<p>Stocks were raised at 25&#176;C on standard cornmeal medium. <italic>Shep</italic> P-element insertion alleles, <italic>shep</italic> deficiencies, <italic>Act5C</italic>::Gal4, <italic>Mef2</italic>::Gal4, and <italic>l(3)31-1</italic>::Gal4 were obtained from the Bloomington and Exelixis Stock Centers. Lines expressing <italic>su(Hw)</italic> (10724 GD) or <italic>shep</italic> dsRNA (37863 GD) were obtained from the Vienna <italic>Drosophila</italic> RNAi Center. UAS::luciferase constructs were inserted into the <italic>attP3</italic> landing site <xref ref-type="bibr" rid="pgen.1003069-Markstein1">[38]</xref>. The <italic>ct<sup>6</sup></italic> phenotype was scored in flies on the first day after eclosion. For all genotypes, males show a more severe wing notching phenotype than females. The <italic>y<sup>2</sup></italic> phenotype was scored in flies aged for 1 d at 25&#176;C. Larvae for luciferase insulator assays and whole mount immunofluorescence were raised at 25&#176;C. Larvae for polytene chromosome staining were raised at 18&#176;C. Embryos aged 0&#8211;24 h for nuclear extracts and immunofluorescence were collected from a population cage as described <xref ref-type="bibr" rid="pgen.1003069-Lei1">[20]</xref>. Anterior thirds of larvae were used for Western blotting.</p><p>Coding regions of FlyBase annotated Shep isoforms RA, RB/RD, and RE were amplified by PCR from Trizol (Invitrogen) extracted, Oregon R embryonic cRNA that was oligo-dT primed and reverse transcribed by Superscript III (Invitrogen). The isoform RA clone obtained differs from FlyBase annotations in 2 locations where either a downstream splice site was used, as in isoforms B, D, and E, resulting in 12 extra amino acids and an additional unannotated exon was included resulting in an additional 11 amino acids; RRM domains remain intact in this isoform A variant. The <italic>shep</italic>, <italic>su(Hw)</italic>, and <italic>mod(mdg4)2.2</italic> cDNAs were inserted into pENTR/D-TOPO and recombined into pDEST 15 (Invitrogen) to generate N-terminal GST fusion constructs. All plasmids were sequenced for verification.</p><p>Expression of GST, GST-Shep, GST-Su(Hw), GST-Mod(mdg4)2.2, His-Su(Hw), and His-Mod(mdg4)2.2 was induced in <italic>E. coli BL21</italic> cells by 1 mM Isopropyl &#946;-D-1-thiogalactopyranoside at 37&#176; C. Proteins were purified under native conditions by affinity using Glutathione-Agarose (Pierce) or Ni-NTA-Agarose (Qiagen). 2 &#181;g immobilized GST or GST fusion proteins were incubated with 35 &#181;g soluble His-Mod(mdg4)2.2 in PBSMT (137 mM NaCl, 2.7 mM KCl, 10 mM NaH<sub>2</sub>PO<sub>4</sub>, 1.8 mM KH<sub>2</sub>PO<sub>4</sub>, 250 mM MgCl<sub>2</sub>, 0.3% Triton X-100 supplemented with Complete protease inhibitors (Roche), 1 mM PMSF, 1 mM DTT, and 100 mg/mL BSA) in a volume of 350 &#181;L. Binding reactions were carried out at 4&#176;C rotating for 2 h. Unbound protein was removed, and beads were washed 5 times in PBSMT. Bound His-Mod(mdg4)2.2 was eluted in sample buffer, separated by SDS-PAGE, and detected by Western blotting. His-Su(Hw) binding reactions were carried out in the same manner except in HBSM (50 mM HEPES, pH 6.7; 150 mM NaCl; 5 mM KCl; 2.5 mM MgCl<sub>2</sub>) supplemented with 0.3% Triton-X 100, 0.2 M KCl, protease inhibitors, 1 mM PMSF, 1 mM DTT, and 100 mg/mL BSA.</p><p>6X-His-Shep isoform E (AA169&#8211;368), CP190 antigen <xref ref-type="bibr" rid="pgen.1003069-Pai1">[4]</xref>, Su(Hw) antigen <xref ref-type="bibr" rid="pgen.1003069-Moshkovich2">[41]</xref> and Mod(mdg4)2.2 antigen <xref ref-type="bibr" rid="pgen.1003069-Mongelard1">[47]</xref> were expressed in BL21 cells, affinity purified by Ni-NTA-agarose (Qiagen) according to the manufacturer's protocol under denaturing conditions and used to immunize rabbits and guinea pigs using standard procedures (Covance Research Products). For Western blotting, guinea pig &#945;-Shep serum was used at 1&#8758;2000, guinea pig &#945;-CP190 was used at 1&#8758;10,000, guinea pig &#945;-Mod(mdg4)2.2 <xref ref-type="bibr" rid="pgen.1003069-Moshkovich2">[41]</xref> was used at 1&#8758;1000, guinea pig &#945;-Su(Hw) <xref ref-type="bibr" rid="pgen.1003069-Moshkovich2">[41]</xref> was used at 1&#8758;7500, &#945;-Pc <xref ref-type="bibr" rid="pgen.1003069-Moshkovich1">[32]</xref> was used at 1&#8758;1000, &#945;-E(z) <xref ref-type="bibr" rid="pgen.1003069-Kahn1">[48]</xref> was used at 1&#8758;1000, and &#945;-Pep <xref ref-type="bibr" rid="pgen.1003069-Amero1">[49]</xref> was used at 1&#8758;1000. For insulator body staining, rabbit &#945;-CP190 <xref ref-type="bibr" rid="pgen.1003069-Pai1">[4]</xref> was used at 1&#8758;30,000. The monoclonal &#945;-Elav9AF89 was obtained from the Developmental Studies Hybridoma Bank and used at 1&#8758;1000, and guinea pig &#945;-Shep serum was used at 1&#8758;200 for IF.</p><p>Nuclei from 20 g of embryos were prepared as described <xref ref-type="bibr" rid="pgen.1003069-Lei1">[20]</xref>. Nuclei were lysed in 4 mL HBSM supplemented with 0.3% TritonX-100 (HBSMT), complete protease inhibitors and 1 mM PMSF by dounce homogenization with the B pestle. Extracts were cleared of insoluble material by centrifugation, and half of the supernatant was incubated with pre-immune serum and half with &#945;-Shep serum pre-conjugated to protein A sepharose. IPs were carried out for 1 h at 4&#176;C, rotating. Unbound protein was removed and beads were washed 4 X in HBSMT and 1 X in HBSM. Bound protein was eluted in sample buffer, separated by SDS-PAGE, and detected by Western blotting.</p><p>Polytene chromosome spreads were prepared as described previously <xref ref-type="bibr" rid="pgen.1003069-Gerasimova2">[8]</xref>. Brains and imaginal discs were dissected from at least 5 larvae of each genotype per experiment and stained as described previously <xref ref-type="bibr" rid="pgen.1003069-Moshkovich1">[32]</xref>. Chromosomes and discs were imaged using a Leica DM5000B epifluorescent microscope and captured using OpenLab software.</p><p>Indirect immunofluorescence of mixed stage Oregon R embryos was carried out as described <xref ref-type="bibr" rid="pgen.1003069-DeFalco1">[50]</xref>, <xref ref-type="bibr" rid="pgen.1003069-Patel1">[51]</xref>. Blocked embryos were incubated rotating with primary antibodies overnight at 4&#176;C and secondary antibodies for 2 h at 37&#176;C. After washing, embryos were incubated in DAPI and mounted in 2.5% DABCO (Sigma) in 70% glycerol. Embryos were imaged on a Zeiss 510 confocal microscope.</p><p>Females homozygous for <italic>attP3</italic>::UAS-luciferase transgenes were crossed to Gal4 expressing males; luciferase in individual F1 male larvae was quantified. Any homozygous lethal Gal4 lines were selected against GFP expressing balancer chromosomes. Larvae were collected on dry ice and stored at &#8722;80&#176;C until use, at which time they were homogenized in 30 &#181;L Glo Lysis buffer (Promega) and incubated at room temperature for 10 min. Debris was cleared from extracts by centrifugation, and 20 &#181;L soluble material was dispensed into opaque 96-well plates; the same volume of luciferase reagent (Promega) was added to each well, and plates were incubated in the dark for 10 min. Light emission was quantified using a Spectramax II Gemini EM plate reader (Molecular Devices). Luciferase values were normalized to total protein determined by Bradford assay carried out in parallel. Luciferase values between genotype populations were log transformed to obtain a normal distribution and compared by one-way ANOVA. Tukey HSD <italic>post hoc</italic> tests were used to determine pairwise <italic>p</italic> values between genotypes. For further information including additional Gal4 lines tested, see <xref ref-type="supplementary-material" rid="pgen.1003069.s007">Text S1</xref>.</p><p>BG3-c2 cells were grown in S2 medium (Sigma) supplemented with 10% fetal calf serum and 10 &#181;g/mL insulin. Cells were maintained in monolayer at 25&#176;C.</p><p>Cells were fixed in 1% formaldehyde added directly to cells in culture medium for 10 min at RT with gentle agitation; formaldehyde was quenched by addition of glycine to 0.125 M with gentle agitation for 5 min at RT. 5&#215;10<sup>6</sup> to 10<sup>7</sup> cells were used per IP. Cells were pelleted at 400 rcf and washed twice in ice cold PBS. Cells were resuspended in 1 mL ice cold cell lysis buffer (5 mM PIPES, pH 8, 85 mM KCl, 0.5% NP-40) supplemented with protease inhibitors, and nuclei were released by Dounce homogenization with the B pestle and pelleted by centrifugation at 9190 rcf for 5 min at 4&#176;C. Nuclei and chromatin were further processed as described <xref ref-type="bibr" rid="pgen.1003069-Moshkovich2">[41]</xref>. Chromatin was fragmented to an average size of 300 bp by sonication and validated by agarose gel electrophoresis. Sequencing libraries were prepared according to the standard Illumina ChIP-seq protocol. Highly similar profiles were obtained with two independent &#945;-Shep antibodies; therefore, the antibody (guinea pig) displaying the highest signal to noise ratio was utilized for subsequent analyses. Rabbit &#945;-Su(Hw) <xref ref-type="bibr" rid="pgen.1003069-Gerasimova4">[16]</xref> and rabbit &#945;-Mod(mdg4)2.2 <xref ref-type="bibr" rid="pgen.1003069-VanBortle1">[42]</xref> were used for ChIP-seq. Highly similar profiles were obtained with two independent &#945;-Mod(mdg4)2.2 antibodies <xref ref-type="bibr" rid="pgen.1003069-Moshkovich2">[41]</xref>; therefore, the antibody displaying the highest signal to noise ratio was utilized for subsequent analyses. Libraries were constructed with TruSeq adapters and sequenced on an Illumina HiSeq multiplexed in a single lane. For directed ChIP, quantitative PCR was performed as previously reported <xref ref-type="bibr" rid="pgen.1003069-Moshkovich1">[32]</xref>.</p><p>36 bp reads from the Illumina HiSeq 2000 sequencer were mapped to dm3 chromosomes except chrUextra, using Bowtie v0.12.7 with parameters &#8220;&#8211;best &#8211;strata -m1 -n2 &#8211;tryhard&#8221;. Reads from repetitive regions were removed, and duplicates were removed with MarkDuplicates from Picard 1.49. Peak-calling was performed with SPP <xref ref-type="bibr" rid="pgen.1003069-Kharchenko1">[43]</xref> using default parameters (e.g., FDR&#8202;=&#8202;0.01, z-threshold&#8202;=&#8202;3) with the exception of &#8220;srange&#8202;=&#8202;c(50, 200)&#8221; when calculating binding site characteristics to improve symmetry of the auto-correlation curve. Broad peak regions were added to binding site point positions, and final peaks were merged. Sequence data are deposited in the Gene Expression Omnibus under accession number GSE40797.</p><p>Intersections, classification, and randomizations in the below analyses were performed with pybedtools v0.6 <xref ref-type="bibr" rid="pgen.1003069-Dale1">[52]</xref>, gffutils v0.8, and BEDTools v2.16.2 <xref ref-type="bibr" rid="pgen.1003069-Quinlan1">[53]</xref>.</p><p>Feature classes [TSSs (1 bp transcript start position), CDSs, introns, 5&#8242;UTRs, and 3&#8242;UTRs] were extracted from all annotated isoforms of all annotated genes in FlyBase release 5.33. Intergenic regions were defined as the remainder of dm3. Since a ChIP-seq peak can fall in more than one class, we classified a peak by its highest priority annotation class, where the priorities from highest to lowest are TSS, CDS, intron, 5&#8242;UTR, 3&#8242;UTR, and intergenic.</p><p>Data files containing called peaks were downloaded from GEO and modENCODE <xref ref-type="bibr" rid="pgen.1003069-Celniker1">[54]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1003069-Richter1">[56]</xref> and converted to BED files. Shep peaks that overlapped either a Su(Hw) peak or a Mod(mdg4)2.2 peak by at least one base were filtered out to create a set of non-<italic>gypsy</italic> Shep peaks. Enrichment scores were calculated as follows: For each pairwise comparison between files A and B, the Jaccard statistic (intersection of bp divided by union of bp; as described previously <xref ref-type="bibr" rid="pgen.1003069-Favorov1">[57]</xref>) was computed to obtain the &#8220;actual&#8221; statistic. Then, features in file A were shuffled to a random position on the same chromosome, and the Jaccard statistic was again calculated. After 1000 such shufflings, the actual statistic was divided by the median of the empirical distribution to get an enrichment score, (actual+1)/(median randomized+1), for the comparison. The full enrichment matrix was hierarchically clustered using correlation as a distance metric and complete linkage clustering as implemented in SciPy, with rows clustered identically as columns. Selected rows from the full clustered matrix in <xref ref-type="supplementary-material" rid="pgen.1003069.s004">Figure S4</xref> are shown in <xref ref-type="fig" rid="pgen-1003069-g006">Figure 6D</xref>.</p><p>To assess the possibility of Shep and Mod(mdg4)2.2 binding mutually exclusively to Su(Hw) sites, we created an N&#215;M binary matrix of binding sites where N&#8202;=&#8202;8194 is the number of binding sites containing any of Su(Hw), Shep, or Mod(mdg4)2.2 (using the pybedtools.contrib.plotting.binary_heatmap() function) and M&#8202;=&#8202;3 for the three factors. We then took the set of 1356 Su(Hw) sites with Shep, Mod(mdg4)2.2, or both, and extracted the Shep and Mod(mdg4)2.2 vectors for these sites representing a total of 663 Shep+Su(Hw) and 964 Mod(mdg4)2.2+Su(Hw) sites. There were 271 Shep+Mod(mdg4)2.2 colocalization events in these vectors. We then randomly shuffled the vectors 10,000 times, computing colocalization each time, and obtained a mean of 472 colocalization events with no iteration giving less than 438 colocalization events. Therefore, of the Su(Hw) sites containing either Shep or Mod(mdg4)2.2, the actual Shep+Mod(mdg4)2.2 overlap of 271 suggests Shep, Mod(mdg4)2.2, and Su(Hw) colocalize significantly less often than expected (empirical <italic>p</italic>&lt;1&#215;10<sup>&#8722;4</sup>). These results are also consistent with a hypergeometric test using n&#8202;=&#8202;1356, n1&#8202;=&#8202;663, n2&#8202;=&#8202;964, and m&#8202;=&#8202;271 (<italic>p</italic>&#8202;=&#8202;2.2&#215;10<sup>&#8722;16</sup>).</p><p>The same analysis was performed for 1403 Mod(mdg4)2.2 sites with Su(Hw), Shep, or both as well as 964 Mod(mdg4)2.2+Su(Hw) and 710 Mod(mdg4)2.2+Shep extracted sites. The mean of randomized iterations was 488 and none had less than 454 colocalization events (empirical <italic>p</italic>&lt;1&#215;10<sup>&#8722;4</sup>) and (<italic>p</italic>&#8202;=&#8202;2.2&#215;10<sup>&#8722;16</sup>, hypergeometric test).</p><p>For 1102 Shep sites with Su(Hw), Mod(mdg4)2.2, or both as well as 710 Shep+Mod(mdg4)2.2 and 663 Shep+Su(Hw) extracted sites. The mean of randomized iterations was 427 and none had less than 393 colocalization events (empirical <italic>p</italic>&lt;1&#215;10<sup>&#8722;4</sup>) and (<italic>p</italic>&#8202;=&#8202;2.2&#215;10<sup>&#8722;16</sup>, hypergeometric test).</p>