<p>All procedures performed in the UK were in accordance with the UK Animals (Scientific Procedures) Act 1986, and those performed in the United States were approved by the IACUC in accordance with NIH guidelines.</p><p>Montu mice were identified in a screen where BALB/c male mice were mutagenized using ENU and crossed to C3H/He for screening. Zuma mice were identified in an ENU screen sensitized by Sox10 haploinsufficiency (<italic>Sox10<sup>LacZ/+</sup></italic>), where BALB/cJ mice were mutagenized using ENU and crossed to C57Bl/6J mice for further pedigree analysis. <italic>Trp53</italic> null mice (B6.129S2-Trp53<sup>tm1Tyj/J</sup>, stock #002101) were purchased from JAX Mice. For behavioral studies, mice were housed in facilities with a 12&#8758;12 light&#8758;dark cycle at a temperature of 22&#177;1&#176;C with a 60&#8211;70% humidity. Upon weaning, mice were separated into single-sex littermate groups and food was available <italic>ad libitum</italic>.</p><p>For body weight analysis, each gender/genotype group represented in <xref ref-type="fig" rid="pgen-1003094-g001">Figure 1B</xref> was weighed over 10 weeks (average N&#8202;=&#8202;9 for each <xref ref-type="fig" rid="pgen-1003094-g001">Figure 1B</xref> data point, range N&#8202;=&#8202;2&#8211;19). A Student's t-test was used to compare wild-type to mutant within gender-matched groups at each age. Significance was assessed using the Bonferonni method to correct for multiple testing, with p&lt;0.001 deemed significant. In females, all time points after 1 week were significantly different and in males all time points after 2 weeks were significantly different, with the exception of 8.5 weeks where only 2 mutant mice were weighed (p&lt;0.003).</p><p>Alizarin red and alcian blue staining of skeletons was conducted using standard techniques. Briefly, embryos were skinned and eviscerated, fixed 24 hours in ethanol, then fixed 24 hours in acetone. Staining was performed for 3&#8211;4 hours at 37&#176;C and then 3&#8211;4 days at room temperature. After staining, embryos were rinsed in water and cleared in 1% KOH for 3 hours at room temperature and moved to fresh 1% KOH overnight. After clearing, embryos were serially transferred through 20%/1% KOH, 50% glycerol/1% KOH, and 80% glycerol/1% KOH.</p><p>FACS analysis was completed using standard techniques. Briefly, fetal liver samples were collected in ice cold PBS and dispersed into a single cell suspension with passage through a 21 gauge needle. Samples were incubated for 20 minutes with the addition of 10 &#181;l of each antibody (BD Parmingen Ter119-APC and CD71-FITC), then washed once with PBS before FACS analysis. At each age ANOVA analysis was performed with a post-hoc test to compare selected pairs: (<italic>Rps7</italic>+/+ versus <italic>Rps7<sup>Zma</sup>/+</italic>) and (<italic>Rps7<sup>Zma</sup>/+</italic> versus <italic>Rps7<sup>Zma</sup>/+</italic>; <italic>Trp53<sup>KO</sup>/+</italic>).</p><p>DNA was extracted from mice using standard techniques. For <italic>Mtu</italic>, the mutation was mapped using 11 affected animals and 51 polymorphic MIT makers that were amplified by PCR and visualized on a 4% agarose gel stained with ethidium bromide. Fine mapping was performed with 13 additional SNPs that distinguished between BALB/c and 101. Seventeen known and predicted genes were identified in the <italic>Mtu</italic> critical interval from the Ensembl database (ENSMUSG00000020633, ENSMUSG00000066544, ENSMUSG00000020636, ENSMUSG00000036655, ENSMUSG00000061477, ENSMUSG00000020630, ENSMUSG00000020629, ENSMUSG00000020628, ENSMUSG00000036613, ENSMUSG00000061911, ENSMUSG00000020674, ENSMUSG00000020673, ENSMUSG00000020672, ENSMUSG00000043061, ENSMUSG00000044573, ENSMUSG00000020669, ENSMUSG00000036136). All exons and adjacent splice sites for these 17 genes were sequenced using a BigDye dideoxy-terminator system and analyzed on an ABI3700 sequencer (Applied Biosystems). For <italic>Zma</italic>, DNA from 8 offspring was analyzed using the Illumina GoldenGate assay medium density linkage panel (Illumina, San Diego, CA). The addition of 24 microsatellite markers, from 4 regions with suggestive linkage (Chr 4, 6, 12, 17) in 8 additional affected mice, allowed localization to chromosome 12 (D12Mit182 -D12Mit60). <italic>Rps7</italic> was sequenced as a candidate gene using the same methods as for <italic>Mtu</italic>.</p><p>mRNA was extracted from wild-type and <italic>Rps7<sup>Mtu</sup>/+</italic> brains, and cDNA was prepared using reverse transcriptase (Invitrogen). A high fidelity PCR polymerase (KOD, Novagen) was then employed to amplify the <italic>Rps7</italic> transcript encoding a fusion protein tagged at either the C- or N-terminus with a FLAG epitope, prior to cloning into the pcDNA3.1(+) vector. Constructs were transfected into HeLa cells grown on glass coverslips, before fixation and staining 48 hours later. The primary antibody employed was a rabbit anti-FLAG (Abcam, ab21536, 1&#8758;1000), followed by a fluorescent secondary (1&#8758;200, Molecular Probes). Five independent transfections were performed. Images were captured on a Zeiss LSM 510 confocal microscope.</p><p>For Western blot experiments, DNA constructs encoding wild-type RPS7, RPS7 p.V156G, and RPS7 p.Y177S fused to the FLAG epitope at the N- or C-terminus were used in transient transfection experiments in human embryonic kidney (HEK) 293 cells. Neomycin phosphotransferase II (NPT2) expressed by the cloning vector pcDNA3.1 was used as a control for transfection efficiency. After transfection, proteins were separated on a 12% SDS-PAGE, transferred onto a nitrocellulose Protran membrane (Schleicher and Schuell) and incubated with a rabbit anti-FLAG or anti-neomycin phosphotransferase II (NPT2) antibody for 1 hr. Following three consecutive washes in PBS/Tween, the membrane was incubated with horseradish peroxidase-conjugated goat anti-rabbit Ab (Jackson Immunoresearch) and visualized using a chemiluminescence detection kit (Pierce) and a LAS3000 Image System (Fuji). Equal loading was checked by Ponceau-S red staining of the membranes before Western blot analysis.</p><p>For Northern blots for rRNA processing, RNA was extracted from mouse tissues with Trizol (Invitrogen) according to the manufacturer's protocol. Total RNA was fractionated on formaldehyde-agarose gels and transferred to Gene Screen Plus membrane (NEN). To detect rRNA precursor transcripts, ITS1 probe was prepared by 5&#8242; end-labeling of a 28-mer oligonucleotide (<named-content content-type="gene">GCTCCTCCACAGTCTCCCGTTAATGATC</named-content>) with 25 &#181;Ci of &#947; [32P]-ATP and T4 polynucleotide kinase according to standard protocols. Hybridization was carried out overnight at 42&#176;C in 6&#215; SSPE, 1% SDS, 0.25 mg/ml ssDNA and 5&#215; Denhardt's solution. After hybridization, the blot was washed twice with 1% SDS in 2&#215; SSPE for 30 min at 37&#176;C. Quantitation of signals was obtained by phosphor screen scanning with a STORM PhosphorImager and ImageQuant software analysis (Molecular Dynamics). The experiments were performed on tissues from <italic>Rps7</italic>+/+ and <italic>Rps7<sup>Mtu</sup>/+</italic> littermate mice (2 <italic>Rps7</italic>+/+ and 2 <italic>Rps7<sup>Mtu</sup>/+</italic> for liver, 3 <italic>Rps7</italic>+/+ and 3 <italic>Rps7<sup>Mtu</sup>/+</italic> for brain, each in triplicate). Quantitation is reported as the ratio between 30S and 21S rRNA precursors and was significantly different between genotypes in both brain (p&lt;0.01) and liver (p&lt;0.01) (two-way ANOVA with post hoc t-test).</p><p>For ribosome isolation, cytoplasmic extracts from RPS7 p.V156G transiently transfected HEK-293 cells were fractionated through ultracentrifugation on a sucrose cushion. After 2 hours of centrifugation at 100,000 g, ribosomes and ribosomal subunits were recovered in the pellet and free cytoplasmic proteins were recovered from the supernatant.</p><p>For sucrose gradients separating polysomes and ribosomal subunits, cytoplasmic extracts from <italic>Mtu/+</italic> liver were loaded onto a 10%&#8211;30% linear sucrose gradient containing 30 mM Tris-HCl (pH 7.5), 100 mM NaCl, and 10 mM MgCl<sub>2</sub>. Gradients were centrifuged in a Beckman SW 41 rotor for 5 hr at 37,000 rpm and then the absorbance profile of the gradient was used to evaluate the ratio between the peaks of ribosomal subunits (60S and 40S).</p><p>For whole mount, <italic>Pmel17</italic>-containing plasmid (Riken cDNA clone G370069C13; GenBank Acc: BB766987) was digested with Kpn1 and transcribed with T3 polymerase to generate DIG-labeled <italic>in situ</italic> probes. Hybridizations were performed using published protocols <xref ref-type="bibr" rid="pgen.1003094-Wilkinson1">[61]</xref> with the following modifications. After probe hybridization, Ribonuclease A digestion was omitted, and Tris-buffered saline was used in place of PBS. BM-purple substrate (Roche, Molecular Biochemicals) was used in place of 5-bromo-4-chloro-3-indolyl phosphate/nitroblue tetrazolium. Embryos were photographed using a Zeiss SteREO Discovery V12 microscope with a Zeiss AxioCam camera. Probes to detect <italic>Rps7</italic> by <italic>in situ</italic> hybridization were amplified by PCR and cloned into the pCRII- TOPO vector (Invitrogen) using the following primer pairs: R7_IN_F <named-content content-type="gene">AAGGAAATCGAAGTTGGTG</named-content> and R7_IN_ R <named-content content-type="gene">AATTAACATCCTTGCCTGTG</named-content>. Mouse embryos were fixed in 2&#8211;4% (w/v) paraformaldehyde, cryoprotected with 30% (w/v) sucrose in phosphate buffer before sectioning (10&#8211;20 &#181;m) on a cryostat. <italic>In situ</italic> hybridization was carried out as previously described <xref ref-type="bibr" rid="pgen.1003094-PhanDinhTuy1">[62]</xref>. Briefly, hybridization of sections with <sup>35</sup>S-UTP-labeled RNA probes was carried out in a 50% (v/v) formamide solution at 60&#176;C. Sections were washed in 50% (v/v) formamide, an RNAse A treatment was performed for 30 min at 37&#176;C, and then successively stringent SSC solution washes were performed, with a final wash at 0.1&#215; SSC at 60&#176;C.</p><p>For Nissl staining, adult mice were perfused with 0.9% NaCl and 4% (w/v) paraformaldehyde before the brain was dehydrated in 30% sucrose, sectioned (40 &#181;m) on a freezing microtome, and stained.. For embryonic histology, embryos were fixed in either Bouin's fixative or 4% paraformaldehyde overnight, washed extensively in PBS, and dehydrated in 70% ethanol before sectioning (5&#8211;7 &#181;m paraffin) and hematoxylin and eosin (H and E) staining. For immunohistochemistry, embryos were fixed in 4% paraformaldehyde overnight, washed with PBS, dehydrated in 10% sucrose followed by 20% sucrose, then embedded in Neg-50 (Thermo Scientific) for cryosectioning (14 &#181;m). Antibodies included anti-TRP53 (Leica NCL-p53-CM5p, 1&#8758;200), anti-CC3 (Cell Signaling #9661, 1&#8758;200), and anti-PH3 (Millipore #06-570, 1&#8758;200) and resulting stains were imaged using a Zeiss Observer.D1 microscope. For apoptosis and proliferation experiments, transverse cryo-sections collected at the level of the forelimb from 3 embryos of each genotype, 6 sections per embryo, were counted for CC3+ and PH3+ cells. InStat software (GraphPad Software, Inc.) was used for one-way ANOVA statistical analysis. No significant difference among the three genotypes was observed in PH3+ cell counts (p&#8202;=&#8202;0.902). CC3 counts were significantly different among genotypes (p&#8202;=&#8202;0.0005) and a Tukey-Kramer multiple comparison post test was used for pairwise comparison among all genotypes with the following results: <italic>Rps7</italic>+/+ versus <italic>Rps7<sup>Zma</sup>/+</italic> (p&lt;0.001), <italic>Rps7</italic>+/+ versus <italic>Rps7<sup>Zma</sup>/+</italic>; <italic>Trp53<sup>KO</sup>/+</italic> (P&gt;0.05, no significant difference), <italic>Rps7<sup>Zma</sup>/+</italic> versus <italic>Rps7<sup>Zma</sup>/+</italic>; <italic>Trp53<sup>KO</sup>/+</italic> (p&lt;0.01).</p><p>Behavioral phenotyping was performed on mutant and unaffected sex-matched littermates between 8 and 12 weeks of age. Open-field tests were conducted for 5 minutes in a brightly lit, 60 cm diameter, enclosed white arena (field), and monitored by an automated tracking system to measure the total distance travelled and the amount of time spent in the center versus the border of the field. The elevated plus maze test was conducted with an elevated platform consisting of two enclosed arms and two open arms. An automated tracking system was employed to measure the number of entries into each arm and distance traveled in open and closed arms of the elevated plus maze over 5 minutes. Spontaneous alternation was performed in an enclosed T-maze built from gray plastic (arm dimensions 30 cm&#215;10 cm&#215;30 cm). Each mouse underwent 10 trials with an inter-trial interval of at least 20 minutes. For each trial the mouse was placed in the start arm facing the end wall and allowed to enter a goal arm of its own choice. The mouse was confined in the goal arm for 30 seconds and then put back in the start arm and allowed a free choice of either arm <xref ref-type="bibr" rid="pgen.1003094-Deacon1">[31]</xref>.</p><p>Late gestation (E18.5) embryos were collected, fixed in 4% paraformaldehyde overnight, washed extensively in PBS, and stored in 70% ethanol. Prior to imaging, embryos were rehydrated in a solution of PBS and 0.5% (v/v) Magnevist (Bayer Healthcare Pharmaceuticals, Wayne, NJ), a contrast agent. MRM was performed on specimens using a Bruker 14.1T MR imaging spectrometer (Bruker Biospin, Billerica, MA) using a multi-echo RARE technique <xref ref-type="bibr" rid="pgen.1003094-Hennig1">[63]</xref> with TR/TE&#8202;=&#8202;200/6.9 ms, 8-echoes, and 4 signal averages. The resulting 3D images were acquired in 53 min with an acquisition resolution of 50 microns isotropic. Manual segmentation of the whole brain and 10 major anatomical regions was performed using Amira visualization software (v5.2.2, Visage Imaging Inc., Andover MA, USA) with guidance from an anatomical reference atlas <xref ref-type="bibr" rid="pgen.1003094-Paxinos1">[64]</xref>. The volume of these regions was estimated and 3D rendering with smoothing was performed to generate 3D representations. Student's t-tests were used to compare volumes of each <italic>Rps7</italic>+/+ and <italic>Rps7<sup>Zma</sup></italic>/+ brain region. Significance was assessed using the Bonferonni method to correct for multiple testing, with p&lt;0.005 deemed significant.</p>