<p>Most of the <italic>E. coli</italic> K-12 strains used in this study are derivatives of RW584 (full genotype: <italic>recA730 lexA51</italic>(Def) &#916;<italic>umuDC596</italic>::<italic>ermGT thr-1 araD139</italic> &#916;(<italic>gpt-proA</italic>)<italic>62 lacY1 tsx-33 glnV44 galK2 hisG4 rpsL31 xyl-5 mtl-1 argE3 thi-1 sulA211</italic>
<xref ref-type="bibr" rid="pgen.1003030-Mead1">[52]</xref>. All derivatives were made by standard methods of P1 transduction using P1<italic>vir</italic>
<xref ref-type="bibr" rid="pgen.1003030-Miller1">[53]</xref> (<xref ref-type="table" rid="pgen-1003030-t001">Table 1</xref>). The various alleles were selected by conferring resistance to spectinomycin (20 &#181;g/ml), zeocin (25 &#181;g/ml), chloramphenicol (20 &#181;g/ml) and kanamycin (50 &#181;g/ml) respectively and subsequently confirmed by PCR <xref ref-type="bibr" rid="pgen.1003030-Curti1">[54]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1003030-Baba1">[56]</xref>.</p><p>: <italic>thr-1 araD139</italic> &#916;<italic>(gpt-proA)62 lacY1 tsx-33 glnV44 galK2 hisG4 rpsL31 xyl-5 mtl-1 argE3 thi-1 sulA211</italic>.</p><p>The low-copy-number plasmids used for expression of UmuC variants are derived from pGB2 <xref ref-type="bibr" rid="pgen.1003030-Churchward1">[57]</xref>. Spectinomycin resistant plasmids pRW134, pJM964, pJM963, and pJM952 encode <italic>E.coli</italic> UmuD&#8242; along with wild-type UmuC or F10L, Y11A, and Y11F variants, respectively and the proteins are expressed from the native <italic>umu</italic> promoter <xref ref-type="bibr" rid="pgen.1003030-Vaisman2">[20]</xref>. Ampicillin resistant derivatives were generated by replacing the <italic>Bsp</italic>HI-<italic>Bsp</italic>HI vector fragment encoding resistance to spectinomycin with a <italic>Bsp</italic>HI-<italic>Bsp</italic>HI fragment from pET22b+ encoding resistance to ampicillin. Bacteria harboring plasmids were grown in LB media containing appropriate antibiotics (50 &#181;g/ml spectinomycin, or 100 &#181;g/ml ampicillin).</p><p>Cells were grown overnight at 37&#176;C in Luria&#8211;Bertani (LB) plus spectinomycin. The next morning, the cultures were sequentially diluted 10-fold in eppendorf tubes containing SM buffer <xref ref-type="bibr" rid="pgen.1003030-Sambrook1">[58]</xref>. 10 &#181;l of each serial dilution was then spotted on the surface of a 12&#215;8 cm rectangular LB agar plate (Nunc, ThermoFisher). The plates were irradiated with UV light (254 nm) and incubated overnight at 37&#176;C. Images of the irradiated plates/cultures were captured with a FluorChem HD2 imaging system (Alpha Innotec).</p><p>Cells transformed with the vector plasmid, pGB2, or one of the low-copy number plasmids expressing wild-type pol V or UmuC variants were grown overnight at 37&#176;C in LB media plus spectinomycin. The next day, cultures were diluted 100-fold into 10 ml fresh media and grown at 37&#176;C until they reach an OD<sub>600</sub> of 0.1 (roughly 3 hrs). Cells were centrifuged and resuspended in an equal volume of SM buffer <xref ref-type="bibr" rid="pgen.1003030-Sambrook1">[58]</xref> and transferred to a petri dish. Aliquots were removed and saved as the unirradiated control for the experiments. The culture was irradiated at a UV fluence of &#8764;2 J/m<sup>2</sup> per second and aliquots removed at 20 J/m<sup>2</sup> increments. At least three independent cultures were assayed for each strain and all experiments were performed under yellow light to avoid unwanted photoreactivation.</p><p>For UV survival assays, appropriate serial dilutions (based upon trial assays) were plated on LB agar plates containing spectinomycin and incubated overnight at 37&#176;C. The surviving fraction was determined by dividing the number of viable cells exposed to UV by the number of viable cells in the unirradiated culture. Error bars represent the standard error of the mean (SEM).</p><p>To determine the number of spontaneously arising histidine mutants on the plate, as well as UV-induced mutants, the unirradiated cell culture was seeded on the Davis and Mingioli minimal agar plates <xref ref-type="bibr" rid="pgen.1003030-Davis1">[59]</xref> plus glucose (0.4% wt/vol); agar (1.0% wt/vol); proline, threonine, valine, leucine, and isoleucine (all at 100 &#181;g/ml); thiamine (0.25 &#181;g/ml); and either no histidine, or histidine (1 &#181;g/ml). On the plates containing no histidine, only pre-existing His<sup>+</sup> mutants grew to form colonies. However, on the plates containing 1 &#181;g/ml histidine, 100&#8211;200 His<sup>&#8722;</sup> cells are able to grow on the limiting amount of histidine, so that a viable cell count can be obtained under the exact same conditions where His<sup>+</sup> mutant arise. When &#8764;4&#215;10<sup>7</sup> bacteria were seeded, they grew to form a lawn, concomitantly exhausting the low level of histidine. Spontaneously arising His<sup>+</sup> mutants grew up through the lawn and were counted after 4 days incubation at 37&#176;C.</p><p>To determine the extent of UV-induced mutagenesis, cells that had been irradiated with 20 J/m<sup>2</sup> UV were used for analysis. This UV dose was chosen since even the UV-sensitive strains exhibited minimal cell killing at this exposure. These conditions therefore provide a window to observe UV-induced mutagenesis without the complications associated with differential levels of cell killing in the various strains. The UV-induced mutation frequencies were calculated as previously described <xref ref-type="bibr" rid="pgen.1003030-Sedgwick1">[60]</xref>. This equation not only takes into account the number of mutants spontaneously arising on the low histidine plates, but also any effect of reduced cell viability on the number of pre-existing His<sup>+</sup> mutants in the culture. The data reported in <xref ref-type="fig" rid="pgen-1003030-g004">Figure 4</xref> represent the average number of His<sup>+</sup> mutants from 3 separate experiments (&#177; standard error of the mean [SEM]).</p><p>Cells were grown overnight at 37&#176;C in LB plus appropriate antibiotics. The next morning, cultures were diluted 1&#8758;100 in fresh LB, plus antibiotics and grown with aeration at 37&#176;C until they reached an OD<sub>600</sub> of &#8764;0.5. Cultures were harvested by centrifugation, resuspended in 1&#215; SDS sample buffer (50 mM Tris-HCl [pH 6.8], 10% glycerol, 2.3% sodium dodecyl sulfate [SDS], 0.1% bromophenol blue, 10 mM dithiothreitol), and immediately frozen in dry ice. Cells were lysed by multiple freeze-thaw cycles and boiled for 5 mins at 95&#8211;100&#176;C. Extracts were immediately applied to a 15% SDS-PAGE gel. After separation, proteins were transferred to an Immobilon-P membrane (Millipore) using standard Western blot protocols. The membrane was incubated overnight with a 1&#8758;1000 dilution of mouse monoclonal antibodies raised against the &#945;-subunit of pol III (kindly provided by Charles McHenry, University of Colorado). The membrane was then incubated with secondary anti-mouse alkaline phosphatase conjugated antibodies and visualized using the CSPD-Western light assay (Applied Biosystems). Pictures were captured on a FluorChem HD2 imaging system (Alpha Innotec).</p><p>Wild-type pol V, the UmuC_Y11A variant and pol IV, &#946;-clamp and &#947;-complex were purified as previously described <xref ref-type="bibr" rid="pgen.1003030-Karata2">[46]</xref>. pSOcpd plasmid, containing a unique CPD adduct, was also constructed as previously described <xref ref-type="bibr" rid="pgen.1003030-Karata1">[45]</xref>. All oligonucleotides were synthesized by Lofstrand Laboratories (Gaithersburg, MD) and gel purified prior to use. 5&#8242;-<sup>32</sup>P labeled M13-TT (5&#8242; &#8211; GAT-CGA-TGG-TAC-GGA-CG) primer was annealed to pSOcpd ssDNA templates at a 1.5&#8758;1 molar ratio by heating in an annealing buffer (50 mM Tris-HCl (pH 8), 5 mM MgCl<sub>2</sub>, 50 &#181;g/ml BSA, 1.42 mM 2-mercaptoethanol) for 10 min at 100&#176;C followed by slow cooling to room temperature. 4 mM RecA (New England Biolabs, Ipswich, MA) was incubated with 0.25 &#181;M 48-mer single-stranded oligonucleotide in the presence of 1 mM adenosine 5&#8242;[&#947;-thio]triphosphate (ATP&#947;S, Biolog Life Science Institute, Bremen, Germany) in the 1&#215; reaction buffer [20 mM Tris-HCl pH 7.5, 8 mM MgCl<sub>2</sub>, 8 mM DTT, 80 &#181;g/ml BSA, 4% glycerol] at 37&#176;C for 5 min to form RecA nucleoprotein filament on ssDNA (RecA*). Reaction mixture containing 1 mM ATP, 50 &#181;M dNTPs, 2 nM DNA templates, 100 nM SSB (Epicentre Biotechnologies, Madison WI), 50 nM &#946; clamp, and 5 nM &#947; complex in the 1&#215; reaction buffer was preincubated for 3 min at 37&#176;C. Purified pol V variants (80 nM) were first combined with RecA*(0.25 &#181;M) to form pol V Mut and then added to the reaction mixture. When indicated, Pol V was either substituted, or mixed with indicated amounts of purified pol IV. Reactions were incubated at 37&#176;C for 20 mins and terminated by adding 10 ml of 2&#215; loading buffer [97% formamide, 10 mM EDTA, 0.1% xylene cyanol, 0.1% bromophenol blue]. The products were heat-denatured and resolved by denaturing PAGE (8 M urea, 15% acrylamide), followed by visualization on a Fuji image analyzer FLA-5100.</p>