<p>
<italic>V. cholerae</italic> and <italic>E. coli</italic> strains, and plasmids used in this study are listed in <xref ref-type="supplementary-material" rid="pgen.1003579.s010">Table S1</xref>. ChrII fragments were amplified from N16961 (CVC209) DNA by PCR using Phusion High-Fidelity polymerase (NEB, Beverly, MA). The sequences of primers used for PCR are shown in <xref ref-type="supplementary-material" rid="pgen.1003579.s011">Table S2</xref>. For cloning sequences up to 100 bp, complementary oligonucleotides (IDT, Skokie, IL) were used after annealing the two <xref ref-type="bibr" rid="pgen.1003579-VenkovaCanova3">[29]</xref>. The exact chrII coordinates of each cloned fragment are given in <xref ref-type="supplementary-material" rid="pgen.1003579.s010">Table S1</xref>.</p><p>This was done in L broth cultures of <italic>E. coli</italic> strain BR8706 at OD<sub>600</sub> between 0.4&#8211;0.5, as described <xref ref-type="bibr" rid="pgen.1003579-Pal1">[24]</xref>. To account for any effect that ParB2 might have on the replication of the <italic>lacZ</italic>-reporter plasmid, <italic>&#946;</italic>-galactosidase activities were normalized for the plasmid copy number in all cases. The copy number variation was small; one standard deviation was within 20% of the mean. The copy numbers were measured (see below) from aliquots of the same cultures that were used for <italic>&#946;</italic>-galactosidase measurements. Some of the cultures were simultaneously monitored for ParB2 amounts by Western blotting (<xref ref-type="supplementary-material" rid="pgen.1003579.s002">Figure S2</xref>). Note that +/&#8722; ParB2 refer to cells carrying pTVC501 (that carries <italic>parB2</italic> under IPTG control) with and without IPTG induction, respectively. In <xref ref-type="fig" rid="pgen-1003579-g002">Figure 2</xref> and <xref ref-type="supplementary-material" rid="pgen.1003579.s003">Figure S3</xref> (lanes 4,6), + refers to cells carrying pTVC236, which supplies ParB2 from a constitutive promoter and &#8722; refers to cells carrying the empty vector, pACYC184.</p><p>The copy number of <italic>lacZ</italic>-carrying plasmids (<xref ref-type="fig" rid="pgen-1003579-g002">Figures 2</xref>,<xref ref-type="fig" rid="pgen-1003579-g003">3</xref>, <xref ref-type="supplementary-material" rid="pgen.1003579.s003">S3</xref>, <xref ref-type="supplementary-material" rid="pgen.1003579.s009">S9</xref>) were measured exactly as described <xref ref-type="bibr" rid="pgen.1003579-Das1">[32]</xref>. Briefly, different experimental cultures were grown to log phase and mixed with separately grown cells carrying pNEB193 before plasmid isolation. The latter plasmid helped to account for plasmid loss, if any, during plasmid isolation steps. The copy number of <italic>oriII</italic> plasmids (<xref ref-type="fig" rid="pgen-1003579-g007">Figure 7</xref>) was determined similarly except that cells instead of growing in liquid cultures were obtained by washing out colonies from transformation plates directly, to avoid mutant accumulation, as described <xref ref-type="bibr" rid="pgen.1003579-VenkovaCanova1">[26]</xref>. The origin fragments were first cloned in a plasmid vector driven by the <italic>&#947;</italic>-origin of plasmid R6K, and the clones were maintained in cells that supplied the cognate initiator (&#960;) protein. The clones were electroporated into <italic>E. coli</italic> (BR8706) carrying pTVC499 that supplied RctB (but no &#960; protein) and pTVC501 that supplied <italic>parB2</italic>.</p><p>The DNA probes were made from plasmids by PCR using oligonucleotides TVC286 (<named-content content-type="gene">5&#8242;-TCCGATTACGGCACCAAATCGA-3&#8242;</named-content>) and TVC287 (<named-content content-type="gene">5&#8242;- AACGTGGATAAACTTCCTGTAAT-3&#8242;</named-content>), which allowed amplification of extra 100 bp of vector sequences from each flank of the region of interest. The PCR products were labeled using 30 units T4 Polynucleotide Kinase (NEB) and 50 &#181;Ci of [&#947;<sup>32</sup>-ATP] (Perkin-Elmer) and purified by passing through G-50 columns (Roche diagnostics). Binding was done in the presence of 300 ng poly dI-dC. Other details are as described <xref ref-type="bibr" rid="pgen.1003579-Duigou1">[25]</xref>, except that the binding reactions were run in 0.5&#215;TBE, which improved ParB2 binding. In <xref ref-type="fig" rid="pgen-1003579-g005">Figure 5</xref>, the probe was non-radioactive and was visualized with SYBR Gold nucleic acid gel stain (Molecular Probes) at 0.5 mg/ml for 30 min. As non-specific competitor, supercoiled pUC19 DNA was used instead of poly dI-dC, as the former stayed at the top of gels and did not interfere with visualization of probe bands. The images were recorded using Fuji LAS-3000 imaging system.</p><p>ChIP assay was performed as described <xref ref-type="bibr" rid="pgen.1003579-Demarre1">[50]</xref>. Briefly, cells of <italic>V. cholerae</italic> CVC209 were cultivated in L broth at 37&#176;C to exponential phase and cross-linked with 1% formaldehyde. After cell lysis and sonication, RctB-DNA or ParB2-DNA complexes were immunoprecipitated using RctB or ParB2 antibody, respectively. The precipitated DNA was amplified, labeled and hybridized to a custom Agilent 8 X 60K <italic>V. cholerae</italic> oligonucleotide microarray representing the whole genome according to the manufacturer's protocol and as described <xref ref-type="bibr" rid="pgen.1003579-Cam1">[51]</xref>. The custom tiling array contained 60 bp probes specific for both the Crick and Watson strands. The consecutive probes were separated by 140 bp in each strand and by 10 bp between the Watson and Crick strands. Data was extracted using an Agilent scanner and Agilent Feature Extraction program. Individual ChIP (Cy5) and input (Cy3) signals were first normalized with respect to total Cy5 and Cy3 signals, respectively. Fold change was calculated by dividing the normalized Cy5 signals with normalized Cy3 signals. The values are mean from three independent experiments.</p><p>The effect of roadblock to ParB2 spreading was tested by cloning an array of five consensus P1 plasmid iterons in front of chrII origin of pTVC20, resulting in pBJH218 (<xref ref-type="fig" rid="pgen-1003579-g008">Figure 8</xref>). Consensus iterons were used to avoid the P<italic>repA</italic> promoter present within the array of natural iterons of P1<italic>ori</italic>. To clone the iterons, pTVC20 was modified by creating a <italic>Nde</italic>I restriction enzyme site between <italic>parS2-B</italic> and the 39-mer within <italic>rctA</italic>, using QuikChange II XL site-directed mutagenesis kit (Agilent Technologies) and oligonucleotides BJH472 and BJH473. To the resulting plasmid (pBJH217), the iteron array, amplified from pALA753 <xref ref-type="bibr" rid="pgen.1003579-Brendler1">[52]</xref> using oligonucleotides BJH475 and BJH476, was cloned at the <italic>Nde</italic>I site, resulting in pBJH218. The plasmid pair, pTVC20 and pBJH218, was used in the same genetic background as used in <xref ref-type="fig" rid="pgen-1003579-g007">Figure 7</xref>, and additionally, in an otherwise isogenic host that supplied P1RepA protein from a constitutive promoter (<italic>bla-p2</italic> of pBR322). The P1<italic>repA</italic> gene and the adjoining <italic>bla-p2</italic> promoter was cloned in a &#955;D69 vector and the resulting phage (&#955;DKC311 <xref ref-type="bibr" rid="pgen.1003579-Park1">[53]</xref>) was used to lysogenize BR8706.</p><p>The copy numbers of pTVC20 and pBJH218 were nearly identical, about four-fold lower than pTVC22 used in <xref ref-type="fig" rid="pgen-1003579-g007">Figure 7</xref>, as was found earlier for pTVC20 <xref ref-type="bibr" rid="pgen.1003579-VenkovaCanova3">[29]</xref>. The copy number was estimated to be about one per cell when there should be four <italic>oriC</italic>. The low copy number of pTVC20 and pBJH218, and the lack of an active partitioning system, made the plasmids unstable (130/130 cells lost the plasmids after seven generations of growth without selection). The effect of ParB2 was therefore checked only under selection. Young colonies (&#8804;1 mm) from selection plates were used to inoculate LB medium with appropriate drugs (ampicillin at 50 &#181;g/ml, chloramphenicol at 25 &#181;g/ml and spectinomycin at 40 &#181;g/ml) and inducers (arabinose at 0.02% and, when desired, IPTG at 100 &#181;M), and the cultures were grown to early log phase (OD<sub>600</sub>&#8764;0.1) and stored in 0.01 M MgSO<sub>4</sub> in ice. For calculating generation times, the cultures were diluted to OD<sub>600</sub>&#8202;=&#8202;0.002 and grown to early log phase. Generation times were calculated from OD<sub>600</sub> values in the range 0.02&#8211;0.2. Saturation of growth was avoided to prevent accumulation of faster growing revertants. Relative colony sizes were also determined from the MgSO<sub>4</sub> suspensions on plates with and without IPTG but otherwise identical in volume and contents.</p>