<p>All human research was approved by the relevant institutional review boards, and conducted according to the Declaration of Helsinki. The Framingham Heart Study protocol was approved by the institutional review board (IRB) of the Boston University School of Medicine and all participants provided written, informed consent. The VISP study protocol was approved by the IRBs of the Wake Forest University School of Medicine (coordinating center) and University of North Carolina Chapel Hill School of Medicine (statistical center). The local IRB for each of the individual recruiting sites approved the VISP protocol and all participants provided written, informed consent. The Genomics and Randomized Trial Network (GARNET) analysis of the VISP data was approved by University of Virginia School of Medicine IRB.</p><p>FHS started in 1948 for evaluation of cardiovascular diseases and risk factors <xref ref-type="bibr" rid="pgen.1004214-Dawber1">[43]</xref>. In 1971, 5124 children of the original cohort, and spouses of these children, referred to as Offspring cohort, were enrolled and have been examined approximately every four years <xref ref-type="bibr" rid="pgen.1004214-Feinleib1">[44]</xref>. Genotyping was performed on the Affymetrix 500K mapping array and the Affymetrix 50K supplemental array. Circulating homocysteine levels were measured on 3465 Offspring participants (N&#8202;=&#8202;3464 with tHcy and N&#8202;=&#8202;2999 with POST) during examination cycle 6 (1995&#8211;1998). The study sample for GWAS consists of a subset of 3110 individuals with at least one phenotype and GWAS data (N&#8202;=&#8202;3108 with tHCY, N&#8202;=&#8202;2711 with POST, N&#8202;=&#8202;2710 with &#916; POST). The sample used to examine the association between the SNPs identified in GWAS of homocysteine phenotypes and incident stroke includes 4176 original cohort and Offspring individuals (N stroke&#8202;=&#8202;200 stroke; N ischemic stroke&#8202;=&#8202;168).</p><p>Plasma tHcy levels were measured using high-performance liquid chromatography with fluorescence detection <xref ref-type="bibr" rid="pgen.1004214-Araki1">[45]</xref>.</p><p>Clinical stroke was defined as rapidly developing signs of focal neurologic disturbance of presumed vascular etiology lasting more than 24 hours. Additional details of stroke classification and diagnosis can be found in prior publications <xref ref-type="bibr" rid="pgen.1004214-Araki1">[45]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004214-Wolf1">[48]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Imputation of about 11 million 1000 Genomes SNPs (1000G Phase I Integrated Release Version 3 Haplotypes: 2010&#8211;11 data freeze, 2012-03-14 haplotypes) was performed using MACH version 1.0.16 (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MACH/">http://www.sph.umich.edu/csg/abecasis/MACH/</ext-link>) based on 412,053 good quality SNPs (excluded SNPs were characterized by call rate &lt;97%, pHWE&lt;1E-6, Mishap p&lt;1e-9, &gt;100 Mendel errors, MAF&lt;1%).</p><p>Prior to association analysis, homocysteine phenotypes were normalized by replacing its observed value with the corresponding quantile under normal distribution. For GWAS, linear mixed effects models were fitted with the transformed phenotypes as dependent variables, individual SNP genotype as a fixed effect, and person specific random effects with correlation coefficient between two individuals being twice their kinship coefficient to account for correlation within extended families <xref ref-type="bibr" rid="pgen.1004214-Chen1">[49]</xref>. FHS GWAS has adjusted for age, sex and first 10 eigenstrat principal components in the linear mixed effects mode (<xref ref-type="supplementary-material" rid="pgen.1004214.s007">Table S2</xref>). Cox proportional hazard model with a robust variance to account for familial relationship was fitted to relate SNPs identified in GWAS with stroke outcomes <xref ref-type="bibr" rid="pgen.1004214-Therneau1">[50]</xref>.</p><p>Imputation was performed using all SNPs and samples passing basic quality filters. In brief, SNPs were selected using the recommended composite quality filter that emerged from the genotype data cleaning process. Samples were selected to have an overall missing call rate &lt;2%, while certain sample-chromosome combinations were also excluded where a gross chromosomal anomaly was detected or when the chromosome-specific missing call rate was &gt;5%. These study data were imputed to a phase 1 interim release from the 1000 Genomes (1000G) Project <xref ref-type="bibr" rid="pgen.1004214-Altshuler1">[51]</xref>. Imputation target variants were defined as those with MAF&#8805;0.005 across all 629 1000G samples.</p><p>Imputation was carried out using BEAGLE imputation software <xref ref-type="bibr" rid="pgen.1004214-Browning1">[52]</xref> (v3.3.1) for chromosomes 1&#8211;22 and the X chromosome. The imputed dataset contained total 7,500,450 variants; 766,577 of which (10.2%) were observed from the array genotyping. In addition to the primary imputation analysis, additional imputations were run on chromosome 22 and the X chromosome, masking a random 10% of observed SNPs to empirically assess imputation quality. The squared correlation between observed and imputed allelic dosages (dosage r<sup>2</sup>) was used to summarize the imputation quality. The median dosage r<sup>2</sup> was 0.933 for chromosome 22 masked SNPs and 0.930 for X chromosome masked SNPs. The imputed dataset, along with a detailed report on imputation methodology, is available through the authorized access portion of the VISP dbGaP posting.</p><p>Genotyping was performed on the Illumina HumanOmni1-Quad-v1 array (Illumina, Inc.) at the Center for Inherited Disease Research, Johns Hopkins University. The genome-wide association analysis was conducted using PLINK v1.0.7. Multivariate linear regression model was used to test correlation of quantitative traits and SNP markers. Using the KING software, the top 10 principle components were derived from genotype data <xref ref-type="bibr" rid="pgen.1004214-Manichaikul1">[53]</xref> and subsequently used to adjust for population heterogeneity, in addition, age and gender were also included as covariates in the model. The VISP population consists of genetically confirmed unrelated individuals and no adjustments were made to the analysis for relatedness. To normalize phenotypic traits, inverse normal transformation was applied to values of POST and &#916;POST. The same regression model was employed to perform association tests between the phenotype and expected allele counts.</p><p>Meta-analysis of the 2100 VISP and 3110 FHS cohorts was conducted using the METAL software <xref ref-type="bibr" rid="pgen.1004214-Willer1">[54]</xref>. The sample size of each study was used as weight, and the sign of the beta value of each SNP coded allele was used as the direction for association (<xref ref-type="supplementary-material" rid="pgen.1004214.s008">Table S3</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<xref ref-type="fig" rid="pgen-1004214-g001">Figure 1</xref> regional association plots were created using the locus zoom &#8220;plot your own data&#8221; function (<ext-link ext-link-type="uri" ns0:href="https://statgen.sph.umich.edu/locuszoom/genform.php?type=yourdata">https://statgen.sph.umich.edu/locuszoom/genform.php?type=yourdata</ext-link>). Plots were created utilizing the genome build/LD population hg19/1000 Genomes Mar 2012 EUR.</p><p>Unless otherwise noted, HepG2 cell lines were cultured in complete media containing high glucose Dulbecco's Modified Eagle's Medium (DMEM) (Invitrogen) with 10% (v/v) FBS, 2 mM L-glutamine supplemented with 1&#215; nonessential amino acids, 1 mM sodium pyruvate (Invitrogen) and 1X antibiotic-antimycotic (Invitrogen). Cells were maintained at 37&#176;C in a 5% CO<sub>2</sub> incubator.</p><p>HepG2 cells were culture in complete media as described above for 24 hrs in 25 cm<sup>2</sup> tissue culture flasks or 6 well culture plates. For methionine starvation, after 24 hrs media was removed and replaced with either complete media or complete media lacking L-methionine. After 24 hrs, cells starved of methionine were supplemented with L-methionine (Ameresco) at concentrations of (0.2 mM, 0.4 mM, 0.8 mM).</p><p>RNA isolation was conducted using the Qiagen RNeasy kit according to standard manufacture's protocols. cDNA synthesis was conducted using 1 ug of total RNA and the Verso cDNA Synthesis Kit (Fisher Scientific). A 3&#8758;1 mix (v/v) of random hexamers and anchored oligo-dT was used following standard theromocycling conditions. For <italic>GNMT</italic> quantitative real-time PCR, Taqman MGB probes and primers were used (Hs002219089) (Applied Biosystems). All samples were run in triplicate in 10 &#181;l reaction volumes. PCR conditions were the default settings of the ABI Prism 7900 HT Sequence Detection System (Applied Biosystems) using the standard curve setting to achieve raw data, which was analyzed in Microsoft Excel. The cycle threshold (Ct) was determined during the geometric phase of the PCR amplification plots as automatically set by the 7900 software. Relative differences in transcript levels were quantified using the &#916;&#916;Ct method with <italic>GAPDH</italic> [MIM 138400] (probe 4333764F) mRNA as an endogenous control.</p><p>
<italic>GNMT</italic> promoters were PCR amplified using primers (Forward: <named-content content-type="gene">5&#8242;-CGGGGTACCACAGAGCGAGACTGTGTC-3&#8242;</named-content>, Reverse: <named-content content-type="gene">5&#8242;-GCGAGATCTCCTGCGCCGCGCCTGGCT-3&#8242;</named-content>) as previously described <xref ref-type="bibr" rid="pgen.1004214-Lee1">[14]</xref>. One VISP case was chosen for TA cloning from the high &#916;POST and low haplotype &#916;POST haplotypes. Promoters were cloned into the StrataClone PCR Cloning vector according to the standard protocols (Agilent Technologies). Promoters were next restriction digested and ligated into the pGL3 luciferase plasmid (Promega) using KpnI and SacI enzymes (New England Biolabs) giving rise to GNMT<sup>&#916;HighLuc</sup> and GNMT<sup>&#916;LowLuc</sup>.</p><p>Haploview version 4.2 <xref ref-type="bibr" rid="pgen.1004214-Barrett1">[13]</xref> was used to analyze haplotype blocks on chromosome 6 from the VISP population. 10 SNPs with p&#8804;5&#215;10<sup>&#8722;8</sup> were assessed using version 3, release 27, analysis panel CEU+TSI.</p><p>Average &#916;POST values were taken from the VISP population, and any individuals with missing data from 1 or more SNPs were excluded from the analysis. The top two haplotypes, encompassing 80% of the total VISP population were assessed.</p><p>2&#215;10<sup>4</sup> HepG2 cells were transfected in quadruplicate with GNMT<sup>&#916;HighLuc</sup> or GNMT<sup>&#916;LowLuc</sup> and pGL4.74[hRluc/TK] (Promega) at a 10&#8758;1 ratio using TransIt-LT1 transfection reagent (Mirus biosciences) in 96 well plates. Luciferase assays were conducted following the Dual-glo kit standard protocol (Promega). Luciferase readings were taken using the Beckman Coulter DTX880 luminometer at a 1 second integration time. Firefly luciferase measures from GNMT<sup>&#916;HighLuc</sup> or GNMT<sup>&#916;LowLuc</sup> were taken for each well, followed by treatment for renilla luciferase activity and renilla measurement. Relative luciferase activity of each promoter was calculated by dividing the average firefly luciferase counts from GNMT<sup>&#916;HighLuc</sup> or GNMT<sup>&#916;LowLuc</sup> by pGL4.74[hRluc/TK] for each independent condition. L-Methionine treatment used 0.2 mM reagent. Total relative luciferase activity for each plasmid encompasses the average of 3 biological replicates.</p><p>Pyrosequencing was performed as previously described <xref ref-type="bibr" rid="pgen.1004214-Jack1">[55]</xref>. Primers are as follows, Forward: <named-content content-type="gene">AGTAGAGAAGTGTTAGTTAGGTTTTAT</named-content>, Reverse (Biotin labeled): <named-content content-type="gene">ACCCATACAAAAAAAACAAAAAAAATCTC</named-content>, Sequencing primer: <named-content content-type="gene">TTTGGATTAGGTGGATAG</named-content>.</p><p>Scores were determined by using imputation dosage measures from VISP and FHS. Alleles were assessed for the average &#916;POST values in FHS and VISP. If a dosage for a homozygous SNP was associated with high homocysteine on average (i.e. close to the value 2) the number was not changed. However, if the homozygous allele was not associated with the risk variant (i.e. close to 0) but was represented as a number above 1.5, 2 was subtracted from that number and made positive. After correction, homozygous risk variants would have a dosage value near 2, heterozygous variants would have a value near 1, and non risk variants would be assigned a number near 0. All imputation dosage values were summed. The sum of each risk value was then taken for each individual to give a score from 0 to a possible 10, and that score was rounded to the nearest integer.</p><p>For calculation of the variance explained by the risk score, linear regression was used for VISP, and a linear mixed effects model was used for FHS, as these methods were used in the initial GWAS for each study.</p><p>All statistics for <xref ref-type="fig" rid="pgen-1004214-g002">Figures 2</xref>&#8211;<xref ref-type="fig" rid="pgen-1004214-g004">4</xref> were performed using GraphPad Prism 4. Tests used are indicated in Figure legends. Significance threshold was set at p&#8202;=&#8202;0.05.</p>