<p>The Colombia-0 ecotype of <italic>Arabidopsis thaliana</italic> was used for all experiments. The <italic>35S:6&#215;His-PIF3-5&#215;MYC</italic> (P3M) transgenic line <xref ref-type="bibr" rid="pgen.1003244-AlSady2">[48]</xref>, <italic>pif3</italic>
<xref ref-type="bibr" rid="pgen.1003244-Monte1">[18]</xref>, <italic>pif1pif4pif5</italic> (<italic>pif145</italic>) <xref ref-type="bibr" rid="pgen.1003244-Leivar3">[11]</xref>, <italic>pif1pif3pif4</italic> (<italic>pif134</italic>), <italic>pif1pif3pif5</italic> (<italic>pif135</italic>), <italic>pif3pif4pif5</italic> (<italic>pif345</italic>), and <italic>pif1pif3pif4pif5</italic> (<italic>pifq</italic>) <xref ref-type="bibr" rid="pgen.1003244-Leivar2">[2]</xref> were as described.</p><p>Stratified seeds were irradiated with WL at 21&#176;C for 3 h to induce germination, followed by a FR pulse for 15 min to suppress pseudo dark effects <xref ref-type="bibr" rid="pgen.1003244-Leivar2">[2]</xref>, and grown in darkness at 21&#176;C for 2 d before harvest.</p><p>The ChIP assay was performed using about 2 g of <italic>Arabidopsis</italic> 2-d-old dark-grown whole seedlings as described <xref ref-type="bibr" rid="pgen.1003244-Gendrel1">[49]</xref> under green safelight. Polyclonal anti-MYC antibodies (Abcam, ab9132) were used with BSA-blocked Protein G Agarose beads (Millipore) to immunoprecipitate the P3M-DNA complex. Wild-type <italic>Arabidopsis</italic> seedlings grown under the same conditions were used as the negative control following the same assay procedure.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The ChIP-seq library was constructed according to Illumina's instructions (<ext-link ext-link-type="uri" ns0:href="http://www.illumina.com">www.illumina.com</ext-link>) with some modifications. Four ChIP samples from technical replicates of each biological replicate were pooled together and concentrated to increase the starting amount of DNA. The end repair of DNA fragments was performed using End-It DNA End-Repair Kit (Epicentre). The A-tailing was added to the end-repaired DNA fragments using Klenow Fragment (NEB), and then Illumina's PE adapters were ligated by T4 DNA Ligase (Promega) at 16&#176;C overnight. The adapter-ligated DNA fragments in the 200&#8211;300 bp size-range were selected by the gel purification, and then were amplified using Phusion High-Fidelity DNA Polymerase (NEB) with the Illumina PE PCR primer set. The library was purified using an Agencourt AMPure XP system (Beckman Coulter Genomics), and then validated by Bioanalyzer 2000 (Agilent).</p><p>The parallel libraries from P3M and WT ChIP samples were assayed by single-end sequencing on an Illumina GAIIx platform. The 36-nt reads were aligned to the TAIR9 assembly of the <italic>Arabidopsis</italic> genome using Bowtie <xref ref-type="bibr" rid="pgen.1003244-Langmead1">[50]</xref> with up to 2 mismatches allowed. Only reads mapped uniquely to the nuclear genome with the lowest number of mismatches were retained for binding-peak identification. To increase the uniformity of read-counts across biological replicates, two technical-replicate sequencing runs were performed on the 4 libraries from the 1st and 2nd ChIP experiments (two of the four biological replicates). The aligned reads from the two technical sequencing replicates of each library were combined and processed as single biological replicate data.</p><p>The statistical identification of PIF3-binding peaks was performed separately for each biological replicate using MACS <xref ref-type="bibr" rid="pgen.1003244-Zhang2">[35]</xref> with the default 10<sup>&#8722;5</sup> P-value cutoff. MACS analysis was customized to ensure a more uniform analysis across biological replicates, and to decrease the size of the window for detecting background enrichment (due to the small size of the <italic>Arabidopsis</italic> genome) by employing modified parameters (gsize&#8202;=&#8202;1.1e8, bw&#8202;=&#8202;100, nomodel, shiftsize&#8202;=&#8202;50, slocal&#8202;=&#8202;1000, and llocal&#8202;=&#8202;2000).</p><p>Four independent biological replicates of ChIP-seq data were collected, and replicate-specific binding peaks, identified in at least one other replicate, were defined as reproducible, if the distance between the summits of each replicate were less than 100 bp. For each reproducible peak, a mean summit position was assigned as the average position of the individual replicate-specific summits, and the PIF3-binding sites were defined as the 201 bp windows centered at each reproducible mean-summit position.</p><p>
<italic>De novo</italic> PIF3-binding motif discovery was performed on the 201-bp defined binding sites using MEME <xref ref-type="bibr" rid="pgen.1003244-Bailey1">[36]</xref>, and the enrichment significance of identified G-box and PBE-box motifs beyond the genome background was quantified by 100 random simulations, where in each simulation 1064 randomly selected genomic regions of the same size were searched for the occurrence of each motif. The tight association of PIF3 binding with a specific motif was defined as the distance between the peak summit and the closest motif less than 100 bp.</p><p>Definition of the closest neighboring genes to each binding peak was approached by scanning the regions within +/&#8722;5 kb centered at each peak summit, using CisGenome <xref ref-type="bibr" rid="pgen.1003244-Ji1">[51]</xref>, and the potential target genes downstream of each summit with no intervening genes were selected manually.</p><p>Total RNA was extracted from 2-d-old dark-grown seedlings using QIAshredder column and RNeasy Plus Mini Kit (Qiagen) according to the manufacturer's instructions. The sequencing library construction was adapted from 3&#8242;-end RNA-seq protocol <xref ref-type="bibr" rid="pgen.1003244-Yoon1">[52]</xref>. The mRNA was fractionated from 20 &#181;g of total RNA using Dynabeads Oligo (dT)<sub>25</sub> (Invitrogen), and fragmented using Fragmentation Reagents (Ambion) at 70&#176;C for 2.5 min in 20 &#181;l of reaction. The polyA-tailed 3&#8242;-end fragments were captured by another run of mRNA purification as described above, and then treated by Antarctic Phosphatase (NEB) and T4 Polynucleotide Kinase (NEB) at 37&#176;C for 1 h and 2 h, respectively. The sample was purified using RNeasy MinElute Cleanup Kit (Qiagen) according to Illumina's protocol. The eluted mRNA fragments were ligated with 2.5 &#181;M of Illumina's SRA 5&#8242; adaptor by T4 RNA Ligase 1 (NEB) at 20&#176;C for 4 h. The 3&#8242; cDNA adapter derived from Illumina's v1.5 sRNA 3&#8242; adapter was conjugated with the anchored oligo (dT)<sub>20</sub> primer, and introduced through reverse transcription using the SuperScript III First-Strand Synthesis System (Invitrogen). The first-strand cDNA was purified using the Agencourt AMPure XP system, and then amplified by PCR reaction using Phusion High-Fidelity DNA Polymerase with Illumina's sRNA PCR primer set. The size of purified library DNA was validated by Bioanalyzer 2000.</p><p>Libraries from the 1st biological replicate were assayed by 36-cycle single-end sequencing on the Illumina GAIIx platform, while libraries from the 2nd and 3rd biological replicates were assayed by 50-cycle single-end sequencing on the HiSeq2000 platform. For consistency, only the 5&#8242;-end 36-nt trimmed reads from the 2nd and 3rd replicates, as well as the full-length 36-nt reads from the 1st replicate, were aligned to the TAIR9 representative transcriptome using Bowtie with zero mismatches allowed. Only reads mapping uniquely to the 3&#8242;-end 500-bp region of the coding strand were counted for gene expression. Differentially expressed genes were identified using the edgeR package <xref ref-type="bibr" rid="pgen.1003244-Robinson1">[53]</xref>, and SSTF genes were defined as those that differ by &#8805;2-fold with an adjusted P value &#8804;0.05 as described <xref ref-type="bibr" rid="pgen.1003244-Leivar4">[17]</xref>.</p><p>The recombinant protein GST-PIF3-Flag and the GST control were purified from <italic>E. coli</italic> as described previously <xref ref-type="bibr" rid="pgen.1003244-MartinezGarcia1">[9]</xref>. DNA probes were generated by annealing a 5&#8242; biotinylated oligonucleotide (IDT) to a complementary unmodified oligonucleotide (IDT). The complementary oligonucleotides were diluted in annealing buffer (10 mM Tris-HCl (pH 7.5), 50 mM NaCl, 1 mM EDTA) to a final concentration of 40 &#181;M, heated to 95&#176;C for 5 min, and cooled down slowly (0.1&#176;C/second) to 12&#176;C. The same procedure was followed to generate unmodified dsDNA fragments for competition assays. Probes are listed in <xref ref-type="supplementary-material" rid="pgen.1003244.s020">Table S13</xref>.</p><p>The DPI-ELISA assays were performed as described <xref ref-type="bibr" rid="pgen.1003244-Brand1">[38]</xref>. Biotinylated probes were bound to Reacti-Bind Streptavidin High Binding Capacity Coated 96-Well Plates (Thermo Scientific) by applying 2 pmol/well of the probes in TBS-T buffer (20 mM Tris-HCl (pH 7.5), 180 mM NaCl, 0.1% (v/v) Tween 20) for 1 h at 37&#176;C. The wells were blocked with 5% (w/v) non-fat dry milk in TBS-T buffer for 30 min, and then incubated with 100 ng of GST-PIF3-Flag or GST for 1 h. For competition assays, 2, 10 or 50 pmol/well of the unlabeled probes were added at the same time with the proteins. After incubation, the wells were washed 3 times with TBS-T/PBS-T buffer, and then were incubated with 1&#8758;2000 diluted THE GST Antibody [HRP] (GenScript, A00866) in PBS-T buffer for 1 h. The wells were then washed twice with PBS-T and PBS buffers, respectively, after incubation. The protein binding was detected by adding the OPD solution (Thermo Scientific), and the reaction was stopped by 2.5 M sulfuric acid. The color extinction was measured at 490 nm, using 650 nm as a reference wavelength in the ELISA reader.</p><p>The EMSA assays were performed as described <xref ref-type="bibr" rid="pgen.1003244-MartinezGarcia1">[9]</xref>. 100 ng of recombinant proteins and the biotinylated DNA probes were used in each assay. Gel electrophoresis using native 5% PAGE gel in ice cold 0.5&#215; TBE buffer (280 V, 15 min) was followed by wet-transfer electro blotting to Biodyne B Nylon membrane (Pierce) in 0.5&#215; TBE buffer (80 V, 1 h). The Lightshift Chemiluminescent DNA EMSA kit (Pierce) was used for detection of the biotinylated probes according to the manufacturer's instructions.</p><p>RT-qPCR was performed as described <xref ref-type="bibr" rid="pgen.1003244-Leivar4">[17]</xref>. Each PCR reaction was repeated at least twice, and the mean value of technical replicates was recorded for each biological replicate. Data from biological triplicates were collected, and the mean value with standard error is represented in the bar graphs. Primers and gene accession numbers are listed in <xref ref-type="supplementary-material" rid="pgen.1003244.s020">Table S13</xref>.</p><p>The 1.8 kb <italic>PIL1</italic> promoter region (<italic>pPIL1</italic>) upstream of the ATG was amplified by PCR using the pPIL1-F1/R1 primer set, and then the <italic>Xho</italic>I/<italic>Eco</italic>RI fragment was cloned into the pBluescript II SK (pBSK) vector (Stratagene) to produce pBSK-pPIL1. The G-box mutations were introduced by two-step PCR amplification (using pPIL1-F7/R7, pPIL1-F8/R8, pPIL1-F9/R9, and pPIL1-F10/R10 primer sets), and the <italic>Xho</italic>I/<italic>Mfe</italic>I fragment from the pPIL1-F1/R5 primer set was cloned to replace the unmutated fragment of pBSK-pPIL1 to produce pBSK-mpPIL1. The <italic>Hind</italic>III/<italic>Bam</italic>HI fragment containing the omega-LUC+-rbcS terminator from the pENTR/D-TOPO\arGIp::LUC+ construct was cloned into pBSK-pPIL1 and pBSK-mpPIL1, respectively, to produce pBSK-pPIL1:LUC and pBSK-mpPIL1:LUC. The CDS of Renilla Luciferase (<italic>RLUC</italic>) was amplified by PCR using the Rluc-F1/R1 primer set, and then the <italic>Nco</italic>I/<italic>Pml</italic>I fragment was cloned into the pCAMBIA1302 binary vector to produce pC1302-35S:RLUC. The <italic>Pst</italic>I/<italic>Sac</italic>I fragments from pBSK-pPIL1:LUC and pBSK-mpPIL1:LUC were then sub-cloned into pC1302-35S:RLUC to produce pC1302-pPIL1:LUC-35S:RLUC (<italic>pPIL1:LUC</italic>) and pC1302-mpPIL1:LUC-35S:RLUC (<italic>mpPIL1:LUC</italic>), respectively.</p><p>The 2.5&#8211;3.0 kb promoter regions upstream of the ATG of <italic>PIF3</italic>, <italic>PIF4</italic> and <italic>PIF5</italic> were amplified from <italic>Arabidopsis</italic> (Col-0 ecotype) genomic DNA by PCR using the TOPO-PIF3p-LP1/RP1, TOPO-PIF4p-LP1/RP1 and TOPO-PIF5p-LP1/RP1 primer sets, respectively. The PCR products were cloned into the pENTR/D-TOPO vector (Invitrogen) to produce the pPIF3, pPIF4 and pPIF5 entry clones. For the <italic>PIF1</italic> promoter, the first 2 kb fragment upstream of the ATG was amplified by PCR using the TOPO-PIF1p-LP3/RP1 primer set, and then was cloned into the pENTR/D-TOPO vector to produce the intermediate entry clone. The second fragment of 2&#8211;4 kb upstream of ATG was amplified using the NotI-PIF1p-LP/XcmI-PIF1p-RP primer set, and then the <italic>Not</italic>I/<italic>Xcm</italic>I fragment of the PCR product was subcloned into the intermediate entry clone to produce the pPIF1 entry clone. All four entry clones were subcloned into the gateway compatible pGWB3 binary vector <xref ref-type="bibr" rid="pgen.1003244-Nakagawa1">[54]</xref> using Gateway LR Clonase II Enzyme Mix (Invitrogen) to produce <italic>pPIF:GUS</italic> constructs.</p><p>The constructs were transformed into <italic>Arabidopsis</italic> plants as described <xref ref-type="bibr" rid="pgen.1003244-Zhang3">[55]</xref>, and the individual transgenic lines were selected on MS medium containing 25 mg/L of Hygromycin B (Invitrogen).</p><p>The 2-d dark-grown seedlings of independent transgenic lines were ground in liquid nitrogen, and total protein was extracted in LUC extraction buffer (1&#215; PBS, 4 mM EDTA, 2 mM DTT, 5% glycerol, 1 mg/ml BSA, 2 mM PMSF and 1&#215; complete protease inhibitor cocktail (Roche) at 3&#215; w/v) as described <xref ref-type="bibr" rid="pgen.1003244-Huq1">[7]</xref>. 20 &#181;l of the supernatant were used to measure the LUC and RLUC activity using a Dual-Luciferase Reporter Assay System (Promega) according to the manufacturer's instruction. The relative expression of <italic>LUC</italic> was represented by its enzyme activity compared to the RLUC internal control.</p><p>Histochemical GUS staining assays were performed on 2-d-old dark-grown seedlings as described <xref ref-type="bibr" rid="pgen.1003244-Somers1">[56]</xref> using a modified substrate buffer (1&#215; PBS (pH 7.0), 1 mM K<sub>3</sub>Fe(III)(CN)<sub>6</sub>, 0.5 mM K<sub>4</sub>Fe(II)(CN)<sub>6</sub>, 1 mM EDTA, 1% Triton X-100, 1 mg/ml X-gluc). Data of biological triplicates were collected from two independent transgenic lines, and representative images are shown for each transgene.</p><p>ChIP-seq and RNA-seq data reported in this study have been deposited in the Gene Expression Omnibus database under the accession number GSE39217.</p>