<p>Pericardial fat was measured on CT using protocols determined by the participating studies, as described in the Study-Specific Methods. Sex-specific residuals were created, with adjustment for age, height, and weight, as well as principal components derived from genotypes denoting population stratification where necessary.</p><p>Heritability of pericardial fat was calculated in the Framingham Heart Study using standard methods. Sex-and-cohort specific residuals were created and then pooled for analysis using variance components analysis (SOLAR) <xref ref-type="bibr" rid="pgen.1002705-Almasy1">[31]</xref>.</p><p>We used SOLAR <xref ref-type="bibr" rid="pgen.1002705-Almasy1">[31]</xref> to calculate pair-wise genetic correlations between pericardial fat, visceral fat, and BMI in the Framingham Heart Study. We used residuals adjusted for age and sex. We tested two separate hypotheses: RhoG&#8202;=&#8202;0 is the test for overlapping genetic correlations, whereas RhoG&#8202;=&#8202;1 is the test for non-overlapping genetic correlations.</p><p>
<xref ref-type="supplementary-material" rid="pgen.1002705.s004">Table S1</xref> and the study specific methods describe the genotyping that was conducted. Quality-control filters were used to exclude low-quality samples or SNPs. Each study imputed &#8764;2.5 million Phase 2 HapMap SNPs based on CEU samples; allelic dosage was used in the analysis.</p><p>Each cohort separately conducted the regression analysis, using an additive genetic effect model with accounting for family structure when necessary. Next, we conducted a fixed effects weighted Z-score meta-analysis given possible differences in phenotype scaling between the participating studies using METAL <xref ref-type="bibr" rid="pgen.1002705-Willer1">[32]</xref>. Statistical significance was considered when SNPs reached a meta-analysis <italic>P</italic> value&#8804;5&#215;10<sup>&#8722;8</sup>
<xref ref-type="bibr" rid="pgen.1002705-Peer1">[33]</xref>. Discovery analyses were performed on European ancestry participants. SNPs were filtered at a minor allele frequency&lt;2% and an imputation quality score&lt;0.3.</p><p>We conducted Stage 2 validation using non-white ethnic samples from the MESA study. Statistical significance was achieved when a direction-consistent p-value was at least p&lt;0.05.</p><p>For our lead SNP, we performed look-up in the publically available GIANT datasets <xref ref-type="bibr" rid="pgen.1002705-Heid1">[12]</xref>, <xref ref-type="bibr" rid="pgen.1002705-Speliotes2">[14]</xref>. We also obtained specific look-up results in the CARDIoGRAM (for coronary heart disease) <xref ref-type="bibr" rid="pgen.1002705-Schunkert1">[16]</xref> and MESA (for coronary artery calcification and carotid intimal medial thickness).</p><p>We also tested whether the 25 previously-identified SNPs for coronary heart disease from the CARDIoGRAM consortium <xref ref-type="bibr" rid="pgen.1002705-Schunkert1">[16]</xref> were associated with pericardial fat. To determine statistical significance, we used the false discovery rate q-value; SNPs with q-value&lt;0.05 were considered statistically significant using the QVALUE package in R <xref ref-type="bibr" rid="pgen.1002705-Storey1">[34]</xref>.</p><p>We tested for a formal sex interaction of rs10198628. Each study computed the interaction regression coefficient, standard error, and p-value. For the sex interaction, we included, age, height, weight, and any principal components (and study center) that were used in the original discovery analysis. We additionally added rs10198628 and the cross-product rs10198628*sex. Interaction terms were meta-analyzed using the weighted z-score approach.</p><p>We calculated the variance explained using the following formula: 2*MAF*(1&#8722;MAF)*(((beta)&#8743;2)/((SD)&#8743;2)).</p><p>We searched for eQTLs in a region bounded by the <italic>LIPN1</italic> and <italic>TRIB2</italic> genes using expression SNP (eSNP) datasets availably publically or via collaboration including lymphocytes <xref ref-type="bibr" rid="pgen.1002705-Goring1">[35]</xref>, leukocytes <xref ref-type="bibr" rid="pgen.1002705-Idaghdour1">[36]</xref>, leukocytes from patients with Celiac disease <xref ref-type="bibr" rid="pgen.1002705-Heap1">[37]</xref>, lymphoblastoid cell lines (LCL) from children with asthma <xref ref-type="bibr" rid="pgen.1002705-Dixon1">[38]</xref>, HapMap LCL from 3 populations <xref ref-type="bibr" rid="pgen.1002705-Stranger1">[39]</xref>, a separate study on HapMap CEU LCL <xref ref-type="bibr" rid="pgen.1002705-Kwan1">[40]</xref>, peripheral blood monocytes <xref ref-type="bibr" rid="pgen.1002705-Heinzen1">[41]</xref>, <xref ref-type="bibr" rid="pgen.1002705-Zeller1">[42]</xref>, subcutaneous and omental adipose tissue <xref ref-type="bibr" rid="pgen.1002705-Emilsson1">[43]</xref>, <xref ref-type="bibr" rid="pgen.1002705-Greenawalt1">[44]</xref>, and blood samples <xref ref-type="bibr" rid="pgen.1002705-Emilsson1">[43]</xref>, 2 studies on brain cortex <xref ref-type="bibr" rid="pgen.1002705-Heinzen1">[41]</xref>, <xref ref-type="bibr" rid="pgen.1002705-Webster1">[45]</xref>, three large studies of brain regions including prefrontal cortex, visual cortex and cerebellum (Emilsson, personal communication), liver <xref ref-type="bibr" rid="pgen.1002705-Greenawalt1">[44]</xref>, <xref ref-type="bibr" rid="pgen.1002705-Schadt1">[46]</xref>, osteoblasts <xref ref-type="bibr" rid="pgen.1002705-Grundberg1">[47]</xref>, skin <xref ref-type="bibr" rid="pgen.1002705-Ding3">[48]</xref>, and additional fibroblast, T cell and LCL samples <xref ref-type="bibr" rid="pgen.1002705-Dimas1">[49]</xref>. Statistical significance was considered using the association with gene transcript levels as originally described.</p><p>Adipose tissue was dissected from male C57Bl/6J mice (Jackson Labs) (n&#8202;=&#8202;4) following 20 weeks of <italic>ad libitum</italic> feeding with a 60% high-fat diet (Research Diets, New Brunswick NJ). High fat feeding was used because many adipose depots (such as pericardial fat) are either not visible or extremely limited in size in standard chow fed mice. Animals were sacrificed and adipose tissues were dissected and frozen in liquid nitrogen. The following adipose depots were dissected: inguinal, axillary, gluteal, brown adipose, epididymal, retroperitoneal, mesenteric, omental, pericardial, and perivascular. All animal experiments were done according to procedures approved by the Institutional Animal Care and Use Committee of Beth Israel Deaconess Medical Center.</p><p>Total RNA was isolated using TRIzol (Invitrogen, Carlsbad, CA) combined with RNeasy mini-columns, (Qiagen, Valencia, CA) according to the manufacturer's instructions. For real-time PCR analysis, cDNA was synthesized from RNA using the high capacity cDNA synthesis kit (ABI, Carlsbad, CA). cDNA was used in quantitative PCR containing SYBR-green dye (ABI). mRNA expression levels for each gene were normalized to TBP. Quantitative PCR was performed using an ABI-7900HT PCR machine.</p><p>In addition to Lipin 1 and Trb2, the expression of the adipocyte markers aP2 and PPAR&#947;2 were also measured. The primer sequences were as follows: aP2, forward <named-content content-type="gene">5&#8242; CAT CAG CGT AAA TGG GGA TT 3&#8242;</named-content> reverse <named-content content-type="gene">5&#8242; CCG CCA TCT AGG GTT ATG AT 3&#8242;</named-content>; Lipin 1, forward <named-content content-type="gene">5&#8242; CGT ACG TGC GGC TCT GCG AA 3&#8242;</named-content> reverse <named-content content-type="gene">5&#8242; GCT CGG TCG CGT CAA GCT GA 3&#8242;</named-content>; PPAR&#947;2, forward <named-content content-type="gene">5&#8242; GCA TGG TGC CTT CGC TGA 3&#8242;</named-content> reverse <named-content content-type="gene">5&#8242; TGG CAT CTC TGT GTC AAC CAT G 3&#8242;</named-content>; TBP, forward <named-content content-type="gene">5&#8242; CCC CTT GTA CCC TTC ACC AAT 3&#8242;</named-content> reverse <named-content content-type="gene">5&#8242; GAA GCT GCG GTA CAA TTC CAG 3&#8242;</named-content>; Trib2, forward <named-content content-type="gene">5&#8242; CCC GCC CGA GAC TCC GAA CT 3&#8242;</named-content> reverse <named-content content-type="gene">5&#8242; GCA CAG CGC GGA AAA CGT GG 3&#8242;</named-content>.</p><p>The Framingham Heart Study began in 1948 with the enrollment of the Original Cohort <xref ref-type="bibr" rid="pgen.1002705-Dawber1">[50]</xref>. In 1971, the 5,124 participants were enrolled as part of the Offspring Cohort. Finally, in 2002, the Third Generation cohort was enrolled (n&#8202;=&#8202;4095) <xref ref-type="bibr" rid="pgen.1002705-Splansky1">[51]</xref>. Participants for the present study were derived from the Framingham Heart Study Multi-detector Computed Tomography (MDCT) Sub-Study. Briefly, from June 2002 to April 2005, 3529 participants (2111 Third Generation, 1418 Offspring participants) underwent MDCT.</p><p>Framingham Heart Study participants underwent MDCT utilizing 8-slice MDCT in a supine position (LightSpeed Ultra, General Electric, Milwaukee, WI). On average, 48 contiguous 2.5 mm slices of the heart were acquired with prospectively ECG triggered CT scanning protocol (120 kVp, 400 mA, temporal resolution 330 ms).</p><p>We measured pericardial fat tissue volumes (cm<sup>3</sup>) with a dedicated offline workstation (Aquarius 3D Workstation, TeraRecon Inc., San Mateo, CA) based on the principle that absolute Hounsfield Units (HU) values correspond to tissue property. Thus, we set a predefined image display (window width &#8722;195 to &#8722;45 HU; window center &#8722;120 HU) to identify pixels that correspond with adipose tissue. Pericardial fat was measured across the complete available imaging volume in cm<sup>3</sup>.</p><p>We used a semi-automatic segmentation technique which required the reader to manually trace the pericardium. We defined pericardial fat volume as adipose tissue located within the pericardial sac. Using a random sample of 100 participants, intra-reader (ICC 0.97) and inter-reader (ICC 0.95) reproducibility was excellent <xref ref-type="bibr" rid="pgen.1002705-Rosito1">[5]</xref>.</p><p>Framingham Heart Study participants also underwent eight-slice MDCT imaging of the abdomen with twenty-five contiguous five mm thick slices (120 kVp, 400 mA, gantry rotation time 500 ms, table feed 3&#8758;1) acquired 125 mm beyond the level of S1. Subcutaneous and visceral adipose tissue volumes (SAT and VAT) were assessed as previously described, with excellent inter-class correlations for VAT (0.992) and SAT (0.997) <xref ref-type="bibr" rid="pgen.1002705-MaurovichHorvat1">[52]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The Phase II CEU HapMap panel was used a reference to impute genotypes to roughly 2.5 million HapMap SNPs; details can be found in <xref ref-type="supplementary-material" rid="pgen.1002705.s004">Table S1</xref>. MACH v1.0.15/16 (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MACH/">http://www.sph.umich.edu/csg/abecasis/MACH/</ext-link>) was used along with 200 (101 Men and 99 Women) biologically independent individuals in order to establish parameter estimates. We then used these estimates to infer gene dosage. Imputed genotypes were expressed as allelic dosage (which is a decimal value ranging from 0&#8211;2).</p><p>Linear mixed effects regression modeling was used in order to account for pedigree structure (R lme and kinship package). Given that we observed association with the first principal component (estimated using Eigenstrat <xref ref-type="bibr" rid="pgen.1002705-Price1">[53]</xref>), we included this component in our regression models.</p><p>The Multi-Ethnic Study of Atherosclerosis (MESA) is a National Heart, Lung and Blood Institute-sponsored, population-based investigation of subclinical cardiovascular disease and its progression <xref ref-type="bibr" rid="pgen.1002705-Bild1">[54]</xref>. In brief, a total of 6,809 individuals, aged 45 to 84 years, were recruited from six US communities (Baltimore City and County, MD; Chicago, IL; Forsyth County, NC; Los Angeles County, CA; New York, NY; and St. Paul, MN) between July 2000 and August 2002.</p><p>All MESA participants underwent baseline cardiac CT scans at baseline, which were analyzed for pericardial fat volume (cm3). Cardiac CT scans were performed either with an ECG-triggered (at 80% of the RR interval) electron-beam scanner (Chicago, Los Angeles, and New York field centers; Imatron C-150, Imatron) or with prospectively ECG-triggered scan acquisition at 50% of the RR interval with a multidetector system that acquired 4 simultaneous 2.5-mm slices for each cardiac cycle in a sequential or axial scan mode (Baltimore, Forsyth Country, and St. Paul field centers; Lightspeed, General Electric or Volume Zoom, Siemens). Three experienced CT analysts measured pericardial fat volume on the previously obtained images of the heart. For pericardial fat volume, slices within 15 mm above and 30 mm below the superior extent of the left main coronary artery were included. This region of the heart was selected because it includes the pericardial fat located around the proximal coronary arteries (left main coronary, left anterior descending, right coronary, and circumflex arteries). The anterior border of the volume was defined by the chest wall and the posterior border by the aorta and the bronchus. Volume Analysis software (GE Healthcare, Waukesha, WI) was used to discern fat from other tissues with a threshold of &#8722;190 to &#8722;30 Hounsfield units. The volume was the sum of all voxels containing fat. Our measure of pericardial fat volume was highly correlated with total volume of pericardial fat volume in a random subset of 10 Diabetes Heart Study participants (correlation coefficient: 0.93; p&lt;0.0001). A random sample of 80 MESA participants was selected and their CT scans were reread. The intra-class correlation coefficients of intra-reader and inter-reader reliability were 0.99 and 0.89 for pericardial fat <xref ref-type="bibr" rid="pgen.1002705-Ding1">[6]</xref>.</p><p>Coronary artery calcium was also assessed using the cardiac CT scans at baseline <xref ref-type="bibr" rid="pgen.1002705-Detrano1">[55]</xref>. Briefly, we used the reader&#8211;work station interface to calibrate each tomographic image according to the estimated attenuation of the calcium phantom and then identified and quantified the coronary calcium in each image. We than calculated the coronary calcium score (Agatston score) for each scan. Intraobserver and interobserver agreement was excellent (kappa statistics, 0.93 and 0.90, respectively).</p><p>Carotid intimal medial thickness (IMT) measures were obtained at baseline using ultrasound imaging of the carotid arteries that was performed using a GE scanner. Using videotaped scans, we performed a centralized interpretation of the data. IMT was measured between lumen-intima and media-adventitia interfaces of near and far walls of the common carotid artery (the 1 cm segment proximal to the bifurcation) and the internal carotid artery (including the bifurcation and 1 cm distal to the bifurcation). A maximum IMT for each of these two segments was standardized (by subtraction of the MESA population mean and division by its standard deviation), and the mean of the standardized IMT for the common and the internal carotid maxima was used in analysis <xref ref-type="bibr" rid="pgen.1002705-Sharrett1">[56]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">MESA participants provided consent for genotyping and had DNA extracted at the time of baseline enrollment between 2000&#8211;2002. Genotyping was performed at the Broad Institute of Harvard and MIT (Boston, Massachusetts, USA) and at the Affymetrix Laboratory (Santa Clara, CA, USA) using the Affymetric Genome-Wide Human SNP Array 6.0 (Affymetrix, Santa Clara, California, USA). Genotype results were imputed to 2.5 million SNPs using IMPUTE v.2.1.0 software (<ext-link ext-link-type="uri" ns0:href="http://mathgen.stats.ox.ac.uk/impute/impute.html">http://mathgen.stats.ox.ac.uk/impute/impute.html</ext-link>) <xref ref-type="bibr" rid="pgen.1002705-Howie1">[57]</xref>. Participant-specific quality controls included filters for call rate and number of Mendelian errors per individual. SNP-specific quality controls included filters for call rate and heterozygosity.</p><p>Linear regression modeling was used with adjustment for age, sex, weight, and height. The first principal component (estimated using Eigenstrat <xref ref-type="bibr" rid="pgen.1002705-Price1">[53]</xref>) was included in our regression models.</p>