<p>Cre-Tgfbr2<sup>flox/flox</sup> female and Cre+Tgfbr2<sup>flox/wt</sup> male mice were kindly provided by Dr. Harold Moses, Vanderbilt Cancer center, Nashville, TN. Mice were bred to yield Tgfbr2<sup>fspKO</sup> mice. Cre+Tgfbr2<sup>flox/wt</sup> male mice were crossed with Rosa26 reporter female mice to validate the Cre specificity. All mice were housed at the National Cancer Institute (NCI) animal facility Animal studies were performed under NCI- IACUC approved protocol.</p><p>Forestomach samples were collected from Tgfbr2<sup>fspKO</sup> and Tgfbr2<sup>flox/flox</sup> mice, which then fixed, sectioned and stained using H&amp;E. For immunofluorescence studies, the sections were incubated overnight at 4&#176;C with primary antibodies directed against T&#946;RII (1&#8758;50, Santa Cruz), S100A4 (1&#8758;100, Abcam), K14 (1&#8758;100, Covance), F4/80 (1&#8758;100, BD Transduction Laboratories), NOS2 (1&#8758;100, BD Transduction Laboratories), Cox2 (1&#8758;100, Cell Signaling Technology), p65 (1&#8758;100, Cell Signaling Technology), &#947;H2AX (1&#8758;100, Trevigen), CD4 and IL17A (1&#8758;100, Biolegend). Fluorescence-tagged secondary antibodies were used for visualization (anti-rabbit, 1&#8758;1000, Invitrogen or; anti-mouse, Vector lab, respectively). Slides were examined using fluorescence microscopy (Olympus). For immunohistochemistry, slides were incubated with primary antibodies against CD45 (1&#8758;100, BD Pharmigen), Psmad2 (1&#8758;100, Cell Signaling Technology), and &#946;-galactosidase (1&#8758;100, Abcam). Signals were visualized using Vectastain (Vector Lab) and examined under a light microscope (Carl Zeiss). Quantitative data was measured by counting total number of cells expressing the marker out of all the cells in one field by Image J software. Three different fields were evaluated; percentage was calculated from total number of cells counted and averaged for three independent fields.</p><p>Apoptosis was evaluated by TUNEL (Terminal deoxynucleotidyl transferase dUTP nick end labeling) method using &#8220;In Situ cell death detection kit, Fluorescein&#8221; (Roche Applied Science, Indianapolis, IN, USA). Briefly, tissue sections were pretreated with xylene and ethanol (100%, 95%, 90%, 80%, 70%), washed with 1&#215; PBS, then treated with proteinase K (20 ug/ml in 10 mM Tris-HCL (pH 7.4) and 50 ul tunel mixture for 60 min. Data was acquired using fluorescence microscopy (Olympus). Quantitative data was obtained using Image J software by counting the number of cells expressing the marker out of total cells for each field. Three different fields were evaluated; the data was presented as percentage of stained cells in total cells and averaged for three independent fields.</p><p>Single cell suspensions were made using fresh forestomach tissue from Tgfbr2<sup>fspKO</sup> and Tgfbr2<sup>flox/flox</sup> mice through incubation with Liberase TL (200 U/mL) (Roche Applied Science, Indianapolis, IN) at 37&#176;C for 30 minutes. Forestomach tissue was then crushed and filtered through a 70 &#181;m cell strainer. Cells were labeled with fluorescence-conjugated antibodies against CD45-PE, CD11b-FITC, Gr-1-APC, Ly6G-APC, Ly6C-PE, or 7-AAD (BD Pharmingen). Isotype-matched IgG was used as a control (BD Pharmingen). The flow data was acquired on BD FACS Calibur flow cytometer (BD Biosciences, San Jose, CA) and analyzed using FlowJO.</p><p>The forestomachs were dissected and treated with 0.05% trypsin overnight at 4&#176;C. The forestomach tumor tissues from Tgfbr2<sup>fspKO</sup> or equivalent normal tissues from Tgfbr2<sup>flox/flox</sup> mice were separated by peeling them from the stromal and muscle layers. Protein was extracted, and then separated by gel electrophoresis. Membranes were incubated with primary antibodies against NOS2 (1&#8758;100, BD Transduction Laboratories), &#947;H2AX (1&#8758;1000, Trevigen), HGF (1&#8758;1000, Santa Cruz Biotechnology), Cox2, P-p65, P-p53, Cyclin D1, p21, p15, and p16 (all at 1&#8758;1000, Cell Signaling Technology) or &#946;-actin (1&#8758;5000, Sigma), and horseradish peroxidase-conjugated secondary antibody (1&#8758;5000, Biorad). The blots were developed using a SuperSignal West Pico Chemiluminescent substrate kit (Pierce). Images were scanned in a G: Box (Syngene).</p><p>DNA was isolated from epithelial layers of forestomach as described in &#8220;Western Blotting.&#8221; QIAGEN Genomic-tip 20/G (Qiagen, CA, USA) and modified with Epitech Bisulfite kit (Qiagen, CA, USA) were used in pyrosequencing. PCR templates for pyrosequencing analysis were amplified from 10 ng gDNA using Hotstart Taq Mastermix (Qiagen, CA, USA) and 5 pmol of each primer in a total reaction volume of 25 &#181;l. In all, 1 &#181;l of each PCR reaction was analysed on an Agilent 2100 Bioanalyzer (Santa Clara, CA) using a DNA 1000 kit. Pyrosequencing was carried out on 0.15&#8211;0.5 pmol of each PCR product using the PyroMark MD System (Qiagen, CA, USA) following the manufacturer's instructions with sequencing primers and assay parameters specific to each methylation site. Resulting pyrograms were analysed using the PyroMark MD 1.0 software in &#8216;AQ mode&#8217;. For each assay, duplicate pyrosequencing analysis was performed, and the average of these was taken to represent the identified percentage methylation of the methylated allele.</p><p>Laser capture microdissection of Tgfbr2<sup>flox/flox</sup>, Tgfbr2<sup>fspKO</sup> and Celecoxib treated Tgfbr2<sup>fspKO</sup> mouse tissue was performed using an Arcturus XT (Life Technologies, CA, USA). Frozen tissue sections on PEN membrane frame slides (Applied Biosystems) were H&amp;E stained followed by dehydration using the standard protocol to improve visualization of the cells at the microscope. The epithelia and stroma were identified by morphology, captured using a low-power infrared laser pulse, and transferred onto a cap (Capsure&#8482; Macro LCM Caps, Life Technologies). The DNA was extracted using a QIAamp DNA micro kit (Qiagen, CA, USA).</p><p>Primary epithelial cells (1096, isolated from SCC of FSP-Cre/RII knock out mouse, and kept in low passages) were cultured in 6 well plate with seeding density 0.3&#215;10<sup>6</sup> per well for overnight in DME/F12 medium containing 10% FBS and 1&#215; antibiotics (GIBCO, Life Technologies, CA, USA). DNA methyltransferase inhibitor, 5-aza 2&#8242; deoxycytidine (Decitabine), was added to the culture at 5 uM concentration for 48 hours. Cells were harvested using 0.25% trypsin-EDTA (GIBCO, Life Technologies, CA, USA) and subjected to total RNA extraction by RNeasy mini kit (Qiagen, CA, USA) and protein extraction by standard method.</p><p>Total RNA was isolated from laser captured micro-dissected tissue samples by Arcturus Picopure RNA isolation Kit (Applied Biosystems). Reverse transcription was performed using oligo dT primers and Superscript II (Invitrogen). The primers for qPCR were mCDKN1A (p21) F: <named-content content-type="gene">5&#8242;-ACAGGAGCAAAGTGTGCCGTTGT-3&#8242;</named-content>; mCDKN1A (p21) R: <named-content content-type="gene">5&#8242;- GCTCAGACACCA GAGTGCAAGACA 3&#8242;</named-content>; mGAPDH F: <named-content content-type="gene">5&#8242;-ATGACCACAGTCCATGCCATCACT-3&#8242;</named-content>; mGAPDH R: <named-content content-type="gene">5&#8242;-TGTTGAAGTCGCAGGAGACAACCT-3&#8242;</named-content>. PCR reactions were performed using fast real-time 7500 PCR system (Applied Biosystems). All samples were tested in triplicate. The comparative C<sub>T</sub> method was used for quantification of gene expression. <italic>Gapdh</italic> was used as an endogenous reference. Statistical analysis was performed using SDS v2.1 software (Applied Biosystems) according to the manufacturer's instructions.</p><p>Protein extraction was obtained from forestomach samples of Tgfbr2<sup>fspKO</sup> and Tgfbr2<sup>flox/flox</sup> mice, and analyzed for IFN-&#947; and TNF-&#945; expression, as per manufacturer's instruction. Data was acquired and analyzed using Bio-Plex Manager version 4.0 software (Bio-Rad).</p><p>Genomic DNA was isolated from primary tumor cells (1096) from <italic>Tgfbr2<sup>fspKO</sup></italic> mice and primary normal epithelial cells isolated from forestomach epithelial cell layer with a QIAamp DNA Mini Kit according to manufacturer protocol (Qiagen, Valencia, CA). Array-CGH was performed using test DNA from laser captured epithelia and stroma, 1096 primary tumor cell culture, and reference DNA. DNA was labeled with Cy3 or Cy5 fluorescent dyes (Pharmacia, Piscataway, NJ) according to the BioPrime array CGH genomic labeling protocol (Invitrogen, Carlsbad, CA) and cleaned using Microcon YM-30 filters (Millipore, Billerica, MA). Hybridization was carried out using Mouse Genome CGH Microarray 4&#215;44 K from Agilent Technologies (Santa Clara, CA) according to CGH Procedures for Genomic DNA Analysis (Agilent Technologies). Slides were hybridized for 20 hours, washed, and scanned with an Agilent microarray scanner. Data was analyzed using Feature Extraction&#174; and CGH Analytics&#174; software packages (Agilent Technologies). The array-based CGH data is available, GEO accession number: GSE42773.</p><p>Genomic DNA from laser captured epithelia and stroma described above was used for genomic PCR using the 2&#215; <italic>Taq</italic> master mix (Gene Script, NJ, USA), 50 ng genomic DNA and Exon 1 specific primers of mouse p15 and p16 genes. The primer sequences were p15: forward- <named-content content-type="gene">5&#8242;- GTT GGG CGG CAG CAG TGA C-3&#8242;</named-content> and reverse- <named-content content-type="gene">5&#8242;-CCT CCC GAA GCG GTT CAG-3&#8242;</named-content>, p16: forward- <named-content content-type="gene">5&#8242;-ACT GGT CAC ACG ACT GGG CGA TTG -3&#8242;</named-content> and reverse- <named-content content-type="gene">5&#8242;-AAT CGG GGT ACG ACC GAA AGA G-3&#8242;</named-content>. Actin: forward-<named-content content-type="gene">5&#8242;-TCA TCA GGT AGT CAG TGA GGT CGC-3&#8242;</named-content> and reverse-<named-content content-type="gene">5&#8242;-CAC CAC ACC TTC TAC AAT GAG CTG-3&#8242;</named-content>. The PCR conditions included initial denaturation at 95&#176;C for 5 min, denaturation at 95&#176;C for 1 min, annealing at 60&#176;C for 1 min and extension at 72&#176;C for 1 min for 40 cycles and final extension at 72&#176;C for 7 min. Agarose gel electrophoresis (2%) was used to detect the PCR products and data was recorded using G: Box (syngene).</p><p>C57BL/6NCr wild type female mice were super-ovulated and crossed with the Fsp-Cre male mice. Four or eight cell embryos were transferred to pathogen free females. After birth, pups were examined for infection and genotypes using ELISA and PCR. Male mice with Cre+ Tgfbr2<sup>flox/wt</sup> and female mice with Cre- Tgfbr2<sup>flox/wt</sup> were identified and crossed to obtain pathogen free Tgfbr2<sup>fspKO</sup> mice. The mice re-derivation was performed in the mice re-derivation core facility located at NCI-Frederick, MD. Mice were transferred and housed in Helicobacter free facility at NIH Bethesda, MD.</p><p>Tgfbr2<sup>flox/flox</sup> and Tgfbr2<sup>fspKO</sup> mice were treated with diet pellets containing Celecoxib at 1500 ppm with (D04090202) or without (AIN 76A D1000) active compound, or <italic>N</italic>
<sub>&#969;</sub>-Nitro-L-arginine methyl ester hydrochloride (L-NAME, Sigma) at a dose of 50 mg/kg/day. The pups received the Celecoxib starting the second week after birth, together with the nursing mother mouse in the same cage, due to early inflammation onset (<xref ref-type="fig" rid="pgen-1003251-g001">Figure 1B</xref>). The treatment continued after weaning. The synergistic effect of L-NAME and Celecoxib was also examined. In addition, Mice housed in the Helicobacter free conditions were treated with Celecoxib to evaluate the cooperative effect in survival and phenotype. For gel diet treatment, after weaning on day 21, Tgfbr2<sup>fspKO</sup> and Tgfbr2<sup>flox/flox</sup> pups were fed with gel diet (AIN76A, Clear H<sub>2</sub>O). All mice were monitored daily and sacrificed with signs of poor health including small size, hunched body, slow movements, and weakness in comparison to healthy littermates.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Human ESCC (n&#8202;=&#8202;8) and adjacent normal FFPE tissue slides (n&#8202;=&#8202;8) were previously described <xref ref-type="bibr" rid="pgen.1003251-Mathe1">[57]</xref> (PMID: 19789312), and were stained to detect T&#946;RII expression in FSP1+ stromal cells. p65, NOS2, and 8-Oxo-dG adducts were also examined for inflammation and DNA damage. The immunofluorescence staining procedures are as described above. Oncomine database (<ext-link ext-link-type="uri" ns0:href="http://www.oncomine.com">www.oncomine.com</ext-link>) was utilized to evaluate &#947;-H2AX expression, p15, and p16 loss in human ESCC, specifically mRNA expression datasets, Hu Esophagus (34 samples) and Su Esophagus 2 (106 samples), and DNA copy number datasets, Hu Esophagus 2 (blood vs LCM tumor sample) and Bass Esophagus (cell lines and tissue specimens). Dot plots of H2AX mRNA expression are presented as log2 median-centered intensity. The dataset (PMID: 19789312) from Oncomine was analyzed using GraphPad Prism 5.0 and two-tailed, paired t-test. Heat map of copy number loss of p15 and p16 in ESCC was obtained from the same dataset that measure DNA copy number on a SNP microarray platform. The expression of T&#946;RII in tumor-associated stroma vs adjacent normal was also analyzed using Oncomine mRNA dataset Ma 4 Breast Carcinoma.</p><p>Data was analyzed using the Student t-test, and was expressed as mean &#177; SE. Differences were considered statistically significant at p&lt;0.05. Mouse survival data was examined using SPSS 16 software, and is presented as Kaplan Meier curve. A log rank test was used to calculate statistical differences in survival and median survival of the different groups.</p>