<p>All studies were approved by Institutional Review Boards at their respective sites, and all participants provided informed consent.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">PAGE involves several studies, described briefly below and in more detail in <xref ref-type="supplementary-material" rid="pgen.1003171.s006">Text S1</xref> as well as at the PAGE website (<ext-link ext-link-type="uri" ns0:href="https://www.pagestudy.org">https://www.pagestudy.org</ext-link>) <xref ref-type="bibr" rid="pgen.1003171-Matise1">[40]</xref>. In brief, participants were recruited from Atherosclerosis Risk in Communities Study (ARIC), GenNet, Hypertension Genetic Epidemiology Network (HyperGEN), Multiethnic Cohort (MEC), and Women's Health Study (WHI). ARIC randomly selected and recruited 15,792 participants aged 45&#8211;64 at four U.S. communities <xref ref-type="bibr" rid="pgen.1003171-1">[41]</xref>. GenNet and HyperGEN are two family-based studies designed to investigate the genetics of hypertension and related conditions <xref ref-type="bibr" rid="pgen.1003171-2">[42]</xref>. The Multiethnic Cohort (MEC) is a population-based prospective cohort study of over 215,000 men and women in Hawaii and California aged 45&#8211;75 at baseline (1993&#8211;1996) and primarily of five ancestries <xref ref-type="bibr" rid="pgen.1003171-Kolonel1">[43]</xref>. The WHI encompasses four randomized clinical trials as well as a prospective cohort study of 161,808 post-menopausal women aged 50&#8211;79, recruited (1993&#8211;1998) and followed up at 40 centers across the US <xref ref-type="bibr" rid="pgen.1003171-3">[44]</xref>. All studies collected self-identified racial/ethnic group via questionnaire. We selected all AA participants from ARIC, HyperGEN, and GenNet for genotyping. In MEC, a subset of AAs was selected based on availability of biomarker or as controls for nested case-control studies. WHI included all AAs who provided consent for DNA analysis. We excluded underweight (BMI&lt;18.5 kg/m<sup>2</sup>) and extremely overweight (BMI&gt;70 kg/m<sup>2</sup>) individuals with the assumption that these extremes could be attributable to data coding errors, an underlying illness or possibly to a familial syndrome and hence, a rare mutation. We also limited analysis to adults defined as age &gt;20 years.</p><p>For ARIC HyperGEN, GenNet and WHI, BMI was calculated from height and weight measured at time of study enrollment. In MEC, self-reported height and weight were used to calculate baseline BMI. A validation study within MEC has shown high validity of self-reported height and weight. Specifically this study showed that BMI was under-estimated based on self-reported compared to measured weight, but the difference was small (&lt;1 BMI unit) and comparable to the findings from national surveys <xref ref-type="bibr" rid="pgen.1003171-Gorber1">[45]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">SNPs were included as part of the Metabochip, a 200 k Illumina customized iSelect array developed through the collaborative efforts of several consortia working on metabolic syndrome related diseases. Details on the design can be found elsewhere (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/kang/MetaboChip/">http://www.sph.umich.edu/csg/kang/MetaboChip/</ext-link>). In brief, SNPs within the 16q12.2/<italic>FTO</italic> region were selected based on 1000 Genomes Pilot 1 and HapMap phase 2. The boundaries around each GWAS index SNP were determined by identifying all SNPs with r<sup>2</sup>&#8805;0.5 with the index SNP, and then expanding the initial boundaries by 0.02 cM in either direction using the HapMap-based genetic map <xref ref-type="bibr" rid="pgen.1003171-Frazer1">[46]</xref>. The total interval size of the 16q12.2/<italic>FTO</italic> region was 646 kb. All 1000 Genomes Pilot 1 SNPs obtained from Sanger Institute (August 12, 2009) and the Broad Institute (August 11, 2009) were considered as potential fine mapping SNPs, unless SNP allele frequency was &lt;0.01 in all three HapMap samples (CEU, YRI and HBC/JPT). SNPs were excluded if (a) the Illumina design score was &lt;0.5 or (b) there were SNPs within 15 bp in both directions of the SNP of interest with allele frequency of &gt;0.02 among Europeans (CEU). SNPs annotated as nonsynonymous, essential splice site, or stop codon were included regardless of allele frequency, design score, or nearby SNPs in the primer.</p><p>Samples were genotyped on the Metabochip at the Human Genetics Center of the University of Texas-Houston (ARIC, GenNet and HyperGEN), the University of Southern California Epigenome Center (MEC), and the Translational Genomics Research Institute (WHI). Each center also genotyped 90 HapMap YRI (Yoruba in Ibadan, Nigeria) samples to facilitate cross-study quality control (QC), as well as 2&#8211;3% study-specific blinded replicates to assess genotyping reproducibility. Genotypes were called separately for each study using GenomeStudio with the GenCall 2.0 algorithm. Samples were called using study-specific cluster definitions (based on samples with call rate &gt;95%, ARIC, MEC, WHI) or cluster definitions provided by Illumina (GenNet, HyperGEN) and kept in the analysis if call rate was &gt;95%. We excluded SNPs with GenTrain score &lt;0.6 (ARIC, MEC, WHI) or &lt;0.7 (GenNet, HyperGEN), cluster separation score &lt;0.4, call rate &lt;0.95, and Hardy-Weinberg Equilibrium p&lt;1&#215;10<sup>&#8722;6</sup>. We also excluded SNPs based on Mendelian errors in 30 YRI trios &gt;1, replication errors &gt;2 with discordant calls (when comparing across studies) in 90 YRI samples &gt;3, and discordant calls for 90 YRI genotyped in PAGE versus HapMap database &gt;3. In total, we successfully genotyped 1,694 out of 1,818 variants in the 16q12.2/<italic>FTO</italic> region. After excluding 165 SNPs that were monomorphic or had very low allele frequency (&lt;0.01%) we included 1,529 variants in the analysis.</p><p>For ARIC, MEC and WHI we identified related persons using PLINK by estimating identical-by-descent (IBD) statistics for all pairs. When apparent first-degree relative pairs were identified, we excluded from each pair the member with the lower call rate. We excluded from further analysis samples with an inbreeding coefficient (F) above 0.15 (ARIC, MEC, WHI) <xref ref-type="bibr" rid="pgen.1003171-Weale1">[47]</xref>. We determined principal components of ancestry in each study separately using EIGENSOFT <xref ref-type="bibr" rid="pgen.1003171-Patterson1">[48]</xref>, <xref ref-type="bibr" rid="pgen.1003171-Price1">[49]</xref> and excluded apparent ancestral outliers from further analysis as described elsewhere <xref ref-type="bibr" rid="pgen.1003171-Buyske1">[50]</xref>. In total 240 subjects failed genotyping (ARIC&#8202;=&#8202;27, GenNet&#8202;=&#8202;9, HyperGEN&#8202;=&#8202;26, MEC&#8202;=&#8202;140, and WHI&#8202;=&#8202;27). After further excluding subjects based on age and BMI (see above), a total of 14,162 subjects with Metabochip data were included (3,297 from ARIC, 517 from GenNet, 1,171 from HyperGEN, 3,865 from MEC, and 5,312 from WHI). In addition we included 6,326 WHI participants genotyped as part of the SNP Health Association Resource (SHARe) on the Affymetrix 6.0 platform. Details can be found elsewhere <xref ref-type="bibr" rid="pgen.1003171-Liu1">[51]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To impute to the 1000 Genomes Project we used as the reference panel the haplotypes of the 1092 samples (all populations) from release version 2 of the 1000 Genomes Project Phase I (<ext-link ext-link-type="ftp" ns0:href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/20110521">ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/20110521</ext-link>) <xref ref-type="bibr" rid="pgen.1003171-Durbin1">[52]</xref>. Combining reference data from all populations has been found to improve imputation accuracy of the low-frequency variants <xref ref-type="bibr" rid="pgen.1003171-Howie1">[53]</xref>, and, hence, is recommended. We built the target panel by combining all genotype data in the <italic>FTO</italic> region from all studies. We used genotyped data from the Metabochip for all studies, except for WHI SNP Health Association Resource (SHARe, n&#8202;=&#8202;6,326 samples) where we used genotype data from the Affymetrix 6.0 platform. The target panel was phased using Beagle <xref ref-type="bibr" rid="pgen.1003171-Browning1">[54]</xref>. We then performed a haplotype-to-haplotype imputation to estimate genotypes (as allele dosages) at 1000 Genomes Project variants. The phased target panel was imputed to the interval 53.5&#8211;54.2 Mb on chromosome 16 of the 1000 genomes reference panel using minimac <xref ref-type="bibr" rid="pgen.1003171-Howie2">[55]</xref>. To evaluate the quality of each imputed SNP we calculated Rsq. We excluded imputed SNPs with Rsq&lt;0.9 for SNPs with allele frequency &lt;0.5%, Rsq&lt;0.8 for SNPs with allele frequency &gt;0.5&#8211;1%, Rsq&lt;0.7 for SNPs with allele frequency &gt;1&#8211;3%, Rsq&lt;0.6 for SNPs with allele frequency &gt;3&#8211;5%, and Rsq&lt;0.5 for SNPs with allele frequency &gt;5%. Given the large reference panel this resulted in high imputation quality <xref ref-type="bibr" rid="pgen.1003171-Liu1">[51]</xref>. To calculate pairwise correlation between variants we used the 1000 Genomes Project data, specifically we used 61 African Americans from the South-west (ASW) and 65 European Americans (Utah residents with Northern and Western European ancestry from the CEPH collection, CEU).</p><p>The association between each SNP and natural log-transformed BMI (lnBMI) was estimated using linear regression. SNP genotypes were coded assuming an additive genetic model (i.e., 0, 1, or 2 copies of the coded allele). All analyses were adjusted for age (continuous), sex, and study site (as applicable). All models (except WHI) included sex*age interaction terms to account for possible effect modification by sex. In addition, we adjusted for the top two principal components of ancestry. Family data from GenNet and HyperGEN was analyzed using mixed models (variance component models) to account for relatedness.</p><p>
<xref ref-type="sec" rid="s2">Results</xref> (&#946; and SEs for lnBMI) were combined with fixed-effects meta-analysis weighting the effect size estimates (&#946;-coefficients) by their estimated standard errors, using METAL <xref ref-type="bibr" rid="pgen.1003171-Willer2">[56]</xref>. We evaluated Q-statistic and I<sup>2</sup> as a measure of heterogeneity <xref ref-type="bibr" rid="pgen.1003171-DerSimonian1">[57]</xref>, <xref ref-type="bibr" rid="pgen.1003171-Higgins1">[58]</xref>, to describe the presence or absence of excess variation between the PAGE cohorts. For ease of interpretation, we calculated the % change in BMI per copy of the effect allele based on the beta for the lnBMI. To graphically display the results, we used LocusZoom <xref ref-type="bibr" rid="pgen.1003171-Pruim1">[59]</xref>. We tested for independence of findings by including the most significant variant and each of the other variant into the same model (i.e. we included 2 variants simultaneously in one model). We evaluated if SNPs are independent by investigating the p-value. For an independent SNP the p-value would remain low/similar after adjusted for the most significant SNP.</p><p>We conducted a bioinformatic characterization for the most significant SNP and all SNPs correlated with the most sigificant SNP (r<sup>2</sup>&gt;0.5). We implemented in-house Perl scripts to query bioinformatic databases, and assigned each of the 16 SNPs to one or more of the functional annotation datasets listed in <xref ref-type="supplementary-material" rid="pgen.1003171.s004">Table S3</xref>. These datasets are not mutually exclusive. For example, a SNP can be located within both a candidate regulatory element (dataset #7) and a CTCF binding site (dataset #10). Because FTO is expressed and may have functional relevance in a wide array of tissues, we defined candidate <italic>cis</italic>-regulatory elements (dataset #7) as DNaseI hypersensitive sites (open chromatin loci) that are present in at least one human cell type. For SNPs that occur within predicted transcription factor binding sites (datasets #3 and #8), we computed transcription factor binding affinity for each SNP allele using the PWM-scan algorithm <xref ref-type="bibr" rid="pgen.1003171-Levy1">[60]</xref>, as described previously <xref ref-type="bibr" rid="pgen.1003171-Sethupathy1">[61]</xref>.</p>