<p>Yeast strains used in this study are listed in <xref ref-type="supplementary-material" rid="pgen.1003334.s011">Table S2</xref> and plasmids in <xref ref-type="supplementary-material" rid="pgen.1003334.s012">Table S3</xref>. Cultures were grown in complete (YPD) or minimal (SD) media <xref ref-type="bibr" rid="pgen.1003334-Sherman1">[73]</xref> at 30&#176;C unless otherwise stated. For phenotypic assays, YPD was supplemented with 10 &#181;g/ml sordarin sodium salt from <italic>Sordaria araneosa</italic> (Sigma-Aldrich). Yeast transformations with plasmid DNAs were performed following the lithium acetate protocol <xref ref-type="bibr" rid="pgen.1003334-Gietz1">[74]</xref>. Diphtheria toxin (DT) growth assays in vivo involved expression of the toxin's cytotoxic ADP ribosylase fragment (DTA) from vector pSU8 (p415-<italic>GALS</italic>-DTA), essentially as previously described for <italic>dph1-dph5</italic> mutants <xref ref-type="bibr" rid="pgen.1003334-Br1">[6]</xref>. pSU8 was made by cloning the <italic>Bam</italic>HI fragment encoding DTA from pLMY101 <xref ref-type="bibr" rid="pgen.1003334-Mattheakis1">[30]</xref> into plasmid p415<italic>-GALS</italic>, a single-copy <italic>E. coli</italic>-yeast shuttle vector with a truncated <italic>GAL</italic> promoter <xref ref-type="bibr" rid="pgen.1003334-Mumberg1">[55]</xref>, which allows for conditional DTA induction on galactose-containing media. <xref ref-type="bibr" rid="pgen.1003334-Mumberg1">[55]</xref>. The translational frameshift reporter assay essentially involved previously published protocols together with the described <italic>lacZ</italic> reporter plasmids pJD204.0 (wild-type control), pJD204.&#8722;1 (&#8722;1 frame) and pJD204.+1 (+1 frame) <xref ref-type="bibr" rid="pgen.1003334-Ortiz1">[36]</xref>, <xref ref-type="bibr" rid="pgen.1003334-Harger1">[59]</xref>; the pJD204 plasmid series was kindly provided by T. Kinzy (UMDNJ, USA). The relative values for +1 and &#8722;1 frameshifting were statistically analyzed using one-way ANOVA followed by Dunnett's multiple comparison post test and was performed with Graphpad Prism 5.0 software essentially as previously described <xref ref-type="bibr" rid="pgen.1003334-Roy1">[75]</xref>.</p><p>Details of all primers used in numerous PCR dependent genomic manipulation experiments can be found in <xref ref-type="supplementary-material" rid="pgen.1003334.s013">Table S4</xref>. Gene deletions were performed using in vivo PCR-based one step-gene disruption protocols in combination with marker plasmids YDpKl-L, YDpKl-U or YDpSp-H <xref ref-type="bibr" rid="pgen.1003334-Jablonowski1">[76]</xref> and knockout primers (<xref ref-type="supplementary-material" rid="pgen.1003334.s013">Table S4</xref>) including those previously described <xref ref-type="bibr" rid="pgen.1003334-Br1">[6]</xref>, <xref ref-type="bibr" rid="pgen.1003334-Fichtner2">[25]</xref>, <xref ref-type="bibr" rid="pgen.1003334-Frohloff1">[77]</xref>. Gene deletions were confirmed via diagnostic PCR on genomic DNA preparations using target ORF-specific primer pairs (<xref ref-type="supplementary-material" rid="pgen.1003334.s013">Table S4</xref>) as well as sordarin response assays. C-terminal tagging of <italic>DPH1</italic>, <italic>DPH2</italic>, <italic>DPH5</italic>, <italic>DPH6/YLR143w</italic> and <italic>DPH7/YBR246</italic> was performed according to previously published in vivo PCR-based epitope tagging protocols <xref ref-type="bibr" rid="pgen.1003334-Knop1">[78]</xref> using appropriate S3/S2 primer pairs (<xref ref-type="supplementary-material" rid="pgen.1003334.s013">Table S4</xref>). Tagged genes were confirmed by Western blot detection with anti-HA or anti-c-Myc antibodies (Santa Cruz Biotechnology A-14 and F7, respectively). Detection of HA- or c-Myc-tagged Dph1, Dph2, Dph5, Dph6 and Dph7 as well as Dph3 and Elp2 in co-immune precipitation (Co-IP) assays were performed as previously described <xref ref-type="bibr" rid="pgen.1003334-Br1">[6]</xref>, <xref ref-type="bibr" rid="pgen.1003334-Frohloff1">[77]</xref>, <xref ref-type="bibr" rid="pgen.1003334-Zachariae1">[79]</xref>.</p><p>pSU6 was generated by insertion into YCplac111 <xref ref-type="bibr" rid="pgen.1003334-Gietz2">[80]</xref> of a genomic PCR fragment including <italic>DPH6</italic> together with 829 bp of upstream and 59 bp of downstream sequence flanked by <italic>Eco</italic>RI and <italic>Bam</italic>HI sites incorporated using PCR primers (<xref ref-type="supplementary-material" rid="pgen.1003334.s013">Table S4</xref>). The insert was verified by sequencing and shown to complement a <italic>dph6</italic> knockout strain. pSU7 was made by cloning the <italic>DPH6</italic> insert from pSU6 into YEplac181 <xref ref-type="bibr" rid="pgen.1003334-Gietz2">[80]</xref>. To generate a G216N E220A <italic>dph6</italic> mutant, pSU6 was digested with <italic>Age</italic>I and <italic>Bsm</italic>BI and the small <italic>DPH6</italic> fragment replaced by an identical synthetic fragment (Integrated DNA Technologies) carrying the G216N E220A mutations, generating independent clones pMS61 and pMS62. The replaced region was verified by DNA sequencing. pMS67 and pMS68 were generated from pSU6 by replacing the <italic>Bsm</italic>BI-<italic>Sal</italic>I fragment carrying the C-terminal region of <italic>DPH6</italic> and downstream sequence with a synthetic <italic>Bsm</italic>BI-<italic>Sal</italic>I fragment in which codons 335&#8211;685 were replaced by sequence encoding the linker and triple <italic>myc</italic> tag from pYM23 <xref ref-type="bibr" rid="pgen.1003334-Janke1">[81]</xref>. To generate pMS72, the smaller <italic>Nhe</italic>I-<italic>Spe</italic>I fragment of pSU7 was excised and the large fragment ligated to generate an in-frame fusion that removed <italic>DPH6</italic> codons 347&#8211;471, checking the resulting fusion by DNA sequencing.</p><p>Yeast cell extracts were prepared as described previously <xref ref-type="bibr" rid="pgen.1003334-Liu1">[15]</xref>. ADP ribosylation reactions were performed at 37&#176;C for 1 hour in a volume of 40 &#181;l ADP ribosylation buffer (20 mM Tris-HCl, pH 7.4, 1 mM EDTA, 50 mM DTT) containing 50 &#181;g of yeast extract, 50 ng fully-nicked DT and 10 &#181;M 6-biotin-17-NAD (Trevigen). Samples were then mixed with SDS sample buffer, boiled for 5 min and run on 4&#8211;25% SDS-PAGE gradient gels (Invitrogen). The proteins were transferred to nitrocellulose membranes and Western blotting was performed using streptavidin-IR conjugate (Rockland Immunochemicals, Gilbertsville, PA) and scanned on an Odyssey Infrared Imager (LICOR Biosciences, Lincoln, NE).</p><p>BY4741 wild-type yeast cells as well as <italic>dph1</italic>, <italic>dph5</italic>, <italic>ylr142w/dph6</italic> and <italic>ybr246w/dph7</italic> mutants thereof carrying an <italic>eft2</italic> null-allele were transformed with plasmid pTKB612 (a kind gift from A. R. Merrill, University of Guelph, Ontario, Canada), which expresses a (His)<sub>6</sub>-tagged form of translation elongation factor eEF2 (<xref ref-type="supplementary-material" rid="pgen.1003334.s012">Table S3</xref>) that is fully functional and able to complement an <italic>eft1 eft2</italic> double mutant <xref ref-type="bibr" rid="pgen.1003334-Jrgensen3">[56]</xref>. In order to express and purify (His)<sub>6</sub>-tagged eEF2 for MS/MS analysis, 750 ml of yeast culture were grown in YPD to an OD<sub>600</sub> 2.0 and harvested by centrifugation. The pellet was resuspended in 3 ml B60 buffer (50 mM HEPES-KOH pH 7.3, 60 mM KOAc, 5 mM Mg(OAc)<sub>2</sub>, 0.1% Triton X100, 10% (v/v) glycerol, 1 mM NaF, 20 mM glycerophosphate, complete protease inhibitor [Roche]) without DTT and cells were lysed in a bead beater. The lysate was centrifuged twice at 13,500 rpm for 30 min. and the protein concentration measured with a NanoDrop spectrophotometer. Five mg total protein was applied to 2 mg anti-(His)<sub>6</sub>-tag Dynabeads (Invitrogen, #101-03D) and purified according to manufacturer's instructions. The identity of purified eEF2 fraction was confirmed by SDS-PAGE and Western blot analysis using an anti-(His)<sub>6</sub> antibody (Abcam, #ab18184).</p><p>Crude yeast eEF2 preparations from wild-type and <italic>dph</italic> mutants strains were separated by SDS-PAGE using 4&#8211;12% Bis-Tris precast gels (Invitrogen, Carlsbad, USA) and the area of the gel containing eEF2 was excised after staining with Instant Blue Coomassie (Expedeon, Cambridge, UK). In-gel digests were performed using trypsin, subsequent to reduction and alkylation with dithiothreitol and iodoacetamide, with the resulting peptides cleaned over C18 columns. Peptides were then analyzed via HPLC-MS/MS using a Dionex U300 HPLC (Dionex California) with a 15 cm PepMap C18 column coupled to a Thermo Orbitrap Velos mass spectrometer (Thermo Fisher Scientific, Bremen, Germany). The peptides were eluted from the C18 column at 300 nL/min over 120 min using a linear 5&#8211;90% (v/v) acetonitrile gradient. The Orbitrap Velos was operated in positive ion mode, with an ion source voltage of 1.2 kV and capillary temperature 200&#176;C, using a lock mass of 445.120024. The initial survey scan was performed at 60000 resolution, FTMS scanning from 335&#8211;1800 Da. The top 15 most intense ions were selected for MS/MS sequencing, using collision-induced dissociation (CID; MS/MS charge state 1+ rejected, &gt;2+ accepted). Protein identification was performed using MaxQuant 1.2.2.5 <xref ref-type="bibr" rid="pgen.1003334-Cox1">[82]</xref> against a proteome database generated from the <italic>Saccharomyces</italic> Genome database <xref ref-type="bibr" rid="pgen.1003334-Cherry1">[83]</xref>. Manual annotation of the modified peptide spectra corresponding to the modified EF2 peptide and generation of extracted ion chromatograms were done using the Thermo Xcalibur software for spectra visualization.</p>