<p>All strains in this study are haploid, contain a circular Chr III and were derived from isogenic strains MWJ49 and MWJ50, described in <xref ref-type="bibr" rid="pgen.1003420-Ma2">[48]</xref>. Gene deletions in these strains were created by replacement of the relevant ORF(s) with dominant-resistance cassettes G418 (<italic>kanMX4</italic>), hygromycin (<italic>hphMx4</italic>), or nourseothricin (<italic>natMX4</italic>) as described in <xref ref-type="bibr" rid="pgen.1003420-Goldstein1">[49]</xref>. The I-<italic>Sce</italic>I recognition site and GAL1-I-<italic>Sce</italic>I gene were introduced into circular Chr III strains by integration of GAL1-I-<italic>Sce</italic>I core cassettes amplified from plasmids pGSHU or pGSKU <xref ref-type="bibr" rid="pgen.1003420-Storici1">[50]</xref> using primers</p><p>GSU_LC3 (<named-content content-type="gene">5&#8242;ATCAAATTCGATGACTGGAAATTTTTTGTTAATTTCAGAGGTCGCCTGACGCTAGGGATAACAGGGTAATTTGGATGGACGCAAAGAAGT3&#8242;</named-content>)</p><p>GNU-RC3 (<named-content content-type="gene">5&#8242;ATGAAAAGCCGGTTCCGGCGCTCTCACCTTTCCTTTTTCTCCCAATTTTTCAGTTGAAAATTCGTACGCTGCAGGTCGAC3&#8242;</named-content>).</p><p>The integration site is &#8764;250 bp to the left of the <italic>LEU2</italic> gene on Chr III. The <italic>mre11-nd</italic> mutation was introduced into circular Chr III strains by crossing with KS435 <xref ref-type="bibr" rid="pgen.1003420-Lobachev1">[20]</xref> followed by tetrad dissection and selection of appropriate markers. The <italic>mre11-null</italic> strain used in this study was made by integration of the KanMX-URA3core <xref ref-type="bibr" rid="pgen.1003420-Storici2">[51]</xref> into the <italic>MRE11</italic> locus. Primers</p><p>mre11-3 (<named-content content-type="gene">AGTTCACAAGCAAGCCTGTA</named-content>) and</p><p>mre11-4 (<named-content content-type="gene">ACTTGTGAGGGATCGCTC</named-content>)</p><p>were used to amplify the core cassette to enable targeting of the core cassette. The integration resulted in replacement by the core cassette of 485 bp of the <italic>MRE11</italic> locus beginning 215 bases upstream of the start codon through the first 169 bases of the coding sequence.</p><p>Nocodazole (United States Biological, Swampscott, MA) was used at 20 ug/ml final concentration in logarithmically growing cultures at 30&#176;C in YPDA media (1% yeast extract, 2% Bacto-Peptone, 2% dextrose, 60 &#181;g/ml adenine sulfate). G2 arrest was monitored by cell morphology. Cultures were harvested by centrifugation, washed and resuspended in ice-cold sterile water at &#8764;5&#215;10<sup>7</sup> cells/ml. Cell suspensions were kept on ice during irradiation in a <sup>137</sup>Cs irradiator (J. L. Shepherd Model 431). Following irradiation, cells were centrifuged and resuspended in YPDA at 30&#176;C and nocodazole was added at 20 ug/ml for post-irradiation incubation.</p><p>Cells were cultured logarithmically in YEP lactate as described in <xref ref-type="bibr" rid="pgen.1003420-Westmoreland1">[33]</xref>, and treated with nocodazole as described above for gamma irradiated cultures before adding 2% galactose.</p><p>Plug preparation of samples and PFGE using a CHEF Mapper XA system (Bio-Rad, Hercules, CA) were carried out as in <xref ref-type="bibr" rid="pgen.1003420-Westmoreland1">[33]</xref> except for the details presented below that are specific for the 2D gels displayed in <xref ref-type="fig" rid="pgen-1003420-g002">Figure 2</xref>, <xref ref-type="fig" rid="pgen-1003420-g003">Figure 3</xref>, <xref ref-type="fig" rid="pgen-1003420-g006">Figure 6</xref>, <xref ref-type="supplementary-material" rid="pgen.1003420.s002">Figure S2B</xref>, <xref ref-type="supplementary-material" rid="pgen.1003420.s003">Figure S3</xref>, and <xref ref-type="supplementary-material" rid="pgen.1003420.s004">Figure S4</xref>.</p><p>Plugs were prepared using 0.35% LE (NOTE: not low melting) agarose (Lonza Allendale, NJ) and were formed in Beckman Geneline plug molds having dimensions 2.5&#215;2.5&#215;25 mm. The 1<sup>st</sup> dimension CHEF gels used 0.75% SeaKem Gold agarose (Lonza) and were made using 3 mm thick plastic spacers sandwiched between the Biorad CHEF gel tray and a glass plate to give the gels a uniform thickness of only 3 mm. Uncut 2.5 cm long plugs were equilibrated in running buffer (0.5&#215; TBE) before loading the entire plug in the gel (wide orientation of gel frame). The gels were then run for 22 hr using the auto-algorithm mode, programmed for 250 to 1400 kb.</p><p>Following the 1<sup>st</sup> dimension run, the DNA-containing lanes were 2.5 cm wide (i.e., the plug width) and &#8764;13 cm long. To obtain material for the 2<sup>nd</sup> dimension CHEF, two slices (&#8764;4 mm&#215;13 cm each) were cut and removed from the center of the 1<sup>st</sup> dimension lanes (leaving about 8 mm to either side). The remaining flanks of each lane were stained with SYBR Gold, which enabled the bands of interest to be located. This procedure avoided the need to stain the lanes prior to the 2<sup>nd</sup> run, since the stain interfered with the migration of chromosome fragments in the 2<sup>nd</sup> dimension. After staining the flanks for 10 minutes in SYBR Gold, they were digitally photographed, and a ruler was used in the photograph to identify the region of the unstained lane slice corresponding to &#8764;220 kb to 600 kb (from just below Chr I to just above Chr V). This section was cut from the full-length lane slices and processed for the 2<sup>nd</sup> dimension PFGE as follows: each lane section was equilibrated in 3 changes of 25 ml sterile TE (10 mM Tris, pH 8, 1 mM EDTA), treated with SfiI (New England Biolabs, Ipswich, MA; 400 units in 4 ml total reaction volume including gel slice) for 5 hr at 50&#176;C, followed by overnight treatment in 1% sarkosyl (Sigma, St. Louis, MO) plus 1 mg/ml proteinase K (Invitrogen) at 37&#176;C and subsequent equilibration with running buffer. The map of the positions of the SfiI restriction sites relative to the I-<italic>Sce</italic>I cut site are presented in <xref ref-type="supplementary-material" rid="pgen.1003420.s002">Figure S2A</xref> and results for cutting the DNA by I-<italic>Sce</italic>I <italic>in vitro</italic> (prior to 1<sup>st</sup> dimension) and SfiI (prior to 2<sup>nd</sup> dimension) are presented in <xref ref-type="supplementary-material" rid="pgen.1003420.s002">Figure S2B</xref>.</p><p>For the 2<sup>nd</sup> dimension PFGE, the treated lane slices were placed near the top of the gel tray (long orientation of gel frame to allow for a long run) using an end plate from the casting stand on the anode side of the lane slices to stabilize the position of the slices so that they could be attached to the gel tray with 2 to 4 ml of agarose on the cathode side of the lane slices. After allowing the agarose to cool for 5 to 10 minutes, the end block was carefully removed, leaving the lane slices attached to the gel tray. The gel tray was then mounted in the casting stand, and 180 ml of 1.5% LE agarose (Lonza) in 0.5&#215; TBE was carefully poured over the tray containing the attached lane slices. This was enough volume to cover the lane slices. The 2<sup>nd</sup> dimension CHEF was run using the same autoalgorithm as the 1<sup>st</sup> dimension, 250 to 1400 kb, except that a longer run time of 48 hr was used to compensate for slower electrophoretic mobility in 1.5% agarose.</p><p>Neutral Southern blots, probe preparations, <sup>32</sup>P labeling and hybridizations were carried out as previously described <xref ref-type="bibr" rid="pgen.1003420-Ma2">[48]</xref>. Primers used for PCR amplification of genomic DNA to be used in the preparation of probes are listed in <xref ref-type="supplementary-material" rid="pgen.1003420.s005">Table S1</xref>.</p><p>The DSB repair data shown in <xref ref-type="fig" rid="pgen-1003420-g005">Figure 5B</xref> were calculated using the &#8220;stained gel, multiple band method&#8221; described in <xref ref-type="bibr" rid="pgen.1003420-Westmoreland1">[33]</xref>.</p>