<p>We used primary data from the studies in BCAC. All studies had approval from the relevant ethics committees and all participants gave informed consent. A centralized BCAC database of information about common risk factors and tumor characteristics was constructed to facilitate studies of potential modifications of SNP associations by other risk factors. A multi-step data harmonization procedure was used to reconcile differences in individual study questionnaires. The reference age for cohort studies was calculated at time of enrollment and for case-control studies at date of diagnosis for cases and at date of interview for controls. All time-dependent variables were assessed at reference age. This analysis included only subjects of European ancestry that had genotype data for at least 3 SNPs and provided information on at least one of the established risk factors. Relevant data were available from 24 studies, including 16 population-based studies (14 case-control and 2 prospective cohort studies) and 8 non-population-based studies (<xref ref-type="table" rid="pgen-1003284-t001">Table 1</xref>, <xref ref-type="supplementary-material" rid="pgen.1003284.s001">Table S1</xref>, <xref ref-type="supplementary-material" rid="pgen.1003284.s002">Table S2</xref>). Subsets of data from 19 studies (with 11 population-based) were included in a previous report that assessed interactions between 12 susceptibility variants, reproductive history, BMI and breast cancer risk <xref ref-type="bibr" rid="pgen.1003284-Milne1">[7]</xref>.</p><p>We included 21 SNPs found to be associated with overall breast cancer risk at genome-wide statistical significance (p&lt;5&#215;10<sup>&#8722;7</sup>) <xref ref-type="bibr" rid="pgen.1003284-Broeks1">[10]</xref>, <xref ref-type="bibr" rid="pgen.1003284-Ghoussaini1">[25]</xref>, <xref ref-type="bibr" rid="pgen.1003284-Fletcher1">[34]</xref> and SNPs for <italic>TGFB1</italic> and <italic>CASP8</italic> from candidate gene studies <xref ref-type="bibr" rid="pgen.1003284-Cox1">[17]</xref> (<xref ref-type="supplementary-material" rid="pgen.1003284.s003">Table S3</xref>). For three loci, 14q24.1/<italic>RAD51L1</italic>, 12p11, <italic>CASP8</italic>, a surrogate SNP in high linkage disequilibrium (LD) (<italic>r<sup>2</sup></italic>&#8202;=&#8202;1 in HapMap CEU) was genotyped in a subset (<xref ref-type="table" rid="pgen-1003284-t003">Table 3</xref> footnote) <xref ref-type="bibr" rid="pgen.1003284-Figueroa1">[19]</xref>, <xref ref-type="bibr" rid="pgen.1003284-Ghoussaini1">[25]</xref>, <xref ref-type="bibr" rid="pgen.1003284-Thomas1">[35]</xref>.</p><p>Genotyping was performed in the framework of BCAC by Taqman and iPlex assays and underwent quality control as described previously <xref ref-type="bibr" rid="pgen.1003284-Broeks1">[10]</xref>, <xref ref-type="bibr" rid="pgen.1003284-Figueroa1">[19]</xref>, <xref ref-type="bibr" rid="pgen.1003284-Ghoussaini1">[25]</xref>, <xref ref-type="bibr" rid="pgen.1003284-Fletcher1">[34]</xref>, <xref ref-type="bibr" rid="pgen.1003284-Milne2">[36]</xref>, <xref ref-type="bibr" rid="pgen.1003284-Milne3">[37]</xref>. Genotype data were excluded from analysis on a study-by-study basis according to the following BCAC quality control (QC) guidelines: 1) any sample that consistently failed for &gt;20% of the SNPs within a genotyping round, 2) all samples on any one plate that had a call rate &lt;90% for any one SNP, 3) all genotype data for any SNP where overall call rate was &lt;95%, 4) all genotype data for any SNP where duplicate concordance was &lt;98%. In addition, for any SNP where the P-value for departure from Hardy-Weinberg proportions for controls was &lt;0.005, clustering of the intensity plots was reviewed manually and the data excluded if clustering was judged to be poor.</p><p>We used logistic regression to assess the main effects of the SNP and environmental risk factors on invasive breast cancer risk. Analyses were adjusted for study as a categorical variable and reference age as a continuous variable. Odds ratios (OR) and their 95% confidence intervals (CI) were calculated for the SNP associations assuming a log-additive model and tested for association with a one degree of freedom trend test. All statistical tests were two-sided.</p><p>The assessment of associations with the environmental risk factors was based on data only from the 16 population-based studies to ensure unbiased estimates for comparison with established effect sizes. The variables considered were analyzed as continuous (age at menarche, number of births in parous women, age at first birth, usual BMI, height, duration of oral contraceptive use, duration of current use of estrogen-progestagen combined therapy, duration of current use of estrogen-only therapy, pack-years of cigarette smoking, mean lifetime daily grams of alcohol intake, recent physical activity in hours per week), or as dichotomous (ever parous, ever breastfed, ever OC use, ever smoked, current EPT use, current ET use) (<xref ref-type="table" rid="pgen-1003284-t002">Table 2</xref>). Analyses were performed for all women as well as separately for women aged &lt;54 years and &#8805;54 years, considering the age groups as surrogates of pre- and postmenopausal status, Differential effects by menopausal status were assessed by adding an interaction term. For all categorical variables, the lowest level of exposure (or no use) was used as the reference. For evaluating current use of MHT by type, we used never use of MHT as the reference category and additionally adjusted for former use of MHT and other MHT type, as appropriate.</p><p>To test for interactions between SNPs and environmental risk factors, we fitted for each SNP two logistic models, a model with terms for the SNP and the risk factor of interest and another model with additionally an interaction term for the product between the SNP (number of risk alleles) and the risk factor variable. We modeled the interaction based on the risk factor variable definitions employed for the main effects. All analyses were stratified by study and adjusted for age as a continuous variable. The likelihood ratio test was used to compare the difference between the two models and departure from independent multiplicative effects of the SNP and the risk factor. BMI was the only variable found to show differential effects by menopausal status, which is consistent with the literature <xref ref-type="bibr" rid="pgen.1003284-vandenBrandt1">[38]</xref>. Therefore, interaction between SNPs and BMI was assessed separately for pre- and postmenopausal women whereas all other risk factors were evaluated regardless of menopausal status. To assess study heterogeneity, we calculated odds ratios for interaction for each individual study, adjusting for age, and reported P-values for heterogeneity using a Q-test. Subjects with missing data for a particular SNP or environmental factor were excluded from the respective analysis. We also calculated stratum specific per-allele ORs for each SNP: age at menarche (&#8804;11, 12&#8211;13, &#8805;14 years), number of births (1,2,3, &#8805;4), age at first birth (&lt;20, 20&#8211;24, 25&#8211;29, &#8805;30 years), usual BMI (&lt;25, 25&#8211;29, &#8805;30), height (&lt;160, 160&#8211;164, 165&#8211;169, &#8805;170 cm), duration of oral contraceptive use and of menopausal hormone use (0, &gt;0&#8211;&lt;5, 5&#8211;&lt;10, &#8805;10 years), mean lifetime alcohol intake (0, 0&#8211;&lt;10, 10&#8211;&lt;20, &#8805;20 g/day), pack-years of smoking (0, 1&#8211;&lt;10, 10&#8211;&lt;20, &#8805;20), and physical activity (0, &gt;0&#8211;&lt;3.5, &#8805;3.5&#8211;&lt;7, &#8805;7 h/week).</p><p>For SNP-environment interactions with associated P-value&lt;10<sup>&#8722;3</sup>, we also compared results after adjusting for additional covariates. We performed a total of 414 (23 SNPs x 18 risk variables) tests. To account for chance findings due to multiple comparisons, we calculated the false positive report probability (FPRP) for SNP-environment interactions with associated P-value&lt;10<sup>&#8722;3</sup>
<xref ref-type="bibr" rid="pgen.1003284-Wacholder1">[39]</xref>. The FPRP depends on the prior probability that the association between the SNP and breast cancer is modified by the environmental risk factor, the power of the present study, and the observed P-value. Since the prior probability of the assessed multiplicative interactions varies depending on subjective evaluation of existing evidence, we calculated the FPRPs for prior probabilities ranging from 0.05 to 0.0001. We considered findings with FPRP below 0.2 to be noteworthy results, as previously proposed <xref ref-type="bibr" rid="pgen.1003284-Wacholder1">[39]</xref>.</p><p>In secondary analyses, we examined associations and effect modifications separately for women with ER-positive tumors and ER-negative tumors, each compared to all controls. Effect heterogeneity by ER status was tested using case-case analysis.</p><p>Data harmonization was performed using an ACCESS database and transformation of the data elements was performed using SAS (Release 9.2). All other data analyses were conducted using SAS (Release 9.2) and the R programming language <xref ref-type="bibr" rid="pgen.1003284-R1">[40]</xref>.</p>