<p>For the TPOAb GWAS stage 1 and 2 analyses, and the hypothyroidism, hyperthyroidism and goiter analyses, individuals were recruited from 16 independent community-based and family studies. For the Graves' disease analyses, cases were recruited from the United Kingdom Graves' disease cohort and controls from the British 1958 Birth Cohort. Thyroid cancer cases and controls were recruited from the Nijmegen and Ohio thyroid cancer cohorts. A detailed description of the original cohorts contributing samples is provided in <xref ref-type="table" rid="pgen-1004123-t001">Table 1</xref> and in the Supplementary Material. All participants provided written informed consent and protocols were approved by the institutional review boards or research ethics committees at the respective institutions, and conducted according to the Declaration of Helsinki.</p><p>Serum TPOAb levels were determined with a range of assays. TPOAb-positives were defined as subjects with TPOAb levels above the assay-specific TPOAb-positivity cut-off, as defined by the manufacturer (<xref ref-type="table" rid="pgen-1004123-t001">Table 1</xref>). Serum TSH and free thyroxine (FT4) levels were determined using a range of assays (<xref ref-type="table" rid="pgen-1004123-t001">Table 1</xref>). Assay-specific TSH and FT4 reference ranges were used, as provided by the manufacturer (<xref ref-type="table" rid="pgen-1004123-t001">Table 1</xref>). Overt hypothyroidism was defined as a high TSH (i.e., a TSH level above the TSH reference range) and a low FT4. Increased TSH was defined as a high TSH, including persons with overt hypothyroidism or subclinical hypothyroidism (i.e., high TSH with a normal FT4). Overt hyperthyroidism was defined as a low TSH and a high FT4. Decreased TSH was defined as a low TSH, including persons with subclinical or overt hyperthyroidism.</p><p>The diagnosis of goiter is described in the Supplementary Material, and the diagnosis of Graves' disease and thyroid cancer in the respective cohorts have been described previously <xref ref-type="bibr" rid="pgen.1004123-Gudmundsson1">[41]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Samples were genotyped with a range of GWAS genotyping arrays (Supplementary <xref ref-type="supplementary-material" rid="pgen.1004123.s007">Table S1</xref>). Sample and SNP quality control procedures were undertaken within each study. For each GWAS, over 2.5 million SNPs were imputed using CEU samples from Phase 2 of the International HapMap project (<ext-link ext-link-type="uri" ns0:href="http://www.hapmap.org">www.hapmap.org</ext-link>). Genotyping procedures in the stage 2, Graves' disease and thyroid cancer populations are described in the Supplementary Material.</p><p>The heritabilities of TPOAb-positivity and serum TPOAb levels were estimated, as described in the Supplementary Material.</p><p>In stage 1, we performed a GWAS on TPOAb-positivity as well as a GWAS on continuous TPOAb levels. Persons taking thyroid medication were excluded. Each SNP was tested for association with TPOAb-positivity using logistic regression analyses, adjusting for age and sex. For cohorts with family structure, we approximated the probability of being affected with a linear mixed model adjusting for age and sex. The produced model was used to predict the expected proportion of &#8220;risk&#8221; (effective) alleles in cases and controls, hence giving the means to estimate odds ratios. Only unrelated individuals were considered for the SardiNIA cohort. For the GWAS of continuous TPOAb levels, samples with a TPOAb level lower than the minimum TPOAb assay detection limit (<xref ref-type="table" rid="pgen-1004123-t001">Table 1</xref>) were excluded. TPOAb levels were natural log-transformed, and sex-specific, age adjusted standardized residuals were calculated. Each SNP was tested for association with these TPOAb level residuals using linear regression analyses (additive model), correcting for relatedness in studies with family structure. See Supplementary <xref ref-type="supplementary-material" rid="pgen.1004123.s007">Table S1</xref> for the software used for these analyses.</p><p>Before meta-analysis, SNPs with a minor allele frequency (MAF) &lt;1% or a low imputation quality were excluded (Supplementary Material), after which the results of each GWAS were combined in a population size weighted z-score based meta-analysis using METAL <xref ref-type="bibr" rid="pgen.1004123-Willer1">[71]</xref>. Genomic control was applied to individual studies if &#955;&gt;1.0.</p><p>In stage 2, we followed-up stage 1 GWAS significant SNPs, as well as promising SNPs not reaching GWAS significance, in an attempt to reach GWAS significant associations by increasing sample size (Supplementary Material). Results from stage 1 and 2 were combined in a population size weighted z-score based meta-analysis using METAL <xref ref-type="bibr" rid="pgen.1004123-Willer1">[71]</xref>. A z-score based meta-analysis was used to reduce bias that might be induced by different assays. As this method does not provide betas, and we wanted to provide a rough estimate of the actual effect sizes for convenience, we calculated betas using the fixed effects (inverse variance based) meta-analysis method. Heterogeneity was tested, applying bonferroni based <italic>P</italic>-value thresholds of <italic>P</italic>&#8202;=&#8202;0.004 for the TPOAb-positivity analyses and <italic>P</italic>&#8202;=&#8202;0.005 for the TPOAb level analyses.</p><p>All studies assessed and, if present, corrected for population stratification using principal-component analysis (PCA) and/or multidimensional-scaling (MDS), with the exception of SardiNIA and ValBorbera where the high isolation substantiates a lack of stratification (<xref ref-type="supplementary-material" rid="pgen.1004123.s007">Table S1</xref>) <xref ref-type="bibr" rid="pgen.1004123-Milani1">[72]</xref>, <xref ref-type="bibr" rid="pgen.1004123-Pilia1">[73]</xref>. Lambda values were all &#8764;1, indicating that population stratification was overall properly accounted for (<xref ref-type="supplementary-material" rid="pgen.1004123.s007">Table S1</xref>). To fully remove residual effects, we applied genomic correction to studies were lambda was &gt;1. The final meta-analyses reported a lambda of 1.01 for both the TPOAb-positivity and the TPOAb level GWAS, thus no genomic correction was applied.</p><p>The variances explained by the GWAS significant SNPs were calculated. We subsequently studied the individual as well as the combined effects of the GWAS significant SNPs on the risk of clinical thyroid disease, as specified in the Supplementary Material. In short, to study combined effects, a genetic risk score was calculated for every person as the weighted sum of TPOAb risk alleles. The associations between the individual SNPs, genetic risk scores and the risk of abnormal thyroid function tests were studied using logistic regression analyses. Logistic regression analyses were used to study the associations with goiter, Graves' disease and thyroid cancer (Supplementary Material). The results of each study were combined in a population size weighted z-score based meta-analysis using METAL <xref ref-type="bibr" rid="pgen.1004123-Willer1">[71]</xref>.</p><p>Various bioinformatic tools were searched for evidence for functional relevance of the GWAS significant SNPs and pathway analyses were performed on the Stage 1 lead SNPs (see Supplementary Material).</p>