<p>All human research was approved by the relevant institutional review boards, and conducted according to the Declaration of Helsinki.</p><p>Cohort description, genotyping and statistical methods for individual study cohorts are reported in <xref ref-type="supplementary-material" rid="pgen.1003266.s011">Text S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1003266.s004">Table S1</xref>.</p><p>We carried out a meta-analysis including up to 26,523, individuals from 18 cohorts for TSH and up to 17,520 individuals from 15 cohorts for FT4 (see <xref ref-type="table" rid="pgen-1003266-t001">Table 1</xref>). FT4 measures were not available for all 21,955 individuals with TSH levels of the 15 participating cohorts. We combined evidence of associations from single GWAS using an inverse variance meta-analysis, where weights are proportional to the squared standard error of the beta estimates, as implemented in METAL <xref ref-type="bibr" rid="pgen.1003266-Willer1">[57]</xref>. Prior to GWAS, each study excluded individuals with known thyroid pathologies, taking thyroid medication, who underwent thyroid surgery, and with out-of-range TSH values (&lt;0.4 mIU/L and &gt;4 mIU/L), and an inverse normal transformation was applied to each trait (<xref ref-type="supplementary-material" rid="pgen.1003266.s004">Table S1</xref>). Age, age-squared, and gender were fitted as covariates, as well as principal components axes or additional variables, as required (<xref ref-type="supplementary-material" rid="pgen.1003266.s004">Table S1</xref>). Family-based correction was applied if necessary (see <xref ref-type="supplementary-material" rid="pgen.1003266.s004">Table S1</xref>). Uniform quality control filters were applied before meta-analysis, including MAF &lt;0.01, call rate &lt;0.9, HWE <italic>P</italic>&lt;1&#215;10<sup>&#8722;6</sup> for genotyped SNPs and low imputation quality (defined as r<sup>2</sup>&lt;0.3 or info &lt;0.4 if MACH <xref ref-type="bibr" rid="pgen.1003266-Li1">[58]</xref> or IMPUTE <xref ref-type="bibr" rid="pgen.1003266-Marchini1">[59]</xref>, <xref ref-type="bibr" rid="pgen.1003266-Howie1">[60]</xref> were used, respectively) for imputed SNPs.</p><p>Genomic control was applied to individual studies if lambda was &gt;1.0. The overall meta-analysis showed no significant evidence for inflated statistics (lambda for TSH, FT4 and were 1.05 and 1.03 respectively). To evaluate for heterogeneity in effect sizes across populations, we used a chi-square test for heterogeneity, implemented in METAL <xref ref-type="bibr" rid="pgen.1003266-Willer1">[57]</xref>. The same test was used to evalute heterogeneity related to iodine intake, by comparing effect sizes obtained in a meta-analysis of studies assessing individuals from South Europe (InChianti, MICROS, Val Borbera, SardiNIA, totaling up to 7,488 subjects) with those estimated in a meta-analysis of studies assessing individuals from North America (BLSA, CHS, FHS, OOA, totaling up to 5,407 subjects). Finally, the main meta-analysis was carried out independently by two analysts who obtained identical results.</p><p>To identify independent signals, each study performed GWA analyses for both TSH and FT4 by adding the lead SNPs found in the primary analysis (19 for TSH, and 4 for FT4, see <xref ref-type="table" rid="pgen-1003266-t002">Table 2</xref>) as additional covariates to the basic model, and removing those from the test data set. When lead SNPs were not available, the best proxies (r<sup>2</sup>&gt;0.8) were included. We then performed a meta-analysis on the conditional GWAS results, using the same method and filters as described above. We used the standard genome-wide significance cutoff (<italic>P</italic>&lt;5&#215;10<sup>&#8722;8</sup>) to declare a significant secondary association.</p><p>To identify sex-specific effects, each study performed GWA analyses for each gender separately, using the same covariates and transformation as in the basic model (with the exception of gender covariate). We then performed a meta-analysis on association results using the same method and filters described for the primary analysis. To evaluate sex-specific differences we tested heterogeneity between effect sizes as described above. False-discovery rates (FDRs) on the 26 associated SNPs were calculated with R's p.adjust() procedure via the method of Benjamini and Hochberg <xref ref-type="bibr" rid="pgen.1003266-Benjamini1">[24]</xref>.</p><p>The variance explained by the strongest associated SNPs was calculated, for each trait and in each cohort, as the difference of R<sup>2</sup> adjusted observed in the full and the basic models, where the full model contains all the independent SNPs in addition to the covariates. The estimates from each cohort were combined using a weighted average, with weights proportional to the cohort sample size.</p><p>To evaluate the impact of the detected variants with clinically relevant TSH levels, we compared the allele frequencies observed in different categories of individuals in a case-control approach. Specifically, we compared individuals in the upper and lower TSH tails (individuals with TSH &gt;4 mIU/L and TSH &lt;0.4 mIU/L, respectively, whom were excluded for the GWAS analyses), as well as individuals in each tail with those in the normal TSH range. In the first case, individuals in the lower tail were considered controls and those in the upper tail cases. In the other two cases, we defined individuals in the normal range as controls and individuals on the two tails cases. To avoid sources of bias, individuals taking thyroid medication and/or with thyroid surgery were excluded. Only unrelated individuals were selected from the family-based cohort SardiNIA, while GEE correction was applied to the TwinsUK dataset. <xref ref-type="sec" rid="s2">Results</xref> from single cohorts were then meta-analyzed. We first assessed the global impact of the 20 TSH- and 6 FT4-associated variants by defining a genotype-risk score (GRS) for each individual as the weighted sum of TSH- and FT4-elevating alleles, with weights proportional to the effect estimated in the meta-analysis. For each comparison, we then calculated quartiles from the global distribution (cases+controls) of the genotype score and used quartile 1 as the baseline reference to compare the number of cases and controls in the other quartiles. In addition, for TSH-associated variants we conducted single SNP comparisons. GRS quartile and single SNP analyses were performed by each study separately. Cohort specific results were then meta-analyzed for both the GRS score and single SNP results only if they had at least 50 cases and 50 controls. Specifically, cohorts included were: CHS, Lifelines, PROSPER, RS, SardiNIA and TwinsUK.</p><p>Bivariate analysis was carried out with the software poly <xref ref-type="bibr" rid="pgen.1003266-Pilia1">[26]</xref> in the SardiNIA cohort using the same individuals included in the GWAS and considering the same covariates and transformation for TSH and FT4 levels.</p><p>The URLs for data presented herein are as follows:</p><p xmlns:ns0="http://www.w3.org/1999/xlink">METAL, <ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/cgs/abecasis/metal">http://www.sph.umich.edu/cgs/abecasis/metal</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">MACH, <ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MACH/">http://www.sph.umich.edu/csg/abecasis/MACH/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">IMPUTE, <ext-link ext-link-type="uri" ns0:href="https://mathgen.stats.ox.ac.uk/impute/impute.html">https://mathgen.stats.ox.ac.uk/impute/impute.html</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">LocusZoom, <ext-link ext-link-type="uri" ns0:href="http://csg.sph.umich.edu/locuszoom/">http://csg.sph.umich.edu/locuszoom/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">HapMap, <ext-link ext-link-type="uri" ns0:href="http://www.hapmap.org">http://www.hapmap.org</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Online Mendelian Inheritance in Man (OMIM), <ext-link ext-link-type="uri" ns0:href="http://www.omim.org/">http://www.omim.org/</ext-link>
</p>