<p>Most of the E. coli K-12 strains used in this study are derivatives of RW698 (full genotype: recA730 lexA51(Def) &#916;dinB61::ble &#916;umuDC596::ermGT thr-1 araD139 &#916;(gpt-proA)62 lacY1 tsx-33 glnV44 galK2 hisG4 rpsL31 xyl-5 mtl-1 argE3 thi-1 sulA211) <xref ref-type="bibr" rid="pgen.1003878-McDonald1">[11]</xref>. Repair-deficient &#8220;KEIO&#8221; strains were obtained from the E.coli Genetic Stock Center and isogenic derivatives of RW698 were generated via generalized transduction using P1vir <xref ref-type="bibr" rid="pgen.1003878-Miller1">[64]</xref> (<xref ref-type="table" rid="pgen-1003878-t001">Table 1</xref>). Where noted, Kan<sup>S</sup> strains were obtained by transforming cells with the temperature sensitive ampicillin and chloramphenicol resistant plasmid, pCP20, which expresses the FLP recombinase <xref ref-type="bibr" rid="pgen.1003878-Baba1">[65]</xref>. Transformants were selected on LB plates containing the appropriate antibiotics at 25&#176;C and subsequently re-streaked on LB plates lacking ampicillin, chloramphenicol and kanamycin and incubated overnight at 43&#176;C. Colonies from these plates were subsequently confirmed to be ampicillin, chloramphenicol and kanamycin sensitive at 37&#176;C.</p><p>: <italic>thr-1 araD139</italic> &#916;<italic>(gpt-proA)62 lacY1 tsx-33 glnV44 galK2 hisG4 rpsL31 xyl-5 mtl-1 argE3 thi-1 sulA211.</italic>
</p><p>The following antibiotics were used for selection; Zeocin (25 &#181;g/ml), Kanamycin (50 &#181;g/ml), Tetracycline (15 &#181;g/ml), Chloramphenicol (20 &#181;g/ml), and Ampicillin (100 &#181;g/ml), Spectinomycin (50 &#181;g/ml).</p><p>The low-copy-number spectinomycin resistant plasmids pRW134 and pJM963 which encode <italic>E.coli</italic> wild-type UmuC and the <italic>umuC</italic>_Y11A variant, respectively, along with UmuD&#8242; <xref ref-type="bibr" rid="pgen.1003878-Kuban1">[23]</xref> are derived from pGB2 <xref ref-type="bibr" rid="pgen.1003878-Churchward1">[66]</xref> and express the UmuD&#8242;C proteins at close to physiological levels from their native promoter <xref ref-type="bibr" rid="pgen.1003878-Kuban1">[23]</xref>. Bacteria harboring plasmids were grown in LB media containing appropriate 50 &#181;g/ml spectinomycin.</p><p>Cells transformed with the vector plasmid, pGB2, or the low-copy number plasmid pRW134 expressing wild-type pol V, or pJM963 expressing the <italic>umuC</italic>_Y11A variant <xref ref-type="bibr" rid="pgen.1003878-Kuban1">[23]</xref> were grown overnight at 37&#176;C in LB media plus appropriate antibiotics. The next day, cultures were centrifuged and resuspended in an equal volume of SM buffer <xref ref-type="bibr" rid="pgen.1003878-Miller1">[64]</xref>. To determine the number of spontaneously arising histidine mutants on the plate, the cell cultures were seeded on the Davis and Mingioli minimal agar plates <xref ref-type="bibr" rid="pgen.1003878-Davis1">[67]</xref> plus glucose (0.4% wt/vol); agar (1.0% wt/vol); proline, threonine, valine, leucine, and isoleucine (all at 100 &#181;g/ml); thiamine (0.25 &#181;g/ml); and either no histidine, or histidine (1 &#181;g/ml). On the plates containing no histidine, only pre-existing His<sup>+</sup> mutants grew to form colonies. When &#8764;4&#215;10<sup>7</sup> bacteria were seeded on the 1 &#181;g/ml histidine, they grew to form a lawn, concomitantly exhausting the low level of histidine. Spontaneously arising His<sup>+</sup> mutants grew up through the lawn and were counted after 4 days incubation at 37&#176;C. Spontaneous mutagenesis is expressed as a frequency (mutants per plate), because the number of mutants arising on the plate is independent of the number of cells plated, but is, instead, dependent upon the limiting amount of nutrient (histidine) in the plate <xref ref-type="bibr" rid="pgen.1003878-Maron1">[68]</xref>. The <italic>relative</italic> extent of <italic>umuC</italic>_Y11A mutagenesis was calculated by first subtracting the number of pre-existing His<sup>+</sup> mutants (mutants arising on the plates lacking histidine) and subsequently dividing the number of spontaneously arising mutants on the <italic>umuC</italic>_Y11A (pJM963) plates by the number of spontaneously arising mutants on the wild-type pol V (pRW134) plates. The data reported in <xref ref-type="fig" rid="pgen-1003878-g001">Figs. 1</xref> and <xref ref-type="fig" rid="pgen-1003878-g003">3</xref> represent the average number of His<sup>+</sup> mutants from at least 3 separate experiments (&#177; standard error of the mean [SEM]).</p><p>The mutation spectra were generated using the <italic>rpoB</italic>/Rif<sup>R</sup> mutagenesis assay <xref ref-type="bibr" rid="pgen.1003878-Garibyan1">[69]</xref>, <xref ref-type="bibr" rid="pgen.1003878-Wolff1">[70]</xref>. A single pair of oligonucleotide primers can be used for PCR amplification and a single primer for DNA sequencing because 88% of all <italic>rpoB</italic> mutations are localized in the central 202 bp region of the gene <xref ref-type="bibr" rid="pgen.1003878-Garibyan1">[69]</xref>. <italic>E. coli</italic> strains RW710 [relevant genotype: <italic>lexA</italic>(Def) <italic>recA730</italic> &#916;<italic>dinB</italic> &#916;<italic>umuDC</italic> &#916;<italic>mutL</italic>], RW942 [relevant genotype: <italic>lexA</italic>(Def) <italic>recA730</italic> &#916;<italic>dinB</italic> &#916;<italic>umuDC</italic> &#916;<italic>mutL</italic> &#916;<italic>rnhB</italic> ], RW698 [relevant genotype: <italic>lexA</italic>(Def) <italic>recA730</italic> &#916;<italic>dinB</italic> &#916;<italic>umuDC</italic>], and RW838 [relevant genotype: <italic>lexA</italic>(Def) <italic>recA730</italic> &#916;<italic>dinB</italic> &#916;<italic>umuDC</italic> &#916;<italic>rnhB</italic> ], harboring the <italic>umuC</italic>_Y11A plasmid, pJM963 were diluted from a frozen stock cultures such that the initial inoculum contained &lt;1000 viable cells. Cultures were grown in LB for 24 h at 37&#176;C and appropriate dilutions spread on an LB agar plate containing 100 &#181;g/ml rifampicin. Individual independent Rif<sup>R</sup> colonies were picked from the plate using a pipette tip and subjected to PCR in a 96-well micro-titer plate. An &lt;1 kb central region of the <italic>rpoB</italic> gene was amplified using the PCR primers RpoB1: 5&#8242;-CAC A<bold>C</bold>G GCA TCT GGT TGA TAC AG-3&#8242; and RpoF1: 5&#8242;-TGG CGA AAT GGC GGA AAA C-3 by denaturation at 95&#176;C for 3 min, followed by 30 cycles of 94&#176;C for 30 s, 1 min at 59&#176;C, 2 min at 72&#176;C, followed by a final extension step at 72&#176;C for 7 min. The nucleotide sequence of the &#8764;200 bp target region of <italic>rpoB</italic> in each PCR amplicon was determined by Beckman Coulter Genomics (Danvers, MA) using WOG923AP01 primer (<named-content content-type="gene">5&#8242;-CAG TTC CGC GTT GGC CTG-3&#8242;</named-content>). Only base-pair substitutions occurring between positions 1516 and 1717 of the <italic>rpoB</italic> gene were considered during data analysis. Nucleotide sequences obtained were aligned and analyzed using the ClustalW multiple sequence alignment program (Hinxton, UK). Rates for forward mutations to rifampicin resistance (mutations in <italic>rpoB</italic>) were determined as previously described <xref ref-type="bibr" rid="pgen.1003878-Curti1">[25]</xref>.</p><p>The <italic>Bacillus caldotenax</italic> UvrA and UvrB proteins and the <italic>Thermatoga maritima</italic> UvrC protein were purified as previously described <xref ref-type="bibr" rid="pgen.1003878-Croteau1">[39]</xref>, <xref ref-type="bibr" rid="pgen.1003878-Wang1">[71]</xref>. <italic>E. coli</italic> RNase HII was purchased from New England Biolabs (Ipswich, MA, USA).</p><p>All oligonucleotides were synthesized by Lofstrand Laboratories (Gaithersburg, MD) and gel purified prior to use. The basic sequence of the 50-mer template is: <named-content content-type="gene">5&#8242;-GAC TAC GTA CTG TTA CGG CTC CAT c<bold><underline>a</underline>a</bold> taC CGC AAT CAG GCC AGA TCT GC-3&#8242;</named-content>. The lowercase letters indicate sites at which substrate dNMPs were replaced by rNMPs. Substrates with a single rAMP (first underlined <underline>a</underline> on the 5&#8242; side), two consecutive rAMPs (shown in bold), or five rNMPs were tested. A substrate with the site-specifically placed fluorescein adduct (fT) <xref ref-type="bibr" rid="pgen.1003878-Croteau1">[39]</xref> was used as a control for the activity of NER proteins and has the same sequence except that the aa bases (shown in bold) were replaced with the [fT]C sites. The fT-containing oligonucleotide and the DNA-RNA-DNA hybrids were 5&#8242;- or 3&#8242;-<sup>32</sup>P end-labeled and annealed with either completely complementary DNA strands, or with the DNA strands containing one or two mispaired bases. In the case of the fluorescein-adducted template, the damaged T was either correctly paired with A, or mispaired with C. In the case of the DNA-RNA-DNA hybrids, either the 5&#8242;A, or both As shown in bold were correctly paired with Ts or mispaired with one, or two, Cs or As. Hybridization was performed at a 1.5 molar excess of the unlabeled strand by heating in an annealing buffer (50 mM Tris-HCl (pH 8), 5 mM MgCl<sub>2</sub>, 50 &#181;g/ml BSA, 1.42 mM 2-mercaptoethanol) for 10 min at 100&#176;C followed by slow cooling to room temperature. Prior to initiation of the incision assay, the UvrABC proteins were diluted from stock solutions and preheated for 10 min at 55&#176;C. The 10 nM DNA substrates were incubated with UvrA (40 nM), UvrB (200 nM), and UvrC (100 nM) proteins for 1 hour at 55&#176;C in the presence of 1 mM ATP in a 1&#215; reaction buffer (10 mM Tris, pH 7.5, 10 mM KCl, 2 mM MgCl<sub>2</sub>, 1 mM DTT, 0.2 mM ATP). Cleavage of the 10 nM DNA substrates by RNase HII was performed according to the manufacturer's instructions. Reactions were terminated by the addition of 2&#215; loading buffer (97% formamide, 10 mM EDTA, 0.1% xylene cyanol, 0.1% bromophenol blue) and the incision products were analyzed on a 15% denaturing polyacrylamide gel. The extent of incision was determined for each substrate and expressed as a percentage of radioactivity in the cleaved products relative to the total signal. Data shown below the gels are the mean values calculated from at least two independent experiments.</p>