<p>All animal work must have been conducted according to relevant national and international guidelines. Animal Care and Use Committee of the National Eye Institute approved all mouse protocols. See also <xref ref-type="supplementary-material" rid="pgen.1002649.s011">Text S1</xref>.</p><p>Retina from postnatal day (P) 28 C57Bl/6J mice was used for ChIP experiments with NRL antibody or normal IgG, as previously described <xref ref-type="bibr" rid="pgen.1002649-Oh2">[41]</xref>. Fifteen or 25 ng of ChIP DNA from parallel experiments was used for library preparation and sequencing on Illumina 1G Genome Analyzer or ABI SOLiD V2 system, respectively.</p><p>Raw sequencing reads from Illumina or ABI platforms were mapped to the mouse genome (NCBI build 37) using Genomatix Mining Station (GMS).</p><p>ChIP&#8211;Seq peaks were called using MACS <xref ref-type="bibr" rid="pgen.1002649-Zhang1">[28]</xref> and NGS-Analyzer (Genomatix). The union of overlapping peak regions from both methods was used in subsequent analyses.</p><p>The NRL ChIP&#8211;Seq peaks were compared to the CRX ChIP&#8211;Seq regions <xref ref-type="bibr" rid="pgen.1002649-Corbo1">[22]</xref> using GenomeInspector (Genomatix; <xref ref-type="bibr" rid="pgen.1002649-Quandt1">[31]</xref>).</p><p>Affymetrix microarray data from flow-sorted photoreceptors of WT and <italic>Nrl</italic>
<sup>&#8722;/&#8722;</sup> retina <xref ref-type="bibr" rid="pgen.1002649-Akimoto1">[25]</xref> was analyzed using ChipInsepector (Genomatix).</p><p>Transcription factor binding site enrichment analyses of sequences in ChIP&#8211;Seq peak regions were performed using RegionMiner and MatInspector (Genomatix; <xref ref-type="bibr" rid="pgen.1002649-Quandt1">[31]</xref>, <xref ref-type="bibr" rid="pgen.1002649-Cartharius1">[32]</xref>). The ChIP&#8211;Seq peaks were extended to &#8722;500 bp and +500 bp from the peak center. The sequences were scanned for TFBS matrices (Genomatix MatBase version 8.2) using MatInspector (Genomatix). Positional bias (P) was calculated <xref ref-type="bibr" rid="pgen.1002649-Hughes1">[76]</xref>, and the &#8211;log(P) was plotted against the scan windows' mid-positions. The over-represented TFBS positions for a TF family appear as peaks in these plots.</p><p>ChIP DNA was tested in triplicates by qPCR using SYBR Green <xref ref-type="bibr" rid="pgen.1002649-Zipper1">[77]</xref>. We randomly tested 26 regions with peaks covering the majority range of the peak heights. Five regions without ChIP&#8211;Seq signals served as negative controls. Normal IgG served as the negative antibody control, and <italic>Nrl</italic>
<sup>&#8722;/&#8722;</sup> retina was used as a negative tissue control. The complete ChIP-qPCR procedure was performed twice.</p><p>HEK293T cells were cultured in DMEM and transfected with Fugene 6 (Roche).</p><p>To generate shRNA-resistant dcDNA constructs, silent mutations that confer resistance to shRNA were introduced into <italic>Lman1</italic> cDNA and <italic>Wisp1</italic> cDNA using Quikchange kit (Stratagene).</p><p>To generate enhancer constructs, ChIP&#8211;Seq peak regions were amplified and cloned into pGL3-promoter vector (Promega). HEK293T cells were transfected with these enhancer constructs, a transfection control plasmid expressing Renilla luciferase (Promega), and NRL expression plasmid or empty vector. The luciferase activities were measured 48 hr after transfection. The experiments were performed three times.</p><p>To generate shRNA-sensor constructs for efficiency test, we cloned the shRNA target sequences into the 3&#8242;UTR of a GFP vector. The shRNA-sensor construct and CAG-HcRed (transfection control) were co-transfected with either shRNA against target or <italic>Gapdh</italic> shRNA, For each target gene, three shRNA constructs were evaluated for efficacy indicated by a decrease in GFP. The most efficient one was chosen for <italic>in vivo</italic> knockdown experiments.</p><p>shRNA alone or together with shRNA-resistant dcDNA was introduced in the retina of CD-1 P0 mouse pups by sub-retinal injection followed by <italic>in vivo</italic> electroporation, as previously described <xref ref-type="bibr" rid="pgen.1002649-Kautzmann1">[44]</xref>, <xref ref-type="bibr" rid="pgen.1002649-Matsuda1">[45]</xref>. The retinas were harvested at P7 or P20 for histology or immunohistochemistry.</p><p>Mouse retina was electroporated at P0 with Ub-GFP and <italic>Gapdh</italic> shRNA or <italic>Kdm5b</italic> shRNA and dissected at P20. GFP+ retinal cells were isolated from dissociated retina by FACS (FACSAria; BD Biosciences). RNA was extracted and cDNA was synthesized followed by sense transcript cDNA (ST-cDNA) generation using WT-Ovation Exon module (NuGEN Technologies). The ST-cDNA was fragmented and labeled with Encore Biotin Module (NuGen) and used for hybridization with GeneChip Mouse Exon 1.0 ST array (Affymetrix). The microarray data has been deposited in the Gene Expression Omnibus Database (accession #: will be available soon).</p>