<p>The generation of <italic>Nth1</italic> null (<italic>Nth1<sup>&#8722;/&#8722;</sup></italic>) and <italic>Tert</italic> null (<italic>Tert<sup>&#8722;/&#8722;</sup></italic>) mice was described previously <xref ref-type="bibr" rid="pgen.1003639-Liu2">[52]</xref>, <xref ref-type="bibr" rid="pgen.1003639-Elder1">[53]</xref>. <italic>Nth1<sup>&#8722;/&#8722;</sup></italic> mice were further backcrossed into C57BL/6 background. Wild-type and <italic>Nth1<sup>&#8722;/&#8722;</sup></italic> mice were derived from heterozygous (<italic>Nth1<sup>+/&#8722;</sup></italic>) breeders. To generate mice deficient for both <italic>Nth1</italic> and <italic>Tert</italic>, <italic>Nth1</italic>
<sup>&#8722;/&#8722;</sup> mice was bred with <italic>Tert</italic>
<sup>&#8722;/&#8722;</sup> mice to obtain <italic>Nth1</italic>
<sup>+/&#8722;</sup>
<italic>Tert</italic>
<sup>+/&#8722;</sup>, which were subsequently bred to generate <italic>Nth1</italic>
<sup>+/+</sup>
<italic>Tert</italic>
<sup>&#8722;/&#8722;</sup> and <italic>Nth1</italic>
<sup>&#8722;/&#8722;</sup>
<italic>Tert</italic>
<sup>&#8722;/&#8722;</sup> mice. Primary mouse embryonic fibroblasts (MEFs) were isolated from 13.5 day embryos of <italic>Nth1<sup>+/&#8722;</sup></italic> female bred with <italic>Nth1<sup>+/&#8722;</sup></italic> male and cultured in Dulbecco's Modified Eagle Medium containing 10% fetal bovine serum in CO<sub>2</sub> incubator in the presence of 20% oxygen. Bone marrow cells were flushed from femurs and tibias and cultured in Iscove's modified Dulbecco's medium (Invitrogen) supplemented with 20% fetal calf serum with interleukin 6 (200 U/mL; Peprotech) and stem cell factor (100 ng/mL; Peprotech). Single cell suspensions of spleen were obtained by passing the spleen suspension through a cell strainer (70 &#181;m, BD Falcon). All animal experiments were carried out according to the &#8220;Guide for the Care and Use of Laboratory Animals&#8221; (National Academy Press, USA, 1996), and were approved by the Institutional Animal Care and Use Committee of National Institute on Aging.</p><p>Identification of oxidative base lesions in telomeres was performed as previously described <xref ref-type="bibr" rid="pgen.1003639-OCallaghan1">[27]</xref>. In brief, DNA was isolated from mouse kidney and primary MEFs by salting out <xref ref-type="bibr" rid="pgen.1003639-Lu1">[22]</xref>, <xref ref-type="bibr" rid="pgen.1003639-Wang1">[26]</xref>. To excise oxidative base lesions and generate strand breaks at the resulting abasic sites, DNA was digested with Endonuclease III. 400 ng of duplex oligomer or genomic DNA were incubated overnight with 12 units of Endonuclease III (New England Biolabs) in 1&#215; NEB Endonuclease III buffer (20 mM Tris-HCl, 1 mM EDTA, 1 mM Dithiothreitol, pH 8.0). A duplicate digestion was also set up for each corresponding sample in mock digestion buffer (i.e. enzyme was excluded and substituted with H<sub>2</sub>O). All samples were set up on ice, then incubated at 37&#176;C overnight to allow complete digestion and followed by the quantitative Real-Time amplification (qPCR) <xref ref-type="bibr" rid="pgen.1003639-OCallaghan2">[54]</xref>. Five oligonucleotides containing <named-content content-type="gene">TTAGGG</named-content> repeats with 0, 1, 2, 4 or 8 thymine glycols were used for generation of a standard curve (<xref ref-type="supplementary-material" rid="pgen.1003639.s008">Table S1</xref>). A reverse oligonucleotide was used to construct duplex substrates (GeneWorks, Adelaide). Forward and reverse oligonucleotides were mixed in a 1&#8758;1 molar ratio. Annealing reactions were incubated at 95&#176;C for 10 min and then cooled to room temperature for 30 min. For qPCR, each 20 &#181;L reaction was composed as follows: 40 pg of digested or undigested oligonucleotides or 40 ng of digested or undigested genomic DNA, 1&#215; SYBR Green master mix, 100 nM telo1 forward and 100 nM telo2 reverse primers <xref ref-type="bibr" rid="pgen.1003639-OCallaghan2">[54]</xref>. All samples were run on an ABI 7300 Sequence Detection System with the SDS Ver. 1.9 software (Applied Biosystems). Cycling conditions were: 10 minutes at 95&#176;C, followed by 40 cycles of 95&#176;C for 15 seconds and 60&#176;C for one minute. Each sample was analyzed in triplicate. PCR efficiencies and correlation coefficients for genomic DNA and synthetic oligonucleotides are shown in <xref ref-type="supplementary-material" rid="pgen.1003639.s009">Table S2</xref>. The Ct values for all the samples were obtained from PCR reactions run under the same conditions using the same reagents, and the &#916;CT value (C<sub>t</sub> treated - C<sub>t</sub> mock) for each sample were converted into numbers of Endonuclease III-sensitive lesions by comparison to the standard curve, as described previously <xref ref-type="bibr" rid="pgen.1003639-OCallaghan1">[27]</xref>.</p><p>Flow-FISH: The average telomere fluorescence in populations of splenocytes and bone marrow cells was measured according to a previously published protocol <xref ref-type="bibr" rid="pgen.1003639-Rufer1">[55]</xref>. In each set, data were pooled from at least five individual mice. A telomere-specific FITC conjugated (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> PNA probe (0.3 &#181;g/mL, Panagene) was employed and telomere fluorescent signal intensity was measured by Accuri O6 flow cytometer using FlowJo software.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Q-FISH: Mice were injected with 100 &#181;l of 0.5% colchicine intraperitoneally for approximately 30 minutes before being sacrificed. Bone marrow cells were then collected by flushing 1 ml of PBS from femurs. Cultured bone marrow cells and primary MEFs were incubated with 0.1 &#181;g/mL colcemid for 2&#8211;6 hours at 37&#176;C to allow mitotic cells to accumulate. Metaphase spreads were obtained by incubating mouse cells in 0.075 M KCl for 15 minutes at 37&#176;C, followed by fixing cells in ice-cold (3&#8758;1) methanol and glacial acetic acid and dropping the fixed cells onto slides. Metaphase spreads were hybridized with Cy3-labeled (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> (0.3 &#181;g/mL, Panagene), washed, and then counterstained with 4, 6 diamidino-2-phenylindole (DAPI) as previously described <xref ref-type="bibr" rid="pgen.1003639-Zijlmans1">[56]</xref>. Images were captured using Cytovision software (Applied Imaging Corp.) on a fluorescence microscope (Axio2; Carl Zeiss, Germany); followed by quantification of individual telomere fluorescence signals using the TFL-Telo software (a kind gift from P. Lansdorp, Vancouver, BC). For histograms and box/jitter plots, data from different mice of each genotype were scored using R statistical package (<ext-link ext-link-type="uri" ns0:href="http://www.r-project.org/">http://www.r-project.org/</ext-link>) and Graphpad software. Metaphases from different mice of each genotype were scored for fragile telomeres (a chromatid with &#8805;2 telomere signals) and SFEs (chromosome ends without detectable telomere signals).</p><p>CO-FISH was used to measure T-SCEs as described previously <xref ref-type="bibr" rid="pgen.1003639-Bailey1">[44]</xref>. Briefly, bone marrow cells were cultured in medium with 3&#8758;1 ratio of BrdU/BrdC at a final concentration of 1&#215;10<sup>&#8722;5</sup> M for approximately 12 hours. Colcemid (0.1 &#181;g/ml) was added for the final 2 hours. Metaphase spreads were prepared as described above, stained with Hoechst 33258, exposed to UV light and then digested with exonuclease III to remove newly synthesized strands. Remaining parental strands were hybridized with Cy3-labeled (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> probe, then briefly washed with hybridization buffer and subsequently with Alexa 488&#8211;labeled (<named-content content-type="gene">TTAGGG</named-content>)<sub>3</sub> probes. Hybridization and wash conditions were identical to those described for telomeric FISH. A chromosome with more than two telomeric DNA signals by both probes was scored as T-SCE positive.</p><p>Fibroblasts were grown overnight on chamber slides and bone marrow cells were spun onto Cytospin microscopic slides at 200 rpm for 3 minutes. Cells were washed in PBS, fixed in 2% paraformaldehyde for 10 minutes at room temperature, permeabilized with 0.5% tritonX-100 for 5 minutes on ice and blocked with 10% FBS for 1&#8211;2 hours. Cells were stained overnight at 4&#176;C with a rabbit anti-&#947;H2AX antibody (1&#8758;200, Santa cruz) or a rabbit anti-53BP1 antibody (1&#8758;500, Novus Biologicals), followed by Alexa 488-labeled secondary antibody (1&#8758;500; Molecular Probes) for one hour at room temperature. Cells were washed in PBS, fixed in 2% paraformaldehyde for 10 minutes at room temperature, washed, dehydrated through ethanol series and briefly dried. Slides were immediately hybridized to Cy3-labeled (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> probe (Panagene) for 2 hours at room temperature. Hybridization buffer and wash conditions were identical to those described for telomeric FISH. Slides were counterstained with DAPI. Z-stack images were captured on a fluorescence microscope (Axiovert 200M; Carl Zeiss).</p><p>Apoptosis was quantified using the Annexin V: FITC Apoptosis Detection Kit as per the manufacturer's instructions (BD Pharmingen). Briefly, cells were harvested, washed twice with ice-cold PBS and resuspended in 1&#215; Binding buffer supplied by the manufacturer. Cells were incubated with 5 &#181;L PI and 5 &#181;L FITC-Annexin V at room temperature for 15 minutes and analyzed using an Accuri O6 flow cytometer with FlowJo software.</p><p>Cells were exposed to 5 &#181;M aphidicolin for 8 hours or 2 mM HU for 24 hours as a positive control for Chk1 phosphorylation or 10 Gy Ionizing radiation with 1 hour recovery as a positive control for Chk2 phosphorylation <xref ref-type="bibr" rid="pgen.1003639-Brown1">[37]</xref>. Cells were lysed with lysis buffer (50 mM Tris HCl, pH 7.4, 150 mM NaCl, 1% IGEPAL, 0.25% sodium deoxycholate, 0.5 M EDTA, 20% SDS) supplemented with complete, EDTA-free Protease Inhibitor Cocktail Tablet and PhosSTOP Phosphatase Inhibitor Cocktail Tablet (Roche). Approximately 40&#8211;100 &#181;g cell extracts were examined for phosphorylated-Chk1 or Chk2 and actin by rabbit phospho-Chk1 (S345) (1: 1000, Cell signaling, 2341), mouse Anti-Chk2 (1&#8758;500, BD Biosciences, 611570) or actin (1&#8758;1000, Santa Cruz, SC-1616) antibodies.</p>