<p>The 6,832 African Americans studied are comprised of individuals from the Atherosclerosis Risk in Communities Study (ARIC) <xref ref-type="bibr" rid="pgen.1003379-The2">[62]</xref>, the Multiethnic Cohort Study (MEC) <xref ref-type="bibr" rid="pgen.1003379-Kolonel1">[63]</xref>, and the Women's Health Initiative (WHI) <xref ref-type="bibr" rid="pgen.1003379-The3">[64]</xref>, <xref ref-type="bibr" rid="pgen.1003379-Anderson1">[65]</xref> that are part of Population Architecture using Genomics and Epidemiology (PAGE) consortium <xref ref-type="bibr" rid="pgen.1003379-Matise1">[66]</xref> and from Hypertensive Genetic Epidemiology Network (HyperGEN) <xref ref-type="bibr" rid="pgen.1003379-Williams1">[67]</xref>. The 9,449 East Asian samples are comprised of 1,716 Filipinos from the Cebu Longitudinal Health and Nutrition Survey (CLHNS) <xref ref-type="bibr" rid="pgen.1003379-Adair1">[68]</xref> and 7,733 Chinese from Taiwan-Metabochip Study for Cardiovascular Disease (TAICHI). The 10,829 European samples are comprised of Finnish and Norwegian individuals; the Finns are from the Finland-United States Investigation of NIDDM Genetics (FUSION), Dehko 2D 2007 (D2D2007), Diabetes Prevention Study (DPS), Dose-Responses to Exercise Training (DR's EXTRA), and Metabolic Syndrome in Men (METSIM) <xref ref-type="bibr" rid="pgen.1003379-Scott1">[69]</xref>, <xref ref-type="bibr" rid="pgen.1003379-Stancakova1">[70]</xref>, and the Norwegians were from the cohorts of Nord-Tr&#248;ndelag Health Study (HUNT 2) and the Troms&#248; Study (TROMSO) <xref ref-type="bibr" rid="pgen.1003379-Midthjell1">[71]</xref>, <xref ref-type="bibr" rid="pgen.1003379-Joseph1">[72]</xref>.</p><p>All study protocols were approved by Institutional Review Boards at their respective sites. Brief descriptions of the studies are provided in the <xref ref-type="supplementary-material" rid="pgen.1003379.s012">Text S1</xref>. General characteristics and measurements of TG, HDL-C, and LDL-C in each cohort are summarized in <xref ref-type="supplementary-material" rid="pgen.1003379.s007">Table S1</xref>. Values of triglycerides were natural log transformed to approximate normality in each study sample separately.</p><p>We genotyped all study samples with the Metabochip according to the manufacturer's protocol (Illumina, San Diego, CA, USA). <xref ref-type="supplementary-material" rid="pgen.1003379.s007">Table S1</xref> summarizes the quality control criteria of genotyping, including call rate, sample success rate, Hardy-Weinberg equilibrium, and MAF that varied across studies.</p><p>We applied multiple linear regression models and assumed an additive mode of inheritance to test for association between genotypes and HDL-C, LDL-C, or log-transformed triglycerides. We performed each test of association separately in each of the 11 groups (<xref ref-type="supplementary-material" rid="pgen.1003379.s007">Table S1</xref>) prior to meta-analysis. We constructed principal components (PCs) using the software EIGENSOFT. We used age and sex as covariates in each individual cohort; other cohort-specific covariates including age<sup>2</sup>, enrollment site, socioeconomic status, and principal components varied across studies (<xref ref-type="supplementary-material" rid="pgen.1003379.s007">Table S1</xref>). The European samples include type 2 diabetes (T2D) cases and unaffected controls; to avoid confounding due to T2D status, samples were analyzed separately as Finnish T2D patients, Finnish unaffected individuals, Norwegian T2D patients, and Norwegian unaffected individuals.</p><p>We first conducted the meta-analysis within the African Americans, East Asians, and Europeans separately. We then performed combined trans-ethnic meta-analyses by combining the statistics of each the 11 participating groups to assess the association with the SNPs at the 58 lipids loci.</p><p>At loci that exhibited evidence of association at <italic>P</italic>&lt;10<sup>&#8722;4</sup>, we next performed a series of sequential conditional analyses by adding the most strongly associated SNP into the regression model as a covariate and testing all remaining regional SNPs for association. We conducted a set of sequential conditional analyses until the strongest SNP showed a conditional <italic>P</italic> value&gt;10<sup>&#8722;4</sup> and had no annotation or literature evidence that suggested a functional role.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For single SNP analyses, we applied PLINK (<ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/~purcell/plink/">http://pngu.mgh.harvard.edu/~purcell/plink/</ext-link>) <xref ref-type="bibr" rid="pgen.1003379-Purcell1">[73]</xref> for population-based studies. We used the R package GWAF <xref ref-type="bibr" rid="pgen.1003379-Chen1">[74]</xref> for the family-based study of HyperGEN. We applied an inverse variance-weighted fixed-effect meta-analysis implemented in METAL <xref ref-type="bibr" rid="pgen.1003379-Willer2">[75]</xref>.</p><p>Unless otherwise noted, linkage disequilibrium estimates were obtained from the 1000 Genomes Project November 2010 release. SNP positions correspond to hg18.</p><p>We performed haplotype analysis at LDL-C locus <italic>TOMM40-APOE-APOC4</italic> in 5,593 unrelated African Americans from the PAGE consortium, using the &#8216;haplo.stat&#8217; R package. Haplotypes and haplotype frequencies were estimated using the R function &#8216;haplo.em&#8217;. The association between haplotypes and LDL-C was assessed using the R function &#8216;haplo.glm&#8217;. An additive model was assumed, in which the regression coefficient &#946; represents the expected change in LDL-C level with each additional copy of the specific haplotype compared with the reference haplotype, which was set as the A-A (trait increasing-increasing) haplotype.</p><p>We created the regional association plots using LocusZoom <xref ref-type="bibr" rid="pgen.1003379-Pruim1">[76]</xref>. To plot the association results in Europeans and East Asians, we used the LocusZoom-implemented LD estimates from the 1000 Genomes Project (June 2010) CEU and CHB+JPT samples, whose LD structures are similar to our samples with European and East Asian ancestries. We applied the user-supplied LD calculated from the genotype data of the PAGE African American samples to plot the regional association in African Americans <xref ref-type="bibr" rid="pgen.1003379-Buyske1">[9]</xref>, because the LD patterns may vary from any pre-computed LD sources implemented in LocusZoom.</p><p>We evaluated the proportion of variance explained by a single SNP or any given locus by including the SNP or a set of SNPs into a linear regression model with all covariates used in association analysis and calculating the R<sup>2</sup> for the full model. We subtracted the variance explained by a basic model in which only covariates were included from the variance we obtained from the full model. We performed these analyses using SAS version 9.2 (SAS Institute, Cary, NC, USA).</p>