<p>All <italic>E. coli</italic> strains used are listed in <xref ref-type="table" rid="pgen-1004217-t001">Table 1</xref>. Standard genetic methods including transformation and P1 <italic>vir</italic> transduction were used for strain construction. Recombineering methods for strain construction <xref ref-type="bibr" rid="pgen.1004217-Sawitzke1">[77]</xref>, <xref ref-type="bibr" rid="pgen.1004217-Thomason1">[78]</xref> are described in a section below.</p><p>Cells were grown in Luria-Bertani (LB) medium at 30&#176;C or 32&#176;C, as indicated, for temperature-sensitive (<italic>ts</italic>) strains under permissive conditions and 42&#176;C under non-permissive conditions, or at 37&#176;C for all non-<italic>ts</italic> strains. Optical density readings at 600 nm (OD<sub>600</sub>) were measured using a UV-1601 or UV-1800 spectrophotometer (Shimadzu). LB medium was supplemented with ampicillin (50 &#181;g/ml; Fisher Scientific), kanamycin (50 &#181;g/ml; Sigma-Aldrich), chloramphenicol (20 &#181;g/ml; Acros Organics), tetracycline (10 &#181;g/ml; Sigma-Aldrich), and spectinomycin (100 &#181;g/ml; Sigma-Aldrich), as needed. Gene expression from vectors derived from pET28-, pET15- (Novagen &#8211; EMD Millipore), and pRR48 <xref ref-type="bibr" rid="pgen.1004217-Studdert1">[79]</xref> was induced with 1 mM isopropyl-&#946;-D-galactopyranoside (IPTG) (Fisher Scientific). Gene expression from pBAD33-derived vectors <xref ref-type="bibr" rid="pgen.1004217-Guzman1">[80]</xref> was induced at a final concentration of 0.2% L-(+)-arabinose (Ara) (Sigma-Aldrich). Gene expression from pKG110-derived vectors (J.S. Parkinson, University of Utah) was either uninduced or induced with 0.5 &#181;M sodium salicylate (Mallinkrodt), as indicated. A PCR test <xref ref-type="bibr" rid="pgen.1004217-Powell1">[81]</xref> confirmed that LT447 and LT1566 were monolysogens.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Standard protocols or manufacturers' instructions were used to isolate plasmid DNA, as well as for restriction endonuclease, DNA ligase, PCR, and other enzymatic treatments of plasmids and DNA fragments. Enzymes were purchased from New England BioLabs, Inc. (NEB) or Invitrogen. Plasmid DNA was prepared using the Wizard Plus SV Minipreps DNA Purification Kit, PCR and digest reactions were cleaned up using the Wizard SV Gel and PCR Clean-up System, and chromosomal DNA was prepared using the Wizard Genomic DNA Purification Kit (Promega). KAPA HiFi HotStart DNA polymerase (Kapa Biosystems) or Phusion High-Fidelity DNA polymerase (Thermo Scientific &#8211; NEB) or Platinum Taq (Invitrogen) were used for PCR reactions in a MyCycler Thermal Cycler (Bio-Rad). Oligonucleotides were purchased from Sigma-Aldrich or IDT and their sequences are listed in <xref ref-type="supplementary-material" rid="pgen.1004217.s001">Table S1</xref>. The final versions of all cloning products were sequenced to verify their construction. DNA sequencing was performed by GeneWiz (South Plainfield, NJ), SeqWright (Houston, TX) or SAIC-Frederick, Inc. (Frederick, MD). DNA bands were visualized using an AlphaImager Mini System (ProteinSimple) and DNA concentrations were estimated with a NanoDrop ND-1000 Spectrophotometer (Thermo Scientific). Protein concentrations were determined using the Coomassie Plus Assay (Themo Scientific &#8211; Pierce). Molecular graphics and analyses were performed with the UCSF Chimera package (<ext-link ext-link-type="uri" ns0:href="http://www.cgl.ucsf.edu/chimera">http://www.cgl.ucsf.edu/chimera</ext-link>). Chimera is developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco (supported by NIGMS P41-GM103311).</p><p>All plasmids are listed in <xref ref-type="table" rid="pgen-1004217-t002">Table 2</xref>.</p><p>Kil expression plasmids: For pRR48-<italic>kil</italic>, the <italic>kil</italic> coding sequence was amplified from CC4506 chromosomal DNA with primers DPH199 and 200; the PCR product was digested with <italic>Pst</italic>I and <italic>Spe</italic>I, then ligated into a pRR48-derivative where the original multiple cloning site had been replaced (<italic>Nde</italic>I to <italic>Sal</italic>I) with that of pKG110. For pDH104 (pBAD33-<italic>kil</italic>), the <italic>kil</italic> coding sequence with its native ribosome-binding site was amplified from CC4506 chromosomal DNA with primers DPH217 and 202; the PCR product was digested with <italic>Xma</italic>I and <italic>Pst</italic>I, then ligated into pBAD33.</p><p>Toxic overexpression of <italic>minCD</italic>: For pWM2737 (pDSW210-<italic>his</italic>-<italic>minCD</italic>) the <italic>minD</italic> coding sequence was amplified with primers WM960 &amp; 356, the PCR product was digested with <italic>Sal</italic>I and <italic>Hind</italic>III, then ligated into pWM2735 (pDSW210-<italic>his<sub>6</sub>-minC</italic>) <xref ref-type="bibr" rid="pgen.1004217-Shiomi1">[44]</xref> to create sequence encoding an uninterrupted His-tagged, MinCD translational fusion (<italic>minCD<sub>tf</sub></italic>).</p><p>Complementation of &#916;<italic>zipA</italic> cells with the <italic>zipA<sub>L286Q</sub></italic> allele: pDH145 (pKG110-<italic>zipA<sub>L286Q</sub></italic>) was constructed as previously published for pKG110-<italic>zipA</italic>
<xref ref-type="bibr" rid="pgen.1004217-Shiomi2">[82]</xref>, but using DPH615 (<italic>zipA<sub>L286Q</sub></italic>) chromosomal DNA as template. Plasmid pKG110-<italic>zapA</italic> was constructed by Daisuke Shiomi in the same manner as pKG116-<italic>zapA</italic>
<xref ref-type="bibr" rid="pgen.1004217-Shiomi2">[82]</xref>.</p><p>Plasmids for protein purifications: For pET28-<italic>zipA<sub>C</sub></italic> the C-terminal ZipA domain coding sequence was amplified by Tushar Beuria with primers WM269 &amp; 268; the PCR product was digested with <italic>BamH</italic>I and <italic>Xho</italic>I, then ligated into pET28a. As primer WM268 did not include a stop codon, this construct produces His<sub>6</sub>-ZipA<sub>C</sub>-His<sub>6</sub>. Construction of pDH146 (pET28-<italic>zipA*<sub>C</sub></italic>) was therefore performed identically, but with DPH615 chromosomal DNA as template, to produce a double His-tagged version of ZipA<sub>C</sub> that harbors the L286Q mutation. For pDH139 (pET28-<italic>flag-kil</italic>) the <italic>kil</italic> coding sequence with its stop codon was amplified from CC4506 chromosomal DNA in two steps (primers DPH309 &amp; 170, followed by DPH310 &amp; 170) to add sequence encoding a FLAG-tag to the 5&#8242; end of <italic>kil</italic>, in-frame with the His-tag-encoding codons of the plasmid. The resulting product was digested with <italic>BamH</italic>I and <italic>Hind</italic>III, then ligated into pET28a. Plasmid pDH149 (pET15-<italic>flag-kil</italic>) was constructed by digesting pDH139 with <italic>Nco</italic>I and <italic>Xho</italic>I, then ligating the <italic>his<sub>6</sub>-flag-kil</italic> insert into identically-digested pET15b.</p><p>Recombineering was done using published methods <xref ref-type="bibr" rid="pgen.1004217-Sawitzke1">[77]</xref>, <xref ref-type="bibr" rid="pgen.1004217-Thomason1">[78]</xref>. Briefly, the Red functions were expressed either from the defective prophage or from a recombineering plasmid such as pSIM18, a hygromycin resistant plasmid encoding the temperature-sensitive CI857 repressor and a portion of the <italic>P<sub>L</sub></italic> operon containing <italic>gam</italic>, <italic>exo</italic>, and <italic>bet</italic>. Log phase cells propagated at 32&#176;C were subjected to a 15 minute heat pulse in a 42&#176;C shaking water bath to induce the Red functions, quickly chilled in an ice-water slurry, and prepared for electroporation by washing. DNA, either double- or single-stranded, was introduced into the cells by electroporation. When a drug marker was selected, the cells were allowed to recover for several hours in 1 mL broth before plating. When point mutations were introduced with &#8764;70 base oligonucleotides, the cells were plated non-selectively on LB agar plates after a 30 minute recovery time, either at 42&#176;C for direct selection of Kil-resistant alleles or at 32&#176;C for screening by PCR, as described below. In some cases oligonucleotides with additional sequence changes in third positions (wobble positions) were used to create mutations. As described <xref ref-type="bibr" rid="pgen.1004217-Sawitzke2">[83]</xref>, these additional wobble changes were placed near the change of interest, creating a configuration of mispairs that is not recognized by the <italic>E. coli</italic> mismatch repair system, without changing the amino acid sequence. This allows both high-efficiency recombineering and detection of recombinant chromosomes with PCR. When wobble changes were introduced, between 12&#8211;20 colonies were analyzed, using one oligonucleotide that hybridized specifically to the recombinant sequence, paired with a second nearby oligonucleotide; this primer pair should not yield product with the parental sequence. Positive candidates were purified to single colonies and about a dozen of these single colonies were again subjected to the same PCR analysis. The region of interest from positive candidates identified in the second round of PCR screening was sequenced. Sequences for oligonucleotides used are listed in <xref ref-type="supplementary-material" rid="pgen.1004217.s001">Table S1</xref>.</p><p>A seven-codon in-frame deletion within <italic>kil</italic>, removing codons 20&#8211;26, was initially constructed in XTL241 (HME6 <italic>gam&lt;&gt;cat-sacB</italic>) and then moved to a phage. The deletion was built into a hybrid oligonucleotide, LT793, which along with an oligonucleotide to the downstream <italic>gam</italic> gene, LT795, were used to amplify a PCR product containing the deletion and spanning the region containing the counter-selectable marker. Recombineering was used to replace the <italic>cat-sacB</italic>. After introduction of the PCR product into electrocompetent XTL241 induced for the Red functions, cultures were grown overnight and recombinants were selected on sucrose plates at 32&#176;C. Candidate colonies were purified on sucrose and confirmed to be chloramphenicol-sensitive. Primers AW24 and LT795 were used to amplify the region containing the deletion, which was confirmed by sequencing. In contrast to the <italic>kil</italic>
<sup>+</sup> control, cells containing the deletion did not filament after several hours of growth at 42&#176;C, and formed colonies after overnight growth at 42&#176;C. A lysate of &#955; <italic>c</italic>I<italic>857red3</italic> was grown on cells containing the seven-codon deletion to allow marker rescue from the defective prophage onto the phage. The lysate was plated on LT352. Plaques were picked into 0.5 mL TMG (10 mM Tris base, 10 mM MgS0<sub>4</sub> 0.01% gelatin) and 2 &#181;l of the pickate was used for PCR with oligonucleotides AW24 and LT795: the difference in size between the deletion and the wild-type alleles was resolvable on a 1.2% agarose gel. The deletion phage was purified by another round of plating on LT352 and a high titer plate lysate was grown from a purified plaque on C600.</p><p>CC4506 was transformed with a pBR322-based library containing 1.5&#8211;5 kb <italic>Sau</italic>3A fragments of LE392 genomic DNA cloned into pHDB3 <xref ref-type="bibr" rid="pgen.1004217-Ulbrandt1">[84]</xref> and transformants were selected on LB ampicillin agar at 42&#176;C. Ampicillin-resistant colonies that survived Kil induction were pooled, plasmid DNA was isolated and CC4506 was transformed with this enriched plasmid population to confirm suppression. The <italic>Sau</italic>3A library was kindly provided by Nadim Magdalani (NIH). Plasmid pJP10 was isolated by this procedure. Plasmid pNB15 was isolated by a similar procedure except the pBR322-based library was constructed by cloning 2&#8211;4 Kb <italic>Sau</italic>3A fragments of JS549 genomic DNA into pBR322 cut with <italic>Bam</italic>HI.</p><p>The <italic>ftsZ</italic> gene was amplified from W3110 with oligonucleotides XMZ325 and XMZ326, using standard PCR conditions. This <italic>ftsZ</italic> PCR product was used as a template for mutagenic PCR <xref ref-type="bibr" rid="pgen.1004217-Wilson1">[85]</xref> with the same oligonucleotides. This randomly mutagenized pool of <italic>ftsZ</italic> PCR fragments was used for recombineering into strain AW34, which carries the thermo-sensitive <italic>ftsZ84</italic> allele (G105S) linked to <italic>leuD::</italic>Tn<italic>10</italic>, a defective &#955; prophage with thermo-inducible <italic>kil</italic> expression, and a <italic>mutS&lt;&gt;amp</italic> allele to prevent host mismatch repair during recombination. After a 30 minute recovery in broth at 30&#176;C, aliquots were plated on nitrocellulose filters on LB plates and incubated at 32&#176;C for 3 hours. Filters were then transferred to 42&#176;C pre-warmed LB plates and these plates were subsequently incubated overnight at 42&#176;C to select for growth. This procedure simultaneously selects for recombinants that replace <italic>ftsZ84</italic> with the wild type allele and for resistance to &#955; Kil expression. A lysate of P1 <italic>vir</italic> was grown on a pool of the temperature-resistant isolates. This lysate was used to transduce the Kil-expressing strain AW41, selecting tetracycline resistance at 42&#176;C. Thermo-resistant colonies were purified and their <italic>ftsZ</italic> gene was amplified with colony PCR; the resulting PCR products were sequenced. Several isolates of the <italic>ftsZ</italic>
<sub>V208A</sub> mutation were obtained, and only one isolate of <italic>ftsZ<sub>L169R</sub></italic>. One isolate of each of the mutant types was chosen for further characterization. The mutations were introduced into a fresh background by P1 transduction at 32&#176;C, selecting for the linked <italic>leuD&lt;&gt;</italic>Tn<italic>10</italic> and screening for temperature resistance.</p><p>Competent WM1074 cells were sequentially transformed with pBAD33-<italic>kil</italic> and pRR48-<italic>kil</italic> and the resulting strain was verified to have a <italic>kil<sup>+</sup></italic> phenotype upon induction of <italic>kil</italic> from either plasmid. A culture of this &#8216;double <italic>kil</italic>&#8217; strain was grown to mid-logarithmic phase and induced with IPTG and arabinose simultaneously. 100 &#181;L of stationary phase culture was plated on LB agar supplemented with appropriate antibiotics, IPTG, and arabinose and grown overnight at 37&#176;C. All resulting colonies were purified, then grown in liquid media to freeze samples. Chromosome purification and PCR amplification of the <italic>zipA</italic> locus in the saved isolates showed that all contained a <italic>zipA<sub>L286Q</sub></italic> mutation, while the original WM1074 pBAD33-<italic>kil</italic> pRR48-<italic>kil</italic> strain contained <italic>zipA<sub>WT</sub></italic> sequence.</p><p>A second isolation of spontaneous Kil-resistant mutants was carried out using a similar protocol, with the only difference being that cells were grown in liquid culture under non-inducing conditions into stationary phase, then plated on medium containing IPTG and arabinose. All isolated colonies from this second experiment contained a <italic>zipA<sub>L286R</sub></italic> mutation.</p><p>Recombineering using ssDNA can be very efficient with up to 75% of the cells being recombinants <xref ref-type="bibr" rid="pgen.1004217-Sawitzke2">[83]</xref>. Therefore, recombineering with ssDNA can be used to isolate and map mutations, and these techniques were used throughout this work. For example, when mutagenic PCR was used to isolate <italic>zipA</italic> mutations, in one case sequencing revealed a double mutation, <italic>zipA<sub>A245T,Q290R</sub></italic>. We designed and ordered oligonucleotides to make each mutation separately via recombineering. Recombineering-proficient cells prepared on CC4506 were transformed with each oligonucleotide separately or both together, outgrown for 30 minutes, then diluted and spot-titered on LB plates at 30&#176;C and 42&#176;C. We found that only one of the oligonucleotides, MH82, was necessary and sufficient to create a mutation that suppressed Kil-dependent killing.</p><p>Cells used for spot titers were taken from the same cultures used for fixation unless as noted below. A ten-fold dilution of these cultures was taken approximately forty minutes after induction (or control conditions), unless otherwise noted, and then serially diluted into fresh LB media in a 96-well plate using a multichannel pipette. A flame-sterilized and cooled, metal-pronged tool was then used to replica-plate spots of serially diluted culture onto LB plates with added components and incubation conditions as indicated in the text and figures. Photos of plates were taken in a FluoroChem 8800 system with its accompanying camera and software (Alpha Innotech).</p><p>Spot titers in <xref ref-type="fig" rid="pgen-1004217-g004">Figures 4C</xref> and <xref ref-type="fig" rid="pgen-1004217-g007">7D</xref> were done as follows: An overnight culture was diluted 100-fold and cells were grown at 30&#176; for 2 hours in LB. Ten-fold serial dilutions of these cultures were made in TMG (see above) and 10 &#181;l was spotted on pre-warmed LB plates and incubated at the indicated temperatures.</p><p>To generate the data in <xref ref-type="fig" rid="pgen-1004217-g001">Figure 1D</xref>, an overnight culture of CC4506 was diluted 70-fold into 15 mL of LB broth and grown to an OD<sub>600</sub>&#8764;0.25 at 32&#176;. The culture was diluted in 10-fold increments from 10<sup>0</sup> to 10<sup>&#8722;5</sup>. At time &#8220;0&#8221;, 0.1 mL samples of appropriate dilutions were spread on prewarmed 42&#176; LB plates on which we had placed a sterile 82 mm (diameter) nitrocellulose filter. After the indicated time, sterile forceps were used to move the filter to a 32&#176; plate. These LB plates were incubated overnight at 32&#176; and colonies counted.</p><p>Overnight cultures were started from &#8722;80&#176;C strain stocks and grown under appropriate antibiotic selection and permissive conditions. Overnight cultures were diluted into fresh medium, grown to mid-logarithmic phase, then the OD<sub>600</sub> of individual cultures was adjusted to a uniform OD<sub>600</sub>&#8202;=&#8202;0.025 with fresh medium. Cultures were grown for approximately 2 doublings at permissive conditions and then shifted to non-permissive conditions (or kept as permissive controls) at time point zero. Unless otherwise indicated, samples of culture for microscopy were taken approximately 40 minutes after shifting to non-permissive conditions.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Cells were fixed with methanol and processed for immunofluorescence microscopy (IFM) as previously published <xref ref-type="bibr" rid="pgen.1004217-Levin1">[86]</xref> using lysozyme (Sigma) treatment for 5 minutes and antibodies diluted in bovine serum albumin (Fisher Scientific). Primary polyclonal rabbit &#945;-FtsZ <xref ref-type="bibr" rid="pgen.1004217-Yu2">[87]</xref> was used at 1/2500, secondary goat &#945;-rabbit-AlexaFluor 488 (Molecular Probes) and wheat germ agglutinin conjugated to rhodamine (Molecular Probes), to visualize cell wall, were used at 1/200. For DNA staining, 4&#8242;,6-diamidino-2-phenylindole (DAPI) (Molecular Probes) was used at 0.5 &#181;g/mL. Micrograph images were captured on an Olympus BX60 microsope with a Hamamatsu C8484 camera using HC Image software (Hamamatsu). Cell length and ring frequency measurements were taken with the ObjectJ extension <xref ref-type="bibr" rid="pgen.1004217-Vischer1">[88]</xref> (<ext-link ext-link-type="uri" ns0:href="http://simon.bio.uva.nl/objectj/">http://simon.bio.uva.nl/objectj/</ext-link>) of ImageJ <xref ref-type="bibr" rid="pgen.1004217-Schneider1">[89]</xref> (National Institutes of Health) using a minimum number of 100 cells and images were minimally adjusted for brightness/contrast using Adobe Photoshop CS4. Microsoft Excel 2008 for Mac (v. 12.3.6) was used for data tabulation and calculatio</p><p>Samples were prepared, subjected to SDS-PAGE on 20% acrylamide gels, and transferred as published <xref ref-type="bibr" rid="pgen.1004217-Busiek1">[90]</xref>. Transfer of His<sub>6</sub>-FLAG-Kil samples was done in 10 mM CAPS pH 10.0, 10% methanol transfer buffer with a two-hour transfer time. Mouse monoclonal &#945;-His primary antibody (Sigma-Aldrich) and affinity-purified rabbit polyclonal &#945;-FtsZ <xref ref-type="bibr" rid="pgen.1004217-Yu2">[87]</xref> were used at 1/5000. Affinity-purified rabbit polyclonal &#945;-ZipA <xref ref-type="bibr" rid="pgen.1004217-Geissler1">[49]</xref> was used at 1/1000. Goat &#945;-mouse and &#945;-rabbit secondary antibodies conjugated to horseradish peroxidase (HRP) (Sigma-Aldrich) were used at 1/10,000. A SuperSignal West Pico Chemiluminescent Substrate Kit (Thermo Scientific &#8211; Pierce) was used for HRP detection; blots were exposed to film and developed using a Konica SRX101A Film Processor (Konica Minolta).</p><p>Proteins were induced for purification from pET vectors (Novagen &#8211; EMD Millipore) in BL21(DE3) <xref ref-type="bibr" rid="pgen.1004217-Studier1">[91]</xref> backgrounds as 2 L cultures that were grown to OD<sub>600</sub>&#8764;0.7 at 30&#176;C, at which point 1 mM IPTG was added. (Volumes in this and subsequent steps were scaled down by a factor of 10 for copurification assays). Cultures were left overnight at 30&#176;C to induce protein and were harvested in the morning by spinning in a Beckman J2-21 centrifuge with a JA-17 rotor at 7,000 rpm. Cell pellets were then washed in buffer (50 mM sodium phosphate pH 8.0; 300 mM NaCl), re-centrifuged, and stored as cell pellets at &#8722;80&#176;C.</p><p>For purifications, cell pellets were thawed on ice, resuspended in 30 mL of the sodium phosphate wash buffer (except for His-tagged Kil, see below) with 1 mg/mL lysozyme and an EDTA-Free c0mplete Protease Inhibitor Cocktail Tablet (Roche), and incubated on ice for 30 minutes. Cells were disrupted by sonication on ice (Branson Sonifier 250; 50% level, output 4) in a series of six alternating 30-second periods of sonication and 30-second rest periods. Cell lysates were clarified of debris by spinning at 40,000 rpm at 4&#176;C for 45 minutes in an 80 Ti rotor with an Optima XL-100K Ultracentrifuge (Beckman).</p><p>
<italic>E. coli</italic> FtsZ was purified from WM971 cell lysates by successive 20% and 30% ammonium sulfate cuts. Following the second cut, protein was resuspended in polymerization buffer (50 mM MES pH 6.5; 50 mM KCl; 2.5 mM MgCl<sub>2</sub>; 1 mM EGTA; 10% sucrose), flash-frozen in liquid nitrogen and stored at &#8722;80&#176;C.</p><p>His-tagged ZipA<sub>C</sub> proteins were purified by gravity flow over water-washed and buffer-equilibrated cobalt-conjugated resin with a 5 mL bed volume at 4&#176;C. Imidazole at pH 8.0 was added to 10 mM in cell lysates for loading onto the column. Following loading, columns were washed successively in ten-column volumes of purification buffer (50 mM HEPES pH 7.5; 300 mM NaCl) with 10 mM, 25 mM, and 50 mM imidazole. His-tagged ZipA<sub>C</sub>s were eluted in 10 mL purification buffer+250 mM imidazole. Eluted proteins were concentrated and subjected to buffer exchange into polymerization buffer using Amicon Ultra-10K Centrifugal Filter Devices (Millipore), flash-frozen in liquid nitrogen, and stored at &#8722;80&#176;C.</p><p>His<sub>6</sub>-FLAG-Kil was primarily induced in, and purified from, BL21(DE3) <italic>ftsA<sub>R286W</sub> &#916;zipA::aph</italic> pBS58 cells. The exception was for copurification experiments, where the fusion was also prepared from <italic>zipA<sub>WT</sub><sup>+</sup></italic> or <italic>zipA<sub>L286Q</sub></italic>
<sup>+</sup> backgrounds, as noted. Following induction, His<sub>6</sub>-FLAG-Kil was found to be predominantly associated in insoluble inclusion bodies, leading to very low yields under native conditions, even in an <italic>ftsA<sub>R286W</sub> &#916;zipA::aph</italic> background. We therefore purified His<sub>6</sub>-FLAG-Kil under denaturing conditions according to the QIAexpressionist Handbook (QIAGEN, 2003) protocol, followed by renaturation by dialysis. Purity was estimated at &gt;95% by Coomassie staining.</p><p>Lysates from His<sub>6</sub>-FLAG-Kil-expressing cells were prepared similarly as described above, except cell pellets were resuspended in denaturing lysis buffer (100 mM sodium phosphate, pH 8.0, 10 mM Tris-Cl, 6M guanidine hydrochloride) with an EDTA-Free c0mplete Protease Inhibitor Cocktail Tablet (Roche), but without lysozyme. After incubation for 30 minutes at room temperature, cells were sonicated as described above.</p><p>These His<sub>6</sub>-FLAG-Kil-containing lysates were then loaded by gravity flow onto water-washed and denaturation lysis buffer-equilibrated cobalt-conjugated resin with a 5 mL bed volume at room temperature. Columns were washed in ten-column volumes of freshly prepared denaturing wash buffer (100 mM sodium phosphate, pH 6.3, 10 mM Tris-Cl, 8 M urea). Denatured His<sub>6</sub>-FLAG-Kil was eluted in 10 mL freshly prepared denaturing elution buffer (100 mM sodium phosphate, pH 4.5, 10 mM Tris-Cl, 8 M urea). The eluted protein was then renatured by dialysis into polymerization buffer overnight and through the following day (three 2 L changes of buffer in total). Renatured His<sub>6</sub>-FLAG-Kil was concentrated using Amicon Ultra-3K Centrifugal Filter Devices (Millipore), flash-frozen in liquid nitrogen, and stored at &#8722;80&#176;C.</p><p>To assay interaction between purified proteins, 200 mL samples were prepared in polymerization buffer containing 6 &#181;M bovine serum albumin. FtsZ at 5 &#181;M, His<sub>6</sub>-FLAG-Kil at 10 &#181;M, and/or His<sub>6</sub>-tagged ZipA<sub>C</sub> domains (WT and L286Q) at 5 &#181;M were included as indicated. 50 &#181;L of 50% buffer-equilibrated &#945;-FLAG M2 affinity resin (Sigma-Aldrich) were added and samples were incubated at room temperature, mixing, for one to three hours for binding. Samples were then loaded onto gravity flow columns and the resin was washed with 25 mL polymerization buffer. Following washes, resin was recovered in 250 &#181;L buffer, SDS-PAGE sample buffer was added and samples were boiled and separated by 20% SDS-PAGE followed by Coomassie blue staining.</p><p>Sedimentation assays were performed essentially as previously described <xref ref-type="bibr" rid="pgen.1004217-Haeusser1">[66]</xref> in a Beckman TL-100 Ultracentrifuge, but using a TLA 100.3 rotor at 70,000 rpm with appropriate adaptors and speed-resistant tubes (Beckman). 100 &#181;L samples were prepared in polymerization buffer with FtsZ at 5 &#181;M, His<sub>6</sub>-FLAG-Kil at 10 &#181;M, His<sub>6</sub>-tagged ZipA<sub>C</sub> domains (WT and L286Q) at 5 &#181;M, and GTP at 1 mM. Components were added in the following order: polymerization buffer, Kil buffer or Kil, FtsZ, ZipA<sub>C</sub>, and GTP. For reactions with calcium-induced bundling, CaCl<sub>2</sub> was added to 1 mM after GTP addition.</p><p>GTPase activities were determined using the EnzChek Phosphate Assay Kit (Molecular Probes) in reactions set up with the same concentrations and buffers as for sedimentation assays, but in a 96 well plate and with the required purine nucleoside phosphorylase enzyme and 7-Methyl-6-thio-D-guanosine (MESG) substrate components. Reactions were initiated by adding one half the reaction volume as buffer with FtsZ alone, simultaneously via multi-channel pipette, into the other half of the reaction volume that contained all other components. (For reactions without FtsZ, buffer with GTP alone was used to simultaneously initate reactions). OD<sub>360</sub> readings were taken every 30 seconds using a Synergy Mx Microplate Reader (BioTek). GTP hydrolysis rates were calculated based on a phosphate standard curve.</p><p>One-step growth was done according to Frank Stahl (unpublished). Briefly, 10 mL MG1655 was growth at 39&#176;C in tryptone broth to OD<sub>600</sub>&#8202;=&#8202;0.4 (&#8764;1.5&#215;10<sup>8</sup> per mL). The cells were pelleted and suspended in TMG (see above) and incubated at 37&#176;C for 30 minutes to starve the cells. NaCN was added at 2&#215;10<sup>&#8722;3</sup> M and 1.8 mL was dispensed to two small glass-plating tubes. To initiate the growth curve, 0.2 mL phage stock at 1.5&#215;10<sup>8</sup> per mL was added to the cell suspension. Phage adsorption was monitored over a 30 minute period by mixing 0.1 mL infected cells into 4.9 mL tryptone broth containing 0.25 mL chloroform and plating appropriate dilutions on C600 host cells. After a 30 minute adsorption the infected cells were diluted 100-fold into tryptone broth and two further dilutions were made into 39&#176;C tryptone broth, one 100-fold and one 5&#215;10<sup>3</sup>-fold. At various times, 0.1 mL samples were taken from these dilutions and plated immediately on C600.</p>