<p>We obtained genomic DNA samples processed at the Finnish Institute of Molecular Medicine (NFBC) and US National Human Genome Research Institute (FUSION). All NFBC and FUSION participants included in this study provided informed consent. The studies were carried out in accordance with the approvals of the Ethical Committee of the Northern Ostrobothnia Hospital District (for NFBC), and the University of Michigan Health Sciences and Behavioral Sciences Institutional Review Board (IRB-HSBS) and the Institutional Review Board of the National Public Health Institute (KTL; now part of the National Institute for Health and Welfare, THL) (for FUSION).</p><p>We constructed Illumina multiplexed libraries with 5 &#181;g of whole genome amplified material (see <xref ref-type="supplementary-material" rid="pgen.1004147.s012">Text S1</xref> for description of amplification procedures) or 1 &#181;g native genomic DNA according to the manufacturer's protocol (Illumina Inc, San Diego, CA) with the following modifications: 1) DNA was fragmented using a Covaris E220 DNA Sonicator (Covaris, Inc. Woburn, MA) to between 100 and 400 bp. 2) Illumina adapter-ligated library fragments were amplified in four 50 &#181;L PCR reactions for eighteen cycles. 3) Solid Phase Reversible Immobilization bead cleanup was used for enzymatic purification throughout the library construction process and for final library size selection targeting 300&#8211;500 bp fragments. Samples were multiplexed using Illumina barcoded libraries pooled together in pools of 12 or 18 depending on the sequencing platform. We designed a custom targeted set of 150 bp probes (Agilent Technologies, Santa Clara, CA) and captured &#8764;270 kb of primarily coding sequence from 78 genes. The concentration of each captured library pool was determined through qPCR according to the manufacturer's protocol (Kapa Biosystems, Inc, Woburn, MA) to produce cluster counts appropriate for the Illumina GAIIx and HiSeq 2000 platforms.</p><p>Sample pools of 12 and 18 were loaded on GAIIx and HiSeq machines, respectively, using paired end 101 bp read lengths. We aimed to achieve a coverage metric of 80% of the targeted space covered at &#8805;20&#215; depth. We aligned reads from each sample to the NCBI37/hg19 reference sequence using BWA <xref ref-type="bibr" rid="pgen.1004147-Li2">[31]</xref>. Sample identity was confirmed by comparing sequence data (SAMtools consensus calls) with pre-existing genotype array data. Individuals with &#8805;70% coverage at 20&#215; and &#8805;90% genotype concordance with 51 array SNPs were included in the analysis (6,123 of 6,958 individuals).</p><p>Details on sequencing and generation of center-specific genotype call sets can be found in <xref ref-type="supplementary-material" rid="pgen.1004147.s012">Text S1</xref>. To generate a consensus call set, we pooled together all quality controlled sites discovered by any of the three centers (UCLA, University of Michigan, or Washington University) in the defined target loci (number of markers m&#8202;=&#8202;2,306). We excluded multi-allelic sites or sites with different alternative alleles (m&#8202;=&#8202;72). Each center then re-called SNP genotypes at the remaining sites (m&#8202;=&#8202;2,234). Majority vote was used to generate variant calls. Genotypes concordant between at least two centers were included in the consensus data set; others were set to missing. The overall concordance rate between centers was 99.96% (99.99%, 99.94%, and 99.95% for homozygous reference, heterozygous, and homozygous alternative genotypes, respectively).</p><p>NFBC individuals were previously genotyped on the Illumina 370duo Chip, and all FUSION cases and 774 of 919 FUSION controls on the Illumina HumanHap300 BeadChip (version 1.0). After standard quality control procedures <xref ref-type="bibr" rid="pgen.1004147-Scott1">[6]</xref>, <xref ref-type="bibr" rid="pgen.1004147-Sabatti1">[7]</xref>, high-quality GWAS genotypes were available for 296,978 SNPs for all genotyped individuals. We used PLINK <xref ref-type="bibr" rid="pgen.1004147-Purcell1">[32]</xref> to identify 122,644 SNPs with no more than moderate pair wise linkage disequilibrium (r<sup>2</sup>&lt;0.5) which we used to calculate genetic principal components (PCs) with EIGENSTRAT <xref ref-type="bibr" rid="pgen.1004147-Price1">[33]</xref>.</p><p>We applied logarithm transformations to BMI, WHR, TG, glucose, insulin, and SBP to reduce skewness. For each phenotype (or its logarithm), data from the two studies combined were regressed on age, age<sup>2</sup>, and indicator variables for sex, oral contraceptive use, pregnancy status, and cohort, and residuals from this regression used in association analyses. T2D cases were excluded from analysis of FG and FI. Analyses were repeated using inverse normal transformed variables, and our conclusions were robust to choice of transformation.</p><p>We tested variants with minor allele frequency (MAF)&gt;0.1% for association with phenotype residuals in the combined NFBC/FUSION data set assuming an additive genetic model and including the first five PCs as covariates using PLINK <xref ref-type="bibr" rid="pgen.1004147-Purcell1">[32]</xref>. We used conditional analyses to determine if single-SNP associations were independent of genotypes at the array SNP. Teslovich <italic>et al.</italic>, in a previous GWAS meta-analysis of lipid traits, identified, at several loci, associations independent of their GWAS signals <xref ref-type="bibr" rid="pgen.1004147-Teslovich1">[8]</xref>. We sought to evaluate, at all such GWAS loci highlighted by Teslovich <italic>et al.</italic>, the correlation between the signals identified through our conditional analyses and the &#8220;independent&#8221; signals detected by Teslovich <italic>et al.</italic> We could not make a direct comparison because at most loci the two studies did not have data on the same sets of variants. We instead used imputed data from Finnish reference populations to evaluate pairwise LD between the variants from our study that we compared with the variants from Teslovich <italic>et al.</italic> In this comparison we considered any pair of variants with r<sup>2</sup>&lt;0.20 to be independent.</p><p>We conducted gene-level tests for phenotype residuals for each gene using non-synonymous variants with MAF&lt;1%. Only individuals with complete genotype data for all variant sites were used in a given gene-level test of each gene; sample size for gene-level tests ranged from 4,651 to 5,376 individuals. There were 62 genes with &gt;1 non-synonymous variant site with MAF&lt;1%; the number of such sites ranged from 2 to 33 per gene.</p><p>We used the Combined Multivariate and Collapsing (CMC, <xref ref-type="bibr" rid="pgen.1004147-Li1">[22]</xref>) test that uses a weighted-sum-score-based linear model to test the collective effect of multiple rare variants within a gene; see <xref ref-type="supplementary-material" rid="pgen.1004147.s012">Text S1</xref> for more information on the form of the weighting method used. We also employed the Sequence Kernel Association Test (SKAT, <xref ref-type="bibr" rid="pgen.1004147-Wu1">[23]</xref>), which assumes that the effect sizes for individual variants follow an arbitrary distribution with zero mean and an unknown variance. SKAT uses a score-based variance component approach to test the null hypothesis that the effect size distribution has zero variance. For both CMC and SKAT, we used the first five PCs as covariates.</p><p>For SKAT, we used asymptotic theory p-values, which conformed well to p-values estimated by permutation (data not shown). For the CMC, we estimated p-values based on 10,000 permutations of the phenotype data. To estimate the p-value for the <italic>ABCA1</italic> association, we performed 1,000,000 permutations.</p><p>We employed FDR controlling procedures <xref ref-type="bibr" rid="pgen.1004147-Benjamini1">[24]</xref> over the entire set of single-variant and gene-level tests we conducted. Testing of each phenotype against variants/genes in loci to which the phenotype had prior association resulted in 2,096 tests (1,802 single-variant tests and 294 gene-level tests). FDR control at the 0.05 level resulted in a p-value cut-off of 0.004, and we opted to use a somewhat more stringent p-value threshold of 0.001 corresponding to a FDR of 0.02.</p><p>We used a standard model selection approach to analyze the 39 locus-phenotype combinations and derive the &#8220;best&#8221; multivariate linear model for each phenotype based on all variants at the locus. We excluded singletons from this analysis because the absence of any replication in the study sample renders inference on their effect impossible without making strong parametric assumptions. As a default model selection approach, we used the Bayesian Information Criterion (BIC) using a greedy search built into R (StepAIC). We again included only complete observations in each locus-phenotype combination. For each locus, the model with the smallest number of predictor variables included the first five PCs, and the model with the largest number of predictor variables included all non-singleton variants genotyped.</p>