<p>All studies were performed using protocols approved by the JHU and NIH Animal Care and Use Committee, and mice were kept and cared in pathogen-free animal facilities accredited by the American Association for the Accreditation of Laboratory Animal Care and meet federal (NIH) guidelines for the humane and appropriate care of laboratory animal.</p><p>Fifth-generation C57/BL6 <italic>Pkd1<sup>cko</sup></italic> mice were crossed to fifth-generation C57/BL6 tamoxifen-Cre (B6.Cg-Tg(Cre/Esr1)5Amc/J mice (stock 004682), Jackson Laboratories) and C57/BL6 congenic B6.129S4-<italic>Gt(ROSA)26Sor<sup>tm1Sor</sup></italic>/J (stock 003474, Jackson Laboratories) to produce the mice used in these studies. The <italic>Pkd1<sup>cko</sup></italic> thus obtained were bred to <italic>Hnf4&#945;<sup>cko</sup></italic> mice (B6.129X1(FVB)-<italic>Hnf4&#945;<sup>tm1.1Gonz</sup></italic>/J (stock 004665, Jackson Laboratories). Cre recombinase activity was induced by intraperitoneal injection of tamoxifen (10 mg/40 g) in corn oil (Sigma-Aldrich) in nursing moms of mice&lt;10 days of age. Mice were euthanized by isofluorane treatment followed by cervical dislocation.</p><p>Specimens were fixed in 4% paraformaldehyde buffered solution, pH 7.4 at 4&#176;C overnight, washed in PBS for 2 h, dehydrated in 50% ethanol for 2 h and stored in 70% ethanol at room temperature. The kidneys were cut in half along the longitudinal axis, embedded in paraffin and sectioned in 5-&#181;m slices for hematoxylin and eosin staining. To calculate the cystic index, a composite image of a longitudinal midline kidney section was assembled using Axiovision (Zeiss), and the total kidney and cystic areas were measured using ImageJ <xref ref-type="bibr" rid="pgen.1003053-Rasband1">[51]</xref>. Statistical analysis was performed using Student's t-test.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Total RNA was isolated from kidneys using Trizol reagent (Invitrogen). For mRNA studies, a subsequent step of RNA isolation using RNeasy plus kit was performed (Qiagen). Illumina's MouseRef-8 v1.0 or v2.0 BeadChip expression arrays with sample processing performed by the Johns Hopkins Bayview Medical Campus Genomics Core. For microRNA expession profiling, Agilent miRNA arrays were used with the Johns Hopkins Medical Institute microarray core (<ext-link ext-link-type="uri" ns0:href="http://www.microarray.jhmi.edu">http://www.microarray.jhmi.edu</ext-link>).</p><p>microRNA real time PCR amplification was performed using TaqMan MicroRNA reverse transcription kit (4366596, Applied Biosystems), and the TaqMan microRNA assays has-miR-21, mmu-miR-31, has-miR-15a, normalized to microRNA snoRNA 202 control (respectively, RT 397, RT 185, RT 389, RT 1232; Applied Biosystems).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Gene expression data were normalized using software provided by the Bioconductor project <xref ref-type="bibr" rid="pgen.1003053-Gentleman1">[52]</xref>. Briefly, we imported gene expression data using the lumi package, followed by preprocessing using variance stabilization transformation and quantile normalization <xref ref-type="bibr" rid="pgen.1003053-Du1">[53]</xref>, <xref ref-type="bibr" rid="pgen.1003053-Lin1">[54]</xref>. The data were filtered to include only probes with detection call p value&lt;0.05 in at least one of the arrays and COMBAT software employed to remove batch effects <xref ref-type="bibr" rid="pgen.1003053-Johnson1">[55]</xref>. When combining datasets of the two different Illumina array versions (v1.0 and v2.0), only probes present in both versions were included. For the microRNA data, quantile-normalization was applied. For both mRNA and microRNA normalized data, Partek was used to obtain principal component analysis and heatmap plots, and to identify differentially expressed probes/genes using ANOVA-based methods and fdr multiple comparison correction. For meta-analysis of published gene expression data, CEL files of experiments performed using Affymetrix Mouse Genome 430 2.0 arrays (platform GPL1261) were downloaded from GEO (<ext-link ext-link-type="uri" ns0:href="http://www.wip.ncbi.nlm.nih.gov/geo/">http://www.wip.ncbi.nlm.nih.gov/geo/</ext-link>), imported into Partek using RMA, followed by quantile normalization. All microarray data have been uploaded to the GEO database (Accession GSE32586).</p><p>For weighted gene co-expression network analysis (WGCNA) <xref ref-type="bibr" rid="pgen.1003053-Zhang1">[56]</xref>, the WGCNA R package was used <xref ref-type="bibr" rid="pgen.1003053-Langfelder1">[11]</xref>. Briefly, soft thresholding powers were chosen based on the approximate scale-free topology criterion <xref ref-type="bibr" rid="pgen.1003053-Zhang1">[56]</xref>, and applied blockwise network construction and module detection to the normalized expression matrix. We then correlated the modules with age and/or mutant status.</p><p>For comparative network analysis, multiSetMEs <xref ref-type="bibr" rid="pgen.1003053-Langfelder1">[11]</xref> was used to identify consensus modules for the 4 experimental groups: P12-mutant, P12-control, P14-mutant and P14-control, and to determine module preservation and quantify eigengene correlations <xref ref-type="bibr" rid="pgen.1003053-Langfelder2">[17]</xref>.</p><p>Genomatix Software Suite (Genomatix Inc.) was used to identify gene ontology, and pathway enrichment. To build knowledge-based networks, Ingenuity Pathway Analysis was used (IPA; Ingenuity Systems). Briefly, matrices were generated with fold-change and q-value for each of the comparisons of interest (i.e. age and mutant status) and considered genes with more than 1.2 fold change and q&lt;0.05 as network eligible molecules. IPA was then used to screen the Ingenuity Knowledge Base (January 2011 release) for reported interactions involving these genes, to identify networks that maximize connectivity and to score them based on the number of network eligible molecules they contain. The network score corresponds roughly to &#8211;log10 (p value).</p><p>Urine samples were collected at the time of euthanasia, diluted with 20 volumes of water and spun at 14,000&#215; g for 10 min at 4C. The supernatant was transferred to an autosampler tube and analyzed by ultraperformance liquid chromatography coupled with electrospray ionization quadrupole time-of-flight mass spectrometry. MarkerLynx software was used to align and deconvolute the mass chromatographic data and to generate a data matrix consisting of peak areas corresponding to a unique m/z and retention time. Statistical analysis was carried out using Partek, employing unpaired t-test comparing cre positive and negative animals, and partial least square (PLS) analysis. The relevant m/z to KEGG ids <xref ref-type="bibr" rid="pgen.1003053-Kanehisa1">[57]</xref> were identified by searching the Scripps Center for Metabolomics Database with 5 ppm accuracy and possible modifications (+H, +NH4, +Na, +H-2H2O, +H-H2O in the positive mode; &#8722;H, &#8722;H2O-H, +Na-2H, +Cl, +K-2H in the negative mode) <xref ref-type="bibr" rid="pgen.1003053-Smith1">[58]</xref>. Verification of acetylcarnitine (m/z 204.123) was obtained by comparing fragmentation patterns of urine samples with authentic standard by tandem mass spectrometry. For generation of MS/MS fragments, the mass spectrometer collision energy was ramped from 6 to 40 V. For pathway enrichment analysis, Ingenuity Pathway Analysis was used.</p>