<p>
<italic>C. heterostrophus</italic> strains sequenced by JGI included inbred strains C5 (ATCC 48332, race O, <italic>MAT1-1</italic>, <italic>Tox1&#8722;</italic>) and C4 (ATCC 48331, race T, <italic>MAT1-2</italic>, <italic>Tox1+</italic>) and field strains Hm540 (geographical origin North Carolina, race O, <italic>MAT1-1</italic>, <italic>Tox1&#8722;</italic>), Hm338 (New York, race T, <italic>MAT1-2</italic>, <italic>Tox1+</italic>, ATCC 48317), and PR1x412, (a progeny of a cross between PR1C from Poza Rica, Mexico and strain 412, unknown geographical origin, race T, <italic>MAT1-1</italic>, <italic>Tox1+</italic>). In addition, the genomes of <italic>C. victoriae</italic> strain FI3 (unknown geographical origin, <italic>MAT1-2</italic>, victorin<italic>+</italic>), <italic>C. carbonum</italic> strain 26-R-13 [<italic>MAT1-1</italic>, HC-toxin<italic>+</italic>, a progeny of a cross between <italic>C. carbonum</italic> strains 2-R-6 (<italic>alb2</italic>; <italic>MAT1-1</italic>) and Five Points (unknown geographical origin, <italic>MAT1-2</italic>) performed by Dr. Steve Briggs]. <italic>C. miyabeanus</italic> strain WK1C (Wuankuei, Yulan county China, <italic>MAT1-2</italic>), <italic>C. sativus</italic> isolate ND90Pr (North Dakota, ATCC 201652, <italic>MAT1-2</italic>, pathotype 2 on barley cv. Bowman) and <italic>S. turcica</italic> strain St28A (New York, race 2,3,N, <italic>MAT1-1</italic>) were sequenced by JGI. <italic>C. sativus</italic> isolate ND93-1 (North Dakota, ATCC 201653, <italic>MAT1-1</italic>, pathotype 0 on barley cv. Bowman) was sequenced at the University of Hawaii.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The highly inbred <italic>C. heterostrophus</italic> reference race O lab strain C5 was sequenced using the Sanger whole genome shotgun approach, with paired end reads and improved by manual finishing and fosmid clone sequencing (<ext-link ext-link-type="uri" ns0:href="http://www.jgi.doe.gov/sequencing/protocols/prots_production.html">http://www.jgi.doe.gov/sequencing/protocols/prots_production.html</ext-link>). Four different sized libraries were sequenced: 3.1 kb, 6.8 kb, and two fosmid libraries (32.3 kb and 35.3 kb), to a total coverage of 9.95&#215;. ESTs were generated by growing strains in complete and minimal medium under many conditions [on complete and minimal medium, on sexual reproduction plates, stress medium (-N, -Fe, etc)] and pooled as complete or minimal samples for sequencing and support of gene annotation. The genome of isogenic <italic>C. heterostrophus</italic> race T strain C4 was sequenced using Illumina technology (300 bp insert size, 2&#215;76 bp reads to a nominal depth of 200&#215;), assembled using Velvet <xref ref-type="bibr" rid="pgen.1003233-Zerbino1">[97]</xref> and AllPathsLG <xref ref-type="bibr" rid="pgen.1003233-Gnerre1">[98]</xref> and annotated using ESTs from <italic>C. heterostrophus</italic> strain C5. The genome of <italic>C. sativus</italic> pathotype 2 isolate ND90Pr was sequenced using a hybrid approach, which included 40 kb fosmid Sanger reads, shredded consensus from Velvet assembled Illumina data (300 bp insert size, 2&#215;76 bp reads), Roche (454) standard and Roche (454) 4 kb insert paired ends, all assembled using Newbler <xref ref-type="bibr" rid="pgen.1003233-Margulies1">[99]</xref> and annotated using <italic>C. sativus</italic> ND90Pr ESTs as described below. The genome of the second <italic>C. sativus</italic> isolate, ND93-1, was sequenced at the University of Hawaii by paired end 454 runs and assembled using Newbler. <italic>S. turcica</italic> strain 28A was sequenced using Roche (454), Sanger fosmids, and shredded consensus from Velvet assembled Illumina data; EST libraries were prepared from <italic>S. turcica</italic> strains using conditions described above for <italic>C. heterostrophus</italic>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The JGI annotation pipeline was used to annotate <italic>C. heterostrophus</italic> strains C5 and C4, <italic>C. sativus</italic> ND90Pr, and <italic>S. turcica</italic>. For this, the assembled genomic scaffolds were masked using RepeatMasker <xref ref-type="bibr" rid="pgen.1003233-Smit1">[100]</xref>with the RepBase fungal library of 234 fungal repeats <xref ref-type="bibr" rid="pgen.1003233-Jurka1">[101]</xref> and genome-specific libraries derived using <xref ref-type="bibr" rid="pgen.1003233-Price1">[102]</xref>. Multiple sets of gene models were predicted for each assembly, and automated filtering based on homology and EST support was applied to produce a final non-redundant GeneCatalog representing the best gene model found at each genomic locus. The gene-prediction methods were: <italic>EST-based predictions</italic> with EST map (<ext-link ext-link-type="uri" ns0:href="http://softberry.com">http://softberry.com</ext-link>) using raw ESTs and assembled EST contigs for each genome; <italic>homology-based predictions</italic> with Fgenesh+ <xref ref-type="bibr" rid="pgen.1003233-Salamov1">[103]</xref> and Genewise <xref ref-type="bibr" rid="pgen.1003233-Birney1">[104]</xref>, with homology seeded by BLASTx alignments of the GenBank non-redundant sequence database (NR: <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</ext-link>) to the genomic scaffolds; and <italic>ab initio predictions</italic> using Fgenesh <xref ref-type="bibr" rid="pgen.1003233-Salamov1">[103]</xref>) and GeneMark <xref ref-type="bibr" rid="pgen.1003233-Isono1">[105]</xref>. Genewise models were extended to include 5&#8242; start and/or 3&#8242; stop codons when possible. Additional EST-extended sets were generated using BLAT-aligned <xref ref-type="bibr" rid="pgen.1003233-Kent1">[106]</xref> EST data to add 5&#8242; UTRs, 3&#8242;UTRs, and CDS regions that were supported by ESTs but had been omitted by the initial prediction methods.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All genome annotations can be interactively accessed through MycoCosm <xref ref-type="bibr" rid="pgen.1003233-Grigoriev1">[107]</xref>, <ext-link ext-link-type="uri" ns0:href="http://jgi.doe.gov/fungi">http://jgi.doe.gov/fungi</ext-link>.</p><p>Because the subject genomes are all closely related to the fully sequenced reference <italic>C. heterostrophus</italic> strain C5, additional <italic>C. heterostrophus</italic> field strains Hm540, Hm338 and PR1x412, <italic>C. victoriae</italic>, <italic>C. carbonum</italic>, and <italic>C. miyabeanus</italic>, were (re)-sequenced using using Illumina technology. DNA was randomly sheared into &#8764;200 bp fragments using Covaris E210 according to the manufacturer's recommendation and the resulting fragments were used to create an Illumina library. 2&#215;76 bp reads were assembled using Velvet (version 1.1.04), <xref ref-type="bibr" rid="pgen.1003233-Zerbino1">[97]</xref>. There are no ESTs available for these organisms. Assembled contigs were mapped to the reference <italic>C. heterostrophus</italic> C5 for analysis of genome variation and rearrangements. Assembled reads are called &#8216;nodes&#8217; (scaffolds). Overall sequence assembly and annotation statistics are presented in <xref ref-type="table" rid="pgen-1003233-t003">Table 3</xref>.</p><p>The genomes without JGI annotation pipeline gene predictions (<italic>Ch</italic>Hm540, <italic>Ch</italic>Hm338, <italic>Ch</italic>PR1x412, <italic>C. carbonum</italic>, <italic>C. miyabeanus</italic>, <italic>C. victoriae</italic>) were annotated using Augustus <xref ref-type="bibr" rid="pgen.1003233-Stanke1">[108]</xref>.</p><p>Assembled genomes were mapped individually to the C5 reference strain using the nucmer program of MUMmer v 3.22 <xref ref-type="bibr" rid="pgen.1003233-Kurtz1">[46]</xref>, a program that finds unique, exact matches to build whole genome alignments. SNPs were called and analyzed using the dnadiff wrapper on the filtered MUMmer delta files. Unassembled reads were also aligned to the <italic>C. heterostrophus</italic> C5 reference genome for low coverage analysis using maq-0.7.1. Regions of the reference genome under a depth of three aligned reads were considered &#8220;low coverage&#8221; for our analyses. Adjacent low coverage regions were merged if they were separated by less than 100 bp in order to minimize noise from mis-mapping of occasional low quality reads. After low coverage regions were identified for pairwise comparisons to C5, regions were identified that were shared as low coverage in multiple genomes: i.e., <italic>C. carbonum</italic>, <italic>C. victoriae</italic>, <italic>C. miyabeanus</italic>, and <italic>C. sativus</italic> for <italic>C. heterostrophus</italic> specific regions; <italic>C. heterostrophus</italic> Hm540, Hm338, and PR1x412 for C strain specific regions; and C4, Hm338, and PR1x412 for race O specific regions.</p><p>Cross genome comparisons were visualized using Mauve <xref ref-type="bibr" rid="pgen.1003233-Darling1">[44]</xref>, a multiple genome alignment tool that visualizes localized collinear blocks (LCB) between genomes.</p><p>Cloned RFLP markers <xref ref-type="bibr" rid="pgen.1003233-Tzeng1">[8]</xref> were sequenced and sequences used in blast queries against the <italic>C. heterostrophus</italic> C5 assembly. Top hits were filtered using Bioperl and manually confirmed to span the entire RFLP with very high stringency to rule out markers that might exist as more than one copy. Physical and genetic distances of adjacent RFLPs mapping to the same scaffold were plotted and used to calculate an average ratio of physical to genetic distance (<xref ref-type="supplementary-material" rid="pgen.1003233.s001">Figure S1</xref>, <xref ref-type="supplementary-material" rid="pgen.1003233.s010">Table S2</xref>). Relative RFLP location was used to orient scaffolds along the linkage group when possible.</p><p>A genetic linkage map was generated previously using the mapping population derived from a cross between parental isolates ND93-1 and ND90Pr <xref ref-type="bibr" rid="pgen.1003233-Zhong1">[30]</xref> and amplified fragment length polymorphism (AFLP) and RFLP markers. To add simple sequence repeat (SSR) markers to the map, the draft sequence assembly of the <italic>C. sativus</italic> isolate ND93-1 was screened for SSR loci with di- and tri-nucleotide units tandemly repeated six or more times using a Perl script (provided by Zheng Jin Tu at the University of Minnesota, St. Paul). The SSR-containing sequences from ND93-1 were aligned to the draft genome sequence of isolate (ND90Pr) of <italic>C. sativus</italic>. Only those sequences that were polymorphic between the two <italic>C. sativus</italic> parents (ND93-1 and ND90Pr) were used for primer design and tested for segregation in the mapping population used previously <xref ref-type="bibr" rid="pgen.1003233-Zhong1">[30]</xref>. PCR conditions and detection of SSR markers were as previously described <xref ref-type="bibr" rid="pgen.1003233-Zhong2">[109]</xref>. Map construction was performed by using MAPMAKER version 2.0 <xref ref-type="bibr" rid="pgen.1003233-Lander1">[110]</xref>. A minimum LOD value of 4.0 and a maximum theta of 0.3 were used to group all SSR markers with previously mapped AFLP, RFLP and PCR markers <xref ref-type="bibr" rid="pgen.1003233-Zhong1">[30]</xref>. The Kosambi mapping function was used to calculate the map distance.</p><p>To associate linkage groups to the sequenced scaffolds of <italic>C. sativus</italic>, the sequences of mapped SSR markers were used as queries to blast against the draft genome assembly of <italic>C. sativus</italic> isolate ND90Pr and the coordinates of each SSR marker were recorded for the associated scaffold.</p><p>NRPS and PKS proteins were identified using our custom fungal AMP domain model <xref ref-type="bibr" rid="pgen.1003233-Bushley1">[57]</xref> for the former and an HMM model build from <italic>C. heterostrophus</italic> KS domains plus sequences from the C-terminal and N-terminal ketosynthase (KS) Pfam domains for the latter (PF00109 and PF02801). Proteins were identified in two ways. In the first case, genome nucleotide sequences were searched using Genewise <xref ref-type="bibr" rid="pgen.1003233-Birney2">[111]</xref> and sequences extracted and concatenated by a Perl script utilizing Bioperl's searchIO system <xref ref-type="bibr" rid="pgen.1003233-Stajich1">[112]</xref>. In the second case, Augustus protein models (see above) and JGI protein models (<italic>C. heterostrophus</italic>, <italic>C. sativus</italic>, <italic>S. turcica</italic>) were searched with the PKS KS and NRPS AMP HMM using HMMER 3.0 <xref ref-type="bibr" rid="pgen.1003233-Finn1">[113]</xref>, and the sequences extracted and concatenated using a Perl script with Bioperl's searchIO.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">AMP and KS domains were aligned, separately, with MAFFT (<ext-link ext-link-type="uri" ns0:href="http://mafft.cbrc.jp/alignment/software/">http://mafft.cbrc.jp/alignment/software/</ext-link>) and manually inspected to remove columns of poor alignment. ProtTest <xref ref-type="bibr" rid="pgen.1003233-Abascal1">[114]</xref> was run on both alignments and identified the RTREVF model as the best fit for the AMP domains and the WAGF model as the best fit for the KS domain alignments, respectively. RAxML <xref ref-type="bibr" rid="pgen.1003233-Stamatakis1">[115]</xref> using the RTREVF and WAGF models with a gamma distribution was used to infer maximum likelihood trees and bootstrap support was determined using the fast-bootstrap method with 1000 bootstrap replicates <xref ref-type="bibr" rid="pgen.1003233-Stamatakis2">[116]</xref>. The CIPRES portal (<ext-link ext-link-type="uri" ns0:href="http://www.phylo.org/sub_sections/portal/">http://www.phylo.org/sub_sections/portal/</ext-link>) was used for inference of phylogenetic trees.</p><p>
<italic>C. sativus</italic> ND90Pr NRPS AMP domains were used as blast queries to identify AMP domain orthologs in ND93-1 with the methods above. An ND93-1 AMP was considered orthologous if it was at least 95% identical to the ND90Pr query.</p><p>Candidate small secreted proteins (SSP) were identified by screening the gene catalogue of each genome. Proteins smaller than 200 amino acids and containing more than 2% cysteines were searched for transmembrane domains and secretion tags using Phobius <xref ref-type="bibr" rid="pgen.1003233-Kall1">[71]</xref>. Those without transmembrane domains were retained. EST support and domain prediction for <italic>C. heterostrophus</italic> C5 SSPs was performed using the JGI portal. Cross-genome comparisons were made based on all vs. all reciprocal best hit analysis with an 80% similarity cutoff.</p><p>Three EST libraries were constructed. A mycelia-only library was constructed by harvesting mycelia grown on different media including Potato Dextrose Agar (PDA), minimal medium (MM) <xref ref-type="bibr" rid="pgen.1003233-Tinline1">[117]</xref>, V8PDA (150 ml V8 juice, 850 ml H<sub>2</sub>O, 10 g PDA, 10 g Agar and 3 g CaCO<sub>3</sub>), and water agar (15 g agar, 1000 ml water). Mycelia were harvested after 12, 24, 48, 76 and 96 hours of growth, and time points from different media were mixed together for RNA extraction. Equal amounts of extracted RNA from each of the 5 time points was bulked to construct the mycelia only library To construct the <italic>in planta</italic> cDNA libraries, two week old barley cv. Bowman and 4 week old <italic>Brachypodium distachyon</italic> line Bd21were inoculated with conidia of ND90Pr at a concentration of 5&#215;10<sup>3</sup>/ml <xref ref-type="bibr" rid="pgen.1003233-Fetch1">[118]</xref>. Inoculated plants were incubated in a humid chamber for 24 hours and moved to the greenhouse. Leaves were harvested at 6, 12, 24, 48, 72, and 96 hours after inoculation and total RNA extracted from each sample. The final <italic>in planta</italic> cDNA libraries were constructed by mixing the equal amounts of total RNA from different time points. Total RNA was isolated from all samples using the PureLink RNA Mini Kit (Invitrogen, Carlsbad, CA) and purified by treatment with DNase I (Invitrogen, Carlsbad, CA). These three libraries were sequenced by JGI.</p><p>For <italic>C. sativus</italic>, total RNA extracted as described above at six time points (6, 12, 24, 48, 72, and 96 hours) after inoculation was used for RT-PCR. The reverse transcription reaction was performed on 2 &#181;g of total RNA using the SuperScript III First-Strand Synthesis System (Invitrogen, Carlsbad, CA). cDNA was diluted 20 times and used as the template for quantitative RT-PCR, which was performed with the AB7500 real time PCR system (Applied Biosystems, Foster, CA) (<xref ref-type="supplementary-material" rid="pgen.1003233.s017">Table S9</xref>). For each cDNA sample, three replications were performed. Each reaction mixture (20 &#181;l) contained 5 &#181;l of the cDNA template, 10 &#181;l of SYBR Green PCR Master Mix (Applied Biosystems, Foster, CA) and 0.3 &#181;l of each primer (10 mM). All samples were normalized using RT-Actin-F and RT-Actin-R primers as a control, and values were expressed as the change in the increase/decrease of the relative levels of the control sample (M96, which is the mixture of mycelia harvested from different media including PDA, MM, V8PDA, and water agar).</p><p>For <italic>S. turcica</italic>, leaf samples with lesions were collected at five time points (3, 5, 6, 7 and 8 days) after inoculation with 9&#215;10<sup>4</sup> spores per plant (three weeks old) and total RNA was extracted and qPCR done as described <xref ref-type="bibr" rid="pgen.1003233-Wu1">[119]</xref>. The actin gene was used as internal control using <italic>ATC1</italic> primers <xref ref-type="bibr" rid="pgen.1003233-Wu1">[119]</xref>. The <italic>S. turcica</italic> gene primers corresponding to protein ID 161586 are listed in <xref ref-type="supplementary-material" rid="pgen.1003233.s017">Table S9</xref>. Expression level was expressed as fold change versus mycelial samples harvested on Lactose Casein Agar (LCA) plates.</p><p>Fungal transformation and molecular characterization of gene knockout mutants were conducted according to the methods of <xref ref-type="bibr" rid="pgen.1003233-Leng2">[120]</xref>. The split marker system <xref ref-type="bibr" rid="pgen.1003233-Catlett1">[121]</xref> was used for gene deletion. The 5&#8242; and 3&#8242; flanking sequences of the <italic>NPS</italic> gene encoding protein ID 115356 were amplified from ND90Pr DNA using primer pairs <named-content content-type="gene">GTCGACTGCCATCTGGAAAC/CACTGGCCGTCGTTTTACAACGTCCACTCGACAGGTCCGTAGGT</named-content> and <named-content content-type="gene">TCATGGTCATAGCTGTTTCCTGTGGTATCCACAAAGCCACAGCA/GACGAACCAGA GATGCATGA</named-content>) respectively.</p><p>To verify deletion of the gene corresponding to protein ID 115356, primers CAN1-F3: <named-content content-type="gene">AGTTGTTGGGGAGTTGTTGG</named-content> and CAN1-F4: <named-content content-type="gene">TGAGCCGTTGTCATGTATCG</named-content> matching the deleted portion of the gene were used. The expected PCR product was obtained from WT DNA, but not when DNA of the deletion mutant was used as template. To further confirm that the hygromycin resistance gene replaced the target gene at the native locus, PCR was conducted using a primer located outside the 3&#8242; flank used for gene deletion (CsNPS1-F0: <named-content content-type="gene">GTCCTACGGCAATTGTGGAC</named-content>) and a second primer (HY: <named-content content-type="gene">GGATGCCTCCGCTCGAAGTA</named-content>) located in the hygromycin resistance gene. No amplification occurred when WT DNA was used, while the expected amplicon was observed when DNA of the mutants was used as template.</p><p>For <italic>C. sativus</italic>, virulence of the mutant (ID# 115356, <xref ref-type="fig" rid="pgen-1003233-g007">Figure 7C</xref>) and wild type strains was tested on barley cv. Bowman by spray inoculation as described in Fetch and Steffenson (1999), except 2&#215;10<sup>3</sup> conidia/ml were used. Inoculated plants were incubated in a humid chamber for 18&#8211;24 hours, and then transferred to a greenhouse room (20+/&#8722;2&#176;C).</p><p>For <italic>S. turcica</italic>, three week old W64A maize plants were sprayed with 9&#215;10<sup>4</sup> spores per plant, and plants grown under conditions described previously for <italic>C. heterostrophus</italic>
<xref ref-type="bibr" rid="pgen.1003233-Wu1">[119]</xref>.</p><p>
<italic>MAT1-1</italic> and <italic>MAT1-2</italic> mating type regions were identified by blasting the corresponding known <italic>C. heterostrophus MAT</italic> sequences (<italic>MAT1-1</italic>: accession CAA48465, <italic>MAT1-2</italic>: accession CAA48464) against each genome. Regions immediately 10 kb upstream and downstream were aligned pairwise to <italic>C. heterostrophus</italic> C5 (<italic>MAT1-1</italic>) or C4 (<italic>MAT1-2</italic>) <italic>MAT</italic> regions using ProgressiveMauve <xref ref-type="bibr" rid="pgen.1003233-Darling1">[44]</xref> to generate SNP data, and as a group (with and without <italic>S. turcica</italic> for <italic>MAT1-1</italic>) for visualizing the alignment.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genome assemblies and annotations are available via JGI Genome Portal MycoCosm (<ext-link ext-link-type="uri" ns0:href="http://jgi.doe.gov/fungi">http://jgi.doe.gov/fungi</ext-link>, <xref ref-type="bibr" rid="pgen.1003233-Grigoriev1">[107]</xref> and DDBJ/EMBL/GenBank under the following accessions <italic>Cochliobolus heterostrophus</italic> ATCC 48331 (race T, strain C4): AIHU00000000, <italic>Cochliobolus heterostrophus</italic> ATCC 48332 (race O, strain C5): AIDY00000000, <italic>Cochliobolus sativus</italic> ND90Pr: AEIN00000000, <italic>Setosphaeria turcica</italic> Et28A: AIHT00000000, <italic>C. sativus</italic> ND93-1:PRJNA87041, <italic>Cochliobolus carbonum</italic> 26-R-13: AMCN00000000, <italic>Cochliobolus miyabeanus</italic> ATCC 44560: AMCO00000000, <italic>Cochliobolus victoriae</italic> FI3: AMCY00000000.</p>