<p>Each of the host institutions (<xref ref-type="supplementary-material" rid="pgen.1003173.s007">Table S1</xref>) recruited under ethically-approved protocols. Written informed consent was obtained from all subjects.</p><p>The majority of <italic>BRCA2</italic> mutation carriers were recruited through cancer genetics clinics and some came from population or community-based studies. Studies contributing DNA samples to these research efforts were members of the Consortium of Investigators of Modifiers of <italic>BRCA1/2</italic> (CIMBA) with the exception of one study (NICCC). Eligible subjects were women of European descent who carried a pathogenic <italic>BRCA2</italic> mutation, had complete phenotype information, and were at least 18 years of age. Harmonized phenotypic data included year of birth, age at cancer diagnosis, age at bilateral prophylactic mastectomy and oophorectomy, age at interview or last follow-up, <italic>BRCA2</italic> mutation description, self-reported ethnicity, and breast cancer estrogen receptor status.</p><p>Details of these samples have been described previously <xref ref-type="bibr" rid="pgen.1003173-Gaudet1">[2]</xref>. Data from 899 young (&lt;40 years) affected and 804 older (&gt;40 years) unaffected carriers of European ancestry from 14 countries were used to select SNPs for inclusion on the iCOGS array.</p><p>Forty-seven studies from 24 different countries (including two East-Asian countries) provided DNA from a total of 10,048 <italic>BRCA2</italic> mutations carriers. All eligible samples were genotyped using COGs, including those from the discovery stage.</p><p>The Collaborative Oncological Gene-Environment Study (COGS) consortium developed a custom genotyping array (referred to as the iCOGS array) to provide efficient genotyping of common and rare genetic variants to identify novel loci that are associated with risk of breast, ovarian, and prostate cancers as well as to fine-map known cancer susceptibility loci. SNPs were selected for inclusion on iCOGS separately by each participating consortium: Breast Cancer Association Consortium (BCAC) <xref ref-type="bibr" rid="pgen.1003173-1">[6]</xref>, Ovarian Cancer Association Consortium (OCAC) <xref ref-type="bibr" rid="pgen.1003173-Gayther1">[7]</xref>, Prostate Cancer Association Group to Investigate Cancer Associated Alterations in the Genome (PRACTICAL) <xref ref-type="bibr" rid="pgen.1003173-KoteJarai1">[8]</xref>, and CIMBA. SNP lists from a <italic>BRCA1</italic> GWAS and SNPs in candidate regions were used together with the <italic>BRCA2</italic> GWAS lists to generate a ranked CIMBA SNP list that included SNPs with the following nominal proportions: 55.5% from the <italic>BRCA1</italic> GWAS, 41.6% from the <italic>BRCA2</italic> GWAS and fine mapping, 2.9% for CIMBA candidate SNPs. Each consortium was given a share of the array: nominally 25% of the SNPs each for BCAC, PRACTICAL and OCAC; 17.5% for CIMBA; and 7.5% for SNPs from commonly researched pathways (e.g., inflammation). For the CIMBA <italic>BRCA2</italic> GWAS, we used the iCOGS array as the platform to genotype the extended replication set of the discovery GWAS stage <xref ref-type="bibr" rid="pgen.1003173-Gaudet1">[2]</xref>. SNPs were selected on the basis of the strength of their associations with breast cancer risk in the discovery stage <xref ref-type="bibr" rid="pgen.1003173-Gaudet1">[2]</xref>, using imputed genotype data for 1.4 M SNPs identified through CEU+TSI samples on HapMap3, release 2. A ranked list of SNPs was based on the 1-df trend test statistic, after excluding highly correlated SNPs (r<sup>2</sup>&gt;0.4). The final list included the 39,015 SNPs with the smallest p-values. An additional set of SNPs were selected for fine mapping of the regions surrounding the SNPs found to be associated with breast cancer in the discovery GWAS stage: rs16917302 on 10q21 and rs311499 on 20q13, including SNPs with a MAF &gt;0.05 located 500 kb in both directions of the SNP, based on HapMap 2 data. The final combined list of SNPs for the iCOGS array comprised 220,123 SNPs. Of these, 211,155 were successfully manufactured onto the array. The present analyses are based on the 19,029 SNPs selected on the basis of <italic>BRCA2</italic> GWAS and fine mapping that were included on the iCOGS array.</p><p>The genotyping was performed on DNA samples from 10,048 <italic>BRCA2</italic> mutation carriers at the McGill University and G&#233;nome Qu&#233;bec Innovation Centre (Montreal, Canada). As a quality control measure, each plate included DNA samples from six individuals who were members of two CEPH trios. Some plates also contained three duplicate pairs of quality control samples. Genotypes were called using GenCall <xref ref-type="bibr" rid="pgen.1003173-Kermani1">[9]</xref>. Initial calling was based on a cluster file generated using 270 samples from Hapmap2. To generate the final calls, we first selected a subset of 3,018 individuals, including samples from each of the genotyping centers in the iCOGS project, each of the participating consortia, and each major ethnicity. Only plates with a consistent high call rate in the initial calling were used. We also included 380 samples of European, African, and Asian ethnicity genotyped as part of the Hapmap and 1000 Genomes project, and 160 samples that were known positive controls for rare variants on the array. This subset was used to generate a cluster file that was then applied to call the genotypes for the remaining samples.</p><p>Of the 211,155 SNPs on the iCOGS array, we excluded SNPs for the following reasons (<xref ref-type="supplementary-material" rid="pgen.1003173.s008">Table S2</xref>): on the Y-chromosome, call rate &lt;95%, deviations from Hardy-Weinberg equilibrium (P&lt;10<sup>&#8722;7</sup>) using a stratified 1-d.f. test <xref ref-type="bibr" rid="pgen.1003173-Robertson1">[10]</xref>, and monomorphic. SNPs that gave discrepant genotypes among known duplicates were also excluded. After quality control filtering, 200,908 SNPs were available for analysis (<xref ref-type="supplementary-material" rid="pgen.1003173.s008">Table S2</xref>); 18,086 of which were selected on the basis of the discovery <italic>BRCA2</italic> GWAS <xref ref-type="bibr" rid="pgen.1003173-Gaudet1">[2]</xref>. Cluster plots of all reported SNPs were inspected manually for quality (<xref ref-type="supplementary-material" rid="pgen.1003173.s001">Figure S1</xref>).</p><p>Genotypes for SNPs identified through the 1000 Genomes Phase I data (released Jan 2012) <xref ref-type="bibr" rid="pgen.1003173-2">[11]</xref> were imputed using all SNPs on the iCOGS chip in a region of 500 kb around the novel modifier locus at 6p24. The boundaries were determined according to the linkage disequilibrium (LD) structure in the region based on HapMap data. The imputation was carried out using IMPUTE 2.2 <xref ref-type="bibr" rid="pgen.1003173-Howie1">[12]</xref>. SNPs with imputation information/accuracy r<sup>2</sup>&lt;0.30 were excluded in the analyses.</p><p>Of 10,048 genotyped samples (<xref ref-type="supplementary-material" rid="pgen.1003173.s008">Table S2</xref>), 742 were excluded because they did not meet the phenotypic eligibility criteria or had self-reported non-CEU ethnicity. Samples were then excluded for the following reasons: not female (XXY, XY), call rate &lt;95%, low or high heterozygosity (P&lt;10<sup>&#8722;6</sup>), discordant genotypes from previous CIMBA genotyping efforts, or discordant duplicate samples. For duplicates with concordant phenotypic data, or in cases of cryptic monozygotic twins, only one of the samples was included. Cryptic duplicates for which phenotypic data indicated different individuals were all excluded. Samples of non-European ancestry were identified using multi-dimensional scaling, after combining the <italic>BRCA2</italic> mutation carrier samples with the HapMap2 CEU, CHB, JPT and YRI samples using a set of 37,120 uncorrelated SNPs from the iCOGS array. Samples with &gt;19% non-European ancestry were excluded (<xref ref-type="supplementary-material" rid="pgen.1003173.s002">Figure S2</xref>). A total of 4,330 affected and 3,881 unaffected <italic>BRCA2</italic> mutation carrier women of European ancestry from 42 studies remained in the analysis (<xref ref-type="supplementary-material" rid="pgen.1003173.s007">Table S1</xref>), including 3,234 breast cancer cases and 3,490 unaffected carriers that were not in the discovery set.</p><p>Details of the sample collection, genotyping and quality control process for the <italic>BRCA1</italic> and BCAC samples, are reported elsewhere <xref ref-type="bibr" rid="pgen.1003173-Couch2">[13]</xref>, <xref ref-type="bibr" rid="pgen.1003173-Michailidou1">[14]</xref>.</p><p>The associations between genotype and breast cancer risk were analyzed within a retrospective cohort framework with time to breast cancer diagnosis as the outcome <xref ref-type="bibr" rid="pgen.1003173-Barnes1">[15]</xref>. Each <italic>BRCA2</italic> carrier was followed until the first event: breast or ovarian cancer diagnosis, bilateral prophylactic mastectomy, or age at last observation. Only those with a breast cancer diagnosis were considered as cases in the analysis. The majority of mutation carriers were recruited through genetic counseling centers where genetic testing is targeted at women diagnosed with breast or ovarian cancer and in particular to those diagnosed with breast cancer at a young age. Therefore, these women are more likely to be sampled compared to unaffected mutation carriers or carriers diagnosed with the disease at older ages. As a consequence, sampling was not random with respect to disease phenotype and standard methods of survival analysis (such as Cox regression) may lead to biased estimates of the associations <xref ref-type="bibr" rid="pgen.1003173-Antoniou2">[16]</xref>. We therefore conducted the analysis by modelling the retrospective likelihood of the observed genotypes conditional on the disease phenotypes. This has been shown to provide unbiased estimates of the associations <xref ref-type="bibr" rid="pgen.1003173-Barnes1">[15]</xref>. The implementation of the retrospective likelihoods has been described in detail elsewhere <xref ref-type="bibr" rid="pgen.1003173-Barnes1">[15]</xref>, <xref ref-type="bibr" rid="pgen.1003173-Antoniou3">[17]</xref>. The associations between genotype and breast cancer risk were assessed using the 1degree of freedom score test statistic based on the retrospective likelihood <xref ref-type="bibr" rid="pgen.1003173-Barnes1">[15]</xref>. In order to account for non-independence between relatives, an adjusted version of the score test was used in which the variance of the score was derived taking into account the correlation between the genotypes <xref ref-type="bibr" rid="pgen.1003173-Antoniou4">[18]</xref>. P-values were not adjusted using genomic control because there was little evidence of inflation. Inflation was assessed using the genomic inflation factor, &#955;. Since this estimate is dependent on sample size, we also calculated &#955; adjusted to 1000 affected and 1000 unaffected samples. Per-allele and genotype-specific hazard-ratios (HR) and 95% confidence intervals (CI) were estimated by maximizing the retrospective likelihood. Calendar-year and cohort-specific breast cancer incidences for <italic>BRCA2</italic> were used <xref ref-type="bibr" rid="pgen.1003173-Antoniou1">[1]</xref>. All analyses were stratified by country of residence. The USA and Canada strata were further subdivided by self-reported Ashkenazi Jewish ancestry. The assumption of proportional hazards was assessed by fitting a model that included a genotype-by-age interaction term. Between-country heterogeneity was assessed by comparing the results of the main analysis to a model with country-specific log-HRs. A possible survival bias due to inclusion of prevalent cases was evaluated by re-fitting the model after excluding affected carriers that were diagnosed &#8805;5 years prior to study recruitment. The associations between genotypes and tumor subtypes were evaluated using an extension of the retrospective likelihood approach that models the association with two or more subtypes simultaneously <xref ref-type="bibr" rid="pgen.1003173-Mulligan1">[19]</xref>. To investigate whether any of the significant SNPs were associated with ovarian cancer risk for <italic>BRCA2</italic> mutation carriers and whether the inclusion of ovarian cancer patients as unaffected subjects biased our results, we also analyzed the data within a competing risks framework and estimated HR simultaneously for breast and ovarian cancer using the methods described elsewhere <xref ref-type="bibr" rid="pgen.1003173-Barnes1">[15]</xref>. Analyses were carried out in R using the GenABEL libraries <xref ref-type="bibr" rid="pgen.1003173-Aulchenko1">[20]</xref> and custom-written software. The retrospective likelihood was modeled in the pedigree-analysis software MENDEL <xref ref-type="bibr" rid="pgen.1003173-Lange1">[21]</xref>, as described in detail elsewhere <xref ref-type="bibr" rid="pgen.1003173-Barnes1">[15]</xref>.</p><p>Affymetrix SNP 6.0 genotype calls for normal (non-tumor) breast DNA were downloaded for all available individuals from The Cancer Genome Atlas in September 2011. Analyses were limited to the 401 individuals of European ancestry based on principal component analysis. Expression levels in breast tumor tissue were adjusted for the top two principal components, age, gender (there are some male breast cancer cases in TCGA), and average copy number across the gene in the tumor. Linear regression was then used to test for association between the SNP and the adjusted gene expression level for all genes within one megabase.</p><p>To investigate enrichment of genes associated with breast cancer risk, the gene-set enrichment approach was implemented using Versatile Gene-based Association Study <xref ref-type="bibr" rid="pgen.1003173-Liu1">[22]</xref> based on the ranked P-values from retrospective likelihood analysis. Association List Go Annotator was also used to prioritize gene pathways using functional annotation from gene ontology (GO) <xref ref-type="bibr" rid="pgen.1003173-Ashburner1">[23]</xref> to increase the power to detect association to a pathway, as opposed to individual genes in the pathway. Both analyses were corrected for LD between SNPs, variable gene size, and interdependence of GO categories, where applicable, based on imputation. 100,000 Monte Carlo simulations were performed in VEGAS and 5000 replicate gene lists using random sampling of SNPs and 5000 replicate studies (sampling with replacement) were performed to estimate P-values.</p><p>We estimated the absolute risks of developing breast cancer based on the joint distribution of SNPs associated with breast cancer for <italic>BRCA2</italic> mutation carriers. The methods have been described elsewhere <xref ref-type="bibr" rid="pgen.1003173-Antoniou5">[24]</xref>. To construct the SNP profiles, we considered the single SNP from each region with the strongest evidence of association in the present dataset. We included all loci that had previously been found to be associated with breast cancer risk through GWAS in the general population and demonstrated associations with breast cancer risk for <italic>BRCA2</italic> mutation carriers, and loci that had GWAS level of significance in the current study. We assumed that all loci in the profile were independent (i.e. they interact multiplicatively on <italic>BRCA2</italic> breast cancer risk). Genotype frequencies were obtained under the assumption of Hardy-Weinberg Equilibrium. For each SNP, the effect of each allele was assumed to be consistent with a multiplicative model (log-additive). We assumed that the average, age-specific breast cancer incidences, over all associated loci, agreed with published breast cancer risk estimates for <italic>BRCA2</italic> mutation carriers <xref ref-type="bibr" rid="pgen.1003173-Antoniou1">[1]</xref>.</p>