<p>Our main study population was the Atherosclerosis Risk In Communities (ARIC) population. The ARIC population consists of a large sample of unrelated individuals and some families across North America. The population was recruited from four centers across the United States: Forsyth County, North Carolina; Jackson, Mississippi; Minneapolis, Minnesota; and Washington County, Maryland. For this study, we restricted our analysis to the European-American group. The population was recruited in 1987 from the general population consisting of subjects aged 45 to 64 years. The ARIC population consisted of 8,451 subjects.</p><p>Quality control and genotype calls for common SNPs were evaluated previously for ARIC using the Affymetrix Human SNP Array 6.0. We selected bilallelic autosomal markers based on the following criteria: missingness &lt;0.05, Hardy-Weinberg equilibrium (p&lt;10<sup>&#8722;6</sup>) and minor allele frequency &gt;0.05. Subjects with missingness &gt;0.05 were removed. This resulted in 436,126 retained markers.</p><p>Quality control measurements from dbGAP (GENEVA ARIC Project Quality Control Report Sept 22, 2009) indicate significant population stratification between self-identified white (European-ancestory kind group) and black populations when projected onto HapMap components. Furthermore, principal-components analysis of the European-ancestory group by dbGAP showed that no component explained more than 0.1% of the population variance. For this study we only analyzed the European-ancestory group and treated it as a single population.</p><p>ARIC phenotypes were adjusted for age, sex, and study center. Only single measurements from visit 1 were used for these subjects. We only used subjects with negative diabetes status and with genotype and phenotype information for all traits. This resulted in 8,451 subjects. We standardized all the traits. We first log-transformed BMI, glucose, insulin, triglycerides, HDL, and systolic blood pressure. All laboratory measurements are under fasting conditions. Population trait statistics are in <xref ref-type="supplementary-material" rid="pgen.1002637.s004">Table S1</xref>.</p><p>We estimated <italic>h<sup>2</sup></italic> and <italic>h<sub>g</sub><sup>2</sup></italic> for height and BMI in the Framingham Heart Study population (FHS). The FHS population is a large multi-generational dataset that started in 1948 in Framingham, Massachusetts in the United States. It consists of a number of ethnicities predominantly from the United Kingdom, Ireland, Italy, and Western Europe <xref ref-type="bibr" rid="pgen.1002637-Govindaraju1">[28]</xref>. Markers were screened similarly to ARIC and we also removed any SNPs that did not overlap with the ARIC set, which results in 320,118 SNPs. We used principal components analysis of the linkage disequilibrium (LD) pruned genetic relationship matrix to identify components with variance &gt;0.1%. LD pruning was as in the ARIC 2009 report. This resulted in 73,432 retained SNPs. We found three significant components that were then used as covariates in the REML model. For consistency with ARIC, we restricted the age range at time of exam to 45 to 65 years and randomly selected a single measurement in the case of multiple measurements. Phenotypes were adjusted for age, sex, and generation prior to the REML estimation and standardized. We first log-transformed BMI. Population trait statistics are in <xref ref-type="supplementary-material" rid="pgen.1002637.s005">Table S2</xref>. Our base FHS population consisted of 4,240 subjects.</p><p>We determined <italic>h<sup>2</sup></italic> using the linear mixed-effects model (see derivation below) and related individuals defined as genomic relatedness between 0.35 and 0.65. We assume that the common SNPs are in greater linkage disequilbrium among related individuals and, as such, can be used to estimate the total additive-genetic variance across the allele spectrum as suggested by Visscher et al., 2006 <xref ref-type="bibr" rid="pgen.1002637-Visscher2">[9]</xref>. We constrained the relationship matrix to have at least one related pair per subject. This was done by pruning the entire population relationship matrix by randomly selecting a row and removing the row and its corresponding column if no genomic covariance in the row was between the cutoff values. For all pairs, including unrelated individuals, we used their empirically defined relationship. This resulted in 530 individuals being selected for analysis in ARIC and 3,663 individuals in FHS.</p><p>
<italic>h<sup>2</sup></italic> was estimated with <italic>h<sup>2</sup></italic>&#8202;=&#8202;<italic>var<sub>g</sub></italic>/(<italic>var<sub>g</sub></italic>+<italic>var<sub>e</sub></italic>), where <italic>var<sub>g</sub></italic> and <italic>var<sub>e</sub></italic> are the genetic and residual variance components estimated by the REML model using related individuals. The error was estimated from the inverse Fisher Information (see linear mixed-effects model below) and propagated using a first-order Taylor expansion.</p><p>We used the linear mixed-effects model and only unrelated individuals to estimate the additive-genetic variance attributable to the common SNPs (<italic>h<sub>g</sub><sup>2</sup></italic>). Unrelated individuals were defined as subjects with maximum genomic correlation of &lt;0.025. The genomic relationship matrix was then produced as above based on this cutoff. The cutoff was taken from Yang et al. 2010 <xref ref-type="bibr" rid="pgen.1002637-Yang1">[6]</xref> and is less than the expected coefficient of relatedness between 2<sup>nd</sup> cousins. For these estimates we used the same group of 5,647 unrelated individuals for all estimates in ARIC and 1,489 individuals in FHS. <italic>h<sub>g</sub><sup>2</sup></italic> was estimated as <italic>h<sub>g</sub><sup>2</sup></italic>&#8202;=&#8202;<italic>var<sub>g</sub></italic>/(<italic>var<sub>g</sub></italic>+<italic>var<sub>e</sub></italic>), where <italic>var<sub>g</sub></italic> and <italic>var<sub>e</sub></italic> are the genetic and residual variance components estimated by the REML model using unrelated individuals. The standard error was estimated as above. The height h<sub>g</sub>
<sup>2</sup> versus SNP number analyses were performed over allele frequency range of 0.05 to 0.5 in order of increasing and decreasing frequency.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The genetic correlation (<italic>r<sub>g</sub></italic>) is defined as <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e001.jpg" /></inline-formula>, where (var<sub>g</sub>(<italic>t<sub>i</sub></italic>)) is the additive genetic variance of trait <italic>i</italic> and covariance (cov<sub>g</sub>(<italic>t<sub>i</sub>,t<sub>j</sub></italic>)) is the additive genetic covariance between the traits. The variances and covariances are estimated directly in the multivariate linear mixed-effects model. The error was computed from the estimated errors of the variances and covariance using a first-order Taylor expansion. The residual and phenotypic correlations were analogously defined. Phenotype correlations and error were estimated by linear regression of the standardized phenotypes.</p><p>The mean and errors for proportionality constants between the genetic and phenotypic correlations were determined by randomly sampling over the distributions of the parameter estimates (i.e. Monte Carlo method) assuming that the error around the mean parameter estimate was normally distributed and that the parameters were independent. We then fit a linear function with the y-intercept fixed at 0 (after first confirming that it was not significantly different from zero).</p><p>We assessed significance for correlation coefficients (<italic>r</italic>) using the standardized Fisher transformed estimate of r: arctan(<italic>r</italic>)/arctan(s.e.(r)). We estimated the two-tailed p-value from a normal distribution and significance was determined by p&lt;0.05 and Bonferroni corrected for 21 hypotheses.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Significance for regression coefficient (<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e002.jpg" /></inline-formula>) was estimated using the standardized coefficient <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e003.jpg" /></inline-formula>. We estimated the two-tailed p-value from a t-distribution and 20 degrees of freedom and significance was determined by p&lt;0.05.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Preprocessing of SNPs and phenotypes was done using PLINK <xref ref-type="bibr" rid="pgen.1002637-Purcell1">[29]</xref> (v1.07,<ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/purcell/plink/">http://pngu.mgh.harvard.edu/purcell/plink/</ext-link>) and MATLAB (2010b, MathWorks, Natick, MA). REML optimization was executed using software written in MATLAB.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We considered the following multivariate linear mixed-effects model for <italic>m</italic> individuals, <italic>n</italic> loci and <italic>t</italic> traits <xref ref-type="bibr" rid="pgen.1002637-Yang1">[6]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1002637-Lynch1">[8]</xref>, <xref ref-type="bibr" rid="pgen.1002637-Deary1">[17]</xref>, <xref ref-type="bibr" rid="pgen.1002637-Lee1">[18]</xref>, <xref ref-type="bibr" rid="pgen.1002637-Price1">[30]</xref>:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002637.e004.jpg" /></disp-formula>where <bold>y</bold>
<italic><sub>i</sub></italic> is a <italic>m</italic>&#215;1 vector of trait <italic>i</italic> for m individuals, <bold>X</bold>
<sub>i</sub> is an <italic>m</italic>&#215;<italic>s</italic> fixed effects matrix for trait <italic>i</italic>, <bold>v</bold>
<italic><sub>i</sub></italic> is a <italic>s</italic>&#215;1 vector of fixed effects parameters for trait <italic>i</italic>, <bold>Z</bold> is an <italic>m</italic>&#215;<italic>n</italic> matrix of standardized genotypes, <bold>u</bold>
<italic><sub>i</sub></italic> is an <italic>n</italic>&#215;1 vector of random effects for trait <italic>i</italic> satisfying <bold>u</bold>
<sub>i</sub>&#8764;<bold>N</bold>(<bold>0</bold>,<bold>G</bold>) and <bold>e</bold>
<italic><sub>i</sub></italic> is an <italic>m</italic>&#215;1 vector of residual effects satisfying <bold>e</bold>
<sub>i</sub>&#8764;<bold>N</bold>(<bold>0</bold>,<bold>R</bold>), with matrix blocks <bold>G</bold>
<sub>ij</sub>&#8202;=&#8202;cov<sub>gij</sub>
<bold>I</bold>
<sub>n</sub> and <bold>R</bold>
<sub>ij</sub>&#8202;=&#8202;cov<sub>eij</sub>
<bold>I</bold>
<sub>m</sub> and <bold>I</bold>
<italic><sub>l</sub></italic> is the <italic>l</italic>&#215;<italic>l</italic> identity matrix. This model can be used for single or multiple traits. For two traits, it is called a bivariate model. The model is identical to that used by <xref ref-type="bibr" rid="pgen.1002637-Yang1">[6]</xref>, <xref ref-type="bibr" rid="pgen.1002637-Yang2">[7]</xref>, <xref ref-type="bibr" rid="pgen.1002637-Deary1">[17]</xref>.</p><p>We considered only bi-allelic SNPs in Hardy-Weinberg equilibrium. Denote the minor allele by q and the major allele by Q. Let the minor allele frequency at locus <italic>i</italic> have frequency <italic>p</italic>
<sub>i</sub>. We assign a value of 2 for genotype qq, 1 for genotype qQ and 0 for genotype QQ. The Hardy-Weinberg mean frequency for the genotype at locus <italic>i</italic> is 2<italic>p</italic>
<sub>i</sub> and the variance is 2<italic>p<sub>i</sub></italic>(1&#8722;<italic>p<sub>i</sub></italic>). The standardized genotype entries have values of (2&#8722;2<italic>p</italic>
<sub>i</sub>)/(2<italic>p</italic>
<sub>i</sub>(1&#8722;2<italic>p</italic>
<sub>i</sub>))<sup>1/2</sup> for qq, (1&#8722;2<italic>p</italic>
<sub>i</sub>)/(2<italic>p</italic>
<sub>i</sub>(1&#8722;2<italic>p</italic>
<sub>i</sub>))<sup>1/2</sup> for qQ, and &#8722;2<italic>p</italic>
<sub>i</sub>/(2<italic>p</italic>
<sub>i</sub>(1&#8722;2<italic>p</italic>
<sub>i</sub>))<sup>1/2</sup> for the QQ genotype.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The log of the likelihood function is given by<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002637.e005.jpg" /></disp-formula>where the covariance matrix can be expressed as a tensor product <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e006.jpg" /></inline-formula> with <italic>m</italic>&#215;<italic>m</italic> blocks <bold>V</bold>
<sup>&#8722;1</sup>
<italic><sub>ij</sub></italic> and <bold>A</bold> is the genetic relationship matrix. Following Yang et al. <xref ref-type="bibr" rid="pgen.1002637-Yang1">[6]</xref>, we used a modified covariance matrix for <bold>A</bold>, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e007.jpg" /></inline-formula>, where the diagonals of <bold>A</bold> are computed using the formula<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002637.e008.jpg" /></disp-formula>We use the restricted maximum likelihood (REML) approach <xref ref-type="bibr" rid="pgen.1002637-Lynch1">[8]</xref> where the gradients of the log likelihood are given by<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002637.e009.jpg" /></disp-formula>
<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002637.e010.jpg" /></disp-formula>where <bold>I</bold>
<italic><sub>ij</sub></italic> is a <italic>tm</italic>&#215;<italic>tm</italic> dimensional matrix with zero entries except for a <italic>m</italic>&#215;<italic>m</italic> identity matrix at block location <italic>i</italic>, <italic>j</italic>, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e011.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e012.jpg" /></inline-formula>, where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e013.jpg" /></inline-formula>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We solved the REML equations using an EM algorithm <xref ref-type="bibr" rid="pgen.1002637-Lynch1">[8]</xref>, which was given by<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002637.e014.jpg" /></disp-formula>
<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002637.e015.jpg" /></disp-formula>for iteration <italic>k</italic>+1 in terms of iteration <italic>k</italic>. We iterated until the rate of change of the log likelihood function was less than about 10<sup>&#8722;4</sup>. We also checked that the rate of change of the square of the covariance predictions was less than 10<sup>&#8722;8</sup>. We checked our results against the software developed by Yang et al. (GCTA) <xref ref-type="bibr" rid="pgen.1002637-Yang3">[31]</xref> for the univariate model.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For the multivariate model, we transformed to a coordinate system where the covariance matrices were diagonal <xref ref-type="bibr" rid="pgen.1002637-Lynch1">[8]</xref> to speed up the computation. Let <bold>z</bold>
<sub>j</sub> be the set of phenotypes for individual <italic>j</italic>. We used the canonical transformation <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e016.jpg" /></inline-formula> such that <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e017.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e018.jpg" /></inline-formula>. <bold>Q</bold> can be computed from the formula <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e019.jpg" /></inline-formula> where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e020.jpg" /></inline-formula>, (<bold>S</bold> is the matrix of left eigenvectors of <bold>GR</bold>
<sup>&#8722;1</sup>). The transformed genetic covariances are given by <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e021.jpg" /></inline-formula> and the residual covariances are <bold>I</bold>
<italic><sub>t</sub></italic>. Each step consisted of taking a single step with the univariate EM algorithm for the transformed additive genetic and residual variance followed by a transformation back to the original coordinates. We iterated until the maximum of the magnitudes of the components of the gradient of the log likelihood function was less than approximately <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e022.jpg" /></inline-formula>.</p><p>In our computations, we used both the direct EM algorithm and the canonically transformed algorithm because even though the transformed algorithm was in principle faster, it sometimes had poor convergence properties if the initial guess was not sufficiently close to the maximum likelihood value. We ensured that both give the same results. For computational efficiency, the results shown are computed from the bivariate model for the different trait pairs. We confirmed our results with a multivariate model that included all traits.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Our error estimates were given by the inverse of the Fisher information matrix <bold>F</bold>, which we computed by evaluating the Hessian of the log likelihood at the maximum likelihood predictions. <bold>F</bold> is a <italic>t(t+1)</italic>&#215;<italic>t(t+1)</italic> dimensional matrix with rows corresponding to the genetic and residual variances and covariances (where cov<sub>ij</sub> was set equal to cov<sub>ji</sub>) and with block elements (that are not all contiguous) given by<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002637.e023.jpg" /></disp-formula>for <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e024.jpg" /></inline-formula>and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002637.e025.jpg" /></inline-formula>.</p>