<p>Nineteen independent studies contributed to our analyses. All study protocols were approved by the respective local Institutional Review Boards, and written informed consent for genetic studies was obtained from all participants included in our analyses.</p><p>Of the 19 studies contributing to our analyses, 18 studies came from the CHARGE <xref ref-type="bibr" rid="pgen.1003098-Hancock1">[8]</xref>, <xref ref-type="bibr" rid="pgen.1003098-Psaty1">[55]</xref> or SpiroMeta <xref ref-type="bibr" rid="pgen.1003098-Repapi1">[9]</xref> Consortium: Age, Gene, Environment, Susceptibility (AGES) &#8211; Reykjavik Study <xref ref-type="bibr" rid="pgen.1003098-Harris1">[56]</xref>; Atherosclerosis Risk in Communities (ARIC) Study <xref ref-type="bibr" rid="pgen.1003098-2">[57]</xref>; British 1958 Birth Cohort (B58C) <xref ref-type="bibr" rid="pgen.1003098-Strachan1">[58]</xref>; Coronary Artery Risk Development in Young Adults (CARDIA) <xref ref-type="bibr" rid="pgen.1003098-Hughes1">[59]</xref>, <xref ref-type="bibr" rid="pgen.1003098-Friedman1">[60]</xref>; Cardiovascular Health Study (CHS) <xref ref-type="bibr" rid="pgen.1003098-Fried1">[61]</xref>; European Community Respiratory Health Survey (ECRHS) <xref ref-type="bibr" rid="pgen.1003098-Burney1">[62]</xref>; European Prospective Investigation into Cancer and Nutrition (EPIC, obese cases and population-based subsets) <xref ref-type="bibr" rid="pgen.1003098-Day1">[63]</xref>; Framingham Heart Study (FHS) <xref ref-type="bibr" rid="pgen.1003098-Dawber1">[64]</xref>, <xref ref-type="bibr" rid="pgen.1003098-Feinleib1">[65]</xref>; Health, Aging, and Body Composition (Health ABC) Study <xref ref-type="bibr" rid="pgen.1003098-Yende1">[66]</xref>; Northern Finland Birth Cohort of 1966 (NFBC1966) <xref ref-type="bibr" rid="pgen.1003098-Jarvelin1">[67]</xref>, <xref ref-type="bibr" rid="pgen.1003098-Rantakallio1">[68]</xref>; Multi-Ethnic Study of Atherosclerosis (MESA) <xref ref-type="bibr" rid="pgen.1003098-Bild1">[69]</xref>, <xref ref-type="bibr" rid="pgen.1003098-Rodriguez1">[70]</xref>; Rotterdam Study (RS-I, RS-II, and RS-III) <xref ref-type="bibr" rid="pgen.1003098-Hofman1">[71]</xref>; Swiss Study on Air Pollution and Lung Diseases in Adults (SAPALDIA) <xref ref-type="bibr" rid="pgen.1003098-Martin1">[72]</xref>; Study of Health in Pomerania (SHIP) <xref ref-type="bibr" rid="pgen.1003098-Volzke1">[73]</xref>; and TwinsUK <xref ref-type="bibr" rid="pgen.1003098-Andrew1">[74]</xref>. We reached out to other population-based studies with GWAS genotyping and data available on cigarette smoking and pulmonary function, resulting in the inclusion of LifeLines <xref ref-type="bibr" rid="pgen.1003098-Stolk1">[75]</xref>. Given the greater power needed to detect novel genetic loci with subtle gene-environment interaction regardless of the statistical method used <xref ref-type="bibr" rid="pgen.1003098-Aschard1">[16]</xref>, we chose to maximize statistical power to discover novel genetic loci by combining all available participants and to use the regression coefficients across the many different component studies as evidence for consistency. This approach was similarly taken by another large-scale GWAS consortium for discovering SNP main effects <xref ref-type="bibr" rid="pgen.1003098-Moffatt1">[24]</xref>.</p><p>All studies were included in our previous GWAS meta-analysis of pulmonary function or the follow-up replication analyses, wherein their pulmonary function testing protocols were described <xref ref-type="bibr" rid="pgen.1003098-SolerArtigas1">[10]</xref>. For studies with spirometry at a single visit (B58C, LifeLines, MESA, NFBC1966, SHIP, RS-I, RS-II, and RS-III), we analyzed FEV<sub>1</sub>/FVC and FEV<sub>1</sub> measured at that visit. For studies with spirometry at more than one visit, we analyzed measurements from the baseline visit (AGES, ARIC, CARDIA, CHS, ECRHS, EPIC obese cases, EPIC population-based, Health ABC, and SAPALDIA) or the most recent examination with spirometry data (FHS and TwinsUK).</p><p>Smoking history (current-, past-, and never-smoking) was ascertained by questionnaire at the time of pulmonary function testing. Pack-years of smoking were calculated for current and past smokers by multiplying smoking amount (packs/day) and duration (years smoked). <xref ref-type="supplementary-material" rid="pgen.1003098.s013">Table S11</xref> presents the specific questions used to ascertain smoking history and pack-years in each of the 19 studies.</p><p>Study participants were genotyped on various genotyping platforms, and standard quality control filters for call rate, Hardy-Weinberg equilibrium p-value, MAF, and other measures were applied to the genotyped SNPs (<xref ref-type="supplementary-material" rid="pgen.1003098.s014">Table S12</xref>). To generate a common set of SNPs for meta-analysis, imputation was conducted with reference haplotype panels from HapMap phase II subjects of European ancestry (CEU) (<xref ref-type="supplementary-material" rid="pgen.1003098.s014">Table S12</xref>) <xref ref-type="bibr" rid="pgen.1003098-3">[76]</xref>. Imputed genotype dosage values (estimated reference allele count with a fractional value ranging from 0 to 2.0) were generated for approximately 2.5 million autosomal SNPs. Among participants with genome-wide SNP genotyping data, exclusions were made due to standard quality control metrics (call rate, discordance with prior genotyping, and genotypic and phenotypic sex mismatch among others), missing pulmonary function data, or missing covariate data (<xref ref-type="supplementary-material" rid="pgen.1003098.s015">Table S13</xref>).</p><p>Our analyses included 50,047 participants from 19 studies who passed their study-specific quality control and had complete data on pulmonary function and smoking. Each study transformed the pulmonary function measures to residuals using linear regression of FEV<sub>1</sub>/FVC (%) and FEV<sub>1</sub> (mL) on age, age<sup>2</sup>, sex, and standing height as predictors. Principal component eigenvectors and recruitment site were also included as covariates to adjust for population stratification (if applicable). The residuals were converted to z scores (henceforth referred to as standardized residuals). We confirmed that smoking was inversely associated with the FEV<sub>1</sub>/FVC and FEV<sub>1</sub> standardized residuals in all 19 studies (meta-analysis &#946;&#8202;=&#8202;&#8722;0.0030 and corresponding <italic>P</italic>&lt;1&#215;10<sup>&#8722;6</sup> for pack-years of smoking).</p><p>The FEV<sub>1</sub>/FVC and FEV<sub>1</sub> standardized residuals were used as the phenotypes for genome-wide association testing with linear regression models, which included the following predictor variables: imputed SNP genotype dosages, smoking history (dichotomous variable, 0&#8202;=&#8202;never-smokers and 1&#8202;=&#8202;ever-smokers), smoking status (dichotomous variable, 0&#8202;=&#8202;never- and past-smokers and 1&#8202;=&#8202;current-smokers), pack-years of smoking (continuous variable), and a SNP-by-smoking interaction product term. Two of the 19 studies (FHS and TwinsUK) had much relatedness among participants, and we took appropriate account of relatedness in the association testing (<xref ref-type="supplementary-material" rid="pgen.1003098.s014">Table S12</xref>). Four regression models with interaction terms for ever-smoking or pack-years were specified in relation to standardized residuals for FEV<sub>1</sub>/FVC or FEV<sub>1.</sub> As it has long been advised in studying interactions, the regression models were designed to fully saturate the main smoking effect on pulmonary function, so that the interaction terms do not capture residual main effects <xref ref-type="bibr" rid="pgen.1003098-Shahani1">[77]</xref>. In each of the 19 studies, the genome-wide analyses were implemented with robust variance estimation using the software packages indicated in <xref ref-type="supplementary-material" rid="pgen.1003098.s014">Table S12</xref>.</p><p>Our analyses were aimed at finding novel loci associated with pulmonary function when considering an interaction with cigarette smoking, so we chose to implement JMA of SNP main and interactive SNP-by-smoking effects (two d.f. test of the null hypothesis &#946;<sub>SNP</sub>&#8202;=&#8202;0 and &#946;<sub>INT</sub>&#8202;=&#8202;0) <xref ref-type="bibr" rid="pgen.1003098-Manning2">[15]</xref>. Manning et al. previously compared the joint methods, such as JMA, with other methods that incorporate gene-environment interaction (such as screening by main effects <xref ref-type="bibr" rid="pgen.1003098-Kooperberg1">[78]</xref> or conducting a 1 d.f. meta-analysis of the interaction product term), and they found that the joint methods offer optimal statistical power over a range of scenarios for SNP main and interactive effects <xref ref-type="bibr" rid="pgen.1003098-Manning2">[15]</xref>, <xref ref-type="bibr" rid="pgen.1003098-Kraft1">[33]</xref>. Therefore, our analyses centered on the JMA method, which simultaneously estimates regression coefficients for the SNP and SNP-by-smoking interaction terms, while accounting for their covariance, to generate a joint test of significance <xref ref-type="bibr" rid="pgen.1003098-Manning2">[15]</xref>. It also accounts for the unequal variances from studies of different sample sizes. Secondarily, we implemented meta-analyses of just the &#946; coefficient from the interaction term for comparison with the JMA results. Of note, the two-step gene-environment interaction study designs by Murcray et al. <xref ref-type="bibr" rid="pgen.1003098-Murcray1">[79]</xref>, <xref ref-type="bibr" rid="pgen.1003098-Murcray2">[80]</xref> and Gauderman et al. <xref ref-type="bibr" rid="pgen.1003098-Gauderman2">[81]</xref> are applicable to case-control or case-parent trio studies, respectively, and were thus not considered for our population-based studies of continuous traits.</p><p>The JMA was conducted with fixed effects on approximately 2.5 million SNPs using METAL software (version 2010-02-08) <xref ref-type="bibr" rid="pgen.1003098-Willer1">[82]</xref> and patch source code provided by Manning et al. <xref ref-type="bibr" rid="pgen.1003098-Willer1">[82]</xref>. Genomic control correction was applied by computing &#955;<sub>gc</sub> as the ratio of the observed and expected (2 d.f.) median chi-square statistics and dividing the observed chi-square statistics by &#955;<sub>gc</sub>. SNPs having <italic>P</italic>
<sub>JMA</sub>&lt;5&#215;10<sup>&#8722;8</sup> (the standard Bonferroni-adjusted <italic>P</italic> value) were considered statistically significant <xref ref-type="bibr" rid="pgen.1003098-Peer1">[83]</xref>. Further correction for the four different (albeit related) JMA models yielded a conservative <italic>P</italic>
<sub>JMA</sub> threshold of 1.25&#215;10<sup>&#8722;8</sup>. In addition to reporting the <italic>P</italic>
<sub>JMA</sub> for the most significant SNP from each novel locus, we used the &#946; and standard error (SE) estimates from the JMA results to calculate the <italic>P</italic> values corresponding to the SNP main association (<italic>P</italic>
<sub>SNP</sub>) and the SNP-by-ever-smoking interaction (<italic>P</italic>
<sub>INT</sub>) <xref ref-type="bibr" rid="pgen.1003098-Manning2">[15]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Gene annotation was performed using the gene prediction tracks &#8220;UCSC Genes&#8221; and &#8220;RefSeq Genes&#8221; in the UCSC browser (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu">http://genome.ucsc.edu</ext-link>). The &#8220;sno/miRNA&#8221; track from the USCS browser was used to search for any microRNA within 100 kb upstream or downstream of each SNP, and the &#8220;TFBS Conserved&#8221; track was used to search for conserved transcription factor binding sites (TFBSs) at or near the most significant SNPs. The SNAP program <xref ref-type="bibr" rid="pgen.1003098-Johnson1">[84]</xref> was used to infer LD patterns, based on the HapMap phase II CEU population.</p><p>We used separate types of expression analyses to confirm the biologic plausibility of our findings. First, we carried out mRNA expression profiling to show whether or not the implicated genes are expressed in human tissues relevant to pulmonary function. The mRNA expression profiles of implicated genes were determined using reverse transcription polymerase chain reaction (RT-PCR). RNA was sourced from lung (Ambion/ABI), human bronchial epithelial cells (Clonetics) <xref ref-type="bibr" rid="pgen.1003098-Wadsworth1">[85]</xref>, and peripheral blood mononuclear cells (3H Biomedica). RNA from human airway smooth muscle cells, cultured as previously described from tissue obtained at thoracotomy <xref ref-type="bibr" rid="pgen.1003098-Sayers1">[86]</xref>, was extracted using a commercially available kit (Qiagen). Ethical approval for the use of primary cells was obtained from the local ethics committees. cDNA was generated using 1 &#181;g of RNA template using random hexamers and a SuperScript kit (Invitrogen) as directed by the manufacturer. PCR assays were designed to cross intron-exon boundaries, where possible and where splice variation was known, in order to detect all variants. The <italic>GAPDH</italic> gene was used as a positive control for the cDNA quality, and water was used as a negative control. Primer sequences for the genes of interest are given in <xref ref-type="supplementary-material" rid="pgen.1003098.s016">Table S14</xref>. All PCR were done using Platinum Taq High Fidelity (Invitrogen) with 100 ng of cDNA template in a 25 &#181;L reaction. Cycling conditions were as follows: 94&#176;C for 2 minutes, 35 cycles of 94&#176;C for 45 seconds, 55&#176;C for 30 seconds, and 68&#176;C for 90 seconds. Following PCR, gel bands were directly sequenced to confirm the presence of the gene's transcript.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Second, we used another publically available data repository to investigate whether any of the implicated genes showed evidence for differential expression depending on smoking history. The gene expression profiles of human airway epithelium from healthy smokers (N&#8202;=&#8202;10) and nonsmokers (N&#8202;=&#8202;12) were obtained from the Gene Expression Omnibus site (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</ext-link>) <xref ref-type="bibr" rid="pgen.1003098-Edgar1">[27]</xref>, <xref ref-type="bibr" rid="pgen.1003098-Barrett1">[28]</xref>, based on robust multichip average processing of probe intensities from Affymetrix HG-U133 Plus 2.0 microarrays (GEO dataset number GSE4498) <xref ref-type="bibr" rid="pgen.1003098-Harvey1">[29]</xref>. Mean expression levels of genes around our genome-wide significant findings from the JMA were compared between smokers versus nonsmokers. The <italic>P</italic> value for the difference in means between smokers and nonsmokers was calculated using the nonparametric Mann-Whitney test.</p><p>Third, our genome-wide significant SNPs from novel loci were searched against an expression quantitative trait loci (eQTL) data repository based on lymphoblastoid cell lines <xref ref-type="bibr" rid="pgen.1003098-Dixon1">[32]</xref>, to investigate whether any of the implicated SNP variants might influence the expression of the nearby genes. <italic>P</italic>&lt;5&#215;10<sup>&#8722;8</sup> was used to designate statistically significant eQTL associations.</p>