<p>Our sample included 1189 narcolepsy cases, 1136 of whom were seen at the sleep laboratory of People's Hospital, Peking University, Beijing, a unit in the pulmonary medicine department evaluating patients with sleep disorders and receiving referrals from all over China. In addition, 51 samples came from Taiwan (Dr. Huang, National Taiwan University) and two from Stanford. All patients had either documented hypocretin deficiency (CSF hypocretin-1&#8804;110 pg/ml, n&#8202;=&#8202;119), or clear-cut cataplexy and HLA-DQB1*06:02 <xref ref-type="bibr" rid="pgen.1003880-Hallmayer1">[6]</xref>. Cases were mostly Han descent (87%) and from North China (85%). The majority of our cases were male (67%). A majority of cases are children (70%) <xref ref-type="bibr" rid="pgen.1003880-Han4">[21]</xref>; mean age was 11.2&#177;0.2 years (11.6&#177;0.3 in males versus 10.55&#177;0.39 for females). Clinical data included age of disease onset (earliest onset of cataplexy or sleepiness), presence or absence of cataplexy, sleepiness, sleep paralysis, hypnogogic hallucinations, and disturbed nocturnal sleep, and Multiple Sleep Latency Testing data (mean sleep latency and number of sleep onset REM sleep periods). Delay between disease onset and diagnosis was also noted <xref ref-type="bibr" rid="pgen.1003880-Han2">[9]</xref>
<xref ref-type="bibr" rid="pgen.1003880-Han3">[18]</xref>. Mean &#177; SEM are reported for age of onset, and diagnostic delay. Control genotypes from China came from university employees and students (41% male), and shared controls from GWAS studies underway for colon cancer and Sjogren's syndrome.</p><p>Informed consent in accordance with governing institutions was obtained from all subjects. The research protocols were approved by IRB Panels on Medical Human Subjects at both Stanford University and the Beijing University People's Hospital.</p><p>DNA samples were genotyped on the Affymetrix Axiom CHB array. Genotypes were called using Affymetrix Genotyping Console. Individuals with call rate &lt;95%, or that were outliers following principal components analysis (n&#8202;=&#8202;47), or related (n&#8202;=&#8202;53), were removed, leaving 1189 cases and 1997 controls. For the main association study, we selected SNP variants with MAF &#8805;1%, call rate &#8805;90%, and HWE p value &#8805;0.001 in controls. Because of the near requirement for DQB1*06:02 in narcolepsy, an extended HLA region from rs1419229 to rs9368865 (Chr 6:24112537-35363736) was excluded from the primary association, leaving 603,382 autosomal, non-HLA SNP variants. The extended HLA region contained 3,000 SNPs and was analyzed separately for imputation of HLA haplotypes and effects on clinical presentation.</p><p>The majority of the analysis (including quality control, LD calculations, quantitative trait association, and interactions) was performed using the Plink suite of software (v 1.07) <xref ref-type="bibr" rid="pgen.1003880-Purcell1">[35]</xref>. To control for population stratification, the association was performed using a variance component model implemented in EMMAX <xref ref-type="bibr" rid="pgen.1003880-Kang1">[36]</xref>. Cluster quality of top ranking SNP markers was verified by visual examination of clusters. As EMMAX does not return OR or MAF data, these data were gathered and reported from corresponding analyses in Plink. Inflation statistics were estimated, and QQ plots were generated using estlambda (GenABEL package (v1.7-4) in R (2.15.3) <xref ref-type="bibr" rid="pgen.1003880-Aulchenko1">[37]</xref>
<xref ref-type="bibr" rid="pgen.1003880-Team1">[38]</xref>. In addition, Principal Components was performed, and Manhattan plot generated using SVS v7 (GoldenHelix).</p><p>The top ranked 150 SNPs from associations performed in EMMAX and Plink (a total of 188 SNPs, p&#8804;0.001 for EMMAX, &#8804;0.003 for Plink) were selected for follow-up. After visual check and exclusion of 16 poorly clustered SNPs, we selected the top-ranked SNP from each locus (defined as 2 SNPs within 100 kb) resulting in 80 variants for replication in our previously published European cohorts (Affymetrix 6.0 array study: 807 narcolepsy, 1071 controls; Illumina Immunochip array study 1886 cases, 10,421 controls). The Affymetrix and Illumina Immunochip samples were partially overlapping and thus the replication was first done in the larger Illumina immunochip sample. If a variant was not present or tagged by variants on that array, then replication was next attempted in the smaller Affymetrix 6.0 data set. When a selected CHB variant was not on the corresponding array, an analysis of LD in a 10&#8211;20 kb window was performed in Chinese and Europeans (Ensembl 1000genomes browser LD data) to identify potential proxies with an r2 &#8805;0.8 available on the ImmunoChip or Affymetrix 6.0 array. In the case of rs1551570, the variant was not tagged on either array, but shows strong LD with rs2305795, a SNP found to be associated with narcolepsy following fine mapping <xref ref-type="bibr" rid="pgen.1003880-Kornum1">[7]</xref> where corresponding p values were extracted from the previous analysis. A Bonferroni correction was applied to determine significance.</p><p>Multiple Sleep Latency Testing data (mean sleep latency and number of sleep onset REM sleep periods) and age of onset were studied using genome wide linear regressions as quantitative traits (Plink). Genome-wide significant values obtained by Plink were then re-tested using EMMAX, with EMMAX P values reported in <xref ref-type="table" rid="pgen-1003880-t002">Table 2</xref>. Presence or absence of cataplexy, sleep paralysis, hypnogogic hallucinations, as well as whether or not onset was prior to September 2009, were studied as binary phenotypes (Plink).</p><p>We imputed genotypes in windows that fully surrounded each of the 5 replicated loci. Imputation was performed separately in the Chinese Axiom CHB, and European ImmunoChip, Affymetrix 6.0 using Beagle v3.3 <xref ref-type="bibr" rid="pgen.1003880-Browning1">[39]</xref> against the CHB reference Chinese population, or 4 European populations (286 individuals from CEU, TSI, GBR, IBS) in the 1000 genomes integrated data set (phase 1 release v3). Imputation was performed in the Chinese cohort for all five replicated loci (TRA@, TRB@, ZNF365, PPAN-DNMT-1, IL10RB-IFNAR1 region). These loci were also imputed in European ancestry narcolepsy samples depending on regional SNP coverage on the corresponding genotype array (TRA@ in Affymetrix 6.0, with rs1154155 also in Immunochip; TRB@ in Affymetrix 6.0, not covered in ImmunoChip; ZNF365 in ImmunoChip, not covered on Affymetrix 6.0; PPAN-DNMT1 in Affymetrix 6.0, not covered in ImmunoChip, IL10RB-IFNAR1 Affymetrix 6.0- no SNPs in LD with rs2834188 on ImmunoChip). Imputed genotypes were combined (Chinese+ ImmunoChip, or Chinese+Affymetrix 6.0) and associations were performed using a Mantel Haenszel test (Plink). SNP markers with poor imputation quality scores in either Chinese or Europeans (r2&lt;0.8) were excluded from further analysis. Plots of association statistics were made using LocusZoom <xref ref-type="bibr" rid="pgen.1003880-Pruim1">[40]</xref>.</p><p>High resolution HLA typing had been performed on a subset of 239 narcolepsy cases and 14 controls for HLA DRB1, DQA1 and DQB1 genes <xref ref-type="bibr" rid="pgen.1003880-Han1">[3]</xref>, all positive for DQB1*06:02. These individuals were also genotyped on the Axiom CHB array (see above). In order to impute the HLA genotypes for the rest of the data set, the HLA types, together with array-genotyped SNPs in a 500 kb window surrounding HLA DRB1, DQA1 and DQB1 were submitted as a training set to HIBAG package in R. HIBAG is an HLA imputation tool that uses attribute bootstrap aggregation of several classifiers (SNPs) to select groups of SNPS that predict HLA type <xref ref-type="bibr" rid="pgen.1003880-Zheng1">[41]</xref>. The resulting sets of haplotype predictive SNPs were then used to impute HLA type in the remaining samples (cases and controls). Allele frequencies of DRB1, DQA1 and DQB1 alleles obtained after imputation was consistent with population data (r2&#8202;=&#8202;0.96) <xref ref-type="bibr" rid="pgen.1003880-Hei1">[42]</xref>. Imputation was acceptable in both DQB1*06:02 positive and negative controls, but in consideration of our training set, we only used data from DQB1*06:02 positive individuals, which was imputed with better quality (85.0% in DQB1*06:02 positive individuals and 77.5% in DQB1*06:02 negative individuals). Imputation of DQB1*03:01, DQB1*06:02 and DQB1*01:02, the alleles of principal interest in this study was highly accurate, (DQB1*03:01: 92.8% in controls 97.6% in cases, DQB1*06:02: 90.5% in controls, 97.4% in cases, DQA1*01:02: 92.2% in controls, 97.5% in cases) in the data set. The imputation quality may be overestimated in cases since the narcolepsy population is more homogenous for their HLA haplotype.</p><p>The imputed HLA haplotype data was first used to study association between rs7744020, rs9274477, rs17212223 (age of onset associated), rs9271117, rs9270965 (associated with onset prior versus after September 2009) and various HLA genotypes (<xref ref-type="table" rid="pgen-1003880-t002">Table 2</xref>). To verify whether the HLA association in narcolepsy versus controls was consistent with previous reports in other ethnic groups <xref ref-type="bibr" rid="pgen.1003880-Mignot1">[1]</xref>, including Chinese subjects <xref ref-type="bibr" rid="pgen.1003880-Han1">[3]</xref> (N&#8202;=&#8202;1183 DQB1*06:02 positive cases and N&#8202;=&#8202;438 DQB1*06:02 positive controls), we next categorized subjects into five risk groups based on our prior model of allelic competition <xref ref-type="bibr" rid="pgen.1003880-Han1">[3]</xref>: 1) DQB1*06:02 homozygous (highest risk); 2) DQB1*03:01 carriers (second highest risk); 3) DQA1*01:02-DQB1*05 or DQB1*06 non-06:02 (intermediate) 4) DQA1*01 non-DQA1*01:02 (protective) 5) alleles with no predisposition or protection effects for narcolepsy. These categories were then compared using &#967;<sup>2</sup> square tests between cases versus controls and cases after versus before September 2009. &#967;<sup>2</sup> analyses were performed sequentially, starting with DQB1*06:02 homozygotes, then moving on to DQB1*03:01 carriers, then to DQA1*01:02-DQB1*05 or DQB1*06 non-06:02 carriers, and finally to DQA1*01 non-DQA1*01:02 carriers, as previously reported <xref ref-type="bibr" rid="pgen.1003880-Han1">[3]</xref>, a technique similar to relative predisposition effect statistics <xref ref-type="bibr" rid="pgen.1003880-Hollenbach1">[43]</xref>.</p><p>Potential interactions between narcolepsy risk SNPs and HLA genotypes were analyzed with R version 2.15.3, and with an epistasis model Y &#8764;b0 + b1.A + b2.B + b3.AB +e implemented in Plink. The interaction was performed with narcolepsy, onset after/before 2009 and with age of onset.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://faculty.washington.edu/browning/beagle/beagle.html">http://faculty.washington.edu/browning/beagle/beagle.html</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://bochet.gcc.biostat.washington.edu/beagle/1000_Genomes.phase1_release_v3/">http://bochet.gcc.biostat.washington.edu/beagle/1000_Genomes.phase1_release_v3/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://csg.sph.umich.edu/locuszoom/">http://csg.sph.umich.edu/locuszoom/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://www.R-project.org/">http://www.R-project.org/</ext-link>
</p>