<p>Human sample studies have been approved by the University of Maryland Medical School Institutional Review Board and conform to the Declaration of Helsinki.</p><p>Animal procedures were performed in compliance with protocols approved by the Institutional Animal Care and Use Committee of the University of Maryland.</p><p>Human cervical carcinoma HeLa cells were cultured in DMEM (Gibco BRL, Gaithersburg, MD) supplemented with 10% FBS. Diffuse large B-cell lymphoma (DLBCL) and Burkitt lymphoma (BL) cell lines were cultured in RPMI medium 1640 (Gibco BRL, Gaithersburg, MD) containing 10% FBS. HeLa cells were transfected by using Oligofectamine (Invitrogen) with 50 nM small RNAs Pre-miR-520c-3p (AAAGUGCUUCCUUUUAGAGGGU), control Pre-miR-Ctrl (Ambion), or 20 nM siRNA targeting eIF4GII and control siRNA (Qiagen). Farage cells were transfected with Nucleofector Kit V (Amaxa, Inc., Germany). For transduction of SUDHL4 cells plasmids pCDH-Vector or pCDH-miR520c-3p (System Biosciences, CA) were transfected with PureFection (System Biosciences, CA) in HEK-293TN packaging cells. Afterward, SUDHL4 cells were co-cultivated with the viral particle producing HEK-293TN cells for 48 h, transferred to flasks and subjected to selection with 0.5 &#181;g/ml puromycin. Plasmids pLenti-UTR-Luc-Blank or pLenti-UTR-Luc-eIF4GII(3&#8242;UTR) were purchased from Applied Biological Materials Inc. (BC, Canada), miRZip-Vector and miRZip-520c-3p constructs were from System Biosciences. Cloning of the eIF4GII 3&#8242;UTR fragments containing seed regions (wild-type or mutants) of miR-520c-3p in plasmid pcDNA3.1/NT-GFP-TOPO were performed using NT-GFP Fusion TOPO TA Expression Kit (Invitrogen). The EIF4G3 lentiviral expression vector was generated through amplification of EIF4G3's coding region (CD) from a human cDNA library and cloning the product into pCDH-EF1-MCS-IRES-Puro (Systems Bioscience).</p><p>Human sample studies have been approved by the University of Maryland Medical School Institutional Review Board and conform to the Declaration of Helsinki. Primary normal B cells were purified from human peripheral blood mononuclear cells (PBMCs) obtained from Sanguine Biosciences, Inc. (Valencia, CA). B cells isolation was performed with using human B Cell Isolation Kit (B-CLL) from Miltenyi Biotec Inc (Auburn, CA) according to manufacturer's protocol. Collected B cells purity was determined by flow cytometry. DLBCL primary cells were obtained from lymph node biopsies, followed by cell density ficoll-gradient separation. Tonsils were provided by the UMGCC Pathology Biorepository and Research Core to serve as normal Germinal Center B-cell (GCB) controls. GCBs were isolated from tonsils as previously described <xref ref-type="bibr" rid="pgen.1004105-Greeve1">[58]</xref> with purity greater than 95%, CD19+, CD38+ and IgD-.</p><p>Polysomal fractionations were carried out and microarray analysis was performed as previously described <xref ref-type="bibr" rid="pgen.1004105-MazanMamczarz2">[59]</xref>. Briefly, cell lysates were fractionated by centrifugation through 10&#8211;50% linear sucrose gradients and divided into 11 fractions. RNA was purified from each fraction for microarray and RT-qPCR analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">RNA purified from sucrose fractions was labeled using Illumina TotalPrep RNA Amplification Kit (Ambion; Austin, TX). Human HT-12 v1.0 gene expression BeadChips containing 48,000 RefSeq transcripts (Illumina, San Diego, CA), were used for microarray analysis. Raw microarray data were filtered by the detection p-value&#8804;0.02, normalized by Z-score transformation and tested for significant differences in signal intensity. The sample quality was analyzed by scatter plot, principal component analysis (PCA), and gene sample z-scores based hierarchy clustering to exclude possible outliers. ANOVA test was used to eliminate the genes with larger variances within each comparison group. Genes were considered to be significantly changed after calculating Z-ratio, indicating fold difference (Z&gt;1.5 or &lt;&#8722;1.5), false discovery rate (fdr), which controls for the expected proportion of false rejected hypothesis (fdr&#8804;0.3) and p&lt;0.05. Genes differentially expressed in studied groups were analyzed for representation of Gene Ontology (GO) terms to identify key biologically functional categories that were significantly changed (p&lt;0.05) in miR-520c-3p overexpressed versus control transfected cells as described before <xref ref-type="bibr" rid="pgen.1004105-MazanMamczarz3">[60]</xref>. Ingenuity Pathways Analysis IPA (<ext-link ext-link-type="uri" ns0:href="https://analysis.ingenuity.com/">https://analysis.ingenuity.com/</ext-link>; Ingenuity Systems; Redwood City, CA) was performed to identify the top network functions amongst the genes translationally regulated by miR-520c-3p. Network analysis utilizes a curated knowledge base on known functional interactions and protein functions to algorithmically infer biochemical interactions. Significance of functions and pathways was calculated using the right-tailed Fisher's Test. See <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40489">www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40489</ext-link> for complete array results.</p><p>Total cellular RNA or RNA from polysomal fractions were purified by using Trizol (Invitrogen), reverse transcribed with the iScript cDNA synthesis kit (BioRad, Hercules, CA) and quantified by real-time qPCR analysis using gene-specific primer pairs. qPCR analysis was carried out by using iQ SYBR Green Supermix (Quanta) and BioRad iCycler instrument. RNA from tissue microarrays (TMAs) was isolated using RecoverAll Total Nucleic Acid Isolation Kit for FFPE (Life Technologies) according to manufacturer's protocol. Mature miR-520c-3p and RNU6B were quantified using TaqMan MicroRNA Assays (Applied Biosystems). Each reaction was carried out in triplicate and three independent experiments were run.</p><p>For real-time qPCR quantification of selected miR-520c-3p targets following gene-specific primer pairs were used: <named-content content-type="gene">CCCTTCAAGATGCACCTCAT</named-content> and <named-content content-type="gene">GTGGTCCACAGCATTTCCTT</named-content> for eIF4GII mRNA, <named-content content-type="gene">GATGCATTGCAGAGGCACTA</named-content> and <named-content content-type="gene">CCCTAATGCTGGAAGCACTC</named-content> for FOXO3 mRNA, <named-content content-type="gene">TGCAATCCAGCCAAATACAA</named-content> and <named-content content-type="gene">TTATCCAGCTTCAGCCAGGT</named-content> for VIL2 mRNA, <named-content content-type="gene">GGACCTTGGCCTCAGTGTTA</named-content> and <named-content content-type="gene">GAAAGTTGCAGGCACCAAAT</named-content> for HNRPDL mRNA, <named-content content-type="gene">TCCCCTTTCCCTGATCTTTT</named-content> and <named-content content-type="gene">TGGGATTCTGTGGGAGAAAG</named-content> for HuR mRNA, <named-content content-type="gene">TGCTATCATGGCAGAAGGAA</named-content> and <named-content content-type="gene">ATATGCCACAGCCCATCATT</named-content> for MCT-1 mRNA and, <named-content content-type="gene">CGGAGTCAACGGATTTGGTCGTAT</named-content> and <named-content content-type="gene">AGCCTTCTCCATGGTGGTGAAGAC</named-content> for GAPDH mRNA.</p><p>For analysis of <italic>de novo</italic> translation, 72 h after transfection 10<sup>6</sup> cells were incubated with 1 mCi L-[<sup>35</sup>S]methionine and L-[<sup>35</sup>S]cysteine (Easy Tag EXPRESS; PerkinElmer Life and Analytical Sciences, Waltham, MA) per 6-well plate for 20 minutes. Cell lysates were then resolved by SDS-PAGE and transferred onto PVDF membranes. Radiolabel incorporation was visualized by PhosphorImager (GE Healthcare).</p><p>10&#8211;50 &#181;g of whole cell lysates were resolved by SDS-PAGE (Invitrogen, Carlsbad, CA) and transferred onto PVDF membranes. After incubation with monoclonal antibodies recognizing eIF4GII (Sigma), p53, HuR, VIL2, GFP (all from Santa Cruz), GAPDH and &#946;-Actin (both from Abcam, Cambridge, UK), or polyclonal FOXO3 (Cell Signaling), p16, HNRPDL, eIF4E (all from Santa Cruz), blots were incubated with the secondary antibodies (Santa Cruz). Signals were detected by enhanced chemiluminescence (Pierce, Rockford, IL).</p><p>72 h after transfections cells were fixed with 70% ethanol, washed with PBS and digested with RNAse (100 &#181;m/ml) at 37 &#176;C for 1 h. The cells were then stained with 50 &#181;g/ml PI and analyzed for cell cycle with a flow cytometer. Cell proliferation in real-time was monitored by the xCELLigence System, RTCA DP Instrument (Roche Diagnostics GmbH, Germany). The cells were also stained with Trypan Blue (Sigma-Aldrich) and counted using a hemocytometer. The percentage of apoptotic cells was determined by flow cytometry with the Annexin V staining kit (BD Biosciences, CA). Senescence induction in the cells was tested by measurement of &#946;-galactosidase activity with Senescence &#946;-Galactosidase staining kit (Cell Signaling) and Western blot of p16, p53 and HuR senescence markers.</p><p>For cap-binding assay, 500 &#181;g total cell lysates were incubated with a 7-Methyl GTP-Sepharose beads (GE Healthcare, Little Chalfont, UK) overnight at 4&#176;C. After washing of the beads, the bound proteins were analyzed by Western blot.</p><p>Farage cells were transfected with Pre-miR-Ctrl, Pre-miR-520c-3p, eIF4GII siRNA or Ctrl siRNA and cultured in agarose/medium. 24 h later cells were plated at 7500 cells/ml in a 1&#8758;3 agarose to DMEM mixture on the top of solidified 1&#8758;1 agarose/DMEM feeder layer and covered with MDEM. Two weeks later colonies (&gt;50 cells) were counted and pictures were taken with using Cell Colony Counter (Microbiology International).</p><p>Tissue microarrays, containing 10 reactive lymph nodes, 54 DLBCL and 6 other types of lymphoma were purchased from Biomax and stained with Vectastain Elite ABC kit (Vector Laboratories) as described before <xref ref-type="bibr" rid="pgen.1004105-Dai1">[61]</xref>. Anti-eIF4GII antibody (Abcam) was used at 1&#8758;1000 dilution, 4&#176;C, overnight. Hematoxylin-eosin (HE) staining was used for morphologic examination. The DLBCL was classified into GCB and ABC subtypes, according to well-accepted immunohistochemistry-based algorithms <xref ref-type="bibr" rid="pgen.1004105-Dai1">[61]</xref>. The immunohistochemical staining was evaluated according to the following criteria: negative, &lt;25% neoplastic cells staining; low, 26&#8211;50% neoplastic cells staining; high, &gt;50% neoplastic cells staining. The images were analyzed with a Nikon eclipse 50i microscope (Nikon Instruments).</p><p>Severe combined immunodeficiency (SCID) Beige mice were maintained in a pathogen&#8211;free environment under controlled conditions of light and humidity. Animal procedures were performed in compliance with protocols approved by the Institutional Animal Care and Use Committee of the University of Maryland. Six weeks old male mice were injected with 1.5&#215;10<sup>6</sup> SUDHL4 cells, either expressing empty vector or overexpressing miR-520c-3p into the left and right dorsal flanks, resuspended in 100 &#181;L PBS and mixed with an equal volume of Matrigel. Tumors were measured and volumes were calculated as previously described <xref ref-type="bibr" rid="pgen.1004105-Dai1">[61]</xref>, <xref ref-type="bibr" rid="pgen.1004105-Dai2">[62]</xref>. Thirty-one days after injection, tumors were harvested for further analysis. The effect of miR-520c-3p overexpression on tumor growth was evaluated using analysis of variance (ANOVA) with time as a repeated measure factor.</p>