<p>This study was carried out in strict accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health. The protocol was approved by the Animal Care and Use Committee of the NIAID, NIH (protocol number LPD 68E). Invertebrates are not covered under NIH guidelines.</p><p>
<italic>L. major</italic> Fn/Sat was derived from a strain (MHOM/IL/80/Friedlin) originally isolated from a patient with cutaneous Leishmaniasis acquired in the Jordan Valley, and contains a heterozygous nourseothricin&#8211;resistance (SAT) marker, integrated along with a linked firefly luciferase reporter gene into one allele of the &#8764;20 rRNA cistrons located on chromosome 27 <xref ref-type="bibr" rid="pgen.1003672-MartinezCalvillo1">[31]</xref> using constructs and methods described previously <xref ref-type="bibr" rid="pgen.1003672-Akopyants1">[13]</xref>; <italic>L. major</italic> LV39c5/Hyg was derived from a strain originally isolated from a reservoir rodent host in southern Russia (MHOM/Sv/59/P) <xref ref-type="bibr" rid="pgen.1003672-Marchand1">[32]</xref> and is heterozygous for an allelic replacement of the <italic>LPG5A</italic> on chromosome 24 by a hygromycin B&#8211;resistance cassette <xref ref-type="bibr" rid="pgen.1003672-Capul1">[16]</xref>; <italic>L. major</italic> Sd/Hyg was derived from a strain isolated from a patient with cutaneous lesions acquired in Senegal (MHOM/SN/74/SD) <xref ref-type="bibr" rid="pgen.1003672-Neva1">[33]</xref>, and is also heterozygous for an allelic replacement of the <italic>LPG5A</italic> on chromosome 24 by a hygromycin B&#8211;resistance cassette <xref ref-type="bibr" rid="pgen.1003672-Capul1">[16]</xref>; <italic>L. major</italic> Sd/Bsd is derived from the same Sd strain from Senegal and is heterozygous for allelic replacement of the <italic>LPG5B</italic> on chromosome 18 by a blastocidin-resistance cassette <xref ref-type="bibr" rid="pgen.1003672-Capul1">[16]</xref>; <italic>L. major</italic> Ry/Sat was derived from a strain originally isolated from a lesion biopsy of a laboratory worker accidentally exposed to sand flies that were experimentally infected with a strain of <italic>L. major</italic> (WR2885) originating in Iraq <xref ref-type="bibr" rid="pgen.1003672-Stamper1">[34]</xref>, and also contains a heterozygous SAT&#8211;resistance marker, integrated into one allele of the rRNA cistrons located on chromosome 27 using constructs and methods described previously <xref ref-type="bibr" rid="pgen.1003672-Akopyants1">[13]</xref>. All parasites were grown at 26&#176;C in complete medium 199 (CM199) supplemented with 20% heat-inactivated FCS, 100 U/ml penicillin, 100 &#181;g/ml streptomycin, 2 mM L-glutamine, 40 mM Hepes, 0.1 mM adenine (in 50 mM Hepes), 5 mg/ml hemin (in 50% triethanolamine), and 1 mg/ml 6-biotin, and containing either 25 ug/ml blasticidin S (Invitrogen, Carlsbad, CA), 25 ug/ml hygromycin B (EMD Biosciences, San Diego, CA) or 100 ug/ml SAT (Jena Bioscience, Germany), or combinations of these antibiotics as necessary.</p><p>Three to five day old female <italic>P. duboscqi</italic> or <italic>L. longipalpis</italic> sand flies, obtained from colonies initiated from field specimens collected in Mali and Brazil, respectively, were fed through a chick skin membrane on heparinized mouse blood containing 1&#8211;4&#215;10<sup>6</sup>/ml logarithmic phase promastigotes of each parental line of <italic>L. major</italic>. At various days after the infective feed (3&#8211;18 days), the sand flies were anesthetized with CO<sub>2</sub>, their midguts dissected and homogenized in 100 &#181;l CM199, and the number of viable promastigotes and the percentage of each promastigote developmental stage were determined by counting under a hemacytometer. Promastigote stages were identified based on the morphologic criteria previously described <xref ref-type="bibr" rid="pgen.1003672-Saraiva1">[35]</xref>. For isolation of hybrids, individual midgut homogenates from co-infected flies were transferred to a single well of a 96 well flat bottom plate, and incubated at 26&#176;C over night. An equal volume of CM199 containing 2&#215; of the appropriate antibiotics was added on the following day for the selection of hybrids. Doubly drug-resistant promastigotes were cloned by distribution in 96 well blood agar plates in 0.1 ml CM199 containing both antibiotics. Poisson analysis was used to determine the percentage probability of clonality, and was in each case &gt;95%.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Parasite DNA was extracted as follows: cell extracts were suspended in TELT (0.05 M Tris, 005 M EDTA, 0.25 M LiCl and Triton X-4) and supplemented with an equal Volume of Phenol Chloroform isoamyl alcohol (25&#8758;24&#8758;1). Cell suspensions were centrifuged (15&#8242; 16,000 g 4&#176;C) and the aqueous phase was transferred to a new tube. DNA was precipitated in 96% ethanol and washed with 70% ethanol. DNA was then eluted with 100 ul of nuclease free water. For hybrids genotyping, a set of marker genes (<xref ref-type="supplementary-material" rid="pgen.1003672.s008">Table S1</xref>) was amplified by polymerase chain reaction (PCR) using Applied Biosystems 2&#215; geneAmpPCR mix, 25 pmol of each primer and 10 ng of the DNA. PCR products were cleaned with ExoSAP-IT kit (USB), and sequences confirmed with forward and/or reverse reads by Rocky Mountain Laboratory Genomics Unit DNA Sequencing Center, Division of Intramural Research, Hamilton, Montana. The sequences were analyzed using &#8220;Lasergene&#8221; software. SNP-CAPS analysis was performed as previously described <xref ref-type="bibr" rid="pgen.1003672-Mannaert1">[9]</xref> and was based on SNPs identified in the sequence analysis. Glucose phosphate dehydrogenase (G6PD-LmjF34.0080) was PCR-amplified and cleaned using gel and PCR clean up system (Promega). The clean PCR product (500 ng) was digested with 15 units of SACII (Thermo scientific) for 16 hours. Cleaved DNA was then loaded on ethidium bromide gel. Band intensity quantification was done using image J software (<ext-link ext-link-type="uri" ns0:href="http://rsbweb.nih.gov/ij">http://rsbweb.nih.gov/ij</ext-link>, National Institutes of Health, Bethesda, MD, USA).</p><p>DNA content was determined using flow cytometry as previously described <xref ref-type="bibr" rid="pgen.1003672-Cruz1">[21]</xref>. Briefly, 2.5&#215;10<sup>6</sup> log-phase promastigotes were permeabilized with 4% paraformaldehide solution for 1 minute. Cells were centrifuged, re-suspended in PBS and fixed with absolute methanol for 15 minutes on ice. Cells were washed and resuspended in PBS at room temperature for 10 minutes followed by RNAseA treatment (200 &#181;g/ml RNase A) and propidium iodide (20 &#181;g/ml) staining for 30 minutes at room temperature. Data were acquired on a FacsCANTO 2 flow cytometer (BD Bioscience), counting at least 10,000 cells per sample, and analyzed using FlowJo 9.5.2 software.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The full methods and analysis of the parental and hybrid <italic>L. major</italic> lines studied here will be presented elsewhere (J. Shaik, N. Akopyants, D. Dobson, P. Lawyer, D. Elnaiem, D. Sacks, and S. Beverley, in preparation). Briefly, Leishmania nuclear DNA was isolated and subjected to deep-sequencing using Illumina Hi-seq 2000 machine, yielding approximately 60&#215; coverage. Reads were aligned to the <italic>L. major</italic> reference genome (REF) using novoalign MPI version 2.07.07 (Novocraft, 2012); &#8220;NovoAlign (<ext-link ext-link-type="uri" ns0:href="http://www.novocraft.com">http://www.novocraft.com</ext-link>).&#8221;) and SNPs were identified using the Samtools pileup function using SOAP SNP consensus model <xref ref-type="bibr" rid="pgen.1003672-Li1">[36]</xref>. Custom scripts were developed for subsequent analysis and display. SNPs were filtered to remove those falling within regions of low coverage (read depth less than 10) and to remove SNPs heterozygous within either parent. For the remaining homozygous SNPs, an &#8216;allelic&#8217; density at each homozygous SNP position was estimated, assuming an average ploidy of 2n across the <italic>L. major</italic> genome. While not strictly true for any given parent or hybrid line (all of which vary from each other), the error introduced by this assumption is small. The results of these analyses are displayed as &#8216;bottlebrush&#8217; plots (<xref ref-type="fig" rid="pgen-1003672-g003">Fig. 3</xref>, <xref ref-type="supplementary-material" rid="pgen.1003672.s004">Fig. S4</xref>) where at each SNP position the inferred allelic density is displayed on the Y-axis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To test if there is a change over time in the proportion of hybrids recovered within each cross, we used an exact two-sided Cochran-Armitage test for trend using the day as the score combining across experiments and assuming no experiment effects. To compare the number of promastigotes per gut between flies yielding hybrids versus those not yielding hybrids, we used a stratified (by day) Wilcoxon-Mann-Whitney test. Tests were performed using SAS Version 9.3 (Cochran-Armitage) or the coin package in Hothorn et al. <xref ref-type="bibr" rid="pgen.1003672-Hothorn1">[37]</xref>. URL <ext-link ext-link-type="uri" ns0:href="http://www.jstatsoft.org/v28/i08/">http://www.jstatsoft.org/v28/i08/</ext-link>.].</p>