<p>All yeast strains used in this study are isogenic to CG379 <xref ref-type="bibr" rid="pgen.1003149-Morrison1">[51]</xref> with the following common markers: <italic>MAT&#945; his7-2 leu2-3,112 trp1-289</italic>. The <italic>ADE2</italic>, <italic>CAN1</italic>, and <italic>URA3</italic> genes were deleted from their native locations and reintroduced into the left subtelomeric region of Chromosome V using the <italic>delitto perfetto</italic> approach <xref ref-type="bibr" rid="pgen.1003149-Storici1">[52]</xref>. The <italic>cdc13-1</italic> mutation was introduced using the integration/pop-out technique described in <xref ref-type="bibr" rid="pgen.1003149-Nugent1">[11]</xref>. <xref ref-type="sec" rid="s1">Introduction</xref> of <italic>cdc13-1</italic> was verified by phenotype, restriction digest, and sequencing. <italic>ung1</italic>&#916; and <italic>rev3x</italic>&#916; derivative strains were constructed by one-step gene replacement <xref ref-type="bibr" rid="pgen.1003149-Brachmann1">[53]</xref> using the kanMX4 and natMX4 <xref ref-type="bibr" rid="pgen.1003149-Goldstein1">[54]</xref> antibiotic resistance markers, respectively. Deletion strains were confirmed by phenotype and PCR. Yeast strains were grown on YPDA liquid (1% yeast extract, 2% peptone, 2% dextrose, supplemented with 0.01% adenine sulfate, filter-sterilized) or agar (same recipe as YPDA liquid with 2% agar added, autoclaved), except for strains bearing tetracycline regulatable plasmids, which were maintained on TRP dropout plates.</p><p>The tetracycline regulatable centromeric shuttle plasmid pCM252 <xref ref-type="bibr" rid="pgen.1003149-Bell1">[18]</xref> was a gift from Prof. E. Herrero (Universidad de Lleida, Spain). The APOBEC3G198-384-2K3A ORF <xref ref-type="bibr" rid="pgen.1003149-Chen1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1003149-Harjes1">[17]</xref> was amplified from plasmid pGST-A3G-CTD-2K3A <xref ref-type="bibr" rid="pgen.1003149-Carpenter1">[55]</xref> by PCR using primers that introduced ClaI and StuI restriction sites. The APOBEC3G fragment then was cloned into pCM252 to generate pCM252-A3G. Correctly cloned plasmid isolates were identified by sequencing of the APOBEC3G ORF, as well as flanking sequences extending approximately 200 bp in the 5&#8242; and 3&#8242; directions. Plasmids were transformed into yeast using a standard lithium acetate technique <xref ref-type="bibr" rid="pgen.1003149-Storici1">[52]</xref>.</p><p>Reporter strains bearing pCM252-A3G were maintained on TRP dropout plates. Individual colonies were inoculated each into a 5-mL YPDA liquid culture and grown at 23&#176;C for 48 hours. 0.5 mL of each 48-hour culture was combined with 4.5 mL of fresh YPDA supplemented by 10 &#181;g/mL doxycycline (Sigma-Aldrich, St. Louis, MO), and shifted to 37&#176;C for 6 hours. Then, cells were collected by centrifugation, washed twice in water, and appropriate dilutions were plated onto ARG dropout plates with 60 &#181;g/mL canavanine sulfate and 20 &#181;g/mL adenine sulfate to select for mutants, as well as onto synthetic complete plates to assess viability.</p><p>Individual colonies from each reporter strain each were inoculated into a 5-mL YPDA culture, and grown at 23&#176;C for 72 hours. 0.5 mL of each 72-hour culture was combined with 4.5 mL of fresh YPDA, and shifted to 37&#176;C for 6 hours. Then, cells were collected, washed twice, and counted. 2&#215;10<sup>7</sup> cells from each culture were resuspended in 1 mL of 1% sodium bisulfite in 100 mM sodium citrate buffer, pH 5.2, and incubated at 37&#176;C for 2.5 hours. Then, cells were collected by centrifugation, washed twice in water, and plated as described for APOBEC3G-induced mutagenesis.</p><p>Plates with mutagenized cells were incubated at 23&#176;C for five days, and counted using an aColyte 7510/SYN colony counter (Microbiology International, Frederick, MD). Replicas of selection media plates were made onto ADE dropout, URA dropout, and glycerol plates to identify multi-loss-of-function isolates that retained mitochondrial function. Isolates of interest were streaked onto YPDA and an individual colony from each streaking was tested to verify loss of function in the reporter genes. Genomic DNA was prepared by a QIAcube robot (QIAGEN, Valencia, CA), using the manufacturer's protocol. Each reporter ORF, as well as the 3&#8242; portion of <italic>LYS2</italic>, was amplified by PCR using the primers listed in <xref ref-type="supplementary-material" rid="pgen.1003149.s005">Table S1</xref>. Sequencing of these PCR products was outsourced to Eton Biosciences (Research Triangle Park, NC) using the primers listed in <xref ref-type="supplementary-material" rid="pgen.1003149.s006">Table S2</xref>. Mutations were identified using Seqman software (DNASTAR, Madison, WI) and graphed using Excel (Microsoft, Redmond, WA).</p><p>Prism 6 software (GraphPad Software, LaJolla, CA) was used to evaluate statistical significance of the data. The Kolmogorov-Smirnov test was applied to evaluate statistical significance of differences in viability and frequency of gene inactivation (i.e., data in <xref ref-type="fig" rid="pgen-1003149-g002">Figure 2</xref>, <xref ref-type="fig" rid="pgen-1003149-g004">Figure 4</xref>, and <xref ref-type="fig" rid="pgen-1003149-g006">Figure 6</xref>). The Chi-square test was used to compare the difference in proportions of mutation types or motif preference between WT and <italic>ung1</italic>&#916; cells (i.e., the data in <xref ref-type="table" rid="pgen-1003149-t001">Table 1</xref> and <xref ref-type="table" rid="pgen-1003149-t002">Table 2</xref>).</p>