<p>Samples for the iCOGS study were drawn from 25 studies participating in the PRACTICAL Consortium <xref ref-type="bibr" rid="pgen.1004129-Eeles1">[7]</xref>. The majority of studies were population-based or hospital-based case-control studies, or nested case-control studies. All studies have the relevant IRB approval in each country in accordance with the principles embodied in the Declaration of Helsinki. After exclusion of samples that failed quality control (QC) in the iCOGS study or showed substantial non-European ancestry, genotype data for 20,440 PrCa cases and 21,469 matched controls were available.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To improve imputation performance, Illumina OMNI2.5 SNP array data were available for 677 UK PrCa cases from the UKGPCS study (<ext-link ext-link-type="uri" ns0:href="http://www.icr.ac.uk/ukgpcs">www.icr.ac.uk/ukgpcs</ext-link>); 262 of these cases were also genotyped on the iCOGS array. The rare coding variant rs138213197 was also genotyped in 2476 cases and 2198 controls from the UK (UKGPCS study), and 3024 cases and 2725 controls from Sweden (CAPS and STHM1 study).</p><p>Detailed information relating to the custom iCOGS Illumina Infinium array can be found in Eeles <italic>et al.</italic>, 2013 <xref ref-type="bibr" rid="pgen.1004129-Eeles1">[7]</xref>. With respect to the <italic>HOXB</italic> locus, 747 SNPs spanning the interval chr17:46201311&#8211;47382559 were genotyped on the iCOGS array, submitted by a combination of the PRACTICAL and OCAC consortia (Supplementary <xref ref-type="supplementary-material" rid="pgen.1004129.s001">Figure S1</xref>).</p><p>To boost imputation performance, additional genotyping of 677 PrCa cases from the UK was conducted using the Illumina (San Diego, CA, USA) OMNI2.5 BeadChip according to the manufacturer's instructions. Further genotyping of the rs138213197 variant was carried out by Taqman assay (Applied Biosystems Inc., Foster City, CA, USA) for 2476 cases and 2198 controls from the UK and by MassARRAY iPLEX (Sequenom Inc., San Diego, CA, USA) for 3024 cases and 2725 controls from Sweden.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Imputation was performed on 20,440 case and 21,469 control samples across the 700 iCOGS SNPs from the <italic>HOXB13</italic> interval which passed pre-imputation QC metrics <xref ref-type="bibr" rid="pgen.1004129-Anderson1">[37]</xref> (Supplementary <xref ref-type="supplementary-material" rid="pgen.1004129.s002">Figure S2</xref>). IMPUTE v2.3.0 <xref ref-type="bibr" rid="pgen.1004129-Howie1">[38]</xref>, <xref ref-type="bibr" rid="pgen.1004129-Howie2">[39]</xref> was used to impute the interval Chr17:46200000&#8211;47400000 (GRCh37/hg19). Two panel imputation <xref ref-type="bibr" rid="pgen.1004129-Sampson1">[40]</xref> was performed using OMNI2.5 BeadChip data for 677 PrCa cases from the UKGPCS study (Panel 1) and a 1000 Genomes Phase 1 integrated variant set &#8220;version 3&#8221; (SNPs and in/dels) from 5<sup>th</sup> March 2012 (Panel 0) (<ext-link ext-link-type="uri" ns0:href="http://mathgen.stats.ox.ac.uk/impute/data_download_1000G_phase1_integrated.html">http://mathgen.stats.ox.ac.uk/impute/data_download_1000G_phase1_integrated.html</ext-link>). Imputation concordance was examined using &#8220;leave one out&#8221; internal concordance check. For single panel imputation (Panel 0 only) concordance was 96.5% at SNPs <italic>r<sup>2</sup></italic>&#8805;0.5 and 98.0% at <italic>r<sup>2</sup></italic>&#8805;0.9, which rose to 98.9% at <italic>r<sup>2</sup></italic>&#8805;0.5 and 99.8% at <italic>r<sup>2</sup></italic>&#8805;0.9 respectively for two panel (Panel 0+1) imputation. SNPs with info &lt;0.5, MAF&lt;0.01 were excluded during QC filtering.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Association tests were performed on genotypes in the MaCH dosage format (0&#8211;2) converted from the IMPUTE genotype posterior probabilities using GenABEL <xref ref-type="bibr" rid="pgen.1004129-Aulchenko1">[41]</xref>. Associations between each SNP and PrCa risk were analysed using a per-allele trend test, adjusted for study and six principal components derived from analysis of the whole iCOGS dataset <xref ref-type="bibr" rid="pgen.1004129-Eeles1">[7]</xref>. Odds ratios (OR) and 95% confidence limits were estimated using unconditional logistic regression. Tests of homogeneity of the ORs across strata were assessed using a likelihood ratio test. SNPs significant at <italic>P</italic>&lt;10<sup>&#8722;6</sup> were considered for further analysis. The independence of these associations was assessed by performing a conditional logistic regression analysis. For further assessment of the relationship between rs138213197 and rs117576373, haplotype analyses were performed with Plink 1.07 (<ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/purcell/plink/">http://pngu.mgh.harvard.edu/purcell/plink/</ext-link>) on the subset of samples where both SNPs had been directly genotyped. Haplotypes were first imputed using the Expectation-Maximisation algorithm in order to then perform a case-control association, using 10,000 permutations of the phenotype labels <xref ref-type="bibr" rid="pgen.1004129-Purcell1">[42]</xref>.</p>