<p>All <italic>C. elegans</italic> strains were propagated on standard nematode growth agar (NGA) plates containing streptomycin (200 &#181;g/mL) and spotted with OP50-1, as previously described <xref ref-type="bibr" rid="pgen.1004020-Sulston1">[90]</xref>. For specific experiments, worms were fed HB101, OP50, or HT115 <italic>E. coli</italic> strains using NGA containing streptomycin (HB101) or no antibiotics (OP50 and HT115).</p><p>The following <italic>C. elegans</italic> mutants were obtained from the <italic>C. elegans</italic> Genetics Center, which is supported in part by NIH Office of Research Infrastructure Programs (P40 OD010440): <italic>daf-16(mgDf50)</italic> I, <italic>cat-2(e1112)</italic> II, <italic>pah-1(ok687)</italic> II, <italic>tdc-1(ok914)</italic> II, <italic>crh-1(tz2)</italic> III, <italic>daf-2(e1368)</italic> III, <italic>hpd-1(ok1955)</italic> III, <italic>unc-119(ed3)</italic> III, <italic>akt-1(mg144)</italic> V, <italic>akt-1(mg306)</italic> V, <italic>aak-2(gt33)</italic> X, <italic>akt-2(ok393)</italic> X, <italic>daf-12(rh61rh411)</italic> X, <italic>pdk-1(mg142)</italic> X, <italic>sgk-1(ok538)</italic> X, <italic>muIs84[pAD76(sod-3::GFP)]</italic>, <italic>lpIs14 [daf-16f::GFP+unc-119(+)]</italic>, and <italic>lpIs12 [daf-16a::RFP+unc-119(+)]. tatn-1(baf1)</italic> X has been described previously, and <italic>tatn-1(qd182)</italic> was identified in an unrelated mutagenesis screen and is a gift from Daniel Pagano and Dennis Kim <xref ref-type="bibr" rid="pgen.1004020-Fisher1">[43]</xref>, <xref ref-type="bibr" rid="pgen.1004020-Ferguson1">[44]</xref>. <italic>muIs109[daf-16::GFP]</italic> X has been described previously and is a gift from Malene Hansen <xref ref-type="bibr" rid="pgen.1004020-Berman1">[59]</xref>. <italic>daf-16(mgDf47)</italic>, <italic>hsd-1(mg433)</italic> I, <italic>eak-3(mg344)</italic> III, <italic>eak-4(mg348)</italic> IV, <italic>eak-7(tm3188)</italic> IV, <italic>sdf-9(mg337)</italic> V have been described previously <xref ref-type="bibr" rid="pgen.1004020-Zhang1">[26]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004020-Hu2">[28]</xref>, <xref ref-type="bibr" rid="pgen.1004020-Patel1">[91]</xref>. Double and triple mutants were generated by standard genetic crosses, and the genotypes of strains were confirmed by PCR using oligos which detect gene deletions or RFLP's associated with the mutation (<xref ref-type="supplementary-material" rid="pgen.1004020.s015">Table S6</xref>). Throughout this work <italic>tatn-1</italic> is implied to refer to the <italic>tatn-1(baf1)</italic> allele except specifically as noted otherwise.</p><p>Worm embryos were isolated by sodium hypochlorite treatment, and eggs were transferred to plates, and grown at the indicated temperature in a designated incubator. The plates were scored two or three days later under a dissecting microscope for the presence of L2, dauer, and L3/L4 and older worms. We conducted control experiments to determine the robustness and reproducibility of visual scoring by having several scorers evaluate a series of still images and corresponding movies of larvae of different developmental stages. We then compared the correlation between the scorers for the entire series via the use of a kappa statistic <xref ref-type="bibr" rid="pgen.1004020-Landis1">[92]</xref>. These experiments indicated that scoring was consistent between raters within the lab with all comparisons showing &#8220;substantial&#8221; to &#8220;almost perfect&#8221; agreement (<xref ref-type="supplementary-material" rid="pgen.1004020.s016">Table S7</xref>) <xref ref-type="bibr" rid="pgen.1004020-Landis1">[92]</xref>.</p><p>For each assay, approximately 100 worms were scored from each of two to three plates set up in parallel for each genotype used in an experiment. This resulted in 200 to, more typically, 300 animals being scored for each genotype within an experiment. Each experiment was repeated at least once with comparable results, which resulted in between 400&#8211;600 worms being scored per genotype in total. The percentages of each stage were graphed using Prism5 software, and the graphs show pooled data from a single trial. To perform pairwise comparisons between mutant strains, a contingency table was set up using the counts for L2, dauer, and L3/L4 categories, and p-values were calculated using Fisher's exact contingency test within SAS version 9.3. To determine SDS resistance, worms were washed from plates with 1% SDS, and then incubated for 20 minutes with gentle rocking. Worms were then pelleted and washed with water. Aliquots were scored for survival as demonstrated by movement, and each experiment was repeated at least once. The percentages of living worms were graphed with Prism5.</p><p>Lifespan assays were conducted as previously described at 20&#176;C using either NGA or RNAi plates containing 50 &#181;M FUDR <xref ref-type="bibr" rid="pgen.1004020-Powolny1">[93]</xref>. Lifespan assays for N2, <italic>tatn-1(baf1)</italic>, <italic>eak-7(tm3188)</italic>, and <italic>eak-7(tm3188); tatn-1(baf1)</italic> used NGA plates spotted with HB101, and worms were grown from eggs at 16&#176;C to minimize larval arrest. Lifespan assays for N2, <italic>tatn-1(baf1)</italic>, <italic>aak-2(gt33)</italic>, and <italic>tatn-1(baf1), aak-2(gt33)</italic> used NGA plates spotted with HB101. Lifespan assays for amino acid treated N2 worms used either NGA plates or NGA plates supplemented with 1 mg./mL tyrosine, glycine, or isoleucine, and then spotted with HB101. Lifespan assays using <italic>daf-2</italic> and <italic>daf-16</italic> RNAi treatment used NGA media supplemented with carbenicillin (50 &#181;g/mL) and isopropyl &#946;-d-thiogalactopyranoside (IPTG, 1 mM). RNAi treatment for <italic>daf-16</italic> was started at egg hatching while <italic>daf-2</italic> RNAi treatment started on day 1 of adulthood with larval development occurring on NGA plates spotted with HB101 at 20&#176;C.</p><p>For all lifespan assays three plates containing 40 worms each, for each genotype were set up, as well as an extra plate, with worms to replace worms that had crawled off the plate, bagged, or exploded, to reduce the number of censored events. Prism5 (Graphpad Software) was used to generate graphs and perform log-rank testing for curve comparisons. SAS was used to create lifetables and calculate mean survival.</p><p>Amino acids (Sigma-Aldrich) were dissolved as 45 mg/mL stock solutions in water, and then added to molten NGM to obtain a 1 mg/mL final concentration. These plates were dried and spotted with HB101 before use.</p><p>Worms were grown on HB101 spotted NGM plates for two days at 25&#176;C before being washed from the plates and then being rinsed twice with miliQ water. For <xref ref-type="supplementary-material" rid="pgen.1004020.s009">Figure S9A</xref>, NGM plates either with or without 1 mg./mL tyrosine cast into the agar were used to grow the worm culture. To normalize the samples for worm number, a 5 &#181;L aliquot was removed and scored for worm number. Amino acids were then extracted using aqueous methanol and crushing with a mortar and pestle as previously described <xref ref-type="bibr" rid="pgen.1004020-Geier1">[94]</xref>. The methanol solution was removed by evaporation and the residue stored frozen at &#8722;80&#176;C. Amino acid analysis was performed via liquid chromatography tandem mass spectrometry following reconstitution of the residue in 0.1 mL water as described previously for urine <xref ref-type="bibr" rid="pgen.1004020-Held1">[95]</xref>. Amino acid content was then either normalized to total worm number in the sample (<xref ref-type="fig" rid="pgen-1004020-g009">Figure 9B</xref>, <xref ref-type="fig" rid="pgen-1004020-g009">Figure 9C</xref>, and <xref ref-type="supplementary-material" rid="pgen.1004020.s009">Figure S9A</xref>) or normalized to the levels of individual non-aromatic amino acids and then divided by the average normalized level observed in the wild-type N2 samples (<xref ref-type="fig" rid="pgen-1004020-g009">Figure 9D</xref>).</p><p>Aliquots from a 250 mM AICAR solution dissolved in water (Cell Signaling Technology) were spotted onto NGA plates spotted with HB101 to give a final concentration of 0.125 mM 1 hour before eggs were added to the plates. A comparable volume of water alone was used as a negative control.</p><p>Worm embryos were isolated by sodium hypochlorite treatment from N2 and <italic>eak-4(mg348); tatn-1(baf1)</italic> adults, and the eggs were transferred to NGA plates spotted with HB101. The plates were incubated at 25&#176;C for 24 hours so most of the population was L2 larvae. The worms were washed from plates with water and washed with water to remove bacteria. Nucleotides were then extracted from the worms as previously described <xref ref-type="bibr" rid="pgen.1004020-Apfeld1">[50]</xref>. The resulting extract was stored at &#8722;80&#176;C until analysis. ATP, ADP, and AMP levels were measured by HPLC with UV detection of individual nucleotides.</p><p>Images of worms were obtained with a BX51 fluorescence microscope and quantified using ImageJ software as previously described <xref ref-type="bibr" rid="pgen.1004020-Ferguson1">[44]</xref>.</p><p>For <italic>sod-3</italic> expression, worm embryos were isolated by sodium hypochlorite treatment, and eggs were transferred to NGA plates spotted with HB101 and incubated at 25&#176;C for 24 hours. The worms were washed from plates in water, pelleted by centrifugation, washed with water, and frozen for storage. RNA extraction, reverse transcription, and quantitative PCR were performed as previously described <xref ref-type="bibr" rid="pgen.1004020-Alam1">[27]</xref>, <xref ref-type="bibr" rid="pgen.1004020-Ferguson1">[44]</xref>. The geometric mean level of the control genes <italic>pmp-3, cdc-42</italic>, and <italic>Y45F10D.4</italic> were used to normalize the samples, and the relative levels of <italic>sod-3</italic> expression were determined using the 2<sup>&#8722;&#916;&#916;Ct</sup> approach <xref ref-type="bibr" rid="pgen.1004020-Livak1">[96]</xref>, <xref ref-type="bibr" rid="pgen.1004020-Hoogewijs1">[97]</xref>.</p><p>For <italic>tatn-1</italic> expression, N2 and <italic>daf-2(e1368)</italic> embryos were isolated by sodium hypochlorite treatment, and eggs were transferred to S-basal to arrest the worms at the L1 stage. L1 larvae were added to NGA plates spotted with OP50-1 and the plates were incubated at 20&#176;C for 3 days. Adult worms were washed from plates and RNA was isolated as described above. The geometric mean level of the control genes <italic>pmp-3, cdc-42</italic>, and <italic>Y45F10D.4</italic> were used to normalize the samples, and the relative levels of <italic>tatn-1</italic> expression were determined using the 2<sup>&#8722;&#916;&#916;Ct</sup> approach <xref ref-type="bibr" rid="pgen.1004020-Livak1">[96]</xref>, <xref ref-type="bibr" rid="pgen.1004020-Hoogewijs1">[97]</xref>.</p><p>The oligos used to detect <italic>pmp-3</italic>, <italic>cdc-42</italic>, and <italic>Y45F10D.4</italic> have been previously described <xref ref-type="bibr" rid="pgen.1004020-Hochbaum1">[98]</xref>. The expression of <italic>sod-3</italic> was detected using the oligos <named-content content-type="gene">5&#8242;-CCAACCAGCGCTGAAATTCAATGG-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GGAACCGAAGTCGCGCTTAATAGT-3&#8242;</named-content>
<xref ref-type="bibr" rid="pgen.1004020-Li2">[99]</xref>. The expression of <italic>tatn-1</italic> was detected using <named-content content-type="gene">5&#8242;-CTTGATCAGAGAAGAATCAGTG-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GAGTGTTGATTGAAGTTGCG-3&#8242;</named-content>. These oligos were designed to cross intron-exon boundaries using the PerlPrimer program <xref ref-type="bibr" rid="pgen.1004020-Marshall1">[100]</xref>.</p><p>N2 wild-type control and <italic>tatn-1(qd182)</italic> mutant worms were synchronized via the use of hypochlorite treatment and grown on HB101 spotted NGM plates at 25&#176;C for 2 days. These conditions and time point correspond to the conditions used for the amino acid analysis separately performed using these strains. The worms were washed from the plates and were then washed twice with miliQ water. The worm pellet was then suspended in QIAzol lysis reagent (Qiagen, Valencia, CA) and frozen at &#8722;80&#176;C. Total RNA was isolated using the Qiagen miRNeasy mini kit and the RNA yield was measured by spectrophotometry. Total RNA was sent to Expression Analysis (Durham, NC) for analysis including bioanalyzer electrophoresis to ensure RNA quality followed by library preparation using the Illumina TruSeq RNA sample prep kit. The resulting library was subjected to high-throughput 50 nucleotide paired end sequencing using an Illumina sequencer at a depth of 17 million reads per sample.</p><p>The resulting sequence data was clipped using internally developed software from Expression Analysis and matched to the <italic>C. elegans</italic> genome using RSEM <xref ref-type="bibr" rid="pgen.1004020-Li3">[101]</xref>. The resulting transcript counts were then normalized using the upper quartile normalization approach <xref ref-type="bibr" rid="pgen.1004020-Bullard1">[102]</xref>. Differentially expressed genes were then identified through the use of ANOVA testing and genes with a FDR score of 5% or lower were considered to be differentially expressed. This lead to the identification of 4622 genes as being differentially expressed (890 up-regulated and 3732 down-regulated) between <italic>tatn-1(qd182)</italic> and wild-type N2.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Over-represented gene classes were identified in the up-regulated and down-regulated genes through the use of DAVID <xref ref-type="bibr" rid="pgen.1004020-Huangda1">[60]</xref>. Analysis of the transcriptome data for <italic>daf-16</italic>/FOXO and <italic>crh-1</italic>/CREB regulated genes was performed using the Gene Set Association Analysis for RNA-seq program (available at <ext-link ext-link-type="uri" ns0:href="http://gsaa.unc.edu/login/index.html">http://gsaa.unc.edu/login/index.html</ext-link>) <xref ref-type="bibr" rid="pgen.1004020-Xiong1">[63]</xref>. GSAA calculates a differential expression score for each gene in the entire RNA-seq dataset, 20408 genes in all, and then uses a running weighted Kolmogorov-Smirnov test to examine association of an entire gene set with each phenotypic class. The strength of the association is measured by the association score (AS) where positive scores indicate association of the gene set with the phenotype, and statistical significance is measured by a false discovery rate (FDR) that is adjusted for multiple testing. The <italic>daf-16</italic>/FOXO regulated genes were from previously published microarray data from Murphy et. al., and the <italic>crh-1</italic>/CREB regulated genes were from recently published microarray data from Mair et. al. <xref ref-type="bibr" rid="pgen.1004020-Mair1">[53]</xref>, <xref ref-type="bibr" rid="pgen.1004020-Murphy1">[62]</xref>.</p><p>Clones for the <italic>tatn-1</italic> promoter and cDNA where purchased from Open Biosystems and verified by sequencing <xref ref-type="bibr" rid="pgen.1004020-Lamesch1">[103]</xref>, <xref ref-type="bibr" rid="pgen.1004020-Dupuy1">[104]</xref>. A <italic>tatn-1p:tatn-1 cDNA:GFP</italic> transgene was generated using Gateway cloning and the vector pDEST-MB14 <xref ref-type="bibr" rid="pgen.1004020-Dupuy1">[104]</xref>. The resulting transgene or p<italic>unc-119cbr</italic>, which contains the <italic>unc-119</italic> gene from <italic>Caenorhabditis briggsae</italic>, was used to bombard HT1593 (<italic>unc-119(ed3)</italic>) as previously described <xref ref-type="bibr" rid="pgen.1004020-Hochbaum2">[105]</xref>, <xref ref-type="bibr" rid="pgen.1004020-Ferguson2">[106]</xref>. From bombardment we obtained <italic>bafIs130</italic> from p<italic>unc-119cbr</italic>, which carries the <italic>unc-119</italic> gene from <italic>Caenorhabditis briggsae</italic>, and <italic>bafIs131</italic> from the <italic>tatn-1p:tatn-1 cDNA:GFP</italic> transgene <xref ref-type="bibr" rid="pgen.1004020-Ferguson2">[106]</xref>. Both transgenes were outcrossed with N2 and then mated with <italic>eak-4(mg348); tatn-1(baf1)</italic> to test for rescue of the dauer formation phenotype. The <italic>bafIs131</italic> transgene was also crossed into a <italic>daf-2(e1368)</italic> mutant to generate <italic>daf-2(e1368); bafIs131</italic>.</p><p>Worm embryos were isolated by sodium hypochlorite treatment, and eggs were transferred to NGA plates spotted with HB101 and incubated at 25&#176;C for 24 hours. For treatment with AICAR, the plate was spotted 1 hour before adding the eggs with aliquots from a 250 mM AICAR stock that was dissolved in water. As a positive control, N2 wild-type worms were grown as above, washed off in S-basal, and then exposed to 10 mM sodium azide in S-basal. This dose of S-basal has been previously shown to result in a 50% decrease in ATP concentrations in treated worms <xref ref-type="bibr" rid="pgen.1004020-Lagido1">[54]</xref>. The worms were washed from plates in water, pelleted by centrifugation, washed with water, and suspended in 1&#215; LDS loading buffer (Invitrogen) before being heated to 70&#176;C in a Bransonic sonicator waterbath (Branson) for 30&#8211;40 minutes <xref ref-type="bibr" rid="pgen.1004020-Zanin1">[107]</xref>. After heating the samples were centrifuged and the supernatant retained for analysis. The protein levels were measured using the CB-X protein assay kit (G-Biosciences), and 30 &#181;g of protein was run on a 10% Nupage SDS-PAGE gel (Invitrogen) and blotted to a nitrocellulose membrane. Phospho-AAK-2 was detected using a rabbit monoclonal antibody (Cell Signaling Technology #2535), and actin was detected using a rabbit anti-actin antibody (Cell Signaling Technology #4967) followed by detection by an anti-rabbit HRP secondary antibody and visualization using chemiluminescence (Bio-Rad). The resulting X-ray film was scanned and quantified using gel analysis tools in ImageJ <xref ref-type="bibr" rid="pgen.1004020-AbramoffMDM1">[108]</xref>.</p>