<p>The NZB backcross has been described previously <xref ref-type="bibr" rid="pgen.1002926-Hunter2">[8]</xref>. NZB chromosome 9 subcongenics were generated by breeding the NF9 chromosomal substitution line <xref ref-type="bibr" rid="pgen.1002926-Lancaster1">[9]</xref> to FVB/NJ, backcrossing the F1 progeny to FVB/NJ and screening the N2 backcross progeny for retention of NZB chromosomal segments. Chromosomal segments of interest were made homozygous by breeding to FVB/NJ followed by brother-sister mating of animals containing the segment of interest. Genotyping was performed by using a combination of microsatellite and SNP genotyping.</p><p>The C58 backcross was generated by crossing PyMT male animals with C58/J females to generate PyMT-positive F1 males, which were subsequently bred to FVB/NJ females to generate N2 backcross animals. Tumor phenotyping was performed as previously described <xref ref-type="bibr" rid="pgen.1002926-Lifsted1">[35]</xref>. Genotyping was performed using the Illumina Mouse Medium Density Linkage array at the Center for Inherited Disease Research (CIDR) using spleen DNA isolated by the Wizard DNA purification kit. Quantitative trait mapping (QTL) was performed using J/QTL <xref ref-type="bibr" rid="pgen.1002926-Smith1">[36]</xref>.</p><p>Exons were amplified from genomic DNA then run on a 2% agarose gel. Amplified bands were purified by the Qiagen Gel Extraction Kit and cloned into TOPO-TA vectors (Invitrogen) and transformed into One-shot Top10 chemically competent E. Coli (Invitrogen) as per manufacture's protocol. Plasmid was purified using the Mini Prep kit (Qiagen). Sequencing reactions were conducted on &#8764;350 ng of purified plasmid using Big Dye Terminator reaction mix (Applied Biosystems) following the manufacturers protocol and sequenced by the NCI Sequencing Facility.</p><p>All experiments were conducted using FVB/NJ background mouse mammary tumor explants cell lines 6DT1 and Mvt-1 kindly provided by Dr. Lalage Wakefield <xref ref-type="bibr" rid="pgen.1002926-Pei1">[17]</xref>. Cell lines were cultured in Dulbecco's Modified Eagle Medium (DMEM) (Gibco) supplemented with L-Glutamate (Gibco), 9% fetal bovine serum (FBS) (Gemini BioProducts), and 1% Penicillin and Streptomycin (P/S) (Gemini BioProducts).</p><p>pDonr253 is a Gateway Donor vector modified from pDonr201 (Invitrogen). pDonr253 replaces the kanamycin resistance gene with a gene encoding spectinomycin resistance, and contains several sequencing primer sites to aid in sequence verification of Entry clones. The following oligonucleotides (Eurofins MWG Operon) were used in this study: L10379: <named-content content-type="gene">5&#8242;-GGGGACAACTTTGTACAAAAAAGTTGGCACCATGGCGAGTGCTGTGCTGCCGAGCGGATC</named-content>
</p><p>L10383: <named-content content-type="gene">5&#8242;-GGGGACAACTTTGTACAAGAAAGTTGAGATGAAGTACTCTTTCTTTTCTTC</named-content>
</p><p>The sequences of short hairpin RNA pLKO.1 vectors for <italic>Cadm1</italic> knockdown were</p><p>shRNA14: <named-content content-type="gene">CCGGCGGACTGGTTTGTAAAGGAAACTCGAGTTTCCTTTACAAACCAGTCCGTTTTTG</named-content>
</p><p>shRNA15: <named-content content-type="gene">CCGGCCTGTTCATCAATAACCTAAACTCGAGTTTAGGTTATTGATGAACAGGTTTTTG</named-content>
</p><p>Murine <italic>Cadm1</italic> was cloned into a Gateway Entry clone by PCR from a cDNA template (BC095986, RefSeq NM_018770.3). The Entry clone contains the complete ORF preceded by a Kozak translation initiation sequence and an ATG start codon, and lacking a stop codon at the 3&#8242; end to allow C-terminal fusions to be generated. DNA was amplified using specific primers L10379 and 10383 containing the Gateway recombination sequences, and PCR was carried out using Pfusion polymerase (New England Biolabs). The final PCR product contains the gene of interest flanked on the 5&#8242; side with a Gateway attB1 site and on the 3&#8242; side with a Gateway attB2 site. The PCR product was cleaned using the QiaQuick PCR purification kit (Qiagen), and recombined into pDonr253 using the Gateway BP recombination reaction (Invitrogen) by the manufacturer's protocols. The subsequent Entry clone was sequence verified throughout the entire cloned region.</p><p>A lentiviral vector expressing a C-terminal V5 epitope-tagged fusion of <italic>Cadm1</italic> was generated using Multisite Gateway recombination. An entry clone using the murine Pol2 promoter was recombined with the <italic>Cadm1</italic> entry clone and a C-terminal entry clone encoding the V5 epitope tag (GKPIPNPLLGLDST) into a Gateway destination vector pDest-659. This vector is a modified version of the pFUGW lentiviral vector which contains the enhanced polypurine tract (PPT) and woodchuck regulatory element (WRE) to provide higher titer virus. In addition, it contains an antibiotic resistance gene for blasticidin resistance. Entry clones were subcloned by Gateway Multisite LR recombination using the manufacturer's protocols (Invitrogen). Expression clones were transformed into E. coli STBL3 cells to minimize unwanted LTR repeat recombination, and verified by agarose gel electrophoresis and restriction digest. Transfection-ready DNA for the final clones was prepared using the GenElute XP Maxiprep kit (Sigma). A control vector (8166-M24-658) was generated by standard Gateway LR recombination of a stuffer fragment made up of a non-coding DNA into the pLenti6-V5-DEST vector (Invitrogen). All lentivector constructs and lentivirus particles were generated by the Protein Expression Laboratory and the Viral Technology Group in NCI, Frederick.</p><p>Two milliliter suspensions of 5&#215;10<sup>4</sup> cells were incubated at 37&#176;C in 5% CO<sub>2</sub> overnight. Cells were then infected with 50 uL of concentrated lentivirus suspension, and selected 30 hours post-infection with 5 mg/mL blasticidin for over-expression (Invitrogen) constructs or 10 ug/mL (shRNA) puromycin for shRNA constructs.</p><p>Protein was extracted by cell lysis in 400 uL of Pierce lysis buffer, vigorous homogenization, and incubation on ice for one hour. Twenty micrograms of protein extract per sample in NuPage LDS Sample Buffer and NuPage Reducing Agent (Invitrogen) were used for western blotting. PVDF membrane (Millipore) containing transferred proteins was incubated overnight with the primary antibodies: mouse anti-V5 (Invitrogen), chicken IgY anti-SynCAM/TSLC1/Cadm1 (MBL International), or mouse anti-&#946;-actin (Abcam). The membrane was then incubated with horse-radish peroxidase linked anti-mouse (GE Healthcare) or anti-chicken IgY secondary antibodies (Abcam). Immunoblot was visualized using Amersham ECL Prime Western Blotting Detection System and Amersham Hyperfilm ECL (GE Healthcare). Densitometry data were obtained and analyzed with a ChemiDoc-It Imaging System and VisionWorksLS software (UVP).</p><p>RNA was isolated from tumors and cell lines using RNeasy kit (Qiagen) and reverse transcribed using iScript (Bio-Rad). Real-Time PCR was conducted using QuantiTect SYBR Green PCR kit (Qiagen). See <xref ref-type="supplementary-material" rid="pgen.1002926.s011">Table S2</xref> for primer sequences.</p><p>One-milliliter suspensions of 6&#215;10<sup>5</sup> cells were seeded in triplicate into each well of 24-well Essen ImageLock (Essen Bioscience) plates in selective media and incubated at 37&#176;C and 5% CO<sub>2</sub> overnight. The following day, the cells washed twice with PBS and incubated at 37&#176;C and 5% CO<sub>2</sub> for 3 hours in 10 ug/mL mitomycin C. Scratch wounds were made using Essen 4-channel scratch instrument loaded with Eppendorf 10 uL micropipette tips. Cells were subsequently washed three times with PBS, placed in selective media and placed in Incucyte (Essen Bioscience). Incucyte was programmed to image each well at 2-hour intervals. Data analysis was conducted using Incucyte 2011A software.</p><p>One-milliliter suspensions of 5&#215;10<sup>3</sup> cells were seeded in six replicates into each well of 24-well cell culture plates (Corning, Inc.) in selective media and placed in Incucyte. Incucyte was programmed to image each well at 3-hour intervals. Data analysis was conducted using Incucyte 2011A software.</p><p>Matrigel inserts were hydrated for 2 hours in 500 uL serum-free DMEM then suspensions of 7.5&#215;10<sup>4</sup> cells were seeded in triplicate into each well of the 24-well plate format BD BioCoat Control and Matrigel Invasion Chambers (BD Biosciences) and incubated for 20 hours at 37&#176;C in 5% CO<sub>2</sub>. Enriched media was used as chemoattractant. Membranes were subsequently fixed with methanol, stained with crystal violet, and cells present on the underside of the membrane were counted.</p><p>Cultrex 3-D culture matrix (Trevigen, Inc.) was injected into 8-chamber slides and allowed to solidify at 37&#176;C for 30 minutes. Cells were trypsinized and 5,000 counted then resuspended in 400 uL DMEM, 9%FBS, 5 ug/mLBlasticidin, and 2% Cultrex and incubated at 37&#176;C with 5% CO<sub>2</sub> for 3 days prior to microscopy.</p><p>Female FVB/NJ mice from Jackson Laboratories were injected at 6&#8211;8 weeks of age. Two days prior to orthotopic injections, cells were placed in non-selective media. On the day of injection, 1&#215;10<sup>5</sup> cells were injected orthotopically into the fourth mammary fat pad of age-matched virgin females. After 30 days the mice were euthanized by intraperitoneal injection of 1 mL Tribromoethanol with subsequent cervical dislocation. Primary tumors were resected, weighed, and snap frozen in liquid nitrogen. Lungs were resected, surface metastases were counted; lungs were inflated with 10% nitrate-buffered formalin and sent for sectioning and staining. For tail vein injection, 7&#215;10<sup>5</sup> were injected into the lateral tail vein, mice were euthanized 22 days post-injection. For the CD8-antibody treatment study, orthotopic transplantation was conducted but mice were treated with 0.5 mg of either control rat IgG or rat monoclonal anti-CD8 IgG (Harlan Bioproducts, kindly provided by Dr. Lalage Wakefield) on days &#8722;4, &#8722;3, &#8722;2, +3, +10, +17, +24, and +30; cells were injected on day 0 and mice were euthanized on day 37. All procedures were performed under the Animal Safety Proposal (LCBG-004) and approved by the NCI-Bethesda Animal Care and Use Committee.</p><p>Immunohistochemical staining was performed on LeicaBiosystems' Bond Autostainer on paraffin embedded tissue sections with biotinylated rabbit anti-Goat IgG; primary antibody, anti-V5 rabbit polyclonal antibody (Abcam Catalog #: ab95038); LeicaBiosystems Intense R Detection Kit. All histological analysis including paraffin embedding, sectioning, hematoxylin and eosin staining and immunohistochemistry were conducted by the Pathology/Histotechnology Laboratory, Laboratory Animal Sciences Program, SAIC, Frederick.</p><p>For IFN-&#947; production by T cells in either <italic>Cadm1</italic>-expressing or control tumor bearing mice: Single cell suspensions from the lymph nodes of tumor bearing mice were prepared. 1.6 million cells were loaded into each well of an IFN-&#947; ELISPOT plate, stimulated with anti-CD3 (0.5 ug/ml, eBioscience, San Diego, CA), and cultured overnight. The procedure was done according to recommendations from the manufacture (BD). Three to four mice for each experimental group, and triplicates for each sample were examined. The ELISPOT plate was scanned in ImmunoSpot (Cellular Technology Ltd. Shaker Heights, OH) and quantification was assessed using the CTL Scanning and CTL counting 4.0.</p><p>Statistical analysis comparing two samples were conducted using the Mann-Whitney test on Prism Version 5.03 (GraphPad Software, La Jolla, CA). Multiple-comparison data was analyzed by Kruskal-Wallis test with post-hoc Conover-Inman correction for multiple analyses by R-script. Survival data was conducted with the Mantel-Cox test on Prism.</p>