<p>We used 1,543 samples in total, including 234 samples from <xref ref-type="bibr" rid="pgen.1004254-Decker1">[3]</xref> and 425 samples from <xref ref-type="bibr" rid="pgen.1004254-Gautier1">[4]</xref>, see <xref ref-type="supplementary-material" rid="pgen.1004254.s011">Table S1</xref>. We selected samples that had fewer than 10% missing genotypes, and for breeds with fewer than 20 genotyped samples, we used all available samples which passed the missing genotype data threshold. When pedigree data were absent for a breed, the 20 samples with the highest genotype call rates were selected. For breeds which had pedigree information, we filtered any animals whose sire or dam was also genotyped. For identified half-siblings, we sampled only the sibling with the highest genotype call rate. After removing genotyped animals known to be closely related, we selected the 20 animals with the highest genotype call rate to represent the breed. All DNA samples were collected in an ethical manner under University of Missouri ACUC approved protocol 7505.</p><p>Samples were genotyped with the Illumina BovineSNP50 BeadChip <xref ref-type="bibr" rid="pgen.1004254-Matukumalli1">[56]</xref>. Autosomal SNPs and a single pseudo-autosomal SNP were analyzed, because the data set from Gautier et al. <xref ref-type="bibr" rid="pgen.1004254-Gautier1">[4]</xref> excluded SNPs located exclusively on the X chromosome. We also filtered all SNPs which mapped to &#8220;chromosome unknown&#8221; of the UMD3.1 assembly <xref ref-type="bibr" rid="pgen.1004254-Zimin1">[57]</xref>. In PLINK <xref ref-type="bibr" rid="pgen.1004254-Purcell1">[58]</xref>, <xref ref-type="bibr" rid="pgen.1004254-Purcell2">[59]</xref>, we removed SNPs with greater than 10% missing genotypes and with minor allele frequencies less than 0.0005 (1/[2*Number of Samples]&#8202;=&#8202;0.000324, thus the minor allele had to be observed at least once in our data set). The average total genotype call rate in the remaining individuals was 0.993. Genotype data were deposited at DRYAD (doi:10.5061/dryad.th092) <xref ref-type="bibr" rid="pgen.1004254-Decker2">[60]</xref>.</p><p>The sample genotype covariance matrix was decomposed using SMARTPCA, part of EIGENSOFT 4.2 <xref ref-type="bibr" rid="pgen.1004254-Patterson1">[19]</xref>. To limit the effects of linkage disequilibrium on the estimation of principal components, for each SNP the residual of a regression on the previous two SNPs was input to the principal component analysis (see EIGENSOFT POPGEN README).</p><p>TreeMix <xref ref-type="bibr" rid="pgen.1004254-Pickrell1">[20]</xref> models the genetic drift at genome-wide polymorphisms to infer relationships between populations. It first estimates a dendrogram of the relationships between sampled populations. Next it compares the covariance structure modeled by this dendrogram to the observed covariance between populations. When populations are more closely related than modeled by a bifurcating tree it suggests that there has been admixture in the history of those populations. TreeMix then adds an edge to the phylogeny, now making it a phylogenetic network. The position and direction of these edges are informative; if an edge originates more basally in the phylogenetic network it indicates that this admixture occurred earlier in time or from a more diverged population.</p><p>TreeMix was used to create a maximum likelihood phylogeny of the 134 breeds. Because TreeMix was slow to add migration events (modeled as &#8220;edges&#8221;) to the complete data set of 134 breeds, we also analyzed subsets of the data containing considerably fewer breeds. For these subsets, breeds with fewer than 4 samples were removed. To speed up the analysis, we iteratively used the previous graph with <italic>m</italic>-1 migrations as the starting graph and added one migration edge for a total of <italic>m</italic> migrations. We rooted the graphs with Bali cattle, used blocks of 1000 SNPs, and used the -se option to calculate standard errors of migration proportions. Migration edges were added until 99.8% of the variance in ancestry between populations was explained by the model. We also ensured that the incorporated migration edges were statistically significant. To further evaluate the consistency of migration edges, we ran TreeMix five separate times with -m set to 17.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">ADMIXTURE 1.21 was used to evaluate ancestry proportions for <italic>K</italic> ancestral populations <xref ref-type="bibr" rid="pgen.1004254-Alexander1">[21]</xref>. We ran ADMIXTURE with cross-validation for values of <italic>K</italic> from 1 through 20 to examine patterns of ancestry and admixture in our data set. Map figure was generated in R using rworldmap (<ext-link ext-link-type="uri" ns0:href="http://cran.r-project.org/web/packages/rworldmap/index.html">http://cran.r-project.org/web/packages/rworldmap/index.html</ext-link>). </p><p>The <italic>f<sub>3</sub></italic> and <italic>f<sub>4</sub></italic> statistics are used to detect correlations in allele frequencies that are not compatible with population evolution following a bifurcating tree; these statistics provide support for admixture in the history of the tested populations <xref ref-type="bibr" rid="pgen.1004254-Patterson2">[54]</xref>, <xref ref-type="bibr" rid="pgen.1004254-Reich2">[61]</xref>. The THREEPOP program from TreeMix was used to calculate <italic>f<sub>3</sub></italic> statistics <xref ref-type="bibr" rid="pgen.1004254-Patterson2">[54]</xref> for all possible triplets from the 134 breeds. The FOURPOP program of TreeMix was used to calculate <italic>f<sub>4</sub></italic> statistics for subsets of the breeds.</p>