<p>The strains used in this study, the measurements of spontaneous mutation rates and the sequencing of <italic>URA3</italic> mutants were as previously described <xref ref-type="bibr" rid="pgen.1003016-NickMcElhinny1">[2]</xref>, <xref ref-type="bibr" rid="pgen.1003016-Pursell1">[13]</xref>, <xref ref-type="bibr" rid="pgen.1003016-NickMcElhinny2">[21]</xref>, save that <italic>MSH2</italic> was deleted from haploid <italic>pol2-M644G</italic> strains rather than diploid. The ATT<sub>3</sub> to ATT<sub>0</sub> conversion was made via site-directed mutagenesis and integration pop-out <xref ref-type="bibr" rid="pgen.1003016-Scherer1">[56]</xref> in a strain with wild type polymerases. PCR product containing the ATT<sub>0</sub>
<italic>URA3</italic> allele was then transformed into <italic>msh2&#916;</italic> backgrounds and proper insertion verified via sequencing.</p><p>Genomic DNA was isolated from exponentially growing cultures (grown in YPDA at 30&#176;C) using the Epicentre Yeast DNA purification kit. Five &#181;g of DNA was treated with 0.3 M KOH for 2 h at 55&#176;C and subjected to alkaline-agarose electrophoresis as described <xref ref-type="bibr" rid="pgen.1003016-NickMcElhinny3">[23]</xref>. Following neutralization, DNA was transferred to a charged nylon membrane (Hybond N+) by capillary action and probed by Southern analysis. Strand-specific radiolabeled probes were prepared from a PCR-amplified fragment of <italic>URA3</italic> template, using a previously described procedure and probe design <xref ref-type="bibr" rid="pgen.1003016-Miyabe1">[14]</xref>.</p><p>See <xref ref-type="supplementary-material" rid="pgen.1003016.s006">Text S1</xref>.</p>