<p>Using genomic DNA from AKR/J and DBA/2J mice as templates, PCR was performed to amplify the protein coding region, exons 2 through 24 (<xref ref-type="supplementary-material" rid="pgen.1002735.s005">Table S1</xref>). PCR Products were subjected to agarose gel electrophoresis, bands isolated using the QIAquick Gel Extraction kit (Qiagen) according to manufacturer's recommendations, and used as templates for sequencing. All sequencing runs were performed by the DNA Sequencing and Gene Expression Core, NCI, Bethesda, MD. Genomic sequences for the AKR and DBA alleles were aligned using pairwise BLAST <xref ref-type="bibr" rid="pgen.1002735-Zhang1">[38]</xref> and non-synonymous polymorphisms verified by manual comparison of chromatograms using Chromas software (Technelysium).</p><p>V5-tagged AKR and DBA alleles of <italic>Arid4b</italic> were generated using long range PCR with forward primer <named-content content-type="gene">5&#8242;-AACAAAGGTGCAGGTGAAGC-3&#8242;</named-content> and reverse primer <named-content content-type="gene">5&#8242;-CCTGCACTCAACTGACATTCCATTC-3&#8242;</named-content> to amplify <italic>Arid4b</italic>, and PCR products were cloned into pcDNA3.1/V5-His-TOPO (Invitrogen). FLAG-tagged <italic>Arid4b</italic> vectors were constructed by the Protein Expression Laboratory, SAIC-Frederick, Inc. using Gateway technology (Invitrogen). Briefly, the AKR or DBA allele of <italic>Arid4b</italic> was PCR amplified and cloned into entry vector pDonr-253, then subcloned by Gateway LR recombination into pDest-737 to generate an expression construct with CMV promoter and N-terminal 3xFLAG tag. Full-length BRMS1 was epitope tagged at the N-terminus by PCR with the HA or myc tag sequence incorporated into the forward primer and cloned into pCMV or pcDNA3-hygro (Invitrogen), respectively. Correct sequences of all vectors were confirmed prior to use.</p><p>Met-1 cells <xref ref-type="bibr" rid="pgen.1002735-Borowsky1">[10]</xref> were a gift from Dr. Robert Cardiff (University of California, Davis, CA). 6DT1 cells <xref ref-type="bibr" rid="pgen.1002735-Pei1">[11]</xref> were a gift from Dr. Lalage Wakefield (NCI, NIH, Bethesda, MD). HEK293 cells were purchased from <named-content content-type="gene">ATCC</named-content> (Manassas, VA). Cell lines were maintained in DMEM supplemented with 10% FBS, 2 mM L-glutamine, penicillin and streptomycin. Cells were confirmed to be free of mycoplasma contamination using the MycoAlert detection kit (Lonza).</p><p>Met-1 cells seeded onto 10 cm tissue culture plates were co-transfected with 6 &#181;g of the appropriate V5-tagged <italic>Arid4b</italic> construct described above, or pcDNA3.1/V5-His-TOPO/<italic>lacZ</italic> (Invitrogen) as a control vector, plus 600 ng of pSuper.Retro.Puro (Oligoengine) as a selectable marker, using FuGENE 6 transfection reagent. Cells were selected using 1 mg/ml G418 plus 4 &#181;g/ml puromycin and clones derived by limiting dilution. Stable upregulation of <italic>Arid4b</italic> was verified by performing QRT-PCR using forward primer <named-content content-type="gene">5&#8242;- GGTGAGTGGGAGCTGGTCTA-3&#8242;</named-content> and reverse primer <named-content content-type="gene">5&#8242;- ATAAAGGGCCCACTGAAGGT-3&#8242;</named-content>, and western blotting for endogenous and ectopically expressed ARID4B as described below. 6DT1 cells were transduced with one of five different lentiviral shRNAs targeting <italic>Arid4b</italic> (RMM4534-NM_194262, Open Biosystems) or a scrambled control shRNA in the same pLKO.1 vector. Stable cells were selected using 10 &#181;g/ml puromycin and pooled clones were analyzed for <italic>Arid4b</italic> knockdown by western blot.</p><p>Met-1 or 6DT1 stable lines were orthotopically implanted into the fourth mammary fat pad of six week old female NU/J or FVB/NJ mice using 10<sup>5</sup> cells suspended in 100 &#181;l of PBS per animal. Primary tumors and lungs were harvested 28 days later. All experiments were performed according to the National Cancer Institute Animal Care and Use Committee guidelines.</p><p>Met-1 cells stably expressing <italic>Arid4b</italic> or <italic>lacZ</italic> were seeded at 75,000 cells per well into invasion chambers coated with Matrigel basement membrane matrix (534480, BD Biosciences) or control chambers lacking Matrigel (354578, BD Biosciences). After 24 hours, cells were fixed in 100% methanol, stained with crystal violet, and mounted onto glass slides using mineral oil. Cells were visualized at 400&#215; magnification and five fields were counted for each of three experiments.</p><p>HEK293 cells were transfected with the V5- or FLAG-tagged AKR or DBA allele of <italic>Arid4b</italic>, with or without HA-BRMS1 or myc-BRMS1 where appropriate, using FuGENE 6 transfection reagent. After 30 hours, cells were harvested in mild IP lysis buffer (25 mM Tris-HCl pH 7.4, 150 mM NaCl, 1 mM EDTA, 1% NP-40, 5% glycerol) supplemented with protease inhibitors (11836170001, Roche) and phosphatase inhibitors (P-5726, Sigma). Protein samples were quantitated using Bradford assays. Gammabind G Sepharose beads (17088501, GE Healthcare) were washed twice in NET buffer (50 mM Tris pH 8.0, 150 mM NaCl, 5 mM EDTA, 1% NP-40, 0.5% BSA, 0.04% sodium azide) supplemented with protease and phosphatase inhibitors, and resuspended to form a 50% bead slurry. Lysates were precleared by adding 40 &#181;l of bead slurry and rotating for 30 minutes at 4&#176;C. Samples were centrifuged at 10,000 rpm for 1 minute at 4&#176;C and pre-cleared supernatant transferred to a fresh tube. Anti-V5, anti-FLAG, or anti-myc tag antibodies was added to a final concentration of 1.0 &#181;g/ml and samples rotated for 1 hour at 4&#176;C, then 50 &#181;l of bead slurry was added and co-IPs performed overnight at 4&#176;C. Beads were then washed four times with NET buffer and resuspended in SDS-PAGE sample buffer.</p><p>NuPAGE precast gels and buffers (Invitrogen) were used according to manufacturers recommendations and gels were transferred onto Immobilon-P (Millipore). Membranes were blocked in TBS with 0.5% Tween-20 (TBST) plus 5% nonfat dry milk for 1 hour at room temperature then incubated with the primary antibody diluted in blocking buffer overnight at 4&#176;C. The following primary antibodies and concentrations were used: anti-Arid4b (1&#8758;3,000; A302-233A, Bethyl Laboratories), anti-V5 (1&#8758;5,000; 37&#8211;7500, Invitrogen), anti-mSIN3A (1&#8758;1,000; sc-994, Santa Cruz), anti-mSDS3 (1&#8758;2,000; A300-235A, Bethyl Labs), anti-&#946;-actin (1&#8758;10,000; ab6276, Abcam), anti-FLAG (1&#8758;3,000; F-3165, Sigma), anti-HA (1&#8758;5,000; 11867423001, Roche), anti-myc tag (1&#8758;2,000; 2276, Cell Signaling). After three washes in TBST, membranes were incubated with one of the following horseradish peroxidase conjugated secondary antibodies diluted in TBST plus 0.5% milk for 1 hour at room temperature: anti-mouse (1&#8758;5,000; NA931V, GE Healthcare), anti-rabbit (1&#8758;10,000; sc-2004, Santa Cruz), anti-goat (1&#8758;10,000; sc-2304, Santa Cruz). Membranes were washed an additional three times in TBST and proteins detected using the Amersham ECL Plus system (RPN2132, GE Healthcare) and Amersham Hyperfilm ECL (28906837, GE Healthcare) according to manufacturer's recommendations. Densitometry data were collected and analyzed using a ChemiDoc-It Imaging System and VisionWorksLS software (UVP).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Total RNA was isolated from pooled clones of 6DT1 <italic>Arid4b</italic> knockdown cell lines using RNeasy kits (Qiagen) and then arrayed on Affymetrix GeneChip Mouse Gene 1.0 ST arrays by the Microarray Core in the NCI Laboratory of Molecular Technology. Expression data were normalized using Partek Genomics Suite to identify genes whose expression was significantly different (p&lt;.05) between the <italic>Arid4b</italic> normal cohort (untreated, scrambled control, and H5 lines) and the <italic>Arid4b</italic> knockdown cohort (lines H3 and H4). The gene list and expression values were then analyzed using Ingenuity Pathways Analysis (Ingenuity Systems, <ext-link ext-link-type="uri" ns0:href="http://www.ingenuity.com">www.ingenuity.com</ext-link>) to identify differentially regulated signaling pathways and biological functions. Expression of the <italic>Tpx2</italic> transcriptional network was visualized and figure generated using Cytoscape software <xref ref-type="bibr" rid="pgen.1002735-Shannon1">[39]</xref>. Microarray data are available through the Gene Expression Omnibus under accession number GSE35731.</p>