<p>We performed a genome wide association study (GWAS) meta-analysis of 21,791 individuals (<xref ref-type="supplementary-material" rid="pgen.1002805.s001">Table S1</xref>: 9,390 women, 12,401 men) from ten observational studies. Data from an additional six studies totaling 7,046 individuals (<xref ref-type="supplementary-material" rid="pgen.1002805.s002">Table S2</xref>: 4,509 women; 2,537 men) were used for validation. The proportion of variance explained was estimated in an independent study (InCHIANTI, n&#8202;=&#8202;1,129). The individual study protocols were approved by their respective institution's ethics committee/institutional review board and all participants provided informed consent prior to participation. Individuals known to be taking hormonal contraceptives or hormone replacement therapy at time of SHBG measurement were excluded from analysis. Age, sex and body mass index (BMI) were included as covariates. After applying standard quality control measures, imputed genotypes were available for approximately 2.5 M SNPs. See <xref ref-type="fig" rid="pgen-1002805-g002">Figure 2</xref> for an overview of the analytic plan and the <xref ref-type="supplementary-material" rid="pgen.1002805.s004">Text S1</xref> for further information for individual studies included in this meta-analysis.</p><p>The initial starting point for the conditional analysis was the four <italic>SHBG</italic> locus SNPs that all showed low Hapmap LD (r<sup>2</sup>&lt;0.05) with each other: rs12150660 (lead SNP <xref ref-type="table" rid="pgen-1002805-t001">Table 1</xref>), rs6258 p&#8202;=&#8202;2.7&#215;10<sup>&#8722;46</sup>, rs1625895 p&#8202;=&#8202;1.2&#215;10<sup>&#8722;14</sup> and rs3853894 p&#8202;=&#8202;2.5&#215;10<sup>&#8722;11</sup>. Each cohort fitted a single regression model, fitting SHBG concentrations against these four genome-wide significant SHBG locus SNPs (rs12150660, rs6258, rs1625895 and rs3853894), in addition to age, sex and BMI. After meta-analyzing the results from all cohorts, three of the SNPs retained genome wide significance when regressed against each other, with the fourth SNP narrowly missing that threshold (rs3853894, p&#8202;=&#8202;4.1&#215;10<sup>&#8722;6</sup>).</p><p>We next performed a conditional GWAS meta-analysis, where each study included, as additional covariates to the original analysis plan, the ten genome-wide significant autosomal SNPs (the eight &#8216;Main&#8217; signals from <xref ref-type="table" rid="pgen-1002805-t001">Table 1</xref> and the two unique <italic>SHBG</italic> locus signals described above in addition to the lead SNP rs12150660: rs6258 and rs1625895). Three additional signals (independence based on HapMap r<sup>2</sup>&lt;0.05) at the <italic>SHBG</italic> locus reached genome-wide significance (rs1641537 p&#8202;=&#8202;7.8&#215;10<sup>&#8722;32</sup>, rs6259 p&#8202;=&#8202;1.5&#215;10<sup>&#8722;12</sup> and rs10432029 p&#8202;=&#8202;3&#215;10<sup>&#8722;8</sup>), giving a total of six independent signals in this gene region. In addition, two novel signals reached genome-wide significance in the conditional analysis, at 7q21.3 (rs3779195 p&#8202;=&#8202;1&#215;10<sup>&#8722;8</sup>) and 2p16.3 (rs10454142 p&#8202;=&#8202;3&#215;10<sup>&#8722;8</sup>). After replication, only rs3779195 at the <italic>BAIAP2L1</italic> locus retained genome-wide significance.</p><p>Given the six signals observed at the <italic>SHBG</italic> locus (three through conditional analysis #1 rs12150660, rs6258, rs1625895, three through LD estimates from conditional GWAS #1: rs1641537, rs6259, rs10432029), we sought to confirm which of these six were truly independent by a second round of conditional analysis. All discovery and replication cohorts fitted a single regression model of the six SNPs (SNPs # 1&#8211;6, <xref ref-type="table" rid="pgen-1002805-t002">Table 2</xref>) against SHBG concentrations, using the same parameters and covariates as conditional analysis #1. Four of the six SNPs (#1&#8211;4: rs12150660, rs6258, rs1641537, and rs1625895) retained genome-wide significance when conditioned against each other, with two showing nominal evidence of association (SNP #5 rs6259, p&#8202;=&#8202;0.0001; SNP #6 rs10432029, p&#8202;=&#8202;0.01).</p><p>Finally, we performed a second conditional GWAS analysis, adjusting for the six <italic>SHBG</italic> locus SNPs which had evidence of association from conditional analysis #2. All the discovery cohorts were used in this analysis, in addition to three replication cohorts (total sample size 24,354). This analysis revealed evidence for a further three independent signals at the <italic>SHBG</italic> locus (based on HapMap r<sup>2</sup>&lt;0.01), SNP #7 rs9901675 p&#8202;=&#8202;1.5&#215;10<sup>&#8722;7</sup>, SNP #8 rs8077824 p&#8202;=&#8202;4.6&#215;10<sup>&#8722;5</sup>, and SNP #9 rs9393218 p&#8202;=&#8202;9.9&#215;10<sup>&#8722;6</sup>.</p><p>We performed a sensitivity analysis using samples from the 1966 Northern Finland Birth Cohort (NFBC1966) study to further investigate allelic heterogeneity at the <italic>SHBG</italic> locus (<xref ref-type="supplementary-material" rid="pgen.1002805.s004">Text S1</xref>). The conditional meta-analysis showed evidence for up to nine signals at the <italic>SHBG</italic> locus, but it is possible that these signals could be explaining a much smaller number of causal variants in the region. Since 1000 Genomes imputation allows us to assess the genetic variation associated with a phenotype across a much denser set of markers, it increases our power to detect allelic heterogeneity within a region. Therefore, 1000 Genomes imputation was carried out on all the samples in the NFBC1966 study and forward selection was used to identify the set of SNPs that best explain the variation in the SHBG phenotype. 1000 Genomes imputation was carried out using IMPUTE2. The mean genotype probabilities for each SNP were calculated and used in the model selection step. Only SNPs 250 kb upstream and 250 kb downstream from the <italic>SHBG</italic> locus (7283453&#8211;7786700 bp) were used in the analysis. All SNPs with MAF &lt;0.1% or an imputation quality score less than 0.4 were excluded from the analysis. In total, 1978 <italic>SHBG</italic> region SNPs measured or imputed in 4467 samples from the NFBC1966 study were used in the sensitivity analysis. Forward selection was implemented in R (version 2.13.0) using the stepAIC package to estimate the Akaikie Information Criterion (AIC), an inclusion parameter. Given the high degree of correlation between the SNPs in this region, we increased the penalty (k) on the number of terms included in the model to 12 (where it is usually two), to minimize possible over fitting. The final model included seven SNPs, adjusted for sex and BMI.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We examined potential interactions among the proteins encoded by the <italic>SHBG</italic> locus and the proteins encoded by the 11 genes (<italic>ZBT10</italic>, <italic>TDGF1</italic>, <italic>ZNF652</italic>, <italic>PRMT6</italic>, <italic>JMJD1C</italic>, <italic>GCKR</italic>, <italic>BAIAP2L1</italic>, <italic>LHCGR</italic>, <italic>SLCO1B1</italic>, <italic>UGT2B15</italic>, <italic>NR2F2</italic>) closest to the 11 identified SNPs using pathway analysis with Search Tool for the Retrieval of Interacting Genes/Proteins (STRING) Pathways Analysis (<ext-link ext-link-type="uri" ns0:href="http://www.string-db.org">www.string-db.org</ext-link>). The interactions explored by STRING include direct (physical) and indirect (functional) associations. We then expanded the analysis to examine protein interactions among the <italic>SHBG</italic> gene and the proteins encoded by 67 genes within 500 kb of the 11 identified SNPs.</p><p>We conducted targeted analysis of two strong candidate genes, hepatocyte nuclear factor-4&#945; (<italic>HNF4</italic>&#945;) and peroxisome-proliferating receptor &#947; (<italic>PPAR&#947;</italic>). Statistical significance thresholds were set correcting for the number of SNPs tested in each gene region (&#177;100 kb).</p>