<p>All studies were performed with ethical committee approval, <xref ref-type="supplementary-material" rid="pgen.1004285.s015">Table S7</xref>, and subjects participated in the studies after providing informed consent.</p><p>Cases and controls came from 34 studies forming part of the Breast Cancer Association Consortium (BCAC) included in the COGS Project <xref ref-type="bibr" rid="pgen.1004285-Stacey1">[13]</xref> (<xref ref-type="supplementary-material" rid="pgen.1004285.s009">Table S1</xref>), and GLACIER (A study to investigate the <underline>G</underline>enetics of <underline>L</underline>obul<underline>A</underline>r <underline>C</underline>arcinoma <italic><underline>I</underline>n situ</italic> in <underline>E</underline>u<underline>R</underline>ope MREC 06/Q1702/64), a UK case-only study of lobular breast cancer. BCAC studies recruited all types of breast cancer. Pathological information in BCAC was collected by the studies individually but combined and checked through standardized data control in a central database. A total of 4,152 ILC and 89 LCIS cases were identified by the central BCAC pathology database (see <xref ref-type="supplementary-material" rid="pgen.1004285.s010">Table S2</xref> for number of cases by study).</p><p>The GLACIER study recruited patients from participating centers throughout the UK with the aim of identifying predisposition genes for LCIS and/or ILC. Any patients aged 60 or less at the time of diagnosis, with a current or past history of LCIS (with or without invasive disease of any histological subtype) were eligible. A total of 2,539 cases were recruited: 2,167 were identified from local pathology reports in 97 UK hospitals, 346 cases were identified through the British Breast Cancer Study (BBCS) using UK Cancer Registry data and 26 cases from the Royal Marsden Breast Tissue Bank. Cryptic relatedness analysis showed no evidence of overlap between these samples and the BCAC samples. All these cases were genotyped with the iCOGS chip and compared to 5,000 UK controls selected from four UK studies participating in BCAC and already typed on the iCOGS chip. Controls were randomly selected prior to analysis so that each of these UK studies, including GLACIER, had a case&#8758;control ratio of at least 1&#8758;2 (<xref ref-type="supplementary-material" rid="pgen.1004285.s016">Table S8</xref>). These controls were excluded from case-control comparisons with BCAC cases from the originating study. This report includes only cases of pure LCIS or ILC with or without LCIS. Cases of LCIS with IDC or mixed lobular and ductal carcinoma in GLACIER were excluded in order to perform meta-analyses with the BCAC studies which do not have information on the presence or absence of LCIS associated with an invasive cancer. After excluding individuals based on genotyping quality (see Genotyping and Analysis) and non-European ancestry, data for the GLACIER study available for analyses included 1,782 cases (1,470 ILC (with or without LCIS), 312 pure LCIS) and 4,755 controls.</p><p>A further 516 cases (481 ILC, 35 LCIS) and 1,465 controls were analyzed as part of Phase II. Controls were recruited through the GLACIER study, but were not genotyped in Phase I on the iCOGS chip to reduce costs, and were all white West European. Cases came from the following studies: 232 cases from GLACIER, 176 from BBCS, 71 from DietCompLyf <xref ref-type="bibr" rid="pgen.1004285-Swann1">[38]</xref>, 39 from King's Health Partners Cancer Biobank (KHP-CB). All cases were white West European, apart from the 39 samples from the KHP-CB where there were no associated ethnicity data. For studies that had also participated in Phase I, we selected samples so there was no overlap with the samples in Phase I.</p><p>After DNA extraction from peripheral blood, GLACIER samples were genotyped on the iCOGS custom Illumina iSelect, which contains 211,155 SNPs, at King's College, London. The remaining cases and controls were genotyped as part of the COGS project described in detail elsewhere <xref ref-type="bibr" rid="pgen.1004285-Stacey1">[13]</xref>. The GLACIER cases were analyzed using the same QC criteria as the COGS project. Briefly, genotypes were called using Illumina's proprietary GenCall algorithm and 10,000 SNPs were manually inspected to verify the algorithm calling. Individuals were excluded if genotypically not female, had overall call rate &lt;95% or were ethnic outliers (248 cases) as identified by multi-dimensional scaling, combining the genotyping data with the three Hapmap2 populations. SNPs with a Gencall rate of &lt;0.25, call rate &lt;95% (call rate &lt;99% if MAF &lt;0.1) and HWE&lt;10<sup>&#8722;7</sup> or evidence of poor clustering on inspection of cluster plots were excluded. All SNPs with MAF &lt;0.01 were excluded for this analysis. A cryptic relatedness analysis of the whole dataset was performed using 46,918 uncorrelated SNPs and there was no evidence of any duplicates.</p><p>For GLACIER cases and controls, principal component analysis (PCA) was carried out on a subset of 46,918 uncorrelated SNPs and used to exclude individuals or groups distinct from the main cluster using the first five principal components (PCs), <xref ref-type="supplementary-material" rid="pgen.1004285.s007">Figure S7</xref>. Following removal of outliers (166 cases and 245 controls), the PCA was repeated and the first five PCs included as covariates in the analysis. The adequacy of the case-control matching was evaluated using quantile-quantile plots of test statistics and the inflation factor (<italic>&#955;</italic>) calculated using only uncorrelated SNPs that were not selected by BCAC and were not within one of the four common fine-mapping regions, to minimize selection for SNPs associated with breast cancer, <xref ref-type="supplementary-material" rid="pgen.1004285.s001">Figure S1</xref>. As the majority of the SNPs on the iCOGS array were selected from GWAS of breast, ovarian and prostate cancer the SNPs selected for this analysis were taken from the set of SNPs selected by the prostate consortium, with the assumption that these SNPs were more likely to be representative of common SNPs in terms of population structure in our study than those selected by the breast or ovarian consortia.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For each SNP, we estimated a per-allele log-odds ratio (OR) and standard error by logistic regression, including the 5 PCs as covariates, using PLINK v1.07 (<ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/purcell/plink/">http://pngu.mgh.harvard.edu/purcell/plink/</ext-link>).</p><p>Genotyping and analysis of BCAC studies is described in detail elsewhere <xref ref-type="bibr" rid="pgen.1004285-Michailidou1">[24]</xref>, in brief data were analyzed using the Genotype Library and Utilities (GLU) package to estimate per-allele ORs and standard errors for each SNP using unconditional logistic regression. All analyses were performed in subjects of European ancestry (determined by PC analyses) and adjusted for study and seven principal components.</p><p>Case-control odds ratio (OR) for ILC or LCIS cases vs controls from BCAC and GLACIER were combined using inverse variance-weighted fixed-effects meta-analysis, as implemented in METAL <xref ref-type="bibr" rid="pgen.1004285-Willer1">[39]</xref>. Case-only analyses were also carried out to compare genotype frequencies for ILC vs LCIS (GLACIER and BCAC) and ILC vs IDC (BCAC studies only), and were used as a test for heterogeneity of ORs by tumor subtype. Any study without data on both histological subtypes was dropped from the case-only analysis.</p><p>SNPs showing the strongest evidence for association with lobular tumors (P&lt;5&#215;10<sup>&#8722;5</sup>) in the meta-analysis (after excluding previously reported loci) were genotyped at LGC Genomics (formerly KBiosciences) in Phase II samples. Duplicate samples genotyped on the iCOGS chip were included to assess the concordance of the two genotyping methods. Cluster plots for rs11977670 are shown in <xref ref-type="supplementary-material" rid="pgen.1004285.s008">Figure S8</xref>.</p><p>A pooled analysis of ILC including Phase I (GLACIER and BCAC) and Phase II data was performed. Data were analyzed using STATA v.12 to estimate per-allele ORs and standard errors for each SNP using unconditional logistic regression. Differences in the strength of the associations with ILC, IDC and LCIS were assessed using case-only analyses. A sign test was used to test whether the number of SNPs showing associations in the same direction in two different subtypes (i.e. LCIS vs ILC, and IDC vs ILC) was significantly grater than expected by chance. A likelihood ratio test was used as a global test of the null hypothesis of no differences between subtypes for any of the ORs of the 75 known loci evaluated. Stratum-specific estimates of per-allele OR by categories of age and family history of disease were obtained from logistic regression models and differences in ORs across strata were tested using an interaction term.</p><p>In order to establish the SNP's functional role, a window of 10 kb both up and downstream was formed around the marker and pairwise r<sup>2</sup> values calculated using 1000 genome CEU population data. Three SNPs were identified as being in LD (r<sup>2</sup>&gt;0.5) with rs11977670 and were compared to next generation sequence technologies to elucidate the overlap between chromatin states (ENCODE Project). Two cell lines, normal human mammary epithelial (HMEC), and breast carcinoma (MCF-7), were used to establish these chromatin states, i.e. active or engaged enhancers (H3K27ac), nucleosome-depleted regions (DNase I and FAIRE), and RNA polymerase linked regions (Pol II). Expression data from the Cancer Genome Atlas Network for each gene within a 1 Mb window of rs11977670 was analyzed looking for differential expression in each breast cancer subtype (ER+ ILC, 40 cases; ER+ IDC, 341 cases; and ER-negative IDC, 108 cases). Allele data for surrogate SNP rs13225058 was obtained for all ER+ cases from TCGA. These 335 cases were used to produce genotype specific gene expression data in R. Differences in gene expression between the three genotypes were tested for using one-way-anova, verified by t-test and visually by boxplot. Linear regression was performed across all three genotypes using copy number variation as a co-variate. Level 3 copy number variation data (hg19 build) was obtained from the TCGA data portal.</p>