<p>Our sample set consisted of 111 individuals from 4 different species, namely <italic>H. melpomene</italic>, <italic>H. cydno</italic>, <italic>H. timareta</italic> and <italic>H. heurippa</italic> (<xref ref-type="supplementary-material" rid="pgen.1002752.s003">Table S1</xref>). In total 14 races of <italic>H. melpomene</italic>, 8 races of <italic>H. cydno</italic>, 5 races of <italic>H. timareta</italic> and one of <italic>H. heurippa</italic> were sampled covering most of the geographic distribution of each species from Central to South America (<xref ref-type="fig" rid="pgen-1002752-g001">Figure 1</xref>). <italic>H. numata</italic> was included as outgroup. DNA was extracted using the QIAGEN DNeasy 96 Blood &amp; Tissue Kit. One mitochondrial and eight nuclear fragments (<xref ref-type="supplementary-material" rid="pgen.1002752.s004">Table S2</xref>) were amplified with QIAGEN Taq DNA Polymerase, purified using ExoSAP and sequenced with ABI Big Dye Terminator. Two of the nuclear markers are unlinked to the <italic>HmB</italic> red locus whereas the remaining six are all located across the region (<xref ref-type="supplementary-material" rid="pgen.1002752.s004">Table S2</xref>). From these colour-linked fragments, <italic>optix</italic> and <italic>kinesin</italic> have previously been implicated in red wing pattern determination. The remaining four were identified as regions under divergent selection with high levels of population differentiation associated with red colouration <xref ref-type="bibr" rid="pgen.1002752-Nadeau1">[31]</xref>.</p><p>Sequences were aligned and cleaned using Codon Code Aligner. Haplotype inference for heterozygous calls was conducted with the PHASE algorithm implemented in DNAsp v5.10.01 <xref ref-type="bibr" rid="pgen.1002752-Librado1">[49]</xref>, with 5000 iterations and allowing for recombination. Inferred haplotypes with a confidence higher than 95% were accepted. In the case of the fragments <italic>HmB449k</italic> and <italic>HmB453k</italic> cloning was necessary due to the presence of considerable indel variation. PCR products of these two markers were ligated into the pGEM-T easy vector and five to ten clones per individual were selected and sequenced. Sequences were deposited in GenBank under accession numbers JX003980&#8211;JX005837.</p><p>For each fragment, phased haplotypes were used to construct phylogenetic trees using the Neighbour-Joining method under the P model of uncorrected distance in PAUP* 4.0b10. Node support in the resulting trees was estimated by 1000 bootstrap replicates using a heuristic search. To confirm the phylogenetic groupings obtained by Neighbour-Joining for <italic>HmB453K</italic>, a maximum likelihood phylogeny was also constructed with PhyML <xref ref-type="bibr" rid="pgen.1002752-Guindon1">[50]</xref>, using the GTR+I+G substitution model selected by Modeltest <xref ref-type="bibr" rid="pgen.1002752-Posada1">[51]</xref> and with branch support values obtained by 1000 bootstrap replicates. The stability of the inferred phylogeny for <italic>HmB453k</italic> was examined using the Shimodaira-Hasegawa test (SH test) <xref ref-type="bibr" rid="pgen.1002752-Shimodaira1">[35]</xref> in PAUP* 4.0b10. For all phylogenetic inferences trees were rooted with <italic>H. numata</italic> as outgroup.</p><p>Analysis of molecular variance (AMOVA) with 1000 permutations, implemented in ARLEQUIN v.3.5 <xref ref-type="bibr" rid="pgen.1002752-Excoffier1">[52]</xref>, was used to assess population structure by species, geography and phenotype. For species, four groups were set, corresponding to <italic>H. melpomene</italic>, <italic>H. cydno</italic>, <italic>H. timareta</italic> and <italic>H. heurippa</italic>. In the geography analysis, haplotypes were grouped into six geographic regions: (i) the Guiana shield, (ii) Amazon, (iii) Pacific, (iv) East Andes foothills, (v) Cauca Valley and (vi) Magdalena Valley. These geographic clustering matches the biogeographic provinces (i) Humid Guyana, (ii) Napo+Imeri, (iii) Choco+Wester Ecuador+Arid Ecuador, (iv) North Andean Paramo, (v) Cauca and (vi) Magdalena defined by Morrone <xref ref-type="bibr" rid="pgen.1002752-Morrone1">[53]</xref>. When compared by phenotype, haplotypes were grouped in three groups: the red-banded type [presence of red forewing band], the rayed type [presence of orange rays in the hind wing] and the non-red type [absence of any dorsal red element on the wings]. The outgroup <italic>H. numata</italic> was excluded from these analyses.</p><p>In order to estimate the role and direction of historical gene flow between <italic>H. melpomene</italic> and <italic>H. timareta</italic> (<italic>H. heurippa</italic> was included in <italic>H. timareta</italic> for the purposes of this analysis), we used the Isolation-Migration (IM) Bayesian model <xref ref-type="bibr" rid="pgen.1002752-Hey1">[36]</xref>. IM uses Markov chain Monte Carlo (MCMC) sampling to obtain maximum-likelihood estimates of six parameters: current population sizes, ancestral population size, rates of migration between two populations (<italic>m</italic>
<sub>1</sub> and <italic>m</italic>
<sub>2</sub>) and the timing of divergence (<italic>t</italic>). IM assumes both free recombination between loci and no recombination within them, therefore the software SITES <xref ref-type="bibr" rid="pgen.1002752-Hey2">[54]</xref> was used to select genetic blocks with no recombination within each locus. To fulfill the assumption of free recombination between loci, only the unlinked colour loci and one of the fragments linked to red colouration [<italic>HmB453k</italic>] were selected for this analysis. We ran IM on two modified datasets for each species pair: (i) <italic>H. melpomene</italic> rayed type<italic>-H. timareta</italic> and (ii) <italic>H. melpomene</italic> red-banded -[<italic>H. heurippa</italic> and <italic>H. timareta</italic>]. These groups constituted the maximal units in which we could get enough data without recombination, with the rayed dataset being a block of 379 bp and the postman dataset one of 313 bp. Unfortunately, pairwise comparisons involving all the species' alleles were not possible nor were comparisons involving species in parapatry because such groupings contained small non-recombining blocks that lacked enough informative sites. However, since our main interest was to determine the direction and magnitude of introgression (<italic>m</italic>) within phenotypes, these datasets are sufficient for addressing this question. For all datasets, after searching for the parameter range using preliminary runs, 30 million steps were sampled from the primary chain after a 300,000 burn-in period under the HKY model with 10 chains per set. Mixing properties of the MCMC were assessed by visual inspection of the parameter trend plots and by examining that the effective sample size (ESS) was higher than 50, as recommended <xref ref-type="bibr" rid="pgen.1002752-Hey1">[36]</xref>, <xref ref-type="bibr" rid="pgen.1002752-Strasburg1">[37]</xref>.</p><p>To get biologically meaningful units of gene flow, the maximum likelihood estimates and 90% highest posterior density (HPD) interval for the migration rates (m) were converted into the effective number of gene migrations received by a population per generation (2Nm, in <xref ref-type="table" rid="pgen-1002752-t002">Table 2</xref>). For this conversion, we used a generation time of 35 days and a mutation rate per gene calculated with the calibration time proposed by Wahlberg et al. for Nymphalidae <xref ref-type="bibr" rid="pgen.1002752-Wahlberg1">[55]</xref> coupled with the divergence between the melpomene/cydno clade per locus estimated with the software SITES. Our estimates of mutation rate per locus per year were: 6.2&#215;10<sup>&#8722;6</sup> for COI, 7.1&#215;10<sup>&#8722;8</sup> for GAPDH, 1.2&#215;10<sup>&#8722;6</sup> for Hsp90 and 3.4&#215;10<sup>&#8722;5</sup> for HmB453k.</p><p>We finally compared the model including all six parameters to simpler demographic models in order to statistically test the hypothesis of zero or equal gene flow between populations (m1&#8202;=&#8202;0, m2&#8202;=&#8202;0, m1&#8202;=&#8202;m2&#8202;=&#8202;0, m1&#8202;=&#8202;m2&gt;0). These analyses were conducted using IMa <xref ref-type="bibr" rid="pgen.1002752-Hey1">[36]</xref> by running the initial M-mode output with identical settings in the L-mode and sampling 5&#215;10<sup>5</sup> genealogies.</p><p>We further tested the presence, significance and direction of gene flow per locus between <italic>H. melpomene</italic> and <italic>H. timareta</italic> using a method based on linkage disequilibrium (LD) developed by Machado <italic>et al</italic> in 2002 <xref ref-type="bibr" rid="pgen.1002752-Machado1">[38]</xref>. In this test, a positive difference between the LD among all pairs of shared polymorphisms (D<sub>SS</sub>) and the LD among all pairs of sites for which one member is a shared polymorphism and the other is an exclusive polymorphism (D<sub>SX</sub>), is indicative of gene flow. The magnitude of the difference directly measures the direction of the introgression, with the species with the highest positive value being the recipient <xref ref-type="bibr" rid="pgen.1002752-Machado1">[38]</xref>. The same phenotypic datasets used in the IM analysis and also groups of species in complete sympatry, were subjected to independent runs, each of them with 30000 simulations. The input files were prepared with the program SITES <xref ref-type="bibr" rid="pgen.1002752-Hey3">[56]</xref> calculating D' as a measure of linkage disequilibrium (as suggested by Machado et. al <xref ref-type="bibr" rid="pgen.1002752-Machado1">[38]</xref>) and analysing linkage disequilibrium among shared polymorphism between groups (by choosing the -s and -p options in the LD string).</p><p>Linkage disequilibrium across the <italic>HmB</italic> region was calculated for all populations using the software MIDAS <xref ref-type="bibr" rid="pgen.1002752-Gaunt1">[57]</xref> only considering sites with allele frequency higher than 5%, and visualised with the R package LDheatmap <xref ref-type="bibr" rid="pgen.1002752-Shin1">[58]</xref>.</p>