<p>MCF-7, MDA-MB-231, and 293T cells were obtained from the American Type Culture Collection and cultured at 37&#176;C, 5% CO2 in Dulbecco's Modified Eagle Medium/Nutrient Mixture F-12 (DMEM/F-12) Media containing 10% fetal bovine serum supplemented with penicillin-streptomycin.</p><p>DamID lentiviral vectors, pLgw-RFC1-V5-EcoDam and pLgw-V5-EcoDam were kindly provided by Dr. Bas van Steensel, Netherlands Cancer Institute. Human MBD3 cDNA (BC043619) was amplified and cloned into pENTR/D-TOPO and then recombined by an LR-reaction into destination vector pLgw-RFC1-V5-EcoDam (pLgw-MBD3-V5-EcoDam). Retroviral knockdown constructs, pSMP-Luc (Addgene plasmid 36394) and pSMP-MBD3_3 (Addgene plasmid 36373) were obtained from Addgene. All constructs were verified by DNA sequencing. Lentivirus production and infection were performed as previously described <xref ref-type="bibr" rid="pgen.1004028-Lai1">[40]</xref>, using pLgw-MBD3-V5-EcoDam (MBD3-Dam) or pLgw-V5-EcoDam (Dam-only). Retrovirus production and infection were performed as described <xref ref-type="bibr" rid="pgen.1004028-Onder1">[41]</xref>, using pSMP-Luc or pSMP-MBD3_3.</p><p>The DamID experiments were carried out as previously described <xref ref-type="bibr" rid="pgen.1004028-Greil1">[42]</xref>. Briefly, MCF-7 or MDA &#8722;231 cells were seeded into 6-well plates. Seventy-two hrs after infection, genomic DNA was isolated using Qiagen DNeasy tissue kit. Genomic DNA (2.5 &#181;g) was digested with Dpn I followed by adaptor ligation. The ligated product was digested with Dpn II and amplified by PCR. One microgram amplified product was labeled using Dual-color DNA Labeling kit (Nimblegen) according to manufacturer's protocol and then hybridized to Nimblegen 2.1M Deluxe promoter array or human 2.1 M Whole-Genome Tiling array (Array 5 of 10 covering chromosomes 6&#8211;8) and washed following the manufacturer's directions. The slides were scanned using a DNA microarray scanner (G2565BA; Agilent Technology) and the images were processed with the Nimblegen software.</p><p>A two-step normalization approach was used, where the first step is designed to correct for GC bias and dye bias within a chip (intrachip correction) and the second step corrects for variations across chips (interchip correction). The first step was within-chip normalization. First, all probes were binned according to their GC content. The GC content was computed as a ratio of C and G nucleotides to the total number of nucleotides in the probe sequence. The overall variability in GC content values was used to compute bin width according to zero-stage rule <xref ref-type="bibr" rid="pgen.1004028-Wand1">[43]</xref>, <xref ref-type="bibr" rid="pgen.1004028-Bieda1">[44]</xref>. These bin widths are proven to be approximate L2 optimal; i.e., they minimize mean integrated square error. The bins with fewer probes were then merged so that each bin contains at least 500 probes. Within each bin, Lowess regression was used to predict log-transformed cy5 values as a smooth function of log-transformed cy3 values <xref ref-type="bibr" rid="pgen.1004028-Hastie1">[45]</xref>, <xref ref-type="bibr" rid="pgen.1004028-Scacheri1">[46]</xref>. The scaled (median of absolute residuals is used for scaling) difference between observed and predicted log(cy5) values were used as normalized signal.</p><p>The second step was between-chip normalization. Once the data were corrected for dye and GC bias as described in the first step, we employed quantile normalization independently for each histone mark and DNA methylation to correct for between-chip variation.</p><p>The differentially bound &#8220;peak&#8221; regions for each cell type comparisons were identified using modified ACME algorithm that allows for spooling data across replicates <xref ref-type="bibr" rid="pgen.1004028-Scacheri1">[46]</xref>. This algorithm like ACME, depends on three user-specified tuning parameters: window size (<italic>w</italic>), signal threshold (<italic>s</italic>) and p-value threshold (<italic>p</italic>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To identify MBD3 bound peaks, we first compute the number (<italic>x</italic>) of signal values within window of size <italic>w</italic> (centered at probe) that are greater than 100<italic>s</italic>
<sup>th</sup> percentile across all replicates. Next, we compute enrichment p-value for probe using hypergeometric distribution as following<disp-formula id="pgen.1004028.e001"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004028.e001.jpg" /></disp-formula>where <italic>N</italic> denotes total number of probes, <italic>k</italic> denotes number of probes in window and <italic>r</italic> denotes the number of replicates.</p><p>Finally, the peaks are identified as runs of enrichment p-values that are less than p-value threshold (<italic>p</italic>). The analysis presented here correspond to signal threshold (<italic>s</italic>&#8202;=&#8202;0.95), window size (<italic>w</italic>&#8202;=&#8202;2000) and p-value threshold (<italic>p</italic>&#8202;=&#8202;0.001) with peaks containing less than six probes excluded.</p><p>For each set of differentially bound and cell-specific gene signatures we performed Fisher's exact test to assess enrichment of gene-sets from Molecular Signature Database (MSigDB, version3.0, Broad Institute) and other published Cancer gene sets. The resulting significance p-values were subjected to Benjamin-Hochberg (FDR) multiple test correction.</p><p>We employed chromosome bound circular permutations test described in <xref ref-type="bibr" rid="pgen.1004028-Scacheri1">[46]</xref> to assess whether the observed overlap between two sets of genomic intervals (e.g. peaks) is significantly enriched or depleted compared to expected overlap. To perform overlap analysis, we used disjoined probes from the promoter-array as a unit of overlap. The &#8216;member status&#8217; (ie. whether the disjoined probe belongs to a genomic region of interest or not) of disjoined promoter probes in one of genomic interval set was permuted 20,000 times using chromosome bound circular permutation as following. For each permutation a randomly generated number drawn from uniform distribution between 1 and number of disjoined probes from the analyzed chromosome was used to shift the membership status of all disjoined probes within a chromosome. The permutations resumed at the beginning of chromosome when the shift of status exceeded number of disjoined probes available in a chromosome. The odds-ratio of enrichment/depletion, corresponding significance p-value and 95% confidence interval were calculated by comparing observed overlaps and average overlap across all 20,000 permutations.</p><p>The above approach was utilized to assess enrichment/depletion of overlap in significant DamID peaks (modified ACME peaks with at least 6 promoter probes) and MBD3 ChIP-seq peaks (peaks with fold change&gt;&#8202;=&#8202;2 and false discovery rate&lt;&#8202;=&#8202;0.00001).</p><p>We also utilized the above approach to assess enrichment/depletion of overlap in significant cell specific DamID peaks (uncommon peaks in MCF-7 and MDA-231 cells) with promoter regions (3 kb upstream and 3 kb downstream of TSS) of Luminal vs. Basal breast cancer differentially expressed genes from Charafe et al <xref ref-type="bibr" rid="pgen.1004028-CharafeJauffret1">[47]</xref>.</p><p>ChIP experiments for MBD3 were performed as previously described with several modifications <xref ref-type="bibr" rid="pgen.1004028-Fujita1">[17]</xref>, <xref ref-type="bibr" rid="pgen.1004028-Fujita2">[18]</xref>. MCF-7 or MDA-231 cells were crosslinked with 1.5 mM disuccinimidyl glutarate (Thermo Scientific) for 45 min at room temperature followed by 1% formaldehyde (Sigma) for 10 min at room temperature. Fixation was quenched by 125 mM glycine. Cells were washed twice with ice-cold PBS and stored at &#8722;80&#176;C. Cells were thawed on ice and resuspended with 270 &#181;L lysis buffer [50 mM Tris-HCl (pH 8.0), 10 mM EDTA, 1% SDS and protease inhibitor]. After incubation on ice for 5 min, cells were sonicated with Bioruptor for 6 cycles of 30 sec on and off at HIGH setting. Samples were centrifuged at 16.1 kg for 5 min at 4&#176;C. Supernatant was collected into new tube and pellet was resuspended with 150 &#181;L lysis buffer. The resuspended pellets were sonicated with Bioruptor for 4 cycles of 30 sec on and off at HIGH setting. Samples were centrifuged at 16.1 kg for 5 min at 4&#176;C and supernatant were combined. 100 &#181;L supernatant was diluted 10 times with IP buffer [16.7 mM Tris-HCl (pH 8.0), 1.2 mM EDTA, 0.01% SDS, 1.1% TritonX-100, 167 mM NaCl, 5 mg/mL BSA and protease inhibitor] and incubated with 4 &#181;g of control rabbit IgG (Santa cruz) or anti-MBD3 antibody (Abcam; ab91458) at 4&#176;C overnight. Samples were further incubated with 20 &#181;L of Dynabeads Protein A and G at 4&#176;C for 1.5 hr. The beads were washed once with IP buffer, twice with RIPA buffer [25 mM Tris-HCl (pH 7.6), 150 mM NaCl, 1% NP-40, 1% sodium deoxycholate, 0.1% SDS], twice with high salt RIPA buffer (1&#8758;9 dilution of 5M NaCl with RIPA buffer), twice with LiCl buffer [20 mM Tris-HCl (pH 8.0), 250 mM LiCl, 1% NP-40, 1% sodium deoxycholate, 2 mM EDTA] and twice with TE buffer [10 mM Tris-HCl (pH 8.0), 0.1 mM EDTA]. Washed beads were resuspended with elution buffer (1% SDS and 0.1 M sodium bicarbonate) and incubated at 65&#176;C for 45 min. Eluted DNA was adjusted to 300 mM NaCl and incubated with RNase at 65&#176;C for 4 hr followed by incubation at 65&#176;C for 1 hr with proteinase K. DNA was purified with a MinElute PCR purification kit (QIAGEN). Sequencing libraries were prepared with Truseq DNA kit or Nextera XT kit (following the manufacturer's protocols).</p><p>MBD3 ChIP-seq and Input libraries were sequenced using Illumina GAIIx technologies at the NIH Intramural Sequencing Center (MCF-7 MBD3 ChIP, MCF-7 Input and MDA-231 Input) or using MiSeq technologies at the NIEHS Epigenomics Core Facility (MDA-231 MBD3 ChIP and replicate Input). Standard Illumina CASAVA 1.8 utilities were used to generate .fastq output files. All libraries were sequenced as single end 36mers.</p><p>ChIP experiments for H3K4me3 in MDA-231 cells were carried out using a Magna ChIP kit (Millipore) following the manufacturer's suggested protocols. For each ChIP experiment, 5&#215;10<sup>6</sup> MDA &#8722;231 cells were crosslinked with 1% formaldehyde at 37&#176;C for 10 min. The anti-H3K4me3 (Upstate, no. 04-745) antibody was used. The ChIP-seq libraries were prepared from 10 ng of both Input and ChIP DNA samples using a ChIP-seq sample preparation kit (Illumina) according to the manufacturer's protocol.</p><p>MDA-231 H3K4me3 ChIP-seq and Input libraries were sequenced using Illumina GAII technologies at the University of Missouri DNA Core Facility. For these libraries Illumina CASAVA 1.3 utilities were used to generate .fastq output files. All libraries were sequenced as single end 36mers.</p><p>Public MCF-7 data used for analyses included: H3K4me3 (GSM945269), H3K9me3 (GSM945857), H3K27ac (GSM945854), H3K27me3 (GSM970218), Reduced Representation Bisulfite (GSM683787), and triplicate expression data (GSM425734, GSM425735, GSM425736) and (GSM425737, GSM425738, GSM425739).</p><p>Sequenced reads from MDB3 ChIP-seq and Input libraries were combined for replicate samples and filtered based on a mean base quality score &lt;20. Filtered reads were then aligned to the human reference genome (UCSC assembly hg19, GRCh37) using the Bowtie short-read alignment program (v0.12.8 employing parameters &#8722;v 2, &#8722;m 1) to retain reads mapped to unique genomic locations with at most 2 mismatches. Only non-duplicate reads were used in subsequent peak calling analyses and the generation of coverage tracks. To make the coverage tracks, aligned reads were extended at the 3&#8242; end to a length of 300 bases (the expected genomic fragment size), and bigWig files were generated to visualize aggregate genomic coverageMBD3 peaks for each cell type were identified using SICER with a FDR threshold of 0.001 and the following parameters (redundancy threshold&#8202;=&#8202;1, window size&#8202;=&#8202;200, gap size&#8202;=&#8202;600, fragment size&#8202;=&#8202;300). Basic quality control of MBD3 ChIP-seq data (ours and others) is provided in <xref ref-type="supplementary-material" rid="pgen.1004028.s011">Table S5</xref>.</p><p>hg19 RefSeq transcript model definitions were downloaded from UCSC. Transcripts covering duplicated genomic loci [have exactly same transcription start site (TSS) and end site (TES)] were excluded yielding 31,723 RefSeq transcripts. ChIP-seq peaks were defined as associated with genomic features if they intersect at least 1 bp. Genomic features queried included exons, introns, TSS, TES, promoters (+/&#8722; 3 kb from TSS), gene bodies (TSS to TES), and intergenic regions (not in gene body).</p><p>From the set of 31,723 RefSeq transcripts, we selected full-length gene bodies (i.e. from the transcript start to the 3&#8242; transcript end) larger than 2 kb.</p><p>Read counts were averaged in 70 bp windows for regions +/&#8722; 5 kb from TSS. Within the gene bodies, reads were averaged in windows equal to 1% of the gene length. All window read counts were normalized by the total number of bases in each window and the length of the window.</p><p>The CpG contents and the ratio of observed versus expected CpG dinucleotides were determined in a 1,001 bp window around TSS (+/&#8722; 500 bp) as described <xref ref-type="bibr" rid="pgen.1004028-Weber2">[48]</xref>. Promoter categories were then classified into 3 categories as follows: HCPs (high-CpG promoters) contain a 500 bp area with CpG ratio above 0.75 and GC content above 55%; LCPs (low-CpG promoters) do not contain a 500 bp area with a CpG ratio above 0.48; and ICPs (intermediate CpG promoters) are neither HCPs nor LCPs.</p><p>Heatmaps represent transcript expression, binding profiles of MBD3 and H3K4me3 in MCF-7 and MDA-231 cells at promoter regions (&#8722;7 to +3 kb relative to TSS) and are sorted by MBD3 binding profiles. For DamID data, we iteratively filtered genes as followed: 1) Downloaded hg18/refseq gene table from UCSC browser site (with date stamp October 20, 2009). 2) 31,451 genes from chr1-22 and chrX and chrY were retained. 3) 24,983 genes with non-missing gene expression signal were retained. 4) 15,292 genes with at least one microarray probe for each TSS bin were retained. 5) 11,872 genes with unique TSS were retained. 6) 8,207 genes with no 7 kb upstream and no 3 kb downstream neighbors were used in heatmap. For ChIP-seq data, we filtered genes as followed: 1) Transcripts with duplicated genome locus were eliminated. 2) Transcripts were selected with a MAD (Mean Absolute Deviation) higher than 1.0, to further eliminate transcripts with low variability around promoter regions in MBD3-binding profiles; 25,569 transcripts (MCF-7) and 23,605 transcripts (MDA-231) were retained and used in the heatmap.</p><p>Raw sequencing datasets for MBD3 and histone modifications H3K4me3, H3K27me3, H3K27ac, and H3K9me3 from ENCODE were filtered, mapped, deduplicated, and extended (to 300 bp) as described above. RPKM thresholds for establishing presence or absence of each mark were determined following the method described by Whyte et al for detecting super-enhancers <xref ref-type="bibr" rid="pgen.1004028-Whyte2">[49]</xref>, but evaluating over all non-overlapping 1 kb genomic bins rather than called peaks. If one or more 1 kb bin (evaluated with step size 100) in the region +/&#8722; 3 kb of an annotated RefSeq TSS had RPKM above the established threshold, presence of MBD3 or histone mark was assigned for that TSS. Comparison of MBD3 signal with H3K4me3, H3K27ac, H3K9me3, or H3K27me3 at all TSS is shown by Venn diagram.</p><p>Called MBD3-binding peaks in MCF-7 cells were separated according to whether they overlap with a RefSeq promoter region (TSS +/&#8722; 3 kb). Then a region with 3 kb flanking the center of peaks was extracted accordingly. Reads from MBD3, H3K4me3 and H3K27ac profiles were averaged in 60 bp windows for the above region and normalized by the total mapped reads.</p><p>CpG islands annotation was downloaded from UCSC hg19 genome, and CpG islands were separated according to whether they overlap at least 1 bp with the MBD3-binding peaks. Reduced Representation Bisulfite sequencing data in MCF-7 cells (GSM683787) was downloaded and the methylation levels of the CpG islands associated with MBD3-binding peaks were calculated as the follows: percentage of methylation&#8202;=&#8202;total sequenced reads being methylated/total reads being sequenced.</p><p>The interaction dataset, part of the Pol II Chromatin Interaction Analysis with Paired-End Tag data (Pol II ChIA-PET), GSM970209, were downloaded from GEO database. Then the data were filtered to keep the PET pair ends that have at least one end overlapped with the TSS loci (+/&#8722; 3 kb). Among the retained PET pair ends, each end in the PET pair ends was examined as follows: 1) Assign the nearest hg19 Refseq gene if this end is overlapped with TSS loci (+/&#8722; 3 kb); 2) Assess if this end was overlapped with the MBD3 peaks; 3) Count the reads from MBD3 ChIP-seq data within the region that is 500 bp flanking the center of this end. The reads were normalized to the total mapped reads.</p><p>The data are displayed as follows: 1) Sort the read number at TSS end of the filtered ChIA-PET pair ends in a descending order; 2) Plot the sorted read number in upper panel: y-axis is the sorted read number and x-axis is the sorted PET pairs; 3) Plot the read number in lower panel: y-axis is the corresponding read number in the other end and x-axis is the sorted PET ends in the same order as in the upper panel.</p><p>3C experiment was performed as previously described <xref ref-type="bibr" rid="pgen.1004028-Dekker1">[25]</xref>. Briefly, MCF-7 cells were crosslinked by 1% formaldehyde for 10 min at room temperature and fixation was quenched by 125 mM glycine. Cells were resuspended with lysis buffer (10 mM Tris-HCl, 10 mM NaCl, 0.2% NP40) and incubated on ice for 15 min. The pellet was resuspended with 1.2&#215;NEB2 buffer and incubated at 65&#176;C for 15 min and followed by digestion with EcoRI overnight. Digested chromatin was ligated with T4 DNA ligase (NEB) at 16&#176;C for 2 hr. Purified 3C template (200 ng) was amplified with iQ SYBR mix (BioRad).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Mononucleosomal DNA were prepared from MCF-7 cells (Luc or MBD3 knockdown) as previously described with some modifications <xref ref-type="bibr" rid="pgen.1004028-Li1">[50]</xref>. Briefly, nuclei were collected from 5&#215;10<sup>5</sup> cells and digested by MNase (1.25, 2.5, or 5 units) for 5 min at 37 &#176;C. Mononucleosomal DNA were collected by gel size selection and pooled. Sequencing libraries were prepared from 1 &#181;g of pooled DNA with TruSeq DNA Sample Preparation Kit. The resulting libraries were sequenced on a HiSeq 2000 (Illumina) as paired end 101mers. To ensure that low quality reads were excluded from the analysis, the raw sequence reads were filtered to remove any entries for which either read in a pair had mean base quality score &lt;20. Filtered reads were aligned to the human genome (GRCh37/hg19) via Bowtie (v0.12.8 with parameters -m 1 &#8211;best &#8211;strata &#8211;chunkmbs 1024 -I 0 -X 1000) <xref ref-type="bibr" rid="pgen.1004028-Langmead1">[51]</xref>. Multiple sequencing lanes from the same MNase library were merged prior to downstream analysis, and duplicate reads per library were removed using MarkDuplicates.jar from the Picard tools package (v1.86) (<ext-link ext-link-type="uri" ns0:href="http://picard.sourceforge.net">http://picard.sourceforge.net</ext-link>). To capture only sequenced reads corresponding to the size of a mononucleosome, filtering was applied to retain only fragments 120&#8211;180 bases in length; 90&#8211;95% of mapped, deduplicated read pairs per library passed this filter. Each mapped read pair was then converted to a single point at the midpoint of the fragment. The two replicate libraries per condition were merged, and &#8216;bedGraph&#8217; files were generated (via BEDtools, version 2.17.0 <xref ref-type="bibr" rid="pgen.1004028-Quinlan1">[52]</xref>) for visualization of genomic distribution of mapped fragments.</p><p>Metagene plots of nucleosome occupancy were generated with mapped MNase-seq fragments downsampled to the same read count per condition (N&#8202;=&#8202;156,822,505). Counts per position were aligned relative to each annotated RefSeq TSS, with the RefSeq loci split into quartiles based on the number of MBD3 ChIP-seq reads mapped to the region +/&#8722; 3 kb of the given TSS. The positional counts in each dataset (shMBD3 or shLuc per quartile) were normalized by the mean score over the region +/&#8722; 5 kb of TSS, then smoothed with a moving average (period&#8202;=&#8202;50).</p><p>The microarray data and sequence data from this study are available at the NCBI Gene Expression Omnibus (GEO) under accession number, GSE44737, GSE 44776, and GSE51097.</p>