<p>
<italic>Per3<sup>&#8722;/&#8722;</sup></italic> mice were obtained from David Weaver at the University of Massachusetts. Knockout mice were bred with <italic>mPer2<sup>Luc</sup></italic> reporter mice to obtain homozygous knockouts harboring the <italic>mPer2<sup>Luc</sup></italic> reporter. Wheel-running assays were performed and analyzed as described previously <xref ref-type="bibr" rid="pgen.1004244-Yoo1">[8]</xref>. All animal studies were conducted in accordance with the regulations of the Committees on Animal Care and Use at University of Memphis and UCSD.</p><p>All cell culture media were from HyClone. 3T3 (also known as NIH 3T3) and 3T3-L1 cells were cultured in regular medium in which DMEM was supplemented with 10% FBS and 1&#215; penicillin-streptomycin-glutamine (PSG). For 3T3-L1 differentiation, pre-adipocytes were first grown to confluence (Day 0). On Day 2, cells were fed with induction medium (regular medium with 1 &#181;M dexamethasone, 0.5 mM isobutylmethylxanthine, and 2 &#181;g/ml insulin). On Day 4, cells were changed to maintenance medium (regular medium containing 2 &#181;g/ml insulin). From day 6 onward, cells were grown in regular medium until use. For bioluminescence recording, 3T3 and 3T3-L1 cells were grown in 25 mM HEPES-buffered regular medium (pH 7.4) containing 1 nM forskolin and 1 mM luciferin. Fully differentiated 3T3-L1 cells were used in all experiments.</p><p>MMH-D3 cells were grown in regular medium in which RPMI medium was supplemented with 10% FBS, 1&#215; PSG, 10 &#181;g/ml insulin, 55 ng/ml epidermal growth factor (EGF), and 16 ng/ml insulin like growth factor-II (IGF-II). For differentiation, pre-hepatocytes were first grown to 100% confluence. Two days later, cells were replaced with differentiation medium (regular medium with 2% DMSO). Medium change was repeated every 48 hours for 6&#8211;8 days for cells to be fully differentiated for use. Circadian rhythms of differentiated cells were synchronized with 200 nM dexamethasone followed by bioluminescence recording in 25 mM HEPES-buffered serum-free explant medium (pH 7.4) containing B-27 and 1 mM luciferin, as we have done previously <xref ref-type="bibr" rid="pgen.1004244-Liu3">[43]</xref>. Fully differentiated MMH-D3 cells were used in all experiments.</p><p>SCN explants and dissociated neuronal cells were prepared and cultured as previously described <xref ref-type="bibr" rid="pgen.1004244-Liu3">[43]</xref>. Bioluminescence recording of explants, single cell-imaging of individual SCN neurons, and respective data analysis were performed as previously described <xref ref-type="bibr" rid="pgen.1004244-Liu3">[43]</xref>.</p><p>Lentiviral luciferase reporters of the <italic>Per2</italic> or <italic>Bmal1</italic> promoter were described previously <xref ref-type="bibr" rid="pgen.1004244-Liu2">[17]</xref>, <xref ref-type="bibr" rid="pgen.1004244-Liu3">[43]</xref>. Reporter cells and clonal lines were generated as previously described <xref ref-type="bibr" rid="pgen.1004244-Ramanathan1">[23]</xref>. Briefly, reporter viral particles of high titer (&gt;10<sup>8</sup> viral particles/ml) were obtained by ultracentrifugation and used to infect 3T3, 3T3-L1, and MMH-D3 cells. Clonal cell lines of homogenous cell populations were obtained by single cell sorting and cloning in 96 well plates. We then selected the clones that expressed high levels of luciferase and exhibited circadian properties comparable to infected parental cell populations. These brighter cells were used in high-throughput assays on 96 well plates. Knockdown phenotypes were confirmed to be independent of the reporter, either <italic>Per2</italic>-d<italic>Luc</italic> or <italic>Bmal1</italic>-d<italic>Luc</italic>, and thus phenotypic differences across cell types are unlikely due to different reporter insertion sites.</p><p>We used an optimized shRNA design algorithm adapted from Aza-Blanc et al. <xref ref-type="bibr" rid="pgen.1004244-AzaBlanc1">[72]</xref> for target sequence prediction. This adapted algorithm selects for optimal target sequence for knockdown, and against homologous sequences to minimize off-target effects. We selected 6 target sequences for each gene as listed in <xref ref-type="supplementary-material" rid="pgen.1004244.s011">Table S5</xref>. Each shRNA construct contained a sense and an antisense target sequence of 19 nucleotides (nts) in length, separated by 9 nts for a hairpin loop, and flanked by TTTG at 5&#8242; and GATC at 3&#8242; ends for cohesive end cloning. All oligos (55 nts) were synthesized by Integrated DNA Technologies (IDT). The annealed oligonucleotides were first cloned into the BbsI and SpeI sites of a pGWL-si2/U6 vector, in which the shRNA expression cassette is driven by an RNA polymerase III- based mouse U6 promoter. Subsequently, the U6-shRNA cassette was cloned into the lentiviral pLL3.7GW vector (modified from pLL3.7) <xref ref-type="bibr" rid="pgen.1004244-Liu2">[17]</xref>, <xref ref-type="bibr" rid="pgen.1004244-Rubinson1">[73]</xref> in a Gateway LR Clonase reaction (Life Technologies), according to manufacturer's instructions.</p><p>Viral particles were prepared using standard methods in 293T cells on 12-well plates as previously described <xref ref-type="bibr" rid="pgen.1004244-Ramanathan1">[23]</xref>, <xref ref-type="bibr" rid="pgen.1004244-Tiscornia1">[74]</xref>. Culture medium containing viral particles (&#8764;10<sup>6</sup> viral particles/ml) were collected and used for subsequent infection of reporter cells. Transfection and infection efficiency were estimated by observing GFP co-expressed from a CMV promoter. To produce high titer viruses, crude viral particles were concentrated through ultracentrifugation and appropriate titers were used, as described previously <xref ref-type="bibr" rid="pgen.1004244-Tiscornia1">[74]</xref>. This pipeline allowed us to generate a panel of shRNA constructs against all known clock factors for genetic perturbation and phenotyping.</p><p>We used a LumiCycle luminometer (version 2.31, Actimetrics) for bioluminescence recording of cells grown on 35 mm culture dishes, as described elsewhere <xref ref-type="bibr" rid="pgen.1004244-Liu2">[17]</xref>, <xref ref-type="bibr" rid="pgen.1004244-Ramanathan1">[23]</xref>, <xref ref-type="bibr" rid="pgen.1004244-Liu3">[43]</xref>. The LumiCycle Analysis program version 2.53 (Actimetrics) was used to determine circadian parameters. Briefly, raw data were fitted to a linear baseline, and the baseline-subtracted data were fitted to a sine wave (damped), from which period length and goodness of fit and damping constant were determined. For samples that showed persistent rhythms, goodness-of-fit of &gt;80% was usually achieved. Due to high transient luminescence upon medium change, the first cycle was usually excluded from rhythm analysis. Damping rate&#8202;=&#8202;1/damping constant. For amplitude analysis, raw data from day 3 to day 5 were fitted to a linear baseline, and the baseline-subtracted (polynomial number&#8202;=&#8202;1) data were fitted to a sine wave, from which the amplitude was determined.</p><p>We used a Synergy 2 SL microplate reader (Bio Tek) for bioluminescence recording of cells grown on 96 well plates, as previously described <xref ref-type="bibr" rid="pgen.1004244-Ramanathan1">[23]</xref>. Synergy data were analyzed with the MultiCycle Analysis program (Actimetrics), in which bioluminescence data were baseline-subtracted and fit to a damped sine wave to determine period length, goodness of fit, and amplitude, as with LumiCycle Analysis. Due to the various reporter expression levels and for direct comparison of different rhythms, baseline subtracted data were plotted. Because there is no damping rate output function in the MultiCycle Analysis, we used a curve fitting program of &#8220;CellulaRhythm&#8221; to determine damping rate from Synergy data as previously developed <xref ref-type="bibr" rid="pgen.1004244-Hirota1">[75]</xref>.</p><p>For testing shRNA knockdown efficiency targeting each clock gene, the cDNA was cloned into a p3XFlag-CMV-14 vector. Flag-tagged cDNA was co-transfected with the indicated shRNA in 3T3 or 293T cells. Forty eight hours post-transfection, cells were lysed in RIPA buffer containing complete protease (Roche) and phosphatase inhibitors (Sigma). Protein expression was determined by Western blot analysis using an anti-Flag monoclonal antibody (Sigma). For all Western assays, PVDF membrane was used in protein transfer, and SuperSignal West Pico substrate (Thermo Scientific) was used for chemiluminescent detection.</p><p>For qPCR analysis, parallel infection experiments were performed as with bioluminescence recording. Cells were harvested prior to medium change and were therefore unsynchronized. Total RNAs were prepared using the RNeasy 96 kit (Qiagen), as previously described <xref ref-type="bibr" rid="pgen.1004244-Zhang1">[31]</xref>. Reverse transcription was performed using a high-capacity RNA to cDNA kit (Applied Biosystems), and qPCR was performed using SYBR Green PCR master mix (Thermo Scientific) on an iCycler thermal cycler (BioRad). The primers used in qPCR analysis are listed in <xref ref-type="supplementary-material" rid="pgen.1004244.s012">Table S6</xref>. Transcript levels for each gene were normalized to Gapdh and values were expressed as percentage of expression in NS control cells, as previously described <xref ref-type="bibr" rid="pgen.1004244-Liu2">[17]</xref>.</p>