<p xmlns:ns0="http://www.w3.org/1999/xlink">To determine gene models, we first ran TBLASTN using known hymenopteran Vg sequences against ant genome sequences downloaded from the fourmidable database <xref ref-type="bibr" rid="pgen.1003730-Wurm2">[63]</xref>. Subsequently, ruby/bioruby scripts <xref ref-type="bibr" rid="pgen.1003730-Goto1">[64]</xref> were used to extract relevant subsets of each genome. Gene predictions were generated on each subset using MAKER2 <xref ref-type="bibr" rid="pgen.1003730-Holt1">[65]</xref> s65mand subsequently manually refined using Apollo <xref ref-type="bibr" rid="pgen.1003730-Lewis1">[66]</xref>. Conflicts gene predictions were resolved by using EST data when available, splice prediction algorithms (<ext-link ext-link-type="uri" ns0:href="http://www.fruitfly.org/seq_tools/splice.html">http://www.fruitfly.org/seq_tools/splice.html</ext-link>) and manual verification of splicing consensus sequences.)</p><p>Inaccurate sequence alignment or phylogeny leads to misleading or incorrect results in molecular evolution analyses. We used an approach centered on the use of phylogenetically aware codon-level aligner PRANK, which is likely to minimize the risks of introducing errors <xref ref-type="bibr" rid="pgen.1003730-Tth1">[67]</xref>, <xref ref-type="bibr" rid="pgen.1003730-CapellaGutirrez1">[68]</xref>. This required several steps. We preliminarily aligned the 26 Vg protein sequences with MAFFT linsi <xref ref-type="bibr" rid="pgen.1003730-Katoh1">[69]</xref> and removed ambiguous sections of the alignment using trimAl (option -gappyout) <xref ref-type="bibr" rid="pgen.1003730-CapellaGutirrez2">[70]</xref>. A first tree was built with RAxML (model GTRGAMMAI) <xref ref-type="bibr" rid="pgen.1003730-Stamatakis1">[71]</xref> and rooted with &#8220;nw_root&#8221; (Newick Utilities package <xref ref-type="bibr" rid="pgen.1003730-Junier1">[72]</xref>). This tree was used as a guide tree in PRANK <xref ref-type="bibr" rid="pgen.1003730-Lytynoja1">[73]</xref> to obtain a high-quality codon-level alignment of the 26 Vg coding sequences. Ambiguous sections of the alignment were removed using trimAl (option -gappyout) and a final tree was built with RAxML (GTRGAMMAI model); 10,000 bootstraps were generated to assess its confidence. Selective pressures (dN/dS) on different parts of the phylogenetic tree were estimated using the branch-site codon-substitution model from CodeML (PAML 4.6) <xref ref-type="bibr" rid="pgen.1003730-Yang1">[74]</xref>. Such dN/dS ratios are obtained by computing the number of non-synonymous changes (dN) over synonymous changes (dS) (see <xref ref-type="table" rid="pgen-1003730-t002">Table 2</xref> for more details). Vg site coordinates (S44, E382, N456) are given as in <italic>Apis mellifera</italic> Vg (Uniprot identifier VIT_APIME).</p><p>Selective pressures are divided into three classes: &#969;0 (dN/dS&lt;1, negative selection), &#969;1 (dN/dS&#8202;=&#8202;1, neutral evolution) and &#969;2 (dN/dS &#8805;1, positive selection). p0, p1 and p2a+p2b are their respective proportion. lnL0 and lnL1 are the likelihood values of the null (&#969;2&#8202;=&#8202;1) and alternative (&#969;2&#8805;1) models, respectively. The likelihood ratio test [2*(lnL1-lnL0)] allows discriminating between neutral evolution and positive selection. A p-value&lt;0.01 presents a significant test (marked as **). Sites under positive selection with BEB score &gt;95% are shown.</p><p>
<italic>Pogonomyrmex barbatus</italic> founding queens were collected during nuptial flights on July 15<sup>th</sup>, 2008 in Bowie, Arizona, USA (N32&#176;18&#8242;54&#8243;//W109&#176;29&#8242;03&#8243;). Colonies were then kept in laboratory conditions (30&#176;C, 60% humidity and 12 h/12 h light&#8758;dark cycle) in 15&#215;13&#215;5 cm plastic boxes with water tubes, and were fed once a week with grass seeds and a mixture of eggs, honey and smashed mealworms. Thirty months later, five of these colonies were used to collect samples on December 16<sup>th</sup>, 2010. Task performance in workers is age related, thus nurses tend to be younger than foragers <xref ref-type="bibr" rid="pgen.1003730-Hlldobler1">[2]</xref>. Young ants interacting with the brood in the nest tube were considered as nurses. To collect foragers, each colony was connected with a cardboard-made bridge to a foraging area composed of a plastic box containing grass seeds. Any ant handling a food item in the foraging area was considered as forager. Ant samples were flash-frozen in liquid nitrogen and kept at &#8722;80&#176;C for further RNA extraction.</p><p>Whole body worker samples were used to measure the expression of <italic>Pb_Vg1</italic> and <italic>Pb_Vg2</italic> genes. RNA extractions were performed using a modified protocol including the use of Trizol (Invitrogen) for the initial homogenization step, RNeasy extraction kit and DNAse I (Qiagen) treatment to remove genomic DNA traces. For each individual worker, cDNAs were synthesized using 500 ng of total RNA, random hexamers and Applied Biosystems reagents. Levels of mRNA were quantified by quantitative real-time polymerase chain reaction (qRT-PCR) using ABI Prism 7900 sequence detector and SYBR green. All qPCR assays were performed in triplicates and subject to the heat-dissociation protocol following the final cycle of the qPCR in order to check for amplification specificity. qRT-PCR values of each gene were normalized by using an internal control gene (RP49). Paralog-specific primers (<xref ref-type="supplementary-material" rid="pgen.1003730.s002">Table S1</xref>) were designed using sequence alignment <xref ref-type="bibr" rid="pgen.1003730-Thompson1">[75]</xref> and primer analysis <xref ref-type="bibr" rid="pgen.1003730-Rychlik1">[76]</xref> programs. Primer sequences overlapped coding regions split by introns, allowing the specific amplification of cDNA levels over eventual genomic DNA contaminations. Transcript quantification calculations were performed by using the &#916;&#916;CT method <xref ref-type="bibr" rid="pgen.1003730-Livak1">[77]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All data were analyzed using R (<ext-link ext-link-type="uri" ns0:href="http://www.r-project.org/">http://www.r-project.org/</ext-link>) and the R packages lme4 <xref ref-type="bibr" rid="pgen.1003730-Bates1">[78]</xref> and language R <xref ref-type="bibr" rid="pgen.1003730-Baayen1">[79]</xref>. The effect of caste on gene expression relative values was analyzed using linear mixed effects models. To avoid pseudoreplication, the colony was included as a random effect. We checked for normality and homogeneity by visual inspections of plots of residuals against fitted values. To assess the validity of the mixed effects analyses, we performed likelihood ratio tests to confirm that the models with fixed effects differed significantly from the null models with only the random effects. Throughout the paper, we present MCMC (Markov-chain Monte Carlo) estimated p-values that are considered significant below the 0.05 threshold (all significant results remained significant after Bonferroni correction).</p>