<p>The association panel consisted of 66 diverse switchgrass populations grown from seed in the greenhouse of the USDA-ARS Dairy Forage Research Center in Madison WI in 2007. This panel was mainly composed of northern adapted upland populations (<xref ref-type="supplementary-material" rid="pgen.1003215.s015">Table S1</xref>). In addition, two tetraploid F1 linkage populations were propagated at the same time. One was derived from the bi-parental cross of two upland accessions from the germplasm collection of MDC, named U518 and U418. The other was a half-sib population whose maternal parent is U601. The numbers of individuals in these two populations were 130 and 168, respectively. For the association panel, ten clones from each population were initially planted in replicated field plots in Ithaca, NY and Arlington, WI in 2008. All genotyping was conducted on plants from the Ithaca site.</p><p>The reduced representation libraries were constructed and sequenced according to the published GBS protocol <xref ref-type="bibr" rid="pgen.1003215-Elshire1">[4]</xref> with one modification. Specifically, a titration experiment showed that &#8764;0.1 pmol of each adapter was appropriate for switchgrass (rather than &#8764;0.06 pmol), and that amount was used with 100 ng of genomic DNA. DNA samples were digested with the restriction enzyme <italic>Ape</italic>KI, which has a 4.5 bp cut site (CGWGC, where W&#8202;=&#8202;A or T). The resulting libraries were sequenced on the Illumina HiSeq 2000. Ninety five samples (plus a blank negative control) were sequenced per lane.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The non-reference pipeline UNEAK (<ext-link ext-link-type="uri" ns0:href="http://www.maizegenetics.net/gbs-bioinformatics">http://www.maizegenetics.net/gbs-bioinformatics</ext-link>) was developed for SNP discovery and genotyping in species like switchgrass (<xref ref-type="fig" rid="pgen-1003215-g001">Figure 1</xref>). In UNEAK, Illumina reads are trimmed to 64 bp and stored in bit format, which greatly reduces the amount of storage space and enables relatively fast computation. About 40GB of data from one lane of an Illumina HiSeq 2000 can be processed to SNP genotypes in 20 minutes on a personal computer with 2.67 GHz CPU and 8 GB memory. More technical details of UNEAK are described in <xref ref-type="supplementary-material" rid="pgen.1003215.s018">Text S1</xref>.</p><p>The network filter is the key step for identifying and removing paralogs. The simplest networks, reciprocal tag pairs, are more likely to be real SNPs than tag pairs that are part of complicated networks. Rare tags, with read counts below a specified error tolerance rate (ETR), are assumed to result from sequencing error. For two tags (t1 and t2) with a 1 bp mismatch and read counts c1 and c2, if c1/(c1+c2)&lt;ETR, then t1 is assumed to be a sequencing error of t2. The edges connecting real tags and error tags are then sheared, dividing the network complexes into parts. Remaining reciprocal, real tag pairs are then identified as SNPs (<xref ref-type="supplementary-material" rid="pgen.1003215.s001">Figure S1</xref>).</p><p>According to the frequency distribution of tag pairs in structured populations (<xref ref-type="fig" rid="pgen-1003215-g002">Figure 2A</xref> and <xref ref-type="supplementary-material" rid="pgen.1003215.s002">Figure S2C</xref>), we found the frequencies of most sequencing errors were less than 3%. Therefore, the ETR was specified as 0.03 in this study.</p><p>To calculate the coverage in the SNP data sets, we calculated the sum of the read counts of all of the sites across all the individuals. This sum is then divided by the number of sites and number of individuals. So the measurement of coverage in this study is reads/site/individual.</p><p>The maternal and paternal parents of the full-sib linkage population, U518 and U418, were sequenced via GBS at about 6&#215;, i.e., at six times higher coverage than the F1 individuals in the full-sib population. SNP markers that were homozygous in one parent and heterozygous in the other and which had a minor allele frequency (MAF) in the progeny between 0.2 and 0.3 were chosen for linkage analysis via the pseudo-testcross mapping approach <xref ref-type="bibr" rid="pgen.1003215-Grattapaglia1">[27]</xref> (<xref ref-type="supplementary-material" rid="pgen.1003215.s007">Figure S7</xref>). We selected two sets of 3,000 markers for paternal and maternal linkage groups, respectively.</p><p>Initial linkage groups were constructed based upon the 6,000 markers with the highest call rate (&gt;0.9). The MMC method <xref ref-type="bibr" rid="pgen.1003215-Stone1">[25]</xref> was used to cluster the markers into linkage groups. In the MMC input file, homozygous genotypes were assigned a value of 0 or 2, and heterozygous genotypes and missing data were assigned a value of 1. Spearman's rank correlation coefficient was used by MMC.</p><p>Markers representing the cross of AA&#215;Aa segregate from either a paternal or maternal linkage group. Using the 36 linkage groups produced by MMC from these initial 6,000 markers as seeds, 88,217 markers (0.12&lt;MAF&lt;0.38, call rate &gt;0.5) were then assigned to linkage groups based upon their Spearman's rank correlation coefficient to each seed linkage group.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To order the markers within each linkage group, we relied upon extensive synteny between the switchgrass and foxtail millet genomes. Markers were mapped to the foxtail millet genome (<ext-link ext-link-type="uri" ns0:href="http://www.phytozome.net/foxtailmillet.php">http://www.phytozome.net/foxtailmillet.php</ext-link>) via Basic Local Alignment Search Tool (BLAST) <xref ref-type="bibr" rid="pgen.1003215-Altschul1">[56]</xref> with a <italic>P</italic>-value cutoff of 1e-5. The 7,245 markers that mapped to a single site of the foxtail millet genome, and clustered with one of the syntenic linkage groups of switchgrass, were used to construct the synteny based framework linkage maps. To construct the high density linkage maps, the rest of the 80,972 markers were mapped to the framework marker with highest value of Spearman's rank correlation coefficient within their assigned linkage groups.</p><p>A pairwise genetic distance matrix between individuals was calculated and an un-rooted NJ tree constructed using TASSEL <xref ref-type="bibr" rid="pgen.1003215-Bradbury1">[57]</xref>. All of the 29,221 markers with a call rate greater than 0.5 in the diverse populations were used in this analysis. To assess the robustness of the topology of the tree, 500 bootstrap replicates were performed using MEGA <xref ref-type="bibr" rid="pgen.1003215-Tamura1">[58]</xref>.</p><p>To address the evolutionary trajectory of upland switchgrass, a two-step phylogenetic analysis was performed. In the first step, foxtail millet was used as an outgroup. A NJ tree was reconstructed based on the 3,144 SNPs that could be aligned to unique positions in the foxtail millet genome. This first step identified the lowland ecotype as ancestral to the remaining switchgrass ecotypes studied herein. The second step omitted foxtail millet and used the lowland ecotype as the outgroup. This second NJ tree was reconstructed based on 29,221 markers (alignment to foxtail millet not required). An MDS plot was also generated based upon the kinship matrix of individuals calculated from the 29,221 markers in TASSEL <xref ref-type="bibr" rid="pgen.1003215-Bradbury1">[57]</xref>.</p>