<p>In each human study, the local institutional review board approved the study and participants signed written informed consent, including for DNA analyses. The experimental protocol in mice was approved by the local veterinarian authorities and fulfilled Swiss federal regulations for experiences with animals.</p><p>A list of all discovery and replication studies, their sample size, mean serum calcium levels, age and serum albumin as well as proportion of women can be found in <bold><xref ref-type="supplementary-material" rid="pgen.1003796.s009">Table S1</xref></bold>. We replicated findings using <italic>de novo</italic> genotyping in the Bus Sant&#233; Study and <italic>in silico</italic> data in all other cohorts. In most studies, serum calcium was measured using a colorimetric assay. The size of discovery tables varied from 488 to 9,049 for a total of 39,400 participants. A detailed description of the characteristics of discovery and replication cohorts, including laboratory method for serum calcium measurement, can be found in <bold><xref ref-type="supplementary-material" rid="pgen.1003796.s020">Table S12</xref></bold>.</p><p>Detailed information on the genotyping plateforms and data cleaning procedures for each discovery and replication cohort can be found in <bold><xref ref-type="supplementary-material" rid="pgen.1003796.s021">Table S13</xref></bold>. <italic>De novo</italic> replication genotyping was perfomed in 4670 participants to the Bus Sant&#233; Study using KASPar v4.0 after whole genome amplification by primer extension pre-amplification (PEP) using thermostable DNA polymerases.</p><p>In each discovery study, genotyping was performed using a genome-wide chip and nearly 2.5 million SNPs were genotyped or imputed using the HapMap CEU panels release 22 or 21 as the reference. Each study applied quality control before imputation. Detailed imputation information is provided in <bold><xref ref-type="supplementary-material" rid="pgen.1003796.s021">Table S13</xref></bold>. Each SNP was modeled using an additive genetic effect (allele dosage for imputed SNPs), including age and sex as covariates in the model as well as study-specific covariates if needed (e.g. principal components, study center). The primary dependent variable in each discovery study was untransformed and uncorrected serum calcium expressed in mg/dL. Beta regression coefficients and standard errors were used with at least 5 decimal places. For secondary analyses, albumin-corrected serum calcium was computed using the following formula: ([4-plasma albumin in g/dL]&#215;0.8+serum calcium in mg/dL) and the same model as for the primary analyses was used. Each file of genome-wide summary statistics underwent extensive quality control prior to meta-analysis both for primary and secondary analyses, including (1) boxplots of all beta coefficients, as well as all standard errors multiplied by the square-root of the sample size, for each study separately; (2) the range of P values, MAF, imputation qualities, call rates and Hardy-Weinberg equilibrium P values and (3) QQ plots. In addition, we checked the direction and magnitude of effect at the previously reported rs1801725 <italic>CASR</italic> variant. Genome-wide meta-analyses were conducted in duplicate by two independent analysts. For each SNP, we used a fixed effect meta-analysis using inverse-variance weights as implemented in the meta-analysis utility Metal <xref ref-type="bibr" rid="pgen.1003796-Willer1">[16]</xref>. Results were confirmed by a z-score based meta-analysis. Data were available for 2,612,817 genotyped or imputed autosomal SNPs for the primary and secondary analyses. After the meta-analysis, genomic control correction was applied (&#955;GC was 1.03 for both uncorrected and corrected serum calcium). Our pre-specified criterion to declare genome-wide significance was P value&lt;5E-8 to account for 1 million independent tests according to the Bonferroni correction. We choose to move forward for replication all SNPs with discovery P value&lt;1E-7 in the European sample or genome-wide significant SNP in the overall sample that included Indian Asians. To choose a single SNP per genome-wide associated region for replication, we merged all SNPs within 1 Mb region and selected the lowest P value for each region. Altogether, fourteen SNPs were moved forward for replication. Up to 17,205 participants contributed information to the replication analyses <italic>in silico</italic> and 4,670 participants provided data for <italic>de novo</italic> genotyping. We used fixed-effects inverse-variance weighted meta-analysis to combine discovery and replication meta-analysis results. Replication was considered as present whenever a combined P value&lt;5E-8 together with an effect-concordant one-sided replication P value&lt;0.05 were obtained.</p><p>We conducted look-ups for femoral and lumbar bone density in the GEnetic Factors of OSteoporosis (GEFOS) dataset <xref ref-type="bibr" rid="pgen.1003796-Estrada2">[17]</xref>. Bone mineral density (BMD) is used in clinical practice for the diagnosis of osteoporosis and bone density at different skeletal sites is predictive of fracture risk. BMD was measured in all cohorts at the lumbar spine (either at L1&#8211;L4 or L2&#8211;L4) and femoral neck using dual-energy X-ray absorptiometry following standard manufacturer protocols <xref ref-type="bibr" rid="pgen.1003796-Estrada2">[17]</xref>. Serum phosphorus was looked up from a previously published GWAS meta-analysis, including 16,264 participants of European ancestry <xref ref-type="bibr" rid="pgen.1003796-Kestenbaum1">[18]</xref>. Serum phosphorus concentrations were quantified using an automated platform in which inorganic phosphorus reacts with ammonium molybdate in an acidic solution to form a colored phosphomolybdate complex <xref ref-type="bibr" rid="pgen.1003796-Kestenbaum1">[18]</xref>. The 25-hydroxyvitamin D was looked-up in the SUNLIGHT consortium <xref ref-type="bibr" rid="pgen.1003796-Wang1">[7]</xref>, which includes data from 33,996 individuals of European descent from 15 cohorts. 25-hydroxyvitamin D concentrations were measured by radioimmunoassay, chemiluminescent assay, ELISA, or mass spectrometry <xref ref-type="bibr" rid="pgen.1003796-Wang1">[7]</xref>. PTH was looked-up in the SHIP and SHIP-Trend studies. The serum parathyroid hormone concentration was measured on the IDS-iSYS Multi-Discipline Automated Analyser with the IDS-iSYS Intact PTH assay (Immunodiagnostic Systems Limited, Frankfurt am Main, Germany) according to the instructions for use. This chemiluminescence immunoassay detects the full-length parathyroid hormone (amino acids 1&#8211;84) and the large parathyroid hormone fragment (amino acids 7&#8211;84). The measurement range of the assay was 5&#8211;5000 pg/mL. The limits of blank, detection and quantitation were 1.3 pg/mL, 1.4 pg/mL, and 3.6 pg/mL, respectively. As recommended by the manufacturer, three levels of control material were measured in order to verify a decent working mode. During the course of the study, the coefficients of variation were 14.02% at low, 6.64% at medium, and 6.84% at high serum parathyroid hormone concentrations in the control material in SHIP and the corresponding percentages were 16.8% at low, 10.7% at medium, and 9.0% at high serum parathyroid hormone concentrations in the control material in SHIP-Trend.</p><p>The Hypergene dataset (a 4206 samples case-control study concerning hypertension genotyped using the Illumina 1M chip) has been used to call CNVs and to check their correlation with the SNPs of interest. The CNVs calls have been done using pennCNV software <xref ref-type="bibr" rid="pgen.1003796-Wang2">[19]</xref>. A SNP by sample matrix with the copy number status was created. Then the square correlation (Pearson correlation) between value of each SNP of interest and the SNPs copy number status in a +/&#8722;2 Mb region was calculated. The SNPs of interest for which no correspondence has been found in the Hypergene dataset have been replaced by the closest SNPs in high linkage disequilibrium (LD) and present in the Hypergene dataset. LD between the SNPs of interest and a list of SNPs tagging CNVs from the GIANT consortium has also been calculated. The SNPs from the GIANT list are in LD higher than 0.8 with their corresponding CNV.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We queried the AmiGo 1.8 gene ontology database for each gene located within &#177;250 kb of the seven replicated SNPs, including rs1801725 (<italic>CASR</italic>). (<ext-link ext-link-type="uri" ns0:href="http://amigo.geneontology.org/cgi-bin/amigo/go.cgi">http://amigo.geneontology.org/cgi-bin/amigo/go.cgi</ext-link>, last accessed November 6, 2012). We used <italic>Homo sapiens</italic> as a filter for species.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For each of the 7 replicated SNPs, we identified all proxy SNPs with r<sup>2</sup>&gt;0.8 in HapMap CEU (releases 21, 22, and HapMap 3 vers. 2) using the online SNAP database (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/mpg/snap/">http://www.broadinstitute.org/mpg/snap/</ext-link>). We then queried each of these 40 SNPs in the eQTL database of the University of Chicago (<ext-link ext-link-type="uri" ns0:href="http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/">http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/</ext-link>).</p><p>The rs1801725 SNP encodes a missense variant in exon 7 of the CASR gene leading to an alanine to serine substitution (A986S). Given the key physiological role of CASR in calcium homeostasis (monogenic disorders of calcium balance), this gene was the logical candidate for analysis in mouse at this previously identified locus.</p><p>For the 6 newly identified loci, the precise rationale for gene selection varied from one locus to the other, but the main criteria was to focus on the most biologically relevant gene. Rs1550532 on chromosome 2 is an intronic SNP of <italic>DGKD</italic>, which was the most likely biological candidate for this locus and was therefore selected for analysis in mouse. None of the other genes located in this region (&#177;250 Kb) has a known link with calcium homeostasis (<xref ref-type="sec" rid="pgen-1003796-box001">
<bold>Box 1</bold>
</xref>) and rs1550532 is not in strong linkage disequilibrium with an eQTL (<bold><xref ref-type="supplementary-material" rid="pgen.1003796.s014">Table S6</xref></bold>). We also took into account the fact that another member of the DGK family, namely DGKH was located near one of the other replicated loci, on chromosome 13.</p><p>Rs780094, on chromosome 2, is located in intro 16 of <italic>GCKR</italic> and is in strong linkage disequilibrium (r<sup>2</sup>&#8202;=&#8202;0.93) in Caucasians <xref ref-type="bibr" rid="pgen.1003796-OrhoMelander1">[20]</xref>, with a common non-synonymous SNP (P446L, rs1260326) associated with glucokinase activity <italic>in vitro</italic>
<xref ref-type="bibr" rid="pgen.1003796-OrhoMelander1">[20]</xref>, <xref ref-type="bibr" rid="pgen.1003796-Beer1">[21]</xref>. This SNP has been associated with multiple other phenotypes in previous GWAS and it is in strong linkage disequilibrium with an eQTL (<bold><xref ref-type="supplementary-material" rid="pgen.1003796.s014">Table S6</xref></bold>). Previous fine mapping analysis of this locus has attributed the signal from rs780094 to the functional rs1260326 variant <xref ref-type="bibr" rid="pgen.1003796-OrhoMelander1">[20]</xref>. <italic>The GCKR</italic> locus may indirectly influence calcium concentrations via its association with albumin levels <xref ref-type="bibr" rid="pgen.1003796-Franceschini1">[22]</xref>. In line with this, we observed an attenuation of the association of rs780094 with albumin-corrected serum calcium compared to the association with uncorrected serum calcium and we found <italic>GCKR</italic> not to be expressed in any of the key organs involved in calcium homeostasis that we tested in mice. We selected <italic>GCKR</italic> for analysis in mouse at this locus.</p><p>Rs10491003 on chromosome 10 is located within a long non-coding RNA. For this locus, we selected <italic>GATA3</italic>, the nearest and only gene located within this region, for analysis in mouse. GATA3 is implicated in monogenic disorders of calcium balance.</p><p>Rs7481584 is located within <italic>CARS</italic> (intronic SNP) in an imprinted region known to play a role in multiple cancers, which makes this locus a plausible candidate for malignancy-related hypercalcemia. Other plausible biological candidates in this locus are <italic>NAP1L4, PHLDA2 and CKDN1C</italic> (<xref ref-type="sec" rid="pgen-1003796-box001">
<bold>Box 1</bold>
</xref>). Rs7481584 is in strong LD with 2 eQTLs, one associated with the expression of <italic>NAP1L4</italic> (rs2583435) and the other one associated with the expressions of <italic>SLC22A18</italic> and <italic>SLC22A18AS</italic>. We selected <italic>CARS</italic>, <italic>NAP1L4, PHLDA2 and CKDN1C</italic> for analyses in mouse.</p><p>For rs7336933, we selected the two only genes (<italic>DGKH</italic> and <italic>KIAA0564</italic>) located under this association peak on chromosome 13 for analyses in mouse.</p><p>Finally, rs1570669 is an intronic SNP of <italic>CYP24A1</italic>, a strong biological candidate implicated in monogenic disorders of calcium balance. The two other genes of this region (<italic>BCAS1</italic> and <italic>PFDN4</italic>) have no known link with calcium homeostasis. Furthermore, rs1570669 and <italic>PFDN4</italic> are separated by a recombination hot spot. We selected <italic>CYP24A1</italic> for analysis in mouse.</p><p>As animal experiments started while the replication process was underway, we had also initially selected the following genes for analysis in mouse: RSG14 and <italic>SLC34A1</italic> at locus rs4074995 (discovery P value&#8202;=&#8202;2.4E-07), <italic>VKORC1L1</italic> at locus rs17711722 (discovery P value&#8202;=&#8202;2.8E-11), <italic>PYGB</italic> at locus rs2281558 (discovery P value&#8202;=&#8202;6.4E-07), <italic>CD109</italic> at locus rs9447004 (discovery P value&#8202;=&#8202;8.1E-06). No gene was selected for the rs2885836 and rs11967485 and rs12150338 loci in the absence of obvious candidate. Results for these unreplicated loci can be found in <bold><xref ref-type="supplementary-material" rid="pgen.1003796.s006">Figures S6</xref>, <xref ref-type="supplementary-material" rid="pgen.1003796.s007">S7</xref> and <xref ref-type="supplementary-material" rid="pgen.1003796.s008">S8</xref></bold>. We present these results for quality control purposes: <italic>SLC34A1</italic> (also known as NAPI-3 or NPT2), which encodes solute carrier family 34 (sodium phosphate), member 1, was expressed in the kidney, but neither in duodenum nor in bone, as expected based on current knowledge on this phosphate transporter. In the kidney <italic>SLC34A1</italic> was mainly expressed proximally and <italic>SLC34A1</italic> expression was upregulated under low calcium diet, which is in line with the known function of this gene.</p><p>Five C57bl/6 mice (Janvier) per group were fed, for one week, three different diets in which the percentage of calcium were 0.17% (low calcium diet), 0.82% (normal calcium diet) and 1.69% (high calcium diet) and had free access to water. 12&#8758;12 hours light/dark alternance was imposed. At the end of the week of the specific diet, spot urine were collected and mice were anesthetized. Blood was collected by retro-orbital puncture. Organs were immediately harvested and snap frozen. RNA was extracted using Trizol (Invitrogen) and reversed transcribed with PrimeScriptTM RT reagent Kit (Takara Bio Inc). Calcium, sodium, phosphate and creatinine in plasma and urine were analyzed at the central lab of the Lausanne University hospital using a Cobas-Mira analyzer (Roche).</p><p>A separate set of three mice was kept under normal calcium diet. Proximal Tubule (Prox), thick ascending limb of the loop of Henle (TAL), distal convoluted tubule and connecting tubule (DCT-CNT) and cortical collecting duct (CCD) were isolated by microdissection of the left kidney after the mice were perfused with Liberase TM (Roche Diagnostics) <xref ref-type="bibr" rid="pgen.1003796-Zuber2">[23]</xref>. RNA was extracted from the above mentioned tubules following TRI Reagent Solution protocol (Applied Biosystems) and purified with RNeasy Micro Kit (Qiagen). Reversed transcription was performed with PrimeScriptTM RT reagent Kit (Takara Bio Inc). Quantitative PCRs were performed (7500 Software v 2.0.4.) using TaqMan gene expression assays for the different genes (Applied Biosystems) and comparative CT method was applied. Expression levels were normalised to beta actin as endogenous reference gene.</p><p>Comparison of groups was performed using unpaired Student's t-test.</p>