<p>Strains were maintained at 20&#176;C under standard conditions. Experiments were performed at 20&#176;C unless otherwise noted. Strains used in this study:</p><p>AV334 <italic>unc-119 III; ruIs32 [Ppie-1::GFP-his-11; unc-119(+)] III mnT12 (IV;X)</italic>
</p><p>AZ212 <italic>unc-119 III; ruIs32 [Ppie-1::GFP-his-11; unc-119(+)] III</italic>
</p><p>AV477 <italic>dsb-2(me96) II</italic> 4X outcrossed</p><p>AV501 <italic>rol-1(e91) dsb-2(me96) II</italic>
</p><p>AV511 <italic>rol-1(e91) dsb-2(me96) unc-52(e998) II</italic>
</p><p>AV539 <italic>rol-1(e91) dsb-2(me96)/mnC1 [dpy-10(e128) unc-52(e444)] II</italic>
</p><p>AV727 <italic>meIs8[unc-119(+) pie-1promoter::gfp::cosa-1] II ;; ItIs37[unc-119(+)pie-1::mcherry::histoneH2B]; ltIs38[pAA1;pie-1 promoter::GFP::PH::unc-119(+)]</italic>
</p><p>AV758 <italic>dsb-2(me97) meIs8[unc-119(+) pie-1promoter::gfp::cosa-1] II ;; ItIs37[unc-119(+)pie-1::mcherry::histoneH2B]; ltIs38[pAA1;pie-1 promoter::GFP::PH::unc-119(+)]</italic>
</p><p>AV630 <italic>meIs8[unc-119(+) pie-1promoter::gfp::cosa-1] II</italic>
</p><p>AV645 <italic>spo-11(ok79)/nT1 IV; +/nT1[qIs51] V</italic>
</p><p>AV146 <italic>chk-2(me64) rol-9(sc148)/unc-57(e369) rol-9(sc148) V</italic>
</p><p>AV660 <italic>chk-2(me64) rol-9(sc148)/sC4(s2172)[dpy-21(e428)] V</italic>
</p><p>VC292 <italic>+/nT1 IV; sun-1(gk199)/nT1 V</italic>
</p><p>VC255 <italic>+/nT1 IV, him-17(ok424)/nT1 V</italic>
</p><p>AV158 <italic>+/nT1 IV; rad-50(ok197)/nT1 [unc-?(n754) let-? qIs50] V</italic>
</p><p>TG9 <italic>dpy-13(e184) rad-51(lg8701) IV/nT1[let-?(m435)] (IV;V)</italic>
</p><p>VC531 <italic>rad-54 and tag-157(ok615) I/hT2[gli-4(e937) let(9782) qIs48] I; III</italic>
</p><p>AV449 <italic>zhp-3(me95)/hT2 [bli-4(e937) let-? (q782) qIs48] I</italic>
</p><p>AV603 <italic>msh-5(me23)/nT1 IV; +/nT1[qIs51] V</italic>
</p><p>AV596 <italic>cosa-1(tm3298)/qC1[qIs26] III</italic>
</p><p>AV307 <italic>+/nT1 IV; syp-1(me17)/nT1 V</italic>
</p><p>AV393 <italic>htp-1(gk174) IV/nT1[unc-?(n754) let-? qIs50] (IV;V)</italic>
</p><p>TY4986 <italic>htp-3(y428) ccIs4251 I/hT2[bli-4(e937)let-?(q782) qIs48] (I,III).</italic>
</p><p>AV473 <italic>+/nT1 IV; rad-50(ok197)/nT1[qIs51] V</italic>
</p><p>AV443 <italic>htp-1(gk174)/nT1[ unc-?(n754) let-? qIs50] IV; rad-50 (ok197)/nT1 [qIs51] V</italic>
</p><p>Bristol (N2) wild type</p><p>CB4856 Hawaiian wild type</p><p>The <italic>dsb-2(me96)</italic> allele was isolated in a genetic screen for meiotic mutants exhibiting defects in chiasma formation or chromosome organization in diakinesis-stage oocytes, conducted in collaboration with M. Hayashi <xref ref-type="bibr" rid="pgen.1003674-Rosu2">[47]</xref>. The AV334 strain used for this screen, which allows visualization of chromosomes using a germline-expressed GFP::histone H2B fusion protein, also contains a fusion of chromosomes IV and X. Parental (P0) L4 hermaphrodites were treated with ethyl methanosulfonate (EMS) as in <xref ref-type="bibr" rid="pgen.1003674-Villeneuve1">[48]</xref> and were plated individually. F1 progeny were picked to individual plates to produce progeny, and pools of F2 progeny worms from each F1 plate were mounted on multi-well slides in anesthetic (0.1% tricaine and 0.01% tetramisole in M9 buffer) and their germ lines were visualized for meiotic defects. Two mutations affecting meiotic recombination, <italic>me95</italic> and <italic>me96</italic>, were identified based on the presence of univalents at diakinesis in a the subset of F2s (from independent F1s) and were recovered by plating of siblings; repeated outcrossing (with N2) and selecting for the mutant phenotype removed the chromosome fusion as well as the GFP::H2B transgene. Mapping, complementation testing and sequencing revealed that <italic>me95</italic> is an allele of <italic>zhp-3</italic>, containing a C-to-T transition that results in a premature stop at codon 348 of the predicted 387 amino acid coding sequence of <italic>K02B12.8a</italic>. Mapping of the <italic>me96</italic> mutation (below) indicated that it identified a new component of the meiotic machinery.</p><p>Initial SNP mapping based on the methods of <xref ref-type="bibr" rid="pgen.1003674-Wicks1">[49]</xref> and <xref ref-type="bibr" rid="pgen.1003674-Davis1">[50]</xref> placed <italic>me96</italic> near genetic map position 20 on the right side of chromosome II. To select for informative COs near this region, <italic>rol-1 me96 unc-52</italic> worms were crossed with CB4856 males, and Rol non-Unc F2 progeny were selected and genotyped for SNP pkP2117 at genetic map position 17.9 to select for COs occurring between this marker and <italic>unc-52</italic> (genetic map position 23). Informative recombinants were assessed for <italic>me96</italic> phenotype and typed using additional SNP markers in the region, narrowing down the position of <italic>me96</italic> to a 165 kb interval (between SNP markers uCE2-2315 and uCE2-2332) comprising 36 candidate genes. RNAi of candidate genes was performed using bacterial clones from the RNAi feeding library <xref ref-type="bibr" rid="pgen.1003674-Fraser1">[51]</xref>, <xref ref-type="bibr" rid="pgen.1003674-Kamath1">[52]</xref> as in <xref ref-type="bibr" rid="pgen.1003674-Wignall1">[53]</xref>. Worms used were AZ212 worms, which contain GFP::histone, as this genotype was shown to be more sensitive to RNAi <xref ref-type="bibr" rid="pgen.1003674-Hayashi2">[54]</xref>. RNAi against candidate gene <italic>F26H11.6</italic> (at 15&#176;C, but not at 20&#176;C) phenocopied the <italic>me96</italic> mutation, eliciting a mixture of bivalents and univalents at diakinesis. Sequencing identified a T-to-A transversion generating an early stop at codon 14 (TTA&#8202;=&#8202;&gt;TAA) of the predicted <italic>F26H11.6</italic> coding sequence (280 codons total) in the <italic>me96</italic> mutant.</p><p>A second <italic>dsb-2</italic> allele <italic>(me97)</italic> was isolated independently in a screen for mutants with altered numbers of GFP::COSA-1 foci (which mark CO sites in late pachytene); <italic>me97</italic> fails to complement <italic>me96</italic> and contains a premature stop at codon 168, further confirming the identity of <italic>F26H11.6</italic> as the <italic>dsb-2</italic> gene. Except where otherwise noted, all analyses were conducted using the <italic>me96</italic> allele.</p><p>L4 hermaphrodites were picked to individual plates, allowed to lay eggs, and transferred to fresh plates every 24 hours for three days. Hermaphrodites start laying eggs after they transition from L4 to adult, and lay most eggs in the first three days. Inviable embryos that do not hatch are indicative of autosomal mis-segregation, while male progeny indicate X-chromosome mis-segregation. Eggs from eight <italic>dsb-2(me96); rol-1(e91)</italic> hermaphrodites (where <italic>rol-1</italic> is a marker with no meiotic defects) were counted. The number of eggs counted for each day is: 573 (day 1), 879 (day 2) and 488 (day 3), with an average of 243 eggs per hermaphrodite over the three day interval.</p><p>Numbers of DNA bodies present in diakinesis oocytes were assessed in intact adult hermaphrodites of indicated age, fixed in ethanol and stained with 4&#8242;,6-diamidino-2-phenylindole (DAPI) as in <xref ref-type="bibr" rid="pgen.1003674-Bessler1">[40]</xref>. This method underestimates the frequency of achiasmate chromosomes, as some univalents lie too close to each other to be resolved unambiguously. Numbers of nuclei scored were: 88 and 106 for <italic>dsb-2</italic> worms, 1 day and 2 day post L4 respectively; 43 and 57 for WT worms, 1 day and 2 day post L4 respectively.</p><p>Numbers of GFP::COSA-1 foci in late pachytene nuclei of live anesthetized worms (0.1% tricaine and 0.01% tetramisole in M9 buffer ) were quantified by taking 3D image stacks on a DeltaVision microscope. GFP foci were counted in the last five rows of pachytene nuclei; only nuclei completely contained within the stack were scored, and nuclei with features indicative of apoptosis (compact and bright mCherry::histoneH2B signal) were excluded. 24 h control data were from fixed immunofluorescence images <xref ref-type="bibr" rid="pgen.1003674-Yokoo1">[13]</xref>. Numbers of nuclei scored: <italic>dsb-2</italic> 24 h, n&#8202;=&#8202;127; <italic>dsb-2</italic> 48 h, n&#8202;=&#8202;101; <italic>WT</italic> 24 h, n&#8202;=&#8202;76; <italic>WT</italic> 48 h, n&#8202;=&#8202;78. (Note: Whereas most nuclei in a mutant that lacks DSBs and COs have zero COSA-1 foci, 20% have one or two foci presumably reflecting non-specific aggregation of CO proteins when a suitable substrate is absent <xref ref-type="bibr" rid="pgen.1003674-Yokoo1">[13]</xref>. Thus, a subset of <italic>dsb-2</italic> nuclei with one or two COSA-1 foci may similarly lack COs, especially at 48 h post L4 where nuclei with zero foci are frequent.)</p><p>Immunofluorescence was conducted as in <xref ref-type="bibr" rid="pgen.1003674-Nabeshima1">[55]</xref> with minor modifications. Unless otherwise noted, all experiments were performed at 40&#8211;48 hours post L4. Worms were cut at the vulva to dissect the gonads (in egg buffer with 0.1% Tween-20) and fixed with 1% paraformaldehyde (in egg buffer) for 5 minutes. Slides (Superfost Plus) were covered with a coverslip and frozen in liquid nitrogen. The coverslip was removed, and slides were immersed in cold (&#8722;20&#176;C) methanol for 1 minute. Slides were washed three times for 8&#8211;10 minutes in phosphate-buffered saline containing 0.1% Tween-20 (PBST) and then blocked for one hour with 0.5% bovine serum albumin (BSA) diluted in PBST. Primary antibody solution was added (50 &#181;l) on top of the dissected gonads and covered with a parafilm square. Slides were incubated overnight in a humid chamber at room temperature, then washed three times for 8&#8211;10 minutes in PBST. Secondary antibody solution was added (50 &#181;l) and slides were incubated with parafilm cover for 2 hours at room temperature in the dark. Slides were washed three times with PBST and incubated for 5 minutes with 2 &#181;g/ml DAPI solution in the dark, followed by two more washes. Slides were mounted with Vectashield and the coverslip was sealed with nail polish.</p><p>The following primary antibodies were used at the indicated dilutions in PBST with 0.5% BSA: guinea pig anti-HIM-8 (1&#8758;500) <xref ref-type="bibr" rid="pgen.1003674-Phillips1">[16]</xref>, rabbit anti-HIM-3 (1&#8758;200) <xref ref-type="bibr" rid="pgen.1003674-Zetka1">[17]</xref>, guinea pig anti-SYP-1 (1&#8758;200) <xref ref-type="bibr" rid="pgen.1003674-MacQueen2">[18]</xref>, rabbit anti-RAD-51 (1&#8758;500) <xref ref-type="bibr" rid="pgen.1003674-Colaiacovo1">[21]</xref> , guinea pig anti-SUN1 S8P (1&#8758;1000) <xref ref-type="bibr" rid="pgen.1003674-Penkner1">[23]</xref>, rabbit anti-DSB-2 (1&#8758;5000), rat anti-RAD-51 (1&#8758;250), guinea pig anti-DSB-1 (1&#8758;500) <xref ref-type="bibr" rid="pgen.1003674-Stamper1">[11]</xref>.</p><p>An affinity-purified rabbit polyclonal antibody against DSB-2 was generated by SDIX (Newark, DE) using the C-terminal 100 amino acids of F26H11.6 as the immunogen. Specificity of the antibody was demonstrated both by the lack of chromatin staining in immunofluorescence analysis of <italic>dsb-2</italic> mutant gonads (<xref ref-type="fig" rid="pgen-1003674-g006">Figure 6A</xref>) and by Western blot analysis (<xref ref-type="fig" rid="pgen-1003674-g006">Figure 6B</xref>).</p><p>Rat anti-RAD-51 antibody was generated using a His-tagged fusion protein expressed from plasmid pET28a containing the entire RAD-51S coding sequence <xref ref-type="bibr" rid="pgen.1003674-Rinaldo1">[56]</xref>; immunizations and bleeds were performed by SDIX. Rat anti -RAD-51 was affinity purified against membrane-bound protein as described in <xref ref-type="bibr" rid="pgen.1003674-Koelle1">[57]</xref> with the following modifications: nitrocellulose membrane was blocked in 5% milk in 1&#215;TBST; and, eluates containing rat anti -RAD-51 were further purified by dialysis with 12&#8211;14 kDa dialysis tubing (Spectrum) in 1&#215;TBST for 1 hour and overnight at 4&#176;C. Specificity was demonstrated by showing that rat anti-RAD-51 foci colocalize with rabbit anti-RAD-51 foci <xref ref-type="bibr" rid="pgen.1003674-Colaiacovo1">[21]</xref> by immunofluorescence and that these recombination-dependent foci are eliminated in <italic>spo-11(me44)</italic> gonads.</p><p>All secondary antibodies were Alexa Fluor goat from Invitrogen used at 1&#8758;200 dilution in PBST with 0.5% BSA.</p><p>Immunofluorescence images were acquired using the DeltaVision microscopy system (Applied Precision) and deconvolved using softWoRx software. Images shown are maximum-intensity projections of Z-stacks acquired at 0.3 &#181;m intervals.</p><p>For each wild-type germ line evaluated, RAD-51 foci were quantified in 8 contiguous rows of pachytene nuclei from the region where foci were most abundant. The average distance (in rows of nuclei) between the position of this peak and the end of the transition zone was calculated for wild-type germ lines, and this distance was used to define the corresponding regions to be scored in <italic>dsb-2(me96)</italic> mutant germ lines (in which the abundance of RAD-51 foci was low throughout). Quantitation was carried out on deconvolved 3D image stacks using SoftWoRx software; only nuclei that were completely contained with in the image stack were scored. Occasional atypical nuclei with condensed, bright DAPI signals were excluded. Numbers of nuclei scored: WT, n&#8202;=&#8202;335; <italic>dsb-2</italic>, n&#8202;=&#8202;196.</p><p>For each genotype, sixty adult worms (24 hours post-L4) were picked into M9+0.05% Tween 20, washed gently three times, then lysed by resuspension in 2&#215; Laemmli Sample Buffer (Bio-Rad). Gel electrophoresis was performed on a 4&#8211;15% Criteriot TGX gradient gel (Bio-Rad), followed by transfer of proteins to a PVDF membrane. Blots were probed with rabbit anti-DSB-2 (1&#8758;1000 in 5% milk) for 2 hours, followed by HRP-conjugated secondary antibody and detection by ECL.</p><p>Worms were exposed to 1 kRad (10Gy) of gamma-irradiation using a Cs-137 source. The 1 kRad dose was chosen based on its sufficiency to restore chiasmata to 95% of chromosome pairs in affected nuclei of the <italic>spo-11(ok79)</italic> mutant <xref ref-type="bibr" rid="pgen.1003674-Hayashi1">[6]</xref>. Worms were irradiated at 36 hours post L4, and the number of DNA bodies at diakinesis was assessed in worms fixed at 18 hours post-irradiation, for both <italic>dsb-2</italic> and age-matched <italic>spo-11</italic> mutants. The <italic>dsb-2</italic> worms also carried the <italic>rol-1</italic> marker, which does not affect meiosis. This assay tends to underestimate the incidence of achiasmate chromosomes, as some lie too close together to be resolved. Numbers of nuclei scored were: 71 and 76 for <italic>dsb-2</italic> worms, untreated and irradiated respectively; 76 and 45 for <italic>spo-11</italic> worms, untreated and irradiated respectively.</p><p>Worms were exposed to 5 kRad (50Gy) of gamma-irradiation using a Cs-137 source. Formation of RAD-51 foci was assessed by immunofluorescence in gonads dissected and fixed 1 hour after irradiation. Germ lines from <italic>rad-50</italic> and <italic>htp-1; rad-50</italic> mutants were irradiated at 24 hours post-L4, and stained with DAPI, RAD-51 antibody and DSB-2 antibody. Germ lines from <italic>dsb-2</italic> mutants were irradiated at 48 hours post-L4 and stained with DAPI and RAD-51 antibody.</p>