<p>
<italic>E. coli</italic> and <italic>V. cholerae</italic> strains, and plasmids used in this study are listed in <xref ref-type="supplementary-material" rid="pgen.1004184.s014">Table S3</xref>.</p><p>ChIP-chip assay was performed exactly as described <xref ref-type="bibr" rid="pgen.1004184-VenkovaCanova3">[38]</xref>. Cells used here were <italic>V. cholerae</italic> WT (CVC209) and MCH1 (CVC2099). As before, ChIP (Cy5) signals were divided by corresponding input DNA (Cy3) from three independent experiments. The difference in mean Cy5/Cy3 signals between the WT and MCH1 strains was plotted in <xref ref-type="fig" rid="pgen-1004184-g001">Figure 1</xref>.</p><p>The copy number of p<italic>oriII</italic> was measured using a three-plasmid system as described <xref ref-type="bibr" rid="pgen.1004184-VenkovaCanova1">[26]</xref>. <italic>E. coli</italic> (BR8706) cells harboring the p<italic>oriII</italic> (pTVC25, 31, 35, 210 or 524) and <italic>prctB</italic> (pTVC11) that supplies the initiator from an arabinose inducible promoter, was transformed with a pBR322-derived plasmid containing one of the newly identified RctB binding sites. The plasmid copy number was determined by harvesting overnight grown cells directly from transformation plates containing either 0.002 or 0.2% arabinose, the conditions referred to in the text as supplying RctB at low or high levels, respectively.</p><p>EMSA was performed as described <xref ref-type="bibr" rid="pgen.1004184-VenkovaCanova4">[57]</xref>. A total of 1.2 pmol of DNA fragment, carrying one copy of a binding site with 100 bp of vector sequences (pTVC243) at both flanks, was end-labeled with <sup>32</sup>P and reacted with RctB at two different concentrations (2 and 20 nM). The binding was monitored using a 5% polyacrylamide gel and autoradiography. For competitive binding assay (<xref ref-type="supplementary-material" rid="pgen.1004184.s004">Figure S4</xref>), DNA was not radiolabeled and visualized after staining with SYBR Green EMSA nucleic acid gel stain at 10,000&#215; dilution for 30 min at room temperature (Molecular Probes).</p><p>The binding of RctB was performed under the same condition as was used for EMSA. The binding reactions were treated with 0.01 unit of DNase I (Promega) for 1 min at room temperature. DNaseI was inactivated by adding 15 mM EDTA followed by heating (95&#176;C, 10 min). The DNA was purified using Qiaquick PCR purification kit (Qiagen). Primer extension was performed using 6 FAM-labeled primer and Thermo Sequenase polymerase (USB) according to the manufacturer's protocol. Amplified products were purified using Qiaquick PCR purification kit. The purified products along with GeneScan 500-ROX size standard (Applied Biosystems) were denatured by adding Hi-Di formamide (Applied Biosystems) followed by heating (75&#176;C, 10 min) and immediately chilled on ice. The single stranded DNA was analyzed on an ABI 3130&#215;l Genetic Analyzer (Applied Biosystems) and data was analyzed using GeneMapper (v. 3.7) software.</p><p>The deletion was achieved in two steps. First, the chrI-4 sequence was substituted with an FRT-Zeo-FRT cassette by the allele-exchange method <xref ref-type="bibr" rid="pgen.1004184-Demarre1">[29]</xref>. Second, the Zeo cassette was excised, leaving one FRT site in place of the chrI-4 sequence. To provide homology for the allele exchange, &#8764;one kb natural flank of chrI-4 followed by a FRT site was cloned upstream of the Zeo cassette in pEM7-Zeo vector that resulted in pBJH242. Likewise, another FRT site followed by &#8764;one kb of natural sequences at the other flank chrI-4 was cloned downstream of the Zeo cassette of pBJH242 that resulted in pBJH245. The cloned region of pBJH245 (upstream flank of chrI-4-FRT-Zeo cassette-FRT-downstream flank of chrI-4) was amplified by PCR and the linear product was introduced by natural transformation to CVC1121, a <italic>hapR</italic>+ derivative of N16961. The transformants were selected for Zeocin resistance, and the replacement of the chrI-4 sequences by the Zeo cassette was confirmed by PCR amplification of the region and sequencing of the PCR product. The Zeo cassette of the resulting &#916;<italic>chrI-4</italic>::FRT-Zeo-FRT strain (CVC2540) was excised by further transformation with an unstable plasmid supplying the Flp recombinase (pBLO1218). The plasmid, when not selected, was lost spontaneously from the final strain &#916;<italic>chrI-4</italic>::FRT (CVC2542). The desired deletion/substitution was confirmed by PCR amplifying the mutated region and DNA sequencing. The same strategy was used to make &#916;<italic>chrII-10</italic>::FRT-Zeo-FRT strain (CVC2565). The fragment used for natural transformation was a PCR product that had &#8764;one kb homology to the left flank of chrII-10 (coord. 1026995&#8211;1027995)-FRT-Zeo-FRT- &#8764;one kb homology to the right flank of chrII-10 (coord. 1028034&#8211;1029033).</p><p>Frequencies of four markers, <italic>oriI</italic>, <italic>oriII</italic>, <italic>terI</italic> and <italic>terII</italic>, in exponentially growing cells in L broth were determined by qPCR using a PTC-200 Peltier Thermal Cycler (MJ Research) and a LightCycler 480 SYBR Green I Master (Roche) mix, as described <xref ref-type="bibr" rid="pgen.1004184-Demarre1">[29]</xref>.</p><p>Fragments containing putative promoters were cloned into a promoter-less <italic>lacZ</italic> containing plasmid (pMLB1109). The resultant plasmids were used to transform <italic>E. coli</italic> BR8706 carrying p<italic>rctB</italic> (pTVC11). Cells harboring these two plasmids were cultivated in L broth containing 0, 0.002 or 0.2% arabinose at 37&#176;C to exponential phase and <italic>&#946;</italic>-galactosidase activity was measured, as described <xref ref-type="bibr" rid="pgen.1004184-Miller1">[58]</xref>.</p><p>To localize <italic>oriI</italic> and <italic>oriII</italic>, P1<italic>parS</italic> as a P1<italic>parS</italic>-Km cassette and pMT<italic>parS</italic> as a pMT<italic>parS</italic>-Sp cassette were integrated at approximately +135 kb region on chrI and +40 kb region on chrII, respectively <xref ref-type="bibr" rid="pgen.1004184-Miller1">[58]</xref>. Both cassettes were inserted into WT (CVC1121) and &#916;<italic>chrI-4</italic> (CVC2542) strains, and the resulting strains (CVC2553 and CVC2554) were transformed with a plasmid expressing mCherry-pMT<italic>parB</italic> and <italic>gfp</italic>-P1<italic>parB</italic> (pRN010). Cells were grown in M9+glycerol medium at 37&#176;C to log phase and observed under a fluorescence microscope, as described <xref ref-type="bibr" rid="pgen.1004184-Miller1">[58]</xref>. Locations of fluorescent foci were measured using the ImageJ (rsb.info.nih.gov/ij/index.html).</p>