<p>All procedures were approved by the Tel Aviv University Animal Care Committee and conducted in accordance with the council for experiments on animal subjects, Ministry of Health, Israel.</p><p>The experimental procedure for the DNA microarrays experiment was performed as follows. Adult (0.5&#8211;1.5 years old) transgenic zebrafish, <italic>Tg(aanat2:EGFP)<sup>Y8</sup></italic>, which express enhanced green fluorescent protein (EGFP) in the pineal gland under the control of the <italic>aanat2</italic> regulatory regions, were used <xref ref-type="bibr" rid="pgen.1003116-Gothilf1">[29]</xref>. Fish were raised under 12-hr light&#8758;12-hr dark (LD) cycles, in a temperature controlled room, and transferred to constant darkness (DD) for tissue collection. Fish were anesthetized in 1.5 mM Tricane (Sigma), sacrificed by decapitation, and pineal glands were removed under a fluorescent dissecting microscope. Starting from circadian time (CT) 14, pineal glands were collected at 4-hr intervals for 48 hours (12 time points identified as CT 14, 18, 22, 2, 6, 10, 14b, 18b, 22b, 2b, 6b and 10b). Pools of 12 (DNA microarray) or 20 (RNA-Seq) pineal glands were prepared at each time-point and total RNA was extracted using the RNeasy Lipid Tissue Mini Kit (QIAGEN), according to the manufacturer's instructions.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Labeled RNA preparation and hybridization to DNA microarrays were performed according to the Affymetrix manual with the two-cycle target labeling protocol (<ext-link ext-link-type="uri" ns0:href="http://www.affymetrix.com/support/downloads/manuals/expression_analysis_technical_manual_.pdf">http://www.affymetrix.com/support/downloads/manuals/expression_analysis_technical_manual_.pdf</ext-link>). A total of 12 Affymetrix DNA microarrays were hybridized with RNA-pools of pineal glands from 12 time points throughout two daily cycles. Each DNA microarray was normalized using Affymetrix GeneChip Operating Software (GCOS). The entire DNA microarray dataset, logarithmically transformed, was normalized using quantile normalization to guarantee that the distribution of probe intensities was the same in all the chips <xref ref-type="bibr" rid="pgen.1003116-Bolstad1">[59]</xref>. The microarray data was deposited to the Gene Expression Omnibus (GEO), under accession GSE41696.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Illumina TruSeq protocol was used to prepare libraries from RNA samples. Overall, 12 libraries (12 time points) were run on 2 lanes of Illumina HiSeq2000 machine using the multiplexing strategy of the TruSeq protocol (<italic>Institute of Applied Genomics</italic>). On average, &#8764;30 million paired-end reads were obtained for each library. The reads were 2&#215;100 base pairs for 8 time points (CTs 22, 2, 6, 10, 18b, 2b, 6b, 10b) and 2&#215;50 base pairs for the remaining time points (CTs 14, 18, 14b and 22b). TopHat <xref ref-type="bibr" rid="pgen.1003116-Trapnell1">[60]</xref> was used for aligning the reads against the zebrafish genome allowing only uniquely aligned reads and up to two mismatches per read. On average, 56% of the reads had unique alignment to the zebrafish genome. Reads aligned to the protein coding regions of known RefSeq genes were used. A custom script written in Perl was used to parse the output of TopHat, which is given in Sequence Alignment/Map (SAM) format (<ext-link ext-link-type="uri" ns0:href="http://samtools.sourceforge.net/">http://samtools.sourceforge.net/</ext-link>), and to convert it into raw number of reads aligned to each position in each RefSeq gene. The RefSeq genes information was obtained from the Table Browser of the UCSC genome browser (genome.ucsc.edu/) using the zebrafish Jul. 2010 (Zv9/danRer7) assembly. To avoid PCR duplicates, only paired-end reads that have unique start position in the genome in both pairs were used <xref ref-type="bibr" rid="pgen.1003116-Levin1">[61]</xref>.</p><p>The quality of the sequencing libraries was assessed as described in Levin et al. <xref ref-type="bibr" rid="pgen.1003116-Levin1">[61]</xref> and the data was normalized using Quantile normalization (<xref ref-type="supplementary-material" rid="pgen.1003116.s015">Text S1</xref>). We made sure that the normalization scheme properly corrects for different RNA levels and other technical differences between samples (<xref ref-type="supplementary-material" rid="pgen.1003116.s015">Text S1</xref>). The sequencing data was deposited to the Sequence Read Archive (SRA), under accession SRA054264.</p><p>The time-dependent signal was converted into a frequency-dependent signal using the Fast Fourier Transform (FFT). The extent to which the original signal is circadian was quantified by the ratio (&#8216;g-factor&#8217;) of the power (squared amplitude) of the frequency which corresponds to 24 hr period to the sum of powers of all frequencies <xref ref-type="bibr" rid="pgen.1003116-Levy1">[23]</xref>. The higher the g-factor, the higher is the confidence that the transcript is circadian. We note that changing the definition of the g-factor by adding the powers of higher harmonics of the 24 hr period to the numerator, gave similar results compared to the use of the definition above. To determine the true-positive rate for a list of transcripts constructed using a given g-factor cut-off, permutation analysis was conducted as follows:</p><p>The time-dependent signals of each transcript were randomly shuffled.</p><p>FFT was performed and g-factor was calculated for each transcript.</p><p>The cumulative histogram of the g-factor values (ranging from 0 to 1) was calculated, resulting in the number of transcripts whose calculated g-factor is larger than a given value (&#8216;random detection function&#8217;).</p><p>Steps (1)&#8211;(3) were repeated a thousand times and the averaged random detection function was calculated. This function estimates the number of false-positive detections of circadian genes for any given cutoff value of the g-factor.</p><p>Steps (2)&#8211;(3) were performed on the original data set. The resulting function, providing the number of transcripts exhibiting a g-factor larger than a given value in the real data, was termed the &#8216;detection function&#8217;.</p><p>For any given choice of cutoff for the g-factor, the difference between the detection function and the average random detection function estimates the number of true-positives in the list of transcripts constructed with this cutoff.</p><p>Finally, using the number of transcripts detected for a given g-factor (step 5) and the number of true-positives for a given g-factor (step 6), the true-positive rate as a function of the number of transcripts detected was calculated and further used to identify circadian genes with high accuracy (<xref ref-type="supplementary-material" rid="pgen.1003116.s008">Figure S8</xref>). The procedure described here was implemented using in-house MATLAB (The Mathworks, Inc.) script.</p><p>The 308 circadian RefSeq genes identified using the RNA-seq were analyzed to find over-represented molecular functions (<xref ref-type="supplementary-material" rid="pgen.1003116.s010">Table S2</xref>), using the DAVID bioinformatics tools <xref ref-type="bibr" rid="pgen.1003116-Huang1">[28]</xref> and focusing on over-represented gene ontology (GO) categories and KEGG pathways <xref ref-type="bibr" rid="pgen.1003116-Ashburner1">[62]</xref>, <xref ref-type="bibr" rid="pgen.1003116-Kanehisa1">[63]</xref>. The DAVID's default zebrafish genes background was used. All the significantly enriched (Benjamini-Hochberg adjusted P-value&lt;0.05) GO categories and KEGG pathways are presented in <xref ref-type="supplementary-material" rid="pgen.1003116.s013">Table S5</xref>.</p><p>The temporal expression pattern of candidate genes was determined by whole mount ISH in zebrafish larvae or by quantitative RT-PCR in the adult pineal gland as previously described <xref ref-type="bibr" rid="pgen.1003116-Vatine2">[12]</xref>, <xref ref-type="bibr" rid="pgen.1003116-Ziv2">[27]</xref> (<xref ref-type="supplementary-material" rid="pgen.1003116.s001">Figure S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1003116.s015">Text S1</xref>).</p><p>Morpholino experiments were conducted as previously described <xref ref-type="bibr" rid="pgen.1003116-Ziv1">[11]</xref> (<xref ref-type="supplementary-material" rid="pgen.1003116.s015">Text S1</xref>).</p><p>Rescue experiments were conducted by co-injection of approximately 2 nl volume <italic>of camk1gb</italic> morpholino (1 mM) and <italic>in vitro</italic> transcribed <italic>camk1gb</italic> mRNA (100 ng/&#181;l). The <italic>camk1gb</italic> protein-coding sequence was PCR-amplified with a KAPA HIFI PCR kit (KAPA Biosystems) using the same set of primers that was used for ISH experiments. The PCR products were cloned into a pCS2<sup>+</sup> vector, linearized with <italic>Not</italic>I restriction enzyme and transcribed using the SP6 mMessage mMachine kit (Ambion), according to the manufacturer's instructions, to generate capped <italic>camk1gb</italic> mRNA.</p><p>Embryos were microinjected with either control morpholino, <italic>camk1gb</italic> morpholino or co-injected with <italic>camk1gb</italic> morpholino and <italic>in vitro</italic> transcribed <italic>camk1gb</italic> mRNA and kept under LD conditions for 3 days. On the fourth day post-fertilization, embryos were placed in 48-well plates in the observation chamber of the DanioVision Tracking System (Noldus Information Technology) and exposed, for acclimation, to two days under 12-hr light (3400 lux)&#8758;12-hr dim light (40 lux) regime followed by 3 days of constant dim light. Live video tracking and analysis was conducted using the Ethovision 8.0 software (Noldus Information Technology). Activity was measured at days 6&#8211;8 post fertilization, as the distance moved by a larva in 10 min time bins (<xref ref-type="fig" rid="pgen-1003116-g004">Figure 4</xref>). The activity record of each individual was subjected to Fourier analysis, and scored with a g-factor (see <xref ref-type="sec" rid="s4">Methods</xref> section &#8216;Fourier analysis&#8217;). Significant differences in the g-factor distributions between the control and <italic>camk1gb</italic> morpholino treated groups (n&#8202;=&#8202;75; combining 4 different experiments) as well as between the rescue and <italic>camk1gb</italic> morpholino treated groups (n&#8202;=&#8202;16) were determined by the Kolmogorov-Smirnov test (<xref ref-type="fig" rid="pgen-1003116-g004">Figure 4b</xref> and <xref ref-type="fig" rid="pgen-1003116-g004">Figure 4d</xref>). The percent of larvae which are considered circadian depends on the g-factor used as the detection criteria, but for all values of g-factor cutoff tested (ranging between 0.05 and 0.3, <xref ref-type="fig" rid="pgen-1003116-g004">Figure 4d</xref>), significantly more larvae are considered circadian in the control group.</p><p>To determine whether <italic>camk1gb</italic> knockdown impairs larval movement abilities, locomotor activity levels were tracked under abrupt light to dark transitions <xref ref-type="bibr" rid="pgen.1003116-Burgess1">[34]</xref>. On day 6 post fertilization, control morpholino and <italic>camk1gb</italic> morpholino injected larvae (n&#8202;=&#8202;24) were subjected to 3 dark flashes of 5 sec each during the light phase <xref ref-type="bibr" rid="pgen.1003116-Burgess1">[34]</xref>. Activity was measured as the distance moved by each larva during the dark flash. No statistical difference was observed between the activity of the control morpholino and <italic>camk1gb</italic> morpholino injected groups (Student's <italic>t</italic>-test, P-value&gt;0.2; <xref ref-type="supplementary-material" rid="pgen.1003116.s004">Figure S4</xref>), indicating that the <italic>camk1gb</italic> morpholino does not impair larval movement abilities.</p><p>Transient co-transfection of the Pac-2 cell line with <italic>camk1gb</italic> and <italic>per1b</italic>:luciferase constructs was performed as previously described <xref ref-type="bibr" rid="pgen.1003116-Cavallari1">[64]</xref> (<xref ref-type="supplementary-material" rid="pgen.1003116.s006">Figure S6</xref> and <xref ref-type="supplementary-material" rid="pgen.1003116.s015">Text S1</xref>).</p><p>For double fluorescence ISH we followed the protocol of Machluf and Levkowitz <xref ref-type="bibr" rid="pgen.1003116-Machluf1">[65]</xref> (<xref ref-type="supplementary-material" rid="pgen.1003116.s015">Text S1</xref>).</p>