<p>In order to keep our problem simple, we condition on two important parameters: We assume that the exact site under selection is known from extraneous information, and we furthermore assume that the allele frequency <italic>f</italic>
<sub>cur</sub> of that site at the time of sampling, <italic>t</italic>
<sub>cur</sub>&#8202;=&#8202;0 is known. The interpretation of the parameters is depicted graphically in <xref ref-type="fig" rid="pgen-1003011-g001">Figure 1a</xref>.</p><p>Unless noted otherwise, we assume a panmictic diploid population of size <italic>N</italic> with an additive selection model where the ancestral homozygous, heterozygous and derived homozygous genotypes have fitness 1, 1+ <italic>s/2</italic> and 1 + <italic>s</italic>, respectively. However, the methodology applied here can easily be adapted to more complex scenarios, e.g. models involving multiple populations, more sophisticated demographic models, and other models of selection. For most simulated data sets, we will report the population scaled mutation rate <italic>&#945;&#8202;=&#8202;</italic>4<italic>Ns</italic>, as the shape of the allele frequency trajectory depends only on that compound parameter <xref ref-type="bibr" rid="pgen.1003011-Ewens1">[25]</xref>. However, for most of the genes we analyze previous estimates were made directly on <italic>s</italic> rather than the compound parameter. To facilitate comparisons, we report <italic>s</italic> for the genes we analyzed.</p><p>The sweep from a <italic>de novo</italic> mutation (SDN) models a single selective sweep and has two parameters: the mutation rate <italic>&#956;</italic>, and the selection coefficient <italic>s</italic>. For all simulations, we follow <xref ref-type="bibr" rid="pgen.1003011-Itan1">[80]</xref> and record the time <italic>t</italic>
<sub>0</sub> when the mutation arose, as <italic>t</italic>
<sub>0</sub> depends stochastically on <italic>s</italic>. The prior distributions we use for this model were <italic>&#956;</italic> &#8764; U(0.5e-8,6e-8) and log<sub>10</sub>(<italic>s</italic>) &#8764; U(&#8722;3,&#8722;0.5), where U is a uniform distribution.</p><p>The sweep from standing variation (SSV) model is identical to the <italic>de novo</italic> mutation model, with the exception that we define a frequency <italic>f</italic>
<sub>1</sub> at which the mutation becomes selected. Unless noted otherwise, the priors for <italic>&#956;</italic> and <italic>s</italic> are the same as in the SDN model, and the prior for <italic>f</italic>
<sub>1</sub> is <italic>f</italic>
<sub>1</sub> &#8764; U(0,0.2). In addition to <italic>t</italic>
<sub>0</sub>, which is defined analogous to the SDN model, we are also interested in <italic>t</italic>
<sub>1</sub>, the time when the mutation becomes selectively advantageous (i.e. the time when the mutation reaches frequency <italic>f</italic>
<sub>1</sub>).</p><p>We also consider a neutral model (NT), without any selection. The only free parameter in this model is the mutation rate <italic>&#956;</italic>, with the same prior distribution as described under SDN model. As under the selection model, however, we still condition on one site having reached a final allele frequency of <italic>f</italic>
<sub>cur</sub>, so this model does not correspond to the classical neutral coalescent.</p><p>We use a standard ABC approach <xref ref-type="bibr" rid="pgen.1003011-Tavar1">[31]</xref>, <xref ref-type="bibr" rid="pgen.1003011-Beaumont1">[32]</xref>, using a post-sampling adjustment in the form of a GLM <xref ref-type="bibr" rid="pgen.1003011-Leuenberger1">[96]</xref>. We used the <italic>ABCToolbox</italic> package <xref ref-type="bibr" rid="pgen.1003011-Wegmann2">[40]</xref>, for specifying priors, rejection sampling and post-sampling adjustment. Unless specified otherwise, we perform 10<sup>5</sup> simulations per model, and retained the 100 (0.1%) simulations with associated Euclidean distance between observed and simulated summary statistics closest to zero. To assess how the number of simulations and acceptance rates influence our results, we analyze 10,000 random data sets with up to 10<sup>7</sup> simulations and varying acceptance rates. We show that these parameters have very little impact on the relative error for both the model choice and parameter estimates in <xref ref-type="supplementary-material" rid="pgen.1003011.s009">Table S1</xref>.</p><p>We use a diverse array of summary statistics, with the goal of maximizing the information captured, while not incluing any statistics that just add noise. The statistics we used may be broadly classified into statistics based on haplotype patterns, and statistics based on the site frequency spectrum. The haplotype based statistics we used were iHS <xref ref-type="bibr" rid="pgen.1003011-Voight1">[19]</xref> and EHH <xref ref-type="bibr" rid="pgen.1003011-Sabeti1">[9]</xref>. We recorded EHH in a 10 kb, 20 kb and 50 kb window, centered on the selected site. For the SFS based statistics, we used Tajima's D <xref ref-type="bibr" rid="pgen.1003011-Tajima1">[20]</xref>, Fay &amp; Wu's H <xref ref-type="bibr" rid="pgen.1003011-Fay1">[23]</xref>, the average number of pairwise differences <italic>&#960;</italic>, and the number of segregating sites <italic>S</italic> as statistics. All these statistics are calculated for three regions: A central region of 20 kb around the selected site, an intermediate region consisting of all sites 20&#8211;50 kb away from the selected site, and a faraway region consisting of all sites further than 50 kb away from the selected site. Following <xref ref-type="bibr" rid="pgen.1003011-Wegmann3">[97]</xref>, we linearize all statistics using a Box-Cox-transformation <xref ref-type="bibr" rid="pgen.1003011-Box1">[98]</xref>. To choose a set of informative summary statistics, we used a Partial Least Squares Discriminant Analysis (PLS-DA) <xref ref-type="bibr" rid="pgen.1003011-Tenenhaus1">[99]</xref>, <xref ref-type="bibr" rid="pgen.1003011-LCao1">[100]</xref>. PLS-DA is a variant of Partial Least Squares regression, that, similarly to principal component analysis, extracts orthogonal components from a high-dimensional data set (in this case the summary statistics). In contrast to PCA, in PLS-DA these components are chosen such that the covariance between summary statistics and models is maximized <xref ref-type="bibr" rid="pgen.1003011-Boulesteix1">[see e.g. 101]</xref>. We did our computations using the &#8216;plsda&#8217; function of the mixOmics package for R, and kept the five first PLS-DA axes <xref ref-type="bibr" rid="pgen.1003011-LCao1">[100]</xref>.</p><p>ABC has two crucial parameters independent of the model it is applied to: The number of simulations <italic>n</italic>
<sub>S</sub> and the acceptance rate <italic>&#949;</italic>. To assess the effect of these parameters on inference, we calculate the accuracy of our model choice estimates for various values of <italic>n</italic>
<sub>S</sub> and <italic>&#949;</italic> (<xref ref-type="supplementary-material" rid="pgen.1003011.s009">Table S1</xref>).</p><p>All our data sets used for both the ABC inference and the assessment of our procedure are simulated using a modified version of the coalescent simulator <italic>mbs</italic>
<xref ref-type="bibr" rid="pgen.1003011-Teshima2">[102]</xref>. <italic>Mbs</italic> allows simulation of genetic data sets with a single selected site using the structured coalescent <xref ref-type="bibr" rid="pgen.1003011-Hudson2">[26]</xref>. <italic>mbs</italic> first simulates the allele frequency trajectory of the site under selection, and then generates a data set conditional on that trajectory. We simulate allele frequency trajectories using Euler's method on the unscaled backwards diffusion equation with selection (eq 7.1 in <xref ref-type="bibr" rid="pgen.1003011-Kimura4">[103]</xref>). This equation makes it very easy to incorporate population size changes by just changing the variance term. To simulate sweeps from standing variation, we set the selection coefficient (<italic>s</italic>) to zero the first time the trajectory reaches <italic>f</italic>
<sub>1</sub>. To analyze simulated data sets, we generally simulate a 100 kb region with a recombination rate of 1.5 cM/Mb. For the human genes, we simulate the gene and a 50 kb flanking region on both sides, resulting in regions that are usually between 100 kb and 150 kb wide. Recombination rates and hotspots are modeled by using the HapMap recombination map <xref ref-type="bibr" rid="pgen.1003011-Myers1">[104]</xref> in the application to selected genes. For all simulated data sets we assume a constant-sized population. For the analysis of human genes, we use the population history estimated by <xref ref-type="bibr" rid="pgen.1003011-Li3">[105]</xref>. Specific regions and details of the used regions are given in <xref ref-type="supplementary-material" rid="pgen.1003011.s010">Table S2</xref>. To ensure that our method does not suffer from a high false positive rate, we also analyze regions 5 Mb downstream from the candidate genes, as they are presumably neutral. For three of genes (ASPM, G6PD, and PSCA), no data was available for these downstream regions, so we analyzed the remaining loci. Candidate loci for selection were chosen using the following criteria: i) they were required to have a derived allele frequency between 0.7 and 0.9 and ii) to be as closely to 5 Mb away from the actual candidate locus in the upstream gene as possible.</p><p>We estimate parameters from our models using the standard ABC procedure described above. The parameters we estimate are the mutation rate <italic>&#956;</italic>, the age of the sweep <italic>t</italic>
<sub>1</sub> the selection coefficient <italic>s</italic> and, only under the SSV model, the initial frequency <italic>f</italic>
<sub>1</sub> for the SSV model. In particular we want to determine if our posteriors are unbiased, and if we were able to get reasonable confidence in our estimates. To do this, we simulate data sets with fixed parameters and plotted the average posterior distribution for all parameters in <xref ref-type="fig" rid="pgen-1003011-g002">Figure 2</xref>.</p><p>For model choice, our main goal is to calculate the relative probabilities of the models given the data, i.e. Pr(SSV | data), Pr(SDN | data) and Pr(NT | data), which we calculate using the marginal densities as proposed by <xref ref-type="bibr" rid="pgen.1003011-Leuenberger1">[96]</xref>. To identify parameter regions where there is power to distinguish between the models, we simulate 1,000 data sets each under 30 different scenarios in three series, corresponding to three parameters of interest: The strength of selection <italic>&#945;</italic>, the frequency when the mutation became selective advantageous <italic>f</italic>1 and <italic>f</italic>
<sub>cur</sub>, the frequency at which the mutation is observed.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To test the algorithm for approximating Bayes factors, we also use a simulation approach. The estimator of the posterior probability from <italic>k</italic> simulations, <inline-formula><inline-graphic ns0:href="pgen.1003011.e001.jpg" /></inline-formula>, should have the property <inline-formula><inline-graphic ns0:href="pgen.1003011.e002.jpg" /></inline-formula> where <italic>m</italic> is a model indicator functions for a specific model. Also, for a particular draw from the posterior, <italic>m</italic>
<sup>(0)</sup>, we expect <inline-formula><inline-graphic ns0:href="pgen.1003011.e003.jpg" /></inline-formula>, if the simulation algorithm works properly. In other words, <inline-formula><inline-graphic ns0:href="pgen.1003011.e004.jpg" /></inline-formula>, should asymptotically equal <inline-formula><inline-graphic ns0:href="pgen.1003011.e005.jpg" /></inline-formula>, i.e. if <inline-formula><inline-graphic ns0:href="pgen.1003011.e006.jpg" /></inline-formula>&#8202;=&#8202;<italic>c</italic>, we expect a proportion <italic>c</italic> of simulations to have been obtained from model <italic>m</italic>. Equivalently, for an estimated log Bayes factor, log<sub>10</sub>&#8202;=&#8202;<italic>c</italic>, we expect a proportion 10<italic><sup>c</sup></italic>/(1+10<italic><sup>c</sup></italic>) of draws to be from model <italic>m</italic>. This prediction is tested in <xref ref-type="supplementary-material" rid="pgen.1003011.s008">Figure S8</xref>, based on 10,000 random data sets from both the SDN and SSV model.</p>