<p>We refer to the standard laboratory yeast sequence <xref ref-type="bibr" rid="pgen.1003257-Wilson1">[62]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1003257-Kushnirov3">[64]</xref> as the &#8216;reference sequence&#8217;. Two common sequence polymorphs found within the wild population were used. The first, with deletion of 19 amino acids from residues 66 to 84 and the G162D change, is referred to as &#916;19, and the other includes N109S, G162D, D169E, P186A, T206K, H225D and is denoted E9 <xref ref-type="bibr" rid="pgen.1003257-Bateman1">[20]</xref>. A prion originating with the Sup35p sequence of strain E9, for example, but being propagated in a strain expressing only the reference sequence will be designated [<italic>PSI</italic>
<sup>+</sup>E9]ref, in analogy with similar nomenclature for [URE3] <xref ref-type="bibr" rid="pgen.1003257-Edskes1">[31]</xref>. Cytoductants (see below) generated with strain A as donor and strain B as recipient are denoted A&#8594;B. They have the nuclear genotype of strain B and the cytoplasmic genotype of both A and B. In an abuse of language, we often use &#8220;[<italic>PSI</italic>
<sup>+</sup>E9]ref was transferred to Sup35 &#916;19&#8221; to mean &#8220;[<italic>PSI</italic>
<sup>+</sup>E9]ref was transferred to cells expressing Sup35 &#916;19&#8221;.</p><p>Sup35p is a subunit of the translation termination complex, and the incorporation of a large proportion of Sup35p into the prion amyloid filaments makes it inactive, resulting in increased read-through of termination codons. This is measured by read-through of <italic>ade2-1</italic>, with an ochre termination codon in the middle of the <italic>ADE2</italic> gene. In addition to <italic>ade2-1</italic>, strains carry the <italic>SUQ5</italic> weak suppressor mutation, which leaves cells Ade- unless the [<italic>PSI</italic>
<sup>+</sup>] prion is also present <xref ref-type="bibr" rid="pgen.1003257-Cox1">[2]</xref>.</p><p>The strains used are listed in <xref ref-type="supplementary-material" rid="pgen.1003257.s002">Table S1</xref>. Plasmids used containing reference, &#916;19 or E9 sequences were generated as described <xref ref-type="bibr" rid="pgen.1003257-Bateman1">[20]</xref>. All yeast media and plates contained 20 &#181;M copper sulfate unless noted. Rich and minimal media (YPAD and SD) are as described <xref ref-type="bibr" rid="pgen.1003257-Sherman1">[65]</xref>. Only nutrients required by the strains used in a given experiment were added to minimal plates.</p><p>Cytoplasm may be transferred from one strain to another utilizing the <italic>kar1-1</italic> mutation <xref ref-type="bibr" rid="pgen.1003257-Conde1">[66]</xref>, defective for nuclear fusion. Cells fuse, but the nuclei do not fuse, and nuclei separate at the next cell division. However, cytoplasmic mixing has occurred, and so a genetic element (prion or mitochondrial DNA) present in one strain (identified by its nuclear genotype) will be transferred to the other. We use transfer of mitochondrial DNA as a marker of cytoplasmic transfer, and score prion transfer. Reference, &#916;19 or E9 sequence plasmids were transformed into both laboratory strains 4828 and 4830, loss of p1215 (<italic>URA3 SUP35C</italic>) was selected by growth on 5-fluoroorotic acid media and Ade- transformants were made rho&#176; by growth on YPAD containing 1 mg/ml ethidium bromide. Donor and recipient strains at high density were mixed in water at a ratio of about 5&#8758;1, and the mixture was spotted onto a YPAD plate. After 18 hours at room temperature, the mating mix was streaked for single colonies on media selective against growth of the donor strain. Clones are shown to be cytoductants by their growth on glycerol and failure to grow on media selective for diploids. As further tests of a sample confirm, Ade+ cytoductants are judged to have received and propagated [<italic>PSI</italic>
<sup>+</sup>].</p><p>[<italic>PSI</italic>
<sup>+</sup>] Strain 779-6A <xref ref-type="bibr" rid="pgen.1003257-Jung1">[48]</xref> was streaked to single colonies on &#189; YPD media and twelve colonies were selected, named Y1-Y12. These isolates were streaked to single colonies three additional times, each time selecting just one colony for further propagation. From the third plate a single colony was selected and expanded on &#189; YPD, and cells from this plate were used for cytoduction. From dilution tests there are approximately 2&#215;10<sup>7</sup> cells per colony, indicating a total of at least 75 generations of growth of clones Y1-Y12 before cytoduction. Additional subclones were handled in the same manner with only ten colonies selected from the initial &#189; YPD plate. In experiments to rule out selection during stationary growth phase, subclones of Y1 and Y2 were grown in a 125 ml Erlenmyer flask containing 25 ml of liquid YPD medium. When A<sub>600</sub> reached 0.3, the culture was diluted, transferring 1000 cells of each to a fresh flask. These subclones were propagated in exponential phase for 84 generations and were then streaked for single colonies on &#189; YPD plates. After one day of growth on &#189; YPD, 10 subclones were selected for each of Y1 and Y2, expanded and tested for transmission via cytoduction.</p><p>Strains reported to be [<italic>PSI</italic>
<sup>+</sup>] <xref ref-type="bibr" rid="pgen.1003257-Halfmann1">[49]</xref> were obtained from the UC Davis Department of Viticulture and Enology culture collection. The cultures were first tested to determine if dots were visible using either reference sequence Sup35NM-GFP pDB65 or E9 sequence Sup35NM-GFP pDB81 <xref ref-type="bibr" rid="pgen.1003257-Bateman1">[20]</xref>. Images were obtained with a Nikon Eclipse TE2000-U spinning disc confocal microscope with 100&#215; NA 1.4 Nikon oil lens with 1.5&#215; magnifier and captured with a Hamamatsu EM-CCD ImagEM digital camera with IPLab version 4.08. Wild strains were sporulated and spores were crossed on rich medium with strain 4972 selecting G418-resistant prototrophs. The diploids formed were again sporulated and tetrads were dissected for each wild strain except for strain 978, whose diploid with 4972 would not sporulate. Ade positive segregants were tested for guanidine curing using two successive streaks on YPAD with 5 mM guanidine. <italic>MAT&#945;</italic> strains were cytoduced into strain 4830 carrying pRS316 (<italic>URA3</italic>) for selection. <italic>Lys2</italic> mutants of <italic>MATa</italic> strains were selected on plates with DL-&#945;-aminoadipic acid as a nitrogen source <xref ref-type="bibr" rid="pgen.1003257-Chattoo1">[67]</xref>. Selected strains were retested for Ade positive growth and curing and cytoduced into strain 4828.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The cytoduction data follows the binomial distribution, because each data point expresses two alternative results, transmission of [<italic>PSI<sup>+</sup></italic>] or failure of its transmission. However, because of the large number of observations, the results should be approximately normally distributed. We want to calculate the probability that two sets of data are due to chance. Let p<sub>1</sub> and p<sub>2</sub> be the observed proportions of transmission in cytoductions 1 and 2, and n<sub>i</sub> the number of cytoductants tested in each experiment. Let p&#8202;=&#8202;(p<sub>1</sub>n<sub>1</sub>+p<sub>2</sub>n<sub>2</sub>)/(n<sub>1</sub>+n<sub>2</sub>) be the average of the proportion of transmission in the two experiments. The estimated standard error of the difference between the two proportions is<disp-formula id="pgen.1003257.e001"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003257.e001.jpg" /></disp-formula>The null hypothesis is that cytoductions 1 and 2 are samples from the same population with transmission efficiency p and standard error S. Then the expected proportions are expected to be the same and their difference is expected to be zero. [(p<sub>1</sub>&#8722;p<sub>2</sub>)&#8722;0]/S&#8202;=&#8202;z&#8202;=&#8202;the number of standard deviations that the observed difference in proportions differs from the expected difference (0). The frequency of &#8220;z&#8221; being greater or equal to the observed value (assuming the null hypothesis) is obtained from a table of the normal distribution. The calculated &#8220;p values&#8221; are shown in the tables and at appropriate points in the text.</p><p>Cytoductants examined have been treated as independent since the chance that they represent sister cells is close to zero. This is because cytoductant mixtures were incubated at 20C where the cells divide slowly and because only about 30 cells were examined from several million in the zygote mixture on each plate.</p>