<p>We obtained two cell types from Coriell for this study: skin fibroblast cells and lymphoblastoid cell lines (LCLs). Primary skin fibroblast cells from three human, three chimpanzee, and three macaque individuals. LCLs, which are B cells immortalized with Epstein-Barr Virus, were obtained from the same three human and three chimpanzee individuals that fibroblasts were isolated from (<xref ref-type="supplementary-material" rid="pgen.1002789.s017">Table S1</xref>). EBV does not reliably transfect macaque lymphocyte cells, so matched macaque LCLs cells were not available for this study. Importantly, other recent genome-wide studies that used macaque LCLs were of B-Lymphocyte cells transformed with rhesus herpes papio virus, a close relative of human EBV <xref ref-type="bibr" rid="pgen.1002789-Cain1">[57]</xref>. Cells from all species were grown in standard growth media. Fibroblast growth media consisted of Gibco's MEM (10370-021), L-Glutamine (25030-081), Pen/Strep (15140-122), and 10% FBS (Hyclone SH30070). LCLs growth media consisted of Gibco's RPMI (21870) media with L-Glutamine, Pen/Strep, and 15% FBS. We harvested fifty million cells for each individual biological replicate and allocated 35 million cells for DNase assays (DNase-seq and DNase-chip), 10 million for genomic DNA (used as control for DNase-chip array hybridization), and 5 million for RNA DGE-seq expression analysis.</p><p>DNase-seq libraries we generated as previously described <xref ref-type="bibr" rid="pgen.1002789-Boyle1">[17]</xref>, <xref ref-type="bibr" rid="pgen.1002789-Song1">[18]</xref> and sequenced via Illumina's GAII sequencer. DNase-chip library preparations, used for validation of our DNase-seq results were performed as previously described <xref ref-type="bibr" rid="pgen.1002789-Crawford1">[24]</xref>, <xref ref-type="bibr" rid="pgen.1002789-Shibata1">[25]</xref> and were hybridized to 1% ENCODE Nimblegen arrays <xref ref-type="bibr" rid="pgen.1002789-Consortium1">[19]</xref>. Custom arrays were designed to cover the orthologous regions from chimpanzee and macaque. DNase-chip array intensities were compiled and significant DHS sites were called using ChIPOTle <xref ref-type="bibr" rid="pgen.1002789-Buck1">[62]</xref> (<italic>P</italic>&lt;0.000001 peak cutoff).</p><p>DNase-seq data generated from each species was aligned to the native genome (human hg19, chimpanzee panTro2, and macaque rheMac2) using BWA <xref ref-type="bibr" rid="pgen.1002789-Li1">[63]</xref>. To directly compare three different primate species requires that they be aligned to a single reference sequence. Because both the chimpanzee (panTro2) and macaque (rheMac2) reference sequences were built from the existing human reference, we converted all sequences to human coordinates. To do this, we converted each 20-mer DNase-seq sequence from panTro2 or rheMac2 to hg19 with liftOver <xref ref-type="bibr" rid="pgen.1002789-Hinrichs1">[23]</xref>, using a match setting of 80 percent. After conversion to hg19 coordinates, we used F-seq <xref ref-type="bibr" rid="pgen.1002789-Boyle2">[64]</xref> to identify DNaseI hypersensitive (DHS) sites. The F-seq scores from the top 100,000 peaks from each sample were used to determine how well chromatin openness correlates among all 15 samples (<xref ref-type="fig" rid="pgen-1002789-g001">Figure 1</xref>). This analysis uses a pairwise Pearson correlation to compare the similarity among samples.</p><p>We used the bioconductor edgeR package to define species-specific hypersensitive regions <xref ref-type="bibr" rid="pgen.1002789-Robinson1">[26]</xref>. EdgeR is designed to detect differences in count data among groups of samples. Briefly, it compares within-group variances to between-group variances using a negative binomial model, and selects entries with significant between-group differences. It was designed for differential expression data such as DGE-seq or RNA-seq, but it is similarly applicable to read counts generated by DNase-seq. One key advantage of edgeR is a normalization procedure specifically designed for high-throughput sequencing studies <xref ref-type="bibr" rid="pgen.1002789-Robinson1">[26]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To locate significant differences in DNase-seq signal between species, we first identified the union set of the top 100,000 DHS sites (as scored by F-seq) from all 15 samples (9 fibroblasts and 6 LCLs). We used bx-python (<ext-link ext-link-type="uri" ns0:href="https://bitbucket.org/james_taylor/bx-python">https://bitbucket.org/james_taylor/bx-python</ext-link>) to analyze the data. We divided these regions into windows, attempting to maximize the resolution of the windows while minimizing the number of windows required:</p><p>We divided the union set into overlapping windows of 300 bp. DHS sites smaller than 300 bp were expanded to 300 bp. Regions larger than 300 bp were tiled with overlapping windows; the overlap varies depending on the size of the hypersensitive region to tile. We start by finding the number of windows that would fit completely inside the defined HS site using the default overlap (100 bp). If these windows discard fewer than 10% of the bases on each edge of the HS site, we tile the site using these windows, because the initial peak calls tend to run a bit wide. If using the default overlap would cause us to lose more than this edge threshold (10%), we add another window and adjust the overlap so the windows exactly cover the entire HS region. These windows cover almost all of the HS bases in the original, while minimizing the number of non-HS bases considered for the downstream analysis. After defining the windows, we liftOver each window back to chimp and macaque and discard any windows that don't map. We also eliminated any windows mapping to the Y chromosome, because we have female samples. This resulted in &#8764;1.2 million windows across all DHS sites (Supplemental data file 4 in <xref ref-type="supplementary-material" rid="pgen.1002789.s001">Dataset S1</xref>).</p><p>We next counted the number of reads (DNase cuts) that mapped within each window for each sample. To be certain to compare only regions that are actually found in all 3 primates, we omitted any windows that did not have at least one read count in at least one individual from each species. We believe this is a reasonable filter since there is a basal level of DNase digestion throughout the genome, and regions without any mapping reads in one species are likely caused by alignment ambiguities (such reads are lost in the liftOver process) or reference problems.</p><p>We then used edgeR to call windows with significantly different counts in each pairwise comparison at a false discovery rate of 1%. This defined species-specific HS windows. For example, to define human-fibroblast-specific (human DHS gain) regions, we selected any windows with significantly more counts in humans compared (pairwise) to both chimp and macaque fibroblasts. We then merged any significant neighboring windows.</p><p>To confirm that DNase sites detected in only a single species were not due to a liftOver artifact or copy-number variation (CNV), we performed a reciprocal liftOver. Reciprocal liftOver discards regions that do not map 1&#8758;1 between species assemblies. After defining a species-specific or Common DHS sites (Common DHS described below) in human coordinates, we the lifted these windows back to panTro2 or rheMac2. We then compared number of reads from the original native genome alignment to the read count on the lifted (hg19) reference. We found that, in general, these scores correlate well. However, some windows differ in counts on each reference, indicates liftOver artifacts or CNVs. Windows with native counts exceeding the reference counts could result from counts from the native assembly being lifted to multiple places on hg19. Likewise, windows with more reads on hg19 than on the equivalent native assembly could be caused by several regions from the native genome lifting to a single location on hg19. Since these may possibly artificially appear as chromatin gains or losses, we filter out these scenarios by requiring that the number of reads on hg19 for chimp/macaque data be within 10% of the number of reads on the native assembly.</p><p>Some regions mapped to areas in the genome that are clear artifacts, most likely CNVs. These artifacts are expanses where one of the species has much higher levels of hypersensitivity than the others, and are often located near centromeres or telomeres. A DHS resulting from a massive expansion in one of our samples compared to reference genome appears as a large (kb to Mb-scale) DHS site encompassing the entire CNV and/or DHS sites that cluster closely together. We manually curated a list of such areas and filtered data mapping within these regions (Assembly filter). We also eliminate abnormally sharp peaks, which are likely caused by PCR artifacts, reference assembly inaccuracies, or small differences in copy number between our samples and the reference genomes. If any 30 bp window within a region contains 75% or more of the reads in the entire region, these regions are filtered out as &#8220;sharp peaks&#8221;. Ultimately, we confirmed the regions by visually inspecting many using both the Integrative Genomics Viewer (IGV) <xref ref-type="bibr" rid="pgen.1002789-Robinson2">[65]</xref> and the UCSC Genome Browser <xref ref-type="bibr" rid="pgen.1002789-Hinrichs1">[23]</xref>. These filters apply to DHS gains, losses, and common regions.</p><p>Chang et al. (2002) showed that human skin- and non-skin- fibroblast samples collected from different locations along the body plane showed notable differences in transcriptional profiles <xref ref-type="bibr" rid="pgen.1002789-Chang1">[29]</xref>. Aware of this issue, we made an effort to use fibroblast cultures made from only skin samples and from the same region of the upper arm. All 3 replicates of macaque Fibroblasts and one human Fibroblast were confirmed from Coriell to be from skin biopsies from the upper arm (the other two locations were unknown). Since our analysis poses a strict requirement of DHS sites to be present in all three human individuals to be called a human DHS gain, having at least one human sample with the biopsy site confirmed to be from the upper arm ensures that the human gains are not a result of human DHS gains being a result of, for example, all human fibroblasts isolated from lower leg. Likewise, to be called a human chromatin loss, DHS sites have to be absent in all three human samples (but present in all chimp and macaque samples); this biopsy location bias is again mitigated by at least one human sample being from the upper arm.</p><p>The Yerkes National Primate Center, from where the chimpanzee skin fibroblasts were obtained from, unfortunately do not document the exact location of biopsy. While the standardized skin punch protocol calls for the location of the skin biopsies to be from the ear pinna (personal communication with Fawn Conner-Stroud from Yerkes), we cannot be 100% sure that the samples were isolated from this location. As described above, human DHS losses are a result of signal being present in all three chimpanzee and macaque samples, supporting that these regions are not due to chimp biopsy location. We also want to reiterate that our skin fibroblast results are highly similar in LCL lines that are all uniformly derived from blood samples.</p><p>We find that none of our chromatin gains and losses overlap the Hox genes described in the Chang et al., 2002 paper <xref ref-type="bibr" rid="pgen.1002789-Chang1">[29]</xref>. In addition, a more recent analysis by Rinn et al. <xref ref-type="bibr" rid="pgen.1002789-Rinn1">[30]</xref>, using more comprehensive microarrays and more biopsy sites, identified 337 expression array probes (299 unique genes) that were shown to be highly associated with five different general biopsy site locations. We have compared this list of genes to both species-specific chromatin gains/losses, as well as species-specific gene expression, and find &lt;3% of the species-specific and common DHS overlap with the 299 position specific genes. Similarly, species-specific and commonly expressed genes also show &lt;3% overlap (<xref ref-type="supplementary-material" rid="pgen.1002789.s028">Table S12</xref>).</p><p>In order ensure that our tests for selection were meaningful, we wanted to compare chromatin gains and losses with a set of regions that were open in all species. Rather than simply choose DHS sites that have the highest scores, we wanted to mirror the level of hypersensitivity to that of the species-specific regions. This is important because species-specific DHS sites are not necessarily the strongest DHS sites. We also wanted to select a set of regions similar in size to our sets of gains and losses to retain computational tractability.</p><p>To select a set of matched Common DHS sites, we required that each window be similarly open in all 9 samples from all 3 species. To be considered &#8220;similarly open&#8221; in a given sample, the number of counts must lie between the 20% and 80% quantiles for that sample in the corresponding species-specific regions. For example, we used the human DHS gains identified by edgeR to define the distribution for each of the three human samples, and similarly for chimp and macaque. As such, our set of Common regions is the set of all windows with DNase counts within this range for each of the 9 samples (<xref ref-type="fig" rid="pgen-1002789-g002">Figure 2d</xref>). To reduce the number of Common regions we found to the most representative set (those that <italic>most closely match</italic> the average signal intensity of the differential DHS sites), we narrowed the quantiles until we found a set of around one thousand Common regions, which we reasoned would be a sufficient number to examine summary statistics. To ensure that our results are not biased for a specific set of Common regions, we repeated our experiments on a significantly larger set (&#8764;11,000) of Common DHS sites using less stringent criteria (10%&#8211;90% quantile). This larger set is even more enriched for promoter regions but does not change our conclusions (data not shown).</p><p>After identifying an initial set of potential Common DHS sites, we filtered out any that appeared to be &#8220;appendages&#8221; to other hypersensitive sites. Without this step, many Common DHS sites would map to the edge of a strong hypersensitive sites. To ensure that a Common DHS site is a standalone DHS site, we examined the neighboring windows surrounding the initial set of Common DHS sites. If a Common DHS site window contained fewer than 80% of the number of reads in the adjacent window on either side, we filtered it out as most likely an &#8220;appendage&#8221; to a stronger DHS site. This resulted in a final list of 1259 Common DHS sites matched in intensity to the species-specific DHS sites. We also ran this filter on DHS gains and losses, and found that very few of the gains (3&#8211;5%) and losses (3&#8211;8%) get flagged as &#8220;appendages.&#8221; Of these, many of them are flagged as a result of FDR threshold issues that simply didn't quite highlight a neighboring window, and we would actually still consider this a legitimate gain region. Because gain/loss appendages are relatively rare and are largely due to threshold issues, we elected to retain them in our final list.</p><p>In every comparison, we reported more species-specific gains than losses. The most important factor in determining the size of these lists is the level of significance we set by choosing a FDR. To obtain lists that match in length, we could simply adjust the FDR value for the lists to yield about the same number of regions. Instead, we decided to keep the FDR constant and select varying numbers of DHS sites for each category.</p><p>However, it is still constructive to consider the disparity. In other words, &#8220;at a constant FDR (1%), why are there more species-specific increases than decreases?&#8221; This is possibly a result of purifying selection. Because DHS sites are regulatory (and therefore tend to be conserved), a loss of a DHS site probably confers a fitness disadvantage. In this case, we would expect to see more gains than losses.</p><p>It is also possible that the prevalence of increases is simply a result of the way we constructed the significance test. A DHS site is a sparse signal (there are more &#8220;closed&#8221; regions than &#8220;open&#8221; regions). Combined this with the asymmetry of the evolutionary tree: the chimp and human are more similar to each other than either is to the macaque. A human-specific increase requires both macaque and chimp to be closed (the default), while a human-specific decrease requires both macaque and chimp to be open. This latter scenario will happen less often because the relationship between chimp and human is closer than either to the macaque.</p><p>In short, the greater number of gains than losses in our analysis may reflect purifying selection on DHS sites; however, it may also simply be a result of the way we constructed the test, particularly due to using an outgroup species to polarize the chromatin structure changes.</p><p>We tested for evidence of positive selection using the DHS sites indicated as DHS gains, losses, and commons defined by edgeR and common analyses (see above). A branch model test <xref ref-type="bibr" rid="pgen.1002789-Zhang2">[66]</xref> in HYPHY <xref ref-type="bibr" rid="pgen.1002789-Pond1">[35]</xref> was used to assess evidence for positive selection on each the human and chimp branches. HYPHY uses a likelihood ratio test to compare two opposing models. For the null hypothesis, we specified a composite model that allowed for negative selection, neutral evolution, or relaxed constraint specifically on the branch of interest (<italic>i.e.</italic> the human branch), with negative or neutral evolution across the rest of the tree. The alternative hypothesis modeled positive selection only on the branch of interest, with negative or neutral evolution on the rest of the tree. For each region, HYPHY performed a likelihood ratio test comparing these hypotheses and output a <italic>P</italic> value that can be interpreted as a level of evidence for positive selection. In order to test the likelihood of either the null or alternative hypothesis, we specify both the alignment of the region of interest, as well as a background sequence alignment assumed to be evolving neutrally <xref ref-type="bibr" rid="pgen.1002789-Wong1">[67]</xref>. For the alignment of the region of interest, we used alignments of human, chimp, macaque, and orangutan precomputed at UCSC. For the background sequence, we collected a separate set of local introns for each region to test, following Haygood et al. <xref ref-type="bibr" rid="pgen.1002789-Haygood1">[16]</xref>. To define these background alignments, we started with the UCSC knownGene definition of intron annotations, and then filtered out all first introns, splice junctions, and hypersensitive sites (in any of the 15 samples in this study). In this way, we aimed to select sequences that are evolving neutrally. After defining this set of neutral introns, we used an expanding window centered on the region of interest to collect introns in a region up to 100 kb surrounding the center. We added introns to this collection sequentially as the window expanded until we reached an alignment of 2000 nucleotides. If we were unable to find 2 kb of background introns within 100 kb of sequence, we discarded these regions (this happens rarely). Introns are commonly assumed to be evolving neutrally <xref ref-type="bibr" rid="pgen.1002789-Graur1">[43]</xref>, <xref ref-type="bibr" rid="pgen.1002789-Gilbert1">[68]</xref>, particularly when our filtering steps are taken into account; however, there are still likely to be regulatory sequences present in our background model, either due to sequences containing DNaseI HS sites in other cell types not tested or due to unannotated or mis-annotated transcripts. In order to further correct for this possibility, we performed each likelihood-ratio test 50 times, using 50 different bootstrapped versions of the background model. We then averaged these <italic>P</italic> values to assign a final <italic>P</italic> value for each region. This method has the effect of possibly discarding any elements under selection in some of the bootstrap replicates, increasing our ability to detect positive selection even if we inadvertently chose some background regions under selection. To test significance, for a given set of regions (e.g. human DHS gains) we ordered the <italic>P</italic> values for selection on both hg19 and panTro2, then did a Mann-Whitney test to see if one branch has higher <italic>P</italic> values than the other (<xref ref-type="supplementary-material" rid="pgen.1002789.s026">Table S10</xref>).</p><p>The fibroblast DHS sites where we can polarize the differences using macaque all have significant differences in the direction we expect, while the Common regions do not have significant differences. In the LCLs, where we are unable to polarize (no macaque LCLs were available), we do not detect a significant difference. This is likely due to a combination of two categories (gains and losses) that have competing selection (i.e. LCL human DHS loss&#8202;=&#8202;human DHS loss+chimpanzee DHS gain).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We calculated the observed fraction of overlaps between DHS sites and evolutionarily constrained regions using constrained regions defined by the Genome Evolutionary Rate Profiling (GERP) method <xref ref-type="bibr" rid="pgen.1002789-Cooper1">[40]</xref> executed on Enredo, Pecan, Ortheus (EPO) <xref ref-type="bibr" rid="pgen.1002789-Paten1">[69]</xref>, <xref ref-type="bibr" rid="pgen.1002789-Paten2">[70]</xref> 33-way alignments. EPO alignments and GERP regions are available for download at the Ensembl browser (<ext-link ext-link-type="uri" ns0:href="http://ensembl.org">http://ensembl.org</ext-link>). We then constructed a null distribution of the fraction of expected overlaps by using the conservative Genome Structure Correction (GSC) methodology described previously <xref ref-type="bibr" rid="pgen.1002789-Consortium1">[19]</xref>, <xref ref-type="bibr" rid="pgen.1002789-Parker1">[41]</xref>, <xref ref-type="bibr" rid="pgen.1002789-Bickel1">[42]</xref>. We performed 1000 randomizations and calculated the mean and standard deviation from the null distribution to assess the statistical significance of the observed overlap (<xref ref-type="fig" rid="pgen-1002789-g006">Figure 6d</xref>).</p><p>We also used PhastCons to explore degree of sequence conservation. For each region, we extracted the mean and max PhastCons score from the primate PhastCons table at UCSC. We then compared the distribution of these scores across the regions to see how sequence conservation is related to hypersensitivity conservation (<xref ref-type="supplementary-material" rid="pgen.1002789.s010">Figure S9</xref>).</p><p>Total RNA purified from 5&#8211;10 million cells harvested from the same cell culture used for DNase-seq were also used to generate DGE-seq expression libraries as previously described <xref ref-type="bibr" rid="pgen.1002789-Babbitt1">[12]</xref>, <xref ref-type="bibr" rid="pgen.1002789-tHoen1">[71]</xref>. Polyadenylated RNA is captured for enrichment of mRNA and the oligo dT primer is used to make cDNA. Briefly, DGE-seq is similar to Serial Analysis of Gene Expression (SAGE) where mRNA abundance is assessed via counting short sequences of their cDNA at specific restriction site locations. These DGE-seq libraries were sequenced using Illumina's GAII sequencer, and averaged 10 million 20mer sequences for each sample, which were then aligned to the samples' native reference sequence using BWA. We used EdgeR to detect differences in tag counts across species by comparing intra-species variances to inter-species variances using a negative binomial model, and selects expressed genes with significant between-species differences <xref ref-type="bibr" rid="pgen.1002789-Robinson1">[26]</xref>.</p><p>Unlike the analysis performed for comparing cross-species DHS sites, we did not liftOver any non-human expression sequences to human. Instead, we simply compared the DGE-seq sequence counts that aligned to exonic regions within each species' native sequence alignment (Supplemental data file 5 in <xref ref-type="supplementary-material" rid="pgen.1002789.s001">Dataset S1</xref>). Because of the high level of homology of the exonic regions between the 3 primate species, we directly compared tag count numbers between each of the orthologous genes. For the genome-wide expression correlation comparison (<xref ref-type="fig" rid="pgen-1002789-g001">Figure 1</xref>), we normalized tag counts using edgeR to calculate the total library size for each sample and adjusting the tag counts accordingly so that relative differences between the depth of the sequencing reads did not influence the results. Next, we filtered out genes that did not have at least 10 combined tag counts between all of the samples to decrease the noise associated with genes that fall below the meaningful level as recommended for edgeR. Following these filters, we compared the Spearman correlation values between all of the samples and plotted the results as a heatmap with hierarchical clustering to show similarities within and between species and tissue types.</p><p>Using edgeR, we identified genes that were significantly differentially expressed between the 3 primate species. Comparisons between species were performed on a pairwise manner comparing 3 individuals of one species against 3 individuals from a second species. The same normalization method and filters used in the expression correlation analysis was also used prior to defining the differentially expressed genes. Differential gene expression was defined as genes having a <italic>P</italic> value cutoff of 0.05. Using the Macaque expression result as an outgroup, we identified genes that displayed high or low expression specifically on the human and chimp branch. For example, genes we defined as highly expressed in human (human upregulated genes) are differentially expressed in both human/chimp and human/macaque comparisons, but not differentially expressed in the chimp/macaque comparison. Similar criteria were used to identify genes that display low expression in human compared to chimp and macaque (human downregulated genes).</p><p>To firmly establish the connection between differential chromatin and differential expression, we tested for significance in overlap in both directions: First, we tested if differential DHS sites tend to be located near differentially expressed genes, and second, we tested if differentially expressed genes tend to have differential DHS sites nearby.</p><p>We compared human DHS gain, human DHS loss, chimpanzee DHS gain, and chimpanzee DHS loss regions to species-specific genes (e.g., human upregulated/downregulated genes, etc.). We first assigned each DHS site its nearest gene, then intersected these with each differential expression set. The number of overlaps for every pairwise combination of DHS sites vs. differentially expressed genes are shown in <xref ref-type="supplementary-material" rid="pgen.1002789.s021">Table S5a</xref>. We calculated <italic>P</italic> values by permuting (100,000 times) the set of all expressed Fibroblast genes, randomly selecting the number of genes for each comparison, and intersecting those random sets with the genes nearest species-specific DHS sites (<xref ref-type="supplementary-material" rid="pgen.1002789.s023">Table S7b</xref>&#8211;<xref ref-type="supplementary-material" rid="pgen.1002789.s023">S7c</xref>). This established a null distribution of overlaps in random intersects.</p><p>To establish the connection in the opposite direction, we did a similar analysis in the reverse direction. Using the UCSC knownGene table, we merged all isoform coordinates for each gene and found all DHS sites within a surrounding 20 kb window. We calculated pairwise overlaps of DHS sites between each differential DHS site list and these lists of all nearby DHS sites. If a gene contained a differential DHS site within 20 kb, it was counted as a match. The number of overlaps for every pairwise combination of sites near differentially expressed genes vs. differential DHS sites are shown in <xref ref-type="supplementary-material" rid="pgen.1002789.s022">Table S6a</xref>. We then permuted genes similar to above (1,000 times), located all nearby DHS sites, and tested for overlap to create a null distribution of overlap count (<xref ref-type="supplementary-material" rid="pgen.1002789.s024">Table S8b</xref>&#8211;<xref ref-type="supplementary-material" rid="pgen.1002789.s024">S8c</xref>).</p><p>To connect sequence changes to species-specific DHS sites, we compared JASPAR motif scores across species. We first extracted the orthologous DNA sequences for human, chimpanzee, and macaque for each of our DHS gain, loss, and common sites. We scanned and scored each of these sequences for all the position weight matrices (PWMs) in the JASPAR database. We scored a sequence for a given PWM as the highest motif score anywhere in that sequence. This resulted in a region-by-motif matrix of scores; each score is the highest score for each motif/sequence combination. To compare species, we took the log ratio of scores (human/chimp, human/macaque, and chimp/macaque). Where this score is 0, the highest score does not differ between species. Deviations from 0 indicate the direction of improvement in motif match (in a human/chimp comparison, a positive number means the best match in the human sequence scored higher than the best match in the chimp sequence). After calculating these scores and each pairwise log-ratio, we plotted the log-ratios (Supplemental data files 2&#8211;3 in <xref ref-type="supplementary-material" rid="pgen.1002789.s001">Dataset S1</xref> and <xref ref-type="fig" rid="pgen-1002789-g007">Figure 7</xref>) to examine trends. We calculated the pairwise log-ratios for multiple species comparisons and plotted these on different axis to check whether increases over one species correlate with increases over the other. In these two-dimensional plots, each axis quantifies a different pairwise species relationship. Points that cluster in the upper-right quadrant have higher scores than either of the other species; points that cluster in the lower-left have lower scores. The most interesting variation in these plots is whether the points congregate in the upper-right or lower left. To assess significance, we project each data point onto the y&#8202;=&#8202;x line to reduce the dimensionality to 1. We then used the Wilcoxson rank-sum test to compare each distribution to the distribution of the common regions (<xref ref-type="fig" rid="pgen-1002789-g007">Figure 7g</xref> and Supplemental data file 3 in <xref ref-type="supplementary-material" rid="pgen.1002789.s001">Dataset S1</xref>).</p>