<p>All participants 18 years of age and older gave informed consent. Individuals under 18 years of age gave assent after a parent signed a consent form on their behalf. The Ethics Committees of each participating site approved this research and the research was conducted in accordance with the Declaration of Helsinki.</p><p>The datasets used in this study are described in depth elsewhere <xref ref-type="bibr" rid="pgen.1003864-Scharf1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1003864-Stewart1">[31]</xref>. Briefly, DNA from individuals with TS or OCD and from controls was randomized across plates and genotyped using the Illumina Human610-Quad genotyping array. Additional unscreened controls that were genotyped as a part of the SAGE (genotyped on Illumina HumanHap1Mv1_C) and iControl (genotyped on Illumina HumanHap550v1/v3) datasets were also included in this study. To reduce effects of population stratification, subjects were limited to those with genetically defined European ancestry, based on principal components clustering analysis using genome-wide pairwise identity-by-descent (IBD) information as estimated with EIGENSTRAT 3.0 <xref ref-type="bibr" rid="pgen.1003864-Patterson1">[32]</xref> and including previously defined European population samples as reference (HapMap3.0).</p><p>The first phase of quality control analyses, including assessment of Hardy-Weinberg equilibrium, differential missingness, platform effects, population stratification, and genotyping call rate, was conducted as a part of the recently published GWAS of OCD and TS <xref ref-type="bibr" rid="pgen.1003864-Scharf1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1003864-Stewart1">[31]</xref>. The variance components models in the REML analysis utilized all unpruned genotype data simultaneously. Because all genotypes are fitted together in a given variance component, these components are particularly susceptible to minor technical and experimental artifacts that might only modestly affect each genotype (i.e., in a SNP-by-SNP test of association) but could have a substantial cumulative global effect on the results from a mixed linear model. We thus undertook additional, more stringent quality control measures to minimize any possible persistent population stratification and experimental bias. Prior to case-control comparisons, we first focused solely on the control dataset to develop our QC pipeline. We split the controls by data source (iControl vs. SAGE controls) and performed the following QC steps using PLINK. We implemented stringent thresholds and removed additional SNPs showing low levels of differential missingness between cases and controls (p&lt;0.05), modest deviation from Hardy-Weinberg expectation (p&lt;0.05), and significant platform effect after adjustment for all ten principal components (p&lt;0.001). In addition, individuals with genotype call rate &lt;99.9%, or with a high degree of relatedness (pi-hat&gt;0.05) were removed (<xref ref-type="supplementary-material" rid="pgen.1003864.s012">Table S1</xref>). To assess any residual cross-platform artifacts that might artificially elevate the heritability estimate, we conducted a dummy case-control GWAS by assigning case status to the iControl data (N&#8202;=&#8202;1,104) and control status to the SAGE Controls (N&#8202;=&#8202;2,190). We detected no significant association with platform &#8220;phenotype&#8221; by logistic regression <bold>(<xref ref-type="supplementary-material" rid="pgen.1003864.s001">Figure S1</xref>)</bold> or &#8220;heritability&#8221; between cross-platform controls (h<sup>2</sup>&#8202;=&#8202;10<sup>&#8722;6</sup>, se&#8202;=&#8202;0.11) (<xref ref-type="supplementary-material" rid="pgen.1003864.s013">Table S2</xref>). Additionally, we analyzed ten permutations of the dummy case phenotype and detected no significant heritability in any permuted analysis. In addition to these QC steps, we examined the data for any possible residual population stratification or cryptic relatedness, which is described in depth in the Supplementary Methods <bold>(<xref ref-type="supplementary-material" rid="pgen.1003864.s001">Figures S1</xref>, <xref ref-type="supplementary-material" rid="pgen.1003864.s002">S2</xref>, <xref ref-type="supplementary-material" rid="pgen.1003864.s003">S3</xref>, <xref ref-type="supplementary-material" rid="pgen.1003864.s004">S4</xref>)</bold>. The quality control and matching steps resulted in a final data set of 617 TS cases and 4,116 TS controls genotyped on 393,387 SNPs, as well as 1,061 OCD cases and 4,236 OCD controls genotyped on 373,846 SNPs. Each analysis included the top 20 principal components as covariates.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For each analysis presented, GCTA v1.2 (<xref ref-type="bibr" rid="pgen.1003864-Yang1">[5]</xref>; <ext-link ext-link-type="uri" ns0:href="http://www.complextraitgenomics.com">www.complextraitgenomics.com</ext-link>) was used to create a genetic relationship matrix (GRM) file containing IBD relationship calculations for all pair-wise sets of individuals. Principal components were determined within GCTA, using all genotype data, and the top 20 principal components were applied to each analysis. The REML analysis was then performed using the respective GRMs and principal component quantitative covariates. As this analysis was performed with dichotomous case/control traits, it was necessary to convert the phenotypic variance to an underlying liability scale. This conversion uses population prevalence to adjust for case/control ascertainment in the sample and to modify the phenotypic variance estimate accordingly <xref ref-type="bibr" rid="pgen.1003864-Lee1">[4]</xref>. We conducted primary analyses using 2.5% for OCD prevalence and 0.8% for TS. As a range of prevalence estimates for both OCD and TS are frequently reported, we conducted additional sensitivity analyses to examine the heritability estimates for TS and OCD across a range of reported prevalences <bold>(<xref ref-type="supplementary-material" rid="pgen.1003864.s014">Table S3</xref>)</bold>
<xref ref-type="bibr" rid="pgen.1003864-Karno1">[33]</xref>, <xref ref-type="bibr" rid="pgen.1003864-Khalifa1">[34]</xref>, <xref ref-type="bibr" rid="pgen.1003864-Wang2">[35]</xref>, <xref ref-type="bibr" rid="pgen.1003864-Lanzi1">[36]</xref>. Additionally, we provide heritability results converted to the sibling relative risk scale for further interpretation <bold>(<xref ref-type="supplementary-material" rid="pgen.1003864.s015">Table S4</xref>)</bold>. We conducted three primary analyses (univariate TS heritability, univariate OCD heritability, joint OCD and TS bivariate analysis) and five exploratory analyses (partitioning by chromosome, MAF, genic annotation, brain eQTL annotation, age of onset). For each primary analysis, ten permutations of the phenotype were performed and GCTA was run on each permutation to observe the stability of the heritability estimate.</p><p>In addition, we calculated the genetic correlation between OCD and TS using the GCTA bivariate REML analysis. We split the shared control sample between the TS cases and the OCD cases in a manner that preserved the matched ancestry structure and the proportion of cases to controls for each disorder. An initial analysis included co-morbid TS and OCD cases assigned to either the TS or OCD samples based on their primary diagnosis as determined by the clinical team. We conducted a secondary bivariate analysis limiting the SNPs included to a subset of SNPs previously identified as regulators of gene expression in the brain. A final sensitivity analysis was conducted after removing all 316 case samples with known overlapping comorbidity (83 OCD samples with TS or chronic tics, and 233 TS samples with OCD) to assess the effect of co-morbidity on the cross-disorder genetic correlation. We then applied a likelihood ratio test (LRT) to determine the statistical significance of each genetic correlation.</p><p>Imputation was performed using IMPUTE v2.1.2 and the 1000 Genomes Project data as a reference panel. Only imputed SNPs that were in strong linkage disequilibrium (LD) (info&gt;0.6) with genotyped SNPs and had a high certainty (&gt;90%) of the predicted genotypes were retained. Imputed SNPs that showed significant genotyping platform effects were excluded. Imputed results were converted to MaCH format (i.e., .mldose, .mlinfo) using an in-house script. MaCH dosage data was used to create GRMs for each chromosome. Chromosome specific GRMs were then merged as needed for additional analyses. The total number of imputed SNPs after QC included 7,657,106 SNPs in both the TS and OCD samples.</p><p>A separate GRM was generated for each chromosome. Each GRM was then run in separate REML analysis. An additional analysis was conducted in which all chromosomes were modeled jointly in a single REML analysis.</p><p>We chose not to employ a minor allele frequency (MAF) cutoff in any of the heritability analyses. This decision was based on the observation that minor allele frequency cutoffs did not alter estimates of heritability for the control-control analysis after establishment of stringent differential missingness rates and call rate. We partitioned the directly genotyped and imputed variants according to MAF bin. For the directly genotyped variants we created six bins representing MAFs from 0.001&#8211;.05, &gt;0.05&#8211;0.1, &gt;.1&#8211;.2, &gt;.2&#8211;.3, &gt;.3&#8211;.4, and &gt;.4&#8211;.5 and generated GRMs for each bin. For the imputed genotypes we created two bins representing MAF 0.001&#8211;0.05 and &gt;0.05 to 0.5 and generated GRMs for each bin. For each set of variants (directly genotyped and imputed respectively) we then combined binned GRMs in a single joint REML analysis, allowing the effects of LD to be partitioned by the REML analytic approach.</p><p>We annotated variants for genic and intergenic classification using ANNOVAR (hg18, refGene) <xref ref-type="bibr" rid="pgen.1003864-Wang3">[37]</xref>. Genic variants included all those variants annotated to exons, introns, UTRs and splice sites. Intergenic variants included those not otherwise annotated as genic. Additionally, we annotated directly genotyped and imputed SNPs that we had previously identified as significantly associated with gene expression (p&lt;0.001) in parietal cortex, (GSE35977), cerebellum (GSE35974), and skeletal muscle (GSE40234). Details of the eQTL detection are described in supplementary methods and in previous publications <xref ref-type="bibr" rid="pgen.1003864-Gamazon1">[38]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1003864-Li2">[45]</xref>. Three sets of analyses were conducted using the eQTL annotations. The first analysis simply partitioned the parietal eQTLs and cerebellar eQTLs from their respective complements for all imputed SNPs. The second model included four partitions: 1) brain only eQTLs (those found in cerebellum or parietal tissues but not in muscle), 2) muscle only eQTLs (those found in muscle and not in either brain tissue), 3) eQTLs common to brain and muscle, and 4) a final partition with non-eQTL SNPs. The last analysis included four total partitions to accommodate eQTLs exclusive to each brain tissue (cerebellum and parietal) as well as eQTLs found in both brain tissues, and the remainder of all imputed SNPs. Annotations were applied to the TS and OCD case/control data and used to create partitions. This resulted in a total of four separate annotation-based REML analyses. For each analysis, we created a single GRM for each partition. Finally, for each analysis, we included the functional variant GRM(s) and the respective complement GRM together in one joint REML analysis.</p><p>Multiple studies have reported significantly higher heritability for early-onset OCD than for adult onset OCD <xref ref-type="bibr" rid="pgen.1003864-vanGrootheest1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1003864-Nestadt1">[46]</xref>. Hanna and colleagues (2005) <xref ref-type="bibr" rid="pgen.1003864-Hanna1">[47]</xref> suggested a possible threshold of 14 years to define early-onset OCD, however, as our data was collected retrospectively, potentially introducing a recall bias, we chose to employ a conservative threshold for early-onset of symptoms or diagnosis at age 16. We sought to test the hypothesis that early-onset OCD is more highly heritable than adult-onset OCD by dividing the OCD sample based on symptom onset or age at diagnosis (&#8804;16&#8202;=&#8202;early onset, &gt;16&#8202;=&#8202;adult-onset). A total of 732 cases were diagnosed or reported symptom onset prior to age 16 and were considered early onset. A total of 267 cases were diagnosed or exhibited symptoms later than age 16 and were classified as adult onset. Age of onset data was missing for 62 cases. GCTA analysis was performed on both subsets of samples.</p>