<p>Old English Sheepdogs with canine hereditary ataxia were identified by referral from veterinarians, breeders and owners. DNA samples and pedigrees from these U.S. and Canadian dogs and their unaffected family members were collected. Affected status was determined by compatible clinical signs of cerebellar disease including ataxia, typical age of onset, physical exam, and neurological exam. Cerebrospinal fluid analysis was undertaken in many dogs, as well as a necropsy, when possible. All protocols were performed with approval from North Carolina State University's Institutional Animal Care and Use Committee. All normal dogs reached the age of four years of age with no evidence of clinical signs of ataxia. Dogs that could not definitively be classified as &#8220;affected&#8221; or &#8220;normal&#8221; based on collected information were classified as of &#8220;undetermined&#8221; status. DNA was extracted from whole blood using QIAamp DNA Blood Midi Kit (Qiagen; Valencia, CA), frozen tissues; DNeasy Blood and Tissue Kit (Qiagen; Valencia, CA), buccal swabs; QIAamp DNA Mini Kit (Qiagen; Valencia, CA), and from saliva using Oragene Animal kit (DNA Genotek; Kanata, Ontario). DNA concentrations were measured using a ND-1000 UV-Vis NanoDrop spectrophotometer (Thermo Scientific, Wilmington, DE).</p><p>Related dogs were genotyped using a genome-wide panel of 311 canine microsatellite markers (representing an average of 9 MB resolution), organized into multiplex PCR groups <xref ref-type="bibr" rid="pgen.1003991-Clark1">[31]</xref>. Four fluorescent labels (FAM, VIC, NED and PET) were incorporated into the PCR primers to allow multiplexing. PCR fragments were analyzed on an ABI-3700 automated Genetic Analyzer (Applied Biosystems), and genotypes were assigned using GeneMapper v3.7 software (Applied Biosystems). Homozygous (uninformative) markers were excluded from further analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Multipoint linkage analysis was performed using the lm_Bayes program of Morgan (v2.8.1) <xref ref-type="bibr" rid="pgen.1003991-Thompson1">[32]</xref>. Dogs were assigned affected, control or unknown phenotypes, and the trait frequency was set to default at 61% for the common allele and 39% for the risk allele. Based on the pedigree analysis performed previously <xref ref-type="bibr" rid="pgen.1003991-Steinberg1">[20]</xref>, penetrance was set at 98%. These values were varied once the linked region on CFA4 was identified to ensure that the results were robust. The location of microsatellites was determined from the online canine genetic linkage map at the University of California, Davis Veterinary Genomics Laboratory (<ext-link ext-link-type="uri" ns0:href="http://www.vgl.ucdavis.edu/dogmap/">http://www.vgl.ucdavis.edu/dogmap/</ext-link>). Allele calls not consistent with Mendelian inheritance were re-evaluated, and individuals with multiple Mendelian errors were removed from analysis. Results were evaluated for LOD scores greater than 3.</p><p>Genotyping of SNPs was performed using Illumina's Canine SNP20 (22,362 SNPs) and CanineHD (170,362 SNPs) genotyping beadchips (Illumina; San Diego, CA). Nine cases and 13 control dogs were genotyped on the SNP20 chip and five cases and 28 controls on the CanineHD chip. One control dog was genotyped on both chips, enabling the comparison of genotype calls between them. The assays were performed at the National Institutes of Health's Laboratory of Neurogenetics (Bethesda, MD) according to the manufacturer's instructions. The amplified DNA products were imaged using a BeadArray Reader (Illumina; San Diego, CA) and analyzed using Illumina's Bead Studio and Genome Studio software (Illumina; San Diego, CA).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Data was pruned such that individuals with less than a 95% call rate and those having a significant number of Mendelian errors were removed from further analysis. SNPs having a minor allele frequency of less than 1%, missing genotype calls greater than 10% and showing inconsistent calls between the two chips were also removed from further analysis. The pruned dataset was then used to perform a case-control and family-based association analysis. The PLINK toolset v1.07 (<ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/~purcell/plink/">http://pngu.mgh.harvard.edu/~purcell/plink/</ext-link>) was used to perform data pruning and the case-control allelic and genotypic association tests using a Bonferroni correction for multiple comparisons <xref ref-type="bibr" rid="pgen.1003991-Purcell1">[33]</xref>. As most of the Old English Sheepdogs genotyped belonged to a large pedigree, a family-based association test was also performed using the Family Based Association Test (FBAT) toolkit (v2.0.3) (<ext-link ext-link-type="uri" ns0:href="http://www.hsph.harvard.edu/~fbat/fbat.htm">http://www.hsph.harvard.edu/~fbat/fbat.htm</ext-link>) <xref ref-type="bibr" rid="pgen.1003991-Laird1">[34]</xref>. The FBAT toolkit improves upon the traditional transmission disequilibrium test (TDT) <xref ref-type="bibr" rid="pgen.1003991-Spielman1">[70]</xref> by handling factors such as missing parents, additional family members, different genetic models and phenotypes, as well as controlling for false positive associations due to population structure <xref ref-type="bibr" rid="pgen.1003991-Laird1">[34]</xref>. A Bonferroni correction for multiple comparisons was implemented using JMP Genomics (SAS; Cary, NC) with adjusted P-values less than 0.05 considered significant.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Receiver operator characteristic curve analysis was performed using Stata v10 (<ext-link ext-link-type="uri" ns0:href="http://www.stata.com">www.stata.com</ext-link>).</p><p>A 12 MB genomic region of interest was targeted using a custom array designed and manufactured by Roche NimbleGen (Madison, WI). 19,720 tiled probes approximately 60&#8211;90 bp in length covered approximately 96% of the targeted bases between 34,000,000&#8211;46,000,000 bp on CFA4. Unique probes were determined using the Sequence Search and Alignment by Hashing Algorithm (SSAHA) <xref ref-type="bibr" rid="pgen.1003991-Ning1">[71]</xref>. Targeted bases not captured were due to SSAHA's inability to determine valid probes possibly resulting from non-unique sequence, repetitive sequence, homopolymer runs or ambiguous bases.</p><p>Three micrograms of genomic DNA from three cases and three controls was used to prepare libraries for sequencing. DNA samples were fragmented by sonication (Covaris; Woburn, MA) and the fragments end-repaired, A-tailed, and ligated to indexing oligonucleotide adapters using NEBNext reagents (New England Biolabs; Ipswich, MA). Indexing adapters were provided by the Broad Institute (Cambridge, MA). The indexed DNA fragments were enriched for by PCR using AccuPrime (Life Technologies; Grand Island, NY) and Phusion (New England Biolabs, Ipswich, MA) DNA polymerases. DNA purification was done using QIAquick and MinElute PCR purification kits (Qiagen; Valencia, CA). Size selection and purification was done using Agencourt AMPure XP beads (Beckman Coulter; Beverly, MA). Analysis of DNA libraries was done using the Agilent Bioanalyzer DNA 1000 (Agilent Technologies; Santa Clara, CA) and Quant-iT dsDNA HS assay (Life Technologies, Grand Island, NY). Libraries were hybridized onto the array using a NimbleGen hybridization system (Roche NimbleGen; Madison, WI) at 42&#176;C for 70 hours. Arrays were washed and samples eluted using NimbleGen's elution system (Roche NimbleGen; Madison, WI). Post-capture amplification was done using the primers <named-content content-type="gene">5&#8242;-AAT GAT ACG GCG ACC ACC GAG-3&#8242;</named-content>and <named-content content-type="gene">5&#8242;-GAA GCA GAA GAC GGC ATA CGA-3&#8242;</named-content> with Phusion (New England Biolabs; Ipswich, MA) and AccuPrime (Life Technologies; Grand Island, NY) enzymes. Quantitative PCR (qPCR) using QuantiFast SYBR Green PCR kit (Qiagen; Valencia, CA) was done to estimate relative fold-enrichment of the target region (<xref ref-type="supplementary-material" rid="pgen.1003991.s006">Table S5</xref>).</p><p>Paired-end sequencing was done on Illumina's Hi-Seq 2000 machine (Illumina; San Diego, CA) at the Broad Institute (Cambridge, MA).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Sequence reads were aligned to the canine reference genome CanFam2 using the Burrows-Wheeler Aligner (BWA) <xref ref-type="bibr" rid="pgen.1003991-Li1">[72]</xref>. SNPs and insertion/deletions (indels) were determined using the Genome Analysis Toolkit (GATK; <ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/gatk/">http://www.broadinstitute.org/gatk/</ext-link>) <xref ref-type="bibr" rid="pgen.1003991-McKenna1">[35]</xref>. Sequence data was processed according to the current best practices for data analysis found on the online GATK Wiki (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/gsa/wiki/index.php/Main_Page">http://www.broadinstitute.org/gsa/wiki/index.php/Main_Page</ext-link>) and included; initial SNP and indel calling, correcting alignment errors due to indels and inaccurate base quality scores, SNP and indel calling after the realignment and recalibration, and filtering the data using standard filtering parameters. Sequence and variants were viewed with the Integrative Genomic Viewer (IGV) <xref ref-type="bibr" rid="pgen.1003991-Robinson1">[73]</xref>. Variants were identified in which all three affected dogs were homozygous for the non-reference allele, the two control carriers were heterozygous and the final control dog was either heterozygous or homozygous for the reference allele. Variants resulting in amino acid changes of coding regions of the genome were considered potential causative mutations for hereditary ataxia.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Variants considered as potential causative mutations were genotyped via Sanger sequencing in Old English Sheepdogs and additional dog breeds. The <italic>RAB24</italic> gene was also sequenced in affected dogs of breeds in which the A&gt;C <italic>RAB24</italic> variant was not present. Primers were designed using Primer3 (<ext-link ext-link-type="uri" ns0:href="http://frodo.wi.mit.edu/">http://frodo.wi.mit.edu/</ext-link>) <xref ref-type="bibr" rid="pgen.1003991-Rozen1">[74]</xref> or NCBI's Primer-BLAST (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/tools/primer-blast/">http://www.ncbi.nlm.nih.gov/tools/primer-blast/</ext-link>) <xref ref-type="bibr" rid="pgen.1003991-Ye1">[75]</xref> (<xref ref-type="supplementary-material" rid="pgen.1003991.s007">Table S6</xref>). PCR reactions included: 10&#215; Buffer B (Thermo Fisher Scientific; Waltham, MA), 25 mM MgCl<sub>2</sub> (Thermo Fisher Scientific; Waltham, MA), 10&#215; MasterAmp (Epicentre Biotechnologies Madison, WI), 25 mM dNTPs (Apex BioResearch Products, San Diego, CA), 10 &#181;M forward and reverse primer (IDT; Coralville, IA), Taq DNA polymerase (Apex BioResearch Products; San Diego, CA) and molecular grade water to a volume of 27 &#181;L. DNA amounts varied from approximately 25 ng to 100 ng per reaction. Thermocycler conditions varied by primer set (<xref ref-type="supplementary-material" rid="pgen.1003991.s008">Table S7</xref>). PCR products were purified using either Agencourt AMPure XP kit (Beckman Coulter; Beverly, MA), QIAquick PCR Purification Kit (Qiagen; Valencia, CA), or MinElute PCR Purification Kit (Qiagen; Valencia, CA). The sequencing of each purified PCR product (20&#8211;40 ng/&#181;l for 8 &#181;l) was carried out using the same forward and reverse primers used for PCR. Sequencing was performed by Eurofins MWG Operon (Huntsville, AL) following the standard BigDye Terminator v3.1 manufacture's protocol (Applied Biosystems; Foster, CA) with capillary electrophoresis carried out on the ABI 3730&#215;l DNA Analyzer (Applied Biosystems; Foster, CA).</p><p>Sections of cerebellum were collected from euthanized affected Old English Sheepdogs (n&#8202;=&#8202;2), affected Gordon Setters (n&#8202;=&#8202;2) and normal beagles (n&#8202;=&#8202;2), and the tissue immediately frozen in liquid nitrogen. RNA was extracted from approximately 30 mg of the frozen tissue using Qiagen's RNeasy Mini Kit (Qiagen; Valencia, CA) and concentrations measured using a ND-1000 UV-Vis spectrophotometer (Thermo Scientific; Wilmington, DE).</p><p>Reverse transcription was performed using the Stratagene AffinityScript Multiple Temperature cDNA synthesis kit (Agilent Technologies; Santa Clara, CA) and 0.5 &#181;g oligo(dT) and 0.2 &#181;g random primers. Thermocycler conditions were set to 25&#176;C for 10 minutes, 42&#176;C for 5 minutes, 25&#176;C for 60 minutes, and 72&#176;C for 15 minutes.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Primers to amplify the mRNA were designed using Primer3 (<ext-link ext-link-type="uri" ns0:href="http://frodo.wi.mit.edu/">http://frodo.wi.mit.edu/</ext-link>) <xref ref-type="bibr" rid="pgen.1003991-Rozen1">[74]</xref>, with forward primer <named-content content-type="gene">5&#8242;-CGTGTCTCCAGGCGTAGC-3&#8242;</named-content> and reverse primer <named-content content-type="gene">5&#8242;-ACTGGGGGTAGCTCAGAC-3&#8242;</named-content>. The approximately 850 bp PCR product was excised from an ethidium bromide stained 2% agarose gel and cleaned using the QIAquick Gel Extraction Kit (Qiagen; Valencia, CA). The cDNA of an affected Old English Sheepdog and Gordon Setter as well as a normal beagle were sequenced using the same forward and reverse PCR primers. Sequencing was performed as was done for genotyping the variants.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Quantitative PCR was performed using Applied Biosystems' StepOne Plus instrument and Invitrogen's Power SYBR Green Master Mix (Life Technologies; Grand Island, NY). <italic>RAB24</italic> primers were designed using Primer3 (<ext-link ext-link-type="uri" ns0:href="http://frodo.wi.mit.edu/">http://frodo.wi.mit.edu/</ext-link>) <xref ref-type="bibr" rid="pgen.1003991-Rozen1">[74]</xref> and PerlPrimer (<ext-link ext-link-type="uri" ns0:href="http://perlprimer.sourceforge.net/">http://perlprimer.sourceforge.net/</ext-link>) <xref ref-type="bibr" rid="pgen.1003991-Marshall1">[76]</xref>. The housekeeping gene <italic>RPS19</italic> (primers published <xref ref-type="bibr" rid="pgen.1003991-Brinkhof1">[77]</xref>) was used for normalization. 20 &#181;l reaction volumes containing 200 nM primers were done in triplicate. Cycling conditions consisted of 95&#176;C for 10 minutes followed by 40 cycles of 95&#176;C for 15 seconds and 60&#176;C for 1 minute with a melt curve performed at completion. Reaction efficiencies were calculated using a 6 point 2&#8758;1 dilution series of 100 ng cDNA. <italic>RAB24</italic> and <italic>RPS19</italic> efficiencies were estimated to be 90% and 88% respectively with both r<sup>2</sup> values of 0.999. Relative expression differences were calculated using the &#916;&#916;Ct-method and statistical significance determined using Microsoft Excel (2010) to perform an unpaired t-test on the relative differences between cases and controls.</p><p>For histopathology, tissues were fixed in 10% neutral buffered formalin, embedded in paraffin, sectioned to five micrometers and stained with hematoxylin and eosin, PAS, Luxol Fast Blue/Cresyl Violet and Bielschowsky Silver Stain. Immunohistochemistry for GFAP, ubiquitin and Rab24 was performed on formalin fixed paraffin embedded brain tissue. Rabbit polyclonal antibodies were used for all three antigens purchased from Dako (GFAP), Santa Cruz Biotechnology (ubiquitin) and Proteintech Group Inc. (Rab24). 5 &#181;m sections were cut, deparaffinized and rehydrated. GFAP and ubiquitin immunostaining were performed using the Dako Autostainer. Rab24 immunostaining was performed manually. The autostaining method included a five-minute retrieval for GFAP using Dako Proteinase K (Dako; Carpinteria, CA), followed by blocking of endogenous peroxidase activity using 3% hydrogen peroxide for 10 minutes. The primary antibody was applied (concentrations and incubation time and conditions are given in <xref ref-type="supplementary-material" rid="pgen.1003991.s009">Table S8</xref>), and the product was visualized by incubation with Dako Envision and rabbit polymer (Dako; Carpinteria, CA) for 30 minutes followed by application of DAB for five minutes. Slides were counterstained with hematoxylin, dehydrated through sequential alcohol immersion to xylene and cover-slipped. Rab24 immunostaining involved a citrate retrieval using a Pascal Pressurized Heating Chamber at 120&#176;C, followed by blocking of endogenous peroxidase activity using 3% hydrogen peroxide for 10 minutes. A protein block was then performed using normal goat serum (protein block/normal goat serum, Biogenex; Fremont, CA) applied for 20 minutes, followed by incubation with rabbit anti-Rab24 at a concentration of 1&#8758;50 for 1 hour at room temperature. Slides were washed with phosphate buffered saline (PBS) then biotinylated goat anti-rabbit antibody (Link; Biogenex, Fremont, CA) was applied diluted to 1&#8758;20 and incubated for 20 minutes at room temperature. Following washing, peroxidase conjugated streptavidin (Label; Biogenex, Fremont, CA) was applied and incubated for a further 20 minutes at room temperature. The product was developed under the microscope using DAB for about one minute to an appropriate level of staining. The slides were counterstained with hematoxylin, dehydrated through sequential alcohol immersion to xylene and cover-slipped.</p><p>Sections were prepared for transmission electron microscopy from freshly harvested cerebellar tissue as described previously <xref ref-type="bibr" rid="pgen.1003991-Urkasemsin1">[11]</xref>. Samples of cerebellar cortex were cut into approximately 1 mm<sup>2</sup> cubes and fixed in McDowell's and Trump's 4F:1G fixative prior to incubation in 1% osmium tetroxide for one hour. Following this period of fixation, samples were placed in a 1&#8758;1 mixture of Spurr resin (EMS Spurr resin kit 14300, Hatfield, PA) and acetone for 30 minutes and then placed in 100% resin changed three times at two hourly intervals. The final change of resin was polymerized at 70&#176;C for eight hours. Semi-thin sections (0.25 &#181;m) were cut, stained with 1% toluidine blue-O in 1% sodium borate and used to identify areas of interest. Ultrathin sections (70&#8211;90 nm) of these areas were stained with methanolic uranyl acetate (EMS 22400, Hatfield, PA), followed by lead citrate, and examined by TEM (FEI/Philips EM208S TEM, Oregon, USA). Reagents were obtained from Fisher Scientific (Pittsburgh, PA) unless otherwise indicated.</p><p>All animals evaluated in this study were privately owned pets. DNA samples were obtained with client consent and approval of the Institutional Animal Use and Care Committee or from DNA banks. Euthanasia and necropsies were performed at the owners' request.</p>