<p>Animal care and experimental animal procedures were performed in accordance with the NIH IACUC.</p><p>
<italic>TYR::CreER<sup>T2</sup></italic> and <italic>Sox10<sup>LacZ</sup></italic> (<italic>Sox10<sup>tm1Weg</sup></italic>) mice were rederived on and maintained by outcross to C57BL/6J <xref ref-type="bibr" rid="pgen.1003644-Bosenberg1">[34]</xref>, <xref ref-type="bibr" rid="pgen.1003644-Britsch1">[66]</xref>. <italic>Rosa26<sup>tm1sor</sup></italic> mice were obtained as homozygotes, maintained by intercross and bred together with <italic>TYR::CreER<sup>T2</sup></italic> mice to generate compound heterozygotes <xref ref-type="bibr" rid="pgen.1003644-Soriano1">[67]</xref>. <italic>Sox10<sup>fl</sup></italic> and <italic>Mitf<sup>vga9</sup></italic> mice were rederived on C57BL/6J and maintained by intercross <xref ref-type="bibr" rid="pgen.1003644-Finzsch1">[33]</xref>, <xref ref-type="bibr" rid="pgen.1003644-Tachibana1">[68]</xref>. The <italic>Tg(DctSox10)</italic> line (CF1-10, <xref ref-type="bibr" rid="pgen.1003644-Hakami1">[35]</xref>) was maintained through a combination of outcrossing to C57BL/6J and by intercross.</p><p>Mice were genotyped using DNA isolated from tail tips and PCR analysis. Primers for the <italic>TYR::CreER<sup>T2</sup></italic> allele, <named-content content-type="gene">5&#8242;-TCCGCCGCATAACCAGTGAA-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;- CGGAAATGGTTTCCCGCAGA</named-content>, were used to amplify the <italic>Cre recombinase</italic> sequence under standard PCR conditions (30 cycles of 45 s at 94&#176;C, 45 s at 65&#176;C and 60 s at 72&#176;C). <italic>Mitf<sup>vga9</sup></italic> and <italic>Sox10<sup>LacZ</sup></italic> alleles were detected using PCR primers for &#946;-galactosidase, <named-content content-type="gene">5&#8242;-GATCCGCGCTGGCTACCGGC-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GGATACTGACGAAACGCCTGCC-3&#8242;</named-content>, using the same PCR conditions described above. Primers and cycling conditions for the <italic>Sox10<sup>fl</sup></italic> allele was described previously <xref ref-type="bibr" rid="pgen.1003644-Finzsch1">[33]</xref>. Zygosity for the <italic>Tg(DctSox10)</italic> transgene was determined by TaqMan analysis for two SNPs flanking the transgene on chromosome 1 that distinguish the original FVB donor strain from the C57BL/6 background strain (rs13475895 and rs13475987).</p><p>TAM (T5648, Sigma) was dissolved in corn oil or a combination of ethanol and sunflower oil. TAM treatment was performed by IP injection of lactating mothers or adults with 2 mg/animal for the number of days indicated.</p><p>Morphogenetic and adult hairs were staged according to <xref ref-type="bibr" rid="pgen.1003644-MllerRver1">[69]</xref>, <xref ref-type="bibr" rid="pgen.1003644-Paus1">[70]</xref>. Plucking was performed to synchronize adult hairs. Briefly, mice were anesthetized and hairs were removed by hand over a 1.5 cm&#215;2 cm region on the lower back. Hairs within this region were allowed to regenerate for 7 (7days post plucking, 7dpp) or 21 days (21 days post plucking, 21dpp). At each stage, the regions of the hair follicle were strictly defined based on visible anatomical landmarks (as described in <xref ref-type="bibr" rid="pgen.1003644-AubinHouzelstein1">[29]</xref>).</p><p>After shaving, skin from the lower back was immersed in 2% formaldehyde, and irradiated in a 540W variable wattage microwave (BioWave, Pelco) three times in intervals of 30 s irradiation followed by 60 s on ice. After microwaving, samples remained in fixative for an additional 25 minutes on ice. Skins were cryoprotected in 10% sucrose overnight, embedded in NEG-50 (Thermo Scientific), frozen and sectioned with a cryostat (10 &#181;m). For brightfield imaging, eosin-Y was sometimes used as a counterstain.</p><p>Sections for immunolabeling were first rinsed in PBS with 0.1% Tween 20. For nuclear antigens sections were subjected to antigen retrieval by boiling for 20 minutes in a Tris-EDTA solution and then permeabilized by treating with 1% Triton X-100 for 15 minutes. Sections were blocked for two hours in 1% bovine serum albumin (Sigma) and incubated with primary antibody overnight at 4&#176;C. Primary antibodies include those against DCT (1&#8758;300; TRP2, Santa Cruz Bio, sc-10451), SOX10 (1&#8758;75; Santa Cruz Bio, sc-17342), PAX3 (1&#8758;75, Developmental Studies Hybridoma Bank), MITF (1&#8758;1000; rabbit polyclonal, gift from Heinz Arnheiter, NINDS-NIH), c-KIT (1&#8758;100; ACK4, Cedarlane, CL8936AP), TRP1 and TYR (1&#8758;300; PEP-1 and PEP-7, rabbit polyclonals, gift from Vince Hearing, NCI-NIH), Cre recombinase (1&#8758;1000; Novagen, #69050-3), &#946;-galactosidase (1&#8758;32,000; MP Bio, #08559761), and cleaved Caspase-3 (1&#8758;100; Cell Signaling, #9661). After washing, sections were incubated in the appropriate secondary antibodies (1&#8758;5000; Alexafluor488 or 568, Invitrogen) for two hours at room temperature.</p><p>Sequential immunolabeling was performed for co-detection of DCT and SOX10 as these antibodies were both generated in goat. After labeling for SOX10 using the protocol described above, sections were blocked with rabbit &#945;-goat IgG FAB (1&#8758;10; Jackson Immuno, #305-007-003) for two hours, washed and then labeled for DCT as described above.</p><p>Brightfield and fluorescence microscopy was performed on a Zeiss Observer.D1 compound microscope. Images were obtained with an Axiocam Hrc camera using the Axiovision 4.8.2 software and processed with Adobe Photoshop. Quantitation of hair and cell phenotypes of immunolabeled tissue was performed on every fourth section of sequentially obtained skin sections. Data is presented as the mean &#177; standard deviation. Student's T-test with Bonferroni correction was used to determine statistical significance, unless stated otherwise.</p><p>Skin samples were fixed in 2% formaldehyde/0.2% glutaraldehyde for 30 min at room temperature. Samples were then washed with rinse buffer (2 mM MgCl2/0.1% NP40/PBS) and stained overnight in X-Gal solution consisting of 0.32 mg/ml X-Gal, 5 mM ferrothiocyanide, and 5 mM ferrithiocyanide in rinse buffer.</p><p>RNA from E17.5 skins from wild type and <italic>Tg(DctSox10)/+</italic> mice was reverse transcribed using the High Capacity cDNA Reverse Transcription Kit (ABI). Quantitative PCR was performed using Taqman Fast Universal PCR Master Mix (ABI) and the following Taqman gene expression assays: Sox10 (Mm01300162_m1) and Pax3 (Mm00435493_m1). All experiments were performed with technical and biological replicates of &#8805;3.</p>