<p>Extant <italic>Pax6</italic> and <italic>Mitf</italic> mutants and transgenics are described in <xref ref-type="supplementary-material" rid="pgen.1002757.s009">Table S1</xref> and were kept on a C57BL/6J background (Backcrosses: 20 for <italic>Pax6<sup>Sey-Neu</sup></italic>, 2 for <italic>PAX6<sup>Yac</sup></italic>, 7 for <italic>Mitf<sup>mi-vga9</sup></italic>, strain of origin of <italic>Mitf<sup>mi-rw</sup></italic> is C57BL/6J). C57BL/6J served as wild-type controls. <italic>Mitf<sup>mi-&#916;D</sup>/Mitf<sup>mi-&#916;D</sup></italic> targeted mice were generated using the recombineering technology. To generate the targeting construct, the <italic>D-Mitf</italic> promoter/D-Mitf exon and its flanking regions (15,801 kbp) were cloned using plasmid rescue from BACRP23-9A13. A floxed neomycin resistance expression cassette flanked by 200 bp of sequence flanking the <italic>D-Mitf</italic> promoter/D-Mitf exon was used to replace 5.8 kbp of the <italic>D-Mitf</italic> promoter/D-Mitf exon from the above plasmid and used for standard targeting of LC3 ES cells (genotype [C57BL/6Nx129S6]F1), giving 6 correctly targeted colonies/40 colonies tested. Of several germline transmitting lines, one, officially designated <italic>Mitf<sup>tm3Arnh</sup></italic>; MGI: 5050698, was selected and crossed with C57BL/6J&#8226;129S4-Prm1-Cre deleter mice (Jackson Laboratories, stock 003328, backcrossed twice to C57BL/6J). Offspring lacking the neo-cassette (<italic>Mitf<sup>tm3.1Arnh</sup></italic>; MGI: 5050699) were backcrossed to C57BL/6J twice and then bred to homozygosity with or without the corresponding <italic>Pax6</italic> alleles or transgenes.</p><p>Tyr-Tfec transgenic mice (strain C57BL/6N) were generated using a construct composed of (5&#8242;-3&#8242;) a 4721 bp Tyr RPE-specific enhancer, a 985 bp hsp70 minimal promoter <xref ref-type="bibr" rid="pgen.1002757-Murisier1">[29]</xref>, and a 1203 bp V5-tagged Tfec cDNA. Four transgenic lines were obtained and used for crosses with <italic>Mitf<sup>mi-rw</sup></italic> mice. Genotyping of mice was performed by Southern blot and/or PCR using primers shown in <xref ref-type="supplementary-material" rid="pgen.1002757.s011">Table S3</xref>. All animal experiments were covered by approved animal protocols.</p><p>Immunostaining and in situ hybridizations were performed as described previously, using 16 &#181;m thick coronal cryostat sections <xref ref-type="bibr" rid="pgen.1002757-Bharti2">[14]</xref>. All eye sections are shown with the dorsal side up. Cell proliferation analysis was done by phosphohistone H3 staining of three different embryos. Immunostaining of adult mouse RPE flat-mounts was performed as described by <xref ref-type="bibr" rid="pgen.1002757-Campos1">[43]</xref>. A Zeiss LSM510 confocal (Zeiss, Thornwood, NY) and a Nikon E800 (Nikon, Melville, NY) microscope were used to record immunostainings. A Polyvar microscope (Reichert Jung, Depew, NY) was used for recording in situ hybridizations. For antibodies and in situ probes, see <xref ref-type="supplementary-material" rid="pgen.1002757.s012">Tables S4</xref> and <xref ref-type="supplementary-material" rid="pgen.1002757.s013">S5</xref>.</p><p>For RNA expression, E9.5 and E10.5 optic vesicles or RPE and retinal fractions from E11.5&#8211;E18.5 embryos and P0 mice were prepared as described. RNA was extracted from pools of 20&#8211;40 individual samples and RT-PCR and real time PCR was performed according to previously published protocols <xref ref-type="bibr" rid="pgen.1002757-Bharti2">[14]</xref>. For primers, see <xref ref-type="supplementary-material" rid="pgen.1002757.s009">Table S1</xref>. Statistical analysis was done using two-tailed Student's t-test. Statistical significance of the data is represented as (&#248;) non-significant; (*) p&lt;0.05; (**) p&lt;0.01; and (***) p&lt;0.001. For microarray analysis, the Affymetrix platform was used in collaboration with the NHGRI/NINDS microarray core facility and bioinformatics core.</p><p>Pools of optic vesicles from &#8764;20 individual E10.5 eyes or RPE fractions from &#8764;30 individual E11.5 eyes were prepared as previously described <xref ref-type="bibr" rid="pgen.1002757-Bharti2">[14]</xref>. ChIP assays using ChIP IT kit from Active Motif (Carlsbad, CA) was performed as described. Primers and antibodies are given in <xref ref-type="supplementary-material" rid="pgen.1002757.s011">Tables S3</xref> and <xref ref-type="supplementary-material" rid="pgen.1002757.s013">S5</xref>.</p><p>Reporter assays were performed using ARPE19 (CRL-2302, ATCC, Manassas, VA) and the dual luciferase assay kit (Promega, Madison, WI). Statistical analysis was done using two-tailed Student's t-test. Details about the plasmids used for reporter assays are given in <xref ref-type="supplementary-material" rid="pgen.1002757.s011">Table S3</xref>.</p><p>Cultures were established from E10.0 embryonic heads and maintained for 48&#8211;72 hours using published protocols <xref ref-type="bibr" rid="pgen.1002757-Nguyen1">[7]</xref>. Polyacrylamide beads used for implantation were soaked for an hour with BSA, recombinant DKK3 (R&amp;D Systems, Minneapolis, MN), recombinant FGF2 (R&amp;D Systems, Minneapolis, MN), or their combinations at the indicated concentrations. In situ hybridization/immunostaining of the vesicles was performed on 16 &#181;m thick cryostat sections.</p>