<p>Fungal strains and their sources are listed in <xref ref-type="supplementary-material" rid="pgen.1003323.s006">Table S1</xref>. The <italic>Epichlo&#235;</italic> and <italic>Neotyphodium</italic> species, <italic>Claviceps fusiformis</italic>, <italic>Claviceps paspali</italic>, and <italic>Aciculosporium take</italic> were cultured on potato dextrose agar (PDA) on a cellophane layer, or in potato dextrose broth (PDB) with shaking at 23&#176;C. Mycelia were collected by centrifugation for 20 min at 5525&#215; <italic>g</italic>, frozen and lyophilized prior to DNA isolation. Culture conditions for <italic>C. purpurea</italic> were as in Mey et al. <xref ref-type="bibr" rid="pgen.1003323-Mey1">[80]</xref>.</p><p>Because <italic>Periglandula ipomoeae</italic> is so far nonculturable, the adaxial sides of the leaves of an infected host plant (<italic>Ipomoea asarifolia</italic>) were wetted with deionized water, and mycelia were picked off with a scalpel, placed into a vial with 70% ethanol, and stored at &#8722;20&#176;C. The mycelium was harvested by centrifugation, frozen and lyophilized.</p><p>DNA was isolated by the method of Al-Samarrai et al. <xref ref-type="bibr" rid="pgen.1003323-AlSamarrai1">[81]</xref> or, for <italic>C. purpurea</italic>, by the method of Cenis <xref ref-type="bibr" rid="pgen.1003323-Cenis1">[82]</xref>.</p><p>In order to document the stages of the life cycle of <italic>Epichlo&#235; festucae</italic> Fl1, the fungus was transformed with the plasmid, pCA49, which includes an enhanced cyan fluorescent protein (eCFP) coding sequence controlled by the <italic>Pyrenophora tritici-repentis TOXA</italic> gene promoter <xref ref-type="bibr" rid="pgen.1003323-Andrie1">[83]</xref>. Fungal transformation was performed as previously described <xref ref-type="bibr" rid="pgen.1003323-Spiering3">[72]</xref> and transformants were selected for resistance to hygromycin B. The transformants were introduced into seedlings of perennial ryegrass (<italic>Lolium perenne</italic>) <xref ref-type="bibr" rid="pgen.1003323-Latch1">[84]</xref>, and the symbiotic fungus was detected by tissue-print immunoblot with antiserum raised against a protein extract from <italic>Neotyphodium coenophialum</italic>
<xref ref-type="bibr" rid="pgen.1003323-An1">[85]</xref>. Plants were grown in the greenhouse, and vernalized to induce flowering and seed development <xref ref-type="bibr" rid="pgen.1003323-Chung1">[86]</xref>. Plant tissues were dissected manually with the aid of a dissecting scope, placed on a glass slide in a drop of 50% glycerol, and covered with a coverslip. Confocal micrographs were generated with an Olympus FV1000 point-scanning/point-detection laser scanning confocal microscope, equipped with a 440 nm laser. Emission fluorescence was captured and collected at 467&#177;15 nm through the eCFP filter. Image acquisition was performed at a resolution of 512&#215;512 pixels and a scan rate of 20 &#181;s pixel<sup>&#8722;1</sup>. The objective, Olympus water immersion PLAN APO 20&#215;-Water (NA 0.75), was used for observing and generating micrographs. FLUOVIEW 1.5 software (Olympus) was used to control the microscope and export images as TIFF files.</p><p>Methods of analyses were as described previously for ergot alkaloids <xref ref-type="bibr" rid="pgen.1003323-Haarmann2">[75]</xref>, <xref ref-type="bibr" rid="pgen.1003323-Panaccione1">[87]</xref>, lolines <xref ref-type="bibr" rid="pgen.1003323-Faulkner1">[88]</xref>, peramine <xref ref-type="bibr" rid="pgen.1003323-Spiering4">[89]</xref>, and indole-diterpenes <xref ref-type="bibr" rid="pgen.1003323-Rasmussen1">[90]</xref>.</p><p>For Sanger sequencing of the <italic>E. festucae</italic> E2368 genome, a clone library of randomly sheared genomic DNA was constructed as follows. Nuclear DNA was enriched by bisbenzimide-CsCl isopycnic ultracentrifugation, randomly sheared with a GeneMachines Hydroshear (DigiLab Genomic Solutions, Inc.), twice gel-fractionated to select DNA fragments of 3.5&#8211;4.5 kb, and cloned into pBCKS+ (Stratagene Cloning Systems, La Jolla, CA, USA). The library consisted of approx. 5 million clones, of which 2.5 million cfu were stored at &#8722;80&#176;C as aliquots of transformed T1-phage resistant <italic>Escherichia coli</italic> cells (Electromax DH10B; Invitrogen Corp., Carlsbad, CA, USA), and the remainder as ligation mixture. The <italic>E. coli</italic> transformants were grown on LB agar with chloramphenicol (25 mg/L). Colonies were picked by a QPix robot (Genetix, Hampshire, UK) into 96-deep-well plates with 2&#215; YT medium (1.5 ml per well), and grown overnight in a HiGro (GeneMachines, San Carlos, CA, USA) oxygenated shaking incubator for microtiter plates. The plasmids were purified robotically (Biomek FX, Beckman Coulter Inc, Fullerton, CA, USA) with the Perfect-Prep Plasmid 96 kit (Eppendorf AG, Hamburg Germany). Sequence reactions and capillary electrophoresis were conducted using vector primers and BigDye3.1 (Applied Biosystems, Foster City, CA, USA) at 1/16th reaction strength. The reactions were cleaned by ethanol precipitation and capillary electrophoresis was performed in a model 3730 DNA analyzer (Applied Biosystems). Both ends of each plasmid were sequenced. Sequencing results indicated that 99.8 of the clones contained genomic DNA inserts.</p><p>For the <italic>E. festucae</italic> Fl1 genome, a library was prepared in the fosmid vector pCC1FOS (Epicentre). DNA was fragmented with a Hydroshear equipped with the LARGE assembly, at speed setting 36, for 15 cycles. The fragments were end-repaired with the End-It kit (Epicentre), size-selected by electrophoresis in 0.4% agarose gel with Gelgreen stain (Biotium), imaged with blue light, purified from the agarose with Gelase (Epicentre), and blunt-end ligated to the fosmid arms using Fast-link (Epicentre). <italic>Escherichia coli</italic> Epi-300 T1<sup>R</sup> cells were transformed and selected for chloramphenicol resistance.</p><p>All DNA sequencing was conducted at the University of Kentucky Advanced Genetic Technologies Center. Most sequencing was conducted on a Roche/454 Titanium pyrosequencer. DNA was nebulized and size-selected to approximately 600 bp with AMPure beads (Agencourt), and subjected to shotgun pyrosequencing using the GS FLX Titanium General Library Preparation Kit, GS FLX Titanium LV emPCR Kit (Lib-L), and GS FLX Titanium Sequencing Kit XLR70 (Roche). Paired-end pyrosequencing was also conducted for <italic>E. festucae</italic> Fl1 (2-kb fragments 960,278 true paired end reads), <italic>E. festucae</italic> E2368 (3.0-kb fragments, 113,208 true paired end reads), and <italic>Claviceps purpurea</italic> (3.0-kb fragments, 1,128,137 true paired end reads). For paired ends, DNA was sheared with a Hydroshear with standard assembly, 20 cycles at speed setting of 12, then size selected with AMPure beads (Agencourt). The GS FLX Titanium Paired End adaptor set from Roche was used with <italic>Cre</italic> Recombinase, Exonuclease 1, and <italic>Bst</italic> polymerase from NEB according to the Roche GS FLX Titanium 3 kb Paired End Library Preparation Method Manual. Survey sequencing was conducted on and Ion Torrent PGM (Life Technologies) according to manufacturer's instructions. Sanger sequencing of the <italic>E. festucae</italic> E2368 genome was conducted on paired-ends of a library of cloned 3.8-kb (ave.) DNA fragments as described above (119,114 reads incorporated). In addition, the E2368 sequence assembly incorporated 235 reads of ca. 11-kb clones of genomic DNA in the pJAZZ system (Lucigen), as well as directly cloned telomere-containing fragments <xref ref-type="bibr" rid="pgen.1003323-Farman1">[91]</xref>. Sanger sequencing of the <italic>E. festucae</italic> Fl1 genome was conducted on paired-ends of the fosmid library of cloned 36-kb (ave.) DNA fragments (7259 read-pairs incorporated).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Data sets consisted of 2.3 M to 6.0 M reads per genome. Assemblies of <italic>E. festucae</italic> Fl1 and E2368 genomes incorporated paired-end data from Sanger sequencing in addition to 454 paired-end and single-end pyrosequencing data from a Roche/454 Titanium sequencer. The <italic>C. purpurea</italic> assembly used both single-end and paired-end pyrosequencing reads. All other genome assemblies used single-end pyrosequencing reads, some supplemented with sequences obtained on an Ion Torrent PGM (Life Technologies). Pyrosequencing reads that were duplicates, very short (&lt;80 nt), or very long (&gt;650 nt), or that had more than 1% of uncalled bases, were purged using utility program prinseq-lite-1.5 (<ext-link ext-link-type="uri" ns0:href="http://prinseq.sourceforge.net">http://prinseq.sourceforge.net</ext-link>) as suggested in Huse et al. <xref ref-type="bibr" rid="pgen.1003323-Huse1">[92]</xref>. Ion Torrent reads were trimmed of all base-calls after the first 230 bases. All genomes were assembled using Newbler Assembler ver. 2.5.3 (Roche/4540) with default parameters and the -sio option to ensure proper order of input data, with single-end reads preceding any paired-end data, and paired-end read libraries (Sanger and pyrosequencing) ordered by increasing insert size. Assemblies were uploaded to GenBank (<xref ref-type="supplementary-material" rid="pgen.1003323.s007">Table S2</xref>), and are provided with annotations on GBrowse web sites (<ext-link ext-link-type="uri" ns0:href="http://www.endophyte.uky.edu">www.endophyte.uky.edu</ext-link>). The annotated assembly of the <italic>C. purpurea</italic> genome sequence can be viewed at <ext-link ext-link-type="uri" ns0:href="http://www.ebi.ac.uk/ena/data/view/Project:76493">http://www.ebi.ac.uk/ena/data/view/Project:76493</ext-link>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Repetitive DNA families in the genome sequences were defined by processing a self-BLASTN report from each genome using a custom PERL script (Amyotte S.G. et.al <italic>Manuscript in preparation</italic>) that identified sequences with multiple genome copies and classified these repeats into non-redundant families. The repeat families were then manually curated to correct or remove families misidentified in the automated process above. The genome distribution of repeated sequences was characterized using RepeatMasker version 3.2.9 <xref ref-type="bibr" rid="pgen.1003323-Smit1">[93]</xref>, with Cross_Match <xref ref-type="bibr" rid="pgen.1003323-Ewing1">[94]</xref> version 0.990329, with the final set of repetitive families serving as a custom library. Results have been included in the GBrowse web sites (<ext-link ext-link-type="uri" ns0:href="http://www.endophyte.uky.edu">www.endophyte.uky.edu</ext-link>). All unique repeats identified from the genome custom libraries were compared by reciprocal BLASTn to identify conserved sequences within and between each species. Repeat sequences with BLAST scores greater than 100 were used to develop a matrix table of corresponding repeat numbers. A common number was given to each repeat association to rapidly identify repeat families conserved across species. The matrix table was used to label repeats within each gene cluster with the universal repeat numbers (<xref ref-type="fig" rid="pgen-1003323-g006">Figure 6</xref>, <xref ref-type="fig" rid="pgen-1003323-g007">Figure 7</xref>, <xref ref-type="fig" rid="pgen-1003323-g009">Figure 9</xref>, <xref ref-type="fig" rid="pgen-1003323-g010">Figure 10</xref>, <xref ref-type="fig" rid="pgen-1003323-g011">Figure 11</xref>). Repeats were assigned to putative superfamilies, and families where possible, based on BLASTx analysis and the presence and orientation of terminal repeats (<xref ref-type="supplementary-material" rid="pgen.1003323.s010">Table S5</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Miniature inverted repeat transposable elements (MITEs) previously characterized in <italic>E. festucae</italic>
<xref ref-type="bibr" rid="pgen.1003323-Fleetwood2">[67]</xref> were identified in other genomes by BLASTn using a personal database. To determine whether short repeats found in non-epichloid clusters were MITEs they were analyzed for terminal inverted repeats using einverted (EMBOSS; <ext-link ext-link-type="uri" ns0:href="http://emboss.bioinformatics.nl/cgi-bin/emboss/einverted">http://emboss.bioinformatics.nl/cgi-bin/emboss/einverted</ext-link>). Individual repeat-containing loci were aligned using MUSCLE and manually analyzed for evidence of recombination.</p><p>RIP indices were calculated using a sliding window analysis with a 200 bp window and a step size of 20-bp (in the centromere-to-telomere direction). RIP indices (ApT/TpA) <xref ref-type="bibr" rid="pgen.1003323-Margolin1">[95]</xref> were calculated for each window. The process was repeated until the window met the end of the sequence (i.e partial windows were not counted). These operations were performed automatically using a perl script (<xref ref-type="supplementary-material" rid="pgen.1003323.s011">Protocol S1</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Gene predictions were conducted by various methods available in MAKER version 2.0.3 <xref ref-type="bibr" rid="pgen.1003323-Cantarel1">[96]</xref>. In the MAKER runs, assembled contigs were filtered against RepBase <xref ref-type="bibr" rid="pgen.1003323-Jurka1">[97]</xref> model organism &#8220;fungi,&#8221; using RepeatMasker <xref ref-type="bibr" rid="pgen.1003323-Smit1">[93]</xref> version open-3.2.8. Our MAKER runs used the predictors AUGUSTUS 2.3.1 (<italic>Fusarium</italic> model) <xref ref-type="bibr" rid="pgen.1003323-Stanke1">[98]</xref>, FGENESH 3.1.1 (<italic>Fusarium</italic> model) <xref ref-type="bibr" rid="pgen.1003323-Salamov1">[99]</xref>, GeneMark-ES 2.3a (self-trained), and SNAP 2006-07-28 (trained with <italic>C. purpurea</italic> gene predictions for genus <italic>Claviceps</italic>, and with <italic>E. festucae</italic> E2368 gene predictions for other genera). These <italic>ab initio</italic> predictions were supplemented with evidence from Clavicipitaceae proteins in the NCBI non-redundant protein database and from assembled E. festucae ESTs (unigenes). Relationships of predicted proteins to known protein families were assessed by running InterProScan <xref ref-type="bibr" rid="pgen.1003323-Enright1">[100]</xref> on the inferred protein sequences inferred from the predicted genes. Results, including MAKER and FGENESH predictions and subsequent analyses, have been included in a collection of web sites based on GBrowse 1.70 <xref ref-type="bibr" rid="pgen.1003323-Stein1">[101]</xref>, <xref ref-type="bibr" rid="pgen.1003323-Donlin1">[102]</xref>, posted at <ext-link ext-link-type="uri" ns0:href="http://www.endophyte.uky.edu">www.endophyte.uky.edu</ext-link>.</p><p>Gene modeling for <italic>C. purpurea</italic> was done similarly, by applying three different gene prediction programs: 1) FGENESH <xref ref-type="bibr" rid="pgen.1003323-Salamov1">[99]</xref> with different matrices (trained with <italic>Aspergillus nidulans</italic>, <italic>Neurospora crassa</italic> and a mixed matrix based on different species); 2) GeneMark-ES <xref ref-type="bibr" rid="pgen.1003323-TerHovhannisyan1">[103]</xref> and 3) AUGUSTUS <xref ref-type="bibr" rid="pgen.1003323-Stanke1">[98]</xref> with available ESTs as hints. The different gene structures were displayed in GBrowse <xref ref-type="bibr" rid="pgen.1003323-Stein1">[101]</xref>, <xref ref-type="bibr" rid="pgen.1003323-Donlin1">[102]</xref>, allowing manual validation of all coding sequences (CDSs). Annotation was aided by BLASTx hits between the <italic>C. purpurea</italic> genome and those from <italic>Blumeria graminis</italic>, <italic>Neurospora crassa</italic>, <italic>Fusarium graminearum</italic> and <italic>Ustilago maydis</italic>, respectively. For the cluster regions and selected genes of interest the best fitting model per locus was selected manually and gene structures were adjusted by manually splitting them or redefining exon-intron boundaries based on EST data where necessary.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Orthology analysis was conducted on FGENESH-predicted proteins with length 10 amino acids or greater. Each inferred protein sequence was assigned a unique label with a prefix indicating its source genome. The predicted genes were first compared to the curated ortholog groups in OrthoMCL-DB <xref ref-type="bibr" rid="pgen.1003323-Chen1">[104]</xref> version 4 using the OrthoMCL web service (<ext-link ext-link-type="uri" ns0:href="http://orthomcl.org/cgi-bin/OrthoMclWeb.cgi?rm=proteomeUploadForm">http://orthomcl.org/cgi-bin/OrthoMclWeb.cgi?rm=proteomeUploadForm</ext-link>), to which each predicted proteome was submitted independently. Next, the combined set of inferred proteins from all of the sequenced Clavicipitaceae was analyzed as described in the OrthoMCL algorithm document (<ext-link ext-link-type="uri" ns0:href="http://docs.google.com/Doc?id=dd996jxg_1gsqsp6">http://docs.google.com/Doc?id=dd996jxg_1gsqsp6</ext-link>). The software versions used for this procedure were: OrthoMCL version 2.0.2 <xref ref-type="bibr" rid="pgen.1003323-Li2">[105]</xref>, MCL-bio <xref ref-type="bibr" rid="pgen.1003323-Enright1">[100]</xref>, MCL version 10&#8211;201 (<ext-link ext-link-type="uri" ns0:href="http://micans.org/mcl/">http://micans.org/mcl/</ext-link>) <xref ref-type="bibr" rid="pgen.1003323-Edgar1">[106]</xref>, and NCBI BLAST version 2.2.25 <xref ref-type="bibr" rid="pgen.1003323-Altschul1">[107]</xref>.</p><p>As noted by Li et al. <xref ref-type="bibr" rid="pgen.1003323-Li2">[105]</xref>, an OrthoMCL-derived &#8220;ortholog group&#8221; may contain paralogs as well as orthologs. We used the COCO-CL <xref ref-type="bibr" rid="pgen.1003323-Jothi1">[108]</xref> software distribution to recursively divide the ortholog groups obtained from OrthoMCL into sub-groups. A division was accepted if it had a bootstrap score of 0.75 or greater and a split-score (number of taxa common to both sub-groups, divided by the number of taxa represented in the smaller sub-group) of 0.5 or greater; a high split score indicates that the group is likely to be the result of an ancient duplication event, as many taxa have representative protein sequences on both sides of the split. For these analyses, COCO-CL was slightly modified as follows. The multiple amino-acid-sequence alignments employed MUSCLE version 3.8.31 <xref ref-type="bibr" rid="pgen.1003323-Edgar1">[106]</xref> instead of ClustalW (version 1.83) <xref ref-type="bibr" rid="pgen.1003323-Chenna1">[109]</xref> because rigorous experiments with simulated protein data sets, have shown that MUSCLE is comparable or superior in speed and average accuracy to the best current methods, such as CLUSTALW, MAFFT, and T-Coffee <xref ref-type="bibr" rid="pgen.1003323-Edgar1">[106]</xref>. ClustalW was still used to compute distance matrices from MUSCLE's alignments. The remaining calls to ClustalW in the COCO-CL source code were converted to non-interactive mode, avoiding freezes that can occur when ClustalW prompts the user unexpectedly. Finally, we addressed a potential infinite loop generated by the COCO-CL clustering program when a cluster cannot be partitioned into precisely two subclusters. In our version this situation terminates the clustering program, leaving the cluster unpartitioned.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Results of OrthoMCL and COCO-CL have been included in the genome browser web sites (<ext-link ext-link-type="uri" ns0:href="http://www.endophyte.uky.edu">www.endophyte.uky.edu</ext-link>). Clicking an FGENESH prediction in the browser opens a data page that lists and hotlinks the prediction's homologs and orthologs, as well as a link to download the multiple sequence alignment for that cluster. A patch file is provided in Supporting Information (<xref ref-type="supplementary-material" rid="pgen.1003323.s012">Protocol S2</xref>).</p><p>For phylogenetic analysis, the following steps were performed on the Phylogeny.fr site <xref ref-type="bibr" rid="pgen.1003323-Dereeper1">[110]</xref>: Sequences were aligned with MUSCLE <xref ref-type="bibr" rid="pgen.1003323-Edgar2">[111]</xref>, the phylogenetic tree was inferred with PhyML <xref ref-type="bibr" rid="pgen.1003323-Guindon1">[112]</xref>, and branch support was estimated by the approximate likelihood-ratio test <xref ref-type="bibr" rid="pgen.1003323-Anisimova1">[113]</xref> with the SH-like option. Trees were compared by the Shimodaira-Hasegawa test <xref ref-type="bibr" rid="pgen.1003323-Shimodaira1">[114]</xref> implemented in phangorn <xref ref-type="bibr" rid="pgen.1003323-Schliep1">[115]</xref> with the default parameters and 10,000 bootstrap replications.</p><p>Sequences of cDNA generated by reverse transcription of mRNA provided the information required for manual annotation to refine models of alkaloid biosynthesis genes. Using the Qiagen RNeasy Plant Mini kit, RNA was isolated from symbiota composed of <italic>Lolium pratense</italic> (&#8202;=&#8202;<italic>Schedonorus pratensis</italic>) with <italic>E. festucae</italic> (see <xref ref-type="fig" rid="pgen-1003323-g001">Figure 1D</xref>). Tissues analyzed were newly emerged stromata and pre-anthesis inflorescences. RNA quality and quantity was checked on Bioanalyzer 2100 (Agilent) using plant RNA nano chip. A clone library was constructed by cDNA synthesis with the SMART kit (ClonTech) <xref ref-type="bibr" rid="pgen.1003323-Zhu1">[116]</xref>, normalization, and cloning into the &#955;Triplex2 vector (ClonTech). Transfected <italic>E. coli</italic> BM25.8 cells were grown with ampicillin to select clones, and plasmid DNA was isolated and sequenced by standard Sanger sequencing using BigDye version 3.0 and an Applied Biosystems (Life Technologies) model 3730 xl DNA sequencer.</p><p>For deep cDNA sequencing (RNA-seq), 10 &#181;g of high quality total RNA was used for cDNA library preparation according to the mRNA sequencing sample preparation guide (Illumina, Cat# RS 930-1001). Sources of RNA were inflorescences and stromata of <italic>L. pratense-E. festucae</italic> symbiota, laboratory id numbers 2194 and 2352. The libraries were validated on a 2100 Bioanalyzer using a DNA-1000 chip (Agilent). These libraries were used for bridge-PCR (SR cluster generation kit v4, Illumina) and 82-cycle single-read sequencing was conducted on a Genome Analyzer IIx (Illumina) in the DNA Facility at Iowa State University.</p><p>RNA-seq data from stromata and inflorescences, as well as RNA-seq data previously obtained from wild type <italic>E. festucae</italic> and a <italic>sak</italic>A mutant <xref ref-type="bibr" rid="pgen.1003323-Eaton1">[57]</xref> were used for genome-wide identification and annotation of the expressed protein-coding genes of the <italic>Epichlo&#235; festucae</italic> strains. The RNA-seq data were combined with previously generated RNA-seq data and assembled into TAC contigs (defined as a continuous exonic region with contiguous read coverage) on both E2368 and Fl1 assemblies by MapSplice <xref ref-type="bibr" rid="pgen.1003323-Wang1">[117]</xref>, which performs both spliced and unspliced alignment of RNA-seq reads to the reference genome. Then the combined read alignment coverage of the 6 tissues was used to detect exons. Exon boundaries are determined by splice junctions or the absence of the read coverage. Two TACs can be merged together by a splice junction connecting them. A TAC-contig is a maximum set of TACs that are linked together by splice junctions. If alternative splicing events exist, the alternative splice junction with more read alignment support is preferred. Because intergenic transcription or overlapping transcription of convergent genes sometimes led to merged gene models, junctions that crossed two FGENESH genes were filtered out, and TAC contigs that overlapped with more than two FGENESH-predicted genes were split according to the predicted gene boundaries. The 5&#8242; and 3&#8242; boundaries of gene structures were also trimmed based on the predicted genes.</p><p>All of the genes for ergoline, peramine, indole-diterpene and loline alkaloid biosynthesis, as well as genes used for phylogenetic analysis, were manually annotated. Many of the genes were previously characterized in the same or related species and strains by targeted reverse transcription of their mRNAs followed by cDNA sequencing <xref ref-type="bibr" rid="pgen.1003323-Markert1">[21]</xref>, <xref ref-type="bibr" rid="pgen.1003323-Tsai1">[39]</xref>, <xref ref-type="bibr" rid="pgen.1003323-Lorenz1">[41]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1003323-Young2">[43]</xref>, <xref ref-type="bibr" rid="pgen.1003323-Lorenz2">[70]</xref>, <xref ref-type="bibr" rid="pgen.1003323-Young3">[71]</xref>, <xref ref-type="bibr" rid="pgen.1003323-Young4">[118]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1003323-Wang2">[121]</xref>. Transcriptome information from <italic>E. festucae</italic>, including reads from cloned cDNAs and assembled TAC contigs, was used to model the gene exons. Cross-species comparisons, for example by using tBLASTn or tBLASTx, were employed to refine models in species for which transcript data were unavailable. In some cases, such as the newly discovered <italic>easO</italic> and <italic>easP</italic> genes, mRNA segments were amplified by reverse-transcription-PCR, and sequenced.</p><p>BLASTx and InterproScan <xref ref-type="bibr" rid="pgen.1003323-Enright1">[100]</xref> were employed to identify genes encoding enzymes that are signatures of SM gene clusters in the Ascomycota; namely, nonribosomal peptide synthetases (NRPS; IPR010071, IPR006163, IPR001242), polyketide synthases (PKS; IPR013968), DMATS-family aromatic prenyltransferases (IPR017795, Pfam PF11991), and terpene synthases/cyclases (IPR008949). The probable functions of proteins encoded by nearby genes were similarly assessed, as SM gene clusters contain various families of biosynthetic genes including mono- and dioxygenases, dehydrogenases, reductases, pyridoxal-phosphate (PLP)-cofactor enzymes, hydrolases, prenyltransferases and methyltransferases, as well as ABC or MFS efflux pumps and transcription regulators. However, many members of these enzyme families are involved in primary metabolism. Considering that most Clavicipitaceae can grow on minimal salts medium with sugars and inorganic nitrogen, those genes that had orthologs (identified by COCO-CL) among all of the sequenced genomes were considered probable primary metabolism genes. This interpretation was validated by the observation that most apparently active SM signature genes were flanked on one or both sides by ortholog groups with limited distribution among the 12 sequenced genomes. (Note that even after COCO-CL analysis, NRPS and PKS ortholog groups usually had several members in each genome, making it difficult to discern the distribution of their true orthologs, but this was not generally a problem for nearby genes.).</p><p>Genome and sequence accession numbers are listed in <xref ref-type="supplementary-material" rid="pgen.1003323.s007">Table S2</xref>.</p>