<p>We collected the publicly available data from cancer genome studies reported in PubMed from 2007 through December 2011 <xref ref-type="bibr" rid="pgen.1003816-Ley1">[1]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1003816-Kan1">[15]</xref> together with the 5 largest datasets available from the International Cancer Genome Consortium (ICGC). The cancer genome datasets varied widely in terms of sequencing strategies, mapping techniques and variant-calling algorithms, implying that the power to detect SBSs may differ depending upon the datasets and methodologies used <xref ref-type="bibr" rid="pgen.1003816-ORawe1">[85]</xref>. However, all studies excluded base variants present in matched-control tissues, such that the reported SBSs were changes attributed to somatic mutations in the tumor tissue. Matched controls were used for all patient samples. On average, between 6 <xref ref-type="bibr" rid="pgen.1003816-Kan1">[15]</xref> and 1834 <xref ref-type="bibr" rid="pgen.1003816-Ley1">[1]</xref> tumor-specific SBSs were reported in the EWS studies (between 1012 <xref ref-type="bibr" rid="pgen.1003816-Puente1">[3]</xref> and &gt;50,000 <xref ref-type="bibr" rid="pgen.1003816-Lee1">[8]</xref> in the GWS studies) (<xref ref-type="table" rid="pgen-1003816-t001">Table 1</xref>), which is &#8764;1&#8211;3 orders of magnitude lower than the numbers of non-synonymous and splice-site variants noted on average in whole-exome studies <xref ref-type="bibr" rid="pgen.1003816-Gilissen1">[86]</xref>. In addition to normal-tumor matched samples, single nucleotide polymorphisms present in dbSNP databases or in the Venter and Watson genomes <xref ref-type="bibr" rid="pgen.1003816-Ley1">[1]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Wei1">[10]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Pleasance2">[11]</xref> were also used to exclude common base variants. Differences in variant-calling power were mitigated in our study since we examined relative proportions of mutated sequences, rather than absolute mutation fractions.</p><p>A second source of variation in detecting SBSs among the cancer genome studies was the sequencing instrument used. Illumina sequencers have been reported to yield systematic base-call errors, especially at the last base of context-specific GG<underline>C</underline> and GG<underline>T</underline> sequences, which affect either the forward or reverse strand, and at inverted repeats <xref ref-type="bibr" rid="pgen.1003816-Meacham1">[87]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Nakamura1">[88]</xref>. The sequencing technologies employed included Illumina genome analyzers, SOLiD next-generation DNA sequencing, ion semiconductor sequencing, dubbed cPAL (combinatorial probe-anchor ligation) nanoballs, capillary electrophoresis, 454 pyrosequencing and mass spectrometry, often used in combination to verify variant calling. Illumina sequencers were the most commonly instruments employed in the studies whose data we used <xref ref-type="bibr" rid="pgen.1003816-Ley1">[1]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Puente1">[3]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Wang1">[4]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Stransky1">[6]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Agrawal1">[7]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Wei1">[10]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1003816-Berger1">[14]</xref>. The frequency of such base-call errors has been estimated at &#8764;0.1&#8211;0.3% before filtering, and even lower after filtering (SAMtools) <xref ref-type="bibr" rid="pgen.1003816-Meacham1">[87]</xref>. Considering that sequencing errors tend to occur over long simple repeat tracts, which have low mappability, and that systematic errors at GG<underline>T</underline> were ignored (we analyzed mutations at G&#8226;C bps only), it seems unlikely that base-call errors have biased our analyses by &gt;0.1%, an acceptable limit.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Approximately half of the human genome sequence comprises highly homologous repetitive DNA elements (<italic>Alu</italic> repeats, LINE elements etc.) and simple repeats, and an additional &#8764;3.6% contains Segmental Duplications, <italic>i.e.</italic> segments of &gt;1 kb in length that are present at multiple loci and which share &#8764;90&#8211;98% sequence similarity (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu">http://genome.ucsc.edu</ext-link>). Thus, because only the mappable genome may be scored for mutations, we used various methods to estimate the total number of mappable NGNN sequences to use as denominators in the <italic>f</italic>
<sub>i</sub> fractions (see below). Three methods were used for the GWS studies: 1) the entries with a mappability index of 1 (representing unique sequences) from file wgEncodeDukeMapabilityUniqueness35bp.bigWig (<ext-link ext-link-type="uri" ns0:href="http://hgdownload.soe.ucsc.edu/goldenPath/hg19/database/">http://hgdownload.soe.ucsc.edu/goldenPath/hg19/database/</ext-link>) generated for the ENCODE project by the Duke University Institute for Genome Sciences and Policy (IGSP) and at the European Bioinformatics Institute (EBI), which we refer to as Duke35; 2) we selected sequences from the mappability file wgEncodeCrgMapabilityAlign50mer.bw ((<ext-link ext-link-type="uri" ns0:href="http://hgdownload.soe.ucsc.edu/goldenPath/hg19/database/">http://hgdownload.soe.ucsc.edu/goldenPath/hg19/database/</ext-link>) <xref ref-type="bibr" rid="pgen.1003816-Derrien1">[89]</xref> (Donohue et al., unpublished data), referred to as CRG50; and 3) we retrieved all NGNN sequences in the GRCh37/hg19 release of the human genome assembly (chromFa.tar.gz file at <ext-link ext-link-type="uri" ns0:href="http://hgdownload.soe.ucsc.edu/goldenPath/hg19/bigZips/">http://hgdownload.soe.ucsc.edu/goldenPath/hg19/bigZips/</ext-link>) (T_hg19).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For the EWS studies, the SureSelect Human All Exon Kit (<ext-link ext-link-type="uri" ns0:href="http://www.genomics.agilent.com">http://www.genomics.agilent.com</ext-link>) was the most common platform reported <xref ref-type="bibr" rid="pgen.1003816-Wood1">[2]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Wang1">[4]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Agrawal1">[7]</xref>, <xref ref-type="bibr" rid="pgen.1003816-TCGARN1">[13]</xref>. A custom RefSeq CCDS PCR primer library was used to generate the Glioblastoma dataset <xref ref-type="bibr" rid="pgen.1003816-Parsons1">[5]</xref> and a set of 1,507 genes (oncogenes, tumor suppressors, &#8220;druggable targets&#8221;) were targeted in the Mixed cancer dataset <xref ref-type="bibr" rid="pgen.1003816-Kan1">[15]</xref>. Hence, three methods were used to estimate the number of mappable NGNN sequences in the EWS studies: i.e. the NGNN counts from 1) the file S0293689_Covered.bed (<ext-link ext-link-type="uri" ns0:href="http://www.Agilent.com">http://www.Agilent.com</ext-link>), listing the coordinates of exons targeted by the SureSelect Human All Exon Kit (AgilentV2); 2) the RefSeq exons sequences of CGR50 (CRG50_exon); and 3) the total RefSeq exons from file seq_gene.md at <ext-link ext-link-type="ftp" ns0:href="ftp://ftp.ncbi.nih.gov/genomes/H_sapiens/mapview/seq_gene.md.gz">ftp://ftp.ncbi.nih.gov/genomes/H_sapiens/mapview/seq_gene.md.gz</ext-link> (<italic>T_</italic>exons).</p><p>We defined <italic>f<sub>i</sub></italic>&#8202;=&#8202;<italic>m<sub>i</sub></italic>/<italic>t<sub>i</sub></italic>, where <italic>m<sub>i</sub></italic> was the number of mutations at a specific NGNN&#8226;NNCN sequence (henceforth designated as NGNN) and <italic>t<sub>i</sub></italic> was the total number of that sequence in one of the six &#8220;mappable&#8221; sets described above. The total number of NGNN sequences was doubled for the self-complementary AGCT, CGCG, GGCC and TGCA sequences since, like all NGCN sequences, they contain two mutations sites, one on the forward and one on the reverse strand. In relation to the counts of mutated NGNN motifs, if the .G.. occurred at the same genomic coordinate more than once within a cancer dataset, or if it was a homozygous mutation, it was considered as one count. Custom shell and FORTRAN scripts were used to obtain the total numbers of mappable NGNN and <italic>f</italic>
<sub>i</sub> fractions (see <xref ref-type="supplementary-material" rid="pgen.1003816.s016">Text S1</xref> for sample scripts). The normalized fractions of mutated DGNN sequences were defined as <italic>F<sub>i</sub></italic>&#8202;=&#8202;<italic>f<sub>i</sub>/&#8721;f<sub>i</sub></italic>, thus, <italic>&#8721;F<sub>i</sub></italic> scaled to 1. N indicates any base (A/C/G/T); D indicates A/G/T; B indicates C/G/T. As mentioned, sequence designation implies double-stranded DNA (<italic>i.e.</italic> AGTC&#8202;=&#8202;(<named-content content-type="gene">5&#8242;-AGTC-3&#8242;</named-content>)&#8226;(<named-content content-type="gene">5&#8242;-GACT-3&#8242;</named-content>)). The average base stacking free energies &lt;&#916;G(&#957;)&gt; were obtained from Friedman and Honig <xref ref-type="bibr" rid="pgen.1003816-Friedman1">[51]</xref> by using the &#916;G(&#957;) (&#949;<sub>i</sub>&#8202;=&#8202;2) values for the three base steps (DpG + GpN + NpN)/3. The free energy of base stacking &#916;G(&#957;) is an estimate of the absolute contribution of base stacking to nucleic acid stability in the absence of hydrogen bonding interactions, and contains a contribution from nonpolar plus electrostatic forces, as assessed from a theoretical approach using the Amber force field and a continuum solvation model of water. The largest contribution to &#916;G(&#957;) was found to arise from nonpolar <xref ref-type="bibr" rid="pgen.1003816-Friedman1">[51]</xref>, as opposed to electrostatic, interactions. Nonpolar interactions were contributed for the most part by enhancement in the Lennard-Jones component as a result of close packing, and to a smaller extent from hydrophobic interactions. Thus, the &#916;G(&#957;) values follow the same trend as the nonpolar contributions to free energies of base stacking &#916;G<sub>np</sub>(&#957;), <italic>i.e</italic>. purine-purine &gt;&gt; purine-pyrimidine &gt; pyrimidine-purine &gt; pyrimidine-pyrimidine, in qualitative agreement with experimental determinations <xref ref-type="bibr" rid="pgen.1003816-Friedman1">[51]</xref>. The relative enrichment <italic>E</italic> of sequence <italic>i</italic>, <italic>E<sub>i</sub></italic>, was defined as the ratio <italic>D<sub>i</sub></italic>/<italic>T<sub>i</sub></italic>, where <italic>D<sub>i</sub></italic>&#8202;=&#8202;<italic>d<sub>i</sub></italic>/&#8721;<italic>d<sub>i</sub></italic>, <italic>d<sub>i</sub></italic> being the number of times sequence <italic>i</italic> was mutated at least twice at the same hg19 coordinate and <italic>T<sub>i</sub>&#8202;=&#8202;t<sub>i</sub>/&#8721;t<sub>i</sub></italic>, <italic>t<sub>i</sub></italic> being the total number of occurrences of sequence <italic>i</italic> exome-wide (T_exons). Finally, <italic>S</italic>&#8202;=&#8202;<italic>s<sub>n</sub></italic>/&#8721;<italic>s<sub>n</sub></italic> and <italic>s<sub>n</sub></italic>&#8202;=&#8202;<italic>t<sub>n</sub></italic>/<italic>c<sub>n</sub></italic>, <italic>t<sub>n</sub></italic> being the number of times the combined <italic>(top)</italic> sequences were recurrently mutated and <italic>c<sub>n</sub></italic> being the total number of NS substitutions in a particular type of cancer.</p><p>Three-dimensional structures of the 12 possible double-stranded DGN trinucleotides were constructed using w3DNA <xref ref-type="bibr" rid="pgen.1003816-Zheng1">[90]</xref>. Hydrogen atoms, atomic charges and four neutralizing Na<sup>+</sup> counterions were assigned to each sequence according to the <italic>amber99</italic> force field <xref ref-type="bibr" rid="pgen.1003816-Wang2">[91]</xref>, using UCSF CHIMERA <xref ref-type="bibr" rid="pgen.1003816-Yang1">[92]</xref>. Na<sup>+</sup> counterions were positioned next to the four DNA backbone phosphates. Each trinucleotide was energy minimized <italic>in vacuo</italic> using the 10,000 steps steepest descent algorithm and the <italic>amber99</italic> force field in GROMACS 4.5.1 <xref ref-type="bibr" rid="pgen.1003816-Hess1">[93]</xref>. Ten and 14 &#197; cutoffs were used for Coulomb and van der Waals interactions, respectively.</p><p>VIPs were computed using Kohn-Sham density functional theory (DFT) <xref ref-type="bibr" rid="pgen.1003816-Kohn1">[94]</xref> employing the Minnesota M06-2&#215; functional <xref ref-type="bibr" rid="pgen.1003816-Zhao1">[95]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Zhao2">[96]</xref> with all-electron 6&#8211;31G(d) basis sets <xref ref-type="bibr" rid="pgen.1003816-Harihara1">[97]</xref>, <xref ref-type="bibr" rid="pgen.1003816-Francl1">[98]</xref>, as implemented in the GAMESS electronic structure package <xref ref-type="bibr" rid="pgen.1003816-Schmidt1">[99]</xref>, and including backbone phosphate groups and sodium counter ions in addition to the DGN double-stranded bases. The M06-2&#215; functional was used since this method provides accurate descriptions of hydrogen bonding and stacking interactions between base-pairs. We reasoned that the DGN set would provide the same type of information as the computationally more demanding NDGN set. Molecular orbitals were depicted using the MacMolPlt graphics program <xref ref-type="bibr" rid="pgen.1003816-Bode1">[100]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For individual patient samples, mutations were collated and sorted into lists of genes carrying mutations using customized R scripts (<ext-link ext-link-type="uri" ns0:href="http://www.r-project.org/">http://www.r-project.org/</ext-link>). The gene lists for each sample were entered into our pattern extraction pipeline analysis (PPEP) <xref ref-type="bibr" rid="pgen.1003816-Yi1">[101]</xref>, as implemented in the WPS package <xref ref-type="bibr" rid="pgen.1003816-Yi2">[102]</xref>, to obtain the ListHit of genes (number of genes from each list that are annotated to each pathway) for each of the BioCarta pathways. For each tumor type, each pathway was ranked on the basis of how frequently it was &#8220;hit&#8221; by individual patient samples and the ranking scores were obtained as the percentages of patient samples that had at least one hit in the corresponding pathway, using customized R scripts. The tumor type ranking scores for each pathway were combined and used to rank the pathways for all tumor types. The highest ranked pathways represent the most &#8220;popularly&#8221; hit pathways amongst all types of tumors. For each highly ranked pathway, the genes carrying the mutations were retrieved from each patient sample, ranked and displayed as gene-level heatmaps. For the pathway analysis of recurrent NS substitutions, the relevant genes for each tumor type were collated into lists and subjected to PPEP analysis, as described above.</p><p>Agglomerative hierarchical clustering dendrograms <xref ref-type="bibr" rid="pgen.1003816-Fernandez1">[103]</xref> were built using either the regression coefficients, <italic>r</italic>, between the fractions of mutated DGN sequences, <italic>f</italic>(DGN), and the VIP values, or the absolute orthogonal distances (Manhattan distances) between each <italic>f</italic>(NGNN) data point for all datasets. All-to-all comparisons were performed, allowing the relative estimation of all components of the systems, including the reference VIP branch.</p>