<p>All animal work and experiments have been approved by The Queensland Institute of Medical Research Animal Ethics Committee</p><p>To disrupt the <italic>Setx</italic> gene a highly-effective recombineering approach was employed <xref ref-type="bibr" rid="pgen.1003435-Chan1">[54]</xref>. Briefly, two cassettes, a loxP-F3-PGK-EM7-Neo-F3 (Neo) cassette was inserted into a BAC clone (RP23-389D11), Children's Hospital Oakland Research Institute corresponding to mouse chromosome 2 and covering the <italic>Setx</italic> genomic sequence. The Neo cassette which provides positive selection in ES cells was flanked by a 5&#8242; homology arm of 6.8 kb and a 3&#8242;homology arm of 3 kb. ES cells were then transfected with the linearized targeting vector and selected with 150 &#181;g/ml of G418. Successful recombinant ES clones were determined by Southern blotting with a specific probe and PCR genotyping, and targeted cells (+neo) were subsequently micro-injected into C57BL6/129Sv mice blastocysts to generate chimeras. Excision of the Neo cassette was obtained by crossing the chimeras with a <italic>Cre</italic> deleter stain to generate <italic>Setx<sup>&#8722;/&#8722;</sup></italic> mice containing only a <italic>LoxP</italic> site.</p><p>The mice were weaned at 21 days post-partum and ear clipped for identification. Genotyping was carried out by PCR on genomic DNA isolated from tail tips. Tail tips were lysed in directPCR Lysis eagent (Qiagen, USA) as recommended by the manufacturer. The primers used were In3F: <named-content content-type="gene">5&#8242;-TTTAAGGAACAGTGCTGC-3&#8242;</named-content>, In3R: <named-content content-type="gene">5&#8242;-ATGAAGCAGGTAGGATT-3&#8242;</named-content> and LoxPR: <named-content content-type="gene">5&#8242;-CGAAGTTATATTAAGGGT-3&#8242;</named-content>. PCR Cycling conditions were as follows: 35 cycles, denaturation at 95&#176;C for 30 sec, annealing at 49&#176;C for 30 sec, extension at 72&#176;C for 1 min, with a final cycle and extension of 7 min at 72&#176;C. Two PCR products were generated, a wild-type PCR product of 600 bp, and the targeted PCR product of 339 bp. PCR products were electrophoresed at 100 V for 30 min on 2% TAE Agarose (Boehringer Mannheim, Amresco, Lewes, UK) stained with Ethidium bromide and visualised with UV transillumination using a GelDoc XR (Biorad Laboratories Inc, UK).</p><p>Testes from adult (35-day-old), 4 months, 8 months and 12 month-old mice were collected and fixed in PBS buffered 10% formalin, embedded in paraffin block and sectioned at 4 &#181;m. Sections were stained with Hematoxylin and Eosin (H&amp;E) and Toluidine blue. Slides were examined under light microscope and then scanned using Scanscope CS system (Aperio Technologies, Vista, USA). Images corresponding to &#215;10 and &#215;20 magnification were captured and assembled into Adobe Photoshop 7 (Adobe Systems Inc, USA).</p><p>Total RNA was isolated from 35-day-old wild type and knockout mice testes using the RNeasy mini kit (Qiagen, USA) according to the manufacturer's protocol. RNA concentrations were determined by UV spectrophotometry using a Nanodrop ND-1000 (Thermo scientific, USA). cDNA was made from 5 &#181;g of purified RNA. Briefly, RNA was mixed with 1 &#181;l of random hexamer primers (Bio-Rad Laboratories Inc. USA), 1 &#181;l of 10 mM dNTP mix and DEPC-treated water up to a 14 &#181;l volume. The mixture was heat-denatured at 65&#176;C for 5 min. 4 &#181;l of First Strand buffer (Invitrogen, USA), 1 &#181;l of 1 mM DTT, 1 &#181;l of RNAaseIN (Promega, USA), and 1 &#181;l of SuperScriptIII reverse transcriptase enzyme (Invitrogen, USA) was added to the mixture, and incubated for 10 min at 25&#176;C, then 60 min at 50&#176;C, 15 min at 70&#176;C, and chilled on ice. 1 &#181;l of RNAse H was subsequently added to each tube and incubated for 20 min at 37&#176;C, followed by heat inactivation for 20 min at 65&#176;C. The resulting cDNA were stored at &#8722;20&#176;C prior to use. Gene expression analysis was performed by PCR in a 2720 Thermal Cycler (Applied Biosystem, USA). Reactions (25 &#181;l) contained 14.5 &#181;l of sterile water, 50 ng of cDNA template, 1&#215; PCR Buffer II (Roche, Switzerland), 2.5 mM MgCl<sub>2</sub> (Roche, Switzerland), 20 &#181;M dNTPs, 1 &#181;M of each primer, and 5 &#181;l of AmpliTaq Gold DNA Polymerase (Roche, Switzerland). The primer pairs used for gene expression analysis are described in <xref ref-type="supplementary-material" rid="pgen.1003435.s009">Table S1</xref>. Amplification was for 30 cycles and cycling conditions were as follows: denaturation for 5 min at 95&#176;C for 30 sec, annealing at 55&#176;C for 30 sec, elongation for 1 min at 72&#176;C followed by a final extension step of 7 min at 72&#176;C. PCR reactions were separated on 2% TAE agarose gels and visualised as above.</p><p>Testes from 35 day-old mice were collected and ground with a pestle to disrupt their structure and lysed for 1 h at 4&#176;C on a rotating wheel with lysis buffer (50 mM Tris-HCl pH 7.5, 50 mM &#946;-glycerophosphate, 150 mM NaCl, 10% glycerol, 1% Tween 20, 1 mM PMSF, 5 mM DTT and 1&#215; EDTA-free Complete Protease inhibitor (Roche, Switzerland). Cellular debris were pelleted by centrifugation at 16,100&#215;g at 4&#176;C for 10 min, and protein concentration was determined using Lowry Assay (Bio-Rad Laboratories, Inc, USA). 2 mg of total cell extract were pre-cleared with 50 &#181;l of a mixture of 1&#8758;30 protein G+A beads (Millipore, Germany) for 3 hours at 4&#176;C on a rotating wheel. Extract were centrifuged for 5 min at 2000&#215;g, beads were removed, and 20 &#181;g of anti-human senataxin antibody (Ab1/Ab-3) was added to the extract. Extracts and antibody were incubated overnight at 4&#176;C on a rotating wheel to allow binding of the antibody to mouse senataxin. The next day, 50 &#181;l of protein G+A beads were added to the extract and incubated for 1 h at 4&#176;C on a rotating wheel. The immunoprecipitate was subsequently washed 3 times with lysis buffer and the beads were resuspended in gel loading buffer and separated on 5% SDS-PAGE at constant current (20 mA per gel) for 1.5 h. Once separated, proteins were transferred onto a nitrocellulose membrane (Hybond C, Amersham) for 1 h at 4&#176;C with constant voltage (100 Volts). Immunoblotting with anti-senataxin (Ab-1) antibody was performed using standard procedure as previously described <xref ref-type="bibr" rid="pgen.1003435-Suraweera1">[5]</xref>.</p><p>All spreads were made from testes collected from adult 35-day-old mice or at day 16, 20 and 22 post partum. Briefly, testes were decapsulated, finely chopped and rinsed in GIBCO DME medium (Invitrogen, USA). Large clumps were removed by centrifugation at 6780&#215;g for 5 min at room temperature. The remaining supernatant was centrifuged to pellet the cell suspension and mixed with 0.1M sucrose and spread onto glass slides pre-wetted with 1% paraformaldehyde and 0.1% Triton X-100 in PBS. Cells were fixed on the glass slides for 2 h at room temperature. The slides were subsequently washed with PBS and air-dried in the presence of a wetting agent, 1&#8758;250 Kodak Photo-Flo 200 (Kodak professional, USA). Once dried, spreads were stored at &#8722;80&#176;C. For immunostaining, slides, were rehydrated in dH20, and blocked in blocking buffer (0.2% BSA, 0.2% gelatine in PBS) for 30 min at room temperature. Spreads were incubated with primary antibodies overnight at 4&#176;C in a humidified chamber. Primary antibodies used included anti-SCP3 (1&#8758;100, NB300-230, Novus Biologicals), anti-SCP1-DyeLight conjugated (1&#8758;100, NB300-2201R, Novus Biologicals), anti-&#947;H2AX (1&#8758;100, Y-P1016, Millipore), anti-Rad51 (1&#8758;100, SC-33626, Santa Cruz Biotechnology), anti-Dmc1 (1&#8758;50, 2H12/4, Sapphire Bioscience), anti-Mlh1 (1&#8758;10, G168-15, Sapphire Bioscience), anti-ATR (1&#8758;100, SC-1887, Santa Cruz Biotechnology), anti-senataxin (1&#8758;100, Ab-1, <xref ref-type="bibr" rid="pgen.1003435-Suraweera1">[5]</xref>), anti-R-loop (1&#8758;100, S9.6), anti-RNA Pol II (phospho S2) (1&#8758;100, H5, ab24758, Abcam), anti-mouse Brca1 (1&#8758;300, David Livingston), anti-ubi-H2A (1&#8758;100, Clone E6C5, Millipore). Slides were subsequently washes 4 times for 3 min each in PBS on a rocker, and probed with the appropriate Alexa-Dye488 or Alexa-Dye594-conjugated secondary antibodies (1&#8758;250, Invitrogen, Molecular Probes, USA). Slides were washed again 4 times for 3 min each in PBS. DNA was stained with Hoechst 33342 (1&#8758;10,000) for 10 min at room temperature, and slides were mounted in Celvol 603 medium. Images were captured at room temperature using a digital camera (AxioCam Mrm, Carl Zeiss Microimaging Inc., Germany) attached to a fluorescent microscope (Axioskop 2 mot plus, Carl Zeiss Microimaging Inc., Germany) and the AxioVision 4.8 software (Carl Zeiss, Microimaging Inc. Germany). The objective employed was a 63&#215; Zeiss Plan Apochromat 1,4 Oil DIC (Carl Zeiss, Germany). Images were subsequently assembled in Adobe Photoshop 7 (Adobe Systems Inc, USA), and contrast and brightness were adjusted on the whole image panel at the same time.</p><p>Terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) is a method for detecting DNA fragmentation by labeling the terminal end of nucleic acids. TUNEL is a common method for detecting DNA fragmentation that results from apoptotic signaling cascades. The assay relies on the presence of nicks in the DNA which can be identified by terminal deoxynucleotidyl transferase (TdT), an enzyme that will catalyze the addition of Fluorescein-labeled dUTP. Paraffin sections were dewaxed and rehydrated with Shandon Varistain Gemini ES (Thermo Scientific, USA). TUNEL assay was performed using the Fluorescence <italic>in situ</italic> Cell Death Detection Kit (Roche, Switzerland) following the manufacturer's instructions. Slides were visualised under a fluorescent microscope and images were captured as previously described. The objective employed was a Zeiss Plan Neofluar &#215;10/0.30 (&#215;10 magnification). For double staining, TUNEL was carried out first followed by immunostaining as described below.</p><p>Slides with tissue sections were dewaxed and enzymatic antigen retrieval was performed by incubating the sections with 1&#8758;10 Trypsin dilution in PBS for 20 min at 37&#176;C. Slides were washed 3 times for 5 min with PBS at room temperature for 5 min each. Tissues sections were blocked in (20% FCS, 2% BSA, 0.2% Triton X-100) for 1 h at room temperature. Slides were incubated with anti-R-loop (1&#8758;100, S9.6) <xref ref-type="bibr" rid="pgen.1003435-Boguslawski1">[29]</xref> or anti-&#947;H2AX (1&#8758;100, Y-P1016, Millipore) antibody overnight at 4&#176;C in a humidified chamber. Slides were washed 5 times with 1&#215; PBS containing 0.5% Triton X-100 for 5 min each at room temperature. Alexa-Dye488 or Alexa-Dye594-conjugated secondary antibody was added for 1 h at 37&#176;C in a humidified chamber. Subsequently, slides were washed 3 times as before and Hoechst 33342 was added for 10 min to staining nuclei. Slides were finally washed twice and glass coverslips were mounted for imaging. Imaging was performed as described above. Confirmation of R-loop specific staining was obtained by pre-treating <italic>Setx<sup>&#8722;/&#8722;</sup></italic> testes sections with RNAse H (New England Biolads, USA).</p><p>To investigate a possible interaction between Brca1, ATR and senataxin we employed <italic>in situ</italic> Proximity Ligation Assay (PLA) (Duolink, Olink Bioscience, Uppsala, Sweden) on wild type (<italic>Setx<sup>+/+</sup></italic>) spermatocytes spreads. PLA allows the monitoring of protein interactions and modifications with high specificity and sensitivity. Protein targets can be readily detected and localized with single molecule resolution and objectively quantified in unmodified cells and tissues. Utilizing only a few cells, sub-cellular events, such as transient or weak interactions are revealed <italic>in situ</italic>. Two primary antibodies raised in different species recognize the target antigens of interest. Species-specific secondary antibodies, called PLA probes, each with a unique short DNA strand attached to it, bind to the primary antibodies. When the PLA probes are in close proximity, the DNA strands can interact through a subsequent addition of two other circle-forming DNA oligonucleotides. After joining of the two added oligonucleotides by enzymatic ligation, they are amplified via rolling circle amplification using a polymerase. After the amplification reaction, several-hundredfold replication of the DNA circle has occurred, labeled complementary oligonucleotide probes highlight the product. The resulting high concentration of fluorescence in each single-molecule amplification product is visible as a distinct bright spot when viewed with a fluorescence microscope. The assay was performed according to the manufacturer's protocol using rabbit anti-mouse Brca1 (1&#8758;200, David Livingston), sheep anti-senataxin (1&#8758;200, Ab-1) and goat anti-ATR antibody (1&#8758;100, SC-1887, Santa Cruz Biotechnology) antibodies and the corresponding anti-goat PLA Probe MINUS and anti-rabbit PLA probe PLUS. Identification of pachytene stage spermatocytes was determined by counterstaining with SCP3 antibody. PLA was also carried out on <italic>Setx<sup>&#8722;/&#8722;</sup></italic> spermatocytes spreads as a negative control. Slide mounting and imaging was performed as described above.</p>