<p>Participants included in these analyses were from the Howard University Family Study (HUFS), which has been described in detail previously <xref ref-type="bibr" rid="pgen.1004190-Adeyemo3">[48]</xref>. Briefly, the HUFS followed a population-based selection strategy designed to be representative of African American families living in the Washington, DC metropolitan area. Ethnicity was ascertained by self-report. The HUFS was approved by the Howard University Institutional Review Board, and was conducted in accordance with the Declaration of Helsinki. All participants provided written informed consent.</p><p>There were 1694 individuals included in the study after application of exclusion criteria: extreme phenotype values (HDLC&lt;20 or &gt;100 mg/dl; TG&lt;20 or &gt;500 mg/dl) and missing covariate data (age, body mass index [BMI], and gender). Given known perturbations to serum lipids that co-occur with Type 2 Diabetes, subjects with fasting blood glucose &#8805;126 mg/dl or taking physician-prescribed diabetes medication were excluded. These data include related participants. Family members were included in the common variant analysis, where adjustment for the random effect of family was possible (further description below). For the RV analysis, only unrelated family members were included (with one randomly-selected individual from each family, total n&#8202;=&#8202;919).</p><p>Serum measurements were made on fasting samples. HDLC and TG were determined enzymatically with the COBAS Integra 400 Plus Analyzer (Roche Diagnostics, Indianapolis, IN). <xref ref-type="sec" rid="s4">Methods</xref> were standardized to in-house and other appropriate reference methods: CDC reference methods for HDLC, Isotope dilution-mass spectrometry (ID-MS) for TG by the manufacturer.</p><p>Forty-eight unrelated subjects were selected from the HUFS, with 24 from the &#8220;favorable lipid profile&#8221; category (lowest TG quartile, highest HDLC quartile) and 24 from the &#8220;unfavorable lipid profile&#8221; category (highest TG quartile, lowest HDLC quartile). A sample of 48 individuals (96 chromosomes) provides a 99% probability of finding a sequence variant with a minor allele frequency (MAF) of 0.05, 86% probability of finding a variant with an MAF of 0.02 and 62% probability of finding a variant with an MAF of 0.01. The sequencing strategy applied methods as previously described <xref ref-type="bibr" rid="pgen.1004190-Biesecker1">[49]</xref>. Briefly, we sequenced known or predicted exons (including 5 bp of the flanking introns), &#8764;1 kb of the 5&#8242; untranslated region (UTR)/promoter region, and the 3&#8242; UTR if it is evolutionarily conserved. We also sequenced up to three of the most evolutionarily conserved regions of each gene that are not captured by the above features. Sequencing primers were designed to allow for sufficient overlap in individual sequencing reads. Florescent dye-terminator chemistry was used for bi-directional DNA sequencing, and sequence delineation was performed by automated ABI Prism 3730xl DNA sequencers, which typically give &gt;650 bp Q20/Phred20 read lengths. Mutations and heterozygotes were scored by automated comparative analysis against the provided reference sequence. All mutations were confirmed by manual curation.</p><p>Sequence variants were selected based on frequency &#8805;2% (to exclude variants that were only observed once, which may reflect error), for genotyping in the full HUFS sample. 174 of these identified variants were available from previous genotyping in this sample using the Affymetrix Genome-Wide Human SNP Array 6.0 <xref ref-type="bibr" rid="pgen.1004190-Adeyemo3">[48]</xref>. Primers were designed for an additional 103 SNPs (<xref ref-type="supplementary-material" rid="pgen.1004190.s004">Table S4</xref>), and genotyping was performed using the iPLEX Gold assay on the MassArray platform (Sequenom, San Diego, CA) as previously described <xref ref-type="bibr" rid="pgen.1004190-Mohlke1">[50]</xref>. Briefly, the PCR and extension primers were designed using MassArray designer Software. SNPs were excluded for assay failure (n&#8202;=&#8202;19), lack of variation (n&#8202;=&#8202;13), and genotype success rate &lt;90% (n&#8202;=&#8202;24). After these exclusions, none of the variants failed the filter for departure from Hardy-Weinberg equilibrium (p-value&lt;0.000001).</p><p>As our goal was to analyze the variation present in these genes in the individuals with extreme lipid profiles as comprehensively as possible, the list of variants identified by sequencing strategies was augmented by variants identified for these individuals from previous GWAS and by imputation based on 1000 Genomes dataset. Imputation was performed using MaCH-Admix <xref ref-type="bibr" rid="pgen.1004190-Liu1">[51]</xref>, an imputation tool specifically designed for use in admixed samples, using a cosmopolitan reference panel based on 1000 Genomes data. Imputed variants were filtered on an R<sup>2</sup> of 0.3. Rare variants with a MAF&lt;0.01 were excluded due to diminished confidence in imputation in variants below this threshold.</p><p>With previous (n&#8202;=&#8202;174) and new (n&#8202;=&#8202;47) genotyping, and imputation (n&#8202;=&#8202;1,194), there were a total of 1,415 variants were analyzed (<xref ref-type="supplementary-material" rid="pgen.1004190.s005">Table S5</xref>). These included 921 common variants (CVs; MAF &#8805; 0.05) and 494 less common or rare variants (RVs; MAF &lt; 0.05) distributed as follows within the candidate genes: <italic>ABCA1</italic> (636 CVs, 389 RVs), <italic>LCAT</italic> (6 CVs, 2 RV), <italic>LPL</italic> (139 CVs, 48 RVs), <italic>PON1</italic> (110 CVs, 33 RVs), and <italic>SERPINE1</italic> (30 CVs, 22 RVs).</p><p>TG was log-transformed, but the distribution of HDLC was approximately normal and left untransformed. Principal component analysis to assess population structure in the admixed African Americans was conducted using EIGENSOFT <xref ref-type="bibr" rid="pgen.1004190-Price1">[52]</xref>, and, as reported previously <xref ref-type="bibr" rid="pgen.1004190-Shriner1">[53]</xref>, the first principal component was retained on the basis of Velicer's minimum average partial test, and included in all analyses as a covariate representing overall proportion of African ancestry. All analyses were also adjusted for age, body mass index (BMI), and gender, and P&lt;0.05 after multiple test correction was considered statistically significant. All SNP effects are described in terms of the minor allele (thus, an &#8220;inverse association&#8221; indicates that the minor allele was associated with decreasing phenotypic values relative to the major allele).</p><p>Separate analytical strategies were employed for the analysis of common variants (CVs) and rare variants (RVs). Variants with MAF &#8805; 0.05 were included in the common variant (CV) analysis. For the CV analysis, the associations between variants and phenotypes were assessed in linear mixed models (Proc Mixed) in SAS 9.3 (SAS Institute, Cary, NC) with adjustment for age, BMI, gender, and the overall proportion African ancestry, as well as random clustering within families. For each variant, models were run for additive, dominant, recessive, and heterosis coding. To correct for the number of SNPs tested, P-values were adjusted for the effective number of SNPs (based on LD&gt;0.6) within the gene evaluated, as previously described <xref ref-type="bibr" rid="pgen.1004190-Ramos1">[54]</xref>. Briefly, this method involves conducting a covariance matrix for all of the variants within a gene, using this covariance to determine the LD-adjusted number of independent tests that were conducted when analyzing all of the variants in that gene, and multiplying the individual P-values by that correction factor.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All variants with MAF&lt;0.05 were included in the less common and rare variant analysis (hereafter referred to as the rare variant [RV] analysis). For the RV analysis, SKAT was used (<ext-link ext-link-type="uri" ns0:href="http://www.hsph.harvard.edu/skat/">http://www.hsph.harvard.edu/skat/</ext-link>) <xref ref-type="bibr" rid="pgen.1004190-Lee1">[55]</xref>. SKAT aggregates individual score tests statistics for a set of SNPs, returning a P-value for the set (in our implementation, each gene). Only unrelated participants were included (all unrelated and one randomly-selected individual from each family, n&#8202;=&#8202;919). SKAT accommodates adjustment for covariates, and all analyses were adjusted (as in the CV analysis) for age, BMI, gender, and overall proportion African ancestry. Bootstrap resampling under the null model (considering covariates) was conducted, and statistical significance for the RV analysis was declared after correction for a family-wise error rate of 0.05.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For follow-up in the <italic>LPL</italic> region, local ancestry at the locus was estimated as previously described <xref ref-type="bibr" rid="pgen.1004190-Shriner2">[56]</xref>. Briefly, ancestry at each locus was categorized as having 0, 1, or 2 chromosomes of African ancestry as estimated based on nearly 800,000 markers using LAMPANC version 2.3 <xref ref-type="bibr" rid="pgen.1004190-Sankararaman1">[57]</xref> and HapMap Phase II+III CEU and YRI reference allele frequencies (<ext-link ext-link-type="uri" ns0:href="http://hapmap.ncbi.nlm.nih.gov/downloads/frequencies/2010-08_phaseII+III/">http://hapmap.ncbi.nlm.nih.gov/downloads/frequencies/2010-08_phaseII+III/</ext-link>). A difference in genotype-phenotype association by local ancestry was evaluated in SAS using linear mixed models (PROC MIXED) with a genotype by local ancestry interaction term and evaluating models stratified by local ancestry.</p><p>Haplotype analysis was conducted in all candidate genes using a sliding window approach in PLINK <xref ref-type="bibr" rid="pgen.1004190-Purcell1">[58]</xref>, with up to 5 SNPs included in each haplotype and adjustment for covariates (age, gender, BMI, and overall proportion African ancestry). Permutation testing (1000 permutations) was employed to evaluate statistical significance.</p><p>Identified loci in African Americans were assessed for replication in a West African sample obtained from the African American Diabetes Mellitus study (AADM; described previously <xref ref-type="bibr" rid="pgen.1004190-Rotimi1">[59]</xref>). Briefly, AADM is a large-scale case-control study designed to explore the genetic and environmental determinants of T2D from West Africa, but only non-diabetic controls were included in this analysis. All participants provided written, informed consent. Variants were genotyped using the Affymetrix Axiom Genome-Wide Pan-African Array Set (&#8764;2.2 million markers), which is optimized for coverage of African-ancestry populations. Imputation was also conducted in this sample (as described above). A limited number of participants with genotype data remained after applying the exclusions described above (n&#8804;536). This sample had 80% power to detect an effect of 7% for logTG and 5 mg/dl for HDLC when the variant was common (MAF&#8202;=&#8202;0.05), and 80% power to detect effects of 15% and 11 mg/dl when the variant was less common (MAF&#8202;=&#8202;0.01). The variants for which there was at least moderate power (&gt;60%, QUANTO <xref ref-type="bibr" rid="pgen.1004190-Gauderman1">[60]</xref>) to detect an association of the magnitude observed in African Americans are described in the text.</p>