<p>We selected six samples for genome sequencing and generated single end and long mate pair SOLiD reads. We generated additional paired end (PE) sequence data on the Illumina HiSeq platform (<xref ref-type="supplementary-material" rid="pgen.1004016.s020">Text S1</xref>). For most downstream analyses, we also utilized sequence information from the Boxer reference genome (CanFam 3.0).</p><p>We aligned sequence reads to CanFam 3.0, with post-processing of aligned reads including the removal of duplicates, local realignment, and base quality recalibration (<xref ref-type="supplementary-material" rid="pgen.1004016.s022">Text S3</xref>). We then genotyped each sample individually, using the Genome Analysis Toolkit (GATK) pipeline <xref ref-type="bibr" rid="pgen.1004016-McKenna1">[34]</xref>. For SNV genotyping and analysis, we excluded repeats of recent origin, CpG sites, regions falling in copy number variants, and triallelic sites, and at the sample level we filtered out genotypes proximate to called indels, with excess depth of coverage, with low genotype quality scores, or where the SNV fell within five base pairs of another SNV (<xref ref-type="supplementary-material" rid="pgen.1004016.s023">Text S4</xref>).</p><p>We compared genotype calls based upon sequencing to those from the same samples using the Illumina CanineHD BeadChip, which consists of &gt;170,00 markers evenly spaced throughout the dog genome (<xref ref-type="supplementary-material" rid="pgen.1004016.s024">Text S5</xref>). We also analyzed variants overlapping those generated in a previous SNP array study of a large panel of dogs and wolves <xref ref-type="bibr" rid="pgen.1004016-vonHoldt1">[10]</xref>, and performed PCA on the combined data set to verify that NGS genotypes clustered with array genotypes for the same lineages (<xref ref-type="supplementary-material" rid="pgen.1004016.s024">Text S5</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We delineated segmental duplications in our six genomes by identifying regions with a significant excess depth of coverage (<xref ref-type="supplementary-material" rid="pgen.1004016.s025">Text S6</xref>). For this purpose, we aligned Illumina and SOLiD reads with MrFAST <xref ref-type="bibr" rid="pgen.1004016-Alkan1">[35]</xref> and drFAST <xref ref-type="bibr" rid="pgen.1004016-Hormozdiari1">[36]</xref> respectively. Absolute copy numbers were calculated using mrCaNaVar version 0.31 (<ext-link ext-link-type="uri" ns0:href="http://mrcanavar.sourceforge.net/">http://mrcanavar.sourceforge.net/</ext-link>). In the particular case of the previously reported <italic>AMY2B</italic> expansion in the dog lineage <xref ref-type="bibr" rid="pgen.1004016-Axelsson1">[15]</xref> we also examined patterns of copy number across 52 breed dogs, six Dingoes, and 40 wolves using qPCR (<xref ref-type="supplementary-material" rid="pgen.1004016.s025">Text S6</xref>).</p><p>In order to conduct demographic analyses on putatively neutral genomic regions without any apparent functional annotation, we first identified genic region using annotations from the union of refGene, Ensembl and SeqGene annotation databases, with the condition that all annotated transcripts had proper start and stop codons, and contained no internal stop codons (<xref ref-type="supplementary-material" rid="pgen.1004016.s026">Text S7</xref>). In addition, we defined conserved non-coding elements (CNEs) on the basis of phastCons scores <xref ref-type="bibr" rid="pgen.1004016-Siepel1">[37]</xref> (<xref ref-type="supplementary-material" rid="pgen.1004016.s026">Text S7</xref>).</p><p>We used the PSMC methods developed by Li and Durbin <xref ref-type="bibr" rid="pgen.1004016-Li2">[20]</xref> to infer the trajectory of population sizes across time for the six canid genome sequences (<xref ref-type="supplementary-material" rid="pgen.1004016.s027">Text S8</xref>).</p><p>To investigate the extent of gene flow between wolves and dogs subsequent to their divergence, we employed a method recently developed by Durand et al. <xref ref-type="bibr" rid="pgen.1004016-Durand1">[18]</xref>. This method tests for directional gene flow by testing for asymmetries in allele sharing between a source lineage (P3), and either of two receiving lineages (P1, P2) with reference to an outgroup (O). To focus on gene flow most germane to evolutionary processes influencing wolf-dog divergence, we restricted testing to those cases where one of the dog samples was P3, the other two (P1 and P2) were wolves, and viceversa (P3&#8202;=&#8202;wolf, P1 and P2&#8202;=&#8202;dogs). For more details, see <xref ref-type="supplementary-material" rid="pgen.1004016.s027">Text S8</xref>.</p><p>Our main demographic analysis is based on the Generalized Phylogenetic Coalescent Sampler (<italic>G-PhoCS</italic>) developed by Gronau et al. <xref ref-type="bibr" rid="pgen.1004016-Gronau1">[19]</xref> and which we applied to 16,434 1 kb loci chosen via a strict set of criteria to obtain putatively neutral loci (<xref ref-type="supplementary-material" rid="pgen.1004016.s028">Text S9</xref>). The prior distributions over model parameters was defined by a product of Gamma distributions using the default setting chosen by Gronau et al. <xref ref-type="bibr" rid="pgen.1004016-Gronau1">[19]</xref>. Markov Chain was run for 100,000 burn-in iterations, after which parameter values were sampled for 200,000 iterations every 10 iterations, resulting in a total of 20,001 samples from the approximate posterior. Convergence was inspected manually for each run. We conditioned inference on the population phylogeny based upon the neighbor-joining tree constructed from the genome-wide distance matrix described above (<xref ref-type="fig" rid="pgen-1004016-g004">Fig. 4A</xref>). We also constructed models under a &#8216;regional domestication&#8217; scenario, in which each dog lineage originated from a wolf lineage from the same geographic region, i.e. Basenji from Israeli wolf, Boxer from Croatian wolf, and Dingo from Chinese wolf. We assessed models in which the branch ancestral to dogs was sister to a particular extant wolf population, or one of internal branches in the wolf clade. In addition, we investigated the sensitivity of parameter estimates to choice of locus length, number of loci, intra-locus recombination, distance from coding exons, and selection on linked sites. For more details, see <xref ref-type="supplementary-material" rid="pgen.1004016.s028">Text S9</xref>.</p>