<p>Plasmids and bacterial strains are listed in <xref ref-type="table" rid="pgen-1002832-t001">Table 1</xref>. All three pQ&#223; plasmids express the indicated Q&#223; components constitutively and have been described <xref ref-type="bibr" rid="pgen.1002832-Priano1">[35]</xref>, <xref ref-type="bibr" rid="pgen.1002832-Arora1">[36]</xref>, <xref ref-type="bibr" rid="pgen.1002832-Mills1">[61]</xref>. The RT<sub>IN</sub> mutant carries a tandem duplication of 2158-UUAA-2161 that corresponds to 416&#8211;419 in the target sequence. Cell transformations with the plasmids were performed using CaCl<sub>2</sub>
<xref ref-type="bibr" rid="pgen.1002832-Sambrook1">[62]</xref>. Unless otherwise indicated, RTH cells were grown in Luria-Bertani medium (LB) supplemented with 2 mM CaCl<sub>2</sub> and 100 &#181;g/ml trimethoprim (TMP), while NR16205 cells were grown in LB containing 15 &#181;g/ml tetracycline. Cells and phages were plated using LB bottom agar with 2.0% Bacto agar. The top agar was always made up in distilled water. For counting plaques or scoring mutants, the top agar contained 0.4% Sigma-Aldrich Noble agar; for other uses, it contained 0.8% Bacto agar. All growth was at 37&#176;C.</p><p>NR16205 cells were transformed with pQ&#223;m100 (which expresses wt Q&#223;) and plated on NR16205 lawns to yield wt Q&#223; plaques, which were independently harvested into tubes containing 1 ml D broth (0.2% Bacto tryptone, 0.5% NaCl) and 25 &#181;l of chloroform. For one-step growth curves in RTH cells, 10 &#181;l of phage suspension from a wt Q&#223; isolate was mixed with 1 ml of cells at OD<sub>600</sub>&#8776;0.5 (10<sup>8</sup> cells/ml) at a multiplicity of infection (MOI)&#8776;0.01 for 20 min at room temperature, centrifuged to remove non-adsorbed phages, resuspended, and serially diluted in LB+TMP. Samples diluted 10<sup>3</sup>- and 10<sup>5</sup>-fold were held for 3 h at 37&#176;C with gentle shaking, and 100-&#181;l aliquots were removed from each dilution every 10 min and plated with RTH cells. Plates were incubated overnight and the follow-on titers were used to estimate Q&#223; densities over time. Three one-step growth experiments were conducted in parallel for each wt Q&#223; isolate used to generate one-step lysates. Visual inspection of the resulting curves sufficed to determine the time (&#8776;75 min) for Q&#223; to complete one infection cycle in RTH cells. These one-step curves were also used to estimate the burst size of wt Q&#223; in RTH cells according to the protocol detailed for RT<sub>IN</sub> and RT<sub>SUB</sub> (see <xref ref-type="supplementary-material" rid="pgen.1002832.s003">Table S1</xref>).</p><p>The distribution of <italic>RT</italic>
<sup>+</sup> revertants among <italic>RT</italic>
<sup>&#8722;</sup> bursts was monitored as in a previous study <xref ref-type="bibr" rid="pgen.1002832-Chao1">[8]</xref> using two different <italic>RT</italic>
<sup>&#8722;</sup> mutants (RT<sub>IN</sub> and RT<sub>SUB</sub> as described in <xref ref-type="table" rid="pgen-1002832-t001">Table 1</xref>). Preliminary measurements provided their burst sizes (<xref ref-type="supplementary-material" rid="pgen.1002832.s002">Figure S2</xref>, <xref ref-type="supplementary-material" rid="pgen.1002832.s003">Table S1</xref>) and revertant frequencies, which are needed to conduct the burst experiments. Ten and five independent experiments were carried out with RT<sub>SUB</sub> and RT<sub>IN</sub>, respectively, and &#8776;500 <italic>RT</italic>
<sup>+</sup> revertants were scored per mutant. In each experiment, &#8776;10<sup>6</sup> phages were added to 1 ml of RTH cells at OD<sub>600</sub>&#8776;0.5. After 20 min of adsorption at room temperature, the mixture was centrifuged for 1 min at 8,000 <italic>g</italic> and the pellet was resuspended in 1 ml LB broth. From the supernatant, 100 &#181;l were collected to estimate the amount of non-adsorbed phages. The resuspended pellet was further diluted and 50 aliquots of 100 &#181;l each were distributed into individual tubes, where infection was allowed to continue for &#8776;75 min at 37&#176;C and then stopped with 15 &#181;l of dichloromethane. Lysates were aerated for 30 min at 37&#176;C to allow the dichloromethane to evaporate and their entire volumes were then independently plated on NR16205 lawns. The observed distributions of <italic>RT</italic>
<sup>+</sup> revertants were compared to the expected Poisson distributions using G-tests for goodness-of-fit.</p><p>To limit the number of infection cycles to one before seeking spontaneous mutants, <italic>RT</italic> mutants were scored among the progeny of one-step growth of wt Q&#223; in RTH cells. RTH cells were infected with wt Q&#223; (MOI&#8776;0.01) as above and one-step lysates were recovered by adding chloroform after 75 min of growth. Samples from the lysates were plated on RTH lawns at &#8776;70 plaques per plate and well-isolated plaques were independently sampled into 96-well plates containing 0.6 ml D-broth per well (reserving six un-inoculated wells as cross-contamination controls). For each of four independent lysates, three different rounds of 630 isolations each were performed. In each round, a control plate containing 8 wt and 82 RT<sub>IN</sub> isolates was also established to confirm the ability of RTH cells to complement <italic>RT<sup>&#8722;</sup></italic> mutants and the inability of any RTH cells remaining in the isolates to grow in LB supplemented with tetracycline. Isolates were spotted in parallel on lawns of NR16205 and RTH cells using a 6&#215;8-array replica plater. After a few losses, a total of 7517 plaques were tested. Isolates that grew poorly in NR16205 cells were re-tested in both bacterial strains and the RT-coding genes of two independent sub-isolates per putative <italic>RT</italic> mutant were sequenced. After this first round of sequencing, two additional sub-isolates as well as the original isolate were sequenced for each verified <italic>RT</italic> mutant. The original wt Q&#223; isolate used to develop each lysate and two sub-isolates of it were also sequenced.</p><p>Plate lysates were prepared from RTH cells (0.25 ml at OD<sub>600</sub>&#8776;0.5) mixed with phages at MOI&#8776;0.1 in Noble top agar. After overnight incubation, the plates were covered with 7 ml of SM buffer with gelatin <xref ref-type="bibr" rid="pgen.1002832-Sambrook1">[62]</xref> and were gently rocked for 30 min. The SM buffer was recovered and 100 &#181;l of chloroform were added to each sample. Cell debris was removed by centrifugation at 12,000 <italic>g</italic> for 10 min. The supernatant was supplemented with polyethylene glycol (PEG 8000) to 10% w/v and NaCl to 1 M, incubated for 1 h on ice, and centrifuged at 3,000 <italic>g</italic> for 15 min at 4&#176;C <xref ref-type="bibr" rid="pgen.1002832-Yamamoto1">[63]</xref>. The pellets were resuspended in 2 ml of 10 mM MgSO<sub>4</sub>, 10 mM Tris-HCl, pH 8, and the resulting concentrated phages were used as sources for RNA purification. Phage RNA was isolated using the QIAamp Viral RNA Mini Kit. From the extracted RNAs, 10 &#181;g were then treated with DNase I (New England BioLabs) to degrade residual host DNA. The DNase-treated product was purified using the RNAeasy Mini Kit. From the purified RNA, 1 &#181;g was subjected to reverse transcription with the Omniscript RT Kit and about 25 ng of the RT product was amplified with PfuTurbo DNA polymerase (Stratagene). PCR products were confirmed by agarose gel electrophoresis, purified with the QIAquick PCR Purification Kit, and sequenced using BigDye Terminator v3.1 (Applied Biosystems). All kits were purchased from Qiagen and were used according to the manufacturer's recommendations. Sub-isolates showing secondary mutations were subjected to a second round of RT, amplification and sequencing. The primers utilized in the RT, PCR, and sequencing reactions and the PCR cycling parameters are listed in <xref ref-type="supplementary-material" rid="pgen.1002832.s004">Table S2</xref>.</p><p>The <italic>RT</italic> and genome base compositions were compared using the G-test of independence. This test was also applied to compare the observed distributions of <italic>RT</italic>
<sup>+</sup> revertants among the single-burst reversion tests conducted with each of two different <italic>RT</italic>
<sup>&#8722;</sup> mutants, RT<sub>SUB</sub> and RT<sub>IN</sub>. The G-test for goodness of fit was used to compare the observed and expected Poisson distributions of <italic>RT</italic>
<sup>+</sup> revertants among <italic>RT</italic>
<sup>&#8722;</sup> single-bursts, and the replicated G-test for goodness of fit was applied to compare the G+C content of the local sequence environment (six to seven bases upstream) of the base substitutions observed in <italic>RT</italic> with the expected content according to the base composition of the whole gene. When applying this last test, each upstream position (from +1 to +6 or +7) was considered as an independent replicate. All tests were performed as per Sokal and Rohlf <xref ref-type="bibr" rid="pgen.1002832-Sokal1">[64]</xref>.</p>