<p>The 349 <italic>A. thaliana</italic> accessions for the GWA study were selected from 5810 worldwide accessions as described previously <xref ref-type="bibr" rid="pgen.1002923-Baxter1">[26]</xref>, <xref ref-type="bibr" rid="pgen.1002923-Platt1">[48]</xref>. 82 of the genome re-sequenced accessions used in this paper were obtained from the Arabidopsis Biological Resource Center. Most plants used for elemental analysis by ICP-MS were grown in a controlled environment <xref ref-type="bibr" rid="pgen.1002923-Baxter1">[26]</xref>, <xref ref-type="bibr" rid="pgen.1002923-Lahner1">[43]</xref>. Briefly, seeds were sown on moist soil (Promix; Premier Horticulture) with non essential elements (As, Cd, Co, Li, Ni and Se) added at subtoxic concentrations in a 20-row tray. After stratification at 4&#176;C for 3 days the tray was moved into a climate-controlled room for growth with a photoperiod of 10 h light (90 &#181;mol&#183;m<sup>&#8722;2</sup>&#183;s<sup>&#8722;1</sup>)/14 h dark, humidity of 60% and temperature ranging from 19 to 22&#176;C. Plants were bottom-watered twice a week with modified 0.25&#215; Hoagland solution in which Fe was replaced by 10 &#181;M Fe-HBED (<italic>N,N&#8242;</italic>-di(2-hydroxybenzyl)ethylenediamine- <italic>N,N&#8242;</italic>-diacetic acid monohydrochloride hydrate; Strem Chemicals, Inc.). Plants used for studying the relationship between expression of <italic>HMA3</italic> and leaf Cd concentration were grown in axenic conditions. Briefly, seeds were surface sterilized using 50% bleach and 0.05% SDS for 15 min, washed 8 times with sterilized deionized water and sown on &#189; strength Murashige and Skoog (Sigma-Aldrich, St. Louis, USA) media solidified with agar containing 1% sucrose in Petri dishes. Plates were placed at 4&#176;C for 3 days for seed stratification and then maintained at 16 h light (90&#8211;120 &#181;mol&#183;m<sup>&#8722;2</sup>&#183;s<sup>&#8722;1</sup>)/8 h dark and 22&#176;C. After 3-weeks growth, roots were harvested and used for RNA extraction and shoots were harvested for elemental analysis.</p><p>Seedlings were grafted as previously described <xref ref-type="bibr" rid="pgen.1002923-Rus1">[49]</xref>. Graft unions were examined before transfer to potting mix soil under the stereoscope to identify any adventitious root formation from graft unions or above. Healthy grafted plants were transferred to potting mix soil in a 20-row tray and grown in a controlled environment and after 4-weeks leaf samples were harvested as described above. After harvesting graft unions were examined again, and grafted plants with adventitious roots or without a clear graft union were removed from subsequent analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The determination of leaf elemental concentrations was performed as described previously <xref ref-type="bibr" rid="pgen.1002923-Lahner1">[43]</xref>. One to two leaves (&#8764;2&#8211;4 mg dry weight) were harvested from <italic>A. thaliana</italic> plants grown vegetatively for 5 weeks, leaves were rinsed with 18 M&#8486; water and placed into Pyrex digestion tubes. Samples were placed into an oven at 92&#176;C to dry for 20 hours. After cooling, 7 reference samples from each planted block were weighed. Subsequently, all samples were digested with 0.7 ml concentrated nitric acid (OmniTrace; VWR Scientific Products) and diluted to 6.0 ml with 18 M&#8486; water. Gallium (Ga) was added in the acid prior to digestion to serve as an internal standard for assessing errors in dilution, variations in sample introduction and plasma stability in the ICP-MS instrument. Analytical blanks and standard reference material (NIST SRM 1547) were digested together with plant samples in the same manner. After samples and controls were prepared, elemental analysis was performed with an ICP-MS (Elan DRCe; PerkinElmer) for Li, B, Na, Mg, P, K, Ca, Mn, Fe, Co, Ni, Cu, Zn, As, Se, Mo and Cd. All samples were normalized to calculated weights, as determined with a heuristic algorithm using the best-measured elements, the weights of the seven weighed samples and the solution concentrations, detailed at <ext-link ext-link-type="uri" ns0:href="http://www.ionomicshub.org">www.ionomicshub.org</ext-link>. For GWA analysis data was normalized using common genotypes across experimental blocks as previously described <xref ref-type="bibr" rid="pgen.1002923-Baxter1">[26]</xref>, and this normalized data has been deposited on the iHUB (previously known as PiiMS <xref ref-type="bibr" rid="pgen.1002923-Baxter3">[51]</xref>) for viewing and download through <ext-link ext-link-type="uri" ns0:href="http://www.ionomicshub.org">www.ionomicshub.org</ext-link>.</p><p>The selection and genotyping of accessions for GWA analysis was described previously <xref ref-type="bibr" rid="pgen.1002923-Baxter1">[26]</xref>, <xref ref-type="bibr" rid="pgen.1002923-Platt1">[48]</xref>. Briefly, 5810 <italic>A. thaliana</italic> accessions were collected worldwide and genotyped at 149 genome-wide SNPs <xref ref-type="bibr" rid="pgen.1002923-Baxter1">[26]</xref>, <xref ref-type="bibr" rid="pgen.1002923-Platt1">[48]</xref>. These accessions were classified into 360 groups based on their genotypes at the 149 SNPs. One accession from each group was chosen to make a core set with 360 accessions. Among the core set of 360 accessions, 349 were phenotyped by ICP-MS for ionomic traits. Of this phenotyped subset 337 accessions were genotyped for at least 213,497 SNPs using the custom-designed SNP-tilling array Atsnptile 1 <xref ref-type="bibr" rid="pgen.1002923-Baxter1">[26]</xref>, <xref ref-type="bibr" rid="pgen.1002923-Atwell1">[31]</xref>, <xref ref-type="bibr" rid="pgen.1002923-Platt1">[48]</xref>. The GWA analysis was done using a linear mixed model to correct confounding by population structure <xref ref-type="bibr" rid="pgen.1002923-Yu1">[44]</xref> implemented in the program EMMA (Efficient Mixed-Model Association), which was described previously <xref ref-type="bibr" rid="pgen.1002923-Atwell1">[31]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The SNP-Tilling array-based eXtreme Array Mapping (XAM) was done following the description of Becker et al. <xref ref-type="bibr" rid="pgen.1002923-Becker1">[47]</xref>. First, F2 progeny from an outcross of CS28181 and Col-0 were sorted by leaf Cd concentration. Approximately 25% of the total progeny at each end of the leaf Cd concentration distribution were pooled separately. From these pools approximately 300 ng genomic DNA was labeled separately using the BioPrime DNA labeling system (Invitrogen) and hybridized to the Affymetrix SNP-tilling array Atsnptile 1. The CEL files containing raw data of signal intensity for all probes were read and spatially corrected using R scripts from Borevitz et al. <xref ref-type="bibr" rid="pgen.1002923-Borevitz1">[52]</xref> with the R program and the Bioconductor Affymetrix package. The original CEL files used in this study have been submitted to the Gene Expression Omnibus (GEO) under accession GSE39679. Polymorphic SNPs between the two parents identified previously <xref ref-type="bibr" rid="pgen.1002923-Borevitz1">[52]</xref> were used for further analysis. There are 4 probes for each SNP, antisense and sense probes for two alleles. The allele frequency difference between the two pools for each SNP was then assessed based on the signal intensity difference of the 4 probes. The whole process can be carried out using R scripts that are available at <ext-link ext-link-type="uri" ns0:href="http://ars.usda.gov/mwa/bsasnp">http://ars.usda.gov/mwa/bsasnp</ext-link>
<xref ref-type="bibr" rid="pgen.1002923-Becker1">[47]</xref>.</p><p>PCR-based genotyping was used to further narrow down the mapping interval for the leaf Cd accumulation QTL. All 312 F2 plants that were phenotyped by ICP-MS were genotyped individually at 5 cleaved-amplified polymorphic sequence (CAPS) markers. The primers and restriction enzymes for the CAPS markers are listed in <xref ref-type="supplementary-material" rid="pgen.1002923.s004">Table S2</xref>. Recombinants between marker Fo13M and Fo16M were selected for further analysis. The F2 recombinants with a clear low leaf Cd phenotype similar to CS28181 were directly used for determination of the candidate region. The F2 recombinants without a clear phenotype, or with a low Cd phenotype were selfed and 24 F3 progeny of each F2 individual further phenotyped for leaf Cd contetnt. According to the leaf Cd concentration of the F3's the genotype in the mapping interval was inferred and the region further narrowed.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The candidate genomic region of CS28181 was sequenced through overlapping PCR. Firstly, 20 overlapping fragments were amplified using KOD hot start DNA polymerase (Novagen, EMD Chemicals, San Diego, CA USA) from the genomic region of CS28181 covering <italic>HMA2</italic> and <italic>HMA3</italic> and their promoters. The primers for the PCR reactions were designed using Overlapping Primersets (<ext-link ext-link-type="uri" ns0:href="http://pcrsuite.cse.ucsc.edu/Overlapping_Primers.html">http://pcrsuite.cse.ucsc.edu/Overlapping_Primers.html</ext-link>) and are listed in <xref ref-type="supplementary-material" rid="pgen.1002923.s004">Table S2</xref>. After purification, each fragment was sequenced using its amplification primers in two directions. The sequenced reads were assembled using SeqMan Lasergene software (DNASTAR; <ext-link ext-link-type="uri" ns0:href="http://www.dnastar.com">http://www.dnastar.com</ext-link>), with Col-0 sequence used as the reference. The <italic>HMA3</italic> haplotypes were analyzed using 149 genome re-sequenced <italic>A. thaliana</italic> accessions. Genomic sequence data of the 149 accessions was downloaded from the 1001 Genomes Data Center (<ext-link ext-link-type="uri" ns0:href="http://1001genomes.org/data/MPI/MPICao2010/releases/2011_08_23/full_set/TAIR10">http://1001genomes.org/data/MPI/MPICao2010/releases/2011_08_23/full_set/TAIR10</ext-link>, <ext-link ext-link-type="uri" ns0:href="http://signal.salk.edu/atg1001/index.php,">http://signal.salk.edu/atg1001/index.php,</ext-link>). The genomic sequences of the <italic>HMA3</italic> region were extracted using Text File Splitter 2.0.4 (<ext-link ext-link-type="uri" ns0:href="http://www.softpedia.com/get/System/File-Management/Text-File-Splitter.shtml">http://www.softpedia.com/get/System/File-Management/Text-File-Splitter.shtml</ext-link>). The sequence data was introduced into Microsoft Excel and polymorphic nucleotides identified. The coding sequence (CDS) of each <italic>HMA3</italic> allele was predicted according to the reference cDNA of Col-0. Variations in protein amino acid sequence were identified according to the polymorphic nucleotides in the DNA sequence.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For construction of the expression vector of <italic>A. thaliana HMA3</italic> and <italic>HMA2</italic> two genomic DNA fragments for the two genes were PCR amplified from CS28181 using KOD hot start DNA polymerase and primers as listed in <xref ref-type="supplementary-material" rid="pgen.1002923.s004">Table S2</xref>. The fragment for <italic>HMA3</italic> is &#8764;4.9 kb including 1.6 kb promoter region and 0.8 kb 3&#8242; downstream sequence. The fragment for <italic>AtHMA2</italic> is &#8764;6.7 kb including 2.0 kb promoter region and 0.4 kb 3&#8242; downstream sequence. The fragments were cloned into pCR-XL-TOPO vector (Invitrogen Life Technologies, <ext-link ext-link-type="uri" ns0:href="http://www.invitrogen.com">http://www.invitrogen.com</ext-link>) for sequencing and subsequently recombined into binary vector pCAMBIA1301 by restriction enzymes of <italic>Sal</italic> I and <italic>Bam</italic>H I. The expression vectors with the two genes were transformed into <italic>Agrobacterium tumeraciens</italic> strain GV3101 and were introduced into Col-0 using the floral dip method <xref ref-type="bibr" rid="pgen.1002923-Clough1">[53]</xref>. Transgenic lines were screened on &#189; strength Murashige and Skoog (Sigma-Aldrich, St. Louis, USA) medium solidified with agar containing 50 &#181;g/ml Hygromycin and 1% sucrose.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Total RNA was extracted from 3-week old plants grown on &#189; strength Murashige and Skoog (Sigma-Aldrich, St. Louis, USA) medium solidified with agar containing 1% sucrose using TRIzol Plus RNA Purification kit (Invitrogen Life Technologies, <ext-link ext-link-type="uri" ns0:href="http://www.invitrogen.com">http://www.invitrogen.com</ext-link>). Two microgram of total RNA was used to synthesize first strand cDNA with SuperScript VILO cDNA Synthesis Kit (Invitrogen Life Technologies, <ext-link ext-link-type="uri" ns0:href="http://www.invitrogen.com">http://www.invitrogen.com</ext-link>). Quantitative real-time PCR was performed using SYBR Green PCR Master Mix (Applied Biosystems, USA) with the fist strand cDNA as a template on a Real-Time PCR System (ABI StepOnePlus, Applied Biosystems lco., USA). Primers for qRT-PCR were designed using Primer Express Software Version 3.0 (Applied Biosystems, USA). One primer of a pair was designed to cover an exon-exon junction. The primer sequences are shown in <xref ref-type="supplementary-material" rid="pgen.1002923.s004">Table S2</xref>. Expression data analysis was performed as described previously <xref ref-type="bibr" rid="pgen.1002923-Livak1">[54]</xref>.</p>