<p>All samples were collected from pet dogs after the owners provided informed consent. Study materials were approved by the Animal Care and Use Committees at the collection institutions. All procedures and materials were approved by the Animal Care and Use Committee of the National Human Genome Research Institute. DNA from blood was extracted using standard protocols. Saliva DNA was collected and extracted using the Oragene-Animal collection kit (DNA Genotek, Ontario, Canada).</p><p>SCCD cases were all confirmed with biopsy reports from veterinary pathologists. Controls are &#8805;8 years old at the time of the analysis with pedigree information and unrelated at the grandparent level. For the original STPO controls (n&#8202;=&#8202;34), Briards (n&#8202;=&#8202;18) and Giant Schnauzers (n&#8202;=&#8202;13), dogs were considered controls if born before 2002 and unaffected. In the 5.7 Kb CNV analysis and black versus light colored STPO GWAS, the unrelated black STPO controls (n&#8202;=&#8202;45) and light colored controls (n&#8202;=&#8202;26) were born before 2005, while the young STPOs (n&#8202;=&#8202;34) were born in or after 2005.</p><p>The first GWAS compared 31 STPO cases and 34 unrelated black STPO controls using the Affymetrix v2 Canine SNP Chip (Affymetrix, Santa Clara, CA). The BRLMM-P algorithm was used to genotype the SNPs. SNPs were removed from the analysis if greater than 10% of the data were missing, there were more than 60% heterozygous calls, or the minor allele frequency was &lt;5%. The final dataset consisted of 36,897 SNPs.</p><p>The second GWAS compared 24 unrelated black STPO controls and 24 unrelated light colored STPO controls using the Illumina CanineHD BeadChip (Illumina, San Diego, CA). SNP genotypes were called using the Illumina Genome Studio software package. SNP clusters were evaluated if the call rate was &lt;90%, the heterozygous excess was &#8722;1 to &#8722;0.7 or 0.5 to 1, and if the GenTrain score was &lt;0.5. SNPs were removed from the analysis if the evaluated SNP clusters could not be improved or if the minor allele frequency was &lt;5%. The final set consisted of 126,697 SNPs. The genotypes and phenotypes for both GWASs will be submitted to Gene Expression Omnibus (GEO).</p><p>Both datasets were analyzed for population stratification using principle components analysis. The principle components (PCs) were calculated in Eigenstrat <xref ref-type="bibr" rid="pgen.1003409-Price1">[57]</xref> and Tracy-Widom statistics were utilized to determine if the PCs were statistically significant. For those deemed significant (TW p-value&lt;&#8202;=&#8202;0.05), ANOVA F-statistics were calculated within the assigned populations (either case/control for the first GWAS or black/light in the second GWAS) to determine if the PCs divided the population based on the phenotype of interest.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The SCCD case/control GWAS did not have evidence of population stratification by phenotype and was analyzed by calculating the allelic association (P<sub>raw</sub>) of each SNP with the disease using the statistical package PLINK v1.06 (<ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/purcell/plink/">http://pngu.mgh.harvard.edu/purcell/plink/</ext-link>) <xref ref-type="bibr" rid="pgen.1003409-Purcell1">[58]</xref>. Correction for multiple testing was performed using 100,000 MaxT permutations in PLINK (P<sub>genome</sub>). The results of the MaxT permutations matched those obtained from Bonferroni correction at the 0.05 level.</p><p>The GWAS comparing black/light coat colored STPOs did have evidence for population stratification by phenotype and the allelic association (P<sub>raw</sub>) of each SNP with the phenotype was performed using the program EMMAX to correct for population structure <xref ref-type="bibr" rid="pgen.1003409-Kang1">[59]</xref>. To correct for false positive associations due to multiple testing, phenotypes were randomly permuted and association was repeated 1000 times. P<sub>genome</sub> values were based on the number of permutations out of 1000 that produced an equal or lower result. The results of the EMMAX-based permutations also matched those obtained from Bonferroni correction at the 0.05 level.</p><p>Over 1,268 amplicons were sequenced within the broad original genomic region (CFA15:30,280,088&#8211;35,714,394). Primers were designed using Primer3 v0.4.0 <xref ref-type="bibr" rid="pgen.1003409-Rozen1">[60]</xref>. Primer sequences are available (<xref ref-type="supplementary-material" rid="pgen.1003409.s004">Table S1</xref>). Amplification used standard PCR methods and sequencing was done using BigDye Terminator v3.1 on an ABI 3730xl DNA Analyzer (Applied Biosystems, Life Technologies, Grand Island, NY). Sequences were analyzed using the Phred/Phrap/Consed software packages <xref ref-type="bibr" rid="pgen.1003409-Ewing1">[61]</xref>, <xref ref-type="bibr" rid="pgen.1003409-Ewing2">[62]</xref>, <xref ref-type="bibr" rid="pgen.1003409-Gordon1">[63]</xref> and SNPs were identified using Polyphred <xref ref-type="bibr" rid="pgen.1003409-Nickerson1">[64]</xref>. Typically, 38 STPO cases and 30 STPO controls were resequenced. However, for a small proportion of the <italic>KITLG</italic> introns, the <italic>KITLG</italic> upstream region and the 144.9 Kb overlapping region, a set of six STPO cases and four STPO controls were resequenced. The ten samples were selected to represent the haplotypes in the region. Of the cases, two were homozygous for the risk haplotype, two were heterozygous and two were homozygous for the non risk haplotype. For the controls, three were heterozygous for the risk haplotype and one was homozygous for the non risk haplotype. Genotypes for the variants identified are available (<xref ref-type="supplementary-material" rid="pgen.1003409.s004">Table S1</xref>).</p><p>Recombination mapping was performed with 862 variants identified from resequencing 38 STPO cases and 30 STPO controls in a 1.2 Mb region surrounding the initial peak (CFA15:31,900,000&#8211;33,100,000). Seven new cases were enrolled subsequent to the initial GWAS for a total of 38 STPO cases. We evaluated only the 35 cases which shared at least one copy of the risk-associated allele at the peak SNP, CFA15:32,383,555, for recombination mapping. In this analysis, a position was identified as the location of a recombination event if a case no longer shared at least one copy of the STPO case major allele (i.e. homozygous for the STPO case minor allele). Additionally, the case needed to continue to be homozygous for the STPO case minor allele for more than one Kb. We set this requirement in order to take the most conservative approach, and to only identify persistent changes in the risk-associated haplotype pattern. We note that within the entire 1.2 Mb region only four variants were excluded as recombination events since the change in genotype did not persist for more than one Kb. For two variants, the minor allele only occurred in a subset of dogs with the risk haplotype indicating that these variants are likely mutations that arose on the haplotype after the causal variant. The other two variants were only 611 bp apart and the STPO case minor allele is the STPO control major allele. In this case, the association with disease is equally as strong, either centromeric or telomeric of these variants. As such, these two variants most likely resulted from a series of unique crossover events in different generations on the risk haplotype where we cannot unambiguously determine that the causal variant is within the centromeric or telomeric section.</p><p>The additional association analysis compared all 38 STPO cases to 30 unrelated black STPO controls. Four controls were excluded since they did not have sufficient DNA. The allelic association was calculated for all 862 variants in a 1.2 Mb region surrounding the initial peak (CFA15:31,900,000&#8211;33,100,000). Since none of the variants with minor allele frequencies &lt;10% were significant, the data was plotted with the 658 variants having minor allele frequencies &gt;10%, for clarity (<xref ref-type="fig" rid="pgen-1003409-g003">Figure 3</xref>). As with the GWASs, the allelic association of each variant with the disease phenotype was calculated using PLINK v1.06.</p><p>For the haplotype mapping comparing either Giant Schnauzer (n&#8202;=&#8202;28) or Briard cases (n&#8202;=&#8202;11) to the STPO cases (n&#8202;=&#8202;38), we scanned the STPO three recombination interval (CFA15:32,088,047&#8211;32,901,086) for the largest region of haplotype sharing. In the previous resequencing effort, either the Giant Schnauzer or Briard cases were sequenced with the STPO cases and controls. Variants from this resequencing were included in the interbreed haplotype analysis along with additional variants, which were then genotyped in either breed to make sure that there was a variant, on average, every 1500 bp. First, we started at the centromeric edge of the interval, CFA15:32,088,047, and moved variant by variant through the interval. We identified variants where all Giant Schnauzer or Briard cases had at least one copy of the STPO case major allele (i.e. where no Giant Schnauzer or Briard cases were homozygous for the STPO case minor allele). We then calculated the size of these intervals and the haplotypes within each interval were determined. If the haplotype started within the three recombination region, we included the area until the end of the haplotype sharing with STPO cases. Finally, the regions of interest were intervals where the Giant Schnauzer or Briard cases share at least one copy of the same haplotype as the majority of STPO cases (&gt;50%). Four SNPs, CFA15:32,724,674, CFA15:32,749,603, CFA15:32,795,285 and CFA15:32,832,982, assumed to have arisen on the haplotype after the mutation or as the result of a series of unique crossover events in separate generations, disrupted the haplotype sharing between STPO, Giant Schnauzer and/or Briard cases, and were removed from future analyses.</p><p>The LD block patterns in the 144.9 Kb overlapping region were determined with 186 variants in the STPO cases (n&#8202;=&#8202;38) and controls (n&#8202;=&#8202;30) using the Confidence Interval analysis in Haploview v4.1 <xref ref-type="bibr" rid="pgen.1003409-Barrett1">[27]</xref>. Tagging variants (SNPs or indels) were selected to capture haplotypes predicted by Haploview. There were four SNPs to capture the six haplotypes in LD block A (CFA15:32,782,292 G/A; CFA15:32,782,334 A/G; CFA15:32,796,712 T/C; CFA15:32,796,907 C/A), with the risk-associated haplotype being G/A/T/C at these SNPs, respectively. One SNP captured the two haplotypes in LD block B (CFA15:32,854,334), three SNPs captured the four haplotypes in LD block C (CFA15:32,859,750, CFA15:32,862,724, CFA15:32,870,197), and eight SNPs and indels captured the nine haplotypes in LD block D (CFA15:32,871,555, CFA15:32,876,284, CFA15:32,880,662, CFA15:32,887,141, CFA15:32,888,465, CFA15:32,888,492, CFA15:32,898,733, CFA15:32,899,461).</p><p>The nonradioactive DIG Southern blot system (Roche Applied Science, Gilroy, CA) was used to analyze the 5.7 Kb CNV. Briefly, 1&#8211;2 &#181;g genomic DNA was digested with the PstI enzyme for 3 hours at 37&#176;C. High molecular weight DNA was necessary for the assay. Samples were run at 30 volts for 16 hours on a 0.6% agarose gel in TAE buffer. A control that was heterozygous for the three and four copy alleles was run on every Southern. The bands on the Southern blot were the following sizes after digestion: 8.2 Kb, 13.9 Kb, 19.6 Kb, 25.3 Kb and 31 Kb for the one, two, three, four and five copy alleles, respectively. A subset of samples were sufficiently degraded such that interpretable results were not obtained. All owners of STPO cases were recontacted to provide additional saliva samples. Of the 37 STPO cases without Southern results, 29 had already passed away. For the remaining eight, we were unable to collect a high quality saliva sample. Ultimately, out of the 84 STPO cases that were attempted, Southern blot data was available for 47 (55.6%). We experienced similar results for the STPOs of control age.</p><p>Blots were processed as specified by the DIG Southern blot system (Roche Applied Science) for hybridization targets &#8805;5 Kb. The probe was prepared using the PCR DIG Probe Synthesis kit (Roche Applied Science) with half DIG labeled dNTPs and half regular dNTPs in a two step PCR reaction (68&#176;C annealing and extension temperatures) with the following primers, <named-content content-type="gene">CTGATTCACATTTCCAAGGTGACAATGA</named-content> and <named-content content-type="gene">ACATGGCAGAGAAAGGCAACTAAGACCT</named-content>. The DIG labeled probe was quantitated on a 1% agarose gel by comparing the probe intensity with the intensity of the Low Mass Ladder (Invitrogen, Carlsbad CA). The DIG Easy Hyb buffer (Roche Applied Science) was used for both the pre-hybridization and hybridization solutions. For hybridization, the probe concentration was 10 ng/ml. The low/high stringency washes and the DIG wash and block buffer set washes were performed according to the DIG protocol using CPD-star (Roche Applied Science). Blots were exposed to X-ray film for 30 minutes to two hours depending on the intensity of the signals.</p>