<p>All studies were approved by Institutional Review Boards at their respective sites (details are given in <xref ref-type="supplementary-material" rid="pgen.1003087.s005">Text S1</xref>). The Population Architecture using Genomics and Epidemiology (PAGE) study includes the following epidemiologic collections: Atherosclerosis Risk in Communities (ARIC), Coronary Artery Risk in Young Adults (CARIDA), Cardiovascular Health Study (CHS), the Multiethnic Cohort (MEC), the National Health and Nutrition Examination Surveys (NHANES), Strong Heart Study (SHS), and Women's Health Initiative (WHI). For this PheWAS, data were available from ARIC, CHS, MEC, NHANES III, NHANES 1999&#8211;2002, and WHI (<xref ref-type="table" rid="pgen-1003087-t001">Table 1</xref>). The PAGE study design is described in Matise et al <xref ref-type="bibr" rid="pgen.1003087-Matise2">[26]</xref> and the PAGE PheWAS study design is described in Pendergrass et al <xref ref-type="bibr" rid="pgen.1003087-Pendergrass1">[1]</xref>.</p><p>All SNPs considered for genotyping in PAGE were candidate gene or GWAS-identified variants for phenotypes and traits available in the epidemiologic collections accessed by PAGE study sites. Cohorts and surveys were genotyped using either commercially available genotyping arrays (Affymetrix 6.0, Illumina 370CNV BeadChip), and/or custom mid- and low-throughput assays (TaqMan, Sequenom, Illumina GoldenGate or BeadXpress). Quality control was implemented at each PAGE study site independently. Study specific genotyping details are described in <xref ref-type="supplementary-material" rid="pgen.1003087.s005">Text S1</xref>.</p><p>In this PheWAS, data were available for SNPs previously associated with HDL-C, LDL-C, and triglycerides <xref ref-type="bibr" rid="pgen.1003087-Dumitrescu1">[27]</xref>, body mass index, obesity <xref ref-type="bibr" rid="pgen.1003087-Fesinmeyer1">[28]</xref>, type 2 diabetes, glucose, insulin <xref ref-type="bibr" rid="pgen.1003087-Haiman1">[29]</xref>, and measures of inflammation (C-reactive protein), among other diseases/traits. A total of 83 SNPs overlapped across at least PAGE study sites: ten were specifically selected for body mass index traits replication, three for C-reactive protein, six for coronary/cardiac traits, three for gout/kidney, 41 for lipids, and 20 for type 2 diabetes. <xref ref-type="supplementary-material" rid="pgen.1003087.s001">Table S1</xref> lists these SNPs, along with references reporting phenotypic associations from the NHGRI GWAS catalog <xref ref-type="bibr" rid="pgen.1003087-Hindorff1">[30]</xref> and the open access database of GWAS results of Johnson et al. 2009 <xref ref-type="bibr" rid="pgen.1003087-Johnson1">[31]</xref>. The NHGRI GWAS catalog was most recently accessed in October, 2011. If no references were available from either of those two sources, a PubMed search was performed to retrieve relevant citations.</p><p>All tests of association were performed independently by each PAGE study site using the following analysis protocol: Linear or logistic regressions were performed for continuous or categorical dependent variables, respectively, assuming an additive genetic model (0, 1, or 2 copies of the coded allele). For variables with multiple categories, binning was used to create new variables of the form &#8220;A versus not A&#8221; for each category, and logistic regression was used to model the new binary variable. Linear regressions were repeated following a y to log (y+1) transformation of the response variable with +1 added to all continuous measurements before transformation to prevent variables recorded as zero from being omitted from analysis. All analyses were stratified by race-ethnicity.</p><p>Test of association were calculated for the number of SNPs and phenotypes listed in <xref ref-type="table" rid="pgen-1003087-t001">Table 1</xref>. The software used to calculate the associations for each study was as follows: ARIC (StatSoftware), CHS (R <xref ref-type="bibr" rid="pgen.1003087-Team1">[32]</xref>), MEC (SAS), MEC (SAS v9.2), WHI (R), EAGLE (SAS v9.2 using the Analytic Data Research by Email (ANDRE) portal of the CDC Research Data Center in Hyattsville, MD).</p><p>All association results from the tests of association were reported in standardized templates designed by the PAGE coordinating center to facilitate data sharing. All results were then imported into a relational database (MySQL). The database was also used to match previously reported GWAS data with the SNPs analyzed in this study.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The software PheWAS-View was developed for data visualization of the PheWAS results as well as for plotting &#8220;Sun Plots&#8221; <xref ref-type="bibr" rid="pgen.1003087-Pendergrass2">[33]</xref>. Synthesis-View <xref ref-type="bibr" rid="pgen.1003087-Pendergrass3">[34]</xref>, <xref ref-type="bibr" rid="pgen.1003087-Pendergrass4">[35]</xref> was also used to present results within this manuscript. Both software packages are freely available software for academic users: <ext-link ext-link-type="uri" ns0:href="http://ritchielab.psu.edu/ritchielab/software">http://ritchielab.psu.edu/ritchielab/software</ext-link>, and can be used with a web interface at: <ext-link ext-link-type="uri" ns0:href="http://visualization.ritchielab.psu.edu/">http://visualization.ritchielab.psu.edu/</ext-link>.</p><p>A total of 105 phenotype-classes were developed to manually match related phenotypes across studies. To bin related phenotypes into classes the following steps were used as visualized in <xref ref-type="fig" rid="pgen-1003087-g006">Figure 6</xref>: First, using a MySQL database, the data from EAGLE, MEC, CHS, ARIC, and WHI were independently filtered for any tests of association results at p&lt;0.01, and then lists of the unique phenotypes for each individual PAGE study were generated. The number of phenotypes that passed this significance threshold for each of the four groups was 604 (ARIC), 331 (CHS), 63 (MEC), 324 (EAGLE), 1,342 (WHI). Resulting phenotypes were then manually matched up between ARIC, CHS, MEC, EAGLE and WHI using knowledge about the phenotypes and the known focus of specific PAGE study survey questions (such as bone fracture questions used primarily for collecting information about osteoporosis). For some phenotypes, the specific phenotype existed clearly across more than one PAGE study, such as for the phenotype &#8220;Hemoglobin&#8221;, where hemoglobin measurements were present for ARIC, CHS, EAGLE, and WHI. Other groups of phenotypes that fell within similar phenotypic domains but were not represented in the same form across studies were also collected into phenotype classes. One example is the phenotypes grouped together for the phenotype class of &#8220;Allergy&#8221;. EAGLE collected specific quantitative data from allergy skin testing and had survey questions about the presence of allergies in participants. ARIC and MEC did not have skin allergy testing, but did have survey questions about the presence of allergies. Thus these allergy phenotypes were grouped together. Finally, phenotypes from all studies, regardless of significance from genotype-phenotype tests of association, were matched to the already-defined phenotype classes using the criteria described above. A phenotype that matched a phenotype class but was not associated with a SNP at the significance threshold of p&lt;0.01 for a single study would still be included in the phenotype-class list. Using these criteria, a second curator reviewed the resultant phenotypes and phenotype classes for consistency and accuracy. To provide examples of the phenotype-classes, and which subphenotypes were matched with phenotype-classes, we show three phenotype-class examples in <xref ref-type="table" rid="pgen-1003087-t005">Table 5</xref>, and <xref ref-type="supplementary-material" rid="pgen.1003087.s002">Table S2</xref> contains the matched phenotypes across studies within the phenotype-classes for all phenotype-classes used within this study.</p><p>A MySQL database was used to filter the data from five studies for any results with p&lt;0.01 to generate lists of the unique phenotypes for each individual PAGE study. The number of phenotypes that passed this significance threshold for each of the four groups was 604 (ARIC), 331 (CHS), 63 (MEC), 324 (EAGLE), 1,342 (WHI). Note that during the binning process, a smaller number of phenotypes are listed in <xref ref-type="fig" rid="pgen-1003087-g006">Figure 6</xref> than the total number of phenotypes referred to in the manuscript for the actual associations, in the phenotype matching process we only took into account distinct phenotypes regardless of whether or not they were transformed or untransformed or if they were categorical phenotypes binned into case/control phenotypes. Next, resulting phenotypes were then manually matched up between ARIC, CHS, MEC, EAGLE and WHI using and knowledge about the phenotypes and the known focus of specific PAGE study questions (such as arterial measurements including degree of arterial stenosis). In the last step, phenotypes from all studies, regardless of significance from genotype-phenotype tests of association, were matched to the already-defined phenotype classes using the criteria described above.</p><p>Presented below are examples of phenotypes binned into three phenotype classes, &#8220;Asthma&#8221;, &#8220;BMI&#8221;, and &#8220;CRP&#8221;. <xref ref-type="supplementary-material" rid="pgen.1003087.s002">Table S2</xref> contains the complete list of matched phenotypes across studies within phenotype-classes, for all phenotype-classes used within this study.</p><p>It is important to note resources that can be used for further investigation of the phenotypes listed in <xref ref-type="supplementary-material" rid="pgen.1003087.s002">Table S2</xref>, as well as in the results presented in this paper. The following study websites contain additional information about all collected study information, including how those phenotypes were collected:</p><p xmlns:ns0="http://www.w3.org/1999/xlink">ARIC <ext-link ext-link-type="uri" ns0:href="http://www.cscc.unc.edu/aric/">http://www.cscc.unc.edu/aric/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">CHS <ext-link ext-link-type="uri" ns0:href="http://www.chs-nhlbi.org/CHSData.htm">http://www.chs-nhlbi.org/CHSData.htm</ext-link>, <ext-link ext-link-type="uri" ns0:href="https://biolincc.nhlbi.nih.gov/static/studies/chs/Other_Documents.htm">https://biolincc.nhlbi.nih.gov/static/studies/chs/Other_Documents.htm</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">WHI <ext-link ext-link-type="uri" ns0:href="https://cleo.whi.org/data/Pages/home.aspx">https://cleo.whi.org/data/Pages/home.aspx</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">EAGLE <ext-link ext-link-type="uri" ns0:href="http://www.cdc.gov/nchs/nhanes/nhanes_questionnaires.htm/">http://www.cdc.gov/nchs/nhanes/nhanes_questionnaires.htm/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">MEC <ext-link ext-link-type="uri" ns0:href="http://www.crch.org/multiethniccohort/mec_questionnaires.htm">http://www.crch.org/multiethniccohort/mec_questionnaires.htm</ext-link>
</p><p>After creating phenotype-classes, significant PheWAS tests of association for single genotype-phenotype associations across PAGE studies were identified using a database query. Our criteria for considering a PheWAS test of association significant included a threshold of p&lt;0.01 observed in two or more PAGE studies for the same SNP, phenotype class, and race/ethnicity and consistent direction of effect.</p><p>A total of 111 PheWAS tests of association met our criteria for significance (<xref ref-type="supplementary-material" rid="pgen.1003087.s003">Table S3</xref>). Significant results were then binned based on class of association: known, related, and novel. In this PheWAS, <italic>Known Associations</italic> are positive controls and represent previously reported genotype-phenotype associations. <italic>Related Associations</italic> are SNPs significantly associated in this PheWAS with phenotypes judged to be closely related to phenotypes among Known Associations found here and the literature. <italic>Novel Associations</italic> are significant PheWAS results where 1) the association does not match a known association <italic>and</italic> 2) the phenotype for the PheWAS association is not within a similar phenotypic domain as the phenotype of known association.</p><p>All participating studies were approved by their respective IRBs, and all study participants signed informed consent forms.</p>