<p>The anthropometry of men and women differ in various aspects: Average height, waist circumference, and WHR, is higher for men than for women, whereas average BMI is similar. Variability for all phenotypes is similar for men and women, which can be seen on the example of the KORA study (<xref ref-type="supplementary-material" rid="pgen.1003500.s017">Table S12</xref>) and specifically for WHR adjusted for BMI for all studies (<xref ref-type="supplementary-material" rid="pgen.1003500.s003">Figure S3</xref>).</p><p>The anthropometric traits examined are height (cm), weight (kg), body mass index (BMI, kg/m<sup>2</sup>) computed as weight divided by meter of height squared, waist circumference (WC, cm), hip circumference (HIP, cm), and waist-hip-ratio (WHR). The last three traits were analyzed without and with adjustment for BMI, yielding nine phenotypes in total (height, weight, BMI, WC, HIP, WHR, WCadjBMI, HIPadjBMI, WHRadjBMI). For the further analyses, the traits were all transformed at the study-level by calculating age-adjusted residuals (including age and age<sup>2</sup> into the regression model for trait creation) for men and women separately and adding BMI into the adjustment as indicated above; then &#8211; for all traits except height &#8211; the values were ranked and an inverse normal transformation was applied, whereas a z-score transformation was performed for height.</p><p>For discovery stage, we included 46 studies (up to 60,586 men, 73,137 women) on height, weight and BMI, 34 studies (up to 36,231 men, 45,192 women) on WC, 33 studies (up to 34,942 men, 43,316 women) on HIP and 32 studies (up to 34,629 men, 42,969 women) on WHR. Each study was genotyped using either Affymetrix or Illumina arrays. To enable meta-analyses across different SNP panels, each group performed genotype imputation using HapMap II CEU (build 21 or 22) via MACH <xref ref-type="bibr" rid="pgen.1003500-Li1">[65]</xref>, IMPUTE <xref ref-type="bibr" rid="pgen.1003500-Marchini1">[66]</xref> or BimBam <xref ref-type="bibr" rid="pgen.1003500-Guan1">[67]</xref>. Details are given in <xref ref-type="supplementary-material" rid="pgen.1003500.s006">Tables S1</xref>, <xref ref-type="supplementary-material" rid="pgen.1003500.s008">S3</xref>, <xref ref-type="supplementary-material" rid="pgen.1003500.s009">S4</xref>, <xref ref-type="supplementary-material" rid="pgen.1003500.s010">S5</xref> and <xref ref-type="supplementary-material" rid="pgen.1003500.s020">Text S2</xref>.</p><p>For follow-up, we included (i) 30 studies (up to 42,055 men, 32,785 women) for height, weight and BMI and 27 studies (up to 36,671 men, 28,326 women) for WC, WCadjBMI, HIP, HIPadjBMI, WHR and WHRadjBMI that were genotyped using the custom iSELECT Metabochip array containing &#8764;195K SNPs designed to support large-scale follow-up of putative associations with metabolic and cardiovascular traits, and (ii) 18 studies (20,340 men, 41,872 women) for height, weight, and BMI and 14 studies (11,225 men, 32,610 women) for WC, WCadjBMI, HIP, HIPadjBMI, WHR and WHRadjBMI genotyped using genome-wide SNP chips with subsequent imputation for <italic>in silico</italic> follow up.</p><p>In each study, association was tested separately for men and women. The additive genetic effect for each SNP on each phenotype was estimated using a normal linear regression model using MACH2QTL <xref ref-type="bibr" rid="pgen.1003500-Li2">[68]</xref>, SNPTEST <xref ref-type="bibr" rid="pgen.1003500-Marchini1">[66]</xref>, ProbABEL <xref ref-type="bibr" rid="pgen.1003500-Aulchenko1">[69]</xref>, GenABEL <xref ref-type="bibr" rid="pgen.1003500-Aulchenko2">[70]</xref>, Merlin <xref ref-type="bibr" rid="pgen.1003500-Abecasis1">[71]</xref>, or PLINK <xref ref-type="bibr" rid="pgen.1003500-Purcell1">[72]</xref>. For studies with a case-control design, cases and controls were analyzed separately. Study-specific information was described previously <xref ref-type="bibr" rid="pgen.1003500-LangoAllen1">[25]</xref>, <xref ref-type="bibr" rid="pgen.1003500-Speliotes1">[29]</xref>, <xref ref-type="bibr" rid="pgen.1003500-Heid1">[31]</xref> for discovery studies and in <xref ref-type="supplementary-material" rid="pgen.1003500.s006">Tables S1</xref>, <xref ref-type="supplementary-material" rid="pgen.1003500.s008">S3</xref>, <xref ref-type="supplementary-material" rid="pgen.1003500.s009">S4</xref>, <xref ref-type="supplementary-material" rid="pgen.1003500.s010">S5</xref> for follow-up studies.</p><p>All involved studies were conducted according to the principles expressed in the Declaration of Helsinki. The studies were approved by the local Review Boards and all study participants provided written informed consent for the collection of samples and subsequent analysis.</p><p>All discovery study-specific files were processed in the meta-analysis centers through a standardized cleaning script that included checks of allele frequencies, compliance with Hapmap alleles, file completeness, number of markers, and ranges of test-statistics. We excluded monomorphic SNPs, SNPs with MAF*N&#8804;3 (minor allele frequency multiplied by sample size) and SNPs with poor imputation quality, i.e. r2_hat &lt;0.3 in MACH; observed/expected dosage variance &lt;0.3 in BIMBAM; proper_info &lt;0.4 in IMPUTE; information &lt;0.8 in PLINK <xref ref-type="bibr" rid="pgen.1003500-Li1">[65]</xref>, <xref ref-type="bibr" rid="pgen.1003500-Marchini1">[66]</xref>, <xref ref-type="bibr" rid="pgen.1003500-Purcell1">[72]</xref>, <xref ref-type="bibr" rid="pgen.1003500-Servin1">[73]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Sex-specific standard errors and P-values from each participating study were genomic-control (GC) corrected <xref ref-type="bibr" rid="pgen.1003500-Devlin1">[74]</xref> using the lambda factors as published <xref ref-type="bibr" rid="pgen.1003500-LangoAllen1">[25]</xref>, <xref ref-type="bibr" rid="pgen.1003500-Speliotes1">[29]</xref>, <xref ref-type="bibr" rid="pgen.1003500-Heid1">[31]</xref>, then beta-estimates were meta-analyzed using the inverse-variance weighted fixed effect model as implemented in METAL <xref ref-type="bibr" rid="pgen.1003500-Willer2">[75]</xref>. A sensitivity analysis using the sample-size weighted Z-score meta-analysis approach yielded the same results; only fixed effect model results are shown. The 2,971,914 SNPs in each of 18 analyses (nine phenotypes in two sexes) reduced to 2,846,694 SNPs with available chromosome and position annotation in dbSNP. The genetic position (cM) was extracted from HapMap release 22 (<ext-link ext-link-type="uri" ns0:href="http://hapmap.ncbi.nlm.nih.gov/downloads/recombination/2008-03_rel22_B36/rates/">http://hapmap.ncbi.nlm.nih.gov/downloads/recombination/2008-03_rel22_B36/rates/</ext-link>) or - if unavailable - approximated by the inverse-distance weighted average of the genetic positions of the nearest HapMap SNPs (release 22) on each side.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We conducted two types of genome-wide searches in the discovery stage: (a) In the <italic>sex-specific scan</italic>, we computed sex-specific association P-values for each SNP, concatenated these for all nine phenotypes totaling 50,586,560 P-values (i.e. 2 sexes&#215;9 phenotypes&#215;2.85 Million SNPs), and selected 20,215 SNPs at 5% FDR <xref ref-type="bibr" rid="pgen.1003500-Benjamini1">[32]</xref>. Pruning this list to independent SNPs (starting with the 20,215 SNPs sorted by increasing P-value and deleting SNPs within 0.2 cM of any of the SNPs above) yielded 619 independent SNPs. For each SNP and for the phenotype that the SNP was selected for, we also computed P-values (<italic>P-diff</italic>) testing for difference between the meta-analyzed men-specific and women-specific beta-estimates <inline-formula><inline-graphic ns0:href="pgen.1003500.e001.jpg" /></inline-formula>, <inline-formula><inline-graphic ns0:href="pgen.1003500.e002.jpg" /></inline-formula> with corresponding standard errors <inline-formula><inline-graphic ns0:href="pgen.1003500.e003.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1003500.e004.jpg" /></inline-formula> using the t statistic<disp-formula id="pgen.1003500.e005"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003500.e005.jpg" /></disp-formula>The correlation <inline-formula><inline-graphic ns0:href="pgen.1003500.e006.jpg" /></inline-formula> between <inline-formula><inline-graphic ns0:href="pgen.1003500.e007.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1003500.e008.jpg" /></inline-formula>, computed as the Spearman rank correlation coefficient across all SNPs for each phenotype, ranged from 0.04 to 0.18 across phenotypes. From these 619 SNPs, we selected the 348 SNPs with nominally significant sex-difference (<italic>P-diff</italic>&lt;0.05) to ensure some level of sex-difference in the discovery. Whether the sex-difference was significant was then evaluated in the follow-up stage (see below). (b) In the <italic>sex-difference scan</italic>, <italic>w</italic>e computed <italic>P-diff</italic> for each of the &#8764;2.85 Million SNPs and each of the nine phenotypes and concatenated the totaling 25,293,280 P-values. We had planned to select SNPs for follow-up at an FDR of 5% across all SNPs and phenotypes, but there were none. Power considerations are provided in the <xref ref-type="supplementary-material" rid="pgen.1003500.s019">Text S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1003500.s004">Figure S4A</xref>.</p><p>Study-specific follow-up data were quality-controlled in a similar manner as discovery data with increased attention towards strand-issues. We conducted sex-specific follow-up meta-analyses using the same statistical models as for the discovery. We combined (i) <italic>in silico</italic> studies and (ii) metabochip studies, and then combined results of (i) and (ii) implying a double genomic control correction (<xref ref-type="supplementary-material" rid="pgen.1003500.s019">Text S1</xref>). Additionally, we conducted a joint meta-analysis combining the sex-specific association results of discovery and follow-up.</p><p>For SNPs selected for their small P-values of sex-specific association, the sex-difference estimates and corresponding P-values in the same data set would be inflated (see <xref ref-type="supplementary-material" rid="pgen.1003500.s005">Figure S5</xref>) as the two tests are not independent. We therefore derived sex-difference estimates and corresponding P-values in the follow-up data alone.</p><p>As none of our selected SNPs stemmed from the scan targeted for OED signals (i.e. the sex-difference scan) or showed any evidence of OED in the discovery, we targeted our follow-up analysis for CED or SSE signals. We filtered for a main effect (<italic>P</italic> for both sexes combined &lt;0.01) prior to testing for sex-difference (<italic>P-diff</italic>, as described above), since this increased the power to detect SSE and CED signals (<xref ref-type="supplementary-material" rid="pgen.1003500.s004">Figure S4B</xref>, <xref ref-type="supplementary-material" rid="pgen.1003500.s019">Text S1</xref>), while this filter did not introduce a bias such as a sex-stratified association filter would. SNPs with a <italic>P-diff</italic> in the follow-up at 5% FDR were considered as SNPs with <italic>significant sex difference</italic>.</p><p>We considered a joint (discovery and follow-up combined) association <italic>P-men</italic> or <italic>P-women</italic>&lt;5&#215;10<sup>&#8722;8</sup> as <italic>genome-wide significant</italic>. For each phenotype, we tested whether there were more male-specific or more female-specific associations among the associations with established genome-wide significance in at least one sex compared to the expected binomial distribution.</p><p>For the identified signals with sex-difference, study partners of the discovery and the <italic>in silico</italic> follow-up re-analyzed their data stratified by sex and age group (&#8805;50 years, &lt;50 years) using the same models as described above. Age difference was tested within each sex using the same t statistic as applied for the sex-difference testing.</p><p>Sex-specific associations of the identified signals with metabolic traits were derived requesting a sex-stratified re-analysis from the Global Lipids Genetics Consortium (GLGC, Triglycerides, HDL-, LDL-, and Total cholesterol) <xref ref-type="bibr" rid="pgen.1003500-Teslovich1">[34]</xref>, the Meta-Analyses of Glucose and Insulin-related traits Consortium (MAGIC, fasting insulin, fasting glucose, HOMA-B, HOMA-IR) <xref ref-type="bibr" rid="pgen.1003500-Dupuis1">[35]</xref>, <xref ref-type="bibr" rid="pgen.1003500-Prokopenko1">[76]</xref>; and, the DIAGRAM consortium (type 2 diabetes) <xref ref-type="bibr" rid="pgen.1003500-Zeggini1">[36]</xref>. We tested the overall number of SNPs with consistent nominally significant association for the sex that the SNP was selected for (<italic>P_women</italic> or <italic>P_men</italic> &lt;0.05) compared to a binomial draw with an event rate of 0.05. It needs to be noted that this test does not account for the correlation between the traits nor for the fact that the consortia involve an overlap of studies.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In order to explore whether certain pathways are enriched among the genes depicted by loci with evidence for sex-difference, we applied MAGENTA <xref ref-type="bibr" rid="pgen.1003500-Segre1">[77]</xref>. Briefly, MAGENTA calculates gene-specific scores (for &#8764;18,000 genes) by combining the p-values (here: the sex-difference P-values from our discovery stage for a specific anthropometric phenotype) of SNPs in and around the genes (40 kb down-, 100 kb upstream). The genetic score is corrected for potential confounders, such as gene size, number of independent SNPs, LD pattern, length in genetic distance, and number of recombination hotspots. These scores are ranked and the genes within the top 5% of these scores are tested for enrichment in certain pathways (separately for each phenotype) as given by different databases (GO: <ext-link ext-link-type="uri" ns0:href="http://www.geneontology.org/">http://www.geneontology.org/</ext-link>, KEGG: <ext-link ext-link-type="uri" ns0:href="http://www.genome.jp/kegg/">http://www.genome.jp/kegg/</ext-link>, Ingenuity: <ext-link ext-link-type="uri" ns0:href="http://www.ingenuity.com/products/pathways_analysis.html">http://www.ingenuity.com/products/pathways_analysis.html</ext-link>, and PANTHER: <ext-link ext-link-type="uri" ns0:href="http://www.pantherdb.org/">http://www.pantherdb.org/</ext-link>). MAGENTA determines whether the genes among the 5% top scores link to certain pathways more often than expected by chance. FDR is controlled at 5% via 10,000 permutations (using a random set of genes with the same number of genes as those observed).</p><p>For the seven confirmed sex-difference loci (defined as the regions depicted by SNPs within 1.0 cM of the respective lead SNP showing a certain level of association, P&#8804;100 * P<sub>leadSNP</sub>), we searched several catalogues and data bases to depict potential biologically relevant links or functional entities. We extended the regions of interest to +&#8722;500 kB around the lead SNP if the regions were very small and no gene was inflicted (as for the PPARG, VEGFA, MAP3K1 loci).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">First, we performed an automated search for reported genes or variants in our regions in PubMed (<ext-link ext-link-type="uri" ns0:href="http://www.pubmed.com">http://www.pubmed.com</ext-link>) and OMIM (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/omim">http://www.ncbi.nlm.nih.gov/omim</ext-link>) using Snipper (<ext-link ext-link-type="uri" ns0:href="http://csg.sph.umich.edu/boehnke/snipper">http://csg.sph.umich.edu/boehnke/snipper</ext-link>) or manually inspected UCSC (PMID: 22086951) and Ensembl (PMID: 21045057) genome browsers as well as the NHGRI GWAS catalog <xref ref-type="bibr" rid="pgen.1003500-Hindorff1">[78]</xref>, <xref ref-type="bibr" rid="pgen.1003500-Hindorff2">[79]</xref>. Second, we explored whether SNPs known to provide reliable tags for Copy-Number-Variations (CNVs) in European-descent samples (combining four catalogues including 60167 CNV-tagging SNPs as described previously <xref ref-type="bibr" rid="pgen.1003500-Heid1">[31]</xref>) correlated with our lead SNPs. Third, we performed several online database searches to establish whether known variants within 500 kb of each lead SNP, that are correlated (r<sup>2</sup>&gt;0.8 or 0.5) with our lead SNPs (using SNAP Proxy search <xref ref-type="bibr" rid="pgen.1003500-Johnson1">[80]</xref>), might have putative or predicted function. (i) We searched the SIFT database <xref ref-type="bibr" rid="pgen.1003500-Kumar1">[81]</xref> to determine whether any of these SNPs was predicted to affect protein function. (ii) We used SNPinfo <xref ref-type="bibr" rid="pgen.1003500-Xu1">[82]</xref> to investigate predicted and putative function in several functional classes, including splicing regulation, stop codons, polyphen predictions, SNPs3D predictions, transcription factor binding site (TFBS) prediction, and miRNA binding site prediction.</p><p>We examined transcript expression of genes near each of the seven identified SNP. For human eQTL, we explored four different tissues (subcutaneous adipose tissue, whole blood, and lymphoblastoid cells; details on methodology and tissue samples in the <xref ref-type="supplementary-material" rid="pgen.1003500.s019">Text S1</xref>). We computed sex-specific association including conditional analyses and r<sup>2</sup> measures to identify <italic>cis</italic> eQTL signals that were likely to be coincident with the anthropometric trait signal. For mouse eQTL, we had four types of tissues (inguinal fat, gonadal fat, liver, brown fat) with expression derived by real-time RT-PCR as well as three types of tissue (liver, inguinal fat, gonadal fat) with Illumina assays (details in <xref ref-type="supplementary-material" rid="pgen.1003500.s019">Text S1</xref>). The sex-specific association and a test for sex-differences were computed.</p>