<p>
<italic>C. elegans</italic> strains were obtained from the Caenorhabditis Genetics Center, and are derived from the wild-type isolate N2 (Bristol). The <italic>spe-44(ok1400)</italic> deletion allele was isolated by the Gene Knockout Consortium <xref ref-type="bibr" rid="pgen.1002678-Moerman1">[43]</xref> and backcrossed six times to N2 prior to analysis. Additional mutations include: <italic>dpy-20(e1282)IV, fem-1(hc17)IV, fem-3(q20gf)IV, fog-1(q253)I, let-92(s677)IV, unc-22(s7)IV</italic>. A linked <italic>spe-44(ok1400) dpy-20(e1282)</italic> mutant strain, balanced by <italic>let-92(s677) unc-22(s7)</italic>, was generated to facilitate discrimination of homozygous lines. Homozygous <italic>spe-44</italic> males were obtained by mating balanced heterozygous males to homozygous Dpy Spe hermaphrodites and picking Dpy male progeny. Sodium hypochlorite treatment of gravid adults was used to obtain embryos for age-synchronized analyses. Strains were propagated on OP50-seeded NGM plates and maintained at 15&#176;C unless otherwise indicated. Genetic manipulations were carried out by standard methods <xref ref-type="bibr" rid="pgen.1002678-Brenner1">[74]</xref>. Self-fertility assays were performed by total progeny counts of individual hermaphrodites; cross-fertility was assessed by mating individual hermaphrodites with four males for 24 hours, then counting total progeny.</p><p>Single-worm PCR detection of the <italic>spe-44(ok1400)</italic> deletion utilized primers HES-343, HES-364, and HES-502 (all primers listed in <xref ref-type="supplementary-material" rid="pgen.1002678.s009">Table S5</xref>). Microinjection rescue was performed with plasmid pHS584, a 6.5 kbp fragment of the <italic>spe-44</italic> genomic interval; included in the microinjection mix were plasmid pRF4, which contains the dominant <italic>rol-6(su1006)</italic> as a morphological marker, and genomic DNA, which enhances germline expression by formation of complex arrays <xref ref-type="bibr" rid="pgen.1002678-Mello1">[75]</xref>, <xref ref-type="bibr" rid="pgen.1002678-Kelly2">[76]</xref>. Rescue was determined by injecting balanced <italic>spe-44(ok1400) dpy-20(e1282)/let-92(s677) unc-22(s7)</italic> hermaphrodites, picking individual dumpy rollers from the F2 generation grown at 25&#176;C, and counting total progeny. For qPCR, RNA was isolated via Trizol (Invitrogen) from triplicate samples (each &#8764;500 animals) grown at 25&#176;C at the indicated developmental stage, reverse-transcribed with Superscript II (Invitrogen), and amplified with primers HES-502 and HES-539 (for <italic>spe-44</italic>) or AKA-70 and AKA-71 (for <italic>act-1</italic>). For yeast expression assays, the <italic>spe-44</italic> cDNA was amplified with primers HES-531 and HES-532 and inserted into a derivative of pJG4-5 <xref ref-type="bibr" rid="pgen.1002678-Gyuris1">[77]</xref> that lacks the transcriptional activation domain. Putative promoter fragments were amplified from genomic DNA with primers HES-612 and HES-613 (<italic>spe-7</italic>) or HES-583 and HES-584 (<italic>spe-4</italic>) and inserted into a derivative of pLacZi (Clontech); the same <italic>spe-4</italic> upstream sequence is sufficient to confer <italic>in vivo</italic> transgene rescue of <italic>spe-4</italic> sterility <xref ref-type="bibr" rid="pgen.1002678-LHernault3">[78]</xref>. Yeast media and manipulations followed standard protocols <xref ref-type="bibr" rid="pgen.1002678-Rose1">[79]</xref>.</p><p>Intact gonads were obtained by dissection of individual worms. Sperm spreads were obtained by further applying slight pressure to the coverslip. Antibody staining of dissected gonads followed established protocols <xref ref-type="bibr" rid="pgen.1002678-Shakes1">[e.g., see 46]</xref>. Specimens were incubated with primary antibodies for 2&#8211;3 hours at room temperature unless otherwise indicated. Primary antibodies included: FITC-labeled anti-alpha-tubulin (DMIA, Sigma) (used at 1&#8758;100 dilution), anti-MSP (4D5, gift from D. Greenstein) (1&#8758;200), anti-phospho-histone H3 (serine10) (Upstate Biotechnology) (1&#8758;150), anti-MPM2 (DAKO Corp.) (1&#8758;100 overnight at 4&#176;C), anti-histone H3 dimethylated Lys 9 (Upstate Biotechnology) (1&#8758;50), 1CB4 <xref ref-type="bibr" rid="pgen.1002678-Okamoto1">[49]</xref>. The SPE-44 antibody (1&#8758;100) was generated in rabbits against a C-terminal peptide epitope and affinity-purified (Open Biosystems). Affinity-purified secondary antibodies (Jackson Immunoresearch Laboratories) (1&#8758;100) included goat anti-rabbit TRITC-labeled IgG and FITC- or DyLight-labeled goat anti-mouse IgG. Images were acquired under differential inference contrast or epifluorescence using an Olympus BX60 microscope equipped with a Cooke Sensicam cooled CCD camera and IPLab software. In some cases, images were minimally processed to enhance contrast either with IPLab software or Photoshop.</p><p>
<italic>In situ</italic> hybridization for <italic>spe-44</italic> germ line expression was performed on dissected gonads following fixation <xref ref-type="bibr" rid="pgen.1002678-Lee2">[80]</xref>. Digoxigenin-labeled, single-stranded sense and antisense probes were generated from a 1.3 kb <italic>spe-44</italic> cDNA fragment by linear amplification according to the manufacturer's protocol (Roche). Following hybridization, probe detection was by colorimetric assay with alkaline phosphatase-conjugated anti-digoxigenin antibodies and NBT/BCIP substrate.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Worms for microarray screening were obtained by hand-picking samples of 50 L4 males of the indicated genotype. RNA was isolated by Trizol treatment and ethanol precipitation using linear polyacrylamide (GeneElute LPA; Sigma-Aldrich) as carrier. RNA was amplified and labeled according to the manufacturer's protocol (Nugen). Microarray screening was performed in triplicate using GeneChip <italic>C. elegans</italic> genome arrays (Affymetrix). Microarray data were analyzed by Microarray Suite 5.0 (Affymetrix) and Genomics Suite (Partek) software, using a p-value threshold of 0.05 for differential expression. Sequences for comparative analysis of <italic>Caenorhabditis</italic> species were obtained from WormBase release WS220 [<ext-link ext-link-type="uri" ns0:href="http://www.wormbase.org">www.wormbase.org</ext-link>]. Multiple sequence alignments of <italic>spe-44</italic> homologs and promoters were performed with CLUSTALW2 <xref ref-type="bibr" rid="pgen.1002678-Larkin1">[81]</xref>.</p>