<p>A total of 787 fire ant queens were collected immediately after a nuptial mating flight on May 5<sup>th</sup>, 2010 in a large parking lot (Target, 3970 SW Archer Rd, Gainesville, FL) and weighed. Since the area of collection has a high prevalence of monogyne colonies, we expected these queens to belong to the monogyne social form; this was subsequently confirmed by screening 108 queens for social form using <italic>Gp-9</italic> as a marker following the protocol as described in <xref ref-type="bibr" rid="pgen.1003633-Valles1">[90]</xref>. The remaining 679 queens were split into two groups: 308 queens were set up in pairs (pleometrosis) based on having similar weights (range &#177;0.2 mg) and paint-marked with different colors, while 371 queens were set up individually (haplometrosis) and paint-marked as well. All the queens were provided with a nesting chamber consisting of a glass tube half-filled with water, which was covered by a cotton ball and a layer of dental plaster: this keeps the chamber moist but avoids an excess of water which is deleterious for the young brood. Tubes were sealed with a loose cap to provide air flow. Specimens were reared in the dark at 28&#176;C, 70% relative humidity under claustral conditions (no food and no water) for 1 month.</p><p>After eclosion of the first batch of workers (minims), incipient colonies were provided with water, sugar water and frozen crickets. Glass tubes were set open in pencil boxes coated with Fluon to prevent escape. Queens were subsequently monitored daily until it was possible to identify the social rank of the two cofoundresses in pleometrotic couples. Previous studies have found that the initial cooperation between the two cofoundresses turns into conflict after the emergence of minims, resulting in the execution of one queen <xref ref-type="bibr" rid="pgen.1003633-Balas1">[19]</xref>. Queens that survive the competition (winners) are usually located at the top of the brood pile within the nest chamber and they are generally tended by workers; conversely, queens that will be executed (losers) are normally seen outside the nest chamber, hiding from workers in order to avoid being attacked (<xref ref-type="supplementary-material" rid="pgen.1003633.s004">Figure S4</xref>). We used these observations to establish the social rank of the two pleometrotic queens, i.e. winner and loser. We collected 25 pleometrotic couples and 25 haplometrotic queens in dry ice and stored them at &#8722;80&#176;C to be later processed.</p><p>This assay was performed with 34 couples of pleometrotic queens from the same pool of newly mated queens as experiment 1. The queens were paired and placed in nesting chambers as before. After emergence of minims, queens' behavior was monitored as before. Once the behavioral observation revealed the social rank of the two cofoundresses, queens were weighed again and re-paired with a different partner. We created the following three groups of queens: a) winner+winner (similar weight), b) loser+loser (similar weight), and c) winner+loser (different weights). Again, we monitored the behavior until the social rank of the newly coupled specimens was evident and we collected 4 new behavioral phenotypes in the same way as above: a) winners switched into losers (win/los, N&#8202;=&#8202;7), b) losers switched into winners (los/win, N&#8202;=&#8202;11), c) continuing winners (win/win, N&#8202;=&#8202;12) and d) continuing losers (los/los, N&#8202;=&#8202;5).</p><p>Individual fire ant queens were thawed and dissected under cold RNAlater (Qiagen, Valencia, CA) to confirm the mating status: unmated queens were not included in the analysis. Total RNA was extracted using the RNeasy Plus kit (Qiagen) combined with a RNase-Free DNase step (Qiagen) to remove any possible contamination by genomic DNA. Subsequent steps in the microarray analysis were performed at the Penn State Genomic Core Facility. RNA concentration and purity were assessed using NanoDrop and Qubit and RNA quality was assessed using RNA Nano Chips on the Agilent Bioanalyzer. 1 &#181;g of each sample was amplified using the Ambion (Life Technologies) Amino Allyl MessageAmp II aRNA Amplification Kit (AM1753). 15 &#181;g of aRNA were dyed with Cy3 or Cy5 (GE Health Care #RPN5661) and subsequently purified according to the Ambion Kit instructions. 1.5 &#181;g of a Cy3 labeled sample were combined with 1.5 &#181;g of a Cy5 labeled sample and fragmented using RNA Fragmentation Reagents (Ambion AM8740) according to the manufacturer's instructions. Samples were hybridized with mixing in a MAUI hybridization instrument overnight at 42&#176;C. Arrays were scanned using Axon GenePix 4000B.</p><p>For the first microarray developed to validate the efficiency of probe sequences, we pooled RNA samples (2 &#181;g total) from different castes, developmental instars and social forms as follows: 3 female alates, 15 workers, 5 larvae and 5 pupae from both monogyne and polygyne social forms and 5 males from monogyne colonies only.</p><p>The fire ant genome includes an official gene set of 16,569 protein-coding genes that were generated by a combination of <italic>ab initio</italic>, EST-based, and sequence similarity-based methods <xref ref-type="bibr" rid="pgen.1003633-Wurm1">[27]</xref>. For our microarray studies, we combined the official gene set with a set of ESTs obtained from assemblies of the fire ant transcriptome for a total set of 63,436 sequences (&#8220;transcripts&#8221;). We successfully designed 60-mer probes for 51,531 of these transcripts (Roche NimbleGen, Inc., Madison WI). These sequences/probes were grouped into four categories: ESTs with gene models (EWGM, 7433 transcripts), ESTs without gene models (EWOGM, 40,613), gene models (GM, 3246) and gene models redundant with other models (GMRWOM, 239).</p><p>We developed and used a first microarray (1-plex 385,000 probe capacity, Roche NimbleGen, Inc., Madison WI) to validate the probe design and test multiple probes per transcript. On average, we designed 7 probes per transcript for a total of 355,930 probes. Each probe was tested for both the red (Cy5) and the green (Cy3) dyes. For transcripts with only one probe (N&#8202;=&#8202;296), we verified that the probe had acceptable intensities for both dyes. For the other transcripts we examined the performance of the probes with the green dye only, because these showed consistently higher intensity compared to the red dye. Probes were ranked in the follow manner: a) if there were only 2 probes per transcript (N&#8202;=&#8202;230), we selected the one with higher intensity; b) if there were 3 to 6 probes (N&#8202;=&#8202;744), we calculated the ratio &#8220;probe intensity/median intensity of all probes for that transcript&#8221; and selected the probe with highest ratio if the value was &lt;3, otherwise we selected the probe with the second highest ratio; c) for transcripts with 7 probes (N&#8202;=&#8202;50,261), we followed the procedure as in &#8220;b&#8221; but, in case the probe with the highest ratio was &gt;3, we removed that probe, calculated new ratios and selected a new probe with highest ratio. This procedure allowed us to select the probes with highest intensity that were not outliers.</p><p>Selected probes were printed in pairs on two 12-plex microarrays (each array had a 135,000 probe capacity, Roche NimbleGen, Inc., Madison WI). We used a loop design with dye swaps incorporated, allowing us to hybridize 24 RNA samples to each array. For experiment 1 we hybridized 8 haplometrotic queens, 8 pleometrotic winners and 8 pleometrotic losers (<xref ref-type="supplementary-material" rid="pgen.1003633.s005">Figure S5</xref>) and for experiment 2 we compared 6 win/los, 6 los/win, 6 win/win and 5 los/los (<xref ref-type="supplementary-material" rid="pgen.1003633.s006">Figure S6</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Any spots with an intensity of less than 300 (the background level on the arrays) were removed from the analyses, as were spots present on less than 20 out of 24 arrays. Expression data were log-transformed and normalized using mixed-model normalization (proc MIXED, SAS, Cary, NC) with the following model:<disp-formula id="pgen.1003633.e001"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003633.e001.jpg" /></disp-formula>where Y is expression, dye and block are fixed effects, and array, array*dye and array*block are random effects. Transcripts with significant expression differences between groups were detected by using a mixed-model ANOVA with the model:<disp-formula id="pgen.1003633.e002"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003633.e002.jpg" /></disp-formula>where Y represents the residual from the previous model. Treatment, spot and dye are fixed effects and array is a random effect. P-values were corrected for multiple testing using a false discovery rate of &lt;0.001 for experiment 1 and &lt;0.1 for experiment 2 (proc MULTTEST, SAS). Because the number of differentially regulated transcripts for experiment 1 was very high (&#8764;13,000 out of &#8764;50,000), and to avoid an excess of redundancy among the different groups of transcripts, we included only probes corresponding to GM and EWGM (see above).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Hierarchical clustering, using the Ward method, and principal component analysis (PCA) for global patterns of gene expression were performed in JMP 9.0.2 (SAS, Cary, NC). We used Genesis 1.7.6 (Graz, Austria) to cluster differentially regulated genes based on average linkage and to perform k-means clustering in experiment 1. Gene Ontology analysis was performed using functional annotation chart/clustering in DAVID version 6 <xref ref-type="bibr" rid="pgen.1003633-Huang1">[28]</xref>, <xref ref-type="bibr" rid="pgen.1003633-Huang2">[29]</xref> using DAVID default population background and a cutoff of p&lt;0.05. For all Gene Ontology (GO) analyses, fire ant genes were matched to their <italic>Drosophila</italic> orthologs in FlyBase (<ext-link ext-link-type="uri" ns0:href="http://flybase.org/">http://flybase.org/</ext-link>). CateGOrizer <xref ref-type="bibr" rid="pgen.1003633-Hu1">[31]</xref> was used to count the occurrences of significantly enriched GO terms within each of the pre-defined set of parent/ancestor GO terms. The array data were deposited on the ArrayExpress website according to MIAME standards (ArrayExpress accession: E-MEXP-3886 for experiment 1, E-MEXP-3898 for experiment 2).</p><p>We compared the results from experiment 1 to the following studies:</p><p>
<underline>experiment 2</underline> from this study: we overlapped 548 transcripts that were significantly differentially regulated at FDR&lt;0.1 between win and los in experiment 1 to all the significantly differentially regulated transcripts from experiment 2 (606, FDR&lt;0.1).</p><p>
<underline>aging</underline> in <italic>Drosophila</italic>: we compared directional expressions for all the significantly enriched GO terms obtained from Functional Annotation Chart for experiment 1 and the significantly enriched GO terms that were associated to aging in the fruit fly based on Zhan et al. <xref ref-type="bibr" rid="pgen.1003633-Zhan1">[47]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We performed overlaps between list of transcripts and GO terms with Venny <xref ref-type="bibr" rid="pgen.1003633-Oliveros1">[91]</xref>. In the first comparative study we overlapped fire ant transcripts directly, while in the second study we used <italic>Drosophila</italic> orthologues (FlyBase numbers) to compare fire ant transcripts to the genes of the fruit fly. Statistical significance of the overlap was calculated using a hypergeometric test (<ext-link ext-link-type="uri" ns0:href="http://nemates.org/MA/progs/overlap_stats.html">http://nemates.org/MA/progs/overlap_stats.html</ext-link>). Selected GO analyses based on study overlap were performed in DAVID as above. In the second study, to test for the significant agreement in the patterns of expression between two studies we performed Fisher's Exact Tests in JMP.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We examined gene expression levels of the following candidate genes (<xref ref-type="supplementary-material" rid="pgen.1003633.s017">Table S10</xref>): <italic>Indy</italic> and <italic>Sod2</italic> (determination of adult life span); <italic>Dredd</italic> and <italic>kay</italic> (immune response); <italic>desat1</italic>, <italic>ifc</italic> and <italic>Putative fatty acyl-CoA reductase CG5065</italic> (synthesis and metabolism of fatty acids); <italic>br</italic> and <italic>Btk29A</italic> (reproductive functions); <italic>Sema-5c</italic> and <italic>Mer</italic> (olfactory behavior); <italic>fru</italic> (aggressive behavior) and <italic>woc</italic> (neurogenesis). We used the total RNA extracted from fire ant queens for the microarray analysis and compared gene expression between haplo and los on an ABI Prism 7900 sequence detector (Applied Biosystems, Foster City, CA, USA). cDNA was made using SuperScript III First-Strand Synthesis System for RT-PCR (Invitrogen-Life Technologies, Carlsbad, CA, USA) and Random Hexamers according to the manufacturer's protocol. The cDNA was then diluted 2(x) with ultra-pure water. Amplification was performed in a 10 &#181;l reaction mixture containing 5 &#181;l of 2&#215; SYBR Green Master Mix (Applied Biosystems-Life Technologies, Carlsbad, CA, USA), 1 &#181;l of each primer (10 &#181;M) and 2 &#181;l of cDNA at the following conditions: 50&#176;C for 2 min, 95&#176;C for 10 min, 40 cycles of 95&#176;C for 15 sec and 60&#176;C for 1 min, a dissociation step of 95&#176;C for 15 sec and 60&#176;C for 15 sec. We used 8 queens per group: triplicate reactions were performed for each of the samples and averaged for use in statistical analysis. Expression levels of candidate genes were normalized to the geometric mean of two housekeeping genes, <italic>Rp-9</italic> and <italic>Rp-37</italic>
<xref ref-type="bibr" rid="pgen.1003633-Wurm1">[27]</xref>. Negative control (cDNA reaction without RT enzyme) was also used. Primer sequences were developed in Primer3Plus (<ext-link ext-link-type="uri" ns0:href="http://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi">http://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi</ext-link>) and primer efficiency was first validated using standard curves. Statistical analysis was performed with nonparametric Kruskall-Wallis rank sums in JMP 10 (SAS, Cary, NC). The data were shown normalized to the haplo group.</p>