<p>We analyzed waist-based traits including waist circumference (WC) and waist-hip ratio (WHR), a measure of body fat distribution <xref ref-type="bibr" rid="pgen.1003681-Krotkiewski1">[43]</xref>. Details regarding trait acquisition within each cohort can be found in the <xref ref-type="supplementary-material" rid="pgen.1003681.s007">Text S1</xref>. Individuals less than 20 years of age were excluded from all analyses. Within each cohort, we created two sets of residuals for WC and WHR, one adjusted for age, age<sup>2</sup>, study site (if applicable) and another additionally adjusted for BMI. Analyses were conducted separately for men and women. The raw residuals were then transformed through an inverse normal function for each subgroup and these transformed residuals were used as our phenotypes in the association analyses. The cohorts with related individuals additionally performed sex-combined analysis. We analyzed four phenotypes: waist circumference (WC), waist circumference adjusted for BMI (WC-BMI), waist hip ratio (WHR) and waist hip ratio adjusted for BMI (WHR-BMI).</p><p>We conducted analysis of WC and WHR in up to 33738 and 27489 AA individuals, respectively. Specifically, the analysis included for WC up to 23,564 individuals and WHR up to 19,744 individuals in stage 1 while 10,174 AA individuals with WC and 7,745 AA individuals with WHR in stage 2. Stage 1 cohorts were part of the CARe consortium and other cohorts that were identified with GWAS data at the time the study started. Stage 2 cohorts with in silico GWAS data were identified later. Some participating studies, including CFS, Family Heart Study, GENOA, HUFS, HyperGEN, GeneSTAR, JHS, MESA-family and SIGNET (REGARDS, SUGAR), are family studies. The CARe consortium (ARIC, CARDIA, CFS, JHS, MESA) consists of several population-based studies that included African ancestry individuals. The WHI study was a clinical trial. HANDLS is a community-based study. Family Heart Study is a multicenter family-based study. GeneSTAR is a prospective study of vascular diseases. GENOA and HYPERGEN are cohorts of sibships enriched for hypertension. Health ABC is a random sample of Medicare beneficiaries in and surrounding Pittsburgh, Pennsylvania, and Memphis, Tennessee. HUFS is a population-based family study in the Washington, D.C. metropolitan area. MEC is a prospective cohort study including a nested breast cancer case-control study. WCHS is a case-control study of breast cancer in the New York City and New Jersey. CBCS is population-based case-control study on Breast cancer. Both MEC and MDA are Prostate cancer case-control studies. Black Women's Health Study is an ongoing follow-up study of 59,000 African American women from across the U.S. CHS is a population-based study of risk factors for CHD and stroke. REGARDS is an observational cohort and SUGAR is a community based family studies focusing on Type 2 Diabetes. Each participating study has obtained institutional review board approval on research involving human subjects and all subjects provided written informed consent. Details regarding each cohort can be found in the <xref ref-type="supplementary-material" rid="pgen.1003681.s007">Text S1</xref>.</p><p>Genotype information for each cohort is presented in <bold>Supplementary <xref ref-type="supplementary-material" rid="pgen.1003681.s003">Table S2</xref></bold>. As shown in <xref ref-type="table" rid="pgen-1003681-t001">
<bold>Table 1</bold>
</xref>, the genetic ancestry of our samples, African American, is also partly from European Ancestry. Simply using all YRI sample as reference panel would be inappropriate, given that we generally see an average of 20% CEU admixture. To better capture the genetic structure of our samples, all the genotypes from discovery cohorts were imputed using combined HapMap 1&#8758;1 CEU+YRI as reference panel. This imputation has resulted in an allelic concordance rate of 95.6%, which is compatible to rates calculated with the HapMap 2 YRI individuals <xref ref-type="bibr" rid="pgen.1003681-Huang1">[44]</xref>. Our follow-up stage (Stage 2) included in silico and de novo follow-up cohorts. In-silico follow-up studies similarly use the combined CEU+YRI in HapMap as their reference panel for genotype imputation. Then the expected allele dosage was used in the association analysis to account for the uncertainty introduced by the genotype imputation. More details regarding imputation were in <bold>Supplementary <xref ref-type="supplementary-material" rid="pgen.1003681.s003">Table S2</xref></bold>.</p><p>In each discovery study, genome-wide tests for association between SNPs and phenotypes were conducted separately for men and women using linear regression with principal components adjustment to adjust for global population substructure. In studies from families, men and women were also combined for analyses. Linear mixed effect models, where appropriate, were used to account for the relatedness in family studies.</p><p>In addition to study-specific filters, a centralized quality control procedure was performed to extensively examine and check all study-specific results files before meta-analysis. We examined the plausible values for all reported summary statistics to check for potential errors. The genomic control lambda (&#955;) value for each set of results was checked for potential p-value inflation. We analyzed SNPs with imputation quality scores greater than 0.3 for studies using MACH or BimBam software, and greater than 0.4 for studies that used other imputation software such as IMPUTE. Additionally, we filtered out SNPs where the minor allele frequency times the number of subjects was smaller than or equal to 5, to ensure robust estimates.</p><p>We performed fixed-effects meta-analyses of study-specific genome-wide association results using the inverse-variance weighted approach for the traits described above. Three sets of meta-analyses were conducted for each phenotype using (1) men only results, (2) women only results, and (3) joint men-only and women-only results for studies of unrelated individuals, and sex-combined results for studies with related individuals. The calculated &#955; genomic control (GC) correction was applied to each cohort's result. Recent studies showed that under polygenic inheritance, test statistics in large meta-analyses are expected to be elevated even when there is no population sub-structure <xref ref-type="bibr" rid="pgen.1003681-Yang1">[22]</xref>, <xref ref-type="bibr" rid="pgen.1003681-Lango1">[23]</xref>. To avoid an overly conservative adjustment, we focused on the single GC-corrected result. However, we also report the double GC-corrected p-value for the joint meta-analysis of the stage 1 and stage 2 samples.</p><p>As a sensitivity analysis, we assessed the impact of local ancestry by including SNP specific local ancestry estimates as a covariate in models for genome-wide significant signals in both the CARE and WHI studies. Locus-specific ancestry (i.e. probabilities of whether an individual has 0, 1, or 2 alleles of African ancestry at each locus) was only available for directly genotyped SNPs and was estimated using a Hidden Markov Model and the local haplotype structure to detect transitions in ancestry along the genome <xref ref-type="bibr" rid="pgen.1003681-Tang1">[45]</xref>. We considered signals robust to adjustment for local ancestry when the Beta was numerically similar.</p><p>We applied a procedure to evaluate the transferability of association signals across different ethnicities <xref ref-type="bibr" rid="pgen.1003681-Liu3">[46]</xref>. Specifically, in addition to validating the previously reported index SNPs identified in studies of EA participants <xref ref-type="bibr" rid="pgen.1003681-Heid1">[17]</xref>, we interrogated the surrounding genomic regions in our AA samples. For each reported index EA SNP, we first examined the results in our AA samples and tested for consistency of direction, with respect to the beta coefficients of index SNPs, between EA and AA samples. To accommodate the difference of LD structure across ethnicities, we then interrogated &#177;250 kb regions around the index SNPs and identified the SNP with the smallest p-value in AA within the interrogated genomic region. The loci-specific significance threshold was based on Bonferroni correction, defined as 0.05 divided by the number of independent SNPs within an interrogated region. SNPs meeting genome-wide (p&lt;5.0&#215;10<sup>&#8722;8</sup>) or suggestive (5.0&#215;10<sup>&#8722;8</sup>&lt;p&lt;5.0&#215;10<sup>&#8722;6</sup>) in the Stage 1 meta-analysis were carried forward for follow-up in Stage 2 and joint Stage 1 and 2 meta-analysis. To test the consistency of effect directions between AA and EA samples, a p-value was calculated based on the cumulative binomial distribution for the observed or more extreme number of variants with a consistent direction.</p><p>An analysis approach consistent with the discovery stage (ie Stage 1), described above, was used for the Stage 2 studies. In this stage, the variants of interest identified from our analysis of Stage 1 and interrogation of previously published EA WHR-BMI loci were followed up in different samples for follow-up meta-analysis and confirming the association. We then conducted additional joint meta-analysis, including studies from both Stage 1 and Stage 2 discovery and follow-up data. In Stage 2 analysis, the replication was defined as having a beta coefficient consistent with the discovery stage; follow-up p-values are thus represented as one-sided tests. For the joint analysis, we used the standard threshold of p-value&lt;5&#215;10<sup>&#8722;8</sup> for genome-wide significance and a locus-specific Bonferroni corrected threshold for the regions identified by the GIANT consortium.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For newly identified SNPs from both genome-wide association analyses and previous region interrogation analyses, we performed sex-specific association analyses and also tested the difference of meta-analyzed sex-specific beta-estimates (<inline-formula><inline-graphic ns0:href="pgen.1003681.e001.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1003681.e002.jpg" /></inline-formula>) using the t-statistic<disp-formula id="pgen.1003681.e003"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003681.e003.jpg" /></disp-formula>where <italic>r</italic> is the Spearman rank correlation coefficient between <inline-formula><inline-graphic ns0:href="pgen.1003681.e004.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1003681.e005.jpg" /></inline-formula> across all SNPs. Note that we are comparing two parameters and testing whether their difference is equal to zero. This is basically the setting of a two-sample test. Although based on our sample size (n&gt;5000 in combined analysis), the Z-statistic should work well due to the Central Limit Theory. However, we intended to conservatively use the t-statistic here to calculate the p-value.</p><p>We also examined the GIANT consortium results <xref ref-type="bibr" rid="pgen.1003681-Heid1">[17]</xref> for evidence of association for the novel loci identified in the AA samples. We applied similar interrogation procedures detailed in the previous section of the Interrogation of GIANT Loci in the samples of African Ancestry. Briefly, we first looked up the association results for the AA index SNP in GIANT and followed up with the interrogation of its &#177;250 kb flanking region. The significance was evaluated as 0.05 divided by the number of independent variants within the interrogated region.</p><p>For the newly identified SNPs from both GWAS and the interrogation analysis we performed cross-trait association analyses of metabolic risk factor and related anthropometric measures, including BMI <xref ref-type="bibr" rid="pgen.1003681-Monda1">[26]</xref> , HDL-cholesterol <xref ref-type="bibr" rid="pgen.1003681-Lettre1">[24]</xref>, LDL-cholesterol <xref ref-type="bibr" rid="pgen.1003681-Lettre1">[24]</xref>, triglycerides <xref ref-type="bibr" rid="pgen.1003681-Lettre1">[24]</xref>, glucose and insulin <xref ref-type="bibr" rid="pgen.1003681-Liu1">[25]</xref>, and height (Christopher Haiman, personal communication), in AA samples for the newly identified SNPs from both genome-wide association analysis and the interrogation analysis.</p>