<p>Eligible subjects were female carriers of deleterious mutations in <italic>BRCA1</italic> or <italic>BRCA2</italic> aged 18 years or older <xref ref-type="bibr" rid="pgen.1004256-Antoniou3">[6]</xref>. A total of 55 collaborating CIMBA studies contributed genotypes for the study. Numbers of samples included from each are provided in <xref ref-type="supplementary-material" rid="pgen.1004256.s003">Table S2</xref>. A total of 1,787 mutation carriers (968 with mutations in <italic>BRCA1</italic> and 819 with mutations in <italic>BRCA2</italic>) from the CNIO, HCSC, ICO, Demokritos and MBCSG were genotyped in the first stage of the study. Stage II included 23,463 CIMBA samples (15,252 with mutations in <italic>BRCA1</italic> and 8,211 with mutations in <italic>BRCA2</italic>). All carriers participated in clinical and/or research studies at the host institution under IRB-approved protocols.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Eighteen genes (<italic>UNG</italic>, <italic>SMUG1</italic>, <italic>MBD4</italic>, <italic>TDG</italic>, <italic>OGG1</italic>, <italic>MUTYH</italic>, <italic>NTHL1</italic>, <italic>MPG</italic>, <italic>NEIL1</italic>, <italic>NEIL2</italic>, <italic>APEX1</italic>, <italic>APEX2</italic>, <italic>LIG3</italic>, <italic>XRCC1</italic>, <italic>PNKP</italic>, <italic>POLB</italic>, <italic>PARP1 and PARP2</italic>) involved in the BER pathway were selected, based on the information available at <ext-link ext-link-type="uri" ns0:href="http://www.cgal.icnet.uk/DNA_Repair_Genes.html">http://www.cgal.icnet.uk/DNA_Repair_Genes.html</ext-link> as at the 31<sup>st</sup> December, 2009. Tag SNPs for the selected genes were defined using Haploview v.4.0 (<ext-link ext-link-type="uri" ns0:href="http://www.broad.mit.edu/mpg/haploview">http://www.broad.mit.edu/mpg/haploview</ext-link>) with an r<sup>2</sup> threshold of 0.8 and a minimum minor allele frequency of 0.05. In addition, SNPs with potentially functional effects already described in the literature were selected. A final number of 144 SNPs was included in an oligonucleotide pool assay for genotyping using the Illumina Veracode technology (Illumina Inc., San Diego, CA). Three hundred nanograms of DNA from each sample were genotyped using the GoldenGate Genotyping Assay with Veracode technology according to the published Illumina protocol. Genotype clustering and calling were carried out using the GenomeStudio software. SNPs with a call rate &lt;0.95 were excluded from further analysis. Duplicate samples and CEPH trios (Coriell Cell Repository, Camden, NJ) were genotyped across the plates. SNPs showing Mendelian allele-transmission errors or showing discordant genotypes across duplicates were excluded.</p><p>To test for departure from Hardy-Weinberg equilibrium (HWE), a single individual was randomly selected from each family and Pearson's X<sup>2</sup> Test (1df) was applied to genotypes from this set of individuals. The association of the SNPs with breast cancer risk was assessed by estimating hazard ratios (HR) and their corresponding 95% confidence intervals (CI) using weighted multivariable Cox proportional hazards regression with robust estimates of variance <xref ref-type="bibr" rid="pgen.1004256-Antoniou7">[34]</xref>. For each mutation carrier, we modeled the time to diagnosis of breast cancer from birth, censoring at the first of the following events: bilateral prophylactic mastectomy, breast cancer diagnosis, ovarian cancer diagnosis, death or date last know to be alive. Subjects were considered affected if their age at censoring corresponded to their age at diagnosis of breast cancer and unaffected otherwise. Weights were assigned separately for carriers of mutations in <italic>BRCA1</italic> and <italic>BRCA2</italic>, by age and affection status, so that the weighted observed incidences in the sample agreed with established estimates for mutation carriers <xref ref-type="bibr" rid="pgen.1004256-Antoniou1">[1]</xref>; <xref ref-type="bibr" rid="pgen.1004256-Antoniou7">[34]</xref>.</p><p>We considered log-additive and co-dominant genetic models and tested for departure from HR&#8202;=&#8202;1 by applying a Wald test based on the log-HR estimate and its standard error. Additional independent variables included in all analyses were year of study, centre and country. All statistical analyses were carried out using Stata: Release 10 (StataCorp. 2007. Stata Statistical Software: Release 10.0. College Station, TX: Stata Corporation LP). Robust estimates of variance were calculated using the <italic>cluster</italic> subcommand, applied to an identifier variable unique to each family.</p><p>Stage II of the experiment was performed as part of the iCOGS genotyping experiment. The iCOGS custom array was designed in collaboration between the Breast Cancer Association Consortium (BCAC), the Ovarian Cancer Association Consortium (OCAC), the Prostate Cancer Association Group to Investigate Cancer Associated in the Genome (PRACTICAL) and CIMBA. The final design comprised 211,155 successfully manufactured SNPs of which approximately 17.5% had been proposed by CIMBA. A total of 43 SNPs were nominated for inclusion on iCOGS based on statistical evidence of association in stage I of the present study (p&#8804;0.05). Of these, 36 were successfully manufactured and genotyped in CIMBA mutation carriers.</p><p>Genotyping was performed at Mayo Clinic and the McGill University and G&#233;nome Qu&#233;bec Innovation Centre (Montreal, Canada). Genotypes were called using Illumina's GenCall algorithm. Sample and quality control process have been described in detail elsewhere <xref ref-type="bibr" rid="pgen.1004256-Couch1">[32]</xref>, <xref ref-type="bibr" rid="pgen.1004256-Gaudet1">[35]</xref>. After the quality control process a total of 23,463 carriers were genotyped for the 36 selected SNPs.</p><p>Both breast and ovarian cancer associations were evaluated in stage II. Censoring for breast cancer followed the same approach as in stage I. Censoring for ovarian cancer risk occurred at risk-reducing salpingo-oophorectomy or last follow-up.</p><p>The genotype-disease associations were evaluated within a survival analysis framework, by modelling the retrospective likelihood of the observed genotypes conditional on the disease phenotypes <xref ref-type="bibr" rid="pgen.1004256-Antoniou6">[9]</xref>, <xref ref-type="bibr" rid="pgen.1004256-Antoniou7">[34]</xref>, <xref ref-type="bibr" rid="pgen.1004256-Barnes1">[36]</xref>, <xref ref-type="bibr" rid="pgen.1004256-Barnes2">[37]</xref>. The associations between genotype and breast or ovarian cancer risk were assessed using the 1 d.f. score test statistic based on this retrospective likelihood. To allow for the non-independence among related individuals, we accounted for the correlation between the genotypes by estimating the kinship coefficient for each pair of individuals using the available genomic data <xref ref-type="bibr" rid="pgen.1004256-Antoniou7">[34]</xref>, <xref ref-type="bibr" rid="pgen.1004256-Amin1">[38]</xref>, <xref ref-type="bibr" rid="pgen.1004256-Leutenegger1">[39]</xref>. These analyses were performed in R using the GenABELlibraries and custom-written functions in FORTRAN and Python.</p><p>To estimate the magnitude of the associations (HRs), the effect of each SNP was modeled either as a per-allele HR (multiplicative model) or as genotype-specific HRs, and was estimated on the log-scale by maximizing the retrospective likelihood. The retrospective likelihood was fitted using the pedigree-analysis software MENDEL. The variances of the parameter estimates were obtained by robust variance estimation based on reported family membership. All analyses were stratified by country of residence and based on calendar-year and cohort-specific breast cancer incidence rates for mutation carriers. Countries with small number of mutation carriers were combined with neighbouring countries to ensure sufficiently large numbers within each stratum. USA and Canada were further stratified by reported Ashkenazi Jewish (AJ) ancestry.</p><p>Genotypes were imputed separately for <italic>BRCA1</italic> and <italic>BRCA2</italic> mutation carriers using the v3 April 2012 release (Genomes Project et al., 2012) as reference panel. To improve computation efficiency we used a two-step procedure which involved pre-phasing in the first step and imputation of the phased data in the second. Pre-phasing was carried out using the SHAPEIT software <xref ref-type="bibr" rid="pgen.1004256-Delaneau1">[40]</xref>. The IMPUTE version 2 software was used for the subsequent imputation <xref ref-type="bibr" rid="pgen.1004256-Howie1">[41]</xref>. SNPs were excluded from the association analysis if their imputation accuracy was r2&lt;0.3 or MAF&lt;0.005 in any of the data sets. For the final analysis we only took in account those SNPs with an imputation accuracy r2&gt;0.7, MAF&gt;0.01 and being located in the region comprised within 15 kilo bases (kb) downstream and upstream the gene where the genotyped SNP showing an association was located (<xref ref-type="table" rid="pgen-1004256-t001">Table 1</xref>). Associations between imputed genotypes and breast cancer risk were evaluated using a version of the score test as described above but with the posterior genotype probabilities replacing the genotypes.</p>