<p>We used data from a total of 5126 HIV-1-infected individuals from eight US and one European cohorts: the AIDS Linked to Intravenous Experience (ALIVE), the U.S. military HIV Natural History Study (DoD HIV NHS), the DC Gay Cohort Study (DCG), the Multicenter AIDS Cohort Study (MACS), the Multicenter Hemophilia Cohort Study (MHCS), the Massachusetts General Hospital Controller Cohort (MGH), the San Francisco City Clinic Cohort (SFCCC), the Study on the Consequences of Protease Inhibitor Era (SCOPE) and the Swiss HIV Cohort Study (SHCS). Patients from MACS, MGH, SCOPE and SHCS, including 2685 white and 1306 black patients, were categorized in controller and noncontroller groups for the analysis of <italic>HLA</italic> class I impact on HIV-1 immune control. Longitudinal viral load data were available for 2900 white and 1490 black patients from ALIVE, MACS, MGH, DoD HIV NHS, SCOPE and SHCS. Seroconversion time and AIDS progression data were known for 780 white and 287 black patients from ALIVE, DCG, MHCS and SFCCC.</p><p>This study was approved by the protocol review office of the US National Cancer Institute institutional review board, as well as by the institutional review board of Massachusetts General Hospital. Informed consent was obtained at the study sites from all individuals. Patients' ethnicities were defined based on self-report.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We performed genotyping of the <italic>HLA-A/B/C</italic> following the PCR-SSOP (sequence-specific oligonucleotide probing) typing protocol and PCR-SBT (sequence based typing) recommended by the 13<sup>th</sup> International Histocompatibility Workshop (<ext-link ext-link-type="uri" ns0:href="http://www.ihwg.org">http://www.ihwg.org</ext-link>). All <italic>HLA</italic> class I genotypes were defined to 4-digit resolution with the exceptions of <italic>A*74:01/2</italic>, <italic>C*17</italic> and <italic>C*18</italic>, which were determined to 2-digits.</p><p>LILRB2-HLA binding scores were defined in a previously described experiment <xref ref-type="bibr" rid="pgen.1004196-Jones1">[22]</xref>. Briefly, a set of LILR-Fc fusion proteins was tested for binding with LABScreen HLA class I SABs at a concentration of 0.5, 1 and 2 &#181;M. The level of binding was assessed by measuring the median fluorescence intensity (MFI) of the LILR-Fc bound to the beads using appropriate normalizations, which included subtraction of the Fc-negative control MFI and division of the result by the MFI of W6/32 (monoclonal anti-HLA class I antibodies recognizing &#946;2m-associated HLA molecules). The normalized values were assigned to each HLA allotype as binding scores. Each binding score is a function of avidity of bivalent LILRB2-Fc for HLA, which in turn depends on the affinity of monomeric LILRB to HLA. Therefore, the binding score can be used as a quantitative characteristic of the strength of LILR-HLA interactions. The relative LILR binding to different HLA allotypes was similar at each of the LILR concentrations tested (<xref ref-type="supplementary-material" rid="pgen.1004196.s004">Figure S4</xref>). This consistency between LILR concentrations assured us that the difference in MFIs between the allotypes is mainly due to difference in binding strength, and is not an experimental artifact. The 1 &#181;M concentration results were chosen as a representative dataset. Among the HIV-1-infected patients used for the analyses, frequencies of the <italic>HLA-A/B/C</italic> alleles with unknown binding scores were 2/11/24% in white and 8/13/39% in black patients. To avoid power loss, we used mean values for the corresponding locus for each genotype with unknown score. The pairs of alleles <italic>A*74:01/2</italic> and <italic>B*81:01</italic> differ only at the signal peptide, therefore, they were treated as individual alleles in the context of LILRB2 binding.</p><p>Monocyte Derived Dendritic Cells (MDDC) were prepared as described previously. Briefly, 2 &#215; 10<sup>8</sup> PBMCs were plated in 5% pooled human serum medium and incubated during 60 min at 37&#176;C to adhere monocytes. After discarding non-adherent cells, monocytes were differentiated into MDDC in the presence of RPMI 1640 medium supplemented with 50 &#181;g/ml of GM-CSF (Amgen) and 10% fetal bovine serum. On day 5, immature MDDC were gently detached using PBS with 0.5% BSA and 2 mM EDTA, harvested and plated at 4x10<sup>5</sup> cell/well in a round-bottom 96-well plate (Costar). Next, cells were incubated with beads coated with selected HLA-B allotypes, or uncoated control beads (One Lambda) for 30 min at 37&#176;C, washed, and subsequently matured in the presence of a previously described cytokine cocktail containing 5 ng/ml IL-1&#946;, 5 ng/ml TNF&#945;, 1 &#181;g/ml PGE-2 and 0.15 &#181;g/ml IL-6. After 16 hours, mature MDDC were mixed with negatively-isolated CFSE-labeled allogeneic T cells at a DC:T cell ratio of 1&#8758;100 for mixed lymphocyte reactions. Allogeneic T cell proliferation was determined after 6 days in culture by investigators blinded towards the added HLA class I molecules, using an LSRFortessa flow cytometer (Becton Dickinson).</p><p>To analyze cytokine secretion, immature MDDC were prepared and treated with HLA class I molecules as described above and then matured using 5 &#181;g/ml CL097 (InvivoGen) in the presence of 5 &#181;g/ml brefeldin A. After 20 hours, cells were fixed and permeabilized, stained with antibodies recognizing intracellular IL-12p70, TNF&#945; and IL-6, and processed to flow cytometric acquisition by investigators blinded towards the added HLA class I allotypes.</p><p>10<sup>6</sup> MDDC were suspended in 300 &#181;l Optimem (Gibco) in the presence of 2 nmol of either LILRB2-specific (LILRB2-siRNA) or scramble control siRNA (sc-siRNA) pools (On-TARGET plus SMARTpool, Dharmacon) and transferred to a 4-mm electroporation cuvette (Bio-Rad Laboratories). Cells were left on ice for 10 min, electroporated (900 V, 0.75 msec square wave; Genepulser Xcell; Bio-Rad Laboratories), and transferred back to culture medium for another 24 to 48 hours. Efficiency of specific siRNA-mediated LILRB2 knockdown was determined by flow cytometry using an anti-LILRB2 antibody (clone 42D1, Biolegend).</p><p>We used SAS 9.1 (SAS Institute) for data management and statistical analyses. The effect of <italic>HLA</italic> alleles on viral control was determined by categorical analysis of the allelic frequencies in HIV-1 controllers and noncontrollers. Corresponding ORs were calculated using logistic regression model with SAS procedure PROC LOGISTIC. Relationships between viral loads and LILRB2-HLA binding scores were analyzed by the Spearman correlation test using PROC CORR. Permutation analysis was done by random assignment of binding scores to <italic>HLA-B</italic> alleles (10,000 times) and testing the probability of significant Spearman correlation of the binding scores with ORs with p&lt;0.05.</p><p>LILRB2 binding scores as continuous variables and presence versus absence of all individual <italic>HLA</italic> class I alleles of frequency &#8805;2% were included with stepwise selection in all regression models. Results in the tables are for the models using a threshold of a two-sided p value &lt;0.05 for inclusion of a covariate as a significant independent effect. The stability of regression models was tested using more stringent thresholds of p&lt;0.01 and p&lt;0.001 for inclusion in the model. The results for the binding scores are indicated in the footnotes to the tables.</p><p>Cox proportional hazards model was applied to perform AIDS progression analysis by using PROC PHREG. For this, we estimated the seroconversion date as the midpoint between the first positive and the last negative HIV-1 antibody test (mean interval, 0.79 years; range, 0.07 to 3.0 years). Four end points reflecting disease progression (AIDS outcomes) were evaluated: time to CD4&lt;200 cells/ml; progression to AIDS according to the 1987 definition by the Centers for Disease Control and Prevention (CDC, <xref ref-type="bibr" rid="pgen.1004196-1">[40]</xref>); progression to AIDS according to the 1993 definition by CDC; and AIDS-related death <xref ref-type="bibr" rid="pgen.1004196-2">[41]</xref>.</p><p>Data of <italic>in vitro</italic> experiments were presented as Box and Whisker plots, reflecting the median, minimum, maximum and the 25<sup>th</sup> and 75<sup>th</sup> percentiles. Significance was tested using one-way ANOVA followed by post-hoc analysis with the Tukey multiple comparison test, or using paired t-tests, as appropriate.</p>