<p>The association panel consists of 282 diverse maize lines that have been previously described <xref ref-type="bibr" rid="pgen.1003246-FlintGarcia1">[42]</xref>. These lines can be subdivided into five major subpopulations, namely stiff stalk, non-stiff stalk, tropical or semitropical lines (related to the non-stiff stalk lines), sweet corn and popcorn. The association panel includes the 25 founder lines of the NAM population. The maize NAM population consists of 5,000 RILs (Recombinant Inbred Lines) derived from crossing B73 with 25 diverse maize inbred lines, and then selfing for 5 generations <xref ref-type="bibr" rid="pgen.1003246-Yu2">[43]</xref>.</p><p>Phenotypic data were collected from the NAM population and the 282 association panel, grown in eight environments including Ithaca, NY, Clayton, NC, Champaign, IL, and Colombia, MO, during the summers of 2006 and 2007. Flowering time was measured separately for female flowers (number of days-to-silk) and male flowers (days-to-anthesis) from the day of planting. The flowering date was defined as the day when the anthers or silk were visible on 50% of all plants within a row.</p><p>DNA sequence data were obtained for <italic>d8</italic> from Thornsberry et al. (2001), available at NCBI. Primers were designed for PCR amplification of gene fragments of interest from the 282 lines in the association panel.</p><p>Each PCR product was cleaned by treating the samples with Exonuclease (ExoI) and Shrimp Alkaline Phosphatase (SAP) and incubated at 37&#176;C for 3 min followed by 80&#176;C for 10 min. The samples were prepared for sequencing using a mixture with a total volume of 10 &#181;l containing 0.7 &#181;l forward primer, 0.7 &#181;l reverse primer (5 pmol/&#181;l), 0.5 &#181;l Big Dye terminator, 1.7 &#181;l 5&#215; sequencing buffer, 7.1 &#181;l distilled water and the PCR product. The thermal cycler was set on the following program: Initial denaturation at 96&#176;C for 4 min, followed by 30 cycles at 96&#176;C for 10 sec, 50&#176;C for 5 sec and 60&#176;C 4 min, with a final, incubation at 10&#176;C. Sanger(3730XL) DNA sequencing was performed using an Applied Biosystems Automated 3730 DNA Analyzer. The software BioLign alignment and multiple contig editor with codon code phred-phrap analysis was used for alignment using consensus sequence contigs and sequence quality scores.</p><p>The alignments from NCBI were also used to reanalyze the results published in the initial study by Thornsberry et al. (2001).</p><p>To obtain sequence data for the region between <italic>d8</italic> and <italic>tb1</italic>, the same protocol was used as described above. However, primer sequences were obtained from Camus-Kulandaivelu et al. (2008)</p><p>Best linear unbiased predictions (BLUPs) of the lines in the 282 association panel and the NAM population were the same as those reported in Buckler et al. (2009). These were obtained from a random effects model fitted in ASREML version 2.0 software <xref ref-type="bibr" rid="pgen.1003246-Gilmour1">[44]</xref> that accounts for spatial correlation and field effects.</p><p>TASSEL (Trait Analysis by aSSociation, Evaluation, and Linkage) was used for data processing analysis <xref ref-type="bibr" rid="pgen.1003246-Bradbury1">[45]</xref>, and results were confirmed by using SAS <xref ref-type="bibr" rid="pgen.1003246-SAS1">[46]</xref>. Association between polymorphisms and phenotypes were evaluated using General Linear Model (GLM) and Mixed Linear Model (MLM) by incorporating phenotypic and genotypic data, population structure (Q) and kinship matrix (K).</p><p>Population structure was predicted using a Bayesian approach that estimates the relationship between subpopulations by grouping genotypic correlations at unlinked markers within the population with the software STRUCTURE <xref ref-type="bibr" rid="pgen.1003246-Pritchard3">[18]</xref> as described in <xref ref-type="bibr" rid="pgen.1003246-FlintGarcia1">[42]</xref>. This approach uses the proportion of an individual's genome that originated from each subpopulation to estimate the genetic background matrix (Q).</p><p>In MLM, the familial relatedness between the individuals is taken into consideration through a kinship matrix. This model corrects for spurious associations arising from population structure and familial relatedness <xref ref-type="bibr" rid="pgen.1003246-Stich1">[47]</xref>. In this study we used marker-based kinship, which was determined on the basis of the definition that random pairs of inbreds are unrelated. Kinship was calculated using the software package SPAGeDi <xref ref-type="bibr" rid="pgen.1003246-Hardy1">[48]</xref>. It has been suggested that marker based kinship is more appropriate for association studies than kinship based on pedigree records <xref ref-type="bibr" rid="pgen.1003246-Falush1">[19]</xref>, <xref ref-type="bibr" rid="pgen.1003246-Clark1">[40]</xref>. The same set of markers was used to create the population structure and kinship matrix.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The GLM approach in this study for phenotype, <bold>y</bold>, is:<disp-formula id="pgen.1003246.e001"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003246.e001.jpg" /><label>(1)</label></disp-formula>Where, the <bold>X&#946;</bold> term represents the fixed effects, including genotypes and population structure, Q, and <bold>&#949;</bold> is a vector of residual effects following a multivariate normal distribution with mean 0 and variance-covariance matrix &#963;<sup>2</sup>
<sub>&#949;</sub>I. The na&#239;ve model is the same as GLM without the population structure effect.</p><p>The MLM approach in this study is the same model as used by Yu et al., 2006.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Mixed model, for phenotype, <bold>y</bold>, is:<disp-formula id="pgen.1003246.e002"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003246.e002.jpg" /><label>(2)</label></disp-formula>Where, the <bold>X&#946;</bold> term represents the fixed effects, including genotypes and population structure, Q, and the <bold>Z&#956;</bold> term represents random line effects, including the matrix of kinship coefficients, K, and vector of polygene background effects. <bold>&#949;</bold> is a vector of residual effects following a multivariate normal distribution with mean 0 and variance-covariance matrix &#963;<sup>2</sup>
<sub>&#949;</sub>I.</p><p>Genome-wide association studies (GWAS) were carried out in the 282 association panel using 51,741 SNPs obtained from the Illumina MaizeSNP50 BeadChip and 591,552 SNPs from the genotyping by sequencing (GBS) protocol <xref ref-type="bibr" rid="pgen.1003246-Elshire1">[49]</xref>. Three different approaches that take into account varying degrees of population structure and familial relatedness were undertaken. The first approach, called the na&#239;ve approach, uses a model similar to the one presented in <xref ref-type="disp-formula" rid="pgen.1003246.e001">Equation (1)</xref>, except that the Q matrix representing population structure is not included among the fixed effects. The next approach is the GLM approach, which uses the model in <xref ref-type="disp-formula" rid="pgen.1003246.e001">Equation (1)</xref>, with the first five principal components (PCs) of the non-industry subset of the Illumina MaizeSNP50 BeadChip SNPs (34,368 SNPs) included as fixed effects to represent population structure. The final approach is the MLM approach, with the aforementioned first five PCs representing population structure, and a kinship matrix calculated from the non-industry subset of these SNPs for the variance-covariance matrix of the random line effects. This kinship matrix is calculated using the method of <xref ref-type="bibr" rid="pgen.1003246-Loiselle1">[50]</xref>. In each approach, these models are fitted to each SNP. After all SNPs with minor allele frequencies (MAFs) less than 0.05 are removed from the analysis, the Benjamini-Hochberg <xref ref-type="bibr" rid="pgen.1003246-Benjamini1">[51]</xref> procedure adjusts for the multiple testing problem by controlling the false discovery rate (FDR) at 0.05. This phase of the anlaysis was conducted using the genome association and prediction integrated tool (GAPIT) package in the R programming language <xref ref-type="bibr" rid="pgen.1003246-Lipka1">[52]</xref>.</p><p>Joint Linkage Mapping of BLUPs for the phenotype across environments was performed using the proc GLMSelect in SAS, as described in Buckler et al. (2009) <xref ref-type="bibr" rid="pgen.1003246-Buckler1">[11]</xref>. BLUPs were calculated for each phenotype and used together with imputed genetic marker intervals and stepwise regression to identify QTLs. Missing marker data were imputed by utilizing genetic distance between missing and flanking markers. A permutation procedure was implemented to obtain empirical &#945;&#8202;=&#8202;0.05 thresholds for including and excluding terms in the joint linkage model <xref ref-type="bibr" rid="pgen.1003246-Churchill1">[53]</xref>.</p><p>To calculate the linkage disequilibrium between the SNPs within <italic>d8</italic>, tested in this association study, against the rest of the genome the LD function SitebyAll in the TASSEL software was used <xref ref-type="bibr" rid="pgen.1003246-Bradbury1">[45]</xref>. For genotypic data, the Illumina MaizeSNP50 Beadchip was used, as well as 458k GBS (Genotyping by Sequencing) SNPs <xref ref-type="bibr" rid="pgen.1003246-Elshire1">[49]</xref>.</p>