<p>This analysis uses data from the Colon Cancer Family Registry (CCFR) and the Genetics and Epidemiology of Colorectal Cancer Consortium (GECCO, <xref ref-type="supplementary-material" rid="pgen.1004228.s008">Text S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1004228.s002">Table S1</xref>) as described previously <xref ref-type="bibr" rid="pgen.1004228-Hutter1">[14]</xref>, <xref ref-type="bibr" rid="pgen.1004228-Peters1">[37]</xref>. All cases were defined as colorectal adenocarcinoma and confirmed by medical records, pathologic reports, or death certificate. All studies received ethical approval by their respective Institutional Review Boards and participants gave written informed consent.</p><p>Average sample and SNP call rates, and concordance rates for blinded duplicates have been previously published <xref ref-type="bibr" rid="pgen.1004228-Peters1">[37]</xref>. In brief, genotyped SNPs were excluded based on call rate (&lt;98%), lack of Hardy-Weinberg Equilibrium in controls (HWE, p&lt;1&#215;10<sup>&#8722;4</sup>), and low minor allele frequency (MAF). We imputed the autosomal SNPs of all studies to the CEU population in HapMap II. SNPs were restricted based on per-study minor allele count &gt;5 and imputation accuracy (R<sup>2</sup>&gt;0.3) to avoid missing any interactions. After imputation and quality control (QC) analyses, approximately 2.7M SNPs were used in the analysis.</p><p>All analyses were restricted to individuals of European ancestry, defined as samples clustering with the Utah residents with Northern and Western European ancestry from the CEPH collection (CEU) population in principal component analysis <xref ref-type="bibr" rid="pgen.1004228-Price1">[38]</xref>, including the HapMap II populations as reference.</p><p>Information on basic demographics and environmental risk factors was collected by using in-person interviews and/or structured questionnaires, as detailed previously <xref ref-type="bibr" rid="pgen.1004228-Newcomb1">[39]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004228-Giovannucci1">[48]</xref>. The multi-step data harmonization procedure applied in this study is described in detail by Hutter et al. <xref ref-type="bibr" rid="pgen.1004228-Hutter1">[14]</xref>. Here we focus on selected dietary variables for intake of red and processed meat, fruits, vegetables (all measured in servings per day) and fiber (measured as g/day). These variables were coded as sex- and study-specific quartiles, where the quartile groups were coded 1 to 4 of the quartile within the controls of each study and sex. For studies that due to limited number of questions assessed dietary intake in categories rather than as continuous variables and had less than 4 intake categories, we assigned these categories to the 2<sup>nd</sup> and 3<sup>rd</sup> or 1st to 3<sup>rd</sup> quartile, as appropriate. The lowest category of exposure was used as the reference and each dietary factor was analyzed as an ordinal variable (e.g., 1, 2, 3, 4) in the model. Data harmonization was performed using SAS and T-SQL.</p><p>Statistical analyses of all samples were conducted centrally at the GECCO coordinating center on individual-level data to ensure a consistent analytical approach. Unless otherwise indicated, we adjusted for age at the reference time, sex (when appropriate), center (when appropriate), total energy consumption (if available) and the first three principal components from EIGENSTRAT to account for potential population substructure. The dietary variables were coded as described above. Each directly genotyped SNP was coded as 0, 1, or 2 copies of the variant allele. For imputed SNPs, we used the expected number of copies of the variant allele (the &#8220;dosage&#8221;), which has been shown to give unbiased test statistics <xref ref-type="bibr" rid="pgen.1004228-Jiao1">[49]</xref>. Genotypes were treated as continuous variables (i.e. log-additive effects). Each study was analyzed separately using logistic regression models and study-specific results were combined using fixed-effects meta-analysis methods to obtain summary odds ratios (ORs) and 95% confidence intervals (CIs) across studies. We calculated the heterogeneity p-values by Woolf's test <xref ref-type="bibr" rid="pgen.1004228-Woolf1">[50]</xref>. Quantile-quantile (Q-Q) plots were assessed to determine whether the distribution of the p-values was consistent with the null distribution (except for the extreme tail).</p><p>To test for interactions between SNPs and dietary risk factors, we conduct two primary analyses: 1) conventional case-control logistic regression analysis including a multiplicative interaction term; 2)our newly developed Cocktail method <xref ref-type="bibr" rid="pgen.1004228-Hsu1">[15]</xref>. For the conventional logistic regression analysis, we modeled the SNP by environment (GxE) interaction by the product of the SNP and the dietary variable (which is in this study the E), adjusting for age, sex, study site, energy, principal components and the main effects of the SNP and dietary variable. Adjustment for additional variables, smoking, alcohol, BMI and other dietary variables did not appreciably change the results. A two-sided p-value of 5&#215;10<sup>&#8722;8</sup> for a SNP-diet factor interaction was considered statistically significant, yielding a genome-wide significance level 0.05 assuming about 1 million independent tests across the genome (0.05/1,000,000&#8202;=&#8202;5&#215;10<sup>&#8722;8</sup>) <xref ref-type="bibr" rid="pgen.1004228-Risch1">[51]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004228-Dudbridge1">[56]</xref>.</p><p>Motivated by recent advances in methods development for detecting GxE interaction <xref ref-type="bibr" rid="pgen.1004228-Dai1">[17]</xref>, <xref ref-type="bibr" rid="pgen.1004228-Mukherjee1">[57]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004228-Roeder1">[60]</xref>, our second approach was based on our recently developed Cocktail method. This statistical method combines the most appealing aspect of several newly developed GxE methods with the goal of creating a comprehensive and powerful test for genome-wide detection of GxE <xref ref-type="bibr" rid="pgen.1004228-Hsu1">[15]</xref>. In brief, this method consists of two-steps: a screening step to prioritize SNPs and a testing step for GxE interaction. Specifically, for the screening step, we ranked and prioritized variants through a genome-wide screen of each of the 2.7M SNPs (referred to as &#8220;G&#8221;) by the maximum of the test statistics from marginal association of Gs on disease risk <xref ref-type="bibr" rid="pgen.1004228-Kooperberg1">[58]</xref>, and correlation between G and environmental/dietary variable (E) in cases and controls combined <xref ref-type="bibr" rid="pgen.1004228-Murcray1">[59]</xref>, a combination which allows for identifying variants with different interaction patterns.</p><p>Based on the ranks of these SNPs from screening, we used a weighted hypothesis framework to partition SNPs into groups with higher ranked groups having less stringent alpha-level cut-offs for interaction <xref ref-type="bibr" rid="pgen.1004228-Roeder1">[60]</xref>, <xref ref-type="bibr" rid="pgen.1004228-IonitaLaza1">[61]</xref>. We followed the grouping scheme used by Ionita et al. <xref ref-type="bibr" rid="pgen.1004228-IonitaLaza1">[61]</xref> such that for example, the first 3 groups consist of 5 SNPs (SNP 1 to 5), 10 SNPs (SNP 6 to 15) and 20 SNPs (SNP 16 to 36), and the corresponding cut-offs are &#945;<sub>group 1</sub>&#8202;=&#8202;&#945;/(2*5)&#8202;=&#8202;0.005, &#945;<sub>group 2</sub>&#8202;=&#8202;&#945;/(4*10)&#8202;=&#8202;0.00125 and &#945;<sub>group 3</sub>&#8202;=&#8202;&#945;/(8*20)&#8202;=&#8202;0.0003, respectively, so on and so forth, to maintain the overall genome-wide alpha level of 0.05. To avoid testing correlated SNPs, we pruned SNPs based on proximity (exclude any SNP within +/&#8722;50 kb of the selected SNP) given that LD pruning is difficult to implement for large number of SNPs. While the choice of the group size is arbitrary our simulation study showed that different group size did not impact the results substantially, and importantly, we chose the group size before looking at the results.</p><p>The second step of the Cocktail method is the testing step. We tested each of the G's for GxE interactions using the case-only (CO) logistic regression test. The use of the CO test is justified because we did not observe correlation between G and any of the tested dietary factors, and it has been shown that under the independence assumption the CO test provides substantial efficiency gain over the conventional CC test <xref ref-type="bibr" rid="pgen.1004228-Piegorsch1">[62]</xref>. Since the CO is not independent of the correlation screening (a requirement to avoid inflation of type I error rates) <xref ref-type="bibr" rid="pgen.1004228-Dai2">[63]</xref>, we used CO test only when the maximum screening test statistic came from the marginal association, and the case-control test otherwise.</p><p>In <xref ref-type="supplementary-material" rid="pgen.1004228.s009">Text S2</xref>, we describe two secondary statistical GxE methods that we used to explore other novel GxE methods: the 2-step method by Gauderman et al. method <xref ref-type="bibr" rid="pgen.1004228-Gauderman1">[16]</xref> and a 2 degree of freedom joint test for marginal associations of G and GxE interaction by Dai et al. <xref ref-type="bibr" rid="pgen.1004228-Dai1">[17]</xref>. All analyses were conducted using the R programming language <xref ref-type="bibr" rid="pgen.1004228-4">[64]</xref>.</p>