<p>We used data from 8 genome-wide association studies (GWAS) of European ancestry (<xref ref-type="table" rid="pgen-1003926-t001">Table 1</xref>): (a) three GWAS of the GeneMSA <xref ref-type="bibr" rid="pgen.1003926-DeJager1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1003926-Patsopoulos1">[39]</xref> with samples from the Netherlands (GeneMSA DU), Switzerland (GeneMSA SW), and the United States (GeneMSA US); (b) an early GWAS from the IMSGC <xref ref-type="bibr" rid="pgen.1003926-DeJager1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1003926-Patsopoulos1">[39]</xref>, <xref ref-type="bibr" rid="pgen.1003926-Hafler1">[40]</xref> with samples from the United States (ISMGC US) and the United Kingdom (IMSGC UK), that was collapsed in one stratum removing the UK cases; (c) a GWAS with cases from the Brigham and Women's Hospital and controls from the MIGEN study (BWH) <xref ref-type="bibr" rid="pgen.1003926-DeJager1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1003926-Patsopoulos1">[39]</xref>; (d) the Australia and New Zealand Multiple Sclerosis Genetics Consortium (ANZgene) <xref ref-type="bibr" rid="pgen.1003926-ANZgene1">[41]</xref>; (e) an unpublished GWAS set from Erasmus Medical Center in Rotterdam, the Netherlands; and (f) an unpublished GWAS collection from the Kaiser Permanente MS Research Program (Kaiser Permanente). All the above GWAS data sets were filtered with the same quality control criteria as part of an ongoing meta-analysis of Multiple Sclerosis GWAS. In each of these data sets we performed principal components analysis (PCA) to identify population outliers and to calculate covariates to control for population stratification between cases and controls.</p><p>From each GWAS we extracted SNPs within the extended MHC region (chr6:29,299,390 to 33,883,424; hg18) to impute classical alleles for class I HLA genes (<italic>HLA-A</italic>, <italic>HLA-B</italic>, and <italic>HLA-C</italic>) and class II HLA genes (<italic>HLA-DPA1</italic>, <italic>HLA-DPB1</italic>, <italic>HLA-DQA1</italic>, <italic>HLA-DQB1</italic>, and <italic>HLA-DRB1</italic>), their corresponding amino acid sequences and SNPs not captured in the genotypic platforms used. The imputation was performed with the software BEAGLE <xref ref-type="bibr" rid="pgen.1003926-Browning1">[42]</xref> using a collection of 2,767 individuals of the Type 1 Diabetes Genetics Consortium (T1DGC) with 4-digit classical allele genotyping for the above HLA genes as the reference panel. This method and reference panel have been used for fine-mapping MHC associations in HIV control <xref ref-type="bibr" rid="pgen.1003926-Pereyra1">[8]</xref> and seropositive rheumatoid arthritis <xref ref-type="bibr" rid="pgen.1003926-Raychaudhuri1">[38]</xref>. Cases and controls from each GWAS dataset were imputed together. All variants in the reference panel were coded as biallelic markers (presence vs. absence), allowing us to use BEAGLE for the imputation. Post-imputation we excluded variants with minor allele frequency less than 1% from the analysis. <xref ref-type="supplementary-material" rid="pgen.1003926.s009">Table S5</xref> lists the imputation quality for the identified variants.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We analyzed each variant using a logistic regression model, assuming alleles have an additive effect on the log-odds scale. We also assumed the genetic effects were fixed across all eight GWAS. In each model we included the top 5 principal components to control for within-GWAS population stratification and 7 dummy variables to account for between-GWAS specific effects. Throughout the text we refer to such a model as <italic>univariate</italic> (<italic>M<sub>u</sub></italic>), even if several covariates were included in the model, reflecting the fact that only one MHC-specific variant is included in the model. This is the representation of the univariate model:<disp-formula id="pgen.1003926.e001"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003926.e001.jpg" /><label>(1)</label></disp-formula>
<italic>M<sub>u</sub></italic>, Univariate logistic regression model</p><p>where <italic>y</italic> is the log(odds) for the case-control status, <italic>&#946;<sub>0</sub></italic> is the logistic regression intercept and <italic>&#946;<sub>i,j</sub></italic> is the log-additive effect for the allele <italic>j</italic> of the variant <italic>i</italic> with <italic>p</italic> alleles. In this paper, the term variant is used for any type of SNP (biallelic, triallelic, etc), two-digit HLA allele, four-digit HLA allele and amino acid position. In any case we included <italic>p-1</italic> alleles, with the one excluded being the reference allele. Where possible we tried to select the most frequent variant in the controls as the reference allele. The five included principal components are represented in the model as <italic>l</italic> and the last block in the model represents the dummy variables included for the <italic>n</italic> studies (<italic>n-1</italic> parameters added in the model).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To calculate an omnibus p-value for the variant, regardless of the number of alleles included in the univariate model, we used using a log-likelihood ratio test (2) comparing the likelihood <italic>L<sub>0</sub></italic> of the null model (3) against the likelihood <italic>L<sub>1</sub></italic> of the fitted model:<disp-formula id="pgen.1003926.e002"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003926.e002.jpg" /><label>(2)</label></disp-formula>Log-likelihood ratio test</p><p xmlns:ns0="http://www.w3.org/1999/xlink">where <italic>D</italic> is the log-likelihood test value, also known as deviance. <italic>D</italic> follows an approximate chi-square distribution with <italic>k</italic> degrees of freedom, where <italic>k</italic> is the difference of the regressed parameters between the two models. Representation of the null model:<disp-formula id="pgen.1003926.e003"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003926.e003.jpg" /><label>(3)</label></disp-formula>
<italic>M<sub>0</sub></italic>, Null logistic regression model</p><p>Besides testing variants for association, i.e. SNPs, HLA alleles and amino acids, we also fitted models that estimated the overall effect of the each of the eight HLA genes. We did so, by fitting all respective four-digit alleles of a given HLA gene in the same model. The respective p-values reflect the overall significance of the gene.</p><p>In order to identify the statistically independent effects, we first tested all variants under a univariate logistic regression model and ranked them based on the p-value of the log-likelihood test. Next, in a forward stepwise fashion, we included in the logistic regression model the most statistically significant variant as a covariate, analyzed all remaining variants and ranked them based on the new p-value of the respective log-likelihood test. The models that included at least one variant as covariate are referred to as <italic>conditional</italic> throughout the text. In each iteration the null model used in the log-likelihood test was the original null model (3) with the variants that were used as covariates. We repeated the same steps until no variant or no HLA gene reached the level of statistical significance, which we <italic>a priori</italic> set to be 10<sup>&#8722;5</sup>. This statistical significance threshold accounts of 5,000 independent tests using Bonferroni correction. Although most of the variants analyzed are correlated, we chose this threshold to account also for the multiple stepwise fitted models. If no variant reached the level of significance but an HLA gene did, we kept adding variants in the overall model until the HLA gene p-value was larger than 10<sup>&#8722;5</sup>.</p><p>To compare the effects of two (or more variants), e.g. A and B, we fitted the following models: <italic>M<sub>A</sub></italic> model with variant A, <italic>M<sub>B</sub></italic> model with variant B, and <italic>M<sub>AB</sub></italic> model with both variants A and B. All three model included the same other covariates. Then we used the log-likelihood test to compare <italic>M<sub>AB</sub></italic> vs. <italic>M<sub>B</sub></italic> and <italic>M<sub>AB</sub></italic> vs. <italic>M<sub>A</sub></italic>. These two comparisons represent the effects of variants A and B, respectively, in the presence of the other variant, i.e. B and A. For these comparisons we used the nominal (&#945;&#8202;=&#8202;0.05) level of statistical significance.</p><p>After adjusting for the most statistically significant variant, <italic>DRB1*15:01</italic>, the residual effect of the DRB1 locus, i.e. the effect of all alleles besides <italic>*15:01</italic>, was still the most statistically significant of any of the remaining variants. This led us to the hypothesis that several other DRB1 alleles could explain the overall DRB1 locus effect, already conditioning on <italic>DRB1*15:01.</italic> To identify such effects inside the DRB1 locus, we applied the above forward stepwise logistic regression approach to the four-digit DRB1 alleles. , To test the robustness of the results from the forward stepwise regression, we also applied four other statistical methods for variant selection: i) lasso, <xref ref-type="bibr" rid="pgen.1003926-Tibshirani1">[43]</xref> ii) elastic net, <xref ref-type="bibr" rid="pgen.1003926-Zou1">[44]</xref> iii) least angle regression, <xref ref-type="bibr" rid="pgen.1003926-Efron1">[45]</xref> and iv) forward Stagewise regression. <xref ref-type="bibr" rid="pgen.1003926-Hastie1">[46]</xref> For the lasso and elastic net we selected the largest value of lambda (<italic>l<sub>1</sub></italic>) after 10-fold cross-validation, such that error was within 1 standard error of the minimum mean cross-validated error. In the respective results section, we illustrate that all methods reached the same conclusion independently.</p><p>It has been proposed that extended <italic>DRB1*15:01&#8211;DQB1*06:02</italic> haplotypes confer the risk for MS rather than individual HLA alleles. To test this hypothesis, we used the post-imputation phased data to estimate the <italic>DRB1*15:01&#8211;DQB1*06:02</italic> diplotypes. Then we fitted a logistic regression that estimated the effect of the diplotype under a per-allelic model. Since this approach used phased data, rather than post-imputation probabilities, the imputation uncertainty is not properly accounted for. Thus, we expect the respective p-values to be slightly inflated.</p><p>To investigate the functional potential of the <italic>MICB-LST1</italic> region we queried:</p><p>in-house cis-eQTL (expression quantitative trait loci) in PBMCs of 213 MS subjects <xref ref-type="bibr" rid="pgen.1003926-Raj1">[11]</xref> and CD4+ T cells and CD14+ monocytes of 211 healthy individuals. The PBMCs gene expression levels were quantified with mRNA derived from of 213 subjects of European ancestry with relapsing remitting (RR) multiple sclerosis (MS) via an Affymetrix Human Genome U133 Plus 2.0 Array. The expression levels were adjusted for confounding factors, such as subject's use of immunomodulatory drugs, age, gender, and batch effects via principle components analysis. Associations between SNP genotypes and adjusted expression residual traits were conducted by Spearman rank correlation (SRC). For the <italic>cis</italic> analysis, we considered only SNPs within a 2 Mbps window from the transcript start site (TSS) of genes. Furthermore, we also explored any possible effect to the expression of class I and II HLA classical genes, regardless of their physical distance. Significance of the nominal p-values was determined by comparing the distribution of the most significant p values generated by permuting expression phenotypes 10,000 times independently for each gene. We call a <italic>cis</italic>-eQTL significant if the nominal association p-value is greater than the 0.05 tail of the minimal p-value distribution resulting from the permuted associations, which corresponds to a p-value of 1&#215;10<sup>&#8722;05</sup>. Similar methods were used to evaluate the <italic>cis</italic>-regulatory effects in CD4+ T lymphocytes and CD14+ monocytes data sets consisting of 211 healthy individuals of European ancestry. These analyses were conducted under the auspices of a protocol approved by the institutional review board of Partners Healthcare.</p><p>publicly available data from the ENOCDE <xref ref-type="bibr" rid="pgen.1003926-Bernstein1">[12]</xref> and NIH Epigenomics Roadmap <xref ref-type="bibr" rid="pgen.1003926-Roadmap1">[13]</xref>. Specifically we perused data for functional potential from CD4 memory primary cells (H3K4me3, H3K27ac, H3K36me3, chromatin states), CD4 na&#239;ve primary cells (H3K4me3, H3K27ac, H3K36me3, chromatin states), CD8 memory primary cells (H3K4me3, H3K27ac, H3K36me3, chromatin states), CD8 na&#239;ve primary cells (H3K4me3, H3K27ac, H3K36me3, chromatin states), CD4 primary cells (DNase hypersensitivity sites), CD8 primary cells (DNase hypersensitivity sites), Treg primary cells (H3K4me3, H3K36me3, DNase hypersensitivity sites), Th1 (DNase hypersensitivity sites), Th2 (DNase hypersensitivity sites), Th17 (DNase hypersensitivity sites) and GM12878 cell line (B-lymphocyte, lymphoblastoid, International HapMap Project - CEPH/Utah - European Caucasion, Epstein-Barr Virus; H3K4me3, H3K27ac, H3K36me3, chromatin states, DNase hypersensitivity sites).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used Nagelkerke's pseudo-R <xref ref-type="bibr" rid="pgen.1003926-Nagelkerke1">[47]</xref> to estimate the variance explained<disp-formula id="pgen.1003926.e004"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003926.e004.jpg" /><label>(4)</label></disp-formula>Nagelkerke's pseudo-R<sup>2</sup>
</p><p>where <italic>L<sub>0</sub></italic> and <italic>L<sub>1</sub></italic> are the likelihoods of the null model and fitted model respectively, and <italic>N</italic> is the number of individuals.</p><p>We used PLINK for the initial analysis of the data and to estimate minor allele frequencies and imputation quality metrics, i.e. INFO score. <xref ref-type="bibr" rid="pgen.1003926-Purcell1">[48]</xref> We fitted all models in R using the glm function and package lars and glmnet.</p><p>This investigation has been approved by the Institutional Review Board of Partners Healthcare; the reference number is 2002p000434.</p>