<p>Samples were collected from nine sympatric population pairs of <italic>mexicana</italic> and maize that spanned the known distribution of <italic>mexicana</italic> in Mexico, as well as a single allopatric population of <italic>mexicana</italic> (<xref ref-type="supplementary-material" rid="pgen.1003477.s010">Table S1</xref>; <xref ref-type="fig" rid="pgen-1003477-g001">Figure 1</xref>). Seed samples from 12 maternal individuals per <italic>mexicana</italic> population (N&#8202;=&#8202;120) were selected for genotyping. A single kernel was also sampled from each of 6&#8211;8 maize ears collected from sympatric maize fields (N&#8202;=&#8202;69). The tenth kernel down from the tip of each ear was chosen to help control for potential variation in outcrossing rate along the ear. Seeds were treated with fungicide, germinated on filter paper and grown in standard potting mix to the five-leaf stage. Freshly harvested leaf tips were stored at &#8722;80&#176;C overnight and lyophilized for 48 hours. Tissue was then homogenized with a Mini-Beadbeater-8 (BioSpec Products, Inc., Bartlesville, OK, USA) and DNA was isolated using a modified CTAB protocol <xref ref-type="bibr" rid="pgen.1003477-SaghaiMaroof1">[93]</xref>. Purity of DNA isolations was determined with a NanoDrop spectrophotometer (NanoDrop Technologies, Inc., Wilmington, DE, USA). Samples with 260&#8758;280 ratios &#8805;1.8 were deemed acceptable for genotyping. Concentrations of DNA isolations were determined with a Wallac VICTOR2 fluorescence plate reader (Perkin-Elmer Life and Analytical Sciences, Torrance, CA, USA) using the Quant-iT Picogreen dsDNA Assay Kit (Invitrogen, Grand Island, NY, USA). Single nucleotide polymorphism genotypes were generated using the Illumina MaizeSNP50 Genotyping BeadChip platform and were clustered separately for the two taxa based on the default algorithm of the GenomeStudio Genotyping Module v1.0 (Illumina Inc., San Diego, CA, USA). Clustering for each SNP in each taxon was subsequently inspected and manually adjusted. Of the total of 56,110 markers contained on the chip, 39,029 SNPs that were polymorphic within the entire sample of maize and <italic>mexicana</italic> and contained less than 10% missing data in both taxa were used for further analysis.</p><p>Observed (<italic>H<sub>O</sub></italic>) and expected (<italic>H<sub>E</sub></italic>) heterozygosities were summarized for each taxon in each sympatric population pair using the &#8220;genetics&#8221; package in R <xref ref-type="bibr" rid="pgen.1003477-R1">[94]</xref>. Polymorphisms were further characterized as shared, fixed, or segregating privately within one of each pair of sympatric populations using the sharedPoly program of the libsequence C++ library <xref ref-type="bibr" rid="pgen.1003477-Thornton1">[95]</xref>. Pairwise differentiation between populations (<italic>F<sub>ST</sub></italic>) was calculated based on the method of Weir and Cockerham <xref ref-type="bibr" rid="pgen.1003477-Weir1">[96]</xref> using custom R scripts and the &#8220;hierfstat&#8221; package of R <xref ref-type="bibr" rid="pgen.1003477-Goudet1">[97]</xref>. The <italic>f3</italic> statistic for identification of admixture <xref ref-type="bibr" rid="pgen.1003477-Reich2">[67]</xref> was calculated using a custom R script.</p><p>To characterize patterns of introgression across the genome in each population we used two complementary methods: 1) Identification of ancestry across chromosomal segments with the hidden Markov model approach of HAPMIX <xref ref-type="bibr" rid="pgen.1003477-Price1">[68]</xref>; and 2) A site-by-site analysis of assignment probabilities using the Bayesian linkage model in the program STRUCTURE <xref ref-type="bibr" rid="pgen.1003477-Falush1">[69]</xref>, <xref ref-type="bibr" rid="pgen.1003477-Pritchard1">[70]</xref>. For both HAPMIX and STRUCTURE analyses, we used a subset of 38,262 SNPs anchored in a genetic map based on the Intermated B73&#215;Mo17 (IBM) population of maize (<xref ref-type="bibr" rid="pgen.1003477-Ganal1">[66]</xref>; J.P. Gerke et al., unpublished data). The IBM population has been widely used for genetic map development and for determining the genetic architecture of complex traits in maize <xref ref-type="bibr" rid="pgen.1003477-Lee1">[98]</xref>.</p><p>Patterns of introgression were assessed using the program HAPMIX by comparing unphased data from putatively admixed individuals from our sympatric populations to phased data from reference ancestral populations. To represent ancestral <italic>mexicana</italic> haplotypes, we chose a population near the town of Amatl&#225;n, Morelos state, Mexico that is currently allopatric to maize. An Americas-wide sample of maize landraces collected largely outside the distribution of <italic>mexicana</italic> was chosen as the maize reference population <xref ref-type="bibr" rid="pgen.1003477-Chia1">[65]</xref>. In order to assess putative introgression and/or false positives in these reference populations, we removed each individual and evaluated introgression through comparison to remaining reference samples using a jackknife approach. Evidence for introgression was assessed in both putatively admixed and reference individuals using HAPMIX as described below.</p><p>Initial estimates of ancestry proportions for HAPMIX models were based on a previous admixture analysis of <italic>mexicana</italic> and highland Mexican maize (&#8764;20% introgression of <italic>mexicana</italic> into maize and &#8764;10% introgression of maize into <italic>mexicana</italic>; <xref ref-type="bibr" rid="pgen.1003477-vanHeerwaarden1">[36]</xref>). The number of generations since the time of admixture was varied from 1&#8211;5000 and the maximum likelihood across individuals in a population was used to compare relative time since admixture on a population-by-population basis (<xref ref-type="supplementary-material" rid="pgen.1003477.s004">Figure S4</xref>). Subsequent analyses of HAPMIX output were based on introgression estimates from the highest likelihood run.</p><p>Prior to analysis in STRUCTURE, SNP data were phased using the program fastPHASE (version 1.4.0; <xref ref-type="bibr" rid="pgen.1003477-Scheet1">[99]</xref>). Because STRUCTURE does not account for linkage disequilibrium (LD) due to physical linkage, SNPs were grouped into haplotypes separated by at least 5 kb. After grouping, our data set consisted of 20,035 loci with an average of 3.92 alleles per locus across all sympatric and reference allopatric individuals. We ran the linkage model in STRUCTURE with 5,000 steps of admixture burn-in, a total burn-in of 10,000 steps, and 100,000 subsequent steps retained for analysis. Convergence along the chain and consistency across replicate runs were assessed to ensure an adequate number of steps were included in the analysis. Assignment was carried out for K&#8202;=&#8202;2 groups (<italic>i.e.</italic>, maize and <italic>mexicana</italic>) for each chromosome separately. Probability of assignment was summarized locus by locus across individuals from each population for each taxon.</p><p>To identify SNPs associated with environmental variables, we employed the association method of BAYENV <xref ref-type="bibr" rid="pgen.1003477-Coop1">[77]</xref>, using a covariance matrix of allele frequencies estimated with 10,000 random SNPs to control for population structure. Seventy-six climatic and soil variables were summarized as four principal components that captured 95% of the variance among <italic>mexicana</italic> populations. BAYENV was run five times with 1,000,000 iterations for each SNP. A given SNP was considered a candidate if its Bayes factor was consistently in the 95<sup>th</sup> percentile across all five independent runs and its average Bayes factor was in the 99<sup>th</sup> percentile. Enrichment of significant SNPs in introgressed regions was determined based on bootstrap resampling for each environmental PC.</p><p>Analyses of haplotype sharing/identity by state between <italic>mexicana</italic>, <italic>parviglumis</italic>, and modern maize lines were conducted using the program GERMLINE <xref ref-type="bibr" rid="pgen.1003477-Gusev1">[82]</xref> with haplotypes generated by the program fastPHASE <xref ref-type="bibr" rid="pgen.1003477-Scheet1">[99]</xref> from samples of <italic>parviglumis</italic>
<xref ref-type="bibr" rid="pgen.1003477-Pyhjrvi1">[78]</xref> and modern maize <xref ref-type="bibr" rid="pgen.1003477-Cook1">[80]</xref>. Shared haplotypes were identified with a seed of identical genotypes at five SNPs that were extended until mismatch. Analyses were then based on segments with a minimum size of 3 cM.</p><p>Ten seeds were germinated from each of four maize populations showing little evidence of introgression (Ixtlan and Puruandiro) or fixed introgression (Opopeo and San Pedro) at two loci (one on chromosome 4 and one on chromosome 9; <xref ref-type="supplementary-material" rid="pgen.1003477.s013">Table S4</xref>) putatively linked to highland adaptation <xref ref-type="bibr" rid="pgen.1003477-Lauter1">[42]</xref>, <xref ref-type="bibr" rid="pgen.1003477-Pyhjrvi1">[78]</xref> and showing little evidence of false positives in our reference populations. Plants were grown under highland conditions with 12.5 hours of light at an intensity of 680 &#181;mol/m<sup>2</sup>*s, a daytime temperature of 23&#176;C and a nighttime temperature of 11&#176;C. Daytime relative humidity was set at 60% and nighttime relative humidity at 80%. Height measurements were taken at 15, 30, and 50 days. Pigment extent was measured on the second leaf sheath from the top of the plant as the proportion of the total sheath showing pigment. Macrohairs were also measured on this leaf sheath as the total count one third of the way down from the leaf blade within the field of a dissecting microscope at 2&#215; magnification. In order to contrast plant height from our highland treatment to those under conditions more comparable to the lowlands of western Mexico, we conducted a separate growth chamber experiment with a daytime temperature of 32&#176;C and a nighttime temperature of 25&#176;C and measured plant height at 30 days. All other conditions were identical to those of the highland treatment.</p>