<p>Colonies of <italic>G. morsitans morsitans</italic> were reared at Yale University and the Institute of Zoology at the Slovak Academy of Sciences (SAS). The other two species (<italic>G. pallidipes</italic> and <italic>G. fuscipes</italic>) were reared at SAS. Flies were maintained on blood meals provided through an artificial feeding system at 48 h intervals <xref ref-type="bibr" rid="pgen.1003874-Moloo1">[104]</xref>. Two groups of females were used for transcriptome analysis: the first group carried a third instar larva (lactating) while the second group was examined 48 h post parturition (dry or non-lactating). Developing progeny were removed from each female to ensure transcript changes were representative of differences between the mothers. For sex specific transcript analysis, males and females were collected 16&#8211;18 d after adult emergence. Tissue samples were collected from pregnant females (16&#8211;18 d after adult emergence) carrying third instar larvae 24 h after blood feeding. Samples for temporal expression analyses were collected according to progeny development status based on previous studies <xref ref-type="bibr" rid="pgen.1003874-Benoit1">[9]</xref>, <xref ref-type="bibr" rid="pgen.1003874-Attardo3">[47]</xref>. Flesh flies, <italic>S. crassipalpis</italic>, acquired from Ohio State University were reared according to standard procedures <xref ref-type="bibr" rid="pgen.1003874-Denlinger5">[105]</xref>.</p><p>Total RNA was extracted from individual flies or dissected tissues using Trizol reagent (Invitrogen, Carlsbad, CA, USA), following the recommended protocol. RNA was treated twice with the TURBO DNA Free kit (Ambion, Austin, TX, USA) to remove DNA, alcohol precipitated to remove residual salt, and further cleaned using the RNeasy kit (Qiagen, Maryland, USA). Total RNA (2&#8211;3 &#181;g) was pooled from 10 flies extracted individually for each treatment. Sample quality and concentration was determined using a Bioanalyzer 2100 (Agilent, Palo Alto, CA, USA). Library construction was performed using standard protocols for Illumina mRNA-Seq sequencing by the W. M. Keck Foundation Microarray Resource at the Yale School of Medicine. Each single-end library was sequenced on one lane of the Genome Analyzer II platform (Illumina, San Diego, CA, USA).</p><p>To determine Illumina read quality, FastQC analysis was performed on the transcriptomes generated from dry and lactating flies. Due to the prevalence of tsetse symbiont sequences in the reads, a specific quality control step was included to reduce bacterial sequence reads using the known whole genome sequence data from <italic>Wigglesworthia</italic>
<xref ref-type="bibr" rid="pgen.1003874-Rio1">[106]</xref>, <italic>Wolbachia</italic> (unpublished) and <italic>Sodalis</italic>
<xref ref-type="bibr" rid="pgen.1003874-Toh1">[107]</xref> determined from the same host species <italic>G. morsitans</italic>. Following symbiont specific sequence removal, remaining sequences were trimmed in CLC Genomics (CLC Bio) to remove ambiguous nucleotides. Contig libraries were constructed using Abyss <xref ref-type="bibr" rid="pgen.1003874-Birol1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1003874-Robertson1">[17]</xref> followed by a secondary assembly with Trinity <xref ref-type="bibr" rid="pgen.1003874-Grabherr1">[18]</xref>. Functional annotation was accomplished using the BLASTx algorithm through comparison with sequences included in the NCBI protein database <xref ref-type="bibr" rid="pgen.1003874-Altschul1">[108]</xref> as well as the KOG <xref ref-type="bibr" rid="pgen.1003874-Tatusov1">[109]</xref> and GO databases <xref ref-type="bibr" rid="pgen.1003874-Ashburner1">[110]</xref>. Conserved protein domains were detected using rpsBLAST <xref ref-type="bibr" rid="pgen.1003874-Schaffer1">[111]</xref> searches against the CDD, Pfam and Smart databases <xref ref-type="bibr" rid="pgen.1003874-Bateman1">[112]</xref>. Predicted protein translations were submitted to SignalP to identify potential secretion products by screening for secretion signal motifs <xref ref-type="bibr" rid="pgen.1003874-Nielsen1">[113]</xref>. Additionally, contigs were compared to several proteomes obtained from Flybase <xref ref-type="bibr" rid="pgen.1003874-Tweedie1">[114]</xref> (<italic>D. melanogaster</italic>) and Vectorbase <xref ref-type="bibr" rid="pgen.1003874-Lawson1">[115]</xref> (<italic>An. gambiae</italic>). Each read from each library was compared by BLASTn to the assembled coding sequences (CDS) using a word size of 25, m8 output and low complexity filter turned off. CDS coverage and CDS number of read &#8220;hits&#8221; from each library were computed from the BLAST output file. A hit was only considered significant if it had 97% or better identity to its target and no more than one gap. The same read could be mapped up to three different CDS to the extent that their BLAST scores were identical. Expression levels were determined using CLC Genomics Workbench (CLC bio, Cambridge, MA). Reads were mapped to our <italic>de novo</italic> assembly with an algorithm allowing only two mismatches and a maximum of 10 hits per read. RPKM was used as a measure of gene expression <xref ref-type="bibr" rid="pgen.1003874-Mortazavi1">[116]</xref>. The proportion of read counts for each contig in relation to the total read counts in each sample was determined in order to calculate P-value differences in proportions by a Z-test following Bonferroni correction <xref ref-type="bibr" rid="pgen.1003874-Kal1">[117]</xref>. Fold change was determined as the ratio of RPKM of lactating flies vs. RPKM of dry flies. In addition to the analysis of the complete Illumina libraries, a secondary analysis was conducted featuring Illuminia libraries filtered to eliminate milk-specific contigs to reduce bias by these highly abundant proteins <xref ref-type="bibr" rid="pgen.1003874-Mortazavi1">[116]</xref>, <xref ref-type="bibr" rid="pgen.1003874-Kal1">[117]</xref>. Data from this study are available in Sequence Read Archive (SRA075330).</p><p>Pulled glass capillary tubes were used to collect milk samples by negative pressure from the guts of feeding third instar larvae, which were microscopically dissected from the uterus of pregnant females. Samples were stored in 1&#215; protease inhibitor cocktail (Sigma-Aldrich). Proteins were precipitated with 10% trichloroacetic acid (Fisher Scientific) at 4&#176;C overnight, collected by centrifugation (11,000&#215;g, 30 minutes, 4&#176;C) and washed two times with ice-cold acetone. Protein pellets were briefly dried and dissolved in 10 &#181;l of protein pellet buffer (8M urea, 3M thiourea, and 1% dithiothreitol). Trypsin digestion was performed at 37&#176;C for 12&#8211;16 h following dilution with distilled H<sub>2</sub>O to a final volume of 100 &#181;l. Samples were stored at &#8722;80&#176;C until analysis. Peptides were separated with a Waters nanoAcquity UPLC system (75 &#181;m&#215;150 mm BEH C18 eluted at 500 nl/min at 35&#176;C) with Buffer A (100% water, 0.1% formic acid) and Buffer B (100% CH<sub>2</sub>CN, 0.075% formic acid). A linear gradient was established with 5% Buffer B, increasing to 50% Buffer B at 50 minutes and finally to 85% Buffer B at 51 minutes. MS/MS was acquired with an AB Sciex 5600 Triple Time-of-Flight mass spectrometer using 1 microscan followed by four MS/MS acquisitions. Neutral loss scans were obtained for 98.0, 49.0 and 32.7 amu. Seven separate 1 &#181;l injections at an estimated 0.351 &#181;g/&#181;l concentration for a total of 2.457 &#181;g on the column were used for analysis.</p><p>Mascot algorithm was used to analyze uninterrupted MS/MS spectra <xref ref-type="bibr" rid="pgen.1003874-Hirosawa1">[118]</xref>. The Mascot Distiller program used MS/MS spectra to generate Mascot compatible files by combining sequential MS/MS scans from profile data that have the same precursor ion. Charge states of +2 and +3 were preferentially located with a signal-to-noise ratio of 1.2 or greater. A list of protein sequences was created and used in the BLASTx search against Trinity-assembled library from the pregnancy-specific analysis and positive matches were identified by tBLASTx against the NCBI and Swiss-Prot databases. Mascot scores were based on MOlecular Weight SEarch (MOWSE) relying on multiple matches of more than one peptide to the same predicted protein <xref ref-type="bibr" rid="pgen.1003874-Pappin1">[119]</xref>, <xref ref-type="bibr" rid="pgen.1003874-Perkins1">[120]</xref>. The MOWSE based ions score is equal to (&#8722;10)*(Log<sub>10</sub>P), where P is the absolute probability that a match is random. Matches were considered significant when the probability of a random match fell below 5% (E value&lt;0.05). Therefore, Mascot scores greater than 68 were above the significance threshold when searching the newly assembled library. Proteins were considered to be successfully identified when two or more peptides matched the same predicted protein and the Mascot score exceeded the significance threshold. The exponentially modified protein abundance index (empai) was employed to estimate levels of protein species based on the number of species detected compared to the number of possible peptides for specific protein <xref ref-type="bibr" rid="pgen.1003874-Ishihama1">[121]</xref>, .</p><p>Chromosomal organization of genes and full length mRNA sequences for <italic>mgp2&#8211;10</italic> were obtained by mapping Illumina high-throughput reads against <italic>G. m. morsitans</italic> genomic scaffolds in the CLC Genomics software package. Nucleotide and predicted protein sequences were aligned using PROMALS3D <xref ref-type="bibr" rid="pgen.1003874-Pei1">[123]</xref> and Clustal <xref ref-type="bibr" rid="pgen.1003874-Larkin1">[124]</xref> and formatted with BioEdit <xref ref-type="bibr" rid="pgen.1003874-Hall1">[24]</xref>. Flesh fly, <italic>Sarcophaga crassipalpis</italic>, sequences were obtained from a previous EST project <xref ref-type="bibr" rid="pgen.1003874-Ragland1">[20]</xref>, <xref ref-type="bibr" rid="pgen.1003874-Hahn1">[21]</xref>. Sequences of <italic>mgp2&#8211;10</italic> from other tsetse species (<italic>G. pallidipes</italic> and <italic>G. fuscipes</italic>) were obtained from female cDNA by RT-PCR followed by cloning into T-vector plasmid (Invitrogen) and sequenced at the DNA Analysis Facility at Yale University (New Haven, CT). Pairwise phylogenetic tree construction and bootstrap analysis (10000 replicates) were performed using the MEGA4/5 sequence analysis suite <xref ref-type="bibr" rid="pgen.1003874-Tamura1">[25]</xref>, <xref ref-type="bibr" rid="pgen.1003874-Kumar2">[125]</xref>. dN/dS analyses were performed using the FEL (Fixed Effects Likelihood <xref ref-type="bibr" rid="pgen.1003874-Pond1">[28]</xref>) and PARRIS <xref ref-type="bibr" rid="pgen.1003874-Scheffler1">[27]</xref> algorithms available via DataMonkey <xref ref-type="bibr" rid="pgen.1003874-Delport1">[29]</xref>, <xref ref-type="bibr" rid="pgen.1003874-Pond2">[30]</xref>, which is a web-based implementation of the HyPhy phylogenetic analysis program <xref ref-type="bibr" rid="pgen.1003874-Pond3">[31]</xref>. Sequences were translated, aligned, reverse translated and the stop codons removed in accordance with the requirements for sequence input to DataMonkey. Under the FEL method, posterior probabilities cutoffs were set at 95, which is equivalent to a p-value of 0.05 for the site-specific detection of codons under positive or negative selection. Analysis of specific regions of the MGP2&#8211;10 coding regions was conducted using MEGA5 according to previous milk protein studies <xref ref-type="bibr" rid="pgen.1003874-Pharo1">[32]</xref> and individual regions were based upon protein coding regions with high or low levels of amino acid homology.</p><p>For sex- and tissue-specific RT-PCR expression analyses, total RNA isolated from males and females and from dissected tissues was used as template for the Superscript III reverse transcriptase kit following the manufacturer's protocols (Invitrogen). Fat body and milk gland were analyzed as a combined samples since complete separation is nearly impossible due to the intricate association of these organs. PCR was performed with gene-specific primer pairs (<xref ref-type="supplementary-material" rid="pgen.1003874.s009">Table S1</xref>) using the GoTaq DNA polymerase kit (Promega). The PCR amplification conditions were as follows: 95&#176;C for 3 min, 35 cycles of 30 sec at 95&#176;C, 52&#8211;56&#176;C for 1 min, and 1 min at 70&#176;C using a Bio-Rad DNA Engine Peltier Thermocycler (Hercules, CA).</p><p>For pregnancy-specific transcript abundance determination, qPCR analyses were performed using a CFX PCR detection system (Bio-Rad, Hercules). Data were analyzed with CFX manager software version 3.1 (Bio-Rad). Primer sequences used were the same as used in RT-PCR analyses (<xref ref-type="supplementary-material" rid="pgen.1003874.s009">Table S1</xref>). Comparative Ct values for genes of interest were standardized by Ct values for the control gene (<italic>tubulin</italic>) relative to the average value for the control treatment or newly emerged flies, yielding the delta Ct value. All experiments were analyzed in triplicate and subject to ANOVA followed by Bonferroni correction and Dunnett's test.</p><p>Short interfering RNAs (siRNA) consisting of two Duplex sequences (<xref ref-type="supplementary-material" rid="pgen.1003874.s009">Table S1</xref>) were designed using Integrated DNA Technologies online software (IDT). Control siRNAs were designed against Green Fluorescent Protein (GFP; <xref ref-type="supplementary-material" rid="pgen.1003874.s009">Table S1</xref>). Each oligo, designed to target a single <italic>mgp</italic> gene, was also compared to the reference RNA library/<italic>G. morsitans</italic> genome <xref ref-type="bibr" rid="pgen.1003874-Baumann1">[14]</xref>) and the Trinity contigs library from this study to verify target specificity. The oligos for each strand of the siRNA were combined, and the concentration was determined spectrophotometrically followed by adjustment to 800&#8211;850 ng/&#181;l per siRNA. Each female fly was injected with 2 &#181;l siRNA solution into the thorax 8&#8211;10 d after adult emergence. Five days post-injection, gene expression levels were determined by qPCR and normalized to <italic>tubulin</italic> transcripts. For combined knockdown studies, siMGP constructs were mixed to yield a sample concentration of at least 600 ng/&#181;l for each siRNA targeting a specific MGP transcript. Fecundity following MGP knockdown was assessed as previously described <xref ref-type="bibr" rid="pgen.1003874-Benoit1">[9]</xref>. Finally, milk protein content was determined by Bradford assay (Bio-Rad) after extraction from the larval gut contents as described above.</p><p>Emulsification assays were based on milk turbidity measurements. For this assay, milk was acquired from the guts of actively feeding larvae as before and diluted 10&#215; prior to the assay. Samples were vortexed for 1 min at 10,000 rpm, and absorbance of the diluted emulsion was measured at 500 nm. Changes in absorbance were measured hourly for 10 h. Results were analyzed based on the slope of a regression, where ln (ABS<sub>t</sub>/ABS<sub>0</sub>) is plotted versus time based on the exponential model (ABS<sub>t</sub>&#8202;=&#8202;ABS<sub>0</sub> e<sup>&#8722;kt</sup>). For this model, ABS<sub>t</sub> denotes absorbance at any time t, ABS<sub>0</sub> is the initial absorbance, and k is the rate of absorbance decline in %/h.</p><p>To generate structural models for MGP2&#8211;10, four web-based <italic>de novo</italic> protein modeling programs were consulted. QUARK is a recently developed <italic>ab initio</italic> assembly program that will first break proteins into small sequences, following which full-length sequence models are assembled using Monte Carlo simulations <xref ref-type="bibr" rid="pgen.1003874-Xu1">[126]</xref>. The I-TASSER program first develops a three-dimensional model and subsequently predicts function based on structural similarity with functionally defined proteins <xref ref-type="bibr" rid="pgen.1003874-Roy1">[127]</xref>. Phyre2 is a widely used protein homology/analogy recognition engine that can rapidly predict the structure of 250 residue proteins <xref ref-type="bibr" rid="pgen.1003874-Kelley1">[128]</xref>. Finally, SPARKS-X is a program that performs well in comparison to other programs <xref ref-type="bibr" rid="pgen.1003874-Yang2">[129]</xref>. Each program was run under the default configuration and the resultant predicted protein structures were visualized using Discovery Studio 3.1 (Accelrys).</p>