<p>Seven strains were selected for genomic sequencing based upon their characterized and distinctive biological control properties and their isolation from different habitats (bulk or rhizosphere soil or aerial plant surfaces) (<xref ref-type="table" rid="pgen-1002784-t001">Table 1</xref>). The seven strains and three previously-sequenced strains (Pf-5, Pf0-1, and SBW25) evaluated in this study exhibited the conserved phenotypes of the P. <italic>fluorescens</italic> group: positive for fluorescence under UV light, arginine dihydrolase activity, and oxidase activity; and negative for growth at 41&#176;C and induction of a hypersensitive response on tobacco, determined through standard methods <xref ref-type="bibr" rid="pgen.1002784-Gerhardt1">[138]</xref> (<xref ref-type="supplementary-material" rid="pgen.1002784.s026">Table S16</xref>). The ten strains were subjected to a panel of biochemical and biological assays (nitrate reduction, levan sucrase production, potato soft rot, gelatinase activity, and catabolic spectra) <xref ref-type="bibr" rid="pgen.1002784-Gerhardt1">[138]</xref>, <xref ref-type="bibr" rid="pgen.1002784-Schaad1">[139]</xref> to assign each to a biovar of <italic>P. fluorescens</italic> or to a species of <italic>Pseudomonas</italic>
<xref ref-type="bibr" rid="pgen.1002784-Stanier1">[129]</xref> (<xref ref-type="table" rid="pgen-1002784-t001">Table 1</xref> and <xref ref-type="supplementary-material" rid="pgen.1002784.s026">Table S16</xref>). Strains A506, 30-84, SS101, and BG33R are rifampicin-resistant (100 &#181;g/ml) derivatives of field isolates; previously, spontaneous mutants with resistance to rifampicin were selected to facilitate tracking of these strains in field studies. Strain A506 is known to have a single nucleotide insertion in <italic>rpoS</italic>, which causes a frameshift resulting in a truncated form of the stationary-phase sigma factor RpoS <xref ref-type="bibr" rid="pgen.1002784-Hagen1">[140]</xref>. During the course of this work, we discovered that strain Pf0-1 has a mutation in <italic>gacA</italic>, which encodes a component of the GacA/GacS global regulatory system in <italic>Pseudomonas</italic> spp. <xref ref-type="bibr" rid="pgen.1002784-Lapouge1">[61]</xref>. We sequenced <italic>gacA</italic> and <italic>gacS</italic> from the strain Pf0-1 in our collection and confirmed that the sequences are identical to those in the published genome of Pf0-1 <xref ref-type="bibr" rid="pgen.1002784-Silby2">[32]</xref>. It is not possible to know whether the mutations in A506 and Pf0-1 were present in the strains prior to isolation or if they developed in the laboratory during storage, but all strains have been maintained as frozen stocks (&#8722;80&#176;C) throughout this study and for many years preceding.</p><p>The genome sequences were determined using shotgun sequencing with a combination of Sanger sequencing (to 4&#215; coverage of the genome size) and 454 pyrosequencing technologies with paired end reads <xref ref-type="bibr" rid="pgen.1002784-Fraser1">[141]</xref>, <xref ref-type="bibr" rid="pgen.1002784-Goldberg1">[142]</xref>. A hybrid genome assembly was prepared from these datasets using Newbler 2.3 (Roche) and Celera Assembler 5.42. Each genome was subsequently evaluated for additional assembly improvement with the Celera Assembler 5.42 assembly versions providing the starting points. Multiple gaps were closed by merging overlapping contigs and resolving repetitive gaps. Further physical and sequencing gaps were closed by sequencing of PCR products spanning the gaps. The order of scaffolds in the genomes of strains 30-84 and Q8r1-96 (<xref ref-type="fig" rid="pgen-1002784-g003">Figure 3</xref>) was confirmed by PCR.</p><p>The whole genome shotgun sequencing projects have been deposited at DDBJ/EMBL/GenBank under the accessions AHHJ00000000 (30-84), AHOT00000000 (O6), AGBM00000000 (Q2-87), AHPO00000000 (Q8r1-96), AHPP00000000 (BG33R) and AHPN00000000 (SS101). Accession numbers for the complete genome sequences are: CP003041 for the chromosome of A506 and CP000076 for the updated genome sequence of Pf-5.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Identification of putative protein-encoding genes and annotation of the genomes were performed as previously described <xref ref-type="bibr" rid="pgen.1002784-Paulsen2">[143]</xref>. A set of open reading frames predicted to encode proteins was initially identified using GLIMMER <xref ref-type="bibr" rid="pgen.1002784-Salzberg1">[144]</xref>. Open reading frames consisting of fewer than 30 codons and those containing overlaps were eliminated. Functional assignment, identification of membrane-spanning domains, determination of paralogous gene families, and identification of regions of unusual trinucleotide composition were performed as previously described <xref ref-type="bibr" rid="pgen.1002784-Paulsen2">[143]</xref>. The annotation of each of the genomes has undergone significant manual curation, removing small spurious overlapping ORFs and improving gene function calls. Manual curation of the genomes was performed using the MANATEE program (<ext-link ext-link-type="uri" ns0:href="http://manatee.sourceforge.net/jcvi/index.shtml">http://manatee.sourceforge.net/jcvi/index.shtml</ext-link>). The annotation of the previously-published genome of Pf-5 was updated and manually curated as part of this study. Phylogenetic analyses on the pyocin proteins were performed using MEGA 5 <xref ref-type="bibr" rid="pgen.1002784-Tamura1">[145]</xref>; domain analyses were performed using the InterProScan program, found on the InterPro website <xref ref-type="bibr" rid="pgen.1002784-Hunter1">[146]</xref>. Secondary metabolite production clusters were examined using the antiSMASH program <xref ref-type="bibr" rid="pgen.1002784-Medema1">[147]</xref>. The amino acid composition of products from NRPS sequences were predicted using NRPSpredictor 2 <xref ref-type="bibr" rid="pgen.1002784-Rttig1">[148]</xref>. Transposons were identified using the ISfinder database (<ext-link ext-link-type="uri" ns0:href="http://www-is.biotoul.fr/">http://www-is.biotoul.fr/</ext-link>) <xref ref-type="bibr" rid="pgen.1002784-Siguier1">[149]</xref>; only expectation values of 10<sup>&#8722;5</sup> and below were considered as significant matches during searches. The <italic>Pseudomonas</italic> genome database <xref ref-type="bibr" rid="pgen.1002784-Winsor1">[150]</xref> was consulted to obtain information on previously-published genomes for comparative purposes.</p><p>The seven genomes were compared to other genomes of <italic>Pseudomonas</italic> species using a multiway BLASTp analysis, and putative orthologs were identified with an E-value cutoff of 10<sup>&#8722;5</sup>. Synteny analyses were performed using Progressive MAUVE <xref ref-type="bibr" rid="pgen.1002784-Darling1">[151]</xref>. Phylogenetic relationships among all sequenced <italic>Pseudomonas</italic> species were investigated by generating phylogenetic trees with MrBayes 3.1.2 <xref ref-type="bibr" rid="pgen.1002784-Ronquist1">[152]</xref> using 1) 16S rRNA and 2) concatenated alignments of 10 highly conserved housekeeping genes: <italic>acsA</italic>, <italic>aroE</italic>, <italic>dnaE</italic>, <italic>guaA</italic>, <italic>gyrB</italic>, <italic>mutL</italic>, <italic>ppsA</italic>, <italic>pyrC</italic>, <italic>recA</italic>, and <italic>rpoB</italic>. We also used Hal, a Markov Clustering algorithm based on e-values from reciprocal all-by-all BLASTP analysis <xref ref-type="bibr" rid="pgen.1002784-Robbertse1">[153]</xref>, to determine phylogenetic relationships among the sequenced strains of <italic>Pseudomonas</italic> spp.</p><p>REP elements were defined by searching for repeat sequences greater than 30 nt in length that occurred more than 10 times within individual genomes using RepeatScout <xref ref-type="bibr" rid="pgen.1002784-Price1">[154]</xref>. Overlapping repeat regions were identified using sequence alignments and assembled to generate consensus repeat motifs. The consensus sequences were used to search the genomes with an identity cut-off of &gt;90%. Sequences identified were aligned using ClustalX and HMMs were generated from alignments using HMMER2.</p><p>Ten strains within the <italic>P. fluorescens</italic> group (<xref ref-type="table" rid="pgen-1002784-t001">Table 1</xref>) were tested for phenotypes associated with gene functions. In addition, we tested derivatives of strain Pf0-1 containing cloned <italic>gacA</italic> or <italic>gacS</italic> genes from strain Pf-5. pJEL5965 has a 1.6-kb <italic>Eco</italic>RI/<italic>Hin</italic>dIII fragment containing the <italic>gacA</italic> gene from Pf-5 cloned into pME6000; pJEL5999 has a 6.7-kb <italic>KpnI</italic> fragment containing the <italic>gacS</italic> gene from Pf-5 cloned into pME6000 <xref ref-type="bibr" rid="pgen.1002784-Maurhofer1">[155]</xref>. Mutants of strains Pf-5 and A506 were included as controls in phenotypic tests, including <italic>gacS</italic> and <italic>aprA</italic> mutants of A506 <xref ref-type="bibr" rid="pgen.1002784-Anderson1">[101]</xref>, and ii) <italic>gacA</italic>
<xref ref-type="bibr" rid="pgen.1002784-Hassan1">[69]</xref>, <italic>ofaA</italic>
<xref ref-type="bibr" rid="pgen.1002784-Hassan1">[69]</xref>, <italic>chiC</italic>, <italic>hcnB</italic>, and <italic>aprA</italic> mutants of Pf-5. The <italic>hcnB</italic> mutant was created as described for the <italic>ofaA</italic> mutant <xref ref-type="bibr" rid="pgen.1002784-Hassan1">[69]</xref>, except that the PCR product was digested with <italic>Hin</italic>dIII and cloned into the <italic>Hin</italic>dIII site of pEX18Tc <xref ref-type="bibr" rid="pgen.1002784-Hoang1">[156]</xref>. The <italic>aprA</italic> and <italic>chiC</italic> mutants contain in-frame deletions and were made using previously-described methods for creating an in-frame deletion <xref ref-type="bibr" rid="pgen.1002784-Kidarsa1">[157]</xref>, except that the PCR products were digested with <italic>Bam</italic>HI or <italic>Xba</italic>I and cloned into the <italic>Bam</italic>HI or <italic>Xba</italic>I site of pEX18Tc <xref ref-type="bibr" rid="pgen.1002784-Hoang1">[156]</xref>. Primers used for mutant construction are listed in <xref ref-type="supplementary-material" rid="pgen.1002784.s027">Table S17</xref>. The deletions were confirmed by PCR amplification and sequencing of the mutant alleles. All phenotypic tests were done on duplicate cultures grown at 27&#176;C (unless another temperature is specified), experiments were repeated, and representative results are presented (<xref ref-type="supplementary-material" rid="pgen.1002784.s022">Table S12</xref>, <xref ref-type="supplementary-material" rid="pgen.1002784.s026">Table S16</xref>).</p><p>Extracellular protease was assessed visually as a cleared zone around bacterial colonies on half-strength BBL Litmus milk agar (Becton, Dickinson and Company, Sparks, MD USA) following incubation for 2 or 4 days. <italic>aprA</italic> mutants of strains A506 <xref ref-type="bibr" rid="pgen.1002784-Anderson1">[101]</xref> and Pf-5 served as negative controls. Gelatinase activity was assessed in 12% gelatin incubated at 20&#176;C and examined at 48 h and 1 week post inoculation <xref ref-type="bibr" rid="pgen.1002784-Gerhardt1">[138]</xref>. Lipase activity was assessed in LB agar containing 1% w/v Tween 80, added before autoclaving. A positive result was observed as the formation of a white precipitate around a bacterial colony; plates were examined at 48 h and 1 week post inoculation. Chitinase activity was estimated from cultures grown in KB broth for 4 days with shaking using a methylumbelliferone-based chitinase assay kit (Sigma, St. Louis, MO). A chitinase-deficient mutant (<italic>chiC</italic>) of Pf-5 served as a negative control.</p><p>Cyclic lipopeptide (CLP) production was assessed as surfactant activity in the droplet collapse assay <xref ref-type="bibr" rid="pgen.1002784-Gross2">[35]</xref> and hemolytic activity, detected as a clearing zone surrounding colonies grown for 48 hr at 27&#176;C on BBL&#8482; Blood Agar Base (Becton, Dickinson and Company, Sparks, MD, USA). CLP production also was visualized as clear zones surrounding colonies grown on CAS agar amended with iron, as described by Hartney et al. <xref ref-type="bibr" rid="pgen.1002784-Hartney1">[158]</xref>. Swarming motility was assessed on standard succinate medium (SSM) <xref ref-type="bibr" rid="pgen.1002784-Meyer1">[159]</xref> containing 0.6% agar following 2 days of incubation at room temperature, as described previously <xref ref-type="bibr" rid="pgen.1002784-deBruijn2">[60]</xref>. Mutants deficient in cyclic lipopeptide production serving as negative controls were: an orfamide deficient mutant (<italic>ofaA</italic>) of strain Pf-5 <xref ref-type="bibr" rid="pgen.1002784-Hassan1">[69]</xref>, a viscosin-deficient mutant (<italic>viscA</italic>) of strain SBW25 <xref ref-type="bibr" rid="pgen.1002784-deBruijn2">[60]</xref>, and a massetolide-deficient mutant (<italic>massA</italic>) of strain SS101 <xref ref-type="bibr" rid="pgen.1002784-deBruijn1">[59]</xref>.</p><p>Indole production was assayed in supernatants of cultures of strains in KB broth with 0.2 mg/ml L-tryptophan for 48 h. Salkowski's reagent <xref ref-type="bibr" rid="pgen.1002784-Gordon1">[160]</xref> was added to the supernatants in a 2&#8758;1 ratio and OD<sub>530 nm</sub> was measured after 30 min incubation at room temperature.</p><p>We attempted to detect mangotoxin-associated activity using an established bioassay <xref ref-type="bibr" rid="pgen.1002784-Arrebola1">[72]</xref> evaluating symptoms following wound-inoculation of tomato leaves (cultivars Oregon Spring and Legacy).</p><p>Hydrogen cyanide production was detected as described by Sarniguet et al. <xref ref-type="bibr" rid="pgen.1002784-Sarniguet1">[161]</xref>. A mutant of Pf-5 (<italic>hcnB</italic>) deficient in hydrogen cyanide production served as a negative control.</p><p>The amount of &#945;-ketobutyrate generated by the enzymatic hydrolysis of 1-aminocyclopropane-1-carboxylic acid in cell-free extracts was monitored as described by Honma and Shimomura <xref ref-type="bibr" rid="pgen.1002784-Honma1">[162]</xref>.</p><p>Strains of <italic>Pseudomonas</italic> spp. were grown on LB agar at 25&#176;C overnight. Cells were inoculated into 1&#215; IF-0 media (Biolog, Inc., Hayward, CA, USA) and the transmittance of the suspension measured using a Biolog Turbidimeter (Biolog, Inc.). Cells were added until a uniform suspension of 42% transmittance was achieved. The cell suspension was added to 1&#215; IF-0 media containing Dye A (Biolog, Inc.) in a ratio of 1&#8758;5 to produce a cell suspension with a final transmittance of 85%. 100 &#181;l of cell suspension was transferred to each well of Biolog plates PM01 and PM02A (Biolog, Inc.). Plates were incubated using the OmniLog Phenotype MicroArray System (Biolog, Inc.) at 25&#176;C for 48 h, with measurements recorded at 15 min intervals. Data was visualized using OmniLog File Management/Kinetic Analysis software v1.20.02 and analyzed using OmniLog Parametric Analysis software v1.20.02 (Biolog, Inc.). The total area under the curve was used to compare strain phenotypes.</p><p>Growth on selected compounds as sole carbon sources was tested on minimal medium 925 <xref ref-type="bibr" rid="pgen.1002784-Langley1">[163]</xref> amended with the compounds at 0.1% w/v, unless otherwise noted.</p>