<p>This study was approved by the Institutional Review Boards (IRBs) or the ethnic committees at the institutions where subjects were recruited. All subjects were enrolled after informed consent had been obtained. The overall study was approved by the IRB of the Oklahoma Medical Research Foundation (OMRF).</p><p>To test the association of <italic>IL10</italic> family genes with SLE, we used a large collection of case-control subjects from the collaborative Large Lupus Association Study 2 (LLAS2) <xref ref-type="bibr" rid="pgen.1003870-Scofield1">[79]</xref>, including European American (EA: 4,248 cases vs. 3,818 controls), African American (AA: 1,724 vs. 2,024), Asian (AS: 1,328 vs. 1,348) and Hispanic enriched for the Amerindian-European admixture (HS: 1,622 cases vs. 887 controls). African Americans included 286 Gullahs (155 cases vs. 131 controls), who are subjects with African ancestry. Asians were composed primarily of Koreans (906 cases and 1,012 controls) but also included Chinese, Japanese, Taiwanese and Singaporeans. Cases were defined by meeting at least four of the 1997 American College of Rheumatology (ACR) revised criteria for the classification of SLE <xref ref-type="bibr" rid="pgen.1003870-Hochberg1">[80]</xref>.</p><p>To test functional consequences of SLE-associated variants, SLE patients and healthy controls of European ancestry were recruited at the University of California, Los Angeles and through the Lupus Family Registry and Repository (LFRR, lupus.omrf.org) for blood donations.</p><p>DNA samples were processed at the Lupus Genetics Studies Unit of OMRF. SNP genotyping was performed using an Illumina custom bead array on the iSCAN instrument for 19 tag SNPs covering over 135 kb of <italic>IL10-IL24</italic> region and 347 admixture informative markers (AIMs). SNPs meeting the following criteria were included for subsequent genetic association tests: well-defined cluster scatter plots, SNP call rate &gt;90%, minor allele frequency &gt;1%, total proportion missing &lt;5%, <italic>P</italic>&gt;0.05 for differential missing rate between cases and controls, and Hardy-Weinberg proportion (HWP) test with a <italic>P</italic>&gt;0.01 in controls and <italic>P</italic>&gt;0.0001 in cases.</p><p>Subjects with individual genotyping missing rate &gt;10% (due to low quality), shared identity by descent &gt;0.4 or showing mismatch between the reported and estimated gender were removed. The global ancestry of each subject was estimated based on genotype of AIMs, using principal components analysis (PCA) <xref ref-type="bibr" rid="pgen.1003870-Price1">[81]</xref> and ADMIXMAP <xref ref-type="bibr" rid="pgen.1003870-Hoggart1">[82]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1003870-McKeigue1">[84]</xref>, as described in another LLAS2 study <xref ref-type="bibr" rid="pgen.1003870-Lessard1">[85]</xref>, and then genetic outliers were removed.</p><p>Finally, a total of unrelated 15,533 subjects including EA (3,820 cases vs. 3,412 controls), AA (1,670 vs. 1,904; composed of 92.5% African Americans and 7.5% Gullahs), AS (1,252 vs. 1,249; composed of 74.6% of Koreans, 16.1% of Chinese and subjects from Japan and Singapore) and HS (1,445 vs. 781) were analyzed for 19 SNPs.</p><p>Imputation was performed using IMPUTE 2.1.2 <xref ref-type="bibr" rid="pgen.1003870-Howie1">[86]</xref>, with SNP genotypes of 379 Europeans (CEU, TSI, GBR, FIN and IBS), 246 Africans (YRI, ASW and LWK), 286 Asians (CHB, JPT and CHS) and 181 Americans (MXL, PUR and CLM) from the 1000 Genomes Project (version 3 of the phase 1 integrated data, March 2012 release) as references in imputation for our EA, AA, AS and HS subjects, respectively. Imputed genotypes had to meet the threshold of information score &gt;0.9, as well as the quality control criteria as described above. After imputation, we obtained an additional 109 SNPs for EA, 45 for AA, 80 for AS and 64 for HS (the number varied due to different LD structure) for further analysis.</p><p>Total RNA was purified with TRIzol reagent (Life Technologies) from PBMCs of EA individuals (58 SLE cases and 55 healthy controls) and reverse-transcribed into cDNA with SuperScript II Reverse Transcriptase kit (Life Technologies). Messenger RNA levels of <italic>IL10</italic> and a housekeeping gene <italic>RPLP0</italic> were measured by quantitative real-time PCR using TaqMan assays (<italic>IL10</italic> probe: Hs00961622_m1; <italic>RPLP0</italic> probe: Hs99999902_m1, Applied Biosystems). All samples were run in triplicate. Relative <italic>IL10</italic> mRNA levels were normalized to that of <italic>RPLP0</italic>, calculated by the 2<sup>&#8722;&#916;&#916;Ct</sup> method and Log10 transformed.</p><p>Plasma IL-10 levels from 132 SLE patients and 116 healthy controls of EA ancestry were measured by ELISA (R&amp;D systems).</p><p>To examine whether inhibition of MAPK pathway may affect co-expression of IL-10 and p-Elk-1, control PBMCs (1&#215;10<sup>6</sup>) were cultured in growth medium with or without interferon alpha (IFN&#945;) (1000 U/ml; PBL Biomedical Laboratories) in the presence or absence of one MAPK inhibitor (EMD Millipore), PD 98059 (20 &#181;M; ERK/MEK inhibitor), SP 600125 (20 &#181;M; JNK inhibitor) or SB 203580 (10 &#181;M; p38-MAPK pathway inhibitor), respectively. Addition of Brefeldin A (eBiosciense) to cells in culture blocks intracellular protein (IL-10) transport processes.</p><p>The patients with SLE recruited in this part of study were evaluated for disease activity by the SLE Disease Activity Index (SLEDAI) 2000 <xref ref-type="bibr" rid="pgen.1003870-Gladman2">[87]</xref> at the time of blood draw, and SLEDAI&#8805;4 was considered as active disease <xref ref-type="bibr" rid="pgen.1003870-Gladman1">[20]</xref>, <xref ref-type="bibr" rid="pgen.1003870-Yee1">[21]</xref>. Freshly isolated or cultured PBMCs were incubated with mouse reference serum to block nonspecific binding to Fc&#947; receptors and then incubated with peridinin chlorophyll protein (PerCP)-conjugated anti-human CD3, allophycocyanin (APC)-conjugated anti-human CD19 and phycoerythrin (PE)-conjugated anti-human CD14 (eBiosciense) to identify T cell, B cell and monocyte subpopulations, respectively. For intracellular staining of IL-10 and p-Elk-1, cells were fixed with IC Fixation Buffer (eBiosciense), washed with Permeabilization Buffer (eBiosciense), and stained with fluorescein isothiocyanate (FITC)-conjugated anti-human IL-10 (eBiosciense) and PE- or Alexa Fluor647-conjugated anti-phospho-Elk-1 antibody (BD Biosciences). Background fluorescence was assessed using appropriate isotype- and fluorochrome-matched control antibodies. Cells were collected and analyzed by FACSCalibur flow cytometer equipped with the manufacturer's software (CellQuest; BD Biosciences).</p><p>Bioinformatic analysis using the program TFSEARCH (conducted on 01/18/2011) showed predicted binding to Elk-1 and STRE in DNA sequence containing the minor but not the major allele of rs3122605 (<xref ref-type="supplementary-material" rid="pgen.1003870.s003">Figure S3</xref> and <xref ref-type="supplementary-material" rid="pgen.1003870.s004">S4</xref>). In addition, HSF, ADR1 and MZF1 were predicted to bind with sequence containing either allele of rs3122605. Given that we were interested in identifying transcription factors that preferentially bind to the minor allele of rs3122605, and that the STRE (stress response element) binding factor includes two yeast transcription factors, Msn2p and Msn4p, we prioritized to test Elk-1 using EMSA.</p><p>EMSA and supershift assays were performed as previously described <xref ref-type="bibr" rid="pgen.1003870-Tan1">[88]</xref>. Nuclear extracts were prepared from peripheral blood lymphocytes of SLE patients with active disease using NE-PER Nuclear Extraction Reagent (Thermo scientific) and incubated with biotin-labeled oligodeoxynucleotides (synthesized by Integrated DNA Technologies, depicted in <xref ref-type="supplementary-material" rid="pgen.1003870.s007">Table S3</xref>). EMSAs were performed with the LightShift Chemiluminescent EMSA kit (Thermo scientific). The antibody used in the supershift reactions was polyclonal rabbit anti-human Elk-1 (Santa Cruz Biotechnology).</p><p>Cytoplasmic or nuclear proteins from PBMCs were prepared using NE-PER Nuclear Extraction Reagent (Thermo scientific). Following SDS/PAGE, proteins were transferred onto Immobilon-P membrane (Millipore). After blocking with membrane blocking solution (Invitrogen, Life Technologies), the membrane was successively incubated with the anti-Elk-1 (Santa Cruz Biotechnology) or anti-phospho-Elk-1 (Cell Signaling Technology) primary antibody and the horseradish peroxidase (HRP)-conjugated secondary antibody (Santa Cruz Biotechnology). Blots were developed using the ECLPlus Western Blotting Detection System (GE Healthcare), visualized with ChemiDoc XRS imager and analyzed by Quantity One software (BIO-RAD). &#946;-actin or Lamin B was used as internal control.</p><p>Allelic association tests in each ancestral group and conditional haplotype-based association tests in EA ancestry were performed by PLINK v1.07 software <xref ref-type="bibr" rid="pgen.1003870-Purcell1">[89]</xref> under a logistic regression model adjusted for gender and the first three principal components estimated using AIMs. The Bonferroni corrected <italic>P</italic>-value threshold was adjusted to <italic>P</italic>&lt;3.9&#215;10<sup>&#8722;4</sup> (&#8202;=&#8202;0.05/128 SNPs in EA). Pairwised LD values between SNPs and haplotypic association with SLE were evaluated using Haploview 4.2 <xref ref-type="bibr" rid="pgen.1003870-Barrett2">[90]</xref>. The linear regression test was used to evaluate the association of SNP genotypes with IL-10 mRNA or protein levels. The Student's <italic>t</italic>-test was used to compare the mean values between two groups. A <italic>P</italic> value&lt;0.05 was considered to be statistically significant.</p>