<p>Human K562 cells, HCT1116 cells, U2OS cells and murine erythroleukemia cells (RL4) containing Rep-P variants were grown at 37&#176;C in a 5% CO2 atmosphere in Dulbecco's modified Eagle medium (Invitrogen, Cat. no. 10564-011), supplemented with 10% heat-inactivated fetal calf serum. For RL4 cells containing Rep-P variants, two transgenes containing sequences from the human beta-globin Locus Control Region (HS432), the human beta-globin promoter (GloPro) driving enhanced green fluorescent protein (EGFP) and two variants of the Rep-P replicator were inserted into a single location on murine chromosome 15 in murine erythroleukemia (RL4) cells <xref ref-type="bibr" rid="pgen.1003542-Feng1">[11]</xref>. We introduced the AG1 mutation by site-directed mutagenesis. Details of the mutagenesis methods were published previously <xref ref-type="bibr" rid="pgen.1003542-Huang1">[12]</xref>.</p><p>HCT116 cells were transfected with 25 nM of control siRNA (Dharmacon, D-001810-10 or Dot1L siRNA (Dharmacon, L-014900-01 or Qiagen, GS84444) with DharmaFECT 2 siRNA (Dharmacon, T-2002) or RNAiMAX Transfection Reagent (Invitrogen, 13778) according to the manufacturer's protocol. Cells were transfected 2 or 3 times with a 2 to 3 days interval and harvested 72 h after the last transfection. Whole cell lysates by1XSDS loading buffer were used for western-blot with anti-Histone H3 (dimethylK79) antibody (Abcam, ab3594 or Millipore, 04-835) to verify the knockdown efficiency of dot1L siRNA.</p><p>We performed nascent-strand DNA analysis as described previously <xref ref-type="bibr" rid="pgen.1003542-Wang2">[33]</xref>. DNA was extracted from asynchronous cells, denatured by boiling for 10 minutes, incubated on ice for 10 minutes, and fractionated on a neutral sucrose gradient. We collected 0.5&#8211;1 kb DNA fractions, treated them with &#955; exonuclease to remove non-RNA-primed genomic DNA fragments, purified them, and measured the DNA concentration using a NanoDrop 1000 (Thermo Scientific). We quantified nascent strand DNA by real-time polymerase chain reaction (PCR) in an ABI 7900 thermocycler (primers and probes used for real-time PCR are listed in <xref ref-type="table" rid="pgen-1003542-t003">Table 3</xref>). For whole genome analyses of nascent strand abundance, DNA from K562 cells was isolated using the above procedure and processed for massively parallel sequencing as previously described <xref ref-type="bibr" rid="pgen.1003542-Martin1">[3]</xref>.Three independent biological replicates were sequenced to the depth of 1.4&#215;10<sup>8</sup> reads. All the data from mapping replication initiation events in K562 cells were deposited in GEO, as Series GSE28911.</p><p>We performed ChIP analyses with 1% formaldehyde-fixed K562 and RL4 cells using the Millipore ChIP assay kit (Cat. no. 17-295) following the manufacturer's protocol. Anti-H3K79Me2 antibody was from Abcam (ab3594). We analyzed ChIP samples by real-time PCR in an ABI 7900 thermocycler using primers/probes listed in <xref ref-type="table" rid="pgen-1003542-t003">Table 3</xref>.</p><p>DNA combing analyses of replicating DNA were performed according to previously published methods <xref ref-type="bibr" rid="pgen.1003542-Conti1">[47]</xref>. Briefly, cells were pulse-labeled with 20 &#181;M IdU (Sigma, Cat. no. I-7125) for 20 minutes and then with 50 &#181;M CldU (MP biomedical, Cat. no. 105478) for 20 minutes before harvest. Then the cells were embedded in low-melting point agarose plugs and lysed in the plug with proteinase K lysis buffer at 50&#176;C overnight. After agarose was digested with &#946;-agarase (NewEngland Biolabs), DNA was combed onto silanized surfaces (Microsurfaces, Inc.) and detected with anti-IdU (BD, Cat. no. 347580), anti-CldU (Accurate Chemical, Cat. no. OBT0030), and anti-single strand DNA (Chemicon, Cat. no. MAB3034) antibodies. Images were captured with the Attovision software using the epifluorescence microscope Pathway (Becton Dickinson) and measured the signals using ImageJ (open source from National Cancer Institute, NIH) with custom-made macros <xref ref-type="bibr" rid="pgen.1003542-Conti2">[48]</xref>.</p><p>Cells were cultured in 4-well chamber slides, pulse labeled with 10 &#181;M EdU (Invitrogen) for 1 hour before harvest. EdU staining using the Click-iT EdU kit (Invitrogen) were performed according to manufacturer's protocol. Images were captured with the Attovision software using the epifluorescence microscope Pathway (Becton Dickinson) and measured with the Attovision software for DNA content by DAPI for cells with early S-phase and late-S phase EdU replication patterns.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Asynchronous K562 cells or fractionated G1, S and G2M K562 cells by elutriator (see <xref ref-type="bibr" rid="pgen.1003542-Huang1">[12]</xref> for elutriation details) were used for ChIP as described above. ChIP-Seq samples were sequenced using the Illumina GA II platform. The resultant sequences were aligned to the hg19 (NCBI Build 37) human reference genome using Bowtie <xref ref-type="bibr" rid="pgen.1003542-Langmead1">[49]</xref>. Alignments were converted to .bam and .tdf format using SAMtools and igvtools for visualization in Broad's Integrative Genomics Viewer, <ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/igv/">http://www.broadinstitute.org/igv/</ext-link>
<xref ref-type="bibr" rid="pgen.1003542-Robinson1">[50]</xref>.</p><p>Reads per kilobase per million aligned reads (RPKM) values were calculated for each sample using 100 base genomic bins <xref ref-type="bibr" rid="pgen.1003542-Mortazavi1">[51]</xref>. A Gaussian smoothing algorithm was applied to the bin values. To correct for sequencing biases and copy number variation, an enrichment ratio was defined as the ratio of nascent strand RPKM to control RPKM, and calculated for each 100 base bin. The H3K79Me2 and H3K4Me3 ChIP-Seq data are deposited in GEO Series GSE GSE35294 including 11 sample files from experiments in unsynchronized and cell cycle fractionated cells.</p><p>HCT116 cells transfected with control siRNA or Dot1L siRNA were pulse-labeled with 50 &#181;M of BrdU for 18 hours before harvest. Genomic DNA were purified and sonicated to 500&#8211;4000 bp. Genomic DNA from HCT116 cells with no BrdU incorporation and BrdU incorporation for 48 hours was used as control. 100 &#181;g of DNA was fractionated with 6 ml CsCl (1 g/ml in TE). Samples were spun at 45000 rpm in a Ti75 rotor (Beckman) for 66 hours. Fractions of 250 &#181;l were collected and the refractory index was measured to confirm the formation of the gradient. Same volume samples from each fraction were loaded to a positive charged nylon membrane by a Slot Blot Filtration Manifold (PR648, GE Healthcare life sciences). 0.5 ng, 5 ng, 50 ng, 500 ng and 5000 ng of total genomic DNA labeled with BrdU for 48 hours were also loaded to the membrane as standard for BrdU. BrdU were detected with anti-BrdU antibody.</p>