<p>Clinical information was assimilated from patient records from the Mayo Clinic. Informed consent was obtained for each patient on two ongoing research protocols approved by the Mayo Clinic Institutional Review Board (10-006180 and 10-002879). Clinicopathological features collected included age, gender, stage, histological grade, sites of metastasis, tumor sample assessment for overall cellularity/necrosis and percent tumor cellularity, prior therapies and risk factors (hepatitis B and C, Caroli's disease, obesity, hepatolithiasis and cholelithiasis, primary sclerosing cholangitis, thorotrast exposure and <italic>H. pylori, H. bilis, S. typhi</italic> and <italic>S. paratyphi</italic> infections). All patients were known to not have had prior exposure to liver flukes that have been implicated in biliary carcinogenesis (<italic>O. viverrini and C. sinensis</italic>). Tissue specimens were collected fresh frozen and maintained below &#8722;80&#176;C until nucleic acid extraction. A board certified pathologist who is experienced in biospecimen studies, evaluated a portion of each specimen to confirm the presence of tumor, the degree of necrosis and the percent cellularity.</p><p>1.1 &#181;g genomic DNA was used to generate separate long insert whole genome libraries for each sample using Illumina's (San Diego, CA) TruSeq DNA Sample Prep Kit (catalog# FC-121-2001). In summary, genomic DNAs are fragmented to a target size of 900&#8211;1000 bp on the Covaris E210. 100 ng of the sample was run on a 1% TAE gel to verify fragmentation. Samples were end repaired and purified with Ampure XP beads using a 1&#8758;1 bead volume to sample volume ratio, and ligated with indexed adapters. Samples are size selected at approximately 1000 bp by running samples on a 1.5% TAE gel and purified using Bio-Rad Freeze &#8216;n Squeeze columns and Ampure XP beads. Size selected products are then amplified using PCR and products were cleaned using Ampure XP beads.</p><p>300 ng genomic tumor and normal DNA was used to create whole genome libraries. Samples were fragmented on the Covaris E210 to a target size of 200&#8211;300 bp and 50 ng of the fragmented product was run on a 2% TAE gel to verify fragmentation. Whole genome libraries were prepared using Illumina's TruSeq DNA Sample Prep Kit.</p><p>1.1 &#181;g genomic DNA for each sample was fragmented to a target size of 150&#8211;200 bp on the Covaris E210. 100 ng of fragmented product was run on TAE gel to verify fragmentation. The remaining 1 &#181;g of fragmented DNA was prepared using Agilent's SureSelect<sup>XT</sup> and SureSelect<sup>XT</sup> Human All Exon 50 Mb kit (catalog# G7544C).</p><p>50 ng genomic tumor and normal DNA was used to create exome libraries using Illumina's Nextera Exome Enrichment kit (catalog# FC-121-1204) following the manufacturer's protocol.</p><p>1 &#181;g of each tumor and germline DNA sample was used to generate separate exome libraries. Libraries were prepared using Illumina's TruSeq DNA Sample Prep Kit and Exome Enrichment Kit (catalog# FC-121-1008) following the manufacturer's protocols.</p><p>3 &#181;g of genomic tumor and normal DNA was fragmented on the Covaris E210 to a target size of 150&#8211;200 bp. Exome libraries were prepared with Agilent's (Santa Clara, CA) SureSelectXT Human All Exon V4 library preparation kit (catalog# 5190-4632) and SureSelectXT Human All Exon V4+UTRs (catalog# 5190-4637) following the manufacturer's protocols.</p><p>50 ng total RNA was used to generate whole transcriptome libraries for RNA sequencing. Using the Nugen Ovation RNA-Seq System v2 (catalog# 7102), total RNA was used to generate double stranded cDNA, which was subsequently amplified using Nugen's SPIA linear amplification process. Amplified products were cleaned using Qiagen's QIAquick PCR Purification Kit and quantitated using Invitrogen's Quant-iT Picogreen. 1 &#181;g of amplified cDNA was fragmented on the Covaris E210 to a target size of 300 bp. Illumina's TruSeq DNA Sample Preparation Kit was used to prepare libraries from 1 &#181;g amplified cDNA.</p><p>1 &#181;g of total RNA for each sample was used to generate RNA sequencing libraries using Illumina's TruSeq RNA Sample Prep Kit V2 (catalog# RS-122-2001) following the manufacturer's protocol.</p><p>Libraries with a 1% phiX spike-in were used to generate clusters on HiSeq Paired End v3 flowcells on the Illumina cBot using Illumina's TruSeq PE Cluster Kit v3 (catalog# PE-401-3001). Clustered flowcells were sequenced by synthesis on the Illumina HiSeq 2000 using paired-end technology and Illumina's TruSeq SBS Kit.</p><p>For whole genome and exome sequencing fastq files were aligned with BWA 0.6.2 to GRCh37.62 and the SAM output were converted to a sorted BAM file using SAMtools 0.1.18. BAM files were then processed through indel realignment, mark duplicates, and recalibration steps in this order with GATK 1.5 where dpsnp135 was used for known SNPs and 1000 Genomes' ALL.wgs.low_coverage_vqsr.20101123 was used for known indels. Lane level sample BAMs were then merged with Picard 1.65 if they were sequenced across multiple lanes. Comparative variant calling for exome data was conducted with Seurat <xref ref-type="bibr" rid="pgen.1004135-Christoforides1">[105]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Previously described copy number and translocation detection were applied to the whole genome long insert sequencing data <xref ref-type="bibr" rid="pgen.1004135-Craig1">[59]</xref> and these are made available through <ext-link ext-link-type="uri" ns0:href="https://github.com/davcraig75/tgen_somaticSV">https://github.com/davcraig75/tgen_somaticSV</ext-link>. Copy number detection was based on a log2 comparison of normalized physical coverage (or clonal coverage) across tumor and normal whole genome long-insert sequencing data, where physical coverage was calculated by considering the entire region a paired-end fragment spans on the genome, then the coverage at 100 bp intervals was kept. Normal and tumor physical coverage was then normalized, smoothed and filtered for highly repetitive regions prior to calculating the log2 comparison. Translocation detection was based on discordant read evidence in the tumor whole genome sequencing data compared to its corresponding normal data. In order for the structural variant to be called there needs to be greater than 7 read pairs mapping to both sides of the breakpoint. The unique feature of the long-insert whole-genome sequencing was the long overall fragment size (&#8764;1 kb), where by two 100 bp reads flank a region of &#8764;800 bp. The separation of forward and reverse reads increases the overall probability that the read pairs do not cross the breakpoint and confound mapping.</p><p>For RNA sequencing, lane level fastq files were appended together if they were across multiple lanes. These fastq files were then aligned with TopHat 2.0.6 to GRCh37.62 using ensembl.63.genes.gtf as GTF file. Changes in transcript expression were calculated with Cuffdiff 2.0.2. For novel fusion discovery reads were aligned with TopHat-Fusion 2.0.6 <xref ref-type="bibr" rid="pgen.1004135-Kim1">[106]</xref> (patients 2, 3, 4 and 6). In addition, Chimerascan 0.4.5 <xref ref-type="bibr" rid="pgen.1004135-Iyer1">[107]</xref> was used to detect fusions in patient 1, deFuse 5.0 <xref ref-type="bibr" rid="pgen.1004135-McPherson1">[108]</xref> used in patients 2, 3 and 5 and SnowShoes <xref ref-type="bibr" rid="pgen.1004135-Asmann1">[109]</xref> for patients 2 and 5.</p><p>Mutations of potential clinical relevance were confirmed in a Clinical Laboratory Improvement Amendments (CLIA) laboratory with Sanger sequencing or quantitative PCR.</p><p>The immunohistochemistry was performed following the procedures described previously <xref ref-type="bibr" rid="pgen.1004135-Diep1">[110]</xref>. Briefly, slides were dewaxed, rehydrated and antigen retrieved on-line on the BondMax autostainer (Leica Microsystems, INC Bannockburn, IL). Slides were then subjected to heat-induced epitope retrieval using a proprietary EDTA-based retrieval solution. Endogenous peroxidase was then blocked and slides were incubated with the following antibodies: FGFR2 (BEK, Santa Cruz, catalog# sc-20735), FGFR3 (C-15, Santa Cruz, catalog# sc-123), panAKT (Cell Signaling Technology, catalog# 4685, pAKT (Cell Signaling Technology, catalog# 4060), EGFR (Cell Signaling Technology, catalog# 4267, pEGFR (Cell Signaling Technology, catalog#2234), MAPK/ERK1/2 (Cell Signaling Technology, catalog# 4695), pMAPK/pERK (Cell Signaling Technology, catalog# 4376) and pFRS2 Y436 (Abcam, catalog# ab78195). Sections were visualized using the Polymer Refine Detection kit (Leica) using diaminobenzidine chromogen as substrate.</p><p>FISH was performed on formalin-fixed paraffin-embedded (FFPE) specimens using standard protocols and dual-color break-apart rearrangement probes specific to the FGFR2 gene (Abbott Molecular, Inc. Des Plaines, IL) located at 10q26. The 5&#8242; FGFR2 signal was labeled with Spectrum Orange (orange) and the 3&#8242; FGFR2 signal was labeled with Spectrum Green (green).</p>