<p>The GA-1 strain of <italic>T. castaneum</italic> was used for all experiments. Insects were reared at 30&#176;C in wheat flour containing 5% brewer's yeast under standard conditions as described previously <xref ref-type="bibr" rid="pgen.1003268-Beeman1">[20]</xref>.</p><p>An extensive, genome-wide search for homologs of <italic>DmRtv</italic> in the <italic>T. castaneum</italic> genome database was performed using NCBI programs TBLASTN and BLASTP and using the amino acid sequence of DmRTV as query.</p><p>A DNA fragment containing the complete coding sequence of <italic>TcRtv</italic> (453 bp) was amplified by reverse transcriptase-PCR (RT-PCR) using the gene-specific primers (forward primer <named-content content-type="gene">5&#8242;- ATGGGTCTGTTTAGATCAATTT-3&#8242;</named-content> and the reverse primer <named-content content-type="gene">5&#8242;- TTACAAAAATCGTAAAATCAGTCT-3&#8242;</named-content>) using cDNA prepared from the RNA extracted from different stages of beetle development as template. Additional 5&#8242; and 3&#8242; UTR sequences were obtained by 5&#8242;- and 3&#8242;- RACE. The amplified fragment was cloned into the pGEMT vector. Sequencing of the cDNA clone was carried out at the DNA sequencing facility at Kansas State University.</p><p>Rtv-like proteins were identified by TBLASTN searches of the fully sequenced genomes of <italic>T. castaneum, Anopheles gambiae, Aedes aegypti, Culex quinquefasciatus, Bombyx mori, Acyrthosiphon pisum, Nasonia vitripennis</italic> and <italic>Ixodes scapularis</italic> using the amino acid sequence of DmRtv as query. Multiple sequence alignments of TcRtv and the related Rtv-like proteins from other insects and arthropods were carried out using the ClustalW software prior to phylogenetic analysis. A consensus phylogenetic tree was constructed using MEGA 4.0 neighbor joining method <xref ref-type="bibr" rid="pgen.1003268-Tamura1">[21]</xref>.</p><p>The isolation of RNA and preparation of cDNA from different stages of <italic>T. castaneum</italic> development were carried out as described previously <xref ref-type="bibr" rid="pgen.1003268-Chaudhari1">[10]</xref>. These cDNA templates and the pair of <italic>TcRtv</italic>-specific primers, forward primer <named-content content-type="gene">5&#8242;-ATGGGTCTGTTTAGATCAATTT-3&#8242;</named-content> and reverse primer <named-content content-type="gene">5&#8242;- TTACAAAAATCGTAAAATCAGTCT -3&#8242;</named-content>, were used for RT-PCR to determine the <italic>TcRtv</italic> expression profile. The <italic>T. castaneum ribosomal protein-6</italic> (<italic>TcRpS6</italic>) gene was used as an internal control for equal loading of cDNA templates <xref ref-type="bibr" rid="pgen.1003268-Arakane3">[22]</xref>.</p><p>Two different regions of the <italic>TcRtv</italic> gene were selected for making dsRNAs. Pairs of forward and reverse primers for the chosen regions (<xref ref-type="supplementary-material" rid="pgen.1003268.s005">Table S1</xref>) were used to amplify the dsRNA sequences by using the cloned <italic>TcRtv</italic> cDNA as template. The Ampliscribe T7-Flash Transcription Kit (Epicentre Technologies) was used to synthesize dsRNA as described previously <xref ref-type="bibr" rid="pgen.1003268-Arakane1">[5]</xref>. dsRNA for the tryptophan oxygenase, <italic>dsVermilion (dsVer)</italic> gene, which is required for eye pigmentation, was used as a control. <italic>dsRNAs</italic> for <italic>TcRtv</italic> or <italic>TcVer</italic> were injected into animals in the young larval, last instar larval, or pharate pupal stages of <italic>T. castaneum</italic> development (200 ng per insect, n&#8202;=&#8202;30). Five insects were collected at the young larval, pharate pupal and pharate adult stages of development 3&#8211;5 d after dsRNA treatment. Total RNA was extracted from the collected insects for measuring transcript levels by RT-PCR using gene-specific primer-pairs.</p><p>The full-length <italic>TcRtv</italic> cDNA clone was used as template to amplify the complete coding region of the <italic>TcRtv</italic> gene. A primer pair containing appropriate restriction enzyme sites [forward primer <named-content content-type="gene">5&#8242;-TATCCCGGGATGGGTCTGTTTAGATC-3&#8242;</named-content> (Sma I) and a reverse primer <named-content content-type="gene">5&#8242;-CAGACTGATTTTACGATTTTTGTAATCTAGATAT-3&#8242;</named-content> (Xba I)] was used to facilitate directional cloning of the <italic>TcRtv</italic> open reading frame (ORF) DNA in the pVL1393 baculovirus expression vector (BD Pharmingen). PCR-amplified, full length <italic>TcRtv</italic> DNA and the pVL1393 vector DNA were digested with the same pair of restriction enzymes and ligated as described previously <xref ref-type="bibr" rid="pgen.1003268-Zhu1">[15]</xref>. Hi-5 cells (<italic>Trichoplusia ni</italic> cell line) were used to express the TcRtv protein as described earlier <xref ref-type="bibr" rid="pgen.1003268-Zhu1">[15]</xref>.</p><p>
<italic>TcRtv</italic> dsRNA-treated pharate adult insects were collected and fixed as described previously <xref ref-type="bibr" rid="pgen.1003268-Chaudhari1">[10]</xref>. Cryosections of 20 &#181;m thickness were made and stained for specific proteins using chicken antiserum to <italic>T. castaneum</italic> Rtv (1&#8758;100), rabbit antiserum (1&#8758;100) to <italic>D. melanogaster</italic> Knk and rabbit antiserum to <italic>T. castaneum</italic> Chs-A (1&#8758;50) as primary antibodies. Alexa 488 goat-anti-chicken IgG (1&#8758;1000) or Alexa 488 goat-anti-rabbit IgG (1&#8758;1000) were used as secondary antibodies for the fluorescence detection of TcRtv, TcKnk and TcChs-A proteins. Rhodamine conjugated chitin-binding probe (1&#8758;100) and DAPI (1&#8758;15) were used for staining of chitin and nuclei, respectively. Confocal microscopy was performed with an LSM META 510 laser scanning confocal microscope using laser lines of 405 nm, 488 nm and 543 nm for excitation. Images were taken using an oil objective (40&#215;/1.3 NA) with 8&#215; zoom and processed in Adobe Photoshop 7.0.</p><p>Hi-5 cells were infected with a recombinant baculovirus containing the TcRtv and TcKnk open reading frame and incubated at 30&#176;C for 3 d for expression of proteins. Three days post-infection, Hi-5 cells expressing respective proteins were treated with PI-PLC as described previously <xref ref-type="bibr" rid="pgen.1003268-Chaudhari1">[10]</xref>.</p><p>Following dsRNA treatment, pharate pupae and pharate adults of <italic>T. castaneum</italic> were collected for chitin content analysis by a modified Morgan-Elson method as described previously <xref ref-type="bibr" rid="pgen.1003268-Arakane1">[5]</xref>. <italic>TcVer</italic> dsRNA and <italic>TcChs-A</italic> dsRNA-treated insects were used as negative and positive controls for the experiment, respectively.</p><p>Pharate pupae were injected with dsRNA and pharate adult samples were collected on pupal day 5, fixed and embedded in EMBED 812/Araldite resin as described previously <xref ref-type="bibr" rid="pgen.1003268-Chaudhari1">[10]</xref>. Resin embedded samples were then thin-sectioned (silver to gold section) and imaged on a CM-100 TEM.</p><p>cDNA sequence for <italic>T. castaneum</italic> Retroactive (TcRtv) is deposited at NCBI with accession number JX470185.</p>