<p>All strains used in this study were derived from the genome sequence strain H99 and are listed in <xref ref-type="supplementary-material" rid="pgen.1004292.s005">Table S1</xref>. YEPD contains 1% yeast extract, 2% Bacto-peptone and 2% dextrose. YES medium contains 0.5% yeast extract, 3% glucose and 225 &#181;g/ml each of uracil, adenine, leucine, histidine, and lysine. Low-oxygen conditions (5% CO<sub>2</sub> and 1% O<sub>2</sub>) were maintained using an Invivo2 400 workstation at the indicated temperatures (Ruskinn, UK). For simplicity, we only mention the oxygen concentration without describing the CO<sub>2</sub> concentration unless specified in our study. The GasPakPlus anaerobic system (BRL) was used to generate 0% O<sub>2</sub> conditions. Spot assay was conducted by serial dilutions of cultures, spotted on agar plates, incubated at conditions as indicated for 3&#8211;6 days.</p><p>Zeiss Axiovert fluorescent microscope equipped with an AxioCam MRm digital camera was used to visualize fluorescent and differential interference contrast microscopy (DIC) images. Axiovision (version 4.0) was used to capture the images that were further processed by Adobe Photoshop CS4 software. Percentage of the polarized cells was determined as described <xref ref-type="bibr" rid="pgen.1004292-Ho1">[13]</xref>.</p><p>
<italic>cdc24&#916;</italic> was transformed with a multicopy suppressor library made in a multicopy episomic plasmid, pYCC725, from LP2, a B-3501 derived strain <xref ref-type="bibr" rid="pgen.1004292-Lee2">[62]</xref>. Suppressor clones were selected by plating the transformants on a medium containing 100 &#181;g/ml nourseothricin at 1% O<sub>2</sub> and 28&#176;C. Three separate screens were performed and 17 transformants were isolated. The plasmids in transformants were isolated and reconfirmed for their ability to restore the growth of <italic>cdc24&#916;</italic> at 1% O<sub>2</sub>. The inserts in the episomes of the transformants were PCR-amplified using primers OYC725B and OYC725C and the amplified PCR fragments were sequenced as described <xref ref-type="bibr" rid="pgen.1004292-Lee2">[62]</xref>. The genomic content in the PCR amplified region was identified by a BLAST search of the B-3501 genome sequence. Episomal plasmids from each clone were rescued in <italic>E. coli</italic> and retransformed into the <italic>cdc24&#916;</italic> strain to confirm the phenotype. For <italic>HRD1</italic> and <italic>GEF1</italic>, the entire gene region was cloned by PCR and reconfirmed for its ability to suppress the hypoxia phenotype of <italic>cdc24&#916;</italic>.</p><p>Gene deletion was carried out via homologous recombination by biolistic transformation <xref ref-type="bibr" rid="pgen.1004292-Toffaletti1">[63]</xref>. Overlapping PCR technique was used to generate deletion constructs <xref ref-type="bibr" rid="pgen.1004292-Kuwayama1">[64]</xref>. PCR and Southern hybridization was used to confirm homologous integrations. Wild type genes were PCR amplified from the strain H99, cloned and sequenced as described <xref ref-type="bibr" rid="pgen.1004292-Chang2">[9]</xref>. To generate complementation construct, each gene was either inserted in the multiple cloning site of pYCC744 which contained the NAT gene as a selectable marker <xref ref-type="bibr" rid="pgen.1004292-Chang2">[9]</xref> or cloned by overlapping PCR with Hygromycin gene as selectable maker.</p><p>Cells were grown in YPD medium to early log phase in normoxic conditions, spun down and transferred to 1% O<sub>2</sub> for 1 hr. Equal volume of ice-cold stop buffer (0.9% NaCl, 1 mM NaN<sub>3</sub>, 10 mM Na-EDTA, 50 mM NaF) was added to the culture. Cells were spun down, washed once in ice-cold stop buffer and lyophilized. Lyophilized cells were disrupted with 1-mm zirconia/silica bead using FastPrep-24 (MP Biomedicals, CA) and resuspended in lysis buffer as described <xref ref-type="bibr" rid="pgen.1004292-Bahn2">[36]</xref>. Cell lysates were spun down at 4&#176;C and protein concentrations were determined using Bio-Rad Bradford Protein assay reagent (Richmond, CA). An equal amount of protein (20 &#181;g) was loaded on the Any kD Criterion TGX Stain-Free gel (Bio-Rad, Richmond, CA) and proteins were transferred to a PVDF membrane. The western blot was incubated with a rabbit phospho-p38-MAPK antibody (Cell Signaling, Beverly, MA) and with a secondary anti-rabbit horseradish peroxidase&#8211;conjugated antibody. The blot was developed using the Clarity Western ECL (Bio-Rad, Richmond, CA). The signal was quantitated using ChemiDoc MP imaging system (Bio-Rad, Richmond, CA). The blot was stripped and used for detection of Hog1 with a rabbit polyclonal anti-Hog1 antibody (Santa Cruz Biotechnology, Santa Cruz, CA). The ratio of the signal intensity between phosphorylated Hog1 and total Hog1 was calculated and expressed as relative phosphorylation levels of the wild-type control.</p><p>To isolate RNA from 1% O<sub>2</sub> grown cells, overnight cultures of wild-type and each deletant strain were refreshed in fresh YPD media for 5 h in ambient air at 28&#176;C and shifted to 1% O<sub>2</sub> for 2 h at 28&#176;C. RNA was extracted from cryptococcal cells using Trizol (Invitrogen, Carlsbad, CA) and purified with RNeasy MinElute cleanup kit (Qiagen, Valencia, CA). The RNAseq was performed at RML Research Technologies Section, NIAID, NIH. The Illumina TruSeq RNA Sample Preparation Kit (Illumina, San Diego, CA) and its workflow were used for the preparation of barcoded RNA-Seq libraries. Final library products were quantified by qPCR using a KAPA Illumina GA Library Quantification Kit (KAPA Biosystems, Boston, MA) and sequenced on a HiSeq 2000 (Illumina) to produce paired 100 bp reads. Each of the 12 RNAseq libraries were given a unique barcode and pooled for clustering. We used 4 lanes on the HiSeq for sequencing, each lane containing the 12 libraries. Initial processing was performed using Illumina pipeline (CASAVA 1.8). TruSeq adapters were then trimmed and the reads were quality filtered with the FastXToolkit. All reads were mapped, using TopHat vl. 3.0 <xref ref-type="bibr" rid="pgen.1004292-Trapnell1">[65]</xref>, to the <italic>C. neoformans</italic> H99 reference genome provided by the Broad Institute. The total number of sequence reads ranged from 78 to 102 million pairs of which nearly 87.6%&#8211;88.9% were uniquely aligned and properly paired to the reference genome assembly. The mapped reads were then used downstream by Cufflinks software packages <xref ref-type="bibr" rid="pgen.1004292-Trapnell1">[65]</xref> for transcript assembly, differential expression, and for gene model construction for RNA-Seq. A union of transcripts from all four samples was produced by Cuffcompare from individually assembled transcripts of each sample which yielded 6967 transcripts. Cuffdiff was used to determine statistically significant differences between strains on assembled transcripts. Gene expression was normalized using the number of fragments per kilobase of transcript per million mapped reads (FPKM). Venn's diagram was produced by taking the statistically significant differentially expressed genes with FPKM cut-off of &gt;10. Gene ontology analysis was performed using DAVID Functional Annotation Tools and with the tool of functional annotation clustering and GO Fat database (GOTERM_BP_FAT) <xref ref-type="bibr" rid="pgen.1004292-Dennis1">[37]</xref>. Based on the homology between H99 and JEC21, each of the H99 gene identification designated by Broad institute was converted to the JEC21 gene identification before using DAVID since DAVID did not recognize the Broad institute identification. The analyzed RNA-Seq data has been submitted to the Sequence Read Archive (SRA) at NCBI and can be viewed under accession number of experiment SRX347687.</p><p>Log phase grown cells were spun down and transferred to 1% O<sub>2</sub> for 1, 3 or 5 hrs or 20% O<sub>2</sub> for 2 hrs in YPD medium. Cells were harvested for sterol extraction. Sterol extraction and analysis was carried out as described <xref ref-type="bibr" rid="pgen.1004292-Bahn1">[31]</xref>. Briefly, 10 mM of sodium azide was added to the culture medium immediately before harvest. Cells (1&#215;10<sup>8</sup> total) were harvested by centrifugation and resuspended in 9 ml methanol and 4.5 ml 60% (wt/vol) KOH together with 5 &#181;g cholesterol which was used as an internal recovery standard. Samples were heated to 75&#176;C in a water bath for 2 h to complete the saponification and the sterols were then extracted with hexane. The extracted sterols were analyzed and characterized by GC-MS (gas chromatography-mass spectrometry) using an ISQ mass spectrometer from Thermo Electron, coupled to a Trace GC Ultra chromatograph, from Thermo Electron, in the EI (electron impact ionization) mode. This GC-MS instrument used a Restek 5MS fused silica column (30 m length, 0.25 mm i.d., 25 &#181;m film thickness) with a program temperature from 200&#176;C (1 min) to 300&#176;C at a rate of 10&#176;C/min. Cholesterol and ergosterol were identified by comparison with standards. The mass spectral data used in combination with a database (NIST/EPA/NIH Mass Spectral Library version 2.0) and published spectra from <italic>C. neoformans</italic>
<xref ref-type="bibr" rid="pgen.1004292-Nes1">[66]</xref> allowed the identification of several peaks for which there were no standards available. The nomenclature of the discernable ergosterol intermediates was followed according to the <italic>C. neoformans</italic> study <xref ref-type="bibr" rid="pgen.1004292-Nes1">[66]</xref>. Relative amounts of each ergosterol intermediate were determined by comparing the area under the peaks in the chromatogram versus the area under the cholesterol peak to correct for recovery.</p><p>Isolation and analysis of genomic DNA was carried out as described previously <xref ref-type="bibr" rid="pgen.1004292-Chang6">[67]</xref>. RNA was treated with RNAse-free DNAse (Ambion, Austin, TX) to remove genomic DNA before quantitative real time reverse transcription PCR (qRT-PCR). cDNA was synthesized using high-capacity cDNA archive kit (Applied Biosystems, Foster City, CA). The qRT-PCR was performed using 20 &#181;l triplicate reactions with SYBR select master mix from two biological replicates and the ABI PRISM 7500 sequence detection system (Applied Biosystems, Foster City, CA). The PCR efficiency and CT determination was performed using the algorithm as described <xref ref-type="bibr" rid="pgen.1004292-Zhao1">[68]</xref>. The primers used for RT-PCR are listed in <xref ref-type="supplementary-material" rid="pgen.1004292.s006">Table S2</xref>. Data were normalized with <italic>ACTIN1</italic> level and expressed as the amount in each deletant strain relative to that of H99.</p>