<p>This research was approved by the Institutional Review Boards (IRB) of the National Cancer Institute and Memorial Sloan Kettering Cancer Center. All participants or their parents signed IRB-approved informed consent forms.</p><p>Patient NCI-318 and her family were participants in an IRB-approved longitudinal cohort study at the National Cancer Institute (NCI) entitled &#8220;Etiologic Investigation of Cancer Susceptibility in Inherited Bone Marrow Failure Syndromes&#8221; (NCI 02-C-0052, ClinicalTrials.gov Identifier: NCT00027274). In this study, patients and their family members complete questionnaires and undergo thorough clinical evaluations at the NIH Clinical Center <xref ref-type="bibr" rid="pgen.1003695-Alter1">[2]</xref>. Telomere length was measured by flow cytometry with fluorescent <italic>in situ</italic> hybridization (flow FISH) in leukocytes <xref ref-type="bibr" rid="pgen.1003695-Baerlocher1">[26]</xref>.</p><p>THE MSKCC proband was ascertained on IRB-approved protocol 95-091 entitled &#8220;Collection of Hematopoietic Progenitor Cell and/or Blood Samples From Patients For Research Studies.&#8221; Other family members consented to germline testing in the Clinical genetics Service, as well as MSKCC 93-102 &#8220;Ascertainment of Peripheral Blood or Saliva Samples for Genetic Epidemiology Studies of Familial Cancers,&#8221; as well as a specific consent for the novel homologous recombination gene described in this report.</p><p>Whole exome sequencing for family NCI-318 was performed at the NCI's Cancer Genomics Research Laboratory as previously described <xref ref-type="bibr" rid="pgen.1003695-Ballew1">[6]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Reads were aligned to the hg19 reference genome using Novoalign software version 2.07.14 (<ext-link ext-link-type="uri" ns0:href="http://www.novocraft.com">http://www.novocraft.com</ext-link>), Picard software version 1.67 (<ext-link ext-link-type="uri" ns0:href="http://picard.sourceforge.net/">http://picard.sourceforge.net/</ext-link>) and the Genome Analysis Toolkit (GATK, <ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/gatk/">http://www.broadinstitute.org/gatk/</ext-link>) <xref ref-type="bibr" rid="pgen.1003695-DePristo1">[27]</xref>. Variant discovery, genotype calling, and annotation were performed as described <xref ref-type="bibr" rid="pgen.1003695-Ballew1">[6]</xref> using data from the UCSC GoldenPath database (<ext-link ext-link-type="uri" ns0:href="http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/">http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/</ext-link>), the ESP6500 dataset from the Exome Variant Server, NHLBI Exome Sequencing Project (ESP), Seattle, WA (<ext-link ext-link-type="uri" ns0:href="http://evs.gs.washington.edu/EVS/">http://evs.gs.washington.edu/EVS/</ext-link>) (accessed August 2012), the Institute of Systems Biology KAVIAR (Known VARiants) database (<ext-link ext-link-type="uri" ns0:href="http://db.systemsbiology.net/kaviar/">http://db.systemsbiology.net/kaviar/</ext-link>) <xref ref-type="bibr" rid="pgen.1003695-Glusman1">[28]</xref>, the National Center for Biotechnology Information dbSNP database (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/projects/SNP/">http://www.ncbi.nlm.nih.gov/projects/SNP/</ext-link>) <xref ref-type="bibr" rid="pgen.1003695-Sherry1">[29]</xref> build 137, and the 1000 Genomes (<ext-link ext-link-type="uri" ns0:href="http://www.1000genomes.org/">http://www.1000genomes.org/</ext-link>) <xref ref-type="bibr" rid="pgen.1003695-Consortium1">[12]</xref>. Variants were also annotated for their presence in an in-house database consisting of over 700 whole exomes that were sequenced in parallel with our DC families. Variants within each family were filtered and categorized as indicated in <xref ref-type="supplementary-material" rid="pgen.1003695.s002">Table S1</xref>.</p><p>Validation of exome sequencing findings in the NCI-318 trio was performed by sequencing coding exons of <italic>RTEL1</italic>. Primer sequences are shown in <xref ref-type="supplementary-material" rid="pgen.1003695.s005">Table S4</xref>. All samples were amplified using KAPA2 RobustHotstart Readymix (2&#215;) (Kapa Biosystems, Johannesburg, South Africa) and the following cycling conditions: 3 min at 95&#176;, followed by 30 cycles of 15 sec at 95&#176;, 15 sec at 60&#176;, 15 sec at 72&#176;, followed by 10 min at 72&#176;. Amplicons were purified using Agencourt's Ampure XP beads, then libraries were constructed and barcoded using the Ion Xpress Plus Fragment Library Kit (Life Technologies, Carlsbad, CA, USA). DNA tagged beads were generated for sequencing using Life Technologies' OneTouch and run on an Ion 316 chip on the Ion PGM Sequencer (Life Technologies). The default TMAP aligner and variant caller was used to generate a variant list per sample.</p><p>Targeted resequencing of DNA damage response genes was instrumental in the discovery of the RTEL1 mutation at MSKCC. Genomic enrichment via microfluidic PCR was conducted using the primer pool from Raindance Technologies <xref ref-type="bibr" rid="pgen.1003695-Tewhey1">[30]</xref>. Resulting libraries were prepared for sequencing using the SOLiD 4 sequencer (Life Technologies, Carlsbad). Read alignment and base-calling was done using the ABI Bioscope software with parameters optimal for targeted resequencing. Reads were filtered for mapping quality. <italic>RTEL1</italic> contained the most biologically relevant non-synonymous exonic variant. MSK-41 was included in a panel of 24 cell lines in which targeted DNA sequencing of approximately 300 DNA damage response genes (including <italic>RTEL1</italic>) was carried out (see methods <xref ref-type="bibr" rid="pgen.1003695-Lovejoy1">[13]</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">PolyPhen-2 <xref ref-type="bibr" rid="pgen.1003695-Adzhubei1">[31]</xref> (<ext-link ext-link-type="uri" ns0:href="http://genetics.bwh.harvard.edu/pph2">http://genetics.bwh.harvard.edu/pph2</ext-link>), SIFT <xref ref-type="bibr" rid="pgen.1003695-Kumar1">[32]</xref> (<ext-link ext-link-type="uri" ns0:href="http://sift.jcvi.org">http://sift.jcvi.org</ext-link>), and Condel <xref ref-type="bibr" rid="pgen.1003695-GonzalezPerez1">[33]</xref> (<ext-link ext-link-type="uri" ns0:href="http://bg.upf.edu/condel/home">http://bg.upf.edu/condel/home</ext-link>) were used to predict the severity of RTEL1 amino acid substitutions. Multiple sequence alignments were generated for homologous RTEL1 protein sequences using T-Coffee <xref ref-type="bibr" rid="pgen.1003695-Notredame1">[34]</xref> (<ext-link ext-link-type="uri" ns0:href="http://www.tcoffee.org">www.tcoffee.org</ext-link>) to evaluate conservation. Alignments were generated with NCBI Reference Sequence, GenBank or Ensembl proteins ENSP00000353332 (<italic>Homo sapiens</italic>), NP_001124929.1 (<italic>Pongo abelii</italic>), NP_001091044.1 (<italic>Bos taurus</italic>), and EDL07405.1 (<italic>Mus musculus</italic>).</p><p>Telomere FISH was performed as described <xref ref-type="bibr" rid="pgen.1003695-Attwooll1">[35]</xref>. Images were captured at 100&#215; magnification, with precisely the same exposure time for each genotype (MSK-41 hTERT and BJ hTERT). Sensitivity (gain) is increased to saturation, and chromosome ends for which there still appears no signal are scored as signal-free ends. The heterogeneity observed in <xref ref-type="fig" rid="pgen-1003695-g002">Figure 2C</xref> was reproducible over several experiments, and with different probes (data not shown).</p><p>Cells were collected from 2 to 3 10 cm plates at 70% confluence for each condition. Genomic DNA extraction was performed as described <xref ref-type="bibr" rid="pgen.1003695-MunozJordan1">[36]</xref>. DNA was double digested by AluI/HinfI restriction enzymes overnight before starting TCA assay and then Southern Blot as described <xref ref-type="bibr" rid="pgen.1003695-Zellinger2">[37]</xref> with minor modifications to Phi29 DNA polymerization (MBI Fermentas) with a mammalian telomeric primer and a mammalian telomeric probe for hybridization. Blot images were captured and quantified with Storm 840 scanner and ImageQuant TL software (Amersham Biosciences).</p><p>Sister chromatid exchange analysis and telomere FISH were carried out as described previously <xref ref-type="bibr" rid="pgen.1003695-Attwooll1">[35]</xref>. Mitomycin C sensitivity assays were as described <xref ref-type="bibr" rid="pgen.1003695-Stracker1">[38]</xref>.</p><p>To detect SLX4 levels in the various knockdown conditions, we immunoprecipitated SLX4 (1.5 mg protein lysate, 10 &#181;g of antibody) with rabbit polyclonal antibody (A302-269A) followed by western blotting with polyclonal rabbit antibody A302-270A. Both antibodies were from Bethyl. T-circles were detected and quantified as previously described <xref ref-type="bibr" rid="pgen.1003695-Vannier1">[14]</xref>.</p><p>Immortalized conditional RTEL1F/- MEFs were as previously described <xref ref-type="bibr" rid="pgen.1003695-Vannier1">[14]</xref> and were cultured in DMEM containing 10% fetal bovine serum. Cre recombination was carried out with Ad5-CMV-Cre adenovirus (Vector Biolabs) for 96 hr as described <xref ref-type="bibr" rid="pgen.1003695-Celli1">[39]</xref>. Cells were either not treated or treated with aphidicolin (5 &#181;M) for 24 hrs.</p>