<p>All animal experiments and procedures at Kansas State University were performed according to protocols approved by the Animal Care and Use Committees at Kansas State University (IACUC#: 2961). All animal procedures at Sorbonne University Paris Cit&#233; were performed according to protocols approved by the ethics committee from University Pierre et Marie Curie, and were performed in accordance with the Guide for the Care and Use of Laboratory Animals (NIH publication No. 93-23, revised 1985).</p><p>Human <italic>SLC26A4</italic> cDNA was ligated into a pBluescript vector that contained 6.9 kbp of the human <italic>ATP6V1B1</italic> promoter <xref ref-type="bibr" rid="pgen.1003641-Miller1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1003641-Miller2">[20]</xref>. An SV40 late region polyadenylation signal was cloned downstream of the <italic>SLC26A4</italic> cDNA. The transgene Tg(<italic>B1-hPDS</italic>) included the 5&#8242;-flanking region of the <italic>ATP6V1B1</italic> gene extending to but excluding the endogenous translational start codon, the human <italic>SLC26A4</italic> cDNA, with its own translational start site, and the SV40 late region polyadenylation signal. The integrity of the transgene was confirmed by restriction digest and bidirectional sequencing of ligation sites. In preparation for injection, the transgene was linearized by SalI and NotI digestion, followed by gel purification using an electroelution method and then concentrated using ElutipD columns (Whatman). The transgene was then further concentrated by ethanol precipitation and resuspended in low EDTA injection buffer (10 mM Tris with 0.1 mM EDTA). Tg(<italic>B1-hPDS</italic>) transgenic mice were created by the University of Utah transgenic mouse core facility using standard procedures <xref ref-type="bibr" rid="pgen.1003641-Miller1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1003641-Miller2">[20]</xref>. Genotyping revealed that 63 pups were positive for transgene integration. One founder, which transmitted the transgene in a Mendelian fashion, was crossed with wild-type C57BL/6&#215; CBA F1 mice to establish a colony. Three Tg(<italic>B1-hPDS</italic>);<italic>Slc26a4<sup>+/+</sup></italic> transgenic mice were shipped to Kansas State University in Manhattan, Kansas, USA.</p><p>At Kansas State University, a colony of Tg<italic>(B1-hPDS)</italic>
<sup>Tg/+</sup>;<italic>Slc26a4<sup>&#916;/&#916;</sup></italic> mice was established. Colony management was supported by software (Litter tracker, written in Microsoft Visual Basic and Excel 2010 by P.W.) Tg(<italic>B1-hPDS</italic>)<sup>Tg/+</sup>;<italic>Slc26a4<sup>+/+</sup></italic> mice were crossed with <italic>Slc26a4<sup>&#916;/&#916;</sup></italic> mice to generate Tg(<italic>B1-hPDS</italic>)<sup>Tg/+</sup>;<italic>Slc26a4</italic>
<sup>&#916;/+</sup> mice. Matings of Tg(<italic>B1-hPDS</italic>)<sup>Tg/+</sup>;<italic>Slc26a4</italic>
<sup>&#916;/+</sup> mice generated 28 Tg(<italic>B1-hPDS</italic>)<sup>Tg/+</sup>;<italic>Slc26a4</italic>
<sup>+/+</sup>, 58 Tg(<italic>B1-hPDS</italic>)<sup>Tg/+</sup>;<italic>Slc26a4</italic>
<sup>&#916;/+</sup> and 39 Tg(<italic>B1-hPDS</italic>)<sup>Tg/+</sup>;<italic>Slc26a4</italic>
<sup>&#916;/&#916;</sup> mice in a near Mendelian ratio of 1 &#8758; 2 &#8758; 1 with a 75% rate of transmission for the transgene (based on 169 pups).</p><p>Mice were genotyped for <italic>Slc26a4</italic>
<sup>+</sup> and <italic>Slc26a4</italic>
<sup>&#916;</sup> alleles by PCR using established primers <xref ref-type="bibr" rid="pgen.1003641-Everett1">[14]</xref> and for the transgene <italic>Tg(B1-hPDS)</italic> (Transnetyx, Cordova, TN). Primers for the transgene were designed to amplify a 345 bp PCR-product spanning the <italic>hPDS</italic> cDNA and the SV40 polyadenylation signal sequence (left primer: <named-content content-type="gene">5&#8242;-aga ggg tca agg ttc cat ttt ag-3&#8242;</named-content>; right primer: <named-content content-type="gene">5&#8242;-caa acc aca act aga atg cag tg-3&#8242;</named-content>) <xref ref-type="bibr" rid="pgen.1003641-Thibaut1">[15]</xref>.</p><p>Time-pregnant dams were deeply anesthetized with 4% tri-bromo-ethanol (0.014 ml/g body weight, i.p.) and embryos were harvested by laparotomy. Dams and embryos were sacrificed by decapitation. Gestational age was counted from the day when the vaginal plug was detected. This day was set to embryonic (E) day 0.5. Gestational age, however, was verified by evaluating gross morphological features including limbs, digits and the appearance of the pinna and auditory meatus <xref ref-type="bibr" rid="pgen.1003641-Rugh1">[36]</xref>, <xref ref-type="bibr" rid="pgen.1003641-Theiler1">[37]</xref>.</p><p>The age of mice was counted from the day of birth, which was set to postnatal (P) day 0. Postnatal mice were deeply anesthetized with 4% tri-bromo-ethanol (0.014 ml/g body weight, i.p.) and sacrificed by decapitation or cardiac perfusion with fixative.</p><p>Quantitative RT-PCR was performed on total RNA <xref ref-type="bibr" rid="pgen.1003641-Wangemann2">[17]</xref>. Total RNA was isolated from tissues obtained by microdissection from Tg(&#8722;);<italic>Slc26a4</italic>
<sup>&#916;/+</sup> and Tg(+);<italic>Slc26a4</italic>
<sup>&#916;/&#916;</sup> mice and subjected to quantitative RT-PCR using gene-specific primers for 18S rRNA as well as for mRNA coding for the &#945;-subunit of the mouse Na<sup>+</sup>/K<sup>+</sup> ATPase <italic>Atp1a1</italic>, the B1-subunit of the mouse vH<sup>+</sup>ATPase <italic>Atp6v1b1</italic>, mouse pendrin <italic>Slc26a4</italic> and human pendrin <italic>SLC26A4</italic>, which was introduced via the transgene.</p><p>Postnatal mice were genotyped by PCR prior to tissue collection. Embryonic Tg(&#8722;);<italic>Slc26a4</italic>
<sup>&#916;/+</sup> mice were generated by mating Tg(&#8722;);<italic>Slc26a4</italic>
<sup>&#916;/&#916;</sup> dams and Tg(&#8722;);<italic>Slc26a4</italic>
<sup>+/+</sup> sires, which yielded 100% of the desired genotype. Embryonic Tg(+);<italic>Slc26a4</italic>
<sup>&#916;/&#916;</sup> mice were generated by mating Tg(+);<italic>Slc26a4</italic>
<sup>&#916;/&#916;</sup> dams and sires, which yielded Tg(+);<italic>Slc26a4</italic>
<sup>&#916;/&#916;</sup> and Tg(&#8722;);<italic>Slc26a4</italic>
<sup>&#916;/&#916;</sup> mice in a ratio of 3 &#8758; 1. Since embryonic mice could not be genotyped prior to tissue collection, the desired Tg(+);<italic>Slc26a4</italic>
<sup>&#916;/&#916;</sup> mice among Tg(&#8722;);<italic>Slc26a4</italic>
<sup>&#916;/&#916;</sup> mice were initially identified by visual inspection of the size of the endolymphatic sac and the presence of &#8216;glittering&#8217; otoconia, This phenotypic identification was subsequently confirmed by the presence of human pendrin <italic>SLC26A4</italic> transgene by RT-PCR.</p><p>Tissues were obtained by microdissection. Endolymphatic sacs (8&#8211;10 endolymphatic sacs from 4&#8211;5 animals per sample) were obtained from mice at age E17.5. Cochlear ducts (4 cochlear ducts from 2 animals per sample and 2 cochlear ducts from 1 animal per sample) were obtained from mice at ages E17.5 and P2, respectively. Vestibular labyrinths (6 vestibular labyrinths from 3 animals per sample) were obtained from mice at age P8. Total RNA was isolated from microdissected tissues (RNeasy micro kit, Qiagen, Valencia, CA, USA), treated with DNAse (RNeasy micro kit), combined with RNA storage solution (Applied Biosystems/Ambion, Austin, TX), adjusted to a concentration of 10 ng/&#181;l, and stored at &#8722;80&#176;C.</p><p>Quantity and quality of total RNA were evaluated by microfluidic electrophoresis (BioAnalyzer, Agilent, Santa Clara, CA), by microliter absorption photometry (Nanodrop, Wilmington, DE) and by quantitative RT-PCR of 18S rRNA. RNA samples were accepted for quantitative RT-PCR only when they were free of contamination and excellent RNA quality. RNA quality was quantified by the RNA integrity number (RIN) on a scale from 0 (worst) to 10 (best) (BioAnalyzer, Agilent). RIN numbers for total RNA isolated from E17.5 endolymphatic sac and cochlea were 8.2&#177;0.3 (n&#8202;=&#8202;3) and 9.2&#177;0.1 (n&#8202;=&#8202;10).</p><p>Chemicals were assembled with the assistance of an automatic pipetting station (Biomek NX<sup>p</sup>, Beckman Coulter, Fullerton, CA) with hardware modifications and software programming by P.W. Quantitative RT-PCR reactions were carried out in 96-well plates with each well containing &#8764;10 ng of total RNA, gene specific primers, and an enzyme mix containing reverse transcriptase and DNA polymerase (iScript, BioRad, Hercules, CA) in a total volume of 25 &#181;l. Reverse transcription was performed for 10 min at 50&#176;C and terminated by heating to 95&#176;C for 5 min (OneStepPlus, Applied Biosystems, Foster City, CA). PCR consisted of 40 cycles of 10 s melting at 95&#176;C, 30 s annealing and elongation at 58&#176;C, and 15 s hot-measurement at 78&#176;C (OneStepPlus, Applied Biosystems).</p><p>Left and right primers (exon, product size) were for 18S <named-content content-type="gene">5&#8242;-gag gtt cga aga cga tca ga-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-tcg ctc cac caa cta aga ac-3&#8242;</named-content> (316 bp), for <italic>Atp1a1</italic>
<named-content content-type="gene">5&#8242;-tgc ccg cct caa cat tcc-3&#8242;</named-content> (exon 14) and <named-content content-type="gene">5&#8242;-gac aca tca gag cca aca atc c-3&#8242;</named-content> (exon 16, 291 bp), for <italic>Atp6v1b1</italic>
<named-content content-type="gene">5&#8242;-tga ccc gaa act aca tca cc-3&#8242;</named-content> (exon 1) and <named-content content-type="gene">5&#8242;-gcc aga gcc att gaa aat cc-3&#8242;</named-content> (exon 5, 305 bp), for mouse <italic>Slc26a4</italic>
<named-content content-type="gene">5&#8242;-tct gat gga ggc aga gat ga-3&#8242;</named-content> (exon 20) and <named-content content-type="gene">5&#8242;-ggc cag cct aac aga gac ag-3&#8242;</named-content> (exon 21, 430 bp), and for human <italic>SLC26A4</italic> were <named-content content-type="gene">5&#8242;-tcc caa agt gcc aat cca ta-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-aca tca agt tct tct tcc gtc</named-content> ag-3&#8242; (360 bp). Primer pairs for <italic>Atp1a1</italic>, <italic>Atp6v1b1</italic>, mouse <italic>Slc26a4</italic> spanned introns to prevent amplification of genomic DNA. Primer pairs for mouse <italic>Slc26a4</italic> detected the <italic>Slc26a4<sup>+</sup></italic> allele as well as the <italic>Slc26a4<sup>&#916;</sup></italic> allele that is lacking exon 8 <xref ref-type="bibr" rid="pgen.1003641-Everett1">[14]</xref>. Left and right primers for mouse <italic>Slc26a4</italic> differed by 7 and 10 nucleotides from the corresponding human sequence and left and right primers for human <italic>SLC26A4</italic> differed in 4 and 6 nucleotides from the corresponding mouse sequence, thereby maximizing species-specific amplification. Since the human transgene did not contain introns, some reactions were carried out without reverse transcriptase to determine whether products of <italic>SLC26A4</italic> originated from cDNA rather than from genomic DNA. These experiments revealed no evidence for significant amplification of genomic DNA. Amplification of a single product of the appropriate size was verified by microfluidic electrophoresis (BioAnalyzer, Agilent).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The number of template molecules (<italic>cDNA<sub>Template</sub></italic>) was estimated according to<disp-formula id="pgen.1003641.e001"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1003641.e001.jpg" /></disp-formula>where 6.02&#215;10<sup>23</sup> molecules/mol represents Avogadro's number, <italic>Product<sub>Threshold</sub></italic> is the weight of the PCR-product at threshold (0.49&#215;10<sup>&#8722;9</sup> g) that was obtained from calibration experiments, <italic>Product<sub>Size</sub></italic> is the size of the product in base pairs (bp), <italic>Weight<sub>bp</sub></italic> is average weight of one bp (660 g/mol), <italic>Efficiency</italic> is the PCR-efficiency obtained from the slope of the log-linear phase of the growth curve <xref ref-type="bibr" rid="pgen.1003641-Ramakers1">[38]</xref> and <italic>C<sub>t</sub></italic> is the cycle at which the fluorescence of the product molecules reaches a common threshold chosen in the middle of the log-linear part of the growth curve.</p><p>Bisected heads of embryos age E15.5 were fixed overnight in Bodian's fixative, contained (vol/vol) 75% ethanol, 5% acetic acid, and 5% formalin in water. Heads were then dehydrated overnight in 100% ethanol and cleared in methyl-salicylate <xref ref-type="bibr" rid="pgen.1003641-Kiernan1">[39]</xref>, <xref ref-type="bibr" rid="pgen.1003641-Martin1">[40]</xref>. The membranous labyrinth was injected via the lateral wall of the basal turn of the cochlea and via the endolymphatic sac with diluted paint (Liquid Paper, Newell Rubbermaid, Atlanta, GA, 0.1&#8211;0.2% in methyl-salicylate) using a fine glass-electrode, a manipulator (NM-151 Narishige) and a micrometer-driven oil-filled microinjector (CellTram Vario, Eppendorf, Hamburg, Germany). For each genotype, at least three inner ears were injected.</p><p>Whole mounts of fresh cochlear ducts, endolymphatic sacs and slices of kidney were prepared from E15.5 Tg(<italic>B1-eGFP</italic>) mice and visualized with a fluorescence microscope (AxioScope, Carl Zeiss G&#246;ttingen).</p><p>Soft tissues from 8 cochleae and 8 vestibular labyrinths devoid of endolymphatic sacs from adult mice were collected by microdissection in Cl<sup>&#8722;</sup>-free solution and pooled and homogenized in 500 &#181;l Tris-sucrose buffer. Cl<sup>&#8722;</sup>-free solution contained (mM) 150 Na-gluconate, 1.6 K<sub>2</sub>HPO<sub>4</sub>, 0.4 KH<sub>2</sub>PO<sub>4</sub>, 4 Ca-gluconate<sub>2</sub>, 1 MgSO<sub>4</sub> and 5 glucose, pH 7.4. Tris-sucrose buffer contained (mM) 250 sucrose, 50 Tris-HCl, 5 EDTA, pH&#8202;=&#8202;7.4, and proteinase inhibitor cocktail (Cat# PI-87786 Fisher, Pittsburgh, PA). Homogenization involved a pestle/1.5 ml Eppendorf vial and a Potter-Elvehjem tissue grinder (3 strokes at 1,250 rpm; Cat# K885512-0020, Fisher). Nuclei and debris were sedimented by centrifugation (6 min, 600 g, 4&#176;C, Micromax RF, International equipment, Needham Heights, MA) and the supernatant containing membranes was collected. Pellets were washed 3&#215; with 100 &#181;l Tris-sucrose buffer. Supernatants were pooled and membranes were pelleted by centrifugation (1 hr, 21,000 g, 4&#176;C, Micromax RF). Membranes were suspended in 30 &#181;l Tris-sucrose buffer and protein content was quantified (BCA assay, Cat# 23227, Fisher; Absorption photometer, Nanodrop).</p><p>Kidneys (&#8764;1 g wet weight) were sliced, frozen in liquid N<sub>2</sub>, pulverized, transferred into 8 ml Tris-sucrose buffer and homogenized. Homogenization involved pestles fitting 1.5 ml Eppendorf vials and the Potter-Elvehjem tissue grinder (3 strokes at 1,250 rpm). Nuclei and debris were sedimented by centrifugation (6 min, 600 g, 4&#176;C, Micromax RF) and the supernatant containing membranes was collected. Pellets were washed 3&#215; with 2 ml Tris-sucrose buffer. Supernatants were pooled and membranes were pelleted by centrifugation (1 hr, 21,000 g, 4&#176;C, Micromax RF). Membranes were suspended in 200 &#181;l Tris-sucrose buffer and protein content was quantified (BCA assay).</p><p>Crude membranes were solubilized, denatured and reduced for 1 hr at 37&#176;C in LDS sample buffer (Cat#: NP0007; Invitrogen) supplemented with 50 mM DTT. Proteins were resolved by gel-electrophoresis (4&#8211;12% Bis-Tris gels (Cat#: NP0326, Invitrogen) using a MOPS-SDS running buffer supplemented with an antioxidant (Cat#: NP0005, Invitrogen) and transferred onto PVDF membranes (BioRad, 3 hrs at 30 V). PVDF membranes were blocked for 1 hr at RT in TBST solution containing 3% BSA. TBST solution contained 20 mM Tris-base, 68 mM NaCl, and 1% Tween-20. Blocked PVDF membranes were incubated overnight at 4&#176;C with primary antibody in TBST solution with 3% BSA. Primary antibodies were a rabbit anti-pendrin antibody (Pds #2; 1&#8758;200) and a polyclonal rabbit anti-mouse &#946;-actin antibody (1&#8758; 1,000; Cat#: A2066, Sigma-Aldrich). Pds #2 was raised against the last 15 amino acids of the C-terminal of mouse pendrin <xref ref-type="bibr" rid="pgen.1003641-Choi2">[30]</xref>. Human pendrin differs in only 1 of these 15 amino acids. PVDF membranes were washed 5&#215;3 min in TBST solution and then incubated for 1 hr at RT with the secondary antibody, goat anti-rabbit poly-HRP (Cat#: PI32260, Fisher) diluted in TBST solution with 3% BSA. The dilution of the secondary antibody was 1&#8758;2,500 for labeling the Pds #2 antibody and 1&#8758;10,000 for labeling the &#946;-actin antibody. The membranes were washed 3&#215;5 min in TBST solution. HRP was detected by chemiluminescence (SuperSignal Femto, Cat# PI34095, Fisher). The signal was integrated over 2&#8211;5 min by cooled CCD camera (ImageStation 4000R, Kodak).</p><p>Isolated tissues from embryonic mice were fixed by submersion in PBS-formaldehyde solution at 4&#176;C. PBS-formaldehyde solution consisted of 4% formaldehyde (Electron Microscopy Sciences, Hatfield PA) in a solution that contained (in mM) 137 NaCl, 2.7 KCl, 10.1 Na<sub>2</sub>HPO<sub>4</sub>, 1.8 KH<sub>2</sub>PO<sub>4</sub>. Tissues from postnatal animals were fixed by cardiac perfusion followed by perilymphatic perfusion. Cardiac perfusion was initiated with Cl<sup>&#8722;</sup>-free solution (6 ml, 1 min) and followed by Cl<sup>&#8722;</sup>-free solution with 4% formaldehyde (12 ml, 2 min). Perilymphatic perfusion consisted of an infusion of Cl<sup>&#8722;</sup>-free solution with 4% formaldehyde (25 &#181;l, 5 min) through the round and oval window of the cochlea.</p><p>Fixed temporal bones were decalcified 18&#8211;24 h in 10% EDTA, processed through a sucrose gradient and infiltrated with polyethylene glycol. Mid-modiolar cryosections (12 &#181;m, CM3050S, Leica, Nussloch, Germany) were blocked in PBS-TX (137 mM NaCl, 10.1 mM Na<sub>2</sub>HPO<sub>4</sub>, 1.8 mM KH<sub>2</sub>PO<sub>4</sub>, 2.7 mM KCl, pH 7.4 with 0.15% Triton X 100) and 5% bovine serum albumin. Slides were incubated overnight at 4&#176;C with primary antibody in PBS-TX with 2.5% BSA. Primary antibodies were two rabbit anti-pendrin antibodies (Pds #1; 1&#8758;200; a kind gift from Dr. S&#248;ren Nielsen, University of Aarhus, Denmark and Pds #2, 1&#8758;1,000) and a polyclonal FITC-conjugated goat anti-eGFP antibody (Cat#: GTX26662, GeneTex, San Antonio, TX). Pds #1 was raised against the last 22 amino acids of the C-terminal of mouse pendrin. Human pendrin differs in 6 of these 22 amino acids <xref ref-type="bibr" rid="pgen.1003641-Kim2">[13]</xref>, <xref ref-type="bibr" rid="pgen.1003641-Kim3">[41]</xref>. Slides were washed with PBS-TX and, where appropriate, incubated for 1 h at RT with secondary antibody (1&#8758;1,000, goat anti-rabbit Alexa 594, Invitrogen, Carlsbad, CA). After washing with PBS-TX, sections, where appropriate, were stained with phalloidin 488 (1&#8758;40; Invitrogen) to mark F-actin and DAPI (1&#8758;1,000; Invitrogen) to mark nuclei, washed again and cover-slipped with mounting medium (Vectashield HardSet Mounting Medium, Vector laboratories, Burlingame, CA).</p><p>Tissues were obtained by microdissection from fixed temporal bones, washed in PBS-TX, and blocked with 5% bovine serum albumin in PBS-TX. Tissues were incubated overnight at 4&#176;C with primary antibodies in PBS-TX with 2.5% BSA. Primary antibodies were a rabbit anti-pendrin antibody (Pds #1; 1&#8758;200) or a polyclonal FITC-conjugated goat anti-eGFP antibody (GeneTex). Slides were washed with PBS-TX and, where appropriate, incubated for 1 h at RT with secondary antibody (1&#8758;1,000, goat anti-rabbit Alexa 594, Invitrogen). After washing with PBS-TX, tissues were stained with phalloidin 488 or phalloidin 594 (1&#8758;40; Invitrogen) and DAPI (1&#8758;1000; Invitrogen), washed again and cover-slipped with mounting medium (VectaShield HardSet Mounting Medium, Vector laboratories, Burlingame, CA).</p><p>Immunocytochemistry of cryosections and whole-mounted specimens were viewed by confocal microscopy (LSM 510 Meta, Carl Zeiss, G&#246;ttingen, Germany). Laser scanning bright-field images were collected to aid orientation.</p><p>Mice were anesthetized with 4% tri-bromo-ethanol for <italic>in situ</italic> measurements of the endocochlear potential and pH with double-barreled microelectrodes. Measurements were made in the basal turn of the cochlea by a round-window approach through the basilar membrane of the first turn of the cochlea <xref ref-type="bibr" rid="pgen.1003641-Wangemann3">[18]</xref>, <xref ref-type="bibr" rid="pgen.1003641-Marcus1">[42]</xref>. The surgical cavity was covered with liquid Sylgard 184 (Dow Corning) to limit the loss of tissue CO<sub>2</sub> into ambient air.</p><p>Double-barreled glass microelectrodes were pulled (micropipette puller PD-5; Narishige) from filament-containing glass tubing (1B100F-4; World Precision Instruments) and baked at 180&#176;C for 2 h to ensure dryness. One barrel was silanized by a 30 s exposure to 0.008 ml dimethyldichlorosilane (40136; Fluka). After silanization, microelectrodes were baked again at 180&#176;C for 3 h and tips were broken to a final O.D. of &#8764;3 &#181;m. The reference barrel was filled with 150 mM KCl and the ion-selective barrel was filled at the tip with liquid ion exchanger (Hydrogen ionophore II - Cocktail A, 95297; Fluka) and back-filled with buffer solution (500 mM KCl, 20 mM HEPES, pH 7.4).</p><p>Each barrel of the double-barreled microelectrode was connected via a Ag/AgCl<sub>2</sub> electrode to an electrometer (FD223, World Precision Instruments). A flowing KCl electrode (1 M KCl in 0.2% agar) was inserted under the skin of the animal to serve as ground electrode. Data were recorded in analog (BD12E Flatbed recorder, Kipp &amp; Zonen, Delft, The Netherlands) and digital form (DIGIDATA 1322A and AxoScope 10, Axon Instruments, Union City, CA). pH electrodes were calibrated <italic>in situ</italic> at 37&#176;C using three calibration solutions with different pH values. Calibration solutions contained (in mM): pH 6: 130 NaCl, 20 MES; pH 7: 130 NaCl, 20 HEPES; and pH 8: 130 NaCl, 20 HEPES. pH-sensitive electrodes had a slope of 56.9 &#177; 0.3 mV/pH unit (n&#8202;=&#8202;11).</p><p>Mice were deeply anesthetized with a mixture of dexmedetomidine and ketamine (0.375 mg/Kg body weight dexmedetomidine and 56 mg/kg body weight ketamine; i.p.) and placed on a thermal pad to maintain normal body temperature. The mastoid, vertex and ventral neck region of the animal were connected via sub-dermal platinum needle electrodes (F-E2, Astro Med, Rhode Island, RI) and short (31 cm) leads to the main channel, reference channel and ground of the preamplifier, respectively. Auditory brainstem recordings were performed in a custom constructed, electrically shielded and sound-attenuated chamber (inner dimensions: 23 cm&#215;23 cm&#215;23 cm) using a digital data acquisition system (BioSig32 software, RA4LI Preamplifier, RP2.1 Enhanced Real Time Processor, PA5 Programmable Attenuator, ED1 Electrostatic Speaker Driver, Tucker-Davis Technologies, Alachua, FL). Tone burst stimuli were presented (21 per sec) via a free field electrostatic speaker (SigGen software, ES1 speaker, Tucker Davis). Acoustic stimuli were calibrated using a 1/4 inch condenser microphone (SigCal IRP4.2 software, Tucker Davis, PS9200 microphone, Acoustical Interface, Belmont, CA) placed at the location of the mouse head. Tone bursts (2 ms duration, 0.5 ms gate time; 8, 16 and 32 kHz) were presented with alternating phase (0 and 180&#176;). Responses, recorded over 10 ms, were filtered (300 Hz high pass, 3000 Hz low pass and 60 Hz notch) and 1000 recordings were averaged. Tone burst stimuli were presented at intensities varying between 90 and 0 dB SPL in 5 dB intervals. Auditory thresholds were obtained by a visual comparison of wave forms. After the procedure, mice were rapidly recovered from anesthesia with atipamizole (1.875 mg/kg body weight; i.p.).</p><p>Balance testing consisted of determining the time that mice could balance on a rotating 1&#8243; rod with rotations ramping up from 4 to 40 rpm in 60 s (RotaRod, IITC Life Science, Woodland Hills, CA). Test chambers were cushioned with bubble-foil to provide a soft landing for mice falling off the rod.</p>