<p>The Institutional Review Board at all participating institutions approved the study protocol. This study was conducted according to the principles expressed in the Declaration of Helsinki.</p><p>The Population Architecture using Genomics and Epidemiology (PAGE) study is a National Human Genome Research Institute funded effort examining the epidemiologic architecture of common genetic variants that have been reproducibly associated with human diseases and traits <xref ref-type="bibr" rid="pgen.1002870-Matise1">[35]</xref>. The PAGE study consists of a coordinating center and four consortia with access to large, diverse population-based studies including three National Health and Nutrition Examination Surveys, the Multiethnic Cohort, the WHI, the ARIC study, the Coronary Artery Risk Disease in Young Adults study, the Cardiovascular Health Study, the Hispanic Community Health Study/Study of Latinos, the Strong Heart Study, and the Strong Heart Family Study.</p><p>This PAGE Metabochip study included African American participants from the ARIC and WHI CT studies. Participants from the other PAGE studies were excluded from this effort due to the unavailability of ECG measures and/or genotype data. Genotypes of WHI CT participants were obtained in three phases: two sets of women were directly genotyped on the Metabochip platform by PAGE investigators during wave 1 (n&#8202;=&#8202;797) and wave 2 (n&#8202;=&#8202;1,128) and women (n&#8202;=&#8202;3,531) with Metabochip variants imputed from previous genome-wide SNP data provided by the WHI SHARe <xref ref-type="bibr" rid="pgen.1002870-Liu1">[36]</xref>. Participants meeting the following criteria were excluded from the study: QT unavailable, atrial fibrillation/atrial flutter on ECG, left or right bundle branch block on ECG, QRS duration &gt;120 milliseconds, intraventricular conduction delay on ECG, pacemaker implant antedating ECG, ancestry outlier, excessive heterozygosity, low call rate, or second member of first degree relative pair. Further details on the ARIC and WHI CT studies are provided in <xref ref-type="supplementary-material" rid="pgen.1002870.s012">Text S1</xref> (Participating Studies).</p><p>For each study, certified technicians digitally recorded resting, supine (or semi-recumbent), standard 12-lead ECGs at study baseline for each participant using Marquette MAC PC machines (GE Healthcare, Milwaukee, WI, USA). The ARIC and WHI CT studies used comparable procedures for preparing participants, placing electrodes, recording, transmitting, processing, and controlling quality of the ECGs. QT was measured electronically using the Marquette 12SL algorithm.</p><p>The Metabochip was a custom Illumina iSELECT array that contained approximately 195,000 SNPs and was designed to support large scale follow up of putative associations for cardiovascular and metabolic traits including QT, blood pressure, cholesterol, type 2 diabetes, and anthropometrics. Approximately 33% of the Metabochip SNPs were included as replication targets and 62% for fine-mapping. In total, 257 loci were selected for fine-mapping, with the surrounding regions totaling 45.5 Mb accounting for overlaps (14.2 Mb for the densest fine-mapping regions). Eleven QT loci identified in previous GWA studies in populations of European and Asian ancestry were represented on the Metabochip (<xref ref-type="table" rid="pgen-1002870-t001">Table 1</xref>). The only published QT locus that is not represented on the Metabochip is an intergenic region on 13q14 reported by Marroni et al <xref ref-type="bibr" rid="pgen.1002870-Marroni1">[19]</xref>, but not replicated by other published GWA studies of populations with similar ancestral backgrounds. SNPs reported in the literature but not genotyped on the Metabochip (<italic>NOS1AP</italic>, rs10494366; <italic>NDRG4</italic>, rs7188697, rs37062) were represented by proxies, defined as SNPs in high LD (r<sup>2</sup>&#8805;0.90) with the index SNP using HapMap YRI data.</p><p>Samples were genotyped at the Human Genetics Center of the University of Texas-Houston and the Translational Genomics Research Institute for ARIC and WHI, respectively, following each genotyping center's standard procedures. HapMap YRI (Yoruba in Ibadan, Nigeria) samples were also genotyped independently by each study to facilitate cross-study quality control. Genotypes were called separately for each study, albeit with a common protocol and common personnel, with GenomeStudio using the GenCall 2.0 algorithm. Because the Metabochip includes SNPs with much lower MAFs than are usually called with GenCall, SNPs were recalled using the GenoSNP genotype-calling algorithm <xref ref-type="bibr" rid="pgen.1002870-Giannoulatou1">[37]</xref>. SNPs with call rates &lt;95%, Hardy-Weinberg equilibrium <italic>P</italic>&lt;10<sup>&#8722;6</sup>, &gt;1 Mendelian error (in 30 YRI trios), &gt;2 replication errors, or &gt;3.3% discordant calls in YRI across genotyping centers or against the HapMap database were considered quality control failures. Samples with call rates &lt;0.95 or an inbreeding coefficient F&gt;0.15 were excluded from further analysis <xref ref-type="bibr" rid="pgen.1002870-Buyske1">[38]</xref>.</p><p>Prior to analyses, related participants were identified using PLINK <xref ref-type="bibr" rid="pgen.1002870-Purcell1">[39]</xref> by estimating identical-by-descent statistics for all pairs. When apparent first-degree relative pairs were identified, the member from each pair with the lower call rate was excluded from further analysis. Principal components of ancestry were determined using the Eigensoft software <xref ref-type="bibr" rid="pgen.1002870-Patterson1">[40]</xref>, <xref ref-type="bibr" rid="pgen.1002870-Price1">[41]</xref> and apparent ancestral outliers were excluded from further analysis.</p><p>Briefly, n&#8202;=&#8202;1,962 WHI participants who were genotyped on both the Affymetrix 6.0 and Metabochip genotyping platforms were used to infer Metabochip genotypes to the n&#8202;=&#8202;8,421 population of WHI participants genotyped on the Affymetrix 6.0 array <xref ref-type="bibr" rid="pgen.1002870-Liu1">[36]</xref>. Before phasing and imputation, Affymetrix 6.0 SNPs with genotype call rates &lt;90%, Hardy-Weinberg <italic>P</italic>-values&lt;10<sup>&#8722;6</sup>, or MAF&lt;0.01 were removed. Participants with call rates &lt;95%, those who demonstrated excess heterozygosity, were part of a first-degree relative pair, or who were identified as an ancestry outlier were excluded. This yielded a set of 987,749 SNPs for the 1,962 reference participants. Mean concordance rates for the 23,703 SNPs in common was 99.7%. Haplotypes were reconstructed using MaCH and were used as a reference to impute Metabochip data into the 6,459 WHI participants with only Affymetrix 6.0 data. Liu et al., (2012) demonstrated the ability to impute 99.9% (97.5%, 83.6%, 52.0%, 20.5%) of SNPs with MAF&#8805;0.05 (0.03&#8211;0.05, 0.01&#8211;0.03, 0.005&#8211;0.01, and 0.001&#8211;0.005) with average dosage r<sup>2</sup>&#8202;=&#8202;94.7% (92.1%, 89.0%, 83.1%, and 79.7%), respectively. For this analysis, all imputed SNPs with r<sup>2</sup>&lt;0.95 were excluded.</p><p>To interpret fine-mapping results, LD in our African American PAGE Metabochip sample was calculated in 500 Kb sliding windows using PLINK. In addition, Metabochip LD and frequency information (but not individual-level information) was provided by the Malm&#246; Diet and Cancer Study on 2,143 control participants from a Swedish population <xref ref-type="bibr" rid="pgen.1002870-Berglund1">[42]</xref> to facilitate LD and MAF comparisons to populations of European ancestry. HapMap CEU LD data were used for previously published GWA studies in European populations, as not all European index variants were represented on the Metabochip. Regional association plots use positions from NCBI build 36. Recombination rates were estimated from HapMap phase II data.</p><p>Linear regression models were used to study the association between QT and 6,670 SNPs from 11 regions fine-mapped for QT assuming an additive genetic model and including age, sex, study center, ancestry principal components, and heart rate as covariates. Study-specific association results were combined using an inverse variance meta-analysis approach as implemented in METAL <xref ref-type="bibr" rid="pgen.1002870-Willer1">[43]</xref>.</p><p>For each QT locus, it is expected that SNPs associated with QT in African Americans will be correlated with the index SNP reported in Europeans. Therefore, we first identified and tested SNPs that are correlated (r<sup>2</sup>&#8805;0.20) with the index signals in Europeans using LD statistics estimated in the Malm&#246; Diet and Cancer Study. In order to determine the appropriate multiple testing threshold for declaration of whether the previously identified signals were significantly associated with QT in African Americans, i.e. generalizability, we then estimated the number of tag SNPs needed to capture all common alleles (r<sup>2</sup>&#8805;0.80) using African American LD patterns. The multiple testing threshold for declaring generalization was &#945;<sub>a</sub>&#8202;=&#8202;0.05/415, where 415&#8202;=&#8202;the total number of tags identified using African American LD patterns.</p><p>To identify significant population-specific SNPs influencing QT that were not correlated with the index signal in Europeans (i.e. r<sup>2</sup>&lt;0.20, which was estimated in the Malm&#246; Diet and Cancer Study), we used an efficient Monte Carlo approach that accounts for LD between SNPs at the previously identified QT loci (&#945;<sub>b</sub>&#8202;=&#8202;1.37&#215;10<sup>&#8722;5</sup>) <xref ref-type="bibr" rid="pgen.1002870-Lin1">[44]</xref>. Conditional analyses were then performed to determine the number of independent signals the population-specific SNPs represent. Specifically, analyses were repeated for each locus including the SNP with the smallest <italic>P</italic> &#8211; value as a covariate. This approach was performed adjusting for successively less significant SNPs until no SNPs with <italic>P</italic> &#8211;values lower than &#945;<sub>b</sub>&#8202;=&#8202;1.37&#215;10<sup>&#8722;5</sup> were identified. To facilitate comparability with previous reports examining the proportion of variance in QT explained by common SNPs, heart rate-corrected QT <xref ref-type="bibr" rid="pgen.1002870-Rautaharju1">[45]</xref> was regressed on the six best markers in African Americans and the three population-specific variants assuming an additive genetic model and including age, sex, study center, ancestry principal components as covariates.</p><p>For each of the nine QT SNPs (i.e. the six best markers in African Americans and the three novel SNPs), we identified all SNPs in LD (r<sup>2</sup>&#8805;0.5) using the genotypes from the African American population described in this study. We refer to these SNP sets as Trait Associated SNP (TAS) blocks. We assigned each TAS to one or more of the functional annotation datasets listed in <xref ref-type="supplementary-material" rid="pgen.1002870.s009">Table S5</xref>. These datasets are not mutually exclusive. For example, a TAS can reside in both a candidate regulatory element (dataset #7) and a CTCF binding site (dataset #10). For TASs that occur within predicted transcription factor binding sites (datasets #3 and #8), we calculated transcription factor binding affinity for each TAS allele using PWM-scan <xref ref-type="bibr" rid="pgen.1002870-Levy1">[46]</xref>, as described previously <xref ref-type="bibr" rid="pgen.1002870-Sethupathy1">[47]</xref>. For TASs that occur within 3&#8242; untranslated regions, we used the TargetScanS algorithm to determine whether they disrupt likely microRNA target sites (dataset #5). To define candidate non-promoter regulatory elements of greatest relevance to QT (dataset #7), we restricted the analysis of DNase I hypersensitive sites (open chromatin loci) to only those present in human cardiomyocytes.</p>