<p>All studies were approved by local research ethics committees, and all participants gave informed consent. All procedures in human islets were approved by the ethics committees at the Uppsala and Lund Universities and informed consent obtained by appropriate measures from donors or their relatives.</p><p>Association analyses of insulin secretion and action traits were performed within 11 cohorts participating in the Meta-Analysis of Glucose- and Insulin-related traits Consortium (MAGIC) in a total of up to 10,831 individuals. In the discovery stage 1, we performed a meta-analysis of 6 GWASs (Diabetes Genetics Initiative (DGI), Amish Family Diabetes Study, Sorbs, Helsinki Birth Cohort Study (HBCS), French Obese Adults, and Relationship between Insulin Sensitivity and Cardiovascular disease Study (RISC)) for glucose-stimulated insulin secretion (GSIS) during an oral glucose-tolerance test (OGTT) at 3 time points (fasting, 30 min, 120 min) for primary traits measured as (1) insulin response to glucose after the first 30 min estimated as corrected insulin response (CIR), and (2) overall insulin response to glucose estimated as area under the curve (AUC) for insulin over a total AUC for glucose (AUC<sub>Ins/</sub>AUC<sub>Gluc</sub>) in up to 5,318 non-diabetic individuals (<xref ref-type="supplementary-material" rid="pgen.1004235.s009">Table S1A</xref>).</p><p>As none of the traits gave genome-wide significant association, we selected the top 50 independent signals from both primary and secondary traits (see &#8220;Phenotype Definition&#8221; below) after LD pruning (r<sup>2</sup>&lt;0.2). Signals prioritized for replication were ranked by the number of associations observed at primary traits and/or secondary traits, association p-value and number of times the signal was observed across the traits (more than 2). We selected 14 SNPs for replication genotyping and follow-up analyses, out of which 3 loci were based on biological relevance: <italic>GRB10</italic>
<xref ref-type="bibr" rid="pgen.1004235-Rampersaud1">[13]</xref> (rs933360, discovery <italic>p</italic>-value (CIR)&#8202;=&#8202;5.09&#215;10<sup>&#8722;6</sup>), <italic>UCN3</italic> (rs11253130, discovery <italic>p</italic>-value (Ins30adjBMI)&#8202;=&#8202;9.46&#215;10<sup>&#8722;7</sup>) and <italic>INADL</italic> (rs2476186, discovery <italic>p</italic>-value (AUC<sub>Ins</sub>/AUC<sub>Gluc</sub>)&#8202;=&#8202;1.88&#215;10<sup>&#8722;6</sup>). Replication stage 2A <italic>de novo</italic> genotyping was undertaken in five population-based studies (Botnia-PPP, ULSAM, METSIM, BPS and Haguenau; only GWAS index SNP rs933360 in the latter three; max N&#8202;=&#8202;15,273) (<xref ref-type="supplementary-material" rid="pgen.1004235.s009">Table S1A</xref>, <xref ref-type="supplementary-material" rid="pgen.1004235.s010">S2A</xref>). Replication stage 2B <italic>in silico</italic> was undertaken using an iSelect CardioMetabochip array (CM) (Illumina, San Diego, CA, USA) to genotype data in 5 independent population-based studies (Botnia-PPP, ULSAM, Ely, DR's Extra and METSIM) including up to 5,513 individuals (<xref ref-type="supplementary-material" rid="pgen.1004235.s009">Table S1A</xref>).</p><p>The GWAS/CM (stage 1 and stage 2B) data including 93,896 SNPs were pooled together with the <italic>de novo</italic> genotyping results from stage 2A for non-overlapping individuals. In this meta-analysis, we defined all independent (r<sup>2</sup>&lt;0.2) genome-wide significant (<italic>p</italic>-value&lt;5&#215;10<sup>&#8722;8</sup>) association signals for insulin secretion traits at 8 genomic loci (<xref ref-type="table" rid="pgen-1004235-t001">Table 1</xref>).</p><p>The primary insulin secretion and action indices were: (i) Corrected Insulin Response (CIR)&#8202;=&#8202;(100&#215; insulin at 30 min)/(glucose at 30 min&#215;(glucose at 30 min&#8211;3.89)), and (ii) ratio of the area under the curve (AUC) for AUC insulin/AUC glucose (AUC<sub>Ins</sub>/AUC<sub>Gluc</sub>, mU/mmol) calculated using the trapezium rule <xref ref-type="bibr" rid="pgen.1004235-Matthews1">[28]</xref>. Insulin sensitivity index (ISI)&#8202;=&#8202;10,000/&#8730; (fasting plasma glucose (mg/dl)&#215;fasting insulin&#215;mean glucose during OGTT (mg/dl)&#215;mean insulin during OGTT). Secondary insulin secretion and action indices during OGTT were: (i) disposition index (DI)&#8202;=&#8202;CIR&#215;ISI; (ii) insulin at 30 min (Ins30); (iii) incremental insulin at 30 min (Increm30)&#8202;=&#8202;insulin 30 min &#8211; fasting insulin; (iv) insulin response to glucose during the first 30 min adjusted for BMI (Ins30adjBMI)&#8202;=&#8202;insulin at 30 min/(glucose at 30 min&#215;BMI); (v) area under the curve (AUC) of insulin levels during OGTT (AUC<sub>Ins</sub>, mU*min/l). Individuals with missing data on any of the three time points included in the AUC calculation were excluded.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Linear regression models were used for association of phenotypes (z-score residuals of insulin secretion and action traits) with genotypes coded additively. Discovery (stage 1) GWAS analyses were carried out using a statistical tool that was able to account for genotype uncertainty, SNPTEST <xref ref-type="bibr" rid="pgen.1004235-Marchini1">[29]</xref>, or by using allele dosages in the linear regression model in MACH2QTL <xref ref-type="bibr" rid="pgen.1004235-Li1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1004235-Li2">[31]</xref>, probABEL <xref ref-type="bibr" rid="pgen.1004235-Aulchenko1">[32]</xref>, corrected for residual inflation of the test statistics using the genomic control method <xref ref-type="bibr" rid="pgen.1004235-Devlin1">[33]</xref>. The meta-analyses of effect sizes were performed with the fixed-effect inverse-variance method using GWAMA <xref ref-type="bibr" rid="pgen.1004235-Mgi1">[34]</xref>. The GC correction was applied only once to cohort-specific results before including them into the meta-analyses. Sex-differentiated analyses were performed using GWAMA, with an assumed heterogeneity <italic>p</italic>-value of &lt;0.05. Effect sizes for glucose levels were estimated using a fixed-effect model using the metaphor package for R version 2.14.2 (<ext-link ext-link-type="uri" ns0:href="http://www.r-project.org/">http://www.r-project.org/</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The Trios from Finland and Sweden, Amish Family Diabetes Study and Kuopio Offspring Study (<xref ref-type="supplementary-material" rid="pgen.1004235.s009">Table S1A</xref>, <xref ref-type="supplementary-material" rid="pgen.1004235.s011">S3A</xref>, B) consisted of a father, a mother and an offspring. Genotype phase was determined using Merlin and then analyzed using Solar, which uses the kinship matrix to account for family. Meta-analysis on insulin secretion and action and glucose levels during OGTT (as earlier described) was performed using a fixed-effect model. Analyses were performed on IBM SPSS Statistics 20.0 (IBM Corp., Chicago, IL, USA), R version 2.14.2 with the metaphor package (<ext-link ext-link-type="uri" ns0:href="http://www.r-project.org/">http://www.r-project.org/</ext-link>) and MMAP (MMAP: mixed models analysis for pedigrees and populations, <ext-link ext-link-type="uri" ns0:href="http://edn.som.umaryland.edu/mmap/index.php">http://edn.som.umaryland.edu/mmap/index.php</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The Transmission Disequilibrium Test (TDT) used to compare frequencies of transmission of the two alleles from heterozygote parents to an affected offspring was performed using PLINK (<ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/purcell/plink/">http://pngu.mgh.harvard.edu/purcell/plink/</ext-link>) <xref ref-type="bibr" rid="pgen.1004235-Purcell2">[35]</xref> (<xref ref-type="supplementary-material" rid="pgen.1004235.s011">Table S3A</xref>). The deviations from Mendelian transmissions were assessed and the power of the test was enhanced by incorporating information from phenotypically discordant parents (ParenTDT) <xref ref-type="bibr" rid="pgen.1004235-Purcell1">[14]</xref>. To confirm the association, another independent test was performed, which can accommodate any type of genetic model and family construction, i.e. the Family Based Association Test (FBAT) <xref ref-type="bibr" rid="pgen.1004235-Horvath1">[15]</xref>. Quantitative traits related to glucose metabolism and insulin secretion were assessed in non-diabetic individuals using qTDT and parent-of-origin effect tests. IBD estimates were calculated using Merlin. Permutations were performed using QFAM (PLINK). OGTT values were natural log-transformed and adjusted for BMI.</p><p>The Botnia Prevalence, Prediction and Prevention of diabetes (Botnia-PPP) study is a population-based study from the Botnia region of Western Finland and has previously been described <xref ref-type="bibr" rid="pgen.1004235-Isomaa1">[36]</xref>. For this study, we selected 4,641 non-diabetic individuals above the age of 18. Linear regression analysis assuming an additive genetic risk model was performed to evaluate genotype-phenotype association. Hyperglycemic individuals were identified based on previous diagnosis, fasting plasma glucose levels of 5.5&#8211;6.9 mmol/l and 2 hr plasma glucose levels of 7.8&#8211;11.1 mmol/l (<xref ref-type="supplementary-material" rid="pgen.1004235.s013">Table S5</xref>).</p><p>A subgroup of 203 men with IGT at screening visit selected from the MPP study participated 20 years later in more extensive metabolic studies, including a new OGTT, a euglycemic-hyperinsulinemic clamp combined with indirect calorimetry and infusion of [3-3H]glucose <xref ref-type="bibr" rid="pgen.1004235-Eriksson1">[37]</xref>, <xref ref-type="bibr" rid="pgen.1004235-Eriksson2">[38]</xref>. The men were similar in age, but had varying degrees of glucose tolerance; 69 were in the normal range, 52 had IFG and/or IGT, and 82 had T2D. T2D patients were either treated with diet alone (42%) or with oral hypoglycemic agents, which were withheld the day before the test. Microarray expression data were analyzed as previously described <xref ref-type="bibr" rid="pgen.1004235-Mootha1">[20]</xref>.</p><p>The Malm&#246; Preventive Project (MPP) is a large population-based prospective study from the city of Malm&#246;, Sweden, and has previously been described <xref ref-type="bibr" rid="pgen.1004235-Nilsson1">[17]</xref>. For this study, we selected 16,715 non-diabetic subjects, of whom 2,637 developed T2D during a 24.1 year mean follow-up period. The odds ratio for risk of developing T2D was calculated using logistic regression analysis assuming an additive genetic risk model. The analysis was adjusted for age, sex, BMI, participation period and an interaction term (participation period x sex) <xref ref-type="bibr" rid="pgen.1004235-Lyssenko1">[10]</xref>. IBM SPSS Statistics 20.0 (IBM Corp.) was used for the statistical analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Islets from cadaver donors were provided by the Nordic Islet Transplantation Program (<ext-link ext-link-type="uri" ns0:href="http://www.nordicislets.org">www.nordicislets.org</ext-link>) by the courtesy of Prof. Olle Korsgren, Uppsala University, Sweden. The microarray experiments (Human Gene 1.0 ST whole transcript) were performed on islets isolated from 81 normoglycemic (mean&#177;SEM; age 56.3&#177;1.3 yrs, BMI 25.7&#177;0.4 kg/m<sup>2</sup>, HbA1c 5.5&#177;0.04%) and 46 hyperglycemic (age 60.3&#177;1.2 yrs, BMI 27.8&#177;0.6 kg/m<sup>2</sup>, HbA1c 6.6&#177;0.1%) islet donors. RNA products were fragmented and hybridized to the GeneChip Human HG U 133A Array (Affymetrix, Santa Clara, CA, USA) <xref ref-type="bibr" rid="pgen.1004235-Taneera1">[39]</xref>. Statistical analyses of expression data were performed using two-tailed Spearman's T-test.</p><p>Immunocytochemistry with antibodies for GRB10 (K20)<sup>18</sup> (code sc-1026, Santa Cruz Biotech. Inc., CA, USA), insulin, glucagon and somatostatin was performed on human pancreatic sections as previously described <xref ref-type="bibr" rid="pgen.1004235-Riva1">[40]</xref>.</p><p>RNA from 48 human pancreatic islets donors (24 with HbA1c &lt;5.5% and 24 donors with HbA1c &gt;6.0%) was isolated and purified using miRNeasy kit (Qiagen, Hilden, Germany). As quality thresholds for RNA samples, we demanded RIN values &gt;8, 28S/18S ratio &gt;1.5 and the absorbance ratios 260/280 &gt;1.8 and 260/230 &gt;1. Sample preparation for sequencing reactions was performed using TrueSeq sample prep kit (Illumina). Fragmentation was performed using inbuilt fragmentation in the sample prep kit to obtain fragments of approximately 300 bp in length. Sequencing was performed on the Illumina HiSeq 2000 platform. Obtained sequenced reads were transformed into .qseq files using the Illumina pipeline. Alignment of the reads was performed using the TopHat short read aligner (tophat.cbcb.umd.edu). Cufflinks (cufflinks.cbcb.umd.edu) was used for splice variant calling.</p><p>Analytical RT-PCRs were performed on cDNA from human islet, visceral fat, subcutaneous fat, liver and muscle in 15 &#181;l reactions using 7.5 &#181;l AmpliTaq Gold PCR Master Mix (Applied Biosystems, Foster City, CA, USA) supplemented with 0.75 &#181;l DMSO and 1.5 &#181;mol/l of forward and reverse primers. The following primers were used in various combinations: UN1fw: <named-content content-type="gene">CAAACGCCTGCCTGACGACTG</named-content>, UN1Afw: <named-content content-type="gene">GCCCGGGACAGTCTTGAGC</named-content>, UN2fw: <named-content content-type="gene">GGCGCACACGCAGCGAC</named-content>, UN3fw: <named-content content-type="gene">ACCACCTACATCAGAGCTGACTGCC</named-content>, 1bfw: <named-content content-type="gene">CCTGGGCTACCCTCTGCTTC</named-content>, 3fw: <named-content content-type="gene">GCCTGTACTCGGCCTGCAGC</named-content>, 9fw: <named-content content-type="gene">GCCCCTACAGACCACGGGCT</named-content>, 11fw: <named-content content-type="gene">GCTGTCCCCGTTCTCGACGC</named-content>, 3rv: <named-content content-type="gene">ATGTGCACAGGCTGGGAGCG</named-content>, 7rv: <named-content content-type="gene">CTGGCTGTCATGTCTGCT</named-content>, 11rv: <named-content content-type="gene">CTGCTGAGGGATTCGGT</named-content>, 16rv: <named-content content-type="gene">GGATGCAGTGGTGCTTGA</named-content>, the names referring to the target exon. PCR reactions were carried out with 53&#176;C annealing temperature and over 50 cycles. Products were analyzed on 2% agarose. Prior to sequencing, 2.5 &#181;l PCR product was treated with 0.5 &#181;l ExoSAP-IT (USB, Cleveland, OH, USA) at 37&#176;C for 30 min followed by deactivation at 80&#176;C for 15 min. Subsequently, 1 &#181;l was sequenced in both directions using BigDye 3.1 according to the manufacturer's protocol (Applied Biosystems). The sequence reactions were purified and analyzed by GATC Biotech AG (Konstanz, Germany).</p><p>Allelic imbalance measurements were performed by RT-PCR using the reverse primer 3rv and either of the forward primers UN1afw, UN1fw, UN2fw, UN3fw, 1bfw or 3fw in samples heterozygous for the common SNP rs1800504 (<italic>GRB10</italic> exon3). The individual contribution from each allele was measured at the position of rs1800504 using Sanger sequence traces and the software Mutation Surveyor (Soft Genetics, PA, USA). Allelic imbalance in <italic>GRB10</italic> was also detected by a different method: After extensive quality and coverage filtering, we did a Fisher exact test for comparing the ratio of reference/alternative alleles in the exome sequencing vs. RNA-seq for each sample. Exome sequencing was performed using the Illumina exome sequencing protocols (TruSeq DNA sample preparation Kit v2).</p><p>Sequenom's MassARRAY EpiTYPER protocol was applied to measure DNA methylation (Sequenom, San Diego, CA, USA) in human islets of 96 donors and peripheral blood lymphocytes (PBL) of 6 diabetic offspring trios (18 individuals). EpiDesigner was used for assay design at <italic>GRB10</italic> and the primer sequences were the following; forward: <named-content content-type="gene">aggaagagagGGGAAAGGGTGTTAAATTGTTTATG</named-content>, reverse: <named-content content-type="gene">cagtaatacgactcactatagggagaaggctTTTTAAACCCCTCAAATTCAAAAAT</named-content>. 500 ng genomic DNA was bisulfite-treated with the EZ DNA Methylation kit (Zymo Research, Orange, CA, USA). The spectra were analyzed and the methylation ratios were obtained by the EpiTYPER software v.1.0.1. Global methylation analyses were performed on DNA extracted from PBL on the Illumina Infinium 450 Bead Chip and the chips were scanned on an Illumina iScan as per protocol (Illumina). 1 &#181;g of DNA was bisulfite-treated according to protocol (EZ DNA Methylation Kit, Zymo research). For analysis, the Genome Studio Methylation Module of the Genome studio Genome Browser was used, which facilitates integration of the SNP and CpG location data (NCBI build 37). Methylation status was assessed after normalization to internal controls and background subtraction and expressed as &#946;. The &#946; values for the CpG sites were mapped to the gene and plotted to give an overview of methylation status for the region of interest (<xref ref-type="supplementary-material" rid="pgen.1004235.s007">Figure S7D</xref>).</p><p>Specific silencing of endogenous hGRB10 was achieved using a lentiviral-based shRNA-silencing technique (Santa Cruz Biotech. Inc.). Isolated human islets were incubated at 2.8 mmol/l glucose + Polybree for 90 min. Thereafter, the medium was removed and the islets were washed before culture medium + lentiviral particles containing <italic>GRB10</italic> shRNA (5 &#181;l/ml) was added, and the islets were cultured for 36 h at 5 mmol/l glucose. For comparison, a scramble (lentiviral particles without targeting any specific region) served as control. Insulin and glucagon were measured using a radioimmunoassay kit (Electrobox, Stockhom, Euro-Diagnostica, Malm&#246;, Sweden).</p><p>For <italic>Grb10</italic> small interfering RNA (siRNA) experiments, 20&#8211;25-nucleotide stealth-prevalidated siRNA duplex designed for rat <italic>Grb10</italic> (Santa Cruz Biotech. Inc.) was used. INS-1 832/13 cells were seeded in 24-well plates at a density of &#8764;5&#215;105 cells in culture media without antibiotics and transfected with DharmaFECT 1 (Dharmacon, Lafayette, CO, USA) according to the manufacturer's instructions.</p><p>Human pancreatic &#946;-cell viability assay was performed using a CellTiter 96 AQueous One Solution Cell Proliferation Assay Reagent (Promega, Stockholm, Sweden) according to the manufacturer's instructions. The actual performance is based on the spectrophotometric detection of a colored formazan product converted from a 3-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium (MTS) compound by NADPH or NADH via metabolically active cells.</p>