<p>All <italic>S.cerevisiae</italic> strains used in this study (see <xref ref-type="supplementary-material" rid="pgen.1003736.s001">Table S1</xref> for genotypes) are derived from 1B-D770 <xref ref-type="bibr" rid="pgen.1003736-Shcherbakova4">[69]</xref>. The mutant <italic>ura3&#8211;4</italic> allele in this strain was reverted to wild type by transformation with wild type <italic>URA3</italic> DNA obtained by PCR, yielding the LAN201 strain. LAN211 is an auto-diploid of LAN201 obtained by HO endonuclease expression followed by selection for diploids. The haploid <italic>ung1</italic>-deficient strain LAN200 was described previously <xref ref-type="bibr" rid="pgen.1003736-Lada4">[62]</xref>. Auto-diploidization of LAN200 resulted in the diploid <italic>ung1</italic> strain LAN210.</p><p>Standard yeast media were used <xref ref-type="bibr" rid="pgen.1003736-ShermanF1">[70]</xref>. For selection of mutants we have used synthetic complete (SC) agar plates without arginine with 60 mg/L of L-canavanine or 0.1% of FOA. For induction of deaminase expression, minimal synthetic media with addition of 1% raffinose and 2% galactose was used.</p><p>Mutation frequencies were determined by fluctuation analysis as described previously <xref ref-type="bibr" rid="pgen.1003736-Shcherbakova4">[69]</xref>. For the HAP experiment, independent LAN201 or LAN211 clones were grown in rich YPD media overnight. HAP was added to the media, where applicable, to a final concentration 50 &#181;g/ml. After overnight incubation at 30&#176;C, cultures were plated undiluted on synthetic complete media with canavanine (SC+CAN) to select for <italic>can1</italic> mutants, and with dilution to complete (SC) plates to estimate viability. The <italic>CAN1</italic> gene encodes arginine permease, which transports the toxic arginine analog canavanine into cells. Inactivation of <italic>CAN1</italic> renders cells resistant to canavanine.</p><p>For the PmCDA1 experiments, plasmid pESC-<italic>LEU</italic>-PmCDA1 was constructed as follows. Total RNA was extracted from the blood of sea lamprey (<italic>Petromyzon marinus</italic>) and reverse-transcribed with oligo (dT). <italic>PmCDA1</italic> was amplified with primers NotICDA1N-F (<named-content content-type="gene">5&#8242;-TTTGCGGCCGCACCATGACCGACGCTGAGTAC</named-content>, location 118&#8211;135 in GenBank accession EF094822) and SpeICDA1C-R (<named-content content-type="gene">5&#8242;- TTTACTAGTGCAACAGCAGGACTCTTAGTG</named-content>, location 724&#8211;742 in EF094822) and cloned into pESC-LEU vector (Stratagene). For yeast experiments, LAN200 or LAN210 strains were transformed with the pESC-LEU-PmCDA1 expression plasmid or with the vector only <xref ref-type="bibr" rid="pgen.1003736-Rogozin1">[31]</xref>, <xref ref-type="bibr" rid="pgen.1003736-Lada2">[37]</xref>. Colony-purified transformants were inoculated in 5 ml of synthetic liquid media without leucine containing 1% raffinose. After overnight incubation, galactose was added to cultures at a final concentration of 2%. Galactose activates the GAL1-10 promoter in the pESC-LEU vector which induces the expression of deaminase. After one day of incubation, culture suspensions were plated undiluted on SC+CAN and with dilution on complete plates.</p><p>LAN201 and LAN211 were streaked on YPDAU plates and grown overnight. The next day, they were replica-plated on fresh YPD plates, and a drop of HAP was added to sterile filter paper placed on the agar surface so that different yeast patches receive a similar HAP dose. The next day, streaks were replica-plated on SC+CAN plates to select for mutants. One Can<sup>R</sup> colony was picked from one streak, then colony-purified and frozen as a glycerol stock at &#8722;80&#176;C. To obtain non-mutant HAP-treated clones, yeast from YPD plates with HAP (the same plates used to obtain Can<sup>R</sup> clones) were streaked on YPDAU plates without HAP and then colony-purified. All of the isolated HAP-treated, non-mutant clones were confirmed to be Can<sup>S</sup>.</p><p>In the PmCDA1 experiments, LAN200 and LAN210 were transformed with pESC-LEU-PmCDA1. Individual transformants were then inoculated in 5 ml of liquid synthetic media containing glucose and without leucine, followed by incubation for one day at 30&#176;C with shaking. Cells were then pelleted, washed once with sterile water, then resuspended in 12 ml of synthetic media without leucine containing 2% galactose and 1% raffinose, followed by incubation for 3 days at 30&#176;C with shaking. Aliquots of the resulting yeast suspensions were plated on synthetic complete media containing canavanine to select for <italic>can1</italic> mutants. Aliquots of diluted cultures were plated on synthetic complete (SC) plates to estimate viability. Individual CAN<sup>R</sup> colonies (one per each independent culture) were colony-purified and stored at &#8722;80&#176;C. PmCDA1-treated non-mutant clones were arbitrarily picked up from SC plates. These clones were confirmed to be Can<sup>S</sup>.</p><p>We used the method described in <xref ref-type="bibr" rid="pgen.1003736-Otero1">[71]</xref> with slight modifications. Cells were collected from 30 ml of saturated culture (OD<sub>600</sub>&#8776;10) grown in YPDAU medium, washed once with water, and resuspended in 3 ml of lysis buffer (0.1 M Tris-HCl pH 8.0, 50 mM EDTA, 1% SDS). Then 150 &#181;l of 5 M NaCl and &#8764;1.2 ml of glass beads were added to the suspension. Cells were disrupted by vortexing (2 cycles, 2 min each) in a cold room and then the lysate was centrifuged (13,000 <italic>g</italic>, 10 min). DNA was purified from the supernatant using phenol-chloroform extraction followed by ethanol precipitation. The DNA pellet was dissolved in DNA-grade water and treated with RNAse A (Qiagen, 10 &#181;l of 10 mg/ml per sample, 1 h at 37&#176;C). DNA was purified again by phenol-chloroform extraction followed by ethanol precipitation, and finally resuspended in DNA-grade water. The concentration and quality of DNA preparations were monitored by agarose gel electrophoresis and the use of a NanoDrop spectrophotometer (Thermo Scientific) and a Qubit fluorometer (Invitrogen).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Isolated yeast genomic DNA was used to construct fragment libraries using the recommended kits for sequencing on the UNMC NGS Core Laboratory's HiSeq 2000 instrument. We multiplexed individual yeast libraries, each derived from an individual clone, in a single lane of an Illumina flow cell. Each of the yeast genomes was sequenced at 100&#215; to 300&#215; coverage (depending on the run) by sequencing 101 bp from each end of the individual DNA fragments in the library (101 bp paired-end sequencing), according to Illumina's recommendations. During the instrument run and after sequencing of the yeast libraries was completed, a variety of quality assurance (QA) measurements were made to ensure the integrity of the DNA sequence data. The DNA &#8220;bar codes&#8221; used for multiplexing were first used to partition the reads into their respective sample-specific bins, and then the bar codes were stripped from the reads to yield sample-specific DNA sequences of interest. Base-calling error correction was performed on each sample-specific set of de-multiplexed raw reads using Quake <xref ref-type="bibr" rid="pgen.1003736-Kelley1">[72]</xref>. Raw Illumina resequencing data for the LAN211 strain and for the various mutant and non-mutant clones were deposited in the NCBI Sequence Read Archive (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/sra">www.ncbi.nlm.nih.gov/sra</ext-link>, accession numbers SRA057025 and SRP014741).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">About ten million pair-end reads generated by sequencing of the whole-genome library obtained from the LAN201 reference strain were used for <italic>de novo</italic> genome assembly using CLC Bio's Genomics Workbench software (CLC Bio, Aarhus, Denmark). This resulted in 458 contigs of various lengths (from 200 to 217,386 bp). These contigs were aligned to the genome of the standard yeast strain S288C using batch BLAST. After sorting of the contigs by chromosome, each set was scaffolded (ordered and oriented) against the corresponding chromosome using Geneious Pro software (Biomatters Ltd, Auckland, New Zealand) <xref ref-type="bibr" rid="pgen.1003736-Drummond1">[73]</xref>. Consensus sequences were extracted from the scaffolds and used in the next step of reference genome assembly. Raw sequencing data (the same 10 million reads that were used for the <italic>de novo</italic> assembly) were then assembled to the extracted consensus sequences, and SNVs were detected using Geneious Pro. After manual identification of false positives and the correction of alignments, a new consensus was obtained. This &#8220;version 1&#8221; LAN201 draft genome assembly covers 92.74% of the standard S288C yeast genome downloaded from the Saccharomyces Genome Database (SGD, <ext-link ext-link-type="uri" ns0:href="http://www.yeastgenome.org">www.yeastgenome.org</ext-link>) in October 2011. This draft assembly has a GC content of 38.24% (compared with 38.31% for S288C; see <xref ref-type="supplementary-material" rid="pgen.1003736.s002">Table S2</xref>). LAN211 is identical to LAN201 except for the mating type locus. To obtain references of <italic>ung1</italic>-deficient strains (LAN200 and LAN210), sequencing reads corresponding to LAN210 were assembled on the LAN211 draft in Geneious Pro. SNVs were called, manually checked, and then a consensus sequence was deduced. This resulted in a LAN210 draft genome assembly. The re-assembly of the LAN200 reads on the LAN210 reference followed by SNV calling confirmed that the two strains are isogenic. These draft genome assemblies were used to analyze the genomes of mutant clones. <italic>UNG1</italic> and <italic>ung1</italic> reference genomes are compared in <xref ref-type="supplementary-material" rid="pgen.1003736.s002">Table S2</xref>.</p><p>Ten to 20 million pair-end reads per clone were comparatively assembled on the LAN211 (for HAP) or LAN210 (for PmCDA1) reference genomes using Geneious Pro. SNVs were called using homozygous (SNV frequency &#8805;80%) and heterozygous (40% &#8804; SNV frequency &#8804;80%) modes. The threshold SNV call frequencies (40% and 80%) were selected based on pilot experiments designed to optimize the detection of true SNVs and reduce the number of false positives. Regions of high and low coverage (more than two standard deviations from the mean) were excluded from the analysis. At this point, a fraction of non-expected substitution types was observed in the genomes. These included non-G-C&#8594;A-T and non-A-T&#8594;G-C mutations in HAP-treated genomes and non-G-C to A-T mutations in PmCDA1-treated genomes. The majority of these SNVs were found in the regions where reads were clearly misaligned to the reference (i.e. having low mapping quality). The rest of the non-expected SNVs were found in the otherwise good regions of assembly. We PCR-amplified some of the representative genomic regions where expected and non-expected SNV types were detected, and then sequenced these amplicons using the Sanger method. All SNVs of the expected types were indeed present in the genomes, whereas all non-expected SNVs were found to be assembly errors. For example, we observed frequent putative A&#8594;C &#8220;transversions&#8221; in ACC or ACCCC sequence motifs, but these were not confirmed by Sanger sequencing. Based on these results, we removed all non-standard SNVs from the data set. Finally, detected SNVs were extracted from the alignments for further analyses (<xref ref-type="supplementary-material" rid="pgen.1003736.s006">Table S6</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The predicted effects of SNVs on proteins were analyzed in Geneious Pro <xref ref-type="bibr" rid="pgen.1003736-Drummond1">[73]</xref>. To extract the genomic sequence context of mutations, we used <italic>ad hoc</italic> program Rseq1. Consensus sequences, also called sequence logos (<xref ref-type="fig" rid="pgen-1003736-g003">Fig. 3</xref>) were created using WebLogo 3 (<ext-link ext-link-type="uri" ns0:href="http://weblogo.threeplusone.com/">http://weblogo.threeplusone.com/</ext-link>) <xref ref-type="bibr" rid="pgen.1003736-Crooks1">[74]</xref> with adjustment for the GC composition of the corresponding genomes and reporter genes.</p><p>The Mann-Whitney test as used to compare differences in mutation loads in different types of mutant and non-mutant clones (see <xref ref-type="fig" rid="pgen-1003736-g004">Fig. 4</xref>). This result suggests that the difference between haploid and diploid strains is significant, reflecting differences in their ability to tolerate the high frequency of induced mutations.</p>