<p>Fifty-one measurements were captured using an Immersion MicroScribe Digitizer G2X running Microscribe Utility Software and Diagnostics (v5.0.0.2). In total, 533 canid skulls representative of 120 breeds and 4 gray wolf subspecies located in museums and private collections were documented. Dorsal and ventral landmark datasets were captured separately and merged based on landmarks in common between datasets (landmarks 1, 2, 28, and 29) using File Converter software (Klingenberg lab). Procrustes fit, PCA, and residuals were generated using MorphoJ <xref ref-type="bibr" rid="pgen.1002849-Klingenberg1">[11]</xref>. Residuals of nonallometric shape were calculated as implemented in MorphoJ (v1.03a) using linear regression (pooled by sex and breed), with symmetric component and log(neurocranium centroid) corresponding to dependent and independent variables, respectively. Ten thousand permutations were performed. Refer to <xref ref-type="supplementary-material" rid="pgen.1002849.s001">Figure S1</xref> to see landmarks used by MorphoJ to calculate neurocranium centroid.</p><p>A covariance matrix based on residuals was analyzed by PCA. GWAS was performed using a subset of the CanMap dataset of genotypes <xref ref-type="bibr" rid="pgen.1002849-Boyko1">[3]</xref>. In total, 72 breed-sex averages of PC1 were assigned to CanMap breeds. In 30 instances, only one skull per breed-sex was measured. In such cases, the actual PC1 score was used for CanMap phenotype assignments. Log(neurocranium centroid) values were similarly assigned and used in subsequent analyses as a size covariate for PLINK and GEMMA association analyses (see next section).</p><p>Skull surface scans (1 Pug, 1 gray wolf) were done by Konica Minolta (3D Sensing Labs, Ramsey, NJ). Decimated scans were loaded into Landmark Editor (v3.6) <xref ref-type="bibr" rid="pgen.1002849-Wiley1">[34]</xref>. Skull morphing was done using PC1 landmark coordinates exported from MorphoJ. Coordinate files used for morphing were generated from representatives of dolichocephalic and brachycephalic breeds (a Collie and Pug).</p><p>Base pair positions stated throughout refer to CanFam2 (Broad/May 2005) coordinates. Single marker and haplotype association analyses were done using PLINK (v1.07) <xref ref-type="bibr" rid="pgen.1002849-Purcell1">[15]</xref> or mixed model GEMMA (v0.91) <xref ref-type="bibr" rid="pgen.1002849-Zhou1">[17]</xref> where specified. CanMap markers used in the analysis included SNPs with missingness &lt;0.10 and minor allele frequency &gt;0.01. In the full dataset (all breeds with breed-sex PC1 averages), 61,270 SNPs were analyzed by PLINK from 576 dogs representing 62 American Kennel Club-recognized breeds. In the mixed-model, &#8764;36,685 SNPs were analyzed. Breeds used in size-stratified analyses are listed in <xref ref-type="supplementary-material" rid="pgen.1002849.s008">Table S2</xref>. Significantly associated SNPs surpassed Bonferroni correction at the 0.05 level (&#8722;log10(<italic>P</italic>)&gt;&#8202;=&#8202;5.86). H<sub>O</sub> was calculated by treating CanMap breeds at the polar extremes of PC1 as two comparisons populations (Pug, Pekingese, Boston Terrier, Shih Tzu, Brussels Griffon, French Bulldog, Bulldog, Boxer, Cavalier King Charles Spaniel, Chihuahua versus Collie, Borzoi, Saluki, Scottish Deerhound, Bloodhound, Greyhound, Scottish Terrier, Doberman Pinscher, and Irish Wolfhound). F<sub>ST</sub> was calculated treating brachycephalic breeds (listed above) as a single subpopulation. H<sub>O</sub>, H<sub>R</sub> (the ratio of dolicho- and brachycephalic H<sub>O</sub>), and F<sub>ST</sub> values were calculated using custom R scripts. fastPHASE was used to generate haplotype frequencies by breed, using CanMap genotypes using the clustering parameter k&#8202;=&#8202;15 <xref ref-type="bibr" rid="pgen.1002849-Scheet1">[35]</xref>. &#8220;Extreme brachycephalic breeds&#8221; were designated as such if both PC1 breed-sex averages exceeded 0.15. This cutoff was chosen based on the obvious jump in magnitude of PC1 values (see <xref ref-type="fig" rid="pgen-1002849-g001">Figure 1E</xref>, <xref ref-type="supplementary-material" rid="pgen.1002849.s006">Figure S6</xref>). Breeds that meet this classification include the Pug, Pekingese, Boston Terrier, French Bulldog, Bulldog, Brussels Griffon, and Shih Tzu.</p><p>DNA used in our study was extracted from blood samples as previously described <xref ref-type="bibr" rid="pgen.1002849-Sutter1">[16]</xref>. In addition to whole-genome sequencing (see below), <italic>BMP3</italic> and <italic>PRKG2</italic> were Sanger sequenced using six brachycephalic and six dolichocephalic breeds (data not shown). The <italic>BMP3</italic> 8,196,098 C/A transversion was sequenced in an expanded panel composed of 847 dogs from 113 breeds. Primers were designed with a melting temperature (T<sub>m</sub>) ranging between 68&#8211;72&#176;C, GC content ranging between 20&#8211;80%, length ranging between 18&#8211;32 nucleotides, and included 5&#8242; M13 tags (<xref ref-type="supplementary-material" rid="pgen.1002849.s016">Table S10</xref>). PCR products for sequencing were generated with a 2-step thermocycler program:</p><p>Initial Denaturation: 1&#215;&#8212;95&#176;C, 5 minutes</p><p>Two-step Cycles: 35&#215;&#8212;95&#176;C, 30 seconds; 68&#176;C, 2 minutes</p><p>Extension: 1&#215;&#8212;72&#176;C, 10 minutes</p><p>PCR products were sequenced using a standard protocol <xref ref-type="bibr" rid="pgen.1002849-Sutter1">[16]</xref>. During the course of SNP discovery, we discovered errors in the reference genome sequence for canine <italic>BMP3</italic>, producing two early stop codons in the first exon. Sequencing of 13 dogs, including the individual from which the reference genome sequence was derived, indicates these stop codons are the results of errors in the reference sequence.</p><p>Paired-end libraries were prepared from DNA from eleven dogs of breeds with widely varying skull shapes. Sequencing was conducted on an Illumina HiSeq 2000 sequencer to a depth of 5.6&#8211;8.5&#215; per dog using manufacturer protocols. The resulting 101-base paired-end sequences were mapped to the genome (CanFam2 release May 2005) with bwa version 0.5.9-r16 with read trimming set to 15. SNPs were called with samtools mpileup version 0.1.18 and custom R scripts <xref ref-type="bibr" rid="pgen.1002849-Ihaka1">[36]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1002849-Wickham2">[39]</xref>. Thirteen SNPs in the PC1-associated region overlap with the CanineHD Genotyping BeadChip (Illumina cat. no. WG-440). DNA from four dogs was assayed with the chip; all resulting genotypes were identical in the deep sequencing and chip results. Four hundred and fifty-two SNPs were identified in the critical interval (85.7 kb between 8,152,258 and 8,237,937), and subjected to further filters. Genotypes with a genotype quality score below 8 were reset to &#8220;unknown.&#8221; We performed association analysis using PLINK with options specifying an additive model omitting the Scottish Terrier, a dolichocephalic breed that appears to be an outlier <xref ref-type="bibr" rid="pgen.1002849-Purcell1">[15]</xref>. After correcting for multiple testing, no SNPs were significantly associated due to limited statistical strength of the test. SNPs in the 5th percentile for association scores were retained. Cross-species conservation was assessed by the UCSC phastCons4way calculations <xref ref-type="bibr" rid="pgen.1002849-Siepel1">[40]</xref> downloaded November 30, 2011, which is generated by using the phastCons program to score the extent of conservation between dog, human, mouse and rat. SNPs with a phastCons4way score above 0.7 were retained. SNPs in an exon or within 20 bases of a splice junction were retained.</p><p>Morpholino knockdown experiments of <italic>bmp3</italic> used two translation blockers (MO1: <named-content content-type="gene">5&#8242;-TGACAGCGATCCATGCTGGAGGTGC-3&#8242;</named-content>, MO2 <named-content content-type="gene">5&#8242;-CGGGACTATGGAAGCTGATCTA-3&#8242;</named-content>), which overlapped by one nucleotide. Morpholino injections used 5.1 ng (MO1) or 7.5 ng (MO2), as determined by titrations.</p><p>Zebrafish <italic>bmp3</italic> (IMAGE Id 7052011) and human <italic>BMP3</italic> (Origene clone SC302990) cDNAs were sequenced and determined to be full length. Missense F&#8594;L mutations for mouse <italic>GDF1</italic>
<xref ref-type="bibr" rid="pgen.1002849-Wall1">[41]</xref>, human <italic>Bmp3</italic>, and zebrafish <italic>bmp2b</italic>
<xref ref-type="bibr" rid="pgen.1002849-Nikaido1">[42]</xref> were introduced using site-directed mutagenesis and confirmed by sequencing. Zebrafish <italic>bmp3</italic> wt and F&#8594;L cDNAs were PCR-amplified using gene-specific primers with attB sites. PCR products were subcloned into entry and destination vectors (pCSDest) using Gateway recombination, as previously described <xref ref-type="bibr" rid="pgen.1002849-Villefranc1">[43]</xref>, <xref ref-type="bibr" rid="pgen.1002849-Kwan1">[44]</xref>. To construct the human <italic>BMP3</italic> expression vector, we PCR-amplified the TGF-&#946; signaling domain using primers with XbaI and XhoI restriction sites. PCR products were ligated into an expression vector bearing the Xenopus <italic>BMP2</italic> prodomain, as such heterologous fusion constructs were previously shown to enhance propeptide cleavage and biological activity <xref ref-type="bibr" rid="pgen.1002849-Wall1">[41]</xref>. mRNA was synthesized using Ambion's SP6 mMessage kit from plasmid that was linearized with Not I. Embryo analyses of RNA injections were done based on injections of the following amounts: 25&#8211;300 pg human <italic>BMP3</italic> mRNA, 25&#8211;300 pg mouse <italic>Gdf1</italic> mRNA, 1&#8211;100 pg zebrafish <italic>bmp2b</italic>. mRNA overexpression assays were repeated three or more times at each stated concentration, unless stated otherwise.</p><p>
<italic>In situ</italic> hybridization was completed as described in Thisse and Thisse 2008 <xref ref-type="bibr" rid="pgen.1002849-Thisse1">[45]</xref>, except probes were hydrolyzed for 2 minutes at 65&#176;C, the hybridization solution contained 5% dextran sulfate, and the anti-DIG-AP incubation and subsequent washes were performed in Malic Acid Buffer rather than PBST. Alcian blue stains were done as previously described by Schilling <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1002849-Schilling1">[46]</xref>, except that staining solution was composed of 0.15% Alcian blue, 50% EtOH, and 0.1 M HCl (pH&#8202;=&#8202;&#8764;1).</p><p>Embryos were imaged using Zeiss Axio Imager.M1, Zeiss SteREO Lumar v12, or Leica M216F compound microscope. Zeiss Axiovision v4.8.1 software was used for image capture. Nonspecific background and dissection debris were removed from images of Alcian blue cartilage dissections using Adobe Photoshop CS3. All plots were generated using custom scripts, in conjunction with R Cran packages ggplot2 <xref ref-type="bibr" rid="pgen.1002849-Wickham1">[38]</xref>, reshape2 <xref ref-type="bibr" rid="pgen.1002849-Wickham2">[39]</xref>, and RColorbrewer <xref ref-type="bibr" rid="pgen.1002849-Neuwirth1">[47]</xref>. Manhattan plot and Q-Q plot scripts were adapted from examples posted on the blog &#8220;Getting Genetics Done&#8221; <xref ref-type="bibr" rid="pgen.1002849-Turner1">[48]</xref>. Post-processing of plots was done using Adobe CS4 Creative Suite softwares Photoshop, InDesign, and Illustrator.</p><p>Informed consent was obtained for all collected dog samples. All animal protocols (dog and zebrafish) were approved by the Animal Care and Use Committees of the Intramural Program of the National Human Genome Research Institute at the National Institutes of Health or by Animal Care Committee of the Hospital for Sick Children Research Institute. Wild canids samples were graciously provided by Dr. Robert Wayne, in accordance with UCLA Approved Animal Care and Use Committee Policies.</p>