<p>All animal experimentation was approved by the Australian National University and Monash University Animal Experimentation Ethics Committees and performed in accordance with Australian NHMRC Guidelines on Ethics in Animal Experimentation.</p><p>Point mutant mice were generated as described previously on a C57BL/6 background and outbred to CBA <xref ref-type="bibr" rid="pgen.1002698-Jamsai1">[27]</xref>. Mouse lines containing sterility causing mutations were identified by breeding trials wherein eight G3 brother-sister pairs per line were co-housed and the presence of pups monitored. Lines where male sterility was observed in a ratio of approximately one in four in the G3 generation with apparently normal mating behaviour were selected for further analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Affymetrix 5K mouse SNP Chip arrays were used to map the sterility causing mutation. Genomic DNA from five affected males was hybridized onto the array at the Australian Genome Research Facility and compared to wild type C57BL/6 and CBA sequences. The linkage interval was subsequently narrowed using additional mice and SNPs (<ext-link ext-link-type="uri" ns0:href="http://www.well.ox.ac.uk/mouse/INBREDS/">www.well.ox.ac.uk/mouse/INBREDS/</ext-link>) using the Amplifluor SNP Genotyping System (Chemicon). Plates were read in a BMG Fluostar optima fluorescent microplate reader.</p><p>Following the identification of the causal mutation, mice were specifically genotyped using the Amplifluor SNPs HT genotyping system using a wild type-specific antisense primer <named-content content-type="gene">5&#8242;-GAAGGTCGGAGTCAACGGATTAAGAGCACCCGTACCTGAC-3&#8242;</named-content>, a mutant allele antisense primer <named-content content-type="gene">5&#8242;-GAAGGTGACCAAGTTCATGCTGAAGAGCACCCGTACCTGAA-3&#8242;</named-content>, a sense primer, <named-content content-type="gene">5&#8242;-GGTGGTGAGCTGCATTGAA-3&#8242;</named-content> and Platinum <italic>Taq</italic> DNA Polymerase (Invitrogen). Conditions for amplification were as follows: 4 minute denaturation at 95&#176;C, 35 cycles of denaturation at 95&#176;C for 10 seconds, annealing at 60&#176;C for 20 seconds and elongation at 72&#176;C for 40 seconds, followed by a final 3 minute elongation at 72&#176;C. Following the reaction, plates were read in a BMG Fluostar optima fluorescent plate reader.</p><p>Infertility in the Taily mouse line was classified using the scheme outlined in Borg et al <xref ref-type="bibr" rid="pgen.1002698-Borg1">[46]</xref>. Daily sperm output and total epididymal sperm content were determined as described previously <xref ref-type="bibr" rid="pgen.1002698-Cotton1">[47]</xref>. Sperm motility was assessed using computer assisted sperm analysis <xref ref-type="bibr" rid="pgen.1002698-Gibbs1">[48]</xref> and ultra-structure using electron microscopy <xref ref-type="bibr" rid="pgen.1002698-Arsov1">[49]</xref>. Cauda epididymal sperm morphology was assessed following staining with hematoxylin. Cells undergoing apoptosis were visualized using the Apoptag kit (Millipore) as recommended by the manufacturer.</p><p>The number of germ cells per Sertoli cell were enumerated in 25 &#181;m thick, periodic acid Schiffs (PAS) stained methacrylate sections using the optical disector as previously described <xref ref-type="bibr" rid="pgen.1002698-Saito1">[50]</xref>. Retained elongated spermatids were counted in stage XI-XI <xref ref-type="bibr" rid="pgen.1002698-Saito1">[50]</xref> and expressed as fold wild type.</p><p>RNA was extracted from testes at defined periods throughout post-natal development using TRIzol regents (Life Technologies), treated with DNase I (Ambion) and cDNA sythnesized using oligo-dT primers and SuperScript III reverse transcriptase (Life Technologies). The relative expression of <italic>Katnb1</italic>, <italic>Katna1</italic>, <italic>Katnal1</italic> and <italic>Katnal2</italic> were defined using quantitative PCR using TaqMan assays (Applied Biosystems) Mm01244795_m1, Mm00496172_m1, Mm00463780_m1 and Mm00510701_m1 respectively. Expression of these was normalized against peptidylprolyl isomerase A (Mm002342429_g1).</p><p>Germ cell sub-populations were purified using the Staput method as previously described <xref ref-type="bibr" rid="pgen.1002698-Romrell1">[51]</xref>. Single cell suspensions were loaded onto a 2&#8211;4% continuous BSA gradient and elongated spermatids and round spermatids collected after a 3 hour and 3.5 hour sedimentation period, respectively. For immunofluorescent staining, gradient fractions were pelleted and resuspended in 4% paraformaldehyde (PFA) fixative for 2 hours on ice. Cells were then washed with PBS and spread onto slides.</p><p>Protein was extracted from round spermatid fractions (&gt;90% purity) using 20 &#181;L M-PER buffer (Thermo Scientific). 10 &#181;g of protein was separated on a 10% SDS-PAGE gel and probed for rabbit katanin p80 (HPA041165, which recognizes a C-terminal region of p80, Sigma Aldrich) and actin (A2066, Sigma Aldrich). Bound antibody was detected using a goat anti-rabbit IgG HRP (P0488, Dako) secondary antibody and an enhanced chemiluminescence (ECL Plus) detection kit (Amersham Biosciences).</p><p>Katanin subunits and &#945;-tubulin were localized in testis sections as described <xref ref-type="bibr" rid="pgen.1002698-Gibbs2">[52]</xref>. Primary antibodies included: anti-&#945;-tubulin (T5168, Sigma, diluted 1 in 5000), anti-katanin p60-like 2 (p60AL2, #sc-84855, Santa Cruz, diluted 1 in 100), anti-katanin p80 (diluted 1 in 200) <xref ref-type="bibr" rid="pgen.1002698-Hartman1">[9]</xref> and anti-katanin p60 (diluted 1 in 200) <xref ref-type="bibr" rid="pgen.1002698-McNally6">[53]</xref>. Both p60 and p80 antibodies were affinity-purified from rabbits immunised against full length recombinant human proteins. These antibodies have been validated extensively and have been shown to recognize a single polypeptide in HeLa cells and a range of human tissues <xref ref-type="bibr" rid="pgen.1002698-Hartman1">[9]</xref>, <xref ref-type="bibr" rid="pgen.1002698-McNally6">[53]</xref>. Given the sequence homology between the p60 and p60L1 subunits, and between p80 and the uncharacterized c15orf29 subunit, there remains the possibility of partial cross-reactivity. Secondary antibodies included: Alexa Fluor 555 donkey anti-rabbit IgG (A-31572) and Alexa Fluor 488 donkey anti-mouse IgG (A-21202) (diluted 1 in 500). DNA was labelled using DAPI (Invitrogen). To define the localization of proteins within isolated elongating spermatids, cells were permeabilized in 0.2% Triton X-100 diluted in 10% normal horse serum (NHS) in PBS for one hour at room temperature. Non-specific labelling was minimized by blocking in 10% NHS in PBS for 30 minutes. Primary antibodies were diluted in 10% NHS in PBS and incubated overnight at 4&#176;C. Secondary antibodies were diluted 1 in 200 and incubated at room temperature for 2 hours. DNA was labelled using TOPRO3 (Invitrogen, 1 in 200) or DAPI. Images were taken with an SP5 5-channel (Leica Microsystems) confocal microscope in the Monash University Microimaging facility. Metaphase spindle lengths were measured on &#945;-tubulin and DAPI-stained sections from <italic>Katnb1<sup>WT/WT</sup></italic> and <italic>Katnb1<sup>Taily/Taily</sup></italic> mice using LAS AF (Leica Application Suite Advanced Fluorescence) software. Z-stacks of spindles and manchettes were collected at 0.5 &#181;m intervals. Images were assembled using Adobe Photoshop. Test and subject images were adjusted uniformly across the image and between groups.</p><p>Differences between <italic>Katnb1<sup>WT/WT</sup></italic> and <italic>Katnb1<sup>Taily/Taily</sup></italic> mice were determined using unpaired t tests in GraphPad Prism 5.0.</p>