<p>Yeast strains used are isogenic to strain 779-6A (<italic>MATa</italic>, <italic>SUQ5</italic>, <italic>kar1-1</italic>, <italic>ade2-1</italic>, <italic>his3&#916;202</italic>, <italic>leu2&#916;1</italic>, <italic>trp1&#916;63</italic>, <italic>ura3-52</italic>) <xref ref-type="bibr" rid="pgen.1004720-Jung2">[70]</xref>, which is used for monitoring [<italic>PSI<sup>+</sup></italic>] and [<italic>PIN<sup>+</sup></italic>]. Knockouts and replacements of chromosomal genes were done using standard transformation procedures <xref ref-type="bibr" rid="pgen.1004720-Sherman1">[71]</xref>. Strain MR386 has <italic>E. coli CLPB</italic> in place of the chromosomal <italic>HSP104</italic> gene <xref ref-type="bibr" rid="pgen.1004720-Reidy1">[15]</xref> and contains plasmids expressing <italic>E. coli</italic> dnaK<sup>R167H</sup> (pMR150LG-R167H) and <italic>E. coli</italic> GrpE (pMR134H) under the control of the GPD (glyceraldehyde-3-phosphate dehydrogenase - <italic>TDH3</italic>) and <italic>FES1</italic> promoters, respectively. [<italic>PSI</italic>
<sup>+</sup>] is maintained in strain MR386 by pJ312, which is <italic>HSP104</italic> on a <italic>URA3</italic> plasmid <xref ref-type="bibr" rid="pgen.1004720-Jung3">[72]</xref>. Strain MR502 has <italic>ydj1::KanMX</italic> and carries p316YDJ1, which is <italic>YDJ1</italic> on a <italic>URA3</italic> plasmid. [<italic>PSI<sup>+</sup></italic>] [<italic>PIN<sup>+</sup></italic>] strain 930 has <italic>sis1::KanMX</italic> and carries plasmid pYW17, a <italic>URA3</italic>-based plasmid encoding wild type <italic>SIS1</italic>
<xref ref-type="bibr" rid="pgen.1004720-Yan1">[22]</xref>, <xref ref-type="bibr" rid="pgen.1004720-Kirkland1">[32]</xref>.</p><p>Strain 930a is a [<italic>psi<sup>&#8722;</sup></italic>] [<italic>PIN<sup>+</sup></italic>] version of 930 that carries plasmid p313Rnq1-GFP. It was cured of [<italic>PSI<sup>+</sup></italic>] by transient growth on medium containing 3 mM guanidine and then [<italic>PIN<sup>+</sup></italic>] clones among [<italic>psi<sup>&#8722;</sup></italic>] isolates were identified by punctate Rnq1-GFP fluorescence. Our [<italic>PIN<sup>+</sup></italic>] variant is uncharacterized, but of the single-dot type, which typically has sturdier fibers and a lower seed number per cell than multi-dot [<italic>PIN<sup>+</sup></italic>] <xref ref-type="bibr" rid="pgen.1004720-Sharma4">[61]</xref>. Isogenic strain 1075, for monitoring [URE3], has <italic>ADE2</italic> regulated by the <italic>DAL5</italic> promoter (<italic>P<sub>DAL5</sub>::ADE2</italic>, see below) in place of <italic>ade2-1</italic>
<xref ref-type="bibr" rid="pgen.1004720-Sharma1">[43]</xref>. Strain 1385 is strain 1075 with <italic>sis1::KanMX</italic> and plasmid pYW17, which is <italic>SIS1</italic> on a <italic>URA3</italic> plasmid. Strains 1408 and 1410 are <italic>hsp104&#916;</italic> versions of strain 779-6A and 1075, respectively <xref ref-type="bibr" rid="pgen.1004720-Reidy1">[15]</xref>. Both carry pJ312. Our parental strains carry only one variant of [<italic>PSI<sup>+</sup></italic>], [URE3] or [<italic>PIN<sup>+</sup></italic>].</p><p>
<italic>SIS1</italic> plasmids used contain wild type and mutant <italic>SIS1</italic> alleles on the pRS314 single-copy <italic>TRP1</italic> vector <xref ref-type="bibr" rid="pgen.1004720-Yan1">[22]</xref>, <xref ref-type="bibr" rid="pgen.1004720-Kirkland1">[32]</xref>. Plasmid pRU4 is <italic>LEU2</italic>-based single-copy plasmid pRS415 containing the <italic>GAL1</italic> promoter and <italic>CYC1</italic> terminator flanking the polylinker sites <italic>Spe</italic>I and <italic>Xho</italic>I. For Gal-induced expression, <italic>YDJ1</italic> and hybrid alleles were inserted into pRU4 as <italic>Bam</italic>HI-<italic>Sal</italic>I fragments. All plasmids used in this study are listed in <xref ref-type="table" rid="pgen-1004720-t001">Table 1</xref>. Plasmids encoding <italic>E. coli</italic> genes or yeast Hsp40 genes with D36N mutations are described <xref ref-type="bibr" rid="pgen.1004720-Reidy1">[15]</xref>.</p><p>See <xref ref-type="sec" rid="s4">Materials and Methods</xref> section for details of construction. All plasmids are <italic>CEN</italic> (single copy) unless otherwise indicated with &#8220;2-micron&#8221; (high copy). All pMR and pRU plasmids contain the <italic>CYC1</italic> terminator sequence immediately downstream of the ORF.</p><p>With the exceptions that 1/2YPD plates contain 5 g/L yeast extract, YPAD plates contain 400 mg/L (excess) adenine and solid defined media contain 10 mg/L (limiting) adenine, standard media and growth conditions were used <xref ref-type="bibr" rid="pgen.1004720-Sherman1">[71]</xref>.</p><p>Sis1 and Ydj1 have clearly defined and characterized J-domains, glycine-phenylalanine (GF) rich middle domains, and a C-terminal region that contains two major elements (CTDI and CTDII) and a dimerization domain. The main structural differences between Sis1 and Ydj1 are a glycine-methionine-rich (GM) extension of the GF domain in Sis1, which has GF-redundant functions, and a Zn-finger domain (ZF) embedded between beta-strands 2 and 3 of the CTDI of Ydj1 that is absent in Sis1 <xref ref-type="bibr" rid="pgen.1004720-Wu1">[73]</xref>. Because the ZF of Ydj1 is an integral part of CTDI, we designed our Sis1-Ydj1 hybrids using the GF-CTDI junction to restrict the number of domain swaps and avoid using complicated junctions to swap the ZF region. Rather than designating the GM region as a separate domain, we combined the functionally redundant GF and GM regions of Sis1 into a single domain. Thus, hybrid alleles were made by swapping three regions: the J-domain, the glycine-rich region and the C-terminal portion, which includes CTDI, CTDII and the dimerization domains, herein referred to simply as the CTD (see <xref ref-type="fig" rid="pgen-1004720-g001">Figure 1A</xref>, <xref ref-type="bibr" rid="pgen.1004720-Sondheimer2">[30]</xref>). Hybrid genes were synthesized by GENEWIZ, Inc (South Plainfield, NJ) and sub-cloned into variants of pRS414 that placed the ORF under the control of the <italic>SIS1</italic>, <italic>YDJ1</italic> or GPD promoters and a downstream <italic>CYC1</italic> transcriptional terminator <xref ref-type="bibr" rid="pgen.1004720-Mumberg1">[74]</xref>. All constructs contained a c-terminal c-myc tag that had no noticeable affect on functions in vivo.</p><p>Depletion of the ribosome release factor Sup35 by its sequestration in [<italic>PSI<sup>+</sup></italic>] prion aggregates causes nonsense suppression. We monitored [<italic>PSI<sup>+</sup></italic>] by suppression of the <italic>ade2-1</italic> nonsense allele in our strains. [<italic>PSI<sup>+</sup></italic>] cells are Ade<sup>+</sup> and white, while [<italic>psi<sup>&#8722;</sup></italic>] cells are Ade<sup>&#8722;</sup> and when grown on limiting adenine are red due to accumulation of a metabolite of adenine biosynthesis. The presence of [URE3] was monitored similarly by use of an <italic>ADE2</italic> allele regulated by the <italic>DAL5</italic> promoter (<italic>P<sub>DAL5</sub>::ADE2</italic>) <xref ref-type="bibr" rid="pgen.1004720-Schlumpberger1">[75]</xref>, <xref ref-type="bibr" rid="pgen.1004720-Brachmann1">[76]</xref>. Under standard growth conditions Ure2 represses transcription of nitrogen metabolic genes, such as <italic>DAL5</italic>. [URE3] sequesters Ure2 into prion aggregates, thereby reducing Ure2 function and activating the <italic>DAL5</italic> promoter. Thus, [URE3] cells are Ade<sup>+</sup> and white, while [ure-o] cells are Ade<sup>&#8722;</sup> and red on limiting adenine. We confirmed that Ade<sup>+</sup> phenotypes were due to the presence of prions by their guanidine curability and by crosses with cells lacking prions to produce a dominant, guanidine-curable Ade<sup>+</sup> phenotype. We monitored [<italic>PIN<sup>+</sup></italic>] by assessing aggregation status of a plasmid-expressed Rnq1-GFP fusion protein. GFP fluorescence is diffuse in [<italic>pin<sup>&#8722;</sup></italic>] cells, but noticeably punctate in [<italic>PIN<sup>+</sup></italic>] cells. In this study we used a typical strong [<italic>PSI<sup>+</sup></italic>] strain and single variants of [URE3] and [<italic>PIN<sup>+</sup></italic>] prions.</p><p>Microscopic analysis of Rnq1-GFP fluorescence in live cells was done with a Nikon E-800 microscope with log phase cells grown in medium selecting for the plasmid encoding the Rnq1-GFP fusion protein. Images were captured using IVision software and processed using Adobe Photoshop software.</p><p>Log phase cells grown in medium selecting for plasmids were diluted in fresh medium to an OD<sub>600</sub> of 0.25 and 100 &#181;L was transferred to 0.5 mL test tubes and placed in a PCR machine for thermocycling as indicated. At various times aliquots were removed and placed on ice. Cooled cells were serially diluted and 5 &#181;L drops were spotted onto YPAD plates.</p><p>Strains 930 and 1385 (both <italic>sis1&#916;</italic>) and derivatives of MR502 (<italic>ydj1&#916;</italic>) carrying wild type <italic>SIS1</italic> or <italic>YDJ1</italic> on <italic>URA3</italic>-based plasmids were transformed by <italic>TRP1</italic>-based plasmids carrying wild type, mutant or hybrid alleles. Strain MR502 also carries pMR169 for monitoring <italic>GAL</italic> induction (see below). Transformants were grown as patches of cells on medium lacking both tryptophan and uracil and then replica-plated onto similar medium containing uracil to allow loss of the <italic>URA3</italic> plasmid. These were then replica-plated onto medium containing 5-FOA, which kills cells that did not lose the resident <italic>URA3</italic> plasmid.</p><p>For Sis1, growth on 5-FOA plates shows complementation of functions essential for growth, and growth without adenine shows complementation of Sis1 functions required for prion propagation. To test complementation of Ydj1 function, 5-FOA resistant cells of strain MR502 were grown overnight in medium selecting for the <italic>TRP1</italic> plasmids, normalized to the same cell density (OD<sub>600</sub>&#8202;=&#8202;0.25) and five-fold serially diluted. Aliquots of the dilutions (5 &#181;L) were then dropped onto YPAD plates. Scanned images of the plates were taken after they were incubated at the indicated temperatures for 3&#8211;4 days.</p><p>Aliquots of overnight cultures of MR502 transformants used for growth complementation were transferred to synthetic raffinose medium (SRaf) and grown overnight. These cells carried the <italic>TRP1</italic>-marked hybrid alleles or empty vector control and a <italic>HIS3</italic>-marked plasmid encoding firefly luciferase under the control of the <italic>GAL10</italic> promoter (pMR169). Cell densities were adjusted to OD<sub>600</sub>&#8202;=&#8202;0.3 in fresh medium and the initial reading (t&#8202;=&#8202;0) was taken by mixing 100 &#181;L culture with 50 &#181;L of 1 mM luciferin in 0.1 M sodium citrate, pH 5.0 immediately before reading in a Zylux Femtomaster luminometer, with a 10 s delay and 5 s read time. Galactose from a 20% stock was then added to the cultures to obtain a final concentration of 2% and the cells were incubated on a roller at 30&#176;C. Readings were taken at 30, 60 and 120 minutes after addition of galactose. All samples in three experiments were tested in triplicate. No significant growth occurred during the course of the experiment.</p><p>Overnight SD cultures of cells carrying <italic>YDJ1</italic> or hybrid alleles on pRU4 for galactose induction were used to inoculate SGal medium to OD<sub>600</sub>&#8202;=&#8202;0.05 and incubated with shaking at 30&#176;C. Generations were monitored as doublings of OD<sub>600</sub>. Cultures were sub-cultured as necessary to keep the OD<sub>600</sub> less than 2.0. After 3, 6 and 9 generations of growth in galactose, aliquots were removed and cells plated onto 1/2YPD plates. Loss of [URE3] was assessed by determining the fraction of entirely red (i.e. [ure-o]) colonies after 3 days of incubation at 30&#176;C. To test the ability of overexpressed Sis1 to block Ydj1- or Hsp104- mediated curing of [URE3], strain 1075 was co-transformed with various combinations of CEN plasmids expressing Sis1, Ydj1 or Hsp104 under the control of the GPD promoter.</p><p>Cell lysates for western blots were prepared as described <xref ref-type="bibr" rid="pgen.1004720-Reidy2">[77]</xref>. Briefly, cells were suspended in lysis buffer and broken by agitation with glass beads. For each sample 10 &#181;g of protein was separated on 4&#8211;20% SDS-PAGE gels, transferred to PVDF membranes and probed using anti-c-myc antibody (AbCam #ab9106) and chemiluminescence. After developing, the blots were stained by amido-black (Sigma #A-8181) as a loading and transfer control.</p><p>Hsp104 <xref ref-type="bibr" rid="pgen.1004720-Miot1">[78]</xref>, Ydj1 <xref ref-type="bibr" rid="pgen.1004720-Cyr1">[79]</xref>, and GFP-38 <xref ref-type="bibr" rid="pgen.1004720-Hoskins1">[80]</xref> were purified as described. SSY was isolated as described for Ydj1 <xref ref-type="bibr" rid="pgen.1004720-Cyr1">[79]</xref>. Sis1 was purified as described <xref ref-type="bibr" rid="pgen.1004720-Caplan2">[81]</xref> with some modifications. Briefly, BL21 (DE3) was transformed with a pET11 plasmid containing the Sis1 gene, cultures were grown at 30&#176;C to OD<sub>595</sub> 0.8 and cells were induced with 1 mM IPTG for 3 h. Clarified cell lysates were applied to an S-Sepharose-FF column (GE Healthcare) in 20 mM MES buffer, pH 6.0, 0.1 mM EDTA and 1 mM DTT. Sis1 was eluted with a linear gradient from 0&#8211;1 M NaCl. Peak fractions were pooled, buffer exchanged into 20 mM MES buffer pH 6.0, 0.1 mM EDTA and 1 mM DTT. The sample was applied to a monoS column (GE Healthcare) and eluted with a linear gradient from 0&#8211;1 M NaCl. YYS was purified similarly to Sis1, except that the buffer used was 25 mM HEPES, pH 7.6, 0.1 mM EDTA and 1 mM DTT. For Ssa1, a pET24 plasmid containing the Ssa1 gene was transformed into Rosetta BL21 (DE3) cells. Cultures were grown to 0.8 OD<sub>595</sub> at 30&#176;C and induced with 0.2 mM IPTG overnight. The clarified lysate was applied to a Q-sepharose FF column (GE Healthcare) in 20 mM Tris<sup>.</sup>HCl, pH 7.6, 40 mM KCl, 0.1 mM EDTA and 1 mM DTT. Ssa1was eluted with a linear gradient of 40&#8211;400 mM KCl over 20 column volumes. Peak fractions were collected and buffer exchanged into 25 mM HEPES, pH 7.6, 100 mM KCl, 0.1 mM EDTA and 1 mM DTT and further purified over a monoQ column (GE Healthcare) using a linear gradient of 100&#8211;400 mM KCl over 20 column volumes. Peak fractions were collected, analyzed, supplemented with 10% glycerol, frozen on dry ice and stored at &#8722;80&#176;C.</p><p>Experiments were performed as described previously <xref ref-type="bibr" rid="pgen.1004720-Hoskins1">[80]</xref> with some modifications. Reactivation was carried out in HKE Buffer (25 mM HEPES, 75 mM KCl, 0.1 mM EDTA, pH 7.6) and GFP-38 heat-denaturation was carried out in Buffer D (20 mM Tris&#183;HCl, 100 mM KCl, 0.1 mM EDTA, 10% glycerol, pH 7.5). 14 mM GFP-38 in buffer D was heated at 80&#176;C for 15 min, frozen on dry ice, thawed and immediately used in experiments. Reactivation reactions (100 &#181;l) contained HKE buffer, 4 mM ATP, 10 mM MgCl<sub>2</sub>, an ATP regenerating system (10 mM creatinine phosphate and 0.03 mg/mL creatinine kinase), 0.7 &#181;M denatured GFP-38, 0.2 &#181;M Hsp104, 1 &#181;M Ssa1 and 0.2 &#181;M Hsp40 (Sis1, Ydj1, SSY or YYS). Reactions were initiated by adding MgATP, and reactivation was monitored by measuring GFP fluorescence at 23&#176;C using a TECAN Infinite M200Pro plate reader. GFP-38 reactivation was calculated as a percentage of the initial native GFP-38 fluorescence.</p><p>Luciferase (80 nM; 55 &#181;l) was heat-denatured in HKE buffer containing 0.05 mg/mL BSA, 2 mM DTT and 10 mM MgCl<sub>2</sub> at 45&#176;C for 7 min and then cooled to 4&#176;C for 1 min. Denatured luciferase was added to reactivation reactions (70 &#181;l total) containing HKE, 0.1 mg/mL BSA, 2 mM DTT, 10 mM MgCl<sub>2</sub>, 3 mM ATP, an ATP regenerating system as above, 2 &#181;M Ssa1 and 1 &#181;M Sis1, Ydj1, SSY or YYS and incubated at 23&#176;C. Luciferase reactivation was monitored by removing 5 &#181;L aliquots at 5 min intervals and measuring luminescence in a TECAN Infinite M200Pro plate reader in the presence of luciferin. Reactivation was determined compared to an unheated luciferase control.</p>