<p>We performed a meta-analysis of genome-wide association studies of morning plasma cortisol in 12,597 subjects from 11 western European population-based cohorts: CROATIA-Vis (n&#8202;=&#8202;885), CROATIA-Korcula (n&#8202;=&#8202;898), CROATIA-Split (n&#8202;=&#8202;493), ORCADES (n&#8202;=&#8202;886), Rotterdam Study (n&#8202;=&#8202;2945), NFBC1966 (n&#8202;=&#8202;1195), Helsinki Birth Cohort Study 1934&#8211;44 (n&#8202;=&#8202;451), ALSPAC (n&#8202;=&#8202;1567), InChianti (n&#8202;=&#8202;1207), PREVEND (n&#8202;=&#8202;1151), and PIVUS (n&#8202;=&#8202;919). Replication was tested in 2,795 subjects from three independent cohorts: Raine Study (n&#8202;=&#8202;797), ET2DS (n&#8202;=&#8202;1,069), and MrOS-Sweden (n&#8202;=&#8202;929). Cortisol was measured by immunoassay in blood samples collected from study participants between 0700 and 1100 h. Inclusion criteria were adults aged 17 years or older from Caucasian populations; exclusion criteria were current glucocorticoid use, pregnant or breast feeding women, and twins (exclusion of one). Characteristics of the study populations are presented in <xref ref-type="supplementary-material" rid="pgen.1004474.s001">Table S1</xref> and details of each cohort are provided in <xref ref-type="supplementary-material" rid="pgen.1004474.s008">Text S1</xref>. All participants provided written informed consent and studies were approved by local Research Ethics Committees and/or Institutional Review Boards.</p><p>Each study performed single marker association tests, and study-specific linear regression models which used z-scores of log-transformed cortisol, additive SNP effects, and were adjusted for age and sex (model 1); age, sex, and smoking (model 2); or age, sex, smoking and body mass index (model 3). Imputation of the gene-chip results used the HapMap CEU population, build 36. In cohorts with consanguineous populations (ORCADES and Croatia), adjustments for principal components in kinship matrices were performed using ProbABEL; for other cohorts, Identity-By-Descent coefficients were calculated using PLINK and related participants excluded. In the majority of cohorts, participants were only included if blood samples had been obtained within a 60 minute time interval, when variations in time of sampling were ignored. In a subset of cohorts, samples were obtained over a wider time interval (but always in the morning before 1100 h) and time of blood sampling recorded; for these cohorts, three further models were run as above but also including time of sampling, calculated as minutes from first sampling time, as an additional covariate <xref ref-type="bibr" rid="pgen.1004474-Reynolds3">[43]</xref>.</p><p>Quality control was carried out on the imputed genome-wide data for all 11 studies prior to meta-analysis; this excluded all samples with a minor allele frequency (MAF) &lt;2%, call rate &lt;95%, Hardy-Weinberg equilibrium (HWE)&lt;1&#215;10<sup>&#8722;8</sup> and poor imputation quality (MACH R2_HAT&lt;0.30, IMPUTE PROPER_INFO&lt;0.60, BEAGLE INFO&lt;0.30, as appropriate). Quantile-quantile (QQ) plots and genomic control (lambda) were used to confirm quality control. Sex chromosomes were not analysed.</p><p>We performed fixed effects meta-analysis, which used combined allelic effects weighted by the inverse of their variance for each of the models using the GWAMA program <xref ref-type="bibr" rid="pgen.1004474-Magi1">[44]</xref>. This aligned all studies to the same reference allele at each SNP, thus avoiding strand errors, and excluded SNPs with obvious input errors (eg. discrepancies in effect allele frequencies). The results from analysis with or without genomic control were nearly identical, as expected with &#955;<sub>GC</sub>&#8202;=&#8202;1.005. The genome-wide significance threshold for the meta-analysis was <italic>P</italic>&lt;5&#215;10<sup>&#8722;8</sup>. Percentage variation of cortisol was calculated from meta-analysis results as (2*effect allele frequency)*(1-effect allele frequency)*(beta<sup>2</sup>/sd<sup>2</sup>). A regional plot was generated using LocusZoom <xref ref-type="bibr" rid="pgen.1004474-Pruim1">[45]</xref>, and heat map using snp.plotter <xref ref-type="bibr" rid="pgen.1004474-Luna1">[46]</xref> in R version 2.15.2. Joint analysis of meta-analysis was performed with the GCTA program <xref ref-type="bibr" rid="pgen.1004474-Yang1">[47]</xref>.</p><p>To detect independent top SNPs on the basis of empirical estimates of linkage disequilibrium between the SNPs, we used the clumping function as implemented in PLINK <xref ref-type="bibr" rid="pgen.1004474-Purcell1">[20]</xref>. All the SNPs with a <italic>P</italic>-value&lt;5&#215;10<sup>&#8722;5</sup> in meta-analysis were used for clumping. We grouped the SNPs within 500 kb of the index SNP that have r<sup>2</sup> &gt;0.2 with the index SNP.</p><p>Genes identified in the meta-analysis were evaluated in the Raine Study, MrOS-Sweden, and Edinburgh type 2 Diabetes Study (ET2DS). Raine Study and MrOS-Sweden had GWAS data so we extracted the replication SNP results, and ET2DS was genotyped at the Wellcome Trust Clinical Research Facility Genetics Core Laboratory in Edinburgh using the OpenArray genotyping platform. rs2749527 failed manufacture for the OpenArray, and was replaced with rs2749529 (r<sup>2</sup>&#8202;=&#8202;0.905, D&#8242;&#8202;=&#8202;1.0) in all replication cohorts. Genotypic association analysis in these studies followed the same methods as those described above for the discovery meta-analysis, adjusting for age and sex.</p><p>We performed meta-analyses of sex-specific GWAS and conditional GWAS in a subset of populations, using single marker association tests of z-scores of log-transformed cortisol. For the sex-specific analysis, each study adjusted for age in both men (n&#8202;=&#8202;3,546) and women (n&#8202;=&#8202;5,956). For the conditional analysis, each study adjusted for age, sex, and each of the <italic>SERPINA6</italic> loci SNPs rs12589136 (n&#8202;=&#8202;9,308), rs11621961 (n&#8202;=&#8202;7,687), and rs2749527 (n&#8202;=&#8202;9,307) individually. We then did fixed effects meta-analysis using GWAMA program.</p><p>We used the Versatile Gene-based Association Study (VEGAS) program <xref ref-type="bibr" rid="pgen.1004474-Liu1">[22]</xref> to perform gene-centric analysis. This used individual SNP p-values derived from the meta-analysis results to compute a gene-based p-value. We used two methods: all SNPs within a gene, or a subset of the 10% most significant SNPs in each gene boundary. VEGAS accounted for linkage disequilibrium between SNPs using the HapMap phase 2 population (CEU). SNPs were assigned to &#8764;18,000 genes based on positions in build 36 (hg18), with gene boundaries of &#177;50 kb of the UTR. Bonferroni corrected threshold for gene-wide significance was 3&#215;10<sup>&#8722;6</sup>. The overlap of SNPs included in the gene boundaries in our results indicates this is likely to be an overly conservative correction factor <xref ref-type="bibr" rid="pgen.1004474-Liu1">[22]</xref>.</p><p>A list of 61 candidate genes with known biological function in the regulation of cortisol was compiled by a panel of experts in the field. All SNPs within and &#177;50 kb of these genes were examined in the GWAMA results, and gene-based p values were inspected in VEGAS results.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genotypes for the <italic>SERPINA6/SERPINA1</italic> gene region (chr14:94,770,585&#8211;94,857,029, build 37) in the CROATIA-Korcula samples (n&#8202;=&#8202;898) were extracted from an Illumina Exome Chip v1 analysis. Genotypes were called in GenomeStudio (Illumina) using the CHARGE Consortium joint calling cluster file (<ext-link ext-link-type="uri" ns0:href="http://www.chargeconsortium.com/main/exomechip">http://www.chargeconsortium.com/main/exomechip</ext-link>) <xref ref-type="bibr" rid="pgen.1004474-Grove1">[48]</xref>. 1000 Genomes imputation was performed using genotypes from Illumina HumanHap370CNV after quality control (Individual Call Rate 97%, SNP Call Rate 98%, MAF 0.01, HWE 1&#215;10<sup>&#8722;6</sup>); prephasing was performed using ShapeIt v2 <xref ref-type="bibr" rid="pgen.1004474-Delaneau1">[49]</xref> and imputation using IMPUTE2 <xref ref-type="bibr" rid="pgen.1004474-Howie1">[50]</xref> and the ALL (Phase 1 integrated release v3, April 2012) reference panel. Associations with plasma cortisol were analysed in GenABEL <xref ref-type="bibr" rid="pgen.1004474-Aulchenko1">[51]</xref>.</p><p>More detailed phenotyping was undertaken in samples from 316 participants in the CROATIA-Korcula cohort, comprising 158 age- and sex-matched homozygotes at the top hit variant rs12589136 (53 T/T, 106 G/G; however, one T/T sample had insufficient sample for CBG measurement resulting in 52 T/T and 106 G/G). An additional 160 randomly selected samples had CBG measured, however two samples failed genotyping resulting in an additional 47 T/G and 111 G/G.</p><p>Total plasma cortisol was measured with a commercial radioimmunoassay (MP Biomedicals, UK). Total CBG was also measured in CROATIA-Korcula samples using a commercial radioimmunoassay (DiaSource, Louvain-la-Neuve, Belgium). Total cortisol binding capacity was measured using a ligand-saturation assay, with [<sup>3</sup>H]-cortisol (PerkinElmer Life Sciences, Waltham, MA) as the labelled ligand and dextran-coated charcoal to separate the CBG-bound [<sup>3</sup>H]cortisol, as previously described <xref ref-type="bibr" rid="pgen.1004474-Hammond2">[52]</xref>. &#945;1-Antitrypsin was measured with a commercial ELISA (Genway Biotech, San Diego, USA). Unbound free plasma cortisol was calculated from measured total plasma cortisol and total CBG values using Coolens' equation <xref ref-type="bibr" rid="pgen.1004474-Coolens1">[53]</xref>. Free cortisol was also measured by ELISA following equilibrium dialysis. Briefly, dialysis tubing (12&#8211;14 kD, Medicell, London, UK) was heated to 80C for 30 min in 2% Na bicarbonate and 1 mM EDTA before overnight dialysis of plasma into phosphate buffered saline containing 1% gelatin at 37C and measurement of dialysed free cortisol by commercial ELISA (Salimetrics Europe Ltd, Newmarket, UK). CBG were also measured, as previously described <xref ref-type="bibr" rid="pgen.1004474-Lewis2">[23]</xref>, by ELISAs using monoclonal antibodies 12G2 and 9G12. Antibody 9G12 binds to an epitope in the reactive centre loop (RCL), the elastase cleavage site on CBG, and has been used to infer intact (uncleaved) CBG, whereas antibody 12G2 binds to a distant epitope and measures total (cleaved and uncleaved) CBG.</p><p>As CROATIA-Korcula is a population isolate, we used the polygenic and mmscore functions in GenABEL <xref ref-type="bibr" rid="pgen.1004474-Aulchenko1">[51]</xref>. All regression equations included the first three principal components and kinship matrix derived from GWAS data in this population and used z-scores of the traits. All variables were normalised using log transformation (cortisol, calculated free cortisol, measured free cortisol, CBG, &#945;1-antitrypsin), and reported means are geometric means, the ratio of intact/cleaved was normally distributed.</p>