<p><italic>Drosophila</italic> stocks used in this study are as follows: <italic>GluRIIA</italic><sup><italic>SP16</italic></sup>, <italic>Df(2L)cl</italic><sup><italic>h4</italic></sup>, and <italic>UAS-GluRIIA</italic> [<xref ref-type="bibr" rid="pgen.1005810.ref025">25</xref>] (from A. DiAntonio, Washington University); <italic>hiw</italic><sup><italic>ND8</italic></sup> [<xref ref-type="bibr" rid="pgen.1005810.ref046">46</xref>]; <italic>imp&#223;11</italic><sup><italic>24</italic></sup> and <italic>imp&#223;11</italic><sup><italic>70</italic></sup> [<xref ref-type="bibr" rid="pgen.1005810.ref042">42</xref>]; <italic>twit</italic><sup><italic>MI06552</italic></sup> [<xref ref-type="bibr" rid="pgen.1005810.ref051">51</xref>]; <italic>mad</italic><sup><italic>12</italic></sup> [<xref ref-type="bibr" rid="pgen.1005810.ref091">91</xref>]; <italic>mad</italic> deficiency <italic>Df(2L)C28</italic> [<xref ref-type="bibr" rid="pgen.1005810.ref092">92</xref>]; <italic>UAS-Mad-GFP</italic> [<xref ref-type="bibr" rid="pgen.1005810.ref009">9</xref>] (from M. Gonzalez-Gaitan, University of Geneva); <italic>UAS-Mad-Myc [<xref ref-type="bibr" rid="pgen.1005810.ref062">62</xref>]</italic>; <italic>UAS-T7-Mad</italic><sup><italic>S25A</italic></sup> [<xref ref-type="bibr" rid="pgen.1005810.ref061">61</xref>]; <italic>gbb</italic><sup><italic>1</italic></sup> and <italic>gbb</italic><sup><italic>2</italic></sup> [<xref ref-type="bibr" rid="pgen.1005810.ref093">93</xref>]; <italic>sax</italic><sup><italic>4</italic></sup> [<xref ref-type="bibr" rid="pgen.1005810.ref094">94</xref>]; <italic>wit</italic><sup><italic>&#916;C</italic>.<italic>genomic</italic></sup>, <italic>wit</italic><sup><italic>A12</italic></sup> and <italic>wit</italic><sup><italic>B11</italic></sup> [<xref ref-type="bibr" rid="pgen.1005810.ref015">15</xref>] (from M. O&#8217;Connor, University of Minnesota); <italic>nwk</italic><sup><italic>1</italic></sup>, <italic>nwk</italic><sup><italic>&#947;3</italic></sup> [<xref ref-type="bibr" rid="pgen.1005810.ref095">95</xref>] (from K. O&#8217;Connor-Giles, University of Wisconsin); <italic>nrx</italic><sup><italic>273</italic></sup> and <italic>nlg1</italic><sup><italic>&#916;46</italic></sup> [<xref ref-type="bibr" rid="pgen.1005810.ref096">96</xref>] (from B. Mozer, NIH); <italic>BG380-Gal4</italic> [<xref ref-type="bibr" rid="pgen.1005810.ref097">97</xref>]; <italic>elav-Gal4</italic> (BL-8760); <italic>24B-Gal4</italic> (BL-1716); <italic>G14-Gal4</italic> and <italic>MHC-Gal4</italic> (from C. Goodman, University of California at Berkeley). For <italic>UAS-T7-Mad</italic><sup><italic>S25D</italic></sup>, the T7-tagged Mad<sup>S25D</sup> [<xref ref-type="bibr" rid="pgen.1005810.ref061">61</xref>] was cloned into pUAST and transgenic lines were generated by germline transformation (BestGene). For <italic>RNAi</italic>-mediated knockout we used UAS-put-RNAi and UAS-mad-RNAi (ID 848 and respectively 12635, Vienna Drosophila RNAi Center) and TRiP lines generated by the Transgenic RNAi Project, Harvard Medical School, <italic>GluRIIA</italic> (<italic>P[TRiP</italic>.<italic>JF02647]attP2</italic>), and <italic>mav</italic> (<italic>P[TRiP</italic>. <italic>HMS01125]attP2</italic> and <italic>P[TRiP</italic>. <italic>GL01025]attP40)</italic>.</p><p>The flies were reared on Jazz-Mix food (Fisher Scientific). To control for larvae crowding, 8&#8211;10 females were crossed with 5&#8211;7 males per vial and were passed to fresh vials every 3 days. For rearing on yeast, embryos were collected on grape agar plates for 24 hours, incubated at 25&#176;C for 24 hours and then 50 first instar larvae of appropriate genotype were transferred to a vial of standard fly food containing a 20 mm<sup>2</sup> paper saturated with 20% (w/v) active dry baker&#8217;s yeast in water. Larvae remained on paper and did not burrow into food. Fresh yeast solution was added daily to keep paper saturated. Larvae were kept on yeast at 25&#176;C until reaching third instar stage.</p><p>Larvae were dissected as described previously in ice-cooled Ca<sup>2+</sup>-free HL-3 solution [<xref ref-type="bibr" rid="pgen.1005810.ref098">98</xref>,<xref ref-type="bibr" rid="pgen.1005810.ref099">99</xref>]. The samples were fixed in either 4% formaldehyde (Polysciences, Inc.) for 25 min or in Bouin&#8217;s fixative (Bio-Rad) for 3 min and washed in PBS containing 0.5% Triton X-100. Primary antibodies from Developmental Studies Hybridoma Bank were used at the following dilutions: mouse anti-GluRIIA (MH2B), 1:200; rat anti-Elav (7E8A10), 1:200; mouse anti-Bruchpilot (Brp) (Nc82), 1:200. Other primary antibodies were as follows: rabbit anti-phosphorylated Mothers against decapentaplegic (pMad), 1:500, (a gift from Carl Heldin) [<xref ref-type="bibr" rid="pgen.1005810.ref100">100</xref>]; rabbit anti-pSmad3, 1:500, (Epitomics, [<xref ref-type="bibr" rid="pgen.1005810.ref010">10</xref>]); FITC-, rhodamine-, and Cy5- conjugated goat anti-HRP, 1:1000 (Jackson ImmunoResearch Laboratories, Inc.); rabbit anti-GFP, 1:250 (Abcam); rat anti-Neto, 1:1000 [<xref ref-type="bibr" rid="pgen.1005810.ref037">37</xref>]; Cy5- conjugated goat anti-HRP, 1:1000 (Jackson ImmunoResearch Laboratories, Inc.). The rabbit polyclonal anti-GluRIIB and anti-GluRIIC were generated as previously described [<xref ref-type="bibr" rid="pgen.1005810.ref026">26</xref>] against synthetic peptides ASSAKKKKKTRRIEK, and QGSGSSSGSNNAGRGEKEARV respectively (Pacific Immunology Corp). Alexa Fluor 488-, Alexa Fluor 568-, and Alexa Fluor 647- conjugated secondary antibodies (Molecular Probes) were used at 1:400.</p><p>Larval filets were mounted in ProLong Gold and brains were mounted in SlowFade Gold (Invitrogen). Samples of different genotypes were processed simultaneously and imaged under identical confocal settings using laser scanning confocal microscopes (CarlZeiss LSM780). Boutons were counted using anti-HRP immunoreactivities. All quantifications were performed while blinded to genotype. The numbers of samples analyzed are indicated inside the bars.</p><p>All the pMad data quantified here were obtained using the anti-pMad serum from Carl Heldin. For these analyses, the samples were fixed with either 4% formaldehyde or Bouin fixative as described above, washed extensively in PBS containing 0.5% Triton X-100, marked by distinct cuts per genotype and pooled together in the same tube, and, without any blocking agent, incubated overnight at 4&#176;C with anti-pMad 1:500 plus other relevant primary antibodies. The samples were then washed with PBS containing 0.5% Triton X-100, incubated with secondary antibodies (1:200) and no blocking agent for either 2 hours at room temperature or overnight at 4&#176;C. After extensive washes with PBS containing 0.5% Triton X-100, the samples were mounted as above, then imaged and quantified together.</p><p>For motor neuron nuclei, confocal regions of interest (ROIs) were determined with Imaris software (Bitplane) by using the &#8220;spots&#8221; feature to automatically identify motor nuclei using Elav immunoreactivity. Spots were verified manually and mean center intensity for all nuclei in a given sample was recorded. This procedure was repeated for all samples of a given genotype and the mean was used for comparison between genotypes. For NMJ signal quantifications, mean signal intensity within the ROI encompassing the synaptic area was normalized to HRP signal. To determine Brp intensity per puncta, individual puncta within the ROI were manually counted and total Brp intensity was divided by the number of puncta. Student&#8217;s <italic>t</italic> test was performed using Sigma Plot (Systat) to evaluate statistical significance. All graphs represent mean value of all samples of the given genotype &#177; SEM.</p><p>Samples were prepared as described for immunohistology and mounted using #1.5 cover glasses (Cat. 12-541-B, Fisher Scientific). 5 phases and 3 rotations of 3D SIM images were captured using a Zeiss Elyra microscope. The interval for all z stacks was 110nm. Channels were aligned using parameters obtained from calibration measurements with 100 nm TetraSpeck beads. Zeiss SIM images were taken with a 100X 1.46 NA oil objective and a PCO edge sCMOS camera (16 bit images). Laser power and exposure time were optimized to use a large portion of the camera&#8217;s dynamic range while minimizing bleaching. As a part of the reconstruction processing using the Zeiss Zen software, Wiener filtering was carefully optimized to maximize resolution and minimize artifacts. The estimated resolution after reconstruction was ~100 nm lateral and ~250 nm axial. Using the Zeiss Zen software, we generated intensity profiles across structures of interest and exported the table containing the fluorescence intensity as a function of distance. To measure distances we calculated the distances between intensity peaks. Surface rendering was performed using Imaris software.</p><p>Recordings were performed on muscle 6, segment A3 of third instar larvae as previously reported [<xref ref-type="bibr" rid="pgen.1005810.ref029">29</xref>]. Briefly, wandering third instar larvae were dissected in ice-cold, calcium-free physiological HL-3 saline [<xref ref-type="bibr" rid="pgen.1005810.ref098">98</xref>], and immersed in HL-3 containing Ca<sup>2+</sup> before being shifted to the recording chamber. The calcium-free HL-3 saline contains (in mM): 70 NaCl, 5 KCl, 20 MgCl<sub>2</sub>, 10 HCO3, 5 trehalose, 115 sucrose, 5 HEPES, pH adjusted to 7.2 at room temperature. The recording solution was HL-3 with either 0.4 or 0.5 mM CaCl<sub>2</sub> as described in the text. Intracellular electrodes (borosilicate glass capillaries of 1 mm diameter) were filled with 3 M KCl and resistances ranged from 12 to 25 M&#937;. Recordings were done at room temperature from muscle cells with an initial membrane potential between &#8211;50 and &#8211;70 mV, and input resistances of &#8805; 4 M&#937;. For mEJCs recording the muscle cells were clamped to &#8211;80 mV. To calculate mean amplitudes and frequency of mEJCs or mEJP, 100&#8211;150 events from each muscle were measured and averaged using the Mini Analysis program (Synaptosoft). Minis with a slow rise and falling time arising from neighboring electrically coupled muscle cells were excluded from analysis [<xref ref-type="bibr" rid="pgen.1005810.ref101">101</xref>,<xref ref-type="bibr" rid="pgen.1005810.ref102">102</xref>]. To measure the decay time constant of mEJCs, 20&#8211;30 clear representative events from each recording were averaged and fit by a single exponential function. For evoked EJP recordings, the nerve roots were cut near the exiting site of the ventral nerve cord so that the motor nerve could be picked up by a suction electrode. Following motor nerve stimulation with a suction electrode (100 &#956;s, 5 V), evoked EJPs were recorded. Four to six EJPs evoked by low frequency of stimulation (0.1 Hz) were averaged. Quantal content was calculated by dividing the mean EJP by the mean mEJP. Since data were recorded in low calcium saline (0.4 mM Ca<sup>2+</sup>) no correction was made for nonlinear summation. Statistical analysis used KaleidaGraph 4.5 (Synergy Software).</p><p>Electrical signals were recorded with an Axoclamp 2B amplifier (Axon Instruments). The signals were filtered at 1 kHz and digitized at 10 kHz by using an analog-digital converter (Digidata 1440A) and pCLAMP software (version 10.0, Axon Instruments). Data are presented as mean &#177;SEM. One-way ANOVA followed by a Tukey&#8217;s post hoc test was used to assess statistically significant differences among genotypes. Differences were considered significant at p &lt; 0.05.</p>