<p>All specimens for sequencing were obtained from patients with appropriate consent from the local institutional review board in accordance with the Children's Oncology Group and the National Cancer Institute. Clinical samples were obtained from collaborations with the Cooperative Human Tissue Network, the Children's Hospital of Westmead, Australia, the Children's Oncology Group, and the National Institutes of Health Clinical Center. Tumors were classified as a Ewing sarcoma family tumor by a sarcoma pathologist and the host institution using standing histological techniques. Fifty-two tumors were from the primary disease site and had not been exposed to previous treatment. Fifteen tumors were from recurrent/metastatic sites and for five tumors we lacked this clinical information. Clinical and pathological data for the sequencing cohort are summarized in <xref ref-type="supplementary-material" rid="pgen.1004475.s019">Table S9</xref>. Tumor samples were evaluated by a pathologist for the presence of more than 70% tumor content before DNA/RNA extraction and sequencing. DNA was extracted from qualifying tumor samples and matched blood using either AllPrep Mini (Qiagen) or Agencourt Genefind v2 (Beckman Coulter) DNA extraction kits. RNA was extracted using the RNeasy Micro Kits according to the manufacturer's protocol (Quiagen). Genotyping confirmed independence of these samples.</p><p>All EFT cell lines used in the study underwent short tandem repeat (STR) profiling for independence testing and all were confirmed to have a unique profile. This characterization is described in detail in <xref ref-type="supplementary-material" rid="pgen.1004475.s020">Table S10</xref>.</p><p>Tissue microarrays were obtained from an independent cohort of genetically confirmed Ewing sarcoma cases <xref ref-type="bibr" rid="pgen.1004475-LopezGuerrero2">[30]</xref>. The associated clinical information is summarized in <xref ref-type="supplementary-material" rid="pgen.1004475.s018">Table S8</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Whole genome paired-end sequencing was performed using the Complete Genomics platform. Data analysis was accomplished using the CGA tools package v2.0 <xref ref-type="bibr" rid="pgen.1004475-Drmanac1">[57]</xref>, ANNOVAR v2012-05-25 <xref ref-type="bibr" rid="pgen.1004475-Wang1">[58]</xref>, and Circos v0.52 <xref ref-type="bibr" rid="pgen.1004475-Krzywinski1">[59]</xref> in build hg19. Somatic variants were determined first by comparison to the matched normal DNA. To remove artifacts specific to the sequencing platform, we eliminated any somatic variants also found in normal samples [50 in-house samples and 69 Complete Genomics samples (<ext-link ext-link-type="uri" ns0:href="http://www.completegenomics.com/public-data/69-Genomes/">http://www.completegenomics.com/public-data/69-Genomes/</ext-link>)]. The Somatic Score (<ext-link ext-link-type="uri" ns0:href="http://media.completegenomics.com/documents/DataFileFormats+Cancer+Pipeline+2.0.pdf">http://media.completegenomics.com/documents/DataFileFormats+Cancer+Pipeline+2.0.pdf</ext-link>) is based on a Bayesian model and takes account of read depth, base call quality, mapping/alignment probabilities, and measured priors on sequencing error rate for both the germline and tumor variants. Verification by Sanger sequencing was performed on all high-confidence somatic variant calls (by default Somatic Score&gt;&#8202;=&#8202;&#8722;10) affecting protein coding or a splice site (SNVs, substitutions, insertions, deletions), including 55 SNVs. We determined that more stringent somatic score cut-off was required in our cohort to achieve adequate positive predictive value of variants calls, likely due to the low mutation rate in our tumor type. Relative to all high-confidence variant calls we established a sensitivity of 86.7% and specificity of 90.7% for a Somatic Score cut-off of 3. Somatic mutations at or above this score and all verified mutations with lower scores were used for further analysis.</p><p>The Complete Genomics somatic copy number segmentation is based on 2-kb windows and utilizes coverage in the matched germline sample for normalization of the tumor sample coverage. Lesser allele fraction (LAF) calculations are based on allele read counts in the tumor at loci that are called heterozygous in the matched germline sample. In addition to default filtering done by the Complete Genomics segmentation algorithm, copy number variants were considered high-confidence if they were either large (&gt;&#8202;=&#8202;10 kb AND containing &gt;&#8202;=&#8202;10 heterozygous SNPs for LAF calculation) OR highly altered (homozygous deletions and focal amplifications &gt;&#8202;=&#8202;5 copies) OR supported by somatic junction(s) (ex: junction detected spanning both ends of a region of LOH). Somatic junctions were called using CGAtools and junctions were filtered by footprints smaller than 70 bases, less than 10 discordant mate pairs, under-represented repeats, and presence in the baseline set of 69 Complete Genomics genomes. We additionally filtered junctions that were present in 50 in-house germline DNA samples that were sequenced on the same platform.</p><p>Genomic sequencing was performed using a custom multiplex PCR designed to include the entire coding sequence of the majority of altered genes in the whole-genome sequencing discovery cohort as well as <italic>TP53</italic> and in total encompassing 106.3 kB of target region (<xref ref-type="supplementary-material" rid="pgen.1004475.s021">Table S11</xref>). Primers for the targeted sequencing were designed using the Ion Ampliseq designer (Life Technologies) and sequencing was performed on the IonTorrent PGM (Life Technologies). PGM sequencing data was analyzed using Torrent Suite software v3.2 (Life Technologies) and ANNOVAR. Genomic sequencing was performed to an average mean coverage of 311&#215; and variants were called using high-confidence thresholds and filtered to include only those that are protein altering and unreported or rare (population allele frequency &lt;0.005) in the dbSNP and 1000 genomes databases. Mutations of interest were verified by capillary sequencing with a &lt;5% false positive rate. Sequence coverage data was calculated at a position, exon and gene levels to look for structural alterations of the recurrently mutated genes (<xref ref-type="supplementary-material" rid="pgen.1004475.s008">Figure S8</xref>). Coverage data was visualized using the Integrated Genomics Viewer.</p><p>PolyA selected RNA libraries were prepared for sequencing on the Illumina HiSeq2000 using TruSeq v3 chemistry according to the manufacturer's protocol (Illumina). RNA sequencing was performed with an average yield of 18.6 Gb per sample. Raw reads were mapped using to ENSEMBL reference (hg19) using TopHat2.0 <xref ref-type="bibr" rid="pgen.1004475-Kim2">[60]</xref>. Fusion analysis was done using TopHat 2.0 and DeFuse 0.6 <xref ref-type="bibr" rid="pgen.1004475-McPherson1">[61]</xref>. The 3 alternated fusions described were confirmed using RT-PCR using flanking primers and Sanger sequencing of the resultant product.</p><p>Expression FPKM results were obtained at both gene and transcript level using CuffLinks 2.1 <xref ref-type="bibr" rid="pgen.1004475-Trapnell1">[62]</xref>. The log2 FPKM expression results from TopHat mapping were median-normalized using in-house data from 63 normal tissue samples. Exon level expression was calculated using the formula RPKM&#8202;=&#8202;(r * 10<sup>9</sup>)/(f * R), with r being the number of reads mapped to an exon, f being the exon length, and R being the total read count of the sample. Hierarchical clustering was performed on normalized log2 FPKM expression values at the gene level using Euclidean distance and Ward agglomeration method.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For variant detection, samtools (<ext-link ext-link-type="uri" ns0:href="http://samtools.sourceforge.net/">http://samtools.sourceforge.net/</ext-link>) is used to count the number of reads uniquely mapped to a position found as variant in DNA sequencing of the same sample or a position of interest based on a mutation being present in the TCGA (<ext-link ext-link-type="uri" ns0:href="http://cancergenome.nih.gov/">http://cancergenome.nih.gov/</ext-link>) or compared to the reference genome hg19 in genes of interest. If there are reads supporting a variant base then the total reads supporting it are counted and variant allele frequency is calculated.</p><p>SNP arrays were performed on the 6 tumor whole genome sequencing cohort to confirm copy number findings. SNP array analysis was conducted on HumanOmni2.5 or HumanOmni5 arrays (Illumina) and the data were analyzed with GenomeStudio (Illumina) and Nexus Copy Number v7 (Biodiscovery Inc.). Copy number state and allelic ratio was manually assessed in all areas of copy number variation and structural variation predicted by WGS and was concordant with WGS prediction in 33/35 (94%).</p><p>Individual exons of STAG2 were PCR amplified from genomic DNA using the conditions and primer pairs previously described <xref ref-type="bibr" rid="pgen.1004475-Solomon1">[29]</xref>. PCR products were purified using the Exo/SAP method followed by a Sephadex spin column. Sequencing reactions were performed using BigDye v3.1 (Applied Biosystems) using an M13F primer and were analyzed on an Applied Biosystems 3730&#215;l capillary sequencer. Sequences were analyzed using Mutation Surveyor (SoftGenetics). Traces with putative mutations were reamplified and sequenced from both tumor and matched normal DNA from blood when available.</p><p>A mouse monoclonal antibody to STAG2 from Santa Cruz Biotechnology (clone J-12, sc-81852) was used at a dilution of 1&#8758;100. Immunostaining was performed in an automated immunostainer (Leica Bond-Max) following heat-induced antigen retrieval for 30 min in high pH epitope retrieval buffer (Bond-Max). Primary antibody was applied for 30 min, and Bond-Max polymer was applied for 15 min. Diaminobenzidine was used as the chromogen, and samples were counterstained with hematoxylin. Samples in which both the tumor and normal cells failed to stain for STAG2 were considered antigenically non-viable and were excluded from the analysis.</p><p>Primary antibodies used were STAG2 clone J-12 (Santa Cruz Biotechnology), p53 clone 7F5 (Cell Signaling), p16 (BD Pharmingen #554079), p21 clone DCS60 (Cell Signaling), and &#945;-tubulin Ab-2 clone DM1A (Neomarkers). Protein was isolated from EFT cell lines in RIPA buffer, resolved by SDS-PAGE, and immunoblotted following standard biochemical techniques.</p><p>For the BRCA2 K3326X polymorphism, a two-tailed Fisher Exact Test was used to calculate p-value for the Odds Ratio significantly different from 1. For RNA expressional analysis, a two-tailed student T Test assuming unequal variances was used to calculate a p-value for difference in population means. For tissue microarrays, the p-value for differences in frequency of STAG2 mutation in primary and recurrent/metastatic samples was calculated using two-tailed Fischer Exact Test. P-value for association of STAG2 expression with overall survival was calculated using univariate analysis.</p>