<p>All animal work was performed according to one of the following protocols: 1) the <italic>Guide for the Care and Use of Laboratory Animals</italic> under approved IACUC animal use protocols within the AAALAC accredited program at the University of North Carolina at Chapel Hill (Animal Welfare Assurance Number: A-3410&#8211;01); 2) the requirements of The Jackson Laboratory Animal Ethics Committees under approved protocol #JAX10001; 3) an animal protocol approved by the North Carolina State University Institutional Animal Care and Use Committee (09&#8211;0133-B); or 4) an animal study protocol approved by the NCI Animal Care and Use Committee (ASP# LCBG-013). All animals were euthanized according to the regulations of the governing protocol.</p><p>The G2:F1 population has been previously reported and was genotyped on the Mouse Diversity Array [<xref ref-type="bibr" rid="pgen.1004850.ref072">72</xref>] (MDA). A population of 96 (FVB/NJx(WSB/EiJxPWK/PhJ)F1)G2 mice was previously reported and was genotyped on the MegaMUGA array [<xref ref-type="bibr" rid="pgen.1004850.ref040">40</xref>,<xref ref-type="bibr" rid="pgen.1004850.ref053">53</xref>]. DNAs from selected progeny from previously published (C57BL/6JxSPRET/EiJ)xC57BL/6J and (A/JxSPRET/EiJ)xA/J backcrosses [<xref ref-type="bibr" rid="pgen.1004850.ref026">26</xref>,<xref ref-type="bibr" rid="pgen.1004850.ref043">43</xref>] were regenotyped on the MegaMUGA array. The SPRET/EiJ strain designation had not yet been assigned to the inbred strain at the time the backcross was performed [<xref ref-type="bibr" rid="pgen.1004850.ref026">26</xref>]. Finally, DNA from multiple samples from the (M16ixL6)F2 intercrosses and from generations 4 and 10 of the (HR8xC57BL/6J) advanced intercross line [<xref ref-type="bibr" rid="pgen.1004850.ref029">29</xref>,<xref ref-type="bibr" rid="pgen.1004850.ref031">31</xref>,<xref ref-type="bibr" rid="pgen.1004850.ref032">32</xref>] were genotyped at markers closely linked to <italic>R2d2</italic>.</p><p>
<bold>Crosses 1&#8211;2, 7&#8211;10 and 16&#8211;17 (<xref ref-type="table" rid="pgen.1004850.t001">Table 1</xref>)</bold>. WSB/EiJ and C57BL/6J were used in reciprocal combinations. Male F1 hybrids were backcrossed to C57BL/6J to produce the progeny of crosses 1 and 2. Female F1 hybrids were backcrossed to C57BL/6J to produce the progeny of crosses 16 and 17. The progeny of crosses 7&#8211;10 was produced in a similar way to crosses 16 and 17, except that female F1 of reciprocal matings of WSB/EiJ and CAST/EiJ were used for crosses 7 and 8, and female F1 of reciprocal matings of WSB/EiJ and PWD/PhJ were used for crosses 9 and 10. All breeding was done at the Jackson Laboratory (Bar Harbor, ME).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<bold>All other crosses</bold>. DO mice and standard mouse inbred strains (129S1/SvImJ, A/J, C57BL/6J, CAST/EiJ, FVB/NJ, NU/J, NOD/ShiLtJ, NZO/H1LtJ, PWK/PhJ and WSB/EiJ) were obtained from The Jackson Laboratory (Bar Harbor, ME). CC mice were obtained from the Systems Genetics Core Facility colony at UNC Chapel Hill [<xref ref-type="bibr" rid="pgen.1004850.ref073">73</xref>] (<ext-link ext-link-type="uri" ns0:href="http://csbio.unc.edu/CCstatus/index.py">http://csbio.unc.edu/CCstatus/index.py</ext-link>). Those mice were used to generate the following number and types of hybrid mice: nine (129S1/SvImJxWSB/EiJ)F1 females; two (A/JxWSB/EiJ)F1 females; seven (NOD/ShiLtJxWSB/EiJ)F1 females; six (NZO/HILtJxWSB/EiJ)F1 females; 10 (CC042/GeniUncxCC001/Unc)F1 females; three (CC001/UncxCC039/Unc)F1; nine (DOxCC001/Unc)F1 females, 13 (DOxCC005/Tau Unc)F1 females and five (NU/JxC57BL/6J). F1 females were mated to FVB/NJ males and cages were surveyed three to five times per week. Litter sizes were recorded and pups were sacrificed at birth, and tissue was collected for DNA isolation. The same breeding schema was followed with 127 DO <italic>R2d</italic> heterozygous females used to determine the origin of maternal TRD. All breeding was done at UNC Chapel Hill (Chapel Hill, NC).</p><p>A single G13 DO female (DO-G13&#8211;44) was mated to a male that was the result of an intercross between four CC lines (CC013/GeniUnc, CC053/Unc, CC065/Unc and CC008Geni/Unc; <xref ref-type="fig" rid="pgen.1004850.g004">Fig. 4</xref>). G3 female progeny were weaned, single housed and mated to FVB/NJ males. Cages were surveyed three to five times per week. Litter sizes were recorded and G4 pups were sacrificed at birth, and tissue was collected for DNA isolation.</p><p>TR was measured in G3 dams as described above. Each dam was classified as having TRD (<italic>p</italic> &lt; 0.05 for 1-df <italic>&#935;</italic>
<sup><italic>2</italic></sup> test of null hypothesis TR = 0.5) or not having TRD (<italic>p</italic> &#8805; 0.05). Both G2 parents and G3 dams were genotyped on MegaMUGA and phased haplotypes at <italic>R2d2</italic> were inferred by manual inspection of haplotype reconstructions. In order to isolate the contribution of maternal and paternal alleles to TRD, MegaMUGA markers called as H in the G2 dam and homozygous in the G2 sire were retained for mapping, and presence of TRD was mapped as a binary phenotype using a logistic regression analog to the Haley-Knott method. The procedure was repeated using only markers called as H in the father of the G3 dams and homozygous in the mother. Significance thresholds for LOD scores were obtained by unrestricted permutation.</p><p>
<bold>Crosses 1&#8211;2, 7&#8211;10 and 16&#8211;17 (<xref ref-type="table" rid="pgen.1004850.t001">Table 1</xref>)</bold>. DNA was prepared from spleens of 21-day old mice. DNA extraction and SNP genotyping were carried out as described previously [<xref ref-type="bibr" rid="pgen.1004850.ref074">74</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<bold>All other samples</bold>. DNA for PCR-based genotyping was performed on crude whole genomic DNA extracted by heating tissue in 100ul of 25mM NaOH/0.2mM EDTA at 95&#176;C for 60 minutes followed by the addition of 100ul of 40mM Tris-HCl. The samples were then spun at 2000 rpm for 10 minutes and the supernatant collected for use as PCR template. All primers (<xref ref-type="supplementary-material" rid="pgen.1004850.s018">S6 Table</xref>) used in this study were designed using PrimerQuest software (<ext-link ext-link-type="uri" ns0:href="https://www.idtdna.com/Primerquest">https://www.idtdna.com/Primerquest</ext-link>). PCR reactions contained 1.5&#8211;2 mM MgCl2, 0.2&#8211;0.25 mM dNTPs, 0.2&#8211;1.8 &#956;M of each primer and 0.5&#8211;1 units of GoTaq polymerase (Promega) in a final volume of 10&#8211;50 &#956;L. Cycling conditions were 95&#176;C, 2 min, 35 cycles at 95&#176;, 55&#176; and 72&#176;C for 30 sec each, with a final extension at 72&#176;C, 7 min. PCR products were loaded into a 2% agarose gel and run at 200 V for 40&#8211;120 minutes (depending on the marker). Genotypes were scored and recorded.</p><p>DNA for MegaMUGA genotyping was isolated as described previously [<xref ref-type="bibr" rid="pgen.1004850.ref040">40</xref>,<xref ref-type="bibr" rid="pgen.1004850.ref053">53</xref>]. Briefly, ~2 mm of mouse tail (5 mg) was harvested, flash-frozen on dry ice and digested with proteinase K overnight at 65&#176;C. The following day, DNA was extracted using the QIAGEN Puregene Gentra kit (kit no. 158389; QIAGEN GmbH, Hilden Germany). Genotyping was performed with the MegaMUGA genotyping microarray (Neogen/GeneSeek, Lincoln, NE), a 78,000-probe array based on the Illumina Infinium platform.</p><p>
<bold>Genotyping by TaqMan</bold>. After <italic>R2d2</italic> was established as the causal variant for TRD, a subset of DO-G16 progeny and all (M16i x L6)F2 intercross progeny were genotyped using TaqMan real-time PCR assays for <italic>Cwc22</italic>. Samples heterozygous for a high-copy allele at <italic>R2d2</italic> can be readily distinguished from samples homozygous for a low-copy allele based on the normalized cycle threshold value estimated from the assay (see section &#8220;Copy-number validation&#8221; below).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">
<bold>Deviation from Mendelian transmission</bold>. TR is reported as the ratio of the WSB/EiJ genotype to the total number of genotypes: WSB / (WSB + nonWSB). <italic>P</italic> values for aggregate data were calculated using a <italic>&#935;</italic>
<sup><italic>2</italic></sup> goodness-of-fit test of the observed number of WSB/EiJ genotypes compared to the number of WSB/EiJ genotypes expected under the null hypothesis of equal transmission:
<disp-formula id="pgen.1004850.e001"><alternatives><graphic id="pgen.1004850.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1004850.e001.jpg" /><ns1:math id="M1"><ns1:msup><ns1:mrow><ns1:mi>X</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msup><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msup><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>W</ns1:mi><ns1:mi>S</ns1:mi><ns1:mi>B</ns1:mi><ns1:mo>-</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mi>W</ns1:mi><ns1:mi>S</ns1:mi><ns1:mi>B</ns1:mi><ns1:mo>+</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>W</ns1:mi><ns1:mi>S</ns1:mi><ns1:mi>B</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:mfrac><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msup></ns1:mrow><ns1:mrow><ns1:mfrac><ns1:mrow><ns1:mi>W</ns1:mi><ns1:mi>S</ns1:mi><ns1:mi>B</ns1:mi><ns1:mo>+</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>W</ns1:mi><ns1:mi>S</ns1:mi><ns1:mi>B</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:mfrac></ns1:mrow></ns1:mfrac></ns1:math></alternatives></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For individual dams, the small sample sizes (typically fewer than 50 total offspring) would lead to type II error; therefore, <italic>p</italic>-values were calculated using an exact binomial test. Confidence intervals for TRs were calculated using the binom R package (<ext-link ext-link-type="uri" ns0:href="http://cran.r-project.org/web/packages/binom/">http://cran.r-project.org/web/packages/binom/</ext-link>).</p><p>
<bold>Average litter size</bold>. Average litter size was calculated as the mean number of offspring counted soon after birth per litter per dam (&#177; standard deviation), including the number of viable embryos counted <italic>in utero</italic> in mid-gestation DO dams (unless otherwise noted).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The expected average litter size (ALS) of a dam under a model in which lethality is the sole explanation for TRD is:
<disp-formula id="pgen.1004850.e002"><alternatives><graphic id="pgen.1004850.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1004850.e002.jpg" /><ns1:math id="M2"><ns1:mi>A</ns1:mi><ns1:mi>L</ns1:mi><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>O</ns1:mi><ns1:mi>b</ns1:mi><ns1:mi>s</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mi>A</ns1:mi><ns1:mi>L</ns1:mi><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>E</ns1:mi><ns1:mi>x</ns1:mi><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>(</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>-</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>T</ns1:mi><ns1:mi>R</ns1:mi><ns1:mo>-</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>T</ns1:mi><ns1:mi>R</ns1:mi></ns1:mrow></ns1:mfrac><ns1:mo>)</ns1:mo></ns1:math></alternatives></disp-formula>, 
where <italic>ALS</italic>
<sub><italic>Exp</italic></sub> is the mean ALS in dams with no TRD [<xref ref-type="bibr" rid="pgen.1004850.ref041">41</xref>]. Significance of the deviation of <italic>ALS</italic>
<sub><italic>Obs</italic></sub> from <italic>ALS</italic>
<sub><italic>Exp</italic></sub> was determined using a Wilcox signed rank test.</p><p>
<bold>Inheritance of <italic>R2d2</italic> alleles</bold>. Similarly, the average absolute number of offspring inheriting each <italic>R2d2</italic> allele was calculated as the mean number of offspring per litter per dam having each of the possible genotypes. Significance was determined using a one-tailed Student <italic>t</italic>-test.</p><p>DO and F1 dams were euthanized by CO<sub>2</sub> asphyxiation 12&#8211;18 days after delivery of the previous litter and the uterus was dissected. The number of live embryos and reabsorbed (dead) embryos was recorded. Each live embryo was dissected to isolate DNA for genotyping. Tissue from each live embryo was harvested for DNA extraction and genotyping.</p><p>All MDA arrays were genotyped using MouseDivGeno [<xref ref-type="bibr" rid="pgen.1004850.ref057">57</xref>], and all MegaMUGA arrays were genotyped using Illumina BeadStudio. We plotted number of H and N calls (as a fraction of the total number of genotypes) for each group of similar samples and excluded outliers from further analysis. For CC lines, DO animals, CCxCC F1 females and DOxCC F1 females, we inferred haplotypes using probabilistic methods [<xref ref-type="bibr" rid="pgen.1004850.ref040">40</xref>,<xref ref-type="bibr" rid="pgen.1004850.ref075">75</xref>]. As an additional QC step, we grouped DO samples by generation and plotted the number of recombinations (counted as unique transitions in haplotype reconstructions) and removed outliers.</p><p>
<bold>CAST/EiJ allele in the CC G2:F1</bold>. Thirty-four MDA SNP probe sets were identified within <italic>R2d</italic> in the GRCm38 reference sequence (<xref ref-type="supplementary-material" rid="pgen.1004850.s015">S3 Table</xref>). We ensured that these probes were unique using BLAT [<xref ref-type="bibr" rid="pgen.1004850.ref076">76</xref>] to map them to the reference genome. In order to map the expansion allele present in the CAST/EiJ strain, phenotypes and genotypes were coded as follows. First, we applied a CCS transform [<xref ref-type="bibr" rid="pgen.1004850.ref077">77</xref>] to the mean intensity of all probes in each probe set using MouseDivGeno [<xref ref-type="bibr" rid="pgen.1004850.ref057">57</xref>] and summed the values for each sample to obtain the final phenotype value. Next, the genome was divided into a set of disjoint intervals whose boundaries were defined by the 21,933 unique recombination events inferred in the population [<xref ref-type="bibr" rid="pgen.1004850.ref040">40</xref>], so that no individual would be recombinant within any of the resulting intervals. Then, using haplotype reconstructions, individuals were coded as either heterozygous (CAST/not-CAST) or homozygous (not-CAST/not-CAST) within each interval (there are no CAST homozygous individuals in this population). Of 474 individuals, 144 with a WSB/EiJ allele in the middle of chromosome 2 were excluded to yield a final sample size of 330. A single-locus QTL scan was then performed via Haley-Knott regression [<xref ref-type="bibr" rid="pgen.1004850.ref078">78</xref>], treating the population as a backcross.</p><p>
<bold>WSB/EiJ allele in an intercross population</bold>. Three MegaMUGA SNP probes were identified within <italic>R2d</italic> in the GRCm38 reference (<xref ref-type="supplementary-material" rid="pgen.1004850.s016">S4 Table</xref>). Again, uniqueness was verified using BLAT. In order to map the expansion allele in WSB/EiJ, the sum intensity of these probes was used as a phenotype and genotypes were coded as follows. First the genome was divided into a grid of 1,000 disjoint intervals of approximately equal size, and one MegaMUGA SNP marker segregating between WSB/EiJ and PWK/PhJ was selected per interval. Individuals were coded as heterozygous (WSB/not-WSB) or homozygous (not-WSB/not-WSB) at each marker. A single-locus QTL scan was then performed using Haley-Knott regression as implemented in R/qtl [<xref ref-type="bibr" rid="pgen.1004850.ref079">79</xref>], treating the population as a backcross.</p><p>In order to refine the location of <italic>R2d2</italic>, we identified individual mice with recombinant chromosomes within the candidate interval defined by linkage mapping. These <italic>critical recombinants</italic> define the proximal and distal boundaries of the refined candidate interval.</p><p>
<bold>CAST/EiJ allele</bold>. We partitioned the 330 G2:F1 individuals without a WSB/EiJ allele in the <italic>R2d</italic> locus into two groups according to MDA sum-intensity values. From those with sum-intensity consistent with a non-CAST/EiJ expansion allele, we selected the most distal recombinants from CAST/EiJ to another haplotype. From those with sum-intensity consistent with the CAST/EiJ expansion allele, we selected the most distal recombinant from another haplotype to CAST/EiJ. Together these recombinants define the proximal boundary of the candidate interval in CAST/EiJ. Similarly, in order to define the distal boundary of the candidate interval, we selected the most proximal recombinants from CAST/EiJ to another haplotype that still had sum-intensity consistent with the CAST/EiJ expansion allele.</p><p>
<bold>WSB/EiJ allele</bold>. The boundaries of the WSB/EiJ candidate interval were mapped in the same fashion using 229 individuals spanning generations 10 through 14 of the DO, all of which have been genotyped on MegaMUGA and are recombinant for WSB/EiJ in the initial candidate interval. We first excluded individuals homozygous for WSB/EiJ over any interval with in the interval. Then we selected the most distal recombinants from another haplotype to WSB/EiJ, which also had MegaMUGA sum-intensity values consistent with a non-WSB/EiJ expansion allele. These recombinants define the distal boundary of the candidate interval. We mapped the proximal boundary similarly.</p><p>
<bold>SPRET/EiJ allele</bold>. (C57BL/6JxSPRET/EiJ)xC57BL/6J (<italic>n</italic> = 12) and (A/JxSPRET/EiJ)xA/J progeny (<italic>n</italic> = 17) [<xref ref-type="bibr" rid="pgen.1004850.ref026">26</xref>,<xref ref-type="bibr" rid="pgen.1004850.ref043">43</xref>] genotyped on the MegaMUGA array were used to refine the candidate interval for the expansion allele in SPRET/EiJ. Haplotypes in the relevant region of Chr 2 were inferred by manual inspection of genotype calls. Samples were partitioned according to sum-intensity at the three MegaMUGA SNP probes tracking the expansion allele. Among individuals with sum-intensity consistent with the expansion allele, the most proximal recombinant from SPRET/EiJ to another haplotype defines the distal boundary of the candidate interval. Likewise the most distal recombinant from a non-SPRET/EiJ haplotype to SPRET/EiJ defines the proximal boundary of the candidate interval.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Ten individuals from the HR8 selection line were selected for whole-genome sequencing. Five micrograms of high-molecular-weight DNA were used to construct TruSeq Illumina libraries, using 0.5 &#956;g starting material, with 300- to 400- and 400- to 500-bp fragment sizes. Each library was sequenced on one lane of an Illumina HiSeq2000 flowcell, as paired-end reads, with 100-bp read lengths. We aligned the sequences to the University of California at Santa Cruz Mouse Build mm9. HR8 sequenced reads were aligned to the mouse genome (mm9) using bowtie 2.2.3 [<xref ref-type="bibr" rid="pgen.1004850.ref080">80</xref>] with default options. We removed PCR duplicates and filtered low-quality SNPs using samtools 0.1.19 [<xref ref-type="bibr" rid="pgen.1004850.ref081">81</xref>] and Picard 1.88 (<ext-link ext-link-type="uri" ns0:href="http://picard.sourceforge.net/">http://picard.sourceforge.net/</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We retrieved BAM files of aligned reads (Oct 2012 release) from the Sanger Mouse Genomes Project FTP site (<ext-link ext-link-type="uri" ns0:href="http://ftp-mouse.sanger.ac.uk">ftp-mouse.sanger.ac.uk</ext-link>). We used the mpileup function of samtools [<xref ref-type="bibr" rid="pgen.1004850.ref081">81</xref>] to call sequence variants on the HR8 and Sanger BAM files jointly and to output the read depth at each base. We counted a SNP as private to WSB/EiJ, SPRET/EiJ and the 10 HR8 individuals if those samples all shared a genotype that was different from the seven other CC founder strains. We defined the boundaries of the copy number expansion by identifying consecutive 100bp windows in which the average read depth was at least twice the genome-wide average read depth. We estimated the number of copies of the expansion as the modal per-base read depth.</p><p>We used commercially-available TaqMan assays for <italic>Cwc22</italic> to estimate the copy number of <italic>R2d2</italic>. We used two copy number assays (Life Technologies catalog numbers Mm00644079_cn, Mm00053048_cn) to target the number of <italic>Cwc22</italic> copies (proximal and distal). We also used two reference assays (<italic>Tfrc</italic>, cat. no. 4458366, for target Mm00053048_cn; <italic>Tert</italic>, cat. no. 4458368, for target Mm00644079_cn), for genes known to exist in a single haploid copy in the mouse, to calibrate the amplification curve. Assays were performed according to the manufacturer&#8217;s protocol on an ABI StepOne Plus Real-Time PCR System (Life Technologies, Carlsbad, CA). Cycle thresholds (Ct) for each assay were determined using the ABI CopyCaller v2.0 software with default settings. For each target-reference pair, relative cycle threshold (&#916;Ct) was calculated as</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The &#916;Ct value is proportional to copy-number of the target gene on the log scale but is subject to batch effects. In order to account such effects, normalized &#916;Ct values for each sample were calculated as follows. A standard set of control samples (from C57BL/6J, WSB/EiJ, CAST/EiJ and (WSB/EiJxC57BL/6J)F1 mice), spanning the expected copy-number range for <italic>Cwc22</italic>, were included in duplicate or triplicate in every assay batch. A linear mixed model was fit to raw &#916;Ct values for these control samples, with target-reference pair and batch as random effects, using the lme4 package (<ext-link ext-link-type="uri" ns0:href="http://lme4.r-forge.r-project.org/">http://lme4.r-forge.r-project.org/</ext-link>) for R (<ext-link ext-link-type="uri" ns0:href="http://www.R-project.org/">http://www.R-project.org/</ext-link>). Predicted values (best linear unbiased predictors, BLUPs) from this model capture technical variation orthogonal to variation due to genotype. BLUPs calculated from control samples were subtracted from raw &#916;Ct values for all samples, and the residual was used as the normalized &#916;Ct for copy-number estimation.</p><p>In this manuscript we chose in most cases to present &#916;Ct, rather than extrapolated absolute copy number, because &#916;Ct is the natural scale of the data (<italic>i</italic>.<italic>e</italic>., the log scale). Constant variance (with respect to mean) on the log scale grows exponentially on the linear scale so that estimates of absolute copy number become increasingly uncertain as copy-number grows.</p><p>The use of TaqMan assays for <italic>Cwc22</italic> as a proxy for copy number at <italic>R2d2</italic> was validated by mapping normalized &#916;Ct for target Mm00644079_cn as a quantitative phenotype in 64 members of the (FVB/NJx(WSB/EiJxPWK/PhJ)F1)G2 intercross population described above. The marker selection and mapping procedure were the same as described above for mapping MegaMUGA sum-intensity values.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Chr 2 genotypes and whole-genome sequence that have not been published elsewhere are available at <ext-link ext-link-type="uri" ns0:href="http://csbio.unc.edu/r2d2">http://csbio.unc.edu/r2d2</ext-link>.</p>