<p>An <italic>Rp1-D21</italic>-H95 mutant line was generated from a cross between an <italic>Rp1-D21</italic> variant and the maize inbred line H95. The ensuing F1 progeny was then backcrossed to the H95 parent four times, while selecting for the HR phenotype marked by spontaneous lesion formation. Since the homozygous <italic>Rp1-D21-</italic>H95 mutant lines are sterile, it was maintained in a heterozygous state. For further details of this line see Chaikam et al <xref ref-type="bibr" rid="pgen.1004562-Chaikam1">[24]</xref>. The <italic>Rp1-D21</italic>-H95 line was crossed as a male to each of 3,381 lines from nested association mapping (NAM) population and to 225 lines from the IBM population <xref ref-type="bibr" rid="pgen.1004562-Lee1">[96]</xref> to create a set of F1 families, each of which segregated 1&#8758;1 (wild type:mutant) for the presence and absence of the <italic>Rp1-D21-</italic>induced HR phenotype but which were otherwise isogenic within a family.</p><p>Development of the maize (NAM) populations has been previously described <xref ref-type="bibr" rid="pgen.1004562-McMullen1">[32]</xref>, <xref ref-type="bibr" rid="pgen.1004562-Buckler2">[56]</xref>. This study used a subset from the NAM population that is comprised of 25 bi-parental RIL populations with B73 as a common parent and 200 lines in each population. The B73&#215;HP301 population was not included in our panel of 3,381 lines since HP301 and about 88% of the derived lines in this population carry the <italic>ga1</italic>cross-incompatibility gene, making it very difficult to obtain F1 seed from these lines used as female parents <xref ref-type="bibr" rid="pgen.1004562-Bloom1">[97]</xref>. The selection of 3,381 F1 families was based on the availability of seed. F1 crosses derived from the IBM population were also included in the correlation analysis part of this study but excluded from the joint linkage and genome-wide association (GWA) analysis.</p><p>Each of the 3,606 F1 families (including 225 IBM lines) was evaluated in four environments that comprised two locations (Clayton, NC and West Lafayette, IN) and 2 years (2011 and 2012), with single replicate of the NAM RILs in each location. An augmented randomized incomplete block design was implemented with the <italic>Rp1-D21-</italic>H95 as repeated check to provide an estimate of experimental error. Each NAM family was assigned to a block and a total of 10 sub-blocks within each family block. The <italic>Rp1-D21-</italic>H95 check was included in each of the 10 blocks in each NAM family and in each location. Two rows of a constant genotype were planted around the edges of the field to minimize border row effect. Standard fertilizer, pesticide and herbicide regimes were applied during the trial to ensure normal plant growth. Thinning to desired plant density and overhead irrigation were applied as required. At Clayton, NC, 12 kernels of each line were sown in 2 m rows with an inter-row spacing of 0.97 m and a 0.6 m alley at the end of each plot, while at West Lafayette, IN, 18 seeds were sown in 6 m rows with an inter-row spacing of 0.76 m.</p><p>Four lesion-associated traits, lesion severity (LES), mutant to wild type height ratio (HTR), mutant to wild type stalk width ratio (SWR), and mutant to wild type days to anthesis ratio (DTAR) were scored. Each F1 family segregated 1&#8758;1 for the presence and absence of <italic>Rp1-D21</italic> but was otherwise isogenic. Within a family it was immediately obvious, by the presence or absence of lesions and by the growth habit of the plant, which plants carries <italic>Rp1-D21</italic> and which were wild-type. For the LES trait, only plants carrying <italic>Rp1-D21</italic> were scored, while for HTR, SWR and DTAR, both wild type and mutant classes were phenotyped separately and ratios between them derived.</p><p>In all environments, lesion severity scores were assigned based on a scale of 1 to 10 (<xref ref-type="supplementary-material" rid="pgen.1004562.s005">Figure S5</xref>), with &#8220;1&#8221; representing very few lesions and &#8220;10&#8221; indicating a completely dead plant <xref ref-type="bibr" rid="pgen.1004562-Chaikam1">[24]</xref>. Experiments were scored five times at Clayton, NC in both 2011 and 2012. At West Lafayette, IN, plants were scored six and four times in years 2011 and 2012, respectively. Scoring was started one month after planting and continued at approximately 10&#8211;14 day intervals.</p><p>We scored an aberrant HR defense response rather than a disease symptom in this case, but since the lesion phenotypes are generally similar to disease lesions, we applied a widely-accepted statistic in plant pathology; standardized area under disease progress curve (sAUDPC), to quantitatively measure HR severity <xref ref-type="bibr" rid="pgen.1004562-Shaner1">[98]</xref>. For each environment, the sAUDPC for LES was calculated as follows: The average value of two consecutive ratings was computed and multiplied by the number of days between the ratings. Values were summed over all intervals, and then divided by the total number of days over which evaluations were performed to determine the weighted average.</p><p>Plant height data were collected after flowering from three representative mutant F1 individuals and from three representative wild type F1 individuals within each F1 family. Height means were calculated for each class within each family and the HTR was calculated by dividing the average mutant type height to the average wild type height.</p><p>Stalk width immediately above the primary ear was measured on three representative mutant F1 individuals and three representative wild type F1 individuals within each F1 family. SWR was then calculated by dividing the average mutant stalk width to the wild type average stalk width.</p><p>Plants were monitered every other day for the date when 50% of the wild type and 50% of the mutant plants in an F1 family were shedding pollen. The days from planting to anthesis were computed for the wild type and the mutant plants in each family. DTAR was then calculated by dividing the days to anthesis for mutants by days to anthesis for wild type plants.</p><p>Disease scores for SLB (southern leaf blight; <italic>Cochliobolus heterostrophus</italic>), GLS (gray leaf spot; <italic>Cercospora zeae-maydis</italic>) and NLB (northern leaf blight; <italic>Exserohilum turcicum</italic>) were obtained from previous studies <xref ref-type="bibr" rid="pgen.1004562-Benson1">[33]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004562-Poland1">[35]</xref>. These data were used to evaluate the correlation between disease resistance and HR severity in response to <italic>Rp1-D21</italic> aberrant phenotype (lesion mimics).</p><p>The least square (LS) mean data used for analysis can be found in <xref ref-type="supplementary-material" rid="pgen.1004562.s015">File S6</xref>. <xref ref-type="supplementary-material" rid="pgen.1004562.s016">File S7</xref> and <xref ref-type="supplementary-material" rid="pgen.1004562.s001">Figure S1</xref> gives the metadata for each of the populations. To obtain least square mean values adjusted for environmental effects, data were analyzed with a mixed model considering line as a fixed effect and environment, block within environment, population by environment and line-by-environment interaction within population as random effects using Proc Mixed in SAS v9.3 <xref ref-type="bibr" rid="pgen.1004562-SAS1">[99]</xref>. The REML Wald's Z statistic was used to test the significance of each random factor in the model <xref ref-type="bibr" rid="pgen.1004562-Littell1">[100]</xref>. Least squares means for lines were estimated from this mixed model and used as the input phenotype for association analysis. For the purpose of estimating heritability, a mixed model with all factors, including lines, as random effects was used. Correlations between disease scores (SLB, GLS and NLB) and lesion mimic traits for each line were estimated using Proc Corr in SAS v9.3 <xref ref-type="bibr" rid="pgen.1004562-SAS1">[99]</xref>. The NAM sub-population groups, considered as a covariate, were accounted for during correlation analysis.</p><p>A total of 7386 SNP markers scored on all 4892 available NAM RILs were used for linkage and QTL analyses (<xref ref-type="supplementary-material" rid="pgen.1004562.s017">Files S8</xref>, <xref ref-type="supplementary-material" rid="pgen.1004562.s018">S9</xref>). The marker values were imputed at 0.2 cM intervals based on SNP calls made from short sequence reads using the GBS protocol <xref ref-type="bibr" rid="pgen.1004562-Elshire1">[101]</xref>. Briefly, each sample was digested with the ApeK1 restriction enzyme, PCR amplified, multiplexed, sequenced and then processed through a custom SNP calling pipeline. Because the sequence coverage was low, about 0.5&#215;, two problems arose; First, many sites had more than 50% missing data, and, second, at many heterozygous sites only a single allele of the two possible alleles was detected. As a result, about 80% of the heterozygous sites were scored as homozygous. To deal with these issues, each SNP call was first scored as either the B73 or non-B73 parent. Then the Viterbi algorithm was applied to the resulting sequence to identify probable heterozygous loci and genotype calling errors. Sites were then chosen at 0.2 cM intervals and values for each site imputed as 2 (probability allele came from the non-B73 parent) based on the nearest non-missing flanking markers. Where both flanking markers came from the same parent the value was either 0 or 2. Where the markers came from different parents, the value was intermediate and based on the relative distance from the two markers <xref ref-type="bibr" rid="pgen.1004562-Romay1">[59]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For the GWA analysis, a total of about 26.5 million SNPs polymorphic among the NAM founder lines were obtained from the HapMapv2 project <xref ref-type="bibr" rid="pgen.1004562-Chia1">[57]</xref>. The data at each locus are comprised of two alleles per SNP, with the minor alleles set to 1 and everything else to 0. The data were recoded to reduce computational time during projection by setting B73 alleles (common parental allele) to 0 and non-B73 alleles (alternate allele) to 1. Projection of the 26.5 million HapMap v2 NAM founder genotypes on the NAM RILs was performed based on the genotype of the RIL's diverse founder/parental lines at each of the 26.5 million HapMap v2 SNP, and an individual RIL's genotype calls at the NAM SNP markers flanking the physical position of the HapMap v2 SNP. Because we used the high density 7386 NAM SNP marker linkage map, each interval was only 0.2 cM in genetic distance, permitting very accurate imputation of HapMap v2 SNPs. If the diverse founder/parental line has the same homozygous allele as B73 at both flanking loci, the RIL genotype is assigned a B73 allele (coded as 0) for a SNP position. When the diverse founder/parental line carries an alternate/non-B73 allele homozygous at both flanking loci, the probability that the RIL carries the alternate allele is computed as the weighted average of the flanking mapped NAM SNP genotype, with the weights assigned based on the relative physical position of the HapMap SNP within the interval. This is computed as:<disp-formula id="pgen.1004562.e001"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004562.e001.jpg" /></disp-formula>where <italic>d</italic>
<sub>2</sub> is the physical distance between the HapMap v2 SNP and the right-hand mapped NAM SNP, <italic>G</italic>
<sub>1</sub> is the genotype score at the mapped NAM SNP, <italic>d</italic>
<sub>1</sub> is the physical distance between the HapMap v2 SNP and the left-hand mapped NAM SNP, and <italic>G</italic>
<sub>2</sub> is the genotype score at the right-hand mapped NAM SNP.</p><p>The single/independent population QTL analysis was performed based on composite interval mapping (CIM) and implemented in the Windows QTL Cartographer software v2.5 <xref ref-type="bibr" rid="pgen.1004562-SilvaLda1">[102]</xref>. Permutation tests set to an alpha-level of 0.05 were performed to determine population and trait specific LOD thresholds at aprroximately 3.0. Linkage and QTL maps were prepared using the MapChart software v2.2 <xref ref-type="bibr" rid="pgen.1004562-Voorrips1">[103]</xref>.</p><p>Joint-linkage mapping was implemented as previously described <xref ref-type="bibr" rid="pgen.1004562-Buckler2">[56]</xref>. Before the joint stepwise regression procedure in PROC GLMSELECT was implemented in SAS software v9.3, LOD thresholds were established following 1,000 trait-dependent permutation tests based on alpha set at 0.05 with the model containing a family main effect. Trait specific QTL LOD thresholds of 5.44, 5.00, 4.27 and 8.34 were obtained and applied to detect QTL for LES, HTR, SWR and DTAR, respectively. The model included family main effects and a single marker effects nested within families. The lowest resulting <italic>p</italic> value among marker tests was obtained for each permutation.</p><p>Following initial detection of QTL loci, the stepwise regression model was optimized with an iterative process by sequentially dropping a marker in the model, testing the fit of adjacent markers until the eighth marker (1.6 cM away) from the originally selected marker, and fitting the best marker in the region back into the model. Allele effects at each marker included in the final model were estimated simultaneously using the solution option of Proc GLM in SAS software v.9.3. The <italic>t</italic>-tests of the null hypothesis of zero allele effect were performed for each NAM founder allele effect at each QTL. The QTL support intervals were computed in the SAS software v.9.3 by adding a single flanking marker to the full model one at a time from the QTL at a step of 0.2 cM. This was performed at the left and right side of the QTL. The support interval boundary was considered to be the last marker at which the QTL regained significance at the <italic>p</italic>&#8202;=&#8202;0.05 level.</p><p>To test for significant digenic epistatic interactions, a subset of 1409 SNP markers were obtained from the available 7386 SNP markers at uniform 1 cM intervals and all pairwise combinations of the 1409 markers were tested separately using models that included population main effects, the two marker main effects nested in populations, and the marker-marker interaction nested in populations by extending the method of Holland <xref ref-type="bibr" rid="pgen.1004562-Holland1">[104]</xref>. The analysis was performed across the 24 NAM families while the trait-specific QTL LOD thresholds were estimated based on a permutation test at a critical value with alpha less than 5%. Marker pair interactions with <italic>p</italic>-values less than the permutation test-based threshold were considered for inclusion in the final joint linkage model without regard to the signficance of the main effects of the markers. Each such pair of markers and their interaction were added to the final additive joint linkage model one at a time; the <italic>p</italic>-value of the interaction in this full model was used to determine if the epistatic interaction improved the model fit.</p><p>GWA models were fit for each chromosome, one at a time. For each chromosome, line residual values from the final joint linkage QTL model excluding all markers on the chromosome under consideration were computed in SAS software v.9.3. Using 26.5 million SNPs, GWA (genome wide association) analysis was performed based on forward regression of the HapMap v2 SNPs to subsamples of these phenotypic residual values adjusted for QTL on other chromosomes. To identify SNPs with the most robust associations with traits, we implemented a subsampling (subagging) procedure during the GWA analysis <xref ref-type="bibr" rid="pgen.1004562-Valdar1">[105]</xref>, with forward regression (using a <italic>p</italic>-value threshold for inlcusion in the model of 1&#215;10<sup>&#8722;6</sup>
<xref ref-type="bibr" rid="pgen.1004562-Panagiotou1">[106]</xref>) performed in each of 100 subsample datasets. Each subsample dataset comprised 80% of the RILs from each NAM family <xref ref-type="bibr" rid="pgen.1004562-Kump1">[34]</xref>, <xref ref-type="bibr" rid="pgen.1004562-Tian2">[51]</xref>. A population main effect was included in the model prior to the addition of SNP terms during the forward selection. The effect estimate of each significant SNP in each subsample was also computed and averaged over the 100 subsamples. The resample model inclusion probability (RMIP) was computed for each SNP as the proportion of subsample data sets in which it was included in the final regression model. Only SNPs with RMIP &gt;0.05 are shown on the Manhattan plot. Following Valdar et al. (2006), an RMIP threshold of 0.25 was used to report the most robust SNP associations.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genes co-localizing with or adjacent to associated SNPs were determined using the maize B73 reference genome assembly v2 available on the MaizeGDB genome browser <xref ref-type="bibr" rid="pgen.1004562-Andorf1">[107]</xref> or the <ext-link ext-link-type="uri" ns0:href="http://www.maizesequence.org">www.maizesequence.org</ext-link> genome browser <xref ref-type="bibr" rid="pgen.1004562-Schnable1">[108]</xref>. Functional annotations of the candidate genes were determined using blastp <xref ref-type="bibr" rid="pgen.1004562-Altschul1">[109]</xref>, conserved domain search tools <xref ref-type="bibr" rid="pgen.1004562-MarchlerBauer1">[110]</xref> and annotations avialble at the Maize Genome database (<ext-link ext-link-type="uri" ns0:href="http://gbrowse.maizegdb.org/gb2/gbrowse/maize_v2/">http://gbrowse.maizegdb.org/gb2/gbrowse/maize_v2/</ext-link>). Further annotation was achieved by inspection of the literature specific to each gene and domain (<xref ref-type="supplementary-material" rid="pgen.1004562.s014">File S5</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Wild-type (WT) and mutant plants in B73&#215;<italic>Rp1-D21</italic>-H95 and Mo17&#215;<italic>Rp1-D21-</italic>H95 backgrounds growing in constant 22&#176;C with 12 h-day/12 h-dark were used for RNA extraction. The 3rd true leaves of 5 individual seedlings collected from 18-day old WT or mutant plants were pooled and total RNA was isolated using Trizol reagent (Life Technologies) according to manufacturer's instructions. RNA concentration and quality were monitored by the NanoDrop and agarose gel electrophoresis. mRNA was isolated from the total RNA by Dynabeads oligo(dT) (Life Technologies) following manufacturer's directions. RNA-seq libraries were constructed according to the TruSeq RNA Sample Prep v2 LS protocol (Illumina) and sequenced using the Illumina HiSeq 2000. Two biological replicates, each consisting of 5 individual plants, and two technical replicates (two lanes) were performed, with 100 nt single end reads. The sequences were aligned to maize genome sequence v2 (<ext-link ext-link-type="uri" ns0:href="http://www.maizegdb.org">www.maizegdb.org</ext-link>) via TopHat 2.0 <xref ref-type="bibr" rid="pgen.1004562-Trapnell1">[111]</xref>, using all the default parameter settings. The raw counts per gene were calculated using in-house scripts. Only reads with unique alignments were maintained for subsequent analyses. Genes with total read counts less than 20 were filtered out. Differentially expressed genes (DEGs) were identified using the software package edgeR from the Bioconductor suite <xref ref-type="bibr" rid="pgen.1004562-Robinson1">[112]</xref>. To account for multiple testing, the procedure of Benjamini and Hochberg <xref ref-type="bibr" rid="pgen.1004562-Benjamini1">[113]</xref> for controlling the false discovery rate (FDR) was applied using a threshold of q&lt;&#8202;=&#8202;0.05 to determine significance.</p>