<p>This work was performed under approved Einstein IRB protocol number 2011-356-000.</p><p>10 to 20 ml of peripheral white blood cells were harvested by venipuncture from individuals from family FNY01 and mononuclear cells were isolated by density gradient centrifugation on Histopaque (Sigma-Aldrich) according to the manufacturer's instructions. The purified cells were frozen in 2 million cell aliquots. Two million mononuclear cells were expanded and differentiated into basophilic erythroblasts in culture for two weeks in serum-free Stemspan media (Stem Cells Technologies, VA, CA) containing the cytokine cocktail mix described by Olivier et al. <xref ref-type="bibr" rid="pgen.1004319-Olivier1">[37]</xref>. At the end of the culture, cells were immuno-phenotyped by FACS using antibodies against CD71 and CD235a. Cells were relatively uniform in size and more than 97% of the cells were double-positives demonstrating that the vast majority of cells in the culture were erythroid and at the basophilic stage of differentiation.</p><p>Twenty to fifty million exponentially growing basophilic erythroblasts were harvested on day 14 of culture, washed with ice cold PBS, fixed by dropwise addition of ice-cold 75% ethanol, and stored at 4&#176;C before processing. Fixed cells were treated with RNase A (0.5 mg/ml for 60 minutes, Sigma-Aldrich) and stained for DNA content with propidium iodide (50 &#181;g/ml) (Invitrogen, Carlsbad, CA). Cells in the S phase of the cell cycle were sorted using a FACS Aria II (BD Biosciences). Genomic DNA was prepared using the Wizard Genomic DNA Purification Kit (Promega) and quantified using NanoDrop and PicoGreen methods (Quant-iT PicoGreen dsDNA Assay Kit, Invitrogen).</p><p>Sequencing of family FNY01 to obtain the phased genomes was performed with DNA from peripheral blood white blood cells <xref ref-type="bibr" rid="pgen.1004319-Lajugie1">[36]</xref>. Since more than 99.5% of peripheral blood white blood cells are in the G1 phase of the cell cycle (<xref ref-type="fig" rid="pgen-1004319-g001">Figure 1B</xref>), data from these experiments were used as the G1 control in all the TimEX-seq experiments described in this report.</p><p>Genomic DNA was sheared to 300&#8211;600 bp size using a focused ultra-sonicator from Covaris (Woburn, MA), size purified by agarose gel electrophoresis, and analyzed on a DNA Chip using a 2100 bio-analyzer (Agilent Technologies, Inc.) to verify size distribution. DNA was end-repaired (End-It kit, Epicenter Biotechnologies), an A-overhang was added (Klenow 3&#8242; to 5&#8242; exo minus, NEB), and ligated to Illumina pair-end adapter sequences (Illumina). The ligated libraries were size selected (600&#177;50 bp), PCR amplified for 10 cycles, and purified with SPRI beads (Agencourt AMPure XP; Fisher Scientific). Pair-end library quality was verified on a bio-analyzer as described above. Each library was sequenced on three lanes on the Illumina HiSeq 2000 yielding about 10<sup>9</sup>, one hundred bp pair-end reads corresponding to about 30&#215; coverage for each S phase library.</p><p>After library sequencing, the reads were aligned to hg 19 with bwa <xref ref-type="bibr" rid="pgen.1004319-Li1">[69]</xref> using the default parameters and the SNPs were called using the GATK variant caller in the known-allele mode <xref ref-type="bibr" rid="pgen.1004319-McKenna1">[70]</xref>, providing the vcf file that describes the genomes of family FNY01 <xref ref-type="bibr" rid="pgen.1004319-Lajugie1">[36]</xref> as a reference file. In this mode, the GATK calls only the SNPs at the positions specified in the user-provided reference vcf file.</p><p>Once the SNPs had been called, the vcf files for the S phase libraries was phased using the vcf for family FNY01 as a reference and specific functions in GenPlay multi-genome <xref ref-type="bibr" rid="pgen.1004319-Lajugie2">[71]</xref>. Once the files were phased files (in the BED format) containing the allele depth for the paternal and maternal chromosomes were generated, again using specific functions that are provided in GenPlay.</p><p>The read depth files were binned in to 500 bp windows, normalized to the total number of reads containing a SNP in each track and smoothed using the GenPlay Gaussian filter (sigma&#8202;=&#8202;100 kb). S/G1 ratios for the paternal and maternal chromosomes were then calculated. The resulting numbers were multiplied by a factor 1.4 to take into account the fact that the average copy number in S is higher than in G1. These resulted in TimEX profiles with value ranging between 0.9 and 2.1. No other filter or normalization was applied to the data. Masks were created to eliminate the regions of low SNP-density using GenPlay island finder functions. The resulting masks were applied in selected analysis to ascertain that the results were independent of SNP density. Other smoothing techniques (Loess, moving average) yielded essentially the same results.</p><p>The combined FNY01_3_2&amp;3_3 allele-specific replication profiles were produced by generating four new files: a combined G1 maternal file containing the sum of the allele-depths of the maternal homologs of FNY01_3_2 and 3_2; a combined G1 paternal file containing the sum of the allele depths of the paternal homologs of FNY01_3_2 and 3_2, and similarly, two combined maternal and paternal S files. These four files were then processed as above to generate combined maternal and paternal S/G1 profiles.</p><p>Asynchronously replicated regions were detected with the GenPlay island finder on a data file generated by subtracting the paternal S/G1 simulated profiles from the maternal S/G1 simulated profiles.</p><p>The GenPlay Island finder function is based on the SICER algorithm of Zang et al. <xref ref-type="bibr" rid="pgen.1004319-Zang1">[72]</xref> and identifies broad regions of enriched read counts rather than peaks. The algorithm allows for gaps in the island and is well suited for the allele-specific TimEX data because of the heterogeneous distribution of the SNPs in the human genome. Implementation of the algorithm in GenPlay is described in the GenPlay documentation in the GenPlay web site.</p><p>A TimEX difference threshold of 0.02, a maximum gap sizes of 250,000 and a minimum island size of 50,000 bp were used in the island finder. Statistically significant regions were then identified using a qui-square test for goodness of fit (chisq.test function in R) on 2&#215;2 contingency tables built by calculating the total number of reads obtained in the S and G1 fraction of the maternal and paternal chromosomes for each islands. Q-values were then calculated using the p.adjust function in R (with the fdr parameter) to control for multiple testing. An FDR rate of 5% was used to define the statistically significant asynchronous replication domains. The parameters for the island finder were determined by optimizing by trial and error, the rate of discovery of simulated regions (see below).</p><p>Core detection: Core were detected as ARD except that a TimEX difference threshold of 0.1, a maximum gap sizes of 50,000 and a minimum island size of 50,000 bp were used in the GenPlay Island finder. Statistical significance was determined as above.</p><p>Simulated maternal and paternal pairs of S and G1 resampled control profiles were generated by imputing in a random number generator based on the binomial function (similar to the rbinom function in R), the number of S reads and the total number of reads (S+G1 reads) observed for each informative SNP in the maternal track of FNY01_3_2.</p><p>About 1,000 randomly located simulated asynchronous genomic regions (ranging in size from 125,000 to 2,000,000 bp) were then introduced in the simulated maternal control S profiles by increasing by a fixed percentage (5, 10, 15, 20, 30 or 50%), the total (S+G1) number of reads observed for each SNP and by assigning the extra reads to the S phase profile. These S and S+G1 read numbers were then imputed in the random generator to generate resampled read numbers in these simulated asynchronous regions.</p><p>Once simulated number of reads for the S and G1 profiles of the maternal and paternal homologs had been generated, simulated S/G1 TimEX profiles were generated as described above using Gaussian smoothing (sigma&#8202;=&#8202;100 kb). Bedgraph files were generated to visualize the simulation in GenPlay.</p><p>Asynchronously replicated regions in these simulated profiles were then detected with GenPlay island finder as described above. Simulations were repeated multiple times.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">A Java script implementing this algorithm can be found at <ext-link ext-link-type="uri" ns0:href="https://github.com/JulienLajugie/ReplicationTimingSimulation">https://github.com/JulienLajugie/ReplicationTimingSimulation</ext-link>.</p><p>S and G1 aligned reads obtained after alignment to hg19 were loaded into 1 kb bins in GenPlay, normalized to the total number of reads, S to G1 ratios were calculated, outliers were eliminated (S/G1&gt;2.4), and the profiles were smoothed with the GenPlay Gaussian filter (sigma&#8202;=&#8202;20 kb). Use of other smoothing techniques (Loess, moving average) yielded essentially the same results. For more convenient analysis values were then indexed between 1 and 2.</p><p>Nascent, newly replicated DNA strands from cultured basophilic erythroblasts were isolated following two protocols. First, we have isolated RNA-primed short newly replicated DNA following the procedure described in Martin et al. <xref ref-type="bibr" rid="pgen.1004319-Martin1">[23]</xref>. Briefly, DNA was extracted from asynchronous cultures, denatured and fractionated by centrifugation on a neutral sucrose gradient. DNA fractions within the 0.5 kb to 1 kb size range were collected and DNA was exposed to lambda exonuclease to remove non-RNA-primed genomic DNA fragments. Purified RNA-primed NS were random-primed and the resulting double-stranded nascent DNA (1 &#181;g) was sequenced using the Illumina (Solexa) genome analyzer II.</p><p>Second, to control for possible lambda exonuclease biases, we have also isolated and sequenced short nascent DNA strands that were isolated based on incorporation of the nucleotide analog BrdU as described by Aladjem et al. <xref ref-type="bibr" rid="pgen.1004319-Aladjem3">[40]</xref>. Briefly, we labeled cells with a 30-minute pulse of BrdU, lysed the cells and fractionated small DNA strands (&lt;1 kb) using a neutral sucrose gradient. We then employed immune-precipitation with anti-BrdU antibodies to isolate newly replicated DNA strands on the basis of selective incorporation of BrdU. The resulting NS were subject to massively parallel sequencing. For both methods, sheared genomic DNA was sequenced as a standard to control for mapability and other potential biases.</p><p>Nascent libraries were prepared and sequenced on two lanes of an Illumina GII analyzer. Read length was 36 bp. Reads were aligned to hg19 using bwa and loaded in GenPlay in bins of 100 bp, 3 kb or 20 kb (depending on the analysis). Peaks were called using MACS 1.42 using the default parameters, or the GenPlay Island finder.</p><p>The ratios of the TimEX values of the two chromosome homologs for individuals FNY01 3_2 and 3_3 were calculated in 5 kb windows and averaged over 500 kb intervals (overlapping every 100 kb). Ratios smaller than 1 were inverted, since we expected the SVs to either increase or decrease the timing ratios. SVs present in the family are described in detail in <xref ref-type="bibr" rid="pgen.1004319-Lajugie1">[36]</xref>.</p><p>Similar results were obtained with interval of 1 Mb, with non-overlapping intervals of 0.5 or 1 Mb, or by averaging the distance or the absolute values of the differences between the S/G1 ratios of the homologs rather than their ratios.</p><p>Hi-res TimEX profiles of FNY01 3_2 and 3_3 were averaged and binned at 3 kb. NS were binned at 200 bp and smoothed with the GenPlay Gaussian filter using sigma 3 kb or 20 kb. The tracks were then converted to 3 kb bins after saturating the outliers (i.e. all bins with scores larger than X with x&#8202;=&#8202;mean plus 3 times the standard deviation were set to a score equal to X). The averaged TimEX profiles were then divided into 5 equal fractions termed S1 to S5 based on the TimEX values (with S1 replicating the earliest and S5 the latest). Correlation and linear regression were performed in R or in Statgraphics Centurion XVII (Statpoint Technologies incorporated). Removing the outliers rather than saturating them yielded similar results.</p><p>Peaks were called using MACS v1.42 using the default value (p&lt;10E-5 and FDR&lt;0.2%). Sheared genomic DNA from the same individual was used as the control.</p><p>The number of peaks observed with the BrdU and lambda exonuclease methods was similar (respectively 250,000 and 250,00), the average widths of the peaks were also similar, about 100&#8211;400 bp in both cases. To facilitate the comparison between the methods, all peaks were normalized to a width of 400 bp which resulted in a 50% reduction in peak number and the analysis was performed on the top 100,000 peaks only.</p><p>Peaks were considered to be associated to a genomic features if they overlapping by one base pair or more with a CpG Island, a transcription start site (defined as 1 kb fragment centered on the start site) or with 400 bp fragments centered around a G-quadruplexes or a DNase I hypersensitive site.</p><p>GC content was calculated as the average percent of C and G in 100 kb windows (in 10 kb intervals).</p><p>Because of the very uneven baseline, MACS or the Genplay island finders were not able to satisfactorily detect and call the ripples within the larger TimEX domains that are illustrated in <xref ref-type="fig" rid="pgen-1004319-g004">Figure 4</xref>. To call these peaks we therefore devised a specific strategy illustrated in <xref ref-type="supplementary-material" rid="pgen.1004319.s005">Figure S5</xref>. Briefly, the TimEX profiles were smoothed with the Gaussian filters and a sigma at 20 kb (which revealed the ripples) and at 100 kb (which smooth out these ripples) and the 100 kb smooth was subtracted from the 20 kb smooth. The top 10 percent of the peaks thus obtained were then analyzed since they coincided closely with the major ripples that we wished to characterize. The same strategy was used to define the clusters of NS.</p><p>All statistical analysis were performed using R or using StatGraphics Centurion XVI (Statpoint technologies)</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Most of the data processing and visualization was performed in GenPlay multi-Genome, an application that was written to display, produce and process allele-specific and non-allele-specific sequencing data files. GenPlay is available freely at <ext-link ext-link-type="uri" ns0:href="http://genplay.einstein.yu.edu">http://genplay.einstein.yu.edu</ext-link>.</p><p>The list of known and predicted genes was obtained from the geneimprint.com web site. The list is reproduced in <xref ref-type="supplementary-material" rid="pgen.1004319.s008">Table S3</xref>.</p><p>Location of G-quadruplexes was generously provided by Dr. Julian Huppert and is described in <xref ref-type="bibr" rid="pgen.1004319-Huppert1">[73]</xref>. Data was converted to hg19 coordinate using liftover.</p><p>DNase I hypersensitivity data from Fetal Liver erythroid cells and K562 was generously provided by Dr. John Stamatoyannopoulos (University of Washington Seattle). It is available from the UCSC browser.</p><p>Annotations were downloaded from the UCSC genome browser on January 2013.</p><p>Repeats map were downloaded on January 213 from UCSC and were originally from repbase.</p><p>Processed and raw data files have been deposited at NCBI in the Geo database. Accession number: GSE50978</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The entire dataset can be visualized in the interactive GenPlay environment at <ext-link ext-link-type="uri" ns0:href="http://genplay.einstein.yu.edu/library/projects/TimEX-NascentStrands-2013/">http://genplay.einstein.yu.edu/library/projects/TimEX-NascentStrands-2013/</ext-link>.</p>