<p>The anthropometric traits examined are body mass index (BMI, kg/m&#178;), which is a measure of body mass and a surrogate for total body fat, and waist-to-hip-ratio adjusted for BMI (WHR<sub>adjBMI</sub>), which is a measure of body fat distribution. Traits were transformed before analyses; we first created age- (and BMI) adjusted residuals (including age and age&#178; into the regression for BMI, and additionally BMI for WHR<sub>adjBMI</sub>) for each of the four strata separately (men &#8804;50y, men &gt;50y, women &#8804;50y, and women &gt;50y) and subsequently applied an inverse normal transformation.</p><p>We included up to 92 studies (totalling up to 21,989 men &#8804;50y, 74,324 men &gt;50y, 41,386 women &#8804;50y, and 88,625 women &gt;50y) with genome-wide genotyping chip data using either Affymetrix or Illumina arrays. To enable meta-analyses across different SNP panels, each study group performed genotype imputation using HapMap II CEU (build 21 or 22) via MACH [<xref ref-type="bibr" rid="pgen.1005378.ref063">63</xref>], IMPUTE [<xref ref-type="bibr" rid="pgen.1005378.ref064">64</xref>] or BimBam [<xref ref-type="bibr" rid="pgen.1005378.ref065">65</xref>] yielding ~2.8 Million SNPs. In addition, we included 22 studies (up to 28,106, 18,877, 29,306, 17,872 individuals for each of the strata, respectively) for BMI and WHR<sub>adjBMI</sub> that were genotyped using the custom iSELECT Metabochip array containing ~195K SNPs designed to support large-scale follow-up of putative associations with metabolic and cardiovascular traits [<xref ref-type="bibr" rid="pgen.1005378.ref066">66</xref>].</p><p>In each study, SNP associations were tested separately by age-group and sex (men &#8804;50y, men &gt;50y, women &#8804;50y and women &gt;50y) for autosomal variants. The additive genetic effect for each SNP on each phenotype was estimated via linear regression using MACH2QTL [<xref ref-type="bibr" rid="pgen.1005378.ref067">67</xref>], SNPTEST [<xref ref-type="bibr" rid="pgen.1005378.ref064">64</xref>], ProbABEL [<xref ref-type="bibr" rid="pgen.1005378.ref068">68</xref>], GenABEL [<xref ref-type="bibr" rid="pgen.1005378.ref069">69</xref>], Merlin [<xref ref-type="bibr" rid="pgen.1005378.ref070">70</xref>], PLINK [<xref ref-type="bibr" rid="pgen.1005378.ref071">71</xref>] or QUICKTEST [<xref ref-type="bibr" rid="pgen.1005378.ref072">72</xref>]. For studies with a case-control design, cases and controls were analysed separately. See <bold><xref ref-type="supplementary-material" rid="pgen.1005378.s017">S1</xref>, <xref ref-type="supplementary-material" rid="pgen.1005378.s018">S2</xref></bold> and <bold><xref ref-type="supplementary-material" rid="pgen.1005378.s019">S3</xref> Tables</bold> for study specific genotyping, imputation, analysis, quality control and phenotypic descriptive information. In total we gathered association data from up to 92 studies with imputed GWAS data and 22 studies genotyped on the Metabochip array for BMI including up to 320,485 individuals and 64 studies with imputed GWAS data and 20 studies genotyped on the Metabochip array for WHR<sub>adjBMI</sub> including up to 216,654 individuals.</p><p>All studies were conducted according to the principles expressed in the Declaration of Helsinki. The studies were approved by the local Review Boards and all study participants provided written informed consent for the collection of samples and subsequent analysis.</p><p>All study-specific files were processed in the meta-analysis centers through a standardized quality-control (QC) pipeline [<xref ref-type="bibr" rid="pgen.1005378.ref073">73</xref>]. This involved QC checks on file completeness, range of test statistics, allele frequencies, trait transformation and population stratification as well as filtering on low quality data. Briefly, we excluded monomorphic SNPs, SNPs with MAF*N &#8804; 3 (minor allele frequency multiplied by sample size), imputed SNPs with poor imputation quality: r2_hat &lt; 0.3 in MACH, observed/expected dosage variance &lt; 0.3 in BIMBAM, proper_info &lt; 0.4 in IMPUTE, information &lt; 0.8 in PLINK [<xref ref-type="bibr" rid="pgen.1005378.ref064">64</xref>, <xref ref-type="bibr" rid="pgen.1005378.ref065">65</xref>, <xref ref-type="bibr" rid="pgen.1005378.ref067">67</xref>, <xref ref-type="bibr" rid="pgen.1005378.ref071">71</xref>]; genotyped SNPs with low call-rate (&lt; 95%), and genotyped SNPs that were out of Hardy-Weinberg equilibrium (HWE, P-Value testing for HWE &lt; 10<sup>&#8722;5</sup>). To increase the overlap in the number of SNPs between imputed GWAS and MetaboChip data, we transferred all SNP identifiers to unique SNP names consisting of chromosomal and base position, e.g. using chr1:217820132 instead of rs2820443 in the meta-analysis. Sex- and age-specific standard errors and P-values from each participating study were genomic-control (GC) corrected using study- and strata-specific lambda factors [<xref ref-type="bibr" rid="pgen.1005378.ref074">74</xref>], whereas the lambdas were estimated from all genome-wide available SNPs for imputed GWAS and form a subset of 4,427 QT-interval SNPs for MetaboChip studies.</p><p>Generally, beta-estimates and standard errors were meta-analyzed using an inverse-variance weighted fixed effect model as implemented in METAL [<xref ref-type="bibr" rid="pgen.1005378.ref075">75</xref>].</p><p>We meta-analyzed effect estimates and standard errors from all available studies in each of the four strata separately, yielding <italic>b</italic>
<sub><italic>M</italic>&#8804;50<italic>y</italic></sub>, <italic>b</italic>
<sub><italic>M</italic>&gt;50<italic>y</italic></sub>, <italic>b</italic>
<sub><italic>F</italic>&#8804;50<italic>y</italic></sub>, <italic>b</italic>
<sub><italic>F</italic>&gt;50<italic>y</italic></sub> and <italic>SE</italic>
<sub><italic>M</italic>&#8804;50<italic>y</italic></sub>, <italic>SE</italic>
<sub><italic>M</italic>&gt;50<italic>y</italic></sub>, <italic>SE</italic>
<sub><italic>F</italic>&#8804;50<italic>y</italic></sub>, <italic>SE</italic>
<sub><italic>F</italic>&gt;50<italic>y</italic></sub>. By meta-analyzing <italic>b</italic>
<sub><italic>M</italic>&#8804;50<italic>y</italic></sub> and <italic>b</italic>
<sub><italic>M</italic>&gt;50<italic>y</italic></sub> we obtained the effect and standard error for men (<italic>b</italic>
<sub><italic>M</italic></sub>,<italic>SE</italic>
<sub><italic>M</italic></sub>) and women (<italic>b</italic>
<sub><italic>F</italic></sub>,<italic>SE</italic>
<sub><italic>F</italic></sub>). Similar meta-analyses yielded the age group-specific association statistics, <italic>b</italic>
<sub>&#8804;50<italic>y</italic></sub> and <italic>b</italic>
<sub>&gt;50<italic>y</italic></sub> with standard errors <italic>SE</italic>
<sub>&#8804;50<italic>y</italic></sub> and <italic>SE</italic>
<sub>&gt;50<italic>y</italic></sub>. Meta-analysis of all four strata provided the overall association effect estimate <italic>b</italic>
<sub><italic>overall</italic></sub>, standard error <italic>SE</italic>
<sub><italic>overall</italic></sub>, and P-value <italic>P</italic>
<sub><italic>overall</italic></sub>. A joint meta-analysis based on the pooled stratum-specific estimates was performed according to Aschard et al [<xref ref-type="bibr" rid="pgen.1005378.ref027">27</xref>].</p><p>After the meta-analyses, we performed an additional quality control step on the meta-analytic results: We only included SNPs (i) being available in at least half of the maximum sample size in all strata; and (ii) having chromosome and position annotation in dbSNP.</p><p>Our study aimed at discovering SNPs with (1) <bold><italic>age-different</italic></bold> effects, (2) <bold><italic>sex-different</italic></bold> effects, and (3) <bold><italic>age-dependent sex-different</italic></bold> effects or <bold><italic>sex-dependent age-different</italic></bold> effects.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">To find <italic>age-different</italic> genetic effects, we computed age-difference P-values (<italic>P</italic>
<sub><italic>agediff</italic></sub>) by testing for difference between the age group-specific meta-analyzed beta-estimates <italic>b</italic>
<sub>&#8804;50<italic>y</italic></sub> and <italic>b</italic>
<sub>&gt;50<italic>y</italic></sub> using
<disp-formula id="pgen.1005378.e001"><alternatives><graphic id="pgen.1005378.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005378.e001.jpg" /><ns1:math id="M1"><ns1:msub><ns1:mrow><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>b</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>&#8804;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>b</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>&gt;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mrow><ns1:msqrt><ns1:msubsup><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>&#8804;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>&gt;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>&#8722;</ns1:mo><ns1:mn>2</ns1:mn><ns1:msub><ns1:mrow><ns1:mi>r</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8729;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>&#8804;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8729;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>&gt;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow></ns1:msub></ns1:msqrt></ns1:mrow></ns1:mfrac><ns1:mo>.</ns1:mo></ns1:math></alternatives></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The correlation <italic>r</italic>
<sub><italic>age</italic></sub> between <italic>b</italic>
<sub>&#8804;50<italic>y</italic></sub> and <italic>b</italic>
<sub>&gt;50<italic>y</italic></sub> computed as the Spearman rank correlation coefficient across all SNPs for BMI and WHR<sub>adjBMI</sub> was 0.123 and 0.049, respectively. The analogous test statistic for <italic>sex-different</italic> effects was
<disp-formula id="pgen.1005378.e002"><alternatives><graphic id="pgen.1005378.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005378.e002.jpg" /><ns1:math id="M2"><ns1:msub><ns1:mrow><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>x</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>b</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>M</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>b</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>F</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mrow><ns1:msqrt><ns1:msubsup><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>M</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>F</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>&#8722;</ns1:mo><ns1:mn>2</ns1:mn><ns1:msub><ns1:mrow><ns1:mi>r</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>x</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8729;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>M</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8729;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>F</ns1:mi></ns1:mrow></ns1:msub></ns1:msqrt></ns1:mrow></ns1:mfrac><ns1:mo>,</ns1:mo></ns1:math></alternatives></disp-formula>
with corresponding P-value (<italic>P</italic>
<sub><italic>sexdiff</italic></sub>). The Spearman correlation <italic>r</italic>
<sub><italic>sex</italic></sub> was 0.121 or 0.047 for BMI and WHR<sub>adjBMI</sub>, respectively.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">To test for the three-way interaction of age- and sex-differences, we introduced for the first time a test of difference between age groups in the sex-difference, which is mathematical equivalent to a test of difference between sexes in the age group-difference using the age-sex-difference statistic as
<disp-formula id="pgen.1005378.e003"><alternatives><graphic id="pgen.1005378.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005378.e003.jpg" /><ns1:math id="M3"><ns1:msub><ns1:mrow><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>x</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>b</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>M</ns1:mi><ns1:mo>&#8804;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>b</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>F</ns1:mi><ns1:mo>&#8804;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo><ns1:mo>&#8722;</ns1:mo><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>b</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>M</ns1:mi><ns1:mo>&gt;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>b</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>F</ns1:mi><ns1:mo>&gt;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mrow><ns1:msqrt><ns1:msubsup><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>M</ns1:mi><ns1:mo>&#8804;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>F</ns1:mi><ns1:mo>&#8804;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>M</ns1:mi><ns1:mo>&gt;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>F</ns1:mi><ns1:mo>&gt;</ns1:mo><ns1:mn>50</ns1:mn><ns1:mi>y</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:msqrt></ns1:mrow></ns1:mfrac><ns1:mo>,</ns1:mo></ns1:math></alternatives></disp-formula>
with the corresponding P-value (<italic>P</italic>
<sub><italic>agesexdiff</italic></sub>).</p><p>To maximize statistical power we did not split our samples (artificially) into discovery and replication sets, but meta-analyzed all studies together and verified the absence of cross-study heterogeneity. We screened genome-wide for <italic>P</italic>
<sub><italic>agediff</italic></sub>, <italic>P</italic>
<sub><italic>sexdiff</italic></sub>, and <italic>P</italic>
<sub><italic>agesexdiff</italic></sub> for each of the two traits (BMI, WHR<sub>adjBMI</sub>). These screens have ideal power to detect effects that are of opposite direction across the four strata (<bold><xref ref-type="supplementary-material" rid="pgen.1005378.s009">S9 Fig</xref></bold>). However, searching for effects that are prominent in one or some strata, but not existent or directionally consistent and less pronounced in other strata profits from an a priori filter on the overall association (<italic>P</italic>
<sub><italic>overall</italic></sub> &lt; 10<sup>&#8722;5</sup>) as shown previously [<xref ref-type="bibr" rid="pgen.1005378.ref010">10</xref>, <xref ref-type="bibr" rid="pgen.1005378.ref026">26</xref>] (<bold><xref ref-type="fig" rid="pgen.1005378.g004">Fig 4</xref></bold>). The rationale behind this filter is that SNPs with unequal effects in the different strata have non-zero overall effect when tested in all strata combined. This is true unless these effects are the same magnitude, but in opposite direction (i.e. cancel out in the combined analysis). Hence filtering on overall association P-value possibly enriches our selection with SNPs showing interaction effects. For BMI and WHR<sub>adjBMI</sub> 7,382 and 2,014 SNPs passed this filter.</p><p>For each trait and for each of the 6 approaches (<italic>P</italic>
<sub><italic>agediff</italic></sub>, <italic>P</italic>
<sub><italic>sexdiff</italic></sub>, <italic>P</italic>
<sub><italic>agesexdiff</italic></sub>; with and without a priori filtering), we controlled the False Discovery Rate (FDR) at 5% to account for the multiple testing [<xref ref-type="bibr" rid="pgen.1005378.ref076">76</xref>]. Importantly, controlling the FDR of each single analysis at 5% implies a global FDR control at 5% for the ensemble of discoveries resulting from all the different approaches together.</p><p>To ensure the association of none of our age- or sex-specific loci were driven by ascertainment bias through inclusion of case-series of individuals with type 2 diabetes or coronary artery disease, we performed additional meta-analyses restricted to population-based (i.e. no ascertainment bias) studies and compared the effect-sizes between the original meta-analyses and the meta-analyses restricted to population-based studies.</p><p>Self-reported BMI or WHR may cause systematic measurement error that might lead to biased effect estimates. Few of our studies assessed BMI and WHR by self-report in the sense that they told study participants how to measure BMI and WHR for themselves. In order to ensure that the age- or sex-differences of our identified loci was not driven by the few studies that used self-report data (13 of our 114 studies), which may introduce bias [<xref ref-type="bibr" rid="pgen.1005378.ref077">77</xref>&#8211;<xref ref-type="bibr" rid="pgen.1005378.ref079">79</xref>], we conducted sensitivity meta-analyses limited to studies that measured anthropometric phenotypes (<bold><xref ref-type="supplementary-material" rid="pgen.1005378.s005">S5</xref></bold> and <bold><xref ref-type="supplementary-material" rid="pgen.1005378.s008">S8</xref> Figs</bold>).</p><p>To illustrate the strength and characteristics of the various screens outlined, we analytically computed power by scan (<bold><xref ref-type="supplementary-material" rid="pgen.1005378.s009">S9 Fig</xref></bold>) and for all scans combined (<bold>Figs <xref ref-type="fig" rid="pgen.1005378.g004">4</xref>, <xref ref-type="supplementary-material" rid="pgen.1005378.s010">S10</xref></bold> and <bold><xref ref-type="supplementary-material" rid="pgen.1005378.s011">S11</xref></bold>), for varying configurations of effect size combinations and directions across the four strata. More specifically, we assumed equally sized strata, a total sample size approximately corresponding to the maximum sample size of our study and modelled three categories of SNPs explaining realistic fractions of the phenotypic variance, i.e. small, medium and large effects from Speliotes et al [<xref ref-type="bibr" rid="pgen.1005378.ref028">28</xref>] and from Heid et al [<xref ref-type="bibr" rid="pgen.1005378.ref029">29</xref>]. The power shown in any of the heatplots was calculated based on a fixed effect in women &#8804;50y (set to the known effect), a fixed effect in men &gt;50y (set to 0), and varying effects in women &gt;50y and men &#8804;50y (varying from negative to positive magnitude of the known effect). This strategy allowed us to depict power for most important interaction effects (i.e. for pure sex-difference, pure age-difference, 1-strata interaction and 3-strata interaction) in a single heatplot (see legend of <bold><xref ref-type="fig" rid="pgen.1005378.g004">Fig 4</xref></bold>).</p><p>To identify novel genetic association for BMI and WHR<sub>adjBMI</sub>, we screened (i) the <italic>P</italic>
<sub><italic>Overall</italic></sub> gathered from a four-way meta-analysis of the stratified results and (ii) the <italic>P</italic>
<sub><italic>Joint</italic></sub> gathered from a four-way joint meta-analysis of the stratified results according to Aschard et al [<xref ref-type="bibr" rid="pgen.1005378.ref027">27</xref>]. We used a genome-wide significance level (P&lt;5x10<sup>-8</sup>) for both approaches to correct for the multiple testing and compared the detected regions to previously established loci using a 500kb distance criterion.</p><p>For WHR<sub>adjBMI</sub>, we counted among the sex-different associations (disregarding the opposite effect loci) how many were significantly stronger in men or women. To test whether the observed counts represent significant imbalances between sexes we compared them to the expected binomial distribution (with p = 0.5). Similar exercise was done for age-specific associations for BMI.</p><p>Age-group specific association results of the identified loci were requested for blood-pressure measures (diastolic and systolic blood pressure, mean arterial pressure and pulse pressure) from the Global-BPGen consortium [<xref ref-type="bibr" rid="pgen.1005378.ref030">30</xref>]. The provided effect size and standard error estimates for six age bins (20&#8211;29, 30&#8211;29, &#8230;, 70&#8211;79 years) were combined to derive SNP x AGE interaction effect sizes and P-Values (<bold><xref ref-type="supplementary-material" rid="pgen.1005378.s022">S6 Table</xref></bold>) using meta-regression [<xref ref-type="bibr" rid="pgen.1005378.ref034">34</xref>].</p><p>Sex-specific associations of the identified loci were requested for lipid traits (HDL-C, LDL-C, Total Cholesterol and Triglycerides) from the Global Lipids Genetics Consortium [<xref ref-type="bibr" rid="pgen.1005378.ref031">31</xref>], for type 2 diabetes (T2D) from the DIAGRAM consortium[<xref ref-type="bibr" rid="pgen.1005378.ref032">32</xref>], for glycemic traits (fasting insulin, fasting glucose, HOMA-B, HOMA-IR) from the Meta-Analyses of Glucose and Insulin-related traits (MAGIC) Consortium[<xref ref-type="bibr" rid="pgen.1005378.ref033">33</xref>](personal communication), and for blood-pressure measures (diastolic and systolic blood pressure) from the Global-BPGen consortium [<xref ref-type="bibr" rid="pgen.1005378.ref030">30</xref>] (<bold><xref ref-type="supplementary-material" rid="pgen.1005378.s023">S7</xref>, <xref ref-type="supplementary-material" rid="pgen.1005378.s024">S8</xref></bold> and <bold><xref ref-type="supplementary-material" rid="pgen.1005378.s025">S9</xref> Tables</bold>). The provided men- and women-specific estimates were used to derive sex-difference P-Values.</p><p>To further investigate the identified genetic variants in this study and to gain additional insight into their functionality and possible pleiotropic effects, we searched for previous SNP-trait associations nearby our lead SNPs. PLINK was used to find all SNPs within 500 kb of any of our lead SNPs using 1000 Genomes Project Pilot I genotype data from the CEPH (Utah residents with ancestry from northern and western Europe) population (CEU) [<xref ref-type="bibr" rid="pgen.1005378.ref080">80</xref>, <xref ref-type="bibr" rid="pgen.1005378.ref081">81</xref>]. To identify previous associations, all SNPs within the specified regions were compared with the NHGRI (National Human Genome Research Institute) catalog for overlap and distances between the two SNPs were obtained using SAS, Version 9.2 [citation info below for SAS and PLINK] [<xref ref-type="bibr" rid="pgen.1005378.ref082">82</xref>]. The NHGRI&#8217;s (National Human Genome Research Institute) GWAS catalog contains only the top 30 most significant SNP-trait associations from recent GWAS published results from studies with at least 100,000 SNPs with resulting P-values of less than P&lt;1x10<sup>-5</sup> [<xref ref-type="bibr" rid="pgen.1005378.ref082">82</xref>]. For previous GWAS results not reported in the Catalog when accessed on 10/15/2014, additional SNP-trait associations were pulled from the literature and compared to our lead SNPs using the same PLINK output file to obtain distance and r<sup>2</sup> values [<xref ref-type="bibr" rid="pgen.1005378.ref083">83</xref>&#8211;<xref ref-type="bibr" rid="pgen.1005378.ref091">91</xref>]. All previous associations within 500 kb and with an r<sup>2</sup>&gt;0.1 with our lead SNP that reached genome-wide significance in the previous publication were retained for further interrogation.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Summary statistics from a genome-wide association meta-analyses previously performed by EGG Consortium (<ext-link ext-link-type="uri" ns0:href="http://www.egg-consortium.org/">www.egg-consortium.org</ext-link>) were used to examine whether the 15 age-specific BMI loci associate with birth weight and/or childhood obesity risk. Birth weight (BW) had been transformed to z-scores. Association between each SNP and the birth weight was tested using linear regression assuming an additive genetic model, with sex and, where available, gestational age as covariables [<xref ref-type="bibr" rid="pgen.1005378.ref036">36</xref>]. In the genome-wide association meta-analysis for childhood obesity risk, cases were defined as having an age- and sex-specific BMI &gt; 95th percentile, and controls as having an age- and sex-specific BMI &lt; 50th percentile in children of European ancestry. SNP associations were assessed in a case-control design assuming an additive genetic model [<xref ref-type="bibr" rid="pgen.1005378.ref037">37</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We compared the effect sizes for 15 loci with age related differences in BMI for each of the age strata (&#8804;50y and &gt;50y) in men and women combined with the BMI in young adults ages 16&#8211;25 years [<xref ref-type="bibr" rid="pgen.1005378.ref058">58</xref>]. Nine out of the 14 studies included in the young adult analysis had overlapping samples with the current sample, although the BMI measurements utilized were different (i.e. adolescence/early adulthood versus middle-aged to older adulthood). We used t-tests to compare effect estimates (&#946;) from the younger adults aged 16&#8211;25 years (A) to each of our age strata (&#8804;50y or &gt;50y) (B) adjusting for the correlation due to overlapping samples such that:
<disp-formula id="pgen.1005378.e004"><alternatives><graphic id="pgen.1005378.e004g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005378.e004.jpg" /><ns1:math id="M4"><ns1:msub><ns1:mrow><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>d</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>f</ns1:mi><ns1:mi>f</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>b</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>A</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>b</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>B</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mrow><ns1:msqrt><ns1:msubsup><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>A</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>B</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>&#8722;</ns1:mo><ns1:mn>2</ns1:mn><ns1:mi>r</ns1:mi><ns1:mo>&#8729;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>A</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8729;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>B</ns1:mi></ns1:mrow></ns1:msub></ns1:msqrt></ns1:mrow></ns1:mfrac><ns1:mo>,</ns1:mo></ns1:math></alternatives></disp-formula>
where SE = standard error and r = Spearman correlation coefficient between the effect estimates genome-wide. We calculated the Spearman correlation <italic>r</italic> between our study and GIANT using the combined stages from both studies. The significance level (P-value) was based on a two-tailed t-test.</p><p>We also evaluated the 15 BMI loci showing age-dependent results from genome-wide analyses with weight change across adulthood. Using growth curves generated from multiple measures of weight in individuals between the ages of 20 and 65 years, weight change trajectories were calculated by sex using age as both a random and fixed effect. For each of the 15 loci showing age-differences in BMI, we observationally compared the direction of the effect estimate in the weight change results with the direction of effect seen between our adults aged 18&#8211;50 years and adults &gt;50 years. While assuming constant height across adulthood and no cohort effect between the two age-groups, we hypothesized that for loci where we find a stronger effect for BMI in the adults ages 18&#8211;50 years compared to adults &gt;50 years, the direction of effect estimate in the weight change data would be negative. For the loci where we found a stronger effect for BMI in the adults &gt;50 years compared to the adults ages 18&#8211;50 years we hypothesized that the direction of effect estimate in the weight change data would be positive.</p><p>We examined transcript expression of genes nearby (+/- 1Mb) the 44 identified WHR<sub>adjBMI</sub> SNP in lymphoblastoid human cell lines available in 2,360 human samples from the EGCUT and Groningen cohorts (910 women and 1,450 men) [<xref ref-type="bibr" rid="pgen.1005378.ref039">39</xref>, <xref ref-type="bibr" rid="pgen.1005378.ref040">40</xref>]. We computed sex-specific associations between each of the 44 variants and all genes in their 1Mb vicinity and tested the men- and women-specific eQTLs for sex-difference (<italic>FDR</italic>
<sub><italic>sexdiff</italic></sub>&lt;5% calculated with/without initial filter on overall expression effect <italic>FDR</italic>
<sub><italic>overall</italic></sub>&lt;20%).</p><p>We next examined whether the 15 SNPs identified to be age-dependently associated with BMI impact nearby (+/- 1Mb) transcripts differently in younger (&lt;50y) than in older individuals (&#8805;50y). As such, we analyzed human whole blood transcription in 3,489 unrelated individuals from NESDA and NTR cohorts [<xref ref-type="bibr" rid="pgen.1005378.ref042">42</xref>, <xref ref-type="bibr" rid="pgen.1005378.ref043">43</xref>], which were divided in a &#8805;50y group (N = 958) and a &lt;50y group (N = 2,531). Cis-eQTL analysis for the 15 SNPs was conducted for the two groups separately and age-group specific eQTLs were compared for age-difference (<italic>FDR</italic>
<sub><italic>agediff</italic></sub>&lt;5% calculated with/without initial filter on overall expression effect <italic>FDR</italic>
<sub><italic>overall</italic></sub>&lt;20%).</p><p>We performed a microarray analysis on data from an experiment previously published [<xref ref-type="bibr" rid="pgen.1005378.ref044">44</xref>]. Briefly, 21 male, 21 female, and 21 ovariectomized (OVX) female C57/BL6 mice were fed from day 21 for 12 weeks on an high fat diet (45% calories from fat; Research Diets, Inc., New Brunswick, NJ). All mice (male, female, and OVX) were exposed to sham or OVX surgery. Animals were sacrificed and tissues collected during the first 2h of the beginning of the light cycle after a 12h fast.</p><p>GeneChip microarray (Affymetrix, Santa Clara, CA) was performed according to manufacturer&#8217;s instructions on 7 independent pooled samples (3 mice per pooled sample) per experimental group (male, female, OVX) from gonadal adipose tissue (GWAT) and inguinal adipose tissue (IWAT) fat pads. AMC Project Report Version 12 (6/27/07) GeneChip Operating System parameters &#945;1 and &#945;2 were set to 0.05 and 0.065, respectively. Normalized expression values from the Affymetrix identifier were analyzed with the online software server Genesifter (VizX Labs, Inc., Seattle, WA, USA). For comparisons of microarray data sets, multiple t-tests were used to identify genes with at least a twofold difference in gene expression (with Benjamini and Hochberg correction; P&lt;0.05) and at least an expression level of 100. Genes populated from the GWAS studies were compared to this list of genes that met the minimum criteria of expression, fold difference, and p-value. Those identified as being statistically significant were further validated by qPCR.</p><p>We used a recently developed pathway enrichment method, DEPICT [<xref ref-type="bibr" rid="pgen.1005378.ref047">47</xref>]. The methodology first selects all lead SNPs below a certain threshold with respect to a target P-value (available genome-wide). We tested multiple hypotheses corresponding to different lead SNP selection scenarios. First, we selected SNPs with <italic>P</italic>
<sub><italic>sexdiff</italic></sub> &lt;0.001. Second, focusing on SNPs with concordant effect size direction (CED), but different magnitude we added a marginal filter to boost power by selecting SNPs with <italic>P</italic>
<sub><italic>sexdiff</italic></sub> &lt;0.01 and <italic>P</italic>
<sub><italic>overall</italic></sub> &lt;0.01. In case of CED, SNPs with stronger effect in women may fall into separate pathways from SNPs with stronger effect in men. Hence, we have derived gender-specific sex-difference P-values (P<sub>sexdiff_F</sub>, P<sub>sexdiff_M</sub>). We then looked for women-specific pathway enrichment by selecting SNPs with P<sub>sexdiff_F</sub> &lt;0.01 and <italic>P</italic>
<sub><italic>overall</italic></sub> &lt;0.01 (given the CED framework). Similarly, we created a separate list for men-specific SNPs by a filter of P<sub>sexdiff_M</sub> &lt;0.01 and <italic>P</italic>
<sub><italic>overall</italic></sub> &lt;0.01. All above lists were also created for age-dependent BMI associations by replacing P<sub>sexdiff</sub> by P<sub>agediff</sub>, P<sub>sexdiff_F</sub> by P<sub>agediff_younger</sub> and P<sub>sexdiff_M</sub> by P<sub>agediff_older</sub>.</p><p>For each of the eight SNP lists, leads SNPs were identified. For each lead SNP locus a target region is defined as the smallest interval containing all SNPs with LD&gt;0.5 with the lead SNP of the locus. All genes encompassed in the target regions represent the &#8220;GWAS genes&#8221;, thereby assuming that either the lead SNP is in LD with a functional coding SNP within a gene or that the lead SNP marks a cis-acting regulatory region. We then used the following pre-defined gene sets and pathways: Gene Ontology (GO), Reactome, InWeb protein complexes, Mouse knock-out phenotypes. Gene sets were re-annotated based on co-expression in a large collection (80,000) of gene expression compendium from GEO. Then, for each gene-set the pair-wise similarity between GWAS genes was calculated and compared to that of matching sets of non-GWAS genes to assess significance of enrichment.</p><p>DEPICT also generated a prioritized set of genes at each locus. Briefly, genes within associated loci that are functionally similar to genes from other associated loci are the more likely candidates. DEPICT prioritizes genes in three steps: gene scoring, bias adjustment, and false discovery rate estimation. In the scoring step, the method quantifies the similarity of a given gene to genes from other associated loci. The bias adjustment step controls confounding factors that may bias the gene scores, e.g. gene length. In the last step, experiment-wide false discovery rates are estimated by repeating the scoring and bias adjustment steps 20 times based on top SNPs from pre-computed null GWAS.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Significantly associated loci were further explored using QIAGEN&#8217;s Ingenuity Pathway Analysis (IPA, QIAGEN Redwood City, <ext-link ext-link-type="uri" ns0:href="http://www.qiagen.com/ingenuity">www.qiagen.com/ingenuity</ext-link>) to determine if there was an over-arching functional or disease relationship among these loci and their associated genes using the age and sex specific SNP lists described above. IPA uses publicly available databases (e.g. NHGRI GWAS Catalog, NCBI databases, KEGG) and proprietary databases of gene/protein interaction, expression, and function to identify possible pathways, networks, and overlapping functions of genes. For our analysis, IPA identified potential genes as those genes with coding regions within 2kb upstream or 0.5kb downstream of our list of input dbSNP ids that can unambiguously be mapped to these ids. To perform the analyses, only Ingenuity Knowledge Base genes were used, both direct and indirect relationships that are observed or predicted in mammals (humans, mice, and rats) are strictly considered. All canonical pathways and functional/disease categories and annotations that were statistically significant (P &lt; 0.05 using the Fisher&#8217;s exact test) are reported; however, those that meet significance for multiple test correction (Benjamin-Hochberg corrected P &lt;0.05) are highlighted in the table. Only the top ten predicted networks containing up to 140 genes or endogenous molecules were requested. Only those networks with a score of greater than 2 (Fisher&#8217;s Exact Test result of P &lt;0.01) are considered significant [<xref ref-type="bibr" rid="pgen.1005378.ref092">92</xref>].</p><p>We estimated the broad heritability (H<sup>2</sup>) of BMI and WHR<sub>adjBMI</sub> within the Family Heart Study (FHS) to assess how much of each trait&#8217;s total phenotypic variance may be genetic. A random sample of 1,810 individuals (454 families) was used for this analysis. The sample was stratified by age and sex into 9 groups (all, all &#8804;50y, all &gt;50y, men, women, men &#8804;50y, men &gt;50y, women &#8804;50y, women &gt;50y) to assess how each trait&#8217;s genetic variance may differ across strata. Within each group, BMI and WHR<sub>adjBMI</sub> were adjusted for age, age<sup>2</sup>, genotyping chips (Illumina 560K, 1,000,000K, 610K), 10 principal components and 3 study centers. Residuals for BMI and WHR<sub>adjBMI</sub> were ranked and an inverse normal transformation was applied. Subsequently, SOLAR was used to estimate the H<sup>2</sup> of BMI and WHR<sub>adjBMI</sub> within each group (<bold><xref ref-type="supplementary-material" rid="pgen.1005378.s044">S28 Table</xref></bold>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To explore the contribution of all common (genotyped) SNPs genome-wide to each trait of interest, BMI and WHR<sub>adjBMI</sub>, we estimated the variance explained by all the autosomal SNPs in the combined ARIC, KORA S3/S4, CoLaus, EGCUT and SHIP studies within each of the sex and age strata, using the method proposed by Yang et al [<xref ref-type="bibr" rid="pgen.1005378.ref093">93</xref>] and implemented in the Genome-wide Complex Trait Analysis software package (GCTA <ext-link ext-link-type="uri" ns0:href="http://www.complextraitgenomics.com/software/gcta/">http://www.complextraitgenomics.com/software/gcta/</ext-link>). Each phenotypic trait was transformed in the same form as was used for all meta-analyses.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We estimated the age-group and sex-specific polygenic variance explained by various subsets of SNPs that were based on varying thresholds of overall association (P<sub>Overall</sub>) with BMI or WHR<sub>adjBMI</sub>. First, each subset of SNPs was clumped into independent regions using a physical distance criterion &lt;500kb and for each region the most significantly overall associated SNP (i.e. top SNP) was taken further. For each top SNP, the explained variance was calculated according to
<disp-formula id="pgen.1005378.e005"><alternatives><graphic id="pgen.1005378.e005g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005378.e005.jpg" /><ns1:math id="M5"><ns1:msup><ns1:mrow><ns1:mi>r</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msup><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>+</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow><ns1:mrow><ns1:msup><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msup><ns1:mrow><ns1:mi mathvariant="normal">&#934;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>&#8722;</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msup><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mfrac><ns1:mrow><ns1:mi>P</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:mfrac></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msup></ns1:mrow></ns1:mfrac></ns1:mrow></ns1:mfrac><ns1:mo>&#8211;</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:msup><ns1:mrow><ns1:mi>r</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msup></ns1:mrow><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow></ns1:mfrac></ns1:math></alternatives></disp-formula>
for each age-group and for each sex separately [<xref ref-type="bibr" rid="pgen.1005378.ref094">94</xref>]. Finally, the variance explained by the subset of SNPs was obtained by summing up the single SNP-specific explained variances. The overall association threshold was varied from 1x10<sup>-8</sup> to 0.1.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We examined whether SNPs known to provide reliable tags for Copy-Number-Variations (CNVs) in subjects of European-descent (combining four catalogues including 60,167 CNV-tagging SNPs as described previously [<xref ref-type="bibr" rid="pgen.1005378.ref095">95</xref>]) correlated with our lead SNPs. We also performed several online database searches to establish whether known variants within a 500kb-window on both sides of each lead SNP, that are in high linkage disequilibrium (r&#178; &gt; 0.8) with our lead SNPs (using SNAP Proxy search [<xref ref-type="bibr" rid="pgen.1005378.ref096">96</xref>]), might have putative or predicted function. (i) We searched the SIFT database [<xref ref-type="bibr" rid="pgen.1005378.ref097">97</xref>] to determine whether any of these SNPs were predicted to affect protein function. (ii) We used Annovar [<xref ref-type="bibr" rid="pgen.1005378.ref098">98</xref>] to investigate predicted and putative function in several functional classes, including splicing regulation, stop codons, polyphen predictions. (iii) We used the regulome database (<ext-link ext-link-type="uri" ns0:href="http://regulome.stanford.edu/">http://regulome.stanford.edu/</ext-link>) to search for known and predicted regulatory elements (DNAase hypersensitivity, binding sites of transcription factors, and promoter regions) in the intergenic regions of our age-specific BMI and sex-specific WHR<sub>adjBMI</sub> loci. Additionally, we searched for estrogen, androgen or progesterone receptor motifs around our sex-specific WHR<sub>adjBMI</sub> loci. Source of these data include public datasets from GEO, the ENCODE project, and published literature [<xref ref-type="bibr" rid="pgen.1005378.ref099">99</xref>].</p>