<p>The <italic>CAN1</italic> gene was replaced with <italic>URA3</italic> from the pUG72 plasmid <xref ref-type="bibr" rid="pgen.1004846-Gueldener1">[38]</xref> (primers &#8220;CAN1 Del Ura3&#8221; in <xref ref-type="supplementary-material" rid="pgen.1004846.s006">Materials and Methods S1</xref>) in the RNR mutated strain (<italic>rnr1-Y285A</italic>) previously described <xref ref-type="bibr" rid="pgen.1004846-Kumar1">[16]</xref>. PCR amplified <italic>WT CAN1</italic> in reversed orientation (primers &#8220;CAN1 orientation&#8221; in <xref ref-type="supplementary-material" rid="pgen.1004846.s006">Materials and Methods S1</xref>) was then transformed into the <italic>can1:: URA3</italic> strain to give <italic>rnr1-Y285A CAN1 OR2</italic>. 5-FOA selection allowed the elimination of any <italic>can1:: URA3</italic> cells and the <italic>CAN1</italic> reverse orientation was confirmed by sequencing (&#8220;can1ori scr&#8221; primers in <xref ref-type="supplementary-material" rid="pgen.1004846.s006">Materials and Methods S1</xref>).</p><p>An MMR deficient strain was created by deleting <italic>MSH2</italic> in the AC402 wt (to give <italic>msh2</italic>&#916;) using the pAG32 plasmid and transfection technique <xref ref-type="bibr" rid="pgen.1004846-Goldstein1">[39]</xref> and the primers msh2_hphMX4, shown in <xref ref-type="supplementary-material" rid="pgen.1004846.s006">Materials and Methods S1</xref>. The deletion was confirmed using primers flanking <italic>MSH2</italic>. This strain was then crossed with <italic>rnr1-Y285A</italic>
<xref ref-type="bibr" rid="pgen.1004846-Kumar1">[16]</xref>, sporulated and dissected spores selected on Hygromycin and &#8211;Trp plates for the double mutant <italic>rnr1-Y285A msh2</italic>&#916;.</p><p>All culturing was carried out at 30&#176;C in YPAD (1% yeast extract, 2% bacto-peptone, 20 mg/l adenine, 2% agar for plates) liquid cultures in a shaking incubator at 160rpm. The Canavanine resistance assay was used to calculate mutation rates as previously described <xref ref-type="bibr" rid="pgen.1004846-Kumar2">[17]</xref>, <xref ref-type="bibr" rid="pgen.1004846-Drake1">[40]</xref>, <xref ref-type="bibr" rid="pgen.1004846-Lea1">[41]</xref>. The Can<sup>r</sup> colonies were picked and the <italic>CAN1</italic> gene amplified and sequenced (MWG Eurofins) using published primers <xref ref-type="bibr" rid="pgen.1004846-Kumar2">[17]</xref> to produce the mutation spectra.</p><p>dNTP pools were measured in asynchronous cultures as described previously <xref ref-type="bibr" rid="pgen.1004846-Kumar1">[16]</xref> with minor changes as described in <xref ref-type="bibr" rid="pgen.1004846-Hoch1">[42]</xref>. Briefly, cells were harvested by filtration at a density of 0.4&#215;10<sup>7</sup> to 0.5&#215;10<sup>7</sup> cells/ml and NTPs and dNTPs were extracted in trichloroacetic acid and MgCl<sub>2</sub> followed by a Freon-trioctylamine mix. dNTPs were separated using boronate columns (Affigel 601, Biorad) and analysed by HPLC on a LaChrom Elite UV detector (Hitachi) with a Partisphere SAX column (Hichrom, UK).</p>