<p>Mice were housed and bred on an outcrossed Swiss Webster background in a pathogen-free facility at the National Cancer Institute (NCI), Bethesda, MD. Genotyping of mice was performed by allele-specific PCR on DNA extracted from tail tissue. All experiments were performed in accordance with institutional guidelines according to IACUC-approved protocols. <italic>En1</italic><sup><italic>Cre/+</italic></sup><italic>; &#946;-catenin</italic><sup><italic>flox/&#916;</italic></sup> and control samples were provided by Dr. Radhika P. Atit. Some <italic>Shh</italic><sup><italic>GFPcre/+</italic></sup>; <italic>R26</italic><sup><italic>YFP/+</italic></sup> samples were provided by Dr. Sunny Y. Wong. <italic>Edar</italic><sup><italic>dl-J/+</italic></sup>, <italic>Atoh1</italic><sup><italic>LacZ/+</italic></sup>, <italic>Shh</italic><sup><italic>GFPcre/+</italic></sup>, <italic>Gli1</italic><sup><italic>LacZ/+</italic></sup>, <italic>Gli1</italic><sup><italic>CreER/+</italic></sup>, <italic>K14-Cre</italic>, <italic>Shh</italic><sup><italic>flox/+</italic></sup>, <italic>K5-tTA</italic>, <italic>TRE-cre</italic>, <italic>Smo</italic><sup><italic>flox/+</italic></sup>, <italic>R26</italic><sup><italic>YFP/+</italic></sup>, <italic>R26</italic><sup><italic>LacZ/+</italic></sup>, and <italic>Fgf20</italic><sup><italic>LacZ/+</italic></sup> mice were described previously as cited in the text. For all embryonic and neonatal observations, littermate animals with a wildtype copy of the targeted allele and/or lacking cre recombinase activity were used as controls.</p><p><italic>K5-tTA; TRE-cre; Smo</italic><sup><italic>flox/flox</italic></sup> mice were bred and maintained on a standard rodent diet without doxycycline during embryonic development. Tamoxifen (Sigma) was dissolved in corn oil (20mg/ml) and administered (2mg intraperitoneal injection per gravid mouse) to induce CreER.</p><p>Skin was fixed in 4% paraformaldehyde for 15 minutes (for X-gal staining) or overnight (for immunostaining). Tissue was whole mount-stained or cryoprotected overnight in 30% sucrose, embedded in OCT, and frozen; 12-&#956;m sections were obtained.</p><p>Standard and whole mount immunostaining procedures were performed. Tissue sections on glass slides were fixed in 4% paraformaldehyde for 15 minutes before incubation in 10% serum in 0.1% PBT (0.1% Triton X-100 in PBS) for 1 hour and then in primary antibody (in 5% serum/0.1% PBT) overnight at 4&#176;C. The primary antibodies used were: rat anti-K8 (1:50, University of Iowa), rabbit anti-K17 (1:200, Epitomics), chicken anti-GFP (1:1000, Abcam), rabbit anti-GFP (1:500, Abcam), rabbit anti-Sox2 (1:500, Stemgent), and rabbit anti-NCAM (1:500, Millipore). Alexa Fluor-conjugated secondary antibodies (1:2000, Invitrogen) were used to detect the signals. Whole mount immunostaining followed the online protocol as described [<xref ref-type="bibr" rid="pgen.1006150.ref052">52</xref>]. Concomitant staining of littermate control tissue and control staining where the primary antibody was omitted were used to confirm the specificity of experimental staining. Confocal images were acquired with the Zeiss LSM 710 Confocal system (Carl Zeiss Inc, Thornwood, NY).</p><p>The Troma-1/K8 antibody developed by Dr. Philippe Brulet and Dr. Rolf Kemler was obtained from the Developmental Studies Hybridoma Bank, developed under the auspices of the National Institute of Child Health and Human Development and maintained by The University of Iowa, Department of Biology, Iowa City, IA 52242.</p><p>Merkel cells per touch dome and touch dome per mm<sup>2</sup> numbers were assessed by direct visualization of immunofluorescently stained K8+ Merkel cells in whole mount dorsal trunk skin from experimental and control animals. Counting was performed by a blinded observer. In tissue sections, Merkel cells were visualized by K8 and/or Sox2 staining and were scored for co-staining with GFP. Reporter recombination in primary hair follicles was quantified by counting total GFP staining cells and dividing by total number of DAPI stained nuclei within a preselected region of each follicle.</p><p>Total RNA from embryonic trunk skin was purified using RNeasy Micro Kit (Qiagen) and reverse-transcribed into cDNA following the manufacturer&#8217;s manual (Invitrogen #11752). Then quantitative RT-PCR, with actin as control, using SYBR Green was performed to detect RNA expression. Data are presented as means &#177; SD. The sequences of PCR primers are shown in <xref ref-type="supplementary-material" rid="pgen.1006150.s007">S1 Table</xref>.</p><p>Embryonic skin culture was performed as described [<xref ref-type="bibr" rid="pgen.1006150.ref053">53</xref>]. E13.5 embryos were collected, each embryonic mouse was cut through the sagittal midline and eviscerated, half was cultured in media (DMEM +10% FBS +1% penicillin/streptomycin) as a control group, and half was cultured in media with 25 nM Smo agonist Hh-Ag1.5 (Xcess Biosciences Inc.) as the treated group. Media was changed the next day, and skin was harvested 2 days after culture, followed immediately by RNA isolation.</p><p>Population data sets are shown as the mean values, and error bars represent SD. For comparisons between sets, a two-tailed t-test was applied.</p>