<p>
<italic>P. falciparum</italic> was cultured as previously described <xref ref-type="bibr" rid="pgen.1004382-Trager1">[70]</xref> and synchronized using the sorbitol method <xref ref-type="bibr" rid="pgen.1004382-Lambros1">[71]</xref>. The F1 progeny from the HB3 x Dd2 cross have been described <xref ref-type="bibr" rid="pgen.1004382-Su1">[37]</xref> and were obtained from MR4. The identity of all progeny was verified by PCR using eight highly polymorphic microsatellite markers as described <xref ref-type="bibr" rid="pgen.1004382-Sanchez5">[42]</xref>.</p><p>The transfection vector expressing <italic>pfcrt</italic> tagged in frame with the coding sequence of the green fluorescence protein has recently been described <xref ref-type="bibr" rid="pgen.1004382-Kuhn1">[69]</xref>. Briefly, the <italic>pfcrt</italic> coding sequence from the chloroquine resistant <italic>P. falciparum</italic> strain Dd2 or the wild type sequence from the chloroquine sensitive strain HB3 was cloned into the <italic>Xho</italic>I and <italic>Avr</italic>II restricted pARL1a<sup>+</sup> transfection vector <xref ref-type="bibr" rid="pgen.1004382-Crabb1">[72]</xref> in a manner that allowed <italic>pfcrt</italic> expression to be controlled by its own promoter. The GFP coding sequences was subsequently cloned into <italic>Avr</italic>II and <italic>Xma</italic>I restricted pARL1a<sup>+</sup> vector containing the <italic>pfcrt</italic> coding sequence to create a C terminal PfCRT/GFP fusion. The catalytic domain of PfUT encompassing the amino acids 3653 to 3877 were cloned in frame with the coding sequences of the conditional aggregation domain <xref ref-type="bibr" rid="pgen.1004382-Saridaki1">[73]</xref> and GFP into <italic>Xho</italic>I and <italic>Avr</italic>II restricted pARL1a<sup>+</sup>. Parasites were transfected using 100 &#181;g plasmid DNA, and transfectants were selected using 5 nM WR99210, as previously described <xref ref-type="bibr" rid="pgen.1004382-Fidock2">[74]</xref>. Transfectants were detected in blood smears 14&#8211;21 days post transfection. Transfectants were grown in the presence of 5 nM WR99210 until two days before analysis when the drug pressure was removed to avoid interference of WR99210 with the drug accumulation assay or the growth inhibition assay. Live cell imaging of <italic>P. falciparum</italic>-infected erythrocytes were performed as described <xref ref-type="bibr" rid="pgen.1004382-Kuhn1">[69]</xref>.</p><p>Radiolabeled quinoline antimalarial drugs were obtained from the following vendors: [<sup>3</sup>H]-chloroquine (18.8 Ci/mmol), GE Healthcare; [<sup>3</sup>H]-quinine (20 Ci/mmol) and [<sup>3</sup>H]-quinidine (20 Ci/mmol), American Radiolabeled Chemicals, Inc.</p><p>The drug accumulation assay has been fully described <xref ref-type="bibr" rid="pgen.1004382-Sanchez3">[21]</xref>, <xref ref-type="bibr" rid="pgen.1004382-Sanchez6">[75]</xref>. Briefly, <italic>P. falciparum</italic>-infected erythrocytes were purified using a strong magnet (VarioMACS, Miltenyi Biotec), as described <xref ref-type="bibr" rid="pgen.1004382-Sanchez6">[75]</xref>. This yielded a purity of trophozoite-infected erythrocytes of 95&#8211;100% as determined by light microscopic examination of Giemsa-stained blood smears. The cells were resuspended in prewarmed RPMI 1640 containing 11 mM glucose, 25 mM HEPES, and 2 mM glutamine (pH 7.3 at 37&#176;C) at an haematocrit of 25000 cells/&#181;l. The haematocrit was determined using an automated cell counter (Z1-Coulter Particle Counter, Beckman Coulter Inc.). Cells were then incubated at 37&#176;C in the presence of 40 nM of the respective drug and the amount of label accumulated over time was monitored. The cellular drug accumulation ratio was determined as described <xref ref-type="bibr" rid="pgen.1004382-Sanchez6">[75]</xref>. Throughout the study, trophozoite-infected erythrocytes (20&#8211;28 hrs post invasion) were examined. The accumulation ratios to quinine, quinidine and chloroquine were determined in parallel assays.</p><p>Cell proliferation assays in the presence of different concentrations of chloroquine, quinine, and quinidine, were performed as described <xref ref-type="bibr" rid="pgen.1004382-Ferdig1">[15]</xref>. For the <italic>pfcrt</italic> and <italic>pfut</italic> HECT domain transfectant parasite lines, the IC<sub>50</sub> values to these drugs were determined in parallel assays over a period of four months. The quinine IC<sub>90</sub> values in the presence and absence of 0.89 &#181;M verapamil have been described for the HB3 x Dd2 cross <xref ref-type="bibr" rid="pgen.1004382-Ferdig1">[15]</xref>. The corresponding quinine IC<sub>50</sub> values for the HB3 x Dd2 cross were derived from these original data and are compiled in supplementary <xref ref-type="supplementary-material" rid="pgen.1004382.s007">Table S1</xref>. The quinine and chloroquine IC<sub>50</sub> values for the 50 field isolates and laboratory strains were taken from Mu et al. (2003) <xref ref-type="bibr" rid="pgen.1004382-Mu2">[43]</xref>.</p><p>QTL analysis was performed as described <xref ref-type="bibr" rid="pgen.1004382-Haley1">[76]</xref> and validated using R/QTL. The percent contribution of individual QTLs to the total variance in a response parameter was calculated using R/QTL. The genetic maps have been published <xref ref-type="bibr" rid="pgen.1004382-Sa1">[38]</xref>. The QTL analysis for the field strains was performed in a similar fashion. The quinine susceptibility of a strain was correlated with the presence or absence of a certain polymorphism, yielding the Pearson coefficient and from this, the probability, as the P value. The appropriate LOD score for a locus was computed as the logarithm of the probability P at that locus divided by the mean of the P values taken over all the loci. LOD scores above 2 (a P value one-hundredth that of the mean) were deemed significant.</p><p>Data were analysed using one or two way ANOVA test (Holm-Sidak test), or the Student's t-test, where appropriate. Statistical calculations were done using SigmaPlot 12.5.</p><p>Appropriate regions of the HB3 and Dd2 genomes were downloaded from the Broad Institute MIT database and analyzed using the BLAST algorithm to identify putative polymorphisms. Novel SNPs, indels, and microsatellite markers have been reported to Genbank.</p><p>Rabbit and mouse antisera were generated against the N-terminal (residues 473 to 712), the C-terminal domain of PfUT (residues 3654 to 3875) and against two combined peptides (DYNIKEDDESGSSN and LDDGVRPEKRKT). Immunofluorescence was carried out on magnet purified trophozoites <xref ref-type="bibr" rid="pgen.1004382-Sanchez5">[42]</xref> fixed with 4% EM-grade paraformaldehyde (EMS) and 0.0075% EM-grade glutaraldehyde (Merck) in PBS for 30 min <xref ref-type="bibr" rid="pgen.1004382-Tonkin1">[77]</xref>. Primary antisera were diluted as follows: rabbit &#945;-PfBiP 1&#8758;1000; rat &#945;-PfERD2 1&#8758;500; rabbit N-terminal PfUT 1&#8758;3000; rabbit C-terminal PfUT 1&#8758;3000; mouse N-terminal PfUT 1&#8758;2000. Corresponding secondary antibodies were used at a dilution of 1&#8758;1000. Slides were viewed using a LSM510 laser scanning confocal microscope (Carl Zeiss).</p><p>Immuno electron microscopy was performed as described <xref ref-type="bibr" rid="pgen.1004382-Cyrklaff1">[78]</xref>, using the rabbit antisera against the N-terminal domain of PfUT (1&#8758;100) coupled to 10 nm protein A colloidal gold.</p><p>For Western analysis, the following antisera were used: guinea pig anti-PfCRT (dilution 1&#8758;1000) <xref ref-type="bibr" rid="pgen.1004382-Nessler1">[79]</xref> and as secondary antibody donkey anti guinea pig antibodies conjugated with horseradish peroxidase (POD) (1&#8758;10000, Dianova); monoclonal mouse anti-GFP (dilution 1&#8758;1000, Roche Diagnostics) and as secondary antibody goat anti mouse POD (1&#8758;10000, Dianova); rabbit anti the C-terminal domain of PfUT (residues 3652 to 3875) (1&#8758;5000) and as secondary antibody goat anti rabbit POD (1&#8758;10000, Dianova); monoclonal-mouse anti-ubiquitin (1&#8758;2000, Santa Cruz Biotechnology) and as secondary antibody goat anti mouse POD (1&#8758;10000, Dianova). POD activity was detected using the BM Chemiluminescence Blotting Substrate Kit (Roche Diagnostics).</p><p>For the analysis of native proteins and protein complexes, magnet purified trophozoites were saponin-lysed (0.07% in PBS), washed two times in PBS, and subsequently solubilized using increasing concentrations of Triton X-100 ranging from 0.125% to 1% with mixing at 4&#176;C for 30 min. All solutions were supplemented with protease inhibitors (1 mM PMSF, 50 &#181;g/ml aprotinin, 20 &#181;g/ml leupeptin). Insoluble material was pelleted (14,000 g for 30 min at 4&#176;C), and supernatant fractions were collected. Samples were subsequently analyzed using a native blue gel as described <xref ref-type="bibr" rid="pgen.1004382-Bullen1">[80]</xref>.</p><p>For the analysis of membrane proteins, a Triton X-114 phase separation protocol was used <xref ref-type="bibr" rid="pgen.1004382-Ingmundson1">[81]</xref>. Briefly, saponin-lysed magnet purified trophozoites were incubated in a Triton X-114 buffer (1% Triton X-114, 150 mM NaCl, 10 mM Tris-Cl pH 7.4) for 3 min at 30&#176;C before centrifuged at 300 g. The supernatant was removed and subject to a second round of extraction (0.5% final concentration of Triton X-114). The lower detergent phase was analyzed by SDS-PAGE on a 3&#8211;8% gradient gel followed by Western analysis.</p><p>The DNA from approximately 2&#215;10<sup>8</sup> parasites was isolated using the DNeasy Blood &amp; Tissue Kit from Qiagen. Copies of pARL plasmids were subsequently determinate by quantitative real time PCR as described <xref ref-type="bibr" rid="pgen.1004382-Lee1">[82]</xref>, using primers to the bla gene. The reaction was performed using the LigthCycler DNA Master SYBR Green I reaction mix and the Biorad CFX96 Real-Time System. In parallel reactions, the amount of genomic DNA was determined by quantitative real time PCR, using primers to the <italic>P. falciparum</italic> &#946;-tubulin gene, as described <xref ref-type="bibr" rid="pgen.1004382-Price1">[83]</xref>. The normalized number of plasmid copies per haploid genome was then obtained. The high copy numbers reported in this study are consistent with episomal maintenance of the transfection vector, although occasional genomic integration cannot be excluded.</p><p>The GFP fusion proteins PfCRT/GFP and PfUT HECT domain/GFP were isolated from the respective transfectant Dd2 lines as described <xref ref-type="bibr" rid="pgen.1004382-Kuhn1">[69]</xref>. Briefly, following synchronisation with 5% sorbitol <xref ref-type="bibr" rid="pgen.1004382-Lambros1">[71]</xref>, infected erythrocytes at the trophozoite stage were purified from 600 ml parasite culture (5% haematocrit and 8% parasitemia) using the Super Macs magnet (Miltenyi Biotech) and the D column. Infected erythrocytes were subsequently treated with 0.07% saponin in PBS. Immunoprecipitation was performed as described <xref ref-type="bibr" rid="pgen.1004382-Saridaki1">[73]</xref>. Briefly, proteins were extracted using the RIPA buffer for PfCRT (1% NP-40, 1% sodium deoxycholate, 0.1% SDS, 150 mM NaCl, 10 mM Na-Phosphate buffer pH 7, 1 mM EDTA) and an TNE buffer for PfUT (20 mM Tris-HCl pH 7.5, 137 mM NaCl, 1% NP40, 2 mM EDTA, plus protease inhibitors: leupeptin 20 &#181;g/ml, aprotinin 50 &#181;g/ml, PMSF 1 mM). The lysate was then diluted with 9 volumes of NETT buffer (10 mM Na-Phosphate buffer pH 7, 150 mM NaCl, 0.1% NP-40, and 1 mM EDTA). Prior to immuno-precipitation, lysates were cleared using goat IgGs. For immunoprecipitation, the ProFound&#8482; Co-Immunoprecipitation Kit (Pierce) was used according to the manufacturer's instructions, using goat anti-GFP antiserum (Rockland). Immuno-precipitated material was washed in buffers with increasing NaCl concentrations ranging from 250 to 350 mM and one additional wash with 50 mM Tris HCl pH 7.5. All buffers used were supplemented with a cocktail of protease and phosphatase inhibitors (50 &#181;g/ml aprotinin, 20 &#181;g/ml leupeptin, 1 mM PMSF).</p><p>
<italic>In vitro</italic> ubiqitination assays were performed as described <xref ref-type="bibr" rid="pgen.1004382-Agrawal1">[50]</xref> and examined by Western analysis. The following reagents purchased from Boston Biochem were used in the ubiquitination assay: 70 &#181;M human recombinant ubiquitin, 200 nM human recombinant ubiquitin activating enzyme UBA, 5 &#181;M human recombinant ubiquitin conjugating enzyme UbcH5a or UbcH13, and 1 x energy regeneration solution. The reactions were size-fractionated by non-reducing SDS PAGE on a 4&#8211;12% gradient gel, transferred to polyvinylidene difluoride membranes, and analyzed using the antisera indicated (see also above).</p>