<p>This investigation included six studies (the Framingham Heart Study (FHS), the Estonian Biobank (EGCUT), the Rotterdam Study (RS) [<xref ref-type="bibr" rid="pgen.1005035.ref008">8</xref>], the InCHIANTI Study, the Cooperative Health Research in the Region of Augsburg (KORA F4) Study [<xref ref-type="bibr" rid="pgen.1005035.ref009">9</xref>], and the Study of Health in Pomerania (SHIP-TREND) [<xref ref-type="bibr" rid="pgen.1005035.ref010">10</xref>], each of which conducted genome-wide genotyping, mRNA expression profiling, and had extensive BP phenotype data. Each of the six studies followed the recommendations of the Declaration of Helsinki. The FHS: Systems Approach to Biomarker Research (SABRe) in cardiovascular disease is approved under the Boston University Medical Center&#8217;s protocol H-27984. Ethical approval of EGCUT was granted by the Research Ethics Committee of the University of Tartu (UT REC). Ethical approval of the InCHIANTI study was granted by the Instituto Nazionale Riposo e Cura Anziani institutional review board in Italy. Ethical approval of RS was granted by the medical ethics committee of the Erasmus Medical Center. The study protocol of SHIP-TREND was approved by the medical ethics committee of the University of Greifswald. KORA F4 is a population-based survey in the region of Augsburg in Southern Germany which was performed between 2006 and 2008. KORA F4 was approved by the local ethical committees. Informed consent was obtained from each study participant.</p><p>Hypertension (HTN) was defined as SBP &#8805;140 mm Hg or DBP &#8805;90 mm Hg. We excluded individuals receiving anti-hypertensive treatment because of the possibility that some of the differentially expressed genes we identified would reflect treatment effects. The eligible study sample included 7017 individuals: 3679 from FHS, 972 from EGCUT, 604 from RS, 597 from InCHIANTI, 565 from KORA F4, and 600 from SHIP-TREND.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">RNA was isolated from whole blood samples that were collected in PaxGene tubes (PreAnalytiX, Hombrechtikon, Switzerland) in FHS, RS, InCHIANTI, KORA F4 and SHIP-TREND, and in Blood RNA Tubes (Life Technologies, NY, USA) in EGCUT. Gene expression in the FHS samples used the Affymetrix Exon Array ST 1.0. EGCUT, RS, InCHANTI, KORA F4, and SHIP-TREND used the Illumina HT12v3 (EGCUT, InCHANTI, KORA F4, and SHIP-TREND) or HT12v4 (RS) array. Raw data from gene expression profiling are available online (FHS [<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/gap">http://www.ncbi.nlm.nih.gov/gap</ext-link>; accession number phs000007], EGCUT [GSE48348], RS [GSE33828], InCHIANTI [GSE48152], KORA F4 [E-MTAB-1708] and SHIP-TREND [GSE36382]). The details of sample collection, microarrays, and data processing and normalization in each cohort are provided in the <bold><xref ref-type="supplementary-material" rid="pgen.1005035.s011">S2 Text</xref></bold>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The association of gene expression with BP was analyzed separately in each of the six studies (<xref ref-type="disp-formula" rid="pgen.1005035.e001">Equation 1</xref>). A linear mixed model was used in the FHS in order to account for family structure. Linear regression models were used in the other five studies. In each study, gene expression level, denoted by geneExp, was included as the dependent variable, and explanatory variables included blood pressure phenotypes (SBP, DBP, and HTN), and covariates included age, sex, body mass index (BMI), cell counts, and technical covariates. A separate regression model was fitted for each gene. The general formula is shown below, and the details of analyses for each study are provided in the <bold><xref ref-type="supplementary-material" rid="pgen.1005035.s005">S2 Text</xref></bold> and <bold><xref ref-type="supplementary-material" rid="pgen.1005035.s009">S6 Table</xref></bold>.
<disp-formula id="pgen.1005035.e001"><alternatives><graphic id="pgen.1005035.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e001.jpg" /><ns1:math id="M1"><ns1:mrow><ns1:mi mathvariant="italic">geneExp</ns1:mi><ns1:mo>=</ns1:mo><ns1:mi>B</ns1:mi><ns1:mi>P</ns1:mi><ns1:mo>+</ns1:mo><ns1:mrow><ns1:munderover><ns1:mo stretchy="true">&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>m</ns1:mi></ns1:munderover></ns1:mrow><ns1:mi mathvariant="italic">covariates</ns1:mi></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p>The overall analysis framework is provided in <bold><xref ref-type="supplementary-material" rid="pgen.1005035.s001">S1 Fig.</xref></bold>. We first identified differentially expressed genes associated with BP (BP signature genes) in the FHS samples (Set 1) and attempted replication in the meta-analysis results from the Illumina cohorts (Set 2, see <xref ref-type="sec" rid="sec009">Methods</xref>, <xref ref-type="sec" rid="sec013">Meta-analysis</xref>). We next identified BP signature genes in the Illumina cohorts (Set 2), and then attempted replication in the FHS samples (Set 1). The significance threshold for pre-selecting BP signature genes in discovery was at Bonferroni corrected <italic>p</italic> = 0.05 (in FHS, corrected for 17,318 measured genes [17,873 transcripts], and in illumina cohorts, corrected for 12,010 measured genes [14,222 transcripts] that passed quality control). Replication was established at Bonferroni corrected <italic>p</italic> = 0.05, correcting for the number of pre-selected BP signatures genes in the discovery set. We computed the <italic>pi1</italic> value to estimate the enrichment of significant <italic>p</italic> values in the replication set (the Illumina cohorts) for BP signatures identified in the discovery set (the FHS) by utilizing the R package <italic>Qvalue</italic> [<xref ref-type="bibr" rid="pgen.1005035.ref011">11</xref>]. <italic>Pi1</italic> is defined as <italic>1-pi0</italic>. <italic>Pi0</italic> value provided by the <italic>Qvalue</italic> package, represents overall probability that the null hypothesis is true. Therefore, <italic>pi1</italic> value represents the proportion of significant results. For genes passing Bonferroni corrected <italic>p</italic>&lt;0.05 in the discovery set for SBP, DBP and HTN, we calculated <italic>pi1</italic> values for each gene set in the replication set.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We performed meta-analysis of the five Illumina cohorts (for discovery and replication purposes), and then performed meta-analysis of all six cohorts. An inverse variance weighted meta-analysis was conducted using fixed-effects or random-effects models by the <italic>metagen()</italic> function in the R package Meta (<ext-link ext-link-type="uri" ns0:href="http://cran.r-project.org/web/packages/meta/index.html">http://cran.r-project.org/web/packages/meta/index.html</ext-link>). At first, we tested heterogeneity for each gene using Cochran&#8217;s Q statistic. If the heterogeneity p value is significant (<italic>p</italic>&lt;0.05), we will use a random-effects model for the meta-analysis, otherwise use a fixed-effects model. The Benjamini-Hochberg (BH) method [<xref ref-type="bibr" rid="pgen.1005035.ref028">28</xref>] was used to calculate FDR for differentially expressed genes in relation to BP following the meta-analysis of all six cohorts. We also used a more stringent threshold to define BP signature genes by utilizing <italic>p</italic>&lt;6.5e-6 (Bonferroni correction for 7717 unique genes [7810 transcript] based on the overlap of FHS and illumina cohort interrogated gene sets).</p><p>To estimate the proportion of variances in SBP or DBP explained by a group of differentially expressed BP signature genes (gene 1, gene 2, &#8230;, gene n), we used the following two models:</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Full model:
<disp-formula id="pgen.1005035.e002"><alternatives><graphic id="pgen.1005035.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e002.jpg" /><ns1:math id="M2"><ns1:mrow><ns1:mi>B</ns1:mi><ns1:mi>P</ns1:mi><ns1:mo>=</ns1:mo><ns1:munderover><ns1:mo stretchy="true">&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>n</ns1:mi></ns1:munderover><ns1:mi mathvariant="italic">gene</ns1:mi><ns1:mspace width="thickmathspace" /><ns1:mi mathvariant="italic">i</ns1:mi><ns1:mo>+</ns1:mo><ns1:munderover><ns1:mo stretchy="true">&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>m</ns1:mi></ns1:munderover><ns1:mi mathvariant="italic">covariates</ns1:mi></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Null model:
<disp-formula id="pgen.1005035.e003"><alternatives><graphic id="pgen.1005035.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e003.jpg" /><ns1:math id="M3"><ns1:mrow><ns1:mi>B</ns1:mi><ns1:mi>P</ns1:mi><ns1:mo>=</ns1:mo><ns1:munderover><ns1:mo stretchy="true">&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>m</ns1:mi></ns1:munderover><ns1:mi mathvariant="italic">covariates</ns1:mi></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The proportion of variance in BP attributable to the group of differentially expressed BP signature genes (<inline-formula id="pgen.1005035.e004"><alternatives><graphic id="pgen.1005035.e004g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e004.jpg" /><ns1:math id="M4"><ns1:msubsup><ns1:mrow><ns1:mi>h</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>B</ns1:mi><ns1:mi>P</ns1:mi><ns1:mo>_</ns1:mo><ns1:mi>s</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>g</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula>) was calculated as:
<disp-formula id="pgen.1005035.e005"><alternatives><graphic id="pgen.1005035.e005g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e005.jpg" /><ns1:math id="M5"><ns1:mrow><ns1:msubsup><ns1:mrow><ns1:mi>h</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>B</ns1:mi><ns1:mi>P</ns1:mi><ns1:mo>_</ns1:mo><ns1:mi>s</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>g</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>=</ns1:mo><ns1:mtext>max</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>G</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>r</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>&#8722;</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>G</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>f</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>&#8722;</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>r</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>f</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:mrow><ns1:mrow><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>B</ns1:mi><ns1:mi>P</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:mrow></ns1:mfrac><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
where <inline-formula id="pgen.1005035.e006"><alternatives><graphic id="pgen.1005035.e006g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e006.jpg" /><ns1:math id="M6"><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>B</ns1:mi><ns1:mi>P</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> is the total phenotypic variance of SBP or DBP, <inline-formula id="pgen.1005035.e007"><alternatives><graphic id="pgen.1005035.e007g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e007.jpg" /><ns1:math id="M7"><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>G</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>f</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> and <inline-formula id="pgen.1005035.e008"><alternatives><graphic id="pgen.1005035.e008g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e008.jpg" /><ns1:math id="M8"><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>r</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>f</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> are the variance and error variance when modeling with the tested group of gene expression traits (gene 1, gene 2, &#8230;, gene n), and <inline-formula id="pgen.1005035.e009"><alternatives><graphic id="pgen.1005035.e009g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e009.jpg" /><ns1:math id="M9"><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>G</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> and <inline-formula id="pgen.1005035.e010"><alternatives><graphic id="pgen.1005035.e010g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e010.jpg" /><ns1:math id="M10"><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>r</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> are the variance and error variance when modeling without the tested group of gene expression traits.</p><p>The proportion of the variance in BP phenotypes attributable to the FHS BP signature genes was estimated in the five Illumina cohorts, respectively, and then the average proportion values were reported. In turn, the proportion of the variance in BP phenotypes attributable to the Illumina BP signature genes was estimated in the FHS.</p><p>SNPs associated with altered gene expression (i.e. eQTLs) were identified using genome-wide genotype and gene expression data in all available FHS samples (n = 5257) at FDR&lt;0.1 (Joehanes R, submitted, 2014, and a brief summary of methods and results are provided in the <bold><xref ref-type="supplementary-material" rid="pgen.1005035.s011">S2 Text</xref></bold>). A <italic>cis-</italic>eQTL was defined as an eQTL within 1 megabase (MB) flanking the gene. Other eQTLs were defined as <italic>trans-</italic>eQTLs. We combined the eQTL list generated in the FHS with the eQTLs generated by meta-analysis of seven other studies (n = 5300) that were also based on whole blood expression[<xref ref-type="bibr" rid="pgen.1005035.ref012">12</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">For every BP signature gene, we estimated the proportion of variance in the transcript attributable to the corresponding <italic>cis-</italic> or <italic>trans-</italic>eQTLs (<inline-formula id="pgen.1005035.e011"><alternatives><graphic id="pgen.1005035.e011g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e011.jpg" /><ns1:math id="M11"><ns1:msubsup><ns1:mrow><ns1:mi>h</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>Q</ns1:mi><ns1:mi>T</ns1:mi><ns1:mi>L</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula>) using the formula:
<disp-formula id="pgen.1005035.e012"><alternatives><graphic id="pgen.1005035.e012g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e012.jpg" /><ns1:math id="M12"><ns1:mrow><ns1:msubsup><ns1:mrow><ns1:mi>h</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>Q</ns1:mi><ns1:mi>T</ns1:mi><ns1:mi>L</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>=</ns1:mo><ns1:mi>m</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>x</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>Q</ns1:mi><ns1:mi>T</ns1:mi><ns1:mi>L</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>r</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>&#8722;</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>Q</ns1:mi><ns1:mi>T</ns1:mi><ns1:mi>L</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>f</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>&#8722;</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>r</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>f</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:mrow><ns1:mrow><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:mrow></ns1:mfrac><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
where <inline-formula id="pgen.1005035.e013"><alternatives><graphic id="pgen.1005035.e013g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e013.jpg" /><ns1:math id="M13"><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> was the total phenotypic variance of a gene expression trait; <inline-formula id="pgen.1005035.e014"><alternatives><graphic id="pgen.1005035.e014g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e014.jpg" /><ns1:math id="M14"><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>Q</ns1:mi><ns1:mi>T</ns1:mi><ns1:mi>L</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>f</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> and <inline-formula id="pgen.1005035.e015"><alternatives><graphic id="pgen.1005035.e015g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e015.jpg" /><ns1:math id="M15"><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>r</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>f</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> were the variance and the residual error, respectively, when modeling with the tested eQTL; <inline-formula id="pgen.1005035.e016"><alternatives><graphic id="pgen.1005035.e016g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e016.jpg" /><ns1:math id="M16"><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>Q</ns1:mi><ns1:mi>T</ns1:mi><ns1:mi>L</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> and <inline-formula id="pgen.1005035.e017"><alternatives><graphic id="pgen.1005035.e017g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005035.e017.jpg" /><ns1:math id="M17"><ns1:msubsup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>r</ns1:mi><ns1:mo>.</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> were the variance and the residual error when modeling without the tested eQTL.</p><p>In order to understand the biological themes within the global gene expression changes in relation to BP, we performed gene set enrichment analysis[<xref ref-type="bibr" rid="pgen.1005035.ref029">29</xref>] to test for enrichment of any gene ontology (GO) biology process[<xref ref-type="bibr" rid="pgen.1005035.ref030">30</xref>] or KEGG pathways[<xref ref-type="bibr" rid="pgen.1005035.ref031">31</xref>]. &#8220;Metric for ranking gene&#8221; parameters were configured to the beta value of the meta-analysis, in order to look at the top enriched functions for BP associated up-regulated and down-regulated gene expression changes respectively. One thousand random permutations were conducted and the significance level was set at FDR&#8804; 0.25 to allow for exploratory discovery [<xref ref-type="bibr" rid="pgen.1005035.ref029">29</xref>].</p><p>Steering Committee (alphabetical)</p><p>Gon&#231;alo Abecasis, Murielle Bochud, Mark Caulfield (co-chair), Aravinda Chakravarti, Dan Chasman, Georg Ehret (co-chair), Paul Elliott, Andrew Johnson, Louise Wain, Martin Larson, Daniel Levy (co-chair), Patricia Munroe (co-chair), Christopher Newton-Cheh (co-chair), Paul O'Reilly, Walter Palmas, Bruce Psaty, Kenneth Rice, Albert Smith, Harold Snider, Martin Tobin, Cornelia Van Duijn, Germaine Verwoert.</p><p>Members</p><p>Georg B. Ehret<sup>1,2,3</sup>, Patricia B. Munroe<sup>4</sup>, Kenneth M. Rice<sup>5</sup>, Murielle Bochud<sup>2</sup>, Andrew D. Johnson<sup>6,7</sup>, Daniel I. Chasman<sup>8,9</sup>, Albert V. Smith<sup>10,11</sup>, Martin D. Tobin<sup>12</sup>, Germaine C. Verwoert<sup>13,14,15</sup>, Shih-Jen Hwang<sup>6,16,7</sup>, Vasyl Pihur<sup>1</sup>, Peter Vollenweider<sup>17</sup>, Paul F. O'Reilly<sup>18</sup>, Najaf Amin<sup>13</sup>, Jennifer L Bragg-Gresham<sup>19</sup>, Alexander Teumer<sup>20</sup>, Nicole L. Glazer<sup>21</sup>, Lenore Launer<sup>22</sup>, Jing Hua Zhao<sup>23</sup>, Yurii Aulchenko<sup>13</sup>, Simon Heath<sup>24</sup>, Siim S&#245;ber<sup>25</sup>, Afshin Parsa<sup>26</sup>, Jian'an Luan<sup>23</sup>, Pankaj Arora<sup>27</sup>, Abbas Dehghan<sup>13,14,15</sup>, Feng Zhang<sup>28</sup>, Gavin Lucas<sup>29</sup>, Andrew A. Hicks<sup>30</sup>, Anne U. Jackson<sup>31</sup>, John F Peden<sup>32</sup>, Toshiko Tanaka<sup>33</sup>, Sarah H. Wild<sup>34</sup>, Igor Rudan<sup>35,36</sup>, Wilmar Igl<sup>37</sup>, Yuri Milaneschi<sup>33</sup>, Alex N. Parker<sup>38</sup>, Cristiano Fava<sup>39,40</sup>, John C. Chambers<sup>18,41</sup>, Ervin R. Fox<sup>42</sup>, Meena Kumari<sup>43</sup>, Min Jin Go<sup>44</sup>, Pim van der Harst<sup>45</sup>, Wen Hong Linda Kao<sup>46</sup>, Marketa Sj&#246;gren<sup>39</sup>, D. G. Vinay<sup>47</sup>, Myriam Alexander<sup>48</sup>, Yasuharu Tabara<sup>49</sup>, Sue Shaw-Hawkins<sup>4</sup>, Peter H. Whincup<sup>50</sup>, Yongmei Liu<sup>51</sup>, Gang Shi<sup>52</sup>, Johanna Kuusisto<sup>53</sup>, Bamidele Tayo<sup>54</sup>, Mark Seielstad<sup>55,56</sup>, Xueling Sim<sup>57</sup>, Khanh-Dung Hoang Nguyen<sup>1</sup>, Terho Lehtim&#228;ki<sup>58</sup>, Giuseppe Matullo<sup>59,60</sup>, Ying Wu<sup>61</sup>, Tom R. Gaunt<sup>62</sup>, N. Charlotte Onland-Moret<sup>63,64</sup>, Matthew N. Cooper<sup>65</sup>, Carl G.P. Platou<sup>66</sup>, Elin Org<sup>25</sup>, Rebecca Hardy<sup>67</sup>, Santosh Dahgam<sup>68</sup>, Jutta Palmen<sup>69</sup>, Veronique Vitart<sup>70</sup>, Peter S. Braund<sup>71,72</sup>, Tatiana Kuznetsova<sup>73</sup>, Cuno S.P.M. Uiterwaal<sup>63</sup>, Adebowale Adeyemo<sup>74</sup>, Walter Palmas<sup>75</sup>, Harry Campbell<sup>35</sup>, Barbara Ludwig<sup>76</sup>, Maciej Tomaszewski<sup>71,72</sup>, Ioanna Tzoulaki<sup>77,78</sup>, Nicholette D. Palmer<sup>79</sup>, CARDIoGRAM consortium<sup>80</sup>, CKDGen Consortium<sup>80</sup>, KidneyGen Consortium<sup>80</sup>, EchoGen consortium<sup>80</sup>, CHARGE-HF consortium<sup>80</sup>, Thor Aspelund<sup>10,11</sup>, Melissa Garcia<sup>22</sup>, Yen-Pei C. Chang<sup>26</sup>, Jeffrey R. O'Connell<sup>26</sup>, Nanette I. Steinle<sup>26</sup>, Diederick E. Grobbee<sup>63</sup>, Dan E. Arking<sup>1</sup>, Sharon L. Kardia<sup>81</sup>, Alanna C. Morrison<sup>82</sup>, Dena Hernandez<sup>83</sup>, Samer Najjar<sup>84,85</sup>, Wendy L. McArdle<sup>86</sup>, David Hadley<sup>50,87</sup>, Morris J. Brown<sup>88</sup>, John M. Connell<sup>89</sup>, Aroon D. Hingorani<sup>90</sup>, Ian N.M. Day<sup>62</sup>, Debbie A. Lawlor<sup>62</sup>, John P. Beilby<sup>91,92</sup>, Robert W. Lawrence<sup>65</sup>, Robert Clarke<sup>93</sup>, Rory Collins<sup>93</sup>, Jemma C Hopewell<sup>93</sup>, Halit Ongen<sup>32</sup>, Albert W. Dreisbach<sup>42</sup>, Yali Li<sup>94</sup>, J H. Young<sup>95</sup>, Joshua C. Bis<sup>21</sup>, Mika K&#228;h&#246;nen<sup>96</sup>, Jorma Viikari<sup>97</sup>, Linda S. Adair<sup>98</sup>, Nanette R. Lee<sup>99</sup>, Ming-Huei Chen<sup>100</sup>, Matthias Olden<sup>101,102</sup>, Cristian Pattaro<sup>30</sup>, Judith A. Hoffman Bolton<sup>103</sup>, Anna K&#246;ttgen<sup>104,103</sup>, Sven Bergmann<sup>105,106</sup>, Vincent Mooser<sup>107</sup>, Nish Chaturvedi<sup>108</sup>, Timothy M. Frayling<sup>109</sup>, Muhammad Islam<sup>110</sup>, Tazeen H. Jafar<sup>110</sup>, Jeanette Erdmann<sup>111</sup>, Smita R. Kulkarni<sup>112</sup>, Stefan R. Bornstein<sup>76</sup>, J&#252;rgen Gr&#228;ssler<sup>76</sup>, Leif Groop<sup>113,114</sup>, Benjamin F. Voight<sup>115</sup>, Johannes Kettunen<sup>116,126</sup>, Philip Howard<sup>117</sup>, Andrew Taylor<sup>43</sup>, Simonetta Guarrera<sup>60</sup>, Fulvio Ricceri<sup>59,60</sup>, Valur Emilsson<sup>118</sup>, Andrew Plump<sup>118</sup>, In&#234;s Barroso<sup>119,120</sup>, Kay-Tee Khaw<sup>48</sup>, Alan B. Weder<sup>121</sup>, Steven C. Hunt<sup>122</sup>, Yan V. Sun<sup>81</sup>, Richard N. Bergman<sup>123</sup>, Francis S. Collins<sup>124</sup>, Lori L. Bonnycastle<sup>124</sup>, Laura J. Scott<sup>31</sup>, Heather M. Stringham<sup>31</sup>, Leena Peltonen<sup>119,125,126,127</sup>, Markus Perola<sup>125</sup>, Erkki Vartiainen<sup>125</sup>, Stefan-Martin Brand<sup>128,129</sup>, Jan A. Staessen<sup>73</sup>, Thomas J. Wang<sup>6,130</sup>, Paul R. Burton<sup>12,72</sup>, Maria Soler Artigas<sup>12</sup>, Yanbin Dong<sup>131</sup>, Harold Snieder<sup>132,131</sup>, Xiaoling Wang<sup>131</sup>, Haidong Zhu<sup>131</sup>, Kurt K. Lohman<sup>133</sup>, Megan E. Rudock<sup>51</sup>, Susan R Heckbert<sup>134,135</sup>, Nicholas L Smith<sup>134,136,135</sup>, Kerri L Wiggins<sup>137</sup>, Ayo Doumatey<sup>74</sup>, Daniel Shriner<sup>74</sup>, Gudrun Veldre<sup>25,138</sup>, Margus Viigimaa<sup>139,140</sup>, Sanjay Kinra<sup>141</sup>, Dorairajan Prabhakaran<sup>142</sup>, Vikal Tripathy<sup>142</sup>, Carl D. Langefeld<sup>79</sup>, Annika Rosengren<sup>143</sup>, Dag S. Thelle<sup>144</sup>, Anna Maria Corsi<sup>145</sup>, Andrew Singleton<sup>83</sup>, Terrence Forrester<sup>146</sup>, Gina Hilton<sup>1</sup>, Colin A. McKenzie<sup>146</sup>, Tunde Salako<sup>147</sup>, Naoharu Iwai<sup>148</sup>, Yoshikuni Kita<sup>149</sup>, Toshio Ogihara<sup>150</sup>, Takayoshi Ohkubo<sup>149,151</sup>, Tomonori Okamura<sup>148</sup>, Hirotsugu Ueshima<sup>152</sup>, Satoshi Umemura<sup>153</sup>, Susana Eyheramendy<sup>154</sup>, Thomas Meitinger<sup>155,156</sup>, H.-Erich Wichmann<sup>157,158,159</sup>, Yoon Shin Cho<sup>44</sup>, Hyung-Lae Kim<sup>44</sup>, Jong-Young Lee<sup>44</sup>, James Scott<sup>160</sup>, Joban S. Sehmi<sup>160,41</sup>, Weihua Zhang<sup>18</sup>, Bo Hedblad<sup>39</sup>, Peter Nilsson<sup>39</sup>, George Davey Smith<sup>62</sup>, Andrew Wong<sup>67</sup>, Narisu Narisu<sup>124</sup>, Alena Stan&#269;&#225;kov&#225;<sup>53</sup>, Leslie J. Raffel<sup>161</sup>, Jie Yao<sup>161</sup>, Sekar Kathiresan<sup>162,27</sup>, Chris O'Donnell<sup>163,27,9</sup>, Stephen M. Schwartz<sup>134</sup>, M. Arfan Ikram<sup>13,15</sup>, W. T. Longstreth Jr.<sup>164</sup>, Thomas H. Mosley<sup>165</sup>, Sudha Seshadri<sup>166</sup>, Nick R.G. Shrine<sup>12</sup>, Louise V. Wain<sup>12</sup>, Mario A. Morken<sup>124</sup>, Amy J. Swift<sup>124</sup>, Jaana Laitinen<sup>167</sup>, Inga Prokopenko<sup>51,168</sup>, Paavo Zitting<sup>169</sup>, Jackie A. Cooper<sup>69</sup>, Steve E. Humphries<sup>69</sup>, John Danesh<sup>48</sup>, Asif Rasheed<sup>170</sup>, Anuj Goel<sup>32</sup>, Anders Hamsten<sup>171</sup>, Hugh Watkins<sup>32</sup>, Stephan J.L. Bakker<sup>172</sup>, Wiek H. van Gilst<sup>45</sup>, Charles S. Janipalli<sup>47</sup>, K. Radha Mani<sup>47</sup>, Chittaranjan S. Yajnik<sup>112</sup>, Albert Hofman<sup>13</sup>, Francesco U.S. Mattace-Raso<sup>13,14</sup>, Ben A. Oostra<sup>173</sup>, Ayse Demirkan<sup>13</sup>, Aaron Isaacs<sup>13</sup>, Fernando Rivadeneira<sup>13,14</sup>, Edward G Lakatta<sup>174</sup>, Marco Orru<sup>175,176</sup>, Angelo Scuteri<sup>174</sup>, Mika Ala-Korpela<sup>177,178,179</sup>, Antti J Kangas<sup>177</sup>, Leo-Pekka Lyytik&#228;inen<sup>58</sup>, Pasi Soininen<sup>177,178</sup>, Taru Tukiainen<sup>180,181,177</sup>, Peter W&#252;rtz<sup>177,18,180</sup>, Rick Twee-Hee Ong<sup>56,57,182</sup>, Marcus D&#246;rr<sup>183</sup>, Heyo K. Kroemer<sup>184</sup>, Uwe V&#246;lker<sup>20</sup>, Henry V&#246;lzke<sup>185</sup>, Pilar Galan<sup>186</sup>, Serge Hercberg<sup>186</sup>, Mark Lathrop<sup>24</sup>, Diana Zelenika<sup>24</sup>, Panos Deloukas<sup>119</sup>, Massimo Mangino<sup>28</sup>, Tim D. Spector<sup>28</sup>, Guangju Zhai<sup>28</sup>, James F. Meschia<sup>187</sup>, Michael A. Nalls<sup>83</sup>, Pankaj Sharma<sup>188</sup>, Janos Terzic<sup>189</sup>, M. J. Kranthi Kumar<sup>47</sup>, Matthew Denniff<sup>71</sup>, Ewa Zukowska-Szczechowska<sup>190</sup>, Lynne E. Wagenknecht<sup>79</sup>, F. Gerald R. Fowkes<sup>191</sup>, Fadi J. Charchar<sup>192</sup>, Peter E.H. Schwarz<sup>193</sup>, Caroline Hayward<sup>70</sup>, Xiuqing Guo<sup>161</sup>, Charles Rotimi<sup>74</sup>, Michiel L. Bots<sup>63</sup>, Eva Brand<sup>194</sup>, Nilesh J. Samani<sup>71,72</sup>, Ozren Polasek<sup>195</sup>, Philippa J. Talmud<sup>69</sup>, Fredrik Nyberg<sup>68,196</sup>, Diana Kuh<sup>67</sup>, Maris Laan<sup>25</sup>, Kristian Hveem<sup>66</sup>, Lyle J. Palmer<sup>197,198</sup>, Yvonne T. van der Schouw<sup>63</sup>, Juan P. Casas<sup>199</sup>, Karen L. Mohlke<sup>61</sup>, Paolo Vineis<sup>200,60</sup>, Olli Raitakari<sup>201</sup>, Santhi K. Ganesh<sup>202</sup>, Tien Y. Wong<sup>203,204</sup>, E Shyong Tai<sup>205,57,206</sup>, Richard S. Cooper<sup>54</sup>, Markku Laakso<sup>53</sup>, Dabeeru C. Rao<sup>207</sup>, Tamara B. Harris<sup>22</sup>, Richard W. Morris<sup>208</sup>, Anna F. Dominiczak<sup>209</sup>, Mika Kivimaki<sup>210</sup>, Michael G. Marmot<sup>210</sup>, Tetsuro Miki<sup>49</sup>, Danish Saleheen<sup>170,48</sup>, Giriraj R. Chandak<sup>47</sup>, Josef Coresh<sup>211</sup>, Gerjan Navis<sup>212</sup>, Veikko Salomaa<sup>125</sup>, Bok-Ghee Han<sup>44</sup>, Xiaofeng Zhu<sup>94</sup>, Jaspal S. Kooner<sup>160,41</sup>, Olle Melander<sup>39</sup>, Paul M Ridker<sup>8,213,9</sup>, Stefania Bandinelli<sup>214</sup>, Ulf B. Gyllensten<sup>37</sup>, Alan F. Wright<sup>70</sup>, James F. Wilson<sup>34</sup>, Luigi Ferrucci<sup>33</sup>, Martin Farrall<sup>32</sup>, Jaakko Tuomilehto<sup>215,216,217,218</sup>, Peter P. Pramstaller<sup>30,219</sup>, Roberto Elosua<sup>29,220</sup>, Nicole Soranzo<sup>119,28</sup>, Eric J.G. Sijbrands<sup>13,14</sup>, David Altshuler<sup>221,115</sup>, Ruth J.F. Loos<sup>23</sup>, Alan R. Shuldiner<sup>26,222</sup>, Christian Gieger<sup>157</sup>, Pierre Meneton<sup>223</sup>, Andre G. Uitterlinden<sup>13,14,15</sup>, Nicholas J. Wareham<sup>23</sup>, Vilmundur Gudnason<sup>10,11</sup>, Jerome I. Rotter<sup>161</sup>, Rainer Rettig<sup>224</sup>, Manuela Uda<sup>175</sup>, David P. Strachan<sup>50</sup>, Jacqueline C.M. Witteman<sup>13,15</sup>, Anna-Liisa Hartikainen<sup>225</sup>, Jacques S. Beckmann<sup>105,226</sup>, Eric Boerwinkle<sup>227</sup>, Ramachandran S. Vasan<sup>6,228</sup>, Michael Boehnke<sup>31</sup>, Martin G. Larson<sup>6,229</sup>, Marjo-Riitta J&#228;rvelin<sup>18,230,231,232,233</sup>, Bruce M. Psaty<sup>21,135*</sup>, Gon&#231;alo R Abecasis<sup>19*</sup>, Aravinda Chakravarti<sup>1</sup>, Paul Elliott<sup>18,233*</sup>, Cornelia M. van Duijn<sup>13,234*</sup>, Christopher Newton-Cheh<sup>27,115</sup>, Daniel Levy<sup>6,16,7</sup>, Mark J. Caulfield<sup>4</sup>, Toby Johnson<sup>4</sup>
</p><p>Affiliations</p><p>Center for Complex Disease Genomics, McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA</p><p>Institute of Social and Preventive Medicine (IUMSP), Centre Hospitalier Universitaire Vaudois and University of Lausanne, Bugnon 17, 1005 Lausanne, Switzerland</p><p>Cardiology, Department of Specialties of Internal Medicine, Geneva University Hospital, Rue Gabrielle-Perret-Gentil 4, 1211 Geneva 14, Switzerland</p><p>Clinical Pharmacology and The Genome Centre, William Harvey Research Institute, Barts and The London School of Medicine and Dentistry, Queen Mary University of London, London EC1M 6BQ, UK</p><p>Department of Biostatistics, University of Washington, Seattle, WA, USA</p><p>Framingham Heart Study, Framingham, MA, USA</p><p>National Heart Lung, and Blood Institute, Bethesda, MD, USA</p><p>Division of Preventive Medicine, Brigham and Women's Hospital, 900 Commonwealth Avenue East, Boston MA 02215, USA</p><p>Harvard Medical School, Boston, MA, USA</p><p>Icelandic Heart Association, Kopavogur, Iceland</p><p>University of Iceland, Reykajvik, Iceland</p><p>Department of Health Sciences, University of Leicester, University Rd, Leicester LE1 7RH, UK</p><p>Department of Epidemiology, Erasmus Medical Center, PO Box 2040, 3000 CA, Rotterdam, The Netherlands</p><p>Department of Internal Medicine, Erasmus Medical Center, Rotterdam, The Netherlands</p><p>Netherlands Consortium for Healthy Aging (NCHA), Netherland Genome Initiative (NGI), The Netherlands</p><p>Center for Population Studies, National Heart Lung, and Blood Institute, Bethesda, MD, USA</p><p>Department of Internal Medicine, Centre Hospitalier Universitaire Vaudois, 1011 Lausanne, Switzerland</p><p>Department of Epidemiology and Biostatistics, School of Public Health, Imperial College London, Norfolk Place, London W2 1PG, UK</p><p>Center for Statistical Genetics, Department of Biostatistics, University of Michigan School of Public Health, Ann Arbor, MI 48103, USA</p><p>Interfaculty Institute for Genetics and Functional Genomics, Ernst-Moritz-Arndt-University Greifswald, 17487 Greifswald, Germany</p><p>Cardiovascular Health Research Unit, Departments of Medicine, Epidemiology and Health Services, University of Washington, Seattle, WA, USA</p><p>Laboratory of Epidemiology, Demography, Biometry, National Institute on Aging, National Institutes of Health, Bethesda, Maryland 20892, USA</p><p>MRC Epidemiology Unit, Institute of Metabolic Science, Cambridge CB2 0QQ, UK</p><p>Centre National de G&#233;notypage, Commissariat &#224; L'Energie Atomique, Institut de G&#233;nomique, Evry, France</p><p>Institute of Molecular and Cell Biology, University of Tartu, Riia 23, Tartu 51010, Estonia</p><p>University of Maryland School of Medicine, Baltimore, MD, USA, 21201, USA</p><p>Center for Human Genetic Research, Cardiovascular Research Center, Massachusetts General Hospital, Boston, Massachusetts, 02114, USA</p><p>Department of Twin Research &amp; Genetic Epidemiology, King's College London, UK</p><p>Cardiovascular Epidemiology and Genetics, Institut Municipal d'Investigacio Medica, Barcelona Biomedical Research Park, 88 Doctor Aiguader, 08003 Barcelona, Spain</p><p>Institute of Genetic Medicine, European Academy Bozen/Bolzano (EURAC), Viale Druso 1, 39100 Bolzano, Italy&#8212;Affiliated Institute of the University of L&#252;beck, Germany</p><p>Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, Michigan, 48109, USA</p><p>Department of Cardiovascular Medicine, The Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, OX3 7BN, UK</p><p>Clinical Research Branch, National Institute on Aging, Baltimore MD 21250, USA</p><p>Centre for Population Health Sciences, University of Edinburgh, EH89AG, UK</p><p>Centre for Population Health Sciences and Institute of Genetics and Molecular Medicine, College of Medicine and Vet Medicine, University of Edinburgh, EH8 9AG, UK</p><p>Croatian Centre for Global Health, University of Split, Croatia</p><p>Department of Genetics and Pathology, Rudbeck Laboratory, Uppsala University, SE-751 85 Uppsala, Sweden</p><p>Amgen, 1 Kendall Square, Building 100, Cambridge, MA 02139, USA</p><p>Department of Clinical Sciences, Lund University, Malm&#246;, Sweden</p><p>Department of Medicine, University of Verona, Italy</p><p>Ealing Hospital, London, UB1 3HJ, UK</p><p>Department of Medicine, University of Mississippi Medical Center, USA</p><p>Genetic Epidemiology Group, Epidemiology and Public Health, UCL, London, WC1E 6BT, UK</p><p>Center for Genome Science, National Institute of Health, Seoul, Korea</p><p>Department of Cardiology, University Medical Center Groningen, University of Groningen, The Netherlands</p><p>Departments of Epidemiology and Medicine, Johns Hopkins University, Baltimore MD, USA</p><p>Centre for Cellular and Molecular Biology (CCMB), Council of Scientific and Industrial Research (CSIR), Uppal Road, Hyderabad 500 007, India</p><p>Department of Public Health and Primary Care, University of Cambridge, CB1 8RN, UK</p><p>Department of Basic Medical Research and Education, and Department of Geriatric Medicine, Ehime University Graduate School of Medicine, Toon, 791-0295, Japan</p><p>Division of Community Health Sciences, St George's University of London, London, SW17 0RE, UK</p><p>Epidemiology &amp; Prevention, Division of Public Health Sciences, Wake Forest University School of Medicine, Winston-Salem, NC 27157, USA</p><p>Division of Biostatistics and Department of Genetics, School of Medicine, Washington University in St. Louis, Saint Louis, Missouri 63110, USA</p><p>Department of Medicine, University of Eastern Finland and Kuopio University Hospital, 70210 Kuopio, Finland</p><p>Department of Preventive Medicine and Epidemiology, Loyola University Medical School, Maywood, IL, USA</p><p>Department of Laboratory Medicine &amp; Institute of Human Genetics, University of California San Francisco, 513 Parnassus Ave. San Francisco CA 94143, USA</p><p>Genome Institute of Singapore, Agency for Science, Technology and Research, Singapore, 138672, Singapore</p><p>Centre for Molecular Epidemiology, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, 117597, Singapore</p><p>Department of Clinical Chemistry, University of Tampere and Tampere University Hospital, Tampere, 33521, Finland</p><p>Department of Genetics, Biology and Biochemistry, University of Torino, Via Santena 19, 10126, Torino, Italy</p><p>Human Genetics Foundation (HUGEF), Via Nizza 52, 10126, Torino, Italy</p><p>Department of Genetics, University of North Carolina, Chapel Hill, NC, 27599, USA</p><p>MRC Centre for Causal Analyses in Translational Epidemiology, School of Social &amp; Community Medicine, University of Bristol, Bristol BS8 2BN, UK</p><p>Julius Center for Health Sciences and Primary Care, University Medical Center Utrecht, Heidelberglaan 100, 3508 GA Utrecht, The Netherlands</p><p>Complex Genetics Section, Department of Medical Genetics&#8212;DBG, University Medical Center Utrecht, 3508 GA Utrecht, The Netherlands</p><p>Centre for Genetic Epidemiology and Biostatistics, University of Western Australia, Crawley, WA, Australia</p><p>HUNT Research Centre, Department of Public Health and General Practice, Norwegian University of Science and Technology, 7600 Levanger, Norway</p><p>MRC Unit for Lifelong Health &amp; Ageing, London, WC1B 5JU, UK</p><p>Occupational and Environmental Medicine, Department of Public Health and Community Medicine, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, 40530 Gothenburg, Sweden</p><p>Centre for Cardiovascular Genetics, University College London, London WC1E 6JF, UK</p><p>MRC Human Genetics Unit and Institute of Genetics and Molecular Medicine, Edinburgh, EH2, UK</p><p>Department of Cardiovascular Sciences, University of Leicester, Glenfield Hospital, Leicester, LE3 9QP, UK</p><p>Leicester NIHR Biomedical Research Unit in Cardiovascular Disease, Glenfield Hospital, Leicester, LE3 9QP, UK</p><p>Studies Coordinating Centre, Division of Hypertension and Cardiac Rehabilitation, Department of Cardiovascular Diseases, University of Leuven, Campus Sint Rafa&#235;l, Kapucijnenvoer 35, Block D, Box 7001, 3000 Leuven, Belgium</p><p>Center for Research on Genomics and Global Health, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><p>Columbia University, NY, USA</p><p>Department of Medicine III, Medical Faculty Carl Gustav Carus at the Technical University of Dresden, 01307 Dresden, Germany</p><p>Epidemiology and Biostatistics, School of Public Health, Imperial College, London, W2 1PG, UK</p><p>Clinical and Molecular Epidemiology Unit, Department of Hygiene and Epidemiology, University of Ioannina School of Medicine, Ioannina, Greece</p><p>Wake Forest University Health Sciences, Winston-Salem, NC 27157, USA</p><p>A list of consortium members is supplied in the Supplementary Materials</p><p>Department of Epidemiology, School of Public Health, University of Michigan, Ann Arbor, MI 48109, USA</p><p>Division of Epidemiology, Human Genetics and Environmental Sciences, School of Public Health, University of Texas at Houston Health Science Center, 12 Herman Pressler, Suite 453E, Houston, TX 77030, USA</p><p>Laboratory of Neurogenetics, National Institute on Aging, Bethesda, MD 20892, USA</p><p>Laboratory of Cardiovascular Science, Intramural Research Program, National Institute on Aging, NIH, Baltimore, Maryland, USA</p><p>Washington Hospital Center, Division of Cardiology, Washington DC, USA</p><p>ALSPAC Laboratory, University of Bristol, Bristol, BS8 2BN, UK</p><p>Pediatric Epidemiology Center, University of South Florida, Tampa, FL, USA</p><p>Clinical Pharmacology Unit, University of Cambridge, Addenbrookes Hospital, Hills Road, Cambridge CB2 2QQ, UK</p><p>University of Dundee, Ninewells Hospital &amp;Medical School, Dundee, DD1 9SY, UK</p><p>Genetic Epidemiology Group, Department of Epidemiology and Public Health, UCL, London WC1E 6BT, UK</p><p>Pathology and Laboratory Medicine, University of Western Australia, Crawley, WA, Australia</p><p>Molecular Genetics, PathWest Laboratory Medicine, Nedlands, WA, Australia</p><p>Clinical Trial Service Unit and Epidemiological Studies Unit, University of Oxford, Oxford, OX3 7LF, UK</p><p>Department of Epidemiology and Biostatistics, Case Western Reserve University, 2103 Cornell Road, Cleveland, OH 44106, USA</p><p>Department of Medicine, Johns Hopkins University, Baltimore, USA</p><p>Department of Clinical Physiology, University of Tampere and Tampere University Hospital, Tampere, 33521, Finland</p><p>Department of Medicine, University of Turku and Turku University Hospital, Turku, 20521, Finland</p><p>Department of Nutrition, University of North Carolina, Chapel Hill, NC, 27599, USA</p><p>Office of Population Studies Foundation, University of San Carlos, Talamban, Cebu City 6000, Philippines</p><p>Department of Neurology and Framingham Heart Study, Boston University School of Medicine, Boston, MA, 02118, USA</p><p>Department of Internal Medicine II, University Medical Center Regensburg, 93053 Regensburg, Germany</p><p>Department of Epidemiology and Preventive Medicine, University Medical Center Regensburg, 93053 Regensburg, Germany</p><p>Department of Epidemiology, Johns Hopkins University, Baltimore MD, USA</p><p>Renal Division, University Hospital Freiburg, Germany</p><p>D&#233;partement de G&#233;n&#233;tique M&#233;dicale, Universit&#233; de Lausanne, 1015 Lausanne, Switzerland</p><p>Swiss Institute of Bioinformatics, 1015 Lausanne, Switzerland</p><p>Division of Genetics, GlaxoSmithKline, Philadelphia, Pennsylvania 19101, USA</p><p>International Centre for Circulatory Health, National Heart &amp; Lung Institute, Imperial College, London, UK</p><p>Genetics of Complex Traits, Peninsula Medical School, University of Exeter, UK</p><p>Department of Community Health Sciences &amp; Department of Medicine, Aga Khan University, Karachi, Pakistan</p><p>Medizinische Klinik II, Universit&#228;t zu L&#252;beck, L&#252;beck, Germany</p><p>Diabetes Unit, KEM Hospital and Research Centre, Rasta Peth, Pune-411011, Maharashtra, India</p><p>Department of Clinical Sciences, Diabetes and Endocrinology Research Unit, University Hospital, Malm&#246;, Sweden</p><p>Lund University, Malm&#246; 20502, Sweden</p><p>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, Massachusetts, 02139, USA</p><p>Department of Chronic Disease Prevention, National Institute for Health and Welfare, FIN-00251 Helsinki, Finland</p><p>William Harvey Research Institute, Barts and The London School of Medicine and Dentistry, Queen Mary University of London, London EC1M 6BQ, UK</p><p>Merck Research Laboratory, 126 East Lincoln Avenue, Rahway, NJ 07065, USA</p><p>Wellcome Trust Sanger Institute, Hinxton, CB10 1SA, UK</p><p>University of Cambridge Metabolic Research Labs, Institute of Metabolic Science Addenbrooke's Hospital, CB2 OQQ, Cambridge, UK</p><p>Division of Cardiovascular Medicine, Department of Internal Medicine, University of Michigan Medical School, Ann Arbor, MI, USA</p><p>Cardiovascular Genetics, University of Utah School of Medicine, Salt Lake City, UT, USA</p><p>Department of Physiology and Biophysics, Keck School of Medicine, University of Southern California, Los Angeles, California 90033, USA</p><p>National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892,USA</p><p>National Institute for Health and Welfare, 00271 Helsinki, Finland</p><p>FIMM, Institute for Molecular Medicine, Finland, Biomedicum, P.O. Box 104, 00251 Helsinki, Finland</p><p>Broad Institute, Cambridge, Massachusetts 02142, USA</p><p>Leibniz-Institute for Arteriosclerosis Research, Department of Molecular Genetics of Cardiovascular Disease, University of M&#252;nster, M&#252;nster, Germany</p><p>Medical Faculty of the Westfalian Wilhelms University Muenster, Department of Molecular Genetics of Cardiovascular Disease, University of Muenster, Muenster, Germany</p><p>Division of Cardiology, Massachusetts General Hospital, Boston, MA, USA</p><p>Georgia Prevention Institute, Department of Pediatrics, Medical College of Georgia, Augusta, GA, USA</p><p>Unit of Genetic Epidemiology and Bioinformatics, Department of Epidemiology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands</p><p>Department of Biostatical Sciences, Division of Public Health Sciences, Wake Forest University School of Medicine, Winston-Salem, NC 27157, USA</p><p>Department of Epidemiology, University of Washington, Seattle, WA, 98195, USA</p><p>Group Health Research Institute, Group Health Cooperative, Seattle, WA, USA</p><p>Seattle Epidemiologic Research and Information Center, Veterans Health Administration Office of Research &amp; Development, Seattle, WA 98108, USA</p><p>Department of Medicine, University of Washington, 98195, USA</p><p>Department of Cardiology, University of Tartu, L. Puusepa 8, 51014 Tartu, Estonia</p><p>Tallinn University of Technology, Institute of Biomedical Engineering, Ehitajate tee 5, 19086 Tallinn, Estonia</p><p>Centre of Cardiology, North Estonia Medical Centre, S&#252;tiste tee 19, 13419 Tallinn, Estonia</p><p>Division of Non-communicable disease Epidemiology, The London School of Hygiene and Tropical Medicine London, Keppel Street, London WC1E 7HT, UK</p><p>South Asia Network for Chronic Disease, Public Health Foundation of India, C-1/52, SDA, New Delhi 100016, India</p><p>Department of Emergency and Cardiovascular Medicine, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, 41685 Gothenburg, Sweden</p><p>Department of Biostatistics, Institute of Basic Medical Sciences, University of Oslo, 0317 Oslo, Norway</p><p>Tuscany Regional Health Agency, Florence, Italy</p><p>Tropical Medicine Research Institute, University of the West Indies, Mona, Kingston, Jamaica</p><p>University of Ibadan, Ibadan, Nigeria</p><p>Department of Genomic Medicine, and Department of Preventive Cardiology, National Cerebral and Cardiovascular Research Center, Suita, 565-8565, Japan</p><p>Department of Health Science, Shiga University of Medical Science, Otsu, 520-2192, Japan</p><p>Department of Geriatric Medicine, Osaka University Graduate School of Medicine, Suita, 565-0871, Japan</p><p>Tohoku University Graduate School of Pharmaceutical Sciences and Medicine, Sendai, 980-8578, Japan</p><p>Lifestyle-related Disease Prevention Center, Shiga University of Medical Science, Otsu, 520-2192, Japan</p><p>Department of Medical Science and Cardiorenal Medicine, Yokohama City University School of Medicine, Yokohama, 236-0004, Japan</p><p>Department of Statistics, Pontificia Universidad Catolica de Chile, Vicu&#241;a Mackena 4860, Santiago, Chile</p><p>Institute of Human Genetics, Helmholtz Zentrum Munich, German Research Centre for Environmental Health, 85764 Neuherberg, Germany</p><p>Institute of Human Genetics, Klinikum rechts der Isar, Technical University of Munich, 81675 Munich, Germany</p><p>Institute of Epidemiology, Helmholtz Zentrum Munich, German Research Centre for Environmental Health, 85764 Neuherberg, Germany</p><p>Chair of Epidemiology, Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-Universit&#228;t, 81377 Munich, Germany</p><p>Klinikum Grosshadern, 81377 Munich, Germany</p><p>National Heart and Lung Institute, Imperial College London, London, UK, W12 0HS, UK</p><p>Medical Genetics Institute, Cedars-Sinai Medical Center, Los Angeles, CA, USA</p><p>Medical Population Genetics, Broad Institute of Harvard and MIT, 5 Cambridge Center, Cambridge MA 02142, USA</p><p>National Heart, Lung and Blood Institute and its Framingham Heart Study, 73 Mount Wayte Ave., Suite #2, Framingham, MA 01702, USA</p><p>Department of Neurology and Medicine, University of Washington, Seattle, USA</p><p>Department of Medicine (Geriatrics), University of Mississippi Medical Center, Jackson, MS, USA</p><p>Department of Neurology, Boston University School of Medicine, USA</p><p>Finnish Institute of Occupational Health, Finnish Institute of Occupational Health, Aapistie 1, 90220 Oulu, Finland</p><p>Wellcome Trust Centre for Human Genetics, University of Oxford, UK</p><p>Lapland Central Hospital, Department of Physiatrics, Box 8041, 96101 Rovaniemi, Finland</p><p>Center for Non-Communicable Diseases Karachi, Pakistan</p><p>Atherosclerosis Research Unit, Department of Medicine, Karolinska Institute, Stockholm, Sweden</p><p>Department of Internal Medicine, University Medical Center Groningen, University of Groningen, The Netherlands</p><p>Department of Medical Genetics, Erasmus Medical Center, Rotterdam, The Netherlands</p><p>Gerontology Research Center, National Institute on Aging, Baltimore, MD 21224, USA</p><p>Istituto di Neurogenetica e Neurofarmacologia, Consiglio Nazionale delle Ricerche, Cittadella Universitaria di Monserrato, Monserrato, Cagliari, Italy</p><p>Unita`Operativa Semplice Cardiologia, Divisione di Medicina, Presidio Ospedaliero Santa Barbara, Iglesias, Italy</p><p>Computational Medicine Research Group, Institute of Clinical Medicine, University of Oulu and Biocenter Oulu, 90014 University of Oulu, Oulu, Finland</p><p>NMR Metabonomics Laboratory, Department of Biosciences, University of Eastern Finland, 70211 Kuopio, Finland</p><p>Department of Internal Medicine and Biocenter Oulu, Clinical Research Center, 90014 University of Oulu, Oulu, Finland</p><p>Institute for Molecular Medicine Finland FIMM, 00014 University of Helsinki, Helsinki, Finland</p><p>Department of Biomedical Engineering and Computational Science, School of Science and Technology, Aalto University, 00076 Aalto, Espoo, Finland</p><p>NUS Graduate School for Integrative Sciences &amp; Engineering (NGS) Centre for Life Sciences (CeLS), Singapore, 117456, Singapore</p><p>Department of Internal Medicine B, Ernst-Moritz-Arndt-University Greifswald, 17487 Greifswald, Germany</p><p>Institute of Pharmacology, Ernst-Moritz-Arndt-University Greifswald, 17487 Greifswald, Germany</p><p>Institute for Community Medicine, Ernst-Moritz-Arndt-University Greifswald, 17487 Greifswald, Germany</p><p>U557 Institut National de la Sant&#233; et de la Recherche M&#233;dicale, U1125 Institut National de la Recherche Agronomique, Universit&#233; Paris 13, Bobigny, France</p><p>Department of Neurology, Mayo Clinic, Jacksonville, FL, USA</p><p>Imperial College Cerebrovascular Unit (ICCRU), Imperial College, London, W6 8RF, UK</p><p>Faculty of Medicine, University of Split, Croatia</p><p>Department of Internal Medicine, Diabetology, and Nephrology, Medical University of Silesia, 41-800, Zabrze, Poland</p><p>Public Health Sciences section, Division of Community Health Sciences, University of Edinburgh, Medical School, Teviot Place, Edinburgh, EH8 9AG, UK</p><p>School of Science and Engineering, University of Ballarat, 3353 Ballarat, Australia</p><p>Prevention and Care of Diabetes, Department of Medicine III, Medical Faculty Carl Gustav Carus at the Technical University of Dresden, 01307 Dresden, Germany</p><p>University Hospital M&#252;nster, Internal Medicine D, M&#252;nster, Germany</p><p>Department of Medical Statistics, Epidemiology and Medical Informatics, Andrija Stampar School of Public Health, University of Zagreb, Croatia</p><p>AstraZeneca R&amp;D, 431 83 M&#246;lndal, Sweden</p><p>Genetic Epidemiology &amp; Biostatistics Platform, Ontario Institute for Cancer Research, Toronto</p><p>Samuel Lunenfeld Institute for Medical Research, University of Toronto, Canada</p><p>Faculty of Epidemiology and Population Health, London School of Hygiene and Tropical Medicine, UK</p><p>Department of Epidemiology and Public Health, Imperial College, Norfolk Place London W2 1PG, UK</p><p>Research Centre of Applied and Preventive Cardiovascular Medicine, University of Turku and the Department of Clinical Physiology, Turku University Hospital, Turku, 20521, Finland</p><p>Department of Internal Medicine, Division of Cardiovascular Medicine, University of Michigan Medical Center, Ann Arbor, Michigan, USA</p><p>Singapore Eye Research Institute, Singapore, 168751, Singapore</p><p>Department of Ophthalmology, National University of Singapore, Singapore, 119074, Singapore</p><p>Department of Medicine, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, 119074, Singapore</p><p>Duke-National University of Singapore Graduate Medical School, Singapore, 169857, Singapore</p><p>Division of Biostatistics, Washington University School of Medicine, Saint Louis, MO, 63110, USA</p><p>Department of Primary Care &amp; Population Health, UCL, London, UK, NW3 2PF, UK</p><p>BHF Glasgow Cardiovascular Research Centre, University of Glasgow, 126 University Place, Glasgow, G12 8TA, UK</p><p>Epidemiology Public Health, UCL, London, UK, WC1E 6BT, UK</p><p>Departments of Epidemiology, Biostatistics, and Medicine, Johns Hopkins University, Baltimore MD, USA</p><p>Division of Nephrology, Department of Internal Medicine, University Medical Center Groningen, University of Groningen, The Netherlands</p><p>Division of Cardiology, Brigham and Women's Hospital, 900 Commonwealth Avenue East, Boston MA 02215, USA</p><p>Geriatric Rehabilitation Unit, Azienda Sanitaria Firenze (ASF), Florence, Italy</p><p>National Institute for Health and Welfare, Diabetes Prevention Unit, 00271 Helsinki, Finland</p><p>Hjelt Institute, Department of Public Health, University of Helsinki, 00014 Helsinki, Finland</p><p>South Ostrobothnia Central Hospital, 60220 Sein&#228;joki, Finland</p><p>Red RECAVA Grupo RD06/0014/0015, Hospital Universitario La Paz, 28046 Madrid, Spain</p><p>Department of Neurology, General Central Hospital, 39100 Bolzano, Italy</p><p>CIBER Epidemiolog&#237;a y Salud P&#250;blica, 08003 Barcelona</p><p>Department of Medicine and Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115, USA</p><p>Geriatric Research and Education Clinical Center, Veterans Administration Medical Center, Baltimore, MD, USA</p><p>U872 Institut National de la Sant&#233; et de la Recherche M&#233;dicale, Centre de Recherche des Cordeliers, Paris, France</p><p>Institute of Physiology, Ernst-Moritz-Arndt-University Greifswald, 17487 Greifswald, Germany</p><p>Institute of Clinical Medicine/Obstetrics and Gynecology, University of Oulu, Finland</p><p>Service of Medical Genetics, Centre Hospitalier Universitaire Vaudois, 1011 Lausanne, Switzerland</p><p>Human Genetics Center, 1200 Hermann Pressler, Suite E447 Houston, TX 77030, USA</p><p>Division of Epidemiology and Prevention, Boston University School of Medicine, Boston, MA, USA</p><p>Department of Mathematics, Boston University, Boston, MA, USA</p><p>Institute of Health Sciences, University of Oulu, BOX 5000, 90014 University of Oulu, Finland</p><p>Biocenter Oulu, University of Oulu, BOX 5000, 90014 University of Oulu, Finland</p><p>National Institute for Health and Welfare, Box 310, 90101 Oulu, Finland</p><p>MRC-HPA Centre for Environment and Health, School of Public Health, Imperial College London, Norfolk Place, London W2 1PG, UK</p><p>Centre of Medical Systems Biology (CMSB 1&#8211;2), NGI Erasmus Medical Center, Rotterdam, The Netherlands</p>