<p>This eQTL meta-analysis is based on gene expression intensities measured in whole blood samples. RNA was isolated with either PAXgene Tubes (Becton Dickinson and Co., Franklin Lakes, NJ, USA) or Tempus Tubes (Life Technologies). To measure gene expression levels, Illumina Whole-Genome Expression Beadchips were used (HT12-v3 and HT12-v4 arrays, Illumina Inc., San Diego, USA). Although different identifiers are used across these different platforms, many probe sequences are identical. Meta-analysis could thus be performed if probe-sequences were equal across platforms. Integration of these probe sequences was performed as described before [<xref ref-type="bibr" rid="pgen.1005223.ref022">22</xref>]. Genotypes were harmonized using HapMap2-based imputation using the Central European population [<xref ref-type="bibr" rid="pgen.1005223.ref032">32</xref>]. In total, the eQTL genotype x environment interaction meta-analysis was performed on seven independent cohorts, comprising a total of 5,863 unrelated individuals (full descriptions of these cohorts can be found in the Supplementary Note). Mix-ups between gene expression samples and genotype samples were corrected using <italic>MixupMapper</italic> [<xref ref-type="bibr" rid="pgen.1005223.ref033">33</xref>].</p><p>Each cohort performed gene expression normalization individually: gene expression data was quantile normalized to the median distribution then log<sub>2</sub> transformed. The probe and sample means were centered to zero. Gene expression data was then corrected for possible population structure by removing four multi-dimensional scaling components (MDS components obtained from the genotype data using PLINK) using linear regression. Additionally, we corrected for possible confounding factors due to arrays of poor RNA quality. We reasoned that arrays of poor RNA quality generally show expression for genes that are normally lowly expressed within the tissue (e.g. expression for brain genes in whole blood data). As such, the expression profiles for such arrays will deviate overall from arrays with proper RNA quality. To capture such variable arrays, we calculated the first PC from the sample correlation matrix and correlated the first PC with the sample gene expression measurements. Samples with a correlation &lt; 0.9 were removed from further analysis (<xref ref-type="supplementary-material" rid="pgen.1005223.s009">S9 Fig</xref>).</p><p>In order to improve statistical power to detect cell-type mediated eQTLs, we corrected the gene expression for technical and batch effects (here we applied principal component analysis and removed per cohort the 40 strongest principal components that affect gene expression). Such procedures are commonly used when conducting <italic>cis-</italic>eQTL mapping [<xref ref-type="bibr" rid="pgen.1005223.ref002">2</xref>,<xref ref-type="bibr" rid="pgen.1005223.ref005">5</xref>,<xref ref-type="bibr" rid="pgen.1005223.ref022">22</xref>,<xref ref-type="bibr" rid="pgen.1005223.ref030">30</xref>,<xref ref-type="bibr" rid="pgen.1005223.ref031">31</xref>,<xref ref-type="bibr" rid="pgen.1005223.ref034">34</xref>]. To minimize the amount of genetic variation removed by this procedure, we performed QTL mapping for each principal component, to ascertain whether genetic variants could be detected that affected the PC. If such an effect was detected, we did not correct the gene expression data for that particular PC [<xref ref-type="bibr" rid="pgen.1005223.ref022">22</xref>]. As a result, this procedure also removed the majority of the variation that explained the correlation between neutrophil percentage and gene expression (<xref ref-type="supplementary-material" rid="pgen.1005223.s010">S10 Fig</xref>), minimizing issues with possible collinearity when testing the interaction effects. We chose to remove 40 PCs based on our previous study results, which suggested that this was the optimum for detecting eQTLs [<xref ref-type="bibr" rid="pgen.1005223.ref022">22</xref>]. We would like to stress that while PC-corrected gene expression data was then used as the outcome variable in our gene x environment interaction model, we used gene expression data that was not corrected for PCs to initially create the neutrophil cell percentage proxy.</p><p>To be able to determine whether a <italic>cis</italic>-eQTL is mediated by neutrophils, we reasoned that such a <italic>cis</italic>-eQTL would show a larger effect size in individuals with a higher percentage of neutrophils than in individuals with a lower percentage. However, this required the percentage of neutrophils in whole blood to be known, and cell-type percentage measurements were not available for all of the cohorts. We therefore created a proxy phenotype that reflected the actual neutrophil percentage that would also be applicable to datasets without neutrophil percentage measurements. In the EGCUT dataset, we first quantile normalized and log<sub>2</sub> transformed the raw expression data. We then correlated the gene expression levels of individual probes with the neutrophil percentage, and selected 58 gene expression probes showing a high positive correlation (Spearman R &gt; 0.57). Here, we chose to use the quantile normalized, log<sub>2</sub> transformed gene expression data that was not corrected for principal components, since correction for principal components would remove the correlation structure between gene expression and neutrophil percentage (<xref ref-type="supplementary-material" rid="pgen.1005223.s010">S10 Fig</xref>).</p><p>In each independent cohort, we corrected for possible confounding factors due to arrays with poor RNA quality, by correlating the quantile normalized and log<sub>2</sub> transformed gene expression measurements against the first PC determined from the sample correlation matrix. Only samples with a high correlation (r &#8805; 0.9) were included in further analyses. Then, for each cohort, we calculated a correlation matrix for the neutrophil proxy probes (the probes selected from the EGCUT cohort). The gene expression data used was quantile normalized, log<sub>2</sub> transformed and corrected for MDS components. Applying PCA to the correlation matrix, we then obtained PCs that described the variation among the probes selected from the EGCUT cohort. As the first PC (PC1) contributes the largest amount of variation, we considered PC1 as a proxy-phenotype for the cell type percentages.</p><p>Considering the overlap between the cohorts in this study and our previous study, we limited our analysis to the 13,124 <italic>cis</italic>-eQTLs having a significant effect (false discovery rate, FDR &lt; 0.05) in our previous study [<xref ref-type="bibr" rid="pgen.1005223.ref022">22</xref>]. This included 8,228 unique Illumina HT12v3 probes and 10,260 unique SNPs (7,674 SNPs that showed the strongest effect per probe, and 2,586 SNPs previously associated with complex traits and diseases, as reported in the Catalog of Published Genome-Wide Association Studies [<xref ref-type="bibr" rid="pgen.1005223.ref001">1</xref>], on 23<sup>rd</sup> September, 2013).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We defined the model for single marker <italic>cis</italic>-eQTL mapping as follows:
<disp-formula id="pgen.1005223.e001"><alternatives><graphic id="pgen.1005223.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005223.e001.jpg" /><ns1:math id="M1"><ns1:mrow><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>Y</ns1:mi></ns1:mstyle><ns1:mo>&#8776;</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>I</ns1:mi></ns1:mstyle><ns1:mo>+</ns1:mo><ns1:msub><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>&#946;</ns1:mi></ns1:mstyle><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mn>1</ns1:mn></ns1:mstyle></ns1:msub><ns1:mo>*</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>G</ns1:mi></ns1:mstyle><ns1:mo>+</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>e</ns1:mi></ns1:mstyle></ns1:mrow></ns1:math></alternatives></disp-formula>
where <bold>Y</bold> is the gene expression of the gene, <bold>&#946;</bold>
<sub><bold>1</bold></sub> is the slope of the linear model, <bold>G</bold> is the genotype, <bold>I</bold> is the intercept with the y-axis, and <bold>e</bold> is the general error term for any residual variation not explained by the rest of the model.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We then extended the typical linear model for single marker <italic>cis</italic>-eQTL mapping to include a covariate as an independent variable, and captured the interaction between the genotype and the covariate using an interaction term:
<disp-formula id="pgen.1005223.e002"><alternatives><graphic id="pgen.1005223.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005223.e002.jpg" /><ns1:math id="M2"><ns1:mrow><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>Y</ns1:mi></ns1:mstyle><ns1:mo>&#8776;</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>I</ns1:mi></ns1:mstyle><ns1:mo>+</ns1:mo><ns1:msub><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>&#946;</ns1:mi></ns1:mstyle><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mn>1</ns1:mn></ns1:mstyle></ns1:msub><ns1:mo>*</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>G</ns1:mi></ns1:mstyle><ns1:mo>+</ns1:mo><ns1:msub><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>&#946;</ns1:mi></ns1:mstyle><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mn>2</ns1:mn></ns1:mstyle></ns1:msub><ns1:mo>*</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>P</ns1:mi></ns1:mstyle><ns1:mo>+</ns1:mo><ns1:msub><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>&#946;</ns1:mi></ns1:mstyle><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mn>3</ns1:mn></ns1:mstyle></ns1:msub><ns1:mo>*</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>P</ns1:mi></ns1:mstyle><ns1:mi mathvariant="bold">:</ns1:mi><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>G</ns1:mi></ns1:mstyle><ns1:mo>+</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>e</ns1:mi></ns1:mstyle></ns1:mrow></ns1:math></alternatives></disp-formula>
where P (cell-type proxy) is the covariate, and P:G is the interaction term between the covariate and the genotype. We used gene expression data corrected for 40 PCs as the predicted variable (<bold>Y</bold>). The interaction terms were then meta-analyzed over all cohorts using a Z-score method, weighted for the sample size [<xref ref-type="bibr" rid="pgen.1005223.ref035">35</xref>].</p><p>Since the gene-expression data has a correlated structure (i.e. co-expressed genes) and the genotype data also has a correlated structure (i.e. linkage disequilibrium between SNPs), a Bonferroni correction would be overly stringent. We therefore first estimated the effective number of uncorrelated tests by using permuted eQTL results from our previous <italic>cis-</italic>eQTL meta-analysis [<xref ref-type="bibr" rid="pgen.1005223.ref022">22</xref>]. The most significant P-value in these permutations was 8.15 x 10<sup>&#8211;5</sup>, when averaged over all permutations. As such, the number of effective tests = 0.5 / 8.15 x 10<sup>&#8211;5</sup> &#8776; 6134, which is approximately half the number of correlated <italic>cis</italic>-eQTL tests that we conducted (= 13,124). Next, we controlled the FDR at 0.05 for the interaction analysis: for a given P-value threshold in our interaction analysis, we calculated the number of expected results (given the number of effective tests and a uniform distribution) and determined the observed number of eQTLs that were below the given P-value threshold (FDR = number of expected p-values below threshold / number of observed p-values below threshold). At an FDR of 0.05, our nominal p-value threshold was 0.009 (corresponding to an absolute interaction effect Z-score of 2.61).</p><p>For each trait in the GWAS catalog, we pruned all SNPs with a GWAS association P-value below 5 x 10<sup>&#8211;8</sup>, using an r<sup>2</sup> threshold of 0.2. We only considered traits that had more than 20 significant eQTL SNPs after pruning (irrespective of cell-type mediation). Then, we determined the proportion of pruned neutrophil-mediated <italic>cis</italic>-eQTLs for the trait relative to all the neutrophil-mediated <italic>cis</italic>-eQTLs. The difference between both proportions was then tested using a binomial test.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The source code and documentation for this type of analysis are available as part of the eQTL meta-analysis pipeline at <ext-link ext-link-type="uri" ns0:href="https://github.com/molgenis/systemsgenetics">https://github.com/molgenis/systemsgenetics</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Summary results are available from <ext-link ext-link-type="uri" ns0:href="http://www.genenetwork.nl/celltype">http://www.genenetwork.nl/celltype</ext-link>
</p><p>Discovery cohorts: Fehrmann (GSE 20142), SHIP-TREND (GSE 36382), Rotterdam Study (GSE 33828), EGCUT (GSE 48348), DILGOM (E-TABM-1036), InCHIANTI (GSE 48152), KORA F4 (E-MTAB-1708). Replication Cohorts: Stranger (E-MTAB-264), Oxford (E-MTAB-945).</p>