<p>All participants gave written informed consent, with protocols approved by institutional review board of the National Institute on Aging (04-AG-N317).</p><p>Recent analyses of mtDNA variants[<xref ref-type="bibr" rid="pgen.1005306.ref011">11</xref>,<xref ref-type="bibr" rid="pgen.1005306.ref012">12</xref>] have taken an approach that determines homoplasmic and heteroplasmic sites directly based on allele counts of sequence reads. This approach does not account for error rates in sequence reads, and hence potentially results in both false positive and false negative variant calls. We propose a likelihood-based model that takes into account the sequencing error rate at each base in each sequence read. The algorithm builds on the conventional autosomal DNA variant callers[<xref ref-type="bibr" rid="pgen.1005306.ref006">6</xref>,<xref ref-type="bibr" rid="pgen.1005306.ref007">7</xref>], but is modified to allow for allele fractions (i.e., heteroplasmic levels) at a variant site to vary across individuals.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We aim to predict the genotype at each mtDNA position, one individual at a time; therefore, the subscripts indicating mtDNA position and individual are suppressed for simplicity. At each mtDNA base position, we use <bold><italic>G</italic></bold> to denote one individual&#8217;s genotype. In theory, <bold><italic>G</italic></bold> can take 15 possible values {A, C, G, T, A/C, A/G, A/T, C/G, C/T, G/T, A/C/G, A/C/T, A/G/T, C/G/T, A/C/G/T}. If <bold><italic>G</italic></bold> takes one of the first four possible genotypes (with a single allele), it is by definition called a homoplasmy. However, by convention, if the genotype is the same as the standard reference allele, it is not counted as a homoplasmy. The remaining 11 possible genotypes are called heteroplasmies. We estimate the posterior probability of having a certain genotype <bold><italic>G</italic></bold> given all the observed sequence reads:
<disp-formula id="pgen.1005306.e001"><alternatives><graphic id="pgen.1005306.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005306.e001.jpg" /><ns1:math id="M1"><ns1:mrow><ns1:mi>P</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>G</ns1:mi><ns1:mo>|</ns1:mo><ns1:mi>r</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>d</ns1:mi><ns1:mi>s</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mspace width="0.25em" /><ns1:mo>&#8733;</ns1:mo><ns1:mi>P</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>r</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>d</ns1:mi><ns1:mi>s</ns1:mi><ns1:mo>|</ns1:mo><ns1:mi mathvariant="bold-italic">G</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mspace width="0.25em" /><ns1:mo>&#215;</ns1:mo><ns1:mspace width="0.25em" /><ns1:mi>P</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi mathvariant="bold-italic">G</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives><label>(1)</label></disp-formula>
where <italic>P</italic>(<bold><italic>G</italic></bold>|<italic>reads</italic>) is the posterior probability, <italic>P</italic>(<italic>reads</italic>|<bold><italic>G</italic></bold>) is called &#8220;genotype likelihood&#8221;, and <italic>P</italic>(<bold><italic>G</italic></bold>) is the prior probability of genotype <bold><italic>G</italic></bold>, which can be assigned by researchers. We can then assign the genotype with highest posterior probability to the site for the individual. So the key is to estimate 15 genotype likelihoods: <italic>P</italic>(<italic>reads</italic>|<bold><italic>G</italic></bold>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Assuming the mtDNA site of interest is covered by N sequence reads, we use r<sub>i</sub> to denote the called base from read i (i = 1, 2, &#8230;, N) and e<sub>i</sub> to denote the corresponding sequencing error rate of that base for read i, which can be estimated from the corresponding Phred-like base quality score Qi (<inline-formula id="pgen.1005306.e002"><alternatives><graphic id="pgen.1005306.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005306.e002.jpg" /><ns1:math id="M2"><ns1:mrow><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>=</ns1:mo><ns1:mspace width="0.15em" /><ns1:msup><ns1:mrow><ns1:mn>10</ns1:mn></ns1:mrow><ns1:mrow><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mi>Q</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>/</ns1:mo><ns1:mn>10</ns1:mn></ns1:mrow></ns1:msup></ns1:mrow></ns1:math></alternatives></inline-formula>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We assume that each sequence read represents an independent random sampling from the multiple copies of mtDNA in a cell, and therefore a genotype likelihood can be estimated by:
<disp-formula id="pgen.1005306.e003"><alternatives><graphic id="pgen.1005306.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005306.e003.jpg" /><ns1:math id="M3"><ns1:mrow><ns1:mi>P</ns1:mi><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>r</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>d</ns1:mi><ns1:mi>s</ns1:mi><ns1:mo stretchy="false">|</ns1:mo><ns1:mi mathvariant="bold-italic">G</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mo>=</ns1:mo><ns1:msubsup><ns1:mo>&#8719;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>N</ns1:mi></ns1:msubsup><ns1:mspace width="0.15em" /><ns1:mi>P</ns1:mi><ns1:mo stretchy="false">(</ns1:mo><ns1:msub><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mi mathvariant="bold-italic">G</ns1:mi><ns1:mo stretchy="false">)</ns1:mo></ns1:mrow></ns1:math></alternatives><label>(2)</label></disp-formula>
When <bold><italic>G</italic></bold> has only one allele (i.e. <bold><italic>G</italic></bold> &#8712; {<italic>A</italic>, <italic>C</italic>, <italic>G</italic>, <italic>T</italic>}), <italic>P</italic>(<italic>r</italic>
<sub><italic>i</italic></sub>|<bold><italic>G</italic></bold>) can be directly estimated while considering the base error rate:
<disp-formula id="pgen.1005306.e004"><alternatives><graphic id="pgen.1005306.e004g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005306.e004.jpg" /><ns1:math id="M4"><ns1:mrow><ns1:mi>P</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mi mathvariant="bold-italic">G</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mrow><ns1:mo>{</ns1:mo><ns1:mrow><ns1:mtable columnalign="left"><ns1:mtr><ns1:mtd><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:mspace width="1.5em" /><ns1:mi>i</ns1:mi><ns1:mi>f</ns1:mi><ns1:mspace width="0.25em" /><ns1:msub><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>=</ns1:mo><ns1:mi mathvariant="bold-italic">G</ns1:mi></ns1:mrow></ns1:mtd></ns1:mtr><ns1:mtr><ns1:mtd><ns1:mrow><ns1:mspace width="1.4em" /><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:mrow><ns1:mn>3</ns1:mn></ns1:mfrac><ns1:mo>,</ns1:mo><ns1:mspace width="1.5em" /><ns1:mi>i</ns1:mi><ns1:mi>f</ns1:mi><ns1:mspace width="0.25em" /><ns1:msub><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>&#8800;</ns1:mo><ns1:mi mathvariant="bold-italic">G</ns1:mi></ns1:mrow></ns1:mtd></ns1:mtr></ns1:mtable></ns1:mrow></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Because a cell has multiple copies of mtDNA and each individual can have different fractions for the alleles, when <bold><italic>G</italic></bold> has more than one allele (i.e. <bold><italic>G</italic></bold> &#8712; {<italic>A</italic>/<italic>C</italic>, <italic>A</italic>/<italic>G</italic>, <italic>A</italic>/<italic>T</italic>, <italic>C</italic>/<italic>G</italic>, <italic>C</italic>/<italic>T</italic>, <italic>G</italic>/<italic>T</italic>, <italic>A</italic>/<italic>C</italic>/<italic>G</italic>, <italic>A</italic>/<italic>C</italic>/<italic>T</italic>, <italic>A</italic>/<italic>G</italic>/<italic>T</italic>, <italic>C</italic>/<italic>G</italic>/<italic>T</italic>, <italic>A</italic>/<italic>C</italic>/<italic>G</italic>/<italic>T</italic>}), we need to introduce allele fractions as unknown parameters of interest to be estimated by maximum likelihood. For example, when <bold><italic>G</italic></bold> has two alleles <italic>G</italic>
<sub><italic>1</italic></sub> and <italic>G</italic>
<sub><italic>2</italic></sub>, <italic>G</italic>
<sub>1</sub>, <italic>G</italic>
<sub>2</sub> &#8712; {<italic>A</italic>, <italic>C</italic>, <italic>G</italic>, <italic>T</italic>}, we denote by <italic>f</italic>
<sub><italic>1</italic></sub> and <italic>f</italic>
<sub><italic>2</italic></sub> the allele fractions of <italic>G</italic>
<sub><italic>1</italic></sub> and <italic>G</italic>
<sub><italic>2</italic></sub> for the individual, respectively. Then:
<disp-formula id="pgen.1005306.e005"><alternatives><graphic id="pgen.1005306.e005g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005306.e005.jpg" /><ns1:math id="M5"><ns1:mrow><ns1:mi>P</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mi mathvariant="bold-italic">G</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mrow><ns1:mo>{</ns1:mo><ns1:mrow><ns1:mtable columnalign="left"><ns1:mtr><ns1:mtd><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>&#215;</ns1:mo><ns1:msub><ns1:mi>f</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>+</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:mrow><ns1:mn>3</ns1:mn></ns1:mfrac><ns1:mo>&#215;</ns1:mo><ns1:msub><ns1:mi>f</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:mspace width="1.5em" /><ns1:mi>i</ns1:mi><ns1:mi>f</ns1:mi><ns1:mspace width="0.25em" /><ns1:msub><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>=</ns1:mo><ns1:msub><ns1:mi>G</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:mtd></ns1:mtr><ns1:mtr><ns1:mtd><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>&#215;</ns1:mo><ns1:msub><ns1:mi>f</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub><ns1:mo>+</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:mrow><ns1:mn>3</ns1:mn></ns1:mfrac><ns1:mo>&#215;</ns1:mo><ns1:msub><ns1:mi>f</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:mspace width="1.5em" /><ns1:mi>i</ns1:mi><ns1:mi>f</ns1:mi><ns1:mspace width="0.25em" /><ns1:msub><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>=</ns1:mo><ns1:msub><ns1:mi>G</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:mtd></ns1:mtr><ns1:mtr><ns1:mtd><ns1:mrow><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:mrow><ns1:mn>3</ns1:mn></ns1:mfrac><ns1:mo>,</ns1:mo><ns1:mspace width="6em" /><ns1:mi>i</ns1:mi><ns1:mi>f</ns1:mi><ns1:mspace width="0.25em" /><ns1:msub><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>&#8800;</ns1:mo><ns1:msub><ns1:mi>G</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>&#8800;</ns1:mo><ns1:msub><ns1:mi>G</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:mtd></ns1:mtr></ns1:mtable></ns1:mrow></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p>Because <italic>f</italic>
<sub><italic>1</italic></sub>+<italic>f</italic>
<sub><italic>2</italic></sub> = 1, we effectively have a single unknown parameter in the above equation. Similarly, we can derive the corresponding formulas when <bold><italic>G</italic></bold> has three or four alleles with more allele fractions as unknown parameters (see <xref ref-type="supplementary-material" rid="pgen.1005306.s016">S1 Text</xref>).</p><p>We next consider all the reads covering the mtDNA site of interest and update the genotype likelihoods as a function of allele fractions using <xref ref-type="disp-formula" rid="pgen.1005306.e003">Eq 2</xref>. When <bold><italic>G</italic></bold> has more than one allele, we maximize the genotype likelihood functions using allele fractions as parameters (the Simplex method for maximization is implemented in our variant calling algorithm). We then use maximized genotype likelihood to estimate posterior probabilities in <xref ref-type="disp-formula" rid="pgen.1005306.e001">Eq 1</xref>.</p><p>We note that a similar idea was recently proposed by Ye et al.[<xref ref-type="bibr" rid="pgen.1005306.ref013">13</xref>] to identify mtDNA heteroplasmies in sequences from 1000 Genomes Project participants. Our publicly available software package was developed independently and further accounts for the circularity of the mtDNA genome (see below) and identifies both homoplasmies and heteroplasmies.</p><p>Even though mtDNA has a circular genome, current studies investigating the mtDNA variation from sequencing data employ a linear genome as reference to align sequence reads. Typically, researchers use the revised Cambridge Reference Sequence (rCRS, Gene Bank number NC_012920) as the &#8220;mtDNA reference&#8221;, with a &#8220;breakpoint&#8221; introduced in the replication control region to &#8220;start&#8221; at position 1 and &#8220;end&#8221; at position 16,569. As a result, sequence reads that link the two &#8220;ends&#8221; (i.e., those that cover the artificial breakpoint in the circular genome) are not aligned and are discarded. Consequently, the variant caller uses an incomplete set of reads to identify variants. We propose a &#8220;double alignment&#8221; strategy, using two linear reference genomes in sequence read alignment (<xref ref-type="supplementary-material" rid="pgen.1005306.s001">S1 Fig</xref>). The conventional rCRS is the reference for the first sequence alignment, and variants are then identified only in the internal region of the genome (e.g., from position 4,000 to position 12,000), skipping the two artificial ends of the rCRS. A &#8220;shifted&#8221; rCRS reference is then created by making a breakpoint in the middle of the circular sequence (position 8,000); the &#8220;shifted&#8221; rCRS thus starts at the original position 8,000 and ends at the original position 7,999 (<xref ref-type="supplementary-material" rid="pgen.1005306.s001">S1 Fig</xref>). The sequence alignment is then repeated using the shifted rCRS as reference. Intuitively, one could simply align the whole-genome sequence reads to the human nuclear genome plus the shifted rCRS reference, but we propose a more efficient approach that combines the unmapped reads and reads mapped to rCRS reference from the first alignment, and then uses them as the input reads for the second alignment. This approach guarantees that all the reads covering the original break point are included in the analysis. We then use the variant caller to call variants between original positions 1 to 4,000 and between 12,000 to 16,569. Variant calling in these two regions is not affected by the breakpoint at position 8,000. Finally, we combine the two sets of called variants from the two alignments to obtain a complete set of variants for the whole mtDNA genome. The pipeline for the &#8220;double alignment&#8221; procedure is outlined in <xref ref-type="fig" rid="pgen.1005306.g001">Fig 1</xref>. When the &#8220;double alignment&#8221; strategy was applied in our SardiNIA data set, the most significant read depth increase was observed in the 100 bp region on each side of the breakpoint, with a 2.1-fold increase (or raw coverage increase of 114X) in average depth (see <xref ref-type="supplementary-material" rid="pgen.1005306.s016">S1 Text</xref> for more discussion on the coverage).</p><p>The mtDNA variant caller was applied to whole-genome sequence data of 2,077 individuals selected among 6,921 participants in the &#8220;SardiNIA&#8221; study of the genetics of quantitative traits in the Sardinian founder population [<xref ref-type="bibr" rid="pgen.1005306.ref008">8</xref>]. The details on the selection of individuals to be sequenced are in <xref ref-type="supplementary-material" rid="pgen.1005306.s016">S1 Text</xref>.</p><p>Sequence data were generated at the University of Michigan Medical School Core Sequencing Lab. DNA was extracted by a standard salting-out method from whole blood samples after a red blood cell lysing step. Libraries were generated from 3&#8211;5 &#956;g of genomic DNA using sample preparation kits from Illumina and New England Biolabs. Paired-end sequence reads (typically, 100 to 120 bp in length) were generated with Illumina Genome Analyzer IIx and Illumina HiSeq 2000 instruments. Samples were sequenced to an average depth of 4.2X.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Reads were aligned to the human reference genome (GRCh37 assembly with decoy sequences, as available in the 1000 Genomes Project ftp site, <ext-link ext-link-type="uri" ns0:href="http://ftp://ftp.1000genomes.ebi.ac.uk">ftp://ftp.1000genomes.ebi.ac.uk</ext-link>) using BWA (version 0.5.9), allowing at most 5 differences (mismatches or gaps), and trimming read tails with average base quality &lt;15. After alignment, base qualities were recalibrated and duplicate reads were flagged and excluded from analysis. We reviewed summary metrics generated using QPLOT and verifyBamId for each aligned sample, to remove samples with low sequencing depth, poor coverage of regions with high or low GC content, or evidence for sample contamination.</p><p>We then extracted from all sequence reads those that were uniquely mapped to the mtDNA reference genome by bwa with a mapping quality score &#8805;20 (i.e., the theoretical probability of wrong alignment &#8804;1%). The mtDNA variant caller was applied only to these mtDNA reads. <xref ref-type="fig" rid="pgen.1005306.g001">Fig 1</xref> outlines the pipeline for aligning sequence reads to the whole genome reference, extracting mtDNA reads, and applying the mtDNA variant caller taking into account the circularity of the mtDNA genome.</p><p>In implementing the likelihood-based model to identify mtDNA variants, we also applied quality control filters to help avoid the inclusion of false variants because of sequencing errors. At a position of interest in mtDNA, we considered only reads with base sequencing error rate &#8804;1% (i.e., recalibrated base quality score &#8805;20). We also applied sequencing depth filters: we required an overall mtDNA median depth &gt; 100 for an individual to be included in the analysis; and at each base of interest for variant calling, we required a raw depth &#8805;40 and a depth &#8805;10 after base quality score filtering at 20. To call a heteroplasmy, we further required that 1) all alleles of the called genotype are observed at least once in both forward and reverse strand sequence reads, and 2) the minor allele fraction (MAF) for an individual is &#8805; 4%. The MAF threshold was chosen based on simulations in which we mimicked the SardiNIA sequencing experiments and simulated similar coverage data with reads of comparable quality scores (see <xref ref-type="supplementary-material" rid="pgen.1005306.s016">S1 Text</xref> for details). An MAF threshold of 4% corresponded to an empirical false discovery rate (i.e., the proportion of false heteroplasmies among all identified heteroplasmies) of 2%. We also used data from a deeply sequenced parent-child trio (~80-fold average coverage for nuclear DNA and ~6,000-fold average coverage for mtDNA) to evaluate the accuracy of variant calling for the same three individuals from the low-pass sequencing data. Using the results from deep-sequencing data as gold standards, we confirmed all the heteroplasmies identified in the three individuals by our variant caller with the 4% MAF cut-off (We found no false negatives in the child or the mother of the trio, but did see one false negative in the father). By contrast, if we lowered the MAF cut-off to a less stringent threshold of 1.6%, we observed an average false discovery rate of 30%. Looking at deep sequencing data from a few individuals could not provide a definite guideline about the MAF cut-off, but supports well the choice of a 4% MAF cut-off for this dataset.</p><p>We have also considered the possibility that nuclear copies of parts of mtDNA sequence (i.e., nuclear mitochondrial DNA, or NUMTs) might be the source of false positives for heteroplasmies. In our analysis, as mentioned above, we included only reads that were uniquely mapped to mtDNA. With further analyses on sampled individual cases, we found that any representation of NUMTs is minimal, and should therefore not restrict the utility of the method (See <xref ref-type="supplementary-material" rid="pgen.1005306.s016">S1 Text</xref> for a detailed discussion).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used publicly available online software HaploGrep (<ext-link ext-link-type="uri" ns0:href="http://haplogrep.uibk.ac.at/">http://haplogrep.uibk.ac.at/</ext-link>) to classify SardiNIA individuals into different haplogroups based on mtDNA, and ANNOVAR[<xref ref-type="bibr" rid="pgen.1005306.ref014">14</xref>] to annotate the called variants on mtDNA and assess whether homoplasmies and heteroplasmies show different distributions in functional categories.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Assuming autosomal and mtDNA are handled and sequenced with no significant differences, average sequencing coverage should be proportional to DNA copy number for autosomal and mtDNA:
<disp-formula id="pgen.1005306.e006"><alternatives><graphic id="pgen.1005306.e006g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005306.e006.jpg" /><ns1:math id="M6"><ns1:mrow><ns1:mfrac><ns1:mrow><ns1:mi>m</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>D</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>A</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>a</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>m</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>D</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>A</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>p</ns1:mi><ns1:mi>y</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>b</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>p</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow></ns1:mfrac><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mi>a</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>l</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>D</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>A</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>a</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>a</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>l</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>D</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>A</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>p</ns1:mi><ns1:mi>y</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>b</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>p</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow></ns1:mfrac></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p>As a proof of principle, we looked at the average depth of coverage across the 22 autosomal chromosomes for 100 randomly selected individuals and observed that as expected, sequencing depth was largely flat across 22 chromosomes for each individual (<xref ref-type="supplementary-material" rid="pgen.1005306.s002">S2 Fig</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Because there are two copies of autosomal DNA in a cell, we could infer the mtDNA copy number by:
<disp-formula id="pgen.1005306.e007"><alternatives><graphic id="pgen.1005306.e007g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005306.e007.jpg" /><ns1:math id="M7"><ns1:mrow><ns1:mi>m</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>D</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>A</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>p</ns1:mi><ns1:mi>y</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>b</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>p</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mi>m</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>D</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>A</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>a</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>a</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>l</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>D</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>A</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>a</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow></ns1:mfrac><ns1:mo>&#215;</ns1:mo><ns1:mn>2</ns1:mn></ns1:mrow></ns1:math></alternatives><label>(3)</label></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used SAMtools (<ext-link ext-link-type="uri" ns0:href="http://samtools.sourceforge.net/">http://samtools.sourceforge.net/</ext-link>) to obtain the coverage of each base in the genome from the aligned bam[<xref ref-type="bibr" rid="pgen.1005306.ref015">15</xref>] files. The average coverages for autosomal DNA and mtDNA were then calculated accordingly.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We applied our computational method to the same whole-genome sequencing data from 2,077 Sardinians and estimated mtDNA copy number for each sample. We also used a NovaQUANT Human Mitochondrial to Nuclear DNA Ratio Kit (EMD Chemicals Inc.) to validate a random group of 18 samples experimentally by qPCR (see <xref ref-type="supplementary-material" rid="pgen.1005306.s016">S1 Text</xref> for more details). We tested for age and gender effects on the mtDNA copy number, and also assessed any association of mtDNA copy number with eleven quantitative traits collected for the cohort that include 5 anthropometric traits (height, weight, BMI, waist circumference, and waist-hip ratio), 2 frailty traits (walking speed and grip strength), and 4 lipid traits (HDL-cholesterol, LDL-cholesterol, total cholesterol, and triglycerides). In addition, POLY software (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/chen/public/software/poly/">http://www.sph.umich.edu/csg/chen/public/software/poly/</ext-link>) was used to estimate the heritability of mtDNA copy number based on the known family structure in the SardiNIA cohort.</p><p>We note that a similar framework has been suggested by Chu and colleagues[<xref ref-type="bibr" rid="pgen.1005306.ref016">16</xref>] to estimate copy number. However, their method identified only reads mapped to mtDNA and counted all the remaining reads as mapped to nuclear DNA. As a result, it would include unmappable reads in calculations and would thus overestimate the nuclear DNA sequence coverage. Furthermore, in calculating the effective length of human genome, their method did not exclude the regions in the nuclear DNA that could not be covered by sequencing. Our computational method avoids those pitfalls and hence should provide more accurate copy number estimates. Here we also validate the method with Q-PCR and implement it to estimate mtDNA copy number in a large-scale population study.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Software programs implementing our methods are freely available at <ext-link ext-link-type="uri" ns0:href="http://lgsun.irp.nia.nih.gov/hsgu/software/mitoAnalyzer/index.html">http://lgsun.irp.nia.nih.gov/hsgu/software/mitoAnalyzer/index.html</ext-link>.</p>