<p>Mouse studies were approved by the Institutional Animal Care and Use Committee at University of California, San Diego (Protocol number: S07365). Every effort was made to minimize the number of animals used, and their suffering.</p><p><italic>Clock</italic><sup><italic>-/-</italic></sup> mice [<xref ref-type="bibr" rid="pgen.1005882.ref010">10</xref>] were crossed to mPer2<sup>Luc</sup> -SV40 reporter mice [<xref ref-type="bibr" rid="pgen.1005882.ref019">19</xref>,<xref ref-type="bibr" rid="pgen.1005882.ref023">23</xref>]. From the heterozygous offspring, we created a double homozygous <italic>Clock</italic><sup><italic>-/-</italic></sup>; <italic>mPer2</italic><sup><italic>Luc</italic></sup> and a <italic>Clock</italic><sup><italic>+/+</italic></sup>; <italic>mPer2</italic><sup><italic>Luc</italic></sup> line, both with the same mixed genetic background. Henceforth, for convenience, these mice will be referred to as <italic>Clock</italic><sup><italic>-/-</italic></sup> and WT. All experiments for this study were carried out in 2&#8211;4 month old male mice, except SCN cultures which were from neonatal (3&#8211;6 day old) male and female mice. Mice were maintained in LD 12:12 cycles (12 h light, 12 h dark, lights on at 06:00 hr) with <italic>ad libitum</italic> access to food and water.</p><p>Primary fibroblasts were obtained from tail or ears. Tissues from each mouse were chopped into small pieces with a scalpel and incubated twice in a 0.25% trypsin solution for 30 min at 37&#176;C. The tryspin digestion was stopped by adding culture medium (high glucose, pyruvate DMEM [Gibco, #11995] with 5% FBS and 50 U/ml penicillin, 50 &#956;g/ml streptomycin). Cells were centrifuged (0.5 x g, 5 min, room temperature) and resuspended in fresh culture medium. About 2x10<sup>5</sup> cells/dish were seeded onto 35 mm dishes and incubated at 37&#176;C with 5% CO<sub>2</sub>. When cells reached ~70% confluence, culture medium was replaced with explant medium formulated for equilibration with 5% CO<sub>2</sub> (high glucose DMEM [Mediatech, Manassas, VA, USA], 14 mM sodium carbonate, 10 mM HEPES, 52 U/ml penicillin, 52 &#956;g/ml streptomycin, 4 mM L-glutamine, 2% B-27 [GIBCO, Grand Island, NY, USA], 0.1 mM luciferin [BioSynth, Itasca, IL, USA]).</p><p>Tissues were isolated and kept in half-frozen Hank&#8217;s Balanced Salt Solution (HBSS). 300 &#956;m brain slices were prepared with a vibratome (Leica VT1200S, Buffalo Grove, IL, USA). Tissues were immediately transferred to tissue culture inserts (EMD Millipore, Billerica, MA, USA) and cultured in 35 mm dishes containing 1 ml of explant medium formulated for equilibration with air (high glucose DMEM [Mediatech, Manassas, VA, USA], 4 mM sodium carbonate, 10 mM HEPES, 52 U/ml penicillin, 52 &#956;g/ml streptomycin, 4 mM L-glutamine, 2% B-27 [GIBCO, Grand Island, NY, USA], 0.1 mM luciferin [BioSynth, Itasca, IL, USA]). When indicated, tissues were treated with 10 &#956;M forskolin for 2 hours in order to enhance rhythmicity.</p><p>Luminescence measurements were taken at 10 min intervals using a LumiCycle luminometer (Actimetrics) that was placed inside a 37&#176;C incubator without CO<sub>2</sub>. Period, peak phases, goodness of fit, and amplitude were determined over 7 days by fitting a sine wave [Sin fit (Damped) for period, phase, and damping constant (days to reach 1/e of initial amplitude), or LM fit (Sin) for amplitude] to 24 h running average baseline-subtracted data using LumiCycle Analysis software (Actimetrics, Wilmette, IL, USA). The first day of measurement was excluded from analyses. Amplitude was normalized to total brightness in order to account for different sizes of brain tissue and technical differences between slices. Explants failing to show significant &#967;<sup>2</sup> periodogram values near 24 h [<xref ref-type="bibr" rid="pgen.1005882.ref024">24</xref>], a goodness of fit &gt;0, or a minimum of two mPer2<sup>Luc</sup> peaks were determined to be arrhythmic and were excluded from further quantification.</p><p>Single-cell mPer2<sup>Luc</sup> measurements were carried out as described elsewhere [<xref ref-type="bibr" rid="pgen.1005882.ref025">25</xref>,<xref ref-type="bibr" rid="pgen.1005882.ref026">26</xref>]. Briefly, a sealed culture in air-equilibrated explant medium was placed on the stage of an inverted microscope (Olympus IX-71, Tokyo, Japan) in a dark, windowless room. A heated lucite chamber, custom-engineered to fit around the microscope stage (Solent Scientific, Segensworth, UK), kept the sample at a constant 36&#176;C. Light from the sample was collected by an Olympus 4x XLFLUOR objective (NA 0.28) and transmitted directly to a cooled charge-coupled-device (CCD) camera (Spectral Instruments, Tucson, AZ, USA) mounted on the bottom port of the microscope. The camera contained a back-thinned CCD thermoelectrically cooled to -90&#176;C with a rated quantum efficiency of 92% at 560 nm. The signal-to-noise ratio was increased by 4 &#215; 4 binning of the 1056 &#215; 1032 pixel array. Images were collected at intervals of 30 min, with 29.5 min exposure duration, for 4&#8211;7 days. Images were acquired and saved to a computer with SI Image SGL D software (Spectral Instruments), and analyzed with MetaMorph (Molecular Devices, Sunnyvale, CA, USA). Period, goodness of fit, and amplitude were determined over 7 days by fitting a sine wave [Sin fit (Damped) for period, phase, and damping constant (time to reach 1/e of initial amplitude), or LM fit (Sin) for amplitude] to 24 h running average baseline-subtracted data using LumiCycle Analysis software (Actimetrics, Wilmette, IL, USA). Cells failing to show significant &#967;<sup>2</sup> periodogram values near 24 h [<xref ref-type="bibr" rid="pgen.1005882.ref024">24</xref>], a goodness of fit &gt;0, or a minimum of two mPer2<sup>Luc</sup> peaks were determined to be arrhythmic and were excluded from further quantification.</p><p>Quantitative real-time PCR (qPCR) was performed with a CFX384 thermocycler system (Bio-Rad, Hercules, CA) with GoTaq SYBR Master Mix (Promega, Madison, WI). Relative quantification of expression levels by a modified &#916;&#916;CT calculation was performed as described [<xref ref-type="bibr" rid="pgen.1005882.ref027">27</xref>]. <italic>&#223;-Actin</italic> was used as a reference gene. PCR primer sequences are listed in <xref ref-type="supplementary-material" rid="pgen.1005882.s007">S2 Table</xref>.</p><p>Primary fibroblasts were transfected with either <italic>Npas2</italic> shRNA (GCTCCGAGAATCGAATGTGAT and GCAAGAACATTCCGAAGTTTA) or scrambled control shRNA (TCGTTTACCACCTCCTGCA) lentiviral particles that also contained a GFP marker sequence. Cells were split and grown until ~70% confluency was reached (max. 48 hours). Medium was reduced and polybrene (4 &#956;g/ml) was added. 10 &#956;l of each <italic>Npas2</italic> and 20 &#956;l of the control virus stock solution were added and cells were incubated for 3 hours at 37&#176;C. Afterwards medium was changed. Seven days after transfection, cells were used for single cell mPer2<sup>Luc</sup> measurements. The efficiency of the <italic>Npas2</italic> knockdown was tested with qPCR after cells were sorted by flow cytometry according to their expression of GFP (<xref ref-type="supplementary-material" rid="pgen.1005882.s005">S5 Fig</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Statistical analyses were conducted using GraphPad Prism. Rhythmicity and phase of qPCR clock gene expression profiles were determined using CircWave v1.4 (developed by Roelof Hut, University of Groningen, Netherlands, <ext-link ext-link-type="uri" ns0:href="http://www.euclock.org/results/item/circ-wave.html">http://www.euclock.org/results/item/circ-wave.html</ext-link>). CircWave v1.4 fits data to a sine wave with added harmonics. Significance of the curve fit is tested against a fitted horizontal line through the overall average. Phase is calculated as the Center of Gravity of the fitted wave form. For raster plots, bioluminescence intensity values were normalized to mean intensity for each cell. Using Gene Cluster 3.0 and Treeview (developed by Dr. Michael Eisen while at Stanford University, USA), data were color coded, with green for positive and red for negative values. Details about statistical tests used for individual experiments are indicated in the figure legends.</p>