<p>To generate <italic>neto-&#946;</italic> alleles, the Minos transposomal element Mi(ET1)Neto[MB07125] was mobilized with Minos transposase [<xref ref-type="bibr" rid="pgen.1005191.ref025">25</xref>]. Several lines with precise excisions and imprecise excisions/small deletions were isolated and characterized by PCR amplification over the deficiencies and DNA sequencing. The genomic fragments removed in various <italic>neto</italic> alleles were as follows: <italic>neto</italic>
<sup><italic>203</italic></sup> X: 13,415,115&#8211;13,418,117, including part of the exon 13 and 14 of the predicted <italic>neto</italic> gene, and <italic>neto</italic>
<sup><italic>204</italic></sup> X: 13,414,477&#8211;13,417,931, containing exons 12, 13 and 14. The <italic>UAS-neto-&#946;</italic> lines (B lines) were generated by insertion of the <italic>neto-&#946;</italic> cDNA (from RE42119) in pUAST vector and germline transformation (BestGene, Inc.). Similarly, for the <italic>UAS-neto&#916;intra</italic> (the H4 line) the Neto coding sequence M<sup>1</sup>-R<sup>471</sup> followed by a short linker (DVPALE) was placed in frame with the eGFP and cloned in pUAST. The <italic>neto-&#945;</italic>
<sup><italic>RNAi</italic></sup> lines were generated by insertion of a <italic>neto-&#945;</italic> specific PCR fragment in pUAST-R57 followed by germline transformation. The PCR primers utilized were: RNAi-F (5&#8242;-AAGGCCTACATGGCCGGACCGGCGAACAAATGGAGGAAGACG-3&#8242;) and RNAi-Rev (5&#8242;-AATCTAGAGGTACCTGATTTTGTGCAGGAACTTGAGG-3&#8242;).</p><p>Other fly stocks used in this study were as follows: <italic>neto</italic>
<sup><italic>36</italic></sup>, <italic>neto</italic>
<sup><italic>109</italic></sup>, <italic>and UAS-neto-&#945; -V5 [<xref ref-type="bibr" rid="pgen.1005191.ref012">12</xref>]; UAS-neto-&#945;</italic> (A lines) [<xref ref-type="bibr" rid="pgen.1005191.ref027">27</xref>]; <italic>neto(CUB1)</italic>
<sup><italic>RNAi</italic></sup> [<xref ref-type="bibr" rid="pgen.1005191.ref014">14</xref>]; <italic>Dp(1</italic>:<italic>3)DC270</italic> [<xref ref-type="bibr" rid="pgen.1005191.ref026">26</xref>]; <italic>pak</italic>
<sup><italic>6</italic></sup>, <italic>pak</italic>
<sup><italic>11</italic></sup>, and <italic>pak</italic>
<sup><italic>myr</italic></sup> [<xref ref-type="bibr" rid="pgen.1005191.ref034">34</xref>]; <italic>dpix</italic>
<sup><italic>1</italic></sup> [<xref ref-type="bibr" rid="pgen.1005191.ref016">16</xref>]; <italic>GluRIIA</italic>
<sup><italic>SP16</italic></sup>, and <italic>Df(2L)cl</italic>
<sup><italic>h4</italic></sup> [<xref ref-type="bibr" rid="pgen.1005191.ref007">7</xref>] (from A. DiAntonio, Washington University). The <italic>G14-Gal4</italic> was obtained from C. Goodman (University of California at Berkeley).</p><p>The rabbit polyclonal anti-Neto-&#946; antibodies were generated against two synthetic peptides: &#946;1 (GRSHYGGLLVTSTAKQP) and &#946;2 (LDDVSNRYYREAVPL) (21st Century Biochemicals) and separated by affinity purification. The rabbit polyclonal anti-GluRIIB and anti-GluRIIC were generated as previously described [<xref ref-type="bibr" rid="pgen.1005191.ref008">8</xref>] against synthetic peptides ASSAKKKKKTRRIEK, and respectively QGSGSSSGSNNAGRGEKEARV (Pacific Immunology Corp).</p><p>To analyze muscle proteins, wandering third instar larvae were dissected, and all tissues except for the body wall (muscle and cuticle) were removed. The body walls were mechanically disrupted and lysed in lysis buffer (50 mM Tris-HCl, 150 mM NaCl, 1% Triton X-100, 1% deoxycholate, protease inhibitor cocktail (Roche) for 30 min on ice. The lysates were separated by SDS-PAGE on 4%&#8211;12% NuPAGE gels (Invitrogen) and transferred onto PVDF membranes (Millipore). <italic>Drosophila</italic> S2 cells were used for the production of recombinant proteins [<xref ref-type="bibr" rid="pgen.1005191.ref012">12</xref>]. Full-length cDNA for <italic>neto-&#946;</italic> from RE42119 was subcloned in pAcPA-based plasmids for expression in S2 cells under the actin promoter. The Neto-&#946; truncation (<italic>neto</italic>
<sup><italic>203</italic></sup>-like) was generated by looping out the deleted fragment using the QuickChange site-directed mutagenesis kit. All constructs were verified by DNA sequencing.</p><p>Primary antibodies were used at the following dilutions: rat anti-Neto-ex [<xref ref-type="bibr" rid="pgen.1005191.ref012">12</xref>], 1:1000; anti-Tubulin (Sigma-Aldrich),1:1000; rabbit anti-Neto-&#946;, 1:1000; rabbit anti-GluRIIC, 1:1000; rabbit anti-PAK, 1:5000 (a gift from Nicholas Harden) [<xref ref-type="bibr" rid="pgen.1005191.ref035">35</xref>]; mouse anti-Dlg (4F3), 1:1000; mouse anti-V5 (Invitrogen), 1:1000. Immune complexes were visualized using secondary antibodies coupled with IR-Dye 700 or IR-Dye 800 followed by scanning with the Odyssey infrared imaging system (LI-COR Biosciences).</p><p>Wandering third instar larvae were dissected as described previously in ice-cooled Ca<sup>2+</sup>-free HL-3 solution [<xref ref-type="bibr" rid="pgen.1005191.ref049">49</xref>,<xref ref-type="bibr" rid="pgen.1005191.ref050">50</xref>]. Embryos at 18h after egg laying (AEL) were dechorinated and genotyped and, after an additional incubation of 2 h at room temperature, were dissected as described previously [<xref ref-type="bibr" rid="pgen.1005191.ref012">12</xref>]. First instar larvae were similarly dissected within 2 h from hatching. The samples were fixed in 4% paraformaldehyde (Polysciences, Inc.) for 20 min or in Bouin&#8217;s fixative (Bio-Rad) for 3 min and washed in PBS containing 0.5% Triton X-100.</p><p>Primary antibodies from Developmental Studies Hybridoma Bank were used at the following dilutions: mouse anti-GluRIIA (8B4D2), 1:100; mouse anti-Dlg (4F3), 1:1000; mouse anti-Brp (Nc82), 1:200; mouse anti-CSP (6D6), 1:1000: mouse anti-&#945;-Spectrin (3A9), 1:50: mouse anti-FasII (1D4),1:10. Other primary antibodies were utilized as follow: rabbit anti-PAK, 1:2000; rat anti-Neto-ex, 1:1000 [<xref ref-type="bibr" rid="pgen.1005191.ref012">12</xref>]; and Cy5- conjugated goat anti-HRP, 1:1000 (Jackson ImmunoResearch Laboratories, Inc.). Alexa Fluor 488-, Alexa Fluor 568-, and Alexa Fluor 647- conjugated secondary antibodies (Molecular Probes) were used at 1:200. All samples were mounted in ProLong Gold (Invitrogen).</p><p>Samples of different genotypes were processed simultaneously and imaged under identical confocal settings in the same imaging session with a laser scanning confocal microscope (CarlZeiss LSM780). All images were collected as 0.2m optical sections and the <italic>z</italic>-stacks were analyzed with Imaris software (Bitplane). To detect positive puncta we used the spot finding Imaris algorithm followed by manual inspection and correction. To quantify fluorescence intensities synaptic ROI areas surrounding anti-HRP immunoreactivities were selected and the signals measured individually at NMJs (muscle 4, segment A3) from ten or more different larvae for each genotype (number of samples is indicated in the graph bar). The signal intensities were calculated relative to HRP volume and subsequently normalized to control. Boutons were counted in preparations double labeled with anti-HRP and anti-Dlg. All quantifications were performed while blinded to genotype. The numbers of samples analyzed are indicated inside the bars. Statistical analyses were performed using the Student t-test with a two-tailed distribution and a two-sample unequal variance. Error bars in all graphs indicate standard deviation &#177;SEM. ***; p&lt;0.001, **; p&lt;0.005, *; p&lt;0.05, ns; p&gt;0.05.</p><p>The standard larval body wall muscle preparation first developed by Jan and Jan (1976) [<xref ref-type="bibr" rid="pgen.1005191.ref051">51</xref>] was used for electrophysiological recordings [<xref ref-type="bibr" rid="pgen.1005191.ref052">52</xref>]. Wandering third instar larvae were dissected in physiological saline HL-3 saline [<xref ref-type="bibr" rid="pgen.1005191.ref049">49</xref>], washed, and immersed in HL-3 containing 0.8 mM Ca<sup>2+</sup> or 0.3 mM Ca<sup>2+</sup> using a custom microscope stage system [<xref ref-type="bibr" rid="pgen.1005191.ref053">53</xref>]. The nerve roots were cut near the exiting site of the ventral nerve cord so that the motor nerve could be picked up by a suction electrode. Intracellular recordings were made from muscle 6. Data were used when the input resistance of the muscle was &gt;5 M&#937; and the resting membrane potential was between -60 mV and -80 mV. The input resistance of the recording microelectrode (backfilled with 3 M KCl) ranged from 20 to 25 M&#937;. Muscle synaptic potentials were recorded using an Axon Clamp 2B amplifier (Axon Instruments) and pClamp 10 software. Following motor nerve stimulation with a suction electrode (100 &#956;sec, 5 V), evoked EJPs were recorded. Four to six EJPs evoked by low frequency of stimulation (0.1 Hz) were averaged. For mini recordings, TTX (1 &#956;M) was added to prevent evoked release [<xref ref-type="bibr" rid="pgen.1005191.ref049">49</xref>]. To calculate mEJP mean amplitudes, 50&#8211;200 events from each muscle were measured and averaged using the Mini Analysis program (Synaptosoft). Minis with a slow rise and falling time arising from neighboring electrically coupled muscle cells were excluded from analysis [<xref ref-type="bibr" rid="pgen.1005191.ref054">54</xref>,<xref ref-type="bibr" rid="pgen.1005191.ref055">55</xref>]. Quantal content was calculated by dividing the mean EJP by the mean mEJP after correction of EJP amplitude for nonlinear summation according to previously described methods [<xref ref-type="bibr" rid="pgen.1005191.ref056">56</xref>,<xref ref-type="bibr" rid="pgen.1005191.ref057">57</xref>]. Corrected EJP amplitude = E[Ln[E/(E&#8212;recorded EJP)]], where E is the difference between reversal potential and resting potential. The reversal potential used in this correction was 0 mV [<xref ref-type="bibr" rid="pgen.1005191.ref057">57</xref>,<xref ref-type="bibr" rid="pgen.1005191.ref058">58</xref>]. Data are presented as mean &#177;SEM, unless otherwise specified; EJP amplitudes and quantal contents after the nonlinear correction are shown. Student T-test was used to assess statistically significant differences among the genotypes.</p><p>Wandering third instar larvae were dissected in physiological saline HL-3 saline and fixed for 30 min on dissection plate in fixation buffer (0.1 M Sodium Cacodylate buffer, pH7.2; 2 mM MgCl<sub>2</sub>; 1% glutaraldehyde; 4% paraformaldehyde). The samples were trimmed to include only the abdominal segments A2 and A3, transferred in a 1.5mL Eppendorf tube, fixed overnight at 4&#176;C, then washed extensively with 0.1 M Sodium Cacodylate buffer with 132 mM Sucrose, pH 7.2. The samples were further processed and analyzed according to published protocols [<xref ref-type="bibr" rid="pgen.1005191.ref059">59</xref>] at the Microscopy and Imaging Facility, NICHD.</p>