<p>MTD-GAL4, UASp-mCherry-Atg8a <italic>(y<sup>1</sup> w<sup>1118</sup>; P{UASp-mCherry-Atg8a}2; Dr<sup>1</sup>/TM3, Ser<sup>1</sup>)</italic>, UASp-GFP-mCherry- Atg8a (<italic>y<sup>1</sup> w<sup>1118</sup>; P{UASp-GFP-mCherry-Atg8a}2, wdr24<sup>1</sup></italic> (<italic>CG7609<sup>1</sup></italic>), UAS-Tsc1 RNAi (<italic>y<sup>1</sup> sc* v<sup>1</sup>; P{TRiP.GL00012}attP2</italic>), HS-FLP; Ubi-GFP FRT82B/TM3 and Df(3R)BSC547 lines were obtained from the Bloomington Stock Center. The <italic>mio<sup>1</sup></italic>, <italic>mio<sup>2</sup></italic>, <italic>seh1<sup>&#916;15</sup></italic>, <italic>nprl <sup>RNAi</sup></italic> and <italic>nprl3 <sup>RNAi</sup></italic> lines were described previously [<xref ref-type="bibr" rid="pgen.1006036.ref018">18</xref>,<xref ref-type="bibr" rid="pgen.1006036.ref064">64</xref>]. GFP-Lamp1 <italic>(w<sup>1118</sup>; P{W+, Tub&gt;GFP-Lamp1}1/</italic>CyO; TM6b<italic>, Hu boss<sup>1</sup>/Sb boss<sup>1</sup>)</italic> was kindly provided by Helmut Kramer (UT Southwestern). Nanos-Gal4 <italic>(P{NANOS GAL4 VP-16}</italic>, <italic>yw;</italic> D/TM3, Ser, Sb<italic>)</italic> was kindly provided by Sharon Bickel (Dartmouth College)<italic>. atg7<sup>d77</sup>/</italic>Cyo-GFP <italic>and atg7<sup>d14</sup>/</italic>CyO-GFP were kindly provided by Thomas P. Neufeld (University of Minnesota) [<xref ref-type="bibr" rid="pgen.1006036.ref041">41</xref>]. All fly stocks were maintained on JAZZ-mix <italic>Drosophila</italic> food (Fisher Scientific) at 25&#8451;.</p><p>The <italic>wdr24</italic> coding region was cloned into a pENTR-1A vector (Invitrogen). The <italic>mCherry</italic> coding region was inserted into pENTR-wdr24 plasmid. The pENTR-<italic>wdr24</italic> and pENTR-<italic>wdr24-mCherry</italic> plasmids were recombined with pPGW vector (DGRC) and pPFHW vector (DGRC) separately and to generate UASp-<italic>GFP</italic>-<italic>wdr24 and</italic> UASp<italic>-wdr24-mCherry</italic> using Gateway LR Clonase II Enzyme (Invitrogen). UASp-<italic>GFP-wdr24</italic> and UASP-<italic>wdr24-mCherry</italic> plasmids were used to generate transgenic lines (Best gene Inc). All primers used for PCR amplification are listed in <xref ref-type="supplementary-material" rid="pgen.1006036.s015">S2 Table</xref>.</p><p>Ovaries from the transgenic flies that stably express Mio proteins tagged with FLAG and HA were homogenized in 50 mM Tris-HCl (pH 7.4), 300 mM NaCl, 5 mM EDTA, and 1% Triton X-100 supplemented with proteinase inhibitor cocktail (Roche) [<xref ref-type="bibr" rid="pgen.1006036.ref027">27</xref>]. Cell lysates were cleared by centrifugation at 15,000 &#215; <italic>g</italic> for 15 min and Proteins were purified using FLAG-HA tandem affinity purification kit (Sigma) Proteins were precipitated with 10% trichloroacetic acid (TCA), washed with acetone, air-dried, and analyzed by liquid chromatography (LC)/MS at the Taplin MS facility (Harvard Medical School).</p><p>S2 cell transfection and immunoprecipitation were performed as previously described [<xref ref-type="bibr" rid="pgen.1006036.ref018">18</xref>]. 2&#8201;&#956;g GFP antibody (Roche), 20&#8201;&#956;l protein G agarose (Millipore) and 10&#8201;&#956;l protein A agarose (Roche) were used for each experiment.</p><p>Whole flies or HeLa cells were lysed in RIPA buffer containing complete protease inhibitors and phosphatase inhibitors (Roche). Western blots were performed as described previously [<xref ref-type="bibr" rid="pgen.1006036.ref018">18</xref>]. Antibodies were used at the following concentrations: rabbit anti-P-S6K T398 at 1:1000 (Cell Signaling), guinea pig anti-S6K at 1:10,000 (24), mouse anti-actin at 1:10,000 (Abcam), rabbit anti-LC3A/B at 1:1000 (Cell Signaling), rabbit anti-P-S6K T389 at 1:1000 (Cell Signaling), rabbit anti-S6K at 1:1000 (Cell Signaling), rabbit anti-GAPDH at 1:3000 (Cell Signaling), rabbit anti-P-4E-BP1at 1:1000 (Cell Signaling), rabbit anti-4E-BP1 at 1:1000 (Cell Signaling), rabbit anti-GFP at 1:500 (Cell Signaling), rabbit anti-SQSTM1/p62 at 1:500 (Cell Signaling), Mouse anti- SQSTM1/p62 at 1:1000 (Novus Biologicals), rabbit anti-WDR24 at 1:1000 (Novus Biologicals) and goat anti-Cathepsin D at 1:500 (Santa Cruz). The band intensity was quantified using Image J analysis tool (NIH).</p><p>Immunofluorescence was performed as described [<xref ref-type="bibr" rid="pgen.1006036.ref065">65</xref>], using the following antibodies: goat anti-GFP FITC conjugated (1:400, Abcam), rabbit anti-LC3A/B (1:1000, Cell Signaling), mouse anti- SQSTM1/p62 (1:100, Novus Biological), rabbit anti-TFEB (1:100, Cell Signaling), rabbit anti-CathD (1:100, Calbiochem) and rabbit anti-Atg8a (1:200, Abcam). Anti-rabbit and anti-mouse Alexa Fluor secondary antibodies (Invitrogen) were used at 1:1000. Nuclei were visualized by staining the DNA with DAPI or Hoechst 33324 (Invitrogen). Images were acquired using a Leica TCS SP5 confocal microscope. Live cell images were obtained as previously described [<xref ref-type="bibr" rid="pgen.1006036.ref018">18</xref>].</p><p>To generate <italic>wdr24<sup>1</sup></italic> homozygous clones, HS-FLP; Ubi-GFP FRT82B/<italic>wdr24<sup>1</sup></italic> FRT82B third instar larvae were heat shocked for 1hr in a 37&#176;C water bath two times per day. The adult females were collected and aged for 5 to 7 days. Fat bodies or ovaries from adult flies were dissected and stained with GFP antibody (Abcam) and Hoechst 33324 (Invitrogen). Homozygous <italic>wdr24<sup>1</sup></italic> clones were marked by the absence of GFP expression.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To generate knock out <italic>nprl3</italic> flies, guide RNAs (gRNA) that target <italic>nprl3</italic> were designed using the online CRIPR design tool (<ext-link ext-link-type="uri" ns0:href="http://crispr.mit.edu/">http://crispr.mit.edu/</ext-link>). To make the deletion mutants, two different gRNAs were cloned into pBFv-U6.2B as previously described [<xref ref-type="bibr" rid="pgen.1006036.ref066">66</xref>]. pBFv-U6.2B-<italic>nprl3</italic> plasmids were injected into y[<xref ref-type="bibr" rid="pgen.1006036.ref001">1</xref>], vas-Cas9, w<sup>[1118]</sup> embryos. All the oligonucleotides used for cloning and screening are listed in <xref ref-type="supplementary-material" rid="pgen.1006036.s015">S2 Table</xref>.</p><p>To generate knock out <italic>WDR24</italic> HeLa cells, two pairs of oligonucleotide encoding the guide RNAs were cloned into px330 vector [<xref ref-type="bibr" rid="pgen.1006036.ref042">42</xref>]. On day one, 50,000 cells were seeded into 24-well plate. Each well was transfected with a total of 1 &#956;g of a pair of <italic>px330</italic> guide constructs. The third day, cells were pooled into 10 cm dishes with 200 cells each dish. After 10 days, all the clones were collected and seeded into 24 well plates. Cells were grown and expanded, and the positive colonies were identified by PCR and sequencing. All primers used for PCR amplification are listed in <xref ref-type="supplementary-material" rid="pgen.1006036.s014">S1 Table</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">HeLa cells and knock out cells were cultured at 37&#176;C, 5% CO2 in DMEM medium (Life Technology) supplemented with 10% fetal bovine serum (FBS, Life Technology), 50&#8201;U/ml penicillin and 50&#8201;&#956;g/ml streptomycin. Plasmid transfection was performed using Lipofectamine 2000 (Life Technologies). <ext-link ext-link-type="uri" ns0:href="http://www.addgene.org/46335/">HAWDR24 pRK5</ext-link> (Plasmid #46335) was obtained from Addgene. For the DQ BSA Red (Life Technology) assay, Cells were incubated for 4 h with DQ BSA (10 &#956;g/ml) and then washed twice with HBSS. Subsequently the cells were incubated in the starvation medium (HBSS, starvation media; Life Technology) to induce autophagy. After 1 hour the DQ BSA fluorescence was detected using a Leica TCS SP5 confocal microscope.</p><p>Cells grown on coverslips were fixed in 2% (wt/vol) formaldehyde and 2% (wt/vol) glutaraldehyde in 0.1 M sodium cacodylate (pH 7.4), postfixed in 1% aqueous OsO4, and stained en bloc with 2% (wt/vol) uranyl acetate. Upon dehydration and embedding in EMBed-812 (EM Science, Horsham, PA), the coverslips were removed by hydrofluoric acid, cells were thin-sectioned parallel to the glass, and sections were stained with uranyl acetate. The samples were examined on an FEI Tecnai 20 transmission electron microscope operated at 80 kV, and images were recorded on a Gatan Ultrascan CCD camera.</p><p>Lysosomal pH measurements were performed as describe [<xref ref-type="bibr" rid="pgen.1006036.ref067">67</xref>]. In brief, cells were stained with 1&#956;M LysoSensor Green DND-189 in DMED regular medium for 20 min at 37&#176;C, 5% CO2. Subsequently, cells were washed with PBS twice and analyzed by a microplate reader (485/530 nm) in triplicate. Lysosomal pH quantification was performed using LysoSensor Yellow/Blue DND-160. Cells were labeled with 1&#956;M LysoSensor Yellow/Blue DND-160 for 30 min at 37&#176;C, 5% CO2 in DMEM medium and then washed twice with PBS. To generate a calibration curve cells were treated for 10 min with 10&#956;M monensin and 10&#956;M nigericin in 25mM MES calibration buffer, pH 3.5&#8211;6.0, containing 5 mM NaCl, 115 mM KCl, and 1.2 mM MgSO. The fluorescence was measured with a microplate reader (340/440 nm and 380/530 nm) at 37&#176;C. These experiments were performed in triplicate.</p>