<p>The DISC and REP1 cohorts were comprised of adults meeting the consensus criteria for the diagnosis of NF1 <xref ref-type="bibr" rid="pgen.1004575-XX1">[63]</xref>, <xref ref-type="bibr" rid="pgen.1004575-Gutmann1">[64]</xref> who were willing to travel to the NIH Clinical Center in Bethesda, Maryland and who had both living biological parents willing to donate a blood sample. The parents did not need to be affected with NF1. Exclusion criteria for probands included: 1) any past or present history of radiation therapy, chemotherapy or biologic agents that might be expected to alter the natural history of neurofibroma growth, 2) any history of surgery to remove multiple neurofibromas or spinal neurofibromas, 3) cognitive delay that would preclude sedation to obtain an MRI, 4) presence or suspected presence of surgical hardware (<italic>e.g.</italic>, Harrington rods) or metallic objects that would preclude MRI imaging and 5) inability or unwillingness to tolerate an extended (one hour or more) MRI protocol. Study participants were recruited via a variety of means (<italic>e.g.</italic>, Google advertising, letters to NF1 clinics) from throughout the United States. Travel and lodging costs were covered by the protocol. Lymphoblastoid cell lines (LCLs) from the first 79 participants (&#8220;EXPR&#8221;) were used in the gene expression screen to identify putative modifiers. For tests of association of variants in putative modifiers identified in the EXPR screen, 99 participants were used as a discovery cohort (DISC) where 70 samples of the EXPR cohort were included in the DISC sample. An additional independent 33 and 81 participants were used as validation cohorts (REP1 and REP2, respectively). This study was approved by the National Human Genome Research Institute and National Cancer Institute institutional review boards and all participants provided written, informed consent.</p><p>We sought to quantify the NF1 phenotype in a comprehensive two-day visit to the NIH Clinical Center. A single observer (DRS) performed a history and physical exam (with measurements), Wood's lamp exam, slit-lamp exam, and collected photographs of the skin. NF1-specific abnormalities were noted (<italic>e.g.</italic>, presence/absence of intertriginous freckling, bony abnormalities, dysmorphic features) and a clinical assessment of the probability of mosaic NF1 was made. Whole-body cutaneous neurofibroma burden (lesions projecting above the skin) was estimated within a set of ranges (0, 1&#8211;10, 11&#8211;50, 51&#8211;100, 101&#8211;500, 500+). In addition, a paper frame with a 100 cm<sup>2</sup> cut-out at the center was placed on the mid-back, abdomen and left thigh of each participant and a photograph was taken. Within the 100 cm<sup>2</sup>, all protruding cutaneous neurofibromas greater than 2 mm were counted, marked with water-soluble ink and re-photographed. The number of cherry hemangiomas, an under-recognized feature associated with NF1 <xref ref-type="bibr" rid="pgen.1004575-Wertelecki1">[65]</xref>, <xref ref-type="bibr" rid="pgen.1004575-Wertelecki2">[66]</xref>, was also counted within each frame at the three different sites. The number, size and distribution of CALM and other dermatologic abnormalities were counted, measured and mapped with a Wood's lamp and ruler in a darkened room. The CALM count was defined as the total number of caf&#233;-au-late spots greater than 5 mm in any dimension. A slit-lamp exam was used to enumerate and photograph Lisch nodules in the eye, as previously described <xref ref-type="bibr" rid="pgen.1004575-Boley1">[58]</xref>. From the physical exam we measured height, weight and head circumference. Growth charts specific for the NF1 population (recruited at an Italian center) were used to determine centile rankings of height and weight <xref ref-type="bibr" rid="pgen.1004575-Clementi1">[67]</xref>. Centile charts for adult head circumference (adjusting for gender and height) were also used for NF1-affected individuals <xref ref-type="bibr" rid="pgen.1004575-Bushby1">[68]</xref>. We obtained demographic and self-reported ethnicity data, a pedigree and associated data (parity, presence of consanguinity, age of parents at birth), subject and parental heights, an MRI of the spine and clinical photographs, and referred participants to the dental clinic at the NIH Clinical Center for a cephalogram, panograph, and intra-oral photography. All participants received genetic counseling. Blood samples for DNA extraction, RNA extraction (PaxGene tubes) and for LCL production were drawn on morning of the second day of the evaluation.</p><p>Patients with NF1 were enrolled in the &#8220;Neurofibromatosis Type 1 Natural History Study&#8221; (NCT00924196), approved by the NCI Institutional Review Board. Patients or their guardians were provided written informed consent. Eligibility criteria included a clinical diagnosis of NF1 or presence of an <italic>NF1</italic> germline mutation. A detailed skin evaluation at the time of enrollment by a single observer (AMB) was used. The number, size and distribution of CALM &gt;5 mm in any dimension were recorded. They were measured with a ruler and documented on a standard form utilized on the natural history study.</p><p>All LCLs were established from peripheral white blood cells at the Lombardi Comprehensive Cancer Center, Georgetown University, using standard procedures. Cells were stored in liquid nitrogen until needed for an experiment. To minimize batch effects, 79 cell lines were thawed on the same day and seeded at initial density of 500,000 cells per mL in 12-well plates. The cultures were maintained in an incubator at 37&#176;C with 5% CO<sub>2</sub> in RPMI 1640 medium supplemented with 2 mM L-glutamine, 100 Units/mL penicillin, 100 mg/mL streptomycin and 15% heat-inactivated fetal bovine serum. The cells were fed every other day and harvested on the same day after 10 days of culturing. The cell densities in the fastest and slowest growing cultures were 1.9 and 1.1 million cells/mL on the day of harvesting, respectively. The majority of LCLs exhibited similar growth rates and were at density of 1.3 to 1.7 million cells/mL at the time of harvesting. For harvesting, the cells were transferred into 15 mL tubes, spun at 400 g for 5 min at room temperature, washed once with PBS (no Ca<sup>++</sup> or Mg<sup>++</sup>), spun again, and the pellets were lysed in 1 mL of Trizol reagent. The lysates were stored at &#8722;80&#176;C prior to RNA extraction. All reagents were from Life Technologies (Grand Island, NY, USA).</p><p>For RNA isolation, Trizol cell lysates were mixed with chloroform (1/5 of lysate volume), vortexed for one minute and centrifuged in a table-top centrifuge at 13,000 rpm for 15 min at 4&#176;C. The aqueous phase containing RNA was mixed with an equal volume of 70% ethanol and immediately loaded onto RNeasy mini columns (Qiagen, Valencia, CA, USA), with subsequent steps performed as per the manufacturer's protocol. The RNA quality was estimated on a 2100 Bioanalyzer, RNA 6000 Nano Chips (Agilent, Santa Clara, CA, USA). Samples with RNA integrity number (RIN) of 8.0 and above were used for further analysis. For microarray analysis of RNA, all reagents, consumables, lab-ware, instruments, and software were obtained from Illumina, Inc (San Diego, CA, USA) unless otherwise indicated. RNA amplification/labeling, microarray hybridization, and microarray washing/staining and scanning procedures were done according to the Illumina protocols without modifications. Amplified biotinylated cRNA (1.5 &#181;g) was hybridized to HumanRef-8_v2 Sentrix BeadChips. Samples were hybridized to microarrays at 55&#176;C for 16&#8211;17 hours. Microarrays were washed to remove non-specifically bound cRNA, stained with 1 mg/mL Streptavidin-Cy3 (GE Healthcare, Piscataway, NJ, USA), dried, and scanned in an Illumina BeadStation 500 scanner. Image acquisition and initial image analysis were done with Illumina BeadScan and BeadStudio applications. Raw expression data were quintile normalized, background subtracted, floored to remove negative values and transformed by calculating logarithm, base 2, for each value (for better approximation to a normal distribution).</p><p>Simple linear regression analyses between specific NF1 quantitative phenotypes (height, head circumference, total number of CALM count, cutaneous neurofibroma burden, Lisch nodule count and cherry hemangioma count) and expression values obtained for each individual in the EXPR set (<xref ref-type="table" rid="pgen-1004575-t001">Table 1</xref>) were performed for each of the 22,177 transcripts on the microarray. The FDR calculation procedure was applied to correct for multiple testing <xref ref-type="bibr" rid="pgen.1004575-Benjamini1">[69]</xref>. All phenotype-transcript regression pairs with an FDR below 0.3 were considered significant. The output was further filtered by subtracting phenotype-transcript pairs with expression level of transcripts below 6 (mean log<sub>2</sub>), expression range (difference between maximum and minimum expression) below 2 and considering biological significance of the candidate genes. In some cases, genes with an expression level below 6 and an expression range below 2 were still considered for validation because of their biological importance.</p><p>Filtered transcripts with putative phenotype/expression correlates (<xref ref-type="supplementary-material" rid="pgen.1004575.s002">Table S1</xref>: &#8220;Set of 80&#8221;) were investigated for outliers by generating scatter plots of quantitative phenotype <italic>vs.</italic> transcript expression values. Twenty-one select transcripts (<xref ref-type="supplementary-material" rid="pgen.1004575.s002">Table S1</xref>: &#8220;Set of 21&#8221;) plus an endogenous control (<italic>GAPDH</italic>) were interrogated by qPCR in all 79 samples that were analyzed on microarrays on 384-well microfluidic cards (Applied Biosystems, Carlsbad, CA, USA). The microfluidic cards were processed and analyzed per the manufacturer's instructions without modifications. Relative expression of each gene was calculated using the standard &#8220;double delta Ct&#8221; method, per the manufacturer's protocol. Simple linear regression analysis of the qPCR expression values and corresponding quantitative phenotypes was performed as described above. For a given transcript, correlation of qPCR expression with phenotype with a nominal <italic>p</italic> value less than 0.05 was considered significant.</p><p>Coding and limited evolutionarily conserved non-coding sequences of <italic>MSH6</italic>, <italic>MSH2</italic>, <italic>MLH1</italic>, <italic>MED21</italic> and <italic>DPH2</italic> were sequenced from germline DNA using the dideoxynucleotide chain termination method (Sanger). The genes <italic>MSH6</italic> and <italic>MED21</italic> were sequenced because of prior validation by qPCR. We included <italic>MSH2</italic> and <italic>MLH1</italic> because the protein products of these genes are known to associate with <italic>MSH6</italic> in functional MMR complexes. Despite not being verified by qPCR, <italic>DPH2</italic> was included because of its biological significance. The concentration of genomic DNA (gDNA) used in sequencing was determined using a DyNA Quant 200 fluorometer (Hoefer, Holliston, MA USA) and dsDNA-specific Hoechst Dye 22358 according to the manufacturer's protocol. The gDNA sample was then tested for functionality in PCR reactions with positive and negative control primers:</p><p>Pos_For: <named-content content-type="gene">TGTAAAACGACGGCCAGTATCCCACTGTTAGGAGAACTGC</named-content>
</p><p>Pos_Rev: <named-content content-type="gene">CAGGAAACAGCTATGACCGGTCAGGAAAGGGACACAGATA</named-content>
</p><p>Negative control primers are the forward and reverse sequencing primers to lac-Z of M13:</p><p>M13_For: <named-content content-type="gene">TGTAAAACGACGGCCAGT</named-content>
</p><p>M13_Rev: <named-content content-type="gene">CAGGAAACAGCTATGACC</named-content>
</p><p>To each gDNA sample, a trace amount of a plasmid with a unique non-human insert was added to serve as a biological barcode; the identifying inserts were amplified and checked using the universal sequencing primers above. The gDNAs were diluted to a working concentration of 2.5 ng/&#181;L. To amplify gDNA, primers were obtained from Eurofins MWG Operon (Huntsville, AL, USA) in individual tubes and reconstituted to 100 &#181;M in 10 mM TRIS, pH 8.0, 0.1 mM EDTA. The primers pairs were tested at a concentration of 0.16 &#181;M each in 10 &#181;L PCR reactions containing KAPA 2G Fast HS ReadyMix PCR Kit (KAPA Biosystems, Woburn, MA, USA) and 5 ng of control human DNA (Coriell Institute, Camden, NJ, USA). Cycling conditions: 1) activate enzyme at 95&#176;C for 3 min, 2) 40 cycles at 95&#176;C for 10 sec, 60&#176;C for 10 sec and then 72&#176;C for 30 sec and, 3) hold at 10&#176;C. A 5 &#181;L aliquot of the PCR reaction was examined by agarose gel to assess multiple or missing bands. The PCR products were then diluted to 0.4 ng/&#181;L and sequenced in 6 &#181;L reactions using M13 universal forward and reverse primers and BDT version 3.1 (Applied Biosystems) using standard ABI protocols. The reactions were then analyzed on 3730 DNA Sequencers (Applied Biosystems). The sequence traces were individually inspected for quality. Primer pairs that did not lead to high-quality traces were retested using one additional control DNA. Primers failing both rounds were redesigned. PCR amplification of amplimers was performed in 10 &#181;L reactions in 384-well plates, as described above. Prior to sequencing, the PCR products were diluted to 0.4 ng/&#181;L. Sequencing was performed on an Applied Biosystem 3730 Sequencer using BigDye Terminator version 3.1. Three &#181;L of diluted PCR products were used in sequencing reaction volumes of 6 &#181;L. Sequencing primer sequences are as above. Reaction cleanup was accomplished through alcohol precipitation. Reaction precipitates are dissolved in 10 &#181;L water immediately before sequencing. All genomic coordinates reference the hg18 (March 2006) build.</p><p>For genotyping <italic>MSH6</italic> SNP rs1800934 in an independent set of germline DNA from 33 samples, a 632 bp amplimer was generated from gDNA using: <named-content content-type="gene">5&#8242;-GTAGTCCGCCCACCTAAGC</named-content> (forward) and <named-content content-type="gene">5&#8242;-CCCTAGCTCTCTACTTCTTACCAAAA</named-content> (reverse). The primers were appended with universal sequences at their 5&#8242;-ends: <named-content content-type="gene">5&#8242;-TGTAAAACGACGGCCAGT</named-content> (forward), <named-content content-type="gene">5&#8242;-CAGGAAACAGCTATGACC</named-content> (reverse). PCR and sequencing was performed as described above.</p><p>For genotyping <italic>DPH2</italic> SNPs rs4660761 and rs7161, we used genotyping calls obtained from Illumina Human OmniQuad-1M SNP-arrays. SNP-array analysis was done according to the manufacturer's protocol using GenomeStudio (v. 2010.2) software (Illumina).</p><p>A targeted, multiplex PCR primer panel was designed using the custom Ion Ampliseq Designer v3.0 (Thermo Fisher Scientific, Life Technologies, Carlsbad, CA, USA). The primer panel covered 11 kb of sequence that includes the specific variants of interest in the <italic>MSH6</italic> and <italic>DPH2</italic> loci. Each site was 100% covered in the design. Average amplicon size was 225 bp. Sample DNA was amplified using this custom Ampliseq primer pool, and libraries were prepared following the manufacturer's Ion Ampliseq Library Preparation protocol (Life Technologies, Carlsbad, CA, USA). Individual samples were barcoded, pooled, templated, and sequenced on the Ion Torrent PGM Sequencer using the Ion PGM Template OT2 200 and Ion PGM Sequencing 200v2 kits per manufacturer's instructions. Mean read length after sequencing was 159 bp.</p><p>The CALM count trait was used both as the untransformed (unt) and as the log-transformed trait (log) by log<sub>10</sub>(x+10). Single nucleotide variants (SNVs) from the five selected candidate genes (<italic>MSH6</italic>, <italic>MSH2</italic>, <italic>MLH1</italic>, <italic>MED21</italic> and <italic>DPH2</italic>) were filtered based on the with the following: 1) if the polyphred score for an SNV was less than 99, the genotype was deemed &#8220;missing&#8221;; 2) if a sample had greater than 30% of its SNVs &#8220;missing&#8221; then the sample was excluded; 3) if an SNV had a greater than 20% &#8220;missing&#8221; rate in all of the samples, the SNV was excluded; and 4) all monomorphic SNVs were excluded. In those cases in which family data were available, SNVs were checked for Mendelian segregation with PedCheck <xref ref-type="bibr" rid="pgen.1004575-OConnell1">[70]</xref>. Hardy-Weinberg equilibrium (HWE) proportions were tested with PEDSTATS <xref ref-type="bibr" rid="pgen.1004575-Wigginton1">[71]</xref> on 70 unrelated individuals. Two SNVs not in HWE (<italic>P</italic> value&lt;0.03) were flagged but retained for analysis since removing SNVs not in HWE in highly-ascertained samples may remove causative SNVs. Self-reported European-American samples were included for further analyses. After filtering, there were a total of 91 individuals with 118 SNVs.</p><p>Simple linear regression was performed on each SNV after adjusting for age and sex. &#8220;Tiled Regression&#8221; as implemented in TRAP (Tiled Regression Analysis Package) <xref ref-type="bibr" rid="pgen.1004575-Sung1">[72]</xref> was used to identify the set of independent significant variants considering all the SNVs in the sample that affected the number of CALM after pre-adjusting for age and sex. Both simple linear regression and tiled regression were performed under two models, additive and dominant for both the untransformed and log-transformed traits. Briefly, in tiled regression, the genome is divided into independent segments based on predefined regions called tiles. In this study, tiles were defined by hotspot-based regions, delimited by the location of recombination hotspots in Human Genome Sequence build 36, yielding 5 independent tiles. Each tile was first analyzed with multiple linear regression of the trait on all SNVs in the tile and by simple linear regression of the trait on each SNV separately. Only those tiles for which the overall multiple linear regression showed a significant relationship to trait variation at the level of 0.2, or in which simple linear regression on any single SNV was significant at the level of 0.05, were retained for subsequent analyses. A forward stepwise regression with backward look then was performed within each tile to select the important individual independent SNVs identified with a critical level of 0.05 for entry and for retention in the model. Thereafter, the SNVs retained from each tile were combined across tiles for higher order stepwise regressions at chromosome and then whole genome levels using the same critical values. The end result was a multiple linear regression model that included the set of variants independently contributing to trait variation.</p><p>Genotypes for 118 SVs, including both 58 common and 60 rare variants (defined by minor allele frequency &lt;0.05), were used as provided (un-collapsed), and with the rare variants (RVs) collapsed. Several coding schemes were considered: 1) the number of minor alleles (add) for each common variant (CV) and for each un-collapsed RV, 2) the presence or absence of the minor allele (dom) for each SNV and for each un-collapsed RV, 3) the number of minor alleles (add) for each CV and, with a new collapsed variant that was coded as proportion of a minor allele at any RV; in other words, collapsing multiple RVs into a single region-wide variant within hotspot-based region definition.</p><p>To confirm the association of SNVs in the DISC sample set with CALM count, we genotyped the variants in germline DNA in an independent set of 33 samples (REP1) and an additional independent set of 81 samples (REP2). Since none of the significant SNVs in the DISC set were significantly associated with CALM count by simple linear regression at the level of 0.05 based on 29 and 62 European-American samples in REP1 and REP2, respectively (<xref ref-type="table" rid="pgen-1004575-t002">Table 2</xref>), we performed additional analyses. In the meta-analysis, p-values of these three (DISC, REP1, REP2) datasets were combined using Liptak's method <xref ref-type="bibr" rid="pgen.1004575-Liptak1">[73]</xref> by weighting each p-value by its square root of the sample size (<xref ref-type="table" rid="pgen-1004575-t002">Table 2</xref>). In the mega-analysis, three (DISC, REP1, REP2) datasets including 180 samples were combined and simple linear regression was performed on each SNV by adjusting for age, sex and each dataset (<xref ref-type="table" rid="pgen-1004575-t002">Table 2</xref>). Tiled regression was not performed in the replication study since the method requires genotyping all variants, not just markers of interest. We did not attempt to validate rare variants due to limited size of the additional set.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To explore whether SNPs rs4660761 and rs7161 might have potential regulatory functions in skin cells (including melanocytes), we used custom tracks on the UCSC Genome browser (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu">http://genome.ucsc.edu</ext-link>) to screen Roadmap and ENCODE data containing the implicated SNP regions for evidence for regulatory relevance <xref ref-type="bibr" rid="pgen.1004575-Rosenbloom1">[74]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004575-Chadwick1">[76]</xref>, such as overlapping with chromatin marks and interactions, CpG-site methylation and transcription factor binding motifs. We also used the online tools HaploReg (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/mammals/haploreg/haploreg.php">http://www.broadinstitute.org/mammals/haploreg/haploreg.php</ext-link>) and RegulomeDB (<ext-link ext-link-type="uri" ns0:href="http://regulome.stanford.edu">http://regulome.stanford.edu</ext-link>) as a complementary analysis and to confirm the location of each SNP in relation to annotated protein-coding genes and/or non-coding RNA (ncRNA) genes.</p><p>The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products or organizations imply endorsement by the U.S. Government.</p>