<p>Fly lines were generated by standard germline transformation of pUAST-based plasmids containing various <italic>neto</italic> constructs (BestGene, Inc). Other stocks used in this study were as follows: <italic>neto</italic> null and hypomorph alleles, <italic>neto</italic>
<sup><italic>36</italic></sup> and respectively <italic>neto</italic>
<sup><italic>109</italic></sup> [<xref ref-type="bibr" rid="pgen.1004988.ref023">23</xref>], <italic>neto</italic>
<sup><italic>RNAi</italic></sup> [<xref ref-type="bibr" rid="pgen.1004988.ref033">33</xref>], <italic>G14-Gal4</italic> and <italic>MHC-Gal4</italic> (obtained from C. Goodman, University of California at Berkeley), <italic>da-Gal4</italic> (BL-5460), <italic>24B-Gal4</italic> (BL-1716), and <italic>elav-Gal4</italic> (BL-8760). For <italic>RNAi</italic>-mediated knockout we used the following TRiP lines generated by the Transgenic RNAi Project: <italic>GluRIIC</italic> (<italic>P[TRiP</italic>.<italic>JF01854}attP2</italic>), and <italic>fur1 (P[TRiP</italic>.<italic>GL01340] attP40)</italic>. The control is <italic>y</italic>
<sup><italic>1</italic></sup>
<italic>w</italic>
<sup><italic>1118</italic></sup> unless otherwise specified.</p><p>For rescue analyses, <italic>neto</italic> transgenes were introduced into <italic>neto</italic>
<sup><italic>36</italic></sup> null mutant background using tissue-specific promoters. Since <italic>neto</italic> is on the X-chromosome we used only <italic>FM7-GFP</italic> balanced stocks to eliminate any meiotic non-disjunction event. The F1 progenies were genotyped during late embryogenesis and reared at the indicated temperatures. After 24 hours, crawling larvae were removed, counted, and kept at the same temperatures for further analyses or adult viability testing.</p><p>Neto variants were generated using QuikChange site-directed mutagenesis kit (Stratagene) as described previously [<xref ref-type="bibr" rid="pgen.1004988.ref023">23</xref>]. CA-Neto has a deletion that joins A<sup>51</sup>-Q<sup>130</sup> and loops out the Neto prodomain. PM-Neto has two point mutations: R<sup>123</sup>I and R<sup>126</sup>I. Double-tagged Neto constructs were generated by QuikChange loop-in of various Neto fragments in a previously described AcPA-SP-Myc-V5/His plasmid [<xref ref-type="bibr" rid="pgen.1004988.ref064">64</xref>]. This actin promoter/terminator plasmid contains the sequences coding for the Tolloid-related signal peptide, the 5xMyc cassette, a multiple cloning site, followed by the V5 and RGS-6xHis epitopes. All constructs were verified by DNA sequencing.</p><p>For RNA interference, PCR primers for Furins that carry the T7 promoter sequence at the 5&#8217; end were designed as previously described [<xref ref-type="bibr" rid="pgen.1004988.ref036">36</xref>]. The primers were as follows:</p><p>dFur1-F 5&#8217;-TAATACGACTCACTATAGGGACGCAAAGATCCTCTGTGGCA;</p><p>dFur1-R 5&#8217;- TAATACGACTCACTATAGGGACATTGCTCCCGGAACTGC;</p><p>dFur2-F 5&#8217;- TAATACGACTCACTATAGGGACGCTAGAGGCCAATCCGGAA;</p><p>dFur2-R 5&#8217;- TAATACGACTCACTATAGGGACCCTTCTCGCCCCAAAAGTG;</p><p>Amon-F 5&#8217;- TAATACGACTCACTATAGGGACCCACATGGAGCTGGCTGT;</p><p>Amon-R 5&#8217;- TAATACGACTCACTATAGGGACCCTGACTTTGCCGCCATT.</p><p>PCR products were amplified from genomic DNA or S2 cells cDNA. In vitro transcribed dsRNA was produced using the MEGAscript kit (Ambion). RNAi treatment was carried out by transfections of 5 mg/ml of dsRNA into S2 cells.</p><p>S2 cells were transfected with indicated constructs and harvested after five days incubation. Total RNA was extracted using TRIZOL reagent (Invitrogen) according to manufacturer's instructions. AccuScript High Fidelity First-Strand cDNA Synthesis Kit (Agilent) was used to generate cDNAs from the extracted total RNAs according to manufacturer&#8217;s instructions. PCR reaction for each target gene was executed using the cDNAs as templates with specific primer pairs (above) and &#946;-Actin as a reaction standard (Actin-Forward: 5&#8217;-CTGGCACCACACCTTCTACAATG-3&#8217;, Actin-Reverse: 5&#8217;-GCTTCTCCTTGATGTCACGGAC-3&#8217;).</p><p>Wandering third instar larvae were dissected as described previously in ice-cooled Ca<sup>2+</sup>-free HL-3 solution [<xref ref-type="bibr" rid="pgen.1004988.ref065">65</xref>,<xref ref-type="bibr" rid="pgen.1004988.ref066">66</xref>]. Dissecting larval tissues were fixed in either 4% formaldehyde or Bouin's fixative (Polysciences, Inc.) for 20 min or 5 min respectively. PBS containing 0.5% Triton X-100 was used for washing and antibody reaction. For detergent-free staining, 1X PBS was used. Primary antibodies from Developmental Studies Hybridoma Bank were used at the following dilutions: mouse anti-GluRIIA (MH2B), 1:100; mouse anti-Dlg (4F3), 1:1000; mouse anti-Brp (Nc82), 1:100; mouse anti-CSP (6D6), 1:100; mouse anti-&#945;-spectrin (3A9), 1:100. Other primary antibodies were as follows: rat anti-Neto, 1:1000 [<xref ref-type="bibr" rid="pgen.1004988.ref023">23</xref>], rabbit anti-GluRIIB, 1:2000 (a gift from David Featherstone) [<xref ref-type="bibr" rid="pgen.1004988.ref067">67</xref>]; rabbit anti-GluRIIC, 1:2000 [<xref ref-type="bibr" rid="pgen.1004988.ref033">33</xref>]; rabbit anti-PAK, 1:2000 (a gift from Nicholas Harden) [<xref ref-type="bibr" rid="pgen.1004988.ref068">68</xref>]; FITC-, rhodamine-, and Cy5-conjugated goat anti-HRP, 1:1000 (Jackson ImmunoResearch Laboratories, Inc.). Alexa Fluor 488-, Alexa Fluor 568-, and Alexa Fluor 647-conjugated secondary antibodies (Molecular Probes) were used at 1:400. All samples were mounted with ProLong Gold reagent (Invitrogen) and incubated for 24 hours at RT. Confocal images were acquired using Carl Zeiss LSM 780 or 510 laser scanning microscopic system with Plan-Apochromat 63X/1.4 oil DIC objective using ZEN software. Z-stacked images were collected, processed, and analyzed using Imaris X64 (7.6.0, Bitplane) or ImageJ (NIH) software. In each experiment, samples of different genotypes were processed simultaneously and imaged under identical confocal settings. To quantify fluorescence intensities, confocal regions of interest (ROIs) surrounding anti-HRP immunoreactivities were selected and the signals measured individually at NMJs from ten or more different larvae for each genotype (number of samples is indicated in the graph bar). The signal intensities were calculated relative to HRP volume and subsequently normalized to control. For the extrajunctional, cell surface GluRIIC staining, where the GluRIIC positive signals are predominantly in the form of puncta at both Neto-depleted and Neto-excess NMJs, intensities from several size-matched areas of the muscles were collected and averaged using Image J software. The numbers of muscles analyzed per genotype are indicated inside the bars. Quantification of NMJ morphological features was performed at muscle 4 of abdominal segment 4 using the filament tracing function of Imaris software. Boutons were counted manually, while blind to the genotype, using anti-HRP and anti-Dlg staining. Statistical analyses were performed using the Student&#8217;s t-test with a two-tailed distribution and a two-sample unequal variance. All graphs represent mean value of all samples of the given genotype &#177; SEM.</p><p>Transiently transfected <italic>Drosophila</italic> S2 cells were used for producing recombinant proteins as previously described [<xref ref-type="bibr" rid="pgen.1004988.ref069">69</xref>]. The S2 cells were maintained in M3 (Shields and Sang M3 insect medium, Sigma) with 1x insect medium supplement (Sigma) and Penicillin/Streptomycin (Sigma), and sub-cultured every 7 days at 2 X 10<sup>6</sup> cells/ml. For the transfection, dimethyldioctadecyl-ammonium bromide (DDAB) solution (250 &#956;g/ml) was mixed with M3 media at 1:2 ratio and incubated 5 min at RT, then the DNA was added to the DDAB-M3 mixture (1&#956;g of plasmid DNA to 100 &#956;l suspension). The mixture was incubated for 20 min and transfected into S2 cells (100 &#956;l mixture to 2 X 10<sup>6</sup> cells/ml culture). After five days, the secreted proteins were collected for analysis, and membrane proteins were extracted by homogenizing cells in lysis buffer (50 mM Hepes-NaOH, 150 mM NaCl, 0.2 mM EDTA, 0.5% NP-40, 0.1% SDS, 2mM AEBSF [MP BIO], and protease inhibitor cocktail [Roche]) for 30 min on ice. The lysates were collected by centrifugation at 13,000rpm for 30 min at 4&#176;C, separated by SDS-PAGE on 4%&#8211;12% NuPAGE gels (Invitrogen) and transferred onto PVDF membranes (Millipore). Primary antibodies were used at the following dilutions: rat anti-Neto, 1:1000; chicken anti-GFP (Abcam), 1:2000; anti-GluRIIC, 1:1000; anti-tubulin (Sigma), 1:1000. Immune complexes were visualized using secondary antibodies coupled with IR-Dye 700 or IR-Dye 800 followed by scanning with the Odyssey infrared imaging system (Li-Cor Biosciences).</p><p>To analyze muscle proteins, wandering third instar larvae were dissected, and the body walls were mechanically homogenized in lysis buffer for 30 min on ice. The lysates were analyzed by Western blotting. For co-immunoprecipitation, the lysates were incubated with rabbit anti-GFP antibody (Invitrogen) for 1 hr at 4&#176;C. Protein A/G UltraLink Resin (50% slurry, Thermo Scientific) was added and incubated overnight at 4&#176;C. The beads were washed with lysis buffer. Proteins were eluted with 1x SDS sample buffer and analyzed by Western blotting.</p><p>Secreted and processed Neto fragment (CUB1-V5/His) was purified using His-Trap affinity column equipped with AKTA FPLC system (Pharmacia) and separated by SDS-PAGE. A specific gel band was isolated and analyzed at Microchemistry and Proteomics Analysis Facility, Harvard University.</p><p>Wandering third instar larvae were dissected in Jan's saline containing 0.1 mM Ca<sup>2+</sup> and processed as previously described [<xref ref-type="bibr" rid="pgen.1004988.ref070">70</xref>]. Dissected larvae were fixed in EM fixative (4% <italic>p</italic>-formaldehyde, 1% glutaraldehyde, 0.1 M sodium cacodylate, and 2 mM MgCl<sub>2</sub>, pH 7.2) for 20 min at room temperature followed by incubation overnight at 4&#176;C, then washed extensively (0.1 M sodium cacodylate, and 132 mM sucrose, pH 7.2). The samples were processed and analyzed at the Microscopy and Imaging Core Facility, NICHD.</p><p>The standard larval body wall muscle preparation first developed by Jan and Jan (1976) was used for electrophysiological recordings [<xref ref-type="bibr" rid="pgen.1004988.ref071">71</xref>,<xref ref-type="bibr" rid="pgen.1004988.ref072">72</xref>]. Wandering third instar larvae were dissected in physiological saline HL-3 [<xref ref-type="bibr" rid="pgen.1004988.ref065">65</xref>], washed, and immersed in HL-3 containing 0.8 mM Ca<sup>2+</sup> using a custom microscope stage system [<xref ref-type="bibr" rid="pgen.1004988.ref073">73</xref>]. The nerve roots were cut near the exiting site of the ventral nerve cord so that a suction electrode could pick up the motor nerve later. Intracellular recordings were made from muscle 6. Data were used when the input resistance of the muscle was &gt;5 M&#937; and the resting membrane potential was between &#8722;60 mV and &#8722;80 mV for the entire duration of the experiment. The input resistance of the recording microelectrode (backfilled with 3 M KCl) ranged from 20 to 25 M&#937;. Muscle synaptic potentials were recorded using Axon Clamp 2B amplifier (Axon Instruments) and pClamp software. Following motor nerve stimulation with a suction electrode (100 &#956;sec, 5 V), evoked EJPs were recorded. Three to five EJPs evoked by low frequency of stimulation (0.1 Hz) were averaged. For mini recordings, TTX (1 &#956;M) was added to prevent evoked release [<xref ref-type="bibr" rid="pgen.1004988.ref065">65</xref>]. To calculate mEJP mean amplitudes, 50&#8211;100 events from each muscle were measured and averaged using the Mini Analysis program (Synaptosoft). Minis with a slow rise and falling time arising from neighboring electrically coupled muscle cells were excluded from analysis [<xref ref-type="bibr" rid="pgen.1004988.ref072">72</xref>,<xref ref-type="bibr" rid="pgen.1004988.ref074">74</xref>]. In addition, when comparing mini sizes between preparations, the Kolmogorov-Smirnov test was administrated. Quantal content was calculated by dividing the mean EJP by the mean mEJP after correction of EJP amplitude for nonlinear summation according to the methods described [<xref ref-type="bibr" rid="pgen.1004988.ref075">75</xref>,<xref ref-type="bibr" rid="pgen.1004988.ref076">76</xref>]. Corrected EJP amplitude = E[Ln[E/(E &#8722; recorded EJP)]], where E is the difference between reversal potential and resting potential. The reversal potential used in this correction was 0 mV [<xref ref-type="bibr" rid="pgen.1004988.ref075">75</xref>,<xref ref-type="bibr" rid="pgen.1004988.ref077">77</xref>]. Data are presented as mean &#177; SEM, unless otherwise specified; EJP amplitudes and quantal contents after the nonlinear correction are shown. A one-way analysis of variance followed by Tukey's HSD test was used to assess statistically significant differences among the genotypes. Differences were considered significant at <italic>p &lt; 0</italic>.<italic>05</italic>.</p>