<p>T47D/A1-2 cells were cultured in DMEM containing 10% FBS. Prior to RNA isolation, cells were cultured in medium supplemented with 5% charcoal dextran-treated serum for at least 24 hours. The A1-2 cell line is a previously described derivative of the T47D breast cancer cell line that overexpresses rat glucocorticoid receptor (GR) and contains a stably-integrated MMTV luciferase reporter gene [<xref ref-type="bibr" rid="pgen.1006224.ref029">29</xref>]. Short capped RNA was isolated from T47D/A1-2 cells as previously described [<xref ref-type="bibr" rid="pgen.1006224.ref016">16</xref>]. Libraries were generated using the Illumina TruSeq small RNA kit. Two independent replicates were performed. A primary dataset was generated from combined Illumina HiSeq and MiSeq runs. This data set was used in all TSS calling and downstream analysis. A secondary validation data set with fewer reads was generated on an Illumina MiSeq run. This data set was used to validate reproducibility of read density at called TSSs (<xref ref-type="supplementary-material" rid="pgen.1006224.s004">S1A Fig</xref>). For this and other in-house sequencing experiments, libraries were prepared and sequenced by the NIH Intramural Sequencing Center (NISC).</p><p>Start-seq reads were first filtered by quality score; reads with an average Sanger score less than 20 were removed from analysis. Following quality filtering, Cutadapt (version 1.2.1) was used to remove adapter sequences [<xref ref-type="bibr" rid="pgen.1006224.ref030">30</xref>]. Alignment of Start-seq reads was performed using Bowtie (version 0.12.8) to hg19 or mm9 genome assemblies [<xref ref-type="bibr" rid="pgen.1006224.ref031">31</xref>]. From each uniquely mapped Start-seq fragment, the 5&#8217; end was taken forward into the TSS calling procedure.</p><p>Identification of TSSs was performed based on methods described previously [<xref ref-type="bibr" rid="pgen.1006224.ref011">11</xref>,<xref ref-type="bibr" rid="pgen.1006224.ref016">16</xref>], TSS identification was guided by RefSeq annotation (retrieved 05/09/2014) [<xref ref-type="bibr" rid="pgen.1006224.ref032">32</xref>]. From the reference annotation, a list of non-redundant TSSs was taken from all mRNA RefSeq IDs (&#8220;NM_&#8221;). 2000-nt search windows were created about each RefSeq TSS. If search windows overlapped and had the same common gene name, those search windows were merged. For other overlapping search windows, boundaries were defined as the midpoint between associated TSSs. The intersection of search windows and 5&#8217; Start-seq ends was then determined.</p><p>TSSs were called within each window in which the strand-specific 5&#8217; end read count at any given nucleotide position met or exceeded a threshold of 5 reads. This threshold was determined in a previously described method [<xref ref-type="bibr" rid="pgen.1006224.ref033">33</xref>]. In short, the FDR was estimated based upon the distribution of Start-seq reads across the genome and a background model where the probability of finding a given number of aligned reads by chance is given by a sum of Poisson probabilities. The read threshold was selected to allow less than 1 expected false positive by this measure.</p><p>In those windows where a single nucleotide position met or exceeded the read threshold, a gene TSS was called. The calling method aims to select as the TSS the position with the highest read counts in window region with the highest read density. To accomplish this, two potential TSSs were first determined. The first was the position with the most aligned 5&#8217; ends across the entire window. For the second, the search window was divided into 200-nt bins at every 10 nt across the search window. At the 200-nt bin with the most overlapped 5&#8217; ends, the second potential TSS was called as the position with the most aligned 5&#8217; ends. Of the two putative sites, the TSS closest to the associated annotated RefSeq TSS was selected.</p><p>Following gene TSS identification, uaTSSs and daTSSs were found. For uaTSSs and daTSSs, search spaces were defined in antisense orientation as 1 to 1000 nt upstream and 1 to 2000 nt downstream, respectively. uaTSSs and daTSSs were called at the position with the most aligned 5&#8217; ends within the search window if the count at any single nucleotide position met or exceeded a threshold of 5 reads. To ensure that identified daTSSs were not simply mis-called gene TSSs or uaTSSs, additional criteria were used to filter daTSS calls. A daTSS call was filtered if (1) within 1000 nt upstream of the daTSS there was a genomic position with Start-seq reads greater than or equal to 10% of reads at the associated gene TSS on the same strand as the gene TSS, implicating the daTSS as a potential uaTSS for an uncalled gene TSS or (2) it was within 1000 nt of an annotated TSS on the same strand, implicating the daTSS as a potential gene TSS.</p><p>Prior to alignment, Pol II ChIP-seq reads from MCF-7 cells [<xref ref-type="bibr" rid="pgen.1006224.ref023">23</xref>] (see also Supplemental <xref ref-type="supplementary-material" rid="pgen.1006224.s001">Table 1</xref>) were filtered based on quality; reads with an average Sanger quality score less than 20 were removed from analysis. Following quality filtering, Cutadapt was used to remove adapter sequences [<xref ref-type="bibr" rid="pgen.1006224.ref030">30</xref>]. Alignment was performed using Bowtie [<xref ref-type="bibr" rid="pgen.1006224.ref031">31</xref>]. Fragment lengths were estimated using Homer (version 4.6) [<xref ref-type="bibr" rid="pgen.1006224.ref034">34</xref>]. From each uniquely mapped fragment, the fragment was extended based on the estimated length, and the fragment center was subsequently found.</p><p>Considering each TSS group separately, the nucleotide composition of each position in a -1000 to +999 window was determined and reported as a percentage. Logo plots were generated using Web Logo 3 considering sequences in a -5 to +5 window about identified TSSs [<xref ref-type="bibr" rid="pgen.1006224.ref035">35</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To identify occurrences of Pol II-associated and known vertebrate motifs, FIMO (version 4.10.0) was used considering a p &lt; 0.0001 significance cutoff and a 0-order Hidden Markov Model from promoter regions as background [<xref ref-type="bibr" rid="pgen.1006224.ref036">36</xref>]. Publically available position weight matrices from JASPAR were used in motif identification [<xref ref-type="bibr" rid="pgen.1006224.ref018">18</xref>]. The JASPAR POLII database (2008 version; 13 motifs) and vertebrate motifs in the JASPAR CORE database (2014 version; 205 motifs) were used for Pol II-associated and known vertebrate motif identification, respectively. Across all promoter regions, on the order of 10<sup>5</sup> Pol II-associated and 10<sup>7</sup> known vertebrate motifs were identified. Motifs and additional information are available at <ext-link ext-link-type="uri" ns0:href="http://jaspar.genereg.net/">http://jaspar.genereg.net/</ext-link>.</p><p><italic>De novo</italic> motif discovery was performed using MEME (version 4.10.0) with default parameters [<xref ref-type="bibr" rid="pgen.1006224.ref037">37</xref>]. Sequence windows from -35 to -20 and from -5 to +5 relative to TSS positions were used. Sequences from each TSS class were combined prior to motif analysis. Logo plots were generated using Web Logo 3 after aligning identified motifs [<xref ref-type="bibr" rid="pgen.1006224.ref035">35</xref>].</p><p>CpG island heatmaps reflect the intersection of annotated CpG islands retrieved from UCSC Genome Browser [<xref ref-type="bibr" rid="pgen.1006224.ref017">17</xref>] with TSS-centered windows.</p><p>Sequence conservation heatmaps were generated using phyloP scores from placental mammal alignments retrieved from UCSC Genome Browser [<xref ref-type="bibr" rid="pgen.1006224.ref019">19</xref>]. Each position in the heatmap represents the average score over all positions in a 40-bp bin for which phyloP scores were available.</p><p>GRO-cap data from K562 and GM12878 cells [<xref ref-type="bibr" rid="pgen.1006224.ref007">7</xref>] were compared to daTSSs identified in T47D/A1-2 cells. If a given site did not have GRO-cap signal in either K562 or GM12878 cells, that daTSS was considered T47D/A1-2 specific. The list of genes with T47D/A1-2-specific daTSSs was applied to Ingenuity Pathway Analysis [<xref ref-type="bibr" rid="pgen.1006224.ref038">38</xref>] considering experimentally observed associations over mammalian tissues and cell lines.</p><p>Nuclei were harvested from cultured T47D/A1-2 cells and digested for 5 minutes at 37&#176;C with a range of MNase (Worthington) concentrations. Reactions were stopped by the addition of EDTA and then treated with RNase and proteinase K. Digested DNA was isolated by phenol/chloroform extraction and ethanol precipitation. Libraries were prepared using an Illumina TruSeq sample preparation kit and sequenced on an Illumina HiSeq for paired-end 50 base reads.</p><p>Publically available MNase-seq data [<xref ref-type="bibr" rid="pgen.1006224.ref022">22</xref>] and data generated in this work were prepared in the same way. Prior to alignment, MNase-seq reads were filtered based on quality; reads with an average Sanger quality score less than 20 were removed from analysis. Following quality filtering, Cutadapt was used to remove adapter sequences [<xref ref-type="bibr" rid="pgen.1006224.ref030">30</xref>]. Alignment of MNase-seq read pairs was performed using Bowtie [<xref ref-type="bibr" rid="pgen.1006224.ref031">31</xref>]. From each uniquely mapped fragment, the fragment center was found. Any report of MNase-seq coverage only considers the fragment-center position.</p><p>FAIRE-seq data were collected as described previously [<xref ref-type="bibr" rid="pgen.1006224.ref024">24</xref>]. Publically available FAIRE-seq data [<xref ref-type="bibr" rid="pgen.1006224.ref024">24</xref>] and data generated in this work were prepared in the same way. Prior to alignment, FAIRE-seq reads were filtered based on quality; reads with an average Sanger quality score less than 20 were removed from analysis. Following quality filtering, Cutadapt was used to remove adapter sequences [<xref ref-type="bibr" rid="pgen.1006224.ref030">30</xref>]. Alignment of FAIRE-seq reads was performed using Bowtie [<xref ref-type="bibr" rid="pgen.1006224.ref031">31</xref>]. Aligned FAIRE-seq reads were then de-duplicated using Picard (version 1.118) [<xref ref-type="bibr" rid="pgen.1006224.ref039">39</xref>]. FAIRE-seq fragment lengths were estimated using Homer [<xref ref-type="bibr" rid="pgen.1006224.ref034">34</xref>]. For each uniquely mapped fragment, the fragment was extended based on the estimated length, and the estimated fragment center subsequently found. Reported FAIRE-seq coverage only considers the fragment-center position.</p><p>Over biological triplicates, total RNA was harvested using an RNeasy Kit (Qiagen) with on-column DNase treatment. RNA quality was validated by Bioanalyzer (Agilent). Paired-end strand-specific poly-A enriched libraries were sequenced on an Illumina HiSeq 2500 for 125 base paired-end reads.</p><p>Prior to alignment, RNA-seq reads were filtered based on quality; reads with an average Sanger quality score less than 20 were removed from analysis. Following quality filtering, Cutadapt was used to remove adapter sequences [<xref ref-type="bibr" rid="pgen.1006224.ref030">30</xref>]. Insert lengths were estimated by transcriptome alignment using Bowtie [<xref ref-type="bibr" rid="pgen.1006224.ref031">31</xref>]. Sequence alignment was then performed using TopHat (version 2.0.4) [<xref ref-type="bibr" rid="pgen.1006224.ref040">40</xref>]. Following de-duplication by Picard [<xref ref-type="bibr" rid="pgen.1006224.ref039">39</xref>], alignments from individual replicates were merged. FPKM values were calculated using Cufflinks (version 2.2.1) [<xref ref-type="bibr" rid="pgen.1006224.ref041">41</xref>].</p><p>For publically available data sets (with the exceptions of MCF7 Pol II ChIP-seq data and of FAIRE-seq and MNase-seq validation data sets), read coverage files were retrieved from public depositories (<xref ref-type="supplementary-material" rid="pgen.1006224.s001">S1 Table</xref>). Given that reported read densities were considered, the data processing of the original authors was effectively observed. deepTools was used to generate matrices describing the intersection of read coverage with TSS-centered genomic windows observing strand specificity when appropriate [<xref ref-type="bibr" rid="pgen.1006224.ref042">42</xref>]. These matrices were then used to generate heatmaps.</p><p>Heatmaps consider 40-bp/40-nt bins over TSS-centered windows. Unless otherwise noted, each position in a heatmap gives the number of reads or other features overlapping with that bin. Heatmap images were generated using Partek (version 6.6) [<xref ref-type="bibr" rid="pgen.1006224.ref043">43</xref>]. Two-dimensional plots, unless otherwise noted, consider 10-bp/10-nt bins and report average values across all TSSs considered. To test enrichment of ChIP-seq signal at daTSS positions, ChIP-seq coverage was found in a 100-bp window about all identified daTSSs. Equivalent regions were found at genes without daTSSs by selecting a 100-bp window shifted downstream of TSSs by the median observed TSS-daTSS distance (507 nts). The significance of enrichment was then calculated by Wilcoxon test comparing the two groups.</p><p>To generate the gene TSS-centered panels in <xref ref-type="supplementary-material" rid="pgen.1006224.s008">S5</xref>, <xref ref-type="supplementary-material" rid="pgen.1006224.s009">S6</xref>, <xref ref-type="supplementary-material" rid="pgen.1006224.s010">S7</xref> and <xref ref-type="supplementary-material" rid="pgen.1006224.s011">S8</xref> Figs, uaTSS- and daTSS-centered plots were first reflected across uaTSS and daTSS positions, respectively, to orient these plots relative to gene TSSs. These plots were then translated upstream or downstream by the median distances observed between gene TSSs and uaTSSs or between gene TSSs and daTSSs across calls made in T47D/A1-2 cells.</p><p>The data sets supporting the results of this article are available in the GEO repository, GSE74308.</p>