<p>All studies were approved by their local ethics committees; all patients gave informed consent.</p><p>The initial dataset consisted of 6,778 ischaemic stroke cases of European ancestry and 12,095 ancestry-matched controls from the Wellcome Trust Case-Control Consortium II project in ischaemic stroke <xref ref-type="bibr" rid="pgen.1004469-Bellenguez1">[1]</xref>, as well as a cohort from Milan, Italy <xref ref-type="bibr" rid="pgen.1004469-Traylor2">[16]</xref>. These included 2,858 cases and 5,716 matched controls genotyped using the Immunochip platform; and 3,940 cases genotyped using either the Illumina 610 k or 660 k platforms matched with 6,379 controls genotyped on the Illumina Human 1.2M Duo (UK), Illumina Human 550 k (German) and Illumina 610 k platforms (Italian) (<xref ref-type="table" rid="pgen-1004469-t001">Table 1</xref>). The Immunochip cases were described in the previous WTCCC2 ischaemic study, where they formed the replication effort <xref ref-type="bibr" rid="pgen.1004469-Bellenguez1">[1]</xref>, as well as in a recent paper <xref ref-type="bibr" rid="pgen.1004469-Kilarski1">[37]</xref>. Genotyping of the five Immunochip case cohorts on the commercially available Immunochip array (Illumina, San Diego, CA, USA) was performed at the Sanger Centre, Hinxton, Cambridge UK. Swedish controls were provided and genotyped by the Swedish SLE network, Uppsala, Sweden. Belgian control samples were provided through the efforts of the International Multiple Sclerosis Genetics Consortium (IMSGC). German controls were derived from the PopGen biobank, <xref ref-type="bibr" rid="pgen.1004469-Krawczak1">[38]</xref>. UK controls were derived from the 1958 Birth cohort. Any of the 1958 Birth controls overlapping with those from the WTCCC2 datasets, as assessed by IBD estimates, were removed prior to analysis. Standard quality control procedures were undertaken on all centres, before centre-wise imputation to the 1000 Genomes phase 1 integrated variant set (March 2012), using IMPUTE v2.2.0 <xref ref-type="bibr" rid="pgen.1004469-Howie1">[39]</xref>, <xref ref-type="bibr" rid="pgen.1004469-Marchini1">[40]</xref>. SNPs with poor imputation quality (info&lt;0.3) or low minor allele frequency (MAF&lt;0.01) were discarded.</p><p>Ischemic stroke was defined as a typical clinical syndrome with radiological confirmation; ascertained cases were classified into individual stroke subtypes using the Trial of Org 10172 in acute stroke (TOAST) criteria in all centres <xref ref-type="bibr" rid="pgen.1004469-Adams1">[41]</xref>. Age-at-onset was defined as age at first hospital admission for stroke; where this information was unavailable, age at blood draw was used (7.3% of cases). The age-at-onset and gender distributions of the populations are given in <xref ref-type="supplementary-material" rid="pgen.1004469.s007">Table S3</xref>. Age-at-onset quantiles were calculated from all the cases from the discovery datasets in the four stroke phenotypes (all IS and the three stroke subtypes: CE, LAA, SVD) and these were used to evaluate associated loci at different age-at-onset thresholds.</p><p>The prevalence of ischaemic stroke by age was obtained from a recent publication <xref ref-type="bibr" rid="pgen.1004469-Seshadri2">[9]</xref>; gender-specific estimates were averaged, and prevalences within each of the stroke subtypes were assumed to be approximately 20% of the overall total, similar to proportions seen in population-based studies <xref ref-type="bibr" rid="pgen.1004469-Lovett1">[42]</xref>. We modeled phenotype data using a continuous unobserved quantitative trait called the disease liability, which we used to approximate the effect of age-at-onset on the liability scale, based on estimates of ischaemic stroke prevalence by age from epidemiological data (full details in <xref ref-type="supplementary-material" rid="pgen.1004469.s011">Text S2</xref>). We developed two models for our analysis; one based on the prevalence rates for all ischaemic stroke cases, and secondly for the three stroke subtypes. We used these models to calculate posterior mean liabilities after conditioning on age-at-onset for the four stroke phenotypes separately. Controls were modeled in the same way, but were assumed to take the posterior mean from the lower (unaffected) portion of the distribution in the liability threshold model. Where age data was missing, individuals were assigned the median age value. Full descriptions of the models used and the formulae used to calculate posterior mean liabilities are given in <xref ref-type="supplementary-material" rid="pgen.1004469.s011">Text S2</xref>. Regression was then performed on posterior liabilities by multiplying the number of samples by the squared correlation between the expected genotype dosage and posterior mean liabilities for each of the discovery cohorts in the four ischaemic stroke phenotypes (CE, LAA, SVD, IS), following a previous approach <xref ref-type="bibr" rid="pgen.1004469-Zaitlen1">[17]</xref>. Ancestry-informative principal components were included where appropriate (6 of 8 centres), using the EIGENSTRAT procedure <xref ref-type="bibr" rid="pgen.1004469-Price1">[43]</xref>. All analysis was performed using the R statistical software.</p><p>The results from each centre were meta-analysed for each of the four phenotypes using Stouffer's Z-score weighted approach, as implemented in METAL <xref ref-type="bibr" rid="pgen.1004469-Willer1">[44]</xref>. Genomic control was used to correct for any residual inflation due to population stratification <xref ref-type="bibr" rid="pgen.1004469-Devlin1">[45]</xref>. Between-study heterogeneity was assessed using Cochran's Q statistic. We considered only SNPs present in at least 75% of the cases, and with no evidence of heterogeneity (Cochran's Q p-value&gt;0.001). All SNPs analysed were either genotyped or imputed in both the Immunochip and the genome-wide datasets. After meta-analysis, the resulting p-values were compared with the equivalent values from an unconditioned analysis. For SNPs more significant in the age-at-onset informed analysis and with p&lt;5&#215;10<sup>&#8722;6</sup>, we determined the evidence of a true age-at-onset effect by generating 1000 permutations of age-at-onset and rerunning the age-at-onset informed analysis, meta-analysing as previously. We calculated an empirical p-value by dividing the number of permuted observations showing greater significance in the meta-analysis than the observed results by the number of permutations. Any novel SNP with a meta-analysis p&lt;5&#215;10<sup>&#8722;6</sup> and evidence of an age-at-onset effect at p&lt;0.05 were taken forward for replication. We set the experiment-wide significance threshold at p&lt;5&#215;10<sup>&#8722;8</sup>.</p><p>Replication of an associated variant was performed in a further 10 cohorts from METASTROKE. Nine of the centres used a cross-sectional design, while one was a large prospective, population based cohort (ARIC). Nine of the centres were of European ancestry, while one consisted of individuals of Pakistani ancestry (RACE) (<xref ref-type="table" rid="pgen-1004469-t002">Table 2</xref>). All centres used a case-control methodology; centres with a cross sectional design used logistic regression to model the association of genotype dosages from imputation with the dichotomous outcome of ischaemic stroke and prospective cohorts used Cox proportional-hazards models to evaluate time to first stroke, fitting an additive model relating genotype dose to the stroke outcome. European ancestry replication centres were meta-analysed using a fixed effects inverse-variance weighted method. To assess the evidence for association of the SNP for replication samples of all ancestries, we performed a trans-ethnic meta-analysis using a random-effects model to control for any resulting heterogeneity <xref ref-type="bibr" rid="pgen.1004469-Han1">[18]</xref>. To evaluate the overall evidence for association, the results of the discovery and replication analyses were combined using Fisher's Method.</p><p>Expression of the two genes proximal to the associated variant was tested in atherosclerotic plaques from the Tampere Vascular study <xref ref-type="bibr" rid="pgen.1004469-Oksala1">[27]</xref>, <xref ref-type="bibr" rid="pgen.1004469-Levula1">[46]</xref>, <xref ref-type="bibr" rid="pgen.1004469-Niinisalo1">[47]</xref>, <xref ref-type="bibr" rid="pgen.1004469-Oksala2">[48]</xref>, <xref ref-type="bibr" rid="pgen.1004469-Oksala3">[49]</xref>. Carotid, femoral, and aortic atherosclerotic plaques constituting the intima and inner media were prospectively obtained between 2005 and 2009 from patients fulfilling the following inclusion criteria: (1) carotid endarterectomy attributable to asymptomatic or symptomatic &gt;70% carotid stenosis, or (2) femoral or (3) aortic endarterectomy with aortoiliac or aortobifemoral bypass attributable to symptomatic peripheral arterial disease. Whole thickness left internal thoracic artery samples obtained during coronary artery bypass surgery and identified as being microscopically atherosclerosis free were used as controls. The patients were consecutively recruited and stratified according to indication for surgery. All open vascular surgical procedures were performed at the Division of Vascular Surgery and Heart Center, Tampere University Hospital.</p><p>Fresh tissue samples were immediately soaked in RNALater solution (Ambion Inc) and homogenized using an Ultra-Turrax T80 homogenizer (IKA). RNA was extracted with the Trizol reagent (Invitrogen) and miRNEasy Mini-Kit (Qiagen) with the RNase-Free DNase Set (Qiagen) according to manufacturer instructions. The RNA isolation protocol was validated by analyzing the integrity of the RNA with the RNA 6000 Nano Chip Kit (Agilent). The expression levels were analyzed with an Illumina HumanHT-12 v3 Expression BeadChip (Illumina). In brief, 300&#8211;500 ng of RNA was reverse transcribed in cRNA and biotin-UTP labeled using the IlluminaTotalPrep RNA Amplification Kit (Ambion), and 1500 ng of cRNA was then hybridized to the Illumina HumanHT-12 v3 Expression BeadChip.</p><p>The BeadChips were scanned with the Illumina iScan system. After background subtraction, raw intensity data were exported using the Illumina Genome Studio software. Further data processing was conducted by means of R language and appropriate Bioconductor modules. Data were log2-transformed, and robust multichip average and robust spline normalization (rma_rsn) were used. Accuracy of the expression array was validated with qRT-PCR <xref ref-type="bibr" rid="pgen.1004469-Raitoharju1">[50]</xref>. mRNA Expression levels in the tissues were determined; a fold change statistic was estimated between the two tissues, and significance was calculated using a t test.</p><p>Recent evidence indicates that a significant proportion of GWAS SNPs fall within regions that are likely to affect binding of nearby proteins, such as transcription factor binding sites <xref ref-type="bibr" rid="pgen.1004469-Schork1">[51]</xref>, <xref ref-type="bibr" rid="pgen.1004469-Dunham1">[52]</xref>. We used the RegulomeDB database to access regulatory information from ENCODE and other existing publications <xref ref-type="bibr" rid="pgen.1004469-Boyle1">[19]</xref>, investigating the evidence that the SNPs in the associated locus have a regulatory function. First, the linkage-disequilibrium (LD) patterns amongst the most strongly associated SNPs were determined. We then used PLINK to determine the LD structure of the associated region, using LD-patterns from the 85 Utah residents from the 1000 Genomes project <xref ref-type="bibr" rid="pgen.1004469-Abecasis1">[53]</xref>, <xref ref-type="bibr" rid="pgen.1004469-Purcell1">[54]</xref>. All SNPs with r<sup>2</sup>&gt;0.6 were identified within a 2,000 kb window from the index SNP. All of the SNPs identified were then investigated using RegulomeDB to determine the evidence that any of the SNPs have a regulatory function.</p><p>Permutation analysis was performed to evaluate the age-at-onset informed approach, to show that including age at onset information directly led to the increased significance, due solely to inclusion of age-at-onset information at tested SNPs. First, we identified a set of SNPs enriched for true association in the case control analysis of ischaemic stroke and subtypes. An expanded set of discovery and METASTROKE studies were analysed using standard case control methods and subsequent meta-analysis (see <xref ref-type="supplementary-material" rid="pgen.1004469.s006">Table S2</xref>). SNPs with p&lt;0.05 and no evidence of heterogeneity (p&gt;0.0001) were extracted and pruned for LD (300 kb window, r<sup>2</sup>&lt;0.25), leaving a set of almost independent SNPs for further analysis. Each retained SNP represented the most significant association in each LD block, as determined by the &#8220;clump&#8221; procedure in PLINK, based on LD patterns from the CEU individuals from 1000 Genomes. The number of SNPs used in each analysis is given in <xref ref-type="supplementary-material" rid="pgen.1004469.s009">Table S5</xref>. These SNP subsets were derived for ischaemic stroke, and for each stroke subset and then used in the age-at-onset informed analysis. Analysis was performed as previously for each stroke subtype using the age-at-onset informed method within studies and meta-analysis across studies (giving <italic>observed</italic> results, as obtained above). We then performed a permutation study to obtain the expected distribution of p-values at these SNPs. Age at onset for cases was permuted within stroke subtypes within each study, and then the data were re-analysed, for 1000 permutations. Two summary statistics were constructed: (1) within permutations, we compared p-values from analysis of permuted age at onset with p-values from the observed data, and tabulated the proportion of SNPs with increased significance in the observed data set than in the permuted data set; across permutations, we calculated the median proportion of SNPs with increased significance in the observed data; (2) Within permutations, we converted each SNP p-value to a Z score and summed the absolute value of the Z score across SNPs (sumZ). An empirical p-value for the age-informed analysis was calculated from the proportion of simulated data sets where sumZ exceeded the value in the observed analysis. This analysis was performed at SNP subsets defined from four SNP p-value thresholds in the discovery and METASTROKE studies: p&lt;0.05, p&lt;0.005, p&lt;0.0005, and p&lt;0.00005.</p><p>Finally, we assessed the evidence of an age-at-onset effect at the four stroke loci identified in the METASTROKE ischaemic stroke collaboration (9p21, <italic>HDAC9</italic>, <italic>PITX2</italic>, <italic>ZFHX3</italic>) <xref ref-type="bibr" rid="pgen.1004469-Traylor2">[16]</xref>. For each SNP, we generated an empirical p-value from the proportion of permutations showing stronger association than in the observed age-at-onset informed analysis.</p>