<p>
<italic>E.coli</italic> RU1 (hence forth referred to as <italic>E. coli</italic>) and <italic>C. freundii</italic> RU2 (hence forth referred to as <italic>C. freundii</italic>) were isolated from the stool of healthy humans (<xref ref-type="supplementary-material" rid="pgen.1004872.s013">S1 Text</xref>, <xref ref-type="supplementary-material" rid="pgen.1004872.s005">S5 Figure</xref>). To minimize any potential for adaptation during the isolation process, we plated stool samples on MacConkey agar plates. After a single overnight growth, half of a single colony was flash frozen at &#8722;80&#176;C in Trypticase soy broth with 20% Glycerol (BD, USA) and the other half was used for phenotypic strain characterization. Strain identification by 16S sequencing was done from the frozen sample. All experimental evolution studies started from the frozen primary isolates by using a single clonalized colony derived from the initial snap frozen isolate. The identity of the wild "un-adapted" strains was based on the results of API 20 E (Biomerieux, USA) test strips and species-specific PCR (using forward primer: AGAGTTTGATCMTGGCTCAG, reverse primer: GWATTACCGCGGCKGCTG).</p><p>Twelve clones of <italic>E. coli</italic> RU1 were evolved independently for 200 days by daily transfers of a single, randomly chosen colony to a fresh LB plate (<xref ref-type="fig" rid="pgen-1004872-g001">Fig. 1</xref>). Every 15 days, a sample of the transferred colony was frozen at &#8722;80&#176;C. To calculate the generations per transfer, we assessed the number of cells in a colony after 24 hours of growth on LB agar and calculated the number of doublings as log<sub>2</sub>(number of cells/colony) at generation 0, 105 and 200 for two randomly chosen lines. The average number of generations per transfer was calculated as 29.4, which results in 5888 generations over 200 single cell bottlenecks.</p><p>A single colony from the frozen cultures of <italic>E. coli</italic> RU1 and C. <italic>freundii</italic> RU2 were each used to inoculate twelve independent cultures in Luria Broth Miller (LB, BD) and brain-heart infusion (BBL BHI, BD) (<xref ref-type="fig" rid="pgen-1004872-g001">Fig. 1</xref>, <xref ref-type="supplementary-material" rid="pgen.1004872.s012">S3 Table</xref>). Populations evolved in liquid media were transferred as described previously <xref ref-type="bibr" rid="pgen.1004872-Saxer1">[10]</xref>, <xref ref-type="bibr" rid="pgen.1004872-Saxer2">[11]</xref>. LB-evolved populations were grown in 10 ml liquid LB Miller broth in 25mm test tubes at 250 rpm and 37&#176;C, and transferred daily by 100-fold dilutions into fresh media (&#8764;6.6 generations/day) for a total of 75 days or 500 generations. Every 15 days, we froze a sample of the populations at &#8722;80&#176;C for further analyses. The BHI-evolved populations were cultured in 10 ml of BHI broth under the same conditions as the LB cultures, but transferred daily by 40,000-fold dilutions into fresh media (&#8764;15.2 generation/day) for a total of 50 days or 765 generations. This dilution was chosen to achieve the same effective population size at transfer of 10<sup>5</sup> cells/ml as in a parallel experiment performed with <italic>Yersinia pestis</italic> to allow direct comparisons. The BHI-evolved populations were frozen at &#8722;80&#176;C after 25 and 50 days.</p><p>Assays were performed either at the population level or the single colony level. Cells or populations were grown in their selective media (LB or BHI) and grown in liquid media or plated on agar plates made with their selective media, unless otherwise stated. Adaptation to the selective environments was assessed as changes in lag time and growth rate by measuring OD<sub>600</sub> over 24 hours of growth in liquid media following Walkiewicz <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1004872-Walkiewicz1">[74]</xref>. To test for changes in the pH of spent media, we grew the populations to stationary phase and measured the pH of the media after removing the cells. To test for genetic variation within the populations, we plated the populations at low density on tetrazolium arabinose plates and observed considerable variation in both colony size and in the ability to utilize arabinose. We plated the populations on the selective media supplemented with agar and isolated eight randomly chosen colonies from each of the 12 populations per treatment. These test sets of 96 individual isolates per species and environment were used for three phenotypic assays: 1) the redox state by plating on methylene blue (0.065 g/liter); 2) differences in exopolysaccharides by plating on Congo Red (0.15 g/liter); and 3) loss of motility by plating cells on soft agar (0.25% DIFCO). For more information see <xref ref-type="supplementary-material" rid="pgen.1004872.s013">S1 Text</xref>.</p><p>Genomic DNA of the two ancestral strains was isolated using the Ultra Clean Microbial DNA isolation Kit (Mo Bio Laboratories, Inc.) and sequenced on the Roche 454 platform according to standard sequencing methods. The reference sequences of <italic>E. coli</italic> RU1 and <italic>C. freundii</italic> RU2 were assembled <italic>de novo</italic> using Newbler v2.6 and annotated using RAST <xref ref-type="bibr" rid="pgen.1004872-Aziz1">[80]</xref>. The annotated genes were grouped by function and assigned to subsystems using SEED <xref ref-type="bibr" rid="pgen.1004872-Overbeek1">[81]</xref> (<xref ref-type="supplementary-material" rid="pgen.1004872.s006">S6 Figure</xref>).</p><p>Genomic DNA of single clones (MA lines: MA1&#8211;MA12) and whole populations (LB1&#8211;LB12, LB25&#8211;LB36, and BHI1&#8211;BHI24) was isolated as described for the ancestors and sequenced on Illumina HiSeq in 100 bp paired-end reads. Reads were aligned to the reference sequence using the breseq-0.24.rc6 pipeline (with options &#8211;j2 &#8211;c &#8211;p)<xref ref-type="bibr" rid="pgen.1004872-Deatherage1">[82]</xref>. Mutations that occurred at a minimum frequency of 0.05, which is an order of magnitude above the overall error rate as determined following Saxer et al. <xref ref-type="bibr" rid="pgen.1004872-Saxer3">[44]</xref> were considered. Because we used such a low acceptance threshold for mutations in polymorphic populations, we used additional measures to reduce the false positive rate, that can be due to errors in the reference sequence, duplications that were incorrectly resolved in the reference sequence, repeat regions or sequence properties, or methylation patterns <xref ref-type="bibr" rid="pgen.1004872-Ross1">[83]</xref>. Mutations arise at random, and under a model of selective neutrality, the frequency of a mutation should be proportional to the time it arose, while the frequency of a beneficial mutation should be determined by the time it arose and by its selection coefficient. Therefore, mutations that arise at random cannot be expected to occur at almost the same frequency in replicated polymorphic populations. In clonal samples, polymorphic sites could arise when a single colony is grown to high density for DNA isolation. However, such mutations should be rare within a single sample and very unlikely to occur in multiple independent clonal samples. In addition, the relaxed selection experience by the MA lines makes it unlikely that the same mutation would arise in multiple independently evolved lines. While beneficial mutations can evolve more than once in mutation accumulation experiments, it is unlikely that such mutations would occur at the exact same site (nucleotide) at approximately the same frequencies in independently evolved mutation accumulation lines, and that these same mutations would be polymorphic in a clonal sample (i.e. at lower frequency than 0.8&#8211;0.9 among reads aligning to that site).</p><p>We aligned the reads of the clonal MA lines as if they were polymorphic populations and identified mutations that occurred in 4 or more lines. These sites were removed from BHI- and LB-evolved populations. We performed the same removal procedure for mutations that occurred in one to twelve MA lines and observed a significant change in the number of mutations removed based on their presence in three or four MA lines. This resulted in the removal of 233 mutations that occurred in three treatments (LB, BHI and MA), 72 mutations that occurred in two treatments (MA and LB, or MA and BHI) and 3 mutations that occurred in the MA lines. The majority of these mutations were in intergenic regions, in transposable elements, hypothetical genes and in the large and small ribosomal subunits. In addition, we removed mutations in genes encoding for genes annotated as core proteins, the Magnesium and cobalt transport protein, CorA and tRNA dihydrouridine synthase A.</p><p>Based on our experience with <italic>E. coli</italic> and the work of Lang et al. <xref ref-type="bibr" rid="pgen.1004872-Lang1">[84]</xref>, we used a different approach to filter out false positive mutations for <italic>C. freundii</italic>. Adaptive mutations that arise in multiple populations evolve at different times and in populations with a different assortment of genotypes, which is reflected in different frequencies in the populations. The frequencies of false positive mutations that are a result of systematic errors on the other hand will be very similar among populations. Therefore, we filtered out any site for which the frequencies in BHI and LB did not significantly differ based on Levene's test <xref ref-type="bibr" rid="pgen.1004872-Whitlock1">[85]</xref>.</p><p>Cell extracts and spent media samples were prepared by growing the evolved populations and ancestral populations to stationary phase, separating the cells from the supernatant by centrifugation and inactivating remaining cells with 70% (v/v) ethyl alcohol.</p><p>Protein was extracted by re-suspending cell pellets in 50 &#181;L lysis buffer [6 M urea (Sigma U-0631) and 14.3 mM 2-mercaptoethanol (Sigma M6240) in 100 mM triethylammonium bicarbonate (TEAB), pH 9 (Sigma T7408)]. The extracted protein was digested with trypsin and the tryptic peptides analyzed by LC-MS using the accurate mass and time (AMT) tag proteomics approach <xref ref-type="bibr" rid="pgen.1004872-Zimmer1">[86]</xref>. See <xref ref-type="supplementary-material" rid="pgen.1004872.s013">S1 Text</xref> for details. The software program DAnTE <xref ref-type="bibr" rid="pgen.1004872-Polpitiya1">[87]</xref> was employed to perform an abundance roll-up procedure to convert peptide abundance information to protein information, thereby inferring protein abundances. ANOVA analyses were applied to protein abundance data sets (<italic>p</italic>-value &#8804;0.01) to identify statistically significant differences in protein expression levels (<xref ref-type="supplementary-material" rid="pgen.1004872.s011">S2 Table</xref>, <xref ref-type="supplementary-material" rid="pgen.1004872.s009">S9 Figure</xref>).</p><p>Putrescine concentration in the cell extract and in spent media was measured during GC-MS analyses of carbohydrates. For specific details see <xref ref-type="supplementary-material" rid="pgen.1004872.s013">S1 Text</xref>.</p>