<p>The FX PM and the <italic>PolB</italic>
<sup>c</sup> mice were generated as previously described [<xref ref-type="bibr" rid="pgen.1005181.ref029">29</xref>,<xref ref-type="bibr" rid="pgen.1005181.ref054">54</xref>]. Mice were maintained in accordance with the guidelines of the NIDDK Animal Care and Use Committee and with the <italic>Guide for the Care and Use of Laboratory Animals</italic> (NIH publication no. 85&#8211;23, revised 1996). No <italic>PolB</italic>
<sup><italic>c/c</italic></sup> mice were ever obtained from <italic>PolB</italic>
<sup><italic>+/C</italic></sup> crosses in the C57BL/6 background or after backcrossing for more than four generations onto a 129S1/SvImJ background. Since there is data to suggest that there would be fewer expansions in 129S1 background than a C57BL/6 background [<xref ref-type="bibr" rid="pgen.1005181.ref031">31</xref>], we therefore confined our analysis to heterozygous animals in the C57BL/6 background.</p><p>Genomic DNA from tails and different mouse organs was extracted using a Maxwell&#174;16 Mouse tail DNA purification kit (Promega, Madison, WI) according to the manufacturer&#8217;s instructions. Sperm was isolated from mouse epididymis using standard procedures. Pol&#946; genotyping was carried out by PCR analysis of tail DNA using the forward primer Y265C-F1: 5&#8242;AGAAAAGCAGCTTCCAGCAG and reverse primer Y265C-R4: 5&#8242;CAGACTTTCCAAGTGCAGGAT. PCR was carried out at 95&#176;C for 3 min followed by 30&#8211;40 cycles of 95&#176;C denaturation for 30 s, 60&#176;C annealing for 15 s, and 72&#176;C extension for 1 min. The WT allele produced a 440 bp PCR product and the mutant a 490 bp product [<xref ref-type="bibr" rid="pgen.1005181.ref029">29</xref>]. The presence of the expanded CGG&#8226;CCG-repeat tract was determined as described previously using the frax-m4 (5&#8217;-CTTGAGGCCCAGCCGCCGTCGGCC-3&#8217;) and frax-m5 (5&#8217;-CGGGGGGCGTGCGGTAACGGCCCAA-3&#8217;) primers [<xref ref-type="bibr" rid="pgen.1005181.ref055">55</xref>]. The primer binding sites are located immediately adjacent to the repeat and the PCR product corresponds to the length of the repeat with 49 bases of flanking sequence. For genotyping the PCR products were resolved by electrophoresis on a 1.5% agarose gel and stained with ethidium bromide.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For determination of the repeat number the repeat was amplified using as described previously except that primer frax-m4 was labeled with 6-carboxyfluorescein (FAM) [<xref ref-type="bibr" rid="pgen.1005181.ref055">55</xref>]. For determination of the repeat number or profile in bulk DNA, 50&#8211;100 ng of DNA was used for the PCR. For small pool PCR analysis from sperm, the DNA was diluted to 3pg/&#956;l (roughly 1 haploid genome equivalent/&#956;l). The diluted DNA was then subjected to nested PCR. The first round of PCR was carried out using the primers frax-C (5&#8217;-gctcagctccgtttcggtttcacttccggt-3&#8217;) and frax-F (5&#8217;-agccccgcacttccaccaccagctcctcca-3&#8217;) in a 25 &#956;l reaction using the same PCR conditions used previously [<xref ref-type="bibr" rid="pgen.1005181.ref055">55</xref>]. One microliter of this PCR reaction was then subjected to a second round of PCR using frax-m4 and frax-m5. Our PCR conditions allow us to amplify a wide range of repeat lengths without significant allele dropout or bias. Under these conditions &lt;50% of samples yielded a PCR product. This is consistent with the idea that each positive PCR reaction likely represented the products of amplification of DNA from a single sperm cell. The total number of positive PCR reactions was considered to represent the total number of alleles analyzed. The PCR products were resolved by capillary electrophoresis on an ABI Genetic Analyzer and the PCR profiles analyzed using GeneMapper Software 5. Alleles that were smaller, larger or the same size as the parental allele were counted and the proportion of each allele size class calculated as a fraction of the total number of alleles analyzed. The somatic instability index from different organs was calculated using 3 mice of each genotype as previously described [<xref ref-type="bibr" rid="pgen.1005181.ref036">36</xref>]. Statistical analysis of these data were carried out using the Graphpad Quickcalcs website (<ext-link ext-link-type="uri" ns0:href="http://www.graphpad.com/quickcalcs/">http://www.graphpad.com/quickcalcs/</ext-link>). The 95% confidence intervals for the proportion of expansions, contractions and unchanged alleles were determined using the Graphpad implementation of the modified Wald method. The significance of the differences in the frequency with which these classes were observed in <italic>PolB</italic>
<sup><italic>+/+</italic></sup> mice and <italic>PolB</italic>
<sup><italic>+/C</italic></sup> mice was determined using Fisher&#8217;s exact test. The significance of the differences in the SII and the distribution of derivative alleles in gametes was determined using Student&#8217;s t-test.</p><p>Protein extracts were prepared from flash frozen tissues. Tissues were homogenized using a tissue homogenizer (Precellys&#174; 24,Bertin Technologies, Rockville, MD) with T-PER protein extraction reagent (Pierce Biotechnology, Inc, Rockford, IL) supplemented with complete, Mini, EDTA-free protease inhibitor cocktail (Roche Applied Science, Indianapolis, IN) and phosphatase inhibitor cocktail-3 (Sigma-Aldrich, St. Louis, MO) according to the supplier&#8217;s instructions. The protein concentration was determined using a Bio-Rad protein assay kit (Bio-Rad, Hercules, CA). Because of tissue-specific differences in the expression of proteins often used as normalizing controls, including &#946;-actin, tubulin and GAPDH, care was taken to load equal amounts of protein in each gel lane. Prior to loading, the protein samples were heated for 10 min at 70&#176;C in LDS-Sample Buffer (Life Technologies, Grand Island, NY) supplemented with 50 mM DTT. The proteins were then resolved by electrophoresis on 3&#8211;8% NuPAGE Novex Tris-Acetate gels (Life Technologies) and electro-blotted onto nitrocellulose membranes using NuPAGE Transfer Buffer (Life Technologies) supplemented with 10% methanol. Transfer was carried out at 100 V at room temperature for 1 hour. Membranes were blocked for one hour at room temperature in 5% ECL Prime blocking agent (GE Healthcare Bio-Sciences, Pittsburgh, PA) in TBST pH 7.5 (10 mM Tris-HCl, 0.15 mM NaCl and 0.01% Tween 20), then incubated overnight at 4&#176;C with antibodies to the following proteins APE1 (ab137708), DNA ligase 1 (ab76232), FEN1 (ab70815); NEIL1 (ab21337) and Pol&#946; (ab26343) from Abcam (Cambridge, MA); OGG1 (15125-1-AP) from Proteintech Group (Chicago, IL) and DNA ligase 3 (611876) from BD Biosciences (Sparks, MD). After incubation with the appropriate secondary antibody and the addition of the ECL Prime detection reagent (GE Healthcare Bio-Sciences), the blot was imaged using a Fluorchem M imaging system (Proteinsimple, Santa Clara, CA).</p>