<p>All human research was approved by the relevant institutional review boards, and conducted according to the Declaration of Helsinki and all patients provided written informed consent. FIN-D2D 2007, DPS, DR&#8217;s EXTRA, FINRISK 2007, FUSION, and METSIM were approved by the University of Michigan Health Sciences and Behavioral Sciences Institutional Review Board (ID: H03-00001613-R2). The Danish studies (Health 2006, Inter99, and Vejle Biobank) were approved by the local Ethical Committees of Capital Region (approval # H-3-2012-155, KA 98155 and KA-20060011) and Region of Southern Denmark (approval # S-20080097). The GoDARTS study was approved by EoS REC 09/S1402/44. The Twins UK study was approved by EC04/015. The OBB study was a approved by South Central, Oxford C, 08/H0606/107+5, IRAS project 136602. The PIVUS study is approved by 00&#8211;419 and ULSAM study by 251/90 and 2007/338. The PPP study was approved by the Committee On the Use of Humans as Experimental Subjects at MIT (IRB 0912003615).</p><p>
<bold>Study participants.</bold> FG was measured in mmol/L and FI in pmol/L. Individuals were excluded from the analysis if they had a physician diagnosis of diabetes, were on diabetes treatment (oral or insulin), or had a fasting plasma glucose concentration &#8805;7 mmol/L or &#8805;11.1 mmol/L following a 2-hour oral glucose tolerance test. Individual studies applied further sample exclusions, including for pregnancy, non&#173;fasting measurements, and type 1 diabetes (<xref ref-type="supplementary-material" rid="pgen.1004876.s004">S1 Table</xref>). Measures of fasting glucose made in whole blood were corrected to approximate plasma level by multiplying by 1.13 [<xref ref-type="bibr" rid="pgen.1004876.ref035">35</xref>].</p><p>
<bold>Trait transformations and adjustment.</bold> To achieve approximate normality of the traits within each study, FG and natural logarithm&#8211;transformed FI levels were adjusted for age, sex, BMI, and study specific covariates followed by inverse normalization of the residuals. Inverse normalized residuals were used as the dependent quantitative trait in genetic association models to calibrate type 1 error. Effect estimates were obtained using untransformed FG and natural logarithm&#8211;transformed FI levels after adjusting for age, sex, BMI, and study specific covariates.</p><p>The Illumina HumanExome Beadchip array was genotyped by individual studies. This custom array was designed to facilitate large-scale genotyping of 247,870 mostly rare (MAF&lt;0.5%) and low-frequency (MAF 0.5&#8211;5%) protein altering variants selected from sequenced exomes and genomes of ~12,000 individuals (see <xref ref-type="sec" rid="sec004f">URLs</xref>). Details of genotype calling and quality control are presented in <xref ref-type="supplementary-material" rid="pgen.1004876.s004">S1 Table</xref>. To confirm the genotyping quality of all the variants discussed in the manuscript, we compared the genotype calls in 2,312 to 4,000 individuals for which we have both exome-array genotypes and exome-sequence data (<xref ref-type="supplementary-material" rid="pgen.1004876.s011">S8 Table</xref>). The results are highly concordant for all variants (99% heterozygous genotype concordance and 100% concordance observed for non-reference homozygotes). Call rate and Hardy-Weinberg equilibrium p-values for each variant are provided in <xref ref-type="supplementary-material" rid="pgen.1004876.s012">S9 Table</xref>.</p><p>
<bold>Single-variant analysis.</bold> We tested single variants for association with FG- and FI-derived inverse normalized residuals assuming an additive genetic model using a linear mixed model to account for relatedness with EMMAX [<xref ref-type="bibr" rid="pgen.1004876.ref005">5</xref>]. Study-specific QQ plots and genomic lambdas are shown in the <xref ref-type="supplementary-material" rid="pgen.1004876.s003">S3 Fig.</xref> We repeated single-variant association analyses with untransformed FG and natural logarithm&#8211;transformed FI residuals to obtain effect estimates. We then combined the association summary statistics across studies by using a fixed-effects meta-analysis (sample size Z-score weighted) using METAL [<xref ref-type="bibr" rid="pgen.1004876.ref036">36</xref>]. Genotype cluster plots of all variants described here were inspected visually in all studies.</p><p>
<bold>Single-variant conditional analysis.</bold> Conditional analysis was performed to identify additional association signals at known or novel loci. The analysis included the genotypes of the lead variant(s) at the conditioning loci as covariate(s) in the regression analysis in EMMAX. We then performed meta-analysis of the association summary statistics across studies by using a fixed-effects meta-analysis (sample size Z-score weighted).</p><p>
<bold>Gene-based analysis.</bold> For gene-based testing, we first computed single-variant score statistics and their covariance matrices for all polymorphic markers within each study. We then combined the single-variant score statistics from all studies using the Cochran-Mantel-Haenszel method and computed corresponding variance-covariance matrices centrally [<xref ref-type="bibr" rid="pgen.1004876.ref006">6</xref>]. These variance-covariance matrices were used to compute gene-level statistics. We applied a frequency-weighted burden test which assumes variants have similar effect sizes and SKAT, a dispersion test that performs well when both protective and deleterious variants are present [<xref ref-type="bibr" rid="pgen.1004876.ref008">8</xref>, <xref ref-type="bibr" rid="pgen.1004876.ref009">9</xref>]. Test-specific asymptotic distributions were used to evaluate significance. For gene-based analyses, we used only unrelated individuals (n = 32,223 for FG and n = 29,848 for FI) and included principal components as covariates to adjust for population structure. We generated four variant lists using frequency data and functional annotation. Variants were mapped to transcripts in Ensembl 66 (GRCh37.66). Using annotations from CHAoS v0.6.3, SnpEff v3.1, and VEP v2.7, we identified variants predicted to be protein-truncating (PTV; e.g. nonsense, frameshift, essential splice site) or protein-altering (e.g. missense, in-frame indel, non-essential splice site) in at least one mapped transcript (by at least one of the three algorithms) [<xref ref-type="bibr" rid="pgen.1004876.ref037">37</xref>, <xref ref-type="bibr" rid="pgen.1004876.ref038">38</xref>]. We additionally used the procedure described by Purcell <italic>et al</italic>. to identify subsets of missense variants meeting &#8220;strict&#8221; or &#8220;broad&#8221; criteria for being deleterious, using annotation predictions from Polyphen2-HumDiv, PolyPhen2-HumVar, LRT, MutationTaster, and SIFT [<xref ref-type="bibr" rid="pgen.1004876.ref010">10</xref>]. Masks 1 (&#8220;PTV-only&#8221;) and 3 (&#8220;PTV + NS<sub>strict</sub>&#8221;) are restricted to variants with predicted major effect on protein function, and, as a result, disproportionately favor inclusion of rare variants, whilst masks 2 (&#8220;PTV + missense&#8221;) and 4 (&#8220;PTV + NS<sub>broad</sub>&#8221;) are more permissive. In total, we tested 1,028, 14,465, 4,603, and 13,093 genes having at least two such variants in the above four categories respectively. For gene-based tests reaching exome-wide significance, if the conditional analysis showed that the signal was driven by a single variant, we required the variant to achieve exome-wide significance in the single-variant test as well.</p><p>
<bold>Gene-based conditional analysis.</bold> We performed conditional gene-level analysis to evaluate whether rare or low-frequency variants associated with the trait in single-variant analysis could account for or were due to a gene-based test association signal [<xref ref-type="bibr" rid="pgen.1004876.ref006">6</xref>]. The genotypes of the variant(s) at the conditioning locus were included as covariate(s) in this analysis.</p><p>
<bold>Estimating phenotypic variance explained by SNPs.</bold> We used a subset of Finnish samples (FIN-D2D 2007, DPS, DR&#8217;s EXTRA, FINRISK 2007, and METSIM; n = 10,266) to calculate variance explained by <italic>G6PC2</italic>. We ran a model regressing BMI adjusted FG on the four <italic>G6PC2</italic> variants: intronic GWAS lead SNP rs560887, p.His177Tyr (rs138726309), p.Tyr207Ser (rs2232323), and p.Val219Leu (rs492594), assuming an additive model (and adjusting for sex, age, age<sup>2</sup>, and study origin). A separate model was run excluding the GWAS SNP to determine the additional variance captured with the three coding variants.</p><p>
<bold>Power calculations.</bold>We had &gt;99.9% power to identify variants that explain &gt;0.3% of the phenotypic variance and 80% power to detect coding variants that explain &gt;0.1% of the phenotypic variance. To achieve &gt;80% power for variants with MAF&lt;0.05% would require effect sizes of at least 1 SD unit of residuals of mmol/L for FG and pmol/L for FI.</p><p>When we estimate power to detect association for aggregation tests, we make many assumptions: i) proportion of variants contributing to trait variation, ii) direction of effects, iii) number of variants aggregated, and iv) allele frequency distribution of the variants [<xref ref-type="bibr" rid="pgen.1004876.ref034">34</xref>]. In this study, assuming 100% of the variants contribute to trait variation, we had &gt;99.9% power to detect association for genes that explain &gt;0.25% of the phenotypic variance and 80% to detect genes that explain &gt;0.1% of the phenotypic variance using a burden test and &gt;0.5% of the phenotypic variance using SKAT. In a less favorable scenario, for example assuming a gene explains 0.25% of the phenotypic variance, 25% of the variants contribute to trait variation, different directions of effect, 20 variants tested, and variants sampled from the reported allele frequency distribution in the exome-array design, power to detect association in this study may be near 0% for a burden test and 63% for SKAT [<xref ref-type="bibr" rid="pgen.1004876.ref034">34</xref>].</p><p>
<bold>Haplotype analysis.</bold> In 4,442 individuals from the Oxford Biobank, we used an expectation-maximization (EM) algorithm to obtain the posterior distribution of haplotypes consistent with the observed genotypes at four <italic>G6PC2</italic> variants: intronic GWAS lead SNP rs560887, p.His177Tyr (rs138726309), p.Tyr207Ser (rs2232323), and p.Val219Leu (rs492594). Haplotype association with FG- and FI-derived residuals (after adjustment for age, sex, and BMI) was tested in a linear regression framework, as a function of haplotype dosage from posterior distribution, and including principal components as covariates to account for population structure using the most frequent haplotype as baseline.</p><p>
<bold><italic>In-silico</italic> mutation analysis.</bold> SIFT [<xref ref-type="bibr" rid="pgen.1004876.ref039">39</xref>], PolyPhen-2 [<xref ref-type="bibr" rid="pgen.1004876.ref040">40</xref>], and Condel [<xref ref-type="bibr" rid="pgen.1004876.ref041">41</xref>] algorithms were used to predict the functional effects of the associated non-synonymous variants on protein function. Genomic Evolutionary Rate Profiling (GERP) [<xref ref-type="bibr" rid="pgen.1004876.ref042">42</xref>] scores were calculated to indicate the degree of evolutionary conservation at a given human nucleotide based on multiple genomic sequence alignments and were measured as site-specific &#8216;rejected substitutions&#8217;: higher scores indicate greater conservation.</p><p>
<bold>Site directed mutagenesis.</bold> Human <italic>G6PC2</italic> cDNA (NM_021176.2) within a pCMV6-Entry vector (with a C-terminal FLAG-tag) was purchased from OriGene (RC211146). We generated non-synonymous variants in the <italic>G6PC2</italic> coding sequence of the clone using Quikchange II Site-Directed Mutagenesis (Agilent). All mutations were verified by Sanger sequencing; in each case, only the desired nucleotide substitutions was introduced.</p><p>
<bold>Western blot analyses.</bold> HEK293 and INS-1E 832&#8211;13 cells were transfected with each FLAG-tagged wild type or mutant G6PC2 construct using Lipofectamine 2000 (Invitrogen). In protein degradation assays, cells were treated with 10&#956;M MG-132 (Calchembio) or 100&#956;M chloroquine (Sigma) for 15h. At 48h after transfection, cells were collected and homogenized by sonication in lysis buffer. Total cellular protein was separated by 4&#8211;12% SDS-PAGE (Invitrogen) and transferred to nitrocellulose membranes. We determined G6PC2 expression by immunoblotting using a mouse monoclonal FLAG M2 antibody (Sigma, F1804). A rabbit polyclonal antibody against &#946; tubulin (Santa Cruz, sc-9104) was used as a loading control. Secondary antibodies specific to mouse or rabbit IgG were obtained from Thermo Fisher Scientific. Protein bands were detected using the ECL reagent (Pierce Thermo Fisher Scientific). Western blots were quantified by densitometry analysis using ImageJ and paired t tests of densitometric data were carried out in GraphPad Prism.</p><p>
<bold>Immunofluorescence.</bold> HEK293 cells were transfected with each FLAG-tagged G6PC2 construct using FuGene 6 transfection reagent (Promega) in 4-well chamber slides (BD Biosciences). After 48h, cells were fixed with 4% paraformaldehyde in PBS for 15 min. Cells were permeabilized with 0.05% Triton X-100 in PBS for 15 min, and blocked for 1 h with 10% BSA in PBS-Tween 20. We carried out double immunostaining of cells using FLAG M2 (Sigma, F1804) and calnexin (Santa Cruz, sc-11397) primary antibodies followed by anti-mouse fluorescein (Vector Labs) and anti-rabbit TRITC (Dako). DRAQ5 fluorescent probe (Thermo Fisher Scientific) was applied at 20&#956;M as a far-red DNA stain. Slides were mounted with Vectashield mounting medium (Vector Labs) and visualized on a BioRad Radiance 2100 confocal microscope with a 60&#215; 1.0 N.A. objective. Images were acquired with different laser settings that were optimized for each sample and therefore fluorescent intensities are not comparable across samples.</p><p>Exome-array design,</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://genome.sph.umich.edu/wiki/Exome_Chip_Design">http://genome.sph.umich.edu/wiki/Exome_Chip_Design</ext-link>
</p><p>EPACTS,</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://genome.sph.umich.edu/wiki/EPACTS">http://genome.sph.umich.edu/wiki/EPACTS</ext-link>
</p><p>METAL,</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/Metal/download/">http://www.sph.umich.edu/csg/abecasis/Metal/download/</ext-link>
</p><p>RareMETALS,</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://genome.sph.umich.edu/wiki/RareMETALS">http://genome.sph.umich.edu/wiki/RareMETALS</ext-link>
</p><p>The 1000 Genomes Project,</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://www.1000genomes.org">www.1000genomes.org</ext-link>
</p>