<p>Sequencing reads from 3 Southeast Asian (Cambodia, n = 253; Thailand, n = 91; Vietnam, n = 12) and 2 African populations (Gambia, n = 55; Malawi, n = 60) of <italic>P</italic>. <italic>falciparum</italic> clinical isolates [<xref ref-type="bibr" rid="pgen.1005131.ref002">2</xref>] were aligned to the <italic>P</italic>. <italic>falciparum</italic> 3D7 genome (version 3) using <italic>smalt</italic> (described in [<xref ref-type="bibr" rid="pgen.1005131.ref002">2</xref>]). For all samples the average coverage across the whole genome was at least 35-fold. Variants were called using <italic>samtools</italic> [<xref ref-type="bibr" rid="pgen.1005131.ref039">39</xref>] and <italic>vcftools</italic> [<xref ref-type="bibr" rid="pgen.1005131.ref040">40</xref>] with default settings. Genotypes at SNP positions were called using ratios of coverage, with heterozygous calls converted to the majority genotype on a 70:30 or greater coverage ratio [<xref ref-type="bibr" rid="pgen.1005131.ref004">4</xref>]. SNPs that reflected possible multiplicity of infection were removed as the imputation methods under evaluation cannot tolerate haploid and mixed genotypes simultaneously. Low-coverage <italic>var</italic> and subtelomeric regions (<xref ref-type="supplementary-material" rid="pgen.1005131.s020">S9 Table</xref>) were discarded. The 85,967 biallelic and non-singleton SNPs with less than 10% missing genotypes were retained, which is approximately the inflection point beyond which SNPs experience a rapid increase in missingness (<xref ref-type="supplementary-material" rid="pgen.1005131.s011">S11 Fig</xref>). Allowing more missing data by relaxing the threshold would lead to a decrease in imputation accuracy [<xref ref-type="bibr" rid="pgen.1005131.ref017">17</xref>].</p><p>Performance of the imputation methods was evaluated with a leave-one-out cross-validation approach using markers on chromosome 13, where 215 (~2%) SNPs were masked in each haplotype and imputed using the others as references. This procedure produces an accuracy measure based on the squared Pearson correlation (allelic <italic>r</italic>
<sup><italic>2</italic></sup>) between the true and imputed allele probability for each marker [<xref ref-type="bibr" rid="pgen.1005131.ref018">18</xref>]. To prevent sampling bias from choosing only markers with completely known genotypes, each population panel was split into several subpopulations of 25&#8211;30 isolates each, and independent rounds of cross-validation were performed on a distinct set of markers in each subpopulation. This procedure introduced representation of markers at varying levels of missingness, and increased the total number of validated markers. A concordance rate between true and best-guess genotypes on a calling threshold &#8805;0.9 was also calculated. Only the concordance of minor alleles is considered to mitigate the possibility of near-monomorphic SNPs inflating overall concordance rates.</p><p>Imputation software established primarily for human genotype analysis were adapted to enable their application to parasite sequence data. Beagle v3.3 [<xref ref-type="bibr" rid="pgen.1005131.ref018">18</xref>] does not require recombination rates to inform its LD model and processes sporadically missing genotypes, but only imputes diploid genotypes. Therefore, we duplicated all positions to make sequences homozygous diploid before each imputation round, then heterozygous imputed genotype probabilities were allocated equally between the 2 alternative alleles when computing allelic dosage. For IMPUTE v2.0, controls allowing haploid imputation (-<italic>chrX</italic>, -<italic>use_prephased_g</italic>, -<italic>known_haps_g)</italic> were activated. Previously published recombination maps from 2 high-quality genetic crosses between laboratory clones GB4x7G8 [<xref ref-type="bibr" rid="pgen.1005131.ref006">6</xref>,<xref ref-type="bibr" rid="pgen.1005131.ref021">21</xref>,<xref ref-type="bibr" rid="pgen.1005131.ref038">38</xref>,<xref ref-type="bibr" rid="pgen.1005131.ref041">41</xref>] and HB3xDD2 [<xref ref-type="bibr" rid="pgen.1005131.ref006">6</xref>,<xref ref-type="bibr" rid="pgen.1005131.ref022">22</xref>,<xref ref-type="bibr" rid="pgen.1005131.ref038">38</xref>,<xref ref-type="bibr" rid="pgen.1005131.ref041">41</xref>] were used to supply 2 sets of experimentally informed rates. These two maps were considered separately, and not combined. An alternative set of population recombination rates (2<italic>N</italic>
<sub><italic>e</italic></sub>
<italic>r</italic>) was inferred using the coalescent-based LDhat program, <italic>interval</italic> [<xref ref-type="bibr" rid="pgen.1005131.ref042">42</xref>]. Pre-supplied likelihood lookup files with a population mutation rate (<italic>&#952;</italic>) value of 0.01 were used. The effective population size (<italic>N</italic>
<sub><italic>e</italic></sub> = 30,000) was derived by re-scaling the compound map distance for Gambian isolates inferred from <italic>interval</italic>, to match the empirical map distance estimated from the GB4x7G8 cross. The scaling factor was treated as a pseudo-<italic>N</italic>
<sub><italic>e</italic></sub> used to adjust compound rates uniformly on the other populations. To test the robustness of assuming the above parameter values for <italic>&#952;</italic> and <italic>N</italic>
<sub><italic>e</italic></sub>, we computed accuracy using different values for one African (Gambia) and one Southeast Asian (Thailand) population (<xref ref-type="supplementary-material" rid="pgen.1005131.s013">S2 Table</xref>).</p><p>Multiple haplotype sets were sampled from the marginal posterior genotype distribution under the best-performing imputation designs. This procedure allowed the computation of population genetics metrics under a multiple imputation-like schema, accounting for uncertainty in the true identity of genotypes. Ten SNP datasets were completed for each imputation strategy through sampling from posterior genotype probabilities, and relevant population genetics metrics at each SNP or gene were computed and averaged across the sets. Two metrics were used to infer recent positive selection, the integrated haplotype score (<italic>iHS)</italic> and <italic>Rsb</italic>. <italic>iHS</italic> is the standardized log ratio of the integrated extended-haplotype homozygosity (EHH) between the ancestral and derived allele at a core SNP [<xref ref-type="bibr" rid="pgen.1005131.ref043">43</xref>], capturing evidence of unusually long haplotypes surrounding a particular allele within the population. The ancestral and derived alleles were defined to be the major and minor allele at Malawian SNPs, as this population is ancestrally ancient [<xref ref-type="bibr" rid="pgen.1005131.ref006">6</xref>,<xref ref-type="bibr" rid="pgen.1005131.ref016">16</xref>] and the rapid LD decay made it a natural reference point for detecting extended haplotype homozygosity in other populations. Absolute values of <italic>iHS</italic> were used to capture long haplotypes centred around either type of allele, using a minimum MAF filter of 5%. <italic>Rsb</italic> is the standardized log ratio of the integrated site-specific EHH between populations at a core SNP, where site-specific EHH refers to the weighted average of EHH surrounding a core SNP according to squared-allele frequencies [<xref ref-type="bibr" rid="pgen.1005131.ref026">26</xref>]. Because site-specific EHH does not require markers to be polymorphic within the population, it can detect selective sweeps for alleles that have risen to fixation. Both <italic>iHS</italic> and <italic>Rsb</italic> were calculated using the <italic>scan_hh</italic> program in the R package <italic>rehh</italic> [<xref ref-type="bibr" rid="pgen.1005131.ref044">44</xref>], which excludes markers with any missing data. The allele frequency-based <italic>Tajima's D</italic> approach [<xref ref-type="bibr" rid="pgen.1005131.ref011">11</xref>] was used to detect balancing selection at each gene, with greater evidence from increasing values from zero. Whilst different populations have different demographic histories, it is difficult to account for this and establish multiple thresholds for the different null distributions of <italic>Tajima&#8217;s D</italic>. To negate any confounding effects of population expansion and difficulties in establishing significance levels, we report genes with <italic>Tajima&#8217;s D</italic> values in excess of one [<xref ref-type="bibr" rid="pgen.1005131.ref006">6</xref>]. The function <italic>neutrality</italic>.<italic>stats</italic> from the R package <italic>PopGenome</italic> was used [<xref ref-type="bibr" rid="pgen.1005131.ref045">45</xref>]. Parasite population structure was analysed using principal components analysis, implemented via the classical multidimensional scaling method using the function <italic>cmdscale</italic> in R. In addition, pairwise population <italic>Fst</italic> with 95% bootstrap confidence intervals was computed using the function <italic>stamppFst</italic> in the R package <italic>StAMPP</italic> [<xref ref-type="bibr" rid="pgen.1005131.ref046">46</xref>].</p>