<p>The strains CB1157 (<italic>unc-54(e1157)</italic>), CL2166 (<italic>dvIs19[pAF15 (gst-4</italic>::<italic>GFP</italic>::<italic>NLS)]</italic>) [<xref ref-type="bibr" rid="pgen.1005823.ref053">53</xref>], CL2070 (<italic>dvIs70[hsp-16</italic>.<italic>2</italic>::<italic>GFP; rol-6 (su1006)]</italic>) [<xref ref-type="bibr" rid="pgen.1005823.ref054">54</xref>], DA2123 (<italic>adIs2122 [lgg-1p</italic>::<italic>GFP</italic>::<italic>lgg-1 + rol-6(su1006)]</italic>) [<xref ref-type="bibr" rid="pgen.1005823.ref110">110</xref>], HZ1688 (<italic>atg-13(bp414)</italic>) [<xref ref-type="bibr" rid="pgen.1005823.ref088">88</xref>], MAH236 (<italic>sqIs13 [lgg-1p</italic>::<italic>GFP</italic>::<italic>lgg-1 + odr-1p</italic>::<italic>RFP]</italic>) [<xref ref-type="bibr" rid="pgen.1005823.ref086">86</xref>], SJ4001 (<italic>zcIs1[aip-1</italic>::<italic>GFP]</italic>) [<xref ref-type="bibr" rid="pgen.1005823.ref037">37</xref>], and VC1369 (<italic>rpn-10(ok1865)</italic>) were obtained from the <italic>Caenorhabditis</italic> Genetics Center which is funded by NIH Office of Research Infrastructure Programs (P40 OD010440). The <italic>rrIs1[elt-2</italic>::<italic>lacZ</italic>::<italic>GFP]</italic> transgene which expresses nuclear localized GFP in the intestine has been previously described [<xref ref-type="bibr" rid="pgen.1005823.ref111">111</xref>]. The <italic>rpn-10(ok1865)</italic> mutant was outcrossed against N2 five times, and one resulting outcrossed homozygous line was used for all subsequent crosses and experiments. The presence of the <italic>rpn-10(ok1865)</italic> allele was determined by single-worm PCR using oligonucleotides designed to amplify the wild-type <italic>rpn-10</italic> allele (F 5&#8217;-AAGAGAACAACGCGCATCTT-3&#8217;; R 5&#8217;-GTGTGCCCCTTTGAGGAGTA-3&#8217;) and to detect the deletion present in the <italic>rpn-10(ok1865)</italic> allele (F 5&#8217;-CCCATTCCAATTGTTGCTCT-3&#8217;; R 5&#8217;-TGCACCAACAACTCCACATT-3&#8217;). The strain AM140 (<italic>rmIs132[unc-54p</italic>::<italic>Q35</italic>::<italic>YFP]</italic>) was kindly provided by Dr. Richard Morimoto and previously described by our group [<xref ref-type="bibr" rid="pgen.1005823.ref026">26</xref>, <xref ref-type="bibr" rid="pgen.1005823.ref112">112</xref>]. The strain AM446 (<italic>rmIs223[phsp70</italic>::<italic>gfp]; pRF4[rol-6(su1006)]</italic>) was kindly provided by Dr. Richard Morimoto [<xref ref-type="bibr" rid="pgen.1005823.ref055">55</xref>]. The strain PP608 <italic>(hhIs64[unc-119(+); sur-5</italic>::<italic>UbV-GFP]; hhIs73[unc-119(+); sur-5</italic>::<italic>mCherry])</italic> was kindly provided by Dr. Thorsten Hoppe [<xref ref-type="bibr" rid="pgen.1005823.ref036">36</xref>]. The strain BC14890 (<italic>sIs14010[rpn-7</italic>::<italic>GFP]</italic>) was kindly provided by Dr. David Baillie [<xref ref-type="bibr" rid="pgen.1005823.ref113">113</xref>]. The strain OG412 (<italic>drIs20[vha-6p</italic>::<italic>Q44</italic>::<italic>YFP + rol-6(su1006)]</italic>) was kindly provided by Dr. Todd Lamitina [<xref ref-type="bibr" rid="pgen.1005823.ref048">48</xref>]. The strain JM168 (<italic>elt-2(ca15); caIs20[elt-2p</italic>::<italic>elt-2</italic>::<italic>GFP</italic> + <italic>unc-119(+)]</italic>) was kindly provided by Dr. James McGhee. This strain was produced by gamma irradiation of JM73, which carried the transgene as an extrachromosomal array, to integrate the array into the genome followed by backcrossing to remove extraneous mutations [<xref ref-type="bibr" rid="pgen.1005823.ref114">114</xref>]. Strains containing transgenes, genetic mutations, and the <italic>rpn-10(ok1865)</italic> mutation were generated by standard crossing and genotyped by PCR.</p><p>Worms expressing a RPN-10::GFP fusion protein were generated via biolistic bombardment [<xref ref-type="bibr" rid="pgen.1005823.ref115">115</xref>]. The 6236103120536928 H12 fosmid clone which contains the entire <italic>rpn-10</italic> coding sequence in fosmid WRM0618DC02 fused to GFP at the C-terminus as well as &gt;10 kilobases of 5&#8217; and 3&#8217; flanking sequences was requested from the TransgenOme project, and the presence and location of the GFP insert was confirmed by PCR and sequencing [<xref ref-type="bibr" rid="pgen.1005823.ref033">33</xref>]. The fosmid was purified from <italic>E</italic>. <italic>coli</italic> and used to generate transgenic animals via bombardment using the DP38 (<italic>unc-119(ed3)</italic>) strain as previously described [<xref ref-type="bibr" rid="pgen.1005823.ref115">115</xref>, <xref ref-type="bibr" rid="pgen.1005823.ref116">116</xref>]. Transgenic animals were identified via rescue of the mobility and body size defects of the <italic>unc-119</italic> mutant. This resulted in the isolation of the ALF85 (<italic>bafEx85</italic>) transgenic strain, which was outcrossed with N2 and then used for further study.</p><p>The <italic>hsf-1</italic>, <italic>pas-6</italic>, <italic>pbs-6</italic>, <italic>pbs-7</italic>, <italic>rpn-12</italic>, <italic>skn-1</italic>, and <italic>wdr-23</italic> RNAi clones were previously described [<xref ref-type="bibr" rid="pgen.1005823.ref026">26</xref>]. The <italic>atg-13</italic>, <italic>atg-16</italic>.<italic>2</italic>, <italic>elt-2</italic>, <italic>prmt-1</italic>, vha<italic>-15</italic>, and <italic>rpn-10</italic> RNAi clones were retrieved from the Ahringer RNAi library and confirmed by sequencing [<xref ref-type="bibr" rid="pgen.1005823.ref117">117</xref>]. For RNAi treatment, NGA plates containing 50&#956;g/ml carbenicillin and 0.2% &#946;-lactose (in place of IPTG for dsRNA induction) were spotted with overnight cultures of RNAi bacteria inoculated from discrete individual colonies [<xref ref-type="bibr" rid="pgen.1005823.ref118">118</xref>]. Due to the adverse developmental effects of the <italic>elt-2</italic> and <italic>vha-15</italic> RNAi, these clones were typically diluted 1:10 with bacteria containing the empty vector control RNAi clone prior to spotting on the plates. Unless otherwise noted, eggs isolated by hypochlorite treatment were then placed on spotted RNAi plates and incubated at 20&#176;C.</p><p>Approximately 50&#8211;100 eggs isolated via hypochlorite treatment were placed on NGA plates spotted with <italic>E</italic>. <italic>coli</italic> strain OP50-1 and grown to adulthood at 20&#176;C. Digital images of day 1 adult worms were either captured with an Olympus BX51 upright microscope and DP70 camera as previously described or with a Nikon Eclipse Ti inverted microscope with a 14-bit CoolSNAP HQ2 (Photometrics) CCD camera and Nikon Elements software [<xref ref-type="bibr" rid="pgen.1005823.ref026">26</xref>, <xref ref-type="bibr" rid="pgen.1005823.ref119">119</xref>]. Fluorescence intensity of the respective reporters was then quantified using ImageJ and statistical analysis of the resulting image data was completed in Prism6 (GraphPad Software, San Diego, CA) [<xref ref-type="bibr" rid="pgen.1005823.ref120">120</xref>].</p><p>To perform heat-shock studies with the <italic>hsp-16</italic>::<italic>GFP</italic> and <italic>hsp-70</italic>::<italic>GFP</italic> reporters, approximately 20 day 1 adult animals drawn from the same synchronized populations used for the initial baseline imaging were transferred to fresh, spotted NGA plates. These worms were incubated at 35&#176;C for one hour and allowed to recover at 20&#176;C overnight (approximately 14 hours) before capturing the post-heat shock images.</p><p>To assess effects of the <italic>rpn-10</italic> mutation on GFP::LGG-1 expression the <italic>adIs2122 [lgg-1p</italic>::<italic>GFP</italic>::<italic>lgg-1 + rol-6(su1006)]</italic> transgene was outcrossed into N2 and <italic>rpn-10</italic> mutant animals via standard crosses. Synchronized day 1 adult animals were mounted and GFP fluorescence was measured via the analysis of digital images with ImageJ. To assess changes in autophagic activity, the <italic>sqIs13 [lgg-1p</italic>::<italic>GFP</italic>::<italic>lgg-1 + odr-1p</italic>::<italic>RFP]</italic> transgene was outcrossed into N2 and <italic>rpn-10</italic> mutant animals via standard crosses. Synchronized day 1 adult animals were mounted and the animals were photographed using a GFP filter set. GFP::LGG-1 puncta in individual seam cells were counted and then analyzed for mean and statistical significance using Prism6 (GraphPad Software, San Diego, CA). GFP::LGG-1 puncta in the intestine were counted by using the &#8220;Find Maxima&#8221; function in ImageJ, and the puncta counts were then analyzed for mean and statistical significance using Prism6.</p><p>Lifespan assays were conducted at 25&#176;C using either NGA or RNAi plates containing 50 &#956;M FUDR as previously described [<xref ref-type="bibr" rid="pgen.1005823.ref119">119</xref>]. All worms were synchronized by hypochlorite treatment, hatched, and grown to adulthood at 20&#176;C on NGA plates supplemented with streptomycin (0.2 mg/mL) and spotted with <italic>E</italic>. <italic>coli</italic> strain OP50-1. They were transferred to plates containing FUDR (and RNAi when appropriate) on the first day of adulthood and placed at 25&#176;C. The worms were transferred to a second FUDR plate on the second day and left at 25&#176;C for the remainder of the assay. Lifespan assays without RNAi treatment were conducted on NGA plates containing streptomycin (0.2 mg/mL) as well as FUDR (50 &#956;M) and spotted with OP50-1. RNAi treatment lifespans were conducted on NGA plates supplemented with FUDR, carbenicillin (50 mg/mL), and isopropyl &#946;-d-thiogalactopyranoside (IPTG,1 mM). These plates were spotted with OP50(xu363) bacteria, which is an OP50-derived bacterial strain that can deliver RNAi to worms, that had been transformed with RNAi-expressing plasmids [<xref ref-type="bibr" rid="pgen.1005823.ref121">121</xref>]. Prism6 (Graphpad Software) was used to generate graphs and perform log-rank testing for curve comparisons. STATA 8 was used to create lifetables and calculate mean survival.</p><p>The worms were synchronized by hypochlorite treatment and eggs were plated on NGA plates spotted with <italic>E</italic>. <italic>coli</italic> strain OP50-1 and grown to adulthood at 20&#176;C. Starting at 48 hours after synchronization, worms were scored for development to adulthood by microscopy every 8&#8211;16 hours until the entire population had reached adulthood. Three plates of approximately 100 worms each were scored for each genotype.</p><p>For heat stress assays, N2 and outcrossed <italic>rpn-10(ok1865)</italic> worms were synchronized via hypochlorite treatment and grown to the L4 stage on <italic>E</italic>. <italic>coli</italic> OP50-1-spotted NGA plates at 20&#176;C. Forty L4 individuals of each strain were then transferred to fresh plates in duplicate and incubated at 35&#176;C. Scoring for survival was performed every hour beginning six hours after initiation of heat stress, with dead worms being identified by lack of responsiveness to gentle prodding with a pick.</p><p>Oxidative stress assays using <italic>tert-</italic>butyl hydroperoxide (tBHP) were performed as described with 40 L4 animals being transferred to <italic>E</italic>. <italic>coli</italic> OP50-1-spotted NGA plates containing 7mM tBHP and scored three times per day using the parameters detailed above until all animals died or were censored [<xref ref-type="bibr" rid="pgen.1005823.ref122">122</xref>]. A minimum of two trials with comparable results were performed for each assay.</p><p>Oxidative stress assays using 5-Hydroxy-1,4-naphthoquinone (juglone) were performed using WT and <italic>rpn-10</italic> worms which were treated with control and <italic>skn-1</italic> RNAi from egg hatching for 48 hours. The worms were then washed from plates and suspended in M9 buffer. Juglone was added to a final concentration of 100 &#956;M from a 100X stock made fresh in 100% ethanol, and the worms were then exposed to juglone for one hour with nutation. The worms were then washed twice with S-basal and returned to NGA. They were scored for survival 48 hours later.</p><p>The aggregation of the muscle-expressed Q35::YFP fusion protein was assessed by adding eggs isolated from <italic>rmIs132[unc-54p</italic>::<italic>Q35</italic>::<italic>YFP]</italic> and <italic>rpn-10(ok1865); rmIs132[unc-54p</italic>::<italic>Q35</italic>::<italic>YFP]</italic> animals via hypochlorite treatment to NGA plates, and then incubating the plates at 23&#176;C for 72 hours [<xref ref-type="bibr" rid="pgen.1005823.ref112">112</xref>]. After this time essentially all of the animals were gravid adults. The number of individual Q35::YFP aggregates were scored using a fluorescent stereomicroscope as previously described, and digital images were captured at 6X magnification using a Nikon Eclipse inverted compound microscope equipped with epifluorescence illumination and a Nikon Endow GFP filter cube [<xref ref-type="bibr" rid="pgen.1005823.ref026">26</xref>]. The aggregation of the intestine-expressed Q44::YFP fusion protein was assessed by adding eggs isolated from <italic>drIs20[vha-6p</italic>::<italic>Q44</italic>::<italic>YFP + rol-6(su1006)]</italic> and <italic>rpn-10(ok1865); drIs20[vha-6p</italic>::<italic>Q44</italic>::<italic>YFP + rol-6(su1006)]</italic> animals via hypochlorite treatment to NGA plates, and then incubating the plates at 23&#176;C for 72 hours [<xref ref-type="bibr" rid="pgen.1005823.ref048">48</xref>]. At this point ~100 worms were transferred to either NGA plates containing 50 &#956;M FUDR or RNAi plates containing 50 &#956;M FUDR and spotted with the indicated RNAi clone, and the plates were returned to 23&#176;C for 4 days before scoring. The <italic>drIs20</italic> worms contained large numbers of aggregates at this point, so the percentage of animals with any aggregates present in the intestine was measured by scoring with a fluorescent microscope. For <xref ref-type="supplementary-material" rid="pgen.1005823.s015">S9 Fig</xref>, we also counted individual aggregates in the <italic>drIs20</italic> worms, and to enhance the aggregate numbers in the <italic>rpn-10</italic> mutant animals, we incubated the plates at 23&#176;C for 5 days instead of 4 days. The aggregate numbers were scored using a fluorescent microscope after briefly incubating the plates on ice to reduce worm activity.</p><p>A previously described protocol for assessing the phenotypic effects of shifts from permissive to non-permissive temperature on the function of the meta-stable UNC-54 protein in the <italic>unc-54(e1157)</italic> mutant was adapted for the analysis of larval animals [<xref ref-type="bibr" rid="pgen.1005823.ref049">49</xref>]. Briefly, synchronized L1 larval populations of <italic>unc-54(e1157)</italic>, and <italic>rpn-10</italic>, <italic>unc-54</italic> animals were spotted on NGA plates and grown at 16&#176;C (permissive temperature) for 24 hours. At this time, a total of approximately 300 worms per genotype were transferred to three separate fresh NGA plates, and shifted to 25&#176;C (non-permissive temperature) for 20 hours. The plates were then allowed to equilibrate at room temperature for 20 minutes before being scoring for paralysis by prodding with a platinum worm-pick. Worms that failed to respond to touch were scored as paralyzed.</p><p>Two transcription factor RNAi libraries were independently screened to identify transcription factors that permitted survival of the <italic>rpn-10(ok1865)</italic> mutant. These libraries were (1) a subset library sold by Source Bioscience which was created from Ahringer RNAi library clones, and (2) a library created from clones in the Ahringer and Vidal RNAi libraries (<xref ref-type="supplementary-material" rid="pgen.1005823.s006">S6 Table</xref>). For each screen, individual wells of 24-well plates containing NGA with 50&#956;g/ml carbenicillin and 0.2% &#946;-lactose were spotted with 20&#956;L of overnight culture for a clone in the transcription factor library and allowed to dry at room temperature. Each well was then seeded with 70&#8211;100 <italic>rpn-10(ok1865); zcIs1[aip-1</italic>::<italic>GFP]</italic> L1 larvae, which had hatched from eggs that were isolated by hypochlorite treatment and then placed in S-basal to arrest the progeny at the L1 larval stage. Each plate was then incubated at 20&#176;C and visually screened for phenotypic effects after three days. Clones that caused developmental arrest or sickness were then re-screened, and clones that were again found to produce these phenotypes in the second round were used to treat larger populations of <italic>zcIs1[aip-1</italic>::<italic>GFP]</italic> animals both with and without the <italic>rpn-10(ok1865)</italic> allele in order to find genes required for the development and survival of mutant but not wild-type animals. In this manner clones determined to be hits from the initial screen were sequentially narrowed down to those that consistently impaired normal development only in the <italic>rpn-10(ok1865)</italic> background.</p><p>Code sets that recognize the indicated genes were synthesized by Nanostring Technologies (Seattle, WA) and used with the Nanostring nCounter system to measure the levels of each transcript in 100 ng aliquots of total RNA. The resulting nCounter data were analyzed using the Nanostring nSolver data analysis software with normalization to the geometric mean of the level of the <italic>cdc-42</italic>, <italic>pmp-3</italic>, and <italic>Y45F10D</italic>.<italic>4</italic> transcripts in each sample [<xref ref-type="bibr" rid="pgen.1005823.ref123">123</xref>]. The normalized expression data were then exported to MS Excel for further analysis.</p><p>RNA for Nanostring studies was isolated from wild-type (N2) or <italic>rpn-10(ok1865)</italic> animals grown from egg hatching on control, <italic>skn-1</italic>, or 1:10 diluted <italic>elt-2</italic> RNAi for 48 hours. This time point was selected because no visible differences in worm morphology were observed in any of the treatment groups, so any changes in gene expression likely occurred before the animals became ill due to RNAi treatment. After washing the animals from the plates in S-basal, the worm pellet was then suspended in QIAzol lysis reagent and frozen at -80&#176;C. Total RNA was isolated using the Qiagen miRNeasy kit. The yield and quality of each RNA sample was evaluated using a Nanodrop spectrophotometer and also by running an aliquot on an Agilent Bioanalyzer. For each genotype-RNAi treatment pair, six biological replicates were performed.</p><p>Three independent populations of N2 control and <italic>rpn-10(ok1865)</italic> mutant worms were synchronized via the use of hypochlorite treatment and grown on <italic>E</italic>. <italic>coli</italic> OP50-1 spotted NGA plates at 20&#176;C for 3 days. The worms were then washed from the plates and washed twice with milliQ-purified water. The worm pellet was then suspended in QIAzol lysis reagent and frozen at -80&#176;C. Total RNA was isolated using the Qiagen miRNeasy mini kit, and the RNA yield was measured by spectrophotometry. Total RNA was sent to Expression Analysis (Durham, NC) for analysis including Agilent Bioanalyzer electrophoresis to ensure RNA quality followed by library preparation using the Illumina TruSeq RNA sample prep kit. The resulting library was subjected to high-throughput 50 nucleotide paired end sequencing using an Illumina sequencer at a depth of 17 million reads per sample.</p><p>The resulting sequence data were analyzed as previously described [<xref ref-type="bibr" rid="pgen.1005823.ref119">119</xref>]. Briefly, the sequence reads were clipped using internally developed software by Expression Analysis and matched to the <italic>C</italic>. <italic>elegans</italic> genome using RSEM [<xref ref-type="bibr" rid="pgen.1005823.ref124">124</xref>]. The resulting transcript counts were then normalized using the upper quartile normalization approach [<xref ref-type="bibr" rid="pgen.1005823.ref125">125</xref>]. Differentially expressed genes were then identified through the use of serial <italic>t-</italic>testing coupled with Benjamini-Hochberg correction and genes with an adjusted p-value score less than 0.05 were considered to be differentially expressed. This led to the identification of 171 genes as being differentially expressed (111 up-regulated and 60 down-regulated) between <italic>rpn-10(ok1865)</italic> and wild-type N2 (<xref ref-type="supplementary-material" rid="pgen.1005823.s002">S2</xref> and <xref ref-type="supplementary-material" rid="pgen.1005823.s003">S3</xref> Tables). Over-represented gene classes were identified in the up-regulated and down-regulated genes through the use of DAVID [<xref ref-type="bibr" rid="pgen.1005823.ref126">126</xref>].</p><p>Lysosomes were stained via two complementary approaches. The first approach utilized the Lysotracker Red stain (Life Technologies #L7528) and Lysosensor Green stain (Life Technologies #L7535), which concentrate in the low pH environment of the lysosome, while the second utilized the Magic Red cathepsin B and cathepsin L substrates (ImmunoChemistry Technologies #938 and #942) [<xref ref-type="bibr" rid="pgen.1005823.ref089">89</xref>]. These cathepsin B and L substrates are cell-permeable cresyl violet-conjugated peptides containing either the Arg-Arg or Phe-Arg sequence cleaved by the respective cathepsin in the lysosome, and this cleavage event relieves an intramolecular quenching of the cresyl violet fluorophore and produces red fluorescence.</p><p>Lysotracker Red and Lysosensor Green staining were both performed by spotting NGA plates with an aliquot of dye from a 1mM working stock diluted in S-basal to produce a final concentration of 2 &#956;M [<xref ref-type="bibr" rid="pgen.1005823.ref127">127</xref>]. The spotted plates were allowed to dry for one hour at room temperature before L4 larval animals were added. The worms were stained overnight at 20&#176;C, and then transferred to unspotted NGA plates for one hour to clear residual dye from the intestinal lumen. The animals were then mounted on slides and imaged using a Nikon Eclipse Ti inverted microscope using a Y-2E/C filter cube. Images were captured at 20X magnification using a CoolSNAP HQ2 (Photometrics) CCD camera and Nikon Elements software. Fluorescence intensity was measured using ImageJ [<xref ref-type="bibr" rid="pgen.1005823.ref120">120</xref>]. We did not stain the control and <italic>elt-2</italic> RNAi treated animals with Lysotracker Red because preliminary experiments demonstrated greater penetration of Lysotracker into the <italic>elt-2</italic> RNAi treated animals. Particularly, we observed the staining of tissues, like the hypodermis, that are not seen in animals stained on control RNAi or NGA, which suggested that the absorption or distribution of the dye is not similar between the RNAi treatments thus precluding reliable comparisons.</p><p>Magic Red staining was performed by spotting NGA, control RNAi, or <italic>elt-2</italic> RNAi plates with an aliquot of dye from a 260X stock, prepared by dissolving the powdered dye in DMSO following the manufacturer&#8217;s instructions, to give a final 1X concentration. To facilitate spreading of the dye on the plate, the aliquot was mixed with water to produce a final volume of 20 &#956;L prior to pipetting onto the plate. To conserve dye, we performed these experiments in 12 well plates containing 3 mL of agar per well. The plates were allowed to dry for one hour at room temperature before L4 larval animals were added. The animals were incubated at 20&#176;C overnight, and then cleared of residual dye and imaged as described above.</p>