<p>Barley (<italic>Hordeum vulgare</italic> L.), bread wheat (<italic>Triticum aestivum</italic> L.) and <italic>Brachypodium distachyon</italic> plants were grown in a growth chamber under a 16 hrs/8 hrs, 20&#176;C/18&#176;C day/night cycle, respectively, with 70% relative humidity; Rice (<italic>O. sativa</italic>) was maintained in a growth chamber under a 16 hrs/8 hrs, 28&#176;C/26&#176;C day/night cycle, respectively, with 90% relative humidity; Maize (<italic>Z. mays</italic>) was grown in a chamber at 22&#176;C under a 16 hrs/8 hrs day/night cycle under 70% relative humidity; <italic>N. benthamiana</italic> and <italic>A. thaliana</italic> were grown in greenhouse at 24&#177;1&#176;C with a 16 hr light period.</p><p>
<italic>Blumeria graminis</italic> f. sp. <italic>hordei</italic> (<italic>Bgh</italic>) isolates K1 (<italic>AVR<sub>a1</sub></italic>, <italic>vir<sub>a6</sub></italic>, <italic>vir<sub>a10</sub></italic>, <italic>vir<sub>a12</sub></italic>) and A6 (<italic>AVR<sub>a6</sub></italic>, <italic>AVR<sub>a10</sub></italic>, <italic>AVR<sub>a12</sub></italic>, <italic>vir<sub>a1</sub></italic>) were maintained on barley cultivar &#8216;I10&#8217; (containing <italic>Mla12</italic>) and &#8216;P01&#8217; (containing <italic>Mla1</italic>), respectively, and kept at 70% relative humidity, and a 16 hrs/8 hrs, 20&#176;C/18&#176;C day/night cycle.</p><p>Seven-day-old barley leaves of barley isogenic line P01 were inoculated with <italic>Bgh</italic> A6 and K1 for 22 hrs and total RNAs were isolated using TRIzol solution (Invitrogen 15596-026) according to the manufacturer's instructions. Small RNAs of 18&#8211;30 nt were excised and isolated from 5 to 10 &#181;g total RNAs electrophoresed on 15% polyacrylamide denaturing gel, and then were ligated with 5&#8242; and 3&#8242; adapters. The ligated small RNAs were used as templates for cDNA synthesis followed by PCR amplification. The obtained libraries were sequenced using the Solexa sequencing platform (BGI, Beijing).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The mature miRNAs sequences were used in a BLASTn search against the barley genome sequencing database (<ext-link ext-link-type="uri" ns0:href="http://webblast.ipk-gatersleben.de/barley/">http://webblast.ipk-gatersleben.de/barley/</ext-link>) and bread wheat expressed sequence tag (EST) database (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</ext-link>). The secondary structure of flanking sequence around perfectly matched site was predicted using the RNA-folding program Mfold <xref ref-type="bibr" rid="pgen.1004755-Zuker1">[103]</xref>.</p><p>Primers for cloning of <italic>MIR9863</italic> precursors were designed according to the flanking sequences of the hairpin structures. Primers (J19/J20) for <italic>hvu-MIR9863a</italic> cloning were designed according to the sequence of <italic>tae-MIR9863a</italic>. <italic>hvu-MIR9863b</italic> was PCR-amplified using barley &#8216;Morex&#8217; genomic DNA as template (primers, J15/J16); <italic>tae-MIR9863</italic> cluster containing <italic>tae-MIR9863a</italic> and <italic>tae-MIR9863b</italic> was amplified using bread wheat &#8216;Chancellor&#8217; genomic DNA as template and primer J05/J06; <italic>tae-MIR9863c</italic> was amplified using bread wheat &#8216;Chancellor&#8217; genomic DNA as template and primer J07/J08. All primer sequences are shown in supplemental <bold><xref ref-type="supplementary-material" rid="pgen.1004755.s013">S2 Table</xref></bold>.</p><p>RLM-RACE kit (TaKaRa, Code D315) was used for 5&#8242; RACE according to the manufacturer's instruction. Total RNAs were isolated from 7-day-old leaves of barley line P01 infected with <italic>Bgh</italic> K1 spores, and mRNAs were enriched from 100 &#181;g of total RNAs using the PolyATtract mRNA isolation kit (Promega, Z5210). The RNA Oligo adaptor was ligated to mRNAs without calf intestinal phosphatase treatment. For the first round PCR, the 5&#8242; RACE Outer Primer J01 together with <italic>Mla1</italic> gene specific outer primer J03 were used. Nested PCR amplification was performed using the 5&#8242; RACE Inner Primer J02 and <italic>Mla1</italic> specific inner primer J04.</p><p>Plasmids constructed in this study are based on several expression vectors previously described and listed in supplemental <bold><xref ref-type="supplementary-material" rid="pgen.1004755.s014">S3 Table</xref></bold>.</p><p>CTAPi-GW-3HA was used as the transient expression vector for <italic>Mla</italic> genes in <italic>N. benthamiana</italic>, and derived plasmids were constructed using Gateway technology (Invitrogen) following the instructions of the manufacturer.</p><p>For construction of miRNAs overexpression constructs used in <italic>N. benthamiana</italic> expression assays, indicated <italic>MIRNAs</italic> precursor sequences were PCR amplified, double digested by KpnI/HindIII, and ligated into 35S-pKANNIBAL vector. Precursor <italic>aMIR9863a</italic>, <italic>aMIR9863b.1</italic> and <italic>aMIR9863b.2</italic> were constructed by replacing the miR173/miR173* duplex in <italic>ath-MIR173</italic> with amiR9863a/amiR9863a*, amiR9863b.1/amiR9863b.1* or amiR9863b.2/amiR9863b.2* (<bold><xref ref-type="supplementary-material" rid="pgen.1004755.s005">S5 Figure</xref></bold>) using overlapping primers J35/J36/J37, J31/J32/J33 and J36/J37/J38, respectively.</p><p>pTRV2-derivatives used in TRV-VIGS analysis were constructed based on pTRV2-LIC by ligation-independent cloning (LIC) as described previously <xref ref-type="bibr" rid="pgen.1004755-Dong1">[104]</xref>. J44/J45, J46/J47, J48/J49 and J50/J51 were used for amplifying anti-sense fragment from cDNA of <italic>N. benthamiana</italic> for silencing <italic>NbAGO1-1</italic>, <italic>NbAGO1-2</italic>, <italic>NbAGO4-1</italic> and <italic>NbAGO4-2</italic>, respectively. pTRV2-LIC vector was digested with PstI and ligated with PCR fragments.</p><p>For BSMV mediated silencing of miR9863 members in barley, STTM-miR9863 and STTM-EV were first designed and amplified by using overlapping primer groups J52/J53/J54 and J55/J56, according to <xref ref-type="fig" rid="pgen-1004755-g005">
<bold>Fig. 5A</bold>
</xref> and Yan and associates <xref ref-type="bibr" rid="pgen.1004755-Yan1">[80]</xref>. Then PCR products were cloned into pCaBS-&#947;bLIC vector through LIC strategy as described above.</p><p>For single-cell transient overexpression of miR9863 members, the <italic>tae-MIR9863a</italic>, <italic>tae-MIR9863c</italic> or <italic>hvu-MIR9863b</italic> precursors with <italic>attB</italic> sites were cloned into pUbi-GATE vector by Gateway technology described above. Similarly, pUbi-GW-mYFP vector was used for expression of <italic>Mla1</italic> or <italic>Mla10</italic> cDNA.</p><p>Point mutations for different <italic>Mla</italic> cDNAs or <italic>MIRNA</italic> precursors were introduced using one-step site-directed mutagenesis as described previously with minor modifications <xref ref-type="bibr" rid="pgen.1004755-Zheng1">[105]</xref>. The entry vectors harboring wild-type <italic>Mla</italic> allele cDNA or 35S-pKANNIBAL vectors containing indicated wild-type <italic>MIRNA</italic> precursor sequences were used as templates for PCR reaction to introduce mutations.</p><p>Total RNAs were extracted from plant materials using TRIzol solution, and treated with RNase-free DNase I (TaKaRa). About 2 &#181;g of total RNA and M-MLV Reverse Transcriptase (Promega) were further used for reverse transcription. For coding genes reverse transcription, first-strand cDNA was synthesized using Oligo (dT)<sub>18</sub>. For small RNA reverse transcription, specifically designed stem-loop reverse transcription primers were used, and followed the procedures described by Chen and colleagues <xref ref-type="bibr" rid="pgen.1004755-Chen4">[106]</xref>. Primer J71, J72, J75 and J76 were used for miR9863a/b.2, miR9863c/b.1, phasiRNAI and phasiRNAII, respectively; primer J81 was used for <italic>U6</italic> (<italic>U6</italic> stands for <italic>U6</italic> spliceosomal RNA), and obtained cDNA was diluted 10 times and used for further analysis. Real-time qPCR was performed using StepOne real-time system (Applied Biosystems) and GoTaq qPCR Master Mix (Promega, A6001); for the determination of three <italic>Mla1</italic> amplicons (<xref ref-type="fig" rid="pgen-1004755-g007">
<bold>Fig. 7</bold>
</xref>), primer pairs J65/J66, J67/J68 and J69/J70 were used; Primer J73, J74, J77 and J78 were respectively used with J79 to quantify the level of miR9863a/b.2, miR9863c/b.1, phasiRNAI and phasiRNAII; Primer pair J80/J81 and J82/J83 were used for the detection of <italic>U6</italic> and <italic>Actin</italic>. Semi-quantitative RT-PCR was performed as described previously <xref ref-type="bibr" rid="pgen.1004755-Bai1">[56]</xref>. Primer pair J61/J62, J63/J64 were designed to determine the silencing efficiency of <italic>NbAGO1</italic> and <italic>NbAGO4</italic>, respectively.</p><p>RNA gel blot analysis was performed as described previously with minor modifications <xref ref-type="bibr" rid="pgen.1004755-Liu1">[107]</xref>. Total RNAs of 10 to 15 &#181;g were separated on a 15% polyacrylamide denaturing gel by electrophoresis and cross-linking was performed as described previously <xref ref-type="bibr" rid="pgen.1004755-Pall1">[108]</xref>. The complementary sequences corresponding to miRNAs were used as probes after labeled with [&#947;-<sup>32</sup>P]ATP using T4 polynucleotide kinase (New England Biolabs). Probe J84, J85, J86 and J88 were used to detect the mature miR9863a, miR9863b.1, miR9863b.2 and miR156a signal, respectively; probe J87 was used for <italic>Mla1</italic> derived phasiRNAs in <bold><xref ref-type="supplementary-material" rid="pgen.1004755.s008">S8 Figure</xref></bold> according to Chen and colleagues <xref ref-type="bibr" rid="pgen.1004755-Chen2">[44]</xref>.</p><p>
<italic>Agrobacterium</italic>-mediated transient expression in <italic>N. benthamiana</italic> was performed as described previously <xref ref-type="bibr" rid="pgen.1004755-Bai1">[56]</xref>, <xref ref-type="bibr" rid="pgen.1004755-VanOoijen1">[109]</xref>. For single expression, <italic>Agrobacterium</italic> suspensions expressing gene of interest were infiltrated into 4- to 5-week old <italic>N. benthamiana</italic> leaves. For co-expressions, <italic>Agrobacterium</italic> suspensions expressing <italic>MIRNA</italic> precursor were first infiltrated, and 24 hrs later, <italic>Agrobacterium</italic> suspensions expressing gene candidate was infiltrated into the same position. Samples were collected from the infiltrated sites at 36 hrs post infiltration. Total soluble proteins were extracted using 180 &#181;l of 2&#215; Laemmli buffer <xref ref-type="bibr" rid="pgen.1004755-Laemmli1">[110]</xref> from 60 mg leaf samples, and detected by immunoblotting as described previously <xref ref-type="bibr" rid="pgen.1004755-Bai1">[56]</xref>. Rat anti-HA antibody (1&#8758;5000; Roche, 11867423001) and anti-rat IgG conjugated with horseradish peroxidase (HRP) (1&#8758;10000; Sigma, A5795) were used for HA-tagged proteins detection. The levels of &#946;-actin were determined using an anti-&#946;-actin antibody (1&#8758;1000; CWBIO, CW0264) coupled with anti-mouse IgG conjugated with HRP (1&#8758;75000; Sigma, A9044).</p><p>Single-cell transient gene expression assay using biolistic delivery of plasmid DNA into barley epidermal cells was performed as previously described <xref ref-type="bibr" rid="pgen.1004755-Shen3">[77]</xref>. For <italic>MIRNAs</italic> overexpression, the &#946;-glucuronidase (<italic>GUS</italic>) reporter gene was mixed with respective plasmids (molar ratio 1&#8758;1) before coating of gold particles. Barley leaf epidermal cells were transformed with the biolistic particle delivery system (Bio-Rad, Model PDS-1000/He), and incubated 4 hrs before inoculation of <italic>Bgh</italic> spores. To identify transformed cells, bombarded leaves were fixed in solution at 48 hrs after <italic>Bgh</italic> infection and further stained for GUS activity. For <italic>MIRNAs</italic> and <italic>Mla</italic> co-expression, the GUS reporter and <italic>MIR9863b</italic> plasmids were co-coated with <italic>Mla1-mYFP</italic> or <italic>Mla10-mYFP</italic> plasmids (molar ratio 1&#8758;1&#8758;1).</p><p>Trypan blue staining was described previously <xref ref-type="bibr" rid="pgen.1004755-Bai1">[56]</xref>. Briefly, <italic>N. benthamiana</italic> leaves were boiled for 5 min in staining solution, and were then de-stained in 2.5 g ml<sup>-1</sup> chloral hydrate in distilled water for at least 3 days.</p><p>TRV-based virus induced gene silencing assay was performed as described <xref ref-type="bibr" rid="pgen.1004755-Bai1">[56]</xref>, <xref ref-type="bibr" rid="pgen.1004755-Liu2">[111]</xref>. Briefly, pTRV1 and pTRV2-derived constructs (pTRV2: EV, pTRV2: <italic>NbPDSas</italic>, pTRV2: <italic>NbAGO1-1as</italic>, pTRV2: <italic>NbAGO1-2as</italic>, pTRV2: <italic>NbAGO4-1as</italic>, pTRV2: <italic>NbAGO4-2as</italic>) were transformed into <italic>A. tumefaciens</italic> strain GV3101. Cell suspensions containing pTRV1 and pTRV2-derived constructs were collected and mixed at 1&#8758;1 ratio and infiltrated into the third to fifth leaf of three-week-old <italic>N. benthamiana</italic> plants. Three weeks after infiltration, the upper newly expanded leaves were selected for further analysis.</p><p>For BSMV-mediated STTM-VIGS assay of miR9863 members, constructs of pCaBS-&#945;, pCaBS-&#946;, and pCaBS-&#947;bLIC derivatives (pCaBS-&#947;STTM-EV and pCaBS-&#947;STTM-miR9863) were transformed into the <italic>A. tumefaciens</italic> strain EHA105, respectively. The <italic>Agrobacterium</italic> suspensions of OD<sub>600</sub>&#8202;=&#8202;0.8 were mixed at 1&#8758;1&#8758;1 ratio and infiltrated in <italic>N. benthamiana</italic> leaves. The <italic>N. benthamiana</italic> sap was extracted from leaves with BSMV symptom at about 12 days post infiltration, and inoculated to the first two emerging leaves of barley leaves. About 15 days later, the newly grown upper barley leaves with virus symptom were collected for further analysis.</p><p>Sequence data for genes used in this article can be found under GenBank accession numbers AY009939 (<italic>HvuMla1</italic>), GU245938 (<italic>HvuMla2</italic>), GU245939 (<italic>HvuMla3</italic>), AJ302292 (<italic>HvuMla6</italic>), AY266444 (<italic>HvuMla7</italic>), GU245940 (<italic>HvuMla8</italic>), GU245941 (<italic>HvuMla9</italic>), AY266445 (<italic>HvuMla10</italic>), AY196347 (<italic>HvuMla12</italic>), AF523683 (<italic>HvuMla13-1</italic>), GU245942 (<italic>HvuMla16-1</italic>), GU245943 (<italic>HvuMla18-1</italic>), GU245944 (<italic>HvuMla18-2</italic>), GU245945 (<italic>HvuMla19-1</italic>), GU245946 (<italic>HvuMla22</italic>), GU245947 (<italic>HvuMla23</italic>), GU245948 (<italic>HvuMla25-1</italic>), GU245949 (<italic>HvuMla27-1</italic>), GU245950 (<italic>HvuMla27-2</italic>), GU245951 (<italic>HvuMla28</italic>), GU245952 (<italic>HvuMla30</italic>), GU245953 (<italic>HvuMla31-1</italic>), GU245954 (<italic>HvuMla32</italic>), GU245955 (<italic>HvuMla34</italic>), GU245956 (<italic>HvuMla35</italic>), GU245957 (<italic>HvuMla36-1</italic>), GU245958 (<italic>HvuMla37-1</italic>), GU245959 (<italic>HvuMla38-1</italic>), GU245960 (<italic>HvuMla39-1</italic>), KJ619975 (<italic>hvu-MIR9863a</italic>), AK364228 (<italic>hvu-MIR9863b</italic>), CK193889 (<italic>tae-MIR9863a</italic>, <italic>tae-MIR9863b</italic>, <italic>tae-MIR9863a/b</italic> cluster), DR736484 (<italic>tae-MIR9863c</italic>), DQ321488 (<italic>NbAGO1-1</italic>), DQ321489 (<italic>NbAGO1-2</italic>), DQ321490 (<italic>NbAGO4-1</italic>), DQ321491 (<italic>NbAGO4-2</italic>), ACZ65507 (<italic>HchMla1</italic>), ADX06722 (<italic>TmoMla1</italic>), AGP75918 (<italic>TmoSr35</italic>), KF031291 (<italic>AetSr33</italic>), ACG63536 (<italic>TduLr10</italic>), AAG42168 (<italic>TaeYr10</italic>), AAY21626 (<italic>TaePm3a</italic>), AAQ96158 (<italic>TaePm3b</italic>), ABB78077 (<italic>TaePm3c</italic>) and AAY21627 (<italic>TaePm3d</italic>).</p>