<p xmlns:ns0="http://www.w3.org/1999/xlink">Deletion strains used to screen for fusion mutants and strains constructed for this study are listed in <xref ref-type="supplementary-material" rid="pgen.1004783.s015">Table S2</xref>. Strains were grown on Vogel's minimal medium (VMM) <xref ref-type="bibr" rid="pgen.1004783-Vogel1">[67]</xref> (with supplements as required) and were crossed on Westergaard's medium <xref ref-type="bibr" rid="pgen.1004783-Westergaard1">[68]</xref>. Transformations and other <italic>N. crassa</italic> molecular techniques were performed as described <xref ref-type="bibr" rid="pgen.1004783-Colot1">[69]</xref> or using protocols available at the Neurospora home page at the FGSC (<ext-link ext-link-type="uri" ns0:href="http://www.fgsc.net/Neurospora/NeurosporaProtocolGuide.htm">http://www.fgsc.net/Neurospora/NeurosporaProtocolGuide.htm</ext-link>).</p><p>To construct the <italic>ham-5</italic> alleles, PCR was performed with the restriction enzyme linkers included in the primer region. We amplified <italic>ham-5</italic> alleles using primers 1-5: <italic>ham-5</italic>FXbaI tttttctagaATGTCGGTCCCCGGACACA; <italic>ham-5</italic>RpacI aaaattaattaaGATCATCTCACTATGATGCAAC; <italic>ham-5</italic> WD40onlyR tttttaattaaGTTAGCAGGATGTTGAACGTTG; <italic>ham-5</italic>RWD40 tttatgcatatttaaaTCATGGTGGCAGCATACAATC; <italic>ham-5</italic>FWD40 tttatttaaatCCTGCTAACATGTTACCTCC; and cloned the fragments into pCR-Blunt vector (Invitrogen). For constructing the point mutations at the predicted phosphorylation site we used fusion PCR strategy with primers <named-content content-type="gene">CGTCATCGGGGGCGCGCGGCACCTCATGGCG</named-content> and <named-content content-type="gene">GGTGCCGCGCGCCCCCGATGACGCGAAAGTTGT</named-content> (S &#8594; A) and <named-content content-type="gene">CGTCATCGGGCTCGCGCGGCACCTCATGGCG</named-content> and <named-content content-type="gene">GGTGCCGCGCGAGCCCGATGACGCGAAAGTTGT</named-content> (S &#8594; E) together with the primers <named-content content-type="gene">TTTGATGCATCACAATGCTGACC</named-content> and <named-content content-type="gene">TTAAGGGCCGAATTCTTCGC</named-content>. The mutated constructs were ligated into the <italic>Nsi</italic>I and <italic>Eco</italic>RI sites of the <italic>HAM-5</italic> gene. For constructing the mutation at the predicted docking site, we used primers ggtcgctgacaaactcgaat and tttgccggctgcCTCGGAACTGCGCGCGCGG that has the restriction site <italic>Nae</italic>I in the linker and <named-content content-type="gene">TTTCGGCCAGCATCATGAGA</named-content> and tttagcgctCCTCCAGCACTCGACCTTCGC that has the restriction site <italic>Afe</italic>I in the linker. The two respective products were digested with <italic>Nsi</italic>I and <italic>Nae</italic>I and <italic>Afe</italic>I and <italic>Tth</italic>111I, respectively and ligated using a three-point ligation in the vector with <italic>HAM-5</italic> cut open with <italic>Nsi</italic>I and <italic>Tth</italic>111I.</p><p>We sequenced and digested the constructs from the pCR-Blunt vector with the appropriate restriction enzymes. The different fragments were ligated into plasmid pMF272 (AY598428) <xref ref-type="bibr" rid="pgen.1004783-Margolin1">[70]</xref>, <xref ref-type="bibr" rid="pgen.1004783-Freitag1">[71]</xref>. For the MAK-2-mCherry, SO-mCherry, MEK-2-mCherry and NRC-1-mCherry strains, plasmid TSL84C was used. To generate plasmid pTSL84C, sGFP, from plasmid pMF272, was removed by digestion with <italic>Pac</italic>I and <italic>Eco</italic>RI restriction enzymes and replaced by a version of mCherry that is codon-optimized for <italic>N. crassa</italic> and includes a C-terminal 6x-His-tag (mCherryNc-6xHis). Plasmid pMFP26 <xref ref-type="bibr" rid="pgen.1004783-CastroLongoria1">[72]</xref> contains untagged codon-optimized mCherryNc and was used as a template for PCR using forward primer OTS177 (<named-content content-type="gene">AAATTAATTAACGTGAGCAAGGGCGAGGAGGATAAC</named-content>) and reverse primer OTS202 (<named-content content-type="gene">AAAGAATTCCTAGTGGTGGTGGTGGTGGTGGCTGCCCTTGTACAGCTCGTCCATGCCGCCG</named-content>), which contained information for the 6xHis tag and a stop codon. Plasmid derivatives with the <italic>tef-1</italic> promoter instead of <italic>ccg-1</italic> promoter were obtained by swapping the <italic>ccg-1</italic> promoter for the <italic>tef-1</italic> promoter using the restriction enzymes <italic>Not</italic>I and <italic>Xba</italic>I. A tandem construct of <italic>tef-1-ham-5-gfp</italic> and <italic>tef-1-mak-2-mCherry</italic> was created by digesting the <italic>tef-1-mak-2-mCherry</italic> pMF272 plasmid with restriction enzymes <italic>Psp</italic>OMI and <italic>Bst</italic>BI and the <italic>tef-1-ham-5-gfp</italic> construct with <italic>Not</italic>I and <italic>Bst</italic>BI. The latter fragment was ligated into the <italic>tef-1-mak-2-mCherry</italic> pMF272 plasmid to create <italic>tef-1-mak-2-mCherry</italic> and <italic>tef-1-ham-5-gfp</italic>. All constructs were transformed into the WT <italic>his-3</italic> strain with selection for His+ prototrophy. Homokaryotic strain was obtained via microconidial purification <xref ref-type="bibr" rid="pgen.1004783-Pandit1">[73]</xref>. A strain bearing cytoplasmic GFP was obtained by transformation of the empty pMF272 plasmid into the WT <italic>his-3</italic> strain. All micrographs with HAM-5-GFP are with strains bearing the <italic>tef-1-ham-5-gfp</italic> constructs unless stated otherwise.</p><p>Deletion strains were obtained from the FGSC <xref ref-type="bibr" rid="pgen.1004783-McCluskey1">[74]</xref> that were generated as part of the <italic>N. crassa</italic> functional genomics project <xref ref-type="bibr" rid="pgen.1004783-Colot1">[69]</xref>, <xref ref-type="bibr" rid="pgen.1004783-Dunlap1">[75]</xref>. For each deletion strain, both the mating type <italic>A</italic> and mating type <italic>a</italic> strains were analyzed, if available.</p><p>To assess the ability of fusion between conidia of a deletion strain as compared to wild type, slant tubes containing the strains were grown for 4&#8211;6 days or until significant conidiation occurred. Conidia were harvested by vortexing the slant tube with 2 ml ddH<sub>2</sub>O and subsequently filtered by pouring over cheesecloth to remove hyphal fragments. Conidia were diluted to a concentration of 3.3&#215;10<sup>7</sup> conidia/ml. For each sample, 300 &#181;l of spore suspension was spread on a 9 cm solid VMM plate. The plates were dried in a fume hood for 20-30 minutes and incubated for 3&#8211;4 hours at 30&#176;C. Squares of 1 cm were excised and observed with a Zeiss Axioskop 2 using a 40&#215; Plan-Neofluor oil immersion objective. The ability of germlings to communicate was determined by evaluating whether germlings displayed homing behavior when germinated conidia were within &#8764;15 um of each other.</p><p>Samples for protein extraction were grown for 4.5 h at 30&#176;C and 200 rpm. Shaking was stopped and samples were grown for 20 minutes longer to encourage cell-cell interaction. The <italic>mak-2<sup>Q100G</sup></italic> strain was treated with 10 &#181;M 1NM-PP1 final concentration in DMSO or with DMSO alone for 10 minutes. Cells were harvested by filtration and frozen before protein extraction. Protein was extracted using the Trizol procedure (according to manufacture's protocols) and was kept in a solution of 6 M guanidine, 50 mM ammonium bicarbonate at pH 7.4. 200 &#181;g for each sample was used for guanidine digestion: 1) pH was adjusted to pH 7.4 with the iTRAQ resuspending buffer (500 mM), 2) 1 hr reduction using 5 mM DTT at 56&#176;C, 3) 1 h alkylation using 10 mM iodoacetamide at RT in the dark, 4) samples were diluted 10 fold with 25 mM NH<sub>4</sub>HCO<sub>3</sub>, pH 7.8, 5) 2 mM CaCl<sub>2</sub> and trypsin was added at a 1&#8758;50 (trypsin-to-protein) ratio and incubated for 3 hr at 37&#176;C with gentle shaking, 6) trypsin was added again in the same ratio and samples were incubated over-night at 37&#176;C with gentle shaking, 7) subsequently, a standard C18 Solid phase extraction (SPE) was performed with 80% ACN and no TFA, 8) samples were dried in a Speed-Vac and a bicinchoninic acid (BCA) assay was performed.</p><p>For 4-plex iTRAQ labeling, 100 &#181;g lyophilized sample per iTRAQ label was used: 1) samples were reconstituted with 30.0 &#181;L of dissolution buffer (500 mM triethylammonium bicarbonate), sonicated and vortexed to resuspend the peptides, 2) sample pH was checked (&#8764;pH 8.5), 3) each vial of iTRAQ reagent (114, 115, 116 and 117) was brought to room temperature and 70 &#181;L of ethanol was added to each iTRAQ reagent vial, 4) samples were vortexed for 1 min and spun down, 5) each labeled reaction mix was added to one separate sample, 6) samples were vortexed, spun down and incubated for 1 hr at RT. Samples were subsequently hydrolyzed by adding 300 uL of 0.05% TFA (3 times the volume), vortexed, spun down and incubated at room temperature for another 30 min, 7) samples were concentrated to 40 &#181;L using a Speed-Vac, 8) samples were pooled into a fresh 2 mL silanized tube and concentrated to &#8764;100 &#181;L using a Speed-Vac before a desalting (SPE C18) step was performed.</p><p>For phosphopeptide enrichment, magnetic Ni-NTA-agarose beads were obtained from Qiagen (Valencia, CA Part N#36111): 1) 50 &#181;L of the 5% suspension metal ion activated NTA was used for 100 ug peptides, 2) beads were first prepared by washing 3&#215; with nano-pure water (800.0 &#181;L of water per 1.0 mL of bead suspension), 3) beads were then treated with 100 mM EDTA, pH 8.0 (800.0 &#181;L of 100 mM EDTA per 1.0 mL of bead suspension) for 30 min with end-over-end rotation, 4) EDTA solution was removed, and beads were washed 3&#215; with nano-pure water (at the same ratio). Subsequently, the beads were treated with 10 mM aqueous metal ion solution (800.0 uL of 100 mM FeCl<sub>3</sub> per 1.0 mL of bead suspension) for 30 min with end-over-end rotation, 5) after removing excess metal ions, beads were washed 3&#215; with water (at the same ratio), and resuspended in 1&#8758;1&#8758;1 acetonitrile/methanol/0.01% acetic acid for aliquoting into microcentrifuge tubes, 6) peptide samples were resuspended in 200.0 &#181;l wash/resuspension buffer (80% acetonitrile, 0.1% TFA), 7) beads were washed with of 80% acetonitrile with 0.1% TFA (200 &#181;L of 80% acetonitrile per 50.0 &#181;L of beads) and precipitated using the magnetic stand; the supernatant was discarded. The resuspended samples (100 &#181;g peptides in 200 &#181;l of 80% acetonitrile, 0.1% TFA) were added to the activated beads and incubated for 30 min with end-over-end rotation, 8) beads were precipitated using the magnetic stand and washed for 1 min with 80% acetonitrile, 0.1% TFA. This step was repeated three more times, 9) phosphopeptides were eluted from the beads using an appropriate amount of elution buffer (50.0 &#181;L of the elution buffer per every 50.0 uL of beads/100.0 ug of peptides) after incubating for 5 min, 10) the samples were then acidified to pH 4.0 by concentrating the samples down to 5&#8722;10 &#181;l in a Speed-Vac and reconstituted in 30 &#181;L with 0.1% TFA.</p><p>All peptide samples were analyzed using an automated home-built constant flow nano LC system (Agilent) coupled to an LTQ Orbitrap Velos mass spectrometer (Thermo Fisher Scientific) operating in data-dependent mode <xref ref-type="bibr" rid="pgen.1004783-Nguyen1">[76]</xref>. Electrospray emitters were custom made using either 360 &#181;m o.d. &#215;20 &#181;m i.d. chemically etched fused silica. The nano LC system for phosphoproteomics analysis has an online 4-cm&#215;360 &#181;m o.d. &#215;150 &#181;m i.d. C18 SPE column (5- &#181;m Jupiter C18, Phenomenex, Torrence, CA) to desalt each phosphopeptide sample (20 &#181;L), which is connected to a home-made 60-cm&#215;360 &#181;m o.d. &#215;50 &#181;m i.d. capillary column (3- &#181;m Jupiter C18, Phenomenex, Torrence, CA). Mobile phase flow rate was 100 nL/min and consisted of 0.1 M acetic acid in water and 0.1 M acetic acid in 70&#8758;30 (v/v) acetonitrile:water. For each sample, three technical replicates of LC-MS analyses were performed as shown in <xref ref-type="supplementary-material" rid="pgen.1004783.s001">Figure S1</xref>. These included (i) an LC gradient of 300 min with the LTQ Orbitrap Velos mass spectrometer acquiring higher-energy collisional dissociation (HCD) scans; (ii) an LC gradient of 300 min with the LTQ Orbitrap Velos mass spectrometer acquiring alternating collision-induced dissociation (CID), ETD (electron transfer dissociation), and higher-energy collisional dissociation (HCD) scans; (iii) an LC gradient of 180 min with the LTQ Orbitrap Velos mass spectrometer acquiring alternating collision-induced dissociation (CID), ETD (electron transfer dissociation), and higher-energy collisional dissociation (HCD) scans.</p><p>For peptide identification, MS/MS spectra were searched against a decoy <italic>Neurospora</italic> protein sequence database using SEQUEST <xref ref-type="bibr" rid="pgen.1004783-Eng1">[77]</xref>. Search parameters included: trypsin enzyme specificity with a maximum of two missed cleavages, +/- 50 ppm precursor mass tolerance, +/- 0.05 Da product mass tolerance, and carbamidomethylation of cysteines and iTRAQ labeling of lysines and peptide N-termini as fixed modifications. Allowed variable modifications were phosphorylation of serine, threonine or tyrosine residues. MSGF spectra probability value <xref ref-type="bibr" rid="pgen.1004783-Kim1">[78]</xref> was also calculated for peptides identified from SEQUEST search. Measured mass accuracy and MSGF spectra probability were used to filter identified peptides to &lt;1% false discovery rate (FDR) at spectrum level.</p><p>iTRAQ reporter ions were extracted using the MASIC software <xref ref-type="bibr" rid="pgen.1004783-Monroe1">[79]</xref> within 10 ppm mass tolerance of each expected iTRAQ reporter ion from each MS/MS spectrum. The sum of the individual iTRAQ reporter ion values from all MS/MS spectra for a given peptide was used for calculating their relative abundance across different conditions. To correct any systematic error due to pipetting, data were normalized by the median of iTRAQ reporter ion of the individual sample. Fold change of each phosphopeptide was calculated by dividing the data points from the two different conditions (i.e. control and 1NM-PP1-treated) and transformed into Log<sub>2</sub> scale. Statistical analyses were performed using Students t-test. Only data with changes exceeding 1.5&#215; greater in control versus 1NM-PP1-treated (p&lt;0.05) were considered differential.</p><p>The strain used to cross <italic>so-gfp</italic> into a &#916;<italic>ham-5</italic> strain was AF-SoT8 and the strain used to cross <italic>mak-2-gfp</italic> into a &#916;<italic>ham-5</italic> strain was AF-M512 (<xref ref-type="supplementary-material" rid="pgen.1004783.s015">Table S2</xref>).</p><p>Oscillation studies performed with HAM-5-GFP and mCherry tagged strains were prepared as described above with modifications from <xref ref-type="bibr" rid="pgen.1004783-Fleissner3">[20]</xref>. Images were taken using a Leica SD6000 microscope with a 100&#215;1.4 NA oil-immersion objective equipped with a Yokogawa CSU-X1 spinning disk head and a 488-nm or 561-nm laser controlled by Metamorph software. Multiple pairs of interacting germlings were analyzed per experiment and representative pairs are shown for each strain. The ImageJ software was used for image analysis.</p><p>Harvested conidia (1&#215;10<sup>6</sup>/ml) were inoculated in 100 ml VMM in flasks and incubated for 2.5 hrs at 30&#176;C with shaking at 200 rpm, then an additional 2.5 hrs at 30&#176;C without shaking. Germlings from 3 flasks were harvested by vacuum filtration over a nitrocellulose membrane and frozen in liquid nitrogen. Protein extraction from ground mycelium was performed using 1 ml lysis buffer described in <xref ref-type="bibr" rid="pgen.1004783-Pandey1">[8]</xref> containing complete protease inhibitors, phosphatase inhibitor and Triton X-100. 20 &#181;l supernatant was used for western blotting and the remaining fraction was used for immunoprecipitation using Protein G Dynabeads (Invitrogen), according to manufacturer's instructions, with the following exceptions: mouse or rabbit anti-GFP antibody (Roche or Life Technologies, respectively) or rabbit anti-mCherry antibody (Bio-vision) was covalently bound to the beads using BS<sup>3</sup> (Sulfo-DSS, Fisher scientific) or DMP (dimethylpimelimidate) according to manufacturer's instructions.</p><p>Supernatant samples were incubated with the beads overnight at 4&#176;C. Beads were washed with standard PBS for three times before protein was removed from the beads by heating at 70&#176;C for 10 min in 1&#215; loading buffer, and samples were run on a 4&#8211;12% Nu-Page Bis-Tris GelGel (NOVEX, Life Technologies). Protein gels were subjected to Western blot analysis using standard methods. Samples for the MAK-1 and MAK-2 phosphorylation western blots were treated similarly, except, after protein extraction with 1 ml lysis buffer, 25 &#181;l of protein sample was directly loaded on a 7% NuPage Bis-Tris GelGel (NOVEX, Life Technologies) protein gel. Gels were subjected to Western blot analysis using standard methods and detection of phosphorylated MAK-1 and MAK-2 was carried out using anti-phospho p44/42 MAP kinase antibodies (1&#8758;3000 dilution) (PhosphoPlus antibody kit; Cell Signaling Technology) as described <xref ref-type="bibr" rid="pgen.1004783-Pandey1">[8]</xref>. Detection of phosphorylated HAM-5-GFP was performed using anti-phosphothreonine-proline/phosphoserine-proline antibodies (Abcam).</p>