<p xmlns:ns0="http://www.w3.org/1999/xlink">Since the large majority of L1s are truncated at the 5&#8242; end <xref ref-type="bibr" rid="pgen.1004395-Fanning1">[86]</xref>, the copy number of 3&#8242; ends better represents the history of retrotransposition events. Therefore, we used 575 bp in the 3&#8242; end of L1 ORF2 (as reconstructed below) to get a comprehensive view of L1 retrotransposition. Using the megabat L1 lineage 1 <xref ref-type="bibr" rid="pgen.1004395-Cantrell2">[50]</xref> consensus as the query sequence, we ran CENSOR 4.2 <xref ref-type="bibr" rid="pgen.1004395-Jurka3">[87]</xref> against the &#8764;2&#215; genome trace files of <italic>P. vampyrus</italic> (Baylor College of Medicine, ftp.ncbi.nlm.nih.gov/pub/TraceDB/pteropus_vampyrus/) to find detectable sequences with &gt;60% identity and &gt;90% coverage of the query. Using 2000 random sequences from the CENSOR run, subfamilies were identified based on shared sequence variants (co-segregating mutations) with COSEG 0.2.1 (<ext-link ext-link-type="uri" ns0:href="http://www.repeatmasker.org/COSEGDownload.html">http://www.repeatmasker.org/COSEGDownload.html</ext-link>) <xref ref-type="bibr" rid="pgen.1004395-Smit2">[55]</xref> following the default parameters. Nine subfamilies were generated and their consensuses used as query sequences for a second round of CENSOR against the <italic>P. vampyrus</italic> genome. All identified L1 sequences from the second CENSOR run were used for a second round of COSEG, which required the additional parameter of at least 250 sequences to form a subfamily. Consensuses of the 64 subfamilies thus generated were used as query sequences to run CENSOR for a third time. Each hit's percent identity to the corresponding query was used to assign it to a L1 subfamily, and the copy numbers in each subfamily were counted. Seven subfamilies containing less than 250 sequences were removed. Consensuses from each of the remaining 57 subfamilies were used as query sequences to run CENSOR for a fourth time and all detected L1s were assigned to their subfamilies by the percent identity of each hit to its query. The 57 subfamily consensuses were aligned with ancestral mammalian L1s from RepBase <xref ref-type="bibr" rid="pgen.1004395-Jurka1">[56]</xref>, <xref ref-type="bibr" rid="pgen.1004395-Jurka2">[57]</xref>, reconstructed by Smit <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1004395-Smit3">[58]</xref> and Wade <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1004395-Wade1">[59]</xref>, with the Lasergene software suite (DNASTAR, Madison, WI), and a distance matrix was calculated. Based on the alignment, a maximum likelihood tree was constructed using PhyML <xref ref-type="bibr" rid="pgen.1004395-Guindon1">[88]</xref> with the GTR+I+G model and 100 bootstrap replicates (<xref ref-type="supplementary-material" rid="pgen.1004395.s001">Figure S1</xref>). L1s were then assigned to families based on a &lt;3.5% within-family pairwise distance from their subfamily consensuses. Sequence specificity of L1 families was determined by BLAST <xref ref-type="bibr" rid="pgen.1004395-Altschul1">[89]</xref> against the NCBI whole genome sequencing databases. The consensus sequences of subfamilies 1, 5, 7, 3, 40, 36, 34, 0 and 29 were used as the BLAST queries representing families 1A, 1B, 1C, 2A, 2B, 2C, 2D, 3 and 4, respectively. A subfamily and its corresponding family were considered bat-specific only if &lt;5 of the top 100 BLAST hits were not from bats.</p><p>Histograms of L1 age distribution were generated by the R <xref ref-type="bibr" rid="pgen.1004395-R1">[90]</xref> histogram function using a window size of 0.5% (<xref ref-type="fig" rid="pgen-1004395-g001">Figures 1</xref> and <xref ref-type="fig" rid="pgen-1004395-g002">2</xref>). Percent identities corresponding to retrotransposition peaks of individual families (<xref ref-type="fig" rid="pgen-1004395-g002">Figure 2</xref>) were determined by R using the kernel smoothing function with 0.2% bandwidth.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">A full-length consensus sequence of the most recently active L1 from megabat lineage 1 was reconstructed by a series of progressive steps. The seed for the reconstruction was a conserved 575 bp region in the 3&#8242; half of ORF2 (<xref ref-type="fig" rid="pgen-1004395-g003">Figure 3A</xref>). This region was previously amplified by degenerate PCR and a consensus sequence was determined <xref ref-type="bibr" rid="pgen.1004395-Cantrell3">[91]</xref>. Walks were performed in the 5&#8242; and 3&#8242; directions away from the cloned region and continued in both directions until full-length L1s were reconstructed. To aid with the reconstruction, a software pipeline was developed consisting of Perl (<ext-link ext-link-type="uri" ns0:href="http://www.perl.org/">http://www.perl.org/</ext-link>), Ruby (<ext-link ext-link-type="uri" ns0:href="https://www.ruby-lang.org/en/">https://www.ruby-lang.org/en/</ext-link>) and Bash (<ext-link ext-link-type="uri" ns0:href="http://www.gnu.org/software/bash/">http://www.gnu.org/software/bash/</ext-link>) scripts. The pipeline queried, filtered and extracted data from the genome of <italic>P. vampyrus</italic>. An individual step resulted in the addition of 100&#8211;500 bp of sequence to the consensus, depending on the quality of the alignment at the ends, which was then used in the next step of the walk and in the final L1 reconstruction. Candidate sequences were identified in the database using BLAST with default parameters and an e-value of 1&#215;10<sup>&#8722;50</sup>, parsed through the BioPerl SearchIO module (<ext-link ext-link-type="uri" ns0:href="http://www.bioperl.org">http://www.bioperl.org</ext-link>) and screened based on their similarity to the input sequence. Only hits with at least 92% identity were retained to assure that the reconstruction did not include older lineages, and then a Ruby script extracted those sequences with overhangs of at least 100 bp. Alignments for each end were created and hand-edited to yield consensuses of clean read which were aligned into a master alignment. A 300&#8211;500 bp region from each end was selected to act as the seeds for the next step in the walk. The process was repeated until the entire element was reconstructed. Upon completion of the full-length L1, a 500 bp seed was chosen arbitrarily from the final consensus and the pipeline was run again to verify the reconstruction. Methylated CpG sites evolve rapidly and must be corrected in the final consensus. CpG sites were identified by their high variation and the presence of dinucleotide sequence CG, CA, TG or TA; these were examined, manually edited and designated as CG in the final consensus. This pipeline also reconstructed the most recently active L1 lineage of four additional species listed in <xref ref-type="supplementary-material" rid="pgen.1004395.s004">Table S1</xref>, but required higher percent identities for the walks to reduce the noise introduced by older lineages.</p><p>To compare the reconstruction of the extinct L1 to other L1s, sequences from a range of mammalian species were either reconstructed as described above, or selected from the RepBase report of February 2013 <xref ref-type="bibr" rid="pgen.1004395-Jurka1">[56]</xref>. L1 consensuses of all species available in RepBase were aligned except those of dolphin and American opossum which had problematic regions of non-homology. When multiple L1 consensus sequences for the same species were present in RepBase, the one with highest average percent identity to its genomic sequence was chosen to represent the most recent master L1 in the genome. Some of the RepBase L1 sequences were out of frame at regions containing adenosine runs or contained in-frame stop codons, both resulting in significantly shorter ORFs. The following corrections brought these sequences into the correct reading frame: L1-1_Cpo, ignored an in-frame stop codon at bp 3050&#8211;3052 and used the original sequence for the alignment; L1-1_DV, added a N after bp 6015; and, L1A_Mim, deleted an A at bp 1590&#8211;1591 and bp 5336&#8211;5337.</p><p>The backbone plasmid for chimera constructions used in the retrotransposition assays was based on pL1PA1tag, a gift from Dr. Astrid Roy-Engel. pL1PA1tag contains a codon-optimized consensus of the PA1 family of human L1 in a pBSSK<sup>&#8722;</sup> (Agilent Technologies, Inc., Santa Clara, CA) backbone. A puromycin resistance gene and its affiliated promoter pPGKpuro (Addgene, Cambridge, MA) were cloned into pL1PA1tag, creating plasmid pLY1004. The L1 insert of pLY1004 was removed by <italic>Nhe</italic>I and <italic>Eco</italic>RI digestion, creating the final plasmid backbone (<xref ref-type="fig" rid="pgen-1004395-g003">Figures 3B and 3C</xref>).</p><p>The reconstructed L1 and manipulated IGR sequences were commercially synthesized by GenScript USA, Inc. (Piscataway, NJ). Reconstructed L1s were synthesized in two blocks consisting of ORF1+IGR and ORF2. The manipulated IGRs were synthesized separately or in combinations containing distinct cloning sites. The synthesized sequences were cloned into pUC57 with flanking ends compatible to the linearized pLY1004 backbone and with <italic>Bsa</italic>I or <italic>Bsm</italic>BI sites to generate compatible overhangs after digestion. ORF1 and IGR were subcloned into separate pUC57 plasmids. <xref ref-type="fig" rid="pgen-1004395-g003">Figure 3B</xref> illustrates the principle underlying the construction of the chimeric L1s. L1 ORFs and IGRs were amplified from these plasmids by PCR with Phusion high-fidelity polymerase (ThermoFisher Scientific, Waltham, MA) using primers designed to generate compatible overhangs when the PCR products are digested with <italic>Bsa</italic>I, <italic>Btg</italic>ZI or <italic>Eco</italic>RI. Human L1rp segments were cloned from pWA192 <xref ref-type="bibr" rid="pgen.1004395-An1">[67]</xref>, a gift from Dr. Wenfeng An, using the same principle. The L1 ORFs, IGRs and the linearized backbone plasmid pLY1004 were joined together by a multi-way ligation using T4 DNA ligase. All restriction enzymes and DNA modifying enzymes were from New England BioLabs, Inc. (Ipswich, MA) unless otherwise specified. All constructs were confirmed by sequencing the L1 insert.</p><p>Retrotransposition rates were tested in an assay derived from Moran <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1004395-Moran2">[61]</xref>, in which the number of cell colonies surviving G418 antibiotic selection represents the retrotransposition rate (<xref ref-type="fig" rid="pgen-1004395-g003">Figure 3C</xref>). Briefly, the transcription and retrotransposition of L1 trigger the splicing of the transcript and excision of the intron of the inverse-oriented <italic>neo</italic> cassette, granting the cell resistance to the antibiotic G418.</p><p>The HeLa cell line (ATCC CCL-2) was a gift from Dr. Wenfeng An and maintained in Dulbecco's Modified Eagle Medium with 4500 mg/L glucose and 110 mg/L sodium pyruvate (ThermoFisher Scientific) supplemented by 10% fetal bovine serum (Atlanta Biologicals, Lawrenceville, GA), 2 mM l-alanyl-l-glutamine dipeptide and 100 units/mL Penicillin-Streptomycin (ThermoFisher Scientific). The assay was conducted as described by An <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1004395-An1">[67]</xref>. The culture medium for antibiotic selection was similar to the cell maintenance medium except 2.5 ug/mL puromycin (CALBIOCHEM, Billerica, MA) or 50 mg/mL G418 (CALBIOCHEM) was added. Plasmids for transfection were prepared with the Promega (Fitchburg, WI) PureYield Plasmid Midiprep System and the cells were transfected with FuGENE HD transfection reagent (Promega) following the manufacturer's protocol. Retrotransposition assays of the chimeric L1s were repeated at least 12 times in three different batches and manipulated IGR assays were repeated at least four times.</p><p>To confirm retrotransposition, two retrotransposition-positive colonies of each chimeric L1 construct were isolated with cloning rings, dissociated with trypsin (ThermoFisher Scientific), seeded on T75 flasks and allowed to grow into confluence. Cells were harvested and their genomic DNA was extracted with the QIAamp DNA mini kit (QIAGEN, Germantown, MD). Genotyping PCRs were conducted with primers bracketing the intron of the G418 reporter gene as described by An <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1004395-An2">[92]</xref>. Briefly, genotyping PCR primers were designed to the <italic>neo</italic> cassette so that cells hosting retrotransposition events, and the corresponding spliced cassettes, yield 653 bp PCR products. pLY1101, a self-ligated version of the linearized pLY1004 without a L1 insertion, was constructed as a positive control; genotyping PCR of pLY1101 yields a 1556 bp construct corresponding to the unspliced <italic>neo</italic> cassette.</p>