<p>All experiments were performed in an Association for Assessment and Accreditation of Laboratory Animal Care International-accredited vivarium following Standard Operating Procedures. The studies were approved by the NIDCR Institutional Animal Care and Use Committee. All studies were littermate controlled. Matriptase-deficient (<italic>St14<sup>&#8722;/&#8722;</sup></italic>), PAR-2-deficient (<italic>F2rl1<sup>&#8722;/&#8722;</sup></italic>), and <italic>Meox2-Cre</italic> transgenic mice have been described previously <xref ref-type="bibr" rid="pgen.1004470-List1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1004470-List3">[18]</xref>, <xref ref-type="bibr" rid="pgen.1004470-Tallquist1">[33]</xref>, <xref ref-type="bibr" rid="pgen.1004470-Schmidlin1">[63]</xref>. Prostasin-deficient (<italic>Prss8<sup>&#8722;/&#8722;</sup></italic>) mice were generated by standard blastocyst injection of C57BL/6J-derived embryonic stem cells carrying a gene trap insertion in the <italic>Prss8</italic> gene (clone IST10122F12, Texas A&amp;M Institute for Genomic Research, College Station, TX). Details on mouse generation will be published separately. Ear or tail clips of newborn or two week old mice were subjected to genomic DNA extraction and genotyped by PCR (see <xref ref-type="supplementary-material" rid="pgen.1004470.s003">Table S1</xref> for primer sequences).</p><p>
<italic>In situ</italic> hybridization data were retrieved from the Eurexpress transcriptome atlas database <xref ref-type="bibr" rid="pgen.1004470-DiezRoux1">[30]</xref>. Gene names <italic>St14</italic>, <italic>Prss8</italic>, and <italic>F2rl1</italic>, respectively, were individually searched to determine expression at embryonic day 14.5.</p><p>Breeding females were checked for vaginal plugs in the morning and the day on which the plug was found was defined as the first day of pregnancy (E0.5). Pregnant females were euthanized in the mid-day at designated time points by CO<sub>2</sub> asphyxiation. Embryos were extracted by Caesarian section and the individual embryos and placentas were dissected and processed. Only living embryos with detectable heartbeat were used for further analysis. Tail clips or yolk sacs of individual embryos were washed twice in phosphate buffered saline, subjected to genomic DNA extraction, and genotyped by PCR (<xref ref-type="supplementary-material" rid="pgen.1004470.s003">Table S1</xref>). Newborn pups were euthanized by CO<sub>2</sub> inhalation at 0&#176;C. For histological analysis, the embryos and newborn pups were fixed for 24 h in 4% paraformaldehyde (PFA), processed into paraffin, sectioned, and stained with hematoxylin and eosin (H&amp;E), or used for immunohistochemistry as described below. To evaluate the development of embryonic and placental tissues, H&amp;E-stained midline sagittal embryonic and the midline cross sections from at least three <italic>F2rl1<sup>&#8722;/&#8722;</sup>;St14<sup>&#8722;/&#8722;</sup></italic> and three corresponding littermate controls were inspected by a certified pathologist for possible abnormalities.</p><p>Placental tissues from living E12.5 and E13.5 embryos were extracted and processed into paraffin as described above. To analyze the overall thickness of the placental labyrinth, a single midline cross-section at the level of the umbilical cord was stained with hematoxylin &amp; eosin and the maximum perpendicular distance between the undifferentiated chorionic epithelium and the labyrinth-supporting spongiotrophoblast layer was measured under a light microscope by an investigator blinded to the genotypes of individual embryos. The volume of the placental labyrinth was estimated using Cavalieri's principle. Briefly, PFA-fixed placental tissue was embedded in paraffin, cross sections 200 &#181;m apart covering the entire volume of the tissue were stained with h&amp;e, scanned using ScanScope system (Aperio Technologies, Vista, CA), and the area of the labyrinth was measured on each section using Image J 1.46r software (National Institutes of Health, MD). The total volume of the labyrinth was then estimated as the sum of partial volumes calculated as area of the labyrinth on each section multiplied by 200 &#181;m.</p><p>To evaluate branching of the fetal vasculature in the E12.5 and E13.5 placentas, a single midline cross section of each placenta at the level of the umbilical cord was immunostained with an anti-CD31 antibody (see below), followed by the manual counting of individual profiles of CD31-stained vessels within the placental labyrinth.</p><p>Antigens from 5 &#181;m paraffin sections were retrieved by incubation for 10 min at 100&#176;C with 1 mM EDTA, pH 8.0 for CD31 staining, or by incubation for 20 min at 100&#176;C in 0.01 M sodium citrate buffer, pH 6.0, for all other antigens. The sections were blocked with 2% bovine serum albumin (fraction V, MP Biomedicals, Solon, OH) in phosphate-buffered saline (PBS), and incubated overnight at 4&#176;C with 2 ug/ml rabbit anti-human CD31 (Santa Cruz Biotechnology, Santa Cruz, CA), or sheep anti-human matriptase (R&amp;D Systems, Minneapolis, MN), primary antibodies. Bound antibodies were visualized using biotin-conjugated anti-rabbit, or -sheep secondary antibodies (Vector Laboratories, Burlingame, CA) and a Vectastain ABC kit (Vector Laboratories) using 3,3&#8242;-diaminobenzidine as the substrate (Sigma-Aldrich, St. Louis, MO). All microscopic images were acquired on an Olympus BX40 microscope using an Olympus DP70 digital camera system (Olympus, Melville, NY).</p><p>Placentas were extracted from embryos at E12.5 or E13.5. The embryonic portion of each placenta was manually separated from maternal decidua using a dissection microscope. The tissues were then homogenized in ice-cold 62.5 mM Tris/HCl, pH 6.8; 2% SDS; 10% glycerol buffer supplemented with 1&#215; protease inhibitor cocktail (Sigma-Aldrich) and incubated on ice for 10 min. The lysates were centrifuged for 10 min at 20,000 g at 4&#176;C to remove tissue debris and the supernatant was used for further analysis as described below.</p><p>The protein concentration in cleared lysates from embryonic, placental, and newborn tissues was determined by BCA assay (Pierce, Rockford, IL). 80 &#181;g of total protein was loaded on 4&#8211;12% reducing SDS-PAGE and analyzed by Western blotting using a polyclonal sheep anti-human matriptase (R&amp;D Systems), mouse anti-cow desmoglein 1 and 2 (Fitzgerald Industries International, Acton, MA), mouse anti-human Gcm1 (Abcam, Cambridge, MA), rabbit anti-human syncytinA (SantaCruz Biotechnology), rabbit anti-human pan-cadherin, rabbit anti-human GAPDH (both Cell Signaling Technology, Danvers, MA), rabbit anti-human claudin-1, and mouse anti-human claudin-2 (both Invitrogen, Carlsbad, CA) primary antibodies, and goat anti-mouse, mouse anti-rabbit (both DakoCytomation), or donkey anti-sheep (Sigma-Aldrich) secondary antibodies conjugated to alkaline phosphatase. Alkaline phosphatase activity was then visualized using nitro-blue tetrazolium and 5-bromo-4-chloro-3&#8242;-indolyphosphate substrates. Where indicated, protein signal quantification was performed using ImageJ 1.46r software.</p><p>Pregnant female mice from breeding pairs heterozygous &#946;-galactosidase-tagged <italic>F2rl1</italic> knock-in mice were euthanized at E12.5 by cervical dislocation. Embryos were extracted by Caesarian section, and the individual embryos and placentas were dissected and placed in 2% PFA with 0.2% glutaraldehyde. Fixed tissues were stained overnight at room temperature in 1 mg/ml X-gal, 5 mM potassium ferricyanide, 5 mM potassium ferrocyanide, 2 mM magnesium chloride and 0.02% NP-40 in PBS. The tissues were post-fixed for 16 h in 4% PFA, embedded in paraffin, and sectioned. The sections were counterstained with nuclear fast red. All microscopic images were acquired on a Zeiss AxioImager Z1 light microscope (Carl Zeiss AG, Gottingen, Germany) using a Qicam FAST1394 digital camera from Qimaging.</p><p>100 &#181;l of PBS containing 1 uCi of 3-O-[methyl-<sup>14</sup>C]-D-glucose or [carboxyl-<sup>14</sup>C]-inulin (both Perkin-Elmer, Hanover, MD) was injected into the tail vein of pregnant female mice from <italic>St14;F2rl1</italic> double-heterozygous breeding pairs at 12.5 or 13.5 days of gestation. The mice were euthanized two min later by CO<sub>2</sub> inhalation, the embryos were extracted as described above, weighted, and lysed overnight at 60&#176;C in 500 &#181;l of 2% potassium hydroxide solution. The lysate was then neutralized with 50 &#181;l of 14 M hydrochloric acid, mixed with 10 ml of ScintiSafe scintillation liquid (Fisher Scientific, Pittsburg, PA), and [<sup>14</sup>C] activity was measured using Beckman LS6000IC scintillation counter (Beckman Coulter, Brea, CA). To evaluate the differences in placental function between the individual embryos, the activity was normalized to gram of embryonic tissue.</p><p>The survival of newborn and pre-weaning mice from PAR-2/matriptase double heterozygous or PAR-2/prostasin double heterozygous breeding pairs was statistically evaluated by using chi-square analysis of the observed versus the expected distribution of mice wildtype, heterozygous, and deficient for PAR-2 (<italic>F2rl1<sup>+/+</sup></italic>; <italic>F2rl1<sup>+/&#8722;</sup></italic>; and <italic>F2rl1<sup>&#8722;/&#8722;</sup></italic>, respectively) among living offspring carrying two (<italic>St14<sup>+/+</sup></italic> or <italic>Prss8<sup>+/+</sup></italic>), one (<italic>St14<sup>+/&#8722;</sup></italic> or <italic>Prss8<sup>+/&#8722;</sup></italic>), or no functional alleles (<italic>St14<sup>&#8722;/&#8722;</sup></italic> or <italic>Prss8<sup>&#8722;/&#8722;</sup></italic>) of the gene encoding the corresponding protease.</p><p>To evaluate the effect of matriptase or prostasin deficiency on the embryonic survival of PAR-2-deficient mice, chi-square analysis was performed on the observed versus the expected distribution of protease-expressing (<italic>St14<sup>+/+</sup></italic> and <italic>St14<sup>+/&#8722;</sup></italic>, or <italic>Prss8<sup>+/+</sup></italic> and <italic>Prss8<sup>+/&#8722;</sup></italic>, respectively) and protease-deficient (<italic>St14<sup>&#8722;/&#8722;</sup></italic> or <italic>Prss8<sup>&#8722;/&#8722;</sup></italic>) animals among the PAR-2-deficient (<italic>F2rl1<sup>&#8722;/&#8722;</sup></italic>) embryos extracted at different embryonic stages, as indicated in the text.</p><p>Morphometric parameters of the placental development were analyzed using tissues from at least five control and five <italic>F2rl1<sup>&#8722;/&#8722;</sup>;St14<sup>&#8722;/&#8722;</sup></italic> double-deficient animals and the observed values were statistically evaluated using a two-sample Student's t-test, two-tailed.</p>