<p>All mice were housed under a constant room temperature (22&#176;C), humidity level (55%), and a 12-h light:12-h dark cycle with food (W.F. Fisher &amp; Son; Somerville, NJ) and water available <italic>ad libitum</italic>. The procedures for the generation of Acer3 null mice were approved under the number AR2123 by the Institutional Animal Care and Use Committee (IACUC) of the Medical University of South Carolina, SC, USA, and other animal studies were approved under the number 580863 by the IACUC of Stony Brook University, Stony Brook, NY, USA. All the animal studies reported in this study were conducted according to the Animal Research: Reporting In Vivo Experiments (ARRIVE) guidelines, developed by the National Centre for the Replacement, Refinement &amp; Reduction of Animals in Research (NC3Rs).</p><p>The mouse Acer3 gene consists of 11 exons (<xref ref-type="fig" rid="pgen.1005591.g002">Fig 2A</xref>). An Acer3 targeting vector was constructed so that upon homologous recombination between the targeting vector and the WT Acer3 allele, exon 8 of the Acer3 gene was replaced by the neomycin resistant gene (<italic>Neo</italic>) cassette (<xref ref-type="fig" rid="pgen.1005591.g002">Fig 2A</xref>). To construct the <italic>Acer3</italic> targeting vector, with PCR, we amplified a 1.3 kb fragment (the short arm) and 5.8 kb fragment (the long arm) upstream and downstream, respectively, of exon 8 from the genomic DNA of 129 SvEv embryonic stem (ES) cells. The short and long arms were inserted upstream and downstream, respectively, of the <italic>Neo</italic> gene in the vector OSDUPDEL (<xref ref-type="fig" rid="pgen.1005591.g002">Fig 2A</xref>), a gift from Dr. Oliver Smithies at the University of North Carolina at Chapel Hill, Chapel Hill, North Carolina. The resulting Acer3 targeting construct was linearized by the restriction enzyme <italic>Not1</italic> and transfected into 129SvEv ES cells by electroporation as described [<xref ref-type="bibr" rid="pgen.1005591.ref065">65</xref>]. Transfected ES cells were cultured in medium containing G418 (200 &#956;g/ml), and the resulting G418-resistant ES clones in which one copy of the Acer3 alleles was correctly targeted by <italic>Neo</italic> were screened by Southern blot analyses (<xref ref-type="fig" rid="pgen.1005591.g002">Fig 2B</xref>), expanded, and injected into mouse C57BL/6J blastocysts using standard procedures as described [<xref ref-type="bibr" rid="pgen.1005591.ref065">65</xref>]. The injected blastocysts were transplanted into the uteri of pseudopregnant C57BL/6J mice, and the resulting chimeric mice were crossed to WT C57BL/6J mice to generate Acer3<sup>+/-</sup> mice. Acer3<sup>+/-</sup> mice with a mixed genetic background were backcrossed to WT C57BL/6J mice for 16 generations to obtain Acer3<sup>+/-</sup> mice with the sole C57BL/6J genetic background. These heterozygous mice were inbred to generate Acer3<sup>-/-</sup> mice and their Acer3<sup>+/+</sup> littermates, which were used for further studies. DNA was isolated from mouse tail clips as described [<xref ref-type="bibr" rid="pgen.1005591.ref066">66</xref>] and subjected to PCR-based genotyping using two different primer pairs (F1/B1 and F1/B2) as shown in <xref ref-type="fig" rid="pgen.1005591.g002">Fig 2A</xref>. Acer3<sup>+/+</sup>, Acer3<sup>+/-</sup>, and Acer3<sup>-/-</sup> mice were identified according to the PCR product patterns as shown in <xref ref-type="fig" rid="pgen.1005591.g002">Fig 2C</xref>.</p><p>Protein concentrations were determined with bovine serum albumin (BSA) as a standard using bicinchoninic acid (BCA) protein determination kits (Thermo Scientific; Waltham, MA) according to the manufacturer&#8217;s instructions.</p><p>Tissues were collected from mice euthanized by CO<sub>2</sub> suffocation followed by cervical dislocation, rinsed with phosphate buffered saline (PBS), snap-frozen in liquid nitrogen, and stored at -80&#176;C. The tissues were homogenized on ice with an electric tissue tearor (Biospec Products; Bartlesville, OK) in Buffer A (25 mM Tris-HCl, pH 7.4, 150 mM NaCl and 0.25 M sucrose) supplemented with a protease inhibitor mixture (Roche; Indianapolis, IN). After brief sonication, the tissue homogenates were centrifuged at 1,000 <italic>g</italic> at 4&#176;C for 5 min to sediment nuclei and tissue debris, and the resulting supernatants were centrifuged at 100,000 <italic>g</italic> at 4&#176;C for 45 min to pellet all cell membranes, which were resuspended by brief sonication in Buffer B (25 mM Tris, pH7.4, 5mM CaCl<sub>2</sub> and 150 mM NaCl). Membrane homogenates (20 &#956;g or 40 &#956;g of protein per tissue) were measured for alkaline ceramidase activity using NBD-C<sub>12</sub>-PHC as a substrate by a thin layer chromatography (TLC) method [<xref ref-type="bibr" rid="pgen.1005591.ref030">30</xref>] or using regular ceramides as substrates by liquid chromatography tandem-mass spectrometric analysis (LC-MS/MS) method [<xref ref-type="bibr" rid="pgen.1005591.ref035">35</xref>]. Briefly, NBD-C<sub>12</sub>-PHC or a regular ceramide was dispersed by water bath sonication in Buffer C (25 mM glycine-NaOH, pH 9.4, 5 mM CaCl<sub>2</sub>, 150 mM NaCl, and 0.3% Triton X-100). The lipid-detergent mixtures were boiled for 30 s and chilled on ice immediately to form homogeneous lipid-detergent micelles, which were mixed with an equal volume of membrane homogenates suspended in Buffer B. Enzymatic reaction was carried out at 37&#176;C for 40 min, and stopped by heating at 100&#176;C on a heating block. The reaction mixtures were completely dried, cooled down to room temperature, and dissolved in chloroform/methanol (2:1). For the TLC method, 20 &#956;l of each reaction mixture was spotted onto TLC plates, which were developed in a solvent system consisting of chloroform, methanol, and 25% ammonium hydroxide (90:30:0.5). The TLC plates were dried and scanned by an imaging system (Typhoon FLA 7000, GE Healthcare Life Sciences; Pittsburgh, PA) set at the fluorescence mode. The fluorescent band of NBD-C<sub>12</sub>-fatty acid (NBD-C<sub>12</sub>-FA) released from NBD-C<sub>12</sub>-PHC was identified according to the standard NBD-C<sub>12</sub>-FA spotted on the same TLC plate. The content of NBD-C<sub>12</sub>-FA in each reaction was determined according to a standard curve generated from known concentrations of the standard NBD-C<sub>12</sub>-FA. For the LC-MS/MS method, lipids were extracted from dried reaction mixtures with chloroform and methanol. Sphingoid bases in the lipid extracts were determined by LC-MS/MS with D-e-C<sub>17</sub>-SPH as an internal standard as described [<xref ref-type="bibr" rid="pgen.1005591.ref035">35</xref>].</p><p>Tissues collected from Acer3<sup>+/+</sup> or Acer3<sup>-/-</sup> mice were homogenized on ice as previously described in buffer E (25 mM Tris&#8211;HCl, pH 7.4, 150 mM NaCl, 1mM EDTA, and 1mM EGTA). Lipids from tissue homogenates (2 mg protein per sample) were extracted with ethyl acetate/isopropanol/water (60/30/10, v/v). The lipid extracts were dried under N<sub>2</sub> gas stream and reconstituted in methanol, and sphingolipids were determined by LC-MS/MS performed on a TSQ 7000 triple quadrupole mass spectrometer (Thermo Finnigan; Ringoes, NJ) as described [<xref ref-type="bibr" rid="pgen.1005591.ref035">35</xref>]. Amounts of sphingolipids in different samples were normalized to protein contents.</p><p>Mice were deeply anesthetized with urethane (1.5 g/kg) and perfused transcardially with PBS followed by 4% paraformaldehyde (PFA) as described [<xref ref-type="bibr" rid="pgen.1005591.ref059">59</xref>]. Brains, left lateral lobes of liver, and left lungs were removed and post-fixed in 4% PFA before dehydration and embedding in paraffin. Sagittal sections from one individual mouse brain were stained with an antibody against the PC marker calbindin-D-28K (Sigma-aldrich, St. Louis, MO) using a Histostain-Plus IHC staining kit (Invitrogen; Grand Island, NY). Cells stained positive for calbindin D in the cerebellum were enumerated from 3 random 10&#215; fields of view under an Imager M2 microscope (Zeiss; Thornwood, NY) in a blind manner, and three serial sections were scored and the average score of each mouse cerebellum was calculated. Lung and liver tissue sections were stained with anti-Ki67 antibody (Biocare Medical; Concord, CA) using a Histostain-Plus IHC staining kit (Invitrogen; Grand Island, NY). Ki67-positive cells were scored under an Imager M2 microscope (Zeiss; Thornwood, NY) in a blind manner. In lungs, Ki67-positive cells in the bronchial epithelium and alveoli were scored from 4 random 10&#215; and 4 random 20&#215; microscopic fields of view, respectively, and the percentage of Ki67-positive cells was calculated. In liver, Ki67 positive cells were enumerated from 4 random 10&#215; fields of view.</p><p>After mice were perfused with 4% PFA under deep anesthesia, brains were removed and cut into sagittal slices on a cryostat. Cryosections were subjected to TUNEL assays using an <italic>in situ</italic> cell death detection kit (Roche; Indianapolis, IN) according to the manufacturer&#8217;s instructions. After TUNEL staining, sections were subjected to IFS with calbindin D-28K antibody (Sigma-aldrich, St. Louis, MO). TUNEL and IFS were analyzed under an Imager M2 microscope (Zeiss; Thornwood, NY) in a blind manner. Lung and liver paraffin-embedded sections were prepared as described above and subjected to TUNEL assays using TACS<sup>&#174;</sup> 2 TdT diaminobenzidine kit (Trevigen; Gaithersburg, MD) according to the manufacturer&#8217;s instructions. TUNEL-positive cells were scored in a blind manner as described earlier. In the lung sections, positive cells in bronchial epithelium and in alveoli were counted from 4 random 10&#215; and 4 random 20&#215; microscopic fields of view, respectively. In the liver sections, positive cells were enumerated from 4 random 10&#215; fields of view.</p><p>RNAs were extracted from fresh brain tissues using RNeasy mini kits (Qiagen; Valencia, CA) according to the manufacturer&#8217;s instructions. The RNAs were reversely transcribed into cDNAs, which were subjected to qPCR analyses as described [<xref ref-type="bibr" rid="pgen.1005591.ref067">67</xref>] using primer pairs specific to each of the following genes: <italic>Acer1</italic> (5&#8217;-ATGCTCATAGGTCTGTTCTC-3&#8217; and 5'-AGTGGTTATAGTTACCAGGC-3&#8217;), <italic>Acer2</italic> (5&#8217;-GTGTGGCATATTCTCATCTG-3&#8217; and 5&#8217;-TAAGGGACACCAATAAAAGC-3&#8217;), <italic>Acer3</italic> (5&#8217;-GTGTGGCATATTCTCATCTG-3&#8217; and 5&#8217;-TAAGGGACACCAA TAAAAGC), and <italic>&#946;-Actin</italic> (5&#8217;- GATGTATGAAGGCTTTGGTC-3&#8217; and 5&#8217;-TGTGCACTTTTATTGGTCTC-3&#8217;). qPCR was performed on an ABI Prism 7000 sequence detection system and mRNA levels for each gene were analyzed with the ABI Prism 7000 software (Applied Biosystems). Relative mRNA levels of Acer1, Acer2, or Acer3 were estimated using &#916;&#916;Ct method as described [<xref ref-type="bibr" rid="pgen.1005591.ref068">68</xref>] with &#946;-Actin as internal control.</p><p>Paraffin cerebellar sections from 8-month-old mice were subjected to Luxol fast blue staining and myelin sheaths were assessed as described [<xref ref-type="bibr" rid="pgen.1005591.ref059">59</xref>,<xref ref-type="bibr" rid="pgen.1005591.ref069">69</xref>]. Briefly, images were taken using a black and white camera then pseudo-colored using Imagepro software. The width of myelinated tracts was measured as an indicator for alteration of myelin.</p><p>Mouse tissues were minced with a razor blade and homogenized as previously described in Buffer F (50 mM Tris-HCl pH 7.5, 250 mM NaCl, 1mM EDTA, 1mM EGTA, 1% Triton X-100, and 1% SDS) supplemented with protease inhibitor mixture (Roche; Indianapolis, IN) and phosphatase inhibitor cocktail (Thermo Scientific; Waltham, MA). Proteins were separated on SDS-polyacrylamide gels and transferred onto nitrocellulose membranes, which were probed with the primary antibody against Ki67 (Biocare Medical; Concord, CA), proliferating cell nuclear antigen (PCNA), caspase 3 (Cell Signaling; Beverly, MA), cleaved caspase 3 (Cell Signaling; Beverly, MA), MBP (Sigma-aldrich; St. Louis, MO), or &#946;-actin (Santa Cruz; Dallas, Texas). Band density was quantified by densitometry using NCBI ImageJ.</p><p>Open field tests were performed as described [<xref ref-type="bibr" rid="pgen.1005591.ref070">70</xref>] with slight modification. The open field consists of a square arena (40&#215;40 cm), with a floor enclosed by continuously 25-cm-high walls. The animals were tested in this field during the light phase of their light/dark cycle. The test was initiated by placing a single mouse in the center of the arena and letting it move freely for 5 min. Mouse behavior was continuously videotaped by a video camera placed over the structure and then encoded using a continuous sampling method by Ethovision XT 7.0 (Noldus; Attleboro, MA). The arena was cleaned with ethanol after every test. A number of parameters were collected during the session. They comprised: (1) Velocity: the distance of the mouse moved per second (cm/s); (2) Distance: the distance (cm) the mouse moved in the five min of the test; (3) Latency to corner: the time (s) from when the mouse was first placed in the arena to when the mouse faced into a corner; (4) Rearing: the number of times the mouse reared on its hindpaws.</p><p>Mice were placed in the center of a wooden-framed wire mesh, and the frame was inverted at a height of 35 cm above soft padding. Mice were observed for movement while hanging from the wire mesh, and the hang time was measured from the time the frame was inverted to the time the mouse fell. The maximum length of time the frame was inverted was one minute [<xref ref-type="bibr" rid="pgen.1005591.ref071">71</xref>].</p><p>The muscular strength of mouse forepaws or hindpaws was measured using a grip strength meter (TSE Instruments International; Chesterfield, MO). Two attachments were used&#8212;a bar specific for mouse forepaws and a wire grid that allowed for simultaneous grip with all four paws. In both testing scenarios, the mouse was held by the base of its tail and allowed to firmly grab the attachment with either the forepaws, or with forepaws and hindpaws combined. A gentle and consistent tug backwards on the tail was applied to ensure it was firmly gripping the attachment before being quickly pulled off perpendicular to the force gauge in one motion [<xref ref-type="bibr" rid="pgen.1005591.ref072">72</xref>,<xref ref-type="bibr" rid="pgen.1005591.ref073">73</xref>]. The software for the meter displayed the peak force achieved during this motion. Five successive measurements were taken and the three highest measurements were averaged to give the strength score. Each mouse was given fifteen seconds of rest between each trial. The peak force (N) of each trial for the hindpaws was calculated by subtracting the peak force for the forepaws from the peak force of all four paws.</p><p>Rotarod tests were performed using a rotarod device (Coulbourn Instruments; Whitehall, Pennsylvania). Mice were trained for 10 min, 6 days a week, for a total of 9 sessions. Each session included 3 trials. The first two trials (3 min each) consisted of assisted training where the mice were gently supported by the trainer&#8217;s fingers or hand while walking on the rod. The third trial was consisted of independent walking (for 4 min). For each trial, animals were trained at their maximum tolerable speed. Training was truncated at 25 rpm for both assisted and independent stepping. Mice that achieved independent stepping at 25 rpm before 9 sessions were still given the same training duration until 9 sessions were met. Note that this baseline training was crucial to ensure that mice were falling off the rod not because they were unfamiliar with the task, but because they were not able to meet the demands of the rotarod task. Mice were tested for two consecutive days immediately following their final training day. Each testing day consisted of three constant speeds; the latency to fall at each constant running speed was used as measures for rotarod tests [<xref ref-type="bibr" rid="pgen.1005591.ref074">74</xref>,<xref ref-type="bibr" rid="pgen.1005591.ref075">75</xref>] and the highest score of both days was recorded.</p><p>Footprint tests were performed as described [<xref ref-type="bibr" rid="pgen.1005591.ref076">76</xref>]. Briefly, the hindfeet and forefeet of the mice were coated with blue and red nontoxic paints, respectively. Mice were then allowed to walk along a 40-cm-long, 10-cm-wide runway (with 15-cm-high walls) into an enclosed box. All mice had three training runs and were then given one testing run. A fresh sheet of white paper was placed on the floor of the runway for each run. The footprint patterns were analyzed for four parameters (all measured in centimeters): (1) stride length measured as the average distance of forward movement between each stride, (2) hindpaw step width measured as the average distance between left and right hind footprints, (3) forepaw step width measured as the average distance between and left and right front footprints, and (4) distance between ipsilateral forepaw and hindpaw footprints measured to indicate the extent of overlap between ipsilateral forepaws and hindpaws (paw overlap). For each parameter, three values were measured, excluding where the animal was initiating and finishing movement, respectively. The mean value of each set of three values was used in subsequent datum analyses.</p><p>Beam walking tests were performed as described [<xref ref-type="bibr" rid="pgen.1005591.ref076">76</xref>] with slight modification. Two customized round beams with diameters of 17 and 10 mm were used. The beams were placed horizontally, 50 cm above the bench surface, with one end mounted on a narrow support and the other end attached to an enclosed box (20&#215;10&#215;10 cm) into which mice could escape. During training, mice were placed at the start of the beam and trained over 3 days (4 trials per day) to traverse the beam to the enclosed box. Once the mice were trained, they received 3 consecutive trials on each beam, progressing from the 17-mm to 10-mm beam. Mice were allowed up to 60 s to traverse each beam. The time to traverse each beam was recorded. If a mouse fell off the beam, its latency was recorded as 60 s. The walking distance was measured for each trial. Mouse hindlimb movement was continuously monitored by a video camera, and the recorded videos were used for enumerating the times that the hindfeet of mice slip off each beam in each trial. For each parameter, the mean value of the three trials for each beam was used in subsequent datum analyses.</p><p>Hindlimb-clasping tests were carried out as described [<xref ref-type="bibr" rid="pgen.1005591.ref077">77</xref>]. Mice were individually lifted by grasping their tails near the base and their hindlimb positions were observed for 10 s. If the hindlimbs were consistently splayed outward and away from the abdomen, it was scored as 0. If one hindlimb was retracted toward the abdomen for more than 5 s, it was scored as 1. If both hindlimbs were partially retracted toward the abdomen for more than 5 s, it was scored as 2. If its hindlimbs were entirely retracted and touching the abdomen for more than 5 s, it was scored as 3.</p><p>Cerebella were dissected from mice perfused with 3.4% PFA/1.25% glutaraldehyde under deep anesthesia and sliced into 60 &#956;m sagittal sections under a microtome [<xref ref-type="bibr" rid="pgen.1005591.ref078">78</xref>]. The tissue sections were then processed for TEM as describe[<xref ref-type="bibr" rid="pgen.1005591.ref079">79</xref>]. Briefly, the tissue sections were washed with 0.1 M sodium cacodylate buffer (pH 7.4) containing 2 mM CaCl<sub>2</sub> and post-fixed for 1 h in a reduced osmium solution containing 2% osmium tetroxide, 1.5% potassium ferrocyanide 2 mM CaCl<sub>2</sub> in 0.15 mM sodium cacodylate buffer (pH 7.4). After being washed with double-distilled water (ddH<sub>2</sub>O), the tissue sections were incubated with a 1% thiocarbohydrazide (TCH) solution in ddH<sub>2</sub>O for 20 minutes at room temperature (RT). After being washed with ddH<sub>2</sub>O, the tissue sections were fixed with 2% osmium tetroxide in ddH<sub>2</sub>O for 30 min at Rt, followed by 1% uranyl acetate (aqueous) overnight at 4&#176;C. After being washed with ddH<sub>2</sub>O, the tissue sections were subjected to <italic>en bloc</italic> Walton&#8217;s lead aspartate staining in a 60&#176;C oven for 30 min as described [<xref ref-type="bibr" rid="pgen.1005591.ref080">80</xref>] After five 3-min rinses, the tissue sections were dehydrated sequentially in 20%, 50%, 70%, 90%, 100%, and 100% ice-cold ethanol (anhydrous), 5 min each, followed by anhydrous ice-cold acetone at RT for 10 min. The tissue sections were infiltrated with 30% Epon-Aradite for 2 h, 50% Epon-Aradite for another 2 h, 75% Epon-Aradite overnight, and 100% Epon-Aradite for a day with one change before being embedded in two permanox slides by flat-embedding procedure in a 60&#176;C oven for 48 h. The polymerized tissue blocks were cut into ultrathin sections (80 nm), which were collected on 150 mesh grids. and examined under an FEI CM120 transmission electron microscope (equipped with a Gatan GIF100 image filter) operating at a beam energy of 120 keV. Images were acquired using a Gatan 1 k &#215; 1 k cooled charge-coupled device (CCD) camera.</p><p>Statistical analyses were performed using the Student&#8217;s t-test or two-way Anova using Graphpad Prism 5.0. Post-hoc Bonferroni&#8217;s correction was used for multiple comparisons. <italic>p</italic>-values&lt;0.05 were considered statistically significant.</p>