<p>We studied a total of 57 individuals (18 pampas cat, 16 kodkod, and 23 Geoffroy&#8217;s cat) whose origin and phenotype are listed in Table S1 in <xref ref-type="supplementary-material" rid="pgen.1004892.s001">S1 Data</xref>. Biological material was either blood (collected from wild-caught individuals in the context of field ecology studies) or tissue (from road-killed animals encountered during routine wildlife surveys); in all cases, sampling was performed following appropriate national regulations for handling animals and biological materials. DNA was prepared by extraction with phenol/chloroform, and assessed for concentration and quality by fluorometry and agarose gel electrophoresis, respectively.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">When this work was initiated, we made use of an annotated but incomplete (1.9x) genome assembly of the domestic cat, felCat3 [<xref ref-type="bibr" rid="pgen.1004892.ref039">39</xref>]. During the course of the work, two additional assemblies became available, and all coordinates used in the manuscript now refer to felCat5/Felis_catus-6.2, <ext-link ext-link-type="uri" ns0:href="http://genome.wustl.edu/genomes/detail/felis-catus/">http://genome.wustl.edu/genomes/detail/felis-catus/</ext-link>.</p><p>After PCR amplification, protein-coding exons of <italic>ASIP</italic> [<xref ref-type="bibr" rid="pgen.1004892.ref008">8</xref>] and <italic>MC1R</italic> (Table S2 in <xref ref-type="supplementary-material" rid="pgen.1004892.s001">S1 Data</xref>) were analyzed by automated capillary sequencing. We first examined 8 individuals of each species (4 of each phenotype), then extended those results to all available samples (Supplemental Tables S1, S4, S5, S6). Sequencing electropherograms were verified and corrected with Sequencher 4.2 (GeneCodes Corporation), and every potential variant was carefully inspected for confirmation. Homologous nucleotide and amino acid sequences of each gene from additional mammalian species were obtained from GenBank (Table S3 in <xref ref-type="supplementary-material" rid="pgen.1004892.s001">S1 Data</xref>) and aligned using ClustalW.</p><p>DNA libraries for Illumina sequencing were generated using standard protocols. Briefly, 700ng to 2.5ug of input DNA (depending on the level of DNA degradation assessed by agarose gel electrophoresis before shearing) was sheared to a size range of 100&#8211;500 bp, and custom inline barcodes were added during adapter ligation for paired-end sequencing.</p><p>For target selection, we utilized a set of fosmid libraries that had been previously end-sequenced and mapped to a 3x draft assembly, V17e/felCat4 [<xref ref-type="bibr" rid="pgen.1004892.ref040">40</xref>]; identity of all fosmids was verified by PCR of each end.</p><p>A detailed protocol for probe preparation, hybridization, and capture is described elsewhere (Day et al., manuscript submitted). Briefly, fosmids were pooled according to their inferred individual mass, a total of 1.5ug of input was sheared to 100&#8211;500 bp, and DNA fragments were used to prepare biotinylated RNA with a Megascript T7 kit (Ambion). After DNAse treatment and removal of unincorporated nucleotides, hybridizations were carried out in a volume of 26ul that contained 300 ng of probe and 500ng of library DNA (125ng each of 4 inline barcoded libraries). Selected DNA was recovered with magnetic beads and amplified by PCR (20 cycles using standard Illumina primers). Library 4-plex pools were sequenced as 12-plex library sets on a single Illumina HiSeq 2000 lane.</p><p>We used <italic>ASIP</italic> and <italic>MC1R</italic> regions from chromosomes A3 and E2, respectively, as reference sequences, extracted from the Felis_catus-6.2 assembly. The genomic reference was masked for transposable elements and low complexity regions with <italic>RepeatMasker</italic> [<xref ref-type="bibr" rid="pgen.1004892.ref041">41</xref>]. To minimize genotyping errors related to inter-species differences in allele structure and distribution, we first mapped all raw sequence reads against the domestic cat reference using the <italic>Burrows-Wheeler Aligner</italic> (<italic>BWA</italic>) with default parameters [<xref ref-type="bibr" rid="pgen.1004892.ref042">42</xref>], then used <italic>SAMtools</italic> [<xref ref-type="bibr" rid="pgen.1004892.ref043">43</xref>] to compute consensus sequences for these preliminary alignments, and to <italic>de novo</italic> assemble targeted regions in each of the three species separately.</p><p>Sequence reads were then remapped to these <italic>de novo</italic> species-specific consensus sequences and subsequently used to for variant calling according to <italic>GATK</italic> best practices guidelines [<xref ref-type="bibr" rid="pgen.1004892.ref044">44</xref>]. Briefly, we started by applying alignment quality control procedures available in <italic>GATK</italic> to detect sequence intervals with low quality mappings (<italic>i.e.</italic> possibly related to the presence of sequence variants, such as small insertions or deletions, in subsets of the analyzed samples). In all such cases, a thorough local realignment of the reads was performed to minimize the number of mismatching bases. The <italic>GATK Unified Genotyper</italic> (UG) tool was applied on realigned reads to infer the genotype structure simultaneously across all samples for each species. The raw genotype calls were subjected to a filtering procedure by imposing thresholds on a set of quality criteria, including minor allele frequency MAF&gt;10%, Phred-scaled mapping quality MQ&gt;40, UG quality by depth QD&gt;2 and UG HaplotypeScore&gt;13. The filtered calls were further restricted to a small subset of high quality calls, to satisfy an average variant density threshold of ~1 per 1kb of target sequence. Finally, the <italic>BEAGLE</italic> genetic analysis software package [<xref ref-type="bibr" rid="pgen.1004892.ref045">45</xref>] was used to check genotype consistency across all samples of each species and infer the haplotype phase of selected variants.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Curated genotype and haplotype data was then used for downstream analyses, including nucleotide diversity and polymorphism statistics with <italic>DnaSP</italic> 5.10 [<xref ref-type="bibr" rid="pgen.1004892.ref046">46</xref>], EHH [<xref ref-type="bibr" rid="pgen.1004892.ref047">47</xref>], and generation of median-joining haplotype networks (<italic>Network</italic> 4.5.0.0; <ext-link ext-link-type="uri" ns0:href="http://www.fluxus-engineering.com">http://www.fluxus-engineering.com</ext-link>). Haplotype bifurcation plots in <xref ref-type="fig" rid="pgen.1004892.g005">Fig. 5</xref> were generated with <italic>Sweep</italic> 1.1 (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/mpg/sweep/index.html">http://www.broadinstitute.org/mpg/sweep/index.html</ext-link>) and <italic>rehh</italic> 1.0 [<xref ref-type="bibr" rid="pgen.1004892.ref048">48</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">DNA sequences reported in this manuscript are publicly available from DRYAD (<ext-link ext-link-type="uri" ns0:href="http://doi:10.5061/dryad.pq482">doi:10.5061/dryad.pq482</ext-link>).</p>