<p>Ethical approval for this work was obtained from the Cambridgeshire Regional Ethics Committee (REC08/H0306/21). All participants provided written informed consent.</p><p>We recruited a cohort of adult healthy volunteers, free from autoimmune disease or any other chronic condition and not taking any regular medication at the time of enrolment. Adult patients with active inflammatory bowel disease (Crohn&#8217;s disease or ulcerative colitis) were recruited from a specialist inflammatory bowel disease clinic at Addenbrooke&#8217;s hospital in Cambridge, UK, prior to commencing treatment, as described previously [<xref ref-type="bibr" rid="pgen.1005908.ref032">32</xref>]. Diagnosis was made using standard endoscopic, histopathological, and radiological criteria. Patients receiving immunomodulators or corticosteroids were excluded to avoid potential confounding effects on gene expression. Patients with a diagnosis of ANCA-associated vasculitis (either granulomatosis with polyangiitis, formerly Wegener&#8217;s granulomatosis, or microscopic polyangiitis, but not eosinophilic granulomatosis with polyangiitis, formerly Churg-Strauss syndrome) and evidence of active disease were recruited from a specialist clinic at Addenbrooke&#8217;s Hospital. Active disease was defined by Birmingham Vasculitis Activity Score (BVAS), elevated acute phase markers, and the clinical assessment that induction immunosuppression would be required to manage disease, as described previously [<xref ref-type="bibr" rid="pgen.1005908.ref033">33</xref>]. 27 of 46 AAV patients were venesected at the time of first clinical presentation prior to treatment. The remainder had clearly defined flares of previously quiescent disease on stable maintenance therapy, and required treatment with induction immunosuppression. See <xref ref-type="supplementary-material" rid="pgen.1005908.s025">S5 Table</xref> for more detailed demographics.</p><p>Cell separations were performed using positive selection with magnetic-activated cell sorting (MACS) as previously described [<xref ref-type="bibr" rid="pgen.1005908.ref034">34</xref>]. Blood samples (100 ml) were collected into 4% sodium citrate and PBMC isolated by centrifugation over Histopaque 1077 (Sigma). The PBMC sample was split into two aliquots and CD14 monocytes were isolated from one aliquot and CD19 B cells from the other by magnetic cell sorting using CD14 and CD19 microbeads (Miltenyi Biotec) according to the manufacturer&#8217;s instructions. CD4 and CD8 T cells were then isolated from the CD14 and CD19 negative fractions, respectively, by magnetic cell sorting using CD4 and CD8 microbeads as described by the manufacturer. CD16 neutrophils were obtained from the red cell/granulocyte pellet by red cell lysis followed by positive selection using CD16 microbeads as described by the manufacturer.</p><p>RNA was extracted from cell lysates using either Qiagen RNEasy Mini Kits or Qiagen Allprep kits according to the manufacturer&#8217;s instructions. RNA quality was assessed using an Agilent Bioanalyser 2100 (Agilent Technologies) and quantified by spectrophotometry using a NanoDrop ND-1000 spectrophotometer (Thermo Scientific). Genomic DNA was extracted using Qiagen Allprep kits according to the manufacturer&#8217;s instructions and quantified as above.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Patients with IBD and healthy controls were genotyped using the Illumina Human OmniExpress12v1.0 BeadChip at the Wellcome Trust Sanger Institute according to the manufacturer&#8217;s protocol. After genotype calling with Illumina GenCall software, SNP data was processed in the following sequence using PLINK [<xref ref-type="bibr" rid="pgen.1005908.ref035">35</xref>]. Samples with a sex mismatch, abnormal heterozygosity, proportion of missing SNPs &gt;5%, and duplicated or related samples (identified using Identity by State) were removed. SNPs with missing calls &gt;5%, extreme deviation from Hardy-Weinberg Equilibrium (p-value &lt;1&#215;10<sup>-8</sup>), or which were monomorphic were removed. Finally, principal components analysis of the post-QC genotype calls combined with calls from HapMap3 founder individuals (downloaded from <ext-link ext-link-type="uri" ns0:href="http://hapmap.ncbi.nlm.nih.gov/">http://hapmap.ncbi.nlm.nih.gov/</ext-link>), using the R/Bioconductor snpStats package, confirmed all samples were of European ancestry (<xref ref-type="supplementary-material" rid="pgen.1005908.s018">S18 Fig</xref>). Genotype data was read into R version 3.02 (<ext-link ext-link-type="uri" ns0:href="http://www.R-project.org/">http://www.R-project.org/</ext-link>) using the snpStats package, and stored as a SnpMatrix object. SNPs with minor allele frequency (MAF) &lt;5% were excluded from eQTL analysis, leaving 626,858 autosomal SNPs. These data have been deposited in the European Genome-phenome Archive (EGA) and are available on request (EGAS00001001251).</p><p>Genotype data for AAV patients were available from a previous GWAS [<xref ref-type="bibr" rid="pgen.1005908.ref036">36</xref>]. Briefly, genotyping was performed by AROS Applied Biotechnology (Aarhus, Denmark) using the Affymetrix SNP 6.0 Platform. SNP genotypes were called using the CRLMM algorithm [<xref ref-type="bibr" rid="pgen.1005908.ref037">37</xref>], and standard QC performed as described previously [<xref ref-type="bibr" rid="pgen.1005908.ref036">36</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">200 ng RNA was processed for hybridization onto Affymetrix Human Gene ST 1.1 microarrays, according to the manufacturer&#8217;s instructions, prior to scanning. Microarrays were processed with the automated GeneTitan instrument which uses a 96-well plate. Chip probe intensity files (.CEL) were read into R version 3.02 using the Bioconductor oligo package [<xref ref-type="bibr" rid="pgen.1005908.ref038">38</xref>]. Raw intensity values were pre-processed with the Robust Multi-array Average (RMA) algorithm which performs background correction, quantile normalization, log<sub>2</sub> transformation and summarization to gene level via a median polish. Probe annotation was obtained from the pd.hugene.1.1.st.v1 Bioconductor package. The resulting expression matrix and phenotype data was stored as an ExpressionSet object. The raw intensities for the expression data for each cell type were read in and normalised separately. The arrayQualityMetrics package was run on the normalised expression matrices before and after batch correction with ComBat (see below) and poor quality samples removed. Microarray data are available in the ArrayExpress database (<ext-link ext-link-type="uri" ns0:href="https://www.ebi.ac.uk/arrayexpress">www.ebi.ac.uk/arrayexpress</ext-link>) under accession number E-MTAB-3554.</p><p>Principal components analysis of the expression matrices indicated batch effects. To address this, we used two alternative approaches. The first approach was removal of unwanted expression heterogeneity by adjustment for latent factors using Probabilistic Estimation of Expression Residuals (PEER) [<xref ref-type="bibr" rid="pgen.1005908.ref039">39</xref>]. The use of PEER has increased eQTL discovery compared to standard methods in a number of studies [<xref ref-type="bibr" rid="pgen.1005908.ref004">4</xref>, <xref ref-type="bibr" rid="pgen.1005908.ref040">40</xref>]. PEER can model both known confounders and hidden latent factors in the expression data. We ran PEER specifying batch and sex as known confounders, and used the expression residuals as the dependent variable in the subsequent eQTL scan. For the analysis of cell-type specificity, comparison of direction of effect between cell types, and for the intersection of eQTLs with GWAS SNPs, we used eQTLs detected from association scans using PEER residuals.</p><p>Alternatively, we performed batch correction using the ComBat function from the sva (Surrogate Variable Analysis) Bioconductor package [<xref ref-type="bibr" rid="pgen.1005908.ref041">41</xref>]. ComBat was implemented specifying diagnosis as a covariate of biological interest whose effect should be preserved. This method was used for the genotype &#215; disease interaction analysis and the pre- and post-treatment analysis.</p><p>The GGtools Bioconductor package [<xref ref-type="bibr" rid="pgen.1005908.ref042">42</xref>] was used to combine SnpMatrix and ExpressionSet objects for patients for whom we had both good quality genotype and expression data to create an smlSet object. <xref ref-type="table" rid="pgen.1005908.t001">Table 1</xref> shows the numbers of samples available for analysis by disease and cell type. For the Bayesian joint analysis across cell types presented in <xref ref-type="fig" rid="pgen.1005908.g001">Fig 1</xref>, we used all available samples, even if this meant differing sample sizes for each cell type. For the estimates of eQTL cell-type specificity from one-at-a-time cell-type analysis, we confined eQTL mapping to individuals with expression data available for all cell types to ensure equal power for each cell type. In the latter scenario, subsetting was performed after pre-processing and adjustment for latent factors or batch correction. In order to make direct comparisons with results from separate cell-type analysis, we also used the Bayesian method on the subset of individuals with expression data available for all cell types (<xref ref-type="supplementary-material" rid="pgen.1005908.s019">S19 Fig</xref>). For the detection of eQTLs to intersect with GWAS SNPs, we used all available samples to maximise power, even if this meant differing sample sizes for each cell type. The expression versus genotype boxplots in Figs <xref ref-type="fig" rid="pgen.1005908.g002">2</xref>, <xref ref-type="fig" rid="pgen.1005908.g003">3</xref>, <xref ref-type="fig" rid="pgen.1005908.g006">6</xref>, <xref ref-type="fig" rid="pgen.1005908.g007">7</xref> and <xref ref-type="fig" rid="pgen.1005908.g008">8</xref> show all individuals for whom we have good quality expression and genotype data rather than restricting to the subset of individuals with expression data available for all cell types.</p><p>Filtering of probesets in the expression data was performed prior to the eQTL scan. Filtering of expression data has been shown to increase power to detect differentially expressed genes, and should not bias FDR estimates as long as the variable of interest is not used in the filtering step [<xref ref-type="bibr" rid="pgen.1005908.ref043">43</xref>]. Probesets on sex chromosomes and those which did not map to an Entrez gene ID were removed. The remaining probesets were then filtered by variance (Bioconductor genefilter package). As well as increasing power, filtering by variance has two additional benefits. First, genes which are expressed homogeneously are not informative in an eQTL analysis. Second, unexpressed genes will show little variability between samples and thus filtering on the basis of lack of expression variability is a reliable method for detecting and excluding unexpressed genes from further analysis [<xref ref-type="bibr" rid="pgen.1005908.ref044">44</xref>]. In the downstream analysis we compared eQTLs detected in each cell type. In order make these comparisons valid, it was necessary to perform eQTL mapping on the same filtered list of probesets in each cell type. To achieve this, we retained any gene whose variance across samples was in the upper quartile in any one of the cell types. For the joint IBD-HV analysis, this resulted in 9,041 probesets when using all available samples, and 9,242 when restricting to the 65 individuals with expression data available for all cell types. For the AAV analysis, the same filtering procedure resulted in 8,916 probesets when using all available samples, and 8,979 when restricting analysis to the 33 patients with complete expression data available for CD4 T cells, CD8 T cells, monocytes and neutrophils.</p><p>This analysis was performed using the IBD-HV dataset, restricted to the individuals with expression data available for all 5 cell types (n = 65). To identify probesets targeting robustly expressed genes in each cell type, the following procedure was performed. First we calculated the mean expression of each probeset across all samples from a given cell type (using expression data that had been RMA-normalised and then batch-corrected with ComBat). To determine thresholds for declaring probesets as expressed or non-expressed in that cell type, we used the normalmixEM function from R package mixtools to fit a mixture of two Gaussian distributions to the vector of mean probeset expression values. Starting values for the expectation&#8211;maximization (EM) algorithm were chosen assuming 50% of expressed probesets, and the initial mean values of the underlying Gaussian distributions were set equal to the first and third quartile of the probeset expression distribution. The distribution with the lowest mean expression was considered as that corresponding to non-expressed genes. A detection threshold was set by taking the 95th percentile of this distribution. Probesets whose mean expression was above the detection threshold were considered to be expressed in that cell type. Subsequently, probesets that did not map to an Entrez gene ID were removed. The lists of probesets expressed in each cell type were then intersected to find those expressed in all cell types. Probesets on the sex chromosomes, and those which did not map uniquely to an Entrez ID were removed. This resulted in 5,186 probesets. The matrix of PEER-adjusted expression data was then subsetted to retain only these probesets, and the eQTL scan was performed using the residuals from PEER as the dependent variable.</p><p>We performed <italic>cis</italic> eQTL mapping with the All.cis function in the GGtools Bioconductor package. Previous work has suggested that the most <italic>cis</italic> eQTLs are approximately centred around the transcribed region, with few eQTLs lying more than 50kB from the gene [<xref ref-type="bibr" rid="pgen.1005908.ref006">6</xref>, <xref ref-type="bibr" rid="pgen.1005908.ref007">7</xref>, <xref ref-type="bibr" rid="pgen.1005908.ref045">45</xref>]. For each gene, we defined <italic>cis</italic> SNPs as those lying in the region spanning 100kB up and downstream from the gene start and end positions respectively. SNP location information was obtained using the Bioconductor package SNPlocs.Hsapiens.dbSNP.20120608 (dbSNP Build 137). Sensitivity analysis using varying distances to define <italic>cis</italic> found no increase in eQTL discovery for <italic>cis</italic> distances greater than 100 kB using a 5% FDR significance threshold.</p><p>The All.cis function calculates association statistics (score tests) for each <italic>cis</italic> SNP-transcript pair. Specifically, the function fits a generalized linear model with transcript abundance as the continuous dependent variable and genotype as the predictor variable. Optionally, potential confounders of an expression-genotype association can be included as covariates. Genotype is a discrete variable coded 0, 1, or 2 according to allelic dose (i.e. homozygous SNP genotypes are coded 0 or 2 and heterozygous genotypes are coded 1). For each gene, each SNP in the <italic>cis</italic> region was tested in turn for association with mRNA abundance (after adjustment with PEER) using an additive genetic model; after the base model was fitted (i.e. a model with an intercept and any specified covariates), a score test was then performed for addition of genotype to the model. The tests used are asymptotic chi-squared tests based on the vector of first and second derivatives of the log-likelihood with respect to the parameters of the additive model. Where the dataset comprised more than one disease grouping (i.e. IBD and healthy individuals), the diagnostic grouping was included as an additional covariate. We did not include sex as a covariate, as sex had already been specified as a known confounder when running PEER.</p><p>GGtools provides estimates of the FDR through estimation of the distribution of statistics under the null by breaking expression-genotype relationships through permutation of sample labels (the &#8216;plug-in FDR&#8217; method [<xref ref-type="bibr" rid="pgen.1005908.ref046">46</xref>]- see <xref ref-type="supplementary-material" rid="pgen.1005908.s026">S1 Text</xref>). A 5% FDR was used to declare statistical significance.</p><p>To evaluate eQTL cell-type specificity, we used the eQTLBMA package which implements the method described by Flutre <italic>et al</italic> [<xref ref-type="bibr" rid="pgen.1005908.ref016">16</xref>]. We ran this according to the instructions in the user manual. Briefly, we first transformed the expression level of each gene into the quantiles of a standard Normal distribution. Ties were broken randomly. We then ran the <monospace>eqtlbma_bf</monospace> programme, to compute the Bayes factors assessing the support in the data for each probeset-SNP pair being an eQTL. We analysed the IBD patient and HV data together, specifying disease status as a covariate. To maximise power, we used all available samples (from 91 IBD patients and 43 HVs), even if this meant that for some individuals we did not have full expression data for all cell types. In this scenario we used the option <monospace>--error hybrid</monospace>. Samples from AAV patients were analysed separately to avoid difficulties that might arise from combining genotype data from a separate platform. Again, we used all available samples. The output file containing the Bayes factors was then fed into the <monospace>eqtl_hm</monospace> programme, to fit the hierarchical model with an EM algorithm, and get to maximum-likelihood estimates of hyper-parameters and the configuration probabilities. To obtain the posterior probabilities, we need an estimate of the probability for a gene to have no eQTL in any tissue, <italic>&#960;</italic><sub>0</sub>. To do this, we implemented the EBF procedure, using the gene-level Bayes factors averaged over the grid and configuration weights (estimated via the EM algorithm). Finally we ran <monospace>eqtlbma_avg_bfs</monospace> to obtain (i) the posterior probability (PP) for the gene to have an eQTL in at least one cell type, (ii) the PP for a SNP to be &#8220;the&#8221; eQTL, assuming one eQTL per gene, (iii) the PP for the SNP to be an eQTL, (iv) the PP for the eQTL to be active in a given tissue, and (v) the PP for the eQTL to be active in a given configuration. We used a PP corresponding to a 5% Bayes FDR as the significance threshold.</p><p>To evaluate estimates of eQTL cell-type specificity from one-at-a-time cell-type analysis, we limited the IBD-HV analysis to 65 individuals (47 IBD patients, 18 HVs) in whom samples passing expression quality control were available for all five cell types. By using the same set of individuals, the sample size and genotype matrix (predictor variables) were identical for each cell type. In order to make a fair comparison between the results from the one-at-a-time cell-type analysis and those from eQTLBMA, we re-ran eQTLBMA on these same 65 individuals with full expression data for all 5 cell types. In this latter scenario we used the option <monospace>--error mvlr</monospace> (<xref ref-type="supplementary-material" rid="pgen.1005908.s019">S19 Fig</xref> and <xref ref-type="supplementary-material" rid="pgen.1005908.s026">S1 Text</xref>).</p><p>For all possible pairings of the cell types, we took those eSNP-gene associations that were statistically significant in both members of the pair in the one-at-a-time cell-type analyses (FDR &lt;0.05), and compared the direction of effect on expression in each. We did this by plotting the estimated coefficient (beta) for the genotype term in the first cell type against that in the second. The betas were estimated from regressing PEER-adjusted expression on genotype using the <monospace>lm</monospace> function in R. For the IBD-HV analysis, disease status was included as a covariate. eQTLs with opposing directions of effect between cell types have a positive beta in one cell type but a negative beta in the other (see <xref ref-type="supplementary-material" rid="pgen.1005908.s026">S1 Text</xref> for more detailed discussion).</p><p>To identify eQTLs influenced by the presence or absence of inflammatory disease we analysed the joint IBD-HV dataset using a linear model with a genotype &#215; disease interaction term. Expression data for each cell type was RMA normalised separately, but within each cell type expression data from healthy individuals and IBD patients were normalised together. We used expression data that had been batch-corrected with ComBat, as adjustment for latent factors with PEER might remove genuine disease effects. We again restricted testing for each gene to <italic>cis</italic> SNPs. For each gene-SNP pair, a linear model was fitted using expression as the dependent variable, and genotype (coded 0,1, or 2), disease status (coded 0 and 1 for healthy and IBD respectively), and a genotype &#215; disease interaction term as the predictor variables. This was performed in R using the <monospace>lm</monospace> function.</p><p>Fitting a model with an interaction term is statistically more robust than the na&#239;ve approach of separate analysis of IBD and HV cohorts, followed by comparison of the resulting lists to find eQTLs common to both groups, and those identified only in IBD or only in HVs. The latter approach would result in an excess of false declarations of group-specific eQTLs. In particular, the comparison of lists approach is unduly influenced by the difference in statistical power to detect eQTLs in each group when the group sample sizes are different. The IBD group was larger than the HV group, and so there was greater power to detect eQTLs in the former. This would have led to many false declarations of IBD-specific eQTLs had we used the comparison of lists approach (see <xref ref-type="supplementary-material" rid="pgen.1005908.s026">S1 Text</xref> for more detailed discussion).</p><p>The analysis was performed for CD4 T cells, CD8 T cells, monocytes and neutrophils. B cells were not analysed due to the low number of samples available from healthy individuals. For each cell type, we used all available IBD and HV samples (see <xref ref-type="table" rid="pgen.1005908.t001">Table 1</xref> for sample sizes). We limited testing of SNPs to those with MAF of 10% or higher, as power decreases with decreasing MAF. Probesets were filtered in the following order: first, removal of probesets that were non/lowly expressed in that cell type, and those which did not map to an Entrez gene ID; second, filtering of the remaining probesets by variance to retain only those in the upper 40%.</p><p>Interaction effects are harder to detect than main effects, so to improve our power we employed a &#8216;two-step&#8217; procedure adapted from methods used to detect genotype-environment interactions in GWAS [<xref ref-type="bibr" rid="pgen.1005908.ref047">47</xref>]. &#8216;Two-step&#8217; procedures aim to reduce the multiplicity burden associated with having a large number of variables to explore by applying some filter (step 1) on the number of variables actually tested (step 2). In order to prevent biasing of FWER/FDR estimates, the filtering procedure used in step 1 should be independent of the final test statistics in step 2. In step 1 we performed linear regression of expression on genotype (main effect of genotype), with no covariates. Importantly, by not including disease status as a covariate, we ensured independence of the step 2 statistics [<xref ref-type="bibr" rid="pgen.1005908.ref047">47</xref>]. SNP-gene pairs passing the step 1 threshold <italic>&#945;</italic><sub>1</sub> were eligible for step 2 testing. We used a p-value of 5&#215;10<sup>-5</sup> for <italic>&#945;</italic><sub>1</sub>. A more liberal <italic>&#945;</italic><sub>1</sub> allows more SNP-gene pairs through to step 2, but at the price of more multiple testing and a more stringent threshold in step 2. In addition, because there were multiple correlated SNPs due to LD, where multiple SNPs were significantly associated with the same gene at the threshold <italic>&#945;</italic><sub>1</sub>, only the most significant SNP per gene was taken forward to step 2.</p><p>In step 2, we fitted the full model with regression of expression on genotype, disease and a genotype &#215; disease interaction term. P-values for the interaction terms were then adjusted for the number of tests performed in step 2 with the p.adjust function in R using the Benjamini-Hochberg procedure, and a 0.15 FDR threshold used to declare significance. Q-values were also calculated.</p><p>Each cell type was analysed separately. We used eQTLBMA to jointly analyse expression data from the 3 timepoints: baseline, 3 months, and 12 months. We used we used the option <monospace>--error hybrid</monospace> as the individuals in each subgroup were overlapping but not identical. For genes with a significant eQTL (5% Bayes FDR), we took the SNP with the highest posterior probability for being the eQTL. For this list of SNP-gene associations, we then assessed the posterior probabilities for each of the 2<sup>3</sup> &#8722; 1 = 7 possible configurations.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To identify the effects of disease-associated SNPs on gene expression, and how this varies between cell types, we examined the overlap of SNPs associated with complex traits from the NIH National Human Genome Research Institute GWAS catalogue (<ext-link ext-link-type="uri" ns0:href="http://www.genome.gov/gwastudies/">http://www.genome.gov/gwastudies/</ext-link>) (download date 20 May 2015) with eQTL SNPs (eSNPs) discovered in each cell type in our analysis. The SNPs identified in eQTL studies or GWAS are not necessarily themselves causal variants, but instead may be in LD with them. Therefore GWAS tag SNPs, and proxy SNPs in high LD with them (<italic>r<sup>2</sup></italic> &gt;0.8), were cross-referenced with <italic>cis</italic> eQTLs found in each cell type. SNAP was used to identify proxy SNPs (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/mpg/snap/">http://www.broadinstitute.org/mpg/snap/</ext-link>), using the 1000 Genomes (<ext-link ext-link-type="uri" ns0:href="http://www.1000genomes.org/about">http://www.1000genomes.org/about</ext-link>) pilot 1 data (CEU individuals) as the reference population.</p><p>To define the eSNPs for cross-referencing, we repeated the analysis using two levels of stringency for declaring colocalisation of eQTL and GWAS signals. For the basic criteria, a GWAS signal was declared to colocalise with the eQTL if the GWAS SNP or one of its proxies was a significant eQTL (FDR &lt;0.05). For the more stringent criteria, in addition to the basic criteria, the GWAS SNP or one of its proxies had to be the most significant <italic>cis</italic> eSNP for that gene (<xref ref-type="supplementary-material" rid="pgen.1005908.s026">S1 Text</xref>, <xref ref-type="supplementary-material" rid="pgen.1005908.s016">S16 Fig</xref>).</p><p>For the detailed examination of the effects of IBD-associated SNPs on gene expression, we took SNPs identified in the IBD GWAS meta-analysis [<xref ref-type="bibr" rid="pgen.1005908.ref016">16</xref>] as hits in either Crohn&#8217;s disease (CD), ulcerative colitis (UC) or both (IBD) (Supplementary Table 2 of that paper), and cross-referenced these SNPs and proxies (<italic>r<sup>2</sup></italic> &gt;0.8) with the eQTL SNPs from our analysis as above (see <xref ref-type="supplementary-material" rid="pgen.1005908.s026">S1 Text</xref>).</p><p>Colocalisation testing was performed using the coloc package [<xref ref-type="bibr" rid="pgen.1005908.ref021">21</xref>]. We used the publically available summary statistics for IBD, CD and UC GWAS meta-analysis by Jostins <italic>et al</italic> [<xref ref-type="bibr" rid="pgen.1005908.ref017">17</xref>] along with our eQTL data.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">1) Expression data has been deposited into ArrayExpress (accession number E-MTAB-3554). 2) Genotype data has been deposited into the EGA (study accession number EGAS00001001251, url <ext-link ext-link-type="uri" ns0:href="https://www.ebi.ac.uk/ega/studies/EGAS00001001251">https://www.ebi.ac.uk/ega/studies/EGAS00001001251</ext-link>). Controlled access for scientific researchers is available via requests to the data access committee (EGAC00001000338). 3) Summary statistics for all significant eQTLs for each cell type and dataset are included as a compressed tarball (<xref ref-type="supplementary-material" rid="pgen.1005908.s027">S1 Data</xref>).</p>