<p>Strains were grown using standard culture conditions, as described by Brenner [<xref ref-type="bibr" rid="pgen.1005221.ref049">49</xref>]. Analyses were performed at 20&#176;C, unless otherwise noted. Cholesterol depletion conditions were following those described in Merris et al. [<xref ref-type="bibr" rid="pgen.1005221.ref080">80</xref>] by replacing agar with agarose, and by growing bacteria OP50 and <italic>C</italic>. <italic>elegans</italic> worms on defined media, which contains 3.5mM Tris.HCl, 2mM Tris, 34mM NaCl, and 3.1g/L of ether-extracted peptone. Eggs or L4 hermaphrodite animals were placed on cholesterol-depleted plates and the resulting adult animals were scored for M lineage phenotypes.</p><p>The following mutations and integrated transgenes were used: Linkage group I (LG I): <italic>drag-1(jj4)</italic>, <italic>arIs37(secreted CC</italic>::<italic>gfp)</italic>, <italic>bre-4(ok3167)</italic>, <italic>bre-5(ye17);</italic> LG II: <italic>sma-6(e1482)</italic>, <italic>pod-2(ye60)</italic>, <italic>cgt-3(ok2877)/mIn1[mIs14 dpy-10(e128)]</italic>, <italic>jjIs2437[CXTim50</italic>.<italic>19[pCXT51(5*RLR</italic>::<italic>deleted pes-10p</italic>::<italic>gfp) + LiuFD61(mec-7p</italic>::<italic>rfp)]</italic>, <italic>sptl-1(ok1693)</italic>; LG III: <italic>daf-4(m63)</italic>, <italic>daf-7(m62)</italic>, <italic>sma-2(e502)</italic>, <italic>sma-3(e491)</italic>, <italic>sma-4(e729)</italic>, <italic>lon-1(e185)</italic>, <italic>cup-5(ar465)</italic>, <italic>ina-1(gm144)</italic>, <italic>ina-1(gm39)</italic>, <italic>bre-2(ye31)</italic>, <italic>bre-3(ye26)</italic>, <italic>bre-3(ye28)</italic>, <italic>lin-12(n676n930ts)</italic>, <italic>hT2[qIs48]</italic>, <italic>ccIs4438[intrinsic CC</italic>::<italic>gfp];</italic> LG IV: <italic>daf-1(m40)</italic>, <italic>daf-1(m213)</italic>, <italic>fat-2(wa17)</italic>, <italic>fat-3(wa22)</italic>, <italic>fat-6(tm331)</italic>, <italic>tsp-12(ok239)</italic>, <italic>nT1[qIs51]</italic>; LG V: <italic>dbl-1(wk70)</italic>, <italic>fat-7(wa36)</italic>, <italic>sma-10(wk89)</italic>, <italic>sma-10(ok2224)</italic>, <italic>sma-1(ru18)</italic>, <italic>lon-3(ct417)</italic>, <italic>crm-1(tm2218)</italic>, <italic>him-5(e1467)</italic>, <italic>bre-1(ye4)</italic>, <italic>bre-5(ye17)</italic>, <italic>cgt-1(ok1045)</italic>, <italic>acs-1(gk3066)V/nT1[qIs51]IV;V;</italic> LG X: <italic>lon-2(e678)</italic>, <italic>tsp-21(tm6269)</italic>, <italic>sma-9(cc604)</italic>, <italic>jjIs2433[RAD-SMAD</italic>: <italic>CXTim50</italic>.<italic>1[pCXT51(5*RLR</italic>::<italic>deleted pes-10p</italic>::<italic>gfp) + LiuFD61(mec-7p</italic>::<italic>rfp)</italic>]].</p><p>
<italic>tsp-21</italic> and <italic>sma-9</italic> are located 0.79 map unit apart from each other on the X chromosome. We therefore separated the <italic>tsp-21(jj77)</italic> mutation from <italic>sma-9(cc604)</italic> via recombination. Specifically, progeny from <italic>tsp-21(jj77) sma-9(cc604)/+ +</italic> heterozygous parents were scored for the number of CCs. Animals with 6 CCs (<italic>jj77 cc604/+ +</italic> or <italic>+ +/+ +</italic> or <italic>jj77 cc604/jj77 +</italic>) were genotyped for <italic>jj77</italic> homozygosity by PCR. <italic>jj77 cc604/jj77 +</italic> animals were selected and their progeny were further genotyped by sequencing the <italic>sma-9</italic> gene in order to obtain <italic>jj77 +/jj77 +</italic> animals. Four independent recombinants were obtained, #570, #778, #898 and #954. Each recombinant was then outcrossed with N2 three more times before further phenotypic analysis. All four recombinants behaved similarly regarding body size, RAD-SMAD and male tail patterning phenotypes.</p><p>The <italic>lon-2(e678) tsp-21(jj77)</italic> double mutant was generated from a <italic>lon-2(e678) egl-15(n484)/tsp-21(jj77)</italic> heterozygous worms by identifying Lon-non-Egl recombinants, and scoring for the presence of the <italic>tsp-21(jj77)</italic> and the <italic>lon-2(e678)</italic> mutations by PCR genotyping.</p><p>
<italic>let-381(RNAi)</italic> was performed via feeding following the protocol described in [<xref ref-type="bibr" rid="pgen.1005221.ref033">33</xref>]. Other RNAi experiments were performed by injection. In general, gene specific fragments were amplified using RNAi clones from the Ahringer library [<xref ref-type="bibr" rid="pgen.1005221.ref081">81</xref>] or the Vidal library [<xref ref-type="bibr" rid="pgen.1005221.ref082">82</xref>], or using N2 genomic DNA as template. dsRNAs were generated using the T7 Ribomax RNA Production System (Promega) and injected into gravid adult hermaphrodite animals of specific genotypes carrying <italic>CC</italic>::<italic>gfp</italic>. The resulting progeny were scored at the adult stage for the number of CCs.</p><p>
<italic>arIs37(secreted CC</italic>::<italic>gfp)</italic> I<italic>; cup-5(ar465)</italic> III<italic>; sma-9(cc604)</italic> X animals lacking M-derived coelomocytes (having a total of 4 CCs) were treated with 50 mM ethyl methanesulfonate (EMS). Individual F1 animals were picked to 3F1s per plate and their combined F2 progeny were screened for the restoration of M lineage-derived coelomocytes (having a total of 5&#8211;6 CCs) by direct visual examination using a fluorescence stereomicroscope. Plates that segregated 5&#8211;25% of animals with 6 CCs were kept for further analysis, including determining whether the mutations bred true, the degree of suppression for each suppressor mutation when homozygous and whether the mutations are dominant or recessive.</p><p>By screening through 5,300 haploid genomes using the above method, we isolated 37 true-breeding <italic>sma-9</italic> suppressors, named <italic>susm</italic> (<underline>su</underline>ppressor of <underline><italic>sm</italic></underline>
<italic>a-9</italic>) mutations (<italic>jj49-jj85</italic>, <xref ref-type="table" rid="pgen.1005221.t002">Table 2</xref>). Four of these, <italic>jj68</italic>, <italic>jj80</italic>, <italic>jj81</italic> and <italic>jj84</italic> showed a relatively low degree of suppression (near 30%, <xref ref-type="table" rid="pgen.1005221.t002">Table 2</xref>), and were not further characterized in this work. <italic>jj58</italic> might be a dominant mutation and was not further analyzed. All of the remaining <italic>susm</italic> alleles appear to be recessive, single locus mutations, although some suppressors exhibited partial dominance in their Susm phenotype (Tables <xref ref-type="table" rid="pgen.1005221.t001">1</xref> and <xref ref-type="table" rid="pgen.1005221.t002">2</xref>). The suppressor mutations were then mapped to chromosome X or chromosome III based on their linkage to <italic>sma-9(cc604)</italic> X or to <italic>cup-5(ar465)</italic> III. Further complementation tests were carried out between each suppressor mutation and mutations in each known members of the Sma/Mab pathway, and between different suppressor mutations that did not affect known genes in the Sma/Mab pathway.</p><p>LW0214, which has <italic>arIs37(secreted CC</italic>::<italic>gfp)</italic> I and <italic>sma-9(cc604)</italic> X introgressed into the CB4856 Hawaiian strain by 6x backcrossing, was used for mapping the <italic>sma-9</italic> suppressors via snip-SNP mapping [<xref ref-type="bibr" rid="pgen.1005221.ref083">83</xref>] and whole genome sequencing (WGS) [<xref ref-type="bibr" rid="pgen.1005221.ref084">84</xref>]. LW0214 was tested using a panel of SNP markers and subsequently by WGS, and found to contain CB4856 SNPs for all six chromosomes except for the following regions that still contain N2 SNP markers: chromosome I&#8212;from the left end to -12 and from +24 to the right end; chromosome II&#8212;from the left end to -18; and chromosome X&#8212;between +1.73 and +11. Snip-SNP markers used were described in Wicks et al. [<xref ref-type="bibr" rid="pgen.1005221.ref083">83</xref>] and Davis et al. [<xref ref-type="bibr" rid="pgen.1005221.ref085">85</xref>].</p><p>For the <italic>sma-9</italic> suppressor mutations that appeared to affect known genes in the Sma/Mab pathway, either by complementation tests, or by whole genome sequencing (WGS, see below), their molecular lesions were identified by sequencing PCR products spanning the entire genomic regions of the corresponding genes, which include <italic>dbl-1</italic>, <italic>daf-4</italic>, <italic>sma-6</italic>, <italic>sma-2</italic>, <italic>sma-3</italic>, <italic>sma-4</italic>, <italic>lon-1</italic>, <italic>sma-10 and unc-40</italic>. For <italic>jj69</italic> that contains a single base pair change in the upstream regulatory region of <italic>sma-6</italic>, a plasmid pJKL1060, which contains 3kb of upstream sequences, the genomic coding region and 2kb of downstream sequences of <italic>sma-6</italic>, was used to rescue the Susm phenotype of <italic>jj69</italic>.</p><p>Direct WGS of the homozygous suppressor mutant DNA was performed for some <italic>sma-9</italic> suppressors. For others, the suppressors were simultaneously mapped and identified using the SNP-WGS method of Doitsidou <italic>et al</italic>. [<xref ref-type="bibr" rid="pgen.1005221.ref084">84</xref>]. For the SNP-WGS method, each <italic>sma-9; suppressor</italic> mutant was crossed with LW0214, which has <italic>sma-9</italic> introgressed into the polymorphic Hawaiian strain CB4856 (described above). Between 36 and 59 F2 progeny that were homozygous for both <italic>sma-9</italic> and the suppressor mutation were collected. F3 generation worms from these F2 progeny were pooled for DNA extraction and library construction. Worm genomic DNA was prepared using the Qiagen Gentra Puregene Kit. 5&#956;g of genomic DNA was used to prepare the sequencing library using the NEBNext DNA Sample Prep Master Mix Set 1. Single-end 50bp short-read (51 cycle) sequencing was performed on the HiSeq 2000 instrument (Illumina), yielding 38 ~ 78 million reads (20 ~ 41 fold coverage) per sample.</p><p>For direct WGS (<italic>jj58</italic>, <italic>jj60</italic>, <italic>jj61</italic>, <italic>jj71</italic>, <italic>jj77</italic>), data analysis was done using the MAQGene platform [<xref ref-type="bibr" rid="pgen.1005221.ref086">86</xref>, <xref ref-type="bibr" rid="pgen.1005221.ref087">87</xref>] with the default setting. SNP variants on the X chromosome compared to the reference <italic>C</italic>. <italic>elegans</italic> genome ce6 W221 were analyzed. Genes with missense SNP variants in <italic>jj60</italic> and <italic>jj77</italic>, but not in <italic>jj58</italic> and <italic>jj71</italic>, were among the candidate genes that were targeted by RNAi for their ability to suppress the <italic>sma-9(cc604)</italic> M lineage defects by injection. These included C41A3.1, K09C4.8 and C17G1.8. Further PCR and sequencing confirmed the <italic>jj60</italic> and <italic>jj77</italic> mutations in C17G1.8 (<italic>tsp-21</italic>).</p><p>For mapping additional suppressors using either direct WGS (for <italic>jj2</italic>, <italic>jj5</italic>, <italic>jj7</italic>, <italic>jj50</italic>, <italic>jj52</italic> and <italic>jj70</italic>) or SNP-WGS (for <italic>jj49</italic>, <italic>jj57</italic>, <italic>jj62</italic>, <italic>jj69</italic>, <italic>jj71</italic>, <italic>jj73</italic>, <italic>jj78</italic> and <italic>jj83</italic>), sequence data were aligned to <italic>C</italic>. <italic>elegans</italic> reference genome version WS220 using BFAST [<xref ref-type="bibr" rid="pgen.1005221.ref088">88</xref>] with default parameters. SNP calling was performed by SAMTOOLS [<xref ref-type="bibr" rid="pgen.1005221.ref089">89</xref>]. A valid SNP call required a minimum read depth of three. ANNOVAR [<xref ref-type="bibr" rid="pgen.1005221.ref090">90</xref>] was used for annotation of SNP coding potential. For SNP-WGS, Hawaiian SNPs were annotated with a custom Perl script. Scatter plots of heterozygous (0.2&#8211;0.7 fraction of total reads) Hawaiian SNPs were generated as chromosome position vs. fractional total graphs. Mapping intervals were defined by visual inspection for gaps (i.e., Hawaiian SNP fraction &lt;0.2). Candidate suppressor genes were identified as homozygous (fraction &gt;0.8), non-Hawaiian, nonsynonymous SNPs in the mapped interval. The SNPs in the identified suppressor genes were verified by PCR and sequencing.</p><p>
<italic>jj61</italic> was mapped via SNP-WGS to the region on the X chromosome where <italic>lon-2</italic> is located. Direct inspection of the sequence reads around the <italic>lon-2</italic> region showed that <italic>jj61</italic> contains a large deletion (11.8kb) spanning the <italic>lon-2</italic> region, which was subsequently verified by PCR and sequencing.</p><p>
<underline><italic>sma-6</italic> reporter and rescuing constructs</underline>
</p><p>pJKL840: <italic>sma-6p</italic>::<italic>nls</italic>::<italic>rfp</italic>::<italic>lacZ</italic>::<italic>unc-54 3&#8217;UTR</italic>
</p><p>pJKL1048: <italic>sma-6(jj69)p</italic>::<italic>nls</italic>::<italic>rfp</italic>::<italic>lacZ</italic>::<italic>unc-54 3&#8217;UTR</italic>
</p><p>pJKL1060: <italic>sma-6p</italic>::<italic>sma-6</italic> rescuing construct</p><p>
<underline><italic>tsp-21</italic> reporter constructs</underline>
</p><p>pJKL1005: <italic>5kb tsp-21p</italic>::<italic>tsp-21 genomic ORF</italic>::<italic>1</italic>.<italic>7kb tsp-21 3&#8217;UTR</italic>
</p><p>pJKL1004: <italic>5kb tsp-21p</italic>::<italic>tsp-21 genomic ORF</italic>::<italic>gfp</italic>::<italic>1</italic>.<italic>7kb tsp-21 3&#8217;UTR</italic>
</p><p>pJKL998: <italic>5kb tsp-21p</italic>::<italic>nls</italic>::<italic>gfp</italic>::<italic>lacZ</italic>::<italic>unc-54 3&#8217;UTR</italic>
</p><p>pZL11: <italic>5kb tsp-21p</italic>::<italic>tsp-21 genomic ORF (sgRNA target site modified)</italic>::<italic>gfp</italic>::<italic>1</italic>.<italic>7kb tsp-21 3&#8217;UTR</italic>
</p><p>
<underline>Constructs for tissue-specific expression of <italic>tsp-21</italic></underline>
</p><p>pJKL1015: <italic>tsp-21p</italic>::<italic>tsp-21 cDNA</italic>::<italic>tsp-21 3&#8217;UTR</italic>
</p><p>pJKL1017: <italic>rol-6p</italic>::<italic>tsp-21 cDNA</italic>::<italic>tsp-21 3&#8217;UTR</italic>
</p><p>pJKL1018: <italic>elt-3p</italic>::<italic>tsp-21 cDNA</italic>::<italic>tsp-21 3&#8217;UTR</italic>
</p><p>pJKL1019: <italic>elt-2p</italic>::<italic>tsp-21 cDNA</italic>::<italic>tsp-21 3&#8217;UTR</italic>
</p><p>pJKL1020: <italic>hlh-8p</italic>::<italic>tsp-21 cDNA</italic>::<italic>tsp-21 3&#8217;UTR</italic>
</p><p>pJKL1021: <italic>myo-2p</italic>::<italic>tsp-21 cDNA</italic>::<italic>tsp-21 3&#8217;UTR</italic>
</p><p>The full-length <italic>tsp-21</italic> cDNA clone <italic>yk1449c02</italic>, which contains a SL1 trans-splice leader sequence, and full length 5&#8217; and 3&#8217; UTRs, was kindly provided by Dr. Yuji Kohara (National Institute of Genetics, Japan). A point mutation in the coding region of <italic>tsp-21</italic> in <italic>yk1449c02</italic> was corrected by site-directed mutagenesis to generate pJKL994. Transgenic animals were generated using the plasmid pRF4 or pJKL449 (<italic>myo-2p</italic>::<italic>gfp</italic>::<italic>unc-54 3&#8217;UTR</italic>) as markers. Integrated transgenic lines carrying pJKL1004[TSP-21::GFP] (<italic>jjIs3113</italic> and <italic>jjIs3114</italic>) were generated using gamma-irradiation. pJKL840[<italic>sma-6p</italic>::<italic>nls</italic>::<italic>rfp</italic>::<italic>lacZ</italic>::<italic>unc-54 3&#8217;UTR</italic>] was used for co-localization of TSP-21::GFP and <italic>sma-6p</italic>::<italic>nls</italic>::<italic>rfp</italic>. pTAA1[<italic>hlh-8p</italic>::<italic>cgt-3</italic>.<italic>1a ORF</italic>::<italic>unc-54 3&#8217;UTR</italic>] was used to test for function of <italic>cgt-3</italic> in the M lineage.</p><p>The following mix of plasmid DNAs was injected into the N2 gravid adults: (1) a Cas9 expression plasmid pDD162 [<xref ref-type="bibr" rid="pgen.1005221.ref091">91</xref>], (2) a <italic>tsp-21</italic>-specific sgRNA plasmid pZL10, which has GAAACTGACACGGTAGAAGA<underline>TGG</underline> replacing the <italic>unc-119</italic> sgRNA in plasmid 46169 [<xref ref-type="bibr" rid="pgen.1005221.ref092">92</xref>], (3) the homologous repair template pZL11: <italic>5kb tsp-21p</italic>::<italic>tsp-21 genomic ORF (sgRNA target site modified)</italic>::<italic>gfp</italic>::<italic>1</italic>.<italic>7kb tsp-21 3&#8217;UTR</italic>, (4) a co-injection marker pCFJ90[<italic>myo-2p</italic>::<italic>mCherry]</italic> [<xref ref-type="bibr" rid="pgen.1005221.ref093">93</xref>]. GFP knock-in events were screened via PCR using a primer in GFP (ZL21: CGCATATCTTGGACGCCTAATTTG) and a primer in the <italic>tsp-21</italic> 3&#8217; region outside of the sequences included in pZL11 (ZL22: TCCACACAATCTGCCCTTTCG). Single worm PCR of 250 F1s failed to detect any germline integration event. However, we checked the F2 generation for high transmission efficiency lines (<italic>myo-2</italic>::<italic>mCherry</italic> positive) and screened via PCR 5&#8211;10 F3 progeny from each of the three high transmission efficiency lines (&gt;50%). One of the three transgenic lines gave us two homozygous GFP knock-in strains: LW3670: <italic>jj93(tsp-21</italic>::<italic>gfp)</italic> and LW3671: <italic>jj94(tsp-21</italic>::<italic>gfp)</italic>.</p><p>Total RNA was isolated from mixed-stage N2 or <italic>sma-6(jj69)</italic> worms using TRIzol Reagent (Invitrogen). Reverse transcription was performed with SuperScript III First-Strand Synthesis System (Invitrogen) following the manufacturer&#8217;s instructions. The primers used to detect the cDNAs of <italic>sma-6</italic> and <italic>act-1</italic> are: <italic>sma-6</italic>, MLF-34 and MLF-44; <italic>act-1</italic>, NMA-163 and NMA-164.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Body size measurement and RAD-SMAD reporter assay were carried out as described in Tian et al. [<xref ref-type="bibr" rid="pgen.1005221.ref022">22</xref>]. Dauer formation assay was carried out as described in Tian et al. [<xref ref-type="bibr" rid="pgen.1005221.ref021">21</xref>]. Statistical analyses were performed using Microsoft Excel and GraphPad Prism (<ext-link ext-link-type="uri" ns0:href="http://www.graphpad.com/scientific-software/prism/">http://www.graphpad.com/scientific-software/prism/</ext-link>).</p><p>GFP and RFP epifluorescence in transgenic animals was visualized either on a Leica DMRA2 compound microscope, where the images were captured by a Hamamatsu Orca-ER camera using the OPENLAB software, or on a Zeiss LSM 710 confocal microscope. Subsequent image analysis was performed using ImageJ and Photoshop CC.</p><p>We identified tetraspanin homologs by running hmmsearch from HMMER 3.1b1 [<xref ref-type="bibr" rid="pgen.1005221.ref094">94</xref>] with the hidden Markov model (HMM) profile for tetraspanins (PF00335.15) from PFAM 27.0 [<xref ref-type="bibr" rid="pgen.1005221.ref095">95</xref>] against the reference proteome set of the Quest for Orthologs consortium ([<xref ref-type="bibr" rid="pgen.1005221.ref096">96</xref>]; source URL, <italic>ftp</italic>:<italic>/ftp</italic>.<italic>ebi</italic>.<italic>ac</italic>.<italic>uk/pub/databases/reference_proteomes/QfO/QfO_release_2014_04</italic>.<italic>tar</italic>.<italic>gz</italic>). hmmsearch was run with the arguments '<italic>-E 1e-06&#8212;domE 1e-06&#8212;incE 1e-06&#8212;incdomE 1e-06-A [alignment]</italic>', which generated aligned regions of similarity to these core domains. Since the regions of homology were extracted from full-length proteins with an HMM, the specific residues extracted were generally a subset of the full protein; moreover, it was possible for two or more such regions to be independently extracted from a single protein chain, although this proved rare for tetraspanins.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">These regions were then realigned with MAFFT v7.158b [<xref ref-type="bibr" rid="pgen.1005221.ref097">97</xref>] in L-INS-i, its slowest and most reliable mode, using the arguments '<italic>&#8212;localpair&#8212;maxiterate 1000</italic>'. The resulting alignments were purged of poorly aligned members by first running trimal v1.4.rev15 [<xref ref-type="bibr" rid="pgen.1005221.ref098">98</xref>] using the argument '<italic>-gt 0</italic>.<italic>5</italic>', and then running BMGE 1.1 [<xref ref-type="bibr" rid="pgen.1005221.ref099">99</xref>] using the arguments '<italic>-t AA-h 1-g 0</italic>.<italic>5</italic>:<italic>1</italic>'. This purged the alignments of any columns in which over 50% of the columns' positions consisted of gaps rather than amino acid residues, and then any sequences in which over 50% of the residues were gapped, yielding global alignments that lacked excessive loops and gaps. From the filtered alignments, we computed protein maximum-likelihood phylogenies, with a WAG model of amino acid evolution [<xref ref-type="bibr" rid="pgen.1005221.ref100">100</xref>] and with pseudocounts for gaps, via FastTree 2.1.7 [<xref ref-type="bibr" rid="pgen.1005221.ref101">101</xref>], using the arguments '<italic>-pseudo-wag</italic>'. Confidence values for the branches of trees (ranging from 0.00 to 1.00) were automatically computed by FastTree with 1,000 internal replicates. We visualized the resulting trees with FigTree 1.4.2 (<italic><ext-link ext-link-type="uri" ns0:href="http://tree.bio.ed.ac.uk/software/figtree">http://tree.bio.ed.ac.uk/software/figtree</ext-link></italic>). Branch lengths were measured in average substitutions (when comparing full sequences or their profiles) among non-gap positions in the aligned sequences, with distances derived from the BLOSUM45 matrix, a correction for multiple substitutions, and an allowed maximum of 3.0 substitutions per individual site [<xref ref-type="bibr" rid="pgen.1005221.ref102">102</xref>].</p><p>The split-ubiquitin yeast two-hybrid experiments were carried out following the detailed method described in Grefen et al. [<xref ref-type="bibr" rid="pgen.1005221.ref103">103</xref>]. The bait CubPLV and prey NubG constructs were generated via PCR and recombinational in vivo cloning in yeast [<xref ref-type="bibr" rid="pgen.1005221.ref103">103</xref>]. The resulting fusion constructs were recovered from yeast and transformed into <italic>E</italic>. <italic>coli</italic> and confirmed by sequencing. The primers, cDNA templates, and the names of the resulting bait and prey constructs are summarized in <xref ref-type="supplementary-material" rid="pgen.1005221.s005">S2 Table</xref>. The bait and prey constructs were transformed into the haploid yeast strains THY.AP4 (MATa) and THY.AP5 (MAT&#945;), respectively, and the resulting yeast strains were mated to generate diploid yeast cells carrying specific combinations of bait and prey constructs [<xref ref-type="bibr" rid="pgen.1005221.ref103">103</xref>]. Interactions among each pair of bait and prey constructs were visualized by streaking diploid cells on SC-Trp,-Leu,-Ade,-His,-Ura,-Met plates that were supplemented with four different concentrations of methionine: 0mM, 0.075mM, 0.150mM and 0.300mM, respectively. Methionine can repress the expression of the CubPLV fusion, which is under the control of the Met-repressible <italic>MET25</italic> promoter [<xref ref-type="bibr" rid="pgen.1005221.ref042">42</xref>]. Growth was monitored for 2&#8211;9 days at 30&#176;C.</p><p>The plasmids KAT-1-Cub-PLV, NubG-KAT-1 (in pNX33 vector) and KAT-1-NubG (in pXN21 vector) [<xref ref-type="bibr" rid="pgen.1005221.ref042">42</xref>], HMT-1-Cub-PLV and HMT-1-NubG (in pXN21 vector) [<xref ref-type="bibr" rid="pgen.1005221.ref046">46</xref>] were kindly provided by Sungjin Kim (Cornell University) and used as specificity controls. NubG fusions for PAR-4, a protein unexpected to interact with any of the proteins tested, was included as another control for specificity of the interactions. The empty NubG vector was used as a control to determine if any Cub-PLV fusions can auto-activate the reporters. The vector expressing soluble wild-type Nub (NubWT) was used as a control to indicate expression of the Cub-PLV fusion. Additional confirmation of expression of each fusion protein came from western blot analysis using rabbit polyclonal anti-VP16 antibodies (ab4808, Abcam, for CubPLV fusions) and monoclonal anti-HA antibodies (Clone 12CA5, Sigma, for NubG fusions).</p>