<p><italic>Pseudocercospora fijiensis</italic> isolate CIRAD86 (<italic>mat1-1</italic> mating type, originating from Cameroon in 1988) was chosen for sequencing because it is the epitype for the species, has been the subject of intensive analyses previously and is one parent of an existing mapping population [<xref ref-type="bibr" rid="pgen.1005876.ref081">81</xref>]. CIRAD139a (<italic>mat1-2</italic>, originating from Colombia in 1990) was used for resequencing. CIRAD86 is maintained at the CBS-KNAW Fungal Biodiversity Centre (CBS 120258).</p><p>Mycelia for DNA extraction were grown in 1L Erlenmeyer flasks containing 200 mL of PDB (potato dextrose broth; Becton Dickinson, NJ, USA) shaken at 120 rpm at 28&#176;C. Mycelial mats produced during culture were filtered to remove the broth and lyophilized. Samples containing 50 mg of lyophilized mycelia were placed in 2 mL tubes and ground with a Hybaid Ribolyser (model n&#176; FP120HY-230) for 10 s at 2500 rpm with a tungsten-carbide bead. DNA was extracted from the ground mycelia using the Wizard Magnetic DNA Purification system (Promega, Netherlands) for food according to instructions provided by the manufacturer.</p><p>Whole-genome shotgun sequencing and assembly of the <italic>P</italic>. <italic>fijiensis</italic> genome were done using Sanger sequencing of three different-sized libraries (3- and 8-kb plasmids, and 40-kb fosmids) as described previously for <italic>Z</italic>. <italic>tritici</italic> [<xref ref-type="bibr" rid="pgen.1005876.ref017">17</xref>] and other species [<xref ref-type="bibr" rid="pgen.1005876.ref082">82</xref>]. The initial version 1 assembly was improved by aligning the physical scaffolds to a genetic linkage map constructed using Joinmap V 4.0 software [<xref ref-type="bibr" rid="pgen.1005876.ref083">83</xref>] to analyze the segregation data for 322 markers that were scored on 135 progeny of the cross between isolates CIRAD86 and CIRAD 139A [<xref ref-type="bibr" rid="pgen.1005876.ref081">81</xref>]. For each molecular locus, a goodness-of-fit analysis was performed to test for deviation from the expected 1:1 segregation ratio at a 1% significance level. Linkage groups were established using a minimum LOD score of 9.0 and final mapping was achieved by combining two or more linkage groups belonging to the same chromosome. The order of the markers on each chromosome was determined using a minimum LOD score of 1.0, recombination threshold of 0.4, jump of 5.0, ripple value of 1 (default) and Haldane&#8217;s mapping function as parameters. In cases of uncertainty, some markers were removed and the order was recalculated until a more stable order was achieved.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Three methods were used to identify the <italic>P</italic>. <italic>fijiensis</italic> repetitive sequences. Repeated sequences in the genome were identified <italic>de novo</italic> using RECON [<xref ref-type="bibr" rid="pgen.1005876.ref084">84</xref>] and the k-mer based method RepeatScout [<xref ref-type="bibr" rid="pgen.1005876.ref085">85</xref>]. A custom set of repeats and the RepBase Update library of 234 fungal repeats [<xref ref-type="bibr" rid="pgen.1005876.ref086">86</xref>] were then used to mask the <italic>P</italic>. <italic>fijiensis</italic> genome using RepeatMasker (<ext-link ext-link-type="uri" ns0:href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</ext-link>) [<xref ref-type="bibr" rid="pgen.1005876.ref087">87</xref>].</p><p>Repeat families with 10 or more elements identified by RepeatScout were annotated and classified into categories based on the presence of protein domains (BLAST [<xref ref-type="bibr" rid="pgen.1005876.ref088">88</xref>]). Structural features including Long Terminal Repeats (LTRs) and Terminal Inverted Repeats (TIRs) were verified using the EMBOSS [<xref ref-type="bibr" rid="pgen.1005876.ref089">89</xref>] software package. Sequences with no known proteins or structural features were grouped into the unclassified category.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Identification and annotation of protein-coding genes were performed using the JGI Annotation Pipeline, which takes multiple inputs (scaffolds, ESTs, and known genes), runs several analytical tools for gene prediction and annotation, and deposits the results in the JGI fungal genome portal MycoCosm (<ext-link ext-link-type="uri" ns0:href="http://jgi.doe.gov/fungi">http://jgi.doe.gov/fungi</ext-link>) [<xref ref-type="bibr" rid="pgen.1005876.ref090">90</xref>] for further analysis and manual curation.</p><p>Several gene-prediction programs falling into three general categories were used to annotate the repeat-masked assembly as described by Ohm et al. [<xref ref-type="bibr" rid="pgen.1005876.ref019">19</xref>]. The resulting set of putative genes was then filtered for the best models based on EST and similarity support to produce a non-redundant representative set. This representative set of filtered gene models from the automated annotation pipeline was subject to further analysis and manual curation as described by Goodwin et al. for <italic>Z</italic>. <italic>tritici</italic> [<xref ref-type="bibr" rid="pgen.1005876.ref017">17</xref>] and by Ohm et al. [<xref ref-type="bibr" rid="pgen.1005876.ref082">82</xref>] for more recently sequenced species. Measures of model quality included proportions of the models complete with start and stop codons (88% of models), those that were consistent with ESTs (30% of models) and those supported by similarity with proteins from the NCBI NR database (74% of models) as summarized in <xref ref-type="supplementary-material" rid="pgen.1005876.s018">S8 Table</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Functional annotations for all predicted gene models were made using SignalP [<xref ref-type="bibr" rid="pgen.1005876.ref091">91</xref>], TMHMM [<xref ref-type="bibr" rid="pgen.1005876.ref092">92</xref>], InterProScan [<xref ref-type="bibr" rid="pgen.1005876.ref093">93</xref>], and BLASTp [<xref ref-type="bibr" rid="pgen.1005876.ref088">88</xref>] against the nr, SwissProt (<ext-link ext-link-type="uri" ns0:href="http://www.expasy.org/sprot/">http://www.expasy.org/sprot/</ext-link>), KEGG [<xref ref-type="bibr" rid="pgen.1005876.ref094">94</xref>] and KOG [<xref ref-type="bibr" rid="pgen.1005876.ref095">95</xref>] databases as described by Ohm et al. [<xref ref-type="bibr" rid="pgen.1005876.ref019">19</xref>]. Multigene families were predicted with the Markov clustering algorithm (MCL) [<xref ref-type="bibr" rid="pgen.1005876.ref096">96</xref>] to cluster the proteins using BLASTp alignment scores between proteins as a similarity metric. Functional annotations are summarized in <xref ref-type="supplementary-material" rid="pgen.1005876.s019">S9 Table</xref>. Manual curation of the automated annotations was performed using the web-based interactive editing tools of the JGI Genome Portal to assess predicted gene structures, assign gene functions, and report supporting evidence. Gene models predicted by the JGI annotation pipeline were also analyzed using the program Blast2GO [<xref ref-type="bibr" rid="pgen.1005876.ref097">97</xref>] with an E-value of &lt; 10<sup>&#8722;6</sup>. Blast2GO assigns GO terms based on the BLAST definitions. Comparisons between groups of genes for enrichment of GO terms were done by using Fisher&#8217;s exact test implemented in the Blast2GO program.</p><p>Potential secreted proteins were identified with a python script made to run all gene models through SignalP 3.0 [<xref ref-type="bibr" rid="pgen.1005876.ref098">98</xref>] and subsequently filtered for proteins that had no transmembrane domains, no signal anchor motifs, were fewer than 300 amino acids in length and had at least four cysteine residues. The gene models that fulfilled these criteria were considered as potential SSPs.</p><p>For re-sequencing of isolate CIRAD 139A, a paired-end library was made using the standard Illumina library prep protocol with NEB reagents. Average insert size of the library was 272 base pairs. Sequencing was done on an Illumina GAIIx in one lane of a 54-cycle paired-end run using 36-cycle version 5 SBS Kits. The flow cell was built using a version 4 paired-end cluster generation kit. Eventually, 37 million reads were obtained for a total yield of 4 gigabases. Paired reads were aligned to the <italic>P</italic>. <italic>fijiensis</italic> v2 Assembly reference scaffolds using GSNAP (2010-03-09 release), allowing up to 3 mismatches or 1 indel and with end trimming enabled. Uniquely aligned reads were then used to call variant sites using the Alpheus pipeline, requiring that a variant have support from at least two reads with an average quality of bases of at least phred 10 and at least 80% of the reads covering the site calling that variant. Nonsynonymous SNP differences were assessed against the coding regions in the <italic>P</italic>. <italic>fijiensis</italic> v2 Frozen Gene Catalog 20100402.</p><p>To survey the non-synonymous SNPs in the annotated protein set of CIRAD86, a simple analysis of functional bias in variant proteins was conducted using a ranking comparison approach. All genes were ranked based on their non-synonymous SNP count (normalized for coding sequence length) and two selected gene sets were compared with the whole-genome set. The two sets used for comparison were the 1500 most-variant proteins (Set V) and the 1500 least-variant proteins (Set C), taken from the list of genes ranked by non-synonymous SNP count. The annotation files for the <italic>P</italic>. <italic>fijiensis</italic> v2 Frozen Gene Catalog 20100402 were used as the source of GO terms for the genes. The ranked frequency of occurrence of GO terms in the gene annotations for the whole genome was compared with those for Sets C and V.</p><p>For each repetitive element family, a subset of elements with lengths within 50% of the longest element was aligned using clustalX [<xref ref-type="bibr" rid="pgen.1005876.ref099">99</xref>]. These alignments were submitted to RIPCAL [<xref ref-type="bibr" rid="pgen.1005876.ref028">28</xref>] to determine the dinucleotide bias observed in repetitive elements. RIPCAL estimates 'RIP dominance' for each dinucleotide containing a cytosine. It is the ratio of a given dinucleotide (e.g., CA) to the sum of the other three dinucleotides (CG/CC/CT).</p><p>To test for isochores in the <italic>P</italic>. <italic>fijiensis</italic> genome, a contiguous stretch of sequence with an arbitrarily chosen average GC content of less than 45% was categorized as an AT block. Custom python scripts were used to calculate the percent GC across the genome, to generate AT blocks and to calculate the average percent GC across the AT blocks and those fewer than 500 bp apart were merged into blocks of at least 1 kb in length, which were retained and analyzed for their composition and distribution of repetitive sequences and genes.</p><p>To estimate and compare the amount of RIP between the AT-rich blocks and the rest of the genome, a custom python script was written using a 500-bp sliding window with a step size of 100 bp. The amount of RIP was calculated as an index (CpA+TpG)/(ApC+GpT) and estimated separately for each of the AT-rich versus AT-poor regions in the genome. The RIP index measures the depletion of the RIP targets CpA and TpG; thus, lower values of (CpA+TpG)/(ApC+GpT) are indicative of a higher degree of RIP [<xref ref-type="bibr" rid="pgen.1005876.ref028">28</xref>].</p><p>To estimate the ages of transposon insertions, LTR retrotransposons were identified and annotated using the LTRharvest [<xref ref-type="bibr" rid="pgen.1005876.ref100">100</xref>] and LTRdigest [<xref ref-type="bibr" rid="pgen.1005876.ref101">101</xref>] modules in GenomeTools [<xref ref-type="bibr" rid="pgen.1005876.ref102">102</xref>]. LTR sequences from these elements were aligned using ClustalW [<xref ref-type="bibr" rid="pgen.1005876.ref099">99</xref>] and manually curated to estimate the numbers of mutations that had accumulated over time. All transition mutations were ignored in this analysis to remove the bias caused by RIP. Age of the LTR retrotransposons was calculated using the average rate of 1.09 &#215; 10<sup>&#8722;9</sup> substitutions/site/year as proposed for fungal sequences [<xref ref-type="bibr" rid="pgen.1005876.ref103">103</xref>].</p><p>To test whether the transposons in the <italic>P</italic>. <italic>fijiensis</italic> genome were unique, a comparison was made to transposable elements (TEs) in the genome of <italic>C</italic>. <italic>fulvum</italic>, the only other sequenced fungus in the Capnodiales with an expanded genome. RepeatMasker [<xref ref-type="bibr" rid="pgen.1005876.ref085">85</xref>] was used to mask the <italic>C</italic>. <italic>fulvum</italic> genome using the repeat database from the <italic>P</italic>. <italic>fijiensis</italic> genome. The resulting file was parsed using the 80:80 rule of Wicker et al. [<xref ref-type="bibr" rid="pgen.1005876.ref027">27</xref>], i.e., 80% identity across 80% length to identify repeats in common between the two genomes. Another run was done at a 70:70 cutoff to allow for greater divergence generated by RIP.</p><p>The CIRAD86 strain was grown in three culture media for production of cDNA libraries: yeast-glucose broth as a rich medium (10 g of yeast extract and 30 g of glucose per liter); minimal nutrient medium (1 g of KH<sub>2</sub>PO<sub>4</sub>, 1 g of KNO<sub>3</sub>, 0.5 g of MgSO<sub>4</sub>&#183;7H<sub>2</sub>O, 0.5 g of KCl, 0.2 g of glucose, 0.2 g of sucrose per liter); and minimal nutrient medium without a nitrogen source (as above but without KNO<sub>3</sub>). Fungal mycelia were grown in each medium at 25&#176;C for 10 days with a photoperiod of 12 hours using cool-white fluorescent light on a rotary shaker at 100 rpm. Mycelia derived from all three <italic>in vitro</italic> conditions were harvested by filtration and ground in liquid nitrogen. The RNA was isolated by the trizol method with the RNeasy kit (Qiagen, Netherlands) with 2 g of starting material. RNA quality and quantity were assessed by spectrophotometer and by gel electrophoresis according to standard procedures. For cDNA library construction, first-strand cDNA synthesis was done using polyA+ RNA, reverse transcriptase (SuperScriptII (Invitrogen, CA, USA)) and an oligo dT-<italic>Not</italic>I primer (5' GACTAGTTCTAGATCGCGAGCGGCCGCCCT15VN 3'). Second-strand synthesis was done by <italic>E</italic>. <italic>coli</italic> DNA ligase, polymerase I and RNaseH before end repair with T4 DNA polymerase. The <italic>Sal</italic>I adaptor (5' TCGACCCACGCGTCCG and 5' CGGACGCGTGGG) was ligated to the cDNA and digested with <italic>Not</italic>I before selecting the size range by gel electrophoresis. Sizes were 0.6&#8211;2 kb and 2&#8211;10 kb. The cDNA of <italic>P</italic>. <italic>fijiensis</italic> grown in yeast-glucose medium was divided into libraries CBBT and CBHU (0.6-2kb) and CBHT (2-10kb). The cDNA from culture on minimal nutrient medium was divided into libraries CBBW and CBHX (0.6-2kb) and CBBU and CBHW (2-10kb), and for the libraries of culture on minimal nutrient medium without nitrogen source, cDNA was divided into libraries CBBX and CBHY(0.6-2kb). The size-selected inserts were cloned into the pCMVSPORT6 vector (Invitrogen) and digested with <italic>Sal</italic>I and <italic>Not</italic>I. Ligated vectors were transformed into ElectroMAX T1 DH10B cells (Invitrogen).</p><p>Sequence reads from cDNA libraries were trimmed of vector, linker, adapter, poly-A/T, and other artifact sequences with the Cross-match software. Internally developed software at the JGI-DOE identified short patterns and low-quality regions (Q15). The longest high-quality region of each read was counted as an EST. Clustering of ESTs was performed based on pairwise alignments generated using the Malign software, a modified version of the Smith&#8211;Waterman algorithm [<xref ref-type="bibr" rid="pgen.1005876.ref104">104</xref>], which was developed at the JGI for use in whole-genome shotgun assembly. ESTs sharing an alignment of at least 98% identity with 150-bp overlap were assigned to the same cluster. For each cluster of EST sequences, a consensus sequence was generated by running the Phrap software [<xref ref-type="bibr" rid="pgen.1005876.ref105">105</xref>,<xref ref-type="bibr" rid="pgen.1005876.ref106">106</xref>].</p><p>Plantlets of <italic>M</italic>. <italic>acuminata</italic> ssp. <italic>burmannicoides</italic> var. Calcutta 4 (recognized as a resistant standard for BLSD) were multiplied and rooted <italic>in vitro</italic>, whereas Cavendish &#8220;Grand Naine&#8221; tissue culture plants were hardened for three to four weeks in a greenhouse environment. Subsequently, all plantlets were grown for three months (small plants), and some plants of var. Calcutta 4 and &#8220;Grand Naine&#8221; were grown for eight months (large plants) in a controlled-environment greenhouse compartment at 25&#176;C with a relative humidity of &gt;80% and 16 hours of light per day.</p><p>Plants of tomato (<italic>Lycopersicum esculentum</italic>) cv. Moneymaker (MM), which has no known <italic>Cf</italic> resistance genes (<italic>Cf0</italic>), or an isogenic line previously transformed with the <italic>Cf4</italic> resistance gene were grown under greenhouse conditions as described by Stergiopoulos et al. [<xref ref-type="bibr" rid="pgen.1005876.ref032">32</xref>] during 3&#8211;4 weeks.</p><p>The mature protein from the <italic>P</italic>. <italic>fijiensis</italic> putative effector gene <italic>PfAvr4</italic> was produced heterologously by culturing <italic>Pichia pastoris</italic> isolate GS115 in a fermentor as described previously [<xref ref-type="bibr" rid="pgen.1005876.ref107">107</xref>]. Following production in the fermentor, the protein was further purified from excess liquid medium and smaller proteins by filtration through a 3-kDa membrane (Amicon Ultra-15 Centrifugal filter unit, Millipore, USA).</p><p>Infiltration of the complete fermentor product or purified PfAVR4 protein into banana and tomato leaves was done by injection with a 1-mL syringe with no needle. Infiltrations on banana leaves were done at the original concentration and a six-fold higher concentration. For tomato, all infiltration materials were diluted fifty or seventy-five times prior to infiltration. Samples of the fermentor medium and water were infiltrated separately as negative controls for all plants. At each infiltration point, the observed water soaking of tissue was marked with a permanent marker. Observations were recorded with an Olympus C-8080 digital camera at four and 10 days post infiltration (dpi) on banana leaves, and at 6 dpi for tomato plants. Protein preparation and controls were infiltrated in at least two banana leaves from each genotype in small and large sizes, with at least 3 repetitions per leaf. Infiltrations in tomato plants were performed in at least 4 leaves with one repetition.</p><p>The thermal denaturation method of Marmur and Doty [<xref ref-type="bibr" rid="pgen.1005876.ref108">108</xref>], performed basically as described by Smith et al. [<xref ref-type="bibr" rid="pgen.1005876.ref109">109</xref>], was used to estimate G+C contents of DNA from <italic>P</italic>. <italic>fijiensis</italic> isolate CIRAD86 (CBS120258) plus that from the closely related banana pathogens <italic>P</italic>. <italic>musae</italic> (isolate UQ430; CBS121371) and <italic>P</italic>. <italic>eumusae</italic> (isolate CBS122457) as well as the previously sequenced <italic>Z</italic>. <italic>tritici</italic> isolate IPO323 (CBS115943) [<xref ref-type="bibr" rid="pgen.1005876.ref017">17</xref>]. Genomic DNA was isolated from cultures grown in PD broth at 25&#176;C on a rotary shaker (150 rpm) following the procedure described by Raeder and Broda [<xref ref-type="bibr" rid="pgen.1005876.ref110">110</xref>] and was dissolved in 0.1X SSC. Melting curves were obtained on a Perkin-Elmer &#955;25 spectrophotometer equipped with a thermal programmer. The G+C contents were calculated from the T<sub>m</sub> values (melting/transition temperature) derived from the peaks of the first derivatives of the melting curves [<xref ref-type="bibr" rid="pgen.1005876.ref111">111</xref>]. DNA from <italic>Candida parapsilosis</italic> isolate CBS604 (T<sub>m</sub> in 0.1 x SSC, 70.6&#176;C) [<xref ref-type="bibr" rid="pgen.1005876.ref003">3</xref>] was used as a calibration control. Determinations were performed at least twice for each isolate.</p><p>Phylogenetic analysis showing the placement of species within the Capnodiales with expanded genomes was done using Internal Transcribed Spacer regions (ITS). DNA sequences were downloaded from GenBank with the following accession numbers: AF181692 for <italic>Z</italic>. <italic>tritici</italic>, EU514233 for <italic>P</italic>. <italic>eumusae</italic>, EU514265 for <italic>P</italic>. <italic>musae</italic> and EU514248 for <italic>P</italic>. <italic>fijiensis</italic> or obtained from genome data available at the Fungal Genome portal at JGI. Sequence alignment was done using MUSCLE [<xref ref-type="bibr" rid="pgen.1005876.ref112">112</xref>] and the phylogenetic tree was generated with MEGA 6.0 [<xref ref-type="bibr" rid="pgen.1005876.ref113">113</xref>] using a Maximum Likelihood statistical method and the Tamura 3-parameter substitution model. Support for the nodes of the tree was estimated by bootstrapping with 1000 replications.</p><p>Isolates of <italic>P</italic>. <italic>fijiensis</italic> grown for 3 weeks in PD broth at 28&#176;C, 150 rpm, were blended and grown for 48 hrs in the same medium at 20% strength amended with 1 &#956;M tricyclazole. Decanted culture was washed with 1 M sorbitol, and added to 40 mL of OM buffer (1.2 M MgSO4, 10 mM K phosphate, pH 5.8 with 700 mg of glucanase (Sigma, Germany), 256 mg of yatalase (Takara, Japan), 7500 U of &#946;-glucuronidase (Sigma) and 0.8 g of driselase (Sigma)) in a ratio of ~1:3 (mycelium:buffer). The enzymatic treatment was incubated at 33&#176;C and shaken at 50 rpm for 4.5 hrs.</p><p>Protoplasts were filtered through a plastic mesh of 30 &#956;m and washed 3 times with 1 M sorbitol in sterile conditions. When the concentration was at least 1 &#215; 10<sup>8</sup> per mL, protoplasts were embedded in low-melting point (SeaKem Gold) agarose at a final concentration of 0.5%. Agarose plugs were treated with proteinase K as described previously [<xref ref-type="bibr" rid="pgen.1005876.ref114">114</xref>], washed with cold 50 mM EDTA, and kept in the same solution at 4&#176;C until used.</p><p>Chromosomes of <italic>P</italic>. <italic>fijiensis</italic> were separated in a CHEF DR-II system (Bio-Rad, Netherlands). Small chromosomal bands were discriminated as described before [<xref ref-type="bibr" rid="pgen.1005876.ref057">57</xref>] using the chromosomes of <italic>Saccharomyces cerevisiae</italic> (Bio-Rad) as a high molecular weight (HMW) standard. Large chromosomes were separated in a 0.8% low-melting point (SeaKem Gold) agarose gel, with 0.5% TBE buffer at 11&#176;C, and 50 V for 195 hrs with switching times from 4800 to 1800 sec, and 24 hrs from 1800 to 1300 sec, followed by 20 hrs at 60 V from 1300 to 800 sec, and finally 27 hrs of 800 to 600 sec at 80 V. HMW standards were <italic>Schizosaccharomyces pombe</italic> and <italic>Hansenula wingei</italic> chromosomes (BioRad). Agarose gels were stained with SYBRGold (Invitrogen) and destained in water for 30 and 20 min, respectively, observed under a UV transilluminator and recorded with an Eagle Eye II (Stratagene) still video system.</p><p>Two tools, Circos [<xref ref-type="bibr" rid="pgen.1005876.ref115">115</xref>] and MUMmer [<xref ref-type="bibr" rid="pgen.1005876.ref116">116</xref>], were used for structural analysis of the <italic>P</italic>. <italic>fijiensis</italic> genome. A nucleotide-based similarity search was done between the masked <italic>P</italic>. <italic>fijiensis</italic> and <italic>Z</italic>. <italic>tritici</italic> genomes and visualized using Circos [<xref ref-type="bibr" rid="pgen.1005876.ref115">115</xref>], whereas protein comparisons between the masked genomes were done using Promer [<xref ref-type="bibr" rid="pgen.1005876.ref116">116</xref>]. Proteins with greater than 60% identity were reported.</p><p>Proteins in <italic>P</italic>. <italic>fijiensis</italic> and <italic>Z</italic>. <italic>tritici</italic> with at least 50% amino-acid identity and match length were grouped as orthologs using OrthoMCL [<xref ref-type="bibr" rid="pgen.1005876.ref117">117</xref>] and synteny blocks were determined using Orthocluster [<xref ref-type="bibr" rid="pgen.1005876.ref118">118</xref>]. The <italic>Z</italic>. <italic>tritici</italic> protein dataset also was compared to two other phylogenetically distant Dothideomycetes in the order Pleosporales, <italic>P</italic>. <italic>nodorum</italic> and <italic>P</italic>. <italic>tritici</italic>-<italic>repentis</italic>.</p><p>To analyze the frequencies of molecular marker alleles and fungicide resistance within populations, four farms in Costa Rica were sampled during 2008 (<xref ref-type="supplementary-material" rid="pgen.1005876.s010">S8 Fig</xref>). San Pablo, Zent, and Cartagena are located in Lim&#243;n province, where bananas are grown at high density on large plantations and diseases are controlled by using chemical fungicides. These farms are located in the main Costa Rica banana production area with approximate sizes of 285, 342 and 64 ha, respectively. A fourth farm, San Carlos (0.5 ha), is located in Alajuela province and is isolated geographically from the principal banana-production area. Leaf tissue was collected from ten banana plants from each farm. Ascospores were discharged from the pseudothecia onto water agar [<xref ref-type="bibr" rid="pgen.1005876.ref119">119</xref>] and single ascospores were transferred immediately to 15 x 100-mm petri dishes filled with potato dextrose agar (PDA). Between eight to ten ascospores from each sample point were placed on each dish of PDA. After 4 days colonies were transferred to Mycophil agar (Becton Dickinson Microbiology Systems, Cockeysville, MD) and incubated for 15 days at 25&#176;C under continuous fluorescent light for colony growth and conidial production. Eventually, 649 isolates were collected and analyzed for phenotypic and molecular variability.</p><p>To obtain DNA for population genetics analyses, mycelia of 190 isolates from each of the three commercial plantations and of 95 isolates from the San Carlos population were harvested and lyophilized for 24 hours. Genomic DNA was extracted using the Wizard Magnetic DNA Purification System for Food Kit (Promega, Madison, WI, USA) according to the manufacturer&#8217;s instructions and 2 &#956;L per sample were quantified using a NanoDrop ND-1000 Spectrophotometer (NanoDrop Technologies, Wilmington, DE, USA). Mating type (<italic>mat</italic>) PCR assays [<xref ref-type="bibr" rid="pgen.1005876.ref120">120</xref>] were performed in a 50-&#956;L total volume containing 50 ng of template genomic DNA, 2 mM MgCl<sub>2</sub>, 600 &#956;M dNTPs, 5 &#956;M of each primer, and 0.4 U of Taq DNA polymerase (Roche, Mannhein, Germany). Temperature cycling was carried out with the following program: 94&#176;C for 2 min, 40 cycles of 94&#176;C for 1 min, 70&#176;C for 30 s and 72&#176;C for 1 min, and a final elongation period of 10 min at 72&#176;C.</p><p>Analysis of VNTR markers was done as reported previously [<xref ref-type="bibr" rid="pgen.1005876.ref121">121</xref>]. For genotyping strobilurin resistance, primers were developed on the basis of the G143A mutation in the <italic>cytb</italic> gene [<xref ref-type="bibr" rid="pgen.1005876.ref122">122</xref>] to identify sensitive and resistant <italic>P</italic>. <italic>fijiensis</italic> field strains (<xref ref-type="supplementary-material" rid="pgen.1005876.s020">S10 Table</xref>) in 20-&#956;L aliquots containing 50 ng of template genomic DNA, 2 mM MgCl<sub>2</sub>, 600 &#956;M dNTPs, 5 &#956;M of each primer and 0.4 U of Taq DNA polymerase (Roche, Mannhein, Germany). Temperature cycling was conducted with the following program: 94&#176;C for two min, 40 cycles of 94&#176;C for one min, 70&#176;C for 30 s and 72&#176;C for one min, and a final elongation period of 10 min at 72&#176;C. Amplicons were separated by electrophoresis using 1% (for <italic>mat</italic> and <italic>cytb</italic> assays) or 3% (for VNTRs) agarose gels containing 0.3 &#956;g/mL ethidium bromide, in 0.5&#215; TBE buffer at 120 V for approximately 1 h (for <italic>mat</italic> and <italic>cytb</italic> assays) or 5 h (for VNTRs) and were visualized and photographed using a UV transilluminator and Eagle Eye II (Stratagene) still video system.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To analyze the data, frequencies of the two mating types within each population and in the overall sample were tested for deviation from a 1:1 ratio with &#967;<sup>2</sup> tests. A molecular multilocus haplotype was constructed for each isolate by combining the allelic data at all five VNTR loci. Gene diversity within each population (<italic>H</italic><sub><italic>S</italic></sub>) in total and by locus was calculated using GenAlEx 6.4 [<xref ref-type="bibr" rid="pgen.1005876.ref123">123</xref>]. Total diversity over the entire sample (<italic>H</italic><sub><italic>T</italic></sub>), mean gene diversity within populations (<italic>H</italic><sub><italic>S</italic></sub>), genetic differentiation among populations (<italic>G</italic><sub><italic>ST</italic></sub>) and the corrected, standardized measure of genetic differentiation (<italic>G&#8221;</italic><sub><italic>ST</italic></sub>) were calculated using GENODIVE Beta version 2.0 [<xref ref-type="bibr" rid="pgen.1005876.ref124">124</xref>]. In all cases, <italic>H</italic><sub><italic>T</italic></sub> and <italic>H</italic><sub><italic>S</italic></sub> refer to the unbiased estimates as developed by Nei [<xref ref-type="bibr" rid="pgen.1005876.ref125">125</xref>]. Pairwise estimates of <italic>G&#8221;</italic><sub><italic>ST</italic></sub> and of Jost&#8217;s differentiation (<italic>D</italic>) [<xref ref-type="bibr" rid="pgen.1005876.ref126">126</xref>] also were calculated with GENODIVE. Multilocus haplotype diversity was calculated with multilocus (<ext-link ext-link-type="uri" ns0:href="http://www.bio.ic.ac.uk/evolve/software/">http://www.bio.ic.ac.uk/evolve/software/</ext-link>).</p>