<p>The FIND was completed in accordance with the principles of the Declaration of Helsinki. Written informed consent was obtained from all participants. The Institutional Review Board at each participating center (Case Western Reserve University, Cleveland, OH, Harbor-University of California Los Angeles Medical Center, Johns Hopkins University, Baltimore, National Institute of Diabetes and Digestive and Kidney Diseases, Phoenix, AZ, University of California, Los Angeles, CA, University of New Mexico, Albuquerque, NM, University of Texas Health Science Center at San Antonio, San Antonio, TX, Wake Forest School of Medicine, Winston-Salem, NC) approved all procedures, and all study subjects provided written informed consent. A certificate of confidentiality was filed at the National Institutes of Health.</p><p>FIND is a multi-ancestry family study of severe DKD.[<xref ref-type="bibr" rid="pgen.1005352.ref011">11</xref>] Index cases had advanced DKD, likely to progress to ESKD based on clinical criteria, and at least one informative sibling with either DKD or long-standing DM without nephropathy. Detailed phenotype criteria for enrollment have been reported.[<xref ref-type="bibr" rid="pgen.1005352.ref026">26</xref>,<xref ref-type="bibr" rid="pgen.1005352.ref043">43</xref>,<xref ref-type="bibr" rid="pgen.1005352.ref044">44</xref>] Index cases of AA, EA, MA and AI ethnicity were included in the Discovery GWAS; all had DM duration &gt;5 years and/or DR, with UACR &gt;1 g/g or ESKD. Unrelated controls had DM duration &#8805;9 years, UACR &lt;30 mg/g (equating to overnight albumin excretion &lt;20 mcg/min), and serum creatinine &lt;1.6 mg/dl ([122 &#956;mol/L] men) or &lt;1.4 mg/dl ([107 &#956;mol/L] women). In AA, EA and MA only unrelated cases and controls were included; since AI participants were largely recruited from relatively small communities all available cases and controls meeting criteria were included, regardless of relationships.</p><p>Additional non-FIND-study DKD cases and controls (with and without DM) were genotyped to increase power (<bold><xref ref-type="supplementary-material" rid="pgen.1005352.s002">S1 Table</xref></bold>). Non-FIND samples included unrelated DKD cases and controls of self-reported African American ethnicity recruited at Wake Forest,[<xref ref-type="bibr" rid="pgen.1005352.ref045">45</xref>] Case Western Reserve [<xref ref-type="bibr" rid="pgen.1005352.ref046">46</xref>] and Howard Universities;[<xref ref-type="bibr" rid="pgen.1005352.ref047">47</xref>] unrelated cases and controls of EA ethnicity recruited at Wake Forest [<xref ref-type="bibr" rid="pgen.1005352.ref048">48</xref>] and Case Western Reserve;[<xref ref-type="bibr" rid="pgen.1005352.ref049">49</xref>,<xref ref-type="bibr" rid="pgen.1005352.ref050">50</xref>] cases and controls of AI ethnicity recruited at NIDDK-Phoenix; [<xref ref-type="bibr" rid="pgen.1005352.ref051">51</xref>] and cases and controls of MA ethnicity recruited in San Antonio and Los Angeles.[<xref ref-type="bibr" rid="pgen.1005352.ref052">52</xref>]</p><p>The FIND Large Replication (FILR) Study was comprised of samples independent from the Discovery cohorts. AA and EA replication cohorts were unrelated individuals recruited at Wake Forest, Johns Hopkins, Case Western Reserve and Harbor UCLA Universities and out-of-study control data from the Genetic Association Information Network (GAIN) consortium;[<xref ref-type="bibr" rid="pgen.1005352.ref053">53</xref>] AI replication cohorts consisted of pedigree data from NIDDK-Phoenix [<xref ref-type="bibr" rid="pgen.1005352.ref051">51</xref>] and from the Dakota and Oklahoma centers of the Strong Heart Family Study.[<xref ref-type="bibr" rid="pgen.1005352.ref054">54</xref>] Replication cases had DM duration &gt;5 years and/or DR, UACR &#8805;0.3 g/g (equating to overnight albumin excretion &gt;200 mcg/min) and/or proteinuria &gt;500 mg/day or ESKD. DM controls had an eGFR &gt;60 ml/min/1.73 m<sup>2</sup>, UACR &lt;30 mg/g after 10 year DM duration or UACR &lt;100 mg/g after 15 year DM duration. GAIN study subjects with and without DM were used as controls; no kidney function data were available for these individuals. GAIN samples were excluded for specific SNPs, if MAFs were inconsistent with those in FIND controls. Additional MA subjects were not available for inclusion in FILR.</p><p>Based on ancestry, the FIND discovery GWAS samples included: (i) AA: 1564 DKD cases (633 in FIND, 931 out of study), 369 controls with DM lacking nephropathy (277 in FIND, 92 out of study) and 1,288 non-diabetic non-nephropathy controls (all out of study); (ii) AI: 538 DKD cases, 319 controls with DM lacking nephropathy; (iii) EA: 342 DKD cases, 404 controls with DM lacking nephropathy; and (iv) MA: 779 DKD cases and 594 controls with DM lacking nephropathy. The FILR replication study included: (i) AA: 950 DKD cases, 50 controls with DM lacking nephropathy and 1,887 non-diabetic non-nephropathy controls; (ii) AI: 471 DKD cases, 340 controls with DM lacking nephropathy and 486 non-diabetic non-nephropathy controls and (iii) EA: 582 DKD cases, 205 controls with DM lacking nephropathy and 2,568 non-diabetic non-nephropathy controls. FILR samples were genotyped at loci including the top associated SNPs from the Discovery GWAS, eQTL associations, literature-based candidate gene polymorphisms and ancestry informative markers (AIMs). <xref ref-type="supplementary-material" rid="pgen.1005352.s002">S1 Table</xref> delineates the sample sources in the Discovery GWAS and FILR, stratified by ancestry.</p><p>See Supplementary Methods (<bold><xref ref-type="supplementary-material" rid="pgen.1005352.s001">S1 Text</xref></bold>).</p><p>The DNA samples that comprise the Discovery cohorts, plus an additional 244 blind duplicates were genotyped on the Affymetrix Genome-Wide Human 6.0 SNP array (see <xref ref-type="supplementary-material" rid="pgen.1005352.s001">S1 Text</xref> Supplemental Methods for details). The FILR replication samples were genotyped for 3,937 SNPs selected based on the strength of the statistical association from the Discovery GWAS. Additional SNPs were included based on the FIND eQTL association and candidate gene SNPs previously reported to be associated with DKD (see <xref ref-type="supplementary-material" rid="pgen.1005352.s001">S1 Text</xref> Supplemental Methods for details). Specifically, within each ancestry group, the SNPs with the strongest statistical evidence of association were identified; a few additional SNPs from each region with supportive but weaker evidence of association were also identified (i.e., associations due to LD but r<sup>2</sup>&lt;0.95 with the primary associated SNP). This redundancy was designed to limit the number of regions not represented in the replication study due to genotyping failure. In total, 3,019 SNPs (821 AA, 790 AI, 608 EA, and 800 MA) were genotyped for FILR based solely on statistical association with DKD within an ethnicity. The trans-ethnic meta-analysis of the discovery cohort identified another 436 SNPs nominally associated with DKD (p&lt;0.0003). In addition, 482 SNPs (121 AA, 133 AI, 122 EA, 14 MA, meta-analysis 92) were chosen with the smallest L<sub>2</sub>-norm (i.e., Euclidean distance) of the&#8211;log<sub>10</sub> (<italic>p</italic>-values) from GWAS and eQTL association analyses, provided that <italic>p</italic> &lt;0.01 from GWAS. Here, the L<sub>2</sub>-norm was defined relative to the maximum of the&#8211;log<sub>10</sub> (<italic>p</italic>-values) from the GWAS and eQTL and provides an ordering of the combined evidence for eQTL and association with DKD. SNP associations in FILR were considered &#8220;replicated&#8221; if both the association reached statistical significance and direction of the association was consistent with the Discovery analysis. Finally, 278 AIMs were genotyped to allow for adjustment of potential population substructure. Thus, FILR was designed as a replication study and not a large-scale trans-ethnic fine-mapping study. Subsequent studies will complete fine-mapping to localize associations.</p>