<p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We first consider a single (bi-allelic) SNV. Let <italic>G</italic> be the genotype (coded as the number of minor alleles) at the variant site and let <italic>D</italic> be the disease status. We denote the genotype distribution under Hardy-Weinberg equilibrium (HWE) by <italic>P</italic><sub><italic>&#960;</italic></sub>(<italic>G</italic>), where <italic>&#960;</italic> is the minor allele frequency (MAF). Note that the HWE assumption has a minimal effect for rare variants, as homozygotes of minor alleles are not expected. Instead of observing <italic>G</italic>, we observe the total number of reads mapped to the SNV and the number of reads carrying the minor allele, denoted by <italic>T</italic> and <italic>R</italic>, respectively. Similar to SAMtools, GATK, and seqEM [<xref ref-type="bibr" rid="pgen.1006040.ref015">15</xref>], we assume that <italic>R</italic> given <italic>T</italic> and <italic>G</italic> follows a binomial distribution
<disp-formula id="pgen.1006040.e001"><alternatives><graphic id="pgen.1006040.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e001.jpg" /><ns1:math id="M1"><ns1:mrow><ns1:msub><ns1:mi>P</ns1:mi><ns1:mi>&#1013;</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>R</ns1:mi><ns1:mo>|</ns1:mo><ns1:mi>T</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>G</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mfenced close="" open="{" separators=""><ns1:mtable><ns1:mtr><ns1:mtd columnalign="left"><ns1:mrow><ns1:mtext>Binomial</ns1:mtext><ns1:mo>(</ns1:mo><ns1:mi>T</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>&#1013;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mtd><ns1:mtd columnalign="left"><ns1:mrow><ns1:mtext>if</ns1:mtext><ns1:mspace width="3.33333pt" /><ns1:mi>G</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:mtd></ns1:mtr><ns1:mtr><ns1:mtd columnalign="left"><ns1:mrow><ns1:mtext>Binomial</ns1:mtext><ns1:mo>(</ns1:mo><ns1:mi>T</ns1:mi><ns1:mo>,</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>.</ns1:mo><ns1:mn>5</ns1:mn><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mtd><ns1:mtd columnalign="left"><ns1:mrow><ns1:mtext>if</ns1:mtext><ns1:mspace width="3.33333pt" /><ns1:mi>G</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow></ns1:mtd></ns1:mtr><ns1:mtr><ns1:mtd columnalign="left"><ns1:mrow><ns1:mtext>Binomial</ns1:mtext><ns1:mo>(</ns1:mo><ns1:mi>T</ns1:mi><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>-</ns1:mo><ns1:mi>&#1013;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mtd><ns1:mtd columnalign="left"><ns1:mrow><ns1:mtext>if</ns1:mtext><ns1:mspace width="3.33333pt" /><ns1:mi>G</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>2</ns1:mn><ns1:mo>,</ns1:mo></ns1:mrow></ns1:mtd></ns1:mtr></ns1:mtable></ns1:mfenced></ns1:mrow></ns1:math></alternatives><label>(1)</label></disp-formula>
where <italic>&#1013;</italic> is the probability that a read allele is different from the true allele and is referred to as the error rate. The &#8220;errors&#8221; here comprise both base-calling and alignment errors. We treat <italic>&#1013;</italic> as a free parameter that is locus-specific and will be estimated from the read data [<xref ref-type="bibr" rid="pgen.1006040.ref015">15</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">To account for case-control sampling, we adopt the retrospective likelihood with individual contribution
<disp-formula id="pgen.1006040.e002"><alternatives><graphic id="pgen.1006040.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e002.jpg" /><ns1:math id="M2"><ns1:mrow><ns1:mtext>Pr</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>R</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>D</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:msub><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>g</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:mtext>Pr</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>R</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>D</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>Pr</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>D</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:msub><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>g</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:mtext>Pr</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>R</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>Pr</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>D</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>)</ns1:mo><ns1:mo>,</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
where the second equation follows from two assumptions: first, the binomial distribution for read count data depends only on the underlying genotype, not on the disease status; second, the genotype distribution depends only on the disease status, not on the read depth. Thus, the likelihood based on <italic>n</italic> subjects takes the form
<disp-formula id="pgen.1006040.e003"><alternatives><graphic id="pgen.1006040.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e003.jpg" /><ns1:math id="M3"><ns1:mrow><ns1:msub><ns1:mi>L</ns1:mi><ns1:mtext>CC</ns1:mtext></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>&#960;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#960;</ns1:mi><ns1:mn>0</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#1013;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#1013;</ns1:mi><ns1:mn>0</ns1:mn></ns1:msub><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:munder><ns1:mo>&#8719;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>&#8712;</ns1:mo><ns1:msub><ns1:mi mathvariant="script">D</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:munder><ns1:munder><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>g</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>2</ns1:mn></ns1:mrow></ns1:munder><ns1:msub><ns1:mi>P</ns1:mi><ns1:msub><ns1:mi>&#1013;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>R</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>P</ns1:mi><ns1:msub><ns1:mi>&#960;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:munder><ns1:mo>&#8719;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>&#8712;</ns1:mo><ns1:msub><ns1:mi mathvariant="script">D</ns1:mi><ns1:mn>0</ns1:mn></ns1:msub></ns1:mrow></ns1:munder><ns1:munder><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>g</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>2</ns1:mn></ns1:mrow></ns1:munder><ns1:msub><ns1:mi>P</ns1:mi><ns1:msub><ns1:mi>&#1013;</ns1:mi><ns1:mn>0</ns1:mn></ns1:msub></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>R</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>P</ns1:mi><ns1:msub><ns1:mi>&#960;</ns1:mi><ns1:mn>0</ns1:mn></ns1:msub></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>,</ns1:mo></ns1:mrow></ns1:math></alternatives><label>(2)</label></disp-formula>
where <inline-formula id="pgen.1006040.e004"><alternatives><graphic id="pgen.1006040.e004g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e004.jpg" /><ns1:math id="M4"><ns1:msub><ns1:mi mathvariant="script">D</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:math></alternatives></inline-formula> and <inline-formula id="pgen.1006040.e005"><alternatives><graphic id="pgen.1006040.e005g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e005.jpg" /><ns1:math id="M5"><ns1:msub><ns1:mi mathvariant="script">D</ns1:mi><ns1:mn>0</ns1:mn></ns1:msub></ns1:math></alternatives></inline-formula> denote the sets of cases and controls, respectively, <italic>&#960;</italic><sub><italic>d</italic></sub> denotes the allele frequency for <italic>D</italic> = <italic>d</italic>, and (<italic>&#960;</italic><sub>1</sub>, <italic>&#1013;</italic><sub>1</sub>) and (<italic>&#960;</italic><sub>0</sub>, <italic>&#1013;</italic><sub>0</sub>) are separate parameters for cases and controls. Note that in writing <xref ref-type="disp-formula" rid="pgen.1006040.e003">Eq (2)</xref> we assume that the depth <italic>T</italic> is independent of the genotype <italic>G</italic>. Also note that this formulation obviates the need to model other covariates (e.g., age and environmental exposures) as long as they are not confounders. The null hypothesis of the association test is <italic>H</italic><sub>0</sub>: <italic>&#960;</italic><sub>1</sub> = <italic>&#960;</italic><sub>0</sub>. We re-parameterize (<italic>&#960;</italic><sub>1</sub>, <italic>&#960;</italic><sub>0</sub>) in terms of (<italic>&#945;</italic>, <italic>&#946;</italic>) such that <italic>&#960;</italic><sub>0</sub> = <italic>e</italic><sup><italic>&#945;</italic></sup>/(1 + <italic>e</italic><sup><italic>&#945;</italic></sup>) and <italic>&#960;</italic><sub>1</sub> = <italic>e</italic><sup><italic>&#945;</italic>+<italic>&#946;</italic></sup>/(1 + <italic>e</italic><sup><italic>&#945;</italic>+<italic>&#946;</italic></sup>); then the null hypothesis is <italic>H</italic><sub>0</sub>: <italic>&#946;</italic> = 0. The score function for <italic>&#946;</italic> under <italic>H</italic><sub>0</sub>, as derived in <xref ref-type="supplementary-material" rid="pgen.1006040.s001">S1 Text</xref>, can be written as
<disp-formula id="pgen.1006040.e006"><alternatives><graphic id="pgen.1006040.e006g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e006.jpg" /><ns1:math id="M6"><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mo>=</ns1:mo><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>n</ns1:mi></ns1:munderover><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>D</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>-</ns1:mo><ns1:mfrac><ns1:msub><ns1:mi>n</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mi>n</ns1:mi></ns1:mfrac><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>G</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo></ns1:mrow></ns1:math></alternatives><label>(3)</label></disp-formula>
where
<disp-formula id="pgen.1006040.e007"><alternatives><graphic id="pgen.1006040.e007g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e007.jpg" /><ns1:math id="M7"><ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>G</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>g</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:mi>g</ns1:mi><ns1:msub><ns1:mi>P</ns1:mi><ns1:msub><ns1:mover accent="true"><ns1:mi>&#1013;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:msub><ns1:mi>D</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:msub></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>R</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>P</ns1:mi><ns1:msub><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>0</ns1:mn></ns1:msub></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mrow><ns1:msub><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>g</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>P</ns1:mi><ns1:msub><ns1:mover accent="true"><ns1:mi>&#1013;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:msub><ns1:mi>D</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:msub></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>R</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>P</ns1:mi><ns1:msub><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>0</ns1:mn></ns1:msub></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:mfrac><ns1:mo>,</ns1:mo></ns1:mrow></ns1:math></alternatives></disp-formula>
<italic>n</italic><sub>1</sub> is the number of cases, and <inline-formula id="pgen.1006040.e008"><alternatives><graphic id="pgen.1006040.e008g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e008.jpg" /><ns1:math id="M8"><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>0</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mover accent="true"><ns1:mi>&#1013;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mover accent="true"><ns1:mi>&#1013;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>0</ns1:mn></ns1:msub><ns1:mo>)</ns1:mo></ns1:mrow></ns1:math></alternatives></inline-formula> are restricted maximum likelihood estimates (MLEs) under the null; these restricted MLEs can be obtained via the expectation-maximization (EM) algorithm described in <xref ref-type="supplementary-material" rid="pgen.1006040.s002">S2 Text</xref>. <inline-formula id="pgen.1006040.e009"><alternatives><graphic id="pgen.1006040.e009g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e009.jpg" /><ns1:math id="M9"><ns1:msub><ns1:mover accent="true"><ns1:mi>G</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mi>i</ns1:mi></ns1:msub></ns1:math></alternatives></inline-formula> can be interpreted as the posterior dosage of the minor allele (estimated under the null hypothesis); as the read depth increases, <inline-formula id="pgen.1006040.e010"><alternatives><graphic id="pgen.1006040.e010g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e010.jpg" /><ns1:math id="M10"><ns1:msub><ns1:mover accent="true"><ns1:mi>G</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mi>i</ns1:mi></ns1:msub></ns1:math></alternatives></inline-formula> converges to the underlying genotype <italic>G</italic><sub><italic>i</italic></sub> and <italic>S</italic> reduces to the standard score statistic <inline-formula id="pgen.1006040.e011"><alternatives><graphic id="pgen.1006040.e011g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e011.jpg" /><ns1:math id="M11"><ns1:mrow><ns1:msubsup><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>n</ns1:mi></ns1:msubsup><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>D</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>-</ns1:mo><ns1:msub><ns1:mi>n</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>/</ns1:mo><ns1:mi>n</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>G</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:mrow></ns1:math></alternatives></inline-formula>. Finally, we construct the burden statistic <italic>W</italic> as a (weighted) sum of the score statistics at a set of SNVs in the gene of interest. The variance estimator <italic>V</italic> for <italic>W</italic> is calculated as the empirical variance of the efficient score functions [<xref ref-type="bibr" rid="pgen.1006040.ref016">16</xref>]. When true SNVs are used, the test statistic <inline-formula id="pgen.1006040.e012"><alternatives><graphic id="pgen.1006040.e012g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e012.jpg" /><ns1:math id="M12"><ns1:mrow><ns1:mi>Z</ns1:mi><ns1:mo>=</ns1:mo><ns1:mi>W</ns1:mi><ns1:mo>/</ns1:mo><ns1:msqrt><ns1:mi>V</ns1:mi></ns1:msqrt></ns1:mrow></ns1:math></alternatives></inline-formula> is asymptotically normal with mean 0 and variance 1.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The score statistic of the Derkach test [<xref ref-type="bibr" rid="pgen.1006040.ref014">14</xref>] has the same form as <xref ref-type="disp-formula" rid="pgen.1006040.e006">Eq (3)</xref>, as it also uses the posterior dosage <inline-formula id="pgen.1006040.e013"><alternatives><graphic id="pgen.1006040.e013g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e013.jpg" /><ns1:math id="M13"><ns1:msub><ns1:mover accent="true"><ns1:mi>G</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mi>i</ns1:mi></ns1:msub></ns1:math></alternatives></inline-formula>. The only difference is that the Derkach test substitutes the genotype likelihood <inline-formula id="pgen.1006040.e014"><alternatives><graphic id="pgen.1006040.e014g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e014.jpg" /><ns1:math id="M14"><ns1:mrow><ns1:msub><ns1:mi>P</ns1:mi><ns1:msub><ns1:mover accent="true"><ns1:mi>&#1013;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:msub><ns1:mi>D</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:msub></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>R</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula> that is provided in the output of standard genotype calling packages [<xref ref-type="bibr" rid="pgen.1006040.ref008">8</xref>, <xref ref-type="bibr" rid="pgen.1006040.ref009">9</xref>], which calculate error rates based on <italic>phred</italic> scores.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">In reality, the locations of rare SNVs are not available without calling genotypes. In order to include the maximum set of variants in the burden test without calling genotypes, we develop a screening algorithm to screen every locus (i.e., base pair) in the genome and filter out only loci that are &#8220;uninformative&#8221; in the sense that they yield <italic>S</italic> = 0 and thus do not contribute to the test statistic. Specifically, we consider the likelihood <inline-formula id="pgen.1006040.e015"><alternatives><graphic id="pgen.1006040.e015g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e015.jpg" /><ns1:math id="M15"><ns1:mrow><ns1:msub><ns1:mi>L</ns1:mi><ns1:mi mathvariant="normal">S</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>&#1013;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:msubsup><ns1:mo>&#8719;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:msup><ns1:mi>n</ns1:mi><ns1:mo>&#8242;</ns1:mo></ns1:msup></ns1:msubsup><ns1:msub><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>g</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:mtext>&#8201;</ns1:mtext><ns1:msub><ns1:mi>P</ns1:mi><ns1:mi>&#1013;</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>R</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>,</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>P</ns1:mi><ns1:mi>&#960;</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>g</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula> which is based on a homogenous group (i.e., cases or controls only) of <italic>n</italic>&#8242; subjects. Let <inline-formula id="pgen.1006040.e016"><alternatives><graphic id="pgen.1006040.e016g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e016.jpg" /><ns1:math id="M16"><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover></ns1:math></alternatives></inline-formula> be the MLE based on <italic>L</italic><sub>S</sub>(<italic>&#960;</italic>, <italic>&#1013;</italic>)under the constraint that <italic>&#960;</italic> &#8712; [0, 1] and note that <inline-formula id="pgen.1006040.e017"><alternatives><graphic id="pgen.1006040.e017g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e017.jpg" /><ns1:math id="M17"><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula> indicates no mutation in this group at this locus. Fortunately, we can easily determine whether <inline-formula id="pgen.1006040.e018"><alternatives><graphic id="pgen.1006040.e018g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e018.jpg" /><ns1:math id="M18"><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula> without iteratively solving for <inline-formula id="pgen.1006040.e019"><alternatives><graphic id="pgen.1006040.e019g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e019.jpg" /><ns1:math id="M19"><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover></ns1:math></alternatives></inline-formula>. By definition, <inline-formula id="pgen.1006040.e020"><alternatives><graphic id="pgen.1006040.e020g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e020.jpg" /><ns1:math id="M20"><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover></ns1:math></alternatives></inline-formula> also maximizes the profile likelihood <italic>pl</italic>(<italic>&#960;</italic>) = max<sub><italic>&#1013;</italic></sub> log <italic>L</italic><sub>S</sub>(<italic>&#960;</italic>, <italic>&#1013;</italic>). Because we have shown in <xref ref-type="supplementary-material" rid="pgen.1006040.s003">S3 Text</xref> that <italic>pl</italic>(<italic>&#960;</italic>) is a concave function of <italic>&#960;</italic>, a negative derivative of <italic>pl</italic>(<italic>&#960;</italic>) at <italic>&#960;</italic> = 0 leads to <inline-formula id="pgen.1006040.e021"><alternatives><graphic id="pgen.1006040.e021g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e021.jpg" /><ns1:math id="M21"><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula>. At <italic>&#960;</italic> = 0, the <italic>&#1013;</italic> maximizing log <italic>L</italic><sub>S</sub>(<italic>&#960;</italic>, <italic>&#1013;</italic>) can be easily determined because, in the absence of any minor alleles, all reads carrying the minor allele must be errors. Therefore, we check the sign of the derivative of <italic>pl</italic>(<italic>&#960;</italic>) at <italic>&#960;</italic> = 0 for cases and controls separately and screen out the loci at which both signs are negative. If <inline-formula id="pgen.1006040.e022"><alternatives><graphic id="pgen.1006040.e022g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e022.jpg" /><ns1:math id="M22"><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula> in both cases and controls, then <inline-formula id="pgen.1006040.e023"><alternatives><graphic id="pgen.1006040.e023g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e023.jpg" /><ns1:math id="M23"><ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>0</ns1:mn></ns1:msub><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula> in the combined sample, where <inline-formula id="pgen.1006040.e024"><alternatives><graphic id="pgen.1006040.e024g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e024.jpg" /><ns1:math id="M24"><ns1:msub><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>0</ns1:mn></ns1:msub></ns1:math></alternatives></inline-formula> was defined in the text following expression <xref ref-type="disp-formula" rid="pgen.1006040.e006">Eq (3)</xref>. From <inline-formula id="pgen.1006040.e025"><alternatives><graphic id="pgen.1006040.e025g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e025.jpg" /><ns1:math id="M25"><ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>0</ns1:mn></ns1:msub><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula>, we have <inline-formula id="pgen.1006040.e026"><alternatives><graphic id="pgen.1006040.e026g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e026.jpg" /><ns1:math id="M26"><ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>G</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula> for all individuals and thus <italic>S</italic> = 0. This screening algorithm only involves evaluating simple (derivative) functions twice at each locus without any iteration, and is thus computationally extremely efficient.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Although most monomorphic loci are &#8220;uninformative&#8221; and will be screened out, there are exceptions. It is possible that a truly monomorphic locus has <inline-formula id="pgen.1006040.e027"><alternatives><graphic id="pgen.1006040.e027g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e027.jpg" /><ns1:math id="M27"><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mo>&gt;</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula> in one disease group or both, if by chance some individuals have more errors than expected. If a truly monomorphic locus has <inline-formula id="pgen.1006040.e028"><alternatives><graphic id="pgen.1006040.e028g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e028.jpg" /><ns1:math id="M28"><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mo>&gt;</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula> in the control group but <inline-formula id="pgen.1006040.e029"><alternatives><graphic id="pgen.1006040.e029g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e029.jpg" /><ns1:math id="M29"><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#960;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula> in the case group, the score statistic <italic>S</italic> of this locus will have a negative mean. Such loci will accumulate over the gene when controls have systematically lower depth (or higher error rate) than cases, and then the expected value of the burden statistic <italic>W</italic> will be substantially biased below zero, even when allele frequencies are identical among cases and controls at true SNVs. Consequently, screening for SNVs in the presence of differential sequencing qualities between cases and controls will invalidate the asymptotic version of our test.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We thus propose a bootstrap procedure for assessing the significance of the observed test statistic <italic>Z</italic>. The idea is to generate bootstrap datasets that mimic the original data in terms of read depth and error rate, have the same number of truly monomorphic loci and true SNVs, but have no difference in allele frequencies among cases and controls. To this end, we condition on the observed depth <italic>T</italic> and simulate the minor-allele read count <italic>R</italic> using the estimated error rates <inline-formula id="pgen.1006040.e030"><alternatives><graphic id="pgen.1006040.e030g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e030.jpg" /><ns1:math id="M30"><ns1:msub><ns1:mover accent="true"><ns1:mi>&#1013;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>1</ns1:mn></ns1:msub></ns1:math></alternatives></inline-formula> and <inline-formula id="pgen.1006040.e031"><alternatives><graphic id="pgen.1006040.e031g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e031.jpg" /><ns1:math id="M31"><ns1:msub><ns1:mover accent="true"><ns1:mi>&#1013;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>0</ns1:mn></ns1:msub></ns1:math></alternatives></inline-formula> once the underlying genotype <italic>G</italic> is simulated. However, it is nontrivial to simulate <italic>G</italic>, because we do not know how many loci in the gene are true SNVs and what are allele frequencies at these SNVs. To obtain this information, we first form a &#8220;monomorphic&#8221; dataset by simulating <italic>R</italic> at every locus in the gene assuming that all <italic>G</italic>s are zero; thus, each read for the minor allele is an error that occurs with rate <inline-formula id="pgen.1006040.e032"><alternatives><graphic id="pgen.1006040.e032g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e032.jpg" /><ns1:math id="M32"><ns1:msub><ns1:mover accent="true"><ns1:mi>&#1013;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>1</ns1:mn></ns1:msub></ns1:math></alternatives></inline-formula> or <inline-formula id="pgen.1006040.e033"><alternatives><graphic id="pgen.1006040.e033g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e033.jpg" /><ns1:math id="M33"><ns1:msub><ns1:mover accent="true"><ns1:mi>&#1013;</ns1:mi><ns1:mo>&#732;</ns1:mo></ns1:mover><ns1:mn>0</ns1:mn></ns1:msub></ns1:math></alternatives></inline-formula>, depending on the disease status. This dataset should provide a good approximation to the truly monomorphic loci in the original data, as the proportion of true SNVs in the original data should be small. Let <italic>M</italic><sub><italic>s</italic></sub> be the number of loci that are screened in from the original data and let <italic>F</italic><sub><italic>s</italic></sub>(<italic>&#960;</italic>) be the cumulative distribution function (CDF) of estimated MAFs at the <italic>M</italic><sub><italic>s</italic></sub> loci. Let <italic>M</italic><sub><italic>m</italic></sub> and <italic>F</italic><sub><italic>m</italic></sub>(<italic>&#960;</italic>) be their counterparts in the monomorphic dataset. The CDF of allele frequencies at true SNVs, denoted by <italic>F</italic><sub><italic>p</italic></sub>(<italic>&#960;</italic>), is related to <italic>F</italic><sub><italic>s</italic></sub>(<italic>&#960;</italic>) and <italic>F</italic><sub><italic>m</italic></sub>(<italic>&#960;</italic>) through the equation
<disp-formula id="pgen.1006040.e034"><alternatives><graphic id="pgen.1006040.e034g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e034.jpg" /><ns1:math id="M34"><ns1:mrow><ns1:msub><ns1:mi>F</ns1:mi><ns1:mi>s</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mi>&#981;</ns1:mi><ns1:msub><ns1:mi>F</ns1:mi><ns1:mi>m</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>-</ns1:mo><ns1:mi>&#981;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>F</ns1:mi><ns1:mi>p</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>,</ns1:mo></ns1:mrow></ns1:math></alternatives></disp-formula>
where <italic>&#981;</italic> is the proportion of monomorphic loci among loci that are screened in. This equation expresses the fact that the distribution of observed (non-zero) allele frequencies <italic>F</italic><sub><italic>s</italic></sub>(<italic>&#960;</italic>) in the original data is a mixture of the distributions for allele frequencies of true SNVs <italic>F</italic><sub><italic>p</italic></sub>(<italic>&#960;</italic>) and artifactual SNVs <italic>F</italic><sub><italic>m</italic></sub>(<italic>&#960;</italic>) that actually correspond to monomorphic loci. We estimate <italic>&#981;</italic> by <inline-formula id="pgen.1006040.e035"><alternatives><graphic id="pgen.1006040.e035g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e035.jpg" /><ns1:math id="M35"><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#981;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mo>=</ns1:mo><ns1:msub><ns1:mi>M</ns1:mi><ns1:mi>m</ns1:mi></ns1:msub><ns1:mo>/</ns1:mo><ns1:msub><ns1:mi>M</ns1:mi><ns1:mi>s</ns1:mi></ns1:msub></ns1:mrow></ns1:math></alternatives></inline-formula> and <italic>F</italic><sub><italic>p</italic></sub> by <inline-formula id="pgen.1006040.e036"><alternatives><graphic id="pgen.1006040.e036g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e036.jpg" /><ns1:math id="M36"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo stretchy="true">^</ns1:mo></ns1:mover></ns1:mrow><ns1:mi>p</ns1:mi></ns1:msub><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mo>=</ns1:mo><ns1:msup><ns1:mrow><ns1:mo stretchy="false">(</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:mover accent="true"><ns1:mi>&#981;</ns1:mi><ns1:mo stretchy="true">^</ns1:mo></ns1:mover><ns1:mo stretchy="false">)</ns1:mo></ns1:mrow><ns1:mrow><ns1:mo>&#8722;</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msup><ns1:mrow><ns1:mo>{</ns1:mo><ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo stretchy="true">^</ns1:mo></ns1:mover><ns1:mi>s</ns1:mi></ns1:msub><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mo>&#8722;</ns1:mo><ns1:mover accent="true"><ns1:mi>&#981;</ns1:mi><ns1:mo stretchy="true">^</ns1:mo></ns1:mover><ns1:msub><ns1:mrow><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo stretchy="true">^</ns1:mo></ns1:mover></ns1:mrow><ns1:mi>m</ns1:mi></ns1:msub><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo stretchy="false">)</ns1:mo></ns1:mrow><ns1:mo>}</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula>, where <inline-formula id="pgen.1006040.e037"><alternatives><graphic id="pgen.1006040.e037g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e037.jpg" /><ns1:math id="M37"><ns1:msub><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mi>s</ns1:mi></ns1:msub></ns1:math></alternatives></inline-formula> and <inline-formula id="pgen.1006040.e038"><alternatives><graphic id="pgen.1006040.e038g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e038.jpg" /><ns1:math id="M38"><ns1:msub><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mi>m</ns1:mi></ns1:msub></ns1:math></alternatives></inline-formula> are empirical CDF estimators of <italic>F</italic><sub><italic>s</italic></sub>(<italic>&#960;</italic>) and <italic>F</italic><sub><italic>m</italic></sub>(<italic>&#960;</italic>) respectively. To ensure that <inline-formula id="pgen.1006040.e039"><alternatives><graphic id="pgen.1006040.e039g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e039.jpg" /><ns1:math id="M39"><ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mi>p</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula> is monotonically increasing, we refine <inline-formula id="pgen.1006040.e040"><alternatives><graphic id="pgen.1006040.e040g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e040.jpg" /><ns1:math id="M40"><ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mi>p</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula> by fitting an isotonic regression to data points of <inline-formula id="pgen.1006040.e041"><alternatives><graphic id="pgen.1006040.e041g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e041.jpg" /><ns1:math id="M41"><ns1:mrow><ns1:msup><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>-</ns1:mo><ns1:mover accent="true"><ns1:mi>&#981;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mrow><ns1:mo>-</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msup><ns1:mo>{</ns1:mo><ns1:msub><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mi>s</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>-</ns1:mo><ns1:mover accent="true"><ns1:mi>&#981;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:msub><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mi>m</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>}</ns1:mo></ns1:mrow></ns1:math></alternatives></inline-formula> evaluated at the pooled (<italic>M</italic><sub><italic>s</italic></sub> + <italic>M</italic><sub><italic>m</italic></sub>) MAFs by the pooled-adjacent-violator algorithm (PAVA) [<xref ref-type="bibr" rid="pgen.1006040.ref017">17</xref>]. After the largest value of MAF, we set <inline-formula id="pgen.1006040.e042"><alternatives><graphic id="pgen.1006040.e042g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e042.jpg" /><ns1:math id="M42"><ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mi>p</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula>. Finally, starting from the monomorphic dataset, we select <inline-formula id="pgen.1006040.e043"><alternatives><graphic id="pgen.1006040.e043g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e043.jpg" /><ns1:math id="M43"><ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>M</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mi>p</ns1:mi></ns1:msub><ns1:mo>=</ns1:mo><ns1:msub><ns1:mi>M</ns1:mi><ns1:mi>s</ns1:mi></ns1:msub><ns1:mo>-</ns1:mo><ns1:msub><ns1:mi>M</ns1:mi><ns1:mi>m</ns1:mi></ns1:msub></ns1:mrow></ns1:math></alternatives></inline-formula> loci to be SNVs, sample <italic>&#960;</italic> from <inline-formula id="pgen.1006040.e044"><alternatives><graphic id="pgen.1006040.e044g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e044.jpg" /><ns1:math id="M44"><ns1:msub><ns1:mover accent="true"><ns1:mi>F</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mi>p</ns1:mi></ns1:msub></ns1:math></alternatives></inline-formula>, and re-generate <italic>G</italic> and <italic>R</italic> at these SNVs to form a final bootstrap dataset. Note that, for a small <italic>&#960;</italic>, we may need to resample <italic>G</italic> repeatedly until each truly polymorphic locus screens in. The bootstrap statistic is then calculated based on all the loci that were screened in from the final bootstrap dataset. The entire procedure is repeated to generate multiple bootstrap replicates.</p><p>Although bootstrap tests are computationally intensive in general, we can save considerable time by adopting a sequential stopping rule [<xref ref-type="bibr" rid="pgen.1006040.ref018">18</xref>]. We stop after generating <italic>L</italic><sub>min</sub> bootstrap replicates, if these early replicates suggest a large <italic>p</italic>-value. When <italic>L</italic><sub>min</sub> = 5, the number of replicates at termination has a median of only 10 for a gene having no SNVs that affect the trait. We also use a closed sampling scheme, in which we restrict the total number of bootstrap replicates to be at most <italic>K</italic><sub>max</sub>. If we stop when <italic>L</italic><sub>min</sub> bootstrap statistics exceed the observed <italic>Z</italic> and <italic>K</italic><sub>obs</sub> (&#8804;<italic>K</italic><sub>max</sub>) replicates have been collected, we set the <italic>p</italic>-value to <italic>L</italic><sub>min</sub>/<italic>K</italic><sub>obs</sub>. If we stop when <italic>K</italic><sub>max</sub> replicates are reached and only <italic>L</italic><sub>obs</sub> (&lt;<italic>L</italic><sub>min</sub>) values exceed <italic>Z</italic>, we set the <italic>p</italic>-value to (<italic>L</italic><sub>obs</sub> + 1)/(<italic>K</italic><sub>max</sub> + 1).</p><p>The MLEs of error rates may not recover the true distribution of error rates, which is essential for generating valid bootstrap replicates. In particular, when the true error rates are very small (e.g., &#8764; 0.02%), the MLEs tend to be over-dispersed. Therefore, we propose the following &#8220;adjusted&#8221; empirical Bayes (EB) estimator of the error rate to be used in bootstrap (instead of the MLE), which is calculated separately among cases and controls. We assume a prior beta distribution for error rates, i.e., <italic>&#1013;</italic><sub><italic>j</italic></sub> &#8764; <italic>Beta</italic>(<italic>a</italic>, <italic>b</italic>), where <italic>j</italic> = 1, &#8230;, <italic>M</italic>, <italic>M</italic> is the total number of loci in the gene, and <italic>a</italic> and <italic>b</italic> are hyperparameters that can be consistently estimated by the method of moments (see <xref ref-type="supplementary-material" rid="pgen.1006040.s004">S4 Text</xref>). While the EB estimator is easily obtained (<xref ref-type="supplementary-material" rid="pgen.1006040.s004">S4 Text</xref>), it is known that the distribution of EB estimators is over-shrunk [<xref ref-type="bibr" rid="pgen.1006040.ref019">19</xref>]. Louis and Shen [<xref ref-type="bibr" rid="pgen.1006040.ref019">19</xref>] proposed estimators that have good distribution, rank and expected value, but these are cumbersome to compute. We use a simplified version of the Louis and Shen estimator in which we first calculate the EB estimators but then replace the (ordered) EB estimators by (ordered) quantiles of the prior beta distribution evaluated using the method-of-moments estimators of <italic>a</italic> and <italic>b</italic>. Because the sample size <italic>M</italic> is typically on the order of a few hundred, <italic>a</italic> and <italic>b</italic> are accurately estimated, ensuring that the distribution of the adjusted EB estimates will closely resemble the prior (true) distribution of error rates.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We have observed that a small proportion of read data (<italic>R</italic>, <italic>T</italic>) do not fit the binomial <xref ref-type="disp-formula" rid="pgen.1006040.e001">model (1)</xref>. This may be due to genotype mosaicism (i.e., the presence of two or more populations of cells with different genotypes in one individual), experimental artifacts, sample contamination, or copy number variants. To detect data that do not fit the binomial model, for each individual at each locus that screens in, we calculated a likelihood-ratio-type statistic for the goodness of fit to the binomial model
<disp-formula id="pgen.1006040.e045"><alternatives><graphic id="pgen.1006040.e045g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006040.e045.jpg" /><ns1:math id="M45"><ns1:mrow><ns1:mi>Q</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>2</ns1:mn><ns1:mo form="prefix">log</ns1:mo><ns1:mfenced close="}" open="{" separators=""><ns1:msup><ns1:mfenced close=")" open="(" separators=""><ns1:mi>R</ns1:mi><ns1:mo>/</ns1:mo><ns1:mi>T</ns1:mi></ns1:mfenced><ns1:mi>R</ns1:mi></ns1:msup><ns1:msup><ns1:mfenced close=")" open="(" separators=""><ns1:mn>1</ns1:mn><ns1:mo>-</ns1:mo><ns1:mi>R</ns1:mi><ns1:mo>/</ns1:mo><ns1:mi>T</ns1:mi></ns1:mfenced><ns1:mrow><ns1:mi>T</ns1:mi><ns1:mo>-</ns1:mo><ns1:mi>R</ns1:mi></ns1:mrow></ns1:msup><ns1:mo>/</ns1:mo><ns1:munder><ns1:mo form="prefix" movablelimits="true">max</ns1:mo><ns1:mrow><ns1:mi>g</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>2</ns1:mn></ns1:mrow></ns1:munder><ns1:mfenced close="]" open="[" separators=""><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>g</ns1:mi></ns1:msub><ns1:msup><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#1013;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mi>R</ns1:mi></ns1:msup><ns1:msup><ns1:mfenced close="}" open="{" separators=""><ns1:mn>1</ns1:mn><ns1:mo>-</ns1:mo><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>g</ns1:mi></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#1013;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mfenced><ns1:mrow><ns1:mi>T</ns1:mi><ns1:mo>-</ns1:mo><ns1:mi>R</ns1:mi></ns1:mrow></ns1:msup></ns1:mfenced></ns1:mfenced><ns1:mo>,</ns1:mo></ns1:mrow></ns1:math></alternatives></disp-formula>
where <italic>e</italic><sub><italic>g</italic></sub>(<italic>&#1013;</italic>) = <italic>&#1013;</italic>, 0.5, and 1 &#8722; <italic>&#1013;</italic> for <italic>g</italic> = 0, 1, and 2, respectively. Then, we mask an individual at a variant (by setting <italic>T</italic> and <italic>R</italic> to zero) if <italic>Q</italic> is greater than 10 and remove a variant altogether if more than 5 individuals are masked at that locus. We can also identify individuals with problematic data by checking for the presence of an excessive number of <italic>Q</italic>s greater than 10.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The proposed methods are implemented in the C/C++ program <monospace>TASER</monospace>, which is publicly available at <ext-link ext-link-type="uri" ns0:href="http://web1.sph.emory.edu/users/yhu30/software.html">http://web1.sph.emory.edu/users/yhu30/software.html</ext-link>.</p>