<p>The <italic>Arabidopsis thaliana exo70B1-3</italic> mutant was identified from a T-DNA insertion population [<xref ref-type="bibr" rid="pgen.1004945.ref014">14</xref>]. T-DNA insertion lines <italic>exo70B1-1</italic> (N328818), <italic>exo70B1-2</italic> (N717829), <italic>exo70B2-1</italic> (SALK_091877C) were from the Arabidopsis Biological Resource Center or Nottingham Arabidopsis Stock Center. The mutant alleles used for the double mutant analyses were as described: <italic>pad4-1</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref047">47</xref>], <italic>npr1-1</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref016">16</xref>], <italic>eds5-1</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref045">45</xref>], <italic>sid2-2</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref046">46</xref>], <italic>ein2-1</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref068">68</xref>], <italic>coi1-1</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref069">69</xref>], <italic>edr1</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref070">70</xref>], <italic>edr2</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref012">12</xref>], <italic>pmr4-1</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref009">9</xref>], <italic>pen1-1</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref015">15</xref>]. All of the mutants used were in the Columbia-0 (Col-0) background, except for <italic>coi1-1</italic> (in Col-6 background). Double mutants were created by standard genetic crosses and confirmed by PCR.</p><p>
<italic>Arabidopsis</italic> plants were grown in a growth room at 22&#8211;24&#176;C, at a light intensity of 8000 lux, and a 9-h-light /15-h-dark photoperiod for phenotype analysis or 16-h-light / 8-h-dark photoperiod for seed setting, as described previously [<xref ref-type="bibr" rid="pgen.1004945.ref071">71</xref>]. <italic>Nicotiana benthamiana</italic> plants were grown in the same short-day conditions as <italic>Arabidopsis</italic>.</p><p>Adapted powdery mildew strain <italic>Golovinomyces cichoracearum</italic> UCSC1 [<xref ref-type="bibr" rid="pgen.1004945.ref004">4</xref>] was maintained on susceptible <italic>pad4-1</italic> mutant plants. Four-week-old plants were inoculated with powdery mildew using a settling tower at low or high densities [<xref ref-type="bibr" rid="pgen.1004945.ref014">14</xref>] [<xref ref-type="bibr" rid="pgen.1004945.ref042">42</xref>]. Low spore density was used to quantify the number of conidiophores per colony and high density was used to evaluate hyphae growth and lesion formation. The plants were moved to a growth chamber immediately after infection with <italic>G. cichoracearum</italic> to facilitate growth of the fungi. Hyphae and dead plant cells in the infected leaves were visualized by trypan blue staining [<xref ref-type="bibr" rid="pgen.1004945.ref070">70</xref>]. H<sub>2</sub>O<sub>2</sub> accumulation was detected at 2 dpi by 3,3&#8217;-diamino benzidine hydrochloride (DAB) staining [<xref ref-type="bibr" rid="pgen.1004945.ref072">72</xref>]. Callose deposition was detected by aniline blue staining [<xref ref-type="bibr" rid="pgen.1004945.ref073">73</xref>]. The samples were observed and photographed with an Olympus BX60 microscope. Non-adapted wheat powdery mildew <italic>Blumeria graminis f. sp. tritici</italic> E09 was grown on susceptible Chinese wheat variety Jing411 [<xref ref-type="bibr" rid="pgen.1004945.ref074">74</xref>]. Assessment of epidermal cell death in response to <italic>graminis f. sp. tritici</italic> E09 was performed as described [<xref ref-type="bibr" rid="pgen.1004945.ref016">16</xref>] with minor modifications. Briefly, four-week-old plants were inoculated with <italic>B. graminis f. sp. tritici</italic> using a settling tower, and the infected leaves was stained with trypan blue, and epidermal single cell death in the infected siytes was visulized with an Olympus BX60 microscope.</p><p>
<italic>Pseudomonas syringae</italic> infection was performed by leaf infiltration (OD<sub>600</sub> = 5 &#215;10<sup>&#8722;4</sup> for growth curve analysis) as described [<xref ref-type="bibr" rid="pgen.1004945.ref075">75</xref>], unless otherwise indicated. For spray infection, 5-week-old plants were sprayed with bacterial suspensions (OD<sub>600</sub> = 0.2) in 10 mM MgCl<sub>2</sub> containing 0.025% Silwet L-77, and the infected plants were placed under plastic domes for 24 hr.</p><p>Inoculation of <italic>Hyaloperonospora arabidopsidis</italic> (<italic>H. a.</italic>) Noco<italic>2</italic> (5 &#215; 10<sup>4</sup> spores/mL) was performed according to [<xref ref-type="bibr" rid="pgen.1004945.ref076">76</xref>].</p><p>The <italic>exo70B1-3</italic> seeds were incubated with 0.3% EMS for 16 h. Around 60,000 M2 plants derived from 5,000 M1 plants were scored by infection with <italic>G. cichoracearum</italic>. Plants showing wild type-like phenotypes were selected for further characterization.</p><p>The <italic>exo70B1-3</italic> mutant was crossed to ecotype Landsberg <italic>erecta</italic> (L<italic>er</italic>). F2 plants displaying <italic>exo70B1-3</italic> mutant phenotypes were selected for mapping. Using rough mapping markers [<xref ref-type="bibr" rid="pgen.1004945.ref077">77</xref>] and our newly developed SSLP, CAPS or dCAPS markers, we narrowed down the mutation to an 83-kb region spanning BAC MCK7 (GenBank accession AB019228) and MQJ2 (GenBank accession AB025632) on chromosome 5. We subsequently designed PCR primers to amplify the coding sequences of these genes. The suppressor gene <italic>TN2</italic> was also identified by standard map-based cloning.</p><p>For complementation analysis, a 4.3-kb genomic sequence including 1531 bp upstream of the start codon, and 835 bp downstream of the stop codon of <italic>EXO70B1</italic> was amplified and cloned into binary vector pCAMBIA1300. The construct was then introduced into <italic>Agrobacterium tumefaciens</italic> strain GV3101, and transformed into <italic>exo70B1-3</italic> plants by the floral dip method [<xref ref-type="bibr" rid="pgen.1004945.ref078">78</xref>]. Transformants were selected on 1/2 MS medium containing 80 &#956;g/mL hygromycin. T1 transgenic plants were used for phenotyping.</p><p>Extraction of total RNA, synthesis of first-strand cDNA and analysis of gene expression were performed as described previously [<xref ref-type="bibr" rid="pgen.1004945.ref079">79</xref>].</p><p>The promoter region of <italic>EXO70B1</italic>, including 1531 bp upstream of the start codon, was cloned into the pBI121 vector. Homozygous transgenic plants in the T3 generation were used for GUS activity analysis according to [<xref ref-type="bibr" rid="pgen.1004945.ref080">80</xref>].</p><p>SA extraction and HPLC assays were conducted as described previously [<xref ref-type="bibr" rid="pgen.1004945.ref049">49</xref>,<xref ref-type="bibr" rid="pgen.1004945.ref081">81</xref>].</p><p>The MATCHMAKER GAL4 Two-Hybrid System 3 (Clontech) was used to examine the interactions between EXO70B1 and other exocyst subunits, as well as PEN1 and SNAP33. <italic>EXO70B1</italic>-pGBKT7 was co-transformed into <italic>Saccharomyces cerevisiae</italic> strain AH109 with SEC3a, SEC5a, SEC6, SEC8, SEC10, SEC15b, EXO84bN (the N terminal domain of EXO84b), PEN1 and SNAP33 fused with pGADT7. The constructs containing SEC3a, SEC5a, or SEC15b fused with pGADT7 were kindly provided by Dr. Viktor Z&#225;rsk&#253; [<xref ref-type="bibr" rid="pgen.1004945.ref025">25</xref>]. The constructs containing SEC6, SEC8, SEC10, EXO84bN, PEN1 and SNAP33 fused with pGADT7 were made in this study using primers listed in <xref ref-type="supplementary-material" rid="pgen.1004945.s021">S1 Table</xref>. A single colony grown on SD/-Leu/-Trp plate was incubated in liquid media, the overnight culture was diluted to OD<sub>600</sub> = 0.5 with sterile water, and a 10 &#956;L drop of dilution was plated on SD/-Ade/-His/-Leu/-Trp plates to test the interaction.</p><p>The interaction of EXO70B1 and TN2 was assayed using Gateway-modified versions of the traditional pEG202 and pJG4-5 vectors [<xref ref-type="bibr" rid="pgen.1004945.ref082">82</xref>]. pDONR207 Gateway Entry clones containing ORFs (<xref ref-type="supplementary-material" rid="pgen.1004945.s021">S1 Table</xref> for cloning primers) were recombined into pEG202 and pJG4-5 using LR clonase II (Invitrogen). The resulting vectors were sequence verified and then transformed into EGY48 (pEG202) and RFY206 containing the LacZ reporter pSH18-34 (pJG4-5). The resulting strains were mated and diploids selected on synthetic media (SD-HUW). Diploids were spotted onto SD-HUW +X-gal (80 &#956;g/mL). Protein accumulation of fusion proteins in diploids was verified with standard Western blotting using anti-LexA (rabbit pAB, AbCam) and anti-HA (rat mAB, Roche) antibodies.</p><p>To create the constructs used in BiFC analysis, the coding sequences of <italic>EXO70B1</italic> and <italic>TN2</italic> were first cloned into pSY736 vector in frame with YFP<sup>N</sup>. The derived YFP<sup>N</sup>-fused-sequences were then amplified and inserted into the Gateway entry vector pDONR207 and subsequently the destination vector pMDC32. Similarly, to create YFP<sup>C</sup> fused to <italic>EXO70B1, SNAP33, VAMP721</italic> and <italic>PEN1</italic>, the coding sequence of each gene was cloned into pSY735 vector in frame with YFP<sup>C</sup>. The inserts were subcloned into pDONR207 and pMDC32 vectors. <italic>Agrobacterium</italic> GV3101 carrying different YFP<sup>N</sup> and YFP<sup>C</sup> pairs were infiltrated into four-week-old <italic>Nicotiana benthamiana</italic> [<xref ref-type="bibr" rid="pgen.1004945.ref083">83</xref>]. YFP fluorescence was observed by confocal microscopy (Carl Zeiss LSM710) after 48hr.</p><p>To create the constructs used in coimmunoprecipitation analysis in <italic>N. benthamiana, EXO70B1</italic> was cloned into the pGWB11 vector with C-terminal FLAG fusion using gateway system (Invitrogen). Similarly, <italic>PEN1</italic> was cloned into pEarleyGate 201 vector with 35S promoter and N-terminal HA fusion, while <italic>SNAP33</italic> and <italic>TN2</italic> were cloned into pEarleyGate 203 vector with 35S promoter and N-terminal Myc fusion. Transient expression, total protein extractions and immunoblots were performed as described previously [<xref ref-type="bibr" rid="pgen.1004945.ref058">58</xref>]. Briefly, about 20 &#956;g protein was resuspended in sample buffer and separated by SDS-PAGE. The protein was detected by immunoblot with primary antibody anti-GFP (Roche 1:1000) or anti-HA (Sigma 1:1000), and then with anti-mouse (rabbit) HRP-linked secondary antibody, followed by detection with the chemiluminescent HRP substrate kit (Millipore).</p><p>For immunoprecipitation, proteins prepared from NB1 buffer [<xref ref-type="bibr" rid="pgen.1004945.ref084">84</xref>] were incubated with primary antibody (3 &#956;L /1 mL), at 4 &#176;C for 3 hours with gentle shaking, and 20 &#956;L protein G agarose beads (Millipore) were added and kept shaking for another 3 hours. The beads were washed 4 times with PBS buffer, and resuspended in PBS buffer. The samples were separated by SDS-PAGE and analyzed by immunoblotting. To produce an EXO70B1-GFP fusion, a 3.4-kb genomic sequence, containing 1531bp promoter and 1872 bp coding sequence of <italic>EXO70B1</italic> without the stop codon, was cloned into the binary vector pEGAD [<xref ref-type="bibr" rid="pgen.1004945.ref085">85</xref>] between the StuI and AgeI cloning sites to remove the 35S promoter from the vector. The derived <italic>EXO70B1pro:EXO70B1-GFP</italic> construct was introduced into the <italic>A. tumefaciens</italic> strain GV3101 and transformed into <italic>exoB1-3</italic> plants using the floral dip method [<xref ref-type="bibr" rid="pgen.1004945.ref078">78</xref>]. To test the interaction between SNAP33 and EXO70B1 in <italic>Arabidopsis</italic>, we amplified <italic>SNAP33</italic> genomic sequence (without stop codon) and fused with an HA tag, and cloned it into pMDC163 vector using gateway system (Invitrogen). And <italic>exo70B1-3</italic> plants expressing <italic>EXO70B1pro:EXO70B1-GFP</italic> were transformed with derived <italic>SNAP33pro:SNAP33-HA</italic> construct, and the T1 plants were used for coimmunoprecipitation immunoblot assays.</p><p>To detect MAP kinase activation, 4-week-old plants were inoculated with <italic>G. cichoracearum</italic> for 3 days, and the infected leaves were excised and quickly frozen in liquid nitrogen. Total protein was isolated and subjected to immunoblot with rabbit anti-pTEpY (Cell Signaling), anti-MPK3 or anti-MPK6 (Sigma) as the primary antibody, following the manufacturer&#8217;s instructions.</p><p>The primers used in this work are listed in <xref ref-type="supplementary-material" rid="pgen.1004945.s021">S1 Table</xref>.</p>