<p xmlns:ns0="http://www.w3.org/1999/xlink">HapMap (<ext-link ext-link-type="uri" ns0:href="http://hapmap.ncbi.nlm.nih.gov/">http://hapmap.ncbi.nlm.nih.gov/</ext-link>) CEU human lymphoblastoid cell lines GM06993, GM11992, and GM12878 (Coriell Cell Repositories, Camden, NJ) were cultured in RPMI 1640 medium supplemented with 15% heat-inactivated fetal bovine serum (Life Technologies) and 1X antibiotics/antimycotics (Life Technologies). Cells were incubated at 37&#176;C with 5% CO<sub>2.</sub> During experiments, cells were grown in petri dishes and treated with 0.3 &#181;g/ml doxorubicin (Sigma, St. Louis, MO) for ChIP-seq, DNase I hypersensitivity, and gene expression.</p><p>GM12878 cells were cultured in triplicate and were either untreated or treated with 0.3 &#181;g/ml doxorubicin for 4 and 18 hrs. Total RNAs was extracted using total RNA Miniprep kit (QIAGEN) with DNase 1 digestion. Total RNA was subjected to the standard cDNA synthesis, dye labeling and hybridization as per Affymetrix's protocols for the Human Exon ST 1.0 array (Affymetrix, Santa Clara, CA), and processed at the NIEHS Microarray Core. Exon expression data was analyzed through Affymetrix Expression Console using gene-level RMA summarization and sketch-quantile normalization methods. The <italic>t</italic>-test was used to calculate the significance of gene expression change. All the Affymetrix microarray data used here have been deposited into the GEO database (GSE51709). DXR induced gene expression array data was validated in a separate cell line using the Illumina gene expression array platform (<xref ref-type="supplementary-material" rid="pgen.1004885.s011">Data File S1</xref>), as well as RT-PCR for selected genes. For replication by qRT-PCR, the cDNA was generated using the First-Strand Synthesis system (Life Technologies). For each biological replicate, target genes were amplified in triplicate using FAM probes designed to span exon junctions for each target gene and PCR Master Mix (Life Technologies). Studies investigating the relationship between gene expression in lymphoblast cell lines and <italic>in vitro</italic> p53 binding have been reported previously (<xref ref-type="bibr" rid="pgen.1004885-Bandele1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1004885-Noureddine1">[31]</xref>).</p><p>Lymphoblastoid cells were seeded in 15-cm dishes and treated with 0.3 &#181;g/mL doxorubicin or vehicle water for 18 hrs. Chromatin from 100&#215;10<sup>6</sup> cells was used for each p53 ChIP-seq experiment. ChIP for p53 was carried out essentially as described in the Agilent Mammalian ChIP-on Chip protocol Version 20.0 (Agilent, Santa Clara, CA) using mouse monoclonal antibody DO-7 (BD Biosciences, San Jose, CA) or a non-specific rabbit IgG conjugated to secondary Dynal magnetic beads (Life Technologies). H3K4me3 ChIP was carried out with the H3K4me3 antibody (EMD Millipore #07-473, Billerica, MA). ChIP and input DNA were quantified using Qubit (Invitrogen). NIH Intramural Sequencing Center (NISC, Bethesda, MD) and the National Center for Genome Resources (NCGR, Santa Fe, NM) created the libraries using the standard Illumina ChIP-seq protocol. ChIP or input libraries were sequenced using Illumina Genome Analyzer IIx. For validation of occupancy, p53 ChIP-PCR was performed in triplicate using Sybr primers designed to span p53RE peaks in promoters of p53 target genes on a 7900 HT real-time PCR machine (Applied Biosystems).</p><p>Genome-wide mapping of DNase I-hypersensitive sites (DHS) was carried out on 2 replicates of untreated and DXR-treated GM12878 cells for 4 and 18 hrs. For DHS profiling, intact nuclei were treated with DNase I, and the DNase I-hypersensitive fraction was analyzed by sequencing as previously described <xref ref-type="bibr" rid="pgen.1004885-Song1">[62]</xref>. Data from two biological replicates under each experimental condition are displayed as tracks in the UCSC genome browser in <xref ref-type="fig" rid="pgen-1004885-g004">Figs. 4</xref>, <xref ref-type="supplementary-material" rid="pgen.1004885.s003">S3</xref>, <xref ref-type="supplementary-material" rid="pgen.1004885.s009">S9</xref>. For analysis of DHS relative to ENCODE genome wide chromatin state, we downloaded ENCODE/OpenChrom (Duke University, Durham, NC) GM12878 DHS data (<xref ref-type="fig" rid="pgen-1004885-g001">Fig. 1</xref>, track K) <xref ref-type="bibr" rid="pgen.1004885-Thurman1">[28]</xref>, <xref ref-type="bibr" rid="pgen.1004885-Ernst1">[33]</xref>. These data (DHS signal intensity) were averaged across p53 binding regions, transformed (log2 +1), grouped by PWM decile and plotted versus p53 peak density (<xref ref-type="fig" rid="pgen-1004885-g007">Figs. 7D </xref>and <xref ref-type="supplementary-material" rid="pgen.1004885.s007">S7C</xref>).</p><p>p53 ChIP-seq, H3K4me3 ChIP-seq, and input DNA sequence reads generated from the Illumina GAIIx were aligned against the human reference sequence (GRCh37p5, or hg19, June 2011) using the Burrows-Wheeler Alignment (BWA) Tool (Li et al., 2008). The uniquely mapped short reads were used to identify regions of the genome with significant enrichment in p53-associated DNA sequences. The peak detection was performed by QuEST 2.4 software <xref ref-type="bibr" rid="pgen.1004885-Valouev1">[36]</xref> using the &#8216;Transcription factor binding site&#8217; setting (bandwidth of 30&#8201;bp, region size of 300&#8201;bp) or the &#8220;Histone-type ChIP&#8221; setting (bandwidth of 100&#8201;bp, region size of 1000&#8201;bp) for H3K4me3 ChIP-seq, and the &#8216;stringent peak calling&#8217; parameters (corresponding to 50-fold ChIP to input enrichment for seeding the regions and 3-fold ChIP enrichment for extending the regions).</p><p>We also used three published studies of p53 ChIP-seq from Nutlin-treated MCF7 cells <xref ref-type="bibr" rid="pgen.1004885-Nikulenkov1">[8]</xref>, 5FU-treated MCF7 cells <xref ref-type="bibr" rid="pgen.1004885-Nikulenkov1">[8]</xref>, 5FU-treated IMR90 cells <xref ref-type="bibr" rid="pgen.1004885-Botcheva1">[7]</xref>, Nutlin and DXR-treated Human osteosarcoma U2OS cells <xref ref-type="bibr" rid="pgen.1004885-Menendez1">[9]</xref>, which deposited the raw reads at the NCBI SRA database. All sequence reads were aligned against the human reference sequence (GRCh37p5, or hg19, June 2011) using the Burrows-Wheeler Alignment (BWA) Tool (Li et al., 2008). For these published studies the peak detection was also performed by QuEST 2.4 software <xref ref-type="bibr" rid="pgen.1004885-Valouev1">[36]</xref> using the &#8216;Transcription factor binding site&#8217; setting (bandwidth of 30&#8201;bp, region size of 300&#8201;bp) for p53 ChIP-seq and the &#8216;stringent peak calling&#8217; parameters. In <xref ref-type="fig" rid="pgen-1004885-g001">Fig. 1</xref> our H3K4me3 ChIP-seq tracks were prepared using the MACS program in order to display comparability with ENCODE tracks.</p><p>MEME Suite <xref ref-type="bibr" rid="pgen.1004885-Bailey1">[40]</xref> was used for the <italic>de novo</italic> motif analysis of the genomic DNA sequences identified by ChIP-seq experiments. Briefly, repeat-masked DNA sequence for each peak was trimmed to 100 bp, centered to the maximal signal of original peak. The MEME algorithm was then applied to identify the top 10 most enriched motifs with a length from 6 to 25.</p><p>Position weight matrix (PWM) score was used to computationally estimate the binding strength to p53. The p53 PWM model was built on 103 published, experimentally validated p53RE sequences from the literature (NCBI PUBMED database), by converting nucleotide frequency values to a position weight matrix score as described <xref ref-type="bibr" rid="pgen.1004885-Wasserman1">[63]</xref>. Next the PWM score for any putative p53RE was calculated by summing the individual matrix values that correspond to the observed nucleotide at each position in that site. The potential p53REs in a genomic sequence were detected by sliding a window along the input sequence, considering the spacer in p53REs <xref ref-type="bibr" rid="pgen.1004885-Wang3">[64]</xref>. Briefly, at each chromosomal position, PWM score calculations were performed on DNA sequences with 20&#8211;23 nucleotides, corresponding to p53REs with a spacer of 0, 1, 2, and 3, respectively. For DNA sequences longer than 20, the first 10 nucleotides and the last 10 nucleotides were concatenated first to make a sequence of 20 nucleotides. For fast searching, a pattern composed of at least 3 of 4 core C and G nucleotides in the p53RE consensus was applied before PWM calculation. In addition, the same calculations were performed on the reversed complimentary sequences. In total, at each chromosomal position, eight calculations were performed. At a given location a putative p53RE was reported based on the following three criteria: (i) the sequence matches the search pattern; and (ii) the PWM score is above the PWM score threshold; and (iii) it contains the shortest spacer and has the largest PWM score. A second computational analysis based on measured binding in an <italic>in vitro</italic> binding assay, termed a binding PWM, was determined using a method described in Noureddine et. al. <xref ref-type="bibr" rid="pgen.1004885-Noureddine1">[31]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Datasets and sample information for histone modifications and CTCF binding for the GM12878 cell line were from the ENCODE project (<ext-link ext-link-type="uri" ns0:href="http://genomes.ucsc.edu">http://genomes.ucsc.edu</ext-link>) via the laboratory of Bradley Bernstein (Broad Institute) <xref ref-type="bibr" rid="pgen.1004885-Ram1">[34]</xref>, at the Massachusetts General Hospital/Harvard Medical School. The chromatin state segmentation was produced in Manolis Kellis's Computational Biology group at the Massachusetts Institute of Technology <xref ref-type="bibr" rid="pgen.1004885-Ernst1">[33]</xref>, <xref ref-type="bibr" rid="pgen.1004885-Ernst2">[54]</xref>. We created analysis files based on 2932 p53 ChIP-seq peaks that had&#8805;30 overlapping sequence reads (<xref ref-type="supplementary-material" rid="pgen.1004885.s011">Data File S1</xref>) and 2415 genes associated with these peaks (many genes had multiple peaks, <xref ref-type="supplementary-material" rid="pgen.1004885.s012">Data File S2</xref>). We merged ENCODE data (ChromHMM15 and ChromHMM7 Combined models, PhyloP, DHS) into our data analysis files that included ChIP-seq peak density (p53, H3K4me3), gene expression (intensity at each time point, fold change). Examples of these data are displayed in <xref ref-type="fig" rid="pgen-1004885-g001">Fig. 1</xref>. In the various analyses we grouped p53 peaks (or genes) based on the H3K4me3 status of nearby genes, the ChromHMM7 Combined status at p53 peaks or at the TSS of nearby genes, or on the computationally calculated binding strength (PWM). ChromHMM7 Combined model failed to call states for some regions and the ChromHMM15 was used in these cases. ChromHMM15 states 14 and 15 (repetitive) were dropped from analysis as no p53 ChIPseq peaks were uniquely mapped to these regions. We also specifically looked at expression levels of genes with &#8220;poised&#8221; promoters carrying H3K4me1/2 marks and H3K27me3 marks (ChromHMM15 state 3). To visualize linear trends related to H3K4me3 or PWM scores we used decile analysis in which PWM values were ordered from low to high and grouped into deciles. We then evaluated other parameters relative to these linear trends. Regression, <italic>t</italic>-tests, and other statistics were calculated using Graphpad Prism (GraphPad Software).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To evaluate the evolution constraint on p53 binding sites, we utilized the placental mammal basewise conservation scores (PhyloP, phylogenetic p-values) downloaded from the UCSC genome browser website. These scores are computed from the PHAST package (<ext-link ext-link-type="uri" ns0:href="http://compgen.bscb.cornell.edu/phast/">http://compgen.bscb.cornell.edu/phast/</ext-link>) for multiple alignments of 45 vertebrate genomes to the human genome. The phyloP scores measure acceleration (faster evolution than expected under neutral drift) as well as conservation (slower than expected evolution). Sites predicted to be conserved are assigned positive scores, while sites predicted to be fast-evolving are assigned negative scores. The absolute values of the scores represent log p-values under a null hypothesis of neutral evolution. The sequence alignment and human-mouse conservation score were analyzed as previously described <xref ref-type="bibr" rid="pgen.1004885-Horvath1">[45]</xref>. Transposon element repeats cataloged by Repeat Masker were accessed at UCSC genome browser website and aligned with p53REs to determine overlap.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Sequence reads for ChIP-seq experiments are deposited in the NCBI SRA database (p53 GSE46991, H3K4me3 GSE51713) and the NCBI Gene Expression Omnibus (GEO: <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo">http://www.ncbi.nlm.nih.gov/geo</ext-link>). Gene expression data are available at GEO under accession number GSE51709.</p>