<p>The inbred lines B73 and Mo17 and their reciprocal F1 hybrids B73XMo17 (BM) and Mo17XB73 (MB) were used for this study. All maize plants were grown at 600 &#956;mol m<sup>&#8722;2</sup> s<sup>&#8722;1</sup> under 16L:8D (hours of light:dark) cycle with temperature 28&#176;C (light) and 23&#176;C (dark) except for the experiments to measure photosynthetic rates, starch and sucrose content. They were carried out in a glasshouse where natural sunlight (maximum photosynthetically active radiation 598 &#177; 354 &#956;mol m<sup>&#8722;2</sup> s<sup>&#8722;1</sup>, mean &#177; SD) was supplemented with fluorescent lighting to achieve a 16L:8D cycle with daily maximum temperature at 26.7 &#177; 2.1&#176;C (mean &#177; SD), daily minimum temperature at 19.4 &#177; 0.4&#176;C (mean &#177; SD), relative humidity at 57 &#177; 7% (mean &#177; SD), and net CO<sub>2</sub> exchange examined during the day. The plants were placed in a randomized design and rotated on a daily basis to minimize positional effects.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The growth traits included seed mass, aerial biomass, plant height, and leaf-blade length and area. For seed weight, 10 seeds randomly selected from three replicates for each genotype were weighed. For aerial biomass, aboveground seedlings for each genotype were collected at 3, 4, 5 and 6 days after planting (DAP) (n = 4&#8211;8). The biomass was weighed after desiccation for 48 h at 80&#176;C. Plant height and leaf-blade length for each genotype were daily measured from 5 DAP to 12 DAP (n = 5). Plant height was measured as the distance from the ground to the top of seedlings. Leaf-blade length was measured as the distance from the base to the tip of the 2<sup>nd</sup> leaf. The 2<sup>nd</sup> leaf at 12 DAP for each genotype was photographed and processed for leaf area using NIH ImageJ software (<ext-link ext-link-type="uri" ns0:href="http://imagej.nih.gov/ij/">http://imagej.nih.gov/ij/</ext-link>) [<xref ref-type="bibr" rid="pgen.1006197.ref070">70</xref>] (n = 6). All experiments were replicated three times, unless noted otherwise.</p><p xmlns:ns0="http://www.w3.org/1999/xlink"><italic>Arabidopsis</italic> seedlings were plated onto Murashige-Skoog media (Sigma, St. Louis, MO) supplemented with 0.8% agar, 3% sucrose (MS agar) and appropriate antibiotics. After stratification in the dark at 4&#176;C for 2 days, plates were transferred to a growth room with a 16L:8D cycle (80 &#956;mol m<sup>&#8722;2</sup> s<sup>&#8722;1</sup>) at 22&#176;C. Luminescence recordings were performed on a TopCount (Packard Bioscience, Shelton, CT) over 7 days, as previously described [<xref ref-type="bibr" rid="pgen.1006197.ref010">10</xref>]. The data were analyzed by fast Fourier transform&#8211;nonlinear least squares (FFT&#8211;NLLS) [<xref ref-type="bibr" rid="pgen.1006197.ref071">71</xref>] using the interface provided by the Biological Rhythms Analysis Software System version 3.0 (BRASS) (<ext-link ext-link-type="uri" ns0:href="http://www.amillar.org/">http://www.amillar.org</ext-link>).</p><p>The construct for <italic>ZmCCA1b</italic> (<italic>ZM2G014902</italic>) overexpression in the binary vector pTF101.1gw1 [<xref ref-type="bibr" rid="pgen.1006197.ref072">72</xref>] (generously provided by Dr. Kan Wang, Iowa State University, Ames, IA) consisted the <italic>ZmUBI</italic> promoter-<italic>ZmUBI</italic> intron (<italic>ZmUBI</italic><sub><italic>pro</italic></sub>) cassette immediately upstream of the <italic>ZmCCA1</italic> coding sequence and the <italic>octopine synthase</italic> (<italic>OCS</italic>) terminator immediately downstream. The <italic>ZmUBI</italic><sub><italic>pro</italic></sub> cassette was PCR amplified from the pANIC6A vector [<xref ref-type="bibr" rid="pgen.1006197.ref073">73</xref>] with primers F 5&#8217;-CACCGT<underline>AAGCTT</underline>AATGCAGTGCAGCGTGACCC-3&#8217; and R 5&#8217;-GT<underline>AAGCTT</underline>TGCAGAAGTAACACCAAACAACAGGGTG-3&#8217;, which added the HindIII restriction site (underlined) onto each side of the fragment. The <italic>OCS</italic> terminator was PCR amplified from the pANIC6A vector with primers F 5&#8217;-CCTGCTTTAATGAGATATGCGAGA CGC-3&#8217; and R 5&#8217;-CACCAAAACGACGGCCAGTGCCAA G-3&#8217;. The <italic>ZmCCA1b</italic> coding sequence was PCR amplified with primers F 5&#8217;-CACCATGGAGGTGAATTCCTCTGGTGAGGAAAC-3&#8217; and R 5&#8217;-TTATGTTGATGCTTCACTATCAAGACGAATCCTCTT-3&#8217;. The <italic>ZmUBI</italic><sub><italic>pro</italic></sub> cassette was subcloned into the Hind III site upstream of <italic>attR1</italic> site in pTF101.1gw1. The <italic>ZmCCA1b</italic> coding sequence was moved into with LR Clonase II according to the manufacturer&#8217;s recommendations (Thermo-Fisher, Waltham, Massachusetts). The <italic>OCS</italic> terminator was subcloned into the AscI site downstream of the <italic>ZmCCA1b</italic> coding sequence. The Plant Transformation Facility (Iowa State University, Ames, IA) transformed the complete <italic>ZmCCA1b</italic> overexpression binary construct into the B104 maize inbred by Agrobacterium-mediated transformation according to their published protocol [<xref ref-type="bibr" rid="pgen.1006197.ref074">74</xref>]. Pollen from T0 plants, which were confirmed to carry at least copy of the one construct, was used for crosses to the B104 inbred. Transgenic lines were maintained in a hemizygous state by crosses to the B104 inbred.</p><p>Transgenic plants were screened using fluorescent probe-based endpoint qPCR. Genomic DNA was extracted from leaf tissue taken from the newest expanded leaf when plants were between V3 and V8. The presence of ADH and BAR was assessed together in PCR reactions with primers 5&#8217;-TGTTGAGCAGATCTCGGTGAC-3&#8217;and 5&#8217;-GTTTCTGGCAGCTGGACTTC-3&#8217;, with probe 5&#8217;-[HEX]AGGACCGGACGGGGCGGTA[BHQ1]-3&#8217;, for the <italic>bar</italic> gene in pTF101.1gw1; and primers 5&#8217;-GAATGTGTGTTGGGTTTGCAT-3&#8217; and 5&#8217;-TCCAGCAATCCTTGCACCTT-3&#8217;, with probe 5&#8217;-[FAM]TGCAGCCTAACCATGCGCAGGGTA[BHQ1]-3&#8217;, for ADH1, which is a single copy gene in the maize genome [<xref ref-type="bibr" rid="pgen.1006197.ref075">75</xref>]. Samples with PCR amplification for both ADH and BAR were scored as transgenic and those with ADH alone were scored as non-transgenic.</p><p>Field trials took place at Oxford Tract in Berkeley, CA during summer 2015. The field was sown in a randomized complete block design with two trials separated by one week, planted in late May. In the greenhouse, plants were grown under 16L:8D cycle, supplemented by LumiGrow Pro 325 LED lights (LumiGrow, Inc., Novato, CA), with 25&#176;C days and 20&#176;C nights. Two individual trials were replicated in greenhouse conditions. 5 plants of each family along with 5 non-transgenic siblings were maintained for each trial.</p><p>In field trials, weekly height measurements were taken beginning at the V8 stage until final height was reached. In greenhouse trials, weekly height measurements were taken starting when plants were at the V7 stage until final height was reached. Height was measured from prop roots to the collar of the last fully expanded leaf. Node length was measured at the end of the field or greenhouse trial after plants had reached maturity. Measurements were taken from the upper prop roots to the final visible node, or the base of the tassel.</p><p>Plant tissue for chlorophyll analysis was harvested using a Harris Uni-Core 2.5 mm biopsy punch (Ted Pella, Redding, CA). 4 punches were taken midway from the base to the tip of the leaf, equidistant from the mid-vein and the leaf edge. Tissue was immediately placed in 1 mL DMSO and incubated at 65&#176;C for 30 min. Absorbance at 645 nm and 663 nm was used to determine chlorophyll content based on Arnon&#8217;s equation [<xref ref-type="bibr" rid="pgen.1006197.ref076">76</xref>].</p><p>CCA1 protein sequences are AtCCA1 (At2g46830), AtLHY (At1g01060), SbCCA1 (Sb07g003870), OsCCA1 (Os08g0157600), PnLHY1 (AB429410), PnLHY2 (AB429411), BraA.LHY.a (Bra030496), McCCA1 (AY371287) and GmLCL1 (Glyma11g15580), At: <italic>Arabidopsis thaliana</italic>; Sb: <italic>Sorghum bicolor</italic>; Os: <italic>Oryza sativa</italic>; Pn: <italic>Populus nigra</italic>; BraA: <italic>Brassica rapa</italic>; Mc: <italic>Mesembryanthemum crystallinum</italic>; and Gm: <italic>Glycine max</italic>. The protein sequences were aligned using the ClustalW module [<xref ref-type="bibr" rid="pgen.1006197.ref077">77</xref>]. The phylogenetic tree was constructed using the Neighbor-Joining method [<xref ref-type="bibr" rid="pgen.1006197.ref078">78</xref>]. The bootstrap values were calculated with 1,000 replicates and shown next to the branches. The evolutionary distances were computed using the Poisson correction method [<xref ref-type="bibr" rid="pgen.1006197.ref079">79</xref>] with the units of the number of amino acid substitutions per site. All positions containing gaps and missing data were eliminated. A total of 472 positions were analyzed using MEGA6 [<xref ref-type="bibr" rid="pgen.1006197.ref080">80</xref>].</p><p>For gene expression analysis, aerial tissues were harvested and immediately frozen in the liquid nitrogen. Total RNA was extracted from the frozen tissues by PureLink Plant RNA reagent (Invitrogen, Carlsbad, CA) and treated with RQ1 DNaseI (Promega, Madison, WI) according to the manufacturers&#8217; instructions. For cDNA synthesis, 1 &#956;g of DNaseI-treated total RNA was incubated with Omniscript reverse transcriptase (Qiagen, Valencia, CA) in the presence of 10 &#956;M random hexamer (GeneLink, Hawthorne, NY). For qPCR, FastStart Universal SYBR Green Master (ROX) (Roche Applied Science, Indianapolis, IN) was used in the presence of gene-specific primers and template cDNAs in an ABI7500 (Applied Biosystems, Foster City, CA) or LightCycler 96 machine (Roche Applied Science, Indianapolis, IN). The control was <italic>18S rRNA</italic> to estimate the relative expression levels of each gene in three biological replicates. A list of primers for gene expression analysis is provided in <xref ref-type="supplementary-material" rid="pgen.1006197.s013">S4 Table</xref>.</p><p>Photosynthetic rate was measured on fully expanded 2<sup>nd</sup> leaves of seedlings at 12 DAP (n = 9), grown under 16L:8D cycle with natural sunlight, every 2 hours from dawn (ZT0) to dusk (ZT14). A LI-6400XT portable photosynthesis analyzer (LICOR Environmental, Lincoln, NE) equipped with a light source (LI-6400-40) and CO<sub>2</sub> mixer (LI-6400-01) was used to determine net CO<sub>2</sub> assimilation at a reference CO<sub>2</sub> concentration of 400 &#956;mol mol<sup>&#8722;1</sup>, with cuvette conditions set to match ambient conditions at the start of each measurement period. The experiment was reproduced twice, in February and April 2012, respectively. Data from the April experiment is shown. Starch and sucrose contents were measured on source (middle-tip of the 2<sup>nd</sup> leaf) and sink (base of the 2<sup>nd</sup> leaf) of additional replicate plants grown alongside those used for photosynthesis assays. Pooled plants in three biological replicates at ZT14 were used for testing. Leaf discs were collected using an 11 mm-diameter cork borer, weighed and immediately frozen in liquid nitrogen. The frozen tissues were ground, mixed with a homogenization buffer (500 mM MOPS pH 7.5, 5 mM EDTA, 10% ethyl glycol), and then filtered through Miracloth (CalBiochem, San Diego, CA). After centrifugation, pellets were dissolved in DMSO to extract the insoluble carbohydrate fraction, while supernatant was transferred to a new tube as the soluble carbohydrate fraction. The starch content was measured from the insoluble carbohydrate fraction using a commercial assay kit according to the manufacturer&#8217;s instruction (R-Biopharm, Darmstadt, Germany). Sucrose content was measured from the soluble carbohydrate fraction using a commercial assay kit according to the manufacturer&#8217;s instruction (K-SUFRG Megazyme, Bray, Ireland) and as previously described [<xref ref-type="bibr" rid="pgen.1006197.ref009">9</xref>].</p><p>The coding sequence (CDS) of <italic>ZM2G474769</italic> (<italic>ZmCCA1a</italic>) was amplified from B73 cDNA by the primer pair 5&#8217;-<underline>GAATTC</underline>ATGCCCTTGAGCAATGAG-3&#8217; (<italic>Eco</italic>R&#1030;, underlined) and 5&#8217;-<underline>GTCGAC</underline>TCATGTTGATGCTTCACTAT-3&#8217; (<italic>Sal</italic>&#1030;). The full-length <italic>ZmCCA1b</italic> (<italic>ZM2G014902</italic>) CDS was amplified from B73 cDNA by the primer pair 5&#8217;- <underline>GGATCC</underline>ATGGAGGTGAATTCCTCTGGC-3&#8217; (<italic>Bam</italic>H&#1030;) and 5&#8217;- <underline>GTCGAC</underline>TTATGTGGATGCTTCGCTATC-3&#8217; (<italic>Sal</italic>&#1030;). The <italic>ZmCCA1a</italic> or <italic>ZmCCA1b</italic> cDNA fragment was cloned into a pGEM-T (Promega, Madison, Wisconsin). After sequence verification, the <italic>ZmCCA1a</italic> or <italic>ZmCCA1b</italic> CDSs was subcloned into pMAL-C2 (New England BioLabs, Beverly, MA) through <italic>Eco</italic>R&#1030;/<italic>Sal</italic>&#1030; and <italic>BamH</italic>&#1030;/<italic>Sal</italic>&#1030; restriction sites, respectively. <italic>E</italic>. <italic>coli</italic> strain Rosetta-gami B competent cells (Novagen, Madison, WI) was used to transform empty pMAL (expressing maltose-binding protein, MBP), pMAL-ZmCCA1a or pMAL-ZmCCA1b, which were grown in 4 ml of Luria-Bertani (LB) media with Carbenicillin (100 mg/L) at 37&#176;C for 18 h. The overnight cultures of Rosetta-gami B cells containing pMAL, pMAL-ZmCCA1a or pMAL-ZmCCA1b construct were diluted into 1:100 in 80 ml LB media with Carbinicillin (100 mg/L) and grown at 37&#176;C to an OD<sub>600</sub> value of 0.5, when isopropyl-&#946;-D-thiogalactoside (IPTG) (0.1 mM) was added. After 20 h of additional incubation at 16&#176;C, cells were harvested after centrifugation at 4,000 <italic>g</italic> at 4&#176;C for 10 min and resuspended in 2 ml of Column buffer (20 mM Tris-HCl, 200 mM NaCl, 1 mM EDTA). After frozen at -20&#176;C for 18 h, cells were lysed by a Bioruptor Sonicator (Diagenode, Sparta, NJ) and centrifuged at 20,000 <italic>g</italic> at 4&#176;C for 20 min. The cleared cell lysates were diluted 1:5 with Column buffer, and loaded on amylose-coupled agarose resin columns prepared according to the manufacturer&#8217;s instruction (New England BioLabs, Beverly, MA). After columns were washed with 12 volumes of Column buffer, MBP, rZmCCA1a and rZmCCA1b were eluted with elution buffer (20 mM Tris-HCl, 200 mM NaCl, 1 mM EDTA, 10 mM Maltose). After filtration by Amicon Ultra 100 K (Millipore, Darmstadt, Germany), the purified MBP, rZmCCA1a or rZmCCA1b was aliquoted and stored at -80&#176;C.</p><p>ChIP was performed as previously described with following modifications [<xref ref-type="bibr" rid="pgen.1006197.ref081">81</xref>]. Aerial tissues of three biological replicates were harvested at ZT3, ZT9 and ZT15 and were completely submerged in fresh prepared formaldehyde buffer (0.4 M Sucrose, 10 mM Tris-HCl, 1 mM PMSF, 3% Formaldehyde, 5 mM &#946;-mercaptoethanol). A vacuum was applied for 20 min; after adding glycine to a final concentration 125 mM, another vacuum was applied for 5 min. After the formaldehyde/glycine buffer was removed, the cross-linked tissues were washed with sterilized water, briefly dried with paper towels, immediately frozen in liquid nitrogen and stored at -80&#176;C for experimental use. Sonication was performed with 10 cycles of 30 s pulses on and 30 s pulses off to achieve an average fragment size of 400-bp using a Bioruptor Sonicator (Diagenode, Sparta, NJ). Samples were subjected to centrifugation at 13,800 <italic>g</italic> at 4&#176;C for 10 min, and the supernatant containing chromatin was transferred to a new 1.5 ml tube. Immunoprecipitation (IP) was performed using 600 &#956;l of sonicated chromatin with 5 &#956;g of anti-CCA1 antibody. For each IP sample, a mock (no antibody) and input (no IP) were included. Purified DNA (IP, mock and input) from ChIP was resuspended in TE, pH 7.5 and used for qPCR and ChIP-seq library preparation.</p><p>ChIP-seq libraries for ChIP and input samples from two biological replicates were constructed using the standard NEB protocol (New England BioLabs, Beverly, MA) using custom made adapters containing barcodes used to pool multiple samples for sequencing. ChIP DNA was subject to end repair, dA-tailing, ligation with the adapters, and amplification by 18 cycles of PCR using Next High-Fidelity 2XPCR Master Mix (New England BioLabs, Beverly, MA). Pair-end (2X100-bp) sequencing was performed on an Illumina HiSeq 2500 at The University of Texas at Austin Genomic Sequencing and Analysis Facility.</p><p>For qPCR, purified ChIP and mock DNA was diluted 2 times, and input DNA was diluted 5 times. The diluted DNA (2 &#956;l) was used for qPCR in an ABI7500 machine (Applied Biosystems, Foster City, CA) using FastStart Universal SYBR Green Master (ROX) (Roche Applied Science, Indianapolis, IN). Enrichment of the binding in IP and mock samples was normalized relative to the corresponding input sample. A list of primers for ChIP-qPCR analysis is provided in <xref ref-type="supplementary-material" rid="pgen.1006197.s013">S4 Table</xref>.</p><p>Raw reads were subject to quality trimming and adaptor clipping using FASTX-Toolkit (hannonlab.cshl.edu/fastx_toolkit) followed by removing orphan reads. The filtered pair-end reads were mapped to the maize reference genome (Zmays_284_AGPv2, release 5b.60) and Mo17 genome (reference-guided assembly based on the B73 genome) [<xref ref-type="bibr" rid="pgen.1006197.ref044">44</xref>, <xref ref-type="bibr" rid="pgen.1006197.ref082">82</xref>], using Bowtie (version 2.1.0) [<xref ref-type="bibr" rid="pgen.1006197.ref083">83</xref>], allowing 1 mismatch in the 20-bp &#8216;seed&#8217; with options &#8216;&#8212;score-min L,0,-0.3 -X 1000&#8212;no-mixed&#8212;no-discordant&#8217;. Particularly, for B73 and Mo17, the filtered pair-end reads were mapped to the maize B73 reference genome and the Mo17 genome, respectively. For F1 hybrids, the filtered pair-end reads were mapped to both B73 and Mo17 reference genomes, and the best alignment was selected for each read. Only reads concordantly mapping to the genome exactly 1 time was kept and used for peak calling. Duplicate reads were removed from the Bowtie output using samtools rmdup command [<xref ref-type="bibr" rid="pgen.1006197.ref084">84</xref>]. To normalize sequencing depth of the mapped reads among samples from different genotypes and time-points, we adopted a normalization method based on the previously published paper [<xref ref-type="bibr" rid="pgen.1006197.ref052">52</xref>]. In order to adjust different sequencing depths among genotype and time-points, the uniquely and concordantly mapped paired-end reads were &#8220;down-sampled&#8221; to the lowest number of samples without PCR duplicates (<xref ref-type="supplementary-material" rid="pgen.1006197.s010">S1 Table</xref>). For example, if B73, Mo17, BM and MB had 4, 5, 6 and 7 million reads respectively, all samples were down-sampled to 4 million reads. This will ensure that numbers of binding peaks are comparable among genotypes and time-points. The criteria for identifying targets were based on abundance of peak enrichment using ANOVA test (p-value &lt; 0.05 as a cut-off). When target genes showed same abundance 2 or more time-points, they were excluded from the phase-shift analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The latest version of model-based analysis of ChIP-seq algorithm (MACS, version 2.0.1) [<xref ref-type="bibr" rid="pgen.1006197.ref053">53</xref>] was used to identify enriched peaks on each ChIP-seq file against the corresponding input file using a mappable genome size of&#8211;g 2.07e+09 and cut off p-value of 1e-3. Peaks were defined if they overlapped in two biological replicates. To make a master-peak list from the three time-points, the peaks obtained from each time-point were merged for each genotype. Integrated Genomics Viewer (version 2.3.46) [<xref ref-type="bibr" rid="pgen.1006197.ref085">85</xref>] was used to visualize duplicate-filtered input subtracted ChIP signals. For the normalization, modules of the deepTools suite (<ext-link ext-link-type="uri" ns0:href="http://deeptools.ie-freiburg.mpg.de/">http://deeptools.ie-freiburg.mpg.de</ext-link>) [<xref ref-type="bibr" rid="pgen.1006197.ref086">86</xref>] were used. <italic>De novo</italic> motif analysis was performed with the total master-peaks using MEME suite [<xref ref-type="bibr" rid="pgen.1006197.ref087">87</xref>] in which JASPAR CORE (2014) plants database was selected for motif database. GO enrichment analysis was performed using the web interface of agriGO (<ext-link ext-link-type="uri" ns0:href="http://bioinfo.cau.edu.cn/agriGO/">http://bioinfo.cau.edu.cn/agriGO/</ext-link>) [<xref ref-type="bibr" rid="pgen.1006197.ref088">88</xref>] with false-discovery rate adjusted p-value &lt; 0.05 (Hypergeometric test) as a cutoff. Biological process among ontology categories was used. The heatmap for GO analysis was generated using R package gplots. Peak distribution was determined with respect to the maize gene model (release 5b.60).</p><p>To create <italic>CAB2</italic>:<italic>LUC</italic>, a native <italic>Arabidopsis CAB2</italic> (<italic>AT1G29920</italic>) promoter fragment was amplified from Col-0 wild-type genomic DNA by the primer pair 5&#8217;- <underline>CTCGAG</underline>TTATATTAATGTTTCGATCATC-3&#8217; (<italic>Xho</italic>&#1030;) and 5&#8217;- <underline>CCATGG</underline>GTTCGATAGTGTTGGATTATA-3&#8217; (<italic>Nco</italic>&#1030;) and cloned into pGEM-T (Promega, Madison, WI). After sequence validation, the <italic>CAB2</italic> promoter fragment was subcloned into pFAMIR binary vector (Basta resistance), which was fused with <italic>LUC</italic> CDS, via <italic>Xho</italic>&#1030;/<italic>Nco</italic>&#1030; restriction sites. The <italic>CAB2</italic>:<italic>LUC</italic> construct was transformed into the Col-0 and WS wild-types, respectively, using a floral dip method [<xref ref-type="bibr" rid="pgen.1006197.ref089">89</xref>]. Homozygous T3 lines in each background were selected for the following analysis. To generate <italic>CCA1-OX</italic>, <italic>ZmCCA1-OX</italic> and <italic>ZmCCA1b-OX</italic> transgenic lines, <italic>CCA1</italic>, <italic>ZmCCA1a</italic> and <italic>ZmCCA1b</italic> cDNAs were amplified by the following primer pairs, respectively, <italic>CCA1</italic> 5&#8217;-<underline>CTCGAG</underline>ATGGAGACAAATTCGTCTGG-3&#8217; (<italic>Xho</italic>&#1030;) and 5&#8217;- <underline>GGATCC</underline>TCATGTGGAAGCTTGAGTTTC-3&#8217; (<italic>Bam</italic>H&#1030;), <italic>ZmCCA1a</italic> 5&#8217;- <underline>CTCGAG</underline>ATGCCCTTGAGCAATGAG-3&#8217; (<italic>Xho</italic>&#1030;) and 5&#8217;- GGATCCTCATGTTGATGCTTCACTATC-3&#8217; (<italic>Bam</italic>H&#1030;), and <italic>ZmCCA1b</italic> 5&#8217;- CAAG<underline>CTCGAG</underline>ATGGAGGTGAATTCCTCTGGC-3&#8217; (<italic>Xho</italic>&#1030;) and 5&#8217;- GGATCCTTATGTGGATGCTTCGCTATC-3&#8217; (<italic>Bam</italic>H&#1030;), and cloned into pGEM-T (Promega, Madison, WI). After sequence validation, <italic>CCA1</italic> CDS, <italic>ZmCCA1a</italic> CDS or <italic>ZmCCA1b</italic> CDS was subcloned into pF35SE binary vector (conferring Kanamycin resistance) through <italic>Xho</italic>&#1030;/<italic>Bam</italic>H&#1030; restriction sites. The resulting constructs designated <italic>CCA1-OX</italic>, <italic>ZmCCA1a-OX</italic> and <italic>ZmCCA1b-OX</italic>, respectively. The construct of vector control, <italic>CCA1-OX</italic>, <italic>ZmCCA1-OX</italic> or <italic>ZmCCA1b-OX</italic> was introduced into the transgenic Col-0 plants that express <italic>CAB2</italic>:<italic>LUC</italic>. T2 lines were used for the luciferase and biomass analysis. For the <italic>cca1-11</italic> complementation analysis with maize <italic>CCA1</italic> homologs, a native <italic>Arabidopsis CCA1</italic> (<italic>AT2G46830</italic>.<italic>1</italic>) promoter fragment was amplified from WS wild-type genomic DNA by the primer pair, 5&#8217;- <underline>GAATTC</underline>GCCACGTCCTTCCTTCAATC-3&#8217; (<italic>Eco</italic>R&#1030;) and 5&#8217;- <underline>CTCGAG</underline>CACTAAGCTCCTCTACACAA-3&#8217; (<italic>Xho</italic>&#1030;), and cloned into pGEM-T (Promega, Madison, WI). After sequence validation, the <italic>CCA1</italic> promoter fragment was subcloned into pFAMIR binary vector (Hygromycin resistance), via <italic>Eco</italic>R&#1030;/<italic>Xho</italic>&#1030; restriction sites. To create <italic>CCA1</italic>:<italic>CCA1</italic>, the <italic>CCA1</italic> CDS was amplified from the WS cDNA by the primer pair, 5&#8217;- <underline>ATTTAAAT</underline>ATGGAGACAAATTCGTCTGG-3&#8217; (<italic>Swa</italic>&#1030;) and 5&#8217;- <underline>GGATCC</underline>TCATGTGGAAGCTTGAGTTTC-3&#8217; (<italic>Bam</italic>H&#1030;), and cloned into pGEM-T (Promega, Madison, WI). To generate <italic>CCA1</italic>:<italic>ZmCCA1a</italic>, the <italic>ZmCCA1a</italic> CDS was amplified from <italic>Z</italic>. <italic>mays</italic> B73 cDNA by the primer pair, 5&#8217;- <underline>CTCGAG</underline>ATGCCCTTGAGCAATGAG-3&#8217; (<italic>Xho</italic>&#1030;) and 5&#8217;- <underline>GGATCC</underline>TCATGTTGATGCTTCACTATC-3&#8217; (<italic>Bam</italic>H&#1030;), and cloned into the pGEM-T (Promega, Madison, WI). To generate <italic>CCA1</italic>:<italic>ZmCCA1b</italic>, the <italic>ZmCCA1b</italic> CDS was amplified from <italic>Z</italic>. <italic>mays</italic> B73 cDNA by the primer pair, 5&#8217;- CAAG<underline>CTCGAG</underline>ATGGAGGTGAATTCCTCTGGC-3&#8217; (<italic>Xho</italic>&#1030;) and 5- ACCGGATCCTTATGTGGATGCTTCGCTATC-3&#8217; (<italic>Bam</italic>H&#1030;), and cloned into the pGEM-T (Promega, Madison, WI). After sequence validation, <italic>CCA1</italic> CDS, <italic>ZmCCA1a</italic> CDS or <italic>ZmCCA1b</italic> CDS was subcloned into the pFAMIR (Hygromycin resistance), which harbors the <italic>CCA1</italic> promoter fragment, through the respective restriction sites, generating <italic>CCA1</italic>:<italic>CCA1</italic>, <italic>CCA1</italic>:<italic>ZmCCA1a</italic>, and <italic>CCA1</italic>:<italic>ZmCCA1b</italic> constructs, respectively. The construct of <italic>CCA1</italic>:<italic>CCA1</italic>, <italic>CCA1</italic>:<italic>ZmCCA1a</italic> or <italic>CCA1</italic>:<italic>ZmCCA1b</italic> was transformed into <italic>A</italic>. <italic>thaliana cca1-11</italic> plants (obtained from the Arabidopsis Biological Resources Center, ABRC, CS9865), which express <italic>CAB2</italic>:<italic>LUC</italic>, generating <italic>CCA1</italic>:<italic>CCA1 cca1-11</italic>, <italic>CCA1</italic>:<italic>ZmCCA1a cca1-11</italic> or <italic>CCA1</italic>:<italic>ZmCCA1b cca1-11</italic> transgenic lines, respectively. T1 transgenic lines were selected for the luciferase analysis.</p><p>DNA probes were generated by annealing PAGE-purified sense and antisense oligonucleotides (<xref ref-type="supplementary-material" rid="pgen.1006197.s014">S5 Table</xref>). The double-stranded oligonucleotides were [<sup>32</sup>P] end-labeled using a T4 Polynucleotide Kinase according to the manufacturer&#8217;s instruction (New England BioLabs, Beverly, MA). The recombinant proteins (0.5 to 2 pmol) were mixed with 20 fmol of the radiolabeled probes, without or with variable amounts of unlabeled competitor DNA in reaction buffer (25 mM HEPES-KOH pH7.5, 2.5 mM DTT, 75 mM KCl, 10% glycerol, 1.25 ng poly-dIdC). Each reaction was incubated on room temperature for 10 min without the probes and then incubated on ice for 20 min with the radiolabeled probes. The competitor concentrations were at 0 (-), 0.5 (25X), 1 (50X) and 2 (100X) pmol. After the incubation, the reaction mixtures were resolved by electrophoresis on a 5% non-denaturing polyacrylamide gel. Gels were dried in a gel dryer (Bio-Rad, Richmond, CA) and exposed to X-ray film (Kodak, Rochester, NY).</p><p>Total crude protein was extracted from the aerial tissues of seedlings at 5 DAP using an extraction buffer (50 mM Sodium Phosphate pH 8.0, 1 mM EDTA, 5 mM DTT, 10% Glycerol). 30 &#956;g of total crude proteins were resolved in 10% SDS-PAGE gel and transferred to PVDF membrane (GE Healthcare, Piscataway, NJ). ZmCCA1s were detected using anti-CCA1 antibody (1:2,000) that was generated against the N-terminus MYB DNA-binding domain (epitope: residues 11&#8211;77 of ZmCCA1s, <xref ref-type="supplementary-material" rid="pgen.1006197.s002">S2B Fig</xref>), followed by HRP-conjugated anti-rabbit secondary antibody (1:20,000) (12&#8211;348 Upstate, Lake Placid, NY). Loading control was detected using an anti-H3 antibody (1:5,000) (Ab1791 Abcam, Cambridge, MA). Immunoreactive protein was visualized on X-ray film (Kodak, Rochester, NY) using SuperSignal West Pico Substrate (Thermo Scientific, Waltham, MA)</p>