<p xmlns:ns0="http://www.w3.org/1999/xlink">Unless otherwise stated, all analyses were performed with in-house bioinformatics pipelines written in SAS, R, Perl, or Java. Source code for the various scripts is included in <xref ref-type="supplementary-material" rid="pgen.1004845.s008">S3 Dataset</xref>. All analyses were done with using the maize B73 genome (version AGPv2) as reference. The maize filtered gene set was taken from maizesequence.org and is available at <ext-link ext-link-type="ftp" ns0:href="ftp://ftp.gramene.org/pub/gramene/maizesequence.org/release-5b/filtered-set/ZmB73_5b_WGS_to_FGS.txt">ftp://ftp.gramene.org/pub/gramene/maizesequence.org/release-5b/filtered-set/ZmB73_5b_WGS_to_FGS.txt</ext-link>. [verified 13 Oct 2014]</p><p>The NAM population was planted in Aurora, New York, USA in May 2007. Samples were all taken within one week at the beginning of August (when most NAM lines are flowering) between 10:00 AM and 2:00 PM on the sampling date. Two samples were taken from each row (RIL), one from the end plant and the other from four middle plants (&#8764;12,000 raw samples total). Tissue was punched in the base part of the first leaf below the flag leaf and immediately frozen in liquid nitrogen, then stored at &#8722;80&#176;C until extraction.</p><p>&#8764;50 mg (fresh weight) of tissue was extracted twice with 80% ethanol and once with 50% ethanol as in Geigenberger et al. <xref ref-type="bibr" rid="pgen.1004845-Geigenberger1">[36]</xref> (the final volume of each was 650 &#181;l). Protein and starch were extracted from the pellet with 100 mM NaOH <xref ref-type="bibr" rid="pgen.1004845-Hendriks1">[37]</xref> and measured according to established protocols <xref ref-type="bibr" rid="pgen.1004845-Hendriks1">[37]</xref>, <xref ref-type="bibr" rid="pgen.1004845-Bradford1">[38]</xref>. Immediately after extraction, chlorophyll content was determined using the protocol in Arnon <xref ref-type="bibr" rid="pgen.1004845-Arnon1">[39]</xref>. Total free amino acids were assayed using fluorescamine <xref ref-type="bibr" rid="pgen.1004845-BantanPolak1">[40]</xref>. Nitrate levels were quantified as in Tschoep <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1004845-Tschoep1">[41]</xref>, while malate and fumarate were measured as described in Nunes-Nesi <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1004845-NunesNesi1">[42]</xref>. Glutamate was determined by pipetting 10 &#181;l aliquots of extract or standards (0&#8211;20 nmol) into a microplate with 100 mM Tricine/KOH pH 9, 3 mM NAD+, 1 mM methylthiazolyldiphenyl-tetrazolium bromide, 0.4 mM phenazine ethosulphate and 0.5% v/v Triton X-100. The absorbance at 570 nm was read for 5 min, then 1 U of glutamate dehydrogenase was added and the absorbance monitored until it reached stability. Sucrose, glucose, and fructose (in ethanolic extracts) were determined as per Jelitto <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1004845-Jelitto1">[43]</xref>. All assays were prepared in 96-well polystyrene microplates using a JANUS automated workstation robot (Perkin-Elmer, Zaventem, Belgium). Absorbances at 340 or 570 nm were read in either an ELX-800 or an ELX-808 microplate reader (Bio-Tek, Bad Friedrichshall, Germany). A Synergy microplate reader (Bio-Tek, Bad Friedrichshall, Germany) was used to determine absorbances at 595, 645 or 665 nm and fluorescence (405 nm excitation, 485 nm emission).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Best linear unbiased predictors (BLUPs) for each line within each trait were calculated using ASReml (version 2.0; <ext-link ext-link-type="uri" ns0:href="http://www.vsni.co.uk/software/asreml">http://www.vsni.co.uk/software/asreml</ext-link>). The final BLUPs are the result from controlling for several potential confounding factors, specifically: spatial effects within the field; the level of nitrogen, phosphorous and potassium in the soil before planting; the tissue sampling date and time; the researcher who performed the sampling; the batch effect of the plate samples were stored in; and the batch effect of the plate the measurements occurred in. BLUPs were also calculated for flowering time (defined as the time from sowing to when 50% of plants in a row are shedding pollen), correcting for the spatial field effects. Most metabolites correlate with flowering time, so we used Proc GLM in SAS (<ext-link ext-link-type="uri" ns0:href="http://www.sas.com/">http://www.sas.com/</ext-link>) to regress the values for all 12 metabolites on flowering time. Partial correlation analysis (part of Proc GLM) was used to confirm that resulting values were significantly different, and a Holm-Bonferroni correction <xref ref-type="bibr" rid="pgen.1004845-Holm1">[44]</xref> at &#945;&#8202;=&#8202;0.05 was used to correct for multiple testing. Principal components were calculated with Proc PRINCOMP in SAS after fitting a linear model to account for the effect of flowering time (days to anthesis) as a covariate. (That is, the principal components are of the residuals after factoring out flowering time.)</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Phenotype data for GWAS analysis was taken from previous studies by our lab and others on a variety of traits, along with the metabolite data included herein (<xref ref-type="table" rid="pgen-1004845-t001">Table 1</xref>). In the majority of cases phenotypic data had already been processed by fitting a joint-linkage model <xref ref-type="bibr" rid="pgen.1004845-Li2">[45]</xref> with 1,106 high-confidence SNP markers across NAM. Chromosome-specific residuals were then determined by fitting a model that included as covariates all identified quantitative trait loci (QTL) except those on the given chromosome. For traits without precomputed residuals, the same process was followed but with an updated list of &#8764;7,000 SNPs derived from genotyping-by-sequencing <xref ref-type="bibr" rid="pgen.1004845-Elshire1">[22]</xref>. All genotypes are available at <ext-link ext-link-type="uri" ns0:href="http://www.panzea.org">http://www.panzea.org</ext-link>; chromosome-specific residuals are included in <xref ref-type="supplementary-material" rid="pgen.1004845.s007">S2 Dataset</xref>.</p><p>Forward-regression genome-wide association was then performed with the NamGwasPlugin in TASSEL version 4.1.32 <xref ref-type="bibr" rid="pgen.1004845-Bradbury1">[46]</xref>. This plugin was created specifically to run stepwise forward regression on the Maize NAM population, and takes as input the chromosome-specific residuals, a genetic map, anchor genotypes in the progeny, and founder genotypes to be imputed. Each chromosome was analyzed separately for each phenotype via 100 forward-regression iterations, each of which excluded a random 20% of NAM lines to destabilize spurious associations <xref ref-type="bibr" rid="pgen.1004845-Valdar1">[47]</xref>. The cutoff for polymorphism inclusion in the model was a raw p-value &lt;9.50&#215;10<sup>&#8722;8</sup>, which was empirically determined by permutation testing with the days to anthesis phenotype to correspond to a genome-wide Type I error rate of 0.01. The resample model inclusion probability (RMIP) <xref ref-type="bibr" rid="pgen.1004845-Valdar1">[47]</xref> of each polymorphism was determined as the proportion of iterations in which a specific polymorphism was called as significant; only polymorphisms with an RMIP &#8805;0.05 are considered in this study.</p><p>The input SNPs were a union of all SNPs in Hapmap1 and Hapmap2. A small portion of SNPs are duplicated between the datasets&#8212;that is, they were independently discovered in both studies&#8212;but in almost every case they have different (and sometimes conflicting) allele calls. Thus we made no attempt to merge such SNPs and instead let each original call be tested individually. After running GWAS, we found a single case of ambiguity in determining which SNP had been chosen by the model, due to two SNPs having identical positions and allele codings. In this case we retained both to maintain consistency with the input dataset.</p><p>Putative CNVs were determined by two methods. First, Hapmap2 sequencing reads aligned to the maize genome were counted in 2 kb-windows and compared to a high-coverage B73 sample with edgeR <xref ref-type="bibr" rid="pgen.1004845-Robinson1">[48]</xref>. This procedure had been done previously <xref ref-type="bibr" rid="pgen.1004845-Chia1">[21]</xref>, and our analysis was primarily to update the results to a newer version of the <italic>Zea mays</italic> reference genome (AGPv2). The B73 sample from Hapmap2 itself served as the null distribution to determine the cutoff corresponding to an empirical, genome-wide Type I error rate of 0.05. CNVs that had been previously identified within annotated genes by the same method <xref ref-type="bibr" rid="pgen.1004845-Chia1">[21]</xref> were also included in the analysis but with updated gene coordinates based on their stable Ensembl gene identifiers.</p><p>Independently, the mapped reads were also analyzed by CNVnator <xref ref-type="bibr" rid="pgen.1004845-Abyzov1">[49]</xref> to identify putative CNVs based on shifts in mean read depth across 500 bp bins. Interestingly, although many CNVnator CNVs showed consistent segregation across the NAM founders, GWAS hits came almost exclusively from the edgeR-derived CNVs. Looking at the characteristics of each, this disparity is probably due to two factors: (1) the edgeR-derived CNVs are generally much smaller than those found by CNVnator, and smaller CNVs have previously been shown to have more significant GWAS hits in this population <xref ref-type="bibr" rid="pgen.1004845-Chia1">[21]</xref>; and (2) edgeR also detects many more CNVs than CNVnator to begin with, presumably because small CNVs are more common than large ones.</p><p>Since there were a total of three separate sources of CNVs in this analysis, a single genome region could potentially contribute to multiple CNVs and thus be tested multiple times per GWAS run. The contribution to each set of CNVs will be different, however, since each one depends on all regions within its limits and is then collapsed to a single score of 0 or 1.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Putative SNP effects were determined by running all 28.9 million SNPs through the Ensembl Variant Effect Predictor (VEP) <xref ref-type="bibr" rid="pgen.1004845-McLaren1">[23]</xref> using a local copy of the <italic>Zea mays</italic> Ensembl database (version 68). Since the VEP annotates effects relative to any gene model (not just quality-filtered ones), it was run with both the &#8220;&#8211;most-severe&#8221; and &#8220;&#8211;per-gene&#8221; options to get lists of the worst overall effect per SNP and the worst per gene, respectively. (Note that the VEP considers that changing an existing amino acid is more severe than in-frame insertions and deletions, so small indels that do both get classified as &#8220;missense.&#8221; These make up &lt;0.1% of the input polymorphisms and only 3 GWAS-hit ones, however, so altering the annotation would not significantly affect the results.) The two results were then combined with in-house Perl scripts to create a list of the worst overall SNP effect with respect to only those genes in the <italic>Zea mays</italic> 5b.60 filtered gene set (available at <ext-link ext-link-type="uri" ns0:href="http://www.gramene.org">http://www.gramene.org</ext-link>).</p><p>Polymorphisms classes were tallied for both the input SNPs dataset and for the GWAS-hit SNPs. Using the input dataset as the null, we then removed any categories with &lt;5 expected counts in the GWAS dataset. Total counts in the remaining groups were then tested for significance by a Chi-square test using the Stats package in R <xref ref-type="bibr" rid="pgen.1004845-R1">[50]</xref>. Individual categories were then tested for enrichment by a two-sided exact binomial test, also in R.</p><p>Due to the possibility that linkage disequilibrium could distort the results from the above test, we also ran 1 million circular permutations of the hits to generate a null distribution of what would be expected by chance. Circular permutation in this case refers to keeping the order of all elements intact while randomly changing the &#8220;start&#8221; location along the chromosome. This maintains the structure of the original data while randomizing its relationship to genomic features. The resulting counts formed a normal distribution, which was used to extrapolate the p-values in <xref ref-type="supplementary-material" rid="pgen.1004845.s004">S1 Table</xref>.</p><p>Marginal variance explained by polymorphisms classes (genic, gene-proximal, intergenic, and CNVs) was calculated by fitting linear models to each trait and comparing the difference in variance explained (adjusted R<sup>2</sup>) between a model with all identified SNPs and a model with all SNPs except those in the chosen category.</p><p>Standardized effect sizes for each polymorphism were determined by first taking all effect sizes the NAM-GWAS model identified for each trait and fitting an empirical cumulative distribution function with ecdf() in R <xref ref-type="bibr" rid="pgen.1004845-R1">[50]</xref>. This function was then used to determine the quantile of each effect. Mean quantile scores were then calculated for each polymorphism that passed RMIP&#8805;0.05 filtering. Each point in the distribution thus represents a specific trait-polymorphism combination.</p><p>Gene Ontology term analysis was performed with agriGO <xref ref-type="bibr" rid="pgen.1004845-Du1">[51]</xref> using all genes with GWAS hits within 5 kb of their annotated transcript. Statistical analysis was performed in R <xref ref-type="bibr" rid="pgen.1004845-R1">[50]</xref> via a two-sided Fisher's exact test with Benjamini-Yekutieli control of the false discovery rate (FDR) to analyze for both enrichment and depletion.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Maize paralogs were taken from an existing list <xref ref-type="bibr" rid="pgen.1004845-Schnable1">[52]</xref> (available at <ext-link ext-link-type="uri" ns0:href="http://genomevolution.org/CoGe">http://genomevolution.org/CoGe</ext-link>). The number of genes with paralogs in the GWAS hit dataset was compared to those in the maize filtered gene set and significance of the difference tested by a two-sided exact binomial test in R <xref ref-type="bibr" rid="pgen.1004845-R1">[50]</xref>.</p>