<p>All patient sample collection was approved by the institutional review board of the participating facility. All de-linked and de-identified patient sample information was collected under the approved institutional review board of the National Cancer Institute protocol 10-C-0086. Samples were assembled from collections at the Pediatric Oncology Branch of the National Cancer Institute, Children&#8217;s Oncology Group, and the Tumor Bank at The Children&#8217;s Hospital at Westmead (New South Wales, Australia). All tumors were collected at initial diagnosis and prior to any therapy with the exception of samples NCI0040 and NCI0080 which were collected at relapse. Samples were de-identified and histologic diagnosis and clinical information were compiled. Quality control genotyping for the whole genome samples was performed to ensure the match of tumor normal pairs.</p><p>Approximately 6 micrograms of DNA was sequenced using the paired end sequencing method of Complete Genomics. Data analysis was accomplished using CGA tools package v2.0 as well as a number of in house tools described in [<xref ref-type="bibr" rid="pgen.1005075.ref023">23</xref>].</p><p>A custom panel of oligonucleotides designed to incorporate somatic single nucleotide variants discovered in samples RMS2110 and RMS2107 was generated using Ion Ampliseq designer software. 150 base pair amplicon libraries were generated using multiplex PCR according to the Ion Torrent Ampliseq Library 2.0 kit. Individual samples were barcoded and the generated libraries were sequenced using a 318 chip on a Personal Genome Machine (Life Technologies). Compiled reads were mapped to Hg19 and the expected variants were analyzed for coverage and VAF. The targeted sequencing has a depth of 1997x. We used it to verify the VAF estimates of somatic mutations across whole genome of two samples. The result shows VAF estimation accuracy is as high as 90% (<xref ref-type="supplementary-material" rid="pgen.1005075.s002">S1 Fig.</xref>).</p><p>Illumina Omni 2.5M (97 paired plus 30 unmatched tumors) or 5M (10 paired samples) were performed according to the standard procedure from the manufacturer (Illumina, San Diego, CA) at the National Cancer, Cancer Genomics Research Laboratory. The data were previously reported in [<xref ref-type="bibr" rid="pgen.1005075.ref023">23</xref>].</p><p>PolyA selected RNA libraries were prepared for RNA sequencing on Illumina HiSeq2000. 100 bases long paired-end reads were assessed for quality and reads were mapped using CASAVA (Illumina, San Diego, CA). The generated fastq files were analyzed by TopHat2 [<xref ref-type="bibr" rid="pgen.1005075.ref058">58</xref>] and Cufflinks [<xref ref-type="bibr" rid="pgen.1005075.ref059">59</xref>] The data were previously reported in [<xref ref-type="bibr" rid="pgen.1005075.ref023">23</xref>].</p><p>We used the somatic score from Complete Genomics (CG), a highly sensitive and specific somatic mutation-calling criteria [<xref ref-type="bibr" rid="pgen.1005075.ref023">23</xref>,<xref ref-type="bibr" rid="pgen.1005075.ref060">60</xref>,<xref ref-type="bibr" rid="pgen.1005075.ref061">61</xref>], combined with principles previously used [<xref ref-type="bibr" rid="pgen.1005075.ref025">25</xref>,<xref ref-type="bibr" rid="pgen.1005075.ref062">62</xref>] to detect somatic signal nucleotide variants (SNVs). The somatic score is designed by systematically considering sequencing error, mapping error and read count and has been validated as an effective criterion by our previous studies as well as studies from other groups [<xref ref-type="bibr" rid="pgen.1005075.ref023">23</xref>,<xref ref-type="bibr" rid="pgen.1005075.ref061">61</xref>,<xref ref-type="bibr" rid="pgen.1005075.ref063">63</xref>]. To ensure sensitivity and specificity of mutation calling, we set the somatic score cutoff as &#8805;0, with a set of additional filters for removing system artifacts and mapping errors, to select somatic mutations, based on the verification results from an independent sequencing platform [<xref ref-type="bibr" rid="pgen.1005075.ref023">23</xref>]. By SOLiD whole exome sequencing (35x coverage) and Ion Torrent Ampliseq targeted sequencing (300x coverage) on 30 RMS tumor-normal sample pairs sequenced by CG, we verified that our filtering removes 99% of false positives while maintaining 80% of true positives [<xref ref-type="bibr" rid="pgen.1005075.ref023">23</xref>] in CG&#8217;s comprehensive somatic mutation pool (called in a loose criterion to be as inclusive as possible) [<xref ref-type="bibr" rid="pgen.1005075.ref023">23</xref>,<xref ref-type="bibr" rid="pgen.1005075.ref060">60</xref>], which is premium given the less somatic mutations in pediatric tumors [<xref ref-type="bibr" rid="pgen.1005075.ref003">3</xref>].</p><p>For the 44 matched tumor and normal sample pairs with whole genome sequencing, we used somatic copy number segmentation profile provided by Complete Genomics, with customized corrections. For a larger cohort, with 120 matched tumor and normal sample pairs measured by SNP array, we used the copy number profile provided by NEXUS copy number analysis.</p><p>In copy number profiling, an important quantity, lesser allele fraction (LAF), is estimated to represent the fraction of copies coming from each parental allele. Along with total copy number, LAF tells us the allelic copy number status. LAF is defined as the ratio between the copy number of the lesser allele (with fewer copies than the other allele) versus the total copy number. For example, a trisomic chromosome with allelic type &#8220;AAB&#8221; has LAF equal to 1/3. LAF is estimated based on germline heterozygous single nucleotide variants that are present in cancer genome. For each genomic segment of tumor sample, sites that have heterozygous single nucleotide variants in matched normal samples were selected and LAF was estimated by the ratio between the read count of lesser allele and the total read count for these sites.</p><p>In order to study the evolutionary history of RMS, we performed an integrated statistical analysis, using information including germline single-nucleotide variants, somatic mutations (single-nucleotide variants), somatic copy number alterations (CNA) and junctions (mapped breaking points), to estimate the normal cell contamination, intra-tumor heterogeneity (due to subclones) and the timing of somatic variants. The methods are detailed in the <xref ref-type="supplementary-material" rid="pgen.1005075.s001">S1 Text</xref>.</p><p>Mouse myoblast cell line C2C12 was obtained as a generous gift from Dr. Marc Landanyi. The cells genotype was performed by the NCI Core Genotyping facility and the cell line was confirmed to be myocoplasm negative. All cell culture was performed in DMEM supplemented with 10% FBS. The differentiation assay was performed using 2% horse serum (Life Technologies) as previously described [<xref ref-type="bibr" rid="pgen.1005075.ref033">33</xref>]. For retrovirus production, the pBabe vector system (Addgene) was used.</p><p>Plasmids encoding human <italic>PKN1 c</italic>DNA was purchased from Addgene, and constructs were subcloned into pBABE containing a N-terminal HA tag. A cDNA encoding the PKN1 E216K mutant was generated using the GeneART Site-Directed Mutagenesis kit (Life Technologies) and was subcloned into pBabe. The generated mutation was confirmed by Sanger sequencing. Retroviruses were generated by contransfection of pBabe constructs with pCL-10A1 into 293T cells (American Type Culture Collection (ATCC), CRL-3216) subsequently used to infect the C2C12 cell line as previously described; infected cells were selected with 2 &#956;g/ml puromycin (Life Technologies).</p><p>Cells grown on Nunc chamber slides were fixed with 4% paraformaldehyde and permeabilized in PBS containing 0.5% Triton X-100 and blocked in Block-Aid (Life Technologies) for 1 h at room temperature. Cells were then incubated with MF20 monoclonal antibody (DSHB) against MHC (1:40 dilution;) overnight at 4 degrees. Secondary antibody Alexa Fluor 488-conjugated secondary antibody (1:200 dilution; Life Technologies) for 1 h at room temperature. Cells were mounted with ProLong Gold antifade reagent with DAPI (4&#8242;,6-diamidino-2-phenylindole; Life Technologies).</p><p>Cells were lysed in M-PER lysis buffer (Pierce Biotechnology). Lysates were denatured in 4&#215; sample buffer at 70&#176;C for 10 min, resolved on 4&#8211;12% NuPAGE gels (Life Technologies) and transferred onto PVDF (polyvinylidene fluoride) membranes. Membranes were blocked in 5% nonfat milk in TBST buffer (TBS with Tween-20) for 1 h at room temperature and probed with primary anti-HA antibody obtained from Covance (1:2,000 dilution). Bound antibodies were detected with peroxidase-labeled horse antibody to mouse IgG and visualized using enhanced chemiluminescence reagents (ThermoScientific).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Total cellular RNA was isolated using the RNAeasy mini kit (Qiagen). Cellular RNA (250ng) was in vitro transcribed, fragmented, hybridized and applied to Affymetrix Mouse 430A arrays according to the standard operating procedure of the Laboratory of Molecular Technology core facility (<ext-link ext-link-type="uri" ns0:href="http://atp.ncifcrf.gov/genetics-and-genomics/laboratory-of-molecular-technology">http://atp.ncifcrf.gov/genetics-and-genomics/laboratory-of-molecular-technology</ext-link>) and the manufacturer&#8217;s instructions (Affymetrix, Santa Clara, CA). For gene set enrichment analysis (GSEA), the normalized gene expression data were z-scored and ranked according to absolute fold-change expression over the control. GSEA analysis (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/gsea/index.jsp">http://www.broadinstitute.org/gsea/index.jsp</ext-link>) was performed using default parameter settings.</p>