<p>For family PKDF406, screening for linkage to the reported recessive deafness loci was performed using at least three short tandem repeat (STR) markers each for these loci. Data was analyzed using GeneMapper software (Applied Biosystems). LOD scores were calculated using a recessive model of inheritance assuming a fully penetrant disorder and a disease allele frequency of 0.001.</p><p>For family LS06, genomic DNA was extracted from fibroblast cell lines using the Gentra-Puregen kit (Qiagen) according to the manufacturer&#8217;s instructions. Exon capture and enrichment was performed (Illumina TruSeq Exome) in solution and libraries were sequenced as 100 bp paired end reads on an Illumina HiSeq 2000 instrument. Mapping to the human reference sequence hg19 and variant calling was performed using CLC bio genomics workbench 6.5.1 employing the &#8220;default stand-alone mapping&#8221; algorithm. Variant calling parameters were minimally modified with a neighborhood radius of 9 bp, minimum allele frequency of 35% and minimum coverage of 4 bidirectional reads (Mapping statistics are summarized in <xref ref-type="supplementary-material" rid="pgen.1005097.s007">S7 Table</xref>). Coding region and splice junction variants (6bp) common to probands II.1 and II.3 were filtered using the Annovar &#8220;filtered annotation&#8221; functions tool [<xref ref-type="bibr" rid="pgen.1005097.ref059">59</xref>]. We discarded variants with Minor Allele Frequency (MAF) &#8805; 0.01 present in the &#8220;1000 genome project April 2012&#8221;, &#8220;dbSNP138-non flagged&#8221; and &#8220;ensembl annotations popfreq all&#8221; databases. We then selectively analyzed variants compatible with X-linked or autosomal recessive inheritance (<xref ref-type="supplementary-material" rid="pgen.1005097.s003">S3 Table</xref>).</p><p>For family PKDF406 whole exome sequencing analysis used a genomic DNA sample from one affected individual (NimbleGen SeqCap EZ Exome Library v2.0, Roche) and 100bp paired-end sequencing was performed on an Illumina HiSeq2000 instrument. Sequencing data were analyzed following the guidelines that are outlined in the Broad Institute&#8217;s Genome Analysis Toolkit [<xref ref-type="bibr" rid="pgen.1005097.ref060">60</xref>,<xref ref-type="bibr" rid="pgen.1005097.ref061">61</xref>]. The raw data were mapped using the Burrows Wheeler Aligner [<xref ref-type="bibr" rid="pgen.1005097.ref061">61</xref>], the variants were called using the Unified Genotyper, and the data underwent further processing and quality control [<xref ref-type="bibr" rid="pgen.1005097.ref060">60</xref>,<xref ref-type="bibr" rid="pgen.1005097.ref061">61</xref>]. Low-quality reads (less than 10x coverage) were removed, and the remaining variants were filtered against the dbSNP133 database and all of the known variants in the NHLBI 6500 Exome Variant database that had a minor allele frequency (MAF) of greater than 0.05% (<xref ref-type="supplementary-material" rid="pgen.1005097.s003">S3 Table</xref>). We used Sanger sequencing to analyze the segregation of alleles in the other family members. Primers were designed with Primer3 software [<xref ref-type="bibr" rid="pgen.1005097.ref062">62</xref>] to amplify exons as well as flanking introns and untranslated regions (<xref ref-type="supplementary-material" rid="pgen.1005097.s004">S4 Table</xref>). PCR amplified products were purified with exonuclease and alkaline phosphatase (Fermentas) treatments. Purified products were then sequenced with BigDye v3.1 (Applied Biosystems) and run on an Applied Biosystems 3730xl DNA Analyzer.</p><p>Post-natal day 0 (P0), P30 and P90 C57BL/6J mice were used. The mice were obtained from Jackson Laboratories and bred in Cincinnati Children&#8217;s Hospital Medical Center (CCHMC) animal facility.</p><p>Two homology models of NARS2 were constructed, using PYMOL (see <xref ref-type="sec" rid="sec036">Web Resources</xref>). The templates, used as backbone, were the crystal structure of <italic>Pyrococcus horikoshii</italic> AsnRS [<xref ref-type="bibr" rid="pgen.1005097.ref035">35</xref>] (Protein Data Bank access # 1X55) and the dimeric protein form of mt-AsnRS of <italic>Entamoeba histolytica</italic> (Protein Data Bank access # 3M4Q).</p><p>Mouse inner ear tissues were harvested from three to five mice at P0, P30, and P90. The vestibular system and the cochlea were separately dissected and immediately frozen in TriReagent (Ambion). RNA were isolated with RiboPure kit (Ambion) and used to synthetize cDNA (SuperScriptII Reverse Transcriptase, Life Technologies). Inter-exonic primers (<xref ref-type="supplementary-material" rid="pgen.1005097.s005">S5 Table</xref>) were designed with Primer3 software and PCR amplifications were performed with EconoTaq (Lucigen). Primers that amplify <italic>Gapdh</italic> cDNA were used as an internal control (<xref ref-type="supplementary-material" rid="pgen.1005097.s005">S5 Table</xref>). Amplimers were size-separated on a 2% agarose gel and stained with ethidium bromide.</p><p>
<italic>In situ</italic> hybridization was performed on mouse cochlear cryosections at post-natal day 2 (P2). <italic>Nars2 in situ</italic> hybridization probes were subcloned from C57BL/6J inner ear cDNA and ligated into pCRII-TOPO vector (Invitrogen) for <italic>in vitro</italic> transcription. <italic>Nars2</italic> was detected with a ~1.5 kb probe transcribed from NM_153591, nt501&#8211;2078 (<xref ref-type="supplementary-material" rid="pgen.1005097.s005">S5 Table</xref>). <italic>In situ</italic> hybridization positive control probes for <italic>Myo15a</italic> and <italic>NF68</italic> were used as described previously [<xref ref-type="bibr" rid="pgen.1005097.ref026">26</xref>,<xref ref-type="bibr" rid="pgen.1005097.ref027">27</xref>]. RNA hybridization was performed on 12 &#956;m cryo-sections according to standard methods using digoxygenin-labeled probes in weakly acidic hybridization buffer (pH 4.5), anti-digoxigenin-AP Fab fragments (Roche) in TBST buffer, and the NBT/BCIP colorimetric substrate reaction in AP buffer at pH 9.5.</p><p>HEK293T and COS7 cells were grown in DMEM that was supplemented with 10% FBS, 2 mm glutamine, and penicillin/streptomycin (50 U/ml) (Life Technologies) and were maintained at 37&#176;C in 5% CO<sub>2</sub>. Human Fibroblast cells were grown in DMEM (Gibco) supplemented with 10% fetal bovine serum at 37&#176;C and with 5% CO<sub>2</sub>.</p><p>Fibroblast cells were grown in DMEM medium supplemented with glutamax (446 mg/l), 10% fetal calf serum, 50&#956;g/ml uridine and 1mM sodium pyruvate under standard conditions. Cells were fused with 143B rhoo cells as previously described [<xref ref-type="bibr" rid="pgen.1005097.ref063">63</xref>]. Several clones were isolated and the resulting cybrid cells were subsequently expanded. Biochemical assays were performed on isolated mitochondria and/or permeabilized cells. Western blot analysis of a patient&#8217;s muscle biopsy sample and Blue Native gel analysis (BNG) of a patient&#8217;s fibroblast were performed as previously described [<xref ref-type="bibr" rid="pgen.1005097.ref063">63</xref>]. We used &#8220;MitoProfile&#8221; antibody mix (Total OXPHOS human WB antibody cocktail, Abcam) for muscle lysates. For fibroblast lysates we used antibody GRIM19 (Abcam), which corresponds to the mitochondrial complex I subunit NDUFA13.</p><p>The insert of <italic>NARS2</italic> cDNA was cloned into pLVX-IRES-tdTomato lentiviral expression vector using In-Fusion Cloning (Clontech). The construct was packaged into VSV-G pseudotyped viral particles by transfection of HEK293T cells with packaging plasmids and the expression vector for <italic>NARS2</italic>. Lentiviruses were concentrated by ultracentrifugation (CCHMC Vector Core). Transduction of patient fibroblast cell line II.1 was performed according to established methods [<xref ref-type="bibr" rid="pgen.1005097.ref064">64</xref>]. Briefly, fibroblast cells from patient were seeded in six-well plates and transduced 22 hours later with 25 &#956;l of concentrated lentivirus (MOI = 6). Polybrene at 4&#956;g/ml was added to increase the transduction efficiency. Transduced cell were sorted by flow cytometry with standard FITC filter sets, and tdTomato positive cells were selected for NARS2 over-expression. Cell sorting was performed at CCHMC Research Flow Cytometry Core with a BD FACSAria II (BD Biosciences).</p><p>Wild type (WT) human <italic>NARS2</italic> cDNA (clone #Z7860) was obtained from GeneCopoeia and subcloned into pEGFP-N2 vector (Clontech) using InFusion cloning (Clontech). Stratagene QuikChange Lightning mutagenesis (Roche) was used to introduce the c.637G&gt;T transversion and c.1142A&gt;G transition into WT <italic>NARS2</italic> sequence. All constructs were then sub-cloned in pcDNA3.1(+) (Invitrogen) vector and sequence verified. A HA tag was added at the C-terminal part of NARS cDNA using InFusion cloning.</p><p>Constructs were expressed in COS7 cells after transfection with PEI (Polysciences) using a 1:5 ratio (1&#956;g cDNA/5&#956;g PEI). Twenty-four hours post transfection, cells were incubated with 100nM Mito Tracker Red FM (Invitrogen) for 30 minutes, followed by fixation with 4% paraformaldehyde. Fixed cells were mounted with Fluorogel Mounting Medium (EMS) and imaged with a Zeiss LSM700 confocal microscope.</p><p>GFP- and HA-tagged NARS2 constructs were co-expressed in HEK293T cells, after transfection using PEI reagent (Polysciences). Forty-eight hours after transfection, cells were harvested and homogenized with sonication in lysis buffer (50mM Tris HCl pH7.4, 100mM NaCl, 1% NP-40, 2mM Na3VO4) containing a protease inhibitor mixture (#P8340, Sigma). Immunoprecipitation was performed with an anti-GFP antibody as described previously [<xref ref-type="bibr" rid="pgen.1005097.ref065">65</xref>]. The cell lysates and the immunoprecipitates were processed for Western blot analysis [<xref ref-type="bibr" rid="pgen.1005097.ref066">66</xref>]. NARS2 (Abcam), GAPDH (Ambion), GFP (Life Technologies) and HA antibody (Millipore) were used for immunoprecipitation and Western blot analyses.</p><p>Genomic and mitochondrial DNA was extracted from skeletal muscle, whole blood and primary fibroblasts using Gentra (Quiagen) blood and tissue kits according to the manufacturer guidelines. RNA was extracted from primary fibroblast using Trizol reagent (Life Technologies). The 16 Kb mtDNA genome was PCR amplified in 8 overlapping fragments. Nuclear DNA encoded complex I genes were PCR-amplified from patient cDNA. cDNA was synthesized from RNA from patient fibroblast cells using the iScript cDNA synthesis kit (Biorad). Twenty-nine nuclear complex I subunit genes were amplified. PCR products were purified by ExoSAP-IT (Amersham) and directly sequenced using the PRISMTM Ready Reaction Sequencing Kit (PE Applied Biosystems) on an automatic sequencer (ABI 3130, PE Applied Biosystems). Sequence data were analyzed using Sequencer (version 4.0.5, Genecode Corp.) software.</p><p>Total mitochondrial RNA preparations were obtained from mitochondria isolated from fibroblast cell lines (4.0 &#215; 10<sup>7</sup> cells). 2&#956;g of total mitochondrial RNA was electrophoresed through a 10% polyacrylamide-7 M urea gel and then electroblotted onto a positively charged nylon membrane (Roche) for hybridization analyses with specific oligodeoxynucleotide probes [<xref ref-type="bibr" rid="pgen.1005097.ref067">67</xref>]. Nonradioactive DIG-labeled (Roche) oligodeoxynucleotides (mt-tRNA<sup>Asn</sup>: 5&#8217;-CTAGACCAATGGGACTTAAA-3&#8217; and 5S-rRNA: 5&#8217;-GGGTGGTATGGCGGTAGAC-3&#8217;) were used on the same membrane after stripping in 50% formamide, 5% SDS, 50 mM Tris-HCl, pH 7.5 and extensive washes in 2x SSC [<xref ref-type="bibr" rid="pgen.1005097.ref067">67</xref>].</p><p>Total RNA was isolated under acidic conditions. 2&#956;g of total RNA was electrophoresed at 4&#176;C through an acid (pH 5.2) 10% polyacrylamide-7 M urea gel to separate the charged from the uncharged tRNA as detailed elsewhere [<xref ref-type="bibr" rid="pgen.1005097.ref068">68</xref>]. The gels were then electroblotted onto a positively charged nylon membrane (Roche) for hybridization analysis with oligodeoxynucleotide probes for mt-tRNA<sup>Asn</sup> and 5S-rRNA.</p><p>Muscle and skin samples were obtained during autopsy for the first proband, subject II.1, and from a biopsy at 3 months of age from the second proband, subject II.3, from Caucasian family LS06 with parental consents. Spectrophotometric analysis of the respiratory chain complexes was performed in muscle homogenates through the Center for Inherited Disorders of Energy Metabolism (CIDEM), according to established protocols (Case Western Reserve University, Cleveland, OH).</p><p>Oxygen consumption rate (OCR) measurements were performed by using a Seahorse Bioscience XF-24 instrument (Seahorse Biosciences). Cells were seeded in XF24-well microplates in growth medium and the following day growth medium was replaced with assay medium as described [<xref ref-type="bibr" rid="pgen.1005097.ref069">69</xref>].</p><p>Complex I activity was assessed by following the decrease of NADH absorbance at 340 nm, using decylubiquinone as an electron acceptor. The activity of complex II was measured by following the reduction of 2,6-dichlorophenolindophenol (DCPIP) with the decrease of the absorbance at 600 nm of the oxidized DCPIP. Complex III activity was determined by measuring the reduction of cytochrome c by an increase of absorbance at 550 nm. Complex IV activity was measured by monitoring the oxidation of reduced cytochrome c as a decrease of absorbance at 550 nm (Cary 300 UV-Vis Spectrophotometer Agilent, CA.). Complexes I&#8211;IV activities were normalized by citrate synthase activity and then used in the analysis [<xref ref-type="bibr" rid="pgen.1005097.ref070">70</xref>].</p><p>Approval for this study was obtained from the following institutional review boards (IRBs): University of California Irvine (2002&#8211;2608), University of Maryland, School of Medicine, Baltimore, Maryland, USA (HP-00059851), Cincinnati Children&#8217;s Hospital Medical Center, Cincinnati, Ohio, USA (2010&#8211;0291 and 2013&#8211;7868), National Centre of Excellence in Molecular Biology, University of the Punjab, Lahore, Pakistan, and the Combined Neuroscience Institutional Review Board protocol (OH93-DC-0016) at the National Institutes of Health, Bethesda, Maryland, USA. Written informed consent was obtained from all the participating family members.</p><p>All experiments and procedures were approved by the Institutional Animal Care and Use Committees of CCHMC and the University of Maryland, School of Medicine.</p><p>The URLs for the data presented herein are as follows:</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Allen Brain Atlas, <ext-link ext-link-type="uri" ns0:href="http://www.brain-map.org/">http://www.brain-map.org/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">ANNOVAR, <ext-link ext-link-type="uri" ns0:href="http://www.openbioinformatics.org/annovar">www.openbioinformatics.org/annovar</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">DomPred Protein Domain Prediction Server, <ext-link ext-link-type="uri" ns0:href="http://bioinf.cs.ucl.ac.uk/dompred">http://bioinf.cs.ucl.ac.uk/dompred</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Online Mendelian Inheritance in Man (OMIM), <ext-link ext-link-type="uri" ns0:href="http://www.omim.org/">http://www.omim.org/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">NHLBI Exome Sequencing Project (Exome Variant Server), <ext-link ext-link-type="uri" ns0:href="http://evs.gs.washington.edu/EVS/">http://evs.gs.washington.edu/EVS/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">National Center for Biotechnology Information (NCBI), <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Primer3, <ext-link ext-link-type="uri" ns0:href="http://frodo.wi.mit.edu/primer3">http://frodo.wi.mit.edu/primer3</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">PYMOL, <ext-link ext-link-type="uri" ns0:href="http://www.pymol.org">www.pymol.org</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">SHIELD (Shared Harvard Inner-Ear Laboratory Database), <ext-link ext-link-type="uri" ns0:href="https://shield.hms.harvard.edu/">https://shield.hms.harvard.edu/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">STRAP, <ext-link ext-link-type="uri" ns0:href="http://www.bioinformatics.org/strap/">http://www.bioinformatics.org/strap/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">UCSC Genome Bioinformatics, <ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu">http://genome.ucsc.edu</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Unigene, <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/unigene/">http://www.ncbi.nlm.nih.gov/unigene/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">World Health Organization, <ext-link ext-link-type="uri" ns0:href="http://www.who.int">http://www.who.int</ext-link>
</p>