<p>All animal procedures were conducted accordingly to the EU regulations and 3R principles and reviewed and approved by the Bioethics Committee of the University of Cantabria. Mutant mouse lines were described previously: <italic>Sp6</italic> null allele <xref ref-type="bibr" rid="pgen.1004468-Nakamura1">[28]</xref>; <italic>Sp8</italic> null allele <xref ref-type="bibr" rid="pgen.1004468-Treichel1">[24]</xref>; <italic>Sp8</italic> floxed allele <xref ref-type="bibr" rid="pgen.1004468-Zembrzycki1">[36]</xref>; <italic>AP2&#945;Cre</italic>
<xref ref-type="bibr" rid="pgen.1004468-Macatee1">[39]</xref> and <italic>Msx2Cre</italic> lines <xref ref-type="bibr" rid="pgen.1004468-Sun1">[37]</xref>; R26R <xref ref-type="bibr" rid="pgen.1004468-Soriano1">[42]</xref>. Mice and embryos were genotyped by PCR, using genomic DNA extracted from tail biopsies and yolk sacs, respectively.</p><p>After removing skin and viscera, mouse embryos were fixed in 95% ethanol. Alizarin Red and Alcian blue skeletal staining was performed according to standard protocols, cleared by KOH treatment and stored in glycerol.</p><p>In situ hybridization (ISH) was performed in whole-mount and in sections following standard procedures using the previously described <italic>Bmp4</italic>
<xref ref-type="bibr" rid="pgen.1004468-Bandyopadhyay1">[64]</xref>, <italic>Dlx5</italic> and <italic>Dlx6</italic>
<xref ref-type="bibr" rid="pgen.1004468-Robledo1">[53]</xref>, <italic>En1</italic>
<xref ref-type="bibr" rid="pgen.1004468-Loomis2">[23]</xref>
<italic>Fgf8</italic>
<xref ref-type="bibr" rid="pgen.1004468-Crossley1">[43]</xref>, <italic>Fgf10</italic>
<xref ref-type="bibr" rid="pgen.1004468-Harada1">[89]</xref>, <italic>Lmx1b</italic>
<xref ref-type="bibr" rid="pgen.1004468-Chen1">[90]</xref>, <italic>Msx2</italic>
<xref ref-type="bibr" rid="pgen.1004468-Robert1">[91]</xref>, <italic>Tp63</italic>
<xref ref-type="bibr" rid="pgen.1004468-Mills1">[62]</xref>, <italic>Sp6</italic>
<xref ref-type="bibr" rid="pgen.1004468-Nakamura1">[28]</xref> and <italic>Wnt7a</italic>
<xref ref-type="bibr" rid="pgen.1004468-Parr1">[92]</xref> antisense riboprobes.</p><p>Embryonic fore and hind- limb buds were dissected in cold RNAse-free PBS from E10.5 wild type embryos. Total RNA was isolated separately from 3 pools of 8 forelimbs or 8 hindlimbs each. cDNA synthesis was done using standard conditions.</p><p>Real-time RT-PCR was carried out on an Mx3005P cycler, using the SYBRGreen PCR Master Mix (Invitrogen) and the data were analyzed using the MxPro software (Stratagene). Results were tested statistically performing ANOVA and Student-T test, being statistically significant when p&lt;0.05.</p><p>Expression of <italic>Sp6</italic> and <italic>Sp8</italic> was normalized to that of housekeeping gene 18sRNA. The primers used (5&#8242; to 3&#8242; orientation) were: Sp6-F: <named-content content-type="gene">tgctaaccgctgtctgtgg</named-content>; Sp6-R:<named-content content-type="gene">ctggtatgtctggagaggttgc</named-content>; Sp8-F: <named-content content-type="gene">ttatctccaaggtgcacacg</named-content>; Sp8-R:<named-content content-type="gene">gcttgaaccaggactcatacg</named-content>; 18sRNA-R: <named-content content-type="gene">ttggcaatgtttcgctc</named-content>;18sRNA-F: <named-content content-type="gene">cgccgctagaggtgaaattt</named-content>.</p><p>Detection of cell death was performed in sections of paraffin embedded tissue using terminal deoxynucleotidyl transferase mediated dUTP nick-end labelling (TUNEL) with the Apoptag Fluorescein Direct In Situ Apoptosis Detection Kit (Intergen) following the manufacturer's instructions.</p><p>For detection of &#946;-galactosidase activity, <italic>R26R;Ap2&#945;Cre</italic> double transgenic embryos were fixed for 30 min, rinsed in PBS and incubated in the presence of X-gal as described <xref ref-type="bibr" rid="pgen.1004468-Whiting1">[93]</xref>.</p><p>Immunohistochemistry was performed in paraffin sections using the anti E-cadherin (Byoscience, # 610182), anti Laminin (Abcam, # ab11575), anti Tp63 (Abcam, # Ab53039) and anti Connexin43 (Abcam, # ab11370) primary antibodies. Antigen retrieval was performed by incubation with proteinase K (10 &#181;g/ml) for E-cadherin and laminin or with citrate buffer in pressure cooker for Tp63 and Connexin43. Alexa&#174;488 and TexasRED fluorescently tagged secondary antibodies were used. Vecthasield containing DAPI for nuclear counter staining was used as mounting medium. Confocal images were acquired in a SP-5 laser-scan confocal microscope (Leica Microsystems).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Conservation of <italic>En1</italic>, <italic>Sp6</italic> and <italic>Sp8</italic> loci between mouse, human, opossum, chicken and zebrafish was determined using pairwise alignment software (mVista browser, <ext-link ext-link-type="uri" ns0:href="http://genome.lbl.gov/vista/">http://genome.lbl.gov/vista/</ext-link>). Conserved noncoding regions were further analyzed for potential transcription factor binding sites using AliBaba 2.1 (<ext-link ext-link-type="uri" ns0:href="http://www.generegulation.com/pub/programs/alibaba2/index.html">http://www.generegulation.com/pub/programs/alibaba2/index.html</ext-link>) and Sequencher 4.8 (Gene Codes Inc.) informatic software.</p>