<p>In order to explore the level of functional constraint in <italic>IFNL4</italic>, we estimated the level of protein conservation in primate and non-primate mammals. Specifically, we assessed the ratio (dN/dS) of non-synonymous substitutions per non-synonymous site (dN) to synonymous substitutions per synonymous site (dS) across gene orthologs. Since purifying selection eliminates deleterious protein-coding changes, dN/dS decreases with negative selection and increases with relaxed constraint and positive selection.</p><p>We used human <italic>IFNL4</italic> reference sequence NM_001276254.2 to BLAT genomes of other species and generate multiple-species sequence alignment of <italic>IFNL4</italic> coding exons 1 through 5 (<xref ref-type="supplementary-material" rid="pgen.1004681.s020">Table S8</xref>). The panda-predicted <italic>IFNL4</italic> ortholog was subsequently used as BLAT query to extract coding exons for additional non-primate species (<xref ref-type="supplementary-material" rid="pgen.1004681.s020">Table S8</xref>). Further, we sequenced <italic>IFNL4</italic> (exons and introns) in genomic DNA and reconstructed complete <italic>IFNL4</italic> cDNA sequences of chimpanzee (Genbank accession JX867772), baboon (Genbank accession KC525947) and crab-eating macaque (Genbank accession KC525948). The whole <italic>IFNL4</italic> genomic region is absent in mouse or rat. All discovered functional <italic>IFNL4</italic> sequences (<xref ref-type="supplementary-material" rid="pgen.1004681.s020">Table S8</xref>) where used for a multiple-sequence alignment which was created using ClustalW <xref ref-type="bibr" rid="pgen.1004681-Thompson1">[52]</xref> and annotated with Jalview <xref ref-type="bibr" rid="pgen.1004681-Waterhouse1">[53]</xref>.</p><p>The alignment was analyzed with <italic>codeml</italic> (part of PAML4 <xref ref-type="bibr" rid="pgen.1004681-Yang1">[54]</xref>) to test various models of selection. We estimated the overall dN/dS for the complete tree and compared likelihoods for models that allowed: i) free dN/dS for each branch (i.e., lineage heterogeneity); ii) a primate-specific dN/dS; and iii) a human-specific dN/dS. Additionally, we performed tests aimed to detect site-specific signatures of positive selection across the phylogeny (branch models): i) model 1a (neutral) vs. model 2 (positive selection); ii) model 7 (neutral) vs. model 8 (with dN/dS&gt;1); and iii) model 8a (with dN/dS&#8202;=&#8202;1) vs. model 8 (with dN/dS&gt;1).</p><p>We analyzed genome-wide data from the 1000 Genomes release (2010/11/23; phase I) <xref ref-type="bibr" rid="pgen.1004681-McVean1">[24]</xref>. We considered (1) autosomal variants detected in the low coverage sequencing, and (2) populations with information for at least 50 unrelated individuals, which was met by 13 populations from four different continents [African ancestry: YRI (Yoruba in Ibadan, Nigeria), LWK (Luhya in Webuye, Kenya), ASW (African Ancestry in Southwest US); European ancestry: GBR (British from England and Scotland), CEU (Utah residents (CEPH) with Northern and Western European ancestry), FIN (Finnish from Finland), TSI (Toscani in Italia); East Asian ancestry: CHS (Han Chinese South), CHB (Han Chinese in Beijing, China), JPT (Japanese in Toyko, Japan); American ancestry: MXL (Mexican Ancestry in Los Angeles, CA), CLM (Colombian in Medellin, Colombia), PUR (Puerto Rican in Puerto Rico)]. As an exception PUR (Puerto Rico) was analyzed although it contains only 44 individuals. Some analyses were performed both by population and by continent; in these cases the continental groups contain 150 randomly selected, unrelated individuals (America 144) with an equal contribution from each population within the continent.</p><p>For the rs368234815 &#916;G/TT frameshift-substitution variant the 1000 Genomes dataset only contains the T insertion/deletion variant rs11322783 (-/T, chr19:39739154, dbSNP b138), while the substitution rs74597329 (G/T, chr19:39739155, dbSNP b138) is absent. This is due to the automatic variant caller failing to correctly identify an insertion and a substitution in the same genomic position. We sequenced an amplicon containing rs368234815 in 153 individuals included both in the 1000 Genomes and HapMap sets (CEU, YRI and CHB/JPT). Sequencing confirmed the presence of only two alleles (&#916;G and TT) and showed good concordance with the 1000 Genomes data between our &#916;G/TT genotypes and 1000 Genomes genotypes for the overlapping insertion/deletion variant rs11322783 (4 individuals of 153 tested were discordant, providing an estimated 97.4% genotype and 98.7% allele concordance rate). This validated the use of 1000 Genomes dataset for our subsequent analyses. We used the ancestral allelic state annotated in the 1000 Genomes data, which is based on the Ensembl 59 comparative 32 species alignment <xref ref-type="bibr" rid="pgen.1004681-Paten1">[55]</xref>; only SNPs with a high-confidence ancestral inference were used, and indels were excluded due to their cryptic variation patterns <xref ref-type="bibr" rid="pgen.1004681-Kvikstad1">[56]</xref>.</p><p>We used F<sub>ST</sub>, iHS and XP-EHH to explore the signatures of selection of rs368234815 TT allele. F<sub>ST</sub> is a measure of population differentiation and unusually high F<sub>ST</sub> can indicate population-specific positive selection that drastically increases allele frequency in the population under selection <xref ref-type="bibr" rid="pgen.1004681-Nielsen1">[57]</xref>. To calculate F<sub>ST</sub> we used the Weir and Cockerham <xref ref-type="bibr" rid="pgen.1004681-Weir1">[25]</xref> estimator implemented in vcf-tools <xref ref-type="bibr" rid="pgen.1004681-Danecek1">[58]</xref>.</p><p>Positively selected alleles rapidly increase in frequency with recombination having little chance to break their association with nearby variants. If the selected allele was originally in few haplotype backgrounds and it has not reached fixation, it will be associated with extended haplotype homozygosity (EHH), a pattern that will be absent for the non-selected allele. We used two statistics to explore this expectation. First, iHS <xref ref-type="bibr" rid="pgen.1004681-Voight1">[27]</xref> measures the allele-specific decay of EHH within a population by comparing the associated EHH of ancestral and derived alleles. Second, XP-EHH <xref ref-type="bibr" rid="pgen.1004681-Sabeti1">[26]</xref> that detects alleles that are under selection in one population only, by comparing EHH patterns both among allelic types and across populations; as such XP-EHH has higher power to detect population-specific selection. Low frequency variants break the EHH signal, so following <xref ref-type="bibr" rid="pgen.1004681-Grossman1">[59]</xref> we considered only SNPs with derived allele frequency &#949; 5% for XP-EHH or minor allele frequency &#949; 5% for iHS. Local recombination rate estimates were obtained from a combined recombination map based on HapMap data <xref ref-type="bibr" rid="pgen.1004681-Frazer1">[60]</xref> from Africa, European, and Asian populations. Both statistics were standardized to a mean of zero and a standard deviation of one; for iHS, scores were then binned by frequency (1%) as previously suggested <xref ref-type="bibr" rid="pgen.1004681-Voight1">[27]</xref>. Correlation of F<sub>ST</sub> with XP-EHH (CHS vs. YRI) or iHS (CHS) was calculated for all variants present in the respective dataset with Spearman's rank correlation test implemented in R <xref ref-type="bibr" rid="pgen.1004681-Team1">[61]</xref>.</p><p>We used each of these statistics to analyze every non-African population; for between-population comparisons we used Yoruba as background, unless noted otherwise. To assess the putative effects of this choice of populations we repeated the analyses for continental groups, for different background populations, and for SNPs that have their lowest allele frequency in Yoruba. In all cases the empirical P-values were obtained by comparing the score for rs368234815 to the whole-genome empirical distribution of the respective statistic. Since this is a hypothesis-driven analysis with a single variant analyzed within a single locus, no multiple testing or genome-wide corrections are needed.</p><p>We also applied tests that analyze the signatures of selection in the <italic>IFNL4</italic> genetic region (&#8764;2.5 kb). Here we show results for Fay and Wu's H test <xref ref-type="bibr" rid="pgen.1004681-Fay1">[28]</xref>, which detects the excess of high-frequency derived alleles expected after a recent sweep with recombination. Significance was estimated using 10,000 standard neutral coalescent simulations <xref ref-type="bibr" rid="pgen.1004681-Hudson1">[62]</xref>. Because demography affects the SFS and can cause spurious results if not properly accounted for, our simulations are run under a demographic model which includes inferred parameters for populations of African <xref ref-type="bibr" rid="pgen.1004681-Gravel1">[63]</xref>, European <xref ref-type="bibr" rid="pgen.1004681-Gravel1">[63]</xref>, Asian <xref ref-type="bibr" rid="pgen.1004681-Gravel1">[63]</xref> and American <xref ref-type="bibr" rid="pgen.1004681-Gutenkunst1">[64]</xref> ancestry. A custom made perl program (Neutrality Test Pipeline) was used to calculate the statistic and corresponding P-value.</p><p>To infer the model of selection that best fits <italic>IFNL4</italic> data and estimate the timing and selection strength of the TT allele, we used an Approximate Bayesian Computation (ABC) approach <xref ref-type="bibr" rid="pgen.1004681-Beaumont1">[22]</xref>. In particular, we followed a published approach <xref ref-type="bibr" rid="pgen.1004681-Peter1">[23]</xref>, which has been previously shown to discriminate well between SDN, SSV and neutrality (NTR) <xref ref-type="bibr" rid="pgen.1004681-Peter1">[23]</xref>. In brief, this approach is based on performing a large number of simulations under different selection models, with random parameters drawn from some probability distribution (called the prior distribution). Real data and simulations are compared based on summary statistics, and through a rejection scheme the simulations that most closely resemble real data help inform inferences about the best-fitting model. The parameter values that generate these simulations are then used to obtain the posterior distribution of each parameter, whose mean and standard deviation are used to perform the parameter inferences. We extended the method to consider more than one population, since two-population statistics are most informative in our case.</p><p>Specifically, the approach uses msms <xref ref-type="bibr" rid="pgen.1004681-Ewing1">[65]</xref> to simulate data, custom python scripts to calculate all summary statistics, and ABCtoolbox <xref ref-type="bibr" rid="pgen.1004681-Wegmann1">[66]</xref> for all ABC inferences. Under both selection models, we started with uniform priors with a range as follow (see <xref ref-type="fig" rid="pgen-1004681-g005">Fig. 5A</xref>):</p><p>SDN model - selection strength in Africa <italic>s<sub>A</sub></italic>&#8764;U(0,1.5%); selection strength in non-Africa <italic>s<sub>NA</sub></italic>&#8764;U(0.5,5%); time when selection started <italic>t<sub>mut</sub></italic>&#8764;U(40,70kya)</p><p>SSV model - selection strength in non-Africa <italic>s<sub>NA</sub></italic>&#8764;U(&gt;0,5%); frequency of the allele when selection started <italic>f<sub>0</sub></italic>&#8764;U(0,20%); time when selection started <italic>t<sub>mut</sub></italic>&#8764;U(21,51kya)</p><p>NTR model - time when mutation appears <italic>t</italic>&#8764;U(40,70kya)</p><p>Because simulations with the selected allele fixed are likely to be very different from the observed data, we conditioned on the selected allele segregating in both populations. This resulted in non-uniform prior distributions presented in <xref ref-type="supplementary-material" rid="pgen.1004681.s005">Figure S5</xref> and <xref ref-type="supplementary-material" rid="pgen.1004681.s006">S6</xref>. We used 10<sup>4</sup> simulations to distinguish between the neutral model and the two selection models, and a larger set of 8&#215;10<sup>5</sup> simulations for the more subtle distinction between the two selection models and for parameter estimation. For the simulations, we used the population history model estimated by Gravel et al. <xref ref-type="bibr" rid="pgen.1004681-Gravel1">[63]</xref> and assumed a constant recombination rate of 1.76 cm/Mb throughout the region (average recombination rate in the <italic>IFNL</italic> locus <xref ref-type="bibr" rid="pgen.1004681-Frazer1">[60]</xref>), and a perfectly additive model of dominance (h&#8202;=&#8202;0.5). Lack of an appropriate demographic model for American and non-Yoruba African populations precludes analysis for those populations. The following single-population statistics were calculated: the average number of pairwise differences &#960;, Watterson's &#952;, Fay and Wu's H <xref ref-type="bibr" rid="pgen.1004681-Fay1">[28]</xref> and Tajima's D <xref ref-type="bibr" rid="pgen.1004681-Tajima1">[67]</xref>, all for both 4 kb around the site and a 8 kb (6 kb upstream and 2 kb downstream of the site) interval around the TT allele. The between-population statistics employed were: F<sub>ST</sub>
<xref ref-type="bibr" rid="pgen.1004681-Reynolds1">[68]</xref> for the selected site, F<sub>ST</sub> in 4 kb around the site, F<sub>ST</sub> for the whole region, and XP-EHH on the selected site <xref ref-type="bibr" rid="pgen.1004681-Sabeti1">[26]</xref>. In addition, we also included the frequency of the selected allele in both populations. This resulted in a set of 16 summary statistics, which, following Wegmann et al. <xref ref-type="bibr" rid="pgen.1004681-Wegmann2">[69]</xref> and Peter et al. <xref ref-type="bibr" rid="pgen.1004681-Peter1">[23]</xref>, was reduced to seven summary statistics using PLS-DA <xref ref-type="bibr" rid="pgen.1004681-Tenenhaus1">[70]</xref> for model choice and regular PLS for parameter inference <xref ref-type="bibr" rid="pgen.1004681-Boulesteix1">[71]</xref>. Performance of the ABC model choice and parameter distribution for the SDN model has been assessed for each particular model (<xref ref-type="supplementary-material" rid="pgen.1004681.s010">Note S3</xref>). Confidence in the choice of selection models has been supported with Bayes factors.</p><p>In addition, we investigated the influence of the dominance model in our inferences. We analyzed a recessive model for TT (h&#8202;=&#8202;0), the perfectly additive model above (h&#8202;=&#8202;0.5), and a supra-additive model (h&#8202;=&#8202;0.38), using 500,000 simulations for each model. We run an ABC analysis for model selection with all simulations (from all three dominance models and the three selection models NTR, SDN, and SSV). We then assess the posterior probability of each dominance model regardless of selection model, and the posterior probability (and parameter estimates) of each selection model for the additive and supra-additive dominance models (see <xref ref-type="supplementary-material" rid="pgen.1004681.s011">Note S4</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">1000 Genomes, <ext-link ext-link-type="ftp" ns0:href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/</ext-link>; GENCODE, <ext-link ext-link-type="uri" ns0:href="http://pseudogene.org/psidr/">http://pseudogene.org/psidr/</ext-link>; HapMap, <ext-link ext-link-type="uri" ns0:href="http://hapmap.ncbi.nlm.nih.gov">http://hapmap.ncbi.nlm.nih.gov</ext-link>; XP-EHH and iHS executables, <ext-link ext-link-type="uri" ns0:href="http://hgdp.uchicago.edu/Software/">http://hgdp.uchicago.edu/Software/</ext-link>; VCFtools, <ext-link ext-link-type="uri" ns0:href="http://vcftools.sourceforge.net">http://vcftools.sourceforge.net</ext-link>; ABCtoolbox: <ext-link ext-link-type="uri" ns0:href="http://www.cmpg.iee.unibe.ch/content/softwares__services/computer_programs/abctoolbox/index_eng.html">http://www.cmpg.iee.unibe.ch/content/softwares__services/computer_programs/abctoolbox/index_eng.html</ext-link>; msms: <ext-link ext-link-type="uri" ns0:href="http://www.mabs.at/ewing/msms/">http://www.mabs.at/ewing/msms/</ext-link>
</p>