<p xmlns:ns0="http://www.w3.org/1999/xlink">Multiple sequence alignments were generated using MUSCLE at <ext-link ext-link-type="uri" ns0:href="http://www.ebi.ac.uk/Tools/msa/muscle/">http://www.ebi.ac.uk/Tools/msa/muscle/</ext-link>. ConSurf <xref ref-type="bibr" rid="pgen.1004326-Landau1">[36]</xref> and PyMOL <xref ref-type="bibr" rid="pgen.1004326-DeLano1">[37]</xref> were used to obtain sequence conservation scores and generate the surface representation of sequence conservation on the crystal structure of the yeast Gcn2 KD (pdb: 1ZYC).</p><p>Plasmids employed are listed in <xref ref-type="table" rid="pgen-1004326-t001">Table 1</xref>. QuikChange site-directed mutagenesis (Stratagene) was used to generate the novel derivatives of plasmid p722 (pSL201-pSL242) and p630 (pSL301-pSL311) and <italic>GCN2</italic> was sequenced in its entirety for those alleles exhibiting significant Gcn<sup>&#8722;</sup> or Gcd<sup>&#8722;</sup> phenotypes. For Gcd<sup>&#8722;</sup> mutations identified by random mutagenesis, p722 was subjected to error-prone PCR mutagenesis using the GeneMorph II kit (Stratagene) by using primer pairs PS-1 (<named-content content-type="gene">5&#8242;-ATAGCAAATTTAGAGAAAGAGTTAG-3&#8242;</named-content>) and PS-2 (<named-content content-type="gene">5&#8242;-CTTAACAGCAGTCATCGGTTTTAC-3&#8242;</named-content>). The BlpI -XhoI 1.4-kb <italic>GCN2</italic> fragment encoding the YKD was isolated from plasmid DNA prepared from a pool of <italic>E. coli</italic> transformants harboring mutagenized plasmids and subcloned into p722. Plasmid DNA prepared from a pool of the resulting <italic>E. coli</italic> transformants was introduced into yeast strain H1149 and transformants were selected on minimal (SD) medium containing 0.5 mM 5-FT. Resident plasmids were isolated from colony-purified transformants and subjected to DNA sequence analysis to identify the mutations. As multiple mutations generally occurred, site-directed mutagenesis was used to produce plasmids pSL233, pSL234, pSL235, pSL237, pSL238, pSL239 and pSL240, containing only single mutations in <italic>GCN2</italic>. pSL102-pSL106 were generated by replacing the 1.2-kb BlpI-BspEI fragment encoding the YKD in pSL101 with the corresponding fragment from p722 derivatives harboring the appropriate <italic>GCN2</italic> mutations. The same strategy was used to construct pSL401-pSL405 from pHQ539.</p><p>Yeast strains employed included H1149 (<italic>MAT&#945; gcn2</italic>&#916;<italic>::LEU2 ino1 ura3-52 leu2-3 leu2-112 &lt;HIS4-lacZ&gt;</italic>) <xref ref-type="bibr" rid="pgen.1004326-Wek1">[11]</xref>, HQY132 (<italic>MAT</italic>&#945; <italic>trp1 ura3 his3 lexAop-LEU2 gcn2</italic>&#916;<italic>::hisG</italic>) <xref ref-type="bibr" rid="pgen.1004326-Qiu3">[19]</xref>, and H2684 (<italic>MATa ino1 ura3-52 gcn1</italic>&#916; <italic>gcn2</italic>&#916; <italic>gcn20</italic>&#916;) (M. Marton and A.G.H., unpublished observations).</p><p>Transformants of H2684 bearing plasmids pSL101, pSL102, pSL103, pSL104, pSL105, or pSL106 were grown to saturation in SC-Ura medium, diluted to A<sub>600</sub>&#8202;=&#8202;0.2 in SC-Ura containing 10% galactose as carbon source and grown to A<sub>600</sub> &#8764;2.5. Cells were harvested (&#8764;25 g), washed with cold distilled water containing EDTA-free protease inhibitor cocktail (PIC) (Boehringer Mannheim) and 0.5 mM PMSF, resuspended in ice-cold binding buffer (BB) (100 mM sodium phosphate [pH 7.4], 500 mM NaCl, 0.1% Triton X-100, EDTA-free PIC, 1 &#181;g/ml leupeptin, and 1 mM PMSF) and disrupted using SPEX freezer mill (model <italic>6870</italic>). Lysates were clarified by centrifugation at 39,000&#215;g for 2 h at 4&#176;C and mixed with 1 ml of M2-FLAG affinity resin (Sigma) overnight at 4&#176;C. The resin was washed three times with 10 vol of BB and Gcn2 was eluted with 100 units of AcTEV protease in 500 &#181;l of 1&#215; TEV buffer (50 mM Tris pH 8, 0.5 mM EDTA, 1 mM DTT). The eluates were concentrated with an Amicon Centricon filter (exclusion limit of M<sub>r</sub> 10,000) and dialyzed against 10 mM Tris-HCl [pH 7.4], 50 mM NaCl, 20% glycerol and stored at &#8722;80&#176;C. The eIF2&#945;&#8722;&#916;C protein was purified from <italic>E. coli</italic> as previously described <xref ref-type="bibr" rid="pgen.1004326-Zhu1">[13]</xref>.</p><p>Assays of &#946;-galactosidase activity in WCEs were performed as described previously <xref ref-type="bibr" rid="pgen.1004326-Moehle1">[38]</xref>. For Western analysis, WCEs were prepared by trichloroacetic acid extraction, as described previously <xref ref-type="bibr" rid="pgen.1004326-Reid1">[39]</xref>, and immunoblot analysis was conducted as described <xref ref-type="bibr" rid="pgen.1004326-Qiu3">[19]</xref> using phosphospecific antibodies against eIF2&#945;-P (Biosource International) and polyclonal antibodies against eIF2&#945; <xref ref-type="bibr" rid="pgen.1004326-Cigan1">[40]</xref> or Gcn2 <xref ref-type="bibr" rid="pgen.1004326-Romano1">[16]</xref>.</p><p>Assaying autophosphorylation and eIF2&#945; phosphorylation by purified Gcn2 was conducted as described previously <xref ref-type="bibr" rid="pgen.1004326-Qiu1">[8]</xref>. Binding of tRNA was measured with a gel mobility shift assay as follows. Total yeast tRNA was purchased from Roche. tRNA was first dephosphorylated using calf intestine alkaline phosphatase (New England BioLabs) for 1 h at 37&#176;C in 1&#215; Dephosphorylation buffer provided with the enzyme and followed by phenol/chloroform extraction and ethanol precipitation. Three micrograms of the dephosphorylated tRNA was phosphorylated using 25 pmol of [&#947;-<sup>32</sup>P]ATP and 10 units polynucleotide kinase for 1 h in 1&#215; kinase buffer provided with the enzyme. The reaction was stopped by adding EDTA to 1 mM and heating for 2 min at 95&#176;C. tRNA was purified from free nucleotides using MicroSpin G-25 Columns (GE Healthcare). Fifty nanograms of [<sup>32</sup>P]-labelled tRNA were mixed with purified Gcn2 (1&#8211;4 &#181;M) and 10 U RNasin Ribonuclease Inhibitor (N2511; Promega), in 1&#215; GMSA buffer (2 mM HEPES [pH 7.4], 15 mM NaCl, 15 mM MgCl<sub>2</sub>, 10% glycerol) in a total volume of 20 &#181;l. After incubating for 30 min at 30&#176;C, 4 &#181;l of 6&#215; nucleic acid loading buffer (30% (v/v) glycerol, 0.25% (w/v) bromophenol blue) were added and the mixture was resolved by electrophoresis on a 1% agarose gel cast in 1&#215; MOPS buffer (Quality Biological, Inc) at 100 V for 1.5 h. RNA and protein molecules were transferred from the gel to a nitrocellulose membrane (162-0097; Bio-Rad, Hercules, CA, USA) by capillary action in 10&#215; SSC for 16 h. [<sup>32</sup>P]tRNA&#8211;Gcn2 complexes were quantified with a phosphorimager (Molecular Dynamics) using the Image Quant software provided by the vendor.</p><p>Coimmunoprecipitations of Gcn2 with LexA-HA-KD fusion protein were conducted as described previously <xref ref-type="bibr" rid="pgen.1004326-Qiu3">[19]</xref> using HA-probe Antibody agarose conjugate (sc-7392 AC) and LexA antibodies (sc-7544, 1&#8758;2000 dilution) purchased from Santa Cruz. Coimmunoprecipitation of [<sup>35</sup>S]-methionine-labeled Gcn2 YKD polypeptides with LexA-HA-KD fusion protein was executed as follows. <italic>In vitro</italic> transcription/translation with [<sup>35</sup>S]-methionine was conducted using the TNT T7 Coupled Reticulocyte Lysate System (Promega) according to the vendor's instructions. <italic>In vitro</italic>-translated proteins were partially purified by ammonium sulfate precipitation as described previously <xref ref-type="bibr" rid="pgen.1004326-Bardwell1">[41]</xref> and resuspended in 50 &#181;l of buffer A (20 mM Tris/HCl pH 7.5, 100 mM NaCl, 0.2 mM EDTA, 1 mM DTT) containing 12.5% glycerol. Fifty &#181;g of WCE prepared from p2825 transformants of HQY132 and 10 &#181;L of <italic>in vitro</italic>-translated proteins were diluted to a final volume of 200 &#181;L with breaking buffer (50 mM Tris/HCl, ph 7.5, 50 mM NaCl, 0.1% Triton-100, 1 mM DTT) containing protease inhibitors (Aprotinin 10 &#181;g/mL, Leupeptin 10 &#181;g/mL, Pepstatin 10 &#181;g/mL and 1 mM PMSF) and pre-incubated with 20 &#181;L of protein A-agarose beads (Santa Cruz, sc-2001) suspended in breaking buffer for 1 h at 4&#176;C with rocking. The beads were removed by centrifugation and the supernatant added to HA-probe Antibody agarose conjugate (Santa Cruz, sc-7392) and incubated at 4&#176;C for 2 h with rocking. The beads were collected by centrifugation, washed three times with 500 &#181;l of breaking buffer, and resuspended in 40 &#181;l of Tris-Glycine SDS Sample Buffer (Novex). Proteins in the immune complexes were resolved by sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE). For detecting [<sup>35</sup>S]-labeled proteins, gels were fixed with 25% ethanol/10% acetic acid, treated with Amplify (Amersham), dried, and subjected to fluorography.</p>