<p>Generation of <italic>en</italic><sup><italic>&#916;1</italic>.<italic>5</italic></sup> and <italic>inv</italic><sup><italic>&#916;24</italic></sup> mutants have been described [<xref ref-type="bibr" rid="pgen.1006200.ref032">32</xref>, <xref ref-type="bibr" rid="pgen.1006200.ref042">42</xref>]. The double mutant <italic>inv</italic><sup><italic>&#916;24</italic></sup>-<italic>en</italic><sup><italic>&#916;1</italic>.<italic>5</italic></sup> was obtained using P-element mediated male recombination [<xref ref-type="bibr" rid="pgen.1006200.ref062">62</xref>]. The deletions were validated both by PCR and by the absence of those sequences in ChIP-seq data. All flies were kept at 25&#176;C. Generation of constructs and transgenic lines has been discussed previously [<xref ref-type="bibr" rid="pgen.1006200.ref032">32</xref>].</p><p>Total RNA was collected from 20 third instar larvae wing imaginal disks using Trizol (Invitrogen) according to manufacturer&#8217;s protocol. One-step RT-qPCR was performed with the SensiFAST SYBR No-ROX One-Step kit (Bioline) on a Roche Lightcycler 480 according to manufacturer instructions [<xref ref-type="bibr" rid="pgen.1006200.ref063">63</xref>]. Primers used are listed in <xref ref-type="supplementary-material" rid="pgen.1006200.s006">S2 Table</xref>.</p><p>For antibody staining, imaginal disks and brains were collected from third instar larvae and fixed for 23 min [0.1 M PIPES (pH 6.9), 1 mM EGTA (pH 6.9), 1.0% Triton X-100, 2mM MgSO4, 4% formaldehyde (Ted Pella, Inc.)]. Fixed tissues were blocked for 2 hrs at 4&#176;C with blocking buffer (1X PBS, 0.1% Tween-20 and 5% normal goat serum). Tissues were incubated over night at 4&#176;C with primary antibody diluted in blocking buffer (rabbit anti-En 1:500 dilution, Santa Cruz Biotechnology, Inc.; mouse anti-&#223;gal 1:500 dilution, Invitrogen). After removal of primary antibody, tissues were washed (1X PBS, 0.1% Tween-20) four times- 15 min each. Washed tissues were incubated in fluorescent tagged secondary antibody diluted in blocking buffer for 2 hrs at 4&#176;C in dark, washed 3 times and mounted with DAPI-Vectashield (Vector Laboratories). All wing imaginal discs were quantified by ImageJ.</p><p>Protocol for carrying out ChIP in larval tissue has been described previously [<xref ref-type="bibr" rid="pgen.1006200.ref063">63</xref>], with minor changes: fixed brains and imaginal discs were dissected from 10 third instar larvae, and before incubating the sonicated chromatin with antibodies, 3.3% of each sample was saved for input reactions. ChIP was performed with 1:100 dilutions of anti-Pho [<xref ref-type="bibr" rid="pgen.1006200.ref064">64</xref>] and anti-Ph antibodies (a kind gift from Donna Arndt-Jovin) and 1:200 dilutions of anti-H3K27me3 (Millipore, 17&#8211;622) antibodies. Quantitative PCR on ChIP samples was performed as described previously [<xref ref-type="bibr" rid="pgen.1006200.ref026">26</xref>]. Lightcycler 480 Real-Time PCR System (Roche Applied Science) and Lightcycler 480 DNA SYBR Green I Master Mix (Roche Applied Science) were used for performing ChIP-qPCR.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Following purification of immunoprecipitated DNA, Illumina libraries were prepared using TruSeq DNA Sample Prep Kit V2 as described (<ext-link ext-link-type="uri" ns0:href="http://ethanomics.wordpress.com/chip-seq-library-construction-using-the-illumina-truseq-adapters/">http://ethanomics.wordpress.com/chip-seq-library-construction-using-the-illumina-truseq-adapters/</ext-link>). Peak calling for Pho was conducted using MACS v1.4.2 [<xref ref-type="bibr" rid="pgen.1006200.ref065">65</xref>]. All ChIP-seq data sets were aligned using Casava (version 1.8) to the Drosophila reference genome (release 5.22/6.02). All ChIP-seq experiments were performed with 2 biological replicates yielding similar results. Data was normalized as reads per million except the embryo Ph and H3K27me3 data in <xref ref-type="fig" rid="pgen.1006200.g001">Fig 1</xref>, normalization of this data is explained in Bowman et al. (Bowman <italic>et al</italic>., 2014 [<xref ref-type="bibr" rid="pgen.1006200.ref038">38</xref>]). The same H3K27me3 data was done in several figures (1, 2D, 3A, 5A). Peak calling for Pho was conducted using MACS v1.4.2 [<xref ref-type="bibr" rid="pgen.1006200.ref065">65</xref>]. Shifting model building for prediction of fragment length was turned off (&#8212;nomodel) and a fixed standard background model for peak calling was used to increase sensitivity (&#8212;nolambda). In addition, we specified a p-value cutoff of 0.00001 for the output peaks. For in depth analysis of the binding regions, output peaks were split with Peaksplitter v0.1 [<xref ref-type="bibr" rid="pgen.1006200.ref040">40</xref>] and ~20% (peak height&#8805; 50) of the split peaks were considered for further analysis to rank them into null, weak and strong binding sites. Sequencing data have been deposited to the Gene Expression Omnibus.</p><p>Chromosome conformation capture (3C) protocol was followed as described by Tolhuis <italic>et al</italic>. [<xref ref-type="bibr" rid="pgen.1006200.ref066">66</xref>] with minor modifications. To generate 4C-library, we followed the protocol described by Tolhuis <italic>et al</italic>. [<xref ref-type="bibr" rid="pgen.1006200.ref067">67</xref>]. Brains and disks from 10 third instar larvae were fixed using freshly prepared fixing solution containing 2% formaldehyde and 1X PBS prepared from 16% formaldehyde (Ted Pella, Inc.). Fixation reaction was quenched using 125 mM Glycine. After fine dissection and nuclei isolation, nuclei pellet was dissolved in 1X NEB buffer 3. After the SDS treatment the chromatin was digested overnight with 5 &#956;l EcoRI (200 U, NEB). Next day, after inactivating EcoRI, the digested chromatin was ligated for 6 hrs with 0.5 &#956;l ligase (200 U, NEB) in 2ml of ligation reaction. The ligated library was purified as stated in the reference mentioned above. Finally the library was dissolved in 400 &#956;l ultrapure water or 10 mM Tris-HCl (pH 8.0).</p><p>100 &#956;l of the 3C library was digested with 5 &#956;l of DpnII (50 U, NEB). After the enzyme was inactivated, digested chromatin was re-ligated using 5 &#956;l of ligase (200 U, NEB) in a total volume of 2 ml ligation reaction to promote self-ligation events. Finally purification of the library was carried out. Inverse PCR was carried out using 8 different custom made reverse primers (containing unique 5 nucleotide bar code) and one universal forward primer to produce Illumina-4C-seq libraries. The primer sequence used for 4C-seq library amplification is mentioned in <xref ref-type="supplementary-material" rid="pgen.1006200.s006">S2 Table</xref>. PCR amplification was carried out using Kapa HiFi HotStart Ready Mix (KAPA Biosystems, cat no. KK2601) with 90 ng of DNA and 2.5 &#956;l of forward primer (10mM) and 1.7 &#956;l of reverse primer (10mM). Amplification program in thermal cycler was: denaturation- 95&#176;C for 30 sec, amplification- 25 cycles of 98&#176;C for 10 sec, 64&#176;C for 30 sec and 72&#176;C for 30 sec; final elongation step was 72&#176;C for 1 min. After pooling together 6 PCR reactions, it was purified using High Pure PCR Product Purification kit (Roche, cat no. 11732676001). Finally 4C-PCR products from WT and mutant lines were combined in preferred ratios for sequencing.</p><p>Procedure to detect significant interactions (peaks) in 4C was adapted from Tolhuis <italic>et al</italic>. and Ghavi-Helm <italic>et al</italic>. [<xref ref-type="bibr" rid="pgen.1006200.ref051">51</xref>, <xref ref-type="bibr" rid="pgen.1006200.ref067">67</xref>]. Read counts were transformed using variance stabilizing transformation from the DESeq2 package in R [<xref ref-type="bibr" rid="pgen.1006200.ref068">68</xref>] and then transformed counts from chr2R (chromosome with bait fragment) were smoothed using loess. Region immediately surrounding bait fragment was excluded from fit (chr2R:7410000..7416000). Interactions with p-value &lt; 0.01 in both replicates and FDR &lt; 0.1 in at least one replicate were selected as bona-fide interactions.</p><p>Reads from all the NGS data were deposited in the NCBI-GEO under accession numbers GSE77342.</p>