<p>We used clinically well-characterized subjects from 6 pretreatment HIV/AIDS cohorts. Study participants (N = 4203) were enrolled in USA-based, prospective natural history HIV/AIDS cohorts: Multicenter AIDS Cohort Study (MACS) [<xref ref-type="bibr" rid="pgen.1005921.ref062">62</xref>], the San Francisco City Clinic Cohort Study (SFCCC) [<xref ref-type="bibr" rid="pgen.1005921.ref063">63</xref>], AIDS Link to the Intravenous Experience (ALIVE) [<xref ref-type="bibr" rid="pgen.1005921.ref064">64</xref>], Hemophilia Growth and Development Study (HGDS) [<xref ref-type="bibr" rid="pgen.1005921.ref065">65</xref>], the Multicenter Hemophilia Cohort Study (MHCS) [<xref ref-type="bibr" rid="pgen.1005921.ref066">66</xref>] and DC gay (DCG) [<xref ref-type="bibr" rid="pgen.1005921.ref067">67</xref>]. These cohorts were initiated during 1980&#8217;s and participants were followed up semi-annually with blood draws for viral load and CD4+ T cell measurements and physical examinations at each visit. These treatment na&#239;ve cohorts have been used to identify multiple HIV-1/AIDS modifying genetic factors [<xref ref-type="bibr" rid="pgen.1005921.ref030">30</xref>,<xref ref-type="bibr" rid="pgen.1005921.ref068">68</xref>].</p><p>The study group includes 707 European American HIV-1 incident seroconverters, 281 African American incident seroconverters, 1135 HIV-uninfected at-risk individuals and 2076 HIV seroprevalent individuals who were HIV infected at study entry. The censoring date was the earliest of the date of the last recorded visit, or July 31, 1997 for the ALIVE cohort, or December 31, 1995 for all other cohorts, to avoid the confounding effect of highly active anti-retroviral therapy (HAART). A later censoring date was used for ALIVE cohort because few ALIVE participants received HAART prior to July 31, 1997 [<xref ref-type="bibr" rid="pgen.1005921.ref069">69</xref>]. PCP prophylaxis status was only available for a subset of the MACS cohort; the first self-reported drugs used included Trimethoprim/sulfamethoxazole, aerosolized Pentamidine and Dapsone; the combination or switch-over of these drugs were used over the clinical course.</p><p>Ethical approval for the study was obtained from the National Institute of Health Office of Human Subjects Research Protections with OHSRP #3314. Ethical approval was obtained from institutional review boards for each of the respective contributing centers in the International Collaboration for the Genomics of HIV. Written informed consent was obtained from all study participants.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We selected six SNPs that were codon-changing, in a predicted splicing or regulatory site (based on SNPinfo web server [<ext-link ext-link-type="uri" ns0:href="http://snpinfo.niehs.nih.gov/">http://snpinfo.niehs.nih.gov</ext-link>]), considering SNP spacing and gene coverage. Several other nonsynonymous variants in A3F are reported in the 1000 genome project but were not genotyped here either because they were rare or tagged by other SNPs: rs2020390 (p. A108S) is in near-absolute LD (<italic>r</italic><sup>2</sup> = 0.94) with rs2076101 (p.I231V) in Europeans. rs34182094 (p. A178T), rs12157816 (p. Y307C), rs13056825 (p.N346S) were absent or infrequent in Utah Europeans (CEU) and Yoruba Africans (YRI).</p><p>SNPs were genotyped using the TaqMan allele discrimination assays on an ABI 7900HT sequencer detector system (Applied Biosystems), according to the manufacturer&#8217;s protocol. For quality control, water controls were included on each plate and 10% of samples were duplicated. No water contamination or genotype mismatches between duplicates was observed. SNP rs2076101 was also genotyped by a second in-house designed TaqMan assay (sequences available upon request); the resulted genotypes were identical by two assays.</p><p>EMSA was performed in nuclear extracts from HeLa cell stimulated with PMA using oligonucleotides carrying <italic>A3F</italic> rs5750728 T/C with an Infrared EMSA Kit (LI-COR), as previously described [<xref ref-type="bibr" rid="pgen.1005921.ref070">70</xref>]. The double-stranded oligonucleotides used were (SNP allele underlined): 5&#8217;-CCCTTTGGCCAGTGCG<underline>T</underline>CCCACCACAT-3&#8217; and 5&#8217;-CCCTTTGGCCAGTGCG<underline>C</underline>CCCACCACAT-3&#8217;.</p><p>HEK293T (ATCC, catalog no. CRL-11268) cells were maintained in Dulbecco's modified Eagle's medium (DMEM, Invitrogen) with 10% fetal bovine serum and penicillin/streptomycin (D-10 medium) and passaged upon confluence. DNA transfection in HEK293T cells was carried out using Lipofectamine 2000 (Invitrogen) as recommended by the manufacturer. For degradation assays, HEK293T cells in 12-well plates were transfected with 1 &#956;g of Vif or 1 &#956;g of empty vector and 0.3 &#956;g of A3F expression vector at a 3:1 ratio. HEK293T cells were collected at 48 h after transfection. Cell lysates were lysed in 1x loading buffer (0.08 M Tris, pH 6.8, with 2.0% SDS, 10% glycerol, 0.1 M dithiothreitol and 0.2% bromophenol blue) and boiled for 5 min before separating the proteins by SDS-PAGE. Membranes were probed with various primary antibodies against proteins of interest. Secondary antibodies were alkaline phosphatase-conjugated anti-sheep, anti-rabbit or anti-mouse (Jackson Immunoresearch) antibodies, and staining was carried out with 5-bromo-4-chloro-3-indolyl phosphate (BCIP) and nitroblue tetrazolium (NBT) solutions prepared from chemicals obtained from Sigma.</p><p>HIV-1 sequences covering the <italic>pol</italic> gene alone (in half of the patients) or the <italic>pol</italic> gene with additional other genes were obtained by the Sanger sequencing from plasma of patients in the Swiss HIV Cohort Study. A3F specific hypermutation was quantified as in the Los Alamos Hypermut2 tool [<xref ref-type="bibr" rid="pgen.1005921.ref038">38</xref>]. Consensus nucleotide sequences of the samples and of the hxb2 reference sequence were translated to amino acid sequence from which a multiple alignment was created with muscle [<xref ref-type="bibr" rid="pgen.1005921.ref071">71</xref>]. Using the amino acid sequence alignment we derived a codon-aware nucleotide alignment, which was used to count the number of possible hypermutation sites. We counted the number of trinucleotides matching the &#8220;GA[A or T or G]&#8221; pattern in the reference sequence as the number of potential A3F hypermutation sites. The number of actual A3F induced mutations in each sample was determined as a subset of potential A3F hypermutation sites where the first nucleotide was an A. It should be noted that GA &#8594; AA editing can be mediated by A3D/F/H [<xref ref-type="bibr" rid="pgen.1005921.ref039">39</xref>,<xref ref-type="bibr" rid="pgen.1005921.ref041">41</xref>]. To quantify the background mutation level we used the following regular expression, which is the same as in the Hypermut2 tool: &#8220;G(([CT][ACGT])|([AG]C))&#8221;. The level of hypermutation of a patient sample was determined as an odds ratio: (number of A3F induced mutations/number of potential A3F specific sites) / (number of background mutations / number of potential background sites). Statistical tests were carried out on the logarithm of the latter quantity using linear regression. The regression included host PCA axes as covariates, the allele dosage of rs2076101, and an indicator variable indicating the SNP genotyping platform.</p><p>We performed analyses using SAS version 9.12 (SAS Institute, Cary, NC).</p><p>We assessed the influences of the <italic>A3F</italic> variants on disease progression to AIDS outcomes by the Cox proportional hazards model (Cox model) and Kaplan-Meier survival curve analyses. We only included seroconverters with known infection dates (midpoint estimation between last seronegative and first seropositive HIV test) in our analysis. The endpoint was clinical AIDS diagnosis using the Centers for Disease Control and Prevention (CDC) 1987 definition of AIDS [<xref ref-type="bibr" rid="pgen.1005921.ref034">34</xref>], i.e. HIV-1 infection plus AIDS-defining illness or AIDS-related death. The median time from seroconversion to AIDS was 10 years in European American seroconverters. As the Kaplan-Meier survival analysis indicated that the haplotype tagged by rs2076101 best fitted a dominant genetic model, we compared the heterozygous (231I/V) and homozygous genotype (231V/V) state to the reference group of homozygotes (231I/I) in a Cox proportional hazards model. European American and African American groups were analyzed separately because the allele frequencies were different between the two groups. We included known genetic factors modifying AIDS progression as covariates in the adjusted Cox model analysis: <italic>CCR5</italic> &#916;32, <italic>CCR5</italic>-59029 (<italic>CCR5</italic>-2459, rs1799987), <italic>HLA</italic>-<italic>B</italic>*27, <italic>HLA</italic>-<italic>B</italic>*57, <italic>HLA</italic>-<italic>B</italic>*35Px group (including <italic>HLA</italic>-<italic>B</italic>*3502, <italic>B</italic>*3503, <italic>B</italic>*3504, and <italic>B</italic>*5301), <italic>HLA-C</italic> [<xref ref-type="bibr" rid="pgen.1005921.ref032">32</xref>] and <italic>HLA</italic> Class I homozygosity for European American (reviewed in [<xref ref-type="bibr" rid="pgen.1005921.ref072">72</xref>]; <italic>HLA</italic>-<italic>B</italic>*57 and <italic>HLA</italic> Class I homozygosity for African American. We stratified the analyses by sex and by age at seroconversion: 0&#8211;20, &gt;20&#8211;40, and &gt; 40 years. P values for Cox model analysis were from the Wald test. Although our previous GWAS for association with HIV phenotypes in EA indicated minimal population substructure (genomic inflation factor &#955; = 1.01), we conservatively include the first two eigenvalues from a PCA analysis (EigenSoft) to correct for population stratification [<xref ref-type="bibr" rid="pgen.1005921.ref073">73</xref>].</p><p>HIV-1 viral load set-point was defined as the mean log<sub>10</sub>-transfromed copies of HIV-1 RNA in plasma measured between months 6&#8211;33 after seroconversion (2&#8211;5 measurements). Viral load measurements exceeding three-fold (0.5 log<sub>10</sub>) from the average of all remaining points were excluded, as previously suggested [<xref ref-type="bibr" rid="pgen.1005921.ref035">35</xref>]. We used the <italic>t</italic> test for analysis of differences between viral load means of the group carrying 0 or 1 231V allele versus the 231I homozygote group.</p><p>The International Collaboration for the Genomics of HIV (ICGH) combined the GWAS data from 25 cohorts from Europe and North America, including cohorts used in this study [<xref ref-type="bibr" rid="pgen.1005921.ref036">36</xref>]. The genotypes of the <italic>A3F</italic> SNPs from ICGH were obtained from the previous GWAS data [<xref ref-type="bibr" rid="pgen.1005921.ref036">36</xref>]. The impact of the <italic>A3F</italic> SNPs on HIV-1 viral load in ICGH was assessed using a fixed-effect inverse-variance weighted meta-analysis as previously described [<xref ref-type="bibr" rid="pgen.1005921.ref036">36</xref>]. A summary of the selected study groups and cohorts used in the meta-analysis is presented in <bold><xref ref-type="supplementary-material" rid="pgen.1005921.s003">S3 Table</xref></bold>. Viral load data from a total of 10395 (7266 European Americans, 3129 African Americans) HIV-1 seropositives was used in the meta-analysis.</p><p>The impact of <italic>A3F</italic> variants on HIV-1 infection susceptibility was assessed by comparing allelic frequencies between the HIV-1 infected group comprising seroconverter and seroprevalent persons and the HIV-1 uninfected group composed of persons at risk for HIV. Odds ratios (OR) and <italic>P</italic> values were obtained by using a conditional logistic regression test. All <italic>P</italic> values were 2-tailed.</p>