<p>See <xref ref-type="table" rid="pgen-1004532-t001">Table 1</xref>. pJS757 is a <italic>LEU2</italic>-based <italic>CEN</italic> vector carrying the <italic>RPB1</italic> ORF, as well as 594 bases upstream and 501 bases downstream of the ORF <xref ref-type="bibr" rid="pgen.1004532-Strathern2">[16]</xref>. pJS725 and pJDI220 are <italic>URA3</italic>-based 2-micron circle plasmids with the same <italic>RPB1</italic> insertion as pJS757. <italic>RPB1</italic> was randomly mutagenized by passing a pool of plasmid pJS757 twice through the mutator (<italic>mutS mutD mutT</italic>) <italic>E. coli</italic> strain XL-1 Red (Stratagene).</p><p>The yeast strains are related to the BY4741 and BY4742 <xref ref-type="bibr" rid="pgen.1004532-Brachmann1">[68]</xref> See <xref ref-type="table" rid="pgen-1004532-t001">Table 1</xref>.</p><p>The <italic>cre</italic> gene used in these experiments has a seven amino acid nuclear localization site from the large SV40 T-antigen added at the N-terminus <xref ref-type="bibr" rid="pgen.1004532-Lewandoski1">[69]</xref>. The <italic>P<sub>GAL1</sub>-cre</italic> gene was made by fusing the <italic>NLS-cre</italic> ORF to the <italic>P<sub>GAL1</sub></italic> promoter inserted into the <italic>BUD5</italic> gene at an autochthonous <italic>EcoR1</italic> site. The <italic>P<sub>HO</sub>-cre</italic> gene was made by overlap PCR and integrated at the normal position for <italic>HO</italic>. The insertions were made by selecting for replacement of <italic>URA3</italic>, previously inserted at those locations.</p><p>
<italic>Direct repeat of loxP:</italic> The <italic>his3-AI2floxkanMX</italic> reporter (<xref ref-type="fig" rid="pgen-1004532-g001">Figure 1B</xref>, construct (iii)) has an artificial intron inserted into <italic>HIS3</italic> at an <italic>MscI</italic> site in codon 70. The 1710 base insertion includes two loxP sites in direct orientation flanking the <italic>kanMX</italic> cassette. <italic>kanMX</italic> is in the same orientation as <italic>HIS3</italic> so that the terminator blocks transcription through the rest of <italic>HIS3</italic> resulting in cells that are His<sup>&#8722;</sup>. The Cre recombinase can remove the 1502 bases including <italic>kanMX</italic> plus one loxP resulting in a spliceable intron (<italic>HIS3-AI2lox</italic>
<xref ref-type="fig" rid="pgen-1004532-g001">Figure 1B, (ii)</xref>) and cells that are His<sup>+</sup>. The <italic>ade6-AI2floxkanMX</italic> reporter has the same artificial intron inserted after the tenth codon of <italic>ADE6</italic> causing cells to be <italic>ade6<sup>&#8722;</sup></italic>, but Cre recombinase can remove <italic>kanMX</italic> and make a functional <italic>ADE6-AI2lox</italic> gene. <italic>Inverted repeat of loxP:</italic> The <italic>his3-AI2floxINV</italic> substrate (<xref ref-type="fig" rid="pgen-1004532-g001">Figure 1B, (iv)</xref>) was constructed on chromosome XV at the <italic>HIS3</italic> locus and surrounding region. This reporter contains the N-terminal portion of <italic>HIS3-AI2</italic> and the splice donor part of the artificial intron ending at the loxP site and is marked by hygromycin resistance (<italic>hygMX</italic>). A lox site, the splice acceptor portion of the intron and the C-terminal portion of the <italic>HIS3-AI2</italic>, marked by zeomycin resistance (<italic>zeoMX</italic>), was inserted in inverted orientation 682 bases downstream of the <italic>GEP3</italic> gene (2.8 kb beyond <italic>HIS3</italic>). Cre-directed recombination inverts the floxed DNA (including the <italic>DED1</italic> gene) creating the functional <italic>HIS3-AI2lox</italic> gene (<xref ref-type="fig" rid="pgen-1004532-g001">Figure 1B-v</xref>). Hygromycin (<italic>hygMX</italic>) is from pAG32 <xref ref-type="bibr" rid="pgen.1004532-Goldstein1">[70]</xref> and Zeocin (<italic>zeoMX</italic>) is from pHybLex/Zeo (Invitrogen).</p><p>Pol II variants carrying wild type or mutant Rpb1 were introduced into a protease-deficient yeast strain GRY3175 by plasmid shuffle. Hexahistidine-tagged Rpb3 <xref ref-type="bibr" rid="pgen.1004532-Kireeva5">[57]</xref> was used to pull down Pol II from the whole cell lysate essentially as described <xref ref-type="bibr" rid="pgen.1004532-Kireeva4">[54]</xref>. Specifically, cells from 3&#8211;4 ml of stationary (2&#8211;3 days) culture were pelleted, washed once with cold water and resuspended in lysis buffer (150 mM Tris&#8211;acetate, pH 7.9, 50 mM potassium acetate, 5 mM MgCl<sub>2</sub>, 10 &#181;M ZnCl<sub>2</sub>, 2 mM 2-mercaptoethanol, 0.5 mM EDTA and protease inhibitors). The cells were disrupted using a Precellys homogenizer according to manufacturer's instructions, the lysate was removed from the glass beads, and KCl was added to 1M final concentration. The debris was precipitated for 15 min at 14,000 rpm in Eppendorf table-top microcentrifuge at 4&#176;C. The supernatant was added to 50 &#181;l of Ni-NTA agarose pre-washed with transcription buffer (TB) containing 20 mM Tris-HCl, pH 7.9, 5 mM MgCl<sub>2</sub>, 10 &#181;M ZnCl<sub>2</sub>, 2 mM 2-mercaptoethanol and 1000 mM KCl (TB1000). Pol II was immobilized on the beads for 30&#8211;50 min at 4&#176;C, and the beads were extensively washed with TB1000 and TB40 (TB with 40 mM KCl). The elongation complex was assembled and purified with the immobilized Pol II exactly as described <xref ref-type="bibr" rid="pgen.1004532-Kireeva4">[54]</xref> using the following oligonucleotides: Cre RNA C9: <named-content content-type="gene">5&#8242; GUC AUG AAC 3&#8242;</named-content>(phosphorylated with T4 polynucleotide kinase in the presence of &#947;-P32-labeled ATP); NDS-Cre-Cys (Non Transcribed DNA Strand) <named-content content-type="gene">5&#8242;GGCTGGACCAATGTAAATATT<underline>GTCATGAAC</underline> TGTATCCGTAA CCTGGATAGTGAAACA 3&#8242;</named-content>; and TDS-Cre-Cys (Transcribed DNA Strand) which is an exact complement of NDS (Cre-Cys). All NTPs used for walking and misincorporation assays were additionally purified <xref ref-type="bibr" rid="pgen.1004532-Kireeva1">[19]</xref>. The elongation complex containing 10-nt RNA (U10) was obtained by 5 min incubation of the assembled and purified complex with 5 &#181;M UTP and washed 4 times with 1 ml of TB40. For the TFIIS sensitivity assay, mismatched A11 elongation complex was obtained by 5 min incubation of U10 with 1 mM ATP, purified by 4 washes with TB40, eluted from the beads with 100 mM imidazole in the presence of 0.2 mg/ml acetylated BSA, diluted 10-fold with TB40 to decrease imidazole concentration and concentrated using Microcon AmiconUltra (Millipore) concentrator to 30 &#181;l. Reactions were stopped by addition of equal volume of gel loading buffer containing 10 M urea and 50 mM EDTA. The reaction products were resolved in 20% polyacrylamide gels (19&#8758;1) in the presence of 1&#215; TBE and 7M urea. To resolve the two 11-nt products of cognate GMP and mismatched AMP incorporation in the competition fidelity assay, the 4-mm thick, 20&#215;40 cm gels were used, and electrophoresis has been performed in 1&#215; TBE at constant power of 65 W until the bromphenol blue tracing dye was within 3 cm of the bottom of the gel. For other experiments the shorter 20&#215;20 cm gels were used and the electrophoresis was performed at 50 W constant power.</p>