<p>Yeast and bacterial culture methods were as previously described [<xref ref-type="bibr" rid="pgen.1005528.ref106">106</xref>, <xref ref-type="bibr" rid="pgen.1005528.ref107">107</xref>] except where noted. Bacterial strains HB101 or DH5&#945; were used for plasmid preparation. All <italic>S</italic>. <italic>cerevisiae</italic> strains were derivatives of BY4741 (<italic>MAT</italic>
<bold>a</bold>
<italic>his3&#916;1 leu2&#916;0 met15&#916;0 ura3&#916;0</italic>). (<xref ref-type="supplementary-material" rid="pgen.1005528.s007">S1 Table</xref>). BY4741 derivatives included strains deleted for Ty3 elements and killer RNA, and strains tagged with fluorescent reporter proteins. Manipulation of strains and plasmids utilized standard molecular genetics. For all &#945;-factor induction, yeast cultures were grown at 24&#176;C to OD<sub>600</sub> of 0.2 and &#945;-factor was added to 6 &#956;M final concentration. A complete description of procedures is provided (<xref ref-type="supplementary-material" rid="pgen.1005528.s014">S1 Text</xref>, Supplemental Materials and Methods; <xref ref-type="supplementary-material" rid="pgen.1005528.s015">S2 Text</xref>, Plasmid construction and sequences; and <xref ref-type="supplementary-material" rid="pgen.1005528.s008">S2 Table</xref>, Primers used in this study and <xref ref-type="supplementary-material" rid="pgen.1005528.s009">S3 Table</xref> Plasmids used in this study.</p><p>Polyclonal rabbit antibodies (Berkeley Antibody Company) were raised against WT Ty3 VLPs purified as previously described [<xref ref-type="bibr" rid="pgen.1005528.ref057">57</xref>] and affinity purified using recombinant Gag3 (this work). Polyclonal rabbit antibodies against Ty3 CA [<xref ref-type="bibr" rid="pgen.1005528.ref060">60</xref>] were used for immunoblot analysis and immunofluorescence. A complete description is provided (<xref ref-type="supplementary-material" rid="pgen.1005528.s014">S1 Text</xref>, Supplemental Materials and Methods).</p><p>Gag3-associated proteins were purified by &#945;-VLP and control IgG affinity chromatography of whole cell extracts. Co-purifying proteins were digested with LysC and trypsin as described [<xref ref-type="bibr" rid="pgen.1005528.ref108">108</xref>]. Peptides were analyzed by 1DLC-MS/MS using LTQ-Orbitrap XL MS (ThermoElectron) as described [<xref ref-type="bibr" rid="pgen.1005528.ref108">108</xref>]. Database searching was performed using Batch-Tag and search Compare within the developmental version (v.5.2.2) of Protein Prospector as described. Proteins were identified by at least two peptides with a false-positive rate of 0.1%. A total of 154 proteins were identified of which 106 are not essential (<xref ref-type="supplementary-material" rid="pgen.1005528.s010">S4 Table</xref>). A complete description is provided (<xref ref-type="supplementary-material" rid="pgen.1005528.s014">S1 Text</xref>, Supplemental Materials and Methods).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">GO term enrichment analysis was performed using the Gene Ontology Term Finder (<ext-link ext-link-type="uri" ns0:href="http://www.yeastgenome.org/cgi-bin/GO/goTermFinder.pl">http://www.yeastgenome.org/cgi-bin/GO/goTermFinder.pl</ext-link>) and the SGD Gene Ontology Slim Mapper(<ext-link ext-link-type="uri" ns0:href="http://www.yeastgenome.org/cgi-bin/GO/goSlimMapper.pl">http://www.yeastgenome.org/cgi-bin/GO/goSlimMapper.pl</ext-link>). Genes were sorted according to biological process, molecular function, and cellular component. For Gene Ontology Term Finder, threshold P-value of 0.01 were used to identify specific enrichment.</p><p>Gag3-associated RNA interaction network was visualized by Cytoscape [<xref ref-type="bibr" rid="pgen.1005528.ref109">109</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Gag3-interacting P-body proteins (boxes) were queried for protein-protein physical interactions using SGD:Yeastmine (<ext-link ext-link-type="uri" ns0:href="http://yeastmine.yeastgenome.org/yeastmine/begin.do">http://yeastmine.yeastgenome.org/yeastmine/begin.do</ext-link>). Cytoscape v3.1.0 (<ext-link ext-link-type="uri" ns0:href="http://www.cytoscape.org/">http://www.cytoscape.org/</ext-link>) was used to graphically map interactions (<xref ref-type="supplementary-material" rid="pgen.1005528.s001">S1 Fig</xref>). Additional details including gene names of all interacting partners used to create the network map (<xref ref-type="supplementary-material" rid="pgen.1005528.s012">S6 Table</xref>).</p><p>BY4741 or derivatives were transformed with a plasmid expressing Ty3 marked with a <italic>HIS3</italic> gene inactivated by the presence of an antisense artificial intron (Ty3-<italic>his3</italic>AI) [<xref ref-type="bibr" rid="pgen.1005528.ref062">62</xref>, <xref ref-type="bibr" rid="pgen.1005528.ref110">110</xref>] under the native promoter (pDM3193). Strains were grown in SD-ura at 24&#176;C to OD<sub>600</sub> = 0.2, induced with &#945;-factor and plated on synthetic dextrose medium (SD)-his, to select for cells that had acquired a chromosomal copy of Ty3-HIS3 in which the artificial intron was spliced out, and plated onto rich medium (1% yeast extract, 2% peptone, 2% dextrose) for total cell counts. At least four independent transformants were assayed for each strain. Transposition frequency was calculated as the frequency of His<sup>+</sup> colonies and <italic>p</italic> values determined using the student t-test.</p><p>Cells were grown at 24&#176;C to OD<sub>600</sub> = 0.2 and induced with 6 &#956;M &#945;-factor (GenScript USA Inc.) for 4 h or left uninduced as control. Fluorescent images of live cells were visualized using either a Zeiss Axioplan2 or a Zeiss LSM510 META inverted confocal scanning Plan-Apochromat fluorescence microscope. FISH was performed with an adaptation of previous methods [<xref ref-type="bibr" rid="pgen.1005528.ref084">84</xref>, <xref ref-type="bibr" rid="pgen.1005528.ref111">111</xref>, <xref ref-type="bibr" rid="pgen.1005528.ref112">112</xref>]. Ty3 mRNA was detected with an equal parts mixture of antisense oligonucleotides complementary to coding sequence in Ty3 Gag3, RT, and IN. Ty3 protein was visualized using &#945;-VLP followed by Alexa-fluor 568-conjugated donkey anti-rabbit IgG (Life Technologies). Cells were stained with 3 &#956;g/ml DAPI (4&#8217;, 6-diamidino-2-phenylindole). FISH was imaged using a Zeiss LSM780 Plan-Apochromat inverted confocal laser scanning microscope. Images were processed for publication with Adobe Photoshop CS3 (Adobe Systems Inc.). For all experiments, images from at least three replicate samples consisting of at least 100 cells each were analyzed, and single plane images are shown. A complete description is provided (<xref ref-type="supplementary-material" rid="pgen.1005528.s014">S1 Text</xref>, Supplemental Methods and Materials).</p><p>Quantitative mating assays were done essentially as described [<xref ref-type="bibr" rid="pgen.1005528.ref113">113</xref>]. In general: Cell cultures were grown at 23&#176;C to OD<sub>600</sub> = 0.2&#8211;0.3 and 10<sup>6</sup> cells of each mating type were mixed, collected onto 25mm-GF/C filter disc (Whatman), and incubated at 30&#176;C for the indicated times. The cells were eluted from the filter in 1 ml medium. For mating efficiency, 100 &#956;l was plated on medium selective for complementation of auxotrophic markers in <italic>MAT</italic>
<bold>a</bold> and <italic>MAT&#945;</italic> cells upon cell fusion. To determine the number of unmated cells, eluted cells were diluted 1&#8211;1,000 and plated onto the appropriate medium. Cells were incubated at 30&#176;C for several days and colonies were counted. Mating efficiency (%) was calculated as the ratio of diploid cells to unmated cells. For specific experiments: For mating of Ty3(WT) BY4741 (<italic>MAT</italic>
<bold>a</bold>) x BY4742(<italic>MAT&#945;</italic>) and Ty3&#916; yVB1913 (<italic>MAT</italic>
<bold>a</bold>) x yDM1456(<italic>MAT&#945;</italic>), cells were grown in YPD medium, mated on YPD medium for 1.5 and 2.5 h. Cells were eluted in SD medium lacking methionine and lysine, plated the same medium for mated cells, and on SD lacking methionine or lysine for unmated cells. For the effect of high level of Ty3 expression on mating, Ty3&#916; strains yVB1672 (<italic>MAT</italic>
<bold>a</bold>) and yVB1926 (<italic>MAT&#945;</italic>) were transformed with either a high-copy-number plasmid containing Ty3 (pDM3194) or a plasmid control (pYES2.0). Cells were grown in SD medium lacking uracil, mated on the same medium for 2.0 or 3.5 h. Cells were eluted in SD medium lacking histidine and methionine and plated on the same medium for mated cells, and on SD lacking histidine or methionine for unmated cells.</p><p>RNA, protein and packaging, and cDNA levels were measured essentially as described [<xref ref-type="bibr" rid="pgen.1005528.ref010">10</xref>]. Statistical significance was determined using the program R [<xref ref-type="bibr" rid="pgen.1005528.ref114">114</xref>] to perform one way ANOVA, with planned contrast between WT and individual mutant samples compared.</p><p>For polysome profilings, cells were grown at 24&#176;C in YPD to OD<sub>600</sub> = 0.2, induced with 6 &#956;M &#945;-factor (GenScript USA Inc.) for 2 h. Polysomes were prepared essentially as described [<xref ref-type="bibr" rid="pgen.1005528.ref115">115</xref>]. Extracts from 10&#8211;12 OD<sub>260</sub> cells were fractionated on a 4&#8211;47% linear sucrose gradient. A complete description is provided (<xref ref-type="supplementary-material" rid="pgen.1005528.s014">S1 Text</xref>, Supplemental Materials and Methods).</p>