<p>Wood wafers (1 cm by 1 cm by 2 mm) were cut from the sapwood of aspen (<italic>Populus tremuloides</italic>), pine (<italic>P. taeda</italic>) and spruce (<italic>Picea glauca</italic>) and sterilized by autoclaving. Following inoculation by contact with mycelium growing on malt extract agar (15 g malt extract [Difco, Detroit, MI] and 15 g agar per liter of water) in Petri dishes, colonized wafers were harvested 30, 60 and 90 days. Noninoculated wood wafers placed on the same media in Petri dishes served as controls. Wafers were removed 30, 60 or 90 days later, weighed and percent weight loss was determined. Additional wafers were removed at the same time period, immediately frozen to &#8722;20&#176;C and prepared for scanning electron microscopy as previously described <xref ref-type="bibr" rid="pgen.1004759-Blanchette2">[68]</xref>.</p><p>The genome was sequenced using Illumina and annotated using the JGI Annotation Pipeline <xref ref-type="bibr" rid="pgen.1004759-Grigoriev2">[69]</xref>. Assembly and annotations are available from JGI portal MycoCosm <xref ref-type="bibr" rid="pgen.1004759-Grigoriev1">[38]</xref> and deposited to DDBJ/EMBL/GenBank under accession AZAG00000000. The version described in this paper is version AZAG01000000. The completeness of the <italic>P. gigantea</italic> genome was assessed by finding 99.1% of CEGMA proteins conserved across sequenced genomes of eukaryotes <xref ref-type="bibr" rid="pgen.1004759-Parra1">[70]</xref>(<xref ref-type="supplementary-material" rid="pgen.1004759.s057">Text S1</xref>; <xref ref-type="supplementary-material" rid="pgen.1004759.s036">Tables S1</xref>, <xref ref-type="supplementary-material" rid="pgen.1004759.s037">S2</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Mycelium was derived from triplicate cultures of 250 ml basal salts containing: i. 1.25 g freshly-harvested, ground (1mm mesh) loblolly pine wood that had been &#8216;spiked&#8217; with acetone and thoroughly dried (NELP); or ii. the same material following extended acetone extraction in a Soxhlet apparatus and drying (ELP). The composition of the extract (<xref ref-type="supplementary-material" rid="pgen.1004759.s057">Text S1</xref>) was determined by GC-MS <xref ref-type="bibr" rid="pgen.1004759-Gutierrez2">[51]</xref>. Duplicate cultures of basal salts medium with glucose as sole carbon source served as a reference. After 5 days incubation, total RNA was purified from frozen mycelium as described <xref ref-type="bibr" rid="pgen.1004759-FernandezFueyo1">[22]</xref>, <xref ref-type="bibr" rid="pgen.1004759-VandenWymelenberg2">[71]</xref>. Multiplexed libraries were constructed and sequenced on an Illumina HiSeq2000. DNAStar Inc (Madison, WI) modules SeqNGen and Qseq were used for mapping reads and statistical analysis. Transcriptome data was deposited to the NCBI Gene Expression Omnibus (GEO) database and assigned accession GSE53112 (Reviewer access via <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=ilovmswixtajjez&amp;acc=GSE53112">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=ilovmswixtajjez&amp;acc=GSE53112</ext-link>). Experimental details are provided in <xref ref-type="supplementary-material" rid="pgen.1004759.s057">Text S1</xref> and all transcriptome analyses are summarized in <xref ref-type="supplementary-material" rid="pgen.1004759.s059">Dataset S2</xref>.</p><p>With minor modification, NanoLC-MS/MS analysis identified extracellular proteins in culture filtrates as described <xref ref-type="bibr" rid="pgen.1004759-FernandezFueyo1">[22]</xref>, <xref ref-type="bibr" rid="pgen.1004759-Ryu1">[72]</xref>. For each of the two woody substrates (e.g NELP and ELP), cultures were harvested after 5, 7 and 9 days. Mass spectrometric protein identifications were accepted if they could be established at greater than 95.0% probability within 0.9% False Discovery Rate and contained at least two identified peptides. Protein probabilities were assigned by the Protein Prophet algorithm <xref ref-type="bibr" rid="pgen.1004759-Nesvizhskii1">[73]</xref>. To verify the effects of pine wood extractives in a well-defined substrate, media containing microcrystalline cellulose (Avicel) were also employed <xref ref-type="bibr" rid="pgen.1004759-FernandezFueyo1">[22]</xref>, <xref ref-type="bibr" rid="pgen.1004759-Martinez2">[45]</xref>, <xref ref-type="bibr" rid="pgen.1004759-VandenWymelenberg3">[74]</xref>. Filtrates from these cultures, with or without addition of loblolly pine wood acetone extract, were collected after 5 days and analyzed. Approximate protein abundance in each of the cultures was expressed as the number of unique peptide and the exponentially modified protein abundance index (emPAI) value <xref ref-type="bibr" rid="pgen.1004759-Ishihama1">[52]</xref> (See <xref ref-type="supplementary-material" rid="pgen.1004759.s057">Text S1</xref> for detailed methods).</p>