<p>Six discovery data sets including 13,312 samples were analyzed in the Stage 1. The participating discovery studies were the Age Gene/Environment Susceptibility Reykjavik Study (AGES, n = 1,548), the Cilento study (Cilento, n = 1,115), the Framingham Heart Study (FHS, n = 7,048), the Ogliastra Genetic Park (OGP, n = 897), the Prospective Investigation of the Vasculature in Uppsala Seniors Study (PIVUS, n = 945), and the Val Borbera study (VB, n = 1,759). Two additional studies, the Gioi population (Gioi, n = 470) and the Sorbs population (Sorbs, n = 659) provided data for an <italic>in-silico</italic> replication (Stage 2). Further a <italic>de-novo</italic> replication (Stage 3) was undertaken in the STANISLAS Family Study (SFS, n = 676) and in a sample of hypertensive adults (HT, n = 995) from the Biological Resources Center (BRC) IGE-PCV &#8220;Interaction G&#232;ne-Environment en Physiopathologie Cardio-Vasculaire. The participating cohorts are described further in Section 1 in <xref ref-type="supplementary-material" rid="pgen.1005874.s008">S1 Text</xref>. The local institutional ethics boards for each study approved the study design. Each subject signed an informed consent before participating to the study. Further details can be found in <xref ref-type="supplementary-material" rid="pgen.1005874.s007">S5 Table</xref>.</p><p>In the discovery and <italic>in-silico</italic> replication cohorts, genotyping was performed using various arrays, and imputation was carried out using the 1000 genome v3 as reference panel in all studies. Details of pre-imputation quality control parameters, genotyping platforms and imputation parameters for each study are provided in <xref ref-type="supplementary-material" rid="pgen.1005874.s003">S1 Table</xref>. In all cohorts blood samples were collected after an overnight fast, and serum/plasma samples were prepared and stored as described in Section 2 in <xref ref-type="supplementary-material" rid="pgen.1005874.s008">S1 Text</xref>. Serum VEGF levels (plasma VEGF were measured in SFS and HT) were measured using commercial ELISA assays as detailed in Section 2 in <xref ref-type="supplementary-material" rid="pgen.1005874.s008">S1 Text</xref>. The <italic>de-novo</italic> genotyping at SFS and HT was undertaken on a competitive allele specific PCR (KASP) chemistry array and variants were called using a FRET-based genotyping system.</p><p>In each individual study, a natural log-transformation of VEGF levels was applied. To do that, in a few studies (AGES, OGP, VB, and Sorbs) where some individuals had VEGF levels below the detection threshold of the assay, half the minimum value of VEGF found in that cohort was arbitrarily assigned to each such participant [<xref ref-type="bibr" rid="pgen.1005874.ref096">96</xref>]. The transformed trait, adjusted for age, sex and additional study-specific covariates (e.g. principal components associated with VEGF levels, study center for multi-site studies), was related to the variant dosages using a linear regression. Studies with familial correlation used linear mixed effect models to account for familial relatedness. Detailed information about the software used in each cohort is reported in the <xref ref-type="supplementary-material" rid="pgen.1005874.s003">S1 Table</xref>. An additive genetic model with 1 degree of freedom was applied. Study specific results of genome-wide per-variant associations underwent additional quality control prior to meta-analysis. Checking of file formatting, data plausibility, and distributions of test statistics and quality measurements was facilitated by the <italic>gwasqc</italic> function of the GWAtoolbox package v1.0.0 in R [<xref ref-type="bibr" rid="pgen.1005874.ref097">97</xref>]. Prior to the meta-analysis, variants with low minor allele frequency (&lt;1%) and poor imputation quality (r<sup>2</sup>&lt; 0.4) were removed.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Meta-analysis was performed in METAL using an effective sample size weighted Z-score method [<xref ref-type="bibr" rid="pgen.1005874.ref098">98</xref>]. This method was chosen over an inverse-variance meta-analysis because of different covariate-adjusted mean values and standard deviations in VEGF levels among studies. The results of meta-analysis were adjusted for genomic control inflation factor. To define the effective sample size, the product of the sample size and the imputation quality for each variant was calculated in each cohort [<xref ref-type="bibr" rid="pgen.1005874.ref099">99</xref>]. The sum of the product of each cohort divided by overall sample size represents the proportion of the effective sample size for each variant <xref ref-type="disp-formula" rid="pgen.1005874.e001">Eq (1)</xref>.
<disp-formula id="pgen.1005874.e001"><alternatives><graphic id="pgen.1005874.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005874.e001.jpg" /><ns1:math id="M1"><ns1:mrow><ns1:mrow><ns1:mo>[</ns1:mo><ns1:mrow><ns1:msubsup><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>C</ns1:mi></ns1:msubsup><ns1:msub><ns1:mi>N</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>&#215;</ns1:mo><ns1:msubsup><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow><ns1:mo>]</ns1:mo></ns1:mrow><ns1:mo>/</ns1:mo><ns1:mn>13</ns1:mn><ns1:mo>,</ns1:mo><ns1:mn>312</ns1:mn><ns1:mo>=</ns1:mo><ns1:mi>E</ns1:mi><ns1:mi>f</ns1:mi><ns1:mi>f</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mo>&#160;</ns1:mo><ns1:mi>s</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>p</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>e</ns1:mi><ns1:mo>&#160;</ns1:mo><ns1:mi>s</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>z</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow></ns1:math></alternatives><label>(1)</label></disp-formula>
where <italic>C</italic> is the total number of participating cohorts, <italic>i</italic> indicates the specific cohort, <italic>N</italic> is the sample size used for the variant association test, and r<sup><italic>2</italic></sup> is imputation quality of the variant. After completing initial quality control checks, 6,705,861 variants, each of which was informative at an effective sample size of &gt;70%, were included in the meta-analysis (Stage 1). The genomic control inflation factor of the metal analysis was 1.003. All variants having a p-value less than 5x10<sup>-8</sup> were considered to be genome-wide significant.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">To identify all independent associations within the loci reaching genome-wide significance, conditional analyses were performed in a forward stepwise fashion, examining the most significant association and including in successive association models the next most significantly associated variant (P&lt;5x10<sup>-8</sup>) in a specific region at each step (referred to as the top variant in <xref ref-type="disp-formula" rid="pgen.1005874.e002">Eq (2)</xref>). We repeated this process until no more genome-wide significant associations were found. The conditional analysis model follows the <xref ref-type="disp-formula" rid="pgen.1005874.e002">formula (2)</xref>.
<disp-formula id="pgen.1005874.e002"><alternatives><graphic id="pgen.1005874.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005874.e002.jpg" /><ns1:math id="M2"><ns1:mrow><ns1:mi>l</ns1:mi><ns1:mi>n</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>V</ns1:mi><ns1:mi>E</ns1:mi><ns1:mi>G</ns1:mi><ns1:mi>F</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mn>0</ns1:mn></ns1:msub><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mi>v</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>t</ns1:mi><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>n</ns1:mi></ns1:msubsup><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mi>C</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>e</ns1:mi><ns1:msub><ns1:mi>s</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>k</ns1:mi></ns1:msubsup><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mi>j</ns1:mi></ns1:msub><ns1:mi>T</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>p</ns1:mi><ns1:mo>&#8201;</ns1:mo><ns1:mi>v</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:msub><ns1:mi>t</ns1:mi><ns1:mi>j</ns1:mi></ns1:msub></ns1:mrow></ns1:math></alternatives><label>(2)</label></disp-formula>
where <italic>n</italic> is the number of covariates used in the primary GWAS, <italic>k</italic> is the number of steps. The conditional analysis was only performed in FHS because it represents the largest cohort in the meta-analysis. The final conditional analysis model included 10 independent variants with p-values less than 5x10<sup>-8</sup> in FHS.</p><p>Genome-wide significant variants identified in the conditional analysis were examined in the two <italic>in-silico</italic> replication cohorts and also carried forward to <italic>de-novo</italic> replication. Furthermore, for each suggestive locus (5x10<sup>-8</sup>&lt;P&lt;1x10<sup>-5</sup>) the lead variant was also examined in the <italic>in-silico</italic> replication sample, and those suggestive variants that reached a genome-wide significant p-value in a meta-analysis of the discovery and <italic>in-silico</italic> replication data (Stage 2) were also carried forward to the <italic>de-novo</italic> replication phase. To check for the presence of other independent variants in the suggestive regions, a clumping procedure implemented in PLINK [<xref ref-type="bibr" rid="pgen.1005874.ref100">100</xref>] was performed. The 1000-genome v3 genotypes were used as reference panel for LD calculation; the physical threshold for clumping was 1 Mb, and the r<sup>2</sup> threshold for clumping was 0.1.</p><p>For selected variants that failed <italic>de-novo</italic> genotyping, a proxy variant having either the highest linkage disequilibrium (LD) value, or the variant in the same region with the next lowest p-value was genotyped instead of the lead variant. We considered as replicated, all variants that reached a genome-wide significance level in the meta-analysis of the discovery and the <italic>in-silico</italic> and <italic>de-novo</italic> replication samples (Stage 3).</p><p>For the replicated variants, an inverse variance-weighted meta-analysis was also performed as a secondary analysis, including in the analysis all the discovery and replication cohorts.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The variants identified after replication stages were used to estimate, in each cohort, a genetic score associated with circulating VEGF levels by summing the product of the beta-estimate and genotype for each variant in a given individual <xref ref-type="disp-formula" rid="pgen.1005874.e003">Eq (3)</xref>.
<disp-formula id="pgen.1005874.e003"><alternatives><graphic id="pgen.1005874.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005874.e003.jpg" /><ns1:math id="M3"><ns1:mrow><ns1:mi>R</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>k</ns1:mi><ns1:mi>S</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>e</ns1:mi><ns1:mo>=</ns1:mo><ns1:msubsup><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mn>10</ns1:mn></ns1:mrow></ns1:msubsup><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>*</ns1:mo><ns1:mi>G</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>y</ns1:mi><ns1:mi>p</ns1:mi><ns1:msub><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:mrow></ns1:math></alternatives><label>(3)</label></disp-formula>
where <italic>i</italic> is the variant, <italic>&#946;</italic> is effect size of the variant in the cohort, and genotype is additively coded genotype of the variant. The proportion of phenotypic variance explained by the variants incorporated in the score was estimated fitting two linear mixed effect models, in which VEGF levels were regressed, respectively, on: 1) gender and age (basic model); 2) gender, age, and genetic risk score (risk score model). The variance explained by the replicated variants was estimated as the difference between the variance explained by the risk score model and that explained by the basic model. The <italic>lmekin</italic> function (R package), which uses the genomic kinship matrix to correct for relatedness between individuals, if any, was applied.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The replicated SNPs and variants in LD with them (r<sup>2</sup>&gt;0.8) were investigated for the presence of chromatin histone marks and hypersensitive DNAse elements using data from ENCODE included in Haploreg_v3 software (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/mammals/haploreg/haploreg_v3.php">http://www.broadinstitute.org/mammals/haploreg/haploreg_v3.php</ext-link>) [<xref ref-type="bibr" rid="pgen.1005874.ref045">45</xref>].</p><p>A database of expression Single Nucleotide Polymorphism (eSNP) was created collecting results from multiple published sources, reported in Section 3 in <xref ref-type="supplementary-material" rid="pgen.1005874.s008">S1 Text</xref>. The eSNP results from each study were included in the database if they met criteria for statistical thresholds for association with gene transcript levels as described in the original references. To search for eQTLs among the associations found in the meta-analysis we queried this database for the replicated variants and their proxies (r<sup>2</sup>&gt;0.8).</p><p>Two different approaches were used to identify biological pathways influencing VEGF variability.</p><p>The GSEA-like statistical test implemented in MAGENTA program was used to test the over-representation of genes containing VEGF-associated variants in a given biological pathway. To do that, all data of meta-analysis results from Stage 1 were used and the gene-set annotations from the Kyoto Encyclopedia of Genes and Genomes (KEGG), PANTHER, INGENUITY, Gene Ontology, REACTOME and BIOCARTA databases were applied. Each gene in the genome was scored by the most significant association p-value among all the SNPs located within a region from 110 kb upstream to 40 kb downstream of each gene&#8217;s transcript boundaries. Confounding effects on gene association scores were identified and corrected for. This &#8220;normalized best gene score&#8221; was used to evaluate the gene enrichment against a null distribution of 10,000 gene sets of identical set size that are randomly sampled from the genome. The 95<sup>th</sup> percentile of all gene scores for the meta-analysis was used as the enrichment cutoff. Genes within the HLA-region were excluded from analysis due to difficulties in accounting for gene density and LD patterns and only gene sets with at least 10 genes were included in the analysis. Significance was determined when an individual pathway reached a false discovery rate (FDR)&lt;0.05.</p><p>The Ingenuity Pathway Analysis software (IPA) was used to explore the functional relationship between genes of interest, selected from candidate regions. For this purpose, a candidate region was defined as comprising all variants between the first and last variants in a chromosomal region that were associated at genome-wide significance with circulating VEGF levels, either in discovery phase (Stage 1) or the combined discovery and replication meta-analysis (Stage 3). The genes of interest were chosen including all within 60kb of each of the candidate regions. A total of 26 genes (listed in the <xref ref-type="supplementary-material" rid="pgen.1005874.s006">S4 Table</xref>) fit this description and served as &#8216;input&#8217; genes for the pathway analysis. Direct and indirect interactions, a reasonable confidence (experimentally observed, highly predicted, or moderately predicted) and a maximum size of 70 genes/proteins per network were used as parameters in the analysis.</p>