<p>
<italic>S</italic>. <italic>pombe</italic> strains used are listed in <xref ref-type="supplementary-material" rid="pgen.1005671.s006">S2 Table</xref>. Cells were grown in minimal media (EMM lacking uracil) or in rich media (YES) to an OD<sub>600</sub> of 1.0. <italic>S</italic>. <italic>pombe</italic> cells were seeded to an OD600 of 0.4, and ten-fold dilutions were plated on the appropriate media as indicated. For liquid growth, overnight cultures were diluted to OD600 of 0.25 and incubated for two hours after which rapamycin (AG Scientific Inc., R1018) at 200 ng/ml, or DMSO alone (Sigma, D2650) was added one hour prior to RNA isolation.</p><p>The <italic>trm1</italic>
<bold><italic>+</italic></bold> open reading frame was amplified from <italic>S</italic>. <italic>pombe</italic> genomic DNA using a forward primer containing sequence for 3X FLAG peptide and XhoI site and a reverse primer with XmaI site. The PCR products were digested with XhoI and XmaI and ligated into XhoI-XmaI digested pREP4X.</p><p>The <italic>maf1+</italic> gene and its 600 bp upstream region was PCR amplified from genomic DNA and cloned into the XhoI-PstI sites of pRep4X (removing the <italic>nmt1+</italic> promoter) resulting in plasmid CB235.</p><p>Total RNA was isolated using hot phenol. In short, 50 ml cultures grown from an OD600 of 0.1 to 0.5 were harvested, washed with water and resuspended in 300 &#956;l TES buffer (10 mM Tris Cl pH 7, 10 mM EDTA, 1% SDS). 300 &#956;l water-equilibrated phenol was added and incubated at 65&#176;C for 45 minutes, with vortex every 15 min. To the samples, 300 &#956;l chloroform was added and centrifuged. Supernatant was extracted twice with acid-phenol-chloroform and once with chloroform before precipitation with ethanol.</p><p>Total RNA was resolved in 6% NuPAGE TBE-Urea gel (Life Technologies) and transferred to positively charged nylon membranes. Probing and washes were done as described previously [<xref ref-type="bibr" rid="pgen.1005671.ref014">14</xref>].</p><p>Using affinity-purified anti-i6A from rabbit [<xref ref-type="bibr" rid="pgen.1005671.ref031">31</xref>] (a kind gift of Anita Hopper, OSU) at 1:50 and processed for chemiluminescence was as described [<xref ref-type="bibr" rid="pgen.1005671.ref028">28</xref>].</p><p>For tRNA isolation, 50 &#956;g of total RNA was separated on a 6% TBE-Urea polyacrylamide gel (21 cm X 18 cm X 0.15 cm) followed gel purification of RNA shorter than 5S rRNA. 300 ng purified tRNA was hydrolyzed in 10 mM bicarbonate buffer pH 9.7 at 90&#176;C for 5 min. The hydrolyzed RNA was dephosphorylated using calf intestinal alkaline phosphatase (NEB) followed by 5' phosphorylation by T4 polynucleotide kinase (NEB). Barcoded, pre-adenylated, 3&#8217; blocked Illumina adapters were ligated to the 3&#8217; end using T4 Rnl2(1&#8211;249)K227Q enzyme (NEB). After heat inactivation, all ligated samples were pooled in ethanol and precipitated. In parallel, two size marker RNA oligos of 19 and 35 nt were radiolabeled using T4 polynucleotide kinase and ligated to pre-adenylated adapter and pooled. Ligated tRNA samples and markers were resolved in a 15% TBE urea acrylamide gel followed by gel purification of tRNA samples between 19 nt and 35 nt guided by the marker lanes. 5&#8217; adapter was ligated to the gel purified samples using T4 RNA ligase I (Thermo). RNA with ligated 5&#8217; and 3&#8217; adapters were gel purified, subjected to reverse transcription (Superscript III, Life Technologies), amplified by PCR (10&#8211;12 cycles) and the band was gel purified and sequenced using Illumina HiSeq 2500.</p><p>tRNA read depths for all samples generally varied with tRNA gene copy number; from 278 for the lowest read from the single copy ArgCCG gene in one replicate to greater than one-million reads for highly abundant tRNAs. Sequence reads were mapped to a reference file comprising sequences of all 61 unique mature <italic>S</italic>. <italic>pombe</italic> tRNAs. Sequences that did not map to this file were then mapped to a file containing all 150 unique <italic>S</italic>. <italic>pombe</italic> precursor-tRNA genes. Read count tables were made using the mapping data and analyzed using DEseq software.</p><p>The fraction of reads that mismatched the reference gene sequence at each position were tabulated. The average values of the fraction of misincorporation among replicates was calculated and plotted (e.g., <xref ref-type="fig" rid="pgen.1005671.g003">Fig 3C</xref>).</p><p>
<italic>S</italic>. <italic>pombe</italic> cells were grown in 10 ml of the noted media to A600 of 0.7&#8211;1, washed with water and resuspended in 400 ul of 20% trichloroacetic acid (TCA). Glass beads (0.5 mm) were added and vortexed for 1 min. The beads were separated from the material, washed with 5% TCA and the wash was pooled with the material recovered. This was centrifuged at 6000 RCF for 10 minutes, the supernatant discarded and the pellet washed twice with 1 ml acetone. The pellet was air dried and dissolved in 400 ul of 1X SDS sample buffer containing fresh beta-mercaptoethanol. Aliquots were resolved on a 4&#8211;12% Bis-Tris gel (Life technologies) followed by transfer to PVDF membrane. The blot was blocked for 1 hour with 5% non-fat milk in PBS. Polyclonal anti-Trm1 antiserum raised against the C-terminal peptide, GPKSKPGKRTIAEVDSKS, in rabbit (Thermo Fisher Scientific, Waltham, MA; animal #PA9064, day 56 bleed) was used at 1:500 in blocking buffer with 0.1% Tween 20. Anti-Tubulin (Sigma, #T5168) was used at 1:4000. After 1 hr. the blot was washed 4 times with PBS-Tween. Appropriate secondary Abs (LI-COR) of different fluorescent emittances were used at 1:20,000 in 5% milk solution in PBS with 0.2% Tween-20 and 0.01% SDS for 1 hour followed by 4 washes in PBS-T. The washed blot was scanned using LI-COR Odyssey Clx system and the images processed and bands quantified using ImageStudioLite software.</p><p>The glyceraldehyde phosphate dehydrogenase (GPD) promoter followed by a 3X-HA tag was amplified from the pYM-16 plasmid (PCR-toolbox, EUROSCARF) and cloned into the SacI-NotI sites of pRS426 to generate the pRS426GPD vector. The <italic>S</italic>. <italic>cerevisiae TRM1</italic> gene starting from the ATG start codon to 855 bp downstream of the stop codon was amplified from genomic DNA and digested with NotI and XhoI present in the primers used. The fragment was inserted in the corresponding site of pRS426GPD. The empty vector (pRS426GPD) and the Trm1 clone were used to transform <italic>S</italic>. <italic>cerevisiae maf1&#916;</italic> strain, MB159-4D<italic>&#916;</italic> [<xref ref-type="bibr" rid="pgen.1005671.ref057">57</xref>] which carries <italic>SUP11</italic> sup-tRNATyrUUA, and plated on SC agar lacking uracil with 10 mg/l adenine.</p>