<p>
<italic>Malassezia</italic> was grown on mDixon media for DNA extraction, and <italic>M</italic>. <italic>globosa</italic> was grown on mDixon or minimal media for RNA extraction. Sequencing was done using Illumina HiSeq 2000. Please see <xref ref-type="supplementary-material" rid="pgen.1005614.s023">S9 Text</xref> for details.</p><p>Genomes were assembled using an in-house pipeline. Specifically, we used a conservative trimming approach (removing Q2 bases and all following bases and discarding paired-end reads with one or both ends shorter than 50 bp) as recommended by Illumina. After read-trimming, <italic>de novo</italic> assemblies (contiging and scaffolding) were constructed using SOAPdenovo [<xref ref-type="bibr" rid="pgen.1005614.ref051">51</xref>] (version 1.05, maximum insert length = 300bp). A range of assembly options were explored including testing various k-mers (41, 51, 61, 71, 81) and down-sampling of coverage on an exponential scale down to 50X to identify an optimal assembly in terms of contiguity statistics (N50). An example of this can be seen in <xref ref-type="supplementary-material" rid="pgen.1005614.s029">S6 Table</xref>. The resulting assembly was then re-scaffolded using the program Opera version 1.4 [<xref ref-type="bibr" rid="pgen.1005614.ref052">52</xref>] with default parameters and we attempted to close remaining gaps in the assembly <italic>in silico</italic> using FinIS [<xref ref-type="bibr" rid="pgen.1005614.ref053">53</xref>]. Details of assembly statistics can be found in <xref ref-type="supplementary-material" rid="pgen.1005614.s030">S7 Table</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Illumina paired-end reads were first trimmed using fastq_quality_trimmer (-t 20 -l 10) in FASTX (<ext-link ext-link-type="uri" ns0:href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</ext-link>). Reads that could not be paired were discarded. Then, reads were assembled into transcripts using Trinity [<xref ref-type="bibr" rid="pgen.1005614.ref054">54</xref>] (&#8212;jaccard_clip&#8212;SS_lib_type FR). The jaccard_clip option was turned on to avoid fusion of transcripts. Assembly was done individually for each sample and all assemblies were combined to generate a complete set of 727,354 transcripts. After removing redundant transcripts that were identical to another transcript, a total of 716,864 transcripts were left. Then, transcripts that were fully contained in another transcript were also removed to generate a set of 402,757 transcripts.</p><p>The <italic>M</italic>. <italic>globosa</italic> 7966 reference genome was originally annotated by combining <italic>in silico</italic> prediction and a limited number of EST sequences [<xref ref-type="bibr" rid="pgen.1005614.ref002">2</xref>]. Therefore, we re-annotated this genome to improve quality with the aid of RNA-seq datasets from two growth conditions and the application of newer <italic>ab initio</italic> gene prediction programs [<xref ref-type="bibr" rid="pgen.1005614.ref055">55</xref>,<xref ref-type="bibr" rid="pgen.1005614.ref056">56</xref>]. We used the MAKER pipeline [<xref ref-type="bibr" rid="pgen.1005614.ref057">57</xref>] to integrate <italic>ab initio</italic> gene prediction (SNAP [<xref ref-type="bibr" rid="pgen.1005614.ref055">55</xref>] and AUGUSTUS [<xref ref-type="bibr" rid="pgen.1005614.ref056">56</xref>]), transcript evidence (from our RNA-seq dataset), and protein evidence to predict genes in an iterative manner. For protein sequences, we downloaded several fungal genomes from Genbank, including <italic>Saccharomyces cerevisiae</italic> [<xref ref-type="bibr" rid="pgen.1005614.ref058">58</xref>], <italic>Ustilago maydis</italic> [<xref ref-type="bibr" rid="pgen.1005614.ref059">59</xref>], <italic>Candida albicans</italic> [<xref ref-type="bibr" rid="pgen.1005614.ref060">60</xref>], <italic>M</italic>. <italic>globosa</italic> [<xref ref-type="bibr" rid="pgen.1005614.ref002">2</xref>], and <italic>M</italic>. <italic>sympodialis</italic> [<xref ref-type="bibr" rid="pgen.1005614.ref014">14</xref>]. The protein sequences of <italic>M</italic>. <italic>globosa</italic> and <italic>M</italic>. <italic>sympodialis</italic> were not used in their own annotation process, respectively. Four sets of annotation were generated using the MAKER pipeline [<xref ref-type="bibr" rid="pgen.1005614.ref057">57</xref>] in an iterative manner to improve <italic>ab initio</italic> gene prediction. First, MAKER was run with protein evidence and <italic>ab initio</italic> predictors trained with core eukaryotic genes predicted by CEGMA [<xref ref-type="bibr" rid="pgen.1005614.ref020">20</xref>]. Then, MAKER was re-run twice additionally. For each time, <italic>ab initio</italic> predictors were retrained with gene models predicted by the previous run.</p><p>The first annotation was generated without transcripts. Compared to the reference annotation, this annotation retrieved longer genes and identified more exons and introns. The number of PFam domains identified in this annotation was slightly higher than that of the reference annotation, suggesting that it annotation captured more sequences with coding potential. The number of supported intron-exon junctions also increased, indicating that a large portion of the newly identified junctions were likely real (<xref ref-type="supplementary-material" rid="pgen.1005614.s025">S2 Table</xref>, 1<sup>st</sup> set). These results suggest that even without transcript evidence, our annotation captured more coding sequences and more complete genes. Upon addition of transcript evidence in the second set, the number of introns increased, while the number of genes decreased, likely due to the false merging of genes by transcripts spanning two genes (<xref ref-type="supplementary-material" rid="pgen.1005614.s025">S2 Table</xref>, 2<sup>nd</sup> set). Since the <italic>M</italic>. <italic>globosa</italic> genome is compact, with 4,223 genes (<xref ref-type="table" rid="pgen.1005614.t001">Table 1</xref>) in a 9.0 Mbp genome, false merging of genes during transcriptome assembly is likely. The number of PFam domains in the protein sequences also decreased. Therefore, we removed any transcript that overlapped with two or more gene models in order to generate the third annotation set (<xref ref-type="supplementary-material" rid="pgen.1005614.s025">S2 Table</xref>, 3<sup>rd</sup> set). The full set of transcripts was aligned to the <italic>M</italic>. <italic>globosa</italic> reference genome using BLAT with minimum identity of 99% [<xref ref-type="bibr" rid="pgen.1005614.ref061">61</xref>] and any transcript that overlapped with more than one gene model were removed to generate the reduced set of transcripts. This reduced the transcript set from 402,757 transcripts to 322,251 transcripts. We also noted that very few genes exhibited evidence for alternatives isoforms (10 with two isoforms and one with four isoforms). Then, gene models in the second set that do not overlap with any gene in the third set were added to generate the fourth annotation set (<xref ref-type="supplementary-material" rid="pgen.1005614.s025">S2 Table</xref>, 4<sup>th</sup> set). Adding these transcripts led to an increased number of supported junctions, but the number of unique PFam domains and total PFam domains did not increase further (<xref ref-type="supplementary-material" rid="pgen.1005614.s025">S2 Table</xref>), suggesting that <italic>ab initio</italic> prediction is sufficient to capture sequences with coding potential. For genomes other than <italic>M</italic>. <italic>globosa</italic> 7966, we used the same iterative approach for annotation without transcript evidence.</p><p>Species trees were constructed using the concatenated sequences of 164 core eukaryotic genes (CEGs) predicted by CEGMA [<xref ref-type="bibr" rid="pgen.1005614.ref020">20</xref>] that are present in all <italic>Malassezia</italic> genomes, <italic>U</italic>. <italic>maydis</italic> genome, and the <italic>S</italic>. <italic>cerevisiae</italic> genome, and the <italic>Malassezia</italic> sequences are at least 90% the length of their <italic>S</italic>. <italic>cerevisiae</italic> orthologs. Sequences were aligned using MUSCLE [<xref ref-type="bibr" rid="pgen.1005614.ref062">62</xref>] and the phylogeny was constructed using maximum likelihood (ML) approaches as implemented by RAxML [<xref ref-type="bibr" rid="pgen.1005614.ref063">63</xref>]. RAxML was run using &#8220;&#8211;f a &#8211;m PROTGAMMAJTT&#8221; with 400 bootstraps. (Ustilaginomycotina tree was built using the same approach.) To test the robustness of our <italic>Malassezia</italic> phylogeny beyond bootstrap values, we applied a Bayesian approach on the same concatenated sequences using MRBAYES [<xref ref-type="bibr" rid="pgen.1005614.ref064">64</xref>]. MRBAYES was run using &#8220;prset aamodelpr = mixed&#8221; and &#8220;mcmc nchains = 1 ngen = 300000&#8221;. We also merged individual ML gene trees for CEGs into a supertree [<xref ref-type="bibr" rid="pgen.1005614.ref065">65</xref>]. Individual gene trees were constructed using the same approach and they were merged into a supertree using Clann [<xref ref-type="bibr" rid="pgen.1005614.ref065">65</xref>]. Using this approach, more genes could be incorporated into the final tree, as missing a strain in a gene tree could be tolerated. Both approaches yielded the same phylogeny as the concatenated ML tree (Bayesian tree in <xref ref-type="supplementary-material" rid="pgen.1005614.s003">S3A Fig</xref>). We also generated a species tree derived from the mitochondrial genomes (Jack Kennell, personal communications, <xref ref-type="supplementary-material" rid="pgen.1005614.s003">S3B Fig</xref>). Despite differences observed on an intra-species level within the <italic>M</italic>. <italic>furfur</italic>, <italic>M</italic>. <italic>globosa</italic>, and <italic>M</italic>. <italic>sympodialis</italic> lineages, we found that the two trees correspond perfectly on an inter-species level.</p><p>For <italic>M</italic>. <italic>furfur</italic>-specific tree (<xref ref-type="supplementary-material" rid="pgen.1005614.s001">S1 Fig</xref>), MCL clusters (described later) with two genes in each of the hybrids (<italic>M</italic>. <italic>furfur</italic> 7710, <italic>M</italic>. <italic>furfur</italic> 1878, <italic>M</italic>. <italic>furfur</italic> 4172, and <italic>M</italic>. <italic>furfur</italic> 7019) and one gene in each of the haploids (<italic>M</italic>. <italic>furfur</italic> 7982 and <italic>M</italic>. <italic>furfur</italic> JPLK23) were used. Nucleotide sequences were used instead of amino acid sequences. The genes from hybrid strains were separated into two groups based on their similarity to the genes of the two haploid strains as measured by BLAST bitscore [<xref ref-type="bibr" rid="pgen.1005614.ref039">39</xref>]. Only clusters with less than 5% of total alignment as indels were used to minimize the effect of assembly and annotation errors. A total of 1,306 clusters were concatenated and aligned using MUSCLE [<xref ref-type="bibr" rid="pgen.1005614.ref062">62</xref>]. To generate a maximum likelihood tree, RAxML was used with &#8220;-f a -# 400 -m GTRGAMMA&#8221; [<xref ref-type="bibr" rid="pgen.1005614.ref063">63</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">PFam domains [<xref ref-type="bibr" rid="pgen.1005614.ref029">29</xref>] were identified in all <italic>Malassezia</italic> strains and other fungal protein sequences using hmmscan (HMMER 3.1b1) with trusted cutoffs (<ext-link ext-link-type="uri" ns0:href="http://hmmer.org">http://hmmer.org</ext-link>). <italic>Malassezia</italic>-specific PFam domains are defined as present in at least seven <italic>Malassezia</italic> species and in at most two other fungi. This approach is limited to the study of protein families having at least one PFam domain. MCL clustering [<xref ref-type="bibr" rid="pgen.1005614.ref066">66</xref>] was used to cluster <italic>Malassezia</italic> and other fungal genes based on their pairwise sequence similarity to construct gene families. This approach does not rely on PFam and can include all genes regardless whether they have PFam domains or not. Yet, unrelated genes with no orthology can be clustered together if they both have a good match to a third gene. BLAST [<xref ref-type="bibr" rid="pgen.1005614.ref039">39</xref>] was used to align <italic>Malassezia</italic> and other fungal protein sequences in an all-against-all fashion with an e-value cutoff of 10<sup>&#8722;5</sup>. MCL was used to cluster protein sequences based on their <italic>E</italic>-values with&#8211;I 2.0. Sequences shorter than 90% of the median length in the cluster were excluded. <italic>Malassezia</italic>-specific clusters were defined as present in at least seven <italic>Malassezia</italic> species and not present in other fungi). To infer gain and loss in PFam families, PFam family phylogenies were constructed using MUSCLE and RAxML as described earlier. Then, species phylogeny and gene family phylogenies were reconciled using NOTUNG [<xref ref-type="bibr" rid="pgen.1005614.ref067">67</xref>] to infer gain and loss events that took place along the <italic>Malassezia</italic> phylogeny.</p><p>For selection tests, we choose singleton PFam families in <italic>Malassezia</italic> species. Protein sequences were first aligned using MUSCLE (described earlier) and nucleotides were then substituted back in the sequences. PAML [<xref ref-type="bibr" rid="pgen.1005614.ref068">68</xref>] was used for selection tests (CODEML, M7 M8 mode), and genes with bonferroni-corrected <italic>p</italic>-value &lt; 0.05 were tabulated (&#967;<sup>2</sup> test, two degrees of freedom).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For <italic>Malassezia</italic> growth in 2X YNB media, cultures were incubated in 2X YNB at 31&#176;C for 24, 48, 72, and 144 hours. Four strains, <italic>M</italic>. <italic>furfur</italic> 7982, <italic>M</italic>. <italic>sympodialis</italic> 42132, <italic>M</italic>. <italic>pachydermatis</italic> 1879 and 7550 (also named by CBS, <ext-link ext-link-type="uri" ns0:href="http://www.cbs.knaw.nl/">http://www.cbs.knaw.nl/</ext-link>), were used. For each strain, two cultures, one with and one without Tween 40 at 1%, were included in this experiment.</p><p>Two separate methods were used to extract lipids from the Bactopeptone media. Please refer to <xref ref-type="supplementary-material" rid="pgen.1005614.s023">S9 Text</xref> for details.</p><p>Briefly, Sabouraud broth was melted with 3% Sea Plaque GTG (low-melt) agarose and equilibrated to 45&#176;C. <italic>Malassezia</italic> cells were counted and diluted to 1x10<sup>5</sup> cells/ml in Sabouraud broth with chloramphenicol, and equilibrated to 31&#176;C. Cell suspension (30ml) and melted agar broth (30ml) were quickly mixed and poured into 150 mm dishes. Once solidified, 18 holes were made with a 2 mm punch biopsy. To each hole, 5 &#956;l of test compound was added and the plates were incubated for 17 days at 34&#176;C. Test compounds are listed in <xref ref-type="supplementary-material" rid="pgen.1005614.s028">S5</xref> and <xref ref-type="supplementary-material" rid="pgen.1005614.s029">S6</xref> Tables. Please refer to [<xref ref-type="bibr" rid="pgen.1005614.ref069">69</xref>,<xref ref-type="bibr" rid="pgen.1005614.ref070">70</xref>] for artificial sebum.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We examined the two mating type loci (<italic>P/R</italic> and <italic>HD</italic>) for all <italic>Malassezia</italic> species. We used the gene sequences from the reference strains for <italic>M</italic>. <italic>sympodialis</italic> and <italic>M</italic>. <italic>globosa</italic>, as queries to BLAST [<xref ref-type="bibr" rid="pgen.1005614.ref039">39</xref>] against the genome assemblies of all <italic>Malassezia</italic> species to identify the locations of the <italic>MAT</italic> loci in these genomes. The alignments of <italic>MAT</italic> loci between <italic>Malassezia</italic> species were generated through WebACT (<ext-link ext-link-type="uri" ns0:href="http://www.webact.org/WebACT/generate">http://www.webact.org/WebACT/generate</ext-link>) using blastn with an <italic>E</italic>-value cutoff of 0.0001.</p><p>
<italic>M</italic>. <italic>sympodialis</italic> 42132 protein sequences were compared to UniProtKB v2015_02 [<xref ref-type="bibr" rid="pgen.1005614.ref071">71</xref>] and the NCBI Nucleotide Collection database using NCBI Blast [<xref ref-type="bibr" rid="pgen.1005614.ref072">72</xref>]. Proteins with the most significant BLAST hit against bacteria in both databases were analyzed further. Bacterial and fungal homologs of each HGT candidate were retrieved from GenBank [<xref ref-type="bibr" rid="pgen.1005614.ref073">73</xref>]. Multiple sequence alignments were generated using MUSCLE v3.8.31 [<xref ref-type="bibr" rid="pgen.1005614.ref062">62</xref>]. Poorly aligned or divergent regions in the alignments were identified and excluded from further analysis using Gblocks v0.91 (with options &#8211;u = y &#8211;t = p) [<xref ref-type="bibr" rid="pgen.1005614.ref074">74</xref>]. Phylogenetic trees and 1000 bootstraps for each alignment were generated using PhyML v20120412 (with options &#8211;m JTT &#8211;d aa) [<xref ref-type="bibr" rid="pgen.1005614.ref075">75</xref>]. Phylogenetic trees were manually inspected and candidate horizontally transferred genes were chosen, if they were closer relatives to their bacterial orthologs than their fungal orthologs.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used PathoScope 2.0 [<xref ref-type="bibr" rid="pgen.1005614.ref076">76</xref>] to estimate the abundances of the 14 <italic>Malassezia</italic> species in different shotgun metagenomics datasets (strain 7982 used for <italic>M</italic>. <italic>furfur</italic>, strain 7877 for <italic>M</italic>. <italic>restricta</italic>, strain 7966 for <italic>M</italic>. <italic>globosa</italic>, and strain 42132 for <italic>M</italic>. <italic>sympodialis</italic>). Before mapping to the reference genomes, the reads were filtered against a set of non-<italic>Malassezia</italic> fungal genomes (all fungal genomes from <ext-link ext-link-type="ftp" ns0:href="ftp://ftp.ncbi.nlm.nih.gov/genomes/Fungi/">ftp://ftp.ncbi.nlm.nih.gov/genomes/Fungi/</ext-link> plus <italic>Ustilago maydis</italic> [<xref ref-type="bibr" rid="pgen.1005614.ref059">59</xref>]) using PathoScope 2.0's MAP module. To further account for potential false positives from PathoScope analysis, <italic>Malassezia</italic> genomes were divided into 1 kb bins and a species was considered present only if &gt;10% of the bins were covered by at least one read. Abundances of <italic>Malassezia</italic> species were renormalized after removing false positives using this filter. The performance of PathoScope 2.0 was benchmarked by mapping the original reads of each <italic>Malassezia</italic> strain to the genomes of all <italic>Malassezia</italic> species using the protocol described here.</p><p>Accession numbers for the datasets studied in this paper are summarized in <xref ref-type="supplementary-material" rid="pgen.1005614.s023">S9 Text</xref>.</p><p>To identify genes with variable copy number in <italic>Malassezia</italic>, we focused on <italic>M</italic>. <italic>restricta</italic> (using <italic>M</italic>. <italic>restricta</italic> 7877 genome), the most abundant species in skin samples [<xref ref-type="bibr" rid="pgen.1005614.ref012">12</xref>]. We selected six samples with highest <italic>M</italic>. <italic>restricta</italic> genome coverage (MET0202, MET0207, MET0259, MET0270, MET0276, MET0278) [<xref ref-type="bibr" rid="pgen.1005614.ref012">12</xref>]. Read counts for each gene were obtained from PathoScope [<xref ref-type="bibr" rid="pgen.1005614.ref076">76</xref>] and normalized across all genes in each sample. Normalized read counts were used to compute the coefficient of variation (= sample standard deviation divided by sample mean). The top 10 genes with highest coefficient of variation are shown in <xref ref-type="fig" rid="pgen.1005614.g002">Fig 2B</xref> and the top 15 are listed in <xref ref-type="supplementary-material" rid="pgen.1005614.s026">S3 Table</xref>.</p>