<p>Heterozygous C57BL/6 staggerer (<italic>ROR&#945;<sup>+/sg</sup></italic>) were obtained from the Jackson Laboratories (Bar Harbor, ME). <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> and <italic>ROR&#945;<sup>sg/sg</sup>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> double knockout (DKO) mice were described previously <xref ref-type="bibr" rid="pgen.1004331-Takeda2">[10]</xref>, <xref ref-type="bibr" rid="pgen.1004331-Kurebayashi1">[36]</xref>. Liver-specific ROR&#947; knockout mice, referred to as <italic>ROR&#947;<sup>fx/fx</sup>Alb-Cre<sup>+</sup></italic>, were generated by crossing B6(Cg)-<italic>Rorc<sup>tm3Litt</sup></italic>/J (<italic>ROR&#947;<sup>fx/fx</sup></italic>) with B6.Cg-Tg(<italic>Alb-cre</italic>)21Mgn/J transgenic mice (Jackson Laboratories). Mice were supplied <italic>ad libitum</italic> with NIH-A31 formula (normal diet, ND) and water, and maintained at 23&#176;C on a constant 12 h light&#8758;12 h dark cycle. Two month-old male mice were fed with a high fat diet (40% kcal fat) (HFD: D12079B Research Diets Inc., New Brunswick, NJ) for 6 weeks. Littermate wild type (WT) mice were used as controls. All animal protocols followed the guidelines outlined by the NIH Guide for the Care and Use of Laboratory Animals and were approved by the Institutional Animal Care and Use Committee at the NIEHS.</p><p>After 16 h fasting, WT and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> mice (n&#8202;=&#8202;8&#8211;10) fed a ND or HFD for 6 weeks were injected intraperitoneally with glucose (2 g/kg), insulin (0.75 U/kg) (Eli Lilly, Indianapolis, IN) or sodium pyruvate (2 g/kg) (Sigma-Aldrich) at ZT4 or ZT18. The blood glucose was measured every 20 min for up to 140 min with glucose test strips (Nova Biomedical, Waltham, MA). These tests were performed in the same way using <italic>ROR&#947;<sup>fx/fx</sup>Alb-Cre<sup>+</sup></italic> and <italic>ROR&#947;<sup>fx/fx</sup>Alb-Cre<sup>&#8722;</sup></italic> mice (n&#8202;=&#8202;11) fed a HFD. ITT and PTT were also performed under red light at CT4 after WT(HFD) and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic>(HFD) mice (n&#8202;=&#8202;12) were kept for 1 day under constant darkness. Total AUC (Area under the curve) was calculated by the trapezoid rule. Two-way ANOVA was performed using GraphPad PRISM software.</p><p>To evaluate insulin signaling, liver, BAT, WAT, and skeletal muscle were isolated from fasting WT(HFD) and mice <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic>(HFD) mice 30 min after injection with either 0.75 U/kg insulin or PBS. Protein from these tissues was extracted with lysis buffer (25 mM Tris-HCl pH 7.6, 150 mM NaCl, 1% Nonidet P-40, 1% sodium deoxycholate, 0.1% SDS). In a separate experiment, primary hepatocytes isolated from WT and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> mice were treated with 20 nM insulin in serum-free 199 medium (Sigma-Aldrich) for 10 min. Phosphorylated Akt (Ser473) and whole Akt proteins were detected by Western blot analysis with antibodies 7408 and 7102 from Cell Signaling Technology. Pklr and Gapdh were detected in liver lysates from WT and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> mice (n&#8202;=&#8202;3) at ZT4 and ZT16 by Western blot analysis with anti-Pklr (22456-1-AP, Proteintech Group Inc., Chicago, IL, USA) and anti-Gapdh (Cell Signaling Technology) antibodies.</p><p>WT and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> mice (n&#8202;=&#8202;5) fed a HFD for 6 weeks underwent surgery under anesthesia to attach catheters to the jugular vein and carotid artery. Mice were left at least 2 days to recover. After a 3.5 h fasting, the basal rates of glucose turnover were measured by continuous infusion of HPLC-purified D-[3-<sup>3</sup>H] glucose (0.05 &#181;Ci/min) (Perkin Elmer, Boston, MA) for 90 minutes following a bolus of 1 &#181;Ci. Blood samples (about 40 &#181;l) were taken from the carotid artery catheter at 75 and 85 min after the infusion to determine the plasma [3-<sup>3</sup>H] glucose concentration. Subsequently the hyperinsulinemic euglycemic clamp test was performed for 120 min in conscious, restrained mice. Human insulin (HumulinR, Eli Lilly) was infused at a constant rate (30 mU/kg/min) through the end of the experiment following a bolus of 90 mU/kg/min for 3 min. Glucose was measured every 10 min in blood from tail vein with glucose test strips. The glucose concentration was maintained at 110&#8211;130 mg/dl by a variable rate of 20% glucose infusion under a continuous infusion of [3-<sup>3</sup>H] glucose (0.1 &#181;Ci/min). Blood samples (about 40 &#181;l) were taken from the carotid artery catheter every 10 min during the last 40 min. [<sup>3</sup>H]-glucose was used to trace hepatic glucose production and glucose turnover. The experiment was performed during daytime at ZT2&#8211;9.</p><p>For the determination of the plasma <sup>3</sup>H-glucose concentration, plasma samples were deproteinized with 0.3 N Ba(OH)<sub>2</sub> and ZnSO<sub>4</sub> and dried to remove <sup>3</sup>H<sub>2</sub>O before the radioactivity was measured in a liquid scintillation counter. Basal hepatic glucose production (Basal HGP) was calculated as the ratio of the preclamp [<sup>3</sup>H]-glucose infusion rate (GIR) (dpm/min) to the specific activity of plasma glucose. Clamp whole-body glucose disappearance (Rd) was calculated as the ratio of the clamp [3-<sup>3</sup>H] GIR (dpm/min) to the specific activity of plasma glucose. Clamp glucose production (Clamp HGP) was determined by subtracting the average GIR in the last 40 min from the Rd.</p><p>Recombinant adenoviruses were generated using the AdEasy adenoviral system (Agilent Technologies, Palo Alto, CA). Full-length <italic>ROR&#947;1</italic> cDNA was inserted to pShuttle-IRES-hrGFP-1 vector, and co-transformed with pAdEasy-1 in BJ5183-AD-1 bacteria by electroporation. The recombinant adenovirus plasmid was then transfected in AD-293 cells. The amplified adenoviruses were purified and concentrated by cesium chloride density gradient centrifugation. The empty control and ROR&#947; expressing adenoviruses were injected into the retro-orbital sinus of <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic>(HFD) mice (n&#8202;=&#8202;6&#8211;7). Pyruvate tolerance test was performed 4 days later and after an additional four days, liver was collected at ZT8 to analyze glycogen accumulation and gene expression.</p><p>Hepatocytes from 2 month-old WT and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> mouse were isolated with a Hepatocyte Isolation System (Worthington Biochemical Corporation, New Jersey, USA) according to the manufacturer's instructions. Primary hepatocytes were cultured in collagen-coated dishes with Medium 199 supplemented with 100 nM dexamethasone, 1 nM insulin, 10 nM triiodothyronine, 5% fetal bovine serum, and penicillin/streptomycin. After 8&#8211;12 h, cells were infected with empty lentivirus pLVX-mCherry-N1 or ROR&#947;1-expressing lentivirus. 24 h later cells were washed twice in PBS and then incubated in serum-free medium 199 in the presence or absence of 100 nM insulin or 100 nM glucagon (Sigma-Aldrich) for 6 h before RNA was isolated. Glucose production was measured with a glucose production buffer (glucose/phenol red-free DMEM (Sigma-Aldrich), 1 mM lactose, 2 mM sodium pyruvate) in <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> hepatocyte infected with lentivirus for each empty and ROR&#947; expression (n&#8202;=&#8202;3). Glucose in the medium was measured with a Glucose assay kit (Sigma-Aldrich).</p><p>Serum and liver samples were collected from WT and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> mice on a HFD (n&#8805;5) every 4 h over a period of 24 h. Serum insulin was measured by a sandwich ELISA with a Rat/Mouse Insulin ELISA kit (EZRMI-13K, Millipore). Glucose stimulated insulin secretion (GSIS) was measured at ZT4 in WT and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> mice on a HFD (n&#8202;=&#8202;5&#8211;6) or ND (n&#8202;=&#8202;2&#8211;3). Serum was collected at 2.5, 5, 15, and 30 min after intraperitoneal injection of glucose (2 g/kg). Pancreatic insulin was determined by rapidly removing the pancreas from WT and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> mice (n&#8202;=&#8202;10&#8211;14) on a HFD. Pancreas was then homogenized and extracted overnight with acid-ethanol at &#8722;20&#176;C. Insulin in the extracts was measured with the insulin ELISA kit. Insulin was normalized by total pancreatic protein. Glycogen extracted from liver with 30% KOH at 100&#176;C for 2 h followed by precipitation by ethanol, was measured with a Glycogen Assay Kit (BioVison Inc., Mountain View, CA).</p><p>To analyze metabolic parameters including oxygen consumption, CO<sub>2</sub> production, respiratory exchange ratio, heat production, and food/water consumptions were measured in WT and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> mice (n&#8202;=&#8202;8) with a LabMaster system (TSE systems Inc., Chesterfield, MO) during 4 successive days.</p><p>The ChIP assay was performed using a ChIP assay kit from Millipore (Billerica, MA) according to the manufacturer's protocol with minor modifications as described previously <xref ref-type="bibr" rid="pgen.1004331-Takeda2">[10]</xref>. Briefly, livers collected from WT, <italic>ROR&#945;<sup>sg/sg</sup></italic>, and <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> mice at ZT10 and ZT22 were homogenized with a polytron PT 3000 (Brinkmann Instruments) and crosslinked by 1% formaldehyde for 10 min at room temperature. After a wash in PBS, an aliquot of the crosslinked chromatin was sonicated and incubated overnight with an anti-ROR&#945; or anti-ROR&#947; antibody <xref ref-type="bibr" rid="pgen.1004331-Takeda2">[10]</xref> generated against amino acids 129&#8211;231 and 121&#8211;213 in mouse ROR&#947;1 and ROR&#945;4, respectively. After incubation with protein G agarose beads for 2 h, DNA-protein complexes were eluted. The crosslinks were reversed by overnight incubation at 65&#176;C in the presence of 25 mM NaCl, digested with RNase A and proteinase K, and then the ChIPed-DNA was purified. The amount of ChIPed-DNA relative to each input DNA was determined by QPCR. All QPCR reactions were carried out in triplicate. Sequences of primers for ChIP-QPCR are listed in <xref ref-type="supplementary-material" rid="pgen.1004331.s011">Table S4</xref>.</p><p>ChIPed-DNA and input DNA as a control were prepared using ROR&#947;- and ROR&#945;-specific antibodies as described previously <xref ref-type="bibr" rid="pgen.1004331-Takeda2">[10]</xref>. ChIP-Seq analysis was performed by the NIH Intramural Sequencing Center and data were analyzed as reported previously <xref ref-type="bibr" rid="pgen.1004331-Narlikar1">[37]</xref>. The sequencing reads were obtained from base-calling of Illumina Genome Analyzer. The wiggle-formatted alignment results were visualized on UCSC Genome Browser using mouse mm9 reference genome. SISSRs (Site Identification from Short Sequence Reads) were used for identification of significant ROR&#947; and ROR&#945; binding sites (P&lt;0.001) that have enriched reads in each ChIPed-DNA versus input control across the whole genome <xref ref-type="bibr" rid="pgen.1004331-Jothi1">[38]</xref>. The distance from each ROR peak to the nearest transcriptional start sites was determined using custom scripts. <italic>De novo</italic> consensus motif search within ROR binding sites was performed using MEME. ChIP-Seq data was compared with gene expression data using Kolmogorov-Smirnov (KS) plot. Gene ontology analysis was performed using the NIH Database for Annotation, Visualization, and Integrated Discovery (DAVID) online web-server, and based on PANTHER Biological process definitions.</p><p>To quantify gene expression during circadian time, liver tissues were collected from WT, <italic>ROR&#947;<sup>&#8722;/&#8722;</sup></italic>, and <italic>ROR&#945;<sup>sg/sg</sup></italic> mice every 4 h over a period of 24 h, processed overnight in RNA<italic>later</italic> solution (Ambion, Austin, TX) at 4&#176;C, and then stored at &#8722;80&#176;C until use. Tissues were then homogenized with a Polytron PT-3000 (Brinkmann Instruments, Westbury, NY). Liver tissues were also collected from <italic>ROR&#945;<sup>sg/sg</sup>ROR&#947;<sup>&#8722;/&#8722;</sup></italic> DKO mice and littermate control WT mice, and <italic>ROR&#947;<sup>fx/fx</sup>Alb-Cre<sup>+</sup></italic> and <italic>ROR&#947;<sup>fx/fx</sup>Alb-Cre<sup>&#8722;</sup></italic> mice at zeitgeber time (ZT) 8 and ZT20. RNA was then extracted using a QIAshredder column and RNeasy Mini kit (Qiagen, Valencia, CA) according to the manufacturer's instructions. The RNA was reverse-transcribed using a High-Capacity cDNA Archive Kit (Applied Biosystems). QPCR analysis was performed using SYBR Green I (Applied Biosystems, Foster City, CA). The reactions were carried out in triplicate using 20 ng of cDNA and the following conditions: 10 min at 95&#176;C, followed by 40 cycles of 15 sec at 95&#176;C and 60 sec at 60&#176;C. The results were normalized by the amount of <italic>Gapdh</italic> mRNA. Primer sequences are listed in <xref ref-type="supplementary-material" rid="pgen.1004331.s011">Table S4</xref>.</p><p>The promoter or intron region of mouse <italic>G6Pase</italic> (promoter; &#8722;500/+58), <italic>Ppar&#948;</italic> (intron 2; +46417/+46987), <italic>Gck</italic> (intron 1; +29709/+30121), <italic>Gckr</italic> (promoter; &#8722;685/+42), <italic>Glut2</italic> (intron 2; +16294/+16805), <italic>Gys2</italic> (promoter; &#8722;256/+345), and <italic>Dlat</italic> (promoter; &#8722;1151/+22) genes was amplified using mouse genomic DNA (Promega, Madison, WI) and cloned into either the promoter-less reporter plasmid pGL4.10 or pGL4.27 containing a minimal promoter (Promega, Madison, WI). Point mutations in ROREs and PPREs were generated using a Quickchange Site-Directed Mutagenesis Kit (Stratagene, La Jolla, CA). Human hepatoma Huh-7 cells were co-transfected with the indicated pGL4 reporter plasmid, pCMV-&#946;-Gal, and p3xFlag-CMV10-ROR&#947;, &#8211;ROR&#945;, -Rev-Erb&#945;, or -PPAR&#945; expression plasmids using lipofectamine 2000 (Invitrogen, Carlsbad, CA). After 24 h incubation, the luciferase and &#946;-galactosidase activities were measured with a Luciferase Assay Substrate kit (Promega) and Luminescent &#946;-galactosidase Detection Kit II (Clontech). All transfections were performed in triplicate and repeated at least twice. In certain experiments cells were treated for 24 h with a ROR&#947;-selective antagonist &#8220;A&#8221;, (R)-N-(1-((4-methoxy-phenyl)sulfonyl)-4-methyl-1,2,3,4-tetrahydroquinolin-7-yl)-2,4,6-trimethylbenzene-sulfonamide provided by Dr. Veronique Birault (GlaxoSmithKline) <xref ref-type="bibr" rid="pgen.1004331-Takeda2">[10]</xref> or with the selective PPAR&#945; antagonist, Wy14,643 (10 &#181;M; Sigma-Aldrich) as indicated.</p>