<p xmlns:ns0="http://www.w3.org/1999/xlink">The 230 target organisms were selected based on i) phylogenetic diversity, ii) relevance to D.O.E mission areas in bioenergy and the environment, iii) the presence of interesting and potentially interpretable RM systems. DNA samples were obtained from commercial sources (American Type Culture Collection, ATCC or DSMZ), or from contributions to the JGI community-sequencing program (<ext-link ext-link-type="uri" ns0:href="http://jgi.doe.gov/collaborate-with-jgi/community-science-program/">http://jgi.doe.gov/collaborate-with-jgi/community-science-program/</ext-link>). A complete list of bacterial strain information and DNA sources is provided in <bold><xref ref-type="supplementary-material" rid="pgen.1005854.s012">S1 Table</xref></bold>.</p><p>All sequenced organisms have publicly available reference genome sequences, and gene annotation files deposited in NCBI and IMG [<xref ref-type="bibr" rid="pgen.1005854.ref051">51</xref>]. Accession numbers and summary statistics of reference genomes used in the analyses are provided in <bold><xref ref-type="supplementary-material" rid="pgen.1005854.s012">S1 Table</xref></bold>. The majority of reference sequences were complete (assembled into a single circular molecule), with more than 90% genomes containing less than 10 scaffolds. The inclusion of data from draft genome sequences increases our overall yield of MTase gene annotations, but limits the ability to comprehensively annotate the methylome in all cases.</p><p>SMRT sequencing was performed using a library construction protocol described previously [<xref ref-type="bibr" rid="pgen.1005854.ref052">52</xref>]. Libraries were sequenced on the Pacific Biosciences RS instrument using either C2, C3 or C4 chemistries. The average SMRT sequence coverage per genome was 130x (ranging from 31x to over 500x), with an average sub-read length of 1.8kb. Sequencing chemistries and sequencing yields for each DNA sample are summarized in <bold><xref ref-type="supplementary-material" rid="pgen.1005854.s012">S1 Table</xref></bold>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">DNA modification detection and motif analysis were performed using the PacBio SMRT analysis platform (protocol version = 2.2.0 method = RS Modification and Motif Analysis.1, <ext-link ext-link-type="uri" ns0:href="http://www.pacb.com/devnet/code.html">http://www.pacb.com/devnet/code.html</ext-link>). Briefly, raw reads were filtered using SFilter, to remove short reads and reads derived from sequencing adapters. Filtered reads were aligned to the reference genome using BLASR(v1) [<xref ref-type="bibr" rid="pgen.1005854.ref053">53</xref>]. Modified sites were then identified through kinetic analysis of the aligned DNA sequence data [<xref ref-type="bibr" rid="pgen.1005854.ref032">32</xref>]. Modified sites were then grouped into motifs using MotifFinder (v1)2. These motifs represent the recognition sequences of MTase genes active in the genome [<xref ref-type="bibr" rid="pgen.1005854.ref054">54</xref>]. All kinetic data files have been deposited in GEO under accession numbers GSE69872, available for review using the following link: <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=ufapcaooxtcdbal&amp;acc=GSE69872">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=ufapcaooxtcdbal&amp;acc=GSE69872</ext-link> The full list of identified modified motifs are in <bold><xref ref-type="supplementary-material" rid="pgen.1005854.s013">S2 Table</xref></bold>.</p><p>Restriction-Modification (RM) genes were assigned using the SEQWARE computer resource (Clark et al. 2012; Murray et al. 2012). It comprises a large suite of program modules with specialized databases containing compilations of protein sequences of bona-fide M system components as well as non-RM system components to weed out false positives. On a daily basis, newly published sequences are collected from GenBank and downloaded for analysis by SEQWARE and incorporation into the SEQWARE databases. Many routines that run in parallel scan the new data, queue different inspection steps, and depending on the preliminary findings, pipe data into further, more detailed analysis loops.</p><p>First, conserved elements of new RM systems are identified by sequence matches to known RM system genes. Most often these are MTases. Characteristic motifs [<xref ref-type="bibr" rid="pgen.1005854.ref055">55</xref>, <xref ref-type="bibr" rid="pgen.1005854.ref056">56</xref>] of the newly detected items are located, and functional domains are mapped. From these, the Type and subtype of the inspected system are inferred, as well as the identity of potentially missing components&#8211;most often restriction enzyme genes. Genes for these missing components are then picked by a contextual analysis, where attribution is guided by marginal similarities in Type/subtype characteristic component order, while skipping genes that show better matches to non-RM system genes. Homologs that harbor non-RM functions (e.g. RNA MTases) generate many false hits in this first round of analysis, but are then filtered out by further matching to a library of known false positives. Occasionally, fusions of RM system genes to genes of unrelated enzymes are observed. To avoid false hits produced solely by the fused parts, the non-RM system domains are masked in the search library.</p><p>Newly detected systems are prepared for expert review. Items are annotated, and background supporting materials are prepared. These include hit lists, summary tables, schematics, plots and selectable alternatives for the resolution of undecided issues (e.g. handling of frame shifts). Following the curator&#8217;s decisions, results are recorded, and the internal databases are rebuilt for the next round of discovery. The program suite part of SEQWARE changes frequently as new modules are incorporated to accommodate new kinds of relevant data (shotgun, methylome), and as our understanding of RM systems expands. SEQWARE has been a prime supporting engine of restriction enzyme research for the last 20 years and is responsible for the bulk of the expansion of REBASE [<xref ref-type="bibr" rid="pgen.1005854.ref005">5</xref>]. RM system gene annotations are summarized in <bold><xref ref-type="supplementary-material" rid="pgen.1005854.s013">S2 Table</xref></bold>.</p><p>MTases without detectable RM genes in the flanking genome sequence were cautiously annotated as &#8216;candidate orphan&#8217; MTases. Since restriction enzymes can be hard to identify, we cannot firmly conclude that a cognate restriction enzyme gene does not exist, and therefore some of these candidate orphan MTases may in reality be part of RM systems. However, our observation that many candidate orphan MTases exhibit incomplete modification of their genomes (<bold><xref ref-type="fig" rid="pgen.1005854.g004">Fig 4</xref></bold>) is one line of evidence to suggest that the majority of these annotations are correct.</p><p>In general, the sequence specificity of each putative MTase gene was predicted based on significant similarity to MTase genes of known specificity. Whenever such a gene was present and a motif of the same specificity was found, then the MTase gene was assumed to be responsible, unless more than one candidate MTase gene of the same specificity was present in which case no match was called. In the case of Type I, Type IIG and Type III genes in many cases only a single candidate gene was present for the particular kind of motif observed. Thus, for Type I genes, the recognition sequences are characteristically bipartite and usually asymmetric. For both Type IIG and Type III MTases methylation is only present on one strand. Again if only one gene was present then it could be matched unambiguously to the motif. In some cases, for Type I systems one half of the recognition sequence would match half of a known specificity in another organism. Often, this would then permit matching of the appropriate S subunit to the motif. In all cases where there were no clear and unambiguous matches, the motif was marked as unmatched (see <bold><xref ref-type="supplementary-material" rid="pgen.1005854.s014">S3 Table</xref></bold> and <bold><xref ref-type="supplementary-material" rid="pgen.1005854.s015">S4 Table</xref></bold>). In some cases reasonable guesses could be made and these are indicated in <bold><xref ref-type="supplementary-material" rid="pgen.1005854.s014">S3 Table</xref></bold> and <bold><xref ref-type="supplementary-material" rid="pgen.1005854.s015">S4 Table</xref></bold> by putting the motifs and the genes likely to match them in parentheses.</p><p>Nine enzymes have been characterized as restriction enzymes. Three typical Type IIG single polypeptide REase-MTase proteins have been cloned and their recognition motif determined from their endonuclease cleavage patterns: SdeAI and PliMI from the MmeI family, and RpaI, a representative of the TaqII family [<xref ref-type="bibr" rid="pgen.1005854.ref033">33</xref>]. Similarly the endonuclease genes for the MjaI, MjaII, MjaIII, MjaIV and MjaV systems from <italic>Methanocaldococcus jannachii</italic> DSM 2661 as well as Csp12AI from <italic>Clostridium sp</italic>. 12(A), have been cloned, expressed and characterized through their endonuclease activity. The modified base and recognition motif for three MTases by cloning the MTase gene into the non-modifying host ER2796 followed by SMRT sequencing and analysis using methods described in Murray et al. 2012 (8). In this way we also characterized the Type III MTase M.Nme18I which recognizes AC<sup><underline>m6</underline></sup><underline>A</underline>CC [<xref ref-type="bibr" rid="pgen.1005854.ref027">27</xref>], and M1.Csp12AI and M2.Csp12AI from <italic>Clostridium sp</italic>. 12(A).</p><p>Amino acid sequences of all annotated Type II MTases were obtained from IMG and used as queries in BLASTP (blastall v2.2.26) searches of protein sequence databases of 35,184 bacterial and archaeal genomes in IMG. Database hits with a similarity score of 35 or more (where similarity score = 100*(bitscore of hit to database / bitscore of hit to self)) were considered potential orthologs of the MTase. Taxonomy information for all database genomes was also obtained from IMG, and used to determine the fraction of organisms across each taxonomic category with a potential MTase ortholog. MTases with orthologs in &gt;50% of species from a taxonomic group were considered &#8216;conserved&#8217;.</p><p>Amino acid sequences of all annotated Type II MTases were obtained from IMG, and split into two groups according to base methylation type (m6A or m4C and m5C). For each group, all versus all alignments were performed with usearch, v8.0.1616_i86linux32 [<xref ref-type="bibr" rid="pgen.1005854.ref057">57</xref>], using the search_global command (with parameters&#8211;fulldp&#8211;id 0 &#8211;uc). Initial clusters of related MTases were identified using usearch&#8211;cluster_agg (with parameters -id 0.35 -linkage min&#8211;fulldp. Using custom perl scripts, MTases were annotated with taxonomic classification of the host organism, presence or absence of cognate REase, and motif specificity. Annotated MTase clusters were then manually inspected to identify individual sub-clusters of orphan MTases with identical or closely related specificities from taxonomically related organisms. The resulting sub-clusters represent putative orphan MTase families.</p><p>The ipdR (inter-pulse duration ratio) is the primary metric in DNA modification detection. It corresponds to the time delay in incorporation of successive bases in a sample versus an unmodified control. Unmethylated motifs were identified using inter pulse duration ratio (ipdR) measurements [<xref ref-type="bibr" rid="pgen.1005854.ref032">32</xref>], and read coverage. For each methylated motif, an &#8216;under-methylated&#8217; ipdR threshold was determined by comparison of ipdR scores of bases in methylated motifs with those in unmethylated, non-motif sequences. ipdR scores for all motif sites in the genome were ranked, and an average motif ipdR calculated across the central 60% values (to minimize the effect of unmethylated sites or other outliers). The average non-motif ipdR was similarly calculated from the central 60% of ranked ipdR scores from all bases of the same type in non-motif sequences in the genome. The under-methylated ipdR threshold was then defined as (0.1*average motif ipdR)+(0.9*average non-motif ipdR), i.e. an approximation of the idpR score if 10% of bases were methylated. For comparison, a &#8216;methylated&#8217; ipdR threshold was defined as (0.5*average motif ipdR)+(0.5*average non-motif ipdR), i.e. an approximation of the idpR score if at least half of bases were methylated. Analysis of unmethylated motifs was only performed for palindromic Type II motifs that have two methylated sites (one on each strand of the genome). Motif instances were considered &#8216;unmethylated&#8217; if both potential methylated bases had at least twenty-fold SMRT sequence coverage, and an ipdR less than the &#8216;under-methylated&#8217; threshold. Importantly, the average SMRT sequence coverage at unmethylated sites is no different from that at methylated sites (<bold><xref ref-type="supplementary-material" rid="pgen.1005854.s004">S4 Fig</xref></bold>). They are therefore high-confidence unmethylated sites, and not simply borderline cases at the thresholds for inclusion in the analysis.</p><p>Gene regulatory regions were defined as 100bp upstream of the CDS start to 50bp downstream of the CDS start. Fold enrichment of all motif sequences in gene regulatory regions was determined by comparison with the average fraction of randomized control sequences in regulatory regions (1000 random samplings of an equal number of sites in the genome with the same length and nucleotide composition as the modified motif). Fold enrichment of unmethylated motifs in gene regulatory regions was determined by comparison with the fraction of methylated motifs in regulatory regions (sites with ipdR scores greater than the &#8216;methylated&#8217; threshold defined above). Significance of enrichment was determined using Fisher&#8217;s exact test. To determine the potential enrichment of unmethylated motifs for specific functional classes of genes, we repeated these analyses using individual subsets of regulatory regions grouped according to the COG category annotations [<xref ref-type="bibr" rid="pgen.1005854.ref058">58</xref>] of their corresponding genes.</p><p>For each orphan MTase family associated with incomplete methylation of the genome (<bold><xref ref-type="fig" rid="pgen.1005854.g004">Fig 4</xref></bold>), we identified conserved unmethylated sites based on conservation of flanking gene sequences (<bold><xref ref-type="table" rid="pgen.1005854.t001">Table 1</xref></bold>). For every unmethylated site, we took the amino acid sequences of the two flanking genes, and identified best hits in each of the other genomes using BLASTP. Pairs of unmethylated sites across the two genomes were considered conserved if their flanking genes were reciprocal best hits in the respective other genome. For select conserved sites, genomic DNA sequences upstream of the putative target gene were subject to multiple alignment using MAFFT [<xref ref-type="bibr" rid="pgen.1005854.ref059">59</xref>] and visualized in Jalview [<xref ref-type="bibr" rid="pgen.1005854.ref060">60</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To identify potential overlap between unmethylated <italic>Dam</italic> GATC motifs and transcription factor binding sites, we obtained curated transcription factor binding site (TFBS) probability matrices from <ext-link ext-link-type="uri" ns0:href="http://regtransbase.lbl.gov/">http://regtransbase.lbl.gov</ext-link> [<xref ref-type="bibr" rid="pgen.1005854.ref061">61</xref>], and searched for matches to these matrices using MAST [<xref ref-type="bibr" rid="pgen.1005854.ref062">62</xref>]. We restricted our search to TFBS that were identified in gammaproteobaceria (using the taxonomic classifications provided by regtransbase), and to gammaproteobacerial genomes from our study that contain at least 4 unmethylated Dam motif sites. We identified overlap between predicted TFBSs and unmethylated Dam motifs using bedtools [<xref ref-type="bibr" rid="pgen.1005854.ref063">63</xref>].</p><p>Candidate enriched clusters of unmethylated motifs were identified as regions of the genome containing at least 3 consecutive unmethylated motifs, each separated from its nearest neighbor by less than the genome-wide average distance between motifs (<bold><xref ref-type="table" rid="pgen.1005854.t002">Table 2</xref></bold>). A cluster of unmethylated sites was considered significant if the probability of observing such a series of consecutive sites by chance was &lt; = 0.01, based on 10,000 iterations of randomly sampling <italic>n</italic> = (number of unmethylated sites) times from an ordered list of length <italic>l</italic> = (total motif sites).</p><p>We searched for unusual clusters of motifs (regardless of methylation state) in non-coding regions of the genome (defined using IMG gene annotations of coding DNA sequences, and excluding RNA gene annotations). For each non-coding region, we calculated the local non-coding motif density (motifs / bp) across the 100 flanking non-coding regions. The expected number of motifs in that region was then estimated as (non-coding region length)*(local motif density). Fold enrichment was determined as observed number of motifs / expected number of motifs. P-values were determined using the pbinom function in R (p_val = 1-pbinom (observed number of motifs, non-coding region length, expected number of motifs)), and subject to Bonferroni correction using the number of non-coding regions in the genome. A p-value threshold of 1e-5 was used to identify non-coding regions with motif enrichment. We identified patterns of non-coding motif enrichment conserved across organisms, based on reciprocal best BLAST hits of flanking genes (as described above).</p><p>For methylated motifs showing conserved patterns of motif enrichment, we extended our analyses to other sequenced genomes in the same taxonomic class (<bold><xref ref-type="fig" rid="pgen.1005854.g006">Fig 6</xref></bold>). For each genome, we calculated motif density in 500bp windows with a 50bp step-size across a 50kb region of the genome centered on the start or end of the gene closest to the motif cluster. BLAST was used to search each genome for orthologs of the responsible MTase, and determine correlation between presence of MTase and non-coding motif clusters.</p>