<p>Mice were treated in accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals (The National Academies Press; Washington DC, 8th edition). The protocol was approved by the Animal Care and Usage Committee of NCI-Frederick (NIH) (Animal Study Proposals: 12&#8211;471 and 15&#8211;471).</p><p>A single base substitution (G&gt;A) was generated in the 6<sup>th</sup> nucleotide of exon 3 to change codon 25 (GGA&gt; AGA, Gly25Arg) of <italic>Brca2</italic> clone in BAC RPCI22-421-23A by recombineering in SW102 cells using the <italic>galK-</italic>based selection-counterselection method as described previously[<xref ref-type="bibr" rid="pgen.1006236.ref021">21</xref>, <xref ref-type="bibr" rid="pgen.1006236.ref047">47</xref>]. The G&gt;A substitution generated a <italic>Mse</italic>I restriction enzyme recognition site. Next, a <italic>loxP</italic>-PGK-Tn5-<italic>neomycin</italic>-bp(A)-<italic>loxP</italic> cassette along with <italic>Ban</italic>II and <italic>Eco</italic>RI restriction sites were inserted into intron 3 by recombineering. Finally, a 10.7 kb genomic region of <italic>Brca2</italic> was retrieved by gap repair from the BAC into pBSK(+) plasmid containing the <italic>Thymidine Kinase</italic> (TK) gene under the control of the MC1 promoter. The resultant plasmid was the G25R knock-in targeting vector (<xref ref-type="supplementary-material" rid="pgen.1006236.s001">S1 Fig</xref>).</p><p>G25R knock-in targeting vector was linearized with <italic>Not</italic>I and electroporated into V6.4 (129 X C57BL/6J) ES cell line as described previously [<xref ref-type="bibr" rid="pgen.1006236.ref048">48</xref>]. G418<sup>R</sup>, FIAU<sup>R</sup> ES cell clones were screened by Southern analysis of <italic>Ban</italic>II and <italic>Eco</italic>RI restriction enzyme digested genomic DNA using 5&#8217; and 3&#8217; probes (<xref ref-type="supplementary-material" rid="pgen.1006236.s001">S1 Fig</xref>). A few correctly targeted ES cell clones were injected into C57BL/6 blastocysts to generate chimeras. One of these chimeras transmitted the targeted allele in the germ line and <italic>Brca2</italic><sup><italic>G25R-Neo/+</italic></sup> pups were obtained.</p><p>To obtain <italic>Brca2</italic><sup><italic>G25R/+</italic></sup> mice, <italic>Brca2</italic><sup><italic>G25R-Neo/+</italic></sup> mice were crossed to &#946;-actin-Cre-deleter strain [<xref ref-type="bibr" rid="pgen.1006236.ref049">49</xref>]. To generate <italic>Palb2</italic><sup><italic>Ko/+</italic></sup> mice, CG0691 ES cells obtained from SIGTR were injected to into C57BL/6 blastocysts. Chimeras were crossed with C57BL/6 females to <italic>Palb2</italic><sup><italic>Ko/+</italic></sup> pups. <italic>Brca2</italic><sup><italic>G25R/Ko</italic></sup> were obtained by breeding <italic>Brca2</italic><sup><italic>G25R/G25R</italic></sup> or <italic>Brca2</italic><sup><italic>G25R/+</italic></sup> mice with <italic>Brca2</italic><sup><italic>ko/+</italic></sup> mice [<xref ref-type="bibr" rid="pgen.1006236.ref050">50</xref>]. <italic>Brca2</italic><sup><italic>G25R/Ko</italic></sup> mice were crossed with <italic>Palb2</italic><sup><italic>Ko/+</italic></sup> and <italic>Trp53</italic><sup><italic>Ko/+</italic></sup> mice to obtain mice of various genotypes used in the study. Mouse colony was maintained on a mixed C57BL/6 and 129/SvEv genetic background. All animal studies were performed as per the protocols outlined in the Guide for the Care and Use of Laboratory Animals and approved by the NCI-Frederick Animal Care and Use Committee.</p><p>For genotyping purposes, genomic DNA was extracted from tail biopsies, fresh or frozen tumor tissues following standard procedures. Genotyping was performed by PCR. Briefly, approximately 50 ng of DNA was amplified with the Taq DNA polymerase using primers listed in Supplementary Information (<xref ref-type="supplementary-material" rid="pgen.1006236.s001">S1A and S1D Fig</xref>, <xref ref-type="supplementary-material" rid="pgen.1006236.s011">S1 Table</xref>).</p><p>For histology, tissues were fixed in 10% formalin overnight, paraffin-embedded, sectioned and stained with H&amp;E (hematoxylin and eosin). Statistical differences in tumor types were assessed via Fisher's exact test, and survival curves were analyzed utilizing Prism Graphpad. For germ-cell visualization 3-week old testis sections were immunostained with germ cell marker (MVH) as previously described [<xref ref-type="bibr" rid="pgen.1006236.ref051">51</xref>]. Antibodies: Rabbit anti-DDX4/MVH (Abcam ab13840; 1:400); goat anti-rabbit Alexa 488 conjugate (Molecular Probes A11008; 1:1,000).</p><p>MEFs were generated from E13.5 embryos and cultured in 10% FBS at 5% O<sub>2</sub>. Cell growth analyses were done by seeding 10cm plates with 5 X 10<sup>5</sup> cells and counting the cells every day. Metaphases were collected after a 24 hour treatment with 100nM MMC and spread as previously described [<xref ref-type="bibr" rid="pgen.1006236.ref052">52</xref>]. Immunofluorescence of RAD51 foci was done by seeding cells on coverslips, irradiating with 1000 rads the following day. Fixed cells were collected after 3 hours and cells were stained with Mouse anti-&#947;H2AX (1:500, Millipore JBW301) and rabbit anti-RAD51 (1:250, Millipore PC130).</p><p>Paraffin sections of testes were TUNEL stained using the In Situ Cell Death Detection Kit (Roche 11684817910) following the directions included with the kit.</p><p>Meiotic spreads were performed as previously described[<xref ref-type="bibr" rid="pgen.1006236.ref053">53</xref>]. Primary antibodies used in this study: rabbit anti-SYCP3 (1:500, Abcam ab13840); mouse anti-&#947;H2AX (1:500, Millipore JBW301); mouse anti-SYCP3 (1:500 Abcam ab20244); rabbit anti-RAD51 (1:250, Millipore PC130).</p><p>A comparative genomic hybridization array was used for 4 tumors from the following genotypes, <italic>Brca2</italic><sup><italic>G25R/+</italic></sup><italic>;Trp53</italic><sup><italic>Ko/+</italic></sup> (2 Lymphomas, 2 Osteosarcoma), <italic>Brca2</italic><sup><italic>G25R/G25R</italic></sup><italic>;Trp53</italic><sup><italic>Ko/+</italic></sup>, (2 Lymphomas, 2 Osteosarcoma) <italic>Brca2</italic><sup><italic>G25R/Ko</italic></sup><italic>;Trp53</italic><sup><italic>Ko/+</italic></sup> (4 Osteosarcomas) and <italic>Brca2</italic><sup><italic>G25R/G25R</italic></sup><italic>;Palb2</italic><sup><italic>Ko/+</italic></sup><italic>;Trp53</italic><sup><italic>Ko/+</italic></sup> (2 Sarcomas, 2 Osteosarcoma). The DNA was isolated using the Qiagen DNeasy Blood and Tissue Kit, wild-type mouse tail DNA was used as the control. The DNA was hybridized to an Agilent SurePrint G3 Mouse CGH Array 4X180K chip. The results from the array were analyzed using Nexus Copy Number from BioDiscovery.</p><p>DNA fiber analysis was performed as previously described with minor modifications [<xref ref-type="bibr" rid="pgen.1006236.ref054">54</xref>, <xref ref-type="bibr" rid="pgen.1006236.ref055">55</xref>]. Briefly, cells were seeded at 50% on the previous day. CldU was added to media for 15 min, cells were washed 3X with pre-warmed 1X PBS. Then IdU was added in pre-warmed media for 15 min. Cells were then treated with 4mM HU for 4 hrs. Cells were washed 2X with pre-warmed 1X PBS, trypsinized and placed in cold PBS at a concentration of 2.5 X 10<sup>5</sup> cells/ml. On pre-cleaned slides 7.5 &#956;L of cell lysis solution (200mM Tris-HCl pH 7.4, 50mM EDTA, 0.5%SDS) was placed near the top of the slide. Then 2.5 &#956;L of cells were added to this drop, mixed and then left at RT for 8 minutes. The slide was then tilted to allow spreading of fibers at approximately 45&#176; angle to allow the drop to slowly run down the slide. The slides were then air-dried. Next, the slides were fixed in methanol/acetic acid 3:1 at 4&#176; O/N. Slides were then re-hydrated with 1X PBS, denatured in 2.5M HCL for 1 hr, washed 5X with PBS. The slides were then blocked (2% BSA, 0.1% Tween 20, 1X PBS) for 40 minutes. The slides were then labeled with mouse anti-BrdU/IdU 1:100 (Becton Dickinson, #347580) and rat anti-BrdU/CldU (1:500) for 2.5hrs at RT or 4&#176; O/N. Slides were then washed 5X with 0.2% PBS-Tween 20. Secondary antibodies, Alexa-fluors anti-rat 594, and anti-mouse 488 were used at 1:300 for 1 hr in dark at RT. Slides were washed again with 0.2% PBST, air-dried and mounted with anti-fade. Fibers were then visualized and imaged at 63X. Measurements of fibers were performed utilizing ImageJ software.</p>