<p>Our study of data use conditions included: (1) a review of guidance from the NIH; (2) a survey of data use conditions for data accessed within a large research institute, the Broad Institute of MIT and Harvard, and for data held within the EGA [<xref ref-type="bibr" rid="pgen.1005772.ref025">25</xref>] of the EMBL-EBI and CRG; and (3) consultation with the international scientific community via the Global Alliance Data Working Group and with the MME project. Taken together, these provided us with a broad overview and evidence from a large number of datasets that are representative of the field.</p><p>The NIH expects that any limitations on the research use of the data, as expressed in the informed consent documents, should be clearly delineated in the institution&#8217;s assurance of the data submission to the dbGaP of the National Center for Biotechnology Information (NCBI) [<xref ref-type="bibr" rid="pgen.1005772.ref026">26</xref>], known as the Institutional Certification [<xref ref-type="bibr" rid="pgen.1005772.ref027">27</xref>]. For example, in some studies in dbGaP, a subset of research participants in a study may have provided informed consent that their data could only be used by not-for-profit companies that have no interest in commercializing a product as a result of the secondary data use. In an effort to provide greater consistency among the commonly used groups of limitations, the NIH developed a library of DULs [<xref ref-type="bibr" rid="pgen.1005772.ref007">7</xref>]. In this case, the NIH standard DUL for this subgroup of participants would be &#8220;not-for-profit use only,&#8221; to indicate that, based on the original informed consent of the study participants, the data can be used only by not-for-profit organizations. To access &#8220;controlled-access&#8221; dbGaP datasets, researchers must submit a data access request (DAR), which includes a proposed research use statement, as well as agreement to the terms of use as described in the Data Use Certification Agreement (DUC) and attestation to the elements in the Genomic Data User Code of Conduct. The DULs for each of the datasets being requested are stipulated in the request process and the final DAR application. The relevant Data Access Committee (DAC) [<xref ref-type="bibr" rid="pgen.1005772.ref028">28</xref>] reviews the DARs, and its decisions are based primarily upon conformity of the proposed research, as described in the DAR, to the DULs specified by the submitting institution.</p><p>The EGA follows a similar procedure for access to the controlled-access datasets it stores: Researchers requesting data apply to a given DAC in order to be granted access to the controlled data [<xref ref-type="bibr" rid="pgen.1005772.ref029">29</xref>]. Each DAC reviews whether the applicant&#8217;s research purpose fits with the provided controlled Data Access Agreement (DAA) with the data donors. A difference with dbGaP is that the DAC process is not centralized, and each DAC has its own policy for gathering the information relating to application acceptance or rejection.</p><p>We also incorporated the findings of a research study that analyzed data use limitations in DUCs and other access agreements for datasets used at the Broad Institute of MIT and Harvard. This identified additional data use conditions as well as provided an estimate of their preponderance. Finally, we also incorporated knowledge of additional data use conditions from the experience of the EGA and of the members of the Global Alliance Data Working Group [<xref ref-type="bibr" rid="pgen.1005772.ref030">30</xref>]. As the material we drew upon stemmed largely from experience with research datasets, we also sought a clinical genetics perspective from leaders of the MME project [<xref ref-type="bibr" rid="pgen.1005772.ref031">31</xref>], a collaborative initiative to share data to enable rare disease gene discovery and supported by the Global Alliance, the International Rare Disease Research Consortium (IRDiRC) [<xref ref-type="bibr" rid="pgen.1005772.ref032">32</xref>], ClinGen [<xref ref-type="bibr" rid="pgen.1005772.ref033">33</xref>], and the individual MME services. We note that the MME and Beacon projects represent distinct approaches to data sharing. The MME provides access to extremely detailed patient data (e.g., matching case-level detail and data owner) and requires logging into an MME service and depositing case-level data such as phenotype and gene/genotype. The structure of the Beacon project allows for very easy access to data, but the data shared are less detailed (currently only the existence of a variant).</p><p>We identified 19 categories of data use conditions and clarified their descriptions. These were then classified into &#8220;primary categories&#8221; or &#8220;secondary categories&#8221; and as &#8220;requirements,&#8221; based on their occurrence and on the resulting criteria for data access.</p><p>For validation purposes and to ensure as much compatibility as possible, we compared our classification to how the NIH ascribed dbGaP Consent Groups based on the NIH DULs, and to the DAA categorization performed by the EGA on the DAAs related to the studies with data stored in it. We included the abbreviations adopted by the NIH for overlapping data use conditions (e.g., GRU, HMB).</p>