<p>
<italic>Chrysochromulina tobin</italic> strain CCMP291, acquired from The National Center for Marine Algae by the Cattolico laboratory in 2006, was designated as P3. These cultures were maintained in 250 mL Erlenmeyer flasks containing 100 mL of RAC-1, a proprietary fresh water medium. Flasks were plugged with silicone sponge stoppers (Bellco Glass, Vineland, NJ) and capped with a sterilizer bag (Propper Manufacturing, Long Island City, NY). Large volume experimental cultures for genomic DNA and transcriptomic RNA harvesting were maintained in 1.0 L of RAC-1 medium that was contained in 2.8 L large-mouth Fernbach flasks. These flasks were plugged with hand-rolled, #50 cheese cloth-covered cotton stoppers and covered with a #2 size Kraft bag (Paper Mart, Orange, CA). All cultures were maintained at 20&#176;C on a 12 hour light: 12 hour dark photoperiod under 100 &#956;Em<sup>-2</sup>s<sup>-1</sup> light intensity using full spectrum T12 fluorescent light bulbs (Philips Electronics, Stamford, CT). No CO<sub>2</sub> was provided and cultures were not agitated.</p><p>Algal cultures were treated in the following manner to minimize bacterial contamination. P3 cultures were subject to re-iterative cell sorting using flow cytometry. <italic>C</italic>. <italic>tobin</italic> cells were stained for identification using BODIPY 505/515 (4,4-difluoro-1,3,5,7-tetramethyl-4-bora-3a,4a-diaza-s-indacene; Invitrogen, Carlsbad, CA), a neutral lipid binding fluorophore. Approximately 10 stained cells were sorted into a single well of a 96 well plate containing 100 &#956;L RAC-1 medium and then transferred to 10 mL of RAC-1 medium in 50 mL plastic tissue culture flasks (Nunc, Roskilde, Denmark). This cell sorting process was carried out 4 times with the resulting culture being designated as P4. Cells obtained from reiterative flow cytometric selection (P4) were then treated in RAC-1 medium that contained either streptomycin (resulting in culture P5.5) or hygromycin (P5.6). Treatment with these two antibiotics was identical. Cells were exposed to a final concentration of 400 &#956;g/mL antibiotic for 18 hours before 5 mL of treated cultures were transferred to 100 mL of antibiotic free RAC-1 medium. Cultures P5.5 and P5.6 were periodically tested for bacterial contamination using liquid LB medium made with RAC-1 medium in replacement of water. Sequencing data and recovery of a cultured bacterial isolate has shown that a single bacterial contaminant is still present in the P5.5 culture.</p><p>Total genomic DNA was collected from each of the P5.5 and P5.6 cultures using the Qiagen Genomic-tip Maxi DNA extraction protocol (Germantown, MD) with the following changes to the standard protocol. 1.5 x 10<sup>8</sup> cells were harvested by centrifugation at 5,378 x g for 20 minutes and resuspended in lysis buffer (20 mM EDTA, pH 8.0; 10 mM Tris-base, pH 8.0; 1% Triton X; 500 mM guanidine; 200 mM NaCl). After 1.0 hour incubation at 37&#176;C, RNase A was added to 200 &#956;g/ml final concentration and the mixture incubated for 30 minutes at 37&#176;C. Following the addition of 600 &#956;L Proteinase K (20 mg/ml) (Sigma-Aldrich) incubation was continued at 50&#176;C for 2.0 hours, mixing every 30 minutes by swirling. DNA preparation was transferred into a Qiagen DNA binding tip (Maxi size) that was equilibrated using the manufacturer&#8217;s instructions, and allowed to pass through the tip by gravity, while maintained at room temperature. The tip was then washed twice using Qiagen buffer QC. Fifteen mL of Buffer QF (at 37&#176;C) was added to the tip to elute the DNA. DNA was precipitated by adding 10.5 mL of 100% room temperature isopropanol followed by centrifugation at 11,220 x g for 20 min at 4&#176;C. The pellet was washed in 4 mL of 4&#176;C 70% ethanol and centrifuged again using the same conditions. The DNA pellet was air dried for 5 min and resuspended in warmed Qiagen buffer EB (50&#176;C) and incubated at 50&#176;C for 2.0 hours. DNA solution was quantitated using a spectrophotometer and subsequently transferred to 1.7 mL Eppendorf tubes and stored at -80&#176;C.</p><p>The <italic>C</italic>. <italic>tobin</italic> genome was sequenced using a combination of Illumina and 454 sequencing. For Illumina, two shotgun libraries (2 X 100 and 1 x 150 base pair) were prepared using standard TruSeq protocols and sequenced on an Illumina HiSeq2000. Using the 454 Titanium platform, shotgun single-end and paired-end (10 kb insert) DNA libraries were prepared generating 4.7 million reads in total. The 454 single end and paired end data (insert size 8180 +/- 1495 bp) were assembled using Newbler, version 2.3 (release 091027_1459) (Roche). The sequences generated by the Illumina platform were assembled separately with VELVET, version 1.0.13 [<xref ref-type="bibr" rid="pgen.1005469.ref095">95</xref>]. Consensus sequences from the VELVET and Newbler assemblies were computationally shredded into 10 kb fragments and were re-assembled with reads from the 454 paired end library using parallel&#8236; Phrap, version 1.080812 (High Performance Software, LLC).</p><p>The final draft genome assembly produced over three thousand contigs. Gene annotation was carried out using the MAKER2 training and annotation pipeline [<xref ref-type="bibr" rid="pgen.1005469.ref096">96</xref>]. After masking repeated genomic elements using Repeatmasker [<xref ref-type="bibr" rid="pgen.1005469.ref097">97</xref>], genes were modeled by combining several methods in MAKER2: a) aligning <italic>C</italic>. <italic>tobin</italic> transcriptomic BLASTn hits as EST evidence; b) aligning to <italic>Emiliania huxleyi</italic> ESTs using tBLASTx; c) using the assembled RNAseq data for gene prediction with Tophat [<xref ref-type="bibr" rid="pgen.1005469.ref098">98</xref>] and Cufflinks [<xref ref-type="bibr" rid="pgen.1005469.ref099">99</xref>]; d) aligning all CEGMA (Core eukaryotic genes) [<xref ref-type="bibr" rid="pgen.1005469.ref100">100</xref>] genes to the <italic>C</italic>. <italic>tobin</italic> contigs using BLASTx; e) Augustus [<xref ref-type="bibr" rid="pgen.1005469.ref101">101</xref>] for <italic>ab initio</italic> models trained on the gene structures of <italic>Chlamydomonas reinhardtii</italic>; f) SNAP [<xref ref-type="bibr" rid="pgen.1005469.ref102">102</xref>] for <italic>ab initio</italic> models trained on Hidden Markov Models (HMMs) of the predicted genes by cufflinks and Tophat models; g) GenemarkES for <italic>ab initio</italic> gene models [<xref ref-type="bibr" rid="pgen.1005469.ref103">103</xref>]. A total of 16,777 genes were annotated using the above method. Of these, 10,293 were supported by BLAST homology using BLAST2GO and 6,484 are considered novel genes.</p><p>BLAST2GO [<xref ref-type="bibr" rid="pgen.1005469.ref104">104</xref>] was used to attach functional annotation to gene call predictions. First, BLASTp was used to search the non-redundant protein database (nr) with a Blast Expect Value cutoff of 1e<sup>-6</sup>. Blast2Go Mapping was performed followed by Annotation using E-Value-Hit-Filter: 1e<sup>-6</sup>, Annotation cutoff of 55 and GO weight of 5. These gene annotations were used in the remainder of the gene analyses unless the manual curation of a gene gave evidence supporting a manual annotation.</p><p>Twelve 1 L cultures were seeded at a starting density of 50,000 cells/ mL, 66 hours prior to the first harvesting time point (Dark hour 6) using inoculation cultures that were maintained for 7 days at standard conditions (see above). Total RNA was purified using a modified TRIzol preparation: 1.5 x 10<sup>8</sup> cells were collected per RNA isolation sample. <italic>C</italic>. <italic>tobin</italic> cells were centrifuged at 8,600 x g for 20 minutes in 500 mL polypropylene centrifuge bottles. The supernatant was decanted and 5 mL of TRIZOL reagent (Invitrogen) was added to the cell pellet. Cells were resuspended by pipetting and vortexing for 1 minute. The homogenate was transferred equally into four microcentrifuge tubes. To each tube, 250 &#956;L of chloroform was added. Each tube was shaken by hand vigorously for 15 seconds and subsequently centrifuged for 15 minutes at 12,000 x g at 4&#176;C. The mixing and centrifugation was repeated once. After the second centrifugation, the top aqueous phase was transferred to a new microcentrifuge tube being sure not to disturb the lower phenol/chloroform phase. Ice cold isopropanol (625 &#956;L) was added to each of the 4 tubes containing the aqueous phase and incubated at -20&#176;C overnight. The samples were then centrifuged at 12,000 x g for 10 minutes at 4&#176;C. The supernatant was removed and the pellet washed with 1.25 mL of 75% ice cold ethanol followed by a 5 minute centrifugation at 7,400 x g. The ethanol wash and centrifugation step was repeated one time. The pellet was dried for 10 minutes and resuspended in 30 &#956;L RNase free water (Qiagen). Four samples were combined into a single tube and treated with RNase free DNase for 90 minutes at 37&#176;C. Samples were then cleaned using a Qiagen RNeasy MinElute clean up protocol as specified by the manufacturer&#8217;s instructions. Samples were stored at -80&#176;C</p><p>Poly-A selection was carried out followed by library preparation using a TruSeq library kit (Illumina). Sequencing was done on the Illumina high-seq and generated 100 bp paired reads. For each time point collected, 15&#8211;30 million reads were generated. Reads were trimmed and groomed [<xref ref-type="bibr" rid="pgen.1005469.ref105">105</xref>]. Tophat version 1.5 [<xref ref-type="bibr" rid="pgen.1005469.ref098">98</xref>] was used to assemble the sequences using the <italic>C</italic>. <italic>tobin</italic> draft genome as a reference. Cufflinks (v2.1.1) was used to estimate FPKM (fragments per kilobase of exon per million mapped reads) for each transcript at each time point [<xref ref-type="bibr" rid="pgen.1005469.ref106">106</xref>]. To determine which subset of the transcriptomic data to include in the global analysis a high expression and high variance selection method was implemented to determine genes that were 1) highly expressed and 2) had great differences in expression between 2 or more time points. This gene selection was done using an in-house derived formula (&#8220;MeanNeighbor&#8221;) that takes each time point and compares the expression level (FPKM value) average across adjacent time points. This method was implemented in R using the following function: MeanNeighbor = function(x) {mean(abs(x[2:length(x)]-x[1:length(x)-1)]))}. The top 1000 genes as scored by MeanNeighbor value were then plotted as a heatmap using a normalization constant so that all values of gene expression FPKM could be represented by a relative level between -3 and +3. First, each individual FPKM value was subtracted from each transcript&#8217;s average FPKM value of all 7 time points. The resulting value at each time point was then divided by the standard deviation, giving a relative expression level to be used in the generation of the heatmap. The global heatmap was generated using the R library &#8220;pheatmap&#8221; [<xref ref-type="bibr" rid="pgen.1005469.ref107">107</xref>] using the &#8220;ward&#8221; clustering method. Fisher&#8217;s exact test was used in Blast2GO to determine GO term overrepresentation in each group based on the annotation of GO terms by Blast2GO. The p-value cutoff for this was set at 0.05. For group 3, over 100 members were obtained so the p-value cutoff was lowered to 1e<sup>-7</sup> to generate the graphs used in <xref ref-type="fig" rid="pgen.1005469.g004">Fig 4</xref>.</p><p>RNA seq data was also assembled <italic>de novo</italic> to identify genes that may have not been present or were mis-assembled in the final <italic>C</italic>. <italic>tobin</italic> nuclear genome draft. Trinity [<xref ref-type="bibr" rid="pgen.1005469.ref108">108</xref>] was used to assemble transcripts, which were used to create an additional BLAST database used in identifying NAD genes, cobalamin synthesis, and polyketide related genes in addition to those found in the nuclear genome draft.</p><p>Total cellular neutral lipid content of the experimental cultures was measured as follows. The BODIPY 505/515 (Invitrogen, Eugene, OR) stock solution was prepared by adding the dry BODIPY 505/515 powder to 99% pure DMSO for a 5 mM final stock concentration. 7.5 &#956;L of the stock solution was diluted 3:1 in 22.5 &#956;L of RAC-1 medium for the working stock solution. A 990 &#956;L aliquot of cell culture and 10 &#956;L of working stock solution were placed into a 12 x 75 mm glass tissue culture tube for use in flow cytometric measurements. The tube was capped, inverted to mix the dye, and incubated in the dark at room temperature for at least one minute. BODIPY 505/515 labeled samples were measured using a BD Accuri C6 flow cytometer in the FL1 channel (excitation: 488 nm; 530/30 nm emission). Unstained cells were used as a control. The BODIPY 505/515 background was negligible. Because BODIPY 505/515 stained samples are spectrally distant and of much higher signal strength than chlorophyll auto-fluorescence and cellular debris, experimental samples are easily gated.</p><p>Samples were collected for total fatty acid analysis when algal cultures were in stationary growth phase (<xref ref-type="supplementary-material" rid="pgen.1005469.s017">S4 Table</xref>). GC/MS analysis was performed using the sub-microscale in-situ method devised in this laboratory [<xref ref-type="bibr" rid="pgen.1005469.ref041">41</xref>]. Briefly, 10 mL culture aliquots (quadruplicate samples) were placed in new 10 mL Pyrex glass tubes (Fisher Scientific, Pittsburgh, PA), centrifuged at 5,900 x g for 20 min at 4&#176;C, and the pelleted cells flash-frozen in liquid nitrogen. Samples were then stored at -80&#176;C before lyophilization and chemical processing. The fatty acids present in the lyophilized samples were transmethylated to fatty acid methyl esters in-situ, catalyzed by boron trifluoride in methanol. A two-component triglyceride surrogate was added to the sample prior to transmethylation to account for any variation in methylation or sample handling prior to internal standardization. After transmethylation, the analytes were separated from the other compounds present in the sample using a two-phase (brine and isooctane), two-step phase separation. An internal standard of deuterated aromatics was then added to the sample. Analyte separation and detection was performed using GC/MS. Quantitation was performed against a 27-component external standard.</p><p>The fluorescein diacetate (FDA) assay used in this study was modified from a study by Jochem [<xref ref-type="bibr" rid="pgen.1005469.ref109">109</xref>]. A 5 mg/mL stock solution of FDA, 3,6-Diacetoxyfluoran, Di-O-acetylfluorescein (Sigma-Aldrich, St. Louis, MO) was prepared in 99.9% anhydrous DMSO (Sigma-Aldrich, St. Louis, MO). This stock solution was stored in a 15 mL Falcon tube at 4&#176;C. For experimental runs, the stock solution was thawed and diluted 100-fold with chilled double-distilled water to make the 50 &#956;g/mL FDA working solution. 330 &#956;L of the working solution was added to 10 mL of cell culture in a 12 x 75 mm glass tissue culture tube (BD Biosciences, San Jose, CA, USA). After gentle vortexing for approximately 5&#8211;10 seconds on a low setting, 1.0 mL of each cell/FDA sample was placed into 8 replicate wells of a clear 96-well plate (BD Biosciences, San Jose, CA, USA). The plate was incubated for 10 minutes at 20&#176;C under normal room illumination or shielded from all light depending on the hour of the light:dark cycle. The FDA signal was measured using an Accuri C6 flow cytometer in the FL1 channel (excitation: 488 nm; emission: 530/30 nm). Unstained cells were used as a control.</p><p>The Rosetta comparative modeling protocol [<xref ref-type="bibr" rid="pgen.1005469.ref110">110</xref>] was used to model the tertiary structure of <italic>C</italic>. <italic>tobin</italic> xanthorhodopsin using the template 3DDL [<xref ref-type="bibr" rid="pgen.1005469.ref075">75</xref>] from the protein data bank. Approximately 41,000 trajectories were performed using Rosetta version 3 [<xref ref-type="bibr" rid="pgen.1005469.ref111">111</xref>] with a new energy function [<xref ref-type="bibr" rid="pgen.1005469.ref112">112</xref>]. Secondary structure prediction for the query sequence was made using PsiPred [<xref ref-type="bibr" rid="pgen.1005469.ref113">113</xref>] and fragments (3- and 9- residue long) were created using Robetta server [<xref ref-type="bibr" rid="pgen.1005469.ref114">114</xref>]. The models were clustered based on their backbone RMSD and top representatives based on lowest full atom Rosetta energy and were visually evaluated using protein structure visualization software, PyMOL (v0.99, Schr&#246;dinger, LLC). Top 10 models by total score (-404 to -400 Rosetta Energy Units or -1.5 REU/residue) out of approximately 41,000 generated models were within around 2 &#197; C&#945;-RMSD from the template (3DDL). The seven helices of the model aligned well (<xref ref-type="fig" rid="pgen.1005469.g009">Fig 9</xref>) with maximum deviation in the loop region connecting second and third helix of the model (residues 71&#8211;93).</p><p>Docking studies of retinal in the putative pocket of <italic>C</italic>. <italic>tobin</italic> xanthorhodopsin structural models were also performed. Retinal molecule coordinates were taken from the structural template 3DDL. Carbonyl oxygen and protons were added to the molecule using Avogadro molecule editor software [<xref ref-type="bibr" rid="pgen.1005469.ref115">115</xref>]. The five rotatable bonds in a retinal molecule were sampled at two more states, &#177;30&#176; from the dihedral angles that were observed in the structure, 3DDL. Approximately 200 conformers of retinal molecule were generated that had intra-molecule full atom repulsive energy, as calculated by Rosetta [<xref ref-type="bibr" rid="pgen.1005469.ref116">116</xref>], not worse than the starting molecule used for conformer generation. Random docking of retinal conformers in top 10 selected comparative models of <italic>C</italic>. <italic>tobin</italic> xanthorhodopsin was achieved by the RosettaLigand protocol [<xref ref-type="bibr" rid="pgen.1005469.ref117">117</xref>]. A total of 6,300 dock trajectories were run for each comparative model and filtered based on ligand binding energy and ligand RMSD from the ligand conformation in the template (3DDL).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">NCBI non-redundant (nr) and MMETSP [<xref ref-type="bibr" rid="pgen.1005469.ref003">3</xref>] databases were mined for CbbX, rhodopsin, and <italic>psbA</italic> sequences for phylogenetic analysis. Relevant CbbX (262 sequences), rhodopsin (81 sequences), and <italic>psbA</italic> (18 taxa) sequences were aligned in MUSCLE [<xref ref-type="bibr" rid="pgen.1005469.ref118">118</xref>] and manually trimmed to leave 257 amino acid, 231 amino acid, and 857 nucleotide alignments, respectively. Best choice protein and nucleotide models were queried using ProtTest 2.4 [<xref ref-type="bibr" rid="pgen.1005469.ref119">119</xref>] and jModelTest [<xref ref-type="bibr" rid="pgen.1005469.ref120">120</xref>], respectively. The WAG+I+G model suited both the CbbX and rhodopsin datasets, and the <italic>psbA</italic> data were best modeled under GTR+I+G. The CbbX, rhodopsin and <italic>psbA</italic> gene trees were generated using the online CIPRES Science Gateway (<ext-link ext-link-type="uri" ns0:href="http://www.phylo.org">www.phylo.org</ext-link>) with MrBayes [<xref ref-type="bibr" rid="pgen.1005469.ref121">121</xref>] using the following conditions: 2 runs of four chains, with 3 million, 5 million, or 500,000 generations, respectively, and 25% burn-in. Convergence of the Bayesian analysis was viewed with Tracer [<xref ref-type="bibr" rid="pgen.1005469.ref122">122</xref>]. Maximum-likelihood analyses were performed using RAxML [<xref ref-type="bibr" rid="pgen.1005469.ref123">123</xref>], also in the CIPRES Science Gateway. The resulting output was visualized in Figtree v1.4.2 (<ext-link ext-link-type="uri" ns0:href="http://tree.bio.ed.ac.uk/software/figtree">http://tree.bio.ed.ac.uk/software/figtree</ext-link>).</p>