<p xmlns:ns0="http://www.w3.org/1999/xlink">The <italic>arid1-1</italic> T-DNA insertion mutant (SALK_047099) was obtained from the ABRC (<ext-link ext-link-type="uri" ns0:href="http://www.arabidopsis.org">www.arabidopsis.org</ext-link>). Seeds of HTR10-RFP and DUO1-RFP were kindly provided by Fred Berger and David Twell, respectively.</p><p>For the construction of ARID1-GFP, ARID1-RFP, and ARID1-Myc plasmids, <italic>ARID1</italic> was amplified from wild type (Columbia-0) genomic DNA with the primer pair ARID1F1 and ARIDR1, cloned into pENTR-D/TOPO, and then transferred into the plant expression destination vector pMDC107 to construct ARID1-GFP, into pMDC163 (GUS was replaced by a mRFP fusion tag) to construct ARID1-RFP, and into pEarleyGate303 to construct ARID1-Myc. For <italic>ARID1</italic> promoter analysis, a 1.5-kb fragment upstream of the ATG was amplified using the primer pair ARID1F1/R4 and was subcloned into pENTR-D/TOPO and then transferred into the plant expression vector pGII-NLS3XGFP. For constructing the antisense <italic>ARID1</italic> plasmid, cDNA was obtained using the primer pair ARID1F5/R5, cloned into pENTR-D/TOPO, then transferred into the plant expression vector pB7WG2 (digested with SacI and SpeI to replace the 35S promoter with the native <italic>ARID1</italic> promoter and then digested with KpnI and ApaI to insert the LAT52-GFP cassette). To construct the LAT52:ARID1 and HTR10:ARID1 plasmids, cDNA was obtained using the primer pair ARID1F6/R1, cloned into pENTR-D/TOPO, then transferred into the plant expression vector pB7WG2 (digested with SacI and SpeI to replace the 35S promoter with the LAT52 promoter or the HTR10 promoter, respectively, then digested with KpnI and ApaI to insert the LAT52-GFP cassette). To construct <italic>ARID1</italic> for the in vitro transcription and translation system, we inserted <italic>ARID1</italic> full length cDNA, amplified with the primer pairs ARID1F2/R6 into the pCMVTNT vector (Promega). For yeast two hybrid experiments, cDNAs of <italic>ARID1</italic> and <italic>ARID1-C</italic> were amplified from pollen RNA using RT-PCR and the primer pairs ARID1F2/R2 and ARID1F4/R2, then cloned into the pGBT9 yeast expression vector; the cDNA of <italic>HDA8</italic> was amplified using the primer pair HDA8F1/R1, then cloned into the pGAD10 yeast expression vector. For GST pulldown experiments, the cDNA of <italic>HDA8</italic> was amplified using primers HDA8F1/R1 and cloned into the pGEX-2TK vector. For the HDA8-YFP construct, <italic>HDA8</italic> was amplified from wild type genomic DNA with the primer pair HDA8F2/R2, cloned into pENTR-D/TOPO, and then transferred into the plant expression destination vector pGWB40. Primer sequences are listed in <xref ref-type="supplementary-material" rid="pgen.1004421.s007">Table S1</xref>.</p><p>The yeast two hybrid assays were performed according to the protocol available on the Clontech website, using strain AH109. Yeast transformation was performed using yeast transformation buffer (0.1 M LiAc, 40% PEG3350 in TE). Transformants were plated and selected on synthetic complete medium that lacked the specified amino acids. Positive colonies were inoculated and spotted with a 100-fold dilution onto synthetic complete medium lacking leucine, tryptophan, and histidine and containing 10 mM 3-amino-1,2,4-triazole. The plates were incubated for 2&#8211;4 d at 28&#176;C for interaction analysis.</p><p>ChIP was performed according to <xref ref-type="bibr" rid="pgen.1004421-Zheng1">[8]</xref>. ARID1 occupancy at the <italic>DUO1</italic> genomic region was determined by ChIP using a GFP antibody (Cat. 632460, Clontech, 1&#8758;200) and inflorescences from wild type and ARID1-GFP transgenic plants, respectively. The occupancy of histone modification marks at <italic>DUO1</italic> was determined by ChIP using Histone 3 antibody (Cat.06-755, Upstate, 1&#8758;50), Histone Lysine 9 acetylation antibody (Cat. 17-10241, Upstate, 1&#8758;200), and Histone Lysine 4 trimethylation antibody (Cat.ab8580, abCam, 1&#8758;200) and inflorescences from wild type and the <italic>arid1-1</italic> mutant, respectively. DNA present in the immunoprecipitates was quantified by qPCR or PCR relative to total input DNA. The results shown were consistent in two biological replicates. The primer sets used for the PCR are listed in <xref ref-type="supplementary-material" rid="pgen.1004421.s007">Table S1</xref>.</p><p>DNA binding assays were performed as described <xref ref-type="bibr" rid="pgen.1004421-Vert1">[41]</xref> with the following modifications. Briefly, biotinylated DNA fragments corresponding to 3, 4, 5, and 10 (<xref ref-type="fig" rid="pgen-1004421-g003">Figure 3</xref>) were generated by PCR using primer pairs DUO1_F3/R3, DUO1_F4/R4, DUO1_F5/R5, and DUO1_F10/R10, with labeling by 5&#8242;biotin at F3, F4, F5, and F10, respectively. Then 100 pmol of the biotinylated DNA fragments were incubated with 50 ul prewashed Streptavidin Agoraose Resin (Thermo, Cat.20349) in IP100 buffer (100 mM potassium glutamate, 50 mM Tris-HCl pH 7.6, 2 mM MgCl2, 0.5% NP40) for 2 h at room temperature with slight rotation, and washed five times in IP100 buffer. In parallel, ARID1 was subjected to TNT T7 in vitro transcription/translation with the TNT Coupled Wheat Germ Extract System (Promega, Cat.L4140). 25 ul freshly translated ARID1 protein was added to DNA-bound beads in the IP buffer plus complete protease inhibitor cocktail, and the mixture was rotated at 4&#176;C for 2 h. Beads were washed eight times with IP100 buffer, then proteins were stripped off the beads by boiling with 2XSDS buffer and then subjected to SDS-PAGE. The ARID1 protein bound by biotinylated DNA was detected by immunoblotting with a 1&#8758;200 dilution of anti-ARID1 (The peptide &#8220;SMVADEDAVDYSKT&#8221; was conjugated to KLH and used to raise rabbit polyclonal antibodies (GL Biochem)).</p><p>The pulldown assay was carried out as described previously <xref ref-type="bibr" rid="pgen.1004421-Li3">[42]</xref>. GST and GST-HDA8 were expressed in <italic>E. coli</italic> BL21. Cells were disrupted by sonication and the proteins were purified by glutathione Sepharose 4B affinity chromatography. 600 &#181;l of protein extract from inflorescences of ARID1-GFP plants was applied to the beads-protein mixture (30 &#181;g total protein) and incubated for 2 h at 4&#176;C on a rotating wheel. The beads were washed 5 times with IP lysis buffer. The bound (pellet) and unbound (supernatant) proteins were detected by immunoblotting with anti-GFP (Cat.632480, Clontech, 1&#8758;2000 dilution) and anti-Hsc70 (Cat.SPA-818, Stressgen, 1&#8758;10000 dilution) antibodies. 1/6 of the pellet fractions, and 1% and 0.1% of the supernatant fractions for anti-GFP and anti-Hsc70 IPs were used, respectively.</p><p>The immunoprecipitation assay was carried out as described <xref ref-type="bibr" rid="pgen.1004421-Li3">[42]</xref>. One gram of inflorescences from wild type or ARID1-Myc; HDA8-YFP doubly transgenic plants were ground in liquid nitrogen and homogenized in 2 ml of protein lysis buffer (50 mM Tris&#8211;HCl at pH 7.5, 150 mM NaCl, 0.2% NP-40, 2 mM DTT, 10% glycerol, complete protease inhibitor cocktail). The lysates were incubated at 4&#176;C for 50 min on a rotating wheel and centrifuged at 16000 g to pellet debris, and then supernatants were pre-cleared for 20 min with protein G agarose beads. Equivalent lysate was mixed with Myc antibodies (Cat. SC-70463, Santa Cruz, 1&#8758;200) pre-coupled to protein G agarose beads or to beads alone, respectively. After incubation for 2 h at 4&#176;C, the immune complexes were washed with lysis buffer. Proteins from 1/6 of the &#8220;No Ab&#8221; (no antibody) IP and 1/6 of the anti-myc IP were analyzed by immunoblotting using anti-Myc, anti-GFP, and anti-Hsc70. Proteins from 1/100 of the input were used for the anti-Myc and anti-GFP blots, while proteins from 1/1000 of the input were used for the anti-Hsc70 blot.</p><p>Inflorescences were fixed for 30 minutes in methanol&#8758;acetone (4&#8758;1, v/v) at room temperature, and anthers were dissected to release tricellular pollen onto a slide covered with liquid pollen germination medium <xref ref-type="bibr" rid="pgen.1004421-Boavida1">[43]</xref>. The slides were allowed to dry for about 20 min at room temperature and then were covered with a thin layer of agarose/gelatin/sucrose (0.94% low melting agarose/0.84% gelatin/0.3% w/v sucrose) for 10 minutes at 37&#176;C. The slides were soaked in blocking buffer (PBS with 5% BSA) for 1 hour at 37&#176;C, and then incubated with H3 antibody (1&#8758;50 dilution, Cat. 06-755, Upstate) or H3K9ac antibody (1&#8758;200 dilution, Cat. 1710241, Upstate), respectively, overnight at 4&#176;C in a dark moist chamber. After washed with blocking buffer, pollen was incubated in blocking buffer containing Alexa Fluor 488 goat anti-rabbit antibody (1&#8758;200 dilution, Cat.711-545-152, Jackson) for 6 h at room temperature. Slides were washed in PBS for five times, and observed with an Axiovert microscope under the GFP channel. Images were acquired using an AxioCamRM camera and AxioVision 4.8.1 software and processed using Adobe Photoshop CS2 (Adobe).</p>