<p>This work was carried according to ARAC guidelines and procedures as outlined in the Guide for the Care and Use of Laboratory Animals, U.S. Government Principles for the Utilization and Care of Vertebrate Animals Used in Testing, Research, and Training and Public Health Service Policy on Humane Care and Use of Laboratory Animals. This work was approved by the NIDDK Animal Care and Use Committee (ASP: K021-LMCB-12 and K021-LMCB-15).</p><p>Oligonucleotides were obtained from Integrated DNA technologies (IDT, Coralville IA) and are listed in <xref ref-type="table" rid="pgen.1006190.t002">Table 2</xref>. Purified human MutS&#945; was a kind gift of Drs Hsieh and Geng (NIDDK, NIH). Purified human MutS&#946; was a kind gift of Drs Yang and Li (NIDDK, NIH). This MutS&#946; complex contained a &#8220;trimmed&#8221; version of MSH3 containing amino acids 211&#8211;1125. This MutS&#946; complex has the same binding affinities for homoduplexes, tailed substrates and IDLs as complexes containing the full length MSH3 protein, as well as the same rate constants and ATPase activities [<xref ref-type="bibr" rid="pgen.1006190.ref030">30</xref>].</p><p>*This oligonucleotide was labeled at the 5&#8217; end with biotin during synthesis for use in EMSA reactions.</p><p>**N represents either A, G, C or T.</p><p>The generation of the FXD mice was described previously [<xref ref-type="bibr" rid="pgen.1006190.ref002">2</xref>]. These mice are on a C57BL/6 background. The <italic>Msh6</italic><sup><italic>+/-</italic></sup> mice were generated previously [<xref ref-type="bibr" rid="pgen.1006190.ref049">49</xref>, <xref ref-type="bibr" rid="pgen.1006190.ref053">53</xref>] and cryopreserved embryos were obtained from the NCI Mouse Repository (Frederick, MD). These mice are also on a predominantly C57BL/6 background. Live born pups were generated from these embryos by implantation into the oviduct of pseudopregnant recipients using standard procedures. F2 <italic>Msh6</italic><sup>+/-</sup> parents were bred to generate <italic>Msh6</italic><sup><italic>+/+</italic></sup>, <italic>Msh6</italic><sup><italic>+/-</italic></sup> and <italic>Msh6</italic><sup><italic>-/-</italic></sup> littermates. Multiple breeding pairs from the same parents were set up for each genotype. The litters for each genotype considered for this analysis had a similar parental age distribution. This was the same genetic background and breeding strategy that we had used previously to examine the effect of the loss of MSH3 on the expansion frequency [<xref ref-type="bibr" rid="pgen.1006190.ref014">14</xref>]. Mice were maintained in accordance with the guidelines of the NIDDK Animal Care and Use Committee and with the <italic>Guide for the Care and Use of Laboratory Animals</italic> (NIH publication no. 85&#8211;23, revised 1996).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Sperm was isolated from the cauda epididymis as previously described [<xref ref-type="bibr" rid="pgen.1006190.ref054">54</xref>], pelleted twice by centrifugation at 500 g for 5 min and the pellet resuspended first in PBS and then in 100 &#956;l of a solution containing a 90:10 mixture of ATL lysis buffer (Qiagen, Valencia, CA) and a 20 mg/ml proteinase K solution (Invitrogen, Carlsbad, CA). The samples were then incubated at 55&#176;C overnight before the addition of 30 &#956;l of 5 M NaCl. The resultant precipitate was pelleted by centrifugation and the supernatant transferred to a new tube and mixed with 130 &#956;l of ethanol. The DNA was then pelleted by centrifugation and dissolved in TE by incubation overnight at 55&#176;C. This protocol results in little, if any, contamination with somatic DNA [<xref ref-type="bibr" rid="pgen.1006190.ref054">54</xref>]. Genomic DNA from mouse tails was extracted using KAPA Mouse Genotyping Kit (KAPA Biosystems, Wilmington, MA). Genomic DNA from other tissues was extracted using a Maxwell16 Mouse tail DNA purification kit (Promega, Madison, WI) according to the manufacturer&#8217;s instructions. <italic>Msh6</italic> genotyping was carried out with Taq DNA polymerase in standard buffer with either the M010/M011 primer pair to detect the WT allele and M012/M013 to detect the mutant allele. The PCR parameters were 1x 94&#176;C for 1 min., 35x (94&#176;C for 1 min., 60&#176;C for 2 min. and 72&#176;C for 1 min), followed by 1x 72&#176;C for 3 min. The presence of the PM allele and its repeat number was determined using a fluorescent PCR assay and FraxM4 and FraxM5 primer pair as described previously [<xref ref-type="bibr" rid="pgen.1006190.ref003">3</xref>]. The somatic instability index (SII) was calculated from the GeneMapper profiles of DNA from different organs as previously described [<xref ref-type="bibr" rid="pgen.1006190.ref003">3</xref>, <xref ref-type="bibr" rid="pgen.1006190.ref022">22</xref>] and used to evaluate the extent of somatic expansion in adult mice. For small pool PCR analysis from sperm, the DNA was diluted to 3 pg/&#956;l (roughly 1 haploid genome equivalent/&#956;l). The diluted DNA was then subjected to nested PCR. The first round of PCR was carried out using the primers FraxC and FraxF in a 25 &#956;l PCR mix as described previously [<xref ref-type="bibr" rid="pgen.1006190.ref055">55</xref>]. One microliter of this PCR mix was used in second round of PCR with the FraxM4 and FraxM5 primers. Roughly 50% of the reactions contained a PCR product, consistent with the idea that each positive PCR likely represents the products of amplification of DNA from a single sperm cell. An exact Jonckheere-Terpstra test of trend in ordered counts was carried out using StatXact software (version 8; Cambridge, Massachusetts). Fisher&#8217;s exact test was carried out using the GraphPad QuickCalcs website (<ext-link ext-link-type="uri" ns0:href="http://www.graphpad.com/quickcalcs">http://www.graphpad.com/quickcalcs</ext-link>). The Mann-Whitney <italic>U</italic> test was carried out using VassarStats (<ext-link ext-link-type="uri" ns0:href="http://vassarstats.net/">http://vassarstats.net/</ext-link>). We set the significance level (&#945;) at 0.050 for the pairwise comparisons. For the comparisons of WT, heterozygous and homozygous null animals this corresponds to p = 0.015 after adjusting for multiple testing using the (relatively conservative) Bonferroni correction. Hartigans&#8217; dip test was calculated using the dip.test command in the R diptest library.</p><p>Total protein extracts were prepared from flash frozen brain, liver, testes and ovary of 6-month old mice. Tissues were homogenized using a tissue homogenizer (Precellys 24, Bertin Technologies, Berlin, Germany) with T-PER protein extraction reagent (Pierce Biotechnology, Inc, Rockford, IL) supplemented with complete, Mini, EDTA-free protease inhibitor cocktail (Roche Applied Science, Indianapolis, IN). Nuclear extracts of liver proteins were prepared using the NE-PER Nuclear and cytoplasmic extraction reagents (Pierce Biotechnology, Inc, Rockford, IL) according to the manufacturer&#8217;s instructions. The protein concentrations were determined using a Bio-Rad protein assay kit (Bio-Rad, Hercules, CA). Proteins were heated for 10 minutes at 70&#176;C in LDS-Sample Buffer (Life Technologies, Grand Island, NY), resolved by electrophoresis on either 3&#8211;8% NuPAGE Novex Tris-Acetate gels (Life Technologies) or 4&#8211;12% NuPAGE Novex Tris-Bis gels (Life Technologies) and transferred to nitrocellulose membranes using the iBlot transfer apparatus (Life Technologies) according to the manufacturer&#8217;s instructions. Membranes were blocked for one hour at room temperature in 5% ECL Prime blocking agent (GE Healthcare Bio-Sciences) in TBST, then incubated overnight at 4&#176;C with antibodies to MSH2 (ab70270, Abcam, Cambridge, MA) at a concentration of (1:10000), MSH3 (sc-271079, Santa Cruz, Dallas, TX) at a concentration of (1:1000) and MSH6 (BD 610918, BD Biosciences, Franklin Lakes, NJ) at a concentration of (1:1000). The secondary antibodies (anti-mouse IgG, NA931V and anti-rabbit IgG, NA934V, GE Healthcare Bio-Sciences) were both used at a dilution of 1:5000. After addition of the ECL Prime detection reagent (GE Healthcare Bio-Sciences), the blot was imaged using a Fluorchem M imaging system (Proteinsimple, Santa Clara, CA). Beta-actin (anti-mouse ab8227, Abcam, Cambridge, MA) was used as a loading control for total cell extracts and lamin B (ab16048, Abcam, Cambridge, MA) for nuclear extracts. A representative example of a full blot of testes protein extracts showing binding to MSH2, MSH3, MSH6 and the loading control &#946;-actin is shown in <xref ref-type="supplementary-material" rid="pgen.1006190.s001">S1 Fig</xref>. Western blots were repeated several times and always included molecular weight markers and extracts from the appropriate null mice as negative controls. To evaluate whether the loss of MSH6 affected the levels of MSH3 in <italic>Msh6</italic><sup><italic>-/-</italic></sup> mice, knowledge of the absolute levels of each protein is not necessary. Since the levels of MSH3 in each group of animals was tested with equivalent amounts of protein using the same antibody on the same gel, the avidity of the MSH3 antibody relative to the avidity of the MSH6 or MSH2 antibodies is not an issue. We thus were able to directly compare the levels of MSH3 in WT, <italic>Msh3</italic><sup><italic>-/-</italic></sup> and <italic>Msh6</italic><sup><italic>-/-</italic></sup> animals by determining the amount of each protein relative to &#946;-actin (total protein extracts) or lamin B1 (nuclear extracts) using the AlphaView software for FluorChem Systems (Proteinsimple, Santa Clara, CA). The levels of MSH2 and MSH6 in these animals were determined in the same way.</p><p>The oligonucleotides used in EMSA were prepared as described previously [<xref ref-type="bibr" rid="pgen.1006190.ref014">14</xref>]. The binding reactions were carried out using the Gelshift chemiluminescent EMSA kit (Active Motif, Carlsbad, CA) according to the manufacturer&#8217;s instructions using the indicated amounts of purified human MutS&#945; and human MutS&#946; and 2 fmoles of the duplexed oligonucleotides as described previously [<xref ref-type="bibr" rid="pgen.1006190.ref014">14</xref>].</p><p>The oligonucleotide used for thermal analysis consisted of a single strand of DNA comprised of 10 copies of CCG with the 5&#8217; end labeled with 5-carboxy-X-rhodamine (ROX) and the 3&#8217; end with IOWA Black RQ. The oligonucleotide was prepared as described previously [<xref ref-type="bibr" rid="pgen.1006190.ref014">14</xref>] and MutS&#945; or BSA was added to 360 nM as indicated. Thermal denaturation was monitored as described previously [<xref ref-type="bibr" rid="pgen.1006190.ref014">14</xref>]. The melting curve was consistent with a two-state model (<xref ref-type="supplementary-material" rid="pgen.1006190.s003">S3 Fig</xref>) and the thermodynamic parameters were thus derived from the melting curve using a two-state model (closed and open states).</p>