<p>LASSO and ENET penalized regression methods are applied to high-dimensional problems with a large number of parameters. The penalization produces a shrinkage of the regression coefficients towards zero given a sparse model reducing the irrelevant parameters. Both methods deal with highly correlated variables though in a different way. LASSO tends to select one variable from a group of correlated features whereas ENET selects the whole group of variables, when evidence for their relevance exists. The shrunk estimators introduce a bias while reducing the variance resulting in a better precision and accuracy model and, therefore, increasing its statistical power.</p><p>Consider the standard linear regression model where <italic>y</italic> = (<italic>y</italic>
<sub>1</sub>, &#8230;<italic>y</italic>
<sub><italic>n</italic></sub>)<sup><italic>t</italic></sup> is the response variable and <italic>x</italic> = (<italic>x</italic>
<sub>1<italic>j</italic></sub>, &#8230;<italic>x</italic>
<sub><italic>nj</italic></sub>)<sup><italic>t</italic></sup>
<italic>j</italic> = 1, &#8230;<italic>p</italic> are the standardized predictors, the LASSO solves the <italic>l</italic>
<sub><italic>1</italic></sub> penalized regression problem, the Ridge regression [<xref ref-type="bibr" rid="pgen.1005689.ref023">23</xref>] solves the <italic>l</italic>
<sub><italic>2</italic></sub> penalized regression problem and the ENET is the combination between the <italic>l</italic>
<sub><italic>1</italic></sub> and <italic>l</italic>
<sub><italic>2</italic></sub> penalized regression problem.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">For the LASSO and ENET estimates <inline-formula id="pgen.1005689.e001"><alternatives><graphic id="pgen.1005689.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e001.jpg" /><ns1:math id="M1"><ns1:mover accent="true"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>0</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover><ns1:mspace width="0.25em" /><ns1:mover accent="true"><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover><ns1:mo>=</ns1:mo><ns1:msup><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mrow><ns1:mover accent="true"><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo><ns1:mo>&#8230;</ns1:mo><ns1:mo>,</ns1:mo><ns1:msub><ns1:mrow><ns1:mover accent="true"><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mrow><ns1:mi>t</ns1:mi></ns1:mrow></ns1:msup></ns1:math></alternatives></inline-formula>; <inline-formula id="pgen.1005689.e002"><alternatives><graphic id="pgen.1005689.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e002.jpg" /><ns1:math id="M2"><ns1:mo>(</ns1:mo><ns1:mover accent="true"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>0</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover><ns1:mspace width="0.15em" /><ns1:mover accent="true"><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover><ns1:mo>)</ns1:mo></ns1:math></alternatives></inline-formula> are defined by
<disp-formula id="pgen.1005689.e003"><alternatives><graphic id="pgen.1005689.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e003.jpg" /><ns1:math id="M3"><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mspace width="0.15em" /><ns1:mover accent="true"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>0</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover><ns1:mover accent="true"><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mtext>arg</ns1:mtext><ns1:mspace width="0.1em" /><ns1:mi>m</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mrow><ns1:mo>{</ns1:mo><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>n</ns1:mi></ns1:munderover><ns1:msup><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mi>y</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mn>0</ns1:mn></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:munderover><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mi>j</ns1:mi></ns1:msub><ns1:msub><ns1:mi>x</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msup><ns1:mo>}</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
<italic>with the restrictions</italic>:
<disp-formula id="pgen.1005689.e004"><alternatives><graphic id="pgen.1005689.e004g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e004.jpg" /><ns1:math id="M4"><ns1:mrow><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>p</ns1:mi></ns1:munderover><ns1:mo stretchy="false">|</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mi>j</ns1:mi></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mo>&#8804;</ns1:mo><ns1:mi>t</ns1:mi><ns1:mspace width="0.25em" /><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>L</ns1:mi><ns1:mi>A</ns1:mi><ns1:mi>S</ns1:mi><ns1:mi>S</ns1:mi><ns1:mi>O</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mo>,</ns1:mo></ns1:mrow></ns1:math></alternatives><label>(1)</label></disp-formula>
<disp-formula id="pgen.1005689.e005"><alternatives><graphic id="pgen.1005689.e005g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e005.jpg" /><ns1:math id="M5"><ns1:mrow><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:munderover><ns1:mo stretchy="false">|</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mo>&#8804;</ns1:mo><ns1:mi>t</ns1:mi><ns1:mspace width="0.25em" /><ns1:mo>,</ns1:mo><ns1:mrow><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:munderover><ns1:mrow><ns1:msubsup><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>j</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:mrow></ns1:mrow><ns1:mo>&#8804;</ns1:mo><ns1:mi>t</ns1:mi><ns1:mspace width="0.5em" /><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>E</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>E</ns1:mi><ns1:mi>T</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mo>.</ns1:mo></ns1:mrow></ns1:math></alternatives><label>(2)</label></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Here, <italic>t</italic> &#8805; 0 is the tuning parameter that controls the amount of shrinkage that is applied to the estimates. For <inline-formula id="pgen.1005689.e006"><alternatives><graphic id="pgen.1005689.e006g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e006.jpg" /><ns1:math id="M6"><ns1:msubsup><ns1:mrow><ns1:mover accent="true"><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover></ns1:mrow><ns1:mrow><ns1:mi>j</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>0</ns1:mn></ns1:mrow></ns1:msubsup></ns1:math></alternatives></inline-formula> the un-penalized least squares estimate, <inline-formula id="pgen.1005689.e007"><alternatives><graphic id="pgen.1005689.e007g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e007.jpg" /><ns1:math id="M7"><ns1:mrow><ns1:msub><ns1:mi>t</ns1:mi><ns1:mn>0</ns1:mn></ns1:msub><ns1:mo>=</ns1:mo><ns1:mi>&#8721;</ns1:mi><ns1:mo>|</ns1:mo><ns1:msubsup><ns1:mrow><ns1:mover accent="true"><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover></ns1:mrow><ns1:mrow><ns1:mi>j</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>0</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo>|</ns1:mo></ns1:mrow></ns1:math></alternatives></inline-formula>. Values of <italic>t</italic> &lt; <italic>t</italic>
<sub>0</sub> will lead to shrinkage towards 0; some coefficients may be exactly equal to 0.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Using the Lagrangian form, this optimization problem is equivalent to (LASSO):
<disp-formula id="pgen.1005689.e008"><alternatives><graphic id="pgen.1005689.e008g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e008.jpg" /><ns1:math id="M8"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mover accent="true"><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover></ns1:mrow><ns1:mrow><ns1:mi>l</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>o</ns1:mi></ns1:mrow></ns1:msub><ns1:mspace width="0.1em" /><ns1:mo>=</ns1:mo><ns1:mspace width="0.1em" /><ns1:mi>a</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mrow><ns1:mo>{</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow></ns1:mfrac><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow></ns1:munderover><ns1:msup><ns1:mrow><ns1:mo stretchy="false">(</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>y</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>x</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mo stretchy="false">)</ns1:mo></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msup><ns1:mo>+</ns1:mo><ns1:mi>&#955;</ns1:mi><ns1:mspace width="0.2em" /><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:munderover><ns1:mo stretchy="false">|</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mi>j</ns1:mi></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mo>}</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives><label>(3)</label></disp-formula>
where &#955; is the penalty parameter related to t. To obtain the optimal penalty, k-fold cross validation (CV) was applied [<xref ref-type="bibr" rid="pgen.1005689.ref024">24</xref>] maximizing the penalized log-likelihood function.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">(ENET):
<disp-formula id="pgen.1005689.e009"><alternatives><graphic id="pgen.1005689.e009g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e009.jpg" /><ns1:math id="M9"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>t</ns1:mi></ns1:mrow></ns1:msub><ns1:mspace width="0.1em" /><ns1:mo>=</ns1:mo><ns1:mspace width="0.1em" /><ns1:mi>a</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mrow><ns1:mo>{</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow></ns1:mfrac><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow></ns1:munderover><ns1:msup><ns1:mrow><ns1:mo stretchy="false">(</ns1:mo><ns1:msub><ns1:mi>y</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mi>x</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mo stretchy="false">)</ns1:mo></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msup><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>&#955;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mspace width="0.15em" /><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:munderover><ns1:mo stretchy="false">|</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mi>j</ns1:mi></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>&#955;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub><ns1:mspace width="0.15em" /><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:munderover><ns1:msubsup><ns1:mi>&#946;</ns1:mi><ns1:mi>j</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>}</ns1:mo></ns1:mrow><ns1:mo>,</ns1:mo></ns1:mrow></ns1:math></alternatives><label>(4)</label></disp-formula>
where &#955;<sub>1</sub>, &#955;<sub>2</sub> are the penalty parameters related to t. In this sense, ENET can be viewed as a penalized least squares method. With <italic>&#945;</italic> = <italic>&#955;</italic>
<sub>2</sub>/(<italic>&#955;</italic>
<sub>1</sub> + <italic>&#955;</italic>
<sub>2</sub>), solving <inline-formula id="pgen.1005689.e010"><alternatives><graphic id="pgen.1005689.e010g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e010.jpg" /><ns1:math id="M10"><ns1:msub><ns1:mrow><ns1:mover accent="true"><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>t</ns1:mi></ns1:mrow></ns1:msub></ns1:math></alternatives></inline-formula> in Eq (<xref ref-type="disp-formula" rid="pgen.1005689.e009">4</xref>) is equivalent to the following optimization problem:
<disp-formula id="pgen.1005689.e011"><alternatives><graphic id="pgen.1005689.e011g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e011.jpg" /><ns1:math id="M11"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mover accent="true"><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mo>^</ns1:mo></ns1:mover></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>t</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mi>a</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mrow><ns1:mo>{</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow></ns1:mfrac><ns1:mrow><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow></ns1:munderover><ns1:mrow><ns1:msup><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>y</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>x</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msup><ns1:mo>+</ns1:mo><ns1:mo stretchy="false">(</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:mi>&#945;</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mspace width="0.15em" /><ns1:mrow><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:munderover><ns1:mrow><ns1:mo stretchy="false">|</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mo stretchy="false">|</ns1:mo></ns1:mrow></ns1:mrow></ns1:mrow></ns1:mrow><ns1:mo>+</ns1:mo><ns1:mi>&#945;</ns1:mi><ns1:mspace width="0.15em" /><ns1:mrow><ns1:munderover><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:munderover><ns1:mrow><ns1:msubsup><ns1:mrow><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>j</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:mrow></ns1:mrow><ns1:mo>}</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives><label>(5)</label></disp-formula>
</p><p>This expression involves a convex combination of the LASSO and ridge penalty. When <italic>&#945;</italic> = 1 the ENET becomes ridge regression and when <italic>&#945;</italic> =0 the ENET becomes LASSO. To obtain the optimal penalty (&#955;), k-fold CV selecting the best <italic>&#945;</italic> was applied. This value was obtained using a vector of <italic>&#945;</italic>
<italic>&#1013;</italic>(0.01, 0.99) <italic>by</italic> 0.01.</p><p>The LASSO and ENET methods described above were applied to our data with the R package glmnet, that relies on cyclical coordinate descent, computed along a regularization path [<xref ref-type="bibr" rid="pgen.1005689.ref025">25</xref>]. To avoid small sample size limitations in variable selection while not introducing an important bias k = 5 was used in the k-fold CV.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">These methods are promising in the context of high-throughput data but one of their drawbacks is that they do not provide p-values to assess statistical significance of relationships, nor give a formal assessment of the overall goodness-of-fit. Therefore, a permutation based strategy was adopted to assess significance of discovered relationships combined with a MT correction approach (MaxT algorithm [<xref ref-type="bibr" rid="pgen.1005689.ref022">22</xref>]) building upon the statistical concept of deviance. The deviance is used to compare two models and in this case we defined it as
<disp-formula id="pgen.1005689.e012"><alternatives><graphic id="pgen.1005689.e012g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e012.jpg" /><ns1:math id="M12"><ns1:mrow><ns1:mi>D</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>e</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>2</ns1:mn><ns1:mo stretchy="false">[</ns1:mo><ns1:mi>l</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>k</ns1:mi><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>f</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:msub><ns1:mrow><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>m</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>d</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow></ns1:msub><ns1:mo stretchy="false">)</ns1:mo><ns1:mo>&#8722;</ns1:mo><ns1:mi>l</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>k</ns1:mi><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>l</ns1:mi><ns1:msub><ns1:mrow><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>m</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>d</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi></ns1:mrow></ns1:msub><ns1:mo stretchy="false">)</ns1:mo><ns1:mo stretchy="false">]</ns1:mo><ns1:mo>.</ns1:mo></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p>Here <italic>loglik</italic> is the loglikelihood function, <italic>full</italic>
<sub><italic>model</italic></sub> refers to the model with the parameters selected by LASSO or ENET, and <italic>null</italic>
<sub><italic>model</italic></sub> is the model with only the intercept estimated. Thus, the interpretation would be, the higher the deviance the better the model.</p><p>MaxT algorithm of Westfall &amp; Young [<xref ref-type="bibr" rid="pgen.1005689.ref022">22</xref>] is a step-down FWER-controlling MT procedure. The method uses the raw p-values or directly the statistics as explained in [<xref ref-type="bibr" rid="pgen.1005689.ref026">26</xref>]. Using this approach, the permutation needed to obtain the p-values was combined with the one needed to apply the MaxT algorithm saving computational time. In this work, we used the deviance obtained per each of the permuted LASSO/ENET model to compute the MaxT algorithm and individuals within gene expression measure were permuted, that is the dependent variable in the models. The algorithm is explained in <xref ref-type="boxed-text" rid="pgen.1005689.box001">Box 1</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">From the original data, order the deviance obtained per each observed statistics:
<disp-formula id="pgen.1005689.e013"><alternatives><graphic id="pgen.1005689.e013g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e013.jpg" /><ns1:math id="M13"><ns1:mrow><ns1:mo stretchy="false">|</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>D</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mo>&#8805;</ns1:mo><ns1:mo stretchy="false">|</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>D</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mo>&#8805;</ns1:mo><ns1:mo stretchy="false">|</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>D</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mn>3</ns1:mn></ns1:mrow></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mo>&#8805;</ns1:mo><ns1:mo>&#8943;</ns1:mo><ns1:mo>&#8805;</ns1:mo><ns1:mo stretchy="false">|</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>D</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mi>m</ns1:mi></ns1:mrow></ns1:msub><ns1:mo stretchy="false">|</ns1:mo><ns1:mo>.</ns1:mo></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p>For the bth permutation, b = 1&#8230;B</p><p>Permute the n individuals of each of the vectors <italic>Y</italic>
<sub><italic>m</italic></sub> = (<italic>y</italic>
<sub>1</sub>, &#8230; <italic>y</italic>
<sub><italic>n</italic></sub>)<sub><italic>m</italic></sub>
</p><p>Compute the statistics <italic>D</italic>
<sub>1<italic>b</italic>,&#8230;</sub>
<italic>D</italic>
<sub><italic>mb</italic></sub>
</p><p>Compute the <italic>U</italic>
<sub><italic>i</italic>,<italic>b</italic></sub> = max<sub><italic>l</italic> = <italic>i</italic>&#8230;<italic>m</italic></sub>|D<sub><italic>sl</italic>,<italic>b</italic></sub>|, the successive step-down procedure is: <italic>U</italic>
<sub><italic>m</italic>,<italic>b</italic></sub> = |D<sub><italic>sm</italic>,<italic>b</italic></sub>|</p><p>&#8230;</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">
<disp-formula id="pgen.1005689.e014"><alternatives><graphic id="pgen.1005689.e014g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e014.jpg" /><ns1:math id="M14"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>U</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mo>,</ns1:mo><ns1:mi>b</ns1:mi></ns1:mrow></ns1:msub><ns1:mspace width="0.1em" /><ns1:mo>=</ns1:mo><ns1:mspace width="0.1em" /><ns1:mtext>max</ns1:mtext><ns1:mrow><ns1:mo>|</ns1:mo><ns1:msub><ns1:mrow><ns1:mtext>D</ns1:mtext></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mn>2</ns1:mn><ns1:mo>,</ns1:mo><ns1:mi>b</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo><ns1:mspace width="0.15em" /><ns1:msub><ns1:mrow><ns1:mtext>D</ns1:mtext></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mn>3</ns1:mn><ns1:mo>,</ns1:mo><ns1:mi>b</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo><ns1:mspace width="0.1em" /><ns1:mo>&#8230;</ns1:mo><ns1:mspace width="0.1em" /><ns1:mo>,</ns1:mo><ns1:msub><ns1:mrow><ns1:mtext>D</ns1:mtext></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mi>m</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>b</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>|</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
<disp-formula id="pgen.1005689.e015"><alternatives><graphic id="pgen.1005689.e015g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e015.jpg" /><ns1:math id="M15"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>U</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mi>b</ns1:mi></ns1:mrow></ns1:msub><ns1:mspace width="0.1em" /><ns1:mo>=</ns1:mo><ns1:mspace width="0.1em" /><ns1:mtext>max</ns1:mtext><ns1:mrow><ns1:mo>|</ns1:mo><ns1:msub><ns1:mrow><ns1:mtext>D</ns1:mtext></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mi>b</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo><ns1:mspace width="0.15em" /><ns1:msub><ns1:mrow><ns1:mtext>D</ns1:mtext></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mn>2</ns1:mn><ns1:mo>,</ns1:mo><ns1:mi>b</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo><ns1:mspace width="0.15em" /><ns1:msub><ns1:mrow><ns1:mtext>D</ns1:mtext></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mn>3</ns1:mn><ns1:mo>,</ns1:mo><ns1:mi>b</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo><ns1:mspace width="0.1em" /><ns1:mo>&#8230;</ns1:mo><ns1:mspace width="0.1em" /><ns1:mo>,</ns1:mo><ns1:msub><ns1:mrow><ns1:mtext>D</ns1:mtext></ns1:mrow><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mi>m</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>b</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>|</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p>The steps are repeated B times and the adjusted p-values are estimated by:</p><p>70 patients with a histologically confirmed UBC were recruited in 2 hospitals during 1997&#8211;1998 as part of the pilot phase of the Spanish Bladder Cancer/EPICURO Study. According to established criteria based on tumor stage and grade for UBC, the tumors were classified as low-grade non-muscle invasive, high-grade non-muscle invasive, and muscle invasive. Three sets of <italic>omics</italic> data were obtained using fresh tumor tissue, including common genetic variation (GSE51641), DNA methylation (GSE71666), and gene expression (GSE71576). The three <italic>omics</italic> data overlapped in 27 individuals that are included in this study and comprise 44% low-grade non-muscle invasive tumors, 30% high-grade non-muscle invasive tumors and 26% muscle invasive tumors. <xref ref-type="supplementary-material" rid="pgen.1005689.s008">S1 Table</xref> shows the IDs of the 27 samples used in the following analysis. The local ethics committee of the participating centers approved the study and written informed consent was obtained from all participants at the time of recruitment.</p><p>Genotyping of tumor samples was performed using Illumina HumanHap 1M array. A total of 1,047,101 SNPs were determined in 46 individuals and, after the standard quality control and filter the SNPs that were in perfect LD (<italic>r</italic>
<sup>2</sup> = 1), they resulted in 567,513 SNPs. The application of multivariable models required no missing values, so genotypes were imputed with BEAGLE 3.0 method [<xref ref-type="bibr" rid="pgen.1005689.ref027">27</xref>]. CpG methylation data was generated using the Infinium Human Methylation 27 BeadChip Kit. At each CpG site, the methylation levels were measured with M-values using the log2 transformation of the &#946;-values since they are more statistically valid due to a better approximation of the homoscedasticity. The initial number of CpGs in the studied array was 27,578 and after background normalization and QC, a total number of 23,034 CpGs were left for analysis. Gene expression data were obtained from 44 tumor samples using the Affymetrix DNA Microarray Human Gene 1.0 ST Array with 32,321 probes. After the application of QC, it resulted in 20,899 probes determined in 37 individuals. Further details about the preprocessing of the data and the quality control applied can be found elsewhere [<xref ref-type="bibr" rid="pgen.1005689.ref012">12</xref>]. The three measures were annotated using the UCSC hg19, NCBI build 37 to make them comparable and homogenize their position in the genome.</p><p>To generate a simulation sample, the association between SNPs and/or CpGs with gene expression was broken and therefore no significant results should be observed. To do that, 10-gene expression probes were randomly selected from our discovery sample showing no correlation structure between the probes and following a multivariate normal distribution. Then, the mean (&#956; = 8.4) and variance (&#963;<sup>2</sup> = 0.4) of all the probes together were obtained. Finally, a simulated set of gene expression probes was generated using the normal distribution obtained and considering the same sample size of the discovery phase (<italic>p</italic> = 20,899 probes and <italic>N</italic> = 27 individuals).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">UBC tumor data were obtained from The Cancer Genome Atlas (TCGA) consortium (<ext-link ext-link-type="uri" ns0:href="https://tcga-data.nci.nih.gov/tcga/">https://tcga-data.nci.nih.gov/tcga/</ext-link>) to replicate our findings. Data was downloaded and processed with the TCGA-Assembler [<xref ref-type="bibr" rid="pgen.1005689.ref028">28</xref>]. The study included only individuals with muscle invasive UBC and the tumors were profiled with genome wide 6.0 Affymetrix, RNASeqV2, and HumanMethylation450K Illumina arrays yielding data for 20,502 gene expression probes, 905,422 SNPs, and 350,271 CpGs. The total number of individuals with overlapping data from the three platforms was 238 and they were used in the replication phase of this contribution. <xref ref-type="supplementary-material" rid="pgen.1005689.s009">S2 Table</xref> shows the IDs corresponding to these 238 samples.</p><p>Penalized regression methods LASSO and ENET were applied to the discovery data in combination with the proposed permutation-based MaxT method to select the SNPs and/or CpGs associated with gene expression levels in the following multivariable models:</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">
<italic><underline>SNP model:</underline></italic>
<disp-formula id="pgen.1005689.e017"><alternatives><graphic id="pgen.1005689.e017g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e017.jpg" /><ns1:math id="M17"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>G</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>E</ns1:mi><ns1:mi>x</ns1:mi><ns1:mi>p</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>l</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>s</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#945;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>P</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#945;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>P</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:mo>&#8943;</ns1:mo><ns1:mo>+</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#945;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>P</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>;</ns1:mo><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn><ns1:mspace width="0.1em" /><ns1:mo>&#8230;</ns1:mo><ns1:mspace width="0.1em" /><ns1:mi>m</ns1:mi></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">
<italic><underline>CPG model:</underline></italic>
<disp-formula id="pgen.1005689.e018"><alternatives><graphic id="pgen.1005689.e018g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e018.jpg" /><ns1:math id="M18"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>G</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>E</ns1:mi><ns1:mi>x</ns1:mi><ns1:mi>p</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>l</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>s</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#947;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>C</ns1:mi><ns1:mi>P</ns1:mi><ns1:mi>G</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#947;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>C</ns1:mi><ns1:mi>P</ns1:mi><ns1:mi>G</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:mo>&#8943;</ns1:mo><ns1:mo>+</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#947;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>C</ns1:mi><ns1:mi>P</ns1:mi><ns1:mi>G</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>;</ns1:mo><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn><ns1:mspace width="0.1em" /><ns1:mo>&#8230;</ns1:mo><ns1:mspace width="0.1em" /><ns1:mi>m</ns1:mi></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">
<italic><underline>Global model = SNP + CPG model:</underline></italic>
<disp-formula id="pgen.1005689.e019"><alternatives><graphic id="pgen.1005689.e019g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005689.e019.jpg" /><ns1:math id="M19"><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>G</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>E</ns1:mi><ns1:mi>x</ns1:mi><ns1:mi>p</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>l</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>s</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#945;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>P</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:mo>&#8943;</ns1:mo><ns1:mo>+</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#945;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mi>N</ns1:mi><ns1:mi>P</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#947;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>C</ns1:mi><ns1:mi>P</ns1:mi><ns1:mi>G</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:mo>&#8943;</ns1:mo><ns1:mo>+</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>&#947;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>C</ns1:mi><ns1:mi>P</ns1:mi><ns1:mi>G</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>;</ns1:mo><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn><ns1:mspace width="0.1em" /><ns1:mo>&#8230;</ns1:mo><ns1:mspace width="0.1em" /><ns1:mi>m</ns1:mi></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p>To apply this integrative idea to our set of data the following steps were performed: (1) SNPs and CpGs that were in a 1MB window upstream and downstream were selected from each probe in the gene expression array; (2) LASSO and ENET were applied to each probe and model (SNP, CpG, and Global models) obtaining the deviance per model; and (3), the permutation-based MaxT method was applied to obtain the adjusted p-values (B = 100 permutations and significant adjusted p-value &lt; 0.1). The scenario and workflow is represented in <xref ref-type="fig" rid="pgen.1005689.g001">Fig 1</xref>.</p><p>The integrative framework proposed is based on three steps. Step 1 corresponds to the selection of SNPs and CpGs in 1MB window upstream and downstream from each probe in the gene expression array. Step 2 corresponds to the application of LASSO and ENET to each probe obtaining the deviance per probe. Step 3 corresponds to the permutation-based MaxT method application where gene expression levels within the individuals are permuted B = 100 times obtaining the deviance per probe.</p><p>Subsequently, this analysis flow was applied to the simulated data set using the same criteria. In the replication scenario, we aimed at determining whether the genes that were significant in the discovery phase were also significant in the replication dataset. Therefore, the analysis was restricted to the genes found to be significant in the discovery phase considering all models (SNP, CPG and/or Global) and methods (LASSO and/or ENET). Following the pipeline shown in <xref ref-type="fig" rid="pgen.1005689.g001">Fig 1</xref>, we focused on the significant genes found in the discovery phase and SNPs and CpGs were selected in 1MB window from the TCGA database, even if the SNPs and CpGs were not the same as those analyzed in the discovery phase. Second, LASSO and/or ENET were conducted to SNP, CPG, and/or Global models. Finally, the permutation-based MaxT method was applied to obtain significance and correct for multiple testing. The replication analysis was performed with the same software and criteria as in the discovery analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To provide a biological interpretation to the results, the entire list of the significant genes identified in the discovery phase by both LASSO and ENET, and by the three models, was used to perform a gene enrichment analysis with the bioinformatics tool DAVID [<xref ref-type="bibr" rid="pgen.1005689.ref029">29</xref>,<xref ref-type="bibr" rid="pgen.1005689.ref030">30</xref>]. The functional annotation clustering analysis module offered by DAVID was used. The gene term annotation is based on 14 annotation categories (Gene Ontology (GO), Biological process, GO Molecular Function, GO Cellular Component, KEGG Pathways, BioCarta Pathways, Swiss-Prot Keywords, BBID Pathways, SMART Domains, NIH Genetics Association DB, UniProt Sequence Features, COG/KOG Ontology, NCBI OMIM, InterPro Domains, and PIR Super-Family Names) collected in the DAVID tool knowledgebase (<ext-link ext-link-type="uri" ns0:href="https://david.ncifcrf.gov/knowledgebase/DAVID_knowledgebase.html">https://david.ncifcrf.gov/knowledgebase/DAVID_knowledgebase.html</ext-link>). The method identifies related genes by measuring the similarity of their global annotation profiles. So, the &#8220;grouping term&#8221; is based on the idea that two genes that have similar annotation profiles are functionally related. Each group term provides an enrichment score (ES) that indicates biological significance when &#8805;1.3 (equivalent to non-log scale 0.05). DAVID also provides a p-value to examine the significance of gene-term enrichment, which is corrected by Benjamini MT [<xref ref-type="bibr" rid="pgen.1005689.ref031">31</xref>].</p>