<p>MCF-7 cells were maintained in DMEM supplemented with 10% calf serum. The stable PAD2-depleted MCF-7 cell line was described previously <xref ref-type="bibr" rid="pgen.1004613-Zhang1">[12]</xref> and maintained in the medium containing 1 &#181;g/ml puromycin (Sigma). Before hormones treatment, the cells were cultured for 3 days in DMEM phenol red-free medium supplemented with 10% charcoal-dextran-treated calf serum.</p><p>ChIP experiments were performed as described previously <xref ref-type="bibr" rid="pgen.1004613-Zhang1">[12]</xref>, <xref ref-type="bibr" rid="pgen.1004613-Guertin2">[33]</xref>. Estrogen-starved MCF-7 cells were subjected to E2 treatment at 100 nM for 0, 2, 5, 10, 40 and 160 min, and followed by crosslinking with a final concentration of 2% paraformaldhyde for one minute at 37&#176;C. Crosslinking was quenched in 125 mM glycine on ice for 5 min. Cell lysates were sonicated under conditions yielding fragments ranging from 100 bp to 200 bp. The material was clarified by centrifugation, diluted 10-fold in dilution buffer, and pre-cleared with protein A-agarose beads. The pre-cleared, chromatin-containing supernatant was used in immunoprecipitation reactions with antibodies against H3Cit26 (Abcam, ab19847, lot 135757), ER&#945; (Santa Cruz, sc-542), non-specific rabbit IgG (Upstate 12-370) or without antibody as a control. ChIP-Western analysis confirmed that the H3R26Cit antibody does not recognize ER (<xref ref-type="supplementary-material" rid="pgen.1004613.s015">Figure S14</xref> and <xref ref-type="supplementary-material" rid="pgen.1004613.s019">Text S1</xref>). The Illumina library preparation was as previously described <xref ref-type="bibr" rid="pgen.1004613-Guertin3">[34]</xref>. Samples were submitted to the Cornell DNA Sequencing and Genotyping Lab and run on the Illumina Genome Analyzer II. Replicates were found to be concordant, with Pearson correlation coefficients all greater than 0.9, for all E2-stimuated data sets where replicates were performed (<xref ref-type="supplementary-material" rid="pgen.1004613.s016">Figure S15</xref>).</p><p>Estrogen-starved MCF-7 cells were treated with ethanol or 100 nM E2 for 10 min. Mononucleosomes were prepared as described previously <xref ref-type="bibr" rid="pgen.1004613-Zhang1">[12]</xref>. The crude chromatin was solubilized with a concentration of MNase (NEB M0247) that produced &#8764;80% mononucleosomes. The mononucleosomes were then immunoprecipitated using anti-H3Cit26 and anti-H3K27ac (Abcam ab4729) antibodies. Advanced Technology Center (Rockville, MD, USA) performed the Illumina library preparation and the paired-end sequencing.</p><p>The primer sequences used for ChIP-qPCR were summarized in <xref ref-type="supplementary-material" rid="pgen.1004613.s018">Table S1</xref>.</p><p>Raw sequence reads were aligned to the hg19 genome using bowtie <xref ref-type="bibr" rid="pgen.1004613-Langmead1">[35]</xref>. Replicate concordance was confirmed and replicate files were merged to call peaks using MACS <xref ref-type="bibr" rid="pgen.1004613-Zhang2">[36]</xref> and a mock IP using IgG as a background data set. Broad regions of enrichment were further subdivided using the subpeaks argument in MACS <xref ref-type="bibr" rid="pgen.1004613-Zhang2">[36]</xref>. Subpeaks files from all time points were combined for each antibody. Redundant peaks (those found in more than one time point) were filtered out if a 100 bp window centered on the subpeak summit was within 30 bp of the adjacent subpeak&#8212;the subpeak with the most tag counts a the summit position was retained. Raw tag count intensity for each peak coordinate was normalized for each time point and peaks with at least one time point with greater intensity than an independent no antibody ChIP were retained (Dataset S1). We generated composite profile plots by taking the average intensity for a given factor in 20 base pairs steps centered on the peak summits.</p><p>We used MEME with default parameters and 100 bp of sequence information surrounding the peak summit to identify motifs de novo <xref ref-type="bibr" rid="pgen.1004613-Bailey1">[37]</xref>. Despite the fact that many H3R26Cit peaks do not overlap with any ER peaks, the canonical ERE is the most significant motif found de novo at these H3R26Cit peak summits (<xref ref-type="supplementary-material" rid="pgen.1004613.s017">Figure S16</xref>).</p><p>DNase data was downloaded from ENCODE (wgEncodeUwDnaseMCF-7Estctrl0hAlnRep*.bam and wgEncodeUwDnaseMCF-7Est100nm1hAlnRep*.bam).</p><p>We used the program DEseq <xref ref-type="bibr" rid="pgen.1004613-Anders1">[19]</xref> to statistically determine differentially DNase sensitivity at all H3R26Cit peaks using the raw number of reads at each peak before and after estrogen treatment. We applied an independent gene filter to the peaks in order to increase the power to detect differences and still control for multiple testing (<xref ref-type="supplementary-material" rid="pgen.1004613.s009">Figure S8</xref>). The joint distribution of p-values and read count total remains unchanged after filtering, indicating that unadjusted p-value and total reads counts for each peak are independent variables (<xref ref-type="supplementary-material" rid="pgen.1004613.s010">Figure S9</xref>).</p><p>Paired-end reads were mapped to the hg19 genome using bowtie2 <xref ref-type="bibr" rid="pgen.1004613-Langmead1">[35]</xref>. Concordant paired-end reads that result in fragment lengths &gt;1 and &lt;500 were considered in the analysis and shown in <xref ref-type="fig" rid="pgen-1004613-g006">Figure 6</xref>.</p><p>MCF-7 Cells grown on slides were subjected to hormones treatment at 100 nM for 45 min, including Estradiol (E2), Dihydrotestosterone (DHT), Progesterone (PRO), and Dexamethasone (Dex). Ethanol (EtOH) was used as a control. Confocal microscopy experiments were described previously <xref ref-type="bibr" rid="pgen.1004613-Wang1">[11]</xref>. In short, after hormones treatment, cells were fixed with a paraformaldehyde fixing solution (1&#215;PBS, 0.1% Triton X- 100, 0.2% NP-40, and 3.7% paraformaldehyde) for 10 min at room temperature. After 3 washes (10 min each) with PBST (1&#215;PBS with 0.2% Triton X-100), cells were blocked with 5% BSA in PBST for at least 1 hr at room temperature. The cells were stained overnight at 4&#176;C with anti-H3Cit26 (Abcam, ab19847) antibody diluted in PBST (1&#8758;100). DNA was stained with DAPI (4,6- diamidino-2-phenylindole) before mounting. Images were collected with LSM 510 laser scanning confocal microscope (Carl Zeiss).</p><p>PAD2 expression levels were measured by provisional RNA-seq (<xref ref-type="supplementary-material" rid="pgen.1004613.s014">Figure S13</xref>) data from The Cancer Genome Atlas (TCGA) <xref ref-type="bibr" rid="pgen.1004613-Cancer1">[25]</xref>, which was curated by UCSC <xref ref-type="bibr" rid="pgen.1004613-Cline1">[26]</xref>, and published TCGA microarray data (<xref ref-type="fig" rid="pgen-1004613-g007">Figure 7A,B</xref>) <xref ref-type="bibr" rid="pgen.1004613-Cancer1">[25]</xref>. P-values were automatically selected for the best expression threshold cut-off and calculated using the log-rank test statistic. In <xref ref-type="fig" rid="pgen-1004613-g007">Figure 7E and 7F</xref>, patient samples were split into two groups according to level of PADI2 expression by above and below the 45% quantiles. In <xref ref-type="supplementary-material" rid="pgen.1004613.s014">Figure S13A</xref>,B patient samples were split into two groups according to level of PADI2 expression by above and below the 65% and 60% quantiles, respectively.</p><p>Four sections from each breast cancer patient were deparaffinized, unmasked, blocked and treated with either rabbit anti-PADI 2 (Proteintech, 12110-1-ap), rabbit anti-ER-&#945; (Santa Cruz, sc-542), rabbit anti-Histone 3 Citrulline 26 (AbCam, ab19847) or non-specific rabbit IgG (Millipore, S-20) using standard methods and developed using the Vectastain Peroxidase Rabbit IgG and Vector DAB substrate kits. Sections stained for ER were examined using a Zeiss AxioObserver microscope and two tumor cell areas on each section which stained positive were chosen and imaged at 5, 10 and 20&#215; using a Zeiss Axiophot color camera. These same two areas were imaged on the corresponding sections stained for PAD2, H3R26Cit and the IgG negative control. For patients who were negative for ER in all areas of the section, two areas containing tumor cells were chosen for imaging and corresponding images from the remaining slides were captured as above.</p><p>In each image the brown DAB staining was scored from 0 to 5, with 0 being no staining and 5 being heavy staining in both nucleus and cytoplasm. The percent of tumor cells in the 10&#215; image at each score level was established and multiplying the stain scores by their corresponding fraction and adding the products together determined a final score for that section area. The following comparisons were made using all of the final scores from both areas of all sections, ER:PADI2, ER:H3R26Cit, PADI2: H3R26Cit.</p><p>H3R26Cit and H3K27ac ChIP-seq data was deposited into GEO with the accession number GSE58177. ER ChIP-seq data was previously published <xref ref-type="bibr" rid="pgen.1004613-Guertin4">[38]</xref> with accession number GSE54855.</p>