<p>The mouse lines employed in this study, <italic>Mitf<sup>&#916;D</sup></italic>
<xref ref-type="bibr" rid="pgen.1004360-Bharti3">[37]</xref>, <italic>Pax6<sup>loxP</sup></italic>
<xref ref-type="bibr" rid="pgen.1004360-AsheryPadan1">[40]</xref> and <italic>DctCre</italic>
<xref ref-type="bibr" rid="pgen.1004360-Davis1">[5]</xref> have been previously described. The latter two were used to establish <italic>Pax6<sup>loxP/loxP</sup>;DctCre</italic> somatic mutants. <italic>Pax6<sup>loxP/loxP</sup></italic> littermates were used as controls. The genetic background of all mice used in this study was C57BL/6J, except for <italic>in situ</italic> staining, for which mice of the outbred ICR genetic background were used. All animal work was conducted according to national and international guidelines and approved by the Tel Aviv University review board.</p><p>All data were examined using two-tailed Student's t-test.</p><p>Immunofluorescence analysis was performed on 10 &#181;m paraffin sections as previously described <xref ref-type="bibr" rid="pgen.1004360-AsheryPadan1">[40]</xref>, using the following primary antibodies: rabbit anti-PAX6 (1&#8758;400, Covance, prb-278b), mouse anti-PAX6 (1&#8758;25, Santa Cruz, sc-32766), rabbit anti-SOX9 (1&#8758;200, Chemicon, ab5535), sheep anti-CHX10 (1&#8758;1000, Exalpha, X1180P), rabbit anti-OTX2 (1&#8758;50, Millipore, AB9566), rabbit anti-MITF <xref ref-type="bibr" rid="pgen.1004360-Bharti2">[18]</xref>, rabbit anti-TYR (1&#8758;1,000, a gift from the Vincent Hearing lab, NCI), rabbit anti-TYRP1 (1&#8758;1,000, a gift from the Vincent Hearing lab, NCI). Secondary antibodies were donkey anti-rabbit conjugated to alexa594 (1&#8758;1000, Invitrogen, A21207) and alexa488 donkey anti mouse/sheep (1&#8758;1000, Invitrogen, A21202/A11015).</p><p>
<italic>In situ</italic> hybridization (ISH) was performed on 14 &#181;m cryo-sections using DIG-labeled RNA probes as previously described <xref ref-type="bibr" rid="pgen.1004360-Yaron1">[78]</xref>. The <italic>Pax6</italic> intron 7 probe was generated from a 849bp PCR fragment (forward, <named-content content-type="gene">5&#8242;-TTTGGAGCCCTCCATCTTTCTC-3&#8242;</named-content>; reverse, <named-content content-type="gene">5&#8242;- TGCACACTTTCGGGCAAGG-3&#8242;</named-content>). Plasmid for antisense transcription of <italic>silver</italic> was kindly provided by the laboratory of Dr. William Pavan (NIH) <xref ref-type="bibr" rid="pgen.1004360-Baxter1">[79]</xref>.</p><p>Flat-mount samples were prepared as follows: Eyes at E19.5 were enucleated and immediately fixed in 4% paraformaldehyde for 30 minutes. The RPE was carefully dissected from the rest of eye structures, sliced radially to four pieces and flattened on membrane filters (Schleicher&amp; Schull, 0.45 &#181;m D-37582). Samples were blocked and stained with phalloidin (1&#8758;100, Invitrogen, A12379). Thereafter, RPE was flattened on its basal side on a slide and sealed for observation.</p><p>The heads of E15.5 embryos and perforated eyes of P1 neonatal mice were fixed in 0.1 M cacodylate-buffered fixative containing 2.5% paraformaldehyde and 2% glutaraldehyde and further processed as described previously <xref ref-type="bibr" rid="pgen.1004360-Baba1">[80]</xref>. Ultrathin sections were cut with a Leica Ultramicrotome UCT (Leica Microsystems), stained with uranyl acetate and lead citrate and analyzed with a H7600 transmission electron microscope (Hitachi).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Exact timed matings were performed by overnight cohabitation of an inbred <italic>Pax6<sup>loxP/loxP</sup>;DctCre</italic> male with <italic>Pax6<sup>loxP/loxP</sup></italic> females. Pregnant females were harvested on day E15.5 and the RPE of the embryos was dissected as previously described <xref ref-type="bibr" rid="pgen.1004360-Bharti2">[18]</xref>. RPEs were pooled into two separate tubes according to their pigmentation intensity, and tubes were stored at &#8722;80&#176;C. Tail cuts of the embryos were collected for genotype verification. Each tube was considered as one biological repeat. RNA was extracted using the QIAshredder and the RNeasy kits (QIAGEN). RNA isolated from three control and three mutant samples was processed for microarray analysis using the Affymetrix GeneChip 1.0ST as described previously <xref ref-type="bibr" rid="pgen.1004360-Shaham3">[81]</xref>. Differentially expressed genes with p-values lower than 0.05 and with a fold-change cutoff of 1.5 are listed in <xref ref-type="supplementary-material" rid="pgen.1004360.s008">Table S1</xref>. The expression data were submitted to the NCBI Gene Expression Omnibus (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo">http://www.ncbi.nlm.nih.gov/geo</ext-link>) under series accession no. GSE56548.</p><p>Reverse transcription of 1 &#181;g of RNA from each sample was performed using the SuperScript III First Strand kit (Invitrogen). cDNA was amplified using the Power SYBR Green Mix (Applied Biosystems) in a 384-well optical reaction plate using ABI Prism 7000 Sequence Detection System (Applied Biosystems). All primer pairs were first tested for specificity and amplicon size using end-point PCR. Formation of a dimer structure was refuted by analyzing the dissociation curve at the end of each amplification reaction. Results were calibrated in relation to an average of two house-keeping genes, <italic>Ppia</italic> and <italic>Tbp</italic>, after verifying that their levels were consistent in normal and mutant RPE. Raw data was processed using the comparative C<sub>t</sub> method by the formula 2<sup>&#8722;&#916;&#916;CT</sup>. Each amplification reaction was performed in triplicate using 20 ng of cDNA for each sample. Primers used to amplify and sequence the two <italic>Pax6&#916;PD</italic> transcript variants are listed in <xref ref-type="supplementary-material" rid="pgen.1004360.s013">Table S6</xref>.</p><p>Reporter assays were performed in HeLa cells using the Dual-Luciferase Reporter Assay System (Promega). Cells were seeded in a 24-well plate and 24 hours later were transfected using jetPEI DNA transfection reagent (Polyplus-transfection). Each well was co-transfected with three types of vectors in a total amount of 1210 ng of DNA: 1) 400 ng of a luciferase reporter vector (pGL3 basic) under the regulation of the examined promoter; 2) A total of 800 ng of expression vector (p3XFlag-CMV-10), either carrying no insert or containing an insert encoding the ORF of <italic>Pax6</italic>, <italic>Pax6&#916;PD</italic> or <italic>A-Mitf</italic>, 400 ng of each; 3) 10 ng normalizing vector (pRL-TK). Cells were harvested 48 hours after transfection and luminescence was evaluated. Each treatment was carried out in duplicate, and each assay was repeated at least three times.</p><p>End-point PCR of 17 cycles was performed using oligonucleotides containing the desired mutated sites (<xref ref-type="supplementary-material" rid="pgen.1004360.s013">Table S6</xref>, mutated nucleotides are in lower case) and the wild-type promoter reporter plasmid (pGL3 basic) as template. The PCR products were treated with 12U <italic>Dpn</italic>I restriction enzyme (Fermentas) for 1 hour at 37&#176;C, and 5 &#181;l of the DNA was transformed into <italic>E. coli XL-1Blue</italic> strain, followed by colony-picking mini-prep and midi-prep plasmid purification (Qiagen). All plasmids were verified by sequencing.</p><p>Transfection into HeLa cells was performed as described in the reporter assay section. Cells were seeded in 90-mm dishes and transfected with total of 10 &#181;g of DNA. Cells were washed with phosphate buffered saline (PBS), scraped in 1 ml lysis buffer (10 mM HEPES pH 8.0, 100 mM NaCl, 1 mM MgCl<sub>2</sub>, 0.5% NP-40) containing protease inhibitor (Roche, complete Mini EDTA-free) and incubated on ice for 30 minutes. Extracts were clarified by centrifugation at 10,000 g for 15 minutes at 4&#176;C. To avoid nonspecific binding of proteins to the beads, extracts were subjected to pre-clearing using 15 &#181;l of protein A agarose beads (Millipore, 16-157) for 2 hours at 4&#176;C, followed by centrifugation at 10,000 g for 1 minute at 4&#176;C. Input samples (50 &#181;l of the supernatant) were kept at &#8722;20&#176;C for input analysis and the cleared extracts were incubated with 5 &#181;l of rabbit anti-MITF (kindly provided by David E. Fisher, MGH <xref ref-type="bibr" rid="pgen.1004360-Levy2">[82]</xref>) with rotation overnight at 4&#176;C. The resulting immuno-complexes were incubated with 30 &#181;l of protein A beads for 2 hours at 4&#176;C. The beads were then washed four times with RIPA buffer (50 mM Tris-HCl pH 8.0, 150 mM NaCl, 1% NP-40, 0.5% Na-deoxycholate, 0.1% SDS) and the complexes were eluted in SDS sample buffer by boiling for 5 minutes. Samples were subjected to SDS&#8211;polyacrylamide gel electrophoresis. Separated proteins were transferred to nitrocellulose membrane and reacted with mouse anti-FLAG antibody (1&#8758;10,000, Sigma F3165) followed by anti-mouse horseradish peroxidase-conjugated secondary antibody. The reaction was examined by enhanced chemiluminescence detection kit (Biological Industries).</p><p>Co-immunoprecipitation from ARPE19 cells was performed essentially as described above, except cells were scraped in RIPA buffer containing protease inhibitor (Roche, complete Mini EDTA-free). Antibodies used for IP were either rabbit anti-PAX6 (Millipore, AB2237) or rabbit anti-MIT (kindly provided by David E. Fisher, MGH). Mouse anti-PAX6 (Santa Cruz, sc-3276) and mouse anti-MITF (kindly provided by David E. Fisher, MGH <xref ref-type="bibr" rid="pgen.1004360-Levy2">[82]</xref>) antibodies were used for immunoblot.</p><p>ChIP was performed as previously described <xref ref-type="bibr" rid="pgen.1004360-Sailaja1">[83]</xref>, <xref ref-type="bibr" rid="pgen.1004360-Sailaja2">[84]</xref>. Briefly, hES-RPE cells were grown as described <xref ref-type="bibr" rid="pgen.1004360-Idelson1">[52]</xref>. Fixed chromatin was extracted from 2&#215;10<sup>7</sup> cells and immunoprecipitated using rabbit anti-PAX6 (Millipore, AB2237) or non-immune rabbit IgG (Rockland) as a negative control. The primers used for ChIP analysis are listed in <xref ref-type="supplementary-material" rid="pgen.1004360.s013">Table S6</xref>.</p><p>HEK-293T cells were transfected with p3XFlag-CMV-10 encoding the ORF of full-length <italic>Pax6</italic>. Nuclear extracts were obtained as previously described <xref ref-type="bibr" rid="pgen.1004360-HayKoren1">[85]</xref>. Nuclear extract (1 ml) or 1&#8758;10 diluted nuclear extract was incubated for 10 minutes on ice in 8.5 mM HEPES pH 7.9, 30 mM KCl, 1.5 mM MgCl<sub>2</sub>, 0.4 mM DTT, 2 mg polydI/dC (Sigma). Binding with 1 ml double-stranded 59-c-ATP-labeled probe (30,000 cpm) was performed at room temperature for 20 minutes and 200 ng of &#8216;&#8216;cold&#8217;&#8217; PAX6 consensus site (PAX6CON) was used for competition <xref ref-type="bibr" rid="pgen.1004360-Wolf1">[86]</xref>.</p>