<p>The Vienna <italic>Drosophila</italic> RNAi Center (VDRC) line used for knockdown experiments was stock #25644 [<xref ref-type="bibr" rid="pgen.1005941.ref037">37</xref>]; experiments were also conducted with #25643 with similar results. The <italic>w;actin-GAL4</italic>,<italic>UAS-GAL4;</italic> animals were generated by recombining the second chromosomes of the <italic>Drosophila</italic> Genetic Resource Center (DGRC) stock <italic>y</italic><sup><italic>1</italic></sup>
<italic>w</italic><sup><italic>1118</italic></sup><italic>; P(w</italic><sup><italic>+mC</italic></sup>
<italic>= UAS-Gal4</italic>.<italic>H)12B</italic> and Bloomington Stock Center stock <italic>y</italic><sup><italic>1</italic></sup><italic>w</italic>*<italic>; P(Act5C-GAL4)25FO1/CyO</italic>,<italic>y</italic><sup><italic>+</italic></sup>; recombinants were screened molecularly and balanced in a <italic>w</italic><sup><italic>1118</italic></sup> background. The following TPI alleles in this study were generated using the GE system: <italic>dTPI</italic><sup><italic>WT</italic></sup>, <italic>hTPI</italic><sup><italic>WT</italic></sup>, <italic>dTPI</italic><sup><italic>WT-CFP</italic></sup>, <italic>dTPI</italic><sup><italic>M80T</italic></sup>, <italic>hTPI</italic><sup><italic>M82T</italic></sup>, <italic>dTPI</italic><sup><italic>T73R</italic></sup>, <italic>dTPI</italic><sup><italic>T73R-CFP</italic></sup>, <italic>dTPI</italic><sup><italic>G74E</italic></sup>, <italic>dTPI</italic><sup><italic>G74E-CFP</italic></sup>, <italic>dTPI</italic><sup><italic>T73R</italic>,<italic>G74E</italic></sup>, <italic>dTPI</italic><sup><italic>&#916;cat</italic></sup>, and <italic>dTPI</italic><sup><italic>&#916;cat-CFP</italic></sup>. The development of the GE system and the production of the <italic>dTPI</italic><sup><italic>WT</italic></sup>, <italic>dTPI</italic><sup><italic>M80T</italic></sup>, <italic>dTPI</italic><sup><italic>&#916;cat</italic></sup>, and <italic>dTPI</italic><sup><italic>&#916;cat-CFP</italic></sup> alleles were initially described elsewhere [<xref ref-type="bibr" rid="pgen.1005941.ref025">25</xref>]. Briefly, GE involves the replacement of the TPI gene locus with a phiC31 integration site through homologous recombination. The phiC31 integration system allows nearly seamless integration of complementary vector constructs directly into the TPI gene locus to maintain endogenous spatial and temporal regulation. The <italic>TPI</italic><sup><italic>null</italic></sup> allele used in <xref ref-type="supplementary-material" rid="pgen.1005941.s001">S1 Fig</xref> was generated previously, formerly known as <italic>TPI</italic><sup><italic>JS10</italic></sup> [<xref ref-type="bibr" rid="pgen.1005941.ref019">19</xref>]. RNAi experiments used the <italic>TPI</italic><sup><italic>M80T</italic></sup> allele formerly known as <italic>TPI</italic><sup><italic>sgk</italic></sup> [<xref ref-type="bibr" rid="pgen.1005941.ref019">19</xref>], due to similarities with the UAS-RNAi and GAL4 genetic backgrounds.</p><p>This study uses the established nomenclature for TPI, assuming the start methionine is removed following translation [<xref ref-type="bibr" rid="pgen.1005941.ref013">13</xref>]; all residue numbering in this study uses the same convention. An alignment is included for clarification (<xref ref-type="fig" rid="pgen.1005941.g001">Fig 1A</xref>). All animal populations assessed were approximately equivalent mixtures of males and females.</p><p>Site directed mutagenesis was performed using the QuikChange Lightening Site-Directed Mutagenesis Kit (Agilent Technologies). Mutagenesis primers were generated (Integrated DNA Technologies) to introduce a Thr-to-Arg codon change at position 73, and a Gly-to-Glu change at position 74 &#8211;both separately and together for the purpose of creating the double-mutant. Mutagenesis was performed on the previously published <italic>pGE-attBTPI</italic><sup><italic>+</italic></sup> plasmid [<xref ref-type="bibr" rid="pgen.1005941.ref025">25</xref>] and confirmed by sequencing. Once the constructs were generated, <italic>TPI</italic> GE was performed using previously published methods [<xref ref-type="bibr" rid="pgen.1005941.ref025">25</xref>,<xref ref-type="bibr" rid="pgen.1005941.ref090">90</xref>,<xref ref-type="bibr" rid="pgen.1005941.ref091">91</xref>]. Briefly, the <italic>PGX-TPI</italic> founder animals were mated to <italic>vasa-phiC31</italic><sup><italic>ZH-2A</italic></sup> animals expressing the integrase on the X chromosome and their progeny injected with <italic>pGE-attBTPI</italic> constructs. Integration events were identified via the <italic>w</italic><sup><italic>+</italic></sup> phenotype and verified molecularly.</p><p>Human TPI enzyme was purified as outlined previously [<xref ref-type="bibr" rid="pgen.1005941.ref018">18</xref>].</p><p>DLS measurements were taken using a DynaPro Plate reader (Wyatt Technology) equipped with a temperature control unit. Purified BSA (Sigma Aldrich), hTPI<sup>+</sup>, and hTPI<sup>M80T</sup> were diluted to concentrations of 3.75 &#956;M, 15 &#956;M and 30 &#956;M in 100 mM triethanolamine (TEA); pH 7.6. Three 75 &#956;l aliquots were loaded onto a 384-well microplate and read at 37&#176;C. Ten measurements were taken per sample and Dynamics V6 software (Wyatt Technology) was used to process the scattering data, generating autocorrelation functions. Autocorrelation functions were then analyzed to obtain the hydrodynamic radii. Student&#8217;s T test was used to compare samples. DLS experiments were performed three times.</p><p>Gel filtration was performed as outlined previously [<xref ref-type="bibr" rid="pgen.1005941.ref026">26</xref>]. Purified TPI samples were diluted to 15 &#956;M in mobile phase, 100 &#956;l were injected in triplicate, and their elution monitored at 280 nm. Experiments were performed three times. Chromatography traces were collected and analyzed using EZStart 7.3 (Shimadzu) to quantify the relative monomer and dimer populations. Curve integration data were compared using Student&#8217;s T test.</p><p>Isomerase activity was determined using an NADH-linked assay as previously detailed [<xref ref-type="bibr" rid="pgen.1005941.ref025">25</xref>,<xref ref-type="bibr" rid="pgen.1005941.ref092">92</xref>]. Lysates were diluted to 0.1 &#956;g/&#956;l in 100 mM TEA pH 7.6 + inhibitors and enzyme activity was assessed. Reaction assays were performed in triplicate using 80 &#956;l mixtures composed of 0.5 mM NADH, 0.752 mM GAP, 1 unit glycerol-3-phosphate dehydrogenase and 1 &#956;g of lysate protein in 100 mM TEA; pH 7.6. Consumption of NADH was monitored at 340 nm and 25&#176;C using a SpectraMax Plus 384 microplate reader (Molecular Devices). All reactions were performed at least three times. Reaction components were purchased from Sigma-Aldrich. Enzyme activity curves were normalized to reactions performed without GAP. A one-way ANOVA was performed to assess variance, and data sets were compared using Tukey&#8217;s post-hoc analysis.</p><p>Mechanical stress sensitivity was examined on Day 1 by vortexing the animals in a standard media vial for 20 seconds and measuring time to recovery, similar to [<xref ref-type="bibr" rid="pgen.1005941.ref093">93</xref>]. Briefly, recovery is defined as two purposeful WT movements including righting, grooming, climbing, or walking. Thermal stress sensitivity was assessed on Day 2 by acutely shifting animals to 38&#176;C and measuring time to paralysis, as previously described [<xref ref-type="bibr" rid="pgen.1005941.ref024">24</xref>,<xref ref-type="bibr" rid="pgen.1005941.ref069">69</xref>]. In these assays, the animals typically seized and either flipped over onto their backs or fell sideways with no successive coordinated movements, i.e. righting, climbing, walking, grooming. Behavioral responses were capped at 360 and 600 seconds where indicated and reported as 360 and 600 sec. Animal lifespan determinations were performed at 25&#176;C as previously described [<xref ref-type="bibr" rid="pgen.1005941.ref069">69</xref>]. All assays used approximately equivalent numbers of males and females. One-way ANOVAs were performed with Tukey's post-hoc analysis to compare behavior, and lifespans were assessed with Log-rank (Mantel&#8211;Cox) survival tests.</p><p>Animals were collected and aged 1&#8211;2 days at room temperature. Ten fly heads were obtained in triplicate from each genotype and processed as described previously [<xref ref-type="bibr" rid="pgen.1005941.ref034">34</xref>]. Blots were incubated with anti-TPI (1:5000; rabbit polyclonal FL-249; Santa Cruz Biotechnology), anti-ATPalpha (1:10,000; mouse monoclonal alpha5; Developmental Studies Hybridoma Bank), or anti-&#946; tubulin (1: 6,000; rabbit polyclonal H-235; Santa Cruz Biotechnology). Densitometric analyses of the scanned films were performed on unsaturated exposures using ImageJ software available from the National Institutes of Health. A one-way ANOVA was performed to assess variance of TPI levels and data sets were compared using Tukey's post-hoc analysis.</p><p>The filter-trap dot blot was modified from methods published previously [<xref ref-type="bibr" rid="pgen.1005941.ref036">36</xref>]. Animals were aged 1&#8211;2 days, collected and homogenized in 1X PBS (2.7 mM KCl, 137 mM NaCl, 2 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM Na<sub>2</sub>HPO<sub>4</sub>; pH 7.4) supplemented with cOmplete mini Protease Inhibitors (Roche Diagnostics), and diluted to 1 &#956;g/&#956;l; wells were loaded as indicated. Samples were diluted 1:2 in 1% SDS, 1X PBS, boiled for 5 min., and filtered through a cellulose acetate membrane (Whatman, 0.2&#956;m pore) using a 96-well vacuum dot blot apparatus. Positive controls were collected from PC12 cells transfected with a construct expressing huntingtin &#8220;exon1&#8221; bearing a stretch of 97 glutamines and C-terminally tagged with EGFP. The membrane was washed four times with 1% SDS-PBS, blocked with Odyssey Blocking Buffer (LiCor), and primary antibodies applied in Odyssey Blocking Buffer. Blots were incubated with anti-TPI (1: 5000) and anti-GFP (1: 5000; rabbit polyclonal; FL; Santa Cruz Biotechnology). The membranes were then washed and incubated with the secondary antibody IRDye 800-conjugated goat anti-rabbit (LiCor) at 1: 20,000 in Odyssey Blocking Buffer. Direct-to-scanner detection and blot visualization were performed using a LiCor Odyssey scanner. Filter-trap experiments were performed twice.</p><p>Coimmunoprecipitation was performed using the Pierce Co-Immunoprecipitation Kit (Thermo Scientific) as per manufacturer&#8217;s instructions. Lysates were generated by mechanically homogenizing 50 animals in 0.5 ml of IP Lysis buffer (25 mM Tris, 150 mM NaCl, 1 mM EDTA, 1% NP-40, 5% glycerol; pH 7.4) supplemented with cOmplete mini Protease Inhibitors. After homogenization, lysates were frozen in liquid nitrogen, thawed, then centrifuged twice at 5,000 <bold><italic>g</italic></bold> to pellet exoskeletal debris. Supernatants were collected and diluted to 1 &#956;g/&#956;l, and 400 &#956;g were loaded onto 25 &#956;l of gel pre-coupled with 10 &#956;g of anti-GFP. A negative control was performed using uncoupled gel and <italic>dTPI</italic><sup><italic>+</italic></sup><italic>/dTPI</italic><sup><italic>&#916;cat</italic></sup> lysate. Samples were incubated overnight at 4&#176;C and washed ten times with IP Lysis buffer at 4&#176;C. Beads were eluted by boiling with 70 &#956;l of 2X SDS&#8211;PAGE sample buffer, separated via SDS-PAGE, immunoblotted, and analyzed as outlined above. Coimmunoprecipitations were performed three times.</p><p>Images were taken with an Olympus BX51WI fluorescence microscope with Till Photonics Polychrome V monochromator excitation, and Hamamatsu C4742-95 digital camera. Heterozygous <italic>TPI</italic><sup><italic>T73R</italic></sup> larvae were maintained over TM6B, and <italic>Tb</italic><sup><italic>+</italic></sup> 3<sup>rd</sup>-instar larvae selected for analysis. Dissection and preparation of larval NMJs were performed as described previously [<xref ref-type="bibr" rid="pgen.1005941.ref046">46</xref>]. FM1-43FX dye [Molecular Probes, Invitrogen] loading was performed as previously detailed [<xref ref-type="bibr" rid="pgen.1005941.ref047">47</xref>]. Briefly, animals were dissected in ice cold 0 mM Ca<sup>2+</sup> HL-3 with 0.5 mM EGTA, motor neurons severed, and the preps heated to room temperature or 38&#176;C over the course of 3 min. Bath temperature was monitored throughout the experiments with a microthermal probe to ensure consistency [Fisher Scientific]. Loading experiments were performed with room temperature or 38&#176;C preheated 90 mM KCl, 1.5 mM CaCl<sub>2</sub> HL-3 supplemented with 4 &#956;M FM1-43FX, and preparations were washed quickly and thoroughly during the experiments to avoid Ca<sup>2+</sup> chelation. After loading, preparations were washed with 15 ml of 0mM Ca<sup>2+</sup> HL-3 with 0.5 mM EGTA at room temperature for 10 min. Unloading experiments were performed as follows: preparations were loaded for 3 min at room temperature; washed with 15 ml of 0mM Ca<sup>2+</sup> HL-3 with 0.5 mM EGTA at room temperature; imaged; washed with 38&#176;C 0 mM Ca<sup>2+</sup> HL-3; heated to 38&#176;C over 3 min.; unloaded using 38&#176;C 90 mM KCl, 1.5 mM CaCl<sub>2</sub> HL-3 for 30 sec.; and the same synapse was imaged again.</p><p>Preparations were imaged with a water immersion 60X objective, using 450 nm excitation and a 500 nm longpass filter [Chroma Technology]. Simple PCI imaging software was used for acquisition and ImageJ for analysis. Two NMJs from muscles 6/7 were assessed per animal, one from segment A2 and A3. Six animals were assessed per genotype per time point for a total of 12 NMJs per experimental condition. After acquisition, images were relabeled by an independent researcher and blinded analysis was performed on the raw images. Boutons were outlined and intensity measured, with background subtracted from adjacent tissue. Pair-wise analyses were performed using a two-tailed Student&#8217;s t test, while comparisons among multiple experimental conditions were performed using a one-way Analysis of Variance (ANOVA) with Tukey&#8217;s post-hoc analysis.</p><p>For NMJ morphological analyses, 3<sup>rd</sup>-instar larvae were collected and dissected as detailed above without transection of the descending motor neurons. Preparations were fixed in 3.5% paraformaldehyde HL-3, permeabilized with 0.1% Triton X-100 in 1X PBS (PBST), and blocked with 0.2% BSA in PBST (PBSTB) for 2 hrs at room temperature. Preps were washed and incubated with goat anti-HRP [Jackson Laboratories] at 1: 200 in PBSTB for 2 hrs at room temperature. Primary antibodies were removed, washed in PBSTB, and incubated with Cy3-labeled donkey anti-goat in PBSTB at 1: 400 for 1.5 hrs at room temperature. Preps were washed, mounted in VectaShield [Vector Laboratories], and imaged within three days. Images were acquired with an Olympus confocal FV1000 microscope, using a 559 nm excitation laser. Z stacks of segment A2 of muscle 6/7 were taken using 1 &#956;m steps. The Z stacks were merged using Olympus FV1000 Fluoview Viewer, and morphology determined. Ten animals were assessed per genotype, one NMJ per animal, for a total of ten NMJs per experimental condition. Boutons were defined as varicosities at least 2 &#956;m in diameter, and branches defined as extensions containing at least 2 boutons. Images were relabeled by an independent researcher for blinded analysis. Variance within the data set was examined using a one-way ANOVA, with comparisons made using Tukey&#8217;s post hoc test.</p><p>All image quantification was performed on raw image files acquired below saturation. Representative images were selected on the basis of raw image measurements, and post-acquisition processing was performed uniformly with grouped images in parallel using ImageJ; in agreement with published guidelines [<xref ref-type="bibr" rid="pgen.1005941.ref094">94</xref>].</p><p>Recombinant hTPI<sup>&#916;cat</sup> containing the K13M substitution was expressed and purified as previously described [<xref ref-type="bibr" rid="pgen.1005941.ref018">18</xref>] using affinity, anion exchange, and size exclusion chromatography. Purified protein was dialyzed into a buffer containing [20 mM Tris pH 8.8, 25 mM NaCl, 2.0% glycerol and 1 mM &#946;&#8211;mercaptoethanol], and concentrated to 6 mg/ml prior to crystallization. Crystals of TPI<sup>&#916;cat</sup> were obtained using the vapor diffusion method with sitting drops containing 1 &#956;l of protein and 2 &#956;l of well solution [28&#8211;34% PEG2000 MME, 50 mM KBr]. Initial crystals grew within 3 days and were improved by successive rounds of microseeding. Crystals were cryoprotected in 40% PEG 2000MME, 20% glycerol, 50 mM KBr, prior to flash freezing in liquid nitrogen.</p><p>Data collection was performed at the National Synchrotron Light Source at beamline X25 and using a Pilutas 6M detector. Diffraction data was integrated, scaled, and merged using HKL2000 [<xref ref-type="bibr" rid="pgen.1005941.ref095">95</xref>]. hTPI<sup>&#916;cat</sup> crystals belong to space group P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> and contain a dimer in the asymmetric unit. Initial phases were estimated for hTPI<sup>&#916;cat</sup> via molecular replacement using a previously determined structure of wild-type as our search model [<xref ref-type="bibr" rid="pgen.1005941.ref018">18</xref>]. Model bias was reduced through simulated annealing and the model was further improved by manual model building combined with positional and anisotropic B-factor refinement within Phenix [<xref ref-type="bibr" rid="pgen.1005941.ref096">96</xref>]. Model quality was validated using MolProbity [<xref ref-type="bibr" rid="pgen.1005941.ref097">97</xref>]. Figs were generated using PyMOL (PyMOL Molecular Graphics System, Schr&#246;dinger, LLC). Coordinates and structure factors for hTPI<sup>&#916;cat</sup> have been deposited within the Protein Databank under accession code 4ZVJ.</p><p>The effect of hT75R and hG76E substitutions were modeled onto hTPI<sup>&#916;cat</sup> or hTPI<sup>WT</sup> structural templates using the RosettaBackrub analysis as implemented within the RosettaBackrub server [<xref ref-type="bibr" rid="pgen.1005941.ref098">98</xref>]. For all predictions, Rosetta version 3.1 was used as the algorithm with a backrub radius of 15 &#197; to ensure that perturbations extending away from the site of the substitution could be sampled. An ensemble of 50 structures was predicted for each TPI model, and their reported Rosetta scores normalized to the starting template. Structures whose Rosetta scores were in the most favorable 10% of the ensemble were used in <xref ref-type="fig" rid="pgen.1005941.g009">Fig 9D</xref>.</p><p>Triosephosphate isomerase research has been split between those studying the enzymatic and structural properties of the protein, and those studying its role in disease. Researchers focusing on the pathology of TPI deficiency typically use the abbreviation &#8220;TPI&#8221;, whereas enzymologists and structural biologists used the abbreviation &#8220;TIM&#8221;. The aim of this study was to determine the molecular mechanism of a disease mutation, and as such we have used the abbreviation &#8220;TPI&#8221;.</p>