<p>Strains used in this work are described in [<xref ref-type="bibr" rid="pgen.1005217.ref028">28</xref>]. To construct <italic>sub1</italic>-deficient LAN210 derivative, we PCR-amplified the <italic>sub1</italic>::<italic>KanMX</italic> cassette from BY4742 <italic>sub1</italic>::<italic>KanMX</italic> strain, and used the resulting PCR product to transform the LAN200 strain (haploid <italic>ung1</italic>, [<xref ref-type="bibr" rid="pgen.1005217.ref023">23</xref>]; genotype <italic>MAT</italic>a <italic>ade5-1 lys2-Tn5-13 trp1-289 his7-2 leu2-3</italic>,<italic>112 ung1</italic>::<italic>hygB</italic>). We PCR-verified the resulting Kan<sup>R</sup> clones and transformed them with plasmid YEpHO encoding for HO endonuclease. Autodiploids (LAN210 <italic>sub1</italic>) have been selected and used for mutagenesis experiments.</p><p>Yeast strains LAN210 or LAN210 <italic>sub1</italic> were transformed with plasmids pESC-<italic>LEU</italic>-PmCDA1, pESC-<italic>LEU</italic>-AID, pESC-<italic>LEU</italic>-APOBEC3G, or pESC-<italic>LEU</italic>-APOBEC1 [<xref ref-type="bibr" rid="pgen.1005217.ref027">27</xref>], and transformants were colony-purified and inoculated into 5 ml of synthetic complete medium (SC) lacking leucine and containing 2% glucose. After one day of incubation, cultures were spun down, washed with water and resuspended in 12 ml of SC containing 1% raffinose and 2% galactose. After three days of culture growth, aliquots were plated on SC plates with 60 mg/L of L-canavanine to select for <italic>can1</italic> mutants and to the synthetic complete plates (with dilutions) to estimate viability. Some mutants were selected on 5-fluoroorotic acid (5-FOA) plates (0.1% 5-FOA) instead of canavanine plates. Clones resistant to either canavanine or 5-FOA were selected, colony purified and stored at -80C.</p><p>Our original tests indicated that glucose-containing medium during the first stages of culture growth can be substituted for the synthetic medium lacking glucose but with raffinose. Later, we found that the mutation frequency can vary under these conditions. We do not know the exact reason for this high variability under raffinose pre-growth conditions.</p><p>In addition to the forward mutation assay (<italic>CAN1</italic> gene) we have analyzed the reversions on <italic>his7-2</italic>, <italic>trp1-289</italic>, <italic>lys2-Tn5-13</italic>, and <italic>ade5-1</italic> mutant alleles [<xref ref-type="bibr" rid="pgen.1005217.ref027">27</xref>] in the spot test (<xref ref-type="supplementary-material" rid="pgen.1005217.s009">S6 Fig</xref>). Yeast of the corresponding strains were streaked on YPDU plates. One set of plates was UV-irradiated (20J/m<sup>2</sup>), another set was HAP-mutagenized as described in [<xref ref-type="bibr" rid="pgen.1005217.ref028">28</xref>], and the third was left untreated to assay for spontaneous mutagenesis. After one day, yeast cells were replica-plated only onto corresponding selective plates. Colonies were scored after three days of growth.</p><p>Genomic DNA was prepared as described in [<xref ref-type="bibr" rid="pgen.1005217.ref028">28</xref>]. We used two methods of library preparation, both from Illumina: TruSeq was used for majority of the clones (containing &#8220;RUN1&#8221; to &#8220;RUN5&#8221; in systematic names of clones, see <xref ref-type="table" rid="pgen.1005217.t001">Table 1</xref>), and Nextera XT (those clones that contain &#8220;RUN6&#8221; in the clone name). All libraries have been sequenced by 100 bp paired-end reads, except for the clones containing &#8220;RUN4&#8221; in the name, which have been sequenced using 100 bp single-end reads. Raw sequencing reads (fastq format) have been deposited to the Sequence Read Archive (SRA), accession numbers SRP056337 (whole genome sequencing files), and SRP056371 (RNA-seq data).</p><p>To improve the quality of our reference genome assembly, we have sequenced wild-type strain LAN200 on the Roche Junior instrument (Research and Resource Center &#8220;Molecular and Cellular Technologies&#8221;, Saint Petersburg State University, Russia) using 454 technology according to the manufacturer&#8217;s recommendations.</p><p>RNA was purified using a RiboPure Yeast Kit (Life Technologies, Ambion, cat. number AM1926). To prepare the libraries for RNA sequencing, we used the ScriptSeq Complete Gold Kit (Yeast) (Epicentre cat. number BGY1306). This kit allows for effective depletion of all yeast rRNA species and also preserves information about RNA strand. Size distribution of the resulting libraries was estimated by both agarose gel electrophoresis followed by SYBR Gold or SYBR Green (both from Life Technologies) staining and visualization on Typhoon scanner, and Bioanalyzer (Agilent Technologies). We have used two RNA samples from the LAN210 strain expressing PmCDA1, and one RNA sample from the LAN210 <italic>sub1</italic> strain expressing PmCDA1. RNA was purified from the same cultures used to measure mutation frequencies. The libraries were pooled together and sequenced on a HiSeq2500 instrument using 100 bp paired-end reads.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All DNA Illumina reads were filtered by base quality (threshold Phred score 20), 5' trimmed (see <xref ref-type="supplementary-material" rid="pgen.1005217.s012">S1 Protocol</xref> for details). Adapters were filtered from the 3' end of reads (<xref ref-type="supplementary-material" rid="pgen.1005217.s012">S1 Protocol</xref>). Minimum read length after filtering was set to 20. The filtering steps were performed using Trim Galore! (<ext-link ext-link-type="uri" ns0:href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</ext-link>). Next, filtered reads were corrected by BayesHammer module of Spades assembler [<xref ref-type="bibr" rid="pgen.1005217.ref078">78</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Roche reads were cut to a length of 479 and then 11 bp from the 5' end of reads were trimmed. After this, reads were filtered by base quality using fastq_quality_filter (parameters-q 20-p 80) from FastX package (<ext-link ext-link-type="uri" ns0:href="http://hannonlab.cshl.edu/fastx_toolkit/index.html">http://hannonlab.cshl.edu/fastx_toolkit/index.html</ext-link>).</p><p>We have improved the genome of our reference strain LAN210 [<xref ref-type="bibr" rid="pgen.1005217.ref028">28</xref>] by taking advantage of longer reads obtained from Roche sequencing run. Assembling and correction of assembling errors were performed in several steps (see diagram in <xref ref-type="supplementary-material" rid="pgen.1005217.s010">S7 Fig</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In the first step, N008-LAN210-wt-NA-NA-RUN2 and Roche reads were assembled <italic>de novo</italic> to contigs using Mira assembler v 4.0 (<ext-link ext-link-type="uri" ns0:href="http://www.chevreux.org/projects_mira.html">http://www.chevreux.org/projects_mira.html</ext-link>). Contigs with a length of 500 bp and more were aligned to standard yeast reference genome S288C_R64 using BWA (mem algorithm) 0.7.5a (<ext-link ext-link-type="uri" ns0:href="http://bio-bwa.sourceforge.net/">http://bio-bwa.sourceforge.net/</ext-link>). Then, the consensus sequence (designated as &#8220;raw reference&#8221;) was extracted from alignment using SAMtools v0.1.19 ([<xref ref-type="bibr" rid="pgen.1005217.ref079">79</xref>], <ext-link ext-link-type="uri" ns0:href="http://samtools.sourceforge.net/">http://samtools.sourceforge.net/</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In the second step, filtered and corrected sequencing reads from the wild-type LAN210 strain (N008-LAN210-wt-NA-NA-RUN2) were aligned to the &#8220;raw reference&#8221;, and variants were extracted from alignment as described in the section <bold>Detection of variants</bold>. Then homozygous variants were used to correct the &#8220;raw reference&#8221; to the &#8220;intermediate reference&#8221; using GATK (<ext-link ext-link-type="uri" ns0:href="https://www.broadinstitute.org/gatk/">https://www.broadinstitute.org/gatk/</ext-link>).</p><p>In the last step, common homozygous variants from alignments of corrected and filtered reads from sequencing LAN200 and LAN201 strains (clones N001-LAN200-wt-NA-NA-RUN1 and N002-LAN201-wt-NA-NA-RUN1) were used in the same way as described previously to get the final reference genome (<xref ref-type="supplementary-material" rid="pgen.1005217.s002">S2 Dataset&#8212;LAN210_v0.10m</xref> genome in FASTA format).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The Saccharomyces Genome Database (SGD, <ext-link ext-link-type="uri" ns0:href="http://www.yeastgenome.org/">www.yeastgenome.org</ext-link>) lacks UTR annotations. We have combined the published UTR annotations [<xref ref-type="bibr" rid="pgen.1005217.ref034">34</xref>] with SGD annotations of standard yeast reference S288C (S288C_R64). These merged annotations were transferred to the LAN210_v0.10m genome sequence using a custom script based on BLAST. The annotations in the GFF format can be found in <xref ref-type="supplementary-material" rid="pgen.1005217.s003">S3 Dataset</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Repeats in LAN210_v0.10m were masked using four sources (see <xref ref-type="supplementary-material" rid="pgen.1005217.s012">S1 Protocol</xref> and <xref ref-type="supplementary-material" rid="pgen.1005217.s010">S7</xref> Fig). First, interspersed repeats were discovered <italic>de novo</italic> using Repeat Modeler (<ext-link ext-link-type="uri" ns0:href="http://www.repeatmasker.org/RepeatModeler.html">http://www.repeatmasker.org/RepeatModeler.html</ext-link>). Then, they were combined with known fungal repeats from RepBase (<ext-link ext-link-type="uri" ns0:href="http://www.girinst.org/repbase/index.html">http://www.girinst.org/repbase/index.html</ext-link>), and both were masked using Repeat Masker (<ext-link ext-link-type="uri" ns0:href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</ext-link>). Second, tandem repeats were raw masked using TRF ([<xref ref-type="bibr" rid="pgen.1005217.ref080">80</xref>], <ext-link ext-link-type="uri" ns0:href="http://tandem.bu.edu/trf/trf.html">http://tandem.bu.edu/trf/trf.html</ext-link>). Then, raw masked tandem repeats were checked for over masking (known problem of TRF) using LAN210_v0.10m annotations: entries in genes were removed. Third, all tRNA features were masked. Fourth, manual masking based on BWA alignment of contigs (see <bold><xref ref-type="sec" rid="sec013">Assembly of LAN210 reference genome</xref></bold>section) to LAN210_v0.10m was performed, resulting in regions with conflicting contigs being masked.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For each clone/sample (see <xref ref-type="table" rid="pgen.1005217.t001">Table 1</xref>), corrected and filtered reads (as described previously) were aligned to the LAN210_v0.10m genome using Bowtie2 v2.1.0 ([<xref ref-type="bibr" rid="pgen.1005217.ref081">81</xref>], <ext-link ext-link-type="uri" ns0:href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</ext-link>) in &#8220;very-sensitive&#8221; mode. Then, the variant call was performed using Unified Genotyper (with options-stand_call_conf 100-stand_emit_conf 40) from GATK, for both indels and SNPs. After that, SNPs and indels were split and independently filtered according to the GATK's Best Practices (<ext-link ext-link-type="uri" ns0:href="https://www.broadinstitute.org/gatk/guide/best-practices">https://www.broadinstitute.org/gatk/guide/best-practices</ext-link>). Finally, filtered indels and SNVs from LAN200, LAN201, and LAN210 (N008-LAN210-wt-NA-NA-RUN2, N001-LAN200-wt-NA-NA-RUN1, and N002-LAN201-wt-NA-NA-RUN1) were combined. Only SNVs were further considered for all other samples. For SNVs in deaminase-induced mutants, the last filtration step was associated with clustering (described below).</p><p>Genomes of diploid yeast mutants induced by deaminases possess hundreds to thousands of mutations (]28] and <xref ref-type="table" rid="pgen.1005217.t001">Table 1</xref>), making these data a good material to detect the hypermutable genomic regions. To increase the sensitivity of detection even further, we pooled the mutations from the clones induced by the same deaminase together. The resulting datasets are characterized by very high mutation densities over the relatively small yeast genome. because of that, we developed a sophisticated clustering procedure that was applied for combined SNV sample sets and consisted of several stages (see scheme in <xref ref-type="supplementary-material" rid="pgen.1005217.s011">S8 Fig</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">
<bold>In the first step</bold>, hierarchical clustering was performed using the SciPy library (<ext-link ext-link-type="uri" ns0:href="http://www.scipy.org/">http://www.scipy.org/</ext-link>). In hierarchical clustering, objects are grouped together (based on distances between them) in nested clusters until a single cluster harboring all objects is formed. The structure of derived clustering can be easily represented as a tree. At this time, the choice of a method to calculate the distances between objects and clusters is highly important. However, there is still no general approach to make such choice. So, one has to choose an appropriate method based on the features of the data he is studying. We choose an average distance (UPGMA) method to calculate the distances between clusters (the reasons are described below). In this case, a distance <italic>d(u</italic>, <italic>v)</italic> between two clusters of objects <italic>u</italic> and <italic>v</italic> is calculated as follows:
<disp-formula id="pgen.1005217.e001"><alternatives><graphic id="pgen.1005217.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005217.e001.jpg" /><ns1:math id="M1"><ns1:mi>d</ns1:mi><ns1:mfenced separators="|"><ns1:mrow><ns1:mi>u</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>v</ns1:mi></ns1:mrow></ns1:mfenced><ns1:mi>&#160;</ns1:mi><ns1:mo>=</ns1:mo><ns1:mi>&#160;</ns1:mi><ns1:mrow><ns1:munder><ns1:mo stretchy="false">&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mi>j</ns1:mi></ns1:mrow></ns1:munder><ns1:mrow><ns1:mfrac><ns1:mrow><ns1:mi>d</ns1:mi><ns1:mo>(</ns1:mo><ns1:mi>u</ns1:mi><ns1:mfenced close="]" open="[" separators="|"><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:mfenced><ns1:mo>,</ns1:mo><ns1:mi>&#160;</ns1:mi><ns1:mi>v</ns1:mi><ns1:mfenced close="]" open="[" separators="|"><ns1:mrow><ns1:mi>j</ns1:mi></ns1:mrow></ns1:mfenced><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mfenced close="|" open="|" separators="|"><ns1:mrow><ns1:mi>u</ns1:mi></ns1:mrow></ns1:mfenced><ns1:mi>*</ns1:mi><ns1:mfenced close="|" open="|" separators="|"><ns1:mrow><ns1:mi>v</ns1:mi></ns1:mrow></ns1:mfenced><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mfrac></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
where <italic>d(u[i]</italic>, <italic>v[j])</italic> is the distance between the <italic>i-th</italic> object from cluster <italic>u</italic> and the <italic>j-th</italic> object from cluster <italic>v</italic>, and |<italic>u</italic>| and |<italic>v</italic>| are numbers of objects in clusters <italic>u</italic> and <italic>v</italic>, respectively. The average distance used to perform hierarchical clustering and the distance criterion used to extract clusters do not set strict limitations for cluster length. Importantly, 1 Kb is not the maximum length of a cluster, as some clusters are longer; the threshold is based on cophenetic distance (see below). Also, the UPGMA method takes into account distances between all mutations in the cluster in comparison to NPA (nearest point algorithm) or FPA (farthest point algorithm).</p><p>If there is no interest in hierarchical clustering process as it but instead one would like to obtain a set of clusters, a method and threshold to extract them from the clustering &#8220;tree&#8221; have to be chosen. We choose a distance criterion as the most appropriate method. This means that all SNVs in each extracted cluster should have no greater than a cophenetic distance than the threshold. The cophenetic distance between two objects is the distance between the largest two clusters containing these objects individually when they are merged into a single cluster that contains both. In other words, it is the height of the tree where the two branches, including the two objects, merge together. For our purposes, &#8220;clusters&#8221; are defined as &#8220;clusters extracted from hierarchical clustering&#8221;. It is important to distinguish them from nested clusters of hierarchical clustering.</p><p>To set a threshold for extraction of clusters, we performed an extraction with different values of threshold in a range from 50 bp to 5000 bp with a 50 bp step. The number of clusters with three or more mutations reaches or nearly reaches a plateau when the threshold was set to a value in the range of 1000 to 2000 bp (<xref ref-type="supplementary-material" rid="pgen.1005217.s005">S2 Fig</xref>). Considering the high density of mutations in our dataset, and <italic>a priori</italic> knowledge of approximate cluster size (rainfall plot) we have set the cutoff for extraction to 1000 bp.</p><p>The distance criterion was used for extraction because, except for clustered mutations, there was a &#8220;noise&#8221; consisting of &#8220;random&#8221; mutations. It was impossible to predict the number of clusters or to directly use more &#8220;natural' methods, such as an inconsistency coefficient (which was used to adjust clusters in the next step).</p><p>
<bold>In the second step</bold>, hierarchical clustering was performed independently for the SNVs in each cluster, and subclusters were extracted using inconsistency coefficient with the threshold set to 0.8 and depth set to two to adjust the borders of clusters. The inconsistency coefficient characterizes each link in a cluster tree by comparing its height with the average height of adjacent links that are less than the depth value below it in the cluster hierarchy. The higher value of this coefficient indicates the lower similarity of the objects connected by the link.</p><p>
<bold>In the third step</bold>, a filtering of clusters was performed. All clusters with one or more SNV in the masked (see <bold><xref ref-type="sec" rid="sec015">Repeat Masking</xref></bold>above) regions were removed. Also, for deaminase sample sets, clusters with more than one non-deaminase-like SNV (non- C-&gt;T or G-&gt;A) were considered as errors of sequencing or SNV call and removed from analysis. In other words, filtering of SNVs for deaminase sample sets is merged with clustering filtration. Clusters with one non-deaminase-like SNV present in only one sample in a set were marked and corresponding SNVs were removed. At this step, SNVs for each sample were independently restored from clustering results. The final SNV datasets can be found in <xref ref-type="supplementary-material" rid="pgen.1005217.s001">S1 Dataset</xref>.</p><p>Finally, clusters were filtered based on their size (only clusters with five and more SNVs were retained) and power (calculated as the size of a cluster divided by the median distance between SNVs in the cluster). The minimum power threshold was set to 0.05 to remove the clusters with a low density of SNV that may be due to high-level &#8220;noise&#8221; because of high mutation densities.</p><p>We have used Top Hat followed by cufflinks [<xref ref-type="bibr" rid="pgen.1005217.ref082">82</xref>] in annotation-guided modes to obtain FPKM values for the genes annotated in our reference genome. FPKM stands for <underline>f</underline>ragments <underline>p</underline>er <underline>k</underline>ilobase per <underline>m</underline>illion mapped reads and represents relative expression values for the genes and/or transcripts.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Analysis of mutation densities around CDS start/UTR/TSS was performed using custom scripts (<xref ref-type="supplementary-material" rid="pgen.1005217.s013">S2 Protocol</xref>). Sequence logos (<xref ref-type="fig" rid="pgen.1005217.g001">Fig 1</xref>) were created from pooled SNV data in WebLogo 3.0 (<ext-link ext-link-type="uri" ns0:href="http://weblogo.berkeley.edu/">http://weblogo.berkeley.edu/</ext-link>), using <italic>S</italic>. <italic>cerevisiae</italic> GC content (38%) for values normalization. Sequences were reverse-complemented where applicable using FastX-Toolkit (<ext-link ext-link-type="uri" ns0:href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</ext-link>). Due to the relatively low number of mutations in the genomes of APOBEC3G-induced mutants and the resulting increased ratio of false positive SNVs to the real mutations, the results of the automatic SNV call were manually checked in the genome browser. The manually filtered SNVs dataset was used for logo creation.</p><p>Most of the statistical tests have been computed in R.</p>