<p>All strains used in this study (<xref ref-type="supplementary-material" rid="pgen.1005335.s013">S6 Table</xref>) are isogenic to BR1919-8B [<xref ref-type="bibr" rid="pgen.1005335.ref069">69</xref>]. Strain variants were created by standard genetic crosses and transformation procedures. Every strain in which the <italic>K</italic>. <italic>l</italic>. <italic>ZIP1</italic> open reading frame replaced the <italic>S</italic>. <italic>c</italic>. <italic>ZIP1</italic> open reading frame was derived from the same parent, CO1. CO1 was created by first inserting <italic>URA3</italic> in place of <italic>S</italic>. <italic>c</italic>. <italic>ZIP1</italic> open reading frame sequences. Next, a PCR product containing the <italic>K</italic>. <italic>l</italic>. <italic>ZIP1</italic> open reading frame (amplified off of genomic DNA extracted from <italic>K</italic>. <italic>lactis</italic> cells) flanked by ~50 bp of homology to the 5&#8217; and 3&#8217; sequences of the <italic>S</italic>. <italic>c</italic>. <italic>ZIP1</italic> endogenous locus was transformed into the <italic>zip1</italic>::<italic>URA3</italic> strain, in order to replace the <italic>URA3</italic> sequences at the <italic>ZIP1</italic> locus with <italic>K</italic>. <italic>l</italic>. <italic>ZIP1</italic> sequences. Primers used for this step were: 5&#8217;TTCTTTGAGATTCGGAAGTAAAATACCCTCGGCGGCTAAATTTTTAGAGA<underline>ATGTCTAACTTCTTCAGAGACAACTCG</underline> 3&#8217; and 5&#8217;ACAAAATGAAATGTATTCGCACAAAACGATTTCAAATTTTCCATTATCCT<underline>TTATCTGAATCTTTTGGTCTTTTTTAATCGAGG</underline> 3&#8217; (underlined regions correspond to <italic>K</italic>. <italic>l</italic>. <italic>ZIP1</italic> sequences). Counterselection against Ura+ was carried out using 5-FOA medium.</p><p>The <italic>K</italic>.<italic>lactis ZIP1</italic>-V5 fusion cassette was created by first inserting <italic>URA3</italic> between the codons for amino acids 472 and 473 of <italic>K</italic>. <italic>lactis</italic> Zip1. Next, a PCR product with flanking homology to <italic>K</italic>.<italic>lactis ZIP1</italic> but carrying an in-frame V5 sequence was used to counterselect against Ura+ cells on 5-FOA medium. DNA sequencing confirmed the position of V5 coding sequences in frame with the codon for amino acid 472 in an otherwise complete <italic>K</italic>.<italic>lactis ZIP1</italic> gene.</p><p>To construct a haploid strain capable of sporulation, <italic>MATa</italic> was integrated at the <italic>THR1</italic> locus in a haploid <italic>MAT&#945;</italic> strain, using the B211 plasmid from Beth Rockmill [<xref ref-type="bibr" rid="pgen.1005335.ref069">69</xref>].</p><p>Strains used for crossover analysis in spores carry a <italic>hphMX</italic> cassette inserted near the chromosome III centromere, <italic>ADE2</italic> inserted upstream of the <italic>RAD18</italic> locus, a <italic>natMX</italic> cassette inserted near the <italic>HMR</italic> locus, <italic>TRP1MX4</italic> was inserted just downstream of the <italic>SPO11</italic> locus, and <italic>LEU2</italic> and <italic>THR1</italic> were inserted on chromosome XI at 152kb, and at 193,424bp, respectively.</p><p>Chromosome III circular <italic>MAT&#945;</italic> strains as well as TY521 and TY522 [<xref ref-type="bibr" rid="pgen.1005335.ref018">18</xref>] were received from the Roeder lab.</p><p>Meiotic nuclei were surface spread on glass slides and imaged as described in [<xref ref-type="bibr" rid="pgen.1005335.ref013">13</xref>]. The following antibodies were used: rabbit anti-Zip1 (created as described in [<xref ref-type="bibr" rid="pgen.1005335.ref010">10</xref>]), rabbit anti-Red1 [<xref ref-type="bibr" rid="pgen.1005335.ref061">61</xref>], guinea pig anti-SUMO [<xref ref-type="bibr" rid="pgen.1005335.ref011">11</xref>], chicken anti-HA (Abcam), mouse anti-MYC (clone 9E10, Invitrogen), rabbit anti-V5 (Abcam). Secondary antibodies conjugated with Alexa Fluor dyes were purchased from Life Technologies and used at a 1:200 dilution.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genetic crossover data was compiled and processed using an Excel Linkage Macro program, created by Jonathan Greene (Rhona Borts, pers. comm.) and donated by Eva Hoffmann (University of Sussex, UK). Final crossover and interference values (and their standard errors) were obtained using the Stahl lab online tools (<ext-link ext-link-type="uri" ns0:href="http://molbio.uoregon.edu/~fstahl/">http://molbio.uoregon.edu/~fstahl/</ext-link>), with the method of Perkins [<xref ref-type="bibr" rid="pgen.1005335.ref092">92</xref>]. All other statistical analyses were carried out using Graphpad Prism or Graphpad InStat (<ext-link ext-link-type="uri" ns0:href="http://www.graphpad.com/">www.graphpad.com</ext-link>).</p><p>Agarose plugs were prepared from meiotic cultures at 0, 40 and 70 hours of sporulation and subjected to pulsed-field gel analysis [<xref ref-type="bibr" rid="pgen.1005335.ref018">18</xref>,<xref ref-type="bibr" rid="pgen.1005335.ref079">79</xref>]. For Southern blotting, a 1 kb probe from the <italic>THR4</italic> region of chromosome III was prepared using a DIG High Prime DNA Labeling and Detection Kit (Roche). A Syngene &#8220;G:Box&#8221; was used to detect chemiluminescence and the Syngene &#8220;Gene-Tools&#8221; program was used to analyze the data. A value for % recombination (<xref ref-type="fig" rid="pgen.1005335.g007">Fig 7</xref>) was calculated by summing twice the intensity of the trimer band (a double crossover product) plus the dimer band (product of a single crossover) over the total intensity of the three bands (trimer, dimer and monomer). Note that circular chromosome III chromatids do not enter the gel, and thus are not included in the calculation to estimate recombination. The average of two experiments is presented.</p><p>Western blotting was performed as described previously [<xref ref-type="bibr" rid="pgen.1005335.ref013">13</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">2D gel electrophoresis followed by Southern analysis to assay JMs was performed as previously described [<xref ref-type="bibr" rid="pgen.1005335.ref035">35</xref>,<xref ref-type="bibr" rid="pgen.1005335.ref066">66</xref>,<xref ref-type="bibr" rid="pgen.1005335.ref093">93</xref>] Probes for detection of JMs at the <italic>ERG1</italic> locus [<xref ref-type="bibr" rid="pgen.1005335.ref077">77</xref>,<xref ref-type="bibr" rid="pgen.1005335.ref081">81</xref>] were amplified from yeast genomic DNA with primers- 5&#8217;-GGCAGCAACATATCTCAAGGCC-3&#8217; and 5&#8217;-TCAATGTAGCCTGAGATTGTGGCG-3&#8217;. Probes for detection of JMs at <italic>YCR047c</italic> [<xref ref-type="bibr" rid="pgen.1005335.ref035">35</xref>,<xref ref-type="bibr" rid="pgen.1005335.ref080">80</xref>] were amplified from yeast genomic DNA using primers 5&#8217;-GGAATTCCGAGAGAATCGACTTGCTAA-3&#8217; and 5&#8217;-GGAATTCCAGCCACCAGTGGGCTTTTC-3&#8217;. Hybridization signal was detected and quantified using a Typhoon FLA 9000 (GE) and the ImageJ software (<ext-link ext-link-type="uri" ns0:href="http://imagej.nih.gov/ij/">http://imagej.nih.gov/ij/</ext-link>).</p>