<p>A genome-wide association (GWA) study was performed in a total of 2,828 subjects of European ancestry with HF from seven samples collected within five large community-based prospective cohort studies including the Atherosclerosis Risk in Communities (ARIC and ARIC2) Study, the Cardiovascular Health Study (CHS), the Framingham Study (FHS), the Health ABC (Health ABC) study and the Rotterdam Study (RS and RS2). Sample characteristics, data collection and clinical definitions have been described previously and are summarized in <bold><xref ref-type="supplementary-material" rid="pgen.1006034.s001">S1 Text</xref></bold>. [<xref ref-type="bibr" rid="pgen.1006034.ref041">41</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006034.ref046">46</xref>] First diagnosis of heart failure (new-onset) was ascertained using a variety of methods based on international published criteria, as detailed in <bold><xref ref-type="supplementary-material" rid="pgen.1006034.s009">S1 Table</xref></bold>. Mortality was ascertained from telephone contacts with relatives and from medical records, death certificates and/or municipal records (<bold><xref ref-type="supplementary-material" rid="pgen.1006034.s001">S1 Text</xref></bold>).</p><p>Genotyping was performed using commercially available assays for genome-wide SNP detection. Imputation of non-genotyped SNPs was performed using CEU reference panels of SNP correlations from the HapMap project phase II (<bold><xref ref-type="supplementary-material" rid="pgen.1006034.s001">S1 Text</xref>)</bold>, to characterize a total of 2.5 million SNPs. Imputation quality was assessed for each SNP from the ratio of observed over expected variance of allele dosage.</p><p>All-cause mortality following initial HF diagnosis was examined for association with additive allele dosage of each genotyped or imputed SNP using Cox proportional hazards models, with censoring at the end of or loss to follow-up. Models were adjusted for age at diagnosis, sex, and recruitment site in multicenter cohorts. In the family-based FHS, Cox models were implemented with clustering on pedigree to account for relatedness. Genomic control was applied to results from each cohort. Cohort-specific GWA results were combined using fixed effects meta-analysis with inverse variance weights. SNPs were excluded from cohort-level analyses if exhibiting implausible beta coefficients (&lt; -5 or &gt; 5) and from the meta-analysis for low heterozygosity (sample size-weighted minor allele frequency &#8804; 0.03, corresponding to &lt; 100 minor allele carriers with an endpoint).</p><p>SNPs passing a <italic>P</italic>-value threshold defined <italic>a priori</italic> as <italic>P</italic> &lt; 5.0x10<sup>-7</sup> in the genome-wide stage 1 were carried forward to the second stage with targeted genotyping in 1,870 HF patients from four independent cohorts. For 2.5 million tests, this threshold limits the expected number of genome-wide false positives to approximately 1, assuming statistical independence of tests. The second stage included four independent cohorts; the Malm&#246; Diet and Cancer Study (MDCS), the Malm&#246; Preventive Project (MPP), the Physicians&#8217; Health Study (PHS) and the Prospective Study of Pravastatin in the Elderly at Risk (PROSPER) [<xref ref-type="bibr" rid="pgen.1006034.ref047">47</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006034.ref050">50</xref>]. Heart failure ascertainment and time of death in these cohorts was similar to in stage 1 cohorts, as shown in <bold><xref ref-type="supplementary-material" rid="pgen.1006034.s001">S1 Table</xref></bold> and <bold><xref ref-type="supplementary-material" rid="pgen.1006034.s001">S1 Text</xref></bold>. Genotyping was performed as outlined in <bold><xref ref-type="supplementary-material" rid="pgen.1006034.s001">S1 Text</xref></bold>. Association analyses and meta-analysis of results were performed as in the first stage. Meta-analysis of stage 1 and 2 was performed, and a combined <italic>P</italic>-value &lt; 5.0x10<sup>-8</sup> was considered significant. Heterogeneity was assessed across the combined stage 1 and 2 cohorts using Cochran&#8217;s Q test for heterogeneity, computed as the sum of the squared deviations of each study&#8217;s effect from the weighted mean over the study variance, and the I<sup>2</sup> test, the percentage of total variation across studies that is due to heterogeneity rather than chance (I<sup>2</sup> = [Q&#8212;df] / Q) [<xref ref-type="bibr" rid="pgen.1006034.ref051">51</xref>, <xref ref-type="bibr" rid="pgen.1006034.ref052">52</xref>].</p><p>The association of replicated SNPs with measures of cardiac structure and function was evaluated from summary results of the following GWA consortia: EchoGen [<xref ref-type="bibr" rid="pgen.1006034.ref019">19</xref>], CHARGE-HF [<xref ref-type="bibr" rid="pgen.1006034.ref020">20</xref>], CHARGE-QRS [<xref ref-type="bibr" rid="pgen.1006034.ref022">22</xref>], natriuretic peptides in 5453 subjects from the Malm&#246; Diet and Cancer study [<xref ref-type="bibr" rid="pgen.1006034.ref021">21</xref>], QT-IGC [<xref ref-type="bibr" rid="pgen.1006034.ref023">23</xref>], and the CHARGE Sudden Cardiac Death consortium (manuscript in preparation). Each of these consortia is described in <bold><xref ref-type="supplementary-material" rid="pgen.1006034.s001">S1 Text</xref></bold>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The correlation of replicated SNPs with known coding SNPs was examined in the databases for the 1000 Genomes Project and phase III of the HapMap project, using SNAP [<xref ref-type="bibr" rid="pgen.1006034.ref053">53</xref>]. The location of SNPs in relation to regulatory motifs was explored using histone methylation patterns generated as part of the ROADMAP Epigenomics project [<xref ref-type="bibr" rid="pgen.1006034.ref024">24</xref>]. Enhancers were identified in each of the 129 ROADMAP tissues using the ChromHMM algorithm [<xref ref-type="bibr" rid="pgen.1006034.ref054">54</xref>] from patterns of monomethylation (H3K4Me1) of the fourth residue (lysine) and acetylation of the 27th residue (H3K27Ac) of histone H3. The location of SNPs in relation to transcription factor binding sites was assessed in silico using HaploReg version 4.1 (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/mammals/haploreg/haploreg.php">http://www.broadinstitute.org/mammals/haploreg/haploreg.php</ext-link>) [<xref ref-type="bibr" rid="pgen.1006034.ref055">55</xref>] and the UCSC Genome Browser (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu/">http://genome.ucsc.edu</ext-link>). In HaploReg, position weight matrices (PWMs; probabilistic representations of DNA sequence) were computed with p-values based on literature sources and ENCODE ChIP-Seq experiments as previously described [<xref ref-type="bibr" rid="pgen.1006034.ref055">55</xref>], and only instances where a motif in the sequence passed a threshold of <italic>P</italic> &lt; 4<sup>&#8722;7</sup> were considered. The NHLH1-binding motif was retrieved into HaploReg from the manually curated TRANSFAC database.</p><p>Complementary DNA oligonucleotides corresponding to the 100 bp genomic region flanking rs9885413 (50 bp on either side of the SNP) were cloned into the luciferase reporter vector pGL3-Promoter (Promega, Madison, WI) using the MluI and BglII sites. Two different sets of oligos were cloned, one corresponding to the major allele of rs9885413 (pGL3P-G) and one to the minor allele (pGL3P-T). Oligonucleotide sequences were as following: major allele sense: CGCGTCCTGCCTCACATAATCTTTTTGTTTGTCCCCCTGAAATGGATTCTCAGCTGTTGCCCAAACATTTCATCTTAGCGTTCCAGGTTTGAACTCGCCCTCACGA, minor allele sense: CGCGTCCTGCCTCACATAATCTTTTTGTTTGTCCCCCTGAAATGTATTC TCAGCTGTTGCCCAAACATTTCATCTTAGCGTTCCAGGTTTGAACTCGCCCTCACGA, and the corresponding antisense sequences. The reporter vectors were co-transfected with the pRL-null vector at a ratio of 10:1 into HEK293 cells using Lipofectamine LTX (Life Technologies) according to the manufacturer&#8217;s instructions. 24 hours post-transfection, luciferase activity was assayed using the Dual-Luciferase Reporter Assay System (Promega) and Glomax 20/20 Luminometer (Promega). The signal from the reporter vector was normalized to the signal from the pRL-null vector.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Samples of left ventricular cardiac tissue from patients undergoing cardiac surgery were genotyped for the SNP rs9885413 and for all five transcripts within +/- 500 kb of the SNP. Samples of cardiac tissue were acquired from patients from the MAGNet consortium (<ext-link ext-link-type="uri" ns0:href="http://www.med.upenn.edu/magnet/">http://www.med.upenn.edu/magnet/</ext-link>). Gene expression levels were determined using the Affymetrix ST1.1 gene expression array (Affymetrix, Santa Clara, CA, USA) in a cohort including 247 heart samples. Genotyping was performed using the Illumina OmniExpress array. Left ventricular free-wall tissue was harvested at time of cardiac surgery from subjects with heart failure undergoing transplantation or from unused transplant donors. In all cases, the heart was perfused with cold cardioplegia prior to cardiectomy to arrest contraction and prevent ischemic damage. Tissue specimens were then obtained and frozen in liquid nitrogen. Genomic DNA from left ventricle was extracted using the Gentra Puregene Tissue Kit (Qiagen) according to manufacturer&#8217;s instruction.</p><p>Total RNA was extracted from left ventricle using the miRNeasy Kit (Qiagen) including DNAse treatment on column. RNA concentration and quality was determined using the NanoVue Plus spectrophotometer (GE Healthcare) and the Agilent 2100 RNA Nano Chip (Agilent). For all samples, genome-wide SNP genotypes were generated using the Illumina OmniExpress Array. Caucasian Ancestry was verified using multi-dimensional scaling of genotypes. For Gene expression array experiments, the Affymetrix ST1.1 Gene array was used. Data were normalized using the Robust Multi-array Average algorithm and batch effects were adjusted for using ComBat. Transcript expression levels were considered significantly higher than background noise if expression values from robust multiarray analysis in at least 10% of either cases or controls exceeded of the 80% quantile of expression of genes on the Y-chromosome in female hearts (5.24). Associations of expression levels for expressed genes with SNP genotypes were tested using a likelihood ratio test. Specifically, we fit a linear regression model Y = &#946;0 + &#946;1*D + &#946;2*g + &#946;3*(g x D) where Y is the log<sub>2</sub> transformed expression level of a given probe, g is the genotype (coded as 0, 1, and 2) of the test SNP, and D is heart failure disease status (D = 1 for heart failure cases and D = 0 for unused donor controls). Association between the probe and test SNP was assessed by testing H0: &#946;2 = &#946;3 = 0 using a likelihood ratio test. Significance of the test statistic was evaluated by comparing with a Chi-squared distribution with two degrees of freedom. All models were additionally adjusted for age, gender, and study site.</p><p>The association of the SNP rs9885413 with DNA methylation was examined in 2408 participants from the FHS Offspring cohort. Methylation at cytosine-guanine dinucleotides (CpG) at the 5q22 locus (+/-500 kb from rs9885413) were ascertained from a gene-centric DNA methylation array (Infinium HumaMethylation450 BeadChip, Illumina, San Diego, CA, USA) which allows interrogation of 485,512 methylation sites across the genome. The array has coverage of at least one methylation site near 99% of RefSeq genes and 96% of CpG islands. Briefly, bisulfite-treated genomic DNA (1 &#956;g) from peripheral blood samples underwent whole-genome amplification, array hybridization and scanning according to manufacturer instructions. Genotyping of rs9885413 was performed as described in <bold><xref ref-type="supplementary-material" rid="pgen.1006034.s001">S1 Text</xref></bold>. Association of rs9885413 and the methylation probe cg02061660 with expression of the five genes at the locus (+/-500 kb from rs9885413) was examined from microarray data (Affymetrix Human Exon Array ST 1.0) in 5257 participants from the FHS Offspring cohort and Third Generation cohort. Procedures for RNA extraction, processing and analysis have been described previously (28). Linear mixed effect (LME) models were fit accounting for familial correlation, cell count heterogeneity and technical covariates to account for batch effects using the pedigreemm package in R [<xref ref-type="bibr" rid="pgen.1006034.ref056">56</xref>]. Specifically, the mQTL model utilized a two-step approach: first, the DNA methylation beta-value (ratio of methylated probe intensity to total probe intensity) was residualized with adjustment for age, sex, cell count proportions (imputed using the Houseman method for granulocytes, monocytes, B-lymphocytes, CD4+ T lymphocytes, CD8+ T lymphocytes and NK cells) [<xref ref-type="bibr" rid="pgen.1006034.ref057">57</xref>], measured technical covariates (row, chip, column), and the family structure covariance matrix. Second, DNA methylation residuals were specified as dependent variable, SNP genotype dosage as independent variable with additional adjustment for 558 SVAs (surrogate variable analysis) [<xref ref-type="bibr" rid="pgen.1006034.ref058">58</xref>] and ten principal components from eigenstrat [<xref ref-type="bibr" rid="pgen.1006034.ref059">59</xref>] to account for unmeasured batch effects. The eQTL models similarly residualized gene expression with adjustment for age, sex, imputed cell count proportions (imputed in Offspring Cohort participants utilizing gene expression markers of cell count proportions developed from the Third Generation participants with both gene expression and measured complete blood counts), and family structure covariance matrix. The residual of gene expression was specified as dependent variable and SNP dosage as independent variable adjusted for 20 PEER (probabilistic estimation of expression residuals) factors [<xref ref-type="bibr" rid="pgen.1006034.ref060">60</xref>] to account for unmeasured technical and batch effects in the gene expression data. The eQTM models specified gene expression residual as dependent variable and DNA methylation residual as independent variable adjusted for 20 methylation SVAs and 20 expression SVAs to account for unmeasured technical and batch effects.</p><p>Replication of the association of rs9885413 with cg02061660 including the same covariates in the model as in FHS was attempted in blood samples from 750 randomly selected participants of the Rotterdam study (RS3) not included in the GWA stage, where information from the same DNA methylation array as FHS was available. DNA was extracted, bisulfite-treated using the Zymo EZ-96 DNA-methylation kit (Zymo Research, Irvine, CA, USA) and hybridized to arrays according to manufacturer instructions. During quality control samples showing incomplete bisulfite treatment were excluded (n = 5) as were samples with a low detection rate (&lt;99%) (n = 7), or gender swaps (n = 4). Probes with a detection <italic>P</italic>-value&gt;0.01 in &gt;1% samples, were filtered out. A total number of 474,528 probes passed the quality control and the filtered &#946; values were normalized with DASEN implemented in the <italic>wateRmelon</italic> package in R statistical software. Genotyping was performed using the Illumina 610quad array. Cell counts were estimated using the same method as in FHS and also directly measured on a Coulter AcT Diff II Hematology Analyzer (Beckman Coulter, Brea, CA) for granulocytes, monocytes, lymphocytes). Models including both estimated and directly measured cell counts were explored.</p><p>HEK293 cells were seeded at 100,000 cells/well in a 6-well plate the day before transfection. Cells were transfected using Lipofectamine and 50 nM of siRNA designed to target human NHLH1 or negative control siRNA (Life Technologies, Carlsbad, CA, USA) according to the manufacturer&#8217;s instructions. After 48 hours, cells were harvested and total RNA extracted using the miRNeasy Mini Kit (Qiagen, Hilden, Germany) according to the manufacturer&#8217;s instructions. cDNA was synthesized using the RevertAid H- First Strand cDNA Synthesis Kit (Thermo Fischer Scientific, Waltham, MA, USA) using random hexamer primers and qPCR was performed with TaqMan assays for NHLH1, TMEM232, SLC25A4, WDR36, TSLP, CAMK4 and GAPDH on a StepOne Plus Real-Time PCR System (Life Technologies). Gene expression was normalized to GAPDH and expressed relative to cells transfected with negative control siRNA according to the &#916;&#916;Ct-method [<xref ref-type="bibr" rid="pgen.1006034.ref061">61</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The frequencies of ancestral and derived alleles of rs9885413 were examined in populations from the International HapMap Project (<ext-link ext-link-type="uri" ns0:href="http://www.hapmap.org/">http://www.hapmap.org/</ext-link>) [<xref ref-type="bibr" rid="pgen.1006034.ref062">62</xref>] and the Human Genome Diversity Project (HGDP, <ext-link ext-link-type="uri" ns0:href="http://hagsc.org/hgdp/">http://hagsc.org/hgdp/</ext-link>) [<xref ref-type="bibr" rid="pgen.1006034.ref063">63</xref>]. The fixation index (F<sub>st</sub>) was estimated as described by Weir and Cockerham [<xref ref-type="bibr" rid="pgen.1006034.ref064">64</xref>], based on allele frequencies in HapMap stage II as also previously described [<xref ref-type="bibr" rid="pgen.1006034.ref065">65</xref>]. The integrated haplotype score (iHS) was calculated from HapMap stage II data as described by Voight et al (<ext-link ext-link-type="uri" ns0:href="http://haplotter.uchicago.edu/">http://haplotter.uchicago.edu/</ext-link>) [<xref ref-type="bibr" rid="pgen.1006034.ref033">33</xref>]. Allele frequency distributions in HGDP populations were visualized using the HGDP selection browser (<ext-link ext-link-type="uri" ns0:href="http://hgdp.uchicago.edu/">http://hgdp.uchicago.edu/</ext-link>) [<xref ref-type="bibr" rid="pgen.1006034.ref066">66</xref>].</p><p>Informed consent was obtained from all participants and all contributing studies were approved by the respective ethics committee as described in <bold><xref ref-type="supplementary-material" rid="pgen.1006034.s001">S1 Text</xref></bold>.</p>