<p>The <italic>Xpg</italic> targeting construct was generated using multiple elements. First, a cassette consisting of a Neomycin (NEO) resistance marker, flanked by <italic>Frt</italic> sites, and followed by a single <italic>LoxP</italic> site was cloned into a modified pBlueScript SK+ vector containing a PGK-DTA negative selection marker, making use of a klenow blunted ApaI (insert)/XbaI(vector) and a NotI restriction site. Second, <italic>Xpg</italic> homologous arms were PCR amplified from C57BL6 genomic DNA (originating from BAC clone RP24-343K18) and cloned into the same plasmid. The following primers (non-homologous regions indicated in italics; the <italic>LoxP</italic> sequence is underlined) were used for amplification of the 5&#8242; and 3&#8242; arm, respectively: LAF2 (<named-content content-type="gene">5&#8242;-<italic>CGCACCCGGG</italic>TGTGATCCTGTGGTCCTGTAGT-3&#8242;</named-content>) and LAR2 (<named-content content-type="gene">5&#8242;-<italic>CCATCGAT</italic>ATCCTCAGAAAGGTATCTCTTAAGCA-3&#8242;</named-content>), yielding a 3.2-kb XmaI-ClaI fragment; RAF1 (<named-content content-type="gene">5&#8242;-<italic>CCCTGCTAGC</italic>GGGATGAGGAATCGTGACTAAGGAG-3&#8242;</named-content>) and RAR1 (<named-content content-type="gene">5&#8242;-<italic>CCGCAGCGGCCGC</italic>AAACAAGGGACCCAAATGTAGGCT-3&#8242;</named-content>), yielding a 2.0-kb NheI-NotI fragment, where the restriction sites were introduced in the PCR primers. Last, the third exon of <italic>Xpg</italic> followed by a PCR-introduced <italic>LoxP</italic> site was amplified using the primers Ex3LoxF2 (<named-content content-type="gene">5&#8242;-<italic>GGGAACCG</italic>GTTTGAGTGTCCTTGGTGACAGG-3&#8242;</named-content>) and Ex3LoxR2 (<named-content content-type="gene">5&#8242;-<italic>CCCTGCTAGC</italic>
<underline>ATAACTTCGTATAGCATACATTATACGAAGTT</underline> ATCC-3&#8242;</named-content>), yielding a 350-bp AgeI-NheI fragment, which was inserted between the neomycin cassette and the 5&#8242; homology arm.</p><p>Next, a total of 10 &#181;g of NotI-linearized targeting vector was electroporated to Ola129 ES cells, and the targeted clones were selected through the use of the Neomycin selection marker (G418 200 &#181;g/ml). Clones resistant for G418 were initially screened by PCR, using a forward primer in exon 3 (F3 <named-content content-type="gene">5&#8242;-GAGACAGGCTCTGAAAACTGCTT-3&#8242;</named-content>) and a reverse primer outside the 3&#8242; homologous region (R3 <named-content content-type="gene">5&#8242;-CACTGAACAAACAAGGGACCCAAA-3&#8242;</named-content>). ES clones showing a 2.2-kb fragment in addition to the wild type 2.3-kb fragment after NheI digestion of the PCR product were further screened by Southern blot. ES genomic DNA was digested with EcoRI and hybridized with a 0.9-kb DpnI restriction fragment from BAC RP24-343K18, spanning the 2<sup>nd</sup> exon of <italic>Xpg</italic>. The probe hybridizes to a 7.4-kb fragment in wild type DNA and to an additional 4.1-kb fragment in targeted DNA.</p><p>ES cells from two independent targeted clones were micro-injected into C57BL6 blastocysts. Heterozygous mutant mice were generated by crossing the male chimeras with C57BL6 females and verified by coat color and PCR genotyping. The Neomycin (NEO) resistance gene was flanked by <italic>Frt</italic> sites to allow specific elimination of this dominant selectable marker by an <italic>Flp</italic> recombinase to avoid potential undesired influence of the <italic>Neo</italic> gene on <italic>Xpg</italic> transcription or mRNA processing. The NEO cassette was removed by crossing mice carrying the targeted allele with <italic>Cag-Flp</italic> recombinase FVB/N transgenic animals <xref ref-type="bibr" rid="pgen.1004686-Schaft1">[121]</xref>. These mice carry the floxed allele, and are referred to as <italic>Xpg<sup>f</sup></italic> throughout this paper. Thereafter, the F<sub>3</sub> offspring was crossed with a <italic>Cag-Cre</italic> C57BL6 transgenic <xref ref-type="bibr" rid="pgen.1004686-Sakai1">[73]</xref>, resulting in Cre-mediated recombination and excision of the third exon. <italic>Xpg<sup>+/&#8722;</sup></italic> animals were backcrossed to C57BL6 and FVB/N in parallel, at least ten times, and interbred to obtain C57BL6, FVB/N and C57BL6/FVB F1 hybrid <italic>Xpg<sup>&#8722;/&#8722;</sup></italic> mice. To achieve liver specific <italic>Xpg</italic> gene inactivation, a transgenic line with Cre recombinase under the control of the albumin promoter (hereafter referred to as <italic>Alb-Cre</italic>) was used <xref ref-type="bibr" rid="pgen.1004686-Postic1">[95]</xref>. Female <italic>Alb-Cre<sup>+</sup></italic> mice were crossed with male <italic>Xpg</italic>
<sup>+/&#8722;</sup> mice (both in a C57BL6 background). Female <italic>Xpg<sup>+/&#8722;</sup> Alb-Cre<sup>+</sup></italic> mice, obtained from these breedings, were crossed with male <italic>Xpg</italic>
<sup>f/f</sup> FVB/N mice to yield hybrid <italic>Xpg</italic>
<sup>f/&#8722;</sup>
<italic>Alb-Cre<sup>+</sup></italic> mice. <italic>Xpg</italic>
<sup>f/&#8722;</sup>
<italic>Alb-Cre<sup>+</sup></italic> mice (in a C57BL6/FVB F1 hybrid background) are heterozygous for <italic>Xpg</italic> in their entire body, except for the hepatocytes in the liver, which are homozygous for <italic>Xpg</italic> after Cre excision of the floxed allele. All littermates, with and without Cre-recombinase expression were used as controls (referred to as <italic>wt</italic>).</p><p>FVB/N <italic>Xpg<sup>f/f</sup></italic> animals were similarly bred to the female offspring from C57BL6 <italic>Xpg</italic>
<sup>+/&#8722;</sup> and C57BL6 <italic>Emx1-Cre</italic> mice <xref ref-type="bibr" rid="pgen.1004686-Iwasato1">[100]</xref> to obtain forebrain specific XPG knock-out animals (referred to as <italic>Emx1-Xpg</italic>). All animals used in the studies described in this paper were of the same C57BL6/FVB F1 hybrid background (unless otherwise stated) and had <italic>ad libitum</italic> access to water and standard mouse food (CRM pellets, SDS BP Nutrition Ltd.; gross energy content 4.39 kcal/g dry mass, digestible energy 3.2 kcal/g or AIN93G synthetic pellets, Research Diet Services B.V.; gross energy content 4.9 kcal/g dry mass, digestible energy 3.97 kcal/g). Since the <italic>Xpg</italic> knockout animals were smaller, food was administered within the cages and water bottles with long nozzles were used from around two weeks of age. Experiments were performed in accordance with the Principles of Laboratory Animal Care (National Institutes of Health publication no. 86-23) and with the guidelines approved by the Erasmus University Animal Care Committee.</p><p>For PCR genotyping the following primers were used: F1 forward primer (<named-content content-type="gene">5&#8242;-TCTGTTTAGGTGGTGCCCATTT-3&#8242;</named-content>) annealing 5&#8242; of the third exon of <italic>Xpg</italic>; NeoF forward primer (<named-content content-type="gene">5&#8242;-GCTTCCTCGTGCTTTACGGTAT-3&#8242;</named-content>) located in the Neomycin resistance marker; R1 reverse primer (<named-content content-type="gene">5&#8242;-CGACAGCACTTCTTTCTCCTTAGT-3&#8242;</named-content>) annealing 3&#8242; of the third exon of <italic>Xpg</italic>. A 711-bp fragment was generated from the targeted allele using primers NeoF and R1, whereas a 495-bp and 227-bp fragment are amplified from the wild type and knockout allele, respectively, using the F1/R1 primerset. Cycling conditions were 95&#176;C for 45 sec, 58&#176;C for 45 sec, 72&#176;C for 1 min (35 cycles), followed by an extension at 72&#176;C for 5 min.</p><p>Total RNA was extracted from wild type and <italic>Xpg<sup>&#8722;/&#8722;</sup></italic> liver using TRIzol reagent and reverse transcribed with SuperScript II Reverse Transcriptase (Life technologies), according to the manufacturer's instructions, to generate cDNA. A 0.4-kb PCR fragment ranging from the 2<sup>nd</sup> to 6<sup>th</sup> of <italic>Xpg</italic> was produced using the following primers: Ex2F (<named-content content-type="gene">5&#8242;-GCTCATCTTCTCACATTATTCC-3&#8242;</named-content>) and Ex6R (<named-content content-type="gene">5&#8242;- GGTAAACTCTTTCATGTCAGTC-3&#8242;</named-content>) and analyzed by Sanger sequencing.</p><p>The mice were weighed and visually inspected weekly, and were scored for the onset of various phenotypical parameters. Clasping was measured by suspending mice from their tails for 20 seconds. A clasping event was scored when retraction of both hind limbs towards the body was observed for at least 5 seconds. Whole body tremor was scored if mice were trembling for a combined total of at least 10 seconds, when put on a flat surface for 20 seconds. Mice showing an abnormal curvature of the spine were scored as having kyphosis.</p><p>Primary MDFs were isolated from the tail of 12&#8211;14 week old <italic>Xpg</italic>
<sup>&#8722;/&#8722;</sup> animals and wild type littermates. Minced tail skin was immersed in F10/DMEM culture medium supplemented with 20% fetal calf serum, 50 &#181;g/ml penicillin/streptomycin, and 1.6 mg/ml type II collagenase (Gibco, Life Technologies). After incubation at 37&#176;C for 24 hours, MDFs were filtered through a 40 &#181;m cell strainer, centrifuged for removal of the collagenase, and cultured at 37&#176;C, 5% CO<sub>2</sub>, and 3% O<sub>2</sub>.</p><p>Whole cell extracts were prepared from cultured MDFs using cells isolated from four wild type mice and four <italic>Xpg<sup>&#8722;/&#8722;</sup></italic> littermates. Proteins from 30 &#181;l of each extract were separated by electrophoresis on 7% SDS-PAGE gels and transferred overnight onto a nitrocellulose membrane. As previously described <xref ref-type="bibr" rid="pgen.1004686-Sarker1">[31]</xref>, XPG protein was detected with a rabbit polyclonal antibody designated R2 (97727) that was raised against a conserved peptide from the spacer region (R-domain) of XPG corresponding to residues 267&#8211;281 of the human protein, which are identical with the same residues in mouse XPG except for amino acid 267, which is E in the human protein but Q in mouse. Whole cell extract from human embryonic kidney 293 cells was used as a positive control for XPG protein. As a loading control, tubulin was detected using a commercial antibody.</p><p>MDFs were seeded in triplicates at equal densities and treated 24 h after seeding as indicated. After 48 h recovery cell survival was determined by cell count on Beckman Coulter, Z2 Coulter particle count and size analyzer.</p><p>UDS was performed using the Click-iT EdU imaging kit (Life technologies). MDFs were seeded on coverslips and 24 h later washed with PBS and irradiated with 16 J/m<sup>2</sup> UV-C (Philips) or mock treated. Cells were directly labelled for 3 h in thymidine-free Ham's F10 medium supplemented with 10% dialyzed serum, 50 &#181;g/ml penicillin/streptomycin, 20 &#181;M Ethynyl-deoxyuridine and 1 &#181;M Fluoro-desoxyuridine. After a PBS wash, the cells were chased with 10 &#181;M non-labelled thymidine in normal medium for 15 minutes and fixed with 3.7% formaldehyde. Slides were washed 3&#215; with PBS/3%BSA and permeabilized with PBS/0.5%Triton-X100 for 20 min. The Click-iT reaction, linking azide-conjugated Alexa dye to ethynyl groups was performed for 30 min in a dark, humid environment. After 3&#215; PBS/3%BSA and 2&#215; PBS wash, the slides were mounted with vectashield containing DAPI (Vector Laboratories) to stain nuclei.</p><p>RRS was performed using the Click-iT EU imaging kit (Life Technologies). MDFs were seeded on coverslips and 8 h later washed with PBS and irradiated with 16 J/m<sup>2</sup> UV-C (Philips) or mock treated. After 14 h recovery, the cells were labelled for 2 h in 0.1 mM EU-containing medium (Ham's F10 medium supplemented with 10% dialyzed serum, 50 &#181;g/ml penicillin/streptomycin and 20 mM HEPES). After a PBS wash, the cells were fixed and processes as described for UDS.</p><p>Force-deflection curves from the left femora of 14-week-old and 16-week-old mice were acquired in a three-point bending assay using a Chatillon TCD series mechanical test frame (Technex BV, The Netherlands), equipped with 3 mm hemi-cylindrical supports with a 8.5 mm total span. Width between the supports was adjusted according to the anatomical landmarks of the femur, i.e. lesser trochanter and condyles. The femora were aligned such that the femoral head was in the horizontal plane and the posterior aspect of the condyles was facing down. All samples were preconditioned for five cycles to 2 Newton (N) at a rate of 0.6 mm/min before testing to failure at a rate of 0.1 mm/min. The obtained force-deflection curves were analyzed for bone strength (N/mm), which was represented by the &#916; force/deflection of the linear part of the curve.</p><p>
<italic>Xpg<sup>&#8722;/&#8722;</sup></italic> and wild type mice were sacrificed by cervical dislocation at scheduled ages (7, 14, 16, 18 weeks), femora were excised and non-osseous tissue was removed. Left femora were placed in PBS and stored at &#8722;20&#176;C until further use for mechanical testing and the right femora were fixated (4% formalin). Two days post-fixation, the right femora were scanned using the Skyscan 1076 in vivo X-Ray computed tomography (Skyscan, Kontich, Belgium) with a voxel size of 8.88 &#181;m. Osseous tissue was distinguished from non-osseous tissue by segmenting the reconstructed grayscale images with an automated algorithm using local thresholds <xref ref-type="bibr" rid="pgen.1004686-Waarsing1">[122]</xref>. The region of interest (ROI), i.e. distal metaphysis of the femora, was selected by using 3D data analysis software. To compensate for bone length differences between the <italic>Xpg<sup>&#8722;/&#8722;</sup></italic> and wild type mice, the length of each ROI was determined relative to the largest specimen femur of the cohort.</p><p>Cortex and trabeculae of the metaphysis were separated using in-house developed automated software. Thickness of the trabeculae and cortices were assessed using 3D analysis software as described <xref ref-type="bibr" rid="pgen.1004686-Botter1">[123]</xref> using the CT analyzer software package (Skyscan). A bone specimen with known bone morphometrics was included within each scan as a quantitative control.</p><p>TRIzol reagent (Life Technologies) was used to isolate total RNA from mouse tissue specimens. 4 &#181;g RNA was reverse transcribed using SuperScript II (Life Technologies). Real-time PCR was performed on a Bio-Rad CFX96 thermocycler using SYBR Green (Sigma-Aldrich) and Platinum Taq polymerase (Life Technologies). Generation of specific PCR products was confirmed by melting-curve analysis and gel electrophoresis. For data analysis, the induction of target cDNA was calculated by the method described by <xref ref-type="bibr" rid="pgen.1004686-Pfaffl1">[124]</xref>. p-values were determined using two-tailed t-tests. The used gene specific real-time PCR primers are listed in Supplementary <xref ref-type="supplementary-material" rid="pgen.1004686.s008">Table S1</xref>.</p><p>Mice were euthanized by CO<sub>2</sub> asphyxiation and blood was immediately collected from the heart. Glucose levels were measured using a Freestyle mini blood glucose meter. Albumin levels were measured in blood plasma using a mouse albumin ELISA kit (Immunology Consultants Laboratory, Inc.).</p><p>Primary antibodies (supplier; dilutions) used in this study were as follows: rabbit anti-Calbindin (Swant; 1&#8758;10,000); goat anti-ChAT (Millipore; 1&#8758;500); rabbit anti-GFAP (DAKO; 1&#8758;8,000); mouse anti-GM130 (BD Transduction; 1&#8758;100); rabbit anti-Hsp25 (Enzo; 1&#8758;8,000); rabbit anti-Iba-1 (Wako; 1&#8758;5,000); rat anti-Ki67 (DAKO; 1&#8758;200); rat anti-Mac2 (Cedarlane;1&#8758;2,000); mouse anti-NeuN (Millipore; 1&#8758;1,000); rabbit anti-p53 (Leica; 1&#8758;1,000). For avidin&#8211; biotin&#8211;peroxidase immunocytochemistry biotinylated secondary antibodies from Vector Laboratories, diluted 1&#8758;200 were used. FITC-, Cy3-, and Cy5-conjugated secondary antibodies raised in donkey (Jackson ImmunoResearch) diluted at 1&#8758;200 were used for confocal immunofluorescence.</p><p>TUNEL staining on liver, brain and retina: To quantify apoptotic cells, specimens were fixed overnight in 10% buffered formalin, paraffin-embedded, sectioned at 5 &#181;m, and mounted on Superfrost Plus slides. Paraffin sections were employed for TdT-mediated dUTP Nick-End Labeling (TUNEL) assay using a commercial kit (Apoptag Plus Peroxidase in situ apoptosis detection kit, Millipore). Sections were deparaffinized and incubated as described by the manufacturer. HE staining on liver, skin and small intestine: Liver, skin and intestine specimens were processed using the same fixation and sectioning methods as described for the TUNEL staining. Paraffin sections were deparaffinized, rehydrated in decreasing concentrations of ethanol and stained with haematoxilin and eosin.</p><p>For immunohistochemistry, paraffin sections of liver and intestine specimens were deparaffinized, rehydrated in decreasing concentrations of ethanol, treated for 10 minutes with 3% H<sub>2</sub>O<sub>2</sub> to quench endogenous peroxidase activity and heated to 100&#176;C for 1 h in 10 mM sodium citrate buffer, pH 6, for antigen retrieval. The amount of damaged (p53) and proliferating (Ki67) cells were subsequently detected using the avidin&#8211;biotin&#8211;immunoperoxidase complex method (ABC, Vector Laboratories, USA) with diaminobenzidine (0.05%) as chromogen.</p><p>Gelatin sections of brain and spinal cord: Mice were anaesthetized with pentobarbital and perfused transcardially with 4% paraformaldehyde. Brain and spinal cord were dissected out, post-fixed for 1 h in 4% paraformaldehyde, cryoprotected, embedded in 12% gelatin, rapidly frozen, and sectioned at 40 &#181;m using a freezing microtome or stored at &#8722;80&#176;C until use. Frozen sections were processed free floating using the ABC method (ABC, Vector Laboratories, USA) or single-, double-, and triple-labelling immunofluorescence. Immunoperoxidase-stained sections were analyzed and photographed using an Olympus BX40 microscope. Immunofluorescence sections were analyzed using a Zeiss LSM700 confocal.</p><p>Average time spent on an accelerating rotarod (Ugo Basile). <italic>Xpg<sup>&#8722;/&#8722;</sup></italic> mice and wild type controls were given four consecutive trials of maximally 5 minutes with inter-trial intervals of 1 hour. <italic>Emx1-Xpg</italic> mice were given two trials per day with a 1 hour inter-trial interval for four consecutive days.</p><p>Grip strength was determined by placing mice with forelimbs or all limbs on a grid attached to a force gauge, and steadily pulling the mice by their tail. Grip strength is defined as the maximum strength produced by the mouse before releasing the grid. For each value the test is performed in triplicate.</p><p>This study was performed in strict accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health. The protocol was approved by the Committee on the Ethics of Animal Experiments of the Erasmus MC (Permit Numbers: 139-03-08, 139-09-03, 139-12-18).</p>