<p>Farmed Asian seabass (<italic>Lates calcarifer</italic>) were obtained from the Marine Aquaculture Centre (Singapore). All experiments were approved by Agri-food and Veterinary Authority (AVA) Institutional Animal Care and Use Committee (IACUC) (approval ID: AVA-MAC-2012-02) and performed according to guidelines set by the National Advisory Committee on Laboratory Animal Research (NACLAR) for the care and use of animals for scientific research in Singapore.</p><p>Backcrossed and inbred fish were genotyped to identify the fish which exhibited the least heterozygosity. Individuals displaying a range of 5&#8211;50% homozygosity could be identified based on the markers used for genotyping [<xref ref-type="bibr" rid="pgen.1005954.ref010">10</xref>]. A backcrossed individual (BC-8) with decreased genomic polymorphism (DGP; ~40% based on the markers analyzed) was chosen and sacrificed to collect blood, as well as the majority of organs to serve as a resource for the Asian Seabass Genome Project.</p><p>We estimated the genome characteristics using a k-mer analysis of the raw Illumina sequencing data and estimated the genome wide rate of heterozygosity by evaluating the shape of the k-mer frequency distribution.</p><p>We counted the frequency of all k-mers (k = 21) in the data using Jellyfish [<xref ref-type="bibr" rid="pgen.1005954.ref074">74</xref>]. The observed k-mer distribution was bimodal with distinct peaks at ~36x and twice this amount at ~72x coverage (<xref ref-type="supplementary-material" rid="pgen.1005954.s002">S1 Fig</xref>), which is characteristic for a heterozygous genome [<xref ref-type="bibr" rid="pgen.1005954.ref075">75</xref>]. We analyzed the distribution by fitting a mixture model of two negative binomial distributions centered at <italic>mu</italic> and <italic>mu*2</italic>, representing the heterozygous and homozygous k-mers, respectively (<xref ref-type="disp-formula" rid="pgen.1005954.e001">Eq 1</xref>). The negative binomial components generalize the Poisson distribution with an additional <italic>size</italic> parameter for the variance. The k-mer model also includes two scaling parameters <italic>s</italic><sub><italic>1</italic></sub> and <italic>s</italic><sub><italic>2</italic></sub> that depend on the genome size and the overall rate of heterozygosity.</p><p><bold><xref ref-type="disp-formula" rid="pgen.1005954.e001">Eq 1</xref>.</bold> k-mer coverage model for heterozygous diploid genomes.</p><p>We determined the parameters of the model using a nonlinear least-squares estimate with the <italic>nls</italic> function in R. The resulting values fit the data well, with the two peaks centered at 36.6x and 73.2x coverage and small residual error (<xref ref-type="supplementary-material" rid="pgen.1005954.s002">S1 Fig</xref>). This also naturally excludes the low coverage k-mers caused from sequencing error that should not be included in the genome size or heterozygosity estimates. The modeling allows us to estimate the number of non-repetitive heterozygous and homozygous k-mers as 96,126,339 and 491,559,122, respectively, by summing the densities for the two components of the model separately. Using these values, we further estimate the overall rate of heterozygosity to be 0.4%-0.5% (<xref ref-type="disp-formula" rid="pgen.1005954.e002">Eq 2</xref>).</p><p><bold><xref ref-type="disp-formula" rid="pgen.1005954.e002">Eq 2</xref>.</bold> Formula to determine rate of heterozygosity.</p><p>The number of heterozygous k-mers in half for both the numerator and denominator is divided in half to evaluate the haploid content. The number of heterozygous k-mers is divided by k to account for that heterozygous base will contribute k heterozygous k-mers (k = 21). This will slightly undercount heterozygous bases that are within K bases of each other, allowing for a range in the overall heterozygosity rate.</p><p>For generating genome sequence data, genomic DNA was isolated from the chosen individual using Qiagen GenomicTip100 (Qiagen, Hilden, DE) as per standard protocol. Genomic DNA from the blood was used for the construction of two separate BAC libraries in pCC1BAC vector (Amplicon Express, Pullman, WA, USA) altogether representing 12X genome coverage. Libraries comprised an <italic>Eco</italic> RI (36,864 clones) library with an average insert size of ~120 Kb and a <italic>Bam</italic> HI BAC library (36,864 clones) with an average insert size of ~115 Kb.</p><p>Sequence data for the genome project was obtained from multiple platforms which are summarized below:</p><p>Sequence data from the PacBio platform was generated in two phases:</p><p>Asian seabass brain genomic DNA (10 &#956;g) was used for generating a single ~10 kb insert size library which was sequenced using 77 SMRT cells (1X120 minutes movie) on PacBio RSII using C2-XL chemistry (DNA Link, Korea). The average data per SMRT cell was 264 Mb (30X coverage). An average read length of 4,498 bp and average base quality score of 0.83 was obtained for the sequence data. A total of 20.3 Gb of data representing ~30X coverage of the Asian seabass genome was generated.</p><p>Asian seabass kidney genomic DNA was used for generating a ~20 kb insert size library which was sequenced using 105 SMRT cells (1x180 minutes movie) on PacBio RSII using P5-C3 chemistry (DNALink, Seoul, Korea). All DNA was size selected (&gt;7 kb) using the Blue Pippin system (Sage Sciences, Beverly, MA, USA) and samples were sheared with g-TUBEs (Covaris, Woburn, MA, USA)). The average data per SMRT cell was 421 Mb (60X coverage). An average read length of 8.09 kb and average base quality score of 0.83 was obtained for the sequence data. A total of 42.14 Gb of data or a ~60X coverage of the Asian seabass genome was generated.</p><p>Asian seabass liver genomic DNA from the chosen individual was used for the construction of two (500 bp and ~750 bp) linear insert TruSeq (Illumina) libraries (as per standard protocol) and sequenced using HiSeq 1500, with one lane for each insert sized library (2X100). A total of 280 million pairs of reads were generated (~56 Gb data), representing 80X coverage of the Asian seabass genome.</p><p>The first 7.5 plates (each containing 384-wells) from each library (<italic>Bam</italic> H1 and <italic>Eco</italic> R1) were sequenced from both ends using M13 forward and reverse primers. The overall data quality of sequences was good, with a pass rate of ~95% and an average of 769 high quality (Phred20) bases per read. The total number of trimmed sequences from <italic>Eco</italic> R1 and <italic>Bam</italic> H1 library (Forward + Reverse) was 5,703 and 5,646, respectively, making a total of 11,349 sequences.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The primary genome assembly with 90x PacBio data was performed using HGAP algorithm followed by polishing using Quiver [<xref ref-type="bibr" rid="pgen.1005954.ref015">15</xref>]; as part of the Pacific Biosciences SMRTAnalysis pipeline. To facilitate genome assembly, a <italic>make</italic> script available at <ext-link ext-link-type="uri" ns0:href="https://github.com/pbjd/smrtmake">https://github.com/pbjd/smrtmake</ext-link> was used. The statistics of the 90X PacBio assembly are detailed in <xref ref-type="supplementary-material" rid="pgen.1005954.s016">S1 Table</xref>.</p><p>QUAKE [<xref ref-type="bibr" rid="pgen.1005954.ref076">76</xref>] and decontamination was performed on the Illumina Hiseq reads and assembled along with BAC end sequences using SOAPDenovo127mer [<xref ref-type="bibr" rid="pgen.1005954.ref077">77</xref>] (k-mer was set from 21&#8211;81). The statistics are summarized in <xref ref-type="supplementary-material" rid="pgen.1005954.s016">S1 Table</xref>.</p><p>For the assembly done at CSHL, instead of using the full Celera assembly pipeline [<xref ref-type="bibr" rid="pgen.1005954.ref078">78</xref>], the algorithms were used separately. BOGART [<xref ref-type="bibr" rid="pgen.1005954.ref079">79</xref>] was used for unitigging (to help resolve some of the repeat issues). PacBio&#8217;s HBAR-DTK [<xref ref-type="bibr" rid="pgen.1005954.ref015">15</xref>] was used to finalize the assembly. The HBAR-DTK software retained the singletons (unlike the Celera assembler [<xref ref-type="bibr" rid="pgen.1005954.ref078">78</xref>] which discards singletons and only keeps consensus contigs). There were totally 3,604 singletons in this assembly and after their removal, the number of contigs reduced to 4,223, the assembly statistics are summarized in <xref ref-type="supplementary-material" rid="pgen.1005954.s016">S1 Table</xref>.</p><p>The assembled seabass transcriptome [<xref ref-type="bibr" rid="pgen.1005954.ref016">16</xref>] was used for scaffolding the 90X PacBio genome assembly using the L_RNA_scaffolder tool [<xref ref-type="bibr" rid="pgen.1005954.ref017">17</xref>] (<xref ref-type="supplementary-material" rid="pgen.1005954.s016">S1 Table</xref>) resulting in the scaffolded genome assembly (v2).</p><p>The assembled genome was evaluated using different metrics described below:</p><p>The assembled genome was evaluated for completeness using the 248 core eukaryotic genes dataset (CEGs)[<xref ref-type="bibr" rid="pgen.1005954.ref018">18</xref>,<xref ref-type="bibr" rid="pgen.1005954.ref019">19</xref>] and for quality using QUAST [<xref ref-type="bibr" rid="pgen.1005954.ref080">80</xref>].</p><p>The 11,191 BAC End Sequences were used for aligning against the genome assembly (BLASTN [<xref ref-type="bibr" rid="pgen.1005954.ref081">81</xref>], minimum alignment length cut-off of 100bp and a threshold e-value of 1e-6). A base level comparison was also done wherein 7,783,146 bp bases representing 11,159 BES were compared (using BLASTN [<xref ref-type="bibr" rid="pgen.1005954.ref081">81</xref>]) to the genome assembly.</p><p>For both the 500 and 750 bp Illumina libraries (totaling to ~80X genome coverage), &gt;99% of the HiSeq paired-end reads mapped correctly to the genome assembly with &gt;97% of the reads in pairs with similarity cut off of 90% and aligned read length 90% (<xref ref-type="supplementary-material" rid="pgen.1005954.s004">S3 Fig</xref>).</p><p>A total of 772 unique marker sequences from the Asian seabass linkage map [<xref ref-type="bibr" rid="pgen.1005954.ref021">21</xref>] were BLAST-searched [<xref ref-type="bibr" rid="pgen.1005954.ref081">81</xref>] against our genome scaffold sequences, retaining the top five BLAST hits for further analyses. For each marker on the linkage group, we selected the best alignment based on marker sequence alignment coverage. If a marker sequence had more than one alignment to different regions of the genome, but with the same alignment coverage, both alignments were retained. Further to that, only alignments that had percentage identity &#8805;90% and marker alignment coverage &#8805;80% were kept. Finally, if a given genome scaffold had markers from multiple linkage groups aligned to it, a manual inspection was performed to select for the linkage group that was represented predominantly. This analysis resulted in 680 (88%) of the marker sequences having an alignment to the genome scaffolds (as shown in <xref ref-type="supplementary-material" rid="pgen.1005954.s012">S11 Fig</xref>).</p><p>Optical map data generation and the whole genome <italic>de novo</italic> assembly process have been described in detail earlier [<xref ref-type="bibr" rid="pgen.1005954.ref082">82</xref>&#8211;<xref ref-type="bibr" rid="pgen.1005954.ref084">84</xref>]. Briefly, high molecular weight (HMW) DNA was obtained from frozen blood of the same Asian seabass individual from which genome sequence information was obtained using OpGen&#8217;s blood processing protocol (OpGen, Gaithersburg, MD, USA). The resulting DNA was evaluated on OpGen&#8217;s Argus Whole Genome Mapping System and quality metrics were reviewed. The average molecule size obtained from this DNA prep was approximately 304 kb. <italic>XbaI</italic> was selected as a suitable enzyme for generation of the optical map data. A total of 11 high-density MapCards were selected from those generated by Argus. On an average, ~68,000 Single Molecular Restriction Maps (SMRM) were marked up on each card. Typically, only molecules longer than 250 kb (total 377,118 SMRMs) were used in the analysis. These SMRMs were assembled into genome-wide Maptigs by using OpGen&#8217;s Gentig software [<xref ref-type="bibr" rid="pgen.1005954.ref085">85</xref>,<xref ref-type="bibr" rid="pgen.1005954.ref086">86</xref>] by aligning SMRMs based on restriction map pattern using a greedy algorithm with limited backtracking for finding an almost optimal scoring set of Maptigs. Gentig also takes into consideration the possible errors of SMRMs such as standard deviation, digestion rate, false cut and missing cut during the assembly process. Totally, 104 Maptigs representing the whole genome of Asian seabass were generated. These assembled Maptigs were then aligned with scaffolds of the assembled genome sequences (&#8805;40 kb) to aid in the orientation and joining of these sequence scaffolds, resulting in the assignment of 577 contig sequences with a combined length of 486.38 Mb (<xref ref-type="supplementary-material" rid="pgen.1005954.s024">S9</xref>&#8211;<xref ref-type="supplementary-material" rid="pgen.1005954.s029">S14</xref> Tables).</p><p>The optical map data also identified 55 sequences that potentially contain mis-assemblies, as different parts of these sequences showed alignment onto different Maptigs (<xref ref-type="supplementary-material" rid="pgen.1005954.s030">S15 Table</xref>).</p><p>We first performed whole genome alignment of the primary Asian seabass genome assembly (HGAP contigs) with the chromosomal scale genome assemblies of European seabass [<xref ref-type="bibr" rid="pgen.1005954.ref022">22</xref>] and three-spined stickleback [<xref ref-type="bibr" rid="pgen.1005954.ref023">23</xref>] using the LAST alignment tool [<xref ref-type="bibr" rid="pgen.1005954.ref087">87</xref>]. The output MAF files were filtered for 1:1 ortholog alignments using single_cov2 [<xref ref-type="bibr" rid="pgen.1005954.ref088">88</xref>]. Subsequently, we combined pairwise alignments into multiple alignments using the multiz tool [<xref ref-type="bibr" rid="pgen.1005954.ref089">89</xref>]. We then used Ragout [<xref ref-type="bibr" rid="pgen.1005954.ref090">90</xref>] to infer the order of Asian sea bass contigs according to colinearity with <italic>G</italic>. <italic>aculeatus</italic> and <italic>D</italic>. <italic>labrax</italic> assuming a closer relationship of <italic>D</italic>. <italic>labrax</italic> and <italic>L</italic>. <italic>calcarifer</italic>. The ordered contigs were written into scaffold sequences and in a second iteration aligned with the <italic>D</italic>. <italic>labrax</italic> genome alone, which enabled us to find further contig links. After each iteration, the resulting order of contigs was manually checked to remove suspicious interchromosomal connections. Subsequently, we compared the results with data from optical mapping and removed contig links that were clearly not in agreement with the contig order from optical mapping. Resulting scaffolds that were supported by shared synteny and optical mapping were relatively large, contained most of the assembled <italic>L</italic>. <italic>calcarifer</italic> sequence and could be ordered into chromosomal sized sequences using information from the <italic>L</italic>. <italic>calcarifer</italic> genetic linkage map [<xref ref-type="bibr" rid="pgen.1005954.ref021">21</xref>].</p><p>We performed BLASTN [<xref ref-type="bibr" rid="pgen.1005954.ref081">81</xref>] alignment (min. alignment identity 95%) between neighbouring contigs (a,b) placed in the assembled chromosomes and found a large number of contig end (a) to contig start (b) overlaps (only +/+ strand overlaps were used). Gaps between overlapping contigs were then closed by trimming the overlap region from the contig (a) and concatenating contig (b) to it. This process was performed by custom scripts written in Linux AWK language (<xref ref-type="supplementary-material" rid="pgen.1005954.s031">S16</xref> and <xref ref-type="supplementary-material" rid="pgen.1005954.s032">S17</xref> Tables).</p><p>As described above, whole genome alignments of the final chromosome scale assembly (v3) of <italic>L</italic>. <italic>calcarifer</italic> with <italic>D</italic>. <italic>labrax</italic> and <italic>G</italic>.<italic>aculeatus</italic> genomes were performed by LAST [<xref ref-type="bibr" rid="pgen.1005954.ref087">87</xref>] and filtered by single_cov2. MAF output files were subsequently converted to the Satsuma format by custom scripts (AWK) and processed by the BlockDisplaySatsuma script from the Satsuma v1.17 package [<xref ref-type="bibr" rid="pgen.1005954.ref091">91</xref>] to result in coordinates of syntenic blocks between the different genomes. This process was done in two iterations. The removal of spurious very short blocks (&lt;6000 bp) after iteration 1 resulted in larger collinear Blocks after iteration 2. Syntenic blocks along the 24 <italic>L</italic>. <italic>calcarifer</italic> chromosomes were plotted using CIRCOS [<xref ref-type="bibr" rid="pgen.1005954.ref092">92</xref>]. Additionally, we plotted links between collinear blocks to underline if they were rearranged in <italic>D</italic>. <italic>labrax</italic> or <italic>G</italic>. <italic>aculeatus</italic>.</p><p>The assembled Asian seabass genome was masked based on known repeats using Repeat Masker [<xref ref-type="bibr" rid="pgen.1005954.ref093">93</xref>] and the RepBase vertebrate libraries [<xref ref-type="bibr" rid="pgen.1005954.ref024">24</xref>] (<xref ref-type="supplementary-material" rid="pgen.1005954.s017">S2 Table</xref>). <italic>De novo</italic> repeat masking was performed on the genome with WindowMasker [<xref ref-type="bibr" rid="pgen.1005954.ref094">94</xref>] and RepeatScout [<xref ref-type="bibr" rid="pgen.1005954.ref095">95</xref>].</p><p>A collection of tandem repeats were obtained using Tandem Repeat Finder (TRF) version 4.07 [<xref ref-type="bibr" rid="pgen.1005954.ref096">96</xref>] and post-processed [<xref ref-type="bibr" rid="pgen.1005954.ref097">97</xref>]. The parameters used for the TRF search were: maximum mismatch 5; maximum period size 2000, and default values for other parameters (<xref ref-type="supplementary-material" rid="pgen.1005954.s018">S3 Table</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Miniature inverted-repeat transposable elements, LTR elements and potential transposon ORFs were also determined from the whole genome assembly using MITE Hunter [<xref ref-type="bibr" rid="pgen.1005954.ref098">98</xref>], LTR-harvester [<xref ref-type="bibr" rid="pgen.1005954.ref099">99</xref>] and TransposonPsi (<ext-link ext-link-type="uri" ns0:href="http://transposonpsi.sourceforge.net/">http://transposonpsi.sourceforge.net/</ext-link>), respectively. The predicted genes from the Asian seabass genome were later searched against transposable elements obtained from the LTRharvest, Mites and TransposonPSI databases (e-value of 1e-06 and 80% percentage alignment length) to remove predicted genes that aligned with transposable elements. Transfer RNAs (tRNAs) were also searched by tRNAscan-SE [<xref ref-type="bibr" rid="pgen.1005954.ref100">100</xref>].</p><p>The consensus sequences for repetitive families generated by RepeatScout (8,248 repeat families; see <xref ref-type="supplementary-material" rid="pgen.1005954.s021">S6 Table</xref> for details) were classified using TEclass [<xref ref-type="bibr" rid="pgen.1005954.ref101">101</xref>] into four categories according to their mechanism of transposition, namely, DNA transposons, LTRs, LINEs and SINEs. Consensus motifs that showed sequence similarity to RefSeq [<xref ref-type="bibr" rid="pgen.1005954.ref102">102</xref>] genes were filtered out, as they are likely to belong to a gene family, or be part of a conserved domain. To assign a repeat classification to the consensus motifs of repetitive regions, they were searched against the transposable elements determined from the genome assembly by MITE Hunter, LTR-harvester and TransposonPsi as well as the repeats present in RepBase [<xref ref-type="bibr" rid="pgen.1005954.ref024">24</xref>].</p><p>We used the Jellyfish software [<xref ref-type="bibr" rid="pgen.1005954.ref074">74</xref>] for computing 23-mer frequencies and choosing a subset of 23-mers with coverage greater than 1,000. We used the Cookiecutter package [<xref ref-type="bibr" rid="pgen.1005954.ref103">103</xref>] for extraction of raw reads containing subset of 23-mers with coverage greater than 1000. The selected reads were used to manually assemble tandem repeat monomer consensus sequences with the help of the targeted <italic>de novo</italic> short-read assembler PRICE [<xref ref-type="bibr" rid="pgen.1005954.ref104">104</xref>].</p><p>Assembled tandem repeats were compared with known Repbase repeats [<xref ref-type="bibr" rid="pgen.1005954.ref024">24</xref>] and all related to transposable elements were excluded. Following which, tandem repeats assembled from raw reads were aligned against the PacBio error corrected reads using BLAST [<xref ref-type="bibr" rid="pgen.1005954.ref081">81</xref>] to improve the consensus sequences by including more individual monomers. The consensus sequences were used to find and estimate the repeat copy number and arrangement in the genome assembly. The interruption of tandem repeats monomers arrays in SMRT reads could be verified by HiSeq reads containing transition fragments.</p><p>The genome was screened using RepeatMasker (version 4.0.5) [<xref ref-type="bibr" rid="pgen.1005954.ref093">93</xref>] against the entire vertebrate repeat library and subsequently against the published <italic>L</italic>. <italic>calcarifer</italic> specific repeats [<xref ref-type="bibr" rid="pgen.1005954.ref012">12</xref>]. Interspersed repeats were hard-masked with Ns and simple repeats were soft-masked. A final round of masking was performed using DustMasker (RMBlast, version 2.2.28) [<xref ref-type="bibr" rid="pgen.1005954.ref105">105</xref>].</p><p>First, putative gene loci were identified. The &#8220;other vertebrates&#8221; protein collection from Genbank [<xref ref-type="bibr" rid="pgen.1005954.ref106">106</xref>] was retrieved and filtered to retain only proteins from ray finned fish species. The resulting 388,340 proteins as well as the 1,184,879 reconstructed transcripts and ESTs were mapped to the genome using a tblastn/blastn run through GenBlastA [<xref ref-type="bibr" rid="pgen.1005954.ref107">107</xref>], which provided coordinates for putative gene loci.</p><p>Protein/transcript alignments were then refined and Augustus hints were generated. The genomic region of the putative gene loci (+25 kb flanking regions) were excised, and the corresponding protein/transcript sequences, linked to these gene loci by GenBlastA, were aligned using exonerate [<xref ref-type="bibr" rid="pgen.1005954.ref108">108</xref>]. Customized scripts were used to convert the exonerate output to GFF3 format with genomic coordinates, and generate an Augustus hints file.</p><p>A <italic>L</italic>. <italic>calcarifer</italic>-specific training annotation file was generated through the Augustus training web interface [<xref ref-type="bibr" rid="pgen.1005954.ref109">109</xref>] using 22,322 <italic>L</italic>. <italic>calcarifer</italic> ESTs. Augustus was run separately for each genomic contig using default parameters, extrinsic.E.XNT.cfg, the contig-specific hints file and the <italic>L</italic>. <italic>calcarifer</italic>-specific training annotation file. Augustus UTR prediction was disabled and only a single transcript was predicted for each putative gene locus.</p><p>In addition, gene predictions were also performed using Maker2 [<xref ref-type="bibr" rid="pgen.1005954.ref110">110</xref>] with assembled Illumina-based transcriptome, PacBio IsoSeq transcriptome, and high quality proteins from Percomorphaceae taxon. Maker2 predicted 29,401 genes and 100,765 different proteins. For mapping PacBio IsoSeq to assemebled genome we used GMAP software [<xref ref-type="bibr" rid="pgen.1005954.ref111">111</xref>].</p><p>The proteins predicted by Augustus and Maker2 were clustered with the proteome of the three-spined stickleback using OrthoMCL [<xref ref-type="bibr" rid="pgen.1005954.ref112">112</xref>]. The resulting clusters were further analysed to verify that Augustus and Maker2 predictions originated from the same genomic locus. Based on these analyses, we classified the genes into four groups: 1) conserved predictions (Augustus and Maker2 proteins are orthologous, have a stickleback ortholog and originate from the same gene locus), 2) species specific predictions (Augustus and Maker2 proteins are orthologous and originate from the same gene locus) 3) unplaced conserved predictions (Augustus and Maker2 proteins are orthologous, have a stickleback ortholog but originate from different gene loci) and 4) uncertain predictions (the remaining predicted genes).</p><p>For the conserved and species-specific categories, the gene prediction (from Augustus or Maker2) that was the longer at the defined locus was chosen as the representative. Since manual verification had shown that Maker2 predictions were often truncated compared to Augustus, for unplaced conserved predictions, the gene at the locus identified by Augustus was chosen as the representative. Uncertain predictions were not included in the final consensus gene dataset. Transcripts in the consensus dataset were then given unique identifiers. The final dataset was filtered for potential duplicates.</p><p>The Asian seabass reference proteins were aligned to proteins annotated in SwissProt and TrEMBL databases [<xref ref-type="bibr" rid="pgen.1005954.ref113">113</xref>,<xref ref-type="bibr" rid="pgen.1005954.ref114">114</xref>] using blastp from NCBI BLAST package [<xref ref-type="bibr" rid="pgen.1005954.ref115">115</xref>] with E-value set to 10-e5 and the best hit was chosen for each protein.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">RunIprScan-1.1.0 client (<ext-link ext-link-type="uri" ns0:href="http://michaelrthon.com/runiprscan/">http://michaelrthon.com/runiprscan/</ext-link>) was used for searching known protein motifs and domains by searching against publicly available databases available in InterPro [<xref ref-type="bibr" rid="pgen.1005954.ref025">25</xref>], including Pfam, PRINTS, PROSITE, ProDom, and SMART. The Gene Ontology (GO) terms were retrieved from RunIprScan-1.1.0 results for each protein. The mapping to KEGG pathways [<xref ref-type="bibr" rid="pgen.1005954.ref027">27</xref>] was computed using KAAS webserver [<xref ref-type="bibr" rid="pgen.1005954.ref116">116</xref>]. The statistics for functional annotation are summarized in <xref ref-type="supplementary-material" rid="pgen.1005954.s041">S26 Table</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We downloaded the proteomes of 19 species from Ensembl [<xref ref-type="bibr" rid="pgen.1005954.ref028">28</xref>] and NCBI RefSeq [<xref ref-type="bibr" rid="pgen.1005954.ref102">102</xref>]. These proteomes contained a total of 389,038 proteins and represented 13 ray-finned fishes (<italic>Astyanax mexicanus</italic>, <italic>Cynoglossus semilaevis</italic>, <italic>Danio rerio</italic>, <italic>Dicentrarchus labrax</italic>, <italic>Gadus morhua</italic>, <italic>Gasterosteus aculeatus</italic>, <italic>Larimichthys crocea</italic>, <italic>Notothenia coriiceps</italic>, <italic>Oreochromis niloticus</italic>, <italic>Oryzias latipes</italic>, <italic>Takifugu rubripes</italic>, <italic>Tetraodon nigroviridis</italic>, <italic>Xiphophorus maculatus</italic>), one cartilaginous fish (<italic>Callorhincus milii</italic>), one lobe-finned fish (<italic>Latimeria chalumnae</italic>), one reptile (<italic>Anolis carolinensis</italic>), one amphibian (<italic>Xenopus tropicalis</italic>), one bird (<italic>Gallus gallus</italic>) and human (<italic>Homo sapiens</italic>). These proteomes were clustered together with the <italic>L</italic>. <italic>calcarifer</italic> genome using blastp [<xref ref-type="bibr" rid="pgen.1005954.ref081">81</xref>] and FastOrtho (<ext-link ext-link-type="uri" ns0:href="http://enews.patricbrc.org/fastortho">http://enews.patricbrc.org/fastortho</ext-link>). All <italic>vs</italic> all BLAST (using blastp [<xref ref-type="bibr" rid="pgen.1005954.ref081">81</xref>]) was performed to identify homologous proteins and these were clustered using FastOrtho and MCL [<xref ref-type="bibr" rid="pgen.1005954.ref117">117</xref>] (FastOrtho is a reimplementation of the OrthoMCL [<xref ref-type="bibr" rid="pgen.1005954.ref112">112</xref>] algorithm in C++ and allowed for fast clustering of proteins).</p><p>Protein clusters were aligned using MAFFT [<xref ref-type="bibr" rid="pgen.1005954.ref118">118</xref>]. The alignments were filtered using a custom script to identify alignments with a low proportion of gaps. Gaps were defined as a column where greater than 40% of sequences in an alignment column was a gap character, and alignments where greater than 50% of the alignment consisting of gaps were considered to have a high proportion of gaps and these alignments were discarded. The remaining alignments were considered for potential gene duplication detection.</p><p>Total RNA was extracted using the mirVana miRNA extraction kit (Life Technologies, Carlsbad, CA, USA) and further purified using the miRCURY RNA isolation kit (Exiqon, Vedbaek, Denmark) according to manufacturer&#8217;s instructions. RNA concentration and integrity was measured on the NanoDrop 8000 Spectrophotometer (ThermoFisherScientific, Waltham, MA, USA) and visually assessed by agarose gel electrophoresis with ethidium bromide staining. RNA was stored at -80&#176;C. Small RNA libraries were constructed with HD adapters as previously described [<xref ref-type="bibr" rid="pgen.1005954.ref119">119</xref>]. Briefly, 2 &#956;g of total RNA was ligated to 3&#8217; and 5&#8217; HD adapters using commercially available enzymes and reagents. Ligated RNA products were reverse transcribed to cDNA and amplified by PCR. The cDNA products expected to contain 19&#8211;25 base pair inserts were selected and purified by 8% PAGE and ethanol precipitation. Libraries were sequenced on the HiSeq 2500 Ultra-High-Throughput Sequencing System (Illumina) at The Genome Analysis Centre (Norwich, United Kingdom).</p><p>FASTQ files were converted to FASTA format and HD adaptor sequences were trimmed by removing the first 4 bases of each read followed by the 3&#8217; adaptor and preceding four bases. Sequences shorter than 18nt and comprised of two or fewer unique bases were removed from further analysis.</p><p>miRNAs were annotated by searching all animal precursor hairpins from miRBase [<xref ref-type="bibr" rid="pgen.1005954.ref120">120</xref>] against the reference genome (E = 10e-6) generating a set of putative pre-miRNA sequences. Overlapping BLAST hits were merged and mature miRNAs annotated in miRBase [<xref ref-type="bibr" rid="pgen.1005954.ref120">120</xref>] were searched against putative precursors using PatMaN [<xref ref-type="bibr" rid="pgen.1005954.ref121">121</xref>], those hairpins with a match to a mature miRNA (with up to one mismatch) were then folded using RNAfold [<xref ref-type="bibr" rid="pgen.1005954.ref122">122</xref>] and those forming a valid pre-miRNA hairpin structure were annotated as miRNAs. In order to further annotate miRBase [<xref ref-type="bibr" rid="pgen.1005954.ref120">120</xref>] orthologues, we mapped our small RNA reads to the putative hairpins. Those with evidence of expression in our samples that were consistent with precise Dicer and Drosha processing were annotated as &#8220;high-confidence&#8221; miRNAs. Those with no evidence of expression or small RNA expression that was not consistent with precise miRNA biogenesis were classified as &#8220;low-confidence&#8221;.</p><p>New miRNAs were predicted using both miRCat [<xref ref-type="bibr" rid="pgen.1005954.ref123">123</xref>] and miRDeep2 [<xref ref-type="bibr" rid="pgen.1005954.ref124">124</xref>] using default parameters. Predictions were merged to obtain a non-redundant set of candidates and known miRNA families from miRBase [<xref ref-type="bibr" rid="pgen.1005954.ref120">120</xref>] identified previously were removed from the predicted novel miRNA set. Small RNA reads were aligned to predicted miRNAs and the read alignment pattern and secondary structure were checked manually to ensure that they are consistent with canonical miRNA biogenesis. Any predictions that did not meet these criteria were removed.</p><p>Best-fit models for each multiple protein sequence alignment were predicted using ProtTest 3.4 [<xref ref-type="bibr" rid="pgen.1005954.ref125">125</xref>]. Phylogeny trees were generated from the multiple sequence alignments using PhyML 3.0 [<xref ref-type="bibr" rid="pgen.1005954.ref126">126</xref>] with default parameters and the model selected by ProtTest, generating trees for 12,741 protein clusters.</p><p>A total of 2,439 alignments containing at least two Asian seabass sequences were identified using a customized python script. A best-fit evolutionary model was predicted for each multiple protein sequence alignment using ProtTest 3.4 prior to reconstructing a phylogenetic tree from each of these alignments using PhyML with 1000 bootstraps. A total of 2,190 phylogenetic trees were reconstructed and rooted using the elephant shark (<italic>Callorhinchus milii</italic>). Where no elephant shark was present in the cluster, the root was chosen by finding the midpoint of the tree using the `get_midpoint_outgroup&#8217; method of the ETE2 software [<xref ref-type="bibr" rid="pgen.1005954.ref127">127</xref>]. Asian seabass duplicate genes were identified by parsing the phylogenetic trees identified using ETE2's get_descendant_events implemented in a python script (show_duplicated_genes.py). Duplication events that yielded two neighbour leaf nodes containing <italic>L</italic>. <italic>calcarifer</italic> proteins were retained.</p><p>The locations of major histocompatibility complex class I (MHC-class I) genes in the Asian seabass genome were searched to determine the continuity of the Asian seabass genome assembly. The genome annotation dataset was mined for MHC-class I genes and their coordinates. A similar analysis was performed for the three-spined stickleback using a previously published list of stickleback MHC genes [<xref ref-type="bibr" rid="pgen.1005954.ref128">128</xref>].</p><p>The list of duplicated proteins was filtered to exclude those proteins lacking GO annotation, yielding a list of 844 proteins. This set was analysed using the BiNGO plugin to Cytoscape [<xref ref-type="bibr" rid="pgen.1005954.ref129">129</xref>]. BiNGO calculates a p-value from a Fisher Exact test that compares the prevalence of GO terms in the query set (genes that were duplicated: 844 proteins mapped to 458 GO terms) to the GO term prevalent in the proteome as a whole (16,671 proteins mapped to 2,984 GO terms). Two analyses were performed, to identify relatively over and under represented terms. The p-values were adjusted using the Benjamini-Hochberg correction. Terms with an adjusted p-value of less than 0.05 were considered significantly differentially expressed and were retained. Retained terms were visualised by mapping onto the GO ontology graph.</p><p>Previously reported one-to-one orthlogues [<xref ref-type="bibr" rid="pgen.1005954.ref022">22</xref>] were used as a starting point to identify the corresponding orthologous sequences from <italic>Cynoglossus semilaevis</italic>, <italic>Larimichthys crocea</italic>, <italic>Notothenia coriiceps</italic> and <italic>L</italic>. <italic>calcarifer</italic> using RSD approach [<xref ref-type="bibr" rid="pgen.1005954.ref130">130</xref>]. In total, we identified 313 strict one-to-one orthologues from the 24 species. Multiple alignments were generated using ClustalW version 2.0.12 [<xref ref-type="bibr" rid="pgen.1005954.ref131">131</xref>]. Alignments were concatenated using an in-house perl script. Ambiguous regions of the alignment were removed using Gblocks version 0.91b [<xref ref-type="bibr" rid="pgen.1005954.ref132">132</xref>]. We used RAxML version 8.1.3 [<xref ref-type="bibr" rid="pgen.1005954.ref133">133</xref>] to generate a Maximum Likelihood (ML) tree. The best-fit substitution model for the alignment was deduced using a perl script (ProteinModelSelection.pl) available at the RAxML webpage [<xref ref-type="bibr" rid="pgen.1005954.ref133">133</xref>]. The JTT+F model, as deduced by the script, was used for the ML analyses. Node support was estimated using 100 bootstrap replicates.</p><p>Fishes were obtained from our selection program based at the Marine Aquaculture Centre (MAC) of the Agri-Food and Veterinary Authority of Singapore (AVA), located on St John's Island, Singapore. Asian seabass larvae at the age of one to two days post-hatching (dph) were sacrificed on ice and used for culturing primary fibroblasts and for preparing chromosomes spreads as described previously [<xref ref-type="bibr" rid="pgen.1005954.ref012">12</xref>].</p><p>Three separate B chromosomes from different chromosome spreads were microdissected and collected using a glass needle coupled with an inverted microscope into collection drop solution as described before [<xref ref-type="bibr" rid="pgen.1005954.ref134">134</xref>]. After incubation at 60&#176;C for an hour, the collection drop solution was transferred to 5&#956;l of water. An initial round of B chromosome DNA amplification was performed using the WGA 1 Kit (Sigma-Aldrich). Primary PCR products were used for probe preparation (for FISH experiments) and amplified for further sequencing. The WGA-PCR-amplified chromosome material was re-amplified with 16-dUTP-biotin and digoxigenin-11-dUTP (both 2 &#956;M, Roche) under the following conditions: (1&#215;) 94&#176;C for 5 min; (35&#215;) 90&#176;C for 30 s, 54&#176;C for 30 s, 72&#176;C for 30s using WGA3 re-amplification kit (Sigma).</p><p>Sequencing libraries were prepared using the NEBNext DNA Library Prep Master Mix Set for Illumina (Illumina) for the ChB6 and ChB5 libraries, and the Nextera DNA Sample Preparation Kit (Illumina) for the ChB1 library. Libraries were sequenced on the Illumina MiSeq System with read length configuration of 2&#1093;250 bp for the ChB6 and ChB5 libraries and 2x300 bp for the ChB1 library. In all, 343,987, 404,427 and 382,627 sequencing reads were generated for ChB1, ChB5 and ChB6, respectively.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All reads with quality score less than 20 bp were removed, adapter sequences (WGA-specific, TGTGTTGGGTGTGTTTGG) were trimmed using the Cutadapt program [<xref ref-type="bibr" rid="pgen.1005954.ref135">135</xref>] and low quality bases were trimmed using &#8220;Trim Galore&#8221; (<ext-link ext-link-type="uri" ns0:href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</ext-link>) with default parameters.</p><p>Clean reads were mapped to assembled reference Asian seabass genome using Bowtie2 [<xref ref-type="bibr" rid="pgen.1005954.ref136">136</xref>] with default parameters. Successfully mapped reads were chained together across gaps less than 10 kb to form B chromosome pseudo-scaffolds. Pseudo-scaffolds were assembled using CAP3 [<xref ref-type="bibr" rid="pgen.1005954.ref137">137</xref>] to remove redundancy with the following parameters: minimum 50 bp overlapping length and 85% of similarity. Contigs were manually checked to reduce potential mis-assemblies (<xref ref-type="supplementary-material" rid="pgen.1005954.s042">S27</xref> and <xref ref-type="supplementary-material" rid="pgen.1005954.s043">S28</xref> Tables).</p><p>Tandem repeat probes were amplified from genomic DNA using the following primers: Lca_217 5&#8217;-GCCATTCTGAGCTGAATAAGCCTC-3&#8217;; Sat_LM 5&#8217;-CCAAAGAGAAGCACTTATGA-3&#8217;; and Lca_38 5&#8217;-Fc- AAAAAATGTCATAGTATAGTATGGCGTCAAAAAACATG-3&#8217;. The FISH procedure, slide preparation and image analysis were performed as described earlier[<xref ref-type="bibr" rid="pgen.1005954.ref012">12</xref>]. Hybridization for precise B chromosomes identification was performed under high-stringency conditions [<xref ref-type="bibr" rid="pgen.1005954.ref134">134</xref>]. Finally, the slides were counterstained with DAPI and mounted in an antifade solution (Vectashield from Vector laboratories, Burlingame, CA, USA).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Images were merged and measured using Image-Pro Express software V5.0 (Media Cybernetics, Rockville, MD, USA). Final image adjustments were performed using Adobe Photoshop CS2. The path of the chromosomes was computationally traced and straightened according to the manual provided by the Image J software V1.41 (<ext-link ext-link-type="uri" ns0:href="http://rsb.info.nih.gov/ij">http://rsb.info.nih.gov/ij</ext-link>).</p><p>The morphometric data was obtained from digital images of 65 individuals (22 from South-East Asia, 22 from India and 21 from Australia&#8211;Papua New Guinea) in a truss network system. The details of the methodology can be found in Ref#[<xref ref-type="bibr" rid="pgen.1005954.ref008">8</xref>]. The statistic Box&#8217;s M which tests the hypothesis of equality of co-variances across groups was non-significant. Therefore, within-groups covariance matrix was used for estimating discriminant functions. Wilks&#8217; lambda which is a measure of individual variable&#8217;s potential indicated that V18 variable is better at discriminating between groups compared to the other variables.</p><p>Only two variables (V18 and V3) contributed significantly to discrimination between groups and hence retained in the model (the remaining 16 variables were excluded from the analysis based on Wilks&#8217; lambda criterion) and used for estimating standardized canonical discriminant function coefficients. The developed discriminant functions could correctly classify about 73.8% of the fish to the respective groups. The cross-validation procedure could correctly assign about 72.3% of fish to the respective groups. Overall, the discriminant function was able to correctly assign three out of every four fish to the respective groups. Principal component analysis was also performed based on the 18 truss measurements normalized by maximum length of fish using correlation matrix in Past 3.7 software [<xref ref-type="bibr" rid="pgen.1005954.ref138">138</xref>].</p><p>The details of 62 Asian seabass samples (the reference individual and 61 additional individuals used for re-sequencing) collected from 13 geographic regions across its range are given in <xref ref-type="supplementary-material" rid="pgen.1005954.s036">S21 Table</xref>. Paired-end genomic DNA libraries were constructed and sequenced using the Illumina platform (<xref ref-type="supplementary-material" rid="pgen.1005954.s037">S22 Table</xref>). The main steps for analyzing the sequence data are outlined in <xref ref-type="supplementary-material" rid="pgen.1005954.s008">S7 Fig</xref>. Mapping quality of at least 40 and base quality of 17 were used for SNP calling. Using GATK unified genotyper [<xref ref-type="bibr" rid="pgen.1005954.ref139">139</xref>], a total of 8,464,441 SNPs with Phred score &gt; 30 were found in the 62 samples. Also, 6,522,041 SNPs with Phred score &gt; 30 were found on repeating SNP-calling using Samtools pipeline [<xref ref-type="bibr" rid="pgen.1005954.ref140">140</xref>]. The combined set of 6,458,484 SNPs common to both the SNP callers were filtered for repeat sequences using Tandem Repeat Finder [<xref ref-type="bibr" rid="pgen.1005954.ref096">96</xref>] and Repeat Scout [<xref ref-type="bibr" rid="pgen.1005954.ref141">141</xref>] (SNP calling in repeats is unreliable because of the high misalignment rate and problematic assembly of repeat sequences) resulting in 5,642,327 SNPs.</p><p>Principal component analysis (PCA) was performed using SNPrelate [<xref ref-type="bibr" rid="pgen.1005954.ref142">142</xref>]. SNPs with less than 5% minor allele frequency and SNPs with linkage disequilibrium (LD) threshold more than 0.2 were removed from the analyses. The final set of results used for PCA analysis consisted of 64,634 SNPs (Figs <xref ref-type="fig" rid="pgen.1005954.g004">4</xref> and <xref ref-type="supplementary-material" rid="pgen.1005954.s009">S8</xref>).</p><p>Model-based inference of ancestry amongst the various seabass populations was performed using ADMIXTURE [<xref ref-type="bibr" rid="pgen.1005954.ref143">143</xref>] software based on ML-optimization. All SNPs with more than 5% missing data were filtered out. ADMIXTURE model cannot incorporate loci in LD, so Plink 1.9 was used to remove SNPs with LD level more than 0.1 in 100 kb window. Final dataset for ADMIXTURE analyses consisted of 27,809 SNPs. The number of Ks which best explain the seen variation was also elucidated (Figs <xref ref-type="fig" rid="pgen.1005954.g005">5</xref> and <xref ref-type="supplementary-material" rid="pgen.1005954.s010">S9</xref>).</p><p>For population phylogenetic analyses, SNPs were filtered out with the following parameters: distance between SNPs of at least 4000 bp (to reduce the linkage effects), MAF = 0.05, maximum 3 missing genotypes per SNP. Final dataset comprised of 123,594 SNPs. Maximum-Likelihood tree was constructed using RaxMl 8 software [<xref ref-type="bibr" rid="pgen.1005954.ref133">133</xref>] with GTR matrix and Gamma parameter using 100 bootstrap replicates. SNPs were annotated using SnpEff software [<xref ref-type="bibr" rid="pgen.1005954.ref144">144</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The scaffolded genome assembly (v2) has been submitted to DDBJ/EMBL/NCBI GenBank under the accession LLXD00000000. Alternatively, it is also available for download at <ext-link ext-link-type="uri" ns0:href="http://seabass.sanbi.ac.za/">http://seabass.sanbi.ac.za/</ext-link>, together with the annotations (for the v2 assembly). The chromosome-level genome assembly (v3) is also available at the above-mentioned website. The Illumina and PacBio reads utilized for the genome assembly, as well as the whole-genome resequencing reads have been submitted to NCBI SRA under BioProject accession numbers SRP069219 and SRP069848, respectively. The BAC end sequences have been submitted to NCBI dbGSS under the accession numbers KS320706&#8212;KS326261 for the <italic>Bam</italic> HI library and KS326262&#8212;KS331896 for the <italic>Eco</italic> RI library.</p>