<p>To infer recombination events and compare sex differences, we extracted a total of 185,917 three-generation families from a large pedigree of Holstein cattle maintained in the Animal Genomics and Improvement Laboratory (AGIL) at USDA, with one offspring, both parents and two grandsires with SNP genotypes in each family (<xref ref-type="fig" rid="pgen.1005387.g001">Fig 1</xref>). In each of the 185,917 families, we inferred recombination events for a paternal meiosis from the sire/offspring pair and a maternal meiosis from the dam/offspring pair. For recombination map construction and GWAS of hotspot usage, we only included the highest-quality or most informative meioses where the offspring, the parent, and the grandsire were genotyped by 50K SNP chips, resulting in a total of 70,715 male and 61,616 female meioses. For GWAS analysis of genome-wide recombination rate, we included all paternal and maternal meioses from the 185,917 families regardless of the number of SNPs genotyped.</p><p>The animals in the selected families were genotyped by various genotyping assays (<xref ref-type="supplementary-material" rid="pgen.1005387.s014">S1 Table</xref>), ranging from 3K to 770K SNPs [<xref ref-type="bibr" rid="pgen.1005387.ref033">33</xref>]. The Illumina BovineSNP50 v1 chip with 56,947 SNPs, v2 chip with 54,609 SNPs, the high-density (HD) chip with 777,962 SNPs, and the GeneSeek HD chip with 77,068 SNPs are referred to as the 50K chip, as we used a combined set of &gt;50K SNPs [<xref ref-type="bibr" rid="pgen.1005387.ref065">65</xref>]. The Zoetis BovineLD chip with 10,555 SNPs, the GeneSeek Genomic Profiler v1 and v2 chips with 8,042 and 8,415 SNPs, the Illumina BovineLD BeadChip with 6,785 SNPs, the Illumina Bovine3K BeadChip with 2,708 SNPs are referred to herein as 10K, 8K, 7K, and 3K, respectively. The chips were designed as mostly nested with the higher-density chip including SNPs on the lower-density ones. Although the offspring and two parents were genotyped by various SNP chips, the grandsires were mostly genotyped by the 50K SNP chips. Note that the granddam was not necessarily genotyped in the selected three-generation families. Depending on the number of genotyped granddams, we collected 67,690, 76,318, and 41,909 families with two, one and zero genotyped granddams, respectively. Note that an animal may appear in more than one family based on the pedigree structure, especially for bulls that have hundreds of progeny.</p><p>To study recombination, we included up to 59,309 genome-wide SNPs after quality control filtering and used the USDA-AGIL SNP coordinates that showed a higher quality than the UMD3.1 assembly [<xref ref-type="bibr" rid="pgen.1005387.ref030">30</xref>,<xref ref-type="bibr" rid="pgen.1005387.ref033">33</xref>]. Previously, several SNPs were relocated from the UMD3.1 assembly to the USDA-AIGL coordinates in cooperation with researchers from the University of Missouri (R. D. Schnabel), and the University of Guelph (M. Sargolzaei and J. Johnston). From pairwise linkage analysis in this study, we also removed nine suspicious SNPs that exhibited suspicious linkage disequilibrium (LD) patterns with SNPs on the same chromosome (<xref ref-type="supplementary-material" rid="pgen.1005387.s002">S2 Fig</xref> and <xref ref-type="supplementary-material" rid="pgen.1005387.s016">S3 Table</xref>). Due to the low quality of the genome assembly, we excluded the X chromosome from recombination calculation in this study.</p><p>To compare sex differences on an equal footing, we phased the raw genotypes for the paternal and maternal meioses within a three-generation family without using the possible additional information from multiple offspring [<xref ref-type="bibr" rid="pgen.1005387.ref066">66</xref>], because bulls generally had many more progeny than cows. In each of the three-generation families, we inferred the paternal and maternal haplotypes of the offspring based on the genotypes of the two parents, and also inferred the paternal and maternal haplotypes for both parents based on the genotypes of the grandparents [<xref ref-type="bibr" rid="pgen.1005387.ref029">29</xref>,<xref ref-type="bibr" rid="pgen.1005387.ref066">66</xref>]. Homozygous genotypes were phased trivially and the heterozygous genotypes were phased whenever the genotypes of the two parents are not heterozygous simultaneously. The parent-of-origin was then assigned to each allele after phasing to determine paternal and maternal haplotypes.</p><p>After phasing the genotypes of the offspring and parents, we inferred recombination events in the paternal and maternal haplotypes of the offspring by comparing the offspring&#8217;s paternal haplotype to the two haplotypes of the sire as well as by comparing the offspring&#8217;s maternal haplotype to the two haplotypes of the dam. In the offspring haplotype, a recombination event was defined as a transition from the parent's paternal to maternal haplotype or vice versa. Note that the recombination events defined here were observed crossovers so that the number of observed crossovers could differ from the number of true crossovers when multiple crossover events occurred at the same site. This potential inconsistency between observed and true crossovers typically is addressed by a map function that translates a recombination frequency into a map distance in terms of crossovers. However, the physical distances between two adjacent SNPs were small so that the use of a map function virtually would not make a numerical difference. Therefore, our estimates of crossovers based on recombinants should be close to the true number of underlying crossovers. To further reduce false positives, we required a crossover call to be supported by at least three consecutive informative heterozygous SNPs [<xref ref-type="bibr" rid="pgen.1005387.ref009">9</xref>]. In total, we identified ~4.5 million paternal and ~4.0 million maternal recombination events from the total of 185,917 paternal and maternal meioses, respectively. To ensure that our results do not depend strongly on the cutoffs, we repeated the analysis by using a different cutoff value of 5 consecutive informative markers and the number of identified crossovers was only reduced by 0.2%.</p><p>A recombination event was assigned to a region spanned by two informative SNPs that may not be adjacent to each other. To construct a recombination map, we used an EM-algorithm to calculate the probability of crossing over per meiosis or recombination rate between each pair of consecutive SNPs based on the observed crossover regions [<xref ref-type="bibr" rid="pgen.1005387.ref009">9</xref>]. After an initiation step to assign an expected count of 1/<italic>m</italic> to each of the <italic>m</italic> adjacent SNP intervals in a crossover region, the EM algorithm proceeded in the following iteration steps: 1) M-step: considering a total of <italic>n</italic> meioses, the overall expected count attributed to a SNP interval divided by <italic>n</italic> were the maximum likelihood estimate of the probability, and 2) E-step: for a crossover region, the expected count assigned to a SNP interval was estimated as proportional to the current estimate of the probability of crossover for that SNP interval. The M and E steps were iterated until convergence.</p><p>We constructed the male and female recombination maps for 59,309 autosomal SNPs based on &gt;1.8 million paternal and &gt;1.4 million maternal recombination crossovers, which were identified from 70,715 male and 61,616 female meioses that are most informative where the offspring, the parent, and the grandsire were genotyped by 50K SNP chips. Note that some granddams were genotyped by a low density chip or even not genotyped. As described earlier, a correction for the number of SNPs of the granddam was employed in the two sexes separately such that the total number of crossovers after correction was the same regardless of the SNP numbers for granddams. We used the expected number of crossovers per meiosis or recombination rate between adjacent SNPs as the genetic distance in our recombination map, since one crossover event on average corresponds to a genetic distance of 1 Morgan and recombination rate is almost the same as genetic distance for small intervals [<xref ref-type="bibr" rid="pgen.1005387.ref009">9</xref>,<xref ref-type="bibr" rid="pgen.1005387.ref067">67</xref>]. Alternatively, using Haldane&#8217;s map function with crossover interference [<xref ref-type="bibr" rid="pgen.1005387.ref068">68</xref>], the male and female maps were slightly longer, 25.6 and 23.3 Morgans in length respectively.</p><p>We defined recombination hotspots as the SNP intervals with a recombination rate &gt;2.5 standard deviations from the genome-wide average in males and females separately, because 2.5 standard deviations are highly significant departures from the average recombination in cattle given our large sample sizes in both sexes, consistent with the observations from a recent recombination study in cattle [<xref ref-type="bibr" rid="pgen.1005387.ref029">29</xref>]. We tested a range of cutoff values, 2, 2.5, 3, 5, and 10 standard deviations, and found that the cutoff value of 2.5 standard deviations showed the clearest signal for the association between <italic>PRDM9</italic> and recombination hotspot usage. For validation purposes, we also defined recombination hotspots via using a standardized recombination rate that was calculated by dividing the original recombination rate between two SNPs by the physical length. After this adjustment of physical lengths, the standard recombination rates vary even less so we used a cutoff value of 0.6 standard deviations to define recombination hotspots.</p><p>To ensure the quality of the hotspots identified, we calculated pairwise linkage disequilibrium (LD) statistics between each of the SNPs in or near a recombination hotspot and all other SNPs on the same chromosome using Locusmap [<xref ref-type="bibr" rid="pgen.1005387.ref035">35</xref>]. The pairwise LD was evaluated by a LOD score and an estimated recombination rate. For a SNP with correct physical position, the LOD scores should peak near the SNP and decrease when moving away in the two directions. The recombination rate should follow the opposite pattern with small values near the SNP and increasing with the distance. Any obvious deviations from these expected LD patterns suggest a possible error in the SNP coordinate and thus a false recombination hotspot (<xref ref-type="supplementary-material" rid="pgen.1005387.s002">S2 Fig</xref>). Nine suspicious SNPs exhibiting unexpected LD patterns together with seven originally identified recombination hotspots were removed (<xref ref-type="supplementary-material" rid="pgen.1005387.s018">S5 Table</xref>).</p><p>As expected, the power for identifying recombination events was affected by the number of genotyped SNPs. For male meiosis, the average number of crossovers identified varied from 25.9 to 17.7 depending on the number of measured SNPs of the animals in a three-generation family (<xref ref-type="supplementary-material" rid="pgen.1005387.s015">S2 Table</xref>). Overall, more crossovers were identified when the number of SNPs increased, except for a few categories that had a small sample size and thus large noise. A similar pattern was also observed for female meioses, where the number of crossovers ranged between 23.9 and 16.0 (<xref ref-type="supplementary-material" rid="pgen.1005387.s016">S3 Table</xref> To account for this effect of SNP numbers, we used the highest-quality meioses where all animals were genotyped by 50K SNP chip as a reference to correct the number of crossovers identified in other meioses. After correction, the average number of crossovers was equal to 25.5 in males and 23.2 in females.</p><p>To evaluate the power for identifying crossovers using 50K SNP genotypes, we simulated 50K SNP genotypes for all animals in the real pedigree and used the same phasing and crossover identification procedures as described in previous sections. By defining a positive result as an identified crossover interval overlapping with the true location of a crossover, we calculated the power of crossover identification as the proportion of positive results across all simulated crossover events. As a result, our approach had a power of 97.6% for the three-generation families genotyped by 50K SNP chip, which means that on average only 2.4% crossovers were missed in our recombination map.</p><p>To evaluate the relationship between recombination features and potentially related factors, we fitted a smooth spline model of the male and female recombination rates on relative chromosomal locations or time using the smooth.spline function implemented in R 3.1.1 [<xref ref-type="bibr" rid="pgen.1005387.ref069">69</xref>]. We calculated a relative physical position for each of the SNP intervals by dividing the original physical position by the corresponding chromosome length. For the analysis of chromosome location effect on recombination, we used those meioses for which all the required individuals were genotyped by 50K SNP chip and removed 2-Mb regions to the end of all chromosomes where the power of identifying crossovers is low [<xref ref-type="bibr" rid="pgen.1005387.ref009">9</xref>]. For the analysis of time trend, we used all the meioses and adjusted for the effects of SNP chips and inbreeding, as well as effect of influential bulls by accounting for the number of progeny for bulls. Specifically, the correction was conducted with a linear model for the number of crossovers by fitting fixed effects for the categories of SNP chips of the offspring, parent and two grandparents, genomic inbreeding coefficient of the parent, the number of phased heterozygous SNPs of the offspring and parent, quadratic and cubic terms of the two numbers of informative SNPs, and the interaction terms between them. As expected, we found a negative association between inbreeding coefficient and number of recombination events in both sexes (Males: <italic>&#946;</italic> = &#8722;0.28 and <italic>P</italic>-value = 0.006; Females: <italic>&#946;</italic> = &#8722;0.11 and <italic>P</italic>-value = 3.2 &#215; 10<sup>&#8722;12</sup>). To investigate the common pattern across the 29 autosomes, we pooled all the autosomes together using the relative physical position and fitted a smooth spline model for all the data combined (<xref ref-type="fig" rid="pgen.1005387.g004">Fig 4</xref>). We also fitted a smooth spline model for each of the chromosomes individually (<xref ref-type="supplementary-material" rid="pgen.1005387.s003">S3 Fig</xref>). A degree of freedom of five was used in all the smooth spline modeling. A similar smooth spline model was fitted for the analysis of time trend of recombination rate and hotspot usage (<xref ref-type="supplementary-material" rid="pgen.1005387.s008">S8</xref>, <xref ref-type="supplementary-material" rid="pgen.1005387.s009">S9</xref>, <xref ref-type="supplementary-material" rid="pgen.1005387.s010">S10</xref> and <xref ref-type="supplementary-material" rid="pgen.1005387.s012">S12</xref> Figs).</p><p>We estimated the number of recombination events for maternal and paternal meioses in each of the 185,917 three-generation families, which were then assigned to the sire and dam in the family. Each sire or dam may have multiple phenotypic measurements when appearing in more than one family, and we calculated the average of the multiple measurements as the phenotype for genome-wide recombination rate. A total of 3,224 bulls and 53,125 cows were included in the GWAS of genome-wide recombination rate. We corrected the originally estimated number of recombination events by the number of measured SNPs of the animals for each meiosis so that the average number of crossovers was the same regardless of the genotyping assays (<xref ref-type="supplementary-material" rid="pgen.1005387.s015">S2</xref> and <xref ref-type="supplementary-material" rid="pgen.1005387.s016">S3</xref> Tables).</p><p>We calculated the genome-wide proportions of crossovers occurring in the hotspots, i.e. hotspot usage, for the 70,715 male and 61,616 female meioses that are most informative where the offspring, the parent and the grandsire were genotyped by 50K SNP chips. After assigning the estimated hotspot usage to the sire or dam for each meiosis, we used the average of the multiple measurements as the phenotype, resulting in a sample size of 1,772 and 12,756 in males and females, respectively.</p><p>Imputed genotypes of 777,962 SNPs on the Illumina BovineHD Genotyping BeadChip (HD) were obtained by running Findhap on measured genotypes with 3K to 50K SNP chips using a reference population of 2,433 animals directly measured with HD SNP chips [<xref ref-type="bibr" rid="pgen.1005387.ref033">33</xref>,<xref ref-type="bibr" rid="pgen.1005387.ref070">70</xref>]. After filtering SNPs exhibiting redundancy, very high linkage disequilibrium (<italic>r</italic> &gt; 0.95), or small minor allele frequency (MAF &lt; 0.001), we retained over 310K genome-wide SNPs in the association studies [<xref ref-type="bibr" rid="pgen.1005387.ref033">33</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We tested for association between each SNP and a phenotype using a linear mixed model with variable residual variances that are inversely proportional to the number of repeated measures of the phenotype, i.e., residual variance is smaller for individuals with more measurements of the phenotype. The model equation in matrix notation is
<disp-formula id="pgen.1005387.e001"><alternatives><graphic id="pgen.1005387.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005387.e001.jpg" /><ns1:math id="M1"><ns1:mrow><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>y</ns1:mi></ns1:mstyle><ns1:mo>=</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>X</ns1:mi><ns1:mi>g</ns1:mi></ns1:mstyle><ns1:mo>+</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>Z</ns1:mi><ns1:mi>a</ns1:mi></ns1:mstyle><ns1:mo>+</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>e</ns1:mi></ns1:mstyle></ns1:mrow></ns1:math></alternatives></disp-formula>
where <bold>y</bold> = a vector of the phenotype, <bold>X</bold> = a design matrix of the fixed effects <bold>g</bold>, including a population mean and the additive effect of the candidate SNP, <bold>Z</bold> = a design matrix for a random animal effect <bold>a</bold>, and <bold>e</bold> = a vector of random residuals. We assume that <inline-formula id="pgen.1005387.e002"><alternatives><graphic id="pgen.1005387.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005387.e002.jpg" /><ns1:math id="M2"><ns1:mrow><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>a</ns1:mi></ns1:mstyle><ns1:mo>&#160;</ns1:mo><ns1:mo>~</ns1:mo><ns1:mo>&#160;</ns1:mo><ns1:mi>N</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>A</ns1:mi></ns1:mstyle><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mi>a</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula> and <inline-formula id="pgen.1005387.e003"><alternatives><graphic id="pgen.1005387.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005387.e003.jpg" /><ns1:math id="M3"><ns1:mrow><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>e</ns1:mi></ns1:mstyle><ns1:mo>&#160;</ns1:mo><ns1:mo>~</ns1:mo><ns1:mo>&#160;</ns1:mo><ns1:mi>N</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:mstyle mathsize="normal" mathvariant="bold"><ns1:mi>R</ns1:mi></ns1:mstyle><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mi>e</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula>, where <bold>A</bold> is the genomic relationship matrix and <bold>R</bold> is a diagonal matrix with the <italic>i</italic>th diagonal element equal to 1/<italic>w</italic>, where <italic>w</italic> is the number of phenotypic measurements for the <italic>i</italic>th animal.</p><p>This model has been implemented in the MMAP software package with optimized computing [<xref ref-type="bibr" rid="pgen.1005387.ref071">71</xref>,<xref ref-type="bibr" rid="pgen.1005387.ref072">72</xref>], which can finish a GWAS analysis with 53,125 samples and 310K SNPs in hours using 32 CPU cores of the high-performance computer at USDA-AGIL. The model was empirically validated by observing no inflation in the quantile-quantile plots (QQ-plot) of the GWAS <italic>P</italic>-values for both recombination rate and hotspot usage in this study (<xref ref-type="supplementary-material" rid="pgen.1005387.s005">S5</xref> and <xref ref-type="supplementary-material" rid="pgen.1005387.s006">S6</xref> Figs).</p>