<p>Jmjd3 floxed mice (<italic>Jmjd3<sup>f/f</sup></italic>) and <italic>Jmjd3<sup>&#8722;/&#8722;</sup></italic> mice were generated by homologous recombination technique <xref ref-type="bibr" rid="pgen.1004524-Zhao1">[17]</xref>. Mice were bred and maintained at the pathogen-free animal facilities of Baylor Medical School and the Houston Methodist Research Institute. Animal procedures were approved by the Animal Care and Use Committee at Baylor College of Medicine and Houston Methodist Research Institute. To generate inducible <italic>Jmjd3</italic> deletion in mice, TM (Sigma T5648) was dissolved in corn oil (Sigma) at a concentration of 10 mg/ml and intraperitoneally (IP) injected into 8-week-old mice at 9 mg TM/40 g of body weight <xref ref-type="bibr" rid="pgen.1004524-Parr1">[48]</xref>, <xref ref-type="bibr" rid="pgen.1004524-Hayashi1">[49]</xref>. Pregnant mice received 3&#8211;4 mg TM/40 g of body weight by IP injection for 2 consecutive days. Pregnant mice were injected with TM at different time points to recover E4.5, E9.5, and E14.5 embryos. The newborn pups were genotyped with tail and lung tissue, and lung structure and development were assessed. The <italic>Wnt1-Cre</italic> strain was obtained from the Jackson Laboratory (022137). <italic>SPC-Cre</italic>, and <italic>CCSP-Cre</italic> and <italic>Utx<sup>floxed</sup></italic> mice were kindly provided by Dr. Brigid Hogan (Duke University Medical Center), Dr. Frank DeMayo (Baylor College of Medicine) and Dr. Kai Ge (National Institutes of Health), respectively.</p><p>Total RNAs were isolated from E17.5 lung tissues and primary MEFs using Trizol reagent (Invitrogen). A total of 1 &#181;g of RNA was converted to cDNA using Superscript III Reverse Transcriptase (Qiagen) with random hexamer primers. qPCR was carried out using SYBR Green mix on the ABI 7000 PCR machine. <italic>Jmjd3</italic> expression in heterozygous and homozygous WT MEFs and WT and <italic>Jmjd3</italic>
<sup>&#8722;/&#8722;</sup> E17.5 lung tissues was determined using the following Jmjd3 primer set: (forward) <named-content content-type="gene">5&#8242;-AAGTGGGGACAAGGAGACCT</named-content> and (reverse) <named-content content-type="gene">5&#8242;-AAGTGGGGACAAGGAGACCT</named-content>. WT and <italic>Jmjd3</italic>
<sup>&#8722;/&#8722;</sup> E17.5 lung tissues were also evaluated for the expression of SP-A (<named-content content-type="gene">5&#8242;-CTCCAGACCTGTGCCCATATG</named-content> and <named-content content-type="gene">5&#8242;-ACCTCCAGTCATGGCACAGTAA</named-content>), <italic>SP-B</italic> (<named-content content-type="gene">5&#8242;-ACGTCCTCTGGAAGCCTTCA</named-content> and <named-content content-type="gene">5&#8242;-TGTCTTCTTGGAGCCACAACAG</named-content>), <italic>SP-C</italic> (<named-content content-type="gene">5&#8242;-ACCCTGTGTGGAGAGCTACCA</named-content> and <named-content content-type="gene">5&#8242;-TTTGCGGAGGGTCTTTCCT</named-content>), <italic>SP-D</italic> (<named-content content-type="gene">5&#8242;-CTCTGAGGCAGCAGATGGA</named-content> and <named-content content-type="gene">5&#8242;-ATCAGGGAACAATGCAGCTT</named-content>), <italic>CC10</italic> (<named-content content-type="gene">5&#8242;-TCCTAACAAGTCCTCTGTGTAAGA</named-content> and <named-content content-type="gene">5&#8242;-AGGAGACACAGGGCAGTGACA</named-content>), <italic>AQP-5</italic> (<named-content content-type="gene">5&#8242;-ATGAACCCAGCCCGATCTTT</named-content> and <named-content content-type="gene">5&#8242;-ACGATCGGTCCTACCCAGAAG</named-content>), <italic>ABCA3</italic> (<named-content content-type="gene">5&#8242;-TTGCCCTCATTGGAGAGCCTG</named-content> and <named-content content-type="gene">5&#8242;-TCCGGCCATCCTCAGTGGTGGG</named-content>), <italic>Nkx2.1</italic> (<named-content content-type="gene">5&#8242;-TCCAGCCTATCCCATCTGAACT</named-content> and <named-content content-type="gene">5&#8242;-CAAGCGCATCTCACGTCTCA</named-content>), <italic>UTX</italic> (<named-content content-type="gene">5&#8242;-ATCCCAGCTCAGCAGAAGTT</named-content> and <named-content content-type="gene">5&#8242;-GGAGGAAAGAAAGCATCACG</named-content>), <italic>Ezh2</italic> (<named-content content-type="gene">5&#8242;-GCCAGACTGGGAAGAAATCTG</named-content> and <named-content content-type="gene">5&#8242;-TGTGCTGGAAAATCCAAGTCA</named-content>), <italic>Hoxa5</italic> (<named-content content-type="gene">5&#8242;-TCTCGTTGCCCTAATTCATCTTT</named-content> and <named-content content-type="gene">5&#8242;-CATTCAGGACAAAGAGATGAACAGAA</named-content>), <italic>Hoxb1</italic> (<named-content content-type="gene">5&#8242;-TTGCCCTGGAAACTGTAAAG</named-content> and <named-content content-type="gene">5&#8242;-AATTTGCCAACAACCCATC</named-content>), <italic>Esrrg</italic> (<named-content content-type="gene">5&#8242;- TGCACCGGGCTCTGTCAAGGAA</named-content> and <named-content content-type="gene">5&#8242;-AATCCATGTGCGCCCGACAACC</named-content>), <italic>Reln</italic> (<named-content content-type="gene">5&#8242;-TCCAGGCTCAGCACCAAGCCAA</named-content> and <named-content content-type="gene">5&#8242;- TGGATCTTGCCTTCTGACGCCCTT</named-content>), <italic>Kirrel3</italic> (<named-content content-type="gene">5&#8242;-AGAAAGTCACAGCTCCGCTCGGT</named-content> and <named-content content-type="gene">5&#8242;-ACGGGAGGGTTGCAGAAAGGCT</named-content>), <italic>March4</italic> (<named-content content-type="gene">5&#8242;-GCCCCTCCCCTTGGTTCCATCAAA</named-content> and <named-content content-type="gene">5&#8242;-CGAGGAGGAGAAAGCGAAGCCACT</named-content>), <italic>Fog2</italic> (<named-content content-type="gene">5&#8242;-CGCCTTTGTGGTGGACTTTGACT</named-content> and <named-content content-type="gene">5&#8242;-GCTTCTCGTTGCCTCCCACTACA</named-content>), <italic>COUP-TFII</italic> (<named-content content-type="gene">5&#8242;-AGTACTGCCGCCTCAAAAAG</named-content> and <named-content content-type="gene">5&#8242;-CGTTGGTCAGGGCAAACT</named-content>), <italic>Wt1</italic> (<named-content content-type="gene">5&#8242;-AATGCGCCCTACCTGCCCA</named-content> and <named-content content-type="gene">5&#8242;- CCGTCGAAAGTGACCGTGCTGTAT</named-content>) and <italic>&#946;-Actin</italic> (<named-content content-type="gene">5&#8242;-GTGGGCCGCTCTAGGCACCA</named-content> and <named-content content-type="gene">5&#8242;-TGGCCTTAGGGTTCAGGG</named-content>).</p><p>Microarray analysis was performed on RNA samples prepared from E17.5 <italic>Jmjd3<sup>+/+</sup></italic> and <italic>Jmjd3<sup>&#8722;/&#8722;</sup></italic> lungs using the Affymetrix Genechip microarray system. RNA quality assessment and microarray analysis were performed by the Microarray Core Facility of Baylor College of Medicine. RNA quality was checked using the NanoDrop ND-1000 spectrophotometer and Agilent 2100 Bioanalyzer. Total RNA samples of 250 ng were labeled using the 3&#8242; IVT Express Kit with the new standard Affymetrix linear amplification protocol and then reverse transcribed to produce double-stranded cDNA. The cDNA product was used as a template for the <italic>in vitro</italic> transcription of biotin-labeled cRNA. The labeled cRNA was quantified using the NanoDrop ND-1000 spectrophotometer. A total of 15.0 &#181;g of the labeled cRNA was fragmented and the concentration rechecked. A hybridization cocktail containing Affymetrix spike-in controls and fragmented labeled cRNA was loaded onto a GeneChip array. The array was hybridized for 16 h at 45&#176;C with rotation at 60 rpm and then washed and stained with a strepavidin, R-phycoerythrin conjugate stain. Signal amplification was done using biotinylated anti-streptavidin. The stained array was scanned on the Affymetrix GeneChip Scanner 3000. The images were then analyzed and quality control metrics were recorded using Affymetrix GCOS software version 1.1.2. Pairwise comparisons were made between WT and <italic>Jmjd3</italic>-deficient lungs. Raw <italic>P</italic> values were adjusted by the Benjamini-Hochberg method for 5% false discovery rate to yield adjusted <italic>p</italic> values. The criteria for significance of differentially regulated genes were established as &#8805;1.3-fold change with an adjusted <italic>P</italic> value&lt;0.05. Pathways were analyzed (Ingenuity Systems) to determine the ratios of known genes within each pathway that were significantly changed between WT and KO lungs.</p><p>Lung tissues were collected, ground into fine powders in liquid nitrogen, and transferred to lysis buffer (50 mM Tris-HCl [pH 7.4], 0.1% SDS) supplemented with protease inhibitor cocktail (cOmplete Mini Protease Inhibitor, 14132300) and 2 &#181;M phenylmethylsulphonyl fluoride (PMSF). Cells were washed in phosphate-buffered saline (PBS), harvested, and lysed in RIPA buffer supplemented with protease inhibitor cocktail and 2 &#181;M PMSF. After sonication and centrifugation, protein supernatants were collected, and protein concentrations were determined using a Bradford Protein Assay Kit (Sigma). Protein extracts were boiled in SDS sample buffer for 5 min, loaded directly onto a 4&#8211;12% SDS gel, transferred onto nitrocellulose membranes (Bio-Rad), blocked with 5% milk, and incubated with primary and corresponding secondary antibodies. The following antibodies were used: anti-H3K4me1 (07-436), anti-H3K4me2 (07-030), anti-H3K4me3 (07-473), anti-H3K9me1 (07-450), anti-H3K9me2 (07-441), anti-H3K9me3 (07-442), anti-H3K27me1 (07-448), anti-H3K27me2 (07-452), and anti-H3K27me3 (07-449) from Millipore; anti-SP-A (sc-7700), anti-AQP-5 (sc-9891); anti-SP-B (sc-13978), anti-SP-C (sc-13979), anti-SP-D (sc-13980), and anti-CC10 (sc-25555) from Santa Cruz Biotechnology; anti-H3(9715) and Ezh2 (4905) from Cell Signaling; anti-CD11b (17-0113-81), anti-F4/80 (15-4801-82), and anti-Gr1 (17-5931-81) from eBioscience; mouse anti-Flag, anti-HRP-flag, and anti-&#946;-actin from Sigma; rabbit anti-UTX (ab36938) from (Abcam); and anti-Jmjd3 (ab1022a) from Abgent.</p><p>Tissues and embryos were fixed in 10% neutral formalin. The fixation time was dependent on embryonic stage: E&lt;12.5 were fixed for 2 h, E12.5-E17 for 4 h, and E&gt;17.5 for 24&#8211;48 h as previously described <xref ref-type="bibr" rid="pgen.1004524-Crawford1">[50]</xref>. After fixation, tissues and embryos were embedded in paraffin and cut into 5-&#181;m thick sections. H&amp;E staining and immunostaining were performed according to standard protocols <xref ref-type="bibr" rid="pgen.1004524-Li1">[51]</xref>. MEFs were isolated from WT and <italic>Jmjd3</italic> KO embryos at E11.5 <xref ref-type="bibr" rid="pgen.1004524-Li1">[51]</xref>. Immunocytofluorescence was performed according to a standard protocol using primary antibodies as mentioned above and corresponding goat anti-FITC-conjugated rabbit secondary antibodies (115-096-003; Jackson ImmunoResearch Laboratories) or goat anti-Texas Red-conjugated mouse secondary antibodies (81&#8211;6114; ZyMax Grade).</p><p>Chromatin was prepared from lung tissues of E17.5 embryos. ChIP-PCR and ChIP-qPCR were performed using Jmjd3 antibody (Abgent) and selected histone marks, respectively. Immunoprecipitated chromatin DNA and input were used for ChIP-PCR and ChIP-qPCR with specific primers. Briefly, whole lungs of E17.5 embryos were quickly chopped into tiny pieces with a scalpel and fixed in 1% formaldehyde in 1X PBS at room temperature for 15 min. Glycine was added to a final concentration of 0.125 M. Samples were rotated at room temperature for another 5 min, centrifuged at low speed for 5 min, and the supernatants decanted. The pellets were washed once with ice-cold 1X PBS and disaggregated mechanically through a 0.5-&#181;m cell strainer in 1 ml of ice-cold 1X PBS. Immunoprecipitation was performed on single cell suspensions using the ChIP Assay Kit according to the manufacturer's instructions (Millipore, 17&#8211;295). Antibodies used included anti-Jmjd3; anti-mono-, di-, and trimethylated H3K4 antibodies; anti-mono-, di-, and trimethylated H3K9 antibodies; and anti-mono-, di-, and trimethylated H3K27 antibodies. DNA binding of Jmjd3 and methylation of lung marker genes including <italic>AQP-5</italic>, <italic>SP-A</italic>, <italic>SP-B</italic>, <italic>SP-C</italic>, <italic>SP-D</italic>, and <italic>CC10</italic> were evaluated by ChIP-PCR and ChIP-qPCR using the following specific primers: AQP-5-AF, <named-content content-type="gene">5&#8242;-AACCTGCGGAGGGGGAAGGT</named-content>; AQP-5-AR, <named-content content-type="gene">5&#8242;-CGTCCCCACCCCCACTCCAT</named-content>; AQP-5-BF, <named-content content-type="gene">5&#8242;-CACCACCCCAGGGTCCCCAA</named-content>; AQP-5-BR, <named-content content-type="gene">5&#8242;-CCTGCTCTGCGCTCGGCAAT</named-content>; AQP-5-CF, <named-content content-type="gene">5&#8242;-CAGGAGAGCCCAGCACGCAC</named-content>; AQP-5-CR, <named-content content-type="gene">5&#8242;-CAGTGTAGACTGGCCCGGCG</named-content>, SP-A-AF, <named-content content-type="gene">5&#8242;-GCACACGTACGGAAGCCGGA</named-content>; SP-A-AR, <named-content content-type="gene">5&#8242;-CCTGCGGTGCTCAGTGGCAA</named-content>; SP-A-BF, <named-content content-type="gene">5&#8242;-CCTTTCTGCTTCTTTCCTATGGCCG</named-content>; SP-A-BR, <named-content content-type="gene">5&#8242;-AGCAAAACATCAGAACAACCAAAACTA</named-content>; SP-A-CF, <named-content content-type="gene">5&#8242;-ATGGCTGCTTCCTGTCCGGC</named-content>; SP-A-CR, <named-content content-type="gene">5&#8242;-CCCGCACACAGAGCCTGCAA</named-content>, SP-B-AF, <named-content content-type="gene">5&#8242;-AGGGCCCGGACACATAGAGG</named-content>; SP-B-AR, <named-content content-type="gene">5&#8242;-CTGAGGCCCAGGGCAGAGGT</named-content>; SP-B-BF, <named-content content-type="gene">5&#8242;-GGGTGTGAGGTGACACGCCG</named-content>; SP-B-BR, <named-content content-type="gene">5&#8242;-CTGTGGTGGGGGTGACCACG</named-content>; SP-B-CF, <named-content content-type="gene">5&#8242;-TGTCACCAGTGGCACAGTGGAA</named-content>; SP-B-CR, <named-content content-type="gene">5&#8242;-AGGTGAGCACTGCCATACCAGG</named-content>; SP-B-DF: <named-content content-type="gene">5&#8242;-GTAGAGGATTGAGAAGCCT</named-content>; SP-B-DR, <named-content content-type="gene">5&#8242;-CAGCCTGACTTTGTTCAC</named-content>; SP-B-EF, <named-content content-type="gene">5&#8242;-CGTGACTCTCTGAAGAAGGC</named-content>; SP-B-ER: <named-content content-type="gene">5&#8242;-AAACGACACCCTGGAAGTG</named-content>; SP-B-FF, <named-content content-type="gene">5&#8242;-CCCTTGTTTGACGGTGAA</named-content>; SP-B-FR: <named-content content-type="gene">5&#8242;-TGGCTGCCTACTGCTTAGT</named-content>; SP-C-AF, <named-content content-type="gene">5&#8242;-ACAGGCAATCCCAGATCGCTGA</named-content>; SP-C-AR: <named-content content-type="gene">5&#8242;-GTCCCAGCCATCTCTGCCCCT</named-content>; SP-C-BF, <named-content content-type="gene">5&#8242;-AGAGGGGCAGAGATGGCTGGG</named-content>; SP-C-BR, <named-content content-type="gene">5&#8242;-GCTTGGGACAGCACCTGGGG</named-content>; SP-C-CF, <named-content content-type="gene">5&#8242;-AGGCCCAGTCCTTCACCCCTG</named-content>; SP-C-CR, <named-content content-type="gene">5&#8242;-GCCTACTGGAGGATGGACAGTCA</named-content>; SP-D-AF, <named-content content-type="gene">5&#8242;-TCAGAGGACTAATGACAGCCTT</named-content>; SP-D-AR, <named-content content-type="gene">5&#8242;-CAGCAGGGACAGACATACCA</named-content>; SP-D-BF, <named-content content-type="gene">5&#8242;-GGAAGATGGAAGAACAAGGG</named-content>; SP-D-BR, <named-content content-type="gene">5&#8242;-CTACAAAGGCAGCAACCTG</named-content>; SP-D-CF, <named-content content-type="gene">5&#8242;-TGTGTGTGTGTGTGTGTGTGA</named-content>; SP-D-CR, <named-content content-type="gene">5&#8242;-CACCTCTGTTTGTCAGGCTC</named-content>; CC10-AF, <named-content content-type="gene">5&#8242;-TCCCACCAGCACCATAGTA</named-content>; CC10-AR, <named-content content-type="gene">5&#8242;-CACCTTCTCCATTTCCACA</named-content>; CC10-BF, <named-content content-type="gene">5&#8242;-CTAACAATGCCCAAGAATCG</named-content> ; CC10-BR, <named-content content-type="gene">5&#8242;-GGAGACCCTTCAGGAATCA</named-content>; CC10-CF, <named-content content-type="gene">5&#8242;-CTCCACTGCCTGAATACTCC</named-content>; CC10-CR, <named-content content-type="gene">5&#8242;-ACTTGGTCATCTTCTCCGTG</named-content>.</p><p>Lungs of E17.5 <italic>Jmjd3<sup>+/+</sup></italic> and <italic>Jmjd3<sup>&#8722;/&#8722;</sup></italic> embryos were collected and prepared for ChIP-PCR as previously described <xref ref-type="bibr" rid="pgen.1004524-Zhao1">[17]</xref>. A total of 200 ng of DNA was used for the ChIP-Seq library construction. Illumina sequencing was performed as previously described <xref ref-type="bibr" rid="pgen.1004524-Barski1">[52]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004524-Wang1">[54]</xref>. Sequencing reads from H3K4me3 and H3K27me3 ChIP-Seq libraries were aligned to the mouse mm8 genome using ELAND software. To reduce PCR amplification bias, only one uniquely mapped read per genomic position was retained. The histone modification changes upon <italic>Jmjd3</italic> knockout were assessed using the fold changes of background-subtracted read densities around 5 kb and 2 kb of the TSSs. The statistical significance of the fold change was assessed using the MA-plot-based method. The fold change and the false discovery rate cutoffs were set to 1.5 and 10%, respectively.</p><p>Data are represented as mean &#177; standard deviation (SD) where indicated. Statistical analysis was performed using the Student's <italic>t</italic> test with GraphPad Prism 4.0 software. A <italic>P</italic> value&lt;0.05 was considered significant.</p>