<p>FOXO3a (sc-11351) and &#946;-actin (sc-1616) were obtained from Santa Cruz Biotechnology (Santa Cruz, CA, USA). Fibronectin (ab23750), vimentin (ab92547), and &#945;-SMA (ab7817) were purchased from Abcam (Massachusetts, US). CD31 (550274) were from BD Biosciences (San Jose, CA, USA). Anti-Mouse CSF1R-PE (AFS98) and Anti-Mouse F4/80 Antigen-APC (BM8) were from eBioscience (San Diego, CA, USA). CCL2 (279-MC), CCL5 (278-RN), IL-6 (206-IL), IL-8 (208-IL), and VEGFA (293-VE) were purchased from R&amp;D systems (Minneapolis, MN, USA).</p><p>HEK293T, A549 and H460 cells were purchased from ATCC (American Type Culture Collection, Manassas, VA, USA). HEK 293T, A549 and H460 cells were maintained in a medium of RPMI 1640 medium supplemented with 10% FBS and 1% penicillin/streptomycin. NFs and CAFs were isolated as previously described [<xref ref-type="bibr" rid="pgen.1006244.ref015">15</xref>]. We isolated CAFs from 15 human lung carcinomas and their corresponding counterpart NFs from their matched non-malignant adjacent tissues, taken at least 10 cm from the outer tumor margin. Both CAFs and NFs expressed fibroblastic markers, such as fibronectin and vimentin. The expression of &#945;-SMA were much stronger in CAFs than corresponding counterpart NFs. NFs and CAFs were maintained in 1:1 mixture of DMEM and F12 medium supplemented with 400 ng/ml hydrocortisone, 200 ng/ml insulin, 15% FBS and 1% penicillin/streptomycin. Cells were maintained in 5% CO<sub>2</sub> incubator at 37&#176;C.</p><p>A total of 15 paired fresh-frozen surgically resected lung tumors (8 adenocarcinomas, 6 squamous cell carcinomas, 1 larger cell carcinomas) and matched non-malignant adjacent tissues were obtained from the first affiliated hospital of Nanjing Medical University. The study protocol has been approved by Ethics Committee of the First Affiliated Hospital and Nanjing Medical University (approval No. 2015-SR-041). Participating subjects provided written informed consent Lung carcinoma samples and normal tissues were confirmed by a pathologist.</p><p>Peripheral blood (10 ml) was drawn from each subject using standardized phlebotomy procedures. EDTA-Blood samples were collected without anti-coagulant and placed into two 5-ml red top vacutainers. After blood coagulation, serum was separated by centrifugation. All specimens were immediately frozen and stored at &#8722;80&#176;C. For miRNAs levels analysis, 25 fmol of spiked-in cel-miR-39 (Applied Biosystems) was added in each plasma sample as an external control to monitor the quality of RNA extraction and normalization analysis.</p><p>Cells were washed and lysed in RIPA lysis buffer with protease inhibitors (Thermo Scientific). The total proteins were separated by 8 or 10% gradients SDS-PAGE gels. Proteins were transferred to a polyvinylidenedifluoride membrane and blocked with 5% nonfat milk. Then the membrane was incubated overnight with primary antibodies. Protein bands were detected by incubation with horseradish peroxidase (HRP)-conjugated antibodies and visualized with an enhanced chemiluminesence reagent.</p><p>The expression levels of miRNAs were analyzed using Taqman MicroRNA Assay Kits (Applied Biosystems, Foster City, CA) specific for hsa-miR-1, hsa-miR-206 and hsa-miR-31. Expression of RNU6B (U6 small nuclear RNA) was used as an endogenous control. Normalization strategy for analysis of serum levels of hsa-miR-1, hsa-miR-206 and hsa-miR-31 were previously described [<xref ref-type="bibr" rid="pgen.1006244.ref050">50</xref>, <xref ref-type="bibr" rid="pgen.1006244.ref051">51</xref>]. Briefly, the raw C<sub>T</sub> data for plasma miRNAs were first normalized using the C<sub>T</sub> RNU6B and then scaled to the spiked-in cel-miR-39 to correct for differences in extraction efficiency. To determine the quantity of VEGF, CCL2, MMP9, CCL5, and SDF-1 mRNA, the cDNA was amplified by real-time PCR with Power SYBR Green PCR Master Mix (Applied Biosystems), and the housekeeping gene GAPDH was used as the internal control. A relative fold change in expression of the target gene transcript was determined using the comparative cycle threshold method (2&#8722;&#916;&#916;CT). All experiments were performed in triplicate. All primers used are listed in <xref ref-type="supplementary-material" rid="pgen.1006244.s013">S1 Table</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Microarray hybridization was performed as previously described [<xref ref-type="bibr" rid="pgen.1006244.ref052">52</xref>]. Briefly, the miRCURY LNATM (Locked Nucleic Acid) microRNA version 11.0 microarray (Exiqon, Denmark, <ext-link ext-link-type="uri" ns0:href="http://www.exiqon.com/microrna-microarray-analysis">http://www.exiqon.com/microrna-microarray-analysis</ext-link>) was used, and each probe was repeated 4 times in the microarray. MiRNAs were labelled with Hy3TM or Hy5TM fluorescent groups using the miRCURYTM Array Power Labeling reagent kit to form fluorescent probes. The background was removed from the signal value, and scale normalization was done. The ratio between groups &gt; 1.5 times or &lt; 0.65 times, and a P value &lt; 0.05 revealed by t test indicated the miRNAs were differentially expressed. The ratio &gt; 1.5 times was defined as up-regulation, while the ratio &lt; 0.65 times was defined as down-regulation.</p><p>CCL2 and FOXO3a 3&#8217;-UTRs containing predicted miR-1, miR-206 and miR-31 binding sites and corresponding mutant sites were amplified by PCR from genomic DNA (HEK293T cells) using pfu DNA polymerase (Stratagene, CA, USA), respectively. The PCR productions containing the wild-type or mutant putative target sites of the CCL2 and FOXO3a 3&#8217;-UTR regions were inserted into untranslated region (UTR) downstream of the luciferase gene in the pMIR-reporter luciferase vector (Ambion). Cells were cotransfected using Lipofectamine (Invitrogen) with 200 ng of wild-type or mutant luciferase reporter plasmid, 100 ng of &#946;-galactosidase (&#946;-gal) plasmid, and 100 nmol of pre-miR-1, pre-miR-206, pre-miR-31 or negative control precursor. Luciferase activity was measured 48 hours after transfection using &#946;-gal for normalization. Experiments were performed in triplicate in three independent experiments. All primers used are listed in <xref ref-type="supplementary-material" rid="pgen.1006244.s013">S1 Table</xref>.</p><p>Cell migration assays were performed using migration chambers. LCCs were co-cultured with CAFs or NFs in complete medium at a ratio 1:1, in 24-well plates for 24 hours. LCCs cells were seeded into a Boyden chamber with serum-free medium. Cells that had not migrated were removed from the interior sides of the chamber by cotton swabs. The exterior sides were fixed with 100% cold methanol and stained with crystal violet. Cells were counted under a microscope.</p><p>Tumor samples were fixed with Z-Fix solution (Anatech LTD, MI, USA) for 24 h and processed by the paraffin-embedded method. The tissues sections (5 mm thick) were then heat-immobilized or pepsin-immobilized according to the manufacturer&#8217;s instructions. Antibodies against vimentin or CD31 were used for the immunostaining and detected through the Dako Envision two-step method of immunohistochemistry (Carpinteria, CA, USA). The relative angiogenesis levels were calculated by microvessel density (MVD) as described [<xref ref-type="bibr" rid="pgen.1006244.ref053">53</xref>]. In short, slides were first scanned under low power (&#215;40) in order to determine three areas with the maximum number of microvessels that were consequently evaluated at &#215; 200 magnifications.</p><p>For immunofluorescence assay, samples were stained with fibronectin, vimentin, and &#945;-SMA after blocking with bovine serum albumin. Samples were incubated with Alexa Fluor 568-conjugated secondary antibody (red) or Alexa Fluor 488-conjugated secondary antibody (green) (Life technologies, Frederick, MD, USA). Microscopic observation was performed under a fluorescence microscope (Zeiss, Thornwood, NY, USA).</p><p>Cells were washed and resuspended in HEPES buffer containing recombinant PI and Annexin V-FITC (BD Biosciences, San Jose, CA, USA). The stained cells were analyzed with flow cytometry.</p><p>For colony formation assay, 1 ml of 0.5% SeaPlaque agarose (BMA, ME, USA) was added to each well of 6-well plates. After solidation, 5 &#215; 10<sup>3</sup> cells were mixed with 2 ml of 0.5% SeaPlaque agarose and added onto the top of the well. The CM were added to the wells and be replaced by every two days. After 12 days of culture, colonies were fixed with 100% methanol for 15 min and stained with 0.1% crystal violet. Colonies with diameter more than 1.5 mm were counted.</p><p>Tumor tissue was minced and digested with dissociation buffer (100 U/ml Collagenase type IV and 100 &#956;g/ml DNase in RPMI + 10% FBS) in a shaking incubator at 37&#176;C for 30 min. Digested tissues were filtered through 70-&#956;m cell strainers. Cells were incubated with Fc Block. To identify TAMs, cells were stained with CSF1R-PE and F4/80-APC at 4&#176;C for 30 min. Unstained control and single-stained cells were prepared in every experiment for gating. Dead cells were gated out by side-scatter and forward-scatter analysis.</p><p>The cells were seeded 24 hours, and transfected with 100 nM of miRNA precursors or 100 nM anti-miRNA inhibitors (Ambion, TX, USA) using Lipofectamine RNAiMAX (Invitrogen, CA, USA) according to the manufacturer&#8217;s instructions. The expression levels of miRNAs were verified by stem-loop qRT-PCR. Cells transfected with scramble oligonucleotides were used as a negative control.</p><p>Lentiviruses were generated by transfection of HEK293T cells with transducing vector and packaging vectors. Virus particles in the medium were initially harvested after 24 h and harvested every 12 or 24 h thereafter. Collected virus particles were filtered and transduced into target cells. Cells underwent two rounds of selection with appropriate antibiotics. Lentiviral plasmids expressing shScr, shCCL2, and shVEGFA were obtained from Sigma Aldrich (St. Louis, MO, USA). Lentiviral plasmids expressing CCL2, VEGFA and FOXO3a were obtained from GeneCopoeia (Rockville, MD).</p><p>The levels of cytokines, growth factors and chemokines in the culture media were assessed by Bio-PlexPro human cytokine, chemokine, and growth factor array (Bio-Rad Life Sciences, CA, USA) using a Luminex 100 plate reader (Bio-Rad Life Sciences, CA, USA) according to the manufacturer&#8217;s protocols. CCL2 and VEGFA protein levels in CM were measured by ELISA kits (R&amp;D systems, Minneapolis, MN, USA) according to the manufacturer&#8217;s protocol.</p><p>4&#8211;6 week aged nude mice of the BALB/c strain were purchased from the Nanjing General Hospital of Nanjing Military Command (Nanjing, China). All nude mice were raised in and all experiments were conducted under SPF-level barrier system. A549 tumor cells (1&#215;10<sup>6</sup>) were commingled with fibroblasts (NFs or CAFs), and mixed with Growth Factor Reduced Matrigel (BD Lifesciences) and injected subcutaneously into the right flank of each animal. Primary tumors and lung tissues were harvested from mice after 6 weeks after injection. Lungs were paraffin-embedded and serial sections were histologically examined with hematoxylin and eosin (H&amp;E) stain.</p><p>For quantitation of lung tumor foci, tumor numbers of 5 serial sections per lung were counted and totaled. The lung metastasis index for each mouse was calculated as the ratio of the number of foci colonies observed in the lungs divided by the mass of the primary tumor (in grams) and normalized to WT as fold changes [<xref ref-type="bibr" rid="pgen.1006244.ref014">14</xref>]. (Mean &#177; SEM; n = 8).</p><p>For neutralizing antibodies (R&amp;D system) treatment, mice received i.p injections of single or combination of anti-mouse CCL2 (MAB-497), anti-mouse VEGF164 (AF-493) or mouse IgG isotype control (MAB002) twice a week starting on day 7 after tumor cell implantation for up to 6 weeks (2 mg/kg/dose). For miRNAs injection treatment, miRNAs were formulated with MaxSuppressor in vivo RNALancerII (Bioo Scientific, Austin, TX, USA) according to the manufacturer&#8217;s instructions. Each does contained 20&#956;g of formulated oligo, which equals 1 mg/kg per mouse with an average weight of 20g. Formulated miRNAs were intravenously (i.v) by tail vein injections every 5 days starting on day 5 after tumor cells implantation.</p><p>Results were analyzed using the version 13 SPSS statistical software (SPSS, Chicago, IL, USA). Quantitative variables were analyzed between two groups using Student's t-test or among multiple groups using one-way analysis of variance (ANOVA). Differences were considered significant at p&lt;0.05.</p>