<p>Two replicate HAD and LAD lines were initiated from the N/NIH HS heterogeneous stock [<xref ref-type="bibr" rid="pgen.1006178.ref071">71</xref>] (i.e., with different parents in two different colonies) and selectively bred using within family selection and a rotational breeding program [<xref ref-type="bibr" rid="pgen.1006178.ref010">10</xref>]. During selection both lines were given free access to food, water and a 10% (v/v) ethanol solution. The selection criteria for the HAD lines were consumption of at least 5.0 g of ethanol/kg body weight/day, with an ethanol to water ratio of at least 2:1, while LAD rats were required to drink less than 1.5 g/kg/day with an ethanol to water ratio of less than 0.5:1. Bi-directional selection for AP was repeated continuously for 30 generations, followed by generations of selection that were interspersed with relaxed selection. Sixty generations were completed at the time of sampling, of which 40 were for and against AP in each replicate.</p><p>Spleen samples from 10 alcohol na&#239;ve rats (5 males and 5 females) of each line and replicate were collected and stored at -80&#176;C. DNA was isolated using Gentra Puregene Kits (Qiagen, Redwood City, CA, USA). About 25 mg of spleen was homogenized in cell lysis solution, followed by proteinase K digestion, RNase A incubation, and DNA precipitation as described in manufacture&#8217;s protocol. DNA purity (260/280 ratio) measured 1.79&#8211;1.94, with the majority &gt;1.85. The DNA yield was more than 200 &#956;g in most samples. One of the samples was removed from the analysis as genetic testing indicated its ID was incorrectly recorded.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Individual DNA from each rat was sequenced in the IU Center for Medical Genomics using an ABI SOLiD 5500xl platform using a 35/75 bp paired end fragments protocol in the first replicate and 75bp single end protocol in the second. We reached an approximate depth of 5x per sample and 50x across each line within a replicate. Reads were mapped to the reference genome (Rattus_norvegicus.Rnor_5.0.71) using LifeScope (<ext-link ext-link-type="uri" ns0:href="http://www.lifescopecloud.com">http://www.lifescopecloud.com</ext-link>). Duplicate reads were removed using Picard (<ext-link ext-link-type="uri" ns0:href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</ext-link>), and reads with mapping quality &lt; 8 were also removed. Variants (SNPs) were called using GATK multi-sample variant calling [<xref ref-type="bibr" rid="pgen.1006178.ref008">8</xref>]. Reads with coverage greater than 4 times the expected read depth across all samples were removed as possible copy number variants or other genome duplication events. A 0.3% sequencing error rate was assumed [<xref ref-type="bibr" rid="pgen.1006178.ref072">72</xref>] and screened for by combining all samples and if the total number of SNPs at that location was less than 0.3% of total reads, the location was removed.</p><p>Allele frequencies in each line and replicate at a given locus were estimated as the total number of non-reference (NR) alleles present across samples within a replicate to the total reads at that locus (N). For estimating LD and HB, genotypes were called for each individual at each polymorphic locus. Gametic phase disequilibrium (or Linkage disequilibrium, LD) was estimated pair wise between all polymorphic loci on the same chromosome as the coefficient of determination (R<sup>2</sup>) using the iterative method [<xref ref-type="bibr" rid="pgen.1006178.ref033">33</xref>]. Only loci with a minor allele frequency (MAF) &gt; 10% were used to calculate LD to avoid issues related to rare alleles. In addition, following the methods of Wang and associates [<xref ref-type="bibr" rid="pgen.1006178.ref073">73</xref>], haplotype blocks (HB) were defined as chromosomal regions flanked by at least one historical recombination event, termed the Four Gamete Rule (FGR) [<xref ref-type="bibr" rid="pgen.1006178.ref023">23</xref>], and as implemented by Haploview [<xref ref-type="bibr" rid="pgen.1006178.ref043">43</xref>]. In brief, blocks [<xref ref-type="bibr" rid="pgen.1006178.ref025">25</xref>] are searched from the start of a region by sequential addition of the next locus, the number of unique gametes between the initial and last locus are counted. When all four gametes are observed between the last locus and any of previous loci in the block, the position of the last locus is regarded as the putative starting point of a new block, and the block size is determined as the sequence length between the start and end positions. The only explanation for observing all four gametes between a pair of loci is the occurrence of at least one historical recombination event [<xref ref-type="bibr" rid="pgen.1006178.ref023">23</xref>]. While strict use of the FGR would require only one recombination event to infer an HB, for purposes of examination of HB as units of selection, a more conservative Minor Gamete Frequency (MGF) was used. Increasing the MGF results in larger HB. The MGF was therefore increased until a near continuous spine of HB resulted as defined and visualized by Haploview. A continuous spine results when nearly all SNPs are contained in HB and a new HB starts where the previous ends. This process of finding a continuous spine will tend to only find common HB, which for our purposes is the desired outcome. However, for purposes of comparison of HB size to the NIH-HS-S (see <xref ref-type="sec" rid="sec019">Methods</xref>-<italic>Estimation of allele frequencies in the NIH-HS parent lines</italic>) we used a MGF of 6.25% (1/16) because allele frequencies in that population were coarse with the smallest expected non-zero MGF = 1/2N for N = 8 lines.</p><p>The initial allele frequency was estimated from the parent lines that composed the NIH-HS. The 8 parent lines of the NIH-HS i.e., ACI, BN, BUF, F344, M520, MR, WKY and WN, were recently sequenced (Rat Genome Sequencing and Mapping Consortium, 2013). We downloaded the genomic data (ERP001923) from the NCBI Sequence Read Archive (SRA) and mapped the sequence to the reference genome used in this study (Rattus_norvegicus.Rnor_5.0.71). Our results were essentially the same of that of the Consortium, except we detected that some SNP loci were still segregating within some lines, although the differences were minor. Allele frequency was estimated within each line and locus as the number of SNP reads at that locus over the read depth at that locus. These were then averaged over the 8 parent lines. We refer to the "<italic>in silico</italic>" NIH-HS as the NIH-HS-S line.</p><p>Signatures of selection (SS) were detected as excessive differentiation measured by Wright's <italic>Fst</italic> statistic [<xref ref-type="bibr" rid="pgen.1006178.ref023">23</xref>], but estimated as the intra-class correlation due to between line differentiation (&#952;), based on a simple extension of the method presented by Weir and Cockerham [<xref ref-type="bibr" rid="pgen.1006178.ref015">15</xref>]. Weir and Cockerham (1984) showed that it is possible to define 'F'-statistics based on the method of moments from the analysis of variance. In a nested analysis, variance components can be estimated for 1) between population, 2) within population between individuals, and 3) between alleles within individuals, and 4) total variation. Based on various ratios of these they defined: F, f, and &#952; which correspond to Wright's Fit, Fis, and Fst. Because Fst, estimated as &#952;, was defined as a ratio of the between population variance component to the total variance, we refer to this statistic as an intraclass correlation as defined by Fisher [<xref ref-type="bibr" rid="pgen.1006178.ref074">74</xref>]. The <italic>Fst</italic> statistic is usually found in running windows of arbitrary size [<xref ref-type="bibr" rid="pgen.1006178.ref017">17</xref>]. We chose a window size based on the coefficient of determination (R<sup>2</sup>) between loci as suggested by Weir and colleagues [<xref ref-type="bibr" rid="pgen.1006178.ref034">34</xref>]. The quantity (1- R<sup>2</sup>) is the proportion of the covariation between loci that is independent of linkage. With an R<sup>2</sup> &lt; 50%, the majority of covariation is independent. Thus at a minimum, the distance between loci should be greater than R<sup>2</sup> = 50%. We chose a more conservative critical value of R<sup>2</sup> = 25% (LD_25). The median number of SNPs between loci associated with the LD_25 was used as the size of the sliding window for the <italic>Fst</italic> statistic. Details of the method are as follows</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Following Weir and Cockerham [<xref ref-type="bibr" rid="pgen.1006178.ref015">15</xref>], the <italic>Fst</italic> statistic was estimated based on the following linear model where Y is the presence or absence of the SNP at the q<sup>th</sup> locus in the m<sup>th</sup> gamete
<disp-formula id="pgen.1006178.e002"><alternatives><graphic id="pgen.1006178.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006178.e002.jpg" /><ns1:math id="M2"><ns1:mrow><ns1:msub><ns1:mi>Y</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mi>j</ns1:mi><ns1:mi>k</ns1:mi><ns1:mi>m</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mi>&#956;</ns1:mi><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>L</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>R</ns1:mi><ns1:mrow><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>i</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>S</ns1:mi><ns1:mrow><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>i</ns1:mi><ns1:mi>j</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mi>k</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>G</ns1:mi><ns1:mrow><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>i</ns1:mi><ns1:mi>j</ns1:mi><ns1:mi>k</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mi>m</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow></ns1:math></alternatives><label>(1)</label></disp-formula>
for the i<sup>th</sup> Line (i = 1,2), of the j<sup>th</sup> Replicate (j = 1,2), in the k<sup>th</sup> Sample (k = 1,10), and m<sup>th</sup> gamete (m = 1,2) The expected mean squares for this model, in terms of drift variance, was given by Muir [<xref ref-type="bibr" rid="pgen.1006178.ref032">32</xref>]. From the analysis of variance, variance components for each term were estimated for each locus containing a polymorphic SNP. The <italic>Fst</italic> statistic was estimated as &#952; in sliding windows of size w following Equation 10 from Weir and Cockerham [<xref ref-type="bibr" rid="pgen.1006178.ref015">15</xref>] for multiple loci as
<disp-formula id="pgen.1006178.e003"><alternatives><graphic id="pgen.1006178.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006178.e003.jpg" /><ns1:math id="M3"><ns1:mrow><ns1:msub><ns1:mi>&#952;</ns1:mi><ns1:mrow><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>q</ns1:mi><ns1:mo stretchy="false">)</ns1:mo></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mstyle displaystyle="true"><ns1:munder><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>q</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mi>w</ns1:mi></ns1:mrow></ns1:munder><ns1:mrow><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mi>L</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:msub><ns1:mrow /><ns1:mi>q</ns1:mi></ns1:msub></ns1:mrow></ns1:mstyle></ns1:mrow><ns1:mrow><ns1:mstyle displaystyle="true"><ns1:munder><ns1:mo>&#8721;</ns1:mo><ns1:mrow><ns1:mi>q</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>,</ns1:mo><ns1:mi>w</ns1:mi></ns1:mrow></ns1:munder><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mrow><ns1:mi>L</ns1:mi><ns1:mi>q</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mrow><ns1:mi>R</ns1:mi><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>L</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mi>q</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mrow><ns1:mi>S</ns1:mi><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>R</ns1:mi><ns1:mi>L</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mi>q</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mrow><ns1:mi>G</ns1:mi><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>R</ns1:mi><ns1:mi>L</ns1:mi><ns1:mi>I</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mi>q</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:mstyle></ns1:mrow></ns1:mfrac></ns1:mrow></ns1:math></alternatives></disp-formula></p><p>Statistical Significance was determined by genome wide permutations [<xref ref-type="bibr" rid="pgen.1006178.ref075">75</xref>] whereby the design classifications were randomized onto gametes, following which the statistic was evaluated at all loci across the genome and the largest value recorded. This process was repeated 100,000 times. Among the ordered 100,000 largest values, the genome wide critical <italic>&#952;</italic> value (GWp) corresponded to the p% largest ordered <italic>&#952;</italic> value. Because all SNP locations across the genome were included in the permutation analysis, the critical value is automatically adjusted for multiple comparisons. As an additional level of caution, permutations were also conducted at the individual level, where the design classifications were randomized onto individuals, with their complete set of genotypes across loci. This method of permutation preserves the covariance structure among the gametes within individuals. These were also permutated 100,000 times and the GWp estimated in the same manner. The larger critical value from the two permutation methods was used as the GWp.</p><p>To give a global perspective of the genome-wide magnitude and spatial distribution of SS, we created a Manhattan plot based on the ' &#952; ' values on Chrs 1&#8211;20 and the X. The plot was generated using MATLAB 8.3 (The MathWorks, Inc., Natick, MA, USA).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Significant ' &#952; ' values in windows of running averages were annotated by gene regions. Gene locations and regions were based on the current Ensemble annotation of the rat genome (<ext-link ext-link-type="uri" ns0:href="http://useast.ensembl.org/info/genome/index.html">http://useast.ensembl.org/info/genome/index.html</ext-link>) version 5.0 and included: exon, untranslated region (UTR) and intron. The promoter region was defined as a 10kb domain upstream of the transcriptional start site in order to capture the proximal promoter, closely-linked enhancer elements and regions subject to epigenetic regulation.</p><p>Critical <italic>cis</italic>-acting splicing elements within the intron include the 5&#8217; splice site, the branch point, the polypyrimidine tract and the 3&#8217; splice site were included in the intron-exon junction fragment. Although the branch point is most commonly found within the last 40 nucleotides (nt) of the intron, its position is highly variable and has been mapped up to 400 nt away from the 3&#8217; splice site [<xref ref-type="bibr" rid="pgen.1006178.ref076">76</xref>, <xref ref-type="bibr" rid="pgen.1006178.ref077">77</xref>]. Analysis of several hundred human exons indicates that the majority of branch points are located within 150 bp of the 3&#8217; splice site [<xref ref-type="bibr" rid="pgen.1006178.ref078">78</xref>]. To ensure we captured the majority of branch point sequences, we divided introns into two components: a) intron-exon junction regions (JR) 150bp upstream and downstream of each exon for multi-exon genes and b) intron-non junction (INJR) regions, which capture intronic sequences that extend beyond 150 bp from the exon. The JR include some, but not necessarily all alternative splice sites, while the INJR may pick up additional alternative spice sites and other non-coding regulatory factors, such as enhancers and non-coding RNA binding elements.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Effects of SS in the exonal region were determined by the Ensembl Variant Effect Predictor (VEP): <ext-link ext-link-type="uri" ns0:href="http://www.ensembl.org/info/docs/tools/vep/index.html">http://www.ensembl.org/info/docs/tools/vep/index.html</ext-link> [<xref ref-type="bibr" rid="pgen.1006178.ref079">79</xref>], which uses the <italic>Sorting Intolerant From Tolerant</italic> (SIFT) software (<ext-link ext-link-type="uri" ns0:href="http://sift.jcvi.org">http://sift.jcvi.org</ext-link>) [<xref ref-type="bibr" rid="pgen.1006178.ref080">80</xref>] to predict the effect of SNPs in protein function [<xref ref-type="bibr" rid="pgen.1006178.ref081">81</xref>]. The current version of Ensembl VEP uses rat genome Rnor_6.0 assembly, we therefore convert our data from Rnor_5.0 to Rnor_6.0 using Ensembl Assembly Converter (<ext-link ext-link-type="uri" ns0:href="http://useast.ensembl.org/common/Tools/AssemblyConverter">http://useast.ensembl.org/common/Tools/AssemblyConverter</ext-link>) before uploading data to VEP. To predict the effects of promoter SS on altering putative transcription factor binding, we retrieved 20 bp of flanking sequence (10bp on each side of the SS; 21bp total) from the RGSC Rnor_5.0 rat genome assembly, and put the sequence into PROMO (<ext-link ext-link-type="uri" ns0:href="http://alggen.lsi.upc.es/">http://alggen.lsi.upc.es/</ext-link>) [<xref ref-type="bibr" rid="pgen.1006178.ref082">82</xref>]. The HAD and LAD sequences were analyzed separately for putative transcription factor binding sites and additions or loss of CpG dinucleotide. For the prediction of RNA binding factors on junctional SS, similar to promoter SS, 10bp of flanking RNA sequence (converted T to U) on each side of the junctional SS were input to <ext-link ext-link-type="uri" ns0:href="http://cisbp-rna.ccbr.utoronto.ca/TFTools.php">http://cisbp-rna.ccbr.utoronto.ca/TFTools.php</ext-link> [<xref ref-type="bibr" rid="pgen.1006178.ref083">83</xref>, <xref ref-type="bibr" rid="pgen.1006178.ref084">84</xref>], using the model of PWMs-Log Odds, and threshold of 6. For genes expressed from the Watson strand, we used the published sequence; for genes expressed on the Crick strand, we used the reverse complement. The HAD and LAD sequences were analyzed separately for putative RNA factor binding sites.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Pathways of biological interest were defined and visualized using QIAGEN&#8217;s Ingenuity Pathway Analysis (IPA; Qiagen, Redwood City, CA, USA, <ext-link ext-link-type="uri" ns0:href="http://www.qiagen.com/ingenuity">www.qiagen.com/ingenuity</ext-link>). We did not do an enrichment or over-representation analysis common in RNA-seq studies because, as opposed to an RNA based expression analysis where all pathway members have the opportunity to be expressed, at the DNA level not all genes in a pathway may have had the opportunity to be selected upon due to lack of new mutation within those genes, or intolerance of those genes to new mutations, i.e. pathway analysis would fail to detect pathways under selection if selection did not have an opportunity to function because no new mutations were present to act upon. As such, pathway analysis is expected to exhibit a large number of false negatives and should not be used to prioritize pathways found via number of genes with SS. But, it is also reasonable to assume that multiple genes from a crucial pathway could be under selection and contribute to collaborative function.</p>