<p>All strains used in this study are congenic derivatives of <italic>B. subtilis</italic> PY79 <xref ref-type="bibr" rid="pgen.1004526-Youngman1">[82]</xref>. Genotypes of strains used are provided in <xref ref-type="supplementary-material" rid="pgen.1004526.s011">Table S1</xref>. <italic>B. subtilis</italic> competent cells were prepared as described previously <xref ref-type="bibr" rid="pgen.1004526-Wilson1">[83]</xref>. &#946;-galactosidase activity of cell samples collected at time points indicated was measured with modifications as described previously <xref ref-type="bibr" rid="pgen.1004526-Eswaramoorthy2">[84]</xref>, <xref ref-type="bibr" rid="pgen.1004526-Nicholson1">[85]</xref>. To express <italic>divIVA-linker-cfp</italic> from the <italic>amy</italic> locus, <italic>P<sub>divIVA</sub>-divIVA</italic> without the stop codon was PCR amplified from PY79 chromosomal DNA using primers that abutted <italic>Hind</italic>III and <italic>Nhe</italic>I sites (&#8220;oP10&#8221; <named-content content-type="gene">5&#8242;-AAAAAGCTTTCGTGTTTTCTGAGACA</named-content> and &#8220;oP11&#8221; <named-content content-type="gene">5&#8242;-AAAGCTAGCTTCCTTTTCCTCAAATAC</named-content>). <italic>linker-cfp</italic> was PCR amplified from DS4152 chromosomal DNA <xref ref-type="bibr" rid="pgen.1004526-Patrick1">[44]</xref> using primers abutting <italic>Nhe</italic>I and <italic>BamH</italic>I sites (&#8220;oP54&#8221; <named-content content-type="gene">5&#8242;-AAAGCTAGCGGT TCCGCTGGCTCCGCTGCTGGT TCTGGCCTC</named-content> and &#8220;oP55&#8221; <named-content content-type="gene">5&#8242;-AAAGGATCCTTACTTATAAAGTTCGTCCATGCCAAGTGTAATGCC</named-content>). Both fragments ligated into integration vector pDG1662 to create pPE17. To express <italic>divIVA-linker-gfp</italic> from the <italic>amy</italic> locus, pPE17 was digested with <italic>Hind</italic>III and <italic>Xho</italic>I to liberate <italic>P<sub>divIVA</sub>-divIVA</italic> and part of the linker sequence. Next, <italic>gfp mut2</italic> was PCR amplified from pKC2 <xref ref-type="bibr" rid="pgen.1004526-Ramamurthi2">[86]</xref> using primers that abutted the remaining linker sequences, and 5&#8242; <italic>Xho</italic>I and 3&#8242; <italic>BamH</italic>I sites (&#8220;oP62&#8221; <named-content content-type="gene">5&#8242;-AAACTCGAGGGTTCCGGAATGAGTAAAGGAGAAGAACTTTTC</named-content> and &#8220;oP47&#8221; <named-content content-type="gene">5&#8242;-AAAGGATCCTTATTTGTATAGTTCATCCATGCC</named-content>). Both fragments were ligated into pDG1662 to create pKR227. To replace <italic>divIVA</italic> at the native locus with <italic>divIVA-FLAG-ssrA<sup>Ec</sup></italic>, the final 400 nucleotides of <italic>divIVA</italic> (omitting the stop codon) were PCR amplified using primers that abutted <italic>BamH</italic>I and <italic>Xba</italic>I sites (&#8220;DivIVA-C4005&#8242;Bam&#8221; <named-content content-type="gene">5&#8242;-AAAGGATCCAGTCAGAAAAGATTACGAAATTG</named-content> and &#8220;DivIVA-C4003&#8242; Xba&#8221; <named-content content-type="gene">5&#8242; AAATCTAGATTCCTTTTCCTCAAATACAGCG</named-content>), and ligated into Campbell integration vector pKG1268 <xref ref-type="bibr" rid="pgen.1004526-Griffith1">[52]</xref> to create pKR226. To express <italic>divIVA-FLAG</italic> from <italic>amyE</italic>, <italic>P<sub>divIVA</sub>-divIVA</italic> was PCR amplified from PY79 chromosomal DNA using primers that abutted 5&#8242; <italic>Hind</italic>III and 3&#8242; <italic>BamH</italic>I sites, as well as a 3&#8242; FLAG tag sequence (&#8220;DivIVAprom5&#8242;Hind&#8221; <named-content content-type="gene">5&#8242;-CCCAAGCTTTCGTGTTTTCTGAGACAGCAG</named-content> and &#8220;DivIVA3&#8242;FLAGBam&#8221; <named-content content-type="gene">5&#8242;-CGCGGATCCTTACTTGTCGTCATCGTCTTTGTAGTCTTCCTTTTCCTCAAATACAG</named-content>), and ligated into pDG1662 to create pKR200. To express <italic>spoIIE-gfp</italic> from <italic>amyE</italic> under the control of an IPTG-inducible promoter, <italic>spoIIE-gfp</italic> was PCR amplified from SB201 <xref ref-type="bibr" rid="pgen.1004526-GartiLevi1">[87]</xref> using primers that abutted <italic>Sal</italic>I and <italic>Sph</italic>I sites (&#8220;oP44&#8221; <named-content content-type="gene">5&#8242;- AAAGTCGACACATAAGGAGGAACTACTATGGAAAAAGCAGAAAGAAGAGTGAACGGG</named-content> and &#8220;oP24&#8221; <named-content content-type="gene">5&#8242;- GCCGCATGCTTATTTGTATAGTTCATCCATGCC</named-content>), and ligated into integration vector pDR111 to create pPE19. To express IPTG-inducible <italic>spoIIE-FLAG</italic> from <italic>amyE</italic>, plasmid pPE48 was created by appending 3&#215;-flag tag sequence (<named-content content-type="gene">GATTATAAGGATCATGATGGTGATTATAAGGATCATGATATCGACTACAAAGACGATGACGACAAG</named-content>) followed by a stop codon to the 3&#8242; end of the <italic>spoIIE</italic> coding sequence in pPE19 via QuikChange mutagenesis (Agilent). Strain KR610 (&#916;<italic>spoIIE::tet</italic>) was created by the long flanking homology method <xref ref-type="bibr" rid="pgen.1004526-Wach1">[88]</xref> using primers (&#8220;spoIIEKO-1&#8221; <named-content content-type="gene">5&#8242;-GCAAGTAGCCTTGTTGACAC</named-content>, &#8220;spoIIEKO-2&#8221; <named-content content-type="gene">5&#8242;-CAATTCGCCCTATAGTGAGTCGTTCCTCTCATCTCCCACCTG</named-content>, &#8220;spoIIEKO-3&#8221; <named-content content-type="gene">5&#8242;-CCAGCTTTTGTTCCCTTTAGTGAGCGCTTCCGTATAAATCAAATTTC</named-content>, and &#8220;spoIIEKO-4&#8221; <named-content content-type="gene">5&#8242;-TTTCAAGACATTCACTTCAGAAG</named-content>). For immunoblot analysis, cells were grown as described below, harvested, and cell extracts for immunoblot analysis were prepared by lysozyme treatment as described previously <xref ref-type="bibr" rid="pgen.1004526-Castaing1">[89]</xref>. Extracts were separated by SDS-PAGE and immunoblotted using antisera raised against purified DivIVA-GFP, &#963;<sup>A</sup>, GFP (Covance, Inc.), or <italic>E. coli</italic> FtsZ (courtesy of Sue Wickner; <xref ref-type="bibr" rid="pgen.1004526-Camberg1">[90]</xref>) as indicated. Where indicated, band intensities were quantified using ImageQuant software (GE).</p><p>
<italic>B. subtilis</italic> overnight cultures grown at 22&#176;C in casein hydrolysate (CH) medium were diluted 1&#8758;20 into fresh CH medium and grown until OD<sub>600</sub> reached &#8764;0.5 at 37&#176;C. For induction of sporulation, cells were spun down and resuspended in Sterlini and Mandelstam (SM) medium as described previously <xref ref-type="bibr" rid="pgen.1004526-Sterlini1">[91]</xref>, for the time indicated. Sporulation efficiency was measured by growing cells in Difco sporulation medium (DSM; KD Medical) for at least 24 h at 37&#176;C. The number of heat-resistant colony forming units (cfu) was obtained after incubation at 80&#176;C for 10 min. For strains harboring genes that disrupted the <italic>thr</italic> locus, L-threonine (40 &#181;g/ml final concentration) was added to the culture as a nutritional supplement. Where specified, IPTG (1 mM final concentration) was added at the indicated time to induce genes; xylose (0.5% final concentration), where indicated, was added at the time of resuspension.</p><p>Cells were visualized as described previously <xref ref-type="bibr" rid="pgen.1004526-Ebmeier1">[92]</xref>. Briefly, culture pellets (from 1 mL culture) that were grown as described above was washed with PBS and resuspened in 50&#8211;100 &#181;L PBS containing 1 &#181;g/ml (final concentration) of the fluorescent dye FM4-64 or 46 &#181;g/ml (final concentration) TMA-DPH to visualize membranes and, where indicated, 2 &#181;g/ml (final concentration) of DAPI to visualize DNA. Cells (5 &#181;l) were then placed on a poly-L-lysine coated glass bottom culture dish (Mattek Corp.; poly-L-lysine did not appreciably affect localization of DivIVA-GFP at the polar septum (<xref ref-type="supplementary-material" rid="pgen.1004526.s002">Fig. S2A</xref>)). A pad made of 1% agarose in distilled water (or SM media containing IPTG and FM4-64 for time lapse microscopy) was cut to size and placed above the cells. Cells were viewed at room temperature (or 32&#176;C for timelapse) with a DeltaVision Core microscope system (Applied Precision) equipped with a Photometrics CoolSnap HQ2 camera and an environmental chamber. Seventeen planes were acquired every 200 nm at room temperature and the data were deconvolved using SoftWorx software. Imaris software was used for three dimensional surface rendering of fluorescence data. For 3D-SIM, cells were prepared as described above and imaged using Delta Vision OMX Blaze (Applied Precision). For MSIM, cells (with fluorescent beads when indicated) were labeled with FM4-64 as described above and placed on top of a glass slide and a freshly prepared poly-L-lysine coated coverslip (#1.5 thickness, VWR) was placed on top of the cell suspension; coverslips and slides were cleaned as described previously <xref ref-type="bibr" rid="pgen.1004526-Shroff1">[93]</xref>. Fluorescence images were acquired using a custom MSIM system equipped with a 60&#215; 1.45NA oil objective (Olympus) and appropriate filters (Chroma, zt405/488/561 (dichroic) LP02-488RE-25, NF03-561E-25 (emission filters). Total exposure times for each 2D slice were either 1 s or 2 s depending on the signal intensity. 2D slices were acquired every 200 nm along the Z-axis for construction of 3D volumes, comprising a total axial range of 10&#8211;12 &#181;m for each sample. The longer wavelength channel (red) was collected prior to the shorter wavelength channel (green) for all samples. Multicolor 100 nm diameter Tetraspeck microspheres (Life Technologies, T-7279) were immobilized on the coverslip surface to enable precise alignment of each image channel. Alignment of the two channels was completed by translating the red channel relative to the green channel to maximize overlap of the reference microspheres using ImageJ. Images were deconvolved with a 3D Richardson-Lucy algorithm implemented in the python programming language (40 iterations, using a Gaussian PSF with x, y, z FWHM values 3.8&#215;3.8&#215;2.4 pixels, code available at code.google.com/p/msim) <xref ref-type="bibr" rid="pgen.1004526-York1">[64]</xref>. ISIM imaging was performed using a previously described system <xref ref-type="bibr" rid="pgen.1004526-York2">[65]</xref>. Cells were prepared as described above. Fluorescence from labeled membranes was excited at 561 nm and GFP signal from fusion proteins was excited at 488 nm. A 525 nm bandpass filter (Semrock, FF01-525/50-25) was used for GFP collection and a 561 nm notch filter (Semrock, NF03-561E-25) was used for collecting membrane fluorescence. Exposure times for imaging the membrane probe were 40 ms or 80 ms (depending on sample brightness) and 400 ms for imaging GFP fusion constructs. 3D stacks were collected with a z step size of 200 nm. Resulting images were deconvolved using Richardson-Lucy deconvolution, and image channels were aligned using the multicolor beads for reference as described above. Line scans were completed using the line scan function in ImageJ, setting the line width to 5-pixels. Line scan measurements were exported to Microsoft Excel for rendering and display.</p><p>FLAG-tagged proteins were immunoprecipitated from strains coproducing FLAG- and GFP-tagged proteins using the FLAG Immunoprecipitation kit (Sigma). Samples were processed largely as described previously <xref ref-type="bibr" rid="pgen.1004526-Eswaramoorthy2">[84]</xref>. Briefly, a 20 ml culture of cells was induced to sporulate for 1.5 h, harvested, resuspended in 500 &#181;l protoplast buffer (0.5 M sucrose, 20 mM MgCl<sub>2</sub>, 10 mM potassium phosphate [pH 6.8], 0.1 mg/ml lysozyme), and incubated at 37&#176;C for 15 min to remove cell wall <xref ref-type="bibr" rid="pgen.1004526-Ramamurthi3">[94]</xref>. Protoplasts were harvested and cell extracts were prepared by resuspension in 1 ml of lysis buffer (50 mM Tris [pH 7.4], 150 mM NaCl, 1 mM EDTA, 1% Triton X-100). 100 &#181;l of the extract was retained for analysis as the &#8220;load&#8221; fraction. The remaining cell extract was added to 20 &#181;l of anti-FLAG affinity resin and incubated overnight at 4&#176;C with light shaking. The antibody resin was centrifuged and supernatant was collected as the &#8220;unbound&#8221; fraction. The resin was then washed extensively with lysis buffer. Proteins associated with the resin were eluted through competitive elution with FLAG peptide-containing lysis buffer. Fractions were analyzed by immunoblotting using specific antisera as described above.</p>