<p xmlns:ns0="http://www.w3.org/1999/xlink">All sequence alignment, genotyping, and quality-filtering methods were described previously [<xref ref-type="bibr" rid="pgen.1005851.ref019">19</xref>]. Genotypes for all six canid genomes in that study were benchmarked against high quality genotypes from the Illumina CanineHD BeadChip, and showed a high degree of concordance with the chip data (e.g., 99.4% &#8722; 99.9% of heterozygous genotypes are confirmed by the CanineHD BeadChip). Sequence data are available at <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/bioproject/PRJNA274504">http://www.ncbi.nlm.nih.gov/bioproject/PRJNA274504</ext-link>. Vcf files can be obtained via the Dryad data repository at doi:<ext-link ext-link-type="uri" ns0:href="http://dx.doi.org/10.5061/dryad.sk3p7">10.5061/dryad.sk3p7</ext-link>.</p><p>We chose lineages to sequence with the goal of elucidating the timing, demographic context, and geographic origins of dogs. We selected the Basenji and Dingo for sequencing, as they represent two divergent breeds basal on the dog phylogeny [<xref ref-type="bibr" rid="pgen.1005851.ref035">35</xref>]. We also utilized the Boxer reference genome as an additional haploid chromosome set. The Chinese, Croatian, and Israeli wolves represent lineages sampled geographically from the three regions from which dogs were previously hypothesized to have originated (East Asia, Europe, and Middle East, respectively). The golden jackal was chosen as an outgroup. This sampling strategy is also informative for understanding selection early in the dog lineage, as it captures the range of variation found in both dogs and wolves, thus minimizing the confusion of selection signals from later, lineage-specific effects, such as might occur were we to bias sampling towards modern breeds of European origin. All genotypes initially generated in CanFam 3.0 reference genome coordinates by Freedman et al. [<xref ref-type="bibr" rid="pgen.1005851.ref019">19</xref>] were converted to the most current version, CanFam 3.1.</p><p>Demographic factors, such as population expansions, bottlenecks, or population structure, are confounders that distorts expected signatures of recent positive selection [<xref ref-type="bibr" rid="pgen.1005851.ref009">9</xref>,<xref ref-type="bibr" rid="pgen.1005851.ref011">11</xref>,<xref ref-type="bibr" rid="pgen.1005851.ref012">12</xref>,<xref ref-type="bibr" rid="pgen.1005851.ref087">87</xref>]. Since most domesticated species experience a population bottleneck [<xref ref-type="bibr" rid="pgen.1005851.ref088">88</xref>], and a frequent mode of selection during domestication is selection from standing variation [<xref ref-type="bibr" rid="pgen.1005851.ref040">40</xref>,<xref ref-type="bibr" rid="pgen.1005851.ref089">89</xref>], detection of positive selection during domestication is difficult. To detect selective sweeps on the dog lineage during domestication, we selected three statistics that have been shown to have the highest power to detect selection under these conditions [<xref ref-type="bibr" rid="pgen.1005851.ref089">89</xref>]: F<sub>ST</sub> [<xref ref-type="bibr" rid="pgen.1005851.ref040">40</xref>], &#916;&#960; [<xref ref-type="bibr" rid="pgen.1005851.ref040">40</xref>], and &#916;TD [<xref ref-type="bibr" rid="pgen.1005851.ref040">40</xref>,<xref ref-type="bibr" rid="pgen.1005851.ref090">90</xref>]. We used a sliding window approach in which we divided the reference genome into overlapping windows of size 100kb with 10kb increments. For each 100kb-window, we computed summary statistics using only sites that passed the genome and sample-level filters [<xref ref-type="bibr" rid="pgen.1005851.ref019">19</xref>]. We considered the boxer reference haplotype when we compute statistics within the dog sample or between the dog and wolf sample. Because our analysis included a mixture of haploid (boxer reference) and diploid samples, we calculated F<sub>ST</sub> from estimates of nucleotide diversity, i.e. (&#960;<sub>between</sub> &#8722; &#960;<sub>within</sub>)/ &#960;<sub>between</sub>, where nucleotide diversities are average per site estimates calculated across all pass filter sites within a 100kb window. We computed &#916;&#960; as &#960;<sub>wolf</sub>/&#960;<sub>dog</sub> in each window, and report values on a log-scale (i.e. log(&#916;&#960;)) and &#916;TD is computed as the difference in Tajima&#8217;s D between the wolf and dog sequences. In cases where &#960;<sub>dog</sub> was zero, we added a small fractional increment so that &#916;&#960; would still be computable. In cases where no segregating sites within dogs exist in a window, we did not calculate &#916;TD.</p><p>We obtained 195,998 sliding windows of size 100kb with 10kb increments genome-wide. We then discarded any windows in which the number of fully observed sites is less than 30kb, because it is more likely that those windows are within or close to repeat/CNV regions or regions of poor sequencing quality.</p><p>In order to minimize the confounding of neutral, demographic signals with those produced by positive selection, we developed an approach to control for the false positive rate with simulations. Using the posterior mean parameter estimates from the best-fitting model (Fig 5A in [<xref ref-type="bibr" rid="pgen.1005851.ref019">19</xref>]), we first simulated 200,000 100 kb windows using the program ms [<xref ref-type="bibr" rid="pgen.1005851.ref091">91</xref>] under the demographic model that we previously inferred from the same seven genomes (including boxer) analyzed here [<xref ref-type="bibr" rid="pgen.1005851.ref019">19</xref>], and computed &#916;&#960;, F<sub>ST</sub>, and &#916;TD for those windows. Next, for each window in our observed data, we computed the proportion of simulated windows with a test statistic &#8805; that in an observed window, i.e. a p-value defined as the probability of the observed data under neutrality. We then computed FDR using the Benjamini-Hochberg procedure for our set of windows based upon our empirical data. For each test statistic, we retained windows with FDR&#8804;0.01. We considered the effects of simulation strategy on FDR calculations and outlier window identification by drawing 1000 samples from the joint posterior distribution of our best-fitting demographic model and simulating each of these samples 200 times for a total of 200,000 100kb windows. As the sliding window approach means that windows are often clustered across the genome (<xref ref-type="fig" rid="pgen.1005851.g001">Fig 1</xref>), we collapsed windows across statistics into outlier regions if they were within 200kb of each other. As a heuristic to rank outlier regions, we took a &#8220;composite of multiple signals&#8221; (CMS) approach, similar to that employed in a recent selection scan in humans [<xref ref-type="bibr" rid="pgen.1005851.ref043">43</xref>]. Specifically, we computed CMS<sub>1-FDR</sub> as (1-FDR<sub>&#916;&#960;</sub>)*(1-FDR<sub>FST</sub>)*(1-FDR<sub>&#916;TD</sub>), and ranked regions according to the maximum CMS<sub>1-FDR</sub> observed in any of the 100kb windows comprising a region. We note that given &#916; Tajima&#8217;s D cannot be calculated for windows with no polymorphism, we have excluded windows with strong signals in F<sub>ST</sub> and &#916;&#960; that are potentially of interest (orange points on far right of <xref ref-type="fig" rid="pgen.1005851.g004">Fig 4A</xref>).</p><p>To contrast our demography-informed approach of controlling for FDR with neutral simulations, we examined overlap of outliers based upon this approach with a fundamentally different, &#8220;demography-free&#8221; approach that examines an arbitrary percentage of a top set of outliers, similar to many genome-wide selection scans that don&#8217;t explicitly include demography. In this approach, for each summary statistic (F<sub>ST</sub>, &#916;&#960; and &#916;TD), we computed empirical percentiles by ranking each window by the summary statistic in question and transforming the ranks to percentiles (% F<sub>ST</sub>, % &#916;&#960; and % &#916;TD). We then calculated a &#8220;joint&#8221; empirical percentile by computing the product of the empirical percentiles obtained for the three summary statistics in each window [(% Product) = (% F<sub>ST</sub>) * (% &#916;&#960;) * (% &#916;TD)] and then ranking each window by the products (% Product) and transforming the ranks to percentiles (% Joint). In order to draw Manhattan plots, we transformed the joint empirical percentiles defined for each window into joint empirical p-values. Joint empirical p-values are defined as a probability of obtaining a joint empirical percentile greater than or equal to that observed for the window in question. For the joint empirical percentile, we defined the top 1% windows as outlier windows. As with our primary approach described above, windows &#8804; 200kb apart were collapsed into outlier regions. For this method, we ranked outlier regions by the maximum joint percentile.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For our analysis of the distribution of sites fixed between dogs and wolves, we first identified sites where the Basenji and Dingo were homozygous for the Boxer reference derived allele, and where the three wolves and golden jackal were fixed for an alternative (i.e. the ancestral) allele. We then reduced this set of candidate sites by only including sites where the dog derived allele was observed across the 12-breed genome sequences at a frequency &#8805; 0.75. We evaluated the functional consequences of dog-specific non-synonymous variants using Ensembl&#8217;s Variant Effect Predictor (<ext-link ext-link-type="uri" ns0:href="http://www.ensembl.org/info/docs/tools/vep/index.html">http://www.ensembl.org/info/docs/tools/vep/index.html</ext-link>).</p><p>To detect functional enrichment within the genes intersecting our outlier regions, we used the program DAVID [<xref ref-type="bibr" rid="pgen.1005851.ref074">74</xref>], with the <italic>Canis lupus</italic> gene set as background. We focused on the genes falling within 25kb of the peak CMS<sub>1-FDR</sub> signal for the top 100 regions, minus those regions that did not also show a reduction in diversity in the 12-breed data set. Because enrichment analyses require a relatively large input set of genes in order to detect enrichment patterns, and given that we already perform statistical inference to identify regions under selection, we report all categories with FDR &#8804; 10%. We also report uncorrected P-values as well as P-values corrected for multiple comparisons using Benjamini&#8217;s method, although the latter are generally considered to be extremely conservative [<xref ref-type="bibr" rid="pgen.1005851.ref074">74</xref>].</p>