<p>All procedures of this study involving animals were reviewed and approved by the Animal Welfare and Research Committee at Lawrence Berkeley National Laboratory.</p><p>Embryonic heart or limb tissue was isolated from CD-1 strain mouse embryos at E11.5 by microdissection in cold PBS <xref ref-type="bibr" rid="pgen.1004610-Visel4">[27]</xref>. A single sample consisting of tissue pooled from multiple embryos was analyzed for either tissue. After washing, about 1 ml TRIzol reagent (Life Technologies, 15596-026) was added to every 100 mg of tissue sample, followed by homogenization using a glass dounce homogenizer. Total RNA from individual tissues were extracted following the manufacturer's instructions. Genomic DNA contamination was removed by using the TURBO DNA-free kit (Applied Biosystems, AM1907) following manufacture's protocol, and the RNA samples were stored at &#8722;80&#176;C before further processing.</p><p>In order to perform the transcriptome analysis by Illumina sequencing, ribosomal RNAs was removed from total RNA (5&#8764;10 &#181;g per reaction) by using two rounds of the RiboMinus Eukaryote Kit for RNA-Seq (Life Technologies, A10837-08) following the manufacturer's instructions. The quality of total RNA after rRNA removal was analyzed on RNA 6000 Pico chip (Agilent, 5067-1513) to assure that rRNA contamination was less than 30%. 100 ng total RNA after rRNA removal were used to construct the individual sequencing libraries for Illumina sequencing. Strand-specific RNA-Seq libraries were created following in-house protocols. Briefly, RNA samples were fragmented with 10&#215;Fragment buffer (Ambion, AM9938) to achieve an average fragment size of 200&#8211;300 nt. First strand cDNA synthesis was performed with random hexamer and Superscript II reverse transcriptase (Life Technologies, 18064-014). During the second strand synthesis, dUTP was used instead of dTTP to introduce strand-specificity. After adaptor ligation and size selection, the second strand containing dUTP was cleaved by AmpErase UNG (Life Technologies, N8080096). The resulting strand-specific cDNA was subjected to 12 cycles of PCR amplification and sequenced with HiSeq 2000 instrument. 50 sequencing cycles were carried out.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Raw Illumina reads (50 bp) were first filtered using the Illumina CASAVA-1.8 FASTQ Filter module (<ext-link ext-link-type="uri" ns0:href="http://cancan.cshl.edu/labmembers/gordon/fastq_illumina_filter/">http://cancan.cshl.edu/labmembers/gordon/fastq_illumina_filter/</ext-link>). The remaining sequence tags were mapped back to the mouse genome (NCBI build 37, mm9) using bowtie2 <xref ref-type="bibr" rid="pgen.1004610-Langmead1">[41]</xref>, and the alignments were extended to 200 bp in the 3&#8242; direction to account for the average length of DNA fragments. Repetitively mapped reads were excluded from the following analysis. For <italic>de novo</italic> peak calling, a sliding window method EnrichedRegionMaker module from USEQ <xref ref-type="bibr" rid="pgen.1004610-Nix1">[42]</xref> was employed. For eRNA-based enhancer predictions, a conservative threshold of 10 or more reads (without considering strand specificity) was chosen based on the observation that in retrospective comparison with <italic>in vivo</italic> validated enhancers, 40.7% of enhancers met or exceeded this expression threshold, compared to 4.5% of random control regions (p&#8202;=&#8202;5.5E-108, <bold><xref ref-type="supplementary-material" rid="pgen.1004610.s006">Table S2</xref></bold>). Enriched regions overlapping with refGene, mouse mRNA, or ESTs (mm9) were also removed before the downstream analysis. This process was performed individually for heart and limb RNA-Seq data. To generate <bold><xref ref-type="supplementary-material" rid="pgen.1004610.s002">Figure S2B</xref></bold>, 10% to 100% of sequencing reads were randomly selected from the raw sequencing data, and <italic>de novo</italic> peak calling was individually performed to identify the enriched intergenic regions.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Among raw enriched regions, tissue-specifically transcribed regions (TSTRs) were defined as non-coding regions with significantly higher expression in this tissue compared with the other tissue (p&lt;0.01, two-proportion z-test; <xref ref-type="fig" rid="pgen-1004610-g003">
<bold>Figure 3A&#8211;B</bold>
</xref>) <xref ref-type="bibr" rid="pgen.1004610-Kal1">[43]</xref> with the equation shown below:<disp-formula id="pgen.1004610.e001"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004610.e001.jpg" /></disp-formula>where <inline-formula><inline-graphic ns0:href="pgen.1004610.e002.jpg" /></inline-formula> (n represents mappable reads within each TSTR in heart or limb, and N represent the total number of mappable reads excluding ribosomal regions in the corresponding tissue) and <inline-formula><inline-graphic ns0:href="pgen.1004610.e003.jpg" /></inline-formula>. RPKM&lt;2<sup>&#8722;9</sup> were arbitrarily set to 2<sup>&#8722;9</sup> for visualization purposes in <xref ref-type="fig" rid="pgen-1004610-g003">
<bold>Figure 3A&#8211;B</bold>
</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Candidate transcription start sites (TSSs) marked by CAGE peaks were downloaded from <ext-link ext-link-type="uri" ns0:href="http://fantom.gsc.riken.jp/5/">http://fantom.gsc.riken.jp/5/</ext-link>
<xref ref-type="bibr" rid="pgen.1004610-XX1">[30]</xref> and extended to 1 kb each side from the peak midpoint. For each TSTR (1 kb around the peak center), the overlapping candidate TSSs were identified by BEDTools <xref ref-type="bibr" rid="pgen.1004610-Quinlan1">[44]</xref>. Random control peaks were also generated using BEDTools with the same number and size of sequences and excluding known genes, mouse mRNAs and ESTs.</p><p>We compared tissue-derived RNA signatures at intergenic regions to enhancer-associated p300 <xref ref-type="bibr" rid="pgen.1004610-Visel4">[27]</xref> and H3K27ac marks from the same tissues and time-point. H3K27ac ChIP-Seq datasets are described in more detail in Nord et al. <xref ref-type="bibr" rid="pgen.1004610-Nord1">[29]</xref> and Attanasio et al. <xref ref-type="bibr" rid="pgen.1004610-Attanasio2">[45]</xref>. Candidate tissue-specific intergenic enhancers were predicted by ChIP-Seq of p300 (171 in heart, 656 in limb) or H3K27ac (6965 in heart, 2174 in limb) as described previously <xref ref-type="bibr" rid="pgen.1004610-Visel4">[27]</xref>. Briefly, uniquely aligned sequencing reads were extended to 300 bp in the 3&#8242; direction. Enriched regions (peaks) were identified with MACS <xref ref-type="bibr" rid="pgen.1004610-Zhang1">[46]</xref> (p&#8804;1E-5) using matched input as controls. Peaks overlapping with repetitive regions, known genes, mouse mRNAs and ESTs were removed for further analysis.</p><p>Summary eRNA coverage plots were generated for p300- and/or H3K27ac-derived intergenic enhancers within a 10 kb window, centering on the maximum ChIP-seq coverage. Using the mapped reads, normalized mean eRNA coverage values were calculated for 25 bp windows across the 10 kb regions scaled by total mapped reads. For mean calculations, only the 5<sup>th</sup>&#8211;95<sup>th</sup> percentiles were used to reduce the effect of outliers. Coverage was calculated separately for antisense and sense reads, and as a combined value. For the summary plots, a loess best fit line was plotted for each of the eRNA datasets (limb and heart), separating into sense and antisense reads (<xref ref-type="fig" rid="pgen-1004610-g002">
<bold>Figure 2C&#8211;F</bold>
</xref>).</p><p>Pre-computed conservation scores (phastCons scores) generated from 30 vertebrate genome alignments were download from the UCSC Genome Browser <xref ref-type="bibr" rid="pgen.1004610-Siepel1">[47]</xref>. For each TSTR (1 kb around the peak center), the conservation score was defined as the most highly constrained overlapping phastCons element in the mouse mm9 genome. Random control peaks were generated using BEDTools with the same number and size of sequences and excluding known genes, mouse mRNAs and ESTs <xref ref-type="bibr" rid="pgen.1004610-Quinlan1">[44]</xref>. The percentages of TSTRs and random control regions overlapping phastCons elements were plotted in <xref ref-type="fig" rid="pgen-1004610-g004">
<bold>Figure 4A</bold>
</xref>.</p><p>Tissue-specific TSTRs were classified as enriched in p300 and/or H3K27ac if the relative ChIP-seq coverage was equal to or greater than the 95<sup>th</sup> percentile of experiment background coverage estimated across 1 Mb of unique sequence. After classification, coverage heatmaps were generated for ChIP-seq data using normalized coverage values, with input corrections. Coverage was plotted for 25 bp windows centered on the peak RNA coverage and extending 25 kb on either side. For plotting purposes, coverage was centered and scaled using mean and SD in order to compare signal across datasets. TSTRs were organized as no H3K27ac and p300 signal, enriched in H3K27ac signal only, enriched in p300 signal only and enriched in both marks from the top to the bottom in <xref ref-type="fig" rid="pgen-1004610-g004">
<bold>Figure 4C&#8211;D</bold>
</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Known heart or limb enhancers were downloaded from Vista Enhancer Browser (<ext-link ext-link-type="uri" ns0:href="http://enhancer.lbl.gov">http://enhancer.lbl.gov</ext-link>). For known enhancer regions, the expression level of individual eRNAs was defined as the mapped sequencing reads within a 2 kb window around the center of <italic>in vivo</italic> tested enhancers. For eRNAs only expressed in one tissue, the mapped number of reads was arbitrarily set to 1 in the other tissue in order to compute the absolute fold change for plotting purposes in <xref ref-type="fig" rid="pgen-1004610-g002">
<bold>Figure 2B</bold>
</xref>. Fold change was defined as higher expression level divided by lower expression of each eRNA in two tissues. For the volcano plot, y axis represents p-value for the expression differences of each known enhancer, which was computed by two-proportion z-test <xref ref-type="bibr" rid="pgen.1004610-Kal1">[43]</xref>.</p><p>Coverage of randomly selected control regions (excluding known genes, mRNA and ESTs) was also computed and iterated 100 times to estimate the genome-wide background based on normal distribution. The percentages of enhancers or the average percentage of control regions with indicated numbers of uniquely mapped reads in either tissue are listed in <bold><xref ref-type="supplementary-material" rid="pgen.1004610.s006">Table S2</xref></bold>, as well as associated p-values.</p><p>After peak calling, for each individual TSTR, normalized RPKM (Reads Per Kilobase per Million mapped reads) was calculated in two tissues (heart and limb) with the raw mapped RNA-Seq data within a 2 kb window around the center of each TSTR. Then, a tissue-specificity index was computed as (s&#8722;u)/(s+u), in which s is the expression of TSTR in the matching tissue and u is its expression in the other tissue. The expression of mouse refGene (mm9) was also analyzed in the same way by computing the RPKM across annotated cDNA regions in two tissues.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The tissue-specific expression correlation between TSTRs and their nearby genes was computed as described <xref ref-type="bibr" rid="pgen.1004610-Kim1">[18]</xref> with minor modifications. Briefly, we paired each TSTR with the nearby genes. For each set of genes with the same ranked distance to TSTRs (the first to the fifth closest genes), genes were ranked based on tissue-specificity indices and grouped into 20 genes per bin. Average tissue-specificity indices from each bin were used to compute the correlation. The Pearson correlation between nearby genes and the corresponding TSTRs was conducted with the statistics module in the R package (<ext-link ext-link-type="uri" ns0:href="http://cran.r-project.org/">http://cran.r-project.org/</ext-link>).</p><p>Gene ontology analysis for the genes near TSTR regions was performed by GREAT version 2.02 <xref ref-type="bibr" rid="pgen.1004610-McLean1">[48]</xref>. Enriched GO biological processes with a binomial p-value and fold enrichment were listed in <xref ref-type="table" rid="pgen-1004610-t001">
<bold>Table 1</bold>
</xref>.</p><p>For TSTRs in heart and limb, enriched motifs were computed within a 2 kb window around the center of individual TSTRs by the motif finding module of HOMER (Hypergeometric Optimization of Motif EnRichment) <xref ref-type="bibr" rid="pgen.1004610-Heinz1">[49]</xref>. Known motifs for transcription factors with a p-value less than 10<sup>&#8722;2</sup> compared with random genomic sequences were reporter in <bold><xref ref-type="supplementary-material" rid="pgen.1004610.s009">Table S5</xref></bold> and <bold><xref ref-type="supplementary-material" rid="pgen.1004610.s010">Table S6</xref></bold>.</p><p>For directionality analysis, the expression of individual TSTRs in sense and antisense strands was defined as the strand-specific mapped sequencing reads within a 2 kb window around the center of TSTRs in either heart or limb. Then the directionality index was defined as |f&#8722;r|/(f+r), in which f is the expression of TSTR in one strand and r is its expression in the other strand in the same tissue.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Total RNA was extracted from independently collected pools of heart or limb tissues with the same method as described before and synthesized into cDNA by reverse transcription using the SuperScript First-Strand Synthesis System (Invitrogen). Candidate TSTRs for RT-PCR validations were randomly selected from the top 30% differentially expressed regions ranked by Z scores. Expression analysis of candidate TSTRs was carried out by real-time PCR using gene-specific primers (<bold><xref ref-type="supplementary-material" rid="pgen.1004610.s008">Table S4</xref></bold>) and KAPA SYBR FAST qPCR Master Mix (KAPA Biosystems) on a Roche LightCycler 480. All primers were designed <italic>in silico</italic> using Primer3 (<ext-link ext-link-type="uri" ns0:href="http://primer3.wi.mit.edu/">http://primer3.wi.mit.edu/</ext-link>) and tested for amplification efficiency. Target gene expression was calculated with the 2<sup>&#8722;&#916;&#916;C<sub>T</sub></sup> method <xref ref-type="bibr" rid="pgen.1004610-Livak1">[50]</xref> and normalized to the <italic>Gapdh</italic> housekeeping gene.</p><p>Candidate enhancers for <italic>in vivo</italic> testing were selected randomly from TSTRs with a p-value less than 0.01. The tested regions included up to 2 kb genomic DNA flanking the TSTRs on either sides. This general transgenic procedure has been described before <xref ref-type="bibr" rid="pgen.1004610-Pennacchio1">[25]</xref>, <xref ref-type="bibr" rid="pgen.1004610-Visel4">[27]</xref>. Briefly, the selected regions were PCR amplified from mouse genomic DNA and cloned into the Hsp68-promoter-LacZ reporter <xref ref-type="bibr" rid="pgen.1004610-Kothary1">[51]</xref>, <xref ref-type="bibr" rid="pgen.1004610-Nobrega1">[52]</xref>. Genomic coordinates and the PCR primers for the cloned regions are listed in <bold>Table 8</bold>. The transgenic embryos were assayed at E11.5 for expression patterns. A positive enhancer is defined as an element with reproducible expression pattern in at least three embryos resulting from independent transgenic integration events <xref ref-type="bibr" rid="pgen.1004610-Visel4">[27]</xref>. For histological analysis, selected embryos were embedded in paraffin and sectioned using standard methods.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">RNA-seq data is available through GEO under accession number GSE58157. <italic>In vivo</italic> transgenic data is available through the Vista Enhancer Browser under the identifiers used throughout this study (<ext-link ext-link-type="uri" ns0:href="http://enhancer.lbl.gov">http://enhancer.lbl.gov</ext-link>).</p>