<p>Mouse studies were performed in strict accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health. All mouse studies were performed at the University of North Carolina (Animal Welfare Assurance #A3410-01) using protocols approved by the UNC Institutional Animal Care and Use Committee (IACUC).</p><p>Recombinant mouse-adapted SARS-CoV (MA15) was propagated and titered on Vero E6 cells. For virus titration half of the right lung was used to assess plaque forming units (PFU) per lung using Vero E6 cells with a detection limit of 100 PFU [<xref ref-type="bibr" rid="pgen.1005504.ref071">71</xref>]. All experiments were performed in a Class II biological safety cabinet in a certified biosafety level 3 laboratory containing redundant exhaust fans while wearing personnel protective equipment including Tyvek suits, hoods, and HEPA-filtered powered air-purifying respirators (PAPRs).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">8&#8211;12 week old female animals from the 8 founder strains (A/J, C57BL/6J, 129S1/SvImJ, NOD/ShiLtJ, NZO/HILtJ, CAST/EiJ, PWK/PhJ, and WSB/EiJ) were obtained from the Jackson labs (<ext-link ext-link-type="uri" ns0:href="http://jax.org">jax.org</ext-link>), and bred at UNC Chapel Hill under specific pathogen free conditions. 8&#8211;20 week old female pre-CC mice were bred at Oak Ridge National Laboratories under specific pathogen free conditions, and transferred directly into a BSL&#8211;3 containment laboratory at UNC Chapel Hill. One preCC mouse per line was infected, for the founder strains at day four n = 2 (A/J), n = 3 (C57BL/6J, 128S1/SvImJ, NOD/ShiLtJ, CAST/EiJ, PWK/PhJ and WSB/EiJ) and n = 5 (NZO/HILtJ). <italic>Trim55</italic>
<sup>-/-</sup> (<italic>Murf2</italic>
<sup>-/-</sup>) mice on a C57BL/6 background were a kind gift from Christian Witt at the University of Mannheim. Validation experiments used 8&#8211;12 week old female mice. All experiments were approved by the UNC Chapel Hill Institutional Animal Care and Use Committee. Animals were maintained in SealSafe ventilated caging system in a BSL3 laboratory, equipped with redundant fans as previously described by our group.</p><p>Animals were lightly anesthetized via inhalation with Isoflurane (Piramal). Following anesthesia, animals were infected intranasally with 10<sup>5</sup> pfu of mouse adapted SARS-CoV (MA15) in 50 &#956;L of phosphate buffered saline (PBS, Gibco), while mock infected animals received only 50 &#956;L of PBS. Animals were weighed daily and at four days post infection, animals were euthanized via Isoflurane overdose and tissues were taken for various assays. No blinding was used in any animal experiments and animals were not randomized; group sample size was chosen based on availability of age-matched mice. Pearson&#8217;s correlation was used to determine any correlation between weight loss and log-transformed viral load in the lung.</p><p>The left lung was removed and submerged in 10% buffered formalin (Fisher) without inflation for 1 week. Tissues were embedded in paraffin, and 5 &#956;m sections were prepared by the UNC Lineberger Comprehensive Cancer Center histopathology core facility. To determine the extent of inflammation, sections were stained with hematoxylin and eosin (H &amp; E) and scored in a blinded manner by for a variety of metrics relating to the extent and severity of immune cell infiltration and pathological damage on a 0&#8211;3 (none, mild, moderate, severe) scale. Significant differences in lung pathology were determined by a two-sample student&#8217;s t test. Images were captured using an Olympus BX41 microscope with an Olympus DP71 camera.</p><p>The right lung of each mouse was used for flow cytometric staining of inflammatory cells. Mice were perfused with PBS through the right ventricle before harvest, lung tissue was dissected and digested in RPMI (Gibco) supplemented with DNAse and Collagenase (Roche). Samples were strained using a 70 micron filter (BD) and any residual red blood cells were lysed using ACK lysis buffer. The resulting single cell suspension was stained with two antibody panels using the following stains (1) FITC anti-Ly-6C clone AL21 (BD), PE anti-SigLecF clone E50-2440 (BD), PETR anti-CD11c clone N418 (MP), PerCP anti-B220 clone RA3-6B2 (MP), PE-Cy7 anti-Gr&#8211;1 clone RB6-8C5 (eBio), eF450 anti-CD11b clone M1/70 (eBio), APC anti-LCA clone 30-F11 (eBio), APC-eF780 anti-MHC class II clone M5/114 (eBio) or (2) FITC anti-CD94 clone 18d3 (eBio), PE anti-CD3&#400; clone 145-2C11 (eBio), PETR anti-CD4 clone RM4-5 (MP), PerCP anti-CD8 clone 53&#8211;6.7 (BD), PE-Cy7 anti-CD49b clone DX5 (eBio), eF450 anti-LCA clone 30-F11 (eBio), AF647 anti-CD19 clone 6D5 (Biolegend), APC-eF780 anti-B220 clone RA3-6B2 (eBio). While this FACS analysis was solely performed on mice of a C57BL6/J background, these antibodies have all been shown recognize the relevant antigens in each of the CC founder lines. Samples were run on a Beckman Coulter CyAN, and data analyzed within the Summit software. Significant differences in lung inflammatory cell populations were determined by a two-sample student&#8217;s t test.</p><p>Genotyping and haplotype reconstruction were done as described in [<xref ref-type="bibr" rid="pgen.1005504.ref023">23</xref>]. Briefly, each pre-CC animal was genotyped using the Mouse Diversity Array [<xref ref-type="bibr" rid="pgen.1005504.ref072">72</xref>] (Affymetrix) at 372,249 well performing SNPs which were polymorphic across the founder strains [<xref ref-type="bibr" rid="pgen.1005504.ref031">31</xref>]. Once genotypes were determined, founder strain haplotype probabilities were computed for all genotyped loci using the HAPPY algorithm [<xref ref-type="bibr" rid="pgen.1005504.ref073">73</xref>]. Genetic map positions were based on the integrated mouse genetic map using mouse genome build 37 [<xref ref-type="bibr" rid="pgen.1005504.ref074">74</xref>].</p><p>Linkage mapping was done as described in [<xref ref-type="bibr" rid="pgen.1005504.ref023">23</xref>]. Briefly, QTL mapping was conducted using the BAGPIPE package [<xref ref-type="bibr" rid="pgen.1005504.ref075">75</xref>] to regress each phenotype on the computed haplotypes in the interval between adjacent genotype markers, producing a LOD score in each interval to evaluate significance. Genome-wide significance was determined by permutation test, with 250 permutations conducted per scan. Phenotype data for mapping either satisfied the assumptions of normality or were log transformed to fit normality (titer data). For the likely regions of identified QTL peaks, SNP data for the eight founder strains from the Sanger Institute Mouse Genomes Project [<xref ref-type="bibr" rid="pgen.1005504.ref037">37</xref>] were downloaded and analyzed as described in Ferris et al [<xref ref-type="bibr" rid="pgen.1005504.ref027">27</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Bagpipe is freely available at <ext-link ext-link-type="uri" ns0:href="http://valdarlab.unc.edu/software.html">http://valdarlab.unc.edu/software.html</ext-link>.</p><p>At two and four days after infection, mice were euthanized and a lung portion placed in RNAlater (Applied Biosystems/Ambion) and then stored at &#8722;80&#176;. The tissues were subsequently homogenized in TriZol (Life Technologies), and RNA extracted as previously described [<xref ref-type="bibr" rid="pgen.1005504.ref012">12</xref>]. RNA samples were spectroscopically verified for purity, and the quality of the intact RNA was assessed using an Agilent 2100 Bioanalyzer. cRNA probes were generated from each sample by the use of an Agilent one-color Quick-Amp labeling kit. Each cRNA sample was then hybridized to Agilent mouse whole-genome oligonucleotide microarrays (4 x 44) based on the manufacturer&#8217;s instructions. Slides were scanned with an Agilent DNA microarray scanner, and the output images were then analyzed using Agilent Feature Extractor software. Microarray data has been deposited in the National Center for Biotechnology Information&#8217;s Gene Expression Omnibus database and is accessible through GEO accession GSE64660.</p><p>Raw Agilent Microarray files were feature extracted Agilent feature extractor version 10.7.3.1. Raw Microarray files were background corrected using the &#8220;norm-exp&#8221; method with an offset of 1 and quantile normalized using Agi4x44PreProcess [<xref ref-type="bibr" rid="pgen.1005504.ref076">76</xref>] in the R statistical software environment. Replicate probes were mean summarized, and all probes were required to pass Agilent QC flags for 75% replicates of at least one infected time point (41,267 probes passed). This microarray analysis was performed only on animals with a C56BL6/J background; thus it was not necessary to correct for probes with SNPs caused by the genetic variation of the other founder lines.</p><p>Differential expression was determined by comparing MA15 infected samples (C57Bl6/J mice vs. <italic>Trim55</italic>
<sup><italic>-/-</italic></sup> mice) with mock and each other to fit a linear model for each probe using the R package Limma. Criteria for differential expression was an absolute log2 FC of 1 and a q value of &lt; 0.05 calculated using a moderated t test with subsequent Benjamini-Hochberg correction. Differentially expressed (DE) genes were observed for both C57BL/6J and <italic>Trim55</italic>
<sup><italic>-/-</italic></sup> infected mice compared to time matched mocks at two and day four post infection. DE analysis was also run on the <italic>Trim55</italic>
<sup><italic>-/-</italic></sup> infected mice against the C57BL/6J infected mice which provided a direct observation of the transcription signatures in the <italic>Trim55</italic>
<sup><italic>-/-</italic></sup> against the MA15 infected mouse background. To identify genes with similar patterns of variation at early and late times post infection, day two and day four gene signatures were intersected separately and then combined. There was no intersection of DE genes between the day two and four time points when the <italic>Trim55</italic>
<sup><italic>-/-</italic></sup> infected mice were run against the C57BL/6J infected mice.</p><p>Functional analysis of statistically significant gene expression changes was performed using the Ingenuity Pathways Knowledge Base (IPA; Ingenuity Systems) [<xref ref-type="bibr" rid="pgen.1005504.ref077">77</xref>]. Functional enrichment scores were calculated in IPA using all probes that passed our QC filter as the background data set.</p>