<p>The Institutional Animal Care and Use Committee approved all procedures. Animals were treated under guidelines of ethical treatment of animals, and approved by IACUC protocol #274005 at Lawrence Berkeley Laboratory. All animal work was conducted according to relevant national and international guidelines.</p><p>The following mouse models were used: homozygous <italic>Hdh(Q150/150)</italic> and heterozygous <italic>Hdh(Q150/wt)</italic> (Lin et al., 2001) and ogg1 null mice (Klungland et al., 1999), which have been previously described and characterized. The <italic>Hdh(Q150/wt)</italic> and <italic>ogg1(-/-)</italic> mice were generated on the C57BL6 background. Therefore, we used <italic>C57BL6</italic> (Cnt) mice as controls, referred to as <italic>Hdh(wt/wt)</italic>. Heterozygous <italic>Hdh(Q150/wt)</italic> (Lin et al., 2001) and ogg1(+/-) mice (Klungland et al., 1999) were bred on C57BL6 background to obtain <italic>Hdh(Q150/Q150)</italic> and <italic>ogg1(-/-)</italic> mice. Each line was characterized by histology and for the level of expressed protein. Genescan analysis, as described previously, determined the size of the CAG repeat tracts as a function of age and brain region (Kovtun et al., 2007). Data were analyzed using GeneMapper software v4. Genescan analysis of the XJB-5-131 treated tissue was performed at LBNL.</p><p>Primary antibodies were: mouse OGG1 (1:1000, a kind gift from Tapas Hazra at University of Texas Medical Branch), mouse monoclonal Huntingtin (HTT) (1:1000, MAB2170, EMD Millipore, MA), and actin-HRP conjugated (1:5000, sc-1616, Santa Cruz Biotechnology). Tissue extracts were prepared in NP-40 lysis buffer (50 mM Tris-HCl pH 8.0, 150 mM NaCl, 1% Igepal Ca-630 and protease inhibitors (Complete, Roche). Tissue was washed twice with ice- cold PBS and resuspended in NP-40 lysis buffer, and kept on ice for 30 min. Then, the cellular suspension was centrifuged at 21,000xg for 5 min and the protein concentration in the supernatant was determined with BioRad DC Protein Assay Kit using albumin as a standard. Twenty five-fifty microgram of protein was separated using 10% SDS-PAGE. Anti-mouse HRP linked secondary antibody (1:1000, #7076S, Cell Signaling) was used and membrane was visualized with Pierce ECL Western Blotting Substrate (#32106, Thermo Scientific) using G:BOX with GeneSnap software form SynGene.</p><p>At least three mice from all nine genotypes and ages were taken into analysis. Mice were decapitated with a guillotine and the brains isolated. Brain hemispheres were post-fixed for at least 24 hours in buffered, 4% PFA. Paraffin- embedded, 4-&#956;m-thick coronal sections were stained using a BondMax&#8482; (Leica Microsystems GmbH/Menarini, Germany) automated immunostaining system. Analysis was conducted on 5&#8211;10 sections per mouse. Sections were pretreated with Citrate, EDTA or Enzyme 1 pretreatment solutions (Menarini, Germany) and immunostained using anti-IBA1 (EDTA pretreatment 20 min, 1:1,000 for 15 min, Wako GmbH, Germany), anti-GFAP (Enzyme 1 pretreatment, 1:500 for 15 min, DAKO, Germany), anti-NeuN (clone A60, Citrate pretreatment 20min, 1:500 for 15 min, Chemicon, Germany), anti-Ubiquitin (clone Ubi-1, EDTA pretreatment 20 min, 1:10,000 for 15 min, Millipore, Germany) and the Bond&#8482; Polymer Refine Detection kit (Menarini, Germany) as described in (Scheffler et al., 2012). Whole tissue sections were fully digitized at a resolution of 230nm using a Mirax Midi slide scanner (Zeiss, Germany) as described in (Krohn et al., 2011) and 10 fields of view (FOV) at a natural magnification (1:1, 230nm per pixel, 53,3 fold on a 24&#8221; screen) were analyzed semi-automatically using the BX Analysis software package and a custom programmed macro (Keyence, Germany).</p><p>Motor testing encompassed both rotarod and grip test, as described (Trushina et al., 2014; Xun et al., 2012). Weight and littersize were also quantified. Animals in each group were evaluated for rotarod performance and grip strength at the indicated ages. Mice were lowered onto the already spinning Rota-Rod (Ugo Basile) at the required speed (10 and 20 rpm were used in this study). The amount of time the animals stayed on the Rota-Rod was determined by a built-in magnetic trip-switch, which was stopped when the animal fell off. Mice were timed on the Rota-Rod for a maximum of 120s, with three attempts given for each mouse to attain 120s. Animals were tested for one session each day at each speed, for 5 consecutive days, and the best times for each trial were averaged for each animal. For grip strength test, mice were lowered onto a parallel rod (D &lt; 0.25 cm) placed 50 cm above a padded surface. The mice were allowed to grab the rod with their forelimbs, after which they were released and scored for their success in holding onto the bar for 30 s. Mice were allowed three attempts to pass the bar test each day of testing, and were tested for 5 consecutive days. Any one successful attempt to hold onto the bar was scored as a pass. The percentage of animals that fell (and failed the test) was measured and recorded as a percentage of the total number of animals tested per genotype and age group. Mice were immediately sacrificed at the end of the 5-day testing period. Average number of mice tested per genotype and age group was greater than 12, with an approximately equal male:female ratio (407 males:351 females) in the 6 genotypes that were focused on for analysis.</p><p>DNA was prepared from mouse brain tissues and tails at the age indicated using the MasterPure Complete DNA and RNA Purification Kit (Epicentre Biotechnologies). Samples were incubated with Proteinase K, RNase A treated, followed by protein precipitation and centrifugation to remove cellular debris. DNA was precipitated with isopropanol, washed and resuspended in H2O. Amplification of CAG repeats from Hdh(Q150/wt)/ogg1(+/+), Hdh(Q150/Q150)/ogg1(+/+), Hdh(Q150/wt)/ogg1(-/-), and Hdh(Q150/Q150)/ogg1(-/-) mouse DNA was performed with a HEX-labeled forward primer (CCCATTCATTGCCTTGCTG) and reverse primer (GCGGCTGAGGGGGTTGA) in 15 &#956;l reactions containing 0.2 mM dNTPs, 2 M betaine, AM buffer (67 mM Tris&#183;HCl, pH 8.8/ 16.6 mM (NH4)SO4/ 2 mM MgCl2/ 0.17 mg/ml BSA) and 1 unit of Kapa Taq HotStart DNA Polymerase (Kapa Biosystems). Cycling conditions were as follows; 5 min at 94&#176;C; 30 s at 94&#176;C, 30 s at 60&#176;C, 3 min at 72oC for 5 cycles; 30 s at 94&#176;C, 30 s at 55&#176;C, 5 min at 72&#176;C for 38 cycles; 5 min at 72&#176;C. Genescan analysis was performed using GeneMapper software v4. The statistical program R was used to separate partially overlapping curves in homozygous animals.</p><p>There are three groups. The initial allele distribution of the inherited repeat is subtracted from the somatic repeats at the age of interest to normalize changes. R uses an iterative curve fitting routine to a Gaussian simple peak shape model. The heterozygous (HdhQ150/wt) animals have only one peak to fit. For homozygous animals (HdhQ150/Q150), if the two peak distributions are coincident or are very far apart, the initial allele distribution of the inherited repeat is the same as for heterozygous animals. If the peaks are partially overlapping, we use iterative fitting routines of R (the statistical program) to resolve them. Mathematical resolution of the two peaks occurs only once (i.e., we do not follow the same animals with time and compound errors by re-fitting the results from the same animals at multiple ages). In our case, we fit to a Gaussian function using two non-linear parameters: peak position and peak width (the peak height is a linear parameter and is determined by regression). In R, peak resolution is not performed by linear least-squares methods because such signals cannot be modeled as polynomials with linear coefficients (the positions and widths of the peaks are not linear functions). Compared to the simpler polynomial least-squares methods for measuring peaks, the iterative method has the advantage of using all the data points across the entire peak, including zero and negative points. This method can be applied to resolve multiple overlapping peaks to a high degree of accuracy.</p><p>Least squares regression analysis (Cohen et al., 1993) was used to compare genotype and motor performance. The ogg1(-/-) and Hdh(Q150/Q150) genotypes could affect overall performance (represented by different intercepts). The ogg1(-/-) and Hdh(Q150/Q150) genotypes could also interact in affecting performance, and potentially include six separate intercepts: six separate age effects, and all their interactions, in addition to covariates. To simplify the model, we included separate intercepts for each genotype in a model that included sex and separate age effects for each Hdh(Q150/Q150) genotype. This allowed us to combine certain genotypic-specific intercepts and age effects into a simpler form that included three intercepts and a regression slope for age. Values were expressed as mean &#177; standard error of the mean (SEM), unless otherwise stated. P-values were obtained from the unpaired two-tailed Student's t-test.</p><p>Statistical analyses of means for three or more groups were performed using one-way analysis of variance (ANOVA) with the categories of genotype and age as independent factors followed by the Newman-Keuls post-hoc test for multiple comparison. For analyses of means involving only two groups with a sample size n&lt;30, the F-test was used to determine whether the variances between the two groups were significantly different. For samples with a significant difference in variance, the Welch&#8217;s t-test was applied. Student&#8217;s t-test was applied for the samples (n &#8805; 30) with an insignificant difference in variance. The significance level was set at 0.05 for all analyses. All statistical computations were carried out using Prism (Graphpad Software).</p><p>Treatment using XJB-5-131 and the motor testing results are previously described (Xun et al., 2012). The tissue used for sizing of the somatic repeats length was obtained from the same animals whose motor performance was reported (Xun et al., 2012). XJB-5-131 was synthesized as described previously (Wipf et al., 2005). Hdh(Q150/Q150)/ogg1(+/+) mice were intraperitoneally injected with 1 mg/kg of XJB-5-131 or phosphate buffered saline three times per week from 7 to 57 weeks. At least seven animals were tested in each age group per genotype.</p><p>The relative level of mtDNA abundance in mouse cerebral cortex was performed as previously described (Ayala- Torres et al., 2000; Siddiqui et al., 2012). The determination of mtDNA abundance consisted of amplifying a 116 bp mtDNA fragment by performing an initial denaturation for 45 s at 94&#176;C, followed by 22 cycles of denaturation for 15 s at 94&#176;C, annealing/extension at 61&#176;C for 45 s, and a final extension for 45 s at 72&#176;C. We used the following primer nucleotide sequences: 5&#8242;-CCCAGCTACTACCATCATTCAAGT-3&#8242; (forward) and 5&#8242;-GATGGTTTGGGAGATTGGTTGATGT-3&#8242; (reverse). The relative copy numbers were calculated as the relative amplification of the Hdh(Q150/Q150)/ogg1(+/+) cortex or the Hdh(Q150/Q150)/ogg1(-/-) cortex compared to the wild-type Hdh(wt/wt)/ogg1(+/+) controls. The results were derived from two qPCR assays in duplicate on each animal. Six mice were used in each analysis.</p>