<p>Written informed consent was obtained from the family in accordance with the Declaration of Helsinki and the study was approved by the Newcastle and North Tyneside 1 Ethics Committee.</p><p>Standard histological and histochemical analyses, including cytochrome <italic>c</italic> oxidase (COX), of a skeletal muscle biopsy were performed according to established protocols <xref ref-type="bibr" rid="pgen.1004424-Old1">[31]</xref>, on fresh-frozen skeletal muscle sections (10 &#181;m). Mitochondrial respiratory chain complex activities were determined in skeletal muscle homogenates as previously described, and expressed relative to the activity of the matrix marker enzyme, citrate synthase <xref ref-type="bibr" rid="pgen.1004424-Kirby1">[32]</xref>.</p><p>Total DNA was extracted by standard procedures from all available tissues obtained with consent from familial relatives, and mtDNA rearrangements were excluded by long-range PCR. Direct sequencing of the entire mitochondrial genome was performed on homogenate skeletal muscle DNA.</p><p>Genomic DNA from the two affected siblings (II&#8211;1 and II&#8211;3) was isolated from blood (DNeasy, Qiagen, Valencia, CA); fragmented to 150&#8211;200 bp with the use of Adaptive Focused Acoustics (Covaris); end-repaired, adenylated, and ligated to adapters (Illumina Paired-End Sample Preparation Kit). Ligated libraries were hybridized with whole-exome baits that covered 27,184 genes (Agilent SureSelect Human All Exon Kit Version 2) with modifications for the SureSelect Human All Exon Kit Illumina Paired-end Sequencing Library (Version 2.0.1). Captured fragments were purified, clonally amplified and sequenced on 2 lanes of an Illumina Genome Analyser IIx using 75 bp paired-end reads.</p><p>The sequence was aligned to the human reference genome (UCSC hg19) with Burrows Wheeler Aligner (BWA) <xref ref-type="bibr" rid="pgen.1004424-Li1">[33]</xref>, then reformatted with the use of SAMtools v0.1.18 <xref ref-type="bibr" rid="pgen.1004424-Lill1">[34]</xref>. 83.1% of exon target sequence was covered by &gt;10 reads. Single base variants were identified with Varscan v2.2 <xref ref-type="bibr" rid="pgen.1004424-Koboldt1">[35]</xref> and Indels were identified with Dindel v1.01 <xref ref-type="bibr" rid="pgen.1004424-Albers1">[36]</xref>. Variants were annotated using wANNOVAR <xref ref-type="bibr" rid="pgen.1004424-Chang1">[37]</xref>. Lists of on-target variants were filtered against data from the National Heart, Lung and Blood Institute (NHLBI, NIH, Bethesda, MD) Exome Sequencing Project (ESP) 6500 exomes, the 1000 Genomes project, and the exome sequences of 315 unrelated in-house control exomes to identify rare homozygous variants with a Minor allele frequency (MAF) &lt;0.01.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Variant filtering led to a final list of 40 rare, homozygous, protein-altering variants of which 4 were mitochondrial according to the Gene-Ontology database. These genes included <italic>TRIT1, CCDC19, ARSB and SYNJ2</italic> of which <italic>TRIT1</italic> segregated with disease in the family. Targeted resequencing and familial segregation studies were performed by cycle sequencing using an ABI 3130xl (Applied Biosystems) system and BigDye Terminator v3.1 technology. The following primer pairs, including universal tags, were employed: forward primer, <named-content content-type="gene">5&#8242;-TGTAAAACGACGGCCAGTAGGGAAAATGCACACTGGAG-3&#8242;</named-content>, and reverse primer, <named-content content-type="gene">5&#8242;-CAGGAAACAGCTATGACCTTCCCTTAGGTCAGATCCAAAA-3&#8242;</named-content>. Analysis of the evolutionary conservation of the mutated amino acid across a range of homologous proteins was performed by the freely available Clustal Omega multiple sequence alignment software (<ext-link ext-link-type="uri" ns0:href="http://www.ebi.ac.uk/Tools/msa/clustalo/">http://www.ebi.ac.uk/Tools/msa/clustalo/</ext-link>) <xref ref-type="bibr" rid="pgen.1004424-Larkin1">[38]</xref>.</p><p>Primary human fibroblast cell lines were established from the patient as well as from controls according to standard protocols and cultured at 37&#176;C, in a humidified, 5% CO<sub>2</sub> atmosphere. Fibroblasts were maintained as monolayers in Minimum Essential Media (MEM) (Life Technologies #21090) supplemented with FBS to 10%, 1x MEM-vitamins, 21 mM L-Glutamine, 1 mM sodium pyruvate, 1x penicillin/streptomycin, 1x non-essential amino acids, and 0.41 &#181;M uridine.</p><p>TRIT1 wild-type and p.Arg323Gln mutant open reading frames were cloned into the pOP retroviral vector (Radichev et al., 2006) in frame with FLAG and HA epitope tags at the 5&#8242; end using the XhoI and NotI sites, and sequencing was performed for confirmation. The preparation of the retroviral supernatants and the transduction of the control and patient fibroblasts were done as described <xref ref-type="bibr" rid="pgen.1004424-Radichev1">[39]</xref>.</p><p>Live cell respiration studies were performed by micro-scale oxygraphy using the Seahorse XF<sup>e</sup> Extracellular Flux Analyzer 24 (Seahorse Bioscience) according to manufacturer's instructions. Fibroblasts were seeded at a density of 30,000 cells/well. Mitochondrial function was assayed through the sequential addition of oligomycin (to 1.3 &#181;M) to block the ATP synthase, 2 additions of carbonyl cyanide 4-(trifluoromethoxy)-phenylhydrazone (FCCP), a respiratory uncoupler which drives maximal respiration (to 2 &#181;M and then to 3 &#181;M), and antimycin (to 2.5 &#181;M) to inhibit Complex III.</p><p>Oxygen consumption rate (OCR) and proton production rate (PPR) measurements for each well were normalized by cell number. Non-mitochondrial respiration was subtracted from all OCR values prior to analysis; spare respiratory capacity (SRC) equals maximal OCR - basal OCR, ATP coupling efficiency equals (basal OCR - oligomycin-inhibited OCR)/(basal OCR*100). Seven separate control cell lines underwent multiple testing and the means were combined to calculate control data (mean &#177; SD; n&#8202;=&#8202;7). Patient fibroblasts were tested multiple times (n&#8202;=&#8202;21). An unpaired, two-tailed Student's t-test was performed to determine the significance of differences between the data sets and P-values were considered significant at the 95% confidence interval.</p><p>Total cellular protein was extracted from patient and control fibroblasts (as well as transfected cell lines), size separated on a 10% separating gel by SDS-PAGE and transferred to a methanol-activated PVDF membrane. Immunoblotting was performed using primary antibodies to NDUFA9, NDUFB8, NDUFA13, SDHA, UQCRC2, MTCOI, MTCOII, COXIV and ATPB (all from Abcam), TRIT1 (GeneTex GTX120508) and &#946;-actin (Sigma A5316) as a loading control and TOMM20 (Abcam) as a non-respiratory chain protein mitochondrial control. Chemiluminescent detection of the bands was achieved using the Amersham ECL Prime Kit (GE Healthcare) for signal development, following manufacturer's instructions and the membrane was viewed using the ChemiDoc<sup>TM</sup>MP Imaging System (Bio-Rad).</p><p>Subcellular fractions were prepared as described previously <xref ref-type="bibr" rid="pgen.1004424-Bruni1">[40]</xref>. The same amount of protein (40 &#181;g) from whole cell lysate, post-mitochondrial supernatant and mitochondrial subfractions was loaded onto a 12% SDS-PAGE gel, transferred to a PVDF membrane and analysed by immunoblotting using primary antibodies to TOMM20 (Santa Cruz), AIF (NEB), GDH (custom made against mature recombinant protein), NDUFA9 (Mitosciences), eIF4E (Cell Signaling). Chemiluminescent detection of the bands was achieved as described before.</p><p>The translation of proteins encoded by the mtDNA in patient fibroblasts was assessed by labelling with <sup>35</sup>S-methionine/<sup>35</sup>S-cysteine (Perkin Elmer) as described previously <xref ref-type="bibr" rid="pgen.1004424-McKenzie1">[41]</xref>. Cytosolic translation was inhibited by co-incubation of the radioisotopes with 100 &#181;g/ml emetine dihydrochloride. Total cell protein was extracted from both control and patient fibroblasts and 50 &#181;g loaded onto a 15%&#8211;20% gradient gel for SDS-PAGE. Assessment of protein loading was achieved by Coomassie blue staining, and the gel was visualised by exposure to a blank PhosphorImager screen that was imaged using a Typhoon system (GE Healthcare).</p><p>The <italic>in vitro</italic> modification activity of both wild-type and mutant TRIT1 was determined as previously described <xref ref-type="bibr" rid="pgen.1004424-Lamichhane2">[11]</xref> using recombinantly-expressed enzyme from <italic>E. coli</italic> recovered using a Histidine tag <xref ref-type="bibr" rid="pgen.1004424-Makrides1">[42]</xref>. Synthetic RNA minihelixes representing the target anticodon stem/loop (ASL) sequences of various tRNAs were used as templates for modification with <sup>14</sup>C-labelled dimethylallyl pyrophosphate (DMAPP). In this assay, the isopentenyl group of DMAPP is transferred to A37 in substrate tRNAs by the IPTase TRIT1. The following RNA oligos were designed with an additional G-C base pair added to each ASL to stabilize the stem, and purchased from Integrated DNA Technologies (IDT): rGrUrGrCrArGrGrCrUrUrCrArArArCrCrUrGrUrArC (cy-tRNA<sup>Sec(UCA)</sup>), rGrArUrGrGrArCrUrUrGrArArArUrCrCrArUrC (cy-tRNA<sup>Ser(UGA)</sup>), rGrGrGrUrUrGrGrCrUrUrGrArArArCrCrArGrCrUrC (mt-tRNA<sup>Ser(UCN)</sup>), rGrGrGrUrUrGrGrCrUrUrGrArArGrCrCrArGrCrUrC (mt-tRNA<sup>Ser(UCN)</sup>-A7480G), rGrUrUrGrArArUrUrGrCrArArArUrUrCrGrArC (mt-tRNA<sup>Cys</sup>) and rGrUrArArArArCrUrUrArArArArCrUrUrUrArC (mt-tRNA<sup>Leu(UUR)</sup>).</p><p>Decreased hybridization efficiency of complimentary DNA oligos due to the incorporation of the isopentenyl group in synthetic RNA minihelixes was confirmed by running an <italic>in vitro</italic> modification reaction with unlabeled DMAPP and recombinant His-TRIT1. We used the same protocol for the <italic>in vitro</italic> modification reaction as described previously <xref ref-type="bibr" rid="pgen.1004424-Lamichhane2">[11]</xref>, replacing <sup>14</sup>C-labeled DMAPP with unlabeled DMAPP (100 nmol). For each sample, two reactions were performed. In mock-treated samples, all the components of the <italic>in vitro</italic> reaction were added except His-TRIT1. After the reaction, the RNA sample was purified by a phenol-chloroform extraction and loaded on a 15% TBE-Urea gel. The RNA was transferred to a GeneScreen Plus Hybridization Transfer Membrane (Perkin Elmer, catalog # NEF986001PK) and hybridized with <sup>32</sup>P-labeled anticodon loop (ACL) oligos as described in the legend of <xref ref-type="fig" rid="pgen-1004424-g006">Figure 6D</xref>. The sequences of DNA oligos used for this experiment are the same as used elsewhere in this paper.</p><p>Total RNA isolation from skeletal muscle and human primary cell lines was performed using TRIzol according to manufacturer's protocol. The impact of the TRIT1 mutation on <italic>in vivo</italic> levels of the i<sup>6</sup>A37 modification in both cytosolic and mitochondrial tRNAs was assessed by the <bold>P</bold>ositive <bold>H</bold>ybridisation in the <bold>A</bold>bsence of i<bold><sup>6</sup></bold>A (PHA6) assay, which is an adaptation of high-resolution northern analysis <xref ref-type="bibr" rid="pgen.1004424-Lamichhane2">[11]</xref>. The following anticodon loop (ACL) and body (BP) probes were used (all written as 5&#8242;-3&#8242;): mt-tRNA<sup>Cys</sup> ACL, <named-content content-type="gene">TCTTCGAATTTGCAATTCAATATG</named-content> and BP, <named-content content-type="gene">AGCCCCGGCAGGTTTGAAGCT</named-content>, cy-tRNA<sup>Ser(UGA)</sup> ACL, <named-content content-type="gene">CCCATTGGATTT CAAGTCCAACGC</named-content>, and BP, <named-content content-type="gene">GCAGGATTCGAACCTGCGCGGG</named-content>, wild-type mt-tRNA<sup>Ser(UCN)</sup> ACL, <named-content content-type="gene">CAAAGCTGGTTTCAAGCCAACCCC</named-content> (used for analysis of the patient carrying the <italic>TRIT1</italic> mutation), mutant mt-tRNA<sup>Ser(UCN)</sup> ACL, <named-content content-type="gene">CAAAGCTGG<underline>C</underline>TTCAAGCCAACCCC</named-content> (both the wild-type and mutation-bearing probes were used together as a &#8216;double ACL probe&#8217;; the complement of the mutated base is underlined) and mt-tRNA<sup>Ser(UCN)</sup> BP, <named-content content-type="gene">AAGGAAGGAATCGAACCCCCC</named-content>, mt-tRNA<sup>Leu(UUR)</sup> BP, <named-content content-type="gene">GTTAAGAAGAGG AATTGAACCTC</named-content> and U5 probe, <named-content content-type="gene">TCCTCTCCACGGAAATCTTTA</named-content>.</p><p>The wild-type <italic>TRIT1</italic> or mutant <italic>TRIT1</italic> was cloned into the pREP82X plasmid under the <italic>nmt1</italic>
<bold><italic><sup>+</sup></italic></bold> promoter before transformation into a yNB5 (tit1-&#916;) strain of <italic>S. pombe</italic>. The experiments related to tRNA-mediated anti-suppression and growth deficiency in glycerol were performed as described previously <xref ref-type="bibr" rid="pgen.1004424-Lamichhane1">[10]</xref>, <xref ref-type="bibr" rid="pgen.1004424-Lamichhane2">[11]</xref>.</p><p>The <italic>S. cerevisiae</italic> yeast strain used was BY4742 <italic>mod5-&#916;</italic> (<italic>MAT&#945; his3&#916;1 leu2&#916;0 lys2&#916;0 ura3&#916;0 mod5::KanMX4</italic>) from the Euroscarf collection. The <italic>MOD5</italic> gene was PCR-amplified with Kod HiFi Polymerase using primers MOD5CFw (gactagaaaatcgatgtgtcagg) and MOD5CSalIRv (ccgccGTCGACgcttgtcat cctccctttcc), digested with <italic>Kpn</italic>I and <italic>Sal</italic>I and cloned in the centromeric vector, pFL38 <xref ref-type="bibr" rid="pgen.1004424-Bonneaud1">[43]</xref>, thus obtaining the plasmid pFL38<italic>MOD5</italic>. The <italic>mod5<sup>K294R</sup></italic> humanized and <italic>mod5<sup>K294Q</sup></italic> mutant alleles were obtained by site-directed mutagenesis as described previously <xref ref-type="bibr" rid="pgen.1004424-Ho1">[44]</xref>, on a <italic>MOD5</italic> gene fragment obtained through amplification with the upstream forward primer MOD5MUTFw (ggagcccctgcagcttcatg) and the reverse mutagenic primer MOD5hK294RFw (cgagaacacgtcaatacgcaCGCaggcaggtaaaatggatcaag) or MOD5K294QFw (cgagaacacgtc aatacgcaCaaaggcaggtaaaatggatcaag). The amplified fragments were digested with <italic>Bam</italic>HI and <italic>Sal</italic>I and subcloned in <italic>Bam</italic>HI-<italic>Sal</italic>I-digested pFL38<italic>MOD5</italic>.</p><p>Plasmids were introduced in a BY4742 <italic>mod5&#916;</italic> strain according to <xref ref-type="bibr" rid="pgen.1004424-Gietz1">[45]</xref>. Growth assays were performed at 28&#176;C in SD medium (0.69% YNB (Formedium, Norfolk, UK), without amino acids for which the strain is auxotroph) supplemented with 2% glucose (w/v) or 2% ethanol (v/v). Images of the colonies in the spots were acquired at 40X magnification with a Zenith inverted microscope through an Optikam 3 Digital Camera (Optika).</p><p>Oxygen consumption rate was measured at 30&#176;C from suspensions of yeast cells cultured for 24 hours at 28&#176;C in SD medium supplemented with glucose at a non-repressing concentration of 0.6% using a Clark-type oxygen electrode (Oxygraph System Hansatech Instruments England) with 1 ml of air-saturated respiration buffer (0.1 M phthalate &#8211; KOH, pH 5.0), 0.5% glucose. The reaction was started by addition of 20 mg of wet-weight cells.</p>