<p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Our proposed method uses the Fixed Effect Model (FEM) and the Random Effect Model (REM) iteratively. The FEM is employed to test <italic>m</italic> genetic markers, one at a time. Pseudo QTNs are included as covariates to control false positives. Specifically, the FEM can be written as follows:
<disp-formula id="pgen.1005767.e006"><alternatives><graphic id="pgen.1005767.e006g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005767.e006.jpg" /><ns1:math id="M6"><ns1:mrow><ns1:msub><ns1:mtext>y</ns1:mtext><ns1:mtext>i</ns1:mtext></ns1:msub><ns1:mo>=</ns1:mo><ns1:msub><ns1:mtext>M</ns1:mtext><ns1:mrow><ns1:mtext>i</ns1:mtext><ns1:mn>1</ns1:mn></ns1:mrow></ns1:msub><ns1:msub><ns1:mtext>b</ns1:mtext><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>+</ns1:mo><ns1:msub><ns1:mrow><ns1:mtext>M</ns1:mtext></ns1:mrow><ns1:mrow><ns1:mtext>i</ns1:mtext><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msub><ns1:msub><ns1:mtext>b</ns1:mtext><ns1:mn>2</ns1:mn></ns1:msub><ns1:mo>+</ns1:mo><ns1:mo>&#8230;</ns1:mo><ns1:msub><ns1:mrow><ns1:mo>+</ns1:mo><ns1:mtext>M</ns1:mtext></ns1:mrow><ns1:mrow><ns1:mtext>it</ns1:mtext></ns1:mrow></ns1:msub><ns1:msub><ns1:mtext>b</ns1:mtext><ns1:mtext>t</ns1:mtext></ns1:msub><ns1:msub><ns1:mrow><ns1:mo>+</ns1:mo><ns1:mspace width="0.15em" /><ns1:mtext>S</ns1:mtext></ns1:mrow><ns1:mrow><ns1:mtext>ij</ns1:mtext></ns1:mrow></ns1:msub><ns1:msub><ns1:mtext>d</ns1:mtext><ns1:mtext>j</ns1:mtext></ns1:msub><ns1:msub><ns1:mrow><ns1:mo>+</ns1:mo><ns1:mtext>e</ns1:mtext></ns1:mrow><ns1:mtext>i</ns1:mtext></ns1:msub></ns1:mrow></ns1:math></alternatives><label>(3)</label></disp-formula>
where y<sub>i</sub> is the observation on the i<sup>th</sup> individual; M<sub>i1</sub>, M<sub>i2</sub>,&#8230;, M<sub>it</sub> are the genotypes of t pseudo QTNs, initiated as an empty set; b<sub>1</sub>, b<sub>2</sub>, &#8230;, b<sub>j</sub> are the corresponding effects of the pseudo QTNs; S<sub>ij</sub> is the genotype of the i<sup>th</sup> individual and j<sup>th</sup> genetic marker; d<sub>j</sub> is the corresponding effect of the j<sup>th</sup> genetic marker; e<sub>i</sub> is the residual having a distribution with zero mean and variance of <inline-formula id="pgen.1005767.e007"><alternatives><graphic id="pgen.1005767.e007g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005767.e007.jpg" /><ns1:math id="M7"><ns1:mrow><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mi>e</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:math></alternatives></inline-formula>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The REM is employed to optimize the selection of pseudo QTNs from markers based on their testing statistics (i.e., P values) and positions by using the SUPER algorithm[<xref ref-type="bibr" rid="pgen.1005767.ref024">24</xref>]. Mathematically, the REM can be written as follow:
<disp-formula id="pgen.1005767.e008"><alternatives><graphic id="pgen.1005767.e008g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005767.e008.jpg" /><ns1:math id="M8"><ns1:mrow><ns1:msub><ns1:mtext>y</ns1:mtext><ns1:mtext>i</ns1:mtext></ns1:msub><ns1:mo>=</ns1:mo><ns1:msub><ns1:mtext>u</ns1:mtext><ns1:mtext>i</ns1:mtext></ns1:msub><ns1:mo>+</ns1:mo><ns1:msub><ns1:mtext>e</ns1:mtext><ns1:mtext>i</ns1:mtext></ns1:msub></ns1:mrow></ns1:math></alternatives><label>(4)</label></disp-formula>
where y<sub>i</sub> and e<sub>i</sub> stay the same as in Eq (<xref ref-type="disp-formula" rid="pgen.1005767.e006">3</xref>) and u<sub>i</sub> is total genetic effect of the i<sup>th</sup> individual. The expectations of the individuals&#8217; total genetic effects are zeros. The variance and covariance matrix of the individuals&#8217; total genetic effects is <inline-formula id="pgen.1005767.e009"><alternatives><graphic id="pgen.1005767.e009g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005767.e009.jpg" /><ns1:math id="M9"><ns1:mrow><ns1:mi>G</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>2</ns1:mn><ns1:mi>K</ns1:mi><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mi>a</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:math></alternatives></inline-formula>, where <inline-formula id="pgen.1005767.e010"><alternatives><graphic id="pgen.1005767.e010g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005767.e010.jpg" /><ns1:math id="M10"><ns1:mrow><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mi>a</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:math></alternatives></inline-formula> is an unknown genetic variance and <italic>K</italic> is the kinship matrix defined by pseudo QTNs.</p><p>The iterative usage of the FEM (1) and the REM (2) is specifically described by the following steps:</p><p>Step 1: Set known candidate QTNs as pseudo QTNs, otherwise leave pseudo QTN set empty.</p><p>Step 2: Perform association tests by using the FEM with pseudo QTNs as covariates. Genetic markers are fitted as fixed effects, one at a time.</p><p>Step 3: Substitution: When the testing marker is one of the pseudo QTNs, the testing marker is not solvable. As each pseudo QTN is examined for every marker, we use the most significant P value for the corresponding marker. We call this process substitution.</p><p>Step 4: If no marker passes a threshold (e.g., 1%) after multiple test corrections, take the result from Step 3 as final; otherwise, go to Step 5.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Step 5: Whole genome is divided into bins, and the SNP with the most significant P value in each bin represents the bin. Bin size (e.g. 500; 5,000; and 50,000 kilo base pairs) and number of bins (e.g. from 10 to 100 with step of 10) are optimized in the REM. The combination of size and number with the minimum REML value will be used to select pseudo QTNs. In each iteration, the number of pseudo QTNs is bounded by a function of sample size (e.g., <inline-formula id="pgen.1005767.e011"><alternatives><graphic id="pgen.1005767.e011g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005767.e011.jpg" /><ns1:math id="M11"><ns1:mrow><ns1:msqrt><ns1:mrow><ns1:mtext>n</ns1:mtext><ns1:mo>/</ns1:mo><ns1:msub><ns1:mtext>log</ns1:mtext><ns1:mn>10</ns1:mn></ns1:msub><ns1:mo stretchy="false">(</ns1:mo><ns1:mi>n</ns1:mi><ns1:mo stretchy="false">)</ns1:mo></ns1:mrow></ns1:msqrt></ns1:mrow></ns1:math></alternatives></inline-formula>) to reflect the ability to detect QTNs in the sample.</p><p>Step 6: Elimination linear dependence, or near linear dependence among pseudo QTNs. When two pseudo QTNs have Pearson correlation coefficients above a threshold (e.g., 0.7), remove the less significant pseudo QTN.</p><p>Step 7: If no new pseudo QTNs are added, or if the specified maximum number of iterations is reached, stop. Otherwise, go back to Step 2.</p><p>We used previously published datasets from multiple species that included <italic>Arabidopsis thaliana</italic>, human, maize, mouse, and pig.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used two datasets of <italic>Arabidopsis thaliana</italic>. The first dataset includes 199 samples, with 216,130 SNPs and 107 phenotypes[<xref ref-type="bibr" rid="pgen.1005767.ref021">21</xref>]. When phenotypes had less than 100 records, a minor allele frequency (MAF) of 0.05 was set to filter the SNPs. The second dataset includes 1,179 samples with 214,545 SNPs. One sample was removed because one-half of the genotypic data is missing (URL: <ext-link ext-link-type="uri" ns0:href="http://archive.gramene.org/db/diversity/diversity_view">http://archive.gramene.org/db/diversity/diversity_view</ext-link>). The kinship matrix and principal components were calculated by GAPIT[<xref ref-type="bibr" rid="pgen.1005767.ref042">42</xref>] using 10% of SNPs sampled randomly.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">One human dataset, &#8220;WTCCC1 controls dataset&#8221;, ID # EGAD00000000002, was obtained from EMBL-EBI (The European Molecular Biology Laboratory&#8211;The European Bioinformatics Institute)[<xref ref-type="bibr" rid="pgen.1005767.ref038">38</xref>]. Respecting the privacy of individual level data, the data is only available under the permission of MalariGen Data Access Committee. This dataset contains 1,500 samples. All samples were genotyped by the Affymetrix_500k SNP Chip and 495,473 markers were used in our simulation study (URL: <ext-link ext-link-type="uri" ns0:href="https://www.ebi.ac.uk/ega/datasets/EGAD00000000002">https://www.ebi.ac.uk/ega/datasets/EGAD00000000002</ext-link>). The kinship matrix and principal components were calculated by GAPIT using 10% of SNPs sampled randomly.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The other human dataset, &#8220;East Asian lung cancer dataset&#8221;, ID # phs000716.v1.p1, was obtained from dbGaP[<xref ref-type="bibr" rid="pgen.1005767.ref027">27</xref>]. Respecting the privacy and intentions of research participants, the data is only available under the permission of NIH (National Institute of Health) and Intramural NCI (National Cancer Institute). The authors applied and got the data through dbGaP Authorized Access. A total of 8,807 samples were used that contain 4,962 lung cancer cases and 3,845 controls. All samples were genotyped by the Illumina Human610_Quadv1_B and Human660W-Quad_v1_A platforms and each sample has 629,968 SNPs (URL: <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000716.v1.p1">http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000716.v1.p1</ext-link>). The kinship matrix and principal components were calculated using 10% randomly sampled or total SNPs by GAPIT and PLINK, respectively.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The maize genotype dataset includes 2,279 inbred lines, each with 681,258 SNPs. The phenotype is flowering time measured as days to silk[<xref ref-type="bibr" rid="pgen.1005767.ref029">29</xref>] (URL: <ext-link ext-link-type="uri" ns0:href="http://www.panzea.org/!#genotypes/cctl">http://www.panzea.org/!#genotypes/cctl</ext-link>). The kinship matrix and principal components were calculated by GAPIT using 10% of SNPs sampled randomly.</p><p>The mouse genotype dataset has 1,940 samples (1000 males and 940 females from a heterogeneous stock mice population owned by the Welcome Trust Centre for Human Genetics) with 12,226 SNPs. The phenotype is weight growth intercept[<xref ref-type="bibr" rid="pgen.1005767.ref030">30</xref>]. The kinship matrix and principal components were calculated by GAPIT using all SNPs.</p><p>The pig genotype dataset has 820 samples (412 Large White and 408 crosses from Large White and Landrace) with 64,212 SNPs. The phenotype is last rib back-fat thickness[<xref ref-type="bibr" rid="pgen.1005767.ref031">31</xref>]. All SNPs were used to build the kinship matrix and principal components.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We used real genotype datasets from human and <italic>Arabidopsis thaliana</italic> to simulate genetic effects and generate phenotypes by adding residual effects. The QTNs underlying these phenotypes were randomly sampled from the real genotypes. The QTN effects followed a geometric distribution with an additive effect of parameter <italic>a</italic>. The effect of the i<sup>th</sup> QTN was <italic>a</italic><sup><italic>i</italic></sup>. The parameter <italic>a</italic> was set to 0.9, 0.95, and 1 as described in previous studies[<xref ref-type="bibr" rid="pgen.1005767.ref009">9</xref>,<xref ref-type="bibr" rid="pgen.1005767.ref017">17</xref>]. Phenotype was simulated as: y = additive effect + residual effect. Additive effect was calculated as: additive effect = QTN matrix * QTN effects. The residual effect, following a Gaussian distribution with mean of 0 and variance of <inline-formula id="pgen.1005767.e012"><alternatives><graphic id="pgen.1005767.e012g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005767.e012.jpg" /><ns1:math id="M12"><ns1:mrow><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mi>e</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:math></alternatives></inline-formula>, was calculated as: <inline-formula id="pgen.1005767.e013"><alternatives><graphic id="pgen.1005767.e013g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005767.e013.jpg" /><ns1:math id="M13"><ns1:mrow><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mi>e</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>=</ns1:mo><ns1:mo stretchy="false">(</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:msup><ns1:mi>h</ns1:mi><ns1:mn>2</ns1:mn></ns1:msup><ns1:mo stretchy="false">)</ns1:mo><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mi>a</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>/</ns1:mo><ns1:msup><ns1:mi>h</ns1:mi><ns1:mn>2</ns1:mn></ns1:msup></ns1:mrow></ns1:math></alternatives></inline-formula>, where <inline-formula id="pgen.1005767.e014"><alternatives><graphic id="pgen.1005767.e014g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1005767.e014.jpg" /><ns1:math id="M14"><ns1:mrow><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mi>a</ns1:mi><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:math></alternatives></inline-formula> is the variance of additive effect and <italic>h</italic><sup>2</sup> is heritability. Simulations were performed using a variety of QTN numbers and heritability values, and with QTNs included and excluded from the genotypic data for association tests. For each combination of factors, simulations were repeated either 100 or 1,000 times, specified for each experiment.</p><p>Statistical power, Type I error, and FDR were examined simultaneously in association tests on simulated phenotypes with known QTNs, using the method described by Segura et.al[<xref ref-type="bibr" rid="pgen.1005767.ref025">25</xref>] and two methods from our previous studies&#8212;SUPER[<xref ref-type="bibr" rid="pgen.1005767.ref024">24</xref>] and Enriched CMLM[<xref ref-type="bibr" rid="pgen.1005767.ref022">22</xref>]. A QTN was considered identified if a positive marker was within a prescribed interval distance (e.g. 50 kb). Power was defined as the proportion of QTNs identified at a threshold of Type I error or FDR. Markers were used to derive the null distribution of negative control if no QTN was within the interval. The null distribution of Type I error was derived from the non-QTN markers. FDR was defined as the proportion of the non-QTN markers among the positive markers.</p><p>The flowering time candidate genes from the database reported by Atwell et. al, (2010, Nature) were used to evaluate the associated SNPs on 23 flowering time traits in <italic>Arabidopsis thaliana</italic>. The whole <italic>Arabidopsis thaliana</italic> genome was divided into gene regions and non-gene regions. The genes and their extensions, 10,000 base pairs on either side, were considered gene regions with a total length of 4,552,815 base pairs (3.9% of whole genome). The remaining areas were considered non-gene regions with total length of 114,616,742 base pairs (96.1% of whole genome). The average hit per base pair was defined by number of associated SNPs divided by total length. The ratio of average hit on gene regions to the average hit on non-gene regions was used as the enrichment coefficient. The random hits were expected to have an enrichment coefficient of 1.</p>