<p>C57BL/6J-Tg(TRAMP)824Ng/J (TRAMP) and NOD/ShiLtJ mice were obtained from The Jackson Laboratory (Bar Harbor, ME). F1 mice were generated by crossing TRAMP females, which were hemizygous for PB-TAg transgene (Tg), to NOD/ShiLtJ males. F2 mice were generated by crossing Tg+ F1 females with Tg- F1 males. All animals were handled, housed and used in the experiments humanely in accordance with the NHGRI Animal Care and Use Committee guidelines. All work was performed under Animal Study Protocol G-09-2. Mouse tail genomic DNA was extracted from F1 progeny with the HotSHOT method <xref ref-type="bibr" rid="pgen.1004809-Truett1">[42]</xref> for genotyping analysis. PCR screening was performed as described <xref ref-type="bibr" rid="pgen.1004809-Hurwitz1">[14]</xref> to identify the hemizygous PB-TAg transgene positive F1 and F2 mice.</p><p>As described previously in <xref ref-type="bibr" rid="pgen.1004809-Patel1">[15]</xref>, (TRAMP &#215; NOD/ShiLtJ) F2 male mice were sacrificed by pentobarbital overdose at 30 weeks of age or humane endpoint, whichever was achieved first. Humane experimental endpoints for this study were rapid weight loss, hunched posture, labored breathing, trauma, impaired mobility, dysuria, or difficulty in obtaining food or water. Prostate tumor, seminal vesicles, lungs, liver, and lymph nodes were harvested from (TRAMP &#215; NOD/ShiLtJ) F2 males. Prostate tumor and seminal vesicles were weighed to quantify tumor burden. Visible, enlarged lymph nodes in para-aortic region were weighed to quantify metastatic lymph node burden. Lungs were collected to determine isolated tumor cell infiltrates in lung parenchyma and microscopic metastatic lesions. Other organs displaying macroscopic metastatic lesions through gross observation were also collected for histology. These collected tissues were fixed in buffered formalin (10% w/v phosphate buffered formaldehyde, Fisher Scientific) overnight and then transferred to 70% ethanol. Fixed tissues were embedded in paraffin, sectioned to a thickness of 4 &#181;m and stained with hematoxylin and eosin (H&amp;E). Histology slides were scanned with Scanscope Digital microscope (Aperio, Vista, CA).</p><p>Genomic DNA was extracted from F2 tail biopsies using a Gentra Puregene DNA Extraction Kit (Qiagen, Valencia, CA), per the manufacturers protocol. Five microliters of DNA at 75 ng/&#181;l was used for SNP genotyping using the 1536 plex assay kit and GoldenGate Assay Mouse Medium Density Linkage Array following the manufacturers protocol (Illumina, San Diego, CA). The intensity data for each SNP for 228 samples were normalized and the genotypes assigned using Illumina GenomeStudio Genotyping Analysis Module version 1.9.4. SNPs with a GC score &lt;0.7 and non-informative (homozygous) SNPs were excluded from further analysis. SNP Hardy&#8211;Weinberg equilibrium (HWE) P-values were estimated with PLINK. SNPs were omitted if the HWE <italic>P</italic>&lt;0.001.</p><p>As described previously in <xref ref-type="bibr" rid="pgen.1004809-Giubellino1">[43]</xref>, total RNA extractions from (TRAMP &#215; NOD/ShiLtJ) F2 tumor samples were carried out using TRIzol Reagent (Life Technologies, Inc.) according to the standard protocol. RNA quality and quantity was ensured using the Bioanalyzer (Agilent, Inc., Santa Clara, CA) and NanoDrop (Thermo Scientific, Inc., Waltham, MA), respectively. Per RNA labeling, 200 ng of total RNA was used in conjunction with the Affymetrix (Santa Clara, CA) recommended protocol for the GeneChip 2.0 ST chips. Hybridization cocktails containing the fragmented and labeled cDNAs were hybridized to Affymetrix Mouse Genome 2.0 ST GeneChip. Chips were washed and stained by the Affymetrix Fluidics Station using the standard format and protocols as described by Affymetrix. Probe arrays were stained with streptavidin phycoerythrin solution (Molecular Probes, Carlsbad, CA) and enhanced by using an antibody solution containing 0.5 mg/mL of biotinylated anti-streptavidin (Vector Laboratories, Burlingame, CA). An Affymetrix Gene Chip Scanner 3000 was used to scan the probe arrays. Gene expression intensities were calculated using Affymetrix AGCC software. Partek Genomic Suite was used to RMA normalize (Robust Multichip Analysis), summarize, log2 transform the data, run ANOVA analysis and unsupervised hierarchical clustering. To account for genes expressed below the threshold of detection, average levels of gene expression across all samples were calculated and genes expressed in the lower 10<sup>th</sup> percentile excluded. This encompassed the average experiment-wide background intensity of 3.04&#177;0.12.</p><p>Microarray data are available through Gene Expression Omnibus (accession no. GSE58829).</p><p>QTL analysis was performed using J/qtl <xref ref-type="bibr" rid="pgen.1004809-Smith1">[20]</xref>. Mapping of QTLs was performed for all traits using a single-QTL analysis, using a binary model for binary traits (e.g., distant metastasis free survival [DMFS]) and a non-parametric model for all other traits. Significance levels were computed using permutation testing <xref ref-type="bibr" rid="pgen.1004809-Churchill1">[44]</xref>, using 10,000 permutations. Age of death was used as an additive covariate for tumor-related traits (primary tumor burden, seminal vesicle tumor burden). Age and primary tumor burden were used as additive covariates for all metastasis-related traits. Confidence intervals for QTLs identified were estimated using 2-LOD support intervals, which is on the chromosome where the LOD score did not fall below 2.0 of its maximum <xref ref-type="bibr" rid="pgen.1004809-Lander1">[45]</xref>. Only those QTLs reaching a genome-wide &#945;&lt;0.05 were considered to be of interest.</p><p>eQTL analysis was performed using Matrix-eQTL in R <xref ref-type="bibr" rid="pgen.1004809-Shabalin1">[24]</xref>. A linear model was used to test for association between gene expression and SNPs, with age and primary tumor burden used as covariates. A SNP that mapped &#8804;1 Mb upstream or downstream of the transcription start site was used to define proximal eQTLs. Correction for multiple testing was performed using the Benjamini-Hochberg FDR method. An FDR &lt;0.05 was used as the threshold for significant eQTLs.</p><p>Pearson correlation coefficients and associated <italic>P</italic>-values were calculated for all traits other than those with a binary distribution by correlating the log2 transformed expression intensities of all probes mapped to a given QTL with the relevant QTL trait using MedCalc (Ostend, Belgium). For the latter, student's t-tests were performed to test the significance of transcript-trait correlations. Correction for multiple testing was performed using the Benjamini-Hochberg FDR method using the QVALUE module in R <xref ref-type="bibr" rid="pgen.1004809-Storey1">[46]</xref>. An FDR &lt;0.05 was used as the threshold for significant correlations.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">QTL candidate gene expression levels were analyzed in the cBioPortal for Cancer Genomics database (<ext-link ext-link-type="uri" ns0:href="http://www.cbioportal.org">http://www.cbioportal.org</ext-link>; <xref ref-type="bibr" rid="pgen.1004809-Gao1">[27]</xref>). Two human prostate cancer datasets possessed sufficient gene expression and clinical data to facilitate assessment of candidate genes: a) TCGA (Provisional) &#8211; the Cancer Genome Atlas provisional data (<ext-link ext-link-type="uri" ns0:href="https://tcga-data.nci.nih.gov/tcga/tcgaCancerDetails.jsp?diseaseType=PRAD&amp;diseaseName=Prostate%20adenocarcinoma">https://tcga-data.nci.nih.gov/tcga/tcgaCancerDetails.jsp?diseaseType=PRAD&amp;diseaseName=Prostate%20adenocarcinoma</ext-link>); and b) GSE21032 - Prostate Oncogenome Project, Taylor et al. <xref ref-type="bibr" rid="pgen.1004809-Taylor1">[28]</xref>. The gene expression levels in TCGA (Provisional) dataset available on the cBioPortal website are provided by The Cancer Genome Atlas. Here, level 3 expression data were generated from RNA-seq data by first generating &#8216;Reads per Kilobase per Million mapped reads&#8217; (RPKM; <xref ref-type="bibr" rid="pgen.1004809-Mortazavi1">[47]</xref>) counts. This is followed by utilization of MapSplice <xref ref-type="bibr" rid="pgen.1004809-Wang1">[48]</xref> to align sequence reads and &#8216;RNA-Seq by Expectation Maximization&#8217; (RSEM) values <xref ref-type="bibr" rid="pgen.1004809-Li1">[49]</xref> to perform gene quantitation. cBioPortal reports higher or lower levels of gene expression by a z-score of &#8805;2 or &#8804;&#8722;2, respectively, where the z-score is the standard deviation of static levels of transcript expression in a given case compared to the mean transcript expression in diploid tumors. Diploid tumors were used for the purposes of normalization since candidate gene ploidy could presumably impact average expression levels of candidate genes.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In the GSE21032 cohort, gene up- or down-regulation in a given case is again provided by cBioPortal as a z-score of &#8805;2 or &#8804;&#8722;2, respectively. However, here a z-score of 2 was defined as an array probe-set intensity that is two standard deviations greater than the mean of the probe set intensity in the matched normal tissue, with the opposite being true for down-regulated genes. Therefore, to make candidate gene expression levels more comparable to those reported for TCGA (Provisional) cohort, raw gene expression data for GSE21032 were downloaded from cBioPortal (<ext-link ext-link-type="uri" ns0:href="http://cbio.mskcc.org/cancergenomics/prostate/data/MSKCC_PCa_mRNA_data.zip">http://cbio.mskcc.org/cancergenomics/prostate/data/MSKCC_PCa_mRNA_data.zip</ext-link>). The expression levels of the 29 QTL candidate genes were subsequently extracted of all primary tumors with mRNA data (n&#8202;=&#8202;131), average expression levels and standard deviations calculated, and z-scores for candidate gene expression in individual tumors calculated using the following formula: ([gene expression in individual tumor &#8211; average population gene expression]/population expression standard deviation).</p><p>Logistic regression and Kaplan-Meyer survival analyses were performed using MedCalc (Ostend, Belgium). Logistic regression was performed using the stepwise method, with individual dichotomized clinical variables (<xref ref-type="table" rid="pgen-1004809-t003">Table 3</xref>; <xref ref-type="supplementary-material" rid="pgen.1004809.s004">Figure S4</xref>) as dependent variables and z-scores for all 29 candidate genes as independent variables. Kaplan-Meyer survival curves were constructed by comparing the time to recurrence in cases from either cohort with higher levels of tumor candidate gene expression <italic>versus</italic> all other cases.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The clinical characteristics of the CGEMS GWAS cohort have been described extensively elsewhere (dbGaP Study Accession: phs000207.v1.p1; <xref ref-type="bibr" rid="pgen.1004809-Yeager1">[31]</xref>). All SNPs analyzed were either located within a given QTL candidate gene or no more than 100,000 bp upstream or downstream. SNP HWE <italic>P</italic>-values were estimated with PLINK. SNPs were omitted if the HWE <italic>P</italic>&lt;0.001. Association analysis between aggressive prostate cancer phenotype and SNP or haplotype was performed using a generalized linear model (glm). Age and PC1, PC2 and PC3 were included as covariates in the glm. Analysis of aggressive vs. non-aggressive disease phenotypes were performed as per the comparisons described in <xref ref-type="table" rid="pgen-1004809-t003">Table 3</xref>. Correction for multiple testing was performed using permutation testing (n&#8202;=&#8202;10,000 permutations) using the glm on NIH biowulf super cluster computer system (<ext-link ext-link-type="uri" ns0:href="http://biowulf.nih.gov">http://biowulf.nih.gov</ext-link>). Specifically, permutation testing was performed for each phenotype against one SNP under rearrangements of the labels on all individuals with 10,000 times. Permutation tests were performed only in instances where the uncorrected <italic>P</italic>&lt;0.01. Manhattan plots were constructed in R. For haplotype analysis, genome-wide LD blocks were estimated by using the Solid Spine algorithm of Haploview software with the default parameters, and fastPHASE was performed to generate haplotypes for each individual based on the LD blocks on NIH biowulf super cluster computer system (<ext-link ext-link-type="uri" ns0:href="http://biowulf.nih.gov">http://biowulf.nih.gov</ext-link>). FDR <italic>P</italic>-values were calculated by the MULTITEST package of R. All analyses were performed by using R.</p>