<p>Deoxynucleotide analog-based mutagenic PCR was used to construct a C1 library as described [<xref ref-type="bibr" rid="pgen.1006253.ref029">29</xref>], in three segments. For lib-A, primers C1-Xho-Fwd and C-NcoI-Rev, lib-B, C1-NcoI-Fwd and C1-BSTAP1-Rev, and lib-C, C1-BstAP1-Fwd and C1-Xma1-Rev were used (all DNA oligos used for this and other applications including northern blotting are listed in <xref ref-type="table" rid="pgen.1006253.t004">Table 4</xref>). The products were cloned into XhoI/XmaI of pRep4X and transformed into UltraMax DH5&#945;-FT cells (30 plates). ~150,000 bacterial Transformants were scraped from the plates and plasmid DNA library prepared.</p><p>Suppressor-tRNA genes and <italic>S</italic>. <italic>pombe</italic> strains are listed in <xref ref-type="supplementary-material" rid="pgen.1006253.s003">S1 Table</xref>. The Rpc2-C-His-FLAG tagged strain, yKR22 was constructed using pFA6a-kanMX6 vector. SpRpc2 was amplified with Rpc2-AscI-F and Rpc2-CHF-BglII-R (contains C-terminus His-FLAG sequence) primers and cloned into the pFA6a-kanMX6 in AscI and BglII site generating pFA6a-kanMX6-Rpc2. One kb sequences upstream and downstream of Rpc2 were also amplified from chromosomal DNA and cloned into pFA6a-KanMX6-Rpc2 in Xma1/AscI and PmeI/SacI sites respectively. For integration, the cloned fragment with kanamycin maker was amplified by PCR and integrated into yKR1 strain as described [<xref ref-type="bibr" rid="pgen.1006253.ref081">81</xref>]. The <italic>maf1-&#916;</italic> strain yKR101 was created as was the yNB1 <italic>maf1-&#916;</italic> strain described previously [<xref ref-type="bibr" rid="pgen.1006253.ref082">82</xref>] but carrying the tRNA reporter, pMSer-3T, with a 3T test terminator followed by cRT region and the 8T failsafe terminator: <italic>h</italic>&#8722; <italic>leu1-32</italic>::[<italic>tRNAmSer3T-leu1+</italic>] <italic>ura4-D18 ade6-704 maf1&#916;</italic>::<italic>KanMX6</italic>.</p><p>SpRpc1-Wt and selected mutants were cloned by PCR amplification from plasmids with forward primer (C1-XhoI-Flag-Fwd) and reverse primer (C1-XmaI&#8211;Rev). The forward primer contains 1X-Flag sequence at the N-terminus after the start codon. The PCR amplified fragments were gel purified, digested with XhoI and XmaI, and cloned into pRep4X at the corresponding sites.</p><p>Libraries were transformed into <italic>S</italic>. <italic>pombe</italic> yKR1, plated on EMM lacking uracil with 10 mg/L adenine. Colonies showing suppression (white) were isolated, the plasmid was recovered and retransformed for conformation, and sequenced.</p><p>For expression analysis, overnight cultures of Wt and mutants in EMM media lacking uracil were transferred to 50 ml and further grown to A600 of 0.8&#8211;1.0, washed with water, residual liquid aspirated and 200 &#956;l of lysis buffer (50 mM HEPES (pH 7.3), 250 mM KCl, 5 mM EDTA, 2.5 mM DTT) was added. Glass beads (0.5 mm) were added and cells disrupted in mini-beadbeater-8 (Biospec) at 4 x 45 seconds keeping in ice for 1 min between intervals. Samples were then spun in tabletop centrifuge for 20 min at top speed, supernatant transferred to a new tube and protein concentration determined. 25 &#956;g total protein was resolved in 4&#8211;12% Nu-PAGE gel (Life Technologies), transferred to nitrocellulose membrane. The blot was blocked for one hour with 5% non-fat milk in PBS. Monoclonal anti-flag (Sigma, #F1804) was used at 1:2000 in blocking buffer with 0.05% NP-40. Anti-Tubulin (Sigma, #T5168) was used at 1:5000. After one hour the blot was washed four times with PBS-NP-40. Appropriate secondary antibodies (LI-COR) of different fluorescence were used at 1:10,000 in 5% milk solution in PBS-NP-40 for one hour. The blot was washed with PBS-NP-40 four times and scanned using LI-COR Odyssey Clx system and the images processed with ImageStudioLite Software.</p><p>yKR1 transformed cells were grown in 2 liters of EMM lacking uracil until mid-log phase (OD600 = 2.0), harvested immediately, washed with ice-cold water, and as much liquid as possible aspirated leaving ~6 gm wet cell pellet which was frozen in dry ice. For S100 extract, the frozen cells were mixed with dry ice and shattered in a coffee grinder for 5 min, transferred to a beaker and 1 ml/gm cell pellet of extraction buffer (50 mM HEPES, 250 mM KCl, 5 mM EGTA, 1 mM EDTA) was added. The lysate was spun at 32K for 90 min. The supernatant, ~4 ml at 15&#8211;20 mg/ml was collected, aliquotted and immediately frozen on dry ice, leaving about 1/3 as the pellet.</p><p>Promoter dependent transcription was carried out by preincubating 3 &#956;l of S100, 100 ng plasmid DNA containing Mser-tRNA genes with different test terminators (2T-8T) in transcription buffer (100 mM KCl, 20 mM HEPES pH 7.5, 7 mM MgCl<sub>2</sub> and 3 mM DTT) at 25&#176;C. After 30 min the NTPs were added at 0.5 mM ATP, CTP, UTP, and 25 &#956;M GTP plus 5 &#956;Ci [&#945;-32P]GTP (10 mCi/ml; 3000 Ci/mmol) in final volume of 25 &#956;l. After another 30 min the reactions were quenched with 200 &#956;l stop buffer (10 mM EDTA, 1% SDS, 25 &#956;g/ml Proteinase K) and incubated for 10 min. Products were recovered by phenol-chloroform extraction, ethanol precipitation, resolved by 6% polyacrylamide-urea gel and visualized by a Typhoon 9400 phosphorImager (GE Healthcare). Image flies were analyzed using Multigauge Software (Fuji).</p><p>Briefly, 300 &#956;l S100 extract (~6 mg protein) was mixed with 25 &#956;l of EZview Red Anti-Flag-M2 Affinity Gel (Sigma, #F2426) and incubated at 4&#176;C for 1 hr. The beads were spun, supernatant removed and beads washed 3X with 500 &#956;l buffer (20 mM HEPES, 200 mM KCl, 1 mM EDTA). The beads with immobilized RNAP III was suspended in 100 &#956;l transcription buffer. To this, tailed template and water was added, mixed by pipetting and distributed equally to multiple tubes for time course reactions, each carried out in 40 &#956;l volume with 50 ng tailed template and other ingredients as indicated for promoter dependent transcription. The RNAP III stall-chase experiment was as above except using a tailed template lacking T in the first 22 nucleotides. The initial reactions were carried out in the absence of UTP for 2 min after which the reactions were made 50 &#956;M UTP. Products were processed at the various time points thereafter as above.</p><p>For the excess template experiment, RNAP III was pull-down with EZview Red Anti-Flag-M2 Affinity Gel as above and incubated with excess of template (2 &#956;g) in transcription buffer without NTPs for 10 min. After 10 min incubation at room temperature, the complex was washed with 500 &#956;l of transcription buffer three times. After washing the complex was suspended into 300 &#956;l transcription buffer and distributed into eight tubes for time course reaction. Time course was carried out with NTPs (50 &#956;M for ATP, CTP, UTP and 25 &#956;M GTP plus 5 &#956;Ci [&#945;-<sup><bold>32</bold></sup>P]GTP and buffer as above.</p><p>RNA stabilities were visualized by inhibiting cellular transcription in <italic>S</italic>. <italic>pombe</italic> with 1,10-Phenanthroline (Sigma Aldrich, Cat # 131377-5G) [<xref ref-type="bibr" rid="pgen.1006253.ref053">53</xref>, <xref ref-type="bibr" rid="pgen.1006253.ref054">54</xref>]. Overnight cultures of C1-Wt and C1-E850K in yAS68 grown in EMM media lacking uracil were transferred to 500 ml fresh media and further grown to A600 of 0.5 OD. 1,10-Phenanthroline in DMSO was added at a final concentration of 350 &#956;g/ml. At the times indicated thereafter 25 ml was withdrawn from the cultures, washed with water, spun down to aspirate residual water, and immediately frozen in dry ice. Total RNA was separated on 6% polyacrylamide-urea gel, and processed for northern blotting as described.</p><p>Total RNA was isolated using hot phenol. RNA from control strains and mutants were separated on 6% or 10% polyacrylamide-urea gel, and transformed to a nylon membrane using the IBlot apparatus (Invitrogen, Grand Island, NY). Membranes were UV cross linked, vacuum dried at 80&#176;C and hybridized with RNA specific DNA oligos as indicated and listed in <xref ref-type="table" rid="pgen.1006253.t004">Table 4</xref>. After washing, blots were exposed to phosphorimager screen, scanned by a Typhoon 9400 phosphorImager and quantified using Multigauge software (Fuji).</p>