<p>A plasmid containing SRA sequence (BC067895.1) was purchased from Open Biosystems. The SRA coding sequence was subcloned into pLITMUS28i (New England Biolabs) for <italic>in vitro</italic> transcription (see below). The following plasmids were used for in <italic>vitro</italic> transcription/translation: pSG5-MYC encoding p68 and p72 (gift from Prof. Frances V. Fuller-Pace, University of Dundee, UK); pcDNA3.1-NANOG (Addgene).</p><p>See <xref ref-type="supplementary-material" rid="pgen.1005615.s019">S1 Table</xref> for the list of antibodies used in this study.</p><p>Human pluripotent stem cell line NTERA2 was grown in DMEM supplemented with 10% FBS (Gibco) at 37&#176;C under a humidified atmosphere of 5% CO<sub>2</sub> in air. At confluent, cells were passaged every three days using 0.25% trypsin (Gibco). For establishment of NTERA2 stable knockdown cell lines, the plasmids pMLP-shRNA targeting SRA, p68 or scramble control (transOMIC) were linearized by <italic>Nde</italic>I and transfected into 1x10<sup>6</sup> cells using nucleofector (Amaxa) according to manufacturer&#8217;s protocol. Cells were immediately grown in DMEM-F12 plus 10% FBS. On day 3, stable cell lines were selected using puromycin at 3 &#956;g/ml final concentration.</p><p>RNA pull down experiments were performed as previously described [<xref ref-type="bibr" rid="pgen.1005615.ref041">41</xref>]. First, DNA fragments encoding full length, 5&#8217; and 3&#8217; domains of lncRNA SRA were cloned into pLITMUS28i (New England Biolabs), and the DNA sequence was confirmed by sequencing. To generate antisense or sense SRA transcripts, the plasmid containing full length SRA was linearized by <italic>Stu</italic>I or <italic>Bgl</italic>I, respectively. Biotinylated SRA was <italic>in vitro</italic> transcribed using HiScribe T7 <italic>In Vitro</italic> transcription kit (New England Biolabs) in the presence of biotin-14-CTP (Invitrogen) according to the instruction manuals. Transcribed RNA products were DNase-treated (Ambion), purified by ethanol precipitation and verified by northern blotting.</p><p>For RNA pull downs using nuclear extract, 3 &#956;g of <italic>in vitro</italic> transcribed RNA was prepared in RNA structure buffer (Tris-Cl pH 7.5, 0.1 M KCl, 10 mM MgCl<sub>2</sub>) and incubated at 78&#176;C for 3 min. The RNA was then gradually cooled down to 37&#176;C. Five hundred micrograms of NTERA2 nuclear extract, prepared using NE-PER Nuclear Protein Extraction Kit (Pierce), was mixed with the RNA in immunoprecipitation buffer (PBS plus 0.1% Triton X-100, 1 mM DTT, protease inhibitor cocktail, PMSF, 80 U RNase inhibitor) in a total volume of 500 &#956;L. The reaction was incubated for 4 hr at 4&#176;C with rotation. MyOne Streptavidin C1 beads were prepared according to manufacturer&#8217;s recommendation, and used at 50 &#956;L per sample. The RNA-beads complex was further incubated overnight. Beads were washed five times with immunoprecipitation buffer and boiled with 50 &#956;L of SDS loading buffer. Twenty microliters was loaded onto Novex precast gel (Invitrogen). For RNA pull down using recombinant proteins, 0.3 &#956;g of RNA was used per pull down reaction with 3 &#956;g of protein complex or 1 &#956;g of individual protein. TrxG and PRC2 complexes were purchased from BPS Bioscience and Cayman Chemical. NANOG, OCT4 and SOX2 were purchased from Fitzgerald Industries International. The RNA helicases p68 and p72 were <italic>in vitro</italic> translated using the TNT Coupled Reticulocyte Lysate System (Promega). A plasmid encoding luminescence protein was used as negative control (Promega). Recombinant NANOG was also produced by <italic>in vitro</italic> translation using a Wheat Germ System (Promega). All <italic>in vitro</italic> translated proteins were verified by western blotting. Ten microliters of translated protein product was used per RNA pull down reaction.</p><p>For <italic>in vitro</italic> co-immunoprecipitation in the presence of antisense or sense SRA, the RNAs were transcribed without Biotin-14-CTP. Three micrograms of TrxG and PRC2 complexes or 10 &#956;L of <italic>in vitro</italic> translated p68 and NANOG were used for co-immunoprecipitation in 200 &#956;L of immunoprecipitation buffer. Antibodies for immunoprecipitation were used at 3 &#956;g including mouse anti-RBBP5 (MABE220, Upstate), mouse anti-EZH2 (MA5-15101, Thermo Scientific) and rabbit anti-DDX5 (A300-523A, Bethyl Laboratories).</p><p>For co-immunoprecipitation using nuclear extract, 500 &#956;g of NTERA2 nuclear extract was mixed with 3 &#956;g of relevant antibodies in a total of 500 &#956;L of immunoprecipitation buffer. The reaction was incubated for 4 hr at 4&#176;C with rotation. Protein A and protein G conjugated magnetic beads were prepared according to manufacturer&#8217;s recommendation (Invitrogen), and used at 50 &#956;L per sample. The complex was then further incubated overnight. Beads were washed five times with immunoprecipitation buffer and boiled with 50 &#956;L of SDS loading buffer. Twenty microliters was loaded onto Novex precast gel (Invitrogen).</p><p>RNA was extracted using TRIzol reagent (Invitrogen) and DNase-treated (DNA-free kit, Ambion). Complementary DNA synthesis was performed with 1 &#956;g RNA using a Maxima First Strand cDNA Synthesis Kit (Thermo Scientific). qPCR was carried on by using Power SYBR Green PCR Master Mix (Applied Biosystems) in a total volume of 20 &#956;l each well with 7900HT real-time PCR system (Applied Biosystems). Gene expression was normalized by expression level of <italic>ACTB</italic>. Primer sequences are available upon request.</p><p>Twenty million cells were fixed with 1% formaldehyde in PBS for 10 min at room temperature. The fixation was quenched by adding glycine at 125 mM final concentration and incubated further for 5 min. Cells were washed and collected by centrifugation at 1500 rpm for 5 min. Nuclear extract was prepared by using NE-PER Nuclear Protein Extraction Kit (Pierce). Three micrograms of antibody was added to 500 &#956;g of the nuclear extract in immunoprecipitation buffer (PBS, 1 mM DTT, protease inhibitor cocktail, PMSF, 80 U RNase inhibitor) in a total volume of 500 &#956;L. The complex was incubated at 4&#176;C for 4 hr. Protein A and protein G conjugated magnetic beads were used at 50 &#956;L per sample. The complex was then further incubated overnight. Beads were washed five times and resuspended in 100 &#956;L proteinase K buffer (10 mM Tris-Cl pH 7.5, 100 mM NaCl, 1 mM EDTA, 0.5% SDS) with 5 &#956;L proteinase K (New England Biolabs). Samples were incubated at 50&#176;C for 45 min with shaking, and boiled at 95&#176;C for 10 min. Samples were mixed with 500 &#956;L Qiazol by vigorous vortexing, and were incubated at room temperature for 10 min. RNA extraction was then performed using miRNeasy mini kit (Qiagen). qPCR was employed to detect RNA binding.</p><p>ChIP was performed according to the manufacturer&#8217;s instruction (Active Motif). Briefly, 2 x 10<sup>7</sup> cells were fixed with 1% formaldehyde in PBS for 10 min at room temperature. The fixation was then quenched by adding glycine. Cells were washed and collected by centrifugation at 1500 rpm for 5 min. Nuclei were sonicated twice using Bioruptor (Diagenode) at maximum power, 30 sec ON and 30 sec OFF for 7.5 min to obtain chromatin fragments ranging from 200&#8211;1000 bp. Fifty micrograms of sheared chromatin was used per IP with 3 &#956;g antibody. Retrieved DNA fragments were purified by QIAquick PCR Purification Kit (Qiagen) or ethanol precipitation. Primer sequences for ChIP are listed in <xref ref-type="supplementary-material" rid="pgen.1005615.s020">S2 Table</xref>.</p><p>ChIRP analysis was performed according to published protocols with minor modifications based on ChIRP and Capture Hybridization Analysis of RNA Targets (CHART) techniques [<xref ref-type="bibr" rid="pgen.1005615.ref019">19</xref>, <xref ref-type="bibr" rid="pgen.1005615.ref042">42</xref>, <xref ref-type="bibr" rid="pgen.1005615.ref043">43</xref>]. Briefly, 3x10<sup>7</sup> cells were fixed with 1% glutaraldehyde for 10 min at room temperature with shaking. The fixation was stopped by adding glycine. Crosslinked cells were washed with PBS, and resuspended in 1 ml swelling buffer (25 mM HEPES pH 7.3, 10 mM KCl, 0.1% NP-40, 1 mM DTT, PMSF). Samples were incubated at 4&#176;C for 30 min with shaking, and were collected by centrifugation. The pellet was resuspended with 350 &#956;L of ChIRP lysis buffer, and was sonicated using Bioruptor (Diagenode) at maximum power, 30 sec ON and 30 sec OFF for 7.5 min of 6 cycles to obtain chromatin fragments ranging from 100&#8211;1000 bp. Sheared chromatin was then collected by centrifugation. Two hundred micrograms of sheared chromatin sample was pre-cleared for 1 hour using 100 &#956;L of Ultralink-streptavidin beads (Pierce) at room temperature with shaking. The sample was then centrifuged, and supernatant was collected. The pre-cleared chromatin was used per hybridization reaction with 10 &#956;L of 100 &#956;M pooled 3&#8217; Biotin TEG oligonucleotide probes (Integrated DNA Technologies). SRA probes were designed to cover SRA transcript at nucleotide position 124 to 1473 (accession number NR_045587.1) (See <xref ref-type="supplementary-material" rid="pgen.1005615.s021">S3 Table</xref> for the probe sequences). LacZ probes were employed as negative control [<xref ref-type="bibr" rid="pgen.1005615.ref019">19</xref>]. The sample and the probes were hybridized at 37&#176;C for 4 hours with shaking. Once the hybridization was completed, 100 &#956;L of C-1 magnetic beads (Invitrogen) was mixed with the sample to pull down the biotinylated probes. DNA was eluted in the presence of 12.5 mM D-Biotin (Invitrogen). DNA was ethanol precipitated and subjected to library preparation.</p><p>Library preparation was performed using TruSeq ChIP Sample Preparation Kit (Illumina) or MicroPlex Library Preparation Kit (Diagenode) according to manufacturer&#8217;s instruction. Three biological triplicates were used for ChIRP-seq and ChIP-seq. Briefly, 5&#8211;10 ng of DNA starting material, which was quantified by Qubit (Invitrogen), was used for each biological sample. The DNA was end-repaired, 3&#8217; adenylated, and ligated with adapters. Then the ligated DNA was size-selected to obtain DNA fragments at 250&#8211;300 bp by agarose gel electrophoresis. The purified DNA was amplified to enrich the library. The final PCR product was purified by Agencourt AMPure XP beads (Beckman Coulter) and was submitted to the NIDDK Genomic Core Facility for high-throughput sequencing using Illumina HiSeq2500. The sequencing was performed with the run type of single-end, 50 bp read. Data were aligned against the human genome version human_hg19, and were exported into BAM file format.</p><p>Aligned reads of SRA ChIRP-seq and p68 ChIP-seq were filtered with SAMTools program to remove duplicates and tags with a map quality score less than 20 [<xref ref-type="bibr" rid="pgen.1005615.ref044">44</xref>]. MACS version 1.4.2 was used for peak calling with a threshold of <italic>p-</italic>value less than 10<sup>&#8722;5</sup> for p68 and 10<sup>&#8722;4</sup> for SRA [<xref ref-type="bibr" rid="pgen.1005615.ref045">45</xref>]. The resulting BED files from each of biological triplicate samples were intersected using the Bioconductor package ChIPpeakAnno [<xref ref-type="bibr" rid="pgen.1005615.ref046">46</xref>]. For SRA ChIRP-seq analysis, the SRA probe dataset was further subtracted against a LacZ probe dataset using bedtools [<xref ref-type="bibr" rid="pgen.1005615.ref047">47</xref>]. The dataset was further screened for possible homologies with the RNA probes used in the analysis. No significant number of sequences was found with zero, one or two mismatches. Intersection between any of SRA, p68, NANOG and CTCF was performed with maximum distance of 500 bp between peaks. For the binding comparison between the above factors and H3K4me3 and H3K27me3, a distance less than 2 kb was allowed. BED files were visualized and exported using IGV [<xref ref-type="bibr" rid="pgen.1005615.ref048">48</xref>]. H3K4me3 and H3K27me3 ChIP-seq data were taken from the ENCODE project of NTERA2 cells. NANOG and CTCF ChIP-seq data were taken from the ENCODE project of human embryonic stem cells H1. Gene classification analysis was performed using GREAT [<xref ref-type="bibr" rid="pgen.1005615.ref049">49</xref>]. The Fishers Exact test to measure peak enrichment was taken from the Fisher&#8217;s exact function from the R package for statistical computing [<xref ref-type="bibr" rid="pgen.1005615.ref050">50</xref>]. SRA ChIRP- and p68 ChIP-Sequencing data were submitted to GEO Datasets under accession number GSE58641.</p><p>Human fibroblast cell line WI-38 at 1x10<sup>6</sup> cells were transfected with a single plasmid encoding the four reprogramming factors OCT4, SOX2, c-MYC and KLF4 [<xref ref-type="bibr" rid="pgen.1005615.ref051">51</xref>] using Nucleofector with scrambled siRNA or ON-TARGETplus siRNA targeting <italic>SRA</italic> (Dharmacon). Transfected fibroblasts were plated in six-well plates under Essential 8 medium (Invitrogen). The siRNA knockdown was also performed consecutively at first and second week post-transfection using Lipofectamine RNAiMAX Reagent (Invitrogen). Plates were collected on day 30 and were analyzed for expression of surface markers of human pluripotent stem cells. Alkaline phosphatase staining was performed using Alkaline Phosphatase Detection Kit (Millipore) per manufacturer&#8217;s instruction.</p><p>Cells were fixed with 4% PFA and were incubated with a monoclonal antibody against SSEA3 (gift from Prof. Peter W. Andrews, University of Sheffield, UK). A goat anti-mouse secondary antibody (IgG+IgM) conjugated with FITC (Cayman Chemical) was then used for visualization under a fluorescence microscope (EVOS).</p><p>Single cells were collected by trypsinization, and resuspended with 10% FBS in PBS buffer. Primary antibodies SSEA3 and A2B5 (1 in 10 dilution) were added into cell suspensions containing 1x10<sup>5</sup> cells in 100 &#956;L volume. P3X was used as a negative control antibody. The reaction was incubated on-ice for 30 min, washed with PBS and resuspended with 10% FBS in PBS. One &#956;L of FITC-conjugated Goat anti-Mouse antibody (Cayman Chemical) was added to the reaction. The reaction was incubated on-ice for another 30 min. Flow cytometric analysis was performed using Cytomic FC500 (Beckman Coulter).</p>