<p>
<italic>S. cerevisiae</italic> strains were grown at 30&#176;C in YP (yeast extract 1%, peptone 2%) with 2% glucose or glycerol as carbon sources or synthetic complete media. <italic>Escherichia coli</italic> strains were grown in standard LB media at 37&#176;C. When appropriate, gentamicin and ampicillin were added to YPD (0.2 mg/ml) and LB (0.1 mg/ml), respectively.</p><p>The plasmid, pFL39, which contains <italic>MIP1</italic> on a centromeric plasmid was previously described <xref ref-type="bibr" rid="pgen.1004748-Foury2">[31]</xref>. Site-directed mutagenesis of plasmid-encoded <italic>MIP1</italic> was performed using the QuikChange Site-Directed Mutagenesis Kit (Invitrogen) as described previously. The construction of plasmids containing <italic>ACT1, COX1</italic>, and <italic>COX2</italic> fragments, used for mtDNA quantitation was previously described.</p><p>All <italic>S. cerevisiae</italic> strains were derived from E134 (MAT&#945; <italic>ade5-1 his7-2 lys2-A14 leu2-3,112 ura3-52 trp1-289</italic>) <xref ref-type="bibr" rid="pgen.1004748-Tran1">[54]</xref> and its MATa isogenic strain, YH747. Heteroallelic <italic>mip</italic>1 strains were made by transforming pFL39-<italic>MIP1</italic> or a mutant derivative into E134 by selecting for TRP. Strains that measure deletions between direct repeats were created by transforming <italic>TRP1</italic>-containing plasmids PFL39 <xref ref-type="bibr" rid="pgen.1004748-Stumpf2">[12]</xref> containing wild type <italic>MIP1</italic> or <italic>mip1</italic> encoding an exonuclease-deficient mutant variant (<italic>mip1-exo</italic> [Het]; JSY114) into <italic>trp1</italic>::G418 NPY75 (JSY77) <xref ref-type="bibr" rid="pgen.1004748-Stumpf2">[12]</xref>. Chromosomal <italic>mip1</italic> mutations were created in haploid E134 with wild type <italic>MIP1</italic> using a PCR-based <italic>delitto perfetto</italic> method as previously described. All strains were checked phenotypically for the absence of the CORE casette used in <italic>delitto perfetto</italic> (ie, screening for Ura<sup>&#8722;</sup> and gentamicin sensitive cells) and were sequenced to confirm the presence of the mutation and the absence of undesired nearby mutations. The <italic>mip1</italic> haploid mutant strains were mated with the isogenic E134 with mating type a.</p><p>Resistance to erythromycin is conferred by one of several missense mutations in the 21S rRNA gene in mitochondrial DNA <xref ref-type="bibr" rid="pgen.1004748-Kalifa1">[46]</xref>, <xref ref-type="bibr" rid="pgen.1004748-Vanderstraeten1">[47]</xref>, <xref ref-type="bibr" rid="pgen.1004748-Sor1">[55]</xref>. Yeast strains were replica plated onto YPD plates with or without 1&#8211;5 mM MMS or 1&#8211;5 &#181;M CdCl<sub>2</sub> and grown at 30C for 2 days. In most cases 3 mM MMS was the highest exposure that allowed growth and was used for the experiments. After plating on CdCl<sub>2</sub>, increased chromosomal mutagenesis was confirmed qualitatively by the presence of colonies on media lacking lysine. From these plates, 20&#8211;40 independent colonies from each strain were used to inoculate into 4 ml of synthetic media without tryptophan (heteroallelic strains) or YPD (heterozygous strains), and these cultures were incubated to saturation (for 2 days) at 30&#176;C. The cells were plated on YPEG (1.7% ethanol, 2% glycerol) with 4 g/L erythromycin. A small aliquot of 5&#8211;10 cultures were used to titer the number of rho<sup>+</sup> cells by plating 10<sup>&#8722;5</sup> dilutions on YPG. Erythromycin resistant colonies were counted after 6 days of incubation at 30&#176;C. The mutant frequency was the median number of erythromycin colonies per 10<sup>8</sup> rho<sup>+</sup> cells plated. To determine the spectrum of erythromycin-resistant mutations, DNA was extracted from one mutant per culture and used for as a template in a PCR reaction to amplify two regions, approximately 200 nucleotides flanking nucleotides 1950 (using <named-content content-type="gene">5&#8242;-GAGGTCCCGCATGAATGACG</named-content> and <named-content content-type="gene">5&#8242;-CGATCTATCTAATTACAGTAAAGC</named-content>) and 3993 (using <named-content content-type="gene">5&#8242;-CTATGTTTGCCACCTCGATGTC</named-content> and <named-content content-type="gene">5&#8242;-CAATAGATACACCATGGGTTGATTC</named-content>). The resulting amplified DNA was the template for the sequence reactions.</p><p>To measure deletions between direct repeats, all strains were replica plated onto YPD with or without MMS exposure as described above. Independent cultures were grown from at least 20 colonies at 30&#176;C in YP (yeast extract 1%, peptone 2%) with 2% glucose and adenine for 2 days. Appropriate dilutions of samples from the saturated cultures were plated on synthetic complete media lacking arginine to determine total number of cells with mtDNA. The cultures were plated onto YP with 2% glycerol and deletion mutants were counted after 4 days. The mutant frequency was determined as the median number of mutant colonies per 10<sup>8</sup> Arg<sup>+</sup> cells. In all mutagenesis experiments, 95% confidence levels were determined using the method of the median.</p><p>Petite frequencies are the frequency of rho<sup>&#8722;</sup> cells (petites) in the total population. Rho<sup>&#8722;</sup>cells are devoid of mitochondrial functions but are not necessarily devoid of mtDNA (rho<sup>0</sup>). To determine petite frequency in heteroallelic strains, at least 12 fresh transformants per strain were diluted in water and between 200&#8211;1000 colonies were plated onto YPD. For monoallelic strains, rho<sup>+</sup> cells derived from several tetrad dissections were single colony purified on YPD and then assayed for petite frequency. Cells were incubated at 30&#176;C for 2 days. Rho<sup>+</sup> cells were identified either by the accumulation of red pigment as a result of mutations in the <italic>ade</italic> biosynthetic pathway or by the inability to grow on YPG. Using either or both method, at least 300 colonies per plate were counted, and petite colonies were identified. Frequencies were determined for each plate, and the median number of the frequencies was calculated. 95% confidence levels were determined by using the method of the median (52). For monoallelic strains determined to be 100% petite, no rho<sup>+</sup> cells could be isolated from cells derived from 10 different haploid spores.</p><p>Mitochondrial DNA copy number was quantified relative to nuclear DNA copy number using real time PCR. Primers and probes designed to specifically amplify within the mitochondrial-encoded <italic>COX1</italic> gene and the nuclear-encoded <italic>ACT1</italic> gene. Real time PCR reactions using Taqman Universal PCR Master Mix (Applied Biosystems) were performed at 40 cycles of 95 for 30 sec and 50 degrees for 30 sec. Known concentrations of plasmid molecules containing <italic>COX1</italic>and <italic>ACT1</italic> were quantified as a positive control <xref ref-type="bibr" rid="pgen.1004748-Stumpf1">[9]</xref> and real time PCR was performed on 7 different dilutions to determine a logarithmic equation of a curve (R<sup>2</sup> values&gt;0.98) that represents numbers of molecules as a function of the critical threshold of every reaction. Every reaction was done in triplicate, and three replicates were tested for each experimental condition. Data represent the average ratio (&#177; SEM) of the number of <italic>COX1</italic> molecules to the number of <italic>ACT1</italic> molecules.</p>