<p>ES cell gene trap lines with a &#946;-geo reporter construct inserted in intron 1 of the <italic>Ppib</italic> gene were produced by BayGenomics (UCSF, CA) and obtained from the Mutant Mouse Regional Resource Center (Davis, CA) <xref ref-type="bibr" rid="pgen.1004465-Stryke1">[63]</xref>. The gene trap construct contains intron 1 and a portion of exon 2 to include splice acceptor sequence from the mouse <italic>En2</italic> gene, followed by a &#946;-galactosidase/neomycin (&#946;-geo) reporter-selection cassette and SV40 polyadenylation signal. Cells were expanded for isolation of mRNA and quantitation of <italic>Ppib</italic> expression. A heterozygous clone from cell line RST139 was shown to have half-normal <italic>Ppib</italic> expression by real-time RT-PCR, and was injected into C57BL/6 blastocysts. Founders were generated by mating chimeric males with 129/P2/OlaHsd females to retain the 129 background of the ES cell line. A second line was generated for experiments by backcrossing F1 mice into C57BL/6 for 5 generations.</p><p>Genomic DNA for genotyping was isolated using the Red Extract-n-Amp tissue PCR kit (Sigma) and amplified by hemi-nested PCR using a sense primer located in <italic>Ppib</italic> exon 1 (<named-content content-type="gene">5&#8242;-TGCCCGGAC CCTCCGTGGCCAACGATAAGA-3&#8242;</named-content>), an antisense primer corresponding to intron 1 of <italic>En2</italic> (<named-content content-type="gene">5&#8242;-GGCATCTCCCCTTCAGTCTTCCTGTCCAGG-3&#8242;</named-content>), and an antisense primer downstream of the inserted construct internal to <italic>Ppib</italic> intron 1 (<named-content content-type="gene">5&#8242;-GGGGGGCTGGGGGAGTCTGGGTTATTCTCT-3&#8242;</named-content>). Complete absence of <italic>Ppib</italic> transcripts was verified by real-time RT-PCR of mRNA isolated from femoral tissue and skin of F2 <italic>Ppib</italic> homozygous knock-out mice. Animal care and experiments were performed in accordance with a protocol approved by the NICHD ACUC committee.</p><p>For skeletal staining, skin and viscera were removed from dead P1 newborn pups. Pups were fixed in 95% ethanol for 7 days and stained with 0.3% Alcian Blue 8GS and 0.1% Alizarin Red S <xref ref-type="bibr" rid="pgen.1004465-McLeod1">[64]</xref>. Excess stain was removed with 1% KOH and increasing concentrations of glycerol. For growth curves, mice were fed regular rodent chow and weighed weekly from 3 to 24 weeks of age. Skeletal characteristics of F6 wild-type, heterozygous and homozygous <italic>Ppib</italic>-null littermates were analyzed at age 8 weeks. Radiographs were performed by Faxitron (30 kV for 1 min). Areal bone mineral density (aBMD) scans of mouse femurs were acquired using a GE Lunar PIXImus2 (GE Healthcare) and internal calibration standards. To determine femoral length, femora were dissected and cleaned of soft tissue, leaving the epiphyses intact. Femurs were measured from the proximal head to the distal end of the medial and lateral condyles with a digital caliper.</p><p>Left femora of 8 week-old mice were analyzed by &#956;QCT for both structural and mineral parameters using a SkyScan1174 compact micro-CT scanner (MicroPhotonics) operating at 50 kV with an X-ray source current of 800 &#181;A, according to manufacturer directions. BMD was calibrated with hydroxyapatite phantoms. Morphometric analyses of trabecular and cortical regions was performed using CTAn software (v.1.13) and 3D images were generated using CTvol software (v.2.2). The trabecular region of interest (ROI) was located just proximal to the distal femoral growth plate and extended 10% total femoral length. The diaphyseal cortical ROI centered on the femur midpoint, spanning 15% total femoral length.</p><p>Femora were loaded to failure in four-point bending as previously described <xref ref-type="bibr" rid="pgen.1004465-Sinder1">[65]</xref>. Femora were loaded to failure in four-point bending at 0.5 mm/s in the anterior-posterior direction with the posterior surface under tension using a servohydraulic testing device (MTS 858; MiniBionix; MTS Systems Corporation). Force was recorded by a 50lb load cell (Sensotec) and vertical displacement by an external linear variable differential transducer (Lucas Schavitts) at 2000 Hz. Load-displacement curves were used to calculate stiffness, yield load, yield displacement, ultimate load, failure displacement, post-yield displacement, and energy to failure.</p><p>Primary fibroblast (FB) and calvarial osteoblast (OB) cultures were derived from 3-day old pups by standard procedures <xref ref-type="bibr" rid="pgen.1004465-Bakker1">[66]</xref>. Cells from digestions 3&#8211;5 were plated at a density of 5,000 cells/cm<sup>2</sup> and cultured in &#945;MEM with 10% FBS, 2 mM glutamine, 1% pen-strep and 8% CO<sub>2</sub>. FB cultures were derived from dermal tissue dissected from the abdomens of newborn pups. FB were allowed to grow out from dermal samples for 2 weeks, released by trypsin digestion and cultured in DMEM containing 10% fetal bovine serum, 2 mM glutamine, 1% pen-strep and 5% CO<sub>2</sub>.</p><p>Total RNA was extracted from cell cultures or tissues dissected from F6 mice using TriReagent (Molecular Research Center) according to the manufacturer's protocol. Total RNA was treated with DNA-free (Life Technologies), then reverse-transcribed using a High Capacity cDNA Archive Kit (Life Technologies). Real-time RT-PCR was performed using Taqman Assays on Demand (Life Technologies, <italic>Ppib</italic>, Mm00478295_m1; <italic>Plod1</italic>, Mm01255760_m1; <italic>Plod3</italic>, Mm00478798_m1; <italic>Glt25d1</italic>, Mm00600638_m1; <italic>Glt25d2</italic>, Mm01290012_m1; <italic>Gapdh</italic>, Mm99999915_g1). Relative expression of genes of interest was measured in triplicate, normalized to <italic>Gapdh</italic> transcripts, and quantified relative to wild-type calvarial OB.</p><p>Two independent fibroblast (FB) and osteoblast (OB) cultures for each genotype were lysed in RIPA buffer (150 mM NaCl, 1% NP-40, 0.5% Na-deoxycholate, 0.1% SDS, 50 mM Tris, pH 7.4) supplemented with protease inhibitor cocktail (Sigma). Protein concentration was determined using the BCA Protein Assay Kit (Thermo Scientific). Samples (15 &#181;g protein) were subjected to SDS-PAGE on 10% acrylamide gels under denaturing conditions and electroblotted onto nitrocellulose membranes. The membranes were blocked overnight in 5% non-fat milk in TBST. After washing in TBST, membranes were incubated overnight at 4&#176;C in TBST containing 2.5% non-fat milk and primary antibody (diluted 1&#8758;1000). After washing in 1&#215; TBST, membranes were incubated with the corresponding IRDye infrared secondary antibody (diluted 1&#8758;10,000) (LI-COR Biosciences). Proteins were visualized using an Odyssey Infrared Imaging System (LI-COR Biosciences). Quantitation of proteins was performed using the Odyssey 3.0.30 software with normalization to Actin levels. The analysis was repeated to ensure reproducibility. The primary antibodies used include anti-P3H1 (Abnova), rabbit anti-actin (Santa Cruz Biotechnology), rabbit anti-CyPB (Abnova), rabbit anti-PLOD1 (Santa Cruz Biotechnology), rabbit anti-PLOD3 (Proteintech), and goat anti-GLT25D1 (Santa Cruz Biotechnology). Anti-mouse CRTAP antibody was a generous gift from Dr Brendan Lee, Baylor College of Medicine.</p><p>Steady-state collagen analysis was performed as previously described <xref ref-type="bibr" rid="pgen.1004465-Bonadio1">[67]</xref>. Collagens were prepared by pepsin digestion (50 &#181;g/ml) of procollagen samples, separated on 6% SDS-urea- polyacrylamide gels and visualized by autoradiography.</p><p>For analysis of collagen modification in cell culture, confluent FB and OB were stimulated for collagen synthesis in DMEM or &#945;MEM containing 0.1% FBS and 100 &#181;g/ml ascorbate for three days, with daily collection. Collected medium was buffered with 100 mM Tris-HCl, pH 7.4, and cooled to 4&#176;C. Protease inhibitors were added to the following final concentrations: 25 mM EDTA, 0.02% NaN<sub>3</sub>, 1 mM phenylmethylsulfonylfluoride, 5 mM benzamidine, and 10 mM N-ethylmaleimide. Procollagens were precipitated from media with ammonium sulfate overnight at 4&#176;C. Procollagen was collected by centrifugation, resuspended in 0.5 M acetic acid and digested with 0.1 mg/ml pepsin at 4&#176;C overnight. Selective salt precipitation of collagen with 0.9 M NaCl in 0.5 M acetic acid was performed twice. Purified collagen samples were resuspended in 0.5 M acetic acid, dialyzed against 5 mM acetic acid overnight and lyophilized before further analyses.</p><p>Differential Scanning Calorimetry (DSC) scans were performed as previously described <xref ref-type="bibr" rid="pgen.1004465-Makareeva1">[68]</xref>. Thermograms were recorded in 0.2 M sodium phosphate, 0.5 M glycerol, pH 7.4, from 10 to 50&#176;C at 0.125 and 1&#176;C/min heating rates in a Nano III DSC instrument (Calorimetry Sciences Corporation).</p><p>Analysis of tissue-derived collagens was performed using skin, femora and humeri from 2 month old mice (n&#8202;=&#8202;5). After marrow was flushed with cold PBS, bone samples were pulverized in liquid N<sub>2</sub>, demineralized with EDTA at 4&#176;C for 2 weeks, and lyophilized. Skin dissected from the backs of mice was minced and lyophilized after removal of hair, fat and muscle. Two mg of the dried samples were then reduced with standardized NaB<sup>3</sup>H<sub>4</sub>, hydrolyzed with 6N HCl and subjected to amino acid and cross-link analyses as previously described <xref ref-type="bibr" rid="pgen.1004465-Yamauchi2">[69]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004465-Eyre6">[71]</xref>. The extent of Lys hydroxylation in collagen was calculated as hydroxylysine (Hyl)/hydroxyproline (Hyp)&#215;300 (i.e. &#8764;300 residues of Hyp/collagen). The cross-link precursor aldehydes and reducible cross-links were measured as their reduced forms and all cross-links were quantified as moles/mole of collagen.</p><p>Collagen 3-hydroxylation was analyzed by in-gel tryptic digestion of SDS-PAGE-purified type I collagen alpha chains. Electrospray mass spectrometry was performed on the tryptic peptides using an LCQ Deca XP ion-trap mass spectrometer equipped with in-line liquid chromatography (Thermo Finnigan) using a C8 capillary column (300 &#181;m&#215;150 mm; Grace Vydac 208MS5.315) eluted at 4.5 &#181;l per min.</p><p>Site-specific modification of lysyl residues was determined by LC/MS/MS on a Waters Q-Tof Premier mass spectrometer coupled to a nanoACQUITY UPLC system (Waters Corporation) as reported <xref ref-type="bibr" rid="pgen.1004465-Sricholpech2">[25]</xref>. Tryptic peptides containing Lys residues, their hydroxylated and/or glycosylated forms were identified from the LC/MS/MS analyses using manual interpretation of the MS/MS spectra. Relative quantitation of lysine (Lys), hydroxylysine (Hyl), galactosyl-Hyl (G-Hyl) and glucosylgalactosyl-Hyl (GG-Hyl) at a specific glycosylation site was performed by dividing the total ion abundance determined for each species by the sum of the ion abundances of all observed species containing that particular site.</p><p>An intracellular collagen folding assay was performed as described <xref ref-type="bibr" rid="pgen.1004465-Steinmann1">[17]</xref>. Confluent cells were stimulated overnight in media with 10% FBS and 100 &#181;g/ml ascorbic acid, and then incubated in serum free media containing 100 &#181;g/ml ascorbic acid for 2 hr. Cells were pulsed with 1.4 &#181;Ci/ml <sup>14</sup>C-proline for 15 min to label procollagen chains, followed by collection of the cell layer every 5 min. Each sample was digested for 2 min at 20&#176;C with 0.2% Triton X-100, 100 &#181;g/ml trypsin, and 250 &#181;g/ml chymotrypsin in PBS (Sigma). Digestions were stopped by addition of 1 mg/ml soybean trypsin inhibitor (Sigma). Samples were precipitated overnight, collected by centrifugation, electrophoresed on 3&#8211;8% Tris-acetate gels (Life Technologies) and quantitated by densitometry of autoradiograms. Assays were performed in duplicate on two independent cultures for each genotype.</p><p>For pulse-chase assays, performed as described <xref ref-type="bibr" rid="pgen.1004465-Forlino2">[72]</xref>, wild-type and homozygous <italic>Ppib</italic>-null FB were grown to confluence. For each cell line, two wells were used for cell counts. Procollagens were harvested at the indicated times, digested with pepsin and precipitated. Samples were loaded for equivalent cell number on 3&#8211;8% Tris-acetate gels. Collagen alpha chains were quantitated by densitometry and expressed as the percent secreted at each time point, as determined by (media)/(cell + media)&#215;100.</p><p>Wild-type and homozygous <italic>Ppib</italic>-null FB and OB were grown to confluence and stimulated every other day for 14 days with fresh DMEM (fibroblasts) or &#945;MEM (osteoblasts) containing 10% FBS and 100 &#181;g/ml ascorbic acid, as described <xref ref-type="bibr" rid="pgen.1004465-Cabral2">[73]</xref>. Matrix collagens were sequentially extracted at 4&#176;C, with neutral salt for newly incorporated collagen, then acetic acid for collagens with acid-labile cross-links, and, finally, by pepsin digestion for collagens with mature cross-links <xref ref-type="bibr" rid="pgen.1004465-Bateman1">[74]</xref>. All fractions were electrophoresed on 6% polyacrylamide-urea-SDS gels. Samples were loaded for equivalent densitometry signal; the total signal for each fraction was calculated by adjusting the gel signal by the total volume of that fraction. In separate experiments, quantitation of matrix deposited in culture was performed using Raman microspectroscopy. Cultures were fixed in 1% paraformaldehyde and analyzed as previously described <xref ref-type="bibr" rid="pgen.1004465-Barnes3">[36]</xref>. Matrix collagen:cell organics ratios were evaluated from decomposition of corrected spectra of collagen-free cytoplasm and purified collagen in the amide III spectral region.</p><p>A dermal biopsy was obtained from abdominal skin of wild-type and homozygous null mice, then processed as described <xref ref-type="bibr" rid="pgen.1004465-Cabral3">[75]</xref>. Representative areas of the stained grids were photographed in a Zeiss EM10 CA transmission electron microscope (JFE Enterprises).</p>