<p>Stage 1 discovery samples included 17 T2D GWAS studies (ARIC, CARDIA, CFS, CHS, FamHS, GeneSTAR, GENOA, HANDLS, Health ABC, HUFS, JHS, MESA, MESA Family, SIGNET-REGARDS, WFSM, FIND, and WHI) with up to 23,827 African American subjects (8,284 cases and 15,543 controls). Stage 2 replication samples included up to 11,544 African American subjects (6,061 cases and 5,483 controls), using <italic>in silico</italic> replication of GWAS data from eMERGE and IPM Biobank and <italic>de novo</italic> genotyping in IRAS, IRASFS, SCCS, and WFSM. In general, T2D cases were defined as having at least one of the following: fasting plasma glucose &#8805;126 mg/dl, 2 hour glucose during oral glucose tolerance test (OGTT) &#8805;200 mg/dl, random glucose &#8805;200 mg/dl, oral hypoglycemic agent or insulin treatment, or physician-diagnosed diabetes. All cases were diagnosed at &#8805;25 years (or age at study &#8805;25 years if age at diagnosis was not available). For cohort studies, individuals who met the criteria at any of the visits were defined as cases. Controls with normal glucose tolerance (NGT) were defined by satisfying all the following criteria: fasting plasma glucose &lt;100 mg/dl, 2 hour OGTT&lt;140 mg/dl (if available), no treatment of diabetes, and age &#8805;25 years. For cohort studies, individuals who met the criteria at all visits were defined as controls. All study participants provided written informed consent, except for eMERGE that use an opt out program, and approval was obtained from the institutional review board (IRB) from the respective local institutions. Detailed descriptions of the participating studies are provided in <xref ref-type="supplementary-material" rid="pgen.1004517.s014">Text S1</xref>.</p><p>For stage 1 and 2 GWAS studies, genotyping was performed with Affymetrix or Illumina genome-wide SNP arrays. Imputation of missing genotypes was performed using MACH <xref ref-type="bibr" rid="pgen.1004517-Li2">[49]</xref>, IMPUTE2 <xref ref-type="bibr" rid="pgen.1004517-Howie1">[50]</xref> or BEAGLE <xref ref-type="bibr" rid="pgen.1004517-Browning1">[51]</xref> using HapMap reference haplotypes. For each study, samples reflecting duplicates, low call rate, gender mismatch, or population outliers were excluded. In general, SNPs were excluded by the following criteria: call rate &lt;0.95, minor allele frequency (MAF)&lt;0.01, minor allele count &lt;10, Hardy-Weinberg P-value &lt;1&#215;10<sup>&#8722;4</sup>, or imputation quality score &lt;0.5 (<xref ref-type="supplementary-material" rid="pgen.1004517.s007">Table S3</xref>). For <italic>de novo</italic> replication studies, genotyping was performed using the Sequenom MassArray platform (Sequenom; San Diego, CA). Sample and SNP quality controls were performed as with GWAS data.</p><p>Single SNP association was performed for each study by regressing T2D case/control status on genotypes. To account for uncertainty of genotype calls during imputation, genotype probabilities or dosage were used for association tests in imputed SNPs. The association tests assumed an additive genetic model and adjusted for age, sex, study centers, and principal components. Principal components were included to control for confounding effects of admixture proportion and population structure. Secondary analysis with additional adjustment for BMI was performed for SNPs with <italic>P</italic>&lt;1&#215;10<sup>&#8722;5</sup> in stage 1 meta-analysis and index SNPs previously reported to be associated with T2D or glucose homeostasis traits. BMI adjustment allows increasing power to detect T2D loci independent of BMI effect and diminish associations at T2D loci with effects modulated through BMI. Logistic regression was used for samples of unrelated individuals. Generalized estimating equations <xref ref-type="bibr" rid="pgen.1004517-Chen1">[52]</xref> or SOLAR <xref ref-type="bibr" rid="pgen.1004517-Almasy1">[53]</xref> were used for samples of related individuals. Association results with extreme values (absolute beta coefficient or standard error &gt;10), primarily due to low cell counts resulting from small sample sizes and/or low minor allele frequencies, were excluded (<xref ref-type="supplementary-material" rid="pgen.1004517.s007">Table S3</xref>).</p><p>In stage 1, association results were combined by a fixed effect model with inverse variance weighted method using the METAL software <xref ref-type="bibr" rid="pgen.1004517-Willer1">[12]</xref>. Genomic control correction <xref ref-type="bibr" rid="pgen.1004517-Devlin1">[11]</xref> was applied to each study before meta-analysis, and to the overall results after meta-analysis. Results from SNPs genotyped in &lt;10,000 samples and those with allele frequency difference &gt;0.3 among studies were excluded. A total of 2,579,389 SNPs were analyzed in the meta-analysis (<xref ref-type="supplementary-material" rid="pgen.1004517.s007">Table S3</xref>). In stage 2a, association results from African American replication studies were also combined using a fixed effect inverse variance weighted method. To assess the overall effects in African Americans (stage 1+2a) and both African Americans and Europeans (stage 1+2a+2b), association results from studies in the respective stages were combined using a fixed effect inverse variance weighted method. Genome-wide significance is declared at <italic>P</italic>&lt;5&#215;10<sup>&#8722;8</sup> from the meta-analysis result of all stages, which has better power than the replication-based strategy <xref ref-type="bibr" rid="pgen.1004517-Skol1">[54]</xref>.</p><p>Among the 51 SNPs carried forward for replication, heterogeneity of effect sizes across studies within each stage was assessed using Cochran's Q statistic implemented in METAL. Meta-analysis results from stages 1 and 2a, stage 1+2a and 2b were used to assess heterogeneity of effect sizes between discovery and replication stages in African Americans, and between African Americans and Europeans, respectively. For SNPs with significant heterogeneous effect size after multiple comparison corrections (<italic>P</italic>
<sub>het</sub>&lt;0.001), meta-analysis results including studies of all stages assessed by the random effect model implemented in GWAMA <xref ref-type="bibr" rid="pgen.1004517-Magi1">[55]</xref> were reported. Heterogeneous associations may partly due to differences in ascertainment scheme across studies. For index SNPs reported in prior studies, assessment of heterogeneity using Cochran's Q statistic between prior studies and this study were also reported.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Index SNPs associated with T2D or glucose homeostasis traits from prior GWAS and candidate gene studies were examined for association with T2D in African Americans (<xref ref-type="supplementary-material" rid="pgen.1004517.s009">Table S5</xref>). For the index SNP association tests, a per-SNP <italic>P</italic> value &lt;0.05 was defined as significant. In the locus-wide analysis, the boundaries of a locus were defined by the most distant markers (within &#177;500 kb) using the 1KGP CEU data with r<sup>2</sup>&#8805;0.3 with the index SNP. All MEDIA SNPs within these bounds were examined for association analysis. All pairwise LD values within each locus were estimated using the 1KGP CEU and ASW data. To estimate the effective number of SNPs at a locus, we retrieved genotypes from the 1KGP ASW data for markers present in MEDIA, estimated the sample covariance matrix from those genotypes, and spectrally decomposed the covariance matrix <xref ref-type="bibr" rid="pgen.1004517-Ramos1">[24]</xref>. The effective number of SNPs was estimated using the relationship <inline-formula><inline-graphic ns0:href="pgen.1004517.e001.jpg" /></inline-formula>, in which <italic>&#955;<sub>k</sub></italic> is the <italic>k</italic>
<sup>th</sup> eigenvalue of the <italic>K</italic>&#215;<italic>K</italic> covariance matrix for the <italic>K</italic> SNPs in the locus <xref ref-type="bibr" rid="pgen.1004517-Ramos1">[24]</xref>. The per-locus significance level was defined as 0.05/effective number of SNPs (<xref ref-type="supplementary-material" rid="pgen.1004517.s010">Table S6</xref>). By accounting for all SNPs within the bounds of LD, the per-locus significance level is corrected to account for markers in LD with the index SNP as well as markers not in LD with the index SNP, thereby potentially allowing for discovery of new associations at markers not tagged by the index SNP.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For each independent locus, we estimated the sibling relative risk using the most strongly associated SNP within that locus. Let <italic>p<sub>i</sub></italic> and <italic>&#968;<sub>i</sub></italic> be the risk allele frequency and the corresponding odds ratio at the <italic>i</italic>
<sup>th</sup> SNP, respectively. Assuming the additive genetic model and independence between SNPs, the contribution to the sibling relative risk <italic>&#955;<sub>s</sub></italic> for a set of <italic>N</italic> SNPs is given by <inline-formula><inline-graphic ns0:href="pgen.1004517.e002.jpg" /></inline-formula>
<xref ref-type="bibr" rid="pgen.1004517-Lin1">[56]</xref>. Let <italic>K</italic> be the disease prevalence. The liability-scale variance <inline-formula><inline-graphic ns0:href="pgen.1004517.e003.jpg" /></inline-formula> explained by the set of <italic>N</italic> SNPs is given by <inline-formula><inline-graphic ns0:href="pgen.1004517.e004.jpg" /></inline-formula>, in which <inline-formula><inline-graphic ns0:href="pgen.1004517.e005.jpg" /></inline-formula>, <inline-formula><inline-graphic ns0:href="pgen.1004517.e006.jpg" /></inline-formula>, and <inline-formula><inline-graphic ns0:href="pgen.1004517.e007.jpg" /></inline-formula>, with <inline-formula><inline-graphic ns0:href="pgen.1004517.e008.jpg" /></inline-formula> representing the standard normal quantile function and <italic>z</italic> representing the standard normal density at <italic>T</italic>
<xref ref-type="bibr" rid="pgen.1004517-Wray1">[57]</xref>.</p><p>The coverage of MEDIA SNPs to the human genome was estimated using HaploView <xref ref-type="bibr" rid="pgen.1004517-Barrett2">[58]</xref> via pairwise tagging at the r<sup>2</sup>&#8202;=&#8202;0.8 threshold. We used all SNPs with minor allele frequencies &#8805;1% in both MEDIA and the 1KGP ASW sequence data. Coverage was estimated using non-overlapping bins of 1,000 SNPs.</p><p>Study power was calculated using the genetic power calculator <xref ref-type="bibr" rid="pgen.1004517-Purcell1">[59]</xref>. For SNPs with MAF&#8805;0.3, our study had &gt;80% power to detect odds ratios for T2D at OR&#8805;1.06 and &#8805;1.13 at <italic>P</italic>&lt;0.05 and <italic>P</italic>&lt;5&#215;10<sup>&#8722;8</sup>, respectively, in stage 1 samples under an additive model. The observed odds ratios among our stage 1 most significantly associated SNPs with <italic>P</italic>&lt;1&#215;10<sup>&#8722;5</sup> ranged from 1.11 to 1.56 (<xref ref-type="supplementary-material" rid="pgen.1004517.s008">Table S4</xref>). Given our African American sample size in stage 1+2a, our study had &gt;80% power to detect OR&#8805;1.1 at <italic>P</italic>&lt;5&#215;10<sup>&#8722;8</sup> at MAF&#8805;0.3, thus provided good power to detect genome-wide significance among the most significantly associated SNPs using all African American samples. For T2D SNPs reported from the literature, power was also calculated from the reported effect size using the risk allele frequency from this study for stage 1 samples at <italic>P</italic>&lt;0.05 and <italic>P</italic>&lt;5&#215;10<sup>&#8722;8</sup>, respectively (<xref ref-type="supplementary-material" rid="pgen.1004517.s009">Table S5</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The MuTHER resource (<ext-link ext-link-type="uri" ns0:href="http://www.muther.ac.uk">www.muther.ac.uk</ext-link>) includes lymphoblastoid cell lines (LCLs), skin, and adipose tissue derived simultaneously from a subset of well-phenotyped healthy female twins from the TwinsUK adult registry <xref ref-type="bibr" rid="pgen.1004517-Nica1">[60]</xref>. Whole-genome expression profiling of the samples, each with either two or three technical replicates, was performed using the Illumina Human HT-12 V3 BeadChips (Illumina Inc.) according to the protocol supplied by the manufacturer. Log<sub>2</sub>-transformed expression signals were normalized separately per tissue as follows: quantile normalization was performed across technical replicates of each individual followed by quantile normalization across all individuals. Genotyping was performed with a combination of Illumina arrays (HumanHap300, HumanHap610Q, 1M-Duo, and 1.2MDuo 1M). Untyped HapMap2 SNPs were imputed using the IMPUTE2 software package. In total, 776 adipose and 777 LCL samples had both expression profiles and imputed genotypes. Association between all SNPs (MAF&gt;5%, IMPUTE info &gt;0.8) within a gene or within 1 Mb of the gene transcription start or end site and normalized expression values were performed with the GenABEL/ProbABEL packages <xref ref-type="bibr" rid="pgen.1004517-Aulchenko1">[61]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004517-Aulchenko2">[62]</xref> using the polygenic linear model incorporating a kinship matrix in GenABEL followed by the ProbABEL mmscore score test with imputed genotypes. Age and experimental batch were included as cofactors.</p><p>Genotype and gene expression in LCL in HapMap samples were also available <xref ref-type="bibr" rid="pgen.1004517-Stranger1">[63]</xref>. Association of genotypes and gene expression of transcripts within 1 MB of tested SNPs were analyzed separately for CEU and YRI populations. The variance components model implemented in SOLAR was used for association analysis which accounts for correlation among related individuals <xref ref-type="bibr" rid="pgen.1004517-Almasy1">[53]</xref>.</p><p>In this study, we examined the association of the most significantly associated SNPs from the six genome-wide significant loci and their proxies (r<sup>2</sup>&#8805;0.8 in ASW) within 1 Mb of the associated SNPs with <italic>cis</italic>-expression quantitative trait loci (eQTLs) in peripheral blood leukocytes (LCL) and adipose tissue (<xref ref-type="supplementary-material" rid="pgen.1004517.s012">Table S8</xref>).</p><p>We examined putative function of non-coding genome-wide significant SNPs and their proxies within 1 Mb (r<sup>2</sup>&#8805;0.8 in 1KGP ASW) using HaploReg <xref ref-type="bibr" rid="pgen.1004517-Ward1">[30]</xref> and RegulomeDB <xref ref-type="bibr" rid="pgen.1004517-Boyle1">[64]</xref>. These databases interrogated multiple chromatin features from the Encyclopedia of DNA Elements (ENCODE) project <xref ref-type="bibr" rid="pgen.1004517-Dunham1">[29]</xref>. High priority was given to variants annotated as protein-binding via ChIP-seq, and motif-changing via position weight matrices, with the respective transcription factors implicated in diabetes pathogenesis and related biological processes.</p>