<p>Yeast strains were grown at 30&#176;C in rich YPD medium (1% yeast extract, 2% peptone and 2% glucose) or defined synthetic complete (SC) medium (1.45g yeast nitrogen base, 5g ammonium sulfate, 2% glucose and 2g amino acid mix per liter) lacking leucine, uracil or histidine wherever appropriate for selection of plasmids; and lacking isoleucine and valine (Ile/Val) for treatment with sulfometuron (SM) at 0.5 &#181;g/ml. Increasing threonine in SC medium from approximately 1 mM to 2.5 mM diminished the slow growth phenotype of <italic>hom6&#916;</italic> cells, and eliminated that of <italic>thr1&#916;</italic> and <italic>thr4&#916;</italic> cells, in SC medium lacking Ile/Val. Therefore, overnight growth to saturation was achieved in SC medium supplemented with 2.5 mM threonine and thereafter yeast strains were cultured in SC with 1 mM threonine. Accordingly, a moderate threonine limitation was imposed in our experiments and, as threonine is a precursor in Ile/Val biosynthesis (<xref ref-type="fig" rid="pgen-1004534-g001">Fig. 1A</xref>), this should intensify the limitation for Ile/Val provoked by SM treatment.</p><p>Yeast strains used in this study are listed in <xref ref-type="table" rid="pgen-1004534-t001">Table 1</xref>. Yeast strains purchased from Research Genetics or previously reported were verified for all auxotrophic requirements indicated in the genotype; and gene deletions were confirmed by PCR amplification of predicted deletion junctions using primers described in <xref ref-type="supplementary-material" rid="pgen.1004534.s009">Table S1</xref>. To generate <italic>HOM6</italic> deletion strains, the appropriate <italic>hphMX4</italic> gene deletion cassette conferring hygromycin B resistance <xref ref-type="bibr" rid="pgen.1004534-Goldstein1">[40]</xref> was PCR-amplified from plasmid pAG32 using primers HOM6-MX4-F and HOM6-MX4-R, thus introducing homologous flanking sequences upstream and downstream of <italic>HOM6</italic> coding sequences, and used to delete <italic>HOM6</italic> by transforming the appropriate strains to hygromycin B resistance on YPD agar plates. <italic>HOM6</italic> deletion was further confirmed by demonstrating acquisition of threonine auxotrophy, except when deleted in <italic>hom2&#916;</italic> or <italic>hom3&#916;</italic> strains F2057 and F1929, respectively; and by PCR-amplification of predicted junction fragments containing <italic>hphMX4</italic> and sequences upstream or downstream of <italic>HOM6</italic> coding sequences using primer pairs HOM6-A/HphMX-R1 and HOM6-DN-R/HphMX-F1 respectively. To generate <italic>SRB10-myc<sub>13</sub></italic> and <italic>PHO85-myc<sub>13</sub></italic> strains, a <italic>myc<sub>13</sub>::HIS3MX6</italic> cassette was PCR-amplified from plasmid pFA6a-13myc-HIS3MX6 using primer pairs SRB10-MYC13-F/SRB10-MYC13-R or PHO85-MYC13-F/PHO85-MYC13-R, respectively, and used to transform strains BY4741 and YR001to His<sup>+</sup>. Cassette insertions were confirmed by PCR analysis of genomic DNA using the appropriate primers specific for the <italic>myc<sub>13</sub>::HIS3MX6</italic> cassette and <italic>SRB10</italic> or <italic>PHO85</italic>, and by Western analysis of whole cell extracts (WCEs) using anti-Myc antibodies (Roche). To generate <italic>GCN4</italic> deletion strains, plasmid pHQ1240 containing a <italic>gcn4&#916;::hisG::URA3::hisG</italic> cassette was digested with SspI and used to transform strains F947 and YR006 to Ura<sup>+</sup>. Deletion of <italic>GCN4</italic> was indicated by acquisition of SM-sensitivity and verified by PCR amplification of <italic>gcn4&#916;::hisG::URA3::hisG</italic> from chromosomal DNA using primer pairs specific for sequences upstream and downstream of the <italic>GCN4</italic> CDS. The <italic>URA3</italic> gene was subsequently evicted by selecting for growth on medium containing 5-fluoroorotic acid.</p><p>NA, not applicable.</p><p>All plasmids used in this study are listed in <xref ref-type="table" rid="pgen-1004534-t002">Table 2</xref>, and primers used in plasmid constructions are listed in <xref ref-type="supplementary-material" rid="pgen.1004534.s009">Table S1</xref>. To construct pYPR010, <italic>HOM6</italic> (chrX:689,322.690,749) was PCR-amplified from chromosomal yeast DNA of strain BY4741, using primers HOM6-HindIII-F and HOM6-BamHI-R, and inserted between the <italic>Hind</italic>III and <italic>Bam</italic>HI sites in YCplac111. To construct pYPR018, pYPR020, pYPR022 and PYPR024, <italic>HOM6</italic> in pYPR010 was mutagenized using the QuikChange II XL Site-Directed Mutagenesis Kit (Stratagene) to produce <italic>hom6</italic> mutant alleles encoding the K117A, E208D, E208L and D219L substitutions, respectively, using sets of complementary primer pairs harboring the corresponding mutations (<xref ref-type="supplementary-material" rid="pgen.1004534.s009">Table S1</xref>).</p><p>sc, single copy.</p><p>lc, low copy.</p><p>hc, high copy.</p><p>To construct pYPR028, <italic>HOM3</italic> (chrV:256,132.258,737) was PCR-amplified from chromosomal yeast DNA of BY4741 using primers HOM3-F1 and HOM3-R1 and inserted between the <italic>Spe</italic>I and <italic>Eco</italic>RI sites in pRS313. pYPR030, containing <italic>HOM3-E282D (HOM3<sup>fbr</sup></italic>), was generated by fusion-PCR using HOM3-F1 and HOM3-R1 as outside primers and complementary primers HOM3-E282D-F and HOM3-E282D-R encoding the appropriate mutation, and inserted between the <italic>Spe</italic>I and <italic>Eco</italic>RI sites of pRS313. To construct pYPR013, the <italic>Apa</italic>I-<italic>SpeI</italic> fragment containing <italic>GCN4</italic> was isolated from plasmid p164, polishing the <italic>Apa</italic>I end using Klenow polymerase exonuclease activity, and inserted between the <italic>Sma</italic>I and <italic>Spe</italic>I sites of YCplac111. pYPR038 and pYPR047 were constructed by fusion-PCR using Gcn4c-SphI-F and GCN4c-SpeI-R as outside primers in combination with primers GCN4-K50,58R-F and GCN4-K50,58R-R or primers GCN4-T165A-F and GCN4-T165A-R, respectively, and pYPR013 as PCR template. The PCR products were inserted between the <italic>Sph</italic>I and <italic>Spe</italic>I sites in YCplac111.</p><p>Yeast strains transformed with plasmids pHYC2 (<italic>UAS<sub>GCRE</sub>-CYC1-lacZ</italic>), p367 (<italic>HIS4-lacZ</italic>), or p180 (<italic>GCN4-lacZ</italic>) were grown to saturation and diluted in two identical cultures in SC-Ura/Ile/Val at A<sub>600</sub>&#8202;=&#8202;0.5, and after 2.5 h of growth, 0.5 &#181;g/ml SM was added to one set of cultures. Cells were harvested from untreated (unstarved) cultures after a total 6 h of growth and SM-treated cultures grown for 6 h in the presence of SM <xref ref-type="bibr" rid="pgen.1004534-Swanson1">[7]</xref>. Whole cell extracts (WCEs) were prepared and assayed for &#946;-galactosidase activity as previously described <xref ref-type="bibr" rid="pgen.1004534-Moehle1">[41]</xref>. Mean specific activities were calculated from results obtained from three independent transformants.</p><p>Yeast strains were cultured to an A<sub>600</sub> of 0.4&#8211;0.6 in SC-Ile/Val, achieving at least two cell doublings, and treated with 0.5 &#181;g/ml SM for the indicated times or left untreated. Total RNA was isolated by hot phenol extraction as previously described <xref ref-type="bibr" rid="pgen.1004534-Schmitt1">[42]</xref>. RNA concentration was quantified by Nanodrop spectroscopy and analyzed for integrity by agarose gel electrophoresis and ethidium bromide staining. An aliquot of 1 &#181;g total RNA was used for cDNA synthesis using SuperScript III First-strand Synthesis Supermix for qRT-PCR (Invitrogen) and the resulting cDNA was diluted 10-fold. qRT-PCR was performed using Brilliant III Ultra-Fast qPCR Master Mix (Agilient Technologies) using the diluted cDNA in multiplex PCR and the appropriate TaqMan probes (<xref ref-type="supplementary-material" rid="pgen.1004534.s009">Table S1</xref>) to quantify <italic>ACT1</italic> (labelled with FAM), <italic>ARG1</italic>, or <italic>HIS4</italic> (both labelled with HEX). qRT-PCR reactions were performed in triplicate using cDNA synthesized from RNA extracted from at least two independent cultures. <italic>ARG1</italic> or <italic>HIS4</italic> cDNA abundance was normalized to that of <italic>ACT1</italic> by calculating 2<sup>(&#8722;&#916;C</sup>
<sub>t</sub>
<sup>)</sup>, where &#916;C<sub>t</sub> is (C<sub>t (Target)</sub>- C<sub>t (<italic>ACT1</italic>)</sub>). Fold changes in mRNA abundance were normalized to those measured in uninduced WT cells, or as indicated, and plotted.</p><p>ChIP assays were conducted essentially as described previously <xref ref-type="bibr" rid="pgen.1004534-Swanson1">[7]</xref>, <xref ref-type="bibr" rid="pgen.1004534-Gaur1">[43]</xref>. Yeast strains were cultured in 100 ml SC-Ile/Val as described above for RNA isolation, treated with 0.5 &#181;g/ml SM for 2 h or as indicated, cross-linked for 15 min with 10 ml formaldehyde solution (50 mM HEPES KOH, pH 7.5, 1 mM EDTA, 100 mM NaCl and 11% formaldehyde) and quenched with 15 ml of 2.5 M glycine. WCEs were prepared by glass beads lysis in 400 &#181;l FA lysis buffer (50 mM HEPES KOH, pH 7.5, 1 mM EDTA, 150 mM NaCl, 1% TritonX-100 and 0.1% Na-deoxycholate) with protease inhibitors for 45 min at 4&#176;C and the supernatant collected after removing the beads was pooled with 600 &#181;l FA lysis buffer used for washing the beads. The resulting lysate was sonicated to yield DNA fragments of 300&#8211;500 bp and cleared by centrifugation. 50 &#181;l aliquots of lysates were immunoprecipitated for 2 h at 4&#176;C with &#945;-Gcn4, (Rabbit) <xref ref-type="bibr" rid="pgen.1004534-Zhang1">[13]</xref> or &#945;-Rpb3 antibodies (Mouse, Neoclone) coupled with &#945;-rabbit IgG or &#945;-mouse IgG conjugated magnetic beads (Dynabeads, Invitrogen), respectively, or with &#945;-c-Myc (Rabbit, Roche) coupled with &#945;-rabbit IgG conjugated magnetic beads. Recovered immune complexes were washed and eluted as described <xref ref-type="bibr" rid="pgen.1004534-Gaur1">[43]</xref>. For matched input and IP samples, the crosslinks were reversed by incubation at 65&#176;C overnight, treated with proteinase K, extracted twice with phenol:chloroform:isoamyl alcohol (25&#8758;24&#8758;1) and once with chloroform:isoamyl alcohol (24&#8758;1), and ethanol precipitated, resuspending the resulting pellets in 30&#8211;40 &#181;l TE containing RNAase as described earlier <xref ref-type="bibr" rid="pgen.1004534-Gaur1">[43]</xref>. Quantitative PCRs were performed in the presence of [<sup>33</sup>P]-dATP with undiluted IP DNA and 500-fold diluted input DNA and further analyzed as previously described <xref ref-type="bibr" rid="pgen.1004534-Swanson1">[7]</xref>, <xref ref-type="bibr" rid="pgen.1004534-Gaur1">[43]</xref>. The primers employed for ChIP analysis are listed in <xref ref-type="supplementary-material" rid="pgen.1004534.s009">Table S1</xref>.</p><p>WCEs were prepared in denaturing conditions with trichloroacetic acid, as described previously <xref ref-type="bibr" rid="pgen.1004534-Reid1">[44]</xref> and analyzed by immunoblotting with &#945;-Gcd6 <xref ref-type="bibr" rid="pgen.1004534-Cigan1">[45]</xref> and affinity purified &#945;-Gcn4 antibodies <xref ref-type="bibr" rid="pgen.1004534-Zhang1">[13]</xref>. Western signals were quantified by ImageJ software.</p><p>The analysis was performed essentially as previously described <xref ref-type="bibr" rid="pgen.1004534-Zhang1">[13]</xref>, <xref ref-type="bibr" rid="pgen.1004534-Kornitzer2">[46]</xref>. Yeast cells collected from a 10 ml culture at A<sub>600</sub>&#8202;=&#8202;0.4&#8211;0.6 were washed with SC-Met/Ile/Val, inoculated into 0.5 ml SC-Met/Ile/Val containing 1 &#181;g/ml SM and incubated for 15 min in a shaking water bath at 30&#176;C; after which 1.0 mCi [<sup>35</sup>S]methionine/cysteine labelling mix was added and incubation continued for an additional 15 min. Cells were collected, transferred to 5 ml of pre-warmed SC-Ile/Val containing 10 mM methionine and 10 mM cysteine, and an 1 ml aliquots were removed immediately or after appropriate times of chase. Aliquots were denatured with 170 &#181;l of 1.85 M NaOH, 7.4% 2-marcaptoethanol and precipitated with 70 &#181;l 100% TCA on ice, washed with chilled acetone and dried under vacuum in a SpeedVac. The dried pellets were resuspended in 120 &#181;l of 2.5% SDS, 5 mM EDTA, 1 mM PMSF by vortexing, boiled for 1 min, and cleared by centrifugation. Incorporation of label was measured by scintillation counting <xref ref-type="bibr" rid="pgen.1004534-Zhang1">[13]</xref> and aliquots of extract containing equal amounts of radioactivity (5.7&#215;10<sup>5</sup> cpm) were combined with 1 ml of immunoprecipitation (IP) buffer (50 mM Na-HEPES [pH 7.5], 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 1 mM PMSF) containing 1 mg/ml BSA and 1 &#181;l affinity-purified &#945;-Gcn4 antibodies and mixed by rotating at 4&#176;C for 2 h. Twenty &#181;l of a 50% slurry of protein A-agarose beads pretreated with IP buffer containing BSA (1 mg/ml) was added, and mixing continued for 2 h. The beads were washed thrice with 500 &#181;l cold IP buffer containing 0.1% SDS, resuspended in loading buffer, boiled, and resolved by SDS-PAGE using 4 to 20% gels. The gel was dried and subjected to autoradiography, and the [<sup>35</sup>S]-labeled Gcn4 was quantified by phosphorimaging analysis.</p><p>A modification of a previously described protocol was employed <xref ref-type="bibr" rid="pgen.1004534-Montpetit1">[47]</xref>, as follows. Yeast strains were cultured and treated with SM as described above for ChIP analysis. 40&#8211;60 A<sub>600</sub> units of cells were lysed at 4&#176;C with glass beads by 10 cycles of vortexing, 30s-on and 30s-off, in 500 &#181;L of chilled lysis buffer (50 mM Tris-HCl [pH 8.0], 5 mM EDTA, 150 mM NaCl, 0.2% Triton &#215;100 and 1 mM PMSF) containing 10 mM sodium ethyl maleimide (NEM) and protease inhibitors. The resulting lysate was cleared by centrifugation at 13,000 rpm for 30 min at 4&#176;C and soluble protein concentration was determined by the Bio-Rad protein assay. For each sample, a 40 &#181;l suspension of magnetic beads conjugated with &#945;-Rabbit IgG (Dynabeads, Invitrogen) was washed twice with lysis buffer containing 5 mg/mL BSA and rotated with 1 &#181;l affinity purified &#945;-Gcn4 antibody <xref ref-type="bibr" rid="pgen.1004534-Zhang1">[13]</xref> in 200 &#181;l lysis buffer/BSA for 3 h at 4&#176;C. The magnetic beads coupled with &#945;-Gcn4 antibody were washed twice with lysis buffer/BSA to remove unbound antibody and resuspended in 200 &#181;l lysis buffer/BSA. Aliquots containing 1 mg of protein were added to the magnetic beads suspension, adjusting the final volume to 500 &#181;l with lysis buffer/BSA, and further rotated for 2 h at 4&#176;C. IP samples were washed thrice with lysis buffer containing 0.1% SDS and resuspended in 30 &#181;l 1&#215; Novex tris-glycine SDS sample buffer (Invitrogen) and boiled for 3 min. Aliquots of 5 &#181;l and 25 &#181;l were subjected to Western analysis with &#945;-Gcn4 antibodies <xref ref-type="bibr" rid="pgen.1004534-Zhang1">[13]</xref> and &#945;-SUMO (&#945;-Smt3) polyclonal antibodies <xref ref-type="bibr" rid="pgen.1004534-Montpetit1">[47]</xref>.</p>