<p>hMSCs were purchased from ATCC and grown in hMSC medium (ATCC). U2OS, SAOS2 and HOS-MNNG cells were purchased from ATCC and grown in DMEM+10% FBS+antibiotics. Hu09-M112 is a subclone (generous gift from Dr. Jun Yokota, Biology Division, National Cancer Center Research Institute, Japan) from Hu09 cells and grown in RPMI1640+10% FBS+antibiotics [<xref ref-type="bibr" rid="pgen.1005884.ref039">39</xref>]. mMSCs were isolated from the bone marrow of p53 knockout mice, as previously described [<xref ref-type="bibr" rid="pgen.1005884.ref018">18</xref>]. Dunn cells and MC3T3-E1 cells (ATCC) were grown in DMEM+10% FBS+antibiotics.</p><p>Ten million SAOS2 cells were transduced with lentivirus expressing shLuc, shRUNX2_3 or shRUNX2_4 for 24 hours. Cells were re-suspended in 100 ul of PBS buffer+25mM HEPES, and mixed with 50 ul Matrigel before being transplanted into hind limb muscle of NSG mice. 43 days after transplantation, tumors were harvested, weighed and fixed in 10% neutral buffered formalin for 16 hours before Hematoxylin and Eosin staining. For Hu09-M112 cells, 1 million cells were transplanted into NSG mice. 69 days after transplantation, tumors were harvested, weighed and fixed in 10% neutral buffered formalin for 16 hours before Hematoxylin and Eosin staining. Mice were maintained under the strict guidelines of the Institutional Animal Care and Use Committee (IACUC)-approved protocols of the National Cancer Institute and National Heart, Lung, and Blood Institute.</p><p>To calculate the cumulative cell numbers, cells were split at a constant ratio (splitting ratio) for each passage. At each splitting, the cell number was counted and multiplied by the splitting ratio. The final number is the cumulative cell number. The final curve was generated by graphing the cumulative cell numbers over several passages. Note that for each cell line, a no-virus transduction control was included to determine the 0-day of counting, which is the first day when the no-virus control was completely dead after drug selection.</p><p>ChIP-seq was performed in the next generation sequencing facility at the National Cancer Institute (NCI), as previously described [<xref ref-type="bibr" rid="pgen.1005884.ref018">18</xref>,<xref ref-type="bibr" rid="pgen.1005884.ref022">22</xref>]. Peaks were identified by the MACS algorithm [<xref ref-type="bibr" rid="pgen.1005884.ref040">40</xref>]. For calculating peak intensity, the number of tags at each nucleotide within a peak was calculated and summed up for all the nucleotides across the peak. The sum was defined as peak intensity, which was used to calculate the correlation between CBFB and RUNX2 binding.</p><p>To assign the peak to specific genes, we used the approach we developed previously [<xref ref-type="bibr" rid="pgen.1005884.ref022">22</xref>]. Briefly, we arbitrarily defined the promoter region as the region between 5 kb upstream to 5 kb downstream of the transcription start site (TSS) of a transcript. The rest of the region within the gene body (5 kb downstream of TSS to the end of transcription) is defined as the gene body region. We also define regions 25 kb away from the transcript as distal.</p><p>ChIP assay were done in the same way as ChIP-seq. The amplicon enrichment was measured by realtime PCR and calculated as percentage of input. Primers for amplifying the location on the MYC gene are: forward, 5&#8217;-ACTCACAGGACAAGGATGCG-3&#8217;; Reverse, 5&#8217;-TGCTCCTCCGTAGCAGTACT-3&#8217;.</p><p>SAOS2 cells were transduced with lentiviruses expressing shLuc, shRUNX2_3 or shRUNX2_4 for 4 days when we observed significant RUNX2 protein reduction but no obvious apoptosis. RNA-seq was performed as previously described [<xref ref-type="bibr" rid="pgen.1005884.ref018">18</xref>]. We reasoned that at this time point, the underlying transcriptional changes precede the cellular event&#8212;apoptosis. After RNA extraction with Trizol, 1ug total RNA was subjected to deep sequencing in the NextSeq 500 machine at the NCI next generation sequencing facility. Reads were aligned to the human genome (Build hg19). The cufflinks algorithm was used to calculate reads per kilobase per million (RPKM) for each RefSeq transcript.</p><p>shRNAs were cloned into a pLKO.1 backbone, which carries a puromycin resistant gene. For the rescue experiment, SAOS2 cells were transduced with a plenti6-GW-MYC vector and selected in 10 ug/ml of Blasticidin. Stable cells were then transduced with a RUNX2 or CBFB shRNA vector. shRNA sequences are as follows:</p><p>RUNX2(h)_3, top: 5&#8217;-CCGGTGCACTATCCAGCCACCTTTACTCGAGTAAAGGTGGCTGGATAGTGCATTTTT-3&#8217;</p><p>RUNX2(h)_3, bottom: 5&#8217;-AATTAAAAATGCACTATCCAGCCACCTTTACTCGAGTAAAGGTGGCTGGATAGTGCA-3&#8217;</p><p>RUNX2(h)_4, top: 5&#8217;-CCGGGCTACCTATCACAGAGCAATTCTCGAGAATTGCTCTGTGATAGGTAGCTTTTT-3&#8217;</p><p>RUNX2(h)_4, bottom: 5&#8217;-AATTAAAAAGCTACCTATCACAGAGCAATTCTCGAGAATTGCTCTGTGATAGGTAGC-3&#8217;</p><p>RUNX2(m)_4, top: 5&#8217;-CCGGGCAGAATGGATGAGTCTGTTTCTCGAGAAACAGACTCATCCATTCTGCTTTTT-3&#8217;</p><p>RUNX2(m)_4, bottom: 5&#8217;-AATTAAAAAGCAGAATGGATGAGTCTGTTTCTCGAGAAACAGACTCATCCATTCTGC-3&#8217;</p><p>RUNX2(m)_5, top: 5&#8217;-CCGGCCGAGTCATTTAAGGCTGCAACTCGAGTTGCAGCCTTAAATGACTCGGTTTTT-3&#8217;</p><p>RUNX2(m)_5, bottom: 5&#8217;-AATTAAAAACCGAGTCATTTAAGGCTGCAACTCGAGTTGCAGCCTTAAATGACTCGG-3&#8217;</p><p>CBFB(h)_1, top: 5&#8217;- CCGGGAGAAGCAGGCAAGGTATATTCTCGAGAATATACCTTGCCTGCTTCTCTTTTT -3&#8217;</p><p>CBFB(h)_1, bottom: 5&#8217;-AATTAAAAAGAGAAGCAGGCAAGGTATATTCTCGAGAATATACCTTGCCTGCTTCTC-3&#8217;</p><p>CBFB(h)_2, top: 5&#8217;-CCGGCCGCGAGTGTGAGATTAAGTACTCGAGTACTTAATCTCACACTCGCGGTTTTT -3&#8217;</p><p>CBFB(h)_2, bottom: 5&#8217;-AATTAAAAACCGCGAGTGTGAGATTAAGTACTCGAGTACTTAATCTCACACTCGCGG-3&#8217;</p><p>MLL1(h)_1, top: 5&#8217;-CCGGGATTCGAACACCCAGTTATTCCTCGAGGAATAACTGGGTGTTCGAATCTTTTT -3&#8217;</p><p>MLL1(h)_1, bottom: 5&#8217;-AATTAAAAAGATTCGAACACCCAGTTATTCCTCGAGGAATAACTGGGTGTTCGAATC-3&#8217;</p><p>MLL1(h)_5, top: 5&#8217;-CCGGTGCCTGGAAGGAGCCTATTATCTCGAGATAATAGGCTCCTTCCAGGCATTTTT-3&#8217;</p><p>MLL1(h)_5, bottom: 5&#8217;-AATTAAAAATGCCTGGAAGGAGCCTATTATCTCGAGATAATAGGCTCCTTCCAGGCA-3&#8217;</p><p>Menin(h)_2, top: 5&#8217;-CCGGCTGTACCTGAAAGGATCATACCTCGAGGTATGATCCTTTCAGGTACAGTTTTT-3&#8217;</p><p>Menin(h)_2, bottom: 5&#8217;-AATTAAAAACTGTACCTGAAAGGATCATACCTCGAGGTATGATCCTTTCAGGTACAG -3&#8217;</p><p>Menin(h)_5, top: 5&#8217;- CCGGTCTACGACGGCATCTGCAAATCTCGAGATTTGCAGATGCCGTCGTAGATTTTT -3&#8217;</p><p>Menin(h)_5, bottom: 5&#8217;-AATTAAAAATCTACGACGGCATCTGCAAATCTCGAGATTTGCAGATGCCGTCGTAGA-3&#8217;</p><p>MYC(h)_4, top: 5&#8217;-CCGGCCTGAGACAGATCAGCAACAACTCGAGTTGTTGCTGATCTGTCTCAGGTTTTT-3&#8217;</p><p>MYC(h)_4, bottom: 5&#8217;-AATTAAAAACCTGAGACAGATCAGCAACAACTCGAGTTGTTGCTGATCTGTCTCAGG-3&#8217;</p><p>MYC(h)_5, top: 5&#8217;-CCGGCCTGAGACAGATCAGCAACAACTCGAGTTGTTGCTGATCTGTCTCAGGTTTTT-3&#8217;</p><p>MYC(h)_5, bottom: 5&#8217;-AATTAAAAACCTGAGACAGATCAGCAACAACTCGAGTTGTTGCTGATCTGTCTCAGG-3&#8217;</p><p>CRISPR targeting sequences (shown in Figure) were cloned into the pX330 vector (Addgene: #42230). To delete a region in the Exon 4 of the human p53 gene, a pair of CRISPR constructs was co-transfected into the cells together with an EGFP expression vector. EGFP-positive cells were selected by flow cytometry and plated at a single cell density. Colonies were picked, propagated, and genotyped by PCR.</p><p>We used the following antibodies for immunoblotting: RUNX2 (Cell Signaling, Cat:8486), p53 (DO1, Santa Cruz, Cat:sc-126), &#946;-actin (Sigma, Cat:A5316), Cleaved PARP (Promega, Cat: G7341), Tubulin (Sigma, Cat: T3526), CBFB (Cell Signaling, Cat: 12902s), and MYC (Epitomics, Cat: 1472&#8211;1).</p><p>We used the following antibodies for ChIP: RUNX2 (house made using recombinant RUNX2 fragments), Menin (Bethyl, Cat: A300-105A), CBFB (Bethyl, Cat: A303-549A), RNA polymerase II, N20 (Santa Cruz, Cat: sc-899), H3K4me3 (Abcam, Cat: ab8580-100), H3K79me2 (Abcam, Cat: ab3594-100).</p><p>MYC immunohistochemistry was performed with a mouse monoclonal antibody of MYC, 9E11 (Santa Cruz, sc-47694).</p><p>Mi-2 and Mi-3 were purchased from Selleckchem, and Mi-nc from Cayman Chemical. All the inhibitors were dissolved in DMSO to make 50 mM stock solution. Working concentrations were 50 uM.</p><p>We established a stable cell line for SAOS2 and Hu09-M112 cells by transducing them with the plenti6-GW-FLAG-RUNX2 vector and selecting transduced cells with 10 ug/ml of Blasticidin. After the stable cell line was established, cells from ten to twenty 10cm plates (around 60 million cells) were used to perform a FLAG pulldown. Briefly, cell pellets were lysed in 10ml of NET buffer (50 mM Tris, pH 7.5, 250 mM NaCl, 5 mM EDTA, 0.1% NP40, 10% Glycerol with freshly added protease inhibitors). Clear cell lysate was incubated with 100ul of anti-FLAG M2 affinity gel (200ul slurry) (A2220, Sigma) overnight at 4&#176;C. FLAG_RUNX2(h) bound anti-FLAG M2 affinity gel was washed three times with NET buffer (50mM Tris, pH 7.5, 250mM NaCl, 5mM EDTA, 1% NP40, 10% Glycerol, freshly added proteinase inhibitors) and eluted four times with 500ug/ml 3xFLAG peptide (F4799, Sigma). Combined eluted materials were concentrated with acetone and resolved by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) and silver staining (SilverQuest Staining Kit, LC6070, Invitrogen). Silver stained bands were cut out from the SDS-PAGE gel and analyzed by Mass Spectrometry.</p><p>Formalin fixed paraffin embedded slides were deparaffinized in Xylene, 100% ethanol and 95% ethanol. Antigens were retrieved by boiling slides in 10 mM sodium citrate for 10 minutes. After cooling, slides were treated with 3% H2O2 for 10 minutes followed by PBS+0.1% Tween 20 washing and blocked with serum. Slides were then incubated with 1:200 MYC antibody (Epitomics) for 1 hour at room temperature, washed three times with PBS, incubated with biotinylated goat anti-rabbit IgG secondary antibody (VECTASTAIN ABC Kit) for 1 hour at room temperature. After washing with biotin-avidin solution for 30 minutes at room temperature, slides were rinsed with PBS three times, DAB solution was added to allow color development for 2&#8211;5 minutes.</p><p>Eighty eight OS patient tumors were collected under an Institutional Review Board (IRB) approved protocol of the Montefiore Medical Center. A TMA was generated using these 88 tumors. Immunohistochemistry (IHC) procedures of using this TMA were approved under an exemption of IRB (#12806) by the Office of Human Subjects of Research (OHSR) at the National Institutes of Health.</p><p>Formalin fixed paraffin embedded slides were deparaffinized in Xylene, 100% ethanol and 95% ethanol. Masson&#8217;s trichrome staining was performed according to the instructions of the NovaUltra Masson Trichrome Stain Kit (IHCWorld, IW-3006). Blue color indicates collagen, black color the nuclei, and red color the cytoplasm.</p>