<p>The research described in this study was performed under the Animal Study Protocol LCBG-004, approved by the NCI Bethesda Animal Use and Care Committee. Animal euthanasia was performed by cervical dislocation after anesthesia by Avertin.</p><p>The mouse mammary carcinoma cell lines 4T1, 6DT1, Mvt-1 [<xref ref-type="bibr" rid="pgen.1005820.ref028">28</xref>] (provided by Dr. Lalage Wakefield) and human embryonic kidney HEK293 cells were cultured in Dulbecco&#8217;s Modified Eagle Medium (DMEM) (Gibco) supplemented with L-Glutamate (Gibco), 9% fetal bovine serum (FBS) (Gemini BioProducts), and 1% Penicillin and Streptomycin (P/S) (Gemini BioProducts).</p><p>Two milliliter suspensions of 10<sup>5</sup> cells were incubated at 37&#176;C in 5% CO<sub>2</sub> overnight. Cells were then infected with lentivirus suspension, and selected 30 hours post-infection with 5mg/mL blasticidin for over-expression (Invitrogen) constructs or 10ug/mL (shRNA) puromycin for shRNA constructs.</p><p>Co-immunoprecipitation was conducted as described in [<xref ref-type="bibr" rid="pgen.1005820.ref057">57</xref>] using mouse origin anti-FLAG and Protein G Dynabeads magnetic beads (Invitrogen).</p><p>Proliferation and wound healing assays were performed on the Incucyte ZOOM (Essen BioScience) system following the previously described protocols [<xref ref-type="bibr" rid="pgen.1005820.ref058">58</xref>]. For soft agar assays, 5,000 trypsinized cells were seeded in triplicate in 0.4% low-melting-point agarose (Sigma) on top of a 1% agarose layer and colonies enumerated 21 days later.</p><p>Array-based transcriptome profiling of <italic>Cnot7</italic>-knockdown and CNOT7-overexpressing 4T1 tumor cells was performed on Affymetrix GeneChip Mouse Gene 1.0 ST arrays by the Microarray Core in the NCI Laboratory of Molecular Technology.</p><p>Library preparation was performed using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina with NEBNext multiplexing oligos using manufacturer&#8217;s protocol. RNA sequencing was conducted on the Illumina HiSeq 2500.</p><p>RNA was isolated from tumors and cell lines using RNeasy kit (Qiagen) or TriPure (Roche) and reverse transcribed using iScript (Bio-Rad). Real-Time PCR was conducted using VeriQuest SYBR Green qPCR Master Mix (Affymetrix).</p><p>cDNA sequences of human FLAG-CNOT7, FLAG-CNOT7-D40A, and FLAG-CNOT7-E247A/Y260A were describe previously [<xref ref-type="bibr" rid="pgen.1005820.ref036">36</xref>]. Lentiviral expression vectors were produced with Multisite Gateway recombination. An entry clone using the murine Pol2 promoter was recombined with the cDNA entry clone and N-terminal entry clone encoding the MYC (EQKLISEEDL) or FLAG (DYKDDDDK) epitope tag into a Gateway destination vector pDest-658. pDest-658 is a modified version of the pFUGW lentiviral vector which contains the enhanced polypurine tract (PPT) and woodchuck regulatory element (WRE) to provide higher titer virus. It also contains an antibiotic resistance gene for blasticidin resistance. Entry clones were subcloned by Gateway Multisite LR recombination using the manufacturer&#8217;s protocols (Invitrogen). Expression clones were transformed into E. coli STBL3 cells to minimize unwanted LTR repeat recombination, and verified by agarose gel electrophoresis and restriction digest. Transfection-ready DNA for the final clones was prepared using the GenElute XP Maxiprep kit (Sigma). A control vector (8166-M24-658) was generated by standard Gateway LR recombination of a stuffer fragment made up of a non-coding DNA into the pLenti6-V5-DEST vector (Invitrogen). CNOT7 and CNOT7-D40A lentivector constructs were generated by the Protein Expression Laboratory and the Viral Technology Group in NCI, Frederick, MD.</p><p>Site directed mutagenesis was employed to generate CNOT7-M141R mutant using the primers listed in <xref ref-type="supplementary-material" rid="pgen.1005820.s011">S6 Table</xref>. The cDNA segment containing these mutations was subcloned into the wild type <italic>CNOT7</italic> lentivector by restriction digest and ligation.</p><p>4T1 cells expressing 8166-M24-658 control vector or FLAG-CNOT7 constructs were grown to ~85% confluence in ten 15cm culture plates. Cells were then rinsed with 15mL ice cold sterile PBS, scraped off, pelleted (~1mL pellet), and snap frozen in liquid nitrogen. Cells were thawed on ice and lysed in 3.5mL lysis buffer (100mM NaCl, 5mM MgCl<sub>2</sub>, 10mM HEPES (pH 7.3), 0.5% NP40, 200 units RNasin (Promega), Protease inhibitor cocktail (Roche)). The resulting 4mL of cell lysate was spun down twice at 21,000*g for 20 minutes at 4&#176;C. 40uL of lysate was saved 1% input to confirm immunoprecipitation. 200uL of lysate was saved for 5% RNA-seq input and was purified by TriPure (Roche) RNA extraction. 25uL beads per 1mL lysate of Protein G Dynabeads were blocked with 1mL 0.5% BSA in PBS at room temperature for 20 minutes then washed twice with 1mL NT2 wash buffer (50mM Tris-HCl (pH 7.4), 150mM NaCl, 1mM MgCl<sub>2</sub>, 0.05% NP40). 24ug antibody was added to each sample and incubated at 4&#176;C overnight then beads were added to each sample and rotated for 30 minutes at room temperature. Beads were washed four times in 1mL NT2 buffer. Eighty five percent of beads were subjected to RNA extraction with TriPure. Protein from the remaining 15% of beads was eluted with Laemli buffer to confirm immunoprecipitation.</p><p>Protein was extracted with Pierce lysis buffer, vigorously homogenized, and incubated on ice for twenty minutes. 20ug lysate per sample in NuPage LDS Sample Buffer and NuPage Reducing Agent (Invitrogen) was used for western blotting. PVDF membrane (Millipore) containing transferred proteins was incubated overnight in solution of 5% milk protein, tris-buffered saline supplemented with 0.05% Tween-20, and primary antibody. The membrane was then incubated with horse-radish peroxidase linked anti-mouse (GE Healthcare), anti-rat, or anti-rabbit (Santa Cruz Biotechnology) IgG secondary antibodies. Immunoblot was visualized using Amersham ECL Prime Western Blotting Detection System and Amersham Hyperfilm ECL (GE Healthcare). Rabbit origin anti-CNOT7 was a generous provided by G. Sebastiaan Winkler. Commercial antibodies used in this study include rabbit origin anti-TOB1 (GeneTex), rabbit origin anti-CNOT1 (Protein Tech), rat origin anti-HA (Roche), mouse origin monoclonal anti-FLAG (M2, Sigma).</p><p>Female FVB/NJ or Balb/cJ mice from Jackson Laboratories were injected at 6&#8211;8 weeks of age. Two days prior to orthotopic injections, cells were placed in non-selective media. On the day of injection, 1x10<sup>5</sup> cells were injected orthotopically into the fourth mammary fat pad of age-matched virgin females. After 30 days the mice were euthanized by intraperitoneal injection of 1mL Tribromoethanol with subsequent cervical dislocation. Primary tumors were resected, weighed, and snap frozen in liquid nitrogen. Lungs were resected, surface metastases were counted; lungs were inflated with 10% nitrate-buffered formalin and sent for sectioning and staining. For tail vein injection, 10<sup>5</sup> were injected into the lateral tail vein, mice were euthanized 22 days post-injection. All procedures were performed under the Animal Safety Proposal (LCBG-004) and approved by the NCI-Bethesda Animal Care and Use Committee.</p><p>Statistical analysis comparing two samples were conducted using the Mann-Whitney test on Prism Version 5.03 (GraphPad Software, La Jolla, CA). Multiple-comparison data was analyzed by Kruskal-Wallis test with post-hoc Conover-Inman correction for multiple analyses by R-script. Survival data was conducted with the Mantel-Cox test on Prism.</p><p>RBP motif sites were mapped on the 3&#8217;UTR regions of genome genes (27305 mRNA FASTA format) by using a Perl script. RBP motif site enrichment analysis was performed by the random sampling the genes from genome in the same number (the final overlapping gene number) and calculate the numbers of the gene with the RBP motif site and of the RBP motif site and repeat the sampling 1000 times. The p-value was estimated by one tail t-test. Differential gene expression analyses were done by t-test using the software package R.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Array-based gene expression and RIP-seq studies from this study have been submitted to the NCBI Gene Expression Omnibus (GEO; <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</ext-link> under the accession numbers GSE73296 (array) and GSE73366 (RIP-seq).</p>