<p>Oregon R or <italic>UAS-lacZ</italic>
<sup>RNAi</sup> was used as the wild type. <italic>Mtor</italic>
<sup>k03905</sup> was previously described [<xref ref-type="bibr" rid="pgen.1005750.ref046">46</xref>] and obtained from the Bloomington stock center. The P-element insertion in the <italic>Mtor</italic>
<sup>k03905</sup> resulted in a 9-base pair duplication, including 8 base pairs of upstream genomic sequence and a duplicated +1 residue and may represent a null mutation.</p><p>To generate transgenic strains of <italic>UAS-mad2</italic>, <italic>UAS-mps1</italic>, and <italic>UAS-Mtor</italic>, the corresponding full-length cDNAs of the <italic>Drosophila</italic> genes were amplified by PCR and inserted into the <italic>pUAST</italic> transformation vector [<xref ref-type="bibr" rid="pgen.1005750.ref057">57</xref>]. Second-chromosome <italic>UAS-Apc2</italic> transgenic flies were from David Roberts. <italic>UAS-DEFL</italic> (full-length <italic>shg</italic>) #6&#8211;3 was obtained from Kyoto stock center. <italic>UAS-Egfr</italic>
<sup>DN</sup> was obtained from the Bloomington stock center (BL5364).</p><p>All constructs were confirmed by DNA sequencing. The UAS constructs were injected into <italic>w</italic>
<sup>1118</sup> embryos using standard procedures.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">RNAi stocks used in this study: <italic>Mtor</italic>
<sup>RNAi-1</sup> (Vienna <italic>Drosophila</italic> RNAi Center-VDRC) Transformant ID 110218 [v110218]), <italic>Mtor</italic>
<sup>RNAi-2</sup> (BL32941), <italic>Mtor</italic>
<sup>RNAi-3</sup> (v24265), and <italic>rl</italic>
<sup>RNAi</sup> (v35641). The sequences used for VDRC knockdown strains are available for each line at <ext-link ext-link-type="uri" ns0:href="https://stockcenter.vdrc.at/"><underline>https://stockcenter.vdrc.at</underline></ext-link> and sequences for Bloomington knock-down strains are available for each line at <ext-link ext-link-type="uri" ns0:href="http://flystocks.bio.indiana.edu/"><underline>http://flystocks.bio.indiana.edu</underline></ext-link>.</p><p>The following Gal4 alleles were used to drive UAS lines: <italic>Nos-Gal4</italic> (<italic>nanos-Gal4VP16</italic>) [<xref ref-type="bibr" rid="pgen.1005750.ref058">58</xref>], obtained from the Bloomington stock center (BL4937), and <italic>upd-Gal4</italic> and <italic>c587-Gal4</italic>, provided by Ting Xie. <italic>esg-Gal4</italic> was obtained from Shigeo Hayashi. <italic>Collagen (Cg)-Gal4</italic> [<xref ref-type="bibr" rid="pgen.1005750.ref037">37</xref>], <italic>Dilp2-Gal4</italic> [<xref ref-type="bibr" rid="pgen.1005750.ref038">38</xref>], <italic>LE-Gal4</italic> [<xref ref-type="bibr" rid="pgen.1005750.ref039">39</xref>], <italic>pumpless (ppl)-Gal4</italic> [<xref ref-type="bibr" rid="pgen.1005750.ref040">40</xref>], <italic>brachyenteron (byn)-Gal4</italic> [<xref ref-type="bibr" rid="pgen.1005750.ref041">41</xref>], <italic>serpent</italic>
<sup><italic>hemo</italic></sup>
<italic>(srp</italic>
<sup><italic>hemo</italic></sup>
<italic>)-Gal4</italic> [<xref ref-type="bibr" rid="pgen.1005750.ref042">42</xref>], <italic>Aug21-Gal4</italic> [<xref ref-type="bibr" rid="pgen.1005750.ref043">43</xref>], <italic>hemolectin (hml)-Gal4</italic> [<xref ref-type="bibr" rid="pgen.1005750.ref044">44</xref>], and <italic>MS1096-Gal4</italic> [<xref ref-type="bibr" rid="pgen.1005750.ref045">45</xref>].</p><p>Flies were raised on standard fly food at 25&#176;C and at 65% humidity, unless otherwise indicated.</p><p>Clones of mutant GSCs were generated as previously described [<xref ref-type="bibr" rid="pgen.1005750.ref006">6</xref>]. To generate <italic>Mtor</italic>-mutant GSC clones, <italic>FRT</italic>
<sup>42D</sup>
<italic>+</italic> and <italic>FRT</italic>
<sup>42D</sup>
<italic>Mtor</italic>
<sup>k03905</sup>/<italic>Cyo</italic> virgin females were mated with males of genotype <italic>FRT</italic>
<sup>42D</sup>
<italic>arm-lacZ</italic>/<italic>Cyo</italic>; <italic>MKRS</italic>, <italic>hs-flp/+</italic>, or <italic>FRT</italic>
<sup>G13</sup>
<italic>Mtor</italic>
<sup>k03905</sup>/<italic>Cyo</italic> virgin females were mated with males of genotype <italic>FRT</italic>
<sup>G13</sup>
<italic>arm-lacZ</italic>/<italic>Cyo</italic>; <italic>MKRS</italic>, <italic>hs-flp/+</italic>. One- or 2-day-old adult males carrying an <italic>arm-lacZ</italic> transgene in <italic>trans</italic> to the mutant-bearing chromosome were heat shocked four times at 37&#176;C for 1 hr, at intervals of 8&#8211;12 hr. The males were transferred to fresh food every day at 25&#176;C. The testes were removed 1, 2, or 7 days after the first heat-shock treatment and processed for antibody staining.</p><p>To induce MARCM clones of <italic>FRT</italic>
<sup>42D</sup>
<italic>-piM</italic> (as a wild-type control) and <italic>FRT</italic>
<sup>42D</sup>
<italic>-Mtor</italic>
<sup>k03905</sup>, we generated the following flies: <italic>FRT</italic>
<sup>42D</sup>
<italic>tub-Gal80/FRT</italic>
<sup>42D</sup>
<italic>Mtor</italic>
<sup>k03905</sup> (or <italic>piM</italic>); <italic>MKRS</italic>, <italic>hs-flp/ tub-Gal4</italic>,<italic>UAS-mCD8</italic>.<italic>GFP</italic>. Three- or 4-day-old adult male flies were heat-shocked twice at 37&#176;C for 45 min, with an interval of 8&#8211;12 hr. The flies were transferred to fresh food daily after the final heat shock. The testes were removed at 1, 2, or 7 days after the first heat-shock treatment and processed for antibody staining.</p><p>Male <italic>UAS-RNAi</italic> transgene flies were crossed with female virgins of genotype <italic>Nos-Gal4</italic>, <italic>upd-Gal4</italic>, <italic>c587-Gal4; tub-Gal80</italic>
<sup>ts</sup> (<italic>c587</italic>
<sup>ts</sup>), or <italic>esg-Gal4</italic>, <italic>UAS-GFP; tub-Gal80</italic>
<sup>ts</sup> (<italic>esg</italic>
<sup>ts</sup>). The flies were cultured at 18&#176;C (Gal4 is inactive and Gal80 is active). Three- to 5-day-old adult flies with the appropriate genotype were transferred to new vials at 29&#176;C (Gal4 is active and Gal80 is inactive) for 3 or 7 days before dissection.</p><p>Normal immunofluorescence staining was performed as described previously with some modifications [<xref ref-type="bibr" rid="pgen.1005750.ref006">6</xref>]. Briefly, testes were dissected in phosphate-buffered saline (PBS), transferred to 4% formaldehyde in PBS, and fixed for 30 minutes. The testes were then washed in PBST (PBS containing 0.1% Triton X-100) for 3 times, 10 Minutes each time, then blocked with 5% goat serum in PBST for 1 hour. Samples were the incubated with primary antibody in PBST at 4&#176;C overnight. Samples were washed for 30 minutes (three 10-minute washes) in PBST, incubated with secondary antibody in PBST at room temperature for 2 hours, washed as above, and mounted in VECTASHIELD with DAPI (Vector Labs).</p><p>For the &#947;-tubulin staining, testes were dissected in PBS, transferred to 4% formaldehyde in PBS, and fixed for 20 minutes, followed by incubation with methanol for 10 minutes. Then washed for 10 minutes with PBST, and two 10-minutes washes with 5% goat serum PBST. Then incubated in primary antibody in 5% goat serum in PBST overnight at 4&#176;C. Then washed three times, 15 min each in PBST, followed with 2 hrs incubation with secondary antibody in 5% goat serum in PBST. Then washed for at least an hour with PBST, and mounted as above.</p><p>Caspase-3 activity was assessed using Live Green Caspase Detection Kits (I35106, Molecular Probes) according to standard protocol.</p><p>Confocal images were obtained by using a Zeiss LSM510 system, and were processed with Adobe Photoshop 7.0. GSCs were scored as Vasa-positive cells adjacent to the hub (detected using Fas3) and containing dot spectrosome (detected using 1B1). Only image with a clear view of the complete hub were used.</p><p>The following antisera were used: rabbit polyclonal anti-Vasa antibody (1:5000; gift from R. Lehmann), rabbit polyclonal anti-&#946;-Gal antibody (1:1000; Cappel), mouse monoclonal anti-&#946;-Gal antibody (1:100; Invitrogen), mouse monoclonal anti-Hts antibody 1B1 (1:4; Developmental Studies Hybridoma Bank [DSHB]), mouse monoclonal anti-Fas 3 antibody (1:10; DSHB), rat polyclonal anti-Tj (1:400; Li et al. [<xref ref-type="bibr" rid="pgen.1005750.ref059">59</xref>], mouse monoclonal anti-Eya (1:20, DSHB), mouse monoclonal anti-Dl (1:20; DSHB), mouse monoclonal anti-Pros (1:50; DSHB), rabbit polyclonal anti-GFP antibody (1:200; Molecular Probes), mouse monoclonal anti-GFP antibody (1:100; Invitrogen), rabbit polyclonal anti-Caspase 3 antibody (1:1000; gift from B. Hay); rabbit polyclonal anti-Thr3-phosphorylated histone H3 antibody (1:200; upstate), mouse monoclonal anti-&#945;-tubulin antibody (1:100; Sigma), mouse monoclonal anti-&#947;-tubulin antibody (1:100; Sigma), guinea pig polyclonal anti-Zfh1 (1:2000; gift from J. Skeath), mouse monoclonal anti-Mtor antibody (1:100; gift from K. Johansen), rabbit polyclonal anti-Apc2 antibody (1:5000; gift from M. Bienz). Secondary antibodies were goat anti-mouse, goat anti-guinea pig, and goat anti-rabbit IgG conjugated to Alexa 488 or Alexa 568 (1:400; Molecular Probes). DAPI (Molecular Probes) was used to stain DNA.</p><p>We scored the centrosome misorientation and spindle misorientation following the protocol described by Yamashita et al. [<xref ref-type="bibr" rid="pgen.1005750.ref019">19</xref>, <xref ref-type="bibr" rid="pgen.1005750.ref060">60</xref>]. Specifically, centrosome misorientation was noted when neither of two centrosomes were closely associated with hub-GSC interface during interphase and at mitosis. Spindle misorientation was scored when neither of the two spindle poles was closely associated with hub-GSC interface during mitosis [<xref ref-type="bibr" rid="pgen.1005750.ref019">19</xref>, <xref ref-type="bibr" rid="pgen.1005750.ref060">60</xref>].</p><p>Statistical analyses were performed using Microsoft Excel 2010 or GraphPad Prism 6 software. Data are shown as means &#177; SD or standard error of the mean (SEM). P-values were obtained between two groups using the Student&#8217;s t-test or between more than two groups by analysis of variance (ANOVA).</p>