<p xmlns:ns0="http://www.w3.org/1999/xlink">Throughout this paper, we shall use <inline-formula><inline-graphic ns0:href="pgen.1004787.e165.jpg" /></inline-formula> to index SNPs, <inline-formula><inline-graphic ns0:href="pgen.1004787.e166.jpg" /></inline-formula> to index GWAS data sets, and <inline-formula><inline-graphic ns0:href="pgen.1004787.e167.jpg" /></inline-formula> to index the annotation data sets. We first consider the simplest case where we only have summary statistics (<italic>p</italic>-values) from just one GWAS data set, and then extend our model to handle multiple GWAS data sets and annotation data. Suppose we have performed hypothesis testing of genome-wide SNPs and obtained their <italic>p</italic>-values: <disp-formula id="pgen.1004787.e168"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e168.jpg" /><label>(3)</label></disp-formula>where <inline-formula><inline-graphic ns0:href="pgen.1004787.e169.jpg" /></inline-formula> is the number of SNPs. Consider the &#8220;two-groups model&#8221; <xref ref-type="bibr" rid="pgen.1004787-Efron1">[46]</xref>, i.e., the obtained <italic>p</italic>-values are assumed to come from the mixture of null and non-null, with probability <inline-formula><inline-graphic ns0:href="pgen.1004787.e170.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1004787.e171.jpg" /></inline-formula>, respectively. Let <inline-formula><inline-graphic ns0:href="pgen.1004787.e172.jpg" /></inline-formula> be the latent variables indicating whether the <italic>j</italic>-th SNP is null or non-null, where <inline-formula><inline-graphic ns0:href="pgen.1004787.e173.jpg" /></inline-formula>, <inline-formula><inline-graphic ns0:href="pgen.1004787.e174.jpg" /></inline-formula>, and <inline-formula><inline-graphic ns0:href="pgen.1004787.e175.jpg" /></inline-formula>, because a SNP can only be either null or non-null. Here <inline-formula><inline-graphic ns0:href="pgen.1004787.e176.jpg" /></inline-formula> means un-associated (null) and <inline-formula><inline-graphic ns0:href="pgen.1004787.e177.jpg" /></inline-formula> means associated (non-null). Then we have the following two-groups model: <disp-formula id="pgen.1004787.e178"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e178.jpg" /><label>(4)</label></disp-formula>where the <italic>p</italic>-values from the null group are from the Uniform distribution on [0,1], denoted as <inline-formula><inline-graphic ns0:href="pgen.1004787.e179.jpg" /></inline-formula>, and the <italic>p</italic>-values from the non-null group are from the Beta distribution with parameters (<inline-formula><inline-graphic ns0:href="pgen.1004787.e180.jpg" /></inline-formula>), where <inline-formula><inline-graphic ns0:href="pgen.1004787.e181.jpg" /></inline-formula>. We put the constraint <inline-formula><inline-graphic ns0:href="pgen.1004787.e182.jpg" /></inline-formula> to model that a smaller <italic>p</italic>-value is more likely than a larger <italic>p</italic>-value when it is from the non-null group <xref ref-type="bibr" rid="pgen.1004787-Pounds1">[47]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To incorporate information from functional annotation data, we extend the basic model as follows. Suppose we have collected information from <inline-formula><inline-graphic ns0:href="pgen.1004787.e183.jpg" /></inline-formula> functional annotation sources in the annotation matrix: <inline-formula><inline-graphic ns0:href="pgen.1004787.e184.jpg" /></inline-formula>, where <inline-formula><inline-graphic ns0:href="pgen.1004787.e185.jpg" /></inline-formula> indicates whether the <italic>j</italic>-th SNP is annotated in the <italic>d</italic>-th functional annotation source. For example, when there are two annotation sources &#8211; eQTL data and DNase I hypersensitivity sites (DHS) data &#8211; then <inline-formula><inline-graphic ns0:href="pgen.1004787.e186.jpg" /></inline-formula> is an <inline-formula><inline-graphic ns0:href="pgen.1004787.e187.jpg" /></inline-formula> matrix. If the <italic>j</italic>-th SNP is an eQTL, then <inline-formula><inline-graphic ns0:href="pgen.1004787.e188.jpg" /></inline-formula>, otherwise <inline-formula><inline-graphic ns0:href="pgen.1004787.e189.jpg" /></inline-formula>; if it is located in a DHS, then <inline-formula><inline-graphic ns0:href="pgen.1004787.e190.jpg" /></inline-formula>, otherwise <inline-formula><inline-graphic ns0:href="pgen.1004787.e191.jpg" /></inline-formula>. Now we model the relationship between <inline-formula><inline-graphic ns0:href="pgen.1004787.e192.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1004787.e193.jpg" /></inline-formula> as <disp-formula id="pgen.1004787.e194"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e194.jpg" /><label>(5)</label></disp-formula>Clearly, <inline-formula><inline-graphic ns0:href="pgen.1004787.e195.jpg" /></inline-formula> can be interpreted as the proportion of null SNPs being annotated in the <italic>d</italic>-th annotation, and <inline-formula><inline-graphic ns0:href="pgen.1004787.e196.jpg" /></inline-formula> corresponds to the proportion of non-null SNPs being annotated in the <italic>d</italic>-th annotation. Therefore, <inline-formula><inline-graphic ns0:href="pgen.1004787.e197.jpg" /></inline-formula> implies that there exists enrichment for the <italic>d</italic>-th annotation. The statistical inference about enrichment of annotation data will be discussed in details in Section &#8220;Hypothesis testing of annotation enrichment and pleiotropy&#8221;.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Now we extend the above model to handle multiple GWAS data sets. To keep the notation uncluttered, we present the model for the case of two GWAS data sets. Suppose we have <italic>p</italic>-values from two GWAS: <disp-formula id="pgen.1004787.e198"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e198.jpg" /><label>(6)</label></disp-formula>Let <inline-formula><inline-graphic ns0:href="pgen.1004787.e199.jpg" /></inline-formula> be the matrix collecting all the <italic>p</italic>-values, where <inline-formula><inline-graphic ns0:href="pgen.1004787.e200.jpg" /></inline-formula> denotes the <italic>p</italic>-value of the <italic>j</italic>-th SNP in the <italic>k</italic>-th GWAS. Similarly, we introduce latent variables <inline-formula><inline-graphic ns0:href="pgen.1004787.e201.jpg" /></inline-formula> indicating the association between the <italic>j</italic>-th SNP and the two phenotypes: <inline-formula><inline-graphic ns0:href="pgen.1004787.e202.jpg" /></inline-formula> means the <italic>j</italic>-th SNP is associated with neither of them, <inline-formula><inline-graphic ns0:href="pgen.1004787.e203.jpg" /></inline-formula> means it is only associated with the first one, <inline-formula><inline-graphic ns0:href="pgen.1004787.e204.jpg" /></inline-formula> means it is only associated with the second one, and <inline-formula><inline-graphic ns0:href="pgen.1004787.e205.jpg" /></inline-formula> means it is associated with both. The two-groups model (4) is extended to the following &#8220;four-groups model&#8221;: <disp-formula id="pgen.1004787.e206"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e206.jpg" /><label>(7)</label></disp-formula>where <inline-formula><inline-graphic ns0:href="pgen.1004787.e207.jpg" /></inline-formula>. When the genetic bases of the two phenotypes are independent of each other (i.e., no pleiotropy), then we have <inline-formula><inline-graphic ns0:href="pgen.1004787.e208.jpg" /></inline-formula> by expectation. Therefore, the difference between <inline-formula><inline-graphic ns0:href="pgen.1004787.e209.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1004787.e210.jpg" /></inline-formula> can be used to characterize pleiotropy. Statistical inference on pleiotropy is given in Section &#8220;Hypothesis testing of annotation enrichment and pleiotropy&#8221;.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To incorporate annotation information into the multiple GWAS model (7), similarly, we model the relationship between <inline-formula><inline-graphic ns0:href="pgen.1004787.e211.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1004787.e212.jpg" /></inline-formula> as <disp-formula id="pgen.1004787.e213"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e213.jpg" /><label>(8)</label></disp-formula>where <inline-formula><inline-graphic ns0:href="pgen.1004787.e214.jpg" /></inline-formula> is the probability of a null SNP being annotated, <inline-formula><inline-graphic ns0:href="pgen.1004787.e215.jpg" /></inline-formula> is the probability of the first phenotype associated-SNP being annotated, <inline-formula><inline-graphic ns0:href="pgen.1004787.e216.jpg" /></inline-formula> is the probability of the second phenotype associated-SNP being annotated, and <inline-formula><inline-graphic ns0:href="pgen.1004787.e217.jpg" /></inline-formula> is the probability of jointly associated-SNP being annotated. Assuming the independence of SNP markers, the joint distribution <inline-formula><inline-graphic ns0:href="pgen.1004787.e218.jpg" /></inline-formula> can be written as <disp-formula id="pgen.1004787.e219"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e219.jpg" /><label>(9)</label></disp-formula>where <inline-formula><inline-graphic ns0:href="pgen.1004787.e220.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1004787.e221.jpg" /></inline-formula> are the <italic>j</italic>-th row of <inline-formula><inline-graphic ns0:href="pgen.1004787.e222.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1004787.e223.jpg" /></inline-formula>; the second equation holds by assuming the independence between <inline-formula><inline-graphic ns0:href="pgen.1004787.e224.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1004787.e225.jpg" /></inline-formula>, conditional on <inline-formula><inline-graphic ns0:href="pgen.1004787.e226.jpg" /></inline-formula>; and the third equation holds by further assuming the independence between <inline-formula><inline-graphic ns0:href="pgen.1004787.e227.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1004787.e228.jpg" /></inline-formula> for <inline-formula><inline-graphic ns0:href="pgen.1004787.e229.jpg" /></inline-formula>, conditional on <inline-formula><inline-graphic ns0:href="pgen.1004787.e230.jpg" /></inline-formula>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Parameters in the GPA model can be estimated using the Expectation-Maximization (EM) algorithm <xref ref-type="bibr" rid="pgen.1004787-Dempster1">[48]</xref>, which is computationally efficient because we have explicit solutions for estimation of all the parameters in the M-step. Standard errors for parameter estimates can be approximated using the empirical observed information matrix <xref ref-type="bibr" rid="pgen.1004787-McLachlan1">[49]</xref>. Note that in the GPA model, the sample size for estimating the empirical observed information matrix corresponds to the number of SNPs and as a result, we have a very large sample size (<inline-formula><inline-graphic ns0:href="pgen.1004787.e231.jpg" /></inline-formula>) to estimate standard errors accurately. More details of the EM algorithm and the estimation of standard errors are provided in Sections 1 and 3 in <xref ref-type="supplementary-material" rid="pgen.1004787.s001">Text S1</xref>.</p><p>After we estimate the parameters in the GPA model, SNPs can be prioritized based on their local false discovery rates <xref ref-type="bibr" rid="pgen.1004787-Efron2">[50]</xref>. Note that separate analysis of single GWAS data based on their summary statistics is equivalent to the analysis of single GWAS data using GPA without any annotation data. Hence, separate analysis of single GWAS data can be considered as a special case of our GPA approach.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To present the local false discovery rate based on our GPA model, we begin with the simplest case: single GWAS without annotation data. In this case, there are only two groups: null and non-null. The false discovery rate is defined as the probability that the <italic>j</italic>-th SNP belongs to the null group given its <italic>p</italic>-value, i.e., <disp-formula id="pgen.1004787.e232"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e232.jpg" /><label>(10)</label></disp-formula>For joint analysis of two GWAS data sets, we are interested in the local false discovery rate of the <italic>j</italic>-th SNP, if it is claimed to be associated with the first phenotype, the second one, and both, i.e., <disp-formula id="pgen.1004787.e233"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e233.jpg" /><label>(11)</label></disp-formula>Similarly, when annotation data are available, the false discovery rates can be calculated as <disp-formula id="pgen.1004787.e234"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e234.jpg" /><label>(12)</label></disp-formula>Then, we use the <italic>direct posterior probability approach</italic>
<xref ref-type="bibr" rid="pgen.1004787-Newton1">[51]</xref> to control global false discovery rates. More details for the estimation of false discovery rates are provided in Section 2 in <xref ref-type="supplementary-material" rid="pgen.1004787.s001">Text S1</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Given an annotation file, we may be interested in whether there is statistical evidence for enrichment of GWAS signals in this annotation file. We propose to use the likelihood ratio test (LRT) to assess the statistical significance. To keep the notation simple, we drop the index of annotation files. Specifically, the significance of enrichment of an annotation for single-GWAS data analysis can be assessed by the following test: <disp-formula id="pgen.1004787.e235"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e235.jpg" /><label>(13)</label></disp-formula>The LRT statistic is: <disp-formula id="pgen.1004787.e236"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e236.jpg" /></disp-formula>where <inline-formula><inline-graphic ns0:href="pgen.1004787.e237.jpg" /></inline-formula> collects the parameter estimates obtained under <inline-formula><inline-graphic ns0:href="pgen.1004787.e238.jpg" /></inline-formula>, and the superscript <italic>A</italic> indicates the Annotation enrichment test. Note that <inline-formula><inline-graphic ns0:href="pgen.1004787.e239.jpg" /></inline-formula> can be easily obtained by running the GPA algorithm without incorporating the annotation data. Under the null, the test statistic <inline-formula><inline-graphic ns0:href="pgen.1004787.e240.jpg" /></inline-formula> asymptotically follows the <inline-formula><inline-graphic ns0:href="pgen.1004787.e241.jpg" /></inline-formula> distribution with 1 degree of freedom <xref ref-type="bibr" rid="pgen.1004787-Shao1">[52]</xref>. We reject <inline-formula><inline-graphic ns0:href="pgen.1004787.e242.jpg" /></inline-formula> if <inline-formula><inline-graphic ns0:href="pgen.1004787.e243.jpg" /></inline-formula>, where <inline-formula><inline-graphic ns0:href="pgen.1004787.e244.jpg" /></inline-formula> is the <inline-formula><inline-graphic ns0:href="pgen.1004787.e245.jpg" /></inline-formula>-th quantile of <inline-formula><inline-graphic ns0:href="pgen.1004787.e246.jpg" /></inline-formula> distribution with <inline-formula><inline-graphic ns0:href="pgen.1004787.e247.jpg" /></inline-formula>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For joint analysis of two GWAS with annotation data, test (13) becomes <disp-formula id="pgen.1004787.e248"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e248.jpg" /><label>(14)</label></disp-formula>Under the null, the test statistic asymptotically follows a <inline-formula><inline-graphic ns0:href="pgen.1004787.e249.jpg" /></inline-formula> distribution with <inline-formula><inline-graphic ns0:href="pgen.1004787.e250.jpg" /></inline-formula>. Similarly, the test of annotation enrichment can be extended to handle <inline-formula><inline-graphic ns0:href="pgen.1004787.e251.jpg" /></inline-formula> GWAS. In this case, the test statistic asymptotically follows <inline-formula><inline-graphic ns0:href="pgen.1004787.e252.jpg" /></inline-formula> distribution with <inline-formula><inline-graphic ns0:href="pgen.1004787.e253.jpg" /></inline-formula> under the null.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Now we consider testing pleiotropy between two GWAS. When there is no pleiotropy, i.e., the signals from the two GWAS are independent of each other, testing pleiotropy can be formulated by testing the following hypothesis: <disp-formula id="pgen.1004787.e254"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e254.jpg" /><label>(15)</label></disp-formula>where <inline-formula><inline-graphic ns0:href="pgen.1004787.e255.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1004787.e256.jpg" /></inline-formula>. The LRT statistic is constructed as follows: <disp-formula id="pgen.1004787.e257"><graphic orientation="portrait" position="anchor" ns0:href="pgen.1004787.e257.jpg" /></disp-formula>where <inline-formula><inline-graphic ns0:href="pgen.1004787.e258.jpg" /></inline-formula> represents the parameter estimates obtained under <inline-formula><inline-graphic ns0:href="pgen.1004787.e259.jpg" /></inline-formula>, and the superscript <italic>P</italic> indicates the Pleiotropy test. The test statistic (<inline-formula><inline-graphic ns0:href="pgen.1004787.e260.jpg" /></inline-formula>) asymptotically follows <inline-formula><inline-graphic ns0:href="pgen.1004787.e261.jpg" /></inline-formula> distribution with <inline-formula><inline-graphic ns0:href="pgen.1004787.e262.jpg" /></inline-formula> under the null.</p>