<p>Gene Expression Omnibus dataset GSE38196, first described in [<xref ref-type="bibr" rid="pgen.1006133.ref043">43</xref>], was used to identify genes upregulated independently of <italic>atfs-1</italic> following mitochondrial dysfunction (<italic>spg-7</italic> RNAi). Full details of our procedure to isolate <italic>atfs-1</italic> independent genes from this dataset is provided in <xref ref-type="supplementary-material" rid="pgen.1006133.s018">S1 Text</xref>. The MEME Suite of tools (v4.10.1) [<xref ref-type="bibr" rid="pgen.1006133.ref120">120</xref>] was used to identify enriched DNA elements (ungapped) among the promoter regions of the identified gene subset. We limited our search to 400 bp of the most proximal 5&#8217; sequence of each gene. MAST [<xref ref-type="bibr" rid="pgen.1006133.ref121">121</xref>], was used to locate DAF-16 binding sites using a weighted matrix based on the consensus identified by Kumar and colleagues [<xref ref-type="bibr" rid="pgen.1006133.ref051">51</xref>].</p><p>A complete list of <italic>C</italic>. <italic>elegans</italic> strains used in this study is provided in <xref ref-type="supplementary-material" rid="pgen.1006133.s017">S6 Table</xref>. All strains were maintained at 20&#176;C on standard NGM-agar plates [<xref ref-type="bibr" rid="pgen.1006133.ref020">20</xref>].</p><p>Recombinant array construction, microinjection procedures and choice of strain background are detailed in <xref ref-type="supplementary-material" rid="pgen.1006133.s012">S1 Text</xref> and <xref ref-type="supplementary-material" rid="pgen.1006133.s017">S6 Table</xref>.</p><p>Feeding RNAi and RNAi dilution studies were performed as previously described [<xref ref-type="bibr" rid="pgen.1006133.ref020">20</xref>]. Details regarding either the source or construction of feeding RNAi constructs is provided in Supplemental Experimental Procedures (<xref ref-type="supplementary-material" rid="pgen.1006133.s018">S1 Text</xref>).</p><p>Images of first day adult worms were captured using an Olympus DP71 CCD camera connected to an Olympus SZX16 fluorescence dissecting microscope. Where relevant, images were quantified using ImageJ software (NIH). A one-way ANOVA, or Student&#8217;s t-test with correction applied for multiple testing was employed, as indicated in figure legends.</p><p>qRT-PCR was used to measure the efficacy of <italic>atfs-1</italic> RNAi knockdown in <xref ref-type="fig" rid="pgen.1006133.g004">Fig 4C</xref>. Details of strain culturing, mRNA extraction, cDNA synthesis, primer design for qRT-PCR analysis, data normalization and statistical testing are provided in <xref ref-type="supplementary-material" rid="pgen.1006133.s018">S1 Text</xref>.</p><p>Lifespan studies were performed as described previously [<xref ref-type="bibr" rid="pgen.1006133.ref020">20</xref>]. Use of FudR was avoided. The first day of adulthood was designated as day one. Data was analyzed using the log rank test and Cox&#8217;s proportional hazard model. A full description of all lifespan experiments is provided in <xref ref-type="supplementary-material" rid="pgen.1006133.s015">S4 Table</xref>. Raw lifespan data is provided in <xref ref-type="supplementary-material" rid="pgen.1006133.s016">S5 Table</xref>.</p>