<p>This study was approved by the NICHD IRB (Protocol #04-CH-0077). Cells from Proband 3 were obtained as part of the diagnostic process. Written, informed consent was obtained from all subjects or their respective guardians.</p><p>Proband and three independent normal control primary fibroblast (FB) cell lines were cultured at 37&#176;C and 5% CO<sub>2</sub> in DMEM with GlutaMAX (Life Technologies) containing 10% fetal bovine serum and 1% pen-strep. Primary osteoblast (OB) cultures were established from surgical bone chips of normal control and Proband 2. Following digestion of bone chips for 2 h at 37&#176;C with 0.3 units/ml collagenase P in serum-free &#945;MEM, osteoblasts were cultured in &#945;MEM supplemented with 10% fetal bovine serum and 1% pen-strep at 37&#176;C and 8% CO<sub>2</sub> [<xref ref-type="bibr" rid="pgen.1006156.ref060">60</xref>]. For generation of conditioned media, cells were cultured in media containing 0.1% fetal bovine serum and 100 &#956;g/ml ascorbate for 24 h in the presence or absence of 50 &#956;M MG132 (Sigma-Aldrich). For ER stress experiments, cells were cultured in media supplemented with 100 &#956;g/ml ascorbate with or without N-[N-(N-Acetyl-L-leucyl)-L-leucyl]-L-norleucine (ALLN, Sigma-Aldrich) for 8 h.</p><p>Total RNA was extracted from fibroblast and osteoblast cultures in the presence or absence of emetine (100 &#956;g/ml) using TriReagent (Molecular Research Center) according to the manufacturer&#8217;s protocol. Total RNA was treated with DNA-free (Life Technologies) prior to further analysis. Since the majority of cases of OI are caused by dominant mutations in <italic>COL1A1</italic> and <italic>COL1A2</italic>, molecular screening for a causative defect began with these genes. No sequence variants, however, were detected in cDNA obtained from proband fibroblast cultures.</p><p>Genomic DNA (gDNA) was prepared from fibroblasts or whole blood using the Puregene DNA extraction kit (Qiagen). Additional molecular screening was performed on proband genomic DNA utilizing Next Generation Sequencing (NGS) covering a panel of genes associated with autosomal dominant and recessive forms of OI (<italic>ALPL</italic>, <italic>BMP1</italic>, <italic>COL1A1</italic>, <italic>COL1A2</italic>, <italic>CRTAP</italic>, <italic>FKBP10</italic>, <italic>IFITM5</italic>, <italic>LEPRE1</italic>, <italic>PLOD2</italic>, <italic>PPIB</italic>, <italic>SERPINF1</italic>, <italic>SERPINH1</italic>, <italic>SP7</italic>, <italic>TMEM38B</italic> and <italic>WNT1</italic>; Connective Tissue Gene Tests).</p><p>For Proband 1, fibroblast gDNA (500 ng) was used to amplify the region surrounding the putative deletion inclusive of exon 4 in <italic>TMEM38B</italic> (NG_032971.1), which was previously characterized by Volodarsky <italic>et al</italic> [<xref ref-type="bibr" rid="pgen.1006156.ref021">21</xref>]. A fully nested PCR was performed using 15 pmol primers flanking exon 4, which are specific to the normal allele (WT-S, 5&#8217;-TGG TGA AAG GGA AGA ATT GC-3&#8217;; WT-AS, 5&#8217;-GAG AAA CCC ACA GAG AAG GAA A-3&#8217;), and primers flanking the deletion (Exon 3s, 5&#8217;-CCG CAT GAC CTA GTT TCC CAG GGC TAT TCA-3&#8217;; Intron 4as, 5&#8217;-CAA AGC AGG ATG AGG TTT GGA CAA CAG ACC-3&#8217;), 1% Perfect Match (Agilent Technologies), 3% DMSO, and 1.0 U High Fidelity Platinum Taq polymerase (Life Technologies). After an initial 5 minute denaturation at 94&#176;C, reactions were incubated for 35 cycles of 94&#176;C for 1 min, 51&#176;C for 30 sec and 72&#176;C for 1 min, followed by 7 min at 72&#176;C. PCR products were then sequenced (Macrogen). Genomic DNA was unavailable for further testing of the parents or siblings.</p><p>For Proband 2, NGS (Connective Tissue Gene Tests) identified only a single mutant allele, which was verified by Sanger sequencing. Genomic DNA from the proband and his family members was used to amplify the region surrounding exon 1 of TMEM38B by PCR. Reactions contained 15 pmol primers (5&#8217;UTRs, 5&#8217;-TCT CCT ACT CCT CAC CGC-3&#8217;; Intron 1as, 5&#8217;-CGA CTT AGA CGG TCC TCG-3&#8217;) and 1.25 U Amplitaq Gold (Life Technologies). Following an initial 10 minute denaturation at 95&#176;C, reactions were incubated for 40 cycles of 94&#176;C for 30 sec, 54&#176;C for 30 sec and 72&#176;C for 30 sec, followed by 7 min at 72&#176;C. PCR products were then sequenced (Macrogen). The second mutant allele in Family 2 was detected by high-density targeted (HDT) array, which suggested copy number variation (CNV) in the first two exons of <italic>TMEM38B</italic> for the proband, his sibling and father (Connective Tissue Gene Tests). Based on the molecular findings reported by Rubinato <italic>et al</italic> [<xref ref-type="bibr" rid="pgen.1006156.ref022">22</xref>], heminested primers were designed to amplify the region encompassing a 35 Kb region encompassing exons 1 and 2 of <italic>TMEM38B</italic>. PCR amplification, using 15 pmol primers (5&#8217;BP-S, 5&#8217;-GCA GCC TCA ACC TCC TGG GCT CAG ATG ATT-3&#8217;; 5&#8217;WT-AS, TAG GCC TGG CTG GTG GCT CAT GCC TGT AAT-3&#8217;; 3BP-AS, 5&#8217;-GCT TTT GAA ACT GAC CCA CCA GGG CTC TCT-3&#8217;) and the same conditions described above, generated 216 bp and 339 bp fragments for the mutant and normal alleles, respectively.</p><p>Proband 3 genomic DNA was amplified using 3% DMSO, 1.0 U High Fidelity Platinum Taq polymerase (Life Technologies), and 15 pmol primers encompassing exon 4 of <italic>TMEM38B</italic> (INT3-S, 5&#8217;-TTG ACA TGC TGG TGA AAG GGA AGA ATT GCA-3&#8217;; INT4-AS, 5&#8217;-CAG CTA TAT TCC ACC CAC ATT TAA AGT CCC-3&#8217;). An initial 5 minute denaturation at 94&#176;C was followed by 35 cycles of 94&#176;C for 1 min, 60&#176;C for 30 sec and 72&#176;C for 1 min, followed by 7 min at 72&#176;C. Amplification products were then sequenced (Macrogen).</p><p>Gene transcript levels were quantitated by real-time RT-PCR following reverse-transcription using a High Capacity cDNA Archive Kit and Taqman Assays on Demand (Life Technologies, <italic>TMEM38A</italic>, Hs00225325_m1; <italic>TMEM38B</italic>, Hs00216531_m1; <italic>COL1A1</italic>, Hs00164004_m1; <italic>FKBP10</italic>, Hs01000263_m1; <italic>PLOD1</italic>, Hs00609368_m1; <italic>PLOD2</italic>, Hs01118190_m1; <italic>PLOD3</italic>, Hs01126617_m1; <italic>P4HB/PDI</italic>, Hs00168586_m1; <italic>HSPA5</italic>, Hs99999174_m1; <italic>PPIB</italic>, Hs00168719_m1; <italic>ATP2A/SERCA2</italic>, Hs00544877_m1; <italic>ITPR1</italic>, Hs00181881_m1; <italic>RYR1</italic>, Hs00166991_m1; <italic>GAPDH</italic>, Hs99999905_m1; <italic>ACTB</italic>, Hs99999903_m1; <italic>B2M</italic>, Hs99999907_m1). Relative expression of genes of interest was measured in triplicate in 3 independent cultures against 3 reference genes (<italic>ACTB</italic>, <italic>B2M</italic> and <italic>GAPDH</italic>). To validate gene expression levels in normal control fibroblast cell lines, transcript levels were determined independently for each control cell line and compared to cDNA derived from pooled RNA samples (<xref ref-type="supplementary-material" rid="pgen.1006156.s008">S8 Fig</xref>). Gene expression in proband fibroblasts was then quantified relative to pooled normal control fibroblast values. Gene transcripts in proband osteoblasts are expressed relative to a single normal control primary osteoblast cell line.</p><p>Identification of <italic>TMEM38B</italic> splice variants in P1 mRNAwas performed as follows: DNase-treated total RNA (1 &#956;g) was reverse transcribed using an antisense primer (5&#8217;-AAC GGC TGC TGC CAG CCA AAT AGC ATC CAA-3&#8217;) complementary to exon 6 of the <italic>TMEM38B</italic> transcript (NM_018112.2) and murine leukemia virus (MuLV) reverse transcriptase (Life Technologies). Subsequent PCR used the same antisense primer and a sense primer corresponding to exon 2 sequence (5&#8217;-GGA GCA GCT GCA TTG GCA TGG AAG AAT CCT-3&#8217;). Reaction products were electrophoresed on 1.5% agarose gels and visualized with ethidium bromide. Alternatively, spliced transcripts were purified by extraction of individual gel bands using the QIAquick gel extraction kit (Qiagen), followed by cloning into plasmids (TOPO TA cloning kit, Life Technologies) and sequencing using vector-specific primers. To quantitate splice products, RT-PCR reactions were repeated in the presence of 2.5 &#956;Ci of 111TBq/mmol [&#945;-<sup>32</sup>P] dCTP for 20 or 25 cycles. PCR products were electrophoresed on 6% TBE gels and individual bands were quantitated by densitometry of autoradiographs.</p><p>For <italic>XBP1</italic> splicing, 1 &#956;g of DNase-treated total RNA was reverse-transcribed using the SuperScript One-Step RT-PCR System (Life Technologies). The resulting cDNA was used as a template for PCR amplification across the region of the <italic>XBP1</italic> cDNA containing the intronic target of IRE1&#945; ribonuclease activity using 15 &#956;M sense (5&#8217;-TCA GCT TTT ACG AGA GAA AAC TCA TGG CCT-3&#8217;) and antisense primers (5&#8217;-AGA ACA TGA CTG GGT CCA AGT TGT CCA GAA-3&#8217;). Following a 30 min incubation at 50&#176;C, reactions were cycled 30 times at 94&#176;C, 60&#176;C and 72&#176;C for 30 sec at each temperature. Reaction products were electrophoresed on 6% TBE gels (Life Technologies) and visualized by ethidium bromide staining.</p><p>Fibroblast cultures were lysed in RIPA buffer (150 mM NaCl, 1% NP-40, 0.5% Na-deoxycholate, 0.1% SDS, 50 mM Tris, pH 7.4) supplemented with protease inhibitor cocktail (Sigma-Aldrich). Protein concentration was determined using the BCA Protein Assay Kit (Thermo Scientific). Samples (20 &#956;g protein) were subjected to SDS-PAGE on 4&#8211;15% acrylamide gels (BioRad) under denaturing conditions and transferred to nitrocellulose membranes. The membranes were blocked overnight in 5% non-fat milk in TBST. After washing in TBST, membranes were incubated overnight at 4&#176;C in TBST containing 2.5% non-fat milk and primary antibody (diluted 1:500&#8211;1000). After washing in 1X TBST, membranes were incubated with the corresponding IRDye infrared secondary antibody (diluted 1:10,000) (LI-COR Biosciences). Proteins were visualized using an Odyssey Infrared Imaging System (LI-COR Biosciences) and quantitated using the Odyssey 3.0.30 software with normalization to actin levels. The primary antibodies used include rabbit anti-TMEM38B (Thermo Fisher; PA5-20859), mouse anti-TMEM38A (Abnova; H00079041-B01P), rabbit anti-IP3R1 (Novus Biologicals; NB120-5908), rabbit anti-SERCA2 (Novus Biologicals; NBP1-59203), rabbit anti-GRP78 (Abcam; ab32618), mouse anti-FKBP65 (Abnova; H00060681-M13), rabbit anti-cyclophilin B/PPIB (Abcam; ab16045), rabbit anti-PLOD1 (Abgent; AP12656c), rabbit anti-PLOD2 (Proteintech; 21214-1-AP), rabbit anti-ATF4 (Proteintech; 10835-1-AP), mouse anti-ATF6 (Active Motif; 40962), and mouse anti-PDI (Novus Biologicals; H00005034-B01P). Rabbit anti-&#945;1(I) C-telopeptide (LF68) was a generous gift from Larry Fisher, NIDCR, NIH. Rabbit (Santa Cruz Biotechnology; sc-1616-R) and mouse (Sigma-Aldrich; A5441) anti-ACTIN antibodies were used as loading controls. Western analyses were performed in duplicate on independent cultures to ensure reproducibility.</p><p>For determination of [Ca<sup>2+</sup>] flux into the cytoplasm, normal control and proband fibroblasts and osteoblasts were cultured in 96-well plates for 3 days prior to loading with 5 &#956;M Fura-2AM (Life Technologies) in HBSS for 30 min at 37&#176;C in 5% CO<sub>2</sub>. The Ca<sup>2+</sup> transients were recorded as the 340/380 nm ratio (R) of the resulting 510-nm emissions using a Mithras LB 940 plate reader (Berthold Technologies), as previously described [<xref ref-type="bibr" rid="pgen.1006156.ref061">61</xref>]. Intracellular [Ca<sup>2+</sup>] was calculated using the equation [Ca<sup>2+</sup>]<sub>i</sub> = K<sub>d</sub> (R-R<sub>min</sub>)/(R<sub>max</sub>-R)(F<sup>380</sup><sub>max</sub>/F<sup>380</sup><sub>min</sub>), where R<sub>min</sub> is the ratio at zero [Ca<sup>2+</sup>], R<sub>max</sub> is the ratio when Fura-2 is completely saturated with Ca<sup>2+</sup>, F<sup>380</sup><sub>min</sub> is the fluorescence at 380 nm for zero [Ca<sup>2+</sup>], and F<sup>380</sup><sub>max</sub> is the fluorescence at saturating [Ca<sup>2+</sup>] and K<sub>d</sub> = 224 nM [<xref ref-type="bibr" rid="pgen.1006156.ref062">62</xref>]. To stimulate Ca<sup>2+</sup> release, 200 &#956;M of ATP or 2 &#956;g/ml ionomycin was injected into cultures using the Mithras 940. For inhibition experiments, cells were incubated for 30 min before analysis with either 100 &#956;M 2-APB to block IP<sub>3</sub>R, or 100 nM thapsigargin for SERCA ER Ca<sup>2+</sup> pump inactivation.</p><p>Endoplasmic reticulum luminal free [Ca<sup>2+</sup>] was quantified using D1ER, a cameleon class calcium FRET sensor containing YFP and CFP, as previously described [<xref ref-type="bibr" rid="pgen.1006156.ref025">25</xref>,<xref ref-type="bibr" rid="pgen.1006156.ref026">26</xref>]. Normal and proband fibroblasts (5 x 10<sup>5</sup> cells) were transfected with 2 &#956;g D1ER or with 1 &#956;g of pmaxGFP plasmid by electroporation using a 4D-Nucleofector system (Lonza), seeded into covered chamber slides and cultured at 37&#176;C in 5% CO<sub>2</sub> for 48 hr. Following a 30 min incubation in Ca<sup>2+</sup>-free HBSS, cells were treated with 1 mM ATP to evoke transient depletion of ER calcium. Cells were then treated with 5 &#956;M ionomycin (Sigma) and 3 mM EGTA (Sigma) to obtain a minimum ER calcium signal reference value. Images were obtained on a Zeiss 510 laser scanning microscope using a 20x 0.8 NA objective. D1ER was excited at 405 nm. CFP emission was collected at 420&#8211;480 nm and the FRET signal was collected at 505&#8211;550 nm. Ratios were calculated in ImageJ (v1.50i, NIH, USA) by first subtracting the background using the Rolling Ball plugin (50 pixel radius, paraboloid) and filtering using a Gaussian blur (2 pixel radius) for each image and then dividing the FRET image by the CFP image. The ratio was then normalized to the average ratio obtained 6 frames before ATP application (R<sub>0</sub>). R<sub>min</sub> was determined 1 min after ionomycin application from the new stable baseline using a 6-frame average. Calcium signal changes between baseline (before ATP treatment) and depletion (after ionomycin treatment) were evaluated using &#916;R = R<sub>min</sub>/R<sub>0</sub>&#8722;1.</p><p>Steady-state collagen analysis was performed as previously described [<xref ref-type="bibr" rid="pgen.1006156.ref063">63</xref>]. Normal control and proband fibroblasts or osteoblasts were grown to confluence in 6-well culture dishes. Cells were incubated with 437.5 &#956;Ci/ml of 3.96 TBq/mmol L-[2,3,4,5-<sup>3</sup>H] proline for 18 h prior to collection. Procollagens were precipitated overnight at 4&#176;C with 176 mg/ml ammonium sulfate, pelleted by centrifugation at 37,000 x g, and digested with pepsin (50 &#956;g/ml) for 4 h. Collagen alpha chains were separated on 6% SDS-urea-polyacrylamide gels and visualized by autoradiography.</p><p>For analysis of collagen modification, confluent cell cultures were stimulated for collagen synthesis in DMEM containing 0.1% FBS and 100 &#956;g/ml ascorbate for three days, with daily collection. Collected medium was buffered with 100 mM Tris-HCl, pH 7.4, and protease inhibitors were added to the following final concentrations: 25 mM EDTA, 0.02% NaN<sub>3</sub>, 1 mM phenylmethylsulfonylfluoride, 5 mM benzamidine, and 10 mM N-ethylmaleimide. Procollagens were then precipitated with 176 mg/ml ammonium sulfate overnight at 4&#176;C and pelleted by centrifugation at 16,000 x g for 2 h. Procollagens were resuspended in 0.5 M acetic acid and digested with 0.1 mg/ml pepsin at 4&#176;C overnight. Selective salt precipitation of type I collagen with 0.9 M NaCl in 0.5 M acetic acid was performed twice, followed by dialysis against 5mM acetic acid overnight and lyophilization for further analyses.</p><p>Differential Scanning Calorimetry (DSC) scans were performed as previously described [<xref ref-type="bibr" rid="pgen.1006156.ref064">64</xref>]. Thermograms were recorded in 0.2 M sodium phosphate, 0.5 M glycerol, pH 7.4, from 10 to 50&#176;C at 0.125&#176;C/min heating rates in a Nano III DSC instrument (Calorimetry Sciences Corporation).</p><p>Amino acid analysis to quantify total 4-hydroxyproline, proline, 5-hydroxylysine and lysine content of type I collagen was performed by high-pressure liquid chromatography (AIBiotech). For determination of modification of specific residues pro-alpha chains of type I procollagen were resolved by SDS-PAGE, followed by in-gel trypsin digestion for analysis of targeted peptides. Electrospray mass spectrometry was performed using an LTQ XL ion-trap mass spectrometer equipped with in-line liquid chromatography (Thermo Scientific) using a C4 5 &#956;m capillary column (300 &#956;m &#215; 150 mm; Higgins Analytical RS-15M3-W045). The LC mobile phase consisted of buffer A (0.1% formic acid in MilliQ water) and buffer B (0.1% formic acid in 3:1 acetonitrile:n-propanol v/v). The LC sample stream was introduced into the mass spectrometer by electrospray ionization (ESI) with a spray voltage of 4 kV. Proteome Discoverer software (Thermo Scientific) was used for peptide identification using the NCBI protein database.</p><p>Procollagen chain incorporation was followed as previously described [<xref ref-type="bibr" rid="pgen.1006156.ref065">65</xref>]. Procollagen alpha chains were labeled with a pulse of 140 &#956;Ci/ml of 3.96 TBq/mmol L-[2,3,4,5-<sup>3</sup>H] proline for 80 min, then chased with DMEM containing 10% serum, 50 &#956;g/ml ascorbic acid and 10 mM proline for the time intervals specified. Samples were ethanol precipitated, separated on 5% SDS-Urea-polyacrylamide gels and visualized by autoradiography.</p><p>For pulse-chase assays, performed in triplicate, normal control and proband fibroblasts or osteoblasts were grown to confluence in 6-well culture dishes [<xref ref-type="bibr" rid="pgen.1006156.ref066">66</xref>]. For each cell line, two wells were used for cell counts. Cells were labeled for 4 h with 1.75 &#956;Ci [<sup>14</sup>C]-proline and then chased with fresh medium containing 2 mM cold proline. Medium and cell layer procollagens were harvested at the indicated times, digested with pepsin and precipitated with 2M NaCl. Samples were loaded for equivalent cell number on 3&#8211;8% Tris-acetate gels. Collagen alpha chains were quantitated by densitometry and expressed as the percent secreted at each time point, as determined by (media)/(cell + media) x 100.</p><p>Alternatively, cells were grown to confluence in 75 cm<sup>2</sup> culture flasks and incubated in phenol red-free DMEM (Life Technologies) containing 0.1% FBS and 100 &#956;g/ml ascorbic to generate conditioned media. After 24 h, media was collected, supplemented with protease inhibitor cocktail (Sigma-Aldrich), and concentrated using Amicon Ultra-4 centrifugal filter devices (Millipore). Total collagen derived from each flask was quantitated using the Sircol Soluble Collagen Assay (Biocolor Ltd) according to the manufacturer&#8217;s instructions. Immediately following media collection, cultures were trypsinized and collected for determination of cell count from each flask. Data is expressed as amount of collagen secreted per cell per hour and were generated from 3 fibroblast and osteoblast cultures per proband, 3 replicates of the normal osteoblast control cultures, and 6 normal control fibroblast cultures.</p><p>Processing of procollagens secreted by fibroblasts was examined by labeling confluent cells with 260 &#956;Ci/ml of 3.96 TBq/mmol L-[2,3,4,5-<sup>3</sup>H] proline for 24 h and then chased with DMEM containing 2 mM non-radioactive proline and 10% fetal bovine serum. Media from independent wells were harvested at 24-h intervals over a 5-day period as previously described, precipitated with ammonium sulfate, electrophoresed on 6% polyacrylamide-urea-SDS gels, and visualized by autoradiography [<xref ref-type="bibr" rid="pgen.1006156.ref067">67</xref>].</p><p>Normal control and proband primary fibroblasts were grown to confluence and stimulated every other day for 14 days with fresh DMEM containing 10% FBS and 100 &#956;g/ml ascorbic acid, as described [<xref ref-type="bibr" rid="pgen.1006156.ref068">68</xref>], and then labeled for 24 h with 437.5 &#956;Ci/ml L-[2,3,4,5-<sup>3</sup>H] proline. Matrix collagens were sequentially extracted at 4&#176;C, with neutral salt for newly incorporated collagen, acetic acid for collagens with acid-labile cross-links, and pepsin digestion for collagens with mature cross-links [<xref ref-type="bibr" rid="pgen.1006156.ref069">69</xref>]. All fractions were electrophoresed on 6% polyacrylamide-urea-SDS gels. Samples were loaded for equivalent densitometry signal, and the total signal for each fraction calculated by adjusting the signal by the total volume of that fraction.</p><p>Data were analyzed using an unpaired Student <italic>t</italic>-test with two-tailed analysis and considered statistically significant at p &lt; 0.05.</p>