<p>Disomic BIR strains are derivatives of AM1003 with genotype <italic>MAT</italic>
<bold>a</bold>-<italic>LEU2-</italic>tel/<italic>MAT</italic>&#945;-<italic>inc ade1 met13 ura3 leu2-3</italic>,<italic>112/leu2 thr4 lys5 hml</italic>&#916;::<italic>ADE1/hml</italic>&#916;::<italic>ADE3 hmr</italic>&#916;::<italic>HYG ade3</italic>::<italic>Gal-HO</italic> FS2&#916;::<italic>NAT/FS2</italic> [<xref ref-type="bibr" rid="pgen.1004990.ref038">38</xref>]. Ectopic BIR strains are derivatives of JRL346 [<xref ref-type="bibr" rid="pgen.1004990.ref040">40</xref>]. HU sensitivity was tested on derivatives of JKM139 with genotype <italic>ho</italic>&#916; <italic>MAT</italic>
<bold>a</bold>
<italic>hml</italic>&#916;::<italic>ADE1 hmr</italic>&#916;::<italic>ADE1 ade1-100 leu2-3</italic>,<italic>112 lys5 trp1</italic>::<italic>hisG&#8217; ura3-52 ade3</italic>::<italic>GAL</italic>::<italic>HO</italic> [<xref ref-type="bibr" rid="pgen.1004990.ref064">64</xref>]. JKM161, YMV80, tGI354 were used for mating-type switch, SSA, and ectopic recombination assays, respectively [<xref ref-type="bibr" rid="pgen.1004990.ref065">65</xref>,<xref ref-type="bibr" rid="pgen.1004990.ref066">66</xref>]. Gene conversion with a 3.4-kb gap strain was constructed by insertion of a <italic>TRP1</italic> containing DNA fragment originating from the pFA6a-<italic>TRP1</italic> plasmid into the <italic>MAT</italic>
<bold>a</bold>-inc template of tGI354. The yeast strain expressing MYC epitope tags and degron fusion was constructed by PCR amplification of degron cassette as described in [<xref ref-type="bibr" rid="pgen.1004990.ref017">17</xref>,<xref ref-type="bibr" rid="pgen.1004990.ref067">67</xref>]. The strain list is shown in <xref ref-type="supplementary-material" rid="pgen.1004990.s016">S2 Table</xref>.</p><p>The assay was performed as previously described with minor changes in cell culture[<xref ref-type="bibr" rid="pgen.1004990.ref038">38</xref>]. BIR derivative strains were cultured in SC-Ade medium overnight and then transferred to YEP-glycerol medium and cultured for more than 6 h before plating on YEP-galactose plates. Cells cultured in this way are selected against spontaneous chromosome loss. An aliquot of cells was also plated on YEPD plates. After colonies grew up on the plates for 3&#8211;5 days, they were replica-plated onto SC-Leu and SC-Ade plates to determine their genotypes. At least 500 individual colonies were scored and at least 2 different isolates with the same genotype were analyzed. Colonies replica-plated from YEPD plates were also examined to determine the background chromosome loss. BIR frequency of ectopic BIR assay was determined as described in [<xref ref-type="bibr" rid="pgen.1004990.ref005">5</xref>].</p><p>To physically monitor BIR products, 3&#215;10<sup>7</sup> cells were harvested to make DNA plugs for each time point. Pulsed-field gel electrophoresis (PFGE) was run in a 1.2% agarose gel with switch time 10&#8211;35s, voltage 6v/cm, angle 120&#176;, 14&#176;C for 32&#8211;40 h (disomic BIR assay) or with switch time 50&#8211;70 s, voltage 6v/cm, angle 120&#176;, 14&#176;C for 21&#8211;24 h (ectopic BIR assay). PCR products from <italic>ADE1</italic> gene or <italic>MCH2</italic> fragment were radiolabeled and used to detect BIR bands by Southern blot assay.</p><p>To test BIR in Hst3-depleted cells or those treated with NAM, <italic>HST4</italic>-deleted cells were arrested in G1 by treating them with 10 &#181;g/ml alpha-factor, subjected to incubation at 37&#176;C (non-permissive conditions) for an additional 1.5 h, releasing cells into nocodazole containing (15 &#181;g/ml) media for 2.5 h and inducing HO endonuclease by addition of 2% (w/v) galactose. Aliquots of cells were harvested at each indicated time point and subjected to Western blot to investigate the level of Hst3, H3K56 acetylation using anti-MYC (Sigma) and H3K56 acetylation antibodies (Upstate) and FACS analysis for cell cycle progression. Cells were harvested to prepare DNA plugs for PFGE to analyze BIR products and PCR analysis of repair synthesis. PCR products from <italic>ADE1</italic> gene or <italic>MCH2</italic> fragment were radiolabeled and used to detect BIR bands or initial repair products by Southern blot.</p><p>Logarithmically growing cells were alpha-factor arrested for 3 h and released into YEPD medium containing 150 mM HU. Cells were harvested at each time point, washed twice with water and plated onto YEPD plates. The survival rate was calculated by the number of colonies treated with HU divided by the number of colonies before HU treatment. For the <italic>mrc1AQ</italic> experiment, Uracil dropout medium instead of YEPD medium was used for cell culture.</p><p>Images were taken with a DeltaVision microscopy system (Applied Precision). Each original image contained 12&#8211;15 Z-stacks. After deconvolution, images were subjected to quick-projection. The projected pictures were then counted for Rad52-GFP foci using SoftwoRx.</p><p>3&#8217; flap cleavage efficiency during gene conversion was determined by measuring the retention frequency of yeast centromeric plasmids pFP120 and pFP122 after HO induced DSB as described in [<xref ref-type="bibr" rid="pgen.1004990.ref068">68</xref>]. The percentage of plasmid retention is calculated as the fraction of colonies retaining the repaired plasmid on SC&#8211;Ura divided by the total number of colonies on YEPD.</p><p>Chromatin immunoprecipitation (ChIP) assays were performed as described previously [<xref ref-type="bibr" rid="pgen.1004990.ref069">69</xref>]. Briefly, JKM161 cell cultures grown to a density between 1 &#215; 10<sup>7</sup> and 2 &#215; 10<sup>7</sup> cells/ml in pre-induction medium (YEP-glycerol) were induced for HO endonuclease by adding 2% galactose. For immunoprecipitation, the sonicated extracts were incubated with Rad51 antibody (kindly provided by Dr. Patrick Sung P). Quantitative PCR was performed using <italic>MAT</italic>
<bold>a</bold> (break) specific primers and <italic>HML</italic> (donor) specific primers.</p><p>JKM139 derivative cells grown to a density between 1 &#215; 10<sup>7</sup> and 2 &#215; 10<sup>7</sup> cells/ml in pre-induction medium (YEP-glycerol) were induced for HO endonuclease by adding 2% galactose. Five ml of cells at different time points were harvested and processed for genomic DNA isolation using Masterpure yeast genomic DNA purification kit (Epicentre). DNA equal to 0.4 ml of cells was digested by 20 units of <italic>Bsa</italic>J1 enzyme or mock digested without <italic>Bsa</italic>J1 for 2 h in a 50 &#956;l reaction volume. Samples were then diluted 10 times and subject to quantitative PCR to determine the percentage of resection. The principle of this assay is shown in <xref ref-type="supplementary-material" rid="pgen.1004990.s008">S8 Fig.</xref> The rate of single stranded DNA formation (r) is calculated by r = (RS<sub><italic>Bsa</italic>J1 digested</sub>/ RS<sub>Mock digested</sub>)/(Input<sub><italic>Bsa</italic>J1 digested</sub>/ Input<sub>Mock digested</sub>), where RS indicates PCR value at the <italic>Bsa</italic>J1 restriction site flanking the HO break and Input indicates PCR value at a site from a different chromosome and does not contain the <italic>Bsa</italic>J1 restriction site. Since r = X/(X+2(1-X)) where X is the rate of resection, the percentage of resection was calculated by: 100X% = 200r/(r+1). Primer sequences for determining resection at 3.3 kb, 12.6 kb or input are 5&#8217;TCCGAACCAATGTCGTCATCCATAGTATC and 5&#8217;GGCGCCTTTAATTCATGGTCTTTACCTTT; 5&#8217;TTCTACGGCGACTGGTGGAATTGC and 5&#8217;ATGTAGCTTGGCTCTTGCTCAAATGC; 5&#8217;ACTTAGTTCTGGAGATTATCGCCTTATCG and 5&#8217;CTGTCTTTCGCTTAGTTCACCTCTACC, respectively.</p><p>Semi-quantitative PCR was performed with Iproof polymerase (Biorad) with a 2 minute extension time and 20 cycles (allelic BIR assay) or 28 cycles (ectopic BIR assay) using Forward primer F (5&#8217; TACTGGAGTTAGTTGAAGCATTAGG3&#8217;) upstream of the HO break inside the <italic>URA3</italic> gene and Reverse primer R1 (5&#8217; TGACTTCCAGACGCTATCCTGTGAA 3&#8217;) inside the <italic>MAT</italic> locus at an alpha-specific sequence or R2 (5&#8217;GATATTAAGTCCTCCGTCCAATCTG3&#8217;) downstream of <italic>MAT</italic>alpha-inc, inside the <italic>TAF2</italic> gene (allelic BIR assay) or Forward primer F2 (5&#8217;AATATTGTGTGTATGGGCACAAACCCTTG3&#8217;) inside the <italic>NPR2</italic> gene and Reverse primer R3 (5&#8217; CCTCAATGTCTCTTCTATCGGAAT3&#8217;) that anneals to the carboxy terminus of <italic>CAN1</italic> gene (ectopic BIR assay). Control PCR was performed with 1 minute extension time for 20 cycles using Forward primer F3 (5&#8217;TCCATGCTAGATTAGCACACAGTAA3&#8217;) and reverse primer R4 (5&#8217;CTTTTGTAGGTGTCCTTAATTTCCA3&#8217;). PCR products were resolved in a 1% agarose gel and stained with ethidium bromide. The gel pictures were captured by GEL-LOGIC system and PCR bands were quantified by Carestream software. Inversed images were showed in the <xref ref-type="fig" rid="pgen.1004990.g006">Fig. 6</xref> and <xref ref-type="supplementary-material" rid="pgen.1004990.s011">S11 Fig.</xref>
</p><p>A fluctuation-based test was performed to determine gross chromosomal rearrangements [<xref ref-type="bibr" rid="pgen.1004990.ref070">70</xref>]. In brief, 3&#8211;5 ml of 11 confluent cultures in YEPD from single colonies were plated on SC medium lacking arginine and supplemented with 60 mg/L L-canavanine and 1 g/L 5-FOA for scoring GCRs. Cells were also diluted and plated on YEPD plates for counting total number of cells. GCR plates were cultured for 5&#8211;10 days at 30&#176;C and colony numbers counted. Calculation of GCR rate and determination of junction type (<italic>de novo</italic> telomere formation or translocation) were as previously described [<xref ref-type="bibr" rid="pgen.1004990.ref070">70</xref>].</p>