<p>The CLOZUK study has UK National Research Ethics approval (Ethics Committee WALES REC 2, Study ID: 10/WSE02/15). The CLOZUK samples were collected anonymously from across the UK (thus without express consent), consistent with the UK Human Tissue Act and with the approval of the above ethics committee.</p><p>We collated the available clinical and molecular data from large (&gt;400 cases), systematically ascertained CNV studies of SZ from the literature, or known to us via our collaborations. We similarly collected data on other neurodevelopmental disorders such as DD/ASD and multiple congenital anomalies (MCA), focusing on two large studies based on referrals to genetic clinics and on three studies that specifically determined the parental origin of these duplications and reported on the presence of triplications/idic15 (<xref ref-type="table" rid="pgen.1005993.t001">Table 1</xref>). We used the reported information on parental origin of the 15q11.2-q13.3 duplications or, where possible, obtained DNA samples to establish this (details in <xref ref-type="supplementary-material" rid="pgen.1005993.s001">S1 Text</xref>). Most teams that we approached responded to our requests, but some had no access to patient DNA and could not take part.</p><p>The control cohorts included 149,780 individuals (<xref ref-type="table" rid="pgen.1005993.t001">Table 1</xref>). The largest sample was from Iceland (~115,000 genotyped individuals at the time writing), a population-based cohort containing related individuals and those affected with medical and/or neuropsychiatric conditions. An ideal control cohort would only have one individual per family to avoid biases from over-sampling within population lineages. However, the Icelandic sample is designed to achieve total population coverage; therefore it is impractical to single out unrelated individuals. The rate of 15q11.2-q13.3 duplications in this population was assumed to be unbiased without excluding relatives of individuals with or without 15q11.2-q13.3 duplications. In fact, this population approaches the ideal general population ascertainment, that should give the best estimates for the various analyses performed here, therefore we decided to retain the two pairs of relatives who were carriers in this population.</p><p>Penetrance was estimated according to the formula originally proposed by Vassos et al. [<xref ref-type="bibr" rid="pgen.1005993.ref037">37</xref>], updated for the joint effect of DD/ASD/MCA and for SZ, as we proposed earlier [<xref ref-type="bibr" rid="pgen.1005993.ref026">26</xref>]. Briefly, we first estimated the rate of the duplications in the general population. We assumed that the control (healthy) population comprises 95% of the population, as explained in our previous work [<xref ref-type="bibr" rid="pgen.1005993.ref011">11</xref>]. The rest of the population is made up of approximately 1% SZ and 4% DD/ASD/MCA cases. From this we estimated the proportion of 15q11.2-q13.3 duplication carriers in the general population that develop SZ or DD/ASD/MCA (the penetrance). Although many duplication carriers might have both SZ and DD/ASD/MCA, we made the simplifying assumption that they would usually be ascertained only once. As the DD/ASD/MCA patients referred for genetic testing and the SZ patients recruited for studies might represent a more severely affected sub-group, we repeated the estimates after reducing by half these population rates (down to 0.5% for SZ and 2% for DD/ASD/MCA) and show these results in the <xref ref-type="supplementary-material" rid="pgen.1005993.s001">S1 Text</xref>, <xref ref-type="supplementary-material" rid="pgen.1005993.s006">S3 Table</xref>. Estimating 95% confidence intervals for the frequencies in each population followed the method we used in our previous paper [<xref ref-type="bibr" rid="pgen.1005993.ref026">26</xref>]. Briefly, we first estimated the binomial CIs for the frequencies of CNVs in each population, including controls and the general population, using the Wilson score interval. Upper and lower 95% bounds for penetrance were estimated from the upper or lower bounds of CNV frequencies in patients and the lower or upper bounds of the frequencies in the general population.</p><p>According to the mutation-selection balance theory, pathogenic mutations in the general population are found at low frequencies, where the addition of <italic>de novo</italic> mutations is balanced by the selection pressure against them (<italic>q = &#956;/s</italic>), where <italic>q</italic> is the allele frequency in the general population, <italic>&#956;</italic> is the mutation frequency and <italic>s</italic> is the selection coefficient [<xref ref-type="bibr" rid="pgen.1005993.ref011">11</xref>,<xref ref-type="bibr" rid="pgen.1005993.ref026">26</xref>]. In order to estimate the selection coefficients (<italic>s</italic>) of 15q11.2-q13.3 duplications of maternal and paternal origin, we used the methods outlined in our previous work [<xref ref-type="bibr" rid="pgen.1005993.ref011">11</xref>,<xref ref-type="bibr" rid="pgen.1005993.ref026">26</xref>] as follows: The selection coefficient of a CNV can be approximated as the proportion of <italic>de novo</italic> CNVs out of the total number of CNVs (<italic>de novo</italic> and inherited) that are observed in an unbiased sample of CNV carriers. This is because the number of CNVs filtered out by natural selection is approximately equal to those introduced in the population as <italic>de novo</italic> mutations, after making the simplifying assumption that the frequency of the CNV does not change from generation to generation. We then estimated the mutation rate of the duplications, using the above formula. We should point out that the actual differences between maternal and paternal mutation rates are prone to error, e.g. due to the flipping of maternal and paternal duplications according to the gender of transmitting parents and possible differences in fecundity between affected male and female carriers (<xref ref-type="sec" rid="sec007">Discussion</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We established the parental origin of 15q11.2-q13.3 duplications for carriers from studies where it was previously unknown or not reported (<xref ref-type="table" rid="pgen.1005993.t001">Table 1</xref> and <xref ref-type="supplementary-material" rid="pgen.1005993.s004">S1 Table</xref>). These were the studies/datasets by Gawlick, Alexic, the BBGRE database [<xref ref-type="bibr" rid="pgen.1005993.ref018">18</xref>] (<ext-link ext-link-type="uri" ns0:href="https://bbgre.brc.iop.kcl.ac.uk/">https://bbgre.brc.iop.kcl.ac.uk/</ext-link>), and new carriers in the Icelandic population and three published datasets [<xref ref-type="bibr" rid="pgen.1005993.ref012">12</xref>,<xref ref-type="bibr" rid="pgen.1005993.ref016">16</xref>,<xref ref-type="bibr" rid="pgen.1005993.ref023">23</xref>]. Parental origin was established by one of several methods. For DNA available to the Cardiff laboratory, we used a methylation-sensitive high-resolution melt curve analysis, as described previously [<xref ref-type="bibr" rid="pgen.1005993.ref008">8</xref>,<xref ref-type="bibr" rid="pgen.1005993.ref019">19</xref>,<xref ref-type="bibr" rid="pgen.1005993.ref038">38</xref>], (<xref ref-type="supplementary-material" rid="pgen.1005993.s001">S1 Text</xref> and <xref ref-type="supplementary-material" rid="pgen.1005993.s002">S1 Fig</xref>). The same method was used for the samples from Canada [<xref ref-type="bibr" rid="pgen.1005993.ref007">7</xref>]. In the Icelandic sample parental origin was either determined with methylation-sensitive Southern analysis [<xref ref-type="bibr" rid="pgen.1005993.ref005">5</xref>] or by long-range haplotype analysis (<xref ref-type="supplementary-material" rid="pgen.1005993.s001">S1 Text</xref>). Microsatellite analysis was used for the BBGRE dataset [<xref ref-type="bibr" rid="pgen.1005993.ref018">18</xref>].</p>