<p xmlns:ns0="http://www.w3.org/1999/xlink">Eighteen Ascomycetes and Basidiomycetes species were selected in this study as sample sets of closely related species for genomes comparisons. Publicly available genomic assemblies were downloaded from the Joint Genome Institute&#8217;s fungal genome portal MycoCosm [<xref ref-type="bibr" rid="pgen.1006108.ref068">68</xref>] (<ext-link ext-link-type="uri" ns0:href="http://jgi.doe.gov/fungi">http://jgi.doe.gov/fungi</ext-link>), the Broad Institute (<ext-link ext-link-type="uri" ns0:href="https://www.broadinstitute.org/">https://www.broadinstitute.org/</ext-link>) and FungiDB [<xref ref-type="bibr" rid="pgen.1006108.ref069">69</xref>]. The genome sequences of the <italic>P</italic>. <italic>ostreatus</italic> monokaryotic strains PC15 v2.0 [<xref ref-type="bibr" rid="pgen.1006108.ref034">34</xref>] and PC9 v1.0, which were obtained by de-dikaryotization of the dikaryotic strain N001 [<xref ref-type="bibr" rid="pgen.1006108.ref035">35</xref>], were used as models for building the pipelines described in this paper.</p><p xmlns:ns0="http://www.w3.org/1999/xlink"><italic>De novo</italic> identification of repetitive sequences in the genome assemblies was performed by running the RECON [<xref ref-type="bibr" rid="pgen.1006108.ref070">70</xref>] and RepeatScout [<xref ref-type="bibr" rid="pgen.1006108.ref071">71</xref>] programs (integrated into the RepeatModeler pipeline). LTRharvest [<xref ref-type="bibr" rid="pgen.1006108.ref036">36</xref>] was used to improve the detection of full length LTR-retrotransposons. LTRharvest results were filtered to avoid false positives as follows: elements were de-duplicated and used as queries for BLASTN searches (cutoff E-value = 10<sup>&#8722;15</sup>) against the genome assembly and for BLASTX (cutoff E-value = 10<sup>&#8722;5</sup>) against the Repbase peptide database [<xref ref-type="bibr" rid="pgen.1006108.ref054">54</xref>]. Only sequences longer than 400 bp with more than five copies or yielding a significant hit to a described LTR-retrotransposon were kept for further analysis. The outputs of the above programs were merged and clustered at 80% similarity using USEARCH [<xref ref-type="bibr" rid="pgen.1006108.ref072">72</xref>] to create species-specific (i.e., <italic>P</italic>. <italic>ostreatus</italic> PC15 and PC9) or genus-specific (i.e., <italic>F</italic>. <italic>oxisporum</italic> and <italic>F</italic>. <italic>graminearum</italic>) TE libraries. Each consensus sequences library was classified using BLASTX against the Repbase peptide database, and the final libraries were used as input for RepeatMasker (<ext-link ext-link-type="uri" ns0:href="http://www.repeatmasker.org">http://www.repeatmasker.org</ext-link>). Consensus sequences without similarity to any Repbase entry were labeled as &#8216;unknown&#8217;. The RepeatMasker output was parsed using the <italic>One</italic>_<italic>code</italic>_<italic>to</italic>_<italic>find</italic>_<italic>them</italic>_<italic>all</italic> script [<xref ref-type="bibr" rid="pgen.1006108.ref073">73</xref>] to reconstruct TE fragments into full-length copies and estimate the fraction of the genome occupied by each TE family.</p><p>To identify solo-LTRs, the left terminal repeat of every autonomous copy was extracted, and a BLASTN against each assembly was performed. The flanking sequences of every hit (5,000 bp, cutoff E-value = 10<sup>&#8722;15</sup>) were extracted and screened for retrotransposon internal sequences. Solo-LTRs were defined as those hits lacking internal retrotransposon sequences at the flanking sites.</p><p>To determine whether TEs were non-randomly distributed, the distribution of inter-TE distances was compared (Mann-Whitney-Wilcoxon text) with that of the inter-element distances of a randomly generated subset of 1,196 elements. In addition, TEs and gene model annotations were merged and used as reference for a hypergeometric test to test for the presence of regions enriched in TEs. The analysis was performed using REEF [<xref ref-type="bibr" rid="pgen.1006108.ref074">74</xref>] with a Q-value of 0.05 (FDR 5%), a window width of 100 kb with a shift of 10 kb and a minimum number of 10 features in clusters.</p><p>The <italic>P</italic>. <italic>ostreatus</italic> PC15 and PC9 genome assemblies were aligned using the Mercator and MAVID pipeline [<xref ref-type="bibr" rid="pgen.1006108.ref075">75</xref>], using the fully assembled PC15 genome as a reference. Gene model positions and TE hits of the PC15 strain were used to extract individual alignments and to check the homozygous <italic>vs</italic>. heterozygous nature of the insertions. A locus was considered homozygous if the alignment spanned at least 80% of the whole locus length, and heterozygous when the PC9 allele was absent.</p><p>Long Terminal Repeats of every intact, full-length element were extracted and aligned. Kimura 2-Parameter distance was obtained using a Python script and transformed to My using the approach described in [<xref ref-type="bibr" rid="pgen.1006108.ref039">39</xref>] and the fungal substitution rate of 1.05 &#215; 10<sup>&#8722;9</sup> nucleotides per site per year [<xref ref-type="bibr" rid="pgen.1006108.ref040">40</xref>].</p><p>Mycelia were harvested, frozen and ground in a sterile mortar in the presence of liquid nitrogen. DNA was extracted using a Fungal DNA Mini Kit (Omega Bio-Tek, Norcross, GA, USA). Sample concentrations were measured using a Qubit 2.0 Fluorometer (Life Technologies, Madrid, Spain), and purity was measured using a NanoDrop 2000 (Thermo-Scientific, Wilmington, DE, USA). PCR reactions were performed according to Sambrook <italic>et al</italic> [<xref ref-type="bibr" rid="pgen.1006108.ref076">76</xref>] using primers designed after the TE flanking sequences (<xref ref-type="supplementary-material" rid="pgen.1006108.s010">S1 Text</xref>, Supplementary Information). Total RNA was extracted from 200 mg of deep frozen tissue using Fungal RNA E.Z.N.A Kit (Omega Bio-Tek, Norcross, GA, USA), and its integrity was estimated by denaturing electrophoresis on 1% (w/v) agarose gels. Nucleic acid concentrations were measured using a Nanodrop 2000 (Thermo Scientific, Wilmington, DE, USA), and the purity of the total RNA was estimated by the 260/280 nm absorbance ratio. Messenger RNA was purified using a MicroPoly(A) Purist kit (Ambion, USA). Transcriptome libraries were generated and sequenced by Sistemas Genomicos S.L. (Valencia, Spain) on a SOLiD platform, following the manufacturers&#8217; recommendations (Life Technologies, CA, USA).</p><p xmlns:ns0="http://www.w3.org/1999/xlink"><italic>P</italic>. <italic>ostreatus</italic> RNA-seq datasets corresponding to PC15 and PC9 strains (8.4 and 9.7 million reads in PC15 and PC9, respectively) cultured in SMY medium and harvested during the exponential growth phase, were used to analyze the transcription of genes and TEs. The quality of the SOLiD RNA-seq reads was verified using FastQC (<ext-link ext-link-type="uri" ns0:href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</ext-link>), and they were mapped to their corresponding PC15 v2.0 or PC9 v1.0 assemblies using TopHat [<xref ref-type="bibr" rid="pgen.1006108.ref077">77</xref>], restricting the multihits option to 1. HTseq-count [<xref ref-type="bibr" rid="pgen.1006108.ref078">78</xref>] was used to determine the number of reads mapping to every feature. SAMtools [<xref ref-type="bibr" rid="pgen.1006108.ref079">79</xref>], BEDTools [<xref ref-type="bibr" rid="pgen.1006108.ref080">80</xref>] and custom Python scripts were used to manipulate the data, to calculate RPKMs and to obtain genome coverages. Public RNA-seq data from other species were downloaded from the NCBI SRA database and were analyzed using the same pipeline (accessions SRR1257938 <italic>Saccharomyces cerevisiae</italic> S288C [<xref ref-type="bibr" rid="pgen.1006108.ref081">81</xref>], SRR1284049 <italic>Botrytis cinerea</italic> B05.10 [<xref ref-type="bibr" rid="pgen.1006108.ref082">82</xref>], SRR1592424 <italic>F</italic>. <italic>graminearum</italic> [<xref ref-type="bibr" rid="pgen.1006108.ref083">83</xref>] and SRR1165053 <italic>Laccaria bicolor</italic> [<xref ref-type="bibr" rid="pgen.1006108.ref084">84</xref>]).</p><p>For analyzing the expression of TE families, reads were mapped to the extracted transposon sequences using Bowtie [<xref ref-type="bibr" rid="pgen.1006108.ref085">85</xref>] and allowing multi-mapping. RSEM software was used to calculate TE expression because its algorithm is especially designed to handle multi-mapped reads [<xref ref-type="bibr" rid="pgen.1006108.ref086">86</xref>]. Afterwards, the FPKMs of each family were normalized to the number of elements.</p><p>Gene and TE annotations were intersected to obtain TE-associated genes (genes overlapping with any TE) and non-TE genes (genes not overlapping with any TE). Afterwards, the closest TE upstream and downstream to each non-TE gene was obtained at a maximum distance of 1 kb. The resulting genes were organized in three groups: i) genes with an upstream TE, ii) genes with a downstream TE and iii) genes with both upstream and downstream TEs. Control groups were obtained by subtracting target genes (three previous scenarios) to all the non-TE genes.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The predicted proteomes of all species were downloaded from the Mycocosm database (<ext-link ext-link-type="uri" ns0:href="http://genome.jgi.doe.gov/programs/fungi/index.jsf">http://genome.jgi.doe.gov/programs/fungi/index.jsf</ext-link>). After all-by-all BLASTP, proteins were clustered with MCL [<xref ref-type="bibr" rid="pgen.1006108.ref087">87</xref>] using an inflation value of 2. Clusters containing single copy genes of each genome were retrieved (allowing two missing taxa per cluster) and proteins were aligned with MAFFT [<xref ref-type="bibr" rid="pgen.1006108.ref088">88</xref>]. The alignments were concatenated after discarding poorly aligned positions with Gblocks [<xref ref-type="bibr" rid="pgen.1006108.ref089">89</xref>]. Maximum-likelihood phylogeny was constructed using RaxML [<xref ref-type="bibr" rid="pgen.1006108.ref090">90</xref>] under PROTGAMMAWAGF substitution model and 100 rapid bootstraps.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Using the <italic>P</italic>. <italic>ostreatus</italic> JGI browser we identified the internal transposase gene of a full length element of TIR_1 family. This protein was used as query for BLASTP searches (cutoff = E<sup>-5</sup>) against NCBI RefSeq protein database (independent searches were carried out against animal, plant and bacterial databases). The best five animal, plant and bacterial hits were retrieved when possible (i.e. only one hit was obtained using plant database). The same search was performed in the JGI database to retrieve the best five basidiomycete hits, and the best five non-basidiomycete hits. Proteins were aligned with MUSCLE [<xref ref-type="bibr" rid="pgen.1006108.ref091">91</xref>], and the alignments were trimmed using trimAl [<xref ref-type="bibr" rid="pgen.1006108.ref092">92</xref>] with the default parameters. An approximate maximum likelihood tree was constructed using FastTree [<xref ref-type="bibr" rid="pgen.1006108.ref093">93</xref>] and edited with Figtree (<ext-link ext-link-type="uri" ns0:href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</ext-link>). Transposases from <italic>P</italic>. <italic>patens</italic>, <italic>Wolbachia</italic> and <italic>Rhizopus oryzae</italic> were further analyzed to exclude the possibility of being a result of database contamination: Using TBLASTN against NCBI Whole-genome shotgun contigs or JGI genomic scaffolds, we identified their genomic position and verified that they were assembled in long scaffolds and surrounded by other host genes.</p><p>Raw sequencing data was deposited in GEO database under the accession number GSE81586.</p>