<p>Fly lines w<sup>1118</sup>, frt82B, frt40A, UAS::nlsGFP, UAS::hsc3, UAS::Xbp1<sup>RNAi</sup> (TRip:HMS03015) were obtained from the Bloomington <italic>Drosophila</italic> stock center. The following RNAi lines were obtained from the Vienna Drosophila RNAi Center: UAS::Xbp1<sup>RNAi</sup> (v109312, v15347), UAS::Hrd1RNAi (v6870), UAS::bsk<sup>RNAi</sup>.</p><p>The following fly lines were generously provided as indicated: <italic>y1w1</italic>; esg::Gal4/+ by Dr. S Hayashi; UAS::xbp1<sup>EP2112</sup> by Dr. Kyoung Sang Cho; UAS::xbp1<sup>d08698</sup> by Dr. P. Fernandez-Funez; esg<sup>ts</sup>F/O by Dr. H. Jiang; Su(H)Gbe::Gal4 by Dr. S. Bray; puc<sup>E69</sup>::lacZ by Dr. E. Mart&#237;n-Blanco; UAS::xbp1<sup>spliced</sup> by Dr. P. Domingos, UAS::bsk<sup>DN</sup> by Dr. M. Mlodzik.</p><p>The Hrd1 loss of function allele <italic>Hrd1<sup>Delta</sup></italic> was made by FRT-mediated deletion of sequences between the Pbac insertion lines Pbac{PB}sip3 c00467 and Pbac{PB}faf06363. hs-FLP was expressed in flies in which these Pbac insertions were <italic>in trans</italic>, deleting Hrd1 and its nearby gene CG2126.</p><p>All flies were raised on yeast/molasses-based food at 25&#176;C and 65% humidity on a 12 hr light/dark cycle, unless otherwise noted.</p><p>For tunicamycin or paraquat exposure, flies were starved in empty vials for 6&#8211;8 hrs and fed with a 5% sucrose solution&#177; 50 &#181;M tunicamycin or &#177;5 mM paraquat for 24 hrs followed by dissection in PBS.</p><p>For TARGET experiments, flies were raised at 18&#176;C and shifted to 29&#176;C at certain time points after eclosion. For MARCM clone induction, adult flies were aged for 1&#8211;2 days and then heat shocked at 37&#176;C for 45 min.</p><p>The DNA fragment containing an enhancer of Su(H)GBE and a mini promoter of HSP70 was amplified from Su(H)GBE-Gal4 <xref ref-type="bibr" rid="pgen.1004568-Zeng1">[39]</xref> using PCR, with the following primers:</p><p>
<named-content content-type="gene">5&#8242;-AGT<bold>GAATTC</bold>AATTAGGCCT<bold>A</bold>CTAGACTTG-3&#8242;</named-content> (the 20th nucleotide &#8220;T&#8221; is replaced by &#8220;A&#8221; to eliminate the endogenous XbaI site).</p><p>
<named-content content-type="gene">5&#8242;-AGT<bold>TCTAGA</bold>TCATGAT<bold>GCGGCCGC</bold>TCAGGAGGCTTGCTTCAAGCTTG-3&#8242;</named-content> (a NotI site was introduced in this primer).</p><p>The amplified DNA was cut and ligated into EcoRI and XbaI digested pCasper-Tub-Gal80 <xref ref-type="bibr" rid="pgen.1004568-Kaser1">[1]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1004568-Adolph1">[7]</xref>, <xref ref-type="bibr" rid="pgen.1004568-Lee1">[59]</xref> to produce the construct pCasper-Su(H)GBE. Then the DNA fragment containing Gal80 and Sv40 polyA was cut from pCasper-Tub-Gal80 at the NotI and XhoI sites, and ligated into NotI- and XhoI-digested pCasper-Su(H)GBE to produce the Su(H)GBE-Gal80 construct.</p><p>The construct was sequenced, purified, and microinjected into embryos using the standard method.</p><p>Guts were dissected in PBS, fixed for 45 min at room temperature in 100 mM glutamic acid, 25 mM KCl, 20 mM MgSO<sub>4</sub>, 4 mM sodium phosphate, 1 mM MgCl<sub>2</sub>, and 4%formaldehyde, washed for 1 hr, and incubated with primary antibodies and second antibodies in washing buffer (PBS, 0.5% BSA, 0.1% Triton X-100).</p><p>The following primary antibodies were used:, guinea-pig anti-hsc3 antibody antibody <xref ref-type="bibr" rid="pgen.1004568-Ryoo3">[60]</xref> (1&#8758;150), mouse anti-Delta (Developmental Studies Hybridoma Bank, 1&#8758;100), rat anti-Delta (gift from Dr. MD Rand, University of Rochester, 1&#8758;1000); rabbit anti-PH3 (phosphorylated histone H3, Upstate, 1&#8758;1000), mouse anti-&#946;-galactosidase (Developmental Studies Hybridoma Bank, 1&#8758;500), rabbit anti-&#946;-galactosidase (Cappel, 1&#8758;5000), rabbit anti-peIF2&#945; antibody (Cell Signaling: 3597, 1&#8758;150), mouse anti-pJNK antibody (Cell Signaling: 9255,1&#8758;150).</p><p>For Delta antibody staining, guts were fixed using a methanol-heptane method as descried <xref ref-type="bibr" rid="pgen.1004568-Lin1">[61]</xref>.</p><p>Fluorescent secondary antibodies were purchased from Jackson ImmunoResearch Laboratories. DNA was stained using DAPI. Confocal imaging was performed on a Zeiss LSM700 confocal microscope and processed using ImageJ and Adobe Illustrator.</p><p>Total RNA from young female samples were extracted using Trizol (Invitrogen) and cDNA was synthesized using Superscript III (Invitrogen). Real time RCR was performed on a Bio-Rad CFX96 detection system. Expression Values were normalized to RP49 expression levels. Primers included: total Xbp1 transcripts (Forward:<named-content content-type="gene">TGGGAGGAGAAAGTGCAAAG</named-content>, Reverse:<named-content content-type="gene">TCCGTTCTGTCTGTCAGCTC</named-content>), Spliced Xbp1 (Forward: <named-content content-type="gene">ACCAACCTTGGATCTGCCG</named-content>, Reverse:<named-content content-type="gene">CGCCAAGCATGTCTTGTAGA</named-content>), Hrd1 (Forward:<named-content content-type="gene">GCAGTTGGTCTTTGGCTTTG</named-content>, Reverse: <named-content content-type="gene">ATGGGCAGCGCGTATATTT</named-content>), RP49(Forward:<named-content content-type="gene">TCCTACCAGCTTCAAGATGAC</named-content>, Reverse:<named-content content-type="gene">CACGTTGTGCACCAGGAACT</named-content>).</p><p>ROS levels were measured as described before <xref ref-type="bibr" rid="pgen.1004568-Hochmuth1">[19]</xref>. Briefly, guts were dissected in Schneider's medium, incubated in 30 &#181;M (Invitrogen) for 5 min at room temperature in the dark, washed twice and mounted to be imaged immediately. GFP expressed under the control of esg::Gal4, Su(H)::Gal80 was used to identify ISCs and/or EBs.</p>