<p>
<italic>S. cerevisiae</italic> cells were grown in YEPD (Sherman et al. 1986) or the appropriate AA-synthetic drop-out media. AA drop-out media is similar to SD media described by Sherman et al. (1986) except that all amino acids, uracil, adenine, Myo-inisitol are 85 &#181;g/mL, except for leucine, which is at 170 &#181;g/mL, and para-aminobenzoic acid and which is at 17 &#181;g/mL. Drop out plates were only missing the noted amino acid. Canavanine was added at 100 &#181;g/ml and cyclohexamide at 5 &#181;g/ml. To identify red colonies on minimal media the adenine was reduced to 20 &#181;g/ml. Amar spore medium is 2% potassium acetate supplemented with 100 &#181;g/ml adenine and uracil, 50 &#181;g/ml histidine, leucine, lysine, tryptophan, methionine and arginine, 35 &#181;g/ml phenylalanine and 10 &#181;g/ml proline, and is designed to support sporulation without growth of the culture.</p><p>All strains used in this study are listed in <xref ref-type="table" rid="pgen-1004910-t001">Table 1</xref>. Strain GRY2691, a <italic>MAT</italic> a parent (<xref ref-type="table" rid="pgen-1004910-t001">Table 1</xref>) was constructed by transformation of GRY1600 with a PvuII fragment from plasmid pMush22 <xref ref-type="bibr" rid="pgen.1004910-Rattray1">[14]</xref> containing <italic>CAN1</italic> and <italic>HIS3</italic> genes transcribing away from one another (<xref ref-type="fig" rid="pgen-1004910-g001">Fig. 1A</xref>) resulting in an insertion of the 3.8 kb cassette 1.85 kb proximal to the <italic>MAT</italic> locus. The sequences of <italic>CAN1</italic> present in the cassette are from &#8722;148 to +1973 relative to the <italic>CAN1</italic> start codon. The strain also harbors a deletion of <italic>CAN1</italic> that excludes sequences from &#8722;151 to +1995 relative to the ATG, thus there is no homology present between the two loci. The <italic>HIS3</italic> insertion includes sequences from &#8722;191 to +857 of the <italic>HIS3</italic> ORF. The <italic>his3-&#916;200</italic> mutation extends from &#8722;205 to +835, thus there are only 22 bp of homology on the 3&#8242; end of <italic>HIS3</italic>, and no homology on the 5&#8242; end. The <italic>MAT&#945;</italic> parent (GRY2690, <xref ref-type="table" rid="pgen-1004910-t001">Table 1</xref>, <xref ref-type="fig" rid="pgen-1004910-g001">Fig. 1B</xref>) was constructed by insertion of a <italic>natMX</italic> cassette from pAG25 <xref ref-type="bibr" rid="pgen.1004910-Goldstein1">[58]</xref> between <italic>FEN1</italic> and <italic>SNM1</italic> on chromosome III and selection on nourseothricin (clonNAT, Werner BioAgents) essentially as described for creating the yeast knockout libraries by providing 45 bp of homology on either side of the cassette to the target locus <xref ref-type="bibr" rid="pgen.1004910-Wach1">[59]</xref>. The <italic>kanMX</italic> cassette was inserted into GRY2690 by PCR of <italic>yih::kanMX</italic> from the yeast <italic>MAT</italic>a knockout collection strain (Open Biosystems) with an additional &#8764;250 bp flanking homology and selection on G418 (Genticin, US Biologicals). To construct the promotorless <italic>can1</italic> gene (<xref ref-type="fig" rid="pgen-1004910-g001">Fig. 1</xref> C) we replaced <italic>HIS3</italic>, the <italic>CAN1</italic> promoter, and the first 6 amino acids of the <italic>CAN1</italic> ORF of pMush22 with a <italic>LEU2</italic> marker by recombineering <xref ref-type="bibr" rid="pgen.1004910-Thomason1">[60]</xref> with a PCR fragment of <italic>LEU2</italic> containing 35 bp of flanking homology to either side of the <italic>HIS3</italic> gene. Once the construct was verified and sequenced, it was inserted into strain GRY2690 by one step transplacement <xref ref-type="bibr" rid="pgen.1004910-Rothstein1">[61]</xref>. Hotspot constructs were made by PCR of the <italic>HIS3 CAN1</italic> and the <italic>LEU2 can1</italic> cassettes with 5&#8242; end- tailed primers containing 50 bp homology on either side to a site between <italic>ARE1</italic> and <italic>BUD23</italic>. The <italic>ARE1</italic> and <italic>BUD23</italic> genes are transcribed away from one another. The insertion did not delete any base pairs, and was positioned so that it was between &#8722;333 of <italic>BUD23</italic> and &#8722;147 of <italic>ARE1</italic>.</p><p>Strains deleted for <italic>SPO11</italic> and <italic>SPO13</italic> were constructed by PCR of the appropriate gene disruption from the yeast knockout collection <xref ref-type="bibr" rid="pgen.1004910-Giaever1">[44]</xref> and subsequent transformation into strains GRY2690 and GRY2691 followed by selection on G418 as described <xref ref-type="bibr" rid="pgen.1004910-Wach1">[59]</xref>. Disruptions were confirmed by PCR, Southern blot and phenotypic analysis where possible. Strains deleted for <italic>rev3</italic> were obtained by one-step transplacement of an Xba1 fragment containing a <italic>rev3::LEU2</italic> disruption from plasmid pAM56 (kindly provided by Alan Morrison). Transformants were selected on media lacking leucine, and further confirmed by PCR and a reduced level of UV induced papillation to canavanine resistance for strains carrying the <italic>HIS3-CAN1</italic> cassette. Strains deleted for <italic>sae2</italic> were PCR amplified from a <italic>sae2::</italic>HygMX mutant strain, which was constructed by marker replacement of the knockout collection <xref ref-type="bibr" rid="pgen.1004910-Giaever1">[44]</xref>, <xref ref-type="bibr" rid="pgen.1004910-Goldstein1">[58]</xref>.</p><p>Tetrads were isolated by patching the various diploid strains on YEPD, then growing up a 5 ml culture in Amar spore media for a minimum of 5 days. Tetrads were treated with zymolyase and dissected onto YEPD. After growth on YEPD, the tetrads were examined for each of the relevant markers by replica plating. Genetic distance was calculated using the Perkins equation (cM&#8202;=&#8202;100X&#8202;=&#8202; (100 (6N+T))/(2(P+N+T)) <xref ref-type="bibr" rid="pgen.1004910-Perkins1">[62]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To analyze the rate of the mutations occurring during mitotic cell divisions we used a Luria-Delbruck fluctuation test <xref ref-type="bibr" rid="pgen.1004910-Luria1">[63]</xref> performed as follows: The relevant haploid strains were mated, and zygotes were isolated by micromanipulation on YEPD. Individual zygote colonies were struck for single cells on YEPD. 9 colonies from each independent zygote colony were inoculated into 5 ml of YEPD. Cells were shaken at 30&#176; overnight and then diluted 1&#8758;50 into 10 ml fresh YEPD and incubated for &#8764;4&#8211;6 hours at 30&#176; to mid-log phase (2&#8211;4&#215;10<sup>7</sup> cells/ml) with shaking (for the <italic>spo13</italic> diploids, we grew 100 ml of culture). After washing cultures in sterile water, half of the cells were removed and used to determine the number of His<sup>+</sup> cells and the number of His<sup>+</sup> Can<sup>r</sup> mutants from each culture. The remaining diploid cells were then resuspended into 5 ml sporulation media and incubated at 30&#176; with shaking until &gt;90% of the cells had sporulated by microscopic examination (&#8764;5 days). Mutation rates during mitosis were estimated by the Ma-Sandri-Sarkar Maximum Likelihood Method <xref ref-type="bibr" rid="pgen.1004910-Sarkar1">[64]</xref>. This is a recursive algorithm that is the product of the probabilities <italic>p<sub>r</sub></italic> for the experimental results <italic>r</italic> (the number of mutants per culture). <italic>m</italic> is the number of mutations per culture, and <italic>c</italic> is the number of cultures. The proportion of cultures with no mutations is <inline-formula><inline-graphic ns0:href="pgen.1004910.e001.jpg" /></inline-formula>, and cultures with 1,2&#8230;<italic>i</italic> mutations are calculated by <inline-formula><inline-graphic ns0:href="pgen.1004910.e002.jpg" /></inline-formula>. The recursive function works as follows:<inline-formula><inline-graphic ns0:href="pgen.1004910.e003.jpg" /></inline-formula> to find the best estimate of <italic>m</italic> from the fluctuation analysis. The mutation frequencies (His<sup>+</sup> Can<sup>r</sup>/His<sup>+</sup>) ranged from &#8764;1&#215;10<sup>&#8722;5</sup> to 1&#215;10<sup>&#8722;8</sup> with majority of cultures falling in the 2&#8211;5&#215;10<sup>&#8722;7</sup> range. To determine the 95% confidence intervals we used the following equations <inline-formula><inline-graphic ns0:href="pgen.1004910.e004.jpg" /></inline-formula> and <inline-formula><inline-graphic ns0:href="pgen.1004910.e005.jpg" /></inline-formula> from Foster <xref ref-type="bibr" rid="pgen.1004910-Foster1">[65]</xref>.</p><p>For determination of mutation rates during meiosis, we used a random spore analysis as described <xref ref-type="bibr" rid="pgen.1004910-Amberg1">[66]</xref>. Briefly, The ascospores were washed twice in 5 ml water and then resuspended in 5 ml water with 0.25 ml 1 mg/ml Zymolyase-100T and 10 &#181;l 2-ME. Cells were incubated overnight at 30&#176;C with gentle shaking. 5 ml 1.5% NP40 (Roche # 11754599001) was added along with 2 ml acid washed glass beads and incubated on a roller drum at room temperature for 2 hours with occasional vigorous vortexing. Disruption of spores was monitored microscopically. This procedure lyses all unsporulated cells. Appropriate dilutions were plated onto media to determine the total number of His<sup>+</sup> and His<sup>+</sup> Can<sup>r</sup> cells. Determination of the meiotic frequency/rate was calculated by subtracting the mitotic frequency from the meiotic frequency for each culture (Meiotic His<sup>+</sup>Can<sup>r</sup>/His<sup>+</sup> per ml minus Mitotic His<sup>+</sup>Can<sup>r</sup>/His<sup>+</sup> per ml). The resulting frequencies were then used to determine a median value reflecting the rate of mutations arising during meiosis, since meiosis involves a single division. For linkage analysis we selected doubly recessive red (<italic>ade2-1</italic>) Cyh<sup>r</sup> His<sup>+</sup> Can<sup>r</sup> random spores to maximize analysis of haploid cells versus cells that mated after plating.</p><p>To identify crossovers among the mitotic mutants we patched &#8764;200 His<sup>+</sup> Can<sup>r</sup> events onto YPD, phenotypically scored the markers, and replica plated them to sporulation media. After 6&#8211;7 days at 30&#176;, spore patches were replica plated to YPD, grown overnight and mated to freshly grown strains GRY633 and GRY634. Diploids were selected on media lacking both histidine and uracil. Because the <italic>HIS3 CAN1</italic> cassette is located near the <italic>MAT</italic>a locus, non-crossovers mate with strain GRY634 (<italic>MAT&#945;</italic>), and give rise to Nat<sup>s</sup> colonies. Crossovers between <italic>natMX</italic> and <italic>HIS3 can1</italic> also only mate with strain GRY634 (<italic>MAT&#945;</italic>) but form Nat<sup>r</sup> progeny. Finally, crossovers between <italic>HIS3 can1</italic> and <italic>MAT</italic> mate with strain GRY633 (<italic>MAT</italic>a) resulting in Nat<sup>s</sup> colonies. Double crossovers between the same chromatids (of which none were detected) would mate with GRY633 and become Nat<sup>r</sup>. No crossovers between <italic>HIS3</italic> and <italic>MAT</italic> were observed among the mitotic diploids analyzed. A similar scheme was used to analyze for mitotic crossover events for the strains with the construct in the hotspot except that the <italic>kanMX</italic> marker was also scored.</p><p>After plating for random spores we colony purified 300&#8211;600 red His<sup>+</sup> Can<sup>r</sup> Cyh<sup>r</sup> spore clones from each strain onto media lacking Histidine. These were then retested for Can<sup>r</sup>, patched and replica plated to test the relevant markers and determine mating type (by crossing with strains DC14 and DC17). Approximately 50 of the spore colonies from each strain that presented evidence of a crossover were mated to GRY1600 or GRY1601 (depending on mating type) and 5&#8211;10 tetrads were dissected from each to further confirm linkage.</p><p>The entire <italic>CAN1</italic> gene was PCR amplified from mutant candidates and sequenced on both strands. Three polymorphisms were noted in our <italic>CAN1</italic> gene as compared to the published S288c <italic>CAN1</italic> sequence. The sequence of the mutations identified in mitosis or meiosis are listed in <xref ref-type="supplementary-material" rid="pgen.1004910.s001">Supplemental Tables S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1004910.s002">S2</xref>, respectively. Sequencing was done by the Laboratory of Molecular Technologies Sequencing Facility- SAIC-Frederick, FNLCR.</p><p>DNA isolation and Southern blots were carried out as described by Sun et al. <xref ref-type="bibr" rid="pgen.1004910-Sun1">[67]</xref>. For the hotspot analysis DNA was digested with AatII and run on a 0.7% agarose gel. For the coldspot analysis DNA was digested with PvuII and run on a 1% agarose gel. <sup>32</sup>P-dCTP labeled probes were made with the Agilent Prime it II kit according to manufacturers instructions. Hotspot probe used a PCR fragment spanning coordinates 213283&#8211;213848 of Chromosme III. Coldspot probes used a PCR fragment spanning 195735&#8211;196774 of Chromosome III.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Statistical analyses were calculated by the chi square test for tetrads and random spores: in tetrads we compared PD, NPD and TT between the various strains, for random spore analysis we compared the number of crossovers in each interval for each strain. For distribution analysis along the length of <italic>CAN1</italic> the gene was divided into 200 bp windows as described previously <xref ref-type="bibr" rid="pgen.1004910-Rattray2">[25]</xref>. For comparison between mutation rates from mitotic growth and from meiosis we used a student's <italic>t</italic> test (<ext-link ext-link-type="uri" ns0:href="http://en.wikipedia.org/wiki/Student's_t-test">http://en.wikipedia.org/wiki/Student's_t-test</ext-link>).</p>