<p>Strains were acquired as desiccated cells from the American Type Culture Collection (ATCC) in Manassas, Virginia, USA; the Leibniz Institute DSMZ German Collection of Microorganisms and Cell Cultures (DSM) in Braunschweig, Germany; and the Japan Collection of Microorganisms (JCM) in Ibaraki, Japan, as indicated in <xref ref-type="supplementary-material" rid="pgen.1004784.s025">Table S2</xref>. Cells were rehydrated in recommended media according to culture collection center protocols and grown to stationary phase at 37&#176;C in liquid culture. Genomic DNA was harvested with Wizard Genomic DNA purification kit (Promega).</p><p>Sequence libraries were constructed using a combination of standard fragmentation (200&#8211;500 bp), mate-pair fragmentation (6 Kbp), and PCR-free transposon-mediated insertion of sequencing primers (Epicentre Nextera) <xref ref-type="bibr" rid="pgen.1004784-Adey1">[81]</xref>. Transposase was purified from E. coli BL21 (DE3) containing the cloning vector pWH1891. For mate-pair libraries, 6 Kb pair-end libraries were constructed and the terminal 50 bases of each end were sequenced, according to standard protocols. Additional libraries were constructed by combining DNA fragments from haloarchaeal species distantly enough related to enable unambiguous assignment of non-barcoded reads (&#8220;SOUP&#8221;). All sequencing was performed on Illumina HiSeq and GAII platforms. The paired-end information and trimming information were specified using annotation strings on the description line of the reads. Reads were assembled using the a5 pipeline <xref ref-type="bibr" rid="pgen.1004784-Tritt1">[18]</xref>. Following assembly, genomic DNA contamination arising from transpose-mediated library preparation was removed by searching assembled reads against a local BLAST database consisting of E. coli BL21 (DE3) genomic DNA and the cloning vector pWH1891. BLAST hits with an E-value &#8804; 10&#8722;20 were considered to be significant matches and candidates for contamination. Contigs with matches covering &#8805;80% of the contig length and contigs &#8804;1 Kbp with matches covering any portion of the contig were treated as contamination and discarded from further analysis. Long contigs for which only a small portion of the contig matched to the local BLAST database were also discarded if there were either no annotated features, or if the annotated features were E. coli genes. These criteria resulted in a total of 497 contigs equaling 265.57 Kbp being removed from the final assemblies.</p><p>A dual annotation pipeline was implemented in order to take advantage of the strengths of different existing automated annotation tools. Assembled genomes were first submitted to the Rapid Annotation using Subsystem Technology (RAST) server at the National Microbial Pathogen Data Resource. RAST-based gene calls and annotations were used for building of protein families, core genome and pan genome analyses, phylogenetic reconstruction, analysis of general transcription factor expansions, GC-bias analysis, building of molecular marker sets and phylogenetically informed re-annotation. These annotations can be accessed as a loadable &#8220;popular genome set&#8221; through the genome context viewer JContextExplorer <xref ref-type="bibr" rid="pgen.1004784-Seitzer1">[17]</xref> as well as a custom MySQL database (see <xref ref-type="supplementary-material" rid="pgen.1004784.s049">Text S1</xref> for instructions). The RAST annotation system was particularly useful in enabling comparison of our genomes with previously sequenced haloarchaea, by allowing standardization via rapid reannotation of existing genomes.</p><p>In addition, the newly sequenced genomes were annotated using NCBI's Prokaryotic Genome Annotation Pipeline (PGAAP) <xref ref-type="bibr" rid="pgen.1004784-NCBI1">[14]</xref>. PGAAP gene calls and annotations were used for COG analysis, proteome acidification calculations, and genera-based genomic feature comparisons. These annotations can be accessed through the NCBI website using the accession numbers listed in <xref ref-type="supplementary-material" rid="pgen.1004784.s025">Table S2</xref> as well as through our custom MySQL database.</p><p>A phylogeny was constructed for all archaeal genomes available through the Integrated Microbial Genomes database along with our sequenced haloarchaea using a concatenated set of 40 conserved marker genes <xref ref-type="bibr" rid="pgen.1004784-Wu1">[20]</xref>. Peptide sequences were downloaded for all archaeal genomes from the IMG 4.0 database on January 4, 2013. HMM profiles of 40 bacterial and archaeal PhyEco markers were searched against these peptide sequences. We excluded genomes with less than 35 of these markers, as well as duplicate genomes which were included in both our analysis and the IMG database. A total of 151 IMG archaeal genomes and all 80 haloarchaeal genomes were included in the phylogenetic tree building. For each PhyEco marker family, only single-copy members from the genomes were included. Independent alignments were built using MUSCLE <xref ref-type="bibr" rid="pgen.1004784-Edgar1">[82]</xref> for each gene families and then concatenated. A phylogenetic tree was built from the concatenated alignment using PHYML 3.0 <xref ref-type="bibr" rid="pgen.1004784-Guindon1">[83]</xref> with the LG substitution model. Tree topology and branch lengths were optimized by the program and aLRT SH-like statistics was used for branch support estimation. The R package ape was used to remove duplicate genomes <xref ref-type="bibr" rid="pgen.1004784-Paradis1">[84]</xref> and the final tree was visualized using FigTree <xref ref-type="bibr" rid="pgen.1004784-Rambaut1">[85]</xref>. The tree files are available as <xref ref-type="supplementary-material" rid="pgen.1004784.s032">Dataset S3</xref> (all archaea) and <xref ref-type="supplementary-material" rid="pgen.1004784.s040">Dataset S11</xref> (haloarchaea only).</p><p>Homologous protein families were constructed using a Markov clustering-based approach. First, an all-vs-all BLASTp search of all protein coding genes called by RAST in the 80 haloarchaeal genomes was conducted using an E-value cutoff of 10<sup>&#8722;5</sup>, soft masking, and the Smith-Waterman alignment algorithm. These options have been shown to improve homolog detection over other BLAST methods <xref ref-type="bibr" rid="pgen.1004784-MorenoHagelsieb1">[86]</xref>. To enable detection of large homology groups, the number of returned matches was set at 100,000. To remove spurious, domain-level matches, BLAST results were post-filtered and matches retained only if bi-directional query-match coverage was &#8805;80%, and each sequence was &#8805;75% of the length of the other. Matches with E-values&gt;10<sup>&#8722;10</sup> were also excluded at this stage (<xref ref-type="supplementary-material" rid="pgen.1004784.s050">Text S2</xref>).</p><p>BLAST results were then clustered into homology groups using the clustering algorithm TRIBE-MCL <xref ref-type="bibr" rid="pgen.1004784-Enright1">[16]</xref>, which utilizes the protein-protein similarity network implicit in the BLAST scores. This method has the benefit of building inclusive homology families including orthologs, xenologs and both in- and out-paralogs, unlike reciprocal best BLAST methods which often overlook complex gene relationships in the search for one-to-one ortholog mapping. To determine an appropriate inflation parameter for TRIBE-MCL clustering, homology clusters derived from inflation parameters ranging from 1.4 to 8.0 were compared and benchmarked using manually curated protein families for haloarchaeal TATA-binding proteins and opsins. The inflation parameter I&#8202;=&#8202;2.5 was selected, as it most closely recapitulated known ortholog/paralog relationships for the selected protein families. A total of 17,591 homology clusters (protein families) were obtained with this method, comprising 276,364 of 303,129 total haloarchaeal proteins. The remaining proteins were singletons, lacking significant sequence similarity to other sequences in the dataset. These sequences may represent species-specific innovation, recent gene influx from horizontal gene transfer, or simply a lack of sequencing depth along some sampled haloarchaeal lineages.</p><p>Random subsets of haloarchaeal genomes (ranging from zero to eight genomes) were selected from the set of RAST-annotated haloarchaea, using the Java Samplers package <xref ref-type="bibr" rid="pgen.1004784-Heinrich1">[87]</xref>. For each selection the following quantities were tabulated: (1) the number of unique COGs, (2) the number of unique TRIBE-MCL determined homology clusters, and (3) the number of unique non-singleton TRIBE-MCL determined homology clusters. Ten thousand random selections were performed for each sample size and the average number of unique protein families obtained for each sample were plotted in Microsoft Excel (<xref ref-type="fig" rid="pgen-1004784-g003">Figure 3</xref>).</p><p>The 1,479 protein families used in the analysis described in <xref ref-type="bibr" rid="pgen.1004784-NelsonSathi1">[25]</xref> were obtained from the authors. All genes were renamed with an &#8220;E&#8221;, &#8220;A&#8221;, or &#8220;H&#8221; tag prepended to their original NCBI gene ID number, according to their taxonomic classification as, respectively, (eu)bacteria, archaea, or haloarchaea. Proteins from 65 haloarchaeal species not included in the original study were assembled into a set of &#8220;additional haloarchaeal proteins&#8221;, and named with an &#8220;H&#8221; tag and unique ID number. Five haloarchaeal genomes in our set were either different strains of a species included in the original study, or a re-sequencing of the same strain, and so were excluded from re-analysis. Gene trees were re-constructed for all 1,479 of the original protein families, using the alignment program MAFFT <xref ref-type="bibr" rid="pgen.1004784-Katoh1">[88]</xref>, with command line options &#8211;legacygappenalty, &#8211;anysymbol, and &#8211;quiet; the protein model determination tool ProtTest <xref ref-type="bibr" rid="pgen.1004784-Darriba1">[89]</xref>, using command line options &#8211;all-matrices, all-distributions, -S 2 &#8211;F &#8211;t1; and the gene tree creation tool PhyML <xref ref-type="bibr" rid="pgen.1004784-Guindon1">[83]</xref>. All command line options were obtained from the authors of <xref ref-type="bibr" rid="pgen.1004784-NelsonSathi1">[25]</xref>). Gene trees were determined to be single transfers from the bacteria only if (1) all haloarchaeal homologs formed a monophyletic group and (2) this monophyletic group rooted with bacterial rather than archaeal homologs. For gene trees matching these criteria, corresponding protein families were extended to include homologs predicted from the set of &#8220;additional haloarchaeal proteins&#8221;, using BLAST with an E-value cutoff of 10<sup>&#8722;10</sup> and a minimum percent identity of 30%. Gene trees were generated for these extended protein families using the parameters described above. Extended gene trees were re-evaluated according to the criteria described above to determine whether they could still be classified as single bacterial transfers. Some (17.2%) of the extended protein families contained a sufficiently large number of members such that determination of gene trees was infeasible, and so were excluded from re-analysis. Gene tree classification results are provided in <xref ref-type="table" rid="pgen-1004784-t001">Table 1</xref>. Phylogenetic trees visualized in <xref ref-type="supplementary-material" rid="pgen.1004784.s004">Figures S4</xref> and <xref ref-type="supplementary-material" rid="pgen.1004784.s003">S3</xref> were made using Phyfi <xref ref-type="bibr" rid="pgen.1004784-Fredslund1">[90]</xref>.</p><p>A list of 74 genes involved in ion transport and compatible solute transport or synthesis was compiled. Representative protein sequences for each of the selected genes were obtained by searching the NCBI protein database by annotation (<xref ref-type="supplementary-material" rid="pgen.1004784.s041">Dataset S12</xref>). When available, sequences from the model organisms <italic>Escherichia coli</italic> and <italic>Bacillus subtilis</italic> were selected. Genes with &#8220;MM&#8221; prefix derive from <italic>Methanosarcina mazei</italic> Go1. BLASTp searches for these query sequences were conducted against a local BLAST database containing 80 haloarchaeal genomes. In order to obtain all potential homologs, the maximum target sequences parameter was set to 100,000 and an E-value cutoff of 1 was used. Resulting matches were then filtered using custom perl scripts to remove spurious domain-level matches by requiring a bidirectional query-target coverage of &#8805;80%, and a bidirectional query-target length of &#8805;75%. To reduce the number of non-specific family-level matches (for example, to differentiate between the closely related Trk H<sup>+</sup>/K<sup>+</sup> and Ktr Na<sup>+</sup>/K<sup>+</sup> symporters), sequences were also filtered to remove matches with an E-value of&gt;1e<sup>&#8722;20</sup>. A presence/absence matrix was constructed from these filtered BLAST results using custom bash and R scripts, and visualized in iTOL <xref ref-type="bibr" rid="pgen.1004784-Letunic1">[91]</xref>, <xref ref-type="bibr" rid="pgen.1004784-Letunic2">[92]</xref>. Genes were grouped into functional categories based on substrate specificity and directionality for ion transporters, substrate imported for compatible solute transporters, or product for compatible solute biosynthesis proteins. As 20 of the query genes were not detected in the set of investigated species, only 54 columns appear in <xref ref-type="supplementary-material" rid="pgen.1004784.s005">Figure S5</xref>. Gene presence/absence data was scaffold on the multi-marker concatenated haloarchaeal phylogeny (<xref ref-type="supplementary-material" rid="pgen.1004784.s040">Dataset S11</xref>).</p><p>Whole-genome feature data for the 59 newly sequenced, PGAAP-annotated haloarchaea were retrieved from an in-house MySQL database using custom scripts <xref ref-type="bibr" rid="pgen.1004784-Becker1">[67]</xref>. For each organism the genome size, number of contigs, percent G+C content, percent coding, number of coding regions, and number of insertion sequence elements were extracted. Genomes were organized into genera and box plots for each of the above traits were drawn for each genus using MATLAB's Statistics toolbox <xref ref-type="bibr" rid="pgen.1004784-MATLAB1">[93]</xref>. Images were exported using the supplementary export_fig function <xref ref-type="bibr" rid="pgen.1004784-Woodward1">[94]</xref>. See <xref ref-type="supplementary-material" rid="pgen.1004784.s023">Figure S23</xref>.</p><p>All isoelectric point analyses were carried out exclusively on the PGAAP gene call set. The pI of every protein coding sequence was computationally predicted using the protein sequence isoelectric point prediction method included in the BioJava framework <xref ref-type="bibr" rid="pgen.1004784-Holland1">[95]</xref>. pI histograms for each genome were computed using MATLAB's statistics toolbox <xref ref-type="bibr" rid="pgen.1004784-MATLAB1">[93]</xref>, sorting proteins into 100 equally spaced bins ranging from a pI of 2.0 to 13.0. All proteins with a pI of 7.5 or greater were collected and a list of all represented gene annotations was procured. For each gene annotation in this list, all instances were tabulated, including the number of instances with a pI greater than or equal to 7.5. A table showing these results, sorted by frequency of instances with high pI can be found in <xref ref-type="supplementary-material" rid="pgen.1004784.s028">Table S5</xref>. All proteins annotated as ribosomal subunits were collected, and all subunits which could confidently be mapped to experimentally characterized structures for the large (1QVG) <xref ref-type="bibr" rid="pgen.1004784-Schmeing1">[96]</xref> and small (1FKA) <xref ref-type="bibr" rid="pgen.1004784-Schluenzen1">[97]</xref> ribosomal subunits were mapped. Subunits with a pI&gt;7.5 in 60% or more of cases were colored red, and subunits with a pI&lt;7.5 in 60% or more of cases were colored blue. Any subunits which could not be confidently mapped or which had variable pI across species were left the default color of green. All mappings were done using the PyMOL protein visualization program <xref ref-type="bibr" rid="pgen.1004784-The1">[98]</xref>.</p><p>Comparative acidification visualizations of TBP and ribosome elongation factor &#945;-1 proteins for haloarchaea and non-halophilic organisms were generated using the SWISS-MODEL web interface <xref ref-type="bibr" rid="pgen.1004784-Arnold1">[99]</xref> and 1D3U <xref ref-type="bibr" rid="pgen.1004784-Littlefield1">[100]</xref> and 3VMF <xref ref-type="bibr" rid="pgen.1004784-Kobayashi1">[101]</xref> as the structural templates, respectively.</p><p>To investigate localized variations in G+C content across each genome, custom scripts were created (<xref ref-type="supplementary-material" rid="pgen.1004784.s051">Text S3</xref> &amp; <xref ref-type="supplementary-material" rid="pgen.1004784.s052">S4</xref>). First, %G+C was calculated for 100 bp windows across the genome, with a 20 bp step size. The terminal &lt;100 bps of each contig were not included in calculation, so as not to artificially inflate the final G+C calculation for each contig. Windows with&gt;10% ambiguous nucleotides were assumed to have %G+C equal to the contig mean. Plots were generated showing a) the overall mean G+C percent for the entire genome as a horizontal green line, b) the contig boundaries as vertical red lines, c) the mean G+C percent for each contig as horizontal blue lines, and d) all G+C percentages for individual 100 bp segments as a black line <xref ref-type="bibr" rid="pgen.1004784-Becker1">[67]</xref>. Steps where the mean %G+C inflects away from the local mean were calculated using the R changepoint package <xref ref-type="bibr" rid="pgen.1004784-Killick1">[102]</xref>, and these changepoints were plotted over the stepwise G+C percentages as vertical green lines <xref ref-type="bibr" rid="pgen.1004784-Becker1">[67]</xref>. After manual curation of changepoints for each species, annotated features between changepoints were retrieved from each species' general feature format (GFF) file. Ten species were excluded from this analysis, either due to having a very large number of contigs, or having no observable inflections in local mean G+C content. For a list of excluded organisms and reasons, see <xref ref-type="supplementary-material" rid="pgen.1004784.s049">Text S1</xref>. The set of features extracted from regions of abnormal G+C content were analyzed for enrichment of homologous protein families (TRIBE-MCL clusters). The frequency of each protein family in the abnormal G+C feature set was compared to its frequency in the entire genome set, and families with at least eight-fold enrichment were investigated further. To avoid including families which were enriched artificially by having a small number of members which are all present in abnormal G+C regions, families with fewer than five members were excluded from enrichment analysis.</p><p>A local BLAST database was constructed using RAST-derived gene calls for the 80 haloarchaea. This database, and the NCBI non-redundant protein database (as of October 30<sup>th</sup>, 2012), were queried for homologs to the archaeal and eukaryotic general transcription factors TATA-binding protein (TBP) and transcription factor B/IIB (TFB) using a query set of 19 curated TBP homologs and 27 curated TFB homologs from six archaeal and one eukaryotic species (<xref ref-type="supplementary-material" rid="pgen.1004784.s047">Datasets S18</xref> &amp; <xref ref-type="supplementary-material" rid="pgen.1004784.s048">S19</xref>). A BLASTp search was conducted (BLAST+2.2.27) using a maximum expect value of 10<sup>&#8722;5</sup> and maximum target sequence of 100,000. For each set of homologs, a multi-sequence alignment was constructed using MUSCLE v3.8.31 <xref ref-type="bibr" rid="pgen.1004784-Edgar1">[82]</xref> and manually curated to remove poorly aligning regions. Highly divergent and fragmentary sequences were manually removed. One thousand bootstrapped alignments were created by resampling the curated alignment using Seqboot in the Phylip toolkit (v3.69) <xref ref-type="bibr" rid="pgen.1004784-Felsenstein1">[103]</xref>. Phylogenetic trees were created for each bootstrapped alignment using FastTree v2.1.5 SSE3 <xref ref-type="bibr" rid="pgen.1004784-Price1">[104]</xref> and a consensus tree was produced by comparing these trees to a guide tree, also constructed using FastTree, using the tree comparison tool CompareToBootstrap.pl <xref ref-type="bibr" rid="pgen.1004784-Price2">[105]</xref>. For all clades in the guide tree, the fraction of bootstrapped trees in which that clade appeared were tabulated and recorded as the bootstrap support values for that clade. Because this approach is limited in that only clades appearing in the initial guide tree are considered, we also constructed a consensus tree with the Consense program in the Phylip package <xref ref-type="bibr" rid="pgen.1004784-Price2">[105]</xref> using the extended majority rule option. Each clade in the resulting consensus tree represents the most frequent grouping of those species in the 1000 bootstrap replicate trees, independent of any guide tree. For Consense tree files, see Datasets S20&#8211;S23. Branch lengths and node labels in Consense tree represent the number of bootstrapped tree replicates for which each clade was observed, rather than sequence divergence. Trees were visualized in FigTree <xref ref-type="bibr" rid="pgen.1004784-Rambaut1">[85]</xref>.</p><p>Marker genes were identified by the following procedure: Homology clusters were identified that were (1) universal to the genus of interest, (2) not found in any other haloarchaeal species, and (3) single copy. Homologs were aligned using MUSCLE <xref ref-type="bibr" rid="pgen.1004784-Edgar1">[82]</xref> with default parameters, and a hidden Markov model was generated from each alignment using HMMer <xref ref-type="bibr" rid="pgen.1004784-Eddy1">[106]</xref> with default parameters. Additionally, a list of marker homology clusters was generated, as were the genomic coordinates for each marker gene <xref ref-type="bibr" rid="pgen.1004784-Becker1">[67]</xref>. This list may be loaded into JContextExplorer <xref ref-type="bibr" rid="pgen.1004784-Seitzer1">[17]</xref> as a custom context set.</p><p>Putative missed gene calls were identified using JContextExplorer <xref ref-type="bibr" rid="pgen.1004784-Seitzer1">[17]</xref>, with the Haloarchaea genome set loaded. To retrieve the context surrounding Tribe4688 (with gene calls in 12 <italic>Haloferax</italic> species) a &#8220;Between&#8221; context set was created with a 5 Kbp limit, and the query &#8220;966; 458&#8221; was carried out under &#8220;Cluster Search&#8221;. Regions were visualized by selecting all instances and pushing the &#8220;View Contexts&#8221; button. This produced a set of either two or three gene groupings, in the order Tribe458 (always) &#8211; Tribe4688 (sometimes) &#8211; Tribe966 (always). The sequence between instances of Tribe458 and Tribe966 in the three <italic>Haloferax</italic> species that did not contain an instance of Tribe4688 was extracted, and every contiguous region of 117 nucleotides was translated into a protein using MATLAB's bioinformatics toolbox <xref ref-type="bibr" rid="pgen.1004784-MATLAB2">[107]</xref>. Each match was manually compared to the gene members of protein sequences of Tribe4688, and three matches were identified, one in each of the three organisms lacking an instance of Tribe4688 (<italic>Haloferax alexandrinus, Hfx. elongans,</italic> and <italic>Hfx. sulfurifontis</italic>). The sequence alignments were visualized, with amino acids colored according to the properties of their side chains. To retrieve the region of poor gene call consistency in <italic>Haloarcula</italic>, the same between context set was used with a 5 Kbp limit, with the query &#8220;254; 369&#8221; under &#8220;Cluster Search&#8221;. Genomic segments were visualized by selecting all instances and pushing the &#8220;View Contexts&#8221; button.</p><p>A presence/absence matrix was constructed for each TRIBE-MCL protein family across the 80 haloarchaea and transformed into a distance matrix using Euclidean distances. Hierarchical clustering of this distance matrix was performed using Mev <xref ref-type="bibr" rid="pgen.1004784-Eisen1">[108]</xref>, and the resulting matrix was visualized. Groups of protein families which clustered together have a similar phylogenetic dispersal pattern, and may therefore perform functionally related tasks. This distribution was manually interrogated to find groups of clustered protein families with functionally related annotations. Unannotated protein families in that cluster were then hypothesized to also be involved in that task.</p>