<p xmlns:ns0="http://www.w3.org/1999/xlink">All shown experiments used <italic>E. coli</italic> strain MG1655 and its derivatives. Genetic deficiencies were introduced into MG1655 by P1 transduction using P1<italic>virA</italic>. The <italic>optA1</italic> allele of <italic>dgt</italic> was introduced linked to transposon <italic>zad-220</italic>::Tn<italic>10</italic> as described <xref ref-type="bibr" rid="pgen.1004310-Itsko1">[43]</xref>. The <italic>gpt</italic>::<italic>kan</italic> allele was obtained from the National BioResource Project (NIG) of Japan (<ext-link ext-link-type="uri" ns0:href="http://www.shigen.nig.ac.jp/ecoli/strain/top/top.jsp">http://www.shigen.nig.ac.jp/ecoli/strain/top/top.jsp</ext-link>). The p<italic>urR</italic>::<italic>cat</italic>, <italic>recA</italic>::<italic>cat</italic> and <italic>sulA</italic>::<italic>cat</italic> mutants were generated by the method of Datsenko and Wanner <xref ref-type="bibr" rid="pgen.1004310-Datsenko1">[47]</xref> using primers described in <xref ref-type="table" rid="pgen-1004310-t001">Table 1</xref>. For testing SOS induction, the relevant strains were transformed with plasmid pSK1002, which contains the <italic>lacZ</italic> reporter gene fused to the <italic>umuDC</italic> promoter <xref ref-type="bibr" rid="pgen.1004310-Shinagawa1">[24]</xref>.</p><p>For strain construction, maintenance, and determination of viable counts, LB medium was used with supplementation of the following antibiotics, where appropriate: tetracycline (15 &#181;g/ml) for <italic>optA1</italic> linked with <italic>zad-22</italic>::Tn<italic>10</italic>, kanamycin (50 &#181;g/ml) for <italic>gpt</italic>::<italic>kan</italic>, chloramphenicol (25 &#181;g/ml) for the <italic>purR</italic>::<italic>cat</italic>, <italic>recA</italic>::<italic>cat</italic> and <italic>sulA</italic>::<italic>cat</italic> alleles, and ampicilin (100 &#181;g/ml) for pSK1002 transformants. For experiments relating to starvation, cells were grown at 37&#176;C in minimal medium containing Vogel-Bonner salts <xref ref-type="bibr" rid="pgen.1004310-Vogel1">[48]</xref> containing glucose (0.4%), casamino acids (1%) (Becton-Dickinson), D-pantothenic acid (5 &#181;M), and hypoxanthine (50 &#181;g/ml). To assay the differential responses in media with or without purine source, two aliquots were filtered through a 25, 47 or 90-mm diameter polycarbonate membrane filter (0.4 &#181;m pore size; Millipore) and diluted up to 10-fold in the identical medium with or without hypoxanthine (50 &#181;g/ml). Aliquots for different assays were withdrawn at densities not exceeding 0.2 OD<sub>630 nm</sub>.</p><p>Culture aliquots (300&#8211;350 ml) harvested at OD<sub>630 nm</sub>&#8202;=&#8202;0.2 were filtered through a 90-mm diameter polycarbonate membrane filter (0.4 &#181;m pore size; Millipore). The filter was transferred to a Petri dish lid containing 10 ml of 60% aqueous methanol at <bold>&#8722;</bold>20<bold>&#176;</bold>C. After 2 h at <bold>&#8722;</bold>20<bold>&#176;</bold>C the filter was removed and the liquid suspension boiled for 5 min, followed by centrifugation for 15 min at 17,000&#215; g and lyophilization of the supernatant. The residue was dissolved in 1 ml of sterile water, filtered through syringe filter (Millipore, 0.22 &#181;m pore size) and lyophilized again. The final residue was dissolved 50 &#181;l sterile water. HPLC analysis of the extracted dNTPs was performed by reversed-phase chromatography on an Agilent 1100 high-pressure liquid chromatography instrument with UV detection at 254 nm. Nucleotides were separated on a Zorbax Eclipse XDBC18 3.5 &#181;M (150 by 4.6 mm) column equipped with a Zorbax Eclipse XDBC18 guard column, adapting a prior method used for the separation of nucleotides <xref ref-type="bibr" rid="pgen.1004310-Ahluwalia1">[38]</xref>. At a flow rate of 0.8 ml/min, a linear gradient of 70&#8758;30 buffer A to buffer B was run to 40&#8758;60 over 30 min. The gradient was then changed over 60 min from 40&#8758;60&#8758;0 to 0&#8758;87.5&#8758;12.5 for buffer A - buffer B - buffer C. To wash the column between samples the gradient was first changed from 0&#8758;87.5&#8758;12.5 to 0&#8758;70&#8758;30 over 10 minutes with a final stepwise change to 70&#8758;30&#8758;0 for an additional 20 min.</p><p>In a later set of experiments aimed at quantifying specifically dGTP during an extended starvation time course (<xref ref-type="fig" rid="pgen-1004310-g004">Fig. 4D</xref>), a modified protocol was used, as follows. At a flow rate of 1 ml/min, a linear gradient of 75&#8758;25 buffer A to buffer B was changed to 52&#8758;48 over 23 min. The gradient was then changed over 12 min from 52&#8758;48 to 49&#8758;51 and for an additional 10 min from 49&#8758;51 to 40&#8758;60. To wash the column between samples the gradient was first changed from 40&#8758;60&#8758;0 to 0&#8758;77.5&#8758;22.5 for buffer A - buffer B - buffer C over 15 minutes and for an additional 10 min from 0&#8758;77.5&#8758;22.5 to 0&#8758;70&#8758;30 with a final stepwise change to 70&#8758;30&#8758;0 for an additional 10 min.</p><p>Buffer A consisted of 5 mM tetrabutyl ammonium phosphate (PicA Reagent; Waters), 10 mM KH<sub>2</sub>PO<sub>4</sub>, and 0.25% methanol adjusted to pH 6.9. Buffer B consisted of 5 mM tetrabutyl ammonium phosphate, 50 mM KH<sub>2</sub>PO<sub>4</sub>, and 30% methanol (pH 7.0). Buffer C was acetonitrile. Nucleotide standards were obtained from Sigma.</p><p>For chromosomal DNA extraction, 7- or 13-ml culture aliquots were harvested at various time points into the same volume of ice-cold PBS solution containing 20 mM NaN<sub>3</sub>. DNA extraction was performed with the Easy-DNA kit (Invitrogen) and quantitated by staining with Picogreen (Invitrogen). For determination of run-out DNA synthesis (<xref ref-type="fig" rid="pgen-1004310-g003">Fig. 3D</xref>), rifampicin was added at time zero at a concentration 300 &#181;g/ml. For ori/ter determination, the DNA was digested with <italic>EcoR</italic>I and subjected to Quantitative PCR (Stratagene Mx 3000) with SIBR Green detection using primers (<xref ref-type="table" rid="pgen-1004310-t002">Table 2</xref>) specific to the origin and the terminus regions of the <italic>E. coli</italic> chromosome <xref ref-type="bibr" rid="pgen.1004310-Stokke1">[49]</xref>.</p><p>For determination of the DNA synthesis rate by pulse labeling, <italic>optA1 gpt</italic> cultures were grown with hypoxanthine to OD<sub>630 nm</sub> 0.1, filtered, resuspended in the identical media with or without hypoxanthine, and brought to the same turbidity. Every 10&#8211;15 minutes 0.5 ml samples were withdrawn and pulse-labeled with 1 &#181;Ci of [methyl-<sup>3</sup>H]-thymidine at specific activity 0.5 &#181;Ci/nmole for 3 minutes. Samples were quenched with 0.5 ml of cold trichloroacetic acid (TCA 10%) containing 500 &#181;g/ml of unlabeled thymidine to a final concentration 5% TCA and 250 &#181;g/ml of unlabeled thymidine and kept on ice bath at least for 30 minutes. The entire samples were then collected on pre-wet 25-mm glass microfibre filters (Whatman), and washed with cold TCA (5% with 250 &#181;g/ml of unlabeled thymidine) and 100% ethanol. The radioactivity on the filters was determined in a LS6500 liquid scintillation counter (Beckman) with Ecolume liquid scintillation cocktail (MP Biomedicals).</p><p>Aliquots of growing cultures were fixed with 0.25% formaldehyde and stained with DAPI. The cells were visualized by Nomarsky and DAPI fluorescence microscopy (NIKON eclipse E600) and photographed using a Micropublisher CCD color Camera (QImaging). Live/Dead stain (Invitrogen) was used to test viability of the cells.</p><p>5.0-ml samples were removed, and the cells were pelleted in a microcentrifuge. &#946;-Galactosidase assays were performed essentially as described by Miller <xref ref-type="bibr" rid="pgen.1004310-Miller1">[18]</xref>. Cell pellets were resuspended in 1 ml of Z-buffer (60 mM Na<sub>2</sub>HPO<sub>4</sub>, 40 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM KCl, 1 mM MgSO<sub>4</sub>, 10 mM dithiothreitol). 80 &#181;l of chloroform and 40 &#181;l of 0.1% sodium dodecyl sulfate (SDS) were added to the cell suspension, which was then vortexed vigorously for 10 s. To start the reactions, 200 &#181;l of ONPG (4 mM) was added, and the reaction mixtures were incubated at 30&#176;C for 4.5 min. The reactions were stopped with 0.5 ml of 1 M sodium bicarbonate, and the cellular debris was pelleted. The optical density was recorded with a BECKMAN DU 640 spectrophotometer with a 405-nm filter. Miller units were calculated as follows: units&#8202;=&#8202;1,000[(OD<sub>405</sub>/(t&#215;v&#215;OD<sub>630 nm</sub>)], where OD<sub>405 nm</sub> denotes the optical density at 405 nm. OD<sub>630 nm</sub> reflects the cell density at 630 nm, t is the reaction time in minutes, and v is the volume of culture used in the assay.</p><p>To test whether suppression of the sensitivity of the <italic>optA1 gpt</italic> strains to hypoxanthine deprivation resulted from loss of the <italic>optA1</italic> allele, a large number of clones that survived on the starvation plates at 42&#176;C (see <xref ref-type="fig" rid="pgen-1004310-g006">Fig. 6C</xref>) were tested for their ability to support growth of the bacteriophage T4 <italic>tsL141</italic> mutant <xref ref-type="bibr" rid="pgen.1004310-Muzyczka1">[50]</xref>, as described <xref ref-type="bibr" rid="pgen.1004310-Itsko1">[43]</xref>. Clones that restrict growth of this phage at 30&#176;C are <italic>optA1</italic>
<xref ref-type="bibr" rid="pgen.1004310-Gauss1">[51]</xref>. Wild-type phage T4D was used as a positive control. The T4 phages were obtained from Dr J.W. Drake, NIEHS.</p>