<p>Potassium phosphate, potassium chloride, magnesium acetate, EDTA, and glycerol were purchased from Fisher. Ammonium sulfate was purchased from MP biomedicals. Dithiothreitol (DTT) was purchased from Research Organics. Tris base was purchased from RPI. Restriction endonucleases and the 2-log DNA molecular weight marker were purchased from New England Biolabs. Chromatography resins were purchased from GE Healthcare except for the ceramic hydroxyapatite, which was purchased from Bio-Rad. A photo-reactive p-benzoyl-phenylalanine, Bpa, was purchased from Bachem. All other chemicals and reagents were purchased from Sigma unless otherwise noted.</p><p>
<bold>pEAW242</bold> is <italic>recA</italic> S117F in the overproduction vector pET21d (Novagen). pAIR79 was used as a PCR template with primers consisting of bases 427 to 410 of pET21d and a second primer consisting of bases 370 to 341 of the <italic>recA</italic> gene, with the G at base 353 changed to A to make the complement of <italic>recA</italic> S117F. The PCR product was digested with SgrAI and inserted into pAIR79 SgrAI digest. The resulting plasmid was directly sequenced to confirm the presence of <italic>recA</italic> S117F.</p><p>
<bold>pEAW739</bold> is <italic>recA</italic> D112R N113R in the overproduction vector pET21a (Novagen). For ease of cloning, the wt <italic>recA</italic> gene was first cloned into pUC19 by digesting pEAW260 at the BglII site before the T7 promoter, and the HindIII site after the end of the <italic>recA</italic> gene, and inserting it into pUC19 digested with BamHI and HindIII. BamHI has compatible cohesive ends with BglII. The resulting plasmid was designated pEAW546. The pEAW260 plasmid was used as PCR template with primers consisting of a NdeI site followed by the first 24 bases of the <italic>recA</italic> gene. The ATG in the NdeI site is also the start codon for <italic>recA</italic>. The other primer consisted of bases 370&#8211;327 of the <italic>recA</italic> gene, with the Asp at aa112 and the Asn at aa113 changed to Arg. Bases 263&#8211;270 of the <italic>recA</italic> gene code for a SgrAI site. The PCR product was digested with SgrAI and NdeI and inserted into pEAW546 digested with the same enzymes. The resulting plasmid was directly sequenced to confirm the presence of <italic>recA</italic> D112R N113R. The <italic>recA</italic> gene was excised from the plasmid by digestion with NdeI and BamHI, the restriction enzyme site directly following the stop codon of <italic>recA</italic>, and ligated into pET21a digested with the same enzymes. The resulting plasmid was designated pEAW739.</p><p>
<bold>pEAW861</bold> is <italic>recA</italic> in the overproduction vector pET21a (Novagen) with N113 changed to a TAG stop codon by oligonucleotide mediated mutagenesis for incorporation of the unnatural amino acid p-benzoyl-phenylalanine.</p><p>
<bold>pHRB1 and pARAR1</bold> are vectors that express His-tagged RumB and untagged RumA' respectively. They were derived from the UmuC expression vector, pHUC25, and the UmuD' expression vector pARAD2 [<xref ref-type="bibr" rid="pgen.1005066.ref043">43</xref>] respectively, as follows: pHRB1 was constructed by replacing the PciI&#8212;XhoI fragment of pHUC25, which includes <italic>umuC</italic> (excluding His-tag) and <italic>umuD</italic>', with a PCR amplified fragment of <italic>rumB</italic> from pRW320 [<xref ref-type="bibr" rid="pgen.1005066.ref052">52</xref>]. pHRB1 therefore expresses His-tagged RumB, i.e. MHHHHHH-MPVFA&#8230;LPRVK (1~422 aa of RumB), at a marginal level without a defined promoter. pARAR1 was constructed by replacing <italic>umuD</italic>'of pARAD2 with <italic>rumA</italic>' and it expresses untagged RumA', i.e. M-GFPSP&#8230;MRRKS (32~149 aa of RumA), under the control of an arabinose promoter.</p><p>
<bold>EAW105&#8212;WT MG1655 with FRT-Kan<sup>R</sup></bold>. EAW105 has a mutant pJFS42 FRT-Kan<sup>R</sup>-wt FRT cassette following the <italic>recX</italic> gene on the chromosome of <italic>E</italic>. <italic>coli</italic> MG1655 [<xref ref-type="bibr" rid="pgen.1005066.ref084">84</xref>]. EAW105 was constructed using a variation of the procedure of Datsenko and Wanner [<xref ref-type="bibr" rid="pgen.1005066.ref085">85</xref>]. A plasmid with the <italic>recX</italic> gene, followed by a pJFS42 FRT-Kanamycin resistance gene-wt FRT and the 212 bp downstream of the end of the <italic>recX</italic> gene was used as a PCR template. The primers consisted of the first 19 bases of the <italic>recX</italic> gene, and bases 212&#8211;192 after the end of the <italic>recX</italic> gene. The product was electroporated into MG1655/pKD46, and a Kanamycin resistant colony was selected. PCR was used to confirm the presence of the pJFS42 FRT-Kan<sup>R</sup>-wt FRT after the end of the <italic>recX</italic> gene on the chromosome.</p><p>
<bold>EAW166&#8212;MG1655 <italic>recA</italic> D112R</bold>. EAW166 is <italic>recA</italic> D112R on the chromosome of E. coli MG1655, replacing the wild-type <italic>recA</italic>. EAW166 was constructed in a manner similar to EAW105, with the plasmid used as PCR template consisting of <italic>recA</italic> D112R, <italic>recX</italic> in the wild-type position, the pJFS42 FRT-KanR- wt FRT cassette, and the 218 bp downstream of the <italic>recX</italic> gene. The primers consisted of the first 27 bases of the <italic>recA</italic> gene, and bases 212&#8211;192 after the end of the <italic>recX</italic> gene.</p><p>
<bold>EAW196&#8212;MG1655 <italic>recA</italic> D112R N113R</bold>. EAW196 is <italic>recA</italic> D112R N113R on the chromosome of E. coli MG1655. EAW196 was constructed in a manner similar to EAW166, with the plasmid used as PCR template consisting of <italic>recA</italic> D112R N113R instead of <italic>recA</italic> D112R.</p><p>
<bold>EAW197&#8212;MG1655 <italic>recA</italic> D112R N113R L114R</bold>. EAW197 is <italic>recA</italic> D112R N113R L114R on the chromosome of <italic>E</italic>. <italic>coli</italic> MG1655. EAW197 was constructed in a manner similar to EAW166, with the plasmid used as PCR template consisting of <italic>recA</italic> D112R N113R L114R instead of <italic>recA</italic> D112R.</p><p>
<bold>EAW198&#8212;MG1655 <italic>recA</italic> D112R L114R</bold>. EAW198 is <italic>recA</italic> D112R N113R L114R on the chromosome of E. coli MG1655. EAW198 was constructed in a manner similar to EAW166, with the plasmid used as PCR template consisting of <italic>recA</italic> D112R L114R instead of <italic>recA</italic> D112R.</p><p>
<bold>RW644</bold>-is a BL21(&#955;DE3) derivative with &#916;<italic>umuDC596</italic>::<italic>ermGT</italic>, &#916;<italic>polB1</italic>::&#937;spec and &#916;<italic>dinB61</italic>::<italic>ble</italic> alleles [<xref ref-type="bibr" rid="pgen.1005066.ref043">43</xref>].</p><p>The wild-type <italic>E</italic>. <italic>coli</italic> RecA protein was purified as described [<xref ref-type="bibr" rid="pgen.1005066.ref086">86</xref>]. The RecA concentration was determined using the extinction coefficient &#949;<sub>280</sub> = 2.23&#215;10<sup>4</sup> M<sup>-1</sup> cm<sup>-1</sup> [<xref ref-type="bibr" rid="pgen.1005066.ref087">87</xref>]. The SSB protein was purified as described [<xref ref-type="bibr" rid="pgen.1005066.ref088">88</xref>]; its concentration was determined using the extinction coefficient &#949;<sub>280</sub> = 2.38&#215;10<sup>4</sup> M<sup>-1</sup> cm<sup>-1</sup>. The RecA D112R protein was purified as described [<xref ref-type="bibr" rid="pgen.1005066.ref065">65</xref>].</p><p>The RecA D112R N113R and S117F proteins were purified using a modified protocol for the RecA D112R protein. All purification steps were carried out at 4&#176;C. Cell paste containing RecA D112R N113R or RecA S117F protein was flash-frozen with liquid N<sub>2</sub> and then thawed overnight on ice in a solution of 250 mM Tris-HCl (80% cation, pH 7.8) and 25% (w/v) sucrose. The resuspended cells were adjusted to 20% (w/v). The cells were lysed for 45 minutes with a lysozyme solution (2.5 mg/ml final concentration lysozyme in 250 mM Tris-HCl (80% cation, pH 7.8)), followed by addition of 0.4 mL 25 mM EDTA per ml of lysis solution. RecA D112R N113R or RecA S117F protein was precipitated from the lysate supernatant with 0.111 ml of 5% (w/v) polyethyleneimine per ml of lysate. The pellet was washed with R-buffer (20 mM Tris-HCl (80% cation, pH 7.8), 10% glycerol, 0.1 mM EDTA, 1 mM DTT), and RecA D112R N113R or RecA S117F was then extracted from the pellet twice with R-buffer + 150 mM ammonium sulfate. The extracted RecA protein was precipitated with 0.28 g/ml ammonium sulfate and centrifuged. The pellet was resuspended in R-buffer + 1 M ammonium sulfate and then loaded on a 120 mL CV butyl-Sepharose column. A 5 CV linear gradient of R buffer + 1 M ammonium sulfate to R buffer was run and RecA D112R N113R or RecA S117F came off during the gradient. The eluted RecA mutant protein was dialyzed into P-buffer (20 mM potassium phosphate, 10% glycerol, 0.1 mM EDTA, and 1 mM DTT) and applied to a ceramic hydroxyapatite column. RecA D112R N113R or RecA S117F protein was eluted from the column with a 10 CV linear gradient from P-buffer to 1 M P-buffer (same as P-buffer, except with 1 M potassium phosphate). Peak fractions were analyzed by SDS-PAGE. Pooled fractions were dialyzed into R-buffer and applied to a Source 15-Q column. RecA D112R N113R or RecA S117F protein was eluted from the Source 15-Q column with a 10 column volume linear gradient of R-buffer to R-buffer + 1 M KCl. The eluted RecA mutant protein was concentrated by ammonium sulfate precipitation and resuspended in R-buffer + 1 M KCl. The RecA mutant was then applied to a HiPrep 16/60 Sephacryl S-300 HR size exclusion column fractionated by isocratic elution with R-buffer + 1 M KCl. Peak fractions were analyzed by SDS-PAGE. Fractions containing the mutant RecA protein were pooled and dialyzed into R-buffer for storage. The concentration was determined using the wild-type RecA protein extinction coefficient. The purified RecA variants were free of detectable nuclease and polymerase activity.</p><p>For RecA N113Bpa, pEAW861 was first transformed into BLR (DE3) (BL21 (<italic>F</italic>&#8722;, <italic>dcm</italic>, <italic>ompT</italic>, <italic>hsdS</italic>(<italic>rB</italic>&#8722;<italic>mB</italic>&#8722;), <italic>gal</italic>, (&#955; <italic>DE3</italic>), <italic>recA</italic>) cells. This was followed by transformation of pSup-BpaRS-6TRN(D286R), encoding an amber suppressor tyrosyl tRNA and the engineered tyrosyl tRNA synthetase derived from <italic>M</italic>. <italic>jannaschii</italic> for the incorporation of Bpa into the amber codon [<xref ref-type="bibr" rid="pgen.1005066.ref089">89</xref>]. Cells were grown at 30&#176;C on LB broth supplemented with 1 mM Bpa to the mid-log phase, and then the expression of amber-mutated RecA N113 were induced by the addition of 0.5 mM IPTG for 4&#8211;5 hours. The RecA N113Bpa was purified similar to the above proteins with slightly different chromatography steps. Successive chromatography steps included butyl-Sepharose, ceramic hydroxyapatite, and Source 15Q. The protein was dialyzed into R + 100 mM KCl for storage and concentration was determined using the wild-type extinction coefficient. Molecular weight and incorporation of Bpa were verified by mass spectrometry.</p><p>His-tagged RumA&#8242;<sub>2</sub>B was purified as previously described for UmuD&#8242;<sub>2</sub>C [<xref ref-type="bibr" rid="pgen.1005066.ref043">43</xref>] from the <italic>E</italic>. <italic>coli</italic> strain RW644, containing plasmids pARAR1 and pHRB1 (<xref ref-type="supplementary-material" rid="pgen.1005066.s003">S3 Fig</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Acquisition of resistance to rifampicin (Rif<sup>s</sup>&#8594;Rif<sup>r</sup> assay) was used to determine mutation rates by fluctuation analysis. Rif<sup>s</sup>&#8594;Rif<sup>r</sup> mutation rates were calculated from 15 tube experiments as follows. A fresh single colony was picked from LB agar supplemented with kanamycin (40 &#956;g/mL), and grown overnight at 37&#176;C with aeration in LB medium. The culture was diluted 1:100 in fresh LB with kanamycin (40 &#956;g/mL) and grown to OD<sub>600</sub> ~0.3&#8211;0.35. Cells were spun down and resuspended in an equal volume of M9 medium. The cells were then treated with 0 J/m<sup>2</sup> or 60 J/m<sup>2</sup> using a Spectrolinker XL-1000 UV cross-linker. Cells were spun down again and resuspended in an equal volume of fresh LB and kanamycin and 1 mL aliquots were dispensed into 15 sterile 8 mL culture tubes. Cultures were grown to saturation overnight at 37&#176;C with aeration in the dark. The following day 0.1 mL of each culture was plated on LB agar supplemented with 100 &#956;g/mL rifampicin (Sigma). For wt RecA strains a 1:100 dilution in M9 was made for countable plates. To determine viable cell count, three randomly selected cultures were serially diluted in M9 and plated on LB agar without antibiotics. Plates were incubated at 37&#176;C for 36 hours before counting colonies. The mean numbers of mutations per culture and their confidence limits were obtained with the Ma-Sandri-Sarkar (MSS) maximum-likelihood method [<xref ref-type="bibr" rid="pgen.1005066.ref090">90</xref>,<xref ref-type="bibr" rid="pgen.1005066.ref091">91</xref>] implemented with the FALCOR web tool found at <ext-link ext-link-type="uri" ns0:href="http://www.mitochondria.org/protocols/FALCOR.html">http://www.mitochondria.org/protocols/FALCOR.html</ext-link> [<xref ref-type="bibr" rid="pgen.1005066.ref092">92</xref>], corrected, when required, for plating only 1/10th of the culture. These values were divided by twice the total number of cells per culture to obtain the mutation rates per cell per generation and their confidence limits [<xref ref-type="bibr" rid="pgen.1005066.ref093">93</xref>].</p><p>Western blotting of UmuD cleavage to UmuD' was done as previously described [<xref ref-type="bibr" rid="pgen.1005066.ref094">94</xref>]. Strains harboring RecA variants on the chromosome where transformed with pJM103 (T7 promoter followed by UmuD, Amp<sup>R</sup>)[<xref ref-type="bibr" rid="pgen.1005066.ref095">95</xref>] and pT7 pol26 (Tet<sup>R</sup>) in order to visualize UmuD cleavage. Briefly, a 3 mL overnight with Amp (100 &#956;g/mL) and Tet (50 &#956;g/mL), of each strain was grown at 37&#176;C. A 1:100 dilution of each strain was made the following day in LB with appropriate antibiotics and 0.07 mM IPTG and grown to an OD<sub>600</sub> 0.3&#8211;0.4. Cells were then spun down, resuspended in M9 media, and treated with a UV dosage of 60 J/m<sup>2</sup> using a Spectrolinker XL-1000 UV cross-linker to induce the SOS response. Cells were spun down again and resuspended in LB, Amp/Tet, and 0.2 mM IPTG. At time points indicated an OD<sub>600</sub> = 1.0 of each strain was spun down, resuspended in 10 mL of LB, heated for 10 minutes at 95&#176;C, and then treated with DNase I for 15 minutes. An equal cell OD was loaded and run on a 12% SDS-PAGE. The gel was transferred to an Immobilon-P polyvinylidene difluoride membrane (Millipore) at 300 mA for 1 h and blocked in 5% skim milk in PBS-0.1% Tween solution. The membrane was incubated in a 1:1,000 dilution of affinity-purified rabbit anti-UmuD/UmuD' antibodies [<xref ref-type="bibr" rid="pgen.1005066.ref095">95</xref>] in blocking buffer. The membrane was washed in PBS and 0.1% Tween and then incubated in PBS-Tween with 1:20,000 diluted goat anti-rabbit horseradish peroxidase secondary antibody (Abcam). The membrane was washed again and visualized with Thermo Scientific SuperSignal West Pico enhanced chemiluminescent substrate on Kodak BioMax light film.</p><p>&#946;-galactosidase assay described in [<xref ref-type="bibr" rid="pgen.1005066.ref096">96</xref>] pg 352. An overnight culture of each strain containing the SOS reporter plasmid (pEAW 362 RecN promoter with <italic>lacZ</italic>) was diluted 1:100 into LB with Amp (100 &#956;g/mL). For uninduced SOS, cells were grown to an OD<sub>600</sub> = 0.28&#8211;0.7, chilled on ice, and OD<sub>600</sub> recorded. Immediately, 0.5 mL cells was added to 0.5 mL Z buffer. To lyse cells, 2 drops of chloroform and 1 drop 0.1% SDS were added. Reaction was started by adding 0.2 mL ONPG in Z buffer at 4 mg/mL and stopped by the addition of 0.5 mL 1 M sodium bicarbonate. An OD<sub>420</sub> and OD<sub>550</sub> reading and time to develop color were recorded and used to determine units by the following equation: 1000&#215;(OD<sub>420</sub>&#8211;1.75&#215;OD<sub>550</sub>)/time&#215;volume&#215;OD<sub>600</sub>. For induced SOS cells, cells were grown in LB to OD<sub>600</sub> = 0.4&#8211;0.5, spun down, and resuspended in M9 media. The cells were then treated with 30 J/m<sup>2</sup> using a Spectrolinker XL-1000 UV cross-linker. Cells were spun down again and resuspended in LB + Amp (100 &#956;g/mL) and outgrown for 1 hour. The same protocol as described above was used to measure &#946;-galactosidase activity, but only 100 &#956;L cells were used.</p><p>The three DNA strand exchange reactions were carried out at 37&#176;C in solutions containing 25 mM Tris-Ac (80% cation, pH 7.6), 10 mM magnesium acetate (Mallincraft), 1 mM dithiothreitol (Research Organics), 3 mM potassium glutamate, 5% (w/v) glycerol and an ATP regeneration system consisting of 2 mM phosphoenolpyruvate (PEP) and 10 units/ml pyruvate kinase. Typically, 3.5 &#9633;M RecA and 10 &#9633;M css DNA (M13mp18) were pre-incubated in the reaction buffer and regeneration system for 10 min before addition of 3 mM ATP and 1 &#9633;M SSB. The reactions were then started by adding 20 &#9633;M linear double-stranded (M13mp18) DNA after 10 min of incubation at 37&#176;C with ATP. The reaction was incubated at 37&#176;C for the indicated timepoints. Reaction aliquots (9.5 &#956;l) were deproteinized by addition of a mixture containing 1.2 &#956;l 10% SDS, 0.3 &#956;l 0.5M EDTA and 0.6 &#956;l 20 mg/ml Proteinase K and incubated for 30 min at 37&#176;C. Aliquots were then mixed with 2.5 &#956;l 6 x loading buffer (15% Ficoll, 0.25% bromophenol blue, 0.25% xylene cyanol FF), loaded on a 1% agarose gel, and electrophoresed at 25&#8211;35 V for 16 hours at room temperature. To visualize the DNA bands, the gels were stained with ethidium bromide, and exposed to ultraviolet light. Gel images were captured with a digital CCD camera utilizing GelExpert software (Nucleotech).</p><p>UmuD cleavage was performed essentially as previously described [<xref ref-type="bibr" rid="pgen.1005066.ref014">14</xref>]. Standard reaction mixtures (30 &#956;l) contained 40 mM Tris-HCI (pH 8.0), 10mM MgCl<sub>2</sub>, 30 mM NaCl, 2 mM dithiothreitol, 5 &#956;M of M13mp18 circular single-stranded DNA, 3 mM adenosine 5'-(&#947;-thio)triphosphate (ATP&#947;S), and UmuD, LexA, and RecA as noted. Incubation was at 37&#176;C for the times indicated. The products of the reaction were separated on a 4&#8211;15% gradient SDS-PAGE and visualized by staining with Coomassie blue. Where indicated, the intensity of the protein bands was quantitated with the software package TotalLab v1.10 from Phoretix.</p><p>A coupled, spectrophotometric enzyme assay [<xref ref-type="bibr" rid="pgen.1005066.ref008">8</xref>,<xref ref-type="bibr" rid="pgen.1005066.ref097">97</xref>] was used to measure the ATPase activities of the wt RecA, D112R, and D112R N113R on circular ssDNA. The regeneration of ATP from ADP and PEP was coupled with the oxidation of NADH and monitored by observing the decrease in absorbance at 380 nm over time. An NADH extinction coefficient of &#949;<sub>380</sub> = 1.21 mM<sup>-1</sup>cm<sup>-1</sup> was used to calculate the rate of ATP hydrolysis. The assays were carried out in a Varian Cary 300 dual beam spectrophotometer equipped with a temperature controller and a 12-position cell changer. The cell path lengths are 1 cm or 0.5 cm and band pass was 2 nm.</p><p>The reactions were carried out at 37&#176;C with 25 mM Tris-Ac (80% cation, pH 7.6), 10 mM magnesium acetate, 1 mM DTT, 3 mM potassium glutamate, 5% (w/v) glycerol, an ATP regeneration system (3 mM PEP and 10 U/ml pyrvate kinase), and a coupling system (10 U/ml lactate dehydrogenase, 2 mM NADH for 1-cm cuvettes or 3 mM NADH for 0.5 cm cuvettes). The concentrations of RecA protein, SSB, and DNA, and the time for pre-incubation, are indicated in the Fig legends.</p><p>The interaction between pol V and RecA variants was probed as described previously [<xref ref-type="bibr" rid="pgen.1005066.ref063">63</xref>]. Briefly, wt RecA, D112R, and D112R N113R were labeled using a Fluorescein-EX Protein Labeling Kit (Molecular Probes). Reaction mixtures (110 &#956;L) contained pol V standard reaction buffer (20 mM Tris (pH 7.5), 8 mM magnesium chloride, 5 mM DTT, 0.1 mM EDTA, 25 mM sodium glutamate, and 4% (vol/vol) glycerol), and 20 nM labeled RecA protein. The fluorescence anisotropy was measured at 25&#176;C using a Tecan Infinite M1000 instrument with 470-nm excitation and 535-nm emission wavelengths. All experiments were repeated four times. The average FA values were plotted with one standard deviation of the mean shown as error. Prism software was used to convert FA values to the percent of RecA bound and apparent dissociation (K<sub>d, app</sub>) constants were determined using one-site, specific binding.</p><p>Pol V activity was assayed for transactivation by RecA as described previously [<xref ref-type="bibr" rid="pgen.1005066.ref064">64</xref>]. Reactions were performed in pol V standard reaction buffer (20 mM Tris (pH 7.5), 8 mM magnesium chloride, 5 mM DTT, 0.1 mM EDTA, 25 mM sodium glutamate, and 4% (vol/vol) glycerol). First the standard reaction buffer with1 mM DTT, 2 mM ATP&#947;S, 2.5 &#956;M poly dT<sub>45</sub>, and 18 &#956;M RecA was incubated at 37&#176;C for 10 minutes to allow RecA filament formation. This was followed by addition of 1mM dNTPs, 1 &#956;M pol V, and 150 nM 65-mer hairpin primer-template DNA which incubated for 60 minutes at 37&#176;C. Reactions (20 &#956;L) were stopped with 40 &#956;L 20 mM EDTA in 95% formamide and ethanol precipitated at-20 &#176;C overnight. Ethanol precipitations were spun down, washed with 70% ethanol and resuspended in 10 &#956;L stop solution. Samples were heated at 95&#176;C for 10 minutes, quick cooled on an ice-water bath for 5 minutes and immediately loaded on a 10% urea denaturing polyacrylamide gel. The synthesis products were imaged using the FAM settings on a Typhoon FLA 9000 (GE Healthcare). Gel band intensities were analyzed to determine primer utilization using ImageQuant TL software (GE Healthcare).</p><p>To transactivate of pol V and RumA&#8242;<sub>2</sub>B (400 nM each; <xref ref-type="fig" rid="pgen.1005066.g007">Fig. 7</xref>), 400nM RecA filaments were preformed by preincubating RecA (4 &#956;M) with 30nt ssDNA (0.4 &#956;M) and ATP&#947;S (500 &#956;M) at 37&#176;C for 3 mins in reaction buffer. Subsequently pol V or RumA&#8242;<sub>2</sub>B was mixed in with RecA*, 5'- <sup>32</sup>P-labeled primer-template 3 nucleotide overhang HP (100 nM) and dNTPs (500 &#956;M) to a final volume of 10 &#956;l and incubated for 30 mins (or 1 h for RecA*D112R transactivation) at 37&#176;C. Reactions were quenched in stop solution (90% Formaldehyde and 0.05M EDTA) and resolved on a 20% denaturing polyacrylamide gel allowing single nucleotide separation. Gel band intensities were detected by phosphorimaging and quantified using IMAGEQUANT software.</p><p>Pol V Mut was assembled as previously described [<xref ref-type="bibr" rid="pgen.1005066.ref046">46</xref>]. Briefly, 5' amino-modified 45nt ssDNA was covalently attached to Cyanogen-Bromide Sepharose resin according to manufacturer&#8217;s protocol (Sigma-Aldrich) and transferred into a spin column. Stable RecA* was formed on the ssDNA by mixing RecA (20 &#956;M) and ATP&#947;S (500 &#956;M) then incubating at 37&#176;C for 20 mins. This was followed by the removal of excess RecA and ATP&#947;S via multiple centrifugations of the spin column until no RecA was detected in the flow through. His-tagged pol V (2 nmole) was resuspended in reaction buffer and mixed with RecA*-resin to form pol V Mut. To separate pol V Mut from the RecA*-resin, the spin column was gently centrifuged one last time. The concentration of pol V Mut collected in the flow through was determined by SDS-PAGE gel.</p><p>Pol V Mut binding to &#949;-ATP (Life Technologies) was measured as a change in rotational anisotropy. In a 70 &#956;l reaction, &#949;-ATP was mixed in standard reaction buffer with pol V Mut, pol V + RecA, or RecA (400 nM). Rotational anisotropy was measured using a QuantaMaster (QM-1). Samples were excited with vertically polarized light at 410 nm and both vertical and horizontal emission was monitored at 425 nm.</p><p>Binding to p/t DNA was measured by changes in steady-state fluorescence depolarization (rotational anisotropy). Flourescein-labeled 12 nt oh HP (50 nM) was mixed with pol V Mut, pol V + RecA, or RecA alone (400 nM each) and ATP&#947;S was titered to a final concentration of 2000 &#956;M. All reactions (70 &#956;l) were carried out at 37&#176;C in standard reaction buffer. Rotational anisotropy was measured using a QuantaMaster (QM-1) fluorometer (Photon Technology International) with a single emission channel. Samples were excited with vertically polarized light at 495 nm and both vertical and horizontal emission was monitored at 520 nm.</p><p>In order to probe the RecA/pol V interaction, the RecA N113Bpa variant was incubated pol V in several conditions. Pol V-Mut active complex was formed as described above. Once the pol V-Mut was formed, 9.2 &#956;M ATP, 100 &#956;M ATP&#947;S, or 5 &#956;M primer-template DNA and 100 &#956;M ATP&#947;S were added. Different concentrations of ATP and ATP&#947;S were used due to stock concentration limitations of ATP. These mixtures were incubated at 37&#176;C for 5 minutes before samples were subjected to UV light (CalSun unit with four 15-watt bulbs). Samples were kept in 1.5 mL Eppendorf tubes and subjected to 45 mintues of UV at 4 &#176;C. Reactions were separated by SDS-PAGE and stained with Coomassie blue or analyzed via Western blotting for molecular composition of cross-linked complexes.</p><p>To form inactive RecA + UmuD&#8242;<sub>2</sub>C complexes, reactions contained pol V standard reaction buffer, 1 mM DTT, 5 &#956;M RecA, and 5 &#956;M UmuD&#8242;<sub>2</sub>C. When present, ATP&#947;S is at 2 mM and poly dT 45-mer is at 1 &#956;M. It should be noted that pol V storage buffer contains 1 M NaCl, and that the final concentration of NaCl present from addition of pol V is 380 mM. The reactions were incubated at 37&#176;C for 30 minutes before subjected to UV light. Samples were kept in 1.5 mL Eppendorf tubes and subjected to 45 minutes UV at 4&#176;C. Samples were also separated by SDS-PAGE and stained with Coomassie blue or analyzed via Western blotting for molecular composition of cross-linked complexes.</p><p>&#8220;In Gel&#8221; digestion and mass spectrometric analysis was done at the Mass Spectrometry Facility (Biotechnology Center, University of Wisconsin-Madison). In short, Coomassie R-250 stained gel pieces were de-stained completely in MeOH/H<sub>2</sub>0/NH<sub>4</sub>HCO<sub>3</sub> (50%/50%/100 mM), dehydrated for 2 min in ACN/H<sub>2</sub>0/NH<sub>4</sub>HCO<sub>3</sub> (50%/50%/25mM) then once more for 30 sec in 100% ACN. Dried in a Speed-Vac for 1min, reduced in 25mM DTT (Dithiotreitol in 25 mM NH<sub>4</sub>HCO<sub>3</sub>) for 30 min at 56&#176;C, alkylated with 55mM IAA (Iodoacetamide in 25 mM NH<sub>4</sub>HCO<sub>3</sub>) in darkness at room temperature for 30 min, washed once in H<sub>2</sub>0, dehydrated for 2 min in ACN/H<sub>2</sub>0/NH<sub>4</sub>HCO<sub>3</sub> (50%:50%:25mM) then once more for 30 sec in 100% ACN. Dried again and rehydrated with 20 &#956;l of trypsin solution with 0.01% ProteaseMAX surfactant (10ng/&#956;l <italic>Trypsin Gold</italic> from Promega Corp. in 25 mM NH<sub>4</sub>HCO<sub>3</sub>/0.01% w/v of ProteaseMAX from Promega Corp.). Let stand for 2 min at room temperature then additional 30 &#956;l of overlay solution (25mM NH<sub>4</sub>HCO<sub>3</sub>/0.01% w/v of ProteaseMAX) was added to keep gel pieces immersed throughout the digestion. The digestion was conducted for 3 hrs at 42&#176;C then peptides generated from tryptic digestion were transferred to a new Protein LoBind tubes (~50 &#956;l volume) and secondary digestion followed by addition of 5 &#956;l AspN proteinase (20ng/&#956;l endoproteinase AspN from Roche Diagnostics in 25 mM NH<sub>4</sub>HCO<sub>3</sub>). That digestion was conducted for 2 hrs at 37&#176;C. Concurrently, the gel pieces post tryptic digestion were dehydrated with 10 &#956;l of ACN and re-hydrated with 20 &#956;l of secondary digestion mix composed of 5 &#956;l AspN proteinase (20ng/&#956;l endoproteinase AspN from Roche Diagnostics in 25mM NH<sub>4</sub>HCO<sub>3</sub>), 13 &#956;l 25 mM NH<sub>4</sub>HCO<sub>3</sub> and 2 &#956;l of 0.1% ProteaseMAX surfactant. This digestion was allowed to proceed for 2 hrs at 37&#176;C after which the solution was pulled off and combined with the primary &#8216;in liquid&#8217; AspN digestion.</p><p>Proteolysis was terminated by acidification with 2.5% TFA (Trifluoroacetic Acid) to 0.3% final (10 &#956;l added). Degraded ProteaseMAX was removed via centrifugation (max speed, 10 minutes) and the peptides solid phase extracted (<italic>ZipTip C18</italic> pipette tips from Millipore). Peptides were eluted off the C18 column with 1 &#956;l of acetonitrile/H<sub>2</sub>O/TFA (60%:40%:0.1%) diluted to 30 &#956;l total volume with 0.1% Formic acid and 8 &#956;l was loaded on the instrument.</p><p>Peptides were analyzed by nanoLC-MS/MS using the Agilent 1100 nanoflow system (Agilent) connected to a hybrid linear ion trap-orbitrap mass spectrometer (LTQ-Orbitrap XL, Thermo Fisher Scientific) equipped with an EASY-Spray electrospray source. Chromatography of peptides prior to mass spectral analysis was accomplished using capillary emitter column (PepMap C18, 3 &#956;M, 100 &#197;, 150x0.075 mm, Thermo Fisher Scientific) onto which extracted peptides were automatically loaded. NanoHPLC system delivered solvents A: 0.1% (v/v) formic acid in water, and B: 99.9% (v/v) acetonitrile, 0.1% (v/v) formic acid at 0.60 &#956;L/min to load the peptides and 0.3 &#956;l/min to elute peptides directly into the nano-electrospray over a 35 minutes 0% (v/v) B to 40% (v/v) B followed by 5 minute 40% (v/v) B to 100% (v/v) B gradient. As peptides eluted from the HPLC-column/electrospray source survey MS scans were acquired in the Orbitrap with a resolution of 100,000 and up to 5 most intense peptides per scan were fragmented and detected in the ion trap over the 300 to 2000 m/z; redundancy was limited by dynamic exclusion. Raw MS/MS data were converted to mgf file format using MSConvert (ProteoWizard: Open Source Software for Rapid Proteomics Tools Development). Resulting mgf files were used to search against <italic>Escherichia coli</italic> database containing target constructs plus common lab protein contaminants. Peptide mass tolerance was set at 20 ppm and fragment mass at 0.8 Da.</p><p>Assignment of amino acid residues involved in the cross-linking was done using new generation hybrid linear ion trap-orbitrap mass spectrometer (LTQ-Orbitrap Elite, Thermo Fisher Scientific) equipped with an EASY-Spray electrospray source. Chromatography of peptides was conducted under the same conditions as above. As peptides eluted from the HPLC-column MS1 scans from 300 to 2000 m/z were acquired in the Orbitrap with a resolution of 120,000 followed by MS2 fragmentation of 10 peptides from the parent mass list (containing masses observed for candidate peptides in the discovery phase using Orbitrap XL system) and 10 additional most intense peptides from each unique MS1 scan; redundancy was limited by dynamic exclusion. Raw MS/MS data was manually interrogated to precisely map the residue involved in the particular cross-linking.</p><p>To determine the identity of cross-linked products a targeted database search of raw mass spec data was conducted using only the sequence of proteins used in the cross-linking experiment plus common lab protein contaminants. Cross-linked candidates were generated using StavroX software (StavroX Freeware version 3.4.5 from University of Halle-Wittenberg) with BPA chosen as a cross-linker, Lysine, Arginie and Aspartate with 2 missed cleavages as a protease sites and static Cysteine carbamidomethylation plus variable Methionie oxidation as possible modifications. These candidates were manually evaluated for the quality of MS1 and MS/MS data, matched with their theoretical isotopic and fragmentation distribution and finally cross-checked against the non-UV treated peptide mass spec data from proteins used in the cross-linking experiment. In order to define the residue or approximate the location of cross-linking within the peptide a manual analysis of MS/MS data for isomeric cross-linked species (same cross-linked peptide with differential LC elution profile) was conducted.</p>