<p>The research described in this study was performed under the Animal Study Protocols LPG-002 and LCBG-004, approved by the NCI Bethesda Animal Use and Care Committee. Animal euthanasia was performed by cervical dislocation after anesthesia by Avertin.</p><p>NZB/B1NJ and NZW/LacJ DNA was obtained from The Jackson Laboratory DNA Repository. Library preparation and sequencing on Illumina HiSeq instruments was performed following the manufacturers recommended protocols. Four lanes of 102 base paired end sequence per strain was performed to achieve approximately 40x coverage. Reads were aligned to the reference genome (GRCm38) using BWA version 0.7.5a-r406 [19451168]. All lanes from the same library were then merged into a single BAM file using Picard tools and PCR duplicates were marked using Picard 'MarkDuplicates' [19505943]. Local realignment was carried out using GATK-v3.0 using default parameters to generate the set of intervals for realignment [20644199]. SNP and indel discovery was performed with the Samtools v1.1 (samtools mpileup -t DP,DV,DP4,SP,DPR,INFO/DPR -E -Q 0 -pm3 -F0.25) and calling was performed with Bcftools call v1.1 (bcftools call -mv -f GQ,GP -p 0.99). Raw sequencing data is available under accession ERP000927.</p><p>DNase hypersensitive site (DHS) data for the mouse 3134 mammary adenocarcinoma cells was downloaded from the Encode project websites.</p><p xmlns:ns0="http://www.w3.org/1999/xlink"><ext-link ext-link-type="uri" ns0:href="http://hgdownload.cse.ucsc.edu/goldenPath/mm9/encodeDCC/wgEncodeUwDnase/">http://hgdownload.cse.ucsc.edu/goldenPath/mm9/encodeDCC/wgEncodeUwDnase/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink"><ext-link ext-link-type="uri" ns0:href="http://hgdownload.cse.ucsc.edu/goldenPath/mm9/encodeDCC/wgEncodeUwDgf/">http://hgdownload.cse.ucsc.edu/goldenPath/mm9/encodeDCC/wgEncodeUwDgf/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genotype data for FVB/NJ was downloaded from the Welcome Trust Sanger Centre website: <ext-link ext-link-type="uri" ns0:href="http://www.sanger.ac.uk/resources/mouse/genomes/">http://www.sanger.ac.uk/resources/mouse/genomes/</ext-link> The NZB/B1NJ and NZW/LacJ genotype data is also available at this site.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All computations were performed on NIH helix/biowulf system, documentation of which is available at <ext-link ext-link-type="uri" ns0:href="https://helix.nih.gov/">https://helix.nih.gov</ext-link>. We used R computing environment, perl scripts, bedtools, and ucsc liftOver for most of the analyses.</p><p>The workflow consisted of the following. The UCSC liftOver tool was used to convert between mm9 and mm10 as necessary when using bedtools intersectBed to intersect two bed files. 1) The Encode DHS data were filtered for the regions overlapping with polymorphic sites. Since the DHS data were generated in Genome Build mm9, we used UCSC mm9 snp128 data to restrict the DHS sites. 2) The mice NZB/B1NJ genotype data in vcf was filtered to retain the SNPs that overlap with the DHS present in the 3134 cells. 3) We then removed SNPs in the DHS that are present in the mouse FVB/NJ strain.</p><p>BED files containing the subtracted NZB/B1NJ specific, DHS associated SNPs were loaded into the GREAT tool website using the default settings for gene assignment. GREAT calculates statistics by associating genomic regions with nearby genes and applying the gene annotations to the regions. Association is a two-step process. First, every gene is assigned a regulatory domain. Then, each genomic region is associated with all genes whose regulatory domain it overlaps. The default association settings included assignment of basal regulatory elements 5 kb upstream and 1 kb downstream of transcriptional start sites (regardless of other nearby genes). In addition, the gene regulatory domain was extended up to 1 megabase in both directions to the nearest gene's basal domain but no more than the maximum extension in one direction [<xref ref-type="bibr" rid="pgen.1005989.ref016">16</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The Diversity Outcross x MMTV-PyMT G5 CEL files have been deposited in the Gene Expression Omnibus (GEO) database (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo">www.ncbi.nlm.nih.gov/geo</ext-link>) under the accession number GSE48565. The G7 CEL files are deposited under the accession number GSE64522. Data was imported into BRB ArrayTools version 4.3.2 (<ext-link ext-link-type="uri" ns0:href="http://brb.nci.nih.gov/BRB-ArrayTools/">http://brb.nci.nih.gov/BRB-ArrayTools/</ext-link>) and normalized using the median array as a reference. Batch correction was not performed before BRB ArrayTool analysis. Only genes with a Log Intensity Variation of p&lt; 0.05 were considered for further analysis. Distant metastasis free survival analysis of these data was performed using the Find Genes Associated with Survival tool (DMFS genes). Genes were considered associated with DMFS if p&lt; 0.05. Genes correlated with metastatic disease was performed using the Spearman Correlation Test option of the Find Genes Associated with Quantitative Trait tool (metastasis correlated genes). Genes with a p&lt; 0.05 were considered to be significantly associated with metastatic disease.</p><p>To provide direction and relative importance to each gene, weights for each gene in the signature was determined from the DO mouse crosses. For the DMFS signature, the hazard ratios associated with DMFS in the DO G5 cross was log2 transformed to provide direction and relative strength for each gene in the signature. For the metastasis-correlated signature, the correlation coefficient for each gene with metastasis number in the DO G7 data was used. The weighted gene was then uploaded to the GOBO tool and analyzed using the default settings for the ER+ and ER- subsets of breast cancers. P values reported are those generated by the GOBO analytical tool.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">2x10<sup>5</sup> Mvt1[<xref ref-type="bibr" rid="pgen.1005989.ref048">48</xref>] mammary tumor cells were plated in 6 well dishes. 24 hours later the cells were treated with 500 uM 8-Br-cAMP (Sigma) dissolved in 0.1% DMSO for 24 hours, then RNA harvested using Trizol following the manufacturers recommended protocol. Vehicle control or 8-Br-cAMP samples were performed in triplicate. Transcriptome analysis performed by the NCI Laboratory of Molecular Technology using Affymetrix MOE430 v2 chips. The data was analyzed using the Class Comparison tool of BRB ArrayTools. For the gene signature analysis, the differentially expressed genes were filtered for those with greater than 10-fold change in expression, either up- or down-regulated, compared to the vehicle alone. The fold change in gene expression was used to weight the individual genes, and the ability of the weighted gene signature to discriminate breast cancer patient outcome assessed using the webtool GOBO (<ext-link ext-link-type="uri" ns0:href="http://co.bmc.lu.se/gobo/">http://co.bmc.lu.se/gobo/</ext-link>).</p><p>The 6DT1 and Mvt1 cell lines[<xref ref-type="bibr" rid="pgen.1005989.ref048">48</xref>] were obtained from the laboratory of Robert Dickson, George Washington University. Microsatellite genotyping validated that these cell lines originated from an FVB/NJ animal. Both cell lines have been demonstrated to be mycoplasma-free.</p><p>shRNA lentiviral vectors were purchased from Sigma-Aldrich (cat. # SHCLNG-NM_021424). Stable <italic>Pvrl1</italic> knockdown cell lines were generated by lentiviral transduction into Mvt1 cells and knockdown validated by qRT-PCR. 8x10<sup>5</sup> Mvt1 or 10<sup>5</sup> 6DT1 cells were inoculated into the four mammary fat pad of 6&#8211;8 week old FVB/NJ female mice, 10 animals per group. Animals were euthanized at 5 weeks (experiment 1) or 4 weeks (experiment 2) after implantation. One-way Anova with Dunnett&#8217;s correction for multiple testing was performed for each experiment using GraphPad Prism. The results of the replicate experiments were then combined using Fisher&#8217;s combined probability test. All procedures were performed as approved by the NCI-Bethesda Animal Care and Use Committee.</p><p>Epitope-tagged <italic>Zbtb16</italic> over-expression was performed by lentiviral transduction into the cell line Mvt1. Orthotopic implantation was carried out as described above and analyzed using the Mann-Whitney test in GraphPad Prism. Autochthonous tumor assays were performed by breeding MMTV-PyMT male animals to <italic>Zbtb16</italic><sup>+/-</sup> female animals to generate PyMT<sup>+</sup>/<italic>Zbtb16</italic><sup><italic>+/-</italic></sup> female animals. 20 PyMT<sup>+</sup>/<italic>Zbtb16</italic><sup><italic>+/-</italic></sup> animals was selected for analysis based on previous experience which suggested this as an appropriate group size to achieve statistical significance. Statistical differences were assessed using the Mann-Whitney test in GraphPad Prism. All procedures were performed as approved by the NCI-Bethesda Animal Care and Use Committee.</p><p>100,000 cells/mouse of the mammary tumor cell line 6DT1 were orthotopically implanted into the fourth mammary fat pad of female FVB/NJ mice (6&#8211;8 weeks old). 10 days post-implantation all of the animals were combined into a single cage then randomly assigned to treatment or control group by alternating assignment to new cages. Roziglitazone (100&#956;M) or vehicle (DMSO, 0.17%) was added to drinking water 10 days post tumor implantation and available to mice ad libitum. Drinking water with Roziglitazone or DMSO was refreshed every week. Tumor growth was monitored and animals were euthanized 28 days post implantation. Tumors and lungs were evaluated for weight and surface metastases respectively. The experiment was performed twice and the results of the Mann-Whitney test combined using Fisher combined probability test. All procedures were performed as approved by the NCI-Bethesda Animal Care and Use Committee.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">NZB/B1NJ sequence polymorphism data available data is available at <ext-link ext-link-type="uri" ns0:href="http://www.sanger.ac.uk/resources/mouse/genomes/">http://www.sanger.ac.uk/resources/mouse/genomes/</ext-link> Raw data is available at <ext-link ext-link-type="uri" ns0:href="http://sra.dnanexus.com/studies/ERP000927">http://sra.dnanexus.com/studies/ERP000927</ext-link>. The Diversity Outcross x MMTV-PyMT G5 CEL files have been deposited in the Gene Expression Omnibus (GEO) database (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo">www.ncbi.nlm.nih.gov/geo</ext-link>) under the accession number GSE48565. The G7 CEL files are deposited under the accession number GSE64522.</p>