<p>Patients were enrolled in the NIH Undiagnosed Diseases Program [<xref ref-type="bibr" rid="pgen.1006481.ref005">5</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006481.ref007">7</xref>] and in protocol 76-HG-0238, &#8220;Diagnosis and Treatment of Patients with Inborn Errors of Metabolism and Other Genetic Disorders&#8221;, approved by the NHGRI Institutional Review Board. All patients involved in this study gave written, informed consent. All zebrafish experiments were approved by the UCLA Animal Care and Use Committee and the NHGRI ACUC Committee.</p><p>To establish a mutant <italic>ATP6V1H</italic> zebrafish line, gRNA targeting the fourth exon of ATP6V1H was designed and shown to induce indels by injection with Cas9 mRNA in zebrafish. Several mutant lines were established using gRNA with the sequence 5&#8217;-GTGTGTCATCAATCAGGGTCAGG; the line used in this study carried a 17 bp insertion and produced a truncated peptide of 94aa, whereas the wild type ATP6V1H encodes a 463aa protein. Stable heterozygous carriers of the mutation were mated to produce homozygous mutants. Details of mutation generation and characterization are presented in the supplementary section and <xref ref-type="supplementary-material" rid="pgen.1006481.s003">S3 Fig</xref>. Staining for bone and cartilage cells was performed as described [<xref ref-type="bibr" rid="pgen.1006481.ref033">33</xref>,<xref ref-type="bibr" rid="pgen.1006481.ref034">34</xref>].</p><p>Total RNA was extracted in Trizol Reagent (Life Technologies) according to the manufacturer&#8217;s protocol. cDNA was synthesized using Superscript III (Life Technologies) and oligodT (Life Technologies) following the manufacturer&#8217;s protocol. Quantitative real-time PCR was performed using FastStart SYBR Green Master (Roche). Primer sequences are listed in <xref ref-type="supplementary-material" rid="pgen.1006481.s011">S1 Table</xref>. Values of three independent samples (n = 50 each sample) are shown.</p><p>See supplementary materials.</p><p>The probes listed in the figures were generated by PCR using zebrafish embryonic cDNA (Primers in <xref ref-type="supplementary-material" rid="pgen.1006481.s011">S1 Table</xref>). Regular and double fluorescence whole-mount in situ hybridization was, respectively, performed as previously described [<xref ref-type="bibr" rid="pgen.1006481.ref035">35</xref>,<xref ref-type="bibr" rid="pgen.1006481.ref036">36</xref>].</p><p>For X-ray imaging, adult zebrafish were fixed in 10% neutral-buffered formalin for 24 hours and X-ray images were taken using Faxitron MX-20 System Digital Radiography system (Faxitron Bioptics, Tucson, Arizona). For micro-computed tomography (&#956;-CT), adult zebrafish were fixed in 10% neutral-buffered formalin for 24hours and mounted in 1% low-melt agarose (Sigma) in a plastic vial. Specimens were scanned using SkyScan1172 Micro-CT scanner (Bruker MicroCT, Kontich, Belgium) with the X-ray power at 55kVp and 185&#956;A, and specimens were scanned at a 10 &#956;m voxel resolution. A three-dimensional reconstruction was generated with NRecon software from the set of scans. The images were then analyzed using CTAn (v.1.14.4) and CTVol (v.2.2) softwares. The first caudal bones were assessed for bone mineral density (BMD), bone volume (BV), and bone surface (BS). For TRAP staining, fixed fish were decalcified and sectioned (5 um) after paraffin embedding. TRAP staining was performed using a kit (Sigma 387A-KT) according to the manufacturer's instructions.</p><p>MMP9 Inhibitor II (Cat# 444293), MMP13 Inhibitor (Cat# 444283) and MMP9/13 Inhibitor I (Cat# 444252) were purchased from EMD Millipore. Zebrafish were treated with designated concentrations of each inhibitor from 1 to 6 days post fertilization in 2 ml of fish water.</p>