<p>The METSIM study was performed in accordance with the Helsinki Declaration and was approved by the Research Ethics Committee, Hospital District of Northern Savo (number 171/2004). All study participants gave their written informed consent.</p><p>Among the 10,197 participants in the METSIM study, we analyzed 8,372 non-diabetic individuals (mean age 57&#177;7 SD years and BMI 26.8&#177;3.8 kg/m<sup>2</sup>)[<xref ref-type="bibr" rid="pgen.1007079.ref055">55</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We measured the 72 lipid and lipoprotein traits from blood serum samples by proton NMR, as previously described[<xref ref-type="bibr" rid="pgen.1007079.ref056">56</xref>]. Briefly, lipid samples are extracted and measured by proton NMR, and the NMR-spectra and automated phasing are compared to plate, background, and serum controls. Regression modeling is used to quantify the spectral areas to produce the quantified molecular data. The samples included 60 lipoprotein subclasses, 6 cholesterol and triglyceride measures, 3 cholesterol diameter measures, and 3 apolipoprotein measurements (<xref ref-type="supplementary-material" rid="pgen.1007079.s008">S1 Table</xref>). Definition of the subclass traits has been previously described[<xref ref-type="bibr" rid="pgen.1007079.ref056">56</xref>,<xref ref-type="bibr" rid="pgen.1007079.ref057">57</xref>]. We visualized the Pearson correlation matrix between lipoprotein traits using a corrgram with the ellipse (<ext-link ext-link-type="uri" ns0:href="https://cran.r-project.org/web/packages/ellipse/">https://cran.r-project.org/web/packages/ellipse/</ext-link>) and lattice packages (<ext-link ext-link-type="uri" ns0:href="http://lattice.r-forge.r-project.org/">http://lattice.r-forge.r-project.org/</ext-link>) within R (<xref ref-type="supplementary-material" rid="pgen.1007079.s002">S2 Fig</xref>)[<xref ref-type="bibr" rid="pgen.1007079.ref058">58</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We genotyped the study samples using the HumanOmniExpress-12v1_C BeadChip and Infinium HumanExome-12 v1.0 BeadChip, resulting in 631,879 and 236,849 variants, respectively. Imputation was performed using the GoT2D reference panel of &gt;19M variants (SNPs, in-dels, and large deletions) based on whole-genome sequence of 2,657 Europeans consisting of German, Swedish, Finnish, and British participants; with the majority of the cohort comprised of Finns[<xref ref-type="bibr" rid="pgen.1007079.ref059">59</xref>]. The resulting 15,144,991 variants were subjected to quality controls including sample- and variant-level controls for detecting sample contamination, sex and relatedness confirmation, and detection of sample outliers using principal-component analysis. To exclude samples with evidence of DNA contamination, we used BAFRegress v0.9 (<ext-link ext-link-type="uri" ns0:href="http://genome.sph.umich.edu/wiki/BAFRegress">http://genome.sph.umich.edu/wiki/BAFRegress</ext-link>). Based on principal component analysis, eighteen exome array sample duplicates, one individual each from seven monozygotic twin pairs, and twelve population outliers were removed from analysis. Due to sex chromosome inconsistencies, fourteen OmniExpress samples were removed. Samples with low genotype call rate (&lt;95%) for either array were removed. Variants with low-mapping quality to build hg19, low genotype completeness (&lt;95% for OmniExpress and &lt;98% for exome array), or multi-allelic variants were removed. The remaining high quality variants were phased using Shape-It v2[<xref ref-type="bibr" rid="pgen.1007079.ref060">60</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We tested for association using imputed dosages for all variants with summed minor allele count dosage &gt;1 with each of the 72 lipid and lipoprotein traits assuming an additive model and accounting for cryptic relatedness using the EMMAX linear mixed model approach as implemented in EPACTS (<ext-link ext-link-type="uri" ns0:href="http://genome.sph.umich.edu/wiki/epacts">http://genome.sph.umich.edu/wiki/epacts</ext-link>). Traits were adjusted for age, age<sup>2</sup>, smoking status, and lipid lowering medication. Residuals were inverse normalized. To assess the level of genomic inflation, we calculated the genomic control statistic (&#955;<sub>GC</sub>) for all of the trait-variant associations using R[<xref ref-type="bibr" rid="pgen.1007079.ref058">58</xref>] (<xref ref-type="supplementary-material" rid="pgen.1007079.s001">S1 Fig</xref>). Reported effect size regression coefficients (betas) sizes are given in standard deviation units. The rare lead associated variants that were imputed and had MAF&lt;0.01 were tested for genotype accuracy by using TaqMan assays (Thermo Fisher Scientific) or Sanger sequencing in 499 METSIM participants who carried one or more rare alleles at these variants. Variants that had &gt;10% discordance between the imputed genotype and the sequenced genotype in the examined individuals were removed from the analysis. Variants with MAF&lt;0.001 were excluded from analysis (<xref ref-type="supplementary-material" rid="pgen.1007079.s012">S5 Table</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To identify variant association signals distinct or independent from those reported previously, we identified variants previously reported to be associated with any cholesterol, lipid, lipoprotein, or triglyceride trait. We performed a literature review of GWAS and sequencing studies using PubMed (<ext-link ext-link-type="uri" ns0:href="https://www.ncbi.nlm.nih.gov/pubmed/">https://www.ncbi.nlm.nih.gov/pubmed/</ext-link>) and Google Scholar (<ext-link ext-link-type="uri" ns0:href="https://scholar.google.com/">https://scholar.google.com/</ext-link>), screened a GWAS Catalog (<ext-link ext-link-type="uri" ns0:href="http://www.ebi.ac.uk/gwas/">http://www.ebi.ac.uk/gwas/</ext-link>), and used SNIPPER (<ext-link ext-link-type="uri" ns0:href="https://csg.sph.umich.edu/boehnke/snipper/">https://csg.sph.umich.edu/boehnke/snipper/</ext-link>) to query publicly accessible databases. The resulting curated list contained 1,714 variants, at &gt;150 loci from 33 studies (<xref ref-type="supplementary-material" rid="pgen.1007079.s010">S3 Table</xref>). The resulting curated list contained 1,714 variants, at &gt;150 loci from 33 studies (<xref ref-type="supplementary-material" rid="pgen.1007079.s010">S3 Table</xref>). We used this list to represent the known genome-wide lipid and lipoprotein-associated variants.</p><p>We LD-pruned (r<sup>2</sup>&gt;0.95) the compiled list of 1,714 variants (<xref ref-type="supplementary-material" rid="pgen.1007079.s010">S3 Table</xref>) to 885 variants (<xref ref-type="supplementary-material" rid="pgen.1007079.s011">S4 Table</xref>) and we used this list (n = 885) in genome-wide conditional tests of association; this stringent LD threshold facilitates conditioning on multiple known signals at a locus. Signals that remained significant (<italic>P</italic><sub><italic>conditional</italic></sub>&lt;5&#215;10<sup>&#8722;8</sup>) after genome-wide conditional analysis were considered novel and further tested by single-variant conditional analyses to determine independence. At each of the five loci (<xref ref-type="table" rid="pgen.1007079.t001">Table 1</xref>), variants within 1 Mb up- and downstream of the lead variant and on the compiled list of 1,714 variants were included in single-marker conditional analyses (<xref ref-type="supplementary-material" rid="pgen.1007079.s013">S6 Table</xref>). Signals that remained significant (<italic>P</italic><sub><italic>single</italic></sub>&lt;5&#215;10<sup>&#8722;8</sup>) after single-variant conditional analysis were considered independent. Signals that only achieved a significance threshold of <italic>P</italic><sub><italic>single</italic></sub>&lt;5&#215;10<sup>&#8722;6</sup> after single-variant conditional analysis were considered distinct. The single-variant conditional analyses considered variants within 1 Mb of the signal, which accounts for &lt;1% of the genome. Therefore, the significance thresholds for the distinct and independent additional signals are conservative. At each locus, we validated that signals were distinct/independent by reciprocal conditional analysis with the putative novel lead associated variant for the trait. Additionally, the association data for the novel signals was adjusted for each of the four conventional traits (HDL-C, LDL-C, TC, and TG), and the effect of the association is reported in <xref ref-type="supplementary-material" rid="pgen.1007079.s014">S7 Table</xref>.</p><p>For each of the 885 lipid/lipoprotein-associated variants (<xref ref-type="supplementary-material" rid="pgen.1007079.s011">S4 Table</xref>), we determined whether the variant showed stronger association with one of the 68 subclass traits compared to the four conventional lipid traits (TC, TG, LDL-C, HDL-C). Variants were included for comparison if the variant association with a subclass trait satisfied <italic>P</italic>&lt;5&#215;10<sup>&#8722;5</sup> and if&#8211;log<sub>10</sub>pvalue for subclass trait association was greater than any of the four conventional lipid traits.</p><p>To determine the contribution of rare coding variants, we used the Optimal Sequence Kernel Association Test (SKAT-O) with EMMAX, as implemented in EPACTS, to test for gene-based associations with the 72 lipoprotein traits[<xref ref-type="bibr" rid="pgen.1007079.ref061">61</xref>]. Only coding variants directly genotyped on the OmniExpress or Exome array were included, resulting in 709,600 variants. Since SKAT-O requires no missing data, we imputed missing genotype data with the variant mean genotype. We annotated coding variants using VEP. These annotations were the basis for four masks that we implemented in the gene-based tests, as previously described[<xref ref-type="bibr" rid="pgen.1007079.ref053">53</xref>]. Briefly, the four masks were: <underline>Protein-Truncating Variants (PTV)</underline>: no MAF limit; variants are nonsense, frameshift, or essential splice variants. <underline>PTV+missense</underline>: MAF&lt;1%; all PTVs and missense variants. <underline>PTV+Nonsynonymous strict (NS</underline><sub>strict</sub>): no MAF limit; all PTVs and missense variants predicted as deleterious by five variant annotation algorithms: LRT, Mutation Taster, PolyPhen2-HumDiv, PolyPhen2-HumVar, and SIFT. <underline>PTV+NS</underline><sub>strict</sub><underline>+NS</underline><sub>broad</sub>: MAF&lt;1%; all variants in PTV+NS<sub>strict</sub> and variants predicted to be deleterious by any of the five algorithms above. Only genes containing two or more variants in a given mask were tested. We conducted gene-based conditional analyses to determine whether a single variant or a net-effect of multiple variants could explain the observed association signal.</p><p>To better characterize the novel signals from this study, we determined whether the associated lead variants and LD proxies (<italic>r</italic><sup><italic>2</italic></sup>&gt;0.7 in METSIM) at each signal were within ChIP-seq peaks of epigenomic transcriptional regulatory elements (<xref ref-type="supplementary-material" rid="pgen.1007079.s015">S8 Table</xref>). We built lists of such elements using data from the ENCODE Consortium[<xref ref-type="bibr" rid="pgen.1007079.ref062">62</xref>] and Roadmap Epigenomics Project[<xref ref-type="bibr" rid="pgen.1007079.ref063">63</xref>]. We used datasets from three lipid and cholesterol relevant tissues (adipose, blood, and liver datasets) that were comprised of experimentally defined regions of transcription factor binding sites (ChIP-seq), open chromatin (DNase- and FAIRE-seq), and histone modification marks (H3K4me1, H3K4me2, and H3K4me3, H3K27ac, and H3K9ac).</p>