<p>Bacterial strains, plasmids and primers are listed in <xref ref-type="supplementary-material" rid="pgen.1007426.s015">S1</xref>&#8211;<xref ref-type="supplementary-material" rid="pgen.1007426.s017">S3</xref> Tables, respectively. Cloned regions were confirmed by sequencing. <italic>E</italic>. <italic>coli</italic> DH5&#916;<italic>lac</italic> (BR2846) and Top10 (Invitrogen, Carlsbad, CA) were used for standard plasmid manipulations and propagation. <italic>V</italic>. <italic>cholerae</italic> strains used were derivatives of CVC1121, a <italic>hapR</italic><sup>+</sup> derivative of the sequenced strain N16961. <italic>E</italic>. <italic>coli</italic> strains were grown at 37 &#176;C and <italic>V</italic>. <italic>cholerae</italic> at 30 &#176;C. Antibiotics and their concentrations used in <italic>E</italic>. <italic>coli</italic> were: ampicillin 100 &#956;g/ml, chloramphenicol 25 &#956;g/ml, kanamycin 25 &#956;g/ml, nourseothricin (Nat) (Gold Biotechnology, St. Louis, MO) 25 &#956;g/ml, spectinomycin 50 &#956;g/ml and zeocin (Invitrogen) 25 &#956;g/ml. In <italic>V</italic>. <italic>cholerae</italic>, antibiotic concentrations used were the same except that chloramphenicol was used at 5 &#956;g/ml and kanamycin at 12.5 &#956;g/ml when cultures were grown in supplemented 1XM63 medium (1X M63 buffer, 1 mM CaCl<sub>2</sub>, 1 mM MgSO<sub>4</sub>, 0.001% vitamin B1, 0.2% fructose and 0.1% casamino acids) for microscopy. Inducers were used at the following concentrations when desired: IPTG at 100 &#956;M, arabinose at 0.2%, and anhydrotetracycline (aTc) at 20 ng/ml.</p><p>This was used for most genetic manipulations of the <italic>V</italic>. <italic>cholerae</italic> chromosomes [<xref ref-type="bibr" rid="pgen.1007426.ref047">47</xref>]. Cells were grown to log phase in LB medium at 30 &#176;C and 1 ml of culture was washed twice in 1X M9 medium supplemented with 32 mM MgCl<sub>2</sub> and 5 mM CaCl<sub>2</sub>. The washed cells were incubated with autoclaved chitin flakes (Sigma-Aldrich, St. Louis, MO) for 24 h at 30 &#176;C without shaking. About 200 ng of linear DNA fragments (generated by either PCR or restriction digestion) with ~1 kb homology region at each flank were added to the cells and gently mixed, and the mixture was incubated for 24 h at 30 &#176;C. The chitin flakes were vortexed to facilitate detachment of cells from the chitin surface and plated to select for transformants. The genetic alterations were confirmed by PCR, using primers flanking the altered regions.</p><p>The plasmid pRR06, containing <italic>araC</italic>, <italic>pbad</italic>-<italic>tus</italic>, <italic>lacI</italic><sup>q</sup>, <italic>ptac</italic>-(<italic>tdTomato</italic>-pMT<italic>parB gfp</italic>-P1<italic>parB</italic>) and the <italic>cat</italic> cassette, was constructed using the Gibson Assembly kit (New England Biolabs, MA). The <italic>tdTomato</italic> gene was amplified from pALA2486 using primers RR53 and RR40, and pMT<italic>parB</italic> gene from pALA2819 using primers RR51 and RR41. The two genes were fused in frame and inserted between <italic>ptac</italic> and <italic>gfp</italic>-P1<italic>parB</italic> in pALA2705 replacing the <italic>mCherry-pMTparB</italic> to yield plasmid pRR01. The 6 kb <italic>lacI</italic><sup>q</sup>
<italic>ptac</italic>-(<italic>tdTomato</italic>-pMT<italic>parB gfp</italic>-P1<italic>parB t1t2</italic>) from pRR01 was an amplified using primers RR07 and RR29. The <italic>araC pbad</italic>-<italic>tus</italic> fragment was amplified from plasmid pBAD30-<italic>tus</italic> using primers RR05 and RR06, the <italic>cat</italic> gene from pACYC184 using primers RR09 and RR10, the 1 kb upstream and downstream flanking regions of <italic>lacZ</italic> from genomic DNA of CVC1121 using primers RR03 and RR04, and RR11 and RR12, respectively. Using the Gibson kit, the above fragments were assembled in the sequence: upstream <italic>lacZ</italic> flank, <italic>araC pbad-tus</italic>, <italic>lacI</italic><sup>q</sup>
<italic>ptac</italic>-(<italic>tdTomato</italic>-pMT<italic>parB gfp</italic>-P1<italic>parB t1t2</italic>), <italic>cat</italic>, and downstream <italic>lacZ</italic> flank in pGEM-T-easy vector (Promega, WI). The ~12 kb region was excised using Not1 and used in natural transformation of CVC1121. Following selection on chloramphenicol plates, the resulting strain (CVC3001) was confirmed for Tus expression under arabinose control by Western blotting, and for tdTomato-pMTParB and GFP-P1ParB expression under IPTG control by fluorescence microscopy.</p><p>P1<italic>parS</italic> Kn (at +135 kb on Chr1) and pMT<italic>parS</italic> Sp (at +40 kb on Chr2) were inserted in CVC1121 by natural transformation with genomic DNA from CVC1146 [<xref ref-type="bibr" rid="pgen.1007426.ref017">17</xref>]. While the resulting strain CVC3009 showed the GFP-P1ParB (<italic>ori1</italic>) focus in all cells, it did not show tdTomato-pMTParB (<italic>ori2</italic>) focus in ~20% of the cells possibly due to slow maturation of tdTomato. This discouraged us from using tdTomato except when simultaneous monitoring of the two colors was necessary (such as time-lapse microscopy of Chr1-blocked cells). In situations where a more accurate count of <italic>ori2</italic> foci was desired, such as in Figs <xref ref-type="fig" rid="pgen.1007426.g003">3A</xref>, <xref ref-type="fig" rid="pgen.1007426.g006">6B and 6C</xref>, <xref ref-type="supplementary-material" rid="pgen.1007426.s004">S4</xref> and <xref ref-type="supplementary-material" rid="pgen.1007426.s009">S9B</xref> Figs, GFP was used to mark <italic>ori2</italic> instead of tdTomato and <italic>ori1</italic> was not marked. In these cases, P1<italic>parS</italic>-Kn was inserted at +40 kb from <italic>ori2</italic> by natural transformation of strain CVC3001 with genomic DNA from CVC1166 that resulted in strain CVC3058.</p><p>The <italic>E</italic>. <italic>coli ter</italic> sites (23 bp) when properly oriented and bound by Tus protein serve as roadblocks to replication fork progression [<xref ref-type="bibr" rid="pgen.1007426.ref018">18</xref>]. <italic>E</italic>. <italic>coli ter</italic> sites were inserted on either side of <italic>ori1</italic> in its active orientation, and the <italic>E</italic>. <italic>coli</italic> Tus protein was expressed from <italic>pbad</italic>. The <italic>ter</italic> sites were inserted by amplifying the <italic>ori1</italic> region of plasmid pGD101 with primers (RP017 and RP018) that added the 23 bp <italic>ter</italic> sites. In pGD101, <italic>ori1</italic> is linked to a <italic>zeo</italic> cassette that allows selection of transformants. A 1 kb region from each flank of <italic>ori1</italic> was amplified from <italic>gidA</italic> and <italic>mioC</italic> genes to provide homology for recombination in natural transformation experiments. The three fragments, <italic>gidA-terB</italic>, <italic>terB-ori1-zeo-terB</italic> and <italic>terB-mioC</italic> were joined in order by overlap extension PCR and the product was transferred to the chromosome by natural transformation. The resulting strain, CVC1157, proved to be inefficient in blocking replication forks of Chr1, necessitating addition of a second <italic>ter</italic> site in tandem to either flanks of <italic>ori1</italic>. The <italic>terB</italic>-flanked <italic>ori1</italic> region of CVC1157 was amplified in three ~1 kb segments as before using primers that added the 23 bp <italic>terC</italic> sites: RR66 and RR72 for <italic>gidA-terC</italic>, RR78 and RR79 for <italic>terC-terB-ori1-terB-terC</italic>, and RR80 and RR71 for <italic>terC-mioC</italic>. The three fragments were joined in an order to the pGEM-T-easy vector using the Gibson kit that resulted in pRR08. The assembled region (<italic>gidA terC terB ori zeo terB terC mioC</italic>, about 3 kb) was amplified from pRR08 and the product used in the natural transformation of CVC3009 that generated CVC3019. The Kn cassette of P1<italic>parS</italic> Kn was excised by supplying the Flp recombinase from pBRFLP as the cassette was flanked with FRT sites. The resulting Kn-sensitive derivative of CVC3019, CVC3022, was used in <xref ref-type="fig" rid="pgen.1007426.g001">Fig 1</xref>.</p><p>The copy number was approximated by measuring the relative marker frequency of the kanamycin resistance gene, <italic>kanR2</italic>, present in p<italic>crtS</italic>, to <italic>ori1</italic> on the chromosome. An aliquot of cell culture grown similarly as for time-lapse microscopy was diluted 1:100 in H<sub>2</sub>O, heated at 95 &#176;C for 10 mins and frozen at -20 &#176;C prior to being used directly as template for qPCR, which was performed as described previously [<xref ref-type="bibr" rid="pgen.1007426.ref012">12</xref>]. The ratios were normalized to the ratio obtained from the DNA of CVC3013, which contains a <italic>kanR2</italic> inserted into <italic>lacZ</italic>, thus possessing a <italic>kanR2/ori1</italic> marker frequency ratio of ~1. Data represents mean &#177; standard deviation and was averaged from two biological replicates analyzed in duplicate.</p><p>A copy of <italic>crtS</italic> was inserted 10 kb upstream of the native <italic>crtS</italic> locus in the intergenic region of VC0754 and VC0755. The upstream 1 kb (805058 to 805991) and downstream 1 kb (806014 to 83000) regions of the insertion site, and <italic>crtS</italic> were amplified from N16961 genomic DNA, and the <italic>nat1</italic> gene conferring nourseothricin resistance was amplified from pAM101 [<xref ref-type="bibr" rid="pgen.1007426.ref048">48</xref>]. The amplified DNA fragments were assembled using the Gibson kit into pGEM-T-easy vector in the following order: Upstream 1 kb region, <italic>crtS</italic>, <italic>nat1</italic> and downstream 1 kb region. The assembled region (~ 3.3 kb) of the resulting plasmid, pRR16, was isolated by Not1 digestion and the fragment was used in natural transformation of CVC3022 and CVC3058 to generate strains CVC3056 and CVC3061, respectively, upon selection on nourseothricin plates. A copy of <italic>crtS</italic> was also inserted 900 kb downstream of the native <italic>crtS</italic> locus in the intergenic region of VC1710 and VC1711 at 1.84 Mb in Chr1. The regions flanking the insertion site were already present in pPS64. The <italic>crtS</italic> linked to <italic>nat1</italic> present in pRR16 was cloned into the PstI site of pPS64, resulting in pRR20. A Not1 fragment of pRR20 was used in natural transformation of CVC3022 and CVC3058 to generate CVC3092 and CVC3093, respectively. To insert a third <italic>crtS</italic>, the <italic>nat1</italic> gene was replaced with a Kn cassette in pRR16, and the Not1 fragment from the resultant plasmid, pRR25, was used in natural transformation of CVC3092 and CV3093 to generate strains CVC3150 and CVC3151, respectively, by selection on Kan plates.</p><p>Plasmid pBJH245 contains the natural flanks of <italic>crtS</italic> and a <italic>zeo</italic> cassette in place of <italic>crtS</italic> [<xref ref-type="bibr" rid="pgen.1007426.ref014">14</xref>]. The <italic>zeo</italic> cassette and the flanks were amplified from pBJH245. This product was used in natural transformation of CVC3061 to generate CVC3112 with the <italic>zeo</italic> cassette replacing the native <italic>crtS</italic> locus at 0.81 Mb and retaining only the <italic>crtS</italic> copy 10 kb upstream of the native locus.</p><p><italic>crtS</italic> was excised in real-time using the site-specific recombination system of &#934;C31 phage. <italic>crtS</italic> was flanked with &#934;C31 <italic>attP</italic> and <italic>attB</italic> sites, and the &#934;C31 <italic>int</italic> gene was expressed from an aTc (IBA Lifesciences, Germany) inducible promoter, <italic>ptet</italic>. To monitor the loss of <italic>crtS</italic>, the gene was linked to a <italic>nat</italic> cassette [<xref ref-type="bibr" rid="pgen.1007426.ref048">48</xref>]. The <italic>crtS-nat</italic> region was amplified from pRR16. The <italic>attP</italic> and <italic>attB</italic> sites were amplified separately from pTF6. pTF6 contains 50 bp <italic>attB</italic> and <italic>attP</italic> sites flanking the origin of pACYC184 and was constructed by inserting an <italic>attP</italic> p15A<italic>ori attB</italic> double-stranded IDT geneblock ligated into pACYC184 digested with SacII and XbaI. The cloning was done using the Gibson kit. The <italic>attP</italic> and <italic>attB</italic> sequences used were AGTAGTGCCCCAACTGGGGTAACCTTTGAGTTCTCTCAGTTGGGGGCGTA and CCGCGGTGCGGGTGCCAGGGCGTGCCCTTGGGCTCCCCGGGCGCGTACTCC, respectively. The natural flanks of <italic>crtS</italic> and the vector backbone were amplified from pBJH245. The amplified fragment (left-flank-of-<italic>crtS</italic> vector right-flank-of-<italic>crtS</italic>) was combined with <italic>attP</italic>, <italic>crtS-nat</italic> and attB fragments using the Gibson kit that generated pRR17 (which is same as pBJH245 except for the presence of <italic>attP</italic>-<italic>crtS</italic>-<italic>nat</italic>-<italic>attB</italic> in place of the <italic>zeo</italic> cassette). A linear 3.3 kb Not1 digested fragment that contained <italic>crtS</italic> and the flanking regions but not the vector sequences was used in natural transformation of CVC3058 to generate CVC3068 upon selection on nourseothricin plates. To supply the &#934;C31 integrase under the control of <italic>ptet</italic> promoter, plasmid pRR21 was constructed which carried in order <italic>bla</italic>, <italic>tetR</italic> and <italic>ptet</italic>- &#934;C31<italic>int</italic> genes in place of <italic>crtS</italic> of pRR16. The homology regions present in pRR16 allowed insertion of the elements 10 kb upstream of the native <italic>crtS</italic> locus. The <italic>int</italic> gene was obtained from plasmid pInt (Addgene plasmid #18941) [<xref ref-type="bibr" rid="pgen.1007426.ref049">49</xref>] and was cloned downstream of <italic>ptet</italic> of plasmid pASK-IBA32 (IBA Life Sciences, Germany). The latter plasmid was also the source of <italic>bla</italic> and <italic>tetR</italic>. The <italic>bla</italic> gene allowed selection of transformants. The start codon of the <italic>int</italic> gene was changed from ATG to TTG to reduce the uninduced level of Int expression. The elements were assembled into a pGEM-T-easy vector by the Gibson kit that resulted in pRR21. A 4 kb Not1 fragment of pRR21 was used in the natural transformation of CVC3068 to generate CVC3082 upon selection on ampicillin plates. Genomic DNA of CVC3082 was used in the natural transformation of CVC3022 to generate CVC3101 upon selection on ampicillin and nourseothricin plates. CVC3101 contains the inducible systems to delete <italic>crtS</italic> and to block Chr1 replication by the Tus-<italic>ter</italic> complex.</p><p>Single colonies grown overnight on LB plates with antibiotics were used to inoculate the supplemented M63 medium and the cultures were grown at 30 &#176;C (for better folding of fluorescent proteins) to log phase (OD<sub>600nm</sub> 0.3). They were diluted two-fold in the same medium to which 0.2% arabinose or 20 ng/ml aTc was added and the growth was continued 90 min or 45 min, respectively. 5 &#956;l of cell culture was spotted on the center of a glass P35 dish (MatTek corporation, Ashland, MA) and the spot was overlaid with 1% agarose gel pad made with the same supplemented M63 medium. Time-lapse microscopy was performed starting at 90 min after inducer addition, with images taken every 15 min, unless otherwise specified, on a Nikon TiE inverted microscope with Nikon 100x/1.4 Oil Plan Apo Ph3 DM lens, Lumencor sola light engine (Beaverton, OR), and ImageEM EMCCD camera (Hamamatsu, Japan). The fluorescence from GFP was detected using 10% laser output and 300 msec exposure, and from td-Tomato using 30% laser output and 700 msec exposure.</p><p>Single colonies grown overnight on LB plates were used to inoculate the supplemented M63 medium and the cultures were grown at 30 &#176;C to log phase (OD<sub>600nm</sub> 0.1). They were diluted 10-fold and loaded onto a CellASIC ONIX B04A Microfluidic Bacteria plate (Millipore, Billerica, MA) primed with the supplemented M63 medium with or without 0.2% arabinose. The experiments were carried out in a CellAsic ONIX2 manifold (Millipore) with 5 &#956;l/h flowrate (2 psi), with aeration at 37 &#176;C and images were taken every 10 min. Subsequent image processing was performed on the Fiji platform as described in the main text.</p><p>Microscopy images were processed to subtract background and noise using the Fiji platform [<xref ref-type="bibr" rid="pgen.1007426.ref050">50</xref>]. The data from the images (number of foci, cell length etc.) were determined using the MicrobeJ plugin [<xref ref-type="bibr" rid="pgen.1007426.ref051">51</xref>]. All statistical analyses were performed using Prism7 (GraphPad Software, CA). Images were captured for each strain, quantified and the data presented as mean (from three biological replicates) &#177; standard error of mean (SEM). Unpaired two-tailed Student&#8217;s <italic>t</italic>-tests were performed on the pairs of triplicate values being compared.</p><p>Frequencies of four markers, <italic>ori1</italic>, <italic>ori2</italic>, <italic>ter1</italic> and <italic>ter2</italic>, from exponentially growing cells in LB medium with or without 0.2% arabinose were determined by qPCR using a MJ Research PTC-200 Peltier Thermal Cycler (Bio-rad, Hercules, CA) and a LightCycler 480 SYBR Green I master mix (Roche Life Sciences, IN), as described [<xref ref-type="bibr" rid="pgen.1007426.ref012">12</xref>].</p><p>Chemically competent DH5&#945; cells carrying p<italic>rctB</italic> (pTVC14) or p<italic>rctB</italic>L156R (pJJ263, which supplies a DnaK-interaction defective RctBL156R mutant) and either p<italic>crtS</italic> (pBJH188) or the vector (pACYC177) were mixed with 100 ng of p<italic>ori2</italic> (pTVC31). 10 &#956;l of the mixture, after allowing 2 h for expression of drug-resistance in LB medium, was used to inoculate 200 &#956;l of LB supplemented with chloramphenicol (12.5 &#956;g/ml) in 96-well plates. The plates were incubated at 37 &#176;C with shaking in the Synergy HT plate reader (BioTek, USA) and OD at 600nm was monitored.</p><p>Cells were inoculated in LB medium and grown at 30 &#176;C overnight. Genomic DNA was extracted from one ml of the culture using the DNeasy Tissue Kit (Qiagen, Hilden, Germany). DNA samples were quantified using a Bioanalyzer (Life Technologies, CA) and sequenced by the NCI CCR genomics sequencing core on the Illumina MiSeq platform. 2-5 million reads were obtained per sample, which were trimmed and mapped to the published N16961 reference genome [<xref ref-type="bibr" rid="pgen.1007426.ref052">52</xref>] on the CLC genomics workbench (Qiagen).</p><p>Total 4.0 OD units of cells grown in supplemented M63 medium was collected and resuspended in 100 &#956;l of 3x SDS sample buffer, boiled for 10 min and centrifuged for 1 min at 13,000 rpm. Aliquots were loaded onto a precast 8&#8211;16% Criterion TGX gel (Bio-Rad, Hercules, CA). Following electrophoresis, the proteins were transferred onto a nitrocellulose membrane and probed with RctB or Tus primary antibodies, as previously described [<xref ref-type="bibr" rid="pgen.1007426.ref053">53</xref>].</p>
