<p>The Ksp-cadherin, FLAG-MITF-A fusion gene was generated by cloning the full-length mouse MITF-A tagged at the 5&#8242;end by a FLAG epitope into the Ksp/BGH/link plasmid (kindly provided by Peter Igarashi) containing the minimal Ksp-cadherin promoter followed by the beta-globin enhancer element [<xref ref-type="bibr" rid="pgen.1007093.ref023">23</xref>]. The poly(A) sequence for the human beta-globin gene was isolated and cloned into the Ksp-cadherin-FLAG-MITF-A plasmid. Transgenic mouse lines were generated by microinjecting the purified DNA construct into fertilized mouse oocytes derived from a pure genetic background FVB/N. Twenty FLAG-MITF-A transgenic founder lines were identified by PCR analysis of tail DNA using primers specific to the MITF-A transgene (<bold><xref ref-type="supplementary-material" rid="pgen.1007093.s003">S3 Table</xref></bold>).</p><p>To generate the targeting construct, the potential <italic>Mitfa</italic> promoter/<italic>Mitfa</italic> exon and its flanking regions (15,942 bp) were cloned using plasmid rescue from BAC RP23-77E9. A floxed neomycin resistance expression cassette flanked by 200 bp of sequence flanking the <italic>Mitfa</italic> promoter/<italic>Mitfa</italic> exon was used to replace 5,956 bp of the putative <italic>Mitfa</italic> promoter/<italic>Mitfa</italic> exon from the above plasmid and used for standard targeting of LC3 ES cells (genotype [C57BL/6Nx129S6]F1). A correctly targeted ES cell colony was used to generate chimeric animals. Of several germline-transmitting lines, one was selected and crossed with C57BL/6JN/129S4-Prm1-Cre deleter mice (Jackson Laboratories, stock 003328, backcrossed twice to C57BL/6J) to remove the floxed neomycin cassette. Offspring lacking the neo-cassette were backcrossed to C57BL/6J mice for nine generations and then bred to homozygosity. Wild-type littermates were always used as controls. The primers used for genotyping are indicated in <bold><xref ref-type="supplementary-material" rid="pgen.1007093.s003">S3 Table</xref></bold>.</p><p>To generate MITF-A<sup>wt/tgMITF-A</sup><italic>;Ret</italic><sup><italic>wt/-</italic></sup> mice, heterozygous FVB/N mice overexpressing MITF-A (see above) were crossed with heterozygous C57BL/6 <italic>Ret</italic> knockout mice [<xref ref-type="bibr" rid="pgen.1007093.ref038">38</xref>]. Double heterozygous offsprings seemed normal and were born in the expected proportion.</p><p>Animals were fed ad libitum and housed at constant ambient temperature in a 12-hour light cycle. Animal procedures were approved by the Departmental Director of &#8220;Services V&#233;t&#233;rinaires de la Pr&#233;fecture de Police de Paris&#8221; and by the ethical committee of the Paris Descartes University as well as by the NIH/NINDS intramural program.</p><p>For the post-natal characterization of the MITF-A transgenic line, transgenic mice and wild-type littermates were sacrificed 2, 4, 6 and 12 months after birth (n = 5 to 12 for each genotype and time point). For renal function studies, urine samples were collected one week before sacrifice using metabolic cages over the course of 24 hours and blood samples were obtained at time of sacrifice (n = 4 to 6 for each genotype and time point) and kidneys dissected for appropriate studies. In addition, liver, spleen, heart, lung and brain were removed at 2 months for mRNA studies. For developmental studies, except where stated otherwise, the number of kidneys removed from E13.5 to P0 ranged between 14 and 25 for each genotype. For genetic interaction between <italic>Mitfa</italic> and <italic>Ret</italic> studies, mice were sacrificed at 3 weeks, kidneys were removed and the number of glomeruli was counted in females, exclusively (n = 3 to 7 for each genotype, i.e. MITF-A<sup><italic>wt/wt</italic></sup><italic>;Ret</italic><sup><italic>wt/wt</italic></sup>, MITF-A<sup>wt/tgMITF-A</sup><italic>;Ret</italic><sup><italic>wt/wt</italic></sup>, MITF-A<sup><italic>wt/wt</italic></sup><italic>;Ret</italic><sup><italic>wt/-</italic></sup>, <italic>and</italic> MITF-A<sup>wt/tgMITF-A</sup>;<italic>Ret</italic><sup><italic>wt/-</italic></sup><italic>)</italic>.</p><p>Plasma urea levels and urinary albumin, protein and creatinine levels were measured using an Olympus multiparametric autoanalyser (Instrumentation Laboratory).</p><p>For morphological analysis, kidneys were fixed in 4% paraformaldehyde, paraffin embedded, and 4-&#956;m sections were stained with periodic acid Schiff (PAS) or Hematoxylin &amp; Eosin (HE). A pathologist, blinded to the nature of the group, examined and evaluated all the sections. For morphometric analysis, glomerular and tubular surfaces were measured on PAS-stained sections, using a Nikon digital camera Dx/m/1200 and Lucia software (Laboratory Imaging Ltd., Prague, Czech Republic). Twenty randomly selected microscopic fields of the cortex were studied (X200). At least 20 glomeruli were analyzed for each animal. Tubular morphometric analysis was performed on sagittal kidney sections. The area of external profile of proximal tubules and the area of the lumen were measured, and the epithelial surface was calculated as the difference between these two areas. At least 35 tubular cross sections were analyzed for each animal. Tubular cell size was evaluated on the same tubular cross sections and was calculated as the ratio of epithelial surface to nuclei number for each tubular section.</p><p>Kidneys of two month-old mice were de-capsulated and macerated in 5N HCL for 30 minutes at 37&#176;C. After rinsing with distilled water, the suspension was stored overnight at 4&#176;C. The flask was shaken on the following day and the tubules and glomeruli were suspended in 25 ml of distilled water. Glomeruli were counted on five different 250 &#956;l aliquots using a Nikon Eclipse E800 microscope.</p><p>Kidneys from E13.5 mice were dissected and fixed in cold methanol for 15 minutes. After washing in PBS for 15 minutes, the entire kidneys were treated with 0.1% Triton in PBS for 2 hours. To visualize the branches and tips of the ureteric tree, kidneys were incubated overnight with primary mouse antibody against Calbindin-D28K (Sigma-Aldrich, 1:200), a calcium-binding protein expressed in the ureteric epithelium. Metanephroi were then washed in PBS and incubated with the secondary antibody AlexaFluor 555 goat anti-mouse IgG (Invitrogen, Molecular Probes, 1:100). The number of UB tips was counted using a Nikon Eclipse E800 microscope.</p><p>For cell proliferation, 4-&#956;m sections of paraffin-embedded kidneys were incubated with a rabbit anti-phospho-histone H3 antibody (Millipore) at 1:200 and a mouse anti-PCNA antibody (DAKO) at 1:50, followed by the appropriate secondary antibody. The number of proliferating cells was determined in UB and expressed as the number of pH3- or PCNA- positive cells per UB structure on whole kidney sections.</p><p>Apoptosis was detected in 4-&#956;m sections of paraffin-embedded kidneys by TUNEL assay using the In Situ Cell Death Detection kit (Roche) according to the manufacturer&#8217;s protocol. The number of apoptotic cells was determined as the number of TUNEL&#8211;positive nuclei per microscopic field in whole kidney sections. Ten microscopic fields were scored for each kidney in 5 embryos of each genotype.</p><p>Four &#956;m sections were retrieved with Tris-EDTA (TE) buffer (pH = 9.0) at 90&#176;C for 20 minutes, then incubated first overnight at 4&#176;C with a rabbit anti-MITF-A antibody [<xref ref-type="bibr" rid="pgen.1007093.ref021">21</xref>] 1/200, then with a secondary biotinylated anti-rabbit antibody (Vector) 1/500, followed by streptavidine-peroxidase (Dako) 1/500. Staining was revealed by 3&#8211;3&#8217;-diaminobenzidine (DAB).</p><p><italic>In situ</italic> hybridization was carried out on 10-&#956;m cryosections from OCT embedded kidneys frozen in isopentane solution under liquid nitrogen. Digoxigenin-dUTP&#8211;labeled sense and antisense RNA probes were synthesized using Roche reagents. The following cDNA templates were used: <italic>Ret</italic>, <italic>Wnt11</italic>, <italic>Pax2</italic>, <italic>BMP7</italic>, <italic>Wnt9b</italic> and <italic>Spry1</italic> (a kind gift from Isabelle Gross, INSERM U682). An RNA probe against MITF-A was obtained from cDNA encompassing both exon 1A and exon 1B. The resulting RNA probe could hybridize with the endogenous MITF-A and the MITF-A transgene and potentially with other MITF isoforms containing exon 1B (MITF-H, MITF-C, MITF-J, MITF-B, MITF-Mc). We designed this probe, since we failed to get a specific staining with a shorter probe (177 nucleotides) that overlaps only with MITF-A sequence. Cryosections were fixed with 4% paraformaldehyde for 10 min, acetylated for 10 min and then pre-hybridized for 3 hours. Hybridization was performed overnight at 60&#176;C in the presence of 1 &#956;g/ml for each RNA probe. After successive washings at 60&#176;C in 2X and 0.2X SCC buffer, the sections were washed in Tris buffer pH 7.6 (0.1 M Tris, 0.15 M NaCl) incubated in 2% blocking reagent solution (Roche) for 3 hours, and then incubated overnight with anti-digoxigenin antibody (alkaline phosphatase-conjugated Fab, Roche, 1:1,500). Sections were successively washed in Tris buffer, pH 7.6 and in Tris buffer, pH 9.5 (0.1 M Tris, 0.1M NaCl, 50 mM Mgcl2) and finally incubated with NBT/BCIP AP substrate solution (Roche).</p><p>For colocalization staining, the sections were successively fixed in 4% paraformaldehyde and incubated overnight with a rabbit antibody anti-laminin (Sigma-Aldrich), at 1:200, followed by an Alexa-Fluor 555 donkey anti-rabbit antibody (Invitrogen, Molecular Probes), at 1/200.</p><p>MITF-A immunoblotting was performed on nuclear extracts obtained after a 100,000 g ultracentrifugation of kidneys homogenized in 10 mM Hepes /1.7 M sucrose buffer (pH 7.9) supplemented with DTT (0.5 mM), spermine (0.15 mM), spermidine (0.5 mM), benzamidine (2 mM), anti-protease and anti-phosphatase inhibitors. The nuclear pellet was suspended in 8 M urea buffer and centrifuged. The supernatant was assayed for MITF-A immunoblotting using a rabbit antibody raised against a recombinant GST-MITF-A protein [<xref ref-type="bibr" rid="pgen.1007093.ref021">21</xref>]. A polyclonal antibody against Lamin-A/C (Epitomics) was used as a control for nuclear proteins.</p><p>Kidneys were incubated in RNA later (Ambion) overnight and stored at -80&#176;C until extraction. Total RNA was extracted from embryonic kidneys using RNeasy Micro Kit from Qiagen. RNAs were DNase treated (DNase I RNase-free, Qiagen) and reverse transcribed according to the manufacturer's protocols (Superscript II, Invitrogen). Quantitative RT-PCR was carried out using an ABI PRISM 7700 Sequence Detection system (Applied Biosystems). HPRT was used as the normalization control. The primers used (Eurogentec) are described in <bold><xref ref-type="supplementary-material" rid="pgen.1007093.s003">S3 Table</xref></bold>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Positional weight matric (PWM) was generated from 47 published functional MITF binding sites [<xref ref-type="bibr" rid="pgen.1007093.ref027">27</xref>] and by using the program MEME 3.5.0 (<ext-link ext-link-type="uri" ns0:href="http://meme.nbcr.net/meme/">http://meme.nbcr.net/meme/</ext-link>) to search both strands for putative motifs. The informative portion of the aligned sequences was converted into a PWM by assigning to each position/nucleotide the frequency of that given nucleotide in that given position. PWM sequence logo representation was created by submitting MEME informative sequences to WebLogo (<ext-link ext-link-type="uri" ns0:href="http://weblogo.berkeley.edu/">http://weblogo.berkeley.edu/</ext-link>) The identification of the conserved predicted MITF-A binding sites among the 102 kidney developmental genes (MGI abnormal kidney development data base) was then carried out as described previously [<xref ref-type="bibr" rid="pgen.1007093.ref026">26</xref>].</p><p>Data are expressed as means &#177; SEM. Differences between the experimental groups were evaluated using ANOVA, and, when significant (<italic>P</italic> &lt; 0.05), followed by the Tukey-Kramer test. When only two groups were compared, the Mann-Whitney test was used. The Pearson&#8217;s correlation coefficient was used to test correlation between variables. The statistical analysis was performed using Graph Prism Software.</p>