<p>We constructed an outbred population of flies using ten lines from the <italic>Drosophila</italic> Genetic Reference Panel (DGRP) [<xref ref-type="bibr" rid="pgen.1007098.ref080">80</xref>, <xref ref-type="bibr" rid="pgen.1007098.ref081">81</xref>] with the most extreme night sleep phenotypes for both sexes combined. Five lines had the shortest average night sleep for both males and females in the population: DGRP_38, DGRP_310, DGRP_365, DGRP_808, and DGRP_832 [<xref ref-type="bibr" rid="pgen.1007098.ref018">18</xref>]. The other five lines had the longest average night sleep in the population: DGRP_235, DGRP_313, DGRP_335, DGRP_338, and DGRP_379 [<xref ref-type="bibr" rid="pgen.1007098.ref018">18</xref>]. All ten lines were crossed in a full diallel design, resulting in 100 crosses. Two virgin females and two males from the F1 of each cross were then randomly mixed together and placed into 20 bottles, with 10 males and 10 females in each bottle. At each generation thereafter, 20 virgin females and 20 males from each bottle were randomly mixed with the other flies, and a new generation of 20 bottles was reared. Each generation of random mating had a census population size of 800. The genomes of these flies were allowed to recombine in this manner for 21 generations, resulting in an advanced intercross population of flies we named the Sleep Advanced Intercross Population (SAIP).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Flies were maintained and assayed on standard food (<ext-link ext-link-type="uri" ns0:href="http://flystocks.bio.indiana.edu/Fly_Work/media-recipes/bloomfood.htm">http://flystocks.bio.indiana.edu/Fly_Work/media-recipes/bloomfood.htm</ext-link>) in constant temperature (25&#176;C), humidity (60%), and light:dark cycle (12hr:12hr). To measure sleep, we collected male and female virgins from each population. Virgins were maintained in single-sex vials of 20 flies for four days before the experiment began (i.e., five vials per sex per population) to control for any potential effects on sleep of mating status [<xref ref-type="bibr" rid="pgen.1007098.ref125">125</xref>] and social exposure [<xref ref-type="bibr" rid="pgen.1007098.ref147">147</xref>]. Individual flies were loaded into Drosophila Activity Monitors (DAM2, Trikinetics, Waltham, MA) and sleep and activity parameters were recorded for five days. The monitors use an infra-red beam to detect activity counts in individual flies as they move past it; five minutes without an activity count is defined as sleep [<xref ref-type="bibr" rid="pgen.1007098.ref042">42</xref>, <xref ref-type="bibr" rid="pgen.1007098.ref043">43</xref>]. To mitigate the effects of CO<sub>2</sub> anesthesia or any other potential acclimation effects, the first day of data recording was discarded. Flies were visually examined after the sleep and activity recordings were completed; data from any flies that did not survive the recording period was discarded. Sleep Analysis v6-1 software (R. S. Barnes, personal communication) was used to calculate night and day sleep duration in minutes, night and day sleep bout number, and night and day average sleep bout length; it also calculated sleep latency, the time in minutes to the first sleep bout after lights are turned off and waking activity, the number of activity counts per minute spent awake. In addition, we calculated the coefficient of environmental variation (<italic>CV</italic><sub>E</sub>) for each sleep trait as (<italic>&#963;</italic><sub>E</sub>/<italic>&#956;</italic>)&#215;100 [<xref ref-type="bibr" rid="pgen.1007098.ref148">148</xref>], where <italic>&#963;</italic><sub>E</sub> is the within-replicate population standard deviation, and <italic>&#956;</italic> is the within-replicate mean. We used an in-house C# program (R. S. Barnes, personal communication) to partition the activity counts for each fly as asleep, awake, or in a transient pause (1 to 4 minutes of inactivity). We used this information to calculate the percentage of flies in each state over time.</p><p>At Generation 0 of the selection procedure, we divided the outbred population into six populations by seeding six bottles with 25 randomly chosen flies of each sex. Two populations served as replicate unselected controls (C1 and C2), two populations were selected for long night sleep (L1 and L2), and two populations were selected for short night sleep (S1 and S2). We measured sleep in 100 males and 100 females from each of these populations prior to initiating the artificial selection procedure. We calculated differences in sleep among these populations using an ANOVA model (see Quantitative genetics of selection below for the model). The resulting analyses indicated that there were no significant differences in any sleep trait among these populations, with the exception of night bout number (<xref ref-type="supplementary-material" rid="pgen.1007098.s001">S1 Fig</xref>; Generation 0 of <xref ref-type="supplementary-material" rid="pgen.1007098.s010">S2 Table</xref>). Thus, each population had essentially the same phenotypic characteristics prior to selection.</p><p>We implemented the following artificial selection procedure each generation. First, 100 virgins of each sex were collected from each population bottle and placed into the sleep monitors. Sleep and activity were monitored continuously over a five-day period. Second, we calculated all sleep parameters for every fly in each selection population. For the four selection populations (L1, L2, S1, and S2), the 25 males and females with the most extreme (high or low) night sleep within each population were chosen as parents for the next generation of that population. For the control populations (C1 and C2), we chose 25 males and 25 females at random to be the parents for the next generation of each population. We repeated this procedure for 13 generations. After generation 13, the six populations were maintained by seeding culture bottles with 25 randomly chosen flies of each sex.</p><p>We subjected artificially-selected flies to sleep deprivation at Generation 60 using the following procedure. Individual flies were loaded into the DAM2 Trikinetics monitors using the protocol above. After allowing the flies to acclimate in the monitors for one day, we recorded normal sleep and activity for two days. We deprived the flies of sleep on the night of the third day using a standard vortexer (Troemner, Thorofare, NJ) with a custom mounting plate (Trikinetics, Waltham, MA). The vortexer gently shook (speed = 2.0) the flies for five seconds every minute of the 12-hour night sleep period. We allowed the flies to recover for two additional days. An identical set of flies was placed into Trikinetics monitors on a different shelf in the same incubator to serve as a non-sleep-deprived control. We repeated these experiments two times. We used this protocol to deprive the long- (L1 and L2) and short-sleeper (S1 and S2) populations. We also applied this protocol to deprive the <italic>Minos</italic> element lines (see below) of sleep. We used an in-house Python program to calculate the number of flies asleep during any given minute, and to compute the average amount asleep per 30 minutes. We examined the overall efficacy of the sleep deprivation protocol for each population/line separately using the ANOVA model <italic>Y</italic> = <italic>&#181;</italic> + <italic>T</italic> + <italic>D</italic> + <italic>S</italic> + <italic>T</italic>&#215;<italic>D</italic> + <italic>T</italic>&#215;<italic>S</italic> + <italic>D</italic>&#215;<italic>S</italic> + <italic>T</italic>&#215;<italic>D</italic>&#215;<italic>S</italic> + <italic>&#949;</italic>, where <italic>T</italic> is the control or sleep deprived treatment, <italic>D</italic> is day, and <italic>S</italic> is sex. To determine whether night and/or day sleep changed across days for the sleep-deprived group and their respective controls, we analyzed the data for each population/line separately using the reduced ANOVA model <italic>Y</italic> = <italic>&#956;</italic> + <italic>D</italic> + <italic>S</italic> + <italic>D</italic>&#215;<italic>S</italic> + <italic>&#949;</italic>, where <italic>D</italic> and <italic>S</italic> are as defined above. Post-hoc Tukey analysis was used to determine which days had significant changes in sleep duration. In addition, to assess sleep deprivation in the <italic>Minos</italic> lines, we assessed differences between mutants and controls per day using the model <italic>Y</italic> = <italic>&#956;</italic> + <italic>T</italic> + <italic>L</italic> + <italic>S</italic> + <italic>T</italic>&#215;<italic>L</italic> + <italic>T</italic>&#215;<italic>S</italic> + <italic>S</italic>&#215;<italic>L</italic> + <italic>T</italic>&#215;<italic>L</italic>&#215;<italic>S</italic> + <italic>&#949;</italic>, where <italic>T</italic> and <italic>S</italic> are as defined above and <italic>L</italic> is Line.</p><p>We evaluated two different life-history traits in the selected and control populations of flies. At generation 14, flies were assayed for lifespan. 100 male and female virgins of each population were placed into 10 replicate vials per sex per population, with 10 same-sex flies per vial. Every two days, the flies were tipped into vials containing fresh food, and deaths were recorded until all flies were dead. Flies were assayed for egg-to-adult viability for nine generations, beginning at generation 4 and continuing through generation 12. Two replicate fly cultures per population per generation were seeded with 25 flies of each sex. Parents were cleared from the vials after 3 days, and the numbers of adult flies emerging were counted from days 10&#8211;14.</p><p>To evaluate the effectiveness of the selection procedure, we used the equation <italic>Y</italic> = <italic>&#956;</italic> + <italic>Sel</italic> + <italic>Rep</italic>(<italic>Sel</italic>) + <italic>Sex</italic> + <italic>Gen</italic> + <italic>Sel</italic>&#215;<italic>Sex</italic> + <italic>Sel</italic>&#215;<italic>Gen</italic> + <italic>Rep</italic>(<italic>Sel</italic>)&#215;<italic>Sex</italic> + <italic>Rep</italic>(<italic>Sel</italic>)&#215;<italic>Gen</italic> + <italic>Sex</italic>&#215;<italic>Gen</italic> + <italic>Sel</italic>&#215;<italic>Sex</italic>&#215;<italic>Gen</italic> + <italic>Rep</italic>(<italic>Sel</italic>)&#215;<italic>Sex</italic>&#215;<italic>Gen</italic> + <italic>&#949;</italic>, where <italic>Y</italic> is night sleep duration, <italic>Sel</italic> is the selection procedure (Short or Long night sleep); <italic>Gen</italic> is the fixed effect of generation; <italic>Rep</italic> is the random effect of replicate population; and <italic>&#949;</italic> is the variance within populations [<xref ref-type="bibr" rid="pgen.1007098.ref068">68</xref>]. A significant (<italic>P</italic> &lt; 0.05) <italic>Sel</italic> term indicates that the selection procedure was effective, and a significant <italic>Sel</italic>&#215;<italic>Sex</italic> term reveals sex-specific responses to selection. A significant <italic>Sel</italic>&#215;<italic>Gen</italic> (<italic>Sel</italic>&#215;<italic>Sex</italic>&#215;<italic>Gen</italic>) term implies that the response to selection differed across generations (and sexes). We used the same model to determine whether the other sleep parameters and egg-to-adult viability had a correlated response to selection for night sleep. To further characterize sex- and generation-specific responses to selection for night sleep, we used the reduced model <italic>Y</italic> = <italic>&#956;</italic> + <italic>Sel</italic> + <italic>Rep</italic>(<italic>Sel</italic>) + <italic>Gen</italic> + <italic>Sel</italic>&#215;<italic>Gen</italic> + <italic>Rep</italic>(<italic>Sel</italic>)&#215;<italic>Gen</italic> + <italic>&#949;</italic> to evaluate sleep characteristics in each sex separately; and the reduced models <italic>Y</italic> = <italic>&#956;</italic> + <italic>Sel</italic> + <italic>Rep</italic>(<italic>Sel</italic>) + <italic>Sex</italic> + <italic>Sel</italic>&#215;<italic>Sex</italic> + <italic>Rep</italic>(<italic>Sel</italic>)&#215;<italic>Sex</italic> + <italic>&#949;</italic> and <italic>Y</italic> = <italic>&#956;</italic> + <italic>Sel</italic> + <italic>Rep</italic>(<italic>Sel</italic>) + <italic>&#949;</italic> to analyze the response to selection in each generation separately and for each generation in each sex separately, respectively. The model <italic>Y</italic> = <italic>&#956;</italic> + <italic>Sel</italic> + <italic>Rep</italic>(<italic>Sel</italic>) + <italic>Sex</italic> + <italic>Sel</italic>&#215;<italic>Sex</italic> + <italic>Rep</italic>(<italic>Sel</italic>)&#215;<italic>Sex</italic> + <italic>&#949;</italic> was used for the lifespan analysis. All analyses were implemented using SAS (Cary, NC). To assess the response of night sleep to selection, we used the breeder&#8217;s equation &#931;<italic>R</italic> = <italic>h</italic><sup>2</sup>&#931;<italic>S</italic> to calculate the realized heritability <italic>h</italic><sup>2</sup>, where &#931;<italic>R</italic> is the cumulated response to selection, calculated as the mean night sleep of the offspring minus the mean night sleep of the parental generation; and &#931;<italic>S</italic> is the cumulated selection differential, calculated as the mean night sleep of the selected parents minus the mean night sleep of the parental generation [<xref ref-type="bibr" rid="pgen.1007098.ref067">67</xref>]. The same equation was used to assess changes in night sleep in the unselected control, except that the cumulated response is more properly termed cumulative difference, and the slope is the coefficient of the regression rather than the realized heritability.</p><p>For each generation of selection, we flash-froze 35 flies of each sex from each of the selection and control populations. We extracted DNA from these flies using a cell lysis solution {1.58 g of Tris-HCl (Quality Biological, Gaithersburg, MD), 37.22 g EDTA disodium salt (Quality Biological, Gaithersburg, MD) and filled to 1 liter with RNase/DNase-free water, adjusting the pH to 8.0 with 10 M NaOH (Sigma Aldrich, St. Louis, MO) when necessary}. Flies were homogenized in 300 &#956;l of cell lysis solution and four to six 2.38 mm metal beads (Omni International, Kennesaw, GA) using an Omni Bead Ruptor {5 cycles of 5 seconds ON/ 5 seconds OFF&#8212;Setting 5 m/s, equal to 3100 rpm (Omni International, Kennesaw, GA)}. An additional 400 &#956;l of cell lysis solution, 120 &#956;l of 10% SDS (Invitrogen, Grand Island, NY), and 60 &#956;l of Proteinase K (Life Technologies, Grand Island, NY) were added and the samples were incubated at 65&#176;C for 1 hour. The lysate was then transferred to a clean 1.5 ml tube, leaving the beads behind. RNA was eliminated from the lysate by adding 3.5 &#956;l RNase A (20 mg/ml) (Life Technologies, Grand Island, NY), mixing, and incubating at 37&#176;C for 15 minutes. Protein precipitation was initiated by adding 200 &#956;l of Ammonium Acetate (Quality Biological, Gaithersburg, MD) solution (289.1 g of Ammonium Acetate in 500 ml of RNase/DNase-free water) to tubes chilled on ice for 5 minutes. Samples were vortexed at high speed for 20 seconds to mix, incubated on ice for 15 minutes, and then centrifuged at maximum speed for 5 minutes. The supernatant was then transferred to a clean 1.5 ml tube, and 3 &#956;l of glycogen (Qiagen, Valencia, CA) was added. 700 &#956;l of 100% isopropanol (VWR International, Radnor, PA) was added and mixed to precipitate the DNA; samples were incubated for 1 hour at -20&#176;C. The samples were then centrifuged at maximum speed for 5 minutes. The supernatant was poured off and the DNA pellet was washed with 600 &#956;l of 75% ethanol (NIH Supply Center, Gaithersburg, MD), then centrifuged at maximum speed for 5 minutes. The ethanol was poured off and the pellet air-dried for 15 minutes. Samples were re-hydrated in 120 &#956;l of RNase/DNase-free water.</p><p>DNA samples were then purified using a phenol-chloroform extraction. First, 80 &#956;l of 10 mM Tris (Quality Biological, Gaithersburg, MD), 1 mM EDTA, pH 7.8 and 200 &#956;l of fresh phenol:chloroform:isoamyl alcohol (25:24:1) (Sigma Aldrich, St. Louis, MO) were added to the DNA sample. The samples were vortexed for 30 seconds, then centrifuged at maximum speed in a 4&#176;C centrifuge for 5 minutes. The resulting upper layer (~ 170 &#956;l) was transferred to a fresh 1.5-ml tube. An equal volume of chloroform (NIH Supply Center, Gaithersburg, MD) (200 &#956;l) was added. The sample was vortexed and centrifuged at maximum speed in a 4&#176;C centrifuge for 5 minutes. The resulting upper layer (~ 150 &#956;l) was transferred to a fresh 1.5-ml tube. Next, 20 &#956;l of sodium acetate (NaOAc) (Sigma Aldrich, St. Louis, MO), 500 &#956;l of pure ethanol, and 1 &#956;l glycogen were added. The sample was mixed and placed in ice for 15 minutes. Afterwards, the samples were centrifuged at maximum speed in a 4&#176;C centrifuge for 30 minutes. The supernatant was removed, and the pellet washed with 0.5 ml of 70% ethanol. The sample was centrifuged at maximum speed for 5 minutes. The supernatant was removed and the sample air-dried for 5 minutes. The resulting DNA pellet was dissolved in 25 &#956;l sterile 10 mM Tris, 0.1 mM EDTA, pH 7.8 and heated for 2 minutes at 55&#176;C. DNA quantity was measured and quality evaluated (260/280 ratios of 1.8 or greater) using the Nanodrop 8000 (Thermo Fisher Scientific, Asheville, NC).</p><p>We sequenced DNA from each selection population and sex for the following generations: 0, 1, 2, 5, 8, 10, and 12. One microgram of genomic DNA was sheared to ~350 bp using a Covaris E210 (Covaris, Woburn, MA) with settings: duty cycle 10%; intensity 5; cycles/burst 200; time 45s. Libraries were constructed using the Tru-Seq DNA PCR-Free LT Sample Prep Kit (Illumina, San Diego, CA) according to the manufacturer&#8217;s protocol. The libraries were pooled in groups of 24 and run on a HiSeq2500 (Illumina, San Diego, CA) with version 3 sequencing reagents to generate a minimum of 37 million paired-end 126 base reads per library. The HiSeq data was processed using RTA1.18.64 and CASAVA 1.8.2.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Sequences were aligned and mapped to the <italic>D</italic>. <italic>melanogaster</italic> assembly BDGP Release 6, UCSC version dm6 using with Burrows-Wheeler Alignment-MEM version 0.7.12 [<xref ref-type="bibr" rid="pgen.1007098.ref149">149</xref>] and Novoalign version 3.02.07 (Novocraft Technologies, Selangor, Malaysia), using the -t 400 option to optimize alignment speed. We used previously identified informative SNPs and indels in Freeze 2.0 of the DGRP [<xref ref-type="bibr" rid="pgen.1007098.ref081">81</xref>], after converting the coordinates to the Release 6 assembly [<xref ref-type="bibr" rid="pgen.1007098.ref150">150</xref>]. Reads were realigned around indels from this set of polymorphisms with GATK version 2.8.1 [<xref ref-type="bibr" rid="pgen.1007098.ref151">151</xref>]. Duplicate reads were removed with samtools version 0.1.18 after alignment, merging, and sorting of the reads. Novel variants were discovered using LoFreq version 2.1.2 [<xref ref-type="bibr" rid="pgen.1007098.ref082">82</xref>] with default parameters. We retained variants called by LoFreq that occurred in 14 or more samples. Allele counts for all single nucleotide variant sites (both known DGRP Freeze 2.0 and novel LoFreq polymorphisms) were determined using the &#8216;bamcounts&#8217; command of the bardCNV package (<ext-link ext-link-type="uri" ns0:href="http://github.com/nhansen/BardCNV">http://github.com/nhansen/BardCNV</ext-link>) with the option -minqual 20 to filter reads for a minimum phred quality of 20. Counts for reads spanning indels were performed by first widening indel variants to their narrowest unambiguous region, then tallying spanning reads both with and without the indel using the perl module Bio::SamTools (<ext-link ext-link-type="uri" ns0:href="http://search.cpan.org/~lds/Bio-SamTools/lib/Bio/DB/Sam.pm">http://search.cpan.org/~lds/Bio-SamTools/lib/Bio/DB/Sam.pm</ext-link>). Confidence interval boundaries with the highest posterior density for the estimated read allele proportions were calculated in R using CRAN &#8216;binom&#8217; package&#8217;s &#8216;binom.bayes&#8217; function. For downstream analyses, we considered all bialleleic polymorphisms with total coverage of 10 and less than 2000 in each population [<xref ref-type="bibr" rid="pgen.1007098.ref056">56</xref>].</p><p>We used the Cochran-Mantel-Haenszel (CMH) Test to detect changes in allele frequency for all known segregating 2,222,264 and putative de novo 258,268 polymorphisms [<xref ref-type="bibr" rid="pgen.1007098.ref083">83</xref>] between generations 0&#8211;1, 1&#8211;2, 2&#8211;5, 5&#8211;8, 8&#8211;10, and 10&#8211;12. This test is unbiased, easy to apply, and has been previously used to identify candidate polymorphisms in evolve and re-sequence studies [<xref ref-type="bibr" rid="pgen.1007098.ref056">56</xref>, <xref ref-type="bibr" rid="pgen.1007098.ref059">59</xref>]. We removed polymorphisms with less than 10 total read counts or more than 2000 total read counts from the analysis. We used a Bonferroni-corrected <italic>P</italic>-value (2.3 &#215; 10<sup>&#8722;8</sup>) as the threshold of significance for the CMH tests in order to account for multiple tests. We made this calculation for both sexes combined as sleep in both sexes responded to the selection procedure (<italic>P</italic><sub>sel&#215;sex</sub> = 0.1501, <xref ref-type="supplementary-material" rid="pgen.1007098.s009">S1 Table</xref>). Despite the similarity in phenotypic changes among replicate long- or short-sleeper populations, few polymorphisms with significant allele frequency changes overlapped between replicate populations. However, significant CMH tests cannot distinguish among allele frequency changes that are responsible for the sleep phenotypes we observed, and changes due to other forces that might alter allele frequency. For example, large numbers of frequency changes were potentially due to random genetic drift or laboratory adaptation, as the numbers of significant polymorphisms in the control populations indicate. Nor can the CMH test account for differences in allele counts that arise due to differences in sequencing coverage [<xref ref-type="bibr" rid="pgen.1007098.ref059">59</xref>, <xref ref-type="bibr" rid="pgen.1007098.ref065">65</xref>], sampling bias, or sequencing error. We examined the overlap between candidate polymorphisms that were significant for the CMH test in the control population with those in the selected populations. We eliminated polymorphisms that overlapped with the control populations as being indistinguishable from random genetic drift. This left 69,188 polymorphisms that overlapped between the long- and short-sleeping populations. We then applied a logistic regression model to the read count data of the form: <italic>Y</italic> = <italic>&#946;</italic><sub>0</sub> + <italic>&#946;</italic><sub>1</sub><italic>Gen</italic> + <italic>&#946;</italic><sub>2</sub><italic>Sel</italic> + <italic>&#946;</italic><sub>3</sub><italic>Gen</italic>&#215;<italic>Sel</italic>, where <italic>Y</italic> is allele type (major or minor), <italic>Gen</italic> is generation, and <italic>Sel</italic> is selection scheme (Long or Short). Both long and short selection population replicates for all seven generations and both sexes were put into the logistic regression analysis. We required that the <italic>P</italic>-value for the lack of fit be &#8805; 0.05 (higher <italic>P</italic>-value indicates that there is no evidence to reject the null hypothesis that the model fits); the <italic>P</italic>-value for <italic>Sel</italic> to be &#8804; 0.05, and the <italic>P</italic>-value for <italic>Gen</italic>&#215;<italic>Sel</italic> to be &#8804; 0.05 [<xref ref-type="bibr" rid="pgen.1007098.ref152">152</xref>]. All analyses were implemented using SAS (Cary, NC).</p><p>We chose 3,600 SNPs at random per chromosome arm and calculated linkage disequilibrium among these SNPs for the 10 DGRP lines using plink 1.0.7 [<xref ref-type="bibr" rid="pgen.1007098.ref153">153</xref>]. To estimate LD between SNPs in the SAIP and resulting artificial selection populations, we used the following procedure. For each SNP pair, we designated one SNP as SNP A and the other as SNP B. We binned the minor allele frequency of SNP A, <italic>p</italic><sub>a</sub>, in 0.01 increments from 0.01 to 0.5. We then calculated the lower and upper bounds of the corresponding minor allele frequency at SNP B, <italic>p</italic><sub>b</sub>, that would result in an <italic>r</italic><sup>2</sup> of 0.8 or greater using the following formulations from VanLiere <italic>et al</italic>. [<xref ref-type="bibr" rid="pgen.1007098.ref089">89</xref>]:</p><p>The lower bound of <italic>p</italic><sub>b</sub> is <italic>r</italic><sup>2</sup><italic>p</italic><sub>a</sub>/(1 + <italic>r</italic><sup>2</sup><italic>p</italic><sub>a</sub>&#8722;<italic>p</italic><sub>a</sub>).</p><p>The upper bound of <italic>p</italic><sub>b</sub> is the minimum of <italic>p</italic><sub>a</sub>/(<italic>r</italic><sup>2</sup> &#8211;<italic>r</italic><sup>2</sup><italic>p</italic><sub>a</sub> + <italic>p</italic><sub>a</sub>), or 0.5.</p><p>We determined whether the actual minor allele frequency at SNP B was within the upper and lower bounds given by these two equations. If it was, we assumed that SNP B was in high LD (i.e., <italic>r</italic><sup>2</sup> &#8805; 0.8) with SNP A.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In addition, we used the <italic>Drosophila melanogaster</italic> Recombination Rate Calculator (<ext-link ext-link-type="uri" ns0:href="http://petrov.stanford.edu/RRcalculator.html">http://petrov.stanford.edu/RRcalculator.html</ext-link>) to estimate recombination rates for the 126 variants we identified using logistic regression. Estimates based on Marey map equations [<xref ref-type="bibr" rid="pgen.1007098.ref154">154</xref>] and on empirical observations of recombination [<xref ref-type="bibr" rid="pgen.1007098.ref086">86</xref>] are included in <xref ref-type="supplementary-material" rid="pgen.1007098.s015">S7 Table</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We compared the actual allele frequency changes in the control populations with simulated random genetic drift. We first estimated the variance effective population size, <italic>N</italic><sub>e</sub>, of our populations as
<disp-formula id="pgen.1007098.e001"><alternatives><graphic id="pgen.1007098.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1007098.e001.jpg" /><ns1:math id="M1"><ns1:msub><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>e</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mo>&#8722;</ns1:mo><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>l</ns1:mi><ns1:mi>n</ns1:mi><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msup><ns1:mrow><ns1:mi>&#963;</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msup></ns1:mrow><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>q</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>0</ns1:mn></ns1:mrow></ns1:msub><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>q</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>0</ns1:mn></ns1:mrow></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mfrac></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mfrac></ns1:math></alternatives></disp-formula>
where <italic>t</italic> is the time in generations, <italic>q</italic><sub>0</sub> is the starting allele frequency at generation 0, and <italic>&#963;</italic><sup>2</sup> is the variance in allele frequencies [<xref ref-type="bibr" rid="pgen.1007098.ref155">155</xref>]. Using the SNPs that had significant CMH tests in the control populations (i.e., those SNPs presumed to be drifting), we binned the allele frequencies at generation 0 from 0.05&#8211;0.50. We then calculated the variance in allele frequency across SNPs in the control populations at generation 12 [<xref ref-type="bibr" rid="pgen.1007098.ref065">65</xref>]. We estimated <italic>N</italic><sub>e</sub> using the equation above for each allele frequency bin at Generation 12. The median <italic>N</italic><sub>e</sub> was 42.5 (21 males and 21 females) which we applied to the drift simulations. Drift was simulated for a single SNP by random and independent sampling of parental genotypes over 12 generations for a given starting minor allele frequency [<xref ref-type="bibr" rid="pgen.1007098.ref065">65</xref>]. We simulated drift using starting minor allele frequencies that ranged from 0.01 to 0.5 in 0.01-frequency increments. Each simulation for a given starting minor allele frequency was repeated 10,000 times. The entire process was replicated 2 times for the autosomes and 2 times for the <italic>X</italic> chromosome, to represent the long-sleeper and short-sleeper populations with replication. We then calculated the absolute value of the average difference between short and long sleeper populations for each starting minor allele frequency. The 99.9% quantile was used as an upper bound of the potential allele frequency change due to drift for each starting minor allele frequency (65).</p><p>We tested <italic>Minos</italic> element <italic>Mi{ET1}</italic> insertion lines [<xref ref-type="bibr" rid="pgen.1007098.ref156">156</xref>] in three candidate genes: <italic>CG33156</italic>, <italic>fz</italic>, and <italic>sgg</italic>. The alleles tested were <italic>CG33156</italic><sup>MB05931</sup>, <italic>fz</italic><sup>MB07478</sup>, and <italic>sgg</italic><sup>MB03827</sup> (Bloomington, IN stock center). The <italic>Mi{ET1}CG33156</italic><sup>MB05931</sup> insertion is located on chromosome <italic>2R</italic> at position 13,549,134, within an exon common to all 5 isoforms of <italic>CG33156</italic>. The <italic>Mi{ET1}fz</italic><sup>MB07478</sup> insertion is located at position 14,303,880 on chromosome <italic>3L</italic> in the second intron of both isoforms of <italic>fz</italic>. The <italic>Mi</italic>{<italic>ET1</italic>}<italic>sgg</italic><sup>MB03827</sup> insertion is located at position 2,673,583 on the <italic>X</italic> chromosome; this insertion is in an exon common to all 17 isoforms of <italic>sgg</italic>. These insertions were created in an isogenic <italic>w</italic><sup>1118</sup> background (<italic>w</italic><sup>1118</sup>{5905}); we used this background as a control [<xref ref-type="bibr" rid="pgen.1007098.ref156">156</xref>]. Sleep phenotypes were measured as described above for 16 flies/sex/genotype in both the homozygous mutant lines and the trans-heterozygous lines, and the measures were repeated two times. Sleep phenotypes in each mutant were compared to the control using the ANOVA model <italic>Y</italic> = &#956; + <italic>G</italic> + <italic>S</italic> + <italic>R</italic> + <italic>G&#215;S</italic> + <italic>G</italic>&#215;<italic>R</italic> + <italic>R</italic>&#215;<italic>S</italic> + <italic>G</italic>&#215;<italic>S</italic>&#215;<italic>R</italic> + <italic>&#949;</italic>, where <italic>G</italic> is genotype, <italic>S</italic> is sex, <italic>R</italic> is replicate, and <italic>&#949;</italic> is the error term. Males and females were analyzed separately using the reduced ANOVA model <italic>Y</italic> = &#956; + <italic>G</italic> + <italic>R</italic> + <italic>G</italic>&#215;<italic>R</italic> + <italic>&#949;</italic>.</p><p>We also investigated potential interactions among these candidate genes by crossing them together. We crossed the three <italic>Minos</italic> lines in a full diallel design, so that all possible trans-heterozygous combinations were represented. Five virgin females of each line were crossed to five males of each line. <italic>sgg</italic> is located on the <italic>X</italic> chromosome; thus, male progeny from male <italic>sgg</italic> parents did not have the <italic>sgg</italic> mutation. We therefore restricted our analysis to males and females separately. We compared differences in sleep across trans-heterozygous genotypes using the ANOVA model <italic>Y</italic> = &#956; + <italic>G</italic> + <italic>R</italic> + <italic>G</italic>&#215;<italic>R</italic> + <italic>&#949;</italic> as defined above. Genotypic effects can be decomposed into general combining ability (GCA), specific combining ability (SCA) and reciprocal effect (REC) using the full diallel, fixed effects design (Method 1, Model I) of Griffing [<xref ref-type="bibr" rid="pgen.1007098.ref117">117</xref>]. The GCA of a mutation is the mean phenotype of the progeny bearing that mutation calculated as a deviation from the overall mean of all crosses; SCA is the deviation from the GCA and overall mean and reflects both dominance and epistatic interactions [<xref ref-type="bibr" rid="pgen.1007098.ref067">67</xref>, <xref ref-type="bibr" rid="pgen.1007098.ref118">118</xref>]. Mean sleep phenotypes of all crosses in the full diallel design were designated as <italic>Y</italic><sub><italic>ij</italic></sub>, where <italic>Y</italic> is the mean of the <italic>ij</italic>th genotype. We estimated the GCA of a mutation as (1/2n)(<italic>X</italic><sub>i.</sub> + <italic>X</italic><sub>.j</sub>)-(1/n<sup>2</sup>)<italic>X</italic><sub>..</sub>, where <italic>X</italic><sub><italic>i</italic>.</sub> and <italic>X</italic><sub>.j</sub> are the sums of mean sleep phenotypes of heterozygotes with the <italic>i</italic>th and <italic>j</italic>th <italic>Minos</italic> element, respectively; <italic>X</italic><sub>..</sub> is the sum of mean sleep phenotypes for all genotypes; and n equals the number of mutations (3 in this case) [<xref ref-type="bibr" rid="pgen.1007098.ref117">117</xref>]. We estimated SCA as (1/2)(<italic>Y</italic><sub>ij</sub> + <italic>Y</italic><sub>ji</sub>)&#8211;(1/2n)(<italic>X</italic><sub>i.</sub> + <italic>X</italic><sub>.i</sub> + <italic>X</italic><sub>j.</sub> + <italic>X</italic><sub>.j</sub>) + (1/n<sup>2</sup>)<italic>X</italic><sub>..</sub> [<xref ref-type="bibr" rid="pgen.1007098.ref117">117</xref>]. REC effects were estimated using the formula (1/2)(<italic>Y</italic><sub>ij</sub>&#8722;<italic>Y</italic><sub>ji</sub>) [<xref ref-type="bibr" rid="pgen.1007098.ref117">117</xref>]. The GCA is the average of a heterozygote relative to the other heterozygous effects, while SCA is a measure of dominance and/or epistatic effects [<xref ref-type="bibr" rid="pgen.1007098.ref157">157</xref>, <xref ref-type="bibr" rid="pgen.1007098.ref158">158</xref>]. We partitioned the genetic variance for GCA, SCA, and REC components for each sex using the ANOVA model <italic>Y</italic> = &#956; + <italic>GCA</italic> + <italic>SCA</italic> + <italic>REC</italic> + <italic>GCA</italic>&#215;<italic>R</italic> + SCA&#215;<italic>R</italic> + REC&#215;<italic>R</italic> +<italic>&#949;</italic>, where &#956; is the overall mean and <italic>&#949;</italic> is the error term [<xref ref-type="bibr" rid="pgen.1007098.ref117">117</xref>]. We used DIALLEL-SAS to evaluate the ANOVA models and estimate GCA, SCA, and REC effects [<xref ref-type="bibr" rid="pgen.1007098.ref159">159</xref>].</p><p>No <italic>Mi{ET1}</italic> lines were available for <italic>mip120</italic>, the candidate gene common to this study and the genome-wide association of sleep. However, many candidate genes were located along a 99.6-kb region on chromosome <italic>2R</italic>, including <italic>crowded by cid</italic>, <italic>mip120</italic> and <italic>CG33156</italic> from the present study, and <italic>arrow</italic>, <italic>downstream of receptor kinase</italic>, <italic>mars</italic>, <italic>mip120</italic>, <italic>Vmat</italic>, <italic>CG13330</italic>, and <italic>CG13331</italic> from the genome-wide association study [<xref ref-type="bibr" rid="pgen.1007098.ref018">18</xref>]. We therefore tested three deficiency lines [<xref ref-type="bibr" rid="pgen.1007098.ref160">160</xref>] <italic>Df(2R)BSC273</italic>, <italic>Df(2R)BSC274</italic>, and <italic>Df(2R)BSC306</italic>, which have breakpoints from 13159579&#8211;13502150, 13430464&#8211;13593272, and 13364133&#8211;13539889, respectively. These deletions were created in an isogenic <italic>w</italic><sup>1118</sup> background (<italic>w</italic><sup>1118</sup> {6326}); we used these backgrounds as a control [<xref ref-type="bibr" rid="pgen.1007098.ref160">160</xref>]. The chromosomal deficiency stocks were maintained over a <italic>CyO</italic> balancer chromosome. We crossed 5 virgin females of each deficiency line to 5 males of the <italic>w</italic><sup>1118</sup> {6326} control and compared the heterozygotes bearing the deficiency to the control line. Sleep phenotypes were measured as described above for 16 flies/sex/genotype, and the measures were repeated two times. Sleep phenotypes were analyzed using the ANOVA model <italic>Y</italic> = &#956; + <italic>G</italic> + <italic>S</italic> + <italic>G&#215;S</italic> + <italic>R(G&#215;S)</italic> + <italic>&#949;</italic>, where <italic>G</italic> is deficiency genotype, <italic>S</italic> is sex, <italic>R</italic> is replicate, and <italic>&#949;</italic> is the error term. All stocks were obtained from the Bloomington, Indiana stock center. All analyses were implemented using SAS (Cary, NC).</p><p><bold>Data availability:</bold> Raw DNA sequence data have been deposited in the NCBI SRA database under BioProject PRJNA369048, SRA database SRP098556.</p>