<p>Four studies from the CHARGE consortium genotyped a total of 1393 AD cases and 8141 cognitively intact controls for an ExomeChip (EC) genotyping array. All participants in the discovery phase of the analysis were of European or European American descent (<bold><xref ref-type="table" rid="pgen.1006327.t001">Table 1</xref></bold>). In the follow-up analysis we included Icelandic individuals from the AGES-followup cohort, African-American participants of the Cardiovascular Health Study (CHS), and European or European American individuals from the Alzheimer&#8217;s Disease Genetics Consortium (ADGC) and Genetic and Environmental Risk in Alzheimer&#8217;s Disease consortium (GERAD). Further details on phenotyping and other cohort characteristics are in the <bold>Supporting methods in <xref ref-type="supplementary-material" rid="pgen.1006327.s001">S1 Text</xref></bold>.</p><p>All participants provided informed consent and all studies were approved by their respective ethics committees.</p><p>All four CHARGE cohorts were genotyped for the HumanExome BeadChip v1.0 from Illumina (San Diego, CA, USA). Genotype calling and quality control was performed centrally as described previously [<xref ref-type="bibr" rid="pgen.1006327.ref030">30</xref>]. Each study also performed quality control locally (<bold>Supporting methods in <xref ref-type="supplementary-material" rid="pgen.1006327.s001">S1 Text</xref></bold>). Post-analysis quality control on the top results included visual inspection of cluster plots (<bold><xref ref-type="supplementary-material" rid="pgen.1006327.s002">S1 Fig</xref></bold>) and re-genotyping of <italic>TM2D3</italic> carriers using a TaqMan assay (genotype calls were 100% validated). <italic>TM2D3</italic> variant was genotyped in the AGES-followup cohort using the TaqMan assay. In ADGC and GERAD consortia the <italic>SKAP2</italic> and <italic>TM2D3</italic> variants were gentoyped on the Illumina HumanExome BeadChip v1.0 or v1.1 from Illumina (<bold>Supporting methods in <xref ref-type="supplementary-material" rid="pgen.1006327.s001">S1 Text</xref></bold>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In the discovery phase of our analysis we used score tests [<xref ref-type="bibr" rid="pgen.1006327.ref027">27</xref>] for the single-variant analysis and the Sequence Kernel Association Test (SKAT [<xref ref-type="bibr" rid="pgen.1006327.ref029">29</xref>]) to test for the aggregate effect of multiple low-frequency variants within a gene on LOAD. Each study adjusted for age, sex, and for those principal components associated with LOAD (see <bold>Supporting methods in <xref ref-type="supplementary-material" rid="pgen.1006327.s001">S1 Text</xref></bold>). The discovery analysis was not adjusted for <italic>APOE</italic> genotypes; however this important LOAD risk factor was considered in secondary analyses of top findings. Both tests were performed in R with the seqMeta package (<ext-link ext-link-type="uri" ns0:href="http://cran.r-project.org/web/packages/seqMeta/">http://cran.r-project.org/web/packages/seqMeta/</ext-link>) to meta-analyze study-specific scores and respective variances and covariances (see details in <bold>Supporting methods in <xref ref-type="supplementary-material" rid="pgen.1006327.s001">S1 Text</xref></bold>) [<xref ref-type="bibr" rid="pgen.1006327.ref028">28</xref>]. In the single-variant analysis we included variants present in at least two studies, variants with a minor allele frequency (MAF) &#8805; 0.5% or assuming that rare damaging alleles could be prevalent predominantly in cases we also included variants with minor allele count (MAC) &#8805; 5 in cases. This resulted in 52,026 single-variant tests. In the SKAT analysis we grouped the variants by genes using the start and stop positions as annotated by dbNSFP v2.0 [<xref ref-type="bibr" rid="pgen.1006327.ref055">55</xref>] to define gene boundaries. We included only variants of MAF &lt; 5% in the combined sample, and further annotated as missense, stop-gain, stop-loss, or splice-site variants. Then we required that genes have at least two such variants as well as at least two studies having a polymorphic variant. This resulted in 11,303 SKAT tests. After filtering, the Bonferroni correction thresholds that accounted for multiple testing were 9.6x10<sup>-7</sup> for single-variant tests and 4.4x10<sup>-6</sup> for SKAT. Two analysts independently performed the meta-analysis, leading to identical results.</p><p>We used a score test and Fisher&#8217;s exact test for the follow-up analysis based on an independent Icelandic AGES sample (AGES-followup). We report (<bold><xref ref-type="table" rid="pgen.1006327.t002">Table 2</xref></bold>) estimates of the ORs based on fitting the full model (AD ~ age + sex + <italic>TM2D3</italic>) instead of the one-step approximation used in seqMeta (see <bold>Supporting Methods in <xref ref-type="supplementary-material" rid="pgen.1006327.s001">S1 Text</xref></bold> for details). For the AGES-discovery cohort we similarly update and report estimates of the ORs (<bold><xref ref-type="table" rid="pgen.1006327.t002">Table 2</xref></bold>) based on fitting the full model (<bold><xref ref-type="supplementary-material" rid="pgen.1006327.s008">S1A Table</xref></bold>, <bold><xref ref-type="supplementary-material" rid="pgen.1006327.s009">S2 Table</xref></bold>). In the AGES study, we also used a mixed-model to account for potential confounding effect relatedness might have on the association of P155L in <italic>TM2D3</italic> with LOAD (<bold>Supporting results in <xref ref-type="supplementary-material" rid="pgen.1006327.s001">S1 Text</xref></bold>). Cox regression was used to test for association of P155L with age-at-onset in the AGES data after adjusting for sex and <italic>APOE</italic> genotype. Individuals with LOAD diagnosis at the baseline visit were excluded so only at-risk individuals were included in the analysis. As detailed in the <bold>Supporting methods in <xref ref-type="supplementary-material" rid="pgen.1006327.s001">S1 Text</xref></bold>, the survival analysis accounted for left-truncation (i.e. follow-up begins after 65 years) and right-censoring (i.e. censoring that happens if a participant is lost to follow-up before having an event).</p><p>Detailed methods for our <italic>Drosophila</italic> experiments can be found in the <bold>Supporting methods in <xref ref-type="supplementary-material" rid="pgen.1006327.s001">S1 Text</xref></bold>. Briefly, the rescue activity of <italic>amx</italic>, <italic>TM2D3</italic>, or <italic>TM2D3</italic><sup><italic>P155L</italic></sup> genomic constructs was examined in female flies of the genotype <italic>amx</italic><sup><italic>1</italic></sup><italic>/Df(1)Exel9049; {Rescue Trangene}/+</italic> or <italic>amx</italic><sup><italic>1</italic></sup><italic>/Df(1)Exel9049; {Rescue Trangene}</italic> when crossed to <italic>amx</italic><sup><italic>1</italic></sup> males (with or without the <italic>{Rescue Transgene}</italic>)[<xref ref-type="bibr" rid="pgen.1006327.ref043">43</xref>]. To visualize the developing nervous system, resulting embryos were stained with anti-Hrp (1:1000) [<xref ref-type="bibr" rid="pgen.1006327.ref056">56</xref>], a neuronal membrane marker and anti-ELAV (1:100) [<xref ref-type="bibr" rid="pgen.1006327.ref057">57</xref>], a neuronal nuclear marker. For egg hatching, adults were allowed to lay eggs for 5 hours on grape juice agar plates, and larvae were counted 24 hours later.</p>