<p>W12-derived subclones [<xref ref-type="bibr" rid="pgen.1007179.ref008">8</xref>, <xref ref-type="bibr" rid="pgen.1007179.ref017">17</xref>] and NIKS human keratinocyte [<xref ref-type="bibr" rid="pgen.1007179.ref045">45</xref>] cell lines were maintained in F-medium (3:1 [vol/vol] F-12&#8211;Dulbecco&#8217;s modified Eagle&#8217;s medium, 5% fetal bovine serum, 0.4 &#956;g/ml hydrocortisone, 5 &#956;g/ml insulin, 8.4 ng/ml cholera toxin, 10 ng/ml epidermal growth factor, 24 &#956;g/ml adenine, 100 U/ml penicillin and 100 &#956;g/ml streptomycin). All cells were grown in the presence of irradiated 3T3-J2 feeder cells. C-33A, C-411, CasKi, HeLa, ME-180 and SiHa cervical carcinoma derived cell lines were maintained in Dulbecco&#8217;s modified Eagle&#8217;s medium, 10% fetal bovine serum, 100 U/ml penicillin and 100 &#956;g/ml streptomycin).</p><p>APOT was performed as described previously [<xref ref-type="bibr" rid="pgen.1007179.ref027">27</xref>]. See Supplementary <xref ref-type="supplementary-material" rid="pgen.1007179.s009">S2 Table</xref> for primer sequences. The resulting PCR product band was gel purified using Roche High Pure PCR Product Purification Kit and sequenced. The cellular DNA fused to HPV16 was identified using BLAT Search on the UCSC Genome Browser.</p><p>Total DNA was harvested with the DNeasy Blood and Tissue kit (Qiagen). 1 &#956;g total DNA was digested with either a single-cut linearizing enzyme (BamHI) for the HPV16 genome or a non-cutter (HindIII) to linearize cellular DNA. After digestion, samples were separated on 0.8% Tris-acetate-EDTA (TAE) agarose gels. A known quantity of linear HPV16 DNA was run on the same gel as the samples. DNA was visualized with 0.5 mg/ml ethidium bromide and transferred onto nylon membranes with a TurboBlotter downward transfer system (Whatman). Membranes were UV cross-linked, dried, incubated with prehybridization blocking buffer (3&#215; SSC [1&#215; SSC is 0.15 M NaCl plus 0.015 M sodium citrate], 2% SDS, 5&#215; Denhardt&#8217;s solution, 0.2 mg/ml sonicated salmon sperm DNA), and then incubated overnight with [<sup>32</sup>P]-dCTP-labeled HPV16 DNA in hybridization buffer (3&#215; SSC [1&#215; SSC is 0.15 M NaCl plus 0.015 M sodium citrate], 2% SDS, 5&#215; Denhardt&#8217;s solution, 0.02 mg/ml sonicated salmon sperm DNA, 25 ng [<sup>32</sup>P]-dCTP-labeled HPV16 DNA). Radiolabeled probe was generated from 50 ng of gel-purified linear HPV16 DNA with a Random Prime DNA labeling kit (Roche). Hybridized DNA was visualized and quantitated by phosphorimaging on a Typhoon Scanner (GE Bioscience).</p><p>Total cellular DNA was purified from 2 x 10<sup>6</sup> W12 cells using the DNeasy Blood and Tissue kit (Qiagen) and sequenced (2 x 150 bp paired-end reads) using the Illumina HiSeq 4000 platform (Genomics Resource Center, Institute for Genome Sciences, University of Maryland). Alignment of WGS data was performed using Bowtie 2 (version 2.2.5) [<xref ref-type="bibr" rid="pgen.1007179.ref046">46</xref>, <xref ref-type="bibr" rid="pgen.1007179.ref047">47</xref>].</p><p>To confirm virus-host insertional breakpoints, we designed a custom panel of Agilent SureSelect XT v.G.1 HPV16 genome sequences to detect viral genomic sequences and flanking host genomic DNA sequences in various samples (manuscript in preparation, Symer et al., similar in overall approach to [<xref ref-type="bibr" rid="pgen.1007179.ref048">48</xref>]). We performed the enrichment and library preparation using a protocol on the Agilent Bravo following the manufacturer&#8217;s protocol in the Genomics Shared Resource, Ohio State University Comprehensive Cancer Center. Libraries were sequenced using a HiSeq 2500 (2 x 150-bp paired-end reads) at Nationwide Children&#8217;s Hospital. More than 5 million reads were sequenced per sample (i.e. for cell line 20861, we generated 5.5 million reads, and for 20863, 23 million reads; see Supplementary <xref ref-type="supplementary-material" rid="pgen.1007179.s008">S1 Table</xref>). Reads were aligned against a custom reference genome assembly (i.e. human genome sequence (UCSC hg19 genome) + HPV16 genome) using BWA aligner [<xref ref-type="bibr" rid="pgen.1007179.ref049">49</xref>]. Discordant sequence read pairs having one end aligned to the HPV16 viral genome and the other end aligned to the human genome were re-aligned using the GSNAP aligner [<xref ref-type="bibr" rid="pgen.1007179.ref050">50</xref>]. After removing the duplicate pairs from these reads, we used a custom pipeline involving Hydra [<xref ref-type="bibr" rid="pgen.1007179.ref010">10</xref>, <xref ref-type="bibr" rid="pgen.1007179.ref051">51</xref>] to identify and confirm HPV insertional breakpoints in the human genome. Many of the breakpoints observed in chromosome 2 in the 20863 cell line were located at simple repeat regions, so those calls were supported by low numbers of paired end reads and were made at low confidence (Supplementary <xref ref-type="supplementary-material" rid="pgen.1007179.s008">S1 Table</xref>).</p><p>Cells were cultured on coverslips and fixed at room temperature in 4% paraformaldehyde (PFA)&#8211;PBS for 20 minutes. Cells were permeabilized in 0.1% Triton X-100 and stained with primary and fluorescent secondary antibodies using standard procedures. Coverslips were mounted in ProLong Gold containing 4&#8217;,6-diamidino-2-phenylindole (DAPI) for analysis by confocal microscopy. Anti-BRD4 CW152 rabbit polyclonal antibody was used at 1:100 dilution [<xref ref-type="bibr" rid="pgen.1007179.ref052">52</xref>].</p><p>Cells grown on coverslips were fixed in cold methanol-acetic acid (3:1) for 1&#8211;3 minutes followed by 4% paraformaldehyde&#8211;PBS for 10 minutes. Cells were treated with RNase A for 1 hour at 37&#176;C and dehydrated in a 70%, 90%, and 100% ethanol series for 3 minutes each. Green 5-Fluorescein dUTP labelled BAC clone RP11-1140H22 (chr2: 28,504,596&#8211;28,660,966; hg19) was purchased from Empire Genomics. DNA-FISH probe against full-length HPV16 DNA was fluorescently labeled using the Alexa Fluor-594 FISH Tag DNA Multicolor Kit (Life Technologies) following manufacturer&#8217;s protocol. To each coverslip, 40&#8211;50 ng labeled DNA FISH-probe in hybridization buffer (Empire Genomics) supplemented with 50 &#956;g/ml human Cot-1 DNA (Invitrogen) was added. DNA denaturation was performed at 75&#176;C for 5 minutes, followed by hybridization at 37&#176;C overnight. Cells were washed for 5 minutes each with 1x phosphate-buffered detergent (Qbiogene) at room temperature, 1x wash buffer (0.5x SSC, 0.1% SDS) at 65&#176;C, and 1x phosphate-buffered detergent at room temperature. Coverslips were mounted in ProLong Gold (ThermoFisher) containing DAPI for analysis by confocal microscopy.</p><p>Cells grown on coverslips were fixed in cold methanol-acetic acid (3:1) for 1&#8211;2 minutes followed by 4% paraformaldehyde&#8211;PBS for 10 minutes. Cells were processed for immunofluorescence staining followed by DNA FISH using methods described above and as described previously [<xref ref-type="bibr" rid="pgen.1007179.ref053">53</xref>].</p><p>W12 cells were processed as previously described [<xref ref-type="bibr" rid="pgen.1007179.ref022">22</xref>]. Briefly, 5 x 10<sup>7</sup> cells were crossed-linked with 1% formaldehyde and sheared to 100&#8211;500 bp DNA fragments using a Bioruptor sonicator (Diagonode) on high power settings. An aliquot of the lysate (5%) was processed for input control. Chromatin samples (20 &#956;g per ChIP) were incubated overnight at 4&#176;C with antibodies against Brd4 (Bethyl Laboratories A301-985A, 3 &#956;g) and H3K27ac (Millipore 07&#8211;360, 3 &#956;l). No-antibody controls were included to measure non-specific binding. Chromatin-immunocomplexes were precipitated for 1 hour at 4&#176;C with blocked Dynabeads Protein G (Invitrogen), subjected to multiple wash steps and the chromatin eluted. Recovered DNA was reverse cross-linked overnight at 65&#176;C with 5 M NaCl, followed by RNase A and proteinase K treatment, and purified using the ChIP DNA Clean &amp; Concentrator kit (Zymo Research). ChIP DNA was analyzed by real-time qPCR, using primers listed in Supplementary <xref ref-type="supplementary-material" rid="pgen.1007179.s009">S2 Table</xref>. Real-time qPCR was performed using the ABI Prism 7900HT sequence detector (Applied Biosystems) and SYBR green PCR master mix (Applied Biosystems). All reactions were run in triplicate and compared to standard curves of input chromatin DNA or cloned HPV16 genome from W12 cells, for determination of viral copy number, to generate a standard curve of threshold cycle (CT) versus log10 quantity (fg).</p><p>W12 cells were processed as described above for ChIP-qPCR. ChIP DNA samples were pooled and sequenced (2 x 150 bp paired-end reads) using the Illumina HiSeq 4000 platform (Genomics Resource Center, Institute for Genome Sciences, University of Maryland). Alignment of ChIP-seq data was performed using Bowtie 2 (version 2.2.5) [<xref ref-type="bibr" rid="pgen.1007179.ref046">46</xref>, <xref ref-type="bibr" rid="pgen.1007179.ref047">47</xref>].</p><p>Total RNA was purified from W12 cells using the RNeasy Mini Kit (Qiagen) and RNA integrity determined using the Bioanalyzer 2100 (Agilent Technologies). Polyadenylated RNA was sequenced (2 x 150 bp paired-end reads) using the Illumina HiSeq 4000 platform (Genomics Resource Center, Institute for Genome Sciences, University of Maryland). RNA-seq data was aligned to reference genomes using TopHat2 (version 2.1.0) [<xref ref-type="bibr" rid="pgen.1007179.ref054">54</xref>] and expression analysis of known transcripts (hg19_refseq_16_08_01_v2) performed using Cufflinks (version 2.2.1) [<xref ref-type="bibr" rid="pgen.1007179.ref055">55</xref>].</p><p>HPV16 DNA inserted into the BamHI site of pUC18/19 was digested with BamHI and extracted from a 0.8% agarose gel to remove pUC18/19. Biotinylated HPV16 genome probe was synthesized with BioPrime DNA Labeling System (Invitrogen, 18094011) using 500 ng of digested HPV16 DNA incubated at 37&#176;C for 4 hours. Biotinylated probe was purified using QIAquick PCR Purification Kit (Qiagen, 28104).</p><p>W12 cells were harvested, pelleted for 5 minutes at 1,000 rpm, and resuspended in cold 1x PBS to a dilution of 1 x 10<sup>6</sup> cells per ml. Diluted cells were warmed to 37&#176;C for 1&#8211;2 minutes, and an equal volume of 1.5% SeaPlaque agarose (Lonza, 50100) in 1x PBS, warmed to 45&#176;C, was added. Cell/agarose mixture was aliquoted into plug mold wells (Biorad, 1703713) and solidified for 20 minutes at 4&#176;C. Solid plugs were ejected into lysis buffer (1 mg/ml proteinase K [added directly before use], 100 mM EDTA, 1% N-lauroylsarcosine, 10 mM Tris-HCl, pH 8.0, and incubated overnight in a 50&#176;C water bath. Lysis buffer was discarded and plugs were rinsed with 1x TE. Rinsed plugs were washed three times for 1 hour in 1x TE. Washed plugs were stored for several months in 1x TE at 4&#176;C.</p><p>The washed plug was transferred to a 2 ml tube and melted for 20 minutes at 70&#176;C in 0.1 M MES (pH 6.5). Melted plug was cooled for 5 minutes in a 42&#176;C water bath. 3 units of &#946;-agarase (New England BioLabs, M0392S) were added to cooled DNA solution and incubated overnight in a 42&#176;C water bath. DNA solution was carefully poured into a 2 ml Teflon reservoir. A silanized coverslip was inserted into the DNA solution and incubated for 5 minutes. The coverslip was removed and adhered to a glass slide with cyanoacrylate glue. A test coverslip was pulled, stained with YOYO-1 (2:10,000 YOYO-1 in 0.1 M MES), and visualized with a microscope to ensure quality and density of DNA fibers. Coverslips were baked for 30 minutes at 60&#176;C to crosslink DNA fibers to coverslip.</p><p>DNA fibers were denatured for 20 minutes in 0.2 M NaOH. Slides were neutralized 5 times with 1x PBS (pH 7.4) for 1 minute. Slides were dehydrated for 3 minutes each in successive baths of 70%, 90%, and 100% EtOH and air dried for 8 minutes. 500 ng of HPV16 biotinylated probe was mixed with 5 &#956;g of human Cot-1 DNA and hybridization buffer (50% formamide, 2x SSC, 0.5% SDS, 0.5% N-lauroylsarcosine sodium salt, 1% Blocking reagent [Roche, 11096176001], 10 mM NaCl) up to a total volume of 20 &#956;L. Probe mixture was added to coverslip, a clean coverslip was placed on top and sealed with rubber cement. Slides were denatured for 5 minutes at 75&#176;C and then incubated for 16 hours at 37&#176;C. The rubber cement and top coverslips were removed from the slides and the slides rinsed with 2x SSC. The slides were washed three times with 2x SSC/50% formamide for 5 minutes, and then washed three times with 2x SSC for 5 minutes. Slides were blocked for 1 hour in a humidified chamber with 5% BSA in 1x PBS and washed 3 times for 5 minutes with PBS-T (0.1% Triton X-100 in 1x PBS). To visualize biotinylated probe, slides were incubated in a humidified chamber at 37&#176;C in the following layers. Layer 1: 1:100 streptavidin, Alexa Fluor-488 conjugated (ThermoFisher, S11223) in 5% BSA/1x PBS for 30 minutes. Layer 2: 1:200 biotinylated anti-streptavidin (Vector, BA-0500) in 5% BSA/1x PBS for 1 hour. Layer 3: 1:200 streptavidin, Alexa Fluor-488 conjugated in 5% BSA/1x PBS for 30 minutes. Layer 4: 1:200 biotinylated anti-streptavidin and 1:200 anti-ssDNA IgG2a (EMD Millipore, MAB3034) in 5% BSA/1x PBS for 1 hour. Layer 5: 1:200 streptavidin, Alexa Fluor-488 conjugated, and 1:100 goat anti-mouse IgG2a, Alexa Fluor-647 conjugated (Invitrogen, A-21241), in 5% BSA/1x PBS. Three 5 minute washes with PBS-T were performed in between each layer. Slides were rinsed with 1x PBS, mounted with ProLong Gold (ThermoFisher, P36930).</p><p>Digoxigenin labeled probes were synthesized with BioPrime DNA Labeling System (Invitrogen, 18094011) using 500 ng BsaBI and XcmI digested HPV16 DNA. The biotinylated nucleotide mix from the kit was replaced with PCR DIG Probe Synthesis Mix (10x, PCR DIG Probe Synthesis Kit, Roche, 11636090910) and the labeling reaction incubated at 37&#176;C for 4 hours. The DIG probe was purified using QIAquick PCR Purification Kit (Qiagen, 28104). Biotinylated probes were prepared as described previously. Following combing, DNA fibers were denatured for 20 minutes in 0.2 M NaOH, and neutralized five times with 1x PBS (pH 7.4) for 1 minute. Slides were dehydrated for 3 minutes each in successive baths of 70%, 90%, and 100% EtOH and air dried for 8 minutes. Biotinylated/DIG probes in an equivalent ng/kb ratio, 200 ng of an 8 kb probe and 620 ng of a 25 kb probe, were mixed with 5 &#956;g human Cot-1 DNA in hybridization buffer (50% formamide, 2x SSC, 0.5% SDS, 0.5% N-lauroylsarcosine sodium salt, 1% Blocking reagent [Roche, 11096176001], 10 mM NaCl) to a total volume of 20 &#956;L. Probe mixture was added to the coverslip, a clean coverslip was placed on top and sealed with rubber cement. Slides were denatured for 5 minutes at 75&#176;C and incubated for 16 hours at 37&#176;C. Slides were washed three times with 2x SSC/50% formamide for 5 minutes and three times with 2x SSC for 5 minutes. Slides were blocked for 1 hour in a humidified chamber with 5% BSA in 1x PBS and washed 3 times for 5 minutes with PBS-T (0.1% Triton X-100 in 1x PBS). To visualize the biotin and DIG-labeled probes, slides were incubated in a humidified chamber at 37&#176;C with the following layers: Layer 1: 1:100 streptavidin, Alexa Fluor-555 conjugated (ThermoFisher, S21381), and 1:100 anti-digoxigenin, FITC conjugated (Roch, 11333089001), in 5% BSA/1x PBS for 1 hour. Layer 2: 1:200 biotinylated anti-streptavidin (Vector, BA-0500) and 1:200 anti-FITC (Invitrogen, 711900) in 5% BSA/1x PBS for 45 minutes. Layer 3: 1:200 streptavidin, Alexa Fluor-555 conjugated in 5% BSA/1x PBS for 30 minutes. Layer 4: 1:200 biotinylated anti-streptavidin, 1:200 anti-rabbit, FITC conjugated (Rockland, 611-102-122), and 1:200 anti-ssDNA IgG2a (EMD Millipore, MAB3034) in 5% BSA/1x PBS for 45 minutes. Layer 5: 1:200 streptavidin, Alexa Fluor-555 conjugated, and 1:100 goat anti-mouse IgG2a, Alexa Fluor-647 conjugated (Invitrogen, A-21241), in 5% BSA/1x PBS. Three 5 minute washes with PBS-T were performed in between each layer. Slides were rinsed with 1x PBS, mounted with ProLong Gold (ThermoFisher, P36930). All slides were imaged using a Leica TCS-SP5 laser scanning confocal imaging system.</p><p>Measurement of signal lengths from raw Leica files was performed using Bitplane Imaris x64. Images were analyzed in Surpass with an orthogonal camera. Measurement points of equal diameter were placed on the first and last points of signal for each HPV16 genome. Dots of signal were considered contiguous if they were within 0.6 &#956;m of a previous dot. Genomes continuous in a fiber were measured with a single line of measurement points to determine the length of any interspersing DNA. Base pair length was determined using 1 &#956;m = 2 kb conversion.</p>