<p>RhoJ KO mice were generated as previously described [<xref ref-type="bibr" rid="pgen.1006913.ref029">29</xref>]. Briefly, the ES cells used to generate RhoJ knockout mice were derived from C57Bl/6N-<italic>RhoJ</italic><sup><italic>tm1a(KOMP)Wtsi</italic></sup> and injected into C57B6/J mice to generate chimeric mice. The melanoma prone mice have the genetic background C57BL/6.Cg- <italic>Braf</italic><sup><italic>tm1Mmcm</italic></sup>
<italic>Pten</italic><sup><italic>tm1Hwu</italic></sup>
<italic>Tg(Tyr-cre/ERT2)13Bos/BosJ</italic> (<italic>Braf</italic><sup><italic>CA</italic></sup>, <italic>Pten</italic><sup><italic>lox4-5</italic></sup>, <italic>Tyr</italic>::<italic>Cre</italic><sup><italic>ERT2</italic></sup>) that have been previously characterized [<xref ref-type="bibr" rid="pgen.1006913.ref018">18</xref>]. Both mice strains were crossed to generate <italic>Braf</italic><sup><italic>CA</italic></sup>, <italic>Pten</italic><sup><italic>lox4-5</italic></sup>, <italic>Tyr</italic>::<italic>Cre</italic><sup><italic>ERT2</italic></sup>, <italic>RhoJ</italic><sup><italic>-/-</italic></sup> mice. The entire survival study utilized a total of 19 males and 37 females. The pigmentation of the mouse paws were analyzed by measuring the area of the pigmented region in the paws and normalizing to the entire paw using ImageJ software. The quantification of the paws includes all the paws shown in the figures and supporting figures.</p><p><italic>Tyr</italic>:<italic>Cre</italic><sup><italic>ERT2</italic></sup>; <italic>Braf</italic><sup><italic>V600E</italic></sup><italic>; RhoJ</italic><sup><italic>+/+</italic></sup> and <italic>Tyr</italic>:<italic>Cre</italic><sup><italic>ERT2</italic></sup>; <italic>Braf</italic><sup><italic>V600E</italic></sup><italic>; RhoJ</italic><sup><italic>-/-</italic></sup> mice were shave depilated at p50 (second telogen), euthanized, and immediately imaged ex-vivo (no labeling) with MPM to capture the fluorescence signal from keratin and melanin and second harmonic generated (SHG) signal from collagen using LSM 510 NLO Zeiss system. Fluorescence and second harmonic generation was excited by femtosecond Titanium: Sapphire (Chameleon-Ultra, Coherent) laser at 900 nm. Emission was detected at 390&#8211;465 nm for SHG channel (blue), and 500&#8211;550 nm (green) and 565&#8211;650 (red) fluorescence channels. Each animal was imaged at 8 to 10 randomly chosen locations on depilated skin of the lower back. Stacks of optical sections of 636&#956;m x 636&#956;m at different depths ranging from 0 to 240 &#956;m (5 &#956;m steps) were obtained to allow for 3-D volume reconstruction (LSM Image Browser, Carl Zeiss GMBH).</p><p><italic>Braf</italic><sup><italic>CA</italic></sup>, <italic>Pten</italic><sup><italic>lox4-5</italic></sup>, <italic>Tyr</italic>::<italic>Cre</italic><sup><italic>ERT2</italic></sup> mice were genotyped as previously described [<xref ref-type="bibr" rid="pgen.1006913.ref018">18</xref>]. The Cre-specific primers are: forward 5&#8217;- GGTGTCCAATTTACTGACCGTACA-3&#8217; and reverse 5&#8217;- CGGATCCGCCGCATAACCAGTG -3&#8217;. Topical administration of 4-hydroxytamoxifen (4-OHT) to the back skin on pups and adults was performed as previously described [<xref ref-type="bibr" rid="pgen.1006913.ref018">18</xref>]. Paws of mice were also treated with 4-OHT at P2, P3, and P4. FRAX597 was administered to mice daily via oral gavage daily at a dose of 100 mg/kg [<xref ref-type="bibr" rid="pgen.1006913.ref033">33</xref>] and prepared in 60%:40% PEG-400:DI water. Tumors in adult mice (P21) were induced for three days and allowed to progress for one week, at which point mice began FRAX treatment for six days. The veterinary staff was blinded as to the genotype of experimental animals utilized in the study, and monitored experimental animals on a daily basis. Animals were culled when tumor burden reached ethical limits and if the animals displayed signs of ill health or distress as determined by the veterinary staff. All mouse procedures were approved by UCI&#8217;s IACUC regulations and standards. As an additional control for background strain variability, all survival comparisons were made between littermates.</p><p>Total RNA from RhoJ WT 1&#8211;5 and RhoJ KO 1&#8211;5 was monitored for quality control using the Agilent Bioanalyzer Nano RNA chip (Santa Clara, CA). Samples with a RIN number &gt;8 were used to construct libraries using the Illumina TruSeq RNA v2 kit. The input quantity for total RNA was 1&#956;g and mRNA was enriched using oligo dT magnetic beads. The enriched mRNA was chemically fragmented for four minutes. First strand synthesis used random primers and reverse transcriptase to make cDNA. After second strand synthesis the ds cDNA was cleaned using AMPure XP beads (Beckman Coulter, Beverly, MA) and the cDNA was end repaired and then the 3&#8217; ends were adenylated. Illumina barcoded adapters were ligated on the ends and the adapter-ligated fragments were enriched by nine cycles of PCR. The resulting libraries were validated by qPCR and sized by Agilent Bioanalyzer DNA high sensitivity chip. The concentrations for the libraries were normalized and then multiplexed together. The concentration for clustering on the flow cell was 12.5pM. The multiplexed libraries were sequenced on three lanes using paired end 100 cycles chemistry for the Illumina HiSeq 2500. The version of HiSeq control software was HCS 2.2.38 with real time analysis software, RTA 1.18.61.</p><p>RhoJ staining was optimized at 1:1000 using an Ab from Genetex (Irvine, CA). Slides were developed with either a horse-radish peroxidase liquid 3,3'-diaminobenzidine chromogen system (DAKO, Carpirteria, CA) or an alkaline phosphatase liquid permanent red chromogen system (DAKO), according to the manufacture&#8217;s protocol. Fluorescence labeling was performed using SMA (Abcam 1:100) PMEL, KIT, DCT, and DAPI as described [<xref ref-type="bibr" rid="pgen.1006913.ref049">49</xref>]. The expression or phosphorylation of proteins was detected by western blotting using the following Abs: RhoJ (Abnova 1:250), Cdc42, Rac1/2/3, RhoA, actin, BAD, phospho-Bad<sup>Ser138</sup>, and Vinculin (all from cell signaling 1:1000).</p><p>A Cdc42 activation assay was performed according the manufacturer&#8217;s protocol (Cell Biolabs, San Diego, CA). Briefly, cell lysates expressing high RhoJ were unloaded of guanosine nucleotides and either loaded with GDP or GTP&#947;S. Agarose beads conjugated with the PAK1 PBD domain pulled down Cdc42, Rac, or RhoJ only when GTP-bound. Lysates were then immunoblotted with indicated Abs.</p><p>Approximately 1 &#215; 10<sup>6</sup> melanoma cell treated with FRAX-597 were trypsinized and washed with PBS, the single-cell suspensions were incubated with Alexa Fluor-Annexin V and propidium iodide (PI) (V13245; Invitrogen) per the manufacturer&#8217;s protocol and were subjected to flow cytometric analysis. In all cases, cell debris was gated out based on forward scatter and side scatter analysis. Data was analyzed using FlowJo (Ashland, OR). Cells not treated with FRAX-597 were used to establish gating parameters. WM3248 cells were transfected with BAD<sup>S136E</sup> (3582 pcDNA3 BAD<sup>S136E</sup> Addgene #8800), with Lipofectamine 3000 (Invitrogen) according to the manufacturer&#8217;s protocol. On the day of transfection, 14&#956;g of DNA from the respective constructs were incubated with Plus Reagent in Opti-MEM to a volume of 350&#956;L, and then mixed with 43&#956;L lipofectamine in 350&#956;L Opti-MEM and incubated for 20 min at room temperature. The DNA-Lipofectamine complexes were then added to each plate, and the cells were incubated at 37&#176;C in a CO<sub>2</sub> incubator. The media was replaced after 24hrs and cells selected in G418.</p><p>The power and sample size calculations for the survival studies (<xref ref-type="fig" rid="pgen.1006913.g001">Fig 1A and 1B</xref>) were based on a simulation of 1000 datasets utilizing SAS v9.2. To calculate the power, the simulation under the alternative hypothesis made the following assumptions: 1.) No censoring data for both groups; 2.) The first event occurred on day 30 for the controls; 3.) The first event occurred on day 40 for RhoJ<sup>-/-</sup>; 4.) Hazard ratios used were 1.5, 2, 4, 6; and 5.) Sample size of mice per group was 15, 20, 25, 30, 40, and 50. To calculate the type I error (alpha) under the null hypothesis the following assumptions were made: 1.) Data was not censored for both groups; 2.) The first event occurred on day 30 for both groups; 3.) Hazard ratio is 1; and 4.) Sample size of the mice per group was 15, 20, 25, 30, 40, and 50.</p><p>Continuous outcomes were summarized via mean and standard error and tests of unadjusted means between groups were conducted using a two-sample t-test with unequal variances. Mean RhoJ expression levels adjusting for stage, age and gender were estimated and compared using multiple linear regression analysis. Residual diagnostics were performed in order to assess the functional form of continuous covariates included in the model and to identify potentially influential subjects. Estimated mean differences, corresponding Wald-based 95% confidence intervals, and p-values corresponding to the test of no association in RhoJ levels were reported for all model covariates. No deviations in model fit or influential observations were observed.</p><p>The TCGA analysis was performed as follows. The skin cutaneous melanoma (SKCM) TCGA Exome-sequencing and RNA-sequencing datasets were downloaded June 2016. Single nucleotide variants (SNVs) at the position 1799 of the BRAF gene (c.1799T&gt;C) leading to a non-synonymous alteration at the amino acid 600 of the BRAF protein were annotated as BRAFV600E. Additional, RhoJ expression assessed by RNA-sequencing was normalized using the z-score and the statistical difference on mean expression between BRAF wt and BRAFV600E was assessed by the Student's t-test.</p><p>Mice were housed and maintained by the University Laboratory Animal Resources. All animal experiments were approved by the University of California, Irvine&#8217;s Institutional Animal Care and Use Committee. Melanoma tumor microarray tissues were collected and processed with approval from the Providence Saint John&#8217;s Health Center and John Wayne Cancer Institute joint Institutional Review Board and Western Institutional Review Board.</p>