<p>We used a three-stage design to evaluate genetic associations with BMI and WHR<sub>adjBMI</sub> in sex-combined and sex-stratified samples (<xref ref-type="supplementary-material" rid="pgen.1006719.s001">S1 Fig</xref>). Stage 1 included GWAS meta-analyses in AA individuals and stage 2 included replication of top associations from stage 1. Stage 3 included meta-analysis of top associations from stages 1 and 2 AA studies and EA meta-analysis results. In the discovery stage 1 of AAAGC, 17 GWAS of up to 42,752 AA individuals (16,559 men and 26,193 women; 41,696 African Americans and 1,056 Africans) were included for the BMI analyses. A total of 10 GWAS of up to 20,384 AA individuals (4,783 men and 15,601 women; all African Americans) were included for the WHR<sub>adjBMI</sub> analyses. For variants with <italic>P</italic> &lt; 1&#215;10<sup>&#8722;4</sup> in either the sex-combined or the sex-stratified meta-analyses, stage 2 replication was performed in additional AA individuals from AAAGC (N = 10,143 for BMI, N = 2,711 for WHR<sub>adjBMI</sub>), followed by meta-analysis with EA individuals from the GIANT consortium (322,154 for BMI, 210,086 for WHR<sub>adjBMI</sub>). Variants that reached genome-wide significance (<italic>P</italic> &lt; 5&#215;10<sup>&#8722;8</sup>) were assessed for associations with BMI in two cohorts of children (N = 7,222). All AA participants in these studies provided written informed consent for the research, and approval for the study was obtained from the ethics review boards at all participating institutions. Detailed descriptions of each participating study and measurement and collection of height, weight, waist and hip circumferences are provided in <xref ref-type="supplementary-material" rid="pgen.1006719.s026">S1 Text</xref>, <xref ref-type="supplementary-material" rid="pgen.1006719.s012">S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1006719.s013">S2</xref> Tables.</p><p>Genotyping in each study was performed with Illumina or Affymetrix genome-wide SNP arrays. Pre-phasing and imputation of missing genotypes in each study was performed using MaCH/ minimac [<xref ref-type="bibr" rid="pgen.1006719.ref020">20</xref>] or SHAPEIT2/IMPUTEv2 [<xref ref-type="bibr" rid="pgen.1006719.ref021">21</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006719.ref022">22</xref>] using the 1000 Genomes Project cosmopolitan reference panel (Phase I Integrated Release Version 3, March 2012) [<xref ref-type="bibr" rid="pgen.1006719.ref018">18</xref>]. The details of the array, genotyping and imputation quality-control procedures and sample exclusions for each study are listed in <xref ref-type="supplementary-material" rid="pgen.1006719.s014">S3 Table</xref>. In general, samples reflecting duplicates, low call rates, gender mismatch, or population outliers were excluded. Variants were excluded by the following criteria: call rate &lt; 0.95, minor allele count (MAC) &#8804; 6, Hardy-Weinberg Equilibrium (HWE) <italic>P</italic> &lt; 1&#215;10<sup>&#8722;4</sup>, imputation quality score &lt; 0.3 for minimac or &lt; 0.4 for IMPUTE, or absolute allele frequency difference &gt; 0.3 compared with expected allele frequency (calculated as 1000 Genomes frequency of AFR &#215; 0.8 + EUR &#215; 0.2).</p><p>We evaluated the performance of 1000 Genomes imputation using the largest study, the Women&#8217;s Health Initiative (WHI) (N = 8,054). A total of 25.1 million variants with MAF &#8805; 0.1% were imputed to the 1000 Genomes reference panel. Of these, 98.1% (8.8 million) common variants, 95.4% (9.3 million) low frequency variants (0.5% &#8804; MAF &lt; 5%), and 72.5% (4.6 million) rare variants (0.1% &#8804; MAF &lt; 0.5%) were well imputed with IMPUTE info scores &#8805; 0.3 (<xref ref-type="supplementary-material" rid="pgen.1006719.s015">S4 Table</xref>). Notably, these frequencies are slightly lower than those obtained by imputation using 1000 Genomes phase 1 interim reference panel in Europeans [<xref ref-type="bibr" rid="pgen.1006719.ref054">54</xref>]. However, 72.6%, 95.5% and 99.5% of the common, low frequency and rare variants, respectively, from the 1000 Genomes reference panel were not present in the HapMap and therefore demonstrate deeper coverage of the genome, particularly for the low frequency and rare variants.</p><p>At all stages, genome-wide association analyses were performed by each of the participating studies. BMI was regressed on age, age squared, principal components and study site (if needed) to obtain residuals, separately by sex and case-control status, if needed. WHR was regressed on age, age squared, principal components, BMI and study site to obtain residuals, separately by sex and case-control status. Principal components were included to adjust for admixture proportion and population structure within each study. Residuals were inverse-normally transformed to obtain a standard normal distribution with mean of zero and standard deviation of one. For studies with unrelated subjects, each variant was tested assuming an additive genetic model with each trait by regressing the transformed residuals on the number of copies of the variant effect allele. The analyses were stratified by sex and case-control status (if needed). For studies that included related individuals, family based association tests were conducted that took into consideration the genetic relationships among the individuals. Sex stratified, case-control stratified and combined analyses were performed. Association results with extreme values (absolute beta coefficient or standard error &#8805; 10), primarily due to small sample sizes and/or low minor allele count, were excluded for meta-analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The latest summary statistics of sex-combined and sex-stratified meta-analyses of BMI and WHR<sub>adjBMI</sub> imputed to the HapMap reference panel in EA from the Genetic Investigation of ANthropometric Traits (GIANT) consortium were obtained from <ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium_data_files">http://www.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium_data_files</ext-link> [<xref ref-type="bibr" rid="pgen.1006719.ref007">7</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006719.ref008">8</xref>]. These association summary statistics were used to impute z-scores of unobserved variants at the 1000 Genomes Project EUR reference panel (Phase I Integrated Release Version 3) using the ImpG program [<xref ref-type="bibr" rid="pgen.1006719.ref023">23</xref>]. In brief, palindromic variants (AT/CG) and variants with allele mismatch with the reference were removed from the data. Using the ImpG-Summary method, the z-score of an unobserved variant was calculated as a linear combination of observed z-scores weighted by the variance-covariance matrix between variants induced by LD within a 1 Mb window from the reference haplotypes. The sample size of each unobserved variant was also interpolated from the sample sizes of observed variants using the same weighting method for z-score as <inline-formula id="pgen.1006719.e001"><alternatives><graphic id="pgen.1006719.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006719.e001.jpg" /><ns1:math id="M1"><ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>N</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mrow><ns1:msubsup><ns1:mo stretchy="false">&#8721;</ns1:mo><ns1:mrow><ns1:mi>t</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>t</ns1:mi><ns1:mo>=</ns1:mo><ns1:mi>T</ns1:mi></ns1:mrow></ns1:msubsup></ns1:mrow><ns1:mrow><ns1:mfrac><ns1:mrow><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>w</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>t</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>|</ns1:mo></ns1:mrow><ns1:mrow><ns1:mo stretchy="false">&#8721;</ns1:mo><ns1:mrow><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>w</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>t</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>|</ns1:mo></ns1:mrow></ns1:mrow></ns1:mfrac></ns1:mrow><ns1:msub><ns1:mi>N</ns1:mi><ns1:mi>t</ns1:mi></ns1:msub></ns1:mrow></ns1:math></alternatives></inline-formula>. Here, t = 1,2,&#8230;.,T, where T is the number of observed variants, <italic>w</italic><sub><italic>i</italic>,<italic>t</italic></sub> is the element of the covariance matrix &#931;<sub><italic>i</italic>,<italic>t</italic></sub> for the unobserved variant <italic>i</italic> and the observed variant <italic>t</italic> within window. The performance of imputation was assessed by <italic>r</italic><sup><italic>2</italic></sup><italic>pred</italic>, with similar characteristics as the standard imputation accuracy metric <italic>r</italic><sup><italic>2</italic></sup><italic>hat</italic> [<xref ref-type="bibr" rid="pgen.1006719.ref020">20</xref>]. Results of variants with <italic>r</italic><sup><italic>2</italic></sup><italic>pred</italic> &#8805; 0.6 were used in subsequent analyses.</p><p>In the discovery stage 1, association results were combined across studies in sex-combined and sex-stratified samples using inverse-variance weighted fixed-effect meta-analysis implemented in the program METAL [<xref ref-type="bibr" rid="pgen.1006719.ref055">55</xref>]. The study-specific &#955; values of association ranged from 0.97 to 1.05 for BMI, and 0.98 to 1.05 for WHR<sub>adjBMI</sub> (<xref ref-type="supplementary-material" rid="pgen.1006719.s014">S3 Table</xref>). Genomic control correction [<xref ref-type="bibr" rid="pgen.1006719.ref056">56</xref>] was applied to each study before meta-analysis, and to the overall results after meta-analysis (&#955; = 1.07 for BMI, 1.01 for WHR<sub>adjBMI</sub>). Variants with results generated from &lt; 50% of the total sample size for each trait were excluded. After filtering, the numbers of variants reported in the meta-analyses were 17,972,087 for BMI, and 20,502,658 for WHR<sub>adjBMI</sub>.</p><p>Variants with <italic>P</italic> &lt; 1&#215;10<sup>&#8722;4</sup> in stage 1 sex-combined or sex-stratified meta-analyses were carried forward for replication in additional AA individuals (stage 2) and EA individuals (stage 3). For each of the replication AA studies, trait transformation and association were performed as in stage 1 and results were meta-analyzed using the inverse-variance method in METAL. For the replication study in EA, HapMap imputed summary statistics of each trait from the GIANT consortium were used to impute z-scores of unobserved variants at the 1000 Genomes.</p><p>In stages 1 and 2, meta-analysis results of AA studies were combined using the inverse-variance weighted method. In all stages including both AA and EA studies, meta-analysis results expressed as signed z-scores were combined using the fixed effect sample size weighted method in METAL due to the lack of beta and standard error estimates from the ImpG program [<xref ref-type="bibr" rid="pgen.1006719.ref023">23</xref>]. Evidence of heterogeneity of allelic effects between males and females, within and across stages were assessed by the <italic>I</italic><sup><italic>2</italic></sup> statistic in METAL. Genome-wide significance was declared at <italic>P</italic> &lt; 5&#215;10<sup>&#8722;8</sup> from each of the sex-combined and sex-stratified meta-analysis including AA and/or combined AA and EA individuals. Difference in effects between men and women was assessed using Cochran&#8217;s Q test and nominal <italic>P</italic><sub><italic>het</italic></sub> &lt; 0.05 declared as significant. A lead variant in a locus was defined as the most significant variant within a 1 Mb region. A novel locus was defined as a lead variant with distance &gt; 500 kb from any established lead variants reported in previous studies. By convention, a locus was named by the closest gene(s) to the lead variant.</p><p>For the genome-wide significant loci identified in sex-combined and sex-stratified analyses in AA (stages 1+2), we used the program GCTA [<xref ref-type="bibr" rid="pgen.1006719.ref057">57</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006719.ref058">58</xref>] to select the top independent associated variants from summary statistics of the meta-analyses. This method uses the LD correlations between variants estimated from a reference sample to perform an approximate conditional association analysis. We used 8,054 unrelated individuals of African ancestry from the WHI cohort with ~15.7M variants available as the reference sample for LD estimation. To select the top independent variants in the discovery and replication meta-analysis results, we first selected all variants that had <italic>P</italic> &lt; 5&#215;10<sup>&#8722;8</sup> and conducted analysis conditioning on the selected variants to search for the top variants iteratively via a stepwise model to select the independent variants from this list. Then we proceeded to condition the rest of the variants that had <italic>P</italic> &gt; 5&#215;10<sup>&#8722;8</sup> on the list of independent variants in the same fashion until no variant had conditional <italic>P</italic> that passed the significance level <italic>P</italic> &lt; 5&#215;10<sup>&#8722;8</sup>. Finally, all the selected variants were fitted jointly in the model for effect size estimation.</p><p>We also tested if the genome-wide significant variants identified from sex-combined GWAS in AA and the locus-wide significant variants identified from sex-combined and sex-specific locus transferability studies in AA were independent from nearby established loci identified from EA studies [<xref ref-type="bibr" rid="pgen.1006719.ref007">7</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006719.ref008">8</xref>]. First, the published lead variants from EA studies were used to search for all surrogate variants that were in high LD (r<sup>2</sup>&gt;0.8 in 1000 Genomes Project EUR population). Second, these variants were pruned to select only variants in low LD in AA (r<sup>2</sup>&lt;0.3 in the 1000 Genomes Project AFR population) to avoid collinearity in conditional analysis. Third, association analysis was conducted on the AA significant variants conditioned on the selected EA lead and surrogate variants, using the program GCTA and estimated LD correlation from the WHI cohort. For genome-wide significant loci, an AA derived association signal is considered as independent from the established EA signals when the difference in&#8211;log<italic>P</italic> &lt;3 and difference in effect size &lt; 1 standard error after conditional analysis. For locus-wide significant loci, given the lower level of significance, independence is only considered as difference in effect size &lt; 1 standard error after conditional analysis.</p><p>We investigated the transferability of EA BMI and WHR associated variants and loci in AA individuals from stage 1 sex-combined and sex-stratified meta-analyses. First, we tested for replication of lead variants previously reported to be associated with BMI (176 variants from 170 loci) and WHR<sub>adjBMI</sub> (84 variants from 65 loci) at genome-wide significance in sex-combined and sex-stratified analyses from the GIANT consortium studies [<xref ref-type="bibr" rid="pgen.1006719.ref007">7</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006719.ref009">9</xref>]. We defined SNP transferability as an EA lead variant sharing the same trait-raising allele at nominal <italic>P</italic> &lt; 0.05 in AA individuals. To account for differences in local LD structure across populations, we also interrogated the flanking 0.1cM regions of the lead variants to search for the best variants with the smallest association <italic>P</italic> in AA individuals. Locus-wide significance was declared as <italic>P</italic><sub><italic>locus</italic></sub> &lt; 0.05 by Bonferroni correction for the effective number of tests within a locus, estimated using the Li and Ji approach [<xref ref-type="bibr" rid="pgen.1006719.ref059">59</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We compared the credible set intervals of established loci that showed locus-wide significance (<italic>P</italic><sub><italic>locus</italic></sub> &lt; 0.05) in the sex-combined or sex-specific analyses from this study in summary statistics datasets including the 1000 Genomes imputed results from GIANT, AAAGC and meta-analysis of GIANT and AAAGC. In each dataset, a candidate region is defined as the flanking 0.1cM region of the lead variant reported by the GIANT consortium. Under the assumption of one causal variant in a region of <italic>M</italic> variants, the posterior probability of a variant <italic>j</italic> with association statistics Z driving the association, <italic>P</italic>(<italic>C</italic><sub><italic>j</italic></sub>|<italic>Z</italic>), was calculated using the formula <inline-formula id="pgen.1006719.e002"><alternatives><graphic id="pgen.1006719.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006719.e002.jpg" /><ns1:math id="M2"><ns1:mi>P</ns1:mi><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>C</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mo>|</ns1:mo><ns1:mrow><ns1:mi>Z</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mrow><ns1:mrow><ns1:mi mathvariant="normal">exp</ns1:mi><ns1:mo stretchy="false">(</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:mfrac></ns1:mrow><ns1:mo>&#8289;</ns1:mo><ns1:mrow><ns1:msubsup><ns1:mrow><ns1:mi>z</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>j</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup><ns1:mo stretchy="false">)</ns1:mo></ns1:mrow></ns1:mrow></ns1:mrow><ns1:mrow><ns1:mrow><ns1:msubsup><ns1:mo stretchy="false">&#8721;</ns1:mo><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>M</ns1:mi></ns1:mrow></ns1:msubsup><ns1:mrow><ns1:mi mathvariant="normal">e</ns1:mi><ns1:mi mathvariant="normal">x</ns1:mi><ns1:mi mathvariant="normal">p</ns1:mi><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mfrac><ns1:mrow><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:mfrac><ns1:msubsup><ns1:mrow><ns1:mi>z</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>j</ns1:mi></ns1:mrow><ns1:mrow><ns1:mn>2</ns1:mn></ns1:mrow></ns1:msubsup></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:mrow></ns1:mfrac></ns1:math></alternatives></inline-formula>. A 99% credible set was constructed by ranking all variants by their posterior probability, followed by adding variants until the credible set has a cumulative posterior probability &gt; 0.99 [<xref ref-type="bibr" rid="pgen.1006719.ref053">53</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To determine whether any of our nine novel GWAS lead variants identified in the sex-combined and sex-specific analyses might be tagging potentially functional variants, we identified all variants within 1 Mb and in LD (r<sup>2</sup> &gt; 0.7, 1000 Genomes AFR) with our lead variants. As such, we identified 137 variants and annotated each of them using ANNOVAR [<xref ref-type="bibr" rid="pgen.1006719.ref060">60</xref>]. The predicted functional impact for coding variants were assessed via the Exome Variant Server (<ext-link ext-link-type="uri" ns0:href="http://evs.gs.washington.edu/EVS/">http://evs.gs.washington.edu/EVS/</ext-link>) for PhastCon, GERP [<xref ref-type="bibr" rid="pgen.1006719.ref061">61</xref>], and PolyPhen [<xref ref-type="bibr" rid="pgen.1006719.ref062">62</xref>], as well as SIFT [<xref ref-type="bibr" rid="pgen.1006719.ref063">63</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We further characterized the variants that were in LD with the novel variants using the web-based tool RegulomeDB (<ext-link ext-link-type="uri" ns0:href="http://regulomedb.org/">http://regulomedb.org/</ext-link>) [<xref ref-type="bibr" rid="pgen.1006719.ref024">24</xref>]. The variants that were likely to affect binding and linked to expression of a gene target (scores 1a-1f) based on &#8220;eQTL, transcription factor (TF) binding, matched TF motif, matched DNase footprint and DNase peak&#8221; or were only likely to affect binding (scores 2a-2c) based on &#8220;TF binding, matched TF motif, matched DNase footprint and DNase peak&#8221; were selected. For these variants, the sequence conservation (GERP and SiPhy [<xref ref-type="bibr" rid="pgen.1006719.ref064">64</xref>]), the epigenomic data from the Roadmap Epigenomic project (ChromHMM states corresponding to enhancer or promoter elements, histone modification ChIP-seq peaks, and DNase hypersensitivity data peaks), the regulatory protein binding from the ENCODE project, the regulatory motifs based on commercial, literature and motif-finding analysis of the ENCODE project, and the eQTLs from Genotype-Tissue Expression (GTEx) project [<xref ref-type="bibr" rid="pgen.1006719.ref065">65</xref>] were extract from web-based HaploReg v4 [<xref ref-type="bibr" rid="pgen.1006719.ref025">25</xref>]. For variants within the tractable credible sets in the fine mapping analyses, similar analyses were also conducted.</p><p>To assess whether the novel loci identified in the sex-combined and sex-specific analyses were associated with any related cardiometabolic and anthropometric traits, or may be in high LD with known eQTLs, we examined the NHGRI-EBI GWAS Catalog [<xref ref-type="bibr" rid="pgen.1006719.ref027">27</xref>] and the GRASP (Genome-Wide Repository of Associations Between SNPs and Phenotypes) catalog [<xref ref-type="bibr" rid="pgen.1006719.ref028">28</xref>] for reported variant-trait associations near our lead variants. We supplemented the catalogs with additional genome-wide significant associations of interest from the literature [<xref ref-type="bibr" rid="pgen.1006719.ref007">7</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006719.ref009">9</xref>,<xref ref-type="bibr" rid="pgen.1006719.ref066">66</xref>]. We used PLINK to identify variants within 1 Mb of lead variants. All variants within the specified regions with r<sup>2</sup> &gt; 0.7 (1000 Genomes AFR) were retained from the catalogs for further evaluation.</p><p>Given our sample sizes in the discovery and replication stages in our African ancestry populations, we have &gt;80% power to detect variants explaining 0.08% variance for BMI that corresponds to effect sizes of 0.09 and 0.20 SD units for MAF of 0.05 and 0.01, respectively. For WHR<sub>adjBMI</sub>, we have &gt;80% power to detect variants explaining 0.18% variance that corresponds to effect sizes of 0.14 and 0.30 SD units for MAF of 0.05 and 0.01, respectively.</p>