<p>HeLa, EL-4, MOLT-4, and the JAR/BeWo human trophoblast cell lines were obtained from ATCC (Manassas, VA, USA) and grown in the recommended media. YT cells were cultured in RPMI 1640 media containing 10% fetal bovine serum, 100 U/ml penicillin, 100 U/ml streptomycin, sodium pyruvate and L-glutamine.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Healthy volunteers were recruited through the NCI-Frederick Research Donor Program (<ext-link ext-link-type="uri" ns0:href="http://ncifrederick.cancer.gov/programs/science/rdp/default.aspx">http://ncifrederick.cancer.gov/programs/science/rdp/default.aspx</ext-link>). The <italic>KIR</italic> and <italic>HLA</italic> genotype of each donor was determined as previously described [<xref ref-type="bibr" rid="pgen.1007163.ref030">30</xref>]. NK cells were separated from the peripheral blood of healthy donors by Histopaque (Sigma-Aldrich, St Louis, MO, USA) gradient centrifugation using the RosetteSep Human NK Cell Enrichment Cocktail (STEMCELL Technologies, Vancouver, BC, Canada). Human tissue samples were obtained from surgically removed tissue (liver, spleen, endometrium, decidua, tonsil) at the Karolinska University Hospital, Stockholm, Sweden. Written and oral informed consent was obtained from all patients, and the study was approved by the Regional Ethics Review Board, Stockholm, Sweden (approval numbers: 2017-1659-32, 2013/2285-31/3, 2006/229-31/3, 2013/1324-31/2, 2017-649-31/1). Lymphocytes from liver, decidua, and endometrium were isolated using enzymatic digestion as previously described [<xref ref-type="bibr" rid="pgen.1007163.ref031">31</xref>]. Tonsil and spleen were mechanically dissociated using scalpels followed by filtration. Mononuclear cells were obtained by density centrifugation using Histopaque (Sigma-Aldrich).</p><p>Total RNA from 20 human tissues (Human total RNA master panel II) was obtained from Clontech (Mountain View, CA, USA). Total RNA from purified NK cells or the YT cell line was isolated from 1&#8211;5 x 10<sup>6</sup> cells with the RNeasy kit (Qiagen, Valencia, CA, USA). A cDNA synthesis reaction was carried out using Random Hexamer primer, Taqman Reverse Transcription Reagents kit (Applied Biosystems, Foster City, CA, USA) according to the manufacturer&#8217;s instructions. A forward primer in exon -1a of the NK-Pro transcript (5&#180;-AGAAGGGCTGGAGAAGCAGGAG-3&#180;) was used together with an exon 1<sub>3</sub> reverse primer upstream of the major <italic>HLA-C</italic> TSS (5&#180;-GGACTGCGGAGACGCTGATTGG-3&#180;) for the initial detection of NK-Pro transcripts. Additional <italic>HLA-C</italic> specific exon-1a forward (5&#180;-GGGATGAGAGGGGCAGASAG-3&#180;) and exon 2 reverse (5&#180;-GTGCCTGGCGCTTGTASTTC-3&#180;) primers were used to confirm the NK-specificity of <italic>HLA-C</italic> transcripts (<xref ref-type="fig" rid="pgen.1007163.g001">Fig 1C</xref>). For full-length transcripts, an alternative 3&#180; primer located immediately following the HLA-C stop codon was used (5&#180;-GTCCCACACAGGCAGCTGTCTC-3&#180;). To assay the level of exon 1 skipping, an exon 2 reverse primer was used (5&#180;-GAACTGCGTGTCGTCCACGTAG-3&#180;). PCR products were cloned into the pCR2.1-topo vector (Invitrogen, Carlsbad, CA, USA) and sequenced. The sequences of the alternatively spliced NK-Pro transcripts have been deposited in GenBank, and can be found under accession numbers MF536989-MF536999 for the peripheral blood NK cell cDNAs, and accession numbers MF563479-MF563493 for the spleen and bone marrow cDNAs.</p><p>Full-length <italic>HLA-C</italic> cDNAs with varying 5&#180;-UTR lengths were cloned into the pEF6/V5-His TOPO-TA vector (Thermo Fisher Scientific, Waltham, MA, USA) and verified by sequencing. 1 ug of each construct was transfected into the human JAR trophoblast cell line using HilyMax Transfection Reagent (Dojindo Molecular Technologies Inc., Rockville, MD, USA). Cells were harvested with Nonidet-P40 (NP-40) lysis buffer (1% NP-40, 50 mM Tris-HCl, pH 8.0, 150 mM NaCl) supplemented with complete mini protease inhibitor cocktail tablets (Roche Diagnostics, Indianapolis, IN, USA), and protein concentrations were determined using a Nanodrop 2000 spectrophotometer (Thermo Fisher Scientific). Equal amounts of total protein were separated using sodium dodecyl sulfate-PAGE (SDS-PAGE) on 4&#8211;12% Tris-Glycine gels (Invitrogen) and transferred to Immobilon-P membrane (Sigma-Aldrich). Membranes were blocked for one hour in 5% milk solution in PBST (Phosphate Buffered Saline, pH 7.4, 0.1% Tween 20) and subsequently probed for 1.5 hours with an anti-HLA-C antibody (Abcam, Cambridge, MA, USA; ab126722) diluted 1:5000 in 5% milk PBST solution. Blots were washed 4 times for 5 minutes each with PBST and then were incubated for 20 minutes with an anti-rabbit HRP-linked IgG antibody (Cell Signaling Technology, Danvers, MA, USA) diluted 1:20,000 in 5% milk PBST solution. Blots were washed 4 times for 5 minutes with PBST and proteins were visualized using Amersham ECL Western blotting detection reagents (GE Healthcare, Pittsburgh, PA, USA). Normalization for protein loading was accomplished by first stripping blots with SDS stripping buffer for 10 minutes and then washing 3 times with PBS, pH7.4 for 5 minutes. Blots were then probed for 1 hour with a monoclonal anti-&#946;-Actin antibody (Sigma-Aldrich A2228) at a concentration of 1 &#956;g/ml followed by an anti-mouse HRP linked IgG antibody (Cell Signaling Technology) for 20 minutes diluted 1:20,000 in 5% milk PBST. Levels of &#946;-Actin were visualized using Amersham ECL Western detection (GE healthcare).</p><p>Nuclear extracts were prepared from cell lines using the CellLytic NuCLEAR extraction kit (Sigma-Aldrich). Protein concentration was measured with a Bio-Rad protein assay, and samples were stored at &#8722;70&#176;C until use. Double-stranded DNA oligonucleotide probes containing either the AP1 site (5&#180;-GACACGACC<bold>TGAGTCA</bold>CATTAGC-3&#180;) or the the combined SP1/Ets site (5&#180;-GACAT<bold>GGGCAGGAAGT</bold>GAGGGAC-3&#180;) were synthesized (IDT, Newark, NJ, USA). Probes were labeled with &#945;-[32P]deoxycytidine triphosphate (3000 Ci/mmol; PerkinElmer, Waltham, MA, USA) by fill-in using the Klenow fragment of DNA polymerase I (Invitrogen). 32P-labeled double-stranded oligonucleotides were purified using mini Quick Spin Oligo Columns (Roche). DNA&#8211;protein binding reactions were performed in a 10-&#956;l mixture containing 5 &#956;g nuclear protein and 1 &#956;g poly[dI-dC] (Sigma-Aldrich) in 4% glycerol, 1 mM MgCl2, 0.5 mM EDTA, 0.5 mM DTT, 50 mM NaCl, 10 mM Tris-HCl (pH 7.5). Nuclear extracts were incubated with 1 &#956;l 32P-labeled oligonucleotide probe (10,000 cpm) at room temperature for 20 min and then loaded on a 5% polyacrylamide gel (37:5:1). Electrophoresis was performed in 0.5x TBE for 2 h at 130 V, and the gel was visualized by autoradiography.</p><p>Lymphocytes isolated from whole blood via Ficoll-Paque Plus gradient centrifugation (GE Healthcare) from 16 healthy volunteers were stained extracellularly with antibodies to determine expression of HLA-C on various subsets. The antibodies used were: CD3 [clone UCHT1] BV605, CD19 [clone HIB19] PE-Cy7, CD56 [clone HCD56] BV711 (BioLegend, San Diego, CA, USA); CD4 [clone SK3] PE-Cy7, CD8 [clone RPA-T8] APC (BD Biosciences, Franklin Lakes, NJ, USA); HLA-C [clone DT9] conjugated to AlexaFluor488. Data from the samples collected on a BD LSRFortessa flow cytometer were analyzed using FlowJo v10.1. Mononuclear cells from additional blood donors and from indicated tissues were analyzed by flow cytometry at the Karolinska Institutet, Stockholm, Sweden. The following additional antibodies were used: CD3 [clone UCHT1] PE-Cy5 (Beckman Coulter, Brea, CA, USA); CD19 BV510 (Biolegend); NKG2A [clone Z199] BB515, CD56 [clone NCAM16.2] BUV737, CD57 [clone NK-1] BV605, HLA-C [clone DT9] PE, KIR2DL2/3/S2 [clone CH-L] BUV395 (BD Biosciences); KIR2DL1/S1 [clone EB6] PC-5.5 (Beckman Coulter); KIR3DL1 [clone DX9] BV421 (Biolegend); KIR2DS4 [clone REA860] PE-Vio615 (Miltenyi Biotec, Cambridge, MA, USA). Live/Dead marker Aqua (Thermo Fisher Scientific) was also used. Samples were acquired on a 5 laser BD LSRFortessa flow cytometer and analyzed using FlowJo v10.1.</p><p>The frequency of degranulated human NK cells was quantitated by flow cytometric detection of cell surface CD107a. Purified donor human NK cells (10<sup>5</sup>) were co-cultured with 721.221 tumor cells in a round bottom 96-well plate at an effector target ratio of 1:1 in RPMI 1640 supplemented with 10% FBS, 10 mM Hepes, 2 mM L-glutamine, 0.1 mM nonessential amino acids, 1 mM sodium pyruvate, Penicillin 100 U/ml, Streptomycin 100 ug/ml, and 100 IU/ml recombinant human interleukin 2 (Tecin; Teceleukin, obtained from the Biological Resources Branch of the National Cancer Institute). Anti-human CD107a PE or BV421 (Biolegend) was added at 3 ul to 200 ul of the co-culture and the mixture was gently mixed by pipetting and then spun down at 50 g for 5 minutes and incubated for 1 h at 37&#176;C in 5% CO<sub>2</sub> after which Golgi-Stop (BD Biosciences) was added and cells were incubated for an additional 4 h at 37&#176;C in 5% CO<sub>2</sub>. Cells were washed with sorter buffer (HBSS containing 0.5% BSA, 1mM EDTA, 25mM Hepes, 0.05% sodium azide) and Fc receptors were blocked with human FcR Blocking Reagent (Miltenyi Biotec) for 5 minutes and then then stained with anti-human CD56 PE or APC (clone HCD56, Biolegend) for 10 minutes. Stained cells were washed twice with sorter buffer and fixed with BD Cytofix (BD Biosciences) for 20 minutes. Cells were washed twice with sorter buffer and flow cytometric analysis was performed on a LSRFortessa instrument (BD Biosciences). Degranulated NK cells were analyzed using FlowJo software by gating on single cells (FSC-H x FSC-A) in the lymphocyte gate and then the frequency of CD56<sup>dim</sup> CD107a<sup>+</sup> cells was quantitated.</p>