<p><italic>Trichoderma</italic> strains used for the whole genome sequencing are given in Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s002">S1 Table</xref>. All fungal strains and other organisms used in experiments and their respective accession numbers for DNA barcode sequences deposited in public databases and/or references are given in Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s007">S6 Table</xref>.</p><p>For inoculum preparation, fungi were cultivated on potato dextrose agar (Sigma Aldrich, Steinheim, Germany) at 28&#176;C for 4 days. Spore suspensions (3x10<sup>6</sup> spores/ml) were prepared in 0.9% (w/v) NaCl with 0.025% (w/v) Tween 20 (Carl Roth, Austria). Growth tests were performed in CELLSTAR 24 Well Cell Culture plates (Greiner bio-one International). 1 ml of spore suspension of the fungus to be tested was inoculated on the following substrates: i) heat-treated (100&#176;C for 3 hours) dried fruiting bodies of <italic>Ganoderma lucidum</italic> (Polyporales, Basidiomycota) (0.3% w/v), ii) epiphyte-free dried leaves of <italic>Shorea johorensis</italic> (Malvales, Angiosperms, Plantae) (0.3% w/v), iii) naturally degraded dead wood of <italic>S</italic>. <italic>johorensis</italic> (0.3% w/v), iv) commercial saw dust (local supplier, Vienna, Austria) (0.3% w/v), v) microcrystalline cellulose (0.05 mM research grade; AMS Biotechnology, Milton park, UK) in 0.5% (w/v) Agar-Agar Kobe I (Carl-Roth, Mannheim, Germany), vi) 2% (w/v) pre-treated (steam exploded) wheat straw 3% (w/v) in Agar-Agar Kobe I, vii) 0.3% pectin (w/v) in Agar-Agar Kobe I. Growth in 0.5% (w/v) Agar-Agar Kobe I was also tested as a control. All non-powdered substrates were finely ground and then sterilized at 120&#176;C for 20 min. Experiment was carried out in quadruples. The plates were photographed after incubation at 28&#176;C for seven days in darkness.</p><p>Dual confrontation assays between fungi were done as described in Atanasova <italic>et al</italic>. [<xref ref-type="bibr" rid="pgen.1007322.ref015">15</xref>]. For these experiments, fungi were incubated for 10 days on PDA at 25&#176;C and 12 hours with cyclic illumination. When required, slower growing fungi (such as <italic>Lentinula edodes</italic> and <italic>Leucoagaricus gongylophorus</italic>) were inoculated 2&#8211;3 days prior to the inoculation with fast growing Hypocreales. The set of <italic>Penicillium</italic> spp. and <italic>Pestalotiopsis fici</italic> TUCIM 5788 strains was randomly selected from a pool of strains isolated from phylloplane of <italic>Shorea johorensis</italic> (Dipterocarpaceae, Plantae) from Borneo where <italic>Trichoderma</italic> spp. are common.</p><p>To analyze the interfungal interaction by confocal microscopy, spores of <italic>T</italic>. <italic>reesei</italic> strain TUCIM 4817, carrying a <italic>gfp</italic> gene under the control of a histone 3A promoter, and <italic>Pestalotiopsis fici</italic> TUCIM 5788 were inoculated on two adjacent but separated PDA agar blocks mounted between the glass slides and the cover glass using a modified Riddell slide method [114] (Supporting information <xref ref-type="supplementary-material" rid="pgen.1007322.s011">S3 Fig</xref>). The construct was incubated for 72 hours in a sterile wet chamber that hyphae of both fungi established the contacts. Live-cell imaging was performed using a Nikon C1 confocal laser scanning unit mounted on a Nikon Eclipse TE2000-E inverted microscope base (Nikon GmbH, Vienna, Austria). A Nikon Plan Apo VC 100&#215;/1.4 with oil immersion objective lens was used. GFP was excited with an argon ion laser at 488 nm. The emitted fluorescence was separated by a Nikon MHX40500b/C100332 filter cube and detected with a photomultiplier tube within the range of 500&#8211;530 nm. Bright light images were captured simultaneously with a Nikon C1-TD transmitted light detector mounted behind the condenser turret.</p><p>For scanning electron microscopy, a coverslip (1 cm<sup>2</sup>) was placed on the centre of an agar plate inoculated with partner fungi and incubated until contact between hyphae was established (in average for 72 hours). The hyphae were then fixed with 2.5% (v/v) glutaraldehyde in 0.5 M potassium phosphate buffer and used for examining by SEM (HITACHI S-3000N, Tokyo, Japan).</p><p>The genomes of five <italic>Trichoderma</italic> species (<italic>T</italic>. <italic>longibrachiatum</italic> ATCC 18648, <italic>T</italic>. <italic>citrinoviride</italic> TUCIM 6016, <italic>T</italic>. <italic>harzianum</italic> CBS 226.95, <italic>T</italic>. <italic>guizhouense</italic> NJAU 4742 and <italic>T</italic>. <italic>asperellum</italic> CBS 433.97) were sequenced for this work (Supporting information <xref ref-type="supplementary-material" rid="pgen.1007322.s002">S1 Table</xref>). Four of them <italic>(T</italic>. <italic>longibrachiatum</italic>, <italic>T</italic>. <italic>citrinoviride</italic>, <italic>T</italic>. <italic>harzianum</italic>, and <italic>T</italic>. <italic>asperellum)</italic> were sequenced using an Illumina platform. To this end, Illumina fragments (270 bp insert size) and 4 kbp long mate-pair (LMP) libraries were combined. The fragment libraries were produced from 1 &#956;g of genomic DNA, sheared to 270 bp using an E210 Focused-ultrasonicator (Covaris) and size selection was carried out using SPRIselect (Beckman Coulter). The fragments were treated with end-repair, A-tailing, and ligation of Illumina adapters (Eurofins MWG Operon), using a NEBNext Ultra DNA Library Prep Kit (New England Biolabs Inc.).</p><p>Two types of LMP libraries were used, CLIP (Cre-Lox Inverse PCR) and LFPE (Ligation Free Paired-End), both of which used 15 &#956;g of genomic DNA sheared with HydroShear (Genomic Solutions) using a selection size of 4 kb. For CLIP, the size selected DNA was ligated to adaptors containing loxP and the Illumina specific primer sequence. This adaptor ligated DNA fragments were then circularized via recombination by a Cre (NEB) excision reaction. The circularized DNA templates were then digested with a cocktail of four base cutter restriction enzymes, i.e. NlaIII, MseI, HypCH4IV (NEB), followed by self-ligation. The paired end library was then amplified via inverted PCR using an Illumina specific primer set. The size of the amplified paired end library was selected by running on a 1.8% (w/v) agarose gel followed by gel purification of the desired fragment (300&#8211;600 bp).</p><p>For LFPE library, the sheared DNA was treated with end repair adapters and ligated with biotinylated adapters. The adapter ligated DNA fragments were circularized by intra-molecular hybridization. The circularized DNA templates were digested by T7 Exonuclease and S1 Nuclease (Thermo Fisher Scientific). The digested fragments were treated with A-tailing Enzyme (NEB), followed by immobilization of mate-paired fragments on strepavidin beads (Thermo Fisher Scientific). Illumina compatible adapters (IDT, Inc) were ligated to the mate paired fragments and 12 cycles of PCR was used to enrich for the final library (KAPA Biosystems).</p><p>All prepared libraries were quantified using KAPA Biosystem&#8217;s next-generation sequencing library qPCR kit and run on a Roche LightCycler 480 real-time PCR instrument. The quantified libraries were then prepared for sequencing on the Illumina HiSeq sequencing platform utilizing a TruSeq paired-end cluster kit v3 and Illumina&#8217;s cBot instrument to generate a clustered flowcell for sequencing. Sequencing of the flowcell was performed on the Illumina HiSeq2000 sequencer using a TruSeq SBS sequencing kit 200 cycles v3, following a 2 x 100 bp or 2 x 150 bp run recipe.</p><p>Illumina data were QC filtered for artifact/process contamination and subsequently assembled using Rnnotator [<xref ref-type="bibr" rid="pgen.1007322.ref082">82</xref>] for transcriptomes, and AllPathsLG [<xref ref-type="bibr" rid="pgen.1007322.ref083">83</xref>] for genomes. The Pacific Biosciences library was prepared from 5 &#956;g of gDNA sheared using a Covaris LE220 focused-ultrasonicator with their Blue miniTUBES to generate sheared fragments of 3kb in length. The sheared DNA fragments were then prepared according to the Pacific Biosciences protocol and using their SMRTbell Template Preparation Kit, where the fragments were treated with DNA damage repair, had their ends repaired so that they were blunt-ended, and 5&#8217; phosphorylated. Pacific Biosciences hairpin adapters were then ligated to the fragments to create the SMRTbell template for sequencing. The SMRTbell templates were then purified using exonuclease treatments and size-selected using AMPure PB beads. Sequencing primer was then annealed to the SMRTbell templates and Version C2 sequencing polymerase was bound to them. The prepared SMRTbell template libraries were then sequenced on a Pacific Biosciences RSII sequencer using Version C2 chemistry and 2x45min sequencing movie run times.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genomes were annotated using the JGI Annotation pipeline and made available via JGI fungal genome portal MycoCosm (<ext-link ext-link-type="uri" ns0:href="http://jgi.doe.gov/fungi">jgi.doe.gov/fungi</ext-link> [<xref ref-type="bibr" rid="pgen.1007322.ref084">84</xref>]). They have also been deposited at DDBJ/EMBL/GenBank as specified in Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s002">S1 Table</xref>.</p><p>The genome of <italic>T</italic>. <italic>guizhouense</italic> NJAU 4742 was shotgun sequenced using a Roche 454 GS FLX system at the Chinese National Human Genome Center (Zhangjiang Hi-tech Park, Shanghai, China) with 28.4X coverage. The fragment libraries were produced from 5 &#956;g of genomic DNA, sheared to 300&#8211;500 bp using M220 Ultrasonicator (Covaris, America) and was purified with Agencourt Ampure beads (Beckman, America). The fragment libraries were constructed with Purified DNA fragments by using DNA Library Preparation kit (Roche Applied Science, Switzerland) and fixed on magnetic beads with GS emPCR kit (Roche Applied Science). The 569 Mb raw data were achieved from 454 GS FLX system with 1,435,699 reads.</p><p>For sequence scaffolding, Solexa Mate Pair reads were used to establish the genome scaffolds. 5 &#956;g of genomic DNA was sheared with a Hydroshear device (Gene Machine) to generate 3&#8211;5 kb DNA fragments. The library was prepared by using TruSeq DNA Sample Prep Kit-SetA (illumina, America), and amplified using TruSeq PE Cluster Kit (illumina, America), and then sequenced in Solexa sequencing machine (illumina, America). Gene calls were generated using FGENESH [<xref ref-type="bibr" rid="pgen.1007322.ref085">85</xref>], ExonHunter [<xref ref-type="bibr" rid="pgen.1007322.ref086">86</xref>] and AUGUSTUS version 2.7 [<xref ref-type="bibr" rid="pgen.1007322.ref087">87</xref>].</p><p>Annotation of the genes encoding carbohydrate active enzymes involved in plant cell wall degradation (pcwdCAZome) in the nine <italic>Trichoderma</italic> genomes was performed using the Carbohydrate-Active Enzyme database (CAZy) nomenclature [<xref ref-type="bibr" rid="pgen.1007322.ref088">88</xref>, <xref ref-type="bibr" rid="pgen.1007322.ref089">89</xref>], by comparing each protein model from the genome by the sequence similarity search tool (blastp) to a collection of protein modules corresponding to catalytic and carbohydrate-binding modules derived from CAZy. Individual hits were then compared by HMMer to models corresponding to each CAZy family to allow an assignment of each identified protein.</p><p>Accession numbers of genes composing pcwdCAZome and respective regulatory proteins in <italic>Trichoderma</italic> genomes are given in Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s004">S3 Table</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Principal component analysis and two-way cluster analysis of the Hypocreales pcwdCAZomes (Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s004">S3 Table</xref>) were made with the use of <ext-link ext-link-type="uri" ns0:href="https://biit.cs.ut.ee/clustvis/">https://biit.cs.ut.ee/clustvis/</ext-link> [<xref ref-type="bibr" rid="pgen.1007322.ref090">90</xref>]. Cluster analysis was made with Euclidian distance and complete linkage method.</p><p>To analyse whether the genomic location of the identified pcwdCAZomes would be syntenic among the nine <italic>Trichoderma</italic> species, we used the manually annotated chromosomes of <italic>T</italic>. <italic>reesei</italic> [<xref ref-type="bibr" rid="pgen.1007322.ref045">45</xref>] as a template. Orthologs for each individual gene from the other <italic>Trichoderma</italic> spp. were then located on their genomic scaffolds, and at least five genes flanking its 5&#8217; and 3&#8217; area were retrieved and identified by blastp. A synteny value of 100% was assigned between <italic>T</italic>. <italic>reesei</italic> and another <italic>Trichoderma</italic> sp. when all investigated flanking genes were orthologues and their order was conserved.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">One-hundred genes were randomly selected from the genomes of the nine <italic>Trichoderma</italic> spp. and 12 reference Hypocreales (<italic>Escovopsis weberi</italic>, <italic>Metarhizium acridium</italic>, <italic>M</italic>. <italic>robertsii</italic>, <italic>Calviceps purpurea</italic>, <italic>Ophiocordyceps sinensis</italic>, <italic>Beauveria bassiana</italic>, <italic>Cordyceps militaris</italic>, <italic>Fusarium graminearum</italic>, <italic>F</italic>. <italic>oxysporum</italic> f. sp. <italic>lycopersici</italic>, <italic>F</italic>. <italic>pseudograminearum</italic>, <italic>F</italic>. <italic>fujikuroi</italic>, and <italic>Nectria haematococca;</italic> see Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s002">S1 Table</xref> and <xref ref-type="supplementary-material" rid="pgen.1007322.s003">S2 Table</xref>) based on two requirements: (a) they should display a syntenic position in all genomes, and (b) be true orthologues (no other gene encoding a protein with amino acid similarity &gt;50% present). <italic>Neurospora crassa</italic> and <italic>Chaetomium globosum</italic> (Sordariales) were chosen as outgroups. For each gene the alignments of nucleotide sequences consisting of coding regions were prepared using ClustalW [<xref ref-type="bibr" rid="pgen.1007322.ref091">91</xref>] and analyzed for the neutral evolution [<xref ref-type="bibr" rid="pgen.1007322.ref092">92</xref>] using DNASp V5.10.01 [<xref ref-type="bibr" rid="pgen.1007322.ref093">93</xref>] based on Tajima&#8217;s D test [<xref ref-type="bibr" rid="pgen.1007322.ref094">94</xref>]as described by Rozas [<xref ref-type="bibr" rid="pgen.1007322.ref095">95</xref>] (Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s003">S2 Table</xref>). Multiple sequence alignments of each protein were done using ClustalW [<xref ref-type="bibr" rid="pgen.1007322.ref091">91</xref>]. Resulting alignments were examined in Genedoc [<xref ref-type="bibr" rid="pgen.1007322.ref096">96</xref>] and then subjected to the phylogenetic analysis online in PhyML [<xref ref-type="bibr" rid="pgen.1007322.ref097">97</xref>] based on best amino acid substitution model acquired using &#8220;smart model selection&#8221; option (<ext-link ext-link-type="uri" ns0:href="http://www.atgc-montpellier.fr/phyml-sms/">http://www.atgc-montpellier.fr/phyml-sms/</ext-link>). Maximum likelihood trees assessed using 1000 bootstrap replicates were also constructed individually for each of the 100 protein sequences and the phylome is deposited at <ext-link ext-link-type="uri" ns0:href="http://itol.embl.de/shared/druzhininaetal">http://itol.embl.de/shared/druzhininaetal</ext-link>. Accession numbers of all genes used in phylogenomic analysis are given in Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s003">S2 Table</xref>.</p><p>For a combined analysis, a concatenated set of 100 proteins for each of 23 species was subjected to the alignment algorithm using the stand alone MAFFT tool [<xref ref-type="bibr" rid="pgen.1007322.ref098">98</xref>] with G-INS-i parameters. Selection of conserved blocks was done using &#8220;relaxed&#8221; conditions in Gblocks [<xref ref-type="bibr" rid="pgen.1007322.ref099">99</xref>]. The final concatenated alignment contained 47 726 amino acids. The selection of best amino acid substitution model was done using ProtTest 3 [<xref ref-type="bibr" rid="pgen.1007322.ref100">100</xref>] based on BIC criterion. The Bayesian analysis was performed using MrBayes v3.2.5 [<xref ref-type="bibr" rid="pgen.1007322.ref101">101</xref>, <xref ref-type="bibr" rid="pgen.1007322.ref102">102</xref>], 1 million generations and the Dayhoff I+G+F amino acid substitution model [<xref ref-type="bibr" rid="pgen.1007322.ref103">103</xref>]. Two simultaneous, independent analyses starting from different random trees were run, each using three heated chains and one "cold" chain. Once the analyses were completed, 7500 trees were summarized after discarding the first 25% of the obtained 10,000 trees, resulting in a consensus tree. The parameters of phylogenetic analyses and accession numbers of individual genes are given in Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s003">S2 Table</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To reconstruct a phylogenetic tree that included also all fungi for which putative pcwdCAZyme homologs to <italic>Trichoderma</italic> have been identified (see below), the amino acid sequences from four nuclear genes that have previously been shown to be suitable phylogenetic markers for Ascomycota multilocus phylogeny (histone acetyltransferase subunit of the RNA polymerase II holoenzyme, FG533; NAD-dependent glutamate dehydrogenase, FG570; translation initiation factor eIF-5, FG832; and TSR1p, a protein required for processing of 20S pre-rRNA, MS277) were retrieved from FunyBase [<xref ref-type="bibr" rid="pgen.1007322.ref104">104</xref>] (<ext-link ext-link-type="uri" ns0:href="http://genome.jouy.inra.fr/funybase">http://genome.jouy.inra.fr/funybase</ext-link>), GenBank (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/genbank/">http://www.ncbi.nlm.nih.gov/genbank/</ext-link>), the Joint Genome Institute (<ext-link ext-link-type="uri" ns0:href="http://genome.jgi-psf.org/programs/fungi/index.jsf?projectList">http://genome.jgi-psf.org/programs/fungi/index.jsf?projectList</ext-link>), EnsemblFungi (<ext-link ext-link-type="uri" ns0:href="http://fungi.ensembl.org/index.html">http://fungi.ensembl.org/index.html</ext-link>) and Broad Institute (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/">http://www.broadinstitute.org/</ext-link>) databases. Complete sets of amino acid sequences for 128 fungi (including the nine <italic>Trichoderma</italic> spp.) were prepared. Concatenated alignments (provided in Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s012">S1 Data</xref>) and an MCMC analysis using MrBayes v3.2.5 was performed as described above. Accession numbers of individual genes from Ascomycota fungi are given in Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s008">S7 Table</xref>.</p><p>In the case of swollenin, where the best Blastp hits included plant species, an appropriate species tree was constructed using the NCBI Taxonomy Browser.</p><p>Protein sequences from the nine species that belonged to the same GH family were aligned using CLUSTALW [<xref ref-type="bibr" rid="pgen.1007322.ref088">88</xref>] and subjected to phylogenetic analysis with PhyML embedded in TOPALI v 2.5 [<xref ref-type="bibr" rid="pgen.1007322.ref105">105</xref>]. To retrieve the respective closest pcwdCAZyme neighbors from other fungi, one or more <italic>Trichoderma</italic> proteins from each GH protein family (including PL1 and AA9 auxiliary proteins) were first subjected to a sequence similarity search by blastp against the NCBI database (finished by December 28, 2015). All hits with a query coverage of &gt;90% and an E-value &lt; 10<sup>&#8722;100</sup> were collected. We initially used a less stringent E-value (&lt; 10<sup>&#8722;40</sup>), but found that the validated close neighbors were all characterized by &lt; 10<sup>&#8722;100</sup>. When this analysis failed to retrieve orthologs from closely related species, the analysis was repeated using tBlastn. Only published sequences were used. The final set of sequences was realigned using Muscle 3.8.425 [<xref ref-type="bibr" rid="pgen.1007322.ref106">106</xref>] and integrated in the program Aliview [<xref ref-type="bibr" rid="pgen.1007322.ref107">107</xref>]. Duplicate sequences from the same species were removed. Highly polymorphic regions were removed using the Gblocks [<xref ref-type="bibr" rid="pgen.1007322.ref099">99</xref>] server with unconstrained parameters. The curated alignments were then subjected to evolutionary analysis using MrBayes v3.2.5 [<xref ref-type="bibr" rid="pgen.1007322.ref102">102</xref>] as described above. The number of generations chosen for each phylogenetic tree depended on the number of sequences in the alignment. As a rule, the MCMC analysis was run for 1 million generations for all the alignments containing less than 100 sequences, and these were subsequently increased by 1 million until the standard deviation of split frequencies of the two parallel yet independent runs fell under 0.05 in the case of alignments containing more than 100 sequences. Parameters of individual phylogenetic analyses are given in Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s004">S3 Table</xref>.</p><p>The incongruent topologies of the phylograms of individual proteins from the pcwdCAZome compared to the topology of the Hypocreales phylogenomic tree and the multilocus phylogram of Ascomycota fungi could be the result of gene duplication (GD), gene loss, or LGT. To distinguish between these possibilities, we reconciled the protein trees of each GH/AA9/PL1 family to the multilocus Ascomycota phylogeny (<xref ref-type="fig" rid="pgen.1007322.g006">Fig 6A</xref>) in NOTUNG [<xref ref-type="bibr" rid="pgen.1007322.ref040">40</xref>&#8211;<xref ref-type="bibr" rid="pgen.1007322.ref042">42</xref>]. Using the approach of Wisecaver <italic>et al</italic>.[<xref ref-type="bibr" rid="pgen.1007322.ref043">43</xref>], we assigned costs to GD, LGT, and gene loss and determined the most parsimonious combination of three events to explain the individual pcwdCAZome trees in view of the topology of the Ascomycota phylogeny. An edge weight threshold of 0.9 was applied. To find the most appropriate parameters, we evaluated three different ratios of transfer to GD costs (2, 4 and 6) and compared the predicted gene transfers to those obtained by T-Rex [<xref ref-type="bibr" rid="pgen.1007322.ref108">108</xref>]. The latter infers LGT by quantifying the proximity between two phylogenetic trees using a refinement of the Robinson and Foulds distance using midpoint rooting (see Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s005">S4 Table</xref>). Finally, we used transfer costs twice as much as GD cost because it was the lowest ratio that identified LGT events that were in agreement with those suggested by the discordance of species and protein phylogeny. LGT events were only inferred when a CAZyme tree topology was contradictory to the Ascomycota phylogeny and could not be more parsimoniously reconciled by a combination of differential GD and gene loss. The scores are given in Supporting Information <xref ref-type="supplementary-material" rid="pgen.1007322.s005">S4 Table</xref>.</p>