<p>Strains used in this study are derivatives of BY4741. Genotypes of each strain are provided in <xref ref-type="supplementary-material" rid="pgen.1007232.s007">S1 Table</xref>. Strains containing a chromosomal <italic>his3AI-[&#916;1]</italic>-marked Ty1 element (Ty1<italic>his3AI</italic>-3114) were described previously [<xref ref-type="bibr" rid="pgen.1007232.ref019">19</xref>, <xref ref-type="bibr" rid="pgen.1007232.ref085">85</xref>]. Recombination of the <italic>his3AI-[&#916;1]</italic> allele with the <italic>his3&#916;1</italic> allele present in strain BY4741 derivatives does not result in a functional <italic>HIS3</italic> allele [<xref ref-type="bibr" rid="pgen.1007232.ref036">36</xref>]. Strains containing Mediator subunit gene deletions were constructed via lithium acetate transformation with a <italic>KanMX</italic> allele as described [<xref ref-type="bibr" rid="pgen.1007232.ref086">86</xref>, <xref ref-type="bibr" rid="pgen.1007232.ref087">87</xref>].</p><p>A chromosomal Ty1<italic>kanMXAI</italic> element was introduced into strain BY4741 by inducing the expression of a Ty1<italic>kanMXAI</italic> element fused to the <italic>GAL1</italic> promoter on a yeast <italic>URA3</italic>-marked 2&#956; vector (plasmid pGTy1<italic>kanMXAI</italic> [<xref ref-type="bibr" rid="pgen.1007232.ref039">39</xref>]), as described previously [<xref ref-type="bibr" rid="pgen.1007232.ref019">19</xref>]. Briefly, a transformant of strain BY4741 harboring plasmid pGTy1<italic>kanMXAI</italic> was grown on SC-URA 2% glucose 2% raffinose agar at 20&#176;C. Twenty single colonies were picked and struck for single colonies on SC-URA 2% glucose and grown at 30&#176;C. A single Ura<sup>+</sup> colony harboring cells that had maintained the pGTy1<italic>kanMXAI</italic> plasmid throughout galactose-induction was picked from each streak and struck for single colonies on YPD and grown at 30&#176;C. A single colony was picked from each YPD streak and used to make a 1 cm<sup>2</sup> patch on YPD at 30&#176;C. Each patch was replicated to a fresh YPD agar and grown for three days at 20&#176;C, and then replicated to YPD agar containing 200 &#956;g/ml G418 and grown at 30&#176;C for three days. Strains harboring a chromosomal Ty1<italic>kanMXAI</italic> element were identified by the appearance of G418<sup>R</sup> papillae. A strain with a representative number of G418<sup>R</sup> papillae, JC6464, was chosen for further analysis. A <italic>med15&#916;</italic>:<italic>URA3</italic> derivative of strain JC6464 was constructed by PCR-mediated gene disruption.</p><p>Plasmid Ycp50-Ty1his3AI-[<italic>&#916;</italic>1] is a <italic>URA3-CEN</italic> plasmid containing a Ty1<italic>his3AI-[&#916;1]</italic> element, constructed by replacing the <italic>ClaI</italic> fragment containing the <italic>his3AI</italic> allele in pBDG633 with a <italic>ClaI</italic> fragment containing <italic>his3AI-[&#916;1]</italic> and kindly provided by Dr. David Garfinkel [<xref ref-type="bibr" rid="pgen.1007232.ref088">88</xref>]. Plasmid pBJC1250 is a <italic>LEU2- CEN</italic> vector containing a Ty1<italic>his3AI-[&#916;1]</italic> element wherein the U3 region of the 5&#8242; LTR was replaced with a <italic>TEF1</italic> promoter (herein referred to as P<sub><italic>TEF1</italic></sub>-Ty1<italic>his3AI</italic>; see <xref ref-type="fig" rid="pgen.1007232.g005">Fig 5</xref>). Plasmid pBJC80, herein referred to as pGTy1<italic>&#916;</italic>POL, has been described previously [<xref ref-type="bibr" rid="pgen.1007232.ref069">69</xref>].</p><p>Ty1 retromobility was determined as previously described [<xref ref-type="bibr" rid="pgen.1007232.ref019">19</xref>]. Individual colonies from strains were grown in triplicate in YPD broth at 30&#176;C overnight. Each culture was then diluted in quadruplicate by a factor of 1000 in YPD broth and grown at 20&#176;C to an optical density beyond log growth phase. 1&#956;L of a 1:1000 dilution of each of the resulting 12 cultures was plated on YPD agar to provide an accurate representation of the cell density. In parallel, 100&#956;L to 1mL of each culture was plated on SC-HIS agar. All plates were grown at 30&#176;C for 3&#8211;4 days. Mobility frequency was calculated as a ratio of the number of His<sup>+</sup> colony forming units to the number of colony forming units in each culture as represented by the number of colonies growing on YPD agar. For strains for which no His<sup>+</sup> prototrophs were observed, mobility was reported as an upper limit equal to the ratio of (1/the total number of colony forming units in all three biological replicates).</p><p>For strains containing a plasmid Ty1<italic>his3AI</italic> element, the above protocol was modified such that cultures were grown in their respective selective media (SC-URA or SC-LEU with 2% glucose) at 30&#176;C until confluent, diluted 1:1000 and grown at 20&#176;C in YPD until confluent, and plated on their respective dropout media (SC-URA or SC-LEU with 2% glucose) as well as the corresponding media lacking histidine (SC-URA-HIS or SC-LEU-HIS with 2% glucose).</p><p>For strains containing a chromosomal Ty1<italic>kanMXAI</italic> element and plasmid pBJC1250 containing the P<sub><italic>TEF1</italic></sub>-Ty1<italic>his3AI</italic> element, the above protocol was modified such that cultures were grown in SC-LEU 2% glucose at 30&#176;C overnight. Cultures were diluted 1:1000 into YPD broth and separated into 12 independent cultures that were grown for three days at 20&#176;C. A 1:1000 dilution of each culture was plated on YPD agar to determine the number of colony forming units in each culture, and aliquots of each culture were plated on YPD agar containing 200 &#956;g/ml G418 to determine the number of G418<sup>R</sup> colony forming units per culture, and SC-HIS 2% glucose to determine the number of His<sup>+</sup> colony forming units per culture.</p><p>Cells were grown in YPD broth at 20&#176;C to mid-log phase. Total cellular RNA was extracted using a hot phenol/chloroform extraction protocol [<xref ref-type="bibr" rid="pgen.1007232.ref089">89</xref>]. PolyA<sup>+</sup> RNA was purified from 250&#956;g to 1mg of total cellular RNA using the Magnetic mRNA Isolation Kit (New England Biolabs) following the manufacturer&#8217;s protocol.</p><p>A 20&#956;g aliquot of total cellular RNA was separated on a 1% Seakem GTG agarose gel as previously described [<xref ref-type="bibr" rid="pgen.1007232.ref090">90</xref>]. Three to six &#956;g of poly A<sup>+</sup> RNA was separated on an 0.8% Seakem GTG agarose gel as previously described [<xref ref-type="bibr" rid="pgen.1007232.ref090">90</xref>]. Separated RNA was transferred to a Hybond XL membrane (GE Healthcare) using a gradient of 6X SSC to 10X SSC overnight at room temperature. Synthesis of <sup>32</sup>P-labeled RNA riboprobes was carried out in vitro using SP6 or T7 polymerase (New England Biolabs). Membranes were incubated with probes in NorthernMax PreHyb Buffer (Ambion) at 65&#176;C overnight. Images were scanned using a Typhoon 9400 scanner, and quantified using ImageQuant software (Molecular Dynamics, Sunnyvale, CA).</p><p>Cultures were grown at 20&#176;C in YPD for one cell doubling (OD<sub>600</sub> 0.3 to OD<sub>600</sub> 0.6) after dilution from overnight cultures grown at 30&#176;C. Protein was extracted from total cell lysates as previously described [<xref ref-type="bibr" rid="pgen.1007232.ref091">91</xref>] and resolved on a 10% SDS-PAGE gel. When resolving p18- and p22-Gag, a 15% SDS-PAGE gel was used. Protein was then transferred to a polyvinylidene difluoride (PVDF) membrane. Membranes were blocked in a 5% nonfat milk solution dissolved in phosphate buffered saline (PBS) with 0.1% TWEEN 20. Membranes were then incubated in 0.5% nonfat milk in PBS with 0.1% TWEEN 20 with a 1:7500 dilution of affinity-purified anti-VLP antisera [<xref ref-type="bibr" rid="pgen.1007232.ref053">53</xref>] to detect Gag, a 1:5000 dilution of a polyclonal antibody specific to p18-Gag (a gift from David Garfinkel, described in [<xref ref-type="bibr" rid="pgen.1007232.ref024">24</xref>]), a 1:7500 dilution of anti GAPDH monoclonal antibody (Thermo Fisher Scientific), or a 1:5000 dilution of anti actin monoclonal antibody (Abcam). Med17-myc was detected using a 1:1000 dilution of a monoclonal antibody to c-myc (Sigma-Aldrich). Membranes were subsequently incubated with horseradish peroxidase (HRP)-conjugated secondary antibodies (Millipore). Following terminal washes, membranes were incubated with SuperSignal West Pico chemiluminescence substrate (Pierce, Thermo Fisher Scientific), and exposed to film (Kodak). Antibody was stripped from membranes as described previously [<xref ref-type="bibr" rid="pgen.1007232.ref092">92</xref>]. Images were developed on film using a Model SRX-101A Medical Scanner (Konica Minolta) and scanned using a Cannon MP480 scanner. Protein bands were quantified using ImageJ (NIH). Quantification was performed using film exposed for different durations to ensure that measurements were done within the linear response range.</p><p>Cultures were grown past log growth phase at 20&#176;C in YPD broth. Total genomic DNA was isolated as previously described [<xref ref-type="bibr" rid="pgen.1007232.ref015">15</xref>, <xref ref-type="bibr" rid="pgen.1007232.ref093">93</xref>], and digested with <italic>Sph I</italic> endonuclease. Digested genomic DNA was then fractionated by gel electrophoresis on a 1% GTG agarose gel and subjected to Southern blot analysis with a <sup>32</sup>P-labeled riboprobe specific for <italic>POL</italic> as described previously [<xref ref-type="bibr" rid="pgen.1007232.ref015">15</xref>, <xref ref-type="bibr" rid="pgen.1007232.ref037">37</xref>].</p><p>Polyadenylated RNA was used to synthesize cDNA using the First Strand cDNA Synthesis Kit (Affymetrix). 100ng of polyA<sup>+</sup> mRNA was used per reaction, as were 0.5&#956;M concentration each of primers specific to the &#916;<italic>POL</italic> region of pGTy1&#916;POL (5&#8242;-CCACCCATAATGTAATAGATCTATCGATTCTAGAC-3&#8242;) and to <italic>ACT1</italic> (5&#8242;-ATCGTCCCAGTTGGTGACAATACC-3&#8242;). Reactions were performed according to manufacturer protocols, and run at 44&#176;C for 1 hour, followed by incubation at 92&#176;C for 10 minutes. For comparison of levels of full-length Ty1 and Ty1i transcripts generated from pGTy1&#916;POL (<xref ref-type="fig" rid="pgen.1007232.g006">Fig 6</xref>), cDNA was PCR amplified using primers binding 250 bp upstream of the Ty1i TSS (5&#8242;-GATTCATCCTCAGCGGACTCTG-3&#8242;), or 250 bp downstream of Ty1i TSS (5&#8242;-AGAAGAATGATTCTCGCAGC-3&#8242;), and the &#916;<italic>POL</italic> region of the element (5&#8242;-CCACCCATAATGTAATAGATCTATCGATTCTAGAC-3&#8242;). PCR product was isolated at different cycle numbers and compared with amplification of <italic>ACT1</italic> (forward primer: 5&#8242;-GGTTCTGGTATGTGTAAAGCCGGT-3&#8242;; reverse primer: 5&#8242;-ATCGTCCCAGTTGGTGACAATACC-3&#8242;) to control for relative cDNA abundance. A control set of PCR amplifications was performed using genomic DNA prepared from the same strain.</p><p>Expression of pGTy1&#916;POL relative to expression of <italic>ACT1</italic> was estimated as follows: Amplification of <italic>ACT1</italic> cDNA was observed at 5&#8211;6 fewer cycles than the average amplification for Ty1 and Ty1i, whereas amplification of gDNA was about equal to this average (<xref ref-type="fig" rid="pgen.1007232.g006">Fig 6B</xref>), thus indicating conservatively a 32-fold greater amount of <italic>ACT1</italic> transcript. The amplicon for <italic>ACT1</italic> is about 200 bp, while the amplicons for Ty1&#916;POL and Ty1i&#916;POL are each about 3 kb, and so an equal intensity for <italic>ACT1</italic> and Ty1&#916;POL corresponds to about a 15-fold greater molar quantity of <italic>ACT1</italic>. Combining these ratios indicates that <italic>ACT1</italic> transcript levels are approximately 450 times greater than those for Ty1&#916;POL.</p><p>For analysis of Mediator occupancy at Ty1 elements (<xref ref-type="fig" rid="pgen.1007232.g008">Fig 8</xref>), chromatin immunoprecipitation followed by high throughput sequencing (ChIP-seq) was performed using strains in which Mediator subunits Med15 or Med17 carried 13-myc epitope tags and which were engineered to allow Kin28 inactivation by the anchor away technique (<xref ref-type="supplementary-material" rid="pgen.1007232.s007">S1 Table</xref>) [<xref ref-type="bibr" rid="pgen.1007232.ref063">63</xref>]. For anchor away experiments, yeast were grown in YPD to an OD<sub>600</sub> of 0.8. Rapamycin was then added to a concentration of 1 &#956;g/mL (from a 1 mg/mL stock in ethanol stored at -20&#176;C for not more than one month) and cultures allowed to grow one hr at 30&#176;C prior to cross-linking. ChIP against epitope-tagged Mediator subunits was carried out as described previously [<xref ref-type="bibr" rid="pgen.1007232.ref061">61</xref>], using 2 &#956;g of anti-myc antibody (clone 9E10, Sigma) and protein G sepharose beads for capture (GE Healthcare). Library preparation for Illumina sequencing was performed using the NEBNext Ultra II library preparation kit (New England Biolabs) according to manufacturer&#8217;s directions. Libraries were bar-coded using NEXTflex barcodes (BIOO Scientific) and sequenced on the Illumina NextSeq platform at the Wadsworth Center, New York State Department of Health.</p><p>Unfiltered paired-end sequencing reads were aligned to the <italic>S</italic>. <italic>cerevisiae</italic> reference genome (Saccer3) by using BWA [<xref ref-type="bibr" rid="pgen.1007232.ref094">94</xref>]. Up to one mismatch was allowed for each aligned read; reads mapping to multiple locations were retained and randomly assigned. Because full-length Ty1 elements share sequences with 266 Ty1 delta elements (263 of which are &lt;350 bp in length), some reads from the first ~350 bp of Ty1 elements will also be mapped to these elements. Duplicate reads were removed based on paired end information. Occupancy profiles for Ty1 elements were generated by averaging the signal of all 31 Ty1 elements; thus, behavior of individual elements cannot be assessed. The occupancy is plotted on the window from 2kb upstream of TSS to 2kb downstream of TES (Figs <xref ref-type="fig" rid="pgen.1007232.g005">5</xref> and <xref ref-type="fig" rid="pgen.1007232.g006">6</xref>). In each Ty1 element ORF, from TSS+1kb to TES-1kb, the region is divided into 100 bins and the average occupancy of each bin was calculated. For the flanking regions, the occupancy was calculated for each base pair.</p><p>ChIP-seq data has been deposited in Arrayexpress under accession number E-MTAB-5824. Data used in <xref ref-type="fig" rid="pgen.1007232.g004">Fig 4B</xref> is available from the NCBI Sequence Read Archive under accession number SRP047524.</p>