<p>The performance of PRS is typically improved if genetic markers are pruned for LD. LD-pruning procedures that ignore GWAS <italic>P</italic>-values frequently prune out the most significant SNPs and may reduce performance. Instead, we use the LD-clumping procedure implemented in PLINK [<xref ref-type="bibr" rid="pgen.1006493.ref020">20</xref>] that chooses the most significant SNP from a set of SNPs in LD guided by GWAS <italic>P</italic>-values. After LD-clumping, no SNPs with physical distance less than 500kb have LD <italic>r</italic><sup>2</sup> &#8805; 0.1.</p><p>Suppose <italic>S</italic><sub><italic>1</italic></sub> is a given HP set defined based on external annotation data (see section <italic>Annotation datasets</italic>). Any SNP in high LD with a SNP in <italic>S</italic><sub><italic>1</italic></sub> is also considered to be an HP SNP. Thus, we expanded <italic>S</italic><sub><italic>1</italic></sub> by including all SNPs that were in high LD (<italic>r</italic><sup>2</sup> &#8805; 0.8) with any SNP in the original <italic>S</italic><sub><italic>1</italic></sub>.</p><p>We simulated quantitative traits with specific genetic architecture by conditioning on the genotypes of a lung cancer GWAS [<xref ref-type="bibr" rid="pgen.1006493.ref027">27</xref>], including 11,924 samples of European ancestry and 485,315 autosomal SNPs after quality control. The simulation scheme is summarized in the following steps:</p><p>We performed LD-pruning implemented in PLINK so that no SNPs within 500kb were in LD at threshold <italic>r</italic><sup>2</sup> = 0.1. After LD-pruning, <italic>M</italic> = 53,163 autosomal SNPs (denoted as <italic>S</italic>) were left.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Denote <italic>S</italic><sub>1</sub> as the putative HP SNP set and <italic>S</italic><sub>2</sub> = <italic>S</italic> \ <italic>S</italic><sub>1</sub> as the LP SNP set. We selected a set of 5000 &#8220;causal&#8221; SNPs (denoted as <italic>C</italic>) from the pruned SNP set <italic>S</italic>. If <italic>C</italic> is randomly selected, <italic>i</italic>.<italic>e</italic>., <italic>S</italic><sub>1</sub> is not enriched with causal SNPs, we expect | <italic>S</italic><sub>1</sub> &#8745; <italic>C</italic> | = | <italic>C</italic> || <italic>S</italic><sub>1</sub>| / <italic>M</italic> SNPs overlapping between <italic>S</italic><sub>1</sub> and <italic>C</italic>. Thus, we defined the enrichment fold change for <italic>S</italic><sub>1</sub> as
<disp-formula id="pgen.1006493.e028"><alternatives><graphic id="pgen.1006493.e028g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e028.jpg" /><ns1:math id="M28"><ns1:mrow><ns1:mo>&#916;</ns1:mo><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>S</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>&#8745;</ns1:mo><ns1:mi>C</ns1:mi><ns1:mo>|</ns1:mo></ns1:mrow><ns1:mrow><ns1:mo>|</ns1:mo><ns1:mi>C</ns1:mi><ns1:mo>|</ns1:mo><ns1:mo>|</ns1:mo><ns1:msub><ns1:mi>S</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>|</ns1:mo><ns1:mo>/</ns1:mo><ns1:mi>M</ns1:mi></ns1:mrow></ns1:mfrac><ns1:mtext>&#160;</ns1:mtext><ns1:mo>.</ns1:mo></ns1:mrow></ns1:math></alternatives></disp-formula>
</p><p>The enrichment fold change &#916; ranged from 2 to 4 in simulations.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We simulated quantitative traits according to <italic>y</italic><sub><italic>i</italic></sub> = &#931;<sub><italic>t</italic>&#8712;<italic>C</italic></sub>
<italic>&#946;</italic><sub><italic>t</italic></sub><italic>g</italic><sub><italic>it</italic></sub> + <italic>&#949;</italic><sub><italic>i</italic></sub>, where <italic>&#946;</italic><sub><italic>t</italic></sub><italic>s</italic> were simulated independently from a Gaussian mixture distribution <inline-formula id="pgen.1006493.e029"><alternatives><graphic id="pgen.1006493.e029g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e029.jpg" /><ns1:math id="M29"><ns1:mrow><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mi>t</ns1:mi></ns1:msub><ns1:mo>~</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mi>N</ns1:mi><ns1:mo stretchy="false">(</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mn>1</ns1:mn><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo stretchy="false">)</ns1:mo><ns1:mo>+</ns1:mo><ns1:mo stretchy="false">(</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:mi>N</ns1:mi><ns1:mo stretchy="false">(</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mn>2</ns1:mn><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo stretchy="false">)</ns1:mo></ns1:mrow></ns1:math></alternatives></inline-formula> with <italic>&#960;</italic> = 0.1 Here, <inline-formula id="pgen.1006493.e030"><alternatives><graphic id="pgen.1006493.e030g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e030.jpg" /><ns1:math id="M30"><ns1:mrow><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mn>1</ns1:mn><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:math></alternatives></inline-formula>, <inline-formula id="pgen.1006493.e031"><alternatives><graphic id="pgen.1006493.e031g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e031.jpg" /><ns1:math id="M31"><ns1:mrow><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mn>2</ns1:mn><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:math></alternatives></inline-formula> and <italic>Var</italic>(<italic>&#949;</italic><sub><italic>i</italic></sub>) were scaled so that <italic>Var</italic>(<italic>y</italic><sub><italic>i</italic></sub>) = 1. The phenotypic variances explained by the two components were <inline-formula id="pgen.1006493.e032"><alternatives><graphic id="pgen.1006493.e032g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e032.jpg" /><ns1:math id="M32"><ns1:mrow><ns1:msubsup><ns1:mi>h</ns1:mi><ns1:mn>1</ns1:mn><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>=</ns1:mo><ns1:mo>|</ns1:mo><ns1:mi>C</ns1:mi><ns1:mo>|</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mn>1</ns1:mn><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>=</ns1:mo><ns1:mn>0.1</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula> and <inline-formula id="pgen.1006493.e033"><alternatives><graphic id="pgen.1006493.e033g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e033.jpg" /><ns1:math id="M33"><ns1:mrow><ns1:msubsup><ns1:mi>h</ns1:mi><ns1:mn>2</ns1:mn><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>=</ns1:mo><ns1:mo>|</ns1:mo><ns1:mi>C</ns1:mi><ns1:mo>|</ns1:mo><ns1:mo stretchy="false">(</ns1:mo><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mo stretchy="false">)</ns1:mo><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mn>2</ns1:mn><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>=</ns1:mo><ns1:mn>0.4</ns1:mn></ns1:mrow></ns1:math></alternatives></inline-formula>. We assume the same effect-size distribution for both HP and LP causal SNPs, but the proportions of causal SNPs are higher in the former than the later group. Under this assumption, &#916; also reflects the ratio of heritability explained at a per SNP basis in the HP set compared to LP set.</p><p>We randomly selected 10,000 samples as a discovery set and 1,924 as a validation set. We performed GWAS association analysis for all 485,315 autosomal SNPs in the discovery sample. The summary statistics were used to calculate PRS for each sample in the validation sample. The prediction <italic>R</italic><sup>2</sup> was calculated as max<sub><italic>&#955;</italic></sub>
<italic>cor</italic><sup>2</sup>(<italic>PRS</italic><sub><italic>i</italic></sub>(<italic>&#955;</italic>), <italic>y</italic><sub><italic>i</italic></sub>) for 1D PRS methods and max<sub><italic>&#955;</italic><sub>1</sub></sub>,<sub><italic>&#955;</italic><sub>2</sub></sub>
<italic>cor</italic><sup>2</sup>(<italic>PRS</italic><sub><italic>i</italic></sub>(<italic>&#955;</italic><sub>1</sub>, <italic>&#955;</italic><sub>2</sub>), <italic>y</italic><sub><italic>i</italic></sub>) for 2D PRS methods. We repeated the simulation 50 times for each set of parameters and report the average prediction <italic>R</italic><sup>2</sup>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Recently, Finucane et al. [<xref ref-type="bibr" rid="pgen.1006493.ref023">23</xref>] reported the heritability explained by common SNPs in multiple functional categories for 17 traits. Interestingly, they found that common SNPs located in regions that are conserved in mammals [<xref ref-type="bibr" rid="pgen.1006493.ref028">28</xref>] accounted for about 2.6% of total common SNPs but explained approximately 35% of total heritability in average across these traits, suggesting a 13.5-fold enrichment. Thus, we were motivated to investigate whether SNPs related with the conserved regions (CR) may be useful for 2D PRS methods. We downloaded the CR annotations (<ext-link ext-link-type="uri" ns0:href="http://compbio.mit.edu/human-constraint/data/gff/">http://compbio.mit.edu/human-constraint/data/gff/</ext-link>), identified common SNPs located in any CR and also identified their LD SNPs with <italic>r</italic><sup>2</sup> &#8805; 0.8. These SNPs are referred to as CR-SNPs, which were used as HP <italic>S</italic><sub>1</sub> in simulations. We found 9,940 CR-SNPs overlapping with the 53,163 LD-pruned SNPs. To investigate whether specific genomic locations of CR-SNPs influence the performance of 2D-PRS, we also performed simulations using a set <italic>S</italic><sub>1</sub> of random SNPs that has the same size and associated heritability as the CR-SNPs.</p><p>The Wellcome Trust Case Control Consortium [<xref ref-type="bibr" rid="pgen.1006493.ref030">30</xref>] (WTCCC) data consisted of two control data sets (1958 Cohort samples and NBS control samples) and seven diseases: bipolar disorder (BD), coronary artery disease (CAD), Crohn&#8217;s disease (CD), hypertension (HT), rheumatoid arthritis (RA), Type 1 diabetes (T1D) and Type 2 diabetes (T2D). Since we analyzed T2D using a much larger discovery sample, we did not analyze the T2D data in WTCCC. Because cases and controls were genotyped in different batches, differential errors between cases and controls might cause a serious overestimate of the risk prediction. Thus, we performed very rigorous quality control (QC) by removing duplicate samples, first or second degree relatives, samples with missing rate greater than 5% and non-European samples identified from EigenStrat [<xref ref-type="bibr" rid="pgen.1006493.ref035">35</xref>] analysis. For each disease, we excluded SNPs with MAF&lt;5%, missing rate &gt;2%, missing rate difference &gt;1% between cases and controls or <italic>P</italic><sub>HWE</sub>&lt;10<sup>&#8722;4</sup> in the control samples. For each PRS method and each disease, we estimated the prediction R<sup>2</sup> by five-fold cross-validation.</p><p>We analyzed three cancer GWAS with individual level genotype data: the bladder cancer [<xref ref-type="bibr" rid="pgen.1006493.ref036">36</xref>, <xref ref-type="bibr" rid="pgen.1006493.ref037">37</xref>] GWAS of European ancestry including 5,937 cases and 10,862 controls, the pancreatic cancer GWAS [<xref ref-type="bibr" rid="pgen.1006493.ref038">38</xref>] of European ancestry (after excluding samples with Asian or African ancestry) including 5,066 cases and 8,807 controls, and the Asian non-smoking female lung cancer GWAS [<xref ref-type="bibr" rid="pgen.1006493.ref039">39</xref>] with 5,510 cases and 4,544 controls. After QC, the bladder cancer GWAS had 463,559 autosomal SNPs and the Asian lung cancer GWAS had 329,703 autosomal SNPs. The pancreatic cancer GWAS included samples from three studies that used different genotyping platforms. For convenience, we analyzed 267,935 autosomal SNPs that overlapped in all three platforms. The prediction performance was evaluated using ten-fold cross-validation.</p><p>For T2D, we downloaded the summary statistics of the DIAGRAM (DIAbetes Genetics Replication And Meta-analysis) consortium [<xref ref-type="bibr" rid="pgen.1006493.ref040">40</xref>] with 12,171 cases and 56,862 controls for 2.5 million SNPs imputed to the Hapmap2 reference panel. We also downloaded the GERA (Genetic Epidemiology Research on Adult Health and Aging) GWAS data of European ancestry with 7,131 T2D patients and 49,747 samples without T2D (but may have other medical conditions, e.g., 27.4% with cancers, 25.4% with asthma, 25.4% with allergic rhinitis&#8206; and 12.4% with depression). We randomly selected 5,631 T2D patients and 48,247 non-T2D subjects from GERA as discovery set, performed association analysis adjusting for top 10 PCA scores and meta-analyzed with the summary statistics from DIAGRAM for 353,196 autosomal SNPs overlapping between the two studies. The resulting summary statistics were used to build PRS risk models, which were validated in the remaining 1500 T2D patients and 1500 non-T2D subjects in GERA.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The PGC2 (Psychiatric Genetics Consortium) schizophrenia GWAS meta-analysis consisted of 34,241 cases and 45,604 controls [<xref ref-type="bibr" rid="pgen.1006493.ref041">41</xref>] (<ext-link ext-link-type="uri" ns0:href="http://www.med.unc.edu/pgc/downloads">http://www.med.unc.edu/pgc/downloads</ext-link>). Summary statistics were obtained by meta-analyzing all PGC2 schizophrenia GWAS except the MGS [<xref ref-type="bibr" rid="pgen.1006493.ref042">42</xref>] (Molecular Genetics of Schizophrenia) subjects of European ancestry. The summary statistics were used to build PRS models, which were validated in MGS samples with 2,681 cases and 2653 controls.</p><p>The TRICL (Transdisciplinary Research in Cancer of the Lung) GWAS consortium consisted of 12,537 lung cancer cases and 17,285 controls [<xref ref-type="bibr" rid="pgen.1006493.ref043">43</xref>, <xref ref-type="bibr" rid="pgen.1006493.ref044">44</xref>]. We performed meta-analysis using TRICL samples excluding the samples from the PLCO [<xref ref-type="bibr" rid="pgen.1006493.ref027">27</xref>] (Prostate, Lung, Colon, and Ovary Cohort Study) study. The summary statistics based on 11,300 cases and 15,952 controls were used to build risk models, which were validated in the PLCO lung GWAS samples with 1,237 cases and 1,333 controls.</p><p>For colorectal cancer, we performed meta-analysis for the GECCO (Genetics and Epidemiology of Colorectal Cancer Consortium) [<xref ref-type="bibr" rid="pgen.1006493.ref045">45</xref>] GWAS data after excluding the PLCO GWAS data. The PLCO samples were genotyped using two different genotyping platforms with different marker densities: one had approximately 500K SNPs and the other had only 250K SNPs. Thus, we first imputed the genotypes to the Hapmap2 reference panel using IMPUTE2 [<xref ref-type="bibr" rid="pgen.1006493.ref046">46</xref>] and selected SNPs with imputation <italic>r</italic><sup>2</sup> &#8805; 0.9 for risk prediction. The discovery sample consisted of 9,719 cases and 10,937 controls from 19 studies. The PLCO validation sample had 1,000 cases and 2,302 controls.</p><p>The summary statistics for prostate cancer were obtained from the PRACTICAL (PRostate cancer AssoCiation group To Investigate Cancer Associated aLterations) consortium and The GAME-ON/ELLIPSE (Elucidating Loci Involved in Prostate Cancer Susceptibility) Consortium with samples from populations of European, African, Japanese and Latino ancestry [<xref ref-type="bibr" rid="pgen.1006493.ref005">5</xref>]. The discovery samples consisted of 38,703 cases and 40,796 controls after excluding the NCI Pegsus GWAS samples with 4,600 cases and 2,941 controls, which were used for validation. We analyzed 536,057 autosomal SNPs after QC that overlapped between the validation and the discovery sample summary statistics.</p><p>For many traits, GWAS risk SNPs have been reported to show enrichment for eQTLs, methylation QTLs (meQTLs) and <italic>cis</italic>-regulatory elements (CREs). In addition, recent studies have reported extensive genetic pleiotropy across diseases and traits, e.g. psychiatric diseases [<xref ref-type="bibr" rid="pgen.1006493.ref047">47</xref>, <xref ref-type="bibr" rid="pgen.1006493.ref048">48</xref>], schizophrenia and cardiovascular-disease risk factors, including blood pressure, triglycerides, low- and high-density lipoprotein, body mass index (BMI) and waist-to-hip ratio (WHR) [<xref ref-type="bibr" rid="pgen.1006493.ref049">49</xref>]. This information may potentially improve risk prediction if the SNPs identified from the secondary trait are highly enriched in the GWAS of the primary trait. Thus, we defined the HP SNP set <italic>S</italic><sub>1</sub> using eQTL SNPs (referred to as eSNPs) in blood, tissue specific eSNPs and meQTL SNPs (referred to as meSNPs), SNPs related with CREs (referred to as CRE-SNPs), SNPs related with genomic regions conserved across mammals (referred to as CR-SNPs) and SNPs identified by pleiotropic analyses (referred to as PT-SNPs). Here, LD was calculated based on the genotype data of relevant ancestry in The 1000 Genomes Project [<xref ref-type="bibr" rid="pgen.1006493.ref029">29</xref>]. Note that the availability of functional annotation data depends on tissue types. However, for all diseases studied in the paper, we have used blood eSNPs and CR-SNPs because blood eSNPs are enriched for GWAS of all these traits and CR-SNPs were highly enriched in many traits by a heritability partitioning analysis [<xref ref-type="bibr" rid="pgen.1006493.ref023">23</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Blood <italic>cis</italic>-eSNPs were identified from two large-scale eQTL studies in European populations. One study involved a transcriptome sequencing project of 922 subjects [<xref ref-type="bibr" rid="pgen.1006493.ref050">50</xref>] and the other involved a microarray study of 5,311 subjects [<xref ref-type="bibr" rid="pgen.1006493.ref051">51</xref>] (<ext-link ext-link-type="uri" ns0:href="http://genenetwork.nl/bloodeqtlbrowser/">http://genenetwork.nl/bloodeqtlbrowser/</ext-link>). Because of its very large sample size, the second study had the power to detect eSNPs with even tiny effect sizes which may not have meaningful functional importance. Thus, we included eSNPs with association <italic>P</italic>-value &lt;10<sup>&#8722;6</sup> with any gene in the <italic>cis</italic> region in the second study. For both Asian and European lung cancer GWAS data, we used eSNPs [<xref ref-type="bibr" rid="pgen.1006493.ref052">52</xref>] and meSNPs [<xref ref-type="bibr" rid="pgen.1006493.ref053">53</xref>] based on lung tissues. For T2D, we used eSNPs [<xref ref-type="bibr" rid="pgen.1006493.ref054">54</xref>] and meSNPs [<xref ref-type="bibr" rid="pgen.1006493.ref055">55</xref>] based on adipose tissues (<ext-link ext-link-type="uri" ns0:href="http://www.muther.ac.uk/Data.html">http://www.muther.ac.uk/Data.html</ext-link>). Furthermore, detected <italic>trans</italic>-SNPs are much fewer than <italic>cis</italic>-SNPs and the replication rate of <italic>trans</italic>-eSNPs was much lower than <italic>cis</italic>-SNPs [<xref ref-type="bibr" rid="pgen.1006493.ref054">54</xref>], suggesting that including <italic>trans</italic>-SNPs would be unlikely to improve risk prediction. Thus, we did not include <italic>trans</italic>-SNPs.</p><p>CREs are regions of noncoding DNA regulating the transcription of nearby genes. SNPs located in CREs may change the binding of specific transcription factors and thus the expression of the target genes. Typically, CREs are identified through ChIP-Seq experiments of histone modifications. We downloaded &#8220;peak&#8221; data (each peak represents one CRE) of specific sets of histone methylation markings, acetylation markings and DNase I hypersensitive sites (DHSs) from the ROADMAP project website for relevant cell lines. For each identified CRE (&#8216;peak&#8217;), we identified common SNPs with MAF&gt;1%. For prostate cancer, we used the ChIP-Seq data for H3K27Ac and the transcription factor TCF7L2 [<xref ref-type="bibr" rid="pgen.1006493.ref056">56</xref>] to define HP SNP sets.</p><p>The summary statistics for height [<xref ref-type="bibr" rid="pgen.1006493.ref001">1</xref>, <xref ref-type="bibr" rid="pgen.1006493.ref002">2</xref>], BMI and obesity [<xref ref-type="bibr" rid="pgen.1006493.ref003">3</xref>, <xref ref-type="bibr" rid="pgen.1006493.ref057">57</xref>], WHR [<xref ref-type="bibr" rid="pgen.1006493.ref058">58</xref>], waist circumference (WC) [<xref ref-type="bibr" rid="pgen.1006493.ref058">58</xref>], hip circumference (HIP) [<xref ref-type="bibr" rid="pgen.1006493.ref058">58</xref>] were downloaded from the GIANT consortium website. The summary statistics for GWAS meta-analysis of cardiovascular-disease risk factors [<xref ref-type="bibr" rid="pgen.1006493.ref059">59</xref>], including triglycerides (TG), low-density lipoprotein (LDL) and high-density lipoprotein (HDL), were also used for 2D PRS.</p><p>We investigated whether or not each tentative HP SNP set was enriched for GWAS associations by examining the quantile-quantile (QQ) plot, which was made for HP SNPs vs. LP SNPs after LD-clumping. The SNP sets not enriched for GWAS associations were not expected to improve risk prediction in 2D PRS. Thus, for each disease, we only included HP SNP sets for 2D PRS when they showed strong enrichment in QQ plots. Interestingly, blood eSNPs were enriched for almost all diseases. CR-SNPs showed modest enrichment for majority of the diseases. Thus, blood eSNPs and CR-SNPs were used for 2D PRS for all diseases. In addition, eSNPs and meSNPs derived in lung tissues were enriched in lung cancer GWAS of both European and Asian ancestry. The SNPs related in enhancer and active promoter regions (characterized by H3K4me3, H3K9-14Ac, H3K36me3, H3K4me1, H3K9ac and H3K9me3) were enriched for GWAS associations but SNPs related with the repressive regions (characterized by H3K27me3) were not. Thus, we included SNPs related with these enhancer and active promoter regions for 2D PRS. DHS SNPs were not strongly enriched and thus were excluded. Recently, we have shown significantly shared genetic component between lung cancer and bladder cancer risk [<xref ref-type="bibr" rid="pgen.1006493.ref060">60</xref>]. Thus, we also used HP SNPs derived based on lung tissues or cell lines for predicting bladder cancer risk. Furthermore, we found that SNPs identified through pleiotropic analysis were enriched in multiple diseases. For example, SNPs with <italic>P</italic>-value &lt;0.001 in GWAS of height, HDL, LDL, TC, TG, WC, obesity, HIP and T2D were enriched in lung cancer GWAS. Because our 2D PRS methods required a relatively large number of HP SNPs to achieve improvement, we combined the SNPs with <italic>P</italic>-value &lt;10<sup>&#8722;3</sup> (or 10<sup>&#8722;2</sup>) in at least one trait into a HP SNP set referred as PT-0.001 (or PT-0.01).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">For WTCCC and three cancer GWAS data sets with individual genotype data, we used <italic>K</italic>-fold cross-validation to estimate prediction R<sup>2</sup>. Here, <italic>K</italic> = 5 for WTCCC data and <italic>K</italic> = 10 for cancer GWAS data. We were interested in testing whether the prediction of a new PRS method was significantly better than that of the standard 1D PRS defined in <xref ref-type="disp-formula" rid="pgen.1006493.e004">Eq (1)</xref>. For the <italic>i</italic><sup><italic>th</italic></sup> cross-validation, we denote <inline-formula id="pgen.1006493.e034"><alternatives><graphic id="pgen.1006493.e034g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e034.jpg" /><ns1:math id="M34"><ns1:mrow><ns1:msubsup><ns1:mi>R</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>,</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:math></alternatives></inline-formula> as the maximum prediction for the standard 1D PRS optimized across <italic>P</italic>-value thresholds, <inline-formula id="pgen.1006493.e035"><alternatives><graphic id="pgen.1006493.e035g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e035.jpg" /><ns1:math id="M35"><ns1:mrow><ns1:msubsup><ns1:mi>R</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:math></alternatives></inline-formula> as the maximum prediction for a new PRS method optimized across all <italic>P</italic>-value thresholds for 1D PRS and all pairs of <italic>P</italic>-value thresholds for 2D PRS. We defined <inline-formula id="pgen.1006493.e036"><alternatives><graphic id="pgen.1006493.e036g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e036.jpg" /><ns1:math id="M36"><ns1:mrow><ns1:msub><ns1:mi>&#948;</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>=</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:msubsup><ns1:mi>R</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>,</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>&#160;</ns1:mo><ns1:mo>&#8722;</ns1:mo><ns1:msubsup><ns1:mi>R</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>,</ns1:mo><ns1:mn>0</ns1:mn></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:math></alternatives></inline-formula> and estimated its variance as <inline-formula id="pgen.1006493.e037"><alternatives><graphic id="pgen.1006493.e037g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e037.jpg" /><ns1:math id="M37"><ns1:mrow><ns1:msup><ns1:mover accent="true"><ns1:mi>&#963;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mn>2</ns1:mn></ns1:msup><ns1:mo>=</ns1:mo><ns1:msubsup><ns1:mi>&#931;</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>K</ns1:mi></ns1:msubsup><ns1:msup><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#948;</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:mover accent="true"><ns1:mi>&#948;</ns1:mi><ns1:mo>&#175;</ns1:mo></ns1:mover></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msup><ns1:mo>/</ns1:mo><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>K</ns1:mi><ns1:mo>&#8722;</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula> with <inline-formula id="pgen.1006493.e038"><alternatives><graphic id="pgen.1006493.e038g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e038.jpg" /><ns1:math id="M38"><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#948;</ns1:mi><ns1:mo>&#175;</ns1:mo></ns1:mover><ns1:mo>=</ns1:mo><ns1:msubsup><ns1:mi>&#931;</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>K</ns1:mi></ns1:msubsup><ns1:msub><ns1:mi>&#948;</ns1:mi><ns1:mi>i</ns1:mi></ns1:msub><ns1:mo>/</ns1:mo><ns1:mi>K</ns1:mi></ns1:mrow></ns1:math></alternatives></inline-formula>. We calculated the statistic <inline-formula id="pgen.1006493.e039"><alternatives><graphic id="pgen.1006493.e039g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e039.jpg" /><ns1:math id="M39"><ns1:mrow><ns1:mi>T</ns1:mi><ns1:mo>=</ns1:mo><ns1:mover accent="true"><ns1:mi>&#948;</ns1:mi><ns1:mo>&#175;</ns1:mo></ns1:mover><ns1:mo>/</ns1:mo><ns1:msqrt><ns1:mrow><ns1:msup><ns1:mover accent="true"><ns1:mi>&#963;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mn>2</ns1:mn></ns1:msup><ns1:mo>/</ns1:mo><ns1:mi>K</ns1:mi></ns1:mrow></ns1:msqrt></ns1:mrow></ns1:math></alternatives></inline-formula> and evaluated its significance using the <italic>t</italic>-distribution. For the five diseases with independent validation samples, we used bootstrap to estimate the variance of the R<sup>2</sup> estimates to test significance [<xref ref-type="bibr" rid="pgen.1006493.ref029">29</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Suppose that for a given trait of interest <italic>Y</italic>, there are two predefined SNP sets: the high priority (HP) SNP set <italic>S</italic><sub>1</sub> and the low priority (LP) SNP set <italic>S</italic><sub>2</sub>. SNPs have been pruned and are in linkage equilibrium. We assume that <italic>S</italic><sub>1</sub> has <italic>M</italic><sub>1</sub> independent susceptibility SNPs and <italic>M</italic><sub>3</sub> null SNPs while <italic>S</italic><sub>2</sub> has <italic>M</italic><sub>2</sub> susceptibility SNPs and <italic>M</italic><sub>4</sub> independent null SNPs. Following Chatterjee et al. [<xref ref-type="bibr" rid="pgen.1006493.ref011">11</xref>], we assume that the true relationship between outcome <italic>Y</italic> and independent susceptibility SNPs is modeled as follows:
<disp-formula id="pgen.1006493.e040"><alternatives><graphic id="pgen.1006493.e040g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e040.jpg" /><ns1:math id="M40"><ns1:mrow><ns1:mi>Y</ns1:mi><ns1:mo>=</ns1:mo><ns1:munderover><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:munderover><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>g</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:munderover><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:munderover><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>g</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:munderover><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>k</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>3</ns1:mn></ns1:msub></ns1:mrow></ns1:munderover><ns1:mn>0</ns1:mn><ns1:mo>&#8901;</ns1:mo><ns1:msub><ns1:mi>g</ns1:mi><ns1:mrow><ns1:mn>3</ns1:mn><ns1:mi>k</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:munderover><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>l</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>4</ns1:mn></ns1:msub></ns1:mrow></ns1:munderover><ns1:mn>0</ns1:mn><ns1:mo>&#8901;</ns1:mo><ns1:msub><ns1:mi>g</ns1:mi><ns1:mrow><ns1:mn>4</ns1:mn><ns1:mi>l</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:mi>&#1013;</ns1:mi><ns1:mo>,</ns1:mo></ns1:mrow></ns1:math></alternatives></disp-formula>
where all <italic>Y</italic> and the genotypic values <italic>g</italic>&#8217;s are standardized so that <italic>E</italic>(<italic>Y</italic>) = 0, <italic>Var</italic>(<italic>Y</italic>) = 1, <italic>E</italic>(<italic>g</italic>) = 0 and <italic>Var</italic>(<italic>g</italic>) = 1, and the error term <italic>&#1013;</italic> ~ <italic>N</italic>(0, <italic>&#963;</italic><sup>2</sup>) and is independent of the genotypic values.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">From a discovery GWAS data set of size <italic>N</italic>, we have regression coefficient <inline-formula id="pgen.1006493.e041"><alternatives><graphic id="pgen.1006493.e041g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e041.jpg" /><ns1:math id="M41"><ns1:mrow><ns1:msub><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mi>i</ns1:mi></ns1:msub></ns1:mrow></ns1:math></alternatives></inline-formula>- and two-sided p-value <italic>P</italic><sub><italic>i</italic></sub> for each SNP. We build an additive prediction model by including SNPs in <italic>S</italic><sub>1</sub> with <italic>P</italic>-value &#8804; <italic>&#945;</italic><sub>1</sub> and SNPs in <italic>S</italic><sub>2</sub> with <italic>P</italic>-value &#8804; <italic>&#945;</italic><sub>2</sub>:
<disp-formula id="pgen.1006493.e042"><alternatives><graphic id="pgen.1006493.e042g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e042.jpg" /><ns1:math id="M42"><ns1:mrow><ns1:mover accent="true"><ns1:mi>Y</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mo>&#945;</ns1:mo><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mo>&#945;</ns1:mo><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:munderover><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:munderover><ns1:msub><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>g</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:munderover><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:munderover><ns1:msub><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>g</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:munderover><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>k</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>3</ns1:mn></ns1:msub></ns1:mrow></ns1:munderover><ns1:msub><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mn>3</ns1:mn><ns1:mi>k</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mn>3</ns1:mn><ns1:mi>k</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>g</ns1:mi><ns1:mrow><ns1:mn>3</ns1:mn><ns1:mi>k</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:munderover><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>l</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>4</ns1:mn></ns1:msub></ns1:mrow></ns1:munderover><ns1:msub><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mn>4</ns1:mn><ns1:mi>l</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mn>4</ns1:mn><ns1:mi>l</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>g</ns1:mi><ns1:mrow><ns1:mn>4</ns1:mn><ns1:mi>l</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo></ns1:mrow></ns1:math></alternatives></disp-formula>
where <italic>&#947;</italic> (<italic>&#945;</italic>) = <italic>I</italic> (<italic>P</italic> &#8804; <italic>&#945;</italic>) with <italic>I</italic> (&#8901;) being an indicator function.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The predictive correlation coefficient (PCC) for the predictive model can be expressed as
<disp-formula id="pgen.1006493.e043"><alternatives><graphic id="pgen.1006493.e043g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e043.jpg" /><ns1:math id="M43"><ns1:mtable columnalign="left"><ns1:mtr><ns1:mtd><ns1:mi>P</ns1:mi><ns1:mi>C</ns1:mi><ns1:mi>C</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mo>&#945;</ns1:mo><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mo>&#945;</ns1:mo><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>r</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>Y</ns1:mi><ns1:mo>,</ns1:mo><ns1:mover accent="true"><ns1:mi>Y</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mo>&#945;</ns1:mo><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mo>&#945;</ns1:mo><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mtd></ns1:mtr><ns1:mtr><ns1:mtd><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msubsup><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:msubsup><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:msubsup><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:msubsup><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mrow><ns1:msqrt><ns1:mrow><ns1:msubsup><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:msubsup><ns1:msubsup><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:msub><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:msubsup><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:msubsup><ns1:msubsup><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:msub><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>k</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>3</ns1:mn></ns1:msub></ns1:mrow></ns1:msubsup><ns1:msubsup><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mn>3</ns1:mn><ns1:mi>k</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:msub><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mn>3</ns1:mn><ns1:mi>k</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>l</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>4</ns1:mn></ns1:msub></ns1:mrow></ns1:msubsup><ns1:msubsup><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mrow><ns1:mn>4</ns1:mn><ns1:mi>l</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:msub><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mn>4</ns1:mn><ns1:mi>l</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:msqrt></ns1:mrow></ns1:mfrac><ns1:mo>.</ns1:mo></ns1:mtd></ns1:mtr></ns1:mtable></ns1:math></alternatives></disp-formula></p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Following Chatterjee et al. (2014), one can verify that PCC follows a normal distribution by the central limit theorem and the strong law of large numbers. Therefore, the expected value of PCC can be approximated as
<disp-formula id="pgen.1006493.e044"><alternatives><graphic id="pgen.1006493.e044g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e044.jpg" /><ns1:math id="M44"><ns1:mtable columnalign="left"><ns1:mtr><ns1:mtd><ns1:mi>E</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>P</ns1:mi><ns1:mi>C</ns1:mi><ns1:mi>C</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mo>&#945;</ns1:mo><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mo>&#945;</ns1:mo><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msubsup><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:msubsup><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>e</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>pow</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:msubsup><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:msubsup><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mi>e</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>pow</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mrow><ns1:msqrt><ns1:mrow><ns1:msubsup><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:msubsup><ns1:msub><ns1:mi>&#957;</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>pow</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:msubsup><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>j</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:msubsup><ns1:msub><ns1:mi>&#957;</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>pow</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mn>2</ns1:mn><ns1:mi>j</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>3</ns1:mn></ns1:msub><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:msub><ns1:mi>&#957;</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>4</ns1:mn></ns1:msub><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub><ns1:msub><ns1:mi>&#957;</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:msqrt></ns1:mrow></ns1:mfrac><ns1:mtext>&#160;</ns1:mtext><ns1:mo>,</ns1:mo></ns1:mtd></ns1:mtr><ns1:mtr><ns1:mtd><ns1:mo>&#8776;</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>&#8747;</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:msub><ns1:mi>e</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>pow</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>f</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mi>d</ns1:mi><ns1:mi>&#946;</ns1:mi><ns1:mo>+</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub><ns1:mo>&#8747;</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:msub><ns1:mi>e</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>pow</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>f</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mi>d</ns1:mi><ns1:mi>&#946;</ns1:mi></ns1:mrow><ns1:mrow><ns1:msqrt><ns1:mrow><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mo>&#8747;</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:msub><ns1:mi>&#957;</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>pow</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>f</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mi>d</ns1:mi><ns1:mi>&#946;</ns1:mi><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub><ns1:mo>&#8747;</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:msub><ns1:mi>&#957;</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>pow</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:msub><ns1:mi>f</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mi>d</ns1:mi><ns1:mi>&#946;</ns1:mi><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>3</ns1:mn></ns1:msub><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub><ns1:msub><ns1:mi>&#957;</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>1</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:msub><ns1:mi>M</ns1:mi><ns1:mn>4</ns1:mn></ns1:msub><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub><ns1:msub><ns1:mi>&#957;</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:msub><ns1:mi>&#945;</ns1:mi><ns1:mn>2</ns1:mn></ns1:msub></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mn>0</ns1:mn><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:msqrt></ns1:mrow></ns1:mfrac></ns1:mtd></ns1:mtr></ns1:mtable></ns1:math></alternatives></disp-formula>
where <inline-formula id="pgen.1006493.e045"><alternatives><graphic id="pgen.1006493.e045g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e045.jpg" /><ns1:math id="M45"><ns1:mrow><ns1:msub><ns1:mi>e</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>&#945;</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mi>E</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mo>|</ns1:mo><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#945;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula>, <inline-formula id="pgen.1006493.e046"><alternatives><graphic id="pgen.1006493.e046g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e046.jpg" /><ns1:math id="M46"><ns1:mrow><ns1:msub><ns1:mi>&#957;</ns1:mi><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>&#945;</ns1:mi></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#946;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mi>E</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msup><ns1:mover accent="true"><ns1:mi>&#946;</ns1:mi><ns1:mo>^</ns1:mo></ns1:mover><ns1:mn>2</ns1:mn></ns1:msup><ns1:mo>|</ns1:mo><ns1:mi>&#947;</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>&#945;</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula>, pow (<italic>N</italic>, <italic>&#946;</italic>, <italic>&#945;</italic>) is power to detect a SNP with effect size <italic>&#946;</italic> at a significance level <italic>&#945;</italic> in a GWAS with size <italic>N</italic>, and <italic>f</italic><sub>1</sub>(&#8901;) and <italic>f</italic><sub>2</sub>(&#8901;) are effect-size distributions for HP and LP susceptibility SNPs, respectively.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">In our numerical calculations, we assumed that the effect sizes of the susceptibility SNPs in the HP and LP sets followed the same distribution <inline-formula id="pgen.1006493.e047"><alternatives><graphic id="pgen.1006493.e047g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006493.e047.jpg" /><ns1:math id="M47"><ns1:mrow><ns1:mi>&#946;</ns1:mi><ns1:mo>~</ns1:mo><ns1:mi>&#960;</ns1:mi><ns1:mi>N</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mn>1</ns1:mn><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mo>+</ns1:mo><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:mi>&#960;</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mi>N</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>0</ns1:mn><ns1:mo>,</ns1:mo><ns1:msubsup><ns1:mi>&#963;</ns1:mi><ns1:mn>2</ns1:mn><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula>, consistent with simulations. We performed grid search to identify the p-value thresholds (<italic>&#945;</italic><sub>1</sub>, <italic>&#945;</italic><sub>2</sub>) that maximizes <italic>E</italic>(<italic>PCC</italic>(<italic>&#945;</italic><sub>1</sub>, <italic>&#945;</italic><sub>2</sub>)). For binary disease outcomes, AUC can be expressed as a function of PCC [<xref ref-type="bibr" rid="pgen.1006493.ref011">11</xref>].</p>