<p>Each study followed protocols for phenotype harmonization. For individuals taking anti-hypertensive medications, we added 15 and 10 mm Hg to measured SBP and DBP, respectively, a standard method used in other BP GWAS.[<xref ref-type="bibr" rid="pgen.1006728.ref006">6</xref>, <xref ref-type="bibr" rid="pgen.1006728.ref068">68</xref>] PP was calculated as the difference between SBP and DBP after addition of the constant values. HTN was defined by a SBP &#8805; 140 mm Hg, a DBP &#8805; 90 mm Hg, or use of antihypertensive drugs.[<xref ref-type="bibr" rid="pgen.1006728.ref069">69</xref>]</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Each cohort was genotyped on either Affymetrix or Illumina genotyping platforms. Pre-imputation quality criteria were applied as described in <xref ref-type="supplementary-material" rid="pgen.1006728.s007">S2 Table</xref>, and included exclusion of individuals with discordant self-reported gender and genetic gender. Imputation was performed using the software MACH-ADMIX, MACH-minimac or IMPUTE2 [<xref ref-type="bibr" rid="pgen.1006728.ref070">70</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006728.ref072">72</xref>] using the Phase 1 integrated (March 2012 release) multi-ethnic reference panel from the 1000G Consortium (<ext-link ext-link-type="uri" ns0:href="http://www.internationalgenome.org/">http://www.internationalgenome.org/</ext-link>).[<xref ref-type="bibr" rid="pgen.1006728.ref073">73</xref>]</p><p>Autosomal chromosome SNP associations for SBP, DBP, and PP were assessed by linear regression for unrelated data or by the generalized linear mixed-effects model for family data, under the assumption of an additive genetic model. All models were adjusted for age, age<sup>2</sup>, sex, and body mass index. Up to ten principal components were included, as needed as covariates in the regression models, to control population stratification.[<xref ref-type="bibr" rid="pgen.1006728.ref074">74</xref>, <xref ref-type="bibr" rid="pgen.1006728.ref075">75</xref>] We used standardized pre-meta-analysis QC criteria for all 21 discovery studies.[<xref ref-type="bibr" rid="pgen.1006728.ref076">76</xref>] At the SNP level, we excluded variants with 1) imputation quality r<sup>2</sup> &lt; 0.3 in MACH or &lt;0.4 in IMPUTE2; 2) the number of informative individuals (2&#215;MAF&#215;N&#215;r<sup>2</sup>) &#8804; 30; 3) an effect allele frequency (EAF) difference larger than 0.3 in comparison with the mixture of 80% YRI and 20% CEU of 1000G; and 4) the absolute regression coefficient &#8805; 10. SNPs that passed the QC were carried forward for inverse variance weighted meta-analyses, implemented in METAL.[<xref ref-type="bibr" rid="pgen.1006728.ref077">77</xref>]</p><p>We applied the CPASSOC software to combine association evidence of SBP, DBP, and HTN. CPASSOC provides two statistics, <italic>S</italic><sub><italic>Hom</italic></sub> and <italic>S</italic><sub><italic>Het</italic>,</sub> as previously described.[<xref ref-type="bibr" rid="pgen.1006728.ref039">39</xref>] <italic>S</italic><sub><italic>Hom</italic></sub> is similar to the fixed effect meta-analysis method[<xref ref-type="bibr" rid="pgen.1006728.ref077">77</xref>] but accounts for the correlation of summary statistics of the multi-traits and for overlapping or related samples among the cohorts. <italic>S</italic><sub><italic>Hom</italic></sub> uses the trait sample size as the weight, so that it is possible to combine traits with different measurement scales. <italic>S</italic><sub><italic>Het</italic></sub> is an extension of <italic>S</italic><sub><italic>Hom</italic>,</sub> and it can increase the statistical power over S<sub><italic>Hom</italic></sub> when a variant affects only a subset of traits. The distribution of <italic>S</italic><sub><italic>Het</italic></sub> under the null hypothesis was obtained through an estimated beta distribution. To calculate the statistics, <italic>S</italic><sub><italic>Hom</italic></sub> and <italic>S</italic><sub><italic>Het</italic></sub>, and to account for the correlation among the traits, a correlation matrix is required. In this study, we used the correlation matrix calculated from the residuals of the three BP traits after adjustments for covariates and principal components.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All independent SNPs identified with P &lt; 10<sup>&#8722;6</sup> (threshold chosen for suggestive association) in the discovery stage were carried forward for replication in African-ancestry individuals and in multi-ethnic samples of European Americans, East Asians, or Hispanics/Latinos (<xref ref-type="fig" rid="pgen.1006728.g001">Fig 1</xref>). For single-trait analyses, we conducted fixed effect meta-analyses in the replication sets for each of four BP traits (SBP, DBP, PP and HTN), followed by a combined trans-ethnic meta-analysis of each trait. This was followed by a mega-meta-analyses, combining the results of discovery and replication for single traits using fixed-effects meta-analysis. We also performed a multi-trait CPASSOC analysis of SBP, DBP, and HTN in each replication study. Because CPASSOC only generated test statistics S<sub>Hom</sub>/S<sub>Het</sub> and corresponding P values without effect sizes, we combined the association P values from all four replication populations using Fisher&#8217;s method (<ext-link ext-link-type="uri" ns0:href="http://hal.case.edu/zhu-web/">http://hal.case.edu/zhu-web/</ext-link>). Finally, we combined the CPASSOC meta-analysis results from the discovery and replication stages using Fisher&#8217;s method.</p><p>For a single trait GWAS discovery analysis, we used genome-wide significant level P = 5.0&#215;10<sup>&#8722;8</sup>. We performed six different analyses, four single trait (SBP, DBP, PP and HTN) analyses and two CPASSOC (S<sub>Hom</sub> and S<sub>Het</sub>) analyses for each SNP. For the four single correlated traits (SBP, DBP, PP and HTN), we calculated the number of independent traits using the eigenvalues of the correlation matrix, [<xref ref-type="bibr" rid="pgen.1006728.ref078">78</xref>] which resulted two independent traits. Therefore, we counted four independent analyses, which were two independent single traits and two statistics of CPASSOC analyses, and applied an experimental significance level P = 1.25&#215;10<sup>&#8722;8</sup> for claiming a genome-wide significance when combining discovery and replication samples. We should point out that the two CPASSOC test statistics and a single trait statistic are not independent. Thus, the significance level P = 1.25&#215;10<sup>&#8722;8</sup> is conservative.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Since a locus may consist of multiple independent signals, we applied approximate conditional analysis implemented in GCTA-COJO[<xref ref-type="bibr" rid="pgen.1006728.ref079">79</xref>, <xref ref-type="bibr" rid="pgen.1006728.ref080">80</xref>] using the summary statistics of SNPs with P &lt; 1.0&#215;10<sup>&#8722;6</sup> from both of the individual trait meta-analyses (<ext-link ext-link-type="uri" ns0:href="http://cnsgenomics.com/software/gcta/cojo.html">http://cnsgenomics.com/software/gcta/cojo.html</ext-link>). The LD among variants was estimated from the five African American cohorts from the CARe consortium.[<xref ref-type="bibr" rid="pgen.1006728.ref079">79</xref>]</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Pathway analysis was performed using the Meta-Analysis Gene-set Enrichment of variant Associations (MAGENTA) program (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/mpg/magenta/">http://www.broadinstitute.org/mpg/magenta/</ext-link>).[<xref ref-type="bibr" rid="pgen.1006728.ref041">41</xref>] Using the summary statistics from the four BP traits and two statistics from CPASSOC, from the discovery stage, we tested whether sets of functionally-related genes are enriched for associations. This method first converts the P values of SNPs into gene scores with correcting for confounders, such as gene site, number of variants in a gene, and their LD patterns, and then calculated a gene set enrichment P value for each biological pathway or gene set of interest using a non-parametric statistical test. The nominal GSEA P value refers to the nominal gene set enrichment P value for a gene set. The database of pathway/gene-sets to be tested include Ingenuity (June 2008), KEGG (2010), GO, and the Panther, signaling pathways downloaded from MSigDB and PANTHER (<ext-link ext-link-type="uri" ns0:href="http://www.broad.mit.edu/gsea/msigdb/collections.jsp">http://www.broad.mit.edu/gsea/msigdb/collections.jsp</ext-link>; <ext-link ext-link-type="uri" ns0:href="http://www.pantherdb.org/">http://www.pantherdb.org/</ext-link>).[<xref ref-type="bibr" rid="pgen.1006728.ref081">81</xref>] We applied the parameters suggested by the authors, which includes the 75<sup>th</sup> percentile cut off of gene scores, the nominal GSEA P-value &lt; 0.01 and the false discovery rate (FDR) &lt; 0.3.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The enrichment of heritability of genomic regions to different functional categories, including cell type-specific elements, was evaluated using the method of LD score regression (<ext-link ext-link-type="uri" ns0:href="https://github.com/bulik/ldsc">https://github.com/bulik/ldsc</ext-link>).[<xref ref-type="bibr" rid="pgen.1006728.ref042">42</xref>, <xref ref-type="bibr" rid="pgen.1006728.ref082">82</xref>] This method partitioned the heritability from the discovery GWAS summary statistics of four BP traits (SBP, DBP, PP, and HTN) while accounting for LD among markers.[<xref ref-type="bibr" rid="pgen.1006728.ref042">42</xref>] We calculated enrichment, in functional regions and in expanded regions (+500bp) around each functional class, based on functional annotation, using a &#8220;full baseline model&#8221; previously created from 24 publicly available main annotations that are not specific to any cell type.[<xref ref-type="bibr" rid="pgen.1006728.ref042">42</xref>] Enrichment was calculated based on the ratio of explained heritability and the proportion of SNPs in each annotation category. The standard error of enrichment was estimated with a block jackknife to calculate z scores and P values.[<xref ref-type="bibr" rid="pgen.1006728.ref042">42</xref>] The multiple testing threshold was determined using the Bonferroni correction while accounting for two independent-trait analyses based on Ji and Li&#8217;s method[<xref ref-type="bibr" rid="pgen.1006728.ref078">78</xref>] (P of 0.05/[25 classes &#215; 2 traits]). We also performed cell-type-specific group enrichment analysis using cell-type-specific annotations from four histone marks (H3K4me1, H3K4me3, H3K9ac, and H3K27ac), which corresponded to 220 cell types. We divided the 220 cell-type-specific annotations into 10 groups: adrenal/pancreas, central nervous system (CNS), cardiovascular, connective/bone, gastrointestinal, immune/hematopoietic, kidney, liver, skeletal muscle and other. The analysis characterized cell-type-specific annotations within each group and calculated the enrichment of heritability for each group.[<xref ref-type="bibr" rid="pgen.1006728.ref042">42</xref>]</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We selected sets of variants in LD r<sup>2</sup> &gt; 0.1 from the eleven replicated variants, and calculated Bayes Factors and posterior causal probabilities for each variant from the effect sizes and standard errors, as previously described.[<xref ref-type="bibr" rid="pgen.1006728.ref083">83</xref>] Each distinct variant associated with multiple traits was included in the analysis only once. The genomic annotations of DHS sites for 348 cell types from the ENCODE project were obtained and grouped into cell types associated with 34 tissues (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu/ENCODE/cellTypes.html">http://genome.ucsc.edu/ENCODE/cellTypes.html</ext-link>). Four gene-based annotations&#8212;coding exon, 5-UTR, 3-UTR, and 1kb upstream of transcription start site (TSS)&#8212;from GENCODE transcripts were also obtained. Variants overlapping each of these annotations were then identified. Using the variant annotations and fGWAS (<ext-link ext-link-type="uri" ns0:href="https://github.com/joepickrell/fgwas">https://github.com/joepickrell/fgwas</ext-link>), we tested for enrichment of variants across all signals in 38 DHS categories, including in the four gene-based annotations in each model.[<xref ref-type="bibr" rid="pgen.1006728.ref043">43</xref>]</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used the GTEx pilot database [<xref ref-type="bibr" rid="pgen.1006728.ref082">82</xref>] (<ext-link ext-link-type="uri" ns0:href="http://www.gtexportal.org/home/">http://www.gtexportal.org/home/</ext-link>) to identify eQTLs in the successfully replicated SNPs.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To evaluate population differentiation and natural selection, using Haplotter,[<xref ref-type="bibr" rid="pgen.1006728.ref040">40</xref>] we calculated the integrated haplotype score (iHS) in five cohorts of CARe so that we could measure the amount of extended haplotype homozygosity (<ext-link ext-link-type="uri" ns0:href="http://coruscant.itmat.upenn.edu/whamm/ihs.html">http://coruscant.itmat.upenn.edu/whamm/ihs.html</ext-link>). Hence, we tested the evidence of recent positive selection at five significant SNPs with differences in allele frequency across continental-ancestry populations. The measures were standardized (mean 0, variance 1) empirically to the distribution of observed iHS scores over a range of SNPs with similar derived allele frequencies. This method assesses the evidence for selection by comparing the extended homozygosity for haplotypes on a high frequency derived allele relative to the ancestry background.[<xref ref-type="bibr" rid="pgen.1006728.ref040">40</xref>]</p><p>Experiments were carried out in accordance with local and the National Institutes of Health guidelines. The animal protocol was approved by the University of Virginia Institutional Animal Care and Use Committee. Wild-type male mice on the 129S6 background at ~ 3 months of age were used for gene expression analyses. All mice were maintained on a 12-hour light-dark cycle with free access to standard chow and water in the animal facility of the University of Virginia.</p><p>The hypertension experimental model was induced using Ang II (Sigma-Aldrich, St. Luis, MO) delivered at 600 ng/kg/min for 2 weeks via Alzet mini-osmotic pumps (Durect Corporation, Cupertino, CA, model 2004), as previously described.[<xref ref-type="bibr" rid="pgen.1006728.ref084">84</xref>] For gene expression analyses, RNA from kidney tissue was isolated by RNeasy Mini kit (Qiagen) and transcribed to cDNA by iScript <sup>TM</sup> cDNA synthesis kit (Bio-Rad). Real time PCR analyses were performed on iQ<sup>TM</sup>5 Multicolor real time PCR Bio-Rad instruments using iQ<sup>TM</sup> SYBER<sup>&#174;</sup> Green Supermix. <italic>Hprt</italic> was used as a reference gene for normalization. Sequences of forward and reversed primers (FP and RP) for the gene expression studies are shown in <xref ref-type="supplementary-material" rid="pgen.1006728.s015">S10 Table</xref>.</p><p>All research involving human participants have been approved by the Institutional Review Board (IRB) # 04-95-72 and study-related Publication and Presentation committees. All participants have provided informed consent for DNA research and data are publicly available in dbGap.</p><p>Animal experiments were carried out following the guidelines established locally at the University of Virginia (UVA) and by the National Institutes of Health. The animal protocol was approved by the UVA Institutional Animal Care and Use Committee (Protocol # 3791, Protocol Title: Genes regulating Hypertension and Kidney Disease). Wild-type male mice on the 129S6 background at ~ 3 months of age were used for gene expression analyses. All mice were maintained on a 12-hour light-dark cycle with free access to standard chow and water in the animal facility UVA.</p>