<p>All plasmid constructs are listed in <xref ref-type="table" rid="pgen.1006287.t006">Table 6</xref> were confirmed by DNA sequencing (Iowa State DNA sequencing facility). All <italic>B</italic>. <italic>subtilis</italic> strains are isogenic derivatives of PY79, a prototrophic derivative of <italic>B</italic>. <italic>subtilis</italic> strain 168 [<xref ref-type="bibr" rid="pgen.1006287.ref076">76</xref>] and are listed in <xref ref-type="table" rid="pgen.1006287.t007">Table 7</xref>. <italic>B</italic>. <italic>subtilis</italic> competent cells were prepared by the one-step method previously described [<xref ref-type="bibr" rid="pgen.1006287.ref077">77</xref>].</p><p><sup>a</sup> This study, unless otherwise noted.</p><p><sup>a</sup> This study, unless otherwise noted.</p><p>Site-directed mutants of <italic>rsiV</italic> were constructed using the QuickChange site-directed mutagenesis kit (Agilent Technologies) and Isothermal Assembly (Gibson <italic>et al</italic>., 2009). The sequences of the oligonucleotide primers used are listed in <xref ref-type="supplementary-material" rid="pgen.1006287.s013">S2 Table</xref>. The IPTG-inducible hyper-spank (P<sub><italic>hs</italic></sub>) promoter was placed upstream of <italic>rsiV</italic> by assembling two <italic>rsiV</italic> PCR products into SphI digested pDR111 [<xref ref-type="bibr" rid="pgen.1006287.ref078">78</xref>]. The two <italic>rsiV</italic> PCR products for the <italic>rsiV</italic><sup><italic>S169W</italic></sup> mutant were PCR amplified with CDEP1859/CDEP3092, and CDEP3093/CDEP1860. Each mutation was generated the same way using the appropriate primer pair with CDEP1850 and CDEP1860: S169W (CDEP3092/CDEP3093), P259A (CDEP3108/3109), Y261A (CDEP3110/CDEP3111), P259A and Y261A (CDEP3098/CDEP3099). S169W P259A Y261A, S169W P259A and S169W, Y261A were made using <italic>rsiVS169W</italic> (pJH375) as template and primers for P259A Y261A, P259A, or Y261A. The resulting plasmids: S169W (pJH375) P259A (pJH436), Y261A (pJH397), P259A Y261A (pJH398), S169W P259A (pJH405), S169W Y261A (pJH439), S169W P259A Y261A (pJH386) were transformed into CDE1563 <italic>sigVrsiV</italic>::<italic>kan</italic>, resulting in P<sub><italic>hs</italic></sub>-<italic>rsiV</italic><sup><italic>S169W</italic></sup>
<italic>sigVrsiV</italic>::<italic>kan</italic> (JLH1271), P<sub><italic>hs</italic></sub>-<italic>rsiV</italic><sup><italic>P259A</italic></sup>
<italic>sigVrsiV</italic>::<italic>kan</italic> (JLH1481), P<sub><italic>hs</italic></sub>-<italic>rsiV</italic><sup><italic>Y261A</italic></sup>
<italic>sigVrsiV</italic>::<italic>kan</italic> (JLH1342), P<sub><italic>hs</italic></sub>-<italic>rsiV</italic><sup><italic>P259A Y261A</italic></sup>
<italic>sigVrsiV</italic>::<italic>kan</italic> (JLH1343), P<sub><italic>hs</italic></sub>-<italic>rsiV</italic><sup><italic>S169W P259A</italic></sup>
<italic>sigVrsiV</italic>::<italic>kan</italic> (JLH1326), P<sub><italic>hs</italic></sub>-<italic>rsiV</italic><sup><italic>S169W Y261A</italic></sup>
<italic>sigVrsiV</italic>::<italic>kan</italic> (JLH1504), and P<sub><italic>hs</italic></sub>-<italic>rsiV</italic><sup><italic>S169W P259A Y261A</italic></sup>
<italic>sigVrsiV</italic>::<italic>kan</italic> (JLH1312).</p><p>The site directed mutants were introduced onto the chromosome of PY79 by homologous recombination using the temperature sensitive plasmid pMAD (Arnaud <italic>et al</italic>., 2004). PCR products for <italic>rsiV</italic><sup><italic>S169W</italic></sup> were amplified with <italic>rsiV</italic> + 1 kb upstream using CDEP1892/CDEP3092 and <italic>rsiV</italic> + 1 kb downstream using CDEP1893/CDEP3093. The resulting PCR products were moved into SmaI digested pMAD using isothermal assembly. Each mutation was generated using the appropriate primer pair with CDEP1892/CDEP1893 or CDEP1892/CDEP3136 (adds 1.5 kb downstream <italic>rsiV</italic> to increase recombination efficiency): S169W (CDEP3092/CDEP3093), P259A (CDEP3108/3109), Y261A (CDEP3110/CDEP3111), P259A and Y261A (CDEP3098/CDEP3099). The triple and double mutants (S169W P259A Y261A, S169W P259A, and S169W, Y261A) were made using <italic>rsiV</italic><sup><italic>S169W</italic></sup> (JLH1413) as template with primers for P259A Y261A, P259A, or Y261A respectively. Each plasmid (S169W (pJH381) P259A (pJH403), Y261A (pJH438), P259A Y261A (pJH404), S169W P259A (pJH405), S169W Y261A (pJH437), and S169W P259A Y261A (pJH406)) was transformed into CDE1546 <italic>P</italic><sub><italic>sigV</italic></sub><italic>-lacZ (cat)</italic> resulting in <italic>rsiV</italic><sup><italic>S169W</italic></sup>
<italic>P</italic><sub><italic>sigV</italic></sub><italic>-lacZ</italic> (JLH1413), <italic>rsiV</italic><sup><italic>P259A</italic></sup>
<italic>P</italic><sub><italic>sigV</italic></sub><italic>-lacZ</italic> (JLH1380), <italic>rsiV</italic><sup><italic>Y261A</italic></sup>
<italic>P</italic><sub><italic>sigV</italic></sub><italic>-lacZ</italic> (JLH1510), <italic>rsiV</italic><sup><italic>P259A Y261A</italic></sup>
<italic>P</italic><sub><italic>sigV</italic></sub><italic>-lacZ</italic> (JLH1372), and <italic>rsiV</italic><sup><italic>S169W Y261A</italic></sup>
<italic>P</italic><sub><italic>sigV</italic></sub><italic>-lacZ</italic> (JLH1507), and <italic>rsiV</italic><sup><italic>S169W P259A Y261A</italic></sup>
<italic>P</italic><sub><italic>sigV</italic></sub><italic>-lacZ</italic> (JLH1412).</p><p>To generate constructs for purification the extracellular domain of RsiV <italic>rsiV</italic><sup><italic>59-285</italic></sup> was PCR amplified using primers CDEP1139 and CDEP952 and the already constructed plasmids as template: S169W (pJH375) P259A (pJH436), Y261A (pJH397), P259A Y261A (pJH398), S169W P259A Y261A (pJH386). This product was PCR amplified again with CDEP1140 and CDEP952 to add <italic>2x-flag</italic> and cloned into pEntrD-TOPO, resulting in plasmids: S169W (pJH421) P259A (pJH426), Y261A (pJH424), P259A Y261A (pJH425), S169W P259A (pJH446), S169W Y261A (pJH448), and S169W P259A Y261A (pJH423). To construct recombinant 6x-His-2x-Flag-RsiV<sup>(59&#8211;285)</sup> + site directed mutant, <italic>2x-flag-rsiV</italic><sup><italic>(59&#8211;285)</italic></sup> was moved from each pEntrD-TOPO vector into the T7-inducible 6x-His destination vector pDEST17 (Invitrogen) using LR Clonase II. The following plasmids were moved into BL21&#955;DE3 cells for expression: S169W (pJH427), P259A (pJH450), Y261A (pJH430), P259A Y261A (pJH431), S169W P259A (pJH447), and S169W P259A Y261A (pJH429).</p><p>Antibiotics were used at the following concentrations: chloramphenicol, 5 &#956;g/ml; erythromycin plus lincomycin, 1 &#956;g/ml and 25 &#956;g/ml; kanamycin, 5 &#956;g/ml; spectinomycin, 100 &#956;g/ml; tetracycline, 10 &#956;g/ml; ampicillin 100 &#956;g/ml. The &#946;-galactosidase chromogenic indicator 5-bromo-4-chloro-3-indolyl &#946;-D-galactopyranoside (X-Gal) was used at a concentration 100 &#956;g/ml. Isopropyl &#946;-D-1-thiogalactopyranoside (IPTG) was used at a final concentration of 1 mM unless otherwise noted.</p><p>Strains were grown for 16 hours in LB at 37&#176;C. The cells were subcultured 1:100 in LB+1 mM IPTG at 37&#176;C and grown to an OD<sub>600</sub> of 0.8&#8211;1. The cells were pelleted by centrifugation and resuspended in 100 &#956;l of 2X Laemmli sample buffer and lysed by repeated sonication. Samples were electrophoresed on a 15% SDS polyacrylamide gel (BioRad). The proteins were then blotted onto nitrocellulose. The nitrocellulose was blocked with 5% milk for 30 minutes. The proteins were detected by incubating with a 1:10,000 dilution of anti-RsiV<sup>59&#8211;285</sup> antibodies (Hastie <italic>et al</italic>., 2013) or 1:15,000 dilution of anti-&#963;<sup>A</sup> antibodies followed by 3 washes and incubation in a 1:10,000 dilution of goat anti-rabbit IgG (H+L) IRDye 800CW (Li-Cor) and imaged on an Odyssey CLx (Li-Cor). Quantification of band intensities was performed using Image Studio software (Li-Cor).</p><p>Cultures were grown overnight in LB broth at 30&#176;C and 20 &#956;l were spotted onto LB agar + 0, 2, 5, 10 or 20 &#956;g/ml lysozyme. Plates were incubated at 37&#176;C for 6 hours. Cells were harvested and resuspended in 500 &#956;l of Z buffer (60 mM Na<sub>2</sub>HPO<sub>4</sub>, 40 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM KCl, 1 mM MgSO<sub>4</sub>, 50 mM &#946;-mercaptoethanol pH 7.0). Cells were transferred to a 96 well plate and optical density (OD<sub>600</sub>) determined. Cells were permeabilized by mixing with chloroform and 2% sarkosyl [<xref ref-type="bibr" rid="pgen.1006287.ref005">5</xref>,<xref ref-type="bibr" rid="pgen.1006287.ref079">79</xref>]. Permeabilized cells (100 &#956;l) were mixed with 10 mg/ml ortho-Nitrophenyl-&#946;-galactoside (ONPG, RPI, 50 &#956;l) and OD<sub>405</sub> was measured over time. &#946;-galactosidase activity units (&#956;mol of ONP formed min<sup>&#8722;1</sup>) X 10<sup>3</sup>/(OD<sub>600</sub> X ml of cell suspension) were calculated as previously described [<xref ref-type="bibr" rid="pgen.1006287.ref080">80</xref>]. Experiments were performed in triplicate with the mean and standard deviation shown.</p><p>For each sample, a single colony of the appropriate <italic>B</italic>. <italic>subtilis</italic> strain was inoculated in LB medium and grown overnight. The overnight cultures were subcultured into LB or LB + 1 mM IPTG and grown to -OD<sub>600</sub> of 0.8 at which point lysozyme when necessary was added to a final concentration of 1.25 &#956;g/ml and grown for 1 hour. RNA was extracted using the RNeasy RNA Isolation kit (Qiagen). Contaminating DNA was removed using the Turbo DNA-free kit protocol (Ambion). Samples were tested for DNA contamination by PCR amplification (Thermo Taq polymerase, NEB) using primers CDEP1017 and CDEP1018 (<xref ref-type="supplementary-material" rid="pgen.1006287.s014">S3 Table</xref>).</p><p>To generate cDNA from RNA samples, we used Superscript II (Invitrogen) according to manufacturer&#8217;s protocols. The resulting reverse transcription reactions were diluted 1:5 in DEPC-treated water. For each quantitative RT PCR reaction, 5 &#956;l of sample was added to 10 &#956;l of power Sybr green master mix (Applied Biosystems) and 5 &#956;l gene-specific primers (2 x 2.5 &#956;M). The list of primers used to quantitate cDNA levels of different samples is provided in <xref ref-type="supplementary-material" rid="pgen.1006287.s014">S3 Table</xref>. Experiments were performed on three biologically independent replicates. Data were normalized to RNA levels of the housekeeping gene <italic>rpoB</italic>.</p><p>Samples were prepared the same as previously described [<xref ref-type="bibr" rid="pgen.1006287.ref012">12</xref>]. Briefly, overnight cultures of <italic>E</italic>. <italic>coli</italic> BL21&#955;DE3 containing WT RsiV pKBW201 (pDEST17-<italic>6xhis-2xflag</italic>-<italic>rsiV</italic><sup>59&#8211;285</sup>) or mutant RsiV S169W (pJH427), P259A (pJH450), Y261A (pJH430), P259A Y261A (pJH431), S169W P259A (pJH447), and S169W P259A Y261A (pJH429) were grown at 30&#176;C in LB + ampicillin. The cell cultures were diluted 1:100 into 500 ml of LB + ampicillin in 2 L baffled flasks and incubated at 30&#176;C to an OD<sub>600</sub> of 0.5&#8211;0.6. IPTG was added to a final concentration of 1 mM to induce protein production and the cultures incubated for an additional 4 hours. Cells were chilled on ice and collected by centrifugation at 5000xg. Cell pellets were stored at &#8722;80&#176;C until time for purification. Cell pellets were thawed on ice and resuspended in 5 ml lysis buffer (50 mM NaH<sub>2</sub>PO<sub>4</sub>, 250 mM NaCl, 10 mM imidazole, pH 8.0) per 500 ml of initial culture volume. Cells were lysed by passaging through a Microfluidics LV1 high shear microfluidizer (Newton, MA) twice. Lysate was centrifuged at 15,000x<italic>g</italic>, for 30 minutes at 4&#176;C to pellet cellular debris. Cleared lysate was applied to a nickel affinity column to bind 6xHis-tagged protein (Qiagen). The column was washed with 10 column volumes of wash buffer (50 mM NaH<sub>2</sub>PO<sub>4</sub>, 250 mM NaCl, 20 mM imidazole, pH 8.0). Protein was eluted with elution buffer (50 mM NaH<sub>2</sub>PO<sub>4</sub>, 250 mM NaCl, 250 mM imidazole, pH 8.0) and collected in 0.5 ml fractions. Samples from each fraction were analyzed by SDS-PAGE and elution fractions containing the desired protein were combined. Combined fractions were then dialyzed into lysis buffer to remove the excess imidazole.</p><p>RsiV and HEW-lysozyme mixed in a 1:3 molar ratio were allowed to incubate for 60 minutes, at 4&#176;C, and run through a Superdex-75 size exclusion column (GE). Fractions representing the complex confirmed by running a SDS-PAGE gel stained with Coomassie Blue were pooled and concentrated using a Amicon 30kDa filter to 8.9 mg/ml. Crystallization drops with commercially available screens using a TTP LabTech Mosquito robot were set-up via the hanging-drop vapor diffusion method at 18&#176;C with the drop containing 0.4 &#956;l each of protein and crystallization solutions. Crystals were obtained in a variety of PEG containing conditions within 14 days. In order to perform experimental phasing selenomethionine labeled RsiV protein was prepared. For production for selenomethionine labeled protein, cells were grown at 35&#176;C overnight in M9 minimal media supplemented with 100 &#956;g/ml ampicillin. The following morning cells were subcultured 1:50 into 1L flasks of M9 minimal media with ampicillin and incubated at 35&#176;C. When the cultures reached O.D.600 of 0.5, 100 mg/L of lysine, phenylalanine, threonine, and 50 mg/L of isoleucine, leucine, and valine were added to inhibit methionine synthesis as described [<xref ref-type="bibr" rid="pgen.1006287.ref081">81</xref>]. Selenomethionine was added to the media at 60 mg/L [<xref ref-type="bibr" rid="pgen.1006287.ref081">81</xref>] and the temperature shifted to 28&#176;C. 15 minutes after addition of amino acid cocktail, IPTG was added to a final concentration of 1mM to induce protein expression. Purification of the selenomethionine labeled protein proceeded as described for unlabeled RsiV. Selenomethionine labeled RsiV was purified as described above complexed with lysozyme and crystallization plates set up similar to the wild type protein. Crystals obtained in 0.2 M sodium nitrate, 20% w/v PEG 3350 were harvested for diffraction data collection.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Crystals were flash-cooled in liquid nitrogen and shipped to the 4.2.2 synchotron beamline at the Advanced Light Source (Berkeley, CA, USA) for remote data collection. The data were processed using XDS [<xref ref-type="bibr" rid="pgen.1006287.ref082">82</xref>]. Pointless [<xref ref-type="bibr" rid="pgen.1006287.ref083">83</xref>,<xref ref-type="bibr" rid="pgen.1006287.ref084">84</xref>] and Aimless [<xref ref-type="bibr" rid="pgen.1006287.ref084">84</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006287.ref089">89</xref>] from the CCP4 [<xref ref-type="bibr" rid="pgen.1006287.ref090">90</xref>] software suite were used for conversion of intensities to structure factors and scaling. Structure solution, initial model building and refinements were performed using AutoSol [<xref ref-type="bibr" rid="pgen.1006287.ref091">91</xref>], AutoBuild [<xref ref-type="bibr" rid="pgen.1006287.ref092">92</xref>] and Phenix.refine [<xref ref-type="bibr" rid="pgen.1006287.ref093">93</xref>] from the Phenix suite [<xref ref-type="bibr" rid="pgen.1006287.ref093">93</xref>]. Model building was performed in Coot [<xref ref-type="bibr" rid="pgen.1006287.ref094">94</xref>] and all structural figures were generated using PyMOL [<xref ref-type="bibr" rid="pgen.1006287.ref095">95</xref>]. All crystallography software used were configured and deployed using SBGrid [<xref ref-type="bibr" rid="pgen.1006287.ref096">96</xref>]. The surface interface of RsiV and lysozyme was determined using Protein interfaces, surfaces and assemblies' service PISA at the European Bioinformatics Institute (<ext-link ext-link-type="uri" ns0:href="http://www.ebi.ac.uk/pdbe/prot_int/pistart.html">http://www.ebi.ac.uk/pdbe/prot_int/pistart.html</ext-link>) [<xref ref-type="bibr" rid="pgen.1006287.ref029">29</xref>].</p><p>ITC analysis was performed as previously described [<xref ref-type="bibr" rid="pgen.1006287.ref012">12</xref>]. Briefly, 6xHis-2xFlag-RsiV<sup>59&#8211;285</sup> and mutants were purified as described above and buffer matched with HEW lysozyme (&#8805;98% pure, Sigma Aldrich) by dialysis into 50 mM Na<sub>2</sub>HPO<sub>4</sub>, 200 mM NaCl, and pH 7.0 for 24 h at 4&#176;C. Final protein concentrations as determined by absorbance at OD<sub>280</sub> were adjusted to 6xHis-2xFlag-RsiV<sup>59&#8211;285</sup> (0.01 mM) and HEW lysozyme (0.1 mM) with filtered dialysate. The protein samples were degassed and ITC measurements recorded using a MicroCal VP-ITC System (GE Healthcare) with HEW lysozyme as the injected sample and 6<sub>X</sub>His-2<sub>X</sub>Flag-RsiV<sup>59&#8211;285</sup> as the cell sample. 21 injections of HEW lysozyme were used, with 180 seconds spacing between events. The chamber was kept under constant stirring at 350 rpm and all experiments were performed at 25&#176;C. The binding reaction reached saturation during the experiment and control experiments where HEW lysozyme was injected into buffer showed that the heats of dilution were constant across all injections. The constant heat of dilution, as determined by the average of the last 3&#8211;5 injections, was subtracted and the data are analyzed using the single site binding model provided in the ITC analysis package. The values for affinity were averaged from three separate runs from two different protein preps.</p><p>Protein from 6xHis-2xFlag-RsiV<sup>59&#8211;285</sup> and 6xHis-2xFlag-RsiV<sup>59-285, S169W, P259A, Y261A</sup> was purified as described above. Samples were further purified on a BioRad DuoFlow FPLC with a Superdex 75 (GE) size exclusion column. Fractions were pooled and the concentration was determined by OD<sub>280</sub> with an extinction coefficient calculated for each mutant as described [<xref ref-type="bibr" rid="pgen.1006287.ref097">97</xref>]. Samples were diluted to 20 &#956;M and analyzed using a 1mm cuvette in a Jasco J-815 CD spectropolarimeter.</p><p>Increasing concentrations of purified RsiV (prepared as described above) were mixed with lysozyme (20 &#956;g/ml) for 15 minutes. The purified RsiV + lysozyme (100 &#956;l) mixture was combined with <italic>M</italic>. <italic>lysodekticus</italic> (100 &#956;l, OD<sub>450</sub> = 0.9, Sigma) The OD<sub>450</sub> was measured every minute for 30 minutes to monitor <italic>M</italic>. <italic>lysodekticus</italic> degradation. Lysozyme activity was calculated based on the equation as described by Sigma [<xref ref-type="bibr" rid="pgen.1006287.ref041">41</xref>]. Briefly, units/ml enzyme = (&#916;OD<sub>450</sub>)*(dilution factor)/ (0.001*0.05), Specific Activity = (units/ml enzyme) / (mg solid/ ml enzyme).</p><p>The minimum inhibitor concentration (MIC) was determined by diluting overnight cultures 1:100 in 1:50 LB + 1 mM IPTG. The cells were inoculated into 250 &#956;l of LB + 1 mM IPTG containing serial dilutions of hen egg white lysozyme ranging from 20 &#956;g/ml to 1.875 &#956;g/ml in a round bottom 96 well plate. The cells were grown for 16 hours at 37&#176;C. Growth was defined as OD<sub>600</sub> of greater than 0.05. All assays were performed using all strains listed in triplicate on the same day.</p>