<p>All affected and healthy family members or their legal guardian provided written and informed consent according to a protocol approved by our institutional review board and by the Israel National Committee for Human Genetic Studies in adherence with the Helsinki principles.</p><p>Genomic DNA was extracted from peripheral blood leukocytes using the 5 Prime ArchivePure DNA Blood Kit (5 Prime Inc., Gaithersburg, USA) or from OG-500 saliva collection kit (DNA Genotek Inc., Ottawa, Canada) according to the manufacturer's instructions.</p><p>Exome sequencing of individuals IV-3, IV-4, III-5, III-7 from family A, I-1, I-2 and II-1 from family C was performed by Otogenetics corporation using in-solution hybridization with Agilent AV5 + UTR Exome (71Mb) version 4.0 (Agilent, Santa Clara, USA) followed by massively parallel sequencing (Illumina HiSeq2000) with 100-bp paired-end reads. Reads were aligned to the Genome Reference Consortium Human Build 37 (GRCh37/hg19) using Burrows-Wheeler Aligner (BWA)[<xref ref-type="bibr" rid="pgen.1006369.ref036">36</xref>].</p><p>Duplicate reads, resulting from PCR clonality or optical duplicates, and reads mapping to multiple locations were excluded from downstream analysis. Reads mapping to a region of known or detected insertions or deletions were re-aligned to minimize alignment errors. Single-nucleotide substitutions and small insertion deletions were identified and quality filtered using the Genome Analysis Tool Kit (GATK) [<xref ref-type="bibr" rid="pgen.1006369.ref037">37</xref>]. Rare variants were annotated using ANNOVAR[<xref ref-type="bibr" rid="pgen.1006369.ref038">38</xref>] and identified by filtering the data from dbSNP138, the 1000 Genomes Project, the Exome Variant Server, and an in-house database of sequenced individuals. Variants were classified by predicted protein effects using Polyphen2 [<xref ref-type="bibr" rid="pgen.1006369.ref039">39</xref>] and SIFT [<xref ref-type="bibr" rid="pgen.1006369.ref040">40</xref>]. <xref ref-type="supplementary-material" rid="pgen.1006369.s004">S4 Table</xref> summarizes exome sequencing details.</p><p>Genomic DNA was PCR-amplified using oligonucleotide primer pairs spanning the entire coding sequence as well as intron&#8211;exon boundaries of <italic>WNT10A</italic>, <italic>TP63</italic> and <italic>TSPEAR</italic> (<xref ref-type="supplementary-material" rid="pgen.1006369.s005">S5 Table</xref>) and Taq polymerase (Qiagen, Hilden, Germany). Cycling conditions were as follows: 94&#176;C, 2min; 94&#176;C, 40 sec; 61&#176;C, 40 sec; 72&#176;C 50 sec, for 3 cycles, 94&#176;C, 40 sec; 59&#176;C, 40 sec; 72&#176;C 50 sec, for 3 cycles, 94&#176;C, 40 sec; 57&#176;C, 40 sec; 72&#176;C 50 sec, for 34 cycles. Gel-purified (QIAquick gel extraction kit, QIAGEN, Hilden, Germany) amplicons were subjected to bidirectional DNA sequencing with the BigDye terminator system on an ABI Prism 3100 sequencer (Applied Biosystems, NY, USA).</p><p>To screen for the c.1728delC mutation (families A and B), we PCR-amplified a 148 bp fragment with Taq polymerase (Qiagen, Hilden, Germany) and the following primers 5`- CTCCGTCATCTACGAGCTGAACGTGACCGCGCAGGCCTTTTT-3`and 5`- GATGAGCCTAACGGGGATTCC-3`. The mutation creates a recognition site for endonuclease MseI (New England Biolabs, Frankfurt, Germany). To screen for the c.454_457delCTGG mutation (family B), we PCR-amplified a 770 bp fragment, with Taq polymerase (Qiagen, Hilden, Germany) and the following primers 5`- TCTCACCACCTGTGCTCATC-3`and 5`- CACCTGTTCTCGCCAATGTC -3`. The mutation creates a recognition site for endonuclease BglI (New England Biolabs, Frankfurt, Germany). To screen for the c.1852T&gt;A mutation (family C), we PCR-amplified a 221 bp fragment, with Taq polymerase (Qiagen, Hilden, Germany) and the following primers 5`- GTAGCTTCTGGCCAATCCCC-3`and 5`- GAAGCAAG GCTCTGGGAGG-3`. The mutation creates a recognition site for endonuclease MseI (New England Biolabs, Frankfurt, Germany). To screen for the c.1915G&gt;A mutation (family C), we PCR-amplified a 220 bp fragment, with Taq polymerase (Qiagen, Hilden, Germany) and the following primers 5`- GGATGGAAGAGGCTCAGATG-3`and 5`- AGATGAGG TAGGCACCAGCCGTGGTGCTGAAGGCCTCCGAAT-3`. The mutation creates a recognition site for endonuclease EcoRI (New England Biolabs, Frankfurt, Germany). PCR cycling conditions were as follows: 94&#176;C, 2min; 94&#176;C, 40 sec; 61&#176;C, 40 sec; 72&#176;C 50 sec, for 3 cycles, 94&#176;C, 40 sec; 59&#176;C, 40 sec; 72&#176;C 50 sec, for 3 cycles, 94&#176;C, 40 sec; 57&#176;C, 40 sec; 72&#176;C 50 sec, for 34 cycles. PCR products were incubated with the appropriate enzyme at 37&#176;C for 16 hours followed by 20 min of inactivation at 65&#176;C. The digested PCR products were electrophoresed in ethidium bromide-stained 3% agarose gels.</p><p>For quantitative real-time PCR (qRT-PCR), cDNA was synthesized from 1000 ng of total RNA using qScript kit (Quanta Biosciences, Gaithersburg, MD). cDNA PCR amplification for <italic>TSPEAR</italic> and <italic>GAPDH</italic> (as a control) was performed with TaqMan SNP expression assays # Hs00376562_m1 and Hs02758991_g1 respectively (Applied Biosystems, Forster, CA, USA) according to the manufacturer protocol. cDNA PCR amplification for other genes was carried out with the PerfeCTa SYBR Green FastMix (Quanta Biosciences, Gaithersburg, USA) on a StepOnePlus system (Applied Biosystems, Waltham, USA) with gene-specific intron-crossing oligonucleotide pairs (<xref ref-type="supplementary-material" rid="pgen.1006369.s006">S6 Table</xref>). Cycling conditions were as follows: 95&#176;C, 20 sec and then 95&#176;C, 3 sec; 60&#176;C, 30 sec for 40 cycles. Each sample was analyzed in triplicates. For each set of primers, standard curves were obtained with serially diluted cDNAs. Results were normalized to <italic>GAPDH</italic> mRNA levels. qRT-PCR results were analyzed by <italic>t-test</italic> statistical analysis.</p><p>Keratinocytes (KCs) cell cultures were established from skin biopsies after written informed consent had been obtained as previously described [<xref ref-type="bibr" rid="pgen.1006369.ref041">41</xref>]. Primary KCs were maintained in KC Growth Medium (KGM) supplemented with 0.4% bovine pituitary extract, 0.1% human epidermal growth factor (hEGF), 0.1% insulin, 0.1% hydrocortisone and 0.1% gentamicin/amphotericin B. HaCaT cells were kindly provided by Dr. Dina Ron (Technion, Haifa, Israel). The cells were maintained in MEM media supplemented with 10% fetal calf serum, 1% L-glutamine, 1% streptomycin and 1% amphotericin (Biological Industries, Beit-Haemek, Israel).</p><p>KCs were cultured in 6 well culture plates at 37&#176;C in 5% CO<sub>2</sub> in a humidified incubator and were harvested at 60% confluence. To down regulate <italic>TSPEAR</italic> expression, we used human TSPEAR small interference RNAs (siRNA) (Santa Cruz; sc-62060) (5`-CCUUCUCGGUGAACAGUAUtt-3`, 5`-CAUUGCCGC CACCUAUUUAtt-3`and 5`-CACUCCUGACCUUUCGUAAtt-3`). As control siRNA, we used Stealth RNAi Negative Control Duplex (Invitrogen, Carlsbad, CA). Twenty five pmol of siRNAs were transfected into KCs using Lipofectamine RNAiMax (Invitrogen). The transfection medium was replaced after 6 hours with high calcium (1.4mM)-containing KGM.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Total RNA (200 ng) was reverse transcribed and cRNA prepared using TargetAmp-Nano Labeling Kit (Epicentre Biotechnologies, Madison, WI) according to the manufacturer's protocol. One and a half &#956;g of biotinylated cRNA was hybridized to HumanHT-12 v4 Expression BeadChip (encompassing more than 47,000 transcript targets), washed, and scanned on a BeadArray 500GX Reader using Illumina BeadScan image data acquisition software (version 2.3.0.13). Quality control and quantile normalization of the microarray data was done by BeadStudio 3.0 software (Illumina). The scanning data of the three biological repeats (total of 12 data sets) were exported to JMP genomic Software (SAS, Cary, NC), log transformed and non-expressed genes (detection p-value&lt;0.01), transcripts with low expression (log2 value &lt; 6.5) or with low variation across all samples (variation &lt; 0.05) were removed from the analysis. The data was analyzed using two-way ANOVA and differently expressed genes (DEGs) were defined as transcripts that were statistically significant at corrected p-value &#8804;0.05 using the False Discovery Rate (FDR) with at least 0.75 delta differences. Pathway analysis to identify statistically significant functional categories in the data set was performed using Ingenuity Pathway analysis (IPA 8.0, QIAGEN Redwood City, <ext-link ext-link-type="uri" ns0:href="http://www.qiagen.com/ingenuity">www.qiagen.com/ingenuity</ext-link>).</p><p>For immunofluorescence analysis of skin biopsies, 5 &#956;m paraffin-embedded sections were kept overnight at 37&#176;C and de-paraffinized using xylene/ethanol. Antigen retrieval was done with 0.01M citrate buffer, pH 6.0 (Invitrogen, Carlsbad, CA) in a microwave for 25 min. Sections were blocked with 2% bovine serum albumin (BSA) in phosphate-buffered saline (PBS) for 30 min at room temperature. Primary antibodies used: rabbit anti-TSPEAR primary antibody (Abcam, Cambridge, MA, USA, 1:200 dilution); goat anti-NOTCH primary antibody (Santa Cruz, Dallas, TX, USA, 1:75 dilution). Both antibodies were diluted in 2% BSA PBS and incubated overnight at 4&#176;C. Rhodamine Red-X goat anti rabbit IgG (H+L) (Life Technologies/Invitrogen) and Alexa Fluor 568 donkey anti goat IgG (H+L) (Thermo Fisher Scientific) were used as a secondary antibody and were diluted 1:200 with 2% BSA in PBS followed by incubation for 45 min at room temperature. Coverslips were mounted in DAPI Fluoromount-G (Southern Biotechnologies, Birmingham, AL). Negative controls consisted of slides processed similarly while omitting the primary antibody. As a positive control for TSPEAR staining, we used normal placenta tissue[<xref ref-type="bibr" rid="pgen.1006369.ref042">42</xref>]. Specimens were examined using either a Nikon 50I microscope connected to DS-RI1 digital camera or a Zeiss LSM700 confocal microscope for fluorescence image acquisition.</p><p>HaCaT cells were seeded on hDLL1 (R&amp;D Systems, Minneapolis, MN) coated 24 wells plate (50,000 cells/well). Twenty four hours after seeding, cells were transfected with a Notch response element-containing luciferase reporter construct, kindly obtained from Dr. David Sprinzak (Biochemistry Department, The George S. Wise Faculty of Life Sciences, Tel Aviv University) as well as a Renilla expression vector, and control siRNA (Stealth<sup>&#8482;</sup> RNAi Negative Control Duplex Invitrogen, Carlsbad, CA) or TSPEAR specific siRNA (sc-91435; Santa Cruz Biotechnology, Santa Cruz, CA) using Lipofectamine2000 (Invitrogen, Carlsbad, CA). Forty eight hours after transfection, luciferase activity was read using a dual luciferase assay (Promega, Madison, USA). Luciferase activity was normalized to Renilla luciferase.</p><p><italic>K14-Cre</italic> and <italic>H2B-GFP lox</italic>P mice were purchased from The Jackson Laboratory. <italic>K14-Cre</italic> mice contain a human keratin 14 promoter directing expression of Cre recombinase, while <italic>H2B-GFP</italic> mice have a fusion H2B histone with a C-terminally attached eGFP. <italic>K14-Cre</italic> and <italic>H2B-GFP lox</italic>P mice were crossed, heterozygous littermates interbred, and resulting pups genotyped. Homozygous <italic>K14 H2B-GFP</italic><sup><italic>+/+</italic></sup> mice were selected for and used as breeding pairs. All mice were kept at the University of California, San Diego (UCSD) animal facilities, and all animal experiments were approved by the UCSD Institutional Animal Care and Use Committees and were conducted in accordance with the Guideline for the Care and Use of Laboratory Animals.</p><p>Dorsal skin was isolated from <italic>K14-H2B-GFP</italic> mice and placed dermal side down into individual sterile petri dishes containing 4 ml warmed supplemented William&#8217;s E. Media (WEM) as previously described[<xref ref-type="bibr" rid="pgen.1006369.ref043">43</xref>], with the orientation of the hair parallel to the <italic>longitudinal</italic> axis. Thin dorsal tissue strips were then sliced off, gently abrading to remove any loose hair.</p><p>Tissue strips were transfected with mouse <italic>Tspear</italic> siRNA (sc-270602; Santa Cruz Biotechnology, Santa Cruz, CA) or control siRNA (sc-36869; Santa Cruz Biotechnology). All reagents required for transfection were obtained from Santa Cruz Biotechnology (siRNA transfection reagent, sc-29528; siRNA transfection medium, sc-36868). Transfection was performed as previously described [<xref ref-type="bibr" rid="pgen.1006369.ref043">43</xref>] and following 7 hours of transfection, tissue strips were maintained in 6-well plate with 2 ml supplemented WEM for an additional 24 hours. Following 24 hours, two tissue strips from each treatment group were used for average hair bulb measurement. In detail, two hundred &#956;l of 2% low melt agarose solution (Agarose II; Midsci, MO, USA) dissolved in sterile Dulbecco&#8217;s phosphate buffered saline (DPBS) were used to coat the bottom of 6-well glass bottom plate (MatTek, Ashland, MA). Dorsal tissue strips were positioned parallel to the bottom of the plate, and embedded in agar. One ml of WEM was gently added to the wells. The 6-well plate was then placed in a pre-warmed incubation chamber (37&#176;C, 5% CO<sub>2</sub>) of a Zeiss Axio Observer.Z1 microscope. Axiovision software (4.8.2 SP3) was used to select and mark multiple non-overlapping fields of view on each tissue strip covering a significant area of each strip. Fluorescent images were captured using an automatic exposure time and an excitation wavelength of 470 nm. Data were analyzed in a single blinded manner. Raw data collected were given an alpha-numeric cipher and were subsequently analyzed by a blinded investigator unaware of the conditions tested or grouping. In each hair follicle, three horizontal measurements were done in the area of hair bulb and proximal hair shaft. The average of those three measurements was calculated for each hair follicle from the two different treatment group and was defined as average hair bulb diameter. Three independent experiments were done with 3 different mice. In each mouse, 2 skin samples from each of the two treatment groups (<italic>Tspear</italic>-siRNA vs. control-siRNA) were used for average hair bulb measurement.</p><p>In addition, 3 strips from each treatment group were collected 24 hours following transfection for RNA isolation (RNeasy kit, Qiagen, Valencia, CA) and validation of <italic>Tspear</italic> silencing. For reverse transcription, we used an iScript cDNA synthesis kit (Bio-Rad, Hercules, CA) with 1 &#956;g of RNA as starting material. The resulting cDNA was diluted 1:10 with nuclease free water and 5 &#956;l of the diluted solution was used as template for subsequent qPCR reactions.</p><p>Quantitative real-time PCR was performed on an Applied Biosystems 7300 using TaqMan Universal PCR Master Mix (Applied Biosystems, Carlsbad, CA), template cDNA, and TaqMan primers. TaqMan primers were ordered from Life technologies; <italic>Tspear</italic> (Mm00455327_m1), and <italic>Gapdh</italic> (Mm99999915_g1) which served to normalize data (Life Technologies/Invitrogen, Grand Island, NY). To quantify <italic>Notch1</italic> expression, we used 2x SYBR Green qPCR Master Mix (Biotool, Houston,TX) and Primer bank ID 13177625a1 <italic>Notch1</italic> primers while <italic>Rplp0</italic> served as an internal control (Forward primer 5&#8217;-GAGATTCGGGATATGCTGTTGG-3&#8217; and Reverse primer 5&#8217;-CGGGTCCTAGACCAGTGTTCT-3&#8217;).</p><p>Finally, tissue strips from each treatment group were frozen in liquid nitrogen 48 hours after transfection for immunohistochemistry studies. Seven &#956;m-thick cryosections were prepared and stored at &#8722;80&#176;C until use. For Tspear protein expression, cryosections were first air-dried for 10 min and then fixed in acetone at -20&#176;C for another 10 min. After air drying, the slides were washed three times for 5 min in PBS. Following 20 min incubation of cryosections with 2% goat serum in PBS, cryosections were incubated overnight at 4&#176;C with the rabbit anti-Tspear monoclonal antibody (Abcam, Cambridge, MA) at 1:50 dilution with 2% goat serum in PBS solution. This was followed by incubation with Alexa Fluor<sup>&#174;</sup> 568 Goat Anti-Rabbit secondary Antibody (Life Technologies/Invitrogen) for 45 min at RT in 1:200 dilution with 2% goat serum in PBS solution. Incubation steps were interspersed with three washes, 5 min each, with PBS. Then sections were embedded and counterstained with DAPI for the identification of cell nuclei.</p><p>For Masson-Fontana histochemistry, cryosections were air dried and fixed in ethanol-acetic acid. The sections were washed in Tris-buffered saline (TBS) and distilled water several times. Cryosections were treated with ammoniacal silver solution (Thermo Fisher Scientific, Carlsbad, CA) for 40 min at 56&#176;C in the dark. After washing in distilled water, the sections were treated with 5% aqueous sodium thiosulphate (Sigma-Aldrich, St. Louis, MO) for 1 min. Then, the sections were washed in running tap water for 3 min and were counterstained with haematoxylin for 45 seconds. After washing in distilled water, sections were dehydrated and mounted in Eukitt (Sigma-Aldrich, St. Louis, MO). Tspear immunoreactivity and melanin content by Masson-Fontana histochemistry were compared between test and control sections by quantitative histomorphometry as previously described [<xref ref-type="bibr" rid="pgen.1006369.ref043">43</xref>,<xref ref-type="bibr" rid="pgen.1006369.ref044">44</xref>] using NIH IMAGE software (NIH, Bethesda, MD, USA).</p><p>For apoptosis detection, a kit for DeadEnd Fluorometric TUNEL system analysis (Promega, Madison, WI, USA) was used. Briefly, cryosections were air dried and fixed in 4% formaldehyde in PBS for 25 minutes. Following several washing steps in PBS, permeabilization step with 0.2% Triton in PBS was conducted for 5 min. Following several washing steps in PBS, sections were equilibrated with equilibration buffer for 5&#8211;10 min and then incubated with TdT reaction mix for 60 minutes at 37&#176;C. Following stop reaction step and washing steps with PBS, sections were embedded and counterstained with DAPI for the identification of cell nuclei.</p><p>The URLs for data presented herein are as follows:</p><p xmlns:ns0="http://www.w3.org/1999/xlink">1000 genomes project, <ext-link ext-link-type="uri" ns0:href="http://www.1000genomes.org/">http://www.1000genomes.org/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">ConSurf, <ext-link ext-link-type="uri" ns0:href="http://consurftest.tau.ac.il/">http://consurftest.tau.ac.il/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">dbSNP, <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/SNP/">http://www.ncbi.nlm.nih.gov/SNP/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">Exome Variant Server (<ext-link ext-link-type="uri" ns0:href="http://evs.gs.washington.edu/EVS/">http://evs.gs.washington.edu/EVS/</ext-link>)</p><p xmlns:ns0="http://www.w3.org/1999/xlink">GenBank, <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/Genbank/">http://www.ncbi.nlm.nih.gov/Genbank/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">NHLBI Grand Opportunity Exome Sequencing Project, <ext-link ext-link-type="uri" ns0:href="https://esp.gs.washington.edu/drupal/">https://esp.gs.washington.edu/drupal/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">Online Mendelian Inheritance in Man (OMIM), <ext-link ext-link-type="uri" ns0:href="http://www.omim.org">http://www.omim.org</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">PolyPhen-2, <ext-link ext-link-type="uri" ns0:href="http://genetics.bwh.harvard.edu/pph2/">http://genetics.bwh.harvard.edu/pph2/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">SIFT, <ext-link ext-link-type="uri" ns0:href="http://sift.jcvi.org/">http://sift.jcvi.org/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">UCSC Genome Browser, <ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">Human Gene Mutation Database, <ext-link ext-link-type="uri" ns0:href="http://www.hgmd.cf.ac.uk/ac/index.php">http://www.hgmd.cf.ac.uk/ac/index.php</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">ANNOVAR, <ext-link ext-link-type="uri" ns0:href="http://annovar.openbioinformatics.org/en/latest/">http://annovar.openbioinformatics.org/en/latest/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">Burrows-Wheeler Aligner, <ext-link ext-link-type="uri" ns0:href="http://bio-bwa.sourceforge.net/">http://bio-bwa.sourceforge.net/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">GATK, <ext-link ext-link-type="uri" ns0:href="https://www.broadinstitute.org/gatk/">https://www.broadinstitute.org/gatk/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">Ingenuity Pathway Analysis (IPA), <ext-link ext-link-type="uri" ns0:href="http://www.qiagen.com/ingenuity">www.qiagen.com/ingenuity</ext-link></p>