<p>This study was conducted under IRB-approved protocols, by IRB committees at University of Maryland School of Medicine (UMSOM), USA (HP-00059851), National Centre of Excellence in Molecular Biology, Lahore, Pakistan (FWA00001758), and the National Institutes Health, USA (Combined Neuroscience Blue Panel IRB; OH-93-N-016), and in accord with the Declaration of Helsinki for the release of clinical information, family history, and blood draw. Written informed consent was obtained from all adult participants and parents of minor subjects. Genomic DNA was extracted from blood samples. Clinical histories were obtained for all the individuals of families PKDF041 and PKDF751. A general physical examination was performed to evaluate general health. Pure tone audiometry tests for air and bone conduction were performed at frequencies from 250 to 8,000 Hz. Tympanometry and otoacoustic emission tests were performed to assess middle ear and cochlear function. Vestibular function was evaluated by testing tandem gait ability and by using the Romberg test.</p><p>Exome sequencing on the genomic DNA of one affected individual of family PKDF751 was performed and analyzed as described previously [<xref ref-type="bibr" rid="pgen.1007297.ref039">39</xref>].</p><p>We used the XHMM and CoNIFER methods [<xref ref-type="bibr" rid="pgen.1007297.ref040">40</xref>, <xref ref-type="bibr" rid="pgen.1007297.ref041">41</xref>] to call CNV events in WES data. The segregation of the candidate variants in participating member of family PKDF751 was performed through Sanger sequencing. Primers were designed with Primer3 software, and used to amplify exons as well as flanking introns and untranslated regions (<xref ref-type="supplementary-material" rid="pgen.1007297.s010">S4 Table</xref>).</p><p>All experiments were approved by the Animal Care and Use Committees at the University of Maryland, School of Medicine in accordance with the National Institutes of Health (NIH) Guide for the Care and Use of Laboratory Animals. The <italic>B6N(Cg)-Ppip5k2</italic><sup><italic>tm1a(EUCOMM)Wtsi/J</italic></sup> strain was generated by KOMP. In this study, we designated this strain as <italic>Ppip5k2</italic><sup><italic>K</italic></sup><italic>^</italic> due to its intact kinase domain. The <italic>Ppip5k2</italic><sup><italic>K</italic></sup><italic>^</italic> mice had a gene-trap cassette downstream of exon 13, which is expected to cause premature truncation.</p><p>Lipofectamine 3000 (Life Technologies) was used to transiently transfect HEK293 cells with pDest515 plasmids hosting cDNAs encoding Flag-tagged versions of either full-length PPIP5K2 (isoform 11; accession number NM_001345875), PPIP5K2 p.Arg837His (PPIP5K2<sup>R837H</sup>) or a truncated version of PPIP5K2 containing residues 1 to 466 (PPIP5K2<sup>1-466</sup>). The mutations were created using a Q5 site-directed mutagenesis kit (New England Biolabs). The primers used are as follows (mutagenic codons underlined): R837H mutation, forward: GTCTATTCTT<underline><bold>CAC</bold></underline>TATGGTGCCTTATG; reverse: AGCAAAGAATGTACATGAC. Truncation (stop codon insertion), forward: <underline><bold>TAG</bold></underline>GTCATTTTTCTGGAATAAATCG; reverse: <underline><bold>TTA</bold></underline>CTCTAATACAGTCTTAAGTTG. All mutations were confirmed by sequencing.</p><p>In some experiments, the wild type (WT) and mutant PPIP5K2 constructs were expressed in <italic>PPIP5K</italic><sup>-/-</sup> HEK293 cells that had been radiolabeled by growing them in media containing [<sup>3</sup>H]-inositol. Cellular levels of [<sup>3</sup>H]IP7 and [<sup>3</sup>H]IP8 were determined by HPLC [<xref ref-type="bibr" rid="pgen.1007297.ref012">12</xref>].</p><p>For the purification of WT and mutant PPIP5K2 proteins, the HEK293 cells hosting the various constructs were harvested 16&#8211;20 hr after transfection and lysed in ice-cold buffer containing 10 mM HEPES, 130 mM NaCl, 1% Triton X-100, 10 mM NaF, 10 mM Na<sub>2</sub>HPO<sub>4</sub>, 10 mM Na pyrophosphate and protease inhibitor cocktail (Roche). All subsequent steps were performed on ice in an anaerobic chamber (Bactron CAT180). The FLAG-tagged PPIP5K2 proteins were immunopurified using FLAG M2 affinity gel (Sigma). Purified FLAG-PPIP5K2 proteins were analyzed by SDS-PAGE and stained with Coomassie Blue. Densitometry analysis (ImageJ) was used to calculate the amounts of PPIP5K2 loaded onto the gel, by reference to additional lanes loaded with known amounts of the human recombinant PPIP5K2 kinase domain.</p><p>All enzyme assays were performed using physiologically relevant substrate concentrations at 37&#176;C; the extent of substrate conversion was up to 30%. IP8 phosphatase activities were determined using approximately 70 ng of purified, WT or mutant PPIP5K2s in 30 min incubations with100 ul of assay buffer containing 1 mM Na<sub>2</sub>EDTA, 50 mM KCl, 20 mM HEPES pH 7.2, 2 mM MgCl<sub>2</sub>, 0.5 mg/ml BSA and 1 &#956;M [<sup>3</sup>H]IP8. The 5-IP7 kinase activities were determined using approximately 35 ng of these proteins in 90 min incubations with 100 ul of assay buffer containing 1 mM Na<sub>2</sub>EDTA, 50 mM KCl, 20 mM HEPES pH 7.2, 7 mM MgCl<sub>2</sub>, 5 mM ATP, 0.5 mg/ml BSA and 1 &#956;M 5-[<sup>3</sup>H]IP7. All assays were then acid-quenched, neutralized, and HPLC was used to determine the degree of metabolism [<xref ref-type="bibr" rid="pgen.1007297.ref012">12</xref>].</p><p>Inner ear tissues were harvested from temporal bones of the WT and <italic>Ppip5k2</italic><sup><italic>K</italic></sup><italic>^</italic> mice, fixed and processed for immunostaining as previously described [<xref ref-type="bibr" rid="pgen.1007297.ref042">42</xref>]. The following primary antibodies were used in our studies: anti-Myosin VIIa (1:200; Proteus BioSciences, Ramona, CA), anti-PPIP5K2 (1:200; Stock# ab87054, Abcam, Cambridge, MA) and anti-PPIP5K1 (1:200; Novus Biologicals, Littleton, CO). Samples were mounted using ProLong Gold antifade reagent (Life Technologies, Carlsbad, CA) and imaged using an LSM 510 DUO confocal microscope (Zeiss Microimaging Inc., Thornwood, NY) with a &#215;63, 1.4 N.A. oil immersion objective.</p><p>Briefly, mice were anesthetized with an IP injection of ketamine/xylazine mix at 80&#8211;100 mg/kg and 10&#8211;12.5 mg/kg, body weight respectively. ABR recordings were performed using a TDT RZ6/BioSigRZ system along with RA4PA preamplifier/digitizer. The system was calibrated between 4&#8211;40KHz frequency range from 90 to 5 db using the ACO Pacific type 7016 microphone. For simultaneous recording from both ears, two open filed speakers were placed 10 cm apart from each ear. Two electrodes were placed behind each ear. The reference was placed at the base of the skull, whereas the ground was placed at the base of the tail. Mice were tested at broad band clicks and pure tone frequencies of 8, 16, 24, and 32 kHz for ABRs (n = 7 per genotype at each time point). All the stimuli were tested from 90 dB SPL to10 dB SPL in 5 dB decrements, with a total of 512 responses averaged at each level. Results are displayed as a mean and standard error of the mean (SEM). When no ABR signals were observed at 90db, for calculation purposes, mice were assigned a threshold of 95db. Statistical significance was determined using a 2-way ANOVA. A Bonferroni correction was performed to adjust for multiple statistical tests.</p><p>The whole inner ear was isolated from P150 WT and <italic>Ppip5k2</italic><sup><italic>K</italic></sup><italic>^</italic><sup><italic>/K</italic></sup><italic>^</italic> mice using the RiboPure RNA isolation kit (Life Technologies, Grand Island, NY). cDNA was prepared using an oligo-dT primer and SMARTScribe Reverse Transcriptase enzymes (Clontech, Mountain View, CA). To determine the differential expression of <italic>Ppip5k1</italic>, SYBRGreen based real-time primers were designed using Integrated DNA Technologies online PrimeTime qPCR assay design tool. The real-time PCR assays were performed in triplicate using an ABI StepOnePlus Real-Time thermal cycler (ABI, Foster City, CA). CT values were normalized using <italic>Gapdh</italic> as an endogenous control, and fold changes of <italic>Ppip5k1</italic> transcripts in different tissues were calculated using 2^-&#916;&#916;Ct standard formula. An expression with a 2-fold change and with a P-value less than 0.05 based on a Student&#8217;s t-test analysis was considered significant.</p><p>The URLs for data presented herein are as follows:</p><p xmlns:ns0="http://www.w3.org/1999/xlink">NHLBI Exome Sequencing Project (Exome Variant Server), <ext-link ext-link-type="uri" ns0:href="http://evs.gs.washington.edu/EVS/">http://evs.gs.washington.edu/EVS/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">Primer3, <ext-link ext-link-type="uri" ns0:href="http://frodo.wi.mit.edu/primer3">http://frodo.wi.mit.edu/primer3</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">PrimeTime qPCR assay design tool, <ext-link ext-link-type="uri" ns0:href="http://www.idtdna.com/Scitools/Applications/RealTimePCR/">http://www.idtdna.com/Scitools/Applications/RealTimePCR/</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">gEAR, <ext-link ext-link-type="uri" ns0:href="http://umgear.org/">http://umgear.org</ext-link></p><p xmlns:ns0="http://www.w3.org/1999/xlink">SHIELD (Shared Harvard Inner-Ear Laboratory Database) <ext-link ext-link-type="uri" ns0:href="https://shield.hms.harvard.edu/">https://shield.hms.harvard.edu/</ext-link></p>