<p>NIH3T3 mouse fibroblasts (ATCC) were maintained in DMEM (Quality Biological) containing 10% FBS, 2 mM <sc>l</sc>-glutamine and 1% penicillin/streptomycin. For primary rat limb bud cell culture, fore and hind limb buds were carefully dissected from the body wall of F344 rat embryos at 13.5dpc and pooled. The buds were washed with PBS, briefly digested in 0.05% trypsin/0.53mM EDTA (Corning), gently washed with three exchanges of PBS and resuspended by trituration in basal media consisting of DMEM:F12 (Corning) supplemented as described with the addition of TGF-&#945; (Peprotech), ROCK inhibitor Y27632 (Tocris), FGF2 (Peprotech) and LIF (Millipore) as indicated [<xref ref-type="bibr" rid="pgen.1006610.ref067">67</xref>]. Where indicated, cells were treated with bovine serum albumin or dimethyl sulfoxide (both from Sigma) as controls for parallel treatment by IL-6, OSM (Cell Signaling Technology), LIF (EMD Millipore), or Stat3i (a gift from Nadya Tarasova, National Cancer Institute, Frederick, Maryland, USA).</p><p>Control luciferase plasmids, pGL3-basic and pRL-TK-renilla were from Promega. Sox9 luciferase reporter (pE2.4-Sox9Luc) was a gift from Peter Koopman (University of Queensland, Brisbane, Queensland, Australia). The pE770Sox9-Luc reporter was generated by <italic>KpnI</italic> digestion of pE2.4-Sox9Luc and vector re-ligation. Individual &#916;SRE luciferase mutants were generated using the Quick Change II site-directed mutagenesis kit (Agilent Technologies) and the wild-type pE2.4-Sox9Luc template according to manufacturer&#8217;s protocol. The subsequent tandem &#916;SRE constructs were generated by sequential rounds of site-directed mutagenesis using the appropriate single-site template constructs and all materials were sequence verified before use. The mutagenic primers were as follows: &#916;SRE1 Fwd&#8212;GACATGCAATGCTAGGAACACCGGTTTGAAAAGAAACTTCGACTG, Rev&#8212;CAGTCGAAGTTTCTTTTCAAACCGGTGTTCCTAGCATTGCATGTC; &#916;SRE2 Fwd&#8212;GTTTGAAAAGAAACCCGGACTGAACAGAGTTGTAGCTTGCTGC, Rev&#8212;GCAGCAAGCTACAACTCTGTTCAGTCCGGGTTTCTTTTCAAAC; &#916;SRE3 Fwd&#8212;CCAAATAACAAATGCCCACCCGGGAAGAAAACGAGAGGAAAACG, Rev&#8212;CGTTTTCCTCTCGTTTTCTTCCCGGGTGGGCATTTGTTATTTGG. The following antibodies were used: anti-Sox9 (Millipore); anti-&#946;-actin (Sigma); anti-Stat1, anti-Stat3, anti-phospho-Stat3 (Y705), anti-phospho-Stat1 (Y701), anti-phospho-Stat5 (Y694), and anti-phospho-ERK1/2 (all from Cell Signaling Technology); anti-Stat5A (Santa Cruz); alkaline phosphatase-conjugated anti-digoxigenin Fab (Roche); alexa-488-conjugated anti-rabbit IgG (Invitrogen); IR-680-conjugated anti-rabbit IgG and IR-800CW-conjugated anti-mouse IgG (both from Licor).</p><p><italic>Stat3</italic><sup><italic>flox/flox</italic></sup> mice (a gift from Shizuo Akira, Osaka University, Osaka, Japan) were intercrossed with a &#946;-actinCre deleter strain (a gift from Mark Lewandoski, National Cancer Institute, Frederick, Maryland, USA) to generate germline deletion of a single allele of <italic>Stat3</italic> [<xref ref-type="bibr" rid="pgen.1006610.ref068">68</xref>]. The resultant heterozygous (<italic>Stat3</italic><sup><italic>&#916;/+</italic></sup>) mice were then intercrossed with <italic>TCre</italic> (a gift from Mark Lewandoski), <italic>Prx1Cre</italic> or <italic>Sox9Cre</italic> (both gifts from Susan Mackem, National Cancer Institute, Frederick, Maryland, USA) to generate specific driver strains. These strains were then re-crossed with <italic>Stat3</italic><sup><italic>flox/flox</italic></sup> mice to generate tissue-specific deletion of <italic>Stat3</italic>. Female <italic>R26R</italic><sup><italic>LacZ</italic></sup> reporter mice (a gift from Mark Lewandoski) were crossed with male <italic>TCre</italic> drivers to perfom the indicated lineage trace. All mice were maintained on a C57BL/6 background and routinely genotyped by PCR (primer sequences available upon request).</p><p>Whole mount ISH was performed, essentially, as previously described [<xref ref-type="bibr" rid="pgen.1006610.ref069">69</xref>] except BM purple (Roche) was used as the chromogenic substrate. Briefly, plasmids encoding cDNA for mouse <italic>Stat3</italic>, <italic>Scx</italic>, <italic>MyoG</italic>, or <italic>Sox9</italic> were linearized and DIG-labeled complimentary probes were generated using a DIG-labeling kit (Roche). As indicated, probes were hybridized overnight and detected using alkaline phosphatase-labeled anti-DIG antibody with subsequent chromogenic substrate development. For sectioning, embryos stained as above were embedded into paraffin wax after clearing in CitriSolv (Fisher Scientific) and sectioned serially at 16&#956;M by microtome.</p><p>For gross skeletal analysis, skeletons were prepared essentially as described by McLeod [<xref ref-type="bibr" rid="pgen.1006610.ref070">70</xref>]. Briefly, mice were euthanized according to institutional ACUC protocols, skinned, eviscerated, debulked of subcutaneous fat deposits and fixed in 95% ethanol. Skeletons were then treated in 100% acetone for 24-48hours (depending on age) and subsequently stained with Alcian Blue and Alizarin Red S. Finally, soft tissue digestion and specimen clearing was done in 1% potassium hydroxide. Preparations were stored and photographed in a 1:1 glycerol:ethanol solution using an Olympus SZX16 stereoscope and measurements were made by CellSens Dimension software.</p><p>For histological analyses, paraffin sections of bones were produced from E18.5, P0 and P4-aged mice from the indicated genetic crosses. Limbs were dissected, skinned and fixed in 4% paraformaldehyde in PBS. Tissues were dehydrated through a graded series of ethanol, cleared in xylene, and embedded into paraffin for sectioning. To visualize the growth plates, sections were stained with Alcian Blue.</p><p>For immunolocalization by fluorescence, sections were dewaxed and rehydrated, and heat-mediated antigen retrieval was performed in citrate buffer (pH 6.0). Sections were blocked in 3% normal goat serum (NGS) containing 0.1% Triton X-100 for 1 hour at room temperature and subsequently incubated with antibody specific for the indicated target in 3% NGS overnight at 4&#176;C. Sections were then washed in 1X Tris-buffered saline (TBS) and incubated with Alexafluor-488 conjugated secondary antibody in 3% NGS for 1 hour at room temperature. Sections were washed in TBS and countersained/mounted in vectashield containing DAPI (Vector Labs).</p><p>Humeri from control and mutant animals were dissected, combined, and dounce-homogenized in 1ml Trizol reagent (Invitrogen) at the indicated age. For NIH3T3 cells, cells were grown to 50% confluence in 12-well plates and serum starved overnight. Cells then were treated as indicated and lysed in 500&#956;l of Trizol reagent. Total RNA was recovered using an RNeasy Kit (Qiagen) and quantified by spectrophotometric analysis (NanoDrop). cDNA was reverse-transcribed from equal amounts of total RNA using random hexamer priming and the Verso cDNA Synthesis Kit (Thermo Scientific). qPCR was then performed on equal volumes of cDNA using a SsoFAST EvaGreen 2X PCR master mix (Biorad) in a CFX96 Real-Time System (BioRad). Data were normalized to GAPDH levels. Primers used were as follows: Sox9 Fwd, GAGGAAGTCGGTGAAGAACG, Sox9 Rev, CTGAGATTGCCCAGAGTGCT; Col2a1 Fwd, AAAGGGGCAGAAAGGAGAA, Col2a1 Rev, AGGATTTCCAGGGGTACCAG; GAPDH Fwd, AATGTGTCCGTCGTGGATCTG GAPDH Rev, CTGCTTCACCACCTTCTTGATGT.</p><p>Reverse transfection of indicated siRNAs was performed with RNAiMax (Invitrogen) according to manufacturer&#8217;s protocol in indicated media without antibiotics. Briefly, 60pmols of siRNA was incubated with 1&#956;l RNAiMax for 20 minutes at room temperature. Rat limb bud cells were plated in 12-well human fibronectin-coated plates (BD Biosciences) at a density of 1.5x10<sup>5</sup> cells/well after siRNA complex formation such that final total siRNA concentration was 50nM. The following siRNA constructs were used (Ambion/Thermo Fisher Scientific): Control #1 (catalog 4390843), Stat1 (ID s129044), Stat3-1 (ID s129046), Stat3-2 (ID s129047), Stat5A (s128672) and Stat5B (ID s129049).</p><p>NIH3T3 or primary rat limb bud cells were treated as indicated and harvested for protein analysis by cell-lysis in SDS sample buffer (2% SDS, 10% glycerol and 60mM Tris-HCl, pH 7.5) followed by DNA hydrolysis at 100&#176;C for 10 minutes. Total protein was quantified using the BioRad DC Protein Assay kit and equivalent amounts were subjected to SDS-PAGE on 4&#8211;12% Bis-Tris gels (Invitrogen). Gels were then transferred to nitrocellulose membranes, which were subsequently blocked in Odyssey diluent (Licor) and probed for indicated proteins with specific antibodies. Detection of protein-antibody complexes was carried out by IR dye-labeled secondary antibody and fluorescent capture using an Odyssey scanner (Licor). Color images were subsequently converted to greyscale for publication using Odyssey software (Licor). Where indicated, pixel intensities were captured using Odyssey software for quantitative analyses.</p><p>NIH3T3 cells were plated into 24-well plates and grown to 50% confluency. The indicated control or Sox9 (0.5&#956;g) and <italic>Renilla</italic> (0.05&#956;g) reporter constructs were co-transfected using Fugene 6 (Roche) in serum-free conditions. 1x10<sup>6</sup> rat limb bud cells were nucleofected by Amaxa (Lonza) protocol DN-100 in P3 primary cell solution with control or Sox9 (2&#956;g) and Renilla (0.2&#956;g) reporter constructs and plated in indiciated media. 24 hours post-transfection, cells were treated as indicated in the absence of serum, for a further 24 hours and harvested by passive lysis. Luciferase activities were measured using the Dual-Luciferase Assay Kit (Promega) and a MicroLumat<italic>Plus</italic> luminometer (Berthold Technologies).</p><p>ChIP was carried out essentially as described previously[<xref ref-type="bibr" rid="pgen.1006610.ref071">71</xref>]. Briefly, NIH3T3 cells were grown to 75% confluence in 100-mm dishes and subsequently serum-starved overnight (16h). As indicated, cells were treated in the absence of serum and fixed in formaldehyde (1% in medium) for 15 minutes at room temperature. Nuclear extracts were prepared, and DNA was sheared by sonication in a Bioruptor (Diagenode) at 4&#176;C to an average fragment size of 500bp as visualized by agarose gel. Equivalent volumes of nuclear extract were immunoprecipitated overnight with anti-Stat3 bound to protein G-sepharose beads (Sigma) at 4&#176;C and eluted the following day by boiling in SDS elution buffer (50mM Tris-HCl, pH 8.0, 10mM EDTA, 1% SDS, 50mM NaHCO<sub>3</sub>). Crosslinking of &#8220;input&#8221; and immunoprecipitated samples was reversed in proteinase-K at 65&#176;C overnight, and DNA was precipitated with phenol/chloroform by standard methods. DNA was analyzed by PCR using previously reported specific primers for <italic>Socs3</italic> [<xref ref-type="bibr" rid="pgen.1006610.ref072">72</xref>] or the following regions of the <italic>Sox9</italic> promoter: SRE-Fwd, AGAAACTTCGACTGAACAGAGTTGT, SRE-Rev, AAGTGGGCATTTGTTATTTGG; Control-Fwd, TCGGCTTTGGTTTTCATTG, Control-Rev, AAATGTTTGGGTGACTCAACG.</p><p>NIH3T3 cells were grown on poly-<sc>l</sc> lysine (Sigma) coated coverglass until 50% confluence and subsequently serum starved overnight. Cells were treated, as indicated, for 30 minutes and immediately fixed in 2% paraformaldehyde at room temperature for 10 minutes. Coverslips were then blocked in 3% normal goat serum (NGS) containing 0.1% Triton X-100 for 1h at room temperature and incubated overnight in indicated primary antibodies. Samples were then washed in 1X Tris-buffered saline (TBS) and incubated with Alexafluor-488 and Alexafluor-568 conjugated secondary antibodies for 1 hour at room temperature. Coverglasses were washed in TBS and mounted in vectashield containing DAPI (Vector Labs).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Data were analyzed by Student&#8217;s t-test to ascertain the statistical significance of observations. Statistical differences were considered significant if p&#8804;0.05* or p&#8804;0.01**. All animal comparisons are made based on a minimum of three (n = 3) control and mutant littermates. All other experiments were performed a minimum of three independent times, and graphical data represents mean &#177; SEM (standard error of the mean). The probability of Stat consensus sequence presence was calculated using the following formula:
<disp-formula id="pgen.1006610.e001"><alternatives><graphic id="pgen.1006610.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006610.e001.jpg" /><ns1:math id="M1"><ns1:mrow><ns1:mtext>Probability&#160;of&#160;site&#160;</ns1:mtext><ns1:mo>=</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:mrow><ns1:mo>[</ns1:mo><ns1:mrow><ns1:msup><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>&#8722;</ns1:mo><ns1:mtext>p</ns1:mtext></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mrow><ns1:mtext>N</ns1:mtext><ns1:mo>*</ns1:mo><ns1:mtext>bp</ns1:mtext></ns1:mrow></ns1:msup></ns1:mrow><ns1:mo>]</ns1:mo></ns1:mrow><ns1:mtext>&#160;</ns1:mtext><ns1:mo>*</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:mn>100</ns1:mn></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
where;</p><p>p = 1/(#nucleotides<sup>k-mer</sup>) = 1/(4<sup>10</sup>)</p><p>N = number of strands of DNA available for interrogation = 2</p><p>bp = length in bases of DNA interrogated = 2100</p><p>All animal procedures were performed following the guidelines from the NCI-Frederick Animal Care and Use Committee under an approved animal study proposal. NCI-Frederick is accredited by AAALAC International and follows the Public Health Service Policy for the Care and Use of Laboratory Animals. Animal care was provided in accordance with the procedures outlined in the "Guide for Care and Use of Laboratory Animals" (National Research Council; 1996; National Academy Press; Washington, D.C.).</p>