<p>Detailed GWAS information including sample size, ethnicity and genotyping platform was described in <bold><xref ref-type="supplementary-material" rid="pgen.1007040.s012">S4 Table</xref></bold> and <bold><xref ref-type="supplementary-material" rid="pgen.1007040.s001">S1 Text</xref></bold>. Briefly, p-values of qualified single nucleotide polymorphisms (SNPs) at minor allele frequency &gt; 0.05 and imputation quality &gt; 0.3 for CVD and T2D were collected for all available GWAS datasets (WHI-SHARe, WHI-GARNET, JHS, FHS, CARDIoGRAMplusC4D [<xref ref-type="bibr" rid="pgen.1007040.ref019">19</xref>], and DIAGRAM [<xref ref-type="bibr" rid="pgen.1007040.ref020">20</xref>]). SNPs meeting the following criteria were further filtered out: 1) ranked in the bottom 50% (weaker association) based on disease association p-values and 2) in strong linkage disequilibrium (LD) (r2 &gt; 0.5) according to ethnicity-specific LD data from Hapmap V3 [<xref ref-type="bibr" rid="pgen.1007040.ref053">53</xref>] and 1000 Genomes[<xref ref-type="bibr" rid="pgen.1007040.ref054">54</xref>]. For each GWAS dataset, LD filtering was conducted by first ranking SNPs based on the association p values and then checking if the next highest ranked SNP was in LD with the top SNP. If the r<sup>2</sup> was above 0.5, the SNP with lower rank was removed. The step was repeated by always checking if the next SNP was in LD with any of the already accepted ones.</p><p>Using the Weighted Gene Co-expression Network Analysis (WGCNA)[<xref ref-type="bibr" rid="pgen.1007040.ref055">55</xref>], we constructed gene co-expression modules capturing significant co-regulation patterns and functional relatedness among groups of genes in multiple CVD- or T2D-related tissues (including aortic endothelial cells, adipose, blood, liver, heart, islet, kidney, muscle and brain) obtained from nine human and mouse studies (<bold><xref ref-type="supplementary-material" rid="pgen.1007040.s013">S5 Table</xref></bold>). Modules with size smaller than 10 genes were excluded to avoid statistical artifacts, yielding 2,672 co-expression modules. These coexpression modules were used as a collection of data-driven sets of functionally connected genes for downstream analysis. The potential biological functions of each module were annotated using pathway databases Reactome and KEGG, and statistical significance was determined by Fisher&#8217;s exact test with Bonferroni-corrected p&lt; 0.05.</p><p>eQTLs establish biologically meaningful connections between genetic variants and gene expression, and could serve as functional evidence in support of the potential causal role of candidate genes in pathogenic processes[<xref ref-type="bibr" rid="pgen.1007040.ref056">56</xref>, <xref ref-type="bibr" rid="pgen.1007040.ref057">57</xref>]. We therefore conducted comprehensive curation for significant eQTLs in a total of 19 tissues that have been identified by various consortia (including the Genotype-Tissue Expression (GTEx) [<xref ref-type="bibr" rid="pgen.1007040.ref058">58</xref>], Muther [<xref ref-type="bibr" rid="pgen.1007040.ref059">59</xref>] and Cardiogenics [<xref ref-type="bibr" rid="pgen.1007040.ref060">60</xref>], and additional independent studies; <bold><xref ref-type="supplementary-material" rid="pgen.1007040.s014">S6 Table</xref>)</bold>. Additional functional genomics resources from ENCODE were also curated to complement the eQTLs for SNP-gene mapping (<bold><xref ref-type="supplementary-material" rid="pgen.1007040.s001">S1 Text</xref></bold>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">MSEA was used to identify co-expression modules with over-representation of CVD- or T2D-associated genetic signals for each disease GWAS in each cohort/ethnicity in a study specific manner. MSEA takes into three input: 1) Summary-level results of individual GWAS (WHI, FHS, JHS, CARDIoGRAM+C4D, DIAGRAM) for the LD-filtered SNPs; 2) SNP-gene mapping information, which could be determined by tissue-specific cis-eQTLs, ENCODE based functional annotation and chromosome distance based annotation. Cis-eQTLs is defined as eQTLs within 1MB of the transcription starting sites of genes. For ENCODE, we accessed the Regulome database and used the reported functional interactions to map SNPs to genes by chromosomal distance. Only SNPs within 50kb of the gene region and have functional evidence in Regulome database were kept; 3) Data-driven co-expression modules from multiple human and mouse studies as described above. Tissue-specificity was determined by the tissue origins of eQTLs and ethnic specificity was determined by the ethnicity of each GWAS cohort, since the human disease genetic signals and human eQTL mapping were the main driving factors to determine the significance of the modules. MSEA employs a chi-square like statistic with multiple quantile thresholds to assess whether a co-expression module shows enrichment of functional disease SNPs compared to random chance [<xref ref-type="bibr" rid="pgen.1007040.ref014">14</xref>]. The varying quantile thresholds allows the statistic to be adoptable to studies of varying sample size and statistical power. For the list of SNPs mapped to each gene-set, MSEA tested whether the SNP list exhibited significant enrichment of SNPs with stronger association with disease using a chi-square like statistic: <inline-formula id="pgen.1007040.e001"><alternatives><graphic id="pgen.1007040.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1007040.e001.jpg" /><ns1:math id="M1"><ns1:mi>&#967;</ns1:mi><ns1:mo>=</ns1:mo><ns1:mrow><ns1:msubsup><ns1:mo stretchy="false">&#8721;</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mrow><ns1:mi>n</ns1:mi></ns1:mrow></ns1:msubsup><ns1:mrow><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>O</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mrow><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mrow><ns1:msqrt><ns1:msub><ns1:mrow><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>i</ns1:mi></ns1:mrow></ns1:msub></ns1:msqrt><ns1:mo>+</ns1:mo><ns1:mi>&#954;</ns1:mi></ns1:mrow></ns1:mfrac></ns1:mrow></ns1:mrow></ns1:math></alternatives></inline-formula>, where n denotes the number of quantile points (we used ten quantile points ranging from the top 50% to the top 99.9% based on the rank of GWAS p values), O and E denote the observed and expected counts of positive findings (i.e. signals above the quantile point), and &#954; = 1 is a stability parameter to reduce artefacts from low expected counts for small SNP sets. The null background was estimated by permuting gene labels to generate random gene sets matching the gene number of each co-expression module, while preserving the assignment of SNPs to genes, accounting for confounding factors such as gene size, LD block size and SNPs per loci. For each co-expression module, 10000 permuted gene sets were generated and enrichment P-values were determined from a Gaussian distribution approximated using the enrichment statistics from the 10000 permutations and the statistics of the co-expression module. Finally, Benjami-Hochberg FDR was estimated across all modules tested for each GWAS.</p><p>To evaluate a module across multiple GWAS studies, we employed the Meta-MSEA analysis in Mergeomics, which conducts module-level meta-analysis to retrieve robust signals across studies. Meta-MSEA takes advantage of the parametric estimation of p-values in MSEA by applying Stouffer&#8217;s Z score method to determine the meta-Z score, then converts it back to a meta P-value. The meta-FDR was calculated using Benjamini-Hochberg method. Co-expression modules with meta-FDR &lt; 5% were considered significant and included in subsequent analyses.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We used graphical gene-gene interaction networks including the GIANT networks [<xref ref-type="bibr" rid="pgen.1007040.ref031">31</xref>] and Bayesian networks (BN) from 25 CVD and T2D relevant tissue and cell types <bold>(<xref ref-type="supplementary-material" rid="pgen.1007040.s015">S7 Table</xref></bold>, <bold><xref ref-type="supplementary-material" rid="pgen.1007040.s001">S1 Text</xref></bold>) to identify KDs. If more than one dataset was available for a given tissue, a network was constructed for each dataset and all networks for the same tissue were combined as a union to represent the network of that tissue, with the consistency of each network edge across datasets coded as edge weight. The co-expression modules genetically associated with CVD or T2D identified by Meta-MSEA were mapped onto these graphical networks to identify KDs using the weighted key driver analysis (wKDA) implemented in Mergeomics [<xref ref-type="bibr" rid="pgen.1007040.ref014">14</xref>]. wKDA uniquely consider the edge weight information, either in the form of edge consistency score in the case of BNs or edge confidence score in the case of GIANT networks. Specifically, a network was first screened for suitable hub genes whose degree (number of genes connected to the hub) is in the top 25% of all network nodes. Once the hubs have been defined, their local one-edge neighborhoods, or &#8220;subnetworks&#8221; were extracted. All genes in each of the CVD and T2D-associated gene sets that were discovered by meta-MSEA were overlaid onto the hub subnetworks to see if a particular subnetwork was enriched for the genes in CVD/T2D associated gene sets. The edges that connect a hub to its neighbors are simplified into node strengths (strength = sum of adjacent edge weights) within the neighborhood, except for the hub itself. The test statistic for the wKDA is analogous to the one used for MSEA: <inline-formula id="pgen.1007040.e002"><alternatives><graphic id="pgen.1007040.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1007040.e002.jpg" /><ns1:math id="M2"><ns1:mi>&#967;</ns1:mi><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mi>O</ns1:mi><ns1:mo>&#8722;</ns1:mo><ns1:mi>E</ns1:mi></ns1:mrow><ns1:mrow><ns1:msqrt><ns1:mi>E</ns1:mi></ns1:msqrt><ns1:mo>&#8722;</ns1:mo><ns1:mi>&#954;</ns1:mi></ns1:mrow></ns1:mfrac></ns1:math></alternatives></inline-formula>, except that the values O and E represent the observed and expected ratios of genes from CVD/T2D gene sets in a hub subnetwork. In particular, <inline-formula id="pgen.1007040.e003"><alternatives><graphic id="pgen.1007040.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1007040.e003.jpg" /><ns1:math id="M3"><ns1:mi>E</ns1:mi><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>k</ns1:mi></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mrow><ns1:mi>N</ns1:mi></ns1:mrow></ns1:mfrac></ns1:math></alternatives></inline-formula> is estimated based on the hub degree <italic>N</italic><sub><italic>k</italic></sub>, disease gene set size <italic>N</italic><sub><italic>p</italic></sub> and the order of the full network <italic>N</italic>, with the implicit assumption that the weight distribution is isotropic across the network. Statistical significance of the disease-enriched hubs, henceforth KDs, is estimated by permuting the gene labels in the network for 10000 times and estimating the P-value based on the null distribution. To control for multiple testing, stringent Bonferroni adjustment was used to focus on the top robust KDs. KDs shared by CVD and T2D modules are prioritized based on the following criteria: i) Bonferroni-corrected p&lt; 0.05 in wKDA, ii) replicated by both GIANT networks and Bayesian networks, and iii) the genetic association strength between the KD subnetworks (immediate network neighbors of the KDs) and CVD/T2D in Meta-MSEA. Finally, Cytoscape 3.3.0 [<xref ref-type="bibr" rid="pgen.1007040.ref061">61</xref>] was used for disease subnetwork visualization.</p><p>We used the genetic effect direction of KDs as a proxy for probable effect direction of the KD subnetworks. For each KD, we retrieved their tissue-specific eQTLs as well as variants within 50kb of the gene region, whose genetic association information was available in both CARDIoGRAMplusC4D and DIAGRAM, the two large meta-consortia of GWAS for CVD and T2D. CVD/T2D association beta-values of mapped variants of KDs were then extracted, and the signs of beta-values were examined to ensure they were based on the same reference alleles between GWAS. Lastly, for all mapped variants on each KD, the signs of the beta-value for CVD and T2D were compared and statistical significance of the proportion of variants with similar or opposite effect direction between diseases was determined by z-test.</p><p>We chose to validate the predicted adipose subnetwork of a top ranked KD of both CVD and T2D, <italic>Cav1</italic>, in 3T3-L1 adipocytes. Cells were cultured to confluence and adipocyte differentiation was induced using MDI differentiation medium (<bold><xref ref-type="supplementary-material" rid="pgen.1007040.s001">S1 Text</xref></bold>). Two days after the initiation of differentiation, cells were transfected with 50 nM <italic>Cav1</italic> siRNAs (3 distinct siRNAs were tested and two of the strongest ones were chosen) or a scrambled control siRNA. For each siRNA, two separate sets of transfection experiments were conducted, with three biological replicates in each experiment. Two days after transfection, cells were collected for total RNA extraction, reverse transcription and quantitative PCR measurement of 12 predicted <italic>Cav1</italic> subnetwork genes and 5 random genes not within the subnetwork as negative controls (<bold><xref ref-type="supplementary-material" rid="pgen.1007040.s001">S1 Text</xref></bold>). &#946;-actin was used to normalize the expression level of target genes.</p><p>We accessed the gonadal white tissue gene expression data of 3-month-old wild type and <italic>Cav1</italic><sup><italic>-/-</italic></sup> male mice (N = 3/group) from Gene Expression Omnibus (GEO accession: GSE35431). Detailed description of the data collection procedures has been described previously [<xref ref-type="bibr" rid="pgen.1007040.ref033">33</xref>]. Gene expression was profiled using Illumina MouseWG-6 v2.0 expression beadchip and normalized using robust spline. Differentially expressed genes (DEGs) between genotype groups were identified using linear model implemented in the R package Limma and false discovery rate was estimated using the Benjamini-Hochberg procedure [<xref ref-type="bibr" rid="pgen.1007040.ref062">62</xref>]. DEGs at different statistical cutoffs were compared to <italic>CAV1</italic> subnetwork genes at different levels (i.e., 1, 2, 3, or 4 edges away from <italic>CAV1</italic>) to assess overlap and significance of overlap was evaluated using Fisher&#8217;s exact test.</p><p>To further validate the role of KD subnetworks in CVD and T2D, we incorporated genetic, genomic and transcriptomic data from HMDP (comprised of &gt;100 mouse strains differing by genetic composition) [<xref ref-type="bibr" rid="pgen.1007040.ref021">21</xref>&#8211;<xref ref-type="bibr" rid="pgen.1007040.ref023">23</xref>]. HMDP data was from two panels, one with mice fed with a high-fat diet (HF-HMDP)[<xref ref-type="bibr" rid="pgen.1007040.ref022">22</xref>], and the other with hyperlipidemic mice made by transgenic expression of human APOE-Leiden and CETP gene (ATH-HMDP)[<xref ref-type="bibr" rid="pgen.1007040.ref023">23</xref>]. For HF-HMDP, we retrieved gene-trait correlation data for adipose tissue (due to its relevance to both CVD and T2D) and 7 core cardiometabolic traits including adiposity, fasting glucose level, fasting insulin level, LDL, HDL, triglycerides and homeostatic model assessment-insulin resistance (HOMA-IR). For ATH-HMDP, we retrieved aorta gene-trait correlation (aorta tissue is the main site for CVD in mice) for all 7 traits. In addition to assessing the trait association strengths of individual KDs, we also used MSEA to evaluate the aggregate association strength of the top CVD/T2D KD subnetworks with the traits at both transcription and genetic levels through transcriptome-wide association (TWAS) and GWAS in HF-HMDP and ATH-HMDP (<bold><xref ref-type="supplementary-material" rid="pgen.1007040.s001">S1 Text</xref></bold>).</p>