<p xmlns:ns0="http://www.w3.org/1999/xlink">The generation of the FVB/NJ x [MOLF/EiJ x MMTV-PyMT] backcross was performed as previously described (Cancer Res December 15, 2001 61; 8866). Genotyping was performed by the Center for Inherited Disease Research (<ext-link ext-link-type="uri" ns0:href="http://www.cidr.jhmi.edu/">http://www.cidr.jhmi.edu/</ext-link>). QTL mapping was performed with the R/QTL program using the J/QTL interface [<xref ref-type="bibr" rid="pgen.1006267.ref015">15</xref>]. QTL peaks were considered significant if the p value was less than 0.05 after 10,000 permutations of the data to correct for genome-wide significance.</p><p>The research described in this study was performed under the Animal Study Protocol LCBG-004, approved by the NCI Bethesda Animal Use and Care Committee. Animal euthanasia was performed by cervical dislocation after anesthesia by Avertin. The Shanghai Breast Cancer Study and Shanghai Breast Cancer Survival Study were approved by the institutional review boards of Vanderbilt University, the Shanghai Cancer Institute, and the Shanghai Center for Disease Prevention and Control, and written informed consent was obtained from all participants.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The <italic>Arntl2</italic> knockout mouse strain used for this research project was created from ES cell clone (Arntl2_F05) originally generated by the Wellcome Trust Sanger Institute. ES cells containing a targeted, non-conditional allele (<ext-link ext-link-type="uri" ns0:href="http://www.mousephenotype.org/data/alleles/MGI:2684845/tm1e%28KOMP%29Wtsi">tm1e(KOMP)Wtsi</ext-link>) for Arntl2 were obtained from the KOMP (Knock Out Mouse Project) repository and knock out mice were generated in a C57BL/6 background by SAIC-Frederick (Frederick, MD). Mice were genotyped using cassette-specific primers.</p><p>Mouse mammary carcinoma cell lines 4T1 and 6DT1 were a generous gift from Dr. Lalage Wakefield (NCI, Bethesda, MD). All cell lines were cultured in Dulbecco&#8217;s Modified Eagle Medium (DMEM), supplemented with 10% Fetal Bovine Serum (FBS), 1% Penicillin and Streptomycin (P/S) and 1% Glutamate, and maintained in 37&#176;C degrees with 5% CO2. Short hairpin RNA (shRNA)-mediated knockdown and overexpression cells were cultured in the same conditions with an addition of 10ug/ml puromycin and 5ug/ml blasticidin, respectively.</p><p>TRC lentiviral shRNA constructs against <italic>Arntl2</italic> were obtained from Open Biosystems (now Dharmacon) as glycerol stocks (RHS4533-EG56938). The sequences for all shRNA constructs were as follows:</p><p>C7- AAGTTTGTCCAGTTTACGCGC</p><p>C8- ATTCCAACAATATTTGGAGGC</p><p>C9- TTGTGAAACTAAACCATTGGC</p><p>C10- TTGAAAGAAGATAGGTAAGGG</p><p>C11- ATCTGTTCCAATACTACCAGC</p><p><italic>Overexpression</italic>. MGC Mouse Arntl2 cDNA was purchased from Open Biosystems (now Dharmacon) as a glycerol stock (Accession: BC108965 Clone ID: 40047159). Using directional TOPO-cloning primers Fwd- 5&#8217;-CACC-ATGGAGTTTCCAAGGAAACGCA-3&#8217; and Rev- 5&#8217;-GAGTGCCCACTGGATGTCAC-3&#8217;, Arntl2 cDNA was amplified using Phusion polymerase. DNA was purified using QiaQuick Gel extraction kit (Qiagen) and ligated into a Gateway entry clone using the LR reaction according to the manufacturer&#8217;s protocol (Invitrogen).</p><p>1.1 x 10<sup>6</sup> 293T cells were plated in 6 cm dishes 24 hours prior to transfection in P/S-free 10% FBS DMEM media. Cells were transfected with 1ug of shRNA/cDNA and 1ug of viral packaging plasmids (250ng pMD2.G and 750ng psPAX2) using 6ul of Xtreme Gene 9 transfection reagent (Roche). After 24 hours of transfection, media was refreshed with 10% DMEM, supplemented with 1% P/S and 1% Glutamine. The following day, virus-containing supernatant was passed through a 45um filter to obtain viral particles, which were then transferred to 100,000 4T1/6DT1 cells. 24 hours post-transduction the viral media was removed and fresh 10% DMEM was added. Finally, 48 hours after transduction, the cells were selected with 10ug/ml puromycin- or 5ug/ml blasticidin-containing complete DMEM.</p><p>One day prior to scratch assay, 25,000 cells/well were plated in triplicates in an Essen ImageLock 24-well plate and allowed to grow to confluence. On the day of the assay, cells were treated with 10ug/ml Mitomycin C (Sigma) for 3&#8211;4 hours to inhibit cell proliferation. Scratch wounds were made using an Essen 4-channel scratch instrument loaded with Eppendorf 10uL micropipette tips and displaced cells and debris removed by washing with Phosphate-buffered saline (PBS) three times. The cells were placed into the IncuCyte Kinetic Live Cell Imaging System (Essen BioScience) in complete DMEM media and cell motility imaged for 24 hours.</p><p>Cells were counted and 5000 cells/well were plated in quadruplicates in 24-well cell culture plates (Corning, Inc.) and placed into the IncuCyte Kinetic Live Cell Imaging System (Essen BioScience) and programmed to image each well at 2-hour intervals. Samples were imaged until they reached 100% confluence. Data analysis was conducted using IncuCyte 2011A software.</p><p>RNA was isolated cell lines using TriPure (Roche) and reverse transcribed using iScript (Bio-Rad). Real-Time PCR was conducted using VeriQuest SYBR Green qPCR Master Mix (Affymetrix). Peptidylprolyl isomerase B (Ppib) was used for normalization of expression levels. Expression of mRNA was defined from the threshold cycle, and relative expression levels were calculated using 2- deltaCt after normalization with Ppib. Primer sequences were as follows:</p><p><italic>Arntl2</italic> Fwd: GTCTTCCCCAGAATCCCTTT; <italic>Arntl2</italic> Rev: TTGTCTCTCCGACGCTTTTC <italic>Stk38l</italic> Fwd: TTCCTATGAGCAACCATACCCG; <italic>Stk38l</italic> Rev: TCTAGGCCAAGTCTGGTCCTC; <italic>Ppib</italic> Fwd: GGAGATGGCACAGGAGGAAAGAG; <italic>Ppib</italic> Rev: TGTGAGCCATTGGTGTCTTTGC</p><p>Protein lysate from one million cells were extracted on ice using Golden Lysis Buffer (10 mM Tris pH 8.0, 400 mM NaCl, 1% Triton X-100, 10% Glycerol+Complete protease inhibitor cocktail (Roche), phosphatase inhibitor (Sigma)). Protein concentration was measured using Pierce&#8217;s BCA Protein Assay Kit and analyzed on the Versamax spectrophotometer at a wavelength of 560nm. Appropriate volumes containing 20ug of protein lysates combined with NuPage LDS Sample Buffer and NuPage Reducing Agent (Invitrogen) were run on 4&#8211;12% (or otherwise indicated) NuPage Bis-Tris gels in MOPS buffer. Proteins were transferred onto a PVDF membrane (Millipore), blocked in 5% milk (TBST + dry milk) for one hour and incubated in the primary antibody (in 5% milk) overnight at 4&#176;C. Membranes were washed with 0.05% TBST (TBS + 5% Tween) and secondary antibody incubations were done at room temperature for one hour. Proteins were visualized using Amersham ECL Prime Western Blotting Detection System and Amersham Hyperfilm ECL (GE Healthcare).</p><p>The following primary antibodies were used: mouse anti-Actin (1:10,000; Abcam), mouse anti-myc-tag (1:1000; Cell Signaling). Secondary antibodies goat anti-rabbit (Santa Cruz) and goat-anti-mouse (GE Healthcare) were used at concentrations of 1:10,000.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Single-guided RNA (sgRNA) against the <italic>Arntl2</italic> regulatory region was designed using the Massachusetts Institute of Technology CRISPR algorithm (crispr.mit.edu). The top sgRNA with the following binding sites was selected: 5&#8217;-GGAATCCCCCTCGCGACCGT-3&#8217;. sgRNAs were annealed and ligated into the pSpCas9(BB)-2A-Puro (PX459) (Addgene; 48139) vector (Nat Protoc. 2013 Nov;8(11):2281&#8211;308. doi: <ext-link ext-link-type="uri" ns0:href="http://dx.doi.org/10.1038/nprot.2013.143">10.1038/nprot.2013.143</ext-link>).</p><p>To create single CRISPR clones, 6DT1 cells were plated in 10cm dishes and transfected with 5ug sgRNA-containing PX459 vector and 5ug of linearized 900bp promoter region. Cells were transfected with Xtreme Gene 9 and 24 hours after transfection, cells were FACS-sorted by GPF fluorescence. A total of 1000 cells were sorted and single cell clones were manually plated into 96-well plates. Clones were allowed to grow up until they reached confluence and then gDNA was isolated to check for integration of the MOLF region by PCR-amplification with the following primers: Fwd- 5&#8217;-TAAGCA-ACGCGT-GGGCTGGCTAGGGCTG-3&#8217; and Rev- 5&#8217;- TGCTTA-AGATCT-TACAAGAGAGTTGACAGGTCCAG-3&#8217;. PCR-amplified DNA was purified using QIAquick PCR Purification Kit (Qiagen) and digested using MseI at 37&#176;C overnight.</p><p>Female virgin FVB/nJ or BALB/cJ mice were obtained from Jackson Laboratory at 6&#8211;8 weeks of age. Two days prior to in vivo experiments, cells were plated at one million cells/condition into T-75 flasks (Corning) in non-selective DMEM. A total of 100,000 cells per mouse was injected into the fourth mammary fat pad of FVB/nJ or BALB/cJ mice for 6DT1 or 4T1 cells, respectively. For experimental metastasis assays, 100,000 4T1 tumor cells were injected into the tail vein of BALB/cJ mice. The mice were euthanized between 28&#8211;30 days post-injection. Primary tumor was resected, weighed and lung metastases counted.</p><p>The MOLF and FVB promoter regions were amplified using strain-specific gDNA and the following primers: Fwd /5BiosG/GAAGGTCCACACCCTCTTGC and Rev- /5BiosG/CCTGGACTTGGCCATTGGAA. Phusion taq polymerase was used according to the manufacturer&#8217;s recommendation with the following PCR conditions: 98&#176;C for 15 sec, 35 cycles of 98&#176;C for 10 sec, 72&#176;C for 1.5 min followed by 72&#176;C for 10 min and a final step of holding in 4&#176;C. PCR products were purified and 5ug of DNA was used for each pull-down experiment along with nuclear lysate from 6DT1 cells (FVB background).</p><p>Nuclear lysate was isolated using Pierce&#8217;s NER Extraction kit according to the provided protocol. Following nuclear extraction, 200ug of lysate was incubated with 5ug of biotinylated DNA probes along with 100ul of streptavidin magnetic beads (Dynabeads M270, Thermo-Fisher (former Invitrogen)). The final volume of 500ul was adjusted using NER buffer from the extraction kit. The mixture was placed on a rotator and incubated at room temperature for 1 hour ([<xref ref-type="bibr" rid="pgen.1006267.ref036">36</xref>]). The samples were placed on a magnetic stand and washed with ice cold PBS three times followed by one wash with NER buffer. The beads were resuspended in 25ul of 2x SDS buffer, boiled at 95&#176;C for 5 min and the proteins were separated on a 4&#8211;12% SDS-Page minigel (Invitrogen). The gel was stained using Pierce&#8217;s Silver Stain for Mass Spectrometry according to the manufacturer&#8217;s protocol. Protein bands were sent to the ATRF facility at NCI&#8211;Frederick and identified via mass spectrometry.</p><p>For validation of the pulldown, the same conditions were used to bind the probes to the beads. The beads were washed with PBS three times followed by one wash with NER buffer. The proteins were separated on a 7% Tris-acetate SDS-Page minigel (Invitrogen) and transferred onto a PVDF membrane at 4&#176;C for 1 hour. The membranes were probed with the following antibodies overnight at 4&#176;C: rabbit-anti-PARP1 (Santa Cruz; 1:5000), rabbit-anti-RRP1B (Santa Cruz; K-19: 1:1000) and rabbit-anti-CTCF (Cell Signaling; 1:1000). Secondary antibody goat anti-rabbit (Santa Cruz) was used at concentrations of 1:10,000 at room temperature for 1 hour.</p><p>Samples included in the epidemiological study came from participants of the Shanghai Breast Cancer Study (SBCS) and Shanghai Breast Cancer Survival Study (SBCSS). Details on the methodology of the parent studies have been described previously [<xref ref-type="bibr" rid="pgen.1006267.ref037">37</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006267.ref039">39</xref>]. Briefly, the SBCS is a population-based, case-control study that recruited 3,448 incident breast cancer patients and controls in urban Shanghai between August 1996 and March 1998 and again between April 2002 and February 2005; 90.6% of participants provided a blood or exfoliated buccal cell sample [<xref ref-type="bibr" rid="pgen.1006267.ref037">37</xref>]. The SBCSS, also conducted in urban Shanghai, recruited 5,042 breast cancer patients between March 2002 and April 2006 with 98% of patients providing an exfoliated buccal cell sample [<xref ref-type="bibr" rid="pgen.1006267.ref038">38</xref>]. All participants of both studies provided written informed consent before participating in the study and the Institutional Review Boards of all institutes involved approved the study protocols. Medical charts for breast cancer patients were reviewed to verify cancer diagnosis and obtain tumor characteristic (including estrogen receptor, ER) and treatment information. Cancer patients have been followed for survival status and breast cancer recurrence through a combination of record linkages with the Shanghai Vital Statistics Registry and in-person surveys. A total of 1,758 women with estrogen receptor negative breast cancer and DNA samples from the SBCS and SBCSS were included in the current study. Among them, 726 samples were genotyped using the Affymetrix Genome-Wide Human SNP Array 6.0 [<xref ref-type="bibr" rid="pgen.1006267.ref039">39</xref>] and imputed according to the 1000 Genomes Project Asian data. Genotype from the remaining 1032 samples were genotyped using the iPLEX Sequenom MassARRAY platform. A total of 137 disease progression events were observed in the GWAS patients and 193 events were observed in the Sequenom genotyped patients.</p>