<p>The yeast <italic>S</italic>. <italic>cerevisiae</italic> strains are derivatives of BY4742 (<italic>MAT</italic>&#945; <italic>his3</italic>&#916;<italic>1 leu2</italic>&#916;<italic>0 lys2</italic>&#916;<italic>0 ura3</italic>&#916;<italic>0)</italic>. The wild-type strains are BKDY155 (<italic>MAT</italic>&#945; <italic>his3</italic>&#916;<italic>1 leu2</italic>&#916;<italic>0 lys2</italic>&#916;<italic>0 ura3</italic>&#916;<italic>0 hmr</italic>::<italic>URA3 hml</italic>::<italic>HphMX</italic>), BKDY157 (<italic>MAT</italic>&#945; <italic>his3</italic>&#916;<italic>1 leu2</italic>&#916;<italic>0 lys2</italic>&#916;<italic>0</italic> Chr V::<italic>URA3 hml</italic>::<italic>HphMX</italic>), BKDY438 (<italic>MAT</italic>&#945; <italic>his3</italic>&#916;<italic>1 leu2</italic>&#916;<italic>0 lys2</italic>&#916;<italic>0 ura3</italic>&#916;<italic>0 hml</italic>::<italic>URA3</italic>), BKDY541 (<italic>MAT</italic>&#945; <italic>his3</italic>&#916;<italic>1 leu2</italic>&#916;<italic>0 lys2</italic>&#916;<italic>0 ura3</italic>&#916;<italic>0</italic> Chr VII-L::<italic>URA3</italic>), BKDY834 (<italic>MAT</italic>&#945; <italic>his3</italic>&#916;<italic>1 leu2</italic>&#916;<italic>0 lys2</italic>&#916;<italic>0 ura3</italic>&#916;<italic>0 can1</italic>::<italic>LEU2 hmr</italic>::<italic>CAN1</italic>), and FKY1292 (<italic>MAT</italic>&#945; <italic>his3</italic>&#916;<italic>1 leu2</italic>&#916;<italic>0 lys2</italic>&#916;<italic>0</italic> Chr V::<italic>URA3</italic>). In BKDY157 and FKY1292, a DNA sequence between nucleotides 115,949 and 117,045 of Chr V is replaced with <italic>URA3</italic>. Each of the mutant strains is isogenic to one of the wild-type strains. In the <italic>hmr-E</italic>&#916; strain, the 56-bp <italic>HMR-E</italic> region (Chr III 292,674&#8211;292,729) [<xref ref-type="bibr" rid="pgen.1007074.ref087">87</xref>] was replaced with a <italic>LEU2</italic> cassette. To create the gene deletions, PCR-amplified disruption cassettes were introduced into yeast cells by lithium acetate/PEG4000/DMSO transformation. The presence of each gene deletion was confirmed by locus and disruption cassette-specific PCRs. The <italic>pol2-4</italic> mutation was introduced into the chromosomal <italic>POL2</italic> gene using the integration-excision method, and the <italic>sir2-N345A</italic> mutation was inserted into the chromosomal <italic>SIR2</italic> gene utilizing a previously described technique [<xref ref-type="bibr" rid="pgen.1007074.ref102">102</xref>].</p><p>The spontaneous mutation rates were measured using a fluctuation test. At least 9&#8211;18 cultures, which were started from single colonies of two-four independent isolates of the same genotype, were used to determine the spontaneous mutation rate. The cultures were grown to saturation in 3 ml YPDAU medium (1% yeast extract, 2% bacto-peptone, 2% dextrose, 60 mg/L adenine, 60 mg/L uracil) at 30&#176;C. The saturated cultures were diluted in sterile water, and appropriate dilutions were plated on a synthetic complete (SC) medium to determine the total number of cells in the cultures and on a selective medium to determine the total number of the mutant cells in the cultures. Unless noted otherwise, the selective medium for FOA<sup>R</sup> cells was a SC medium containing 1 g/L 5-FOA and 5 mM NAM (SC + 5-FOA + NAM), and the selective medium for Can<sup>R</sup> cells was a SC medium that lacked arginine and contained 60 mg/L L-canavanine and 5 mM NAM. The plates were incubated for 3&#8211;4 days at 30&#176;C, and the colonies were counted. 5&#8211;70% of the FOA<sup>R</sup> colonies grew on a SC&#8212;Ura + 5 mM NAM medium. FOA<sup>R</sup> cells, which formed these colonies, were excluded from calculations of the mutation rates. To identify FOA<sup>R</sup> colonies that were Ura<sup>+</sup>, FOA<sup>R</sup> colonies formed on the fluctuation test plates were replica-plated onto the SC&#8212;Ura + 5 mM NAM medium, and the plates were incubated for 1 day at 30&#176;C.</p><p>The mutation rates were calculated from the total numbers of cells and mutants in the cultures using the Drake&#8217;s formula <italic>&#956;</italic> = <italic>&#402;</italic>/ln(<italic>N&#956;</italic>) [<xref ref-type="bibr" rid="pgen.1007074.ref103">103</xref>], where &#956; is mutation rate per replication, &#402; is the median mutant frequency, and <italic>N</italic> is population size.</p><p>Where indicated, the significance of the observed differences in the mutation rates was assessed with the Mann-Whitney U two-tailed test (GraphPad Prism 6 software), in which the null hypothesis is that there is no difference between the two data sets.</p><p>To examine the relationship between the nominal variables of spectra and mutation type (<bold><xref ref-type="table" rid="pgen.1007074.t002">Table 2</xref></bold>), categorical variables were summarized with frequencies and percentages, and a &#967;2 test of independence with a Bonferroni correction for multiple comparisons was utilized.</p><p>In a different method, a Monte Carlo modification of the Pearson &#967;2 test of spectra homogeneity [<xref ref-type="bibr" rid="pgen.1007074.ref082">82</xref>] was used to compare mutation distributions (<bold><xref ref-type="supplementary-material" rid="pgen.1007074.s002">S2 Table</xref></bold>). The calculations were done using the COLLAPSE program [<xref ref-type="bibr" rid="pgen.1007074.ref104">104</xref>].</p><p>In order to determine the <italic>ura3</italic> mutation spectrum, ~50&#8211;100 patches each started from a different single colony were grown on YPDAU plates (1% yeast extract, 2% bacto-peptone, 2% dextrose, 60 mg/L adenine, 60 mg/L uracil, 2% agar). The patches were next replica-plated on the SC + 5-FOA + 5 mM NAM, followed by incubation of the plates for 1 day at 30&#176;C. The patches that were formed on the SC + 5-FOA + 5 mM NAM plates were replica-plated on fresh SC + 5-FOA + 5 mM NAM plates, and the plates were incubated for 2&#8211;3 days at 30&#176;C. A single FOA<sup>R</sup> colony was randomly selected from each patch, purified on a SC + 5-FOA + 5 mM NAM plate, and propagated on a YPDAU plate. The patches were then replica-plated on SC&#8212;Ura + 5 mM NAM plates. Patches that grew on the SC&#8212;Ura + 5 mM NAM plates were not analyzed further. Genomic DNAs of the remaining FOA<sup>R</sup> patches were isolated with a MasterPure Yeast DNA purification kit (Epicentre). Each of these genomic DNAs was used as a template to PCR-amplify a 1.4-kb DNA fragment encompassing the entire length of <italic>ura3</italic> ORF with primers #1 (5&#8217;- GAGAATAAGCGCAGGTACTCCTG -3&#8217;) and #2 (5&#8217;- CGCCATATACGAAAATGTTGGTG -3&#8217;). The amplified DNA fragments were purified with a PCR purification kit (Thermo Fisher) and sequenced to determine <italic>ura3</italic> mutations.</p>