<p>We thank the Pachyonychia Congenita Project and Ms. Holly Evans for providing us with clinical information and extracted third molars from a PC patient (20040468&#8211;1057496), and the NIDCR dental clinic for providing extracted third molars from healthy patients (NCT01805869).</p><p>For the COHRA study, written informed consent was provided by all adult participants, and verbal assent with parental written consent was provided by all child participants. All procedures, forms and protocols were approved by the Institutional Review Boards of the University of Pittsburgh and West Virginia University.</p><p>Written informed consent was obtained from the pachyonychia congenita patient, as part of a research registry approved by an institutional review board that complies with all principles of the Helsinki Accord (Western IRB study no. 20040468).</p><p>All animal work was approved by the NIAMS Animal Care and Use Committee.</p><p>The Center for Oral Health Research in Appalachia (COHRA) study was initiated to investigate the community-, family-, and individual-level contributors to oral health [<xref ref-type="bibr" rid="pgen.1007168.ref036">36</xref>]. Participants from rural counties of Pennsylvania and West Virginia were enrolled via a household-based recruitment strategy, with eligible households required to include at least one biological parent-child pair. All other members of eligible households were invited to participate without regard to biological or legal relationships, or oral health status. Written informed consent was provided by all adult participants, and verbal assent with parental written consent was provided by all child participants. All procedures, forms and protocols were approved by the Institutional Review Boards of the University of Pittsburgh and West Virginia University.</p><p>Intra-oral examinations of all participants were performed by licensed dentists and/or research dental hygienists. Each surface of each tooth (excluding third molars) was examined for evidence of decay, from which dental caries indices were generated. Three measurements of caries experience were considered: (1) the number of surfaces with untreated decay (DS/ds); (2) the traditional DMFS/dfs indices which represent the number of decayed (D/d), missing due to decay (M), and filled (F/f) tooth surfaces (S/s) in the permanent (DMFS) and primary (dfs) dentitions; and (3) the partial DMFS and dfs indices limited to the molars and premolar pit and fissure surfaces which are at high risk of decay. DNA samples were collected via blood, saliva or buccal swab. Genotyping for approximately 600,000 polymorphisms was performed by the Center for Inherited Disease Research at Johns Hopkins University using the Illumina Human610-Quadv1_B BeadChip (Illumina). Extensive data cleaning and quality assurance analyses were performed as previously reported [<xref ref-type="bibr" rid="pgen.1007168.ref004">4</xref>]. Imputation of approximately 16 million unobserved genetic variants was performed using the 1000 Genome Project (phase 1 June 2011 release) as reference. In brief, pre-phasing of haplotypes was performed via SHAPEIT2 [<xref ref-type="bibr" rid="pgen.1007168.ref037">37</xref>] and imputation was performed via IMPUTE2 [<xref ref-type="bibr" rid="pgen.1007168.ref038">38</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Linear regression was used to test the association of dental caries experience with genetic polymorphisms under the additive genetic model while adjusting for age and sex. Pairwise SNP-by-SNP interaction effects were tested in the same modeling framework by including main effects of each SNP and their product term for selected variants within the same gene region, along with age and sex. Analyses were performed separately for dental caries in the permanent dentition of adults (ages 25&#8211;50 years) and the primary dentition of children (ages 6&#8211;12 years). All analyses were limited to self-reported non-Hispanic whites (mixed European descent); self-reported race was consistent with genetically-determined ancestry. Analyses were performed using PLINK (v1.9) (<ext-link ext-link-type="uri" ns0:href="http://www.cog-genomics.org/plink/1.9/">http://www.cog-genomics.org/plink/1.9/</ext-link>)[<xref ref-type="bibr" rid="pgen.1007168.ref039">39</xref>] and R (R Foundation for Statistical Computing).</p><p>Third molars were obtained from a patient involved in the International Pachyonychia Congenita Research Registry (IPCRR), under the IRB Protocol number 20040468 and IRB Study number 1057496. Third molars from healthy patients were obtained from the NIDCR OP-1 Dental Clinic that were collected under the IRB Protocol number NCT01805869.</p><p>RNA-seq analysis was performed as described previously [<xref ref-type="bibr" rid="pgen.1007168.ref040">40</xref>]. Briefly, mandibles were dissected from P10 mice, transferred to RNAlater solution (Life Technologies) and stored at 4&#176;C. Enamel organs were dissected from mandibles and homogenized in Trizol reagent (Invitrogen) using the Precellys 24 (Bertin Technologies). Total RNA was extracted and further purified using the RNAeasy mini kit (Qiagen). RNA-seq analysis was performed using the Mondrian SP kit (Illumina) and the Illumina HiSeq 2000 sequencing system.</p><p>Rat mandibles at postnatal day 10 were fixed overnight at 4&#176;C in 4% paraformaldehyde in 1X PBS, dehydrated and embedded in paraffin and 10 &#956;m-thick sections were prepared. Immunohistochemical analysis was performed using a blocking solution containing 5% goat serum and 7.5% BlockHen II (Aves Labs, Tigard, OR) in 1X PBS. Enzymatic antigen retrieval was performed using Ultravision Trypsin (Thermo Fisher Scientific, Waltham MA). Primary antibodies used: Guinea-pig anti-K6 (Progen Biotechnik GmbH, Germany), guinea-pig anti-K17 (Progen Biotechnik GmbH, Germany). Alexa-488 anti-guinea-pig (Thermo Fisher Scientific, Waltham MA) was used as secondary antibody. Images were acquired on a Leica LS5 confocal microscope (Leica Microsystems Inc., Buffalo Grove, IL).</p><p>Ground, polished and etched human teeth were stained with guinea-pig anti-K6 (Progen Biotechnik GmbH, Germany), mouse anti-K6 (Abcam, Cambridge MA), guinea-pig anti-K16 (Progen Biotechnik GmbH, Germany) or guinea-pig anti-K17 (Progen Biotechnik GmbH, Germany) antibody. Alexa-488 anti-guinea-pig and Alexa-555 anti-mouse (Thermo Fisher Scientific, Waltham MA) were used as secondary antibodies.</p><p>Ground, polished and etched human teeth were prepared for SEM as described previously [<xref ref-type="bibr" rid="pgen.1007168.ref019">19</xref>]. Samples were fixed overnight at 4&#176;C in 2% glutaraldehyde, 2% PFA in 0.1M phosphate buffer pH 7.4 and dehydrated through a series of 50%, 70%, 95% and 100% ethanol solutions. They were incubated for 10 min in hexamethyldisilazane, air-dried for 30 min, mounted on aluminum specimen mount stubs covered with conductive carbon adhesive tabs (Electron Microscopy Sciences, Hatfield, PA), sputter-coated with gold and analyzed under a Field Emission Scanning Electron Microscope S4800 (Hitashi, Toronto, Canada) at 10 kV.</p><p>Micro-CT analysis of fixed molars was performed as described previously [<xref ref-type="bibr" rid="pgen.1007168.ref019">19</xref>] using the Skyscan 1172 desktop X-ray microfocus CT scanner and the following parameters: 0.5mm AI + 0.1mm Cu filters, 100 kV, 100 uA, 8.00 micron resolution, 0.4 degrees rotation step over 180 degrees). CTvox software (Bruker microCT) was used for 3D reconstruction.</p><p>The cDNAs encoding the, K6a<sup>N21S</sup>, K6a<sup>G97R</sup>, K6a<sup>S143N</sup>, K6a<sup>Y497C</sup> and K6a<sup>N171K</sup> mutant proteins were generated by site-directed mutagenesis using the QuikChange Site-directed Mutagenesis Kit (Stratagene). The following primers were used: N21S-forward: GGGGTTTCAGTGCCAgCTCAGCCAGGC, N21S-reverse: GCCTGGCTGAGcTGGCACTGAAACCCC, G97R-forward: GGCTTTGGTGGCGCCaGGAGTGGATTGG, G97R-reverse: CCAATCCACTCCtGGCGCCACCAAAGCC, S143N-forward: GTCAACCAGAaTCTCCTGACTCCCCTC, S143N-reverse: GAGGGGAGTCAGGAGAtTCTGGTTGAC, Y497C-forward: CCGTCTCCAGTGGCTgTGGCGGTGCCAG, Y497C-reverse: CTGGCACCGCCAcAGCCACTGGAGACGG, N171K-forward: GATCAAGACCCTCAAaAACAAGTTTGCC, N171K-reverse: GGCAAACTTGTTtTTGAGGGTCTTGATC (lower cases indicate the position of the mutations). The pBi4-GFP vector was used to simultaneously express the reporter protein EGFP with K6a<sup>WT</sup>, K6a<sup>N21S</sup>, K6a<sup>G97R</sup>, K6a<sup>S143N</sup>, K6a<sup>Y497C</sup>, K6a<sup>S143N-Y497C</sup>, or K6a<sup>N171K</sup> under control of a unique tetracycline responsive element.</p><p>Murine Ameloblast-like cells (ALC) were kindly provided by Dr. Sugiyama [<xref ref-type="bibr" rid="pgen.1007168.ref026">26</xref>] and used to produce a tetracycline inducible ameloblast cell line. These cells were stably transfected with a prtTA2-M2/IRES-Neo plamid obtained after subcloning of the rtTA2-M2 cassette [<xref ref-type="bibr" rid="pgen.1007168.ref041">41</xref>] into the pCMV-IRES-Neo (Clontech). rtTA-M2 is a mutagenized form of rtTA that shows a lower basal activity and a higher sensitivity to doxycycline (Dox) than the original rtTA [<xref ref-type="bibr" rid="pgen.1007168.ref041">41</xref>]. The presence of the IRES cassette before the neomycin (Neo) resistance gene allowed coexpression of the rtTA-M2 transactivator and the Neo resistance gene, increasing the chance to select clones that express sufficient amounts of the transactivator in Neo-resistant cells. Clones were isolated and functionality of the tet system was screened by transient transfection with pTRE2-luc (expression of luciferase under the control of tetracycline responsive element). Cells were grown in the presence or absence of 2 ug/ml Dox and luciferase activity was estimated. One clone exhibiting a low basal activity of the transactivator in the absence of Dox and a strong induction in the presence of Dox (+Dox/-Dox ratio) was selected for subsequent experiments and named ALC-TetON. ALC-TetON cells were grown in Dulbecco&#8217;s modified Eagle&#8217;s medium (10% fetal bovine serum, 1% penicillin/streptomycin and 1 ug/ml G418). For transfections, the cells were grown to at least 70% confluence. 2 million cells were used per transfection with either the pBi4-GFP/K6a<sup>WT</sup>, pBi4-GFP/K6a<sup>N21S</sup>, pBi4-GFP/K6a<sup>G97R</sup>, pBi4-GFP/K6a<sup>S143N</sup>, pBi4-GFP/K6a<sup>Y497C</sup>, pBi4-GFP/K6a<sup>S143N-Y497C</sup>, or pBi4-GFP/K6a<sup>N171K</sup> plasmid and the pCMV-K16<sup>WT</sup> plasmid (Amaxa Nucleofactor).</p><p>Transfected cells were seeded on glass coverslips coated with 0.1% gelatin and immediately induced with 2ug/ml doxycycline. Twenty-four hours after induction, cells were washed three times in 1X PBS and fixed with 4% paraformaldehyde in PBS for 15 min at room temperature. A 5 min incubation in 0.2% Triton in 1X PBS was used to permeabilize the cells before blocking unspecific sites using 3% BSA in PBS for 1 h. Primary antibodies diluted in blocking solution were applied for 1 h. Primary antibody used: guinea-pig anti-K6 (Progen). Secondary antibodies diluted in blocking solution were applied for 30 min. Secondary antibodies used: Alexa Fluor 555 anti-guinea pig IgG (Life Technologies). Nuclei were stained using DAPI and coverslips were mounted on glass slides using Mowiol (Calbiochem). Images were acquired using a LEICA Sp5 confocal microscope.</p>