<p>The variant rs10166942 is located ~1 kb upstream of the <italic>TRPM8</italic> gene. We used a combination of bioinformatics tools to investigate possible functional effects of rs10166942 and it neighboring variants in high linkage disequilibrium (LD). We explored the predicted effects on protein sequence using variant effect predictor (VEP) [<xref ref-type="bibr" rid="pgen.1007298.ref059">59</xref>], focusing on the non-synonymous and splice-site SNPs, as well as indels annotated in the 1KGP. We explored effects on gene expression using Regulome DB annotations [<xref ref-type="bibr" rid="pgen.1007298.ref060">60</xref>], GTEx data [<xref ref-type="bibr" rid="pgen.1007298.ref061">61</xref>] and basal root ganglion RNA-Seq data (kindly provided by G. Gisselmann) [<xref ref-type="bibr" rid="pgen.1007298.ref062">62</xref>].</p><p>To investigate the patterns of genetic diversity of <italic>TRPM8</italic> we used genome-wide genotype data from the 1KGP phase III [<xref ref-type="bibr" rid="pgen.1007298.ref034">34</xref>]. African ancestry: ESN (Esan in Nigeria), GWD (Gambian (Mandinka) in Western Divisions in Gambia), YRI (Yoruba in Ibadan, Nigeria), LWK (Luhya in Webuye, Kenya), MSL (Mende in Sierra Leone), ASW (African Ancestry in Southwest USA), ACB (African Caribbean in Barbados); European ancestry: GBR (British from England and Scotland), CEU (Utah Residents, USA, with Northern and Western European ancestry), FIN (Finnish from Finland), TSI (Toscani in Italia), IBS (Iberian Populations in Spain); East Asian ancestry: CHS (Southern Han Chinese), CHB (Han Chinese in Beijing, China), JPT (Japanese in Toyko, Japan), CDX (Chinese Dai in Xishuangbanna, China), KHV (Kinh in Ho Chi Minh City, Vietnam); South Asian ancestry: BEB (Bengali in Bangladesh), GIH (Gujarati Indians in Houston, USA), ITU (Indian Telugu in the UK), PJL (Punjabi in Lahore, Pakistan), STU (Sri Lankan Tamil in the UK). The American populations from the 1KGP have recent admixture with Europeans [<xref ref-type="bibr" rid="pgen.1007298.ref063">63</xref>], and thus are not suited for our analysis and were excluded. Across the 22 populations the lowest sample size is 61 (ASW), so to minimise power differences among populations we randomly down-sampled each population to 61 unrelated individuals.</p><p>We also used the genetic data from the 142 populations of the SGDP project dataset, together with their meta-information (including geographic location) [<xref ref-type="bibr" rid="pgen.1007298.ref041">41</xref>]. For the geographic location, in the southern hemisphere we used the absolute value of the latitude. Most populations have high coverage whole-genome sequencing data for two representative individuals, so we used two individuals from each &#8216;Panel C&#8217; population with a sample size of at least two (110 populations).</p><p>Ancient genomes were used to infer the frequency of rs10166942 T in different pre-historic human populations. The genotype data from ancient paleo-eskimo individuals from the Saqqaq culture [<xref ref-type="bibr" rid="pgen.1007298.ref064">64</xref>] were obtained from the Danish bioinformatics center. Data on early Europeans [<xref ref-type="bibr" rid="pgen.1007298.ref042">42</xref>] was downloaded from the Reich lab webpage. We transformed the binary eigenstrat file to a vcf using eigenstrat2vcf.py and extracted the genotype information for rs10166942. Age information was extracted from Supplementary Data 1 in [<xref ref-type="bibr" rid="pgen.1007298.ref042">42</xref>]. After filtering, we were able to genotype 79 ancient individuals for rs10166942. These individuals lived in Eurasia 3,000 to 8,500 years ago and represent three different ancestry groups: Hunter-Gatherers (8 individuals), Early Farmers (33 individuals), and Steppe pastoralists (38 individuals).</p><p>We inferred the likely place of origin for the rs10166942 T allele by analysing haplotypes carrying the derived T allele, as levels of linked variation should be highest in the population closest to the one where it appeared. Since no homozygous T/T individuals are present in several of the 1KGP populations, we relied on the phased haplotypes across the 65kb region of interest. We calculated pi after removing derived haplotypes with evidence of recombination with ancestral rs10166942 C allele (<xref ref-type="supplementary-material" rid="pgen.1007298.s014">S1 Table</xref>).</p><p>In order to investigate the correlation of allele frequencies with latitude and temperature, we jointly analysed genetic, latitude, and temperature information. For modern humans, we estimated the absolute latitude of the location of each population according to Wikipedia and Google Maps (<xref ref-type="table" rid="pgen.1007298.t001">Table 1</xref>). The CEU population, of central European ancestry, was assigned the coordinates of Brussels. For early modern humans, latitude information was extracted from Supplementary Data 1 in [<xref ref-type="bibr" rid="pgen.1007298.ref042">42</xref>] and updated when necessary (e.g., some individuals lacked geographic coordinates or had problems with the longitude/latitude information).</p><p>Temperature time series information was extracted for 2001&#8211;2010 from a 0.5&#176;x0.5&#176; grid matrix assembled at the Climate Research Unit of the University of East Anglia (version 3.23; [<xref ref-type="bibr" rid="pgen.1007298.ref065">65</xref>]). Data is available since 1960, but we used only the time series from 2001&#8211;2010 to guarantee comparable and high-quality estimates across populations. Using the geographic coordinates of each population we extracted annual mean temperatures.</p><p>To investigate to what extent shared ancestry, latitude and temperature predict rs10166942 T allele frequency in each population we used two different linear models. We first used a PGLS analysis [<xref ref-type="bibr" rid="pgen.1007298.ref040">40</xref>], which can account for the full phylogenetic signal (the population relationships) present in our data [<xref ref-type="bibr" rid="pgen.1007298.ref039">39</xref>]. The response variable is the mean derived allele frequency of the rs10166942 T allele per population. We first conducted a null/full model comparison. The null model contains only the shared ancestry information (the &#8216;phylogeny&#8217;); here, we used the full pairwise F<sub>ST</sub> matrix averaged across all positions polymorphic in that particular population pair. Following Weir and Cockerham, we calculated the genome-wide average F<sub>ST</sub> between two populations as the &#8220;ratio of averages&#8221; (equation 10 in [<xref ref-type="bibr" rid="pgen.1007298.ref049">49</xref>]). A neighbor-joining (NJ) tree was calculated using a matrix of the pairwise F<sub>ST</sub> values with the R package <italic>ape</italic> [<xref ref-type="bibr" rid="pgen.1007298.ref066">66</xref>], and rooted using &#8216;mid-point&#8217; rooting with <italic>Archaeopteryx</italic> [<xref ref-type="bibr" rid="pgen.1007298.ref067">67</xref>]. The full model includes additional predictor variables: <italic>latitude</italic> and annual mean <italic>temperature</italic>. In order to achieve convergence of the model we z-transformed each predictor. We excluded populations one at a time and compared the model estimates derived from the subsets with those obtained from the full data set, which revealed the model to have good stability. We assessed for the full model whether the assumptions of normally distributed and homogenous residuals were fulfilled by visual inspection of a QQ-plot of the residuals and residuals plotted against fitted values [<xref ref-type="bibr" rid="pgen.1007298.ref068">68</xref>], which revealed no issues with these assumptions. As an overall test of the effect of the two test predictors (<italic>latitude</italic> and annual mean <italic>temperature</italic>), we compared the fit of the full model with that of the null model [<xref ref-type="bibr" rid="pgen.1007298.ref069">69</xref>] using a likelihood ratio test [<xref ref-type="bibr" rid="pgen.1007298.ref070">70</xref>].</p><p>We then performed a multi-model inference [<xref ref-type="bibr" rid="pgen.1007298.ref071">71</xref>] to compare the null model and all possible models that could be constructed with the two test predictors (four models in total). To quantify the relative performance of each model, we used Akaike&#8217;s Information Criterion (AIC, corrected for small samples) as a measure of model fit penalized for model complexity, and determined Akaike weights as a measure of the support a model received compared to all other models in the set [<xref ref-type="bibr" rid="pgen.1007298.ref071">71</xref>]. In practice, we use the Akaike weights to derive the 95% best model confidence (comprising the truly best model in the model set with a probability of 0.95) and also to determine Akaike weights for the individual predictors by summing the Akaike weights of the models comprising them. To infer the overall relevance of predictors in the model set we determined whether the null model was included in the 95% best model confidence set [<xref ref-type="bibr" rid="pgen.1007298.ref072">72</xref>]. The analysis was conducted in R [<xref ref-type="bibr" rid="pgen.1007298.ref073">73</xref>] using the function pgls of the package caper [<xref ref-type="bibr" rid="pgen.1007298.ref074">74</xref>].</p><p>To be able to analyze both the 1KGP and the SGDP datasets (which has small sample size for a large number of populations, so allele frequencies cannot be estimated) we also used a GLMM [<xref ref-type="bibr" rid="pgen.1007298.ref075">75</xref>] fitted with binomial error structure and logit link function [<xref ref-type="bibr" rid="pgen.1007298.ref076">76</xref>]. This model conceptually corresponds to a regression; however, it allows more flexibility with regard to the distribution of the response (e.g., normality and homogeneity of the residuals are not necessarily required), and it also allows us to effectively control for non-independence of the data due to multiple observations of the same populations or individuals [<xref ref-type="bibr" rid="pgen.1007298.ref075">75</xref>]. The response variable is the genotype of rs10166942 in each individual, in a 2-column-reponse-matrix (the derived and the ancestral allele counts). For the modern human genetic data, shared ancestry was controlled by adding as an additional fixed effect the genetic distance between each population and YRI, measured as the genome-wide average F<sub>ST</sub>. Population identity was included as a random effect in the model, to account for random genetic drift. We further included a random effect per individual to account for the non-independence of the ancestral and derived allele counts. The model that includes all these effects is the null model.</p><p>To test for the effects of <italic>latitude</italic> and the annual mean <italic>temperature</italic> we included them as test predictor variables with fixed effects. In the analysis of the early Europeans, we added <italic>age</italic> as a further test predictor variable. For the comparison among models (multi model inference [<xref ref-type="bibr" rid="pgen.1007298.ref071">71</xref>]) we considered the null model and all possible models that could be constructed with the two test predictors, totaling four models (eight in the early European analysis). We assessed model stability as in case of the PGLS, which revealed the model to have good stability (<xref ref-type="supplementary-material" rid="pgen.1007298.s016">S3 Table</xref>). Overdispersion was no issue (dispersion parameter of the full model in the 1KGP: 0.97 and the SGDP: 0.67). The models were fitted in R [<xref ref-type="bibr" rid="pgen.1007298.ref073">73</xref>] using the library &#8216;lme4&#8217; [<xref ref-type="bibr" rid="pgen.1007298.ref077">77</xref>].</p><p>Local adaptation on a single variant can lead to a rapid rise in the frequency of the positively selected allele, resulting in strong population differentiation (measured for example by F<sub>ST</sub>) between the population(s) with positive selection and those without it. We calculated per SNP F<sub>ST</sub> with a custom <italic>perl</italic> implementation of the Weir and Cockerham estimator [<xref ref-type="bibr" rid="pgen.1007298.ref049">49</xref>] for each pairwise population comparison.</p><p>The allele under positive selection will rise in frequency together with its background haplotype, raising the frequency of linked alleles. When the favoured allele is young (e.g., under a classic selection from a de-novo mutation model (SDN) hard sweep model), this results in a signature of extended haplotype homozygosity. To test for such signature, we calculated iHS [<xref ref-type="bibr" rid="pgen.1007298.ref045">45</xref>] and XP-EHH [<xref ref-type="bibr" rid="pgen.1007298.ref044">44</xref>] using <italic>selscan</italic> with default parameters [<xref ref-type="bibr" rid="pgen.1007298.ref078">78</xref>]. For iHS, we used SNPs with derived allele frequencies higher than 5% and lower than 95%. For XP-EHH, we used SNPs with derived allele frequency higher than 5% in the test population. These filters follow previously established methods [<xref ref-type="bibr" rid="pgen.1007298.ref079">79</xref>] and prevent signatures of extended LD to be broken by rare variants, while still obtaining XP-EHH values for derived alleles fixed or nearly fixed in the <italic>test</italic> population. For both analyses, only sites with a high confidence inferred ancestral allele were used (part of 1KGP genotype files). Recombination was estimated using the genetic map from HapMap Project, Phase 2 [<xref ref-type="bibr" rid="pgen.1007298.ref080">80</xref>].</p><p>All three statistics were calculated genome-wide, and P-values for SNPs of interest were calculated based on the empirical distribution. Since both tests are sensitive for positive selection, the tail of the empirical distribution is enriched for the targets of positive selection. Our analysis is hypothesis-driven for the migraine risk allele in rs10166942, and, thus, no correction for multiple testing is required.</p><p>To infer the selective history of the gene, we used an Approximate Bayesian Computation (ABC) approach, which allows us to assess the probability of different evolutionary models and their associated parameters [<xref ref-type="bibr" rid="pgen.1007298.ref046">46</xref>]. Following [<xref ref-type="bibr" rid="pgen.1007298.ref007">7</xref>, <xref ref-type="bibr" rid="pgen.1007298.ref050">50</xref>], we compared the genomic observations to simulations under three models with parameters drawn from uniform (U) prior distributions. These models are: (I) SDN, where the selected allele appeared as a single copy between 60,000 and 30,000 years ago (t<sub>mut</sub>~U(30,000, 60,000 years ago)) and was immediately advantageous with a selective coefficient that was allowed to differ between the African (s<sub>A</sub>~U(0,1.5%)) and the non-African (s<sub>NA</sub>~U(0.5,5%)) populations; (II) selection on standing variation (SSV), where a previously neutral allele at a given starting frequency (f<sub>sel</sub>~U(0,20%)) became positively selected (s<sub>NA</sub>~U(&gt;0,5%)) in the non-African population after the out of Africa migration and before the European-Asian split (51,000 to 21,000 years ago; t<sub>mut</sub>~U(21,000, 51,000 years ago)); (III) fully neutral model (NTR), where the allele appeared as in the SDN model (t<sub>mut</sub>~U(30,000, 60,000 years ago)) but was completely neutral.</p><p>We ran one million simulations for each selection model and 100,000 simulations for the neutral model using msms [<xref ref-type="bibr" rid="pgen.1007298.ref081">81</xref>]. Each simulation comprised a stretch of 185 kb with 122 chromosomes of an African (population 1) and a non-African (population 2) population. Human demographic parameters followed the model inferred by Gravel et al. [<xref ref-type="bibr" rid="pgen.1007298.ref082">82</xref>], and in each simulation we analyzed the African population with one non-African population (in Europe or Asia). To simulate the recombination hotspots across the locus, we simulated extended regions with a length that corresponded to the local increase in recombination rate above the baseline recombination rate (<xref ref-type="supplementary-material" rid="pgen.1007298.s012">S12 Fig</xref>). These regions were then removed before calculating summary statistics, such that they contribute recombination events but not mutation events to the data. The baseline recombination rate was the mean recombination rate across the locus excluding the peaks, based on a merged map from several 1KGP populations (<xref ref-type="supplementary-material" rid="pgen.1007298.s012">S12 Fig</xref>).</p><p>For the ABC inference we used five summary statistics: XP-EHH [<xref ref-type="bibr" rid="pgen.1007298.ref044">44</xref>], Fay and Wu&#8217;s H [<xref ref-type="bibr" rid="pgen.1007298.ref047">47</xref>], Tajima&#8217;s D [<xref ref-type="bibr" rid="pgen.1007298.ref048">48</xref>], F<sub>ST</sub> [<xref ref-type="bibr" rid="pgen.1007298.ref049">49</xref>] and derived-allele-frequency. XP-EHH and F<sub>ST</sub> were calculated between YRI and the studied population. We calculated the LD based statistic XP-EHH on the selected allele using the entire simulated region. We calculated the statistics Fay and Wu&#8217;s H, Tajima&#8217;s D, and average F<sub>ST</sub> (across SNPs in a section) in both simulated populations on two separate sections: the first section was the central ~65 kb part (since the genomic data shows strong population differentiation across 65 kb), and the second section were the combined flanking regions, together 120 kb long. We also used the allele frequency of the selected site in the African and non-African population and its F<sub>ST</sub>.</p><p>As in the genomic data, for the XP-EHH statistic we required the variant investigated to have a derived allele frequency &gt; 5% in the <italic>test</italic> non-African population. The absence of a long haplotype associated with the derived allele (XP-EHH) in the presence of strong population differentiation is an important attribute to differentiate between the SDN and the SSV model [<xref ref-type="bibr" rid="pgen.1007298.ref083">83</xref>&#8211;<xref ref-type="bibr" rid="pgen.1007298.ref085">85</xref>]. Thus, we used only simulations where XP-EHH could be calculated, which biased minimally the previously uniform prior.</p><p>All summary statistics were calculated in the same way for the simulations and the real data&#8211;where rs10166942 was used as a proxy for the selected site. The demographic history follows the [<xref ref-type="bibr" rid="pgen.1007298.ref082">82</xref>] model. African demography was based on YRI, all European populations (CEU, GBR, TSI, FIN, IBS) were simulated under the inferred European (CEU) demography, and all Asian populations (CDX, CHB, CHS, KHV, JPT, BEB, GIH, ITU, PJL, STU) under the inferred East Asian (CHB/JPT) demography. The ABC analysis was performed using the ABCtoolbox on BoxCox and PLS transformed summary statistics (following recommendations for ABCtoolbox) [<xref ref-type="bibr" rid="pgen.1007298.ref086">86</xref>] retaining the top 1,000 simulations matching our observation. We used the first five PLS components as they carried most information for each parameter (<xref ref-type="supplementary-material" rid="pgen.1007298.s013">S13 Fig</xref>). The PLS transformed statistics differentiate between the different models and capture the variation observed (<xref ref-type="supplementary-material" rid="pgen.1007298.s011">S11 Fig</xref>), rendering them well-suited for the inference.</p><p>We performed an additional ABC inference considering a halted SDN and a halted SSV model (with all parameters as above, with the only exception that selection ceased 3,000 years ago). Both power estimates and model selection were performed as described above. Lastly, we also performed an ABC analysis with the four selection models (SDN, partial SDN, SSV and partial SSV) to test our power to discriminate among them.</p>