<p>All animal studies were performed according to the approved protocols of the National Cancer Institute Animal Care and Use Committee. The animal study protocol is NCI LMB-036. Mice harboring the mutated <italic>Thra1</italic><sup><italic>PV</italic></sup> gene (<italic>Thra1</italic><sup><italic>PV</italic></sup> mice) were prepared and genotyped by PCR as described earlier [<xref ref-type="bibr" rid="pgen.1006991.ref012">12</xref>]. Wild-type (<italic>Thra1</italic><sup><italic>+/+</italic></sup>) and <italic>Thra1</italic><sup><italic>PV/+</italic></sup> female siblings were used in this study. To induce hypothyroidism, mice were fed a low-iodine diet supplemented with 0.15% propylthiouracil (LoI/PTU) (Cat# TD 95125, Harlan Teklad, Madison, WI) for 10 days. To induce hyperthyroidism, T3 (5 &#956;g; Cat# T2752, Sigma-Aldrich, St. Louis, MO) was injected intraperitoneally to each mouse for 6 days while they were being fed with LoI/PTU diet. The same volume of vehicle (phosphate-buffered saline) was injected in the control group.</p><p>Erythroleukemia K562 cell line was maintained in RPMI1650 (Thermo Fisher Scientific, Waltham, MA) with 10% fetal bovine serum (FBS; GE Healthcare life science, Marlborough, MA) with 50 units/ml penicillin G and 50 &#956;g/ml streptomycin (Thermo Fisher Scientific, Waltham, MA). Bone marrow cells were isolated from femurs and tibiae of wild-type and <italic>Thra1</italic><sup><italic>PV</italic></sup> mice (age: 3&#8211;5 months). Single cell suspensions were prepared by passing bone marrow through a 70 &#956;M cell strainer.</p><p>For analysis of complete blood counts, peripheral blood was collected in a heparinized microtube and analyzed by hematology analyzer (HEMAVET HV950FS, Drew Scientific, Miami Lakes, FL).</p><p>The level of TSH in serum was measured as described [<xref ref-type="bibr" rid="pgen.1006991.ref050">50</xref>]. Total T4 (TT4) and T3 (TT3) levels were determined by using Gamma Coat T4 and T3 assay radioimmunoassay (RIA) kits according to the manufacturer&#8217;s instruction (Cat# 06B256447 and 06B254029, MP Biomedical, LLC, Solon, OH).</p><p>To detect burst-forming units-erythroid (BFU-E) colonies, 5 X 10<sup>4</sup> bone marrow cells were seeded in duplicates in semisolid medium (Methocult M3434; STEMCELL Technologies, Vancouver, BC). To detect colony forming units-erythroid (CFU-E) colonies, 8 X 10<sup>4</sup> bone marrow cells were seeded in duplicates in semisolid medium (Methocult M3334; STEMCELL Technologies, Vancouver, BC). To detect colony- forming units-megakaryocytes (CFU-Mk) colonies, 1 X 10<sup>5</sup> bone marrow cells were seeded in duplicates in semisolid medium (Methocult-c, 04974; STEMCELL Technologies, Vancouver, BC) supplemented with 10 ng/ml Interleukin (IL)-3, 20 ng/ml Interleukin (IL)-6, 50 ng/ml thrombopoietin (TPO) (STEMCELL Technologies, Vancouver, BC). To analyze the colonies of multi-potential progenitor cells (CFU-GEMM) and granulocyte/macrophage progenitor (CFU-GM), 4 x 10<sup>4</sup> bone marrow cells were mixed with semisolid medium (Methocult GF M3434; STEMCELL Technologies, Vancouver, BC) by vortexing. Bone marrow cells (4 x 10<sup>4</sup> cells) from WT mice (n = 4) and <italic>Thra1</italic><sup><italic>PV/+</italic></sup> mice (n = 4) were seeded in 6 wells plate (DENVILLE, SCIENTIFIC INC., quadruplicates) which was cultured in 5% CO<sub>2</sub> humidified incubator at 37&#176;C. The numbers of colonies were counted under inverted microscope (Primo Vert, Ziess) by morphologic criteria 8 days after plating.</p><p>Total RNA was isolated from bone marrow cells using Trizol (Thermo Fisher Scientific, Waltham, MA). RT&#8212;qPCR was performed with one step SYBR Green RT-qPCR Master Mix (Qiagen, Valencia, CA). The mRNA level of each gene was normalized to the GAPDH (glyceraldehyde-3-phosphate dehydrogenase) mRNA level. The primer sequences are listed in <xref ref-type="supplementary-material" rid="pgen.1006991.s001">S1 Table</xref>.</p><p>ChIP assay with bone marrow cells was performed as described previously [<xref ref-type="bibr" rid="pgen.1006991.ref051">51</xref>]. Quantitative PCR was performed to detect the upstream fragment in <italic>Gata1</italic> genes (primer sequences are listed in <xref ref-type="supplementary-material" rid="pgen.1006991.s001">S1 Table</xref>). The fold of changes in binding was relative to the control of IgG level as 1.</p><p>Oligonucleotide probe containing mouse <italic>Gata1</italic> TREs or F2 TRE (positive control) was labeled with [&#945;-32p] dCTP by Klenow fill-in reaction. Assays were performed as described previously [<xref ref-type="bibr" rid="pgen.1006991.ref052">52</xref>].</p><p>The <italic>Gata1</italic> TRE2 and <italic>Gata1</italic> TRE1 luciferase constructs were generated by cloning upstream <italic>Gata1</italic> promoter fragments into the pGL4.23 luciferase plasmid. The <italic>Gata1-</italic> TRE2 luciferase construct was made by insertion of a 1.553 kb XhoI-HindIII fragment representing the sequences between -3.741 kb and -2.189 kb. The <italic>Gata1</italic>-TRE1 luciferase construct was made by insertion of a 140 bp XhoI-HindIII fragment representing the sequences between +1.385 kb and +1.524 kb. Insert sequence validated by DNA sequencing. K562 cells were transfected with the TRE-luc reporters with Genepulse X cell electrophorators (Biorad). Bone marrow cells (2X10<sup>6</sup> cells) were transfected with the GATA1-pLenti6/V5 plasmid (3 &#956;g) provided by Dr. GP Rodgers (NHLBI; [<xref ref-type="bibr" rid="pgen.1006991.ref053">53</xref>]) using 4D nucleofector (Lonza) in accordance with the manufacturer&#8217;s instructions. The Gata1 reporter plasmids cloned in pGL3 basic (5 &#956;g), and the expressing plasmid for TR&#945;1 (pcDNA3.1-TR&#945;1; 10 &#956;g) with or without the expression plasmid for TR&#945;1PV (pcDNA3.1-TR&#945;1PV; 80 &#956;g) were transfected into K562 cells according to Guigon et al [<xref ref-type="bibr" rid="pgen.1006991.ref054">54</xref>]. Luciferase activity was measured using Victor 3 (PerkinElmer Life and Analytical Sciences, Waltham, MA). Luciferase values were standardized to the ratio of &#946;-galactosidase activity and protein concentration. The fold of changes in activity was based on using the values of negative control (no plasmid transfected cell without T3) as 1.</p><p>For lineage depleted bone marrow cell preparation, linage marker positive cells were depleted using the biotin based selection kit (cat# 19856, STEMCELL Technologies, Vancouver, BC) according to the manufacturer's instructions. Lin- BM cells were seeded in fibronectin-coated wells (Corning Inc, Corning, NY). To induce erythropoiesis, Lin- BM cells were cultured as described [<xref ref-type="bibr" rid="pgen.1006991.ref024">24</xref>].</p><p>For May-Gr<bold>&#220;</bold>nwald Giemsa stain, cytocentrifuged cells were stained with May-Gr<bold>&#220;</bold>nwald solution (Cat# MG500, Sigma-Aldrich, St. Louis, MO) for 5 minutes and in Giemsa (Cat# GS500, Sigma-Aldrich, St. Louis, MO) for 20 minutes. For whole bone marrow sections, femurs were fixed in 10% (vol/vol) neutral buffered formalin solution (NBF, approximately 4% formaldehyde) (Sigma-Aldrich, St. Louis, MO). The embedded sections were stained with hematoxylin and eosin (HistoServ, Germantown, MD).</p><p>All antibodies used in flow cytometry were from eBiosciences (Thermo Fisher Scientific, Waltham, MA). The sources of antibodies and fluorophore-labeled antibodies used in FACS analyses are listed in <xref ref-type="supplementary-material" rid="pgen.1006991.s002">S2 Table</xref>. The flow cytometry analyses were performed on a BD LSR II flow cytometer (BD bioscience, San Jose, CA) and analyzed with FloJo, LLC (Tree Star Inc, Ashland, OR).</p><p>The western blot analysis of bone marrow lysates was carried as described previously [<xref ref-type="bibr" rid="pgen.1006991.ref050">50</xref>]. To determine the GATA1 tagged with V5 (The V5 tag is derived from a small epitope found on the P and V proteins of the paramyxovirus of simian virus 5 (SV5). after transfection with GATA1-V5 expression plasmid, anti-V5 antibodies (1:2000 dilution; Thermo Fisher Scientific) was used to detect the expressed GATA1-V5. GAPDH (1:4000 dilution; Cell Signaling Technology (Danvers, MA) was used as a loading control.</p><p>For the detection of GATA1 proteins in the bone marrow of WT and <italic>Thra1</italic><sup><italic>PV/+</italic></sup> mice, bone marrow lysates (600 &#956;g each) were first immunoprecipitated with rat anti-GATA1 antibody (4 &#956;g; Santa Crus Biotecholology, Cat.# Sc-265) or mouse IgG (4 &#956;g; negative controls) followed by pulling down the enriched GATA1-anti-GATA1 antibody-complex with protein G-agarose beads. GATA1 proteins were subsequently detected by western blot analysis as described above using rabbit anti-GATA1 antibody (1:1000 dilution; abcam, Cat.# ab28839).</p><p>All statistical analyses and the graphs were performed and generated using GraphPad Prism version 6.0 (GraphPad Software, La Jolla, CA). <italic>P</italic> &lt; 0.05 is considered statistically significant. All data are expressed as mean &#177; SEM.</p>