<p>Individuals 1&#8211;4 had unexplained ID or developmental delay and were ascertained through family-based whole exome sequencing in a diagnostic setting in the Department of Human Genetics at Radboudumc and MaastrichtUMC. Exome sequencing and data analysis were performed as previously described in the probands and their unaffected parents [<xref ref-type="bibr" rid="pgen.1006864.ref029">29</xref>]. Individual 5 was ascertained during a clinical genetic workup due to failure to thrive and lack of expected normal childhood development at Children's Hospital Colorado (Aurora, Colorado). Chromosomal microarray analysis for individual 5 was performed using both copy number and single-nucleotide polymorphism (SNP) probes on a whole genome array (CytoScan HD platform; Affymetrix, Santa Clara, CA). The <italic>KMT2C</italic> deletion was confirmed to be <italic>de novo</italic> by parental microarray testing. Additionally, this deletion was confirmed using locus-specific qPCR. The primer pairs used were directed to <italic>KMT2C</italic> and two adjacent negative control regions (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s014">S10 Table</xref></bold>). All data were analyzed and reported using the February 2009 (hg19) human genome build.</p><p>Although all individuals harbored <italic>de novo</italic> loss of function mutations in <italic>KMT2C</italic> (<bold><xref ref-type="fig" rid="pgen.1006864.g001">Fig 1A</xref></bold>), exome results revealed that individual 1 and 2 carry one or two additional <italic>de novo</italic> mutations that do not appear to be related to the core phenotype of the individuals presented here. Individual 1 carried a second <italic>de novo</italic> truncating mutation in <italic>PHF21A</italic>, which is located in the 11p11.2 contiguous gene deletion syndrome named Potocki-Shaffer syndrome, PSS (OMIM #601224) [<xref ref-type="bibr" rid="pgen.1006864.ref061">61</xref>, <xref ref-type="bibr" rid="pgen.1006864.ref062">62</xref>]. Three individuals with translocation breakpoints through <italic>PHF21A</italic> were reported previously and it was concluded that this gene was associated with the PSS. However, no intragenic loss of function mutations are reported in association with this syndrome so far. The mutation in individual 1 is present in the last exon of the gene and is therefore not subjected to nonsense mediated RNA decay, however it is possible that this variant may contribute to the phenotype observed in this patient.</p><p>In individual 2 we found the <italic>KMT2C</italic> change c.7550C&gt;G (p.Ser2517*) in mosaic in blood. Individual 2 carries two additional <italic>de novo</italic> missense mutations in <italic>C11orf35</italic> (p.Pro602Leu) and <italic>UBR5</italic> (p.Arg1907His). Protein function or mutations associated with other diseases for <italic>C11orf35</italic> are not known, and thus we cannot rule out that this variant may contribute to the patient phenotype. With respect to <italic>UBR5</italic>, the same mutation has been previously described in a family with adult myoclonic epilepsy [<xref ref-type="bibr" rid="pgen.1006864.ref063">63</xref>], but that family was not associated with ID/Neuro Developmental Delay. The missense change in <italic>UBR5</italic> individual 2 is therefore unlikely to cause his ID and autism. Moreover, no epilepsy is present in this individual at age 10.</p><p>Flies were reared on standard cornmeal-agar media at 25&#176;C on a light/dark cycle of 12h/12h in 50% or 70% humidity. The following stocks were obtained from Bloomington Drosophila Stock Center: <italic>UAS-trr-RNAi</italic> generated by the Transgenic RNAi Project (TRiP) (stock #36916: <italic>y1 sc*v1;P{TRiP</italic>.<italic>HMS01019}attP2</italic>), <italic>UAS-mCD8</italic>::<italic>GFP</italic> (Stock #5137: <italic>y [1] w [*]; P{w [+mC] = UAS-mCD8</italic>::<italic>GFP</italic>.<italic>L}LL5</italic>, <italic>P{UAS-mCD8</italic>::<italic>GFP</italic>.<italic>L}2</italic>), <italic>UAS-mCherry-RNAi</italic> (stock #35785: <italic>y [1] sc [*] v [1]; P{y [+t7</italic>.<italic>7] v [+t1</italic>.<italic>8] = VALIUM20-mCherry}attP2</italic>), <italic>Elav-Gal4; UAS-dicer2</italic> (stock #25750: <italic>P{w [+mW</italic>.<italic>hs] = GawB}elav [C155] w [1118]; P{w [+mC] = UAS-Dcr-2</italic>.<italic>D}2</italic>), and <italic>R14H06-Gal4</italic> (stock #48667: <italic>P{GMR14H06-GAL4}fattP2)</italic> [<xref ref-type="bibr" rid="pgen.1006864.ref033">33</xref>]. <italic>G9a</italic><sup>DD1</sup> mutants and the precise excision control were generated previously [<xref ref-type="bibr" rid="pgen.1006864.ref018">18</xref>]. To study the effects of trr down regulation, the UAS/Gal4 system was used with <italic>UAS-trr-RNAi</italic> combined with pan-neuronal (<italic>elav-Gal4</italic>) or mushroom body (MB) specific (<italic>R14H06-Gal4</italic>) Gal4 driver lines. For all <italic>trr</italic> knockdown experiments, genetic control animals without the RNAi hairpin were generated using the (AttP2) genetic background control strain (Bloomington #36303) or the <italic>mCherry-RNAi</italic> strain (Bloomington #35785). For trr ChIP-seq experiments, a Nijmegen wild type strain was used.</p><p><italic>R14H06-Gal4</italic> combined with <italic>UAS-mCD8</italic>::<italic>GFP</italic> was used to visualize the calyx region of the <italic>Drosophila</italic> MB in controls and <italic>trr</italic> knockdown flies. The following genotypes were analyzed: <italic>UAS-mCD8</italic>::<italic>GFP/+;R14H06-Gal4/UAS-trr-RNAi</italic> (trr knockdown) and <italic>UAS-mCD8</italic>::<italic>GFP/+;R14H06-Gal4/+</italic> (control). Adult brains were dissected in PBS (pH 7.2) and fixed with ice cold methanol for 2 minutes, washed three times with PBS for 5 minutes each and permeabilized in PBS with 0.3% Triton-X 100 (PBT) for 1 hour. Fixed and permeabilized brains were blocked (5% normal goat serum (NGS)) at room temperature for 2 hours and incubated in 300nM DAPI solution for three minutes, followed by three five minute washes in PBT. Next, brains were incubated with the primary antibody (rabbit anti-trr [<xref ref-type="bibr" rid="pgen.1006864.ref024">24</xref>], 1:5000) in PBT with 5% NGS for 72 hour at 4&#176;C followed by five 20 minute washes in PBT. Brains were incubated with the secondary antibody (goat anti-rabbit Alexa Fluor 568, 1:250; Invitrogen) for 48 hour in PBT with 5% NGS at 4&#176;C and washed at room temperature in PBT five times for 20 minutes. Brains were mounted in Vectashield (Vector Laboratories) and imaged on a confocal microscope (Zeiss LSM 510 duo vario confocal microscope). Confocal stacks were processed using ImageJ software [<xref ref-type="bibr" rid="pgen.1006864.ref064">64</xref>].</p><p>Courtship conditioning assays were performed on 5-day-old males raised at 25&#176;C, 70% humidity, and a 12h day/night rhythm as previously described [<xref ref-type="bibr" rid="pgen.1006864.ref018">18</xref>, <xref ref-type="bibr" rid="pgen.1006864.ref035">35</xref>]. The following genotypes were analyzed: <italic>R14H06-Gal4/UAS-mCherry-RNAi</italic> (control) and <italic>R14H06-Gal4/UAS-trr-RNAi</italic> (trr-RNAi). These genotypes were generated by crossing females containing the trr-RNAi or mCherry-RNAi inserted into the TRiP attP2 genetic background (genotypes: trr RNAi&#8212;<italic>y[1] sc[*] v[1]; P{y[+t7</italic>.<italic>7] v[+t1</italic>.<italic>8] = TRiP</italic>.<italic>HMS01019}</italic>, mCherry RNAi&#8212;<italic>y[1] sc[*] v[1]; P{y[+t7</italic>.<italic>7] v[+t1</italic>.<italic>8] = VALIUM20-mCherry}attP2)</italic>, with identical males containing the Gal4 driver (genotype: <italic>P{GMR14H06-GAL4}fattP2</italic>. Males were randomly assigned to either trained (n<sub><italic>control</italic></sub> = 59, n<sub><italic>trr-RNAi</italic></sub> = 62) or na&#239;ve (n<sub><italic>control</italic></sub> = 56, <italic>n</italic><sub><italic>trr-RNAi</italic></sub> = 60) groups. Training was performed by pairing individual males in with 5-day-old mated wild type females. All experiments were conducted with a one-hour training period, and tested after a one-hour isolation period. For each fly pair, a courtship index (CI) was calculated as the percentage of a 10 minute time period spent courting. Courtship behaviour was manually quantified by observing videos for all 237 fly pairs analyzed in this study. Quantification was performed by trained observers that were na&#239;ve to the nature of the experiment. Comparisons of average CI between na&#239;ve and trained groups of the same genotype were calculated using a Mann-Whitney test. A learning index (LI = (CI<sub>naive-</sub>CI<sub>trained</sub>)/CI<sub>naive</sub>) was calculated to compare courtship memory between genotypes. Statistical comparisons between genotypes were conducted using a randomization test [<xref ref-type="bibr" rid="pgen.1006864.ref065">65</xref>] with a custom bootstrapping script created in R (10,000 replicates) [<xref ref-type="bibr" rid="pgen.1006864.ref066">66</xref>].</p><p>Chip was performed using an antibody directed against trr [<xref ref-type="bibr" rid="pgen.1006864.ref014">14</xref>, <xref ref-type="bibr" rid="pgen.1006864.ref034">34</xref>]. This antibody was originally validated by the lack of staining in trr homozygous null mutant embryos [<xref ref-type="bibr" rid="pgen.1006864.ref014">14</xref>]. Additionally, this antibody shows complete lack of signal in mutant clones of the <italic>Drosophila</italic> salivary gland, and has been used for ChIP experiments showing very specific trr binding patterns at EcR response genes during ecdysone-mediated developmental transitions [<xref ref-type="bibr" rid="pgen.1006864.ref034">34</xref>]. Here, we provide additional evidence for specificitiy of this antibody by showing reduced immunofluorescent staining in mushroom body target cells expressing a trr-RNAi construct (<bold><xref ref-type="fig" rid="pgen.1006864.g002">Fig 2</xref></bold>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Chromatin was extracted from 50 &#956;L aliquots of frozen wildtype fly heads, aged between 0 and 5 days old, in biological duplicates. Fly heads were crushed in PBS (Sigma) and crosslinked with formaldehyde (Sigma) at a final concentration of 1% for 30 minutes. Crosslinking was terminated using glycine (Invitrogen) at a final concentration of 125mM and crosslinked material was immediately washed twice in PBS and centrifuged at 13000rpm, for 15 minutes at 4 degrees. The pellet was resuspended in buffer 1 (15 mM Tris-HCl (pH 7.5), 60 mM KCl, 15 mM NaCl, 1 mM EDTA, 0.1 mM EGTA, 0.15 mM spermine, 0.5 mM spermidine, 0.1 mM sucrose) and the solution was homogenized using a QiaShredder column (Qiagen). Cells were lysed using buffer 1 supplemented by 2% triton-X-100 (Sigma) and a crude nuclear extract was collected by centrifugation (6000 rpm, 10 minutes at 4 degrees). Nuclei were re-suspended in incubation buffer (0.15% SDS, 1% triton x-100, 150 mM NaCl, 1 mM EDTA, 0.5 mM EGTA, 10 mM Tris) supplemented with 0.1% BSA (Sigma) and 1x protease inhibitor (Roche) and subjected to sonication (Diagenode Bioruptor) for 30 minutes (30 seconds on/off cycle using the &#8220;high intensity&#8221; mode), yielding average DNA fragments of 150&#8211;300 base pairs. Immunoprecipitation reactions were performed overnight in incubation buffer with 3 ng of anti-trr antibody (gift from Dr. A. Mazo [<xref ref-type="bibr" rid="pgen.1006864.ref014">14</xref>]), protease inhibitor cocktail (Roche), BSA, and pre-blocked protein A/G agarose beads (Santa Cruz). Chromatin-antibody-bead complexes were recovered by centrifugation (4000 rpm, 2 minutes at four degrees) and washed twice with low salt buffer (0.1% SDS, 1% Triton, 2 mM EDTA, 20 mM Tris pH 8, 150 mM NaCl), once with high salt buffer (0.1% SDS, 1% Triton, 2 mM EDTA, 20 mM Tris pH 8, 500 mM NaCl), once with LiCl wash buffer (10 mM Tris pH 8.0, 1% Na-deoxycholate, 1% NP-40, 250 mM LiCl, 1 mM EDTA) and twice with TE buffer. Chromatin was eluted in 1% SDS, 0.1M NaHCO<sub>3</sub>, 200 mM NaCl and de-crosslinked at 65&#176; Celsius (C) for four hours. DNA was purified by phenol/chloroform extraction and ethanol precipitated using linear acrylamide (Ambion) and sodium acetate at -20 degrees. Library preparation for Illumina sequencing was performed using the Truseq DNA sample prep kit V2 (Illumina) with approximately 3 ng of starting DNA 15 PCR cycles for amplification. Library fragment size was assessed using the 2100 Agilent Bioanalyser and was shown to be between 200 and 400 bp. Cluster generation and sequencing-by-synthesis (50bp) was performed using the Illumina Hiseq 2000 according to standard protocols of the manufacturer. The image files generated by the HiSeq were processed to extract DNA sequence data. We obtained between 30 and 54 million reads per sample. Reads were mapped to the <italic>Drosophila</italic> genome (BDGP R5/dm3) using the Burrows Wheeler Aligner (BWA, version 0.6.1) with standard settings allowing 1 mismatch [<xref ref-type="bibr" rid="pgen.1006864.ref067">67</xref>]. Total alignment efficiency was more than 95%. Duplicate reads and reads with a mapping quality score (MAPQ) below 15 were excluded from downstream analysis (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s005">S1 Table</xref></bold>). For each biological replicate, trr binding sites were identified using MACS2 [<xref ref-type="bibr" rid="pgen.1006864.ref036">36</xref>] with input DNA as control (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s005">S1 Table</xref></bold>). Bindings sites on chromosome U, Uextra and mitochondrial genome were not used for further analysis. Remaining putative trr binding regions were visualized in a heatmap that was sorted based on k-means clustering of read intensity around the centre of the peak using the python script fluff_heatmap.py (<ext-link ext-link-type="uri" ns0:href="https://github.com/simonvh/fluff">https://github.com/simonvh/fluff</ext-link>) (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s003">S3A Fig</xref></bold>). Individual clusters were examined visually in the genome browser to assess the quality of the peaks within the clusters. Two clusters (8 and 15) of replicate 1 were identified that contained many false positive peaks with a relatively small binding region and cluster 15 had very few reads around the centre. Peaks within these clusters were removed from downstream analysis. For trr_rep2 all clusters appeared to represent high quality trr binding sites. Trr binding regions from both biological replicates were merged and concatenated and the number of reads present in these regions were counted in each sample individually using HTSeq [<xref ref-type="bibr" rid="pgen.1006864.ref068">68</xref>] and normalized to library size. The ratio and mean of reads in each binding region was calculated between the two biological duplicates to identify trr binding sites that were consistent between the two biological replicates. We included all binding sites with a mean number of reads &gt; 100 and &lt; 2 fold difference in normalized read count between the two biological replicates (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s003">S3B Fig</xref></bold>). Using these criteria we identified 3371 predicted trr binding regions, which appeared to be very consistent upon visual inspection in the UCSC genome browser (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s003">S3D Fig</xref></bold>). These predicted trr binding sites were allocated to the nearest genomic feature using the pearl script annotatePeaks.pl [<xref ref-type="bibr" rid="pgen.1006864.ref069">69</xref>] with standard settings. In order to find out if detected binding sites of trr are overrepresented in any genomic regions, we compared trr binding sites to an equal set of randomly selected genomic positions using <ext-link ext-link-type="uri" ns0:href="https://www.random.org/">https://www.random.org/</ext-link> (<bold><xref ref-type="fig" rid="pgen.1006864.g003">Fig 3B</xref></bold>). The average trr occupancy profile over all transcription start sites (<bold><xref ref-type="fig" rid="pgen.1006864.g003">Fig 3C</xref></bold>) was calculated using ensemble annotations from release 84 [<xref ref-type="bibr" rid="pgen.1006864.ref070">70</xref>] with read count data generated by HTseq-count [<xref ref-type="bibr" rid="pgen.1006864.ref068">68</xref>] in 50-bp bins spanning all transcription start sites.</p><p>To validate the ChIP-seq data, independent ChIP reactions were performed as described above and tested by qPCR. Validation targets were selected based on ChIP-seq data with representative binding regions on the promoters of <italic>mor</italic>, <italic>lis-1</italic>, <italic>Atg9</italic>, <italic>Hsc70-4</italic>, <italic>Socs36e</italic>, <italic>Tsp42ed</italic>, <italic>smid</italic> and <italic>med21</italic>. Additionally, we selected two trr-negative regions in the promoters of <italic>drm99B</italic> and <italic>CG1646</italic> (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s014">S10 Table</xref></bold>). All primers were tested and approved for amplification efficiency according to standard methods. qPCR was performed on the ChIPed and input samples using SYBRgreen master mix (Promega) and the 7900HT Fast Real Time PCR system (Applied biosystems) according to the manufacturer&#8217;s instructions. Fold enrichments per target were calculated using the mean of the percent input of the two negative regions relative to the positive region. Mean fold enrichments are plotted with standard error of the mean as error bars (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s003">S3C Fig</xref></bold>). All regions tested confirmed the ChIP-seq results. In addition we compared our trr ChIP-seq targets to published trr ChIP-seq data from cultured <italic>Drosophila</italic> S2 cells that was obtained using a different trr antibody [<xref ref-type="bibr" rid="pgen.1006864.ref025">25</xref>]. In S2 cells, 1482 genes were identified with a trr peak at the promoter [<xref ref-type="bibr" rid="pgen.1006864.ref025">25</xref>]. 943 of these genes were also identified as trr targets in this study. This is a very high overlap considering the vastly different starting material (cultured S2 cells versus fly heads), suggesting a high degree of concordance between the two antibodies.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Mutant and control lines for <italic>trr</italic> and <italic>G9a</italic> were aged between 1 and 5 days and snap frozen in liquid nitrogen. Frozen fly heads were harvested by vortexing and separated from other body parts through a series of standard laboratory sieves. Total RNA was extracted from 50 &#956;L aliquots of frozen fly heads using the RNAeasy lipid tissue mini kit (Qiagen). For <italic>trr</italic> knockdown, two biological replicates were used, for <italic>G9a</italic> mutants, three biological replicates were used for each condition. mRNA was purified using the Oligotex kit (Qiagen) and cDNA was synthesized using the SuperScript III First Strand synthesis kit (Thermo Fisher) using random hexamers as primers. Second strand cDNA was synthesized using <italic>E</italic>. <italic>Coli</italic> polymerase and T4 ligase (New England Biolabs Inc. (NEB)). Remaining RNA was removed using 2 units RNaseH (NEB) before the cDNA was purified using the MinElute PCR purification kit (Qiagen). DNA end repair was performed followed by ligation of Illumina sequencing adaptors and size selection for 300 bp by 2% E-gel (Invitrogen). Fragments were linearly amplified (15 PCR cycles), as validated by quantitative real-time PCR (qPCR), and sample quality was assessed using the Agilent Bioanalyser 2100. Cluster generation and sequencing-by-synthesis (36bp) was performed using the Illumina Genome Analyser IIx (GAIIx) according to standard protocols of the manufacturer. The image files generated by the GAIIx were processed to extract DNA sequence data. From the GAIIx, we obtained between 27 and 35 million reads. Reads were mapped to the <italic>Drosophila</italic> genome (BDGP R5/dm3) using the Burrows Wheeler Aligner (BWA, version 0.6.1) with standard settings allowing 1 mismatch [<xref ref-type="bibr" rid="pgen.1006864.ref067">67</xref>]. Only the uniquely mapped reads were used for further analysis and total alignment efficiency was between 63% and 79% for G9a samples and between 79% and 81% for trr samples (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s009">S5 Table</xref></bold>). Total read count data was generated by the python script HTSeq-count (<ext-link ext-link-type="uri" ns0:href="http://www-huber.embl.de/HTSeq/doc/overview.html">http://www-huber.embl.de/HTSeq/doc/overview.html</ext-link>) with gene annotations extracted from the file Drosophila_melanogaster.BDGP5.75.gtf, available at <ext-link ext-link-type="uri" ns0:href="http://www.ensembl.org/">http://www.ensembl.org</ext-link>. In all samples, over 96% of aligned reads, mapped unambiguously to exons. The unambiguously mapped reads, ranging from 18 to 25 million reads for G9a samples and from 26 to 28 million reads for trr samples, were used for further analysis of differential gene expression by DeSeq2 [<xref ref-type="bibr" rid="pgen.1006864.ref071">71</xref>]. Hierarchical clustering based on Euclidean distances with Pearson correlation using the normalized expression values and variance stabilizing transformation, revealing a high degree of similarity between biological replicates (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s004">S4A and S4C Fig</xref></bold>). In order to perform statistical comparisons, dispersion values were estimated using the DESEQ method. As expected, we observed a high degree of correlation between gene expression and dispersion values with decreasing dispersion upon increasing expression patterns (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s004">S4B and S4D Fig</xref></bold>). We then used DESeq2 to identify genes that are differentially expressed in mutant fly heads based on negative binominal distribution (adjusted p-value &lt; 0.05, fold change &gt; 1.5, <bold><xref ref-type="fig" rid="pgen.1006864.g004">Fig 4A and 4C</xref>, <xref ref-type="supplementary-material" rid="pgen.1006864.s010">S6</xref> and <xref ref-type="supplementary-material" rid="pgen.1006864.s011">S7 Table</xref></bold>).</p><p>Validation of <italic>trr</italic> knockdown was performed using SYBRgreen master mix (Promega) and the 7900HT Fast Real Time PCR system (Applied Biosystems) according to the manufacturer&#8217;s instructions on a new biological replicate cDNA (described above) performed in triplicate technical replicates. Beta-cop and RP49 were used as reference genes for normalization and calculation of fold change differences upon pan-neuronal knockdown in fly heads using the &#916;&#916; Ct procedure (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s001">S1 Fig</xref></bold>). All RT-qPCR primers (<bold><xref ref-type="supplementary-material" rid="pgen.1006864.s014">S10 Table</xref></bold>) are validated for amplification efficiency according to standard procedures.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Gene Ontology (GO) enrichment analysis was performed using the Panther software version 11.1 [<xref ref-type="bibr" rid="pgen.1006864.ref052">52</xref>] on <ext-link ext-link-type="uri" ns0:href="http://geneontology.org/">http://geneontology.org/</ext-link> (GO Ontology database released on 2015-08-06 and 2016-10-24) using the GO-SLIM function for GO enrichment datasets that showed a high degree of redundancy (<bold><xref ref-type="fig" rid="pgen.1006864.g004">Fig 4</xref> and <xref ref-type="supplementary-material" rid="pgen.1006864.s002">S2 Fig</xref></bold>). For GO enrichment analysis of differential gene expression, list was used with genes expressed in fly heads as background control. This list was generated by exclusion of 4579 genes that had less than 10 reads, leaving 11103 genes. Overlap between datasets was determined and visualized as a Venn diagram by BioVenn [<xref ref-type="bibr" rid="pgen.1006864.ref072">72</xref>] (<bold>Figs <xref ref-type="fig" rid="pgen.1006864.g003">3E</xref> and <xref ref-type="fig" rid="pgen.1006864.g004">4E</xref></bold>). Hypergeometric statistics on overlaps were calculated using <ext-link ext-link-type="uri" ns0:href="https://www.geneprof.org/GeneProf/tools/hypergeometric.jsp">https://www.geneprof.org/GeneProf/tools/hypergeometric.jsp</ext-link>.</p><p>The raw data for RNA-seq and ChIP-seq is available at the NCBI Gene Expression omnibus (GEO), accession number GSE89459.</p><p>Diagnostic whole exome sequencing (WES) was approved by the medical ethics committee of the Radboud University Medical Center, Nijmegen, The Netherlands (registration number 2011&#8211;188). For all patients, written informed consent for WES was obtained after counseling by a clinical geneticist.</p>