<p>All patients in this study fulfilled the 2002 American-European Consensus Group (AECG) criteria for primary SS [<xref ref-type="bibr" rid="pgen.1006820.ref007">7</xref>]. Seropositivity of anti-Ro/SSA autoantibodies was determined by the antibody index &#8805;1 using the Bio-Plex assay (Bio-Rad) following the manufacturer&#8217;s protocol. The present study was approved by the Oklahoma Medical Research Foundation Institutional Review Board (IRB#1&#8212;Biomedical), operation under Federalwide Assurance (FWA) # 00001389 and IRB # 00000114 under IORG 0000079 approved by the Office for Human Research protection (OHRP), Department of Health and Human Services (DHHS). The OMRF IRB is in compliance with local regulations and the regulations of the United States Food and Drug Administration as described in 21 CFR Parts 50, 56 and 11, the International Conference on Harmonization (ICH) E6, and the United States Department of Health and Human Services at 45 CFR 46. The current study was approved under IRB#07&#8211;12 and all patients provided written informed consent.</p><p>SS patients were evaluated by expert clinicians at the University of Minnesota or the Oklahoma Medical Research Foundation (OMRF) as described previously [<xref ref-type="bibr" rid="pgen.1006820.ref090">90</xref>]. Samples subjected to microarray-based transcriptome measurements included 182 patients with SS and 76 healthy controls. After QC assessments (see below), 115 anti-Ro/SSA positive SS cases and 56 healthy controls were included in the transcriptome profiling analysis. We assessed the distribution of the five main nucleated blood cell subpopulations (granulocytes, lymphocytes, monocytes, basophils and eosinophils) derived from complete blood cell counts from SS patients and healthy controls to evaluate whether differences in their proportions might contribute to differential expression of transcriptional signatures (<xref ref-type="supplementary-material" rid="pgen.1006820.s007">S7 Fig</xref>). A total of 178 European subjects (108 anti-Ro/SSA positive SS cases, 55 anti-Ro/SSA negative SS cases, and 15 healthy controls) for whom genotype data were also available were used in the <italic>cis</italic>- eQTL analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All SS cases used in the genetic association analyses were collected through the Sj&#246;gren&#8217;s Genetics Network and organized at the OMRF. Two datasets (Dataset 1A and 1B) were combined for the initial genetic association analysis (<xref ref-type="table" rid="pgen.1006820.t001">Table 1</xref>). All subjects in Dataset 1A have been previously described in a genome-wide association study [<xref ref-type="bibr" rid="pgen.1006820.ref015">15</xref>]. In Dataset 1B, 384 SS cases of European descent were genotyped and subjected to QC measurements outlined below. The genotype data of 3,315 population controls in Dataset 1B were obtained from the database of Genotypes and Phenotypes (dbGaP; <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/gap">http://www.ncbi.nlm.nih.gov/gap</ext-link>). Each SS case was genetically matched to five population controls in Dataset 1A and 1B, respectively, using the identity-by-state (IBS) to assess allele sharing as implemented in PLINK v1.07 [<xref ref-type="bibr" rid="pgen.1006820.ref091">91</xref>]. The remaining controls were used in the replication study (Dataset 2). A total of 622 SS cases and 3,502 population controls were subjected to QC in the replication phase (Dataset 2; <xref ref-type="table" rid="pgen.1006820.t001">Table 1</xref>).</p><p>For RNA-seq experiments, a total of 90 European subjects were evaluated, including 27 anti-Ro/SSA positive SS cases, 33 anti-Ro/SSA negative SS cases, and 30 healthy controls.</p><p>Total RNA was obtained by blood collection into PAXGene tubes (BD Company) and extracted following manufacturer&#8217;s protocols (Qiagen). Excess globin transcripts were removed using GLOBINclearTM (Ambion). RNA concentrations were determined using a NanoDrop spectrophotometer (Thermo Scientific) based on Optical Density values at A260. RNA quality was assessed by Agilent 2100 Bioanalyzer based on 28S/18S ribosomal RNA ratio and RNA integrity number. Double stranded cDNA was synthesized using a T7 promoter, and biotin-labeled cRNA was transcribed using the Illumina TotalPrep RNA Amplification System (Ambion). Samples were hybridized to Human WG-6 v3.0 BeadChip microarrays (Illumina) containing 48,803 mRNA probes in 37,805 unique genes per array. Microarrays were washed under high stringency and labeled with streptavidin-Cy3, and fluorescent intensity-based gene expression data were collected using Illumina&#8217;s BeadStation 500 scanner or iScan.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Unless otherwise stated, all statistical analyses were performed in the R Bioconductor suite. Microarray experiments were performed in two batches, with 93 SS cases and 34 healthy controls in Batch 1 and 89 cases and 42 controls in Batch 2. Raw intensity values for the two datasets were background subtracted separately using Illumina BeadStudio software. Identification of outlier and poor-performing samples was accomplished by applying the package arrayQualityMetrics (AQM) [<xref ref-type="bibr" rid="pgen.1006820.ref092">92</xref>] to log<sub>2</sub>-transformed microarray data from each experiment. QC measures were applied to each dataset to filter out transcripts expressed in &lt;10% of the subjects (detection call threshold <italic>P</italic>&lt;0.05) and probes with differential missingness rates (<italic>P</italic>&lt;0.001 by Fisher&#8217;s exact test) between the two datasets. The remaining probes were then compared against data tables from the Re-annotation and Mapping of Oligonucleotide Array Technologies (ReMOAT) [<xref ref-type="bibr" rid="pgen.1006820.ref093">93</xref>], in which Illumina BeadArray probe quality was extensively assessed and re-annotated. Each dataset was then independently normalized using Robust Multiarray Average (RMA) [<xref ref-type="bibr" rid="pgen.1006820.ref094">94</xref>], followed by log<sub>2</sub> transformation and quantile normalization. The ComBat program was subsequently applied to the combined dataset to adjust for non-biological experimental variation (i.e. batch effects) [<xref ref-type="bibr" rid="pgen.1006820.ref095">95</xref>]. Final re-annotation of un-annotated probes was performed manually using the NCBI probe database (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/probe">http://www.ncbi.nlm.nih.gov/probe</ext-link>) and UCSC Genome Browser alignment (<ext-link ext-link-type="uri" ns0:href="https://genome.ucsc.edu">https://genome.ucsc.edu</ext-link>) [<xref ref-type="bibr" rid="pgen.1006820.ref052">52</xref>]. Probes mapping to un-annotated genes or having inconsistent results from the two databases were removed from the analysis. The results and pipeline of the QC and normalization procedures applied to microarray data are summarized in (<xref ref-type="supplementary-material" rid="pgen.1006820.s008">S8 Fig</xref>). Gene expression comparisons were performed using Welch&#8217;s <italic>t</italic>-test between the mean expression values in SS cases and controls. The clustering of genes and samples was performed in Cluster 3.0 [<xref ref-type="bibr" rid="pgen.1006820.ref096">96</xref>] and visualized in Java TreeView [<xref ref-type="bibr" rid="pgen.1006820.ref097">97</xref>]. Unless otherwise noted, unsupervised hierarchical clustering was performed using centroid linkage with uncentered correlation for both genes and samples.</p><p>Genotyping for Dataset 1A was described previously [<xref ref-type="bibr" rid="pgen.1006820.ref015">15</xref>]. Genotypes for Dataset 1B were obtained using the Illumina OmniExpress arrays at OMRF following the manufacturer&#8217;s protocol. To increase informativeness, imputation was conducted in subjects from the merged datasets (Dataset 1) for the <italic>OAS1</italic> region (chr12: 113294739&#8211;113507712; hg19) meeting the criteria for suggestive association with SS (<italic>P</italic><sub><italic>assoc</italic></sub>&lt;1&#215;10<sup>&#8722;4</sup>). Imputation was performed using IMPUTE2 and the European Impute2 1000 Genomes Phase 1 April 2012 reference panel [<xref ref-type="bibr" rid="pgen.1006820.ref098">98</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006820.ref100">100</xref>]. A probability threshold of 0.9 and information score of &gt;0.5 were applied to the imputed genotypes in addition to the QC criteria described below for the association analyses. Genotypes for the replication study (Dataset 2) were determined using TaqMan probes and reagents (Life Technologies) following the manufacturer&#8217;s protocol.</p><p>Variants and samples in the genetic association analysis were subjected to a strict QC procedure as described previously [<xref ref-type="bibr" rid="pgen.1006820.ref015">15</xref>]: single-nucleotide polymorphism (SNP) call rate &gt;95% in all individuals; minor allele frequency &gt;1%; Hardy-Weinberg proportion test with a <italic>P</italic>&gt;0.001 in controls; and <italic>P</italic>&gt;0.001 for differential missingness between cases and controls. Samples from Dataset 1A and Dataset 1B passing QC were retained if results showed: &gt;95% call rate for all variants; no excessive increased heterozygosity (&gt;5 standard deviations from the mean); and no relatedness determined by identity-by-descent (IBD) &gt;0.4 using PLINK v1.07 [<xref ref-type="bibr" rid="pgen.1006820.ref091">91</xref>]. Population substructure was identified using EIGENSTRAT [<xref ref-type="bibr" rid="pgen.1006820.ref101">101</xref>] with independent genetic markers (<italic>r</italic><sup>2</sup>&lt;0.2 between variants). The resulting Eigenvectors were used to distinguish the four continental ancestral populations with the following HapMap samples: Africans (ASW, LWK, MKK, and YRI), Europeans (CEU and TSI), Hispanic and East Indians (MEX and GIH), and Asians (CHB, CHD, and JPT) [<xref ref-type="bibr" rid="pgen.1006820.ref102">102</xref>, <xref ref-type="bibr" rid="pgen.1006820.ref103">103</xref>]. The first two principal components (PCs) output by EIGENSTRAT were plotted and used to identify samples outside the European cluster [<xref ref-type="bibr" rid="pgen.1006820.ref101">101</xref>, <xref ref-type="bibr" rid="pgen.1006820.ref102">102</xref>] (<xref ref-type="supplementary-material" rid="pgen.1006820.s009">S9 Fig</xref>). Outliers from the European population were removed from further analysis. Population stratification analysis was performed for the replication cohort using genotype data from a previous study for the 622 SS cases [<xref ref-type="bibr" rid="pgen.1006820.ref015">15</xref>] and genotype data from the respective dbGaP studies.</p><p>A logistic regression model was used to test the association of genetic variants with SS susceptibility in PLINK v1.07 [<xref ref-type="bibr" rid="pgen.1006820.ref091">91</xref>]. The additive genetic model was calculated for variants within and 50kb flanking the 73 differentially expressed transcripts while adjusting for gender and the first three PCs as determined by the Scree test to evaluate the loading of each PC for the amount of variance explained [<xref ref-type="bibr" rid="pgen.1006820.ref104">104</xref>]. Meta-analysis of rs10774671 between the initial genetic association analysis (Dataset 1) and the replication study (Dataset 2) were calculated using a <italic>Z-</italic>score weighted by the sample size of each dataset in METAL [<xref ref-type="bibr" rid="pgen.1006820.ref105">105</xref>]. We tested our logistic regression model for deviation from additivity using PLINK in both the discovery dataset and replication dataset, and neither model shows significant deviation (<italic>p</italic> = 0.40 and <italic>p</italic> = 0.31, respectively). Logistic regression adjusting for the most significant variants (conditional analysis) was performed to determine independence of the association. LD and probable haplotypes were determined using Haploview [<xref ref-type="bibr" rid="pgen.1006820.ref106">106</xref>].</p><p>Quantitative levels of the differentially expressed transcripts from the microarray analysis were used as phenotypic traits in 178 European subjects described above. Variants showing nominal association with SS (<italic>P</italic><sub><italic>assoc</italic></sub>&lt;0.05) in the genetic association analysis were selected to test for <italic>cis</italic>-eQTLs, defined by variant-transcript pairs within 50kb of the target genes or <italic>trans</italic>-eQTLs for variants at least 1Mb away. Association of genotype with transcript expression was evaluated using both linear regression (adjusted for gender and disease status) and analysis of variance (ANOVA) in Matrix-eQTL [<xref ref-type="bibr" rid="pgen.1006820.ref107">107</xref>]. FDR-adjusted <italic>P</italic> values were calculated to determine the significance of the eQTL. The results of the <italic>cis</italic>-eQTL analyses were plotted in Prism 6. We also used a tool, PEER, based on a Bayesian framework to adjust for unknown non-genetic factors in gene expression [<xref ref-type="bibr" rid="pgen.1006820.ref108">108</xref>]. We transformed our expression values using all the genes that passed QC by running PEER for 15 factors. We then used the PEER residuals from the 44 SS-associated (<italic>P</italic><sub><italic>assoc</italic></sub>&lt;0.05) IFN signature genes as quantitative traits to determine eQTL while adjusting for other known potentially confounding factors: sex, disease status, anti-Ro/SSA status, and age.</p><p>In addition to additive genetic models, we also performed eQTL analyses using linear regression by three other models: recessive (recode genotype from 0,1,2 [where 2 equals to AA] to 0,0,1), dominant (0,1,1), and overdominant (0,1,0). We used the coefficient of determination (<italic>R</italic><sup>2</sup>) to evaluate the goodness-of-fit in each of these models. As shown in <xref ref-type="supplementary-material" rid="pgen.1006820.s013">S4 Table</xref>, both the p42 and p48 isoforms fit the additive model best (highest <italic>R</italic><sup>2</sup>), whereas the recessive model outperformed in the p46 isoform regression (<italic>R</italic><sup><italic>2</italic></sup><sub><italic>rec</italic></sub> = 0.57 vs. <italic>R</italic><sup><italic>2</italic></sup><sub><italic>add</italic></sub> = 0.54). However, the difference of the <italic>R</italic><sup><italic>2</italic></sup> between the dominant and additive models in the p46 eQTL analysis is subtle. Also, the outperformance of the recessive model cannot be confirmed by qPCR (where the additive model has the highest <italic>R</italic><sup><italic>2</italic></sup>). Therefore, we only reported the additive results in the main text. However, the alternative genetic models for the eQTL effect observed in different isoforms may reflect distinct disease mechanisms rendered by these isoforms, and thus detailed contribution of different isoforms on disease susceptibility warrant further functional study.</p><p>The co-localization analysis between genetic association and <italic>cis</italic>-eQTL results in the <italic>OAS1</italic> region was performed using eCAVIAR [<xref ref-type="bibr" rid="pgen.1006820.ref053">53</xref>]. We used the <italic>z</italic>-scores (calculated by &#946;/standard error) and the LD matrix (calculated using PLINK&#8212;r) from both the genetic association and <italic>cis</italic>-eQTL results as input and assumed one causal variant to obtain colocalization posterior probability (CLPP) scores for all the tested 453 variants in the <italic>OAS1</italic> region.</p><p>Peripheral blood mRNA transcripts from 27 anti-Ro/SSA positive SS cases, 33 anti-Ro/SSA negative SS cases, and 30 healthy controls were isolated and measured as described above. RNA-seq was performed using the Illumina HiSeq 2000 employing standard procedures. Multiplexing of 6 samples per lane was utilized. Post sequence data were processed with Illumina Pipeline software v.1.7. Quality of raw sequence data was assessed using FASTQC. We assessed the quality of each sample using AQM [<xref ref-type="bibr" rid="pgen.1006820.ref092">92</xref>] as described above. A total of 6 samples were removed from analysis due to significantly different expression patterns revealed by PC analysis. Raw FASTQ files were aligned to the human reference genome (hg19) using TopHat [<xref ref-type="bibr" rid="pgen.1006820.ref057">57</xref>] that aligns the reads across splicing junctions independent of gene annotations, which benefits <italic>de novo</italic> detection of alternative splicing events. The total gene transcript level was determined by normalized read counts (raw read counts divided by estimated size factor) in DESeq [<xref ref-type="bibr" rid="pgen.1006820.ref109">109</xref>].</p><p>To determine alternative splicing events, the reference-independent construction of the transcripts was performed using Cufflinks [<xref ref-type="bibr" rid="pgen.1006820.ref058">58</xref>] to identify transcripts &gt;1% of the most abundant isoform in each sample. We only kept the transcripts that were detected in more than 10% of the samples for further analysis. The previously annotated isoforms (p46, p42 and p48) and an un-annotated isoform identified across multiple samples (p44) were used as reference to reconstruct the isoforms of <italic>OAS1</italic>. The novel identified isoforms of <italic>OAS1</italic> were also checked manually in the Integrative Genomics Viewer (IGV) [<xref ref-type="bibr" rid="pgen.1006820.ref110">110</xref>] to confirm the transcripts and cross-exon reads. The FPKM values calculated by Cufflinks were used to determine the expression levels of each isoform of <italic>OAS1</italic>.</p><p>Total RNA was extracted using TRIzol reagents (Life Technologies) from EBV-immortalized B cells pre-selected for the presence of target <italic>OAS1</italic> isoforms based on the RNA-seq results from whole blood. Following DNase treatment (Life Technologies) and cDNA synthesis (iScript kit from Bio-Rad), full-length and truncated <italic>OAS1</italic> transcripts were amplified from cDNA using primer sets specific for the different <italic>OAS1</italic> isoforms and truncated forms (<xref ref-type="supplementary-material" rid="pgen.1006820.s012">S3 Table</xref>). Each <italic>OAS1</italic> isoform transcript was individually cloned into pcDNA3.1 (Invitrogen) with an Xpress epitope tag at the 5'-terminus to facilitate the detection of transfected protein using Western-blot with anti-Xpress antibody. The plasmid was transfected into the HEK 293T cells using FuGENE transfection reagents (Promega) following manufacturer&#8217;s protocols.</p><p>The protein expression of OAS1 isoforms was evaluated in EBV-immortalised B cells from SS patients, four independent samples from each genotype group GG, GA and AA, treated or not treated with type I interferon (universal type I IFN, 1500 U/mL, for 24 hours). The cells were lysed in RIPA buffer and cell lysate protein concentration determined using the Qubit Protein Assay kit (Thermo Fisher Scientific). A total of 30 &#956;g protein from each cell extract was separated on a 10% Bis-Tris gel (10% Criterion<sup>&#8482;</sup> XT Bis-Tris Gel, BioRad, Cat #: 3450112) following the manufacturer&#8217;s instructions, the gels cut according to the weight of the OAS1 protein, and simultaneously transferred to a single PVDF membrane, thus ensuring the comparability of Western blot bands from all gels. The OAS1 isoforms were visualized using an anti-OAS1 antibody targeting the shared epitope (Rabbit polyclonal anti-human OAS1, Abcam, Cat #: ab86343) and ECL Prime Western Blotting Detection Reagents (Amersham, Cat #: RPN2232).</p>