<p>The HCHS/SOL is a cohort of 16,415 self-identified Hispanic/Latino persons aged 18&#8211;74 years who were selected from households and census block groups in Chicago, IL, Miami, FL, Bronx, NY, and San Diego, CA, as previously described[<xref ref-type="bibr" rid="pgen.1006760.ref052">52</xref>]. Study participants self-identified as having Hispanic/Latino background in one of six sub-groups, with the total study population including 6,471 participants identifying as having a Mexican background, 2,728 as Puerto Rican, 2,348 as Cuban, 1,730 as Central American, 1,460 as Dominican, and 1,068 as South American. Individuals were recruited to HCHS/SOL between 2008 and 2011, and underwent a baseline clinical exam that included clinical, lifestyle, and sociodemographic assessment[<xref ref-type="bibr" rid="pgen.1006760.ref053">53</xref>]. Based on kinship coefficient among the genotyped individuals, the HCHS/SOL sample includes 204 parent-offspring trios, 1,042 parent-offspring duos, 699 full-sibling pairs, and numerous second- and third-degree relatives. The IRB committees for the HCHS Coordinating Center at UNC Chapel Hill, San Diego State University, University of Illinois at Chicago, University of Miami, and Yeshiva University-Albert Einstein College of Medicine have all reviewed and approved the informed consent documents and study protocol. Written and signed informed consents in the language preferred by the participants are administered and archived at each of the participating field centers. All participants in this publication from HCHS/SOL have consented to use of their genetic and non-genetic data. Anyone not providing consent has been excluded from this analysis. Demographic characteristics and RBC trait descriptive statistics for included study populations are presented in <bold><xref ref-type="supplementary-material" rid="pgen.1006760.s007">S2 Table</xref></bold>.</p><p>Whole blood (approximately 58 to 76ml) was collected at Visit 1 for all consenting HCHS/SOL participants by certified technicians trained at their respective field-center institutions. Supplies and procedures were standardized across all field centers; 4ml of whole blood for complete blood count (hemogram) was collected in a tube containing EDTA as an anticoagulant. CBC values were measured from whole blood using an automated hematology analyzer (Sysmex XE-2100, Sysmex America, Inc., Mundelein, IL 60060) at the central laboratory at the University of Minnesota Medical Center, Fairview, in Minneapolis.</p><p>Of the 16,415 individuals in the HCHS/SOL cohort study, 12,803 consented to genotyping and passed QC. Several individuals from the genotyped subset were excluded from the analysis, including individuals with predominantly Asian ancestry (n = 19), pregnant women (n = 8), participants with &gt;5% immature granulocytes (n = 2), end-stage kidney disease (n = 46), hematologic cancer (n = 28), or those undergoing cancer chemotherapy (n = 54). After exclusions, a total 12,502 participants were included for HCT, HGB, RBC, MCH, and MCV; 12,501 for RDW; and 12,500 for MCHC.</p><p>HCHS/SOL subjects who consented to genetic studies had DNA extracted from whole blood, which was genotyped on the Illumina SOL HCHS Custom 15041502 B3 array. This array comprised the Illumina Omni 2.5M array (HumanOmni2.5-8v1-1) and additional custom content[<xref ref-type="bibr" rid="pgen.1006760.ref051">51</xref>, <xref ref-type="bibr" rid="pgen.1006760.ref054">54</xref>]. In order to capture more Amerindian variation, the Omni2.5M array was modified by the addition of custom content comprised of ~150K SNPs selected from the CLM, MXL, and PUR 1000 Genomes Phase I samples for higher informativeness to identify Amerindian continental ancestry and for higher frequency in Amerindian genomic segments. Standard quality assurance/quality control (QA/QC) methods for SNP- and sample-level quality were applied. Quality metrics used to filter SNPs included Illumina/LA Biomed assay-failure indicator, missing call rate (&gt;2%), deviation from Hardy-Weinberg equilibrium (p&lt;10<sup>&#8722;5</sup>), Mendelian errors (&gt;3 in 1343 trios or duos), and duplicate sample discordance (&gt;2 in 291 sample pairs). Following genotyping QA/QC procedures, there were 12,803 unique study participants and 2,232,944 SNPs available for imputation.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For imputation, we used 1000 Genomes Project phase 1 reference panel and IMPUTE2 software. Genotypes were initially pre-phased using SHAPEIT2 (v2.r644, <ext-link ext-link-type="uri" ns0:href="http://www.shapeit.fr/">www.shapeit.fr</ext-link>), and subsequently imputed using IMPUTE2 software (v2.3.0, <ext-link ext-link-type="uri" ns0:href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html">https://mathgen.stats.ox.ac.uk/impute/impute_v2.html</ext-link>, last accessed Dec 2016)[<xref ref-type="bibr" rid="pgen.1006760.ref054">54</xref>]. Only variants with at least two copies of the minor allele present in any of the four 1000 Genomes continental panels were imputed, yielding a total of 25,568,744 imputed variants (SNPs and indels). Imputed genotype dosages were modeled on a continuous scale from 0 to 2 in order to account for genotype uncertainty. Oevar is an imputation quality metric, defined as the ratio of the observed variance of imputed dosage to the expected binomial variance. Variants with an oevar &lt;0.3 were considered low quality and excluded from analysis. Additional information about imputation and quality metrics is found in Conomos, et al[<xref ref-type="bibr" rid="pgen.1006760.ref055">55</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The SOL Illumina Omni 2.5M array contains five variants (rs2362744, rs4021971, rs4021965, rs11639532, rs2858942) within the 3,811bp alpha-globin structural variant that can be used for determining copy number. Raw probe intensity data (normalized X and Y values) were exported from GenomeStudio as FinalReport files and then imported into the Genvisis software package (<ext-link ext-link-type="uri" ns0:href="http://genvisis.org/">http://genvisis.org</ext-link>, last accessed Jan 2017) in order to use its specialized CNV calling algorithm. The first step in the process is to re-compute the Log R ratios (LRRs) using centroids derived from only high-quality samples (standard deviation of the autosomal LRRs &lt;0.32 and genotype call rate &gt;98%). LRRs from a set of ~50,000 curated markers were included in a principal components analysis (PCA) to capture DNA quality, DNA quantity, and batch effects. After regressing out 60 PCs from the raw intensity data, we recomputed LRRs and determined the median LRR value for the five markers in the alpha-globin region. Copy-number (0, 1, 2, 3, or 4) calling for the structural variant was then performed after visual inspection of the cluster boundaries with median LRR on the x-axis and median absolute difference on the y-axis. For RBC phenotype association analyses, genotypes were then coded and analyzed separately for the presence of the 3.8kb alpha-globin deletion (0, 1, or 2 copies) and the presence of the 3.8kb alpha-globin duplication (0, 1, or 2 copies).</p><p>For replication of discovery associations in HCHS/SOL, 1000 Genomes Project phase 1-imputed GWAS data were utilized from three Hispanic/Latino study populations. These included the Women's Health Initiative (WHI) SNP Health Association Resource (SHARe) project (n = 3,454), the Multi-Ethnic Study of Atherosclerosis (MESA) cohort (n = 782), and Mount Sinai Bio<italic>Me</italic> biobank (n = 2,854)[<xref ref-type="bibr" rid="pgen.1006760.ref056">56</xref>]. Genotyping in WHI-SHARe and MESA was performed using Affymetrix 6.0 array and imputation was performed with MaCH software[<xref ref-type="bibr" rid="pgen.1006760.ref057">57</xref>]. Bio<italic>Me</italic> was genotyped using the Illumina OmniExpressExome beadchip array, phasing was performed using ShapeIt Version 2 release 644 and imputation with Impute version 2.3 using the All 1000 Genomes Project phase 1 integrated variant set (Aug 2012) as the reference.</p><p>All outcomes were analyzed using linear mixed-effect models (LMMs), with random effects accounting for inter-individual correlation (due to either relatedness, shared household, or census block group). The covariates (fixed effects) included age, sex, five principal components, recruitment center, current cigarette smoking, sampling weight, and genetic analysis group (Cuban, Dominican, Puerto Rican, Mexican, Central American and South American)[<xref ref-type="bibr" rid="pgen.1006760.ref054">54</xref>]. When performing analysis on the X chromosome, we included the first two X chromosome-specific principal components as covariates. Additionally, pairwise genetic relatedness as estimated from the X chromosome was included as a random effect along with the autosomal genetic relatedness matrix. Additionally, since males have only one copy of X chromosome, genotypes on the X chromosome were coded 0, 1, 2 for females and 0, 2 for males. We also conducted three additional analyses for the known G6PD locus on the X chromosome: (1) sex-stratified analysis (<bold><xref ref-type="supplementary-material" rid="pgen.1006760.s015">S10 Table</xref>)</bold>; (2) genotype-specific analysis in women, since there is evidence for skewed X chromosome-inactivation with age[<xref ref-type="bibr" rid="pgen.1006760.ref058">58</xref>] (<bold><xref ref-type="supplementary-material" rid="pgen.1006760.s016">S11 Table</xref></bold>); and (3) age-genotype interaction analysis (<bold><xref ref-type="supplementary-material" rid="pgen.1006760.s017">S12 Table</xref></bold>).</p><p>More information about the principal components, kinship matrix computation, and the genetic analysis groups, is provided in Conomos, et al[<xref ref-type="bibr" rid="pgen.1006760.ref054">54</xref>]. Potential inflation was assessed using quantile-quantile plots of the test statistics against the standard normal distribution, and a calculated inflation factor <italic>&#955;</italic><sub><italic>gc</italic></sub>. We report genome-wide significant results at significance threshold of p-value &#8804;5.0x10<sup>-8</sup> and suggestive significance threshold of p-value &lt;1.0x10<sup>-7</sup> in the HCHS/SOL discovery sample for all variants with MAF = 0.01 and imputation oevar &gt;0.3 All SNPs exceeding genome-wide significance threshold of p-value &lt;1x10<sup>-7</sup> are described in <bold><xref ref-type="supplementary-material" rid="pgen.1006760.s014">S9 Table</xref></bold>.</p><p>Local ancestry estimates were previously inferred in the HCHS/SOL[<xref ref-type="bibr" rid="pgen.1006760.ref059">59</xref>]. A genome-wide admixture mapping scan was performed using a linear mixed model with covariates and random effects described above, jointly testing the three ancestries (European, African, Amerindian) at each available locus. On the basis of previous simulation results, a nominal p-value of 5.7x10<sup>-5</sup> yielded a genome-wide type I error of 0.05. There are currently no well-developed, validated methods available for local ancestry estimation on the X chromosome. Hence, we performed admixture mapping analysis only on the autosomes.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Association testing was performed in each of the three Hispanic replication data sets (WHI, Bio<italic>Me</italic>, MESA) using linear regression and the same RBC trait transformation as the discovery samples, adjusted for age, sex, and principal components. Meta-analysis of results from the 3 independent Hispanic replication study samples was performed using the inverse-variance-weighted method implemented in METAL (<ext-link ext-link-type="uri" ns0:href="http://csg.sph.umich.edu//abecasis/Metal">http://csg.sph.umich.edu//abecasis/Metal</ext-link>, last accessed Dec 2016). We defined novel, replicated loci as those which exceeded a Bonferroni-corrected significance threshold of p &lt;0.05/7, or 0.0071 (accounting for 7 SNPs carried forward for replication) and are located &gt;1 megabase (Mb) from a previously reported genome-wide significant association signal.</p><p>We performed step-wise conditional analysis for each RBC phenotype to identify secondary, independent association signals within 500kb of known and newly discovered GWAS loci. In the first round of the conditional analysis for each trait, we used the same regression model as in the discovery GWAS, with additional adjustment for previously reported or novel variants identified in this study. The list of variants used in conditional analysis of each trait is provided in <bold><xref ref-type="supplementary-material" rid="pgen.1006760.s018">S13 Table</xref></bold>. The significance threshold for discovering new, independent association signals was the same as the genome-wide discovery threshold (&#945; = 5.0x10<sup>-8</sup>) as well as MAF&#8805;0.01. Subsequent rounds of conditional analysis were repeated for each genomic region, adding the strongest genome-wide significant variant from the previous round as a covariate in the regression model, until no further genome-wide significant variants satisfying the MAF threshold remained in that region after covariate adjustment. The full models for each trait used in the final round of conditional analysis are listed in <bold><xref ref-type="supplementary-material" rid="pgen.1006760.s019">S14 Table</xref></bold>. After obtaining probe intensity-based CNV calls for the 3.8kb alpha-globin CNV, we conducted conditional analysis on chromosome 16 using the calls from the re-typed CNV. The full models for each trait used in these conditional analysis are also listed in <bold><xref ref-type="supplementary-material" rid="pgen.1006760.s019">S14 Table</xref>.</bold> Conditional analysis with the re-typed 3.8kb alpha-globin CNV was conducted on the subset of 12,390 individuals for whom the re-typed CNV calls were available.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Variants used in generalization analyses were identified by one of two inclusion methods: (1) any variant listed as genome-wide significant for any of the seven RBC traits in our study in the European Bioinformatics Institute GWAS catalog (<ext-link ext-link-type="uri" ns0:href="http://www.ebi.ac.uk/">http://www.ebi.ac.uk</ext-link>, last accessed Jan 2017); or (2) any RBC trait genome-wide-significant variants published in the main text or supplement of an English-language GWAS indexed in PubMed prior to December 2016. (Of note, we did not identify any GWAS published in a language other than English, hence we expect our list of variants identified using these methods to be complete prior to 2017.) We tested each published RBC-associated variant to see whether that association generalized to Hispanics/Latinos. The directional generalization null hypothesis is rejected if there is enough evidence that the published variant is directionally consistent and associated with the outcome in both the discovery study and HCHS/SOL. We evaluated for generalization all available signals previously reported in any GWAS published in English, for all seven traits evaluated in this paper (<bold><xref ref-type="supplementary-material" rid="pgen.1006760.s009">S4 Table</xref></bold>). Most of these SNPs were reported in studies of adults of European ancestry, but we also generalized associations from African- and Japanese-ancestry populations. No variants identified in Danjou, 2015, were included in our genotyped or imputed dataset and hence these variants could not be evaluated for generalization[<xref ref-type="bibr" rid="pgen.1006760.ref060">60</xref>]. To test the generalization null hypotheses, we computed directional FDR r-values for each of the tested SNPs. Directional r-values were calculated based on one-sided p-values from both the &#8220;discovery&#8221; study (reported in the literature) and the HCHS/SOL, and based on the number of tests performed in the discovery study, in order to properly account for multiple testing. A SNP was considered generalized if its r-value was &lt;0.05[<xref ref-type="bibr" rid="pgen.1006760.ref061">61</xref>]. In generalizing associations reported by Ganesh et al. (2009), we did not employ directional control since Ganesh, et al., (2009) did not report effect sizes or directions[<xref ref-type="bibr" rid="pgen.1006760.ref008">8</xref>]. The implication is a slight loss of power.</p><p>Generalization analysis was performed by looking up reported SNPs in HCHS/SOL results, in an analysis that mimics the analysis reported in the discovery study. For example, if a trait was reported as an association analysis with the natural-log-transformed trait, we performed the analysis with the same transformation in the HCHS/SOL population. In some cases, as with Kamatani, et al. (2010), we also matched effect-size reporting methods (standard deviations) for ease of comparison. Transformations, when applicable, are described in <bold><xref ref-type="supplementary-material" rid="pgen.1006760.s006">S1 Table</xref></bold>. Since the same SNP-trait association may be reported by multiple studies, we counted only unique SNP-trait associations. In instances where more than one study reported associations for the same SNPs and trait, but used different trait transformations, we selected the results from the generalization analysis in which the trait transformation matched our primary analysis.</p><p>Since some SNPs are associated with more than one RBC trait, and some genomic regions contain multiple SNPs associated with multiple traits, we summarize the generalization results as follows. Overall, we summarize the number of generalized unique trait-SNP associations (the same SNP may be counted more than once, if associated with more than one trait). Then, for each trait, we summarized (1) the number of unique SNPs, and (2) the number of unique genomic regions. To define genomic regions, we identified specific SNPs, and a 1Mb genomic region around them. Other SNPs within these regions were clumped together. We say that a genomic region generalized for a specific trait if at least one SNP in the region was associated with the trait.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We assessed any novel, replicated red blood cell associated loci to determine potentially causal variants. At each locus, we determined if the lead or proxy variants (r<sup>2</sup> &#8805; 0.8) were located within putative erythroid regulatory elements, defined on the basis of enrichment for various histone-modification and ChIP-Seq signals in either erythroblasts or the erythroleukemia cell line K562[<xref ref-type="bibr" rid="pgen.1006760.ref034">34</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006760.ref036">36</xref>]. We defined these regulatory regions as follows: enrichment for histone H3K4me1 as an enhancer, enrichment for histone H3K4me3 as a promoter. Variants located within a putative promoter or enhancer, and that overlapped a DNaseI hypersensitive site in proerythroblasts or K562 cells, were prioritized as putatively functional [<xref ref-type="bibr" rid="pgen.1006760.ref034">34</xref>, <xref ref-type="bibr" rid="pgen.1006760.ref036">36</xref>, <xref ref-type="bibr" rid="pgen.1006760.ref062">62</xref>]. Regulatory elements often are bound by transcription factors and hence we report ChIP-Seq peak overlaps of key erythroid transcription factors (GATA1, TAL1), and others in proerythroblasts and K562 cells to provide further support for the functional role of putative regulatory elements in erythroid cells[<xref ref-type="bibr" rid="pgen.1006760.ref034">34</xref>, <xref ref-type="bibr" rid="pgen.1006760.ref036">36</xref>, <xref ref-type="bibr" rid="pgen.1006760.ref062">62</xref>]. The ENCODE and BLUEPRINT datasets were accessed through the ENCODE analysis Hub and <ext-link ext-link-type="uri" ns0:href="http://ftp.ebi.ac.uk/pub/databases/blueprint/releases/current_release/homo_sapiens/hub/hub.txt#_blank">Blueprint Hub</ext-link> respectively via the UCSC genome browser[<xref ref-type="bibr" rid="pgen.1006760.ref063">63</xref>, <xref ref-type="bibr" rid="pgen.1006760.ref064">64</xref>]. Datasets from Xu, et al, were accessed from codex (<ext-link ext-link-type="uri" ns0:href="http://codex.stemcells.cam.ac.uk/">http://codex.stemcells.cam.ac.uk</ext-link>, last accessed Dec 2016)[<xref ref-type="bibr" rid="pgen.1006760.ref062">62</xref>, <xref ref-type="bibr" rid="pgen.1006760.ref065">65</xref>]. To hypothesize likely mode of action via which the causal variants influence the trait, we report eQTL targets and or motifs disrupted by prioritized variants using HaploReg v4.1[<xref ref-type="bibr" rid="pgen.1006760.ref066">66</xref>]. All the datasets used for functional annotation were mapped to Human GRCh37/hg19 assembly. Functional annotation is summarized in <bold><xref ref-type="supplementary-material" rid="pgen.1006760.s010">S5 Table</xref></bold>. We also used <italic>in silico</italic> prediction algorithms to annotate variants. These included RegulomeDB, the Combined Annotation Dependent Depletion (CADD) phred score, GWAVA, and deltaSVM[<xref ref-type="bibr" rid="pgen.1006760.ref067">67</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006760.ref070">70</xref>]. These annotations are summarized in <bold><xref ref-type="supplementary-material" rid="pgen.1006760.s011">S6 Table</xref></bold>.</p><p>The CRISPR/Cas9 system was used to mutagenize individual variants or small regions of interest identified during discovery analysis and subsequent bioinformatics interrogation. All oligonucleotide sequences used in CRISPR-Cas9 genome editing experiments are listed in <bold><xref ref-type="supplementary-material" rid="pgen.1006760.s012">S7 Table</xref>.</bold> The human umbilical cord blood derived erythroid progenitor cell line #2 (HUDEP-2) was cultured and used for genome editing as previously described[<xref ref-type="bibr" rid="pgen.1006760.ref071">71</xref>]. Individual and tandem pairs of single chimeric guide RNAs were cloned to lentiviral expression vectors (lentiGuide-Puro, Addgene plasmid 52963). Cells were transduced and selected for lentiviral integrants by antibiotic selection (10 &#956;g/ml blasticidin for lentiCas9-Blast [Addgene plasmid 52962], 1 &#956;g/ml puromycin for lentiGuide-Puro). For <italic>SLC12A2</italic> individual sgRNA promoter editing, indel frequencies were assessed after 7 days by nested PCR followed by amplicon deep sequencing. For <italic>SLC12A2-LINC01184</italic>, <italic>PSMB5</italic>, and <italic>PROX1</italic> interstitial deletions, cells were plated at limiting dilution to isolate clones 7 days after transduction with tandem sgRNAs. Clones with biallelic deletions were characterized by presence of gap PCR amplification with primers outside the deleted segment and absence of PCR amplification from inside the deleted segment. Expression of mRNA of genes of interest was compared to GAPDH expression using quantitative reverse transcription PCR (RT-qPCR) in control and edited HUDEP-2 cells. For <italic>SLC12A2</italic> individual sgRNA promoter editing, the total population of edited cells was evaluated in bulk by RT-qPCR. For <italic>SLC12A2-LINC01184</italic>, <italic>PSMB5</italic>, and <italic>PROX1</italic> interstitial deletions, clones were first identified by PCR screening and then evaluated by RT-qPCR. For differentiation experiments, control and edited HUDEP-2 cells were cultured separately for 4 days in Erythroid Differentiation Media (EDM) with Iscove&#8217;s Modified Dulbecco&#8217;s Medium (IMDM) (Life Technologies) supplemented with 330 mg/ml holo-transferrin (Sigma), 10 mg/ml recombinant human insulin (Sigma), 2 IU/ml heparin (Sigma), 5% human solvent detergent pooled plasma AB (Rhode Island Blood Center), 3 IU/ml erythropoietin, 100 ng/ml human SCF, (R&amp;D), 1 mg/ml doxycycline, 1% L-glutamine, and 2% penicillin/streptomycin. Subsequently the cells were cultured an additional 4 days in EDM lacking SCF, and then an additional 4 days in EDM lacking both SCF and doxycycline. Erythroid maturation was evaluated by flow cytometry staining with CD71 (eBiosciences), CD235a (eBiosciences), CD49f (Miltenyi), and DRAQ5 (eBiosciences) as well as morphology by May-Grunwald-Giemsa staining, Student's t-tests were used for statistical analysis of results.</p><p>Genotype data and GWAS results of discovery analysis of all the seven RBC traits can be requested via dbGaP study accession phs000880. Phenotype data can be requested via dbGaP study accession phs000810</p>