<p>Mouse immortalized melanocytes, Ink4a-ARF double null melan-a cells, were maintained as described previously [<xref ref-type="bibr" rid="pgen.1006636.ref105">105</xref>].</p><p>Primary melanocytes were regularly maintained in culture as described previously [<xref ref-type="bibr" rid="pgen.1006636.ref106">106</xref>], using complete growth medium consisting of Minimum Essential Medium Eagle (MCDB) 153 supplemented with 4% fetal bovine serum, insulin (5 &#956;g/ml), &#945;-tocopherol (1 &#956;g/ml), 1% penicillin/streptomycin/amphotericin, human basic fibroblast growth factor (bFGF, 0.6 ng/ml), phorbol 12-myristate 13-acetate (PMA, 8 nM), and bovine pituitary extract (BPE, 13 &#956;g/ml). Primary melanocytes were grown in T-75 cm<sup>2</sup> flasks. To split flasks, cells were washed for 2 min in EDTA solution and trypsinized for 2 min in 500 &#956;l of a 0.25% trypsin solution.</p><p>The M21 melanoma cell line was obtained from the American Type Culture Collection (ATCC, Manassas, VA), and maintained in RPMI containing 10% FBS and 1% PenStrep.</p><p><italic>tfap2a</italic><sup><italic>-/-</italic></sup> mutant [lockjaw allele, <xref ref-type="bibr" rid="pgen.1006636.ref028">28</xref>] and non-mutant siblings (i.e. wildtype and <italic>tfap2a</italic><sup><italic>+/-</italic></sup> heterozygous) zebrafish larvae were sorted at 36 hpf based on pigmentation phenotype, and heads were removed to eliminate the RPE. Pools of 25&#8211;30 embryos were collected in TRIzol (Invitrogen), and RNA was extracted according to the manufacturer protocol. RNA was further purified using the RNeasy Mini Kit (Qiagen) and tested for quality using an Experion bioanalyzer (Bio-Rad). The SuperScript Double-Stranded cDNA Synthesis Kit (Invitrogen) was used to generate ds-cDNA, which was also tested for quality on the Experion bioanalyzer. ds-cDNA was labeled using Cy3-coupled random nanomers (Nimblegen). Biological replicates were hybridized to sub-arrays of Roche Nimblegen gene expression microarrays (12 x 135K, design number 090505_Zv7_EXPR). The arrays were scanned on an Axon GenePix 4200A microarray scanner (Molecular Devices). Raw data (Pair files) were normalized in ArrayStar software, version 12.0.0 (DNASTAR, Inc). Background correction, quantile normalization, and median polish summarization were performed by applying the robust multichip analysis (RMA) algorithm. A Student&#8217;s <italic>t</italic>-test, corrected for multiple testing by the Benjamini and Hochberg false discovery rate (FDR) method, was performed for each pair-wise comparison. Transcripts were considered differentially expressed if the FDR-adjusted P-value was less than 0.05. Primers for qRT-PCR were designed to amplify fragments of 150&#8211;200 bp spanning exon junctions (listed in <xref ref-type="supplementary-material" rid="pgen.1006636.s021">S11 Table</xref>). qRT-PCR was carried out using iQ SYBR Green Supermix (Bio-Rad) on a CFX96 Real-Time PCR Detection system (Bio-Rad) according to the default settings. Three technical replicates were averaged for each sample, and the 2<sup>-&#916;&#916;Ct</sup> method was used to calculate the difference in gene expression between samples, after normalization to the reference gene <italic>b2m</italic>. Statistical significance was evaluated via a Mann-Whitney U test for each gene.</p><p>Melan-a cells were transfected in triplicate with either of two independent siTfap2a double strand duplex RNAs, siA = sure select siRNA #s74846 (Ambion), si5 = MMS.RNAIN011547.12.5 (Integrated DNA Technologies), or a non-silencing control siRNA NC-1 (Integrated DNA Technologies) using Lipofectamine RNAimax protocol (Invitrogen) with 25pmol of oligo per 6 well dish. Total RNA isolated from each bioreplicate was used to synthesize cDNA for hybridization to Mouse Gene 2.0 ST Arrays (Affymetrix). cDNA from each bioreplicate was then assayed by qRT-PCR with 3X technical replicates using TAQMAN qPCR probes (ABI, listed in <xref ref-type="supplementary-material" rid="pgen.1006636.s021">S11 Table</xref>), on a StepOne machine (Thermo Fisher) with standard fast protocol conditions. Relative sample quantifications were performed using a standard curve method, where gene expression for each bioreplicate was normalized to <italic>Actb</italic>, then plotted relative to non-silencing control samples. Statistical significance was evaluated via one-way ANOVA with Bonferroni multiple test correction for each gene when response to both siA and si5 agreed.</p><p>Melan-a cell ChIP-seq experiments (2 replicates) were performed as described by Gorkin and colleagues with the following alterations [<xref ref-type="bibr" rid="pgen.1006636.ref065">65</xref>]: sonication was performed using a Q500 horn sonicator (Qsonica), 70% amplitude, 25 minute sonication cycle occurring with a 10 second pulse, 20 second off setting. Peaks that overlap by at least 1 bp between replicates were used for subsequent analyses. Anti-TFAP2A and IgG ChIP samples from mouse melan-a cells were validated by ChIP-qPCR with primer sets listed in <xref ref-type="supplementary-material" rid="pgen.1006636.s021">S11 Table</xref>. Quantification of PCR products was performed as per manufacturer&#8217;s instructions using FAST SYBR Green master mix (ABI) and the Step-One Plus Real-time PCR machine (ABI) under FAST standard cycling conditions.</p><p>ChIP was performed on human primary melanocytes that had not exceeded 10 passages using 10 &#956;g anti-TFAP2A (3B5, Santa Cruz Biotechnology, Inc.) or control Ab-108C polyclonal Goat IgG control (R&amp;D Systems). Over 45 million mapped reads were obtained for one biological replicate and one input control sample. Approximately 40 million human primary melanocytes were fixed at room temperature for 10 minutes with 1% formaldehyde diluted in cell culture media (v/v). Following fixation, formaldehyde was quenched with 1.25M glycine for 5 minutes at room temperature. Fixed/quenched cells were subsequently spun at 1,200 rpm for 5 minutes, washed with 1x PBS, and respun. Pelleted cells were then resuspended in 1mL cell lysis buffer (150mM NaCl, 10mM Hepes pH 7.4, 1.5mM MgCl2, 10 mM KCl, 0.5% NP-40, 0.5mM DTT, 1mM EDTA, plus protease inhibitors) and incubated on ice 10 minutes. Following cell lysis, nuclei were pelleted by spinning at 5,000 rpm, for 5 minutes at 4&#176;C. Nuclei were then resuspended in 1 part nuclear lysis buffer (50mM Tris pH 8.0, 10mM EDTA, 1% SDS plus protease inhibitors) and 2 parts ChIP-dilution buffer (150mM NaCl, 16.7mM Tris pH 7.5, 3.3mM EDTA, 1% Triton X-100, 0.1% SDS, 0.5% Na-Doc, plus protease inhibitors), and sonicated using either a Covaris instrument (Model S220) or with a standard probe-tip sonicator (VirTis Virsonic 600). Sonication settings were determined empirically to generate fragments ranging from 200&#8211;500 bp. Following sonication samples were spun at 14,000 rpm for 15 minutes, and supernatant was saved for immunoprecipitation. For immunoprecipitation, Dynabeads (Life Technologies) were first washed 3x with ChIP-dilution buffer and subsequently incubated with equal amounts (5&#8211;10ug) of either anti-TFAP2A (3B5, Santa Cruz Biotechnology, Inc.) or Mouse IgG (Millipore) for 2 hours to overnight at 4&#176;C. Following incubation, antibody-bound beads were washed 3x with ChIP-dilution buffer. Sonicated chromatin was equally split between two tubes, brought to approximately 1 mL with ChIP-dilution buffer, and used to resuspend antibody bound beads. Samples were incubated at 4&#176;C overnight. Antibody/bead/chromatin complexes were then washed with a series of solutions 3x and eluted with 100 &#956;L elution buffer. Crosslinks were reversed overnight at 65&#176;C with 0.2 M NaCl. DNA was further cleaned using a Qiagen PCR purification kit as per manufacturer&#8217;s instructions (Qiagen). Total enriched DNA was quantified using a PicoGreen dsDNA assay kit (Life Technologies) and shearing efficiency was confirmed using a High Sensitivity DNA Analysis Kit and a 2100 Bioanalyzer (Agilent Technologies) on the input sample. Libraries were then constructed using ABI's SOLiD ChIP-seq kit with barcoding (rev. 08/06/2010), as per manufacturer&#8217;s instructions (Life Technologies). Following library construction, samples were sequenced using the ABI SOLiD 3.0 platform, using a 50bp sequencing run. LifeScope Genomic Analysis Software was used for basecalling and alignment to hg19 genome with default specifications for each program. Quality assessment was performed using Phantompeak tools as per the guidelines of the Encode project. Peaks were called using the SPP R program with an FDR setting of 0.05.</p><p>ChIP-qPCR was performed to validate chromatin enrichment at known TFAP2A targets using primer sets listed in <xref ref-type="supplementary-material" rid="pgen.1006636.s021">S11 Table</xref>. Quantification of PCR products was performed as per manufacturer&#8217;s instructions using iQ SYBR Green Supermix (Bio-Rad) on a CFX96 Real-Time PCR Detection system (Bio-Rad) according to the default settings. Three technical replicates were averaged for each sample, and the 2<sup>-&#916;&#916;Ct</sup> method was used to calculate the differences in gene expression.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The ChIP-seq Tool Set in Galaxy was used for all peak overlap analyses [<xref ref-type="bibr" rid="pgen.1006636.ref107">107</xref>]. For comparison of human ChIP-seq peaks to gene expression, we used a published RNA-seq expression profile of human penis foreskin melanocytes from the Roadmap Epigenomics Project (GEO accession number, GSM958174) [<xref ref-type="bibr" rid="pgen.1006636.ref064">64</xref>]. Motif enrichment analysis was carried out using the MEME-ChIP suite [<xref ref-type="bibr" rid="pgen.1006636.ref108">108</xref>], including CentriMo [<xref ref-type="bibr" rid="pgen.1006636.ref060">60</xref>], MEME-ChIP [<xref ref-type="bibr" rid="pgen.1006636.ref069">69</xref>], and AME [<xref ref-type="bibr" rid="pgen.1006636.ref109">109</xref>] tools. For gene set enrichment analysis, we used the Genomic Regions Enrichment of Annotations Tool (GREAT), with the association rule basal plus extension, proximal: 5 kb upstream, 1 kb downstream, plus distal: up to 100 kb [<xref ref-type="bibr" rid="pgen.1006636.ref072">72</xref>]. ChIP-seq read density clustering analysis and quantitative comparisons were performed using <italic>k</italic>-means linear enrichment cluster function in seqMINER with the following parameters: window size = -5K to +5K, read extension = 200bp, seed = 12 [<xref ref-type="bibr" rid="pgen.1006636.ref110">110</xref>] (<ext-link ext-link-type="uri" ns0:href="http://bips.u-strasbg.fr/">http://bips.u-strasbg.fr/</ext-link>). The Panther Classification System was used for GO term enrichment analysis on gene lists [<xref ref-type="bibr" rid="pgen.1006636.ref076">76</xref>,<xref ref-type="bibr" rid="pgen.1006636.ref077">77</xref>].</p><p>Human foreskin melanocyte H3K27ac ChIP-seq alignment results were obtained from the Roadmap Epigenomics Project (GEO accession number, GSM1127072) [<xref ref-type="bibr" rid="pgen.1006636.ref064">64</xref>]. Typical enhancers and super-enhancers were called using HOMER [<xref ref-type="bibr" rid="pgen.1006636.ref111">111</xref>]. Briefly, all the stitched enhancer peaks were generated and sorted based on normalized tag count in descending order. Super-enhancers were defined by slope&gt;1 (slope = &#916;(normalized tag count) / &#916;rank) [<xref ref-type="bibr" rid="pgen.1006636.ref084">84</xref>]. All stitched enhancers were compared with human melanocyte TFAP2A ChIP-seq peaks and MITF ChIP-seq peaks [<xref ref-type="bibr" rid="pgen.1006636.ref018">18</xref>] using BEDTools (v. 2.24.0) [<xref ref-type="bibr" rid="pgen.1006636.ref112">112</xref>]. We then normalized the rank by the total number of stitched enhancers and plotted this against normalized super-enhancer score (obtained by calculating normalized tag counts / highest normalized tag count).</p><p>RNA sequencing (RNA-seq) libraries were prepared with the TrueSeq stranded mRNA kit (Illumina) and sequenced on the Illumina HiSeq 2000 platform. The ten 5&#8217;-most bases were trimmed from all of the raw RNA-seq reads. Reads were aligned to the mouse reference genome sequence (mm9) using the STAR alignment software (v. 2.3.0e). RNA-seq reads derived from rRNAs were removed using the split_bam.py script available in RSeQC (v. 2.3.7), using genomic locations of known rDNAs that were downloaded from UCSC. Counts for RNA-seq reads mapping to Ensembl-annotated transcripts (release 67) were calculated using the htseq-count software (v. 0.5.3p3). These raw RNA-seq counts were used for differential gene expression analysis that was performed using DESeq2 (v. 1.10.1).</p><p>The following versions of the <italic>TRPM1</italic> promoter sequence (748 bp) were obtained as double stranded gBlocks gene fragments from IDT: intact sequence, &#916;AP2A with mutations in four TFAP2A binding sites, and &#916;E1 with a mutation in the main MITF binding site. The consensus TFAP2A binding site G<underline>C</underline>CNNNG<underline>G</underline> was disrupted by changing the two underlined bases to T, whereas the E1 MITF site was changed as previously published by [<xref ref-type="bibr" rid="pgen.1006636.ref086">86</xref>]. Fragments were cloned into a Tol2-cfos-FFluc vector via Gibson assembly and confirmed with Sanger sequencing.</p><p>M21 melanoma cells were grown to 70&#8211;90% confluency in a 24-well culture plate. In each well, the reporter plasmid (1&#956;g) and a &#946;-galactosidase plasmid (100ng) were transfected using Lipofectamine 3000 (Life Technologies). Approximately 48 hours after transfection, luciferase assays were conducted using the Dual-Light System from Applied Biosystems, and 20/20n Luminometer (Turner Biosystems, Sunnyvale, CA) according to the manufacturer protocols. Briefly, cells were washed with cold PBS and incubated in 40 &#956;L lysis solution on a shaker for 15 minutes. Cell lysates were transferred to tubes and centrifuged at 12000 rpm for 2 minutes at 4&#176;C, and supernatant was transferred to a clean tube. 10 &#956;L of cell lysate was added to 25 &#956;L of Buffer A and placed in the luminometer, where the injector adds 100 &#956;L Buffer B/Galacton-Plus substrate and reads the signal after 1 second. Samples were incubated in the dark for 30 minutes before injection of 100 &#956;L Accelerator-II and measurement of the &#946;-gal signal after 1 second. For each version of the <italic>TRPM1</italic> promoter element, transfection was carried out in triplicate. Firefly luciferase reads from each sample were normalized to the respective &#946;-gal reads, and an average signal was calculated within groups.</p><p>Experiments utilizing mice in this study were carried out in strict accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health. The protocol was approved by the Institutional Animal Care and Use Committee of the University of Colorado Denver. Noon on the day a copulatory plug was present was denoted as embryonic day 0.5. Mice used in this study included males that were heterozygous for a <italic>Tfap2a</italic>-null allele [<xref ref-type="bibr" rid="pgen.1006636.ref113">113</xref>], heterozygous for a newly generated <italic>Tfap2b</italic>-null allele (EVO/TW, in preparation), and hemizygous for the <italic>Wnt1-Cre</italic> transgene [<xref ref-type="bibr" rid="pgen.1006636.ref089">89</xref>] (<italic>Tfap2a</italic><sup><italic>null/wt</italic></sup><italic>;Tfap2b</italic><sup><italic>null/wt</italic></sup><italic>;Wnt1-Cre</italic>). These males were crossed with females that were homozygous for both a <italic>Tfap2a</italic>-conditional [<xref ref-type="bibr" rid="pgen.1006636.ref024">24</xref>] and <italic>Tfap2b</italic>-conditional (EVO/TW, in preparation) allele, resulting in a 1:8 frequency of generating single or double conditional mutants, as well as various other genotype combinations. Mice were maintained on an outbred Black Swiss background. Of note, by virtue of this breeding scheme, the single conditional mutants would always be conditionally heterozygous for the alternate paralog. For <italic>r26r</italic> [<xref ref-type="bibr" rid="pgen.1006636.ref090">90</xref>] and &#8216;tomato&#8217; [<xref ref-type="bibr" rid="pgen.1006636.ref114">114</xref>] experiments, the female was also homozygous for the reporter allele. Yolk sacs or tail clips were used for genotyping. DNA for PCR was extracted using DirectPCR Lysis Reagent (Viagen Biotech. Inc) plus 10 ug/ml Proteinase K (Roche) followed by heat inactivation at 85&#176;C for 45 min. Samples were then used directly for PCR-based genotyping using allele specific primers (available upon request) at a concentration of 200 nM using the Qiagen DNA polymerase kit, including the optional Q Buffer solution (Qiagen).</p><p>With the exception of whole-mount &#946;-galactosidase (&#946;-gal) staining, procedures used for mouse embryo analysis (<italic>in situ</italic> hybridization, &#593;-neurofilament immunostaining, and immunofluorescence) have all been previously described [<xref ref-type="bibr" rid="pgen.1006636.ref115">115</xref>]. Following euthanasia, embryos were collected at the indicated time-points in DEPC-PBS and subsequently processed. Briefly, for <italic>in situ</italic> hybridization, after embryo collection, trunks were removed, fixed overnight in 4% paraformaldehyde (PFA), and stained using the indicated riboprobe (<italic>Pmel</italic>, <italic>Dct</italic>, <italic>Sox10</italic>). For immunostaining, embryos were processed using an &#593;-neurofilament primary antibody [<xref ref-type="bibr" rid="pgen.1006636.ref116">116</xref>] (IgG clone 2H3, obtained from the Developmental Studies Hybridoma Bank&#8211;University of Iowa), followed by colorimetric staining using a standard secondary antibody and DAB-detection. For &#946;-gal immunofluorescence, collected embryos underwent a short fixation in 0.25% glutaraldehyde, were taken through a series of sucrose/O.C.T. solutions (Tissue-Tek O.C.T. Compound, Electron Microscopy Sciences), until being embedded in 100% O.C.T. and frozen on dry-ice. Cryosections were then cut at ~12&#956;M on a Leica CM 1900 cryostat (Leica Biosystems Inc.) using the hindlimb as a delimiting rostral-caudal boundary between samples. Following sectioning, slides were washed in PBS, 2 x 10 min, blocked in 3% BSA (in PBS) 1 hr at room temperature, incubated over-night with a rabbit polyclonal anti-&#946;-gal antibody (Product 55976, MP Biomedicals, LLC) diluted 1:200 in block solution. Subsequently, sections were washed 2 x 10 min in PBS, followed by a 1 hr incubation in goat-anti-rabbit Alexa Fluor 488 (Thermo Fisher) and counterstained with DRAQ5 (Abcam), washed again 2 x 10 min in PBS, and then cover-slipped. Processed samples were imaged on a Leica TCS SP5 II confocal microscope and representative images taken. Finally, for &#946;-gal staining, embryos were collected at appropriate time points and fixed ~1hr at room temperature with 0.25% glutaraldehyde in PBS. Subsequently, embryos were washed 3 x 30 minutes in a &#8216;lacZ rinse buffer&#8217; (0.2M sodium phosphate, 2mM magnesium chloride, 0.02% NP40, and 0.01% sodium deoxycholate), and then incubated overnight in a &#8216;lacZ staining solution&#8217; (lacZ rinse buffer plus 5mM potassium ferricyanide, 5mM potassium ferrocyanide, and 1 mg/ml X-gal) at 37&#176;C. Following adequate staining, embryos were post-fixed in 4% PFA overnight, moved to PBS, and subsequently imaged.</p><p>Zebrafish alleles used in this study include <italic>tfap2a</italic><sup><italic>low</italic></sup> [lockjaw allele, <xref ref-type="bibr" rid="pgen.1006636.ref028">28</xref>], <italic>mitfa</italic><sup><italic>w2</italic></sup> [<xref ref-type="bibr" rid="pgen.1006636.ref094">94</xref>], and <italic>mitfa</italic><sup><italic>z25</italic></sup> [<xref ref-type="bibr" rid="pgen.1006636.ref095">95</xref>]. Lines were maintained as <italic>mitfa</italic> trans-heterozygous and <italic>tfap2a</italic> heterozygous (<italic>mitfa</italic><sup><italic>w2/z25</italic></sup><italic>;tfap2a</italic><sup><italic>+/-</italic></sup>) or <italic>mitfa</italic> wildtype and <italic>tfap2a</italic> heterozygous (<italic>tfap2a</italic><sup>+/-</sup>). For genetic interaction experiments, <italic>mitfa</italic><sup><italic>w2/z25</italic></sup><italic>;tfap2a</italic><sup><italic>+/-</italic></sup> animals were incrossed to generate <italic>tfap2a</italic><sup><italic>+/+</italic></sup>, <italic>tfap2a</italic><sup><italic>+/-</italic></sup>, and <italic>tfap2a</italic><sup><italic>-/-</italic></sup> genotypes in each of <italic>mitfa</italic><sup><italic>z25/z25</italic></sup> and <italic>mitfa</italic><sup><italic>w2/z25</italic></sup> backgrounds. To obtain control animals in the <italic>mitfa</italic><sup><italic>+/+</italic></sup> background, <italic>mitfa</italic><sup><italic>+/+</italic></sup><italic>;tfap2a</italic><sup><italic>+/-</italic></sup> animals were also incrossed. At 72 hpf, embryos were binned according to melanocyte phenotype, photographed, and genotyped via High Resolution Melt Analysis (HRMA) using Precision Melt Supermix (Bio-Rad) on a CFX96 Real-Time PCR Detection system (Bio-Rad) according to the default settings. Precision Melt Analysis Software (Bio-Rad) was used to analyze melt curves. HRMA primer sequences are as follows: <italic>tfap2a</italic><sup><italic>low</italic></sup> (forward: GTA GCT ATG TTT CGT GGT TA; reverse: ACA ATA AGC AGC TGC TTT AC), <italic>mitfa</italic><sup><italic>z25</italic></sup> (forward: GCA GAA GTC AGA GCC CTG GC; reverse: ACG GAT CAT TTG ACT TGG GAA TTA AAG). Ventral stripe melanocytes were counted from lateral-view images taken at 72 hpf and statistical significance between groups was tested using a one-way ANOVA with multiple test correction.</p><p>For zebrafish rescue experiments, <italic>mitfa</italic><sup><italic>w2/z25</italic></sup> trans-heterozygotes were incrossed to obtain <italic>mitfa</italic><sup><italic>z25/z25</italic></sup>, <italic>mitfa</italic><sup><italic>w2/z25</italic></sup>, and <italic>mitfa</italic><sup><italic>w2/w2</italic></sup> genotypes, which were clearly distinguishable based on melanocyte phenotype at 48 hpf. 3-way Gateway cloning technology was used to generate a pDestTol2CG2 plasmid [<xref ref-type="bibr" rid="pgen.1006636.ref117">117</xref>] containing 5&#8217; entry <italic>mitfa</italic> promoter [<xref ref-type="bibr" rid="pgen.1006636.ref096">96</xref>], middle entry <italic>tfap2a</italic> cDNA, and 3&#8217; entry 6x Myc epitope tag followed by polyA, resulting in the presence of Myc epitope tags on the carboxy terminus of Tfap2a. This construct was then injected into the above cross along with Tol2 mRNA, both at a concentration of 30 ng/uL. Due to the presence of a <italic>cmlc2</italic>:<italic>GFP</italic> reporter in the vector, injected embryos were screened for successful plasmid integration based on expression of GFP in the heart at 28&#8211;30 hpf. GFP positive embryos and uninjected control embryos were fixed overnight in 4% paraformaldehyde at 48 hpf. After fixation, embryos were rinsed 3x in PBS, blocked in PBDT+2.5% goat serum for 1 hr at room temperature, and incubated in anti-Myc primary overnight at 4&#176;C (9E10, obtained from the Developmental Studies Hybridoma Bank&#8212;University of Iowa, 1:100 diluted in block solution). The following day, embryos were rinsed 4x in PBS + 0.1% Triton X-100 (PBS-Tx) and incubated overnight at 4&#176;C in Alexa Fluor goat-anti-mouse 488 secondary (Thermo Fisher) diluted in block solution. Embryos were then rinsed 4x15 minutes in PBS-Tx and mounted on slides. Ten individual embryos of each genotype with plasmid injection or uninjected control were viewed at 40x and photographed.</p><p>Cells were grown to approximately 75% confluency in 24-well plate on poly-lysine coated disks and fixed with 4% paraformaldehyde for 1 hr at 4&#176;C. Following fixation, cells were rinsed 3x20 minutes with 1x PBS, permeabilized in 1x PBS + 0.3% Triton X-100 for 30 min at 37&#176;C, rinsed 3x10 minutes in PBS-Tx, and blocked (1% BSA in PBS-Tx) overnight at 4&#176;C. Cells were then incubated in anti-TFAP2A primary overnight at 4&#176;C (3B5, Santa Cruz Biotechnology, Inc., 1:100, 250ug/mL stock diluted in block solution). The following day, cells were rinsed 3x20 minutes in PBS-Tx and incubated for 2 hr at room temperature in Alexa Fluor goat-anti-mouse 488 secondary (Thermo Fisher) diluted in block solution. Cells were then rinsed 4x15 minutes in PBS-Tx and disks mounted in Prolong Gold antifade reagent and DAPI SlowFade solution (Life Technologies). Slides were imaged on a Zeiss LSM 700 Flexible Confocal Microscope (Carl Zeiss Microscopy). Briefly, for quantification of immunofluorescent intensity approximately three 20x images were taken of each slide, and three slides of each cell line processed (including a no primary control). Each image included both the antibody of interest as well as DAPI staining to identify cell nuclei.</p><p>All experiments were approved by the University of Iowa or University of Colorado Institutional Animal Care and Use Committee (IACUC). We abide by PHS Policy, USDA-Animal Welfare regulations, the Guide for the Care and Use of Laboratory Animals, University of Iowa policies and regulations and any state and local laws and regulations.</p>