<p>A <italic>Setaria</italic> F7 RIL population comprised of 217 individuals was used for genetic mapping. The RIL population was generated through an interspecific cross between the wild-type <italic>S</italic>. <italic>viridis</italic> accession, A10, and the domesticated <italic>S</italic>. <italic>italica</italic> accession, B100 [<xref ref-type="bibr" rid="pgen.1006841.ref049">49</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006841.ref051">51</xref>]. Plant height measurements of individuals within this population were collected in six different trials. Four of the six trials were conducted at field sites located at University of Illinois-Champaign. Two field trials designed to assess plant phenotypic response to planting density and water availability were each conducted in both 2013 and 2014. The controlled environment trials were performed in a growth chamber at Carnegie Institute for Science, Stanford, CA and using the Bellweather Phenotyping Facility at Donald Danforth Plant Science Center [<xref ref-type="bibr" rid="pgen.1006841.ref052">52</xref>].</p><p>Field experiments were conducted on the South Farms at the University of Illinois Urbana-Champaign in summers 2013 and 2014. The field site is rain-fed, tile-drained, has a deep and organically rich Flanagan/Drummer series type soil [<xref ref-type="bibr" rid="pgen.1006841.ref053">53</xref>].</p><p>For each field experiment, seeds were first germinated in a greenhouse in plug trays (128sq, T.O. Plastics) containing a media mix composed of sphagnum, vermiculite, fine bark, bark ash, gypsum, slow-release nitrogen, dolomitic limestone, and a wetting agent (Metro-Mix 360, Sun Gro Horticulture). Plugs were provided with ample water daily by clear water or by fertigation every other day (EXCEL CAL MAG 15-5-5). Seedlings were hand transplanted into genotype specific subplots (dimensions 7 x 6) within a mechanically tilled field approximately 1 week after sewing. Control over water availability was achieved by growing plants under retractable awning rainout shelters at 5 cm spacing to block precipitation from experimental plots and re-applying differential drip irrigation regimes to wet and dry replicates. Each awning is referred to as a plot and the genotype specific blocks within each awning are called subplots. Planting density experiments were performed by growing each genotype specific subplot at 5 cm (dense) and 25 cm (sparse) spacing.</p><p>Plants (F7 RIL population) were grown in deep pots (Stuewe &amp; Sons, Oregon, D25L) with one plant per pot (one plant per RIL was phenotyped). Seeds were directly germinated in 14 inch Deepots filled with a soil mixture containing 75% Pro-Mix (PRO-MIX(r) PGX soil, Premier Tech, Canada) and 25% river sand, imbibed in water to pot capacity. Pot capacity, defined as the amount of water that soil in a pot can hold against the pull of gravity, was estimated to be 230 ml. Plants were grown in a growth chamber (12 hours light at 31&#176;C and 12 hours dark at 23&#176;C with constant relative humidity at 54&#8211;55%). Well-watered plants were watered back to field capacity once every third day whereas for water deficit experiments, seeds were sown in soil imbibed to pot capacity and no further water was added. To prevent water loss, the bottom of each pot was covered with a plastic bag.</p><p>After a six week stratification in moist long fiber sphagnum moss (Luster Leaf Products Inc., USA) at 4 C, <italic>Setaria</italic> seeds were planted in four inch diameter (10 cm) white pots pre-filled with ~470 cm<sup>3</sup> of Metro-Mix 360 soil (Hummert, USA) and 0.5 g of Osmocote Classic 14-14-14 fertilizer (Everris, USA). After planting, seeds were given 7 days to germinate in a Conviron growth chamber with long day photoperiod (16 h day/8 h night; light intensity 230 &#956;mol/m<sup>2</sup>/s) at 31&#176;C day/21&#176;C night before being loaded onto the Bellweather Phenotyping System using a random block design. Plants were grown on the system for 25 days under long day photoperiod (16 h day/8 h night; light intensity 500 &#956;mol/m<sup>2</sup>/s) cycling the temperature to match the photoperiod (31&#176;C day/21&#176;C night) and maintaining the relative humidity between 40&#8211;80%. Weighing and watering of plants was performed between 2&#8211;3 times per day to maintain soil volumetric water content at either 40% full-capacity (FC) (drought) or 100% FC (well-watered) as determined by [<xref ref-type="bibr" rid="pgen.1006841.ref052">52</xref>]. Water limitation began 15 days after planting. Individual plants were imaged at 4 different angular rotations (0&#176;, 90&#176; 180&#176;, 270&#176;) every other day.</p><p>The effect of pot size was evaluated by growing parental lines of <italic>Setaria viridis</italic> (A10) and <italic>Setaria italica</italic> (B100) in four different size pots: 160 cm<sup>3</sup> (T.O. Plastics, 715348C), 473 cm<sup>3</sup> (P&#246;ppelmann, VCC 10 F US), 950 cm<sup>3</sup> (Stuewe &amp; Sons, MT38) and 6033 cm<sup>3</sup> (Nursery Supplies, C600) filled with Metro-Mix 360 soil (Sun Gro Horticulture). Seeds were given 7 days to germinate in a greenhouse and watered daily by the Donald Danforth Plant Science Center greenhouse staff. Plants were grown for 50 days under long day photoperiod (14 h day/ 10 h night) cycling the temperature to match photoperiod (27&#176;C day/ 23&#176;C night) and maintaining the relative humidity between 40&#8211;100%.</p><p>In the field, plant height was measured as the distance from the plant base to the uppermost leaf collar on the culm. In 2013 this was measured by judging the average height of the canopy in a subplot relative to a yard stick and recorded the measurement using a Janam XP20 hand held scanner [Janam Technologies; Woodbury, NY]. In 2014 height was directly measured repeatedly on three tagged plants in each subplot. In each field experiment, a direct culm height measurement was taken at the final destructive biomass harvest. In 2013 this was done on three representative plants from each subplot; in 2014 this was done on the same-tagged plants as the in-field repeated height measurements.</p><p>In the 2013 Density experiment, height was measured at 19, 25, 31, 38, 45, 52, 59, and 67 days after seed sowing. Final biomass harvest began 87 days after seed sowing. In the 2013 Drought experiment, height was measured at 21, 29, 36, 43, 50, 59, and 67 days after seed sowing. Final biomass harvest began 92 days after seed sowing. In the 2014 Drought experiment, height was measured at 25, 33, 40, and 47 days after seed sowing. Final biomass harvest began 59 days after seed sowing. In the 2014 Density experiment, height was measured at 25 and 46 days after seed sowing. Final biomass harvest began 67 days after seed sowing.</p><p>Panicle emergence was measured as the number of days after sowing at which the panicle head was seen past the collar of the culm flag leaf in at least half of the individuals in a genotype specific subplot.</p><p>At Carnegie, plant height was measured from the base of the plant to the tip of the panicle (at 48 days after seed sowing).</p><p>Five different measurement functions encoded within the PlantCV software package were used to estimate plant height in the images collected at the Bellweather Phenotyping Facility [<xref ref-type="bibr" rid="pgen.1006841.ref052">52</xref>] (<xref ref-type="supplementary-material" rid="pgen.1006841.s001">S1 Fig</xref>). These include length of the plant object along the y-axis (extent_y), length of the plant object from the top of the pot to the maximal y-axis point identified as the plant object (height_above_bound), the distance from the top of the pot to the y-coordinate reported as the center of mass of the plant object (centroid_y), the distance from the top of the pot to the y-coordinate identified as the center of the ellipse that encompasses the plant object (ellipse_y) and an additional function designed to estimate the distance between the top of the pot and y-coordinate where average plant width is maximized (canopy_height). Canopy height is calculated by dividing the height_above_bound measure into 20 different equally sized scoring windows (divide the object into 5% intervals along the y-axis, scoring is based upon median width within each interval) and measuring the median plant width within each window. The function reports the average of y-coordinates that comprises the scoring window and median width within the scoring window where the object (plant) is the widest (canopy_width).</p><p>Centroid_y and ellipse_center_y were the most and least heritable measurements of plant height respectively (<xref ref-type="supplementary-material" rid="pgen.1006841.s002">S2a Fig</xref> and <xref ref-type="supplementary-material" rid="pgen.1006841.s016">S2 Table</xref>). Height_above_bound and extent_y exhibited the largest proportion of variance attributed to water limitation (<xref ref-type="supplementary-material" rid="pgen.1006841.s002">S2b Fig</xref> and <xref ref-type="supplementary-material" rid="pgen.1006841.s017">S3 Table</xref>). Summarization of height measurements derived from four rotational side view images was performed by calculating the mean, median, maximum or minimum of each distribution. Summarization method does not appear to influence the heritability of this these traits (<xref ref-type="supplementary-material" rid="pgen.1006841.s003">S3 Fig</xref>). All height metrics are reported as the average value estimated from all individual rotational images at each plant time point. Lengths of objects measured in pixels were converted to centimeters (cm) using the calibration reported by [<xref ref-type="bibr" rid="pgen.1006841.ref052">52</xref>].</p><p>Plant height of parental lines grown in the greenhouse at the Donald Danforth Plant Science Center were measured as the distance from the base of the plant to most distant leaf collar on the culm using a tape measure. Measurements were recorded every other day during the workweek for a span of 50 days.</p><p>A de novo genetic map of the A10 x B100 <italic>Setaria</italic> RIL population was constructed through genotyping by sequencing (GBS) using the methods described in [<xref ref-type="bibr" rid="pgen.1006841.ref054">54</xref>,<xref ref-type="bibr" rid="pgen.1006841.ref055">55</xref>]. Briefly, leaf tissue was collected from plants in the 2014 planting density experiment in addition to greenhouse trials. High molecular weight DNA was extracted using the CTAB method [<xref ref-type="bibr" rid="pgen.1006841.ref056">56</xref>] and quantified using a Nanodrop 1000 Spectrophotometer [Thermo Scientific; Waltham, MA]. DNA was then double-digested using the restriction enzymes <italic>Pst1</italic> and <italic>MspI</italic> and a unique barcode and common adaptors were ligated onto their respective fragment ends [<xref ref-type="bibr" rid="pgen.1006841.ref057">57</xref>]. After size selection and cleanup using Agencourt AMPure XP beads [Beckman Coulter; Brea, Ca], barcoded samples were pooled and submitted for sequencing (100-bp single end reads) using two Illumina Hi-Seq lanes [Illumina; San Diego, CA].</p><p>The resulting sequence reads were processed using the Tassel 3.0 GBS pipeline [<xref ref-type="bibr" rid="pgen.1006841.ref058">58</xref>]. Raw sequence reads were filtered to remove any tags that were not present 20 times at minimum within in each lane of sequencing and anchored to the <italic>S</italic>. <italic>viridis</italic> genome version 1.1 (Phytozome) using bowtie2 [<xref ref-type="bibr" rid="pgen.1006841.ref059">59</xref>] with default parameters. Anchored SNP polymorphisms exhibiting an inbreeding coefficient less than 0.9 were removed and identical taxa within and between library preps (2 sequencing lanes) were called as heterozygous if the ratio between alleles in and between all identical taxa and/or library preps was less than 0.8. All polymorphisms categorized as heterozygous were called as missing values for the remainder of the analysis, yielding 17,612 SNPs.</p><p>The genetic map was generated using the R/qtl software [<xref ref-type="bibr" rid="pgen.1006841.ref060">60</xref>]. Genotype calls within the combined A10 taxa were compared to the <italic>S</italic>. <italic>viridis</italic> reference genome version 1.1 (Phytozome) and all missing and non-congruent polymorphisms were removed. The remaining sites were filtered to remove SNPs not present in &gt; 90% of the unified taxa yielding 217 RILs and 3,295 markers. All polymorphisms were then recoded to reflect their parent of origin. Dubious genotype calls were identified by examining the recombination frequency between adjacent markers. Based upon manual inspection of histograms generated of recombination fraction between adjacent markers, all markers exhibiting a recombination frequency between adjacent markers greater than 36% (80 recombinations / 217 individuals) were considered to be erroneous and removed from the map. Next imputation of missing genotypes was performed at all sites that did not exhibit recombination between missing makers (using the R/qtl fill.geno() function and specifying the &#8216;no_dbl_XO&#8217; method) and duplicate markers were removed from the genetic map. Finally, the genetic distance between markers was calculated using the Kosambi mapping function assuming a genotype error rate of 0.005. Prior to QTL mapping, individuals missing more than 10% of their genotype calls are removed yielding a genetic map containing 208 individuals genotyped at 1,595 sites with 99.6% coverage. Based on pairwise similarity between identical individuals, the GBS map constructed here is highly consistent with other genetic maps of this population described in the literature [<xref ref-type="bibr" rid="pgen.1006841.ref049">49</xref>,<xref ref-type="bibr" rid="pgen.1006841.ref061">61</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Variance components corresponding to broad sense heritability and total variance explained were estimated using a mixed linear model using the R package lme4 [<xref ref-type="bibr" rid="pgen.1006841.ref062">62</xref>]. Broad sense heritability was calculated using two methods. Within an individual experiment, broad sense heritability on a line-estimate basis was calculated using the following formula:
<disp-formula id="pgen.1006841.e001"><alternatives><graphic id="pgen.1006841.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006841.e001.jpg" /><ns1:math id="M1"><ns1:mrow><ns1:msup><ns1:mtext>H</ns1:mtext><ns1:mn>2</ns1:mn></ns1:msup><ns1:msub><ns1:mrow /><ns1:mrow><ns1:mtext>experiment</ns1:mtext></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:msup><ns1:mo>&#963;</ns1:mo><ns1:mn>2</ns1:mn></ns1:msup><ns1:msub><ns1:mrow /><ns1:mrow><ns1:mtext>genotype</ns1:mtext></ns1:mrow></ns1:msub><ns1:mo>/</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msup><ns1:mo>&#963;</ns1:mo><ns1:mn>2</ns1:mn></ns1:msup><ns1:msub><ns1:mrow /><ns1:mrow><ns1:mtext>genotype</ns1:mtext></ns1:mrow></ns1:msub><ns1:mtext>&#8201;</ns1:mtext><ns1:mo>+</ns1:mo><ns1:mtext>&#8201;</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msup><ns1:mo>&#963;</ns1:mo><ns1:mn>2</ns1:mn></ns1:msup><ns1:msub><ns1:mrow /><ns1:mrow><ns1:mtext>genotype&#160;X&#160;treatment</ns1:mtext></ns1:mrow></ns1:msub><ns1:mo>/</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:msub><ns1:mtext>n</ns1:mtext><ns1:mrow><ns1:mtext>treatment</ns1:mtext></ns1:mrow></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mtext>&#160;</ns1:mtext><ns1:mo>+</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msup><ns1:mo>&#963;</ns1:mo><ns1:mn>2</ns1:mn></ns1:msup><ns1:msub><ns1:mrow /><ns1:mrow><ns1:mtext>residual</ns1:mtext></ns1:mrow></ns1:msub><ns1:mo>/</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:msub><ns1:mtext>n</ns1:mtext><ns1:mrow><ns1:mtext>replicates</ns1:mtext></ns1:mrow></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:math></alternatives></disp-formula>
in which n<sub>treatment</sub> is the harmonic mean of the number of treatment blocks in which each line was observed and n<sub>replicates</sub> is the harmonic mean of number of replicates of each genotype in the experiment. Heritability within treatment blocks was calculated by fitting a linear model with genotype as the only explanatory factor within each treatment block.</p><p>The proportion of variance attributed to genotype divided by total variance within each treatment block is reported as broad sense heritability within treatment.</p><p>Total variance explained was calculated by fitting a linear model including factors genotype, treatment, plot and genotype x treatment effects across all phenotypic values in all treatments. The proportion of variance that is incorporated into these factors divided by the total variance in the experiment is reported as total variance explained.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Best linear unbiased predictors (BLUPs) of plant height for each genotype were predicted from a linear mixed effect model independently within all individual time points of field drought and density experiments in 2013 and 2014 using lme4 [<xref ref-type="bibr" rid="pgen.1006841.ref062">62</xref>]. The following terms were included in the model:
<disp-formula id="pgen.1006841.e003"><alternatives><graphic id="pgen.1006841.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006841.e003.jpg" /><ns1:math id="M3"><ns1:mrow><ns1:msub><ns1:mtext>Y</ns1:mtext><ns1:mrow><ns1:mtext>ijk</ns1:mtext></ns1:mrow></ns1:msub><ns1:msub><ns1:mrow><ns1:mo>=</ns1:mo><ns1:mi>&#956;</ns1:mi><ns1:mo>+</ns1:mo><ns1:mtext>&#160;genotype</ns1:mtext></ns1:mrow><ns1:mtext>i</ns1:mtext></ns1:msub><ns1:msub><ns1:mrow><ns1:mo>+</ns1:mo><ns1:mtext>&#160;treatment</ns1:mtext></ns1:mrow><ns1:mtext>j</ns1:mtext></ns1:msub><ns1:mo>+</ns1:mo><ns1:mtext>&#160;plot</ns1:mtext><ns1:msub><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mtext>treatment</ns1:mtext></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mrow><ns1:mtext>k</ns1:mtext><ns1:mo>(</ns1:mo><ns1:mtext>j</ns1:mtext><ns1:mo>)</ns1:mo></ns1:mrow></ns1:msub><ns1:mo>+</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:msub><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mtext>genotype&#160;x&#160;treatment</ns1:mtext></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mrow><ns1:mtext>ij</ns1:mtext></ns1:mrow></ns1:msub><ns1:mtext>+</ns1:mtext><ns1:msub><ns1:mi>&#949;</ns1:mi><ns1:mrow><ns1:mtext>ijk</ns1:mtext></ns1:mrow></ns1:msub></ns1:mrow></ns1:math></alternatives></disp-formula>
In which Y<sub>ijk</sub> is the individual observation of plant height, <italic>&#956;</italic> is the overall mean, genotype<sub>i</sub> is the effect of the i<sub>th</sub> genotype, treatment<sub>j</sub> is the effect of the j<sub>th</sub> treatment, plot(treatment)<sub>k(j)</sub> is the effect of the k<sub>th</sub> plot nested within the j<sub>th</sub> treatment, and (genotype x treatment)<sub>ij</sub> is a term describing the interaction between the i<sub>th</sub> genotype and j<sub>th</sub> treatment and &#949;<sub>ijk</sub> is the random error term. As fixed effects we entered treatment and plot nested within treatment into the model. The terms genotype and genotype by treatment interactions were modeled as random effects.</p><p>Logistic regression was used to model the line values of plant height in experiments performed at the Bellweather Phenotyping Facility. Three types of logistic regression models considered (three-parameter logistic, four-parameter logistic and Gompertz) and estimates from the model with the lowest Akaike information criterion (AIC) score were used. Modeling was performed using R functions described in [<xref ref-type="bibr" rid="pgen.1006841.ref063">63</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">QTL mapping was performed at each time point within treatment blocks and on the numerical difference, relative difference and trait ratio calculated between treatment blocks using functions encoded within the R/qtl and funqtl package [<xref ref-type="bibr" rid="pgen.1006841.ref060">60</xref>,<xref ref-type="bibr" rid="pgen.1006841.ref064">64</xref>]. The functions were called by a set of custom python and R scripts (<ext-link ext-link-type="uri" ns0:href="https://github.com/maxjfeldman/foxy_qtl_pipeline">https://github.com/maxjfeldman/foxy_qtl_pipeline</ext-link>). Two complimentary analysis methods were utilized. First, a single QTL model genome scan using Haley-Knott regression was performed to identify QTL exhibiting LOD score peaks greater than a permutation based significance threshold (&#945; = 0.05, n = 1000). Next, a stepwise forward/backward selection procedure was used to identify an additive, multiple QTL model based upon maximization of penalized LOD score. Both procedures were performed at each time point, within treatment blocks and on the numerical difference relative difference and trait ratio calculated between phenotypic values measured in treatment blocks at each time point. QTL associated with difference or ratio composite traits may identify loci associated with genotype by environment interaction [<xref ref-type="bibr" rid="pgen.1006841.ref065">65</xref>]. QTLs across different experiments were considered to be the same if they were in a 20cM window around the most commonly identified marker in that window. Tests for epistatic interaction between QTL were performed using the scantwo function in the R/qtl package [<xref ref-type="bibr" rid="pgen.1006841.ref060">60</xref>]. Epistasis between QTL was evaluated by comparing the log<sub>10</sub> likelihood ratio of a model describing the additive and interaction between two QTL with a model only containing the additive interaction between two QTL. The significance threshold was determined through permutation (&#945; = 0.05, n = 100).</p><p>The function-valued approach described by Kwak et al., 2016 [<xref ref-type="bibr" rid="pgen.1006841.ref064">64</xref>] was used to identify QTL associated with the average (SLOD) and maximum (MLOD) score at each locus throughout the experiment [<xref ref-type="bibr" rid="pgen.1006841.ref064">64</xref>]. The genotypic mean height within treatments was estimated using a logistic function and the QTL significance threshold was determined based upon permutation-based likelihood of observing the empirical SLOD or MLOD test statistic. Across all grow outs, separate; independent linkage mapping analysis performed at each time point identified a larger number of QTL locations relative to similar function valued analysis based on the SLOD and MLOD statistics calculated at each individual marker throughout the experimental time course.</p><p>After refinement of QTL position estimates, the significance of fit for the full multiple QTL model was assessed using type III analysis of variance (ANOVA). The contribution of individual loci was assessed using drop-one-term, type III ANOVA. The proportion of variance explained by in addition to the allelic effect size of each locus were determined by comparing the fit of the full model to a sub-model with one of the terms removed. All putative protein coding genes (<italic>Setaria viridis</italic> genome version 1.1) found within a 1.5-logarithm of the odds (LOD) confidence interval were reported for each QTL.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">A search for orthologous genes associated with plant height (putative causal genes) was performed within the 37 unique QTL intervals identified throughout the genome. A curated list of plant height genes cloned from maize, sorghum and rice was generated through literature review and searching Maize GDB with search terms including &#8220;height&#8221; and &#8220;dwarf&#8221;. The coding sequence name and genomic coordinates of each of these genes was downloaded from their species-specific repository (Maize GDB, Rice Genome Annotation Project, Phytozome). Regions of genomic synteny between <italic>Setaria viridis</italic> and maize, sorghum or rice were identified using the synmap function encoded by the CoGe web tool (default parameters) [<xref ref-type="bibr" rid="pgen.1006841.ref066">66</xref>]. The genomic coordinates of coding sequences of interest were extracted from the CoGe text output using a custom python script (<ext-link ext-link-type="uri" ns0:href="https://github.com/maxjfeldman/Feldman_Setaria_Height_2016">https://github.com/maxjfeldman/Feldman_Setaria_Height_2016</ext-link>). Reciprocal BLAST searches were used to identify the location of orthologous genes not found in syntenic blocks. The location of these sequences was plotted across the genome to examine their location relative to the coordinates of the QTL and median confidence interval at each location.</p>