<p>All stocks were obtained from the Bloomington Drosophila stock center unless noted otherwise. All stocks are described in flybase (flybase.net) unless noted otherwise.</p><p>Epitope tags and point mutations were introduced into the endogenous loci of <italic>moi</italic>, <italic>tea</italic> and <italic>ver</italic> using the SIRT gene targeting method that we developed for Drosophila [<xref ref-type="bibr" rid="pgen.1006435.ref019">19</xref>]. Detailed experimental designs are presented in Supplemental Materials. Briefly, an <italic>attP</italic> landing site for the phiC31 integrase was first introduced to the vicinity of the locus of interest by traditional gene targeting by homologous recombination [<xref ref-type="bibr" rid="pgen.1006435.ref035">35</xref>; <xref ref-type="bibr" rid="pgen.1006435.ref036">36</xref>]. A plasmid containing the desired modification, a tag or a mutation, was integrated into the locus of interest by phiC31-mediated site specific integration followed by I-CreI endonuclease-induced reduction of the target locus duplication to generate a final locus containing the desired modification.</p><p>LacZ-based yeast two hybrid assays were performed as described using an identical set of vectors and yeast strains [<xref ref-type="bibr" rid="pgen.1006435.ref032">32</xref>]. We modified the traditional yeast two-hybrid assay into a three-hybrid assay in order to detect tripartite interaction involving all three subunits of MTV. Detailed protocols are given in Supplemental Materials.</p><p>MTV expression in insect cells was based on the Bac-to-Bac Baculovirus Expression system from Invitrogen. The correct DNA clones containing 3xFLAG-Ver, 3xHA-Tea and V5-Moi were confirmed by sequencing. Baculovirus were generated by the service of the Protein Expression Laboratory (PEL) of NCI, Frederick with the titer above 1x10<sup>8</sup> pFU/ml. Detailed protocols for protein expression and purification are provided in Supplemental Materials. Between 50ng to 200ng of purified MTV and 100ng to 400ng of purified MV were used for EMSA.</p><p>Protein-oligo interaction was analyzed using LightShift Chemiluminescent EMSA Kit (Pierce) according to the manufacturer protocols using 5&#8217; Biotin-labeled oligos synthesized by Integrated DNA Technologies. The ExoI protection assay was performed in the same binding buffer with 2U of the enzyme from NEB and incubated for 30 minutes at RT.</p><p>Telomeric localization of Tea, HipHop and HOAP by antibody staining was performed as described in ref [<xref ref-type="bibr" rid="pgen.1006435.ref006">6</xref>]. A rabbit anti-GFP antibody from Torrey Pine Biolabs was used to localize Tea at polytene telomeres using published protocols. For live GFP observation, tissues were dissected in PBS with 0.1% Triton X-100 (PBST), transferred to a DAPI solution in PBST. The tissues were stained for five minutes followed by a wash in PBST, and mounted in PBS.</p>