<p>The ethics statement is available in <xref ref-type="supplementary-material" rid="pgen.1006384.s001">S1 Text</xref>.</p><p>The HBEC3kt cells (a gift from Dr. John Minna, UT Southwestern Medical Center), H441, A549 (ATCC) and HCC78 cells (DMSZ) were cultured as previously described [<xref ref-type="bibr" rid="pgen.1006384.ref022">22</xref>] or according to vendors&#8217; instructions. The primary antibodies for immunoblots are: rabbit anti U2AF1 (1:5000, # NBP1-19121, Novus), rabbit anti GFP (1:5000, #A-11122, Invitrogen), mouse anti ACTB (1:5000, Clone 8H10D10, Cell Signaling). Lentiviruses were produced and titered in HEK293T cells as previously described [<xref ref-type="bibr" rid="pgen.1006384.ref037">37</xref>]. An MOI (multiplicity of infection) of 1&#8211;5 were used for all assays.</p><p>Total RNA was extracted and reverse transcribed as previously described [<xref ref-type="bibr" rid="pgen.1006384.ref038">38</xref>]. Splicing alterations were measured by quantitative PCR on reverse-transcribed cDNA (RT-qPCR), using isoform-specific primers (<xref ref-type="supplementary-material" rid="pgen.1006384.s026">S7 Table</xref>). These primers were designed following a previously described method [<xref ref-type="bibr" rid="pgen.1006384.ref039">39</xref>]. The PCR efficiency and specificity of each primer set were determined before they were used for measuring splicing changes (See Supplemental Materials and Methods for details).</p><p>Clonogenic assay was performed by infecting cells with lentiviruses two days before seeding them into 100 mm dishes (1000 live cells per dish) to grow colonies. Growth media were supplemented with puromycin (1 &#956;lg/ml) for selecting infected cells and were changed once a week for up to three weeks. Cell colonies were stained with 0.03% methylene blue (in 20% methanol) for 5 min. Clonogenicity is defined as colony numbers formed as a percent of those in control cells.</p><p>Details of all DNA constructs used in the study and the genome editing approaches for creating the <italic>U2AF1</italic>S34F allele and allelic-specific disruption of <italic>U2AF1</italic> are described in the Supplemental Materials and Methods.</p><p>High throughput mRNA sequencing (RNA-seq) was conducted in the Sequencing Facility of the National Cancer Institute. RNA quality was assessed by 2100 Bioanalyzer (Agilent). Total RNA with good integrity values (RIN &gt; 9.0) was used for poly A selection and library preparation using the Illumina TruSeq RNA library prep kit. Two or four samples were pooled per lane and ran on the HiSeq2000 or HiSeq2500 instrument using TruSeq V3 chemistry. All samples were sequenced to the depth of 100 million pair-end 101 bp reads per sample.</p><p>Splicing analysis of RNA-seq data from the TCGA LUAD cohort as well as engineered HBEC3kt, H441, and HCC78 cell lines was performed as previously described [<xref ref-type="bibr" rid="pgen.1006384.ref017">17</xref>]. A brief description of the method was provided in the Supplemental Materials and Methods.</p><p>Purification of U2AF1 complexes, as illustrated in <xref ref-type="fig" rid="pgen.1006384.g005">Fig 5</xref>, was explained in Supplemental Materials and Methods. Sequences of synthetic 5&#8242;-labeled fluorescein RNAs (GE Healthcare Dharmacon) and binding curves are given in the Supplementary <xref ref-type="supplementary-material" rid="pgen.1006384.s013">S12 Fig</xref>. Apparent equilibrium affinity constant of the purified U2AF1 complexes with RNA was measured based on changes in fluorescence anisotropy as previously described [<xref ref-type="bibr" rid="pgen.1006384.ref018">18</xref>]. A brief description of this method was provided in the Supplemental Materials and Methods.</p><p>All experiments were independently performed at least three times unless otherwise stated. Statistical significance was determined by two-tailed Student's <italic>t</italic> test or otherwise stated. In all analyses, <italic>p</italic> values &#8804; 0.05 are considered statistically significant.</p>