<p>The participating studies were approved by the medical ethics committees of all participating centres, and all participants gave their written informed consent before entering the study</p><p>We conducted genome-wide association studies of mJSW for each cohort of the discovery stage: Rotterdam Study I (RS-I), Rotterdam Study II (RS-II), TwinsUK, SOF and MrOS using standardized age-, gender and population stratification (four principal components) adjusted residuals from linear regression. Cohort description and details of the single GWAS studies are given in <xref ref-type="supplementary-material" rid="pgen.1006260.s020">S1 Text</xref> and <xref ref-type="supplementary-material" rid="pgen.1006260.s002">S1 Table</xref>. The 6 cohorts used in the discovery stage were combined in a joined meta-analysis using inverse variance weighting with METAL [<xref ref-type="bibr" rid="pgen.1006260.ref039">39</xref>]. Genomic control correction was applied to the standard errors and P-values before meta-analysis. SNPs with a P value &lt; = 5&#215;10&#8722;<sup>6</sup> were selected for replication. The top SNPs for each independent locus were taken for replication in seven studies: the Genetics of Osteoarthritis and Lifestyle (GOAL) study, the Chingford study, CHECK (Cohort Hip &amp; Cohort Knee), Genetics osteoARthritis and progression (GARP) study, the Genetics of Generalized Osteoarthritis (GOGO), the Johnston County Osteoarthritis Project (JoCo) and additionally the Nottingham OA case-control study for association with Hip OA (see Supplemental material for detailed information of the cohorts). Association of the SNPs with mJSW was additionally adjusted for height to test its independence. Secondary analyses included: association of the top SNPs with hip OA using logistic regression analysis (age and gender adjusted and by study centres an/or relatedness when it was pertinent). We used conditional analyses to investigate whether there are any independent signals in the identified associated loci, which were implemented using GCTA-COJO analysis [<xref ref-type="bibr" rid="pgen.1006260.ref040">40</xref>].</p><p>The mJSW was assessed at pelvic radiographs in anteriorposterior position. The mJSW was measured in mm, along a radius from the center of the femoral head, and defined as the shortest distance found from the femoral head to the acetabulum. Within the Rotterdam Study, we used a 0.5 mm graduated magnifying glass laid directly over the radiograph to measure the minimal joint space width of the hip joints [<xref ref-type="bibr" rid="pgen.1006260.ref041">41</xref>]. Within SOF and MrOS, a handheld caliper and reticule was used to measured mJSW to the nearest 0.1mm between the acetabular rim and proximal head of the femur [<xref ref-type="bibr" rid="pgen.1006260.ref042">42</xref>]. For CHECK, mJSW was measured semi-automatic with the Software tool HOLY [<xref ref-type="bibr" rid="pgen.1006260.ref043">43</xref>].</p><p>Radiographic hip OA was defined in the RS-I, RS-II, RSIII, Twins-UK, Chingford, and JoCo studies using Kellgren and Lawrence (K/L) scores. Hip OA cases were defined as a K/L score &#8805; 2 on either side of the hip or THR due to OA. Hip OA controls were defined as no THR for OA and K/L score &#8804; 1 and JSN &#8804; 1. In MrOS and SOF cohorts, radiographic hip OA case-control was defined by a modified Croft grade, as previously described [<xref ref-type="bibr" rid="pgen.1006260.ref044">44</xref>], where cases were defined as a Croft score &#8805; 2 on either side of the hip or THR due to OA and controls were defined as a Croft score &#8804; 1 on both sides of the hip and no THR. Hip OA cases in the GOAL and Nottingham OA studies were defined by having THR, and controls were radiographically free of hip OA, as previously described [<xref ref-type="bibr" rid="pgen.1006260.ref045">45</xref>]. In GARP, hip osteoarthritis was defined as pain or stiffness in the groin and hip region on most days of the preceding month in addition to femoral or acetabular osteophytes or axial joint space narrowing on radiography or prosthesis due to osteoarthritis. In GOGO, hip OA was defined as KL grade &gt; = 2, or minimal joint space width &lt; = 2.5 mm, or the combination of joint space narrowing grade &gt; = 2 and any osteophyte of grade &gt; = 1, or history of joint replacement for OA. In JoCo, hip OA cases were defined as KL grade &gt; = 2 or THR in at least one hip. Hip OA controls were defined as KL grade &lt; = 1 in both hips.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We have used several available tools and publicly available databases to prioritize genes in known and newly discovered osteoarthritis associated regions. Locus gene sets were constructed by taking a region of 500 Kb upstream and 500Kb downstream of the lead SNP of that locus. We analysed 152 genes in 13 independent loci associated with minimal joint space width in the hip joint (mJSW) for 7 loci, hip OA for 4 loci, total joint replacement (TJR) for 1 locus and total hip replacement (THR) for 1 locus [<xref ref-type="bibr" rid="pgen.1006260.ref002">2</xref>]. We analysed the following biological evidence for each gene at all loci; Nearest located genes: Taken from the UCSC genome browser, GRCh37/hg19 [<xref ref-type="bibr" rid="pgen.1006260.ref046">46</xref>]. DEPICT gene prioritization: Data-driven Expression-Prioritized Integration for Complex Traits, a novel tool designed to identify the most likely causal gene in a given locus and to gene sets that are enriched in the genetic associations [<xref ref-type="bibr" rid="pgen.1006260.ref021">21</xref>]. DEPICT was used to prioritize genes in a 1 MB region around the found SNPs that were significant associated with the osteoarthritis phenotype, taking a region of 500 Kb upstream and 500Kb downstream of the lead SNP of that locus. Gene prioritization analysis was performed to directly investigate functional similarities among genes from different associated regions, significance was defined by false discovery rate (FDR &#8804; 5%). GRAIL gene prioritization: Gene Relationships Across Implicated Loci (GRAIL), was used to determine connectively between genes across OA implicated loci based on literature associations [<xref ref-type="bibr" rid="pgen.1006260.ref022">22</xref>]. A GRAIL analysis was performed on 10 independent OA associated loci, based on existing literature in PubMed till August 2014. Mouse gene expression and phenotype: For each investigated gene, expression in mouse bone and/or cartilage tissue during several developmental stages as well as for adult tissue was determined using data from the Jackson lab database (<ext-link ext-link-type="uri" ns0:href="http://www.informatics.jax.org/">http://www.informatics.jax.org/</ext-link>). In addition mouse phenotype data was also obtained for each gene. OMIM phenotype: Using the Online Mendelian Inheritance in Man (OMIM) database we examined which genes were involved in abnormal skeletal growth syndromes when mutated (<ext-link ext-link-type="uri" ns0:href="http://omom.org/">http://omom.org</ext-link>). Expression quantitative trait loci: eQTL information was taken from the Blood eQTL browser (<ext-link ext-link-type="uri" ns0:href="http://genenetwork.nl/bloodeqtlbrowser/">http://genenetwork.nl/bloodeqtlbrowser/</ext-link>) and the eQTL browser (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/projects/gap/eqtl/index.cgi">http://www.ncbi.nlm.nih.gov/projects/gap/eqtl/index.cgi</ext-link>) using the lead SNP in each locus [<xref ref-type="bibr" rid="pgen.1006260.ref020">20</xref>]. Non-synonymous variants: Last we determined if there were any nonsynonymous variants in LD (r<sup>2</sup>&gt;0.06) with the lead SNP of a locus, using HaploReg V2 and the SNP Annotation and Proxy Search (SNAP) tools [<xref ref-type="bibr" rid="pgen.1006260.ref047">47</xref>,<xref ref-type="bibr" rid="pgen.1006260.ref048">48</xref>]. For each gene we assigned a score based on equally weighted lines of evidence.</p><p>We have used cartilage samples from the RAAK study to study gene expression in preserved and affected cartilage from individuals undergoing joint replacement [<xref ref-type="bibr" rid="pgen.1006260.ref019">19</xref>]. The ongoing Research Arthritis and Articular Cartilage (RAAK) study is aimed at the biobanking of blood, joint materials (cartilage, bone and where available ligaments of knees and hips) and bone marrow stem cells (hip joints only) of patients and controls in the Leiden University Medical Center and collaborating outpatient clinics in the Leiden area. At the moment of collection (within 2 hours following surgery) tissue was washed extensively with phosphate buffered saline (PBS) to decrease the risk of contamination by blood, and cartilage was collected of the weight-bearing area of the joint. Cartilage was classified macroscopically and collected separately for macroscopically OA affected and preserved regions. Classification was done according to predefined features for OA related damage based on color/whiteness of the cartilage, based on surface integrity as determined by visible fibrillation/crack formation, and based on depth and hardness of the cartilage upon sampling with a scalpel. During collection with a scalpel, care was taken to avoid contamination with bone or synovium. Collected cartilage was snap frozen in liquid nitrogen and stored at -80&#176;C prior to RNA extraction. Tissues have been stored tailored to apply staining and immunohistochemistry (IHC). Furthermore, DNA and RNA have been isolated from the preserved and affected areas of the respective tissues in order to apply genetic, transcriptomic and epigenomic profiling with respect to the OA pathophysiological process.</p><p>After <italic>in vitro</italic> transcription, amplification, and labeling with biotin-labeled nucleotides (Illumina TotalPrep RNA Amplification Kit) Illumina HumanHT-12 v3 microarrays were hybridized. Sample pairs were randomly dispersed over the microarrays, however each pair was measured on a single chip. Microarrays were read using an Illumina Beadarray 500GX scanner and after basic quality checks using Beadstudio software data were analyzed in R statistical programming language. Intensity values were normalized using the &#8220;rsn&#8221; option in the Lumi-package and absence of large scale between-chip effects was confirmed using the Globaltest-package in which the individual chip numbers were tested for association to the raw data. After removal of probes that were not optimally measured (detection <italic>P</italic> &gt;0.05 in more than 50% of the samples) a paired t-test was performed on all sample pairs while adjusting for chip (to adjust for possible batch effects) and using multiple testing correction as implemented in the &#8220;BH&#8221; (Benjamini and Hochberg) option in the Limma-package. Analyses for differential expression between OA and healthy and between preserved and healthy cartilage was performed likewise, adjusting in addition for sex and for age.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Exome sequencing was performed in 2628 individuals from the Rotterdam Study the average mean coverage was 55x, corresponding to approximately 80% of the targeted regions covered by at least 20 reads. The exome sequencing was performed in house (HuGe-F, <ext-link ext-link-type="uri" ns0:href="http://www.glimDNA.org">www.glimDNA.org</ext-link>). Details of the technical procedure and variant calling are given in <xref ref-type="supplementary-material" rid="pgen.1006260.s021">S2 Text</xref>. We tested the exome variants for association with mJSW and/or hip OA in two ways. Each individual variant was tested for association with mJSW using the single variant option within RV-test, while adjusting for age and sex. In addition, we did a burden test for each of the selected genes by using SNP-set kernel association test (SKAT-O). SKAT aggregates individual score test statistics of SNPs in a SNP set and computes SNP-set level p-values for a gene [<xref ref-type="bibr" rid="pgen.1006260.ref023">23</xref>].</p><p>For each of the top mJSW GWAS associated SNPs the LD region was determined using the 1000G Phase 1 population using the Haploreg tool [<xref ref-type="bibr" rid="pgen.1006260.ref047">47</xref>]. The LD threshold was set at r2&#8805;0.8. For each of these SNPs it was determined if the variant was located in a potential enhancer region using the Roadmap consortium reference epigenomes data set [<xref ref-type="bibr" rid="pgen.1006260.ref027">27</xref>]. Heatmaps were constructed by calculating the percentage of variants in LD with the top mJSW GWAS found SNP located in enhancer regions as defined by the Roadmap epigenome chromatin states. The reference epigenomes were downloaded from the official data portal accompanying [<xref ref-type="bibr" rid="pgen.1006260.ref027">27</xref>]. Reference epigenome data was used from mesenchymal stem cell derived chondrocyte cultured cells, Osteoblast, Bone marrow derived cultured mesenchymal stem cells, K562, HUVEC, HeLA and NHEK cells. Reference epigenomes were chromatin state models based on ChIPseq data of 5 core histone marks (H3K4me3, H3K4me1, H3K36me3, H3K27me3, H3K9me3) and an additional H3K27ac histone mark, the Roadmap expanded 18-state model.</p><p>ChIPseq data of mesenchymal stem cell derived chondrocyte cultured cells, and bone marrow derived cultured mesenchymal stem cells were generated by the NHI roadmap epigenomics project [<xref ref-type="bibr" rid="pgen.1006260.ref028">28</xref>]. ChIPseq data of, Osteoblast, K562, HUVEC, HeLA and NHEK cells were generated by the ENCODE consortium [<xref ref-type="bibr" rid="pgen.1006260.ref026">26</xref>]. All data and annotation tracks were downloaded through the UCSC genome browser table tool. Visualization of all ChIPseq annotation and roadmap full epigenomes tracks was done through the UCSC genome browser on GRCh37/hg19. Heatmaps were plotted in R using the CRAN software packages gplots and RcolorBrewer. Enrichment was calculated according to methods described in Trynka et al [<xref ref-type="bibr" rid="pgen.1006260.ref025">25</xref>].</p>