<p>The overall project was reviewed and approved by the Fred Hutchinson Cancer Research Center Institutional Review Board (approval number: 6501 and 3995). Each study was approved by the local IRB [University of Hawaii Human Studies Program (Colo23 and MEC); University of Utah Institutional Review Board (DALS); Partners Human Research Committee (NHS and PHS); Harvard School of Public Health Institutional Review Board (HPFS); Fred Hutchinson Cancer Research Center Institutional Review Board (VITAL, overall study); Ethics Committee of the Medical Faculty of the University of Heidelberg (DKFZ); NCI Special Studies Institutional Review Board (PLCO)]. For each participating study, participants or the next of kin in the case of deceased participants, provided either written informed consent to participate (Colo23, DACHS, DALS, MEC, PHS, PLCO, VITAL, WHI) or they provided implied written consent by the return of the mailed questionnaires (NHS, HPFS). Additional consent to review medical records was obtained through signed written consent.</p><p>We included 14 study centers from the CCFR and GECCO as described in the <xref ref-type="supplementary-material" rid="pgen.1006296.s001">S1 Text</xref> and <xref ref-type="supplementary-material" rid="pgen.1006296.s003">S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1006296.s004">S2</xref> Tables. All colorectal cancer cases were defined as colorectal adenocarcinoma and confirmed by medical records, pathologic reports, or death certificates. We included advanced colorectal adenoma, a well-defined colorectal cancer precursor [<xref ref-type="bibr" rid="pgen.1006296.ref070">70</xref>, <xref ref-type="bibr" rid="pgen.1006296.ref071">71</xref>], from two studies (<xref ref-type="supplementary-material" rid="pgen.1006296.s001">S1 Text</xref>). Advanced adenoma was defined as an adenoma 1 cm or larger in diameter and/or with tubulovillous, villous, or high-grade dysplasia/carcinoma-in-situ histology. Colorectal adenoma cases were confirmed by medical records, histopathology, or pathologic reports. Controls for adenoma cases had a clean sigmoidoscopic or colonoscopic examination. All participants provided informed consent and studies were approved by their respective Institutional Review Boards.</p><p>Average sample and SNP call rates, and concordance rates for blinded duplicates have been previously published [<xref ref-type="bibr" rid="pgen.1006296.ref003">3</xref>]. In brief, genotyped SNPs were excluded based on call rate (&lt; 98%), lack of Hardy-Weinberg Equilibrium in controls (HWE, p &lt; 1 x 10<sup>&#8722;4</sup>), and low minor allele frequency (MAF&lt;0.05). We imputed the autosomal SNPs of all studies to the Northern Europeans from Utah (CEU population) in HapMap II. SNPs were restricted based on per-study minor allele count &gt; 5 and imputation accuracy (R<sup>2</sup> &gt; 0.3). After imputation and quality-control (QC) exclusion, approximately 2.7M SNPs were used in analysis.</p><p>All analyses were restricted to individuals of European ancestry, defined as samples clustering with the Utah residents with Northern and Western European ancestry from the CEPH collection population in principal component analysis [<xref ref-type="bibr" rid="pgen.1006296.ref072">72</xref>], including the HapMap II populations as reference.</p><p>All information on basic demographics and environmental risk factors were collected through interviews or through self-administered questionnaires. Data for all studies were centrally harmonized at the data coordinating center. We used the risk-factor information at the reference time, which varied across studies (<xref ref-type="supplementary-material" rid="pgen.1006296.s001">S1 Text</xref>). A multi-step data-harmonization procedure which is described in detail in Hutter et al. [<xref ref-type="bibr" rid="pgen.1006296.ref029">29</xref>] was applied to reconcile differences in individual study questionnaires. We converted consumption of alcoholic beverages into grams of alcohol per day (g/day) by summing the alcohol content of each beverage consumed per day. To test if the categorical or continuous variable fitted the association between alcohol intake and CRC risk better we used Akaike Information Criterion (AIC) to compare both models. With our sample size a model with an AIC that is 6 points smaller than the other model is considered a better fitting model[<xref ref-type="bibr" rid="pgen.1006296.ref032">32</xref>]. According to this analysis and consistent with previously described risk profiles [<xref ref-type="bibr" rid="pgen.1006296.ref016">16</xref>, <xref ref-type="bibr" rid="pgen.1006296.ref017">17</xref>, <xref ref-type="bibr" rid="pgen.1006296.ref019">19</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006296.ref022">22</xref>, <xref ref-type="bibr" rid="pgen.1006296.ref073">73</xref>], we grouped study participants as non-/occasional drinkers (drinking &lt; 1 g/day); light-to-moderate drinkers (drinking 1&#8211;28 g/day); and heavy drinkers (drinking &gt;28 g/day, one standard drinking is approximately equal to 14 grams of alcohol). We coded these categories using indicator variables for the genome-wide interaction analysis. Smoking history was defined as never- and ever-smoking; pack-years of smoking was calculated by multiplying the average number of packs of cigarettes smoked per day by smoking duration (years). Smoking history (ever vs. never smoking) and pack-years (treated as a continuous variable) of smoking were used in genome-wide interaction analysis, separately.</p><p>Statistical analyses of all data were conducted centrally at the GECCO coordinating center on individual-level data to ensure a consistent analytical approach. Unless otherwise indicated, we adjusted for age at the reference time, sex (when appropriate), center (when appropriate), and the first three principal components from EIGENSTRAT to account for potential population substructure. The alcohol and smoking variables were coded as described above. Each directly genotyped SNP was coded as 0, 1, or 2 copies of the variant allele. For imputed SNPs, we used the expected number of copies of the variant allele (the &#8220;dosage&#8221;), which has been shown to give unbiased test statistics [<xref ref-type="bibr" rid="pgen.1006296.ref074">74</xref>]. Genotypes were treated as continuous variables (i.e. log-additive effects). Each study was analyzed separately using logistic regression models and study-specific results were combined using fixed-effects meta-analysis methods to obtain summary odds ratios (ORs) and 95% confidence intervals (CIs) across studies. We calculated the heterogeneity p-values using Woolf&#8217;s test [<xref ref-type="bibr" rid="pgen.1006296.ref075">75</xref>]. Quantile-quantile (Q-Q) plots were assessed to determine whether the distribution of the p-values was consistent with the null distribution (except for the extreme tail). Subjects with missing data for SNPs or environmental factors were excluded from the relevant analyses. Considering the potential male-female difference in alcohol metabolism[<xref ref-type="bibr" rid="pgen.1006296.ref076">76</xref>, <xref ref-type="bibr" rid="pgen.1006296.ref077">77</xref>] and the different levels of alcohol consumption between sexes, we conducted the genome-wide interaction analysis for alcohol separately for men and women and used fixed effects meta-analysis to combine their results. All analyses were conducted using the R software (Version 3.0.1).</p><p>Two statistical methods that leverage SNPs and environmental factors interaction (G&#215;E interaction) were used to detect potential disease associated loci. First, we used conventional case-control logistic regression analysis including G&#215;E interaction term(s). As the alcohol consumption variable has three categories there are two interaction terms in the statistical models. Based on an increasing number of publications [<xref ref-type="bibr" rid="pgen.1006296.ref078">78</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006296.ref083">83</xref>] providing a detailed discussion on the appropriate genome-wide significance threshold, which all arrive at similar values in the range of 5 x 10<sup>-7</sup>to 5 x 10<sup>&#8722;8</sup> for European populations, we decided to use an alpha level of 5 x 10<sup>&#8722;8</sup> as the genome-wide significance threshold, assuming about 1 million independent tests across the genome (0.05/1,000,000 = 5 x 10<sup>&#8722;8</sup>). For significant results we used permutation approach to determine the empirical p-value. We defined the number of permutation needed as 1/p-value (i.e., for a p-value of 5 x 10<sup>&#8722;8</sup> 1/5E-08 = 20,000,000). We permutated the case-control status 1/p-value times and calculated the p values for the interaction from each meta-analyses to calculate the permuted p-value.</p><p>Second, we used our recently developed Cocktail method.[<xref ref-type="bibr" rid="pgen.1006296.ref055">55</xref>] In brief, this method consists of two-steps: a screening step to prioritize SNPs and a testing step for GxE interaction. For the screening step, we ranked and prioritized variants through a genome-wide screen of each of the 2.7M SNPs (referred to as &#8220;G&#8221;) by the maximum of the two test statistics from marginal association testing of Gs on disease risk [<xref ref-type="bibr" rid="pgen.1006296.ref084">84</xref>], and correlation testing between G and exposure (E) in cases and controls combined.[<xref ref-type="bibr" rid="pgen.1006296.ref085">85</xref>] Based on the ranks of these SNPs from screening, we used a weighted hypothesis framework to partition SNPs into ordered groups and assigned each group an alpha-level cut-off, with higher ranked groups from the screening stage having less stringent alpha-level cut-offs for interaction [<xref ref-type="bibr" rid="pgen.1006296.ref086">86</xref>, <xref ref-type="bibr" rid="pgen.1006296.ref087">87</xref>]. The second step of the Cocktail method is the testing step. We used either case-control (CC) or case-only (CO) logistic regression to calculate a p-value for the interaction. If the G was assigned based on its low marginal association P value in the screening tests, we used CO test; if it was ranked because of a low correlation screening p-value, we used CC tests. We compared the test step p-value to the alpha-level cutoff for each SNP in a given group.</p><p>We calculated absolute risks for each genotype of the SNP showing significant G&#215;E interaction. Briefly, based upon the Surveillance, Epidemiology, and End Results (SEER) age-adjusted colorectal cancer incidence rate (denoted by &#8220;I&#8221;) between 1982&#8211;2011 among the White population of 42.9 per 100,000 men and women per year, we estimated the reference incidence rate of colorectal cancer (denoted by &#8220;I_{reference}&#8221;) using the following formula: I_{reference} = I/(P(AA, non-E) + OR{Aa, non-E}&#215;P(Aa, non-E) + OR{aa, non-E}&#215;P(aa, non-E) + OR{AA, E}&#215;P(AA, E) + OR{Aa, E}&#215;P(Aa, E)) + OR{aa, E}&#215;P(aa, E)), where P(genotype, E (or non-E)) is the prevalence of light-to-moderate drinking (or non/occasional drinking) in each corresponding genotype category among controls (non-cases). Based on this reference incidence rate of colorectal cancer (i.e., I_{reference}), we further calculated absolute colorectal cancer incidence rates within each subgroup defined by genotype of the SNP according to a light-to-moderate drinking or non/occasional drinking by multiplying the I_{reference} with each corresponding OR. Bootstrap methods were used to calculate the 95% CI of absolute risk estimates [<xref ref-type="bibr" rid="pgen.1006296.ref088">88</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used different types of gene expression data to examine putative expression of genes identified in our genome-wide interaction analysis, and to determine biological plausibility that the variants identified might impact CRC risk. First, we searched the Genotype-Tissue Expression project (GTEx) portal (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/gtex/searchGenes">http://www.broadinstitute.org/gtex/searchGenes</ext-link>)[<xref ref-type="bibr" rid="pgen.1006296.ref034">34</xref>] and the Human Protein Atlas (<ext-link ext-link-type="uri" ns0:href="http://www.proteinatlas.org">http://www.proteinatlas.org</ext-link>)[<xref ref-type="bibr" rid="pgen.1006296.ref035">35</xref>] to establish whether the implicated genes and corresponding proteins are expressed in human colon/rectal tissues. Second, we used several eQTL databases including the Browser at University of Chicago (<ext-link ext-link-type="uri" ns0:href="http://eqtl.uchicago.edu/Home.html">http://eqtl.uchicago.edu/Home.html</ext-link>),the Genevar (GENe Expression VARiation) at the Wellcome Trust Sanger Institute (<ext-link ext-link-type="uri" ns0:href="http://www.sanger.ac.uk/resources/software/genevar">http://www.sanger.ac.uk/resources/software/genevar</ext-link>) [<xref ref-type="bibr" rid="pgen.1006296.ref042">42</xref>], HaploReg (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/mammals/haploreg/haploreg.php">http://www.broadinstitute.org/mammals/haploreg/haploreg.php</ext-link>) (PMID:22064851), and the GTEx Portal Version 4(<ext-link ext-link-type="uri" ns0:href="http://gtexportal.org/home/">http://gtexportal.org/home/</ext-link>) (PMID: 26484569) to investigate whether any of the implicated SNPs may impact the expression of the nearby genes. A cis-eQTL analysis was also performed in TCGA COAD data in 356 Caucasian samples that have demographic and clinical data for 15,008 genes (<xref ref-type="supplementary-material" rid="pgen.1006296.s001">S1 Text</xref>). Third, we analyzed expression data for the implicated genes from 35 pairs of colorectal tumor-normal tissue samples included in the ColoCare Cohort (<xref ref-type="supplementary-material" rid="pgen.1006296.s002">S2 Text</xref>) as well as expression data from the Cancer Genome Atlas (TCGA; <ext-link ext-link-type="uri" ns0:href="http://cancergenome.nih.gov">http://cancergenome.nih.gov</ext-link>) in 50 pairs of colorectal adenocarcinoma-normal tissue samples. We searched the UCSC Cancer Genomics Browser (<ext-link ext-link-type="uri" ns0:href="https://genome-cancer.ucsc.edu">https://genome-cancer.ucsc.edu</ext-link>) [37&#8211;39] to examine whether the implicated genes showed evidence of differential expression in colorectal tumor tissue and normal tissue. Last, we used the publically available data in the Gene Expression Omnibus site (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</ext-link>) [<xref ref-type="bibr" rid="pgen.1006296.ref089">89</xref>, <xref ref-type="bibr" rid="pgen.1006296.ref090">90</xref>] and the gene expression data from normal colon (n = 33) and tumor (n = 28) tissue in the ColoCare Cohort (<xref ref-type="supplementary-material" rid="pgen.1006296.s002">S2 Text</xref>) to investigate whether the expression of implicated genes are correlated with alcohol/smoking history.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We explored potential functional annotations for the SNPs that showed evidence for interactions with either smoking or alcohol in our genome-wide interaction analyses. As detailed in <xref ref-type="supplementary-material" rid="pgen.1006296.s001">S1 Text</xref>, we queried multiple bioinformatics databases using the UCSC genome browser (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu">http://genome.ucsc.edu</ext-link>), HaploReg (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/mammals/haploreg/haploreg.php">http://www.broadinstitute.org/mammals/haploreg/haploreg.php</ext-link>), and literature review of published enhancer signatures of colon cancer.</p>