<p xmlns:ns0="http://www.w3.org/1999/xlink"><italic>N</italic>. <italic>crassa &#916;col-26</italic> (FGSC 11031) and &#916;<italic>gln-1</italic> (FGSC 19959) were obtained from the Fungal Genetics Stock Center (<ext-link ext-link-type="uri" ns0:href="http://www.fgsc.net/">http://www.fgsc.net/</ext-link>). The P<italic>gpd-col-26;</italic> &#916;<italic>col-26</italic> strain was constructed by transforming the &#916;<italic>col-26</italic> mutant with a DNA fragment containing the <italic>A</italic>. <italic>nidulans gpdA</italic> promoter, the open reading frame and 3&#8217; untranslated region (UTR) of <italic>col-26</italic>, and flanking regions homologous to the upstream and downstream genomic sequence of the <italic>csr-1</italic> gene. Transformants were selected for resistance for cyclosporin [<xref ref-type="bibr" rid="pgen.1006737.ref061">61</xref>] and tested for genotypes by diagnostic PCR. The transformants with positive results were backcrossed to FGSC 2489 to obtain a <italic>csr-1</italic>::<italic>PgpdA-col-26;</italic> &#916;<italic>col-26</italic> homokaryotic strain.</p><p>The <italic>T</italic>. <italic>reesei</italic> &#916;<italic>bglR</italic> mutants were created by transforming protoplasts of an uridine auxotrophic strain made from QM6a (&#916;<italic>pyr-4</italic>) [<xref ref-type="bibr" rid="pgen.1006737.ref046">46</xref>] with two split-marker DNA fragments using method described in [<xref ref-type="bibr" rid="pgen.1006737.ref062">62</xref>]. One of the split-marker fragment contains a ~1 kbp sequence homologous to upstream genomic sequence of the <italic>bglR</italic> gene followed by the promoter and the first half of the <italic>pyr-4</italic> coding sequence and the other contained the second half the <italic>pyr-4</italic> coding sequence with ~400 bp of overlap sequence with the first half of the <italic>pyr-4</italic> coding sequence and a ~1 kb sequence homologous to the downstream genomic sequence of the <italic>bglR</italic> gene. Transformants were first grown on the plates with minimal media and subsequently transferred to PDA plates for conidiation. Conidia were tested for correct integration of the <italic>pyr-4</italic> gene at the <italic>bglR</italic> locus using diagnostic PCR. The strains with the <italic>bglR</italic> gene disrupted were subjected to single colony purification. Three verified &#916;<italic>bglR</italic> homokaryotic strains were used for downstream analysis.</p><p><italic>N</italic>. <italic>crassa</italic> cultures were grown on slants, each with 3 mL of Vogel&#8217;s minimal medium (VMM) with 2% sucrose [<xref ref-type="bibr" rid="pgen.1006737.ref017">17</xref>] and 2% agar, at 30&#176;C in dark for 24 hours, followed by 4&#8211;10 days in constant light at 25&#176;C to stimulate conidia production. For growth phenotype testing in 24-well plates, conidia were inoculated at 10<sup>6</sup>/ml into 3 mL of VMM with selected carbon and nitrogen sources in 24-well plates covered with breathable rayon film seal, and the culture were grown at 25&#176;C in constant light with shaking at 200 rpm. The film was taken off before imaging. At least two replicates were included in each experiment and the same experiments were done at least twice.</p><p>For mycelial biomass measurement, conidia were inoculated at 10<sup>6</sup>/ml into 100 mL of VMM with selected carbon and nitrogen sources and grown at 25&#176;C in constant light with shaking at 200 rpm. For crosses, one parental strain was grown on plates with synthetic crossing medium [<xref ref-type="bibr" rid="pgen.1006737.ref063">63</xref>] for 2 weeks at room temperature for protoperithecial development. Conidia of the other parental strain were added to the plates for fertilization. Plates were kept for 3 weeks at room temperature. Ascospores were collected and activated as described [<xref ref-type="bibr" rid="pgen.1006737.ref064">64</xref>], plated on VMM with 1% sucrose, and incubated at room temperature for 18 hrs. Germinated ascospores were transferred to VMM slants supplemented with cyclosporin or hygromycin B and screened for desired genotypes by diagnostic PCR.</p><p><italic>T</italic>. <italic>reesei</italic> cultures were grown in either minimal media [<xref ref-type="bibr" rid="pgen.1006737.ref065">65</xref>] for selecting transformants or with PDA for conidiation. For growth phenotype testing, conidia were inoculated at 10<sup>6</sup>/ml into 3 mL minimal media with a selected carbon source in 24-well plates, and the culture were grown at 28&#176;C in dark with shaking at 200 rpm.</p><p>For RNA-seq experiments on VMM with 2% (w/v) maltose and metabolite analyses, conidia were inoculated at 10<sup>6</sup> conidia/mL into 3 mL VMM with 2% cellobiose and grown at 25&#176;C in constant light and shaking at 200 rpm for 28 hrs. The mycelial biomass was washed twice with VMM-NC, followed by 18 hrs of incubation in VMM-NC. Mycelia were then transferred to VMM with maltose and grown 5.5 hrs for RNA-seq experiments, or transferred to VMM or VMM(Glu) with 2% (w/v) glucose and grown 5.5 hrs for metabolite profiling experiments.</p><p>For RNA-seq experiments on VMM with other carbon sources, conidia were inoculated at 10<sup>6</sup> conidia/mL into 3 mL VMM with 2% sucrose and grown at 25&#176;C in constant light and shaking at 200 rpm for 16 hrs. The mycelial biomass was washed twice with VMM-NC and then transferred to VMM with the selected carbon sources for 4 hrs prior to RNA extraction. Concentrations of carbon sources were glycerol (2 mM), fructose (2 mM), mannose (2 mM), trehalose (2 mM), sorbose (2 mM), xylose (2 mM), sucrose (2% w/v), cellobiose (2 mM), maltose (2 mM), avicel (1% w/v), amylose (1% w/v), amylopectin (1% w/v), xyloglucan (1% w/v).</p><p>Mycelia of cultures were harvested by filtration and flash frozen in liquid nitrogen. RNA was extracted using the Trizol method (Invitrogen) and further purified using RNeasy kits (QIAGEN). RNA-seq libraries of WT and &#916;<italic>col-26</italic> from 2% (w/v) maltose were prepared at the Functional Genomics Lab, a QB3-Berkeley Core Research Facility at UC Berkeley and sequenced on an Illumina HiSeq2000 at the Vincent J. Coates Genomics Sequencing Lab. Other libraries were prepared and sequenced at JGI as part of the Neurospora ENCODE CSP project. Total RNA starting material was 1 &#956;g per sample and 10 cycles of PCR was used for library amplification. The prepared libraries were then quantified using KAPA Biosystem&#8217;s next-generation sequencing library qPCR kit and run on a Roche LightCycler 480 real-time PCR instrument. The quantified libraries were then multiplexed into pools of 9 libraries, and the pool was then prepared for sequencing on the Illumina HiSeq sequencing platform utilizing a TruSeq paired-end cluster kit, v3, and Illumina&#8217;s cBot instrument to generate a clustered flowcell for sequencing. Sequencing of the flowcell was performed on the Illumina HiSeq2000 sequencer using a TruSeq SBS sequencing kit, v3, following a 1x100 indexed run recipe.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The sequencing reads that passed filtering from the CASAVA 1.8 FASTQ files were subjected to quality score checking using the FASTX-Toolkit (<ext-link ext-link-type="uri" ns0:href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</ext-link>). Only reads with all bases scoring greater than 22 were used to map against predicted transcripts from the <italic>N</italic>. <italic>crassa</italic> OR74A genome v12 (<italic>Neurospora crassa</italic> Sequencing Project, Broad Institute of Harvard and MIT <ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/">http://www.broadinstitute.org/</ext-link>) with Tophat v2.0.4 [<xref ref-type="bibr" rid="pgen.1006737.ref066">66</xref>]. The output bam files were sorted and indexed using the SAMtools package [<xref ref-type="bibr" rid="pgen.1006737.ref067">67</xref>] and the indexed files were visualized in Integrative Genomics Viewer [<xref ref-type="bibr" rid="pgen.1006737.ref068">68</xref>]. Transcript abundance reflected in FPKM was estimated with Cufflinks v2.0.2 [<xref ref-type="bibr" rid="pgen.1006737.ref066">66</xref>] mapping against reference isoforms. Profiling data are available at the GEO (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</ext-link>; Series Record GSE GSE92848 and GSE95350). For differential gene expression analysis, the bam files were first processed using the HTSeq package v0.6.0 [<xref ref-type="bibr" rid="pgen.1006737.ref069">69</xref>] to generate raw counts, and the raw counts are subjected to differential analysis using the DESeq2 package version 1.10.1 [<xref ref-type="bibr" rid="pgen.1006737.ref070">70</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The FungiFun2 online resource tool was used in functional enrichment analysis (<ext-link ext-link-type="uri" ns0:href="https://elbe.hki-jena.de/fungifun/fungifun.php">https://elbe.hki-jena.de/fungifun/fungifun.php</ext-link>) [<xref ref-type="bibr" rid="pgen.1006737.ref071">71</xref>]. The gene to category associations was tested for over-representation using hypergeometric distribution and the probability for false discovery rate was controlled by the Benjamini-Hochberg procedure.</p><p>Mycelia from 3 mL cultures in 24-well plates were harvested by filtration followed by a quick wash in distilled water. Half of biological replicates were used for metabolite extraction and the other half were dried for biomass measurement. Washed mycelia for metabolite extraction were quickly put into a tube containing 200 &#956;L zirconia beads (0.5 mm) and 500 &#956;L extraction buffer (80% acetonitrile, 20% water, 0.1 M formic acid) and snap frozen in liquid nitrogen. Samples were stored at -80&#176;C until extraction before mass spectrometry (MS) analysis. For metabolite extraction, the frozen samples were immediately put in a bead-beater (BioSpec) and homogenized for 1 min, and cooled on ice. The homogenate were centrifuged at 4&#176;C at 14 000 rpm for 5 minutes, and the supernatants were subjected to either GC-MS or LC-MS analysis.</p><p>For GC-MS analysis, 20 &#956;L of the supernatant was collected and transferred to 1.5 mL micro-tubes containing 50 &#956;L internal standard solution (d27-Myristic acid in methanol, 250 &#956;M). Samples were dried under reduced pressure using a speedvac (Savant). Samples were derivatized for GC-MS analysis according to the method of Kind et al [<xref ref-type="bibr" rid="pgen.1006737.ref072">72</xref>]. Briefly, 10 &#956;L of methoxyamine hydrochloride dissolved in pyridine (40 mg/mL) was added to each dried sample, and shaken at 30&#176;C at maximum speed for 90 min using a thermomixer (Eppendorf). A mixture of retention time marker standards were prepared by dissolving fatty acid methyl esters (FAMEs) of different linear chain lengths in chloroform (C8, C9, C10, C12, C14, C16 FAMES at 0.8 mg/ml, and C18, C20, C22, C24, C26, C28, C30 at 0.4 mg/ml). The FAME mixture (20 &#956;L) was added to 1 mL of N-methyl-N-trimethylsilytrifluoroacetamide (MSTFA) containing 1% trimethylchlorosilane (TMCS), and 90 &#956;L of the FAMEs/MSTFA solution was added to each sample. Samples were shaken at 37&#176;C at maximum speed in a thermomixer for 30 min, and then transferred to and sealed in amber GC-MS sample vials containing glass inserts (Agilent). Extraction blanks were prepared following the above procedure but starting with empty Eppendorf tubes.</p><p>For LC-MS analysis, supernatant (350 &#956;L) was collected and filtered through 0.2 &#956;m spin filters (Pall) by centrifugation for 1 min at 14000 rpm. Fifty &#956;L of the filtrate was transferred to HPLC vials containing 50 &#956;L of an internal standard mixture solution. Samples were kept at 4&#176;C in the LC-MS autosampler chamber. Extraction blanks were prepared in triplicate by following the above sample preparation procedure with empty microtubes.</p><p>For GC-MS analysis, samples were analyzed using an Agilent 7890 gas chromatograph (Agilent Technologies, Santa Clara, CA) connected to an Agilent 5977 single quadrupole mass spectrometer, all controlled by Agilent GC-MS MassHunter Acquisition software. Samples were injected using a Gerstel automatic liner exchange MPS system (Gerstel, Muehlheim, Germany) controlled by Maestro software. Sample injection volume was 2 &#956;L, and the injector was operated in splitless mode. Samples were injected into the 50&#176;C injector port which was ramped to 270&#176;C in a 12&#176;C/s thermal gradient and held for 3 min. The gas chromatograph was fitted with a 30m long, 0.25mm ID Rtx5Sil-MS column (Restek, Bellefonte, PA), 0.25 mm 5% diphenyl film with a 10 m integrated guard column. Initial oven temperature was set at 50&#176;C, and the over program was as follows: ramp at 5&#176;C/min to 65&#176;C, held for 0.2 min; ramp at 15&#176;C/ min to 80&#176;C, held for 0.2 min; ramp at 15&#176;C/min to 310&#176;C, hold for 12 min. The mass spectrometer transfer line and ion source temperature was 250&#176;C and 230&#176;C, respectively. Electron ionization was at 70 eV and mass spectra were acquired from 50 to 700 m/z at 8 spectra per second. Raw data was visually inspected using Agilent MassHunter Qualitative Analysis software (Agilent Technologies, Santa Clara, CA). Agilent MassHunter Unknowns Analysis software v. B.07.00 (Agilent Technologies, Santa Clara, CA) was used to perform peak deconvolution and library matching. A library match score was calculated for using FAME markers for retention time calibration, and matching mass fragmentation spectra to those in the Fiehn GC-MS Metabolomics RTL Library [<xref ref-type="bibr" rid="pgen.1006737.ref072">72</xref>]. Metabolites of interest were only included in further analysis if their library match scores were greater than 75%. The identities of some metabolites with scores lower than 90% were confirmed by comparing mass spectra and retention times with that of authentic reference standards (<xref ref-type="supplementary-material" rid="pgen.1006737.s006">S6 Table</xref>). Mass spectral and retention time data from identified target metabolites were used to make an analysis method in MassHunter Quantitative Analysis Software for GCMS (v.B.07.00). For each metabolite, a quantifier ion and two qualifier ions were defined to produce an extracted ion chromatogram in a specified retention time window. Integration of the extracted ion chromatogram peaks yielded peak areas that were further normalized by the mean of dry fungal biomass from biological replicates. The normalized peak areas were used for comparing the relative abundance of metabolites across samples.</p><p>Targeted LC-MS analysis was performed for select metabolites not detected by GC-MS (<xref ref-type="supplementary-material" rid="pgen.1006737.s007">S7 Table</xref>). Samples were analyzed on an Agilent 6550 ESI-QTOF LCMS fitted with a Merck SeQuant Zic-HILIC column (150 x 1 mm, 3.5 mm, 100 &#197;) with a guard column. Mobile phase consisted of 5% ammonium acetate in water (solvent A), and 5% ammonium acetate in water-acetonitrile (10:90) (solvent B). The following LC solvent time-table was used: 0 min, 100% B; 1.5 min, 100% B; 25 min, 50% B; 26 min, 35% B; 32 min, 35% B; 33 min, 100% B; 40 min, 100% B. Flow rate: 0.25 ml/min; injection volume: 2 &#956;L. Each sample was analyzed in positive and negative ionization mode. Raw data was analyzed using Agilent MassHunter Qualitative analysis. Extracted ion chromatograms were produced from raw scan data using calculated m/z values for target metabolites, corresponding to their molecular ion and potential adducts: (M+H)<sup>+</sup>, (M+Na)<sup>+</sup>, (M+K)<sup>+</sup> for Positive mode; (M-H)<sup>-</sup>, (M+COOH)<sup>-</sup>, (M+CH3COOH)<sup>-</sup> for Negative mode. The identity of detected ions were confirmed by comparing retention time with reference standards, or checked by performing MS/MS analysis of the target ion, and comparing ion fragments with those in the METLIN online database. Integration of the extracted ion chromatogram peaks yielded peak areas that were further normalized by the mean of dry fungal biomass from biological replicates. Normalized peak areas were used for comparing the relative abundance of target metabolites across samples.</p><p>For differential metabolite analysis, the normalized peak areas were log transformed and then used in the independent t-test of hypothesis that there is no difference between WT and the mutant. P values of less than 0.05 were considered significantly different and values between 0.05 and 0.1 were interpreted as indicating a trend toward statistical significance. Four biological replicates measured by GC-MS and two biological replicates measured by LC-MS were used in differential analysis. All metabolites that were found to be either significantly different or with a trend toward statistical significance were subjected to hierarchical clustering analysis.</p><p>Hierarchical clustering analysis is performed with Cluster 3.0 [<xref ref-type="bibr" rid="pgen.1006737.ref073">73</xref>] using log transformed mean of normalized peak areas from biological replicates. The values were centered to the mean across different growth conditions and normalized on a per metabolite basis. Average linkage clustering was performed with Euclidean distance as the similarity metric.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Protein sequences of selected ascomycetes were downloaded from JGI Mycosm [<xref ref-type="bibr" rid="pgen.1006737.ref074">74</xref>] and used to construct a local protein database using the NCBI BLAST+ application (version 2.2.31) (<xref ref-type="supplementary-material" rid="pgen.1006737.s005">S5 Table</xref>). The putative COL26 orthologs were searched in the database using BLASTP with a cut-off E value less than e-20. All hits were tested by reciprocally BLASTP against the <italic>N</italic>. <italic>crassa</italic> database and only ones that resulted in COL26 as the best hit were retained for protein sequence alignment. Protein sequences of the putative COL26 orthologs from selected species were aligned using three different programs: Clustal Omega [<xref ref-type="bibr" rid="pgen.1006737.ref075">75</xref>], MAFFT [<xref ref-type="bibr" rid="pgen.1006737.ref076">76</xref>], and MUSCLE [<xref ref-type="bibr" rid="pgen.1006737.ref077">77</xref>], and the best alignment was chosen and further trimmed using trimAl [<xref ref-type="bibr" rid="pgen.1006737.ref078">78</xref>]. The trimmed alignment file was used for phylogenetic tree construction by the RAxML program with 200 bootstraps [<xref ref-type="bibr" rid="pgen.1006737.ref079">79</xref>]. The result were visualized and edited in iTOL (<ext-link ext-link-type="uri" ns0:href="http://itol.embl.des/">http://itol.embl.des/</ext-link>) [<xref ref-type="bibr" rid="pgen.1006737.ref080">80</xref>].</p>