<p>All procedures were approved by the Tel-Aviv University Animal Care Committee (L-10-011 and L-15-047) and conducted in accordance with the National Council for Animal Experimentation, Ministry of Health, Israel.</p><p>The <italic>pT2-aanat2</italic>:<italic>EGFP-2A-&#916;clocka-5&#215;MYC</italic> construct (<xref ref-type="fig" rid="pgen.1006445.g001">Fig 1A</xref>) was generated using the backbone of the Tol2 transposable element-containing vector, pT2KXIG&#916;in [<xref ref-type="bibr" rid="pgen.1006445.ref066">66</xref>]. The coding sequences of EGFP and a truncated zebrafish CLOCKa (&#916;CLK; [<xref ref-type="bibr" rid="pgen.1006445.ref038">38</xref>]) linked to 5 Myc tag sequences were cloned downstream of the <italic>aanat2</italic> regulatory regions [<xref ref-type="bibr" rid="pgen.1006445.ref036">36</xref>]. A sequence encoding the 'self-cleaving' 2A peptide was cloned between the open reading frames of EGFP and &#916;CLK, in order to produce two distinct proteins [<xref ref-type="bibr" rid="pgen.1006445.ref039">39</xref>]. The sequence of the <italic>pT2-aanat2</italic>:<italic>EGFP-2A-&#916;clocka-5&#215;MYC</italic> construct is provided in <xref ref-type="supplementary-material" rid="pgen.1006445.s009">S9 Fig</xref>.</p><p>The transgenic line, Tg(<italic>aanat2</italic>:EGFP-&#916;CLK), registered in the Zebrafish Model Organism Database (ZFIN) as <italic>Tg(aanat2</italic>:<italic>EGFP-2A-&#916;clocka-5&#215;MYC)tlv03</italic>, was generated using the Tol2 system as described [<xref ref-type="bibr" rid="pgen.1006445.ref067">67</xref>]. Tol2 plasmids were kindly provided by Koichi Kawakami. Capped Tol2 transposase mRNA was synthesized <italic>in vitro</italic> using the mMESSAGE mMACHINE SP6 Transcription Kit (Ambion) and a linearized pCS-TP plasmid as template. Approximately 1 nl of a DNA/RNA solution containing 25 ng/&#956;l of the <italic>pT2-aanat2</italic>:<italic>EGFP-2A-&#916;clocka-5&#215;MYC</italic> circular DNA and 25 ng/&#956;l of transposase mRNA were injected into fertilized eggs at the single-cell stage. Founder (F0) fish were raised to adulthood and outcrossed to screen for integration of the transgene into the germline. Transgenic EGFP-expressing progeny (F1) were isolated using a fluorescence stereomicroscope. Several transgenic lines were obtained, all of them expressing EGFP specifically in the melatonin-producing pineal cells. F1 fish that showed strong transgene expression were further propagated. F2 progeny from one selected outcrossed F1 fish were further incrossed to generate homozygotes in the F3 generation. F3 homozygotes were further incrossed to produce homozygous transgenic progeny. WT siblings of F3 homozygotes were also maintained and further incrossed to produce control WT progeny. As in the case of transient expression of &#916;CLK in embryos [<xref ref-type="bibr" rid="pgen.1006445.ref038">38</xref>], the stable expression of &#916;CLK in the pineal gland did not affect overall embryonic development.</p><p>Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) larvae (5 dpf) were fixed, and whole-mount immunostaining was carried out as previously described [<xref ref-type="bibr" rid="pgen.1006445.ref068">68</xref>] with mouse anti-Myc antibody 1:100 (clone 9E10; Santa Cruz Biotechnology).</p><p>Anesthetized or immunostained Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) larvae were placed in low melting point agarose. Images were obtained using a Leica TCS SP8 confocal laser scanning microscope equipped with Leica LAS AF image acquisition software.</p><p>Heterozygous Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) larvae and their WT siblings were entrained by seven 12-hr:12-hr LD cycles, transferred to DD and sampled at 4-hr intervals throughout one daily cycle. The collected embryos were fixed and subjected to whole-mount ISH followed by quantification as previously described [<xref ref-type="bibr" rid="pgen.1006445.ref069">69</xref>,<xref ref-type="bibr" rid="pgen.1006445.ref070">70</xref>], using the <italic>aanat2</italic> probe [<xref ref-type="bibr" rid="pgen.1006445.ref040">40</xref>].</p><p>Adult fish were raised in a temperature-controlled recirculation water system under 12-hr:12-hr LD cycles and their pineal glands were collected and cultured in a flow-through system as previously described [<xref ref-type="bibr" rid="pgen.1006445.ref053">53</xref>]. Fish were anesthetized in 1.5 mM Tricane (Sigma-Aldrich), decapitated, and pineal glands were removed by a surgical procedure carried out under a dissecting microscope (Olympus SZX12). Three pineal glands were collected from homozygous Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) fish, and three from control fish (progeny of WT siblings). The pineal glands were kept in culture medium and placed in a flow-through system. Each individual gland was placed in a glass column and continuously perfused with medium (1 ml/hr) delivered by a multi-channel peristaltic pump (Minipuls 3; Gilson). The culture medium was MEM (Sigma-Aldrich), supplemented with 2 mM L-glutamine, 0.1 mM L-tryptophan, 0.02 M sodium bicarbonate, penicillin (100,000 U/l)&#8211;streptomycin (100 mg/l) (Biological Industries) and 2.5 mg/l Fungizone (Amphotericin B; Sigma-Aldrich). The medium was continuously bubbled with a 5% CO<sub>2</sub>: 95% O<sub>2</sub> gas mixture during the experiment. Fractions (1 ml) of medium were collected at 1-hr intervals using a multi-channel fraction collector (FC204; Gilson). The apparatus was placed inside a light- and temperature-controlled incubator and the temperature was maintained at 24&#176;C. Pineal glands were exposed to one DL cycle followed by two DD cycles.</p><p>The concentration of melatonin in the collected medium was determined by high performance liquid chromatography (HPLC) using a 125 &#215; 4.6 mm C18(2) reversed-phase analytic column (Phenomenex Luna) with a particle size of 5 &#956;m and a Dionex UltiMate 3100 fluorescence detector (Thermo Scientific). A volume of 100 &#956;l was injected from samples at 2-hr intervals. The excitation and emission wavelengths were 280 nm and 340 nm, respectively. The mobile phase consisted of 0.1 M Na<sub>2</sub>HPO<sub>4</sub> containing 20% acetonitrile; the pH was adjusted to 6.5 with orthophosphoric acid. The mobile phase flow was 1.5 ml/min and the melatonin retention time (about 7 min) was confirmed using a commercial standard (Sigma-Aldrich).</p><p>To account for variation in the basal levels of melatonin secretion from individual pineal glands, melatonin levels secreted by each pineal gland were normalized by dividing the absolute levels by the maximal night-time levels. Levels of melatonin production by each pineal gland under DD underwent Fourier analysis and were scored with a G-factor ratio to determine circadian rhythmicity (see 'Fourier analysis' in <xref ref-type="supplementary-material" rid="pgen.1006445.s013">S1 Text</xref>). Statistical differences between Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) and control fish in rhythmic melatonin production under DD were determined by the Kolmogorov-Smirnov test.</p><p>Adult homozygous Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) fish were raised in a temperature-controlled recirculation water system under 12-hr:12-hr LD cycles, and transferred to DD at the end of the light period prior to sampling. Pineal glands were sampled at 4-hr intervals throughout two daily cycles under DD at 12 time points corresponding to circadian time (CT) 14, 18, 22, 2, 6, 10, 14b, 18b, 22b, 2b, 6b and 10b (<xref ref-type="fig" rid="pgen.1006445.g004">Fig 4A</xref>), as previously described [<xref ref-type="bibr" rid="pgen.1006445.ref042">42</xref>]. A pool of 16 pineal glands was collected at each time point. In addition, two control pools of 14 pineal glands were collected from Tg(<italic>aanat2</italic>:EGFP) fish at time points corresponding to CT2 and CT14b. Fish were anesthetized in 1.5 mM Tricane (Sigma-Aldrich) and decapitated. Fluorescent pineal glands were selectively removed under a dissecting microscope (Olympus SZX12) equipped with filters for excitation (460&#8211;490 nm) and emission (510&#8211;550 nm) of EGFP. Total RNA for mRNA analysis was isolated using RNeasy Lipid Tissue Mini Kit (Qiagen).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">mRNA-seq data acquisition and analysis were carried out as a replicate of a previously described procedure [<xref ref-type="bibr" rid="pgen.1006445.ref042">42</xref>]. The Illumina TruSeq protocol was used to prepare libraries from RNA samples. Overall, 14 libraries [12 time points from Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) fish and two control samples from Tg(<italic>aanat2</italic>:EGFP) fish] were run on a single flow cell of an Illumina HiSeq2500 machine (rapid run mode) using the multiplexing strategy of the TruSeq protocol (Institute of Applied Genomics, Italy). On average, 14 million paired-end reads were obtained for each library. The reads were of 2&#215;50 base pairs. The sequencing data were deposited in the Sequence Read Archive, under accession SRP016132. TopHat [<xref ref-type="bibr" rid="pgen.1006445.ref071">71</xref>] was used for aligning the reads against the zebrafish genome, keeping only uniquely aligned reads with up to two mismatches per read. On average, 68% of the reads had unique alignment with the zebrafish genome. Reads aligned with the protein coding regions of known NCBI reference sequence (RefSeq) genes were used. A custom script written in Perl was used to parse the output of TopHat, which is given in Sequence Alignment/Map (SAM) format (<ext-link ext-link-type="uri" ns0:href="http://samtools.sourceforge.net/">http://samtools.sourceforge.net/</ext-link>), and to convert it into a raw number of reads aligned to each position in each RefSeq gene. The RefSeq genes data was obtained from the Table Browser of the UCSC genome browser (genome.ucsc.edu/) using the zebrafish July 2010 (Zv9/danRer7) assembly. To avoid PCR duplicates, only paired-end reads with unique start positions in the genome in both pairs were used [<xref ref-type="bibr" rid="pgen.1006445.ref072">72</xref>].</p><p>The 26 mRNA-seq profiles (i.e., the number of reads aligned against each RefSeq gene) corresponding to the 12 time points using pineal glands of Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) fish, the same 12 time points using pineal glands of control Tg(<italic>aanat2</italic>:EGFP) fish (data from Tovin et al. [<xref ref-type="bibr" rid="pgen.1006445.ref042">42</xref>]), and the two control pineal gland samples from Tg(<italic>aanat2</italic>:EGFP) fish (CT2 and CT14b) were normalized together. The logarithmically transformed dataset was normalized using quantile normalization [<xref ref-type="bibr" rid="pgen.1006445.ref073">73</xref>]. Transcripts with low maximum expression values (i.e., their highest level over all time points is in the lower quartile of all transcripts) were not included in the Fourier analysis.</p><p>Fourier analysis was conducted as previously described ([<xref ref-type="bibr" rid="pgen.1006445.ref042">42</xref>]; <xref ref-type="supplementary-material" rid="pgen.1006445.s013">S1 Text</xref>).</p><p>See <xref ref-type="supplementary-material" rid="pgen.1006445.s013">S1 Text</xref>.</p><p>Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) fish were crossed with Tg(&#8722;3.1)<italic>per1b</italic>::luc fish [<xref ref-type="bibr" rid="pgen.1006445.ref043">43</xref>]. Embryos were raised under 12-hr:12-hr LD cycles at 25&#176;C. At 3 dpf, single larvae were transferred into individual wells of a 96-multiwell plate (Nunc) in E3 media (without methylene blue) supplemented with 0.5 mM beetle luciferin potassium salt solution (Promega), and the plate was sealed using an adhesive TopSeal sheet (Packard). Plates were then subjected to two LD cycles, followed by two daily cycles under DD, under which bioluminescence from whole larvae was assayed using a TopCount NXT Scintillation Counter (2-detector model; Packard). Bioluminescence data were analyzed using the Import and Analysis Macro (I&amp;A, Plautz and Kay, Scripps) for Microsoft Excel or CHRONO software [<xref ref-type="bibr" rid="pgen.1006445.ref074">74</xref>]. Short-term and long-term trends were removed from the raw data by an adjacent-averaging method with 3-hr and 2-day running means, respectively. Normalized values were obtained by dividing the bioluminescence values by their average value for each larva. Traces represent the mean value &#177; SD of 23 Tg[<italic>aanat2</italic>:EGFP-&#916;CLK;(&#8722;3.1)<italic>per1b</italic>::luc] larvae and 55 control Tg(&#8722;3.1)<italic>per1b</italic>::luc larvae; each group was comprised of larvae from two separate crosses. To determine the circadian rhythmicity of <italic>per1b</italic> promotor activity, bioluminescence data from each larva underwent Fourier analysis and were scored with a G-factor ratio (see 'Fourier analysis' in <xref ref-type="supplementary-material" rid="pgen.1006445.s013">S1 Text</xref>). Kolmogorov-Smirnov test was applied to compare the distribution of G-factors between Tg[<italic>aanat2</italic>:EGFP-&#916;CLK;(&#8722;3.1)<italic>per1b</italic>::luc] and control Tg(&#8722;3.1)<italic>per1b</italic>::luc larvae.</p><p>See <xref ref-type="supplementary-material" rid="pgen.1006445.s013">S1 Text</xref>.</p><p>Homozygous Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) embryos and control embryos (progeny of WT siblings) were raised in a light- and temperature-controlled incubator under 12-hr:12-hr LD cycles at 28&#176;C. On the 4<sup>th</sup> day of development, larvae were placed in 48-well plates in the observation chamber of the DanioVision tracking system (Noldus Information Technology) for acclimation under controlled temperature (28&#176;C) and lighting conditions (LED; intensity of 'light' and 'dim light' were 1.8 W/m<sup>2</sup> and 0.013 W/m<sup>2</sup>, respectively) according to the desired protocol. Starting from the 6<sup>th</sup> day of development, movement was tracked and analyzed by the Ethovision 11.0 software (Noldus Information Technology).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For the analysis of locomotor activity, the raw data were converted into the total distance moved (cm) by each larva per 10 min time-bins. The data are presented as a moving average (20 sliding points) of 24 larvae in each group, excluding experiments with light-dark or dark-light transitions that trigger a temporary rise in activity (<xref ref-type="fig" rid="pgen.1006445.g007">Fig 7D</xref> and <xref ref-type="supplementary-material" rid="pgen.1006445.s004">S4 Fig</xref>). To determine alterations in circadian rhythms of locomotor activity, individual tracks underwent Fourier analysis and were scored with a G-factor ratio (see 'Fourier analysis' in <xref ref-type="supplementary-material" rid="pgen.1006445.s013">S1 Text</xref>). Differences in G-factor distributions between Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) and control groups were determined by the Kolmogorov-Smirnov test. The periods of locomotor activity rhythms were computed by the chi-square periodogram [<xref ref-type="bibr" rid="pgen.1006445.ref075">75</xref>] with ActogramJ software ([<xref ref-type="bibr" rid="pgen.1006445.ref076">76</xref>]; <ext-link ext-link-type="uri" ns0:href="http://actogramj.neurofly.de/">http://actogramj.neurofly.de/</ext-link>), and statistical differences between Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) and control larvae were determined by <italic>t</italic>-test. Amplitude values were calculated as the difference between the peak of activity at day 7 and the preceding trough, divided by 2, and statistical differences between Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) and control larvae were determined by <italic>t</italic>-test. For analyzing the experiment in which activity was monitored under 3.5-hr light: 3.5-hr dark cycles (masking protocol, <xref ref-type="supplementary-material" rid="pgen.1006445.s005">S5 Fig</xref>), the percentage of activity during the bouts of light and dark was calculated, and the difference between Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) and control larvae was determined by <italic>t-</italic>test.</p><p>For the analysis of sleep and waking activity, the raw data were converted to the number of seconds spent moving per 1-min time bin for each larva, with stop velocity threshold of 0.59 cm/s and start velocity threshold of 0.6 cm/s [<xref ref-type="bibr" rid="pgen.1006445.ref065">65</xref>]. Sleep time was calculated as the number of minutes without movement per 1-hr. Waking activity was computed as the average number of seconds of activity/waking minutes per 1-hr. Data are presented as the average sleep time or average waking activity of 24 larvae in each group, and also as the average sleep time or average waking activity during subjective day and subjective night periods. Repeated-measures ANOVA was applied to compare the sleep time and log-transformed waking activity values of Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) and control groups during subjective day and subjective night periods.</p><p>For place preference, larvae were tested in groups of 15 in chambers of 15 mm &#215; 22 mm &#215; 50 mm (width, length, height). Larvae were initially entrained under LD cycles for 4 days, and were then placed in the chamber under DD. Recordings were initiated after a 24-hr acclimation period, using a &#956;Eye IDS-1545LE-M CMOS camera (1stVision) to capture a snapshot of each group every 15 sec. Backlit illumination was provided by an 880-nm infrared LED array (Advanced Illumination) which was activated for 50 ms, synchronized with image acquisition. Timing was controlled by DAQtimer event control software and image analysis to detect the position of larvae performed using FLOTE [<xref ref-type="bibr" rid="pgen.1006445.ref077">77</xref>]. For analysis, we then calculated the proportion of larvae in each of three identically sized zones of the chamber: top, middle and bottom. For behavioral testing we used homozygous Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) larvae, and progeny of WT siblings as controls.</p><p>The raw data were converted into the average percentage of larvae in the top third of the water column per 5-min time-bins. The data are presented as a moving average (40 sliding points) of four groups of Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) larvae and eight groups of control larvae. To determine alterations in circadian rhythms of environmental positioning, the data underwent Fourier analysis and were scored with a G-factor ratio (see 'Fourier analysis' in <xref ref-type="supplementary-material" rid="pgen.1006445.s013">S1 Text</xref>). Differences in the G-factor distributions between Tg(<italic>aanat2</italic>:EGFP-&#916;CLK) and control groups were determined by the Kolmogorov-Smirnov test. Differences in the average percentage of larvae in the top third of the water column between CT 5&#8211;6 (subjective day) and CT 17&#8211;18 (subjective night), under two daily cycles, were determined by paired <italic>t</italic>-test.</p>