<p>Flies were maintained on standard cornmeal medium at room temperature or 25&#176;C. Larvae for qRT-PCR and western blotting were raised at 25&#176;C. <italic>AGO2</italic><sup><italic>51B</italic></sup> and <italic>AGO2</italic><sup><italic>V966M</italic></sup> mutants were described in [<xref ref-type="bibr" rid="pgen.1007276.ref001">1</xref>,<xref ref-type="bibr" rid="pgen.1007276.ref013">13</xref>], respectively. <italic>AGO2</italic> wild-type genomic rescue was previously described in [<xref ref-type="bibr" rid="pgen.1007276.ref025">25</xref>].</p><p>Kc167 cells were grown in CCM3 media (Thermo Scientific HyClone). Cells were maintained at 25&#176;C. To produce a similar analysis of the larval mutants but in Kc cells, we attempted to generate <italic>AGO2</italic> mutants by CRISPR technology with no successful results perhaps because of the tetraploid nature of the Kc cell line and/or its sensitivity to the clonal dilution procedure.</p><p>Amplicons used for dsRNA knockdowns were designed based on recommendations from the <italic>Drosophila</italic> RNAi Screening Center. Templates were PCR-amplified from genomic DNA using primers containing the T7 promoter sequence. In vitro transcription of PCR templates using the MEGAscript T7 kit (Ambion) was used to produce dsRNAs, and these were purified by phenol-chloroform extraction. Transfections using 2 &#956;g of dsRNA or 100 pmol of siRNA, or no dsRNA/siRNA as mock treatment, were performed using Cell Line Nucleofector kit V (Amaxa Biosystems) transfection reagent using the G-30 program. Three days after transfection, cells were collected and knockdown efficiency was confirmed by Western blotting.</p><p>To generate AGO2-rescues, AGO2 cDNA was cloned into p-ENTR/D-TOPO gateway vector (Invitrogen). Site-directed mutagenesis was carried out to obtain the AGO2<sup>V966M</sup> mutant. These constructs were recombined into the pAFHW (Carnegie vectors) expression vector to generate N-terminal 3xFlag, 3XHA fusion constructs. Each construct was co-transfected with an siRNA designed against the 5&#8217;UTR of AGO2.</p><p>Nuclear extracts from 11 g of 0&#8211;24 h OR embryos were prepared as previously described [<xref ref-type="bibr" rid="pgen.1007276.ref026">26</xref>] and lysed in 2.5 mL HBSMT-0.3% + 1 M KCl (50 mM HEPES, 150 m MNaCl, 1 M KCl, 3 mM MgCl<sub>2</sub>, 0.3% Triton X-100 [v/v] at pH 7) including 1 mM PMSF, and Complete protease inhibitor cocktail (Roche). For AGO2 immunoaffinity purification, 6 mL of 9D6 tissue culture supernatant was covalently crosslinked to rProtA sepharose beads using 20 mM dimethylpimelimidate in 0.2 M sodium borate, pH 9.0 quenched with 0.2 M ethanolamine, pH 7.9. For control sample, 2.4 &#956;g normal mouse IgG (Santa Cruz) was used. Crosslinked beads were incubated with 5.5 mg nuclear lysate overnight at 4&#176;C and washed as described previously [<xref ref-type="bibr" rid="pgen.1007276.ref001">1</xref>]). Samples were eluted in formic acid, trypsin digested, subjected to ultraperformance liquid chromatography on a NanoAcquity system (Waters), and analyzed on an LTQ-Orbitrap-Velos system (Thermo) at the NIDDK Mass Spectrometry Facility. Results were analyzed by Mascot algorithm (Matrix Science).</p><p>Chromatin fractionation was performed according to [<xref ref-type="bibr" rid="pgen.1007276.ref002">2</xref>,<xref ref-type="bibr" rid="pgen.1007276.ref027">27</xref>] with minor modifications. In brief, approximately 1&#8211;2 X 10<sup>7</sup> Kc cells were washed twice with cold PBS. Cells were lysed for 15 min on ice in cold CSKI buffer (10 mM PIPES pH 6.8, 100 mM NaCl, 1 mM EDTA, 300 mM sucrose, 1 mM MgCl<sub>2</sub>, 0.5% (v/v) Triton X-100 supplemented with 1 Mini Complete tablet (Roche) per 10 mL lysis buffer. Cell lysate was divided into two portions, which were centrifuged at 500 x<italic>g</italic> at 4&#176;C for 5 min. The supernatants (S1 fraction), which contain Triton-soluble proteins, were further analyzed. One of the pellets (P1 fraction) was washed twice in CSKI buffer and then resuspended in RIPA buffer (150 mM Tris-HCl pH 8, 150 mM NaCl, 0.5% sodium deoxycholate, 0.1% (w/v) SDS, 1% (v/v) NP-40). The second pellet, after washing in CSKI buffer, was resuspended in CSKII buffer (10 mM PIPES pH 6.8, 50 mM NaCl, 300 mM sucrose, 6 mM MgCl<sub>2</sub>), then treated with 10 U DNase I (Roche) for 30 min and extracted with 250 mM NH<sub>2</sub>SO<sub>4</sub> for 10 min at 25&#176;C. The sample treated with DNase I and salt was then centrifuged at 1,200 x<italic>g</italic> for 5 min at 4&#176;C, and the supernatant (S2 fraction) and pellet (P2 fraction) were collected. The P2 fraction was also resuspended in RIPA buffer. All fractions were analyzed by western blotting.</p><p>Approximately 1 X 10<sup>8</sup> cells were harvested and washed twice with cold PBS. Cells were lysed for 5 min on ice in lysis buffer (Tris-HCl pH 7.4 30 mM, 100 mM NaCl, 10 mM MgCl<sub>2</sub>, 0.1% Triton X-100) supplemented with Complete EDTA free protease inhibitors (Roche) and centrifuged for 5 min at 500 x<italic>g</italic> at 4&#176;C. The supernatant was removed and pellets were washed once with lysis buffer, resuspended in IP buffer (50 mM Tris-HCl pH 7.4, 150 mM NaCl, 0.3 mM MgCl<sub>2</sub>, 0.3% Triton X-100), incubated 10 min on ice and sonicated 1 cycle for 10 sec, to shear DNA (nuclear fraction). Nuclear extracts were incubated with normal mouse serum or a monoclonal antibody against LaminB (ADL67.10) from Developmental Studies Hybridoma Bank (DSHB), and allowed to bind for 1 h at 4&#176;C. Next, 50&#8201;&#956;L of prewashed Protein G beads 50% slurry was added. After incubation for 2 h, unbound supernatant was removed, and the beads were washed once in IP buffer and twice in TBS (50 mM Tris-HCl pH 7.4, 150 mM NaCl). The bound protein was eluted in sample buffer by boiling, separated by using SDS-PAGE, transferred to nitrocellulose in 10 mM CAPS, pH 11 and detected by western blotting. Proteins were detected with the SuperSignal substrate (Pierce).</p><p>neuRNA labeling and capture were performed with Click-iT Nascent RNA Capture Kit (Thermo Fisher Scientific) according to manufacturer&#8217;s protocol with minor modifications. Cells were incubated with EU at 0.2 mM for 1 h and RNA was extracted with TRizol (Thermo Fisher Scientific). The Click-iT reaction was performed with 0.25 mM biotin azide, and biotinylated RNA was captured with 12 &#956;L T1 beads. Nascent RNA was used to generate RNA-seq libraries with Ovation RNA-seq Systems 1&#8211;16 for Model Organisms (Nugen). All samples were sequenced with HiSeq2500 (Illumina) at the NIDDK Genomics Core Facility by 50 bp single-end sequencing.</p><p>Whole female larvae were collected and total RNA extracted with TRizol (Invitrogen). Polyadenylated RNA and ribosomal (rRNA)-depleted RNA was purified from total RNA using the MicroPoly(A)Purist Kit (Ambion) and the RiboMinus Eukaryote Kit for RNA-seq (Invitrogen), respectively. Sequencing libraries were prepared from Poly(A)<sup>+</sup> and rRNA-depleted RNA samples according to the manufacturer&#8217;s protocol (Illumina). All samples were sequenced with HiSeq2500 (Illumina) at the NIDDK Genomics Core Facility by 50 bp single-end sequencing.</p><p>Approximately 1&#8211;2 X 10<sup>7</sup> cells were fixed by addition of 1% formaldehyde to cell media for 10 min at RT with gentle agitation. Formaldehyde was quenched by addition of glycine to 0.125 M with gentle agitation for 5 min at RT. Cells were pelleted at 2000 x<italic>g</italic>, washed twice in PBS, and resuspended in 0.8 mL ice&#8211;cold cell lysis buffer (5 mM PIPES pH 8, 85 mM KCl, 0.5% NP-40) supplemented with Complete protease inhibitors (Roche), incubated on ice 10 min pelleted by centrifugation at 2000 x<italic>g</italic> for 5 min at 4&#176;C. Next, the supernatant was removed and pellets were resuspended in 1 mL nuclear lysis buffer (50 mM Tris-HCl pH 8, 10 mM EDTA.Na2, 1% SDS) and incubated for 10 min at 4&#176;C. Afterwards, 0.5 mL of IP dilution buffer was added (16.7 mM Tris-HCl pH 8, 1.2 mM EDTA, 167 mM NaCl, 1.1% Triton X-100, 0.01% SDS) and chromatin was fragmented to an average size of 300 bp by using PicoBioruptor (Diagenode) using 10 cycles of 30 s on plus 30 s off, maximum output. Samples were centrifuged at max speed for 10 min at 4&#176;C, and the supernatant (sheared chromatin) was saved at -80&#176;C. Chromatin was diluted to 1:5 with IP buffer, and the assayed antibody in addition to 50 &#956;L of prewashed protein A/G 50% slurry was added and rotated overnight at 4&#176;C. The next day, beads were washed as follows:</p><p>3X Low salt IP dilution buffer (20 mM Tris-HCl pH 8, 2 mM EDTA.Na2, 150 mM NaCl, 1% Triton X-100, 0.1% SDS).</p><p>3X High salt IP dilution buffer (20 mM Tris-HCl pH 8, 2 mM EDTA.Na2, 500 mM NaCl, 1% Triton X-100, 0.1% SDS).</p><p>2X times LiCl buffer (10 mM Tris-HCl pH 8, 1 mM EDTA.Na2, 250 mM LiCl, 1% NP-40, 1% DOC).</p><p>Chromatin was eluted twice with 200 &#956;L of elution buffer (500 &#956;L of 1M NaHCO<sub>3</sub>, 250 &#956;L of 20% SDS, 4.25 mL dH<sub>2</sub>O) for 30 min at 65&#176;C each and further incubated overnight at 65&#176;C with 38 &#956;L of decrosslinking solution (20 &#956;L of 5M NaCl, 8 &#956;L of 0.5M EDTA, 10 &#956;L of 1M Tris-HCl pH 8). After de-crosslinking, samples were treated with Proteinase K for 2 h at 50&#176;C and then combined with 1 vol of phenol/chloroform/isoamyl alcohol (25:24:1), vortexed 15 s, and centrifuged for 5 min at 10,000 x<italic>g</italic>. The top layer was transferred to a new tube, and the procedure was repeated using 1 vol chloroform. The top layer was collected and subsequently precipitated with 0.1 vol of 3M NaOAc pH 5.2 and 2.5 vol of 100% ethanol supplemented with 2 &#956;L of Glycoblue (Ambion). After incubating 30 min at -80&#176;C, samples were centrifuged 20 min at 4&#176;C at 10,000 x<italic>g</italic>. Pellets were washed with 70% ethanol and centrifuged 5 min at 4&#176;C at 10,000 x<italic>g</italic>. Pellets were air dried at RT prior to resuspension in 10 &#956;L of dH2O. Samples for ChIP-seq were prepared according to the manufacturer&#8217;s protocol with Clontech or TruSeq adapters (Illumina). All samples were sequenced with HiSeq2500 (Illumina) at the NIDDK Genomics Core Facility by 50 bp single-end sequencing.</p><p>Total RNA was isolated from cells using TRizol reagent (Invitrogen) following the manufacturer's protocol. Reverse transcription of 0.5&#8211;1 &#956;g of total RNA was performed using oligo(dT) or random hexamer as primers and SuperScript III reverse transcriptase (Invitrogen) using the manufacturer's protocol. Transcript levels were quantified in the linear amplification range by real-time PCR using HotStart-IT SYBR green qPCR Master Mix (USB Corporation) by calibration to a standard curve of genomic DNA to account for differences in primer efficiencies.</p><p>Run-On was performed according to [<xref ref-type="bibr" rid="pgen.1007276.ref027">27</xref>] with minor modifications. Nuclei were prepared as follows: cells were centrifuged and washed twice with PBS. The pellet was resuspended in cell lysis buffer (10 mM Tris-HCl pH 7.4, 3 mM MgCl<sub>2</sub>, 10 mM NaCl, 150 mM sucrose, and 0.5% Nonidet P-40 (NP-40)), and a 5 min incubation on ice followed. Nuclei were then collected by centrifugation (4&#176;C, 500 x<italic>g</italic>) and gently washed with cell lysis buffer devoid of NP-40. After centrifugation, the pellet was resuspended in freezing buffer (50 mM Tris-HCl pH 8, 40% glycerol, 5 mM MgCl<sub>2</sub> and 0.1 mM EDTA). One volume of transcription buffer (200 mM KCl, 20 mM Tris-HCl pH 8, 5 mM MgCl<sub>2</sub>, 4 mM dithiothreitol (DTT), 4 mM each of rATP, rGTP and rCTP, 200 mM sucrose and 20% glycerol) was gently added to nuclei in ice, 8 &#956;L 10 mM biotin-16-UTP (Roche) was supplied to the mixture, which was incubated for 30 min at 29&#176;C. Reaction was stopped by adding 250 mM CaCl<sub>2</sub>, 6 &#956;L RNase-free DNase I (10 U/mL; Roche) and incubating for 15 min at 29&#176;C. RNA purification of nuclear Run-On RNA was performed with TRizol reagent (LifeTechnologies) according to manufacturer&#8217;s instructions. Dynabeads M-280 (50 &#956;L; Dynal, A.S., Oslo, Norway) resuspended in binding buffer (10 mM Tris-HCl, pH 7.5, 1 mM EDTA and 2 M NaCl) were mixed to an equal volume of Run-On RNA and incubated 20 min at 42&#176;C and 2 h at room temperature. Beads were separated by the magnetic apparatus and then washed for 15 min in 500 &#956;L 15% formamide and twice with 2X standard saline citrate (SSC), followed by a 5 min wash in 1 mL 2X SSC. Beads were then resuspended in DEPC-treated water and processed for qRT-PCR using random hexamers.</p><p>Crosslinking was performed by adding formaldehyde directly to the media at a final concentration of 1% and incubating for 10 min at RT. Reactions were quenched by adding glycine to a final concentration of 0.125 M and incubating for 5 min at RT. Reactions were incubated on ice 5 min followed by centrifugation at 1200 x<italic>g</italic> at 4&#176;C for 5 min. Cells were then washed with 5 mL of cold PBS and centrifuged at 1200 x<italic>g</italic> at 4&#176;C for 5 min. Lysis was performed by incubating cells in 1 mL of Lysis buffer (10 mM NaCl, 0.2% NP-40, 10 mM Tris pH8, 1 Mini Complete tablet (Roche) per 10 mL lysis buffer) at 37&#176;C for 20 min. Samples were then centrifuged at 4200 x<italic>g</italic> at 4&#176;C for 5 min, and the lysis step was repeated once. 800 &#956;L of Lysis buffer was added, and samples were incubated at 37&#176;C for 20 min. Samples were then centrifuged at 4200 x<italic>g</italic> at 4&#176;C for 5 min, and resuspended in 1 mL Lysis buffer and incubated another 20 min at 37&#176;C. Pellets from cell samples were then washed with 0.8 mL of digestion buffer [(0.2% NP-40, 1X NEBuffer 3 (NEB)] and centrifuged at 4200 x<italic>g</italic> at 4&#176;C for 5 min. Nuclei were resuspended in 1.6 mL of digestion buffer with SDS added to a final concentration of 0.1% and incubated at 65&#176;C for 30 min. Triton X-100 was added to a final concentration of 1% and samples incubated at 37&#176;C for 15 min. A 40 &#956;L aliquot of the sample was taken and used as the undigested control. The remaining sample was digested with 1600 U of <italic>HindIII</italic> (NEB) at 37&#176;C overnight. Samples were incubated 20 min at 65&#176;C to inactivate <italic>HindIII</italic>. A 40 &#956;L aliquot of the sample was taken here and used as the digested control. The remaining sample was then diluted to 4 mL with ligation buffer [final concentrations were 1% Triton X-100, 1X T4 DNA Ligase Reaction Buffer (NEB)] and incubated at 37&#176;C for 30 min. After the addition of 4800 U of T4 DNA Ligase (NEB), each sample was incubated at 16&#176;C overnight. Proteinase K was added to all samples including the controls. Samples were then incubated at 65&#176;C overnight to reverse crosslinking. After de-crosslinking, samples were combined with 1 vol of phenol/chloroform/isoamyl alcohol (25:24:1), vortexed 15 s, and centrifuged for 4 min at 10,000 x<italic>g</italic>. The top layer was transferred to a new tube, and the procedure was repeated using 1 vol chloroform. The top layer was collected and subsequently diluted with 1 vol of water. The sample was combined with 0.1 vol of 3M NaOAc pH 5.2 and 2.5 vol of 100% ethanol, supplemented with 2 &#956;L of Glycoblue (Ambion). After incubating 1 h at -80&#176;C, samples were centrifuged 20 min at 4&#176;C at 10,000 x<italic>g</italic>. Pellets were washed with 75% ethanol and centrifuged 5 min at 4&#176;C at 10,000 x<italic>g</italic>. Pellets were air dried at RT prior to resuspension in 350 &#956;L water. Loading adjustment was performed by SYBR green quantitative PCR to the <italic>blanks</italic> locus, and samples were adjusted accordingly before TaqMan quantitative PCR for 3C [<xref ref-type="bibr" rid="pgen.1007276.ref028">28</xref>]. The values from the qPCR were quantified relative to a standard curve generated using BACs covering the analyzed region (CH321-63B06, CH321-76H10, CH321-88N10, CH321-76C01). BACs were obtained from BACKPACK.</p><p>In brief, the first steps correspond exactly to the 3C protocol described above. After resuspending the final pellet with 350 &#956;L of dH2O, 300 &#956;L of the 3C library was digested with 5 &#956;L of DpnII (50 U, NEB) overnight at 37&#176;C. After the enzyme was inactivated, digested chromatin was re-ligated using 20 &#956;L of ligase (100 U, NEB) in a total volume of 8 mL ligation reaction. Finally purification of the library was carried out by adding 1 vol of phenol/chloroform/isoamyl alcohol (25:24:1), vortexed 15 s, and centrifuged for 4 min at 10,000 x<italic>g</italic>. The top layer was transferred to a new tube, and the procedure was repeated using 1 vol chloroform. The top layer was collected and precipitated with 0.1 vol of 3M NaOAc, pH 5.2 and 2.5 vol of 100% ethanol supplemented with 7 &#956;L of Glycoblue. Inverse PCR was carried out using primers containing a 6 nucleotide barcode and one universal forward primer to produce Illumina-4C-seq libraries. PCR amplification was carried out using Phusion High Fidelity DNA Polymerase (NEB) with 60&#8211;70 ng of DNA and 1 &#956;L of forward primer (10mM) and 1 &#956;L of reverse primer (10mM). Amplification program: 95&#176;C for 3 min, 30 cycles of 98&#176;C for 20 s, 50&#176;C for 1 min and 72&#176;C for 2 min; final elongation step was 72&#176;C for 5 min. After pooling together 3 PCR reactions, samples were purified using Agencourt AMPure XP beads (Beckman Coulter). Finally, 4C-PCR products were sequenced with HiSeq2500 (Illumina) at the NIDDK Genomics Core Facility by 50 bp single-end sequencing.</p><p>Cells were centrifuged 1 min at 2000 xg, washed twice in PBS, allowed to settle on poly-L-lysine-coated slides and fixed in 4% paraformaldehyde (PFA) in PBS at RT for 10 min. Cells were washed twice with PBS, permeabilized and blocked with 0.5% Triton X-100, 1% Bovine serum albumin, 2% goat normal serum in PBS for 1 h at RT. After blocking, cells were first incubated with primary antibody diluted in 0.1% Triton X-100 and 1% BSA in PBS for 1 h at RT and then washed 3 times with PBS. Afterwards, slides were incubated with secondary antibodies diluted in 0.1% Triton X-100 and 1% BSA in PBS for 1 h at RT and washed three times with PBS. Primary antibodies were rabbit anti-AGO2 1:1000 (Liu) or mouse anti-LaminB 1:1000 (ADL67.10, DSHB). Secondary goat antibodies labeled with AlexaFluor 488 or AlexaFluor 594 (Molecular Probes) were used at 1&#8758;1000. Finally, cells were stained in 1 &#956;g/mL DAPI (Molecular Probes) prepared in PBS and mounted using ProLong Gold (Life Technologies).</p><p>Cells were fixed in 4% PFA in PBS for 10 min, washed twice with PBS, permeabilized in 0.5% Saponin (Sigma Aldrich)/0.5% Triton X-100/PBS for 20 min at RT and incubated in 0.1 N HCl for 15 min at RT. Afterwards, cells were washed twice with 2X SSC and treated with RNase A (100 ug/mL) in 2x SSC 1 h at RT. After washing twice with 2X SSC, cells were kept in 50% formamide/2X SSC for at least 30 min at RT. BAC DNAs were purchased from BACPAC (CHORI: CH321-76H10 and CH321-28E07). DNA was used directly as a template in a nick translation reaction including fluorescently labeled dUTPs (ARES kit, Molecular Probes). A probe mix containing 200&#8211;400 ng of each fluorescently labeled probe, 100 &#956;g/mL yeast tRNA (Ambion), 100 &#956;g/mL salmon sperm in 10 &#956;L of hybridization buffer (10% dextran sulfate, 50% formamide, 2x SSC, 1% Tween 20) was first incubated 10 min at 80&#176;C and put on ice, then added to cells incubated at 78&#176;C for 3 min and left to hybridize at 37&#176;C overnight. Excess probe was washed three times with each: 1X SSC and 0.1x SSC at 42&#176;C for 5 min. Finally, cells were stained in 1 &#956;g/mL DAPI (Molecular Probes) prepared in PBS and mounted using ProLong Gold (Life Technologies).</p><p>Cells were imaged on a Zeiss LSM 780 confocal microscope using the 63X oil objective. Image stacks of 8&#8211;10 images at steps of 1 &#956;m were acquired. First, images from the same field of view and channel were maximally projected. Colocalization and radial measurements to nuclear periphery were performed using CellProfiler software and the pipelines for &#8220;colocalization&#8221; and &#8220;speckle&#8221; analysis, respectively. For radial analysis, nuclei were subdivided starting from the center in concentric rings using the DAPI channel and radial distance between objects (nucleus and probe, centroid distances), normalized to nucleus diameter, was calculated. The normalized radial position of the FISH signal was calculated at the spot center.</p><p>For neuRNA-seq and mRNA-seq, raw reads were trimmed with cutadapt [<xref ref-type="bibr" rid="pgen.1007276.ref029">29</xref>] v 1.10 to remove any adapters and poly-A tails while performing light quality trimming using parameters "&#8212;quality-cutoff = 20&#8212;minimum-length = 25&#8212;overlap = 10 -a AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA -a TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT -a AGATCGGAAGAGC". Trimmed reads were mapped to the primary chromosomes (2L, 2R, 3L, 3R, 4, X, Y, and mitochondrion_genome) of the FlyBase release 6.11 dm6 assembly with HISAT2 2.0.4 [<xref ref-type="bibr" rid="pgen.1007276.ref030">30</xref>] using default parameters. Reads were counted in annotated genes in FlyBase release 6.11 using featureCounts v1.5.0-p3 [<xref ref-type="bibr" rid="pgen.1007276.ref031">31</xref>]. Since neuRNA-seq enriches for nascent RNA and the libraries are stranded, we counted reads in full gene bodies using featureCounts parameters "-s 1 -t gene -g gene_id -f". For the unstranded mRNA-seq libraries, we counted reads in exons using parameters "-t exon -g gene_id".</p><p>For neuRNA-seq, counts tables were loaded into DESeq2 v1.10.1 [<xref ref-type="bibr" rid="pgen.1007276.ref032">32</xref>]. Counts tables from independent neuRNA-seq experiments (siAGO2/dsLaminB/mock; dsDcr-2/mock; rescue constructs) were independently imported and normalized using simple design "~treatment" and using otherwise default parameters. Differentially expressed genes were those with an adjusted p-value &lt; 0.1.</p><p>For mRNA-seq, the older libraries were originally analyzed with an older set of programs and algorithms (specifically DESeq v1 [<xref ref-type="bibr" rid="pgen.1007276.ref033">33</xref>] instead of DESeq2). A major difference is that DESeq2 shrinks the log2 fold change estimates for genes with low information (i.e., low number of counts or high variance across replicates). Notably, <italic>nht</italic> is expressed at very low levels in both WT and AGO2 mutant flies. DESeq v1 detects <italic>nht</italic> as statistically significantly up-regulated in <italic>AGO2</italic> mutants. The AGO2 catalytic-independent gene list, GO analysis and follow-up experiments were based on this original observation. However, we found that <italic>nht</italic> is not significantly differentially expressed using DESeq2. Since follow-up RT-PCR confirmed that <italic>nht</italic> is in fact up-regulated in AGO2 mutants, we conclude that DESeq2 gives a false negative for <italic>nht</italic> likely because of its low expression, consistent with the known conservative behavior of DESeq2 in calling differentially expressed genes especially at low expression levels.</p><p>We estimated gene expression by running Salmon v0.8.1 [<xref ref-type="bibr" rid="pgen.1007276.ref034">34</xref>] on an index built from the transcriptome FASTA file from FlyBase (Release 6.11) using a <italic>k</italic> of 25 bp for our data containing 50 bp reads, and using default parameters in addition to the parameter "-l SF". For each gene, the reported transcript-level expression estimates in transcripts per million (TPM) were summed to give a total TPM value for each gene. Gene TPMs were averaged within replicates of each experiment to give a single value for each treatment for each experiment.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genes up-regulated in <italic>AGO2</italic><sup><italic>51B</italic></sup> null mutant but not in <italic>AGO2</italic><sup><italic>V966M</italic></sup> catalytic mutant female larva mRNA-seq libraries were analyzed for Biological Process GO-term enrichment in FlyMine <ext-link ext-link-type="uri" ns0:href="http://www.flymine.org/">http://www.flymine.org</ext-link>, an integrated database for <italic>Drosophila</italic> genomics [<xref ref-type="bibr" rid="pgen.1007276.ref035">35</xref>]. The same gene can appear in more than 1 category. The actual source for GO analysis is the Gene Ontology Consortium <ext-link ext-link-type="uri" ns0:href="http://www.geneontology.org/">http://www.geneontology.org/</ext-link> [<xref ref-type="bibr" rid="pgen.1007276.ref036">36</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Trimming reads for adapters and light quality trimming was performed with cutadapt v1.10 using the parameters "&#8212;quality-cutoff 20 -a AGATCGGAAGAGC&#8212;minimum-length = 25&#8212;overlap = 10". Reads were aligned to the same reference genome described above for RNA-seq using bowtie2 v2.2.8 [<xref ref-type="bibr" rid="pgen.1007276.ref037">37</xref>] with default parameters. Multimapping reads were removed using the "view" program of samtools v1.3.1 [<xref ref-type="bibr" rid="pgen.1007276.ref038">38</xref>] with parameter "-q 20". The Picard tools v2.5.0 (<ext-link ext-link-type="uri" ns0:href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</ext-link>) program MarkDuplicates was used to remove PCR duplicates from mapped reads.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Peak calling methods depended on whether the binding of the protein is punctate (AGO2) or broad (Pol II). In both cases we first used the MACS2 v2.1.1.20160309 callpeak program [<xref ref-type="bibr" rid="pgen.1007276.ref039">39</xref>] and <ext-link ext-link-type="uri" ns0:href="https://github.com/taoliu/MACS">https://github.com/taoliu/MACS</ext-link>) to call peaks on each replicate independently. For both AGO2 and Pol II we used the non-default parameters "&#8212;gsize dm&#8212;bdg&#8212;SPMR". For Pol II we additionally included the parameter "&#8212;broad". We then ran the same peak-calling steps but on the pooled set of replicates for each target protein, by providing all input and IP replicates to MACS2. The signal bedGraph outputs from the pooled peak-calling runs were used in screenshots for figures.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For the punctate AGO2 peaks, we used the Irreproducible Discovery Rate (IDR) method using IDR v2.0.3 [<xref ref-type="bibr" rid="pgen.1007276.ref040">40</xref>], <ext-link ext-link-type="uri" ns0:href="https://github.com/nboley/idr">https://github.com/nboley/idr</ext-link>, and following recommendations from <ext-link ext-link-type="uri" ns0:href="https://sites.google.com/site/anshulkundaje/projects/idr">https://sites.google.com/site/anshulkundaje/projects/idr</ext-link>). Initial lenient peaks were called with MACS2 with the lenient parameters "-p 1e-3&#8212;nomodel&#8212;to-large" on four versions of the IP data: replicates, pooled pseudoreplicates (BAM files generated by pooling all replicates, shuffling the result, and then splitting back out an equal number of reads into separate pseudoreplicate BAM files), self-pseudoreplicates (shuffle-and-split each replicate individually) and pooled replicates. Pooled input was always used as the control for these peak-calling runs. On each pair of replicate, self-pseudoreplicate, and pooled-pseudoreplicate called peaks, we ran IDR with parameters "&#8212;use-best-multisummit-IDR&#8212;soft-idr-threshold 0.05&#8212;rank p.value&#8212;input-file-type narrowPeak&#8212;peak-list $POOLED" where "$POOLED" is the peaks called on all replicates pooled together. From the pooled peaks ranked by increasing p-value, we chose the final AGO2 peaks as the top N peaks were N was the number of peaks in the pooled-pseudoreplicate comparison passing an IDR threshold of 0.05, as recommended by the IDR authors.</p><p>The IDR method has not been tested on broad peaks, and as such the authors do not recommend using it in such cases. We therefore selected final peaks for Pol II as those peaks in the pooled peak-calling run that were also found in at least one of the individual replicates. Specifically, using BEDTools v2.25.0 and pybedtools v0.7.9 [<xref ref-type="bibr" rid="pgen.1007276.ref041">41</xref>,<xref ref-type="bibr" rid="pgen.1007276.ref042">42</xref>] we used "pooled.intersect([replicates], u = True).sort().merge()" where "pooled" points to the pooled peak calls and "[replicates]" is the list of individual peak calls for each replicate.</p><p>When comparing AGO2 ChIP-seq peaks to genes changed upon AGO2 depletion, we considered a gene bound by AGO2 if it had an AGO2 peak overlapping at least one of the gene TSSs by at least 1 bp. In all other comparisons, we considered a gene bound by a protein if the protein overlapped anywhere in the gene body by at least 1 bp.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used the 4c-ker algorithm [<xref ref-type="bibr" rid="pgen.1007276.ref017">17</xref>] to analyze 4C-seq data following the authors' recommendations documented in <ext-link ext-link-type="uri" ns0:href="https://github.com/rr1859/R.4Cker">https://github.com/rr1859/R.4Cker</ext-link>. The reduced genome was created around HindIII sites with the fragment size of 13 bp on either side. The primer sequence was ACCGTTTTTGACAACAGCAGCTGTA and was flanked by a HindIII site starting at genomic position chr2L:15318918 in the dm6 assembly. Results from the "near-bait" analysis mode (defined by 4c-ker to be 5 Mb on either side of the bait) are presented here, and used k = 5, and pval = 0.05. Post-processing of the 4c-ker results files were required to identify the direction of interaction changes (gain/loss). Specifically, the bedGraph signal output, which contains values for overlapping windows of different sizes, was intersected with each differential region for the each treatment and control replicate using BEDTools and pybedtools. The gain/loss fold change was calculated as the weighted average of treatment window signals divided by the weighted average of control window signals intersecting each differential region.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Chromatin colors from [<xref ref-type="bibr" rid="pgen.1007276.ref009">9</xref>] were downloaded from GEO accession GSE22069 and lifted over to the dm6 assembly using liftOver from the UCSC tools [<xref ref-type="bibr" rid="pgen.1007276.ref043">43</xref>]. TADs from [<xref ref-type="bibr" rid="pgen.1007276.ref008">8</xref>] were downloaded from <ext-link ext-link-type="uri" ns0:href="http://chorogenome.ie-freiburg.mpg.de/">http://chorogenome.ie-freiburg.mpg.de</ext-link> [<xref ref-type="bibr" rid="pgen.1007276.ref008">8</xref>] and lifted over to the dm6 assembly. LADs from [<xref ref-type="bibr" rid="pgen.1007276.ref011">11</xref>] were downloaded from supplemental data. Original LAD data were in the dm2 assembly, so we performed a 2-stage liftover first from dm2 to dm3 and then from dm3 to dm6.</p><p>All FETs used the implementation in the Python scipy package (scipy.stats.fisher_exact). When comparing intervals (peaks, TADs) with genes, we considered the entire gene body. Reported p-values are two-tailed. When multiple tests were performed (e.g., for heatmaps in <xref ref-type="fig" rid="pgen.1007276.g003">Fig 3</xref>), we applied the Benjamini-Hochberg multiple test correction (FDR) as implemented in the statsmodels Python package (statsmodels.stats.multitest.fdrcorrection). Where FDR is displayed in heatmaps, if the odds ratio was &lt;1 then the sign of the corresponding FDR was flipped such that negative values indicate depletion and positive values indicate enrichment. Where log-odds ratio is displayed in heatmaps, if the FDR was &gt; 0.05 then the log odds was set to zero.</p><p>For computing colocalization between sets of intervals, we used the Genomic Association Test v1.2.2 framework [<xref ref-type="bibr" rid="pgen.1007276.ref044">44</xref>] using the dm6 assembly as the domain or workspace and otherwise default parameters. In the heatmaps showing log2 fold change enrichment, any comparisons where the q-value was &lt;0.05 was reset to have a log2 fold change of zero, and any self-self comparisons were set to zero.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Data are available from the following GEO accessions: GSE101365 (ChIP-seq), GSE95844 (mRNA-seq), GSE95845 (neuRNA-seq), and GSE95847 (4C-seq). The following link contains the setup, code, and documentation to reproduce the analysis and generate all figures: <ext-link ext-link-type="uri" ns0:href="https://doi.org/10.6084/m9.figshare.5829318">https://doi.org/10.6084/m9.figshare.5829318</ext-link>.</p>