<p>All participants gave written informed consent for tissue donation through the Sample Collection Registry for Quality Control of Biological and Environmental Specimens and Assay Development and Testing protocol (registered with ClinicalTrials.gov, number NCT01087307, and approved by the NIH Institutional Review Board at the National Institute of Environmental Health Sciences (NIEHS), protocol # 10-E-0063). We obtained 4 mm diameter skin punch biopsies from the left and right lateral forearms and hips and whole blood from two Caucasian male donors, ages 58 and 62 (<xref ref-type="fig" rid="pgen.1006385.g001">Fig 1</xref>). Areas for biopsy were identified as free of moles, hair and previous scars. Lateral forearm was away from area that would be covered by a watch. Hip area was under area that would be covered by shorts but away from groin. The skin was sectioned into approximately 6 parts and the sections were allowed to adhere to a cell culture dish. Dulbecco&#8217;s modified eagle&#8217;s medium (Gibco 11965118) supplemented with 1X non-essential amino acids (ThermoFisher 11140&#8211;050), 10% Cosmic Calf Serum (Hyclone SH30087), 10% AmnioMax C-100 supplement (Gibco 12-556-015) and 100&#956;g/ml primocin (Invitrogen ant-pm-1) and the cultures were incubated at 37&#176;C in a 5% carbon dioxide containing incubator. Fibroblasts were selected for their ability to grow well in culture with high cloning efficiency (4&#8211;5%). They were identified based on their elongated shape and ability to grow adhered to the tissue culture dishes in the presence of serum-containing media (<xref ref-type="supplementary-material" rid="pgen.1006385.s001">S1 Fig</xref>). It took approximately 3&#8211;4 weeks for confluent fibroblast growth from the tissue fragments. The fibroblasts were harvested, diluted and plated to obtain single-cell derived clones. The clones were expanded in culture for, 5 to 6 additional passages (4&#8211;6 weeks), to obtain 10<sup>6</sup> cells and genomic DNA was extracted (DNeasy Blood and Tissue kit, Qiagen). Venous blood for DNA isolation was collected in five 8.5 ml PAXgene blood DNA tubes (PreAnalytiX/Qiagen: Valencia, CA).</p><p>We used the KAPA Hyper (KAPA Biosystems) kit for whole genome sequencing (WGS) library preparation. Size selection of constructed libraries was performed using Pippin prep (SAGE Science). DNA libraries were used to obtain Paired-End 100 (Rapid V2 chemistry) or 125 bp (V4 chemistry) reads from HiSeq 2500 sequencer (Illumina). Each DNA library was sequenced on 4 lanes to get an average coverage of 40X to 60X for whole genomes, and 100X to 200X for whole exomes (WES) (<xref ref-type="supplementary-material" rid="pgen.1006385.s013">S1 Table</xref>). For whole exome (37 Mb target size) sequencing, Nextera Rapid Capture Exome Kit (Illumina) was used to prepare the library prior to sequencing. The raw FastQ files were filtered to keep only read pairs with an average base quality score of 20 or more and were then aligned to the reference human genome (GRCh37) using BWA-MEM-0.7.10 [<xref ref-type="bibr" rid="pgen.1006385.ref064">64</xref>] using the default parameters and the resulting BAM files were merged. Duplicate reads were removed from the resulting BAM files using MarkDuplicates from Picard Tools. The BAM files were processed according to the Genome Analysis Toolkit (GATK) best practices pipeline [<xref ref-type="bibr" rid="pgen.1006385.ref039">39</xref>, <xref ref-type="bibr" rid="pgen.1006385.ref065">65</xref>]. The length of the genome with coverage greater than or equal to 10X was estimated using the BedTools genomeCoverageBed command [<xref ref-type="bibr" rid="pgen.1006385.ref066">66</xref>] (<xref ref-type="supplementary-material" rid="pgen.1006385.s013">S1 Table</xref>). The final BAM files and complete lists of somatic mutation calls in WGS and WES samples have been deposited into dbGAP approved study under accession number phs001182.v1.p1.</p><p>The published method, scTea [<xref ref-type="bibr" rid="pgen.1006385.ref067">67</xref>] was applied to detect insertions of retrotransposons (L1, Alu and SVA) in ten fibroblast clones. At least 5 supporting read pairs and the score higher than 0.6 were required. The germline non-reference insertions were filtered out using the blood genome from the same donor. 13 raw insertion candidates were manually inspected using the Integrative Genome Viewer, and two candidates remained as a final call set (<xref ref-type="supplementary-material" rid="pgen.1006385.s014">S2 Table</xref>). The clonality of each event was estimated by calculating the ratio of discordant reads pairs supporting the insertion (i.e., repeat-anchored mate read (RAM) counts) to the sum of the RAM counts and the number of concordant read pairs across the insertion breakpoint (i.e., those supporting the non-insertion allele). The top high confidence insertion candidate found in the clone D1-L-H (L1#1) resides in a genomic region with large tandem duplication in the germline genome thus having an elevated sequencing depth with poor read mappability whereas the other candidate found in the D2-R-F clone (L1#2) resides in a genomic region with the expected depth and good mappability. To more accurately estimate the clonality of the event like L1#1, we adjusted the number of concordant read pairs by the relative sequencing depth to the genomic average depth and recalculated the allele frequency with an assumption that the insertion allele exists in only one of the many duplicated genomic regions. This copy number adjusted clonality provides us an upper bound of the event clonality (<xref ref-type="supplementary-material" rid="pgen.1006385.s002">S2 Fig</xref>).</p><p>Large-scale clonal copy number changes in the clones were detected using the VarScan2 copy number tool on pileup files generated using Samtools mpileup command [<xref ref-type="bibr" rid="pgen.1006385.ref027">27</xref>, <xref ref-type="bibr" rid="pgen.1006385.ref068">68</xref>]. This tool detects changes in coverage in the DNA samples from the clones as compared to their corresponding blood sample. Circular binary segmentation was applied to the output using the DNAcopy package in R [<xref ref-type="bibr" rid="pgen.1006385.ref028">28</xref>, <xref ref-type="bibr" rid="pgen.1006385.ref069">69</xref>, <xref ref-type="bibr" rid="pgen.1006385.ref070">70</xref>] (<xref ref-type="supplementary-material" rid="pgen.1006385.s003">S3 Fig</xref>, <xref ref-type="supplementary-material" rid="pgen.1006385.s016">S4 Fig</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s015">S3 Table</xref>).</p><p>Somatic structural changes in the form of large deletions, inversions, duplications and translocations were detected using the software DELLY [<xref ref-type="bibr" rid="pgen.1006385.ref029">29</xref>] using the default settings (<xref ref-type="supplementary-material" rid="pgen.1006385.s003">S3 Fig</xref>, <xref ref-type="supplementary-material" rid="pgen.1006385.s016">S4 Fig and S4 Table</xref>). Alignments of blood DNA reads were used as the &#8220;matched normal&#8221; to detect clone-specific somatic events. Centromeric and telomeric regions were excluded from the analyses. For each sample a structural variation event was called if &gt; 30% of the reads in the region supported the call and there were 0 reads supporting the event in the corresponding blood sample (<xref ref-type="supplementary-material" rid="pgen.1006385.s016">S4 Table</xref>). The structural variations were further validated by PCR as described below and any variants detected in blood DNA were discarded (<xref ref-type="supplementary-material" rid="pgen.1006385.s005">S5 Fig</xref>).</p><p>Single nucleotide variations (SNVs) for WGS samples were analyzed using three mutation calling tools, the haplotype caller from GATK [<xref ref-type="bibr" rid="pgen.1006385.ref039">39</xref>], VarScan2 somatic mutation caller from Washington University, St. Louis [<xref ref-type="bibr" rid="pgen.1006385.ref027">27</xref>], and MuTect from the Broad Institute of Harvard and MIT [<xref ref-type="bibr" rid="pgen.1006385.ref040">40</xref>]. SNVs were called somatic if they were present in the clone and were not detected in the blood of the same donor. VarScan2 and MuTect were run using the alignment files from whole blood from a donor as the &#8220;normal&#8221; for each clone isolated from this donor. The SNV calls were limited to regions with a minimum of 10X coverage with at least 3 reads supporting the mutation call. The list of SNVs generated were filtered to remove calls that match known dbSNPs (version 138) and calls that fall within known simple repeat regions (UCSC Genome Browser, hg19 build). The consensus between the 3 callers was taken and filtered further for allele frequency to provide the final mutation calls. Calls with an allele frequency between 45% and 55% were denoted as heterozygous mutations, and those with an allele frequency &gt; 90% were denoted as homozygous mutations. Homozygous SNVs may arise in the genome as mutations in genomic regions with copy number = 1 (e.g. X chromosome or autosome region heterozygous for a deletion)or by loss of heterozygosity following a mutation in the autosomes. Mutations with allele frequencies not falling within these criteria were discarded. Since, the samples sequenced were clonal in nature, 61% to 73% of the SNVs called by the three mutation callers were contained within these allele frequencies (<xref ref-type="supplementary-material" rid="pgen.1006385.s006">S6 Fig</xref>, <xref ref-type="supplementary-material" rid="pgen.1006385.s018">S6 Table</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s019">S7 Table</xref>).</p><p>VarScan2 was also used to annotate loss of heterozygosity (LOH) events in the clones as compared to their respective blood samples. The LOH events in the clones were expected to have an allele frequency &gt; 90% and between 45% and 55% in the corresponding blood samples (<xref ref-type="supplementary-material" rid="pgen.1006385.s017">S5 Table</xref>).</p><p>To detect shared SNVs between 2 or more samples, the SNV calls by MuTect were compared for all samples from the 2 donors, respectively. Analyses of allele frequencies for mutation calls by all 3 callers demonstrated that MuTect detects the highest fraction of somatic SNVs. The resulting SNVs were filtered by allele frequency such that calls between 45% and 55% or alleles with frequency &gt; 90% in either of the shared samples were further validated as described below. Only 1 SNV was confirmed to be present in both clones from the left forearm in donor2, and 2 SNVs were validated to be identical between the clones from the left and right forearms of Donor2 (<xref ref-type="supplementary-material" rid="pgen.1006385.s007">S7 Fig</xref>).</p><p>Mutations were annotated for their presence within exons and for their functional consequences and potential roles in cancers, using the software Annovar [<xref ref-type="bibr" rid="pgen.1006385.ref071">71</xref>] and CRAVAT [<xref ref-type="bibr" rid="pgen.1006385.ref072">72</xref>] (<xref ref-type="supplementary-material" rid="pgen.1006385.s018">S6 Table</xref>).</p><p>The bulk fibroblasts that outgrew from the initial biopsies from all 4 sites (forearms and hips) for the 2 donors were used for deep sequencing of the exome with approximately 100-200X coverage (<xref ref-type="supplementary-material" rid="pgen.1006385.s013">S1 Table</xref>) and aligned to the human genome as described above. MuTect was used to call somatic SNVs in these samples. A large fraction (264/387) of the SNVs from the bulk cells were found to be identical to the SNVs detected by the three mutations callers in exomes of the respective clones (<xref ref-type="supplementary-material" rid="pgen.1006385.s018">S6 Table</xref>), thus further corroborating the SNV calls in the clones.</p><p>The bulk cells derived from the biopsy from the left forearm of donor 2 showed a bimodal distribution of variant allele frequencies with the 2 peaks at ~5&#8211;10% and ~35&#8211;45%. 79 of the 83 SNVs present in the exons of D2-L-F (left forearm, donor 2) were present in the bulk sample for the left forearm. 49 of these SNVs had allele frequencies &#8805; 40% (<xref ref-type="supplementary-material" rid="pgen.1006385.s008">S8 Fig</xref>, <xref ref-type="supplementary-material" rid="pgen.1006385.s018">S6 Table</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s020">S8 Table</xref>). These observations indicate that the biopsy segment for the left forearm for donor 2 primarily contained one clonal lineage from which the clone D2-L-F was derived. In the 7 other bulk samples, the majority of the SNVs called were between 5 and 15% (<xref ref-type="supplementary-material" rid="pgen.1006385.s008">S8 Fig</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s020">S8 Table</xref>). These allele frequencies can be explained if each biopsy segment contained a minimum of 10 clonal lineages. Based on this, we estimated the minimum expected exonic mutation loads in a given biopsy segment by multiplying the number of mutations detected in the exome of the clones derived from the bulk cells by 10 (<xref ref-type="supplementary-material" rid="pgen.1006385.s021">S9 Table</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The average length of the telomeric repeats was estimated using TelSeq [<xref ref-type="bibr" rid="pgen.1006385.ref073">73</xref>]. In newborns, the average length of the telomere restriction fragments (TRFs) has been estimated to be approximately 11 kb [<xref ref-type="bibr" rid="pgen.1006385.ref074">74</xref>, <xref ref-type="bibr" rid="pgen.1006385.ref075">75</xref>]. TRFs have been shown to contain 1 kb DNA between the restriction site and telomeric repeats. In order to compare the telomere lengths in our samples to newborns, we therefore added 1 kb to the lengths estimated by TelSeq. Fibroblasts have been shown to lose 50&#8211;100 bp of the telomeres every generation [<xref ref-type="bibr" rid="pgen.1006385.ref075">75</xref>]. Since, each clone rose from a single cell with at least 20 cell generations, we can assess that telomeres in the progenitor cell were 2 kb longer. This implies that the fibroblast cells that gave rise to the clones have 4&#8211;5 kb TRFs. Based on this we can calculate that the cells underwent a minimum of 60 to 70 divisions since the individuals were born. Since the diploid human genome is 6 x 10<sup>9</sup> nucleotides; we calculate the mutation rate per nucleotide in the genome per cell division as:
<disp-formula id="pgen.1006385.e001"><alternatives><graphic id="pgen.1006385.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006385.e001.jpg" /><ns1:math id="M1"><ns1:msub><ns1:mrow><ns1:mi>R</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>p</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>n</ns1:mi><ns1:mi>t</ns1:mi><ns1:mo>,</ns1:mo><ns1:mi>p</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>c</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>d</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:mi>M</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>l</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>d</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>C</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>l</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>d</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>s</ns1:mi><ns1:mo>&#215;</ns1:mo><ns1:mi>d</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>p</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>d</ns1:mi><ns1:mspace width="0.25em" /><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="0.25em" /><ns1:mo>(</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>t</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mfrac></ns1:math></alternatives></disp-formula></p><p>Because the number of prenatal cell divisions is unknown, this calculation represents the maximum estimate of the rate. Therefore, we also calculated mutation rate per genome per year as another characteristics of mutation accumulation (<xref ref-type="supplementary-material" rid="pgen.1006385.s022">S10 Table</xref>).</p><p>Primers to amplify regions containing somatic mutations and LOH events were designed using PrimerBlast such that approximately 500 to 1500 bp region spanning the mutated base would be amplified [<xref ref-type="bibr" rid="pgen.1006385.ref076">76</xref>]. 6 to 10 mutations in the exons of each clone were chosen for amplification and Sanger sequencing. The corresponding blood DNA was also analyzed to ascertain whether the mutations annotated as somatic were present in the blood (germline). 102 SNVs with potential functional consequences were chosen for validation, of these, 87 SNVs were validated as somatic. We were unable to PCR amplify and get high quality Sanger sequencing results for the remaining SNVs due to difficulty obtaining primers unique to the region being amplified. None of the somatic events were identified as germline (mutations in both clone and blood) (<xref ref-type="supplementary-material" rid="pgen.1006385.s018">S6 Table</xref>).</p><p>LOH events that were either present in protein coding regions, or were in regions with CNVs, were taken for validation by PCR and Sanger sequencing. Only 10 such events were validated due to constraints with the amount of DNA available and difficulties in obtaining primers specific to the genomic regions. The LOH event was considered somatic if it was identified as a heterozygous allele in blood DNA and as a homozygous mutant in the clone (<xref ref-type="supplementary-material" rid="pgen.1006385.s017">S5 Table</xref>).</p><p>Structural variants were validated by PCR using primers flanking the novel junction formed and Sanger sequencing the PCR product. The structural change was considered somatic if it was detected in the clone and not detected in the blood DNA (<xref ref-type="supplementary-material" rid="pgen.1006385.s017">S5 Fig</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s016">S4 Table</xref>). The plots depicting all somatic changes in the clones were generated using RCircos [<xref ref-type="bibr" rid="pgen.1006385.ref077">77</xref>].</p><p>The enrichment of UV-induced mutation signatures and mutation load in the clones were calculated similar to [<xref ref-type="bibr" rid="pgen.1006385.ref043">43</xref>, <xref ref-type="bibr" rid="pgen.1006385.ref044">44</xref>] We use the term mutation signature in reference to the characteristics of a given mutation including the mutated residue, the altered base and the nucleotides that surround the residue that occur more frequently than expected with random mutations in genomic DNA. The signatures analyzed in this paper are listed in <xref ref-type="table" rid="pgen.1006385.t002">Table 2</xref>.</p><p>The mutated residue is capitalized in the center of the motif. In the detailed signature motif, possible nucleotides for a position are show in brackets separated by |.</p><p>For UV-signatures, C&#8594;T changes in the context of the yCn motif where n is any base and y is pyrimidine (T or C), and T&#8594;C changes in the 5&#8217; T in the context of the nTt motif, were analyzed. Example of the calculation for enrichment of mutations within the nTt motif is presented below where, the context is derived from 41-nucleotides region containing the mutated residue in the center.</p><p>For each motif, the reverse complement was also used in the calculations. The use of nucleotide context immediately surrounding the mutation rather than the whole-genome helps to account for localized preference of mutagenesis stemming from small range scanning by mutagenic enzymes such as APOBEC, or the preference of lesion occurrence, lesion repair as well as other factors including epigenomic features influencing mutagenesis within localized genomic regions [<xref ref-type="bibr" rid="pgen.1006385.ref078">78</xref>, <xref ref-type="bibr" rid="pgen.1006385.ref079">79</xref>]. These calculations do not exclude any specific genomic region, but rather use the area in each sample, where the mutations actually happened. With large numbers of mutations this approach gives results similar to calculations using whole-genome context [<xref ref-type="bibr" rid="pgen.1006385.ref043">43</xref>].</p><p>To statistically determine the over-representation of the UV-signature mutations a one-sided Fisher&#8217;s exact test was performed. The ratio of the number of C &#8594; T (for yCn and nCg) or T &#8594;C (for nTt) changes that occurred within or out of their respective motifs were compared to the ratio of C or T bases present within or out of the motifs above. To correct P-values for multiple testing the Benjamini-Hochberg method was used.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The minimum estimate of the number of mutations attributable to UV for both nCy and nTt motifs were calculated only for samples with enrichment &gt; 1 also having corrected q-value for enrichment &lt; 0.05. For samples with enrichment <underline>&lt;</underline> 1 and for samples with enrichment q-value &gt; 0.05 the minimum estimate of mutation load was assigned the value = 0. For example, minimum mutation load for nTt&#8594;nCt was calculated as
<disp-formula id="pgen.1006385.e003"><alternatives><graphic id="pgen.1006385.e003g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006385.e003.jpg" /><ns1:math id="M3"><ns1:msub><ns1:mrow><ns1:mi>M</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>L</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>d</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>T</ns1:mi><ns1:mi>t</ns1:mi><ns1:mo>&#8594;</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>C</ns1:mi><ns1:mi>t</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>m</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>s</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>n</ns1:mi><ns1:mi>T</ns1:mi><ns1:mi>t</ns1:mi><ns1:mo>&#8594;</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>C</ns1:mi><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:msub><ns1:mo>&#215;</ns1:mo><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>E</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>h</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>n</ns1:mi><ns1:mi>T</ns1:mi><ns1:mi>t</ns1:mi><ns1:mo>&#8594;</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>C</ns1:mi><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mrow><ns1:msub><ns1:mrow><ns1:mi>E</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>h</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>n</ns1:mi><ns1:mi>T</ns1:mi><ns1:mi>t</ns1:mi><ns1:mo>&#8594;</ns1:mo><ns1:mi>n</ns1:mi><ns1:mi>C</ns1:mi><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:msub></ns1:mrow></ns1:mfrac></ns1:math></alternatives></disp-formula></p><p>Mutagenesis in CpG leading to nCg&#8594; nTg changes was evaluated in a similar way (<xref ref-type="fig" rid="pgen.1006385.g004">Fig 4A and 4B</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s023">S11 Table</xref>).</p><p>Total tandem CC&#8594;TT dinucleotide changes were calculated for each sample (<xref ref-type="fig" rid="pgen.1006385.g004">Fig 4C</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s023">S11 Table</xref>). This mutation signature is known to be unique to UV and the probability of random CC&#8594;TT changes not by UV is extremely low [<xref ref-type="bibr" rid="pgen.1006385.ref050">50</xref>].</p><p>In order to assess the contributions of the -1 nucleotide in the nCg and nTt mutation motifs, we also evaluated rCg&#8594;rTg, yCg&#8594;yTg, rTt&#8594;rCt and yTt&#8594;yCt mutation signatures as described above.</p><p>Replication timing for each 1Mb window of the genome was determined by taking the average of the wavelet-smoothed signal. The average replication timing was binned into 5 equal bins and the mean mutation density per Mb was calculated for each bin (<xref ref-type="fig" rid="pgen.1006385.g005">Fig 5A</xref>, <xref ref-type="supplementary-material" rid="pgen.1006385.s022">S10 Fig</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s024">S12 Table</xref>). The total number of DNase I hypersensitive peaks were calculated across 1Mb windows in the human genome. The data was binned into 5 equal bins and mean mutation density was calculated for each bin. BEDtools suite was used to map peaks to 1Mb intervals (<xref ref-type="fig" rid="pgen.1006385.g005">Fig 5A</xref>, <xref ref-type="supplementary-material" rid="pgen.1006385.s011">S11 Fig</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s024">S12 Table</xref>). Variation of UV-induced mutation density with replication timing and DNaseI hypersensitivity levels was also determined for yCn&#8594;yTn, nTt&#8594;nCt and CC&#8594;TT mutations (<xref ref-type="supplementary-material" rid="pgen.1006385.s012">S12 Fig</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s024">S12 Table</xref>).</p><p>To statistically determine the over-representation of mutations in pyrimidines due to UV in the non-transcribed strand of genes, the number of C&#8594;T and T&#8594;C mutations in the yCn and nTt contexts and CC&#8594;TT mutations were determined for both strands of transcribed genes, and a one sided binomial test was performed expectation that more mutations are present on the non-transcribed strand. To correct P-values for multiple testing the Benjamini-Hochberg method was used. Similar comparison was done for mutations in pyrimidines not conforming to UV-signatures (<xref ref-type="fig" rid="pgen.1006385.g005">Fig 5B</xref> and <xref ref-type="supplementary-material" rid="pgen.1006385.s025">S13 Table</xref>).</p><p>The R-code for analysis of all mutation signatures will be provided on request.</p>