<p>Written, informed consent was obtained for all participants in accordance with the Declaration of Helsinki. This project was also reviewed and approved by the Montreal Heart Institute Ethics Committee and the different recruiting centers (approval number 2014&#8211;1707).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Single variant association results considered in this effort were obtained from participants of European ancestry using an additive genetic model. All phenotypes were corrected for confounding factors (see below) and normalized using inverse normal transformation. The final analyses included samples from BCX1 (up to 157,622 participants), AIRWAVE (up to 14,866), and the first release of the UK Biobank (up to 136,084). Genotypes for BCX1 and AIRWAVE were obtained from the Illumina ExomeChip array; the content is available at: <ext-link ext-link-type="uri" ns0:href="http://genome.sph.umich.edu/wiki/Exome_Chip_Design">http://genome.sph.umich.edu/wiki/Exome_Chip_Design</ext-link>. The ExomeChip includes ~250,000 exonic variants obtained from whole-exome sequencing of ~12,000 participants. The UK Biobank samples were genotyped on the UK Biobank Axiom array; the content of the arrays is available at: <ext-link ext-link-type="uri" ns0:href="http://www.ukbiobank.ac.uk/scientists-3/uk-biobank-axiom-array/">http://www.ukbiobank.ac.uk/scientists-3/uk-biobank-axiom-array/</ext-link>. Whereas the Axiom array targets &gt;800,000 variants, we only analyzed exonic variants that overlap with the content of the ExomeChip.</p><p>Phenotypic information, and ExomeChip quality-control steps and association results from cohorts involved in the BCX1 Consortium and AIRWAVE have been described elsewhere [<xref ref-type="bibr" rid="pgen.1006925.ref010">10</xref>, <xref ref-type="bibr" rid="pgen.1006925.ref011">11</xref>, <xref ref-type="bibr" rid="pgen.1006925.ref043">43</xref>]. Briefly, we excluded variants with low genotyping success rate (&lt;95%, except for WHI that used a cutoff &lt;90%). Samples with call rate &lt;95% after joint or zCALL calling and with outlying heterozygosity rate were also excluded. Other exclusions were deviation from Hardy-Weinberg equilibrium (<italic>P</italic>&lt;1x10<sup>-6</sup>) and gender mismatch. We performed principal component analysis (PCA) or multidimensional scaling (MDS) and excluded sample outliers from the resulting plots, using populations from the 1000 Genomes Project to anchor these analyses. Prior to performing meta-analyses of the association results provided by each participating study, we ran the EasyQC protocol [<xref ref-type="bibr" rid="pgen.1006925.ref044">44</xref>] to check for population allele frequency deviations and proper trait transformation in each cohort. In terms of hematological phenotypes, we excluded individuals with blood cancer, leukemia, lymphoma, bone marrow transplant, congenital or hereditary anemia, HIV, end-stage kidney disease, dialysis, splenectomy, and cirrhosis, and those with extreme measurements of RBC traits. We also excluded individuals on erythropoietin treatment or chemotherapy. Additionally, we excluded pregnant women and individuals with acute medical illness at the time the complete blood count (CBC) was done. For all traits, within each study, we adjusted for age, age-squared, gender, the first 10 principal components, and, where applicable, other study-specific covariates such as study center using a linear regression model. Within each study, we then applied inverse normal transformation on the residuals and tested the phenotypes for association with the ExomeChip variants using either RVtests (version 20140416) [<xref ref-type="bibr" rid="pgen.1006925.ref045">45</xref>] or RAREMETALWORKER.0.4.9 [<xref ref-type="bibr" rid="pgen.1006925.ref046">46</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">A description of methods and quality-control procedures for the blood-cell data for the first release of the UK Biobank can also be found elsewhere [<xref ref-type="bibr" rid="pgen.1006925.ref001">1</xref>]. Description of the exome-component of the genotyping arrays used for the UK Biobank samples can be found at: <ext-link ext-link-type="uri" ns0:href="http://www.ukbiobank.ac.uk/scientists-3/uk-biobank-axiom-array/">http://www.ukbiobank.ac.uk/scientists-3/uk-biobank-axiom-array/</ext-link>. In the UK Biobank, we modelled blood-cell traits using the following covariates: age, sex, menopause status for women, assessment centre where the blood samples were collected, machine counter that processed the blood samples, month, day of the week, time inside the day that the samples were collected, self-reported ethnic background of the individuals, smoking status and smoking frequency, alcohol drinker status, alcohol intake frequency, height and weight. In the first release of the UK Biobank, we tested the association between directly genotyped or imputed variants and normalized hematological traits with BOLT-LMM [<xref ref-type="bibr" rid="pgen.1006925.ref047">47</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We meta-analyzed results from BCX1, AIRWAVE and the UK Biobank using inverse-variance weighting as implemented in METAL [<xref ref-type="bibr" rid="pgen.1006925.ref048">48</xref>]. We limited our analyses to variants with a mean minor allele frequency (MAF) &lt;1% in the meta-analyses that are annotated as coding (missense or nonsense) or splice site (acceptor or donor) using ENSEMBL&#8217;s Variant Effect Predictor (VEP). Furthermore, the variants had to be present on the Illumina exome array used by the BCX1 studies. In total, we tested 137,086 variants against 15 blood-cell traits. These phenotypes are divided between the main three cell types found in blood: red blood cells (red blood cell count (RBC count), hemoglobin concentration (HGB), hematocrit (HCT), mean corpuscular hemoglobin (MCH), mean corpuscular volume (MCV), mean corpuscular hemoglobin concentration (MCHC), and RBC distribution width (RDW)), white blood cells (total white blood cell count (WBC count), neutrophil count (Neutro), lymphocyte count (Lympho), monocyte count (Mono), basophil count (Baso), and eosinophil count (Eosin)), and platelets (platelet count (PLT count) and mean platelet volume (MPV)). The meta-analysis results are available at: <ext-link ext-link-type="uri" ns0:href="http://www.mhi-humangenetics.org/en/resources">http://www.mhi-humangenetics.org/en/resources</ext-link>. We used a conservative &#945; = 5x10<sup>-8</sup> to declare statistical significance. At this statistical threshold, our sample size (N = 308,572) provides &gt;99% power to discover variants with MAF = 0.1% that explain &gt;0.03% of the phenotypic variance.</p><p>To test whether the 31 rare variants newly identified in the meta-analyses are associated with blood-cell traits independently of other known genetic loci, we regressed out the effect of the known blood-cell variants previously identified in the first release of the UK Biobank [<xref ref-type="bibr" rid="pgen.1006925.ref001">1</xref>] from the normalized blood-cell phenotypes. All these analyses were done per phenotype; that is we fitted 15 different models for each of the 15 blood-cell phenotypes tested. We then re-tested in the UK Biobank (using linear regression implemented in R) the association between the &#8220;residual&#8221; blood-cell phenotypes and genotypes at the rare variants identified in the meta-analyses. For instance, for hemoglobin, we conditioned the new rare coding or splice site hemoglobin variants on all variants (across the genome) previously reported to associate with hemoglobin levels. We provide the complete list of variants on which we conditioned in <bold><xref ref-type="supplementary-material" rid="pgen.1006925.s006">S5 Table</xref></bold> per blood-cell trait. If the rare variants were not significantly associated with the residual phenotypes, we then ran pairwise conditional analyses to identify which previously known variants at the locus accounted for the association signal identified in this project. To declare statistical independence from previously reported hematological trait association signals, we used &#945; = 0.002 (Bonferroni correction for 31 variants tested).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We sought association of the <italic>PLG</italic> variant (rs145535174) with platelet reactivity, as well as hemostatic and coagulation factors in the Framingham Heart Study (FHS) [<xref ref-type="bibr" rid="pgen.1006925.ref049">49</xref>]. Genotyping for rs145535174 was conducted using the Illumina Human Exome BeadChip v.1.0 (Illumina, Inc., San Diego, CA) [<xref ref-type="bibr" rid="pgen.1006925.ref050">50</xref>]. Multiple measurements were assessed for platelet reactivity [<xref ref-type="bibr" rid="pgen.1006925.ref051">51</xref>], including maximum percentage platelet aggregation in response to agonists, <italic>i</italic>.<italic>e</italic>. ADP and epinephrine; minimal concentration of each agonist to produce a &gt;50% aggregation response (EC50); and lag time in response to collagen stimulation. Hemostatic factors and coagulation factors, including antigens of plasminogen activator inhibitor-1 (PAI-1), tissue plasminogen activator (tPA), D-dimer, clotting factor VII (FVII) and von Willebrand factor (VWF), were measured using enzyme-linked immunosorbent assays [<xref ref-type="bibr" rid="pgen.1006925.ref052">52</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006925.ref054">54</xref>] while fibrinogen levels were assessed using the Clauss method [<xref ref-type="bibr" rid="pgen.1006925.ref052">52</xref>]. Association analyses in the FHS were conducted using either RAREMETALWORKER (<ext-link ext-link-type="uri" ns0:href="http://genome.sph.umich.edu/wiki/RAREMETALWORKER">http://genome.sph.umich.edu/wiki/RAREMETALWORKER</ext-link>) or seqMeta (<ext-link ext-link-type="uri" ns0:href="http://cran.r-project.org/web/packages/seqMeta/index.html">http://cran.r-project.org/web/packages/seqMeta/index.html</ext-link>). A linear mixed effects model that accounts for familial correlation was used and adjustments were made for age, sex and principal components. The phenotypes were log-transformed or inverse normal transformed, as needed.</p><p>We tested the association between the rare G-allele at <italic>PLG</italic>-rs145535174 and myocardial infarction (MI) and stroke in the UK Biobank. We used the &#8220;Health and Medical History&#8221; records to identify MI and stroke cases. For MI, we used the search terms &#8220;heart attack&#8221;, &#8220;myocardial infarction&#8221;, &#8220;acute myocardial infarction&#8221;, &#8220;subsequent myocardial infarction&#8221; and &#8220;old myocardial infarction&#8221; to retrieve affected individuals. We used the terms &#8220;stroke&#8221;, &#8220;ischaemic stroke&#8221; and &#8220;cerebral infarction&#8221; to define stroke cases. As controls, we excluded UK Biobank participants with MI, stroke or transient ischemic attack, percutaneous coronary intervention, coronary artery bypass graft surgery, peripheral vascular disease, congestive heart failure, and angina. For analysis of venous thromboembolism (VTE), we identified cases in the UK Biobank, the Montreal Heart Institute Biobank, and the Women&#8217;s Health Initiative as individuals with pulmonary embolism or deep vein thrombosis. We tested the genetic association by logistic regression in PLINK or R, correcting for age, sex and the first ten principal components when available.</p><p>We identified asthma, allergic rhinitis (hay fever), and endometriosis cases using the detailed &#8220;Health and Medical History&#8221; UK Biobank participant records. All other individuals were assigned as controls. We tested the genetic association by logistic regression in R, correcting for age, sex and the first ten principal components. To determine if the rare <italic>IL33</italic>-rs146597587 variant is independent from the previously reported common SNPs at the locus, we conditioned on genotypes at these variants (rs343496, rs7032572, rs72699186, rs1342326, rs2381416, rs928413, rs10975519) and re-run the logistic regression model.</p><p>We used the default parameters in DAVID [<xref ref-type="bibr" rid="pgen.1006925.ref055">55</xref>] to perform biological term and pathway enrichment analyses. For these bioinformatic analyses, we used as reference set all genes with at least one rare coding or splice site variant tested in the meta-analyses. We retrieved genes associated with RBC, WBC, or PLT traits from <bold><xref ref-type="table" rid="pgen.1006925.t001">Table 1</xref></bold> (new independent variants from our study) and <bold><xref ref-type="supplementary-material" rid="pgen.1006925.s003">S2 Table</xref></bold> (known positive controls), and tested their enrichment in biological pathways in comparison with the reference set. Due to the relatively low number of genes that were used as input for this kind of analysis, we lowered the initial, minimum number of genes in a seeding group to 3 (default = 4) to ensure that the clustering algorithm will include as many genes as possible into functional groups. All other parameters were left at their default values.</p>