<p>Edinburgh Minimal Medium (EMM) was prepared as described [<xref ref-type="bibr" rid="pgen.1006775.ref117">117</xref>]. PM was identical to EMM except the nitrogen source was 3.74 gm/l monosodium glutamate. Minimal media were supplemented with 2 gm/l of a dropout mixture that contained equal weights of all amino acids and adenine was added to 2.5 times the weight of the other components [<xref ref-type="bibr" rid="pgen.1006775.ref029">29</xref>]. When indicated vitamin B1 was added to a final concentration of &#956;M and 5-Fluoroorotic acid (FOA) (U.S. Biologicals, Swampscott, MA.) was added to a final concentration of 1 mg/ml. When FOA is used in EMM the final concentration of uracil is lowered to 50 &#956;g/ml. The rich medium, yeast extract plus supplements (YES) contained 5 g/l Difco yeast extract, 30 g/l glucose, and 2 g/l dropout powder. When indicated the drug nourseothricin (Nat), (ClonNAT, Jena Bioscience, Germany) was added to a final concentration of 100&#956;g/ml.</p><p>The plasmids for this study are listed in <xref ref-type="supplementary-material" rid="pgen.1006775.s014">S6 Table</xref>. The plasmid pHL2882, used to measure transposition in the deletion strains, includes the <italic>nmt1</italic> promoter to express Tf1 with <italic>nat</italic> disrupted with an artificial intron (<italic>nat</italic>AI) (<xref ref-type="supplementary-material" rid="pgen.1006775.s001">S1 Fig</xref> and <xref ref-type="fig" rid="pgen.1006775.g001">Fig 1A</xref>). pHL2883 and pHL2884 were equivalent to pHL2882 except they have frame shift mutations in PR and IN, respectively. These plasmids were derived from pHL2803, which expressed Tf1 with a <italic>nat</italic> marker that lacks the AI. pHL2803 was constructed starting with pHL2673 by replacing the BsrGI-BamHI fragment containing IN sequence and <italic>neo</italic> with a BsrGI-BamHI fragment that was generated by fusion PCR to introduce restriction sites for AsiSI, SacII, and NotI just upstream of the polypurine tract. The primers for this fusion PCR and all other oligonucleotides are listed in <xref ref-type="supplementary-material" rid="pgen.1006775.s014">S6 Table</xref>. To complete pHL2803, <italic>nat</italic> was PCR amplified with primers containing AsiSI and NotI restriction sites and the product was inserted with <italic>nat</italic> in reverse orientation to Tf1 into pHL2673 with the AsiSI and NotI sites. To produce pHL2804 (PRfs) and pHL2805 (INfs), the AvrII-BsrGI fragments of Tf1 from pHL415-2 (PRfs) and pHL431-25 (INfs) were inserted into the AvrII-BsrGI backbone of pHL2803. pHL2882 was generated by inserting <italic>nat</italic>AI synthesized commercially by DNA 2.0 into the AsiSI and NotI sites of pHL2803 (<xref ref-type="supplementary-material" rid="pgen.1006775.s001">S1 Fig</xref>). The synthetic fragment contained the AI located after the 60<sup>th</sup> amino acid of Nat (<xref ref-type="supplementary-material" rid="pgen.1006775.s001">S1 Fig</xref>). The codon usage of the <italic>nat</italic> ORF was optimized for <italic>S</italic>. <italic>pombe</italic> without changing the amino acid sequence. pHL2883 and pHL2884 were created by inserting the BsrGI-BamHI fragment with <italic>nat</italic>AI from pHL2882 into the backbones of pHL2804 and pHL2805, respectively.</p><p>pHL2898, pHL2900, and pHL2902 express Tf1-<italic>neo</italic>AI from the <italic>nmt1</italic> promoter and encode the IN mutations D987N, D1047N, and E1083Q, respectively. These plasmids were made by replacing the BsrGI-NarI fragment of pHL449-1 with PCR fusion products of the BsrGI-NarI fragment containing the mutations. The primers for these PCRs are listed in <xref ref-type="supplementary-material" rid="pgen.1006775.s015">S7 Table</xref>.</p><p>The deletion library contained 3,004 haploid deletion strains from the V2 library of Bioneer (Alameda, CA, Cat. # M2030) [<xref ref-type="bibr" rid="pgen.1006775.ref034">34</xref>]. The deletions were derived from two haploid parents ED666 (<italic>h+ ade6-M210 ura4-D18 leu1-32</italic>) and ED668 (<italic>h+ ade6-M216 ura4-D18 leu1-32</italic>). These strains and others are listed in <xref ref-type="supplementary-material" rid="pgen.1006775.s016">S8 Table</xref>.</p><p>To transform pHL2882 in all 3,004 deletion strains, we modified previously published protocols [<xref ref-type="bibr" rid="pgen.1006775.ref118">118</xref>]. Using a sterile 96 pin multi-replicator (Model-VP408FS2AS-1, V&amp;P Scientific, Inc, San Diego, California,USA), each 96 well plate of the library was pined onto single well YES agar plates, and incubated at 32&#176;C for 72hrs (<xref ref-type="fig" rid="pgen.1006775.g002">Fig 2A</xref>). Each strain was inoculated with an initial OD<sub>600nm</sub> of 0.05 units in 5ml YES liquid media in 15ml tubes. All 96 deletion strains from each plate were independently transformed with pHL2882 (10&#956;g) and 5&#956;g of sonicated herring sperm DNA. Half of each culture was transformed with herring sperm DNA and no pHL2882 as a control for contamination. The transformed cells were processed as indicated in <xref ref-type="fig" rid="pgen.1006775.g002">Fig 2A</xref>. We isolated four independent transformants for each deletion strain.</p><p>Strains containing Tf1-<italic>nat</italic>AI (pHL2882) were grown as patches on agar plates with PM-U+L+B1. These patches were then replica printed on to agar plates with PM-U+L-B1 to induce the <italic>nmt1</italic> promoter. After 4 days of incubation, the patches were replica printed onto agar plates with EMM+U+L+B1+FOA twice in succession, the first print was incubated 3 days and the second for 2 days. The patches were then replica printed on to YES+Nat+FOA agar and incubated for 44hrs at 32&#176;C (<xref ref-type="fig" rid="pgen.1006775.g003">Fig 3</xref>). Each transposition plate contained patches of PRfs and INfs as controls and the growth of the deletion strains was scored relative to a set of standards (<xref ref-type="supplementary-material" rid="pgen.1006775.s002">S2 Fig</xref>). Four independent transformants of each deletion strain were assayed. An average transposition score was determined if all four transformants had scores within a window of three units. Outliers were excluded from the average if a single transformant had a difference in score three units or greater from the other three. If two transformants had scores that differed by three or more units from the other transformants, the score for the deletion was considered to be inconsistent and were excluded from the screen.</p><p>To measure amounts of Tf1-cDNA in the nucleus, we used a homologous recombination patch assay as described [<xref ref-type="bibr" rid="pgen.1006775.ref029">29</xref>, <xref ref-type="bibr" rid="pgen.1006775.ref036">36</xref>]. Deletion strains containing pHL2882 were grown on PM-U+L+B1 agar plates for 3 days at 32&#176;C. The patches were replica printed onto PM-U+L-B1 for induction. After 4 days of incubation, the patches were replica printed onto YES+Nat agar and incubated for 24hr at 32&#176;C. The patches were compared with the PRfs and INfs controls from the same plate and scored for homologous recombination using standards (<xref ref-type="supplementary-material" rid="pgen.1006775.s003">S3 Fig</xref>). Four independent transformants of each deletion strain were assayed. The adjusted average scores were determined as previously described in the transposition assay.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">Strains tested with the quantitative homologous recombination assay were grown on PM-U+L+B1 plates for 3 days at 32&#176;C (<xref ref-type="supplementary-material" rid="pgen.1006775.s004">S4 Fig</xref>). Cells were then suspended into 5ml of PM-U+L-B1 liquid media, and washed six times with 5 ml of PM-U+L-B1 liquid media to remove residual B1. Cells were then inoculated in 5ml of PM-U+L-B1 media at a starting OD<sub>600nm</sub> of 0.05 units. Following 4 days of incubation the cultures were diluted to OD<sub>600nm</sub> 1.0 (2x10<sup>7</sup> cells/ml) in PM-U+L+B1 medium and serially diluted from 2x10<sup>7</sup> cells/ml to 2 x10<sup>4</sup> cells/ml using PM-U+L+B1, then spread on YES and YES+Nat (100 &#956;g/ml) agar plates and grown for 3 days at 32&#176;C. Colonies were counted per plate, and the homologous recombination frequencies were determined with the following equation:
<disp-formula id="pgen.1006775.e001"><alternatives><graphic id="pgen.1006775.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006775.e001.jpg" /><ns1:math id="M1"><ns1:mi>Q</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>H</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>s</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>R</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>b</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>F</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>q</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>y</ns1:mi><ns1:mspace width="1pt" /><ns1:mo>=</ns1:mo><ns1:mspace width="1pt" /><ns1:mfrac><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>b</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>o</ns1:mi><ns1:mi>f</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>s</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>Y</ns1:mi><ns1:mi>E</ns1:mi><ns1:mi>S</ns1:mi><ns1:mo>+</ns1:mo><ns1:mi>N</ns1:mi><ns1:mi>A</ns1:mi><ns1:mi>T</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo><ns1:mspace width="1pt" /><ns1:mi mathvariant="normal">*</ns1:mi><ns1:mspace width="1pt" /><ns1:mn>100</ns1:mn></ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>n</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>b</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>o</ns1:mi><ns1:mi>f</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>c</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>s</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>Y</ns1:mi><ns1:mi>E</ns1:mi><ns1:mi>S</ns1:mi><ns1:mspace width="1pt" /><ns1:mi mathvariant="normal">*</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>d</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>f</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>r</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mfrac></ns1:math></alternatives></disp-formula></p><p>Recombination frequencies for wild-type Tf1-<italic>nat</italic>AI in wild-type strains without deletions ranged from 3% to 1.5% in individual experiments. Values for each deletion strain were normalized to wild-type strains assayed during the same experiment.</p><p>10 ml cultures were inoculated with a starting OD<sub>600nm</sub> of 0.05 units. After 18 hours, cells were washed with sterile deionized water. The cell pellets were suspended in 0.4ml of extraction buffer consisting of 15 mM KCl, 10 mM HEPES-KOH (pH 7.8), 5 mM EDTA, 5 mM dithiothreitol, protease inhibitor cocktail tablet (Complete, Roche Lifesciences), 2 mM phenylmethylsulfonylfluoride (PMSF), Pepstatin(0.7mg/ml, 1000x stock), leupeptin (0.5mg/ml 1000x stock), and Aprotinin (1.0mg/ml 1000x stock). An equal volume of acid-washed glass beads was added and vortexed in a bead beater for a total of 3 min in 30 sec intervals separated by 30 sec rest. 0.1 ml of extraction buffer was mixed into the extract, and the liquid was removed. Extracts were combined with 2X sample buffer and boiled. The samples were loaded onto an SDS&#8211;10% polyacrylamide gel. The gels were electrotransferred to Immobilon-FL membranes (Millipore). The production bleeds of 660 (anti-Gag) and 657(anti-IN) were used to probe Tf1-IN and Gag protein levels, and monoclonal anti-&#945;-Tubulin antibody (Sigma-Aldrich,USA) was used as a loading control on all immunoblot experiments. The anti-alpha tubulin, 660 (anti-Gag) and 657(anti-IN) were used with 1:5000, 1:10,000, and 1:5000, respectively. The fluorescently-tagged secondary mouse IR-Dye 700 and rabbit anti-body IR-Dye 800 (Rockland Immunochemicals Inc.Limerick, PA) were used with 1:20,000 dilutions. The Immobilon-FL membranes were scanned with an Odyssey infrared imaging system (LI-COR Biosciences). Fluorescence levels from antibodies specific for Gag and IN were normalized to amounts of tubulin and measured with a Li-COR digital instrument (<xref ref-type="sec" rid="sec012">Materials and methods</xref>) (<xref ref-type="supplementary-material" rid="pgen.1006775.s011">S3 Table</xref>).</p><p>For the deletion strains tested two independent transformants were assayed for Tf1-IN and Gag protein levels. Geometric means of Gag and IN levels of these replica pairs of deletion strains were compared to the geometric means of the wild-type strains lacking the deletion.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">The Tf1-IN and Gag protein expression levels were measured and normalized to alpha tubulin. The fold change in Tf1-IN and Gag protein expression were calculated using below equation:
<disp-formula id="pgen.1006775.e002"><alternatives><graphic id="pgen.1006775.e002g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006775.e002.jpg" /><ns1:math id="M2"><ns1:mi>C</ns1:mi><ns1:mi>h</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>I</ns1:mi><ns1:mi>N</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>d</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>G</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>l</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>s</ns1:mi><ns1:mspace width="1pt" /><ns1:mo>=</ns1:mo><ns1:mspace width="1pt" /><ns1:mfrac><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>z</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>d</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>c</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>m</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>o</ns1:mi><ns1:mi>f</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>I</ns1:mi><ns1:mi>N</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>d</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>G</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>p</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>m</ns1:mi><ns1:mi>u</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>t</ns1:mi></ns1:mrow><ns1:mrow><ns1:mi>N</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>z</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>d</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>g</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>m</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>c</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>m</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>o</ns1:mi><ns1:mi>f</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>I</ns1:mi><ns1:mi>N</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>d</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>G</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>g</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>w</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>l</ns1:mi><ns1:mi>d</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>t</ns1:mi><ns1:mi>y</ns1:mi><ns1:mi>p</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="1pt" /><ns1:mi>s</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi></ns1:mrow></ns1:mfrac></ns1:math></alternatives></disp-formula></p><p>Tf1 was expressed by incubating the cells for 2 days in 50 ml of EMM &#8211;B1 starting at OD600 = 0.05 to induce the nmt1 promoter after washing them 4 times in EMM &#8211;B1. Genomic DNAs were isolated from 200 OD units of the resulting cultures. Southern blots were performed as described previously [<xref ref-type="bibr" rid="pgen.1006775.ref036">36</xref>, <xref ref-type="bibr" rid="pgen.1006775.ref047">47</xref>] with the following modifications. The nat probe was produced by digesting 5&#956;g of pHL2597 with 160 units of EcoRI, isolating the 1.2 kb fragment from a 0.7% agarose gel and random-priming labelling with <sup>32</sup>P-CTP. One microgram of gDNAs were digested with 40 units of BsrGI, separated on a 1.0% agarose gel and transferred to a nylon membrane. The blot was hybridized with the nat probe. BsrGI digestion resulted in Tf1 cDNAs being detected at 2.8 kb and the Tf1 expression plasmid at 14kb. Tf1 cDNA was quantified with phosphoimaging and normalized to the amount of expression plasmid. Briefly, the <sup>32</sup>P-signal was detected by phosphoimaging on a Typhoon FLA-9500. The relative level of cDNA was determined by normalizing the signal intensity of the 2.9kb cDNA band to the signal intensity of the 14kb plasmid band.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Tf1 transposition was induced in strains containing Tf1-natAI (pHL2882) and deleted for <italic>pht1</italic>, <italic>rad51</italic>, <italic>set1</italic>, <italic>cwf12</italic>, <italic>snf5</italic>, <italic>nup61</italic>, <italic>rad50</italic> or wild-type (<xref ref-type="supplementary-material" rid="pgen.1006775.s016">S8 Table</xref>) as described previously but with some modifications [<xref ref-type="bibr" rid="pgen.1006775.ref031">31</xref>]. Briefly, cells were washed 4 times in EMM &#8211;B1 before being inoculated at OD600 = 0.05 in EMM &#8211;B1 to induce the <italic>nmt1</italic> promoter, then grown 4 days for each of two passages. Cells with transposition events were selected by incubating them in 50 ml of EMM+B1+FOA for 4 days followed by 4 days in YES+FOA+Nat. Genomic DNAs were isolated from 200 OD units of the resulting cultures. Libraries were prepared for Illumina sequencing according to Chatterjee et al. [<xref ref-type="bibr" rid="pgen.1006775.ref031">31</xref>] and sequenced on a MiSeq System (Illumina) with custom primers. The sequence of linker oligonucleotides and primers used are given in <xref ref-type="supplementary-material" rid="pgen.1006775.s015">S7 Table</xref>. To determine the genome-wide integration profiles raw sequence reads were processed through a custom suite of Perl scripts [<xref ref-type="bibr" rid="pgen.1006775.ref030">30</xref>] modified to accommodate sequences of Tf1-<italic>nat</italic>AI and Illumina technology. Maps of integration relative to ORFs and Sap1 motifs were performed according to previous work [<xref ref-type="bibr" rid="pgen.1006775.ref032">32</xref>]. Density plots were obtained using the R function densCol from the package grDevices [<xref ref-type="bibr" rid="pgen.1006775.ref119">119</xref>]. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL <ext-link ext-link-type="uri" ns0:href="https://www.R-project.org/">https://www.R-project.org/</ext-link>). The sequence data can be obtained from the SRA database with the accession SRA Study: SRP100942.</p><p>The Biological Process slim terms optimized for <italic>S</italic>. <italic>pombe</italic> were applied to genes important for late stages of transposition of Tf1, Ty1 and Ty3. Term enrichments were calculated against the list of non-essential genes available from the Bioneer <italic>S</italic>. <italic>pombe</italic> deletion library and the <italic>S</italic>. <italic>cerevisiae</italic> ORF deletion collection in strain BY4741 from Invitrogen MapPairs. The p-values were calculated using hypergeometric distance and corrected for multiple comparison with false discovery rate.</p><p>Full-length Tf1 IN was fused to the C-terminus of a truncated DNA binding domain of LexA by ligating the IN sequence into the EcoRI and SalI sites of pSH2-1 [<xref ref-type="bibr" rid="pgen.1006775.ref120">120</xref>]. Full-length Tf1 IN was also fused to the C-terminus of the Gal4 activation domain by ligating IN sequence into the XhoI site of pACT [<xref ref-type="bibr" rid="pgen.1006775.ref121">121</xref>]. The host strain used in the two-hybrid screen was <italic>S</italic>. <italic>cerevisiae</italic> strain CTY10-5d (<italic>MAT&#945; ade2 trp1-901 leu2-3</italic>,<italic>112 his3-200 gal4 gal80 URA3</italic>::<italic>lexAop-lacZ ura3-52</italic>) [<xref ref-type="bibr" rid="pgen.1006775.ref122">122</xref>]. The two-hybrid assays detected production of lacZ by lifting colonies to 3MM nitrocellulose filter (Whatman) that was then stored at -80&#176;C overnight. The filters were thawed and at room temperature tested for galactosidase activity using X-gal [<xref ref-type="bibr" rid="pgen.1006775.ref122">122</xref>]. The sequences of Cwf3 (amino acids 3&#8211;284) and Rhp18 (amino acids 16&#8211;308) were inserted into pACT.</p>