<p>Our previous POInT analyses in yeast were based on human curated datasets [<xref ref-type="bibr" rid="pgen.1007267.ref019">19</xref>, <xref ref-type="bibr" rid="pgen.1007267.ref049">49</xref>]. We do not have such inferences for either the At&#8211;&#945; or the grass &#961; event. Instead, using experience from previous projects [<xref ref-type="bibr" rid="pgen.1007267.ref040">40</xref>, <xref ref-type="bibr" rid="pgen.1007267.ref050">50</xref>], we developed a new pipeline for inferring the paralogous genomic regions created by a WGD in the genomes sharing that event. We then merged these regions of DCS [<xref ref-type="bibr" rid="pgen.1007267.ref020">20</xref>, <xref ref-type="bibr" rid="pgen.1007267.ref021">21</xref>] across all polyploid genomes and sought an ancestral gene order that minimized the number of synteny breaks. <xref ref-type="fig" rid="pgen.1007267.g001">Fig 1</xref> shows examples of such DCS blocks for At-&#945;.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The goal of the pipeline is to find a common set of DCS blocks shared by the genomes of the six Brassicaceae species that possess At-&#945;: <italic>Arabidopsis thaliana</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref051">51</xref>], <italic>Arabidopsis lyrata</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref052">52</xref>], <italic>Capsella rubella</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref053">53</xref>], <italic>Shrenkiella parvula</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref054">54</xref>], formerly known as <italic>Thellungiella parvula</italic> or erroneously as <italic>Thellungiella halophila</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref055">55</xref>], <italic>Eutrema salsugineum</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref056">56</xref>], and <italic>Aethionema arabicum</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref057">57</xref>] and for the four grasses with &#961;: <italic>Brachypodium distachyon</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref058">58</xref>], <italic>Oropetium thomaeum</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref059">59</xref>], <italic>Setaria italica</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref060">60</xref>] and <italic>Sorghum bicolor</italic> [<xref ref-type="bibr" rid="pgen.1007267.ref061">61</xref>]. To do so, we used outgroup genomes that lacked the WGD in question. For the At-&#945; event, we used the draft genome of the outgroup plant <italic>Cleome violacea</italic>, which split from the six taxa studied prior to that event [<xref ref-type="bibr" rid="pgen.1007267.ref011">11</xref>]: it likewise lacks the WGD found in other taxa in the Cleomaceae [<xref ref-type="bibr" rid="pgen.1007267.ref009">9</xref>]. The <italic>C</italic>. <italic>violacea</italic> genome is available from the CoGe comparative genomics portal (<ext-link ext-link-type="uri" ns0:href="https://genomevolution.org/coge/">https://genomevolution.org/coge/</ext-link>) under accession number 23822. For the grass &#961; event, we used the genome of the pineapple <italic>Ananas comosus</italic> as an outgroup [<xref ref-type="bibr" rid="pgen.1007267.ref062">62</xref>]. CoGe accession numbers for all plant genomes used are listed in <xref ref-type="supplementary-material" rid="pgen.1007267.s009">S1 Data</xref>.</p><p>The product of a WGD is a set of duplicated genes in a genome that each originate from a single ancestral gene. Here, the <italic>C</italic>. <italic>violacea</italic> and pineapple genomes give us an estimate of these ancestral loci, and we seek to place either one (e.g., a duplicate loss has happened) or two genes (the ohnologs survive) from the duplicated genome in a &#8220;pillar&#8221; with each such ancestral gene (see <xref ref-type="fig" rid="pgen.1007267.g001">Fig 1</xref>). Genome annotation files for these 12 plant genomes were obtained from CoGe [<xref ref-type="bibr" rid="pgen.1007267.ref063">63</xref>]. With these data in hand, the inference of the shared DCS blocks that serve as POInT&#8217;s input is a three step process: 1) a homology search of each polyploid genome against the diploid outgroup, 2) inference of species-specific DCS blocks and 3) inference of a common set of DCS blocks across all genomes along with an estimate of their ancestral order at the time of the polyploidy.</p><p><underline>Step 1: Homology search</underline>. For At-&#945;, we used a fast homology search program based on the SeqAn package [<xref ref-type="bibr" rid="pgen.1007267.ref064">64</xref>, <xref ref-type="bibr" rid="pgen.1007267.ref065">65</xref>] to identify pairs of homologous genes, one from a genome with At-&#945; and one from <italic>C</italic>. <italic>violacea</italic>. We defined a pair of genes as being homologous for the purposes of DCS inference if their protein sequences: 1) share two 7 amino acid residue exact matches, 2) have the shorter sequence having 80% of the length of the longer, and 3) show 70% amino acid identity overall. Because of the greater evolutionary distances involved in the grass &#961; event, we used a slower but more sensitive BLAST-based search, employing our tool GenomeHistory to do so [<xref ref-type="bibr" rid="pgen.1007267.ref066">66</xref>, <xref ref-type="bibr" rid="pgen.1007267.ref067">67</xref>]. In this case, we required a maximal BLAST E-value of 10<sup>&#8722;8</sup> to identify matches between the four duplicated grasses and pineapple: we then used the same 70% identity and 80% aligned length cutoffs as used with At-&#945; to select homologs.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML"><underline>Step 2: Genome-specific DCS inference</underline>. Sequence homology alone is insufficient to identify the DCS blocks given the angiosperms&#8217; history of nested polyploidy [<xref ref-type="bibr" rid="pgen.1007267.ref001">1</xref>]. Instead, for the second step of the pipeline, we used gene order information (synteny) to identify which of the potentially many homologs in each polyploid genome are the WGD-produced ohnologs. We frame this problem as follows. First, we define a set <italic>A</italic> of <italic>n</italic> DCS blocks that consists of ancestral pillars <italic>A</italic><sub><italic>i</italic></sub> such that <italic>A</italic><sub><italic>i</italic></sub> &#8712; <italic>A</italic>|1 &#8804; <italic>i</italic> &#8804; <italic>n</italic>. Each pillar is linked to a unique gene from <italic>C</italic>. <italic>violacea</italic> or pineapple and has elements <italic>A</italic><sub><italic>i</italic></sub><italic>(p</italic><sub><italic>1</italic></sub><italic>)</italic> and <italic>A</italic><sub><italic>i</italic></sub><italic>(p</italic><sub><italic>2</italic></sub><italic>)</italic>, which represent the potential homologous genes created by WGD. Each pillar <italic>A</italic><sub><italic>i</italic></sub> also has associated a set of genes {<italic>h</italic><sub><italic>1</italic></sub><italic>&#8230;h</italic><sub><italic>h</italic></sub>} from the polyploid genome that are homologous to the pillar&#8217;s ancestral gene. A maximum of two of these homologs can be assigned to <italic>A</italic><sub><italic>i</italic></sub><italic>(p</italic><sub><italic>1</italic></sub><italic>)</italic> and <italic>A</italic><sub><italic>i</italic></sub><italic>(p</italic><sub><italic>2</italic></sub><italic>)</italic>. We next define <italic>O(A</italic><sub><italic>1</italic></sub><italic>&#8230;A</italic><sub><italic>n</italic></sub><italic>)</italic> to be the order of the pillars in <italic>A</italic> for our analysis. Hence, <italic>A</italic><sub><italic>O(i)</italic></sub> represents the <italic>i</italic><sup><italic>th</italic></sup> pillar in this ordering. For a given <italic>A</italic><sub><italic>O</italic>(<italic>i</italic>)</sub> (<italic>p</italic><sub><italic>k</italic></sub>)|1 &#8804; <italic>k</italic> &#8804; 2, define <italic>A</italic><sub><italic>O</italic>(<italic>i</italic>+<italic>j</italic>)</sub> (<italic>p</italic><sub><italic>k</italic></sub>) such that <italic>j</italic> = min(<italic>x</italic>; <italic>i</italic>+1&#8804;<italic>x</italic>&#8804;<italic>n</italic>) where <italic>A</italic><sub><italic>O</italic>(<italic>i</italic>+<italic>x</italic>)</sub> (<italic>p</italic><sub><italic>k</italic></sub>) &#8800; &#8709;: in other words, <italic>i+j</italic> is the next pillar after <italic>i</italic> in <italic>O(A</italic><sub><italic>1</italic></sub><italic>&#8230;A</italic><sub><italic>n</italic></sub><italic>)</italic> with an assigned gene for parental genome <italic>k</italic>. We define the score <italic>s</italic> of such a combination of homolog assignments and pillar orders:
<disp-formula id="pgen.1007267.e001"><alternatives><graphic id="pgen.1007267.e001g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1007267.e001.jpg" /><ns1:math id="M1"><ns1:mrow><ns1:mi>s</ns1:mi><ns1:mo>=</ns1:mo><ns1:munderover><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mi>n</ns1:mi></ns1:munderover><ns1:munderover><ns1:mstyle displaystyle="true" mathsize="140%"><ns1:mo>&#8721;</ns1:mo></ns1:mstyle><ns1:mrow><ns1:mi>k</ns1:mi><ns1:mo>=</ns1:mo><ns1:mn>1</ns1:mn></ns1:mrow><ns1:mn>2</ns1:mn></ns1:munderover><ns1:mtable><ns1:mtr><ns1:mtd><ns1:mn>1</ns1:mn></ns1:mtd></ns1:mtr><ns1:mtr><ns1:mtd><ns1:mn>0</ns1:mn></ns1:mtd></ns1:mtr></ns1:mtable><ns1:mo>|</ns1:mo><ns1:mtable><ns1:mtr><ns1:mtd><ns1:mrow><ns1:msub><ns1:mi>A</ns1:mi><ns1:mrow><ns1:mi>O</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mi>i</ns1:mi><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>p</ns1:mi><ns1:mi>k</ns1:mi></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mspace width="4pt" /><ns1:mi>a</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>d</ns1:mi><ns1:mspace width="4pt" /><ns1:msub><ns1:mi>A</ns1:mi><ns1:mrow><ns1:mi>O</ns1:mi><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mo>+</ns1:mo><ns1:mi>j</ns1:mi></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow></ns1:msub><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:msub><ns1:mi>p</ns1:mi><ns1:mi>k</ns1:mi></ns1:msub></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow><ns1:mspace width="4pt" /><ns1:mi>a</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>e</ns1:mi><ns1:mspace width="4pt" /><ns1:mi>n</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>g</ns1:mi><ns1:mi>h</ns1:mi><ns1:mi>b</ns1:mi><ns1:mi>o</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>s</ns1:mi></ns1:mrow></ns1:mtd></ns1:mtr><ns1:mtr><ns1:mtd><ns1:mrow><ns1:mi>o</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>h</ns1:mi><ns1:mi>e</ns1:mi><ns1:mi>r</ns1:mi><ns1:mi>w</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>s</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow></ns1:mtd></ns1:mtr></ns1:mtable></ns1:mrow></ns1:math></alternatives><label>(1)</label></disp-formula></p><p>In other words, the score is the sum of the number of positions in <italic>O(A</italic><sub><italic>1</italic></sub><italic>&#8230;A</italic><sub><italic>n</italic></sub><italic>)</italic> where the genes in each pillar are the genomic neighbors of the genes in the next non-empty position. We cannot simply use the pillar order seen in the outgroup, because neither <italic>C</italic>. <italic>violacea</italic> nor pineapple is the true ancestor of the WGD events in question: both have evolved independently for many millions of years. Instead we must optimize <italic>O(A</italic><sub><italic>1</italic></sub><italic>&#8230;A</italic><sub><italic>n</italic></sub><italic>)</italic>. Note that, throughout this pipeline, neighbor is understood to exclude any genes that are not part of the current analysis set. For instance, a gene in <italic>Arabidopsis thaliana</italic> with no identified <italic>C</italic>. <italic>violacea</italic> homolog is ignored in the neighbor computation because it could never appear in an ancestral pillar. By the same logic, any position for which <italic>A</italic><sub><italic>O</italic>(<italic>i</italic>)</sub> (<italic>p</italic><sub><italic>k</italic></sub>) and <italic>A</italic><sub><italic>O</italic>(<italic>i</italic>+<italic>j</italic>)</sub> (<italic>p</italic><sub><italic>k</italic></sub>) are not neighbors is defined as a synteny break, and, if this situation is true for both <italic>k = 1</italic> and <italic>k = 2</italic>, we refer to position <italic>i</italic> as having a double synteny break.</p><p>To infer the combination of the homolog assignments <italic>A</italic><sub><italic>i</italic></sub> (<italic>p</italic><sub><italic>k</italic></sub>) | 1 &#8804; <italic>i</italic> &#8804; <italic>n</italic>, 1 &#8804; <italic>k</italic> &#8804; 2 and the ordering <italic>O(A</italic><sub><italic>1</italic></sub><italic>&#8230;A</italic><sub><italic>n</italic></sub><italic>)</italic>, we used simulated annealing [<xref ref-type="bibr" rid="pgen.1007267.ref068">68</xref>, <xref ref-type="bibr" rid="pgen.1007267.ref069">69</xref>]. This algorithm proposes random changes to either <italic>O(A</italic><sub><italic>1</italic></sub><italic>&#8230;A</italic><sub><italic>n</italic></sub><italic>)</italic> or to the <italic>A</italic><sub><italic>i</italic></sub><italic>(p</italic><sub><italic>k</italic></sub><italic>)</italic> assignments with the goal of maximizing <italic>s</italic>, which recomputed after each such change. We used the extant <italic>C</italic>. <italic>violacea</italic> and pineapple gene orders as our initial orders and made increasingly long runs until longer run times no longer produced meaningfully higher values of <italic>s</italic>.</p><p><italic>A</italic>. <italic>thaliana</italic> and its relatives share a history of WGD [<xref ref-type="bibr" rid="pgen.1007267.ref026">26</xref>]: prior to the WGD-&#945; event modeled here there was another WGD, termed WGD-&#946;, which is shared with <italic>C</italic>. <italic>violacea</italic>. One might wonder if our simulated annealing algorithm has mistaken synteny blocks surviving from WGD-&#946; for the more recent products of WGD-&#945;. We suspect that any such errors are quite rare for two reasons. First, <italic>C</italic>. <italic>violacea</italic> also experienced WGD-&#946; and hence also possesses the corresponding synteny blocks, meaning that they are accounted for in the inputs to our simulated annealing routines. Second, we only considered homology relationships between genes in <italic>C</italic>. <italic>violacea</italic> and in <italic>A</italic>. <italic>thaliana</italic>, <italic>A</italic>.<italic>lyrata</italic>, <italic>C</italic>. <italic>rubella</italic>, <italic>S</italic>. <italic>parvula</italic> and <italic>E</italic>. <italic>salsugineum</italic> with nonsynonymous divergence (K<sub>a</sub>) less than 0.1 and between <italic>C</italic>. <italic>violacea</italic> and <italic>A</italic>. <italic>arabicum</italic> with K<sub>a</sub>&#8804;0.2. As a result, between 41% and 45% of the genes from <italic>C</italic>. <italic>violacea</italic> have only a single homolog identified in the other 6 genomes and hence cannot represent ambiguous surviving blocks from WGD-&#946; in <italic>C</italic>. <italic>violacea</italic>. Hence, it is difficult to see how ancestral WGD-&#946; blocks would have infiltrated our inferences in significant numbers.</p><p><underline>Step 3: Inferring a global ancestral ordering for POInT analyses</underline>. Using the four/six individually optimized set of ancestral pillars (for &#961; and At-&#945;, respectively) with assigned genes (the <italic>A</italic><sub><italic>i</italic></sub><italic>(p</italic><sub><italic>k</italic></sub><italic>)</italic> values for each genome), we extracted, for each genome, only ancestral pillars for which each gene in the pillar had synteny support (i.e., each gene was a neighbor of at least one other gene in that pillar set). Using the outgroup gene from each ancestral pillar as an index, we then merged all of these inferences. Because we required that at least one gene from each genome be in each pillar, the effect of this merging was to limit our analyses to a set of <italic>m</italic> = 7243 and = 3091 ancestral pillars for At-&#945; and &#961;, respectively. However, those pillars have shared syntenic support across all genomes. The optimal ancestral order for each extant genome differs, so once the ancestral pillars were assembled, we inferred a globally-optimal ancestral order <italic>O(AG</italic><sub><italic>1</italic></sub>..<italic>AG</italic><sub><italic>m</italic></sub>), again using simulated annealing. The optimality criterion here was to maximize the number of neighbor relationships, but in this case the <italic>A</italic><sub><italic>i</italic></sub><italic>(p</italic><sub><italic>k</italic></sub><italic>)</italic> assignments were held constant and only <italic>O(AG</italic><sub><italic>1</italic></sub>..<italic>AG</italic><sub><italic>m</italic></sub>) was changed.</p><p>To assess the influence of the ancestral ordering on POInT&#8217;s estimates, we fit the WGD-<italic>bf</italic> model (<xref ref-type="fig" rid="pgen.1007267.g002">Fig 2B</xref>) to both the initial <italic>C</italic>. <italic>violacea</italic> order and to the 10 inferences of <italic>O(AG</italic><sub><italic>1</italic></sub>..<italic>AG</italic><sub><italic>m</italic></sub>) with the largest simulated annealing scores, using the order with the highest likelihood for further analyses (<xref ref-type="supplementary-material" rid="pgen.1007267.s004">S1 Table</xref>). We similarly used the ancestral ordering of highest likelihood for our &#961; analyses.</p><p>We employed a number of models of the fates of the duplicates produced by WGD. <bold>A)</bold> Statistical relationships between the various models for the yeast WGD (blue), At-&#945; (green) and &#961; (brown) events. The simplest model (WGD-<italic>n</italic>) considers only a balanced process of gene loss. From this model, we can either allow duplicate genes to become fixed (for instance by neo- or sub-functionalization, WGD-<italic>f</italic>) or for one of the two parental subgenomes to lose more genes than the other (WGD-<italic>b</italic>). Using a likelihood ratio test (LRT), we find that, for all three WGD events, allowing duplicate fixation significantly improves the fit of the data to the models (<italic>P&lt;</italic>10<sup>&#8722;10</sup>, LRT, <xref ref-type="sec" rid="sec002">Methods</xref>). However, for the yeast dataset, there is no significant evidence for biased fractionation (<italic>P</italic>&gt;0.5, LRT), while for the two plant WGDs, adding it significantly improves the fit (<italic>P&lt;</italic>10<sup>&#8722;10</sup>; LRT). From these two models, we can then allow the other process. Again, for yeast, there is significant evidence for fixation but not biased fractionation (<italic>P&lt;</italic>10<sup>&#8722;10</sup> and <italic>P</italic>&gt;0.5, respectively, LRT) while for At-&#945; and &#961;, there is significant evidence for both (<italic>P</italic>&lt;10<sup>&#8722;10</sup> in each case, LRT). We also tested a model where the biased fractionation parameter &#949; (see panel <bold>B</bold>) was allowed to differ on the shared root branch of the tree (WGD-<italic>b</italic><sub><italic>t</italic></sub><italic>f</italic>) compared to all of the other branches. For the two plant WGD events, there is no significant evidence that the level of biased fractionation differed early in history of the WGD relative to later in time (<italic>P</italic>&#8805;0.19, <italic>Results</italic>). On the other hand, for the yeast WGD, biased fractionation was much more intense soon after the polyploidy event and weakened later (<italic>P</italic> = 0.001; <italic>Results</italic>). <bold>B)</bold> Model states and parameters. Our model has four states, two duplicated ones (<bold>U</bold> = undifferentiated duplicates and <bold>F</bold> = fixed duplicates) and two single copy states (<bold>S</bold><sub><bold>1</bold></sub> and <bold>S</bold><sub><bold>2</bold></sub>, corresponding to the two parental subgenomes). The base loss rate (&#945;) is compounded with the estimated time to give the branch lengths of <xref ref-type="fig" rid="pgen.1007267.g001">Fig 1</xref>. The relative fixation rate &#947; (0&#8804;&#947;&lt;&#8734;) gives the rate of duplicate fixation relative to the loss rate &#945;. Likewise, the fractionation bias parameter &#949; (0&#8804;&#949;&#8804;1) gives the excess of preservations from subgenome 1 relative to subgenome 2 (assumed to be the more fractionated subgenome).</p><p>To assess if the fragmentation of synteny blocks was artificially leading us to invoke BF, we also extracted from our full At-&#945; dataset a smaller set of ancestral loci with strong syntenic support, including only pillars with full syntenic support in at least one direction (e.g., two links per pillar per genome). The result was a dataset of <italic>m</italic><sub><italic>h</italic></sub> = 4556 ancestral loci for which we also inferred an optimal ancestral ordering. No such analysis was performed for &#961; due to the small total number of ancestral pillars found. <xref ref-type="table" rid="pgen.1007267.t001">Table 1</xref> gives the parameter estimates from all four datasets for various ancestral orders.</p><p><sup>a</sup>: Number of ancestral loci studied.</p><p><sup>b</sup>: Number of synteny breaks across the polyploid genomes.</p><p><sup>c</sup>: Number of cases where both parental subgenomes showed a synteny break after an ancestral locus (see <xref ref-type="sec" rid="sec002">Methods</xref>).</p><p><sup>d</sup>: ln-likelihood from fitting WGD-<italic>bf</italic> to this ancestral order.</p><p><sup>e</sup>: Maximum likelihood estimate of the relative duplicate fixation rate for this ancestral order (see <xref ref-type="fig" rid="pgen.1007267.g002">Fig 2</xref>).</p><p><sup>f</sup>: Maximum likelihood estimate of the relative rate of retention from the more fractionated subgenome for this ancestral order (see <xref ref-type="fig" rid="pgen.1007267.g002">Fig 2</xref>).</p><p><sup>g</sup>: &#949; not significantly different from 1.0; see <xref ref-type="fig" rid="pgen.1007267.g002">Fig 2</xref>.</p><p>We have previously described POInT [<xref ref-type="bibr" rid="pgen.1007267.ref022">22</xref>, <xref ref-type="bibr" rid="pgen.1007267.ref070">70</xref>], which fits a Markov model to duplicate loci created by WGD. The model has four states (<xref ref-type="fig" rid="pgen.1007267.g002">Fig 2B</xref>), namely <bold>U</bold> (undifferentiated duplicated genes), <bold>F</bold> (fixed duplicate genes) and <bold>S</bold><sub><bold>1</bold></sub> and <bold>S</bold><sub><bold>2</bold></sub> (the single copy states): it is a generalization of a model proposed by Lewis [<xref ref-type="bibr" rid="pgen.1007267.ref071">71</xref>]. Note that once the genes of each post-WGD genome have been assembled into ancestral pillars using the simulated annealing approach above, the <italic>sequences</italic> of the genes of the post-WGD genomes are never used again: all of POInT&#8217;s inferences are based on shared DCS information. Since our prior work, we have completely re-written POInT to allow for user-defined evolutionary models, computing the resulting transition probabilities by exponentiating the user-supplied instantaneous rate matrix [<xref ref-type="bibr" rid="pgen.1007267.ref072">72</xref>]. Using this new version of POInT, we fit five models to our four datasets (two from At-&#945; and one each from the yeast and grass WGD events, <xref ref-type="fig" rid="pgen.1007267.g002">Fig 2</xref>). We used likelihood ratio tests to assess whether more complex models better fit the loss data than did simpler models [<xref ref-type="bibr" rid="pgen.1007267.ref073">73</xref>].</p><p>POInT&#8217;s focus on WGD has advantages over applying more general gene birth-death models to polyploid species [<xref ref-type="bibr" rid="pgen.1007267.ref074">74</xref>, <xref ref-type="bibr" rid="pgen.1007267.ref075">75</xref>]. POInT models the process of duplicate loss and retention jointly across all genomes and along a phylogeny. Hence, the probability of a particular model state at a given ancestral locus is conditioned on all other loci and all other genomes. This conditioning is performed by analogy to the linkage analysis model of Lander and Green [<xref ref-type="bibr" rid="pgen.1007267.ref076">76</xref>] using the hidden-Markov approach of Felsenstein and Churchill [<xref ref-type="bibr" rid="pgen.1007267.ref077">77</xref>]. The states the Markov model considers are the set of <italic>2</italic><sup><italic>n</italic></sup> possible orthology relationships between the <italic>2n</italic> different loci (e.g., 2 duplicated loci in each of <italic>n</italic> genomes). The likelihood of site <italic>i+1</italic> having orthology state <italic>j</italic> given that site <italic>i</italic> has that orthology assignment is (1-&#952;), where &#952; is a small constant estimated from data (0.0004&#8804;&#952;&#8804;0.0081 across these analyses). In cases where there is a double break in gene order in a particular genome, &#952; = 0.5.</p><p>From this model structure, we can infer orthologous chromosomal regions produced by WGD between the genomes studied, along with confidence estimates in these inferences (<xref ref-type="fig" rid="pgen.1007267.g001">Fig 1</xref>). The previous version of POInT did not distinguish between states <bold>S</bold><sub><bold>1</bold></sub> and <bold>S</bold><sub><bold>2</bold></sub>. The result was degeneracy in the inferences of orthologous regions. In other words, assigning the first member of each DCS pair to subgenome 1 and the second to subgenome 2 produced orthology assignment 111111 across the six genomes, which was identical in likelihood to assignment 222222. (The computation is completely analogous for the other two WGD events studied.) Effectively, this degeneracy corresponds to flipping the upper and lower panels of <xref ref-type="fig" rid="pgen.1007267.g001">Fig 1</xref>, because each of the <italic>2</italic><sup><italic>n</italic></sup> possible orthology assignments has an equivalent assignment with all 1s converted to 2s and <italic>vice versa</italic>.</p><p>To model the process of BF, we relaxed this assumption by introducing parameter &#949; (<xref ref-type="fig" rid="pgen.1007267.g002">Fig 2B</xref>). This parameter makes losses to state <bold>S</bold><sub><bold>2</bold></sub> potentially less common than to <bold>S</bold><sub><bold>1</bold></sub>. If BF is present in the data, the maximum likelihood estimate of &#949; will be less than 1.0, and the likelihood of orthology assignment 111111 will no longer be the same as 222222. We can then use the POInT model to estimate the posterior probability of the subgenome assignments (the numbers shown above every column in <xref ref-type="fig" rid="pgen.1007267.g001">Fig 1</xref>) at every pillar. For convenience we refer to the resulting two regions as deriving from allopolyploid parents 1 and 2 [<xref ref-type="bibr" rid="pgen.1007267.ref043">43</xref>], respectively, defining parent 1 as containing genes in state <bold>S</bold><sub><bold>1</bold></sub> (e.g., it is potentially less fractionated), similar to Thomas et al., [<xref ref-type="bibr" rid="pgen.1007267.ref039">39</xref>].</p><p>In previous work in yeast [<xref ref-type="bibr" rid="pgen.1007267.ref015">15</xref>, <xref ref-type="bibr" rid="pgen.1007267.ref022">22</xref>, <xref ref-type="bibr" rid="pgen.1007267.ref070">70</xref>], we found evidence for &#8220;convergent&#8221; gene losses that were phylogenetically independent and yet more often from the same subgenome than could be explained by chance. We modeled these events by adding two duplicated converging states to our model, <bold>C</bold><sub><bold>1</bold></sub> and <bold>C</bold><sub><bold>2</bold></sub>. Gene losses from <bold>C</bold><sub><bold>1</bold></sub> were always to <bold>S</bold><sub><bold>1</bold></sub> and similarly for <bold>C</bold><sub><bold>2</bold></sub>. We fit versions of this model both with (0&#8804; &#949;&#8804;1.0) and without (&#949; = 1) BF to our yeast, grass and At-&#945; data: while these models improved the fit relative to the WGD-<italic>bf</italic> model used here, we present our results in terms of the WGD-<italic>bf</italic> model because both model classes give similar parameter estimates (<xref ref-type="supplementary-material" rid="pgen.1007267.s005">S2 Table</xref>), and the more complex models do not add insight for the questions considered here.</p><p>Because we analyzed only four genomes sharing the grass &#961; event, it was possible to use POInT to test all 15 possible rooted phylogenetic trees for these taxa to assess the dependence of our inferences on the inferred phylogeny. We present our results in terms of the optimal tree, but the global parameter estimates for the WGD-<italic>bf</italic> model were very similar for all topologies (0.061&#8804;&#947;&#8804;0.067; 0.719&#8804;&#949;&#8804;0.739; 0.0061&#8804;&#952;&#8804;0068; <xref ref-type="fig" rid="pgen.1007267.g002">Fig 2</xref>).</p><p>We asked whether genes surviving from one or the other of the subgenomes showed patterns of interconnection in the networks of <italic>Arabidopsis thaliana</italic>. We use the BioGrid database [<xref ref-type="bibr" rid="pgen.1007267.ref078">78</xref>] to extract known protein-protein interactions [<xref ref-type="bibr" rid="pgen.1007267.ref079">79</xref>]. We tested for paucity of interactions between the products of genes from different subgenomes with a randomization approach. We thus compared the number of interactions between gene products from alternative subgenomes in the actual data to this value computed after 1000 randomizations of the subgenome assignments. To assess the degree to which our conclusions were potentially affected by errors in the assignment of genes to subgenomes, we conducted our tests at a range of confidences in subgenome assignment (<xref ref-type="fig" rid="pgen.1007267.g003">Fig 3</xref>).</p><p>We extracted single-copy genes for a range of values of POInT&#8217;s overall confidence in pillar assignments to subgenomes (<italic>x</italic>-axis) and computed the <italic>P</italic>-value for the test of the null hypothesis of no fewer protein-protein interactions between products of genes from alternative subgenomes than expected (<italic>y</italic>-axis; panel <bold>A</bold>: see <xref ref-type="sec" rid="sec002">Methods</xref>). We also computed the frequency of such &#8220;crossing&#8221; interactions relative to interactions between products of the same subgenome (<italic>y</italic>-axis, panel <bold>B</bold>).</p><p>We used the Gene List Analysis tool from the PANTHER classification system [<xref ref-type="bibr" rid="pgen.1007267.ref080">80</xref>] to perform statistical overrepresentation tests to find over/under-represented Gene Ontology (GO) terms associated with biological processes, molecular functions, or cellular components. The input of our analysis consists of two sets of genes: the target list to analyze, and a reference list. The expected number of genes for a GO term in the target list was calculated based on the number of genes with that term in the reference list: binomial statistics for each GO term associated with genes in the target list were then computed from these expectations [<xref ref-type="bibr" rid="pgen.1007267.ref081">81</xref>].</p><p>We first performed an overrepresentation test for 4,086 single copy genes from both subgenomes against the reference set of 4,152 surviving duplicated genes. The over/under represented GO terms in the analysis were filtered with a threshold <italic>P</italic>-value &#8804; 0.01 after Bonferroni correction, and only terms with a fold-enrichment larger than 1.5 (overrepresented) or smaller than 0.67 (underrepresented) are reported. We next compared 2,552 single copy genes from subgenome 1 (dominant) relative to the terms for the 1,534 genes from subgenome 2 (more fractionated) with a similar approach. To compensate for the smaller number of terms found to be enriched in this second analysis, we used an FDR-corrected <italic>P</italic>-value of 0.05 as a threshold. Full lists of all significantly enriched terms for any comparison with associated GO identifiers are given as <xref ref-type="supplementary-material" rid="pgen.1007267.s006">S3</xref>&#8211;<xref ref-type="supplementary-material" rid="pgen.1007267.s008">S5</xref> Tables.</p>