<p>Whole blood samples were collected into EDTA or ACD anticoagulant from AKC-registered dogs. Genomic DNA was extracted using a standard phenol-chloroform extraction protocol [<xref ref-type="bibr" rid="pgen.1006661.ref083">83</xref>]. All procedures were reviewed and approved by the NHGRI Animal Care and Use Committee at the National Institutes of Health.</p><p>Standard breed weights and height were obtained from several sources: weights previously listed in Rimbault et al. [<xref ref-type="bibr" rid="pgen.1006661.ref047">47</xref>] were used, although they were updated if weights specified by the AKC [<xref ref-type="bibr" rid="pgen.1006661.ref084">84</xref>] were different. If the AKC did not specify SBW and SBH, we used data from <italic>Atlas of Dog Breeds of the World</italic> [<xref ref-type="bibr" rid="pgen.1006661.ref016">16</xref>]. SBW and SBH (male + female average) were applied to all samples from the same breed and the values used in this study are listed in <xref ref-type="supplementary-material" rid="pgen.1006661.s006">S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1006661.s007">S2</xref> Tables. Analyses by sex did not change the results, thus we retained the genotypes as a single dataset.</p><p>Genotyping was performed using the Illumina 170K Canine HD SNP array containing approximately 170,000 SNPs distributed across the 38 canine autosomes and the X chromosome. Genotypes were called using Illumina Genome Studio software. In total, 855 dogs, 418 males and 437 females, were genotyped [<xref ref-type="bibr" rid="pgen.1006661.ref021">21</xref>]. Dogs belong to 88 different breeds. Eighty-two breeds with nine to 11 dogs were genotyped and six large dog breeds with four to six dogs genotyped. All samples had a call rate greater than 93% (range: 93.57&#8211;99.98, average: 99.84). SNPs with a minor allele frequency &lt;1% or the presence of &gt;5% missing genotypes were pruned, resulting in a final dataset of 150,895 SNPs that were used for the subsequent GWAS. The GWAS was conducted using the software GEMMA v0.94.1 (Genome-wide Efficient Mixed-Model Association) [<xref ref-type="bibr" rid="pgen.1006661.ref054">54</xref>,<xref ref-type="bibr" rid="pgen.1006661.ref055">55</xref>] as a linear mixed-model software using a centered kinship matrix. Pedigrees of dogs used in the study were verified to avoid inclusion of close relatives, <italic>i</italic>.<italic>e</italic>. none shared a common grandparent. In the two regions of interest, pairwise r<sup>2</sup> values were calculated using Plink v1.07 [<xref ref-type="bibr" rid="pgen.1006661.ref085">85</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Fine mapping at both loci used data from 157 individuals who had undergone WGS and for which the data were published or available online from the Sequence Read Archive (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</ext-link>). Six new WGS recently produced by the NIH Intramural Sequencing Center (NISC) were also included. The latter were produced using the Illumina TruSeq DNAPCR-Free Protocol (Cat.FC-121-3001). Reads were aligned to the CanFam 3.1 reference genome (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=canFam3">http://genome.ucsc.edu/cgi-bin/hgGateway?db=canFam3</ext-link>) using BWA 0.7.13 MEM [<xref ref-type="bibr" rid="pgen.1006661.ref086">86</xref>] and sorted using SAMtools 1.3.1 [<xref ref-type="bibr" rid="pgen.1006661.ref087">87</xref>]. PCR duplicates were marked as secondary reads using PicardTools 2.2.4 (<ext-link ext-link-type="uri" ns0:href="http://github.com/broadinstitute/picard">http://github.com/broadinstitute/picard</ext-link>) for those libraries that were not PCR-free. GATK 3.5 [<xref ref-type="bibr" rid="pgen.1006661.ref088">88</xref>,<xref ref-type="bibr" rid="pgen.1006661.ref089">89</xref>] was used to perform local realignment around putative indels events using 714,278 variants published in [<xref ref-type="bibr" rid="pgen.1006661.ref090">90</xref>] as the training set. A total of 172,254 Illumina Canine HD Chip positions and 2,738,537 dbSNP v131 variants were utilized for base recalibration with GATK 3.5. SNV were called per-individual in gVCF mode of HaplotypeCaller [<xref ref-type="bibr" rid="pgen.1006661.ref091">91</xref>], with subsequent joint-calling across all individuals. Variant quality score recalibration was conducted with GATK best practices and default parameters for SNV and indels separately as follows. Indel recalibration: 714,278 variants as truth and training sets with a prior of six [<xref ref-type="bibr" rid="pgen.1006661.ref090">90</xref>]. SNV recalibration: 172,254 Illumina Canine HD Chip variants (known, training, true, prior = 12); 2,738,537 dbSNP v131 variants (known, true, prior = 8); 3,627,539 published variants from [<xref ref-type="bibr" rid="pgen.1006661.ref092">92</xref>] (known, training, prior = 6). We only used genomes with a sequencing depth &gt;10X and retained only variants with a minimum of two alleles and a minor allele frequency &gt;5%. For locus 1, 6,809 variants met the QC criteria, while 1,997 met the criteria for locus 2. These variants were analyzed using GEMMA v0.94.1 as a linear mixed-model software [<xref ref-type="bibr" rid="pgen.1006661.ref054">54</xref>,<xref ref-type="bibr" rid="pgen.1006661.ref055">55</xref>]. A centered kinship matrix was estimated extracting SNPs from the 163 WGS data using the positions of 147,740 SNPs of the Illumina Canine HD SNP array. DELLY and CNVnator were used to analyze structural variants, including indels, inversions and duplications that were &gt;100 bp in length [<xref ref-type="bibr" rid="pgen.1006661.ref092">92</xref>,<xref ref-type="bibr" rid="pgen.1006661.ref093">93</xref>].</p><p>To confirm the distribution of &#8220;large alleles&#8221; in the <italic>IRS4</italic> and <italic>IGSF1</italic> genes, we genotyped a panel of 512 dogs of 93 breeds and 24 wolves (<xref ref-type="supplementary-material" rid="pgen.1006661.s011">S6 Table</xref>). Primer pairs were designed to target regions that included the variants of interest, and two pairs were specifically designed to reveal the absence/presence of the deletion (<xref ref-type="supplementary-material" rid="pgen.1006661.s012">S7 Table</xref>). Targeted regions were assayed using polymerase chain reaction (PCR) with AmpliTaq Gold. PCR products were purified by ExoSap-It reaction (Affymetrix), and then Sanger sequenced using BigDye Terminator v3.1 (Applied Biosystems). Products from sequencing reactions were run on ABI 3730 DNA analyzer. Sequence traces were analyzed using Phred/Phrap/Consed package [<xref ref-type="bibr" rid="pgen.1006661.ref094">94</xref>&#8211;<xref ref-type="bibr" rid="pgen.1006661.ref096">96</xref>]. The absence/presence of the deletion was detected after migration of the PCR products on a 1% agarose gel followed by staining with ethidium bromide. To analyze the <italic>ACSL4</italic> variant, we sequenced a larger set of 985 unrelated dogs and wild canids, including 24 geographically diverse gray wolves from North America, Europe, and Asia, two coyotes and two red wolves (<xref ref-type="supplementary-material" rid="pgen.1006661.s010">S5 Table</xref>). Three hundred and fifteen of these dogs were included in the dataset used for the initial GWAS.</p><p>To estimate the conservation of mutated codons/nucleotides between mammals, we used both protein and gene sequences from <italic>IRS4</italic>, <italic>IGSF1</italic> and <italic>ACSL4</italic> which were available on Ensembl [<xref ref-type="bibr" rid="pgen.1006661.ref097">97</xref>]. We selected proteins for dog, human, mouse, cat, pig, horse, cow, and megabat and we used SIM [<xref ref-type="bibr" rid="pgen.1006661.ref098">98</xref>] and LALNVIEW [<xref ref-type="bibr" rid="pgen.1006661.ref099">99</xref>] to align sequences.</p>