<p>Flies were raised at 25&#176;C in a 12-h light/12-h dark cycle in 60% relative humidity and maintained on cornmeal, yeast, molasses, and agar medium. Canton S flies were used as the wild-type strain. All transgenic fly lines were backcrossed at least 5 generations into a white Canton S background. A UAS-<italic>Obp69aRNAi</italic> line was obtained from the Vienna RNAi collection, <italic>Obp69a-Minos-GAL4</italic> and <italic>Obp28a</italic>-GAL4 flies were obtained from Bloomington, <italic>Lush</italic>-GAL4 was a gift from Richard Benton, <italic>Nomp-A</italic> GAL4 line was a gift from Yun Doo Chung, and UAS <italic>UNC84</italic>-<italic>GFP</italic>, UAS <italic>mCD8-GFP</italic>, <italic>Or67d</italic>-GAL4 and <italic>Or65a</italic>-GAL4 flies were obtained from HHMI Janelia Research Campus.</p><p><italic>Obp69a</italic> GAL4 was generated by inserting a GAL4 coding sequence into the <italic>Obp69a</italic> Locus using homologous recombination. <italic>Obp69a</italic> 5&#8217; and 3&#8217; homology regions of 3Kb were amplified by High Fidelity PCR Kit (Hy Labs) from wild type Canton-S genomic DNA using the following primers: 5' TGTACTTAGGAAAATGGA 3', 5' TTTTGCTTCTCCCCAAAAATTGCTA 3' for the 5&#8217;HA arm and 5' CGCTAACCAACCTAAATA 3', 5' AATTTGCTCAAGTTCCCCA 3' for the 3&#8217;HA arm. The amplified fragments were cloned into pC31B-JMKS4.2 GAL4-KanR donor vector. This vector contains tdTomato marker under the GMR promoter for visualizing positive donor integration into the MiMIC insertion site of Bloomington stock #35109. Integrant lines were isolated to serve as donors of Obp69a GAL4 DNA substrate for homologous recombination [<xref ref-type="bibr" rid="pgen.1007328.ref056">56</xref>] using Bloomington mobilization stock #6934 containing heat-shock-inducible FLP recombinase and I-SceI endonuclease. Transgenic GFP-eyed flies were individually balanced to establish stable lines. The UAS Obp69a-GFP fused transgenic line was generated accordingly; Obp69a coding region was amplified by High Fidelity PCR Kit (Hy Labs) from wild type Canton-S c-DNA library using the following primers: 5&#8217; gctAGATCTatggttgcaaggcatttta 3&#8217; and 5&#8217; attCTCGAGcccaagtagcactattatc 3&#8217; (uppercase letters represent Bglll and XhoI restriction enzyme sites, respectively). The amplified fragment was cloned in frame up-stream to the E-GFP sequence in the pJFRC81-10XUAS-IVS-Syn21-GFP-p10 vector (addgene, UAS) and sent for injection into y1 w67 c23; P{CaryP{attP2 and y1 M{vas-int.Dm}ZH-2A w*;P{CaryP}attP40 sites (BestGene Inc., USA). All transformants were picked from individually injected flies.</p><p>All behavioral observations were performed at 25&#176;C, 65% relative humidity and at the same time of day (1h after Lights ON) with 3&#8211;4 day old flies unless indicated otherwise.</p><p>Newly eclosed flies were anesthetized and collected into vials with food, where they were housed separately for three days at 25&#176;C in a 12/12 light/dark cycle. During that time flies were exposed to different social conditions: Single housed (One fly in each vial), Group housed (5 same sex flies in each vial), Group mated (5 same sex flies with 5 opposite sex flies in each vial), Single/Group housed with male scents or Single/Group housed with cVA. Following exposure to different housing conditions, experimental flies were lightly anesthetized using CO2 and decapitated using a micro-scalpel at 1&#8211;2 hours after Lights ON. Making sure no antennae were damaged, heads were then frozen in dry ice for total-RNA extraction. We chose to use whole heads and not to surgically remove antennae, as this induces additional variability in the RNA measurements.</p><p>Mesh experiments were performed by inserting experimental flies into a food vial containing an Eppendorf tube with food in it. The top of the Eppendorf tube was cut and replaced with a plastic mesh. This enables exposure of scents of group housing for the experimental flies without experiencing direct physical contact with other flies. Five male odor donor flies were placed in the Eppendorf tube. Test flies were placed in the vial for 3&#8211;4 days at 25&#176;C, 65% humidity. At the end of the experiment, experimental flies were removed for further processing (decapitation for RNA extraction or behavioral analysis).</p><p>Exposure to cVA was performed by adding 1&#956;g or 10&#956;g of cVA dissolved in ethanol onto a filter tip. After the ethanol evaporated, tips were placed in fly food vials and experimental flies were inserted into the vial. cVA was replaced every 24 hours for three days. For expression analysis, heads were frozen (as in the above experimental procedure) for RNA extraction and RT-qPCR analysis. For behavioral tests, three-day old female flies were exposed to cVA either 24h or 1h before behavioral experiments. They were then placed into courtship arenas with virgin 4 days old WT males for behavioral courtship analysis.</p><p>Light induced activation of the red shifted Channel Rhodopsin <italic>UAS-CsChrimson</italic> was achieved by placing fly vials over a red LED covered plate (40 Hz, 650nm, 0.6 lm @20mA). Activation protocol consisted of three 15 min long activation, with resting intervals of 45 min. First activation started at 1h after Lights ON. Flies were then lightly anesthetized and decapitated, heads were frozen 20&#8211;25 minutes after final activation ended.</p><p>Male flies expressing the inward rectifying channel Kir2.1 in Or65a or Or67d neurons and genetic controls were subjected to the following social conditions for three days: single housing or group housing (male cohort), and group housing or group housing with male flies (females cohort). Relative Obp69a levels were determined as previously mentioned. Relative fold change between the different condition for each cohort was calculated by comparing single males to group housed males and comparing group housed females to grouped with males. The fold change ratios were converted to a logarithmic scale of base 2 for linearity. For statistical analysis, One-way ANOVA with Tukey post hoc analysis was used.</p><p>Flies expressing Cs-Chrimson and UAS-Shibire<sup>ts</sup> in Or65a or Or67d neurons were subjected to one of three conditions: (1) Three 15 minutes long optogenetic activations spaced by 45 minutes resting intervals (under constant dark) at constant 22&#176;C serve as positive control. (2) Three 15 minutes long sessions at 28&#176;C under constant dark, spaced by 45 minutes at 22&#176;C, also under constant dark. (3) Three 15 minutes long optogenetic activations at 28&#176;C, spaced by 45 minutes resting intervals with no light and under constant 22&#176;C. 20 minutes following last activation, flies were lightly anesthetized with CO2 and decapitated, as described above. Obp69a relative mRNA levels extracted from whole heads were then quantified using RT-qPCR and compared between the three conditions.</p><p><italic>Courtship analysis</italic>. 3-4-day old female flies that were raised in groups, either na&#239;ve, with male scents through a mesh or with cVA, were inserted into round courtship arenas (0.04 cm<sup>3</sup> in volume) at Lights ON + 1h together with 4-day old WT na&#239;ve males. Courtship arenas were placed in behavior chambers, under controlled temperature and humidity (25&#176;C, 70% humidity). Behavior was recorded for one hour from the introduction of male and female pairs using Point-Grey Flea3 cameras (1080&#215;720 pixels at 30 fps). Latency to copulate was quantified for each pair as total time, starting from first wing vibration the male exhibited and ending in successful copulation. Genetic manipulation of Obp69a in female flies was achieved by driving the expression of specific RNAi or Obp69a-GFP using the <italic>Obp69a-Minos-GAL4</italic>, since this enhancer trap does not affect the basal levels of endogenous <italic>Obp69a</italic>.</p><p><italic>Aggression</italic>. 4&#8211;7 day old pairs of Single housed male flies were put into round aggression arenas (about 0.08 cm<sup>3</sup> in volume). A mixture of agarose and apple juice (1% agarose, 50% apple juice) was inserted into arenas to enhance aggressive behavior (described in [<xref ref-type="bibr" rid="pgen.1007328.ref057">57</xref>]). Aggression arenas were placed in behavior chambers, as described above. Experiments were performed in similar time of day (Lights ON + 1h). Flies&#8217; behavior was recorded for 30 min with Point-Grey Flea3 (1080&#215;720 pixels at 60 fps). Aggressive behavior was later quantified by counting the number of lunges for each pair, and latency as the time from start of experiment to first lunge for each pair. Genetic manipulation of Obp69a in male flies was achieved by driving the expression of specific RNAi or Obp69a-GFP using the <italic>Obp69a-Minos-GAL4</italic>, since this enhancer trap does not affect the basal levels of endogenous <italic>Obp69a</italic>.</p><p>Total RNA was extracted from frozen intact fly heads using TRIZOL reagent. Each sample consisting of 15 frozen heads unless otherwise stated. cDNA was synthesized from total RNA extracts using BIORAD cDNA synthesis kit. cDNA samples were used as templates in a RT-qPCR machine (BIORAD CFX96) using primers for <italic>Obp69a</italic>, <italic>lush</italic>, <italic>cyp6a20</italic>, <italic>est-6</italic>, <italic>obp28a</italic>. relative expression was quantified by &#916;&#916;CT method using <italic>rpl32</italic> as a loading control. Each sample was run in triplicates. Each experiment was repeated at least three time using independent sets of genetic crosses.</p><p><italic>Obp69a</italic> Primers:</p><p>F&#8211;CCTACGATCATAAAGCAGGTGAGA</p><p>R&#8211;TCACCGACTTGTCAATCACATCT.</p><p>Lush primers:</p><p>F&#8211;CGCAGGATCTTATGTGCTACAC</p><p>R&#8211;CATTTCCGGGGGAACCAGAT</p><p><italic>Est-6</italic> primers:</p><p>F&#8212;AGCACGCAGGAGTCATTGGA</p><p>R&#8212;CGTCACCGTCTACAGTTCCAAAA</p><p><italic>Cyp6a20</italic> primers:</p><p>F&#8212;TACTGGAAGCGCCGGGGCATTC</p><p>R&#8212;CCTCATGGTCTCATCAATGACC</p><p><italic>Obp28a</italic> primers:</p><p>F&#8212;ATGCCTATCTGCAGGAAATG</p><p>R&#8212;GCGTCCAGAATTCCGATGTT</p><p><italic>RPL32</italic> primers:</p><p>F&#8212;ATCGATATGCTAAGCTGTCGCA</p><p>R&#8212;GGCATCAGATACTGTCCCTTGAAG</p><p>For RT-qPCR experiments, the average relative expression of three, or when indicated 4,5 or 6 independent experiments (every sample within each repeat consisted of 15 flies), were analyzed using Two-Sample T test (Student&#8217;s T test) for two groups, or by using One-Way ANOVA with Tukey post-hoc analysis for 3&#8211;4 groups. Behavioral experiments were analyzed using One-Way ANOVA with Tukey post-hoc analysis for the groups of experimental flies compared with two genetic controls. When comparing only two groups (<xref ref-type="fig" rid="pgen.1007328.g005">Fig 5D</xref>), Two-Sample T-test (Student&#8217;s T test) was used. All behavioral data were tested for normality. Bonferroni correction for multiple hypothesis testing was applied for each set of related tests which aimed to investigate the same goal of research (indicated in figure legends).</p><p>Fluorescent images were captured using Leica SP8 confocal microscope.</p>