<p>All studies were conducted according to the Declaration of Helsinki. The studies were approved by the local ethical review boards and all study participants provided written informed consent for the collection of samples and subsequent analyses.</p><p>We examined three anthropometric traits related to overall adiposity (BMI) or body fat distribution (WC<sub>adjBMI</sub> and WHR<sub>adjBMI</sub>) [<xref ref-type="bibr" rid="pgen.1006528.ref036">36</xref>] that were available from a large number of studies. Before the association analyses, we calculated sex-specific residuals by adjusting for age, age<sup>2</sup>, BMI (for WC<sub>adjBMI</sub> and WHR<sub>adjBMI</sub> traits only), and other necessary study-specific covariates, such as genotype-derived principal components. Subsequently, we normalized the distributions of sex-specific trait residuals using inverse normal transformation.</p><p>Physical activity was assessed and quantified in various ways in the participating studies of the meta-analysis (<xref ref-type="supplementary-material" rid="pgen.1006528.s006">S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1006528.s011">S6</xref> Tables). Aiming to amass as large a sample size as possible, we harmonized PA by categorizing it into a simple dichotomous variable&#8212;physically inactive vs. active&#8212;that could be derived in a relatively consistent way in all participating studies, and that would be consistent with previous findings on gene-physical activity interactions and the relationship between activity levels and health outcomes. In studies with categorical PA data, individuals were defined inactive if they reported having a sedentary occupation and being sedentary during transport and leisure-time (&lt;1 h of moderate intensity leisure-time or commuting PA per week). All other individuals were defined physically active. Previous studies in large-scale individual cohorts have demonstrated that the interaction between <italic>FTO</italic>, or a BMI-increasing genetic risk score, with physical activity, is most pronounced approximately at this activity level [<xref ref-type="bibr" rid="pgen.1006528.ref006">6</xref>, <xref ref-type="bibr" rid="pgen.1006528.ref037">37</xref>, <xref ref-type="bibr" rid="pgen.1006528.ref038">38</xref>]. In studies with continuous PA data, PA variables were standardized by defining individuals belonging to the lowest sex- and age-adjusted quintile of PA levels as inactive, and all other individuals as active. The study-specific coding of the dichotomous PA variable in each study is described in <xref ref-type="supplementary-material" rid="pgen.1006528.s011">S6 Table</xref>.</p><p>We included 42 studies with genome-wide data, 10 studies with Metabochip data, and eight studies with both genome-wide and Metabochip data. If both genome-wide and Metabochip data were available for the same individual, we only included the genome-wide data (<xref ref-type="supplementary-material" rid="pgen.1006528.s006">S1 Table</xref>). Studies with genome-wide genotyped data used either Affymetrix or Illumina arrays (<xref ref-type="supplementary-material" rid="pgen.1006528.s007">S2 Table</xref>). Following study-specific quality control measures, the genotype data were imputed using the HapMap phase II reference panel (<xref ref-type="supplementary-material" rid="pgen.1006528.s007">S2 Table</xref>). Studies with Metabochip data used the custom Illumina HumanCardio-Metabo BeadChip containing ~195K SNPs designed to support large-scale follow-up of known associations with metabolic and cardiovascular traits [<xref ref-type="bibr" rid="pgen.1006528.ref039">39</xref>]. Each study ran autosomal SNP association analyses with BMI, WC<sub>adjBMI</sub> and WHR<sub>adjBMI</sub> across their array of genetic data using the following linear regression models in men and women separately: 1) active individuals only; 2) inactive individuals only; and 3) active and inactive individuals combined, adjusting for the PA stratum. In studies that included families or closely related individuals, regression coefficients were estimated using a variance component model that modeled relatedness in men and women combined, with sex as a covariate, in addition to the sex-specific analyses. The additive genetic effect for each SNP and phenotype association was estimated using linear regression. For studies with a case-control design (<xref ref-type="supplementary-material" rid="pgen.1006528.s006">S1 Table</xref>), cases and controls were analyzed separately.</p><p>All study-specific files for the three regression models listed above were processed through a standardized quality control protocol using the EasyQC software [<xref ref-type="bibr" rid="pgen.1006528.ref040">40</xref>]. The study-specific quality control measures included checks on file completeness, range of test statistics, allele frequencies, trait transformation, population stratification, and filtering out of low quality data. Checks on file completeness included screening for missing alleles, effect estimates, allele frequencies, and other missing data. Checks on range of test statistics included screening for invalid statistics such as P-values &gt;1 or &lt;0, negative standard errors, or SNPs with low minor allele count (MAC, calculated as MAF*N, where MAF is the minor allele frequency and N is the sample size) and where SNPs with MAC&lt;5 in the inactive or the active group were removed. The correctness of trait transformation to inverse normal was examined by plotting 2/median of the standard error with the square root of the sample size. Population stratification was examined by calculating the study specific genomic control inflation factor (&#955;<sub>GC</sub>) [<xref ref-type="bibr" rid="pgen.1006528.ref041">41</xref>]. If a study had &#955;<sub>GC</sub>&gt;1.1, the study analyst was contacted and asked to revise the analyses by adjusting for principal components. The allele frequencies in each study were examined for strand issues and miscoded alleles by plotting effect allele frequencies against the corresponding allele frequencies from the HapMap2 reference panel. Finally, low quality data were filtered out by removing monomorphic SNPs, imputed SNPs with poor imputation quality (r2_hat &lt;0.3 in MACH [<xref ref-type="bibr" rid="pgen.1006528.ref042">42</xref>], observed/expected dosage variance &lt;0.3 in BIMBAM [<xref ref-type="bibr" rid="pgen.1006528.ref043">43</xref>], proper_info &lt;0.4 in IMPUTE [<xref ref-type="bibr" rid="pgen.1006528.ref044">44</xref>]), and genotyped SNPs with a low call-rate (&lt;95%) or that were out of Hardy-Weinberg equilibrium (P&lt;10<sup>&#8722;6</sup>).</p><p>Beta-coefficients and standard errors were combined by an inverse-variance weighted fixed effect method, implemented using the METAL software [<xref ref-type="bibr" rid="pgen.1006528.ref045">45</xref>]. We performed meta-analyses for each of the three models (active, inactive, active + inactive adjusted for PA) in men only, in women only, and in men and women combined. Study-specific GWAS results were corrected for genomic control using all SNPs. Study-specific Metabochip results as well as the meta-analysis results for GWAS and Metabochip combined were corrected for genomic control using 4,425 SNPs included on the Metabochip for replication of associations with QT-interval, a phenotype not correlated with BMI, WC<sub>adjBMI</sub> or WHR<sub>adjBMI</sub>, after pruning of SNPs within 500 kb of an anthropometry replication SNP. We excluded SNPs that 1) were not available in at least half of the maximum sample size in each stratum; 2) had a heterogeneity I<sup>2</sup> &gt;75%, or 3) were missing chromosomal and base position annotation in dbSNP.</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">To identify SNP&#215;PA interactions, we used the EasyStrata R package [<xref ref-type="bibr" rid="pgen.1006528.ref046">46</xref>] to test for the difference in meta-analyzed beta-coefficients between the active and inactive groups for the association of each SNP with BMI, WC<sub>adjBMI</sub> and WHR<sub>adjBMI</sub>. Easystrata tests for differences in effect estimates between the active and inactive strata by subtracting one beta from the other (&#946;<sub>active</sub>&#8722;&#946;<sub>inactive</sub>,) and dividing by the overall standard error of the difference as follows:
<disp-formula id="pgen.1006528.e004"><alternatives><graphic id="pgen.1006528.e004g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006528.e004.jpg" /><ns1:math id="M4"><ns1:mrow><ns1:msub><ns1:mi>Z</ns1:mi><ns1:mrow><ns1:mi>d</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>f</ns1:mi><ns1:mi>f</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mrow><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mi>a</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow></ns1:msub><ns1:mo>&#8722;</ns1:mo><ns1:msub><ns1:mi>&#946;</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow></ns1:msub></ns1:mrow><ns1:mrow><ns1:msqrt><ns1:mrow><ns1:mtext>&#160;</ns1:mtext><ns1:mi>S</ns1:mi><ns1:msubsup><ns1:mi>E</ns1:mi><ns1:mrow><ns1:mi>a</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>&#8722;</ns1:mo><ns1:mi>S</ns1:mi><ns1:msubsup><ns1:mi>E</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>&#8722;</ns1:mo><ns1:mn>2</ns1:mn><ns1:mo>&#160;</ns1:mo><ns1:mi>r</ns1:mi><ns1:mtext>&#160;</ns1:mtext><ns1:mo>*</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:mi>S</ns1:mi><ns1:msubsup><ns1:mi>E</ns1:mi><ns1:mrow><ns1:mi>a</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup><ns1:mo>*</ns1:mo><ns1:mo>&#160;</ns1:mo><ns1:mi>S</ns1:mi><ns1:msubsup><ns1:mi>E</ns1:mi><ns1:mrow><ns1:mi>i</ns1:mi><ns1:mi>n</ns1:mi><ns1:mi>a</ns1:mi><ns1:mi>c</ns1:mi><ns1:mi>t</ns1:mi><ns1:mi>i</ns1:mi><ns1:mi>v</ns1:mi><ns1:mi>e</ns1:mi></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msubsup></ns1:mrow></ns1:msqrt></ns1:mrow></ns1:mfrac></ns1:mrow></ns1:math></alternatives></disp-formula>
where <italic>r</italic> is the Spearman rank correlation coefficient between &#946;<sub>active</sub> and &#946;<sub>inactive</sub> for all genome-wide SNPs. The joint significance of the SNP main and SNP&#215;PA interaction effects was estimated using the method by Aschard et al. [<xref ref-type="bibr" rid="pgen.1006528.ref016">16</xref>] which is a joint test for genetic main effects and gene-environment interaction effects where gene-environment interaction is calculated as the difference in effect estimates between two exposure strata, accounting for 2 degrees of freedom.</p><p>Approximate conditional analyses were conducted using GCTA version 1.24 [<xref ref-type="bibr" rid="pgen.1006528.ref019">19</xref>]. In the analyses for SNPs identified in our meta-analyses of European-ancestry individuals only, LD correlations between SNPs were estimated using a reference sample comprised of European-ancestry participants of the Atherosclerosis Risk in Communities (ARIC) study. In the analyses for SNPs identified in our meta-analyses of all ancestries combined, the reference sample comprised 93% of European-ancestry individuals and 6% of African ancestry participants from ARIC, as well as 1% of CHB and JPT samples from the HapMap2 panel, to approximate the ancestry mixture in our all ancestry meta-analyses. To test if our identified SNPs were independent secondary signals that fell within 1 Mbp of a previously established signal, we used the GCTA&#8212;cojo-cond command to condition our lead SNPs on each previously established SNP in the same locus.</p><p>The replication analysis for the <italic>CDH12</italic> locus included participants from the EPIC-Norfolk (N<sub>INACTIVE</sub> = 4,755, N<sub>ACTIVE</sub> = 11,526) and Fenland studies (N<sub>INACTIVE</sub> = 1,213, N<sub>ACTIVE</sub> = 4,817), and from the random subcohort of the EPIC-InterAct Consortium (N<sub>INACTIVE</sub> = 2,154, N<sub>ACTIVE</sub> = 6,632). PA stratum-specific estimates of the association of <italic>CDH12</italic> with BMI were assessed and meta-analyzed by fixed effects meta-analyses, and the differences between the PA-strata were determined as described above.</p><p>To identify associations of the novel BMI, WC<sub>adjBMI</sub> or WHR<sub>adjBMI</sub> loci with other complex traits in published GWAS, we extracted previously reported GWAS associations within 500 kb and r<sup>2</sup>&gt;0.6 with any of the lead SNPs, from the GWAS Catalog of the National Human Genome Research Institute and European Bioinformatics Institute [<xref ref-type="bibr" rid="pgen.1006528.ref047">47</xref>] (<xref ref-type="supplementary-material" rid="pgen.1006528.s019">S14 Table</xref>).</p><p>We examined the <italic>cis</italic>-associations of the novel BMI, WC<sub>adjBMI</sub> or WHR<sub>adjBMI</sub> loci with the expression of nearby genes from various tissues by performing a look-up in a library of &gt;100 published expression datasets, as described previously by Zhang et al [<xref ref-type="bibr" rid="pgen.1006528.ref048">48</xref>]. In addition, we examined <italic>cis</italic>-associations using gene expression data derived from fasting peripheral whole blood in the Framingham Heart Study [<xref ref-type="bibr" rid="pgen.1006528.ref049">49</xref>] (n = 5,206), adjusting for PA, age, age<sup>2</sup>, sex and cohort. For each novel locus, we evaluated the association of all transcripts &#177;1 Mb from the lead SNP. To minimize the potential for false positives, we only considered associations where our lead SNP or its proxy (r<sup>2</sup>&gt;0.8) was either the peak SNP associated with the expression of a gene transcript in the region, or in strong LD (r<sup>2</sup>&gt;0.8) with the peak SNP.</p><p>We used the Uncovering Enrichment Through Simulation method to combine the genetic association data with the Roadmap Epigenomics Project segmentation data [<xref ref-type="bibr" rid="pgen.1006528.ref022">22</xref>]. First, 10,000 sets of random SNPs were selected among HapMap2 SNPs with a MAF &gt;0.05 that matched the original input SNPs based on proximity to a transcription start site and the number of LD partners (r<sup>2</sup>&gt;0.8 in individuals of European ancestry in the 1000 Genomes Project). The LD partners were combined with their original lead SNPs to create 10,000 sets of matched random SNPs and their respective LD partners. These sets were intersected with the 15-state ChromHMM data from the Roadmap Epigenomics Project and resultant co-localizations were collapsed from total SNPs down to loci, which were then used to calculate an empirical P value when comparing the original SNPs to the random sets. We examined the enrichment for all loci reaching P&lt;10<sup>&#8722;5</sup> for SNP&#215;PA interaction combined, and for all loci reaching P&lt;5x10<sup>-8</sup> in the PA-adjusted SNP main effect model combined. In addition, we examined the variant-specific overlap with regulatory elements for each of the index SNPs of the novel BMI, WC<sub>adjBMI</sub> and WHR<sub>adjBMI</sub> loci and variants in strong LD (r<sup>2</sup>&gt;0.8).</p><p xmlns:ns0="http://www.w3.org/1999/xlink" xmlns:ns1="http://www.w3.org/1998/Math/MathML">We compared variance explained for BMI, WC<sub>adjBMI</sub> and WHR<sub>adjBMI</sub> between the active and inactive groups using two approaches. First, we used a method previously reported by Kutalik et al [<xref ref-type="bibr" rid="pgen.1006528.ref015">15</xref>], and selected subsets of SNPs based on varying P value thresholds (ranging from 5x10<sup>-8</sup> to 0.05) from the SNP main effect model adjusted for PA. Each subset of SNPs was clumped into independent regions using a physical distance criterion of &lt;500kb, and the most significant lead SNP within the respective region was selected. For each lead SNP, the explained variance was calculated as:
<disp-formula id="pgen.1006528.e005"><alternatives><graphic id="pgen.1006528.e005g" mimetype="image" orientation="portrait" position="anchor" ns0:href="pgen.1006528.e005.jpg" /><ns1:math id="M5"><ns1:mrow><ns1:msup><ns1:mi>r</ns1:mi><ns1:mn>2</ns1:mn></ns1:msup><ns1:mo>=</ns1:mo><ns1:mfrac><ns1:mn>1</ns1:mn><ns1:mrow><ns1:mn>1</ns1:mn><ns1:mo>+</ns1:mo><ns1:mtext>&#160;</ns1:mtext><ns1:mfrac><ns1:mi>N</ns1:mi><ns1:mrow><ns1:msup><ns1:mrow><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mi>&#934;</ns1:mi><ns1:mo>&#8722;</ns1:mo><ns1:mn>1</ns1:mn><ns1:mrow><ns1:mo>(</ns1:mo><ns1:mrow><ns1:mfrac><ns1:mi>P</ns1:mi><ns1:mn>2</ns1:mn></ns1:mfrac></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mo>)</ns1:mo></ns1:mrow></ns1:mrow><ns1:mn>2</ns1:mn></ns1:msup></ns1:mrow></ns1:mfrac></ns1:mrow></ns1:mfrac><ns1:mo>&#8722;</ns1:mo><ns1:mfrac><ns1:mn>1</ns1:mn><ns1:mi>N</ns1:mi></ns1:mfrac></ns1:mrow></ns1:math></alternatives></disp-formula>
in the active and inactive groups separately, where <italic>N</italic> is the sample size and <italic>P</italic> is the P value for SNP main effect in active or inactive strata. Finally, the variance explained by each subset of SNPs in the active and inactive strata was estimated by summing up the variance explained by the SNPs.</p><p>Second, we applied the LD Score regression tool developed by Bulik-Sullivan et al [<xref ref-type="bibr" rid="pgen.1006528.ref014">14</xref>] to quantify the proportion of inflation due to polygenicity (heritability) rather than confounding (cryptic relatedness or population stratification) using meta-analysis summary results. LD Score regression leverages LD between causal and index variants to distinguish true signals by regressing meta-analysis summary results on an &#8216;LD Score&#8217;, i.e. the cumulative genetic variation that an index SNP tags. To obtain heritability estimates by PA strata, we regressed our summary results from the genome-wide meta-analyses of BMI, WC<sub>adjBMI</sub> and WHR<sub>adjBMI</sub>, stratified by PA status (active and inactive), on pre-calculated LD Scores available in HapMap3 reference samples of up to 1,061,094 variants with MAF&#8805;1% and N&gt;10<sup>th</sup> percentile of the total sample size.</p>