<p><italic>Drosophila melanogaster</italic> were raised at 25&#176;C on standard yeast/cornmeal medium (Fly Facility, University of Cambridge, UK) for <italic>w</italic><sup><italic>1118</italic></sup> isogenic flies, and on standard fly agar (Bloomington Drosophila Stock Center, Indiana University, IN) for <italic>w</italic><sup><italic>1118</italic></sup>/<italic>OregonR</italic> hybrid flies. Virgin <italic>w</italic><sup><italic>1118</italic></sup> or modENCODE <italic>OregonR</italic> females were crossed to DrosDel strain males and the non-balancer flies were collected and aged for 3&#8211;5 days.</p><p>We used the following DrosDel lines: <italic>Df(2L)ED105</italic>, <italic>Df(2L)ED1050</italic>, <italic>Df(2L)ED1054</italic>, <italic>Df(2L)ED108</italic>, <italic>Df(2L)ED1109</italic>, <italic>Df(2L)ED1161</italic>, <italic>Df(2L)ED1196</italic>, <italic>Df(2L)ED1200</italic>, <italic>Df(2L)ED1202</italic>, <italic>Df(2L)ED1203</italic>, <italic>Df(2L)ED1226</italic>, <italic>Df(2L)ED123</italic>, <italic>Df(2L)ED125</italic>, <italic>Df(2L)ED12527</italic>, <italic>Df(2L)ED1272</italic>, <italic>Df(2L)ED1303</italic>, <italic>Df(2L)ED1305</italic>, <italic>Df(2L)ED1315</italic>, <italic>Df(2L)ED1317</italic>, <italic>Df(2L)ED134</italic>, <italic>Df(2L)ED136</italic>, <italic>Df(2L)ED1378</italic>, <italic>Df(2L)ED1454</italic>, <italic>Df(2L)ED1466</italic>, <italic>Df(2L)ED1473</italic>, <italic>Df(2L)ED19</italic>, <italic>Df(2L)ED206</italic>, <italic>Df(2L)ED334</italic>, <italic>Df(2L)ED343</italic>, <italic>Df(2L)ED347</italic>, <italic>Df(2L)ED353</italic>, <italic>Df(2L)ED354</italic>, <italic>Df(2L)ED369</italic>, <italic>Df(2L)ED384</italic>, <italic>Df(2L)ED385</italic>, <italic>Df(2L)ED40</italic>, <italic>Df(2L)ED441</italic>, <italic>Df(2L)ED4559</italic>, <italic>Df(2L)ED4651</italic>, <italic>Df(2L)ED475</italic>, <italic>Df(2L)ED479</italic>, <italic>Df(2L)ED489</italic>, <italic>Df(2L)ED49</italic>, <italic>Df(2L)ED50001</italic>, <italic>Df(2L)ED690</italic>, <italic>Df(2L)ED695</italic>, <italic>Df(2L)ED700</italic>, <italic>Df(2L)ED7007</italic>, <italic>Df(2L)ED729</italic>, <italic>Df(2L)ED746</italic>, <italic>Df(2L)ED761</italic>, <italic>Df(2L)ED773</italic>, <italic>Df(2L)ED774</italic>, <italic>Df(2L)ED775</italic>, <italic>Df(2L)ED776</italic>, <italic>Df(2L)ED7762</italic>, <italic>Df(2L)ED777</italic>, <italic>Df(2L)ED778</italic>, <italic>Df(2L)ED779</italic>, <italic>Df(2L)ED784</italic>, <italic>Df(2L)ED7853</italic>, <italic>Df(2L)ED793</italic>, <italic>Df(2L)ED80</italic>, <italic>Df(2L)ED800</italic>, <italic>Df(2L)ED8142</italic>, <italic>Df(2L)ED8185</italic>, <italic>Df(2L)ED87</italic>, <italic>Df(2L)ED929</italic>, and <italic>Df(2L)ED94</italic> were analyzed for both the hybrid (<italic>w</italic><sup><italic>1118</italic></sup>/<italic>OregonR</italic>) and the isogenic (<italic>w</italic><sup><italic>1118</italic></sup>) backgrounds. <italic>Df(2L)ED1102</italic>, <italic>Df(2L)ED1231</italic>, <italic>Df(2L)ED21</italic>, <italic>Df(2L)ED243</italic>, <italic>Df(2L)ED247</italic>, <italic>Df(2L)ED250</italic>, <italic>Df(2L)ED270</italic>, <italic>Df(2L)ED279</italic>, <italic>Df(2L)ED280</italic>, <italic>Df(2L)ED284</italic>, <italic>Df(2L)ED285</italic>, <italic>Df(2L)ED292</italic>, <italic>Df(2L)ED299</italic>, <italic>Df(2L)ED3</italic>, <italic>Df(2L)ED320</italic>, <italic>Df(2L)ED33</italic>, <italic>Df(2L)ED499</italic>, <italic>Df(2L)ED501</italic>, <italic>Df(2L)ED508</italic>, <italic>Df(2L)ED548</italic>, <italic>Df(2L)ED578</italic>, <italic>Df(2L)ED5878</italic>, <italic>Df(2L)ED611</italic>, <italic>Df(2L)ED62</italic>, <italic>Df(2L)ED623</italic>, <italic>Df(2L)ED629</italic>, <italic>Df(2L)ED647</italic>, <italic>Df(2L)ED6569</italic>, <italic>Df(2L)ED678</italic>, and <italic>Df(2L)ED680</italic> were analyzed only in the <italic>w</italic><sup><italic>1118</italic></sup> isogenic background, and <italic>Df(2L)ED1004</italic>, <italic>Df(2L)ED632</italic>, and <italic>Df(2L)ED8186</italic> were analyzed only in the <italic>w</italic><sup><italic>1118</italic></sup>/<italic>OregonR</italic> hybrid background. <italic>Df(2L)ED1050</italic>, <italic>Df(2L)ED123</italic>, and <italic>Df(2L)ED611</italic> showed no clear reduction in hemizygous gene expression raising the possibility that they are not deletions, and complemented multiple mutations in genes that should be deleted. We conclude that these three lines are incorrectly identified in the collection. We removed these from the compensation analysis, although inclusion/exclusion of these <italic>Dfs</italic> did not significantly alter overall compensation values at the rounding levels reported here. Data from these <italic>Dfs</italic> is included in the GEO entries.</p><p>Preparation of RNA sequencing libraries for the <italic>w</italic><sup><italic>1118</italic></sup> isogenic background: flies were mixed with 3mm tungsten carbide beads (Qiagen, West Sussex, UK) and 500&#956;l RNAlater solution (Life Technologies, Grand Island, NY, USA) then homogenized using a Qiagen Tissue Lyser II (Qiagen, West Sussex, UK) for 90 seconds. RNA isolation was performed using the RNeasy 96 kit (Qiagen, Valencia, CA, USA) according to the manufacturer's guide (Protocol for Isolation of Total RNA from animal cells using QIAvac96 vacuum manifold, Cat#19504) except that instead of mixing 150&#956;l of lysis buffer and 150&#956;l of 70% ethanol, 150&#956;l of lysis buffer, 120&#956;l of 100% ethanol and 30&#956;l of the homogenate in RNAlater were used. RNA was quantified using the RiboGreen kit (Life Technology, Grand Island, NY, USA). 100ng of total RNA was used to prepare libraries based on the half-size reaction protocol from the standard Illumina guideline using the TruSeq RNA sample preparation kit v2 (set A and B, Cat # RS-122-2001 and Cat # RS-122-2002, San Diego, CA, USA). 10pg (336 samples) or 500pg (60 samples) of External RNA Controls Consortium RNAs were spiked in during the RNA fragmentation step [<xref ref-type="bibr" rid="pgen.1006295.ref090">90</xref>]. These defined pools of RNAs were transcribed from the plasmids of the Standard Reference Material 2374 (Pool 78A and 78B), obtained from National Institute of Standards and Technology (Gaithersburg, MD, USA). RNA-Seq was carried out on a HiSeq 2000 (Illumina, San Diego, CA, USA)</p><p>Single, 3&#8211;5 day post-eclosion adult male or female flies were collected and crushed in 100&#956;l of RNAlater (Life Technologies, Grand Island, NY). Biological samples were prepared in triplicate and crushed files were frozen at -80&#176;C for long-term storage. A Mini-BeadBeater 96 and 1mm glass beads (Biospec Products, Bartlesville, OK) were used to homogenize flies as follows: 100&#956;l of beads were mixed with the flies in RNAlater in 1ml Axygen 96 well plates (Corning, Union City, CA) and homogenized for 1min (repeated three times) with 2min on ice between homogenization rounds. The RNAlater solution was diluted and tissues were lysed in 600&#956;l of RLT buffer (Qiagen, Valencia, CA) following homogenization. RNA extraction was performed using RNeasy 96 kits (Qiagen, Valencia, CA) according to the manufacturer's handbook (Protocol for Isolation of Total RNA from Animal Cells using spin technology, Cat#19504). Quant-iT RiboGreen (Life Technologies, Grand Island, NY) was used to quantify RNA before library preparation.</p><p>400ng of total RNA from the hybrid background flies in 50&#956;l nuclease-free water were mixed with 50&#956;l Dynabeads Oligo(dT)-25 slurry (Life Technologies, Grand Island, NY), which was pre-rinsed and 2:5 diluted with Binding Buffer (20mM Tris-HCl pH7.5, 1.0M LiCl, 2mM EDTA), and incubated at 65&#176;C for 5min. Samples were cooled on ice for 1min. The bead-RNA complex was incubated at room temperature for 15min. Beads were collected using a magnetic stand, and rinsed with 200&#956;l of Washing Buffer (10mM Tris-HCl pH7.5, 0.15M LiCl, 1mM EDTA) with agitation (1min at 1,000rpm with Thermomixer, Eppendorf, Hauppauge, NY). Beads were collected again and poly A<sup>+</sup> RNA was eluted with Elution Buffer (10mM Tris-HCl pH7.5) at 80&#176;C for 2min. poly A<sup>+</sup> RNA was rebound to the beads by adding 50&#956;l of Binding Buffer and rinsing as above. RNA was eluted from the beads with 16&#956;l of Fragmentation Buffer (1:4 dilution of 5X First Strand Buffer from Protoscript II (New England BioLabs, Ipswich, MA), 500ng of random primers (Life Technologies, Grand Island, NY), and 10pg of External RNA Controls Consortium spike-ins from National Institute of Standards Technology) at 94&#176;C for 6min. Pool 78A of Standard Reference Materials 2374 was used as a spike-in [<xref ref-type="bibr" rid="pgen.1006295.ref033">33</xref>, <xref ref-type="bibr" rid="pgen.1006295.ref090">90</xref>]. Any residual beads were removed using a magnetic stand. Reverse transcription was carried out using 100 units of Protoscript II reverse transcriptase, 5mM DTT, and 625uM dNTPs (Enzymatics, Beverly, MA) with 10 units of SuperRase-in (Life Technologies, Grand Island, NY) at 25&#176;C for 10min, 42&#176;C for 50min, and 70&#176;C for 15min. The DNA-RNA hybrid was cleaned up by incubating with 1.9 volumes of MagNA beads [<xref ref-type="bibr" rid="pgen.1006295.ref091">91</xref>], and a 0.85 volume of ethanol for 15min at room temperature. The DNA-RNA-bead complexes were collected using a magnetic stand, beads were rinsed twice with 200&#956;l of 80% ethanol and air-dried for 5min. DNA-RNA hybrids were eluted using 16&#956;l of Elution Buffer. A strand-specific protocol for second strand synthesis was used as follows: 2.5 units of RNase H, 10 units of DNA polymerase I in 1X Blue Buffer, 10mM DTT, 0.5mM each dATP, dCTP, and dGTP (all from Enzymatics) and 1mM dUTP (ThermoFisher Scientific, Waltham, MA, 1mM final) were added to the first strand synthesis and incubated for 5 hours at 16&#176;C. The double strand DNA products were rinsed with MagNA beads as described above. cDNA was eluted with 16&#956;l of Elution Buffer and the same para-magnetic beads were retained for the next purification steps. DNA was end repaired using NEBNext End Repair Module (New England BioLabs) according to the manufacturer&#8217;s protocol in a 20&#956;l volume, 1.9 volumes of XP buffer [<xref ref-type="bibr" rid="pgen.1006295.ref092">92</xref>] added and the adenylated DNA was eluted with 10&#956;l of Elution Buffer. 1&#956;l of 24 differently indexed TruSeq v2 kit adapters (Illumina, San Diego, CA) were added to the eluate in 24&#956;l ligation solution with T4 DNA ligase (Rapid) (Enzymatics, Beverly, MA) and incubated for 20 min at 20&#176;C. The ligation was stopped by adding 8&#956;l of 0.03M EDTA and DNA cleaned with 24&#956;l of XP buffer as described above. DNA was eluted with 30&#956;l Elution buffer, which was subjected to an additional clean-up with 1 volume of XP buffer. Final elution was done with 12&#956;l of Elution Buffer. In order to remove dUTP incorporated strands of DNA, 5 units of Uracil DNA Glycosylase (New England BioLabs) were added and incubated at 37&#176;C for 30min. PCR reactions were performed with 1.5&#956;l of 10&#956;M P5 and P7 primers (Sequences from Illumina were synthesized by Integrated DNA Technology, Coralville, IA). 2X KAPA HiFi HotStart DNA polymerase (KAPA biosystems, Wilmington, MA) was used in 30&#956;l reactions. PCR parameters were 98&#176;C for 45sec; 14 cycles of 98&#176;C for 15sec; 60&#176;C for 30sec; and 72&#176;C for 30sec followed by 72&#176;C for 5min. The amplified DNA was purified with 1 volume of MagNA beads. Libraries were quantified with Quant-iT PicoGreen (Life Technologies) and sequenced on an Illumina HiSeq 2000 (Illumina).</p><p>Base-calling used CASAVA version 1.8.2 (Illumina, San Diego, CA, USA) and the short reads were mapped onto the <italic>Drosophila</italic> reference genome (Release 5, with no &#8220;chrU&#8221; and &#8220;chrUextra&#8221; scaffolds) using TopHat 2.0.11 [<xref ref-type="bibr" rid="pgen.1006295.ref093">93</xref>] with -g 1 and -G options. An annotation file from FlyBase (5.57) was provided in the mapping process. Cufflinks 2.2.1 [<xref ref-type="bibr" rid="pgen.1006295.ref094">94</xref>] was used to measure gene expression levels in Fragments per Kilobase per Million mapped reads (FPKM), using -G, -b, and -u parameters. FPKM values from intergenic regions were measured as described in [<xref ref-type="bibr" rid="pgen.1006295.ref014">14</xref>]: these were used to determine expression cutoff levels as 0.6829118 for the isogenic background and 0.8140542 for the hybrid genetic background. HTseq 0.6.1p1 [<xref ref-type="bibr" rid="pgen.1006295.ref095">95</xref>] was used with default settings but with the matching strand information to obtain raw reads counts at the gene level. Differential gene expression was assessed with &#8220;voom&#8221; in the R limma package [<xref ref-type="bibr" rid="pgen.1006295.ref096">96</xref>] as in the referenced paper except we filtered based on intergenic FPKM values as above. We also filtered genes that have less than 1 reads from more than 50% of samples for our analysis. TMM (Trimmed Mean of M values)-scale normalization was performed using edgeR [<xref ref-type="bibr" rid="pgen.1006295.ref097">97</xref>] to determine effective library size for the voom analysis. For differential expression analyses, replicates from one genotype were compared to those from all other genotypes in the same sex and genetic background. In determining genes with male- or female-biased expression patterns (<bold><xref ref-type="fig" rid="pgen.1006295.g005">Fig 5E</xref></bold>), the same procedures as above were followed, but compared all males to all females from the isogenic genetic background. Benjamini-Hochberg multiple hypothesis correction of the <italic>p</italic> values was used throughout the manuscript, and adjusted <italic>p</italic> values less than 0.05 were called to be significant changes. FPKM values for the spike-ins were separately determined as abundant transcripts influence estimation of gene expression. In the calculation, the number of reads from both genes and spike-ins, not solely from spike-ins, was used as the denominator of FPKM to infer the lowest detectable expression levels of genes. In describing gene expression changes, expression values for genes that produce polyA<sup>+</sup> mRNA in the gene model were reported, since we followed a poly-A purification protocol. For example, expression of histone transcripts were detected, but the values were highly variable between libraries due to their lack of poly-A tails, and essentially followed the values of residual rRNAs in the sequencing libraries [<xref ref-type="bibr" rid="pgen.1006295.ref098">98</xref>]. Expression results presented were robust to normalization methods used in different bioinformatics tools (FPKM, DESeq, and TMM). The Expectation-Maximization method was used to identification of the fully compensated group of genes in <xref ref-type="fig" rid="pgen.1006295.g004">Fig 4</xref> using &#8220;mclust&#8221; package in R [<xref ref-type="bibr" rid="pgen.1006295.ref099">99</xref>]. In the analysis, two ellipsoidal distributions of equal orientation, &#8220;EEE&#8221;, were used as models for the clustering algorithm.</p><p>TopHat was used to identify fusion transcripts. Potential fusion transcripts that have at least 15 bp long anchor sequences were surveyed. For intra-chromosomal fusion events, we investigated potential fusion transcripts with more than 683 bp distances between juxtaposed regions based on the shortest length of <italic>Dfs</italic> (&#8212;fusion-anchor-length 15 and&#8212;fusion-min-dist 683 parameters).</p><p>The RNA-Seq data showed excellent biological replicate profiles (Pearson's <italic>r</italic> &gt;0.9) in all 396 isogenic samples (<bold><xref ref-type="fig" rid="pgen.1006295.g010">Fig 10A</xref></bold>). However, 10% outliers were observed in the single fly profiles and all samples where Pearson's <italic>r</italic> &lt;0.9 were removed, the two duplicates with the highest correlation were used. The sexual identify of a sample is self-reported in the expression profile (<bold><xref ref-type="fig" rid="pgen.1006295.g010">Fig 10B and 10C</xref></bold>). Detection of low-level gene expression is complicated by the contributions of noise, which vary between libraries. To mitigate this problem reads from intergenic regions were measured (trimmed to account for variation in transcription start and stop sites) and the 95th percentile determined as a low expression cutoff (<bold><xref ref-type="fig" rid="pgen.1006295.g010">Fig 10D</xref></bold>). While some of this intergenic expression may be due to strain-specific transcripts or un-annotated genes, much is likely to be due to noise such as ectopic Pol-II initiation, inclusion of contaminating genomic DNA, sequencing, and/or mapping errors. Ratios and data compression measurements are critical for dosage compensation analysis. Pools of ERCC controls were used in each sample library to produce 1.5:1, 1:1, and 1:1.5 ratios across a &gt; 2<sup>15</sup> input concentration range (<bold><xref ref-type="fig" rid="pgen.1006295.g010">Fig 10E</xref></bold>). Ratio measurements show a clear increase in scatter when input was low. However, even at very low input, there was only modest compression, and no evidence of compression in the useful range for this work.</p><p>A) Box plots that display distribution of Pearson&#8217;s correlation coefficient (<italic>r</italic>) between experimental replicates. The second and third boxes demonstrate <italic>r</italic> values between replicates from the hybrid genetic background, where the second one is after filtering at <italic>r</italic> &lt; 0.9. B, C) All sample-to-all sample pairwise comparisons. The opacity of color indicates Pearson&#8217;s <italic>r</italic> as illustrated in the key. D) RNA-Seq signals from intergenic regions are collectively overlaid as density plots across signal in FPKM. Median of the top 95 percentile in the distribution is indicated (dotted lines). E) Signals from ERCC spike-ins. A pair of spike-in sets is composed with known amount of RNA species that have either 1:1 or 1:1.5 ratios between libraries. In the plot, the black horizontal line indicates expected fold difference from the subpool of spike-in RNAs that have 1:1 ratio between the pair. Red and Blue horizontal line show the expected fold differences from subpools of external RNA that have 1:1.5 ratios. Red, Black, and Blue points indicate the actually measured ratio of each spike-in RNA species after normalization, at different abundances of gene expression (x-axis). Dot lines are trend lines from linear modeling.</p><p>The gene expression profiles generated in this study are available in GEO with accession numbers of GSE61509 (isogenic genetic background) and GSE73920 (hybrid genetic background).</p>